<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>neuroscience</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>neuroscience | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Engineered 3D Immuno-Glial-Neurovascular Human Brain Model
Authors: Stanton, A. E.; Bubnys, A.; Agbas, E.; James, B.; Park, D. S.; Jiang, A.; Pinals, R. L.; Truong, N.; Loon, A.; Staab, C.; Liu, L.; Cerit, O.; Wen, H.-L.; Kellis, M.; Blanchard, J. W.; Langer, R. S.; Tsai, L.-H.
Score: 36.4, Published: 2023-08-17 DOI: 10.1101/2023.08.15.553453
Patient-specific, human-based cellular models that integrate biomimetic BBB, immune, and myelinated neuron components are critically needed to enable translationally relevant and accelerated discovery of neurological disease mechanisms and interventions.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/neuroscience/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="neuroscience" />
<meta property="og:description" content="Engineered 3D Immuno-Glial-Neurovascular Human Brain Model
Authors: Stanton, A. E.; Bubnys, A.; Agbas, E.; James, B.; Park, D. S.; Jiang, A.; Pinals, R. L.; Truong, N.; Loon, A.; Staab, C.; Liu, L.; Cerit, O.; Wen, H.-L.; Kellis, M.; Blanchard, J. W.; Langer, R. S.; Tsai, L.-H.
Score: 36.4, Published: 2023-08-17 DOI: 10.1101/2023.08.15.553453
Patient-specific, human-based cellular models that integrate biomimetic BBB, immune, and myelinated neuron components are critically needed to enable translationally relevant and accelerated discovery of neurological disease mechanisms and interventions." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/neuroscience/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-23T10:37:56+00:00" />
<meta property="article:modified_time" content="2023-08-23T10:37:56+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="neuroscience"/>
<meta name="twitter:description" content="Engineered 3D Immuno-Glial-Neurovascular Human Brain Model
Authors: Stanton, A. E.; Bubnys, A.; Agbas, E.; James, B.; Park, D. S.; Jiang, A.; Pinals, R. L.; Truong, N.; Loon, A.; Staab, C.; Liu, L.; Cerit, O.; Wen, H.-L.; Kellis, M.; Blanchard, J. W.; Langer, R. S.; Tsai, L.-H.
Score: 36.4, Published: 2023-08-17 DOI: 10.1101/2023.08.15.553453
Patient-specific, human-based cellular models that integrate biomimetic BBB, immune, and myelinated neuron components are critically needed to enable translationally relevant and accelerated discovery of neurological disease mechanisms and interventions."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "neuroscience",
      "item": "https://trxiv.yorks0n.com/posts/neuroscience/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "neuroscience",
  "name": "neuroscience",
  "description": "Engineered 3D Immuno-Glial-Neurovascular Human Brain Model\nAuthors: Stanton, A. E.; Bubnys, A.; Agbas, E.; James, B.; Park, D. S.; Jiang, A.; Pinals, R. L.; Truong, N.; Loon, A.; Staab, C.; Liu, L.; Cerit, O.; Wen, H.-L.; Kellis, M.; Blanchard, J. W.; Langer, R. S.; Tsai, L.-H.\nScore: 36.4, Published: 2023-08-17 DOI: 10.1101/2023.08.15.553453\nPatient-specific, human-based cellular models that integrate biomimetic BBB, immune, and myelinated neuron components are critically needed to enable translationally relevant and accelerated discovery of neurological disease mechanisms and interventions.",
  "keywords": [
    
  ],
  "articleBody": " Engineered 3D Immuno-Glial-Neurovascular Human Brain Model\nAuthors: Stanton, A. E.; Bubnys, A.; Agbas, E.; James, B.; Park, D. S.; Jiang, A.; Pinals, R. L.; Truong, N.; Loon, A.; Staab, C.; Liu, L.; Cerit, O.; Wen, H.-L.; Kellis, M.; Blanchard, J. W.; Langer, R. S.; Tsai, L.-H.\nScore: 36.4, Published: 2023-08-17 DOI: 10.1101/2023.08.15.553453\nPatient-specific, human-based cellular models that integrate biomimetic BBB, immune, and myelinated neuron components are critically needed to enable translationally relevant and accelerated discovery of neurological disease mechanisms and interventions. By engineering a brain-mimicking 3D hydrogel and co-culturing all six major brain cell types derived from patient iPSCs, we have constructed, characterized, and utilized a multicellular integrated brain (miBrain) immuno-glial-neurovascular model with in vivo-like hallmarks. As proof of principle, here we utilized the miBrain to model Alzheimers Disease pathologies associated with APOE4 genetic risk. APOE4 miBrains differentially exhibit amyloid aggregation, tau phosphorylation, and astrocytic GFAP. Unlike the co-emergent fate specification of glia and neurons in organoids, miBrains integrate independently differentiated cell types in a modular system with unique utility for elucidating cell-type specific contributions to pathogenesis. We here harness this feature to identify that risk factor APOE4 in astrocytes promotes tau pathogenesis and neuronal dysregulation through crosstalk with microglia. One-Sentence SummaryA novel patient-specific brain model with BBB, neuronal, immune, and glial components was developed, characterized, and harnessed to model Alzheimers Disease-associated pathologies and APOE4 genetic risk.\nDecoding reveals the neural representation of held and manipulated musical thoughts\nAuthors: Quiroga-Martinez, D. R.; Fernandez-Rubio, G.; Bonetti, L.; Achyutuni, K. G.; Tzovara, A.; Knight, R. T.; Vuust, P.\nScore: 33.2, Published: 2023-08-17 DOI: 10.1101/2023.08.15.553456\nImagine a song you know by heart. With little effort you could sing it or play it vividly in your mind. However, we are only beginning to understand how the brain represents, holds, and manipulates these musical \"thoughts\". Here, we decoded listened and imagined melodies from MEG brain data (N = 71) to show that auditory regions represent the sensory properties of individual sounds, whereas cognitive control (prefrontal cortex, basal nuclei, thalamus) and episodic memory areas (inferior and medial temporal lobe, posterior cingulate, precuneus) hold and manipulate the melody as an abstract unit. Furthermore, the mental manipulation of a melody systematically changes its neural representation, reflecting the volitional control of auditory images. Our work sheds light on the nature and dynamics of auditory representations and paves the way for future work on neural decoding of auditory imagination. Significance statementImagining vividly a sequence of sounds is a skill that most humans exert with relatively little effort. However, it is unknown how the brain achieves such an outstanding feat. Here, we used decoding techniques and non-invasive electrophysiology to investigate how sequences of sounds are represented in the brain. We report that auditory regions represent the sensory properties of individual sounds while association areas represent melodies as abstract entities. Moreover, we show that mentally manipulating a melody changes its neural representation across the brain. Understanding auditory representations and their volitional control opens the path for future work on decoding of imagined auditory objects and possible applications in cognitive brain computer interfaces.\nA reproducible and generalizable software workflow for analysis of large-scale neuroimaging data collections using BIDS Apps\nAuthors: Zhao, C.; Jarecka, D.; Covitz, S.; Chen, Y.; Eickhoff, S. B.; Fair, D. A.; Franco, A. R.; Halchenko, Y. O.; Hendrickson, T. J.; Hoffstaedter, F.; Houghton, A.; Kiar, G.; Macdonald, A.; Mehta, K.; Milham, M. P.; Salo, T.; Hanke, M.; Ghosh, S. S.; Cieslak, M.; Satterthwaite, T. D.\nScore: 31.5, Published: 2023-08-18 DOI: 10.1101/2023.08.16.552472\nNeuroimaging research faces a crisis of reproducibility. With massive sample sizes and greater data complexity, this problem becomes more acute. Software that operates on imaging data defined using the Brain Imaging Data Structure (BIDS) - BIDS Apps - have provided a substantial advance. However, even using BIDS Apps, a full audit trail of data processing is a necessary prerequisite for fully reproducible research. Obtaining a faithful record of the audit trail is challenging - especially for large datasets. Recently, the FAIRly big framework was introduced as a way to facilitate reproducible processing of large-scale data by leveraging DataLad - a version control system for data management. However, the current implementation of this framework was more of a proof of concept, and could not be immediately reused by other investigators for different use cases. Here we introduce the BIDS App Bootstrap (BABS), a user-friendly and generalizable Python package for reproducible image processing at scale. BABS facilitates the reproducible application of BIDS Apps to large-scale datasets. Leveraging DataLad and the FAIRly big framework, BABS tracks the full audit trail of data processing in a scalable way by automatically preparing all scripts necessary for data processing and version tracking on high performance computing (HPC) systems. Currently, BABS supports jobs submissions and audits on Sun Grid Engine (SGE) and Slurm HPCs with a parsimonious set of programs. To demonstrate its scalability, we applied BABS to data from the Healthy Brain Network (HBN; n=2,565). Taken together, BABS allows reproducible and scalable image processing and is broadly extensible via an open-source development model.\nSituating the parietal memory network in the context of multiple parallel distributed networks using high-resolution functional connectivity\nAuthors: Kwon, Y. H.; Salvo, J. J.; Anderson, N.; Holubecki, A. M.; Lakshman, M.; Yoo, K.; Kay, K.; Gratton, C.; Braga, R. M.\nScore: 29.1, Published: 2023-08-17 DOI: 10.1101/2023.08.16.553585\nA principle of brain organization is that networks serving higher cognitive functions are widely distributed across the brain. One exception has been the parietal memory network (PMN), which plays a role in recognition memory but is often defined as being restricted to posteromedial association cortex. We hypothesized that high-resolution estimates of the PMN would reveal small regions that had been missed by prior approaches. High-field 7T functional magnetic resonance imaging (fMRI) data from extensively sampled participants was used to define the PMN within individuals. The PMN consistently extended beyond the core posteromedial set to include regions in the inferior parietal lobule; rostral, dorsal, medial, and ventromedial prefrontal cortex; the anterior insula; and ramus marginalis of the cingulate sulcus. The results suggest that, when fine-scale anatomy is considered, the PMN matches the expected distributed architecture of other association networks, reinforcing that parallel distributed networks are an organizing principle of association cortex.\nTrigeminal innervation and tactile responses in mouse tongue\nAuthors: Zhang, L.; Nagel, M.; Olson, W. P.; Chesler, A. T.; O'Connor, D. H.\nScore: 26.8, Published: 2023-08-19 DOI: 10.1101/2023.08.17.553449\nThe mammalian tongue is richly innervated with somatosensory, gustatory and motor fibers. These form the basis of many ethologically important functions such as eating, speaking and social grooming. Despite its high tactile acuity and sensitivity, the neural basis of tongue mechanosensation remains largely mysterious. Here we explored the organization of mechanosensory afferents in the tongue and found that each lingual papilla is innervated by Piezo2+ trigeminal neurons. Notably, each fungiform papilla contained highly specialized ring-like sensory neuron terminations that circumscribe the taste buds. Myelinated lingual afferents in the mouse lingual papillae did not form corpuscular sensory end organs but rather had only free nerve endings. In vivo single-unit recordings from the trigeminal ganglion revealed two types of lingual low-threshold mechanoreceptors (LTMRs) with conduction velocities in the A{delta} range or above and distinct response properties: intermediately adapting (IA) units and rapidly adapting (RA) units. IA units were sensitive to static indentation and stroking, while RA units had a preference for tangential forces applied by stroking. Lingual LTMRs were not directly responsive to rapid cooling or chemicals that can induce astringent or numbing sensations. Genetic labeling of lingual afferents in the tongue revealed at least two types of nerve terminal patterns, involving dense innervation of individual fungiform papillae by multiple putatively distinct afferents, and relatively sparse innervation of filiform papillae. Together, our results indicate that fungiform papillae are mechanosensory structures, while suggesting a simple model that links the functional and anatomical properties of tactile sensory neurons in the tongue.\nRetinal origin of orientation but not direction selective maps in the superior colliculus\nAuthors: de Malmazet, D.; Kuhn, N.; Li, C.; Farrow, K.\nScore: 21.5, Published: 2023-08-19 DOI: 10.1101/2023.08.18.553874\nNeurons in the mouse superior colliculus (\"colliculus\") are arranged in ordered spatial maps. While orientationselective (OS) neurons form a concentric map aligned to the center of vision, direction-selective (DS) neurons are arranged in patches with changing preferences across the visual field. It remains unclear if these maps are a consequence of feed-forward input from the retina or local computations in the colliculus. To determine whether these maps originate in the retina, we mapped the local and global distribution of orientation- and direction-selective retinal ganglion cell boutons using in-vivo two-photon calcium imaging. We found that OS boutons formed patches that matched the distribution of OS neurons within the colliculus. DS boutons displayed less regional specializations, better reflecting the organization of DS neurons in the retina. Both eyes convey similar orientation but different DS inputs to the colliculus, as shown in recordings from retinal explants. These data demonstrate that orientation and direction maps within the colliculus are independent, implying that orientation maps are inherited from the retina, but direction maps require local circuits.\nTemporal regularities shape perceptual decisions and striatal dopamine signals\nAuthors: Fritsche, M.; Majumdar, A.; Strickland, L.; Liebana Garcia, S.; Bogacz, R.; Lak, A.\nScore: 21.1, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553829\nPerceptual decisions should depend on sensory evidence. However, such decisions are also influenced by past choices and outcomes. These choice history biases may reflect advantageous strategies to exploit temporal regularities of natural environments. However, it is unclear whether and how observers can adapt their choice history biases to different temporal regularities, to exploit the multitude of temporal correlations that exist in nature. Here, we show that mice adapt their perceptual choice history biases to different temporal regularities. This adaptation is well captured by a normative reinforcement learning algorithm with multi-trial belief states, comprising both current trial sensory and previous trial memory states. We demonstrate that striatal dopamine tracks predictions of the model and behavior, pointing towards the involvement of dopamine in forming adaptive history biases. Our results reveal the adaptive nature of perceptual choice history biases, and shed light on their underlying computational principles and neural implementation.\nDaytime experiences shape neural activity and dream content in the sleeping brain\nAuthors: Kumral, D.; Palmieri, J.; Gais, S.; Schoenauer, M.\nScore: 69.9, Published: 2023-07-29 DOI: 10.1101/2023.07.29.551087\nLearning-related brain activity patterns are replayed during sleep, and memories of recent experiences appear in our dreams. The connection between these phenomena, however, remains unclear. We investigated whether memory reinstatement during sleep contributes to dreaming. Participants listened to audiobooks before falling asleep. We could determine which audiobook they had studied based on dream reports collected during the night. Audiobook content was also reinstated at the neural level, in high-density EEG recordings. Brain activity during rapid eye movement sleep, particularly in the high-frequency beta range, carried information about the audiobook and simultaneously benefitted memory retention. Crucially, when the learning condition was manifest in neural activity, it also emerged in dreams. Reprocessing of daytime experiences during sleep thus shapes our brain activity, our dreams, and our memories.\nLocal activation of CA1 pyramidal cells induces theta phase precession\nAuthors: Sloin, H.; Spivak, L.; Levi, A.; Gattegno, R.; Someck, S.; Stark, E.\nScore: 17.4, Published: 2023-08-21 DOI: 10.1101/2023.08.19.553985\nHippocampal theta phase precession is involved in spatiotemporal coding and generating multineural spike sequences, but how precession originates remains unresolved. To determine whether precession can be generated directly in CA1 and disambiguate multiple competing mechanisms, we used optogenetic activation to impose artificial place fields in pyramidal cells of mice running on a linear track. More than a third of the CA1 artificial fields exhibited synthetic precession that persisted for a full cycle. In contrast, artificial fields in the parietal cortex did not exhibit synthetic precession. The findings are incompatible with precession models based on inheritance, spreading activation, dual-input, or inhibition-excitation summation. Thus, a precession generator resides locally within CA1.\nRegional Differences in Progenitor Consumption Dynamics Shape Brain Growth during Development\nAuthors: Baumann, N.; Wagener, R.; Javed, A.; Abe, P.; Lopes, A.; Lavalley, A.; Fuciec, D.; Magrinelli, E.; Fievre, S.; Jabaudon, D.\nScore: 16.9, Published: 2023-08-22 DOI: 10.1101/2023.08.21.553891\nDeveloping mammalian brains are characterized by disproportionate growth of the forebrain compared to other regions. How this localized expansion occurs is, however, largely unknown. To address this, we identified region-specific neurogenic patterns by creating a single-cell-resolution birthdate atlas of the mouse brain (https://www.neurobirth.org). We report that in forebrain regions, neurogenesis is sustained compared to the hindbrain, where neurogenesis is transient and limited to early brain development. Sustained forebrain neurogenesis reflects lengthened cell cycle and reduced consumptive divisions of ventricular zone progenitors, resulting in a preserved germinal cell pool. Using single-cell RNA sequencing, we identify functional molecular programs of ventricular zone progenitors that spatially and temporally regulate progenitor cycling properties, including through loss-of-function of the forebrain-enriched mitochondrial membrane protein Fam210b. These results reveal a parsimonious mechanism to locally regulate neuronal production, in which the time window during which progenitors generate cells is a critical determinant of region-specific brain expansion.\n",
  "wordCount" : "2161",
  "inLanguage": "en",
  "datePublished": "2023-08-23T10:37:56Z",
  "dateModified": "2023-08-23T10:37:56Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/neuroscience/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      neuroscience
    </h1>
    <div class="post-meta"><span>updated on August 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553453">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553453" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553453">
        <p class="paperTitle">Engineered 3D Immuno-Glial-Neurovascular Human Brain Model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553453" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553453" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stanton, A. E.; Bubnys, A.; Agbas, E.; James, B.; Park, D. S.; Jiang, A.; Pinals, R. L.; Truong, N.; Loon, A.; Staab, C.; Liu, L.; Cerit, O.; Wen, H.-L.; Kellis, M.; Blanchard, J. W.; Langer, R. S.; Tsai, L.-H.</p>
        <p class="info">Score: 36.4, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553453' target='https://doi.org/10.1101/2023.08.15.553453'> 10.1101/2023.08.15.553453</a></p>
        <p class="abstract">Patient-specific, human-based cellular models that integrate biomimetic BBB, immune, and myelinated neuron components are critically needed to enable translationally relevant and accelerated discovery of neurological disease mechanisms and interventions. By engineering a brain-mimicking 3D hydrogel and co-culturing all six major brain cell types derived from patient iPSCs, we have constructed, characterized, and utilized a multicellular integrated brain (miBrain) immuno-glial-neurovascular model with in vivo-like hallmarks. As proof of principle, here we utilized the miBrain to model Alzheimers Disease pathologies associated with APOE4 genetic risk. APOE4 miBrains differentially exhibit amyloid aggregation, tau phosphorylation, and astrocytic GFAP. Unlike the co-emergent fate specification of glia and neurons in organoids, miBrains integrate independently differentiated cell types in a modular system with unique utility for elucidating cell-type specific contributions to pathogenesis. We here harness this feature to identify that risk factor APOE4 in astrocytes promotes tau pathogenesis and neuronal dysregulation through crosstalk with microglia.

One-Sentence SummaryA novel patient-specific brain model with BBB, neuronal, immune, and glial components was developed, characterized, and harnessed to model Alzheimers Disease-associated pathologies and APOE4 genetic risk.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553456">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553456" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553456">
        <p class="paperTitle">Decoding reveals the neural representation of held and manipulated musical thoughts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553456" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553456" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Quiroga-Martinez, D. R.; Fernandez-Rubio, G.; Bonetti, L.; Achyutuni, K. G.; Tzovara, A.; Knight, R. T.; Vuust, P.</p>
        <p class="info">Score: 33.2, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553456' target='https://doi.org/10.1101/2023.08.15.553456'> 10.1101/2023.08.15.553456</a></p>
        <p class="abstract">Imagine a song you know by heart. With little effort you could sing it or play it vividly in your mind. However, we are only beginning to understand how the brain represents, holds, and manipulates these musical &#34;thoughts&#34;. Here, we decoded listened and imagined melodies from MEG brain data (N = 71) to show that auditory regions represent the sensory properties of individual sounds, whereas cognitive control (prefrontal cortex, basal nuclei, thalamus) and episodic memory areas (inferior and medial temporal lobe, posterior cingulate, precuneus) hold and manipulate the melody as an abstract unit. Furthermore, the mental manipulation of a melody systematically changes its neural representation, reflecting the volitional control of auditory images. Our work sheds light on the nature and dynamics of auditory representations and paves the way for future work on neural decoding of auditory imagination.

Significance statementImagining vividly a sequence of sounds is a skill that most humans exert with relatively little effort. However, it is unknown how the brain achieves such an outstanding feat. Here, we used decoding techniques and non-invasive electrophysiology to investigate how sequences of sounds are represented in the brain. We report that auditory regions represent the sensory properties of individual sounds while association areas represent melodies as abstract entities. Moreover, we show that mentally manipulating a melody changes its neural representation across the brain. Understanding auditory representations and their volitional control opens the path for future work on decoding of imagined auditory objects and possible applications in cognitive brain computer interfaces.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.552472">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.552472" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.552472">
        <p class="paperTitle">A reproducible and generalizable software workflow for analysis of large-scale neuroimaging data collections using BIDS Apps</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.552472" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.552472" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhao, C.; Jarecka, D.; Covitz, S.; Chen, Y.; Eickhoff, S. B.; Fair, D. A.; Franco, A. R.; Halchenko, Y. O.; Hendrickson, T. J.; Hoffstaedter, F.; Houghton, A.; Kiar, G.; Macdonald, A.; Mehta, K.; Milham, M. P.; Salo, T.; Hanke, M.; Ghosh, S. S.; Cieslak, M.; Satterthwaite, T. D.</p>
        <p class="info">Score: 31.5, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.552472' target='https://doi.org/10.1101/2023.08.16.552472'> 10.1101/2023.08.16.552472</a></p>
        <p class="abstract">Neuroimaging research faces a crisis of reproducibility. With massive sample sizes and greater data complexity, this problem becomes more acute. Software that operates on imaging data defined using the Brain Imaging Data Structure (BIDS) - BIDS Apps - have provided a substantial advance. However, even using BIDS Apps, a full audit trail of data processing is a necessary prerequisite for fully reproducible research. Obtaining a faithful record of the audit trail is challenging - especially for large datasets. Recently, the FAIRly big framework was introduced as a way to facilitate reproducible processing of large-scale data by leveraging DataLad - a version control system for data management. However, the current implementation of this framework was more of a proof of concept, and could not be immediately reused by other investigators for different use cases. Here we introduce the BIDS App Bootstrap (BABS), a user-friendly and generalizable Python package for reproducible image processing at scale. BABS facilitates the reproducible application of BIDS Apps to large-scale datasets. Leveraging DataLad and the FAIRly big framework, BABS tracks the full audit trail of data processing in a scalable way by automatically preparing all scripts necessary for data processing and version tracking on high performance computing (HPC) systems. Currently, BABS supports jobs submissions and audits on Sun Grid Engine (SGE) and Slurm HPCs with a parsimonious set of programs. To demonstrate its scalability, we applied BABS to data from the Healthy Brain Network (HBN; n=2,565). Taken together, BABS allows reproducible and scalable image processing and is broadly extensible via an open-source development model.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553585">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553585" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553585">
        <p class="paperTitle">Situating the parietal memory network in the context of multiple parallel distributed networks using high-resolution functional connectivity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553585" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553585" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kwon, Y. H.; Salvo, J. J.; Anderson, N.; Holubecki, A. M.; Lakshman, M.; Yoo, K.; Kay, K.; Gratton, C.; Braga, R. M.</p>
        <p class="info">Score: 29.1, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553585' target='https://doi.org/10.1101/2023.08.16.553585'> 10.1101/2023.08.16.553585</a></p>
        <p class="abstract">A principle of brain organization is that networks serving higher cognitive functions are widely distributed across the brain. One exception has been the parietal memory network (PMN), which plays a role in recognition memory but is often defined as being restricted to posteromedial association cortex. We hypothesized that high-resolution estimates of the PMN would reveal small regions that had been missed by prior approaches. High-field 7T functional magnetic resonance imaging (fMRI) data from extensively sampled participants was used to define the PMN within individuals. The PMN consistently extended beyond the core posteromedial set to include regions in the inferior parietal lobule; rostral, dorsal, medial, and ventromedial prefrontal cortex; the anterior insula; and ramus marginalis of the cingulate sulcus. The results suggest that, when fine-scale anatomy is considered, the PMN matches the expected distributed architecture of other association networks, reinforcing that parallel distributed networks are an organizing principle of association cortex.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553449">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553449" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553449">
        <p class="paperTitle">Trigeminal innervation and tactile responses in mouse tongue</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553449" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553449" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, L.; Nagel, M.; Olson, W. P.; Chesler, A. T.; O&#39;Connor, D. H.</p>
        <p class="info">Score: 26.8, Published: 2023-08-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553449' target='https://doi.org/10.1101/2023.08.17.553449'> 10.1101/2023.08.17.553449</a></p>
        <p class="abstract">The mammalian tongue is richly innervated with somatosensory, gustatory and motor fibers. These form the basis of many ethologically important functions such as eating, speaking and social grooming. Despite its high tactile acuity and sensitivity, the neural basis of tongue mechanosensation remains largely mysterious. Here we explored the organization of mechanosensory afferents in the tongue and found that each lingual papilla is innervated by Piezo2&#43; trigeminal neurons. Notably, each fungiform papilla contained highly specialized ring-like sensory neuron terminations that circumscribe the taste buds. Myelinated lingual afferents in the mouse lingual papillae did not form corpuscular sensory end organs but rather had only free nerve endings. In vivo single-unit recordings from the trigeminal ganglion revealed two types of lingual low-threshold mechanoreceptors (LTMRs) with conduction velocities in the A{delta} range or above and distinct response properties: intermediately adapting (IA) units and rapidly adapting (RA) units. IA units were sensitive to static indentation and stroking, while RA units had a preference for tangential forces applied by stroking. Lingual LTMRs were not directly responsive to rapid cooling or chemicals that can induce astringent or numbing sensations. Genetic labeling of lingual afferents in the tongue revealed at least two types of nerve terminal patterns, involving dense innervation of individual fungiform papillae by multiple putatively distinct afferents, and relatively sparse innervation of filiform papillae. Together, our results indicate that fungiform papillae are mechanosensory structures, while suggesting a simple model that links the functional and anatomical properties of tactile sensory neurons in the tongue.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553874">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553874" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553874">
        <p class="paperTitle">Retinal origin of orientation but not direction selective maps in the superior colliculus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553874" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553874" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de Malmazet, D.; Kuhn, N.; Li, C.; Farrow, K.</p>
        <p class="info">Score: 21.5, Published: 2023-08-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553874' target='https://doi.org/10.1101/2023.08.18.553874'> 10.1101/2023.08.18.553874</a></p>
        <p class="abstract">Neurons in the mouse superior colliculus (&#34;colliculus&#34;) are arranged in ordered spatial maps. While orientationselective (OS) neurons form a concentric map aligned to the center of vision, direction-selective (DS) neurons are arranged in patches with changing preferences across the visual field. It remains unclear if these maps are a consequence of feed-forward input from the retina or local computations in the colliculus. To determine whether these maps originate in the retina, we mapped the local and global distribution of orientation- and direction-selective retinal ganglion cell boutons using in-vivo two-photon calcium imaging. We found that OS boutons formed patches that matched the distribution of OS neurons within the colliculus. DS boutons displayed less regional specializations, better reflecting the organization of DS neurons in the retina. Both eyes convey similar orientation but different DS inputs to the colliculus, as shown in recordings from retinal explants. These data demonstrate that orientation and direction maps within the colliculus are independent, implying that orientation maps are inherited from the retina, but direction maps require local circuits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553829">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553829" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553829">
        <p class="paperTitle">Temporal regularities shape perceptual decisions and striatal dopamine signals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553829" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553829" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fritsche, M.; Majumdar, A.; Strickland, L.; Liebana Garcia, S.; Bogacz, R.; Lak, A.</p>
        <p class="info">Score: 21.1, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553829' target='https://doi.org/10.1101/2023.08.18.553829'> 10.1101/2023.08.18.553829</a></p>
        <p class="abstract">Perceptual decisions should depend on sensory evidence. However, such decisions are also influenced by past choices and outcomes. These choice history biases may reflect advantageous strategies to exploit temporal regularities of natural environments. However, it is unclear whether and how observers can adapt their choice history biases to different temporal regularities, to exploit the multitude of temporal correlations that exist in nature. Here, we show that mice adapt their perceptual choice history biases to different temporal regularities. This adaptation is well captured by a normative reinforcement learning algorithm with multi-trial belief states, comprising both current trial sensory and previous trial memory states. We demonstrate that striatal dopamine tracks predictions of the model and behavior, pointing towards the involvement of dopamine in forming adaptive history biases. Our results reveal the adaptive nature of perceptual choice history biases, and shed light on their underlying computational principles and neural implementation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.29.551087">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.29.551087" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.29.551087">
        <p class="paperTitle">Daytime experiences shape neural activity and dream content in the sleeping brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.29.551087" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.29.551087" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kumral, D.; Palmieri, J.; Gais, S.; Schoenauer, M.</p>
        <p class="info">Score: 69.9, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.29.551087' target='https://doi.org/10.1101/2023.07.29.551087'> 10.1101/2023.07.29.551087</a></p>
        <p class="abstract">Learning-related brain activity patterns are replayed during sleep, and memories of recent experiences appear in our dreams. The connection between these phenomena, however, remains unclear. We investigated whether memory reinstatement during sleep contributes to dreaming. Participants listened to audiobooks before falling asleep. We could determine which audiobook they had studied based on dream reports collected during the night. Audiobook content was also reinstated at the neural level, in high-density EEG recordings. Brain activity during rapid eye movement sleep, particularly in the high-frequency beta range, carried information about the audiobook and simultaneously benefitted memory retention. Crucially, when the learning condition was manifest in neural activity, it also emerged in dreams. Reprocessing of daytime experiences during sleep thus shapes our brain activity, our dreams, and our memories.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.19.553985">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.19.553985" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.19.553985">
        <p class="paperTitle">Local activation of CA1 pyramidal cells induces theta phase precession</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.19.553985" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.19.553985" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sloin, H.; Spivak, L.; Levi, A.; Gattegno, R.; Someck, S.; Stark, E.</p>
        <p class="info">Score: 17.4, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.19.553985' target='https://doi.org/10.1101/2023.08.19.553985'> 10.1101/2023.08.19.553985</a></p>
        <p class="abstract">Hippocampal theta phase precession is involved in spatiotemporal coding and generating multineural spike sequences, but how precession originates remains unresolved. To determine whether precession can be generated directly in CA1 and disambiguate multiple competing mechanisms, we used optogenetic activation to impose artificial place fields in pyramidal cells of mice running on a linear track. More than a third of the CA1 artificial fields exhibited synthetic precession that persisted for a full cycle. In contrast, artificial fields in the parietal cortex did not exhibit synthetic precession. The findings are incompatible with precession models based on inheritance, spreading activation, dual-input, or inhibition-excitation summation. Thus, a precession generator resides locally within CA1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.553891">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.553891" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.553891">
        <p class="paperTitle">Regional Differences in Progenitor Consumption Dynamics Shape Brain Growth during Development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.553891" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.553891" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baumann, N.; Wagener, R.; Javed, A.; Abe, P.; Lopes, A.; Lavalley, A.; Fuciec, D.; Magrinelli, E.; Fievre, S.; Jabaudon, D.</p>
        <p class="info">Score: 16.9, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.553891' target='https://doi.org/10.1101/2023.08.21.553891'> 10.1101/2023.08.21.553891</a></p>
        <p class="abstract">Developing mammalian brains are characterized by disproportionate growth of the forebrain compared to other regions. How this localized expansion occurs is, however, largely unknown. To address this, we identified region-specific neurogenic patterns by creating a single-cell-resolution birthdate atlas of the mouse brain (https://www.neurobirth.org). We report that in forebrain regions, neurogenesis is sustained compared to the hindbrain, where neurogenesis is transient and limited to early brain development. Sustained forebrain neurogenesis reflects lengthened cell cycle and reduced consumptive divisions of ventricular zone progenitors, resulting in a preserved germinal cell pool. Using single-cell RNA sequencing, we identify functional molecular programs of ventricular zone progenitors that spatially and temporally regulate progenitor cycling properties, including through loss-of-function of the forebrain-enriched mitochondrial membrane protein Fam210b. These results reveal a parsimonious mechanism to locally regulate neuronal production, in which the time window during which progenitors generate cells is a critical determinant of region-specific brain expansion.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
