<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>neuroscience</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>neuroscience | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A reproducible and generalizable software workflow for analysis of large-scale neuroimaging data collections using BIDS Apps
Authors: Zhao, C.; Jarecka, D.; Covitz, S.; Chen, Y.; Eickhoff, S. B.; Fair, D. A.; Franco, A. R.; Halchenko, Y. O.; Hendrickson, T. J.; Hoffstaedter, F.; Houghton, A.; Kiar, G.; Macdonald, A.; Mehta, K.; Milham, M. P.; Salo, T.; Hanke, M.; Ghosh, S. S.; Cieslak, M.; Satterthwaite, T. D.
Score: 33.7, Published: 2023-08-18 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/neuroscience/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="neuroscience" />
<meta property="og:description" content="A reproducible and generalizable software workflow for analysis of large-scale neuroimaging data collections using BIDS Apps
Authors: Zhao, C.; Jarecka, D.; Covitz, S.; Chen, Y.; Eickhoff, S. B.; Fair, D. A.; Franco, A. R.; Halchenko, Y. O.; Hendrickson, T. J.; Hoffstaedter, F.; Houghton, A.; Kiar, G.; Macdonald, A.; Mehta, K.; Milham, M. P.; Salo, T.; Hanke, M.; Ghosh, S. S.; Cieslak, M.; Satterthwaite, T. D.
Score: 33.7, Published: 2023-08-18 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/neuroscience/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-27T10:37:02+00:00" />
<meta property="article:modified_time" content="2023-08-27T10:37:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="neuroscience"/>
<meta name="twitter:description" content="A reproducible and generalizable software workflow for analysis of large-scale neuroimaging data collections using BIDS Apps
Authors: Zhao, C.; Jarecka, D.; Covitz, S.; Chen, Y.; Eickhoff, S. B.; Fair, D. A.; Franco, A. R.; Halchenko, Y. O.; Hendrickson, T. J.; Hoffstaedter, F.; Houghton, A.; Kiar, G.; Macdonald, A.; Mehta, K.; Milham, M. P.; Salo, T.; Hanke, M.; Ghosh, S. S.; Cieslak, M.; Satterthwaite, T. D.
Score: 33.7, Published: 2023-08-18 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "neuroscience",
      "item": "https://trxiv.yorks0n.com/posts/neuroscience/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "neuroscience",
  "name": "neuroscience",
  "description": "A reproducible and generalizable software workflow for analysis of large-scale neuroimaging data collections using BIDS Apps\nAuthors: Zhao, C.; Jarecka, D.; Covitz, S.; Chen, Y.; Eickhoff, S. B.; Fair, D. A.; Franco, A. R.; Halchenko, Y. O.; Hendrickson, T. J.; Hoffstaedter, F.; Houghton, A.; Kiar, G.; Macdonald, A.; Mehta, K.; Milham, M. P.; Salo, T.; Hanke, M.; Ghosh, S. S.; Cieslak, M.; Satterthwaite, T. D.\nScore: 33.7, Published: 2023-08-18 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " A reproducible and generalizable software workflow for analysis of large-scale neuroimaging data collections using BIDS Apps\nAuthors: Zhao, C.; Jarecka, D.; Covitz, S.; Chen, Y.; Eickhoff, S. B.; Fair, D. A.; Franco, A. R.; Halchenko, Y. O.; Hendrickson, T. J.; Hoffstaedter, F.; Houghton, A.; Kiar, G.; Macdonald, A.; Mehta, K.; Milham, M. P.; Salo, T.; Hanke, M.; Ghosh, S. S.; Cieslak, M.; Satterthwaite, T. D.\nScore: 33.7, Published: 2023-08-18 DOI: 10.1101/2023.08.16.552472\nNeuroimaging research faces a crisis of reproducibility. With massive sample sizes and greater data complexity, this problem becomes more acute. Software that operates on imaging data defined using the Brain Imaging Data Structure (BIDS) - BIDS Apps - have provided a substantial advance. However, even using BIDS Apps, a full audit trail of data processing is a necessary prerequisite for fully reproducible research. Obtaining a faithful record of the audit trail is challenging - especially for large datasets. Recently, the FAIRly big framework was introduced as a way to facilitate reproducible processing of large-scale data by leveraging DataLad - a version control system for data management. However, the current implementation of this framework was more of a proof of concept, and could not be immediately reused by other investigators for different use cases. Here we introduce the BIDS App Bootstrap (BABS), a user-friendly and generalizable Python package for reproducible image processing at scale. BABS facilitates the reproducible application of BIDS Apps to large-scale datasets. Leveraging DataLad and the FAIRly big framework, BABS tracks the full audit trail of data processing in a scalable way by automatically preparing all scripts necessary for data processing and version tracking on high performance computing (HPC) systems. Currently, BABS supports jobs submissions and audits on Sun Grid Engine (SGE) and Slurm HPCs with a parsimonious set of programs. To demonstrate its scalability, we applied BABS to data from the Healthy Brain Network (HBN; n=2,565). Taken together, BABS allows reproducible and scalable image processing and is broadly extensible via an open-source development model.\nRegional Differences in Progenitor Consumption Dynamics Shape Brain Growth during Development\nAuthors: Baumann, N.; Wagener, R.; Javed, A.; Abe, P.; Lopes, A.; Lavalley, A.; Fuciec, D.; Magrinelli, E.; Fievre, S.; Jabaudon, D.\nScore: 29.4, Published: 2023-08-22 DOI: 10.1101/2023.08.21.553891\nDeveloping mammalian brains are characterized by disproportionate growth of the forebrain compared to other regions. How this localized expansion occurs is, however, largely unknown. To address this, we identified region-specific neurogenic patterns by creating a single-cell-resolution birthdate atlas of the mouse brain (https://www.neurobirth.org). We report that in forebrain regions, neurogenesis is sustained compared to the hindbrain, where neurogenesis is transient and limited to early brain development. Sustained forebrain neurogenesis reflects lengthened cell cycle and reduced consumptive divisions of ventricular zone progenitors, resulting in a preserved germinal cell pool. Using single-cell RNA sequencing, we identify functional molecular programs of ventricular zone progenitors that spatially and temporally regulate progenitor cycling properties, including through loss-of-function of the forebrain-enriched mitochondrial membrane protein Fam210b. These results reveal a parsimonious mechanism to locally regulate neuronal production, in which the time window during which progenitors generate cells is a critical determinant of region-specific brain expansion.\nUsing network control theory to study the dynamics of the structural connectome\nAuthors: Parkes, L.; Kim, J. Z.; Stiso, J.; Brynildsen, J. K.; Cieslak, M.; Covitz, S.; Gur, R. E.; Gur, R. C.; Pasqualetti, F.; Shinohara, R. T.; Zhou, D.; Satterthwaite, T. D.; Bassett, D. S.\nScore: 29.3, Published: 2023-08-24 DOI: 10.1101/2023.08.23.554519\nNetwork control theory (NCT) is a simple and powerful tool for studying how network topology informs and constrains dynamics. Compared to other structure-function coupling approaches, the strength of NCT lies in its capacity to predict the patterns of external control signals that may alter dynamics in a desired way. We have extensively developed and validated the application of NCT to the human structural connectome. Through these efforts, we have studied (i) how different aspects of connectome topology affect neural dynamics, (ii) whether NCT outputs cohere with empirical data on brain function and stimulation, and (iii) how NCT outputs vary across development and correlate with behavior and mental health symptoms. In this protocol, we introduce a framework for applying NCT to structural connectomes following two main pathways. Our primary pathway focuses on computing the control energy associated with transitioning between specific neural activity states. Our second pathway focuses on computing average controllability, which indexes nodes' general capacity to control dynamics. We also provide recommendations for comparing NCT outputs against null network models. Finally, we support this protocol with a Python-based software package called network control theory for python (https://github.com/BassettLab/nctpy)\nTrigeminal innervation and tactile responses in mouse tongue\nAuthors: Zhang, L.; Nagel, M.; Olson, W. P.; Chesler, A. T.; O'Connor, D. H.\nScore: 27.3, Published: 2023-08-19 DOI: 10.1101/2023.08.17.553449\nThe mammalian tongue is richly innervated with somatosensory, gustatory and motor fibers. These form the basis of many ethologically important functions such as eating, speaking and social grooming. Despite its high tactile acuity and sensitivity, the neural basis of tongue mechanosensation remains largely mysterious. Here we explored the organization of mechanosensory afferents in the tongue and found that each lingual papilla is innervated by Piezo2+ trigeminal neurons. Notably, each fungiform papilla contained highly specialized ring-like sensory neuron terminations that circumscribe the taste buds. Myelinated lingual afferents in the mouse lingual papillae did not form corpuscular sensory end organs but rather had only free nerve endings. In vivo single-unit recordings from the trigeminal ganglion revealed two types of lingual low-threshold mechanoreceptors (LTMRs) with conduction velocities in the A{delta} range or above and distinct response properties: intermediately adapting (IA) units and rapidly adapting (RA) units. IA units were sensitive to static indentation and stroking, while RA units had a preference for tangential forces applied by stroking. Lingual LTMRs were not directly responsive to rapid cooling or chemicals that can induce astringent or numbing sensations. Genetic labeling of lingual afferents in the tongue revealed at least two types of nerve terminal patterns, involving dense innervation of individual fungiform papillae by multiple putatively distinct afferents, and relatively sparse innervation of filiform papillae. Together, our results indicate that fungiform papillae are mechanosensory structures, while suggesting a simple model that links the functional and anatomical properties of tactile sensory neurons in the tongue.\nMultiplexing cortical brain organoids for the longitudinal dissection of developmental traits at single cell resolution\nAuthors: Caporale, N.; Castaldi, D.; Rigoli, M. T.; Cheroni, C.; Trattaro, S.; Valenti, A.; Bonfanti, M.; Stucchi, S.; Tobon, A. L.; Ricca, D.; Lessi, M.; Pezzali, M.; Vitriolo, A.; Schmid, K.; Heinig, M.; Theis, F. J.; Villa, C. E.; Testa, G.\nScore: 26.4, Published: 2023-08-22 DOI: 10.1101/2023.08.21.553507\nThe combination of brain organoid and single cell omic technologies holds transformative potential to dissect human neurobiology at high resolution and with mechanistic precision. Delivering this promise in the context of human neurodiversity, physiological and pathological alike, requires however a major leap in scalability, given the need for experimental designs that include multiple individuals and, prospectively, population cohorts. To lay the foundation for this, we implemented and benchmarked complementary strategies to multiplex brain organoids. Following an extended longitudinal design with a uniquely informative set of timepoints, we pooled cells from different induced pluripotent stem cell lines either during organoids generation (upstream multiplexing in mosaic models) or before single cell-RNAseq library preparation (downstream multiplexing). We developed a new method, SCanSNP, and an aggregated call to deconvolve organoids cell identities, overcoming current criticalities in doublets prediction and low quality cells identification and improving accuracy over state of the art algorithms. Integrating single cell transcriptomes and analysing cell types across neurodevelopmental stages and multiplexing modalities, we validated the feasibility of both multiplexing methods in charting neurodevelopmental trajectories at high resolution, linking their specificity to genetic variation between individual lines. Together, this multiplexing suite of experimental and computational methods provides an enabling resource for disease modelling at scale and paves the way towards an in vitro epidemiology paradigm.\nRetinal origin of orientation but not direction selective maps in the superior colliculus\nAuthors: de Malmazet, D.; Kuhn, N.; Li, C.; Farrow, K.\nScore: 24.4, Published: 2023-08-19 DOI: 10.1101/2023.08.18.553874\nNeurons in the mouse superior colliculus (\"colliculus\") are arranged in ordered spatial maps. While orientationselective (OS) neurons form a concentric map aligned to the center of vision, direction-selective (DS) neurons are arranged in patches with changing preferences across the visual field. It remains unclear if these maps are a consequence of feed-forward input from the retina or local computations in the colliculus. To determine whether these maps originate in the retina, we mapped the local and global distribution of orientation- and direction-selective retinal ganglion cell boutons using in-vivo two-photon calcium imaging. We found that OS boutons formed patches that matched the distribution of OS neurons within the colliculus. DS boutons displayed less regional specializations, better reflecting the organization of DS neurons in the retina. Both eyes convey similar orientation but different DS inputs to the colliculus, as shown in recordings from retinal explants. These data demonstrate that orientation and direction maps within the colliculus are independent, implying that orientation maps are inherited from the retina, but direction maps require local circuits.\nUltra-high density electrodes improve detection, yield, and cell type specificity of brain recordings\nAuthors: Ye, Z.; Shelton, A. M.; Shaker, J. R.; Boussard, J. M.; Colonell, J.; Minavi, S.; Chen, S.; Windolf, C.; Hurwitz, C.; Namima, T.; Pedraja, F.; Weiss, S.; Raducanu, B.; Ness, T.; Einevoll, G. T.; Laurent, G.; Sawtell, N. B.; Bair, W.; Pasupathy, A.; Mora-Lopez, C.; Dutta, B.; Paninski, L.; Siegle, J. H.; Koch, C.; Olsen, S. R.; Harris, T. D.; Steinmetz, N. A.\nScore: 18.5, Published: 2023-08-25 DOI: 10.1101/2023.08.23.554527\nTo study the neural basis of behavior, we require methods to sensitively and accurately measure neural activity at single neuron and single spike resolution. Extracellular electrophysiology is a principal method for achieving this, but it has biases in the neurons it detects and it imperfectly resolves their action potentials. To overcome these limitations, we developed a silicon probe with significantly smaller and denser recording sites than previous designs, called Neuropixels Ultra (NP Ultra). This device measures neuronal activity at ultra-high densities (\u003e1300 sites per mm, 10 times higher than previous probes), with 6 m center-to-center spacing and low noise. This device effectively comprises an implantable voltage-sensing camera that captures a planar image of a neuron's electrical field. We introduce a new spike sorting algorithm optimized for these probes and use it to find that the yield of visually-responsive neurons in recordings from mouse visual cortex improves ~3-fold. Recordings across multiple brain regions and four species revealed a subset of unexpectedly small extracellular action potentials not previously reported. Further experiments determined that, in visual cortex, these do not correspond to major subclasses of interneurons and instead likely reflect recordings from axons. Finally, using ground-truth identification of cortical inhibitory cell types with optotagging, we found that cell type was discriminable with approximately 75% success among three types, a significant improvement over lower-resolution recordings. NP Ultra improves spike sorting performance, sampling bias, and cell type classification.\nLocal activation of CA1 pyramidal cells induces theta phase precession\nAuthors: Sloin, H.; Spivak, L.; Levi, A.; Gattegno, R.; Someck, S.; Stark, E.\nScore: 17.9, Published: 2023-08-21 DOI: 10.1101/2023.08.19.553985\nHippocampal theta phase precession is involved in spatiotemporal coding and generating multineural spike sequences, but how precession originates remains unresolved. To determine whether precession can be generated directly in CA1 and disambiguate multiple competing mechanisms, we used optogenetic activation to impose artificial place fields in pyramidal cells of mice running on a linear track. More than a third of the CA1 artificial fields exhibited synthetic precession that persisted for a full cycle. In contrast, artificial fields in the parietal cortex did not exhibit synthetic precession. The findings are incompatible with precession models based on inheritance, spreading activation, dual-input, or inhibition-excitation summation. Thus, a precession generator resides locally within CA1.\nSeparating cognitive and motor processes in the behaving mouse\nAuthors: Hasnain, M. A.; Birnbaum, J. E.; Nunez, J. L. U.; Hartman, E.; Chandrasekaran, C.; Economo, M. N.\nScore: 17.5, Published: 2023-08-24 DOI: 10.1101/2023.08.23.554474\nThe cognitive processes supporting complex animal behavior are closely associated with ubiquitous movements responsible for our posture, facial expressions, ability to actively sample our sensory environments, and other central processes. These movements are strongly related to neural activity across much of the brain, making it challenging to dissociate the neural dynamics that support cognitive processes from those supporting movements when they are highly correlated in time. Of critical importance is whether the dynamics supporting cognitive processes and related movements are separable, or if they are both driven by common neural mechanisms. Here, we demonstrate how the separability of cognitive and motor processes can be assessed, and, when separable, how each component can be isolated. We establish a novel two-context behavioral task in mice that involves multiple cognitive processes and show that commonly observed dynamics taken to support cognitive processes are strongly contaminated by movements. When cognitive and motor components are isolated using our analytical approach, we find that they exhibit distinct dynamical trajectories. Further, properly accounting for movement revealed that separate populations of cells encode cognitive and motor variables, in contrast to the mixed selectivity reported by prior work. Accurately isolating the dynamics associated with particular cognitive and motor processes will be essential for developing conceptual and computational models of neural circuit function and evaluating the function of the cell types of which neural circuits are composed.\nDeep sequencing of Phox2a nuclei reveals five classes of anterolateral system neurons\nAuthors: Bell, A. M.; Utting, C.; Hachisuka, J.; Dickie, A. C.; Kucharczyk, M. W.; Quillet, R.; Gutierrez-Mecinas, M.; Razlan, A. N. B.; Cooper, A. H.; Lan, Y.; Weir, G. A.; Bannister, K.; Watanabe, M.; Hoon, M. A.; Macaulay, I. C.; Denk, F.; Todd, A.\nScore: 17.3, Published: 2023-08-21 DOI: 10.1101/2023.08.20.553715\nThe anterolateral system (ALS) is a major ascending pathway from the spinal cord that projects to multiple brain areas and underlies the perception of pain, itch and skin temperature. Despite its importance, our understanding of this system has been hampered by the considerable functional and molecular diversity of its constituent cells. Here we use fluorescence-activated cell sorting to isolate ALS neurons belonging to the Phox2a-lineage for single-nucleus RNA sequencing. We reveal five distinct clusters of ALS neurons (ALS1-5) and document their laminar distribution in the spinal cord using in situ hybridization. We identify 3 clusters of neurons located predominantly in laminae I-III of the dorsal horn (ALS1-3) and two clusters with cell bodies located in deeper laminae (ALS4 \u0026 ALS5). Our findings reveal the transcriptional logic that underlies ALS neuronal diversity in the adult mouse and uncover the molecular identity of two previously identified classes of projection neurons. We also show that these molecular signatures can be used to target groups of ALS neurons using retrograde viral tracing. Overall, our findings provide a valuable resource for studying somatosensory biology and targeting subclasses of ALS neurons. Significance StatementThe anterolateral system (ALS) is a major ascending pathway from the spinal cord that underlies perception of pain, itch and skin temperature. It is therefore an important target for the development of new treatments for chronic pain. Our understanding of this system has been hampered by the considerable diversity of its constituent cells. Here we dissect the complex heterogeneity of these cells by using high-resolution RNA sequencing. We reveal five distinct types of ALS neurons, which are differentially distributed within the spinal cord, and probably represent functional populations. Our data provide novel insights into the molecular architecture of the ALS, and will be important for future studies to define the roles of different ALS cell types in sensory processing.\n",
  "wordCount" : "2586",
  "inLanguage": "en",
  "datePublished": "2023-08-27T10:37:02Z",
  "dateModified": "2023-08-27T10:37:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/neuroscience/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      neuroscience
    </h1>
    <div class="post-meta"><span>updated on August 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.552472">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.552472" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.552472">
        <p class="paperTitle">A reproducible and generalizable software workflow for analysis of large-scale neuroimaging data collections using BIDS Apps</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.552472" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.552472" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhao, C.; Jarecka, D.; Covitz, S.; Chen, Y.; Eickhoff, S. B.; Fair, D. A.; Franco, A. R.; Halchenko, Y. O.; Hendrickson, T. J.; Hoffstaedter, F.; Houghton, A.; Kiar, G.; Macdonald, A.; Mehta, K.; Milham, M. P.; Salo, T.; Hanke, M.; Ghosh, S. S.; Cieslak, M.; Satterthwaite, T. D.</p>
        <p class="info">Score: 33.7, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.552472' target='https://doi.org/10.1101/2023.08.16.552472'> 10.1101/2023.08.16.552472</a></p>
        <p class="abstract">Neuroimaging research faces a crisis of reproducibility. With massive sample sizes and greater data complexity, this problem becomes more acute. Software that operates on imaging data defined using the Brain Imaging Data Structure (BIDS) - BIDS Apps - have provided a substantial advance. However, even using BIDS Apps, a full audit trail of data processing is a necessary prerequisite for fully reproducible research. Obtaining a faithful record of the audit trail is challenging - especially for large datasets. Recently, the FAIRly big framework was introduced as a way to facilitate reproducible processing of large-scale data by leveraging DataLad - a version control system for data management. However, the current implementation of this framework was more of a proof of concept, and could not be immediately reused by other investigators for different use cases. Here we introduce the BIDS App Bootstrap (BABS), a user-friendly and generalizable Python package for reproducible image processing at scale. BABS facilitates the reproducible application of BIDS Apps to large-scale datasets. Leveraging DataLad and the FAIRly big framework, BABS tracks the full audit trail of data processing in a scalable way by automatically preparing all scripts necessary for data processing and version tracking on high performance computing (HPC) systems. Currently, BABS supports jobs submissions and audits on Sun Grid Engine (SGE) and Slurm HPCs with a parsimonious set of programs. To demonstrate its scalability, we applied BABS to data from the Healthy Brain Network (HBN; n=2,565). Taken together, BABS allows reproducible and scalable image processing and is broadly extensible via an open-source development model.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.553891">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.553891" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.553891">
        <p class="paperTitle">Regional Differences in Progenitor Consumption Dynamics Shape Brain Growth during Development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.553891" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.553891" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baumann, N.; Wagener, R.; Javed, A.; Abe, P.; Lopes, A.; Lavalley, A.; Fuciec, D.; Magrinelli, E.; Fievre, S.; Jabaudon, D.</p>
        <p class="info">Score: 29.4, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.553891' target='https://doi.org/10.1101/2023.08.21.553891'> 10.1101/2023.08.21.553891</a></p>
        <p class="abstract">Developing mammalian brains are characterized by disproportionate growth of the forebrain compared to other regions. How this localized expansion occurs is, however, largely unknown. To address this, we identified region-specific neurogenic patterns by creating a single-cell-resolution birthdate atlas of the mouse brain (https://www.neurobirth.org). We report that in forebrain regions, neurogenesis is sustained compared to the hindbrain, where neurogenesis is transient and limited to early brain development. Sustained forebrain neurogenesis reflects lengthened cell cycle and reduced consumptive divisions of ventricular zone progenitors, resulting in a preserved germinal cell pool. Using single-cell RNA sequencing, we identify functional molecular programs of ventricular zone progenitors that spatially and temporally regulate progenitor cycling properties, including through loss-of-function of the forebrain-enriched mitochondrial membrane protein Fam210b. These results reveal a parsimonious mechanism to locally regulate neuronal production, in which the time window during which progenitors generate cells is a critical determinant of region-specific brain expansion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554519">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554519" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554519">
        <p class="paperTitle">Using network control theory to study the dynamics of the structural connectome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554519" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554519" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parkes, L.; Kim, J. Z.; Stiso, J.; Brynildsen, J. K.; Cieslak, M.; Covitz, S.; Gur, R. E.; Gur, R. C.; Pasqualetti, F.; Shinohara, R. T.; Zhou, D.; Satterthwaite, T. D.; Bassett, D. S.</p>
        <p class="info">Score: 29.3, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554519' target='https://doi.org/10.1101/2023.08.23.554519'> 10.1101/2023.08.23.554519</a></p>
        <p class="abstract">Network control theory (NCT) is a simple and powerful tool for studying how network topology informs and constrains dynamics. Compared to other structure-function coupling approaches, the strength of NCT lies in its capacity to predict the patterns of external control signals that may alter dynamics in a desired way. We have extensively developed and validated the application of NCT to the human structural connectome. Through these efforts, we have studied (i) how different aspects of connectome topology affect neural dynamics, (ii) whether NCT outputs cohere with empirical data on brain function and stimulation, and (iii) how NCT outputs vary across development and correlate with behavior and mental health symptoms. In this protocol, we introduce a framework for applying NCT to structural connectomes following two main pathways. Our primary pathway focuses on computing the control energy associated with transitioning between specific neural activity states. Our second pathway focuses on computing average controllability, which indexes nodes&#39; general capacity to control dynamics. We also provide recommendations for comparing NCT outputs against null network models. Finally, we support this protocol with a Python-based software package called network control theory for python (https://github.com/BassettLab/nctpy)</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553449">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553449" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553449">
        <p class="paperTitle">Trigeminal innervation and tactile responses in mouse tongue</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553449" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553449" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, L.; Nagel, M.; Olson, W. P.; Chesler, A. T.; O&#39;Connor, D. H.</p>
        <p class="info">Score: 27.3, Published: 2023-08-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553449' target='https://doi.org/10.1101/2023.08.17.553449'> 10.1101/2023.08.17.553449</a></p>
        <p class="abstract">The mammalian tongue is richly innervated with somatosensory, gustatory and motor fibers. These form the basis of many ethologically important functions such as eating, speaking and social grooming. Despite its high tactile acuity and sensitivity, the neural basis of tongue mechanosensation remains largely mysterious. Here we explored the organization of mechanosensory afferents in the tongue and found that each lingual papilla is innervated by Piezo2&#43; trigeminal neurons. Notably, each fungiform papilla contained highly specialized ring-like sensory neuron terminations that circumscribe the taste buds. Myelinated lingual afferents in the mouse lingual papillae did not form corpuscular sensory end organs but rather had only free nerve endings. In vivo single-unit recordings from the trigeminal ganglion revealed two types of lingual low-threshold mechanoreceptors (LTMRs) with conduction velocities in the A{delta} range or above and distinct response properties: intermediately adapting (IA) units and rapidly adapting (RA) units. IA units were sensitive to static indentation and stroking, while RA units had a preference for tangential forces applied by stroking. Lingual LTMRs were not directly responsive to rapid cooling or chemicals that can induce astringent or numbing sensations. Genetic labeling of lingual afferents in the tongue revealed at least two types of nerve terminal patterns, involving dense innervation of individual fungiform papillae by multiple putatively distinct afferents, and relatively sparse innervation of filiform papillae. Together, our results indicate that fungiform papillae are mechanosensory structures, while suggesting a simple model that links the functional and anatomical properties of tactile sensory neurons in the tongue.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.553507">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.553507" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.553507">
        <p class="paperTitle">Multiplexing cortical brain organoids for the longitudinal dissection of developmental traits at single cell resolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.553507" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.553507" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Caporale, N.; Castaldi, D.; Rigoli, M. T.; Cheroni, C.; Trattaro, S.; Valenti, A.; Bonfanti, M.; Stucchi, S.; Tobon, A. L.; Ricca, D.; Lessi, M.; Pezzali, M.; Vitriolo, A.; Schmid, K.; Heinig, M.; Theis, F. J.; Villa, C. E.; Testa, G.</p>
        <p class="info">Score: 26.4, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.553507' target='https://doi.org/10.1101/2023.08.21.553507'> 10.1101/2023.08.21.553507</a></p>
        <p class="abstract">The combination of brain organoid and single cell omic technologies holds transformative potential to dissect human neurobiology at high resolution and with mechanistic precision. Delivering this promise in the context of human neurodiversity, physiological and pathological alike, requires however a major leap in scalability, given the need for experimental designs that include multiple individuals and, prospectively, population cohorts. To lay the foundation for this, we implemented and benchmarked complementary strategies to multiplex brain organoids. Following an extended longitudinal design with a uniquely informative set of timepoints, we pooled cells from different induced pluripotent stem cell lines either during organoids generation (upstream multiplexing in mosaic models) or before single cell-RNAseq library preparation (downstream multiplexing). We developed a new method, SCanSNP, and an aggregated call to deconvolve organoids cell identities, overcoming current criticalities in doublets prediction and low quality cells identification and improving accuracy over state of the art algorithms. Integrating single cell transcriptomes and analysing cell types across neurodevelopmental stages and multiplexing modalities, we validated the feasibility of both multiplexing methods in charting neurodevelopmental trajectories at high resolution, linking their specificity to genetic variation between individual lines. Together, this multiplexing suite of experimental and computational methods provides an enabling resource for disease modelling at scale and paves the way towards an in vitro epidemiology paradigm.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553874">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553874" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553874">
        <p class="paperTitle">Retinal origin of orientation but not direction selective maps in the superior colliculus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553874" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553874" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de Malmazet, D.; Kuhn, N.; Li, C.; Farrow, K.</p>
        <p class="info">Score: 24.4, Published: 2023-08-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553874' target='https://doi.org/10.1101/2023.08.18.553874'> 10.1101/2023.08.18.553874</a></p>
        <p class="abstract">Neurons in the mouse superior colliculus (&#34;colliculus&#34;) are arranged in ordered spatial maps. While orientationselective (OS) neurons form a concentric map aligned to the center of vision, direction-selective (DS) neurons are arranged in patches with changing preferences across the visual field. It remains unclear if these maps are a consequence of feed-forward input from the retina or local computations in the colliculus. To determine whether these maps originate in the retina, we mapped the local and global distribution of orientation- and direction-selective retinal ganglion cell boutons using in-vivo two-photon calcium imaging. We found that OS boutons formed patches that matched the distribution of OS neurons within the colliculus. DS boutons displayed less regional specializations, better reflecting the organization of DS neurons in the retina. Both eyes convey similar orientation but different DS inputs to the colliculus, as shown in recordings from retinal explants. These data demonstrate that orientation and direction maps within the colliculus are independent, implying that orientation maps are inherited from the retina, but direction maps require local circuits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554527">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554527" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554527">
        <p class="paperTitle">Ultra-high density electrodes improve detection, yield, and cell type specificity of brain recordings</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554527" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554527" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ye, Z.; Shelton, A. M.; Shaker, J. R.; Boussard, J. M.; Colonell, J.; Minavi, S.; Chen, S.; Windolf, C.; Hurwitz, C.; Namima, T.; Pedraja, F.; Weiss, S.; Raducanu, B.; Ness, T.; Einevoll, G. T.; Laurent, G.; Sawtell, N. B.; Bair, W.; Pasupathy, A.; Mora-Lopez, C.; Dutta, B.; Paninski, L.; Siegle, J. H.; Koch, C.; Olsen, S. R.; Harris, T. D.; Steinmetz, N. A.</p>
        <p class="info">Score: 18.5, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554527' target='https://doi.org/10.1101/2023.08.23.554527'> 10.1101/2023.08.23.554527</a></p>
        <p class="abstract">To study the neural basis of behavior, we require methods to sensitively and accurately measure neural activity at single neuron and single spike resolution. Extracellular electrophysiology is a principal method for achieving this, but it has biases in the neurons it detects and it imperfectly resolves their action potentials. To overcome these limitations, we developed a silicon probe with significantly smaller and denser recording sites than previous designs, called Neuropixels Ultra (NP Ultra). This device measures neuronal activity at ultra-high densities (&gt;1300 sites per mm, 10 times higher than previous probes), with 6 m center-to-center spacing and low noise. This device effectively comprises an implantable voltage-sensing camera that captures a planar image of a neuron&#39;s electrical field. We introduce a new spike sorting algorithm optimized for these probes and use it to find that the yield of visually-responsive neurons in recordings from mouse visual cortex improves ~3-fold. Recordings across multiple brain regions and four species revealed a subset of unexpectedly small extracellular action potentials not previously reported. Further experiments determined that, in visual cortex, these do not correspond to major subclasses of interneurons and instead likely reflect recordings from axons. Finally, using ground-truth identification of cortical inhibitory cell types with optotagging, we found that cell type was discriminable with approximately 75% success among three types, a significant improvement over lower-resolution recordings. NP Ultra improves spike sorting performance, sampling bias, and cell type classification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.19.553985">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.19.553985" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.19.553985">
        <p class="paperTitle">Local activation of CA1 pyramidal cells induces theta phase precession</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.19.553985" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.19.553985" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sloin, H.; Spivak, L.; Levi, A.; Gattegno, R.; Someck, S.; Stark, E.</p>
        <p class="info">Score: 17.9, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.19.553985' target='https://doi.org/10.1101/2023.08.19.553985'> 10.1101/2023.08.19.553985</a></p>
        <p class="abstract">Hippocampal theta phase precession is involved in spatiotemporal coding and generating multineural spike sequences, but how precession originates remains unresolved. To determine whether precession can be generated directly in CA1 and disambiguate multiple competing mechanisms, we used optogenetic activation to impose artificial place fields in pyramidal cells of mice running on a linear track. More than a third of the CA1 artificial fields exhibited synthetic precession that persisted for a full cycle. In contrast, artificial fields in the parietal cortex did not exhibit synthetic precession. The findings are incompatible with precession models based on inheritance, spreading activation, dual-input, or inhibition-excitation summation. Thus, a precession generator resides locally within CA1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554474">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554474" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554474">
        <p class="paperTitle">Separating cognitive and motor processes in the behaving mouse</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554474" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554474" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hasnain, M. A.; Birnbaum, J. E.; Nunez, J. L. U.; Hartman, E.; Chandrasekaran, C.; Economo, M. N.</p>
        <p class="info">Score: 17.5, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554474' target='https://doi.org/10.1101/2023.08.23.554474'> 10.1101/2023.08.23.554474</a></p>
        <p class="abstract">The cognitive processes supporting complex animal behavior are closely associated with ubiquitous movements responsible for our posture, facial expressions, ability to actively sample our sensory environments, and other central processes. These movements are strongly related to neural activity across much of the brain, making it challenging to dissociate the neural dynamics that support cognitive processes from those supporting movements when they are highly correlated in time. Of critical importance is whether the dynamics supporting cognitive processes and related movements are separable, or if they are both driven by common neural mechanisms. Here, we demonstrate how the separability of cognitive and motor processes can be assessed, and, when separable, how each component can be isolated. We establish a novel two-context behavioral task in mice that involves multiple cognitive processes and show that commonly observed dynamics taken to support cognitive processes are strongly contaminated by movements. When cognitive and motor components are isolated using our analytical approach, we find that they exhibit distinct dynamical trajectories. Further, properly accounting for movement revealed that separate populations of cells encode cognitive and motor variables, in contrast to the  mixed selectivity reported by prior work. Accurately isolating the dynamics associated with particular cognitive and motor processes will be essential for developing conceptual and computational models of neural circuit function and evaluating the function of the cell types of which neural circuits are composed.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.20.553715">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.20.553715" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.20.553715">
        <p class="paperTitle">Deep sequencing of Phox2a nuclei reveals five classes of anterolateral system neurons</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.20.553715" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.20.553715" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bell, A. M.; Utting, C.; Hachisuka, J.; Dickie, A. C.; Kucharczyk, M. W.; Quillet, R.; Gutierrez-Mecinas, M.; Razlan, A. N. B.; Cooper, A. H.; Lan, Y.; Weir, G. A.; Bannister, K.; Watanabe, M.; Hoon, M. A.; Macaulay, I. C.; Denk, F.; Todd, A.</p>
        <p class="info">Score: 17.3, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.20.553715' target='https://doi.org/10.1101/2023.08.20.553715'> 10.1101/2023.08.20.553715</a></p>
        <p class="abstract">The anterolateral system (ALS) is a major ascending pathway from the spinal cord that projects to multiple brain areas and underlies the perception of pain, itch and skin temperature. Despite its importance, our understanding of this system has been hampered by the considerable functional and molecular diversity of its constituent cells. Here we use fluorescence-activated cell sorting to isolate ALS neurons belonging to the Phox2a-lineage for single-nucleus RNA sequencing. We reveal five distinct clusters of ALS neurons (ALS1-5) and document their laminar distribution in the spinal cord using in situ hybridization. We identify 3 clusters of neurons located predominantly in laminae I-III of the dorsal horn (ALS1-3) and two clusters with cell bodies located in deeper laminae (ALS4 &amp; ALS5). Our findings reveal the transcriptional logic that underlies ALS neuronal diversity in the adult mouse and uncover the molecular identity of two previously identified classes of projection neurons. We also show that these molecular signatures can be used to target groups of ALS neurons using retrograde viral tracing. Overall, our findings provide a valuable resource for studying somatosensory biology and targeting subclasses of ALS neurons.

Significance StatementThe anterolateral system (ALS) is a major ascending pathway from the spinal cord that underlies perception of pain, itch and skin temperature. It is therefore an important target for the development of new treatments for chronic pain. Our understanding of this system has been hampered by the considerable diversity of its constituent cells. Here we dissect the complex heterogeneity of these cells by using high-resolution RNA sequencing. We reveal five distinct types of ALS neurons, which are differentially distributed within the spinal cord, and probably represent functional populations. Our data provide novel insights into the molecular architecture of the ALS, and will be important for future studies to define the roles of different ALS cell types in sensory processing.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
