<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>neuroscience</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>neuroscience | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="SARS-CoV-2 Spike amyloid fibrils specifically and selectively accelerates amyloid fibril formation of human prion protein and the amyloid β peptide
Authors: Larsson, J. N.; Hellstrand, E.; Hammarström, P.; Nyström, S.
Score: 2152.6, Published: 2023-09-01 DOI: 10.1101/2023.09.01.555834
An increasing number of reports suggest an association between COVID-19 infection and initiation or acceleration of neurodegenerative diseases (NDs) including Alzheimers disease (AD) and Creutzfeldt-Jakob disease (CJD). Both these diseases and several other NDs are caused by conversion of human proteins into a misfolded, aggregated amyloid fibril state.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/neuroscience/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="neuroscience" />
<meta property="og:description" content="SARS-CoV-2 Spike amyloid fibrils specifically and selectively accelerates amyloid fibril formation of human prion protein and the amyloid β peptide
Authors: Larsson, J. N.; Hellstrand, E.; Hammarström, P.; Nyström, S.
Score: 2152.6, Published: 2023-09-01 DOI: 10.1101/2023.09.01.555834
An increasing number of reports suggest an association between COVID-19 infection and initiation or acceleration of neurodegenerative diseases (NDs) including Alzheimers disease (AD) and Creutzfeldt-Jakob disease (CJD). Both these diseases and several other NDs are caused by conversion of human proteins into a misfolded, aggregated amyloid fibril state." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/neuroscience/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-17T10:36:50+00:00" />
<meta property="article:modified_time" content="2023-09-17T10:36:50+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="neuroscience"/>
<meta name="twitter:description" content="SARS-CoV-2 Spike amyloid fibrils specifically and selectively accelerates amyloid fibril formation of human prion protein and the amyloid β peptide
Authors: Larsson, J. N.; Hellstrand, E.; Hammarström, P.; Nyström, S.
Score: 2152.6, Published: 2023-09-01 DOI: 10.1101/2023.09.01.555834
An increasing number of reports suggest an association between COVID-19 infection and initiation or acceleration of neurodegenerative diseases (NDs) including Alzheimers disease (AD) and Creutzfeldt-Jakob disease (CJD). Both these diseases and several other NDs are caused by conversion of human proteins into a misfolded, aggregated amyloid fibril state."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "neuroscience",
      "item": "https://trxiv.yorks0n.com/posts/neuroscience/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "neuroscience",
  "name": "neuroscience",
  "description": "SARS-CoV-2 Spike amyloid fibrils specifically and selectively accelerates amyloid fibril formation of human prion protein and the amyloid β peptide\nAuthors: Larsson, J. N.; Hellstrand, E.; Hammarström, P.; Nyström, S.\nScore: 2152.6, Published: 2023-09-01 DOI: 10.1101/2023.09.01.555834\nAn increasing number of reports suggest an association between COVID-19 infection and initiation or acceleration of neurodegenerative diseases (NDs) including Alzheimers disease (AD) and Creutzfeldt-Jakob disease (CJD). Both these diseases and several other NDs are caused by conversion of human proteins into a misfolded, aggregated amyloid fibril state.",
  "keywords": [
    
  ],
  "articleBody": " SARS-CoV-2 Spike amyloid fibrils specifically and selectively accelerates amyloid fibril formation of human prion protein and the amyloid β peptide\nAuthors: Larsson, J. N.; Hellstrand, E.; Hammarström, P.; Nyström, S.\nScore: 2152.6, Published: 2023-09-01 DOI: 10.1101/2023.09.01.555834\nAn increasing number of reports suggest an association between COVID-19 infection and initiation or acceleration of neurodegenerative diseases (NDs) including Alzheimers disease (AD) and Creutzfeldt-Jakob disease (CJD). Both these diseases and several other NDs are caused by conversion of human proteins into a misfolded, aggregated amyloid fibril state. The fibril formation process is self-perpetuating by seeded conversion from preformed fibril seeds. We recently described a plausible mechanism for amyloid fibril formation of SARS-CoV-2 spike protein. Spike-protein formed amyloid fibrils upon cleavage by neutrophil elastase, abundant in the inflammatory response to COVID-19 infection. We here provide evidence of significant Spike-amyloid fibril seeded acceleration of amyloid formation of CJD associated human prion protein (HuPrP) using an in vitro conversion assay. By seeding the HuPrP conversion assay with other in vitro generated disease associated amyloid fibrils we demonstrate that this is not a general effect but a specific feature of spike-amyloid fibrils. We also showed that the amyloid fibril formation of AD associated A{beta}1-42 was accelerated by Spike-amyloid fibril seeds. Of seven different 20-amino acid long peptides, Spike532 (532NLVKNKCVNFNFNGLTGTGV551) was most efficient in seeding HuPrP and Spike601 (601GTNTSNQVAVLYQDVNCTEV620) was most effective in seeding A{beta}1-42, suggesting substrate dependent selectivity of the cross-seeding activity. Albeit purely in vitro, our data suggest that cross-seeding by Spike-amyloid fibrils can be implicated in the increasing number of reports of CJD, AD, and possibly other NDs in the wake of COVID-19.\nXylazine is an agonist at kappa opioid receptors and exhibits sex-specific responses to naloxone administration\nAuthors: Bedard, M. L.; Murray, J. G.; Huang, X.-P.; Nowlan, A. C.; Conley, S. Y.; E, M. S.; Loyack, S. J.; Cline, C. A.; Clodfelter, C. G.; Dasgupta, N.; Roth, B. L.; McElligott, Z. A.\nScore: 217.2, Published: 2023-09-09 DOI: 10.1101/2023.09.08.556914\nXylazine has been found in the unregulated drug supply at increasing rates, usually in combination with fentanyl. It has become critical to understand its basic pharmacology, how it impacts behavior, and how it interacts with fentanyl in rodent models of opioid administration. Despite commentary from scientists, politicians, and public health officials, it is not known if xylazine impacts the efficacy of naloxone, the opioid receptor antagonist used to reverse opioid induced respiratory depression. Furthermore, few studies have examined the effects of xylazine alone, without co-administration of ketamine. Here, we examine the impact of xylazine alone and in combination with fentanyl on several key behaviors in male and female mice. We demonstrate differential locomotor responses by dose and sex to xylazine. Surprisingly, our results further indicate that naloxone precipitates withdrawal from xylazine and a fentanyl/xylazine combination, in both sexes, with enhanced sensitivity in females. Further, we show that xylazine is a full agonist at the kappa opioid receptor, a potential mechanism for its naloxone sensitivity. One-Sentence SummaryWe present surprising new insights into xylazine and fentanyl pharmacology with immediate implications for clinical practice and frontline public health.\nATLAS: A rationally designed anterograde transsynaptic tracer\nAuthors: Rivera, J. F.; Weng, W.; Huang, H.; Rao, S.; Herring, B. E.; Arnold, D. B.\nScore: 71.8, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557425\nNeural circuits, which constitute the substrate for brain processing, can be traced in the retrograde direction, from postsynaptic to presynaptic cells, using methods based on introducing modified rabies virus into genetically marked cell types. These methods have revolutionized the field of neuroscience. However, similarly reliable, transsynaptic, and non-toxic methods to trace circuits in the anterograde direction are not available. Here, we describe such a method based on an antibody-like protein selected against the extracellular N-terminus of the AMPA receptor subunit GluA1 (AMPA.FingR). ATLAS (Anterograde Transsynaptic Label based on Antibody-like Sensors) is engineered to release the AMPA.FingR and its payload, which can include Cre recombinase, from presynaptic sites into the synaptic cleft, after which it binds to GluA1, enters postsynaptic cells through endocytosis and subsequently carries its payload to the nucleus. Testing in vivo and in dissociated cultures shows that ATLAS mediates monosynaptic tracing from genetically determined cells that is strictly anterograde, synaptic, and non-toxic. Moreover, ATLAS shows activity dependence, which may make tracing active circuits that underlie specific behaviors possible.\nRapid, biochemical tagging of cellular activity history in vivo\nAuthors: Zhang, R.; Anguiano, M.; Aarrestad, I. K.; Lin, S.; Chandra, J.; Vadde, S. S.; Olson, D. E.; Kim, C. K.\nScore: 61.0, Published: 2023-09-08 DOI: 10.1101/2023.09.06.556431\nIntracellular calcium (Ca2+) is ubiquitous to cell signaling across all biology. While existing fluorescent sensors and reporters can detect activated cells with elevated Ca2+ levels, these approaches require implants to deliver light to deep tissue, precluding their noninvasive use in freely-behaving animals. Here we engineered an enzyme-catalyzed approach that rapidly and biochemically tags cells with elevated Ca2+ in vivo. Ca2+-activated Split-TurboID (CaST) labels activated cells within 10 minutes with an exogenously-delivered biotin molecule. The enzymatic signal increases with Ca2+ concentration and biotin labeling time, demonstrating that CaST is a time-gated integrator of total Ca2+ activity. Furthermore, the CaST read-out can be performed immediately after activity labeling, in contrast to transcriptional reporters that require hours to produce signal. These capabilities allowed us to apply CaST in a new context to tag prefrontal cortex neurons activated by psilocybin, and to correlate the CaST signal with psilocybin-induced head-twitch responses in untethered mice.\nSynaptic and peptidergic connectomes of the Drosophila circadian clock\nAuthors: Reinhard, N.; Fukuda, A.; Manoli, G.; Derksen, E.; Saito, A.; Möller, G.; Sekiguchi, M.; Rieger, D.; Helfrich-Förster, C.; Yoshii, T.; Zandawala, M.\nScore: 42.4, Published: 2023-09-13 DOI: 10.1101/2023.09.11.557222\nThe circadian clock of animals regulates various physiological and behavioral processes in anticipation of, and adaptation to, daily environmental fluctuations. Consequently, the circadian clock and its output pathways play a pivotal role in maintaining homeostasis and optimizing daily functioning. To obtain novel insights into how diverse rhythmic physiology and behaviors are orchestrated, we have generated the first comprehensive connectivity map of an animal circadian clock using the Drosophila FlyWire brain connectome. We reveal hitherto unknown extensive contralateral synaptic connectivity between the clock neurons, which might contribute to the robustness of the clock by synchronizing clock neurons across the two hemispheres. In addition, we discover novel direct and indirect light input pathways to the clock neurons that could facilitate clock entrainment. Intriguingly, we observe sparse monosynaptic connectivity between clock neurons and downstream higher-order brain centers and neurosecretory cells known to regulate several behaviors and physiology. Therefore, we integrated single-cell transcriptomic analysis and receptor mapping to additionally decipher paracrine peptidergic signaling between clock neurons and with neurosecretory cells. Our analyses identified additional novel neuropeptides expressed in clock neurons and suggest that peptidergic signaling greatly enriches the interconnectivity within the clock network. Neuropeptides also form the basis of output pathways which regulate rhythmic hormonal signaling. The Drosophila circadian clock and neurosecretory center connectomes provide the framework to understand more complex clock and hormonal networks, respectively, as well as the rhythmic processes regulated by them.\nHuman HYPOMAP: A comprehensive spatio-cellular map of the human hypothalamus\nAuthors: Tadross, J. A.; Steuernagal, L.; Dowsett, G.; Kentistou, K. A.; Lundh, S.; Porniece, M.; Klemm, P.; Rainbow, K.; Hvid, H.; Kania, K.; Polex-Wolf, J.; Knudsen, L. B.; Pyke, C.; Perry, J.; Lam, B. Y.; Bruening, J. C.; Yeo, G. S. H.\nScore: 33.1, Published: 2023-09-15 DOI: 10.1101/2023.09.15.557967\nThe hypothalamus is a brain region that plays a key role in coordinating fundamental biological functions. However, our understanding of the underlying cellular components and circuitry, have, until recently, emerged primarily from rodent studies. Here, we combine a single-nucleus sequencing database of 433,369 human hypothalamic cells, with spatial transcriptomics, to present a comprehensive spatio-cellular transcriptional map of the human hypothalamus, the \"HYPOMAP\". Analysing hypothalamic leptin melanocortin pathway neuronal populations that play a role in appetite control, we identify spatially distinct populations of arcuate nucleus POMC and AGRP neurons, and their receptors MC3R and MC4R. Next, we map the cells expressing incretin receptors, targets of the new generation of anti-obesity medications, and uncover transcriptionally distinct GLP1R and GIPR-expressing cellular populations. Finally, out of the 458 hypothalamic cell types in HYPOMAP, we find 182 neuronal clusters are significantly enriched in expression of BMI GWAS genes. This enrichment is driven by 375 effector genes, with rare deleterious variants in 6 of these; MC4R, PCSK1, POMC, CALCR, BSN and CORO1A, the last of which has previously not been linked to obesity; being significantly associated with changes in BMI at the population level. Thus, the HYPOMAP provides a detailed atlas of the human hypothalamus in a spatial context, and serves as an important resource to identify novel druggable targets for treating a wide range of conditions, including reproductive, circadian, and metabolic disorders.\nBrainLM: A foundation model for brain activity recordings\nAuthors: Ortega Caro, J.; Oliveira Fonseca, A. H.; Averill, C.; Rizvi, S. A.; Rosati, M.; Cross, J. L.; Mittal, P.; Zappala, E.; Levine, D.; Dhodapkar, R. M.; Abdallah, C. G.; van Dijk, D.\nScore: 32.3, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557460\nAO_SCPLOWBSTRACTC_SCPLOWWe introduce the Brain Language Model (BrainLM), a foundation model for brain activity dynamics trained on 6,700 hours of fMRI recordings. Utilizing self-supervised masked-prediction training, BrainLM demonstrates proficiency in both fine-tuning and zero-shot inference tasks. Fine-tuning allows for the prediction of clinical variables and future brain states. In zero-shot inference, the model identifies functional networks and generates interpretable latent representations of neural activity. Furthermore, we introduce a novel prompting technique, allowing BrainLM to function as an in silico simulator of brain activity responses to perturbations. BrainLM offers a novel framework for the analysis and understanding of large-scale brain activity data, serving as a \"lens\" through which new data can be more effectively interpreted.\nFunctional connectome through the human life span\nAuthors: Sun, L.; Zhao, T.; Liang, X.; Xia, M.; Li, Q.; Liao, X.; Gong, G.; Wang, Q.; Pang, C.; Yu, Q.; Bi, Y.; Chen, P.; Chen, R.; Chen, Y.; Chen, T.; Cheng, J.; Cheng, Y.; Cui, Z.; Dai, Z.; Deng, Y.; Ding, Y.; Dong, Q.; Duan, D.; Gao, J.-H.; Gong, Q.; Han, Y.; Han, Z.; Huang, C.-C.; Huang, R.; Li, L.; Lin, C.-P.; Lin, Q.; Liu, B.; Liu, C.; Liu, N.; Liu, Y.; Lu, J.; Ma, L.; Men, W.; Qin, S.; Qiu, J.; Qiu, S.; Si, T.; Tan, S.; Tang, Y.; Tao, S.; Wang, D.; Wang, F.; Wang, J.; Wang, P.; Wang, X.; Wang, Y.; Wei, D.; Wu, Y.; Xie, P.; Xu, X.; Xu, Y.; Xu, Z.; Yang, L.; Zeng, Z.; Zhang, H.; Zhang, X.; Zhao, G\nScore: 31.8, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557193\nThe functional connectome of the human brain represents the fundamental network architecture of neural activity, but its normative growth trajectory over the life course remains unknown. Here, we aggregate the largest, quality-controlled multimodal neuroimaging dataset across 119 global sites, including 33,809 task-free fMRI and structural MRI scans of 32,328 individuals aged from 32 postmenstrual weeks to 80 years old. The lifespan growth charts of the connectome are quantified at the whole cortex, system, and regional levels using generalized additive models for location, scale, and shape. We find critical inflection points in the nonlinear growth trajectories of the whole-brain functional connectome, notably peaking in the fourth decade of life. After establishing the first fine-grained, lifespan-spanning suite of system-level brain atlases, we generate person-specific parcellation maps and further elucidate distinct timelines of maturation for functional segregation within various subsystems. We identify a spatiotemporal gradient axis that governs the life-course growth of regional connectivity, transitioning from primary sensory cortices to higher-order association regions. Using the connectome-based normative model, we demonstrate substantial individual heterogeneities at the network level in patients with autism spectrum disorder and patients with major depressive disorder, respectively. Our findings shed light on the functional connectome's life-course evolution, serving as normative references for understanding network growth principles of the human brain and assessing individual variations of patients with neuropsychiatric conditions.\nDual lineage origins of neocortical astrocytes\nAuthors: Zhou, J.; Vitali, I.; Roig Puiggros, S.; Javed, A.; Jabaudon, D.; Mayer, C.; Bocchi, R.\nScore: 30.8, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557313\nAstrocytes represent one of the most abundant cell types in the central nervous system, and play an essential role in nearly all aspects of brain functions1. Recent studies have challenged the notion that cortical astrocytes are a uniform population, and have highlighted their diverse characteristics at the morphological, molecular, and functional levels2-5. However, how this diversity originates and establishes during cortical development, remains largely unknown. Using single-cell RNA sequencing, we identified five distinct astrocyte subtypes displaying unique spatial patterns in the mouse neocortex, and discovered essential regulators for their formation. Furthermore, we used TrackerSeq6, a method that integrates heritable DNA barcodes into the genome of electroporated progenitors, to track clonally related astrocytes, and identified two distinct lineages that give rise to the five astrocyte subtypes. The first lineage derives from Emx1+ multipotent progenitors that first generate neurons and then switch to produce cortical astrocytes. The second lineage stems from a fate-restricted progenitor population that exclusively gives rise to a specific subset of cortical astrocytes, marked by Olig2. The knockout of this gene in cortical progenitors is sufficient to promote a fate switch between the two lineages. These findings offer novel insights into the cellular mechanisms underlying astrocyte diversity, highlighting the presence of multiple progenitor subtypes, responsible for generating distinct subtypes of astrocytes.\nNetworks of descending neurons transform command-like signals into population-based behavioral control\nAuthors: Braun, J.; Hurtak, F.; Wang-Chen, S.; Ramdya, P.\nScore: 30.8, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557103\nTo transform intentions into actions, movement instructions must pass from the brain to downstream motor circuits through descending neurons (DNs). These include small sets of command-like neurons that are sufficient to drive behaviors--the circuit mechanisms for which remain unclear. Here, we show that command-like DNs in Drosophila directly recruit networks of additional DNs to orchestrate flexible behaviors. Specifically, we found that optogenetic activation of command-like DNs previously thought to drive behaviors alone in fact co-activate larger populations of DNs. Connectome analysis revealed that this functional recruitment can be explained by direct excitatory connections between command-like DNs and networks of interconnected DNs in the brain. The size of downstream DN networks is predictive of whether descending population recruitment is necessary to generate a complete behavior: DNs with many downstream descending partners require network recruitment to drive flexible behaviors, while neurons with fewer partners can alone drive stereotyped behaviors and simple movements. Finally, DN networks reside within behavior-specific clusters that inhibit one another. These results support a mechanism for command-like descending control whereby a continuum of stereotyped to flexible behaviors are generated through the recruitment of increasingly large DN networks which likely construct a complete behavior by combining multiple motor subroutines.\n",
  "wordCount" : "2431",
  "inLanguage": "en",
  "datePublished": "2023-09-17T10:36:50Z",
  "dateModified": "2023-09-17T10:36:50Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/neuroscience/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      neuroscience
    </h1>
    <div class="post-meta"><span>updated on September 17, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555834">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555834" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555834">
        <p class="paperTitle">SARS-CoV-2 Spike amyloid fibrils specifically and selectively accelerates amyloid fibril formation of human prion protein and the amyloid β peptide</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555834" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555834" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Larsson, J. N.; Hellstrand, E.; Hammarström, P.; Nyström, S.</p>
        <p class="info">Score: 2152.6, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555834' target='https://doi.org/10.1101/2023.09.01.555834'> 10.1101/2023.09.01.555834</a></p>
        <p class="abstract">An increasing number of reports suggest an association between COVID-19 infection and initiation or acceleration of neurodegenerative diseases (NDs) including Alzheimers disease (AD) and Creutzfeldt-Jakob disease (CJD). Both these diseases and several other NDs are caused by conversion of human proteins into a misfolded, aggregated amyloid fibril state. The fibril formation process is self-perpetuating by seeded conversion from preformed fibril seeds. We recently described a plausible mechanism for amyloid fibril formation of SARS-CoV-2 spike protein. Spike-protein formed amyloid fibrils upon cleavage by neutrophil elastase, abundant in the inflammatory response to COVID-19 infection.

We here provide evidence of significant Spike-amyloid fibril seeded acceleration of amyloid formation of CJD associated human prion protein (HuPrP) using an in vitro conversion assay. By seeding the HuPrP conversion assay with other in vitro generated disease associated amyloid fibrils we demonstrate that this is not a general effect but a specific feature of spike-amyloid fibrils. We also showed that the amyloid fibril formation of AD associated A{beta}1-42 was accelerated by Spike-amyloid fibril seeds. Of seven different 20-amino acid long peptides, Spike532 (532NLVKNKCVNFNFNGLTGTGV551) was most efficient in seeding HuPrP and Spike601 (601GTNTSNQVAVLYQDVNCTEV620) was most effective in seeding A{beta}1-42, suggesting substrate dependent selectivity of the cross-seeding activity.

Albeit purely in vitro, our data suggest that cross-seeding by Spike-amyloid fibrils can be implicated in the increasing number of reports of CJD, AD, and possibly other NDs in the wake of COVID-19.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556914">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556914" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556914">
        <p class="paperTitle">Xylazine is an agonist at kappa opioid receptors and exhibits sex-specific responses to naloxone administration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556914" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556914" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bedard, M. L.; Murray, J. G.; Huang, X.-P.; Nowlan, A. C.; Conley, S. Y.; E, M. S.; Loyack, S. J.; Cline, C. A.; Clodfelter, C. G.; Dasgupta, N.; Roth, B. L.; McElligott, Z. A.</p>
        <p class="info">Score: 217.2, Published: 2023-09-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556914' target='https://doi.org/10.1101/2023.09.08.556914'> 10.1101/2023.09.08.556914</a></p>
        <p class="abstract">Xylazine has been found in the unregulated drug supply at increasing rates, usually in combination with fentanyl. It has become critical to understand its basic pharmacology, how it impacts behavior, and how it interacts with fentanyl in rodent models of opioid administration. Despite commentary from scientists, politicians, and public health officials, it is not known if xylazine impacts the efficacy of naloxone, the opioid receptor antagonist used to reverse opioid induced respiratory depression. Furthermore, few studies have examined the effects of xylazine alone, without co-administration of ketamine. Here, we examine the impact of xylazine alone and in combination with fentanyl on several key behaviors in male and female mice. We demonstrate differential locomotor responses by dose and sex to xylazine. Surprisingly, our results further indicate that naloxone precipitates withdrawal from xylazine and a fentanyl/xylazine combination, in both sexes, with enhanced sensitivity in females. Further, we show that xylazine is a full agonist at the kappa opioid receptor, a potential mechanism for its naloxone sensitivity.

One-Sentence SummaryWe present surprising new insights into xylazine and fentanyl pharmacology with immediate implications for clinical practice and frontline public health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557425">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557425" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557425">
        <p class="paperTitle">ATLAS: A rationally designed anterograde transsynaptic tracer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557425" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557425" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rivera, J. F.; Weng, W.; Huang, H.; Rao, S.; Herring, B. E.; Arnold, D. B.</p>
        <p class="info">Score: 71.8, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557425' target='https://doi.org/10.1101/2023.09.12.557425'> 10.1101/2023.09.12.557425</a></p>
        <p class="abstract">Neural circuits, which constitute the substrate for brain processing, can be traced in the retrograde direction, from postsynaptic to presynaptic cells, using methods based on introducing modified rabies virus into genetically marked cell types. These methods have revolutionized the field of neuroscience. However, similarly reliable, transsynaptic, and non-toxic methods to trace circuits in the anterograde direction are not available. Here, we describe such a method based on an antibody-like protein selected against the extracellular N-terminus of the AMPA receptor subunit GluA1 (AMPA.FingR). ATLAS (Anterograde Transsynaptic Label based on Antibody-like Sensors) is engineered to release the AMPA.FingR and its payload, which can include Cre recombinase, from presynaptic sites into the synaptic cleft, after which it binds to GluA1, enters postsynaptic cells through endocytosis and subsequently carries its payload to the nucleus. Testing in vivo and in dissociated cultures shows that ATLAS mediates monosynaptic tracing from genetically determined cells that is strictly anterograde, synaptic, and non-toxic. Moreover, ATLAS shows activity dependence, which may make tracing active circuits that underlie specific behaviors possible.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556431">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556431" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556431">
        <p class="paperTitle">Rapid, biochemical tagging of cellular activity history in vivo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556431" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556431" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, R.; Anguiano, M.; Aarrestad, I. K.; Lin, S.; Chandra, J.; Vadde, S. S.; Olson, D. E.; Kim, C. K.</p>
        <p class="info">Score: 61.0, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556431' target='https://doi.org/10.1101/2023.09.06.556431'> 10.1101/2023.09.06.556431</a></p>
        <p class="abstract">Intracellular calcium (Ca2&#43;) is ubiquitous to cell signaling across all biology. While existing fluorescent sensors and reporters can detect activated cells with elevated Ca2&#43; levels, these approaches require implants to deliver light to deep tissue, precluding their noninvasive use in freely-behaving animals. Here we engineered an enzyme-catalyzed approach that rapidly and biochemically tags cells with elevated Ca2&#43; in vivo. Ca2&#43;-activated Split-TurboID (CaST) labels activated cells within 10 minutes with an exogenously-delivered biotin molecule. The enzymatic signal increases with Ca2&#43; concentration and biotin labeling time, demonstrating that CaST is a time-gated integrator of total Ca2&#43; activity. Furthermore, the CaST read-out can be performed immediately after activity labeling, in contrast to transcriptional reporters that require hours to produce signal. These capabilities allowed us to apply CaST in a new context to tag prefrontal cortex neurons activated by psilocybin, and to correlate the CaST signal with psilocybin-induced head-twitch responses in untethered mice.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557222">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557222" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557222">
        <p class="paperTitle">Synaptic and peptidergic connectomes of the Drosophila circadian clock</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557222" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557222" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reinhard, N.; Fukuda, A.; Manoli, G.; Derksen, E.; Saito, A.; Möller, G.; Sekiguchi, M.; Rieger, D.; Helfrich-Förster, C.; Yoshii, T.; Zandawala, M.</p>
        <p class="info">Score: 42.4, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557222' target='https://doi.org/10.1101/2023.09.11.557222'> 10.1101/2023.09.11.557222</a></p>
        <p class="abstract">The circadian clock of animals regulates various physiological and behavioral processes in anticipation of, and adaptation to, daily environmental fluctuations. Consequently, the circadian clock and its output pathways play a pivotal role in maintaining homeostasis and optimizing daily functioning. To obtain novel insights into how diverse rhythmic physiology and behaviors are orchestrated, we have generated the first comprehensive connectivity map of an animal circadian clock using the Drosophila FlyWire brain connectome. We reveal hitherto unknown extensive contralateral synaptic connectivity between the clock neurons, which might contribute to the robustness of the clock by synchronizing clock neurons across the two hemispheres. In addition, we discover novel direct and indirect light input pathways to the clock neurons that could facilitate clock entrainment. Intriguingly, we observe sparse monosynaptic connectivity between clock neurons and downstream higher-order brain centers and neurosecretory cells known to regulate several behaviors and physiology. Therefore, we integrated single-cell transcriptomic analysis and receptor mapping to additionally decipher paracrine peptidergic signaling between clock neurons and with neurosecretory cells. Our analyses identified additional novel neuropeptides expressed in clock neurons and suggest that peptidergic signaling greatly enriches the interconnectivity within the clock network. Neuropeptides also form the basis of output pathways which regulate rhythmic hormonal signaling. The Drosophila circadian clock and neurosecretory center connectomes provide the framework to understand more complex clock and hormonal networks, respectively, as well as the rhythmic processes regulated by them.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557967">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557967" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557967">
        <p class="paperTitle">Human HYPOMAP: A comprehensive spatio-cellular map of the human hypothalamus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557967" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557967" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tadross, J. A.; Steuernagal, L.; Dowsett, G.; Kentistou, K. A.; Lundh, S.; Porniece, M.; Klemm, P.; Rainbow, K.; Hvid, H.; Kania, K.; Polex-Wolf, J.; Knudsen, L. B.; Pyke, C.; Perry, J.; Lam, B. Y.; Bruening, J. C.; Yeo, G. S. H.</p>
        <p class="info">Score: 33.1, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557967' target='https://doi.org/10.1101/2023.09.15.557967'> 10.1101/2023.09.15.557967</a></p>
        <p class="abstract">The hypothalamus is a brain region that plays a key role in coordinating fundamental biological functions. However, our understanding of the underlying cellular components and circuitry, have, until recently, emerged primarily from rodent studies. Here, we combine a single-nucleus sequencing database of 433,369 human hypothalamic cells, with spatial transcriptomics, to present a comprehensive spatio-cellular transcriptional map of the human hypothalamus, the &#34;HYPOMAP&#34;. Analysing hypothalamic leptin melanocortin pathway neuronal populations that play a role in appetite control, we identify spatially distinct populations of arcuate nucleus POMC and AGRP neurons, and their receptors MC3R and MC4R. Next, we map the cells expressing incretin receptors, targets of the new generation of anti-obesity medications, and uncover transcriptionally distinct GLP1R and GIPR-expressing cellular populations. Finally, out of the 458 hypothalamic cell types in HYPOMAP, we find 182 neuronal clusters are significantly enriched in expression of BMI GWAS genes. This enrichment is driven by 375 effector genes, with rare deleterious variants in 6 of these; MC4R, PCSK1, POMC, CALCR, BSN and CORO1A, the last of which has previously not been linked to obesity; being significantly associated with changes in BMI at the population level. Thus, the HYPOMAP provides a detailed atlas of the human hypothalamus in a spatial context, and serves as an important resource to identify novel druggable targets for treating a wide range of conditions, including reproductive, circadian, and metabolic disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557460">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557460" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557460">
        <p class="paperTitle">BrainLM: A foundation model for brain activity recordings</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557460" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557460" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ortega Caro, J.; Oliveira Fonseca, A. H.; Averill, C.; Rizvi, S. A.; Rosati, M.; Cross, J. L.; Mittal, P.; Zappala, E.; Levine, D.; Dhodapkar, R. M.; Abdallah, C. G.; van Dijk, D.</p>
        <p class="info">Score: 32.3, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557460' target='https://doi.org/10.1101/2023.09.12.557460'> 10.1101/2023.09.12.557460</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWWe introduce the Brain Language Model (BrainLM), a foundation model for brain activity dynamics trained on 6,700 hours of fMRI recordings. Utilizing self-supervised masked-prediction training, BrainLM demonstrates proficiency in both fine-tuning and zero-shot inference tasks. Fine-tuning allows for the prediction of clinical variables and future brain states. In zero-shot inference, the model identifies functional networks and generates interpretable latent representations of neural activity. Furthermore, we introduce a novel prompting technique, allowing BrainLM to function as an in silico simulator of brain activity responses to perturbations. BrainLM offers a novel framework for the analysis and understanding of large-scale brain activity data, serving as a &#34;lens&#34; through which new data can be more effectively interpreted.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557193">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557193" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557193">
        <p class="paperTitle">Functional connectome through the human life span</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557193" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557193" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sun, L.; Zhao, T.; Liang, X.; Xia, M.; Li, Q.; Liao, X.; Gong, G.; Wang, Q.; Pang, C.; Yu, Q.; Bi, Y.; Chen, P.; Chen, R.; Chen, Y.; Chen, T.; Cheng, J.; Cheng, Y.; Cui, Z.; Dai, Z.; Deng, Y.; Ding, Y.; Dong, Q.; Duan, D.; Gao, J.-H.; Gong, Q.; Han, Y.; Han, Z.; Huang, C.-C.; Huang, R.; Li, L.; Lin, C.-P.; Lin, Q.; Liu, B.; Liu, C.; Liu, N.; Liu, Y.; Lu, J.; Ma, L.; Men, W.; Qin, S.; Qiu, J.; Qiu, S.; Si, T.; Tan, S.; Tang, Y.; Tao, S.; Wang, D.; Wang, F.; Wang, J.; Wang, P.; Wang, X.; Wang, Y.; Wei, D.; Wu, Y.; Xie, P.; Xu, X.; Xu, Y.; Xu, Z.; Yang, L.; Zeng, Z.; Zhang, H.; Zhang, X.; Zhao, G</p>
        <p class="info">Score: 31.8, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557193' target='https://doi.org/10.1101/2023.09.12.557193'> 10.1101/2023.09.12.557193</a></p>
        <p class="abstract">The functional connectome of the human brain represents the fundamental network architecture of neural activity, but its normative growth trajectory over the life course remains unknown. Here, we aggregate the largest, quality-controlled multimodal neuroimaging dataset across 119 global sites, including 33,809 task-free fMRI and structural MRI scans of 32,328 individuals aged from 32 postmenstrual weeks to 80 years old. The lifespan growth charts of the connectome are quantified at the whole cortex, system, and regional levels using generalized additive models for location, scale, and shape. We find critical inflection points in the nonlinear growth trajectories of the whole-brain functional connectome, notably peaking in the fourth decade of life. After establishing the first fine-grained, lifespan-spanning suite of system-level brain atlases, we generate person-specific parcellation maps and further elucidate distinct timelines of maturation for functional segregation within various subsystems. We identify a spatiotemporal gradient axis that governs the life-course growth of regional connectivity, transitioning from primary sensory cortices to higher-order association regions. Using the connectome-based normative model, we demonstrate substantial individual heterogeneities at the network level in patients with autism spectrum disorder and patients with major depressive disorder, respectively. Our findings shed light on the functional connectome&#39;s life-course evolution, serving as normative references for understanding network growth principles of the human brain and assessing individual variations of patients with neuropsychiatric conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557313">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557313" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557313">
        <p class="paperTitle">Dual lineage origins of neocortical astrocytes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557313" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557313" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, J.; Vitali, I.; Roig Puiggros, S.; Javed, A.; Jabaudon, D.; Mayer, C.; Bocchi, R.</p>
        <p class="info">Score: 30.8, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557313' target='https://doi.org/10.1101/2023.09.12.557313'> 10.1101/2023.09.12.557313</a></p>
        <p class="abstract">Astrocytes represent one of the most abundant cell types in the central nervous system, and play an essential role in nearly all aspects of brain functions1. Recent studies have challenged the notion that cortical astrocytes are a uniform population, and have highlighted their diverse characteristics at the morphological, molecular, and functional levels2-5. However, how this diversity originates and establishes during cortical development, remains largely unknown. Using single-cell RNA sequencing, we identified five distinct astrocyte subtypes displaying unique spatial patterns in the mouse neocortex, and discovered essential regulators for their formation. Furthermore, we used TrackerSeq6, a method that integrates heritable DNA barcodes into the genome of electroporated progenitors, to track clonally related astrocytes, and identified two distinct lineages that give rise to the five astrocyte subtypes. The first lineage derives from Emx1&#43; multipotent progenitors that first generate neurons and then switch to produce cortical astrocytes. The second lineage stems from a fate-restricted progenitor population that exclusively gives rise to a specific subset of cortical astrocytes, marked by Olig2. The knockout of this gene in cortical progenitors is sufficient to promote a fate switch between the two lineages. These findings offer novel insights into the cellular mechanisms underlying astrocyte diversity, highlighting the presence of multiple progenitor subtypes, responsible for generating distinct subtypes of astrocytes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557103">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557103" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557103">
        <p class="paperTitle">Networks of descending neurons transform command-like signals into population-based behavioral control</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557103" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557103" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Braun, J.; Hurtak, F.; Wang-Chen, S.; Ramdya, P.</p>
        <p class="info">Score: 30.8, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557103' target='https://doi.org/10.1101/2023.09.11.557103'> 10.1101/2023.09.11.557103</a></p>
        <p class="abstract">To transform intentions into actions, movement instructions must pass from the brain to downstream motor circuits through descending neurons (DNs). These include small sets of command-like neurons that are sufficient to drive behaviors--the circuit mechanisms for which remain unclear. Here, we show that command-like DNs in Drosophila directly recruit networks of additional DNs to orchestrate flexible behaviors. Specifically, we found that optogenetic activation of command-like DNs previously thought to drive behaviors alone in fact co-activate larger populations of DNs. Connectome analysis revealed that this functional recruitment can be explained by direct excitatory connections between command-like DNs and networks of interconnected DNs in the brain. The size of downstream DN networks is predictive of whether descending population recruitment is necessary to generate a complete behavior: DNs with many downstream descending partners require network recruitment to drive flexible behaviors, while neurons with fewer partners can alone drive stereotyped behaviors and simple movements. Finally, DN networks reside within behavior-specific clusters that inhibit one another. These results support a mechanism for command-like descending control whereby a continuum of stereotyped to flexible behaviors are generated through the recruitment of increasingly large DN networks which likely construct a complete behavior by combining multiple motor subroutines.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
