<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>neuroscience</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>neuroscience | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Constructed languages are processed by the same brain mechanisms as natural languages
Authors: Malik-Moraleda, S.; Taliaferro, M.; Shannon, S.; Jhingan, N.; Swords, S.; Peterson, D. J.; Frommer, P.; Okrand, M.; Sams, J.; Cardwell, R.; Freeman, C.; Fedorenko, E.
Score: 125.8, Published: 2023-07-28 DOI: 10.1101/2023.07.28.550667
What constitutes a language? Natural languages share some features with other domains: from math, to music, to gesture. However, the brain mechanisms that process linguistic input are highly specialized, showing little or no response to diverse non-linguistic tasks.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/neuroscience/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="neuroscience" />
<meta property="og:description" content="Constructed languages are processed by the same brain mechanisms as natural languages
Authors: Malik-Moraleda, S.; Taliaferro, M.; Shannon, S.; Jhingan, N.; Swords, S.; Peterson, D. J.; Frommer, P.; Okrand, M.; Sams, J.; Cardwell, R.; Freeman, C.; Fedorenko, E.
Score: 125.8, Published: 2023-07-28 DOI: 10.1101/2023.07.28.550667
What constitutes a language? Natural languages share some features with other domains: from math, to music, to gesture. However, the brain mechanisms that process linguistic input are highly specialized, showing little or no response to diverse non-linguistic tasks." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/neuroscience/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-02T10:38:09+00:00" />
<meta property="article:modified_time" content="2023-08-02T10:38:09+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="neuroscience"/>
<meta name="twitter:description" content="Constructed languages are processed by the same brain mechanisms as natural languages
Authors: Malik-Moraleda, S.; Taliaferro, M.; Shannon, S.; Jhingan, N.; Swords, S.; Peterson, D. J.; Frommer, P.; Okrand, M.; Sams, J.; Cardwell, R.; Freeman, C.; Fedorenko, E.
Score: 125.8, Published: 2023-07-28 DOI: 10.1101/2023.07.28.550667
What constitutes a language? Natural languages share some features with other domains: from math, to music, to gesture. However, the brain mechanisms that process linguistic input are highly specialized, showing little or no response to diverse non-linguistic tasks."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "neuroscience",
      "item": "https://trxiv.yorks0n.com/posts/neuroscience/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "neuroscience",
  "name": "neuroscience",
  "description": "Constructed languages are processed by the same brain mechanisms as natural languages\nAuthors: Malik-Moraleda, S.; Taliaferro, M.; Shannon, S.; Jhingan, N.; Swords, S.; Peterson, D. J.; Frommer, P.; Okrand, M.; Sams, J.; Cardwell, R.; Freeman, C.; Fedorenko, E.\nScore: 125.8, Published: 2023-07-28 DOI: 10.1101/2023.07.28.550667\nWhat constitutes a language? Natural languages share some features with other domains: from math, to music, to gesture. However, the brain mechanisms that process linguistic input are highly specialized, showing little or no response to diverse non-linguistic tasks.",
  "keywords": [
    
  ],
  "articleBody": " Constructed languages are processed by the same brain mechanisms as natural languages\nAuthors: Malik-Moraleda, S.; Taliaferro, M.; Shannon, S.; Jhingan, N.; Swords, S.; Peterson, D. J.; Frommer, P.; Okrand, M.; Sams, J.; Cardwell, R.; Freeman, C.; Fedorenko, E.\nScore: 125.8, Published: 2023-07-28 DOI: 10.1101/2023.07.28.550667\nWhat constitutes a language? Natural languages share some features with other domains: from math, to music, to gesture. However, the brain mechanisms that process linguistic input are highly specialized, showing little or no response to diverse non-linguistic tasks. Here, we examine constructed languages (conlangs) to ask whether they draw on the same neural mechanisms as natural languages, or whether they instead pattern with domains like math and logic. Using individual-subject fMRI analyses, we show that understanding conlangs recruits the same brain areas as natural language comprehension. This result holds for Esperanto (n=19 speakers)-- created to resemble natural languages--and fictional conlangs (Klingon (n=10), Navi (n=9), High Valyrian (n=3), and Dothraki (n=3)), created to differ from natural languages, and suggests that conlangs and natural languages share critical features and that the notable differences between conlangs and natural language are not consequential for the cognitive and neural mechanisms that they engage.\nNonlinear manifolds underlie neural population activity during behaviour\nAuthors: Fortunato, C.; Bennasar-Vazquez, J.; Park, J.; Chang, J. C.; Miller, L. E.; Dudman, J. T.; Perich, M. G.; Gallego, J. A.\nScore: 56.9, Published: 2023-07-21 DOI: 10.1101/2023.07.18.549575\nThere is rich variety in the activity of single neurons recorded during behaviour. Yet, these diverse single neuron responses can be well described by relatively few patterns of neural co-modulation. The study of such low-dimensional structure of neural population activity has provided important insights into how the brain generates behaviour. Virtually all of these studies have used linear dimensionality reduction techniques to estimate these population-wide co-modulation patterns, constraining them to a flat \"neural manifold\". Here, we hypothesised that since neurons have nonlinear responses and make thousands of distributed and recurrent connections that likely amplify such nonlinearities, neural manifolds should be intrinsically nonlinear. Combining neural population recordings from monkey motor cortex, mouse motor cortex, mouse striatum, and human motor cortex, we show that: 1) neural manifolds are intrinsically nonlinear; 2) the degree of their nonlinearity varies across architecturally distinct brain regions; and 3) manifold nonlinearity becomes more evident during complex tasks that require more varied activity patterns. Simulations using recurrent neural network models confirmed the proposed relationship between circuit connectivity and manifold nonlinearity, including the differences across architecturally distinct regions. Thus, neural manifolds underlying the generation of behaviour are inherently nonlinear, and properly accounting for such nonlinearities will be critical as neuroscientists move towards studying numerous brain regions involved in increasingly complex and naturalistic behaviours.\nThe brain hierarchically represents the past and future during multistep anticipation\nAuthors: Tarder-Stoll, H.; Baldassano, C.; Aly, M.\nScore: 49.6, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550399\nMemory for temporal structure enables both planning of future events and retrospection of past events. We investigated how the brain flexibly represents extended temporal sequences into the past and future during anticipation. Participants learned sequences of environments in immersive virtual reality. Pairs of sequences had the same environments in a different order, enabling context-specific learning. During fMRI, participants anticipated upcoming environments multiple steps into the future in a given sequence. Temporal structure was represented in the hippocampus and across visual regions (1) bidirectionally, with graded representations into the past and future and (2) hierarchically, with further events into the past and future represented in successively more anterior brain regions. Further, context-specific predictions were prioritized in the forward but not backward direction. Together, this work sheds light on how we flexibly represent sequential structure to enable planning over multiple timescales.\nRastermap: a discovery method for neural population recordings\nAuthors: Stringer, C.; Zhong, L.; Syeda, A.; Du, F.; Pachitariu, M.\nScore: 46.6, Published: 2023-07-28 DOI: 10.1101/2023.07.25.550571\nNeurophysiology has long progressed through exploratory experiments and chance discoveries. Anecdotes abound of researchers setting up experiments while listening to spikes in real time and observing a pattern of consistent firing when certain stimuli or behaviors happened. With the advent of large-scale recordings, such close observation of data has become harder because high-dimensional spaces are impenetrable to our pattern-finding intuitions. To help ourselves find patterns in neural data, our lab has been openly developing a visualization framework known as \"Rastermap\" over the past five years. Rastermap takes advantage of a new global optimization algorithm for sorting neural responses along a one-dimensional manifold. Displayed as a raster plot, the sorted neurons show a variety of activity patterns, which can be more easily identified and interpreted. We first benchmark Rastermap on realistic simulations with multiplexed cognitive variables. Then we demonstrate it on recordings of tens of thousands of neurons from mouse visual and sensorimotor cortex during spontaneous, stimulus-evoked and task-evoked epochs, as well as on whole-brain zebrafish recordings, widefield calcium imaging data, population recordings from rat hippocampus and artificial neural networks. Finally, we illustrate high-dimensional scenarios where Rastermap and similar algorithms cannot be used effectively.\nAPOE4/4 is linked to damaging lipid droplets in Alzheimer's microglia\nAuthors: Haney, M.; Palovics, R.; Munson, C.; Long, C.; Johansson, P.; Yip, O.; Dong, W.; Rawat, E.; West, E.; Schlachetzki, J.; Tsai, A. P.-Y.; Guldner, I.; Lamichhane, B.; Smith, A.; Schaum, N.; Calcuttawala, K.; Shin, A.; Wang, Y.-H.; Wang, C.; Koutsodendris, N.; Serrano, G. E.; Beach, T.; Reiman, E. M.; Glass, C.; Abu-Remaileh, M. M.; Enejder, A.; Huang, Y.; Wyss-Coray, T.\nScore: 49.5, Published: 2023-07-25 DOI: 10.1101/2023.07.21.549930\nSeveral genetic risk factors for Alzheimers Disease (AD) implicate genes involved in lipid metabolism and many of these lipid genes are highly expressed in glial cells. However, the relationship between lipid metabolism in glia and AD pathology remains poorly understood. Through single-nucleus RNA-sequencing of AD brain tissue, we have identified a microglial state defined by the expression of the lipid droplet (LD) associated enzyme ACSL1 with ACSL1-positive microglia most abundant in AD patients with the APOE4/4 genotype. In human iPSC-derived microglia (iMG) fibrillar A{beta} (fA{beta}) induces ACSL1 expression, triglyceride synthesis, and LD accumulation in an APOE-dependent manner. Additionally, conditioned media from LD-containing microglia leads to Tau phosphorylation and neurotoxicity in an APOE-dependent manner. Our findings suggest a link between genetic risk factors for AD with microglial LD accumulation and neurotoxic microglial-derived factors, potentially providing novel therapeutic strategies for AD.\nSpatial and single-nucleus transcriptomic analysis of genetic and sporadic forms of Alzheimer's Disease\nAuthors: Miyoshi, E.; Morabito, S.; Henningfield, C. M.; Rahimzadeh, N.; Shabestari, S. K.; Das, S.; Michael, N.; Reese, F.; Shi, Z.; Cao, Z.; Scarfone, V.; Arreola, M. A.; Lu, J.; Wright, S.; Silva, J.; Leavy, K.; Lott, I. T.; Doran, E.; Yong, W. H.; Shahin, S.; Perez-Rosendahl, M.; Head, E.; Green, K. N.; Swarup, V.\nScore: 41.0, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550282\nThe pathogenesis of Alzheimers disease (AD) depends on environmental and heritable factors, with remarkable differences evident between individuals at the molecular level. Here we present a transcriptomic survey of AD using spatial transcriptomics (ST) and single-nucleus RNA-seq in cortical samples from early-stage AD, late-stage AD, and AD in Down Syndrome (AD in DS) donors. Studying AD in DS provides an opportunity to enhance our understanding of the AD transcriptome, potentially bridging the gap between genetic mouse models and sporadic AD. Our analysis revealed spatial and cell-type specific changes in disease, with broad similarities in these changes between sAD and AD in DS. We performed additional ST experiments in a disease timecourse of 5xFAD and wildtype mice to facilitate cross-species comparisons. Finally, amyloid plaque and fibril imaging in the same tissue samples used for ST enabled us to directly link changes in gene expression with accumulation and spread of pathology.\nCell-type specific molecular signatures of aging revealed in a brain-wide transcriptomic cell-type atlas\nAuthors: Jin, K.; Yao, Z.; van Velthoven, C. T. J.; Kaplan, E. S.; Glattfelder, K.; Barlow, S. T.; Boyer, G.; Carey, D.; Casper, T.; Chakka, A. B.; Chakrabarty, R.; Clark, M.; Departee, M.; Desierto, M.; Gary, A.; Gloe, J.; Goldy, J.; Guilford, N.; Guzman, J.; Hirschstein, D.; Lee, C.; Liang, E.; Pham, T.; Reding, M.; Ronellenfitch, K.; Ruiz, A.; Sevigny, J.; Shapovalova, N.; Shulga, L.; Sulc, J.; Torkelson, A.; Tung, H.; Levi, B.; Sunkin, S. M.; Dee, N.; Esposito, L.; Smith, K.; Tasic, B.; Zeng, H.\nScore: 40.4, Published: 2023-07-27 DOI: 10.1101/2023.07.26.550355\nBiological aging can be defined as a gradual loss of homeostasis across various aspects of molecular and cellular function. Aging is a complex and dynamic process which influences distinct cell types in a myriad of ways. The cellular architecture of the mammalian brain is heterogeneous and diverse, making it challenging to identify precise areas and cell types of the brain that are more susceptible to aging than others. Here, we present a high-resolution single-cell RNA sequencing dataset containing [~]1.2 million high-quality single-cell transcriptomic profiles of brain cells from young adult and aged mice across both sexes, including areas spanning the forebrain, midbrain, and hindbrain. We find age-associated gene expression signatures across nearly all 130+ neuronal and non-neuronal cell subclasses we identified. We detect the greatest gene expression changes in non-neuronal cell types, suggesting that different cell types in the brain vary in their susceptibility to aging. We identify specific, age-enriched clusters within specific glial, vascular, and immune cell types from both cortical and subcortical regions of the brain, and specific gene expression changes associated with cell senescence, inflammation, decrease in new myelination, and decreased vasculature integrity. We also identify genes with expression changes across multiple cell subclasses, pointing to certain mechanisms of aging that may occur across wide regions or broad cell types of the brain. Finally, we discover the greatest gene expression changes in cell types localized to the third ventricle of the hypothalamus, including tanycytes, ependymal cells, and Tbx3+ neurons found in the arcuate nucleus that are part of the neuronal circuits regulating food intake and energy homeostasis. These findings suggest that the area surrounding the third ventricle in the hypothalamus may be a hub for aging in the mouse brain. Overall, we reveal a dynamic landscape of cell-type-specific transcriptomic changes in the brain associated with normal aging that will serve as a foundation for the investigation of functional changes in the aging process and the interaction of aging and diseases.\nThree-dimensional single-cell transcriptome imaging of thick tissues\nAuthors: Fang, R.; Halpern, A. R.; Rahman, M. M.; Huang, Z.; Lei, Z.; Hell, S. J.; Dulac, C.; Zhuang, X.\nScore: 65.9, Published: 2023-07-25 DOI: 10.1101/2023.07.21.550124\nMultiplexed error-robust fluorescence in-situ hybridization (MERFISH) allows genome-scale imaging of RNAs in individual cells in intact tissues. To date, MERFISH has been applied to image thin tissue samples of [~]10-{micro}m thickness. Here, we present a method to enable three-dimensional (3D) single-cell transcriptome imaging of thick tissue specimens by integrating MERFISH with confocal microscopy for optical sectioning and deep learning for increasing imaging speed and quality. We demonstrated 3D MERFISH on mouse brain tissue sections of up to 200 {micro}m thickness with high detection efficiency and accuracy. We anticipate that 3D thick-tissue MERFISH imaging will broaden the scope of questions that can be addressed by spatial genomics.\nDaytime experiences shape neural activity and dream content in the sleeping brain\nAuthors: Kumral, D.; Palmieri, J.; Gais, S.; Schoenauer, M.\nScore: 31.3, Published: 2023-07-29 DOI: 10.1101/2023.07.29.551087\nLearning-related brain activity patterns are replayed during sleep, and memories of recent experiences appear in our dreams. The connection between these phenomena, however, remains unclear. We investigated whether memory reinstatement during sleep contributes to dreaming. Participants listened to audiobooks before falling asleep. We could determine which audiobook they had studied based on dream reports collected during the night. Audiobook content was also reinstated at the neural level, in high-density EEG recordings. Brain activity during rapid eye movement sleep, particularly in the high-frequency beta range, carried information about the audiobook and simultaneously benefitted memory retention. Crucially, when the learning condition was manifest in neural activity, it also emerged in dreams. Reprocessing of daytime experiences during sleep thus shapes our brain activity, our dreams, and our memories.\nProfiling human hypothalamic neurons reveals a candidate combination drug therapy for weight loss\nAuthors: Chen, H.-J. C.; Yang, A.; Mazzaferro, S.; Mali, I.; Cahn, O.; Kentistou, K.; Rowley, C.; Stewart, N.; Seah, J. W. E.; Pisupati, V.; Kirwan, P.; Aggarwal, S.; Toyohara, T.; Florido, M. H. C.; Cowan, C. A.; Quambusch, L.; Hyvonen, M.; Livesey, M. R.; Perry, J. R. B.; Marioni, J. C.; Merkle, F. T.\nScore: 24.6, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549357\nObesity substantially increases the risk of type 2 diabetes, cardiovascular disease, and other diseases, making it a leading preventable cause of death in developed countries. It has a strong genetic basis, with obesity-associated genetic variants preferentially acting in the brain. This includes the hypothalamic pro-opiomelanocortin (POMC) neurons that inhibit food intake and are stimulated by drugs that agonise glucagon-like 1 peptide receptor (GLP1R) including Semaglutide (Ozempic/Wegovy). We therefore hypothesised that drugs which selectively activate human POMC neurons would suppress appetite and promote weight loss, and that focusing on drugs already approved for use would facilitate rapid clinical translation. We therefore generated POMC neurons from human pluripotent stem cells (hPSCs) and identified enriched genes that were genetically associated with obesity and targeted by approved drugs. We found that human POMC neurons are enriched in GLP1R, reliably activated by Semaglutide, and their responses are further increased by co-administration of Ceritinib, an FDA-approved drug potently and selectively inhibiting anaplastic lymphoma kinase (ALK). Ceritinib reduced food intake and body weight in obese but not lean mice, and upregulated the expression of GLP1R in the mouse hypothalamus and hPSC-derived human hypothalamic neurons. These studies reveal a new potential therapeutic strategy for reducing food intake and body weight, and demonstrate the utility of hPSC-derived hypothalamic neurons for drug discovery.\n",
  "wordCount" : "2267",
  "inLanguage": "en",
  "datePublished": "2023-08-02T10:38:09Z",
  "dateModified": "2023-08-02T10:38:09Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/neuroscience/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      neuroscience
    </h1>
    <div class="post-meta"><span>updated on August 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550667">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550667" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550667">
        <p class="paperTitle">Constructed languages are processed by the same brain mechanisms as natural languages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550667" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550667" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Malik-Moraleda, S.; Taliaferro, M.; Shannon, S.; Jhingan, N.; Swords, S.; Peterson, D. J.; Frommer, P.; Okrand, M.; Sams, J.; Cardwell, R.; Freeman, C.; Fedorenko, E.</p>
        <p class="info">Score: 125.8, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550667' target='https://doi.org/10.1101/2023.07.28.550667'> 10.1101/2023.07.28.550667</a></p>
        <p class="abstract">What constitutes a language? Natural languages share some features with other domains: from math, to music, to gesture. However, the brain mechanisms that process linguistic input are highly specialized, showing little or no response to diverse non-linguistic tasks. Here, we examine constructed languages (conlangs) to ask whether they draw on the same neural mechanisms as natural languages, or whether they instead pattern with domains like math and logic. Using individual-subject fMRI analyses, we show that understanding conlangs recruits the same brain areas as natural language comprehension. This result holds for Esperanto (n=19 speakers)-- created to resemble natural languages--and fictional conlangs (Klingon (n=10), Navi (n=9), High Valyrian (n=3), and Dothraki (n=3)), created to differ from natural languages, and suggests that conlangs and natural languages share critical features and that the notable differences between conlangs and natural language are not consequential for the cognitive and neural mechanisms that they engage.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549575">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549575" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549575">
        <p class="paperTitle">Nonlinear manifolds underlie neural population activity during behaviour</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549575" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549575" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fortunato, C.; Bennasar-Vazquez, J.; Park, J.; Chang, J. C.; Miller, L. E.; Dudman, J. T.; Perich, M. G.; Gallego, J. A.</p>
        <p class="info">Score: 56.9, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549575' target='https://doi.org/10.1101/2023.07.18.549575'> 10.1101/2023.07.18.549575</a></p>
        <p class="abstract">There is rich variety in the activity of single neurons recorded during behaviour. Yet, these diverse single neuron responses can be well described by relatively few patterns of neural co-modulation. The study of such low-dimensional structure of neural population activity has provided important insights into how the brain generates behaviour. Virtually all of these studies have used linear dimensionality reduction techniques to estimate these population-wide co-modulation patterns, constraining them to a flat &#34;neural manifold&#34;. Here, we hypothesised that since neurons have nonlinear responses and make thousands of distributed and recurrent connections that likely amplify such nonlinearities, neural manifolds should be intrinsically nonlinear. Combining neural population recordings from monkey motor cortex, mouse motor cortex, mouse striatum, and human motor cortex, we show that: 1) neural manifolds are intrinsically nonlinear; 2) the degree of their nonlinearity varies across architecturally distinct brain regions; and 3) manifold nonlinearity becomes more evident during complex tasks that require more varied activity patterns. Simulations using recurrent neural network models confirmed the proposed relationship between circuit connectivity and manifold nonlinearity, including the differences across architecturally distinct regions. Thus, neural manifolds underlying the generation of behaviour are inherently nonlinear, and properly accounting for such nonlinearities will be critical as neuroscientists move towards studying numerous brain regions involved in increasingly complex and naturalistic behaviours.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550399">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550399" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550399">
        <p class="paperTitle">The brain hierarchically represents the past and future during multistep anticipation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550399" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550399" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tarder-Stoll, H.; Baldassano, C.; Aly, M.</p>
        <p class="info">Score: 49.6, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550399' target='https://doi.org/10.1101/2023.07.24.550399'> 10.1101/2023.07.24.550399</a></p>
        <p class="abstract">Memory for temporal structure enables both planning of future events and retrospection of past events. We investigated how the brain flexibly represents extended temporal sequences into the past and future during anticipation. Participants learned sequences of environments in immersive virtual reality. Pairs of sequences had the same environments in a different order, enabling context-specific learning. During fMRI, participants anticipated upcoming environments multiple steps into the future in a given sequence. Temporal structure was represented in the hippocampus and across visual regions (1) bidirectionally, with graded representations into the past and future and (2) hierarchically, with further events into the past and future represented in successively more anterior brain regions. Further, context-specific predictions were prioritized in the forward but not backward direction. Together, this work sheds light on how we flexibly represent sequential structure to enable planning over multiple timescales.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550571">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550571" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550571">
        <p class="paperTitle">Rastermap: a discovery method for neural population recordings</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550571" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550571" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stringer, C.; Zhong, L.; Syeda, A.; Du, F.; Pachitariu, M.</p>
        <p class="info">Score: 46.6, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550571' target='https://doi.org/10.1101/2023.07.25.550571'> 10.1101/2023.07.25.550571</a></p>
        <p class="abstract">Neurophysiology has long progressed through exploratory experiments and chance discoveries. Anecdotes abound of researchers setting up experiments while listening to spikes in real time and observing a pattern of consistent firing when certain stimuli or behaviors happened. With the advent of large-scale recordings, such close observation of data has become harder because high-dimensional spaces are impenetrable to our pattern-finding intuitions. To help ourselves find patterns in neural data, our lab has been openly developing a visualization framework known as &#34;Rastermap&#34; over the past five years. Rastermap takes advantage of a new global optimization algorithm for sorting neural responses along a one-dimensional manifold. Displayed as a raster plot, the sorted neurons show a variety of activity patterns, which can be more easily identified and interpreted. We first benchmark Rastermap on realistic simulations with multiplexed cognitive variables. Then we demonstrate it on recordings of tens of thousands of neurons from mouse visual and sensorimotor cortex during spontaneous, stimulus-evoked and task-evoked epochs, as well as on whole-brain zebrafish recordings, widefield calcium imaging data, population recordings from rat hippocampus and artificial neural networks. Finally, we illustrate high-dimensional scenarios where Rastermap and similar algorithms cannot be used effectively.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.549930">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.549930" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.549930">
        <p class="paperTitle">APOE4/4 is linked to damaging lipid droplets in Alzheimer&#39;s microglia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.549930" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.549930" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Haney, M.; Palovics, R.; Munson, C.; Long, C.; Johansson, P.; Yip, O.; Dong, W.; Rawat, E.; West, E.; Schlachetzki, J.; Tsai, A. P.-Y.; Guldner, I.; Lamichhane, B.; Smith, A.; Schaum, N.; Calcuttawala, K.; Shin, A.; Wang, Y.-H.; Wang, C.; Koutsodendris, N.; Serrano, G. E.; Beach, T.; Reiman, E. M.; Glass, C.; Abu-Remaileh, M. M.; Enejder, A.; Huang, Y.; Wyss-Coray, T.</p>
        <p class="info">Score: 49.5, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.549930' target='https://doi.org/10.1101/2023.07.21.549930'> 10.1101/2023.07.21.549930</a></p>
        <p class="abstract">Several genetic risk factors for Alzheimers Disease (AD) implicate genes involved in lipid metabolism and many of these lipid genes are highly expressed in glial cells. However, the relationship between lipid metabolism in glia and AD pathology remains poorly understood. Through single-nucleus RNA-sequencing of AD brain tissue, we have identified a microglial state defined by the expression of the lipid droplet (LD) associated enzyme ACSL1 with ACSL1-positive microglia most abundant in AD patients with the APOE4/4 genotype. In human iPSC-derived microglia (iMG) fibrillar A{beta} (fA{beta}) induces ACSL1 expression, triglyceride synthesis, and LD accumulation in an APOE-dependent manner. Additionally, conditioned media from LD-containing microglia leads to Tau phosphorylation and neurotoxicity in an APOE-dependent manner. Our findings suggest a link between genetic risk factors for AD with microglial LD accumulation and neurotoxic microglial-derived factors, potentially providing novel therapeutic strategies for AD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550282">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550282" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550282">
        <p class="paperTitle">Spatial and single-nucleus transcriptomic analysis of genetic and sporadic forms of Alzheimer&#39;s Disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550282" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550282" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miyoshi, E.; Morabito, S.; Henningfield, C. M.; Rahimzadeh, N.; Shabestari, S. K.; Das, S.; Michael, N.; Reese, F.; Shi, Z.; Cao, Z.; Scarfone, V.; Arreola, M. A.; Lu, J.; Wright, S.; Silva, J.; Leavy, K.; Lott, I. T.; Doran, E.; Yong, W. H.; Shahin, S.; Perez-Rosendahl, M.; Head, E.; Green, K. N.; Swarup, V.</p>
        <p class="info">Score: 41.0, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550282' target='https://doi.org/10.1101/2023.07.24.550282'> 10.1101/2023.07.24.550282</a></p>
        <p class="abstract">The pathogenesis of Alzheimers disease (AD) depends on environmental and heritable factors, with remarkable differences evident between individuals at the molecular level. Here we present a transcriptomic survey of AD using spatial transcriptomics (ST) and single-nucleus RNA-seq in cortical samples from early-stage AD, late-stage AD, and AD in Down Syndrome (AD in DS) donors. Studying AD in DS provides an opportunity to enhance our understanding of the AD transcriptome, potentially bridging the gap between genetic mouse models and sporadic AD. Our analysis revealed spatial and cell-type specific changes in disease, with broad similarities in these changes between sAD and AD in DS. We performed additional ST experiments in a disease timecourse of 5xFAD and wildtype mice to facilitate cross-species comparisons. Finally, amyloid plaque and fibril imaging in the same tissue samples used for ST enabled us to directly link changes in gene expression with accumulation and spread of pathology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550355">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550355" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550355">
        <p class="paperTitle">Cell-type specific molecular signatures of aging revealed in a brain-wide transcriptomic cell-type atlas</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550355" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550355" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jin, K.; Yao, Z.; van Velthoven, C. T. J.; Kaplan, E. S.; Glattfelder, K.; Barlow, S. T.; Boyer, G.; Carey, D.; Casper, T.; Chakka, A. B.; Chakrabarty, R.; Clark, M.; Departee, M.; Desierto, M.; Gary, A.; Gloe, J.; Goldy, J.; Guilford, N.; Guzman, J.; Hirschstein, D.; Lee, C.; Liang, E.; Pham, T.; Reding, M.; Ronellenfitch, K.; Ruiz, A.; Sevigny, J.; Shapovalova, N.; Shulga, L.; Sulc, J.; Torkelson, A.; Tung, H.; Levi, B.; Sunkin, S. M.; Dee, N.; Esposito, L.; Smith, K.; Tasic, B.; Zeng, H.</p>
        <p class="info">Score: 40.4, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550355' target='https://doi.org/10.1101/2023.07.26.550355'> 10.1101/2023.07.26.550355</a></p>
        <p class="abstract">Biological aging can be defined as a gradual loss of homeostasis across various aspects of molecular and cellular function. Aging is a complex and dynamic process which influences distinct cell types in a myriad of ways. The cellular architecture of the mammalian brain is heterogeneous and diverse, making it challenging to identify precise areas and cell types of the brain that are more susceptible to aging than others. Here, we present a high-resolution single-cell RNA sequencing dataset containing [~]1.2 million high-quality single-cell transcriptomic profiles of brain cells from young adult and aged mice across both sexes, including areas spanning the forebrain, midbrain, and hindbrain. We find age-associated gene expression signatures across nearly all 130&#43; neuronal and non-neuronal cell subclasses we identified. We detect the greatest gene expression changes in non-neuronal cell types, suggesting that different cell types in the brain vary in their susceptibility to aging. We identify specific, age-enriched clusters within specific glial, vascular, and immune cell types from both cortical and subcortical regions of the brain, and specific gene expression changes associated with cell senescence, inflammation, decrease in new myelination, and decreased vasculature integrity. We also identify genes with expression changes across multiple cell subclasses, pointing to certain mechanisms of aging that may occur across wide regions or broad cell types of the brain. Finally, we discover the greatest gene expression changes in cell types localized to the third ventricle of the hypothalamus, including tanycytes, ependymal cells, and Tbx3&#43; neurons found in the arcuate nucleus that are part of the neuronal circuits regulating food intake and energy homeostasis. These findings suggest that the area surrounding the third ventricle in the hypothalamus may be a hub for aging in the mouse brain. Overall, we reveal a dynamic landscape of cell-type-specific transcriptomic changes in the brain associated with normal aging that will serve as a foundation for the investigation of functional changes in the aging process and the interaction of aging and diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.550124">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.550124" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.550124">
        <p class="paperTitle">Three-dimensional single-cell transcriptome imaging of thick tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.550124" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.550124" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fang, R.; Halpern, A. R.; Rahman, M. M.; Huang, Z.; Lei, Z.; Hell, S. J.; Dulac, C.; Zhuang, X.</p>
        <p class="info">Score: 65.9, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.550124' target='https://doi.org/10.1101/2023.07.21.550124'> 10.1101/2023.07.21.550124</a></p>
        <p class="abstract">Multiplexed error-robust fluorescence in-situ hybridization (MERFISH) allows genome-scale imaging of RNAs in individual cells in intact tissues. To date, MERFISH has been applied to image thin tissue samples of [~]10-{micro}m thickness. Here, we present a method to enable three-dimensional (3D) single-cell transcriptome imaging of thick tissue specimens by integrating MERFISH with confocal microscopy for optical sectioning and deep learning for increasing imaging speed and quality. We demonstrated 3D MERFISH on mouse brain tissue sections of up to 200 {micro}m thickness with high detection efficiency and accuracy. We anticipate that 3D thick-tissue MERFISH imaging will broaden the scope of questions that can be addressed by spatial genomics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.29.551087">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.29.551087" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.29.551087">
        <p class="paperTitle">Daytime experiences shape neural activity and dream content in the sleeping brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.29.551087" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.29.551087" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kumral, D.; Palmieri, J.; Gais, S.; Schoenauer, M.</p>
        <p class="info">Score: 31.3, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.29.551087' target='https://doi.org/10.1101/2023.07.29.551087'> 10.1101/2023.07.29.551087</a></p>
        <p class="abstract">Learning-related brain activity patterns are replayed during sleep, and memories of recent experiences appear in our dreams. The connection between these phenomena, however, remains unclear. We investigated whether memory reinstatement during sleep contributes to dreaming. Participants listened to audiobooks before falling asleep. We could determine which audiobook they had studied based on dream reports collected during the night. Audiobook content was also reinstated at the neural level, in high-density EEG recordings. Brain activity during rapid eye movement sleep, particularly in the high-frequency beta range, carried information about the audiobook and simultaneously benefitted memory retention. Crucially, when the learning condition was manifest in neural activity, it also emerged in dreams. Reprocessing of daytime experiences during sleep thus shapes our brain activity, our dreams, and our memories.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549357">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549357" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549357">
        <p class="paperTitle">Profiling human hypothalamic neurons reveals a candidate combination drug therapy for weight loss</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549357" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549357" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, H.-J. C.; Yang, A.; Mazzaferro, S.; Mali, I.; Cahn, O.; Kentistou, K.; Rowley, C.; Stewart, N.; Seah, J. W. E.; Pisupati, V.; Kirwan, P.; Aggarwal, S.; Toyohara, T.; Florido, M. H. C.; Cowan, C. A.; Quambusch, L.; Hyvonen, M.; Livesey, M. R.; Perry, J. R. B.; Marioni, J. C.; Merkle, F. T.</p>
        <p class="info">Score: 24.6, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549357' target='https://doi.org/10.1101/2023.07.18.549357'> 10.1101/2023.07.18.549357</a></p>
        <p class="abstract">Obesity substantially increases the risk of type 2 diabetes, cardiovascular disease, and other diseases, making it a leading preventable cause of death in developed countries. It has a strong genetic basis, with obesity-associated genetic variants preferentially acting in the brain. This includes the hypothalamic pro-opiomelanocortin (POMC) neurons that inhibit food intake and are stimulated by drugs that agonise glucagon-like 1 peptide receptor (GLP1R) including Semaglutide (Ozempic/Wegovy). We therefore hypothesised that drugs which selectively activate human POMC neurons would suppress appetite and promote weight loss, and that focusing on drugs already approved for use would facilitate rapid clinical translation. We therefore generated POMC neurons from human pluripotent stem cells (hPSCs) and identified enriched genes that were genetically associated with obesity and targeted by approved drugs. We found that human POMC neurons are enriched in GLP1R, reliably activated by Semaglutide, and their responses are further increased by co-administration of Ceritinib, an FDA-approved drug potently and selectively inhibiting anaplastic lymphoma kinase (ALK). Ceritinib reduced food intake and body weight in obese but not lean mice, and upregulated the expression of GLP1R in the mouse hypothalamus and hPSC-derived human hypothalamic neurons. These studies reveal a new potential therapeutic strategy for reducing food intake and body weight, and demonstrate the utility of hPSC-derived hypothalamic neurons for drug discovery.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
