<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>neuroscience</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>neuroscience | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="SARS-CoV-2 Spike amyloid fibrils specifically and selectively accelerates amyloid fibril formation of human prion protein and the amyloid β peptide
Authors: Larsson, J. N.; Hellstrand, E.; Hammarström, P.; Nyström, S.
Score: 1834.3, Published: 2023-09-01 DOI: 10.1101/2023.09.01.555834
An increasing number of reports suggest an association between COVID-19 infection and initiation or acceleration of neurodegenerative diseases including Alzheimer&#39;s disease (AD) and Creutzfeldt-Jakob disease (CJD). Both these diseases and several other neurodegenerative diseases are caused by conversion of human proteins into a misfolded, aggregated amyloid fibril state.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/neuroscience/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="neuroscience" />
<meta property="og:description" content="SARS-CoV-2 Spike amyloid fibrils specifically and selectively accelerates amyloid fibril formation of human prion protein and the amyloid β peptide
Authors: Larsson, J. N.; Hellstrand, E.; Hammarström, P.; Nyström, S.
Score: 1834.3, Published: 2023-09-01 DOI: 10.1101/2023.09.01.555834
An increasing number of reports suggest an association between COVID-19 infection and initiation or acceleration of neurodegenerative diseases including Alzheimer&#39;s disease (AD) and Creutzfeldt-Jakob disease (CJD). Both these diseases and several other neurodegenerative diseases are caused by conversion of human proteins into a misfolded, aggregated amyloid fibril state." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/neuroscience/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-06T10:39:07+00:00" />
<meta property="article:modified_time" content="2023-09-06T10:39:07+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="neuroscience"/>
<meta name="twitter:description" content="SARS-CoV-2 Spike amyloid fibrils specifically and selectively accelerates amyloid fibril formation of human prion protein and the amyloid β peptide
Authors: Larsson, J. N.; Hellstrand, E.; Hammarström, P.; Nyström, S.
Score: 1834.3, Published: 2023-09-01 DOI: 10.1101/2023.09.01.555834
An increasing number of reports suggest an association between COVID-19 infection and initiation or acceleration of neurodegenerative diseases including Alzheimer&#39;s disease (AD) and Creutzfeldt-Jakob disease (CJD). Both these diseases and several other neurodegenerative diseases are caused by conversion of human proteins into a misfolded, aggregated amyloid fibril state."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "neuroscience",
      "item": "https://trxiv.yorks0n.com/posts/neuroscience/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "neuroscience",
  "name": "neuroscience",
  "description": "SARS-CoV-2 Spike amyloid fibrils specifically and selectively accelerates amyloid fibril formation of human prion protein and the amyloid β peptide\nAuthors: Larsson, J. N.; Hellstrand, E.; Hammarström, P.; Nyström, S.\nScore: 1834.3, Published: 2023-09-01 DOI: 10.1101/2023.09.01.555834\nAn increasing number of reports suggest an association between COVID-19 infection and initiation or acceleration of neurodegenerative diseases including Alzheimer\u0026#39;s disease (AD) and Creutzfeldt-Jakob disease (CJD). Both these diseases and several other neurodegenerative diseases are caused by conversion of human proteins into a misfolded, aggregated amyloid fibril state.",
  "keywords": [
    
  ],
  "articleBody": " SARS-CoV-2 Spike amyloid fibrils specifically and selectively accelerates amyloid fibril formation of human prion protein and the amyloid β peptide\nAuthors: Larsson, J. N.; Hellstrand, E.; Hammarström, P.; Nyström, S.\nScore: 1834.3, Published: 2023-09-01 DOI: 10.1101/2023.09.01.555834\nAn increasing number of reports suggest an association between COVID-19 infection and initiation or acceleration of neurodegenerative diseases including Alzheimer's disease (AD) and Creutzfeldt-Jakob disease (CJD). Both these diseases and several other neurodegenerative diseases are caused by conversion of human proteins into a misfolded, aggregated amyloid fibril state. The fibril formation process is self-perpetuating by seeded conversion from preformed fibril seeds. We recently described a plausible mechanism for amyloid fibril formation of SARS-CoV-2 spike protein. Spike-protein formed amyloid fibrils upon cleavage by neutrophil elastase, abundant in the inflammatory response to COVID-19 infection. We here provide evidence of significant Spike-amyloid fibril seeded acceleration of amyloid formation of CJD associated human prion protein (HuPrP) using an in vitro conversion assay. By seeding the HuPrP conversion assay with other in vitro generated disease associated amyloid fibrils we demonstrate that this is not a general effect but a specific feature of spike-amyloid fibrils. We also showed that the amyloid fibril formation of AD associated A{beta}1-42 was accelerated by Spike-amyloid fibril seeds. Of seven different 20-amino acid long peptides, Spike532 (532NLVKNKCVNFNFNGLTGTGV551) was most efficient in seeding HuPrP and Spike601 (601GTNTSNQVAVLYQDVNCTEV620) was most effective in seeding A{beta}1-42, suggesting substrate dependent selectivity of the cross-seeding activity. Albeit purely in vitro, our data suggest that cross-seeding by Spike-amyloid fibrils can be implicated in the increasing number of reports of CJD, AD, and possibly other neurodegenerative diseases in the wake of COVID-19.\nTranslocation of gut commensal bacteria to the brain\nAuthors: Thapa, M.; Kumari, A.; Chin, C.-Y.; Choby, J.; Jin, F.; Bogati, B.; Chopyk, D.; Koduri, N.; Pahnke, A.; Elrod, E.; Burd, E. M.; Weiss, D. S.; Grakoui, A.\nScore: 630.5, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555630\nThe gut-brain axis, a bidirectional signaling network between the intestine and the central nervous system, is crucial to the regulation of host physiology and inflammation. Recent advances suggest a strong correlation between gut dysbiosis and neurological diseases, however, relatively little is known about how gut bacteria impact the brain. Here, we reveal that gut commensal bacteria can translocate directly to the brain when mice are fed an altered diet that causes dysbiosis and intestinal permeability, and that this also occurs without diet alteration in distinct murine models of neurological disease. The bacteria were not found in other systemic sites or the blood, but were detected in the vagus nerve. Unilateral cervical vagotomy significantly reduced the number of bacteria in the brain, implicating the vagus nerve as a conduit for translocation. The presence of bacteria in the brain correlated with microglial activation, a marker of neuroinflammation, and with neural protein aggregation, a hallmark of several neurodegenerative diseases. In at least one model, the presence of bacteria in the brain was reversible as a switch from high-fat to standard diet resulted in amelioration of intestinal permeability, led to a gradual loss of detectable bacteria in the brain, and reduced the number of neural protein aggregates. Further, in murine models of Alzheimers disease, Parkinsons disease, and autism spectrum disorder, we observed gut dysbiosis, gut leakiness, bacterial translocation to the brain, and microglial activation. These data reveal a commensal bacterial translocation axis to the brain in models of diverse neurological diseases.\nSARS-CoV-2 RNA Persists in the Central Nervous System of Non-Human Primates Despite Clinical Recovery\nAuthors: Li, H.; McLaurin, K. A.; Mactutus, C. F.; Rappaport, J.; Datta, P. K.; Booze, R.\nScore: 129.0, Published: 2023-08-30 DOI: 10.1101/2023.08.29.555368\nAdverse neurological and psychiatric outcomes, collectively termed the post-acute sequelae of SARS-CoV-2 infection (PASC), persist in adults clinically recovered from COVID-19. Effective therapeutic interventions are fundamental to reducing the burden of PASC, necessitating an investigation of the pathophysiology underlying the debilitating neurological symptoms associated with the condition. Herein, eight non-human primates (Wild-Caught African Green Monkeys, n=4; Indian Rhesus Macaques, n=4) were inoculated with the SARS-CoV-2 isolate USA-WA1/2020 by either small particle aerosol or via multiple routes. At necropsy, tissue from the olfactory epithelium and pyriform cortex/amygdala of SARS-CoV-2 infected non-human primates were collected for ribonucleic acid in situ hybridization (i.e., RNAscope). First, angiotensin-converting enzyme 2 (ACE2) and transmembrane serine protease 2 (TMPRSS2) mRNA are downregulated in the pyriform cortex/amygdala of non-human primates clinically recovered from SARS-CoV-2 inoculation relative to wildtype controls. Second, abundant SARS-CoV-2 mRNA was detected in clinically recovered non-human primates; mRNA which is predominantly harbored in pericytes. Collectively, examination of post-mortem pyriform cortex/amygdala brain tissue of non-human primates clinically recovered from SARS-CoV-2 infection revealed two early pathophysiological mechanisms potentially underlying PASC. Indeed, therapeutic interventions targeting the downregulation of ACE2, decreased expression of TMPRSS2, and/or persistent infection of pericytes in the central nervous system may effectively mitigate the debilitating symptoms of PASC.\nA unified open-source platform for multimodal neural recording and perturbation during naturalistic behavior\nAuthors: Newman, J. P.; Zhang, J.; Cuevas-Lopez, A.; Miller, N. J.; Honda, T.; van der Goes, M.-S. H.; Leighton, A. H.; Carvalho, F.; Lopes, G.; Lakunina, A.; Siegle, J. H.; Harnett, M. T.; Wilson, M. A.; Voigts, J.\nScore: 90.1, Published: 2023-09-01 DOI: 10.1101/2023.08.30.554672\nBehavioral neuroscience faces two conflicting demands: long-duration recordings from large neural populations and unimpeded animal behavior. To meet this challenge, we developed ONIX, an open-source data acquisition system with high data throughput (2GB/sec) and low closed-loop latencies (\u003c1ms) that uses a novel 0.3 mm thin tether to minimize behavioral impact. Head position and rotation are tracked in 3D and used to drive active commutation without torque measurements. ONIX can acquire from combinations of passive electrodes, Neuropixels probes, head-mounted microscopes, cameras, 3D-trackers, and other data sources. We used ONIX to perform uninterrupted, long ([~]7 hours) neural recordings in mice as they traversed complex 3-dimensional terrain. ONIX allowed exploration with similar mobility as non-implanted animals, in contrast to conventional tethered systems which restricted movement. By combining long recordings with full mobility, our technology will enable new progress on questions that require high-quality neural recordings during ethologically grounded behaviors.\nThe entire brain, more or less is at work: 'Language regions' are artefacts of averaging\nAuthors: Aliko, S.; Wang, B.; Small, S. L.; Skipper, J. I.\nScore: 81.8, Published: 2023-09-03 DOI: 10.1101/2023.09.01.555886\nModels of the neurobiology of language suggest that a small number of anatomically fixed brain regions are responsible for language functioning. This observation derives from centuries of examining brain lesions causing aphasia and is supported by decades of neuroimaging studies. The latter rely on thresholded measures of central tendency applied to activity patterns resulting from heterogeneous stimuli. We hypothesised that these methods obscure the whole brain distribution of regions supporting language. Specifically, cortical 'language regions' and the corresponding 'language network' consist of input regions and connectivity hubs. The latter primarily coordinate peripheral regions whose activity is variable, making them likely to be averaged out following thresholding. We tested these hypotheses in two studies using neuroimaging meta-analyses and functional magnetic resonance imaging during film watching. Both converged to suggest that averaging over heterogeneous words is localised to regions historically associated with language but distributed throughout most of the brain when not averaging over the sensorimotor properties of those words. The localised word regions are composed of highly central hubs. The film data further shows that these hubs are dynamic, connected to peripheral regions, and only appear in the aggregate across time. Results suggest that 'language regions' are an artefact of indiscriminately averaging across heterogeneous language representations and linguistic processes. Rather, they are mostly dynamic connectivity hubs coordinating whole-brain distributions of networks for processing the complexities of real-world language use, explaining why damage to them results in aphasia.\nUltra-high density electrodes improve detection, yield, and cell type specificity of brain recordings\nAuthors: Ye, Z.; Shelton, A. M.; Shaker, J. R.; Boussard, J. M.; Colonell, J.; Minavi, S.; Chen, S.; Windolf, C.; Hurwitz, C.; Namima, T.; Pedraja, F.; Weiss, S.; Raducanu, B.; Ness, T.; Einevoll, G. T.; Laurent, G.; Sawtell, N. B.; Bair, W.; Pasupathy, A.; Mora-Lopez, C.; Dutta, B.; Paninski, L.; Siegle, J. H.; Koch, C.; Olsen, S. R.; Harris, T. D.; Steinmetz, N. A.\nScore: 121.3, Published: 2023-08-25 DOI: 10.1101/2023.08.23.554527\nTo study the neural basis of behavior, we require methods to sensitively and accurately measure neural activity at single neuron and single spike resolution. Extracellular electrophysiology is a principal method for achieving this, but it has biases in the neurons it detects and it imperfectly resolves their action potentials. To overcome these limitations, we developed a silicon probe with significantly smaller and denser recording sites than previous designs, called Neuropixels Ultra (NP Ultra). This device measures neuronal activity at ultra-high densities (\u003e1300 sites per mm, 10 times higher than previous probes), with 6 {micro}m center-to-center spacing and low noise. This device effectively comprises an implantable voltage-sensing camera that captures a planar image of a neurons electrical field. We introduce a new spike sorting algorithm optimized for these probes and use it to find that the yield of visually-responsive neurons in recordings from mouse visual cortex improves [~]3-fold. Recordings across multiple brain regions and four species revealed a subset of unexpectedly small extracellular action potentials not previously reported. Further experiments determined that, in visual cortex, these do not correspond to major subclasses of interneurons and instead likely reflect recordings from axons. Finally, using ground-truth identification of cortical inhibitory cell types with optotagging, we found that cell type was discriminable with approximately 75% success among three types, a significant improvement over lower-resolution recordings. NP Ultra improves spike sorting performance, sampling bias, and cell type classification.\nCryo-EM structures of Tau filaments from the brains of mice transgenic for human mutant P301S Tau\nAuthors: Schweighauser, M.; Murzin, A. G.; McDonald, J.; Lavenir, I. L.; Crowther, R. A.; Scheres, S.; Goedert, M.\nScore: 26.6, Published: 2023-09-01 DOI: 10.1101/2023.08.31.555656\nMice transgenic for human mutant P301S tau are widely used as models for human tauopathies. They develop neurodegeneration and abundant filamentous inclusions made of human mutant four-repeat tau. Here we used electron cryo-microscopy (cryo-EM) to determine the structures of tau filaments from the brains of Tg2541 and PS19 mice. Both lines express human P301S tau (0N4R for Tg2541 and 1N4R for PS19) on mixed genetic backgrounds and downstream of different promoters (murine Thy1 for Tg2541 and murine Prnp for PS19). The structures of tau filaments from Tg2541 and PS19 mice differ from each other and those of tau filaments from human brains. However, they share a substructure at the junction of repeats 2 and 3, which comprises residues I297-V312 of tau and includes the P301S mutation. The filament core from the brainstem of Tg2541 mice consists of residues K274-H329 of tau and two disconnected protein densities. Two non-proteinaceous densities are also in evidence. The filament core from the cerebral cortex of line PS19 extends from residues G271-P364 of tau. One strong non-proteinaceous density is also present. Unlike the tau filaments from human brains, the sequences following repeat 4 are missing from the cores of tau filaments from the brains of Tg2541 and PS19 mice.\nPrenatal environment is associated with the pace of cortical network development over the first three years of life\nAuthors: Tooley, U.; Latham, A.; Kenley, J. K.; Alexopoulos, D.; Smyser, T.; Warner, B. B.; Shimony, J. S.; Neil, J. J.; Luby, J. L.; Barch, D. M.; Rogers, C. E.; Smyser, C. D.\nScore: 26.3, Published: 2023-08-21 DOI: 10.1101/2023.08.18.552639\nEnvironmental influences on brain structure and function during early development have been well-characterized. In pre-registered analyses, we test the theory that socioeconomic status (SES) is associated with differences in trajectories of intrinsic brain network development from birth to three years (n = 261). Prenatal SES is associated with developmental increases in cortical network segregation, with neonates and toddlers from lower-SES backgrounds showing a steeper increase in cortical network segregation with age, consistent with accelerated network development. Associations between SES and cortical network segregation occur at the local scale and conform to a sensorimotor-association hierarchy of cortical organization. SES-associated differences in cortical network segregation are associated with language abilities at two years, such that lower segregation is associated with improved language abilities. These results yield key insight into the timing and directionality of associations between the early environment and trajectories of cortical development.\nOpto4E-BP, an optogenetic tool for inducible, reversible, and cell type-specific inhibition of translation initiation\nAuthors: Alapin, J. M.; Mohamed, M. S.; Shrestha, P.; Khaled, H. G.; Vorabyeva, A. G.; Bowling, H. L.; Oliveira, M. M.; Klann, E.\nScore: 19.9, Published: 2023-08-31 DOI: 10.1101/2023.08.30.554643\nThe protein kinase mechanistic target of rapamycin complex 1 (mTORC1) is one of the primary triggers for initiating cap-dependent translation. Amongst its functions, mTORC1 phosphorylates eIF4E-binding proteins (4E-BPs), which prevents them from binding to eIF4E and thereby enables translation initiation. mTORC1 signaling is required for multiple forms of protein synthesis-dependent synaptic plasticity and various forms of long-term memory (LTM), including associative threat memory. However, the approaches used thus far to target mTORC1 and its effectors, such as pharmacological inhibitors or genetic knockouts, lack fine spatial and temporal control. The development of a conditional and inducible eIF4E knockdown mouse line partially solved the issue of spatial control, but still lacked optimal temporal control to study memory consolidation. Here, we have designed a novel optogenetic tool (Opto4E-BP) for cell type-specific, light-dependent regulation of eIF4E in the brain. We show that light-activation of Opto4E-BP decreases protein synthesis in HEK cells and primary mouse neurons. In situ, light-activation of Opto4E-BP in excitatory neurons decreased protein synthesis in acute amygdala slices. Finally, light activation of Opto4E-BP in principal excitatory neurons in the lateral amygdala (LA) of mice after training blocked the consolidation of LTM. The development of this novel optogenetic tool to modulate eIF4E-dependent translation with spatiotemporal precision will permit future studies to unravel the complex relationship between protein synthesis and the consolidation of LTM.\nCharacterization of Identified Dopaminergic Neurons in the Mouse Olfactory Bulb and Midbrain\nAuthors: Lau, M. Y. H.; Gadiwalla, S.; Jones, S.; Galliano, E.\nScore: 19.1, Published: 2023-08-31 DOI: 10.1101/2023.08.29.554772\nDopaminergic (DA) neurons play pivotal roles in diverse brain functions, spanning movement, reward processing, and sensory perception. DA neurons are most abundant in the midbrain (Substantia Nigra pars compacta, SNC, and Ventral Tegmental Area, VTA) and the olfactory bulb (OB) in the forebrain. Interestingly, a subtype of OB DA neurons is capable of regenerating throughout life, while a second class is exclusively born during embryonic development. Emerging evidence in SNC and VTA also indicates substantial heterogeneity in terms of morphology, connectivity, and function. To further investigate this heterogeneity and directly compare form and function of midbrain and forebrain DA neurons, we performed immunohistochemistry and whole-cell patch-clamp recordings in ex vivo brain slices from juvenile DAT-tdTomato mice. After confirming the penetrance and specificity of the dopamine transporter (DAT) Cre line, we compared soma shape, passive membrane properties, voltage sags and action potential firing across midbrain and forebrain DA subtypes. We found that each DA subgroup within midbrain and forebrain was highly heterogeneous, and that DA neurons across the two brain areas are also substantially different. These findings complement previous work in rats as well as gene expression and in vivo datasets, further questioning the existence of a single \"dopaminergic\" neuronal phenotype.\n",
  "wordCount" : "2520",
  "inLanguage": "en",
  "datePublished": "2023-09-06T10:39:07Z",
  "dateModified": "2023-09-06T10:39:07Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/neuroscience/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      neuroscience
    </h1>
    <div class="post-meta"><span>updated on September 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555834">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555834" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555834">
        <p class="paperTitle">SARS-CoV-2 Spike amyloid fibrils specifically and selectively accelerates amyloid fibril formation of human prion protein and the amyloid β peptide</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555834" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555834" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Larsson, J. N.; Hellstrand, E.; Hammarström, P.; Nyström, S.</p>
        <p class="info">Score: 1834.3, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555834' target='https://doi.org/10.1101/2023.09.01.555834'> 10.1101/2023.09.01.555834</a></p>
        <p class="abstract">An increasing number of reports suggest an association between COVID-19 infection and initiation or acceleration of neurodegenerative diseases including Alzheimer&#39;s disease (AD) and Creutzfeldt-Jakob disease (CJD). Both these diseases and several other neurodegenerative diseases are caused by conversion of human proteins into a misfolded, aggregated amyloid fibril state. The fibril formation process is self-perpetuating by seeded conversion from preformed fibril seeds. We recently described a plausible mechanism for amyloid fibril formation of SARS-CoV-2 spike protein. Spike-protein formed amyloid fibrils upon cleavage by neutrophil elastase, abundant in the inflammatory response to COVID-19 infection. We here provide evidence of significant Spike-amyloid fibril seeded acceleration of amyloid formation of CJD associated human prion protein (HuPrP) using an in vitro conversion assay. By seeding the HuPrP conversion assay with other in vitro generated disease associated amyloid fibrils we demonstrate that this is not a general effect but a specific feature of spike-amyloid fibrils. We also showed that the amyloid fibril formation of AD associated A{beta}1-42 was accelerated by Spike-amyloid fibril seeds. Of seven different 20-amino acid long peptides, Spike532 (532NLVKNKCVNFNFNGLTGTGV551) was most efficient in seeding HuPrP and Spike601 (601GTNTSNQVAVLYQDVNCTEV620) was most effective in seeding A{beta}1-42, suggesting substrate dependent selectivity of the cross-seeding activity. Albeit purely in vitro, our data suggest that cross-seeding by Spike-amyloid fibrils can be implicated in the increasing number of reports of CJD, AD, and possibly other neurodegenerative diseases in the wake of COVID-19.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555630">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555630" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555630">
        <p class="paperTitle">Translocation of gut commensal bacteria to the brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555630" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555630" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thapa, M.; Kumari, A.; Chin, C.-Y.; Choby, J.; Jin, F.; Bogati, B.; Chopyk, D.; Koduri, N.; Pahnke, A.; Elrod, E.; Burd, E. M.; Weiss, D. S.; Grakoui, A.</p>
        <p class="info">Score: 630.5, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555630' target='https://doi.org/10.1101/2023.08.30.555630'> 10.1101/2023.08.30.555630</a></p>
        <p class="abstract">The gut-brain axis, a bidirectional signaling network between the intestine and the central nervous system, is crucial to the regulation of host physiology and inflammation. Recent advances suggest a strong correlation between gut dysbiosis and neurological diseases, however, relatively little is known about how gut bacteria impact the brain. Here, we reveal that gut commensal bacteria can translocate directly to the brain when mice are fed an altered diet that causes dysbiosis and intestinal permeability, and that this also occurs without diet alteration in distinct murine models of neurological disease. The bacteria were not found in other systemic sites or the blood, but were detected in the vagus nerve. Unilateral cervical vagotomy significantly reduced the number of bacteria in the brain, implicating the vagus nerve as a conduit for translocation. The presence of bacteria in the brain correlated with microglial activation, a marker of neuroinflammation, and with neural protein aggregation, a hallmark of several neurodegenerative diseases. In at least one model, the presence of bacteria in the brain was reversible as a switch from high-fat to standard diet resulted in amelioration of intestinal permeability, led to a gradual loss of detectable bacteria in the brain, and reduced the number of neural protein aggregates. Further, in murine models of Alzheimers disease, Parkinsons disease, and autism spectrum disorder, we observed gut dysbiosis, gut leakiness, bacterial translocation to the brain, and microglial activation. These data reveal a commensal bacterial translocation axis to the brain in models of diverse neurological diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555368">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555368" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555368">
        <p class="paperTitle">SARS-CoV-2 RNA Persists in the Central Nervous System of Non-Human Primates Despite Clinical Recovery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555368" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555368" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, H.; McLaurin, K. A.; Mactutus, C. F.; Rappaport, J.; Datta, P. K.; Booze, R.</p>
        <p class="info">Score: 129.0, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555368' target='https://doi.org/10.1101/2023.08.29.555368'> 10.1101/2023.08.29.555368</a></p>
        <p class="abstract">Adverse neurological and psychiatric outcomes, collectively termed the post-acute sequelae of SARS-CoV-2 infection (PASC), persist in adults clinically recovered from COVID-19. Effective therapeutic interventions are fundamental to reducing the burden of PASC, necessitating an investigation of the pathophysiology underlying the debilitating neurological symptoms associated with the condition. Herein, eight non-human primates (Wild-Caught African Green Monkeys, n=4; Indian Rhesus Macaques, n=4) were inoculated with the SARS-CoV-2 isolate USA-WA1/2020 by either small particle aerosol or via multiple routes. At necropsy, tissue from the olfactory epithelium and pyriform cortex/amygdala of SARS-CoV-2 infected non-human primates were collected for ribonucleic acid in situ hybridization (i.e., RNAscope). First, angiotensin-converting enzyme 2 (ACE2) and transmembrane serine protease 2 (TMPRSS2) mRNA are downregulated in the pyriform cortex/amygdala of non-human primates clinically recovered from SARS-CoV-2 inoculation relative to wildtype controls. Second, abundant SARS-CoV-2 mRNA was detected in clinically recovered non-human primates; mRNA which is predominantly harbored in pericytes. Collectively, examination of post-mortem pyriform cortex/amygdala brain tissue of non-human primates clinically recovered from SARS-CoV-2 infection revealed two early pathophysiological mechanisms potentially underlying PASC. Indeed, therapeutic interventions targeting the downregulation of ACE2, decreased expression of TMPRSS2, and/or persistent infection of pericytes in the central nervous system may effectively mitigate the debilitating symptoms of PASC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.554672">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.554672" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.554672">
        <p class="paperTitle">A unified open-source platform for multimodal neural recording and perturbation during naturalistic behavior</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.554672" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.554672" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Newman, J. P.; Zhang, J.; Cuevas-Lopez, A.; Miller, N. J.; Honda, T.; van der Goes, M.-S. H.; Leighton, A. H.; Carvalho, F.; Lopes, G.; Lakunina, A.; Siegle, J. H.; Harnett, M. T.; Wilson, M. A.; Voigts, J.</p>
        <p class="info">Score: 90.1, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.554672' target='https://doi.org/10.1101/2023.08.30.554672'> 10.1101/2023.08.30.554672</a></p>
        <p class="abstract">Behavioral neuroscience faces two conflicting demands: long-duration recordings from large neural populations and unimpeded animal behavior. To meet this challenge, we developed ONIX, an open-source data acquisition system with high data throughput (2GB/sec) and low closed-loop latencies (&lt;1ms) that uses a novel 0.3 mm thin tether to minimize behavioral impact. Head position and rotation are tracked in 3D and used to drive active commutation without torque measurements. ONIX can acquire from combinations of passive electrodes, Neuropixels probes, head-mounted microscopes, cameras, 3D-trackers, and other data sources. We used ONIX to perform uninterrupted, long ([~]7 hours) neural recordings in mice as they traversed complex 3-dimensional terrain. ONIX allowed exploration with similar mobility as non-implanted animals, in contrast to conventional tethered systems which restricted movement. By combining long recordings with full mobility, our technology will enable new progress on questions that require high-quality neural recordings during ethologically grounded behaviors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555886">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555886" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555886">
        <p class="paperTitle">The entire brain, more or less is at work: &#39;Language regions&#39; are artefacts of averaging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555886" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555886" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aliko, S.; Wang, B.; Small, S. L.; Skipper, J. I.</p>
        <p class="info">Score: 81.8, Published: 2023-09-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555886' target='https://doi.org/10.1101/2023.09.01.555886'> 10.1101/2023.09.01.555886</a></p>
        <p class="abstract">Models of the neurobiology of language suggest that a small number of anatomically fixed brain regions are responsible for language functioning. This observation derives from centuries of examining brain lesions causing aphasia and is supported by decades of neuroimaging studies. The latter rely on thresholded measures of central tendency applied to activity patterns resulting from heterogeneous stimuli. We hypothesised that these methods obscure the whole brain distribution of regions supporting language. Specifically, cortical &#39;language regions&#39; and the corresponding &#39;language network&#39; consist of input regions and connectivity hubs. The latter primarily coordinate peripheral regions whose activity is variable, making them likely to be averaged out following thresholding. We tested these hypotheses in two studies using neuroimaging meta-analyses and functional magnetic resonance imaging during film watching. Both converged to suggest that averaging over heterogeneous words is localised to regions historically associated with language but distributed throughout most of the brain when not averaging over the sensorimotor properties of those words. The localised word regions are composed of highly central hubs. The film data further shows that these hubs are dynamic, connected to peripheral regions, and only appear in the aggregate across time. Results suggest that &#39;language regions&#39; are an artefact of indiscriminately averaging across heterogeneous language representations and linguistic processes. Rather, they are mostly dynamic connectivity hubs coordinating whole-brain distributions of networks for processing the complexities of real-world language use, explaining why damage to them results in aphasia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554527">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554527" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554527">
        <p class="paperTitle">Ultra-high density electrodes improve detection, yield, and cell type specificity of brain recordings</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554527" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554527" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ye, Z.; Shelton, A. M.; Shaker, J. R.; Boussard, J. M.; Colonell, J.; Minavi, S.; Chen, S.; Windolf, C.; Hurwitz, C.; Namima, T.; Pedraja, F.; Weiss, S.; Raducanu, B.; Ness, T.; Einevoll, G. T.; Laurent, G.; Sawtell, N. B.; Bair, W.; Pasupathy, A.; Mora-Lopez, C.; Dutta, B.; Paninski, L.; Siegle, J. H.; Koch, C.; Olsen, S. R.; Harris, T. D.; Steinmetz, N. A.</p>
        <p class="info">Score: 121.3, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554527' target='https://doi.org/10.1101/2023.08.23.554527'> 10.1101/2023.08.23.554527</a></p>
        <p class="abstract">To study the neural basis of behavior, we require methods to sensitively and accurately measure neural activity at single neuron and single spike resolution. Extracellular electrophysiology is a principal method for achieving this, but it has biases in the neurons it detects and it imperfectly resolves their action potentials. To overcome these limitations, we developed a silicon probe with significantly smaller and denser recording sites than previous designs, called Neuropixels Ultra (NP Ultra). This device measures neuronal activity at ultra-high densities (&gt;1300 sites per mm, 10 times higher than previous probes), with 6 {micro}m center-to-center spacing and low noise. This device effectively comprises an implantable voltage-sensing camera that captures a planar image of a neurons electrical field. We introduce a new spike sorting algorithm optimized for these probes and use it to find that the yield of visually-responsive neurons in recordings from mouse visual cortex improves [~]3-fold. Recordings across multiple brain regions and four species revealed a subset of unexpectedly small extracellular action potentials not previously reported. Further experiments determined that, in visual cortex, these do not correspond to major subclasses of interneurons and instead likely reflect recordings from axons. Finally, using ground-truth identification of cortical inhibitory cell types with optotagging, we found that cell type was discriminable with approximately 75% success among three types, a significant improvement over lower-resolution recordings. NP Ultra improves spike sorting performance, sampling bias, and cell type classification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555656">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555656" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555656">
        <p class="paperTitle">Cryo-EM structures of Tau filaments from the brains of mice transgenic for human mutant P301S Tau</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555656" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555656" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schweighauser, M.; Murzin, A. G.; McDonald, J.; Lavenir, I. L.; Crowther, R. A.; Scheres, S.; Goedert, M.</p>
        <p class="info">Score: 26.6, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555656' target='https://doi.org/10.1101/2023.08.31.555656'> 10.1101/2023.08.31.555656</a></p>
        <p class="abstract">Mice transgenic for human mutant P301S tau are widely used as models for human tauopathies. They develop neurodegeneration and abundant filamentous inclusions made of human mutant four-repeat tau. Here we used electron cryo-microscopy (cryo-EM) to determine the structures of tau filaments from the brains of Tg2541 and PS19 mice. Both lines express human P301S tau (0N4R for Tg2541 and 1N4R for PS19) on mixed genetic backgrounds and downstream of different promoters (murine Thy1 for Tg2541 and murine Prnp for PS19). The structures of tau filaments from Tg2541 and PS19 mice differ from each other and those of tau filaments from human brains. However, they share a substructure at the junction of repeats 2 and 3, which comprises residues I297-V312 of tau and includes the P301S mutation. The filament core from the brainstem of Tg2541 mice consists of residues K274-H329 of tau and two disconnected protein densities. Two non-proteinaceous densities are also in evidence. The filament core from the cerebral cortex of line PS19 extends from residues G271-P364 of tau. One strong non-proteinaceous density is also present. Unlike the tau filaments from human brains, the sequences following repeat 4 are missing from the cores of tau filaments from the brains of Tg2541 and PS19 mice.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.552639">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.552639" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.552639">
        <p class="paperTitle">Prenatal environment is associated with the pace of cortical network development over the first three years of life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.552639" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.552639" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tooley, U.; Latham, A.; Kenley, J. K.; Alexopoulos, D.; Smyser, T.; Warner, B. B.; Shimony, J. S.; Neil, J. J.; Luby, J. L.; Barch, D. M.; Rogers, C. E.; Smyser, C. D.</p>
        <p class="info">Score: 26.3, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.552639' target='https://doi.org/10.1101/2023.08.18.552639'> 10.1101/2023.08.18.552639</a></p>
        <p class="abstract">Environmental influences on brain structure and function during early development have been well-characterized. In pre-registered analyses, we test the theory that socioeconomic status (SES) is associated with differences in trajectories of intrinsic brain network development from birth to three years (n = 261). Prenatal SES is associated with developmental increases in cortical network segregation, with neonates and toddlers from lower-SES backgrounds showing a steeper increase in cortical network segregation with age, consistent with accelerated network development. Associations between SES and cortical network segregation occur at the local scale and conform to a sensorimotor-association hierarchy of cortical organization. SES-associated differences in cortical network segregation are associated with language abilities at two years, such that lower segregation is associated with improved language abilities. These results yield key insight into the timing and directionality of associations between the early environment and trajectories of cortical development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.554643">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.554643" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.554643">
        <p class="paperTitle">Opto4E-BP, an optogenetic tool for inducible, reversible, and cell type-specific inhibition of translation initiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.554643" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.554643" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alapin, J. M.; Mohamed, M. S.; Shrestha, P.; Khaled, H. G.; Vorabyeva, A. G.; Bowling, H. L.; Oliveira, M. M.; Klann, E.</p>
        <p class="info">Score: 19.9, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.554643' target='https://doi.org/10.1101/2023.08.30.554643'> 10.1101/2023.08.30.554643</a></p>
        <p class="abstract">The protein kinase mechanistic target of rapamycin complex 1 (mTORC1) is one of the primary triggers for initiating cap-dependent translation. Amongst its functions, mTORC1 phosphorylates eIF4E-binding proteins (4E-BPs), which prevents them from binding to eIF4E and thereby enables translation initiation. mTORC1 signaling is required for multiple forms of protein synthesis-dependent synaptic plasticity and various forms of long-term memory (LTM), including associative threat memory. However, the approaches used thus far to target mTORC1 and its effectors, such as pharmacological inhibitors or genetic knockouts, lack fine spatial and temporal control. The development of a conditional and inducible eIF4E knockdown mouse line partially solved the issue of spatial control, but still lacked optimal temporal control to study memory consolidation. Here, we have designed a novel optogenetic tool (Opto4E-BP) for cell type-specific, light-dependent regulation of eIF4E in the brain. We show that light-activation of Opto4E-BP decreases protein synthesis in HEK cells and primary mouse neurons. In situ, light-activation of Opto4E-BP in excitatory neurons decreased protein synthesis in acute amygdala slices. Finally, light activation of Opto4E-BP in principal excitatory neurons in the lateral amygdala (LA) of mice after training blocked the consolidation of LTM. The development of this novel optogenetic tool to modulate eIF4E-dependent translation with spatiotemporal precision will permit future studies to unravel the complex relationship between protein synthesis and the consolidation of LTM.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.554772">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.554772" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.554772">
        <p class="paperTitle">Characterization of Identified Dopaminergic Neurons in the Mouse Olfactory Bulb and Midbrain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.554772" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.554772" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lau, M. Y. H.; Gadiwalla, S.; Jones, S.; Galliano, E.</p>
        <p class="info">Score: 19.1, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.554772' target='https://doi.org/10.1101/2023.08.29.554772'> 10.1101/2023.08.29.554772</a></p>
        <p class="abstract">Dopaminergic (DA) neurons play pivotal roles in diverse brain functions, spanning movement, reward processing, and sensory perception. DA neurons are most abundant in the midbrain (Substantia Nigra pars compacta, SNC, and Ventral Tegmental Area, VTA) and the olfactory bulb (OB) in the forebrain. Interestingly, a subtype of OB DA neurons is capable of regenerating throughout life, while a second class is exclusively born during embryonic development. Emerging evidence in SNC and VTA also indicates substantial heterogeneity in terms of morphology, connectivity, and function. To further investigate this heterogeneity and directly compare form and function of midbrain and forebrain DA neurons, we performed immunohistochemistry and whole-cell patch-clamp recordings in ex vivo brain slices from juvenile DAT-tdTomato mice. After confirming the penetrance and specificity of the dopamine transporter (DAT) Cre line, we compared soma shape, passive membrane properties, voltage sags and action potential firing across midbrain and forebrain DA subtypes. We found that each DA subgroup within midbrain and forebrain was highly heterogeneous, and that DA neurons across the two brain areas are also substantially different. These findings complement previous work in rats as well as gene expression and in vivo datasets, further questioning the existence of a single &#34;dopaminergic&#34; neuronal phenotype.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
