<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>neuroscience</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>neuroscience | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Constructed languages are processed by the same brain mechanisms as natural languages
Authors: Malik-Moraleda, S.; Taliaferro, M.; Shannon, S.; Jhingan, N.; Swords, S.; Peterson, D. J.; Frommer, P.; Okrand, M.; Sams, J.; Cardwell, R.; Freeman, C.; Fedorenko, E.
Score: 160.1, Published: 2023-07-28 DOI: 10.1101/2023.07.28.550667
What constitutes a language? Natural languages share some features with other domains: from math, to music, to gesture. However, the brain mechanisms that process linguistic input are highly specialized, showing little or no response to diverse non-linguistic tasks.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/neuroscience/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="neuroscience" />
<meta property="og:description" content="Constructed languages are processed by the same brain mechanisms as natural languages
Authors: Malik-Moraleda, S.; Taliaferro, M.; Shannon, S.; Jhingan, N.; Swords, S.; Peterson, D. J.; Frommer, P.; Okrand, M.; Sams, J.; Cardwell, R.; Freeman, C.; Fedorenko, E.
Score: 160.1, Published: 2023-07-28 DOI: 10.1101/2023.07.28.550667
What constitutes a language? Natural languages share some features with other domains: from math, to music, to gesture. However, the brain mechanisms that process linguistic input are highly specialized, showing little or no response to diverse non-linguistic tasks." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/neuroscience/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-06T10:36:48+00:00" />
<meta property="article:modified_time" content="2023-08-06T10:36:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="neuroscience"/>
<meta name="twitter:description" content="Constructed languages are processed by the same brain mechanisms as natural languages
Authors: Malik-Moraleda, S.; Taliaferro, M.; Shannon, S.; Jhingan, N.; Swords, S.; Peterson, D. J.; Frommer, P.; Okrand, M.; Sams, J.; Cardwell, R.; Freeman, C.; Fedorenko, E.
Score: 160.1, Published: 2023-07-28 DOI: 10.1101/2023.07.28.550667
What constitutes a language? Natural languages share some features with other domains: from math, to music, to gesture. However, the brain mechanisms that process linguistic input are highly specialized, showing little or no response to diverse non-linguistic tasks."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "neuroscience",
      "item": "https://trxiv.yorks0n.com/posts/neuroscience/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "neuroscience",
  "name": "neuroscience",
  "description": "Constructed languages are processed by the same brain mechanisms as natural languages\nAuthors: Malik-Moraleda, S.; Taliaferro, M.; Shannon, S.; Jhingan, N.; Swords, S.; Peterson, D. J.; Frommer, P.; Okrand, M.; Sams, J.; Cardwell, R.; Freeman, C.; Fedorenko, E.\nScore: 160.1, Published: 2023-07-28 DOI: 10.1101/2023.07.28.550667\nWhat constitutes a language? Natural languages share some features with other domains: from math, to music, to gesture. However, the brain mechanisms that process linguistic input are highly specialized, showing little or no response to diverse non-linguistic tasks.",
  "keywords": [
    
  ],
  "articleBody": " Constructed languages are processed by the same brain mechanisms as natural languages\nAuthors: Malik-Moraleda, S.; Taliaferro, M.; Shannon, S.; Jhingan, N.; Swords, S.; Peterson, D. J.; Frommer, P.; Okrand, M.; Sams, J.; Cardwell, R.; Freeman, C.; Fedorenko, E.\nScore: 160.1, Published: 2023-07-28 DOI: 10.1101/2023.07.28.550667\nWhat constitutes a language? Natural languages share some features with other domains: from math, to music, to gesture. However, the brain mechanisms that process linguistic input are highly specialized, showing little or no response to diverse non-linguistic tasks. Here, we examine constructed languages (conlangs) to ask whether they draw on the same neural mechanisms as natural languages, or whether they instead pattern with domains like math and logic. Using individual-subject fMRI analyses, we show that understanding conlangs recruits the same brain areas as natural language comprehension. This result holds for Esperanto (n=19 speakers)-- created to resemble natural languages--and fictional conlangs (Klingon (n=10), Navi (n=9), High Valyrian (n=3), and Dothraki (n=3)), created to differ from natural languages, and suggests that conlangs and natural languages share critical features and that the notable differences between conlangs and natural language are not consequential for the cognitive and neural mechanisms that they engage.\nDivergent patterns of healthy aging across human brain regions at single-cell resolution reveal links to neurodegenerative disease\nAuthors: Duffy, M.; Ding, J.; Langston, R. G.; Shah, S. I.; Nalls, M. A.; Scholz, S. W.; Whitaker, D. T.; Auluck, P. K.; Gibbs, J. R.; Marenco, S.; Cookson, M. R.\nScore: 32.1, Published: 2023-08-01 DOI: 10.1101/2023.07.31.551097\nAge is a major common risk factor underlying neurodegenerative diseases, including Alzheimers disease, Parkinsons disease, and amyotrophic lateral sclerosis. Previous studies reported that chronological age correlates with differential gene expression across different brain regions. However, prior datasets have not disambiguated whether expression associations with age are due to changes in cell numbers and/or gene expression per cell. In this study, we leveraged single nucleus RNA-sequencing (snRNAseq) to examine changes in cell proportions and transcriptomes in four different brain regions, each from 12 donors aged 20-30 years (young) or 60-85 years (old). We sampled 155,192 nuclei from two cortical regions (entorhinal cortex and middle temporal gyrus) and two subcortical regions (putamen and subventricular zone) relevant to neurodegenerative diseases or the proliferative niche. We found no changes in cellular composition of different brain regions with healthy aging. Surprisingly, we did find that each brain region has a distinct aging signature, with only minor overlap in differentially associated genes across regions. Moreover, each cell type shows distinct age-associated expression changes, including loss of protein synthesis genes in cortical inhibitory neurons, axonogenesis genes in excitatory neurons and oligodendrocyte precursor cells, enhanced gliosis markers in astrocytes and disease-associated markers in microglia, and genes critical for neuron-glia communication. Importantly, we find cell type-specific enrichments of age associations with genes nominated by Alzheimers disease and Parkinsons disease genome-wide association studies (GWAS), such as apolipoprotein E (APOE), and leucine-rich repeat kinase 2 (LRRK2) in microglia that are independent of overall expression levels across cell types. We present this data as a new resource which highlights, first, region- and cell type-specific transcriptomic changes in healthy aging that may contribute to selective vulnerability and, second, provide context for testing GWAS-nominated disease risk genes in relevant subtypes and developing more targeted therapeutic strategies. The data is readily accessible without requirement for extensive computational support in a public website, https://brainexp-hykyffa56a-uc.a.run.app/ Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=177 SRC=\"FIGDIR/small/551097v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (66K): org.highwire.dtl.DTLVardef@d8e616org.highwire.dtl.DTLVardef@1e9e734org.highwire.dtl.DTLVardef@14a3924org.highwire.dtl.DTLVardef@1ed8cff_HPS_FORMAT_FIGEXP M_FIG *Created using Biorender.com C_FIG HighlightsO_LIEstablishment of a single nuclei atlas of human aging in four brain regions C_LIO_LIEach region and cell type exhibits a unique aging-associated transcriptome signature C_LIO_LIGene expression changes occur in absence of overt cell loss and are categorically unique across cell types C_LIO_LINeurological disease-associated genes have age-associated expression patterns in specific cell types in the context of healthy aging C_LI\nAlpha oscillations support the efficiency of guided visual search by inhibiting both target and distractor features in early visual cortex\nAuthors: Duecker, K.; Shapiro, K.; Hanslmayr, S.; Wolfe, J.; Pan, Y.; Jensen, O.\nScore: 30.3, Published: 2023-08-03 DOI: 10.1101/2023.08.03.551520\nVisual search models have long emphasised that task-relevant items must be prioritized for optimal performance. While it is known that search efficiency also benefits from active distractor inhibition, the underlying neuronal mechanisms are debated. Here, we used MEG in combination with Rapid Invisible Frequency Tagging (RIFT) to understand the neural correlates of feature-guided visual search. RIFT served as a continuous read-out of the neuronal excitability to the search stimuli and revealed evidence for target boosting and distractor suppression in early visual cortex. These findings were complemented by an increase in occipital alpha power predicting faster responses and higher hit rates, as well as reduced RIFT responses to all stimuli, regardless of their task-relevance. We propose that alpha oscillations in early visual regions implement a blanket inhibition that reduces neuronal excitability to both target and distractor features. As the excitability of neurons encoding the target features is boosted, these neurons overcome the inhibition, facilitating guidance towards task-relevant stimuli. Our results provide novel insights on a mechanism in early visual regions that supports selective attention through inhibition.\nDaytime experiences shape neural activity and dream content in the sleeping brain\nAuthors: Kumral, D.; Palmieri, J.; Gais, S.; Schoenauer, M.\nScore: 32.9, Published: 2023-07-29 DOI: 10.1101/2023.07.29.551087\nLearning-related brain activity patterns are replayed during sleep, and memories of recent experiences appear in our dreams. The connection between these phenomena, however, remains unclear. We investigated whether memory reinstatement during sleep contributes to dreaming. Participants listened to audiobooks before falling asleep. We could determine which audiobook they had studied based on dream reports collected during the night. Audiobook content was also reinstated at the neural level, in high-density EEG recordings. Brain activity during rapid eye movement sleep, particularly in the high-frequency beta range, carried information about the audiobook and simultaneously benefitted memory retention. Crucially, when the learning condition was manifest in neural activity, it also emerged in dreams. Reprocessing of daytime experiences during sleep thus shapes our brain activity, our dreams, and our memories.\nThe Human Microglia Atlas (HuMicA) Unravels Changes in Homeostatic and Disease-Associated Microglia Subsets across Neurodegenerative Conditions\nAuthors: Martins-Ferreira, R.; Calafell-Segura, J.; Leal, B.; Rodriguez-Ubreva, J.; Mereu, E.; Pinho e Costa, P.; Ballestar, E.\nScore: 26.8, Published: 2023-08-02 DOI: 10.1101/2023.08.01.550767\nDysregulated microglia activation, leading to neuroinflammation, is crucial in neurodegenerative disease development and progression. The initial M1/M2 dual activation classification for microglia is outdated. Even the disease-associated microglia (DAM) phenotype, firstly described in mice, falls short in representing the diverse microglia phenotypes in pathology. In this study, we have constructed a transcriptomic atlas of human brain immune cells by integrating single-nucleus (sn)RNA-seq datasets from multiple neurodegenerative conditions. Sixteen datasets were included, comprising 295 samples from patients with Alzheimers disease, autism spectrum disorder, epilepsy, multiple sclerosis, Lewy body diseases, COVID-19, and healthy controls. The integrated Human Microglia Atlas (HuMicA) dataset included 60,557 nuclei and revealed 11 microglial subpopulations distributed across all pathological and healthy conditions. Among these, we identified four different homeostatic clusters as well as pathological phenotypes. These included two stages of early and late activation of the DAM phenotype and the disease-inflammatory macrophage (DIM) phenotype, which was recently described in mice, and is also present in human microglia, as indicated by our analysis. The high versatility of microglia is evident through changes in subset distribution across various pathologies, suggesting their contribution in shaping pathological phenotypes. Our analysis showed overall depletion of four substates of homeostatic microglia, and expansion of niche subpopulations within the DAM and DIM spectrum across distinct neurodegenerative pathologies. The HuMicA is invaluable in advancing the study of microglia biology in both healthy and disease settings.\nDistinct roles of putative excitatory and inhibitory neurons in the macaque inferior temporal cortex in core object recognition behavior\nAuthors: Sanghavi, S.; Kar, K.\nScore: 22.4, Published: 2023-08-03 DOI: 10.1101/2023.08.01.551579\nA spatially distributed population of neurons in the macaque inferior temporal (IT) cortex supports object recognition behavior, but the cell-type specificity of the population in forming \"behaviorally sufficient\" object decodes remain unclear. To address this, we recorded neural signals from the macaque IT cortex and compared the object identity information and the alignment of decoding strategies derived from putative inhibitory (Inh) and excitatory (Exc) neurons to the monkeys behavior. We observed that while Inh neurons represented significant category information, decoding strategies based on Exc neural population activity outperformed those from Inh neurons in overall accuracy and their image-level match to the monkeys behavioral reports. Interestingly, both Exc and Inh responses explained a fraction of unique variance of the monkeys behavior, demonstrating a distinct role of the two cell types in generating object identity solutions for a downstream readout. We observed that current artificial neural network (ANN) models of primate ventral stream, designed with AI goals of performance optimization on image categorization, better predict Exc neurons (and its contribution to object recognition behavior) than Inh neurons. Beyond, the refinement of linking propositions between IT and object recognition behavior, our results guide the development of more biologically constrained brain models by offering novel cell-type specific neural benchmarks.\nRastermap: a discovery method for neural population recordings\nAuthors: Stringer, C.; Zhong, L.; Syeda, A.; Du, F.; Pachitariu, M.\nScore: 46.8, Published: 2023-07-28 DOI: 10.1101/2023.07.25.550571\nNeurophysiology has long progressed through exploratory experiments and chance discoveries. Anecdotes abound of researchers setting up experiments while listening to spikes in real time and observing a pattern of consistent firing when certain stimuli or behaviors happened. With the advent of large-scale recordings, such close observation of data has become harder because high-dimensional spaces are impenetrable to our pattern-finding intuitions. To help ourselves find patterns in neural data, our lab has been openly developing a visualization framework known as \"Rastermap\" over the past five years. Rastermap takes advantage of a new global optimization algorithm for sorting neural responses along a one-dimensional manifold. Displayed as a raster plot, the sorted neurons show a variety of activity patterns, which can be more easily identified and interpreted. We first benchmark Rastermap on realistic simulations with multiplexed cognitive variables. Then we demonstrate it on recordings of tens of thousands of neurons from mouse visual and sensorimotor cortex during spontaneous, stimulus-evoked and task-evoked epochs, as well as on whole-brain zebrafish recordings, widefield calcium imaging data, population recordings from rat hippocampus and artificial neural networks. Finally, we illustrate high-dimensional scenarios where Rastermap and similar algorithms cannot be used effectively.\nPatient iPSC models reveal glia-intrinsic phenotypes in multiple sclerosis\nAuthors: Clayton, B. L. L.; Barbar, L.; Sapar, M.; Rusielewicz, T.; Kalpana, K.; Migliori, B.; The NYSCF Global Stem Cell Array, ; Paull, D.; Brenner, K.; Moroziewicz, D.; Sand, I. K.; Casaccia, P.; Tesar, P. J.; Fossati, V.\nScore: 19.0, Published: 2023-08-02 DOI: 10.1101/2023.08.01.551553\nMultiple sclerosis (MS) is considered an inflammatory and neurodegenerative disease of the central nervous system, typically resulting in significant neurological disability that worsens over time. While considerable progress has been made in defining the immune systems role in MS pathophysiology, the contribution of intrinsic CNS-cell dysfunction remains unclear. Here, we generated the largest reported collection of iPSC lines from people with MS spanning diverse clinical subtypes and differentiated them into glia-enriched cultures. Using single-cell transcriptomic profiling, we observed several distinguishing characteristics of MS cultures pointing to glia-intrinsic disease mechanisms. We found that iPSC-derived cultures from people with primary progressive MS contained fewer oligodendrocytes. Moreover, iPSC-oligodendrocyte lineage cells and astrocytes from people with MS showed increased expression of immune and inflammatory genes that match those of glial cells from MS postmortem brains. Thus, iPSC-derived MS models provide a unique platform for dissecting glial contributions to disease phenotypes independent of the peripheral immune system and identify potential glia-specific targets for therapeutic intervention.\nNetwork Statistics of the Whole-Brain Connectome of Drosophila\nAuthors: Lin, A.; Yang, R.; Dorkenwald, S.; Matsliah, A.; Sterling, A. R.; Schlegel, P.; Yu, S.-c.; McKellar, C. E.; Costa, M.; Eichler, K.; Bates, A. S.; Eckstein, N.; Funke, J.; Jefferis, G. S. X. E.; Murthy, M.\nScore: 18.4, Published: 2023-07-29 DOI: 10.1101/2023.07.29.551086\nAnimal brains are complex organs composed of thousands of interconnected neurons. Characterizing the network properties of these brains is a requisite step towards understanding mechanisms of computation and information flow. With the completion of the Flywire project, we now have access to the connectome of a complete adult Drosophila brain, containing 130,000 neurons and millions of connections. Here, we present a statistical summary and data products of the Flywire connectome, delving into its network properties and topological features. To gain insights into local connectivity, we computed the prevalence of two- and three-node network motifs, examined their strengths and neurotransmitter compositions, and compared these topological metrics with wiring diagrams of other animals. We uncovered a population of highly connected neurons known as the \"rich club\" and identified subsets of neurons that may serve as integrators or broadcasters of signals. Finally, we examined subnetworks based on 78 anatomically defined brain regions. The freely available data and neuron populations presented here will serve as a foundation for models and experiments exploring the relationship between neural activity and anatomical structure.\nGrid-like entorhinal representation of an abstract value space during prospective decision making\nAuthors: Nitsch, A.; Garvert, M. M.; Bellmund, J. L. S.; Schuck, N. W.; Doeller, C. F.\nScore: 16.6, Published: 2023-08-03 DOI: 10.1101/2023.08.02.548378\nEveryday decisions require us to predict how valuable different choice options will be in the future. Prior studies have identified a cognitive map in the hippocampal-entorhinal system that encodes relationships between states and enables prediction of future states, but does not inherently convey value during prospective decision making. Here, we investigated whether the entorhinal cortex integrates relational information about changing values by representing an abstract value space. To this end, we combined fMRI with a prospective decision making task that required participants to track and predict changing values of two choice options in a sequence. Such a sequence formed a trajectory through an underlying two-dimensional value space. Our results show that participants successfully integrated and extrapolated changes along the two value dimensions. Participants' choice behavior was explained by a prospective reinforcement learning model and the degree to which they updated values over time correlated with self-reported navigational abilities and preferences. Crucially, while participants traversed the abstract value space, the entorhinal cortex exhibited a grid-like representation, with the phase of the hexadirectional fMRI signal (i.e., the orientation of the estimated grid) being aligned to the most informative axis through the value space. A network of brain regions, including the ventromedial prefrontal cortex (vmPFC), tracked the prospective value difference between options and the occipital-temporal cortex represented the more valuable option. These findings suggest that the entorhinal grid system might support the prediction of future values by representing a cognitive map, which might be used to generate lower-dimensional signals of the value difference between options and their identities for choices. Thus, these findings provide novel insight for our understanding of cognitive maps as a mechanism to guide prospective decision making in humans.\n",
  "wordCount" : "2472",
  "inLanguage": "en",
  "datePublished": "2023-08-06T10:36:48Z",
  "dateModified": "2023-08-06T10:36:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/neuroscience/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      neuroscience
    </h1>
    <div class="post-meta"><span>updated on August 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550667">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550667" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550667">
        <p class="paperTitle">Constructed languages are processed by the same brain mechanisms as natural languages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550667" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550667" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Malik-Moraleda, S.; Taliaferro, M.; Shannon, S.; Jhingan, N.; Swords, S.; Peterson, D. J.; Frommer, P.; Okrand, M.; Sams, J.; Cardwell, R.; Freeman, C.; Fedorenko, E.</p>
        <p class="info">Score: 160.1, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550667' target='https://doi.org/10.1101/2023.07.28.550667'> 10.1101/2023.07.28.550667</a></p>
        <p class="abstract">What constitutes a language? Natural languages share some features with other domains: from math, to music, to gesture. However, the brain mechanisms that process linguistic input are highly specialized, showing little or no response to diverse non-linguistic tasks. Here, we examine constructed languages (conlangs) to ask whether they draw on the same neural mechanisms as natural languages, or whether they instead pattern with domains like math and logic. Using individual-subject fMRI analyses, we show that understanding conlangs recruits the same brain areas as natural language comprehension. This result holds for Esperanto (n=19 speakers)-- created to resemble natural languages--and fictional conlangs (Klingon (n=10), Navi (n=9), High Valyrian (n=3), and Dothraki (n=3)), created to differ from natural languages, and suggests that conlangs and natural languages share critical features and that the notable differences between conlangs and natural language are not consequential for the cognitive and neural mechanisms that they engage.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551097">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551097" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551097">
        <p class="paperTitle">Divergent patterns of healthy aging across human brain regions at single-cell resolution reveal links to neurodegenerative disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551097" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551097" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duffy, M.; Ding, J.; Langston, R. G.; Shah, S. I.; Nalls, M. A.; Scholz, S. W.; Whitaker, D. T.; Auluck, P. K.; Gibbs, J. R.; Marenco, S.; Cookson, M. R.</p>
        <p class="info">Score: 32.1, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551097' target='https://doi.org/10.1101/2023.07.31.551097'> 10.1101/2023.07.31.551097</a></p>
        <p class="abstract">Age is a major common risk factor underlying neurodegenerative diseases, including Alzheimers disease, Parkinsons disease, and amyotrophic lateral sclerosis. Previous studies reported that chronological age correlates with differential gene expression across different brain regions. However, prior datasets have not disambiguated whether expression associations with age are due to changes in cell numbers and/or gene expression per cell. In this study, we leveraged single nucleus RNA-sequencing (snRNAseq) to examine changes in cell proportions and transcriptomes in four different brain regions, each from 12 donors aged 20-30 years (young) or 60-85 years (old). We sampled 155,192 nuclei from two cortical regions (entorhinal cortex and middle temporal gyrus) and two subcortical regions (putamen and subventricular zone) relevant to neurodegenerative diseases or the proliferative niche. We found no changes in cellular composition of different brain regions with healthy aging. Surprisingly, we did find that each brain region has a distinct aging signature, with only minor overlap in differentially associated genes across regions. Moreover, each cell type shows distinct age-associated expression changes, including loss of protein synthesis genes in cortical inhibitory neurons, axonogenesis genes in excitatory neurons and oligodendrocyte precursor cells, enhanced gliosis markers in astrocytes and disease-associated markers in microglia, and genes critical for neuron-glia communication. Importantly, we find cell type-specific enrichments of age associations with genes nominated by Alzheimers disease and Parkinsons disease genome-wide association studies (GWAS), such as apolipoprotein E (APOE), and leucine-rich repeat kinase 2 (LRRK2) in microglia that are independent of overall expression levels across cell types. We present this data as a new resource which highlights, first, region- and cell type-specific transcriptomic changes in healthy aging that may contribute to selective vulnerability and, second, provide context for testing GWAS-nominated disease risk genes in relevant subtypes and developing more targeted therapeutic strategies. The data is readily accessible without requirement for extensive computational support in a public website, https://brainexp-hykyffa56a-uc.a.run.app/

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=177 SRC=&#34;FIGDIR/small/551097v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (66K):
org.highwire.dtl.DTLVardef@d8e616org.highwire.dtl.DTLVardef@1e9e734org.highwire.dtl.DTLVardef@14a3924org.highwire.dtl.DTLVardef@1ed8cff_HPS_FORMAT_FIGEXP  M_FIG *Created using Biorender.com

C_FIG HighlightsO_LIEstablishment of a single nuclei atlas of human aging in four brain regions
C_LIO_LIEach region and cell type exhibits a unique aging-associated transcriptome signature
C_LIO_LIGene expression changes occur in absence of overt cell loss and are categorically unique across cell types
C_LIO_LINeurological disease-associated genes have age-associated expression patterns in specific cell types in the context of healthy aging
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551520">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551520" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551520">
        <p class="paperTitle">Alpha oscillations support the efficiency of guided visual search by inhibiting both target and distractor features in early visual cortex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551520" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551520" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duecker, K.; Shapiro, K.; Hanslmayr, S.; Wolfe, J.; Pan, Y.; Jensen, O.</p>
        <p class="info">Score: 30.3, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551520' target='https://doi.org/10.1101/2023.08.03.551520'> 10.1101/2023.08.03.551520</a></p>
        <p class="abstract">Visual search models have long emphasised that task-relevant items must be prioritized for optimal performance. While it is known that search efficiency also benefits from active distractor inhibition, the underlying neuronal mechanisms are debated. Here, we used MEG in combination with Rapid Invisible Frequency Tagging (RIFT) to understand the neural correlates of feature-guided visual search. RIFT served as a continuous read-out of the neuronal excitability to the search stimuli and revealed evidence for target boosting and distractor suppression in early visual cortex. These findings were complemented by an increase in occipital alpha power predicting faster responses and higher hit rates, as well as reduced RIFT responses to all stimuli, regardless of their task-relevance. We propose that alpha oscillations in early visual regions implement a blanket inhibition that reduces neuronal excitability to both target and distractor features. As the excitability of neurons encoding the target features is boosted, these neurons overcome the inhibition, facilitating guidance towards task-relevant stimuli. Our results provide novel insights on a mechanism in early visual regions that supports selective attention through inhibition.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.29.551087">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.29.551087" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.29.551087">
        <p class="paperTitle">Daytime experiences shape neural activity and dream content in the sleeping brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.29.551087" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.29.551087" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kumral, D.; Palmieri, J.; Gais, S.; Schoenauer, M.</p>
        <p class="info">Score: 32.9, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.29.551087' target='https://doi.org/10.1101/2023.07.29.551087'> 10.1101/2023.07.29.551087</a></p>
        <p class="abstract">Learning-related brain activity patterns are replayed during sleep, and memories of recent experiences appear in our dreams. The connection between these phenomena, however, remains unclear. We investigated whether memory reinstatement during sleep contributes to dreaming. Participants listened to audiobooks before falling asleep. We could determine which audiobook they had studied based on dream reports collected during the night. Audiobook content was also reinstated at the neural level, in high-density EEG recordings. Brain activity during rapid eye movement sleep, particularly in the high-frequency beta range, carried information about the audiobook and simultaneously benefitted memory retention. Crucially, when the learning condition was manifest in neural activity, it also emerged in dreams. Reprocessing of daytime experiences during sleep thus shapes our brain activity, our dreams, and our memories.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.550767">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.550767" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.550767">
        <p class="paperTitle">The Human Microglia Atlas (HuMicA) Unravels Changes in Homeostatic and Disease-Associated Microglia Subsets across Neurodegenerative Conditions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.550767" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.550767" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martins-Ferreira, R.; Calafell-Segura, J.; Leal, B.; Rodriguez-Ubreva, J.; Mereu, E.; Pinho e Costa, P.; Ballestar, E.</p>
        <p class="info">Score: 26.8, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.550767' target='https://doi.org/10.1101/2023.08.01.550767'> 10.1101/2023.08.01.550767</a></p>
        <p class="abstract">Dysregulated microglia activation, leading to neuroinflammation, is crucial in neurodegenerative disease development and progression. The initial M1/M2 dual activation classification for microglia is outdated. Even the  disease-associated microglia (DAM) phenotype, firstly described in mice, falls short in representing the diverse microglia phenotypes in pathology. In this study, we have constructed a transcriptomic atlas of human brain immune cells by integrating single-nucleus (sn)RNA-seq datasets from multiple neurodegenerative conditions. Sixteen datasets were included, comprising 295 samples from patients with Alzheimers disease, autism spectrum disorder, epilepsy, multiple sclerosis, Lewy body diseases, COVID-19, and healthy controls. The integrated Human Microglia Atlas (HuMicA) dataset included 60,557 nuclei and revealed 11 microglial subpopulations distributed across all pathological and healthy conditions. Among these, we identified four different homeostatic clusters as well as pathological phenotypes. These included two stages of early and late activation of the DAM phenotype and the disease-inflammatory macrophage (DIM) phenotype, which was recently described in mice, and is also present in human microglia, as indicated by our analysis. The high versatility of microglia is evident through changes in subset distribution across various pathologies, suggesting their contribution in shaping pathological phenotypes. Our analysis showed overall depletion of four substates of homeostatic microglia, and expansion of niche subpopulations within the DAM and DIM spectrum across distinct neurodegenerative pathologies. The HuMicA is invaluable in advancing the study of microglia biology in both healthy and disease settings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551579">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551579" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551579">
        <p class="paperTitle">Distinct roles of putative excitatory and inhibitory neurons in the macaque inferior temporal cortex in core object recognition behavior</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551579" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551579" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanghavi, S.; Kar, K.</p>
        <p class="info">Score: 22.4, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551579' target='https://doi.org/10.1101/2023.08.01.551579'> 10.1101/2023.08.01.551579</a></p>
        <p class="abstract">A spatially distributed population of neurons in the macaque inferior temporal (IT) cortex supports object recognition behavior, but the cell-type specificity of the population in forming &#34;behaviorally sufficient&#34; object decodes remain unclear. To address this, we recorded neural signals from the macaque IT cortex and compared the object identity information and the alignment of decoding strategies derived from putative inhibitory (Inh) and excitatory (Exc) neurons to the monkeys behavior. We observed that while Inh neurons represented significant category information, decoding strategies based on Exc neural population activity outperformed those from Inh neurons in overall accuracy and their image-level match to the monkeys behavioral reports. Interestingly, both Exc and Inh responses explained a fraction of unique variance of the monkeys behavior, demonstrating a distinct role of the two cell types in generating object identity solutions for a downstream readout. We observed that current artificial neural network (ANN) models of primate ventral stream, designed with AI goals of performance optimization on image categorization, better predict Exc neurons (and its contribution to object recognition behavior) than Inh neurons. Beyond, the refinement of linking propositions between IT and object recognition behavior, our results guide the development of more biologically constrained brain models by offering novel cell-type specific neural benchmarks.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550571">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550571" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550571">
        <p class="paperTitle">Rastermap: a discovery method for neural population recordings</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550571" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550571" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stringer, C.; Zhong, L.; Syeda, A.; Du, F.; Pachitariu, M.</p>
        <p class="info">Score: 46.8, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550571' target='https://doi.org/10.1101/2023.07.25.550571'> 10.1101/2023.07.25.550571</a></p>
        <p class="abstract">Neurophysiology has long progressed through exploratory experiments and chance discoveries. Anecdotes abound of researchers setting up experiments while listening to spikes in real time and observing a pattern of consistent firing when certain stimuli or behaviors happened. With the advent of large-scale recordings, such close observation of data has become harder because high-dimensional spaces are impenetrable to our pattern-finding intuitions. To help ourselves find patterns in neural data, our lab has been openly developing a visualization framework known as &#34;Rastermap&#34; over the past five years. Rastermap takes advantage of a new global optimization algorithm for sorting neural responses along a one-dimensional manifold. Displayed as a raster plot, the sorted neurons show a variety of activity patterns, which can be more easily identified and interpreted. We first benchmark Rastermap on realistic simulations with multiplexed cognitive variables. Then we demonstrate it on recordings of tens of thousands of neurons from mouse visual and sensorimotor cortex during spontaneous, stimulus-evoked and task-evoked epochs, as well as on whole-brain zebrafish recordings, widefield calcium imaging data, population recordings from rat hippocampus and artificial neural networks. Finally, we illustrate high-dimensional scenarios where Rastermap and similar algorithms cannot be used effectively.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551553">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551553" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551553">
        <p class="paperTitle">Patient iPSC models reveal glia-intrinsic phenotypes in multiple sclerosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551553" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551553" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Clayton, B. L. L.; Barbar, L.; Sapar, M.; Rusielewicz, T.; Kalpana, K.; Migliori, B.; The NYSCF Global Stem Cell Array,  ; Paull, D.; Brenner, K.; Moroziewicz, D.; Sand, I. K.; Casaccia, P.; Tesar, P. J.; Fossati, V.</p>
        <p class="info">Score: 19.0, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551553' target='https://doi.org/10.1101/2023.08.01.551553'> 10.1101/2023.08.01.551553</a></p>
        <p class="abstract">Multiple sclerosis (MS) is considered an inflammatory and neurodegenerative disease of the central nervous system, typically resulting in significant neurological disability that worsens over time. While considerable progress has been made in defining the immune systems role in MS pathophysiology, the contribution of intrinsic CNS-cell dysfunction remains unclear. Here, we generated the largest reported collection of iPSC lines from people with MS spanning diverse clinical subtypes and differentiated them into glia-enriched cultures. Using single-cell transcriptomic profiling, we observed several distinguishing characteristics of MS cultures pointing to glia-intrinsic disease mechanisms. We found that iPSC-derived cultures from people with primary progressive MS contained fewer oligodendrocytes. Moreover, iPSC-oligodendrocyte lineage cells and astrocytes from people with MS showed increased expression of immune and inflammatory genes that match those of glial cells from MS postmortem brains. Thus, iPSC-derived MS models provide a unique platform for dissecting glial contributions to disease phenotypes independent of the peripheral immune system and identify potential glia-specific targets for therapeutic intervention.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.29.551086">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.29.551086" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.29.551086">
        <p class="paperTitle">Network Statistics of the Whole-Brain Connectome of Drosophila</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.29.551086" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.29.551086" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lin, A.; Yang, R.; Dorkenwald, S.; Matsliah, A.; Sterling, A. R.; Schlegel, P.; Yu, S.-c.; McKellar, C. E.; Costa, M.; Eichler, K.; Bates, A. S.; Eckstein, N.; Funke, J.; Jefferis, G. S. X. E.; Murthy, M.</p>
        <p class="info">Score: 18.4, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.29.551086' target='https://doi.org/10.1101/2023.07.29.551086'> 10.1101/2023.07.29.551086</a></p>
        <p class="abstract">Animal brains are complex organs composed of thousands of interconnected neurons. Characterizing the network properties of these brains is a requisite step towards understanding mechanisms of computation and information flow. With the completion of the Flywire project, we now have access to the connectome of a complete adult Drosophila brain, containing 130,000 neurons and millions of connections. Here, we present a statistical summary and data products of the Flywire connectome, delving into its network properties and topological features. To gain insights into local connectivity, we computed the prevalence of two- and three-node network motifs, examined their strengths and neurotransmitter compositions, and compared these topological metrics with wiring diagrams of other animals. We uncovered a population of highly connected neurons known as the &#34;rich club&#34; and identified subsets of neurons that may serve as integrators or broadcasters of signals. Finally, we examined subnetworks based on 78 anatomically defined brain regions. The freely available data and neuron populations presented here will serve as a foundation for models and experiments exploring the relationship between neural activity and anatomical structure.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.548378">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.548378" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.548378">
        <p class="paperTitle">Grid-like entorhinal representation of an abstract value space during prospective decision making</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.548378" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.548378" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nitsch, A.; Garvert, M. M.; Bellmund, J. L. S.; Schuck, N. W.; Doeller, C. F.</p>
        <p class="info">Score: 16.6, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.548378' target='https://doi.org/10.1101/2023.08.02.548378'> 10.1101/2023.08.02.548378</a></p>
        <p class="abstract">Everyday decisions require us to predict how valuable different choice options will be in the future. Prior studies have identified a cognitive map in the hippocampal-entorhinal system that encodes relationships between states and enables prediction of future states, but does not inherently convey value during prospective decision making. Here, we investigated whether the entorhinal cortex integrates relational information about changing values by representing an abstract value space. To this end, we combined fMRI with a prospective decision making task that required participants to track and predict changing values of two choice options in a sequence. Such a sequence formed a trajectory through an underlying two-dimensional value space. Our results show that participants successfully integrated and extrapolated changes along the two value dimensions. Participants&#39; choice behavior was explained by a prospective reinforcement learning model and the degree to which they updated values over time correlated with self-reported navigational abilities and preferences. Crucially, while participants traversed the abstract value space, the entorhinal cortex exhibited a grid-like representation, with the phase of the hexadirectional fMRI signal (i.e., the orientation of the estimated grid) being aligned to the most informative axis through the value space. A network of brain regions, including the ventromedial prefrontal cortex (vmPFC), tracked the prospective value difference between options and the occipital-temporal cortex represented the more valuable option. These findings suggest that the entorhinal grid system might support the prediction of future values by representing a cognitive map, which might be used to generate lower-dimensional signals of the value difference between options and their identities for choices. Thus, these findings provide novel insight for our understanding of cognitive maps as a mechanism to guide prospective decision making in humans.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
