<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>neuroscience</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>neuroscience | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Sustaining wakefulness: Brainstem connectivity in human consciousness
Authors: Edlow, B. L.; Olchanyi, M.; Freeman, H. J.; Li, J.; Maffei, C.; Snider, S. B.; Zollei, L.; Iglesias, J. E.; Augustinack, J.; Bodien, Y. G.; Haynes, R. L.; Greve, D. N.; Diamond, B. R.; Stevens, A.; Giacino, J. T.; Destrieux, C.; van der Kouwe, A.; Brown, E. N.; Folkerth, R. D.; Fischl, B.; Kinney, H. C.
Score: 96.0, Published: 2023-07-15 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/neuroscience/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="neuroscience" />
<meta property="og:description" content="Sustaining wakefulness: Brainstem connectivity in human consciousness
Authors: Edlow, B. L.; Olchanyi, M.; Freeman, H. J.; Li, J.; Maffei, C.; Snider, S. B.; Zollei, L.; Iglesias, J. E.; Augustinack, J.; Bodien, Y. G.; Haynes, R. L.; Greve, D. N.; Diamond, B. R.; Stevens, A.; Giacino, J. T.; Destrieux, C.; van der Kouwe, A.; Brown, E. N.; Folkerth, R. D.; Fischl, B.; Kinney, H. C.
Score: 96.0, Published: 2023-07-15 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/neuroscience/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-26T10:37:30+00:00" />
<meta property="article:modified_time" content="2023-07-26T10:37:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="neuroscience"/>
<meta name="twitter:description" content="Sustaining wakefulness: Brainstem connectivity in human consciousness
Authors: Edlow, B. L.; Olchanyi, M.; Freeman, H. J.; Li, J.; Maffei, C.; Snider, S. B.; Zollei, L.; Iglesias, J. E.; Augustinack, J.; Bodien, Y. G.; Haynes, R. L.; Greve, D. N.; Diamond, B. R.; Stevens, A.; Giacino, J. T.; Destrieux, C.; van der Kouwe, A.; Brown, E. N.; Folkerth, R. D.; Fischl, B.; Kinney, H. C.
Score: 96.0, Published: 2023-07-15 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "neuroscience",
      "item": "https://trxiv.yorks0n.com/posts/neuroscience/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "neuroscience",
  "name": "neuroscience",
  "description": "Sustaining wakefulness: Brainstem connectivity in human consciousness\nAuthors: Edlow, B. L.; Olchanyi, M.; Freeman, H. J.; Li, J.; Maffei, C.; Snider, S. B.; Zollei, L.; Iglesias, J. E.; Augustinack, J.; Bodien, Y. G.; Haynes, R. L.; Greve, D. N.; Diamond, B. R.; Stevens, A.; Giacino, J. T.; Destrieux, C.; van der Kouwe, A.; Brown, E. N.; Folkerth, R. D.; Fischl, B.; Kinney, H. C.\nScore: 96.0, Published: 2023-07-15 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " Sustaining wakefulness: Brainstem connectivity in human consciousness\nAuthors: Edlow, B. L.; Olchanyi, M.; Freeman, H. J.; Li, J.; Maffei, C.; Snider, S. B.; Zollei, L.; Iglesias, J. E.; Augustinack, J.; Bodien, Y. G.; Haynes, R. L.; Greve, D. N.; Diamond, B. R.; Stevens, A.; Giacino, J. T.; Destrieux, C.; van der Kouwe, A.; Brown, E. N.; Folkerth, R. D.; Fischl, B.; Kinney, H. C.\nScore: 96.0, Published: 2023-07-15 DOI: 10.1101/2023.07.13.548265\nConsciousness is comprised of arousal (i.e., wakefulness) and awareness. Substantial progress has been made in mapping the cortical networks that modulate awareness in the human brain, but knowledge about the subcortical networks that sustain arousal is lacking. We integrated data from ex vivo diffusion MRI, immunohistochemistry, and in vivo 7 Tesla functional MRI to map the connectivity of a subcortical arousal network that we postulate sustains wakefulness in the resting, conscious human brain, analogous to the cortical default mode network (DMN) that is believed to sustain self-awareness. We identified nodes of the proposed default ascending arousal network (dAAN) in the brainstem, hypothalamus, thalamus, and basal forebrain by correlating ex vivo diffusion MRI with immunohistochemistry in three human brain specimens from neurologically normal individuals scanned at 600-750 {micro}m resolution. We performed deterministic and probabilistic tractography analyses of the diffusion MRI data to map dAAN intra-network connections and dAAN-DMN internetwork connections. Using a newly developed network-based autopsy of the human brain that integrates ex vivo MRI and histopathology, we identified projection, association, and commissural pathways linking dAAN nodes with one another and with cortical DMN nodes, providing a structural architecture for the integration of arousal and awareness in human consciousness. We release the ex vivo diffusion MRI data, corresponding immunohistochemistry data, network-based autopsy methods, and a new brainstem dAAN atlas to support efforts to map the connectivity of human consciousness. One sentence summaryWe performed ex vivo diffusion MRI, immunohistochemistry, and in vivo 7 Tesla functional MRI to map brainstem connections that sustain wakefulness in human consciousness.\nA modular chemigenetic calcium indicator enables in vivo functional imaging with near-infrared light\nAuthors: Farrants, H.; Shuai, Y.; Lemon, W. C.; Hernandez, C. M.; Yang, S.; Patel, R.; Qiao, G.; Frei, M. S.; Grimm, J. B.; Hanson, T. L.; Tomaska, F.; Turner, G. C.; Stringer, C.; Keller, P. J.; Beyene, A. G.; Chen, Y.; Liang, Y.; Lavis, L. D.; Schreiter, E. R.\nScore: 51.5, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549527\nGenetically encoded fluorescent calcium indicators have revolutionized neuroscience and other biological fields by allowing cellular-resolution recording of physiology during behavior. However, we currently lack bright, genetically targetable indicators in the near infrared that can be used in animals. Here, we describe WHaloCaMP, a modular chemigenetic calcium indicator built from bright dye-ligands and protein sensor domains that can be genetically targeted to specific cell populations. Fluorescence change in WHaloCaMP results from reversible quenching of the bound dye via a strategically placed tryptophan. WHaloCaMP is compatible with rhodamine dye-ligands that fluoresce from green to near-infrared, including several dye-ligands that efficiently label the central nervous system in animals. When bound to a near-infrared dye-ligand, WHaloCaMP1a is more than twice as bright as jGCaMP8s, and shows a 7x increase in fluorescence intensity and a 2.1 ns increase in fluorescence lifetime upon calcium binding. We use WHaloCaMP1a with near-infrared fluorescence emission to image Ca2+ responses in flies and mice, to perform three-color multiplexed functional imaging of hundreds of neurons and astrocytes in zebrafish larvae, and to quantitate calcium concentration using fluorescence lifetime imaging microscopy (FLIM).\nA neuronal substrate for translating nutrient state and resource density estimations into foraging decisions\nAuthors: Goldschmidt, D.; Tastekin, I.; Munch, D.; Park, J.-Y.; Haberkern, H.; Serra, L.; Baltazar, C.; Jayaraman, V.; Rubin, G. M.; Ribeiro, C.\nScore: 49.0, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549514\nForaging animals must balance the costs of exploring their surroundings with the potential benefits of finding nutritional resources. Each time an animal encounters a food source it must decide whether to initiate feeding or continue searching for potentially better options. Experimental evidence and patch foraging models predict that this decision depends on both nutritional state and the density of available resources in the environment. How the brain integrates such internal and external states to adapt the so-called exploration-exploitation trade-off remains poorly understood. We use video-based tracking to show that Drosophila regulates the decision to engage with food patches based on nutritional state and travel time between food patches, the latter being a measure of food patch density in the environment. To uncover the neuronal basis of this decision process, we performed a neurogenetic silencing screen of more than 400 genetic driver lines with sparse expression patterns in the fly brain. We identified a population of neurons in the central complex that acts as a key regulator of the decision to engage with a food patch. We show that manipulating the activity of these neurons alters the probability to engage, that their activity is modulated by the protein state of the animal, and that silencing these neurons perturbs the ability of the animal to adjust foraging decisions to the flys travel time between food patches. Taken together, our results reveal a neuronal substrate that integrates nutritional state and patch density information to control a specific foraging decision, and therefore provide an important step towards a mechanistic explanation of the cognitive computations that resolve complex cost-benefit trade-offs.\nThree-dimensional single-cell transcriptome imaging of thick tissues\nAuthors: Fang, R.; Halpern, A. R.; Rahman, M. M.; Huang, Z.; Lei, Z.; Hell, S. J.; Dulac, C.; Zhuang, X.\nScore: 46.3, Published: 2023-07-25 DOI: 10.1101/2023.07.21.550124\nMultiplexed error-robust fluorescence in-situ hybridization (MERFISH) allows genome-scale imaging of RNAs in individual cells in intact tissues. To date, MERFISH has been applied to image thin tissue samples of ~10-micron thickness. Here, we present a method to enable three-dimensional (3D) single-cell transcriptome imaging of thick tissue specimens by integrating MERFISH with confocal microscopy for optical sectioning and deep learning for increasing imaging speed and quality. We demonstrated 3D MERFISH on mouse brain tissue sections of up to 200-micron thickness with high detection efficiency and accuracy. We anticipate that 3D thick-tissue MERFISH imaging will broaden the scope of questions that can be addressed by spatial genomics.\nEvolutionarily conserved fMRI network dynamics in the mouse, macaque, and human brain\nAuthors: Gutierrez-Barragan, D.; Ramirez, J. S. B.; Panzeri, S.; Xu, T.; Gozzi, A.\nScore: 45.6, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549681\nEvolutionarily relevant networks have been previously described in several mammalian species using time-averaged analyses of fMRI time-series. However, fMRI network activity is highly dynamic and continually evolves over timescales of seconds. Whether the dynamic principles that govern intrinsic fMRI network fluctuations are conserved across mammalian species remains unclear. Using frame-wise clustering of fMRI time-series, we find that fMRI network dynamics in awake macaques and humans is characterized by recurrent transitions between a set of 4 dominant, neuroanatomically homologous fMRI coactivation modes (C-modes), three of which are also plausibly represented in the rodent brain. Importantly, in all species the identified C-modes exhibit species-invariant dynamic features, including intrinsic infraslow dynamics and preferred occurrence at specific phases of global fMRI signal fluctuations. Moreover, C-modes occurrence rates in awake humans, macaques and mice reflect temporal trajectories of least energy and predicts ranking of corresponding functional connectivity gradients. Our results reveal a set of species-invariant principles underlying the dynamic organization of fMRI networks in mammalian species, and offer novel opportunities to relate fMRI network findings across the phylogenetic tree.\nCalcium-permeable AMPA receptors govern PV neuron feature selectivity\nAuthors: Hong, I.; Kim, J.; Hainmueller, T.; Kim, D. W.; Johnson, R. C.; Park, S. H.; Limjunyawong, N.; Yang, Z.; Cheon, D.; Hwang, T.; Agarwal, A.; Cholvin, T.; Krienen, F. M.; McCarroll, S. A.; Dong, X.; Leopold, D. A.; Blackshaw, S.; Bergles, D. E.; Bartos, M.; Brown, S. P.; Huganir, R. L.\nScore: 37.3, Published: 2023-07-20 DOI: 10.1101/2023.07.20.549908\nThe brain helps us survive by forming internal representations of the external world1,2. Excitatory cortical neurons are often precisely tuned to specific external stimuli3,4. However, inhibitory neurons, such as parvalbumin-positive (PV) interneurons, are generally less selective5. PV interneurons differ from excitatory cells in their neurotransmitter receptor subtypes, including AMPA receptors6,7. While excitatory neurons express calcium-impermeable AMPA receptors containing the GluA2 subunit, PV interneurons express receptors that lack the GluA2 subunit and are calcium-permeable (CP-AMPARs). Here we demonstrate a causal relationship between CP-AMPAR expression and the low feature selectivity of PV interneurons. We find a low expression stoichiometry of GluA2 mRNA relative to other subunits in PV interneurons which is conserved across ferrets, rodents, marmosets, and humans, causing abundant CP-AMPAR expression. Replacing CP-AMPARs in PV interneurons with calcium-impermeable AMPARs increased their orientation selectivity in the visual cortex. Sparse CP-AMPAR manipulations demonstrated that this increase was cell-autonomous and could occur well beyond development. Interestingly, excitatory-PV interneuron connectivity rates and unitary synaptic strength were unaltered by CP-AMPAR removal, suggesting that the selectivity of PV interneurons can be altered without drastically changing connectivity. In GluA2 knockout mice, where all AMPARs are calcium-permeable, excitatory neurons showed significantly reduced orientation selectivity, suggesting that CP-AMPARs are sufficient to drive lower selectivity regardless of cell type. Remarkably, hippocampal PV interneurons, which usually exhibit low spatial tuning, became more spatially selective after removing CP-AMPARs, indicating that CP-AMPARs suppress the feature selectivity of PV interneurons independent of modality. These results reveal a novel role of CP-AMPARs in maintaining a low-selectivity sensory representation in PV interneurons and suggest a conserved molecular mechanism that distinguishes the unique synaptic computations of inhibitory and excitatory neurons.\nMembrane mechanics dictate axonal morphology and function\nAuthors: Griswold, J. M.; Bonilla-Quintana, M.; Pepper, R.; Lee, C. T.; Raychaudhuri, S.; Ma, S.; Gan, Q.; Syed, S.; Zhu, C.; Bell, M.; Suga, M.; Yamaguchi, Y.; Chereau, R.; Nagerl, V.; Knott, G. W.; Rangamani, P.; Watanabe, S.\nScore: 28.3, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549958\nAxons are thought to be ultrathin membrane cables of a relatively uniform diameter, designed to conduct electrical signals, or action potentials. Here, we demonstrate that unmyelinated axons are not simple cylindrical tubes. Rather, axons have nanoscopic boutons repeatedly along their length interspersed with a thin cable with a diameter of [~]60 nm like pearls-on-a-string. These boutons are only [~]200 nm in diameter and do not have synaptic contacts or a cluster of synaptic vesicles, hence non-synaptic. Our in silico modeling suggests that axon pearling can be explained by the mechanical properties of the membrane including the bending modulus and tension. Consistent with modeling predictions, treatments that disrupt these parameters like hyper- or hypo-tonic solutions, cholesterol removal, and non-muscle myosin II inhibition all alter the degree of axon pearling, suggesting that axon morphology is indeed determined by the membrane mechanics. Intriguingly, neuronal activity modulates the cholesterol level of plasma membrane, leading to shrinkage of axon pearls. Consequently, the conduction velocity of action potentials becomes slower. These data reveal that biophysical forces dictate axon morphology and function and that modulation of membrane mechanics likely underlies plasticity of unmyelinated axons.\nFunctional Connectivity Development along the Sensorimotor-Association Axis Enhances the Cortical Hierarchy\nAuthors: Luo, A.; Sydnor, V. J.; Pines, A.; Larsen, B.; Alexander-Bloch, A. F.; Cieslak, M.; Covitz, S.; Chen, A.; Esper, N. B.; Feczko, E.; Franco, A. R.; Gur, R. E.; Gur, R. C.; Houghton, A.; Hu, F.; Keller, A. S.; Kiar, G.; Mehta, K.; Salum, G. A.; Tapera, T.; Xu, T.; Zhao, C.; Fair, D. A.; Salo, T.; Shinohara, R. T.; Milham, M. P.; Satterthwaite, T. D.\nScore: 28.2, Published: 2023-07-25 DOI: 10.1101/2023.07.20.549090\nHuman cortical maturation has been posited to be organized along the sensorimotor-association (S-A) axis, a hierarchical axis of brain organization that spans from unimodal sensorimotor cortices to transmodal association cortices. Here, we investigate the hypothesis that the development of functional connectivity during childhood through adolescence conforms to the cortical hierarchy defined by the S-A axis. We tested this pre-registered hypothesis in four large-scale, independent datasets (total n = 3,355; ages 5-23 years): the Philadelphia Neurodevelopmental Cohort (n = 1,207), Nathan Kline Institute-Rockland Sample (n = 397), Human Connectome Project: Development (n = 625), and Healthy Brain Network (n = 1,126). In each dataset, the development of functional connectivity systematically varied along the S-A axis. Connectivity in sensorimotor regions increased, whereas connectivity in association cortices declined, refining and reinforcing the cortical hierarchy. These robust and generalizable results establish that the sensorimotor-association axis of cortical organization encodes the dominant pattern of functional connectivity development.\nPolyGR and polyPR knock-in mice reveal a conserved neuroprotective extracellular matrix signature in C9orf72 ALS/FTD neurons\nAuthors: Milioto, C.; Carcole, M.; Giblin, A.; Coneys, R.; Attrebi, O.; Ahmed, M.; Harris, S. S.; Lee, B. I.; Yang, M.; Nirujogi, R. S.; Biggs, D.; Salomonsson, S.; Zanovello, M.; De Oliveira, P.; Katona, E.; Glaria, I.; Mikheenko, A.; Geary, B.; Udine, E.; Vaizoglu, D.; Rademakers, R.; van Blitterswijk, M.; Devoy, A.; Hong, S.; Partridge, L.; Fratta, P.; Alessi, D.; Davies, B.; Busche, M. A.; Greensmith, L.; Fisher, E. M.; Isaacs, A. M.\nScore: 28.0, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549331\nA GGGGCC repeat expansion in C9orf72 is the most common genetic cause of ALS and FTD (C9ALS/FTD). The presence of dipeptide repeat (DPR) proteins, generated by translation of the expanded repeat, is a major pathogenic feature of C9ALS/FTD pathology, but their most relevant effects in a physiological context are not known. Here, we generated C9orf72 DPR knock-in mouse models characterised by physiological expression of 400 codon-optimised polyGR or polyPR repeats, and heterozygous C9orf72 reduction. (GR)400 and (PR)400 knock-in mice exhibit cortical neuronal hyperexcitability, age-dependent spinal motor neuron loss and progressive motor dysfunction, showing that they recapitulate key features of C9FTD/ALS. Quantitative proteomics revealed an increase in extracellular matrix (ECM) proteins in (GR)400 and (PR)400 spinal cord, with the collagen COL6A1 the most increased protein. This signature of increased ECM proteins was also present in C9ALS patient iPSC-motor neurons indicating it is a conserved feature of C9ALS/FTD. TGF-{beta}1 was one of the top predicted regulators of this ECM signature and polyGR expression in human iPSC-neurons was sufficient to induce TGF-{beta}1 followed by COL6A1, indicating TGF-{beta}1 is one driver of the ECM signature. Knockdown of the TGF-{beta}1 or COL6A1 orthologue in Drosophila dramatically and specifically exacerbated neurodegeneration in polyGR flies, showing that TGF-{beta}1 and COL6A1 protect against polyGR toxicity. Altogether, our physiological C9orf72 DPR knock-in mice have revealed a neuroprotective and conserved ECM signature in C9FTD/ALS.\nScalable projected Light Sheet Microscopy for high-resolution imaging of living and cleared samples\nAuthors: Chen, Y.; Chauhan, S.; Gong, C.; Dayton, H.; Xu, C.; De La Cruz, E. D.; Datta, M. S.; Leong, K. W.; Dietrich, L. E. P.; Tomer, R.\nScore: 49.3, Published: 2023-07-18 DOI: 10.1101/2023.05.31.543173\nLight sheet fluorescence microscopy (LSFM) is a widely used imaging technique for living and large cleared samples. However, high-performance LSFM systems are often prohibitively expensive and not easily scalable for high-throughput applications. Here, we introduce a cost-effective, scalable, and versatile high-resolution imaging framework, called projected Light Sheet Microscopy (pLSM), which repurposes readily available off-the-shelf consumer-grade components and an over-the-network control architecture to achieve high-resolution imaging of living and cleared samples. We extensively characterize the pLSM framework and showcase its capabilities through high-resolution, multi-color imaging and quantitative analysis of mouse and post-mortem human brain samples cleared using various techniques. Moreover, we show the applicability of pLSM for high-throughput molecular phenotyping of human induced pluripotent cells (iPSC)-derived brain and vessel organoids. Additionally, we utilized pLSM for comprehensive live imaging of bacterial pellicle biofilms at the air-liquid interface, uncovering their intricate layered architecture and diverse cellular dynamics across different depths. Overall, the pLSM framework has the potential to further democratize LSFM by making high-resolution light sheet microscopy more accessible and scalable.\n",
  "wordCount" : "2568",
  "inLanguage": "en",
  "datePublished": "2023-07-26T10:37:30Z",
  "dateModified": "2023-07-26T10:37:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/neuroscience/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      neuroscience
    </h1>
    <div class="post-meta"><span>updated on July 26, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548265">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548265" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548265">
        <p class="paperTitle">Sustaining wakefulness: Brainstem connectivity in human consciousness</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548265" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548265" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Edlow, B. L.; Olchanyi, M.; Freeman, H. J.; Li, J.; Maffei, C.; Snider, S. B.; Zollei, L.; Iglesias, J. E.; Augustinack, J.; Bodien, Y. G.; Haynes, R. L.; Greve, D. N.; Diamond, B. R.; Stevens, A.; Giacino, J. T.; Destrieux, C.; van der Kouwe, A.; Brown, E. N.; Folkerth, R. D.; Fischl, B.; Kinney, H. C.</p>
        <p class="info">Score: 96.0, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548265' target='https://doi.org/10.1101/2023.07.13.548265'> 10.1101/2023.07.13.548265</a></p>
        <p class="abstract">Consciousness is comprised of arousal (i.e., wakefulness) and awareness. Substantial progress has been made in mapping the cortical networks that modulate awareness in the human brain, but knowledge about the subcortical networks that sustain arousal is lacking. We integrated data from ex vivo diffusion MRI, immunohistochemistry, and in vivo 7 Tesla functional MRI to map the connectivity of a subcortical arousal network that we postulate sustains wakefulness in the resting, conscious human brain, analogous to the cortical default mode network (DMN) that is believed to sustain self-awareness. We identified nodes of the proposed default ascending arousal network (dAAN) in the brainstem, hypothalamus, thalamus, and basal forebrain by correlating ex vivo diffusion MRI with immunohistochemistry in three human brain specimens from neurologically normal individuals scanned at 600-750 {micro}m resolution. We performed deterministic and probabilistic tractography analyses of the diffusion MRI data to map dAAN intra-network connections and dAAN-DMN internetwork connections. Using a newly developed network-based autopsy of the human brain that integrates ex vivo MRI and histopathology, we identified projection, association, and commissural pathways linking dAAN nodes with one another and with cortical DMN nodes, providing a structural architecture for the integration of arousal and awareness in human consciousness. We release the ex vivo diffusion MRI data, corresponding immunohistochemistry data, network-based autopsy methods, and a new brainstem dAAN atlas to support efforts to map the connectivity of human consciousness.

One sentence summaryWe performed ex vivo diffusion MRI, immunohistochemistry, and in vivo 7 Tesla functional MRI to map brainstem connections that sustain wakefulness in human consciousness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549527">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549527" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549527">
        <p class="paperTitle">A modular chemigenetic calcium indicator enables in vivo functional imaging with near-infrared light</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549527" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549527" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Farrants, H.; Shuai, Y.; Lemon, W. C.; Hernandez, C. M.; Yang, S.; Patel, R.; Qiao, G.; Frei, M. S.; Grimm, J. B.; Hanson, T. L.; Tomaska, F.; Turner, G. C.; Stringer, C.; Keller, P. J.; Beyene, A. G.; Chen, Y.; Liang, Y.; Lavis, L. D.; Schreiter, E. R.</p>
        <p class="info">Score: 51.5, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549527' target='https://doi.org/10.1101/2023.07.18.549527'> 10.1101/2023.07.18.549527</a></p>
        <p class="abstract">Genetically encoded fluorescent calcium indicators have revolutionized neuroscience and other biological fields by allowing cellular-resolution recording of physiology during behavior. However, we currently lack bright, genetically targetable indicators in the near infrared that can be used in animals. Here, we describe WHaloCaMP, a modular chemigenetic calcium indicator built from bright dye-ligands and protein sensor domains that can be genetically targeted to specific cell populations. Fluorescence change in WHaloCaMP results from reversible quenching of the bound dye via a strategically placed tryptophan. WHaloCaMP is compatible with rhodamine dye-ligands that fluoresce from green to near-infrared, including several dye-ligands that efficiently label the central nervous system in animals. When bound to a near-infrared dye-ligand, WHaloCaMP1a is more than twice as bright as jGCaMP8s, and shows a 7x increase in fluorescence intensity and a 2.1 ns increase in fluorescence lifetime upon calcium binding. We use WHaloCaMP1a with near-infrared fluorescence emission to image Ca2&#43; responses in flies and mice, to perform three-color multiplexed functional imaging of hundreds of neurons and astrocytes in zebrafish larvae, and to quantitate calcium concentration using fluorescence lifetime imaging microscopy (FLIM).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549514">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549514" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549514">
        <p class="paperTitle">A neuronal substrate for translating nutrient state and resource density estimations into foraging decisions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549514" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549514" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goldschmidt, D.; Tastekin, I.; Munch, D.; Park, J.-Y.; Haberkern, H.; Serra, L.; Baltazar, C.; Jayaraman, V.; Rubin, G. M.; Ribeiro, C.</p>
        <p class="info">Score: 49.0, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549514' target='https://doi.org/10.1101/2023.07.19.549514'> 10.1101/2023.07.19.549514</a></p>
        <p class="abstract">Foraging animals must balance the costs of exploring their surroundings with the potential benefits of finding nutritional resources. Each time an animal encounters a food source it must decide whether to initiate feeding or continue searching for potentially better options. Experimental evidence and patch foraging models predict that this decision depends on both nutritional state and the density of available resources in the environment. How the brain integrates such internal and external states to adapt the so-called exploration-exploitation trade-off remains poorly understood. We use video-based tracking to show that Drosophila regulates the decision to engage with food patches based on nutritional state and travel time between food patches, the latter being a measure of food patch density in the environment. To uncover the neuronal basis of this decision process, we performed a neurogenetic silencing screen of more than 400 genetic driver lines with sparse expression patterns in the fly brain. We identified a population of neurons in the central complex that acts as a key regulator of the decision to engage with a food patch. We show that manipulating the activity of these neurons alters the probability to engage, that their activity is modulated by the protein state of the animal, and that silencing these neurons perturbs the ability of the animal to adjust foraging decisions to the flys travel time between food patches. Taken together, our results reveal a neuronal substrate that integrates nutritional state and patch density information to control a specific foraging decision, and therefore provide an important step towards a mechanistic explanation of the cognitive computations that resolve complex cost-benefit trade-offs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.550124">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.550124" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.550124">
        <p class="paperTitle">Three-dimensional single-cell transcriptome imaging of thick tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.550124" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.550124" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fang, R.; Halpern, A. R.; Rahman, M. M.; Huang, Z.; Lei, Z.; Hell, S. J.; Dulac, C.; Zhuang, X.</p>
        <p class="info">Score: 46.3, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.550124' target='https://doi.org/10.1101/2023.07.21.550124'> 10.1101/2023.07.21.550124</a></p>
        <p class="abstract">Multiplexed error-robust fluorescence in-situ hybridization (MERFISH) allows genome-scale imaging of RNAs in individual cells in intact tissues. To date, MERFISH has been applied to image thin tissue samples of ~10-micron thickness. Here, we present a method to enable three-dimensional (3D) single-cell transcriptome imaging of thick tissue specimens by integrating MERFISH with confocal microscopy for optical sectioning and deep learning for increasing imaging speed and quality. We demonstrated 3D MERFISH on mouse brain tissue sections of up to 200-micron thickness with high detection efficiency and accuracy. We anticipate that 3D thick-tissue MERFISH imaging will broaden the scope of questions that can be addressed by spatial genomics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549681">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549681" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549681">
        <p class="paperTitle">Evolutionarily conserved fMRI network dynamics in the mouse, macaque, and human brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549681" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549681" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gutierrez-Barragan, D.; Ramirez, J. S. B.; Panzeri, S.; Xu, T.; Gozzi, A.</p>
        <p class="info">Score: 45.6, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549681' target='https://doi.org/10.1101/2023.07.19.549681'> 10.1101/2023.07.19.549681</a></p>
        <p class="abstract">Evolutionarily relevant networks have been previously described in several mammalian species using time-averaged analyses of fMRI time-series. However, fMRI network activity is highly dynamic and continually evolves over timescales of seconds. Whether the dynamic principles that govern intrinsic fMRI network fluctuations are conserved across mammalian species remains unclear. Using frame-wise clustering of fMRI time-series, we find that fMRI network dynamics in awake macaques and humans is characterized by recurrent transitions between a set of 4 dominant, neuroanatomically homologous fMRI coactivation modes (C-modes), three of which are also plausibly represented in the rodent brain. Importantly, in all species the identified C-modes exhibit species-invariant dynamic features, including intrinsic infraslow dynamics and preferred occurrence at specific phases of global fMRI signal fluctuations. Moreover, C-modes occurrence rates in awake humans, macaques and mice reflect temporal trajectories of least energy and predicts ranking of corresponding functional connectivity gradients. Our results reveal a set of species-invariant principles underlying the dynamic organization of fMRI networks in mammalian species, and offer novel opportunities to relate fMRI network findings across the phylogenetic tree.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549908">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549908" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549908">
        <p class="paperTitle">Calcium-permeable AMPA receptors govern PV neuron feature selectivity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549908" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549908" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hong, I.; Kim, J.; Hainmueller, T.; Kim, D. W.; Johnson, R. C.; Park, S. H.; Limjunyawong, N.; Yang, Z.; Cheon, D.; Hwang, T.; Agarwal, A.; Cholvin, T.; Krienen, F. M.; McCarroll, S. A.; Dong, X.; Leopold, D. A.; Blackshaw, S.; Bergles, D. E.; Bartos, M.; Brown, S. P.; Huganir, R. L.</p>
        <p class="info">Score: 37.3, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549908' target='https://doi.org/10.1101/2023.07.20.549908'> 10.1101/2023.07.20.549908</a></p>
        <p class="abstract">The brain helps us survive by forming internal representations of the external world1,2. Excitatory cortical neurons are often precisely tuned to specific external stimuli3,4. However, inhibitory neurons, such as parvalbumin-positive (PV) interneurons, are generally less selective5. PV interneurons differ from excitatory cells in their neurotransmitter receptor subtypes, including AMPA receptors6,7. While excitatory neurons express calcium-impermeable AMPA receptors containing the GluA2 subunit, PV interneurons express receptors that lack the GluA2 subunit and are calcium-permeable (CP-AMPARs). Here we demonstrate a causal relationship between CP-AMPAR expression and the low feature selectivity of PV interneurons. We find a low expression stoichiometry of GluA2 mRNA relative to other subunits in PV interneurons which is conserved across ferrets, rodents, marmosets, and humans, causing abundant CP-AMPAR expression. Replacing CP-AMPARs in PV interneurons with calcium-impermeable AMPARs increased their orientation selectivity in the visual cortex. Sparse CP-AMPAR manipulations demonstrated that this increase was cell-autonomous and could occur well beyond development. Interestingly, excitatory-PV interneuron connectivity rates and unitary synaptic strength were unaltered by CP-AMPAR removal, suggesting that the selectivity of PV interneurons can be altered without drastically changing connectivity. In GluA2 knockout mice, where all AMPARs are calcium-permeable, excitatory neurons showed significantly reduced orientation selectivity, suggesting that CP-AMPARs are sufficient to drive lower selectivity regardless of cell type. Remarkably, hippocampal PV interneurons, which usually exhibit low spatial tuning, became more spatially selective after removing CP-AMPARs, indicating that CP-AMPARs suppress the feature selectivity of PV interneurons independent of modality. These results reveal a novel role of CP-AMPARs in maintaining a low-selectivity sensory representation in PV interneurons and suggest a conserved molecular mechanism that distinguishes the unique synaptic computations of inhibitory and excitatory neurons.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549958">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549958" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549958">
        <p class="paperTitle">Membrane mechanics dictate axonal morphology and function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549958" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549958" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Griswold, J. M.; Bonilla-Quintana, M.; Pepper, R.; Lee, C. T.; Raychaudhuri, S.; Ma, S.; Gan, Q.; Syed, S.; Zhu, C.; Bell, M.; Suga, M.; Yamaguchi, Y.; Chereau, R.; Nagerl, V.; Knott, G. W.; Rangamani, P.; Watanabe, S.</p>
        <p class="info">Score: 28.3, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549958' target='https://doi.org/10.1101/2023.07.20.549958'> 10.1101/2023.07.20.549958</a></p>
        <p class="abstract">Axons are thought to be ultrathin membrane cables of a relatively uniform diameter, designed to conduct electrical signals, or action potentials. Here, we demonstrate that unmyelinated axons are not simple cylindrical tubes. Rather, axons have nanoscopic boutons repeatedly along their length interspersed with a thin cable with a diameter of [~]60 nm like pearls-on-a-string. These boutons are only [~]200 nm in diameter and do not have synaptic contacts or a cluster of synaptic vesicles, hence non-synaptic. Our in silico modeling suggests that axon pearling can be explained by the mechanical properties of the membrane including the bending modulus and tension. Consistent with modeling predictions, treatments that disrupt these parameters like hyper- or hypo-tonic solutions, cholesterol removal, and non-muscle myosin II inhibition all alter the degree of axon pearling, suggesting that axon morphology is indeed determined by the membrane mechanics. Intriguingly, neuronal activity modulates the cholesterol level of plasma membrane, leading to shrinkage of axon pearls. Consequently, the conduction velocity of action potentials becomes slower. These data reveal that biophysical forces dictate axon morphology and function and that modulation of membrane mechanics likely underlies plasticity of unmyelinated axons.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549090">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549090" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549090">
        <p class="paperTitle">Functional Connectivity Development along the Sensorimotor-Association Axis Enhances the Cortical Hierarchy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549090" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549090" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Luo, A.; Sydnor, V. J.; Pines, A.; Larsen, B.; Alexander-Bloch, A. F.; Cieslak, M.; Covitz, S.; Chen, A.; Esper, N. B.; Feczko, E.; Franco, A. R.; Gur, R. E.; Gur, R. C.; Houghton, A.; Hu, F.; Keller, A. S.; Kiar, G.; Mehta, K.; Salum, G. A.; Tapera, T.; Xu, T.; Zhao, C.; Fair, D. A.; Salo, T.; Shinohara, R. T.; Milham, M. P.; Satterthwaite, T. D.</p>
        <p class="info">Score: 28.2, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549090' target='https://doi.org/10.1101/2023.07.20.549090'> 10.1101/2023.07.20.549090</a></p>
        <p class="abstract">Human cortical maturation has been posited to be organized along the sensorimotor-association (S-A) axis, a hierarchical axis of brain organization that spans from unimodal sensorimotor cortices to transmodal association cortices. Here, we investigate the hypothesis that the development of functional connectivity during childhood through adolescence conforms to the cortical hierarchy defined by the S-A axis. We tested this pre-registered hypothesis in four large-scale, independent datasets (total n = 3,355; ages 5-23 years): the Philadelphia Neurodevelopmental Cohort (n = 1,207), Nathan Kline Institute-Rockland Sample (n = 397), Human Connectome Project: Development (n = 625), and Healthy Brain Network (n = 1,126). In each dataset, the development of functional connectivity systematically varied along the S-A axis. Connectivity in sensorimotor regions increased, whereas connectivity in association cortices declined, refining and reinforcing the cortical hierarchy. These robust and generalizable results establish that the sensorimotor-association axis of cortical organization encodes the dominant pattern of functional connectivity development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549331">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549331" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549331">
        <p class="paperTitle">PolyGR and polyPR knock-in mice reveal a conserved neuroprotective extracellular matrix signature in C9orf72 ALS/FTD neurons</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549331" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549331" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Milioto, C.; Carcole, M.; Giblin, A.; Coneys, R.; Attrebi, O.; Ahmed, M.; Harris, S. S.; Lee, B. I.; Yang, M.; Nirujogi, R. S.; Biggs, D.; Salomonsson, S.; Zanovello, M.; De Oliveira, P.; Katona, E.; Glaria, I.; Mikheenko, A.; Geary, B.; Udine, E.; Vaizoglu, D.; Rademakers, R.; van Blitterswijk, M.; Devoy, A.; Hong, S.; Partridge, L.; Fratta, P.; Alessi, D.; Davies, B.; Busche, M. A.; Greensmith, L.; Fisher, E. M.; Isaacs, A. M.</p>
        <p class="info">Score: 28.0, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549331' target='https://doi.org/10.1101/2023.07.17.549331'> 10.1101/2023.07.17.549331</a></p>
        <p class="abstract">A GGGGCC repeat expansion in C9orf72 is the most common genetic cause of ALS and FTD (C9ALS/FTD). The presence of dipeptide repeat (DPR) proteins, generated by translation of the expanded repeat, is a major pathogenic feature of C9ALS/FTD pathology, but their most relevant effects in a physiological context are not known. Here, we generated C9orf72 DPR knock-in mouse models characterised by physiological expression of 400 codon-optimised polyGR or polyPR repeats, and heterozygous C9orf72 reduction. (GR)400 and (PR)400 knock-in mice exhibit cortical neuronal hyperexcitability, age-dependent spinal motor neuron loss and progressive motor dysfunction, showing that they recapitulate key features of C9FTD/ALS. Quantitative proteomics revealed an increase in extracellular matrix (ECM) proteins in (GR)400 and (PR)400 spinal cord, with the collagen COL6A1 the most increased protein. This signature of increased ECM proteins was also present in C9ALS patient iPSC-motor neurons indicating it is a conserved feature of C9ALS/FTD. TGF-{beta}1 was one of the top predicted regulators of this ECM signature and polyGR expression in human iPSC-neurons was sufficient to induce TGF-{beta}1 followed by COL6A1, indicating TGF-{beta}1 is one driver of the ECM signature. Knockdown of the TGF-{beta}1 or COL6A1 orthologue in Drosophila dramatically and specifically exacerbated neurodegeneration in polyGR flies, showing that TGF-{beta}1 and COL6A1 protect against polyGR toxicity. Altogether, our physiological C9orf72 DPR knock-in mice have revealed a neuroprotective and conserved ECM signature in C9FTD/ALS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.31.543173">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.31.543173" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.31.543173">
        <p class="paperTitle">Scalable projected Light Sheet Microscopy for high-resolution imaging of living and cleared samples</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.31.543173" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.31.543173" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, Y.; Chauhan, S.; Gong, C.; Dayton, H.; Xu, C.; De La Cruz, E. D.; Datta, M. S.; Leong, K. W.; Dietrich, L. E. P.; Tomer, R.</p>
        <p class="info">Score: 49.3, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.31.543173' target='https://doi.org/10.1101/2023.05.31.543173'> 10.1101/2023.05.31.543173</a></p>
        <p class="abstract">Light sheet fluorescence microscopy (LSFM) is a widely used imaging technique for living and large cleared samples. However, high-performance LSFM systems are often prohibitively expensive and not easily scalable for high-throughput applications. Here, we introduce a cost-effective, scalable, and versatile high-resolution imaging framework, called projected Light Sheet Microscopy (pLSM), which repurposes readily available off-the-shelf consumer-grade components and an over-the-network control architecture to achieve high-resolution imaging of living and cleared samples. We extensively characterize the pLSM framework and showcase its capabilities through high-resolution, multi-color imaging and quantitative analysis of mouse and post-mortem human brain samples cleared using various techniques. Moreover, we show the applicability of pLSM for high-throughput molecular phenotyping of human induced pluripotent cells (iPSC)-derived brain and vessel organoids. Additionally, we utilized pLSM for comprehensive live imaging of bacterial pellicle biofilms at the air-liquid interface, uncovering their intricate layered architecture and diverse cellular dynamics across different depths. Overall, the pLSM framework has the potential to further democratize LSFM by making high-resolution light sheet microscopy more accessible and scalable.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
