<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>immunology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>immunology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="SARS-CoV-2 antibodies cross-react and enhance dengue infection
Authors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.
Score: 363.0, Published: 2023-10-10 DOI: 10.1101/2023.10.09.557914
Dengue disease is highly prevalent in tropical and subtropical regions worldwide.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/immunology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="immunology" />
<meta property="og:description" content="SARS-CoV-2 antibodies cross-react and enhance dengue infection
Authors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.
Score: 363.0, Published: 2023-10-10 DOI: 10.1101/2023.10.09.557914
Dengue disease is highly prevalent in tropical and subtropical regions worldwide." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/immunology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-05T10:37:37+00:00" />
<meta property="article:modified_time" content="2023-11-05T10:37:37+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="immunology"/>
<meta name="twitter:description" content="SARS-CoV-2 antibodies cross-react and enhance dengue infection
Authors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.
Score: 363.0, Published: 2023-10-10 DOI: 10.1101/2023.10.09.557914
Dengue disease is highly prevalent in tropical and subtropical regions worldwide."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "immunology",
      "item": "https://trxiv.yorks0n.com/posts/immunology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "immunology",
  "name": "immunology",
  "description": "SARS-CoV-2 antibodies cross-react and enhance dengue infection\nAuthors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.\nScore: 363.0, Published: 2023-10-10 DOI: 10.1101/2023.10.09.557914\nDengue disease is highly prevalent in tropical and subtropical regions worldwide.",
  "keywords": [
    
  ],
  "articleBody": " SARS-CoV-2 antibodies cross-react and enhance dengue infection\nAuthors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.\nScore: 363.0, Published: 2023-10-10 DOI: 10.1101/2023.10.09.557914\nDengue disease is highly prevalent in tropical and subtropical regions worldwide. However, its pathogenesis is still incompletely understood, particularly in comparison to other endemic viruses. Antibody-dependent enhancement (ADE) is a well-known phenomenon for dengue viruses. Given the recent surge in dengue cases and potential cross-reactivity with SARS-CoV-2 antibodies, this study explores the impact of anti-SARS-CoV-2 antibodies on DENV-2 infection. The study assessed the cross-reactivity of SARS-CoV-2 antibodies with the DENV-2 Virus. Human convalescent plasma samples collected during different waves of COVID-19 and monoclonal and polyclonal antibodies raised against SARS-CoV-2 were examined for their potential to cause ADE of DENV-2 infection using cell-based assays. The study found that anti-SARS-CoV-2 antibodies acquired from natural infection in humans or through experimental immunization in animals were cross-reactive with DENV-2 and had the potential to enhance DENV-2 infection in K562 and U937 cells. In-silico and in-vitro studies indicated a strong interaction between SARS-CoV-2 antibodies and DENV-2 E-protein, providing a molecular basis for these findings. This study is the first to demonstrate that anti-SARS-CoV-2 antibodies can cross-react with DENV-2 and can enhance its infection through ADE. These findings have implications for SARS-CoV-2 vaccine development and deployment strategies in regions where dengue is endemic. SummaryAntibodies against SARS-CoV-2 (RBD and Spike) showed significant cross reactivity with DENV-2 (E protein). Also, anti-SARS-CoV-2-commercial antibodies, immunised animal sera and 46 human convalescent plasma samples (from different waves of pandemic) demonstrated antibody-dependent enhancement (ADE) of DENV-2 infection.\nRapid activation of IL-2 receptor signaling by CD301b+DC-derived IL-2 dictates the outcome of helper T cell differentiation\nAuthors: Tatsumi, N.; El-Fenej, J.; Davila-Pagan, A.; Kumamoto, Y.\nScore: 36.6, Published: 2023-10-31 DOI: 10.1101/2023.10.26.564276\nEffector T helper (Th) cell differentiation is fundamental to functional adaptive immunity. Different subsets of dendritic cells (DCs) preferentially induce different types of Th cells, but the fate instruction mechanism for Th type 2 (Th2) differentiation remains enigmatic, as the critical DC-derived cue has not been clearly identified. Here, we show that CD301b+ DCs, a major Th2-inducing DC subset, drive Th2 differentiation through cognate interaction by kick-starting IL-2 receptor signaling in CD4T cells. Mechanistically, CD40 engagement induces IL-2 production selectively from CD301b+ DCs to maximize CD25 expression in CD4 T cells, which is required specifically for the Th2 fate decision. On the other hand, CD25 in CD301b+ DCs facilitates directed action of IL-2 toward cognate CD4T cells. Furthermore, CD301b+ DC-derived IL-2 skews CD4T cells away from the T follicular helper fate. These results highlight the critical role of DC-intrinsic CD40-IL-2 axis in bifurcation of Th cell fate.\nPan-cancer profiling of tumor-infiltrating natural killer cells through transcriptional reference mapping\nAuthors: Netskar, H. K.; Pfefferle, A.; Goodridge, J. P.; Sohlberg, E.; Dufva, O.; Teichmann, S.; Clancy, T.; horowitz, a.; Malmberg, K.-J.\nScore: 28.0, Published: 2023-10-30 DOI: 10.1101/2023.10.26.564050\nThe functional diversity of natural killer (NK) cell repertoires stems from differentiation, homeostatic receptor-ligand interactions, and adaptive-like responses to viral infections. Here, we generated a single-cell transcriptional reference map of healthy human blood and tissue-derived NK cells, with temporal resolution and fate-specific expression of gene regulator networks defining NK cell differentiation. Using transfer learning, transcriptomes of tumor-infiltrating NK cells from seven solid tumor types (427 patients), combined from 39 datasets, were incorporated into the reference map and interrogated for tumor microenvironment (TME)-induced perturbations. We identified six functionally distinct NK cellular states in healthy and malignant tissues, two of which were commonly enriched for across tumor types: a dysfunctional stressed CD56bright state susceptible to TME-induced immunosuppression and a cytotoxic TME-resistant effector CD56dim state. The ratio of stressed CD56bright and effector CD56dim was predictive of patient outcome in malignant melanoma and osteosarcoma. This resource may inform the design of novel NK cell therapies and can be extended endlessly through transfer learning to interrogate new datasets from experimental perturbations or disease conditions.\nSARS-CoV-2 spike protein induces the cytokine release syndrome by stimulating T cells to produce more IL-2\nAuthors: Niu, C.; Liang, T.; Chen, Y.; Zhu, S.; Zhou, L.; Chen, N.; Qian, L.; Wang, Y.; Li, M.; Zhou, X.; Cui, J.\nScore: 27.6, Published: 2023-11-01 DOI: 10.1101/2023.11.01.565098\nCytokine release syndrome (CRS) is one of the leading causes of mortality in COVID-19 patients caused by the SARS-CoV-2 coronavirus. However, the mechanism of CRS induced by SARS-CoV-2 is vague. This study shows that dendritic cells loaded with spike protein of SARS-CoV-2 stimulate T cells to release much more IL-2, which subsequently cooperates with spike protein to facilitate peripheral blood mononuclear cells to release IL-1{beta}, IL-6, and IL-8. These effects are achieved via IL-2 stimulation of NK cells to release TNF- and IFN-{gamma}, as well as T cells to release IFN-{gamma}. Mechanistically, IFN-{gamma} and TNF- enhance the transcription of CD40, and the interaction of CD40 and its ligand stabilizes the membrane expression of TLR4 which serves as a receptor of spike protein on the surface of monocytes. As a result, there is a constant interaction between spike protein and TLR4, leading to continuous activation of NF-{kappa}B. Furthermore, TNF- also activates NF-{kappa}B signaling in monocytes, which further cooperates with IFN-{gamma} and spike protein to modulate NF-{kappa}B-dependent transcription of CRS-related inflammatory cytokines. Targeting TNF-/IFN-{gamma} in combination with TLR4 may represent a promising therapeutic approach for alleviating CRS in individuals with COVID-19.\nCritical role of CD206+ macrophages in organizing anti-tumor immunity\nAuthors: Ray, A.; Hu, K. H.; Kersten, K.; Kuhn, N. F.; Samad, B.; Combes, A. J.; Krummel, M. F.\nScore: 20.2, Published: 2023-11-02 DOI: 10.1101/2023.10.31.560822\nTumor-associated macrophages (TAMs) are frequently and simplistically categorized as immunosuppressive, and one molecule prominently used to highlight their so-called M2 state is the surface protein CD206. However, direct evidence of the impact of macrophages remains impaired by the lack of sufficiently penetrant and specific tools to manipulate them in vivo. We thus made a novel conditional CD206 knock-in mouse to specifically visualize and/or deplete these TAMs. Early depletion of CD206+ macrophages and monocytes (here, MonoMacs) strikingly led to an indirect loss of a key anti-tumor network of NK cells, conventional type I dendritic cells (cDC1) and CD8 T cells. Among myeloid cells, we found that the CD206+ TAMs are the primary producers of CXCL9, the well-established chemoattractant for CXCR3-expressing NK and CD8 T cells. In contrast, a population of stress-responsive TAMs (Hypoxic or Spp1+) and immature monocytes, which remain following depletion, expressed vastly diminished levels of CXCL9. We confirmed that the missing NK and CD8 T cells are the primary producers of the cDC1-attracting chemokine Xcl1 and cDC1 growth factor Flt3l. Consistent with the loss of this critical network, CD206+ TAM depletion decreased tumor control in mice. Likewise, in humans, the CD206+ MonoMac signature correlated robustly with stimulatory cDC1 signature genes. Together, these findings negate the classification of CD206+ macrophages as immunosuppressive and instead illuminate the role of this majority of TAMs in organizing a critical tumor-reactive archetype of immunity.\nBenchmarking and integrating human B-cell receptor genomic and antibody proteomic profiling\nAuthors: Le Quy, K.; Chernigovskaya, M.; Stensland, M.; Singh, S.; Leem, J.; Revale, S.; Galson, J. D.; Nyman, T. A.; Snapkow, I.; Greiff, V.\nScore: 19.8, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565093\nImmunoglobulins (Ig), which exist either as B-cell receptors (BCR) on the surface of B cells or as antibodies when secreted, play a key role in the recognition and response to antigenic threats. The capability to jointly characterize the BCR and antibody repertoire is crucial in understanding human adaptive immunity. From peripheral blood, bulk BCR sequencing (bulkBCR-seq) currently provides the highest sampling depth, single-cell BCR sequencing (scBCR-seq) allows for paired chain characterization, and antibody peptide sequencing by tandem mass spectrometry (Ab-seq) provides information on the composition of secreted antibodies in the serum. Although still rare, studies combining these three technologies would comprehensively capture the humoral immune response. Yet, it has not been benchmarked to what extent the datasets generated by these three technologies overlap and complement each other. To address this question, we isolated peripheral blood B cells from healthy donors and sequenced BCRs at bulk and single-cell level, in addition to utilizing publicly available sequencing data. Integrated analysis was performed on these datasets, resolved by replicates and across individuals. Simultaneously, serum antibodies were isolated, digested with multiple proteases, and analyzed with Ab-seq. Systems immunology analysis showed high concordance in repertoire features between bulk and scBCR-seq within individuals, especially when replicates were utilized. In addition, Ab-seq identified clonotype-specific peptides using both bulk and scBCR-seq library references, demonstrating the feasibility of combining scBCR-seq and Ab-seq for reconstructing paired-chain Ig sequences from the serum antibody repertoire. Collectively, our work serves as a proof-of-principle for combining bulk sequencing, single-cell sequencing, and mass spectrometry as complementary methods towards capturing humoral immunity in its entirety.\nNK cell-monocyte crosstalk underlies NK cell activation in severe COVID-19\nAuthors: Lee, M. J.; de los Rios Kobara, I.; Barnard, T. R.; Vales Torres, X.; Tobin, N. H.; Ferbas, K. G.; Rimoin, A. W. J.; Yang, O. O.; Aldrovandi, G. M.; Wilk, A. J.; Fulcher, J.; Blish, C. A.\nScore: 17.6, Published: 2023-10-30 DOI: 10.1101/2023.10.27.564440\nNK cells in the peripheral blood of severe COVID-19 patients exhibit a unique profile characterized by activation and dysfunction. Previous studies have identified soluble factors, including type I interferon and TGF{beta}, that underlie this dysregulation. However, the role of cell-cell interactions in mediating changes in NK cell function during COVID-19 remains unclear. To address this question, we combined cell-cell communication analysis on existing single-cell RNA sequencing data with in vitro primary cell co-culture experiments to dissect the mechanisms underlying NK cell dysfunction in COVID-19. We found that NK cells are predicted to interact most strongly with monocytes and that this occurs via both soluble factors and direct interactions. To validate these findings, we performed in vitro co-cultures in which NK cells from healthy donors were incubated with monocytes from COVID-19+ or healthy donors. Co-culture of healthy NK cells with monocytes from COVID-19 patients recapitulated aspects of the NK cell phenotype observed in severe COVID-19, including decreased expression of NKG2D, increased expression of activation markers, and increased proliferation. When these experiments were performed in a transwell setting, we found that only CD56bright CD16- NK cells were activated in the presence of severe COVID-19 patient monocytes. O-link analysis of supernatants from transwell co-cultures revealed that cultures containing severe COVID-19 patient monocytes had significantly elevated levels of proinflammatory cytokines and chemokines as well as TGF{beta}. Collectively, these results demonstrate that interactions between NK cells and monocytes in the peripheral blood of COVID-19 patients contribute to NK cell activation and dysfunction in severe COVID-19. BACKGROUNDNatural killer (NK) cells are innate lymphocytes that are critical antiviral effectors. Because of their role in controlling acute viral infections, multiple studies have evaluated the role of NK cells in SARS-CoV-2 infection. Such studies revealed that NK cell phenotype and function are significantly altered by severe COVID-19; the peripheral NK cells of severe COVID-19 patients are highly activated and proliferative(1-5), with increased expression of cytotoxic molecules, Ki-67, and several surface markers of activation(3, 5-8). However, these NK cells also have dysfunctional cytotoxic responses to both tumor target cells(1, 2, 9, 10) and SARS-CoV-2-infected target cells(9, 10). Given that peripheral NK cells are thought to migrate to the lung during COVID-19(11-13), these results suggest that the NK cells of severe COVID-19 patients may be incapable of mounting a successful antiviral response to SARS-CoV-2 infection. Although the unique phenotype and dysfunctionality of NK cells in severe COVID-19 has been well-characterized, the processes underlying these phenomena have not. Only one study has conducted in vitro mechanistic experiments to identify a possible cause of NK cell dysfunction: Witkowski et al. identified serum-derived TGF{beta} as a suppressor of NK cell functionality in severe COVID-19 patients(9). However, this study did not identify the source of serum TGF{beta}. Additionally, given the high degree of complexity within the immune system, there are likely other causes of NK cell dysfunction in COVID-19 that have thus far remain unexplored. One such mechanism may be the myriad of interactions between NK cells and other peripheral immune cells. NK cells are known to interact with CD4 and CD8 T cells, dendritic cells, neutrophils, and macrophages/monocytes(14), which can prime NK cell cytotoxicity or induce tolerance. Previous work by our lab suggested the potential for NK cell-monocyte crosstalk in severe COVID-19 through the expression of ligands for NK cell activating receptors on the monocytes of these patients(3). Crosstalk between NK cells and monocytes plays a role in regulating the NK cell response to other infections, including HIV-1(15, 16), mouse(17) and human cytomegalovirus(18), and malaria(19) through mechanisms including secretion of NK cell-regulating cytokines by monocytes. In this study, we used a combination of computational and in vitro methods to dissect the interactions between NK cells and monocytes in severe COVID-19. We utilized primary NK cells and monocytes from a large cohort of COVID-19 patients to demonstrate that co-culture of healthy NK cells with monocytes from severe COVID-19 donors can partially recapitulate the activated phenotype observed in the NK cells from COVID-19 patients. We then interrogated the mechanisms by which this activation occurs by performing NK cell-monocyte co-cultures in a transwell setting and using O-link to analyze the cytokines present in this system. Collectively, our work identifies monocytes as a driver of NK cell activation in severe COVID-19 and reveals interactions between NK cells and monocytes that may underlie this process.\nImmunosuppression is a conserved driver of tuberculosis susceptibility\nAuthors: Kotov, D. I.; Lee, O. V.; Ji, D. X.; Jaye, D. L.; Suliman, S.; Gabay, C.; Vance, R. E.\nScore: 16.3, Published: 2023-11-01 DOI: 10.1101/2023.10.27.564420\nMycobacterium tuberculosis (Mtb) causes 1.6 million deaths a year. However, no individual mouse model fully recapitulates the hallmarks of human tuberculosis disease. Here we report that a comparison across three different susceptible mouse models identifies Mtb-induced gene signatures that predict active TB disease in humans significantly better than a signature from the standard C57BL/6 mouse model. An increase in lung myeloid cells, including neutrophils, was conserved across the susceptible mouse models, mimicking the neutrophilic inflammation observed in humans. Myeloid cells in the susceptible models and non-human primates exhibited high expression of immunosuppressive molecules including the IL-1 receptor antagonist, which inhibits IL-1 signaling. Prior reports have suggested that excessive IL-1 signaling impairs Mtb control. By contrast, we found that enhancement of IL-1 signaling via deletion of IL-1 receptor antagonist promoted bacterial control in all three susceptible mouse models. IL-1 signaling enhanced cytokine production by lymphoid and stromal cells, suggesting a mechanism for IL-1 signaling in promoting Mtb control. Thus, we propose that myeloid cell expression of immunosuppressive molecules is a conserved mechanism exacerbating Mtb disease in mice, non-human primates, and humans.\nOne-carbon unit supplementation fuels tumor-infiltrating T cells and augments checkpoint blockade\nAuthors: Xu, X.; Chen, Z.; Bartman, C. R.; Xing, X.; Olszewski, K.; Rabinowitz, J.\nScore: 10.4, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565193\nNucleotides perform important metabolic functions, carrying energy and feeding nucleic acid synthesis. Here, we use isotope tracing-mass spectrometry to quantitate the contributions to purine nucleotides of salvage versus de novo synthesis. We further explore the impact of augmenting a key precursor for purine synthesis, one-carbon (1C) units. We show that tumors and tumor-infiltrating T cells (relative to splenic T cells) synthesize purines de novo. Purine synthesis requires two 1C units, which come from serine catabolism and circulating formate. Shortage of 1C units is a potential bottleneck for anti-tumor immunity. Elevating circulating formate drives its usage by tumor-infiltrating T cells. Orally administered methanol functions as a formate pro-drug, with deuteration enabling control of formate-production kinetics. In MC38 tumors, safe doses of methanol raise formate levels and augment anti-PD-1 checkpoint blockade, tripling durable regressions. Thus, 1C deficiency can gate antitumor immunity and this metabolic checkpoint can be overcome with pharmacological 1C supplementation.\nA novel bioinformatics pipeline for the identification of immune inhibitory receptors as potential therapeutic targets\nAuthors: Singh, A.; Miranda Bedate, A.; von richthofen, H. J.; van der Vlist, M.; Kuhn, R.; Yermanos, A.; Kuball, J. J.; Kesmir, C.; Pascoal Ramos, M. I.; Meyaard, L.\nScore: 8.4, Published: 2023-10-28 DOI: 10.1101/2023.10.24.563834\nBlocking inhibitory receptors like PD-1 and CTLA-4 has revolutionized cancer treatment in recent years. However, despite major successes in melanoma and lung cancer, the majority of cancer types are not responsive to these immunotherapies. As such, there is an ongoing need for the identification of novel inhibitory receptors as drug targets. Most inhibitory receptors signal via immunoreceptor tyrosine-based inhibitory motifs (ITIMs) and previous studies have estimated that our genome contains over 1600 ITIM-bearing transmembrane proteins. However, further testing and development of this large number of candidates requires increased understanding of their expression patterns and likelihood to function as inhibitory receptor. To assist in the selection of novel inhibitory receptor as therapeutic targets, we designed a novel bioinformatics pipeline integrating machine learning-guided structural predictions and sequence-based likelihood models to identify 51 known and 390 putative inhibitory receptors. Using publicly available transcriptomics data of immune cells, we determined the expression of these novel inhibitory receptors, and classified them into previously proposed functional categories. Known and putative inhibitory receptors were expressed across a wide variety of immune cells, and we found cell type-specific patterns in expression of these receptors. We used our pipeline to study inhibitory receptor expression patterns in single cell transcriptomics data of tumour infiltrating T cells. We determined that putative immune inhibitory receptors were expressed differentially in CD4+ and CD8+ T cell subsets, including exhausted CD8+ T cells and CD4+ memory T cells, which could allow for subset-specific targeting. In conclusion, we present an inhibitory receptor pipeline that identifies 51 known and 390 novel inhibitory receptors. This pipeline will support future drug target selection across diseases where therapeutic targeting of immune inhibitory receptors is warranted.\n",
  "wordCount" : "2975",
  "inLanguage": "en",
  "datePublished": "2023-11-05T10:37:37Z",
  "dateModified": "2023-11-05T10:37:37Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/immunology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      immunology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 5, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.557914">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.557914" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.557914">
        <p class="paperTitle">SARS-CoV-2 antibodies cross-react and enhance dengue infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.557914" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.557914" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.</p>
        <p class="info">Score: 363.0, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.557914' target='https://doi.org/10.1101/2023.10.09.557914'> 10.1101/2023.10.09.557914</a></p>
        <p class="abstract">Dengue disease is highly prevalent in tropical and subtropical regions worldwide. However, its pathogenesis is still incompletely understood, particularly in comparison to other endemic viruses. Antibody-dependent enhancement (ADE) is a well-known phenomenon for dengue viruses. Given the recent surge in dengue cases and potential cross-reactivity with SARS-CoV-2 antibodies, this study explores the impact of anti-SARS-CoV-2 antibodies on DENV-2 infection.

The study assessed the cross-reactivity of SARS-CoV-2 antibodies with the DENV-2 Virus. Human convalescent plasma samples collected during different waves of COVID-19 and monoclonal and polyclonal antibodies raised against SARS-CoV-2 were examined for their potential to cause ADE of DENV-2 infection using cell-based assays. The study found that anti-SARS-CoV-2 antibodies acquired from natural infection in humans or through experimental immunization in animals were cross-reactive with DENV-2 and had the potential to enhance DENV-2 infection in K562 and U937 cells. In-silico and in-vitro studies indicated a strong interaction between SARS-CoV-2 antibodies and DENV-2 E-protein, providing a molecular basis for these findings. This study is the first to demonstrate that anti-SARS-CoV-2 antibodies can cross-react with DENV-2 and can enhance its infection through ADE. These findings have implications for SARS-CoV-2 vaccine development and deployment strategies in regions where dengue is endemic.

SummaryAntibodies against SARS-CoV-2 (RBD and Spike) showed significant cross reactivity with DENV-2 (E protein). Also, anti-SARS-CoV-2-commercial antibodies, immunised animal sera and 46 human convalescent plasma samples (from different waves of pandemic) demonstrated antibody-dependent enhancement (ADE) of DENV-2 infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564276">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564276" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564276">
        <p class="paperTitle">Rapid activation of IL-2 receptor signaling by CD301b&#43;DC-derived IL-2 dictates the outcome of helper T cell differentiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564276" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564276" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tatsumi, N.; El-Fenej, J.; Davila-Pagan, A.; Kumamoto, Y.</p>
        <p class="info">Score: 36.6, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564276' target='https://doi.org/10.1101/2023.10.26.564276'> 10.1101/2023.10.26.564276</a></p>
        <p class="abstract">Effector T helper (Th) cell differentiation is fundamental to functional adaptive immunity. Different subsets of dendritic cells (DCs) preferentially induce different types of Th cells, but the fate instruction mechanism for Th type 2 (Th2) differentiation remains enigmatic, as the critical DC-derived cue has not been clearly identified. Here, we show that CD301b&#43; DCs, a major Th2-inducing DC subset, drive Th2 differentiation through cognate interaction by  kick-starting IL-2 receptor signaling in CD4T cells. Mechanistically, CD40 engagement induces IL-2 production selectively from CD301b&#43; DCs to maximize CD25 expression in CD4 T cells, which is required specifically for the Th2 fate decision. On the other hand, CD25 in CD301b&#43; DCs facilitates directed action of IL-2 toward cognate CD4T cells. Furthermore, CD301b&#43; DC-derived IL-2 skews CD4T cells away from the T follicular helper fate. These results highlight the critical role of DC-intrinsic CD40-IL-2 axis in bifurcation of Th cell fate.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564050">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564050" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564050">
        <p class="paperTitle">Pan-cancer profiling of tumor-infiltrating natural killer cells through transcriptional reference mapping</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564050" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564050" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Netskar, H. K.; Pfefferle, A.; Goodridge, J. P.; Sohlberg, E.; Dufva, O.; Teichmann, S.; Clancy, T.; horowitz, a.; Malmberg, K.-J.</p>
        <p class="info">Score: 28.0, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564050' target='https://doi.org/10.1101/2023.10.26.564050'> 10.1101/2023.10.26.564050</a></p>
        <p class="abstract">The functional diversity of natural killer (NK) cell repertoires stems from differentiation, homeostatic receptor-ligand interactions, and adaptive-like responses to viral infections. Here, we generated a single-cell transcriptional reference map of healthy human blood and tissue-derived NK cells, with temporal resolution and fate-specific expression of gene regulator networks defining NK cell differentiation. Using transfer learning, transcriptomes of tumor-infiltrating NK cells from seven solid tumor types (427 patients), combined from 39 datasets, were incorporated into the reference map and interrogated for tumor microenvironment (TME)-induced perturbations. We identified six functionally distinct NK cellular states in healthy and malignant tissues, two of which were commonly enriched for across tumor types: a dysfunctional  stressed CD56bright state susceptible to TME-induced immunosuppression and a cytotoxic TME-resistant  effector CD56dim state. The ratio of  stressed CD56bright and  effector CD56dim was predictive of patient outcome in malignant melanoma and osteosarcoma. This resource may inform the design of novel NK cell therapies and can be extended endlessly through transfer learning to interrogate new datasets from experimental perturbations or disease conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565098">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565098" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565098">
        <p class="paperTitle">SARS-CoV-2 spike protein induces the cytokine release syndrome by stimulating T cells to produce more IL-2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565098" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565098" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Niu, C.; Liang, T.; Chen, Y.; Zhu, S.; Zhou, L.; Chen, N.; Qian, L.; Wang, Y.; Li, M.; Zhou, X.; Cui, J.</p>
        <p class="info">Score: 27.6, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565098' target='https://doi.org/10.1101/2023.11.01.565098'> 10.1101/2023.11.01.565098</a></p>
        <p class="abstract">Cytokine release syndrome (CRS) is one of the leading causes of mortality in COVID-19 patients caused by the SARS-CoV-2 coronavirus. However, the mechanism of CRS induced by SARS-CoV-2 is vague. This study shows that dendritic cells loaded with spike protein of SARS-CoV-2 stimulate T cells to release much more IL-2, which subsequently cooperates with spike protein to facilitate peripheral blood mononuclear cells to release IL-1{beta}, IL-6, and IL-8. These effects are achieved via IL-2 stimulation of NK cells to release TNF- and IFN-{gamma}, as well as T cells to release IFN-{gamma}. Mechanistically, IFN-{gamma} and TNF- enhance the transcription of CD40, and the interaction of CD40 and its ligand stabilizes the membrane expression of TLR4 which serves as a receptor of spike protein on the surface of monocytes. As a result, there is a constant interaction between spike protein and TLR4, leading to continuous activation of NF-{kappa}B. Furthermore, TNF- also activates NF-{kappa}B signaling in monocytes, which further cooperates with IFN-{gamma} and spike protein to modulate NF-{kappa}B-dependent transcription of CRS-related inflammatory cytokines. Targeting TNF-/IFN-{gamma} in combination with TLR4 may represent a promising therapeutic approach for alleviating CRS in individuals with COVID-19.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.560822">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.560822" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.560822">
        <p class="paperTitle">Critical role of CD206&#43; macrophages in organizing anti-tumor immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.560822" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.560822" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ray, A.; Hu, K. H.; Kersten, K.; Kuhn, N. F.; Samad, B.; Combes, A. J.; Krummel, M. F.</p>
        <p class="info">Score: 20.2, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.560822' target='https://doi.org/10.1101/2023.10.31.560822'> 10.1101/2023.10.31.560822</a></p>
        <p class="abstract">Tumor-associated macrophages (TAMs) are frequently and simplistically categorized as immunosuppressive, and one molecule prominently used to highlight their so-called M2 state is the surface protein CD206. However, direct evidence of the impact of macrophages remains impaired by the lack of sufficiently penetrant and specific tools to manipulate them in vivo. We thus made a novel conditional CD206 knock-in mouse to specifically visualize and/or deplete these TAMs. Early depletion of CD206&#43; macrophages and monocytes (here, MonoMacs) strikingly led to an indirect loss of a key anti-tumor network of NK cells, conventional type I dendritic cells (cDC1) and CD8 T cells. Among myeloid cells, we found that the CD206&#43; TAMs are the primary producers of CXCL9, the well-established chemoattractant for CXCR3-expressing NK and CD8 T cells. In contrast, a population of stress-responsive TAMs (Hypoxic or Spp1&#43;) and immature monocytes, which remain following depletion, expressed vastly diminished levels of CXCL9. We confirmed that the missing NK and CD8 T cells are the primary producers of the cDC1-attracting chemokine Xcl1 and cDC1 growth factor Flt3l. Consistent with the loss of this critical network, CD206&#43; TAM depletion decreased tumor control in mice. Likewise, in humans, the CD206&#43; MonoMac signature correlated robustly with stimulatory cDC1 signature genes. Together, these findings negate the classification of CD206&#43; macrophages as immunosuppressive and instead illuminate the role of this majority of TAMs in organizing a critical tumor-reactive archetype of immunity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565093">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565093" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565093">
        <p class="paperTitle">Benchmarking and integrating human B-cell receptor genomic and antibody proteomic profiling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565093" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565093" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Le Quy, K.; Chernigovskaya, M.; Stensland, M.; Singh, S.; Leem, J.; Revale, S.; Galson, J. D.; Nyman, T. A.; Snapkow, I.; Greiff, V.</p>
        <p class="info">Score: 19.8, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565093' target='https://doi.org/10.1101/2023.11.01.565093'> 10.1101/2023.11.01.565093</a></p>
        <p class="abstract">Immunoglobulins (Ig), which exist either as B-cell receptors (BCR) on the surface of B cells or as antibodies when secreted, play a key role in the recognition and response to antigenic threats. The capability to jointly characterize the BCR and antibody repertoire is crucial in understanding human adaptive immunity. From peripheral blood, bulk BCR sequencing (bulkBCR-seq) currently provides the highest sampling depth, single-cell BCR sequencing (scBCR-seq) allows for paired chain characterization, and antibody peptide sequencing by tandem mass spectrometry (Ab-seq) provides information on the composition of secreted antibodies in the serum. Although still rare, studies combining these three technologies would comprehensively capture the humoral immune response. Yet, it has not been benchmarked to what extent the datasets generated by these three technologies overlap and complement each other. To address this question, we isolated peripheral blood B cells from healthy donors and sequenced BCRs at bulk and single-cell level, in addition to utilizing publicly available sequencing data. Integrated analysis was performed on these datasets, resolved by replicates and across individuals. Simultaneously, serum antibodies were isolated, digested with multiple proteases, and analyzed with Ab-seq. Systems immunology analysis showed high concordance in repertoire features between bulk and scBCR-seq within individuals, especially when replicates were utilized. In addition, Ab-seq identified clonotype-specific peptides using both bulk and scBCR-seq library references, demonstrating the feasibility of combining scBCR-seq and Ab-seq for reconstructing paired-chain Ig sequences from the serum antibody repertoire. Collectively, our work serves as a proof-of-principle for combining bulk sequencing, single-cell sequencing, and mass spectrometry as complementary methods towards capturing humoral immunity in its entirety.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564440">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564440" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564440">
        <p class="paperTitle">NK cell-monocyte crosstalk underlies NK cell activation in severe COVID-19</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564440" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564440" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, M. J.; de los Rios Kobara, I.; Barnard, T. R.; Vales Torres, X.; Tobin, N. H.; Ferbas, K. G.; Rimoin, A. W. J.; Yang, O. O.; Aldrovandi, G. M.; Wilk, A. J.; Fulcher, J.; Blish, C. A.</p>
        <p class="info">Score: 17.6, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564440' target='https://doi.org/10.1101/2023.10.27.564440'> 10.1101/2023.10.27.564440</a></p>
        <p class="abstract">NK cells in the peripheral blood of severe COVID-19 patients exhibit a unique profile characterized by activation and dysfunction. Previous studies have identified soluble factors, including type I interferon and TGF{beta}, that underlie this dysregulation. However, the role of cell-cell interactions in mediating changes in NK cell function during COVID-19 remains unclear. To address this question, we combined cell-cell communication analysis on existing single-cell RNA sequencing data with in vitro primary cell co-culture experiments to dissect the mechanisms underlying NK cell dysfunction in COVID-19. We found that NK cells are predicted to interact most strongly with monocytes and that this occurs via both soluble factors and direct interactions. To validate these findings, we performed in vitro co-cultures in which NK cells from healthy donors were incubated with monocytes from COVID-19&#43; or healthy donors. Co-culture of healthy NK cells with monocytes from COVID-19 patients recapitulated aspects of the NK cell phenotype observed in severe COVID-19, including decreased expression of NKG2D, increased expression of activation markers, and increased proliferation. When these experiments were performed in a transwell setting, we found that only CD56bright CD16- NK cells were activated in the presence of severe COVID-19 patient monocytes. O-link analysis of supernatants from transwell co-cultures revealed that cultures containing severe COVID-19 patient monocytes had significantly elevated levels of proinflammatory cytokines and chemokines as well as TGF{beta}. Collectively, these results demonstrate that interactions between NK cells and monocytes in the peripheral blood of COVID-19 patients contribute to NK cell activation and dysfunction in severe COVID-19.

BACKGROUNDNatural killer (NK) cells are innate lymphocytes that are critical antiviral effectors. Because of their role in controlling acute viral infections, multiple studies have evaluated the role of NK cells in SARS-CoV-2 infection. Such studies revealed that NK cell phenotype and function are significantly altered by severe COVID-19; the peripheral NK cells of severe COVID-19 patients are highly activated and proliferative(1-5), with increased expression of cytotoxic molecules, Ki-67, and several surface markers of activation(3, 5-8). However, these NK cells also have dysfunctional cytotoxic responses to both tumor target cells(1, 2, 9, 10) and SARS-CoV-2-infected target cells(9, 10). Given that peripheral NK cells are thought to migrate to the lung during COVID-19(11-13), these results suggest that the NK cells of severe COVID-19 patients may be incapable of mounting a successful antiviral response to SARS-CoV-2 infection.

Although the unique phenotype and dysfunctionality of NK cells in severe COVID-19 has been well-characterized, the processes underlying these phenomena have not. Only one study has conducted in vitro mechanistic experiments to identify a possible cause of NK cell dysfunction: Witkowski et al. identified serum-derived TGF{beta} as a suppressor of NK cell functionality in severe COVID-19 patients(9). However, this study did not identify the source of serum TGF{beta}. Additionally, given the high degree of complexity within the immune system, there are likely other causes of NK cell dysfunction in COVID-19 that have thus far remain unexplored. One such mechanism may be the myriad of interactions between NK cells and other peripheral immune cells. NK cells are known to interact with CD4 and CD8 T cells, dendritic cells, neutrophils, and macrophages/monocytes(14), which can prime NK cell cytotoxicity or induce tolerance. Previous work by our lab suggested the potential for NK cell-monocyte crosstalk in severe COVID-19 through the expression of ligands for NK cell activating receptors on the monocytes of these patients(3). Crosstalk between NK cells and monocytes plays a role in regulating the NK cell response to other infections, including HIV-1(15, 16), mouse(17) and human cytomegalovirus(18), and malaria(19) through mechanisms including secretion of NK cell-regulating cytokines by monocytes.

In this study, we used a combination of computational and in vitro methods to dissect the interactions between NK cells and monocytes in severe COVID-19. We utilized primary NK cells and monocytes from a large cohort of COVID-19 patients to demonstrate that co-culture of healthy NK cells with monocytes from severe COVID-19 donors can partially recapitulate the activated phenotype observed in the NK cells from COVID-19 patients. We then interrogated the mechanisms by which this activation occurs by performing NK cell-monocyte co-cultures in a transwell setting and using O-link to analyze the cytokines present in this system. Collectively, our work identifies monocytes as a driver of NK cell activation in severe COVID-19 and reveals interactions between NK cells and monocytes that may underlie this process.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564420">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564420" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564420">
        <p class="paperTitle">Immunosuppression is a conserved driver of tuberculosis susceptibility</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564420" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564420" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kotov, D. I.; Lee, O. V.; Ji, D. X.; Jaye, D. L.; Suliman, S.; Gabay, C.; Vance, R. E.</p>
        <p class="info">Score: 16.3, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564420' target='https://doi.org/10.1101/2023.10.27.564420'> 10.1101/2023.10.27.564420</a></p>
        <p class="abstract">Mycobacterium tuberculosis (Mtb) causes 1.6 million deaths a year. However, no individual mouse model fully recapitulates the hallmarks of human tuberculosis disease. Here we report that a comparison across three different susceptible mouse models identifies Mtb-induced gene signatures that predict active TB disease in humans significantly better than a signature from the standard C57BL/6 mouse model. An increase in lung myeloid cells, including neutrophils, was conserved across the susceptible mouse models, mimicking the neutrophilic inflammation observed in humans. Myeloid cells in the susceptible models and non-human primates exhibited high expression of immunosuppressive molecules including the IL-1 receptor antagonist, which inhibits IL-1 signaling. Prior reports have suggested that excessive IL-1 signaling impairs Mtb control. By contrast, we found that enhancement of IL-1 signaling via deletion of IL-1 receptor antagonist promoted bacterial control in all three susceptible mouse models. IL-1 signaling enhanced cytokine production by lymphoid and stromal cells, suggesting a mechanism for IL-1 signaling in promoting Mtb control. Thus, we propose that myeloid cell expression of immunosuppressive molecules is a conserved mechanism exacerbating Mtb disease in mice, non-human primates, and humans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565193">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565193" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565193">
        <p class="paperTitle">One-carbon unit supplementation fuels tumor-infiltrating T cells and augments checkpoint blockade</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565193" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565193" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, X.; Chen, Z.; Bartman, C. R.; Xing, X.; Olszewski, K.; Rabinowitz, J.</p>
        <p class="info">Score: 10.4, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565193' target='https://doi.org/10.1101/2023.11.01.565193'> 10.1101/2023.11.01.565193</a></p>
        <p class="abstract">Nucleotides perform important metabolic functions, carrying energy and feeding nucleic acid synthesis. Here, we use isotope tracing-mass spectrometry to quantitate the contributions to purine nucleotides of salvage versus de novo synthesis. We further explore the impact of augmenting a key precursor for purine synthesis, one-carbon (1C) units. We show that tumors and tumor-infiltrating T cells (relative to splenic T cells) synthesize purines de novo. Purine synthesis requires two 1C units, which come from serine catabolism and circulating formate. Shortage of 1C units is a potential bottleneck for anti-tumor immunity. Elevating circulating formate drives its usage by tumor-infiltrating T cells. Orally administered methanol functions as a formate pro-drug, with deuteration enabling control of formate-production kinetics. In MC38 tumors, safe doses of methanol raise formate levels and augment anti-PD-1 checkpoint blockade, tripling durable regressions. Thus, 1C deficiency can gate antitumor immunity and this metabolic checkpoint can be overcome with pharmacological 1C supplementation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563834">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563834" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563834">
        <p class="paperTitle">A novel bioinformatics pipeline for the identification of immune inhibitory receptors as potential therapeutic targets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563834" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563834" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Singh, A.; Miranda Bedate, A.; von richthofen, H. J.; van der Vlist, M.; Kuhn, R.; Yermanos, A.; Kuball, J. J.; Kesmir, C.; Pascoal Ramos, M. I.; Meyaard, L.</p>
        <p class="info">Score: 8.4, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563834' target='https://doi.org/10.1101/2023.10.24.563834'> 10.1101/2023.10.24.563834</a></p>
        <p class="abstract">Blocking inhibitory receptors like PD-1 and CTLA-4 has revolutionized cancer treatment in recent years. However, despite major successes in melanoma and lung cancer, the majority of cancer types are not responsive to these immunotherapies. As such, there is an ongoing need for the identification of novel inhibitory receptors as drug targets. Most inhibitory receptors signal via immunoreceptor tyrosine-based inhibitory motifs (ITIMs) and previous studies have estimated that our genome contains over 1600 ITIM-bearing transmembrane proteins. However, further testing and development of this large number of candidates requires increased understanding of their expression patterns and likelihood to function as inhibitory receptor.

To assist in the selection of novel inhibitory receptor as therapeutic targets, we designed a novel bioinformatics pipeline integrating machine learning-guided structural predictions and sequence-based likelihood models to identify 51 known and 390 putative inhibitory receptors. Using publicly available transcriptomics data of immune cells, we determined the expression of these novel inhibitory receptors, and classified them into previously proposed functional categories.

Known and putative inhibitory receptors were expressed across a wide variety of immune cells, and we found cell type-specific patterns in expression of these receptors. We used our pipeline to study inhibitory receptor expression patterns in single cell transcriptomics data of tumour infiltrating T cells. We determined that putative immune inhibitory receptors were expressed differentially in CD4&#43; and CD8&#43; T cell subsets, including exhausted CD8&#43; T cells and CD4&#43; memory T cells, which could allow for subset-specific targeting.

In conclusion, we present an inhibitory receptor pipeline that identifies 51 known and 390 novel inhibitory receptors. This pipeline will support future drug target selection across diseases where therapeutic targeting of immune inhibitory receptors is warranted.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
