<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>immunology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>immunology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Mucosal Adenoviral-vectored Vaccine Boosting Durably Prevents XBB.1.16 Infection in Nonhuman Primates
Authors: Gagne, M.; Flynn, B. J.; Andrew, S. F.; Flebbe, D. R.; Mychalowych, A.; Lamb, E.; Davis-Gardner, M. E.; Burnett, M. R.; Serebryannyy, L. A.; Lin, B. C.; Pessaint, L.; Todd, J.-P. M.; Ziff, Z. E.; Maule, E.; Carroll, R.; Naisan, M.; Jethmalani, Y.; Case, J. B.; Dmitriev, I. P.; Kashentseva, E. A.; Ying, B.; Dodson, A.; Kouneski, K.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/immunology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="immunology" />
<meta property="og:description" content="Mucosal Adenoviral-vectored Vaccine Boosting Durably Prevents XBB.1.16 Infection in Nonhuman Primates
Authors: Gagne, M.; Flynn, B. J.; Andrew, S. F.; Flebbe, D. R.; Mychalowych, A.; Lamb, E.; Davis-Gardner, M. E.; Burnett, M. R.; Serebryannyy, L. A.; Lin, B. C.; Pessaint, L.; Todd, J.-P. M.; Ziff, Z. E.; Maule, E.; Carroll, R.; Naisan, M.; Jethmalani, Y.; Case, J. B.; Dmitriev, I. P.; Kashentseva, E. A.; Ying, B.; Dodson, A.; Kouneski, K." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/immunology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-15T10:40:26+00:00" />
<meta property="article:modified_time" content="2023-11-15T10:40:26+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="immunology"/>
<meta name="twitter:description" content="Mucosal Adenoviral-vectored Vaccine Boosting Durably Prevents XBB.1.16 Infection in Nonhuman Primates
Authors: Gagne, M.; Flynn, B. J.; Andrew, S. F.; Flebbe, D. R.; Mychalowych, A.; Lamb, E.; Davis-Gardner, M. E.; Burnett, M. R.; Serebryannyy, L. A.; Lin, B. C.; Pessaint, L.; Todd, J.-P. M.; Ziff, Z. E.; Maule, E.; Carroll, R.; Naisan, M.; Jethmalani, Y.; Case, J. B.; Dmitriev, I. P.; Kashentseva, E. A.; Ying, B.; Dodson, A.; Kouneski, K."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "immunology",
      "item": "https://trxiv.yorks0n.com/posts/immunology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "immunology",
  "name": "immunology",
  "description": "Mucosal Adenoviral-vectored Vaccine Boosting Durably Prevents XBB.1.16 Infection in Nonhuman Primates\nAuthors: Gagne, M.; Flynn, B. J.; Andrew, S. F.; Flebbe, D. R.; Mychalowych, A.; Lamb, E.; Davis-Gardner, M. E.; Burnett, M. R.; Serebryannyy, L. A.; Lin, B. C.; Pessaint, L.; Todd, J.-P. M.; Ziff, Z. E.; Maule, E.; Carroll, R.; Naisan, M.; Jethmalani, Y.; Case, J. B.; Dmitriev, I. P.; Kashentseva, E. A.; Ying, B.; Dodson, A.; Kouneski, K.",
  "keywords": [
    
  ],
  "articleBody": " Mucosal Adenoviral-vectored Vaccine Boosting Durably Prevents XBB.1.16 Infection in Nonhuman Primates\nAuthors: Gagne, M.; Flynn, B. J.; Andrew, S. F.; Flebbe, D. R.; Mychalowych, A.; Lamb, E.; Davis-Gardner, M. E.; Burnett, M. R.; Serebryannyy, L. A.; Lin, B. C.; Pessaint, L.; Todd, J.-P. M.; Ziff, Z. E.; Maule, E.; Carroll, R.; Naisan, M.; Jethmalani, Y.; Case, J. B.; Dmitriev, I. P.; Kashentseva, E. A.; Ying, B.; Dodson, A.; Kouneski, K.; Doria-Rose, N. A.; O'Dell, S.; Godbole, S.; Laboune, F.; Henry, A. R.; Marquez, J.; Teng, I.-T.; Wang, L.; Zhou, Q.; Wali, B.; Ellis, M.; Zouantchangadou, S.; Van Ry, A.; Lewis, M. G.; Andersen, H.; Kwong, P. D.; Curiel, D. T.; Foulds, K. E.; Nason, M. C.; Suthar, M\nScore: 260.1, Published: 2023-11-08 DOI: 10.1101/2023.11.06.565765\nWaning immunity and continued virus evolution have limited the durability of protection from symptomatic infection mediated by intramuscularly (IM)-delivered mRNA vaccines against COVID-19 although protection from severe disease remains high. Mucosal vaccination has been proposed as a strategy to increase protection at the site of SARS-CoV-2 infection by enhancing airway immunity, potentially reducing rates of infection and transmission. Here, we compared protection against XBB.1.16 virus challenge 5 months following IM or mucosal boosting in non-human primates (NHP) that had previously received a two-dose mRNA-1273 primary vaccine regimen. The mucosal boost was composed of a bivalent chimpanzee adenoviral-vectored vaccine encoding for both SARS-CoV-2 WA1 and BA.5 spike proteins (ChAd-SARS-CoV-2-S) and delivered either by an intranasal mist or an inhaled aerosol. An additional group of animals was boosted by the IM route with bivalent WA1/BA.5 spike-matched mRNA (mRNA-1273.222) as a benchmark control. NHP were challenged in the upper and lower airways 18 weeks after boosting with XBB.1.16, a heterologous Omicron lineage strain. Cohorts boosted with ChAd-SARS-CoV-2-S by an aerosolized or intranasal route had low to undetectable virus replication as assessed by levels of subgenomic SARS-CoV-2 RNA in the lungs and nose, respectively. In contrast, animals that received the mRNA-1273.222 boost by the IM route showed minimal protection against virus replication in the upper airway but substantial reduction of virus RNA levels in the lower airway. Immune analysis showed that the mucosal vaccines elicited more durable antibody and T cell responses than the IM vaccine. Protection elicited by the aerosolized vaccine was associated with mucosal IgG and IgA responses, whereas protection elicited by intranasal delivery was mediated primarily by mucosal IgA. Thus, durable immunity and effective protection against a highly transmissible heterologous variant in both the upper and lower airways can be achieved by mucosal delivery of a virus-vectored vaccine. Our study provides a template for the development of mucosal vaccines that limit infection and transmission against respiratory pathogens. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=166 HEIGHT=200 SRC=\"FIGDIR/small/565765v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (48K): org.highwire.dtl.DTLVardef@1dff1b1org.highwire.dtl.DTLVardef@174c368org.highwire.dtl.DTLVardef@c8f61borg.highwire.dtl.DTLVardef@1957fa2_HPS_FORMAT_FIGEXP M_FIG C_FIG\nInsights into B Cell and Antibody Kinetics Against SARS-CoV-2 Variants Using Mathematical Modelling\nAuthors: Farhang-Sardroodi, S.; Deng, X.; Portet, S.; Arino, J.; Craig, M.\nScore: 51.9, Published: 2023-11-12 DOI: 10.1101/2023.11.10.566587\nB cells and antibodies are crucial in protecting against infections like SARS-CoV-2. However, antibody levels decline after infection or vaccination, reducing defences against future SARS-CoV-2 infections. To understand antibody production and decline, we developed a mathematical model that predicts germinal center B cell, long-lived plasma cell, memory B cell, and antibody dynamics. Our focus was on B cell activation and antibody generation following both primary and secondary SARS-CoV-2 infections. Aligning our model with clinical data, we adjusted antibody production rates for germinal center B cells and plasma B cells during primary and secondary infections. We also assessed antibody neutralization against Delta and Omicron variants post-primary and secondary exposure. Our findings showed reduced neutralization against Omicron due to its immune evasion. In primary and secondary exposures to Delta and Omicron, our predictions indicated enhanced antibody neutralization in the secondary response within a year of the primary response. We also explored waning immunity, demonstrating how B cell kinetics affect viral neutralization post-primary infection. This study enhances our understanding of humoral immunity to SARS-CoV-2 and can predict antibody dynamics post-infection or vaccination.\nA spatial human thymus cell atlas mapped to a continuous tissue axis\nAuthors: Yayon, N.; Kedlian, V. R.; Boehme, L.; Suo, C.; Wachter, B.; Beuschel, R. T.; Amsalem, O.; Polanski, K.; Koplev, S.; Tuck, E.; Dann, E.; Van Hulle, J.; Perera, S.; Putteman, T.; Predeus, A. V.; Dabrowska, M.; Richardson, L.; Tudor, C.; Kreins, A. Y.; Engelbert, J.; Stephenson, E.; Kleshchevnikov, V.; De Rita, F.; Crossland, D.; Bosticardo, M.; Pala, F.; Prigmore, E.; Chipampe, N.-J.; Prete, M.; Fei, L.; To, K.; Barker, R. A.; He, X.; Van Nieuwerburgh, F.; Bayraktar, O.; Patel, M.; Davies, G. E.; Haniffa, M. A.; Uhlmann, V.; Notarangelo, L. D.; Germain, R. N.; Radtke, A. J.; Marioni, J. C.; Tag\nScore: 39.1, Published: 2023-10-28 DOI: 10.1101/2023.10.25.562925\nT cells develop from circulating precursors, which enter the thymus and migrate throughout specialised sub-compartments to support maturation and selection. This process starts already in early fetal development and is highly active until the involution of the thymus in adolescence. To map the micro-anatomical underpinnings of this process in pre- vs. post-natal states, we undertook a spatially resolved analysis and established a new quantitative morphological framework for the thymus, the Cortico-Medullary Axis. Using this axis in conjunction with the curation of a multimodal single-cell, spatial transcriptomics and high-resolution multiplex imaging atlas, we show that canonical thymocyte trajectories and thymic epithelial cells are highly organised and fully established by post-conception week 12, pinpoint TEC progenitor states, find that TEC subsets and peripheral tissue genes are associated with Hassalls Corpuscles and uncover divergence in the pace and drivers of medullary entry between CD4 vs. CD8 T cell lineages. These findings are complemented with a holistic toolkit for spatial analysis and annotation, providing a basis for a detailed understanding of T lymphocyte development.\nSARS-CoV-2 and its ORF3a, E and M viroporins activate inflammasome in human macrophages and induce of IL-1α in pulmonary epithelial and endothelial cells\nAuthors: Abrozek-Latecka, M.; Kozlowski, P.; Hoser, G.; Bandyszewska, M.; Hanusek, K.; Nowis, D.; Golab, J.; Grzanka, M.; Piekielko-Witkowska, A.; Schulz, L.; Hornung, F.; Deinhardt-Emmer, S.; Kozlowska, E.; Skirecki, T.\nScore: 18.9, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566917\nInflammasome assembly is a potent mechanism responsible for the host protection against pathogens, including viruses. When compromised, it can allow viral replication, while when disrupted, it can perpetuate pathological responses by IL-1 signaling and pyroptotic cell death. SARS-CoV-2 infection was shown to activate inflammasome in the lungs of COVID-19 patients, however, potential mechanisms responsible for this response are not fully elucidated. In this study, we investigated the effects of ORF3a, E and M SARS-CoV-2 viroporins in the inflammasome activation in major populations of alveolar sentinel cells: macrophages, epithelial and endothelial cells. We demonstrated that each viroporin is capable of activation of the inflammasome in macrophages to trigger cell death and IL-1 release from epithelial and endothelial cells. Small molecule NLRP3 inflammasome inhibitors reduced IL-1 release but weakly affected the pyroptosis. Importantly, we discovered that while SARS-CoV-2 could not infect the pulmonary microvascular endothelial cells it induced IL-1 and IL-33 release. Together, these findings highlight the essential role of macrophages as the major inflammasome-activating cell population in the lungs and point to endothelial cell expressed IL-1 as a potential novel component driving the pulmonary immunothromobosis in COVID-19.\nReinstatement of RIG-I in chickens via genetic modification reveals new insights into the dynamic evolution of avian immune sensors\nAuthors: Sid, H.; von Heyl, T.; Schleibinger, S.; Klinger, R.; Guabiraba, R.; Guillory, V.; Schade, B.; Elleder, D.; Sives, S.; Vervelde, L.; Trapp, S.; Schusser, B.\nScore: 34.1, Published: 2023-11-02 DOI: 10.1101/2023.11.01.564710\nRetinoic acid-inducible gene I (RIG-I) activates mitochondrial antiviral signaling proteins, initiating the antiviral response. RIG-I and RNF135, a ubiquitin ligase regulator, are missing in domestic chickens but conserved in mallard ducks. It was long believed that chickens RIG-I loss was linked to increased avian influenza susceptibility. We reinstated both genes in chickens and examined their susceptibility to the avian influenza virus H7N1. Uninfected RIG-I-expressing chickens exhibited shifts in T and B cells, while the H7N1 infection led to severe disease, persistent weight loss, and increased viral replication. Conversely, the co-expression of RIG-I and RNF135 reduced the viral replication and was associated with high inflammatory response. Our data indicate that the loss of RIG-I in chickens likely evolved to counteract deleterious inflammation caused by viral infection. We highlight the effects of restoring evolutionary lost genes in birds and suggest a new immunological approach to reduce viral replication and prevent infection. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=138 SRC=\"FIGDIR/small/564710v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (29K): org.highwire.dtl.DTLVardef@b87535org.highwire.dtl.DTLVardef@17256aorg.highwire.dtl.DTLVardef@19968c3org.highwire.dtl.DTLVardef@acec77_HPS_FORMAT_FIGEXP M_FIG C_FIG\nSoquelitinb, A selective inhibitor of interleukin-2-inducible kinase (ITK), is active in several murine models of T cell-mediated inflammatory disease\nAuthors: Hsu, L.-Y.; Rosenbaum, J. T.; Lindner, S.; Allanore, Y.; Cauvet, A.; Pawar, R. D.; Li, D.; Van Den Brink, M.; Miller, R. A.\nScore: 36.0, Published: 2023-11-01 DOI: 10.1101/2023.10.27.564296\nInterleukin-2-inducible T cell kinase (or ITK) is a tyrosine kinase predominantly expressed by T lymphocytes. It plays a major role in T cell activation, differentiation, and receptor signaling. Studies in mouse models have established that the absence or inhibition of ITK reduces the secretion of Th2 and Th17 cytokines, while favoring the production of Th1 cytokines and the differentiation of T cells to regulatory T cells (T regs). We recently characterized the activity of soquelitinib (SQL), a selective, covalent inhibitor of ITK in vitro using mouse or human T cells and in vivo in murine models of cancer. We hypothesized that selective pharmacologic blockade of ITK could attenuate a variety of T cell-mediated inflammatory diseases including murine models that resemble asthma, pulmonary fibrosis, systemic sclerosis, psoriasis, and acute graft versus host disease. In each model, SQL demonstrated substantial ability to ameliorate the disease process along with effects on cytokines and/or T cell subsets consistent with the reported function of ITK. Our studies demonstrate that selective inhibition of ITK by SQL is potentially a novel therapeutic approach for several forms of T cell-mediated, inflammatory diseases.\nRobust detection of SARS-CoV-2 exposure in population using T-cell repertoire profiling\nAuthors: Vlasova, E. K.; Nekrasova, A. I.; Komkov, A. Y.; Izraelson, M.; Snigir, E. A.; Mitrofanov, S. I.; Yudin, V. S.; Makarov, V. V.; Kesnikov, A. A.; Kirienko, D.; Pivnyuk, A.; Shelyakin, P. V.; Mamedov, I. Z.; Rebrikov, D. V.; Chudakov, D. M.; Yudin, S. M.; Skvortsova, V. I.; Britanova, O.; Shugay, M. A.\nScore: 9.6, Published: 2023-11-09 DOI: 10.1101/2023.11.08.566227\nThe COVID-19 pandemic clearly demonstrates the need to monitor the spread of infectious diseases and population immunity. Probing adaptive immunity by sequencing the repertoire of antigen receptors (Rep-Seq) encoding specificity and immunological memory has become a method of choice for immunology studies. Rep-Seq can detect the imprint of past and ongoing infections and study individual responses to SARS-CoV-2 as shown in a number of recent studies. Here we apply a machine learning approach to two large datasets with more than 1200 high-quality repertoires from healthy and COVID-19-convalescent donor repertoires to infer T-cell receptor (TCR) repertoire features that were induced by SARS-CoV-2 exposure. Proper standardization of Rep-Seq batches, access to human leukocyte antigen (HLA) typing and both - and {beta}-chain sequences of TCRs allowed us to generate a high-quality biomarker database and build a robust and highly accurate classifier for COVID-19 exposure applicable to individual TCR repertoires obtained using different protocols, paving a way to Rep-Seq-based immune status assessment in large cohorts of donors.\nSimultaneous analysis of pMHC binding and reactivity unveils virus-specific CD8 T cell immunity to a concise epitope set\nAuthors: Kristensen, N. P.; Dionisio, E.; Bentzen, A. K.; Tamhane, T.; Kemming, J. S.; Nos, G.; Voss, L. F.; Hansen, U. K.; Lauer, G. M.; Hadrup, S. R.\nScore: 12.6, Published: 2023-11-07 DOI: 10.1101/2023.11.06.565606\nKnowledge of widely recognized T-cell epitopes against common virus infections are vital for immune monitoring and characterization of relevant antigen-specific CD8 T cells and their antigen receptors. We therefore aimed to establish a concise and validated epitope panel for monitoring human virus-specific immunity complete with data on both prevalence of recognition and reactivity in humans. To achieve this, we first establish TCR downregulation, and loss of peptide major histocompatibility (pMHC) multimer-binding, as an early and sensitive marker of T cell reactivity after peptide stimulation. We next applied TCR downregulation in a high-throughput assay by monitoring binding, and loss of binding (i.e. reactivity), to libraries of DNA-barcode labelled pMHC multimers in paired unstimulated/stimulated samples. This novel method allowed us to access T-cell responses in 48 donors towards 929 epitopes recorded in the Immune Epitope Database (IEDB) encompassing 29 virus common infections and 25 different HLA alleles. This yielded a concise panel of 137 virus epitopes, many of which were underrepresented in the public domain, recognized by T cells in peripheral blood. 84% of these epitopes exhibited prevalent reactivity to peptide stimulation, which was associated with effector and long-term memory phenotypes. Conversely, non-reactive responses correlated with naive and immunosenescence phenotypes. This study represents the largest effort to unbiasedly assess T-cell recognition and reactivity to common virus infections in healthy individuals providing a minimal epitope panel for monitoring adaptive immune responses in humans. Significance StatementCD8 T-cell epitopes are widely available in public databases yet many are not recognized in the general population. Here we undertook an exhaustive screening process using \"state-of-the-art\" methods to assess both T-cell recognition and reactivity against common virus infections, which holds significant implications for shaping T-cell immunity and disease protection. We identify 137 commonly recognized epitopes from common virus infections to which T cell responses are expected to occur in human donors. Importantly, several of the verified epitopes were underreported in public databases compared to their observed prevalence of recognition and high cellular frequency making this an important reference dataset and resource for immunologists studying antigen-specific T cells across different immunopathologies and contexts including autoimmunity, infectious disease and cancer immunotherapy.\nIntestinal cDC1s provide IL-12 dependent and independent functions required for CD4+ T cell mediated resistance to Cryptosporidium\nAuthors: Cohn, I. S.; Wallbank, B. A.; Haskins, B. E.; O'Dea, K. M.; Pardy, R. D.; Shaw, S.; Merolle, M. I.; Gullicksrud, J.; Christian, D. A.; Striepen, B.; Hunter, C. A.\nScore: 8.5, Published: 2023-11-13 DOI: 10.1101/2023.11.11.566669\nCryptosporidium is an enteric pathogen that is a prominent cause of diarrheal disease. Control of this infection requires CD4+ T cells, though the processes that lead to T cell-mediated resistance have been difficult to assess. Here, Cryptosporidium parasites that express MHCII-restricted model antigens were generated to dissect the early events that influence CD4+ T cell priming and effector function. These studies highlight that parasite-specific CD4+ T cells are primed in the draining mesenteric lymph node (mesLN) and differentiate into Th1 cells in the gut, where they mediate IFN-{gamma}-dependent control of the infection. Although type 1 conventional dendritic cells (cDC1s) were not required for initial priming of CD4+ T cells, cDC1s were required for CD4+ T cell expansion and gut homing. cDC1s were also a major source of IL-12 that was not required for priming but promoted full differentiation of CD4+ T cells and local production of IFN-{gamma}. Together, these studies reveal distinct roles for cDC1s in shaping CD4+ T cell responses to enteric infection: first to drive early expansion in the mesLN and second to drive effector responses in the gut.\nA modular platform to display multiple hemagglutinin subtypes on a single immunogen\nAuthors: Lamson, D. T.; Mohamed, F. A. N.; Vu, M.; Maurer, D. P.; Ronsard, L.; Lingwood, D.; Schmidt, A. G.\nScore: 8.0, Published: 2023-11-13 DOI: 10.1101/2023.11.09.566478\nNext-generation influenza vaccines aim to elicit cross-reactive humoral responses to multiple influenza subtypes. Such increased breadth would not only improve seasonal vaccines but may afford ''universal'' protection against influenza subtypes including those with pandemic potential. Here, we describe a ''beads-on-a-string'' (BOAS) immunogen, that tandemly links up to eight distinct hemagglutinin (HA) head domains from circulating and non-circulating influenzas. These BOAS are immunogenic in the murine model and elicit comparable serum responses to each individual component. Notably, we also find that BOAS elicit cross-reactive, and neutralizing responses to influenza subtypes not included in the immunizing immunogen. Furthermore, BOAS conjugation to protein-based ferritin nanoparticles does not significantly augment serum responses suggesting that our BOAS platform is sufficient for eliciting cross-reactive responses without off-target effects induced by the nanoparticle scaffold. This mix-and-match immunogen design strategy is a robust platform for eliciting responses to multiple influenza subtypes via a single immunogen, and a potential platform for other viral glycoproteins.\n",
  "wordCount" : "2713",
  "inLanguage": "en",
  "datePublished": "2023-11-15T10:40:26Z",
  "dateModified": "2023-11-15T10:40:26Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/immunology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      immunology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 15, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565765">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565765" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565765">
        <p class="paperTitle">Mucosal Adenoviral-vectored Vaccine Boosting Durably Prevents XBB.1.16 Infection in Nonhuman Primates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565765" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565765" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gagne, M.; Flynn, B. J.; Andrew, S. F.; Flebbe, D. R.; Mychalowych, A.; Lamb, E.; Davis-Gardner, M. E.; Burnett, M. R.; Serebryannyy, L. A.; Lin, B. C.; Pessaint, L.; Todd, J.-P. M.; Ziff, Z. E.; Maule, E.; Carroll, R.; Naisan, M.; Jethmalani, Y.; Case, J. B.; Dmitriev, I. P.; Kashentseva, E. A.; Ying, B.; Dodson, A.; Kouneski, K.; Doria-Rose, N. A.; O&#39;Dell, S.; Godbole, S.; Laboune, F.; Henry, A. R.; Marquez, J.; Teng, I.-T.; Wang, L.; Zhou, Q.; Wali, B.; Ellis, M.; Zouantchangadou, S.; Van Ry, A.; Lewis, M. G.; Andersen, H.; Kwong, P. D.; Curiel, D. T.; Foulds, K. E.; Nason, M. C.; Suthar, M</p>
        <p class="info">Score: 260.1, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565765' target='https://doi.org/10.1101/2023.11.06.565765'> 10.1101/2023.11.06.565765</a></p>
        <p class="abstract">Waning immunity and continued virus evolution have limited the durability of protection from symptomatic infection mediated by intramuscularly (IM)-delivered mRNA vaccines against COVID-19 although protection from severe disease remains high. Mucosal vaccination has been proposed as a strategy to increase protection at the site of SARS-CoV-2 infection by enhancing airway immunity, potentially reducing rates of infection and transmission. Here, we compared protection against XBB.1.16 virus challenge 5 months following IM or mucosal boosting in non-human primates (NHP) that had previously received a two-dose mRNA-1273 primary vaccine regimen. The mucosal boost was composed of a bivalent chimpanzee adenoviral-vectored vaccine encoding for both SARS-CoV-2 WA1 and BA.5 spike proteins (ChAd-SARS-CoV-2-S) and delivered either by an intranasal mist or an inhaled aerosol. An additional group of animals was boosted by the IM route with bivalent WA1/BA.5 spike-matched mRNA (mRNA-1273.222) as a benchmark control. NHP were challenged in the upper and lower airways 18 weeks after boosting with XBB.1.16, a heterologous Omicron lineage strain. Cohorts boosted with ChAd-SARS-CoV-2-S by an aerosolized or intranasal route had low to undetectable virus replication as assessed by levels of subgenomic SARS-CoV-2 RNA in the lungs and nose, respectively. In contrast, animals that received the mRNA-1273.222 boost by the IM route showed minimal protection against virus replication in the upper airway but substantial reduction of virus RNA levels in the lower airway. Immune analysis showed that the mucosal vaccines elicited more durable antibody and T cell responses than the IM vaccine. Protection elicited by the aerosolized vaccine was associated with mucosal IgG and IgA responses, whereas protection elicited by intranasal delivery was mediated primarily by mucosal IgA. Thus, durable immunity and effective protection against a highly transmissible heterologous variant in both the upper and lower airways can be achieved by mucosal delivery of a virus-vectored vaccine. Our study provides a template for the development of mucosal vaccines that limit infection and transmission against respiratory pathogens.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=166 HEIGHT=200 SRC=&#34;FIGDIR/small/565765v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (48K):
org.highwire.dtl.DTLVardef@1dff1b1org.highwire.dtl.DTLVardef@174c368org.highwire.dtl.DTLVardef@c8f61borg.highwire.dtl.DTLVardef@1957fa2_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566587">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566587" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566587">
        <p class="paperTitle">Insights into B Cell and Antibody Kinetics Against SARS-CoV-2 Variants Using Mathematical Modelling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566587" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566587" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Farhang-Sardroodi, S.; Deng, X.; Portet, S.; Arino, J.; Craig, M.</p>
        <p class="info">Score: 51.9, Published: 2023-11-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566587' target='https://doi.org/10.1101/2023.11.10.566587'> 10.1101/2023.11.10.566587</a></p>
        <p class="abstract">B cells and antibodies are crucial in protecting against infections like SARS-CoV-2. However, antibody levels decline after infection or vaccination, reducing defences against future SARS-CoV-2 infections. To understand antibody production and decline, we developed a mathematical model that predicts germinal center B cell, long-lived plasma cell, memory B cell, and antibody dynamics. Our focus was on B cell activation and antibody generation following both primary and secondary SARS-CoV-2 infections. Aligning our model with clinical data, we adjusted antibody production rates for germinal center B cells and plasma B cells during primary and secondary infections. We also assessed antibody neutralization against Delta and Omicron variants post-primary and secondary exposure. Our findings showed reduced neutralization against Omicron due to its immune evasion. In primary and secondary exposures to Delta and Omicron, our predictions indicated enhanced antibody neutralization in the secondary response within a year of the primary response. We also explored waning immunity, demonstrating how B cell kinetics affect viral neutralization post-primary infection. This study enhances our understanding of humoral immunity to SARS-CoV-2 and can predict antibody dynamics post-infection or vaccination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.562925">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.562925" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.562925">
        <p class="paperTitle">A spatial human thymus cell atlas mapped to a continuous tissue axis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.562925" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.562925" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yayon, N.; Kedlian, V. R.; Boehme, L.; Suo, C.; Wachter, B.; Beuschel, R. T.; Amsalem, O.; Polanski, K.; Koplev, S.; Tuck, E.; Dann, E.; Van Hulle, J.; Perera, S.; Putteman, T.; Predeus, A. V.; Dabrowska, M.; Richardson, L.; Tudor, C.; Kreins, A. Y.; Engelbert, J.; Stephenson, E.; Kleshchevnikov, V.; De Rita, F.; Crossland, D.; Bosticardo, M.; Pala, F.; Prigmore, E.; Chipampe, N.-J.; Prete, M.; Fei, L.; To, K.; Barker, R. A.; He, X.; Van Nieuwerburgh, F.; Bayraktar, O.; Patel, M.; Davies, G. E.; Haniffa, M. A.; Uhlmann, V.; Notarangelo, L. D.; Germain, R. N.; Radtke, A. J.; Marioni, J. C.; Tag</p>
        <p class="info">Score: 39.1, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.562925' target='https://doi.org/10.1101/2023.10.25.562925'> 10.1101/2023.10.25.562925</a></p>
        <p class="abstract">T cells develop from circulating precursors, which enter the thymus and migrate throughout specialised sub-compartments to support maturation and selection. This process starts already in early fetal development and is highly active until the involution of the thymus in adolescence. To map the micro-anatomical underpinnings of this process in pre- vs. post-natal states, we undertook a spatially resolved analysis and established a new quantitative morphological framework for the thymus, the Cortico-Medullary Axis. Using this axis in conjunction with the curation of a multimodal single-cell, spatial transcriptomics and high-resolution multiplex imaging atlas, we show that canonical thymocyte trajectories and thymic epithelial cells are highly organised and fully established by post-conception week 12, pinpoint TEC progenitor states, find that TEC subsets and peripheral tissue genes are associated with Hassalls Corpuscles and uncover divergence in the pace and drivers of medullary entry between CD4 vs. CD8 T cell lineages. These findings are complemented with a holistic toolkit for spatial analysis and annotation, providing a basis for a detailed understanding of T lymphocyte development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566917">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566917" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566917">
        <p class="paperTitle">SARS-CoV-2 and its ORF3a, E and M viroporins activate inflammasome in human macrophages and induce of IL-1α in pulmonary epithelial and endothelial cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566917" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566917" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abrozek-Latecka, M.; Kozlowski, P.; Hoser, G.; Bandyszewska, M.; Hanusek, K.; Nowis, D.; Golab, J.; Grzanka, M.; Piekielko-Witkowska, A.; Schulz, L.; Hornung, F.; Deinhardt-Emmer, S.; Kozlowska, E.; Skirecki, T.</p>
        <p class="info">Score: 18.9, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566917' target='https://doi.org/10.1101/2023.11.13.566917'> 10.1101/2023.11.13.566917</a></p>
        <p class="abstract">Inflammasome assembly is a potent mechanism responsible for the host protection against pathogens, including viruses. When compromised, it can allow viral replication, while when disrupted, it can perpetuate pathological responses by IL-1 signaling and pyroptotic cell death. SARS-CoV-2 infection was shown to activate inflammasome in the lungs of COVID-19 patients, however, potential mechanisms responsible for this response are not fully elucidated. In this study, we investigated the effects of ORF3a, E and M SARS-CoV-2 viroporins in the inflammasome activation in major populations of alveolar sentinel cells: macrophages, epithelial and endothelial cells. We demonstrated that each viroporin is capable of activation of the inflammasome in macrophages to trigger cell death and IL-1 release from epithelial and endothelial cells. Small molecule NLRP3 inflammasome inhibitors reduced IL-1 release but weakly affected the pyroptosis. Importantly, we discovered that while SARS-CoV-2 could not infect the pulmonary microvascular endothelial cells it induced IL-1 and IL-33 release. Together, these findings highlight the essential role of macrophages as the major inflammasome-activating cell population in the lungs and point to endothelial cell expressed IL-1 as a potential novel component driving the pulmonary immunothromobosis in COVID-19.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.564710">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.564710" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.564710">
        <p class="paperTitle">Reinstatement of RIG-I in chickens via genetic modification reveals new insights into the dynamic evolution of avian immune sensors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.564710" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.564710" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sid, H.; von Heyl, T.; Schleibinger, S.; Klinger, R.; Guabiraba, R.; Guillory, V.; Schade, B.; Elleder, D.; Sives, S.; Vervelde, L.; Trapp, S.; Schusser, B.</p>
        <p class="info">Score: 34.1, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.564710' target='https://doi.org/10.1101/2023.11.01.564710'> 10.1101/2023.11.01.564710</a></p>
        <p class="abstract">Retinoic acid-inducible gene I (RIG-I) activates mitochondrial antiviral signaling proteins, initiating the antiviral response. RIG-I and RNF135, a ubiquitin ligase regulator, are missing in domestic chickens but conserved in mallard ducks. It was long believed that chickens RIG-I loss was linked to increased avian influenza susceptibility. We reinstated both genes in chickens and examined their susceptibility to the avian influenza virus H7N1. Uninfected RIG-I-expressing chickens exhibited shifts in T and B cells, while the H7N1 infection led to severe disease, persistent weight loss, and increased viral replication. Conversely, the co-expression of RIG-I and RNF135 reduced the viral replication and was associated with high inflammatory response. Our data indicate that the loss of RIG-I in chickens likely evolved to counteract deleterious inflammation caused by viral infection. We highlight the effects of restoring evolutionary lost genes in birds and suggest a new immunological approach to reduce viral replication and prevent infection.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=138 SRC=&#34;FIGDIR/small/564710v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (29K):
org.highwire.dtl.DTLVardef@b87535org.highwire.dtl.DTLVardef@17256aorg.highwire.dtl.DTLVardef@19968c3org.highwire.dtl.DTLVardef@acec77_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564296">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564296" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564296">
        <p class="paperTitle">Soquelitinb, A selective inhibitor of interleukin-2-inducible kinase (ITK), is active in several murine models of T cell-mediated inflammatory disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564296" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564296" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hsu, L.-Y.; Rosenbaum, J. T.; Lindner, S.; Allanore, Y.; Cauvet, A.; Pawar, R. D.; Li, D.; Van Den Brink, M.; Miller, R. A.</p>
        <p class="info">Score: 36.0, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564296' target='https://doi.org/10.1101/2023.10.27.564296'> 10.1101/2023.10.27.564296</a></p>
        <p class="abstract">Interleukin-2-inducible T cell kinase (or ITK) is a tyrosine kinase predominantly expressed by T lymphocytes. It plays a major role in T cell activation, differentiation, and receptor signaling. Studies in mouse models have established that the absence or inhibition of ITK reduces the secretion of Th2 and Th17 cytokines, while favoring the production of Th1 cytokines and the differentiation of T cells to regulatory T cells (T regs). We recently characterized the activity of soquelitinib (SQL), a selective, covalent inhibitor of ITK in vitro using mouse or human T cells and in vivo in murine models of cancer. We hypothesized that selective pharmacologic blockade of ITK could attenuate a variety of T cell-mediated inflammatory diseases including murine models that resemble asthma, pulmonary fibrosis, systemic sclerosis, psoriasis, and acute graft versus host disease. In each model, SQL demonstrated substantial ability to ameliorate the disease process along with effects on cytokines and/or T cell subsets consistent with the reported function of ITK. Our studies demonstrate that selective inhibition of ITK by SQL is potentially a novel therapeutic approach for several forms of T cell-mediated, inflammatory diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566227">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566227" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566227">
        <p class="paperTitle">Robust detection of SARS-CoV-2 exposure in population using T-cell repertoire profiling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566227" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566227" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vlasova, E. K.; Nekrasova, A. I.; Komkov, A. Y.; Izraelson, M.; Snigir, E. A.; Mitrofanov, S. I.; Yudin, V. S.; Makarov, V. V.; Kesnikov, A. A.; Kirienko, D.; Pivnyuk, A.; Shelyakin, P. V.; Mamedov, I. Z.; Rebrikov, D. V.; Chudakov, D. M.; Yudin, S. M.; Skvortsova, V. I.; Britanova, O.; Shugay, M. A.</p>
        <p class="info">Score: 9.6, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566227' target='https://doi.org/10.1101/2023.11.08.566227'> 10.1101/2023.11.08.566227</a></p>
        <p class="abstract">The COVID-19 pandemic clearly demonstrates the need to monitor the spread of infectious diseases and population immunity. Probing adaptive immunity by sequencing the repertoire of antigen receptors (Rep-Seq) encoding specificity and immunological memory has become a method of choice for immunology studies. Rep-Seq can detect the imprint of past and ongoing infections and study individual responses to SARS-CoV-2 as shown in a number of recent studies. Here we apply a machine learning approach to two large datasets with more than 1200 high-quality repertoires from healthy and COVID-19-convalescent donor repertoires to infer T-cell receptor (TCR) repertoire features that were induced by SARS-CoV-2 exposure. Proper standardization of Rep-Seq batches, access to human leukocyte antigen (HLA) typing and both - and {beta}-chain sequences of TCRs allowed us to generate a high-quality biomarker database and build a robust and highly accurate classifier for COVID-19 exposure applicable to individual TCR repertoires obtained using different protocols, paving a way to Rep-Seq-based immune status assessment in large cohorts of donors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565606">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565606" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565606">
        <p class="paperTitle">Simultaneous analysis of pMHC binding and reactivity unveils virus-specific CD8 T cell immunity to a concise epitope set</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565606" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565606" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kristensen, N. P.; Dionisio, E.; Bentzen, A. K.; Tamhane, T.; Kemming, J. S.; Nos, G.; Voss, L. F.; Hansen, U. K.; Lauer, G. M.; Hadrup, S. R.</p>
        <p class="info">Score: 12.6, Published: 2023-11-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565606' target='https://doi.org/10.1101/2023.11.06.565606'> 10.1101/2023.11.06.565606</a></p>
        <p class="abstract">Knowledge of widely recognized T-cell epitopes against common virus infections are vital for immune monitoring and characterization of relevant antigen-specific CD8 T cells and their antigen receptors. We therefore aimed to establish a concise and validated epitope panel for monitoring human virus-specific immunity complete with data on both prevalence of recognition and reactivity in humans. To achieve this, we first establish TCR downregulation, and loss of peptide major histocompatibility (pMHC) multimer-binding, as an early and sensitive marker of T cell reactivity after peptide stimulation. We next applied TCR downregulation in a high-throughput assay by monitoring binding, and loss of binding (i.e. reactivity), to libraries of DNA-barcode labelled pMHC multimers in paired unstimulated/stimulated samples. This novel method allowed us to access T-cell responses in 48 donors towards 929 epitopes recorded in the Immune Epitope Database (IEDB) encompassing 29 virus common infections and 25 different HLA alleles. This yielded a concise panel of 137 virus epitopes, many of which were underrepresented in the public domain, recognized by T cells in peripheral blood. 84% of these epitopes exhibited prevalent reactivity to peptide stimulation, which was associated with effector and long-term memory phenotypes. Conversely, non-reactive responses correlated with naive and immunosenescence phenotypes. This study represents the largest effort to unbiasedly assess T-cell recognition and reactivity to common virus infections in healthy individuals providing a minimal epitope panel for monitoring adaptive immune responses in humans.

Significance StatementCD8 T-cell epitopes are widely available in public databases yet many are not recognized in the general population. Here we undertook an exhaustive screening process using &#34;state-of-the-art&#34; methods to assess both T-cell recognition and reactivity against common virus infections, which holds significant implications for shaping T-cell immunity and disease protection. We identify 137 commonly recognized epitopes from common virus infections to which T cell responses are expected to occur in human donors. Importantly, several of the verified epitopes were underreported in public databases compared to their observed prevalence of recognition and high cellular frequency making this an important reference dataset and resource for immunologists studying antigen-specific T cells across different immunopathologies and contexts including autoimmunity, infectious disease and cancer immunotherapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.11.566669">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.11.566669" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.11.566669">
        <p class="paperTitle">Intestinal cDC1s provide IL-12 dependent and independent functions required for CD4&#43; T cell mediated resistance to Cryptosporidium</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.11.566669" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.11.566669" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cohn, I. S.; Wallbank, B. A.; Haskins, B. E.; O&#39;Dea, K. M.; Pardy, R. D.; Shaw, S.; Merolle, M. I.; Gullicksrud, J.; Christian, D. A.; Striepen, B.; Hunter, C. A.</p>
        <p class="info">Score: 8.5, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.11.566669' target='https://doi.org/10.1101/2023.11.11.566669'> 10.1101/2023.11.11.566669</a></p>
        <p class="abstract">Cryptosporidium is an enteric pathogen that is a prominent cause of diarrheal disease. Control of this infection requires CD4&#43; T cells, though the processes that lead to T cell-mediated resistance have been difficult to assess. Here, Cryptosporidium parasites that express MHCII-restricted model antigens were generated to dissect the early events that influence CD4&#43; T cell priming and effector function. These studies highlight that parasite-specific CD4&#43; T cells are primed in the draining mesenteric lymph node (mesLN) and differentiate into Th1 cells in the gut, where they mediate IFN-{gamma}-dependent control of the infection. Although type 1 conventional dendritic cells (cDC1s) were not required for initial priming of CD4&#43; T cells, cDC1s were required for CD4&#43; T cell expansion and gut homing. cDC1s were also a major source of IL-12 that was not required for priming but promoted full differentiation of CD4&#43; T cells and local production of IFN-{gamma}. Together, these studies reveal distinct roles for cDC1s in shaping CD4&#43; T cell responses to enteric infection: first to drive early expansion in the mesLN and second to drive effector responses in the gut.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566478">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566478" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566478">
        <p class="paperTitle">A modular platform to display multiple hemagglutinin subtypes on a single immunogen</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566478" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566478" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lamson, D. T.; Mohamed, F. A. N.; Vu, M.; Maurer, D. P.; Ronsard, L.; Lingwood, D.; Schmidt, A. G.</p>
        <p class="info">Score: 8.0, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566478' target='https://doi.org/10.1101/2023.11.09.566478'> 10.1101/2023.11.09.566478</a></p>
        <p class="abstract">Next-generation influenza vaccines aim to elicit cross-reactive humoral responses to multiple influenza subtypes. Such increased breadth would not only improve seasonal vaccines but may afford &#39;&#39;universal&#39;&#39; protection against influenza subtypes including those with pandemic potential. Here, we describe a &#39;&#39;beads-on-a-string&#39;&#39; (BOAS) immunogen, that tandemly links up to eight distinct hemagglutinin (HA) head domains from circulating and non-circulating influenzas. These BOAS are immunogenic in the murine model and elicit comparable serum responses to each individual component. Notably, we also find that BOAS elicit cross-reactive, and neutralizing responses to influenza subtypes not included in the immunizing immunogen. Furthermore, BOAS conjugation to protein-based ferritin nanoparticles does not significantly augment serum responses suggesting that our BOAS platform is sufficient for eliciting cross-reactive responses without off-target effects induced by the nanoparticle scaffold. This mix-and-match immunogen design strategy is a robust platform for eliciting responses to multiple influenza subtypes via a single immunogen, and a potential platform for other viral glycoproteins.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
