<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>immunology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>immunology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Antigenicity and infectivity characterization of SARS-CoV-2 BA.2.86
Authors: Yang, S.; Yu, Y.; Jian, F.; Song, W.; Yisimayi, A.; Chen, X.; Xu, Y.; Wang, P.; Yu, L.; Wang, J.; Niu, X.; Wang, J.; Xiao, T.; An, R.; Wang, Y.; Gu, Q.; Shao, F.; Jin, R.; Shen, Z.; Wang, Y.; Cao, Y. R.
Score: 549.1, Published: 2023-09-04 DOI: 10.1101/2023.09.01.555815
The recently identified SARS-CoV-2 variant, BA.2.86, which carries a substantial number of Spike mutations, has raised a global alarm.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/immunology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="immunology" />
<meta property="og:description" content="Antigenicity and infectivity characterization of SARS-CoV-2 BA.2.86
Authors: Yang, S.; Yu, Y.; Jian, F.; Song, W.; Yisimayi, A.; Chen, X.; Xu, Y.; Wang, P.; Yu, L.; Wang, J.; Niu, X.; Wang, J.; Xiao, T.; An, R.; Wang, Y.; Gu, Q.; Shao, F.; Jin, R.; Shen, Z.; Wang, Y.; Cao, Y. R.
Score: 549.1, Published: 2023-09-04 DOI: 10.1101/2023.09.01.555815
The recently identified SARS-CoV-2 variant, BA.2.86, which carries a substantial number of Spike mutations, has raised a global alarm." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/immunology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-13T10:39:03+00:00" />
<meta property="article:modified_time" content="2023-09-13T10:39:03+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="immunology"/>
<meta name="twitter:description" content="Antigenicity and infectivity characterization of SARS-CoV-2 BA.2.86
Authors: Yang, S.; Yu, Y.; Jian, F.; Song, W.; Yisimayi, A.; Chen, X.; Xu, Y.; Wang, P.; Yu, L.; Wang, J.; Niu, X.; Wang, J.; Xiao, T.; An, R.; Wang, Y.; Gu, Q.; Shao, F.; Jin, R.; Shen, Z.; Wang, Y.; Cao, Y. R.
Score: 549.1, Published: 2023-09-04 DOI: 10.1101/2023.09.01.555815
The recently identified SARS-CoV-2 variant, BA.2.86, which carries a substantial number of Spike mutations, has raised a global alarm."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "immunology",
      "item": "https://trxiv.yorks0n.com/posts/immunology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "immunology",
  "name": "immunology",
  "description": "Antigenicity and infectivity characterization of SARS-CoV-2 BA.2.86\nAuthors: Yang, S.; Yu, Y.; Jian, F.; Song, W.; Yisimayi, A.; Chen, X.; Xu, Y.; Wang, P.; Yu, L.; Wang, J.; Niu, X.; Wang, J.; Xiao, T.; An, R.; Wang, Y.; Gu, Q.; Shao, F.; Jin, R.; Shen, Z.; Wang, Y.; Cao, Y. R.\nScore: 549.1, Published: 2023-09-04 DOI: 10.1101/2023.09.01.555815\nThe recently identified SARS-CoV-2 variant, BA.2.86, which carries a substantial number of Spike mutations, has raised a global alarm.",
  "keywords": [
    
  ],
  "articleBody": " Antigenicity and infectivity characterization of SARS-CoV-2 BA.2.86\nAuthors: Yang, S.; Yu, Y.; Jian, F.; Song, W.; Yisimayi, A.; Chen, X.; Xu, Y.; Wang, P.; Yu, L.; Wang, J.; Niu, X.; Wang, J.; Xiao, T.; An, R.; Wang, Y.; Gu, Q.; Shao, F.; Jin, R.; Shen, Z.; Wang, Y.; Cao, Y. R.\nScore: 549.1, Published: 2023-09-04 DOI: 10.1101/2023.09.01.555815\nThe recently identified SARS-CoV-2 variant, BA.2.86, which carries a substantial number of Spike mutations, has raised a global alarm. An immediate assessment of its antigenic properties and infectivity is necessary. Here, we reveal the distinct antigenicity of BA.2.86 compared with previous variants including XBB.1.5. BA.2.86 significantly evades convalescent plasma from XBB breakthrough infection (BTI) and reinfections. Key mutations that mediate the enhanced resistance include N450D, K356T, L452W, A484K, V483del, and V445H on the RBD, while BA.2.86s NTD mutations and E554K on SD1 also largely contribute. However, we found that BA.2.86 pseudovirus exhibits compromised efficiency of infecting HEK293T-hACE2 cells compared to XBB.1.5 and EG.5, which may be caused by K356T, V483del, and E554K, and could potentially limit BA.2.86s transmissibility. In sum, it appears that BA.2.86 has traded its infectivity for higher immune evasion during long-term host-viral evolution. Close attention should be paid to monitoring additional mutations that could improve BA.2.86s infectivity.\nSensitivity of BA.2.86 to prevailing neutralising antibody responses\nAuthors: Sheward, D. J.; Yang, Y.; Westerberg, M.; OÌˆling, S.; Muschiol, S.; Sato, K.; Peacock, T. P.; Karlsson Hedestam, G. B.; Albert, J.; Murrell, B.\nScore: 791.8, Published: 2023-09-04 DOI: 10.1101/2023.09.02.556033\nA new SARS-CoV-2 variant, designated BA.2.86, has recently emerged with over 30 spike mutations relative to its parental BA.2, raising questions about its degree of resistance to neutralising antibodies. Using a spike-pseudotyped virus model we characterise neutralisation of BA.2.86 by clinically relevant monoclonal antibodies and by two cohorts of serum sampled from Stockholm, including both a recent cohort, and one sampled prior to the arrival of XBB in Sweden.\nXBB.1.5 Spike Protein COVID-19 Vaccine Induces Broadly Neutralizing and Cellular Immune Responses Against EG.5.1 and Emerging XBB Variants\nAuthors: Patel, N.; Trost, J. F.; Guebre-Xabier, M.; Zhou, H.; Norton, J.; Jiang, D.; Cai, Z.; Zhu, M.; Marchese, A. M.; Greene, A. M.; Mallory, R. M.; Kalkeri, R.; Dubovsky, F.; Smith, G.\nScore: 487.3, Published: 2023-08-31 DOI: 10.1101/2023.08.30.554497\nMonovalent SARS-CoV-2 Prototype (Wuhan-Hu-1) and bivalent (Prototype + BA.4/5) COVID-19 vaccines have demonstrated a waning of vaccine-mediated immunity highlighted by lower neutralizing antibody responses against SARS-CoV-2 Omicron XBB sub-variants. The reduction of humoral immunity due to the rapid evolution of SARS-CoV-2 has signaled the need for an update to vaccine composition. A strain change for all authorized/approved vaccines to a monovalent composition with Omicron subvariant XBB.1.5 has been supported by the WHO, EMA, and FDA. Here, we demonstrate that immunization with a monovalent recombinant spike protein COVID-19 vaccine (Novavax, Inc.) based on the subvariant XBB.1.5 induces cross-neutralizing antibodies against XBB.1.5, XBB.1.16, XBB.2.3, EG.5.1, and XBB.1.16.6 subvariants, promotes higher pseudovirus neutralizing antibody titers than bivalent (Prototype + XBB.1.5) vaccine, induces SARS-CoV-2 spike-specific Th1-biased CD4+ T-cell responses against XBB subvariants, and robustly boosts antibody responses in mice and nonhuman primates primed with a variety of monovalent and bivalent vaccines. Together, these data support updating the Novavax vaccine to a monovalent XBB.1.5 formulation for the 2023-2024 COVID-19 vaccination campaign.\nAltered DNA methylation underlies monocyte dysregulation and innate exhaustion memory in sepsis\nAuthors: Caldwell, B. A.; Wu, Y.; Wang, J.; Li, L.\nScore: 71.1, Published: 2023-08-31 DOI: 10.1101/2023.08.30.555580\nInnate immune memory is the process by which pathogen exposure elicits cell-intrinsic states to alter the strength of future immune challenges. Such altered memory states drive monocyte dysregulation during sepsis, promoting pathogenic behavior characterized by pro-inflammatory, immunosuppressive gene expression in concert with emergency hematopoiesis. Epigenetic changes, notably in the form of histone modifications, have been shown to underlie innate immune memory, but the contribution of DNA methylation to this process remains poorly understood. Using an ex vivo sepsis model, we discovered broad changes in DNA methylation throughout the genome of exhausted monocytes, including at several genes previously implicated as major drivers of immune dysregulation during sepsis and Covid-19 infection (e.g. Plac8). Methylome alterations are driven in part by Wnt signaling inhibition in exhausted monocytes, and can be reversed through treatment with DNA methyltransferase inhibitors, Wnt agonists, or immune training molecules. Importantly, these changes are recapitulated in septic mice following cecal slurry injection, resulting in stable changes at critical immune genes that support the involvement of DNA methylation in acute and long-term monocyte dysregulation during sepsis.\nThe molecular reach of antibodies determines their SARS-CoV-2 neutralisation potency\nAuthors: Huhn, A.; Nissley, D. A.; Wilson, D. B.; Kutuzov, M. A.; Donat, R.; Tan, T. K.; Zhang, Y.; Barton, M. I.; Liu, C.; Dejnirattisai, W.; Supasa, P.; Mongkolsapaya, J.; Townsend, A.; James, W. S.; Screaton, G.; van der Merwe, P. A.; Deane, C.; Isaacson, S.; Dushek, O.\nScore: 21.9, Published: 2023-09-07 DOI: 10.1101/2023.09.06.556503\nAntibodies play crucial roles in health and disease and are invaluable tools for diagnostics, research, and therapy. Although antibodies bind bivalently, we lack methods to analyse bivalent binding. Here, we introduce a particle-based model and use it to analyse bivalent binding of SARS-CoV-2 RBD-specific antibodies in surface plasmon resonance assays. The method reproduces the monovalent on/off-rates and enables measurements of new parameters, including the molecular reach, which is the maximum antigen separation that supports bivalent binding. We show that the molecular reach (22-46 nm) exceeds the physical size of an antibody (15 nm) and that the variation in reach across 45 patient-isolated antibodies is the best correlate of viral neutralisation. Using the complete set of fitted parameters, the model predicts an emergent antibody binding potency that equals the neutralisation potency. This novel analytical method should improve our understanding and exploitation of antibodies and other bivalent molecules. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=154 SRC=\"FIGDIR/small/556503v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (36K): org.highwire.dtl.DTLVardef@155db69org.highwire.dtl.DTLVardef@1b7d04forg.highwire.dtl.DTLVardef@cd0c09org.highwire.dtl.DTLVardef@1d1309e_HPS_FORMAT_FIGEXP M_FIG C_FIG\nConvergent Sequence Features of Antiviral B Cells\nAuthors: Abu-Shmais, A. A.; Vukovich, M. J.; Wasdin, P. T.; Suresh, Y. P.; Rush, S. A.; Gillespie, R. A.; Sankhala, R. S.; Choe, M.; Joyce, M. G.; Kanekiyo, M.; McLellan, J. S.; Georgiev, I. S.\nScore: 17.1, Published: 2023-09-07 DOI: 10.1101/2023.09.06.556442\nThroughout life, humans experience repeated exposure to viral antigens through infection and vaccination, building diverse antigen-specific antibody repertoires. In recent years, these repertoires have become an important source for novel antibody-based antiviral therapeutics, yet there is still limited understanding of the determinants of antibody-antigen specificity. Here, we generated a large dataset mapping antibody sequence to antigen specificity for thousands of B cells, by screening the repertoires of a set of healthy individuals against twenty viral antigens representing diverse pathogens of biomedical significance. Analysis revealed antigen-specific patterns in variable gene usage, gene pairing, and somatic hypermutation, as well as the presence of convergent antiviral signatures across multiple individuals. These results help define the characteristics of human antibody repertoires simultaneously against an unprecedented number and diversity of viral targets. Understanding the fundamental rules of antibody-antigen interactions can lead to transformative new approaches for the development of antibody therapeutics and vaccines against current and emerging viruses.\nAllergen-induced airway matrix remodelling in mice can be prevented or reversed by targeting chitinase-like proteins.\nAuthors: Parkinson, J. E.; Adamson, A. D.; MacDonald, A. S.; Allen, J. E.; Sutherland, T. E.\nScore: 14.0, Published: 2023-08-21 DOI: 10.1101/2023.08.18.553857\nChitinase-like proteins (CLPs) are biomarkers of inflammation and airway remodelling in asthma, yet their direct contribution towards disease pathogenesis is unknown. Using a mouse model of allergen-induced type 2/type 17 airway inflammation we sought to directly investigate the role of the murine CLPs Ym1 and Ym2 during chronic lung pathology. Data demonstrated distinct chronic inflammatory roles for Ym2, IL-13, and IL-17a signalling pathways. Notably, only CLPs were key for initiating the pathogenic accumulation and re-organisation of the pulmonary extracellular matrix (ECM) environment. Furthermore, inhibition of CLPs after chronic pathology developed, reversed airway remodelling independently of chronic inflammation. These studies disentangle chronic IL-13 and IL-17a signalling from the development of allergic airway remodelling and instead highlight a central role for CLPs, which provides new avenues to therapeutically target aberrant ECM accumulation.\nThe Structures of Secretory IgA in complex with Streptococcus pyogenes M4 and human CD89 provide insights on mucosal host-pathogen interactions\nAuthors: Liu, Q.; Stadtmueller, B.\nScore: 21.9, Published: 2023-08-26 DOI: 10.1101/2023.04.21.537878\nImmunoglobulin (Ig) A functions as monomeric IgA in the serum and Secretory (S) IgA in mucosal secretions. Host IgA Fc receptors (FcRs), including human FcR1/CD89, mediate IgA effector functions; however human pathogen Streptococcus pyogenes has evolved surface-protein virulence factors, including M4, that also engage the CD89 binding site on IgA. Despite human mucosa serving as a reservoir for pathogens, SIgA interactions with CD89 and M4 remain poorly understood. Here we report cryo-EM structures of M4-SIgA and CD89-SIgA complexes, which unexpectedly reveal different SIgA-binding stoichiometry for M4 and CD89. Structural data, supporting experiments, and modeling indicate that copies of SIgA bound to S. pyogenes M4 will adopt similar orientations on the bacterium surface and leave one host FcR binding site open. Results suggest unappreciated functional consequences associated with SIgA binding to host and bacterial FcRs relevant to understanding host-microbe co-evolution, IgA effector functions and to improving the outcomes of group A Streptococcus infection.\nMapping human natural killer cell development in pediatric tonsil by imaging mass cytometry \u0026 high-resolution microscopy\nAuthors: Hegewisch-Solloa, E.; Melsen, J. E.; Ravichandran, H.; Rendeiro, A.; Mundy-Bosse, B.; Freud, A. G.; Melms, J. C.; Eisman, S. E.; Izar, B.; Grunstein, E.; Connors, T. J.; Elemento, O.; Horowitz, A.; Mace, E. M.\nScore: 12.0, Published: 2023-09-06 DOI: 10.1101/2023.09.05.556371\nNatural killer (NK) cells develop from CD34+ progenitors in a stage-specific manner defined by changes in cell surface receptor expression and function. Secondary lymphoid tissues, including tonsil, are sites of human NK cell development. Here we present new insights into human NK cell development in pediatric tonsil using cyclic immunofluorescence and imaging mass cytometry. We show that NK cell subset localization and interactions are dependent on NK cell developmental stage and tissue residency. NK cell progenitors are found in the interfollicular domain in proximity to cytokine-expressing stromal cells that promote proliferation and maturation. Mature NK cells are primarily found in the T-cell rich parafollicular domain engaging in cell-cell interactions that differ depending on their stage and tissue residency. The presence of local inflammation results in changes in NK cell interactions, abundance, and localization. This study provides the first comprehensive atlas of human NK cell development in secondary lymphoid tissue.\nConcurrent administration of COVID-19 and influenza vaccines enhances Spike-specific antibody responses\nAuthors: Barouch, S. E.; Chicz, T. M.; Blanc, R.; Barbati, D. R.; Parker, L. J.; Tong, X.; McNamara, R. P.\nScore: 11.9, Published: 2023-09-12 DOI: 10.1101/2023.09.12.557347\nThe bivalent COVID-19 mRNA boosters became available in fall 2022 and were recommended alongside the seasonal influenza vaccine. However, the immunogenicity of concurrent versus separate administration of these vaccines remains unclear. Here, we analyzed antibody responses in healthcare workers who received the bivalent COVID-19 booster and the influenza vaccine on the same day or different days. IgG1 responses to SARS-CoV-2 Spike were higher at peak immunogenicity and 6 months following concurrent administration compared with separate administration of the COVID-19 and influenza vaccines. These data suggest that concurrent administration of these vaccines may yield higher and more durable SARS-CoV-2 antibody responses.\n",
  "wordCount" : "1861",
  "inLanguage": "en",
  "datePublished": "2023-09-13T10:39:03Z",
  "dateModified": "2023-09-13T10:39:03Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/immunology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      immunology
    </h1>
    <div class="post-meta"><span>updated on September 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555815">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555815" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555815">
        <p class="paperTitle">Antigenicity and infectivity characterization of SARS-CoV-2 BA.2.86</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555815" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555815" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, S.; Yu, Y.; Jian, F.; Song, W.; Yisimayi, A.; Chen, X.; Xu, Y.; Wang, P.; Yu, L.; Wang, J.; Niu, X.; Wang, J.; Xiao, T.; An, R.; Wang, Y.; Gu, Q.; Shao, F.; Jin, R.; Shen, Z.; Wang, Y.; Cao, Y. R.</p>
        <p class="info">Score: 549.1, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555815' target='https://doi.org/10.1101/2023.09.01.555815'> 10.1101/2023.09.01.555815</a></p>
        <p class="abstract">The recently identified SARS-CoV-2 variant, BA.2.86, which carries a substantial number of Spike mutations, has raised a global alarm. An immediate assessment of its antigenic properties and infectivity is necessary. Here, we reveal the distinct antigenicity of BA.2.86 compared with previous variants including XBB.1.5. BA.2.86 significantly evades convalescent plasma from XBB breakthrough infection (BTI) and reinfections. Key mutations that mediate the enhanced resistance include N450D, K356T, L452W, A484K, V483del, and V445H on the RBD, while BA.2.86s NTD mutations and E554K on SD1 also largely contribute. However, we found that BA.2.86 pseudovirus exhibits compromised efficiency of infecting HEK293T-hACE2 cells compared to XBB.1.5 and EG.5, which may be caused by K356T, V483del, and E554K, and could potentially limit BA.2.86s transmissibility. In sum, it appears that BA.2.86 has traded its infectivity for higher immune evasion during long-term host-viral evolution. Close attention should be paid to monitoring additional mutations that could improve BA.2.86s infectivity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.02.556033">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.02.556033" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.02.556033">
        <p class="paperTitle">Sensitivity of BA.2.86 to prevailing neutralising antibody responses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.02.556033" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.02.556033" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sheward, D. J.; Yang, Y.; Westerberg, M.; OÌˆling, S.; Muschiol, S.; Sato, K.; Peacock, T. P.; Karlsson Hedestam, G. B.; Albert, J.; Murrell, B.</p>
        <p class="info">Score: 791.8, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.02.556033' target='https://doi.org/10.1101/2023.09.02.556033'> 10.1101/2023.09.02.556033</a></p>
        <p class="abstract">A new SARS-CoV-2 variant, designated BA.2.86, has recently emerged with over 30 spike mutations relative to its parental BA.2, raising questions about its degree of resistance to neutralising antibodies. Using a spike-pseudotyped virus model we characterise neutralisation of BA.2.86 by clinically relevant monoclonal antibodies and by two cohorts of serum sampled from Stockholm, including both a recent cohort, and one sampled prior to the arrival of XBB in Sweden.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.554497">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.554497" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.554497">
        <p class="paperTitle">XBB.1.5 Spike Protein COVID-19 Vaccine Induces Broadly Neutralizing and Cellular Immune Responses Against EG.5.1 and Emerging XBB Variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.554497" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.554497" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Patel, N.; Trost, J. F.; Guebre-Xabier, M.; Zhou, H.; Norton, J.; Jiang, D.; Cai, Z.; Zhu, M.; Marchese, A. M.; Greene, A. M.; Mallory, R. M.; Kalkeri, R.; Dubovsky, F.; Smith, G.</p>
        <p class="info">Score: 487.3, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.554497' target='https://doi.org/10.1101/2023.08.30.554497'> 10.1101/2023.08.30.554497</a></p>
        <p class="abstract">Monovalent SARS-CoV-2 Prototype (Wuhan-Hu-1) and bivalent (Prototype &#43; BA.4/5) COVID-19 vaccines have demonstrated a waning of vaccine-mediated immunity highlighted by lower neutralizing antibody responses against SARS-CoV-2 Omicron XBB sub-variants. The reduction of humoral immunity due to the rapid evolution of SARS-CoV-2 has signaled the need for an update to vaccine composition. A strain change for all authorized/approved vaccines to a monovalent composition with Omicron subvariant XBB.1.5 has been supported by the WHO, EMA, and FDA. Here, we demonstrate that immunization with a monovalent recombinant spike protein COVID-19 vaccine (Novavax, Inc.) based on the subvariant XBB.1.5 induces cross-neutralizing antibodies against XBB.1.5, XBB.1.16, XBB.2.3, EG.5.1, and XBB.1.16.6 subvariants, promotes higher pseudovirus neutralizing antibody titers than bivalent (Prototype &#43; XBB.1.5) vaccine, induces SARS-CoV-2 spike-specific Th1-biased CD4&#43; T-cell responses against XBB subvariants, and robustly boosts antibody responses in mice and nonhuman primates primed with a variety of monovalent and bivalent vaccines. Together, these data support updating the Novavax vaccine to a monovalent XBB.1.5 formulation for the 2023-2024 COVID-19 vaccination campaign.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555580">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555580" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555580">
        <p class="paperTitle">Altered DNA methylation underlies monocyte dysregulation and innate exhaustion memory in sepsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555580" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555580" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Caldwell, B. A.; Wu, Y.; Wang, J.; Li, L.</p>
        <p class="info">Score: 71.1, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555580' target='https://doi.org/10.1101/2023.08.30.555580'> 10.1101/2023.08.30.555580</a></p>
        <p class="abstract">Innate immune memory is the process by which pathogen exposure elicits cell-intrinsic states to alter the strength of future immune challenges. Such altered memory states drive monocyte dysregulation during sepsis, promoting pathogenic behavior characterized by pro-inflammatory, immunosuppressive gene expression in concert with emergency hematopoiesis. Epigenetic changes, notably in the form of histone modifications, have been shown to underlie innate immune memory, but the contribution of DNA methylation to this process remains poorly understood. Using an ex vivo sepsis model, we discovered broad changes in DNA methylation throughout the genome of exhausted monocytes, including at several genes previously implicated as major drivers of immune dysregulation during sepsis and Covid-19 infection (e.g. Plac8). Methylome alterations are driven in part by Wnt signaling inhibition in exhausted monocytes, and can be reversed through treatment with DNA methyltransferase inhibitors, Wnt agonists, or immune training molecules. Importantly, these changes are recapitulated in septic mice following cecal slurry injection, resulting in stable changes at critical immune genes that support the involvement of DNA methylation in acute and long-term monocyte dysregulation during sepsis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556503">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556503" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556503">
        <p class="paperTitle">The molecular reach of antibodies determines their SARS-CoV-2 neutralisation potency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556503" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556503" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huhn, A.; Nissley, D. A.; Wilson, D. B.; Kutuzov, M. A.; Donat, R.; Tan, T. K.; Zhang, Y.; Barton, M. I.; Liu, C.; Dejnirattisai, W.; Supasa, P.; Mongkolsapaya, J.; Townsend, A.; James, W. S.; Screaton, G.; van der Merwe, P. A.; Deane, C.; Isaacson, S.; Dushek, O.</p>
        <p class="info">Score: 21.9, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556503' target='https://doi.org/10.1101/2023.09.06.556503'> 10.1101/2023.09.06.556503</a></p>
        <p class="abstract">Antibodies play crucial roles in health and disease and are invaluable tools for diagnostics, research, and therapy. Although antibodies bind bivalently, we lack methods to analyse bivalent binding. Here, we introduce a particle-based model and use it to analyse bivalent binding of SARS-CoV-2 RBD-specific antibodies in surface plasmon resonance assays. The method reproduces the monovalent on/off-rates and enables measurements of new parameters, including the molecular reach, which is the maximum antigen separation that supports bivalent binding. We show that the molecular reach (22-46 nm) exceeds the physical size of an antibody (15 nm) and that the variation in reach across 45 patient-isolated antibodies is the best correlate of viral neutralisation. Using the complete set of fitted parameters, the model predicts an emergent antibody binding potency that equals the neutralisation potency. This novel analytical method should improve our understanding and exploitation of antibodies and other bivalent molecules.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=154 SRC=&#34;FIGDIR/small/556503v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (36K):
org.highwire.dtl.DTLVardef@155db69org.highwire.dtl.DTLVardef@1b7d04forg.highwire.dtl.DTLVardef@cd0c09org.highwire.dtl.DTLVardef@1d1309e_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556442">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556442" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556442">
        <p class="paperTitle">Convergent Sequence Features of Antiviral B Cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556442" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556442" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abu-Shmais, A. A.; Vukovich, M. J.; Wasdin, P. T.; Suresh, Y. P.; Rush, S. A.; Gillespie, R. A.; Sankhala, R. S.; Choe, M.; Joyce, M. G.; Kanekiyo, M.; McLellan, J. S.; Georgiev, I. S.</p>
        <p class="info">Score: 17.1, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556442' target='https://doi.org/10.1101/2023.09.06.556442'> 10.1101/2023.09.06.556442</a></p>
        <p class="abstract">Throughout life, humans experience repeated exposure to viral antigens through infection and vaccination, building diverse antigen-specific antibody repertoires. In recent years, these repertoires have become an important source for novel antibody-based antiviral therapeutics, yet there is still limited understanding of the determinants of antibody-antigen specificity. Here, we generated a large dataset mapping antibody sequence to antigen specificity for thousands of B cells, by screening the repertoires of a set of healthy individuals against twenty viral antigens representing diverse pathogens of biomedical significance. Analysis revealed antigen-specific patterns in variable gene usage, gene pairing, and somatic hypermutation, as well as the presence of convergent antiviral signatures across multiple individuals. These results help define the characteristics of human antibody repertoires simultaneously against an unprecedented number and diversity of viral targets. Understanding the fundamental rules of antibody-antigen interactions can lead to transformative new approaches for the development of antibody therapeutics and vaccines against current and emerging viruses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553857">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553857" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553857">
        <p class="paperTitle">Allergen-induced airway matrix remodelling in mice can be prevented or reversed by targeting chitinase-like proteins.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553857" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553857" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parkinson, J. E.; Adamson, A. D.; MacDonald, A. S.; Allen, J. E.; Sutherland, T. E.</p>
        <p class="info">Score: 14.0, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553857' target='https://doi.org/10.1101/2023.08.18.553857'> 10.1101/2023.08.18.553857</a></p>
        <p class="abstract">Chitinase-like proteins (CLPs) are biomarkers of inflammation and airway remodelling in asthma, yet their direct contribution towards disease pathogenesis is unknown. Using a mouse model of allergen-induced type 2/type 17 airway inflammation we sought to directly investigate the role of the murine CLPs Ym1 and Ym2 during chronic lung pathology. Data demonstrated distinct chronic inflammatory roles for Ym2, IL-13, and IL-17a signalling pathways. Notably, only CLPs were key for initiating the pathogenic accumulation and re-organisation of the pulmonary extracellular matrix (ECM) environment. Furthermore, inhibition of CLPs after chronic pathology developed, reversed airway remodelling independently of chronic inflammation. These studies disentangle chronic IL-13 and IL-17a signalling from the development of allergic airway remodelling and instead highlight a central role for CLPs, which provides new avenues to therapeutically target aberrant ECM accumulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.04.21.537878">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.04.21.537878" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.04.21.537878">
        <p class="paperTitle">The Structures of Secretory IgA in complex with Streptococcus pyogenes M4 and human CD89 provide insights on mucosal host-pathogen interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.04.21.537878" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.04.21.537878" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, Q.; Stadtmueller, B.</p>
        <p class="info">Score: 21.9, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.04.21.537878' target='https://doi.org/10.1101/2023.04.21.537878'> 10.1101/2023.04.21.537878</a></p>
        <p class="abstract">Immunoglobulin (Ig) A functions as monomeric IgA in the serum and Secretory (S) IgA in mucosal secretions. Host IgA Fc receptors (FcRs), including human FcR1/CD89, mediate IgA effector functions; however human pathogen Streptococcus pyogenes has evolved surface-protein virulence factors, including M4, that also engage the CD89 binding site on IgA. Despite human mucosa serving as a reservoir for pathogens, SIgA interactions with CD89 and M4 remain poorly understood. Here we report cryo-EM structures of M4-SIgA and CD89-SIgA complexes, which unexpectedly reveal different SIgA-binding stoichiometry for M4 and CD89. Structural data, supporting experiments, and modeling indicate that copies of SIgA bound to S. pyogenes M4 will adopt similar orientations on the bacterium surface and leave one host FcR binding site open. Results suggest unappreciated functional consequences associated with SIgA binding to host and bacterial FcRs relevant to understanding host-microbe co-evolution, IgA effector functions and to improving the outcomes of group A Streptococcus infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556371">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556371" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556371">
        <p class="paperTitle">Mapping human natural killer cell development in pediatric tonsil by imaging mass cytometry &amp; high-resolution microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556371" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556371" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hegewisch-Solloa, E.; Melsen, J. E.; Ravichandran, H.; Rendeiro, A.; Mundy-Bosse, B.; Freud, A. G.; Melms, J. C.; Eisman, S. E.; Izar, B.; Grunstein, E.; Connors, T. J.; Elemento, O.; Horowitz, A.; Mace, E. M.</p>
        <p class="info">Score: 12.0, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556371' target='https://doi.org/10.1101/2023.09.05.556371'> 10.1101/2023.09.05.556371</a></p>
        <p class="abstract">Natural killer (NK) cells develop from CD34&#43; progenitors in a stage-specific manner defined by changes in cell surface receptor expression and function. Secondary lymphoid tissues, including tonsil, are sites of human NK cell development. Here we present new insights into human NK cell development in pediatric tonsil using cyclic immunofluorescence and imaging mass cytometry. We show that NK cell subset localization and interactions are dependent on NK cell developmental stage and tissue residency. NK cell progenitors are found in the interfollicular domain in proximity to cytokine-expressing stromal cells that promote proliferation and maturation. Mature NK cells are primarily found in the T-cell rich parafollicular domain engaging in cell-cell interactions that differ depending on their stage and tissue residency. The presence of local inflammation results in changes in NK cell interactions, abundance, and localization. This study provides the first comprehensive atlas of human NK cell development in secondary lymphoid tissue.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557347">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557347" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557347">
        <p class="paperTitle">Concurrent administration of COVID-19 and influenza vaccines enhances Spike-specific antibody responses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557347" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557347" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Barouch, S. E.; Chicz, T. M.; Blanc, R.; Barbati, D. R.; Parker, L. J.; Tong, X.; McNamara, R. P.</p>
        <p class="info">Score: 11.9, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557347' target='https://doi.org/10.1101/2023.09.12.557347'> 10.1101/2023.09.12.557347</a></p>
        <p class="abstract">The bivalent COVID-19 mRNA boosters became available in fall 2022 and were recommended alongside the seasonal influenza vaccine. However, the immunogenicity of concurrent versus separate administration of these vaccines remains unclear. Here, we analyzed antibody responses in healthcare workers who received the bivalent COVID-19 booster and the influenza vaccine on the same day or different days. IgG1 responses to SARS-CoV-2 Spike were higher at peak immunogenicity and 6 months following concurrent administration compared with separate administration of the COVID-19 and influenza vaccines. These data suggest that concurrent administration of these vaccines may yield higher and more durable SARS-CoV-2 antibody responses.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
