<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>immunology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>immunology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Immune Correlates of Hyperglycemia and Vaccination in a Non-human Primate Model of Long-COVID
Authors: Palmer, C. S.; Perdios, C.; Abdel-Mohsen, M.; Mudd, J.; Datta, P.; Maness, N.; Lehmicke, G.; Golden, N.; Hellmer, L.; Coyne, C.; Moore Green, K.; Midkiff, C.; Williams, K.; Tiburcio, R.; Fahlberg, M.; Boykin, K.; Kenway, C.; Russell-Lodrigue, K.; Bohm, R.; Blair, R. V.; Dufour, J.; Fischer, T.; Saied, A.; Rappaport, J.
Score: 36.3, Published: 2023-09-25 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/immunology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="immunology" />
<meta property="og:description" content="Immune Correlates of Hyperglycemia and Vaccination in a Non-human Primate Model of Long-COVID
Authors: Palmer, C. S.; Perdios, C.; Abdel-Mohsen, M.; Mudd, J.; Datta, P.; Maness, N.; Lehmicke, G.; Golden, N.; Hellmer, L.; Coyne, C.; Moore Green, K.; Midkiff, C.; Williams, K.; Tiburcio, R.; Fahlberg, M.; Boykin, K.; Kenway, C.; Russell-Lodrigue, K.; Bohm, R.; Blair, R. V.; Dufour, J.; Fischer, T.; Saied, A.; Rappaport, J.
Score: 36.3, Published: 2023-09-25 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/immunology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-27T10:39:11+00:00" />
<meta property="article:modified_time" content="2023-09-27T10:39:11+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="immunology"/>
<meta name="twitter:description" content="Immune Correlates of Hyperglycemia and Vaccination in a Non-human Primate Model of Long-COVID
Authors: Palmer, C. S.; Perdios, C.; Abdel-Mohsen, M.; Mudd, J.; Datta, P.; Maness, N.; Lehmicke, G.; Golden, N.; Hellmer, L.; Coyne, C.; Moore Green, K.; Midkiff, C.; Williams, K.; Tiburcio, R.; Fahlberg, M.; Boykin, K.; Kenway, C.; Russell-Lodrigue, K.; Bohm, R.; Blair, R. V.; Dufour, J.; Fischer, T.; Saied, A.; Rappaport, J.
Score: 36.3, Published: 2023-09-25 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "immunology",
      "item": "https://trxiv.yorks0n.com/posts/immunology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "immunology",
  "name": "immunology",
  "description": "Immune Correlates of Hyperglycemia and Vaccination in a Non-human Primate Model of Long-COVID\nAuthors: Palmer, C. S.; Perdios, C.; Abdel-Mohsen, M.; Mudd, J.; Datta, P.; Maness, N.; Lehmicke, G.; Golden, N.; Hellmer, L.; Coyne, C.; Moore Green, K.; Midkiff, C.; Williams, K.; Tiburcio, R.; Fahlberg, M.; Boykin, K.; Kenway, C.; Russell-Lodrigue, K.; Bohm, R.; Blair, R. V.; Dufour, J.; Fischer, T.; Saied, A.; Rappaport, J.\nScore: 36.3, Published: 2023-09-25 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " Immune Correlates of Hyperglycemia and Vaccination in a Non-human Primate Model of Long-COVID\nAuthors: Palmer, C. S.; Perdios, C.; Abdel-Mohsen, M.; Mudd, J.; Datta, P.; Maness, N.; Lehmicke, G.; Golden, N.; Hellmer, L.; Coyne, C.; Moore Green, K.; Midkiff, C.; Williams, K.; Tiburcio, R.; Fahlberg, M.; Boykin, K.; Kenway, C.; Russell-Lodrigue, K.; Bohm, R.; Blair, R. V.; Dufour, J.; Fischer, T.; Saied, A.; Rappaport, J.\nScore: 36.3, Published: 2023-09-25 DOI: 10.1101/2023.09.22.559019\nHyperglycemia, and exacerbation of pre-existing deficits in glucose metabolism, are major manifestations of the post-acute sequelae of SARS-CoV-2 (PASC). Our understanding of lasting glucometabolic disruptions after acute COVID-19 remains unclear due to the lack of animal models for metabolic PASC. Here, we report a non-human primate model of metabolic PASC using SARS-CoV-2 infected African green monkeys (AGMs). Using this model, we have identified a dysregulated chemokine signature and hypersensitive T cell population during acute COVID-19 that correlates with elevated and persistent hyperglycemia four months post-infection. This persistent hyperglycemia correlates with elevated hepatic glycogen, but there was no evidence of long-term SARS-CoV-2 replication in the liver and pancreas. Finally, we report a favorable glycemic effect of the SARS-CoV-2 mRNA vaccine, administered on day 4 post-infection. Together, these data suggest that the AGM metabolic PASC model exhibits important similarities to human metabolic PASC and can be utilized to assess therapeutic candidates to combat this syndrome.\nTAPIR: a T-cell receptor language model for predicting rare and novel targets\nAuthors: Fast, E.; Dhar, M.; Chen, B.\nScore: 48.7, Published: 2023-09-15 DOI: 10.1101/2023.09.12.557285\nT-cell receptors (TCRs) are involved in most human diseases, but linking their sequences with their targets remains an unsolved grand challenge in the field. In this study, we present TAPIR (T-cell receptor and Peptide Interaction Recognizer), a T-cell receptor (TCR) language model that predicts TCR-target interactions, with a focus on novel and rare targets. TAPIR employs deep convolutional neural network (CNN) encoders to process TCR and target sequences across flexible representations (e.g., beta-chain only, unknown MHC allele, etc.) and learns patterns of interactivity via several training tasks. This flexibility allows TAPIR to train on more than 50k either paired (alpha and beta chain) or unpaired TCRs (just alpha or beta chain) from public and proprietary databases against 1933 unique targets. TAPIR demonstrates state-of-the-art performance when predicting TCR interactivity against common benchmark targets and is the first method to demonstrate strong performance when predicting TCR interactivity against novel targets, where no examples are provided in training. TAPIR is also capable of predicting TCR interaction against MHC alleles in the absence of target information. Leveraging these capabilities, we apply TAPIR to cancer patient TCR repertoires and identify and validate a novel and potent anti-cancer T-cell receptor against a shared cancer neoantigen target (PIK3CA H1047L). We further show how TAPIR, when extended with a generative neural network, is capable of directly designing T-cell receptor sequences that interact with a target of interest.\nCellular RNA interacts with MAVS to promote antiviral signaling\nAuthors: Gokhale, N. S.; Somfleth, K.; Thompson, M. G.; Sam, R. K.; Marciniak, D. M.; Chu, L. H.; Park, M.; Dvorkin, S.; Oberst, A.; Horner, S. M.; Ong, S.-E.; Gale, M.; Savan, R.\nScore: 21.2, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559083\nImmune signaling needs to be well-regulated to promote clearance of pathogens, while preventing aberrant inflammation. Interferons (IFNs) and antiviral genes are activated by the detection of viral RNA by RIG-I-like receptors (RLRs). Signal transduction downstream of RLRs proceeds through a multi-protein complex organized around the central adaptor protein MAVS. Recent work has shown that protein complex function can be modulated by RNA molecules providing allosteric regulation or acting as molecular guides or scaffolds. Thus, we hypothesized that RNA plays a role in organizing MAVS signaling platforms. Here, we show that MAVS, through its central intrinsically disordered domain, directly interacts with the 3' untranslated regions of cellular mRNAs. Importantly, elimination of RNA by RNase treatment disrupts the MAVS signalosome, including newly identified regulators of RLR signaling, and inhibits phosphorylation of the transcription factor IRF3. This supports the hypothesis that RNA molecules scaffold proteins in the MAVS signalosome to induce IFNs. Together, this work uncovers a function for cellular RNA in promoting signaling through MAVS and highlights a generalizable principle of RNA regulatory control of cytoplasmic immune signaling complexes.\nWeakly supervised identification and generation of adaptive immune receptor sequences associated with immune disease status\nAuthors: Slabodkin, A.; Sollid, L. M.; Sandve, G. K.; Robert, P. A.; Greiff, V.\nScore: 21.0, Published: 2023-09-24 DOI: 10.1101/2023.09.24.558823\nAdaptive immune receptor (AIR) repertoires carry immune signals as sequence motif imprints of past and present encounters with antigen (immune status). Machine learning (ML)-based identification and generation of antigen-specific immune receptors is potentially immense value for public health. The ideal training data for such ML tasks would be AIR datasets, where each sequence is labeled with its cognate antigen. However, given current technological constraints, sequence-labeled datasets are scarce, contrasted by an abundance of repertoire-labeled ones - AIR repertoire datasets where only the repertoire dataset, but not the individual AIRs, are labeled. Therefore, an unmet need exists for an ML approach that enables predictive identification and generation of disease-specific novel AIR sequences using exclusively repertoire-level immune status information. To address this need, we developed AIRRTM, an end-to-end generative model using an encoder-decoder architecture and Topic Modeling (TM) that requires exclusively repertoire-labeled AIR sequencing data as input. We validated AIRRTM's capacity to identify and generate novel disease-associated receptors on several ground truth synthetic datasets of increasingly complex immune signals and experimental data. AIRRTM broadens the discovery space for immunotherapeutics by enabling the exploitation of large-scale and broadly available immune repertoire data previously deemed largely unsuitable for this task.\nThe Inflammasome Pathway is Activated by Dengue Virus Non-structural Protein 1 and is Protective During Dengue Virus Infection\nAuthors: Wong, M. P.; Juan, E. Y.; Chelluri, S. S.; Wang, P.; Pahmeier, F.; Castillo-Rojas, B.; Blanc, S. F.; Biering, S. B.; Vance, R. E.; Harris, E.\nScore: 19.8, Published: 2023-09-21 DOI: 10.1101/2023.09.21.558875\nDengue virus (DENV) is a medically important flavivirus causing an estimated 50-100 million dengue cases annually, some of whom progress to severe disease. DENV non-structural protein 1 (NS1) is secreted from infected cells and has been implicated as a major driver of dengue pathogenesis by inducing endothelial barrier dysfunction. However, less is known about how DENV NS1 interacts with immune cells and what role these interactions play. Here we report that DENV NS1 can trigger activation of inflammasomes, a family of cytosolic innate immune sensors that respond to infectious and noxious stimuli, in mouse and human macrophages. DENV NS1 induces the release of IL-1{beta} in a caspase-1 dependent manner. Additionally, we find that DENV NS1-induced inflammasome activation is independent of the NLRP3, Pyrin, and AIM2 inflammasome pathways, but requires CD14. Intriguingly, DENV NS1-induced inflammasome activation does not induce pyroptosis and rapid cell death; instead, macrophages maintain cellular viability while releasing IL-1{beta}. Lastly, we show that caspase-1/11-deficient, but not NLRP3-deficient, mice are more susceptible to lethal DENV infection. Together, these results indicate that the inflammasome pathway acts as a sensor of DENV NS1 and plays a protective role during infection.\nAntigenicity and infectivity characterization of SARS-CoV-2 BA.2.86\nAuthors: Yang, S.; Yu, Y.; Jian, F.; Song, W.; Yisimayi, A.; Chen, X.; Xu, Y.; Wang, P.; Yu, L.; Wang, J.; Niu, X.; Wang, J.; Xiao, T.; An, R.; Wang, Y.; Gu, Q.; Shao, F.; Jin, R.; Shen, Z.; Wang, Y.; Cao, Y. R.\nScore: 653.2, Published: 2023-09-04 DOI: 10.1101/2023.09.01.555815\nThe recently identified SARS-CoV-2 variant, BA.2.86, which carries a substantial number of Spike mutations, has raised a global alarm. An immediate assessment of its antigenic properties and infectivity is necessary. Here, we reveal the distinct antigenicity of BA.2.86 compared with previous variants including XBB.1.5. BA.2.86 significantly evades convalescent plasma from XBB breakthrough infection (BTI) and reinfections. Key mutations that mediate the enhanced resistance include N450D, K356T, L452W, A484K, V483del, and V445H on the RBD, while BA.2.86s NTD mutations and E554K on SD1 also largely contribute. However, we found that BA.2.86 pseudovirus exhibits compromised efficiency of infecting HEK293T-hACE2 cells compared to XBB.1.5 and EG.5, which may be caused by K356T, V483del, and E554K, and could potentially limit BA.2.86s transmissibility. In sum, it appears that BA.2.86 has traded its infectivity for higher immune evasion during long-term host-viral evolution. Close attention should be paid to monitoring additional mutations that could improve BA.2.86s infectivity.\nTrem2 Agonist Reprograms Foamy Macrophages to Promote Atherosclerotic Plaque Stability\nAuthors: Patterson, M. T.; Xu, Y.; Hillman, H.; Osinski, V.; Schrank, P. R.; Kennedy, A. E.; Zhu, A.; Tollison, S.; Shekhar, S.; Stromnes, I.; Tassi, I.; Wu, D.; Binstadt, B.; Williams, J. W.\nScore: 14.8, Published: 2023-09-23 DOI: 10.1101/2023.09.21.558810\nObjective: Trem2, a surface lipid receptor, is expressed on foamy macrophages within atherosclerotic lesions and regulates cell survival, proliferation, and anti-inflammatory responses. Studies examining the role of Trem2 in atherosclerosis have shown that deletion of Trem2 leads to impaired foamy macrophage lipid uptake, proliferation, survival, and cholesterol efflux. Thus, we tested the hypothesis that administration of a validated Trem2 agonist antibody (AL002a) to atherogenic mice could drive macrophage survival and decrease necrotic core formation to improve plaque stability. Approach and Results: To model a therapeutic intervention approach, atherosclerosis-prone mice (Ldlr-/-) were fed a high fat diet (HFD) for 8 weeks, then transitioned to treatment with AL002a or isotype control for an additional 8 weeks while continuing on an HFD. AL002a-treated mice had increased lesion size in both the aortic sinus and whole mount aorta, which correlated with an expansion of plaque macrophage area. This expansion was due to increased macrophage survival and proliferation in plaques. Importantly, plaques from AL002a-treated mice showed improved features of plaque stability, including smaller necrotic cores, increased fibrous caps, and greater collagen deposition. Single cell RNA sequencing of whole aorta suspensions from isotype and AL002a-treated atherosclerotic mice revealed that Trem2 agonism dramatically altered foamy macrophage transcriptome. This included upregulation of oxidative phosphorylation and increased expression of collagen genes. In vitro studies validated that Trem2-agonism with AL002a promoted foamy macrophage oxLDL uptake, survival, and cholesterol efflux in culture. Conclusions: Trem2 agonist expands plaque macrophages by promoting cell survival and proliferation but improves features of plaque stability by rewiring foamy macrophage function to enhance collagen deposition.\nB cells occupy a unique, dynamic, biological niche within the human myocardium\nAuthors: Bermea, K. C.; Duque, C.; Cohen, C. D.; Bhalodia, A.; Rousseau, S.; Lovell, J.; Zita, M. D.; Mugnier, M. C.; Adamo, L.\nScore: 10.1, Published: 2023-09-23 DOI: 10.1101/2023.09.21.558902\nBackgroundA growing body of evidence from animal models indicates that the myocardium hosts a population of B cells that play a role in the development of cardiomyopathy. However, there is minimal data on human myocardial B cells and their biological niche within the heart remains mostly unexplored. MethodsWe integrated single-cell and single-nuclei datasets from 45 healthy human hearts, 70 hearts with dilated cardiomyopathy (DCM), and 8 hearts with Arrhythmogenic Right Ventricular Cardiomyopathy (ARVC). Interactions between B cells and other cell types were investigated using the CellChat Package. Differential gene expression analysis comparing B cells across conditions was performed using DESeq2. Pathway analysis was performed using Ingenuity, KEGG, and GO pathways analysis. ResultsOut of 1,200,752 nuclei and 49,723 cells, we identified 1,100 B cells, including naive B cells and plasma cells. B cells showed an extensive network of interactions within the healthy myocardium that included outgoing signaling to macrophages, T cells, endothelial cells, and pericytes, and incoming signaling from endothelial cells, pericytes, and fibroblasts. This biological niche relied on both ECM-receptor interactions, cell-cell contact interactions, and paracrine interaction, it changed significantly in the context of cardiomyopathy and had disease-specific features. Differential gene expression analysis showed that in the context of DCM both naive and plasma myocardial B cells upregulated several pathways related to immune activation, including upregulation of oxidative phosphorylation, upregulation of leukocyte extravasation and, in naive B cells, antigen processing and presentation. ConclusionsThe human healthy and diseased myocardium contains naive B cells and plasma cells, integrated in a diverse and dynamic biological niche. Naive myocardial-associated B cells likely contribute to the pathogenesis of human dilated cardiomyopathy.\ntidytcells: standardizer for TCR/MHC nomenclature\nAuthors: Nagano, Y.; Chain, B.\nScore: 7.3, Published: 2023-09-23 DOI: 10.1101/2023.09.21.558778\nT cell receptors (TCRs) underpin the diversity and specificity of T cell activity. As such, TCR repertoire data is valuable both as an adaptive immune biomarker, and as a way to identify candidate therapeutic TCRs. Analysis of TCR repertoires relies heavily on computational analysis, and therefore it is of vital importance that the data is standardized and computer-readable. However in practice, the usage of different abbreviations and non-standard nomenclature in different datasets makes this data pre-processing non-trivial. tidytcells is a lightweight, platform-independent Python package that provides easy-to-use standardization tools specifically designed for TCR nomenclature. The software is open-sourced under the MIT license and is available to install from the Python Package Index (PyPI).\nSingle-Cell Profiling Reveals a Naive-Memory Relationship between CD56bright and Adaptive Human Natural Killer Cells\nAuthors: Panjwani, M. K.; Grassmann, S.; Sottile, R.; Le Ludeuc, J.-B.; Kontopoulos, T.; van der Ploeg, K.; Sun, J. C.; Hsu, K. C.\nScore: 7.2, Published: 2023-09-24 DOI: 10.1101/2023.09.23.559062\nDevelopment of antigen-specific memory upon pathogen exposure is a hallmark of the adaptive immune system. While natural killer (NK) cells are considered part of the innate immune system, humans exposed to the chronic viral pathogen cytomegalovirus (CMV) often possess a distinct NK cell population lacking in individuals who have not been exposed, termed \"adaptive\" NK cells. To identify the \"naive\" population from which this \"memory\" population derives, we performed phenotypic, transcriptional, and functional profiling of NK cell subsets. We identified immature precursors to the Adaptive NK cells that are equally present in both CMV+ and CMV- individuals, resolved an Adaptive transcriptional state distinct from most mature NK cells and sharing a common gene program with the immature CD56bright population, and demonstrated retention of proliferative capacity and acquisition of superior IFN{gamma} production in the Adaptive population. Furthermore, we distinguish the CD56bright and Adaptive NK populations by expression of the transcription factor CXXC5, positioning these memory NK cells at the inflection point between innate and adaptive lymphocytes.\n",
  "wordCount" : "2332",
  "inLanguage": "en",
  "datePublished": "2023-09-27T10:39:11Z",
  "dateModified": "2023-09-27T10:39:11Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/immunology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      immunology
    </h1>
    <div class="post-meta"><span>updated on September 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.559019">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.559019" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.559019">
        <p class="paperTitle">Immune Correlates of Hyperglycemia and Vaccination in a Non-human Primate Model of Long-COVID</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.559019" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.559019" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Palmer, C. S.; Perdios, C.; Abdel-Mohsen, M.; Mudd, J.; Datta, P.; Maness, N.; Lehmicke, G.; Golden, N.; Hellmer, L.; Coyne, C.; Moore Green, K.; Midkiff, C.; Williams, K.; Tiburcio, R.; Fahlberg, M.; Boykin, K.; Kenway, C.; Russell-Lodrigue, K.; Bohm, R.; Blair, R. V.; Dufour, J.; Fischer, T.; Saied, A.; Rappaport, J.</p>
        <p class="info">Score: 36.3, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.559019' target='https://doi.org/10.1101/2023.09.22.559019'> 10.1101/2023.09.22.559019</a></p>
        <p class="abstract">Hyperglycemia, and exacerbation of pre-existing deficits in glucose metabolism, are major manifestations of the post-acute sequelae of SARS-CoV-2 (PASC). Our understanding of lasting glucometabolic disruptions after acute COVID-19 remains unclear due to the lack of animal models for metabolic PASC. Here, we report a non-human primate model of metabolic PASC using SARS-CoV-2 infected African green monkeys (AGMs). Using this model, we have identified a dysregulated chemokine signature and hypersensitive T cell population during acute COVID-19 that correlates with elevated and persistent hyperglycemia four months post-infection. This persistent hyperglycemia correlates with elevated hepatic glycogen, but there was no evidence of long-term SARS-CoV-2 replication in the liver and pancreas. Finally, we report a favorable glycemic effect of the SARS-CoV-2 mRNA vaccine, administered on day 4 post-infection. Together, these data suggest that the AGM metabolic PASC model exhibits important similarities to human metabolic PASC and can be utilized to assess therapeutic candidates to combat this syndrome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557285">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557285" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557285">
        <p class="paperTitle">TAPIR: a T-cell receptor language model for predicting rare and novel targets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557285" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557285" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fast, E.; Dhar, M.; Chen, B.</p>
        <p class="info">Score: 48.7, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557285' target='https://doi.org/10.1101/2023.09.12.557285'> 10.1101/2023.09.12.557285</a></p>
        <p class="abstract">T-cell receptors (TCRs) are involved in most human diseases, but linking their sequences with their targets remains an unsolved grand challenge in the field. In this study, we present TAPIR (T-cell receptor and Peptide Interaction Recognizer), a T-cell receptor (TCR) language model that predicts TCR-target interactions, with a focus on novel and rare targets. TAPIR employs deep convolutional neural network (CNN) encoders to process TCR and target sequences across flexible representations (e.g., beta-chain only, unknown MHC allele, etc.) and learns patterns of interactivity via several training tasks. This flexibility allows TAPIR to train on more than 50k either paired (alpha and beta chain) or unpaired TCRs (just alpha or beta chain) from public and proprietary databases against 1933 unique targets. TAPIR demonstrates state-of-the-art performance when predicting TCR interactivity against common benchmark targets and is the first method to demonstrate strong performance when predicting TCR interactivity against novel targets, where no examples are provided in training. TAPIR is also capable of predicting TCR interaction against MHC alleles in the absence of target information. Leveraging these capabilities, we apply TAPIR to cancer patient TCR repertoires and identify and validate a novel and potent anti-cancer T-cell receptor against a shared cancer neoantigen target (PIK3CA H1047L). We further show how TAPIR, when extended with a generative neural network, is capable of directly designing T-cell receptor sequences that interact with a target of interest.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559083">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559083" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559083">
        <p class="paperTitle">Cellular RNA interacts with MAVS to promote antiviral signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559083" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559083" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gokhale, N. S.; Somfleth, K.; Thompson, M. G.; Sam, R. K.; Marciniak, D. M.; Chu, L. H.; Park, M.; Dvorkin, S.; Oberst, A.; Horner, S. M.; Ong, S.-E.; Gale, M.; Savan, R.</p>
        <p class="info">Score: 21.2, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559083' target='https://doi.org/10.1101/2023.09.25.559083'> 10.1101/2023.09.25.559083</a></p>
        <p class="abstract">Immune signaling needs to be well-regulated to promote clearance of pathogens, while preventing aberrant inflammation. Interferons (IFNs) and antiviral genes are activated by the detection of viral RNA by RIG-I-like receptors (RLRs). Signal transduction downstream of RLRs proceeds through a multi-protein complex organized around the central adaptor protein MAVS. Recent work has shown that protein complex function can be modulated by RNA molecules providing allosteric regulation or acting as molecular guides or scaffolds. Thus, we hypothesized that RNA plays a role in organizing MAVS signaling platforms. Here, we show that MAVS, through its central intrinsically disordered domain, directly interacts with the 3&#39; untranslated regions of cellular mRNAs. Importantly, elimination of RNA by RNase treatment disrupts the MAVS signalosome, including newly identified regulators of RLR signaling, and inhibits phosphorylation of the transcription factor IRF3. This supports the hypothesis that RNA molecules scaffold proteins in the MAVS signalosome to induce IFNs. Together, this work uncovers a function for cellular RNA in promoting signaling through MAVS and highlights a generalizable principle of RNA regulatory control of cytoplasmic immune signaling complexes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.24.558823">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.24.558823" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.24.558823">
        <p class="paperTitle">Weakly supervised identification and generation of adaptive immune receptor sequences associated with immune disease status</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.24.558823" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.24.558823" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Slabodkin, A.; Sollid, L. M.; Sandve, G. K.; Robert, P. A.; Greiff, V.</p>
        <p class="info">Score: 21.0, Published: 2023-09-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.24.558823' target='https://doi.org/10.1101/2023.09.24.558823'> 10.1101/2023.09.24.558823</a></p>
        <p class="abstract">Adaptive immune receptor (AIR) repertoires carry immune signals as sequence motif imprints of past and present encounters with antigen (immune status). Machine learning (ML)-based identification and generation of antigen-specific immune receptors is potentially immense value for public health. The ideal training data for such ML tasks would be AIR datasets, where each sequence is labeled with its cognate antigen. However, given current technological constraints, sequence-labeled datasets are scarce, contrasted by an abundance of repertoire-labeled ones - AIR repertoire datasets where only the repertoire dataset, but not the individual AIRs, are labeled. Therefore, an unmet need exists for an ML approach that enables predictive identification and generation of disease-specific novel AIR sequences using exclusively repertoire-level immune status information. To address this need, we developed AIRRTM, an end-to-end generative model using an encoder-decoder architecture and Topic Modeling (TM) that requires exclusively repertoire-labeled AIR sequencing data as input. We validated AIRRTM&#39;s capacity to identify and generate novel disease-associated receptors on several ground truth synthetic datasets of increasingly complex immune signals and experimental data. AIRRTM broadens the discovery space for immunotherapeutics by enabling the exploitation of large-scale and broadly available immune repertoire data previously deemed largely unsuitable for this task.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558875">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558875" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558875">
        <p class="paperTitle">The Inflammasome Pathway is Activated by Dengue Virus Non-structural Protein 1 and is Protective During Dengue Virus Infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558875" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558875" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wong, M. P.; Juan, E. Y.; Chelluri, S. S.; Wang, P.; Pahmeier, F.; Castillo-Rojas, B.; Blanc, S. F.; Biering, S. B.; Vance, R. E.; Harris, E.</p>
        <p class="info">Score: 19.8, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558875' target='https://doi.org/10.1101/2023.09.21.558875'> 10.1101/2023.09.21.558875</a></p>
        <p class="abstract">Dengue virus (DENV) is a medically important flavivirus causing an estimated 50-100 million dengue cases annually, some of whom progress to severe disease. DENV non-structural protein 1 (NS1) is secreted from infected cells and has been implicated as a major driver of dengue pathogenesis by inducing endothelial barrier dysfunction. However, less is known about how DENV NS1 interacts with immune cells and what role these interactions play. Here we report that DENV NS1 can trigger activation of inflammasomes, a family of cytosolic innate immune sensors that respond to infectious and noxious stimuli, in mouse and human macrophages. DENV NS1 induces the release of IL-1{beta} in a caspase-1 dependent manner. Additionally, we find that DENV NS1-induced inflammasome activation is independent of the NLRP3, Pyrin, and AIM2 inflammasome pathways, but requires CD14. Intriguingly, DENV NS1-induced inflammasome activation does not induce pyroptosis and rapid cell death; instead, macrophages maintain cellular viability while releasing IL-1{beta}. Lastly, we show that caspase-1/11-deficient, but not NLRP3-deficient, mice are more susceptible to lethal DENV infection. Together, these results indicate that the inflammasome pathway acts as a sensor of DENV NS1 and plays a protective role during infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555815">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555815" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555815">
        <p class="paperTitle">Antigenicity and infectivity characterization of SARS-CoV-2 BA.2.86</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555815" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555815" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, S.; Yu, Y.; Jian, F.; Song, W.; Yisimayi, A.; Chen, X.; Xu, Y.; Wang, P.; Yu, L.; Wang, J.; Niu, X.; Wang, J.; Xiao, T.; An, R.; Wang, Y.; Gu, Q.; Shao, F.; Jin, R.; Shen, Z.; Wang, Y.; Cao, Y. R.</p>
        <p class="info">Score: 653.2, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555815' target='https://doi.org/10.1101/2023.09.01.555815'> 10.1101/2023.09.01.555815</a></p>
        <p class="abstract">The recently identified SARS-CoV-2 variant, BA.2.86, which carries a substantial number of Spike mutations, has raised a global alarm. An immediate assessment of its antigenic properties and infectivity is necessary. Here, we reveal the distinct antigenicity of BA.2.86 compared with previous variants including XBB.1.5. BA.2.86 significantly evades convalescent plasma from XBB breakthrough infection (BTI) and reinfections. Key mutations that mediate the enhanced resistance include N450D, K356T, L452W, A484K, V483del, and V445H on the RBD, while BA.2.86s NTD mutations and E554K on SD1 also largely contribute. However, we found that BA.2.86 pseudovirus exhibits compromised efficiency of infecting HEK293T-hACE2 cells compared to XBB.1.5 and EG.5, which may be caused by K356T, V483del, and E554K, and could potentially limit BA.2.86s transmissibility. In sum, it appears that BA.2.86 has traded its infectivity for higher immune evasion during long-term host-viral evolution. Close attention should be paid to monitoring additional mutations that could improve BA.2.86s infectivity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558810">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558810" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558810">
        <p class="paperTitle">Trem2 Agonist Reprograms Foamy Macrophages to Promote Atherosclerotic Plaque Stability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558810" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558810" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Patterson, M. T.; Xu, Y.; Hillman, H.; Osinski, V.; Schrank, P. R.; Kennedy, A. E.; Zhu, A.; Tollison, S.; Shekhar, S.; Stromnes, I.; Tassi, I.; Wu, D.; Binstadt, B.; Williams, J. W.</p>
        <p class="info">Score: 14.8, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558810' target='https://doi.org/10.1101/2023.09.21.558810'> 10.1101/2023.09.21.558810</a></p>
        <p class="abstract">Objective: Trem2, a surface lipid receptor, is expressed on foamy macrophages within atherosclerotic lesions and regulates cell survival, proliferation, and anti-inflammatory responses. Studies examining the role of Trem2 in atherosclerosis have shown that deletion of Trem2 leads to impaired foamy macrophage lipid uptake, proliferation, survival, and cholesterol efflux. Thus, we tested the hypothesis that administration of a validated Trem2 agonist antibody (AL002a) to atherogenic mice could drive macrophage survival and decrease necrotic core formation to improve plaque stability. Approach and Results: To model a therapeutic intervention approach, atherosclerosis-prone mice (Ldlr-/-) were fed a high fat diet (HFD) for 8 weeks, then transitioned to treatment with AL002a or isotype control for an additional 8 weeks while continuing on an HFD. AL002a-treated mice had increased lesion size in both the aortic sinus and whole mount aorta, which correlated with an expansion of plaque macrophage area. This expansion was due to increased macrophage survival and proliferation in plaques. Importantly, plaques from AL002a-treated mice showed improved features of plaque stability, including smaller necrotic cores, increased fibrous caps, and greater collagen deposition. Single cell RNA sequencing of whole aorta suspensions from isotype and AL002a-treated atherosclerotic mice revealed that Trem2 agonism dramatically altered foamy macrophage transcriptome. This included upregulation of oxidative phosphorylation and increased expression of collagen genes. In vitro studies validated that Trem2-agonism with AL002a promoted foamy macrophage oxLDL uptake, survival, and cholesterol efflux in culture. Conclusions: Trem2 agonist expands plaque macrophages by promoting cell survival and proliferation but improves features of plaque stability by rewiring foamy macrophage function to enhance collagen deposition.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558902">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558902" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558902">
        <p class="paperTitle">B cells occupy a unique, dynamic, biological niche within the human myocardium</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558902" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558902" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bermea, K. C.; Duque, C.; Cohen, C. D.; Bhalodia, A.; Rousseau, S.; Lovell, J.; Zita, M. D.; Mugnier, M. C.; Adamo, L.</p>
        <p class="info">Score: 10.1, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558902' target='https://doi.org/10.1101/2023.09.21.558902'> 10.1101/2023.09.21.558902</a></p>
        <p class="abstract">BackgroundA growing body of evidence from animal models indicates that the myocardium hosts a population of B cells that play a role in the development of cardiomyopathy. However, there is minimal data on human myocardial B cells and their biological niche within the heart remains mostly unexplored.

MethodsWe integrated single-cell and single-nuclei datasets from 45 healthy human hearts, 70 hearts with dilated cardiomyopathy (DCM), and 8 hearts with Arrhythmogenic Right Ventricular Cardiomyopathy (ARVC). Interactions between B cells and other cell types were investigated using the CellChat Package. Differential gene expression analysis comparing B cells across conditions was performed using DESeq2. Pathway analysis was performed using Ingenuity, KEGG, and GO pathways analysis.

ResultsOut of 1,200,752 nuclei and 49,723 cells, we identified 1,100 B cells, including naive B cells and plasma cells. B cells showed an extensive network of interactions within the healthy myocardium that included outgoing signaling to macrophages, T cells, endothelial cells, and pericytes, and incoming signaling from endothelial cells, pericytes, and fibroblasts. This biological niche relied on both ECM-receptor interactions, cell-cell contact interactions, and paracrine interaction, it changed significantly in the context of cardiomyopathy and had disease-specific features. Differential gene expression analysis showed that in the context of DCM both naive and plasma myocardial B cells upregulated several pathways related to immune activation, including upregulation of oxidative phosphorylation, upregulation of leukocyte extravasation and, in naive B cells, antigen processing and presentation.

ConclusionsThe human healthy and diseased myocardium contains naive B cells and plasma cells, integrated in a diverse and dynamic biological niche. Naive myocardial-associated B cells likely contribute to the pathogenesis of human dilated cardiomyopathy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558778">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558778" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558778">
        <p class="paperTitle">tidytcells: standardizer for TCR/MHC nomenclature</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558778" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558778" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nagano, Y.; Chain, B.</p>
        <p class="info">Score: 7.3, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558778' target='https://doi.org/10.1101/2023.09.21.558778'> 10.1101/2023.09.21.558778</a></p>
        <p class="abstract">T cell receptors (TCRs) underpin the diversity and specificity of T cell activity. As such, TCR repertoire data is valuable both as an adaptive immune biomarker, and as a way to identify candidate therapeutic TCRs. Analysis of TCR repertoires relies heavily on computational analysis, and therefore it is of vital importance that the data is standardized and computer-readable. However in practice, the usage of different abbreviations and non-standard nomenclature in different datasets makes this data pre-processing non-trivial. tidytcells is a lightweight, platform-independent Python package that provides easy-to-use standardization tools specifically designed for TCR nomenclature. The software is open-sourced under the MIT license and is available to install from the Python Package Index (PyPI).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.23.559062">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.23.559062" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.23.559062">
        <p class="paperTitle">Single-Cell Profiling Reveals a Naive-Memory Relationship between CD56bright and Adaptive Human Natural Killer Cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.23.559062" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.23.559062" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Panjwani, M. K.; Grassmann, S.; Sottile, R.; Le Ludeuc, J.-B.; Kontopoulos, T.; van der Ploeg, K.; Sun, J. C.; Hsu, K. C.</p>
        <p class="info">Score: 7.2, Published: 2023-09-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.23.559062' target='https://doi.org/10.1101/2023.09.23.559062'> 10.1101/2023.09.23.559062</a></p>
        <p class="abstract">Development of antigen-specific memory upon pathogen exposure is a hallmark of the adaptive immune system. While natural killer (NK) cells are considered part of the innate immune system, humans exposed to the chronic viral pathogen cytomegalovirus (CMV) often possess a distinct NK cell population lacking in individuals who have not been exposed, termed &#34;adaptive&#34; NK cells. To identify the &#34;naive&#34; population from which this &#34;memory&#34; population derives, we performed phenotypic, transcriptional, and functional profiling of NK cell subsets. We identified immature precursors to the Adaptive NK cells that are equally present in both CMV&#43; and CMV- individuals, resolved an Adaptive transcriptional state distinct from most mature NK cells and sharing a common gene program with the immature CD56bright population, and demonstrated retention of proliferative capacity and acquisition of superior IFN{gamma} production in the Adaptive population. Furthermore, we distinguish the CD56bright and Adaptive NK populations by expression of the transcription factor CXXC5, positioning these memory NK cells at the inflection point between innate and adaptive lymphocytes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
