<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>immunology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>immunology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Heterotypic responses against nsp12/nsp13 from prior SARS-CoV-2 infection associates with lower subsequent endemic coronavirus incidence
Authors: Bean, D. J.; Monroe, J.; Liang, Y. M.; Borberg, E.; Senussi, Y.; Swank, Z.; Chalise, S.; Walt, D.; Weinberg, J.; Sagar, M.
Score: 143.7, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563621
Immune responses from prior SARS-CoV-2 infection and COVID-19 vaccination do not prevent re-infections and may not protect against future novel coronaviruses (CoVs). We examined the incidence of and immune differences against human endemic CoVs (eCoV) as a proxy for response against future emerging CoVs.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/immunology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="immunology" />
<meta property="og:description" content="Heterotypic responses against nsp12/nsp13 from prior SARS-CoV-2 infection associates with lower subsequent endemic coronavirus incidence
Authors: Bean, D. J.; Monroe, J.; Liang, Y. M.; Borberg, E.; Senussi, Y.; Swank, Z.; Chalise, S.; Walt, D.; Weinberg, J.; Sagar, M.
Score: 143.7, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563621
Immune responses from prior SARS-CoV-2 infection and COVID-19 vaccination do not prevent re-infections and may not protect against future novel coronaviruses (CoVs). We examined the incidence of and immune differences against human endemic CoVs (eCoV) as a proxy for response against future emerging CoVs." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/immunology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-01T10:38:40+00:00" />
<meta property="article:modified_time" content="2023-11-01T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="immunology"/>
<meta name="twitter:description" content="Heterotypic responses against nsp12/nsp13 from prior SARS-CoV-2 infection associates with lower subsequent endemic coronavirus incidence
Authors: Bean, D. J.; Monroe, J.; Liang, Y. M.; Borberg, E.; Senussi, Y.; Swank, Z.; Chalise, S.; Walt, D.; Weinberg, J.; Sagar, M.
Score: 143.7, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563621
Immune responses from prior SARS-CoV-2 infection and COVID-19 vaccination do not prevent re-infections and may not protect against future novel coronaviruses (CoVs). We examined the incidence of and immune differences against human endemic CoVs (eCoV) as a proxy for response against future emerging CoVs."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "immunology",
      "item": "https://trxiv.yorks0n.com/posts/immunology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "immunology",
  "name": "immunology",
  "description": "Heterotypic responses against nsp12/nsp13 from prior SARS-CoV-2 infection associates with lower subsequent endemic coronavirus incidence\nAuthors: Bean, D. J.; Monroe, J.; Liang, Y. M.; Borberg, E.; Senussi, Y.; Swank, Z.; Chalise, S.; Walt, D.; Weinberg, J.; Sagar, M.\nScore: 143.7, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563621\nImmune responses from prior SARS-CoV-2 infection and COVID-19 vaccination do not prevent re-infections and may not protect against future novel coronaviruses (CoVs). We examined the incidence of and immune differences against human endemic CoVs (eCoV) as a proxy for response against future emerging CoVs.",
  "keywords": [
    
  ],
  "articleBody": " Heterotypic responses against nsp12/nsp13 from prior SARS-CoV-2 infection associates with lower subsequent endemic coronavirus incidence\nAuthors: Bean, D. J.; Monroe, J.; Liang, Y. M.; Borberg, E.; Senussi, Y.; Swank, Z.; Chalise, S.; Walt, D.; Weinberg, J.; Sagar, M.\nScore: 143.7, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563621\nImmune responses from prior SARS-CoV-2 infection and COVID-19 vaccination do not prevent re-infections and may not protect against future novel coronaviruses (CoVs). We examined the incidence of and immune differences against human endemic CoVs (eCoV) as a proxy for response against future emerging CoVs. Assessment was among those with known SARS-CoV-2 infection, COVID-19 vaccination but no documented SARS-CoV-2 infection, or neither exposure. Retrospective cohort analyses suggest that prior SARS-CoV-2 infection, but not COVID-19 vaccination alone, protects against subsequent symptomatic eCoV infection. CD8+ T cell responses to the non-structural eCoV proteins, nsp12 and nsp13, were significantly higher in individuals with previous SARS-CoV-2 infection as compared to the other groups. The three groups had similar cellular responses against the eCoV spike and nucleocapsid, and those with prior spike exposure had lower eCoV-directed neutralizing antibodies. Incorporation of non-structural viral antigens in a future pan-CoV vaccine may improve protection against future heterologous CoV infections.\nPan-cancer profiling of tumor-infiltrating natural killer cells through transcriptional reference mapping\nAuthors: Netskar, H. K.; Pfefferle, A.; Goodridge, J. P.; Sohlberg, E.; Dufva, O.; Teichmann, S.; Clancy, T.; horowitz, a.; Malmberg, K.-J.\nScore: 25.4, Published: 2023-10-30 DOI: 10.1101/2023.10.26.564050\nThe functional diversity of natural killer (NK) cell repertoires stems from differentiation, homeostatic receptor-ligand interactions, and adaptive-like responses to viral infections. Here, we generated a single-cell transcriptional reference map of healthy human blood and tissue-derived NK cells, with temporal resolution and fate-specific expression of gene regulator networks defining NK cell differentiation. Using transfer learning, transcriptomes of tumor-infiltrating NK cells from seven solid tumor types (427 patients), combined from 39 datasets, were incorporated into the reference map and interrogated for tumor microenvironment (TME)-induced perturbations. We identified six functionally distinct NK cellular states in healthy and malignant tissues, two of which were commonly enriched for across tumor types: a dysfunctional stressed CD56bright state susceptible to TME-induced immunosuppression and a cytotoxic TME-resistant effector CD56dim state. The ratio of stressed CD56bright and effector CD56dim was predictive of patient outcome in malignant melanoma and osteosarcoma. This resource may inform the design of novel NK cell therapies and can be extended endlessly through transfer learning to interrogate new datasets from experimental perturbations or disease conditions.\nSARS-CoV-2 antibodies cross-react and enhance dengue infection\nAuthors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.\nScore: 260.9, Published: 2023-10-10 DOI: 10.1101/2023.10.09.557914\nDengue disease is highly prevalent in tropical and subtropical regions worldwide. However, its pathogenesis is still incompletely understood, particularly in comparison to other endemic viruses. Antibody-dependent enhancement (ADE) is a well-known phenomenon for dengue viruses. Given the recent surge in dengue cases and potential cross-reactivity with SARS-CoV-2 antibodies, this study explores the impact of anti-SARS-CoV-2 antibodies on DENV-2 infection. The study assessed the cross-reactivity of SARS-CoV-2 antibodies with the DENV-2 Virus. Human convalescent plasma samples collected during different waves of COVID-19 and monoclonal and polyclonal antibodies raised against SARS-CoV-2 were examined for their potential to cause ADE of DENV-2 infection using cell-based assays. The study found that anti-SARS-CoV-2 antibodies acquired from natural infection in humans or through experimental immunization in animals were cross-reactive with DENV-2 and had the potential to enhance DENV-2 infection in K562 and U937 cells. In-silico and in-vitro studies indicated a strong interaction between SARS-CoV-2 antibodies and DENV-2 E-protein, providing a molecular basis for these findings. This study is the first to demonstrate that anti-SARS-CoV-2 antibodies can cross-react with DENV-2 and can enhance its infection through ADE. These findings have implications for SARS-CoV-2 vaccine development and deployment strategies in regions where dengue is endemic. SummaryAntibodies against SARS-CoV-2 (RBD and Spike) showed significant cross reactivity with DENV-2 (E protein). Also, anti-SARS-CoV-2-commercial antibodies, immunised animal sera and 46 human convalescent plasma samples (from different waves of pandemic) demonstrated antibody-dependent enhancement (ADE) of DENV-2 infection.\nNK cell-monocyte crosstalk underlies NK cell activation in severe COVID-19\nAuthors: Lee, M. J.; de los Rios Kobara, I.; Barnard, T. R.; Vales Torres, X.; Tobin, N. H.; Ferbas, K. G.; Rimoin, A. W. J.; Yang, O. O.; Aldrovandi, G. M.; Wilk, A. J.; Fulcher, J.; Blish, C. A.\nScore: 16.6, Published: 2023-10-30 DOI: 10.1101/2023.10.27.564440\nNK cells in the peripheral blood of severe COVID-19 patients exhibit a unique profile characterized by activation and dysfunction. Previous studies have identified soluble factors, including type I interferon and TGF{beta}, that underlie this dysregulation. However, the role of cell-cell interactions in mediating changes in NK cell function during COVID-19 remains unclear. To address this question, we combined cell-cell communication analysis on existing single-cell RNA sequencing data with in vitro primary cell co-culture experiments to dissect the mechanisms underlying NK cell dysfunction in COVID-19. We found that NK cells are predicted to interact most strongly with monocytes and that this occurs via both soluble factors and direct interactions. To validate these findings, we performed in vitro co-cultures in which NK cells from healthy donors were incubated with monocytes from COVID-19+ or healthy donors. Co-culture of healthy NK cells with monocytes from COVID-19 patients recapitulated aspects of the NK cell phenotype observed in severe COVID-19, including decreased expression of NKG2D, increased expression of activation markers, and increased proliferation. When these experiments were performed in a transwell setting, we found that only CD56bright CD16- NK cells were activated in the presence of severe COVID-19 patient monocytes. O-link analysis of supernatants from transwell co-cultures revealed that cultures containing severe COVID-19 patient monocytes had significantly elevated levels of proinflammatory cytokines and chemokines as well as TGF{beta}. Collectively, these results demonstrate that interactions between NK cells and monocytes in the peripheral blood of COVID-19 patients contribute to NK cell activation and dysfunction in severe COVID-19.\nReduced Monocyte Proportions and Responsiveness in Convalescent COVID-19 Patients\nAuthors: Ravkov, E.; Williams, E. S. C. P.; Elgort, M.; Spivak, A.; Barker, A. P.; Planelles, V.; Delgado, J.; Lin, L.; Hanley, T.\nScore: 11.8, Published: 2023-10-26 DOI: 10.1101/2023.10.25.563806\nThe clinical manifestations of acute severe acute respiratory syndrome coronavirus type 2 (SARS-CoV-2) infection and COVID-19 suggest a dysregulation of the host immune response that leads to inflammation, thrombosis, and organ dysfunction. It is less clear whether these dysregulated processes persist during the convalescent phase of disease or during long COVID. We investigated the effects of SARS-CoV-2 infection on the proportions of classical, intermediate, and non-classical monocytes, their activation status, and their functional properties in convalescent COVID-19 patients and uninfected control subjects. We found that the percentage of total monocytes was decreased in convalescent COVID-19 patients compared to uninfected controls. This was due to decreased intermediate and non-classical monocytes. Classical monocytes from convalescent COVID-19 patients demonstrated a decrease in activation markers, such as CD56, in response to stimulation with bacterial lipopolysaccharide (LPS). In addition, classical monocytes from convalescent COVID-19 patients showed decreased expression of CD142 (tissue factor), which can initiate the extrinsic coagulation cascade, in response to LPS stimulation. Finally, we found that monocytes from convalescent COVID-19 patients produced less TNF- and IL-6 in response to LPS stimulation, than those from uninfected controls. In conclusion, SARS-CoV-2 infection exhibits a clear effect on the relative proportions of monocyte subsets, the activation status of classical monocytes, and proinflammatory cytokine production that persists during the convalescent phase of disease.\nImmune Epitopes of SARS-CoV-2 Spike Protein and Considerations for Universal Vaccine Development\nAuthors: Magazine, N.; Zhang, T.; Bungwon, A. D.; McGee, M. C.; Wu, Y.; Veggiani, G.; Huang, W.\nScore: 10.0, Published: 2023-10-27 DOI: 10.1101/2023.10.26.564184\nDespite the success of global vaccination programs in slowing the spread of COVID-19, these efforts have been hindered by the emergence of new SARS-CoV-2 strains capable of evading prior immunity. The mutation and evolution of SARS-CoV-2 have created a demand for persistent efforts in vaccine development. SARS-CoV-2 Spike protein has been the primary target for COVID-19 vaccine development, but it is also the hotspot of mutations directly involved in host susceptibility and immune evasion. Our ability to predict emerging mutants and select conserved epitopes is critical for the development of a broadly neutralizing therapy or a universal vaccine. In this article, we review the general paradigm of immune responses to COVID-19 vaccines, highlighting the immunological epitopes of Spike protein that are likely associated with eliciting protective immunity resulting from vaccination. Specifically, we analyze the structural and evolutionary characteristics of the SARS-CoV-2 Spike protein related to immune activation and function via the toll-like receptors (TLRs), B cells, and T cells. We aim to provide a comprehensive analysis of immune epitopes of Spike protein, thereby contributing to the development of new strategies for broad neutralization or universal vaccination.\nResolving haplotype variation and complex genetic architecture in the human immunoglobulin kappa chain locus in individuals of diverse ancestry\nAuthors: Engelbrecht, E.; Rodriguez, O. L.; Shields, K.; Schulze, S.; Tieri, D.; Jana, U.; Yaari, G.; Lees, W.; Smith, M. L.; Watson, C. T.\nScore: 8.9, Published: 2023-10-25 DOI: 10.1101/2023.10.23.563321\nImmunoglobulins (IGs), critical components of the human immune system, are composed of heavy and light protein chains encoded at three genomic loci. The IG Kappa (IGK) chain locus consists of two large, inverted segmental duplications. The complexity of IG loci has hindered effective use of standard high- throughput methods for characterizing genetic variation within these regions. To overcome these limitations, we leverage long-read sequencing to create haplotype-resolved IGK assemblies in an ancestrally diverse cohort (n=36), representing the first comprehensive description of IGK haplotype variation at population-scale. We identify extensive locus polymorphism, including novel single nucleotide variants (SNVs) and a common novel [~]24.7 Kbp structural variant harboring a functional IGKV gene. Among 47 functional IGKV genes, we identify 141 alleles, 64 (45.4%) of which were not previously curated. We report inter-population differences in allele frequencies for 14 of the IGKV genes, including alleles unique to specific populations within this dataset. Finally, we identify haplotypes carrying signatures of gene conversion that associate with enrichment of SNVs in the IGK distal region. These data provide a critical resource of curated genomic reference information from diverse ancestries, laying a foundation for advancing our understanding of population-level genetic variation in the IGK locus.\nA novel bioinformatics pipeline for the identification of immune inhibitory receptors as potential therapeutic targets\nAuthors: Singh, A.; Miranda Bedate, A.; von richthofen, H. J.; van der Vlist, M.; Kuhn, R.; Yermanos, A.; Kuball, J. J.; Kesmir, C.; Pascoal Ramos, M. I.; Meyaard, L.\nScore: 8.4, Published: 2023-10-28 DOI: 10.1101/2023.10.24.563834\nBlocking inhibitory receptors like PD-1 and CTLA-4 has revolutionized cancer treatment in recent years. However, despite major successes in melanoma and lung cancer, the majority of cancer types are not responsive to these immunotherapies. As such, there is an ongoing need for the identification of novel inhibitory receptors as drug targets. Most inhibitory receptors signal via immunoreceptor tyrosine-based inhibitory motifs (ITIMs) and previous studies have estimated that our genome contains over 1600 ITIM-bearing transmembrane proteins. However, further testing and development of this large number of candidates requires increased understanding of their expression patterns and likelihood to function as inhibitory receptor. To assist in the selection of novel inhibitory receptor as therapeutic targets, we designed a novel bioinformatics pipeline integrating machine learning-guided structural predictions and sequence-based likelihood models to identify 51 known and 390 putative inhibitory receptors. Using publicly available transcriptomics data of immune cells, we determined the expression of these novel inhibitory receptors, and classified them into previously proposed functional categories. Known and putative inhibitory receptors were expressed across a wide variety of immune cells, and we found cell type-specific patterns in expression of these receptors. We used our pipeline to study inhibitory receptor expression patterns in single cell transcriptomics data of tumour infiltrating T cells. We determined that putative immune inhibitory receptors were expressed differentially in CD4+ and CD8+ T cell subsets, including exhausted CD8+ T cells and CD4+ memory T cells, which could allow for subset-specific targeting. In conclusion, we present an inhibitory receptor pipeline that identifies 51 known and 390 novel inhibitory receptors. This pipeline will support future drug target selection across diseases where therapeutic targeting of immune inhibitory receptors is warranted.\nEarly antiviral CD4 and CD8 T cell responses are associated with upper respiratory tract clearance of SARS-CoV-2\nAuthors: Ramirez, S. I.; Lopez, P. G.; Faraji, F.; Parikh, U. M.; Heaps, A.; Ritz, J.; Moser, C.; Eron, J. J.; Wohl, D. A.; Currier, J. S.; Daar, E. S.; Greninger, A. L.; Klekotka, P.; Grifoni, A.; Weiskopf, D.; Sette, A.; Peters, B.; Hughes, M. D.; Chew, K. W.; Smith, D. M.; Crotty, S.; ACTIV-2/A5401 Study Team, Score: 8.0, Published: 2023-10-26 DOI: 10.1101/2023.10.25.564014\nT cells are involved in protective immunity against numerous viral infections. Limited data have been available regarding roles of human T cell responses controlling SARS-CoV-2 viral clearance in primary COVID-19. Here, we examined longitudinal SARS-CoV-2 upper respiratory tract viral RNA levels and early adaptive immune responses from 95 unvaccinated individuals with acute COVID-19. Acute SARS-CoV-2-specific CD4 and CD8 T cell responses were evaluated in addition to antibody responses. Most individuals with acute COVID-19 developed rapid SARS-CoV-2-specific T cell responses during infection, and both early CD4 T cell and CD8 T cell responses correlated with reduced upper respiratory tract SARS-CoV-2 viral RNA, independent of neutralizing antibody titers. Overall, our findings indicate a distinct protective role for SARS-CoV-2-specific T cells during acute COVID-19.\nmRNA vaccines encoding membrane-anchored receptor-binding domains of SARS-CoV-2 mutants induce strong humoral responses and can overcome immune imprinting\nAuthors: Al-Wassiti, H. A.; Fabb, S. A.; Grimley, S. L.; Kochappan, R.; Ho, J. K.; Wong, C. Y.; Tan, C. W.; Payne, T. J.; Takanashi, A.; Sicilia, H.; Teo, S. L. Y.; McAuley, J. L.; Ellenberg, P.; Cooney, J. P.; Davidson, K. C.; Bowen, R.; Pellegrini, M.; Rockman, S.; Godfrey, D. I.; Nolan, T. M.; Wang, L.-F.; Deliyannis, G.; Purcell, D. F. J.; Pouton, C. W.\nScore: 27.9, Published: 2023-10-04 DOI: 10.1101/2023.10.04.560777\nTo address the limitations of whole-spike COVID vaccines, we explored mRNA vaccines encoding membrane-anchored receptor-binding domain (RBD-TMs), each a fusion of a variant RBD, the transmembrane (TM) and cytoplasmic tail (CT) fragments of the SARS-CoV-2 spike protein. In naive mice, RBD-TM mRNA vaccines against ancestral SARS-CoV-2, Beta, Delta, Delta-plus, Kappa, Omicron BA.1 or BA.5, all induced strong humoral responses against the target RBD. Multiplex surrogate viral neutralization (sVNT) assays indicated broad neutralizing activity against a range of variant RBDs. In the setting of a heterologous boost, against the background of exposure to ancestral whole spike vaccines, sVNT studies suggested that RBD-TM vaccines were able to overcome the detrimental effects of immune imprinting. Omicron BA.1 and BA.5 RBD-TM booster vaccines induced serum antibodies with 12 and 22-fold higher neutralizing activity against the target RBD than their equivalent whole spike variants. Boosting with BA.1 or BA.5 RBD-TM provided good protection against more recent variants including XBB and XBB.1.5. Each RBD-TM mRNA is 28% of the length of its whole-spike equivalent. This advantage will enable tetravalent mRNA vaccines to be developed at well-tolerated doses of formulated mRNA. One Sentence SummarymRNA vaccines encoding membrane-anchored RBDs of SARS-CoV-2 mutants are effective vaccines that can overcome immune imprinting in mice\n",
  "wordCount" : "2548",
  "inLanguage": "en",
  "datePublished": "2023-11-01T10:38:40Z",
  "dateModified": "2023-11-01T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/immunology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      immunology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 1, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563621">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563621" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563621">
        <p class="paperTitle">Heterotypic responses against nsp12/nsp13 from prior SARS-CoV-2 infection associates with lower subsequent endemic coronavirus incidence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563621" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563621" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bean, D. J.; Monroe, J.; Liang, Y. M.; Borberg, E.; Senussi, Y.; Swank, Z.; Chalise, S.; Walt, D.; Weinberg, J.; Sagar, M.</p>
        <p class="info">Score: 143.7, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563621' target='https://doi.org/10.1101/2023.10.23.563621'> 10.1101/2023.10.23.563621</a></p>
        <p class="abstract">Immune responses from prior SARS-CoV-2 infection and COVID-19 vaccination do not prevent re-infections and may not protect against future novel coronaviruses (CoVs). We examined the incidence of and immune differences against human endemic CoVs (eCoV) as a proxy for response against future emerging CoVs. Assessment was among those with known SARS-CoV-2 infection, COVID-19 vaccination but no documented SARS-CoV-2 infection, or neither exposure. Retrospective cohort analyses suggest that prior SARS-CoV-2 infection, but not COVID-19 vaccination alone, protects against subsequent symptomatic eCoV infection. CD8&#43; T cell responses to the non-structural eCoV proteins, nsp12 and nsp13, were significantly higher in individuals with previous SARS-CoV-2 infection as compared to the other groups. The three groups had similar cellular responses against the eCoV spike and nucleocapsid, and those with prior spike exposure had lower eCoV-directed neutralizing antibodies. Incorporation of non-structural viral antigens in a future pan-CoV vaccine may improve protection against future heterologous CoV infections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564050">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564050" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564050">
        <p class="paperTitle">Pan-cancer profiling of tumor-infiltrating natural killer cells through transcriptional reference mapping</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564050" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564050" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Netskar, H. K.; Pfefferle, A.; Goodridge, J. P.; Sohlberg, E.; Dufva, O.; Teichmann, S.; Clancy, T.; horowitz, a.; Malmberg, K.-J.</p>
        <p class="info">Score: 25.4, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564050' target='https://doi.org/10.1101/2023.10.26.564050'> 10.1101/2023.10.26.564050</a></p>
        <p class="abstract">The functional diversity of natural killer (NK) cell repertoires stems from differentiation, homeostatic receptor-ligand interactions, and adaptive-like responses to viral infections. Here, we generated a single-cell transcriptional reference map of healthy human blood and tissue-derived NK cells, with temporal resolution and fate-specific expression of gene regulator networks defining NK cell differentiation. Using transfer learning, transcriptomes of tumor-infiltrating NK cells from seven solid tumor types (427 patients), combined from 39 datasets, were incorporated into the reference map and interrogated for tumor microenvironment (TME)-induced perturbations. We identified six functionally distinct NK cellular states in healthy and malignant tissues, two of which were commonly enriched for across tumor types: a dysfunctional stressed CD56bright state susceptible to TME-induced immunosuppression and a cytotoxic TME-resistant effector CD56dim state. The ratio of stressed CD56bright and effector CD56dim was predictive of patient outcome in malignant melanoma and osteosarcoma. This resource may inform the design of novel NK cell therapies and can be extended endlessly through transfer learning to interrogate new datasets from experimental perturbations or disease conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.557914">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.557914" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.557914">
        <p class="paperTitle">SARS-CoV-2 antibodies cross-react and enhance dengue infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.557914" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.557914" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.</p>
        <p class="info">Score: 260.9, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.557914' target='https://doi.org/10.1101/2023.10.09.557914'> 10.1101/2023.10.09.557914</a></p>
        <p class="abstract">Dengue disease is highly prevalent in tropical and subtropical regions worldwide. However, its pathogenesis is still incompletely understood, particularly in comparison to other endemic viruses. Antibody-dependent enhancement (ADE) is a well-known phenomenon for dengue viruses. Given the recent surge in dengue cases and potential cross-reactivity with SARS-CoV-2 antibodies, this study explores the impact of anti-SARS-CoV-2 antibodies on DENV-2 infection.

The study assessed the cross-reactivity of SARS-CoV-2 antibodies with the DENV-2 Virus. Human convalescent plasma samples collected during different waves of COVID-19 and monoclonal and polyclonal antibodies raised against SARS-CoV-2 were examined for their potential to cause ADE of DENV-2 infection using cell-based assays. The study found that anti-SARS-CoV-2 antibodies acquired from natural infection in humans or through experimental immunization in animals were cross-reactive with DENV-2 and had the potential to enhance DENV-2 infection in K562 and U937 cells. In-silico and in-vitro studies indicated a strong interaction between SARS-CoV-2 antibodies and DENV-2 E-protein, providing a molecular basis for these findings. This study is the first to demonstrate that anti-SARS-CoV-2 antibodies can cross-react with DENV-2 and can enhance its infection through ADE. These findings have implications for SARS-CoV-2 vaccine development and deployment strategies in regions where dengue is endemic.

SummaryAntibodies against SARS-CoV-2 (RBD and Spike) showed significant cross reactivity with DENV-2 (E protein). Also, anti-SARS-CoV-2-commercial antibodies, immunised animal sera and 46 human convalescent plasma samples (from different waves of pandemic) demonstrated antibody-dependent enhancement (ADE) of DENV-2 infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564440">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564440" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564440">
        <p class="paperTitle">NK cell-monocyte crosstalk underlies NK cell activation in severe COVID-19</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564440" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564440" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, M. J.; de los Rios Kobara, I.; Barnard, T. R.; Vales Torres, X.; Tobin, N. H.; Ferbas, K. G.; Rimoin, A. W. J.; Yang, O. O.; Aldrovandi, G. M.; Wilk, A. J.; Fulcher, J.; Blish, C. A.</p>
        <p class="info">Score: 16.6, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564440' target='https://doi.org/10.1101/2023.10.27.564440'> 10.1101/2023.10.27.564440</a></p>
        <p class="abstract">NK cells in the peripheral blood of severe COVID-19 patients exhibit a unique profile characterized by activation and dysfunction. Previous studies have identified soluble factors, including type I interferon and TGF{beta}, that underlie this dysregulation. However, the role of cell-cell interactions in mediating changes in NK cell function during COVID-19 remains unclear. To address this question, we combined cell-cell communication analysis on existing single-cell RNA sequencing data with in vitro primary cell co-culture experiments to dissect the mechanisms underlying NK cell dysfunction in COVID-19. We found that NK cells are predicted to interact most strongly with monocytes and that this occurs via both soluble factors and direct interactions. To validate these findings, we performed in vitro co-cultures in which NK cells from healthy donors were incubated with monocytes from COVID-19&#43; or healthy donors. Co-culture of healthy NK cells with monocytes from COVID-19 patients recapitulated aspects of the NK cell phenotype observed in severe COVID-19, including decreased expression of NKG2D, increased expression of activation markers, and increased proliferation. When these experiments were performed in a transwell setting, we found that only CD56bright CD16- NK cells were activated in the presence of severe COVID-19 patient monocytes. O-link analysis of supernatants from transwell co-cultures revealed that cultures containing severe COVID-19 patient monocytes had significantly elevated levels of proinflammatory cytokines and chemokines as well as TGF{beta}. Collectively, these results demonstrate that interactions between NK cells and monocytes in the peripheral blood of COVID-19 patients contribute to NK cell activation and dysfunction in severe COVID-19.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.563806">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.563806" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.563806">
        <p class="paperTitle">Reduced Monocyte Proportions and Responsiveness in Convalescent COVID-19 Patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.563806" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.563806" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ravkov, E.; Williams, E. S. C. P.; Elgort, M.; Spivak, A.; Barker, A. P.; Planelles, V.; Delgado, J.; Lin, L.; Hanley, T.</p>
        <p class="info">Score: 11.8, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.563806' target='https://doi.org/10.1101/2023.10.25.563806'> 10.1101/2023.10.25.563806</a></p>
        <p class="abstract">The clinical manifestations of acute severe acute respiratory syndrome coronavirus type 2 (SARS-CoV-2) infection and COVID-19 suggest a dysregulation of the host immune response that leads to inflammation, thrombosis, and organ dysfunction. It is less clear whether these dysregulated processes persist during the convalescent phase of disease or during long COVID. We investigated the effects of SARS-CoV-2 infection on the proportions of classical, intermediate, and non-classical monocytes, their activation status, and their functional properties in convalescent COVID-19 patients and uninfected control subjects. We found that the percentage of total monocytes was decreased in convalescent COVID-19 patients compared to uninfected controls. This was due to decreased intermediate and non-classical monocytes. Classical monocytes from convalescent COVID-19 patients demonstrated a decrease in activation markers, such as CD56, in response to stimulation with bacterial lipopolysaccharide (LPS). In addition, classical monocytes from convalescent COVID-19 patients showed decreased expression of CD142 (tissue factor), which can initiate the extrinsic coagulation cascade, in response to LPS stimulation. Finally, we found that monocytes from convalescent COVID-19 patients produced less TNF- and IL-6 in response to LPS stimulation, than those from uninfected controls. In conclusion, SARS-CoV-2 infection exhibits a clear effect on the relative proportions of monocyte subsets, the activation status of classical monocytes, and proinflammatory cytokine production that persists during the convalescent phase of disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564184">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564184" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564184">
        <p class="paperTitle">Immune Epitopes of SARS-CoV-2 Spike Protein and Considerations for Universal Vaccine Development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564184" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564184" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Magazine, N.; Zhang, T.; Bungwon, A. D.; McGee, M. C.; Wu, Y.; Veggiani, G.; Huang, W.</p>
        <p class="info">Score: 10.0, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564184' target='https://doi.org/10.1101/2023.10.26.564184'> 10.1101/2023.10.26.564184</a></p>
        <p class="abstract">Despite the success of global vaccination programs in slowing the spread of COVID-19, these efforts have been hindered by the emergence of new SARS-CoV-2 strains capable of evading prior immunity. The mutation and evolution of SARS-CoV-2 have created a demand for persistent efforts in vaccine development. SARS-CoV-2 Spike protein has been the primary target for COVID-19 vaccine development, but it is also the hotspot of mutations directly involved in host susceptibility and immune evasion. Our ability to predict emerging mutants and select conserved epitopes is critical for the development of a broadly neutralizing therapy or a universal vaccine. In this article, we review the general paradigm of immune responses to COVID-19 vaccines, highlighting the immunological epitopes of Spike protein that are likely associated with eliciting protective immunity resulting from vaccination. Specifically, we analyze the structural and evolutionary characteristics of the SARS-CoV-2 Spike protein related to immune activation and function via the toll-like receptors (TLRs), B cells, and T cells. We aim to provide a comprehensive analysis of immune epitopes of Spike protein, thereby contributing to the development of new strategies for broad neutralization or universal vaccination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563321">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563321" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563321">
        <p class="paperTitle">Resolving haplotype variation and complex genetic architecture in the human immunoglobulin kappa chain locus in individuals of diverse ancestry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563321" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563321" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Engelbrecht, E.; Rodriguez, O. L.; Shields, K.; Schulze, S.; Tieri, D.; Jana, U.; Yaari, G.; Lees, W.; Smith, M. L.; Watson, C. T.</p>
        <p class="info">Score: 8.9, Published: 2023-10-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563321' target='https://doi.org/10.1101/2023.10.23.563321'> 10.1101/2023.10.23.563321</a></p>
        <p class="abstract">Immunoglobulins (IGs), critical components of the human immune system, are composed of heavy and light protein chains encoded at three genomic loci. The IG Kappa (IGK) chain locus consists of two large, inverted segmental duplications. The complexity of IG loci has hindered effective use of standard high- throughput methods for characterizing genetic variation within these regions. To overcome these limitations, we leverage long-read sequencing to create haplotype-resolved IGK assemblies in an ancestrally diverse cohort (n=36), representing the first comprehensive description of IGK haplotype variation at population-scale. We identify extensive locus polymorphism, including novel single nucleotide variants (SNVs) and a common novel [~]24.7 Kbp structural variant harboring a functional IGKV gene. Among 47 functional IGKV genes, we identify 141 alleles, 64 (45.4%) of which were not previously curated. We report inter-population differences in allele frequencies for 14 of the IGKV genes, including alleles unique to specific populations within this dataset. Finally, we identify haplotypes carrying signatures of gene conversion that associate with enrichment of SNVs in the IGK distal region. These data provide a critical resource of curated genomic reference information from diverse ancestries, laying a foundation for advancing our understanding of population-level genetic variation in the IGK locus.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563834">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563834" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563834">
        <p class="paperTitle">A novel bioinformatics pipeline for the identification of immune inhibitory receptors as potential therapeutic targets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563834" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563834" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Singh, A.; Miranda Bedate, A.; von richthofen, H. J.; van der Vlist, M.; Kuhn, R.; Yermanos, A.; Kuball, J. J.; Kesmir, C.; Pascoal Ramos, M. I.; Meyaard, L.</p>
        <p class="info">Score: 8.4, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563834' target='https://doi.org/10.1101/2023.10.24.563834'> 10.1101/2023.10.24.563834</a></p>
        <p class="abstract">Blocking inhibitory receptors like PD-1 and CTLA-4 has revolutionized cancer treatment in recent years. However, despite major successes in melanoma and lung cancer, the majority of cancer types are not responsive to these immunotherapies. As such, there is an ongoing need for the identification of novel inhibitory receptors as drug targets. Most inhibitory receptors signal via immunoreceptor tyrosine-based inhibitory motifs (ITIMs) and previous studies have estimated that our genome contains over 1600 ITIM-bearing transmembrane proteins. However, further testing and development of this large number of candidates requires increased understanding of their expression patterns and likelihood to function as inhibitory receptor. To assist in the selection of novel inhibitory receptor as therapeutic targets, we designed a novel bioinformatics pipeline integrating machine learning-guided structural predictions and sequence-based likelihood models to identify 51 known and 390 putative inhibitory receptors. Using publicly available transcriptomics data of immune cells, we determined the expression of these novel inhibitory receptors, and classified them into previously proposed functional categories. Known and putative inhibitory receptors were expressed across a wide variety of immune cells, and we found cell type-specific patterns in expression of these receptors. We used our pipeline to study inhibitory receptor expression patterns in single cell transcriptomics data of tumour infiltrating T cells. We determined that putative immune inhibitory receptors were expressed differentially in CD4&#43; and CD8&#43; T cell subsets, including exhausted CD8&#43; T cells and CD4&#43; memory T cells, which could allow for subset-specific targeting. In conclusion, we present an inhibitory receptor pipeline that identifies 51 known and 390 novel inhibitory receptors. This pipeline will support future drug target selection across diseases where therapeutic targeting of immune inhibitory receptors is warranted.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.564014">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.564014" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.564014">
        <p class="paperTitle">Early antiviral CD4 and CD8 T cell responses are associated with upper respiratory tract clearance of SARS-CoV-2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.564014" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.564014" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ramirez, S. I.; Lopez, P. G.; Faraji, F.; Parikh, U. M.; Heaps, A.; Ritz, J.; Moser, C.; Eron, J. J.; Wohl, D. A.; Currier, J. S.; Daar, E. S.; Greninger, A. L.; Klekotka, P.; Grifoni, A.; Weiskopf, D.; Sette, A.; Peters, B.; Hughes, M. D.; Chew, K. W.; Smith, D. M.; Crotty, S.; ACTIV-2/A5401 Study Team,  </p>
        <p class="info">Score: 8.0, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.564014' target='https://doi.org/10.1101/2023.10.25.564014'> 10.1101/2023.10.25.564014</a></p>
        <p class="abstract">T cells are involved in protective immunity against numerous viral infections. Limited data have been available regarding roles of human T cell responses controlling SARS-CoV-2 viral clearance in primary COVID-19. Here, we examined longitudinal SARS-CoV-2 upper respiratory tract viral RNA levels and early adaptive immune responses from 95 unvaccinated individuals with acute COVID-19. Acute SARS-CoV-2-specific CD4 and CD8 T cell responses were evaluated in addition to antibody responses. Most individuals with acute COVID-19 developed rapid SARS-CoV-2-specific T cell responses during infection, and both early CD4 T cell and CD8 T cell responses correlated with reduced upper respiratory tract SARS-CoV-2 viral RNA, independent of neutralizing antibody titers. Overall, our findings indicate a distinct protective role for SARS-CoV-2-specific T cells during acute COVID-19.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560777">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560777" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560777">
        <p class="paperTitle">mRNA vaccines encoding membrane-anchored receptor-binding domains of SARS-CoV-2 mutants induce strong humoral responses and can overcome immune imprinting</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560777" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560777" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Al-Wassiti, H. A.; Fabb, S. A.; Grimley, S. L.; Kochappan, R.; Ho, J. K.; Wong, C. Y.; Tan, C. W.; Payne, T. J.; Takanashi, A.; Sicilia, H.; Teo, S. L. Y.; McAuley, J. L.; Ellenberg, P.; Cooney, J. P.; Davidson, K. C.; Bowen, R.; Pellegrini, M.; Rockman, S.; Godfrey, D. I.; Nolan, T. M.; Wang, L.-F.; Deliyannis, G.; Purcell, D. F. J.; Pouton, C. W.</p>
        <p class="info">Score: 27.9, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560777' target='https://doi.org/10.1101/2023.10.04.560777'> 10.1101/2023.10.04.560777</a></p>
        <p class="abstract">To address the limitations of whole-spike COVID vaccines, we explored mRNA vaccines encoding membrane-anchored receptor-binding domain (RBD-TMs), each a fusion of a variant RBD, the transmembrane (TM) and cytoplasmic tail (CT) fragments of the SARS-CoV-2 spike protein. In naive mice, RBD-TM mRNA vaccines against ancestral SARS-CoV-2, Beta, Delta, Delta-plus, Kappa, Omicron BA.1 or BA.5, all induced strong humoral responses against the target RBD. Multiplex surrogate viral neutralization (sVNT) assays indicated broad neutralizing activity against a range of variant RBDs. In the setting of a heterologous boost, against the background of exposure to ancestral whole spike vaccines, sVNT studies suggested that RBD-TM vaccines were able to overcome the detrimental effects of immune imprinting. Omicron BA.1 and BA.5 RBD-TM booster vaccines induced serum antibodies with 12 and 22-fold higher neutralizing activity against the target RBD than their equivalent whole spike variants. Boosting with BA.1 or BA.5 RBD-TM provided good protection against more recent variants including XBB and XBB.1.5. Each RBD-TM mRNA is 28% of the length of its whole-spike equivalent. This advantage will enable tetravalent mRNA vaccines to be developed at well-tolerated doses of formulated mRNA.

One Sentence SummarymRNA vaccines encoding membrane-anchored RBDs of SARS-CoV-2 mutants are effective vaccines that can overcome immune imprinting in mice</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
