<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>immunology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>immunology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Fast evolution of SARS-CoV-2 BA.2.86 to JN.1 under heavy immune pressure
Authors: Yang, S.; Yu, Y.; Xu, Y.; Jian, F.; Song, W.; Yisimayi, A.; Wang, P.; Wang, J.; Liu, J.; Yu, L.; Niu, X.; Wang, J.; Wang, Y.; Shao, F.; Jin, R.; Wang, Y.; Cao, Y.
Score: 220.2, Published: 2023-11-17 DOI: 10.1101/2023.11.13.566860
While the BA.2.86 variant demonstrated significant antigenic drift and enhanced ACE2 binding affinity, its ability to evade humoral immunity was relatively moderate compared to dominant strains like EG.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/immunology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="immunology" />
<meta property="og:description" content="Fast evolution of SARS-CoV-2 BA.2.86 to JN.1 under heavy immune pressure
Authors: Yang, S.; Yu, Y.; Xu, Y.; Jian, F.; Song, W.; Yisimayi, A.; Wang, P.; Wang, J.; Liu, J.; Yu, L.; Niu, X.; Wang, J.; Wang, Y.; Shao, F.; Jin, R.; Wang, Y.; Cao, Y.
Score: 220.2, Published: 2023-11-17 DOI: 10.1101/2023.11.13.566860
While the BA.2.86 variant demonstrated significant antigenic drift and enhanced ACE2 binding affinity, its ability to evade humoral immunity was relatively moderate compared to dominant strains like EG." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/immunology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-22T10:41:31+00:00" />
<meta property="article:modified_time" content="2023-11-22T10:41:31+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="immunology"/>
<meta name="twitter:description" content="Fast evolution of SARS-CoV-2 BA.2.86 to JN.1 under heavy immune pressure
Authors: Yang, S.; Yu, Y.; Xu, Y.; Jian, F.; Song, W.; Yisimayi, A.; Wang, P.; Wang, J.; Liu, J.; Yu, L.; Niu, X.; Wang, J.; Wang, Y.; Shao, F.; Jin, R.; Wang, Y.; Cao, Y.
Score: 220.2, Published: 2023-11-17 DOI: 10.1101/2023.11.13.566860
While the BA.2.86 variant demonstrated significant antigenic drift and enhanced ACE2 binding affinity, its ability to evade humoral immunity was relatively moderate compared to dominant strains like EG."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "immunology",
      "item": "https://trxiv.yorks0n.com/posts/immunology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "immunology",
  "name": "immunology",
  "description": "Fast evolution of SARS-CoV-2 BA.2.86 to JN.1 under heavy immune pressure\nAuthors: Yang, S.; Yu, Y.; Xu, Y.; Jian, F.; Song, W.; Yisimayi, A.; Wang, P.; Wang, J.; Liu, J.; Yu, L.; Niu, X.; Wang, J.; Wang, Y.; Shao, F.; Jin, R.; Wang, Y.; Cao, Y.\nScore: 220.2, Published: 2023-11-17 DOI: 10.1101/2023.11.13.566860\nWhile the BA.2.86 variant demonstrated significant antigenic drift and enhanced ACE2 binding affinity, its ability to evade humoral immunity was relatively moderate compared to dominant strains like EG.",
  "keywords": [
    
  ],
  "articleBody": " Fast evolution of SARS-CoV-2 BA.2.86 to JN.1 under heavy immune pressure\nAuthors: Yang, S.; Yu, Y.; Xu, Y.; Jian, F.; Song, W.; Yisimayi, A.; Wang, P.; Wang, J.; Liu, J.; Yu, L.; Niu, X.; Wang, J.; Wang, Y.; Shao, F.; Jin, R.; Wang, Y.; Cao, Y.\nScore: 220.2, Published: 2023-11-17 DOI: 10.1101/2023.11.13.566860\nWhile the BA.2.86 variant demonstrated significant antigenic drift and enhanced ACE2 binding affinity, its ability to evade humoral immunity was relatively moderate compared to dominant strains like EG.5 and HK.3. However, the emergence of a new subvariant, JN.1 (BA.2.86.1.1), which possesses an additional spike mutation, L455S, compared to BA.2.86, showed a markedly increased prevalence in Europe and North America, especially in France. Here, we found that L455S of JN.1 significantly enhances immune evasion capabilities at the expense of reduced ACE2 binding affinity. This mutation enables JN.1 to effectively evade Class 1 neutralizing antibodies, offsetting BA.2.86s susceptibility and thus allowing it to outcompete both its precursor BA.2.86 and the prevailing variants HV.1 (XBB.1.5+L452R+F456L) and JD.1.1 (XBB.1.5+L455F+F456L+A475V) in terms of humoral immune evasion. The rapid evolution from BA.2.86 to JN.1, similar to the earlier transition from BA.2.75 to CH.1.1, highlights the importance of closely monitoring strains with high ACE2 binding affinity and distinct antigenicity, despite their unremarkable immune evasion capabilities. Such strains have the potential to quickly accumulate mutations that enhance their immune escape during transmission, often at the cost of receptor binding.\nSARS-CoV-2 and its ORF3a, E and M viroporins activate inflammasome in human macrophages and induce of IL-1Î± in pulmonary epithelial and endothelial cells\nAuthors: Abrozek-Latecka, M.; Kozlowski, P.; Hoser, G.; Bandyszewska, M.; Hanusek, K.; Nowis, D.; Golab, J.; Grzanka, M.; Piekielko-Witkowska, A.; Schulz, L.; Hornung, F.; Deinhardt-Emmer, S.; Kozlowska, E.; Skirecki, T.\nScore: 33.4, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566917\nInflammasome assembly is a potent mechanism responsible for the host protection against pathogens, including viruses. When compromised, it can allow viral replication, while when disrupted, it can perpetuate pathological responses by IL-1 signaling and pyroptotic cell death. SARS-CoV-2 infection was shown to activate inflammasome in the lungs of COVID-19 patients, however, potential mechanisms responsible for this response are not fully elucidated. In this study, we investigated the effects of ORF3a, E and M SARS-CoV-2 viroporins in the inflammasome activation in major populations of alveolar sentinel cells: macrophages, epithelial and endothelial cells. We demonstrated that each viroporin is capable of activation of the inflammasome in macrophages to trigger cell death and IL-1 release from epithelial and endothelial cells. Small molecule NLRP3 inflammasome inhibitors reduced IL-1 release but weakly affected the pyroptosis. Importantly, we discovered that while SARS-CoV-2 could not infect the pulmonary microvascular endothelial cells it induced IL-1 and IL-33 release. Together, these findings highlight the essential role of macrophages as the major inflammasome-activating cell population in the lungs and point to endothelial cell expressed IL-1 as a potential novel component driving the pulmonary immunothromobosis in COVID-19.\nMonocyte-derived microglia with Dnmt3a mutation cause motor pathology in aging mice\nAuthors: Kim, J.-S.; Trzebanski, S.; Shin, S.-H.; Ilani, N. C.; Kaushansky, N.; Scheller, M.; Solomon, A.; Liu, Z.; Aust, O.; Boura-Halfon, S.; Amann, L.; Prinz, M.; Ginhoux, F.; Avraham, R.; Mueller-Tidow, C.; Uderhardt, S.; Milenkovic, I.; Shlush, L.; Jung, S.\nScore: 19.5, Published: 2023-11-17 DOI: 10.1101/2023.11.16.567402\nMicroglia are established in embryogenesis forming a self-containing cellular compartment resisting seeding with cells derived from adult definitive hematopoiesis. We report that monocyte-derived macrophages (MoM{Phi}) accumulate in the brain of aging mice with distinct topology, including the nigrostriatum and medulla, but not the frontal cortex. Parenchymal MoM{Phi} adopt bona fide microglia expression profiles. Unlike microglia, these monocyte-derived microglia (MoMg) are due to their hematopoietic origin targets of clonal hematopoiesis (CH). Using a chimeric transfer model, we show that hematopoietic expression of DNMT3AR822H, a prominent mutation in human CH, renders MoMg pathogenic promoting motor deficits resembling atypical Parkinsonian disorders. Collectively, these data establish in a mouse model that MoMg progressively seed the brains of aging healthy mice, accumulate in selected areas, and, when carrying a somatic mutation associated with CH, can contribute to brain pathology.\nRobust detection of SARS-CoV-2 exposure in population using T-cell repertoire profiling\nAuthors: Vlasova, E. K.; Nekrasova, A. I.; Komkov, A. Y.; Izraelson, M.; Snigir, E. A.; Mitrofanov, S. I.; Yudin, V. S.; Makarov, V. V.; Kesnikov, A. A.; Kirienko, D.; Pivnyuk, A.; Shelyakin, P. V.; Mamedov, I. Z.; Rebrikov, D. V.; Chudakov, D. M.; Yudin, S. M.; Skvortsova, V. I.; Britanova, O.; Shugay, M. A.\nScore: 13.4, Published: 2023-11-09 DOI: 10.1101/2023.11.08.566227\nThe COVID-19 pandemic clearly demonstrates the need to monitor the spread of infectious diseases and population immunity. Probing adaptive immunity by sequencing the repertoire of antigen receptors (Rep-Seq) encoding specificity and immunological memory has become a method of choice for immunology studies. Rep-Seq can detect the imprint of past and ongoing infections and study individual responses to SARS-CoV-2 as shown in a number of recent studies. Here we apply a machine learning approach to two large datasets with more than 1200 high-quality repertoires from healthy and COVID-19-convalescent donor repertoires to infer T-cell receptor (TCR) repertoire features that were induced by SARS-CoV-2 exposure. Proper standardization of Rep-Seq batches, access to human leukocyte antigen (HLA) typing and both - and {beta}-chain sequences of TCRs allowed us to generate a high-quality biomarker database and build a robust and highly accurate classifier for COVID-19 exposure applicable to individual TCR repertoires obtained using different protocols, paving a way to Rep-Seq-based immune status assessment in large cohorts of donors.\nIntegration of Î¶-deficient CARs into the CD3-zeta gene conveys potent cytotoxicity in T and NK cells\nAuthors: Kath, J.; Franke, C.; Drosdek, V.; Du, W.; Glaser, V.; Fuster-Garcia, C.; Stein, M.; Zittel, T.; Schulenberg, S.; Porter, C. E.; Andersch, L.; Kuenkele, A.; Alcaniz, J.; Hoffmann, J.; Abken, H.; Abou-el-Enein, M.; Pruss, A.; Suzuki, M.; Cathomen, T.; Stripecke, R.; Volk, H.-D.; Reinke, P.; Schmueck-Henneresse, M.; Wagner, D. L.\nScore: 11.9, Published: 2023-11-14 DOI: 10.1101/2023.11.10.565518\nI.Chimeric antigen receptor (CAR)-reprogrammed immune cells hold significant therapeutic potential for oncology, autoimmune diseases, transplant medicine, and infections. All approved CAR-T therapies rely on personalized manufacturing using undirected viral gene transfer, which results in non-physiological regulation of CAR-signaling and limits their accessibility due to logistical challenges, high costs and biosafety requirements. Here, we propose a novel approach utilizing CRISPR-Cas gene editing to redirect T cells and natural killer (NK) cells with CARs. By transferring shorter, truncated CAR-transgenes lacking a main activation domain into the human CD3{zeta} (CD247) gene, functional CAR fusion-genes are generated that exploit the endogenous CD3{zeta} gene as the CARs activation domain. Repurposing this T/NK-cell lineage gene facilitated physiological regulation of CAR-expression and reprogramming of various immune cell types, including conventional T cells, TCR{gamma}/{delta} T cells, regulatory T cells, and NK cells. In T cells, CD3{zeta} in-frame fusion eliminated TCR surface expression, reducing the risk of graft-versus-host disease in allogeneic off-the-shelf settings. CD3{zeta}-CD19-CAR-T cells exhibited comparable leukemia control to T cell receptor alpha constant (TRAC)-replaced and lentivirus-transduced CAR-T cells in vivo. Tuning of CD3{zeta}-CAR-expression levels significantly improved the in vivo efficacy. Compared to TRAC-edited CAR-T cells, integration of a Her2-CAR into CD3{zeta} conveyed similar in vitro tumor lysis but reduced susceptibility to activation-induced cell death and differentiation, presumably due to lower CAR-expression levels. Notably, CD3{zeta} gene editing enabled reprogramming of NK cells without impairing their canonical functions. Thus, CD3{zeta} gene editing is a promising platform for the development of allogeneic off-the-shelf cell therapies using redirected killer lymphocytes. Key pointsO_LIIntegration of {zeta}-deficient CARs into CD3{zeta} gene allows generation of functional TCR-ablated CAR-T cells for allogeneic off-the-shelf use C_LIO_LICD3{zeta}-editing platform allows CAR reprogramming of NK cells without affecting their canonical functions C_LI\nIdentification of mouse CD4+ T cell epitopes in SARS-CoV-2 BA.1 spike and nucleocapsid for use in peptide:MHCII tetramers\nAuthors: Bricio Moreno, L.; Barreto de Albuquerque, J.; Neary, J. M.; Nguyen, T.; Hastie, K. M.; Landeras-Bueno, S.; Hariharan, C.; Nathan, A.; Getz, M. A.; Gayton, A. C.; Khatri, A.; Gaiha, G. D.; Saphire, E. O.; Luster, A. D.; Moon, J. J.\nScore: 11.4, Published: 2023-11-17 DOI: 10.1101/2023.11.16.566918\nUnderstanding adaptive immunity against SARS-CoV-2 is a major requisite for the development of effective vaccines and treatments for COVID-19. CD4+ T cells play an integral role in this process primarily by generating antiviral cytokines and providing help to antibody-producing B cells. To empower detailed studies of SARS-CoV-2-specific CD4+ T cell responses in mouse models, we comprehensively mapped I-Ab-restricted epitopes for the spike and nucleocapsid proteins of the BA.1 variant of concern via IFN{gamma} ELISpot assay. This was followed by the generation of corresponding peptide:MHCII tetramer reagents to directly stain epitope-specific T cells. Using this rigorous validation strategy, we identified 6 reliably immunogenic epitopes in spike and 3 in nucleocapsid, all of which are conserved in the ancestral Wuhan strain. We also validated a previously identified epitope from Wuhan that is absent in BA.1. These epitopes and tetramers will be invaluable tools for SARS-CoV-2 antigen-specific CD4+ T cell studies in mice.\nSpatially resolved single-cell atlas of the lung in fatal Covid19 in an African population reveals a distinct cellular signature and an interferon gamma dominated response\nAuthors: Nyirenda, J.; Hardy, O.; Silva Filho, J. L.; Herder, V.; Attipa, C.; Ndovi, C.; Siwombo, M.; Namalima, T.; Suwedi, L.; Nyasulu, W.; Ngulube, T.; Nyirenda, D.; Mvaya, L.; Phiri, J.; Chasweka, D.; Eneya, C.; Makwinja, C.; Phiri, C.; Ziwoya, F.; Tembo, A.; Makwangwala, K.; Khoswe, S.; Banda, P.; Morton, B.; Hilton, O.; Lawrence, S.; Frere dos Reis, M.; Cardoso Melo, G.; Vinicious Guimaros de Lecerda, M.; Trindande Maranhao Costa, F.; Marcelo Monteiro, W.; Carlos de Lima Fereirra, L.; Johnson, C.; Mcguinness, D.; Jambo, K.; Haley, M. J.; Kumwenda, B.; Palmarini, M.; Barnes, K. G.; Denno, D. M.; Vo\nScore: 6.6, Published: 2023-11-17 DOI: 10.1101/2023.11.16.566964\nPostmortem single-cell studies have transformed understanding of lower respiratory tract diseases (LRTD) including Covid19 but there is almost no data from African settings where HIV, malaria and other environmental exposures may affect disease pathobiology and treatment targets. We used histology and high-dimensional imaging to characterise fatal lung disease in Malawian adults with (n=9) and without (n=7) Covid19, and generated single-cell transcriptomics data from lung, blood and nasal cells. Data integration with other cohorts showed a conserved Covid19 histopathological signature, driven by contrasting immune and inflammatory mechanisms: in the Malawi cohort, by response to interferon-gamma in lung-resident alveolar macrophages, in USA, European and Asian cohorts by type I/III interferon responses, particularly in blood-derived monocytes. HIV status had minimal impact on histology or immunopathology. Our study provides data resources and highlights the importance of studying the cellular mechanisms of disease in underrepresented populations, indicating shared and distinct targets for treatment.\nPrenatal inflammation reprograms hyperactive ILC2s that promote allergic lung inflammation and airway dysfunction\nAuthors: Lopez, D. A.; Griffin, A.; Aguilar, L. M.; Deering-Rice, C.; Myers, E. J.; Warren, K. J.; Welner, R. S.; Beaudin, A. E.\nScore: 5.8, Published: 2023-11-21 DOI: 10.1101/2023.11.20.567899\nAllergic asthma is a chronic respiratory disease that initiates in early life, but causal mechanisms are poorly understood. Here we examined how prenatal inflammation shapes allergic asthma susceptibility by reprogramming lung immunity from early development. Induction of Type I interferon-mediated inflammation during development provoked expansion and hyperactivation of group 2 innate lymphoid cells (ILC2s) seeding the developing lung. Hyperactivated ILC2s produced increased IL-5 and IL-13, and were associated with acute Th2 bias, eosinophilia, and decreased Tregs in the lung. The hyperactive ILC2 phenotype was recapitulated by adoptive transfer of a fetal liver precursor following exposure to prenatal inflammation, indicative of developmental programming. Programming of ILC2 function and subsequent lung immune remodeling by prenatal inflammation led to airway dysfunction at baseline and in response to papain, indicating increased asthma susceptibility. Our data provide a link by which developmental programming of progenitors by early-life inflammation drives lung immune remodeling and asthma susceptibility through hyperactivation of lung-resident ILC2s.\nImproving influenza A vaccine strain selection through deep evolutionary models\nAuthors: Shi, W.; Wohlwend, J.; Wu, M.; Barzilay, R.\nScore: 5.1, Published: 2023-11-16 DOI: 10.1101/2023.11.14.567037\nEven though vaccines have the potential to significantly alleviate the disease burden of epidemics such as the seasonal flu, current influenza vaccines offer limited protection. According to the Centers for Disease Control and Prevention (CDC), vaccine effectiveness has hovered below 50% for the past decade. Identifying the optimal strains to use in a vaccine is central to increasing its efficacy. However, this task is challenging due to the antigenic drift that occurs during the flu season. In this paper, we propose to select vaccines based on their escapability score, a metric that quantifies the antigenic similarity of vaccine strains with future dominant strains and demonstrates a strong correlation with clinical vaccine effectiveness. We introduce a deep learning-based approach that predicts both the antigenic properties of vaccine strains and the dominance of future circulating viruses, enabling efficient virtual screening of a large number of vaccine compositions. We utilized historical antigenic analysis data from the World Health Organization (WHO) to demonstrate that our model selects vaccine strains that reliably improve over the recommended ones.\nThe helminth TGF-beta mimic TGM4 is a modular ligand that binds CD44, CD49d and TGF-beta receptors to preferentially target myeloid cells\nAuthors: Singh, S.; Smyth, D. J.; Cunningham, K.; Mukundan, A.; Byron, C.-H.; Hinck, C. S.; White, M. P. J.; Ciancia, C.; Wosowska, N.; Sanders, A.; Jin, R.; Lilla, S.; Zanivan, S.; Schoenherr, C.; Inman, G.; van Dinther, M. A. H.; ten Dijke, P.; Hinck, A. P.; Maizels, R. M.\nScore: 5.0, Published: 2023-11-15 DOI: 10.1101/2023.11.13.566701\nThe murine helminth parasite Heligmosomoides polygyrus expresses a family of modular proteins which, replicating the functional activity of the immunomodulatory cytokine TGF-{beta}, have been named TGM (TGF-{beta} Mimic). Multiple domains bind to different receptors, including TGF-{beta} receptors T{beta}RI (ALK5) and T{beta}RII through domains 1-3, and prototypic family member TGM1 binds the cell surface co-receptor CD44 through domains 4-5. This allows TGM1 to induce T lymphocyte Foxp3 expression, characteristic of regulatory (Treg) cells, and to activate a range of TGF-{beta}-responsive cell types. In contrast, a related protein, TGM4, targets a much more restricted cell repertoire, primarily acting on myeloid cells, with less potent effects on T cells and lacking activity on other TGF-{beta}-responsive cell types. TGM4 binds avidly to myeloid cells by flow cytometry, and can outcompete TGM1 for cell binding. Analysis of receptor binding in comparison to TGM1 reveals a 10-fold higher affinity than TGM1 for TGF{beta}R-I (T{beta}RI), but a 100-fold lower affinity for T{beta}RII through Domain 3. Consequently, TGM4 is more dependent on co-receptor binding; in addition to CD44, TGM4 also engages CD49d (Itga4) through Domains 1-3, as well as CD206 and Neuropilin-1 through Domains 4 and 5. TGM4 was found to effectively modulate macrophage populations, inhibiting lipopolysaccharide-driven inflammatory cytokine production and boosting interleukin (IL)-4-stimulated responses such as Arginase-1 in vitro and in vivo. These results reveal that the modular nature of TGMs has allowed the fine tuning of the binding affinities of the T{beta}R- and co-receptor binding domains to establish cell specificity for TGF-{beta} signalling in a manner that cannot be attained by the mammalian cytokine.\n",
  "wordCount" : "2479",
  "inLanguage": "en",
  "datePublished": "2023-11-22T10:41:31Z",
  "dateModified": "2023-11-22T10:41:31Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/immunology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      immunology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 22, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566860">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566860" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566860">
        <p class="paperTitle">Fast evolution of SARS-CoV-2 BA.2.86 to JN.1 under heavy immune pressure</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566860" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566860" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, S.; Yu, Y.; Xu, Y.; Jian, F.; Song, W.; Yisimayi, A.; Wang, P.; Wang, J.; Liu, J.; Yu, L.; Niu, X.; Wang, J.; Wang, Y.; Shao, F.; Jin, R.; Wang, Y.; Cao, Y.</p>
        <p class="info">Score: 220.2, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566860' target='https://doi.org/10.1101/2023.11.13.566860'> 10.1101/2023.11.13.566860</a></p>
        <p class="abstract">While the BA.2.86 variant demonstrated significant antigenic drift and enhanced ACE2 binding affinity, its ability to evade humoral immunity was relatively moderate compared to dominant strains like EG.5 and HK.3. However, the emergence of a new subvariant, JN.1 (BA.2.86.1.1), which possesses an additional spike mutation, L455S, compared to BA.2.86, showed a markedly increased prevalence in Europe and North America, especially in France. Here, we found that L455S of JN.1 significantly enhances immune evasion capabilities at the expense of reduced ACE2 binding affinity. This mutation enables JN.1 to effectively evade Class 1 neutralizing antibodies, offsetting BA.2.86s susceptibility and thus allowing it to outcompete both its precursor BA.2.86 and the prevailing variants HV.1 (XBB.1.5&#43;L452R&#43;F456L) and JD.1.1 (XBB.1.5&#43;L455F&#43;F456L&#43;A475V) in terms of humoral immune evasion. The rapid evolution from BA.2.86 to JN.1, similar to the earlier transition from BA.2.75 to CH.1.1, highlights the importance of closely monitoring strains with high ACE2 binding affinity and distinct antigenicity, despite their unremarkable immune evasion capabilities. Such strains have the potential to quickly accumulate mutations that enhance their immune escape during transmission, often at the cost of receptor binding.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566917">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566917" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566917">
        <p class="paperTitle">SARS-CoV-2 and its ORF3a, E and M viroporins activate inflammasome in human macrophages and induce of IL-1Î± in pulmonary epithelial and endothelial cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566917" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566917" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abrozek-Latecka, M.; Kozlowski, P.; Hoser, G.; Bandyszewska, M.; Hanusek, K.; Nowis, D.; Golab, J.; Grzanka, M.; Piekielko-Witkowska, A.; Schulz, L.; Hornung, F.; Deinhardt-Emmer, S.; Kozlowska, E.; Skirecki, T.</p>
        <p class="info">Score: 33.4, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566917' target='https://doi.org/10.1101/2023.11.13.566917'> 10.1101/2023.11.13.566917</a></p>
        <p class="abstract">Inflammasome assembly is a potent mechanism responsible for the host protection against pathogens, including viruses. When compromised, it can allow viral replication, while when disrupted, it can perpetuate pathological responses by IL-1 signaling and pyroptotic cell death. SARS-CoV-2 infection was shown to activate inflammasome in the lungs of COVID-19 patients, however, potential mechanisms responsible for this response are not fully elucidated. In this study, we investigated the effects of ORF3a, E and M SARS-CoV-2 viroporins in the inflammasome activation in major populations of alveolar sentinel cells: macrophages, epithelial and endothelial cells. We demonstrated that each viroporin is capable of activation of the inflammasome in macrophages to trigger cell death and IL-1 release from epithelial and endothelial cells. Small molecule NLRP3 inflammasome inhibitors reduced IL-1 release but weakly affected the pyroptosis. Importantly, we discovered that while SARS-CoV-2 could not infect the pulmonary microvascular endothelial cells it induced IL-1 and IL-33 release. Together, these findings highlight the essential role of macrophages as the major inflammasome-activating cell population in the lungs and point to endothelial cell expressed IL-1 as a potential novel component driving the pulmonary immunothromobosis in COVID-19.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567402">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567402" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567402">
        <p class="paperTitle">Monocyte-derived microglia with Dnmt3a mutation cause motor pathology in aging mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567402" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567402" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, J.-S.; Trzebanski, S.; Shin, S.-H.; Ilani, N. C.; Kaushansky, N.; Scheller, M.; Solomon, A.; Liu, Z.; Aust, O.; Boura-Halfon, S.; Amann, L.; Prinz, M.; Ginhoux, F.; Avraham, R.; Mueller-Tidow, C.; Uderhardt, S.; Milenkovic, I.; Shlush, L.; Jung, S.</p>
        <p class="info">Score: 19.5, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567402' target='https://doi.org/10.1101/2023.11.16.567402'> 10.1101/2023.11.16.567402</a></p>
        <p class="abstract">Microglia are established in embryogenesis forming a self-containing cellular compartment resisting seeding with cells derived from adult definitive hematopoiesis. We report that monocyte-derived macrophages (MoM{Phi}) accumulate in the brain of aging mice with distinct topology, including the nigrostriatum and medulla, but not the frontal cortex. Parenchymal MoM{Phi} adopt bona fide microglia expression profiles. Unlike microglia, these monocyte-derived microglia (MoMg) are due to their hematopoietic origin targets of clonal hematopoiesis (CH). Using a chimeric transfer model, we show that hematopoietic expression of DNMT3AR822H, a prominent mutation in human CH, renders MoMg pathogenic promoting motor deficits resembling atypical Parkinsonian disorders. Collectively, these data establish in a mouse model that MoMg progressively seed the brains of aging healthy mice, accumulate in selected areas, and, when carrying a somatic mutation associated with CH, can contribute to brain pathology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566227">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566227" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566227">
        <p class="paperTitle">Robust detection of SARS-CoV-2 exposure in population using T-cell repertoire profiling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566227" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566227" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vlasova, E. K.; Nekrasova, A. I.; Komkov, A. Y.; Izraelson, M.; Snigir, E. A.; Mitrofanov, S. I.; Yudin, V. S.; Makarov, V. V.; Kesnikov, A. A.; Kirienko, D.; Pivnyuk, A.; Shelyakin, P. V.; Mamedov, I. Z.; Rebrikov, D. V.; Chudakov, D. M.; Yudin, S. M.; Skvortsova, V. I.; Britanova, O.; Shugay, M. A.</p>
        <p class="info">Score: 13.4, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566227' target='https://doi.org/10.1101/2023.11.08.566227'> 10.1101/2023.11.08.566227</a></p>
        <p class="abstract">The COVID-19 pandemic clearly demonstrates the need to monitor the spread of infectious diseases and population immunity. Probing adaptive immunity by sequencing the repertoire of antigen receptors (Rep-Seq) encoding specificity and immunological memory has become a method of choice for immunology studies. Rep-Seq can detect the imprint of past and ongoing infections and study individual responses to SARS-CoV-2 as shown in a number of recent studies. Here we apply a machine learning approach to two large datasets with more than 1200 high-quality repertoires from healthy and COVID-19-convalescent donor repertoires to infer T-cell receptor (TCR) repertoire features that were induced by SARS-CoV-2 exposure. Proper standardization of Rep-Seq batches, access to human leukocyte antigen (HLA) typing and both - and {beta}-chain sequences of TCRs allowed us to generate a high-quality biomarker database and build a robust and highly accurate classifier for COVID-19 exposure applicable to individual TCR repertoires obtained using different protocols, paving a way to Rep-Seq-based immune status assessment in large cohorts of donors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.565518">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.565518" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.565518">
        <p class="paperTitle">Integration of Î¶-deficient CARs into the CD3-zeta gene conveys potent cytotoxicity in T and NK cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.565518" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.565518" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kath, J.; Franke, C.; Drosdek, V.; Du, W.; Glaser, V.; Fuster-Garcia, C.; Stein, M.; Zittel, T.; Schulenberg, S.; Porter, C. E.; Andersch, L.; Kuenkele, A.; Alcaniz, J.; Hoffmann, J.; Abken, H.; Abou-el-Enein, M.; Pruss, A.; Suzuki, M.; Cathomen, T.; Stripecke, R.; Volk, H.-D.; Reinke, P.; Schmueck-Henneresse, M.; Wagner, D. L.</p>
        <p class="info">Score: 11.9, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.565518' target='https://doi.org/10.1101/2023.11.10.565518'> 10.1101/2023.11.10.565518</a></p>
        <p class="abstract">I.Chimeric antigen receptor (CAR)-reprogrammed immune cells hold significant therapeutic potential for oncology, autoimmune diseases, transplant medicine, and infections. All approved CAR-T therapies rely on personalized manufacturing using undirected viral gene transfer, which results in non-physiological regulation of CAR-signaling and limits their accessibility due to logistical challenges, high costs and biosafety requirements. Here, we propose a novel approach utilizing CRISPR-Cas gene editing to redirect T cells and natural killer (NK) cells with CARs. By transferring shorter, truncated CAR-transgenes lacking a main activation domain into the human CD3{zeta} (CD247) gene, functional CAR fusion-genes are generated that exploit the endogenous CD3{zeta} gene as the CARs activation domain. Repurposing this T/NK-cell lineage gene facilitated physiological regulation of CAR-expression and reprogramming of various immune cell types, including conventional T cells, TCR{gamma}/{delta} T cells, regulatory T cells, and NK cells. In T cells, CD3{zeta} in-frame fusion eliminated TCR surface expression, reducing the risk of graft-versus-host disease in allogeneic off-the-shelf settings. CD3{zeta}-CD19-CAR-T cells exhibited comparable leukemia control to T cell receptor alpha constant (TRAC)-replaced and lentivirus-transduced CAR-T cells in vivo. Tuning of CD3{zeta}-CAR-expression levels significantly improved the in vivo efficacy. Compared to TRAC-edited CAR-T cells, integration of a Her2-CAR into CD3{zeta} conveyed similar in vitro tumor lysis but reduced susceptibility to activation-induced cell death and differentiation, presumably due to lower CAR-expression levels. Notably, CD3{zeta} gene editing enabled reprogramming of NK cells without impairing their canonical functions. Thus, CD3{zeta} gene editing is a promising platform for the development of allogeneic off-the-shelf cell therapies using redirected killer lymphocytes.

Key pointsO_LIIntegration of {zeta}-deficient CARs into CD3{zeta} gene allows generation of functional TCR-ablated CAR-T cells for allogeneic off-the-shelf use
C_LIO_LICD3{zeta}-editing platform allows CAR reprogramming of NK cells without affecting their canonical functions
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.566918">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.566918" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.566918">
        <p class="paperTitle">Identification of mouse CD4&#43; T cell epitopes in SARS-CoV-2 BA.1 spike and nucleocapsid for use in peptide:MHCII tetramers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.566918" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.566918" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bricio Moreno, L.; Barreto de Albuquerque, J.; Neary, J. M.; Nguyen, T.; Hastie, K. M.; Landeras-Bueno, S.; Hariharan, C.; Nathan, A.; Getz, M. A.; Gayton, A. C.; Khatri, A.; Gaiha, G. D.; Saphire, E. O.; Luster, A. D.; Moon, J. J.</p>
        <p class="info">Score: 11.4, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.566918' target='https://doi.org/10.1101/2023.11.16.566918'> 10.1101/2023.11.16.566918</a></p>
        <p class="abstract">Understanding adaptive immunity against SARS-CoV-2 is a major requisite for the development of effective vaccines and treatments for COVID-19. CD4&#43; T cells play an integral role in this process primarily by generating antiviral cytokines and providing help to antibody-producing B cells. To empower detailed studies of SARS-CoV-2-specific CD4&#43; T cell responses in mouse models, we comprehensively mapped I-Ab-restricted epitopes for the spike and nucleocapsid proteins of the BA.1 variant of concern via IFN{gamma} ELISpot assay. This was followed by the generation of corresponding peptide:MHCII tetramer reagents to directly stain epitope-specific T cells. Using this rigorous validation strategy, we identified 6 reliably immunogenic epitopes in spike and 3 in nucleocapsid, all of which are conserved in the ancestral Wuhan strain. We also validated a previously identified epitope from Wuhan that is absent in BA.1. These epitopes and tetramers will be invaluable tools for SARS-CoV-2 antigen-specific CD4&#43; T cell studies in mice.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.566964">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.566964" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.566964">
        <p class="paperTitle">Spatially resolved single-cell atlas of the lung in fatal Covid19 in an African population reveals a distinct cellular signature and an interferon gamma dominated response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.566964" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.566964" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nyirenda, J.; Hardy, O.; Silva Filho, J. L.; Herder, V.; Attipa, C.; Ndovi, C.; Siwombo, M.; Namalima, T.; Suwedi, L.; Nyasulu, W.; Ngulube, T.; Nyirenda, D.; Mvaya, L.; Phiri, J.; Chasweka, D.; Eneya, C.; Makwinja, C.; Phiri, C.; Ziwoya, F.; Tembo, A.; Makwangwala, K.; Khoswe, S.; Banda, P.; Morton, B.; Hilton, O.; Lawrence, S.; Frere dos Reis, M.; Cardoso Melo, G.; Vinicious Guimaros de Lecerda, M.; Trindande Maranhao Costa, F.; Marcelo Monteiro, W.; Carlos de Lima Fereirra, L.; Johnson, C.; Mcguinness, D.; Jambo, K.; Haley, M. J.; Kumwenda, B.; Palmarini, M.; Barnes, K. G.; Denno, D. M.; Vo</p>
        <p class="info">Score: 6.6, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.566964' target='https://doi.org/10.1101/2023.11.16.566964'> 10.1101/2023.11.16.566964</a></p>
        <p class="abstract">Postmortem single-cell studies have transformed understanding of lower respiratory tract diseases (LRTD) including Covid19 but there is almost no data from African settings where HIV, malaria and other environmental exposures may affect disease pathobiology and treatment targets. We used histology and high-dimensional imaging to characterise fatal lung disease in Malawian adults with (n=9) and without (n=7) Covid19, and generated single-cell transcriptomics data from lung, blood and nasal cells. Data integration with other cohorts showed a conserved Covid19 histopathological signature, driven by contrasting immune and inflammatory mechanisms: in the Malawi cohort, by response to interferon-gamma in lung-resident alveolar macrophages, in USA, European and Asian cohorts by type I/III interferon responses, particularly in blood-derived monocytes. HIV status had minimal impact on histology or immunopathology. Our study provides data resources and highlights the importance of studying the cellular mechanisms of disease in underrepresented populations, indicating shared and distinct targets for treatment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567899">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567899" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567899">
        <p class="paperTitle">Prenatal inflammation reprograms hyperactive ILC2s that promote allergic lung inflammation and airway dysfunction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567899" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567899" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lopez, D. A.; Griffin, A.; Aguilar, L. M.; Deering-Rice, C.; Myers, E. J.; Warren, K. J.; Welner, R. S.; Beaudin, A. E.</p>
        <p class="info">Score: 5.8, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567899' target='https://doi.org/10.1101/2023.11.20.567899'> 10.1101/2023.11.20.567899</a></p>
        <p class="abstract">Allergic asthma is a chronic respiratory disease that initiates in early life, but causal mechanisms are poorly understood. Here we examined how prenatal inflammation shapes allergic asthma susceptibility by reprogramming lung immunity from early development. Induction of Type I interferon-mediated inflammation during development provoked expansion and hyperactivation of group 2 innate lymphoid cells (ILC2s) seeding the developing lung. Hyperactivated ILC2s produced increased IL-5 and IL-13, and were associated with acute Th2 bias, eosinophilia, and decreased Tregs in the lung. The hyperactive ILC2 phenotype was recapitulated by adoptive transfer of a fetal liver precursor following exposure to prenatal inflammation, indicative of developmental programming. Programming of ILC2 function and subsequent lung immune remodeling by prenatal inflammation led to airway dysfunction at baseline and in response to papain, indicating increased asthma susceptibility. Our data provide a link by which developmental programming of progenitors by early-life inflammation drives lung immune remodeling and asthma susceptibility through hyperactivation of lung-resident ILC2s.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567037">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567037" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567037">
        <p class="paperTitle">Improving influenza A vaccine strain selection through deep evolutionary models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567037" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567037" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shi, W.; Wohlwend, J.; Wu, M.; Barzilay, R.</p>
        <p class="info">Score: 5.1, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567037' target='https://doi.org/10.1101/2023.11.14.567037'> 10.1101/2023.11.14.567037</a></p>
        <p class="abstract">Even though vaccines have the potential to significantly alleviate the disease burden of epidemics such as the seasonal flu, current influenza vaccines offer limited protection. According to the Centers for Disease Control and Prevention (CDC), vaccine effectiveness has hovered below 50% for the past decade. Identifying the optimal strains to use in a vaccine is central to increasing its efficacy. However, this task is challenging due to the antigenic drift that occurs during the flu season. In this paper, we propose to select vaccines based on their escapability score, a metric that quantifies the antigenic similarity of vaccine strains with future dominant strains and demonstrates a strong correlation with clinical vaccine effectiveness. We introduce a deep learning-based approach that predicts both the antigenic properties of vaccine strains and the dominance of future circulating viruses, enabling efficient virtual screening of a large number of vaccine compositions. We utilized historical antigenic analysis data from the World Health Organization (WHO) to demonstrate that our model selects vaccine strains that reliably improve over the recommended ones.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566701">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566701" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566701">
        <p class="paperTitle">The helminth TGF-beta mimic TGM4 is a modular ligand that binds CD44, CD49d and TGF-beta receptors to preferentially target myeloid cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566701" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566701" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Singh, S.; Smyth, D. J.; Cunningham, K.; Mukundan, A.; Byron, C.-H.; Hinck, C. S.; White, M. P. J.; Ciancia, C.; Wosowska, N.; Sanders, A.; Jin, R.; Lilla, S.; Zanivan, S.; Schoenherr, C.; Inman, G.; van Dinther, M. A. H.; ten Dijke, P.; Hinck, A. P.; Maizels, R. M.</p>
        <p class="info">Score: 5.0, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566701' target='https://doi.org/10.1101/2023.11.13.566701'> 10.1101/2023.11.13.566701</a></p>
        <p class="abstract">The murine helminth parasite Heligmosomoides polygyrus expresses a family of modular proteins which, replicating the functional activity of the immunomodulatory cytokine TGF-{beta}, have been named TGM (TGF-{beta} Mimic). Multiple domains bind to different receptors, including TGF-{beta} receptors T{beta}RI (ALK5) and T{beta}RII through domains 1-3, and prototypic family member TGM1 binds the cell surface co-receptor CD44 through domains 4-5. This allows TGM1 to induce T lymphocyte Foxp3 expression, characteristic of regulatory (Treg) cells, and to activate a range of TGF-{beta}-responsive cell types. In contrast, a related protein, TGM4, targets a much more restricted cell repertoire, primarily acting on myeloid cells, with less potent effects on T cells and lacking activity on other TGF-{beta}-responsive cell types. TGM4 binds avidly to myeloid cells by flow cytometry, and can outcompete TGM1 for cell binding. Analysis of receptor binding in comparison to TGM1 reveals a 10-fold higher affinity than TGM1 for TGF{beta}R-I (T{beta}RI), but a 100-fold lower affinity for T{beta}RII through Domain 3. Consequently, TGM4 is more dependent on co-receptor binding; in addition to CD44, TGM4 also engages CD49d (Itga4) through Domains 1-3, as well as CD206 and Neuropilin-1 through Domains 4 and 5. TGM4 was found to effectively modulate macrophage populations, inhibiting lipopolysaccharide-driven inflammatory cytokine production and boosting interleukin (IL)-4-stimulated responses such as Arginase-1 in vitro and in vivo. These results reveal that the modular nature of TGMs has allowed the fine tuning of the binding affinities of the T{beta}R- and co-receptor binding domains to establish cell specificity for TGF-{beta} signalling in a manner that cannot be attained by the mammalian cytokine.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
