<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>immunology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>immunology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="SARS-CoV-2 antibodies cross-react and enhance dengue infection
Authors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.
Score: 202.5, Published: 2023-10-10 DOI: 10.1101/2023.10.09.557914
Dengue disease is highly prevalent in tropical and subtropical regions worldwide.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/immunology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="immunology" />
<meta property="og:description" content="SARS-CoV-2 antibodies cross-react and enhance dengue infection
Authors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.
Score: 202.5, Published: 2023-10-10 DOI: 10.1101/2023.10.09.557914
Dengue disease is highly prevalent in tropical and subtropical regions worldwide." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/immunology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-18T10:38:41+00:00" />
<meta property="article:modified_time" content="2023-10-18T10:38:41+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="immunology"/>
<meta name="twitter:description" content="SARS-CoV-2 antibodies cross-react and enhance dengue infection
Authors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.
Score: 202.5, Published: 2023-10-10 DOI: 10.1101/2023.10.09.557914
Dengue disease is highly prevalent in tropical and subtropical regions worldwide."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "immunology",
      "item": "https://trxiv.yorks0n.com/posts/immunology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "immunology",
  "name": "immunology",
  "description": "SARS-CoV-2 antibodies cross-react and enhance dengue infection\nAuthors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.\nScore: 202.5, Published: 2023-10-10 DOI: 10.1101/2023.10.09.557914\nDengue disease is highly prevalent in tropical and subtropical regions worldwide.",
  "keywords": [
    
  ],
  "articleBody": " SARS-CoV-2 antibodies cross-react and enhance dengue infection\nAuthors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.\nScore: 202.5, Published: 2023-10-10 DOI: 10.1101/2023.10.09.557914\nDengue disease is highly prevalent in tropical and subtropical regions worldwide. However, its pathogenesis is still incompletely understood, particularly in comparison to other endemic viruses. Antibody-dependent enhancement (ADE) is a well-known phenomenon for dengue viruses. Given the recent surge in dengue cases and potential cross-reactivity with SARS-CoV-2 antibodies, this study explores the impact of anti-SARS-CoV-2 antibodies on DENV-2 infection. The study assessed the cross-reactivity of SARS-CoV-2 antibodies with the DENV-2 Virus. Human convalescent plasma samples collected during different waves of COVID-19 and monoclonal and polyclonal antibodies raised against SARS-CoV-2 were examined for their potential to cause ADE of DENV-2 infection using cell-based assays. The study found that anti-SARS-CoV-2 antibodies acquired from natural infection in humans or through experimental immunization in animals were cross-reactive with DENV-2 and had the potential to enhance DENV-2 infection in K562 and U937 cells. In-silico and in-vitro studies indicated a strong interaction between SARS-CoV-2 antibodies and DENV-2 E-protein, providing a molecular basis for these findings. This study is the first to demonstrate that anti-SARS-CoV-2 antibodies can cross-react with DENV-2 and can enhance its infection through ADE. These findings have implications for SARS-CoV-2 vaccine development and deployment strategies in regions where dengue is endemic. SummaryAntibodies against SARS-CoV-2 (RBD and Spike) showed significant cross reactivity with DENV-2 (E protein). Also, anti-SARS-CoV-2-commercial antibodies, immunised animal sera and 46 human convalescent plasma samples (from different waves of pandemic) demonstrated antibody-dependent enhancement (ADE) of DENV-2 infection.\nGenetic ablation of adhesion ligands averts rejection of allogeneic immune cells\nAuthors: Hammer, Q.; Perica, K.; van Ooijen, H.; Mbofung, R.; Momayyezi, P.; Varady, E.; Martin, K. E.; Pan, Y.; Jelcic, M.; Groff, B.; Abujarour, R.; Krokeide, S.; Lee, T.; Williams, A.; Goodridge, J. P.; Valamehr, B.; Onfelt, B.; Sadelain, M.; Malmberg, K.-J.\nScore: 16.8, Published: 2023-10-09 DOI: 10.1101/2023.10.09.557143\nAllogeneic cell therapies hold promise for broad clinical implementation, but face limitations due to potential rejection by the recipient immune system. Silencing of beta-2-microglobulin (B2M) expression is commonly employed to evade T cell-mediated rejection, although absence of B2M triggers missing-self responses by recipient natural killer (NK) cells. Here, we demonstrate that deletion of the adhesion ligands CD54 and CD58 on targets cells robustly dampens NK cell reactivity across all sub-populations. Genetic deletion of CD54 and CD58 in B2M-deficient allogeneic chimeric antigen receptor (CAR) T and multi-edited induced pluripotent stem cell (iPSC)-derived NK cells reduces their susceptibility to rejection by NK cells in vitro and in vivo without affecting their anti-tumor effector potential. Thus, these data suggest that genetic ablation of adhesion ligands effectively alleviates rejection of allogeneic immune cells for immunotherapy.\nHERV activation segregates ME/CFS from fibromyalgia and defines a novel nosological entity for patients fulfilling both clinical criteria\nAuthors: Gimenez-Orenga, K.; Martin-Martinez, E.; Nathanson, L.; Oltra, E.\nScore: 129.9, Published: 2023-10-06 DOI: 10.1101/2023.10.05.561025\nMyalgic Encephalomyelitis/Chronic Fatigue Syndrome (ME/CFS) and fibromyalgia (FM) are chronic diseases with poorly understood pathophysiology and diagnosis based on clinical assessment of unspecific symptoms. The recent post-COVID-19 condition, which shares similarities with ME/CFS and FM, has raised concerns about viral-induced transcriptome changes in post-viral syndromes. Viral infections, and other types of stress, are known to unleash human endogenous retroviruses (HERV) repression that if maintained could lead to symptom chronicity. This study evaluated this possibility for ME/CFS and FM on a selected cohort of female patients complying with diagnosis criteria for ME/CFS, FM, or both, and matched healthy controls (n=43). The results show specific HERV fingerprints for each disease, confirming biological differences between ME/CFS and FM. Unexpectedly, HERV profiles segregated patients that met both ME/CFS and FM clinical criteria from patients complying only with ME or FM criteria, while clearly differentiating patients from healthy subjects, supporting that the highly prevalent comorbidity condition must constitute a different nosological entity. Moreover, HERV profiles exposed significant quantitative differences within the ME/CFS group that correlated with differences in immune gene expression and patient symptomatology, supporting ME/CFS patient subtyping and confirming immunological disturbances in this disease. Pending issues include validation of HERV profiles as disease biomarkers of post-viral syndromes and understanding the role of HERV during infection and beyond.\nMast-Cell Expressed Membrane Protein-1 (MCEMP1) is expressed in classical monocytes and alveolar macrophages in Idiopathic Pulmonary Fibrosis and regulates cell chemotaxis, adhesion, and migration in a TGFβ dependent manner.\nAuthors: Perrot, C. Y.; Karampitsakos, T.; Unterman, A.; Adams, T.; Marlin, K.; Arsenault, A.; Zhao, A.; Kaminski, N.; Katlaps, G.; Patel, K.; Bandyopadhyay, D.; Herazo-Maya, J. D.\nScore: 7.8, Published: 2023-10-13 DOI: 10.1101/2023.10.07.561349\nBackgroundMast-Cell Expressed Membrane Protein-1 (MCEMP1) is higher in Idiopathic Pulmonary Fibrosis (IPF) patients with increased risk of death and poor outcomes. Here we seek to establish the mechanistic role of MCEMP1 in pulmonary fibrosis. MethodsMCEMP1 expression was analyzed by single-cell RNA sequencing, immunofluorescence in Peripheral Blood Mononuclear Cells (PBMC) as well as in lung tissues from IPF patients and controls. Chromatin Immunoprecipitation (ChiP) and Proximity Ligation Assay (PLA) were used to study the transcriptional regulation of MCEMP1. Transient RNA interference and lentivirus transduction were used to knockdown and knock-in MCEMP1 in THP-1 cells to study chemotaxis, adhesion, and migration. Bulk RNA sequencing was used to identify the mechanisms by which MCEMP1 participates in monocyte function. Active RHO pull-down assay was used to validate bulk RNA sequencing results. ResultsWe identified increased MCEMP1 expression in classical monocytes and alveolar macrophages in IPF compared to controls. MCEMP1 was upregulated by TGF{beta} at the mRNA and protein levels in THP-1. TGF{beta}-mediated MCEMP1 upregulation results from the cooperation of SMAD3 and SP1 via concomitant binding to SMAD3/SP1 cis-regulatory elements within the MCEMP1 promoter. In terms of its function, we found that MCEMP1 regulates TGF{beta}-mediated monocyte chemotaxis, adhesion, and migration. 400 differentially expressed genes were found to increase after TGF{beta} stimulation of THP-1, further increased in MCEMP1 knock-in cells treated with TGF{beta} and decreased in MCEMP1 knockdown cells treated with TGF{beta}. GO annotation analysis of these genes showed enrichment for positive regulation of RHO GTPase activity and signal transduction. While TGF{beta} enhanced RHO GTPase activity in THP-1 cells, this effect was attenuated following MCEMP1 knockdown. ConclusionMCEMP1 is highly expressed in circulating classical monocytes and alveolar macrophages in IPF. MCEMP1 is regulated by TGF{beta} and participates in the chemotaxis, adhesion, and migration of circulating monocytes by modulating the effect of TGF{beta} in RHO activity. Our results suggest that MCEMP1 may regulate the migration and transition of monocytes to monocyte-derived alveolar macrophages during pulmonary fibrosis development and progression.\nHuman effector CD8+ T cells with an exhausted-like phenotype control tumor growth in vivo\nAuthors: Mietz, J.; Kaulfuss, M.; Egli, L.; Opitz, L.; Munz, C.; Chijioke, O.\nScore: 5.1, Published: 2023-10-14 DOI: 10.1101/2023.10.11.561856\nThe functional relevance of CD8+ T cell phenotypes in human cancer is not well established. In preclinical studies, murine CD8+ T cells exhibiting an exhausted-like phenotype have been described to have drastically decreased anticancer activity while non-exhausted TCF-1+ progenitor CD8+ T cells have been thought to be mainly responsible for driving protective antitumor responses. However, human data has mostly been correlative by nature. By establishing a novel autologous humanized mouse tumor model, we found a CD8+ dominated T cell expansion upon tumor challenge and identified tumor-reactive CD8+ T cells through expression of CD137. This subpopulation of de novo arising human CD8+ T cells displayed a highly proliferative, fully activated effector and exhausted-like phenotype with enhanced expression of exhaustion markers like PD-1, CD39 and TOX, the senescence marker CD57 (B3GAT1) and cytolytic effector molecules such as PRF1, GZMH and NKG7. Moreover, these CD137+CD8+ T cells exhibited tumor-specific clonal expansion and presented signature overlap with tumor-reactive CD8+ T cells described in human cancer. We demonstrate superior anticancer activity of this exhausted-like human CD8+ T cell subset by adoptive transfer experiments using recipients bearing autologous human tumors. Mice adoptively transferred with CD137+CD8+ T cells showed reduced tumor growth, higher CD8+ T cell persistence as well as tumor infiltration, correlating with control of human tumors. In summary, defined effective anticancer activity of human effector CD8+ T cells with an exhausted-like phenotype, supporting clinical exploration of such cells in adoptive T cell therapies.\nDeep learning-guided selection of antibody therapies with enhanced resistance to current and prospective SARS-CoV-2 Omicron variants\nAuthors: Frei, L.; Gao, B.; Han, J.; Taft, J. M.; Irvine, E. B.; Weber, C. R.; Kumar, R.; Eisinger, B.; Reddy, S. T.\nScore: 5.1, Published: 2023-10-10 DOI: 10.1101/2023.10.09.561492\nMost COVID-19 antibody therapies rely on binding the SARS-CoV-2 receptor binding domain (RBD). However, heavily mutated variants such as Omicron and its sublineages, which are characterized by an ever increasing number of mutations in the RBD, have rendered prior antibody therapies ineffective, leaving no clinically approved antibody treatments for SARS-CoV-2. Therefore, the capacity of therapeutic antibody candidates to bind and neutralize current and prospective SARS-CoV-2 variants is a critical factor for drug development. Here, we present a deep learning-guided approach to identify antibodies with enhanced resistance to SARS-CoV-2 evolution. We apply deep mutational learning (DML), a machine learning-guided protein engineering method to interrogate a massive sequence space of combinatorial RBD mutations and predict their impact on angiotensin-converting enzyme 2 (ACE2) binding and antibody escape. A high mutational distance library was constructed based on the full-length RBD of Omicron BA.1, which was experimentally screened for binding to the ACE2 receptor or neutralizing antibodies, followed by deep sequencing. The resulting data was used to train ensemble deep learning models that could accurately predict binding or escape for a panel of therapeutic antibody candidates targeting diverse RBD epitopes. Furthermore, antibody breadth was assessed by predicting binding or escape to synthetic lineages that represent millions of sequences generated using in silico evolution, revealing combinations with complementary and enhanced resistance to viral evolution. This deep learning approach may enable the design of next-generation antibody therapies that remain effective against future SARS-CoV-2 variants.\nThe RNA binding protein, HNRNPA2B1, regulates IFNG signaling in macrophages\nAuthors: Mohammed Salih, M.; Weindel, C. G.; Malekos, E.; Sudek, L.; Katzman, S.; Mabry, C. J.; Coleman, A.; Azam, S.; Watson, R. O.; Patrick, K. L.; Carpenter, S.\nScore: 4.0, Published: 2023-10-15 DOI: 10.1101/2023.10.12.562050\nHeterogeneous nuclear ribonucleoprotein A2B1 (HNRNPA2B1) is a well known RNA binding protein but the mechanisms by which it contributes to innate immune gene regulation are poorly understood. Here we report that HNRNPA2B1 functions in macrophages to regulate IFNG (IFN-{gamma}) signaling through alternative splicing of the IFNG receptor. Specific deletion of HNRNPA2B1 in macrophages resulted in altered cytokine responses in both an endotoxic shock model and following Salmonella infection. Interestingly, while HNRNPA2B1 can function as a viability gene, we observed increased macrophage and neutrophil numbers in the KO mice following LPS induced endotoxic shock. We also discovered that HNRNPA2B1 restricts replication of Salmonella enterica in vivo. Mechanistically, loss of HNRNPA2B1 resulted in an increase in NGO transcripts, which lack a start codon, of the IFNG receptor (Ifngr) leading to lower expression of the receptor at the cell surface impacting the downstream IFNG signaling cascade. Collectively, our data highlight an important role for HNRNPA2B1 in regulating IFNG signaling and restricting intracellular bacterial pathogens in macrophages.\nAntigen-level resolution of commensal-specific B cell responses enabled by phage-display screening and B cell tetramers\nAuthors: Verma, S.; Dufort, M. J.; Olsen, T. M.; Labuda, J. C.; Kimmel, S.; Scharffenberger, S.; McGuire, A.; Harrison, O. J.\nScore: 3.2, Published: 2023-10-17 DOI: 10.1101/2023.10.13.562275\nInduction of adaptive immune responses to commensal microbes is critical for tissue homeostasis, and perturbation of these responses is associated with multiple chronic inflammatory disorders. However, the mechanisms underlying the induction and regulation of mucosal B cells targeting commensal microbes remain poorly understood, in part due to a lack of tools to identify commensal-specific B cells ex vivo. To address this, we identified immunogenic protein epitopes recognized by Segmented Filamentous Bacteria (SFB)-specific serum antibodies using a whole-genome phage display screen and identified immunogenic proteins engaging IgA, IgG1 and IgG2b responses. Using these antigens, we generated B cell tetramers to identify and track SFB-specific B cell responses in the gut associated lymphoid tissue during natural and de novo colonization. We revealed a compartmentalized response in SFB-specific B cell activation between Peyer's patches and mesenteric lymph nodes, with a gradient of IgA, IgG1 and IgG2b isotypes along the small intestine, and selective production of IgG2b with the mesenteric lymph node chain. VDJ sequencing analyses and generation of SFB-specific monoclonal antibodies identified that somatic hypermutation drives affinity maturation to SFB derived antigens under homeostatic conditions. By combining phage display screening and B cell tetramer technologies, we now enable antigen-level based studies of immunity to intestinal microbes, which will advance our understanding of the ontogeny and function of commensal-specific B cell responses in tissue immunity, inflammation and repair.\nDisruption of myelin structure and oligodendrocyte maturation in a pigtail macaque model of congenital Zika infection\nAuthors: Tisoncik-Go, J.; Stokes, C.; Whitmore, L.; Newhouse, D. J.; Voss, K.; Gustin, A.; Song, C.-J.; Smith, E.; Stencel-Baerenwald, J.; Parker, E. D.; Snyder, J. M.; Shaw, D.; Rajagopal, L.; Kapur, R. P.; Waldorf, K. A.; Gale, M.\nScore: 3.2, Published: 2023-10-14 DOI: 10.1101/2023.10.11.561759\nZika virus (ZikV) infection during pregnancy can cause congenital Zika syndrome (CZS) and neurodevelopmental delay in non-microcephalic infants, of which the pathogenesis remains poorly understood. We utilized an established pigtail macaque maternal-to-fetal ZikV infection/exposure model to study fetal brain pathophysiology of CZS manifesting from ZikV exposure in utero. We found prenatal ZikV exposure led to profound disruption of fetal myelin, with extensive downregulation in gene expression for key components of oligodendrocyte maturation and myelin production. Immunohistochemical analyses revealed marked decreases in myelin basic protein intensity and myelinated fiber density in ZikV-exposed animals. At the ultrastructural level, the myelin sheath in ZikV-exposed animals showed multi-focal decompaction consistent with perturbation or remodeling of previously formed myelin, occurring concomitant with dysregulation of oligodendrocyte gene expression and maturation. These findings define fetal neuropathological profiles of ZikV-linked brain injury underlying CZS resulting from ZikV exposure in utero. Because myelin is critical for cortical development, ZikV-related perturbations in oligodendrocyte function may have long-term consequences on childhood neurodevelopment, even in the absence of overt microcephaly.\nTime- and lineage-resolved transcriptional profiling uncovers gene expression programs and clonal relationships that underlie human T lineage specification\nAuthors: Michaels, Y. S.; Major, M. C.; Bonham-Carter, B.; Zhang, J.; Heydari, T.; Edgar, J. M.; Greenstreet, L.; Vilarrasa-Blasi, R.; Kim, S.; Castle, E. L.; Forrow, A.; Ibanez-Rios, M. I.; Zimmerman, C.; Chung, Y.; Stach, T.; Werschler, N.; Knapp, D. J.; Vento-Tormo, R.; Schiebinger, G.; Zandstra, P. W.\nScore: 6.0, Published: 2023-10-10 DOI: 10.1101/2023.10.06.561277\nT cells develop from multi-potent hematopoietic progenitors in the thymus and provide adaptive protection against pathogens and cancer. However, the emergence of human T cell-competent blood progenitors, and their subsequent specification to the T lineage, has been challenging to capture in real time. Here, we leveraged a pluripotent stem cell differentiation system to understand the transcriptional dynamics and cell fate restriction events that underlie this critical developmental process. Time-resolved single cell RNA sequencing revealed that cell-cycle exit, downregulation of the multipotent hematopoietic program, and upregulation of \u003e90 lineage-associated transcription factors all occur within a highly co-ordinated and narrow developmental window. Computational gene-regulatory network inference elucidated the transcriptional logic of T lineage specification, uncovering an important role for YBX1. We mapped the differentiation cell fate hierarchy using transcribed lineage barcoding and mathematical trajectory inference and discovered that mast and myeloid potential bifurcate from each other early in haematopoiesis, upstream of T lineage restriction. Collectively, our analyses provide a quantitative, time-resolved model of human T cell specification with relevance for regenerative medicine and developmental immunology.\n",
  "wordCount" : "2617",
  "inLanguage": "en",
  "datePublished": "2023-10-18T10:38:41Z",
  "dateModified": "2023-10-18T10:38:41Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/immunology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      immunology
    </h1>
    <div class="post-meta"><span>updated on October 18, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.557914">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.557914" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.557914">
        <p class="paperTitle">SARS-CoV-2 antibodies cross-react and enhance dengue infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.557914" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.557914" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.</p>
        <p class="info">Score: 202.5, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.557914' target='https://doi.org/10.1101/2023.10.09.557914'> 10.1101/2023.10.09.557914</a></p>
        <p class="abstract">Dengue disease is highly prevalent in tropical and subtropical regions worldwide. However, its pathogenesis is still incompletely understood, particularly in comparison to other endemic viruses. Antibody-dependent enhancement (ADE) is a well-known phenomenon for dengue viruses. Given the recent surge in dengue cases and potential cross-reactivity with SARS-CoV-2 antibodies, this study explores the impact of anti-SARS-CoV-2 antibodies on DENV-2 infection.

The study assessed the cross-reactivity of SARS-CoV-2 antibodies with the DENV-2 Virus. Human convalescent plasma samples collected during different waves of COVID-19 and monoclonal and polyclonal antibodies raised against SARS-CoV-2 were examined for their potential to cause ADE of DENV-2 infection using cell-based assays. The study found that anti-SARS-CoV-2 antibodies acquired from natural infection in humans or through experimental immunization in animals were cross-reactive with DENV-2 and had the potential to enhance DENV-2 infection in K562 and U937 cells. In-silico and in-vitro studies indicated a strong interaction between SARS-CoV-2 antibodies and DENV-2 E-protein, providing a molecular basis for these findings. This study is the first to demonstrate that anti-SARS-CoV-2 antibodies can cross-react with DENV-2 and can enhance its infection through ADE. These findings have implications for SARS-CoV-2 vaccine development and deployment strategies in regions where dengue is endemic.

SummaryAntibodies against SARS-CoV-2 (RBD and Spike) showed significant cross reactivity with DENV-2 (E protein). Also, anti-SARS-CoV-2-commercial antibodies, immunised animal sera and 46 human convalescent plasma samples (from different waves of pandemic) demonstrated antibody-dependent enhancement (ADE) of DENV-2 infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.557143">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.557143" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.557143">
        <p class="paperTitle">Genetic ablation of adhesion ligands averts rejection of allogeneic immune cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.557143" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.557143" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hammer, Q.; Perica, K.; van Ooijen, H.; Mbofung, R.; Momayyezi, P.; Varady, E.; Martin, K. E.; Pan, Y.; Jelcic, M.; Groff, B.; Abujarour, R.; Krokeide, S.; Lee, T.; Williams, A.; Goodridge, J. P.; Valamehr, B.; Onfelt, B.; Sadelain, M.; Malmberg, K.-J.</p>
        <p class="info">Score: 16.8, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.557143' target='https://doi.org/10.1101/2023.10.09.557143'> 10.1101/2023.10.09.557143</a></p>
        <p class="abstract">Allogeneic cell therapies hold promise for broad clinical implementation, but face limitations due to potential rejection by the recipient immune system. Silencing of beta-2-microglobulin (B2M) expression is commonly employed to evade T cell-mediated rejection, although absence of B2M triggers missing-self responses by recipient natural killer (NK) cells. Here, we demonstrate that deletion of the adhesion ligands CD54 and CD58 on targets cells robustly dampens NK cell reactivity across all sub-populations. Genetic deletion of CD54 and CD58 in B2M-deficient allogeneic chimeric antigen receptor (CAR) T and multi-edited induced pluripotent stem cell (iPSC)-derived NK cells reduces their susceptibility to rejection by NK cells in vitro and in vivo without affecting their anti-tumor effector potential. Thus, these data suggest that genetic ablation of adhesion ligands effectively alleviates rejection of allogeneic immune cells for immunotherapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561025">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561025" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561025">
        <p class="paperTitle">HERV activation segregates ME/CFS from fibromyalgia and defines a novel nosological entity for patients fulfilling both clinical criteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561025" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561025" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gimenez-Orenga, K.; Martin-Martinez, E.; Nathanson, L.; Oltra, E.</p>
        <p class="info">Score: 129.9, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561025' target='https://doi.org/10.1101/2023.10.05.561025'> 10.1101/2023.10.05.561025</a></p>
        <p class="abstract">Myalgic Encephalomyelitis/Chronic Fatigue Syndrome (ME/CFS) and fibromyalgia (FM) are chronic diseases with poorly understood pathophysiology and diagnosis based on clinical assessment of unspecific symptoms. The recent post-COVID-19 condition, which shares similarities with ME/CFS and FM, has raised concerns about viral-induced transcriptome changes in post-viral syndromes. Viral infections, and other types of stress, are known to unleash human endogenous retroviruses (HERV) repression that if maintained could lead to symptom chronicity. This study evaluated this possibility for ME/CFS and FM on a selected cohort of female patients complying with diagnosis criteria for ME/CFS, FM, or both, and matched healthy controls (n=43). The results show specific HERV fingerprints for each disease, confirming biological differences between ME/CFS and FM. Unexpectedly, HERV profiles segregated patients that met both ME/CFS and FM clinical criteria from patients complying only with ME or FM criteria, while clearly differentiating patients from healthy subjects, supporting that the highly prevalent comorbidity condition must constitute a different nosological entity. Moreover, HERV profiles exposed significant quantitative differences within the ME/CFS group that correlated with differences in immune gene expression and patient symptomatology, supporting ME/CFS patient subtyping and confirming immunological disturbances in this disease. Pending issues include validation of HERV profiles as disease biomarkers of post-viral syndromes and understanding the role of HERV during infection and beyond.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.07.561349">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.07.561349" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.07.561349">
        <p class="paperTitle">Mast-Cell Expressed Membrane Protein-1 (MCEMP1) is expressed in classical monocytes and alveolar macrophages in Idiopathic Pulmonary Fibrosis and regulates cell chemotaxis, adhesion, and migration in a TGFβ dependent manner.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.07.561349" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.07.561349" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Perrot, C. Y.; Karampitsakos, T.; Unterman, A.; Adams, T.; Marlin, K.; Arsenault, A.; Zhao, A.; Kaminski, N.; Katlaps, G.; Patel, K.; Bandyopadhyay, D.; Herazo-Maya, J. D.</p>
        <p class="info">Score: 7.8, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.07.561349' target='https://doi.org/10.1101/2023.10.07.561349'> 10.1101/2023.10.07.561349</a></p>
        <p class="abstract">BackgroundMast-Cell Expressed Membrane Protein-1 (MCEMP1) is higher in Idiopathic Pulmonary Fibrosis (IPF) patients with increased risk of death and poor outcomes. Here we seek to establish the mechanistic role of MCEMP1 in pulmonary fibrosis.

MethodsMCEMP1 expression was analyzed by single-cell RNA sequencing, immunofluorescence in Peripheral Blood Mononuclear Cells (PBMC) as well as in lung tissues from IPF patients and controls. Chromatin Immunoprecipitation (ChiP) and Proximity Ligation Assay (PLA) were used to study the transcriptional regulation of MCEMP1. Transient RNA interference and lentivirus transduction were used to knockdown and knock-in MCEMP1 in THP-1 cells to study chemotaxis, adhesion, and migration. Bulk RNA sequencing was used to identify the mechanisms by which MCEMP1 participates in monocyte function. Active RHO pull-down assay was used to validate bulk RNA sequencing results.

ResultsWe identified increased MCEMP1 expression in classical monocytes and alveolar macrophages in IPF compared to controls. MCEMP1 was upregulated by TGF{beta} at the mRNA and protein levels in THP-1. TGF{beta}-mediated MCEMP1 upregulation results from the cooperation of SMAD3 and SP1 via concomitant binding to SMAD3/SP1 cis-regulatory elements within the MCEMP1 promoter. In terms of its function, we found that MCEMP1 regulates TGF{beta}-mediated monocyte chemotaxis, adhesion, and migration. 400 differentially expressed genes were found to increase after TGF{beta} stimulation of THP-1, further increased in MCEMP1 knock-in cells treated with TGF{beta} and decreased in MCEMP1 knockdown cells treated with TGF{beta}. GO annotation analysis of these genes showed enrichment for positive regulation of RHO GTPase activity and signal transduction. While TGF{beta} enhanced RHO GTPase activity in THP-1 cells, this effect was attenuated following MCEMP1 knockdown.

ConclusionMCEMP1 is highly expressed in circulating classical monocytes and alveolar macrophages in IPF. MCEMP1 is regulated by TGF{beta} and participates in the chemotaxis, adhesion, and migration of circulating monocytes by modulating the effect of TGF{beta} in RHO activity. Our results suggest that MCEMP1 may regulate the migration and transition of monocytes to monocyte-derived alveolar macrophages during pulmonary fibrosis development and progression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561856">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561856" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561856">
        <p class="paperTitle">Human effector CD8&#43; T cells with an exhausted-like phenotype control tumor growth in vivo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561856" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561856" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mietz, J.; Kaulfuss, M.; Egli, L.; Opitz, L.; Munz, C.; Chijioke, O.</p>
        <p class="info">Score: 5.1, Published: 2023-10-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561856' target='https://doi.org/10.1101/2023.10.11.561856'> 10.1101/2023.10.11.561856</a></p>
        <p class="abstract">The functional relevance of CD8&#43; T cell phenotypes in human cancer is not well established. In preclinical studies, murine CD8&#43; T cells exhibiting an exhausted-like phenotype have been described to have drastically decreased anticancer activity while non-exhausted TCF-1&#43; progenitor CD8&#43; T cells have been thought to be mainly responsible for driving protective antitumor responses. However, human data has mostly been correlative by nature. By establishing a novel autologous humanized mouse tumor model, we found a CD8&#43; dominated T cell expansion upon tumor challenge and identified tumor-reactive CD8&#43; T cells through expression of CD137. This subpopulation of de novo arising human CD8&#43; T cells displayed a highly proliferative, fully activated effector and exhausted-like phenotype with enhanced expression of exhaustion markers like PD-1, CD39 and TOX, the senescence marker CD57 (B3GAT1) and cytolytic effector molecules such as PRF1, GZMH and NKG7. Moreover, these CD137&#43;CD8&#43; T cells exhibited tumor-specific clonal expansion and presented signature overlap with tumor-reactive CD8&#43; T cells described in human cancer. We demonstrate superior anticancer activity of this exhausted-like human CD8&#43; T cell subset by adoptive transfer experiments using recipients bearing autologous human tumors. Mice adoptively transferred with CD137&#43;CD8&#43; T cells showed reduced tumor growth, higher CD8&#43; T cell persistence as well as tumor infiltration, correlating with control of human tumors. In summary, defined effective anticancer activity of human effector CD8&#43; T cells with an exhausted-like phenotype, supporting clinical exploration of such cells in adoptive T cell therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561492">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561492" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561492">
        <p class="paperTitle">Deep learning-guided selection of antibody therapies with enhanced resistance to current and prospective SARS-CoV-2 Omicron variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561492" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561492" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Frei, L.; Gao, B.; Han, J.; Taft, J. M.; Irvine, E. B.; Weber, C. R.; Kumar, R.; Eisinger, B.; Reddy, S. T.</p>
        <p class="info">Score: 5.1, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561492' target='https://doi.org/10.1101/2023.10.09.561492'> 10.1101/2023.10.09.561492</a></p>
        <p class="abstract">Most COVID-19 antibody therapies rely on binding the SARS-CoV-2 receptor binding domain (RBD). However, heavily mutated variants such as Omicron and its sublineages, which are characterized by an ever increasing number of mutations in the RBD, have rendered prior antibody therapies ineffective, leaving no clinically approved antibody treatments for SARS-CoV-2. Therefore, the capacity of therapeutic antibody candidates to bind and neutralize current and prospective SARS-CoV-2 variants is a critical factor for drug development. Here, we present a deep learning-guided approach to identify antibodies with enhanced resistance to SARS-CoV-2 evolution. We apply deep mutational learning (DML), a machine learning-guided protein engineering method to interrogate a massive sequence space of combinatorial RBD mutations and predict their impact on angiotensin-converting enzyme 2 (ACE2) binding and antibody escape. A high mutational distance library was constructed based on the full-length RBD of Omicron BA.1, which was experimentally screened for binding to the ACE2 receptor or neutralizing antibodies, followed by deep sequencing. The resulting data was used to train ensemble deep learning models that could accurately predict binding or escape for a panel of therapeutic antibody candidates targeting diverse RBD epitopes. Furthermore, antibody breadth was assessed by predicting binding or escape to synthetic lineages that represent millions of sequences generated using in silico evolution, revealing combinations with complementary and enhanced resistance to viral evolution. This deep learning approach may enable the design of next-generation antibody therapies that remain effective against future SARS-CoV-2 variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.12.562050">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.12.562050" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.12.562050">
        <p class="paperTitle">The RNA binding protein, HNRNPA2B1, regulates IFNG signaling in macrophages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.12.562050" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.12.562050" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mohammed Salih, M.; Weindel, C. G.; Malekos, E.; Sudek, L.; Katzman, S.; Mabry, C. J.; Coleman, A.; Azam, S.; Watson, R. O.; Patrick, K. L.; Carpenter, S.</p>
        <p class="info">Score: 4.0, Published: 2023-10-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.12.562050' target='https://doi.org/10.1101/2023.10.12.562050'> 10.1101/2023.10.12.562050</a></p>
        <p class="abstract">Heterogeneous nuclear ribonucleoprotein A2B1 (HNRNPA2B1) is a well known RNA binding protein but the mechanisms by which it contributes to innate immune gene regulation are poorly understood. Here we report that HNRNPA2B1 functions in macrophages to regulate IFNG (IFN-{gamma}) signaling through alternative splicing of the IFNG receptor. Specific deletion of HNRNPA2B1 in macrophages resulted in altered cytokine responses in both an endotoxic shock model and following Salmonella infection. Interestingly, while HNRNPA2B1 can function as a viability gene, we observed increased macrophage and neutrophil numbers in the KO mice following LPS induced endotoxic shock. We also discovered that HNRNPA2B1 restricts replication of Salmonella enterica in vivo. Mechanistically, loss of HNRNPA2B1 resulted in an increase in NGO transcripts, which lack a start codon, of the IFNG receptor (Ifngr) leading to lower expression of the receptor at the cell surface impacting the downstream IFNG signaling cascade. Collectively, our data highlight an important role for HNRNPA2B1 in regulating IFNG signaling and restricting intracellular bacterial pathogens in macrophages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562275">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562275" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562275">
        <p class="paperTitle">Antigen-level resolution of commensal-specific B cell responses enabled by phage-display screening and B cell tetramers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562275" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562275" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Verma, S.; Dufort, M. J.; Olsen, T. M.; Labuda, J. C.; Kimmel, S.; Scharffenberger, S.; McGuire, A.; Harrison, O. J.</p>
        <p class="info">Score: 3.2, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562275' target='https://doi.org/10.1101/2023.10.13.562275'> 10.1101/2023.10.13.562275</a></p>
        <p class="abstract">Induction of adaptive immune responses to commensal microbes is critical for tissue homeostasis, and perturbation of these responses is associated with multiple chronic inflammatory disorders. However, the mechanisms underlying the induction and regulation of mucosal B cells targeting commensal microbes remain poorly understood, in part due to a lack of tools to identify commensal-specific B cells ex vivo. To address this, we identified immunogenic protein epitopes recognized by Segmented Filamentous Bacteria (SFB)-specific serum antibodies using a whole-genome phage display screen and identified immunogenic proteins engaging IgA, IgG1 and IgG2b responses. Using these antigens, we generated B cell tetramers to identify and track SFB-specific B cell responses in the gut associated lymphoid tissue during natural and de novo colonization. We revealed a compartmentalized response in SFB-specific B cell activation between Peyer&#39;s patches and mesenteric lymph nodes, with a gradient of IgA, IgG1 and IgG2b isotypes along the small intestine, and selective production of IgG2b with the mesenteric lymph node chain. VDJ sequencing analyses and generation of SFB-specific monoclonal antibodies identified that somatic hypermutation drives affinity maturation to SFB derived antigens under homeostatic conditions. By combining phage display screening and B cell tetramer technologies, we now enable antigen-level based studies of immunity to intestinal microbes, which will advance our understanding of the ontogeny and function of commensal-specific B cell responses in tissue immunity, inflammation and repair.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561759">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561759" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561759">
        <p class="paperTitle">Disruption of myelin structure and oligodendrocyte maturation in a pigtail macaque model of congenital Zika infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561759" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561759" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tisoncik-Go, J.; Stokes, C.; Whitmore, L.; Newhouse, D. J.; Voss, K.; Gustin, A.; Song, C.-J.; Smith, E.; Stencel-Baerenwald, J.; Parker, E. D.; Snyder, J. M.; Shaw, D.; Rajagopal, L.; Kapur, R. P.; Waldorf, K. A.; Gale, M.</p>
        <p class="info">Score: 3.2, Published: 2023-10-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561759' target='https://doi.org/10.1101/2023.10.11.561759'> 10.1101/2023.10.11.561759</a></p>
        <p class="abstract">Zika virus (ZikV) infection during pregnancy can cause congenital Zika syndrome (CZS) and neurodevelopmental delay in non-microcephalic infants, of which the pathogenesis remains poorly understood. We utilized an established pigtail macaque maternal-to-fetal ZikV infection/exposure model to study fetal brain pathophysiology of CZS manifesting from ZikV exposure in utero. We found prenatal ZikV exposure led to profound disruption of fetal myelin, with extensive downregulation in gene expression for key components of oligodendrocyte maturation and myelin production. Immunohistochemical analyses revealed marked decreases in myelin basic protein intensity and myelinated fiber density in ZikV-exposed animals. At the ultrastructural level, the myelin sheath in ZikV-exposed animals showed multi-focal decompaction consistent with perturbation or remodeling of previously formed myelin, occurring concomitant with dysregulation of oligodendrocyte gene expression and maturation. These findings define fetal neuropathological profiles of ZikV-linked brain injury underlying CZS resulting from ZikV exposure in utero. Because myelin is critical for cortical development, ZikV-related perturbations in oligodendrocyte function may have long-term consequences on childhood neurodevelopment, even in the absence of overt microcephaly.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561277">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561277" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561277">
        <p class="paperTitle">Time- and lineage-resolved transcriptional profiling uncovers gene expression programs and clonal relationships that underlie human T lineage specification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561277" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561277" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Michaels, Y. S.; Major, M. C.; Bonham-Carter, B.; Zhang, J.; Heydari, T.; Edgar, J. M.; Greenstreet, L.; Vilarrasa-Blasi, R.; Kim, S.; Castle, E. L.; Forrow, A.; Ibanez-Rios, M. I.; Zimmerman, C.; Chung, Y.; Stach, T.; Werschler, N.; Knapp, D. J.; Vento-Tormo, R.; Schiebinger, G.; Zandstra, P. W.</p>
        <p class="info">Score: 6.0, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561277' target='https://doi.org/10.1101/2023.10.06.561277'> 10.1101/2023.10.06.561277</a></p>
        <p class="abstract">T cells develop from multi-potent hematopoietic progenitors in the thymus and provide adaptive protection against pathogens and cancer. However, the emergence of human T cell-competent blood progenitors, and their subsequent specification to the T lineage, has been challenging to capture in real time. Here, we leveraged a pluripotent stem cell differentiation system to understand the transcriptional dynamics and cell fate restriction events that underlie this critical developmental process. Time-resolved single cell RNA sequencing revealed that cell-cycle exit, downregulation of the multipotent hematopoietic program, and upregulation of &gt;90 lineage-associated transcription factors all occur within a highly co-ordinated and narrow developmental window. Computational gene-regulatory network inference elucidated the transcriptional logic of T lineage specification, uncovering an important role for YBX1. We mapped the differentiation cell fate hierarchy using transcribed lineage barcoding and mathematical trajectory inference and discovered that mast and myeloid potential bifurcate from each other early in haematopoiesis, upstream of T lineage restriction. Collectively, our analyses provide a quantitative, time-resolved model of human T cell specification with relevance for regenerative medicine and developmental immunology.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
