<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>immunology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>immunology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Megakaryocyte infection by SARS-CoV-2 drives the formation of pathogenic afucosylated IgG antibodies in mice
Authors: Mabrouk, M.; Jalali, F.; Zaid, N.; Naya, A.; Khyatti, M.; Khalki, L.; Guessous, F.; Zaid, Y.
Score: 697.3, Published: 2023-07-20 DOI: 10.1101/2023.07.14.549113
More than 90% of total human plasma immunoglobulin G (IgG) is found in a fucosylated form, but specific IgGs with low core fucosylation (afucosylated IgGs) are found in response to infections with enveloped viruses and to alloantigens on blood cells.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/immunology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="immunology" />
<meta property="og:description" content="Megakaryocyte infection by SARS-CoV-2 drives the formation of pathogenic afucosylated IgG antibodies in mice
Authors: Mabrouk, M.; Jalali, F.; Zaid, N.; Naya, A.; Khyatti, M.; Khalki, L.; Guessous, F.; Zaid, Y.
Score: 697.3, Published: 2023-07-20 DOI: 10.1101/2023.07.14.549113
More than 90% of total human plasma immunoglobulin G (IgG) is found in a fucosylated form, but specific IgGs with low core fucosylation (afucosylated IgGs) are found in response to infections with enveloped viruses and to alloantigens on blood cells." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/immunology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-13T10:36:01+00:00" />
<meta property="article:modified_time" content="2023-08-13T10:36:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="immunology"/>
<meta name="twitter:description" content="Megakaryocyte infection by SARS-CoV-2 drives the formation of pathogenic afucosylated IgG antibodies in mice
Authors: Mabrouk, M.; Jalali, F.; Zaid, N.; Naya, A.; Khyatti, M.; Khalki, L.; Guessous, F.; Zaid, Y.
Score: 697.3, Published: 2023-07-20 DOI: 10.1101/2023.07.14.549113
More than 90% of total human plasma immunoglobulin G (IgG) is found in a fucosylated form, but specific IgGs with low core fucosylation (afucosylated IgGs) are found in response to infections with enveloped viruses and to alloantigens on blood cells."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "immunology",
      "item": "https://trxiv.yorks0n.com/posts/immunology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "immunology",
  "name": "immunology",
  "description": "Megakaryocyte infection by SARS-CoV-2 drives the formation of pathogenic afucosylated IgG antibodies in mice\nAuthors: Mabrouk, M.; Jalali, F.; Zaid, N.; Naya, A.; Khyatti, M.; Khalki, L.; Guessous, F.; Zaid, Y.\nScore: 697.3, Published: 2023-07-20 DOI: 10.1101/2023.07.14.549113\nMore than 90% of total human plasma immunoglobulin G (IgG) is found in a fucosylated form, but specific IgGs with low core fucosylation (afucosylated IgGs) are found in response to infections with enveloped viruses and to alloantigens on blood cells.",
  "keywords": [
    
  ],
  "articleBody": " Megakaryocyte infection by SARS-CoV-2 drives the formation of pathogenic afucosylated IgG antibodies in mice\nAuthors: Mabrouk, M.; Jalali, F.; Zaid, N.; Naya, A.; Khyatti, M.; Khalki, L.; Guessous, F.; Zaid, Y.\nScore: 697.3, Published: 2023-07-20 DOI: 10.1101/2023.07.14.549113\nMore than 90% of total human plasma immunoglobulin G (IgG) is found in a fucosylated form, but specific IgGs with low core fucosylation (afucosylated IgGs) are found in response to infections with enveloped viruses and to alloantigens on blood cells. Afucosylated IgGs mediate immunopathology in severe COVID-19 and dengue fever in humans. In COVID-19, the early formation of non-neutralizing afucosylated IgG against the spike protein predicts and directly mediates disease progression to severe form. IgG lacking core fucosylation causes dramatically increased antibody-dependent cellular toxicity mediated by intense Fc{gamma}R-mediated stimulation of macrophages, monocytes, natural killer cells, and platelets. The mechanism and the context within which afucosylated IgG formation occurs in response to enveloped virus antigens have remained elusive thus far in COVID-19, dengue fever, and other infections. This study demonstrates that administration of human bone marrow megakaryocytes infected by SARS-CoV-2 into the circulation of K18-hACE2 transgenic mice drives the formation of pathogenic afucosylated anti-spike IgG antibodies, and is sufficient to reproduce severe COVID-19 manifestations of pulmonary vascular thrombosis, acute lung injury, and death in mice.\nEvaluation of the Ability of AlphaFold to Predict the Three-Dimensional Structures of Antibodies and Epitopes\nAuthors: Polonsky, K.; Pupko, T.; Freund, N.\nScore: 17.0, Published: 2023-08-06 DOI: 10.1101/2023.08.03.551715\nBeing able to accurately predict the three-dimensional structure of an antibody can facilitate fast and precise antibody characterization and epitope prediction, with important diagnostic and clinical implications. In the current study, we evaluate the ability of AlphaFold to predict the structures of 222 recently published, non-redundant, high resolution Fab heavy and light chain structures of antibodies from different species (human, Macaca mulatta, mouse, rabbit, rat) directed against different antigens. Our analysis reveals that while the overall prediction quality of antibody chains is in line with the results available in CASP14, other antibody regions like the complementarity-determining regions (CDRs) of the heavy chain, which are prone to higher genetic variation, generate a less accurate prediction. Moreover, we discovered that AlphaFold often mis-predicts the bending angles between the variable and constant domains within a Fab. To evaluate the ability of AlphaFold to model antibody:antigen interactions based only on sequence, we used AlphaFold-multimer in combination with ZDOCK docking to predict the structures of 26 known antibody:antigen complexes. ZDOCK succeeded in predicting 11, and AlphaFold only two, out of 26 models with medium or high accuracy, with significant deviations in the docking contacts predicted in the rest of the molecules. In summary, our study provides important information about the abilities and limitations of using AlphaFold to predict antibody:antigen interactions and suggests areas for possible improvement. Key PointsO_LIAlphaFold was used to predict 222 new 3D hi-res atomic structures of Ab chains. C_LIO_LILow accuracy was observed in the prediction of HC-CDR3 and the elbow angles. C_LIO_LIPredicting Ab-Ag complexes and epitope mapping using AlphaFold-Multimer was limited. C_LI\nA comparison of clustering models for inference of T cell receptor antigen specificity\nAuthors: Koohy, H.; Hudson, D.; Lubbock, A.; Basham, M.\nScore: 10.5, Published: 2023-08-07 DOI: 10.1101/2023.08.04.551940\nThe vast potential sequence diversity of TCRs and their ligands has presented an historic barrier to computational prediction of TCR epitope specificity, a holy grail of quantitative immunology. One common approach is to cluster sequences together, on the assumption that similar receptors bind similar epitopes. Here, we provide an independent evaluation of widely used clustering algorithms for TCR specificity inference, observing some variability in predictive performance between models, and marked differences in scalability. Despite these differences, we find that different algorithms produce clusters with high degrees of similarity for receptors recognising the same epitope. Our analysis highlights an unmet need for improvement of complex models over a simple Hamming distance comparator, and strengthens the case for use of clustering models in TCR specificity inference.\nAllergic inflammation triggers dyslipidemia via IgG signalling\nAuthors: Fernandez-Gallego, N.; Castillo-Gonzalez, R.; Moreno-Serna, L.; Garcia-Civico, A. J.; Sanchez-Martinez, E.; Lopez-Sanz, C.; Fontes, A. L.; Pimentel, L. L.; Gradillas, A.; Obeso, D.; Ramirez-Huesca, M.; Ruiz-Fernandez, I.; Nunez-Borque, E.; Carrasco, Y. R.; Ibanez, B.; Martin, P.; Blanco, C.; Barbas, C.; Barber, D.; Rodriguez-Alcala, L. M.; Villasenor, A.; Esteban, V.; Sanchez-Madrid, F.; Jimenez-Saiz, R.\nScore: 9.8, Published: 2023-08-06 DOI: 10.1101/2023.08.04.551996\nAllergic diseases begin early in life and are often chronic, thus creating an inflammatory environment that may lead to metabolic disorders, although the underlying mechanisms remain incompletely understood. Here, we show that allergic inflammation induces diet-independent dyslipidemia in a mouse model of allergy and atherosclerosis. Using untargeted lipidomics in mouse plasma, we found that allergic inflammation induces a unique lipid signature that extends beyond acute and late inflammation and that is characterized by triglyceride (TG) changes in circulation. Alterations in blood TGs following an allergic reaction are independent of T-cell-driven late phase inflammation. On the contrary, the humoral component is sufficient to induce a TG increase and a unique lipid profile through the IgG-mediated alternative pathway of anaphylaxis. Lastly, we demonstrated blood TG changes in patients after undergoing an allergic reaction. Overall, this study reveals the importance of IgG-mediated allergic inflammation insofar as it regulates lipid metabolism, which may contribute to atherosclerosis and, ultimately, to cardiovascular events.\nOntogeny of the organized nasopharynx-associated lymphoid tissue in rainbow trout\nAuthors: Garcia, B. J.; Reyes, A.; Martinez, C.; Serra dos Santos, Y.; Salinas, I.\nScore: 8.2, Published: 2023-08-07 DOI: 10.1101/2023.08.04.552019\nUnderstanding the ontogeny of teleost mucosa-associated lymphoid tissues (MALT) is critical for determining the earliest timepoint for effective mucosal vaccination of young fish. Here, we describe the developmental sequence that leads to the formation of an organized MALT structure in rainbow trout, the organized nasopharynx-associated lymphoid tissue (O-NALT). Control rainbow trout were sampled between 340 and 1860 degree days (DD) and routine histology and immunofluorescence staining were used to determine cellular changes in immune cells in the nasal cavity as well as O-NALT formation. We identified that O-NALT is first seeded by CD8+ T cells at 700DD followed by IgM+ B cells and CD4-2b+ T cells at 1000DD. Histomorphologically, trout O-NALT is fully formed at 1400DD when it is composed of 67% CD4-2b+ cells, 20% IgM+ cells, 13% CD8+ T cells, and no IgT+ B cells. Whole body gene expression analyses uncovered waves of igmh, cd4-2b, and cd8a expression that recapitulate the cellular seeding sequence of O-NALT by specific lymphocyte subsets. Our results indicate that 1) O-NALT formation results from a specific sequence of lymphocyte subset colonization pioneered by CD8+ T cells and 2) the presence of the full O-NALT structure at 1400DD may mark this timepoint as the earliest developmental stage at which mucosal vaccines can induce long lasting, specific immune responses.\nThe T cell receptor sequence influences the likelihood of T cell memory formation\nAuthors: Lagattuta, K. A.; Nathan, A.; Rumker, L.; Birnbaum, M. E.; Raychaudhuri, S.\nScore: 55.3, Published: 2023-07-23 DOI: 10.1101/2023.07.20.549939\nT cell differentiation depends on activation through the T cell receptor (TCR), whose amino acid sequence varies cell to cell. Particular TCR amino acid sequences nearly guarantee Mucosal-Associated Invariant T (MAIT) and Natural Killer T (NKT) cell fates. To comprehensively define how TCR amino acids affects all T cell fates, we analyze the paired {beta}TCR sequence and transcriptome of 819,772 single cells. We find that hydrophobic CDR3 residues promote regulatory T cell transcriptional states in both the CD8 and CD4 lineages. Most strikingly, we find a set of TCR sequence features, concentrated in CDR2, that promotes positive selection in the thymus as well as transition from naive to memory in the periphery. Even among T cells that recognize the same antigen, these TCR sequence features help to explain which T cells form immunological memory, which is essential for effective pathogen response.\nTIM3 is a context-dependent co-regulator of cytotoxic T cell function\nAuthors: Alamir, H.; Wong, C. W. C.; Alsubaiti, A.; Edmunds, G. L.; Grant, T.; Alsulaimani, S.; Boyd, J.; Holland, C. J.; Morgan, D. J.; Gallimore, A. M.; Wuelfing, C.\nScore: 6.8, Published: 2023-08-05 DOI: 10.1101/2023.08.03.551797\nTIM3 is a co-regulatory receptor that is highly expressed on multiple immune cell types, including on T cells after prolonged exposure to antigen. It marks functionally suppressed cytotoxic T lymphocytes (CTL) in the tumor microenvironment. However, it is unresolved whether TIM3 acts directly on suppressed CTL. Moreover, the nature of TIM3 ligands remains controversial. Paradoxically, TIM3 combines inhibitory function in vivo with costimulatory signaling capability in vitro. Here we have investigated TIM3 in the direct interaction of suppressed murine and human CTL with tumor target cell using spheroids. TIM3 directly inhibited the function of such CTL. TIM3 regulated the ability of suppressed CTL to polarize their cytoskeleton as a required step in cytolysis. Expression of CEACAM1 in cis, on the CTL, blocked TIM3 function, expression of CEACAM1 and galectin9 in trans, on the tumor target cells, enhanced TIM3 function. TIM3 only functioned as an inhibitory receptor on the spheroid-suppressed CTL, not on active CTL in a two-dimensional tissue culture model. These data suggest that TIM3 amplifies T cell function, serving as a co-inhibitory or co-stimulatory receptor depending on the functional context of the T cell it is expressed on.\nMatching or genetic engineering of HLA Class I and II facilitates successful allogeneic 'off-the-shelf' regulatory T cell therapy\nAuthors: McCallion, O.; Du, W.; Glaser, V.; Milward, K.; Franke, C.; Kath, J.; Valkov, M.; Yang, M.; Künkele, A.; Polansky, J. K.; Schmueck-Henneresse, M.; Volk, H.-D.; Reinke, P.; Wagner, D. L.; Hester, J.; Issa, F.\nScore: 5.2, Published: 2023-08-06 DOI: 10.1101/2023.08.06.551956\nThe potential to harness regulatory T cells (Tregs) for the treatment of autoimmune diseases and transplant rejection has been restricted by several barriers: donor variability, manufacturing complications, and time-consuming expansion processes. These issues further complicate the use of autologous Tregs during acute disease phases or when Tregs are low in number or dysfunctional. Here we explore the potential of off-the-shelf allogeneic Tregs, from healthy donors or universal sources, to provide a more practical solution. We discover that the efficacy of these cells is undermined by the recipients immune response, and that that rigorous matching of HLA classes I and II overcomes this barrier. Importantly, genetically manipulating HLA expression enables the use of unmatched allogeneic Tregs with in vivo efficacy. Our findings underscore the transformative potential of HLA-engineered Tregs, offering a novel, ready-to-use therapeutic avenue for treating a wide array of inflammatory diseases. One-Sentence SummaryMatching or engineering of HLA-I and HLA-II facilitates allogeneic off-the-shelf regulatory T cells for immunoregulation.\nResident memory T cell precursors in tumor draining lymph nodes require type-1 IFN for optimal differentiation\nAuthors: Khatwani, N.; Searles, T.; Han, J.; Messier, C.; Mittal, N.; Molodtsov, A.; Ramirez, D.; Hawkes, A.; Williams, O. M.; Huang, Y. H.; Kolling, F. W.; Rosato, P.; Turk, M. J.\nScore: 9.8, Published: 2023-08-05 DOI: 10.1101/2023.08.02.551658\nResident memory (Trm) cells play an essential role in anti-tumor immunity. However, little is known about the precursors that differentiate into protective Trm populations against cancer. Here we employed an established model of B16 melanoma neoadjuvant anti-CD4 therapy, to track tumor antigen-specific CD8+ T cells through tissues and across time; from their priming as effectors to their differentiation into Trm. We show that tumor-draining lymph nodes (TDLNs) contain Teff cells that begin to express canonical Trm markers CD103 and CD69. These tumor-specific Teff cells seeded skin and tumor during the effector phase of the response, although egress from these tissues was not required Trm development in LNs. Paired scRNAseq/scTCRseq was used to identify Teff clonotypes in TDLNs and trace their differentiation, in real-time, into Trm populations. We found that expanded clonotypes favored the Trm fate and were unlikely to co-differentiate into other lineages. Precursors of Trm (pre-Trm) clonotypes that subsequently seeded populations throughout tumors, LNs, and skin, were characterized by early expression of tissue residency, stemness, and type-1 IFN sensing genes. These multipotent pre-Trm cells sensed plasmacytoid dendritic cell-derived type-1 interferons in TDLNs, and their expression of interferon alpha receptor was required for their formation of Trm populations in LNs but not in skin. These findings reveal the defining features of pre-Trm cells in response to tumor antigens, and reveal a previously unappreciated role for type-1 IFNs in programming regional Trm immunity to cancer. One Sentence SummaryAnti-tumor effector CD8 T cells adopt early characteristics of tissue residency and stemness, and rely on the sensing of type-1 interferons for their local differentiation into resident memory T cells.\nIn-depth immunometabolic profiling by measuring single-cell protein translation inhibition via bioorthogonal noncanonical amino acid tagging (SCENITH-BONCAT)\nAuthors: Vrieling, F.; van der Zande, H. J. P.; Smeehuijzen, L.; van den Bossche, J.; Kersten, S.; Stienstra, R.\nScore: 3.6, Published: 2023-08-05 DOI: 10.1101/2023.08.03.551800\nMotivationSeahorse Extracellular Flux (XF) analysis has been influential in immunometabolism research, measuring cellular oxygen consumption rate (OCR) and extracellular acidification rate (ECAR) to determine immune cell metabolic profiles. However, Seahorse XF has several limitations, relying on purified adherent cells, relatively high cell numbers, and specialized equipment and reagents. Recently, a novel flow cytometry-based technique called SCENITH (Single Cell Energetic metabolism by profiling Translation inhibition) was introduced, which measures cellular protein synthesis through puromycin staining as a proxy for metabolic activity in single cells. The current limitation of SCENITH is its reliance on a specific anti-puromycin antibody that is not commercially available. To address this, we propose an alternative approach using biorthogonal noncanonical amino acid tagging (BONCAT) to measure protein synthesis. SummaryThe field of immunometabolism has revealed that cellular energy metabolism significantly contributes to immune cell function. Disturbances in immune cell metabolism have been associated with various diseases, including obesity, atherosclerosis, and cancer. To further advance immunometabolic research, developing novel methods to study the metabolism of immune cells in complex samples is required. Here, we introduce SCENITH-BONCAT, an adapted version of SCENITH which utilizes click-labeling of homopropargylglycine (HPG), an alkyne-bearing methionine analog, to measure protein synthesis as a proxy of metabolic activity. SCENITH-BONCAT successfully reproduced known metabolic signatures of immune cell activation. Specifically, LPS/IFN{gamma}-induced classical activation increased glycolytic capacity, and IL-4-induced alternative activation enhanced mitochondrial dependence in human macrophages. The assays applicability was further explored in peripheral blood mononuclear cells (PBMCs) from healthy volunteers, revealing diverse metabolic rewiring in immune cell subsets upon stimulation with different activators. Cryopreserved PBMCs showed reduced HPG incorporation and mitochondrial dependence compared to freshly isolated cells, warranting attention to cellular mitochondrial health during cryopreservation. Finally, SCENITH-BONCAT was used to analyze murine tissue-resident immune cells from different organs. Principal component analysis (PCA) revealed tissue-specific clustering based on metabolic profiles, likely driven by microenvironmental priming of tissue-resident immune cells. In conclusion, SCENITH-BONCAT offers valuable insights into immune cell metabolic responses and presents a powerful platform for studying immune cell metabolism in complex samples and tissue-resident immune populations in both human and murine studies.\n",
  "wordCount" : "2447",
  "inLanguage": "en",
  "datePublished": "2023-08-13T10:36:01Z",
  "dateModified": "2023-08-13T10:36:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/immunology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      immunology
    </h1>
    <div class="post-meta"><span>updated on August 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.549113">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.549113" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.549113">
        <p class="paperTitle">Megakaryocyte infection by SARS-CoV-2 drives the formation of pathogenic afucosylated IgG antibodies in mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.549113" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.549113" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mabrouk, M.; Jalali, F.; Zaid, N.; Naya, A.; Khyatti, M.; Khalki, L.; Guessous, F.; Zaid, Y.</p>
        <p class="info">Score: 697.3, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.549113' target='https://doi.org/10.1101/2023.07.14.549113'> 10.1101/2023.07.14.549113</a></p>
        <p class="abstract">More than 90% of total human plasma immunoglobulin G (IgG) is found in a fucosylated form, but specific IgGs with low core fucosylation (afucosylated IgGs) are found in response to infections with enveloped viruses and to alloantigens on blood cells. Afucosylated IgGs mediate immunopathology in severe COVID-19 and dengue fever in humans. In COVID-19, the early formation of non-neutralizing afucosylated IgG against the spike protein predicts and directly mediates disease progression to severe form. IgG lacking core fucosylation causes dramatically increased antibody-dependent cellular toxicity mediated by intense Fc{gamma}R-mediated stimulation of macrophages, monocytes, natural killer cells, and platelets. The mechanism and the context within which afucosylated IgG formation occurs in response to enveloped virus antigens have remained elusive thus far in COVID-19, dengue fever, and other infections. This study demonstrates that administration of human bone marrow megakaryocytes infected by SARS-CoV-2 into the circulation of K18-hACE2 transgenic mice drives the formation of pathogenic afucosylated anti-spike IgG antibodies, and is sufficient to reproduce severe COVID-19 manifestations of pulmonary vascular thrombosis, acute lung injury, and death in mice.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551715">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551715" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551715">
        <p class="paperTitle">Evaluation of the Ability of AlphaFold to Predict the Three-Dimensional Structures of Antibodies and Epitopes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551715" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551715" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Polonsky, K.; Pupko, T.; Freund, N.</p>
        <p class="info">Score: 17.0, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551715' target='https://doi.org/10.1101/2023.08.03.551715'> 10.1101/2023.08.03.551715</a></p>
        <p class="abstract">Being able to accurately predict the three-dimensional structure of an antibody can facilitate fast and precise antibody characterization and epitope prediction, with important diagnostic and clinical implications. In the current study, we evaluate the ability of AlphaFold to predict the structures of 222 recently published, non-redundant, high resolution Fab heavy and light chain structures of antibodies from different species (human, Macaca mulatta, mouse, rabbit, rat) directed against different antigens. Our analysis reveals that while the overall prediction quality of antibody chains is in line with the results available in CASP14, other antibody regions like the complementarity-determining regions (CDRs) of the heavy chain, which are prone to higher genetic variation, generate a less accurate prediction. Moreover, we discovered that AlphaFold often mis-predicts the bending angles between the variable and constant domains within a Fab. To evaluate the ability of AlphaFold to model antibody:antigen interactions based only on sequence, we used AlphaFold-multimer in combination with ZDOCK docking to predict the structures of 26 known antibody:antigen complexes. ZDOCK succeeded in predicting 11, and AlphaFold only two, out of 26 models with medium or high accuracy, with significant deviations in the docking contacts predicted in the rest of the molecules. In summary, our study provides important information about the abilities and limitations of using AlphaFold to predict antibody:antigen interactions and suggests areas for possible improvement.

Key PointsO_LIAlphaFold was used to predict 222 new 3D hi-res atomic structures of Ab chains.
C_LIO_LILow accuracy was observed in the prediction of HC-CDR3 and the elbow angles.
C_LIO_LIPredicting Ab-Ag complexes and epitope mapping using AlphaFold-Multimer was limited.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.551940">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.551940" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.551940">
        <p class="paperTitle">A comparison of clustering models for inference of T cell receptor antigen specificity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.551940" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.551940" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koohy, H.; Hudson, D.; Lubbock, A.; Basham, M.</p>
        <p class="info">Score: 10.5, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.551940' target='https://doi.org/10.1101/2023.08.04.551940'> 10.1101/2023.08.04.551940</a></p>
        <p class="abstract">The vast potential sequence diversity of TCRs and their ligands has presented an historic barrier to computational prediction of TCR epitope specificity, a holy grail of quantitative immunology. One common approach is to cluster sequences together, on the assumption that similar receptors bind similar epitopes. Here, we provide an independent evaluation of widely used clustering algorithms for TCR specificity inference, observing some variability in predictive performance between models, and marked differences in scalability. Despite these differences, we find that different algorithms produce clusters with high degrees of similarity for receptors recognising the same epitope. Our analysis highlights an unmet need for improvement of complex models over a simple Hamming distance comparator, and strengthens the case for use of clustering models in TCR specificity inference.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.551996">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.551996" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.551996">
        <p class="paperTitle">Allergic inflammation triggers dyslipidemia via IgG signalling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.551996" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.551996" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fernandez-Gallego, N.; Castillo-Gonzalez, R.; Moreno-Serna, L.; Garcia-Civico, A. J.; Sanchez-Martinez, E.; Lopez-Sanz, C.; Fontes, A. L.; Pimentel, L. L.; Gradillas, A.; Obeso, D.; Ramirez-Huesca, M.; Ruiz-Fernandez, I.; Nunez-Borque, E.; Carrasco, Y. R.; Ibanez, B.; Martin, P.; Blanco, C.; Barbas, C.; Barber, D.; Rodriguez-Alcala, L. M.; Villasenor, A.; Esteban, V.; Sanchez-Madrid, F.; Jimenez-Saiz, R.</p>
        <p class="info">Score: 9.8, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.551996' target='https://doi.org/10.1101/2023.08.04.551996'> 10.1101/2023.08.04.551996</a></p>
        <p class="abstract">Allergic diseases begin early in life and are often chronic, thus creating an inflammatory environment that may lead to metabolic disorders, although the underlying mechanisms remain incompletely understood. Here, we show that allergic inflammation induces diet-independent dyslipidemia in a mouse model of allergy and atherosclerosis. Using untargeted lipidomics in mouse plasma, we found that allergic inflammation induces a unique lipid signature that extends beyond acute and late inflammation and that is characterized by triglyceride (TG) changes in circulation. Alterations in blood TGs following an allergic reaction are independent of T-cell-driven late phase inflammation. On the contrary, the humoral component is sufficient to induce a TG increase and a unique lipid profile through the IgG-mediated alternative pathway of anaphylaxis. Lastly, we demonstrated blood TG changes in patients after undergoing an allergic reaction. Overall, this study reveals the importance of IgG-mediated allergic inflammation insofar as it regulates lipid metabolism, which may contribute to atherosclerosis and, ultimately, to cardiovascular events.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552019">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552019" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552019">
        <p class="paperTitle">Ontogeny of the organized nasopharynx-associated lymphoid tissue in rainbow trout</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552019" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552019" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garcia, B. J.; Reyes, A.; Martinez, C.; Serra dos Santos, Y.; Salinas, I.</p>
        <p class="info">Score: 8.2, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552019' target='https://doi.org/10.1101/2023.08.04.552019'> 10.1101/2023.08.04.552019</a></p>
        <p class="abstract">Understanding the ontogeny of teleost mucosa-associated lymphoid tissues (MALT) is critical for determining the earliest timepoint for effective mucosal vaccination of young fish. Here, we describe the developmental sequence that leads to the formation of an organized MALT structure in rainbow trout, the organized nasopharynx-associated lymphoid tissue (O-NALT). Control rainbow trout were sampled between 340 and 1860 degree days (DD) and routine histology and immunofluorescence staining were used to determine cellular changes in immune cells in the nasal cavity as well as O-NALT formation. We identified that O-NALT is first seeded by CD8&#43; T cells at 700DD followed by IgM&#43; B cells and CD4-2b&#43; T cells at 1000DD. Histomorphologically, trout O-NALT is fully formed at 1400DD when it is composed of 67% CD4-2b&#43; cells, 20% IgM&#43; cells, 13% CD8&#43; T cells, and no IgT&#43; B cells. Whole body gene expression analyses uncovered waves of igmh, cd4-2b, and cd8a expression that recapitulate the cellular seeding sequence of O-NALT by specific lymphocyte subsets. Our results indicate that 1) O-NALT formation results from a specific sequence of lymphocyte subset colonization pioneered by CD8&#43; T cells and 2) the presence of the full O-NALT structure at 1400DD may mark this timepoint as the earliest developmental stage at which mucosal vaccines can induce long lasting, specific immune responses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549939">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549939" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549939">
        <p class="paperTitle">The T cell receptor sequence influences the likelihood of T cell memory formation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549939" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549939" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lagattuta, K. A.; Nathan, A.; Rumker, L.; Birnbaum, M. E.; Raychaudhuri, S.</p>
        <p class="info">Score: 55.3, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549939' target='https://doi.org/10.1101/2023.07.20.549939'> 10.1101/2023.07.20.549939</a></p>
        <p class="abstract">T cell differentiation depends on activation through the T cell receptor (TCR), whose amino acid sequence varies cell to cell. Particular TCR amino acid sequences nearly guarantee Mucosal-Associated Invariant T (MAIT) and Natural Killer T (NKT) cell fates. To comprehensively define how TCR amino acids affects all T cell fates, we analyze the paired {beta}TCR sequence and transcriptome of 819,772 single cells. We find that hydrophobic CDR3 residues promote regulatory T cell transcriptional states in both the CD8 and CD4 lineages. Most strikingly, we find a set of TCR sequence features, concentrated in CDR2, that promotes positive selection in the thymus as well as transition from naive to memory in the periphery. Even among T cells that recognize the same antigen, these TCR sequence features help to explain which T cells form immunological memory, which is essential for effective pathogen response.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551797">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551797" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551797">
        <p class="paperTitle">TIM3 is a context-dependent co-regulator of cytotoxic T cell function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551797" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551797" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alamir, H.; Wong, C. W. C.; Alsubaiti, A.; Edmunds, G. L.; Grant, T.; Alsulaimani, S.; Boyd, J.; Holland, C. J.; Morgan, D. J.; Gallimore, A. M.; Wuelfing, C.</p>
        <p class="info">Score: 6.8, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551797' target='https://doi.org/10.1101/2023.08.03.551797'> 10.1101/2023.08.03.551797</a></p>
        <p class="abstract">TIM3 is a co-regulatory receptor that is highly expressed on multiple immune cell types, including on T cells after prolonged exposure to antigen. It marks functionally suppressed cytotoxic T lymphocytes (CTL) in the tumor microenvironment. However, it is unresolved whether TIM3 acts directly on suppressed CTL. Moreover, the nature of TIM3 ligands remains controversial. Paradoxically, TIM3 combines inhibitory function in vivo with costimulatory signaling capability in vitro. Here we have investigated TIM3 in the direct interaction of suppressed murine and human CTL with tumor target cell using spheroids. TIM3 directly inhibited the function of such CTL. TIM3 regulated the ability of suppressed CTL to polarize their cytoskeleton as a required step in cytolysis. Expression of CEACAM1 in cis, on the CTL, blocked TIM3 function, expression of CEACAM1 and galectin9 in trans, on the tumor target cells, enhanced TIM3 function. TIM3 only functioned as an inhibitory receptor on the spheroid-suppressed CTL, not on active CTL in a two-dimensional tissue culture model. These data suggest that TIM3 amplifies T cell function, serving as a co-inhibitory or co-stimulatory receptor depending on the functional context of the T cell it is expressed on.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.551956">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.551956" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.551956">
        <p class="paperTitle">Matching or genetic engineering of HLA Class I and II facilitates successful allogeneic &#39;off-the-shelf&#39; regulatory T cell therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.551956" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.551956" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McCallion, O.; Du, W.; Glaser, V.; Milward, K.; Franke, C.; Kath, J.; Valkov, M.; Yang, M.; Künkele, A.; Polansky, J. K.; Schmueck-Henneresse, M.; Volk, H.-D.; Reinke, P.; Wagner, D. L.; Hester, J.; Issa, F.</p>
        <p class="info">Score: 5.2, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.551956' target='https://doi.org/10.1101/2023.08.06.551956'> 10.1101/2023.08.06.551956</a></p>
        <p class="abstract">The potential to harness regulatory T cells (Tregs) for the treatment of autoimmune diseases and transplant rejection has been restricted by several barriers: donor variability, manufacturing complications, and time-consuming expansion processes. These issues further complicate the use of autologous Tregs during acute disease phases or when Tregs are low in number or dysfunctional. Here we explore the potential of  off-the-shelf allogeneic Tregs, from healthy donors or universal sources, to provide a more practical solution. We discover that the efficacy of these cells is undermined by the recipients immune response, and that that rigorous matching of HLA classes I and II overcomes this barrier. Importantly, genetically manipulating HLA expression enables the use of unmatched allogeneic Tregs with in vivo efficacy. Our findings underscore the transformative potential of HLA-engineered Tregs, offering a novel, ready-to-use therapeutic avenue for treating a wide array of inflammatory diseases.

One-Sentence SummaryMatching or engineering of HLA-I and HLA-II facilitates allogeneic  off-the-shelf regulatory T cells for immunoregulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551658">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551658" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551658">
        <p class="paperTitle">Resident memory T cell precursors in tumor draining lymph nodes require type-1 IFN for optimal differentiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551658" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551658" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khatwani, N.; Searles, T.; Han, J.; Messier, C.; Mittal, N.; Molodtsov, A.; Ramirez, D.; Hawkes, A.; Williams, O. M.; Huang, Y. H.; Kolling, F. W.; Rosato, P.; Turk, M. J.</p>
        <p class="info">Score: 9.8, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551658' target='https://doi.org/10.1101/2023.08.02.551658'> 10.1101/2023.08.02.551658</a></p>
        <p class="abstract">Resident memory (Trm) cells play an essential role in anti-tumor immunity. However, little is known about the precursors that differentiate into protective Trm populations against cancer. Here we employed an established model of B16 melanoma neoadjuvant anti-CD4 therapy, to track tumor antigen-specific CD8&#43; T cells through tissues and across time; from their priming as effectors to their differentiation into Trm. We show that tumor-draining lymph nodes (TDLNs) contain Teff cells that begin to express canonical Trm markers CD103 and CD69. These tumor-specific Teff cells seeded skin and tumor during the effector phase of the response, although egress from these tissues was not required Trm development in LNs. Paired scRNAseq/scTCRseq was used to identify Teff clonotypes in TDLNs and trace their differentiation, in real-time, into Trm populations. We found that expanded clonotypes favored the Trm fate and were unlikely to co-differentiate into other lineages. Precursors of Trm (pre-Trm) clonotypes that subsequently seeded populations throughout tumors, LNs, and skin, were characterized by early expression of tissue residency, stemness, and type-1 IFN sensing genes. These multipotent pre-Trm cells sensed plasmacytoid dendritic cell-derived type-1 interferons in TDLNs, and their expression of interferon alpha receptor was required for their formation of Trm populations in LNs but not in skin. These findings reveal the defining features of pre-Trm cells in response to tumor antigens, and reveal a previously unappreciated role for type-1 IFNs in programming regional Trm immunity to cancer.

One Sentence SummaryAnti-tumor effector CD8 T cells adopt early characteristics of tissue residency and stemness, and rely on the sensing of type-1 interferons for their local differentiation into resident memory T cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551800">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551800" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551800">
        <p class="paperTitle">In-depth immunometabolic profiling by measuring single-cell protein translation inhibition via bioorthogonal noncanonical amino acid tagging (SCENITH-BONCAT)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551800" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551800" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vrieling, F.; van der Zande, H. J. P.; Smeehuijzen, L.; van den Bossche, J.; Kersten, S.; Stienstra, R.</p>
        <p class="info">Score: 3.6, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551800' target='https://doi.org/10.1101/2023.08.03.551800'> 10.1101/2023.08.03.551800</a></p>
        <p class="abstract">MotivationSeahorse Extracellular Flux (XF) analysis has been influential in immunometabolism research, measuring cellular oxygen consumption rate (OCR) and extracellular acidification rate (ECAR) to determine immune cell metabolic profiles. However, Seahorse XF has several limitations, relying on purified adherent cells, relatively high cell numbers, and specialized equipment and reagents. Recently, a novel flow cytometry-based technique called SCENITH (Single Cell Energetic metabolism by profiling Translation inhibition) was introduced, which measures cellular protein synthesis through puromycin staining as a proxy for metabolic activity in single cells. The current limitation of SCENITH is its reliance on a specific anti-puromycin antibody that is not commercially available. To address this, we propose an alternative approach using biorthogonal noncanonical amino acid tagging (BONCAT) to measure protein synthesis.

SummaryThe field of immunometabolism has revealed that cellular energy metabolism significantly contributes to immune cell function. Disturbances in immune cell metabolism have been associated with various diseases, including obesity, atherosclerosis, and cancer. To further advance immunometabolic research, developing novel methods to study the metabolism of immune cells in complex samples is required. Here, we introduce SCENITH-BONCAT, an adapted version of SCENITH which utilizes click-labeling of homopropargylglycine (HPG), an alkyne-bearing methionine analog, to measure protein synthesis as a proxy of metabolic activity. SCENITH-BONCAT successfully reproduced known metabolic signatures of immune cell activation. Specifically, LPS/IFN{gamma}-induced classical activation increased glycolytic capacity, and IL-4-induced alternative activation enhanced mitochondrial dependence in human macrophages. The assays applicability was further explored in peripheral blood mononuclear cells (PBMCs) from healthy volunteers, revealing diverse metabolic rewiring in immune cell subsets upon stimulation with different activators. Cryopreserved PBMCs showed reduced HPG incorporation and mitochondrial dependence compared to freshly isolated cells, warranting attention to cellular mitochondrial health during cryopreservation. Finally, SCENITH-BONCAT was used to analyze murine tissue-resident immune cells from different organs. Principal component analysis (PCA) revealed tissue-specific clustering based on metabolic profiles, likely driven by microenvironmental priming of tissue-resident immune cells. In conclusion, SCENITH-BONCAT offers valuable insights into immune cell metabolic responses and presents a powerful platform for studying immune cell metabolism in complex samples and tissue-resident immune populations in both human and murine studies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
