<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>immunology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>immunology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="XBB.1.5 monovalent mRNA vaccine booster elicits robust neutralizing antibodies against emerging SARS-CoV-2 variants
Authors: Wang, Q.; Guo, Y.; Bowen, A.; Mellis, I. A.; Valdez, R.; Gherasim, C.; Gordon, A.; Liu, L.; Ho, D. D.
Score: 1219.7, Published: 2023-12-06 DOI: 10.1101/2023.11.26.568730
COVID-19 vaccines have recently been updated with the spike protein of SARS-CoV-2 XBB.1.5 subvariant alone, but their immunogenicity in humans has yet to be fully evaluated and reported, particularly against emergent viruses that are rapidly expanding.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/immunology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="immunology" />
<meta property="og:description" content="XBB.1.5 monovalent mRNA vaccine booster elicits robust neutralizing antibodies against emerging SARS-CoV-2 variants
Authors: Wang, Q.; Guo, Y.; Bowen, A.; Mellis, I. A.; Valdez, R.; Gherasim, C.; Gordon, A.; Liu, L.; Ho, D. D.
Score: 1219.7, Published: 2023-12-06 DOI: 10.1101/2023.11.26.568730
COVID-19 vaccines have recently been updated with the spike protein of SARS-CoV-2 XBB.1.5 subvariant alone, but their immunogenicity in humans has yet to be fully evaluated and reported, particularly against emergent viruses that are rapidly expanding." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/immunology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-20T10:36:27+00:00" />
<meta property="article:modified_time" content="2023-12-20T10:36:27+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="immunology"/>
<meta name="twitter:description" content="XBB.1.5 monovalent mRNA vaccine booster elicits robust neutralizing antibodies against emerging SARS-CoV-2 variants
Authors: Wang, Q.; Guo, Y.; Bowen, A.; Mellis, I. A.; Valdez, R.; Gherasim, C.; Gordon, A.; Liu, L.; Ho, D. D.
Score: 1219.7, Published: 2023-12-06 DOI: 10.1101/2023.11.26.568730
COVID-19 vaccines have recently been updated with the spike protein of SARS-CoV-2 XBB.1.5 subvariant alone, but their immunogenicity in humans has yet to be fully evaluated and reported, particularly against emergent viruses that are rapidly expanding."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "immunology",
      "item": "https://trxiv.yorks0n.com/posts/immunology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "immunology",
  "name": "immunology",
  "description": "XBB.1.5 monovalent mRNA vaccine booster elicits robust neutralizing antibodies against emerging SARS-CoV-2 variants\nAuthors: Wang, Q.; Guo, Y.; Bowen, A.; Mellis, I. A.; Valdez, R.; Gherasim, C.; Gordon, A.; Liu, L.; Ho, D. D.\nScore: 1219.7, Published: 2023-12-06 DOI: 10.1101/2023.11.26.568730\nCOVID-19 vaccines have recently been updated with the spike protein of SARS-CoV-2 XBB.1.5 subvariant alone, but their immunogenicity in humans has yet to be fully evaluated and reported, particularly against emergent viruses that are rapidly expanding.",
  "keywords": [
    
  ],
  "articleBody": " XBB.1.5 monovalent mRNA vaccine booster elicits robust neutralizing antibodies against emerging SARS-CoV-2 variants\nAuthors: Wang, Q.; Guo, Y.; Bowen, A.; Mellis, I. A.; Valdez, R.; Gherasim, C.; Gordon, A.; Liu, L.; Ho, D. D.\nScore: 1219.7, Published: 2023-12-06 DOI: 10.1101/2023.11.26.568730\nCOVID-19 vaccines have recently been updated with the spike protein of SARS-CoV-2 XBB.1.5 subvariant alone, but their immunogenicity in humans has yet to be fully evaluated and reported, particularly against emergent viruses that are rapidly expanding. We now report that administration of an updated monovalent mRNA vaccine (XBB.1.5 MV) to uninfected individuals boosted serum virus-neutralization antibodies significantly against not only XBB.1.5 (27.0-fold) and the currently dominant EG.5.1 (27.6-fold) but also key emergent viruses like HV.1, HK.3, JD.1.1, and JN.1 (13.3-to-27.4-fold). In individuals previously infected by an Omicron subvariant, serum neutralizing titers were boosted to highest levels (1,504-to-22,978) against all viral variants tested. While immunological imprinting was still evident with the updated vaccines, it was not nearly as severe as the previously authorized bivalent BA.5 vaccine. Our findings strongly support the official recommendation to widely apply the updated COVID-19 vaccines to further protect the public.\nEngineering self-propelled tumor-infiltrating CAR T cells using synthetic velocity receptors\nAuthors: Johnston, A. C.; Alicea, G. M.; Lee, C. C.; Patel, P. V.; Hanna, E. A.; Vaz, E.; Forjaz, A.; Wan, Z.; Nair, P. R.; Lim, Y.; Chen, T.; Du, W.; Kim, D.; Nichakawade, T. D.; Rebecca, V. W.; Bonifant, C. L.; Fan, R.; Kiemen, A. L.; Wu, P.-H.; Wirtz, D.\nScore: 133.3, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571595\nChimeric antigen receptor (CAR) T cells express antigen-specific synthetic receptors, which upon binding to cancer cells, elicit T cell anti-tumor responses. CAR T cell therapy has enjoyed success in the clinic for hematological cancer indications, giving rise to decade-long remissions in some cases. However, CAR T therapy for patients with solid tumors has not seen similar successes. Solid tumors constitute 90% of adult human cancers, representing an enormous unmet clinical need, ripe for new cell therapies. Current approaches do not solve the central problem of limited ability of therapeutic cells to migrate through the stromal matrix. We discover that T cells at low and high density display low- and high-migration phenotypes, respectively. The highly migratory phenotype is mediated by a paracrine pathway from a group of self-produced inflammatory cytokines that include IL5, TNF, IFN{gamma}, and IL8. We exploit this finding to \"lock-in\" a highly migratory phenotype by developing and expressing receptors, which we call velocity receptors (VRs). VRs target these cytokines and signal through these cytokines cognate receptors to increase T cell motility and infiltrate lung, ovarian, and pancreatic tumors in large numbers and at doses for which control CAR T cells remain confined to the tumor periphery. In contrast to CAR therapy alone, VR-CAR T cells significantly attenuate tumor growth and extend overall survival. This work suggests that approaches to the design of immune cell receptors that focus on migration signaling will help current and future CAR cellular therapies to infiltrate deep into solid tumors.\nMulti-variate statistical and machine learning reveals the interplay between sex and age in antibody responses to de novo SARS-CoV-2 infection and vaccination\nAuthors: Cuperlovic-Culf, M.; Bennett, S.; Galipeau, Y.; McCluskie, P. S.; Arnold, C.; Bagheri, S.; Cooper, C. L.; Langlois, M.-A.; Fritz, J.; A. Piccirillo, C.; Crawley, A. M.\nScore: 34.5, Published: 2023-12-05 DOI: 10.1101/2023.12.05.569965\nPrevention of negative COVID19 infection outcomes and infection/vaccine-acquired immunity is associated with the quality of antibody responses, whose variance by age and sex are poorly understood. Integrated, network approaches, identified sex and age effects in antibody responses and neutralization potential of de novo infection and vaccination throughout the Covid-19 pandemic. Cluster analysis found neutralization values followed SARS-CoV-2 specific receptor binding RIgG, spike SIgG and S and RIgA levels based on COVID19 status. Stochastic behavior tests and other analytical methods revealed sex differences only in persons \u003c40y.o. Serum IgA antibody titers correlated with neutralization only in females 40-60y.o. Network analysis found males could improve IgA responses after vaccination dose 2, unlike \u003e60y.o. females. Complex correlation analyses found vaccination induced less antibody isotype switching and neutralization in older persons, especially in females. Sex dependent antibody \u0026 neutralization behavior decayed fastest in older males and with vaccination. Such sex and age characterization by machine learning can direct studies integrating cell mediated responses to define yet elusive correlates of protection and inform age and sex precision-focused vaccine design.\nMacrophages control pathological interferon responses during viral respiratory infection\nAuthors: Hoagland, D. A.; Rodriguez-Morales, P.; Mann, A. O.; Yu, S.; Lai, A.; Baez Vazquez, A.; Pope, S. D.; Lim, J.; Li, S.; Zhang, X.; Li, M. O.; Medzhitov, R.; Franklin, R. A.\nScore: 31.0, Published: 2023-12-17 DOI: 10.1101/2023.12.16.572019\nAntiviral immune mediators, including interferons and their downstream effectors, are critical for host defense yet can become detrimental when uncontrolled. Here, we identify a macrophage-mediated anti-inflammatory mechanism that limits type I interferon (IFN-I) responses. Specifically, we found that cellular stress and pathogen recognition induce Oncostatin M (OSM) production by macrophages. OSM-deficient mice succumbed to challenge with influenza or a viral mimic due to heightened IFN-I activation. Macrophage-derived OSM restricted excessive IFN-I production by lung epithelial cells following viral stimulation. Furthermore, reconstitution of OSM in the respiratory tract was sufficient to protect mice lacking macrophage-derived OSM against morbidity, indicating the importance of local OSM production. This work reveals a host strategy to dampen inflammation in the lung through the negative regulation of IFN-I by macrophages.\nTransposable elements may enhance antiviral resistance in HIV-1 elite controllers\nAuthors: Singh, M.; Leddy, S. M.; Iniguez, L. P.; Bendall, M. L.; Feschotte, C.; Nixon, D. F.\nScore: 28.7, Published: 2023-12-12 DOI: 10.1101/2023.12.11.571123\nLess than 0.5% of people living with HIV-1 are elite controllers (ECs) - individuals who have a replication-competent viral reservoir in their CD4+ T cells but maintain undetectable plasma viremia without the help of antiretroviral therapy. While the EC CD4+ T cell transcriptome has been investigated for gene expression signatures associated with disease progression (or, in this case, a lack thereof), the expression and regulatory activity of transposable elements (TEs) in ECs has not been explored. Yet previous studies have established that TEs can directly impact the immune response to pathogens, including HIV-1. Thus, we hypothesize that the regulatory activities of TEs could contribute to the natural resistance of ECs against HIV-1. We perform a TE-centric analysis of previously published multi-omics data derived from EC individuals and other populations. We find that the CD4+ T cell transcriptome and retrotranscriptome of ECs are distinct from healthy controls, treated patients, and viremic progressors. However, there is a substantial level of transcriptomic heterogeneity among ECs. We categorize individuals with distinct chromatin accessibility and expression profiles into four clusters within the EC group, each possessing unique repertoires of TEs and antiviral factors. Notably, several TE families with known immuno-regulatory activity are differentially expressed among ECs. Their transcript levels in ECs positively correlate with their chromatin accessibility and negatively correlate with the expression of their KRAB zinc-finger (KZNF) repressors. This coordinated variation is seen at the level of individual TE loci likely acting or, in some cases, known to act as cis-regulatory elements for nearby genes involved in the immune response and HIV-1 restriction. Based on these results, we propose that the EC phenotype is driven in part by the reduced availability of specific KZNF proteins to repress TE-derived cis-regulatory elements for antiviral genes, thereby heightening their basal level of resistance to HIV-1 infection. Our study reveals considerable heterogeneity in the CD4+ T cell transcriptome of ECs, including variable expression of TEs and their KZNF controllers, that must be taken into consideration to decipher the mechanisms enabling HIV-1 control.\nMassively parallel base editing screens to map variant effects on anti-tumor hallmarks of primary human T cells\nAuthors: Walsh, Z. H.; Shah, P.; Kothapalli, N.; Nikolenyi, G.; Shah, S.; Leuzzi, G.; Mu, M.; Ho, P.; Abuzaid, S.; Brodtman, Z. D.; Vasan, N.; AlQuaraishi, M. N.; Milner, J. D.; Ciccia, A.; Melms, J. C.; Izar, B.\nScore: 23.6, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571465\nBase editing enables generation of single nucleotide variants, but large-scale screening in primary human T cells is limited due to low editing efficiency, among other challenges1. Here, we developed a high-throughput approach for high-efficiency and massively parallel adenine and cytosine base-editor screening in primary human T cells. We performed multiple large-scale screens editing 102 genes with central functions in T cells and full-length tiling mutagenesis of selected genes, and read out variant effects on hallmarks of T cell anti-tumor immunity, including activation, proliferation, and cytokine production. We discovered a broad landscape of gain- and loss-of-function mutations, including in PIK3CD and its regulatory subunit encoded by PIK3R1, LCK, AKT1, CTLA-4 and JAK1. We identified variants that affected several (e.g., PIK3CD C416R) or only selected (e.g. LCK Y505C) hallmarks of T cell activity, and functionally validated several hits by probing downstream signaling nodes and testing their impact on T cell polyfunctionality and proliferation. Using primary human T cells in which we engineered a T cell receptor (TCR) specific to a commonly presented tumor testis antigen as a model for cellular immunotherapy, we demonstrate that base edits identified in our screens can tune specific or broad T cell functions and ultimately improve tumor elimination while exerting minimal off-target activity. In summary, we present the first large-scale base editing screen in primary human T cells and provide a framework for scalable and targeted base editing at high efficiency. Coupled with multi-modal phenotypic mapping, we accurately nominate variants that produce a desirable T cell state and leverage these synthetic proteins to improve models of cellular cancer immunotherapies.\nOpposing effects of pre-existing antibody and memory T cell help on the dynamics of recall germinal centers\nAuthors: Schiepers, A.; van 't Wout, M. F. L.; Hobbs, A.; Mesin, L.; Victora, G. D.\nScore: 20.8, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571936\nRe-exposure to an antigen generates serum antibody responses that greatly exceed in magnitude those elicited by primary antigen encounter, while simultaneously driving the formation of recall germinal centers (GCs). Although recall GCs in mice are composed almost entirely of naive B cells, recall antibody titers derive overwhelmingly from memory B cells, suggesting a division between cellular and serum compartments. Here, we show that this schism is at least partly explained by a marked decrease in the ability of recall GC B cells to detectably bind antigen. Variant priming and plasmablast ablation experiments show that this decrease is largely due to suppression by preexisting antibody, whereas hapten-carrier experiments reveal a role for memory T cell help in allowing B cells with undetectable antigen binding to access GCs. We propose a model in which antibody-mediated feedback steers recall GC B cells away from previously targeted epitopes, thus enabling specific targeting of variant epitopes.\n50-color phenotyping of the human immune system with in-depth assessment of T cells and dendritic cells\nAuthors: Konecny, A. J.; Mage, P.; Tyznik, A. J.; Prlic, M.; Mair, F.\nScore: 19.5, Published: 2023-12-15 DOI: 10.1101/2023.12.14.571745\nWe report the development of an optimized 50-color spectral flow cytometry panel designed for the in-depth analysis of the immune system in human blood and tissues, with the goal of maximizing the amount of information that can be collected using currently available flow cytometry platforms. We established and tested this panel using peripheral blood mononuclear cells (PBMCs), but included CD45 to enable its use for the analysis of human tissue samples. The panel contains lineage markers for all major immune cell subsets, and an extensive set of phenotyping markers focused on the activation and differentiation status of the T cell and dendritic cell (DC) compartment. We outline the biological insight that can be gained from the simultaneous measurement of such a large number of proteins and propose that this approach provides a unique opportunity for the comprehensive exploration of the immune status in tissue biopsies and other human samples with a limited number of cells. Of note, we tested the panel to be compatible with cell sorting for further downstream applications. Furthermore, to facilitate the wide-spread implementation of such a panel across different cohorts and samples, we established a trimmed-down 45-color version which can be used with different spectral cytometry platforms. Finally, to generate this panel, we utilized not only existing panel design guidelines, but also developed new metrics to systematically identify the optimal combination of 50 fluorochromes and evaluate fluorochrome-specific resolution in the context of a 50-color unmixing matrix.\nTargeting HLA-E Positive Cancers with a Novel NKG2A/C Switch Receptor\nAuthors: Saetersmoen, M. L.; Kotchetkov, I.; Torralba-Raga, L.; Masilla-Soto, J.; Sohlberg, E.; Krokeide, S. Z.; Hammer, Q.; Sadelain, M.; Malmberg, K.-J.\nScore: 13.9, Published: 2023-12-15 DOI: 10.1101/2023.12.14.571754\nHLA-E is overexpressed by approximately 80% of solid tumors, including malignant glioblastoma, and is emerging as a major checkpoint for NKG2A+ CD8+ T cells and NK cells in the tumor microenvironment and circulation. This axis operates side-by-side with PD-L1 to shut down effector responses by T and NK cells. Here, we engineered a novel chimeric A/C switch receptor, combining the strong HLA-E binding affinity of the NKG2A receptor ectodomain with the activating signaling of the NKG2C receptor endodomain. We found that A/C Switch-transduced NK and T cells displayed superior and specific cytotoxic function when challenged with tumor cells exhibiting medium to high HLA-E expression. Furthermore, A/C Switch-expressing human T cells demonstrated enhanced anti-tumor function in a xenograft model of glioblastoma. Importantly, the activity of the modified T cells was governed by an equilibrium between A/C Switch transduction level and HLA-E expression, creating a therapeutic window to safeguard against on-target off-tumor toxicities. Indeed, normal cells remained insensitive to A/C Switch engineered T cells even after pre-treatment with IFN-{gamma} to induce HLA-E expression. We propose that this novel A/C switch receptor may operate alone to control tumor cells expressing high levels of HLA-E or in combination with other engineered specificities to overcome the suppressive NKG2A/HLA-E checkpoint.\nCovalent Targeting of Splicing in T Cells.\nAuthors: Scott, K. A.; Kojima, H.; Ropek, N.; Warren, C. D.; Zhang, T. L.; Hogg, S. J.; Webster, C.; Zhang, X.; Rahman, J.; Melillo, B.; Cravatt, B. F.; Lyu, J.; Abdel-Wahab, O.; Vinogradova, E. V.\nScore: 12.8, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572199\nDespite significant interest in therapeutic targeting of splicing, few chemical probes are available for the proteins involved in splicing. Here, we show that elaborated stereoisomeric acrylamide chemical probe EV96 and its analogues lead to a selective T cell state-dependent loss of interleukin 2 inducible T cell kinase (ITK) by targeting one of the core splicing factors SF3B1. Mechanistic investigations suggest that the state-dependency stems from a combination of differential protein turnover rates and availability of functional mRNA pools that can be depleted due to extensive alternative splicing. We further introduce a comprehensive list of proteins involved in splicing and leverage both cysteine- and protein-directed activity-based protein profiling (ABPP) data with electrophilic scout fragments to demonstrate covalent ligandability for many classes of splicing factors and splicing regulators in primary human T cells. Taken together, our findings show how chemical perturbation of splicing can lead to immune state-dependent changes in protein expression and provide evidence for the broad potential to target splicing factors with covalent chemistry.\n",
  "wordCount" : "2506",
  "inLanguage": "en",
  "datePublished": "2023-12-20T10:36:27Z",
  "dateModified": "2023-12-20T10:36:27Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/immunology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      immunology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 20, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.26.568730">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.26.568730" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.26.568730">
        <p class="paperTitle">XBB.1.5 monovalent mRNA vaccine booster elicits robust neutralizing antibodies against emerging SARS-CoV-2 variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.26.568730" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.26.568730" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Q.; Guo, Y.; Bowen, A.; Mellis, I. A.; Valdez, R.; Gherasim, C.; Gordon, A.; Liu, L.; Ho, D. D.</p>
        <p class="info">Score: 1219.7, Published: 2023-12-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.26.568730' target='https://doi.org/10.1101/2023.11.26.568730'> 10.1101/2023.11.26.568730</a></p>
        <p class="abstract">COVID-19 vaccines have recently been updated with the spike protein of SARS-CoV-2 XBB.1.5 subvariant alone, but their immunogenicity in humans has yet to be fully evaluated and reported, particularly against emergent viruses that are rapidly expanding. We now report that administration of an updated monovalent mRNA vaccine (XBB.1.5 MV) to uninfected individuals boosted serum virus-neutralization antibodies significantly against not only XBB.1.5 (27.0-fold) and the currently dominant EG.5.1 (27.6-fold) but also key emergent viruses like HV.1, HK.3, JD.1.1, and JN.1 (13.3-to-27.4-fold). In individuals previously infected by an Omicron subvariant, serum neutralizing titers were boosted to highest levels (1,504-to-22,978) against all viral variants tested. While immunological imprinting was still evident with the updated vaccines, it was not nearly as severe as the previously authorized bivalent BA.5 vaccine. Our findings strongly support the official recommendation to widely apply the updated COVID-19 vaccines to further protect the public.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571595">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571595" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571595">
        <p class="paperTitle">Engineering self-propelled tumor-infiltrating CAR T cells using synthetic velocity receptors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571595" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571595" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnston, A. C.; Alicea, G. M.; Lee, C. C.; Patel, P. V.; Hanna, E. A.; Vaz, E.; Forjaz, A.; Wan, Z.; Nair, P. R.; Lim, Y.; Chen, T.; Du, W.; Kim, D.; Nichakawade, T. D.; Rebecca, V. W.; Bonifant, C. L.; Fan, R.; Kiemen, A. L.; Wu, P.-H.; Wirtz, D.</p>
        <p class="info">Score: 133.3, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571595' target='https://doi.org/10.1101/2023.12.13.571595'> 10.1101/2023.12.13.571595</a></p>
        <p class="abstract">Chimeric antigen receptor (CAR) T cells express antigen-specific synthetic receptors, which upon binding to cancer cells, elicit T cell anti-tumor responses. CAR T cell therapy has enjoyed success in the clinic for hematological cancer indications, giving rise to decade-long remissions in some cases. However, CAR T therapy for patients with solid tumors has not seen similar successes. Solid tumors constitute 90% of adult human cancers, representing an enormous unmet clinical need, ripe for new cell therapies. Current approaches do not solve the central problem of limited ability of therapeutic cells to migrate through the stromal matrix. We discover that T cells at low and high density display low- and high-migration phenotypes, respectively. The highly migratory phenotype is mediated by a paracrine pathway from a group of self-produced inflammatory cytokines that include IL5, TNF, IFN{gamma}, and IL8. We exploit this finding to &#34;lock-in&#34; a highly migratory phenotype by developing and expressing receptors, which we call velocity receptors (VRs). VRs target these cytokines and signal through these cytokines cognate receptors to increase T cell motility and infiltrate lung, ovarian, and pancreatic tumors in large numbers and at doses for which control CAR T cells remain confined to the tumor periphery. In contrast to CAR therapy alone, VR-CAR T cells significantly attenuate tumor growth and extend overall survival. This work suggests that approaches to the design of immune cell receptors that focus on migration signaling will help current and future CAR cellular therapies to infiltrate deep into solid tumors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.569965">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.569965" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.569965">
        <p class="paperTitle">Multi-variate statistical and machine learning reveals the interplay between sex and age in antibody responses to de novo SARS-CoV-2 infection and vaccination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.569965" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.569965" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cuperlovic-Culf, M.; Bennett, S.; Galipeau, Y.; McCluskie, P. S.; Arnold, C.; Bagheri, S.; Cooper, C. L.; Langlois, M.-A.; Fritz, J.; A. Piccirillo, C.; Crawley, A. M.</p>
        <p class="info">Score: 34.5, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.569965' target='https://doi.org/10.1101/2023.12.05.569965'> 10.1101/2023.12.05.569965</a></p>
        <p class="abstract">Prevention of negative COVID19 infection outcomes and infection/vaccine-acquired immunity is associated with the quality of antibody responses, whose variance by age and sex are poorly understood. Integrated, network approaches, identified sex and age effects in antibody responses and neutralization potential of de novo infection and vaccination throughout the Covid-19 pandemic. Cluster analysis found neutralization values followed SARS-CoV-2 specific receptor binding RIgG, spike SIgG and S and RIgA levels based on COVID19 status. Stochastic behavior tests and other analytical methods revealed sex differences only in persons &lt;40y.o. Serum IgA antibody titers correlated with neutralization only in females 40-60y.o. Network analysis found males could improve IgA responses after vaccination dose 2, unlike &gt;60y.o. females. Complex correlation analyses found vaccination induced less antibody isotype switching and neutralization in older persons, especially in females. Sex dependent antibody &amp; neutralization behavior decayed fastest in older males and with vaccination. Such sex and age characterization by machine learning can direct studies integrating cell mediated responses to define yet elusive correlates of protection and inform age and sex precision-focused vaccine design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.16.572019">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.16.572019" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.16.572019">
        <p class="paperTitle">Macrophages control pathological interferon responses during viral respiratory infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.16.572019" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.16.572019" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoagland, D. A.; Rodriguez-Morales, P.; Mann, A. O.; Yu, S.; Lai, A.; Baez Vazquez, A.; Pope, S. D.; Lim, J.; Li, S.; Zhang, X.; Li, M. O.; Medzhitov, R.; Franklin, R. A.</p>
        <p class="info">Score: 31.0, Published: 2023-12-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.16.572019' target='https://doi.org/10.1101/2023.12.16.572019'> 10.1101/2023.12.16.572019</a></p>
        <p class="abstract">Antiviral immune mediators, including interferons and their downstream effectors, are critical for host defense yet can become detrimental when uncontrolled. Here, we identify a macrophage-mediated anti-inflammatory mechanism that limits type I interferon (IFN-I) responses. Specifically, we found that cellular stress and pathogen recognition induce Oncostatin M (OSM) production by macrophages. OSM-deficient mice succumbed to challenge with influenza or a viral mimic due to heightened IFN-I activation. Macrophage-derived OSM restricted excessive IFN-I production by lung epithelial cells following viral stimulation. Furthermore, reconstitution of OSM in the respiratory tract was sufficient to protect mice lacking macrophage-derived OSM against morbidity, indicating the importance of local OSM production. This work reveals a host strategy to dampen inflammation in the lung through the negative regulation of IFN-I by macrophages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.571123">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.571123" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.571123">
        <p class="paperTitle">Transposable elements may enhance antiviral resistance in HIV-1 elite controllers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.571123" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.571123" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Singh, M.; Leddy, S. M.; Iniguez, L. P.; Bendall, M. L.; Feschotte, C.; Nixon, D. F.</p>
        <p class="info">Score: 28.7, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.571123' target='https://doi.org/10.1101/2023.12.11.571123'> 10.1101/2023.12.11.571123</a></p>
        <p class="abstract">Less than 0.5% of people living with HIV-1 are elite controllers (ECs) - individuals who have a replication-competent viral reservoir in their CD4&#43; T cells but maintain undetectable plasma viremia without the help of antiretroviral therapy. While the EC CD4&#43; T cell transcriptome has been investigated for gene expression signatures associated with disease progression (or, in this case, a lack thereof), the expression and regulatory activity of transposable elements (TEs) in ECs has not been explored. Yet previous studies have established that TEs can directly impact the immune response to pathogens, including HIV-1. Thus, we hypothesize that the regulatory activities of TEs could contribute to the natural resistance of ECs against HIV-1. We perform a TE-centric analysis of previously published multi-omics data derived from EC individuals and other populations. We find that the CD4&#43; T cell transcriptome and retrotranscriptome of ECs are distinct from healthy controls, treated patients, and viremic progressors. However, there is a substantial level of transcriptomic heterogeneity among ECs. We categorize individuals with distinct chromatin accessibility and expression profiles into four clusters within the EC group, each possessing unique repertoires of TEs and antiviral factors. Notably, several TE families with known immuno-regulatory activity are differentially expressed among ECs. Their transcript levels in ECs positively correlate with their chromatin accessibility and negatively correlate with the expression of their KRAB zinc-finger (KZNF) repressors. This coordinated variation is seen at the level of individual TE loci likely acting or, in some cases, known to act as cis-regulatory elements for nearby genes involved in the immune response and HIV-1 restriction. Based on these results, we propose that the EC phenotype is driven in part by the reduced availability of specific KZNF proteins to repress TE-derived cis-regulatory elements for antiviral genes, thereby heightening their basal level of resistance to HIV-1 infection. Our study reveals considerable heterogeneity in the CD4&#43; T cell transcriptome of ECs, including variable expression of TEs and their KZNF controllers, that must be taken into consideration to decipher the mechanisms enabling HIV-1 control.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571465">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571465" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571465">
        <p class="paperTitle">Massively parallel base editing screens to map variant effects on anti-tumor hallmarks of primary human T cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571465" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571465" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Walsh, Z. H.; Shah, P.; Kothapalli, N.; Nikolenyi, G.; Shah, S.; Leuzzi, G.; Mu, M.; Ho, P.; Abuzaid, S.; Brodtman, Z. D.; Vasan, N.; AlQuaraishi, M. N.; Milner, J. D.; Ciccia, A.; Melms, J. C.; Izar, B.</p>
        <p class="info">Score: 23.6, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571465' target='https://doi.org/10.1101/2023.12.13.571465'> 10.1101/2023.12.13.571465</a></p>
        <p class="abstract">Base editing enables generation of single nucleotide variants, but large-scale screening in primary human T cells is limited due to low editing efficiency, among other challenges1. Here, we developed a high-throughput approach for high-efficiency and massively parallel adenine and cytosine base-editor screening in primary human T cells. We performed multiple large-scale screens editing 102 genes with central functions in T cells and full-length tiling mutagenesis of selected genes, and read out variant effects on hallmarks of T cell anti-tumor immunity, including activation, proliferation, and cytokine production. We discovered a broad landscape of gain- and loss-of-function mutations, including in PIK3CD and its regulatory subunit encoded by PIK3R1, LCK, AKT1, CTLA-4 and JAK1. We identified variants that affected several (e.g., PIK3CD C416R) or only selected (e.g. LCK Y505C) hallmarks of T cell activity, and functionally validated several hits by probing downstream signaling nodes and testing their impact on T cell polyfunctionality and proliferation. Using primary human T cells in which we engineered a T cell receptor (TCR) specific to a commonly presented tumor testis antigen as a model for cellular immunotherapy, we demonstrate that base edits identified in our screens can tune specific or broad T cell functions and ultimately improve tumor elimination while exerting minimal off-target activity. In summary, we present the first large-scale base editing screen in primary human T cells and provide a framework for scalable and targeted base editing at high efficiency. Coupled with multi-modal phenotypic mapping, we accurately nominate variants that produce a desirable T cell state and leverage these synthetic proteins to improve models of cellular cancer immunotherapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571936">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571936" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571936">
        <p class="paperTitle">Opposing effects of pre-existing antibody and memory T cell help on the dynamics of recall germinal centers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571936" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571936" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schiepers, A.; van &#39;t Wout, M. F. L.; Hobbs, A.; Mesin, L.; Victora, G. D.</p>
        <p class="info">Score: 20.8, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571936' target='https://doi.org/10.1101/2023.12.15.571936'> 10.1101/2023.12.15.571936</a></p>
        <p class="abstract">Re-exposure to an antigen generates serum antibody responses that greatly exceed in magnitude those elicited by primary antigen encounter, while simultaneously driving the formation of recall germinal centers (GCs). Although recall GCs in mice are composed almost entirely of naive B cells, recall antibody titers derive overwhelmingly from memory B cells, suggesting a division between cellular and serum compartments. Here, we show that this schism is at least partly explained by a marked decrease in the ability of recall GC B cells to detectably bind antigen. Variant priming and plasmablast ablation experiments show that this decrease is largely due to suppression by preexisting antibody, whereas hapten-carrier experiments reveal a role for memory T cell help in allowing B cells with undetectable antigen binding to access GCs. We propose a model in which antibody-mediated feedback steers recall GC B cells away from previously targeted epitopes, thus enabling specific targeting of variant epitopes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571745">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571745" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571745">
        <p class="paperTitle">50-color phenotyping of the human immune system with in-depth assessment of T cells and dendritic cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571745" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571745" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Konecny, A. J.; Mage, P.; Tyznik, A. J.; Prlic, M.; Mair, F.</p>
        <p class="info">Score: 19.5, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571745' target='https://doi.org/10.1101/2023.12.14.571745'> 10.1101/2023.12.14.571745</a></p>
        <p class="abstract">We report the development of an optimized 50-color spectral flow cytometry panel designed for the in-depth analysis of the immune system in human blood and tissues, with the goal of maximizing the amount of information that can be collected using currently available flow cytometry platforms. We established and tested this panel using peripheral blood mononuclear cells (PBMCs), but included CD45 to enable its use for the analysis of human tissue samples. The panel contains lineage markers for all major immune cell subsets, and an extensive set of phenotyping markers focused on the activation and differentiation status of the T cell and dendritic cell (DC) compartment.

We outline the biological insight that can be gained from the simultaneous measurement of such a large number of proteins and propose that this approach provides a unique opportunity for the comprehensive exploration of the immune status in tissue biopsies and other human samples with a limited number of cells. Of note, we tested the panel to be compatible with cell sorting for further downstream applications. Furthermore, to facilitate the wide-spread implementation of such a panel across different cohorts and samples, we established a trimmed-down 45-color version which can be used with different spectral cytometry platforms.

Finally, to generate this panel, we utilized not only existing panel design guidelines, but also developed new metrics to systematically identify the optimal combination of 50 fluorochromes and evaluate fluorochrome-specific resolution in the context of a 50-color unmixing matrix.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571754">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571754" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571754">
        <p class="paperTitle">Targeting HLA-E Positive Cancers with a Novel NKG2A/C Switch Receptor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571754" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571754" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saetersmoen, M. L.; Kotchetkov, I.; Torralba-Raga, L.; Masilla-Soto, J.; Sohlberg, E.; Krokeide, S. Z.; Hammer, Q.; Sadelain, M.; Malmberg, K.-J.</p>
        <p class="info">Score: 13.9, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571754' target='https://doi.org/10.1101/2023.12.14.571754'> 10.1101/2023.12.14.571754</a></p>
        <p class="abstract">HLA-E is overexpressed by approximately 80% of solid tumors, including malignant glioblastoma, and is emerging as a major checkpoint for NKG2A&#43; CD8&#43; T cells and NK cells in the tumor microenvironment and circulation. This axis operates side-by-side with PD-L1 to shut down effector responses by T and NK cells. Here, we engineered a novel chimeric A/C switch receptor, combining the strong HLA-E binding affinity of the NKG2A receptor ectodomain with the activating signaling of the NKG2C receptor endodomain. We found that A/C Switch-transduced NK and T cells displayed superior and specific cytotoxic function when challenged with tumor cells exhibiting medium to high HLA-E expression. Furthermore, A/C Switch-expressing human T cells demonstrated enhanced anti-tumor function in a xenograft model of glioblastoma. Importantly, the activity of the modified T cells was governed by an equilibrium between A/C Switch transduction level and HLA-E expression, creating a therapeutic window to safeguard against on-target off-tumor toxicities. Indeed, normal cells remained insensitive to A/C Switch engineered T cells even after pre-treatment with IFN-{gamma} to induce HLA-E expression. We propose that this novel A/C switch receptor may operate alone to control tumor cells expressing high levels of HLA-E or in combination with other engineered specificities to overcome the suppressive NKG2A/HLA-E checkpoint.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572199">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572199" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572199">
        <p class="paperTitle">Covalent Targeting of Splicing in T Cells.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572199" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572199" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Scott, K. A.; Kojima, H.; Ropek, N.; Warren, C. D.; Zhang, T. L.; Hogg, S. J.; Webster, C.; Zhang, X.; Rahman, J.; Melillo, B.; Cravatt, B. F.; Lyu, J.; Abdel-Wahab, O.; Vinogradova, E. V.</p>
        <p class="info">Score: 12.8, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572199' target='https://doi.org/10.1101/2023.12.18.572199'> 10.1101/2023.12.18.572199</a></p>
        <p class="abstract">Despite significant interest in therapeutic targeting of splicing, few chemical probes are available for the proteins involved in splicing. Here, we show that elaborated stereoisomeric acrylamide chemical probe EV96 and its analogues lead to a selective T cell state-dependent loss of interleukin 2 inducible T cell kinase (ITK) by targeting one of the core splicing factors SF3B1. Mechanistic investigations suggest that the state-dependency stems from a combination of differential protein turnover rates and availability of functional mRNA pools that can be depleted due to extensive alternative splicing. We further introduce a comprehensive list of proteins involved in splicing and leverage both cysteine- and protein-directed activity-based protein profiling (ABPP) data with electrophilic scout fragments to demonstrate covalent ligandability for many classes of splicing factors and splicing regulators in primary human T cells. Taken together, our findings show how chemical perturbation of splicing can lead to immune state-dependent changes in protein expression and provide evidence for the broad potential to target splicing factors with covalent chemistry.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
