<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>immunology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>immunology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="SARS-CoV-2 antibodies cross-react and enhance dengue infection
Authors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.
Score: 187.0, Published: 2023-10-10 DOI: 10.1101/2023.10.09.557914
Dengue disease is highly prevalent in tropical and subtropical regions worldwide.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/immunology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="immunology" />
<meta property="og:description" content="SARS-CoV-2 antibodies cross-react and enhance dengue infection
Authors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.
Score: 187.0, Published: 2023-10-10 DOI: 10.1101/2023.10.09.557914
Dengue disease is highly prevalent in tropical and subtropical regions worldwide." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/immunology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-15T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-10-15T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="immunology"/>
<meta name="twitter:description" content="SARS-CoV-2 antibodies cross-react and enhance dengue infection
Authors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.
Score: 187.0, Published: 2023-10-10 DOI: 10.1101/2023.10.09.557914
Dengue disease is highly prevalent in tropical and subtropical regions worldwide."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "immunology",
      "item": "https://trxiv.yorks0n.com/posts/immunology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "immunology",
  "name": "immunology",
  "description": "SARS-CoV-2 antibodies cross-react and enhance dengue infection\nAuthors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.\nScore: 187.0, Published: 2023-10-10 DOI: 10.1101/2023.10.09.557914\nDengue disease is highly prevalent in tropical and subtropical regions worldwide.",
  "keywords": [
    
  ],
  "articleBody": " SARS-CoV-2 antibodies cross-react and enhance dengue infection\nAuthors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.\nScore: 187.0, Published: 2023-10-10 DOI: 10.1101/2023.10.09.557914\nDengue disease is highly prevalent in tropical and subtropical regions worldwide. However, its pathogenesis is still incompletely understood, particularly in comparison to other endemic viruses. Antibody-dependent enhancement (ADE) is a well-known phenomenon for dengue viruses. Given the recent surge in dengue cases and potential cross-reactivity with SARS-CoV-2 antibodies, this study explores the impact of anti-SARS-CoV-2 antibodies on DENV-2 infection. The study assessed the cross-reactivity of SARS-CoV-2 antibodies with the DENV-2 Virus. Human convalescent plasma samples collected during different waves of COVID-19 and monoclonal and polyclonal antibodies raised against SARS-CoV-2 were examined for their potential to cause ADE of DENV-2 infection using cell-based assays. The study found that anti-SARS-CoV-2 antibodies acquired from natural infection in humans or through experimental immunization in animals were cross-reactive with DENV-2 and had the potential to enhance DENV-2 infection in K562 and U937 cells. In-silico and in-vitro studies indicated a strong interaction between SARS-CoV-2 antibodies and DENV-2 E-protein, providing a molecular basis for these findings. This study is the first to demonstrate that anti-SARS-CoV-2 antibodies can cross-react with DENV-2 and can enhance its infection through ADE. These findings have implications for SARS-CoV-2 vaccine development and deployment strategies in regions where dengue is endemic.\nQuantifying Adaptive Evolution of the Human Immune Cell Landscape\nAuthors: Salvador-Martinez, I.; Murga-Moreno, J.; Nieto, J. C.; Alsinet, C.; Enard, D.; Heyn, H.\nScore: 41.7, Published: 2023-10-06 DOI: 10.1101/2023.10.06.559946\nThe human immune system is under constant evolutionary pressure, primarily through its role as first line of defence against pathogens. Accordingly, population genomics studies have shown that immune-related genes have a high rate of adaptive evolution. These studies, however, are mainly based on protein-coding genes without cellular context, leaving the adaptive role of cell types and states uncharted. Inferring the rate of protein-coding genes adaptation in developing and adult immune cells at cellular resolution, we found cell types from both the lymphoid and myeloid compartments to harbour significantly increased adaptation rates. Specific cell states, such as foetal Pre-Pro B cells and adult T resident memory CD8+ cells show highly elevated rates of adaptation. We further analysed activated cell states, specifically, iPSC-derived macrophages responding to various challenges, including pro- and anti-inflammatory cytokines or bacterial and viral infections, the latter simulating the evolutionary arms race between humans and pathogens. Here, we found positive selection to be concentrated in early immune responses, suggesting benefits for the host to adapt to early stages of infection to control pathogen numbers and spread. Together, our study reveals spatio-temporal and functional biases in human immune populations with evidence of rapid adaptive evolution and provides a retrospect of forces that shaped the complexity, architecture, and function of the human body.\nHERV activation segregates ME/CFS from fibromyalgia and defines a novel nosological entity for patients fulfilling both clinical criteria\nAuthors: Gimenez-Orenga, K.; Martin-Martinez, E.; Nathanson, L.; Oltra, E.\nScore: 129.7, Published: 2023-10-06 DOI: 10.1101/2023.10.05.561025\nMyalgic Encephalomyelitis/Chronic Fatigue Syndrome (ME/CFS) and fibromyalgia (FM) are chronic diseases with poorly understood pathophysiology and diagnosis based on clinical assessment of unspecific symptoms. The recent post-COVID-19 condition, which shares similarities with ME/CFS and FM, has raised concerns about viral-induced transcriptome changes in post-viral syndromes. Viral infections, and other types of stress, are known to unleash human endogenous retroviruses (HERV) repression that if maintained could lead to symptom chronicity. This study evaluated this possibility for ME/CFS and FM on a selected cohort of female patients complying with diagnosis criteria for ME/CFS, FM, or both, and matched healthy controls (n=43). The results show specific HERV fingerprints for each disease, confirming biological differences between ME/CFS and FM. Unexpectedly, HERV profiles segregated patients that met both ME/CFS and FM clinical criteria from patients complying only with ME or FM criteria, while clearly differentiating patients from healthy subjects, supporting that the highly prevalent comorbidity condition must constitute a different nosological entity. Moreover, HERV profiles exposed significant quantitative differences within the ME/CFS group that correlated with differences in immune gene expression and patient symptomatology, supporting ME/CFS patient subtyping and confirming immunological disturbances in this disease. Pending issues include validation of HERV profiles as disease biomarkers of post-viral syndromes and understanding the role of HERV during infection and beyond.\nGenetic ablation of adhesion ligands averts rejection of allogeneic immune cells\nAuthors: Hammer, Q.; Perica, K.; van Ooijen, H.; Mbofung, R.; Momayyezi, P.; Varady, E.; Martin, K. E.; Pan, Y.; Jelcic, M.; Groff, B.; Abujarour, R.; Krokeide, S.; Lee, T.; Williams, A.; Goodridge, J. P.; Valamehr, B.; Onfelt, B.; Sadelain, M.; Malmberg, K.-J.\nScore: 16.8, Published: 2023-10-09 DOI: 10.1101/2023.10.09.557143\nAllogeneic cell therapies hold promise for broad clinical implementation, but face limitations due to potential rejection by the recipient immune system. Silencing of beta-2-microglobulin (B2M) expression is commonly employed to evade T cell-mediated rejection, although absence of B2M triggers missing-self responses by recipient natural killer (NK) cells. Here, we demonstrate that deletion of the adhesion ligands CD54 and CD58 on targets cells robustly dampens NK cell reactivity across all sub-populations. Genetic deletion of CD54 and CD58 in B2M-deficient allogeneic chimeric antigen receptor (CAR) T and multi-edited induced pluripotent stem cell (iPSC)-derived NK cells reduces their susceptibility to rejection by NK cells in vitro and in vivo without affecting their anti-tumor effector potential. Thus, these data suggest that genetic ablation of adhesion ligands effectively alleviates rejection of allogeneic immune cells for immunotherapy.\nTime- and lineage-resolved transcriptional profiling uncovers gene expression programs and clonal relationships that underlie human T lineage specification\nAuthors: Michaels, Y. S.; Major, M. C.; Bonham-Carter, B.; Zhang, J.; Heydari, T.; Edgar, J. M.; Greenstreet, L.; Vilarrasa-Blasi, R.; Kim, S.; Castle, E. L.; Forrow, A.; Ibanez-Rios, M. I.; Zimmerman, C.; Chung, Y.; Stach, T.; Werschler, N.; Knapp, D. J.; Vento-Tormo, R.; Schiebinger, G.; Zandstra, P. W.\nScore: 6.0, Published: 2023-10-10 DOI: 10.1101/2023.10.06.561277\nT cells develop from multi-potent hematopoietic progenitors in the thymus and provide adaptive protection against pathogens and cancer. However, the emergence of human T cell-competent blood progenitors, and their subsequent specification to the T lineage, has been challenging to capture in real time. Here, we leveraged a pluripotent stem cell differentiation system to understand the transcriptional dynamics and cell fate restriction events that underlie this critical developmental process. Time-resolved single cell RNA sequencing revealed that cell-cycle exit, downregulation of the multipotent hematopoietic program, and upregulation of \u003e90 lineage-associated transcription factors all occur within a highly co-ordinated and narrow developmental window. Computational gene-regulatory network inference elucidated the transcriptional logic of T lineage specification, uncovering an important role for YBX1. We mapped the differentiation cell fate hierarchy using transcribed lineage barcoding and mathematical trajectory inference and discovered that mast and myeloid potential bifurcate from each other early in haematopoiesis, upstream of T lineage restriction. Collectively, our analyses provide a quantitative, time-resolved model of human T cell specification with relevance for regenerative medicine and developmental immunology.\nMast-Cell Expressed Membrane Protein-1 (MCEMP1) is expressed in classical monocytes and alveolar macrophages in Idiopathic Pulmonary Fibrosis and regulates cell chemotaxis, adhesion, and migration in a TGFβ dependent manner.\nAuthors: Perrot, C. Y.; Karampitsakos, T.; Unterman, A.; Adams, T.; Marlin, K.; Arsenault, A.; Zhao, A.; Kaminski, N.; Katlaps, G.; Patel, K.; Bandyopadhyay, D.; Herazo-Maya, J. D.\nScore: 5.5, Published: 2023-10-13 DOI: 10.1101/2023.10.07.561349\nBackground: Mast-Cell Expressed Membrane Protein-1 (MCEMP1) is higher in Idiopathic Pulmonary Fibrosis (IPF) patients with increased risk of death and poor outcomes. Here we seek to establish the mechanistic role of MCEMP1 in pulmonary fibrosis. Methods: MCEMP1 expression was analyzed by single-cell RNA sequencing, immunofluorescence in Peripheral Blood Mononuclear Cells (PBMC) as well as in lung tissues from IPF patients and controls. Chromatin Immunoprecipitation (ChiP) and Proximity Ligation Assay (PLA) were used to study the transcriptional regulation of MCEMP1. Transient RNA interference and lentivirus transduction were used to knockdown and knock-in MCEMP1 in THP-1 cells to study chemotaxis, adhesion, and migration. Bulk RNA sequencing was used to identify the mechanisms by which MCEMP1 participates in monocyte function. Active RHO pull-down assay was used to validate bulk RNA sequencing results. Results: We identified increased MCEMP1 expression in classical monocytes and alveolar macrophages in IPF compared to controls. MCEMP1 was upregulated by TGF{beta} at the mRNA and protein levels in THP-1. TGF{beta}-mediated MCEMP1 upregulation results from the cooperation of SMAD3 and SP1 via concomitant binding to SMAD3/SP1 cis-regulatory elements within the MCEMP1 promoter. In terms of its function, we found that MCEMP1 regulates TGF{beta}-mediated monocyte chemotaxis, adhesion, and migration. 400 differentially expressed genes were found to increase after TGF{beta} stimulation of THP-1, further increased in MCEMP1 knock-in cells treated with TGF{beta} and decreased in MCEMP1 knockdown cells treated with TGF{beta}. GO annotation analysis of these genes showed enrichment for positive regulation of RHO GTPase activity and signal transduction. While TGF{beta} enhanced RHO GTPase activity in THP-1 cells, this effect was attenuated following MCEMP1 knockdown. Conclusion: MCEMP1 is highly expressed in circulating classical monocytes and alveolar macrophages in IPF. MCEMP1 is regulated by TGF{beta} and participates in the chemotaxis, adhesion, and migration of circulating monocytes by modulating the effect of TGF{beta} in RHO activity. Our results suggest that MCEMP1 may regulate the migration and transition of monocytes to monocyte-derived alveolar macrophages during pulmonary fibrosis development and progression.\nmRNA vaccines encoding membrane-anchored receptor-binding domains of SARS-CoV-2 mutants induce strong humoral responses and can overcome immune imprinting\nAuthors: Al-Wassiti, H. A.; Fabb, S. A.; Grimley, S. L.; Kochappan, R.; Ho, J. K.; Wong, C. Y.; Tan, C. W.; Payne, T. J.; Takanashi, A.; Sicilia, H.; Teo, S. L. Y.; McAuley, J. L.; Ellenberg, P.; Cooney, J. P.; Davidson, K. C.; Bowen, R.; Pellegrini, M.; Rockman, S.; Godfrey, D. I.; Nolan, T. M.; Wang, L.-F.; Deliyannis, G.; Purcell, D. F. J.; Pouton, C. W.\nScore: 20.6, Published: 2023-10-04 DOI: 10.1101/2023.10.04.560777\nTo address the limitations of whole-spike COVID vaccines, we explored mRNA vaccines encoding membrane-anchored receptor-binding domain (RBD-TMs), each a fusion of a variant RBD, the transmembrane (TM) and cytoplasmic tail (CT) fragments of the SARS-CoV-2 spike protein. In naive mice, RBD-TM mRNA vaccines against ancestral SARS-CoV-2, Beta, Delta, Delta-plus, Kappa, Omicron BA.1 or BA.5, all induced strong humoral responses against the target RBD. Multiplex surrogate viral neutralization (sVNT) assays indicated broad neutralizing activity against a range of variant RBDs. In the setting of a heterologous boost, against the background of exposure to ancestral whole spike vaccines, sVNT studies suggested that RBD-TM vaccines were able to overcome the detrimental effects of immune imprinting. Omicron BA.1 and BA.5 RBD-TM booster vaccines induced serum antibodies with 12 and 22-fold higher neutralizing activity against the target RBD than their equivalent whole spike variants. Boosting with BA.1 or BA.5 RBD-TM provided good protection against more recent variants including XBB and XBB.1.5. Each RBD-TM mRNA is 28% of the length of its whole-spike equivalent. This advantage will enable tetravalent mRNA vaccines to be developed at well-tolerated doses of formulated mRNA. One Sentence SummarymRNA vaccines encoding membrane-anchored RBDs of SARS-CoV-2 mutants are effective vaccines that can overcome immune imprinting in mice\nDeep learning-guided selection of antibody therapies with enhanced resistance to current and prospective SARS-CoV-2 Omicron variants\nAuthors: Frei, L.; Gao, B.; Han, J.; Taft, J. M.; Irvine, E. B.; Weber, C. R.; Kumar, R.; Eisinger, B.; Reddy, S. T.\nScore: 4.3, Published: 2023-10-10 DOI: 10.1101/2023.10.09.561492\nMost COVID-19 antibody therapies rely on binding the SARS-CoV-2 receptor binding domain (RBD). However, heavily mutated variants such as Omicron and its sublineages, which are characterized by an ever increasing number of mutations in the RBD, have rendered prior antibody therapies ineffective, leaving no clinically approved antibody treatments for SARS-CoV-2. Therefore, the capacity of therapeutic antibody candidates to bind and neutralize current and prospective SARS-CoV-2 variants is a critical factor for drug development. Here, we present a deep learning-guided approach to identify antibodies with enhanced resistance to SARS-CoV-2 evolution. We apply deep mutational learning (DML), a machine learning-guided protein engineering method to interrogate a massive sequence space of combinatorial RBD mutations and predict their impact on angiotensin-converting enzyme 2 (ACE2) binding and antibody escape. A high mutational distance library was constructed based on the full-length RBD of Omicron BA.1, which was experimentally screened for binding to the ACE2 receptor or neutralizing antibodies, followed by deep sequencing. The resulting data was used to train ensemble deep learning models that could accurately predict binding or escape for a panel of therapeutic antibody candidates targeting diverse RBD epitopes. Furthermore, antibody breadth was assessed by predicting binding or escape to synthetic lineages that represent millions of sequences generated using in silico evolution, revealing combinations with complementary and enhanced resistance to viral evolution. This deep learning approach may enable the design of next-generation antibody therapies that remain effective against future SARS-CoV-2 variants.\nA novel HIV triple broadly neutralizing antibody (bNAb) combination-based passive immunization of infant rhesus macaques achieves durable protective plasma neutralization levels and mediates anti-viral effector functions.\nAuthors: Dankwa, S.; Kosman, C.; Dennis, M.; Giorgi, E. E.; Vuong, K.; Pahountis, I.; Garza, A.; McCarthy, J.; Mayer, B. T.; Ngo, J. T.; Enemuo, C. A.; Carnathan, D. G.; Stanfield-Oakley, S.; Berendam, S. J.; Weinbaum, C.; Engelman, K.; Magnani, D. M.; Chan, C.; Ferrari, G.; Silvestri, G.; Amara, R. R.; Chahroudi, A.; Permar, S. R.; Fouda, G. G.; Goswami, R.\nScore: 4.1, Published: 2023-10-10 DOI: 10.1101/2023.10.09.561559\nTo eliminate vertical HIV transmission and achieve therapy-free viral suppression among children living with HIV, novel strategies beyond antiretroviral therapy (ART) are necessary. Our group previously identified a triple bNAb combination comprising of 3BNC117, PGDM1400 and PGT151 that mediates robust in vitro neutralization and non-neutralizing effector functions against a cross-clade panel of simian human immunodeficiency viruses (SHIVs). In this study, we evaluated the safety, pharmacokinetics, and antiviral potency of this bNAb combination in infant rhesus macaques (RMs). We demonstrate that subcutaneous infusion of the triple bNAb regimen was well tolerated in pediatric monkeys and resulted in durable systemic and mucosal distribution. Plasma obtained from passively-immunized RMs demonstrated potent HIV-neutralizing and Fc-mediated antiviral effector functions. Finally, using the predicted serum neutralization 80% inhibitory dilution titer (PT80) biomarker, which was recently identified as a surrogate endpoint for evaluation of the preventative efficacy (PE) of bNAbs against mucosal viral acquisition in human clinical trials, we demonstrated that our regimen has a predicted PE[\u0026ge;]90% against a large panel of plasma and breast milk-derived HIV strains and cross-clade SHIV variants. This data will guide the development of combination bNAbs for eliminating vertical HIV transmission and for achieving ART-free viral suppression among children living with HIV.\nJAK/STAT mediated insulin resistance in muscles is essential for effective immune response\nAuthors: McMullen, E.; Strych, L.; Chodakova, L.; Krebs, A.; Dolezal, T.\nScore: 4.0, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560867\nThe metabolically demanding nature of immune response requires nutrients to be preferentially directed towards the immune system at the expense of peripheral tissues. We study the mechanisms by which this metabolic reprograming occurs using the parasitoid infection of Drosophila larvae. To overcome such an immune challenge hemocytes differentiate into lamellocytes, which encapsulate and melanize the parasitoid egg. Hemocytes acquire the energy for this process by expressing JAK/STAT ligands upd2 and upd3, which activates JAK/STAT signaling in muscles and redirects carbohydrates away from muscles in favor of immune cells. Absence of upd/JAK/STAT signaling leads to an impaired immune response and increased mortality. We demonstrate how JAK/STAT signaling in muscles leads to suppression of insulin signaling through activation of ImpL2, the inhibitor of Drosophila insulin like peptides. We reveal the crucial function of muscles during immune response and show the benefits of insulin resistance as an adaptive mechanism that is necessary for survival.\n",
  "wordCount" : "2619",
  "inLanguage": "en",
  "datePublished": "2023-10-15T10:38:01Z",
  "dateModified": "2023-10-15T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/immunology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      immunology
    </h1>
    <div class="post-meta"><span>updated on October 15, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.557914">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.557914" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.557914">
        <p class="paperTitle">SARS-CoV-2 antibodies cross-react and enhance dengue infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.557914" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.557914" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.</p>
        <p class="info">Score: 187.0, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.557914' target='https://doi.org/10.1101/2023.10.09.557914'> 10.1101/2023.10.09.557914</a></p>
        <p class="abstract">Dengue disease is highly prevalent in tropical and subtropical regions worldwide. However, its pathogenesis is still incompletely understood, particularly in comparison to other endemic viruses. Antibody-dependent enhancement (ADE) is a well-known phenomenon for dengue viruses. Given the recent surge in dengue cases and potential cross-reactivity with SARS-CoV-2 antibodies, this study explores the impact of anti-SARS-CoV-2 antibodies on DENV-2 infection. The study assessed the cross-reactivity of SARS-CoV-2 antibodies with the DENV-2 Virus. Human convalescent plasma samples collected during different waves of COVID-19 and monoclonal and polyclonal antibodies raised against SARS-CoV-2 were examined for their potential to cause ADE of DENV-2 infection using cell-based assays. The study found that anti-SARS-CoV-2 antibodies acquired from natural infection in humans or through experimental immunization in animals were cross-reactive with DENV-2 and had the potential to enhance DENV-2 infection in K562 and U937 cells. In-silico and in-vitro studies indicated a strong interaction between SARS-CoV-2 antibodies and DENV-2 E-protein, providing a molecular basis for these findings. This study is the first to demonstrate that anti-SARS-CoV-2 antibodies can cross-react with DENV-2 and can enhance its infection through ADE. These findings have implications for SARS-CoV-2 vaccine development and deployment strategies in regions where dengue is endemic.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.559946">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.559946" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.559946">
        <p class="paperTitle">Quantifying Adaptive Evolution of the Human Immune Cell Landscape</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.559946" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.559946" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Salvador-Martinez, I.; Murga-Moreno, J.; Nieto, J. C.; Alsinet, C.; Enard, D.; Heyn, H.</p>
        <p class="info">Score: 41.7, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.559946' target='https://doi.org/10.1101/2023.10.06.559946'> 10.1101/2023.10.06.559946</a></p>
        <p class="abstract">The human immune system is under constant evolutionary pressure, primarily through its role as first line of defence against pathogens. Accordingly, population genomics studies have shown that immune-related genes have a high rate of adaptive evolution. These studies, however, are mainly based on protein-coding genes without cellular context, leaving the adaptive role of cell types and states uncharted. Inferring the rate of protein-coding genes adaptation in developing and adult immune cells at cellular resolution, we found cell types from both the lymphoid and myeloid compartments to harbour significantly increased adaptation rates. Specific cell states, such as foetal Pre-Pro B cells and adult T resident memory CD8&#43; cells show highly elevated rates of adaptation. We further analysed activated cell states, specifically, iPSC-derived macrophages responding to various challenges, including pro- and anti-inflammatory cytokines or bacterial and viral infections, the latter simulating the evolutionary arms race between humans and pathogens. Here, we found positive selection to be concentrated in early immune responses, suggesting benefits for the host to adapt to early stages of infection to control pathogen numbers and spread. Together, our study reveals spatio-temporal and functional biases in human immune populations with evidence of rapid adaptive evolution and provides a retrospect of forces that shaped the complexity, architecture, and function of the human body.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561025">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561025" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561025">
        <p class="paperTitle">HERV activation segregates ME/CFS from fibromyalgia and defines a novel nosological entity for patients fulfilling both clinical criteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561025" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561025" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gimenez-Orenga, K.; Martin-Martinez, E.; Nathanson, L.; Oltra, E.</p>
        <p class="info">Score: 129.7, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561025' target='https://doi.org/10.1101/2023.10.05.561025'> 10.1101/2023.10.05.561025</a></p>
        <p class="abstract">Myalgic Encephalomyelitis/Chronic Fatigue Syndrome (ME/CFS) and fibromyalgia (FM) are chronic diseases with poorly understood pathophysiology and diagnosis based on clinical assessment of unspecific symptoms. The recent post-COVID-19 condition, which shares similarities with ME/CFS and FM, has raised concerns about viral-induced transcriptome changes in post-viral syndromes. Viral infections, and other types of stress, are known to unleash human endogenous retroviruses (HERV) repression that if maintained could lead to symptom chronicity. This study evaluated this possibility for ME/CFS and FM on a selected cohort of female patients complying with diagnosis criteria for ME/CFS, FM, or both, and matched healthy controls (n=43). The results show specific HERV fingerprints for each disease, confirming biological differences between ME/CFS and FM. Unexpectedly, HERV profiles segregated patients that met both ME/CFS and FM clinical criteria from patients complying only with ME or FM criteria, while clearly differentiating patients from healthy subjects, supporting that the highly prevalent comorbidity condition must constitute a different nosological entity. Moreover, HERV profiles exposed significant quantitative differences within the ME/CFS group that correlated with differences in immune gene expression and patient symptomatology, supporting ME/CFS patient subtyping and confirming immunological disturbances in this disease. Pending issues include validation of HERV profiles as disease biomarkers of post-viral syndromes and understanding the role of HERV during infection and beyond.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.557143">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.557143" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.557143">
        <p class="paperTitle">Genetic ablation of adhesion ligands averts rejection of allogeneic immune cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.557143" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.557143" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hammer, Q.; Perica, K.; van Ooijen, H.; Mbofung, R.; Momayyezi, P.; Varady, E.; Martin, K. E.; Pan, Y.; Jelcic, M.; Groff, B.; Abujarour, R.; Krokeide, S.; Lee, T.; Williams, A.; Goodridge, J. P.; Valamehr, B.; Onfelt, B.; Sadelain, M.; Malmberg, K.-J.</p>
        <p class="info">Score: 16.8, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.557143' target='https://doi.org/10.1101/2023.10.09.557143'> 10.1101/2023.10.09.557143</a></p>
        <p class="abstract">Allogeneic cell therapies hold promise for broad clinical implementation, but face limitations due to potential rejection by the recipient immune system. Silencing of beta-2-microglobulin (B2M) expression is commonly employed to evade T cell-mediated rejection, although absence of B2M triggers missing-self responses by recipient natural killer (NK) cells. Here, we demonstrate that deletion of the adhesion ligands CD54 and CD58 on targets cells robustly dampens NK cell reactivity across all sub-populations. Genetic deletion of CD54 and CD58 in B2M-deficient allogeneic chimeric antigen receptor (CAR) T and multi-edited induced pluripotent stem cell (iPSC)-derived NK cells reduces their susceptibility to rejection by NK cells in vitro and in vivo without affecting their anti-tumor effector potential. Thus, these data suggest that genetic ablation of adhesion ligands effectively alleviates rejection of allogeneic immune cells for immunotherapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561277">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561277" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561277">
        <p class="paperTitle">Time- and lineage-resolved transcriptional profiling uncovers gene expression programs and clonal relationships that underlie human T lineage specification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561277" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561277" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Michaels, Y. S.; Major, M. C.; Bonham-Carter, B.; Zhang, J.; Heydari, T.; Edgar, J. M.; Greenstreet, L.; Vilarrasa-Blasi, R.; Kim, S.; Castle, E. L.; Forrow, A.; Ibanez-Rios, M. I.; Zimmerman, C.; Chung, Y.; Stach, T.; Werschler, N.; Knapp, D. J.; Vento-Tormo, R.; Schiebinger, G.; Zandstra, P. W.</p>
        <p class="info">Score: 6.0, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561277' target='https://doi.org/10.1101/2023.10.06.561277'> 10.1101/2023.10.06.561277</a></p>
        <p class="abstract">T cells develop from multi-potent hematopoietic progenitors in the thymus and provide adaptive protection against pathogens and cancer. However, the emergence of human T cell-competent blood progenitors, and their subsequent specification to the T lineage, has been challenging to capture in real time. Here, we leveraged a pluripotent stem cell differentiation system to understand the transcriptional dynamics and cell fate restriction events that underlie this critical developmental process. Time-resolved single cell RNA sequencing revealed that cell-cycle exit, downregulation of the multipotent hematopoietic program, and upregulation of &gt;90 lineage-associated transcription factors all occur within a highly co-ordinated and narrow developmental window. Computational gene-regulatory network inference elucidated the transcriptional logic of T lineage specification, uncovering an important role for YBX1. We mapped the differentiation cell fate hierarchy using transcribed lineage barcoding and mathematical trajectory inference and discovered that mast and myeloid potential bifurcate from each other early in haematopoiesis, upstream of T lineage restriction. Collectively, our analyses provide a quantitative, time-resolved model of human T cell specification with relevance for regenerative medicine and developmental immunology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.07.561349">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.07.561349" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.07.561349">
        <p class="paperTitle">Mast-Cell Expressed Membrane Protein-1 (MCEMP1) is expressed in classical monocytes and alveolar macrophages in Idiopathic Pulmonary Fibrosis and regulates cell chemotaxis, adhesion, and migration in a TGFβ dependent manner.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.07.561349" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.07.561349" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Perrot, C. Y.; Karampitsakos, T.; Unterman, A.; Adams, T.; Marlin, K.; Arsenault, A.; Zhao, A.; Kaminski, N.; Katlaps, G.; Patel, K.; Bandyopadhyay, D.; Herazo-Maya, J. D.</p>
        <p class="info">Score: 5.5, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.07.561349' target='https://doi.org/10.1101/2023.10.07.561349'> 10.1101/2023.10.07.561349</a></p>
        <p class="abstract">Background: Mast-Cell Expressed Membrane Protein-1 (MCEMP1) is higher in Idiopathic Pulmonary Fibrosis (IPF) patients with increased risk of death and poor outcomes. Here we seek to establish the mechanistic role of MCEMP1 in pulmonary fibrosis. Methods: MCEMP1 expression was analyzed by single-cell RNA sequencing, immunofluorescence in Peripheral Blood Mononuclear Cells (PBMC) as well as in lung tissues from IPF patients and controls. Chromatin Immunoprecipitation (ChiP) and Proximity Ligation Assay (PLA) were used to study the transcriptional regulation of MCEMP1. Transient RNA interference and lentivirus transduction were used to knockdown and knock-in MCEMP1 in THP-1 cells to study chemotaxis, adhesion, and migration. Bulk RNA sequencing was used to identify the mechanisms by which MCEMP1 participates in monocyte function. Active RHO pull-down assay was used to validate bulk RNA sequencing results. Results: We identified increased MCEMP1 expression in classical monocytes and alveolar macrophages in IPF compared to controls. MCEMP1 was upregulated by TGF{beta} at the mRNA and protein levels in THP-1. TGF{beta}-mediated MCEMP1 upregulation results from the cooperation of SMAD3 and SP1 via concomitant binding to SMAD3/SP1 cis-regulatory elements within the MCEMP1 promoter. In terms of its function, we found that MCEMP1 regulates TGF{beta}-mediated monocyte chemotaxis, adhesion, and migration. 400 differentially expressed genes were found to increase after TGF{beta} stimulation of THP-1, further increased in MCEMP1 knock-in cells treated with TGF{beta} and decreased in MCEMP1 knockdown cells treated with TGF{beta}. GO annotation analysis of these genes showed enrichment for positive regulation of RHO GTPase activity and signal transduction. While TGF{beta} enhanced RHO GTPase activity in THP-1 cells, this effect was attenuated following MCEMP1 knockdown. Conclusion: MCEMP1 is highly expressed in circulating classical monocytes and alveolar macrophages in IPF. MCEMP1 is regulated by TGF{beta} and participates in the chemotaxis, adhesion, and migration of circulating monocytes by modulating the effect of TGF{beta} in RHO activity. Our results suggest that MCEMP1 may regulate the migration and transition of monocytes to monocyte-derived alveolar macrophages during pulmonary fibrosis development and progression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560777">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560777" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560777">
        <p class="paperTitle">mRNA vaccines encoding membrane-anchored receptor-binding domains of SARS-CoV-2 mutants induce strong humoral responses and can overcome immune imprinting</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560777" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560777" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Al-Wassiti, H. A.; Fabb, S. A.; Grimley, S. L.; Kochappan, R.; Ho, J. K.; Wong, C. Y.; Tan, C. W.; Payne, T. J.; Takanashi, A.; Sicilia, H.; Teo, S. L. Y.; McAuley, J. L.; Ellenberg, P.; Cooney, J. P.; Davidson, K. C.; Bowen, R.; Pellegrini, M.; Rockman, S.; Godfrey, D. I.; Nolan, T. M.; Wang, L.-F.; Deliyannis, G.; Purcell, D. F. J.; Pouton, C. W.</p>
        <p class="info">Score: 20.6, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560777' target='https://doi.org/10.1101/2023.10.04.560777'> 10.1101/2023.10.04.560777</a></p>
        <p class="abstract">To address the limitations of whole-spike COVID vaccines, we explored mRNA vaccines encoding membrane-anchored receptor-binding domain (RBD-TMs), each a fusion of a variant RBD, the transmembrane (TM) and cytoplasmic tail (CT) fragments of the SARS-CoV-2 spike protein. In naive mice, RBD-TM mRNA vaccines against ancestral SARS-CoV-2, Beta, Delta, Delta-plus, Kappa, Omicron BA.1 or BA.5, all induced strong humoral responses against the target RBD. Multiplex surrogate viral neutralization (sVNT) assays indicated broad neutralizing activity against a range of variant RBDs. In the setting of a heterologous boost, against the background of exposure to ancestral whole spike vaccines, sVNT studies suggested that RBD-TM vaccines were able to overcome the detrimental effects of immune imprinting. Omicron BA.1 and BA.5 RBD-TM booster vaccines induced serum antibodies with 12 and 22-fold higher neutralizing activity against the target RBD than their equivalent whole spike variants. Boosting with BA.1 or BA.5 RBD-TM provided good protection against more recent variants including XBB and XBB.1.5. Each RBD-TM mRNA is 28% of the length of its whole-spike equivalent. This advantage will enable tetravalent mRNA vaccines to be developed at well-tolerated doses of formulated mRNA.

One Sentence SummarymRNA vaccines encoding membrane-anchored RBDs of SARS-CoV-2 mutants are effective vaccines that can overcome immune imprinting in mice</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561492">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561492" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561492">
        <p class="paperTitle">Deep learning-guided selection of antibody therapies with enhanced resistance to current and prospective SARS-CoV-2 Omicron variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561492" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561492" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Frei, L.; Gao, B.; Han, J.; Taft, J. M.; Irvine, E. B.; Weber, C. R.; Kumar, R.; Eisinger, B.; Reddy, S. T.</p>
        <p class="info">Score: 4.3, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561492' target='https://doi.org/10.1101/2023.10.09.561492'> 10.1101/2023.10.09.561492</a></p>
        <p class="abstract">Most COVID-19 antibody therapies rely on binding the SARS-CoV-2 receptor binding domain (RBD). However, heavily mutated variants such as Omicron and its sublineages, which are characterized by an ever increasing number of mutations in the RBD, have rendered prior antibody therapies ineffective, leaving no clinically approved antibody treatments for SARS-CoV-2. Therefore, the capacity of therapeutic antibody candidates to bind and neutralize current and prospective SARS-CoV-2 variants is a critical factor for drug development. Here, we present a deep learning-guided approach to identify antibodies with enhanced resistance to SARS-CoV-2 evolution. We apply deep mutational learning (DML), a machine learning-guided protein engineering method to interrogate a massive sequence space of combinatorial RBD mutations and predict their impact on angiotensin-converting enzyme 2 (ACE2) binding and antibody escape. A high mutational distance library was constructed based on the full-length RBD of Omicron BA.1, which was experimentally screened for binding to the ACE2 receptor or neutralizing antibodies, followed by deep sequencing. The resulting data was used to train ensemble deep learning models that could accurately predict binding or escape for a panel of therapeutic antibody candidates targeting diverse RBD epitopes. Furthermore, antibody breadth was assessed by predicting binding or escape to synthetic lineages that represent millions of sequences generated using in silico evolution, revealing combinations with complementary and enhanced resistance to viral evolution. This deep learning approach may enable the design of next-generation antibody therapies that remain effective against future SARS-CoV-2 variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561559">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561559" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561559">
        <p class="paperTitle">A novel HIV triple broadly neutralizing antibody (bNAb) combination-based passive immunization of infant rhesus macaques achieves durable protective plasma neutralization levels and mediates anti-viral effector functions.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561559" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561559" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dankwa, S.; Kosman, C.; Dennis, M.; Giorgi, E. E.; Vuong, K.; Pahountis, I.; Garza, A.; McCarthy, J.; Mayer, B. T.; Ngo, J. T.; Enemuo, C. A.; Carnathan, D. G.; Stanfield-Oakley, S.; Berendam, S. J.; Weinbaum, C.; Engelman, K.; Magnani, D. M.; Chan, C.; Ferrari, G.; Silvestri, G.; Amara, R. R.; Chahroudi, A.; Permar, S. R.; Fouda, G. G.; Goswami, R.</p>
        <p class="info">Score: 4.1, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561559' target='https://doi.org/10.1101/2023.10.09.561559'> 10.1101/2023.10.09.561559</a></p>
        <p class="abstract">To eliminate vertical HIV transmission and achieve therapy-free viral suppression among children living with HIV, novel strategies beyond antiretroviral therapy (ART) are necessary. Our group previously identified a triple bNAb combination comprising of 3BNC117, PGDM1400 and PGT151 that mediates robust in vitro neutralization and non-neutralizing effector functions against a cross-clade panel of simian human immunodeficiency viruses (SHIVs). In this study, we evaluated the safety, pharmacokinetics, and antiviral potency of this bNAb combination in infant rhesus macaques (RMs). We demonstrate that subcutaneous infusion of the triple bNAb regimen was well tolerated in pediatric monkeys and resulted in durable systemic and mucosal distribution. Plasma obtained from passively-immunized RMs demonstrated potent HIV-neutralizing and Fc-mediated antiviral effector functions. Finally, using the predicted serum neutralization 80% inhibitory dilution titer (PT80) biomarker, which was recently identified as a surrogate endpoint for evaluation of the preventative efficacy (PE) of bNAbs against mucosal viral acquisition in human clinical trials, we demonstrated that our regimen has a predicted PE[&amp;ge;]90% against a large panel of plasma and breast milk-derived HIV strains and cross-clade SHIV variants. This data will guide the development of combination bNAbs for eliminating vertical HIV transmission and for achieving ART-free viral suppression among children living with HIV.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560867">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560867" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560867">
        <p class="paperTitle">JAK/STAT mediated insulin resistance in muscles is essential for effective immune response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560867" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560867" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McMullen, E.; Strych, L.; Chodakova, L.; Krebs, A.; Dolezal, T.</p>
        <p class="info">Score: 4.0, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560867' target='https://doi.org/10.1101/2023.10.04.560867'> 10.1101/2023.10.04.560867</a></p>
        <p class="abstract">The metabolically demanding nature of immune response requires nutrients to be preferentially directed towards the immune system at the expense of peripheral tissues. We study the mechanisms by which this metabolic reprograming occurs using the parasitoid infection of Drosophila larvae. To overcome such an immune challenge hemocytes differentiate into lamellocytes, which encapsulate and melanize the parasitoid egg. Hemocytes acquire the energy for this process by expressing JAK/STAT ligands upd2 and upd3, which activates JAK/STAT signaling in muscles and redirects carbohydrates away from muscles in favor of immune cells. Absence of upd/JAK/STAT signaling leads to an impaired immune response and increased mortality. We demonstrate how JAK/STAT signaling in muscles leads to suppression of insulin signaling through activation of ImpL2, the inhibitor of Drosophila insulin like peptides. We reveal the crucial function of muscles during immune response and show the benefits of insulin resistance as an adaptive mechanism that is necessary for survival.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
