<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>immunology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>immunology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="HERV activation segregates ME/CFS from fibromyalgia and defines a novel nosological entity for patients fulfilling both clinical criteria
Authors: Gimenez-Orenga, K.; Martin-Martinez, E.; Nathanson, L.; Oltra, E.
Score: 107.5, Published: 2023-10-06 DOI: 10.1101/2023.10.05.561025
Myalgic Encephalomyelitis/Chronic Fatigue Syndrome (ME/CFS) and fibromyalgia (FM) are chronic diseases with poorly understood pathophysiology and diagnosis based on clinical assessment of unspecific symptoms. The recent post-COVID-19 condition, which shares similarities with ME/CFS and FM, has raised concerns about viral-induced transcriptome changes in post-viral syndromes.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/immunology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="immunology" />
<meta property="og:description" content="HERV activation segregates ME/CFS from fibromyalgia and defines a novel nosological entity for patients fulfilling both clinical criteria
Authors: Gimenez-Orenga, K.; Martin-Martinez, E.; Nathanson, L.; Oltra, E.
Score: 107.5, Published: 2023-10-06 DOI: 10.1101/2023.10.05.561025
Myalgic Encephalomyelitis/Chronic Fatigue Syndrome (ME/CFS) and fibromyalgia (FM) are chronic diseases with poorly understood pathophysiology and diagnosis based on clinical assessment of unspecific symptoms. The recent post-COVID-19 condition, which shares similarities with ME/CFS and FM, has raised concerns about viral-induced transcriptome changes in post-viral syndromes." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/immunology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-08T10:37:16+00:00" />
<meta property="article:modified_time" content="2023-10-08T10:37:16+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="immunology"/>
<meta name="twitter:description" content="HERV activation segregates ME/CFS from fibromyalgia and defines a novel nosological entity for patients fulfilling both clinical criteria
Authors: Gimenez-Orenga, K.; Martin-Martinez, E.; Nathanson, L.; Oltra, E.
Score: 107.5, Published: 2023-10-06 DOI: 10.1101/2023.10.05.561025
Myalgic Encephalomyelitis/Chronic Fatigue Syndrome (ME/CFS) and fibromyalgia (FM) are chronic diseases with poorly understood pathophysiology and diagnosis based on clinical assessment of unspecific symptoms. The recent post-COVID-19 condition, which shares similarities with ME/CFS and FM, has raised concerns about viral-induced transcriptome changes in post-viral syndromes."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "immunology",
      "item": "https://trxiv.yorks0n.com/posts/immunology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "immunology",
  "name": "immunology",
  "description": "HERV activation segregates ME/CFS from fibromyalgia and defines a novel nosological entity for patients fulfilling both clinical criteria\nAuthors: Gimenez-Orenga, K.; Martin-Martinez, E.; Nathanson, L.; Oltra, E.\nScore: 107.5, Published: 2023-10-06 DOI: 10.1101/2023.10.05.561025\nMyalgic Encephalomyelitis/Chronic Fatigue Syndrome (ME/CFS) and fibromyalgia (FM) are chronic diseases with poorly understood pathophysiology and diagnosis based on clinical assessment of unspecific symptoms. The recent post-COVID-19 condition, which shares similarities with ME/CFS and FM, has raised concerns about viral-induced transcriptome changes in post-viral syndromes.",
  "keywords": [
    
  ],
  "articleBody": " HERV activation segregates ME/CFS from fibromyalgia and defines a novel nosological entity for patients fulfilling both clinical criteria\nAuthors: Gimenez-Orenga, K.; Martin-Martinez, E.; Nathanson, L.; Oltra, E.\nScore: 107.5, Published: 2023-10-06 DOI: 10.1101/2023.10.05.561025\nMyalgic Encephalomyelitis/Chronic Fatigue Syndrome (ME/CFS) and fibromyalgia (FM) are chronic diseases with poorly understood pathophysiology and diagnosis based on clinical assessment of unspecific symptoms. The recent post-COVID-19 condition, which shares similarities with ME/CFS and FM, has raised concerns about viral-induced transcriptome changes in post-viral syndromes. Viral infections, and other types of stress, are known to unleash human endogenous retroviruses (HERV) repression that if maintained could lead to symptom chronicity. This study evaluated this possibility for ME/CFS and FM on a selected cohort of female patients complying with diagnosis criteria for ME/CFS, FM, or both, and matched healthy controls (n=43). The results show specific HERV fingerprints for each disease, confirming biological differences between ME/CFS and FM. Unexpectedly, HERV profiles segregated patients that met both ME/CFS and FM clinical criteria from patients complying only with ME or FM criteria, while clearly differentiating patients from healthy subjects, supporting that the highly prevalent comorbidity condition must constitute a different nosological entity. Moreover, HERV profiles exposed significant quantitative differences within the ME/CFS group that correlated with differences in immune gene expression and patient symptomatology, supporting ME/CFS patient subtyping and confirming immunological disturbances in this disease. Pending issues include validation of HERV profiles as disease biomarkers of post-viral syndromes and understanding the role of HERV during infection and beyond.\nNeutralisation of SARS-CoV-2 Omicron subvariants BA.2.86 and EG.5.1 by antibodies induced by earlier infection or vaccination\nAuthors: Lassauniere, R.; Polacek, C.; Baig, S.; Ellegaard, K.; Escobar-Herrera, L. A.; Fomsgaard, A.; Spiess, K.; Schwartz, O.; Planas, D.; Simon-Loriere, E.; Schneider, U. V.; Sieber, R. N.; Stegger, M.; Krause, T. G.; Ullum, H.; Jokelainen, P.; Rasmussen, M.\nScore: 91.6, Published: 2023-10-02 DOI: 10.1101/2023.10.01.560365\nHighly mutated SARS-CoV-2 Omicron subvariant BA.2.86 emerged in July 2023. We investigated the neutralisation of isolated virus by antibodies induced by earlier infection or vaccination. The neutralisation titres for BA.2.86 were comparable to those for XBB.1 and EG.5.1, by antibodies induced by XBB.1.5 or BA.4/5 breakthrough infection or BA.4/5 vaccination.\nQuantifying Adaptive Evolution of the Human Immune Cell Landscape\nAuthors: Salvador-Martinez, I.; Murga-Moreno, J.; Nieto, J. C.; Alsinet, C.; Enard, D.; Heyn, H.\nScore: 28.4, Published: 2023-10-06 DOI: 10.1101/2023.10.06.559946\nThe human immune system is under constant evolutionary pressure, primarily through its role as first line of defence against pathogens. Accordingly, population genomics studies have shown that immune-related genes have a high rate of adaptive evolution. These studies, however, are mainly based on protein-coding genes without cellular context, leaving the adaptive role of cell types and states uncharted. Inferring the rate of protein-coding genes adaptation in developing and adult immune cells at cellular resolution, we found cell types from both the lymphoid and myeloid compartments to harbour significantly increased adaptation rates. Specific cell states, such as foetal Pre-Pro B cells and adult T resident memory CD8+ cells show highly elevated rates of adaptation. We further analysed activated cell states, specifically, iPSC-derived macrophages responding to various challenges, including pro- and anti-inflammatory cytokines or bacterial and viral infections, the latter simulating the evolutionary arms race between humans and pathogens. Here, we found positive selection to be concentrated in early immune responses, suggesting benefits for the host to adapt to early stages of infection to control pathogen numbers and spread. Together, our study reveals spatio-temporal and functional biases in human immune populations with evidence of rapid adaptive evolution and provides a retrospect of forces that shaped the complexity, architecture, and function of the human body.\nHuman intestinal organoids with an autologoustissue-resident immune compartment\nAuthors: Recaldin, T.; Gjeta, B.; Steinacher, L.; Harter, M. F.; Adam, L.; Nikolaev, M.; Krese, R.; Kilik, U.; Popovic, D.; Almato-Bellavista, M.; Kromer, K.; Bscheider, M.; Cabon, L.; Camp, J. G.; Gjorevski, N.\nScore: 23.0, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560810\nThe intimate relationship between the epithelium and the immune system is crucial for maintaining tissue homeostasis, with perturbations in epithelial-immune interactions linked to autoimmune disease and cancer. Whereas stem cell-derived organoids are powerful models of tissue-specific epithelial function, these structures lack tissue-resident immune cells that are essential for capturing organ-level processes. We describe human intestinal immuno-organoids (IIOs), formed through self-organization of epithelial organoids and autologous tissueresident lymphocytes (TRMs), a portion of which integrate within the IIO epithelium and survey the barrier. IIO formation was driven by TRM migration and interaction with epithelial cells, as orchestrated by TRM- enriched transcriptomic programs governing cell motility and epithelial inspection. We combined IIOs and single-cell transcriptomics to investigate intestinal inflammation triggered by cancer-targeting biologics in patients, and found that the system recapitulates clinical outcomes and the underlying cellular mechanisms. Inflammation was associated with the emergence of an activated population of CD8+ T cells, which progressively acquired intraepithelial and cytotoxic features. The appearance of this effector population was preceded and likely mediated by a Th1- like CD4+ population, which initially displayed a cytokine producing character and subsequently became cytotoxic itself. A system amenable to direct perturbation and interrogation, IIOs allowed us to identify the Rho pathway as a novel target for mitigating immunotherapy-associated intestinal inflammation. Given that they recapitulate both the phenotypic outcomes and the underlying inter-lineage immune interactions, IIOs can be used to broadly study tissue-resident immune responses in the context of tumorigenesis, infectious and autoimmune diseases.\nProtective non-neutralizing mAbs targets conserved opsonic epitopes on SARS-CoV-2 variants\nAuthors: Izadi, A.; Godzwon, M.; Ohlin, M.; Nordenfelt, P.\nScore: 33.7, Published: 2023-09-29 DOI: 10.1101/2023.09.29.560084\nAntibodies play a central role in the immune defense against SARS-CoV-2. Strong evidence has shown that non-neutralizing antibodies (nnAbs) are important for anti-SARS-Cov-2 immunity through Fc-mediated effector functions. These nnAbs bind to epitopes that could be less subjected to mutations in the emerging variants. When protective, such nnAbs would constitute a more promising alternative to neutralizing mAbs (nAbs). Here, we show that six nnAbs retain binding to Omicron, while two nAbs do not. Furthermore, two of our nnAbs, which are protective in vivo, retained binding to XBB, XBB.1.5, and BQ.1.1. They appear to bind to conserved epitopes on the N-terminal and receptor binding domain (RBD), respectively. As a proof of concept, we show that these protective non-neutralizing antibodies retain potent Fc-mediated opsonic function against BQ.1.1 and XBB. We also show that the Fc-mediated function is further enhanced by expressing the antibodies in the IgG3 subclass and combining them into a dual antibody cocktail. Our work suggests that opsonizing nnAbs could be a viable strategy for anti-SARS-CoV-2 mAb therapies against current and future SARS-CoV-2 variants.\nmRNA vaccines encoding membrane-anchored receptor-binding domains of SARS-CoV-2 mutants induce strong humoral responses and can overcome immune imprinting\nAuthors: Al-Wassiti, H. A.; Fabb, S. A.; Grimley, S. L.; Kochappan, R.; Ho, J. K.; Wong, C. Y.; Tan, C. W.; Payne, T. J.; Takanashi, A.; Sicilia, H.; Teo, S. L. Y.; McAuley, J. L.; Ellenberg, P.; Cooney, J. P.; Davidson, K. C.; Bowen, R.; Pellegrini, M.; Rockman, S.; Godfrey, D. I.; Nolan, T. M.; Wang, L.-F.; Deliyannis, G.; Purcell, D. F. J.; Pouton, C. W.\nScore: 17.2, Published: 2023-10-04 DOI: 10.1101/2023.10.04.560777\nTo address the limitations of whole-spike COVID vaccines, we explored mRNA vaccines encoding membrane-anchored receptor-binding domain (RBD-TMs), each a fusion of a variant RBD, the transmembrane (TM) and cytoplasmic tail (CT) fragments of the SARS-CoV-2 spike protein. In naive mice, RBD-TM mRNA vaccines against ancestral SARS-CoV-2, Beta, Delta, Delta-plus, Kappa, Omicron BA.1 or BA.5, all induced strong humoral responses against the target RBD. Multiplex surrogate viral neutralization (sVNT) assays indicated broad neutralizing activity against a range of variant RBDs. In the setting of a heterologous boost, against the background of exposure to ancestral whole spike vaccines, sVNT studies suggested that RBD-TM vaccines were able to overcome the detrimental effects of immune imprinting. Omicron BA.1 and BA.5 RBD-TM booster vaccines induced serum antibodies with 12 and 22-fold higher neutralizing activity against the target RBD than their equivalent whole spike variants. Boosting with BA.1 or BA.5 RBD-TM provided good protection against more recent variants including XBB and XBB.1.5. Each RBD-TM mRNA is 28% of the length of its whole-spike equivalent. This advantage will enable tetravalent mRNA vaccines to be developed at well-tolerated doses of formulated mRNA.\nComparative single-cell analysis reveals IFN-γ as a driver of respiratory sequelae post COVID-19\nAuthors: Sun, J.; Li, C.; Qian, W.; Wei, X.\nScore: 14.5, Published: 2023-10-04 DOI: 10.1101/2023.10.03.560739\nPost-acute sequelae of SARS-CoV-2 infection (PASC) represents an urgent public health challenge, with its impact resonating in over 60 million individuals globally. While a growing body of evidence suggests that dysregulated immune reactions may be linked with PASC symptoms, most investigations have primarily centered around blood studies, with few focusing on samples derived from post-COVID affected tissues. Further, clinical studies alone often provide correlative insights rather than causal relationships. Thus, it is essential to compare clinical samples with relevant animal models and conduct functional experiments to truly understand the etiology of PASC. In this study, we have made comprehensive comparisons between bronchoalveolar lavage fluid (BAL) single-cell RNA sequencing (scRNAseq) data derived from clinical PASC samples and relevant PASC mouse models. This revealed a strong pro-fibrotic monocyte-derived macrophage response in respiratory PASC (R-PASC) in both humans and mice, and abnormal interactions between pulmonary macrophages and respiratory resident T cells. IFN-g emerged as a key node mediating the immune anomalies in R-PASC. Strikingly, neutralizing IFN-g post the resolution of acute infection reduced lung inflammation, tissue fibrosis, and improved pulmonary gas-exchange function in two mouse models of R-PASC. Our study underscores the importance of performing comparative analysis to understand the root cause of PASC for developing effective therapies.\nConcurrent administration of COVID-19 and influenza vaccines enhances Spike-specific antibody responses\nAuthors: Barouch, S. E.; Chicz, T. M.; Blanc, R.; Barbati, D. R.; Parker, L. J.; Tong, X.; McNamara, R. P.\nScore: 232.3, Published: 2023-09-12 DOI: 10.1101/2023.09.12.557347\nThe bivalent COVID-19 mRNA boosters became available in fall 2022 and were recommended alongside the seasonal influenza vaccine. However, the immunogenicity of concurrent versus separate administration of these vaccines remains unclear. Here, we analyzed antibody responses in healthcare workers who received the bivalent COVID-19 booster and the influenza vaccine on the same day or different days. IgG1 responses to SARS-CoV-2 Spike were higher at peak immunogenicity and 6 months following concurrent administration compared with separate administration of the COVID-19 and influenza vaccines. These data suggest that concurrent administration of these vaccines may yield higher and more durable SARS-CoV-2 antibody responses.\nMultimodal single-cell profiling of T cell specificity and reactivity in lung cancer\nAuthors: Bieberich, F.; Vazquez-Lombardi, R.; Jin, H.; Hong, K.-L.; Herzig, P.; Trefny, M.; Trueb, M.; Laubli, H.; Lardinois, D.; Mertz, K.; Matter, M.; Zippelius, A.; Reddy, S. T.\nScore: 10.9, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560863\nAdoptive transfer of autologous tumor-infiltrating lymphocyte T cells (TILs) offers one of the most promising approaches for cancer immunotherapy. However, high variability in patient responses highlight the need for an enhanced understanding of the transcriptional phenotypes of TILs and reactivity of their T cell receptors (TCR). Here, we employ single-cell multiomics approaches and TCR functional screening to investigate TILs from treatment-naive non-small cell lung cancer patients. This comprehensive analysis integrates scRNA-seq, scTCR-seq, and scATAC-seq, enabling a high-resolution examination of TILs within lung cancer tissue, as well as the adjacent non-tumor tissue. We apply a cellular functional screening platform to identify reactive TCRs that represent \u003e1,000 TILs and have specificity towards a multitude of targets, including primary tumor cells, neoantigens, tumor-associated antigens, and viral antigens. Tumor-reactive TILs were primarily associated with dysfunctional phenotypes, whereas viral antigen-reactive TCRs were found in effector phenotype clusters. Key marker genes were identified and used to construct a tumor or viral reactivity score. Comparing clones shared in tumor and non-tumor tissue, a higher fraction of exhausted cells was observed in the tumor tissue, whereas non-tumor adjacent tissue possessed more effector cells, thus providing insight into potential sources for therapeutic T cells. Elucidating the specific T cell populations within TILs and their associated TCRs may support strategies to enhance the efficacy of TIL-based therapies.\nInferring B cell phylogenies from paired heavy and light chain BCR sequences with Dowser\nAuthors: Jensen, C. G.; Sumner, J. A.; Kleinstein, S. H.; Hoehn, K. B.\nScore: 10.2, Published: 2023-10-02 DOI: 10.1101/2023.09.29.560187\nAntibodies are vital to human immune responses and are composed of genetically variable heavy and light chains. These structures are initially expressed as B cell receptors (BCRs). BCR diversity is shaped through somatic hypermutation and selection during immune responses. This evolutionary process produces B cell clones, cells that descend from a common ancestor but differ by mutations. Phylogenetic trees inferred from BCR sequences can reconstruct the history of mutations within a clone. Until recently, BCR sequencing technologies separated heavy and light chains, but advancements in single cell sequencing now pair heavy and light chains from individual cells. However, it is unclear how these separate genes should be combined to infer B cell phylogenies. In this study, we investigated strategies for using paired heavy and light chain sequences to build phylogenetic trees. We found incorporating light chains significantly improved tree accuracy and reproducibility across all methods tested. This improvement was greater than the difference between tree building methods and persisted even when mixing bulk and single cell sequencing data. However, we also found that many phylogenetic methods estimated significantly biased branch lengths when some light chains were missing, such as when mixing single cell and bulk BCR data. This bias was eliminated using maximum likelihood methods with separate branch lengths for heavy and light chain gene partitions. Thus, we recommend using maximum likelihood methods with separate heavy and light chain partitions, especially when mixing data types. We implemented these methods in the R package Dowser: https://dowser.readthedocs.io.\n",
  "wordCount" : "2272",
  "inLanguage": "en",
  "datePublished": "2023-10-08T10:37:16Z",
  "dateModified": "2023-10-08T10:37:16Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/immunology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      immunology
    </h1>
    <div class="post-meta"><span>updated on October 8, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561025">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561025" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561025">
        <p class="paperTitle">HERV activation segregates ME/CFS from fibromyalgia and defines a novel nosological entity for patients fulfilling both clinical criteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561025" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561025" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gimenez-Orenga, K.; Martin-Martinez, E.; Nathanson, L.; Oltra, E.</p>
        <p class="info">Score: 107.5, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561025' target='https://doi.org/10.1101/2023.10.05.561025'> 10.1101/2023.10.05.561025</a></p>
        <p class="abstract">Myalgic Encephalomyelitis/Chronic Fatigue Syndrome (ME/CFS) and fibromyalgia (FM) are chronic diseases with poorly understood pathophysiology and diagnosis based on clinical assessment of unspecific symptoms. The recent post-COVID-19 condition, which shares similarities with ME/CFS and FM, has raised concerns about viral-induced transcriptome changes in post-viral syndromes. Viral infections, and other types of stress, are known to unleash human endogenous retroviruses (HERV) repression that if maintained could lead to symptom chronicity. This study evaluated this possibility for ME/CFS and FM on a selected cohort of female patients complying with diagnosis criteria for ME/CFS, FM, or both, and matched healthy controls (n=43). The results show specific HERV fingerprints for each disease, confirming biological differences between ME/CFS and FM. Unexpectedly, HERV profiles segregated patients that met both ME/CFS and FM clinical criteria from patients complying only with ME or FM criteria, while clearly differentiating patients from healthy subjects, supporting that the highly prevalent comorbidity condition must constitute a different nosological entity. Moreover, HERV profiles exposed significant quantitative differences within the ME/CFS group that correlated with differences in immune gene expression and patient symptomatology, supporting ME/CFS patient subtyping and confirming immunological disturbances in this disease. Pending issues include validation of HERV profiles as disease biomarkers of post-viral syndromes and understanding the role of HERV during infection and beyond.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.560365">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.560365" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.560365">
        <p class="paperTitle">Neutralisation of SARS-CoV-2 Omicron subvariants BA.2.86 and EG.5.1 by antibodies induced by earlier infection or vaccination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.560365" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.560365" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lassauniere, R.; Polacek, C.; Baig, S.; Ellegaard, K.; Escobar-Herrera, L. A.; Fomsgaard, A.; Spiess, K.; Schwartz, O.; Planas, D.; Simon-Loriere, E.; Schneider, U. V.; Sieber, R. N.; Stegger, M.; Krause, T. G.; Ullum, H.; Jokelainen, P.; Rasmussen, M.</p>
        <p class="info">Score: 91.6, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.560365' target='https://doi.org/10.1101/2023.10.01.560365'> 10.1101/2023.10.01.560365</a></p>
        <p class="abstract">Highly mutated SARS-CoV-2 Omicron subvariant BA.2.86 emerged in July 2023. We investigated the neutralisation of isolated virus by antibodies induced by earlier infection or vaccination. The neutralisation titres for BA.2.86 were comparable to those for XBB.1 and EG.5.1, by antibodies induced by XBB.1.5 or BA.4/5 breakthrough infection or BA.4/5 vaccination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.559946">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.559946" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.559946">
        <p class="paperTitle">Quantifying Adaptive Evolution of the Human Immune Cell Landscape</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.559946" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.559946" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Salvador-Martinez, I.; Murga-Moreno, J.; Nieto, J. C.; Alsinet, C.; Enard, D.; Heyn, H.</p>
        <p class="info">Score: 28.4, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.559946' target='https://doi.org/10.1101/2023.10.06.559946'> 10.1101/2023.10.06.559946</a></p>
        <p class="abstract">The human immune system is under constant evolutionary pressure, primarily through its role as first line of defence against pathogens. Accordingly, population genomics studies have shown that immune-related genes have a high rate of adaptive evolution. These studies, however, are mainly based on protein-coding genes without cellular context, leaving the adaptive role of cell types and states uncharted. Inferring the rate of protein-coding genes adaptation in developing and adult immune cells at cellular resolution, we found cell types from both the lymphoid and myeloid compartments to harbour significantly increased adaptation rates. Specific cell states, such as foetal Pre-Pro B cells and adult T resident memory CD8&#43; cells show highly elevated rates of adaptation. We further analysed activated cell states, specifically, iPSC-derived macrophages responding to various challenges, including pro- and anti-inflammatory cytokines or bacterial and viral infections, the latter simulating the evolutionary arms race between humans and pathogens. Here, we found positive selection to be concentrated in early immune responses, suggesting benefits for the host to adapt to early stages of infection to control pathogen numbers and spread. Together, our study reveals spatio-temporal and functional biases in human immune populations with evidence of rapid adaptive evolution and provides a retrospect of forces that shaped the complexity, architecture, and function of the human body.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560810">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560810" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560810">
        <p class="paperTitle">Human intestinal organoids with an autologoustissue-resident immune compartment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560810" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560810" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Recaldin, T.; Gjeta, B.; Steinacher, L.; Harter, M. F.; Adam, L.; Nikolaev, M.; Krese, R.; Kilik, U.; Popovic, D.; Almato-Bellavista, M.; Kromer, K.; Bscheider, M.; Cabon, L.; Camp, J. G.; Gjorevski, N.</p>
        <p class="info">Score: 23.0, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560810' target='https://doi.org/10.1101/2023.10.04.560810'> 10.1101/2023.10.04.560810</a></p>
        <p class="abstract">The intimate relationship between the epithelium and the immune system is crucial for maintaining tissue homeostasis, with perturbations in epithelial-immune interactions linked to autoimmune disease and cancer. Whereas stem cell-derived organoids are powerful models of tissue-specific epithelial function, these structures lack tissue-resident immune cells that are essential for capturing organ-level processes. We describe human intestinal immuno-organoids (IIOs), formed through self-organization of epithelial organoids and autologous tissueresident lymphocytes (TRMs), a portion of which integrate within the IIO epithelium and survey the barrier. IIO formation was driven by TRM migration and interaction with epithelial cells, as orchestrated by TRM- enriched transcriptomic programs governing cell motility and epithelial inspection. We combined IIOs and single-cell transcriptomics to investigate intestinal inflammation triggered by cancer-targeting biologics in patients, and found that the system recapitulates clinical outcomes and the underlying cellular mechanisms. Inflammation was associated with the emergence of an activated population of CD8&#43; T cells, which progressively acquired intraepithelial and cytotoxic features. The appearance of this effector population was preceded and likely mediated by a Th1- like CD4&#43; population, which initially displayed a cytokine producing character and subsequently became cytotoxic itself. A system amenable to direct perturbation and interrogation, IIOs allowed us to identify the Rho pathway as a novel target for mitigating immunotherapy-associated intestinal inflammation. Given that they recapitulate both the phenotypic outcomes and the underlying inter-lineage immune interactions, IIOs can be used to broadly study tissue-resident immune responses in the context of tumorigenesis, infectious and autoimmune diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560084">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560084" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560084">
        <p class="paperTitle">Protective non-neutralizing mAbs targets conserved opsonic epitopes on SARS-CoV-2 variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560084" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560084" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Izadi, A.; Godzwon, M.; Ohlin, M.; Nordenfelt, P.</p>
        <p class="info">Score: 33.7, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560084' target='https://doi.org/10.1101/2023.09.29.560084'> 10.1101/2023.09.29.560084</a></p>
        <p class="abstract">Antibodies play a central role in the immune defense against SARS-CoV-2. Strong evidence has shown that non-neutralizing antibodies (nnAbs) are important for anti-SARS-Cov-2 immunity through Fc-mediated effector functions. These nnAbs bind to epitopes that could be less subjected to mutations in the emerging variants. When protective, such nnAbs would constitute a more promising alternative to neutralizing mAbs (nAbs). Here, we show that six nnAbs retain binding to Omicron, while two nAbs do not. Furthermore, two of our nnAbs, which are protective in vivo, retained binding to XBB, XBB.1.5, and BQ.1.1. They appear to bind to conserved epitopes on the N-terminal and receptor binding domain (RBD), respectively. As a proof of concept, we show that these protective non-neutralizing antibodies retain potent Fc-mediated opsonic function against BQ.1.1 and XBB. We also show that the Fc-mediated function is further enhanced by expressing the antibodies in the IgG3 subclass and combining them into a dual antibody cocktail. Our work suggests that opsonizing nnAbs could be a viable strategy for anti-SARS-CoV-2 mAb therapies against current and future SARS-CoV-2 variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560777">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560777" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560777">
        <p class="paperTitle">mRNA vaccines encoding membrane-anchored receptor-binding domains of SARS-CoV-2 mutants induce strong humoral responses and can overcome immune imprinting</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560777" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560777" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Al-Wassiti, H. A.; Fabb, S. A.; Grimley, S. L.; Kochappan, R.; Ho, J. K.; Wong, C. Y.; Tan, C. W.; Payne, T. J.; Takanashi, A.; Sicilia, H.; Teo, S. L. Y.; McAuley, J. L.; Ellenberg, P.; Cooney, J. P.; Davidson, K. C.; Bowen, R.; Pellegrini, M.; Rockman, S.; Godfrey, D. I.; Nolan, T. M.; Wang, L.-F.; Deliyannis, G.; Purcell, D. F. J.; Pouton, C. W.</p>
        <p class="info">Score: 17.2, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560777' target='https://doi.org/10.1101/2023.10.04.560777'> 10.1101/2023.10.04.560777</a></p>
        <p class="abstract">To address the limitations of whole-spike COVID vaccines, we explored mRNA vaccines encoding membrane-anchored receptor-binding domain (RBD-TMs), each a fusion of a variant RBD, the transmembrane (TM) and cytoplasmic tail (CT) fragments of the SARS-CoV-2 spike protein. In naive mice, RBD-TM mRNA vaccines against ancestral SARS-CoV-2, Beta, Delta, Delta-plus, Kappa, Omicron BA.1 or BA.5, all induced strong humoral responses against the target RBD. Multiplex surrogate viral neutralization (sVNT) assays indicated broad neutralizing activity against a range of variant RBDs. In the setting of a heterologous boost, against the background of exposure to ancestral whole spike vaccines, sVNT studies suggested that RBD-TM vaccines were able to overcome the detrimental effects of immune imprinting. Omicron BA.1 and BA.5 RBD-TM booster vaccines induced serum antibodies with 12 and 22-fold higher neutralizing activity against the target RBD than their equivalent whole spike variants. Boosting with BA.1 or BA.5 RBD-TM provided good protection against more recent variants including XBB and XBB.1.5. Each RBD-TM mRNA is 28% of the length of its whole-spike equivalent. This advantage will enable tetravalent mRNA vaccines to be developed at well-tolerated doses of formulated mRNA.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560739">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560739" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560739">
        <p class="paperTitle">Comparative single-cell analysis reveals IFN-γ as a driver of respiratory sequelae post COVID-19</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560739" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560739" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sun, J.; Li, C.; Qian, W.; Wei, X.</p>
        <p class="info">Score: 14.5, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560739' target='https://doi.org/10.1101/2023.10.03.560739'> 10.1101/2023.10.03.560739</a></p>
        <p class="abstract">Post-acute sequelae of SARS-CoV-2 infection (PASC) represents an urgent public health challenge, with its impact resonating in over 60 million individuals globally. While a growing body of evidence suggests that dysregulated immune reactions may be linked with PASC symptoms, most investigations have primarily centered around blood studies, with few focusing on samples derived from post-COVID affected tissues. Further, clinical studies alone often provide correlative insights rather than causal relationships. Thus, it is essential to compare clinical samples with relevant animal models and conduct functional experiments to truly understand the etiology of PASC. In this study, we have made comprehensive comparisons between bronchoalveolar lavage fluid (BAL) single-cell RNA sequencing (scRNAseq) data derived from clinical PASC samples and relevant PASC mouse models. This revealed a strong pro-fibrotic monocyte-derived macrophage response in respiratory PASC (R-PASC) in both humans and mice, and abnormal interactions between pulmonary macrophages and respiratory resident T cells. IFN-g emerged as a key node mediating the immune anomalies in R-PASC. Strikingly, neutralizing IFN-g post the resolution of acute infection reduced lung inflammation, tissue fibrosis, and improved pulmonary gas-exchange function in two mouse models of R-PASC. Our study underscores the importance of performing comparative analysis to understand the root cause of PASC for developing effective therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557347">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557347" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557347">
        <p class="paperTitle">Concurrent administration of COVID-19 and influenza vaccines enhances Spike-specific antibody responses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557347" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557347" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Barouch, S. E.; Chicz, T. M.; Blanc, R.; Barbati, D. R.; Parker, L. J.; Tong, X.; McNamara, R. P.</p>
        <p class="info">Score: 232.3, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557347' target='https://doi.org/10.1101/2023.09.12.557347'> 10.1101/2023.09.12.557347</a></p>
        <p class="abstract">The bivalent COVID-19 mRNA boosters became available in fall 2022 and were recommended alongside the seasonal influenza vaccine. However, the immunogenicity of concurrent versus separate administration of these vaccines remains unclear. Here, we analyzed antibody responses in healthcare workers who received the bivalent COVID-19 booster and the influenza vaccine on the same day or different days. IgG1 responses to SARS-CoV-2 Spike were higher at peak immunogenicity and 6 months following concurrent administration compared with separate administration of the COVID-19 and influenza vaccines. These data suggest that concurrent administration of these vaccines may yield higher and more durable SARS-CoV-2 antibody responses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560863">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560863" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560863">
        <p class="paperTitle">Multimodal single-cell profiling of T cell specificity and reactivity in lung cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560863" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560863" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bieberich, F.; Vazquez-Lombardi, R.; Jin, H.; Hong, K.-L.; Herzig, P.; Trefny, M.; Trueb, M.; Laubli, H.; Lardinois, D.; Mertz, K.; Matter, M.; Zippelius, A.; Reddy, S. T.</p>
        <p class="info">Score: 10.9, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560863' target='https://doi.org/10.1101/2023.10.04.560863'> 10.1101/2023.10.04.560863</a></p>
        <p class="abstract">Adoptive transfer of autologous tumor-infiltrating lymphocyte T cells (TILs) offers one of the most promising approaches for cancer immunotherapy. However, high variability in patient responses highlight the need for an enhanced understanding of the transcriptional phenotypes of TILs and reactivity of their T cell receptors (TCR). Here, we employ single-cell multiomics approaches and TCR functional screening to investigate TILs from treatment-naive non-small cell lung cancer patients. This comprehensive analysis integrates scRNA-seq, scTCR-seq, and scATAC-seq, enabling a high-resolution examination of TILs within lung cancer tissue, as well as the adjacent non-tumor tissue. We apply a cellular functional screening platform to identify reactive TCRs that represent &gt;1,000 TILs and have specificity towards a multitude of targets, including primary tumor cells, neoantigens, tumor-associated antigens, and viral antigens. Tumor-reactive TILs were primarily associated with dysfunctional phenotypes, whereas viral antigen-reactive TCRs were found in effector phenotype clusters. Key marker genes were identified and used to construct a tumor or viral reactivity score. Comparing clones shared in tumor and non-tumor tissue, a higher fraction of exhausted cells was observed in the tumor tissue, whereas non-tumor adjacent tissue possessed more effector cells, thus providing insight into potential sources for therapeutic T cells. Elucidating the specific T cell populations within TILs and their associated TCRs may support strategies to enhance the efficacy of TIL-based therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560187">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560187" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560187">
        <p class="paperTitle">Inferring B cell phylogenies from paired heavy and light chain BCR sequences with Dowser</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560187" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560187" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jensen, C. G.; Sumner, J. A.; Kleinstein, S. H.; Hoehn, K. B.</p>
        <p class="info">Score: 10.2, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560187' target='https://doi.org/10.1101/2023.09.29.560187'> 10.1101/2023.09.29.560187</a></p>
        <p class="abstract">Antibodies are vital to human immune responses and are composed of genetically variable heavy and light chains. These structures are initially expressed as B cell receptors (BCRs). BCR diversity is shaped through somatic hypermutation and selection during immune responses. This evolutionary process produces B cell clones, cells that descend from a common ancestor but differ by mutations. Phylogenetic trees inferred from BCR sequences can reconstruct the history of mutations within a clone. Until recently, BCR sequencing technologies separated heavy and light chains, but advancements in single cell sequencing now pair heavy and light chains from individual cells. However, it is unclear how these separate genes should be combined to infer B cell phylogenies. In this study, we investigated strategies for using paired heavy and light chain sequences to build phylogenetic trees. We found incorporating light chains significantly improved tree accuracy and reproducibility across all methods tested. This improvement was greater than the difference between tree building methods and persisted even when mixing bulk and single cell sequencing data. However, we also found that many phylogenetic methods estimated significantly biased branch lengths when some light chains were missing, such as when mixing single cell and bulk BCR data. This bias was eliminated using maximum likelihood methods with separate branch lengths for heavy and light chain gene partitions. Thus, we recommend using maximum likelihood methods with separate heavy and light chain partitions, especially when mixing data types. We implemented these methods in the R package Dowser: https://dowser.readthedocs.io.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
