<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>immunology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>immunology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Heterotypic responses against nsp12/nsp13 from prior SARS-CoV-2 infection associates with lower subsequent endemic coronavirus incidence
Authors: Bean, D. J.; Monroe, J.; Liang, Y. M.; Borberg, E.; Senussi, Y.; Swank, Z.; Chalise, S.; Walt, D.; Weinberg, J.; Sagar, M.
Score: 143.7, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563621
Immune responses from prior SARS-CoV-2 infection and COVID-19 vaccination do not prevent re-infections and may not protect against future novel coronaviruses (CoVs). We examined the incidence of and immune differences against human endemic CoVs (eCoV) as a proxy for response against future emerging CoVs.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/immunology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="immunology" />
<meta property="og:description" content="Heterotypic responses against nsp12/nsp13 from prior SARS-CoV-2 infection associates with lower subsequent endemic coronavirus incidence
Authors: Bean, D. J.; Monroe, J.; Liang, Y. M.; Borberg, E.; Senussi, Y.; Swank, Z.; Chalise, S.; Walt, D.; Weinberg, J.; Sagar, M.
Score: 143.7, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563621
Immune responses from prior SARS-CoV-2 infection and COVID-19 vaccination do not prevent re-infections and may not protect against future novel coronaviruses (CoVs). We examined the incidence of and immune differences against human endemic CoVs (eCoV) as a proxy for response against future emerging CoVs." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/immunology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-29T10:37:40+00:00" />
<meta property="article:modified_time" content="2023-10-29T10:37:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="immunology"/>
<meta name="twitter:description" content="Heterotypic responses against nsp12/nsp13 from prior SARS-CoV-2 infection associates with lower subsequent endemic coronavirus incidence
Authors: Bean, D. J.; Monroe, J.; Liang, Y. M.; Borberg, E.; Senussi, Y.; Swank, Z.; Chalise, S.; Walt, D.; Weinberg, J.; Sagar, M.
Score: 143.7, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563621
Immune responses from prior SARS-CoV-2 infection and COVID-19 vaccination do not prevent re-infections and may not protect against future novel coronaviruses (CoVs). We examined the incidence of and immune differences against human endemic CoVs (eCoV) as a proxy for response against future emerging CoVs."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "immunology",
      "item": "https://trxiv.yorks0n.com/posts/immunology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "immunology",
  "name": "immunology",
  "description": "Heterotypic responses against nsp12/nsp13 from prior SARS-CoV-2 infection associates with lower subsequent endemic coronavirus incidence\nAuthors: Bean, D. J.; Monroe, J.; Liang, Y. M.; Borberg, E.; Senussi, Y.; Swank, Z.; Chalise, S.; Walt, D.; Weinberg, J.; Sagar, M.\nScore: 143.7, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563621\nImmune responses from prior SARS-CoV-2 infection and COVID-19 vaccination do not prevent re-infections and may not protect against future novel coronaviruses (CoVs). We examined the incidence of and immune differences against human endemic CoVs (eCoV) as a proxy for response against future emerging CoVs.",
  "keywords": [
    
  ],
  "articleBody": " Heterotypic responses against nsp12/nsp13 from prior SARS-CoV-2 infection associates with lower subsequent endemic coronavirus incidence\nAuthors: Bean, D. J.; Monroe, J.; Liang, Y. M.; Borberg, E.; Senussi, Y.; Swank, Z.; Chalise, S.; Walt, D.; Weinberg, J.; Sagar, M.\nScore: 143.7, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563621\nImmune responses from prior SARS-CoV-2 infection and COVID-19 vaccination do not prevent re-infections and may not protect against future novel coronaviruses (CoVs). We examined the incidence of and immune differences against human endemic CoVs (eCoV) as a proxy for response against future emerging CoVs. Assessment was among those with known SARS-CoV-2 infection, COVID-19 vaccination but no documented SARS-CoV-2 infection, or neither exposure. Retrospective cohort analyses suggest that prior SARS-CoV-2 infection, but not COVID-19 vaccination alone, protects against subsequent symptomatic eCoV infection. CD8+ T cell responses to the non-structural eCoV proteins, nsp12 and nsp13, were significantly higher in individuals with previous SARS-CoV-2 infection as compared to the other groups. The three groups had similar cellular responses against the eCoV spike and nucleocapsid, and those with prior spike exposure had lower eCoV-directed neutralizing antibodies. Incorporation of non-structural viral antigens in a future pan-CoV vaccine may improve protection against future heterologous CoV infections.\nSARS-CoV-2 antibodies cross-react and enhance dengue infection\nAuthors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.\nScore: 255.7, Published: 2023-10-10 DOI: 10.1101/2023.10.09.557914\nDengue disease is highly prevalent in tropical and subtropical regions worldwide. However, its pathogenesis is still incompletely understood, particularly in comparison to other endemic viruses. Antibody-dependent enhancement (ADE) is a well-known phenomenon for dengue viruses. Given the recent surge in dengue cases and potential cross-reactivity with SARS-CoV-2 antibodies, this study explores the impact of anti-SARS-CoV-2 antibodies on DENV-2 infection. The study assessed the cross-reactivity of SARS-CoV-2 antibodies with the DENV-2 Virus. Human convalescent plasma samples collected during different waves of COVID-19 and monoclonal and polyclonal antibodies raised against SARS-CoV-2 were examined for their potential to cause ADE of DENV-2 infection using cell-based assays. The study found that anti-SARS-CoV-2 antibodies acquired from natural infection in humans or through experimental immunization in animals were cross-reactive with DENV-2 and had the potential to enhance DENV-2 infection in K562 and U937 cells. In-silico and in-vitro studies indicated a strong interaction between SARS-CoV-2 antibodies and DENV-2 E-protein, providing a molecular basis for these findings. This study is the first to demonstrate that anti-SARS-CoV-2 antibodies can cross-react with DENV-2 and can enhance its infection through ADE. These findings have implications for SARS-CoV-2 vaccine development and deployment strategies in regions where dengue is endemic. SummaryAntibodies against SARS-CoV-2 (RBD and Spike) showed significant cross reactivity with DENV-2 (E protein). Also, anti-SARS-CoV-2-commercial antibodies, immunised animal sera and 46 human convalescent plasma samples (from different waves of pandemic) demonstrated antibody-dependent enhancement (ADE) of DENV-2 infection.\nReduced Monocyte Proportions and Responsiveness in Convalescent COVID-19 Patients\nAuthors: Ravkov, E.; Williams, E. S. C. P.; Elgort, M.; Spivak, A.; Barker, A. P.; Planelles, V.; Delgado, J.; Lin, L.; Hanley, T.\nScore: 11.1, Published: 2023-10-26 DOI: 10.1101/2023.10.25.563806\nThe clinical manifestations of acute severe acute respiratory syndrome coronavirus type 2 (SARS-CoV-2) infection and COVID-19 suggest a dysregulation of the host immune response that leads to inflammation, thrombosis, and organ dysfunction. It is less clear whether these dysregulated processes persist during the convalescent phase of disease or during long COVID. We investigated the effects of SARS-CoV-2 infection on the proportions of classical, intermediate, and non-classical monocytes, their activation status, and their functional properties in convalescent COVID-19 patients and uninfected control subjects. We found that the percentage of total monocytes was decreased in convalescent COVID-19 patients compared to uninfected controls. This was due to decreased intermediate and non-classical monocytes. Classical monocytes from convalescent COVID-19 patients demonstrated a decrease in activation markers, such as CD56, in response to stimulation with bacterial lipopolysaccharide (LPS). In addition, classical monocytes from convalescent COVID-19 patients showed decreased expression of CD142 (tissue factor), which can initiate the extrinsic coagulation cascade, in response to LPS stimulation. Finally, we found that monocytes from convalescent COVID-19 patients produced less TNF- and IL-6 in response to LPS stimulation, than those from uninfected controls. In conclusion, SARS-CoV-2 infection exhibits a clear effect on the relative proportions of monocyte subsets, the activation status of classical monocytes, and proinflammatory cytokine production that persists during the convalescent phase of disease.\nResolving haplotype variation and complex genetic architecture in the human immunoglobulin kappa chain locus in individuals of diverse ancestry\nAuthors: Engelbrecht, E.; Rodriguez, O. L.; Shields, K.; Schulze, S.; Tieri, D.; Jana, U.; Yaari, G.; Lees, W.; Smith, M. L.; Watson, C. T.\nScore: 8.9, Published: 2023-10-25 DOI: 10.1101/2023.10.23.563321\nImmunoglobulins (IGs), critical components of the human immune system, are composed of heavy and light protein chains encoded at three genomic loci. The IG Kappa (IGK) chain locus consists of two large, inverted segmental duplications. The complexity of IG loci has hindered effective use of standard high-throughput methods for characterizing genetic variation within these regions. To overcome these limitations, we leverage long-read sequencing to create haplotype-resolved IGK assemblies in an ancestrally diverse cohort (n=36), representing the first comprehensive description of IGK haplotype variation at population-scale. We identify extensive locus polymorphism, including novel single nucleotide variants (SNVs) and a common novel ~24.7 Kbp structural variant harboring a functional IGKV gene. Among 47 functional IGKV genes, we identify 141 alleles, 64 (45.4%) of which were not previously curated. We report inter-population differences in allele frequencies for 14 of the IGKV genes, including alleles unique to specific populations within this dataset. Finally, we identify haplotypes carrying signatures of gene conversion that associate with enrichment of SNVs in the IGK distal region. These data provide a critical resource of curated genomic reference information from diverse ancestries, laying a foundation for advancing our understanding of population-level genetic variation in the IGK locus.\nEarly antiviral CD4 and CD8 T cell responses are associated with upper respiratory tract clearance of SARS-CoV-2\nAuthors: Ramirez, S. I.; Lopez, P. G.; Faraji, F.; Parikh, U. M.; Heaps, A.; Ritz, J.; Moser, C.; Eron, J. J.; Wohl, D. A.; Currier, J. S.; Daar, E. S.; Greninger, A. L.; Klekotka, P.; Grifoni, A.; Weiskopf, D.; Sette, A.; Peters, B.; Hughes, M. D.; Chew, K. W.; Smith, D. M.; Crotty, S.; ACTIV-2/A5401 Study Team, Score: 8.6, Published: 2023-10-26 DOI: 10.1101/2023.10.25.564014\nT cells are involved in protective immunity against numerous viral infections. Limited data have been available regarding roles of human T cell responses controlling SARS-CoV-2 viral clearance in primary COVID-19. Here, we examined longitudinal SARS-CoV-2 upper respiratory tract viral RNA levels and early adaptive immune responses from 95 unvaccinated individuals with acute COVID-19. Acute SARS-CoV-2-specific CD4 and CD8 T cell responses were evaluated in addition to antibody responses. Most individuals with acute COVID-19 developed rapid SARS-CoV-2-specific T cell responses during infection, and both early CD4 T cell and CD8 T cell responses correlated with reduced upper respiratory tract SARS-CoV-2 viral RNA, independent of neutralizing antibody titers. Overall, our findings indicate a distinct protective role for SARS-CoV-2-specific T cells during acute COVID-19.\nGeneration of circulating autoreactive pre-plasma cells fueled by naive B cells in celiac disease\nAuthors: Lindeman, I.; Hoydahl, L. S.; Christophersen, A.; Risnes, L. F.; Jahnsen, J.; Lundin, K. E. A.; Sollid, L. M.; Iversen, R.\nScore: 8.5, Published: 2023-10-23 DOI: 10.1101/2023.10.20.563227\nAutoantibodies against the enzyme transglutaminase 2 (TG2) are characteristic of celiac disease (CeD), and TG2-specific IgA plasma cells are abundant in gut biopsies of patients. We here describe the corresponding population of autoreactive B cells in blood. Circulating TG2-specific IgA cells were found in untreated patients on a gluten-containing diet but not in controls. They were clonally related to TG2-specific small intestinal plasma cells, and they expressed gut-homing molecules, indicating that they are plasma cell precursors. Unlike other IgA-switched cells, the TG2-specific cells were negative for CD27, placing them in the double negative (IgD-CD27-) category. They had a plasmablast or activated memory B-cell phenotype, and they harbored fewer variable region mutations than other IgA cells. Based on their similarity to naive B cells, we propose that autoreactive IgA cells in CeD are generated mainly through chronic recruitment of naive B cells via an extrafollicular response involving gluten-specific CD4+ T cells.\nImmune Epitopes of SARS-CoV-2 Spike Protein and Considerations for Universal Vaccine Development\nAuthors: Magazine, N.; Zhang, T.; Bungwon, A. D.; McGee, M. C.; Wu, Y.; Veggiani, G.; Huang, W.\nScore: 8.4, Published: 2023-10-27 DOI: 10.1101/2023.10.26.564184\nDespite the success of global vaccination programs in slowing the spread of COVID-19, these efforts have been hindered by the emergence of new SARS-CoV-2 strains capable of evading prior immunity. The mutation and evolution of SARS-CoV-2 have created a demand for persistent efforts in vaccine development. SARS-CoV-2 Spike protein has been the primary target for COVID-19 vaccine development, but it is also the hotspot of mutations directly involved in host susceptibility and immune evasion. Our ability to predict emerging mutants and select conserved epitopes is critical for the development of a broadly neutralizing therapy or a universal vaccine. In this article, we review the general paradigm of immune responses to COVID-19 vaccines, highlighting the immunological epitopes of Spike protein that are likely associated with eliciting protective immunity resulting from vaccination. Specifically, we analyze the structural and evolutionary characteristics of the SARS-CoV-2 Spike protein related to immune activation and function via the toll-like receptors (TLRs), B cells, and T cells. We aim to provide a comprehensive analysis of immune epitopes of Spike protein, thereby contributing to the development of new strategies for broad neutralization or universal vaccination.\nmRNA vaccines encoding membrane-anchored receptor-binding domains of SARS-CoV-2 mutants induce strong humoral responses and can overcome immune imprinting\nAuthors: Al-Wassiti, H. A.; Fabb, S. A.; Grimley, S. L.; Kochappan, R.; Ho, J. K.; Wong, C. Y.; Tan, C. W.; Payne, T. J.; Takanashi, A.; Sicilia, H.; Teo, S. L. Y.; McAuley, J. L.; Ellenberg, P.; Cooney, J. P.; Davidson, K. C.; Bowen, R.; Pellegrini, M.; Rockman, S.; Godfrey, D. I.; Nolan, T. M.; Wang, L.-F.; Deliyannis, G.; Purcell, D. F. J.; Pouton, C. W.\nScore: 27.4, Published: 2023-10-04 DOI: 10.1101/2023.10.04.560777\nTo address the limitations of whole-spike COVID vaccines, we explored mRNA vaccines encoding membrane-anchored receptor-binding domain (RBD-TMs), each a fusion of a variant RBD, the transmembrane (TM) and cytoplasmic tail (CT) fragments of the SARS-CoV-2 spike protein. In naive mice, RBD-TM mRNA vaccines against ancestral SARS-CoV-2, Beta, Delta, Delta-plus, Kappa, Omicron BA.1 or BA.5, all induced strong humoral responses against the target RBD. Multiplex surrogate viral neutralization (sVNT) assays indicated broad neutralizing activity against a range of variant RBDs. In the setting of a heterologous boost, against the background of exposure to ancestral whole spike vaccines, sVNT studies suggested that RBD-TM vaccines were able to overcome the detrimental effects of immune imprinting. Omicron BA.1 and BA.5 RBD-TM booster vaccines induced serum antibodies with 12 and 22-fold higher neutralizing activity against the target RBD than their equivalent whole spike variants. Boosting with BA.1 or BA.5 RBD-TM provided good protection against more recent variants including XBB and XBB.1.5. Each RBD-TM mRNA is 28% of the length of its whole-spike equivalent. This advantage will enable tetravalent mRNA vaccines to be developed at well-tolerated doses of formulated mRNA. One Sentence SummarymRNA vaccines encoding membrane-anchored RBDs of SARS-CoV-2 mutants are effective vaccines that can overcome immune imprinting in mice\nVagus nerve stimulation limits the germinal center B cell response via CD4+ T cell-derived acetylcholine\nAuthors: Kurata-Sato, I.; Mughrabi, I. T.; Rana, M.; Gerber, M.; Al-Abed, Y.; Sherry, B.; Zanos, S.; Diamond, B.\nScore: 5.0, Published: 2023-10-27 DOI: 10.1101/2023.10.25.563818\nNeural signals are known to contribute to immune regulation and modulation of the innate immune response downstream of the vagus nerve has been well studied. The effects of vagus nerve activity on antibody production, however, have been largely unexplored. Here we use a chronic vagus nerve stimulation (VNS) mouse model to study the effect of vagal activation on T-dependent B cell responses. We observed lower titers of high-affinity serum IgG and fewer antigen-specific germinal center (GC) B cells in the spleen. GC B cells from chronic VNS mice expressed more active caspase-3 and exhibited an altered gene expression profile suggesting increased susceptibility to apoptosis and impaired maturation. Follicular dendritic cell (FDC) cluster dispersal and altered FDC gene expression suggested poor FDC function. These alterations were diminished in the absence of a subset of acetylcholine-producing CD4+ T cells. In vitro studies revealed that 7 and 9 nicotinic acetylcholine receptors (nAChRs) directly regulated B cell production of TNF, a cytokine crucial to FDC clustering. Engagement of the 4 nAChR subunit on B cells impaired Akt phosphorylation, presumably decreasing B cell survival. Thus, VNS-induced GC impairment can be attributed, at least in part, to the effect of acetylcholine on B cell intrinsic pathways, resulting in hindered B cell survival and maturation and leading to an alteration in FDC function. Our findings identify a potential therapeutic target to prevent immunosuppression in conditions associated with increased vagal activity.\nScarring hair follicle destruction is driven by the collapse of EGFR-protected JAK-STAT1-sensitive stem cell immune privilege\nAuthors: Strobl, K.; Klufa, J.; Jin, R.; Artner-Gent, L.; Krauss, D.; Novoszel, P.; Strobl, J.; Stary, G.; Vujic, I.; Griss, J.; Holcmann, M.; Farlik, M.; Homey, B.; Sibilia, M.; Bauer, T.\nScore: 4.6, Published: 2023-10-14 DOI: 10.1101/2023.10.11.561653\nThe hair follicle stem cell niche is an immune-privileged microenvironment, characterised by suppressed antigen presentation, thus shielding against permanent immune-mediated tissue damage. In this study, we demonstrate the protective role of hair follicle-specific epidermal growth factor receptor (EGFR) from scarring hair follicle degeneration. Mechanistically, disruption of EGFR signalling generates a cell intrinsic hypersensitivity within the JAK-STAT1 pathway, compromising the immune privilege in the context of CD8 T cell and NK cell-mediated inflammation. Genetic depletion of either JAK1/2 or STAT1 or topical therapeutic inhibition of JAK1/2 restores the immune privilege and activates stem cells to resume hair growth in mouse models of epidermal and hair follicle specific EGFR deletion. Skin biopsies from EGFR inhibitor-treated and from EGFR-independent cicatricial alopecia patients indicate active STAT1 signalling within the hair follicles. Notably, a case study of folliculitis decalvans, characterised by progressive hair loss, scaling and perifollicular erythema, demonstrates successful treatment with systemic JAK1/2 inhibition. Our findings offer mechanistic insights and present a therapeutic strategy for addressing scarring hair follicle destruction associated with EGFR-inhibitor therapy and cicatricial alopecia.\n",
  "wordCount" : "2400",
  "inLanguage": "en",
  "datePublished": "2023-10-29T10:37:40Z",
  "dateModified": "2023-10-29T10:37:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/immunology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      immunology
    </h1>
    <div class="post-meta"><span>updated on October 29, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563621">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563621" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563621">
        <p class="paperTitle">Heterotypic responses against nsp12/nsp13 from prior SARS-CoV-2 infection associates with lower subsequent endemic coronavirus incidence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563621" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563621" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bean, D. J.; Monroe, J.; Liang, Y. M.; Borberg, E.; Senussi, Y.; Swank, Z.; Chalise, S.; Walt, D.; Weinberg, J.; Sagar, M.</p>
        <p class="info">Score: 143.7, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563621' target='https://doi.org/10.1101/2023.10.23.563621'> 10.1101/2023.10.23.563621</a></p>
        <p class="abstract">Immune responses from prior SARS-CoV-2 infection and COVID-19 vaccination do not prevent re-infections and may not protect against future novel coronaviruses (CoVs). We examined the incidence of and immune differences against human endemic CoVs (eCoV) as a proxy for response against future emerging CoVs. Assessment was among those with known SARS-CoV-2 infection, COVID-19 vaccination but no documented SARS-CoV-2 infection, or neither exposure. Retrospective cohort analyses suggest that prior SARS-CoV-2 infection, but not COVID-19 vaccination alone, protects against subsequent symptomatic eCoV infection. CD8&#43; T cell responses to the non-structural eCoV proteins, nsp12 and nsp13, were significantly higher in individuals with previous SARS-CoV-2 infection as compared to the other groups. The three groups had similar cellular responses against the eCoV spike and nucleocapsid, and those with prior spike exposure had lower eCoV-directed neutralizing antibodies. Incorporation of non-structural viral antigens in a future pan-CoV vaccine may improve protection against future heterologous CoV infections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.557914">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.557914" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.557914">
        <p class="paperTitle">SARS-CoV-2 antibodies cross-react and enhance dengue infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.557914" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.557914" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jakhar, K.; Sonar, S.; Singh, G.; Sarkar, T.; Tiwari, M.; Kaur, J.; Rathore, D. K.; Lal, B.; Kumar, S.; Srivastav, P.; Kumar, S.; Phagna, V.; Kumar, L.; Gupta, V.; Kshetrapal, P.; Singh, S.; Wadhwa, N.; Thiruvengadam, R.; Raghavan, S.; Gosain, M.; Shrivastava, T.; Bhattacharyya, S.; Bhattacharya, J.; Asthana, S.; Mani, S.</p>
        <p class="info">Score: 255.7, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.557914' target='https://doi.org/10.1101/2023.10.09.557914'> 10.1101/2023.10.09.557914</a></p>
        <p class="abstract">Dengue disease is highly prevalent in tropical and subtropical regions worldwide. However, its pathogenesis is still incompletely understood, particularly in comparison to other endemic viruses. Antibody-dependent enhancement (ADE) is a well-known phenomenon for dengue viruses. Given the recent surge in dengue cases and potential cross-reactivity with SARS-CoV-2 antibodies, this study explores the impact of anti-SARS-CoV-2 antibodies on DENV-2 infection.

The study assessed the cross-reactivity of SARS-CoV-2 antibodies with the DENV-2 Virus. Human convalescent plasma samples collected during different waves of COVID-19 and monoclonal and polyclonal antibodies raised against SARS-CoV-2 were examined for their potential to cause ADE of DENV-2 infection using cell-based assays. The study found that anti-SARS-CoV-2 antibodies acquired from natural infection in humans or through experimental immunization in animals were cross-reactive with DENV-2 and had the potential to enhance DENV-2 infection in K562 and U937 cells. In-silico and in-vitro studies indicated a strong interaction between SARS-CoV-2 antibodies and DENV-2 E-protein, providing a molecular basis for these findings. This study is the first to demonstrate that anti-SARS-CoV-2 antibodies can cross-react with DENV-2 and can enhance its infection through ADE. These findings have implications for SARS-CoV-2 vaccine development and deployment strategies in regions where dengue is endemic.

SummaryAntibodies against SARS-CoV-2 (RBD and Spike) showed significant cross reactivity with DENV-2 (E protein). Also, anti-SARS-CoV-2-commercial antibodies, immunised animal sera and 46 human convalescent plasma samples (from different waves of pandemic) demonstrated antibody-dependent enhancement (ADE) of DENV-2 infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.563806">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.563806" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.563806">
        <p class="paperTitle">Reduced Monocyte Proportions and Responsiveness in Convalescent COVID-19 Patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.563806" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.563806" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ravkov, E.; Williams, E. S. C. P.; Elgort, M.; Spivak, A.; Barker, A. P.; Planelles, V.; Delgado, J.; Lin, L.; Hanley, T.</p>
        <p class="info">Score: 11.1, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.563806' target='https://doi.org/10.1101/2023.10.25.563806'> 10.1101/2023.10.25.563806</a></p>
        <p class="abstract">The clinical manifestations of acute severe acute respiratory syndrome coronavirus type 2 (SARS-CoV-2) infection and COVID-19 suggest a dysregulation of the host immune response that leads to inflammation, thrombosis, and organ dysfunction. It is less clear whether these dysregulated processes persist during the convalescent phase of disease or during long COVID. We investigated the effects of SARS-CoV-2 infection on the proportions of classical, intermediate, and non-classical monocytes, their activation status, and their functional properties in convalescent COVID-19 patients and uninfected control subjects. We found that the percentage of total monocytes was decreased in convalescent COVID-19 patients compared to uninfected controls. This was due to decreased intermediate and non-classical monocytes. Classical monocytes from convalescent COVID-19 patients demonstrated a decrease in activation markers, such as CD56, in response to stimulation with bacterial lipopolysaccharide (LPS). In addition, classical monocytes from convalescent COVID-19 patients showed decreased expression of CD142 (tissue factor), which can initiate the extrinsic coagulation cascade, in response to LPS stimulation. Finally, we found that monocytes from convalescent COVID-19 patients produced less TNF- and IL-6 in response to LPS stimulation, than those from uninfected controls. In conclusion, SARS-CoV-2 infection exhibits a clear effect on the relative proportions of monocyte subsets, the activation status of classical monocytes, and proinflammatory cytokine production that persists during the convalescent phase of disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563321">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563321" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563321">
        <p class="paperTitle">Resolving haplotype variation and complex genetic architecture in the human immunoglobulin kappa chain locus in individuals of diverse ancestry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563321" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563321" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Engelbrecht, E.; Rodriguez, O. L.; Shields, K.; Schulze, S.; Tieri, D.; Jana, U.; Yaari, G.; Lees, W.; Smith, M. L.; Watson, C. T.</p>
        <p class="info">Score: 8.9, Published: 2023-10-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563321' target='https://doi.org/10.1101/2023.10.23.563321'> 10.1101/2023.10.23.563321</a></p>
        <p class="abstract">Immunoglobulins (IGs), critical components of the human immune system, are composed of heavy and light protein chains encoded at three genomic loci. The IG Kappa (IGK) chain locus consists of two large, inverted segmental duplications. The complexity of IG loci has hindered effective use of standard high-throughput methods for characterizing genetic variation within these regions. To overcome these limitations, we leverage long-read sequencing to create haplotype-resolved IGK assemblies in an ancestrally diverse cohort (n=36), representing the first comprehensive description of IGK haplotype variation at population-scale. We identify extensive locus polymorphism, including novel single nucleotide variants (SNVs) and a common novel ~24.7 Kbp structural variant harboring a functional IGKV gene. Among 47 functional IGKV genes, we identify 141 alleles, 64 (45.4%) of which were not previously curated. We report inter-population differences in allele frequencies for 14 of the IGKV genes, including alleles unique to specific populations within this dataset. Finally, we identify haplotypes carrying signatures of gene conversion that associate with enrichment of SNVs in the IGK distal region. These data provide a critical resource of curated genomic reference information from diverse ancestries, laying a foundation for advancing our understanding of population-level genetic variation in the IGK locus.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.564014">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.564014" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.564014">
        <p class="paperTitle">Early antiviral CD4 and CD8 T cell responses are associated with upper respiratory tract clearance of SARS-CoV-2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.564014" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.564014" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ramirez, S. I.; Lopez, P. G.; Faraji, F.; Parikh, U. M.; Heaps, A.; Ritz, J.; Moser, C.; Eron, J. J.; Wohl, D. A.; Currier, J. S.; Daar, E. S.; Greninger, A. L.; Klekotka, P.; Grifoni, A.; Weiskopf, D.; Sette, A.; Peters, B.; Hughes, M. D.; Chew, K. W.; Smith, D. M.; Crotty, S.; ACTIV-2/A5401 Study Team,  </p>
        <p class="info">Score: 8.6, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.564014' target='https://doi.org/10.1101/2023.10.25.564014'> 10.1101/2023.10.25.564014</a></p>
        <p class="abstract">T cells are involved in protective immunity against numerous viral infections. Limited data have been available regarding roles of human T cell responses controlling SARS-CoV-2 viral clearance in primary COVID-19. Here, we examined longitudinal SARS-CoV-2 upper respiratory tract viral RNA levels and early adaptive immune responses from 95 unvaccinated individuals with acute COVID-19. Acute SARS-CoV-2-specific CD4 and CD8 T cell responses were evaluated in addition to antibody responses. Most individuals with acute COVID-19 developed rapid SARS-CoV-2-specific T cell responses during infection, and both early CD4 T cell and CD8 T cell responses correlated with reduced upper respiratory tract SARS-CoV-2 viral RNA, independent of neutralizing antibody titers. Overall, our findings indicate a distinct protective role for SARS-CoV-2-specific T cells during acute COVID-19.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563227">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563227" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563227">
        <p class="paperTitle">Generation of circulating autoreactive pre-plasma cells fueled by naive B cells in celiac disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563227" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563227" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lindeman, I.; Hoydahl, L. S.; Christophersen, A.; Risnes, L. F.; Jahnsen, J.; Lundin, K. E. A.; Sollid, L. M.; Iversen, R.</p>
        <p class="info">Score: 8.5, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563227' target='https://doi.org/10.1101/2023.10.20.563227'> 10.1101/2023.10.20.563227</a></p>
        <p class="abstract">Autoantibodies against the enzyme transglutaminase 2 (TG2) are characteristic of celiac disease (CeD), and TG2-specific IgA plasma cells are abundant in gut biopsies of patients. We here describe the corresponding population of autoreactive B cells in blood. Circulating TG2-specific IgA cells were found in untreated patients on a gluten-containing diet but not in controls. They were clonally related to TG2-specific small intestinal plasma cells, and they expressed gut-homing molecules, indicating that they are plasma cell precursors. Unlike other IgA-switched cells, the TG2-specific cells were negative for CD27, placing them in the double negative (IgD-CD27-) category. They had a plasmablast or activated memory B-cell phenotype, and they harbored fewer variable region mutations than other IgA cells. Based on their similarity to naive B cells, we propose that autoreactive IgA cells in CeD are generated mainly through chronic recruitment of naive B cells via an extrafollicular response involving gluten-specific CD4&#43; T cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564184">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564184" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564184">
        <p class="paperTitle">Immune Epitopes of SARS-CoV-2 Spike Protein and Considerations for Universal Vaccine Development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564184" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564184" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Magazine, N.; Zhang, T.; Bungwon, A. D.; McGee, M. C.; Wu, Y.; Veggiani, G.; Huang, W.</p>
        <p class="info">Score: 8.4, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564184' target='https://doi.org/10.1101/2023.10.26.564184'> 10.1101/2023.10.26.564184</a></p>
        <p class="abstract">Despite the success of global vaccination programs in slowing the spread of COVID-19, these efforts have been hindered by the emergence of new SARS-CoV-2 strains capable of evading prior immunity. The mutation and evolution of SARS-CoV-2 have created a demand for persistent efforts in vaccine development. SARS-CoV-2 Spike protein has been the primary target for COVID-19 vaccine development, but it is also the hotspot of mutations directly involved in host susceptibility and immune evasion. Our ability to predict emerging mutants and select conserved epitopes is critical for the development of a broadly neutralizing therapy or a universal vaccine. In this article, we review the general paradigm of immune responses to COVID-19 vaccines, highlighting the immunological epitopes of Spike protein that are likely associated with eliciting protective immunity resulting from vaccination. Specifically, we analyze the structural and evolutionary characteristics of the SARS-CoV-2 Spike protein related to immune activation and function via the toll-like receptors (TLRs), B cells, and T cells. We aim to provide a comprehensive analysis of immune epitopes of Spike protein, thereby contributing to the development of new strategies for broad neutralization or universal vaccination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560777">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560777" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560777">
        <p class="paperTitle">mRNA vaccines encoding membrane-anchored receptor-binding domains of SARS-CoV-2 mutants induce strong humoral responses and can overcome immune imprinting</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560777" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560777" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Al-Wassiti, H. A.; Fabb, S. A.; Grimley, S. L.; Kochappan, R.; Ho, J. K.; Wong, C. Y.; Tan, C. W.; Payne, T. J.; Takanashi, A.; Sicilia, H.; Teo, S. L. Y.; McAuley, J. L.; Ellenberg, P.; Cooney, J. P.; Davidson, K. C.; Bowen, R.; Pellegrini, M.; Rockman, S.; Godfrey, D. I.; Nolan, T. M.; Wang, L.-F.; Deliyannis, G.; Purcell, D. F. J.; Pouton, C. W.</p>
        <p class="info">Score: 27.4, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560777' target='https://doi.org/10.1101/2023.10.04.560777'> 10.1101/2023.10.04.560777</a></p>
        <p class="abstract">To address the limitations of whole-spike COVID vaccines, we explored mRNA vaccines encoding membrane-anchored receptor-binding domain (RBD-TMs), each a fusion of a variant RBD, the transmembrane (TM) and cytoplasmic tail (CT) fragments of the SARS-CoV-2 spike protein. In naive mice, RBD-TM mRNA vaccines against ancestral SARS-CoV-2, Beta, Delta, Delta-plus, Kappa, Omicron BA.1 or BA.5, all induced strong humoral responses against the target RBD. Multiplex surrogate viral neutralization (sVNT) assays indicated broad neutralizing activity against a range of variant RBDs. In the setting of a heterologous boost, against the background of exposure to ancestral whole spike vaccines, sVNT studies suggested that RBD-TM vaccines were able to overcome the detrimental effects of immune imprinting. Omicron BA.1 and BA.5 RBD-TM booster vaccines induced serum antibodies with 12 and 22-fold higher neutralizing activity against the target RBD than their equivalent whole spike variants. Boosting with BA.1 or BA.5 RBD-TM provided good protection against more recent variants including XBB and XBB.1.5. Each RBD-TM mRNA is 28% of the length of its whole-spike equivalent. This advantage will enable tetravalent mRNA vaccines to be developed at well-tolerated doses of formulated mRNA.

One Sentence SummarymRNA vaccines encoding membrane-anchored RBDs of SARS-CoV-2 mutants are effective vaccines that can overcome immune imprinting in mice</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.563818">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.563818" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.563818">
        <p class="paperTitle">Vagus nerve stimulation limits the germinal center B cell response via CD4&#43; T cell-derived acetylcholine</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.563818" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.563818" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kurata-Sato, I.; Mughrabi, I. T.; Rana, M.; Gerber, M.; Al-Abed, Y.; Sherry, B.; Zanos, S.; Diamond, B.</p>
        <p class="info">Score: 5.0, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.563818' target='https://doi.org/10.1101/2023.10.25.563818'> 10.1101/2023.10.25.563818</a></p>
        <p class="abstract">Neural signals are known to contribute to immune regulation and modulation of the innate immune response downstream of the vagus nerve has been well studied. The effects of vagus nerve activity on antibody production, however, have been largely unexplored. Here we use a chronic vagus nerve stimulation (VNS) mouse model to study the effect of vagal activation on T-dependent B cell responses. We observed lower titers of high-affinity serum IgG and fewer antigen-specific germinal center (GC) B cells in the spleen. GC B cells from chronic VNS mice expressed more active caspase-3 and exhibited an altered gene expression profile suggesting increased susceptibility to apoptosis and impaired maturation. Follicular dendritic cell (FDC) cluster dispersal and altered FDC gene expression suggested poor FDC function. These alterations were diminished in the absence of a subset of acetylcholine-producing CD4&#43; T cells. In vitro studies revealed that 7 and 9 nicotinic acetylcholine receptors (nAChRs) directly regulated B cell production of TNF, a cytokine crucial to FDC clustering. Engagement of the 4 nAChR subunit on B cells impaired Akt phosphorylation, presumably decreasing B cell survival. Thus, VNS-induced GC impairment can be attributed, at least in part, to the effect of acetylcholine on B cell intrinsic pathways, resulting in hindered B cell survival and maturation and leading to an alteration in FDC function. Our findings identify a potential therapeutic target to prevent immunosuppression in conditions associated with increased vagal activity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561653">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561653" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561653">
        <p class="paperTitle">Scarring hair follicle destruction is driven by the collapse of EGFR-protected JAK-STAT1-sensitive stem cell immune privilege</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561653" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561653" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Strobl, K.; Klufa, J.; Jin, R.; Artner-Gent, L.; Krauss, D.; Novoszel, P.; Strobl, J.; Stary, G.; Vujic, I.; Griss, J.; Holcmann, M.; Farlik, M.; Homey, B.; Sibilia, M.; Bauer, T.</p>
        <p class="info">Score: 4.6, Published: 2023-10-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561653' target='https://doi.org/10.1101/2023.10.11.561653'> 10.1101/2023.10.11.561653</a></p>
        <p class="abstract">The hair follicle stem cell niche is an immune-privileged microenvironment, characterised by suppressed antigen presentation, thus shielding against permanent immune-mediated tissue damage. In this study, we demonstrate the protective role of hair follicle-specific epidermal growth factor receptor (EGFR) from scarring hair follicle degeneration. Mechanistically, disruption of EGFR signalling generates a cell intrinsic hypersensitivity within the JAK-STAT1 pathway, compromising the immune privilege in the context of CD8 T cell and NK cell-mediated inflammation. Genetic depletion of either JAK1/2 or STAT1 or topical therapeutic inhibition of JAK1/2 restores the immune privilege and activates stem cells to resume hair growth in mouse models of epidermal and hair follicle specific EGFR deletion. Skin biopsies from EGFR inhibitor-treated and from EGFR-independent cicatricial alopecia patients indicate active STAT1 signalling within the hair follicles. Notably, a case study of folliculitis decalvans, characterised by progressive hair loss, scaling and perifollicular erythema, demonstrates successful treatment with systemic JAK1/2 inhibition. Our findings offer mechanistic insights and present a therapeutic strategy for addressing scarring hair follicle destruction associated with EGFR-inhibitor therapy and cicatricial alopecia.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
