<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>immunology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>immunology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="XBB.1.5 monovalent mRNA vaccine booster elicits robust neutralizing antibodies against emerging SARS-CoV-2 variants
Authors: Wang, Q.; Guo, Y.; Bowen, A.; Mellis, I. A.; Valdez, R.; Gherasim, C.; Gordon, A.; Liu, L.; Ho, D. D.
Score: 984.1, Published: 2023-11-27 DOI: 10.1101/2023.11.26.568730
COVID-19 vaccines have recently been updated with the spike protein of SARS-Co-V-2 XBB.1.5 subvariant alone, but their immunogenicity in humans has yet to be fully evaluated and reported, particularly against emergent viruses that are rapidly expanding.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/immunology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="immunology" />
<meta property="og:description" content="XBB.1.5 monovalent mRNA vaccine booster elicits robust neutralizing antibodies against emerging SARS-CoV-2 variants
Authors: Wang, Q.; Guo, Y.; Bowen, A.; Mellis, I. A.; Valdez, R.; Gherasim, C.; Gordon, A.; Liu, L.; Ho, D. D.
Score: 984.1, Published: 2023-11-27 DOI: 10.1101/2023.11.26.568730
COVID-19 vaccines have recently been updated with the spike protein of SARS-Co-V-2 XBB.1.5 subvariant alone, but their immunogenicity in humans has yet to be fully evaluated and reported, particularly against emergent viruses that are rapidly expanding." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/immunology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-03T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-12-03T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="immunology"/>
<meta name="twitter:description" content="XBB.1.5 monovalent mRNA vaccine booster elicits robust neutralizing antibodies against emerging SARS-CoV-2 variants
Authors: Wang, Q.; Guo, Y.; Bowen, A.; Mellis, I. A.; Valdez, R.; Gherasim, C.; Gordon, A.; Liu, L.; Ho, D. D.
Score: 984.1, Published: 2023-11-27 DOI: 10.1101/2023.11.26.568730
COVID-19 vaccines have recently been updated with the spike protein of SARS-Co-V-2 XBB.1.5 subvariant alone, but their immunogenicity in humans has yet to be fully evaluated and reported, particularly against emergent viruses that are rapidly expanding."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "immunology",
      "item": "https://trxiv.yorks0n.com/posts/immunology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "immunology",
  "name": "immunology",
  "description": "XBB.1.5 monovalent mRNA vaccine booster elicits robust neutralizing antibodies against emerging SARS-CoV-2 variants\nAuthors: Wang, Q.; Guo, Y.; Bowen, A.; Mellis, I. A.; Valdez, R.; Gherasim, C.; Gordon, A.; Liu, L.; Ho, D. D.\nScore: 984.1, Published: 2023-11-27 DOI: 10.1101/2023.11.26.568730\nCOVID-19 vaccines have recently been updated with the spike protein of SARS-Co-V-2 XBB.1.5 subvariant alone, but their immunogenicity in humans has yet to be fully evaluated and reported, particularly against emergent viruses that are rapidly expanding.",
  "keywords": [
    
  ],
  "articleBody": " XBB.1.5 monovalent mRNA vaccine booster elicits robust neutralizing antibodies against emerging SARS-CoV-2 variants\nAuthors: Wang, Q.; Guo, Y.; Bowen, A.; Mellis, I. A.; Valdez, R.; Gherasim, C.; Gordon, A.; Liu, L.; Ho, D. D.\nScore: 984.1, Published: 2023-11-27 DOI: 10.1101/2023.11.26.568730\nCOVID-19 vaccines have recently been updated with the spike protein of SARS-Co-V-2 XBB.1.5 subvariant alone, but their immunogenicity in humans has yet to be fully evaluated and reported, particularly against emergent viruses that are rapidly expanding. We now report that administration of an updated monovalent mRNA vaccine (XBB.1.5 MV) to uninfected individuals boosted serum virus-neutralization antibodies significantly against not only XBB.1.5 (27.0-fold) and the currently dominant EG.5.1 (27.6-fold) but also key emergent viruses like HV.1, HK.3, JD.1.1, and JN.1 (13.3-to-27.4-fold). In individuals previously infected by an Omicron subvariant, serum neutralizing titers were boosted to highest levels (1,764-to-22,978) against all viral variants tested. While immunological imprinting was still evident with the updated vaccines, it was not nearly as severe as the previously authorized bivalent BA.5 vaccine. Our findings strongly support the official recommendation to widely apply the updated COVID-19 vaccines to further protect the public.\nPersistent immune imprinting after XBB.1.5 COVID vaccination in humans\nAuthors: Tortorici, M. A.; Addetia, A.; Seo, A. J.; Brown, J.; Sprouse, K.; Logue, J.; Clarke, E.; Franko, N.; Chu, H.; Veesler, D.\nScore: 92.8, Published: 2023-11-30 DOI: 10.1101/2023.11.28.569129\nImmune imprinting - also known as original antigenic sin - describes how the first exposure to a virus shapes the immunological outcome of subsequent exposures to antigenically related strains. SARS-CoV-2 Omicron breakthrough infections and bivalent COVID-19 vaccination were shown to primarily recall cross-reactive memory B cells and antibodies induced by prior mRNA vaccination with the Wuhan-Hu-1 spike rather than priming naive B cells that recognize Omicron-specific epitopes. These findings underscored a strong immune imprinting resulting from repeated Wuhan-Hu-1 spike exposures. To understand if immune imprinting can be overcome, we investigated memory and plasma antibody responses after administration of the updated XBB.1.5 COVID mRNA vaccine booster. Our data show that the XBB.1.5 booster elicits neutralizing antibody responses against current variants that are dominated by recall of pre-existing memory B cells previously induced by the Wuhan-Hu-1 spike. These results indicate that immune imprinting persists even after multiple exposures to Omicron spikes through vaccination and infection, including post XBB.1.5 spike booster mRNA vaccination, which will need to be considered to guide the design of future vaccine boosters.\nFast evolution of SARS-CoV-2 BA.2.86 to JN.1 under heavy immune pressure\nAuthors: Yang, S.; Yu, Y.; Xu, Y.; Jian, F.; Song, W.; Yisimayi, A.; Wang, P.; Wang, J.; Liu, J.; Yu, L.; Niu, X.; Wang, J.; Wang, Y.; Shao, F.; Jin, R.; Wang, Y.; Cao, Y.\nScore: 326.7, Published: 2023-11-17 DOI: 10.1101/2023.11.13.566860\nWhile the BA.2.86 variant demonstrated significant antigenic drift and enhanced ACE2 binding affinity, its ability to evade humoral immunity was relatively moderate compared to dominant strains like EG.5 and HK.3. However, the emergence of a new subvariant, JN.1 (BA.2.86.1.1), which possesses an additional spike mutation, L455S, compared to BA.2.86, showed a markedly increased prevalence in Europe and North America, especially in France. Here, we found that L455S of JN.1 significantly enhances immune evasion capabilities at the expense of reduced ACE2 binding affinity. This mutation enables JN.1 to effectively evade Class 1 neutralizing antibodies, offsetting BA.2.86s susceptibility and thus allowing it to outcompete both its precursor BA.2.86 and the prevailing variants HV.1 (XBB.1.5+L452R+F456L) and JD.1.1 (XBB.1.5+L455F+F456L+A475V) in terms of humoral immune evasion. The rapid evolution from BA.2.86 to JN.1, similar to the earlier transition from BA.2.75 to CH.1.1, highlights the importance of closely monitoring strains with high ACE2 binding affinity and distinct antigenicity, despite their temporarily unremarkable immune evasion capabilities. Such strains could survive and transmit at low levels, since their large antigenic distance to dominant strains allow them to target distinct populations and accumulate immune-evasive mutations rapidly, often at the cost of receptor binding affinity.\nInterpretable Inflammation Landscape of Circulating Immune cells\nAuthors: Jimenez-Gracia, L.; Maspero, D.; Aguilar-Fernandez, S.; Craighero, F.; Ruiz, S.; Marchese, D.; Caratu, G.; Elosua-Bayes, M.; Abdalfatah, M.; Sanzo-Machuca, A.; M. Corraliza, A.; Massoni-Badosa, R.; A. Tran, H.; Normand, R.; Nestor, J.; Hong, Y.; Kole, T.; van der Velde, P.; Alleblas, F.; Pedretti, F.; Aterido, A.; Banchero, M.; Soriano, G.; Roman, E.; van den Berge, M.; Salas, A.; Carrascosa, J. M.; Fernandez Nebro, A.; Domenech, E.; Canete, J.; Tornero, J.; Perez-Gisbert, J.; Choy, E.; Girolomoni, G.; Siegmund, B.; Julia, A.; Serra, V.; Elosua, R.; Tejpar, S.; Vidal, S.; Nawijn, M. C.; Marsal\nScore: 51.0, Published: 2023-11-28 DOI: 10.1101/2023.11.28.568839\nInflammation is a biological phenomenon involved in a wide variety of physiological and pathological processes. Although a controlled inflammatory response is beneficial for restoring homeostasis, it can become unfavorable if dysregulated. In recent years, major progress has been made in characterizing acute and chronic inflammation in specific diseases. However, a global, holistic understanding of inflammation is still elusive. This is particularly intriguing, considering the crucial function of inflammation for human health and its potential for modern medicine if fully deciphered. Here, we leverage advances in the field of single-cell genomics to delineate the full spectrum of circulating immune cell activation underlying inflammatory processes during infection, immune-mediated inflammatory diseases and cancer. Our single-cell atlas of \u003e2 million peripheral blood mononuclear cells from 356 patients and 18 diseases allowed us to learn a foundation model of inflammation in circulating immune cells. The atlas expanded our current knowledge of the biology of inflammation of acute (e.g. inflammatory bowel disease, sepsis) and chronic (e.g. cirrhosis, asthma, and chronic obstructive pulmonary disease) disease processes and laid the foundation to develop a precision medicine framework using unsupervised as well as explainable machine learning. Beyond a disease-centered classification, we charted altered activity of inflammatory molecules in peripheral blood cells, depicting functional biomarkers to further understand mechanisms of inflammation. Finally, we have laid the groundwork for developing precision medicine diagnostic tools for patients experiencing severe acute or chronic inflammation by learning a classifier for inflammatory diseases, presenting cells in circulation as a powerful resource for patient stratification.\nAntiviral innate immune memory in alveolar macrophages following SARS-CoV-2 infection.\nAuthors: Lercher, A.; Cheong, J.-G.; Jiang, C.; Hoffmann, H.-H.; Ashbrook, A. W.; Yin, Y. S.; Quirk, C.; DeGrace, E. J.; Chiriboga, L.; Rosenberg, B. R.; Josefowicz, S. Z.; Rice, C. M.\nScore: 33.2, Published: 2023-11-27 DOI: 10.1101/2023.11.24.568354\nPathogen encounter results in long-lasting epigenetic imprinting that shapes diseases caused by heterologous pathogens. The breadth of this innate immune memory is of particular interest in the context of respiratory pathogens with increased pandemic potential and wide-ranging impact on global health. Here, we investigated epigenetic imprinting across cell lineages in a disease relevant murine model of SARS-CoV-2 recovery. Past SARS-CoV-2 infection resulted in increased chromatin accessibility of type I interferon (IFN-I) related transcription factors in airway-resident macrophages. Mechanistically, establishment of this innate immune memory required viral pattern recognition and canonical IFN-I signaling and augmented secondary antiviral responses. Past SARS-CoV-2 infection ameliorated disease caused by the heterologous respiratory pathogen influenza A virus. Insights into innate immune memory and how it affects subsequent infections with heterologous pathogens to influence disease pathology could facilitate the development of broadly effective therapeutic strategies.\nMgl2+ cDC2s coordinate fungal allergic airway type 2, but not type 17, inflammation.\nAuthors: Cook, P. C.; Brown, S. L.; Houlder, E. L.; Baker, S.; Svedberg, F. R.; Howell, G.; Bertuzzi, M.; Boon, L.; Konkel, J. E.; Allen, J. E.; MacDonald, A. S.\nScore: 16.5, Published: 2023-11-25 DOI: 10.1101/2023.11.24.568263\nFungal spores are abundant in the environment and a major cause of asthma. Originally characterised as a type 2 inflammatory disease, allergic airway inflammation that underpins asthma can also involve type 17 inflammation, which can exacerbate disease causing failure of treatments tailored to inhibit type 2 factors. However, the mechanisms that determine the host response to fungi, which can trigger both type 2 and type 17 inflammation in allergic airway disease, remain unclear. We found that CD11c+ DCs and CD4+ T cells are essential for development of both type 2 and type 17 airway inflammation in mice when repeatedly exposed to inhaled spores. Single cell RNA-sequencing enabled the development of multi-parameter cytometry that identified allergic inflammation dramatically altered the proportion of numerous DC clusters in the lung, but that only two of these (Mgl2+ cDC2s and CCR7+ DCs) migrated to the dLNs. Targeted removal of several DC subsets revealed that only Mgl2+ cDC2 depletion dramatically reduced type 2, but not type 17, anti-fungal allergic airway inflammation. These data highlight distinct DC subsets are potential therapeutic targets for the treatment of pulmonary fungal disease.\nPathogenic IgE-fated B cell memory retains functional plasticity\nAuthors: Bruton, K.; Phelps, A.; Ariaz, A.; Walker, T.; Wen, J.; Khavkine-Binstock, S. S.; Fang, A.; Mann-Delany, O.; Knudsen, N. P. H.; Gadkar, S.; Grydziuszko, E.; Koenig, J. F.; Gagnon, A.; Waserman, S.; Andersen, P. S.; Jordana, M.\nScore: 8.5, Published: 2023-11-29 DOI: 10.1101/2023.11.28.567094\nLong-lived immunoglobulin (Ig) E responses against innocuous environmental and dietary antigens (Ags) are maintained by an IgG1-dominant memory B cell (MBC) compartment primed for IL-4 responsiveness. In this work, we investigated the plasticity of the MBC compartment fated for IgE class switch recombination upon Ag re-exposure. Antibody-mediated IL-4R( blockade augmented the germinal center response and uncovered an IL-4/IL-13 dependency within the type 2 memory B cell (MBC2) skewed phenotype. In the absence of IL-4/IL-13 signaling during the recall response, a long-lived Ag-specific IgG2c MBC population emerged, shifting the MBC response away from a type 2 phenotype and towards type 1. The emergence of this IgG2c response was dependent on IFN-( signaling and arose from both unswitched and class-switched Ag-specific B cells in vivo. We further demonstrated that a type 1-polarizing adjuvant can augment the redirection of type 2-polarized class-switched MBCs, suggesting with the correct cues pathogenic MBCs can be functionally reprogrammed. One Sentence SummaryB cell responses to allergens can be reprogrammed away from a pathogenic fate through IL-4/IL-13 signaling blockade.\nLong-term B cell memory emerges at uniform relative rates in the human immune response\nAuthors: Cvijovic, I.; Swift, M.; Quake, S. R.\nScore: 8.4, Published: 2023-11-28 DOI: 10.1101/2023.11.27.568934\nB cells generate pathogen-specific antibodies and play an essential role in providing adaptive protection against infection. Antibody genes are modified in evolutionary processes acting on the B cell populations within an individual. These populations proliferate, differentiate, and migrate to long-term niches in the body. However, the dynamics of these processes in the human immune system are primarily inferred from mouse studies. We addressed this gap by sequencing the antibody repertoire and transcriptomes from single B cells in four immune-rich tissues from six individuals. We find that B cells descended from the same pre-B cell (\"lineages\") often co-localize within the same tissue, with the bone marrow harboring the largest excess of lineages without representation in other tissues. Within lineages, cells with different levels of somatic hypermutation are uniformly distributed among tissues and functional states. This suggests that the relative probabilities of localization and differentiation decisions change negligibly during affinity maturation, and quantitatively agrees with a simple dynamical model of B cell differentiation. While lineages strongly co-localize, we find individual B cells nevertheless make independent differentiation decisions. Proliferative antibody secreting cells, however, deviate from these global patterns. These cells are often clonally expanded, their clones appear universally distributed among all sampled organs, and form lineages with an excess of cells of the same type. Collectively, our findings show the limits of peripheral blood monitoring of the immune repertoire, and provide a probabilistic model of the dynamics of antibody memory formation in humans.\nAn Environmental Restriction impairs HIV-1 virion fusion and triggers innate immune recognition\nAuthors: Sid Ahmed, S.; Zimmermann, L.; Imle, A.; Wuebben, K.; Tibroni, N.; Rauch-Wirth, L.; Muench, J.; Chlanda, P.; Graw, F.; Fackler, O. T.\nScore: 7.0, Published: 2023-11-23 DOI: 10.1101/2023.11.22.568261\nIn vivo, HIV-1 replicates within 3D tissues, yet the impact of tissue-like environments on viral spread is largely unknown. Our previous research identified that synthetic 3D environments impose an Environmental Restriction to cell-free Virus Infectivity (ERVI) that diminishes HIV-1 particle infectivity. Here, mechanistic studies reveal that ERVI is implemented within minutes, saturable and induced by different adhesive tissue-like 3D matrices. ERVI reduces infectivity across a wide range of primary HIV-1 strains and virions bearing distinct viral glycoproteins but does not damage virion morphology or affect their binding to target cells. Rather, ERVI impairs virion fusion with target cells and infectivity enhancing peptide nanofibrils can restore efficient infection. In addition, ERVI sensitizes HIV-1 particles for recognition by monocyte-derived macrophages via toll-like receptors 4 and 8, triggering pronounced pro-inflammatory cytokine secretion. These results suggest that ERVI represents a broadly acting, tissue-intrinsic barrier to virus spread that reduces the fusogenicity of cell-free virions and sensitizes them for innate immune recognition.\nEpithelial organoid supports resident memory CD8 T cell differentiation\nAuthors: Ulibarri, M.; Lin, Y.; Ramprashad, J. R.; Han, G.; Hasan, M. H.; Mithila, F. J.; Ma, C.; Gopinath, S.; Zhang, N.; Milner, J. J.; Beura, L.\nScore: 5.2, Published: 2023-12-01 DOI: 10.1101/2023.12.01.569395\nResident Memory T cells (TRM) play a vital role in regional immune defense in barrier organs. Although laboratory rodents have been extensively used to study fundamental TRM biology, poor isolation efficiency, sampling bias and low cell survival rates have limited our ability to conduct TRM-focused high-throughput assays. Here, we engineered a murine vaginal epithelial organoid (VEO)-CD8 T cell coculture system that supports CD8 TRM differentiation in vitro. The three-dimensional VEOs established from murine adult stem cells resembled stratified squamous vaginal epithelium and induced gradual differentiation of activated CD8 T cells into epithelial TRM. These in vitro generated TRM were phenotypically and transcriptionally similar to in vivo TRM, and key tissue residency features were reinforced with a second cognate-antigen exposure during coculture. TRM differentiation was not affected even when VEOs and CD8 T cells were separated by a semipermeable barrier, indicating soluble factors' involvement. Pharmacological and genetic approaches showed that TGF-{beta} signaling played a crucial role in their differentiation. We found that the VEOs in our model remained susceptible to viral infections and the CD8 T cells were amenable to genetic manipulation; both of which will allow detailed interrogation of antiviral CD8 T cell biology in a reductionist setting. In summary, we established a robust model which captures bonafide TRM differentiation that is scalable, open to iterative sampling, and can be subjected to high throughput assays that will rapidly add to our fundamental understanding of TRM.\n",
  "wordCount" : "2380",
  "inLanguage": "en",
  "datePublished": "2023-12-03T10:38:01Z",
  "dateModified": "2023-12-03T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/immunology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      immunology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 3, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.26.568730">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.26.568730" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.26.568730">
        <p class="paperTitle">XBB.1.5 monovalent mRNA vaccine booster elicits robust neutralizing antibodies against emerging SARS-CoV-2 variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.26.568730" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.26.568730" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Q.; Guo, Y.; Bowen, A.; Mellis, I. A.; Valdez, R.; Gherasim, C.; Gordon, A.; Liu, L.; Ho, D. D.</p>
        <p class="info">Score: 984.1, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.26.568730' target='https://doi.org/10.1101/2023.11.26.568730'> 10.1101/2023.11.26.568730</a></p>
        <p class="abstract">COVID-19 vaccines have recently been updated with the spike protein of SARS-Co-V-2 XBB.1.5 subvariant alone, but their immunogenicity in humans has yet to be fully evaluated and reported, particularly against emergent viruses that are rapidly expanding. We now report that administration of an updated monovalent mRNA vaccine (XBB.1.5 MV) to uninfected individuals boosted serum virus-neutralization antibodies significantly against not only XBB.1.5 (27.0-fold) and the currently dominant EG.5.1 (27.6-fold) but also key emergent viruses like HV.1, HK.3, JD.1.1, and JN.1 (13.3-to-27.4-fold). In individuals previously infected by an Omicron subvariant, serum neutralizing titers were boosted to highest levels (1,764-to-22,978) against all viral variants tested. While immunological imprinting was still evident with the updated vaccines, it was not nearly as severe as the previously authorized bivalent BA.5 vaccine. Our findings strongly support the official recommendation to widely apply the updated COVID-19 vaccines to further protect the public.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569129">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569129" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569129">
        <p class="paperTitle">Persistent immune imprinting after XBB.1.5 COVID vaccination in humans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569129" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569129" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tortorici, M. A.; Addetia, A.; Seo, A. J.; Brown, J.; Sprouse, K.; Logue, J.; Clarke, E.; Franko, N.; Chu, H.; Veesler, D.</p>
        <p class="info">Score: 92.8, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569129' target='https://doi.org/10.1101/2023.11.28.569129'> 10.1101/2023.11.28.569129</a></p>
        <p class="abstract">Immune imprinting - also known as  original antigenic sin - describes how the first exposure to a virus shapes the immunological outcome of subsequent exposures to antigenically related strains. SARS-CoV-2 Omicron breakthrough infections and bivalent COVID-19 vaccination were shown to primarily recall cross-reactive memory B cells and antibodies induced by prior mRNA vaccination with the Wuhan-Hu-1 spike rather than priming naive B cells that recognize Omicron-specific epitopes. These findings underscored a strong immune imprinting resulting from repeated Wuhan-Hu-1 spike exposures. To understand if immune imprinting can be overcome, we investigated memory and plasma antibody responses after administration of the updated XBB.1.5 COVID mRNA vaccine booster. Our data show that the XBB.1.5 booster elicits neutralizing antibody responses against current variants that are dominated by recall of pre-existing memory B cells previously induced by the Wuhan-Hu-1 spike. These results indicate that immune imprinting persists even after multiple exposures to Omicron spikes through vaccination and infection, including post XBB.1.5 spike booster mRNA vaccination, which will need to be considered to guide the design of future vaccine boosters.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566860">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566860" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566860">
        <p class="paperTitle">Fast evolution of SARS-CoV-2 BA.2.86 to JN.1 under heavy immune pressure</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566860" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566860" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, S.; Yu, Y.; Xu, Y.; Jian, F.; Song, W.; Yisimayi, A.; Wang, P.; Wang, J.; Liu, J.; Yu, L.; Niu, X.; Wang, J.; Wang, Y.; Shao, F.; Jin, R.; Wang, Y.; Cao, Y.</p>
        <p class="info">Score: 326.7, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566860' target='https://doi.org/10.1101/2023.11.13.566860'> 10.1101/2023.11.13.566860</a></p>
        <p class="abstract">While the BA.2.86 variant demonstrated significant antigenic drift and enhanced ACE2 binding affinity, its ability to evade humoral immunity was relatively moderate compared to dominant strains like EG.5 and HK.3. However, the emergence of a new subvariant, JN.1 (BA.2.86.1.1), which possesses an additional spike mutation, L455S, compared to BA.2.86, showed a markedly increased prevalence in Europe and North America, especially in France. Here, we found that L455S of JN.1 significantly enhances immune evasion capabilities at the expense of reduced ACE2 binding affinity. This mutation enables JN.1 to effectively evade Class 1 neutralizing antibodies, offsetting BA.2.86s susceptibility and thus allowing it to outcompete both its precursor BA.2.86 and the prevailing variants HV.1 (XBB.1.5&#43;L452R&#43;F456L) and JD.1.1 (XBB.1.5&#43;L455F&#43;F456L&#43;A475V) in terms of humoral immune evasion. The rapid evolution from BA.2.86 to JN.1, similar to the earlier transition from BA.2.75 to CH.1.1, highlights the importance of closely monitoring strains with high ACE2 binding affinity and distinct antigenicity, despite their temporarily unremarkable immune evasion capabilities. Such strains could survive and transmit at low levels, since their large antigenic distance to dominant strains allow them to target distinct populations and accumulate immune-evasive mutations rapidly, often at the cost of receptor binding affinity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.568839">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.568839" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.568839">
        <p class="paperTitle">Interpretable Inflammation Landscape of Circulating Immune cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.568839" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.568839" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jimenez-Gracia, L.; Maspero, D.; Aguilar-Fernandez, S.; Craighero, F.; Ruiz, S.; Marchese, D.; Caratu, G.; Elosua-Bayes, M.; Abdalfatah, M.; Sanzo-Machuca, A.; M. Corraliza, A.; Massoni-Badosa, R.; A. Tran, H.; Normand, R.; Nestor, J.; Hong, Y.; Kole, T.; van der Velde, P.; Alleblas, F.; Pedretti, F.; Aterido, A.; Banchero, M.; Soriano, G.; Roman, E.; van den Berge, M.; Salas, A.; Carrascosa, J. M.; Fernandez Nebro, A.; Domenech, E.; Canete, J.; Tornero, J.; Perez-Gisbert, J.; Choy, E.; Girolomoni, G.; Siegmund, B.; Julia, A.; Serra, V.; Elosua, R.; Tejpar, S.; Vidal, S.; Nawijn, M. C.; Marsal</p>
        <p class="info">Score: 51.0, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.568839' target='https://doi.org/10.1101/2023.11.28.568839'> 10.1101/2023.11.28.568839</a></p>
        <p class="abstract">Inflammation is a biological phenomenon involved in a wide variety of physiological and pathological processes. Although a controlled inflammatory response is beneficial for restoring homeostasis, it can become unfavorable if dysregulated. In recent years, major progress has been made in characterizing acute and chronic inflammation in specific diseases. However, a global, holistic understanding of inflammation is still elusive. This is particularly intriguing, considering the crucial function of inflammation for human health and its potential for modern medicine if fully deciphered. Here, we leverage advances in the field of single-cell genomics to delineate the full spectrum of circulating immune cell activation underlying inflammatory processes during infection, immune-mediated inflammatory diseases and cancer. Our single-cell atlas of &gt;2 million peripheral blood mononuclear cells from 356 patients and 18 diseases allowed us to learn a foundation model of inflammation in circulating immune cells. The atlas expanded our current knowledge of the biology of inflammation of acute (e.g. inflammatory bowel disease, sepsis) and chronic (e.g. cirrhosis, asthma, and chronic obstructive pulmonary disease) disease processes and laid the foundation to develop a precision medicine framework using unsupervised as well as explainable machine learning. Beyond a disease-centered classification, we charted altered activity of inflammatory molecules in peripheral blood cells, depicting functional biomarkers to further understand mechanisms of inflammation. Finally, we have laid the groundwork for developing precision medicine diagnostic tools for patients experiencing severe acute or chronic inflammation by learning a classifier for inflammatory diseases, presenting cells in circulation as a powerful resource for patient stratification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.24.568354">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.24.568354" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.24.568354">
        <p class="paperTitle">Antiviral innate immune memory in alveolar macrophages following SARS-CoV-2 infection.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.24.568354" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.24.568354" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lercher, A.; Cheong, J.-G.; Jiang, C.; Hoffmann, H.-H.; Ashbrook, A. W.; Yin, Y. S.; Quirk, C.; DeGrace, E. J.; Chiriboga, L.; Rosenberg, B. R.; Josefowicz, S. Z.; Rice, C. M.</p>
        <p class="info">Score: 33.2, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.24.568354' target='https://doi.org/10.1101/2023.11.24.568354'> 10.1101/2023.11.24.568354</a></p>
        <p class="abstract">Pathogen encounter results in long-lasting epigenetic imprinting that shapes diseases caused by heterologous pathogens. The breadth of this innate immune memory is of particular interest in the context of respiratory pathogens with increased pandemic potential and wide-ranging impact on global health. Here, we investigated epigenetic imprinting across cell lineages in a disease relevant murine model of SARS-CoV-2 recovery. Past SARS-CoV-2 infection resulted in increased chromatin accessibility of type I interferon (IFN-I) related transcription factors in airway-resident macrophages. Mechanistically, establishment of this innate immune memory required viral pattern recognition and canonical IFN-I signaling and augmented secondary antiviral responses. Past SARS-CoV-2 infection ameliorated disease caused by the heterologous respiratory pathogen influenza A virus. Insights into innate immune memory and how it affects subsequent infections with heterologous pathogens to influence disease pathology could facilitate the development of broadly effective therapeutic strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.24.568263">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.24.568263" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.24.568263">
        <p class="paperTitle">Mgl2&#43; cDC2s coordinate fungal allergic airway type 2, but not type 17, inflammation.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.24.568263" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.24.568263" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cook, P. C.; Brown, S. L.; Houlder, E. L.; Baker, S.; Svedberg, F. R.; Howell, G.; Bertuzzi, M.; Boon, L.; Konkel, J. E.; Allen, J. E.; MacDonald, A. S.</p>
        <p class="info">Score: 16.5, Published: 2023-11-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.24.568263' target='https://doi.org/10.1101/2023.11.24.568263'> 10.1101/2023.11.24.568263</a></p>
        <p class="abstract">Fungal spores are abundant in the environment and a major cause of asthma. Originally characterised as a type 2 inflammatory disease, allergic airway inflammation that underpins asthma can also involve type 17 inflammation, which can exacerbate disease causing failure of treatments tailored to inhibit type 2 factors. However, the mechanisms that determine the host response to fungi, which can trigger both type 2 and type 17 inflammation in allergic airway disease, remain unclear. We found that CD11c&#43; DCs and CD4&#43; T cells are essential for development of both type 2 and type 17 airway inflammation in mice when repeatedly exposed to inhaled spores. Single cell RNA-sequencing enabled the development of multi-parameter cytometry that identified allergic inflammation dramatically altered the proportion of numerous DC clusters in the lung, but that only two of these (Mgl2&#43; cDC2s and CCR7&#43; DCs) migrated to the dLNs. Targeted removal of several DC subsets revealed that only Mgl2&#43; cDC2 depletion dramatically reduced type 2, but not type 17, anti-fungal allergic airway inflammation. These data highlight distinct DC subsets are potential therapeutic targets for the treatment of pulmonary fungal disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.567094">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.567094" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.567094">
        <p class="paperTitle">Pathogenic IgE-fated B cell memory retains functional plasticity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.567094" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.567094" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bruton, K.; Phelps, A.; Ariaz, A.; Walker, T.; Wen, J.; Khavkine-Binstock, S. S.; Fang, A.; Mann-Delany, O.; Knudsen, N. P. H.; Gadkar, S.; Grydziuszko, E.; Koenig, J. F.; Gagnon, A.; Waserman, S.; Andersen, P. S.; Jordana, M.</p>
        <p class="info">Score: 8.5, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.567094' target='https://doi.org/10.1101/2023.11.28.567094'> 10.1101/2023.11.28.567094</a></p>
        <p class="abstract">Long-lived immunoglobulin (Ig) E responses against innocuous environmental and dietary antigens (Ags) are maintained by an IgG1-dominant memory B cell (MBC) compartment primed for IL-4 responsiveness. In this work, we investigated the plasticity of the MBC compartment fated for IgE class switch recombination upon Ag re-exposure. Antibody-mediated IL-4R( blockade augmented the germinal center response and uncovered an IL-4/IL-13 dependency within the type 2 memory B cell (MBC2) skewed phenotype. In the absence of IL-4/IL-13 signaling during the recall response, a long-lived Ag-specific IgG2c MBC population emerged, shifting the MBC response away from a type 2 phenotype and towards type 1. The emergence of this IgG2c response was dependent on IFN-( signaling and arose from both unswitched and class-switched Ag-specific B cells in vivo. We further demonstrated that a type 1-polarizing adjuvant can augment the redirection of type 2-polarized class-switched MBCs, suggesting with the correct cues pathogenic MBCs can be functionally reprogrammed.

One Sentence SummaryB cell responses to allergens can be reprogrammed away from a pathogenic fate through IL-4/IL-13 signaling blockade.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568934">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568934" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568934">
        <p class="paperTitle">Long-term B cell memory emerges at uniform relative rates in the human immune response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568934" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568934" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cvijovic, I.; Swift, M.; Quake, S. R.</p>
        <p class="info">Score: 8.4, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568934' target='https://doi.org/10.1101/2023.11.27.568934'> 10.1101/2023.11.27.568934</a></p>
        <p class="abstract">B cells generate pathogen-specific antibodies and play an essential role in providing adaptive protection against infection. Antibody genes are modified in evolutionary processes acting on the B cell populations within an individual. These populations proliferate, differentiate, and migrate to long-term niches in the body. However, the dynamics of these processes in the human immune system are primarily inferred from mouse studies. We addressed this gap by sequencing the antibody repertoire and transcriptomes from single B cells in four immune-rich tissues from six individuals. We find that B cells descended from the same pre-B cell (&#34;lineages&#34;) often co-localize within the same tissue, with the bone marrow harboring the largest excess of lineages without representation in other tissues. Within lineages, cells with different levels of somatic hypermutation are uniformly distributed among tissues and functional states. This suggests that the relative probabilities of localization and differentiation decisions change negligibly during affinity maturation, and quantitatively agrees with a simple dynamical model of B cell differentiation. While lineages strongly co-localize, we find individual B cells nevertheless make independent differentiation decisions. Proliferative antibody secreting cells, however, deviate from these global patterns. These cells are often clonally expanded, their clones appear universally distributed among all sampled organs, and form lineages with an excess of cells of the same type. Collectively, our findings show the limits of peripheral blood monitoring of the immune repertoire, and provide a probabilistic model of the dynamics of antibody memory formation in humans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568261">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568261" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568261">
        <p class="paperTitle">An Environmental Restriction impairs HIV-1 virion fusion and triggers innate immune recognition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568261" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568261" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sid Ahmed, S.; Zimmermann, L.; Imle, A.; Wuebben, K.; Tibroni, N.; Rauch-Wirth, L.; Muench, J.; Chlanda, P.; Graw, F.; Fackler, O. T.</p>
        <p class="info">Score: 7.0, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568261' target='https://doi.org/10.1101/2023.11.22.568261'> 10.1101/2023.11.22.568261</a></p>
        <p class="abstract">In vivo, HIV-1 replicates within 3D tissues, yet the impact of tissue-like environments on viral spread is largely unknown. Our previous research identified that synthetic 3D environments impose an Environmental Restriction to cell-free Virus Infectivity (ERVI) that diminishes HIV-1 particle infectivity. Here, mechanistic studies reveal that ERVI is implemented within minutes, saturable and induced by different adhesive tissue-like 3D matrices. ERVI reduces infectivity across a wide range of primary HIV-1 strains and virions bearing distinct viral glycoproteins but does not damage virion morphology or affect their binding to target cells. Rather, ERVI impairs virion fusion with target cells and infectivity enhancing peptide nanofibrils can restore efficient infection. In addition, ERVI sensitizes HIV-1 particles for recognition by monocyte-derived macrophages via toll-like receptors 4 and 8, triggering pronounced pro-inflammatory cytokine secretion. These results suggest that ERVI represents a broadly acting, tissue-intrinsic barrier to virus spread that reduces the fusogenicity of cell-free virions and sensitizes them for innate immune recognition.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569395">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569395" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569395">
        <p class="paperTitle">Epithelial organoid supports resident memory CD8 T cell differentiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569395" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569395" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ulibarri, M.; Lin, Y.; Ramprashad, J. R.; Han, G.; Hasan, M. H.; Mithila, F. J.; Ma, C.; Gopinath, S.; Zhang, N.; Milner, J. J.; Beura, L.</p>
        <p class="info">Score: 5.2, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569395' target='https://doi.org/10.1101/2023.12.01.569395'> 10.1101/2023.12.01.569395</a></p>
        <p class="abstract">Resident Memory T cells (TRM) play a vital role in regional immune defense in barrier organs. Although laboratory rodents have been extensively used to study fundamental TRM biology, poor isolation efficiency, sampling bias and low cell survival rates have limited our ability to conduct TRM-focused high-throughput assays. Here, we engineered a murine vaginal epithelial organoid (VEO)-CD8 T cell coculture system that supports CD8 TRM differentiation in vitro. The three-dimensional VEOs established from murine adult stem cells resembled stratified squamous vaginal epithelium and induced gradual differentiation of activated CD8 T cells into epithelial TRM. These in vitro generated TRM were phenotypically and transcriptionally similar to in vivo TRM, and key tissue residency features were reinforced with a second cognate-antigen exposure during coculture. TRM differentiation was not affected even when VEOs and CD8 T cells were separated by a semipermeable barrier, indicating soluble factors&#39; involvement. Pharmacological and genetic approaches showed that TGF-{beta} signaling played a crucial role in their differentiation. We found that the VEOs in our model remained susceptible to viral infections and the CD8 T cells were amenable to genetic manipulation; both of which will allow detailed interrogation of antiviral CD8 T cell biology in a reductionist setting. In summary, we established a robust model which captures bonafide TRM differentiation that is scalable, open to iterative sampling, and can be subjected to high throughput assays that will rapidly add to our fundamental understanding of TRM.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
