<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The receptor-like kinase BIR1 inhibits elicitor-induced plasmodesmata callose deposition and PTI gene expression and requires EDS1 and SOBIR1 to cause dose-dependent cell-death in Arabidopsis
Authors: GUZMAN-BENITO, I.; ROBINSON, C.; HUA, C.; SEDE, A. R.; ELVIRA-GONZALEZ, L.; PUNZON, I.; HEINLEIN, M.; NURNBERGER, T.; LLAVE, C.
Score: 16.6, Published: 2023-06-23 DOI: 10.1101/2023.06.23.546234
O_LIThe receptor-like kinase BIR1 functions as a negative regulator of cell death and defense in Arabidopsis. Previous studies showed that BIR1 expression is up-regulated during infections with microbes and viruses.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="The receptor-like kinase BIR1 inhibits elicitor-induced plasmodesmata callose deposition and PTI gene expression and requires EDS1 and SOBIR1 to cause dose-dependent cell-death in Arabidopsis
Authors: GUZMAN-BENITO, I.; ROBINSON, C.; HUA, C.; SEDE, A. R.; ELVIRA-GONZALEZ, L.; PUNZON, I.; HEINLEIN, M.; NURNBERGER, T.; LLAVE, C.
Score: 16.6, Published: 2023-06-23 DOI: 10.1101/2023.06.23.546234
O_LIThe receptor-like kinase BIR1 functions as a negative regulator of cell death and defense in Arabidopsis. Previous studies showed that BIR1 expression is up-regulated during infections with microbes and viruses." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-02T10:39:32+00:00" />
<meta property="article:modified_time" content="2023-07-02T10:39:32+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="The receptor-like kinase BIR1 inhibits elicitor-induced plasmodesmata callose deposition and PTI gene expression and requires EDS1 and SOBIR1 to cause dose-dependent cell-death in Arabidopsis
Authors: GUZMAN-BENITO, I.; ROBINSON, C.; HUA, C.; SEDE, A. R.; ELVIRA-GONZALEZ, L.; PUNZON, I.; HEINLEIN, M.; NURNBERGER, T.; LLAVE, C.
Score: 16.6, Published: 2023-06-23 DOI: 10.1101/2023.06.23.546234
O_LIThe receptor-like kinase BIR1 functions as a negative regulator of cell death and defense in Arabidopsis. Previous studies showed that BIR1 expression is up-regulated during infections with microbes and viruses."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "The receptor-like kinase BIR1 inhibits elicitor-induced plasmodesmata callose deposition and PTI gene expression and requires EDS1 and SOBIR1 to cause dose-dependent cell-death in Arabidopsis\nAuthors: GUZMAN-BENITO, I.; ROBINSON, C.; HUA, C.; SEDE, A. R.; ELVIRA-GONZALEZ, L.; PUNZON, I.; HEINLEIN, M.; NURNBERGER, T.; LLAVE, C.\nScore: 16.6, Published: 2023-06-23 DOI: 10.1101/2023.06.23.546234\nO_LIThe receptor-like kinase BIR1 functions as a negative regulator of cell death and defense in Arabidopsis. Previous studies showed that BIR1 expression is up-regulated during infections with microbes and viruses.",
  "keywords": [
    
  ],
  "articleBody": " The receptor-like kinase BIR1 inhibits elicitor-induced plasmodesmata callose deposition and PTI gene expression and requires EDS1 and SOBIR1 to cause dose-dependent cell-death in Arabidopsis\nAuthors: GUZMAN-BENITO, I.; ROBINSON, C.; HUA, C.; SEDE, A. R.; ELVIRA-GONZALEZ, L.; PUNZON, I.; HEINLEIN, M.; NURNBERGER, T.; LLAVE, C.\nScore: 16.6, Published: 2023-06-23 DOI: 10.1101/2023.06.23.546234\nO_LIThe receptor-like kinase BIR1 functions as a negative regulator of cell death and defense in Arabidopsis. Previous studies showed that BIR1 expression is up-regulated during infections with microbes and viruses. However, the biological consequences of BIR1 induction remain unknown. C_LIO_LIHere, we use a DEX-inducible expression system in Arabidopsis to investigate the outputs associated with physiological and non-physiological levels of BIR1 expression. C_LIO_LIWe show that BIR1 induction at physiological levels significantly interferes with gene expression and plasmodesmata callose deposition triggered by canonical PTI elicitors. We found that plants that accumulated non-physiological doses of BIR1 displayed morphological defects that concur with transcriptomic changes in multiple plant defense genes. We provide experimental evidence that EDS1 and SOBIR1 are required for the ETI-type cell death phenotypes associated to non-physiological levels of BIR1 C_LIO_LIHere we propose that BIR1 induction may represent a pathogen-triggered mechanism to modulate plant defenses during infection. Our model posits that when BIR1 regulation is lost, BIR1 integrity is sensed by one or several guarding resistance (R) proteins to initiate an ETI-like response, in which SOBIR1 cooperates with EDS1 to transduce signals downstream of R proteins. C_LI\nControl of stem cell behavior by CLE-JINGASA signaling in the shoot apical meristem in Marchantia polymorpha\nAuthors: Takahashi, G.; Kiyosue, T.; Hirakawa, Y.\nScore: 12.0, Published: 2023-06-23 DOI: 10.1101/2023.06.23.545176\nLand plants undergo indeterminate growth by the activity of meristems in both gametophyte (haploid) and sporophyte (diploid) generations1-3. In the sporophyte of the flowering plant Arabidopsis thaliana, the apical meristems are located at the shoot and root tips, in which a number of regulatory gene homologs are shared for their development, implying deep evolutionary origins4-7. However, little is known about their functional conservation with gametophytic meristems in distantly related land plants such as bryophytes8-17, even though genomic studies have revealed the subfamily-level diversity of regulatory genes is mostly conserved throughout land plants18-23. Here we show that a NAM/ATAF/CUC (NAC) domain transcription factor, JINGASA (MpJIN), acts downstream of CLV3/ESR-related (CLE) peptide signaling and controls stem cell behavior in the gametophytic shoot apical meristem of the liverwort Marchantia polymorpha. We found that MpJIN shows heterogeneous expression in space and time within the stem cell zone, associated with the loss and de novo specification of stem cell identity at the branching event. Consistent with the expression pattern, induction of MpJIN results in ectopic periclinal cell division in the stem cell zone and meristem termination. Comparative expression analysis suggests that the function of JIN/FEZ subfamily genes was shared between the shoot apical meristems in the gametophyte and sporophyte generations in early land plants but was lost in certain lineages including the flowering plant A. thaliana.\nPolyploidy enhances desiccation tolerance in the grass Microchloa caffra\nAuthors: Marks, R. A.; Delgado, P.; Makonya, G. M.; Cooper, K.; VanBuren, R.; Farrant, J. M.\nScore: 10.1, Published: 2023-06-22 DOI: 10.1101/2023.06.20.545583\nDesiccation tolerance evolved recurrently across diverse plant lineages to enable survival in water limited conditions. Many resurrection plants are polyploid and several groups have hypothesized that polyploidy enabled the evolution of desiccation tolerance. However, due to the vast evolutionary divergence between resurrection plant lineages, the rarity of desiccation tolerance, and the prevalence of polyploidy in plants, this hypothesis has been difficult to test. Here, we surveyed variation in morphological, reproductive, and desiccation tolerance traits across natural populations of a single species that has differing ploidies and tested for links between polyploidy and resilience. We sampled multiple populations of the resurrection grass Microchloa caffra across an environmental gradient ranging from mesic to xeric in South Africa. We describe two distinct ecotypes of M. caffra that occupy different ends of the environmental gradient and exhibit consistent differences in ploidy, morphological, reproductive, and desiccation tolerance traits in both field and common growth conditions. Interestingly, plants with more polyploid genomes were consistently more desiccation tolerant, less reproductive, and larger than plants with smaller genomes and lower ploidy. These data suggest that polyploidy enhances desiccation tolerance and that stronger selective pressures in increasingly xeric sites may play a role in maintaining and increasing desiccation tolerance.\nPlant Expression Omnibus - a comparative transcriptomic database for 103 Archaeplastida\nAuthors: Koh, E.; Goh, W.; Villanueva, E.; Julca, I.; Mutwil, M.\nScore: 31.4, Published: 2023-06-22 DOI: 10.1101/2023.06.22.546049\nThe Plant Expression Omnibus (PEO) is a web application that provides biologists with access to gene expression insights across over 100 plant species, [~]60,000 manually annotated RNA-seq samples, and more than four million genes. The tool allows users to explore the expression patterns of genes across different organs, identify organ-specific genes, and discover top co-expressed genes for any gene of interest. PEO also provides functional annotations for each gene, allowing for the identification of genetic modules and pathways. PEO is designed to facilitate comparative kingdom-wide gene expression analysis and provide a valuable resource for plant biology research. We provide two case studies to demonstrate the utility of PEO in identifying candidate genes in pollen coat biosynthesis and investigating the biosynthetic pathway components of capsaicin in Capsicum annuum. The database is freely available at https://expression.plant.tools/.\nFungal small RNAs ride in extracellular vesicles to enter plant cells through clathrin-mediated endocytosis\nAuthors: He, B.; Wang, H.; Liu, G.; Chen, A.; Cai, Q.; Jin, H.\nScore: 20.9, Published: 2023-06-30 DOI: 10.1101/2023.06.15.545159\nSmall RNAs (sRNAs) of the fungal pathogen Botrytis cinerea can enter plant cells and hijack host Argonaute protein 1 (AGO1) to silence host immunity genes. However, the mechanism by which these fungal sRNAs are secreted and enter host cells remains unclear. Here, we demonstrate that B. cinerea utilizes extracellular vesicles (EVs) to secrete Bc-sRNAs, which are then internalized by plant cells through clathrin-mediated endocytosis (CME). The B. cinerea tetraspanin protein, Punchless 1 (BcPLS1), serves as an EV biomarker and plays an essential role in fungal pathogenicity. We observe numerous Arabidopsis clathrin-coated vesicles (CCVs) around B. cinerea infection sites and the colocalization of B. cinerea EV marker BcPLS1 and Arabidopsis CLATHRIN LIGHT CHAIN 1, one of the core components of CCV. Meanwhile, BcPLS1 and the B. cinerea-secreted sRNAs are detected in purified CCVs after infection. Arabidopsis knockout mutants and inducible dominant-negative mutants of key components of CME pathway exhibit increased resistance to B. cinerea infection. Furthermore, Bc-sRNA loading into Arabidopsis AGO1 and host target gene suppression are attenuated in those CME mutants. Together, our results demonstrate that fungi secrete sRNAs via EVs, which then enter host plant cells mainly through CME.\nImplementing a framework of carbon and nitrogen feedback responses into a plant resource allocation model\nAuthors: Holland, B. L.; Monk, N. A. M.; Clayton, R. H.; Osborne, C. P.\nScore: 3.7, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546727\nThe allocation of resources to roots and shoots can greatly alter total plant mass. Allocation is thought to be the consequence of growth processes (i.e uptake rates, transport rates, growth rates) and the communication between them via signalling mechanisms. Feedbacks that alter growth processes are induced in nature by changes in the internal pools of carbon and nitrogen, but how these function together to define allocation remains unclear. We introduce a framework model of internal feedback responses to changes in plant carbon and nitrogen concentrations. We evaluate how well the model responds to changes in carbon and nitrogen availability by simulating external environmental perturbations that influence the uptake of resources. The model reflects experimental results when looking at the effect of atmospheric CO2 and soil nitrogen concentrations on total plant mass and replicates observed responses to leaf defoliation events. Overall this shows that a combination of known signalling mechanisms are sufficient to reproduce experimentally observed responses to external resource availability. Model simulations highlight key areas of uncertainty where more empirical data is needed. In particular, quantitative data is needed to establish the strengths and rates at which feedback responses to carbon and nitrogen substrate concentrations alter growth and uptake rates.\nThe molecular basis of parental conflict driven regulation of endosperm cellularization\nAuthors: Kohler, C.; Butel, N.; Qiu, Y.; Xu, W.; Santos-Gonzalez, J.\nScore: 16.9, Published: 2023-06-22 DOI: 10.1101/2023.06.22.546051\nThe endosperm is a seed tissue supporting embryo growth, similar to the placenta in mammals. It originates after fertilization of the maternal central cell by one of the paternal sperm cells. In the early stages of Arabidopsis thaliana seed development, nuclei divisions in the endosperm are not followed by cellularization. After a defined number of mitotic cycles, the endosperm cellularizes and stops dividing. The timing of endosperm cellularization impacts on final seed size and is differentially controlled by maternal and paternal genome contributions. While increased maternal genome dosage causes early endosperm cellularization and the formation of small seeds, the opposite is caused by increased paternal genome dosage. The parental factors controlling the differential timing of endosperm cellularization remain largely unexplored. Here, we show that a family of maternally expressed auxin response factors (ARFs) promotes endosperm cellularization and regulates final seed size. One-Sentence SummaryEndosperm cellularization is under antagonistic parental control that converges on Auxin Response Factors.\nGenomic characterization of a nematode tolerance locus in sugar beet\nAuthors: Sielemann, K.; Pucker, B.; Orsini, E.; Elashry, A.; Schulte, L.; Viehoever, P.; Mueller, A. E.; Schechert, A.; Weisshaar, B.; Holtgraewe, D.\nScore: 13.2, Published: 2023-06-22 DOI: 10.1101/2023.06.22.546034\nInfection by beet cyst nematodes (BCN, Heterodera schachtii) causes a serious disease of sugar beet, and climatic change is expected to improve the conditions for BCN infection. Yield and yield stability under adverse conditions are among the main breeding objectives. Breeding of BCN tolerant sugar beet cultivars offering high yield in the presence of the pathogen is therefore of high relevance. To identify causal genes providing tolerance against BCN infection, we combined several experimental and bioinformatic approaches. Relevant genomic regions were detected through mapping-by-sequencing using a segregating F2 population. DNA sequencing of contrasting F2 pools and analyses of allele frequencies for variant positions identified a single genomic region which confers nematode tolerance. The genomic interval was confirmed and narrowed down by genotyping with newly developed molecular markers. To pinpoint the causal genes within the nematode tolerance locus, we generated long read-based genome sequence assemblies of the tolerant parental breeding line Strube U2Bv and the susceptible reference line 2320Bv. We analyzed continuous sequences of the locus with regard to functional gene annotation and differential gene expression upon BCN infection. A cluster of genes with similarity to the Arabidopsis thaliana gene encoding nodule inception protein-like protein 7 (NLP7) was identified. Gene expression analyses confirmed transcriptional activity and revealed clear differences between susceptible and tolerant genotypes. Our findings provide new insights into the genomic basis of plant-nematode interactions that can be used to design and accelerate novel management strategies against BCN.\nMultifactorial analysis of terminator performance on heterologous gene expression in Physcomitrella\nAuthors: Niederau, P. A.; Egle, P.; Willig, S.; Parsons, J.; Hoernstein, S. N. W.; Decker, E. L.; Reski, R.\nScore: 2.8, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547182\nThe production of recombinant proteins for health applications accounts for a large share of the biopharmaceutical market. While many drugs are produced in microbial and mammalian systems, plants gain more attention as expression hosts to produce eukaryotic proteins. In particular the GMP-compliant moss Physcomitrella (Physcomitrium patens) has outstanding features such as excellent genetic amenability, reproducible bioreactor cultivation, and humanized protein glycosylation patterns. In this study, we selected and characterized new terminators for their effects on heterologous gene expression. The Physcomitrella genome contains 53,346 unique 3 prime UTRs (untranslated regions) of which 7,964 transcripts contain at least one intron. Over 91% of 3 prime UTRs exhibit more than one polyadenylation site, indicating the prevalence of alternative polyadenylation in Physcomitrella. Out of all 3 prime UTRs, 14 terminator candidates were selected and characterized via transient Dual Luciferase assays, yielding a collection of endogenous terminators performing equally high as established heterologous terminators CaMV35S, AtHSP90, and NOS. High performing candidates were selected for testing as double terminators which impact reporter levels, dependent on terminator identity and positioning. Testing of 3 prime UTRs among the different promoters NOS, CaMV35S, and PpActin5 showed an increase of more than 1,000-fold between promoters PpActin5 and NOS, whereas terminators increased reporter levels by less than 10-fold, demonstrating the stronger effect promoters play as compared to terminators. The number of polyadenylation sites as well as polyadenylation signals were found to be major determinants of terminator performance. Our results improve the biotechnology platform Physcomitrella and further our understanding of how terminators influence gene expression in plants in general.\nThe long non-coding RNA LINDA restrains cellular collapse following DNA damage in Arabidopsis thaliana\nAuthors: Herbst, J.; Nagy, S. H.; Vercauteren, I.; De Veylder, L.; Kunze, R.\nScore: 2.1, Published: 2023-06-29 DOI: 10.1101/2023.06.28.546876\nThe genomic integrity of every organism is endangered by various intrinsic and extrinsic stresses. To maintain the genomic integrity, a sophisticated DNA damage response (DDR) network is activated rapidly after DNA damage. Notably, the fundamental DDR mechanisms are conserved in eukaryotes. However, knowledge about many regulatory aspects of the plant DDR is still limited. Important, yet little understood, regulatory factors of the DDR are the long non-coding RNAs (lncRNAs). In humans, 13 lncRNAs functioning in DDR have been characterized to date, whereas no such lncRNAs have been characterized in plants yet. By meta-analysis, we identified the long intergenic non-coding RNA induced by DNA damage (LINDA) that responds strongly to various DNA double-strand break-inducing treatments, but not to replication stress induced by mitomycin C. After DNA damage, LINDA is rapidly induced in an ATM- and SOG1-dependent manner. Intriguingly, the transcriptional response of LINDA to DNA damage is similar to that of its flanking hypothetical protein-encoding gene. Phylogenetic analysis of putative Brassicales and Malvales LINDA homologs indicates that LINDA lncRNAs originate from duplication of a flanking small protein-encoding gene followed by pseudogenization. We demonstrate that LINDA is not only needed for the regulation of this flanking gene, but also for fine-tuning of the DDR after the occurrence of DNA double-strand breaks. Moreover, {Delta}linda mutant root stem cells are unable to recover from DNA damage, most likely due to hyper-induced cell death.\n",
  "wordCount" : "2350",
  "inLanguage": "en",
  "datePublished": "2023-07-02T10:39:32Z",
  "dateModified": "2023-07-02T10:39:32Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta"><span>updated on July 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546234">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546234" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546234">
        <p class="paperTitle">The receptor-like kinase BIR1 inhibits elicitor-induced plasmodesmata callose deposition and PTI gene expression and requires EDS1 and SOBIR1 to cause dose-dependent cell-death in Arabidopsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546234" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546234" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: GUZMAN-BENITO, I.; ROBINSON, C.; HUA, C.; SEDE, A. R.; ELVIRA-GONZALEZ, L.; PUNZON, I.; HEINLEIN, M.; NURNBERGER, T.; LLAVE, C.</p>
        <p class="info">Score: 16.6, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546234' target='https://doi.org/10.1101/2023.06.23.546234'> 10.1101/2023.06.23.546234</a></p>
        <p class="abstract">O_LIThe receptor-like kinase BIR1 functions as a negative regulator of cell death and defense in Arabidopsis. Previous studies showed that BIR1 expression is up-regulated during infections with microbes and viruses. However, the biological consequences of BIR1 induction remain unknown.
C_LIO_LIHere, we use a DEX-inducible expression system in Arabidopsis to investigate the outputs associated with physiological and non-physiological levels of BIR1 expression.
C_LIO_LIWe show that BIR1 induction at physiological levels significantly interferes with gene expression and plasmodesmata callose deposition triggered by canonical PTI elicitors. We found that plants that accumulated non-physiological doses of BIR1 displayed morphological defects that concur with transcriptomic changes in multiple plant defense genes. We provide experimental evidence that EDS1 and SOBIR1 are required for the ETI-type cell death phenotypes associated to non-physiological levels of BIR1
C_LIO_LIHere we propose that BIR1 induction may represent a pathogen-triggered mechanism to modulate plant defenses during infection. Our model posits that when BIR1 regulation is lost, BIR1 integrity is sensed by one or several guarding resistance (R) proteins to initiate an ETI-like response, in which SOBIR1 cooperates with EDS1 to transduce signals downstream of R proteins.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.545176">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.545176" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.545176">
        <p class="paperTitle">Control of stem cell behavior by CLE-JINGASA signaling in the shoot apical meristem in Marchantia polymorpha</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.545176" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.545176" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Takahashi, G.; Kiyosue, T.; Hirakawa, Y.</p>
        <p class="info">Score: 12.0, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.545176' target='https://doi.org/10.1101/2023.06.23.545176'> 10.1101/2023.06.23.545176</a></p>
        <p class="abstract">Land plants undergo indeterminate growth by the activity of meristems in both gametophyte (haploid) and sporophyte (diploid) generations1-3. In the sporophyte of the flowering plant Arabidopsis thaliana, the apical meristems are located at the shoot and root tips, in which a number of regulatory gene homologs are shared for their development, implying deep evolutionary origins4-7. However, little is known about their functional conservation with gametophytic meristems in distantly related land plants such as bryophytes8-17, even though genomic studies have revealed the subfamily-level diversity of regulatory genes is mostly conserved throughout land plants18-23. Here we show that a NAM/ATAF/CUC (NAC) domain transcription factor, JINGASA (MpJIN), acts downstream of CLV3/ESR-related (CLE) peptide signaling and controls stem cell behavior in the gametophytic shoot apical meristem of the liverwort Marchantia polymorpha. We found that MpJIN shows heterogeneous expression in space and time within the stem cell zone, associated with the loss and de novo specification of stem cell identity at the branching event. Consistent with the expression pattern, induction of MpJIN results in ectopic periclinal cell division in the stem cell zone and meristem termination. Comparative expression analysis suggests that the function of JIN/FEZ subfamily genes was shared between the shoot apical meristems in the gametophyte and sporophyte generations in early land plants but was lost in certain lineages including the flowering plant A. thaliana.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545583">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545583" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545583">
        <p class="paperTitle">Polyploidy enhances desiccation tolerance in the grass Microchloa caffra</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545583" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545583" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Marks, R. A.; Delgado, P.; Makonya, G. M.; Cooper, K.; VanBuren, R.; Farrant, J. M.</p>
        <p class="info">Score: 10.1, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545583' target='https://doi.org/10.1101/2023.06.20.545583'> 10.1101/2023.06.20.545583</a></p>
        <p class="abstract">Desiccation tolerance evolved recurrently across diverse plant lineages to enable survival in water limited conditions. Many resurrection plants are polyploid and several groups have hypothesized that polyploidy enabled the evolution of desiccation tolerance. However, due to the vast evolutionary divergence between resurrection plant lineages, the rarity of desiccation tolerance, and the prevalence of polyploidy in plants, this hypothesis has been difficult to test. Here, we surveyed variation in morphological, reproductive, and desiccation tolerance traits across natural populations of a single species that has differing ploidies and tested for links between polyploidy and resilience. We sampled multiple populations of the resurrection grass Microchloa caffra across an environmental gradient ranging from mesic to xeric in South Africa. We describe two distinct ecotypes of M. caffra that occupy different ends of the environmental gradient and exhibit consistent differences in ploidy, morphological, reproductive, and desiccation tolerance traits in both field and common growth conditions. Interestingly, plants with more polyploid genomes were consistently more desiccation tolerant, less reproductive, and larger than plants with smaller genomes and lower ploidy. These data suggest that polyploidy enhances desiccation tolerance and that stronger selective pressures in increasingly xeric sites may play a role in maintaining and increasing desiccation tolerance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546049">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546049" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546049">
        <p class="paperTitle">Plant Expression Omnibus - a comparative transcriptomic database for 103 Archaeplastida</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546049" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546049" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koh, E.; Goh, W.; Villanueva, E.; Julca, I.; Mutwil, M.</p>
        <p class="info">Score: 31.4, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546049' target='https://doi.org/10.1101/2023.06.22.546049'> 10.1101/2023.06.22.546049</a></p>
        <p class="abstract">The Plant Expression Omnibus (PEO) is a web application that provides biologists with access to gene expression insights across over 100 plant species, [~]60,000 manually annotated RNA-seq samples, and more than four million genes. The tool allows users to explore the expression patterns of genes across different organs, identify organ-specific genes, and discover top co-expressed genes for any gene of interest. PEO also provides functional annotations for each gene, allowing for the identification of genetic modules and pathways. PEO is designed to facilitate comparative kingdom-wide gene expression analysis and provide a valuable resource for plant biology research. We provide two case studies to demonstrate the utility of PEO in identifying candidate genes in pollen coat biosynthesis and investigating the biosynthetic pathway components of capsaicin in Capsicum annuum. The database is freely available at https://expression.plant.tools/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.15.545159">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.15.545159" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.15.545159">
        <p class="paperTitle">Fungal small RNAs ride in extracellular vesicles to enter plant cells through clathrin-mediated endocytosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.15.545159" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.15.545159" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: He, B.; Wang, H.; Liu, G.; Chen, A.; Cai, Q.; Jin, H.</p>
        <p class="info">Score: 20.9, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.15.545159' target='https://doi.org/10.1101/2023.06.15.545159'> 10.1101/2023.06.15.545159</a></p>
        <p class="abstract">Small RNAs (sRNAs) of the fungal pathogen Botrytis cinerea can enter plant cells and hijack host Argonaute protein 1 (AGO1) to silence host immunity genes. However, the mechanism by which these fungal sRNAs are secreted and enter host cells remains unclear. Here, we demonstrate that B. cinerea utilizes extracellular vesicles (EVs) to secrete Bc-sRNAs, which are then internalized by plant cells through clathrin-mediated endocytosis (CME). The B. cinerea tetraspanin protein, Punchless 1 (BcPLS1), serves as an EV biomarker and plays an essential role in fungal pathogenicity. We observe numerous Arabidopsis clathrin-coated vesicles (CCVs) around B. cinerea infection sites and the colocalization of B. cinerea EV marker BcPLS1 and Arabidopsis CLATHRIN LIGHT CHAIN 1, one of the core components of CCV. Meanwhile, BcPLS1 and the B. cinerea-secreted sRNAs are detected in purified CCVs after infection. Arabidopsis knockout mutants and inducible dominant-negative mutants of key components of CME pathway exhibit increased resistance to B. cinerea infection. Furthermore, Bc-sRNA loading into Arabidopsis AGO1 and host target gene suppression are attenuated in those CME mutants. Together, our results demonstrate that fungi secrete sRNAs via EVs, which then enter host plant cells mainly through CME.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546727">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546727" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546727">
        <p class="paperTitle">Implementing a framework of carbon and nitrogen feedback responses into a plant resource allocation model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546727" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546727" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Holland, B. L.; Monk, N. A. M.; Clayton, R. H.; Osborne, C. P.</p>
        <p class="info">Score: 3.7, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546727' target='https://doi.org/10.1101/2023.06.27.546727'> 10.1101/2023.06.27.546727</a></p>
        <p class="abstract">The allocation of resources to roots and shoots can greatly alter total plant mass. Allocation is thought to be the consequence of growth processes (i.e uptake rates, transport rates, growth rates) and the communication between them via signalling mechanisms. Feedbacks that alter growth processes are induced in nature by changes in the internal pools of carbon and nitrogen, but how these function together to define allocation remains unclear. We introduce a framework model of internal feedback responses to changes in plant carbon and nitrogen concentrations. We evaluate how well the model responds to changes in carbon and nitrogen availability by simulating external environmental perturbations that influence the uptake of resources. The model reflects experimental results when looking at the effect of atmospheric CO2 and soil nitrogen concentrations on total plant mass and replicates observed responses to leaf defoliation events. Overall this shows that a combination of known signalling mechanisms are sufficient to reproduce experimentally observed responses to external resource availability. Model simulations highlight key areas of uncertainty where more empirical data is needed. In particular, quantitative data is needed to establish the strengths and rates at which feedback responses to carbon and nitrogen substrate concentrations alter growth and uptake rates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546051">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546051" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546051">
        <p class="paperTitle">The molecular basis of parental conflict driven regulation of endosperm cellularization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546051" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546051" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kohler, C.; Butel, N.; Qiu, Y.; Xu, W.; Santos-Gonzalez, J.</p>
        <p class="info">Score: 16.9, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546051' target='https://doi.org/10.1101/2023.06.22.546051'> 10.1101/2023.06.22.546051</a></p>
        <p class="abstract">The endosperm is a seed tissue supporting embryo growth, similar to the placenta in mammals. It originates after fertilization of the maternal central cell by one of the paternal sperm cells. In the early stages of Arabidopsis thaliana seed development, nuclei divisions in the endosperm are not followed by cellularization. After a defined number of mitotic cycles, the endosperm cellularizes and stops dividing. The timing of endosperm cellularization impacts on final seed size and is differentially controlled by maternal and paternal genome contributions. While increased maternal genome dosage causes early endosperm cellularization and the formation of small seeds, the opposite is caused by increased paternal genome dosage. The parental factors controlling the differential timing of endosperm cellularization remain largely unexplored. Here, we show that a family of maternally expressed auxin response factors (ARFs) promotes endosperm cellularization and regulates final seed size.

One-Sentence SummaryEndosperm cellularization is under antagonistic parental control that converges on Auxin Response Factors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546034">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546034" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546034">
        <p class="paperTitle">Genomic characterization of a nematode tolerance locus in sugar beet</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546034" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546034" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sielemann, K.; Pucker, B.; Orsini, E.; Elashry, A.; Schulte, L.; Viehoever, P.; Mueller, A. E.; Schechert, A.; Weisshaar, B.; Holtgraewe, D.</p>
        <p class="info">Score: 13.2, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546034' target='https://doi.org/10.1101/2023.06.22.546034'> 10.1101/2023.06.22.546034</a></p>
        <p class="abstract">Infection by beet cyst nematodes (BCN, Heterodera schachtii) causes a serious disease of sugar beet, and climatic change is expected to improve the conditions for BCN infection. Yield and yield stability under adverse conditions are among the main breeding objectives. Breeding of BCN tolerant sugar beet cultivars offering high yield in the presence of the pathogen is therefore of high relevance. To identify causal genes providing tolerance against BCN infection, we combined several experimental and bioinformatic approaches. Relevant genomic regions were detected through mapping-by-sequencing using a segregating F2 population. DNA sequencing of contrasting F2 pools and analyses of allele frequencies for variant positions identified a single genomic region which confers nematode tolerance. The genomic interval was confirmed and narrowed down by genotyping with newly developed molecular markers. To pinpoint the causal genes within the nematode tolerance locus, we generated long read-based genome sequence assemblies of the tolerant parental breeding line Strube U2Bv and the susceptible reference line 2320Bv. We analyzed continuous sequences of the locus with regard to functional gene annotation and differential gene expression upon BCN infection. A cluster of genes with similarity to the Arabidopsis thaliana gene encoding nodule inception protein-like protein 7 (NLP7) was identified. Gene expression analyses confirmed transcriptional activity and revealed clear differences between susceptible and tolerant genotypes. Our findings provide new insights into the genomic basis of plant-nematode interactions that can be used to design and accelerate novel management strategies against BCN.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547182">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547182" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547182">
        <p class="paperTitle">Multifactorial analysis of terminator performance on heterologous gene expression in Physcomitrella</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547182" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547182" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Niederau, P. A.; Egle, P.; Willig, S.; Parsons, J.; Hoernstein, S. N. W.; Decker, E. L.; Reski, R.</p>
        <p class="info">Score: 2.8, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547182' target='https://doi.org/10.1101/2023.06.30.547182'> 10.1101/2023.06.30.547182</a></p>
        <p class="abstract">The production of recombinant proteins for health applications accounts for a large share of the biopharmaceutical market. While many drugs are produced in microbial and mammalian systems, plants gain more attention as expression hosts to produce eukaryotic proteins. In particular the GMP-compliant moss Physcomitrella (Physcomitrium patens) has outstanding features such as excellent genetic amenability, reproducible bioreactor cultivation, and humanized protein glycosylation patterns. In this study, we selected and characterized new terminators for their effects on heterologous gene expression. The Physcomitrella genome contains 53,346 unique 3 prime UTRs (untranslated regions) of which 7,964 transcripts contain at least one intron. Over 91% of 3 prime UTRs exhibit more than one polyadenylation site, indicating the prevalence of alternative polyadenylation in Physcomitrella. Out of all 3 prime UTRs, 14 terminator candidates were selected and characterized via transient Dual Luciferase assays, yielding a collection of endogenous terminators performing equally high as established heterologous terminators CaMV35S, AtHSP90, and NOS. High performing candidates were selected for testing as double terminators which impact reporter levels, dependent on terminator identity and positioning. Testing of 3 prime UTRs among the different promoters NOS, CaMV35S, and PpActin5 showed an increase of more than 1,000-fold between promoters PpActin5 and NOS, whereas terminators increased reporter levels by less than 10-fold, demonstrating the stronger effect promoters play as compared to terminators. The number of polyadenylation sites as well as polyadenylation signals were found to be major determinants of terminator performance. Our results improve the biotechnology platform Physcomitrella and further our understanding of how terminators influence gene expression in plants in general.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546876">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546876" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546876">
        <p class="paperTitle">The long non-coding RNA LINDA restrains cellular collapse following DNA damage in Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546876" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546876" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Herbst, J.; Nagy, S. H.; Vercauteren, I.; De Veylder, L.; Kunze, R.</p>
        <p class="info">Score: 2.1, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546876' target='https://doi.org/10.1101/2023.06.28.546876'> 10.1101/2023.06.28.546876</a></p>
        <p class="abstract">The genomic integrity of every organism is endangered by various intrinsic and extrinsic stresses. To maintain the genomic integrity, a sophisticated DNA damage response (DDR) network is activated rapidly after DNA damage. Notably, the fundamental DDR mechanisms are conserved in eukaryotes. However, knowledge about many regulatory aspects of the plant DDR is still limited. Important, yet little understood, regulatory factors of the DDR are the long non-coding RNAs (lncRNAs). In humans, 13 lncRNAs functioning in DDR have been characterized to date, whereas no such lncRNAs have been characterized in plants yet. By meta-analysis, we identified the long intergenic non-coding RNA induced by DNA damage (LINDA) that responds strongly to various DNA double-strand break-inducing treatments, but not to replication stress induced by mitomycin C. After DNA damage, LINDA is rapidly induced in an ATM- and SOG1-dependent manner. Intriguingly, the transcriptional response of LINDA to DNA damage is similar to that of its flanking hypothetical protein-encoding gene. Phylogenetic analysis of putative Brassicales and Malvales LINDA homologs indicates that LINDA lncRNAs originate from duplication of a flanking small protein-encoding gene followed by pseudogenization. We demonstrate that LINDA is not only needed for the regulation of this flanking gene, but also for fine-tuning of the DDR after the occurrence of DNA double-strand breaks. Moreover, {Delta}linda mutant root stem cells are unable to recover from DNA damage, most likely due to hyper-induced cell death.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
