<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Plant pathogenic fungi hijack phosphate starvation signaling with conserved enzymatic effectors
Authors: McCombe, C. L.; Wegner, A.; Zamora, C. S.; Casanova, F.; Aditya, S.; Greenwood, J. R.; Wirtz, L.; de Paula, S.; England, E.; Shang, S.; Ericsson, D. J.; Oliveira-Garcia, E.; Williams, S. J.; Schaffrath, U.
Score: 25.5, Published: 2023-11-14 DOI: 10.1101/2023.11.14.566975
Phosphate availability modulates plant immune function and regulates interactions with beneficial, phosphate-providing, microbes. Here, we describe the hijacking of plant phosphate sensing by a family of Nudix hydrolase effectors from pathogenic Magnaporthe oryzae and Colletotrichum fungi.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="Plant pathogenic fungi hijack phosphate starvation signaling with conserved enzymatic effectors
Authors: McCombe, C. L.; Wegner, A.; Zamora, C. S.; Casanova, F.; Aditya, S.; Greenwood, J. R.; Wirtz, L.; de Paula, S.; England, E.; Shang, S.; Ericsson, D. J.; Oliveira-Garcia, E.; Williams, S. J.; Schaffrath, U.
Score: 25.5, Published: 2023-11-14 DOI: 10.1101/2023.11.14.566975
Phosphate availability modulates plant immune function and regulates interactions with beneficial, phosphate-providing, microbes. Here, we describe the hijacking of plant phosphate sensing by a family of Nudix hydrolase effectors from pathogenic Magnaporthe oryzae and Colletotrichum fungi." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-19T10:38:34+00:00" />
<meta property="article:modified_time" content="2023-11-19T10:38:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="Plant pathogenic fungi hijack phosphate starvation signaling with conserved enzymatic effectors
Authors: McCombe, C. L.; Wegner, A.; Zamora, C. S.; Casanova, F.; Aditya, S.; Greenwood, J. R.; Wirtz, L.; de Paula, S.; England, E.; Shang, S.; Ericsson, D. J.; Oliveira-Garcia, E.; Williams, S. J.; Schaffrath, U.
Score: 25.5, Published: 2023-11-14 DOI: 10.1101/2023.11.14.566975
Phosphate availability modulates plant immune function and regulates interactions with beneficial, phosphate-providing, microbes. Here, we describe the hijacking of plant phosphate sensing by a family of Nudix hydrolase effectors from pathogenic Magnaporthe oryzae and Colletotrichum fungi."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "Plant pathogenic fungi hijack phosphate starvation signaling with conserved enzymatic effectors\nAuthors: McCombe, C. L.; Wegner, A.; Zamora, C. S.; Casanova, F.; Aditya, S.; Greenwood, J. R.; Wirtz, L.; de Paula, S.; England, E.; Shang, S.; Ericsson, D. J.; Oliveira-Garcia, E.; Williams, S. J.; Schaffrath, U.\nScore: 25.5, Published: 2023-11-14 DOI: 10.1101/2023.11.14.566975\nPhosphate availability modulates plant immune function and regulates interactions with beneficial, phosphate-providing, microbes. Here, we describe the hijacking of plant phosphate sensing by a family of Nudix hydrolase effectors from pathogenic Magnaporthe oryzae and Colletotrichum fungi.",
  "keywords": [
    
  ],
  "articleBody": " Plant pathogenic fungi hijack phosphate starvation signaling with conserved enzymatic effectors\nAuthors: McCombe, C. L.; Wegner, A.; Zamora, C. S.; Casanova, F.; Aditya, S.; Greenwood, J. R.; Wirtz, L.; de Paula, S.; England, E.; Shang, S.; Ericsson, D. J.; Oliveira-Garcia, E.; Williams, S. J.; Schaffrath, U.\nScore: 25.5, Published: 2023-11-14 DOI: 10.1101/2023.11.14.566975\nPhosphate availability modulates plant immune function and regulates interactions with beneficial, phosphate-providing, microbes. Here, we describe the hijacking of plant phosphate sensing by a family of Nudix hydrolase effectors from pathogenic Magnaporthe oryzae and Colletotrichum fungi. Structural and enzymatic analyses of the Nudix effector family demonstrate that they selectively hydrolyze inositol pyrophosphates, a molecule used by plants to monitor phosphate status and regulate starvation responses. In M. oryzae, gene deletion and complementation experiments reveal that the enzymatic activity of a Nudix effector significantly contributes to pathogen virulence. Further, we show that this conserved effector family induces phosphate starvation signaling in plants. Our study elucidates a molecular mechanism, utilized by multiple phytopathogenic fungi, that manipulates the highly conserved plant phosphate sensing pathway to exacerbate disease. One-Sentence SummaryA family of conserved enzyme effectors from pathogenic fungi manipulate plant phosphate sensing to promote infection.\nPhotobodies enable the phase-separation and counterbalance of opposing phytochrome B actions in PIF5 degradation and stabilization\nAuthors: Kim, R. J. A.; Fan, D.; He, J.; Kim, K.; Du, J.; Chen, M.\nScore: 14.8, Published: 2023-11-15 DOI: 10.1101/2023.11.12.566724\nPhotoactivation of the plant photoreceptor and thermosensor phytochrome B (PHYB) triggers its condensation into subnuclear photobodies (PBs). However, the function of PBs remains frustratingly elusive. Here, we show that PHYB condensation enables the co-occurrence and competition of two antagonistic phase-separated signaling actions. We found that PHYB recruits PHYTOCHROME-INTERACTING FACTOR5 (PIF5) to PBs and, surprisingly, that PHYB exerts opposing roles in degrading and stabilizing PIF5. Perturbing PB size by overproducing PHYB provoked a biphasic PIF5 response: while a moderate increase in PHYB enhanced PIF5 degradation, further elevating the PHYB level stabilized PIF5 by retaining more of it in enlarged PBs. Our results support a model in which PHYB condensation stabilizes PIF5 in PBs to counteract PIF5 degradation in the surrounding nucleoplasm, thereby enabling an environmentally sensitive counterbalancing mechanism to titrate nucleoplasmic PIF5 and its transcriptional output. This PB-enabled signaling mechanism provides a framework for regulating a plethora of PHYB-interacting signaling molecules in diverse plant environmental responses. We propose that this function of PBs represents a general function of biomolecular condensates to allow distinct variations of a cellular process or signaling pathway to coexist and interact to generate dynamically adjustable integrated outputs within a single subcellular space.\nAn Arabidopsis leaf expression atlas across diurnal and developmental scales\nAuthors: Vong, G.; McCarthy, K.; Claydon, W.; Davis, S. J.; Redmond, E. J.; Ezer, D.\nScore: 12.9, Published: 2023-11-13 DOI: 10.1101/2023.11.10.566572\nMature plant leaves are a composite of distinct cell types, including epidermal, mesophyll and vascular cells. Notably the proportion of these cells, and the relative transcript concentrations within different cell types may change over time. While gene expression data at a single-cell level can provide cell-type specific expression values, it is often too expensive to perform this on high resolution time series. Although bulk RNA-seq can be performed in a high resolution time series, the RNA-seq in whole leaves measures the average gene expression values across all cell types in each sample. In this study, we combined single cell RNA-seq data with time-series data from whole leaves to infer an atlas of cell type-specific gene expression changes over time for Arabidopsis thaliana. We inferred how relative transcript concentrations of cell types vary across diurnal and developmental time scales. Importantly this analysis revealed three sub-groups of mesophyll cells that have distinct temporal profiles of expression. Finally, we develop tissue-specific gene networks that form a new community resource: An Arabidopsis Leaf Time-Dependent Atlas (AraLeTa), which allows users to extract gene networks that are confirmed by transcription factor binding data and specific to certain cell types, at certain times of day and certain developmental stages, which is available at: https://regulatorynet.shinyapps.io/araleta/.\nThe Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment\nAuthors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium, ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J. H.; Schmutz, J.; Webber, J.; Grimwood, J.; Jacquet, C.; Dunand, C.; Nelson, J.; Roux, F.; Philippe, H.; Schornack, S.; Bonhomme, M.; Delaux, P.-M.\nScore: 66.3, Published: 2023-10-29 DOI: 10.1101/2023.10.27.564390\nPlant adaptation to a terrestrial life 450 million years ago played a major role in the evolution of life on Earth. This shift from an aquatic environment has been mostly studied by focusing on flowering plants. Here, we gathered a collection of 133 accessions of the non-vascular plants Marchantia polymorpha and studied its intraspecific diversity using selection signature analyses, genome-environment association study and a gene-centered pangenome. We identified adaptive features shared with flowering plants, such as peroxidases or nucleotide-binding and leucine-rich repeat (NLR), which likely played a role in the adaptation of the first land plants to the terrestrial habitat. The M. polymorpha pangenome also harbored lineage-specific accessory genes absent from seed plants. We conclude that different land plants lineages still share many elements from the genetic toolkit evolved by their most recent common ancestor to adapt to the terrestrial habitat, refined by lineage specific polymorphisms and gene family evolutions.\nConservation of heat stress acclimation by the inositol polyphosphate multikinase, IPMK responsible for 4/6-InsP7 production in land plants\nAuthors: Yadav, R.; Liu, G.; Rana, P.; Pullagurla, N. J.; Qiu, D.; Jessen, H. J.; Laha, D.\nScore: 6.3, Published: 2023-11-18 DOI: 10.1101/2023.11.17.567642\nInositol pyrophosphates (PP-InsPs) are soluble cellular messengers that integrate environmental cues to induce adaptive responses in eukaryotes. In plants, the biological functions of various PP-InsP species are poorly understood, largely due to the absence of canonical enzymes present in other eukaryotes. The recent identification of a new PP-InsP isomer with yet unknown enantiomeric identity, 4/6-InsP7 in the eudicot Arabidopsis thaliana, further highlights the intricate PP-InsP signalling network employed by plants. The abundance of 4/6-InsP7 in land plants, the enzyme(s) responsible for its synthesis, and the physiological functions of this species are all currently unknown. In this study, we show that 4/6-InsP7 is the major PP-InsP species present across land plants. Our findings demonstrate that the Arabidopsis inositol polyphosphate multikinase (IPMK) homolog, AtIPK2 generates 4/6-InsP7 in vitro. Furthermore, the cellular level of 4/6-InsP7 is controlled by the two Arabidopsis IPMK isoforms, AtIPK2 and AtIPK2{beta}. Notably, the activity of these IPMK proteins is critical for heat stress acclimation in Arabidopsis. During heat stress, the expression of genes encoding various heat shock proteins controlled by the heat shock factors (HSFs) is affected in the AtIPK2-deficient plants. Furthermore, we show that the transcription activity of HSF is regulated by the AtIPK2 proteins. Our parallel investigations using the liverwort Marchantia polymorpha suggest that the InsP6 kinase activity of IPMK and the role of IPMK in regulating the heat stress response are evolutionarily conserved. Collectively, our study indicates that IPMK has played a critical role in transducing environmental cues for biological processes during land plant evolution.\nPseudomonas syringae pv. tomato DC3000 induces defense responses in diverse maize inbred lines\nAuthors: Jaiswal, N.; Helm, M.\nScore: 6.0, Published: 2023-11-17 DOI: 10.1101/2023.11.17.567553\nMany phytopathogens translocate virulence (effector) proteins into plant cells to circumvent host immune responses during infection. One such pathogen is Pseudomonas syringae pv. tomato DC3000, which secretes at least twenty-nine effectors into host cells, of which a subset elicits host defense responses in crop plant species. However, it is unknown whether P. syringae pv. tomato DC3000 activates immune responses in diverse maize inbreds. Here, we screened a diverse maize germplasm collection for effector-dependent recognition of this bacterial pathogen. As a control, we infiltrated Pseudomonas syringae DC3000(D36E), a derivative of P. syringae pv. tomato DC3000 that lacks all endogenous effectors. In our evaluations, we observed a variety of responses to P. syringae pv. tomato DC3000 in maize and scored the phenotypes as either no observable response (N) or as one of three responses: weak chlorosis (WC), chlorosis (C) with minimal cell death, and hypersensitive reaction (HR)-like cell death. Of the twenty-six maize inbreds screened, 13 were scored as N, 2 as WC, 2 as C, and 9 as HR-like cell death. Importantly, no maize line responded to P. syringae DC3000(D36E), demonstrating the responses observed are likely dependent upon recognition of one or more Pseudomonas effectors. Importantly, maize inbreds that recognize P. syringae pv. tomato DC3000 accumulated detectable hydrogen peroxide as well as an increase in transcript expression of a subset of maize defense genes. Collectively, our results will likely stimulate new research aimed at identifying the cognate maize disease resistance proteins that recognize the activities of one or more bacterial effectors.\nDistinct mechanisms of plant immune resilience revealed by natural variation in warm temperature-modulated disease resistance among Arabidopsis accessions\nAuthors: Rossi, C. A. M.; Patel, D. N.; Castroverde, C. D. M.\nScore: 5.6, Published: 2023-11-17 DOI: 10.1101/2023.11.15.565111\nElevated temperature suppresses production of the key plant defence hormone salicylic acid (SA). Heat-mediated SA suppression and resulting plant vulnerability are due to downregulated expression of CALMODULIN BINDING PROTEIN 60-LIKE G (CBP60g) and SYSTEMIC ACQUIRED RESISTANCE DEFICIENT 1 (SARD1), which encode master regulators of plant immunity. However, previous studies in Arabidopsis thaliana plants have primarily focused on the accession Columbia-0 (Col-0), while the mechanisms governing the intraspecific variation in Arabidopsis immunity under elevated temperature has remained unknown. Here we show that BASIC HELIX LOOP HELIX 059 (bHLH059), a thermosensitive SA regulator at non-stress temperatures, does not regulate immune suppression under warmer temperatures. In agreement, temperature-resilient and -sensitive Arabidopsis accessions based on disease resistance to the bacterial pathogen Pseudomonas syringae pv. tomato (Pst) DC3000 did not correlate with bHLH059 sequence polymorphisms. Finally, we found that different temperature-resilient accessions exhibit varying CBP60g and SARD1 expression profiles, potentially revealing both CBP60g/SARD1-dependent and independent mechanisms of plant immune resilience to warming temperature. Collectively, this study has unveiled the intraspecific diversity of Arabidopsis immune responses under warm temperatures, which is differentially mediated by the master transcription factors CBP60g and SARD1. Dissecting the mechanisms underlying temperature-modulated plant immunity aid in predicting plant responses to climate change and provide foundational knowledge for climate-resilient crop engineering.\nComparative mutant analyses reveal a novel mechanism of ARF regulation in land plants\nAuthors: Prigge, M. J.; Morffy, N.; De Neve, A.; Szutu, W.; Juarez, M. J. A.; Johnson, K.; Do, N.; Lavy, M.; Hake, S.; Strader, L.; Estelle, M.; Richardson, A. E.\nScore: 5.5, Published: 2023-11-13 DOI: 10.1101/2023.11.09.566459\nA major challenge in plant biology is to understand how the plant hormone auxin regulates diverse transcriptional responses throughout development, in different environments, and in different species. The answer may lie in the specific complement of auxin signaling components in each cell. The balance between activators (class-A AUXIN RESPONSE FACTORS) and repressors (class-B ARFs) is particularly important. It is unclear how this balance is achieved. Through comparative analysis of novel, dominant mutants in maize and the moss Physcomitrium patens, we have discovered a [~]500-million-year-old mechanism of class-B ARF protein level regulation, important in determining cell fate decisions across land plants. Thus, our results add a key piece to the puzzle of how auxin regulates plant development.\nGenetic factors acting prior to dormancy in sour cherry influence bloom time the following spring\nAuthors: Goeckeritz, C. Z.; Grabb, C.; Grumet, R.; Iezzoni, A. F.; Hollender, C. A.\nScore: 5.1, Published: 2023-11-14 DOI: 10.1101/2023.11.09.566501\nBloom time is central to tree fruit production, and for Prunus species floral development leading up to bloom spans four seasons. Understanding this entire process is crucial for developing strategies to manipulate bloom time to prevent crop loss due to climate change. Here, we present a detailed examination of flower development from initiation until bloom for early- and late-blooming sour cherries (Prunus cerasus) from a population segregating for a major bloom time QTL on chromosome 4. Using a new staging system, we identified floral buds from early-blooming trees were persistently more advanced than those from late-blooming siblings. A gDNA coverage analysis revealed the late-blooming haplotype of this QTL, k, is located on a subgenome originating from the late-blooming P. fruticosa progenitor. Transcriptome analyses identified a large number of genes within this QTL as differentially expressed between early- and late-blooming trees during the vegetative-to-floral transition. From these, we identified candidate genes for the late bloom phenotype, including multiple transcription factors homologous to REproductive Meristem (REM) B3 domain-containing proteins. Additionally, we determined the basis of k in sour cherry is likely separate from candidate genes found in sweet cherry - suggesting several major regulators of bloom time are located on Prunus chromosome 4. HIGHLIGHTDormancy is a main effector of bloom time in fruit trees. However, developmental, genetic, and transcriptomic analyses indicate differences in flower development before dormancy significantly influence flowering time in cherry.\nGuanylate cyclase activity of TIR1/AFB auxin receptorsin rapid auxin responses\nAuthors: Qi, L.; Kwiatkowski, M.; Kulich, I.; Chen, H.; Gao, Y.; Yun, P.; Li, L.; Shabala, S.; Farmer, E.; Jaworski, K.; Friml, J.\nScore: 4.0, Published: 2023-11-18 DOI: 10.1101/2023.11.18.567481\nThe major developmental signal in plants, auxin is perceived by TIR1/AFB receptors. It triggers transcriptional reprogramming via well-established canonical mechanism but also elusive rapid, non-transcriptional responses. Here we demonstrate that TIR1/AFB receptors have, next to recently identified adenylate cyclase, also guanylate cyclase activity. Auxin perception activates independently the cAMP and cGMP production by TIR1/AFBs in vitro and increases cAMP and cGMP levels in planta with a slow and fast dynamics, respectively. Exogenous cGMP but not cAMP application induces rapid cytosolic Ca2+ transients and root growth inhibition, suggesting that TIR1/AFB-derived cGMP mediates rapid auxin responses. This unprecedented combination of adenylate and guanylate cyclase activities in a hormone receptor provides a new paradigm for how a single perception mechanism can mediate a multitude of diverse downstream responses.\n",
  "wordCount" : "2316",
  "inLanguage": "en",
  "datePublished": "2023-11-19T10:38:34Z",
  "dateModified": "2023-11-19T10:38:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 19, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.566975">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.566975" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.566975">
        <p class="paperTitle">Plant pathogenic fungi hijack phosphate starvation signaling with conserved enzymatic effectors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.566975" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.566975" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McCombe, C. L.; Wegner, A.; Zamora, C. S.; Casanova, F.; Aditya, S.; Greenwood, J. R.; Wirtz, L.; de Paula, S.; England, E.; Shang, S.; Ericsson, D. J.; Oliveira-Garcia, E.; Williams, S. J.; Schaffrath, U.</p>
        <p class="info">Score: 25.5, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.566975' target='https://doi.org/10.1101/2023.11.14.566975'> 10.1101/2023.11.14.566975</a></p>
        <p class="abstract">Phosphate availability modulates plant immune function and regulates interactions with beneficial, phosphate-providing, microbes. Here, we describe the hijacking of plant phosphate sensing by a family of Nudix hydrolase effectors from pathogenic Magnaporthe oryzae and Colletotrichum fungi. Structural and enzymatic analyses of the Nudix effector family demonstrate that they selectively hydrolyze inositol pyrophosphates, a molecule used by plants to monitor phosphate status and regulate starvation responses. In M. oryzae, gene deletion and complementation experiments reveal that the enzymatic activity of a Nudix effector significantly contributes to pathogen virulence. Further, we show that this conserved effector family induces phosphate starvation signaling in plants. Our study elucidates a molecular mechanism, utilized by multiple phytopathogenic fungi, that manipulates the highly conserved plant phosphate sensing pathway to exacerbate disease.

One-Sentence SummaryA family of conserved enzyme effectors from pathogenic fungi manipulate plant phosphate sensing to promote infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.12.566724">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.12.566724" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.12.566724">
        <p class="paperTitle">Photobodies enable the phase-separation and counterbalance of opposing phytochrome B actions in PIF5 degradation and stabilization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.12.566724" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.12.566724" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, R. J. A.; Fan, D.; He, J.; Kim, K.; Du, J.; Chen, M.</p>
        <p class="info">Score: 14.8, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.12.566724' target='https://doi.org/10.1101/2023.11.12.566724'> 10.1101/2023.11.12.566724</a></p>
        <p class="abstract">Photoactivation of the plant photoreceptor and thermosensor phytochrome B (PHYB) triggers its condensation into subnuclear photobodies (PBs). However, the function of PBs remains frustratingly elusive. Here, we show that PHYB condensation enables the co-occurrence and competition of two antagonistic phase-separated signaling actions. We found that PHYB recruits PHYTOCHROME-INTERACTING FACTOR5 (PIF5) to PBs and, surprisingly, that PHYB exerts opposing roles in degrading and stabilizing PIF5. Perturbing PB size by overproducing PHYB provoked a biphasic PIF5 response: while a moderate increase in PHYB enhanced PIF5 degradation, further elevating the PHYB level stabilized PIF5 by retaining more of it in enlarged PBs. Our results support a model in which PHYB condensation stabilizes PIF5 in PBs to counteract PIF5 degradation in the surrounding nucleoplasm, thereby enabling an environmentally sensitive counterbalancing mechanism to titrate nucleoplasmic PIF5 and its transcriptional output. This PB-enabled signaling mechanism provides a framework for regulating a plethora of PHYB-interacting signaling molecules in diverse plant environmental responses. We propose that this function of PBs represents a general function of biomolecular condensates to allow distinct variations of a cellular process or signaling pathway to coexist and interact to generate dynamically adjustable integrated outputs within a single subcellular space.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566572">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566572" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566572">
        <p class="paperTitle">An Arabidopsis leaf expression atlas across diurnal and developmental scales</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566572" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566572" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vong, G.; McCarthy, K.; Claydon, W.; Davis, S. J.; Redmond, E. J.; Ezer, D.</p>
        <p class="info">Score: 12.9, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566572' target='https://doi.org/10.1101/2023.11.10.566572'> 10.1101/2023.11.10.566572</a></p>
        <p class="abstract">Mature plant leaves are a composite of distinct cell types, including epidermal, mesophyll and vascular cells. Notably the proportion of these cells, and the relative transcript concentrations within different cell types may change over time. While gene expression data at a single-cell level can provide cell-type specific expression values, it is often too expensive to perform this on high resolution time series. Although bulk RNA-seq can be performed in a high resolution time series, the RNA-seq in whole leaves measures the average gene expression values across all cell types in each sample. In this study, we combined single cell RNA-seq data with time-series data from whole leaves to infer an atlas of cell type-specific gene expression changes over time for Arabidopsis thaliana. We inferred how relative transcript concentrations of cell types vary across diurnal and developmental time scales. Importantly this analysis revealed three sub-groups of mesophyll cells that have distinct temporal profiles of expression. Finally, we develop tissue-specific gene networks that form a new community resource: An Arabidopsis Leaf Time-Dependent Atlas (AraLeTa), which allows users to extract gene networks that are confirmed by transcription factor binding data and specific to certain cell types, at certain times of day and certain developmental stages, which is available at: https://regulatorynet.shinyapps.io/araleta/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564390">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564390" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564390">
        <p class="paperTitle">The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564390" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564390" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium,  ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J. H.; Schmutz, J.; Webber, J.; Grimwood, J.; Jacquet, C.; Dunand, C.; Nelson, J.; Roux, F.; Philippe, H.; Schornack, S.; Bonhomme, M.; Delaux, P.-M.</p>
        <p class="info">Score: 66.3, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564390' target='https://doi.org/10.1101/2023.10.27.564390'> 10.1101/2023.10.27.564390</a></p>
        <p class="abstract">Plant adaptation to a terrestrial life 450 million years ago played a major role in the evolution of life on Earth. This shift from an aquatic environment has been mostly studied by focusing on flowering plants. Here, we gathered a collection of 133 accessions of the non-vascular plants Marchantia polymorpha and studied its intraspecific diversity using selection signature analyses, genome-environment association study and a gene-centered pangenome. We identified adaptive features shared with flowering plants, such as peroxidases or nucleotide-binding and leucine-rich repeat (NLR), which likely played a role in the adaptation of the first land plants to the terrestrial habitat. The M. polymorpha pangenome also harbored lineage-specific accessory genes absent from seed plants. We conclude that different land plants lineages still share many elements from the genetic toolkit evolved by their most recent common ancestor to adapt to the terrestrial habitat, refined by lineage specific polymorphisms and gene family evolutions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.567642">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.567642" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.567642">
        <p class="paperTitle">Conservation of heat stress acclimation by the inositol polyphosphate multikinase, IPMK responsible for 4/6-InsP7 production in land plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.567642" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.567642" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yadav, R.; Liu, G.; Rana, P.; Pullagurla, N. J.; Qiu, D.; Jessen, H. J.; Laha, D.</p>
        <p class="info">Score: 6.3, Published: 2023-11-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.567642' target='https://doi.org/10.1101/2023.11.17.567642'> 10.1101/2023.11.17.567642</a></p>
        <p class="abstract">Inositol pyrophosphates (PP-InsPs) are soluble cellular messengers that integrate environmental cues to induce adaptive responses in eukaryotes. In plants, the biological functions of various PP-InsP species are poorly understood, largely due to the absence of canonical enzymes present in other eukaryotes. The recent identification of a new PP-InsP isomer with yet unknown enantiomeric identity, 4/6-InsP7 in the eudicot Arabidopsis thaliana, further highlights the intricate PP-InsP signalling network employed by plants. The abundance of 4/6-InsP7 in land plants, the enzyme(s) responsible for its synthesis, and the physiological functions of this species are all currently unknown. In this study, we show that 4/6-InsP7 is the major PP-InsP species present across land plants. Our findings demonstrate that the Arabidopsis inositol polyphosphate multikinase (IPMK) homolog, AtIPK2 generates 4/6-InsP7 in vitro. Furthermore, the cellular level of 4/6-InsP7 is controlled by the two Arabidopsis IPMK isoforms, AtIPK2 and AtIPK2{beta}. Notably, the activity of these IPMK proteins is critical for heat stress acclimation in Arabidopsis. During heat stress, the expression of genes encoding various heat shock proteins controlled by the heat shock factors (HSFs) is affected in the AtIPK2-deficient plants. Furthermore, we show that the transcription activity of HSF is regulated by the AtIPK2 proteins. Our parallel investigations using the liverwort Marchantia polymorpha suggest that the InsP6 kinase activity of IPMK and the role of IPMK in regulating the heat stress response are evolutionarily conserved. Collectively, our study indicates that IPMK has played a critical role in transducing environmental cues for biological processes during land plant evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.567553">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.567553" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.567553">
        <p class="paperTitle">Pseudomonas syringae pv. tomato DC3000 induces defense responses in diverse maize inbred lines</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.567553" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.567553" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jaiswal, N.; Helm, M.</p>
        <p class="info">Score: 6.0, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.567553' target='https://doi.org/10.1101/2023.11.17.567553'> 10.1101/2023.11.17.567553</a></p>
        <p class="abstract">Many phytopathogens translocate virulence (effector) proteins into plant cells to circumvent host immune responses during infection. One such pathogen is Pseudomonas syringae pv. tomato DC3000, which secretes at least twenty-nine effectors into host cells, of which a subset elicits host defense responses in crop plant species. However, it is unknown whether P. syringae pv. tomato DC3000 activates immune responses in diverse maize inbreds. Here, we screened a diverse maize germplasm collection for effector-dependent recognition of this bacterial pathogen. As a control, we infiltrated Pseudomonas syringae DC3000(D36E), a derivative of P. syringae pv. tomato DC3000 that lacks all endogenous effectors. In our evaluations, we observed a variety of responses to P. syringae pv. tomato DC3000 in maize and scored the phenotypes as either no observable response (N) or as one of three responses: weak chlorosis (WC), chlorosis (C) with minimal cell death, and hypersensitive reaction (HR)-like cell death. Of the twenty-six maize inbreds screened, 13 were scored as N, 2 as WC, 2 as C, and 9 as HR-like cell death. Importantly, no maize line responded to P. syringae DC3000(D36E), demonstrating the responses observed are likely dependent upon recognition of one or more Pseudomonas effectors. Importantly, maize inbreds that recognize P. syringae pv. tomato DC3000 accumulated detectable hydrogen peroxide as well as an increase in transcript expression of a subset of maize defense genes. Collectively, our results will likely stimulate new research aimed at identifying the cognate maize disease resistance proteins that recognize the activities of one or more bacterial effectors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.565111">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.565111" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.565111">
        <p class="paperTitle">Distinct mechanisms of plant immune resilience revealed by natural variation in warm temperature-modulated disease resistance among Arabidopsis accessions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.565111" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.565111" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rossi, C. A. M.; Patel, D. N.; Castroverde, C. D. M.</p>
        <p class="info">Score: 5.6, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.565111' target='https://doi.org/10.1101/2023.11.15.565111'> 10.1101/2023.11.15.565111</a></p>
        <p class="abstract">Elevated temperature suppresses production of the key plant defence hormone salicylic acid (SA). Heat-mediated SA suppression and resulting plant vulnerability are due to downregulated expression of CALMODULIN BINDING PROTEIN 60-LIKE G (CBP60g) and SYSTEMIC ACQUIRED RESISTANCE DEFICIENT 1 (SARD1), which encode master regulators of plant immunity. However, previous studies in Arabidopsis thaliana plants have primarily focused on the accession Columbia-0 (Col-0), while the mechanisms governing the intraspecific variation in Arabidopsis immunity under elevated temperature has remained unknown. Here we show that BASIC HELIX LOOP HELIX 059 (bHLH059), a thermosensitive SA regulator at non-stress temperatures, does not regulate immune suppression under warmer temperatures. In agreement, temperature-resilient and -sensitive Arabidopsis accessions based on disease resistance to the bacterial pathogen Pseudomonas syringae pv. tomato (Pst) DC3000 did not correlate with bHLH059 sequence polymorphisms. Finally, we found that different temperature-resilient accessions exhibit varying CBP60g and SARD1 expression profiles, potentially revealing both CBP60g/SARD1-dependent and independent mechanisms of plant immune resilience to warming temperature. Collectively, this study has unveiled the intraspecific diversity of Arabidopsis immune responses under warm temperatures, which is differentially mediated by the master transcription factors CBP60g and SARD1. Dissecting the mechanisms underlying temperature-modulated plant immunity aid in predicting plant responses to climate change and provide foundational knowledge for climate-resilient crop engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566459">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566459" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566459">
        <p class="paperTitle">Comparative mutant analyses reveal a novel mechanism of ARF regulation in land plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566459" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566459" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Prigge, M. J.; Morffy, N.; De Neve, A.; Szutu, W.; Juarez, M. J. A.; Johnson, K.; Do, N.; Lavy, M.; Hake, S.; Strader, L.; Estelle, M.; Richardson, A. E.</p>
        <p class="info">Score: 5.5, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566459' target='https://doi.org/10.1101/2023.11.09.566459'> 10.1101/2023.11.09.566459</a></p>
        <p class="abstract">A major challenge in plant biology is to understand how the plant hormone auxin regulates diverse transcriptional responses throughout development, in different environments, and in different species. The answer may lie in the specific complement of auxin signaling components in each cell. The balance between activators (class-A AUXIN RESPONSE FACTORS) and repressors (class-B ARFs) is particularly important. It is unclear how this balance is achieved. Through comparative analysis of novel, dominant mutants in maize and the moss Physcomitrium patens, we have discovered a [~]500-million-year-old mechanism of class-B ARF protein level regulation, important in determining cell fate decisions across land plants. Thus, our results add a key piece to the puzzle of how auxin regulates plant development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566501">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566501" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566501">
        <p class="paperTitle">Genetic factors acting prior to dormancy in sour cherry influence bloom time the following spring</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566501" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566501" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goeckeritz, C. Z.; Grabb, C.; Grumet, R.; Iezzoni, A. F.; Hollender, C. A.</p>
        <p class="info">Score: 5.1, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566501' target='https://doi.org/10.1101/2023.11.09.566501'> 10.1101/2023.11.09.566501</a></p>
        <p class="abstract">Bloom time is central to tree fruit production, and for Prunus species floral development leading up to bloom spans four seasons. Understanding this entire process is crucial for developing strategies to manipulate bloom time to prevent crop loss due to climate change. Here, we present a detailed examination of flower development from initiation until bloom for early- and late-blooming sour cherries (Prunus cerasus) from a population segregating for a major bloom time QTL on chromosome 4. Using a new staging system, we identified floral buds from early-blooming trees were persistently more advanced than those from late-blooming siblings. A gDNA coverage analysis revealed the late-blooming haplotype of this QTL, k, is located on a subgenome originating from the late-blooming P. fruticosa progenitor. Transcriptome analyses identified a large number of genes within this QTL as differentially expressed between early- and late-blooming trees during the vegetative-to-floral transition. From these, we identified candidate genes for the late bloom phenotype, including multiple transcription factors homologous to REproductive Meristem (REM) B3 domain-containing proteins. Additionally, we determined the basis of k in sour cherry is likely separate from candidate genes found in sweet cherry - suggesting several major regulators of bloom time are located on Prunus chromosome 4.

HIGHLIGHTDormancy is a main effector of bloom time in fruit trees. However, developmental, genetic, and transcriptomic analyses indicate differences in flower development before dormancy significantly influence flowering time in cherry.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.18.567481">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.18.567481" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.18.567481">
        <p class="paperTitle">Guanylate cyclase activity of TIR1/AFB auxin receptorsin rapid auxin responses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.18.567481" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.18.567481" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Qi, L.; Kwiatkowski, M.; Kulich, I.; Chen, H.; Gao, Y.; Yun, P.; Li, L.; Shabala, S.; Farmer, E.; Jaworski, K.; Friml, J.</p>
        <p class="info">Score: 4.0, Published: 2023-11-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.18.567481' target='https://doi.org/10.1101/2023.11.18.567481'> 10.1101/2023.11.18.567481</a></p>
        <p class="abstract">The major developmental signal in plants, auxin is perceived by TIR1/AFB receptors. It triggers transcriptional reprogramming via well-established canonical mechanism but also elusive rapid, non-transcriptional responses. Here we demonstrate that TIR1/AFB receptors have, next to recently identified adenylate cyclase, also guanylate cyclase activity. Auxin perception activates independently the cAMP and cGMP production by TIR1/AFBs in vitro and increases cAMP and cGMP levels in planta with a slow and fast dynamics, respectively. Exogenous cGMP but not cAMP application induces rapid cytosolic Ca2&#43; transients and root growth inhibition, suggesting that TIR1/AFB-derived cGMP mediates rapid auxin responses. This unprecedented combination of adenylate and guanylate cyclase activities in a hormone receptor provides a new paradigm for how a single perception mechanism can mediate a multitude of diverse downstream responses.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
