<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Single nuclei sequencing reveals C4 photosynthesis is based on rewiring of ancestral cell identity networks
Authors: Swift, J.; Luginbuehl, L. H.; Schreier, T.; Donald, R. M.; Lee, T.; Nery, J.; Ecker, J. R.; Hibberd, J. M.
Score: 43.7, Published: 2023-10-27 DOI: 10.1101/2023.10.26.562893
In multicellular systems changes to the patterning of gene expression drive modifications in cell function and trait evolution. One striking example is found in more than sixty plant lineages where compartmentation of photosynthesis between cell types allowed evolution of the efficient C4 pathway from the ancestral C3 state.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="Single nuclei sequencing reveals C4 photosynthesis is based on rewiring of ancestral cell identity networks
Authors: Swift, J.; Luginbuehl, L. H.; Schreier, T.; Donald, R. M.; Lee, T.; Nery, J.; Ecker, J. R.; Hibberd, J. M.
Score: 43.7, Published: 2023-10-27 DOI: 10.1101/2023.10.26.562893
In multicellular systems changes to the patterning of gene expression drive modifications in cell function and trait evolution. One striking example is found in more than sixty plant lineages where compartmentation of photosynthesis between cell types allowed evolution of the efficient C4 pathway from the ancestral C3 state." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-29T10:37:40+00:00" />
<meta property="article:modified_time" content="2023-10-29T10:37:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="Single nuclei sequencing reveals C4 photosynthesis is based on rewiring of ancestral cell identity networks
Authors: Swift, J.; Luginbuehl, L. H.; Schreier, T.; Donald, R. M.; Lee, T.; Nery, J.; Ecker, J. R.; Hibberd, J. M.
Score: 43.7, Published: 2023-10-27 DOI: 10.1101/2023.10.26.562893
In multicellular systems changes to the patterning of gene expression drive modifications in cell function and trait evolution. One striking example is found in more than sixty plant lineages where compartmentation of photosynthesis between cell types allowed evolution of the efficient C4 pathway from the ancestral C3 state."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "Single nuclei sequencing reveals C4 photosynthesis is based on rewiring of ancestral cell identity networks\nAuthors: Swift, J.; Luginbuehl, L. H.; Schreier, T.; Donald, R. M.; Lee, T.; Nery, J.; Ecker, J. R.; Hibberd, J. M.\nScore: 43.7, Published: 2023-10-27 DOI: 10.1101/2023.10.26.562893\nIn multicellular systems changes to the patterning of gene expression drive modifications in cell function and trait evolution. One striking example is found in more than sixty plant lineages where compartmentation of photosynthesis between cell types allowed evolution of the efficient C4 pathway from the ancestral C3 state.",
  "keywords": [
    
  ],
  "articleBody": " Single nuclei sequencing reveals C4 photosynthesis is based on rewiring of ancestral cell identity networks\nAuthors: Swift, J.; Luginbuehl, L. H.; Schreier, T.; Donald, R. M.; Lee, T.; Nery, J.; Ecker, J. R.; Hibberd, J. M.\nScore: 43.7, Published: 2023-10-27 DOI: 10.1101/2023.10.26.562893\nIn multicellular systems changes to the patterning of gene expression drive modifications in cell function and trait evolution. One striking example is found in more than sixty plant lineages where compartmentation of photosynthesis between cell types allowed evolution of the efficient C4 pathway from the ancestral C3 state. The molecular events enabling this transition are unclear. We used single nuclei sequencing to generate a cell level expression atlas for C3 rice and C4 sorghum during photomorphogenesis. In both species a conserved cistrome was identified for each cell type and initiation of photosynthesis gene expression was conditioned by cell identity. Photosynthesis genes switching expression from mesophyll in rice to bundle sheath in sorghum acquire hallmarks of bundle sheath identity. The sorghum bundle sheath has also acquired gene networks associated with C3 guard cells. We conclude C4 photosynthesis is based on rewiring in cis that exapts cell identity networks of C3 plants.\nA fully sequenced collection of homozygous EMS mutants for forward and reverse genetic screens in Arabidopsis thaliana\nAuthors: Carrere, S.; Routaboul, J.-M.; Savourat, P.; Bellenot, C.; Lopez, H.; Quiroz-Monnens, T.; Ricou, A.; Camilleri, C.; Laufs, P.; mercier, r.; Noel, L. D.\nScore: 43.5, Published: 2023-10-27 DOI: 10.1101/2023.10.26.564234\nGenetic screens are powerful tools for biological research and are one of the reasons for the success of the thale cress Arabidopsis thaliana as a model species. Here, we describe the whole-genome sequencing of 871 Arabidopsis lines from the Homozygous EMS Mutant (HEM) collection as a novel resource for forward and reverse genetics. With an average 576 high-confidence mutations per HEM line, over three independent mutations altering protein sequence are found on average per gene in the collection. Pilot reverse genetics experiments on reproductive, developmental and physiological traits confirmed the efficacy of the tool for identifying both null and knockdown alleles. The possibility of conducting subtle repeated phenotyping of HEM lines and the immediate availability of the mutations will empower forward genetic approaches. The sequence resource is searchable with the ATHEM web interface (https://lipm-browsers.toulouse.inra.fr/pub/ATHEM/), and the biological material is distributed by the Versailles Arabidopsis Stock Center.\nThe NRC0 gene cluster of sensor and helper NLR immune receptors is functionally conserved across asterid plants\nAuthors: Sakai, T.; Martinez-Anaya, C.; Contreras, M. P.; Kamoun, S.; Wu, C.-H.; Adachi, H.\nScore: 31.1, Published: 2023-10-23 DOI: 10.1101/2023.10.23.563533\nNLR (nucleotide-binding domain and leucine-rich repeat-containing) proteins can form complex receptor networks to confer innate immunity. NRCs are phylogenetically related nodes that function downstream of a massively expanded network of disease resistance proteins that protect against multiple plant pathogens. Here, we used phylogenomic methods to reconstruct the macroevolution of the NRC family. One of the NRCs, we termed NRC0, is the only family member shared across asterid plants, leading us to investigate its evolutionary history and genetic organization. In several asterid species, NRC0 is genetically clustered to other NLRs that are phylogenetically related to NRC-dependent disease resistance genes. This prompted us to hypothesize that the ancestral state of the NRC network is an NLR helper-sensor gene cluster that was present early during asterid evolution. We validated this hypothesis by demonstrating that NRC0 is essential for the hypersensitive cell death induced by its genetically linked sensor NLR partners in four divergent asterid species: tomato, wild sweet potato, coffee and carrot. In addition, activation of a sensor NLR leads to high-order complex formation of its genetically linked NRC0 similar to other NRCs. Our findings map out contrasting evolutionary dynamics in the macroevolution of the NRC network over the last 125 million years from a functionally conserved NLR gene cluster to a massive genetically dispersed network. One-sentence summaryNRC0 gene cluster is functionally conserved across divergent asterid species and predates the massively expanded NRC network.\nNRC immune receptor networks show diversified hierarchical genetic architecture across plant lineages\nAuthors: Goh, F.-J.; Huang, C.-Y.; Derevnina, L.; Wu, C.-H.\nScore: 17.4, Published: 2023-10-27 DOI: 10.1101/2023.10.25.563953\nPlants developed sophisticated immune systems consisting of nucleotide-binding domain and leucine-rich repeat-containing (NLR) proteins to repel invading pathogens. The NRC (NLR required for cell death) family is a group of helper NLRs that form a complex genetic network with multiple sensor NLRs to provide resistance against various pathogens of solanaceous plants. However, how the NRC network has evolved and how it functions outside of solanaceous plants is currently unknown. We conducted phylogenomic and macroevolutionary analyses comparing NLRs identified from different lineages of asterids and found that NRC networks showed significant lineage-specific expansion patterns in lamiids but not in Ericales and campanulids. Using transient expression assays in Nicotiana benthamiana, we show that the NRC networks in Ericales and campanulids are simple, with one or two NRC nodes, while the NRC networks of lamiids were complex, with multiple partially redundant NRC nodes. Phylogenetic analyses grouped the NRC helper NLRs into three NRC0 subclades that are conserved, and several family-specific NRC subclades of lamiids that show signatures of diversifying selection. Functional analyses of NRCs and NRC-dependent sensor NLRs from different species revealed that members of the NRC0 subclades are partially interchangeable, with several functioning with NRC0-dependent sensor NLRs across asterids. In contrast, family-specific NRC members in lamiids display a lack of interchangeability, with only a few showing compatibility with sensor NLRs across different plant families. Our findings highlight the distinctive evolutionary patterns of the NRC networks in asterids and provide potential insights into transferring disease resistance across plant lineages.\nGenotype-specific nonphotochemical quenching responses to nitrogen deficit are linked to chlorophyll a to b ratios\nAuthors: Sahay, S.; Grzybowski, M.; Schnable, J. C.; Glowacka, K.\nScore: 10.4, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563650\nNon-photochemical quenching (NPQ) protects plants from photodamage caused by excess light energy. The mechanism of NPQ appears to be conserved across flowering plants. However, substantial variation in NPQ has been reported within different genotypes of the same species grown under the environmental conditions. Individual maize genotypes from a diversity panel exhibited a range of responses to low nitrogen with some genotypes exhibiting increased NPQ between control and low nitrogen conditions and others exhibiting no change. These patterns were consistent for the same genotypes across multiple field seasons. NPQ increases under low nitrogen were correlated with shifts in the ratio of chlorophyll a to chlorophyll b consistent with a decrease in reaction centers. Both photosynthetic capacity and dry biomass accumulation decreased more in maize genotypes which were unable to maintain constant NPQ levels between control and low N conditions. Collectively these results suggest that the ability to maintain sufficient numbers of reaction centers under low nitrogen conditions and avoid dissipating a greater proportion of absorbed light energy via the NPQ pathway may play a key role in increasing carbon fixation and productivity in nitrogen-limited environments.\nIntegration of shoot-derived polypeptide signals by root TGA transcription factors is essential for survival under fluctuating nitrogen environments\nAuthors: Kobayashi, R.; Ohkubo, Y.; Izumi, M.; Ota, R.; Yamada, K.; Hayashi, Y.; Yamashita, Y.; Noda, S.; Ogawa-Ohnishi, M.; Matsubayashi, Y.\nScore: 9.2, Published: 2023-10-23 DOI: 10.1101/2023.10.23.563506\nUnlike plants in the field, which experience significant temporal fluctuations in environmental conditions, plants in the laboratory are typically grown in controlled, stable environments. Therefore, signaling pathways evolved for survival in continuously fluctuating environments often remain functionally latent in laboratory settings. Here, we show that TGA1 and TGA4 act as hub transcription factors through which the expression of genes involved in high-affinity nitrate uptake are regulated in response to shoot-derived phloem mobile polypeptides, CEP DOWNSTREAM 1 (CEPD1), CEPD2 and CEPD-like 2 (CEPDL2) as nitrogen (N) deficiency signals, and Glutaredoxin S1 (GrxS1) to GrxS8 as N sufficiency signals. CEPD1/2/CEPDL2 and GrxS1-S8 competitively bind to TGA1/4 in roots, with the former acting as transcription coactivators that enhance the uptake of nitrate, while the latter function as corepressor complexes together with TOPLESS to limit nitrate uptake. Arabidopsis plants deficient in TGA1/4 maintain basal nitrate uptake and exhibit growth similar to wild-type plants in a stable N environment, but were impaired in regulation of nitrate acquisition in response to shoot N demand, leading to defective growth under continuously fluctuating N environments where rhizosphere nitrate ions switch periodically between deficient and sufficient states. TGA1/4 are crucial transcription factors that enable plants to survive under fluctuating and challenging N environmental conditions.\nZinc-finger (ZiF) fold secreted effectors form a functionally diverse family across lineages of the blast fungus Magnaporthe oryzae.\nAuthors: Concepcion, J. C. D. l.; Langner, T.; Fujisaki, K.; Were, V.; Yan, X.; Lam, A. H. C.; Saado, I.; Brabham, H.; Win, J.; Yoshida, K.; Talbot, N. J.; Terauchi, R.; Kamoun, S.; Banfield, M. J.\nScore: 36.6, Published: 2023-10-20 DOI: 10.1101/2023.10.18.562914\nFilamentous plant pathogens deliver effector proteins into host cells to suppress host defence responses and manipulate metabolic processes to support colonization. Understanding the evolution and molecular function of these effectors provides knowledge about pathogenesis and can suggest novel strategies to reduce damage caused by pathogens. However, effector proteins are highly variable, share weak sequence similarity and, although they can be grouped according to their structure, only a few structurally conserved effector families have been functionally characterized to date. Here, we demonstrate that Zinc-finger fold (ZiF) secreted proteins form a functionally diverse effector family in the blast fungus Magnaporthe oryzae. This family relies on the Zinc-finger motif for protein stability and is ubiquitously present, forming different effector tribes in blast fungus lineages infecting 13 different host species. Homologs of the canonical ZiF effector, AVR-Pii from rice infecting isolates, are present in multiple M. oryzae lineages, and the wheat infecting strains of the fungus, for example, possess an allele that also binds host Exo70 proteins and activates the immune receptor Pii. Furthermore, ZiF tribes vary in the host Exo70 proteins they bind, indicating functional diversification and an intricate effector/host interactome. Altogether, we uncovered a new effector family with a common protein fold that has functionally diversified in lineages of M. oryzae. This work expands our understanding of the diversity of M. oryzae effectors, the molecular basis of plant pathogenesis and may ultimately facilitate the development of new sources for pathogen resistance. Author SummaryDiseases caused by filamentous plant pathogens impact global food production, leading to severe economic and humanitarian consequences. These pathogens secrete hundreds of effectors inside the host to alter cellular processes and to promote infection and disease. Effector proteins have weak or no sequence similarity but can be grouped in structural families based on conserved protein folds. However, very few conserved effector families have been functionally characterized. We have identified a family of effectors with a shared Zinc-finger protein fold (ZiF) that is present in lineages of the blast fungus Magnaporthe oryzae that can, collectively, infect 13 different grasses. We characterized the binding of a sub-set of these proteins to putative Exo70 host targets and showed they can be recognized by the plant immune system. Furthermore, we show that other ZiF effectors do not bind Exo70 targets, suggesting functional specialization within this effector family for alternative interactors. These findings shed light on the diversity of effectors and their molecular functions, as well as potentially leading to the development of new sources of blast disease resistance in the future.\nBiolistics-mediated transformation of hornworts and its application to study pyrenoid protein localization\nAuthors: Lafferty, D.; Robison, T. A.; Gunadi, A.; Gunn, L. H.; Van Eck, J.; Li, F.-W.\nScore: 5.0, Published: 2023-10-26 DOI: 10.1101/2023.10.23.563637\nHornworts are a deeply diverged lineage of bryophytes that are sister to mosses and liverworts. Hornworts have an array of unique features that can be leveraged to illuminate not only the early evolution of land plants, but also alternative paths for nitrogen and carbon assimilation via cyanobacterial symbiosis and a pyrenoid-based CO2-concentrating mechanism (CCM), respectively. Despite this, hornworts are one of the few plant lineages with limited available genetic tools. Here we report an efficient biolistics method for generating transient-expression and stable transgenic lines in the model hornwort, Anthoceros agrestis. An average of 569 ({+/-} 268) cells showed transient expression per bombardment, with green fluorescent protein expression observed within 48 hours. A total of 81 stably transformed lines were recovered across three separate experiments, averaging six lines per bombardment. We followed the same method to transiently transform nine additional hornwort species, and obtained stable transformants from one. This method was further used to verify the localization of Rubisco and Rubisco activase in pyrenoids, which are central proteins for CCM function. Together, our biolistics approach offers key advantages over existing methods as it enables rapid transient expression and can be applied to widely diverse hornwort species.\nPhosphorylation Promotes DELLA Activity by Enhancing Its Binding to Histone H2A at Target Chromatin in Arabidopsis\nAuthors: Huang, X.; Zentella, R.; Park, J.; Reser, L.; Bai, D.; Ross, M.; Shabanowitz, J.; Hunt, D.; Sun, T.-p.\nScore: 8.3, Published: 2023-10-13 DOI: 10.1101/2023.10.10.561786\nDELLA proteins are conserved master growth regulators that play a central role in controlling plant development in response to internal and environmental cues. DELLAs function as transcription regulators, which are recruited to target promoters by binding to transcription factors (TFs) and histone H2A via its GRAS domain. Recent studies showed that DELLA stability is regulated post-translationally via two mechanisms, phytohormone gibberellin-induced polyubiquitination for its rapid degradation, and Small Ubiquitin-like Modifier (SUMO)- conjugation to alter its accumulation. Moreover, DELLA activity is dynamically modulated by two distinct glycosylations: DELLA-TF interactions are enhanced by O-fucosylation, but inhibited by O-linked N-acetylglucosamine (O-GlcNAc) modification. However, the role of DELLA phosphorylation remains unclear. Here, we identified phosphorylation sites in REPRESSOR OF ga1-3 (RGA, an AtDELLA) purified from Arabidopsis by tandem mass spectrometry analysis, and showed that phosphorylation of the RGA LKS-peptide in the poly- S/T region enhances RGA-H2A interaction and RGA association with target promoters. Interestingly, phosphorylation does not affect RGA-TF interactions. Our study has uncovered that phosphorylation is a new regulatory mechanism of DELLA activity.\nMulti-Omics Insights into Grapevine Dormancy-Budburst Transition: Gene Expression, miRNA Regulation, and DNA Methylation\nAuthors: Shinde, H.; Raihan, T.; Anand, L.; Perry, S.; Geneve, R.; Rodriguez Lopez, C. M.\nScore: 4.0, Published: 2023-10-23 DOI: 10.1101/2023.10.21.563414\nIn grapevine, the transition from a dormant bud to budburst is a critical developmental process related to vegetative and reproductive growth. We generated a time series analysis (five sampling time points) and used transcriptome, small RNA, and whole-genome bisulfite sequencing to characterize this transition. Ecodormant buds took an average of 17 days to budburst. Transcriptome analysis identified a total of 7002 differentially expressed genes across all sampling times and revealed that the brassinosteroid metabolism and the linoleic acid metabolism pathways are upregulated and downregulated respectively across all time points. Gene expression cluster analysis identified the activation of the photosynthesis pathway and photosynthesis related genes during this transition. miRNA expression analysis identified a steady increase in expression of two miRNAs, miR159a and miR159b during the transition from dormancy to budburst. Our analysis suggests that these two miRNAs promote budburst by repressing the expression of auxin responsive genes. Finally, a continued increase in CG methylation levels was observed during the dormancy to budburst transition. Differential methylation analysis, considering dormant buds as the control stage, yielded 6354 differentially methylated regions across the genome. Two glucosidase genes exhibited increases in promoter methylation and a corresponding decrease in gene expression in our analysis. This study provides a multi-omics view of grapevine bud transition from dormancy to bud burst and unveils the interacting genetic and epigenetic networks regulating this process.\n",
  "wordCount" : "2566",
  "inLanguage": "en",
  "datePublished": "2023-10-29T10:37:40Z",
  "dateModified": "2023-10-29T10:37:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta"><span>updated on October 29, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.562893">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.562893" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.562893">
        <p class="paperTitle">Single nuclei sequencing reveals C4 photosynthesis is based on rewiring of ancestral cell identity networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.562893" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.562893" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Swift, J.; Luginbuehl, L. H.; Schreier, T.; Donald, R. M.; Lee, T.; Nery, J.; Ecker, J. R.; Hibberd, J. M.</p>
        <p class="info">Score: 43.7, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.562893' target='https://doi.org/10.1101/2023.10.26.562893'> 10.1101/2023.10.26.562893</a></p>
        <p class="abstract">In multicellular systems changes to the patterning of gene expression drive modifications in cell function and trait evolution. One striking example is found in more than sixty plant lineages where compartmentation of photosynthesis between cell types allowed evolution of the efficient C4 pathway from the ancestral C3 state. The molecular events enabling this transition are unclear. We used single nuclei sequencing to generate a cell level expression atlas for C3 rice and C4 sorghum during photomorphogenesis. In both species a conserved cistrome was identified for each cell type and initiation of photosynthesis gene expression was conditioned by cell identity. Photosynthesis genes switching expression from mesophyll in rice to bundle sheath in sorghum acquire hallmarks of bundle sheath identity. The sorghum bundle sheath has also acquired gene networks associated with C3 guard cells. We conclude C4 photosynthesis is based on rewiring in cis that exapts cell identity networks of C3 plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564234">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564234" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564234">
        <p class="paperTitle">A fully sequenced collection of homozygous EMS mutants for forward and reverse genetic screens in Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564234" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564234" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carrere, S.; Routaboul, J.-M.; Savourat, P.; Bellenot, C.; Lopez, H.; Quiroz-Monnens, T.; Ricou, A.; Camilleri, C.; Laufs, P.; mercier, r.; Noel, L. D.</p>
        <p class="info">Score: 43.5, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564234' target='https://doi.org/10.1101/2023.10.26.564234'> 10.1101/2023.10.26.564234</a></p>
        <p class="abstract">Genetic screens are powerful tools for biological research and are one of the reasons for the success of the thale cress Arabidopsis thaliana as a model species. Here, we describe the whole-genome sequencing of 871 Arabidopsis lines from the Homozygous EMS Mutant (HEM) collection as a novel resource for forward and reverse genetics. With an average 576 high-confidence mutations per HEM line, over three independent mutations altering protein sequence are found on average per gene in the collection. Pilot reverse genetics experiments on reproductive, developmental and physiological traits confirmed the efficacy of the tool for identifying both null and knockdown alleles. The possibility of conducting subtle repeated phenotyping of HEM lines and the immediate availability of the mutations will empower forward genetic approaches. The sequence resource is searchable with the ATHEM web interface (https://lipm-browsers.toulouse.inra.fr/pub/ATHEM/), and the biological material is distributed by the Versailles Arabidopsis Stock Center.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563533">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563533" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563533">
        <p class="paperTitle">The NRC0 gene cluster of sensor and helper NLR immune receptors is functionally conserved across asterid plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563533" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563533" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sakai, T.; Martinez-Anaya, C.; Contreras, M. P.; Kamoun, S.; Wu, C.-H.; Adachi, H.</p>
        <p class="info">Score: 31.1, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563533' target='https://doi.org/10.1101/2023.10.23.563533'> 10.1101/2023.10.23.563533</a></p>
        <p class="abstract">NLR (nucleotide-binding domain and leucine-rich repeat-containing) proteins can form complex receptor networks to confer innate immunity. NRCs are phylogenetically related nodes that function downstream of a massively expanded network of disease resistance proteins that protect against multiple plant pathogens. Here, we used phylogenomic methods to reconstruct the macroevolution of the NRC family. One of the NRCs, we termed NRC0, is the only family member shared across asterid plants, leading us to investigate its evolutionary history and genetic organization. In several asterid species, NRC0 is genetically clustered to other NLRs that are phylogenetically related to NRC-dependent disease resistance genes. This prompted us to hypothesize that the ancestral state of the NRC network is an NLR helper-sensor gene cluster that was present early during asterid evolution. We validated this hypothesis by demonstrating that NRC0 is essential for the hypersensitive cell death induced by its genetically linked sensor NLR partners in four divergent asterid species: tomato, wild sweet potato, coffee and carrot. In addition, activation of a sensor NLR leads to high-order complex formation of its genetically linked NRC0 similar to other NRCs. Our findings map out contrasting evolutionary dynamics in the macroevolution of the NRC network over the last 125 million years from a functionally conserved NLR gene cluster to a massive genetically dispersed network.

One-sentence summaryNRC0 gene cluster is functionally conserved across divergent asterid species and predates the massively expanded NRC network.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.563953">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.563953" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.563953">
        <p class="paperTitle">NRC immune receptor networks show diversified hierarchical genetic architecture across plant lineages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.563953" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.563953" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goh, F.-J.; Huang, C.-Y.; Derevnina, L.; Wu, C.-H.</p>
        <p class="info">Score: 17.4, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.563953' target='https://doi.org/10.1101/2023.10.25.563953'> 10.1101/2023.10.25.563953</a></p>
        <p class="abstract">Plants developed sophisticated immune systems consisting of nucleotide-binding domain and leucine-rich repeat-containing (NLR) proteins to repel invading pathogens. The NRC (NLR required for cell death) family is a group of helper NLRs that form a complex genetic network with multiple sensor NLRs to provide resistance against various pathogens of solanaceous plants. However, how the NRC network has evolved and how it functions outside of solanaceous plants is currently unknown. We conducted phylogenomic and macroevolutionary analyses comparing NLRs identified from different lineages of asterids and found that NRC networks showed significant lineage-specific expansion patterns in lamiids but not in Ericales and campanulids. Using transient expression assays in Nicotiana benthamiana, we show that the NRC networks in Ericales and campanulids are simple, with one or two NRC nodes, while the NRC networks of lamiids were complex, with multiple partially redundant NRC nodes. Phylogenetic analyses grouped the NRC helper NLRs into three NRC0 subclades that are conserved, and several family-specific NRC subclades of lamiids that show signatures of diversifying selection. Functional analyses of NRCs and NRC-dependent sensor NLRs from different species revealed that members of the NRC0 subclades are partially interchangeable, with several functioning with NRC0-dependent sensor NLRs across asterids. In contrast, family-specific NRC members in lamiids display a lack of interchangeability, with only a few showing compatibility with sensor NLRs across different plant families. Our findings highlight the distinctive evolutionary patterns of the NRC networks in asterids and provide potential insights into transferring disease resistance across plant lineages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563650">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563650" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563650">
        <p class="paperTitle">Genotype-specific nonphotochemical quenching responses to nitrogen deficit are linked to chlorophyll a to b ratios</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563650" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563650" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sahay, S.; Grzybowski, M.; Schnable, J. C.; Glowacka, K.</p>
        <p class="info">Score: 10.4, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563650' target='https://doi.org/10.1101/2023.10.24.563650'> 10.1101/2023.10.24.563650</a></p>
        <p class="abstract">Non-photochemical quenching (NPQ) protects plants from photodamage caused by excess light energy. The mechanism of NPQ appears to be conserved across flowering plants. However, substantial variation in NPQ has been reported within different genotypes of the same species grown under the environmental conditions. Individual maize genotypes from a diversity panel exhibited a range of responses to low nitrogen with some genotypes exhibiting increased NPQ between control and low nitrogen conditions and others exhibiting no change. These patterns were consistent for the same genotypes across multiple field seasons. NPQ increases under low nitrogen were correlated with shifts in the ratio of chlorophyll a to chlorophyll b consistent with a decrease in reaction centers. Both photosynthetic capacity and dry biomass accumulation decreased more in maize genotypes which were unable to maintain constant NPQ levels between control and low N conditions. Collectively these results suggest that the ability to maintain sufficient numbers of reaction centers under low nitrogen conditions and avoid dissipating a greater proportion of absorbed light energy via the NPQ pathway may play a key role in increasing carbon fixation and productivity in nitrogen-limited environments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563506">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563506" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563506">
        <p class="paperTitle">Integration of shoot-derived polypeptide signals by root TGA transcription factors is essential for survival under fluctuating nitrogen environments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563506" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563506" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kobayashi, R.; Ohkubo, Y.; Izumi, M.; Ota, R.; Yamada, K.; Hayashi, Y.; Yamashita, Y.; Noda, S.; Ogawa-Ohnishi, M.; Matsubayashi, Y.</p>
        <p class="info">Score: 9.2, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563506' target='https://doi.org/10.1101/2023.10.23.563506'> 10.1101/2023.10.23.563506</a></p>
        <p class="abstract">Unlike plants in the field, which experience significant temporal fluctuations in environmental conditions, plants in the laboratory are typically grown in controlled, stable environments. Therefore, signaling pathways evolved for survival in continuously fluctuating environments often remain functionally latent in laboratory settings. Here, we show that TGA1 and TGA4 act as hub transcription factors through which the expression of genes involved in high-affinity nitrate uptake are regulated in response to shoot-derived phloem mobile polypeptides, CEP DOWNSTREAM 1 (CEPD1), CEPD2 and CEPD-like 2 (CEPDL2) as nitrogen (N) deficiency signals, and Glutaredoxin S1 (GrxS1) to GrxS8 as N sufficiency signals. CEPD1/2/CEPDL2 and GrxS1-S8 competitively bind to TGA1/4 in roots, with the former acting as transcription coactivators that enhance the uptake of nitrate, while the latter function as corepressor complexes together with TOPLESS to limit nitrate uptake. Arabidopsis plants deficient in TGA1/4 maintain basal nitrate uptake and exhibit growth similar to wild-type plants in a stable N environment, but were impaired in regulation of nitrate acquisition in response to shoot N demand, leading to defective growth under continuously fluctuating N environments where rhizosphere nitrate ions switch periodically between deficient and sufficient states. TGA1/4 are crucial transcription factors that enable plants to survive under fluctuating and challenging N environmental conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.562914">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.562914" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.562914">
        <p class="paperTitle">Zinc-finger (ZiF) fold secreted effectors form a functionally diverse family across lineages of the blast fungus Magnaporthe oryzae.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.562914" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.562914" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Concepcion, J. C. D. l.; Langner, T.; Fujisaki, K.; Were, V.; Yan, X.; Lam, A. H. C.; Saado, I.; Brabham, H.; Win, J.; Yoshida, K.; Talbot, N. J.; Terauchi, R.; Kamoun, S.; Banfield, M. J.</p>
        <p class="info">Score: 36.6, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.562914' target='https://doi.org/10.1101/2023.10.18.562914'> 10.1101/2023.10.18.562914</a></p>
        <p class="abstract">Filamentous plant pathogens deliver effector proteins into host cells to suppress host defence responses and manipulate metabolic processes to support colonization. Understanding the evolution and molecular function of these effectors provides knowledge about pathogenesis and can suggest novel strategies to reduce damage caused by pathogens. However, effector proteins are highly variable, share weak sequence similarity and, although they can be grouped according to their structure, only a few structurally conserved effector families have been functionally characterized to date. Here, we demonstrate that Zinc-finger fold (ZiF) secreted proteins form a functionally diverse effector family in the blast fungus Magnaporthe oryzae. This family relies on the Zinc-finger motif for protein stability and is ubiquitously present, forming different effector tribes in blast fungus lineages infecting 13 different host species. Homologs of the canonical ZiF effector, AVR-Pii from rice infecting isolates, are present in multiple M. oryzae lineages, and the wheat infecting strains of the fungus, for example, possess an allele that also binds host Exo70 proteins and activates the immune receptor Pii. Furthermore, ZiF tribes vary in the host Exo70 proteins they bind, indicating functional diversification and an intricate effector/host interactome. Altogether, we uncovered a new effector family with a common protein fold that has functionally diversified in lineages of M. oryzae. This work expands our understanding of the diversity of M. oryzae effectors, the molecular basis of plant pathogenesis and may ultimately facilitate the development of new sources for pathogen resistance.

Author SummaryDiseases caused by filamentous plant pathogens impact global food production, leading to severe economic and humanitarian consequences. These pathogens secrete hundreds of effectors inside the host to alter cellular processes and to promote infection and disease. Effector proteins have weak or no sequence similarity but can be grouped in structural families based on conserved protein folds. However, very few conserved effector families have been functionally characterized. We have identified a family of effectors with a shared Zinc-finger protein fold (ZiF) that is present in lineages of the blast fungus Magnaporthe oryzae that can, collectively, infect 13 different grasses. We characterized the binding of a sub-set of these proteins to putative Exo70 host targets and showed they can be recognized by the plant immune system. Furthermore, we show that other ZiF effectors do not bind Exo70 targets, suggesting functional specialization within this effector family for alternative interactors. These findings shed light on the diversity of effectors and their molecular functions, as well as potentially leading to the development of new sources of blast disease resistance in the future.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563637">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563637" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563637">
        <p class="paperTitle">Biolistics-mediated transformation of hornworts and its application to study pyrenoid protein localization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563637" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563637" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lafferty, D.; Robison, T. A.; Gunadi, A.; Gunn, L. H.; Van Eck, J.; Li, F.-W.</p>
        <p class="info">Score: 5.0, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563637' target='https://doi.org/10.1101/2023.10.23.563637'> 10.1101/2023.10.23.563637</a></p>
        <p class="abstract">Hornworts are a deeply diverged lineage of bryophytes that are sister to mosses and liverworts. Hornworts have an array of unique features that can be leveraged to illuminate not only the early evolution of land plants, but also alternative paths for nitrogen and carbon assimilation via cyanobacterial symbiosis and a pyrenoid-based CO2-concentrating mechanism (CCM), respectively. Despite this, hornworts are one of the few plant lineages with limited available genetic tools. Here we report an efficient biolistics method for generating transient-expression and stable transgenic lines in the model hornwort, Anthoceros agrestis. An average of 569 ({&#43;/-} 268) cells showed transient expression per bombardment, with green fluorescent protein expression observed within 48 hours. A total of 81 stably transformed lines were recovered across three separate experiments, averaging six lines per bombardment. We followed the same method to transiently transform nine additional hornwort species, and obtained stable transformants from one. This method was further used to verify the localization of Rubisco and Rubisco activase in pyrenoids, which are central proteins for CCM function. Together, our biolistics approach offers key advantages over existing methods as it enables rapid transient expression and can be applied to widely diverse hornwort species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561786">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561786" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561786">
        <p class="paperTitle">Phosphorylation Promotes DELLA Activity by Enhancing Its Binding to Histone H2A at Target Chromatin in Arabidopsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561786" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561786" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, X.; Zentella, R.; Park, J.; Reser, L.; Bai, D.; Ross, M.; Shabanowitz, J.; Hunt, D.; Sun, T.-p.</p>
        <p class="info">Score: 8.3, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561786' target='https://doi.org/10.1101/2023.10.10.561786'> 10.1101/2023.10.10.561786</a></p>
        <p class="abstract">DELLA proteins are conserved master growth regulators that play a central role in controlling plant development in response to internal and environmental cues. DELLAs function as transcription regulators, which are recruited to target promoters by binding to transcription factors (TFs) and histone H2A via its GRAS domain. Recent studies showed that DELLA stability is regulated post-translationally via two mechanisms, phytohormone gibberellin-induced polyubiquitination for its rapid degradation, and Small Ubiquitin-like Modifier (SUMO)- conjugation to alter its accumulation. Moreover, DELLA activity is dynamically modulated by two distinct glycosylations: DELLA-TF interactions are enhanced by O-fucosylation, but inhibited by O-linked N-acetylglucosamine (O-GlcNAc) modification. However, the role of DELLA phosphorylation remains unclear. Here, we identified phosphorylation sites in REPRESSOR OF ga1-3 (RGA, an AtDELLA) purified from Arabidopsis by tandem mass spectrometry analysis, and showed that phosphorylation of the RGA LKS-peptide in the poly- S/T region enhances RGA-H2A interaction and RGA association with target promoters. Interestingly, phosphorylation does not affect RGA-TF interactions. Our study has uncovered that phosphorylation is a new regulatory mechanism of DELLA activity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.21.563414">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.21.563414" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.21.563414">
        <p class="paperTitle">Multi-Omics Insights into Grapevine Dormancy-Budburst Transition: Gene Expression, miRNA Regulation, and DNA Methylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.21.563414" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.21.563414" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shinde, H.; Raihan, T.; Anand, L.; Perry, S.; Geneve, R.; Rodriguez Lopez, C. M.</p>
        <p class="info">Score: 4.0, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.21.563414' target='https://doi.org/10.1101/2023.10.21.563414'> 10.1101/2023.10.21.563414</a></p>
        <p class="abstract">In grapevine, the transition from a dormant bud to budburst is a critical developmental process related to vegetative and reproductive growth. We generated a time series analysis (five sampling time points) and used transcriptome, small RNA, and whole-genome bisulfite sequencing to characterize this transition. Ecodormant buds took an average of 17 days to budburst. Transcriptome analysis identified a total of 7002 differentially expressed genes across all sampling times and revealed that the brassinosteroid metabolism and the linoleic acid metabolism pathways are upregulated and downregulated respectively across all time points. Gene expression cluster analysis identified the activation of the photosynthesis pathway and photosynthesis related genes during this transition. miRNA expression analysis identified a steady increase in expression of two miRNAs, miR159a and miR159b during the transition from dormancy to budburst. Our analysis suggests that these two miRNAs promote budburst by repressing the expression of auxin responsive genes. Finally, a continued increase in CG methylation levels was observed during the dormancy to budburst transition. Differential methylation analysis, considering dormant buds as the control stage, yielded 6354 differentially methylated regions across the genome. Two glucosidase genes exhibited increases in promoter methylation and a corresponding decrease in gene expression in our analysis. This study provides a multi-omics view of grapevine bud transition from dormancy to bud burst and unveils the interacting genetic and epigenetic networks regulating this process.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
