<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://yorks0n.github.io/TRxiv2" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/TRxiv2/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Mapping the landscape of transcription factor promoter activity during vegetative development in Marchantia
Authors: Romani, F.; Sauret-Güeto, S.; Rebmann, M.; Annese, D.; Bonter, I.; Tomaselli, M.; Dierschke, T.; Delmans, M.; Frangedakis, E.; Silvestri, L.; Rever, J.; Bowman, J. L.; Romani, I.; Haseloff, J.
Score: 46.0, Published: 2023-06-20 DOI: 10.1101/2023.06.17.545419
Transcription factors (TFs) are essential for the regulation of gene expression and cell fate determination. Characterising the transcriptional activity of TF genes in space and time is a critical step towards understanding complex biological systems.">
<meta name="author" content="">
<link rel="canonical" href="https://yorks0n.github.io/TRxiv2/posts/plant-biology/">
<link crossorigin="anonymous" href="/TRxiv2/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/TRxiv2/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://yorks0n.github.io/TRxiv2/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://yorks0n.github.io/TRxiv2/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://yorks0n.github.io/TRxiv2/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://yorks0n.github.io/TRxiv2/apple-touch-icon.png">
<link rel="mask-icon" href="https://yorks0n.github.io/TRxiv2/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="Mapping the landscape of transcription factor promoter activity during vegetative development in Marchantia
Authors: Romani, F.; Sauret-Güeto, S.; Rebmann, M.; Annese, D.; Bonter, I.; Tomaselli, M.; Dierschke, T.; Delmans, M.; Frangedakis, E.; Silvestri, L.; Rever, J.; Bowman, J. L.; Romani, I.; Haseloff, J.
Score: 46.0, Published: 2023-06-20 DOI: 10.1101/2023.06.17.545419
Transcription factors (TFs) are essential for the regulation of gene expression and cell fate determination. Characterising the transcriptional activity of TF genes in space and time is a critical step towards understanding complex biological systems." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://yorks0n.github.io/TRxiv2/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-06-28T09:51:07+00:00" />
<meta property="article:modified_time" content="2023-06-28T09:51:07+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="Mapping the landscape of transcription factor promoter activity during vegetative development in Marchantia
Authors: Romani, F.; Sauret-Güeto, S.; Rebmann, M.; Annese, D.; Bonter, I.; Tomaselli, M.; Dierschke, T.; Delmans, M.; Frangedakis, E.; Silvestri, L.; Rever, J.; Bowman, J. L.; Romani, I.; Haseloff, J.
Score: 46.0, Published: 2023-06-20 DOI: 10.1101/2023.06.17.545419
Transcription factors (TFs) are essential for the regulation of gene expression and cell fate determination. Characterising the transcriptional activity of TF genes in space and time is a critical step towards understanding complex biological systems."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://yorks0n.github.io/TRxiv2/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://yorks0n.github.io/TRxiv2/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "Mapping the landscape of transcription factor promoter activity during vegetative development in Marchantia\nAuthors: Romani, F.; Sauret-Güeto, S.; Rebmann, M.; Annese, D.; Bonter, I.; Tomaselli, M.; Dierschke, T.; Delmans, M.; Frangedakis, E.; Silvestri, L.; Rever, J.; Bowman, J. L.; Romani, I.; Haseloff, J.\nScore: 46.0, Published: 2023-06-20 DOI: 10.1101/2023.06.17.545419\nTranscription factors (TFs) are essential for the regulation of gene expression and cell fate determination. Characterising the transcriptional activity of TF genes in space and time is a critical step towards understanding complex biological systems.",
  "keywords": [
    
  ],
  "articleBody": " Mapping the landscape of transcription factor promoter activity during vegetative development in Marchantia\nAuthors: Romani, F.; Sauret-Güeto, S.; Rebmann, M.; Annese, D.; Bonter, I.; Tomaselli, M.; Dierschke, T.; Delmans, M.; Frangedakis, E.; Silvestri, L.; Rever, J.; Bowman, J. L.; Romani, I.; Haseloff, J.\nScore: 46.0, Published: 2023-06-20 DOI: 10.1101/2023.06.17.545419\nTranscription factors (TFs) are essential for the regulation of gene expression and cell fate determination. Characterising the transcriptional activity of TF genes in space and time is a critical step towards understanding complex biological systems. The vegetative gametophyte meristems of bryophytes share some characteristics with the shoot-apical meristems of flowering plants. However, the identity and expression profiles of TFs associated with gametophyte organization are largely unknown. With only [~]450 TF genes, Marchantia polymorpha is an outstanding model system for plant systems biology. We have generated a near-complete collection of promoter elements derived from Marchantia TF genes. We experimentally tested in planta reporter fusions for all the TF promoters in the collection and systematically analysed expression patterns in Marchantia gemmae. This allowed us to build a map of precise expression domains and identify a unique set of TFs expressed in the stem-cell zone, providing new insight into the dynamic regulation of the gametophytic meristem and its evolution. In addition, we provide an online database of expression patterns for all promoters in the collection. We expect that the promoter elements characterised here will be useful for cell-type specific expression, synthetic biology applications, and functional genomics.\nPlant Expression Omnibus - a comparative transcriptomic database for 103 Archaeplastida\nAuthors: Koh, E.; Goh, W.; Villanueva, E.; Julca, I.; Mutwil, M.\nScore: 29.9, Published: 2023-06-22 DOI: 10.1101/2023.06.22.546049\nThe Plant Expression Omnibus (PEO) is a web application that provides biologists with access to gene expression insights across over 100 plant species, [~]60,000 manually annotated RNA-seq samples, and more than four million genes. The tool allows users to explore the expression patterns of genes across different organs, identify organ-specific genes, and discover top co-expressed genes for any gene of interest. PEO also provides functional annotations for each gene, allowing for the identification of genetic modules and pathways. PEO is designed to facilitate comparative kingdom-wide gene expression analysis and provide a valuable resource for plant biology research. We provide two case studies to demonstrate the utility of PEO in identifying candidate genes in pollen coat biosynthesis and investigating the biosynthetic pathway components of capsaicin in Capsicum annuum. The database is freely available at https://expression.plant.tools/.\nThe ALOG domain defines a new family of plant-specific transcription factors acting during Arabidopsis flower development\nAuthors: Rieu, P.; Beretta, V. M.; Caselli, F.; Thevenon, E.; Lucas, J.; Rizk, M.; Franchini, E.; Dumas, R.; Nanao, M. H.; Kater, M. M.; Dumas, R.; Zubieta, C.; Parcy, F.; Gregis, V.\nScore: 24.7, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545689\nThe ALOGs (Arabidopsis LIGHT-DEPENDENT SHORT HYPOCOTYLS 1 and Oryza G1) are Transcription Factors (TFs) from an evolutionarily conserved plant-specific family shown to play critical roles in meristem identity, inflorescence architecture and organ boundaries in diverse species from mosses to higher flowering plants. However, the DNA binding-specificity and molecular determinants of protein-DNA interactions of this family were uncharacterized. Using in vitro genome-wide studies, we identified the conserved DNA motif bound by ALOG proteins from the liverwort Marchantia polymorpha and the flowering plants Arabidopsis, tomato and rice. In order to determine the amino acids responsible for DNA-binding specificity, we solved the 2.1[A] structure of the ALOG DNA binding domain in complex with its cognate DNA. The ALOG DBD is an all-alpha helical domain with a structural zinc ribbon insertion and an N-terminal disordered NLS. The NLS sequence forms an integral part of the DNA binding domain and contributes to direct base read-out. To define the function of a group of redundant ALOG proteins in the model plant Arabidopsis thaliana, we generated a series of alog mutants and uncovered their participation in a gene regulatory network involving the other floral regulators LEAFY, BLADE-ON-PETIOLE and PUCHI, all active in defining boundary regions between reproductive meristems and repressing bracts development. Taken together, this work provides the biochemical and structural basis for DNA-binding specificity of an evolutionarily conserved TF family and reveals its role as a key player in defining organ boundaries in Arabidopsis. Significance StatementTranscription Factors (TFs) are key proteins that bind specific regions in the genome and regulate the expression of associated genes. Not all organisms possess the same set of TFs and some, like the ALOGs, are specific to the plant kingdom. These TFs have been shown to play important roles from mosses to flowering plants. However, it was not known what DNA motif they recognize and how they bind DNA. Here we identify this motif, we show it is widely conserved in evolution and we understand how this new type of DNA binding domain works at the structural level. In addition, we also show that several ALOG genes from Arabidopsis share a redundant function within the genetic network underlying correct floral meristem development.\nWound induced small-peptide mediated signalling cascade regulated by OsPSKR, dictates balance between growth and defense in rice\nAuthors: Harshith, C. Y.; Pal, A.; Chakraborty, M.; NAIR, A. K.; Raju, S.; Shivaprasad, P. V.\nScore: 19.0, Published: 2023-06-22 DOI: 10.1101/2023.06.20.545841\nWounding is a general stress in plants that results from various pest and pathogenic infections in addition to environment induced mechanical damages. Plants have sophisticated molecular mechanisms to recognize and respond to pests and pathogens. Although several molecules such as phytohormones, peptides and receptors have been attributed to wound responses in dicots, such mechanisms for monocots probably having distinct wound responses are less understood. Here, we show the involvement of two distinct categories of temporally separated, endogenously derived peptides, namely, plant elicitor peptides (PEPs) and phytosulfokine (PSK), that mediate wound responses in rice. These peptides trigger a dynamic signal relay in which a novel receptor kinase named OsPSKR played a major role. OsPSKR perceived PSK ligand, acting in association with a co-receptor OsSERK1, to activate downstream responses in a kinase activity-dependent manner. Perturbation of OsPSKR expression in rice led to compromised development and constitutive autoimmune phenotypes. These results suggested that OsPSKR maintains the trade-off between growth and exaggerated defense responses, both during homeostasis and wounding. Collectively, these findings indicate the presence of a stepwise peptide-mediated signal relay that regulates the transition from defense to growth upon wounding in monocots. One line summaryEndogenous peptide signalling initiated wound responses through a receptor-like kinase OsPSKR to maintain the balance between growth and defense responses.\nThe molecular basis of parental conflict driven regulation of endosperm cellularization\nAuthors: Kohler, C.; Butel, N.; Qiu, Y.; Xu, W.; Santos-Gonzalez, J.\nScore: 16.9, Published: 2023-06-22 DOI: 10.1101/2023.06.22.546051\nThe endosperm is a seed tissue supporting embryo growth, similar to the placenta in mammals. It originates after fertilization of the maternal central cell by one of the paternal sperm cells. In the early stages of Arabidopsis thaliana seed development, nuclei divisions in the endosperm are not followed by cellularization. After a defined number of mitotic cycles, the endosperm cellularizes and stops dividing. The timing of endosperm cellularization impacts on final seed size and is differentially controlled by maternal and paternal genome contributions. While increased maternal genome dosage causes early endosperm cellularization and the formation of small seeds, the opposite is caused by increased paternal genome dosage. The parental factors controlling the differential timing of endosperm cellularization remain largely unexplored. Here, we show that a family of maternally expressed auxin response factors (ARFs) promotes endosperm cellularization and regulates final seed size. One-Sentence SummaryEndosperm cellularization is under antagonistic parental control that converges on Auxin Response Factors.\nThe receptor-like kinase BIR1 inhibits elicitor-induced plasmodesmata callose deposition and PTI gene expression and requires EDS1 and SOBIR1 to cause dose-dependent cell-death in Arabidopsis\nAuthors: GUZMAN-BENITO, I.; ROBINSON, C.; HUA, C.; SEDE, A. R.; ELVIRA-GONZALEZ, L.; PUNZON, I.; HEINLEIN, M.; NURNBERGER, T.; LLAVE, C.\nScore: 14.9, Published: 2023-06-23 DOI: 10.1101/2023.06.23.546234\nO_LIThe receptor-like kinase BIR1 functions as a negative regulator of cell death and defense in Arabidopsis. Previous studies showed that BIR1 expression is up-regulated during infections with microbes and viruses. However, the biological consequences of BIR1 induction remain unknown. C_LIO_LIHere, we use a DEX-inducible expression system in Arabidopsis to investigate the outputs associated with physiological and non-physiological levels of BIR1 expression. C_LIO_LIWe show that BIR1 induction at physiological levels significantly interferes with gene expression and plasmodesmata callose deposition triggered by canonical PTI elicitors. We found that plants that accumulated non-physiological doses of BIR1 displayed morphological defects that concur with transcriptomic changes in multiple plant defense genes. We provide experimental evidence that EDS1 and SOBIR1 are required for the ETI-type cell death phenotypes associated to non-physiological levels of BIR1 C_LIO_LIHere we propose that BIR1 induction may represent a pathogen-triggered mechanism to modulate plant defenses during infection. Our model posits that when BIR1 regulation is lost, BIR1 integrity is sensed by one or several guarding resistance (R) proteins to initiate an ETI-like response, in which SOBIR1 cooperates with EDS1 to transduce signals downstream of R proteins. C_LI\nCRISPR/Cas9 generated MeSWEET10a mutants show reduced susceptibility to cassava bacterial blight and produce viable seed\nAuthors: Elliott, K.; Veley, K. M.; Jensen, G.; Gilbert, K. B.; Norton, J.; Kambic, L.; Yoder, M.; Weil, A.; Motomura-Wages, S.; Bart, R.\nScore: 13.2, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545993\nBacteria from the genus Xanthomonas are prolific phytopathogens that elicit disease in over 400 plant species. Xanthomonads carry a repertoire of specialized proteins called transcription activator-like (TAL) effectors that promote disease and pathogen virulence by inducing expression of host susceptibility (S) genes. Xanthomonas phaseoli pv. manihotis (Xpm) causes bacterial blight on the staple food crop, cassava. The Xpm effector, TAL20, induces ectopic expression of the S gene, MeSWEET10a, a sugar transporter that contributes to cassava bacterial blight susceptibility. We used CRISPR/Cas9 to generate multiple cassava lines with edits to the MeSWEET10a TAL20 effector binding site and/or coding sequence. In several of the regenerated lines, MeSWEET10a expression was no longer induced by Xpm and in these cases, we observed reduced cassava bacterial blight disease symptoms post Xpm infection. MeSWEET10a is expressed in cassava flowers. Therefore, we investigated flower development and reproductive function in an MeSWEET10a mutant line. We found that the MeSWEET10a mutant produced phenotypically wildtype cassava flowers and viable F1 seed. Thus, blocking MeSWEET10a induction is a viable strategy for decreasing cassava susceptibility to CBB.\nGenomic characterization of a nematode tolerance locus in sugar beet\nAuthors: Sielemann, K.; Pucker, B.; Orsini, E.; Elashry, A.; Schulte, L.; Viehoever, P.; Mueller, A. E.; Schechert, A.; Weisshaar, B.; Holtgraewe, D.\nScore: 11.9, Published: 2023-06-22 DOI: 10.1101/2023.06.22.546034\nInfection by beet cyst nematodes (BCN, Heterodera schachtii) causes a serious disease of sugar beet, and climatic change is expected to improve the conditions for BCN infection. Yield and yield stability under adverse conditions are among the main breeding objectives. Breeding of BCN tolerant sugar beet cultivars offering high yield in the presence of the pathogen is therefore of high relevance. To identify causal genes providing tolerance against BCN infection, we combined several experimental and bioinformatic approaches. Relevant genomic regions were detected through mapping-by-sequencing using a segregating F2 population. DNA sequencing of contrasting F2 pools and analyses of allele frequencies for variant positions identified a single genomic region which confers nematode tolerance. The genomic interval was confirmed and narrowed down by genotyping with newly developed molecular markers. To pinpoint the causal genes within the nematode tolerance locus, we generated long read-based genome sequence assemblies of the tolerant parental breeding line Strube U2Bv and the susceptible reference line 2320Bv. We analyzed continuous sequences of the locus with regard to functional gene annotation and differential gene expression upon BCN infection. A cluster of genes with similarity to the Arabidopsis thaliana gene encoding nodule inception protein-like protein 7 (NLP7) was identified. Gene expression analyses confirmed transcriptional activity and revealed clear differences between susceptible and tolerant genotypes. Our findings provide new insights into the genomic basis of plant-nematode interactions that can be used to design and accelerate novel management strategies against BCN.\nControl of stem cell behavior by CLE-JINGASA signaling in the shoot apical meristem in Marchantia polymorpha\nAuthors: Takahashi, G.; Kiyosue, T.; Hirakawa, Y.\nScore: 11.8, Published: 2023-06-23 DOI: 10.1101/2023.06.23.545176\nLand plants undergo indeterminate growth by the activity of meristems in both gametophyte (haploid) and sporophyte (diploid) generations1-3. In the sporophyte of the flowering plant Arabidopsis thaliana, the apical meristems are located at the shoot and root tips, in which a number of regulatory gene homologs are shared for their development, implying deep evolutionary origins4-7. However, little is known about their functional conservation with gametophytic meristems in distantly related land plants such as bryophytes8-17, even though genomic studies have revealed the subfamily-level diversity of regulatory genes is mostly conserved throughout land plants18-23. Here we show that a NAM/ATAF/CUC (NAC) domain transcription factor, JINGASA (MpJIN), acts downstream of CLV3/ESR-related (CLE) peptide signaling and controls stem cell behavior in the gametophytic shoot apical meristem of the liverwort Marchantia polymorpha. We found that MpJIN shows heterogeneous expression in space and time within the stem cell zone, associated with the loss and de novo specification of stem cell identity at the branching event. Consistent with the expression pattern, induction of MpJIN results in ectopic periclinal cell division in the stem cell zone and meristem termination. Comparative expression analysis suggests that the function of JIN/FEZ subfamily genes was shared between the shoot apical meristems in the gametophyte and sporophyte generations in early land plants but was lost in certain lineages including the flowering plant A. thaliana.\nITPK1 regulates jasmonate-controlled root development in Arabidopsis thaliana\nAuthors: Laha, D.; Pullagurla, N. J.; Shome, S.; Yadav, R.\nScore: 9.5, Published: 2023-06-20 DOI: 10.1101/2023.06.16.545325\nJasmonic acid (JA) is a plant hormone that regulates a plethora of physiological processes including immunity and development and is perceived by the F-Box protein, Coronatine-insensitive protein 1(COI1). The discovery of inositol phosphates (InsPs) in the COI1 receptor complex highlights their role in JA perception. InsPs are phosphate-rich signaling molecules that control many aspects of plant physiology. Inositol pyrophosphates (PP-InsPs) are diphosphate containing InsP species of which InsP7 and InsP8 are the best characterized ones. Different InsP and PP-InsP species are linked with JA-related plant immunity. However, role of PP-InsP species in regulating JA-dependent developmental processes are poorly understood. Recent identification of ITPK1 kinase responsible for the production of 5-InsP7 from InsP6 in planta provides a platform to interrogate possible involvement of ITPK-derived InsP species in JA-related plant development. Herein this study, we report that ITPK1-defective plants exhibit increased root growth inhibition to bioactive JA treatment. The itpk1 plants also show increased lateral root density when treated with JA. Notably, JA treatment does not induce ITPK1 protein level. Gene expression analyses revealed that JA-biosynthetic genes are not differentially expressed in the ITPK1-deficient plants. We further demonstrate that genes encoding different JAZ repressor proteins are severely downregulated in the ITPK1-defective plants. Taken together, our study highlights the role of ITPK1 in regulating JA-dependent root architecture development through controlling expression of different JAZ repressor proteins.\n",
  "wordCount" : "2479",
  "inLanguage": "en",
  "datePublished": "2023-06-28T09:51:07Z",
  "dateModified": "2023-06-28T09:51:07Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://yorks0n.github.io/TRxiv2/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://yorks0n.github.io/TRxiv2/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://yorks0n.github.io/TRxiv2" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta"><span>updated on June 28, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.17.545419">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.17.545419" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.17.545419">
        <p class="paperTitle">Mapping the landscape of transcription factor promoter activity during vegetative development in Marchantia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.17.545419" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.17.545419" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Romani, F.; Sauret-Güeto, S.; Rebmann, M.; Annese, D.; Bonter, I.; Tomaselli, M.; Dierschke, T.; Delmans, M.; Frangedakis, E.; Silvestri, L.; Rever, J.; Bowman, J. L.; Romani, I.; Haseloff, J.</p>
        <p class="info">Score: 46.0, Published: 2023-06-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.17.545419' target='https://doi.org/10.1101/2023.06.17.545419'> 10.1101/2023.06.17.545419</a></p>
        <p class="abstract">Transcription factors (TFs) are essential for the regulation of gene expression and cell fate determination. Characterising the transcriptional activity of TF genes in space and time is a critical step towards understanding complex biological systems. The vegetative gametophyte meristems of bryophytes share some characteristics with the shoot-apical meristems of flowering plants. However, the identity and expression profiles of TFs associated with gametophyte organization are largely unknown. With only [~]450 TF genes, Marchantia polymorpha is an outstanding model system for plant systems biology. We have generated a near-complete collection of promoter elements derived from Marchantia TF genes. We experimentally tested in planta reporter fusions for all the TF promoters in the collection and systematically analysed expression patterns in Marchantia gemmae. This allowed us to build a map of precise expression domains and identify a unique set of TFs expressed in the stem-cell zone, providing new insight into the dynamic regulation of the gametophytic meristem and its evolution. In addition, we provide an online database of expression patterns for all promoters in the collection. We expect that the promoter elements characterised here will be useful for cell-type specific expression, synthetic biology applications, and functional genomics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546049">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546049" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546049">
        <p class="paperTitle">Plant Expression Omnibus - a comparative transcriptomic database for 103 Archaeplastida</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546049" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546049" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koh, E.; Goh, W.; Villanueva, E.; Julca, I.; Mutwil, M.</p>
        <p class="info">Score: 29.9, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546049' target='https://doi.org/10.1101/2023.06.22.546049'> 10.1101/2023.06.22.546049</a></p>
        <p class="abstract">The Plant Expression Omnibus (PEO) is a web application that provides biologists with access to gene expression insights across over 100 plant species, [~]60,000 manually annotated RNA-seq samples, and more than four million genes. The tool allows users to explore the expression patterns of genes across different organs, identify organ-specific genes, and discover top co-expressed genes for any gene of interest. PEO also provides functional annotations for each gene, allowing for the identification of genetic modules and pathways. PEO is designed to facilitate comparative kingdom-wide gene expression analysis and provide a valuable resource for plant biology research. We provide two case studies to demonstrate the utility of PEO in identifying candidate genes in pollen coat biosynthesis and investigating the biosynthetic pathway components of capsaicin in Capsicum annuum. The database is freely available at https://expression.plant.tools/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545689">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545689" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545689">
        <p class="paperTitle">The ALOG domain defines a new family of plant-specific transcription factors acting during Arabidopsis flower development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545689" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545689" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rieu, P.; Beretta, V. M.; Caselli, F.; Thevenon, E.; Lucas, J.; Rizk, M.; Franchini, E.; Dumas, R.; Nanao, M. H.; Kater, M. M.; Dumas, R.; Zubieta, C.; Parcy, F.; Gregis, V.</p>
        <p class="info">Score: 24.7, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545689' target='https://doi.org/10.1101/2023.06.21.545689'> 10.1101/2023.06.21.545689</a></p>
        <p class="abstract">The ALOGs (Arabidopsis LIGHT-DEPENDENT SHORT HYPOCOTYLS 1 and Oryza G1) are Transcription Factors (TFs) from an evolutionarily conserved plant-specific family shown to play critical roles in meristem identity, inflorescence architecture and organ boundaries in diverse species from mosses to higher flowering plants. However, the DNA binding-specificity and molecular determinants of protein-DNA interactions of this family were uncharacterized. Using in vitro genome-wide studies, we identified the conserved DNA motif bound by ALOG proteins from the liverwort Marchantia polymorpha and the flowering plants Arabidopsis, tomato and rice. In order to determine the amino acids responsible for DNA-binding specificity, we solved the 2.1[A] structure of the ALOG DNA binding domain in complex with its cognate DNA. The ALOG DBD is an all-alpha helical domain with a structural zinc ribbon insertion and an N-terminal disordered NLS. The NLS sequence forms an integral part of the DNA binding domain and contributes to direct base read-out. To define the function of a group of redundant ALOG proteins in the model plant Arabidopsis thaliana, we generated a series of alog mutants and uncovered their participation in a gene regulatory network involving the other floral regulators LEAFY, BLADE-ON-PETIOLE and PUCHI, all active in defining boundary regions between reproductive meristems and repressing bracts development. Taken together, this work provides the biochemical and structural basis for DNA-binding specificity of an evolutionarily conserved TF family and reveals its role as a key player in defining organ boundaries in Arabidopsis.

Significance StatementTranscription Factors (TFs) are key proteins that bind specific regions in the genome and regulate the expression of associated genes. Not all organisms possess the same set of TFs and some, like the ALOGs, are specific to the plant kingdom. These TFs have been shown to play important roles from mosses to flowering plants. However, it was not known what DNA motif they recognize and how they bind DNA. Here we identify this motif, we show it is widely conserved in evolution and we understand how this new type of DNA binding domain works at the structural level. In addition, we also show that several ALOG genes from Arabidopsis share a redundant function within the genetic network underlying correct floral meristem development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545841">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545841" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545841">
        <p class="paperTitle">Wound induced small-peptide mediated signalling cascade regulated by OsPSKR, dictates balance between growth and defense in rice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545841" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545841" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harshith, C. Y.; Pal, A.; Chakraborty, M.; NAIR, A. K.; Raju, S.; Shivaprasad, P. V.</p>
        <p class="info">Score: 19.0, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545841' target='https://doi.org/10.1101/2023.06.20.545841'> 10.1101/2023.06.20.545841</a></p>
        <p class="abstract">Wounding is a general stress in plants that results from various pest and pathogenic infections in addition to environment induced mechanical damages. Plants have sophisticated molecular mechanisms to recognize and respond to pests and pathogens. Although several molecules such as phytohormones, peptides and receptors have been attributed to wound responses in dicots, such mechanisms for monocots probably having distinct wound responses are less understood. Here, we show the involvement of two distinct categories of temporally separated, endogenously derived peptides, namely, plant elicitor peptides (PEPs) and phytosulfokine (PSK), that mediate wound responses in rice. These peptides trigger a dynamic signal relay in which a novel receptor kinase named OsPSKR played a major role. OsPSKR perceived PSK ligand, acting in association with a co-receptor OsSERK1, to activate downstream responses in a kinase activity-dependent manner. Perturbation of OsPSKR expression in rice led to compromised development and constitutive autoimmune phenotypes. These results suggested that OsPSKR maintains the trade-off between growth and exaggerated defense responses, both during homeostasis and wounding. Collectively, these findings indicate the presence of a stepwise peptide-mediated signal relay that regulates the transition from defense to growth upon wounding in monocots.

One line summaryEndogenous peptide signalling initiated wound responses through a receptor-like kinase OsPSKR to maintain the balance between growth and defense responses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546051">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546051" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546051">
        <p class="paperTitle">The molecular basis of parental conflict driven regulation of endosperm cellularization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546051" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546051" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kohler, C.; Butel, N.; Qiu, Y.; Xu, W.; Santos-Gonzalez, J.</p>
        <p class="info">Score: 16.9, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546051' target='https://doi.org/10.1101/2023.06.22.546051'> 10.1101/2023.06.22.546051</a></p>
        <p class="abstract">The endosperm is a seed tissue supporting embryo growth, similar to the placenta in mammals. It originates after fertilization of the maternal central cell by one of the paternal sperm cells. In the early stages of Arabidopsis thaliana seed development, nuclei divisions in the endosperm are not followed by cellularization. After a defined number of mitotic cycles, the endosperm cellularizes and stops dividing. The timing of endosperm cellularization impacts on final seed size and is differentially controlled by maternal and paternal genome contributions. While increased maternal genome dosage causes early endosperm cellularization and the formation of small seeds, the opposite is caused by increased paternal genome dosage. The parental factors controlling the differential timing of endosperm cellularization remain largely unexplored. Here, we show that a family of maternally expressed auxin response factors (ARFs) promotes endosperm cellularization and regulates final seed size.

One-Sentence SummaryEndosperm cellularization is under antagonistic parental control that converges on Auxin Response Factors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546234">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546234" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546234">
        <p class="paperTitle">The receptor-like kinase BIR1 inhibits elicitor-induced plasmodesmata callose deposition and PTI gene expression and requires EDS1 and SOBIR1 to cause dose-dependent cell-death in Arabidopsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546234" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546234" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: GUZMAN-BENITO, I.; ROBINSON, C.; HUA, C.; SEDE, A. R.; ELVIRA-GONZALEZ, L.; PUNZON, I.; HEINLEIN, M.; NURNBERGER, T.; LLAVE, C.</p>
        <p class="info">Score: 14.9, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546234' target='https://doi.org/10.1101/2023.06.23.546234'> 10.1101/2023.06.23.546234</a></p>
        <p class="abstract">O_LIThe receptor-like kinase BIR1 functions as a negative regulator of cell death and defense in Arabidopsis. Previous studies showed that BIR1 expression is up-regulated during infections with microbes and viruses. However, the biological consequences of BIR1 induction remain unknown.
C_LIO_LIHere, we use a DEX-inducible expression system in Arabidopsis to investigate the outputs associated with physiological and non-physiological levels of BIR1 expression.
C_LIO_LIWe show that BIR1 induction at physiological levels significantly interferes with gene expression and plasmodesmata callose deposition triggered by canonical PTI elicitors. We found that plants that accumulated non-physiological doses of BIR1 displayed morphological defects that concur with transcriptomic changes in multiple plant defense genes. We provide experimental evidence that EDS1 and SOBIR1 are required for the ETI-type cell death phenotypes associated to non-physiological levels of BIR1
C_LIO_LIHere we propose that BIR1 induction may represent a pathogen-triggered mechanism to modulate plant defenses during infection. Our model posits that when BIR1 regulation is lost, BIR1 integrity is sensed by one or several guarding resistance (R) proteins to initiate an ETI-like response, in which SOBIR1 cooperates with EDS1 to transduce signals downstream of R proteins.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545993">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545993" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545993">
        <p class="paperTitle">CRISPR/Cas9 generated MeSWEET10a mutants show reduced susceptibility to cassava bacterial blight and produce viable seed</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545993" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545993" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Elliott, K.; Veley, K. M.; Jensen, G.; Gilbert, K. B.; Norton, J.; Kambic, L.; Yoder, M.; Weil, A.; Motomura-Wages, S.; Bart, R.</p>
        <p class="info">Score: 13.2, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545993' target='https://doi.org/10.1101/2023.06.21.545993'> 10.1101/2023.06.21.545993</a></p>
        <p class="abstract">Bacteria from the genus Xanthomonas are prolific phytopathogens that elicit disease in over 400 plant species. Xanthomonads carry a repertoire of specialized proteins called transcription activator-like (TAL) effectors that promote disease and pathogen virulence by inducing expression of host susceptibility (S) genes. Xanthomonas phaseoli pv. manihotis (Xpm) causes bacterial blight on the staple food crop, cassava. The Xpm effector, TAL20, induces ectopic expression of the S gene, MeSWEET10a, a sugar transporter that contributes to cassava bacterial blight susceptibility. We used CRISPR/Cas9 to generate multiple cassava lines with edits to the MeSWEET10a TAL20 effector binding site and/or coding sequence. In several of the regenerated lines, MeSWEET10a expression was no longer induced by Xpm and in these cases, we observed reduced cassava bacterial blight disease symptoms post Xpm infection. MeSWEET10a is expressed in cassava flowers. Therefore, we investigated flower development and reproductive function in an MeSWEET10a mutant line. We found that the MeSWEET10a mutant produced phenotypically wildtype cassava flowers and viable F1 seed. Thus, blocking MeSWEET10a induction is a viable strategy for decreasing cassava susceptibility to CBB.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546034">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546034" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546034">
        <p class="paperTitle">Genomic characterization of a nematode tolerance locus in sugar beet</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546034" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546034" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sielemann, K.; Pucker, B.; Orsini, E.; Elashry, A.; Schulte, L.; Viehoever, P.; Mueller, A. E.; Schechert, A.; Weisshaar, B.; Holtgraewe, D.</p>
        <p class="info">Score: 11.9, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546034' target='https://doi.org/10.1101/2023.06.22.546034'> 10.1101/2023.06.22.546034</a></p>
        <p class="abstract">Infection by beet cyst nematodes (BCN, Heterodera schachtii) causes a serious disease of sugar beet, and climatic change is expected to improve the conditions for BCN infection. Yield and yield stability under adverse conditions are among the main breeding objectives. Breeding of BCN tolerant sugar beet cultivars offering high yield in the presence of the pathogen is therefore of high relevance. To identify causal genes providing tolerance against BCN infection, we combined several experimental and bioinformatic approaches. Relevant genomic regions were detected through mapping-by-sequencing using a segregating F2 population. DNA sequencing of contrasting F2 pools and analyses of allele frequencies for variant positions identified a single genomic region which confers nematode tolerance. The genomic interval was confirmed and narrowed down by genotyping with newly developed molecular markers. To pinpoint the causal genes within the nematode tolerance locus, we generated long read-based genome sequence assemblies of the tolerant parental breeding line Strube U2Bv and the susceptible reference line 2320Bv. We analyzed continuous sequences of the locus with regard to functional gene annotation and differential gene expression upon BCN infection. A cluster of genes with similarity to the Arabidopsis thaliana gene encoding nodule inception protein-like protein 7 (NLP7) was identified. Gene expression analyses confirmed transcriptional activity and revealed clear differences between susceptible and tolerant genotypes. Our findings provide new insights into the genomic basis of plant-nematode interactions that can be used to design and accelerate novel management strategies against BCN.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.545176">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.545176" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.545176">
        <p class="paperTitle">Control of stem cell behavior by CLE-JINGASA signaling in the shoot apical meristem in Marchantia polymorpha</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.545176" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.545176" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Takahashi, G.; Kiyosue, T.; Hirakawa, Y.</p>
        <p class="info">Score: 11.8, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.545176' target='https://doi.org/10.1101/2023.06.23.545176'> 10.1101/2023.06.23.545176</a></p>
        <p class="abstract">Land plants undergo indeterminate growth by the activity of meristems in both gametophyte (haploid) and sporophyte (diploid) generations1-3. In the sporophyte of the flowering plant Arabidopsis thaliana, the apical meristems are located at the shoot and root tips, in which a number of regulatory gene homologs are shared for their development, implying deep evolutionary origins4-7. However, little is known about their functional conservation with gametophytic meristems in distantly related land plants such as bryophytes8-17, even though genomic studies have revealed the subfamily-level diversity of regulatory genes is mostly conserved throughout land plants18-23. Here we show that a NAM/ATAF/CUC (NAC) domain transcription factor, JINGASA (MpJIN), acts downstream of CLV3/ESR-related (CLE) peptide signaling and controls stem cell behavior in the gametophytic shoot apical meristem of the liverwort Marchantia polymorpha. We found that MpJIN shows heterogeneous expression in space and time within the stem cell zone, associated with the loss and de novo specification of stem cell identity at the branching event. Consistent with the expression pattern, induction of MpJIN results in ectopic periclinal cell division in the stem cell zone and meristem termination. Comparative expression analysis suggests that the function of JIN/FEZ subfamily genes was shared between the shoot apical meristems in the gametophyte and sporophyte generations in early land plants but was lost in certain lineages including the flowering plant A. thaliana.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.16.545325">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.16.545325" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.16.545325">
        <p class="paperTitle">ITPK1 regulates jasmonate-controlled root development in Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.16.545325" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.16.545325" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Laha, D.; Pullagurla, N. J.; Shome, S.; Yadav, R.</p>
        <p class="info">Score: 9.5, Published: 2023-06-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.16.545325' target='https://doi.org/10.1101/2023.06.16.545325'> 10.1101/2023.06.16.545325</a></p>
        <p class="abstract">Jasmonic acid (JA) is a plant hormone that regulates a plethora of physiological processes including immunity and development and is perceived by the F-Box protein, Coronatine-insensitive protein 1(COI1). The discovery of inositol phosphates (InsPs) in the COI1 receptor complex highlights their role in JA perception. InsPs are phosphate-rich signaling molecules that control many aspects of plant physiology. Inositol pyrophosphates (PP-InsPs) are diphosphate containing InsP species of which InsP7 and InsP8 are the best characterized ones. Different InsP and PP-InsP species are linked with JA-related plant immunity. However, role of PP-InsP species in regulating JA-dependent developmental processes are poorly understood. Recent identification of ITPK1 kinase responsible for the production of 5-InsP7 from InsP6 in planta provides a platform to interrogate possible involvement of ITPK-derived InsP species in JA-related plant development. Herein this study, we report that ITPK1-defective plants exhibit increased root growth inhibition to bioactive JA treatment. The itpk1 plants also show increased lateral root density when treated with JA. Notably, JA treatment does not induce ITPK1 protein level. Gene expression analyses revealed that JA-biosynthetic genes are not differentially expressed in the ITPK1-deficient plants. We further demonstrate that genes encoding different JAZ repressor proteins are severely downregulated in the ITPK1-defective plants. Taken together, our study highlights the role of ITPK1 in regulating JA-dependent root architecture development through controlling expression of different JAZ repressor proteins.</p>
      </div>
    </div>
  </div>
</div>









<script src="/TRxiv2/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://yorks0n.github.io/TRxiv2">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
