<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment
Authors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium, ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment
Authors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium, ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-15T10:40:26+00:00" />
<meta property="article:modified_time" content="2023-11-15T10:40:26+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment
Authors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium, ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment\nAuthors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium, ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J.",
  "keywords": [
    
  ],
  "articleBody": " The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment\nAuthors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium, ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J. H.; Schmutz, J.; Webber, J.; Grimwood, J.; Jacquet, C.; Dunand, C.; Nelson, J.; Roux, F.; Philippe, H.; Schornack, S.; Bonhomme, M.; Delaux, P.-M.\nScore: 65.8, Published: 2023-10-29 DOI: 10.1101/2023.10.27.564390\nPlant adaptation to a terrestrial life 450 million years ago played a major role in the evolution of life on Earth. This shift from an aquatic environment has been mostly studied by focusing on flowering plants. Here, we gathered a collection of 133 accessions of the non-vascular plants Marchantia polymorpha and studied its intraspecific diversity using selection signature analyses, genome-environment association study and a gene-centered pangenome. We identified adaptive features shared with flowering plants, such as peroxidases or nucleotide-binding and leucine-rich repeat (NLR), which likely played a role in the adaptation of the first land plants to the terrestrial habitat. The M. polymorpha pangenome also harbored lineage-specific accessory genes absent from seed plants. We conclude that different land plants lineages still share many elements from the genetic toolkit evolved by their most recent common ancestor to adapt to the terrestrial habitat, refined by lineage specific polymorphisms and gene family evolutions.\nAn Arabidopsis leaf expression atlas across diurnal and developmental scales\nAuthors: Vong, G.; McCarthy, K.; Claydon, W.; Davis, S. J.; Redmond, E. J.; Ezer, D.\nScore: 10.3, Published: 2023-11-13 DOI: 10.1101/2023.11.10.566572\nMature plant leaves are a composite of distinct cell types, including epidermal, mesophyll and vascular cells. Notably the proportion of these cells, and the relative transcript concentrations within different cell types may change over time. While gene expression data at a single-cell level can provide cell-type specific expression values, it is often too expensive to perform this on high resolution time series. Although bulk RNA-seq can be performed in a high resolution time series, the RNA-seq in whole leaves measures the average gene expression values across all cell types in each sample. In this study, we combined single cell RNA-seq data with time-series data from whole leaves to infer an atlas of cell type-specific gene expression changes over time for Arabidopsis thaliana. We inferred how relative transcript concentrations of cell types vary across diurnal and developmental time scales. Importantly this analysis revealed three sub-groups of mesophyll cells that have distinct temporal profiles of expression. Finally, we develop tissue-specific gene networks that form a new community resource: An Arabidopsis Leaf Time-Dependent Atlas (AraLeTa), which allows users to extract gene networks that are confirmed by transcription factor binding data and specific to certain cell types, at certain times of day and certain developmental stages, which is available at: https://regulatorynet.shinyapps.io/araleta/.\nGibberellin Perception Sensors 1 and 2 reveal cellular GA dynamics articulated by COP1 and GA20ox1 that are necessary but not sufficient to pattern hypocotyl cell elongation\nAuthors: Griffiths, J.; Rizza, A.; Tang, B.; Frommer, W. B.; Jones, A. M.\nScore: 9.9, Published: 2023-11-08 DOI: 10.1101/2023.11.06.565859\nThe phytohormone gibberellin (GA) is critical for environmentally sensitive plant development including germination, skotomorphogenesis and flowering. The FRET biosensor GIBBERELLIN PERCEPTION SENSOR1, which permits single-cell GA measurements in vivo, was previously used to observe a GA gradient correlated with cell length in dark-grown but not light-grown hypocotyls. We sought to understand how light signalling integrates into cellular GA regulation. Here we show how the E3 ligase COP1 and transcription factor HY5 play central roles in directing cellular GA distribution in skoto- and photomorphogenic hypocotyls, respectively. We demonstrate that the expression pattern of biosynthetic enzyme GA20ox1 is the key determinant of the GA gradient in dark-grown hypocotyls and is a target of COP1 signalling. We engineered a second generation GPS2 biosensor with improved orthogonality and reversibility to show the cellular pattern of GA depletion during the transition to growth in the light. This GA depletion partly explains the resetting of hypocotyl growth dynamics during photomorphogenesis. Achieving cell-level resolution has revealed how GA distributions link environmental conditions with morphology and morphological plasticity and the GPS2 biosensor is an ideal tool for GA studies in further conditions, organs and plant species.\nRepression of pattern-triggered immune responses by hypoxia\nAuthors: Mooney, B. C.; Doorly, C. M.; Mantz, M.; Garcia, P.; Huesgen, P. F.; Graciet, E.\nScore: 11.8, Published: 2023-11-07 DOI: 10.1101/2023.11.07.565979\nCombined abiotic/biotic stresses frequently occur in nature. Yet, relatively little is known about how plants regulate the crosstalk between stress response pathways to trigger a coordinated response to combined stresses. Protein degradation by the ubiquitin/proteasome system (UPS) is central to the regulation of plant responses to multiple stresses, including the crosstalk between pathways. The Arg/N-degron pathway, a subset of the UPS, targets proteins based on their N-termini. In plants, it has been implicated in the responses to both biotic and abiotic stresses, while being a key regulator of hypoxia response through the degradation of ERF-VII transcription factors, which orchestrate the onset of the hypoxia response program. Because of its central position in regulating multiple (a)biotic stresses, we hypothesized that the Arg/N-degron pathway could act in the crosstalk between abiotic/biotic stresses. In exploring this possibility using the model pathogen-associated molecular pattern flg22 to elicit pattern-triggered immunity (PTI), we uncovered a link between the transcriptional response programs to hypoxia and to flg22. Combined hypoxia/flg22 treatments further showed that hypoxia represses the flg22 transcriptional program, as well as the expression of pattern recognition receptors, MAPK signalling and callose deposition during PTI, through mechanisms that are at least partly dependent on the ERF-VIIs. These findings are of relevance to understanding the trade-offs between plant responses to combined abiotic/biotic stresses in the context of our efforts to increase crop resilience to global climate change. Our results also show that the well-known repressive effect of hypoxia on innate immunity in animals also applies to plants. Significance statementUnderstanding how plants regulate the crosstalk between stress response pathways is key to our efforts to increase crop resilience and mitigate yield losses caused by global climate change. Despite the urgency to do so, relatively little remains known about how plants respond to combined stresses, which frequently occur in nature. Here, we show that the hypoxia response program and the basal layer of plant immunity (pattern-triggered immunity or PTI) share components. Our data also show that hypoxia represses several key aspects of the plants innate immune response, a situation akin to that discovered in animals decades ago. These findings have implications for our ability to develop resilient crops by limiting the negative trade-offs that exist between hypoxia response and immunity.\nJAGGER localization and function is dependent on GPI anchor addition\nAuthors: Figueiredo, R.; Costa, M.; Moreira, D.; Moreira, M.; Noble, J.; Pereira, L. G.; Melo, P.; Palanivelu, R.; Coimbra, S.; Pereira, A. M.\nScore: 8.2, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565777\nIn flowering plants, successful double fertilization requires the correct delivery of two sperm cells to the female gametophyte inside the ovule. The delivery of a single pair of sperm cells is achieved by the entrance of a single pollen tube into one female gametophyte. To prevent polyspermy, Arabidopsis ovules avoid the attraction of multiples pollen tubes to one ovule - polytubey block. In Arabidopsis jagger mutants, a significant number of ovules attract more than one pollen tube to an ovule due to an impairment in synergid degeneration. JAGGER encodes a putative arabinogalactan protein (AGP) which is predicted to be anchored to the plasma membrane by a glycosylphosphatidylinositol (GPI) anchor. Here, we show that JAGGER fused to citrine yellow fluorescent protein (JAGGER-cYFP) is functional and localizes mostly to the periphery of ovule integuments and transmitting tract cells. We further investigated the importance of GPI-anchor addition domains for JAGGER localization and function. Different JAGGER proteins with deletions in predicted {omega}-site regions and GAS (GPI attachment signal) domain, expected to compromise the addition of the GPI anchor, led to disruption of JAGGER localization in the cell periphery. All JAGGER proteins with disrupted localization were also not able to rescue the polytubey phenotype, pointing to the importance of GPI-anchor addition to in vivo function of the JAGGER protein.\nPhotobodies enable the phase-separation and counterbalance of opposing phytochrome B actions in PIF5 degradation and stabilization\nAuthors: Kim, R. J. A.; Fan, D.; He, J.; Kim, K.; Du, J.; Chen, M.\nScore: 7.3, Published: 2023-11-15 DOI: 10.1101/2023.11.12.566724\nPhotoactivation of the plant photoreceptor and thermosensor phytochrome B (PHYB) triggers its condensation into subnuclear photobodies (PBs). However, the function of PBs remains frustratingly elusive. Here, we show that PHYB condensation enables the co-occurrence and competition of two antagonistic phase-separated signaling actions. We found that PHYB recruits PHYTOCHROME-INTERACTING FACTOR5 (PIF5) to PBs and, surprisingly, that PHYB exerts opposing roles in degrading and stabilizing PIF5. Perturbing PB size by overproducing PHYB provoked a biphasic PIF5 response: while a moderate increase in PHYB enhanced PIF5 degradation, further elevating the PHYB level stabilized PIF5 by retaining more of it in enlarged PBs. Our results support a model in which PHYB condensation stabilizes PIF5 in PBs to counteract PIF5 degradation in the surrounding nucleoplasm, thereby enabling an environmentally sensitive counterbalancing mechanism to titrate nucleoplasmic PIF5 and its transcriptional output. This PB-enabled signaling mechanism provides a framework for regulating a plethora of PHYB-interacting signaling molecules in diverse plant environmental responses. We propose that this function of PBs represents a general function of biomolecular condensates to allow distinct variations of a cellular process or signaling pathway to coexist and interact to generate dynamically adjustable integrated outputs within a single subcellular space.\nPlant pathogenic fungi hijack phosphate starvation signaling with conserved enzymatic effectors\nAuthors: McCombe, C. L.; Wegner, A.; Zamora, C. S.; Casanova, F.; Aditya, S.; Greenwood, J. R.; Wirtz, L.; de Paula, S.; England, E.; Shang, S.; Ericsson, D. J.; Oliveira-Garcia, E.; Williams, S. J.; Schaffrath, U.\nScore: 6.4, Published: 2023-11-14 DOI: 10.1101/2023.11.14.566975\nPhosphate availability modulates plant immune function and regulates interactions with beneficial, phosphate-providing, microbes. Here, we describe the hijacking of plant phosphate sensing by a family of Nudix hydrolase effectors from pathogenic Magnaporthe oryzae and Colletotrichum fungi. Structural and enzymatic analyses of the Nudix effector family demonstrate that they selectively hydrolyze inositol pyrophosphates, a molecule used by plants to monitor phosphate status and regulate starvation responses. In M. oryzae, gene deletion and complementation experiments reveal that the enzymatic activity of a Nudix effector significantly contributes to pathogen virulence. Further, we show that this conserved effector protein family induces phosphate starvation signaling in plants. Our study elucidates a molecular mechanism, utilized by multiple phytopathogenic fungi, that manipulates the highly conserved plant phosphate sensing pathway to exacerbate disease.\nComparative mutant analyses reveal a novel mechanism of ARF regulation in land plants\nAuthors: Prigge, M. J.; Morffy, N.; De Neve, A.; Szutu, W.; Juarez, M. J. A.; Johnson, K.; Do, N.; Lavy, M.; Hake, S.; Strader, L.; Estelle, M.; Richardson, A. E.\nScore: 5.2, Published: 2023-11-13 DOI: 10.1101/2023.11.09.566459\nA major challenge in plant biology is to understand how the plant hormone auxin regulates diverse transcriptional responses throughout development, in different environments, and in different species. The answer may lie in the specific complement of auxin signaling components in each cell. The balance between activators (class-A AUXIN RESPONSE FACTORS) and repressors (class-B ARFs) is particularly important. It is unclear how this balance is achieved. Through comparative analysis of novel, dominant mutants in maize and the moss Physcomitrium patens, we have discovered a ~500-million-year-old mechanism of class-B ARF protein level regulation, important in determining cell fate decisions across land plants. Thus, our results add a key piece to the puzzle of how auxin regulates plant development.\nSingle nuclei sequencing reveals C4 photosynthesis is based on rewiring of ancestral cell identity networks\nAuthors: Swift, J.; Luginbuehl, L. H.; Schreier, T.; Donald, R. M.; Lee, T.; Nery, J.; Ecker, J. R.; Hibberd, J. M.\nScore: 61.0, Published: 2023-10-27 DOI: 10.1101/2023.10.26.562893\nIn multicellular systems changes to the patterning of gene expression drive modifications in cell function and trait evolution. One striking example is found in more than sixty plant lineages where compartmentation of photosynthesis between cell types allowed the evolution of the efficient C4 pathway from the ancestral C3 state. The molecular events enabling this transition are unclear. We used single nuclei sequencing to generate atlases for C3 rice and C4 sorghum during photomorphogenesis. Our analysis revealed that initiation of photosynthesis gene expression is conditioned by cell identity. In both species a conserved cistrome defines each cell type, and photosynthesis genes switching expression from mesophyll in rice to bundle sheath in sorghum acquire hallmarks of bundle sheath identity. The sorghum bundle sheath has also acquired gene networks associated with C3 guard cells. We conclude C4 photosynthesis is based on rewiring in cis that exapts cell identity networks of C3 plants.\nGlobally-deployed sorghum aphid resistance gene RMES1 is vulnerable to biotype shifts but being bolstered by RMES2\nAuthors: VanGessel, C.; Rice, B.; Felderhoff, T.; Charles, J. R.; Pressoir, G.; Nalam, V.; Morris, G. P.\nScore: 4.0, Published: 2023-11-08 DOI: 10.1101/2023.11.07.566092\nDurable host plant resistance (HPR) to insect pests is critical for sustainable agriculture. Natural variation exists for aphid HPR in sorghum (Sorghum bicolor) but the genetic architecture and phenotype has not been clarified for most sources. To assess the threat of a sorghum aphid (Melanaphis sorghi) biotype shift, we characterized the phenotype of Resistance to Melanaphis sorghi 1 (RMES1) and contributing HPR architecture in globally-admixed populations selected under severe aphid infestation in Haiti. We found RMES1 reduces sorghum aphid fecundity but not bird cherry-oat aphid (Rhopalosiphum padi) fecundity, suggesting a discriminant HPR response typical of gene-for-gene interaction. A second resistant gene, RMES2, were more frequent than RMES1 resistant alleles in landraces and historic breeding lines. RMES2 contributes early and mid-season aphid resistance in a segregating F2 population, however RMES1 was only significant with mid-season fitness. In a fixed population with high aphid resistance, RMES1 and RMES2 were selected for demonstrating a lack of significant antagonistic pleiotropy. Associations with resistance co-located with cyanogenic glucoside biosynthesis genes support additional HPR sources. Globally, therefore, a vulnerable HPR source (RMES1) is bolstered by a second common source of resistance in breeding programs (RMES2) which may be staving off a biotype shift. HIGHLIGHTThe globally-deployed sorghum aphid resistance gene, RMES1, reduces aphid reproduction and therefore is vulnerable to a biotype shift. A second major gene, RMES2, and cyanogenesis may increase global durability of resistance.\n",
  "wordCount" : "2404",
  "inLanguage": "en",
  "datePublished": "2023-11-15T10:40:26Z",
  "dateModified": "2023-11-15T10:40:26Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 15, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564390">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564390" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564390">
        <p class="paperTitle">The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564390" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564390" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium,  ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J. H.; Schmutz, J.; Webber, J.; Grimwood, J.; Jacquet, C.; Dunand, C.; Nelson, J.; Roux, F.; Philippe, H.; Schornack, S.; Bonhomme, M.; Delaux, P.-M.</p>
        <p class="info">Score: 65.8, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564390' target='https://doi.org/10.1101/2023.10.27.564390'> 10.1101/2023.10.27.564390</a></p>
        <p class="abstract">Plant adaptation to a terrestrial life 450 million years ago played a major role in the evolution of life on Earth. This shift from an aquatic environment has been mostly studied by focusing on flowering plants. Here, we gathered a collection of 133 accessions of the non-vascular plants Marchantia polymorpha and studied its intraspecific diversity using selection signature analyses, genome-environment association study and a gene-centered pangenome. We identified adaptive features shared with flowering plants, such as peroxidases or nucleotide-binding and leucine-rich repeat (NLR), which likely played a role in the adaptation of the first land plants to the terrestrial habitat. The M. polymorpha pangenome also harbored lineage-specific accessory genes absent from seed plants. We conclude that different land plants lineages still share many elements from the genetic toolkit evolved by their most recent common ancestor to adapt to the terrestrial habitat, refined by lineage specific polymorphisms and gene family evolutions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566572">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566572" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566572">
        <p class="paperTitle">An Arabidopsis leaf expression atlas across diurnal and developmental scales</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566572" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566572" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vong, G.; McCarthy, K.; Claydon, W.; Davis, S. J.; Redmond, E. J.; Ezer, D.</p>
        <p class="info">Score: 10.3, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566572' target='https://doi.org/10.1101/2023.11.10.566572'> 10.1101/2023.11.10.566572</a></p>
        <p class="abstract">Mature plant leaves are a composite of distinct cell types, including epidermal, mesophyll and vascular cells. Notably the proportion of these cells, and the relative transcript concentrations within different cell types may change over time. While gene expression data at a single-cell level can provide cell-type specific expression values, it is often too expensive to perform this on high resolution time series. Although bulk RNA-seq can be performed in a high resolution time series, the RNA-seq in whole leaves measures the average gene expression values across all cell types in each sample. In this study, we combined single cell RNA-seq data with time-series data from whole leaves to infer an atlas of cell type-specific gene expression changes over time for Arabidopsis thaliana. We inferred how relative transcript concentrations of cell types vary across diurnal and developmental time scales. Importantly this analysis revealed three sub-groups of mesophyll cells that have distinct temporal profiles of expression. Finally, we develop tissue-specific gene networks that form a new community resource: An Arabidopsis Leaf Time-Dependent Atlas (AraLeTa), which allows users to extract gene networks that are confirmed by transcription factor binding data and specific to certain cell types, at certain times of day and certain developmental stages, which is available at: https://regulatorynet.shinyapps.io/araleta/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565859">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565859" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565859">
        <p class="paperTitle">Gibberellin Perception Sensors 1 and 2 reveal cellular GA dynamics articulated by COP1 and GA20ox1 that are necessary but not sufficient to pattern hypocotyl cell elongation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565859" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565859" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Griffiths, J.; Rizza, A.; Tang, B.; Frommer, W. B.; Jones, A. M.</p>
        <p class="info">Score: 9.9, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565859' target='https://doi.org/10.1101/2023.11.06.565859'> 10.1101/2023.11.06.565859</a></p>
        <p class="abstract">The phytohormone gibberellin (GA) is critical for environmentally sensitive plant development including germination, skotomorphogenesis and flowering. The FRET biosensor GIBBERELLIN PERCEPTION SENSOR1, which permits single-cell GA measurements in vivo, was previously used to observe a GA gradient correlated with cell length in dark-grown but not light-grown hypocotyls. We sought to understand how light signalling integrates into cellular GA regulation. Here we show how the E3 ligase COP1 and transcription factor HY5 play central roles in directing cellular GA distribution in skoto- and photomorphogenic hypocotyls, respectively. We demonstrate that the expression pattern of biosynthetic enzyme GA20ox1 is the key determinant of the GA gradient in dark-grown hypocotyls and is a target of COP1 signalling. We engineered a second generation GPS2 biosensor with improved orthogonality and reversibility to show the cellular pattern of GA depletion during the transition to growth in the light. This GA depletion partly explains the resetting of hypocotyl growth dynamics during photomorphogenesis. Achieving cell-level resolution has revealed how GA distributions link environmental conditions with morphology and morphological plasticity and the GPS2 biosensor is an ideal tool for GA studies in further conditions, organs and plant species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.565979">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.565979" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.565979">
        <p class="paperTitle">Repression of pattern-triggered immune responses by hypoxia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.565979" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.565979" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mooney, B. C.; Doorly, C. M.; Mantz, M.; Garcia, P.; Huesgen, P. F.; Graciet, E.</p>
        <p class="info">Score: 11.8, Published: 2023-11-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.565979' target='https://doi.org/10.1101/2023.11.07.565979'> 10.1101/2023.11.07.565979</a></p>
        <p class="abstract">Combined abiotic/biotic stresses frequently occur in nature. Yet, relatively little is known about how plants regulate the crosstalk between stress response pathways to trigger a coordinated response to combined stresses. Protein degradation by the ubiquitin/proteasome system (UPS) is central to the regulation of plant responses to multiple stresses, including the crosstalk between pathways. The Arg/N-degron pathway, a subset of the UPS, targets proteins based on their N-termini. In plants, it has been implicated in the responses to both biotic and abiotic stresses, while being a key regulator of hypoxia response through the degradation of ERF-VII transcription factors, which orchestrate the onset of the hypoxia response program. Because of its central position in regulating multiple (a)biotic stresses, we hypothesized that the Arg/N-degron pathway could act in the crosstalk between abiotic/biotic stresses. In exploring this possibility using the model pathogen-associated molecular pattern flg22 to elicit pattern-triggered immunity (PTI), we uncovered a link between the transcriptional response programs to hypoxia and to flg22. Combined hypoxia/flg22 treatments further showed that hypoxia represses the flg22 transcriptional program, as well as the expression of pattern recognition receptors, MAPK signalling and callose deposition during PTI, through mechanisms that are at least partly dependent on the ERF-VIIs. These findings are of relevance to understanding the trade-offs between plant responses to combined abiotic/biotic stresses in the context of our efforts to increase crop resilience to global climate change. Our results also show that the well-known repressive effect of hypoxia on innate immunity in animals also applies to plants.

Significance statementUnderstanding how plants regulate the crosstalk between stress response pathways is key to our efforts to increase crop resilience and mitigate yield losses caused by global climate change. Despite the urgency to do so, relatively little remains known about how plants respond to combined stresses, which frequently occur in nature. Here, we show that the hypoxia response program and the basal layer of plant immunity (pattern-triggered immunity or PTI) share components. Our data also show that hypoxia represses several key aspects of the plants innate immune response, a situation akin to that discovered in animals decades ago. These findings have implications for our ability to develop resilient crops by limiting the negative trade-offs that exist between hypoxia response and immunity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565777">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565777" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565777">
        <p class="paperTitle">JAGGER localization and function is dependent on GPI anchor addition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565777" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565777" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Figueiredo, R.; Costa, M.; Moreira, D.; Moreira, M.; Noble, J.; Pereira, L. G.; Melo, P.; Palanivelu, R.; Coimbra, S.; Pereira, A. M.</p>
        <p class="info">Score: 8.2, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565777' target='https://doi.org/10.1101/2023.11.06.565777'> 10.1101/2023.11.06.565777</a></p>
        <p class="abstract">In flowering plants, successful double fertilization requires the correct delivery of two sperm cells to the female gametophyte inside the ovule. The delivery of a single pair of sperm cells is achieved by the entrance of a single pollen tube into one female gametophyte. To prevent polyspermy, Arabidopsis ovules avoid the attraction of multiples pollen tubes to one ovule - polytubey block. In Arabidopsis jagger mutants, a significant number of ovules attract more than one pollen tube to an ovule due to an impairment in synergid degeneration. JAGGER encodes a putative arabinogalactan protein (AGP) which is predicted to be anchored to the plasma membrane by a glycosylphosphatidylinositol (GPI) anchor. Here, we show that JAGGER fused to citrine yellow fluorescent protein (JAGGER-cYFP) is functional and localizes mostly to the periphery of ovule integuments and transmitting tract cells. We further investigated the importance of GPI-anchor addition domains for JAGGER localization and function. Different JAGGER proteins with deletions in predicted {omega}-site regions and GAS (GPI attachment signal) domain, expected to compromise the addition of the GPI anchor, led to disruption of JAGGER localization in the cell periphery. All JAGGER proteins with disrupted localization were also not able to rescue the polytubey phenotype, pointing to the importance of GPI-anchor addition to in vivo function of the JAGGER protein.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.12.566724">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.12.566724" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.12.566724">
        <p class="paperTitle">Photobodies enable the phase-separation and counterbalance of opposing phytochrome B actions in PIF5 degradation and stabilization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.12.566724" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.12.566724" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, R. J. A.; Fan, D.; He, J.; Kim, K.; Du, J.; Chen, M.</p>
        <p class="info">Score: 7.3, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.12.566724' target='https://doi.org/10.1101/2023.11.12.566724'> 10.1101/2023.11.12.566724</a></p>
        <p class="abstract">Photoactivation of the plant photoreceptor and thermosensor phytochrome B (PHYB) triggers its condensation into subnuclear photobodies (PBs). However, the function of PBs remains frustratingly elusive. Here, we show that PHYB condensation enables the co-occurrence and competition of two antagonistic phase-separated signaling actions. We found that PHYB recruits PHYTOCHROME-INTERACTING FACTOR5 (PIF5) to PBs and, surprisingly, that PHYB exerts opposing roles in degrading and stabilizing PIF5. Perturbing PB size by overproducing PHYB provoked a biphasic PIF5 response: while a moderate increase in PHYB enhanced PIF5 degradation, further elevating the PHYB level stabilized PIF5 by retaining more of it in enlarged PBs. Our results support a model in which PHYB condensation stabilizes PIF5 in PBs to counteract PIF5 degradation in the surrounding nucleoplasm, thereby enabling an environmentally sensitive counterbalancing mechanism to titrate nucleoplasmic PIF5 and its transcriptional output. This PB-enabled signaling mechanism provides a framework for regulating a plethora of PHYB-interacting signaling molecules in diverse plant environmental responses. We propose that this function of PBs represents a general function of biomolecular condensates to allow distinct variations of a cellular process or signaling pathway to coexist and interact to generate dynamically adjustable integrated outputs within a single subcellular space.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.566975">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.566975" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.566975">
        <p class="paperTitle">Plant pathogenic fungi hijack phosphate starvation signaling with conserved enzymatic effectors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.566975" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.566975" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McCombe, C. L.; Wegner, A.; Zamora, C. S.; Casanova, F.; Aditya, S.; Greenwood, J. R.; Wirtz, L.; de Paula, S.; England, E.; Shang, S.; Ericsson, D. J.; Oliveira-Garcia, E.; Williams, S. J.; Schaffrath, U.</p>
        <p class="info">Score: 6.4, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.566975' target='https://doi.org/10.1101/2023.11.14.566975'> 10.1101/2023.11.14.566975</a></p>
        <p class="abstract">Phosphate availability modulates plant immune function and regulates interactions with beneficial, phosphate-providing, microbes. Here, we describe the hijacking of plant phosphate sensing by a family of Nudix hydrolase effectors from pathogenic Magnaporthe oryzae and Colletotrichum fungi. Structural and enzymatic analyses of the Nudix effector family demonstrate that they selectively hydrolyze inositol pyrophosphates, a molecule used by plants to monitor phosphate status and regulate starvation responses. In M. oryzae, gene deletion and complementation experiments reveal that the enzymatic activity of a Nudix effector significantly contributes to pathogen virulence. Further, we show that this conserved effector protein family induces phosphate starvation signaling in plants. Our study elucidates a molecular mechanism, utilized by multiple phytopathogenic fungi, that manipulates the highly conserved plant phosphate sensing pathway to exacerbate disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566459">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566459" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566459">
        <p class="paperTitle">Comparative mutant analyses reveal a novel mechanism of ARF regulation in land plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566459" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566459" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Prigge, M. J.; Morffy, N.; De Neve, A.; Szutu, W.; Juarez, M. J. A.; Johnson, K.; Do, N.; Lavy, M.; Hake, S.; Strader, L.; Estelle, M.; Richardson, A. E.</p>
        <p class="info">Score: 5.2, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566459' target='https://doi.org/10.1101/2023.11.09.566459'> 10.1101/2023.11.09.566459</a></p>
        <p class="abstract">A major challenge in plant biology is to understand how the plant hormone auxin regulates diverse transcriptional responses throughout development, in different environments, and in different species. The answer may lie in the specific complement of auxin signaling components in each cell. The balance between activators (class-A AUXIN RESPONSE FACTORS) and repressors (class-B ARFs) is particularly important. It is unclear how this balance is achieved. Through comparative analysis of novel, dominant mutants in maize and the moss Physcomitrium patens, we have discovered a ~500-million-year-old mechanism of class-B ARF protein level regulation, important in determining cell fate decisions across land plants. Thus, our results add a key piece to the puzzle of how auxin regulates plant development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.562893">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.562893" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.562893">
        <p class="paperTitle">Single nuclei sequencing reveals C4 photosynthesis is based on rewiring of ancestral cell identity networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.562893" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.562893" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Swift, J.; Luginbuehl, L. H.; Schreier, T.; Donald, R. M.; Lee, T.; Nery, J.; Ecker, J. R.; Hibberd, J. M.</p>
        <p class="info">Score: 61.0, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.562893' target='https://doi.org/10.1101/2023.10.26.562893'> 10.1101/2023.10.26.562893</a></p>
        <p class="abstract">In multicellular systems changes to the patterning of gene expression drive modifications in cell function and trait evolution. One striking example is found in more than sixty plant lineages where compartmentation of photosynthesis between cell types allowed the evolution of the efficient C4 pathway from the ancestral C3 state. The molecular events enabling this transition are unclear. We used single nuclei sequencing to generate atlases for C3 rice and C4 sorghum during photomorphogenesis. Our analysis revealed that initiation of photosynthesis gene expression is conditioned by cell identity. In both species a conserved cistrome defines each cell type, and photosynthesis genes switching expression from mesophyll in rice to bundle sheath in sorghum acquire hallmarks of bundle sheath identity. The sorghum bundle sheath has also acquired gene networks associated with C3 guard cells. We conclude C4 photosynthesis is based on rewiring in cis that exapts cell identity networks of C3 plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.566092">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.566092" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.566092">
        <p class="paperTitle">Globally-deployed sorghum aphid resistance gene RMES1 is vulnerable to biotype shifts but being bolstered by RMES2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.566092" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.566092" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: VanGessel, C.; Rice, B.; Felderhoff, T.; Charles, J. R.; Pressoir, G.; Nalam, V.; Morris, G. P.</p>
        <p class="info">Score: 4.0, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.566092' target='https://doi.org/10.1101/2023.11.07.566092'> 10.1101/2023.11.07.566092</a></p>
        <p class="abstract">Durable host plant resistance (HPR) to insect pests is critical for sustainable agriculture. Natural variation exists for aphid HPR in sorghum (Sorghum bicolor) but the genetic architecture and phenotype has not been clarified for most sources. To assess the threat of a sorghum aphid (Melanaphis sorghi) biotype shift, we characterized the phenotype of Resistance to Melanaphis sorghi 1 (RMES1) and contributing HPR architecture in globally-admixed populations selected under severe aphid infestation in Haiti. We found RMES1 reduces sorghum aphid fecundity but not bird cherry-oat aphid (Rhopalosiphum padi) fecundity, suggesting a discriminant HPR response typical of gene-for-gene interaction. A second resistant gene, RMES2, were more frequent than RMES1 resistant alleles in landraces and historic breeding lines. RMES2 contributes early and mid-season aphid resistance in a segregating F2 population, however RMES1 was only significant with mid-season fitness. In a fixed population with high aphid resistance, RMES1 and RMES2 were selected for demonstrating a lack of significant antagonistic pleiotropy. Associations with resistance co-located with cyanogenic glucoside biosynthesis genes support additional HPR sources. Globally, therefore, a vulnerable HPR source (RMES1) is bolstered by a second common source of resistance in breeding programs (RMES2) which may be staving off a biotype shift.

HIGHLIGHTThe globally-deployed sorghum aphid resistance gene, RMES1, reduces aphid reproduction and therefore is vulnerable to a biotype shift. A second major gene, RMES2, and cyanogenesis may increase global durability of resistance.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
