<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment
Authors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium, ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment
Authors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium, ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-01T10:38:40+00:00" />
<meta property="article:modified_time" content="2023-11-01T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment
Authors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium, ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment\nAuthors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium, ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J.",
  "keywords": [
    
  ],
  "articleBody": " The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment\nAuthors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium, ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J. H.; Schmutz, J.; Webber, J.; Grimwood, J.; Jacquet, C.; Dunand, C.; Nelson, J.; Roux, F.; Philippe, H.; Schornack, S.; Bonhomme, M.; Delaux, P.-M.\nScore: 60.7, Published: 2023-10-29 DOI: 10.1101/2023.10.27.564390\nPlant adaptation to a terrestrial life 450 million years ago played a major role in the evolution of life on Earth. This shift from an aquatic environment has been mostly studied by focusing on flowering plants. Here, we gathered a collection of 133 accessions of the non-vascular plants Marchantia polymorpha and studied its intraspecific diversity using selection signature analyses, genome-environment association study and a gene-centered pangenome. We identified adaptive features shared with flowering plants, such as peroxidases or nucleotide-binding and leucine-rich repeat (NLR), which likely played a role in the adaptation of the first land plants to the terrestrial habitat. The M. polymorpha pangenome also harbored lineage-specific accessory genes absent from seed plants. We conclude that different land plants lineages still share many elements from the genetic toolkit evolved by their most recent common ancestor to adapt to the terrestrial habitat, refined by lineage specific polymorphisms and gene family evolutions.\nA fully sequenced collection of homozygous EMS mutants for forward and reverse genetic screens in Arabidopsis thaliana\nAuthors: Carrere, S.; Routaboul, J.-M.; Savourat, P.; Bellenot, C.; Lopez, H.; Quiroz-Monnens, T.; Ricou, A.; Camilleri, C.; Laufs, P.; mercier, r.; Noel, L. D.\nScore: 55.4, Published: 2023-10-27 DOI: 10.1101/2023.10.26.564234\nGenetic screens are powerful tools for biological research and are one of the reasons for the success of the thale cress Arabidopsis thaliana as a model species. Here, we describe the whole-genome sequencing of 871 Arabidopsis lines from the Homozygous EMS Mutant (HEM) collection as a novel resource for forward and reverse genetics. With an average 576 high-confidence mutations per HEM line, over three independent mutations altering protein sequence are found on average per gene in the collection. Pilot reverse genetics experiments on reproductive, developmental and physiological traits confirmed the efficacy of the tool for identifying both null and knockdown alleles. The possibility of conducting subtle repeated phenotyping of HEM lines and the immediate availability of the mutations will empower forward genetic approaches. The sequence resource is searchable with the ATHEM web interface (https://lipm-browsers.toulouse.inra.fr/pub/ATHEM/), and the biological material is distributed by the Versailles Arabidopsis Stock Center.\nSingle nuclei sequencing reveals C4 photosynthesis is based on rewiring of ancestral cell identity networks\nAuthors: Swift, J.; Luginbuehl, L. H.; Schreier, T.; Donald, R. M.; Lee, T.; Nery, J.; Ecker, J. R.; Hibberd, J. M.\nScore: 52.9, Published: 2023-10-27 DOI: 10.1101/2023.10.26.562893\nIn multicellular systems changes to the patterning of gene expression drive modifications in cell function and trait evolution. One striking example is found in more than sixty plant lineages where compartmentation of photosynthesis between cell types allowed evolution of the efficient C4 pathway from the ancestral C3 state. The molecular events enabling this transition are unclear. We used single nuclei sequencing to generate a cell level expression atlas for C3 rice and C4 sorghum during photomorphogenesis. In both species a conserved cistrome was identified for each cell type and initiation of photosynthesis gene expression was conditioned by cell identity. Photosynthesis genes switching expression from mesophyll in rice to bundle sheath in sorghum acquire hallmarks of bundle sheath identity. The sorghum bundle sheath has also acquired gene networks associated with C3 guard cells. We conclude C4 photosynthesis is based on rewiring in cis that exapts cell identity networks of C3 plants.\nNRC immune receptor networks show diversified hierarchical genetic architecture across plant lineages\nAuthors: Goh, F.-J.; Huang, C.-Y.; Derevnina, L.; Wu, C.-H.\nScore: 21.4, Published: 2023-10-27 DOI: 10.1101/2023.10.25.563953\nPlants developed sophisticated immune systems consisting of nucleotide-binding domain and leucine-rich repeat-containing (NLR) proteins to repel invading pathogens. The NRC (NLR required for cell death) family is a group of helper NLRs that form a complex genetic network with multiple sensor NLRs to provide resistance against various pathogens of solanaceous plants. However, how the NRC network has evolved and how it functions outside of solanaceous plants is currently unknown. We conducted phylogenomic and macroevolutionary analyses comparing NLRs identified from different lineages of asterids and found that NRC networks showed significant lineage-specific expansion patterns in lamiids but not in Ericales and campanulids. Using transient expression assays in Nicotiana benthamiana, we show that the NRC networks in Ericales and campanulids are simple, with one or two NRC nodes, while the NRC networks of lamiids were complex, with multiple partially redundant NRC nodes. Phylogenetic analyses grouped the NRC helper NLRs into three NRC0 subclades that are conserved, and several family-specific NRC subclades of lamiids that show signatures of diversifying selection. Functional analyses of NRCs and NRC-dependent sensor NLRs from different species revealed that members of the NRC0 subclades are partially interchangeable, with several functioning with NRC0-dependent sensor NLRs across asterids. In contrast, family-specific NRC members in lamiids display a lack of interchangeability, with only a few showing compatibility with sensor NLRs across different plant families. Our findings highlight the distinctive evolutionary patterns of the NRC networks in asterids and provide potential insights into transferring disease resistance across plant lineages.\nGenotype-specific nonphotochemical quenching responses to nitrogen deficit are linked to chlorophyll a to b ratios\nAuthors: Sahay, S.; Grzybowski, M.; Schnable, J. C.; Glowacka, K.\nScore: 11.3, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563650\nNon-photochemical quenching (NPQ) protects plants from photodamage caused by excess light energy. The mechanism of NPQ appears to be conserved across flowering plants. However, substantial variation in NPQ has been reported within different genotypes of the same species grown under the environmental conditions. Individual maize genotypes from a diversity panel exhibited a range of responses to low nitrogen with some genotypes exhibiting increased NPQ between control and low nitrogen conditions and others exhibiting no change. These patterns were consistent for the same genotypes across multiple field seasons. NPQ increases under low nitrogen were correlated with shifts in the ratio of chlorophyll a to chlorophyll b consistent with a decrease in reaction centers. Both photosynthetic capacity and dry biomass accumulation decreased more in maize genotypes which were unable to maintain constant NPQ levels between control and low N conditions. Collectively these results suggest that the ability to maintain sufficient numbers of reaction centers under low nitrogen conditions and avoid dissipating a greater proportion of absorbed light energy via the NPQ pathway may play a key role in increasing carbon fixation and productivity in nitrogen-limited environments. HighlightsO_LISubstantial variation in NPQ kinetics exists in maize on both low and control N. C_LIO_LIIn early and late-vegetative stages a similar portion of genotypes increased, no-change or decreased NPQmax in response to low N while in the post-flowering stage substantially more genotypes decreased NPQmax. C_LIO_LIIn low nitrogen conditions, the NPQmax strongly correlates with shifts in Chl a/Chl b ratios. C_LI\nThe NRC0 gene cluster of sensor and helper NLR immune receptors is functionally conserved across asterid plants\nAuthors: Sakai, T.; Martinez-Anaya, C.; Contreras, M. P.; Kamoun, S.; Wu, C.-H.; Adachi, H.\nScore: 31.1, Published: 2023-10-23 DOI: 10.1101/2023.10.23.563533\nNLR (nucleotide-binding domain and leucine-rich repeat-containing) proteins can form complex receptor networks to confer innate immunity. NRCs are phylogenetically related nodes that function downstream of a massively expanded network of disease resistance proteins that protect against multiple plant pathogens. Here, we used phylogenomic methods to reconstruct the macroevolution of the NRC family. One of the NRCs, we termed NRC0, is the only family member shared across asterid plants, leading us to investigate its evolutionary history and genetic organization. In several asterid species, NRC0 is genetically clustered to other NLRs that are phylogenetically related to NRC-dependent disease resistance genes. This prompted us to hypothesize that the ancestral state of the NRC network is an NLR helper-sensor gene cluster that was present early during asterid evolution. We validated this hypothesis by demonstrating that NRC0 is essential for the hypersensitive cell death induced by its genetically linked sensor NLR partners in four divergent asterid species: tomato, wild sweet potato, coffee and carrot. In addition, activation of a sensor NLR leads to high-order complex formation of its genetically linked NRC0 similar to other NRCs. Our findings map out contrasting evolutionary dynamics in the macroevolution of the NRC network over the last 125 million years from a functionally conserved NLR gene cluster to a massive genetically dispersed network. One-sentence summaryNRC0 gene cluster is functionally conserved across divergent asterid species and predates the massively expanded NRC network.\nA genetic screen for dominant chloroplast reactive oxygen species signaling mutants reveals life stage-specific singlet oxygen-signaling networks\nAuthors: Lemke, M. D.; Woodson, J. D.\nScore: 5.8, Published: 2023-10-29 DOI: 10.1101/2023.10.26.564295\nPlants employ intricate molecular mechanisms to respond to abiotic stresses, which often lead to the accumulation of reactive oxygen species (ROS) within organelles such as chloroplasts. Such ROS can produce stress signals that regulate cellular response mechanisms. One ROS, singlet oxygen (1O2), is predominantly produced in the chloroplast during photosynthesis and can trigger chloroplast degradation, programmed cell death (PCD), and retrograde (organelle-to-nucleus) signaling. However, little is known about the molecular mechanisms involved in these signaling pathways or how many different signaling 1O2 pathways may exist. The Arabidopsis thaliana plastid ferrochelatase two (fc2) mutant conditionally accumulates chloroplast 1O2, making fc2 a valuable genetic system for studying chloroplast 1O2-initiated signaling. Here, we have used activation tagging in a new forward genetic screen to identify eight dominant fc2 activation-tagged (fas) mutations that suppress chloroplast 1O2-initiated PCD. While fc2 fas mutants all block 1O2-triggered PCD in the adult stage, only two fc2 fas mutants block such cellular degradation at the seedling stage, suggesting that life-stage-specific 1O2-response pathways exist. In addition to PCD, fas mutations generally reduce 1O2-induced retrograde signals. Furthermore, fas mutants have enhanced tolerance to excess light, a natural mechanism to produce chloroplast 1O2. However, general abiotic stress tolerance was only observed in one fc2 fas mutant (fc2 fas2). Together, this suggests that plants can employ general stress tolerance mechanisms to overcome 1O2 production but that this screen was mostly specific to 1O2 signaling. We also observed that salicylic acid (SA) and jasmonate (JA) stress hormone response marker genes were induced in 1O2-stressed fc2 and generally reduced by fas mutations, suggesting that SA and JA signaling is correlated with active 1O2 signaling and PCD. Together, this work highlights the complexity of 1O2 signaling by demonstrating that multiple pathways may exist and introduces a suite of new 1O2 signaling mutants to investigate the mechanisms controlling chloroplast-initiated degradation, PCD, and retrograde signaling.\nMultiple transcription factors mediate acclimation of Chlamydomonas to light stress\nAuthors: Wulf, D.; Kruger, F. J.; Klages, L. J.; Viehoever, P.; Jin, E.; Wobbe, L.; Eisenhut, M.; Kruse, O.; Blifernez-Klassen, O.; Braeutigam, A.\nScore: 5.2, Published: 2023-10-31 DOI: 10.1101/2023.10.30.564712\nLight as a substrate for photosynthesis may be a boon or a bane. To thrive, photosynthetic organisms must constantly respond to changing light and CO2 conditions by balancing energy harvest and consumption in a highly dynamic way. Two major safeguard measures of photoacclimation, that is photoprotection and carbon concentrating mechanism, underlie tight transcriptional control, leading to expression changes under high light and limited CO2 with different dynamics for both systems. Here, by using a consensus gene regulatory network inferred by employing a compendium of 1,869 RNA-seq datasets, we identified and validated in vivo eight candidate transcription factors (TFs) that contribute to photoacclimation in Chlamydomonas reinhardtii. Target gene analyses indicate that the TFs act individually in associated pathways but also influence each other in expression, and function as network parts with partial redundancy with respect to photoprotection. The analyses unveil that stress responses in Chlamydomonas are mediated by a complex, interconnected network of TFs rather than a hierarchical system where multiple regulators can influence each other and target gene expression and thereby mitigate the effects of loss.\nBiolistics-mediated transformation of hornworts and its application to study pyrenoid protein localization\nAuthors: Lafferty, D.; Robison, T. A.; Gunadi, A.; Gunn, L. H.; Van Eck, J.; Li, F.-W.\nScore: 5.0, Published: 2023-10-26 DOI: 10.1101/2023.10.23.563637\nHornworts are a deeply diverged lineage of bryophytes that are sister to mosses and liverworts. Hornworts have an array of unique features that can be leveraged to illuminate not only the early evolution of land plants, but also alternative paths for nitrogen and carbon assimilation via cyanobacterial symbiosis and a pyrenoid-based CO2-concentrating mechanism (CCM), respectively. Despite this, hornworts are one of the few plant lineages with limited available genetic tools. Here we report an efficient biolistics method for generating transient-expression and stable transgenic lines in the model hornwort, Anthoceros agrestis. An average of 569 ({+/-} 268) cells showed transient expression per bombardment, with green fluorescent protein expression observed within 48 hours. A total of 81 stably transformed lines were recovered across three separate experiments, averaging six lines per bombardment. We followed the same method to transiently transform nine additional hornwort species, and obtained stable transformants from one. This method was further used to verify the localization of Rubisco and Rubisco activase in pyrenoids, which are central proteins for CCM function. Together, our biolistics approach offers key advantages over existing methods as it enables rapid transient expression and can be applied to widely diverse hornwort species. HighlightWe developed a new transformation method for hornworts, a lineage of understudied bryophytes, and demonstrated its effectiveness in studying subcellular localization of proteins involved in pyrenoid-based CO2-concentrating mechanism.\nAntagonistic Regulation of Arabidopsis Leaf Senescence by SCOOP10 and SCOOP12 Peptides via MIK2 Receptor-Like Kinase\nAuthors: Zhang, Z.; Gigli-Bisceglia, N.; Li, W.; Testerink, C.; Guo, Y.\nScore: 4.7, Published: 2023-10-29 DOI: 10.1101/2023.10.27.564453\nLeaf senescence plays a critical role in a plant's overall reproductive success due to its involvement in nutrient remobilization and allocation. However, our current understanding of the molecular mechanisms controlling leaf senescence remains limited. In this study, we demonstrate that the receptor-like kinase MALE DISCOVERER 1-INTERACTING RECEPTOR-LIKE KINASE 2 (MIK2) functions as a negative regulator of leaf senescence. We report that the SERINE-RICH ENDOGENOUS PEPTIDES 10 and 12 (SCOOP10 and SCOOP12), previously known to physically interact with MIK2, compete with each other to control MIK2-dependent senescence mechanisms. We observed that increased expression of SCOOP10 or the application of exogenous SCOOP10 peptides accelerated leaf senescence in a MIK2-dependent manner. Conversely, SCOOP12 acted as a suppressor of MIK2-dependent senescence. Thus, the SCOOP10-MIK2 and the SCOOP12-MIK2 modules function antagonistically, allowing for fine-tuning the modulation of the leaf senescence process. Our research sheds light on the complex mechanisms underlying leaf senescence and provides valuable insights into the interplay between receptors, peptides, and the regulation of plant senescence.\n",
  "wordCount" : "2454",
  "inLanguage": "en",
  "datePublished": "2023-11-01T10:38:40Z",
  "dateModified": "2023-11-01T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 1, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564390">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564390" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564390">
        <p class="paperTitle">The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564390" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564390" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium,  ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J. H.; Schmutz, J.; Webber, J.; Grimwood, J.; Jacquet, C.; Dunand, C.; Nelson, J.; Roux, F.; Philippe, H.; Schornack, S.; Bonhomme, M.; Delaux, P.-M.</p>
        <p class="info">Score: 60.7, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564390' target='https://doi.org/10.1101/2023.10.27.564390'> 10.1101/2023.10.27.564390</a></p>
        <p class="abstract">Plant adaptation to a terrestrial life 450 million years ago played a major role in the evolution of life on Earth. This shift from an aquatic environment has been mostly studied by focusing on flowering plants. Here, we gathered a collection of 133 accessions of the non-vascular plants Marchantia polymorpha and studied its intraspecific diversity using selection signature analyses, genome-environment association study and a gene-centered pangenome. We identified adaptive features shared with flowering plants, such as peroxidases or nucleotide-binding and leucine-rich repeat (NLR), which likely played a role in the adaptation of the first land plants to the terrestrial habitat. The M. polymorpha pangenome also harbored lineage-specific accessory genes absent from seed plants. We conclude that different land plants lineages still share many elements from the genetic toolkit evolved by their most recent common ancestor to adapt to the terrestrial habitat, refined by lineage specific polymorphisms and gene family evolutions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564234">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564234" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564234">
        <p class="paperTitle">A fully sequenced collection of homozygous EMS mutants for forward and reverse genetic screens in Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564234" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564234" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carrere, S.; Routaboul, J.-M.; Savourat, P.; Bellenot, C.; Lopez, H.; Quiroz-Monnens, T.; Ricou, A.; Camilleri, C.; Laufs, P.; mercier, r.; Noel, L. D.</p>
        <p class="info">Score: 55.4, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564234' target='https://doi.org/10.1101/2023.10.26.564234'> 10.1101/2023.10.26.564234</a></p>
        <p class="abstract">Genetic screens are powerful tools for biological research and are one of the reasons for the success of the thale cress Arabidopsis thaliana as a model species. Here, we describe the whole-genome sequencing of 871 Arabidopsis lines from the Homozygous EMS Mutant (HEM) collection as a novel resource for forward and reverse genetics. With an average 576 high-confidence mutations per HEM line, over three independent mutations altering protein sequence are found on average per gene in the collection. Pilot reverse genetics experiments on reproductive, developmental and physiological traits confirmed the efficacy of the tool for identifying both null and knockdown alleles. The possibility of conducting subtle repeated phenotyping of HEM lines and the immediate availability of the mutations will empower forward genetic approaches. The sequence resource is searchable with the ATHEM web interface (https://lipm-browsers.toulouse.inra.fr/pub/ATHEM/), and the biological material is distributed by the Versailles Arabidopsis Stock Center.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.562893">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.562893" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.562893">
        <p class="paperTitle">Single nuclei sequencing reveals C4 photosynthesis is based on rewiring of ancestral cell identity networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.562893" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.562893" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Swift, J.; Luginbuehl, L. H.; Schreier, T.; Donald, R. M.; Lee, T.; Nery, J.; Ecker, J. R.; Hibberd, J. M.</p>
        <p class="info">Score: 52.9, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.562893' target='https://doi.org/10.1101/2023.10.26.562893'> 10.1101/2023.10.26.562893</a></p>
        <p class="abstract">In multicellular systems changes to the patterning of gene expression drive modifications in cell function and trait evolution. One striking example is found in more than sixty plant lineages where compartmentation of photosynthesis between cell types allowed evolution of the efficient C4 pathway from the ancestral C3 state. The molecular events enabling this transition are unclear. We used single nuclei sequencing to generate a cell level expression atlas for C3 rice and C4 sorghum during photomorphogenesis. In both species a conserved cistrome was identified for each cell type and initiation of photosynthesis gene expression was conditioned by cell identity. Photosynthesis genes switching expression from mesophyll in rice to bundle sheath in sorghum acquire hallmarks of bundle sheath identity. The sorghum bundle sheath has also acquired gene networks associated with C3 guard cells. We conclude C4 photosynthesis is based on rewiring in cis that exapts cell identity networks of C3 plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.563953">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.563953" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.563953">
        <p class="paperTitle">NRC immune receptor networks show diversified hierarchical genetic architecture across plant lineages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.563953" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.563953" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goh, F.-J.; Huang, C.-Y.; Derevnina, L.; Wu, C.-H.</p>
        <p class="info">Score: 21.4, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.563953' target='https://doi.org/10.1101/2023.10.25.563953'> 10.1101/2023.10.25.563953</a></p>
        <p class="abstract">Plants developed sophisticated immune systems consisting of nucleotide-binding domain and leucine-rich repeat-containing (NLR) proteins to repel invading pathogens. The NRC (NLR required for cell death) family is a group of helper NLRs that form a complex genetic network with multiple sensor NLRs to provide resistance against various pathogens of solanaceous plants. However, how the NRC network has evolved and how it functions outside of solanaceous plants is currently unknown. We conducted phylogenomic and macroevolutionary analyses comparing NLRs identified from different lineages of asterids and found that NRC networks showed significant lineage-specific expansion patterns in lamiids but not in Ericales and campanulids. Using transient expression assays in Nicotiana benthamiana, we show that the NRC networks in Ericales and campanulids are simple, with one or two NRC nodes, while the NRC networks of lamiids were complex, with multiple partially redundant NRC nodes. Phylogenetic analyses grouped the NRC helper NLRs into three NRC0 subclades that are conserved, and several family-specific NRC subclades of lamiids that show signatures of diversifying selection. Functional analyses of NRCs and NRC-dependent sensor NLRs from different species revealed that members of the NRC0 subclades are partially interchangeable, with several functioning with NRC0-dependent sensor NLRs across asterids. In contrast, family-specific NRC members in lamiids display a lack of interchangeability, with only a few showing compatibility with sensor NLRs across different plant families. Our findings highlight the distinctive evolutionary patterns of the NRC networks in asterids and provide potential insights into transferring disease resistance across plant lineages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563650">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563650" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563650">
        <p class="paperTitle">Genotype-specific nonphotochemical quenching responses to nitrogen deficit are linked to chlorophyll a to b ratios</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563650" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563650" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sahay, S.; Grzybowski, M.; Schnable, J. C.; Glowacka, K.</p>
        <p class="info">Score: 11.3, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563650' target='https://doi.org/10.1101/2023.10.24.563650'> 10.1101/2023.10.24.563650</a></p>
        <p class="abstract">Non-photochemical quenching (NPQ) protects plants from photodamage caused by excess light energy. The mechanism of NPQ appears to be conserved across flowering plants. However, substantial variation in NPQ has been reported within different genotypes of the same species grown under the environmental conditions. Individual maize genotypes from a diversity panel exhibited a range of responses to low nitrogen with some genotypes exhibiting increased NPQ between control and low nitrogen conditions and others exhibiting no change. These patterns were consistent for the same genotypes across multiple field seasons. NPQ increases under low nitrogen were correlated with shifts in the ratio of chlorophyll a to chlorophyll b consistent with a decrease in reaction centers. Both photosynthetic capacity and dry biomass accumulation decreased more in maize genotypes which were unable to maintain constant NPQ levels between control and low N conditions. Collectively these results suggest that the ability to maintain sufficient numbers of reaction centers under low nitrogen conditions and avoid dissipating a greater proportion of absorbed light energy via the NPQ pathway may play a key role in increasing carbon fixation and productivity in nitrogen-limited environments.

HighlightsO_LISubstantial variation in NPQ kinetics exists in maize on both low and control N.
C_LIO_LIIn early and late-vegetative stages a similar portion of genotypes increased, no-change or decreased NPQmax in response to low N while in the post-flowering stage substantially more genotypes decreased NPQmax.
C_LIO_LIIn low nitrogen conditions, the NPQmax strongly correlates with shifts in Chl a/Chl b ratios.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563533">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563533" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563533">
        <p class="paperTitle">The NRC0 gene cluster of sensor and helper NLR immune receptors is functionally conserved across asterid plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563533" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563533" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sakai, T.; Martinez-Anaya, C.; Contreras, M. P.; Kamoun, S.; Wu, C.-H.; Adachi, H.</p>
        <p class="info">Score: 31.1, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563533' target='https://doi.org/10.1101/2023.10.23.563533'> 10.1101/2023.10.23.563533</a></p>
        <p class="abstract">NLR (nucleotide-binding domain and leucine-rich repeat-containing) proteins can form complex receptor networks to confer innate immunity. NRCs are phylogenetically related nodes that function downstream of a massively expanded network of disease resistance proteins that protect against multiple plant pathogens. Here, we used phylogenomic methods to reconstruct the macroevolution of the NRC family. One of the NRCs, we termed NRC0, is the only family member shared across asterid plants, leading us to investigate its evolutionary history and genetic organization. In several asterid species, NRC0 is genetically clustered to other NLRs that are phylogenetically related to NRC-dependent disease resistance genes. This prompted us to hypothesize that the ancestral state of the NRC network is an NLR helper-sensor gene cluster that was present early during asterid evolution. We validated this hypothesis by demonstrating that NRC0 is essential for the hypersensitive cell death induced by its genetically linked sensor NLR partners in four divergent asterid species: tomato, wild sweet potato, coffee and carrot. In addition, activation of a sensor NLR leads to high-order complex formation of its genetically linked NRC0 similar to other NRCs. Our findings map out contrasting evolutionary dynamics in the macroevolution of the NRC network over the last 125 million years from a functionally conserved NLR gene cluster to a massive genetically dispersed network.

One-sentence summaryNRC0 gene cluster is functionally conserved across divergent asterid species and predates the massively expanded NRC network.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564295">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564295" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564295">
        <p class="paperTitle">A genetic screen for dominant chloroplast reactive oxygen species signaling mutants reveals life stage-specific singlet oxygen-signaling networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564295" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564295" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lemke, M. D.; Woodson, J. D.</p>
        <p class="info">Score: 5.8, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564295' target='https://doi.org/10.1101/2023.10.26.564295'> 10.1101/2023.10.26.564295</a></p>
        <p class="abstract">Plants employ intricate molecular mechanisms to respond to abiotic stresses, which often lead to the accumulation of reactive oxygen species (ROS) within organelles such as chloroplasts. Such ROS can produce stress signals that regulate cellular response mechanisms. One ROS, singlet oxygen (1O2), is predominantly produced in the chloroplast during photosynthesis and can trigger chloroplast degradation, programmed cell death (PCD), and retrograde (organelle-to-nucleus) signaling. However, little is known about the molecular mechanisms involved in these signaling pathways or how many different signaling 1O2 pathways may exist. The Arabidopsis thaliana plastid ferrochelatase two (fc2) mutant conditionally accumulates chloroplast 1O2, making fc2 a valuable genetic system for studying chloroplast 1O2-initiated signaling. Here, we have used activation tagging in a new forward genetic screen to identify eight dominant fc2 activation-tagged (fas) mutations that suppress chloroplast 1O2-initiated PCD. While fc2 fas mutants all block 1O2-triggered PCD in the adult stage, only two fc2 fas mutants block such cellular degradation at the seedling stage, suggesting that life-stage-specific 1O2-response pathways exist. In addition to PCD, fas mutations generally reduce 1O2-induced retrograde signals. Furthermore, fas mutants have enhanced tolerance to excess light, a natural mechanism to produce chloroplast 1O2. However, general abiotic stress tolerance was only observed in one fc2 fas mutant (fc2 fas2). Together, this suggests that plants can employ general stress tolerance mechanisms to overcome 1O2 production but that this screen was mostly specific to 1O2 signaling. We also observed that salicylic acid (SA) and jasmonate (JA) stress hormone response marker genes were induced in 1O2-stressed fc2 and generally reduced by fas mutations, suggesting that SA and JA signaling is correlated with active 1O2 signaling and PCD. Together, this work highlights the complexity of 1O2 signaling by demonstrating that multiple pathways may exist and introduces a suite of new 1O2 signaling mutants to investigate the mechanisms controlling chloroplast-initiated degradation, PCD, and retrograde signaling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564712">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564712" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564712">
        <p class="paperTitle">Multiple transcription factors mediate acclimation of Chlamydomonas to light stress</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564712" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564712" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wulf, D.; Kruger, F. J.; Klages, L. J.; Viehoever, P.; Jin, E.; Wobbe, L.; Eisenhut, M.; Kruse, O.; Blifernez-Klassen, O.; Braeutigam, A.</p>
        <p class="info">Score: 5.2, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564712' target='https://doi.org/10.1101/2023.10.30.564712'> 10.1101/2023.10.30.564712</a></p>
        <p class="abstract">Light as a substrate for photosynthesis may be a boon or a bane. To thrive, photosynthetic organisms must constantly respond to changing light and CO2 conditions by balancing energy harvest and consumption in a highly dynamic way. Two major safeguard measures of photoacclimation, that is photoprotection and carbon concentrating mechanism, underlie tight transcriptional control, leading to expression changes under high light and limited CO2 with different dynamics for both systems. Here, by using a consensus gene regulatory network inferred by employing a compendium of 1,869 RNA-seq datasets, we identified and validated in vivo eight candidate transcription factors (TFs) that contribute to photoacclimation in Chlamydomonas reinhardtii. Target gene analyses indicate that the TFs act individually in associated pathways but also influence each other in expression, and function as network parts with partial redundancy with respect to photoprotection. The analyses unveil that stress responses in Chlamydomonas are mediated by a complex, interconnected network of TFs rather than a hierarchical system where multiple regulators can influence each other and target gene expression and thereby mitigate the effects of loss.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563637">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563637" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563637">
        <p class="paperTitle">Biolistics-mediated transformation of hornworts and its application to study pyrenoid protein localization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563637" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563637" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lafferty, D.; Robison, T. A.; Gunadi, A.; Gunn, L. H.; Van Eck, J.; Li, F.-W.</p>
        <p class="info">Score: 5.0, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563637' target='https://doi.org/10.1101/2023.10.23.563637'> 10.1101/2023.10.23.563637</a></p>
        <p class="abstract">Hornworts are a deeply diverged lineage of bryophytes that are sister to mosses and liverworts. Hornworts have an array of unique features that can be leveraged to illuminate not only the early evolution of land plants, but also alternative paths for nitrogen and carbon assimilation via cyanobacterial symbiosis and a pyrenoid-based CO2-concentrating mechanism (CCM), respectively. Despite this, hornworts are one of the few plant lineages with limited available genetic tools. Here we report an efficient biolistics method for generating transient-expression and stable transgenic lines in the model hornwort, Anthoceros agrestis. An average of 569 ({&#43;/-} 268) cells showed transient expression per bombardment, with green fluorescent protein expression observed within 48 hours. A total of 81 stably transformed lines were recovered across three separate experiments, averaging six lines per bombardment. We followed the same method to transiently transform nine additional hornwort species, and obtained stable transformants from one. This method was further used to verify the localization of Rubisco and Rubisco activase in pyrenoids, which are central proteins for CCM function. Together, our biolistics approach offers key advantages over existing methods as it enables rapid transient expression and can be applied to widely diverse hornwort species.

HighlightWe developed a new transformation method for hornworts, a lineage of understudied bryophytes, and demonstrated its effectiveness in studying subcellular localization of proteins involved in pyrenoid-based CO2-concentrating mechanism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564453">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564453" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564453">
        <p class="paperTitle">Antagonistic Regulation of Arabidopsis Leaf Senescence by SCOOP10 and SCOOP12 Peptides via MIK2 Receptor-Like Kinase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564453" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564453" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Z.; Gigli-Bisceglia, N.; Li, W.; Testerink, C.; Guo, Y.</p>
        <p class="info">Score: 4.7, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564453' target='https://doi.org/10.1101/2023.10.27.564453'> 10.1101/2023.10.27.564453</a></p>
        <p class="abstract">Leaf senescence plays a critical role in a plant&#39;s overall reproductive success due to its involvement in nutrient remobilization and allocation. However, our current understanding of the molecular mechanisms controlling leaf senescence remains limited. In this study, we demonstrate that the receptor-like kinase MALE DISCOVERER 1-INTERACTING RECEPTOR-LIKE KINASE 2 (MIK2) functions as a negative regulator of leaf senescence. We report that the SERINE-RICH ENDOGENOUS PEPTIDES 10 and 12 (SCOOP10 and SCOOP12), previously known to physically interact with MIK2, compete with each other to control MIK2-dependent senescence mechanisms. We observed that increased expression of SCOOP10 or the application of exogenous SCOOP10 peptides accelerated leaf senescence in a MIK2-dependent manner. Conversely, SCOOP12 acted as a suppressor of MIK2-dependent senescence. Thus, the SCOOP10-MIK2 and the SCOOP12-MIK2 modules function antagonistically, allowing for fine-tuning the modulation of the leaf senescence process. Our research sheds light on the complex mechanisms underlying leaf senescence and provides valuable insights into the interplay between receptors, peptides, and the regulation of plant senescence.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
