<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="LysM-RLK plays an ancestral symbiotic function in plants
Authors: Teyssier, E.; Grat, S.; Rich, M. K.; Delaux, P.-M.; Mbengue, M.
Score: 37.7, Published: 2024-01-17 DOI: 10.1101/2024.01.16.575821
To ensure their water and mineral nutrition, most land plants form arbuscular mycorrhiza (AM) with soil-borne Glomeromycota fungi 1. This [~]450 million years old symbiosis was key in driving land colonisation by plants2. In angiosperms, AM is thought to be initiated via the perception by the host plant of AM-fungi derived chito- and lipochito-oligosaccharides leading to the activation of a conserved signalling pathway referred to as the Common Symbiosis Pathway3.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="LysM-RLK plays an ancestral symbiotic function in plants
Authors: Teyssier, E.; Grat, S.; Rich, M. K.; Delaux, P.-M.; Mbengue, M.
Score: 37.7, Published: 2024-01-17 DOI: 10.1101/2024.01.16.575821
To ensure their water and mineral nutrition, most land plants form arbuscular mycorrhiza (AM) with soil-borne Glomeromycota fungi 1. This [~]450 million years old symbiosis was key in driving land colonisation by plants2. In angiosperms, AM is thought to be initiated via the perception by the host plant of AM-fungi derived chito- and lipochito-oligosaccharides leading to the activation of a conserved signalling pathway referred to as the Common Symbiosis Pathway3." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-24T10:40:04+00:00" />
<meta property="article:modified_time" content="2024-01-24T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="LysM-RLK plays an ancestral symbiotic function in plants
Authors: Teyssier, E.; Grat, S.; Rich, M. K.; Delaux, P.-M.; Mbengue, M.
Score: 37.7, Published: 2024-01-17 DOI: 10.1101/2024.01.16.575821
To ensure their water and mineral nutrition, most land plants form arbuscular mycorrhiza (AM) with soil-borne Glomeromycota fungi 1. This [~]450 million years old symbiosis was key in driving land colonisation by plants2. In angiosperms, AM is thought to be initiated via the perception by the host plant of AM-fungi derived chito- and lipochito-oligosaccharides leading to the activation of a conserved signalling pathway referred to as the Common Symbiosis Pathway3."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "LysM-RLK plays an ancestral symbiotic function in plants\nAuthors: Teyssier, E.; Grat, S.; Rich, M. K.; Delaux, P.-M.; Mbengue, M.\nScore: 37.7, Published: 2024-01-17 DOI: 10.1101/2024.01.16.575821\nTo ensure their water and mineral nutrition, most land plants form arbuscular mycorrhiza (AM) with soil-borne Glomeromycota fungi 1. This [~]450 million years old symbiosis was key in driving land colonisation by plants2. In angiosperms, AM is thought to be initiated via the perception by the host plant of AM-fungi derived chito- and lipochito-oligosaccharides leading to the activation of a conserved signalling pathway referred to as the Common Symbiosis Pathway3.",
  "keywords": [
    
  ],
  "articleBody": " LysM-RLK plays an ancestral symbiotic function in plants\nAuthors: Teyssier, E.; Grat, S.; Rich, M. K.; Delaux, P.-M.; Mbengue, M.\nScore: 37.7, Published: 2024-01-17 DOI: 10.1101/2024.01.16.575821\nTo ensure their water and mineral nutrition, most land plants form arbuscular mycorrhiza (AM) with soil-borne Glomeromycota fungi 1. This [~]450 million years old symbiosis was key in driving land colonisation by plants2. In angiosperms, AM is thought to be initiated via the perception by the host plant of AM-fungi derived chito- and lipochito-oligosaccharides leading to the activation of a conserved signalling pathway referred to as the Common Symbiosis Pathway3. Genetics in legumes and monocots have demonstrated that members of the Lysin motif Receptor-Like Kinase (LysM-RLK) family are important for the perception of these AM-fungi derived molecules, although none of the LysM-RLK mutants or combination of mutants described to date fully abolish AM4. This discrepancy with the phenotypes observed for components of the CSP, which fail to host AM fungi, might be the result of genetic redundancy between the multiple LysM-RLK paralogs found in these species. In contrast to angiosperms, the liverwort Marchantia paleacea contains only four LysM-RLKs. In this study, we demonstrate the essential role of one LysM-RLK for AM in M. paleacea. Furthermore, we present evidence that Marchantias ability to respond to chito- or lipochito-oligosaccharides is not a predictor of its symbiotic ability, suggesting the existence of yet uncharacterized AM-fungi signals. HighlightsO_LIThe LysM-RLK LYKa is essential for arbuscular mycorrhiza in Marchantia paleacea C_LIO_LIThe only LYR from Marchantia paleacea is not required for arbuscular mycorrhiza C_LIO_LILYKa and LYR are both required for chito- and lipochito-oligosaccharide signalling C_LIO_LILipochito- and chito-oligosaccharide signalling is not essential for arbuscular mycorrhiza in Marchantia paleacea C_LI\nFROM DUST TO SEED: A LUNAR CHICKPEA STORY\nAuthors: Atkin, J.; Oliveira Pedro dos Santos, S.\nScore: 33.3, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576311\nFood sustainability is one of the most significant barriers to long-term space travel. Providing resources from Earth is not cost-efficient, and resupply missions are not viable to meet the needs of long-term life in deep space conditions. Plants in space can provide a source of nutrition and oxygen, reducing the reliance on packaged foods, reducing resupply needs, and extending the duration of missions. Using lunar regolith simulant, we employ a novel methodology to create a sustainable and productive growth medium to support the cultivation of horticultural crops on the Moon. Implementing microbial soil regeneration mechanisms derived from Earth, we leverage the interaction between Arbuscular Mycorrhizal Fungi (AMF) and Vermicompost (VC) to create a fertile LRS matrix. These amendments can sequester toxic contaminants, improve soil structure, and increase plant stress tolerance. We demonstrate the ability to produce chickpea (Cicer arietinum) in lunar regolith simulant augmented with AMF and VC under climate-controlled conditions. We cultivated chickpea to seed in a mixture containing 75% Lunar Regolith Simulant. Preliminary results suggest that higher LRS contents induce heightened stress responses. However, plants grown in 100% LRS inoculated with arbuscular mycorrhizal fungi demonstrated an average two-week survival extension compared to non-inoculated plants. This study provides, for the first time, a baseline for chickpea germination in varying mixtures of LRS and VC and will inform future studies as humanity goes back to the Moon.\nConservation of symbiotic signalling across 450 million years of plant evolution\nAuthors: Vernie, T.; Rich, M. K.; Pellen, T.; Teyssier, E.; Garrigues, V.; Chauderon, L.; Medioni, L.; van Beveren, F.; Libourel, C.; Keller, J.; Girou, C.; Lefort, C.; Le Ru, A.; Reinhardt, D.; Kodama, K.; Shimazaki, S.; Morel, P.; Kyozuka, J.; Mbengue, M.; Vandenbussche, M.; Delaux, P.-M.\nScore: 29.2, Published: 2024-01-17 DOI: 10.1101/2024.01.16.575147\nHighlightO_LIThe common symbiotic pathway is activated during arbuscular mycorrhizal symbiosis in Marchantia paleacea C_LIO_LIThe three core members of the common symbiotic pathway are essential for symbiosis in Marchantia paleacea C_LIO_LIThe molecular function of the CCaMK/CYCLOPS module is conserved across land plants C_LIO_LISymbiotic signalling has been conserved in plants for 450 million years C_LI The colonization of land by plants 450 million years ago revolutionized life on Earth1. The fossil record2 and genetic evidence in extant species3 suggest that this transition was facilitated by interactions with symbiotic arbuscular mycorrhizal (AM) fungi4. This ancestral symbiosis relied on the biosynthesis of chemicals by the host plant, both as signals5 and as nutrients3. In angiosperms, a signalling pathway involving the receptor-like kinase SYMRK/DMI26,7, the Calcium and Calmodulin-dependent protein kinase CCaMK/DMI38 and the transcription factor CYCLOPS/IPD39,10 has been described as the common symbiosis pathway (CSP), essential for the establishment of the AM symbiosis and the root-nodule symbiosis11. Phylogenetic and comparative phylogenomic analyses indicated an ancient origin of the CSP, present in all extant land plants forming intracellular symbioses12-15. Trans-complementation assays of the angiosperm mutants with orthologs from diverse species further indicated the conservation of the molecular function of the CSP across the embryophytes9,12,14-16. However, this correlative evidence did not allow testing the ancestral biological function of the CSP. In this study we demonstrate that SYMRK, CCaMK and CYCLOPS are essential for the colonization by AM fungi in bryophytes, indicating that plants have maintained a dedicated signalling pathway to support symbiotic interactions for 450 million years.\nM2WISH: an easy and efficient protocol for whole-mount mRNA in situ hybridization that allows 3D cell resolution of gene expression in Arabidopsis thaliana.\nAuthors: Chelysheva, L.; Palauqui, J.-C.; Morin, H.; Biot, E.; Nicolas, A.; Rech, P.; da Costa, M.; Barel, L.; Laufs, P.\nScore: 22.5, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576197\nGene expression analysis is essential for understanding the mechanisms involved in plant development. Here, we developed M2WISH, a protocol based on MicroWave treatment for Wholemount mRNA In Situ Hybridization in Arabidopsis. By permeabilizing tissues without damaging cellular organisation this protocol results in high and homogeneous hybridization yields that enables systematic analysis of gene expression dynamics. Moreover, when combined with cellular histochemical staining, M2WISH provides a cellular resolution of gene expression on roots, aerial meristems, leaves and embryos in the seed. We applied M2WISH to study the spatial dynamics of WUSCHEL (WUS) and CLAVATA3 (CLV3) expression during in vitro meristematic conversion of roots into shoot apical meristems. Thus, we showed that shoot apical meristems could arise from two different types of root structures that differed by their CLV3 gene expression patterns. We constructed 3D cellular representations of WUS and CLV3 gene co-expression pattern, and stressed the variability inherent to meristem conversion. Thus, this protocol generates a large amount of data on the localization of gene expression, which can be used to model complex systems.\nBioengineering a plant NLR immune receptor with a robust binding interface towards a conserved fungal pathogen effector\nAuthors: Zdrzalek, R.; Xi, Y.; Langner, T.; Bentham, A. R.; Petit-Houdenot, Y.; De la Concepcion, J. C.; Harant, A.; Shimizu, M.; Were, V.; Talbot, N. J.; Terauchi, R. J.; Kamoun, S.; Banfield, M. J.\nScore: 19.3, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576400\nBioengineering of plant immune receptors has emerged as a key strategy for generating novel disease resistance traits to counteract the expanding threat of plant pathogens to global food security. However, current approaches are limited by rapid evolution of plant pathogens in the field and may lack durability when deployed. Here, we show that the rice nucleotide-binding, leucine-rich repeat (NLR) immune receptor Pik-1 can be engineered to respond to a conserved family of effectors from the multihost blast fungus pathogen Magnaporthe oryzae. We switched the effector binding and response profile of the Pik NLR from its cognate rice blast effector AVR-Pik to the host-determining factor Pwl2 by installing a putative host target, OsHIPP43, in place of the native integrated HMA domain (generating Pikm-1OsHIPP43). This chimeric receptor also responded to other PWL alleles from diverse blast isolates. The crystal structure of the Pwl2/OsHIPP43 complex revealed a multifaceted, robust interface that cannot be easily disrupted by mutagenesis, and may therefore provide durable, broad resistance to blast isolates carrying PWL effectors in the field. Our findings highlight how the host targets of pathogen effectors can be used to bioengineer new recognition specificities that have more robust properties compared to naturally evolved disease resistance genes.\nSpeedy benthi: Elongated photoperiods reduce the generation time of the model plant species Nicotiana benthamiana\nAuthors: Castle, M.; Luedke, D.\nScore: 18.9, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576090\nNicotiana benthamiana is increasingly gaining prominence as a model plant species with recently published high-quality genome assemblies, which will further enable forward and reverse genetic approaches (Bally et al., 2018; Derevnina et al., 2019; Kourelis et al., 2019; Ranawaka et al., 2023; Vollheyde et al., 2023). However, the generation time of N. benthamiana poses a bottleneck in the creation of mutant and transgenic plant lines. Speed breeding (SB), by extended photoperiods and adjustments to growth parameters, is an efficient way to reduce generation times for many crop and model plant species (Ghosh et al., 2018; Watson et al., 2018; Hickey et al., 2019; Varshney et al., 2021). We hypothesized that an extended photoperiod could reduce the seed to seed generation time of N. benthamiana. We tested this hypothesis by comparing generation times under SB conditions to traditionally used photoperiods in growth chambers and green house settings. We found that a 22h photoperiod reduced the generation time of N. benthamiana by approximately 2 weeks (16-22%). Fertilization in combination with a far-red light spectrum did not yield a further reduction in generation time when combined with SB conditions. Our findings further contribute to the establishment of N. benthamiana as an important model organism for plant research.\nThe blast effector Pwl2 is a virulence factor that modifies the cellular localisation of host protein HIPP43 to suppress immunity.\nAuthors: Were, V.; Yan, X.; Foster, A.; Sklenar, J.; Langner, T.; Bentham, A.; Zdrzałek, R.; Ryder, L.; Kaimenyi, D.; Gomez De La Cruz, D.; Gentle, A.; Petit-Houdenot, Y.; Eseola, A. B.; Smoker, M.; Bautista, M. J.; Ma, W.; Kourelis, J.; Maclean, D.; Banfield, M. J.; Kamoun, S.; Menke, F. L. H.; Moscou, M. J.; Talbot, N. J.\nScore: 14.8, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576406\nThe rice blast fungus Magnaporthe oryzae secretes a battery of effector proteins during host infection to suppress plant immunity and promote disease. Among these effectors, the MAX effector-Pwl2 was first identified as a host specificity determinant for infection of weeping lovegrass (Eragrostis curvula). However, its biological activity has remained unknown. Here we show that the PWL2 gene is ubiquitous in all host-limited forms of M. oryzae and has undergone substantial copy number expansion indicating that it is a core effector of the blast fungus. We used CRISPR/Cas9-mediated gene editing to delete all three copies of PWL2 and generate a pwl2 null mutant in M. oryzae strain Guy11. This resulted in gain-of-virulence towards weeping lovegrass, but a reduction in the severity of disease symptoms on rice. To further investigate the virulence mechanism of Pwl2, we showed that transgenic rice and barley lines constitutively expressing PWL2 display suppression of reactive oxygen species generation and increased susceptibility to infection by M. oryzae. Also, we identify the barley and rice heavy metal-binding isoprenylated protein HIPP43 as a target of the Pwl2 effector. Transgenic lines overexpressing HIPP43 exhibit attenuated defense responses and increased susceptibility to blast infection, consistent with the hypothesis that Pwl2 binds HIPP43 to modulate immunity. This binding is coupled to changes in cellular localisation of these proteins. Taken together, our results provide evidence that Pwl2 is a virulence factor in M. oryzae that acts by suppressing host immunity through binding and re-localising HIPP43.\nChromosome-scale genome assembly of bread wheat's wild relative Triticum timopheevii\nAuthors: Grewal, S.; Yang, C.; Scholefield, D.; Ashling, S.; Ghosh, S.; Swarbreck, D.; Collins, J.; Yao, E.; Sen, T. Z.; Wilson, M.; Yant, L.; King, I. P.; King, J.\nScore: 11.9, Published: 2024-01-17 DOI: 10.1101/2024.01.16.575864\nWheat (Triticum aestivum) is one of the most important food crops with an urgent need for increase in its production to feed the growing world. Triticum timopheevii (2n = 4x = 28) is an allotetraploid wheat wild relative species containing the At and G genomes that has been exploited in many pre-breeding programmes for wheat improvement. In this study, we report the generation of a chromosome-scale reference genome assembly of T. timopheevii accession PI 94760 based on PacBio HiFi reads and chromosome conformation capture (Hi-C). The assembly comprised a total size of 9.35 Gb, featuring a contig N50 of 42.4 Mb, and 166,325 predicted gene models. DNA methylation analysis showed that the G genome had on average more methylated bases than the At genome. The G genome was also more closely related to the S genome of Aegilops speltoides than to the B genome of hexaploid or tetraploid wheat. In summary, the T. timopheevii genome assembly provides a valuable resource for genome-informed discovery of agronomically important genes for food security.\nStrategy for unlimited cycles of scarless oligonucleotide directed gene editing in Chlamydomonas reinhardtii\nAuthors: Ross, I. L.; Budiman, S.; Le, H. P.; Xiong, D.; Hemker, F.; Millen, E. A.; Oey, M.; Hankamer, B.\nScore: 7.5, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576255\nCRISPR/Cas9 gene editing in the model green alga Chlamydomonas reinhardtii relies on the use of selective marker genes to enrich for non-selectable target mutations. This becomes challenging when many sequential modifications are required in a single cell line, as useful markers are limited. Here we demonstrate a cyclical selection process which only requires a single marker gene to identify an almost infinite sequential series of CRISPR-based target gene modifications. The NIA1 (Nit1, NR; nitrate reductase) gene was this selectable marker. In the forward stage of the cycle, a stop codon was engineered into the NIA1 gene at the CRISPR target location. Cells retaining the wild type NIA1 gene were killed by chlorate, while NIA1 knockout mutants survived. In the reverse phase of the cycle, the stop codon engineered into the NIA1 gene during the forward phase was edited back to the wild type sequence. Using nitrate as the sole nitrogen source, here only the reverted wild type cells survived. By using CRISPR to specifically deactivate and reactivate the NIA1 gene, a marker system was established that flipped back and forth between chlorate- and auxotrophic (nitrate) based selection. This provided a scarless cyclical marker system that enabled an indefinite series of CRISPR edits in other, non-selectable genes. Here, we demonstrate that this Sequential CRISPR via Recycling Endogenous Auxotrophic Markers (SCREAM) technology enables an essentially limitless series of genetic modifications to be introduced to a single cell lineage of C. reinhardtii in a fast and efficient manner to complete complex genetic engineering.\nAbscisic acid signaling gates salt-specific responses of plant roots\nAuthors: Lamers, J.; Zhang, Y.; van Zelm, E.; Meyer, A. J.; De Zeeuw, T.; Verstappen, F.; Veen, M.; Deolu-Ajayi, A. O.; Gommers, C. M. M.; Testerink, C.\nScore: 7.8, Published: 2023-12-28 DOI: 10.1101/2023.12.28.572987\nSoil salinity presents a dual challenge for plants, involving both osmotic and ionic stress. In response, plants deploy distinct yet interconnected mechanisms to cope with these facets of salinity stress. In this investigation, we observed a substantial overlap in the salt (NaCl)-induced transcriptional responses of Arabidopsis roots with those triggered by osmotic stress or the plant stress hormone abscisic acid (ABA), as anticipated. Notably, a specific cluster of genes responded uniquely to sodium (Na+) ions. Surprisingly, expression of sodium-induced genes exhibited a negative correlation with the ABA response and preceded the activation of genes induced by the osmotic stress component of salt. Elevated exogenous ABA levels resulted in the complete abolition of sodium-induced responses. Consistently, ABA signalling mutants displayed prolonged sodium-induced gene expression, coupled with increased root cell damage under high salinity conditions. Moreover, ABA signalling mutants were unable to redirect root growth to avoid high sodium concentrations and failed to contain their root cell swelling in the presence of elevated salt levels. In summary, our findings unveil an unexpected and pivotal role for ABA signaling in mitigating cellular damage induced by salinity stress and modulating sodium-specific responses in plant roots.\n",
  "wordCount" : "2609",
  "inLanguage": "en",
  "datePublished": "2024-01-24T10:40:04Z",
  "dateModified": "2024-01-24T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 24, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575821">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575821" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575821">
        <p class="paperTitle">LysM-RLK plays an ancestral symbiotic function in plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575821" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575821" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Teyssier, E.; Grat, S.; Rich, M. K.; Delaux, P.-M.; Mbengue, M.</p>
        <p class="info">Score: 37.7, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575821' target='https://doi.org/10.1101/2024.01.16.575821'> 10.1101/2024.01.16.575821</a></p>
        <p class="abstract">To ensure their water and mineral nutrition, most land plants form arbuscular mycorrhiza (AM) with soil-borne Glomeromycota fungi 1. This [~]450 million years old symbiosis was key in driving land colonisation by plants2. In angiosperms, AM is thought to be initiated via the perception by the host plant of AM-fungi derived chito- and lipochito-oligosaccharides leading to the activation of a conserved signalling pathway referred to as the Common Symbiosis Pathway3. Genetics in legumes and monocots have demonstrated that members of the Lysin motif Receptor-Like Kinase (LysM-RLK) family are important for the perception of these AM-fungi derived molecules, although none of the LysM-RLK mutants or combination of mutants described to date fully abolish AM4. This discrepancy with the phenotypes observed for components of the CSP, which fail to host AM fungi, might be the result of genetic redundancy between the multiple LysM-RLK paralogs found in these species. In contrast to angiosperms, the liverwort Marchantia paleacea contains only four LysM-RLKs. In this study, we demonstrate the essential role of one LysM-RLK for AM in M. paleacea. Furthermore, we present evidence that Marchantias ability to respond to chito- or lipochito-oligosaccharides is not a predictor of its symbiotic ability, suggesting the existence of yet uncharacterized AM-fungi signals.

HighlightsO_LIThe LysM-RLK LYKa is essential for arbuscular mycorrhiza in Marchantia paleacea
C_LIO_LIThe only LYR from Marchantia paleacea is not required for arbuscular mycorrhiza
C_LIO_LILYKa and LYR are both required for chito- and lipochito-oligosaccharide signalling
C_LIO_LILipochito- and chito-oligosaccharide signalling is not essential for arbuscular mycorrhiza in Marchantia paleacea
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576311">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576311" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576311">
        <p class="paperTitle">FROM DUST TO SEED: A LUNAR CHICKPEA STORY</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576311" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576311" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Atkin, J.; Oliveira Pedro dos Santos, S.</p>
        <p class="info">Score: 33.3, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576311' target='https://doi.org/10.1101/2024.01.18.576311'> 10.1101/2024.01.18.576311</a></p>
        <p class="abstract">Food sustainability is one of the most significant barriers to long-term space travel. Providing resources from Earth is not cost-efficient, and resupply missions are not viable to meet the needs of long-term life in deep space conditions. Plants in space can provide a source of nutrition and oxygen, reducing the reliance on packaged foods, reducing resupply needs, and extending the duration of missions. Using lunar regolith simulant, we employ a novel methodology to create a sustainable and productive growth medium to support the cultivation of horticultural crops on the Moon. Implementing microbial soil regeneration mechanisms derived from Earth, we leverage the interaction between Arbuscular Mycorrhizal Fungi (AMF) and Vermicompost (VC) to create a fertile LRS matrix. These amendments can sequester toxic contaminants, improve soil structure, and increase plant stress tolerance. We demonstrate the ability to produce chickpea (Cicer arietinum) in lunar regolith simulant augmented with AMF and VC under climate-controlled conditions. We cultivated chickpea to seed in a mixture containing 75% Lunar Regolith Simulant. Preliminary results suggest that higher LRS contents induce heightened stress responses. However, plants grown in 100% LRS inoculated with arbuscular mycorrhizal fungi demonstrated an average two-week survival extension compared to non-inoculated plants. This study provides, for the first time, a baseline for chickpea germination in varying mixtures of LRS and VC and will inform future studies as humanity goes back to the Moon.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575147">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575147" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575147">
        <p class="paperTitle">Conservation of symbiotic signalling across 450 million years of plant evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575147" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575147" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vernie, T.; Rich, M. K.; Pellen, T.; Teyssier, E.; Garrigues, V.; Chauderon, L.; Medioni, L.; van Beveren, F.; Libourel, C.; Keller, J.; Girou, C.; Lefort, C.; Le Ru, A.; Reinhardt, D.; Kodama, K.; Shimazaki, S.; Morel, P.; Kyozuka, J.; Mbengue, M.; Vandenbussche, M.; Delaux, P.-M.</p>
        <p class="info">Score: 29.2, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575147' target='https://doi.org/10.1101/2024.01.16.575147'> 10.1101/2024.01.16.575147</a></p>
        <p class="abstract">HighlightO_LIThe common symbiotic pathway is activated during arbuscular mycorrhizal symbiosis in Marchantia paleacea
C_LIO_LIThe three core members of the common symbiotic pathway are essential for symbiosis in Marchantia paleacea
C_LIO_LIThe molecular function of the CCaMK/CYCLOPS module is conserved across land plants
C_LIO_LISymbiotic signalling has been conserved in plants for 450 million years
C_LI

The colonization of land by plants 450 million years ago revolutionized life on Earth1. The fossil record2 and genetic evidence in extant species3 suggest that this transition was facilitated by interactions with symbiotic arbuscular mycorrhizal (AM) fungi4. This ancestral symbiosis relied on the biosynthesis of chemicals by the host plant, both as signals5 and as nutrients3. In angiosperms, a signalling pathway involving the receptor-like kinase SYMRK/DMI26,7, the Calcium and Calmodulin-dependent protein kinase CCaMK/DMI38 and the transcription factor CYCLOPS/IPD39,10 has been described as the common symbiosis pathway (CSP), essential for the establishment of the AM symbiosis and the root-nodule symbiosis11. Phylogenetic and comparative phylogenomic analyses indicated an ancient origin of the CSP, present in all extant land plants forming intracellular symbioses12-15. Trans-complementation assays of the angiosperm mutants with orthologs from diverse species further indicated the conservation of the molecular function of the CSP across the embryophytes9,12,14-16. However, this correlative evidence did not allow testing the ancestral biological function of the CSP. In this study we demonstrate that SYMRK, CCaMK and CYCLOPS are essential for the colonization by AM fungi in bryophytes, indicating that plants have maintained a dedicated signalling pathway to support symbiotic interactions for 450 million years.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576197">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576197" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576197">
        <p class="paperTitle">M2WISH: an easy and efficient protocol for whole-mount mRNA in situ hybridization that allows 3D cell resolution of gene expression in Arabidopsis thaliana.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576197" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576197" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chelysheva, L.; Palauqui, J.-C.; Morin, H.; Biot, E.; Nicolas, A.; Rech, P.; da Costa, M.; Barel, L.; Laufs, P.</p>
        <p class="info">Score: 22.5, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576197' target='https://doi.org/10.1101/2024.01.18.576197'> 10.1101/2024.01.18.576197</a></p>
        <p class="abstract">Gene expression analysis is essential for understanding the mechanisms involved in plant development. Here, we developed M2WISH, a protocol based on MicroWave treatment for Wholemount mRNA In Situ Hybridization in Arabidopsis. By permeabilizing tissues without damaging cellular organisation this protocol results in high and homogeneous hybridization yields that enables systematic analysis of gene expression dynamics. Moreover, when combined with cellular histochemical staining, M2WISH provides a cellular resolution of gene expression on roots, aerial meristems, leaves and embryos in the seed. We applied M2WISH to study the spatial dynamics of WUSCHEL (WUS) and CLAVATA3 (CLV3) expression during in vitro meristematic conversion of roots into shoot apical meristems. Thus, we showed that shoot apical meristems could arise from two different types of root structures that differed by their CLV3 gene expression patterns. We constructed 3D cellular representations of WUS and CLV3 gene co-expression pattern, and stressed the variability inherent to meristem conversion. Thus, this protocol generates a large amount of data on the localization of gene expression, which can be used to model complex systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576400">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576400" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576400">
        <p class="paperTitle">Bioengineering a plant NLR immune receptor with a robust binding interface towards a conserved fungal pathogen effector</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576400" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576400" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zdrzalek, R.; Xi, Y.; Langner, T.; Bentham, A. R.; Petit-Houdenot, Y.; De la Concepcion, J. C.; Harant, A.; Shimizu, M.; Were, V.; Talbot, N. J.; Terauchi, R. J.; Kamoun, S.; Banfield, M. J.</p>
        <p class="info">Score: 19.3, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576400' target='https://doi.org/10.1101/2024.01.20.576400'> 10.1101/2024.01.20.576400</a></p>
        <p class="abstract">Bioengineering of plant immune receptors has emerged as a key strategy for generating novel disease resistance traits to counteract the expanding threat of plant pathogens to global food security. However, current approaches are limited by rapid evolution of plant pathogens in the field and may lack durability when deployed. Here, we show that the rice nucleotide-binding, leucine-rich repeat (NLR) immune receptor Pik-1 can be engineered to respond to a conserved family of effectors from the multihost blast fungus pathogen Magnaporthe oryzae. We switched the effector binding and response profile of the Pik NLR from its cognate rice blast effector AVR-Pik to the host-determining factor Pwl2 by installing a putative host target, OsHIPP43, in place of the native integrated HMA domain (generating Pikm-1OsHIPP43). This chimeric receptor also responded to other PWL alleles from diverse blast isolates. The crystal structure of the Pwl2/OsHIPP43 complex revealed a multifaceted, robust interface that cannot be easily disrupted by mutagenesis, and may therefore provide durable, broad resistance to blast isolates carrying PWL effectors in the field. Our findings highlight how the host targets of pathogen effectors can be used to bioengineer new recognition specificities that have more robust properties compared to naturally evolved disease resistance genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576090">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576090" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576090">
        <p class="paperTitle">Speedy benthi: Elongated photoperiods reduce the generation time of the model plant species Nicotiana benthamiana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576090" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576090" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Castle, M.; Luedke, D.</p>
        <p class="info">Score: 18.9, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576090' target='https://doi.org/10.1101/2024.01.18.576090'> 10.1101/2024.01.18.576090</a></p>
        <p class="abstract">Nicotiana benthamiana is increasingly gaining prominence as a model plant species with recently published high-quality genome assemblies, which will further enable forward and reverse genetic approaches (Bally et al., 2018; Derevnina et al., 2019; Kourelis et al., 2019; Ranawaka et al., 2023; Vollheyde et al., 2023). However, the generation time of N. benthamiana poses a bottleneck in the creation of mutant and transgenic plant lines. Speed breeding (SB), by extended photoperiods and adjustments to growth parameters, is an efficient way to reduce generation times for many crop and model plant species (Ghosh et al., 2018; Watson et al., 2018; Hickey et al., 2019; Varshney et al., 2021). We hypothesized that an extended photoperiod could reduce the seed to seed generation time of N. benthamiana. We tested this hypothesis by comparing generation times under SB conditions to traditionally used photoperiods in growth chambers and green house settings. We found that a 22h photoperiod reduced the generation time of N. benthamiana by approximately 2 weeks (16-22%). Fertilization in combination with a far-red light spectrum did not yield a further reduction in generation time when combined with SB conditions. Our findings further contribute to the establishment of N. benthamiana as an important model organism for plant research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576406">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576406" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576406">
        <p class="paperTitle">The blast effector Pwl2 is a virulence factor that modifies the cellular localisation of host protein HIPP43 to suppress immunity.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576406" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576406" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Were, V.; Yan, X.; Foster, A.; Sklenar, J.; Langner, T.; Bentham, A.; Zdrzałek, R.; Ryder, L.; Kaimenyi, D.; Gomez De La Cruz, D.; Gentle, A.; Petit-Houdenot, Y.; Eseola, A. B.; Smoker, M.; Bautista, M. J.; Ma, W.; Kourelis, J.; Maclean, D.; Banfield, M. J.; Kamoun, S.; Menke, F. L. H.; Moscou, M. J.; Talbot, N. J.</p>
        <p class="info">Score: 14.8, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576406' target='https://doi.org/10.1101/2024.01.20.576406'> 10.1101/2024.01.20.576406</a></p>
        <p class="abstract">The rice blast fungus Magnaporthe oryzae secretes a battery of effector proteins during host infection to suppress plant immunity and promote disease. Among these effectors, the MAX effector-Pwl2 was first identified as a host specificity determinant for infection of weeping lovegrass (Eragrostis curvula). However, its biological activity has remained unknown. Here we show that the PWL2 gene is ubiquitous in all host-limited forms of M. oryzae and has undergone substantial copy number expansion indicating that it is a core effector of the blast fungus. We used CRISPR/Cas9-mediated gene editing to delete all three copies of PWL2 and generate a pwl2 null mutant in M. oryzae strain Guy11. This resulted in gain-of-virulence towards weeping lovegrass, but a reduction in the severity of disease symptoms on rice. To further investigate the virulence mechanism of Pwl2, we showed that transgenic rice and barley lines constitutively expressing PWL2 display suppression of reactive oxygen species generation and increased susceptibility to infection by M. oryzae. Also, we identify the barley and rice heavy metal-binding isoprenylated protein HIPP43 as a target of the Pwl2 effector. Transgenic lines overexpressing HIPP43 exhibit attenuated defense responses and increased susceptibility to blast infection, consistent with the hypothesis that Pwl2 binds HIPP43 to modulate immunity. This binding is coupled to changes in cellular localisation of these proteins. Taken together, our results provide evidence that Pwl2 is a virulence factor in M. oryzae that acts by suppressing host immunity through binding and re-localising HIPP43.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575864">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575864" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575864">
        <p class="paperTitle">Chromosome-scale genome assembly of bread wheat&#39;s wild relative Triticum timopheevii</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575864" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575864" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grewal, S.; Yang, C.; Scholefield, D.; Ashling, S.; Ghosh, S.; Swarbreck, D.; Collins, J.; Yao, E.; Sen, T. Z.; Wilson, M.; Yant, L.; King, I. P.; King, J.</p>
        <p class="info">Score: 11.9, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575864' target='https://doi.org/10.1101/2024.01.16.575864'> 10.1101/2024.01.16.575864</a></p>
        <p class="abstract">Wheat (Triticum aestivum) is one of the most important food crops with an urgent need for increase in its production to feed the growing world. Triticum timopheevii (2n = 4x = 28) is an allotetraploid wheat wild relative species containing the At and G genomes that has been exploited in many pre-breeding programmes for wheat improvement. In this study, we report the generation of a chromosome-scale reference genome assembly of T. timopheevii accession PI 94760 based on PacBio HiFi reads and chromosome conformation capture (Hi-C). The assembly comprised a total size of 9.35 Gb, featuring a contig N50 of 42.4 Mb, and 166,325 predicted gene models. DNA methylation analysis showed that the G genome had on average more methylated bases than the At genome. The G genome was also more closely related to the S genome of Aegilops speltoides than to the B genome of hexaploid or tetraploid wheat. In summary, the T. timopheevii genome assembly provides a valuable resource for genome-informed discovery of agronomically important genes for food security.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576255">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576255" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576255">
        <p class="paperTitle">Strategy for unlimited cycles of scarless oligonucleotide directed gene editing in Chlamydomonas reinhardtii</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576255" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576255" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ross, I. L.; Budiman, S.; Le, H. P.; Xiong, D.; Hemker, F.; Millen, E. A.; Oey, M.; Hankamer, B.</p>
        <p class="info">Score: 7.5, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576255' target='https://doi.org/10.1101/2024.01.18.576255'> 10.1101/2024.01.18.576255</a></p>
        <p class="abstract">CRISPR/Cas9 gene editing in the model green alga Chlamydomonas reinhardtii relies on the use of selective marker genes to enrich for non-selectable target mutations. This becomes challenging when many sequential modifications are required in a single cell line, as useful markers are limited. Here we demonstrate a cyclical selection process which only requires a single marker gene to identify an almost infinite sequential series of CRISPR-based target gene modifications. The NIA1 (Nit1, NR; nitrate reductase) gene was this selectable marker. In the forward stage of the cycle, a stop codon was engineered into the NIA1 gene at the CRISPR target location. Cells retaining the wild type NIA1 gene were killed by chlorate, while NIA1 knockout mutants survived. In the reverse phase of the cycle, the stop codon engineered into the NIA1 gene during the forward phase was edited back to the wild type sequence. Using nitrate as the sole nitrogen source, here only the reverted wild type cells survived. By using CRISPR to specifically deactivate and reactivate the NIA1 gene, a marker system was established that flipped back and forth between chlorate- and auxotrophic (nitrate) based selection. This provided a scarless cyclical marker system that enabled an indefinite series of CRISPR edits in other, non-selectable genes. Here, we demonstrate that this  Sequential CRISPR via Recycling Endogenous Auxotrophic Markers (SCREAM) technology enables an essentially limitless series of genetic modifications to be introduced to a single cell lineage of C. reinhardtii in a fast and efficient manner to complete complex genetic engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.572987">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.572987" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.572987">
        <p class="paperTitle">Abscisic acid signaling gates salt-specific responses of plant roots</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.572987" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.572987" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lamers, J.; Zhang, Y.; van Zelm, E.; Meyer, A. J.; De Zeeuw, T.; Verstappen, F.; Veen, M.; Deolu-Ajayi, A. O.; Gommers, C. M. M.; Testerink, C.</p>
        <p class="info">Score: 7.8, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.572987' target='https://doi.org/10.1101/2023.12.28.572987'> 10.1101/2023.12.28.572987</a></p>
        <p class="abstract">Soil salinity presents a dual challenge for plants, involving both osmotic and ionic stress. In response, plants deploy distinct yet interconnected mechanisms to cope with these facets of salinity stress. In this investigation, we observed a substantial overlap in the salt (NaCl)-induced transcriptional responses of Arabidopsis roots with those triggered by osmotic stress or the plant stress hormone abscisic acid (ABA), as anticipated. Notably, a specific cluster of genes responded uniquely to sodium (Na&#43;) ions. Surprisingly, expression of sodium-induced genes exhibited a negative correlation with the ABA response and preceded the activation of genes induced by the osmotic stress component of salt. Elevated exogenous ABA levels resulted in the complete abolition of sodium-induced responses. Consistently, ABA signalling mutants displayed prolonged sodium-induced gene expression, coupled with increased root cell damage under high salinity conditions. Moreover, ABA signalling mutants were unable to redirect root growth to avoid high sodium concentrations and failed to contain their root cell swelling in the presence of elevated salt levels.

In summary, our findings unveil an unexpected and pivotal role for ABA signaling in mitigating cellular damage induced by salinity stress and modulating sodium-specific responses in plant roots.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
