<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Chemical inhibition of stomatal differentiation by perturbation of the master-regulatory bHLH heterodimer via an ACT-Like domain
Authors: Nakagawa, A.; Sepuru, K. M.; Yip, S. J. A.; Seo, H.; Coffin, C. M.; Segawa, Y.; Iwasaki, R.; Kato, H.; Kim, S.; Aihara, Y.; Kinoshita, T.; Itami, K.; Han, S.-K.; Murakami, K.; Torii, K. U.
Score: 29.3, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565226
Selective perturbation of protein interactions with chemical compounds enables dissection and control of developmental processes.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="Chemical inhibition of stomatal differentiation by perturbation of the master-regulatory bHLH heterodimer via an ACT-Like domain
Authors: Nakagawa, A.; Sepuru, K. M.; Yip, S. J. A.; Seo, H.; Coffin, C. M.; Segawa, Y.; Iwasaki, R.; Kato, H.; Kim, S.; Aihara, Y.; Kinoshita, T.; Itami, K.; Han, S.-K.; Murakami, K.; Torii, K. U.
Score: 29.3, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565226
Selective perturbation of protein interactions with chemical compounds enables dissection and control of developmental processes." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-08T10:39:23+00:00" />
<meta property="article:modified_time" content="2023-11-08T10:39:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="Chemical inhibition of stomatal differentiation by perturbation of the master-regulatory bHLH heterodimer via an ACT-Like domain
Authors: Nakagawa, A.; Sepuru, K. M.; Yip, S. J. A.; Seo, H.; Coffin, C. M.; Segawa, Y.; Iwasaki, R.; Kato, H.; Kim, S.; Aihara, Y.; Kinoshita, T.; Itami, K.; Han, S.-K.; Murakami, K.; Torii, K. U.
Score: 29.3, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565226
Selective perturbation of protein interactions with chemical compounds enables dissection and control of developmental processes."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "Chemical inhibition of stomatal differentiation by perturbation of the master-regulatory bHLH heterodimer via an ACT-Like domain\nAuthors: Nakagawa, A.; Sepuru, K. M.; Yip, S. J. A.; Seo, H.; Coffin, C. M.; Segawa, Y.; Iwasaki, R.; Kato, H.; Kim, S.; Aihara, Y.; Kinoshita, T.; Itami, K.; Han, S.-K.; Murakami, K.; Torii, K. U.\nScore: 29.3, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565226\nSelective perturbation of protein interactions with chemical compounds enables dissection and control of developmental processes.",
  "keywords": [
    
  ],
  "articleBody": " Chemical inhibition of stomatal differentiation by perturbation of the master-regulatory bHLH heterodimer via an ACT-Like domain\nAuthors: Nakagawa, A.; Sepuru, K. M.; Yip, S. J. A.; Seo, H.; Coffin, C. M.; Segawa, Y.; Iwasaki, R.; Kato, H.; Kim, S.; Aihara, Y.; Kinoshita, T.; Itami, K.; Han, S.-K.; Murakami, K.; Torii, K. U.\nScore: 29.3, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565226\nSelective perturbation of protein interactions with chemical compounds enables dissection and control of developmental processes. Differentiation of stomata, cellular valves vital for plant growth and survival, is specified by the basic-helix-loop-helix (bHLH) heterodimers. Harnessing a new amination reaction, we here report a synthesis, target identification, and mode of action of an atypical doubly-sulfonylated imidazolone, Stomidazolone, which triggers stomatal stem cell arrest. Our forward chemical genetics followed by biophysical analyses elucidated that Stomidazolone directly binds to the C-terminal ACT-Like (ACTL) domain of MUTE, a master regulator of stomatal differentiation, and perturbs its heterodimerization with a partner bHLH, SCREAM. Guided by structural docking modeling, we rationally designed MUTE with reduced Stomidazolone binding. These engineered MUTE proteins are fully functional and confer Stomidazolone resistance in vivo. Our study identifies doubly-sulfonylated imidazolone as a direct inhibitor of the stomatal master regulator, further expanding the chemical space for perturbing bHLH-ACTL proteins to manipulate plant development.\nDELLA Proteins Recruit the Mediator Complex Subunit MED15 to Co-activate Transcription in Land Plants\nAuthors: Hernandez-Garcia, J.; Serrano-Mislata, A.; Lozano-Quiles, M.; Urbez, C.; Nohales, M. A.; Blanco-Tourinan, N.; Peng, H.; Ledesma-Amaro, R.; Blazquez, M. A.\nScore: 28.1, Published: 2023-11-01 DOI: 10.1101/2023.11.01.565078\nDELLA proteins are negative regulators of the gibberellin response pathway in angiosperms, acting as central hubs that interact with hundreds of transcription factors and regulators to modulate their activities. While the mechanism of transcription factor sequestration by DELLAs to prevent DNA binding to downstream targets has been extensively documented, the mechanism that allows them to act as co-activators remains to be understood. Here, we demonstrate that DELLAs directly recruit the Mediator complex to specific loci in Arabidopsis, facilitating transcription. This recruitment involves DELLA amino-terminal domain and the conserved MED15 KIX domain. Accordingly, partial loss of MED15 function mainly disrupted processes known to rely on DELLA co-activation capacity; including cytokinin-dependent regulation of meristem function and skotomorphogenic response, gibberellin metabolism feedback, and flavonol production. We have also found that the single DELLA protein in the liverwort Marchantia polymorpha is capable of recruiting MpMED15 subunits, contributing to transcriptional co-activation. The conservation of Mediator-dependent transcriptional co-activation by DELLA between Arabidopsis and Marchantia implies that this mechanism is intrinsic to the emergence of DELLA in the last common ancestor of land plants. Significance StatementDELLA proteins are plant-specific transcriptional hubs integrating environmental signals with endogenous cues. In order to regulate downstream processes, DELLAs modulate the activity of hundreds of transcription factors and transcriptional regulators in various ways. Here, we describe the molecular mechanism underlying DELLA co-activator function. We show that DELLAs act as transcriptional activators in eukaryotic cells by interacting with the Mediator complex subunit MED15. Mediator function is necessary to regulate a subset of DELLA-regulated responses that are mediated by direct co-activation of DELLA-Transcription factors complexes. We further show that this mechanism is present in bryophyte DELLAs, and thus represents a conserved mechanism of DELLA function in land plants.\nPopulation level gene expression can repeatedly link genes to functions in maize\nAuthors: Torres-Rodriguez, J. V.; Li, D.; Turkus, J.; Newton, L.; Davis, J.; Lopez-Corona, L.; Ali, W.; Sun, G.; Mural, R. V.; Grzybowski, M. W.; Thompson, A. M.; Schnable, J. C.\nScore: 27.7, Published: 2023-11-01 DOI: 10.1101/2023.10.31.565032\nTranscriptome-Wide Association Studies (TWAS) can provide single gene resolution for candidate genes in plants, complementing Genome-Wide Association Studies (GWAS) but efforts in plants have been met with, at best, mixed success. We generated expression data from 693 maize genotypes, measured in a common field experiment, sampled over a two-hour period to minimize diurnal and environmental effects, using full-length RNA-seq to maximize the accurate estimation of transcript abundance. TWAS could identify roughly ten times as many genes likely to play a role in flowering time regulation as GWAS conducted data from the same experiment. TWAS using mature leaf tissue identified known true positive flowering time genes known to act in the shoot apical meristem, and trait data from new environments enabled the identification of additional flowering time genes without the need for new expression data. eQTL analysis of TWAS-tagged genes identified at least one additional known maize flowering time gene through trans-eQTL interactions. Collectively these results suggest the gene expression resource described here can link genes to functions across different plant phenotypes expressed in a range of tissues and scored in different experiments.\nConservation of molecular responses upon viral infection in the non-vascular plant Marchantia polymorpha\nAuthors: Moner-Ross, E.; Jimenez-Gongora, T.; Villar-Martin, L.; Gonzalez-Miguel, V. M.; kutnjak, D.; Rubio-Somoza, I.\nScore: 23.6, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565823\nAfter their transition from water to land around 450 million years ago, plants colonized new habitats facing unprecedented pathogenic microbes. That expansion was mostly supported by a growing anatomical complexity based on the acquisition of developmental innovations, such as roots, stomata and vascular tissue. Despite several of those innovations became central for the interaction between plants and their associated microbes, little is known about their impact on plant immune programs and on the diversification of infection strategies of their pathogens. A paradigmatic case is the close relationship between plant vasculature and viruses. Vascular tissues provide a unique cellular environment for viral replication and existence, besides constituting a fast track for viral systemic spread throughout the plant. Since most of our knowledge about plant-virus interactions come from studies in vascular plants, we here present a comparative study to contribute to the understanding of the evolution of plant- virus interactions by molecularly characterizing the interplay between the bryophyte Marchantia polymorpha and viruses. Virome analysis of Marchantia plants shows that they are primarily associated with RNA viruses in natural settings. Additional molecular characterization of the interaction between Marchantia and tobacco mosaic virus (TMV) show conserved basic processes with vascular plants and divergent features. Viral infection triggers an extensive transcriptional reprogramming in Marchantia encompassing broad range defence responses, inhibition of cell cycle and photosynthesis and a sustained wound response that prevents further viral movement. Additionally, infected plants show premature aging and organ maturation. Notably, we found that some core responses that occur in infected areas in Marchantia, were described to be restricted to vascular tissues in Nicotiana, suggesting that evolutionary appearance of developmental innovations that became central in plants-virus interactions resulted in re-routing of defence responses. Finally, we uncover the conserved role of a transcription factor interacting with the TMV silencing suppressor p126 in specifically abrogating TMV infection.\nTargeting editing of tomato SPEECHLESS cis-regulatory regions generates plants with altered stomatal density in response to changing climate conditions\nAuthors: Nir, I.; Budrys, A.; Smoot, N. K.; Erberich, J.; Bergmann, D.\nScore: 22.3, Published: 2023-11-02 DOI: 10.1101/2023.11.02.564550\nFlexible developmental programs enable plants to customize their organ size and cellular composition. In leaves of eudicots, the stomatal lineage produces two essential cell types, stomata and pavement cells, but the total numbers and ratio of these cell types can vary. Central to this flexibility is the stomatal lineage initiating transcription factor, SPEECHLESS (SPCH). Here we show, by multiplex CRISPR/Cas9 editing of SlSPCH cis-regulatory sequences in tomato, that we can identify variants with altered stomatal development responses to light and temperature cues. Analysis of tomato leaf development across different conditions, aided by newly-created tools for live-cell imaging and translational reporters of SlSPCH and its paralogues SlMUTE and SlFAMA, revealed the series of cellular events that lead to the environmental change-driven responses in leaf form. Plants bearing the novel SlSPCH variants generated in this study are powerful resources for fundamental and applied studies of tomato resilience in response to climate change. Significance statementPlants can change their shape, size and cellular composition in response to environmental cues. Here, by precise gene editing of a core stomatal development regulator gene in tomato, we generate new alleles with enhanced or dampened responses to light and temperature cues. Combined with live imaging of development, we show the genetic and cellular pathways that contribute to customization of the leaf epidermis, and how this could lead to better climate-adapted varieties.\nATG8 delipidation is dispensable for plant autophagy\nAuthors: Holla, S.; Zou, Y.; Sabljic, I.; Ohlsson, J. A.; Leong, J. X.; Ballhaus, F.; Krebs, M.; Schumacher, K.; Uestuen, S.; Dagdas, Y.; Bozhkov, P.; Minina, E. A.\nScore: 17.6, Published: 2023-11-03 DOI: 10.1101/2023.08.23.554513\nIntracellular recycling via autophagy is governed by post-translational modifications of the autophagy related (ATG) proteins. One example is ATG4-dependent delipidation of ATG8, which assumes critical but distinct roles in autophagosome formation of yeast and mammals. Here, we discovered that ATG8 delipidation is dispensable for the maturation of autophagosomes, autophagic flux and related stress tolerance in Arabidopsis. In contrast, autophagy in an evolutionary distant Chlamydomonas strictly required ATG4-dependent ATG8 delipidation. Additionally, we uncovered distinct participation of Arabidopsis ATG8 orthologs in autophagosome biogenesis, especially prevalent under nitrogen depletion, thereby providing new insights into potential reasons behind ATG8 gene family expansion in vascular plants. Our findings point to evolutionary diversification of the molecular mechanism governing maturation of autophagosomes in eukaryotic lineages and highlight how this conserved pathway is tailored to diverse eukaryotic organisms.\nA Ralstonia solanacearum type III effector alters the actin and microtubule cytoskeleton to promote bacterial virulence in plants\nAuthors: Hiles, R.; Rogers, A.; Jaiswal, N.; Zhang, W.; Butchacas, J.; Merfa, M.; Klass, T.; Kaser, E.; Jacobs, J. M.; Staiger, C. J.; Helm, M. J.; Iyer-Pascuzzi, A. S.\nScore: 14.3, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565113\nCellular responses to biotic stress frequently involve signaling pathways that are conserved across eukaryotes. These pathways include the cytoskeleton, a proteinaceous network that senses external cues at the cell surface and signals to interior cellular components. During biotic stress, dynamic cytoskeletal rearrangements serve as a platform from which early immune-associated processes are organized and activated. Bacterial pathogens of plants and animals use proteins called type III effectors (T3Es) to interfere with host immune signaling, thereby promoting virulence. We previously found that RipU, a T3E from the soilborne phytobacterial pathogen Ralstonia solanacearum K60 (Rs K60), co-localizes with the plant cytoskeleton. Here, we show that RipU from Rs K60 (RipUK60) physically associates with both actin and tubulin and disrupts actin and microtubule cytoskeleton organization. We find that pharmacological disruption of the tomato (Solanum lycopersicum) cytoskeleton promotes Rs K60 colonization. RipUK60 suppresses cell surface-triggered immune responses including flg22-mediated reactive oxygen species (ROS) production and callose deposition. Importantly, tomato plants inoculated with Rs K60 lacking RipUK60 ({Delta}ripUK60) had reduced wilting symptoms and significantly reduced root colonization when compared to plants inoculated with wild-type Rs K60. Collectively, our data suggest that Rs K60 uses the type III effector RipUK60 to remodel cytoskeletal organization, thereby promoting pathogen virulence.\nGenome-wide evidence of the role of Paf1C in transcription elongation and histone H2B monoubiquitination in Arabidopsis\nAuthors: Blanco, N.; Perez-Alemany, J.; Bourbousse, C.; latrasse, D.; Ait-Mohamed, O.; Benhamed, M.; Barneche, F.; Blazquez, M. A.; Gallego-Bartolome, J.; Alabadi, D.\nScore: 12.7, Published: 2023-10-31 DOI: 10.1101/2023.10.27.564443\nThe evolutionarily conserved Paf1 complex (Paf1C) participates in transcription, and research in animals and fungi suggests that it facilitates RNAPII progression through chromatin remodeling. To obtain evidence that Paf1C acts in transcription elongation in Arabidopsis, we examined the genomic distribution of the ELF7 and VIP3 subunits of Paf1C. The occupancy of both subunits was confined to thousands of gene bodies and positively correlated with RNAPII occupancy and the level of gene expression, supporting a role as a transcription elongation factor. We found that monoubiquitinated histone H2B, which marks most transcribed genes, was strongly reduced genome-wide in elf7 seedlings. Genome-wide profiling of RNAPII revealed that in elf7 mutants, RNAPII occupancy was reduced throughout the gene body and at the transcription end site of Paf1C-targeted genes, suggesting a direct role for the complex in transcription elongation. Overall, our observations suggest that there is a direct functional link between Paf1C activity, mono-ubiquitination of histone H2B, and the transition of RNPII to productive elongation. However, for several genes, Paf1C may also act independently of H2Bub deposition or occupy these genes more stably than H2Bub marking, possibly reflecting the dynamic nature of Paf1C association and H2Bub turnover during transcription.\nPan-genome analysis of different morphotypes reveals genomic basis of Brassica oleracea domestication and differential organogenesis\nAuthors: Guo, N.; Wang, S.; Wang, T.; Duan, M.; Zong, M.; Miao, L.; Han, S.; Wang, G.; Liu, X.; Zhang, D.; Jiao, C.; Xu, H.; Chen, L.; Fei, Z.; Li, J.; Liu, F.\nScore: 11.3, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563711\nThe domestication of Brassica oleracea has resulted in diverse morphological types with distinct patterns of organ development. Here we report a graph-based pan-genome of B. oleracea constructed with high-quality genome assemblies of different morphotypes. The pan-genome harbors over 200 structural variant (SV) hotspot regions enriched with auxin and flowering-related genes. Population genomic analyses reveal that early domestication of B. oleracea focused on leaf or stem selection. Gene flows resulting from agricultural practices and variety improvement are detected among different morphotypes. Selective sweep analysis identifies an auxin-responsive SAUR gene and a CLE family gene as the crucial players in the leaf-stem differentiation during the early stage of B. oleracea domestication, and the BoKAN1 gene as instrumental in shaping the leafy heads of cabbage and Brussels sprouts. Our pan-genome and functional analyses further discover that variations in the BoFLC2 gene play key roles in the divergence of vernalization and flowering characteristics among different morphotypes, and variations in the first intron of BoFLC3 are involved in fine-tuning the flowering process in cauliflower. This study provides a comprehensive understanding of the pan-genome of B. oleracea and sheds light on the domestication and differential organ development of this globally important crop species.\nChloroplasts lacking class I glutaredoxins are functional but show a delayed recovery of protein cysteinyl redox state after oxidative challenge\nAuthors: Bohle, F.; Rossi, J.; Tamanna, S. S.; Jansohn, H.; Schlosser, M.; Reinhardt, F.; Brox, A.; Bethmann, S.; Kopriva, S.; Trentmann, O.; Jahns, P.; Deponte, M.; Schwarzländer, M.; Trost, P.; Zaffagnini, M.; Meyer, A. J.; Müller-Schüssele, S. J.\nScore: 8.9, Published: 2023-11-01 DOI: 10.1101/2023.10.31.564817\nRedox status of protein cysteinyl residues is mediated via glutathione (GSH)/glutaredoxin (GRX) and thioredoxin (TRX)-dependent redox cascades. An oxidative challenge can induce post-translational protein modifications on thiols, such as protein S-glutathionylation. Class I GRX are small thiol-disulfide oxidoreductases that reversibly catalyse S-glutathionylation and protein disulfide formation. TRX and GSH/GRX redox systems can provide partial backup for each other in several subcellular compartments, but not in the plastid stroma where TRX/light-dependent redox regulation of primary metabolism takes place. While the stromal TRX system has been studied at detail, the role of class I GRX on plastid redox processes in vivo is still unknown. We generate knockout lines of GRXC5 as the only chloroplast class I GRX of the moss Physcomitrium patens. While we find that class I PpGRXC5 has high activities in glutathione-dependent oxidoreductase assays using hydroxyethyl disulfide or redox-sensitive GFP2 (roGFP2) as substrates in vitro, {Delta}grxc5 plants show no detectable growth defect or stress sensitivity, in contrast to mutants with a less negative stromal EGSH ({Delta}gr1). Using stroma-targeted roGFP2, we show increased protein Cys oxidation and decreased reduction rates after oxidative challenge in {Delta}grxc5 plants in vivo, indicating kinetic uncoupling of the protein Cys redox state from glutathione redox potential. Protein Cys disulfide and S-glutathionylation formation rates after H2O2 treatment remained unchanged. Lack of class I GRX function in the stroma did not result in impaired carbon fixation. Our observations suggest specific roles for class I GRX in the efficient redox equilibration between EGSH and protein Cys in the plastid stroma as well as negligible cross-talk with metabolic regulation via the TRX system. We propose a model for stromal class I GRX function as efficient kinetic couplers of protein Cys redox state to the dynamic stromal EGSH and highlight the importance of identifying in vivo target proteins of GRXC5. One sentence summaryRemoval of class I GRX activity in the chloroplast stroma of P. patens kinetically uncouples GRX-dependent cysteine redox changes from the local glutathione redox potential, without an effect on NPQ or photosynthetic carbon reactions.\n",
  "wordCount" : "2674",
  "inLanguage": "en",
  "datePublished": "2023-11-08T10:39:23Z",
  "dateModified": "2023-11-08T10:39:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 8, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565226">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565226" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565226">
        <p class="paperTitle">Chemical inhibition of stomatal differentiation by perturbation of the master-regulatory bHLH heterodimer via an ACT-Like domain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565226" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565226" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nakagawa, A.; Sepuru, K. M.; Yip, S. J. A.; Seo, H.; Coffin, C. M.; Segawa, Y.; Iwasaki, R.; Kato, H.; Kim, S.; Aihara, Y.; Kinoshita, T.; Itami, K.; Han, S.-K.; Murakami, K.; Torii, K. U.</p>
        <p class="info">Score: 29.3, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565226' target='https://doi.org/10.1101/2023.11.02.565226'> 10.1101/2023.11.02.565226</a></p>
        <p class="abstract">Selective perturbation of protein interactions with chemical compounds enables dissection and control of developmental processes. Differentiation of stomata, cellular valves vital for plant growth and survival, is specified by the basic-helix-loop-helix (bHLH) heterodimers. Harnessing a new amination reaction, we here report a synthesis, target identification, and mode of action of an atypical doubly-sulfonylated imidazolone, Stomidazolone, which triggers stomatal stem cell arrest. Our forward chemical genetics followed by biophysical analyses elucidated that Stomidazolone directly binds to the C-terminal ACT-Like (ACTL) domain of MUTE, a master regulator of stomatal differentiation, and perturbs its heterodimerization with a partner bHLH, SCREAM. Guided by structural docking modeling, we rationally designed MUTE with reduced Stomidazolone binding. These engineered MUTE proteins are fully functional and confer Stomidazolone resistance in vivo. Our study identifies doubly-sulfonylated imidazolone as a direct inhibitor of the stomatal master regulator, further expanding the chemical space for perturbing bHLH-ACTL proteins to manipulate plant development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565078">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565078" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565078">
        <p class="paperTitle">DELLA Proteins Recruit the Mediator Complex Subunit MED15 to Co-activate Transcription in Land Plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565078" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565078" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hernandez-Garcia, J.; Serrano-Mislata, A.; Lozano-Quiles, M.; Urbez, C.; Nohales, M. A.; Blanco-Tourinan, N.; Peng, H.; Ledesma-Amaro, R.; Blazquez, M. A.</p>
        <p class="info">Score: 28.1, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565078' target='https://doi.org/10.1101/2023.11.01.565078'> 10.1101/2023.11.01.565078</a></p>
        <p class="abstract">DELLA proteins are negative regulators of the gibberellin response pathway in angiosperms, acting as central hubs that interact with hundreds of transcription factors and regulators to modulate their activities. While the mechanism of transcription factor sequestration by DELLAs to prevent DNA binding to downstream targets has been extensively documented, the mechanism that allows them to act as co-activators remains to be understood. Here, we demonstrate that DELLAs directly recruit the Mediator complex to specific loci in Arabidopsis, facilitating transcription. This recruitment involves DELLA amino-terminal domain and the conserved MED15 KIX domain. Accordingly, partial loss of MED15 function mainly disrupted processes known to rely on DELLA co-activation capacity; including cytokinin-dependent regulation of meristem function and skotomorphogenic response, gibberellin metabolism feedback, and flavonol production. We have also found that the single DELLA protein in the liverwort Marchantia polymorpha is capable of recruiting MpMED15 subunits, contributing to transcriptional co-activation. The conservation of Mediator-dependent transcriptional co-activation by DELLA between Arabidopsis and Marchantia implies that this mechanism is intrinsic to the emergence of DELLA in the last common ancestor of land plants.

Significance StatementDELLA proteins are plant-specific transcriptional hubs integrating environmental signals with endogenous cues. In order to regulate downstream processes, DELLAs modulate the activity of hundreds of transcription factors and transcriptional regulators in various ways. Here, we describe the molecular mechanism underlying DELLA co-activator function. We show that DELLAs act as transcriptional activators in eukaryotic cells by interacting with the Mediator complex subunit MED15. Mediator function is necessary to regulate a subset of DELLA-regulated responses that are mediated by direct co-activation of DELLA-Transcription factors complexes. We further show that this mechanism is present in bryophyte DELLAs, and thus represents a conserved mechanism of DELLA function in land plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.565032">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.565032" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.565032">
        <p class="paperTitle">Population level gene expression can repeatedly link genes to functions in maize</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.565032" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.565032" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Torres-Rodriguez, J. V.; Li, D.; Turkus, J.; Newton, L.; Davis, J.; Lopez-Corona, L.; Ali, W.; Sun, G.; Mural, R. V.; Grzybowski, M. W.; Thompson, A. M.; Schnable, J. C.</p>
        <p class="info">Score: 27.7, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.565032' target='https://doi.org/10.1101/2023.10.31.565032'> 10.1101/2023.10.31.565032</a></p>
        <p class="abstract">Transcriptome-Wide Association Studies (TWAS) can provide single gene resolution for candidate genes in plants, complementing Genome-Wide Association Studies (GWAS) but efforts in plants have been met with, at best, mixed success. We generated expression data from 693 maize genotypes, measured in a common field experiment, sampled over a two-hour period to minimize diurnal and environmental effects, using full-length RNA-seq to maximize the accurate estimation of transcript abundance. TWAS could identify roughly ten times as many genes likely to play a role in flowering time regulation as GWAS conducted data from the same experiment. TWAS using mature leaf tissue identified known true positive flowering time genes known to act in the shoot apical meristem, and trait data from new environments enabled the identification of additional flowering time genes without the need for new expression data. eQTL analysis of TWAS-tagged genes identified at least one additional known maize flowering time gene through trans-eQTL interactions. Collectively these results suggest the gene expression resource described here can link genes to functions across different plant phenotypes expressed in a range of tissues and scored in different experiments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565823">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565823" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565823">
        <p class="paperTitle">Conservation of molecular responses upon viral infection in the non-vascular plant Marchantia polymorpha</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565823" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565823" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moner-Ross, E.; Jimenez-Gongora, T.; Villar-Martin, L.; Gonzalez-Miguel, V. M.; kutnjak, D.; Rubio-Somoza, I.</p>
        <p class="info">Score: 23.6, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565823' target='https://doi.org/10.1101/2023.11.06.565823'> 10.1101/2023.11.06.565823</a></p>
        <p class="abstract">After their transition from water to land around 450 million years ago, plants colonized new habitats facing unprecedented pathogenic microbes. That expansion was mostly supported by a growing anatomical complexity based on the acquisition of developmental innovations, such as roots, stomata and vascular tissue. Despite several of those innovations became central for the interaction between plants and their associated microbes, little is known about their impact on plant immune programs and on the diversification of infection strategies of their pathogens. A paradigmatic case is the close relationship between plant vasculature and viruses. Vascular tissues provide a unique cellular environment for viral replication and existence, besides constituting a fast track for viral systemic spread throughout the plant. Since most of our knowledge about plant-virus interactions come from studies in vascular plants, we here present a comparative study to contribute to the understanding of the evolution of plant- virus interactions by molecularly characterizing the interplay between the bryophyte Marchantia polymorpha and viruses. Virome analysis of Marchantia plants shows that they are primarily associated with RNA viruses in natural settings. Additional molecular characterization of the interaction between Marchantia and tobacco mosaic virus (TMV) show conserved basic processes with vascular plants and divergent features. Viral infection triggers an extensive transcriptional reprogramming in Marchantia encompassing broad range defence responses, inhibition of cell cycle and photosynthesis and a sustained wound response that prevents further viral movement. Additionally, infected plants show premature aging and organ maturation. Notably, we found that some core responses that occur in infected areas in Marchantia, were described to be restricted to vascular tissues in Nicotiana, suggesting that evolutionary appearance of developmental innovations that became central in plants-virus interactions resulted in re-routing of defence responses. Finally, we uncover the conserved role of a transcription factor interacting with the TMV silencing suppressor p126 in specifically abrogating TMV infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.564550">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.564550" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.564550">
        <p class="paperTitle">Targeting editing of tomato SPEECHLESS cis-regulatory regions generates plants with altered stomatal density in response to changing climate conditions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.564550" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.564550" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nir, I.; Budrys, A.; Smoot, N. K.; Erberich, J.; Bergmann, D.</p>
        <p class="info">Score: 22.3, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.564550' target='https://doi.org/10.1101/2023.11.02.564550'> 10.1101/2023.11.02.564550</a></p>
        <p class="abstract">Flexible developmental programs enable plants to customize their organ size and cellular composition. In leaves of eudicots, the stomatal lineage produces two essential cell types, stomata and pavement cells, but the total numbers and ratio of these cell types can vary. Central to this flexibility is the stomatal lineage initiating transcription factor, SPEECHLESS (SPCH). Here we show, by multiplex CRISPR/Cas9 editing of SlSPCH cis-regulatory sequences in tomato, that we can identify variants with altered stomatal development responses to light and temperature cues. Analysis of tomato leaf development across different conditions, aided by newly-created tools for live-cell imaging and translational reporters of SlSPCH and its paralogues SlMUTE and SlFAMA, revealed the series of cellular events that lead to the environmental change-driven responses in leaf form. Plants bearing the novel SlSPCH variants generated in this study are powerful resources for fundamental and applied studies of tomato resilience in response to climate change.

Significance statementPlants can change their shape, size and cellular composition in response to environmental cues. Here, by precise gene editing of a core stomatal development regulator gene in tomato, we generate new alleles with enhanced or dampened responses to light and temperature cues. Combined with live imaging of development, we show the genetic and cellular pathways that contribute to customization of the leaf epidermis, and how this could lead to better climate-adapted varieties.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554513">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554513" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554513">
        <p class="paperTitle">ATG8 delipidation is dispensable for plant autophagy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554513" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554513" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Holla, S.; Zou, Y.; Sabljic, I.; Ohlsson, J. A.; Leong, J. X.; Ballhaus, F.; Krebs, M.; Schumacher, K.; Uestuen, S.; Dagdas, Y.; Bozhkov, P.; Minina, E. A.</p>
        <p class="info">Score: 17.6, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554513' target='https://doi.org/10.1101/2023.08.23.554513'> 10.1101/2023.08.23.554513</a></p>
        <p class="abstract">Intracellular recycling via autophagy is governed by post-translational modifications of the autophagy related (ATG) proteins. One example is ATG4-dependent delipidation of ATG8, which assumes critical but distinct roles in autophagosome formation of yeast and mammals. Here, we discovered that ATG8 delipidation is dispensable for the maturation of autophagosomes, autophagic flux and related stress tolerance in Arabidopsis. In contrast, autophagy in an evolutionary distant Chlamydomonas strictly required ATG4-dependent ATG8 delipidation. Additionally, we uncovered distinct participation of Arabidopsis ATG8 orthologs in autophagosome biogenesis, especially prevalent under nitrogen depletion, thereby providing new insights into potential reasons behind ATG8 gene family expansion in vascular plants.

Our findings point to evolutionary diversification of the molecular mechanism governing maturation of autophagosomes in eukaryotic lineages and highlight how this conserved pathway is tailored to diverse eukaryotic organisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565113">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565113" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565113">
        <p class="paperTitle">A  Ralstonia solanacearum  type III effector alters the actin and microtubule cytoskeleton to promote bacterial virulence in plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565113" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565113" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hiles, R.; Rogers, A.; Jaiswal, N.; Zhang, W.; Butchacas, J.; Merfa, M.; Klass, T.; Kaser, E.; Jacobs, J. M.; Staiger, C. J.; Helm, M. J.; Iyer-Pascuzzi, A. S.</p>
        <p class="info">Score: 14.3, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565113' target='https://doi.org/10.1101/2023.11.01.565113'> 10.1101/2023.11.01.565113</a></p>
        <p class="abstract">Cellular responses to biotic stress frequently involve signaling pathways that are conserved across eukaryotes. These pathways include the cytoskeleton, a proteinaceous network that senses external cues at the cell surface and signals to interior cellular components. During biotic stress, dynamic cytoskeletal rearrangements serve as a platform from which early immune-associated processes are organized and activated. Bacterial pathogens of plants and animals use proteins called type III effectors (T3Es) to interfere with host immune signaling, thereby promoting virulence. We previously found that RipU, a T3E from the soilborne phytobacterial pathogen Ralstonia solanacearum K60 (Rs K60), co-localizes with the plant cytoskeleton. Here, we show that RipU from Rs K60 (RipUK60) physically associates with both actin and tubulin and disrupts actin and microtubule cytoskeleton organization. We find that pharmacological disruption of the tomato (Solanum lycopersicum) cytoskeleton promotes Rs K60 colonization. RipUK60 suppresses cell surface-triggered immune responses including flg22-mediated reactive oxygen species (ROS) production and callose deposition. Importantly, tomato plants inoculated with Rs K60 lacking RipUK60 ({Delta}ripUK60) had reduced wilting symptoms and significantly reduced root colonization when compared to plants inoculated with wild-type Rs K60. Collectively, our data suggest that Rs K60 uses the type III effector RipUK60 to remodel cytoskeletal organization, thereby promoting pathogen virulence.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564443">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564443" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564443">
        <p class="paperTitle">Genome-wide evidence of the role of Paf1C in transcription elongation and histone H2B monoubiquitination in Arabidopsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564443" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564443" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Blanco, N.; Perez-Alemany, J.; Bourbousse, C.; latrasse, D.; Ait-Mohamed, O.; Benhamed, M.; Barneche, F.; Blazquez, M. A.; Gallego-Bartolome, J.; Alabadi, D.</p>
        <p class="info">Score: 12.7, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564443' target='https://doi.org/10.1101/2023.10.27.564443'> 10.1101/2023.10.27.564443</a></p>
        <p class="abstract">The evolutionarily conserved Paf1 complex (Paf1C) participates in transcription, and research in animals and fungi suggests that it facilitates RNAPII progression through chromatin remodeling. To obtain evidence that Paf1C acts in transcription elongation in Arabidopsis, we examined the genomic distribution of the ELF7 and VIP3 subunits of Paf1C. The occupancy of both subunits was confined to thousands of gene bodies and positively correlated with RNAPII occupancy and the level of gene expression, supporting a role as a transcription elongation factor. We found that monoubiquitinated histone H2B, which marks most transcribed genes, was strongly reduced genome-wide in elf7 seedlings. Genome-wide profiling of RNAPII revealed that in elf7 mutants, RNAPII occupancy was reduced throughout the gene body and at the transcription end site of Paf1C-targeted genes, suggesting a direct role for the complex in transcription elongation. Overall, our observations suggest that there is a direct functional link between Paf1C activity, mono-ubiquitination of histone H2B, and the transition of RNPII to productive elongation. However, for several genes, Paf1C may also act independently of H2Bub deposition or occupy these genes more stably than H2Bub marking, possibly reflecting the dynamic nature of Paf1C association and H2Bub turnover during transcription.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563711">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563711" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563711">
        <p class="paperTitle">Pan-genome analysis of different morphotypes reveals genomic basis of Brassica oleracea domestication and differential organogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563711" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563711" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guo, N.; Wang, S.; Wang, T.; Duan, M.; Zong, M.; Miao, L.; Han, S.; Wang, G.; Liu, X.; Zhang, D.; Jiao, C.; Xu, H.; Chen, L.; Fei, Z.; Li, J.; Liu, F.</p>
        <p class="info">Score: 11.3, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563711' target='https://doi.org/10.1101/2023.10.24.563711'> 10.1101/2023.10.24.563711</a></p>
        <p class="abstract">The domestication of Brassica oleracea has resulted in diverse morphological types with distinct patterns of organ development. Here we report a graph-based pan-genome of B. oleracea constructed with high-quality genome assemblies of different morphotypes. The pan-genome harbors over 200 structural variant (SV) hotspot regions enriched with auxin and flowering-related genes. Population genomic analyses reveal that early domestication of B. oleracea focused on leaf or stem selection. Gene flows resulting from agricultural practices and variety improvement are detected among different morphotypes. Selective sweep analysis identifies an auxin-responsive SAUR gene and a CLE family gene as the crucial players in the leaf-stem differentiation during the early stage of B. oleracea domestication, and the BoKAN1 gene as instrumental in shaping the leafy heads of cabbage and Brussels sprouts. Our pan-genome and functional analyses further discover that variations in the BoFLC2 gene play key roles in the divergence of vernalization and flowering characteristics among different morphotypes, and variations in the first intron of BoFLC3 are involved in fine-tuning the flowering process in cauliflower. This study provides a comprehensive understanding of the pan-genome of B. oleracea and sheds light on the domestication and differential organ development of this globally important crop species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.564817">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.564817" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.564817">
        <p class="paperTitle">Chloroplasts lacking class I glutaredoxins are functional but show a delayed recovery of protein cysteinyl redox state after oxidative challenge</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.564817" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.564817" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bohle, F.; Rossi, J.; Tamanna, S. S.; Jansohn, H.; Schlosser, M.; Reinhardt, F.; Brox, A.; Bethmann, S.; Kopriva, S.; Trentmann, O.; Jahns, P.; Deponte, M.; Schwarzländer, M.; Trost, P.; Zaffagnini, M.; Meyer, A. J.; Müller-Schüssele, S. J.</p>
        <p class="info">Score: 8.9, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.564817' target='https://doi.org/10.1101/2023.10.31.564817'> 10.1101/2023.10.31.564817</a></p>
        <p class="abstract">Redox status of protein cysteinyl residues is mediated via glutathione (GSH)/glutaredoxin (GRX) and thioredoxin (TRX)-dependent redox cascades. An oxidative challenge can induce post-translational protein modifications on thiols, such as protein S-glutathionylation. Class I GRX are small thiol-disulfide oxidoreductases that reversibly catalyse S-glutathionylation and protein disulfide formation. TRX and GSH/GRX redox systems can provide partial backup for each other in several subcellular compartments, but not in the plastid stroma where TRX/light-dependent redox regulation of primary metabolism takes place. While the stromal TRX system has been studied at detail, the role of class I GRX on plastid redox processes in vivo is still unknown. We generate knockout lines of GRXC5 as the only chloroplast class I GRX of the moss Physcomitrium patens.

While we find that class I PpGRXC5 has high activities in glutathione-dependent oxidoreductase assays using hydroxyethyl disulfide or redox-sensitive GFP2 (roGFP2) as substrates in vitro, {Delta}grxc5 plants show no detectable growth defect or stress sensitivity, in contrast to mutants with a less negative stromal EGSH ({Delta}gr1). Using stroma-targeted roGFP2, we show increased protein Cys oxidation and decreased reduction rates after oxidative challenge in {Delta}grxc5 plants in vivo, indicating kinetic uncoupling of the protein Cys redox state from glutathione redox potential. Protein Cys disulfide and S-glutathionylation formation rates after H2O2 treatment remained unchanged. Lack of class I GRX function in the stroma did not result in impaired carbon fixation.

Our observations suggest specific roles for class I GRX in the efficient redox equilibration between EGSH and protein Cys in the plastid stroma as well as negligible cross-talk with metabolic regulation via the TRX system. We propose a model for stromal class I GRX function as efficient kinetic couplers of protein Cys redox state to the dynamic stromal EGSH and highlight the importance of identifying in vivo target proteins of GRXC5.

One sentence summaryRemoval of class I GRX activity in the chloroplast stroma of P. patens kinetically uncouples GRX-dependent cysteine redox changes from the local glutathione redox potential, without an effect on NPQ or photosynthetic carbon reactions.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
