<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="An oomycete effector co-opts a host RabGAP protein to remodel pathogen interface and subvert defense-related secretion
Authors: Yuen, E. L. H.; Tumtas, Y.; Chan, L. I.; Ibrahim, T.; Evangelisti, E.; Tulin, F.; Sklenar, J.; Menke, F.; Kamoun, S.; Bubeck, D.; Schornack, S.; Bozkurt, T. O.
Score: 22.5, Published: 2024-01-15 DOI: 10.1101/2024.01.11.575225
Pathogens have evolved sophisticated mechanisms to manipulate host cell membrane dynamics, a crucial adaptation to survive in hostile environments shaped by innate immune responses.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="An oomycete effector co-opts a host RabGAP protein to remodel pathogen interface and subvert defense-related secretion
Authors: Yuen, E. L. H.; Tumtas, Y.; Chan, L. I.; Ibrahim, T.; Evangelisti, E.; Tulin, F.; Sklenar, J.; Menke, F.; Kamoun, S.; Bubeck, D.; Schornack, S.; Bozkurt, T. O.
Score: 22.5, Published: 2024-01-15 DOI: 10.1101/2024.01.11.575225
Pathogens have evolved sophisticated mechanisms to manipulate host cell membrane dynamics, a crucial adaptation to survive in hostile environments shaped by innate immune responses." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-17T10:40:22+00:00" />
<meta property="article:modified_time" content="2024-01-17T10:40:22+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="An oomycete effector co-opts a host RabGAP protein to remodel pathogen interface and subvert defense-related secretion
Authors: Yuen, E. L. H.; Tumtas, Y.; Chan, L. I.; Ibrahim, T.; Evangelisti, E.; Tulin, F.; Sklenar, J.; Menke, F.; Kamoun, S.; Bubeck, D.; Schornack, S.; Bozkurt, T. O.
Score: 22.5, Published: 2024-01-15 DOI: 10.1101/2024.01.11.575225
Pathogens have evolved sophisticated mechanisms to manipulate host cell membrane dynamics, a crucial adaptation to survive in hostile environments shaped by innate immune responses."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "An oomycete effector co-opts a host RabGAP protein to remodel pathogen interface and subvert defense-related secretion\nAuthors: Yuen, E. L. H.; Tumtas, Y.; Chan, L. I.; Ibrahim, T.; Evangelisti, E.; Tulin, F.; Sklenar, J.; Menke, F.; Kamoun, S.; Bubeck, D.; Schornack, S.; Bozkurt, T. O.\nScore: 22.5, Published: 2024-01-15 DOI: 10.1101/2024.01.11.575225\nPathogens have evolved sophisticated mechanisms to manipulate host cell membrane dynamics, a crucial adaptation to survive in hostile environments shaped by innate immune responses.",
  "keywords": [
    
  ],
  "articleBody": " An oomycete effector co-opts a host RabGAP protein to remodel pathogen interface and subvert defense-related secretion\nAuthors: Yuen, E. L. H.; Tumtas, Y.; Chan, L. I.; Ibrahim, T.; Evangelisti, E.; Tulin, F.; Sklenar, J.; Menke, F.; Kamoun, S.; Bubeck, D.; Schornack, S.; Bozkurt, T. O.\nScore: 22.5, Published: 2024-01-15 DOI: 10.1101/2024.01.11.575225\nPathogens have evolved sophisticated mechanisms to manipulate host cell membrane dynamics, a crucial adaptation to survive in hostile environments shaped by innate immune responses. Plant-derived membrane interfaces, engulfing invasive hyphal projections of fungal and oomycete pathogens, are prominent junctures dictating infection outcomes. Understanding how pathogens transform these host-pathogen interfaces to their advantage remains a key biological question. Here, we identified a conserved effector, secreted by plant pathogenic oomycetes, that co-opts a host Rab GTPase-activating protein (RabGAP), TBC1D15L, to remodel the host-pathogen interface. The effector, PiE354, hijacks TBC1D15L as a susceptibility factor to usurp its GAP activity on Rab8a - a key Rab GTPase crucial for defense-related secretion. By hijacking TBC1D15L, PiE354 purges Rab8a from the plasma membrane, diverting Rab8a-mediated immune trafficking away from the pathogen interface. This mechanism signifies an uncanny evolutionary adaptation of a pathogen effector in co-opting a host regulatory component to subvert defense-related secretion, thereby providing unprecedented mechanistic insights into the reprogramming of host membrane dynamics by pathogens.\nAnnexin and calcium-regulated priming of legume root cells for endosymbiotic infection\nAuthors: Guillory, A.; Fournier, J.; Kelner, A.; Hobecker, K.; Auriac, M.-C.; Frances, L.; Delers, A.; Le Ru, A.; Keller, J.; Delaux, P.-M.; Gutjahr, C.; Frei Dit Frey, N.; de Carvalho-Niebel, F.\nScore: 14.2, Published: 2024-01-12 DOI: 10.1101/2024.01.10.574854\nLegumes establish endosymbioses with arbuscular mycorrhizal (AM) fungi or rhizobia bacteria to improve mineral nutrition. Symbionts are then hosted in privileged habitats, root cortex (for AM fungi) or nodules (for rhizobia), for efficient nutrient exchanges. To reach them, the plant creates trans-vacuolar cytoplasmic bridges, key for predicting and directing AM fungi hyphae or rhizobia-filled infection threads (ITs) entry. Yet, mechanisms underlying this pre-infection cellular remodelling are poorly studied. Here we show that unique ultrastructural changes and Ca2+ spiking signals closely linked to MtAnn1 annexin cellular dynamics, shape rhizobia-primed cells. Loss of MtAnn1 function in M. truncatula affects peak amplitude, cytoplasm configuration and rhizobia infection, consistent with MtAnn1s role in regulating this priming state. MtAnn1, mainly recruited during evolution in plants establishing endosymbioses, also appears involved in the ancient AM symbiosis in M. truncatula. Together, our work suggests MtAnn1 as part of an ancient Ca2+-regulatory module for transcellular endosymbiotic root infection.\nMitochondrial respiration is essential for photosynthesis-dependent ATP supply of the plant cytosol\nAuthors: Vera-Vives, A. M.; Novel, P.; Zheng, K.; Tan, S.-l.; Schwarzlander, M.; Alboresi, A.; Morosinotto, T.\nScore: 14.0, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574809\nPlants rely on solar energy to synthesize ATP and NADPH for photosynthetic carbon fixation. Since a substantial proportion of cellular ATP is consumed in the cytosol, photosynthesis-derived ATP needs to be supplied there. While the triose phosphate shuttle and mitochondrial respiration can both deliver ATP to the cytosol, the significance of the different mechanisms in vivo has been difficult to assess. Although mitochondrial respiration is essential in plants, whether this is due to heterotrophic bottlenecks during plant development or rather a need for respiration in photosynthetically active cells, has not been resolved. In this study, we examined in vivo changes of cytosolic ATP concentration in response to light, employing a biosensing strategy in the moss Physcomitrium patens. Our measurements revealed increased cytosolic ATP concentration caused by photosynthetic activity. Moss tissue depleted of respiratory complex I showed decreased cytosolic ATP accumulation, highlighting a critical role of mitochondrial respiration in light-dependent ATP supply of the cytosol. Consistently, targeting mitochondrial ATP production directly, through the construction of mutants deficient in mitochondrial ATPase (complex V), led to drastic growth reduction, despite only minor alterations in photosynthetic electron transport activity. Since P. patens is photoautotrophic throughout its development, we conclude that heterotrophic bottlenecks cannot account for the indispensable role of mitochondrial respiration in plants. Instead, our results offer compelling evidence that mitochondrial respiration is essential for ATP provision to the cytosol in actively photosynthesizing cells. Mitochondrial respiration provides metabolic integration, ensuring a reliable supply of cytosolic ATP essential for supporting plant growth and development.\nCEP signaling coordinates plant immunity with nitrogen status\nAuthors: Rzemieniewski, J.; Leicher, H.; Lee, H. K.; Broyart, C.; Nayem, S.; Wiese, C.; Maroschek, J.; Camgoez, Z.; Olsson Lalun, V.; Djordjevic, M. A.; Vlot, A. C.; Hueckelhoven, R.; Santiago, J.; Stegmann, M.\nScore: 29.6, Published: 2024-01-12 DOI: 10.1101/2022.12.20.521212\nPlant endogenous signaling peptides shape growth, development and adaptations to biotic and abiotic stress. Here, we identified C-TERMINALLY ENCODED PEPTIDEs (CEPs) as novel immune-modulatory peptides (phytocytokines) in Arabidopsis thaliana. Our data reveals that CEPs induce immune outputs and are required to mount resistance against the leaf-infecting bacterial pathogen Pseudomonas syringae pv. tomato. We show that effective immunity requires CEP perception by tissue-specific CEP RECEPTOR 1 (CEPR1) and CEPR2. Moreover, we identified the related RECEPTOR-LIKE KINASE 7 (RLK7) as a novel CEP4-specific CEP receptor contributing to CEP-mediated immunity, suggesting a complex interplay of multiple CEP ligands and receptors in different tissues during biotic stress. CEPs have a known role in the regulation of root growth and systemic nitrogen (N)-demand signaling. We now provide evidence that CEPs and their receptors promote immunity in an N status-dependent manner, suggesting a previously unknown molecular crosstalk between plant nutrition and cell surface immunity. We propose that CEPs and their receptors are central regulators for the adaptation of biotic stress responses to plant-available resources.\nGenomes of Aegilops umbellulata provide new insights into unique structural variations and genetic diversity in the U-genome for wheat improvement\nAuthors: Singh, J.; Gudi, S.; Maughan, P. J.; Liu, Z.; Kolmer, J.; Wang, M.; Chen, X.; Rouse, M.; Lasserre-Zuber, P.; Rimbert, H.; Sehgal, S.; Fiedler, J.; Choulet, F.; Acevado, M.; Gupta, R.; Gill, U.\nScore: 8.1, Published: 2024-01-14 DOI: 10.1101/2024.01.13.575480\nAegilops spp. serve as an important reservoir for novel sources of resistance or tolerance to biotic and abiotic stresses. To harness this reservoir, we have generated a high-quality chromosome-level genome assembly of an Ae. umbellulata accession PI 554389 using a combination of PacBio HiFi, Oxford nanopore, and chromosome conformation capture (Hi-C) sequencing technologies and resequenced 20 Ae. umbellulata genomes using Illumina sequencing. We assembled a 4.20 Gb genome spanned over seven chromosomes, rich in repetitive elements (~84%), achieving a QV of 59.54 with 98.14% completeness. The phylogenetic analysis places the U-genome with D-lineage, but major and distinct rearrangements were revealed in the U-genome. Unique transposon landscape of diploid U-genome and complex chromosomal rearrangements, most prominently in 4U and 6U chromosomes uncovered a distinct evolutionary trajectory of Ae. umbellulata. Additionally, the resequencing of geographically and morphologically diverse Ae. umbellulata accessions revealed three distinctive evolutionary sub-populations. Resequencing also identified six new haplotypes for Lr9, the first leaf rust resistance gene introgressed and cloned from Ae. umbellulata. These genomics resources along with high levels of resistance in the resequenced accessions against five devastating wheat diseases affirmed the genetic potential of Ae. umbellulata for wheat improvement.\nNovel Mechanisms of Strigolactone-Induced DWARF14 Degradation in Arabidopsis thaliana\nAuthors: Sanchez Martin-Fontecha, E.; Cardinale, F.; Bürger, M.; Prandi, C.; Cubas, P.\nScore: 7.0, Published: 2024-01-12 DOI: 10.1101/2024.01.10.574527\nStrigolactones (SLs) are carotenoid-derived compounds that regulate various aspects of plant development, physiological responses and plant interactions with microorganisms. In angiosperms, the SL receptor is the /{beta} hydrolase D14 that, upon SL binding, undergoes conformational changes, triggers SL-dependent responses and hydrolyses SLs. Arabidopsis SL signalling involves the formation of a complex between SL-bound D14, the E3-ubiquitin ligase SCFMAX2 and the transcriptional corepressors SMXL6/7/8 (SMXLs), which become ubiquitinated and degraded by the proteasome. However, the sequence of events that result in SL signalling and the requirement of SL hydrolysis for this process remain unclear. In addition, SL destabilises the D14 receptor. The biological significance of SL-induced D14 degradation is unclear, although it has been proposed to create a negative feedback loop in SL signalling. The current model proposes that D14 degradation occurs after SMXLs ubiquitination and proteolysis, via the same E3-ubiquitin ligase that targets the repressors. In this work we quantitatively studied the degradation dynamics of Arabidopsis D14 in response to SLs in planta. For this, we conducted fluorescence and luminescence assays to monitor D14 stability dynamics upon SL treatments, in transgenic lines expressing D14 fused to GREEN FLUORESCENT PROTEIN (GFP) or LUCIFERASE (LUC), in wild-type and SL-signalling mutant backgrounds. Mutant D14 proteins predicted to be non-functional for SL signalling were also examined, and their capability to bind SLs in vitro was studied using Differential Scanning Fluorimetry (DSF). Finally, we used a non-hydrolysable SL to test the requirements of SL hydrolysis for D14 and SMXL7 degradation. Our research revealed that SL-induced D14 degradation may occur in the absence of SCFMAX2 and/or SMXLs by a proteasome-independent mechanism. Additionally, we observed conditions in which the efficiency of SL-induced degradation of D14 is not aligned with that of SMXL7 degradation. Finally, our results indicate that the hydrolysis of SLs is not a prerequisite to trigger either D14 or SMXL7 degradation. These findings suggest the existence of a regulatory mechanism governing D14 degradation more complex than anticipated, and provide novel insights into the dynamics of SL signalling in Arabidopsis.\nTwilight length alters growth, flowering time, and clock function in Arabidopsis via LHY/CCA1\nAuthors: Mehta, D.; Scandola, S.; Kennedy, C.; Lummer, C.; Rodriguez Gallo, M. C.; Grubb, L. E.; Tan, M.; Scarpella, E.; Uhrig, R. G.\nScore: 20.2, Published: 2024-01-09 DOI: 10.1101/2023.06.09.544111\nThe plant circadian clock governs growth and development by syncing biological processes to periodic environmental changes. Decades of research has shown how the clock enables plants to respond to two environmental variables that change at different latitudes and over different seasons: photoperiod and temperature. However, a third variable, twilight length, has so far gone unstudied. The duration of twilight changes across the planet, lengthening with latitude and changing across seasons, and yet most circadian experiments are performed in lab environments with no twilight. Here, using controlled growth setups, we show that twilight length impacts plant growth and changes flowering time via the LHY/CCA1 morning module of the Arabidopsis circadian clock. Using a series of progressively truncated no-twilight photoperiods, we also found that plants are more sensitive to twilight length compared to equivalent changes in solely photoperiod. Further, transcriptome and proteome analyses indicated that twilight length alters the regulation of proteins involved in reactive oxygen species metabolism, photosynthesis, and carbon metabolism. Genetic analyses also implicated a twilight sensing pathway from phytochromes D and E, the morning elements of the circadian clock and modulating flowering time through the GI-FT pathway. Overall, our findings call for more nuanced models of daylength perception in plants and posit that twilight length is an important determinant of plant growth, development, and circadian function.\nEvolution of cell-type-specific accessible chromatin regions and the cis-regulatory elements that drive lineage-specific innovation\nAuthors: Yan, H.; Mendieta, J. P.; Zhang, X.; Marand, A. P.; Liang, Y.; Luo, Z.; Roule, T.; Wagner, D.; Tu, X.; Wang, Y.; Zhong, S.; Wessler, S. R.; Schmitz, R. J.\nScore: 20.4, Published: 2024-01-09 DOI: 10.1101/2024.01.08.574753\nCis-regulatory elements (CREs) are critical in coordinating the regulation of gene expression. Understanding how CREs evolve represents a challenging endeavor. Here, we constructed a comprehensive single-cell atlas of CRE activity in Oryza sativa, integrating data from 104,029 nuclei representing 128 discrete cell states across nine distinct organs. We used a comparative genomic approach to analyze cell-type-specific CRE activity between Oryza sativa and four additional grass species (Zea mays, Sorghum bicolor, Panicum miliaceum, and Urochloa fusca). We revealed a complex interplay whereby accessible chromatin regions (ACRs) had different levels of conservation dependent on if these regions were broadly accessible, or cell-type specific. We found that on average epidermal ACRs were less conserved in regions of synteny compared to other cell types, potentially indicating that large amounts of regulatory evolution have taken place in the epidermal layers of these species. We found a significant overlap between cell-type-specific ACRs and conserved non-coding sequences. Finally, we identified a subset of ACRs that were overlapping the repressive histone modification H3k27me3, which were conserved across evolutionary time, implicating them as potentially critical silencer CREs. Collectively, these results highlight the dynamic evolution of cell-type-specific CRE activity using comparative genomics.\nPolarised subcellular activation of ROPs by specific ROPGEFs drives pollen germination in Arabidopsis thaliana\nAuthors: Bouatta, A. M.; Lepper, A.; Denninger, P.\nScore: 5.8, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575165\nDuring plant fertilisation, excess male gametes compete for a limited number of female gametes. The dormant male gametophyte, encapsulated in the pollen grain, consists of two sperm cells enclosed in a vegetative cell. After reaching the stigma of a compatible flower, quick and efficient germination of the vegetative cell to a tip-growing pollen tube is crucial to ensure fertilisation success. RHO OF PLANTS (ROP) signalling and their activating ROP GUANINE NUCLEOTIDE EXCHANGE FACTORS (ROPGEFs) are essential for initiating polar growth processes in multiple cell types. However, which ROPGEFs activate pollen germination is unknown. We investigated the role of ROPGEFs in initiating pollen germination and the required cell polarity establishment. Of the five pollen-expressed ROPGEFs, we found that GEF8, GEF9, and GEF12 are required for pollen germination and male fertilisation success, as gef8;gef9;gef12 triple mutants showed almost complete loss of pollen germination in vitro and had a reduced allele transmission rate. Live cell imaging and spatiotemporal analysis of subcellular protein distribution showed that GEF8 and GEF9, but not GEF12, displayed transient polar protein accumulations at the future site of pollen germination minutes before pollen germination, demonstrating specific roles for GEF8 and GEF9 during the initiation of pollen germination. Furthermore, this novel GEF accumulation appears in a biphasic temporal manner and can shift its location. We showed that the C-terminal domain of GEF8 and GEF9 confers this protein accumulation and demonstrated that GEFs locally activate ROPs and alter Ca2+ signalling, which is required for pollen tube germination. We demonstrated that GEFs do not act redundantly during pollen germination and described for the first time a polar domain with spatiotemporal flexibility, which is crucial for the de novo establishment of a polar growth domain within a cell and, thus, for pollen function and fertilisation success.\nImplementation of pupylation-based proximity labelling in plant biology reveals regulatory factors in cellulose biosynthesis\nAuthors: Zheng, S.; Khammy, O.; Noack, L.; De Meyer, A.; Khan, G.; De Winne, N.; Eeckhout, D.; van Damme, D.; Persson, S.\nScore: 5.3, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574788\nKnowledge about how and where proteins interact provides a pillar for cell biology. Biotin-related proximity labelling approaches are efficiently monitoring protein interactions but may have labelling-related drawbacks. Here, we introduce pupylation-based proximity labelling (PUP-IT) as a tool for protein interaction detection in plant biology. We show that PUP-IT readily confirmed and extended protein interactions for several known protein complexes, across different types of plant systems. To further demonstrate the power of PUP-IT, we used the system to identify protein interactions of the protein complex that underpin cellulose synthesis in plants. Apart from known complex components, we identified the ARF-GEF BEN1 (BFA-VISUALIZED ENDOCYTIC TRAFFICKING DEFECTIVE1). We show that BEN1 contributes to cellulose synthesis by regulating the trafficking of the cellulose synthesis protein complex between the trans-Golgi network and the plasma membrane. Our results, therefore, introduce PUP-IT as a new and powerful proximity labelling system to identify protein interactions in plant cells.\n",
  "wordCount" : "2535",
  "inLanguage": "en",
  "datePublished": "2024-01-17T10:40:22Z",
  "dateModified": "2024-01-17T10:40:22Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 17, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575225">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575225" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575225">
        <p class="paperTitle">An oomycete effector co-opts a host RabGAP protein to remodel pathogen interface and subvert defense-related secretion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575225" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575225" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yuen, E. L. H.; Tumtas, Y.; Chan, L. I.; Ibrahim, T.; Evangelisti, E.; Tulin, F.; Sklenar, J.; Menke, F.; Kamoun, S.; Bubeck, D.; Schornack, S.; Bozkurt, T. O.</p>
        <p class="info">Score: 22.5, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575225' target='https://doi.org/10.1101/2024.01.11.575225'> 10.1101/2024.01.11.575225</a></p>
        <p class="abstract">Pathogens have evolved sophisticated mechanisms to manipulate host cell membrane dynamics, a crucial adaptation to survive in hostile environments shaped by innate immune responses. Plant-derived membrane interfaces, engulfing invasive hyphal projections of fungal and oomycete pathogens, are prominent junctures dictating infection outcomes. Understanding how pathogens transform these host-pathogen interfaces to their advantage remains a key biological question. Here, we identified a conserved effector, secreted by plant pathogenic oomycetes, that co-opts a host Rab GTPase-activating protein (RabGAP), TBC1D15L, to remodel the host-pathogen interface. The effector, PiE354, hijacks TBC1D15L as a susceptibility factor to usurp its GAP activity on Rab8a - a key Rab GTPase crucial for defense-related secretion. By hijacking TBC1D15L, PiE354 purges Rab8a from the plasma membrane, diverting Rab8a-mediated immune trafficking away from the pathogen interface. This mechanism signifies an uncanny evolutionary adaptation of a pathogen effector in co-opting a host regulatory component to subvert defense-related secretion, thereby providing unprecedented mechanistic insights into the reprogramming of host membrane dynamics by pathogens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574854">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574854" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574854">
        <p class="paperTitle">Annexin and calcium-regulated priming of legume root cells for endosymbiotic infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574854" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574854" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guillory, A.; Fournier, J.; Kelner, A.; Hobecker, K.; Auriac, M.-C.; Frances, L.; Delers, A.; Le Ru, A.; Keller, J.; Delaux, P.-M.; Gutjahr, C.; Frei Dit Frey, N.; de Carvalho-Niebel, F.</p>
        <p class="info">Score: 14.2, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574854' target='https://doi.org/10.1101/2024.01.10.574854'> 10.1101/2024.01.10.574854</a></p>
        <p class="abstract">Legumes establish endosymbioses with arbuscular mycorrhizal (AM) fungi or rhizobia bacteria to improve mineral nutrition. Symbionts are then hosted in privileged habitats, root cortex (for AM fungi) or nodules (for rhizobia), for efficient nutrient exchanges. To reach them, the plant creates trans-vacuolar cytoplasmic bridges, key for predicting and directing AM fungi hyphae or rhizobia-filled infection threads (ITs) entry. Yet, mechanisms underlying this pre-infection cellular remodelling are poorly studied. Here we show that unique ultrastructural changes and Ca2&#43; spiking signals closely linked to MtAnn1 annexin cellular dynamics, shape rhizobia-primed cells. Loss of MtAnn1 function in M. truncatula affects peak amplitude, cytoplasm configuration and rhizobia infection, consistent with MtAnn1s role in regulating this priming state. MtAnn1, mainly recruited during evolution in plants establishing endosymbioses, also appears involved in the ancient AM symbiosis in M. truncatula. Together, our work suggests MtAnn1 as part of an ancient Ca2&#43;-regulatory module for transcellular endosymbiotic root infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574809">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574809" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574809">
        <p class="paperTitle">Mitochondrial respiration is essential for photosynthesis-dependent ATP supply of the plant cytosol</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574809" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574809" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vera-Vives, A. M.; Novel, P.; Zheng, K.; Tan, S.-l.; Schwarzlander, M.; Alboresi, A.; Morosinotto, T.</p>
        <p class="info">Score: 14.0, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574809' target='https://doi.org/10.1101/2024.01.09.574809'> 10.1101/2024.01.09.574809</a></p>
        <p class="abstract">Plants rely on solar energy to synthesize ATP and NADPH for photosynthetic carbon fixation. Since a substantial proportion of cellular ATP is consumed in the cytosol, photosynthesis-derived ATP needs to be supplied there. While the triose phosphate shuttle and mitochondrial respiration can both deliver ATP to the cytosol, the significance of the different mechanisms in vivo has been difficult to assess. Although mitochondrial respiration is essential in plants, whether this is due to heterotrophic bottlenecks during plant development or rather a need for respiration in photosynthetically active cells, has not been resolved. In this study, we examined in vivo changes of cytosolic ATP concentration in response to light, employing a biosensing strategy in the moss Physcomitrium patens. Our measurements revealed increased cytosolic ATP concentration caused by photosynthetic activity. Moss tissue depleted of respiratory complex I showed decreased cytosolic ATP accumulation, highlighting a critical role of mitochondrial respiration in light-dependent ATP supply of the cytosol. Consistently, targeting mitochondrial ATP production directly, through the construction of mutants deficient in mitochondrial ATPase (complex V), led to drastic growth reduction, despite only minor alterations in photosynthetic electron transport activity. Since P. patens is photoautotrophic throughout its development, we conclude that heterotrophic bottlenecks cannot account for the indispensable role of mitochondrial respiration in plants. Instead, our results offer compelling evidence that mitochondrial respiration is essential for ATP provision to the cytosol in actively photosynthesizing cells. Mitochondrial respiration provides metabolic integration, ensuring a reliable supply of cytosolic ATP essential for supporting plant growth and development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.12.20.521212">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.12.20.521212" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.12.20.521212">
        <p class="paperTitle">CEP signaling coordinates plant immunity with nitrogen status</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.12.20.521212" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.12.20.521212" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rzemieniewski, J.; Leicher, H.; Lee, H. K.; Broyart, C.; Nayem, S.; Wiese, C.; Maroschek, J.; Camgoez, Z.; Olsson Lalun, V.; Djordjevic, M. A.; Vlot, A. C.; Hueckelhoven, R.; Santiago, J.; Stegmann, M.</p>
        <p class="info">Score: 29.6, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.12.20.521212' target='https://doi.org/10.1101/2022.12.20.521212'> 10.1101/2022.12.20.521212</a></p>
        <p class="abstract">Plant endogenous signaling peptides shape growth, development and adaptations to biotic and abiotic stress. Here, we identified C-TERMINALLY ENCODED PEPTIDEs (CEPs) as novel immune-modulatory peptides (phytocytokines) in Arabidopsis thaliana. Our data reveals that CEPs induce immune outputs and are required to mount resistance against the leaf-infecting bacterial pathogen Pseudomonas syringae pv. tomato. We show that effective immunity requires CEP perception by tissue-specific CEP RECEPTOR 1 (CEPR1) and CEPR2. Moreover, we identified the related RECEPTOR-LIKE KINASE 7 (RLK7) as a novel CEP4-specific CEP receptor contributing to CEP-mediated immunity, suggesting a complex interplay of multiple CEP ligands and receptors in different tissues during biotic stress. CEPs have a known role in the regulation of root growth and systemic nitrogen (N)-demand signaling. We now provide evidence that CEPs and their receptors promote immunity in an N status-dependent manner, suggesting a previously unknown molecular crosstalk between plant nutrition and cell surface immunity. We propose that CEPs and their receptors are central regulators for the adaptation of biotic stress responses to plant-available resources.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.13.575480">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.13.575480" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.13.575480">
        <p class="paperTitle">Genomes of Aegilops umbellulata provide new insights into unique structural variations and genetic diversity in the U-genome for wheat improvement</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.13.575480" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.13.575480" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Singh, J.; Gudi, S.; Maughan, P. J.; Liu, Z.; Kolmer, J.; Wang, M.; Chen, X.; Rouse, M.; Lasserre-Zuber, P.; Rimbert, H.; Sehgal, S.; Fiedler, J.; Choulet, F.; Acevado, M.; Gupta, R.; Gill, U.</p>
        <p class="info">Score: 8.1, Published: 2024-01-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.13.575480' target='https://doi.org/10.1101/2024.01.13.575480'> 10.1101/2024.01.13.575480</a></p>
        <p class="abstract">Aegilops spp. serve as an important reservoir for novel sources of resistance or tolerance to biotic and abiotic stresses. To harness this reservoir, we have generated a high-quality chromosome-level genome assembly of an Ae. umbellulata accession PI 554389 using a combination of PacBio HiFi, Oxford nanopore, and chromosome conformation capture (Hi-C) sequencing technologies and resequenced 20 Ae. umbellulata genomes using Illumina sequencing. We assembled a 4.20 Gb genome spanned over seven chromosomes, rich in repetitive elements (~84%), achieving a QV of 59.54 with 98.14% completeness. The phylogenetic analysis places the U-genome with D-lineage, but major and distinct rearrangements were revealed in the U-genome. Unique transposon landscape of diploid U-genome and complex chromosomal rearrangements, most prominently in 4U and 6U chromosomes uncovered a distinct evolutionary trajectory of Ae. umbellulata. Additionally, the resequencing of geographically and morphologically diverse Ae. umbellulata accessions revealed three distinctive evolutionary sub-populations. Resequencing also identified six new haplotypes for Lr9, the first leaf rust resistance gene introgressed and cloned from Ae. umbellulata. These genomics resources along with high levels of resistance in the resequenced accessions against five devastating wheat diseases affirmed the genetic potential of Ae. umbellulata for wheat improvement.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574527">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574527" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574527">
        <p class="paperTitle">Novel Mechanisms of Strigolactone-Induced DWARF14 Degradation in Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574527" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574527" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanchez Martin-Fontecha, E.; Cardinale, F.; Bürger, M.; Prandi, C.; Cubas, P.</p>
        <p class="info">Score: 7.0, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574527' target='https://doi.org/10.1101/2024.01.10.574527'> 10.1101/2024.01.10.574527</a></p>
        <p class="abstract">Strigolactones (SLs) are carotenoid-derived compounds that regulate various aspects of plant development, physiological responses and plant interactions with microorganisms. In angiosperms, the SL receptor is the /{beta} hydrolase D14 that, upon SL binding, undergoes conformational changes, triggers SL-dependent responses and hydrolyses SLs. Arabidopsis SL signalling involves the formation of a complex between SL-bound D14, the E3-ubiquitin ligase SCFMAX2 and the transcriptional corepressors SMXL6/7/8 (SMXLs), which become ubiquitinated and degraded by the proteasome. However, the sequence of events that result in SL signalling and the requirement of SL hydrolysis for this process remain unclear. In addition, SL destabilises the D14 receptor. The biological significance of SL-induced D14 degradation is unclear, although it has been proposed to create a negative feedback loop in SL signalling. The current model proposes that D14 degradation occurs after SMXLs ubiquitination and proteolysis, via the same E3-ubiquitin ligase that targets the repressors.

In this work we quantitatively studied the degradation dynamics of Arabidopsis D14 in response to SLs in planta. For this, we conducted fluorescence and luminescence assays to monitor D14 stability dynamics upon SL treatments, in transgenic lines expressing D14 fused to GREEN FLUORESCENT PROTEIN (GFP) or LUCIFERASE (LUC), in wild-type and SL-signalling mutant backgrounds. Mutant D14 proteins predicted to be non-functional for SL signalling were also examined, and their capability to bind SLs in vitro was studied using Differential Scanning Fluorimetry (DSF). Finally, we used a non-hydrolysable SL to test the requirements of SL hydrolysis for D14 and SMXL7 degradation. Our research revealed that SL-induced D14 degradation may occur in the absence of SCFMAX2 and/or SMXLs by a proteasome-independent mechanism. Additionally, we observed conditions in which the efficiency of SL-induced degradation of D14 is not aligned with that of SMXL7 degradation. Finally, our results indicate that the hydrolysis of SLs is not a prerequisite to trigger either D14 or SMXL7 degradation. These findings suggest the existence of a regulatory mechanism governing D14 degradation more complex than anticipated, and provide novel insights into the dynamics of SL signalling in Arabidopsis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.09.544111">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.09.544111" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.09.544111">
        <p class="paperTitle">Twilight length alters growth, flowering time, and clock function in Arabidopsis via LHY/CCA1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.09.544111" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.09.544111" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mehta, D.; Scandola, S.; Kennedy, C.; Lummer, C.; Rodriguez Gallo, M. C.; Grubb, L. E.; Tan, M.; Scarpella, E.; Uhrig, R. G.</p>
        <p class="info">Score: 20.2, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.09.544111' target='https://doi.org/10.1101/2023.06.09.544111'> 10.1101/2023.06.09.544111</a></p>
        <p class="abstract">The plant circadian clock governs growth and development by syncing biological processes to periodic environmental changes. Decades of research has shown how the clock enables plants to respond to two environmental variables that change at different latitudes and over different seasons: photoperiod and temperature. However, a third variable, twilight length, has so far gone unstudied. The duration of twilight changes across the planet, lengthening with latitude and changing across seasons, and yet most circadian experiments are performed in lab environments with no twilight. Here, using controlled growth setups, we show that twilight length impacts plant growth and changes flowering time via the LHY/CCA1 morning module of the Arabidopsis circadian clock. Using a series of progressively truncated no-twilight photoperiods, we also found that plants are more sensitive to twilight length compared to equivalent changes in solely photoperiod. Further, transcriptome and proteome analyses indicated that twilight length alters the regulation of proteins involved in reactive oxygen species metabolism, photosynthesis, and carbon metabolism. Genetic analyses also implicated a twilight sensing pathway from phytochromes D and E, the morning elements of the circadian clock and modulating flowering time through the GI-FT pathway. Overall, our findings call for more nuanced models of daylength perception in plants and posit that twilight length is an important determinant of plant growth, development, and circadian function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574753">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574753" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574753">
        <p class="paperTitle">Evolution of cell-type-specific accessible chromatin regions and the cis-regulatory elements that drive lineage-specific innovation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574753" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574753" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yan, H.; Mendieta, J. P.; Zhang, X.; Marand, A. P.; Liang, Y.; Luo, Z.; Roule, T.; Wagner, D.; Tu, X.; Wang, Y.; Zhong, S.; Wessler, S. R.; Schmitz, R. J.</p>
        <p class="info">Score: 20.4, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574753' target='https://doi.org/10.1101/2024.01.08.574753'> 10.1101/2024.01.08.574753</a></p>
        <p class="abstract">Cis-regulatory elements (CREs) are critical in coordinating the regulation of gene expression. Understanding how CREs evolve represents a challenging endeavor. Here, we constructed a comprehensive single-cell atlas of CRE activity in Oryza sativa, integrating data from 104,029 nuclei representing 128 discrete cell states across nine distinct organs. We used a comparative genomic approach to analyze cell-type-specific CRE activity between Oryza sativa and four additional grass species (Zea mays, Sorghum bicolor, Panicum miliaceum, and Urochloa fusca). We revealed a complex interplay whereby accessible chromatin regions (ACRs) had different levels of conservation dependent on if these regions were broadly accessible, or cell-type specific. We found that on average epidermal ACRs were less conserved in regions of synteny compared to other cell types, potentially indicating that large amounts of regulatory evolution have taken place in the epidermal layers of these species. We found a significant overlap between cell-type-specific ACRs and conserved non-coding sequences. Finally, we identified a subset of ACRs that were overlapping the repressive histone modification H3k27me3, which were conserved across evolutionary time, implicating them as potentially critical silencer CREs. Collectively, these results highlight the dynamic evolution of cell-type-specific CRE activity using comparative genomics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575165">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575165" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575165">
        <p class="paperTitle">Polarised subcellular activation of ROPs by specific ROPGEFs drives pollen germination in Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575165" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575165" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bouatta, A. M.; Lepper, A.; Denninger, P.</p>
        <p class="info">Score: 5.8, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575165' target='https://doi.org/10.1101/2024.01.11.575165'> 10.1101/2024.01.11.575165</a></p>
        <p class="abstract">During plant fertilisation, excess male gametes compete for a limited number of female gametes. The dormant male gametophyte, encapsulated in the pollen grain, consists of two sperm cells enclosed in a vegetative cell. After reaching the stigma of a compatible flower, quick and efficient germination of the vegetative cell to a tip-growing pollen tube is crucial to ensure fertilisation success. RHO OF PLANTS (ROP) signalling and their activating ROP GUANINE NUCLEOTIDE EXCHANGE FACTORS (ROPGEFs) are essential for initiating polar growth processes in multiple cell types. However, which ROPGEFs activate pollen germination is unknown. We investigated the role of ROPGEFs in initiating pollen germination and the required cell polarity establishment. Of the five pollen-expressed ROPGEFs, we found that GEF8, GEF9, and GEF12 are required for pollen germination and male fertilisation success, as gef8;gef9;gef12 triple mutants showed almost complete loss of pollen germination in vitro and had a reduced allele transmission rate. Live cell imaging and spatiotemporal analysis of subcellular protein distribution showed that GEF8 and GEF9, but not GEF12, displayed transient polar protein accumulations at the future site of pollen germination minutes before pollen germination, demonstrating specific roles for GEF8 and GEF9 during the initiation of pollen germination. Furthermore, this novel GEF accumulation appears in a biphasic temporal manner and can shift its location. We showed that the C-terminal domain of GEF8 and GEF9 confers this protein accumulation and demonstrated that GEFs locally activate ROPs and alter Ca2&#43; signalling, which is required for pollen tube germination. We demonstrated that GEFs do not act redundantly during pollen germination and described for the first time a polar domain with spatiotemporal flexibility, which is crucial for the de novo establishment of a polar growth domain within a cell and, thus, for pollen function and fertilisation success.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574788">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574788" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574788">
        <p class="paperTitle">Implementation of pupylation-based proximity labelling in plant biology reveals regulatory factors in cellulose biosynthesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574788" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574788" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zheng, S.; Khammy, O.; Noack, L.; De Meyer, A.; Khan, G.; De Winne, N.; Eeckhout, D.; van Damme, D.; Persson, S.</p>
        <p class="info">Score: 5.3, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574788' target='https://doi.org/10.1101/2024.01.09.574788'> 10.1101/2024.01.09.574788</a></p>
        <p class="abstract">Knowledge about how and where proteins interact provides a pillar for cell biology. Biotin-related proximity labelling approaches are efficiently monitoring protein interactions but may have labelling-related drawbacks. Here, we introduce pupylation-based proximity labelling (PUP-IT) as a tool for protein interaction detection in plant biology. We show that PUP-IT readily confirmed and extended protein interactions for several known protein complexes, across different types of plant systems. To further demonstrate the power of PUP-IT, we used the system to identify protein interactions of the protein complex that underpin cellulose synthesis in plants. Apart from known complex components, we identified the ARF-GEF BEN1 (BFA-VISUALIZED ENDOCYTIC TRAFFICKING DEFECTIVE1). We show that BEN1 contributes to cellulose synthesis by regulating the trafficking of the cellulose synthesis protein complex between the trans-Golgi network and the plasma membrane. Our results, therefore, introduce PUP-IT as a new and powerful proximity labelling system to identify protein interactions in plant cells.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
