<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The WY domain of an RxLR effector drives interactions with a host target phosphatase to mimic host regulatory proteins and promote Phytophthora infestans infection.
Authors: Bentham, A. R.; Wang, W.; Trusch, F.; Varden, F.; Birch, P. R.; Banfield, M. J.
Score: 24.7, Published: 2023-07-25 DOI: 10.1101/2023.07.24.550319
Plant pathogens manipulate the cellular environment of the host to facilitate infection and colonization, often leading to plant diseases. To accomplish this, many specialized pathogens secrete virulence proteins called effectors into the host cell, which subvert processes such as immune signalling, gene transcription, and host metabolism.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="The WY domain of an RxLR effector drives interactions with a host target phosphatase to mimic host regulatory proteins and promote Phytophthora infestans infection.
Authors: Bentham, A. R.; Wang, W.; Trusch, F.; Varden, F.; Birch, P. R.; Banfield, M. J.
Score: 24.7, Published: 2023-07-25 DOI: 10.1101/2023.07.24.550319
Plant pathogens manipulate the cellular environment of the host to facilitate infection and colonization, often leading to plant diseases. To accomplish this, many specialized pathogens secrete virulence proteins called effectors into the host cell, which subvert processes such as immune signalling, gene transcription, and host metabolism." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-02T10:38:09+00:00" />
<meta property="article:modified_time" content="2023-08-02T10:38:09+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="The WY domain of an RxLR effector drives interactions with a host target phosphatase to mimic host regulatory proteins and promote Phytophthora infestans infection.
Authors: Bentham, A. R.; Wang, W.; Trusch, F.; Varden, F.; Birch, P. R.; Banfield, M. J.
Score: 24.7, Published: 2023-07-25 DOI: 10.1101/2023.07.24.550319
Plant pathogens manipulate the cellular environment of the host to facilitate infection and colonization, often leading to plant diseases. To accomplish this, many specialized pathogens secrete virulence proteins called effectors into the host cell, which subvert processes such as immune signalling, gene transcription, and host metabolism."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "The WY domain of an RxLR effector drives interactions with a host target phosphatase to mimic host regulatory proteins and promote Phytophthora infestans infection.\nAuthors: Bentham, A. R.; Wang, W.; Trusch, F.; Varden, F.; Birch, P. R.; Banfield, M. J.\nScore: 24.7, Published: 2023-07-25 DOI: 10.1101/2023.07.24.550319\nPlant pathogens manipulate the cellular environment of the host to facilitate infection and colonization, often leading to plant diseases. To accomplish this, many specialized pathogens secrete virulence proteins called effectors into the host cell, which subvert processes such as immune signalling, gene transcription, and host metabolism.",
  "keywords": [
    
  ],
  "articleBody": " The WY domain of an RxLR effector drives interactions with a host target phosphatase to mimic host regulatory proteins and promote Phytophthora infestans infection.\nAuthors: Bentham, A. R.; Wang, W.; Trusch, F.; Varden, F.; Birch, P. R.; Banfield, M. J.\nScore: 24.7, Published: 2023-07-25 DOI: 10.1101/2023.07.24.550319\nPlant pathogens manipulate the cellular environment of the host to facilitate infection and colonization, often leading to plant diseases. To accomplish this, many specialized pathogens secrete virulence proteins called effectors into the host cell, which subvert processes such as immune signalling, gene transcription, and host metabolism. Phytophthora infestans, the causative agent of potato late blight, employs an expanded repertoire of RxLR effectors with WY domains to manipulate the host through direct interaction with protein targets. However, our understanding of the molecular mechanisms underlying the interactions between WY effectors and their host targets remains limited. In this study, we performed a structural and biophysical characterization of the P. infestans WY effector, Pi04314, in complex with the potato Protein Phosphatase 1-c (PP1c). We elucidate how Pi04314 uses a WY domain and a specialised C-terminal loop carrying a KVxF motif that interact with conserved surfaces on PP1c, known to be used by host regulatory proteins for guiding function. Through biophysical and in planta analyses, we demonstrate that Pi04314 WY or KVxF mutants lose their ability to bind PP1c. The loss of PP1c binding correlates with a reduced capacity to re-localize PP1c from the nucleolus and a decrease in lesion size in plant infection assays. This study provides insights into the manipulation of plant hosts by pathogens, revealing how effectors exploit key regulatory interfaces in host proteins to modify their function and facilitate disease.\nH2O2 sulfenylates CHE linking local infection to establishment of systemic acquired resistance\nAuthors: Cao, L.; Yoo, H.; Chen, T.; Mwimba, M.; Zhang, X.; Dong, X.\nScore: 20.6, Published: 2023-08-01 DOI: 10.1101/2023.07.27.550865\nIn plants, a local infection can lead to systemic acquired resistance (SAR) through increased production of salicylic acid (SA). For 30 years, the identity of the mobile signal and its direct transduction mechanism for systemic SA synthesis in initiating SAR have been hotly debated. We found that, upon pathogen challenge, the cysteine residue of transcription factor CHE undergoes sulfenylation in systemic tissues, enhancing its binding to the promoter of SA-synthesis gene, ICS1, and increasing SA production. This occurs independently of previously reported pipecolic acid (Pip) signal. Instead, H2O2 produced by NADPH oxidase, RBOHD, is the mobile signal that sulfenylates CHE in a concentration-dependent manner. This modification serves as a molecular switch that activates CHE-mediated SA-increase and subsequent Pip-accumulation in systemic tissues to synergistically induce SAR. One Sentence SummaryRBOHD-generated H2O2 sulfenylates transcription factor CHE to establish systemic acquired resistance in plants.\nBacterial pathogens deliver water/solute-permeable channels as a virulence strategy\nAuthors: Nomura, K.; Andreazza, F.; Cheng, J.; Dong, K.; Zhou, P.; He, S. Y.\nScore: 15.7, Published: 2023-07-29 DOI: 10.1101/2023.07.29.547699\nMany animal and plant pathogenic bacteria utilize a type III secretion system to deliver effector proteins into the host cell1,2. Elucidation of how these effector proteins function in the host cell is critical for understanding infectious diseases in animals and plants3-5. The widely conserved AvrE/DspE-family effectors play a central role in the pathogenesis of diverse phytopathogenic bacteria6. These conserved effectors are involved in the induction of \"water-soaking\" and host cell death that are conducive to bacterial multiplication in infected tissues. However, the exact biochemical functions of AvrE/DspE-family effectors have been recalcitrant to mechanistic understanding for three decades. Here we show that AvrE/DspE-family effectors fold into a {beta}-barrel structure that resembles bacterial porins. Expression of AvrE and DspE in Xenopus oocytes results in (i) inward and outward currents, (ii) permeability to water and (iii) osmolarity-dependent oocyte swelling and bursting. Liposome reconstitution confirmed that the DspE channel alone is sufficient to allow the passage of small molecules such as fluorescein dye. Targeted screening of chemical blockers based on the predicted pore size (15-20 [A]) of the DspE channel identified polyamidoamine (PAMAM) dendrimers as inhibitors of the DspE/AvrE channels. Remarkably, PAMAMs broadly inhibit AvrE/DspE virulence activities in Xenopus oocytes and during Erwinia amylovora and Pseudomonas syringae infections. Thus, we have unraveled the enigmatic function of a centrally important family of bacterial effectors with significant conceptual and practical implications in the study of bacterial pathogenesis.\nMetabolic arms race between a plant and a fungal pathogen\nAuthors: Liu, Y.; Mahdi, L.; Porzel, A.; Stark, P.; Esposto, D.; Scherr-Henning, A.; Bathe, U.; Acosta, I. F.; Zuccaro, A.; Balcke, G. U.; Tissier, A. F.\nScore: 15.3, Published: 2023-07-29 DOI: 10.1101/2023.07.29.551000\nIn this work, we uncover a metabolite interaction between barley and the fungal pathogen Bipolaris sorokiniana (Bs), involving hordedanes, a previously undescribed set of labdane-related diterpenoids with antimicrobial properties. Bs infection of barley roots elicits hordedane synthesis from a 600-kb gene cluster. Heterologous reconstruction of the synthesis pathway in yeast produced several hordedanes, including one of the most advanced products 19-b-hydroxy-hordetrienoic acid (19-OH-HTA). Barley mutants in the diterpene synthase genes of the cluster are unable to produce hordedanes but, unexpectedly, show reduced Bs colonization. Accordingly, 19-OH-HTA enhances both germination and growth of Bs, while it inhibits other fungi, and Bs chemically modifies 19-OH-HTA. Thus, plant and pathogen molecular interactions extend beyond protein-protein recognition and the simple detoxification of plant antimicrobial metabolites. One-Sentence SummaryA fungal pathogen uses barley diterpenoid phytoalexins to facilitate root colonization.\nA Proxitome-RNA-capture Approach Reveals that Processing Bodies Repress Co-Regulated Hubs\nAuthors: Liu, C.; Mentzelopoulou, A.; Hatzianestis, I.; Tzagkarakis, E.; SKALTSOGIANNIS, V.; Ma, X.; Michalopoulou, V. A.; Romero-Campero, F. J.; Romero-Losada, A. B.; Sarris, P. F.; Marhavy, P.; Bölter, B.; Kanterakis, A.; Gutierrez-Beltran, E.; Moschou, P. N.\nScore: 12.2, Published: 2023-07-28 DOI: 10.1101/2023.07.26.550742\nCellular condensates are usually ribonucleoprotein assemblies with liquid- or solid-like properties. Because they lack a delineating membrane, the compositional determination of condensates is laborious. Here we set up a pipeline for proximity-biotinylation-dependent capture of RNA to investigate the RNA composition of the condensate in Arabidopsis known as the processing bodies (PBs). Using this pipeline together with in situ protein-protein interaction and RNA detection, in silico, and high-resolution imaging approaches, we studied PBs under normal and heat stress conditions. The composition of PBs in RNAs is much more dynamic than that of the total transcriptome. RNAs involved in cell wall development and regeneration, hormonal signaling, secondary metabolism/defense, and RNA metabolism were enriched in PBs. RNA binding proteins and liquid-to-solid phase transitions modulated specificity of RNA recruitment in PBs. Surprisingly, RNAs were sometimes recruited together with their encoded proteins. In PBs RNAs follow distinct fates, with small liquid-like PBs modulating RNA decay while larger ones storage. The size and properties of PBs are regulated by the actin polymerization cAMP receptor (SCAR)-WASP family verprolin homologous (WAVE) complex. SCAR/WAVE modulates signaling by shuttling RNAs between PBs and the translational machinery adjusting the ethylene signaling pathway. Heat stress leads to the storage of immunity-related RNAs in PBs by reducing PBs dynamics, suggesting why processes such as immunity malfunction under heat stress. In summary, we provide a method to identify RNAs in condensates which allowed us to reveal a mechanism for RNA fate regulation.\nIdentification of cambium stem cell factors and their positioning mechanism\nAuthors: Eswaran, G.; Rutten, J. P.; Han, J.; Iida, H.; Lopez Ortiz, J.; Makila, R.; Wybouw, B.; Vainio, L.; Porcher, A.; Leal Gavarron, M.; Zhang, J.; Blomster, T.; Wang, X.; Dolan, D. W. P.; Smetana, O.; Brady, S.; Kucukoglu Topcu, M.; ten Tusscher, K. H.; Etchells, P.; Mahonen, A. P.\nScore: 36.0, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549889\nWood constitutes the majority of terrestrial biomass. Composed of xylem, it arises from one side of the vascular cambium, a bifacial stem cell niche that also produces phloem on the opposing side. It is currently unknown which molecular factors endow cambium stem cell identity. Here we show that TDIF ligand-activated PXY receptors promote the expression of CAMBIUM AINTEGUMENTA-LIKE (CAIL) transcription factors to define cambium stem cell identity in the Arabidopsis root. By sequestrating the phloem-originated TDIF, xylem-dependent PXY confines the TDIF signaling front, resulting in the activation of CAIL expression and stem cell identity in only a narrow domain. Our findings show how signals emanating from cells on opposing sides ensure robust yet dynamically adjustable positioning of a bifacial stem cell.\nDataPLAN: a web-based data management plan generator for the plant sciences\nAuthors: Zhou, X.-R.; Beier, S.; Brilhaus, D.; Rodrigues, C. M.; Mühlhaus, T.; von Suchodoletz, D.; Twyman, R. M.; Usadel, B.; Kranz, A.\nScore: 24.6, Published: 2023-07-10 DOI: 10.1101/2023.07.07.548147\nResearch Data Management (RDM) is a system for the effective handling of research data that enables scientists to structure their research questions and ensure best practices throughout the data lifecycle, from acquisition, computation and annotation to data publication and re-use. Data management plans (DMPs) are documents that formally set out the RDM of a project and are required by many funding bodies. DMPs help to organize and structure RDM strategies, thus promoting data findability, accessibility, interoperability and reusability (FAIR). Although DMPs incorporate methods and standards that can be reused by different research projects, the standardization of DMP content is not as evident as the standardization of RDM practices and data/metadata. To address this issue in the plant sciences, we developed DataPLAN - a tool that combines a questionnaire with pre-written standardized responses. We wrapped the questionnaire in a serverless single-page web application that can then generate standardized responses from DMP templates. The current templates cater to plant research grant proposals for Horizon 2020, Horizon Europe and the German Research Foundation (Deutsche Forschungsgemeinschaft, DFG). In the future the range of templates will be extended to accommodate other funding schemes, thereby enabling more users to generate their own templates. The DataPLAN web application is open-source and does not require an internet connection. By utilizing DataPLAN, the workload associated with creating, updating, and adhering to DMPs is significantly reduced.\nPloidy-specific transcriptomes shed light on the heterogeneous identity and metabolism of developing pericarp cells\nAuthors: Tourdot, E.; Maza, E.; Djari, A.; Martin, P. G.; Gevaudant, F.; Chevalier, C.; Pirrello, J.; Gonzalez, N.\nScore: 7.7, Published: 2023-07-30 DOI: 10.1101/2023.07.28.550816\nEndoreduplication, during which cells increase their DNA content through successive rounds of full genome replication without cell division, is the major source of endopolyploidy in higher plants. Endoreduplication plays pivotal roles in plant growth and development and is associated with the activation of specific transcriptional programs that are characteristic to each cell type, thereby defining their identity. In plants, endoreduplication is found in numerous organs and cell types and especially in agronomically valuable ones, such as the fleshy fruit (pericarp) of tomato presenting high ploidy levels. We used the tomato pericarp tissue as a model system to explore the transcriptomes associated with endoreduplication progression during fruit growth. We confirmed that expression globally scales with ploidy level and identified sets of genes differentially expressed when comparing ploidy levels at a specific developmental stage. We found that non-endoreduplicated cells are defined by cell division state and cuticle synthesis while endoreduplicated cells are mainly defined by their metabolic activity changing rapidly over time. By combining this dataset with publicly available spatiotemporal pericarp expression data, we proposed a map describing the distribution of ploidy levels within the pericarp. These transcriptome-based predictions were validated by quantifying ploidy levels within the pericarp tissue. This in situ ploidy quantification revealed the dynamic progression of endoreduplication and its cell layer specificity during early fruit development. In summary, the study sheds light on the complex relationship between endoreduplication, cell differentiation, and gene expression patterns in the tomato pericarp. Significance statementThe progression of endoreduplication is very dynamic during early fruit development and displays cell layer specific patterns. The integration of ploidy distribution maps with ploidy-specific transcriptome data revealed that gene expression in the pericarp is controlled in a ploidy-specific manner during the early stages of tomato fruit development, resulting in the spatialization of transcriptional programs.\nChitosan stimulates root hair callose deposition and inhibits root hair growth\nAuthors: Drs, M.; Haluska, S.; Skrabalkova, E.; Krupar, P.; Brejskova, L.; Muller, K.; Serrano, N.; Potocka, A.; Voxeur, A.; Vernhettes, S.; Ortmannova, J.; Caldarescu, G.; Fendrych, M.; Potocky, M.; Zarsky, V.; Pecenkova, T.\nScore: 7.4, Published: 2023-07-31 DOI: 10.1101/2023.07.30.551171\nAlthough angiosperm plants have a general capacity to react after the immunity elicitor chitin or chitosan treatment by the cell wall callose deposition, this response in particular cell types and its evolutionary conservation is not understood. Here we show that also the growing root hairs (RHs) of Arabidopsis can respond to a mild (0.001%) chitosan treatment by the callose deposition and by a deceleration of the RH growth. We demonstrate that the glucan synthase-like 5 (GSL5)/PMR4 is vital for chitosan-induced callose deposition but not for RH growth inhibition. Upon the higher chitosan concentration (0.01%) treatment, RHs do not deposit callose, while growth inhibition is prominent. To understand the specificities of the low and high concentration chitosan treatments, we analysed the PTI signalling components, gene expression, and RH cellular endomembrane and cytoskeleton modifications upon these treatments. Importantly, chitosan-induced callose deposition is also present in the functionally analogous and evolutionarily only distantly related RH-like structures rhizophores (lycopod) and rhizoids (bryophytes). Our results point to the RH callose deposition as a conserved strategy of soil-anchoring plant cells (rhizoids/rhizophores/RHs) to deal with mild biotic stress. At the same time, high chitosan concentration prominently disturbs intracellular dynamics, tip-localised endomembrane compartments and RH growth, precluding callose deposition.\nThe transcriptome landscape of developing barley seeds reveals H3K27me3 dynamics in endosperm tissues\nAuthors: Kovacik, M.; Nowicka, A.; Zwyrtkov, J.; Strejckova, B.; Vardanega, I.; Esteban, E. N.; Pasha, A.; Kaduchova, K.; Krautsova, M.; Safar, J.; Provart, N. J.; Simon, R.; Pecinka, A.\nScore: 6.6, Published: 2023-07-27 DOI: 10.1101/2023.07.26.550659\nCereal grains are an important food and feed. To provide comprehensive spatiotemporal information about biological processes in developing seeds of cultivated barley, we performed a transcriptomic study of the embryo, endosperm, and seed maternal tissues collected from 4 to 32 days after pollination. Weighted gene co-expression network and motif enrichment analyses pointed out specific groups of genes and transcription factors with possible impacts on regulating barley seed tissue development. We defined a set of tissue-specific marker genes and families of transcription factors for functional studies of the pathways controlling barley grain development. Assessment of selected groups of chromatin regulators revealed that epigenetic processes are highly dynamic and likely to play a major role during barley endosperm development. Repressive modification H3K27me3 is globally reduced in endosperm tissues and at specific developmental and storage compound genes. Altogether, this atlas uncovers the complexity of the developmental regulation of gene expression in barley grains. TeaserSpatiotemporal profiling of developing barley seeds revealed loss of H3K27me3 and changes in gene expression in endosperm.\n",
  "wordCount" : "2423",
  "inLanguage": "en",
  "datePublished": "2023-08-02T10:38:09Z",
  "dateModified": "2023-08-02T10:38:09Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta"><span>updated on August 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550319">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550319" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550319">
        <p class="paperTitle">The WY domain of an RxLR effector drives interactions with a host target phosphatase to mimic host regulatory proteins and promote Phytophthora infestans infection.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550319" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550319" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bentham, A. R.; Wang, W.; Trusch, F.; Varden, F.; Birch, P. R.; Banfield, M. J.</p>
        <p class="info">Score: 24.7, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550319' target='https://doi.org/10.1101/2023.07.24.550319'> 10.1101/2023.07.24.550319</a></p>
        <p class="abstract">Plant pathogens manipulate the cellular environment of the host to facilitate infection and colonization, often leading to plant diseases. To accomplish this, many specialized pathogens secrete virulence proteins called effectors into the host cell, which subvert processes such as immune signalling, gene transcription, and host metabolism. Phytophthora infestans, the causative agent of potato late blight, employs an expanded repertoire of RxLR effectors with WY domains to manipulate the host through direct interaction with protein targets. However, our understanding of the molecular mechanisms underlying the interactions between WY effectors and their host targets remains limited. In this study, we performed a structural and biophysical characterization of the P. infestans WY effector, Pi04314, in complex with the potato Protein Phosphatase 1-c (PP1c). We elucidate how Pi04314 uses a WY domain and a specialised C-terminal loop carrying a KVxF motif that interact with conserved surfaces on PP1c, known to be used by host regulatory proteins for guiding function. Through biophysical and in planta analyses, we demonstrate that Pi04314 WY or KVxF mutants lose their ability to bind PP1c. The loss of PP1c binding correlates with a reduced capacity to re-localize PP1c from the nucleolus and a decrease in lesion size in plant infection assays. This study provides insights into the manipulation of plant hosts by pathogens, revealing how effectors exploit key regulatory interfaces in host proteins to modify their function and facilitate disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550865">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550865" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550865">
        <p class="paperTitle">H2O2 sulfenylates CHE linking local infection to establishment of systemic acquired resistance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550865" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550865" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cao, L.; Yoo, H.; Chen, T.; Mwimba, M.; Zhang, X.; Dong, X.</p>
        <p class="info">Score: 20.6, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550865' target='https://doi.org/10.1101/2023.07.27.550865'> 10.1101/2023.07.27.550865</a></p>
        <p class="abstract">In plants, a local infection can lead to systemic acquired resistance (SAR) through increased production of salicylic acid (SA). For 30 years, the identity of the mobile signal and its direct transduction mechanism for systemic SA synthesis in initiating SAR have been hotly debated. We found that, upon pathogen challenge, the cysteine residue of transcription factor CHE undergoes sulfenylation in systemic tissues, enhancing its binding to the promoter of SA-synthesis gene, ICS1, and increasing SA production. This occurs independently of previously reported pipecolic acid (Pip) signal. Instead, H2O2 produced by NADPH oxidase, RBOHD, is the mobile signal that sulfenylates CHE in a concentration-dependent manner. This modification serves as a molecular switch that activates CHE-mediated SA-increase and subsequent Pip-accumulation in systemic tissues to synergistically induce SAR.

One Sentence SummaryRBOHD-generated H2O2 sulfenylates transcription factor CHE to establish systemic acquired resistance in plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.29.547699">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.29.547699" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.29.547699">
        <p class="paperTitle">Bacterial pathogens deliver water/solute-permeable channels as a virulence strategy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.29.547699" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.29.547699" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nomura, K.; Andreazza, F.; Cheng, J.; Dong, K.; Zhou, P.; He, S. Y.</p>
        <p class="info">Score: 15.7, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.29.547699' target='https://doi.org/10.1101/2023.07.29.547699'> 10.1101/2023.07.29.547699</a></p>
        <p class="abstract">Many animal and plant pathogenic bacteria utilize a type III secretion system to deliver effector proteins into the host cell1,2. Elucidation of how these effector proteins function in the host cell is critical for understanding infectious diseases in animals and plants3-5. The widely conserved AvrE/DspE-family effectors play a central role in the pathogenesis of diverse phytopathogenic bacteria6. These conserved effectors are involved in the induction of &#34;water-soaking&#34; and host cell death that are conducive to bacterial multiplication in infected tissues. However, the exact biochemical functions of AvrE/DspE-family effectors have been recalcitrant to mechanistic understanding for three decades. Here we show that AvrE/DspE-family effectors fold into a {beta}-barrel structure that resembles bacterial porins. Expression of AvrE and DspE in Xenopus oocytes results in (i) inward and outward currents, (ii) permeability to water and (iii) osmolarity-dependent oocyte swelling and bursting. Liposome reconstitution confirmed that the DspE channel alone is sufficient to allow the passage of small molecules such as fluorescein dye. Targeted screening of chemical blockers based on the predicted pore size (15-20 [A]) of the DspE channel identified polyamidoamine (PAMAM) dendrimers as inhibitors of the DspE/AvrE channels. Remarkably, PAMAMs broadly inhibit AvrE/DspE virulence activities in Xenopus oocytes and during Erwinia amylovora and Pseudomonas syringae infections. Thus, we have unraveled the enigmatic function of a centrally important family of bacterial effectors with significant conceptual and practical implications in the study of bacterial pathogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.29.551000">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.29.551000" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.29.551000">
        <p class="paperTitle">Metabolic arms race between a plant and a fungal pathogen</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.29.551000" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.29.551000" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, Y.; Mahdi, L.; Porzel, A.; Stark, P.; Esposto, D.; Scherr-Henning, A.; Bathe, U.; Acosta, I. F.; Zuccaro, A.; Balcke, G. U.; Tissier, A. F.</p>
        <p class="info">Score: 15.3, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.29.551000' target='https://doi.org/10.1101/2023.07.29.551000'> 10.1101/2023.07.29.551000</a></p>
        <p class="abstract">In this work, we uncover a metabolite interaction between barley and the fungal pathogen Bipolaris sorokiniana (Bs), involving hordedanes, a previously undescribed set of labdane-related diterpenoids with antimicrobial properties. Bs infection of barley roots elicits hordedane synthesis from a 600-kb gene cluster. Heterologous reconstruction of the synthesis pathway in yeast produced several hordedanes, including one of the most advanced products 19-b-hydroxy-hordetrienoic acid (19-OH-HTA). Barley mutants in the diterpene synthase genes of the cluster are unable to produce hordedanes but, unexpectedly, show reduced Bs colonization. Accordingly, 19-OH-HTA enhances both germination and growth of Bs, while it inhibits other fungi, and Bs chemically modifies 19-OH-HTA. Thus, plant and pathogen molecular interactions extend beyond protein-protein recognition and the simple detoxification of plant antimicrobial metabolites.

One-Sentence SummaryA fungal pathogen uses barley diterpenoid phytoalexins to facilitate root colonization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550742">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550742" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550742">
        <p class="paperTitle">A Proxitome-RNA-capture Approach Reveals that Processing Bodies Repress Co-Regulated Hubs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550742" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550742" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, C.; Mentzelopoulou, A.; Hatzianestis, I.; Tzagkarakis, E.; SKALTSOGIANNIS, V.; Ma, X.; Michalopoulou, V. A.; Romero-Campero, F. J.; Romero-Losada, A. B.; Sarris, P. F.; Marhavy, P.; Bölter, B.; Kanterakis, A.; Gutierrez-Beltran, E.; Moschou, P. N.</p>
        <p class="info">Score: 12.2, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550742' target='https://doi.org/10.1101/2023.07.26.550742'> 10.1101/2023.07.26.550742</a></p>
        <p class="abstract">Cellular condensates are usually ribonucleoprotein assemblies with liquid- or solid-like properties. Because they lack a delineating membrane, the compositional determination of condensates is laborious. Here we set up a pipeline for proximity-biotinylation-dependent capture of RNA to investigate the RNA composition of the condensate in Arabidopsis known as the processing bodies (PBs). Using this pipeline together with in situ protein-protein interaction and RNA detection, in silico, and high-resolution imaging approaches, we studied PBs under normal and heat stress conditions. The composition of PBs in RNAs is much more dynamic than that of the total transcriptome. RNAs involved in cell wall development and regeneration, hormonal signaling, secondary metabolism/defense, and RNA metabolism were enriched in PBs. RNA binding proteins and liquid-to-solid phase transitions modulated specificity of RNA recruitment in PBs. Surprisingly, RNAs were sometimes recruited together with their encoded proteins. In PBs RNAs follow distinct fates, with small liquid-like PBs modulating RNA decay while larger ones storage. The size and properties of PBs are regulated by the actin polymerization cAMP receptor (SCAR)-WASP family verprolin homologous (WAVE) complex. SCAR/WAVE modulates signaling by shuttling RNAs between PBs and the translational machinery adjusting the ethylene signaling pathway. Heat stress leads to the storage of immunity-related RNAs in PBs by reducing PBs dynamics, suggesting why processes such as immunity malfunction under heat stress. In summary, we provide a method to identify RNAs in condensates which allowed us to reveal a mechanism for RNA fate regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549889">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549889" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549889">
        <p class="paperTitle">Identification of cambium stem cell factors and their positioning mechanism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549889" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549889" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eswaran, G.; Rutten, J. P.; Han, J.; Iida, H.; Lopez Ortiz, J.; Makila, R.; Wybouw, B.; Vainio, L.; Porcher, A.; Leal Gavarron, M.; Zhang, J.; Blomster, T.; Wang, X.; Dolan, D. W. P.; Smetana, O.; Brady, S.; Kucukoglu Topcu, M.; ten Tusscher, K. H.; Etchells, P.; Mahonen, A. P.</p>
        <p class="info">Score: 36.0, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549889' target='https://doi.org/10.1101/2023.07.20.549889'> 10.1101/2023.07.20.549889</a></p>
        <p class="abstract">Wood constitutes the majority of terrestrial biomass. Composed of xylem, it arises from one side of the vascular cambium, a bifacial stem cell niche that also produces phloem on the opposing side. It is currently unknown which molecular factors endow cambium stem cell identity. Here we show that TDIF ligand-activated PXY receptors promote the expression of CAMBIUM AINTEGUMENTA-LIKE (CAIL) transcription factors to define cambium stem cell identity in the Arabidopsis root. By sequestrating the phloem-originated TDIF, xylem-dependent PXY confines the TDIF signaling front, resulting in the activation of CAIL expression and stem cell identity in only a narrow domain. Our findings show how signals emanating from cells on opposing sides ensure robust yet dynamically adjustable positioning of a bifacial stem cell.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548147">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548147" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548147">
        <p class="paperTitle">DataPLAN: a web-based data management plan generator for the plant sciences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548147" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548147" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, X.-R.; Beier, S.; Brilhaus, D.; Rodrigues, C. M.; Mühlhaus, T.; von Suchodoletz, D.; Twyman, R. M.; Usadel, B.; Kranz, A.</p>
        <p class="info">Score: 24.6, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548147' target='https://doi.org/10.1101/2023.07.07.548147'> 10.1101/2023.07.07.548147</a></p>
        <p class="abstract">Research Data Management (RDM) is a system for the effective handling of research data that enables scientists to structure their research questions and ensure best practices throughout the data lifecycle, from acquisition, computation and annotation to data publication and re-use. Data management plans (DMPs) are documents that formally set out the RDM of a project and are required by many funding bodies. DMPs help to organize and structure RDM strategies, thus promoting data findability, accessibility, interoperability and reusability (FAIR). Although DMPs incorporate methods and standards that can be reused by different research projects, the standardization of DMP content is not as evident as the standardization of RDM practices and data/metadata. To address this issue in the plant sciences, we developed DataPLAN - a tool that combines a questionnaire with pre-written standardized responses. We wrapped the questionnaire in a serverless single-page web application that can then generate standardized responses from DMP templates. The current templates cater to plant research grant proposals for Horizon 2020, Horizon Europe and the German Research Foundation (Deutsche Forschungsgemeinschaft, DFG). In the future the range of templates will be extended to accommodate other funding schemes, thereby enabling more users to generate their own templates. The DataPLAN web application is open-source and does not require an internet connection. By utilizing DataPLAN, the workload associated with creating, updating, and adhering to DMPs is significantly reduced.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550816">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550816" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550816">
        <p class="paperTitle">Ploidy-specific transcriptomes shed light on the heterogeneous identity and metabolism of developing pericarp cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550816" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550816" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tourdot, E.; Maza, E.; Djari, A.; Martin, P. G.; Gevaudant, F.; Chevalier, C.; Pirrello, J.; Gonzalez, N.</p>
        <p class="info">Score: 7.7, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550816' target='https://doi.org/10.1101/2023.07.28.550816'> 10.1101/2023.07.28.550816</a></p>
        <p class="abstract">Endoreduplication, during which cells increase their DNA content through successive rounds of full genome replication without cell division, is the major source of endopolyploidy in higher plants. Endoreduplication plays pivotal roles in plant growth and development and is associated with the activation of specific transcriptional programs that are characteristic to each cell type, thereby defining their identity. In plants, endoreduplication is found in numerous organs and cell types and especially in agronomically valuable ones, such as the fleshy fruit (pericarp) of tomato presenting high ploidy levels. We used the tomato pericarp tissue as a model system to explore the transcriptomes associated with endoreduplication progression during fruit growth. We confirmed that expression globally scales with ploidy level and identified sets of genes differentially expressed when comparing ploidy levels at a specific developmental stage. We found that non-endoreduplicated cells are defined by cell division state and cuticle synthesis while endoreduplicated cells are mainly defined by their metabolic activity changing rapidly over time. By combining this dataset with publicly available spatiotemporal pericarp expression data, we proposed a map describing the distribution of ploidy levels within the pericarp. These transcriptome-based predictions were validated by quantifying ploidy levels within the pericarp tissue. This in situ ploidy quantification revealed the dynamic progression of endoreduplication and its cell layer specificity during early fruit development. In summary, the study sheds light on the complex relationship between endoreduplication, cell differentiation, and gene expression patterns in the tomato pericarp.

Significance statementThe progression of endoreduplication is very dynamic during early fruit development and displays cell layer specific patterns. The integration of ploidy distribution maps with ploidy-specific transcriptome data revealed that gene expression in the pericarp is controlled in a ploidy-specific manner during the early stages of tomato fruit development, resulting in the spatialization of transcriptional programs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.30.551171">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.30.551171" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.30.551171">
        <p class="paperTitle">Chitosan stimulates root hair callose deposition and inhibits root hair growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.30.551171" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.30.551171" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Drs, M.; Haluska, S.; Skrabalkova, E.; Krupar, P.; Brejskova, L.; Muller, K.; Serrano, N.; Potocka, A.; Voxeur, A.; Vernhettes, S.; Ortmannova, J.; Caldarescu, G.; Fendrych, M.; Potocky, M.; Zarsky, V.; Pecenkova, T.</p>
        <p class="info">Score: 7.4, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.30.551171' target='https://doi.org/10.1101/2023.07.30.551171'> 10.1101/2023.07.30.551171</a></p>
        <p class="abstract">Although angiosperm plants have a general capacity to react after the immunity elicitor chitin or chitosan treatment by the cell wall callose deposition, this response in particular cell types and its evolutionary conservation is not understood. Here we show that also the growing root hairs (RHs) of Arabidopsis can respond to a mild (0.001%) chitosan treatment by the callose deposition and by a deceleration of the RH growth. We demonstrate that the glucan synthase-like 5 (GSL5)/PMR4 is vital for chitosan-induced callose deposition but not for RH growth inhibition. Upon the higher chitosan concentration (0.01%) treatment, RHs do not deposit callose, while growth inhibition is prominent. To understand the specificities of the low and high concentration chitosan treatments, we analysed the PTI signalling components, gene expression, and RH cellular endomembrane and cytoskeleton modifications upon these treatments. Importantly, chitosan-induced callose deposition is also present in the functionally analogous and evolutionarily only distantly related RH-like structures rhizophores (lycopod) and rhizoids (bryophytes). Our results point to the RH callose deposition as a conserved strategy of soil-anchoring plant cells (rhizoids/rhizophores/RHs) to deal with mild biotic stress. At the same time, high chitosan concentration prominently disturbs intracellular dynamics, tip-localised endomembrane compartments and RH growth, precluding callose deposition.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550659">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550659" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550659">
        <p class="paperTitle">The transcriptome landscape of developing barley seeds reveals H3K27me3 dynamics in endosperm tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550659" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550659" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kovacik, M.; Nowicka, A.; Zwyrtkov, J.; Strejckova, B.; Vardanega, I.; Esteban, E. N.; Pasha, A.; Kaduchova, K.; Krautsova, M.; Safar, J.; Provart, N. J.; Simon, R.; Pecinka, A.</p>
        <p class="info">Score: 6.6, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550659' target='https://doi.org/10.1101/2023.07.26.550659'> 10.1101/2023.07.26.550659</a></p>
        <p class="abstract">Cereal grains are an important food and feed. To provide comprehensive spatiotemporal information about biological processes in developing seeds of cultivated barley, we performed a transcriptomic study of the embryo, endosperm, and seed maternal tissues collected from 4 to 32 days after pollination. Weighted gene co-expression network and motif enrichment analyses pointed out specific groups of genes and transcription factors with possible impacts on regulating barley seed tissue development. We defined a set of tissue-specific marker genes and families of transcription factors for functional studies of the pathways controlling barley grain development. Assessment of selected groups of chromatin regulators revealed that epigenetic processes are highly dynamic and likely to play a major role during barley endosperm development. Repressive modification H3K27me3 is globally reduced in endosperm tissues and at specific developmental and storage compound genes. Altogether, this atlas uncovers the complexity of the developmental regulation of gene expression in barley grains.

TeaserSpatiotemporal profiling of developing barley seeds revealed loss of H3K27me3 and changes in gene expression in endosperm.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
