<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Retrotransposon-driven environmental regulation of FLC leads to adaptive response to herbicide
Authors: Raingeval, M.; Leduque, B.; Baduel, P.; Edera, A.; Roux, F.; Colot, V.; Quadrana, L.
Score: 32.2, Published: 2023-09-07 DOI: 10.1101/2023.09.06.556499
The mobilization of retrotransposons yields major-effect mutations. Here, we report an adaptive retrotransposon insertion within the first intron of the Arabidopsis floral-repressor locus FLOWERING LOCUS C (FLC). The insertion-mutation augments the environmental sensitivity of FLC by affecting the balance between coding and non-coding transcript isoforms in response to environmental threads.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="Retrotransposon-driven environmental regulation of FLC leads to adaptive response to herbicide
Authors: Raingeval, M.; Leduque, B.; Baduel, P.; Edera, A.; Roux, F.; Colot, V.; Quadrana, L.
Score: 32.2, Published: 2023-09-07 DOI: 10.1101/2023.09.06.556499
The mobilization of retrotransposons yields major-effect mutations. Here, we report an adaptive retrotransposon insertion within the first intron of the Arabidopsis floral-repressor locus FLOWERING LOCUS C (FLC). The insertion-mutation augments the environmental sensitivity of FLC by affecting the balance between coding and non-coding transcript isoforms in response to environmental threads." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-13T10:39:03+00:00" />
<meta property="article:modified_time" content="2023-09-13T10:39:03+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="Retrotransposon-driven environmental regulation of FLC leads to adaptive response to herbicide
Authors: Raingeval, M.; Leduque, B.; Baduel, P.; Edera, A.; Roux, F.; Colot, V.; Quadrana, L.
Score: 32.2, Published: 2023-09-07 DOI: 10.1101/2023.09.06.556499
The mobilization of retrotransposons yields major-effect mutations. Here, we report an adaptive retrotransposon insertion within the first intron of the Arabidopsis floral-repressor locus FLOWERING LOCUS C (FLC). The insertion-mutation augments the environmental sensitivity of FLC by affecting the balance between coding and non-coding transcript isoforms in response to environmental threads."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "Retrotransposon-driven environmental regulation of FLC leads to adaptive response to herbicide\nAuthors: Raingeval, M.; Leduque, B.; Baduel, P.; Edera, A.; Roux, F.; Colot, V.; Quadrana, L.\nScore: 32.2, Published: 2023-09-07 DOI: 10.1101/2023.09.06.556499\nThe mobilization of retrotransposons yields major-effect mutations. Here, we report an adaptive retrotransposon insertion within the first intron of the Arabidopsis floral-repressor locus FLOWERING LOCUS C (FLC). The insertion-mutation augments the environmental sensitivity of FLC by affecting the balance between coding and non-coding transcript isoforms in response to environmental threads.",
  "keywords": [
    
  ],
  "articleBody": " Retrotransposon-driven environmental regulation of FLC leads to adaptive response to herbicide\nAuthors: Raingeval, M.; Leduque, B.; Baduel, P.; Edera, A.; Roux, F.; Colot, V.; Quadrana, L.\nScore: 32.2, Published: 2023-09-07 DOI: 10.1101/2023.09.06.556499\nThe mobilization of retrotransposons yields major-effect mutations. Here, we report an adaptive retrotransposon insertion within the first intron of the Arabidopsis floral-repressor locus FLOWERING LOCUS C (FLC). The insertion-mutation augments the environmental sensitivity of FLC by affecting the balance between coding and non-coding transcript isoforms in response to environmental threads. We show that this balance is modulated epigenetically by DNA methylation and orchestrated by IBM2, a factor involved in the processing of intronic heterochromatin. The stress-sensitive allele of FLC has recently spread across populations subjected to recurrent chemical weeding, and we demonstrate that retrotransposon-driven acceleration of life cycle represents a rapid response to herbicide. Our findings illustrate how retrotransposition can create environmentally-sensitive alleles that facilitate adaptation to anthropogenic disturbances of the environment.\nAn atypical endomembrane localized CNL-type immune receptor with a conserved deletion in the N-terminal signaling domain functions in cell death and immunity\nAuthors: Sunil, S.; Beeh, S.; Stoebbe, E.; Fischer, K. M.; Wilhelm, F.; Paris, C.; Meral, A.; Teasdale, L.; Jiang, Z.; Zhang, l.; Aguilar Parras, E.; Nuernberger, T.; Weigel, D.; Lozano-Duran, R.; El Kasmi, F.\nScore: 20.6, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556214\nPlants have evolved intracellular nucleotide-binding leucine rich repeat receptors (NLRs) to induce a superior immune response. Upon activation, coiled-coil (CC) domain containing NLRs (CNLs) oligomerize to form apparent cation channels that promote calcium influx and cell death induction, with the alpha-1 helix of the individual CC domains penetrating membranes. Some members of a monophyletic subclass of CNLs, the ancient and autonomous NLRs (ANLs), are characterized by putative N- myristoylation and S-acylation sites at the N-terminus of their CCG10/GA domain, potentially mediating permanent membrane association. Whether these Potentially Membrane Localized NLRs (PMLs) mediate cell death upon activation in a similar way as reported for other CNLs has been unknown. We integrated phylogenetic, cell- biological, and functional studies to uncover the cell death function of an atypical but conserved Arabidopsis PML, PML5, which has a 113 amino acid deletion in its CCG10/GA domain. Active PML5 oligomers localize in Golgi membranes and the tonoplast, changes vacuolar morphology, and induce cell death, with the short N- terminus being sufficient for cell death. Mutant analysis supports a potential key role of PMLs in plant immunity. Similar deletions as in Arabidopsis PML5 are found in several Brassicales paralogs, pointing to the evolutionary importance of this innovation. PML5 is thus a naturally occurring CNL variant with a minimal signaling domain and its further study should help in understanding the functional importance of this minimal domain for NLR signaling.\nA soybean rust effector protease suppresses host immunity and cleaves a 3-deoxy-7-phosphoheptulonate synthase\nAuthors: Chicowski, A. S.; Qi, M.; Variz, H.; Bredow, M.; Montes-Serey, C.; Caiazza, F.; Dong, H.; Margets, A. C.; Mejias, J.; Walley, J.; Craik, C.; Pedley, K. F.; Aung, K.; Innes, R. W.; Whitham, S.\nScore: 15.8, Published: 2023-09-08 DOI: 10.1101/2023.09.07.556260\nThe devastating soybean rust (SBR) pathogen, Phakopsora pachyrhizi, encodes many secreted proteins, but only two have been functionally characterized for their roles in rust virulence. Here, we demonstrate that transient expression of P. pachyrhizi effector candidate 15 (PpEC15), an aspartic protease, leads to enhanced bacterial growth in planta, suppression of callose deposition, reduced expression of plant defense-related marker genes and suppression of pathogen-associated molecular pattern (PAMP)-induced reactive oxygen species (ROS). Stable expression of PpEC15 in soybean suppresses PAMP-induced ROS production and enhances bacterial growth, indicating that, collectively, PpEC15 suppresses host and non-host innate immune responses. Yeast-two-hybrid and proximity labeling identified putative PpEC15 interacting partners including a peptide-chain release factor (PCRF), a NAC83 (NAM, ATAF, and CUC) transcription factor, and a DAHP (3-deoxy-7-phosphoheptulonate) synthase. We further show that PpEC15 is able to cleave DAHP, but does not cleave PCRF or NAC83. Virus-induced gene silencing of NAC83, PCRF and DAHP altered PAMP-induced ROS production and salicylic acid production, indicating that these proteins may be involved in immune signaling. Collectively, our data show that PpEC15 is conserved across P. pachyrhizi isolates and other economically important rust species and is involved in the suppression of plant basal defense responses. Understanding the role of PpEC15 in P. pachyrhizi virulence will provide a foundation for designing targeted intervention strategies to generate rust-resistant crops.\nStacking of PRRs in potato to achieve enhanced resistance against Phytophthora infestans\nAuthors: Ascurra, Y. C. T.; Wouters, D.; Visser, R.; Nuernberger, T.; Vleeshouwers, V.\nScore: 15.2, Published: 2023-09-11 DOI: 10.1101/2023.09.07.556738\nPlants employ pattern recognition receptors (PRRs) to sense pathogen-associated molecular patterns (PAMPs) or apoplastic effectors at the plant cell surface, as well as nucleotide-binding domain leucine-rich-repeat-containing receptors (NLRs) to sense effectors inside the plant cell. Breeding for potato resistance to P. infestans has focused on the use of NLRs, however, these genes are typically quickly overcome since the matching avirulence genes evolve exceptionally quickly. Here, we stacked two PRRs, PERU and RLP23, that recognize the rather conserved Phytophthora PAMPs Pep-13/25 and nlp20, respectively, in the potato cultivar Atlantic, and evaluated their effect on P. infestans resistance. We found that PERU and RLP23 cooperate for the early immune responses like the accumulation of reactive oxygen species (ROS) and production of ethylene by recognizing their corresponding PAMPs. Furthermore, we show that potato plants overexpressing these two PRRs are slightly less affected by P. infestans compared to the single transformants. Together, our data suggest that pyramiding of surface receptors can provide additional enhanced resistance against pathogens, however, more effective or synergistic combinations that may include intracellular NLR receptors should be explored.\nEstablishing a comprehensive web-based analysis platform for Nicotiana benthamiana genome and transcriptome\nAuthors: Kurotani, K.-i.; Hirakawa, H.; Shirasawa, K.; Tagiri, K.; Mori, M.; Ichihashi, Y.; Suzuki, T.; Tanizawa, Y.; Nakamura, Y.; Isobe, S.; Notaguchi, M.\nScore: 15.2, Published: 2023-09-05 DOI: 10.1101/2023.09.03.556139\nNicotiana benthamiana has long served as a crucial plant material extensively used in plant physiology research, particularly in the field of plant pathology, because of its high susceptibility to plant viruses. Additionally, it serves as a production platform to test vaccines and other valuable substances. Among its approximately 3.1 Gb genome, 57,583 genes have been annotated within a 61 Mb region. We created a comprehensive and easy-to-use platform to use transcriptomes for modern annotation. These tools allow to visualize gene expression profiles, draw molecular evolutionary phylogenetic trees of gene families, perform functional enrichment analyses, and facilitate output downloads. To demonstrate their utility, we analyzed the gene expression profiles of enzymes within the nicotine biosynthesis pathway, a secondary metabolic pathway characteristic of the Nicotiana genus. Using the developed tool, expression profiles of the nicotine biosynthesis pathway genes were generated. The expression patterns of eight gene groups in the pathway were strongly expressed in the roots and weakly expressed in leaves and flowers of N. benthamiana. The results were consistent with the established gene expression profiles in Nicotiana tabacum and provided insights into gene family composition and expression trends. The compilation of this database tool can facilitate genetic analysis of N. benthamiana in the future. Significance statementA tool was developed to visualize gene expression profiles, draw molecular evolutionary phylogenetic trees of gene families, perform functional enrichment analyses, and facilitate output downloads of Nicotiana benthamiana. The database developed using the tool can evolve into a comprehensive all-in-one analysis platform by continuously incorporating transcriptome data released to date, newly released RNA-seq data, and annotations in the future.\nWater fluxes contribute to growth patterning in shoot meristems\nAuthors: Alonso-Serra, J.; Cheddadi, I.; Kiss, A.; Cerutti, G.; Lionnet, C.; Godin, C.; Hamant, O.\nScore: 35.1, Published: 2023-08-29 DOI: 10.1101/2023.08.27.554993\nIn multicellular organisms, localized tissue outgrowth creates a new water sink thereby modifying hydraulic patterns at the organ level. These fluxes are often considered passive by-products of development and their patterning and potential contribution to morphogenesis remains largely unexplored. Here, we generated a complete map of cell volumetric growth and deformation across the shoot apex in Arabidopsis thaliana. Within the organ-meristem boundary, we found that a subpopulation of cells next to fast-growing cells experiences volumetric shrinkage. To understand this process, we used a vertex-based model integrating mechanics and hydraulics, informed by the measured growth rates. Organ outgrowth simulations revealed the emerging water fluxes and predicted water deficit with volume loss for a few cells at the boundary. Consistently, in planta, a water-soluble dye is preferentially allocated to fast-growing tissues and fails to enter the boundary domain. Analysis of intact meristems further validated our model by revealing cell shrinkage next to fast-growing cells in different contexts of tissue surface curvature and cell deformation. A molecular signature of water deficit at the boundary further confirmed our conclusion. Taken together, we propose a model where the differential sink strength of emerging organs prescribes the hydraulic patterns that define the boundary domain at the shoot apex.\nYerba mate (Ilex paraguariensis) genome provides new insights into convergent evolution of caffeine biosynthesis\nAuthors: Vignale, F. A.; Hernandez Garcia, A.; Modenutti, C. P.; Sosa, E. J.; Defelipe, L. A.; Oliveira, R. R. M.; Nunes, G. L.; Acevedo, R. M.; Burguener, G. F.; Rossi, M.; Zapata, P. D.; Marti, D. A.; Oliveira, G.; Smith, M. N.; Dubs, N. M.; Nair, S.; Barkman, T. J.; Turjanski, A. G.\nScore: 8.8, Published: 2023-09-10 DOI: 10.1101/2023.09.08.556846\nYerba mate (Ilex paraguariensis) is an economically important crop marketed for the elaboration of mate, the third-most widely consumed caffeine-containing infusion worldwide. Here we report the first genome assembly of this species, which has a total length of 1.06 Gb and contains 53,390 protein-coding genes. Comparative analyses revealed that the large yerba mate genome size is partly due to a whole-genome duplication (Ip-) during the early evolutionary history of Ilex, in addition to the hexaploidization event ({gamma}) shared by core eudicots. Characterization of the genome allowed us to clone the genes encoding methyltransferase enzymes that catalyse multiple reactions required for caffeine production. To our surprise, this species has converged upon a different biochemical pathway compared to that of its relatives, coffee and tea. In order to gain insight into the structural basis for the convergent enzyme activities, we obtained a crystal structure for the terminal enzyme in the pathway that forms caffeine. The structure reveals that convergent solutions have evolved for substrate positioning because different amino acid residues facilitate a different substrate orientation such that efficient methylation occurs in the independently evolved enzymes in yerba mate and coffee. While our results show phylogenomic constraint limits the genes coopted for convergence of caffeine biosynthesis, the x-ray diffraction data suggests structural constraints are minimal for the convergent evolution of individual reactions.\nControl of cell fate specification and patterning by an ancestral microRNA\nAuthors: Aguilar-Cruz, A.; Flores-Sandoval, E.; Gutierrez-Ramos, X.; Oltehua-Lopez, O.; Dorantes-Acosta, A. E.; Trujillo, J. T.; Kato, H.; Ishizaki, K.; Mosher, R. A.; Dolan, L.; Grimanelli, D.; Haseloff, J.; Bowman, J. L.; Arteaga-Vazquez, M. A.\nScore: 8.8, Published: 2023-09-11 DOI: 10.1101/2023.09.09.556951\nThe formation of an organized body requires the establishment and maintenance of cells with structural and functional distinctive characteristics. A central question in developmental biology is how changes in the regulation of genes drive cell specification and patterning1. microRNAs (miRNAs) are small non-coding RNAs that regulate development through mRNA cleavage and/or translational repression2. In plants, miRNAs regulate key aspects including growth, development, stem cell maintenance, vegetative phase change, leaf morphogenesis, floral organ formation and flowering time3. Biogenesis of plant miRNAs depends on the activity of DICER-LIKE 1 (DCL1), an RNase type III endonuclease that processes double stranded RNA to give rise to mature miRNAs 4 . The genomes of today's flora contain at least one bona fide copy of DCL1 5,6. Using Marchantia polymorpha -a model bryophyte that allows comparative approaches to infer characteristics of the ancestral land plant-, we demonstrate that MpDCL1a is required for the biogenesis of miRNAs and uncovered a central role for miR166/Homeodomain Zipper Class III-regulated auxin synthesis in the specification of cell identity, patterning, meristem function, laminar expansion and the development of the body in the last common ancestor of extant land plants.\nHistorical causes for the greater proportion of polyploid plants in higher latitudes\nAuthors: Hagen, E. R.; Vasconcelos, T.; Boyko, J. D.; Beaulieu, J. M.\nScore: 7.7, Published: 2023-09-05 DOI: 10.1101/2023.09.01.555981\nPremise of the StudyThe proportion of polyploid plants in a community increases with latitude, and different hypotheses have been proposed about which factors drive this pattern. Here, we aim to understand the historical causes of the latitudinal polyploidy gradient using a combination of ancestral state reconstruction methods. Specifically, we assess whether (1) polyploidization enables movement to higher latitudes (i.e., polyploidization precedes occurrences in higher latitudes) or (2) higher latitudes facilitate polyploidization (i.e., occurrence in higher latitudes precedes polyploidization). MethodsWe reconstruct the ploidy states and ancestral niches of 1,032 angiosperm species at four paleoclimatic time slices ranging from 3.3 million years ago to the present, comprising taxa from four well-represented clades: Onagraceae, Primulaceae, Solanum (Solanaceae), and Pooideae (Poaceae). We use ancestral niche reconstruction models alongside a customized discrete character evolution model to allow reconstruction of states at specific time slices. Patterns of latitudinal movement are reconstructed and compared in relation to inferred ploidy shifts. Key ResultsWe find that no single hypothesis applies equally well across all analyzed clades. While significant differences in median latitudinal occurrence were detected in the largest clade, Pooideae, no significant differences were detected in latitudinal movement in any clade. ConclusionsOur preliminary study is the first to attempt to connect ploidy changes to continuous latitudinal movement, but we cannot favor one hypothesis over another. Given that patterns seem to be clade-specific, a larger number of clades must be analyzed in future studies for generalities to be drawn.\nSingle-plant-omics reveals the cascade of transcriptional changes during the vegetative-to-reproductive transition\nAuthors: Redmond, E. J.; Ronald, J.; Davis, S. J.; Ezer, D.\nScore: 6.1, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557157\nPlants undergo rapid developmental transitions, as well as gradual developmental processes. Moreover, individual plants within a population will undergo the developmental transitions asynchronously, so it is difficult to assemble a time series to resolve the sequence of transcriptional changes that take place during these rapid transitions. Single-plant-omics has the potential to distinguish between transcriptional events that are associated with these binary and continuous processes. Furthermore, we can utilise single-plant-omics to exploit this developmental asynchrony to order individual plants by their developmental trajectory, revealing a detailed cascade of transcriptional events. Here, we utilise single-plant-transcriptomics to resolve the transcriptional events that coincide with the onset of bolting. We performed RNA-seq on the leaves of individual plants from a large population of wild type Arabidopsis thaliana replicated at one time point during the vegetative-to-reproductive transition. Even though more than half of transcripts were differentially expressed between bolted and unbolted plants, we were able to find a subset of regulators that were more closely associated with gradual developmental traits like leaf size and biomass. Using a novel pseudotime inference algorithm, we determined that some senescence-associated processes, such as the reduction in ribosome biogenesis, are evident in the transcriptome before a bolt is visible. These results show the potential of single-plant-omics to reveal the detailed sequence of events that occur during rapid developmental transitions.\n",
  "wordCount" : "2510",
  "inLanguage": "en",
  "datePublished": "2023-09-13T10:39:03Z",
  "dateModified": "2023-09-13T10:39:03Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta"><span>updated on September 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556499">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556499" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556499">
        <p class="paperTitle">Retrotransposon-driven environmental regulation of FLC leads to adaptive response to herbicide</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556499" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556499" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Raingeval, M.; Leduque, B.; Baduel, P.; Edera, A.; Roux, F.; Colot, V.; Quadrana, L.</p>
        <p class="info">Score: 32.2, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556499' target='https://doi.org/10.1101/2023.09.06.556499'> 10.1101/2023.09.06.556499</a></p>
        <p class="abstract">The mobilization of retrotransposons yields major-effect mutations. Here, we report an adaptive retrotransposon insertion within the first intron of the Arabidopsis floral-repressor locus FLOWERING LOCUS C (FLC). The insertion-mutation augments the environmental sensitivity of FLC by affecting the balance between coding and non-coding transcript isoforms in response to environmental threads. We show that this balance is modulated epigenetically by DNA methylation and orchestrated by IBM2, a factor involved in the processing of intronic heterochromatin. The stress-sensitive allele of FLC has recently spread across populations subjected to recurrent chemical weeding, and we demonstrate that retrotransposon-driven acceleration of life cycle represents a rapid response to herbicide. Our findings illustrate how retrotransposition can create environmentally-sensitive alleles that facilitate adaptation to anthropogenic disturbances of the environment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556214">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556214" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556214">
        <p class="paperTitle">An atypical endomembrane localized CNL-type immune receptor with a conserved deletion in the N-terminal signaling domain functions in cell death and immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556214" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556214" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sunil, S.; Beeh, S.; Stoebbe, E.; Fischer, K. M.; Wilhelm, F.; Paris, C.; Meral, A.; Teasdale, L.; Jiang, Z.; Zhang, l.; Aguilar Parras, E.; Nuernberger, T.; Weigel, D.; Lozano-Duran, R.; El Kasmi, F.</p>
        <p class="info">Score: 20.6, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556214' target='https://doi.org/10.1101/2023.09.04.556214'> 10.1101/2023.09.04.556214</a></p>
        <p class="abstract">Plants have evolved intracellular nucleotide-binding leucine rich repeat receptors (NLRs) to induce a superior immune response. Upon activation, coiled-coil (CC) domain containing NLRs (CNLs) oligomerize to form apparent cation channels that promote calcium influx and cell death induction, with the alpha-1 helix of the individual CC domains penetrating membranes. Some members of a monophyletic subclass of CNLs, the ancient and autonomous NLRs (ANLs), are characterized by putative N- myristoylation and S-acylation sites at the N-terminus of their CCG10/GA domain, potentially mediating permanent membrane association. Whether these Potentially Membrane Localized NLRs (PMLs) mediate cell death upon activation in a similar way as reported for other CNLs has been unknown. We integrated phylogenetic, cell- biological, and functional studies to uncover the cell death function of an atypical but conserved Arabidopsis PML, PML5, which has a 113 amino acid deletion in its CCG10/GA domain. Active PML5 oligomers localize in Golgi membranes and the tonoplast, changes vacuolar morphology, and induce cell death, with the short N- terminus being sufficient for cell death. Mutant analysis supports a potential key role of PMLs in plant immunity. Similar deletions as in Arabidopsis PML5 are found in several Brassicales paralogs, pointing to the evolutionary importance of this innovation. PML5 is thus a naturally occurring CNL variant with a minimal signaling  domain and its further study should help in understanding the functional importance of this minimal domain for NLR signaling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556260">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556260" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556260">
        <p class="paperTitle">A soybean rust effector protease suppresses host immunity and cleaves a 3-deoxy-7-phosphoheptulonate synthase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556260" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556260" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chicowski, A. S.; Qi, M.; Variz, H.; Bredow, M.; Montes-Serey, C.; Caiazza, F.; Dong, H.; Margets, A. C.; Mejias, J.; Walley, J.; Craik, C.; Pedley, K. F.; Aung, K.; Innes, R. W.; Whitham, S.</p>
        <p class="info">Score: 15.8, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556260' target='https://doi.org/10.1101/2023.09.07.556260'> 10.1101/2023.09.07.556260</a></p>
        <p class="abstract">The devastating soybean rust (SBR) pathogen, Phakopsora pachyrhizi, encodes many secreted proteins, but only two have been functionally characterized for their roles in rust virulence. Here, we demonstrate that transient expression of P. pachyrhizi effector candidate 15 (PpEC15), an aspartic protease, leads to enhanced bacterial growth in planta, suppression of callose deposition, reduced expression of plant defense-related marker genes and suppression of pathogen-associated molecular pattern (PAMP)-induced reactive oxygen species (ROS). Stable expression of PpEC15 in soybean suppresses PAMP-induced ROS production and enhances bacterial growth, indicating that, collectively, PpEC15 suppresses host and non-host innate immune responses. Yeast-two-hybrid and proximity labeling identified putative PpEC15 interacting partners including a peptide-chain release factor (PCRF), a NAC83 (NAM, ATAF, and CUC) transcription factor, and a DAHP (3-deoxy-7-phosphoheptulonate) synthase. We further show that PpEC15 is able to cleave DAHP, but does not cleave PCRF or NAC83. Virus-induced gene silencing of NAC83, PCRF and DAHP altered PAMP-induced ROS production and salicylic acid production, indicating that these proteins may be involved in immune signaling. Collectively, our data show that PpEC15 is conserved across P. pachyrhizi isolates and other economically important rust species and is involved in the suppression of plant basal defense responses. Understanding the role of PpEC15 in P. pachyrhizi virulence will provide a foundation for designing targeted intervention strategies to generate rust-resistant crops.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556738">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556738" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556738">
        <p class="paperTitle">Stacking of PRRs in potato to achieve enhanced resistance against Phytophthora infestans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556738" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556738" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ascurra, Y. C. T.; Wouters, D.; Visser, R.; Nuernberger, T.; Vleeshouwers, V.</p>
        <p class="info">Score: 15.2, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556738' target='https://doi.org/10.1101/2023.09.07.556738'> 10.1101/2023.09.07.556738</a></p>
        <p class="abstract">Plants employ pattern recognition receptors (PRRs) to sense pathogen-associated molecular patterns (PAMPs) or apoplastic effectors at the plant cell surface, as well as nucleotide-binding domain leucine-rich-repeat-containing receptors (NLRs) to sense effectors inside the plant cell. Breeding for potato resistance to P. infestans has focused on the use of NLRs, however, these genes are typically quickly overcome since the matching avirulence genes evolve exceptionally quickly. Here, we stacked two PRRs, PERU and RLP23, that recognize the rather conserved Phytophthora PAMPs Pep-13/25 and nlp20, respectively, in the potato cultivar Atlantic, and evaluated their effect on P. infestans resistance. We found that PERU and RLP23 cooperate for the early immune responses like the accumulation of reactive oxygen species (ROS) and production of ethylene by recognizing their corresponding PAMPs. Furthermore, we show that potato plants overexpressing these two PRRs are slightly less affected by P. infestans compared to the single transformants. Together, our data suggest that pyramiding of surface receptors can provide additional enhanced resistance against pathogens, however, more effective or synergistic combinations that may include intracellular NLR receptors should be explored.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.03.556139">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.03.556139" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.03.556139">
        <p class="paperTitle">Establishing a comprehensive web-based analysis platform for Nicotiana benthamiana genome and transcriptome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.03.556139" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.03.556139" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kurotani, K.-i.; Hirakawa, H.; Shirasawa, K.; Tagiri, K.; Mori, M.; Ichihashi, Y.; Suzuki, T.; Tanizawa, Y.; Nakamura, Y.; Isobe, S.; Notaguchi, M.</p>
        <p class="info">Score: 15.2, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.03.556139' target='https://doi.org/10.1101/2023.09.03.556139'> 10.1101/2023.09.03.556139</a></p>
        <p class="abstract">Nicotiana benthamiana has long served as a crucial plant material extensively used in plant physiology research, particularly in the field of plant pathology, because of its high susceptibility to plant viruses. Additionally, it serves as a production platform to test vaccines and other valuable substances. Among its approximately 3.1 Gb genome, 57,583 genes have been annotated within a 61 Mb region. We created a comprehensive and easy-to-use platform to use transcriptomes for modern annotation. These tools allow to visualize gene expression profiles, draw molecular evolutionary phylogenetic trees of gene families, perform functional enrichment analyses, and facilitate output downloads. To demonstrate their utility, we analyzed the gene expression profiles of enzymes within the nicotine biosynthesis pathway, a secondary metabolic pathway characteristic of the Nicotiana genus. Using the developed tool, expression profiles of the nicotine biosynthesis pathway genes were generated. The expression patterns of eight gene groups in the pathway were strongly expressed in the roots and weakly expressed in leaves and flowers of N. benthamiana. The results were consistent with the established gene expression profiles in Nicotiana tabacum and provided insights into gene family composition and expression trends. The compilation of this database tool can facilitate genetic analysis of N. benthamiana in the future.

Significance statementA tool was developed to visualize gene expression profiles, draw molecular evolutionary phylogenetic trees of gene families, perform functional enrichment analyses, and facilitate output downloads of Nicotiana benthamiana. The database developed using the tool can evolve into a comprehensive all-in-one analysis platform by continuously incorporating transcriptome data released to date, newly released RNA-seq data, and annotations in the future.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.27.554993">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.27.554993" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.27.554993">
        <p class="paperTitle">Water fluxes contribute to growth patterning in shoot meristems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.27.554993" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.27.554993" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alonso-Serra, J.; Cheddadi, I.; Kiss, A.; Cerutti, G.; Lionnet, C.; Godin, C.; Hamant, O.</p>
        <p class="info">Score: 35.1, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.27.554993' target='https://doi.org/10.1101/2023.08.27.554993'> 10.1101/2023.08.27.554993</a></p>
        <p class="abstract">In multicellular organisms, localized tissue outgrowth creates a new water sink thereby modifying hydraulic patterns at the organ level. These fluxes are often considered passive by-products of development and their patterning and potential contribution to morphogenesis remains largely unexplored. Here, we generated a complete map of cell volumetric growth and deformation across the shoot apex in Arabidopsis thaliana. Within the organ-meristem boundary, we found that a subpopulation of cells next to fast-growing cells experiences volumetric shrinkage. To understand this process, we used a vertex-based model integrating mechanics and hydraulics, informed by the measured growth rates. Organ outgrowth simulations revealed the emerging water fluxes and predicted water deficit with volume loss for a few cells at the boundary. Consistently, in planta, a water-soluble dye is preferentially allocated to fast-growing tissues and fails to enter the boundary domain. Analysis of intact meristems further validated our model by revealing cell shrinkage next to fast-growing cells in different contexts of tissue surface curvature and cell deformation. A molecular signature of water deficit at the boundary further confirmed our conclusion. Taken together, we propose a model where the differential sink strength of emerging organs prescribes the hydraulic patterns that define the boundary domain at the shoot apex.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556846">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556846" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556846">
        <p class="paperTitle">Yerba mate (Ilex paraguariensis) genome provides new insights into convergent evolution of caffeine biosynthesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556846" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556846" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vignale, F. A.; Hernandez Garcia, A.; Modenutti, C. P.; Sosa, E. J.; Defelipe, L. A.; Oliveira, R. R. M.; Nunes, G. L.; Acevedo, R. M.; Burguener, G. F.; Rossi, M.; Zapata, P. D.; Marti, D. A.; Oliveira, G.; Smith, M. N.; Dubs, N. M.; Nair, S.; Barkman, T. J.; Turjanski, A. G.</p>
        <p class="info">Score: 8.8, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556846' target='https://doi.org/10.1101/2023.09.08.556846'> 10.1101/2023.09.08.556846</a></p>
        <p class="abstract">Yerba mate (Ilex paraguariensis) is an economically important crop marketed for the elaboration of mate, the third-most widely consumed caffeine-containing infusion worldwide. Here we report the first genome assembly of this species, which has a total length of 1.06 Gb and contains 53,390 protein-coding genes. Comparative analyses revealed that the large yerba mate genome size is partly due to a whole-genome duplication (Ip-) during the early evolutionary history of Ilex, in addition to the hexaploidization event ({gamma}) shared by core eudicots. Characterization of the genome allowed us to clone the genes encoding methyltransferase enzymes that catalyse multiple reactions required for caffeine production. To our surprise, this species has converged upon a different biochemical pathway compared to that of its relatives, coffee and tea. In order to gain insight into the structural basis for the convergent enzyme activities, we obtained a crystal structure for the terminal enzyme in the pathway that forms caffeine. The structure reveals that convergent solutions have evolved for substrate positioning because different amino acid residues facilitate a different substrate orientation such that efficient methylation occurs in the independently evolved enzymes in yerba mate and coffee. While our results show phylogenomic constraint limits the genes coopted for convergence of caffeine biosynthesis, the x-ray diffraction data suggests structural constraints are minimal for the convergent evolution of individual reactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.556951">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.556951" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.556951">
        <p class="paperTitle">Control of cell fate specification and patterning by an ancestral microRNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.556951" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.556951" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aguilar-Cruz, A.; Flores-Sandoval, E.; Gutierrez-Ramos, X.; Oltehua-Lopez, O.; Dorantes-Acosta, A. E.; Trujillo, J. T.; Kato, H.; Ishizaki, K.; Mosher, R. A.; Dolan, L.; Grimanelli, D.; Haseloff, J.; Bowman, J. L.; Arteaga-Vazquez, M. A.</p>
        <p class="info">Score: 8.8, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.556951' target='https://doi.org/10.1101/2023.09.09.556951'> 10.1101/2023.09.09.556951</a></p>
        <p class="abstract">The formation of an organized body requires the establishment and maintenance of cells with structural and functional distinctive characteristics. A central question in developmental biology is how changes in the regulation of genes drive cell specification and patterning1. microRNAs (miRNAs) are small non-coding RNAs that regulate development through mRNA cleavage and/or translational repression2. In plants, miRNAs regulate key aspects including growth, development, stem cell maintenance, vegetative phase change, leaf morphogenesis, floral organ formation and flowering time3. Biogenesis of plant miRNAs depends on the activity of DICER-LIKE 1 (DCL1), an RNase type III endonuclease that processes double stranded RNA to give rise to mature miRNAs 4 . The genomes of today&#39;s flora contain at least one bona fide copy of DCL1 5,6. Using Marchantia polymorpha -a model bryophyte that allows comparative approaches to infer characteristics of the ancestral land plant-, we demonstrate that MpDCL1a is required for the biogenesis of miRNAs and uncovered a central role for miR166/Homeodomain Zipper Class III-regulated auxin synthesis in the specification of cell identity, patterning, meristem function, laminar expansion and the development of the body in the last common ancestor of extant land plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555981">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555981" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555981">
        <p class="paperTitle">Historical causes for the greater proportion of polyploid plants in higher latitudes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555981" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555981" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hagen, E. R.; Vasconcelos, T.; Boyko, J. D.; Beaulieu, J. M.</p>
        <p class="info">Score: 7.7, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555981' target='https://doi.org/10.1101/2023.09.01.555981'> 10.1101/2023.09.01.555981</a></p>
        <p class="abstract">Premise of the StudyThe proportion of polyploid plants in a community increases with latitude, and different hypotheses have been proposed about which factors drive this pattern. Here, we aim to understand the historical causes of the latitudinal polyploidy gradient using a combination of ancestral state reconstruction methods. Specifically, we assess whether (1) polyploidization enables movement to higher latitudes (i.e., polyploidization precedes occurrences in higher latitudes) or (2) higher latitudes facilitate polyploidization (i.e., occurrence in higher latitudes precedes polyploidization).

MethodsWe reconstruct the ploidy states and ancestral niches of 1,032 angiosperm species at four paleoclimatic time slices ranging from 3.3 million years ago to the present, comprising taxa from four well-represented clades: Onagraceae, Primulaceae, Solanum (Solanaceae), and Pooideae (Poaceae). We use ancestral niche reconstruction models alongside a customized discrete character evolution model to allow reconstruction of states at specific time slices. Patterns of latitudinal movement are reconstructed and compared in relation to inferred ploidy shifts.

Key ResultsWe find that no single hypothesis applies equally well across all analyzed clades. While significant differences in median latitudinal occurrence were detected in the largest clade, Pooideae, no significant differences were detected in latitudinal movement in any clade.

ConclusionsOur preliminary study is the first to attempt to connect ploidy changes to continuous latitudinal movement, but we cannot favor one hypothesis over another. Given that patterns seem to be clade-specific, a larger number of clades must be analyzed in future studies for generalities to be drawn.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557157">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557157" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557157">
        <p class="paperTitle">Single-plant-omics reveals the cascade of transcriptional changes during the vegetative-to-reproductive transition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557157" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557157" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Redmond, E. J.; Ronald, J.; Davis, S. J.; Ezer, D.</p>
        <p class="info">Score: 6.1, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557157' target='https://doi.org/10.1101/2023.09.11.557157'> 10.1101/2023.09.11.557157</a></p>
        <p class="abstract">Plants undergo rapid developmental transitions, as well as gradual developmental processes. Moreover, individual plants within a population will undergo the developmental transitions asynchronously, so it is difficult to assemble a time series to resolve the sequence of transcriptional changes that take place during these rapid transitions. Single-plant-omics has the potential to distinguish between transcriptional events that are associated with these binary and continuous processes. Furthermore, we can utilise single-plant-omics to exploit this developmental asynchrony to order individual plants by their developmental trajectory, revealing a detailed cascade of transcriptional events. Here, we utilise single-plant-transcriptomics to resolve the transcriptional events that coincide with the onset of bolting. We performed RNA-seq on the leaves of individual plants from a large population of wild type Arabidopsis thaliana replicated at one time point during the vegetative-to-reproductive transition. Even though more than half of transcripts were differentially expressed between bolted and unbolted plants, we were able to find a subset of regulators that were more closely associated with gradual developmental traits like leaf size and biomass. Using a novel pseudotime inference algorithm, we determined that some senescence-associated processes, such as the reduction in ribosome biogenesis, are evident in the transcriptome before a bolt is visible. These results show the potential of single-plant-omics to reveal the detailed sequence of events that occur during rapid developmental transitions.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
