<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment
Authors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium, ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment
Authors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium, ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-05T10:37:37+00:00" />
<meta property="article:modified_time" content="2023-11-05T10:37:37+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment
Authors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium, ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment\nAuthors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium, ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J.",
  "keywords": [
    
  ],
  "articleBody": " The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment\nAuthors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium, ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J. H.; Schmutz, J.; Webber, J.; Grimwood, J.; Jacquet, C.; Dunand, C.; Nelson, J.; Roux, F.; Philippe, H.; Schornack, S.; Bonhomme, M.; Delaux, P.-M.\nScore: 62.1, Published: 2023-10-29 DOI: 10.1101/2023.10.27.564390\nPlant adaptation to a terrestrial life 450 million years ago played a major role in the evolution of life on Earth. This shift from an aquatic environment has been mostly studied by focusing on flowering plants. Here, we gathered a collection of 133 accessions of the non-vascular plants Marchantia polymorpha and studied its intraspecific diversity using selection signature analyses, genome-environment association study and a gene-centered pangenome. We identified adaptive features shared with flowering plants, such as peroxidases or nucleotide-binding and leucine-rich repeat (NLR), which likely played a role in the adaptation of the first land plants to the terrestrial habitat. The M. polymorpha pangenome also harbored lineage-specific accessory genes absent from seed plants. We conclude that different land plants lineages still share many elements from the genetic toolkit evolved by their most recent common ancestor to adapt to the terrestrial habitat, refined by lineage specific polymorphisms and gene family evolutions.\nDELLA Proteins Recruit the Mediator Complex Subunit MED15 to Co-activate Transcription in Land Plants\nAuthors: Hernandez-Garcia, J.; Serrano-Mislata, A.; Lozano-Quiles, M.; Urbez, C.; Nohales, M. A.; Blanco-Tourinan, N.; Peng, H.; Ledesma-Amaro, R.; Blazquez, M. A.\nScore: 27.6, Published: 2023-11-01 DOI: 10.1101/2023.11.01.565078\nDELLA proteins are negative regulators of the gibberellin response pathway in angiosperms, acting as central hubs that interact with hundreds of transcription factors and regulators to modulate their activities. While the mechanism of transcription factor sequestration by DELLAs to prevent DNA binding to downstream targets has been extensively documented, the mechanism that allows them to act as co-activators remains to be understood. Here, we demonstrate that DELLAs directly recruit the Mediator complex to specific loci in Arabidopsis, facilitating transcription. This recruitment involves DELLA amino-terminal domain and the conserved MED15 KIX domain. Accordingly, partial loss of MED15 function mainly disrupted processes known to rely on DELLA co-activation capacity; including cytokinin-dependent regulation of meristem function and skotomorphogenic response, gibberellin metabolism feedback, and flavonol production. We have also found that the single DELLA protein in the liverwort Marchantia polymorpha is capable of recruiting MpMED15 subunits, contributing to transcriptional co-activation. The conservation of Mediator-dependent transcriptional co-activation by DELLA between Arabidopsis and Marchantia implies that this mechanism is intrinsic to the emergence of DELLA in the last common ancestor of land plants.\nPopulation level gene expression can repeatedly link genes to functions in maize\nAuthors: Torres-Rodriguez, J. V.; Li, D.; Turkus, J.; Newton, L.; Davis, J.; Lopez-Corona, L.; Ali, W.; Sun, G.; Mural, R. V.; Grzybowski, M. W.; Thompson, A. M.; Schnable, J. C.\nScore: 26.9, Published: 2023-11-01 DOI: 10.1101/2023.10.31.565032\nTranscriptome-Wide Association Studies (TWAS) can provide single gene resolution for candidate genes in plants, complementing Genome-Wide Association Studies (GWAS) but efforts in plants have been met with, at best, mixed success. We generated expression data from 693 maize genotypes, measured in a common field experiment, sampled over a two-hour period to minimize diurnal and environmental effects, using full-length RNA-seq to maximize the accurate estimation of transcript abundance. TWAS could identify roughly ten times as many genes likely to play a role in flowering time regulation as GWAS conducted data from the same experiment. TWAS using mature leaf tissue identified known true positive flowering time genes known to act in the shoot apical meristem, and trait data from new environments enabled the identification of additional flowering time genes without the need for new expression data. eQTL analysis of TWAS-tagged genes identified at least one additional known maize flowering time gene through trans-eQTL interactions. Collectively these results suggest the gene expression resource described here can link genes to functions across different plant phenotypes expressed in a range of tissues and scored in different experiments.\nTargeting editing of tomato SPEECHLESS cis-regulatory regions generates plants with altered stomatal density in response to changing climate conditions\nAuthors: Nir, I.; Budrys, A.; Smoot, N. K.; Erberich, J.; Bergmann, D.\nScore: 22.3, Published: 2023-11-02 DOI: 10.1101/2023.11.02.564550\nFlexible developmental programs enable plants to customize their organ size and cellular composition. In leaves of eudicots, the stomatal lineage produces two essential cell types, stomata and pavement cells, but the total numbers and ratio of these cell types can vary. Central to this flexibility is the stomatal lineage initiating transcription factor, SPEECHLESS (SPCH). Here we show, by multiplex CRISPR/Cas9 editing of SlSPCH cis-regulatory sequences in tomato, that we can identify variants with altered stomatal development responses to light and temperature cues. Analysis of tomato leaf development across different conditions, aided by newly-created tools for live-cell imaging and translational reporters of SlSPCH and its paralogues SlMUTE and SlFAMA, revealed the series of cellular events that lead to the environmental change-driven responses in leaf form. Plants bearing the novel SlSPCH variants generated in this study are powerful resources for fundamental and applied studies of tomato resilience in response to climate change.\nA fully sequenced collection of homozygous EMS mutants for forward and reverse genetic screens in Arabidopsis thaliana\nAuthors: Carrere, S.; Routaboul, J.-M.; Savourat, P.; Bellenot, C.; Lopez, H.; Quiroz-Monnens, T.; Ricou, A.; Camilleri, C.; Laufs, P.; mercier, r.; Noel, L. D.\nScore: 56.7, Published: 2023-10-27 DOI: 10.1101/2023.10.26.564234\nGenetic screens are powerful tools for biological research and are one of the reasons for the success of the thale cress Arabidopsis thaliana as a model species. Here, we describe the whole-genome sequencing of 871 Arabidopsis lines from the Homozygous EMS Mutant (HEM) collection as a novel resource for forward and reverse genetics. With an average 576 high-confidence mutations per HEM line, over three independent mutations altering protein sequence are found on average per gene in the collection. Pilot reverse genetics experiments on reproductive, developmental and physiological traits confirmed the efficacy of the tool for identifying both null and knockdown alleles. The possibility of conducting subtle repeated phenotyping of HEM lines and the immediate availability of the mutations will empower forward genetic approaches. The sequence resource is searchable with the ATHEM web interface (https://lipm-browsers.toulouse.inra.fr/pub/ATHEM/), and the biological material is distributed by the Versailles Arabidopsis Stock Center.\nSingle nuclei sequencing reveals C4 photosynthesis is based on rewiring of ancestral cell identity networks\nAuthors: Swift, J.; Luginbuehl, L. H.; Schreier, T.; Donald, R. M.; Lee, T.; Nery, J.; Ecker, J. R.; Hibberd, J. M.\nScore: 54.3, Published: 2023-10-27 DOI: 10.1101/2023.10.26.562893\nIn multicellular systems changes to the patterning of gene expression drive modifications in cell function and trait evolution. One striking example is found in more than sixty plant lineages where compartmentation of photosynthesis between cell types allowed the evolution of the efficient C4 pathway from the ancestral C3 state. The molecular events enabling this transition are unclear. We used single nuclei sequencing to generate atlases for C3 rice and C4 sorghum during photomorphogenesis. Our analysis revealed that initiation of photosynthesis gene expression is conditioned by cell identity. In both species a conserved cistrome defines each cell type, and photosynthesis genes switching expression from mesophyll in rice to bundle sheath in sorghum acquire hallmarks of bundle sheath identity. The sorghum bundle sheath has also acquired gene networks associated with C3 guard cells. We conclude C4 photosynthesis is based on rewiring in cis that exapts cell identity networks of C3 plants.\nATG8 delipidation is dispensable for plant autophagy\nAuthors: Holla, S.; Zou, Y.; Sabljic, I.; Ohlsson, J. A.; Leong, J. X.; Ballhaus, F.; Krebs, M.; Schumacher, K.; Uestuen, S.; Dagdas, Y.; Bozhkov, P.; Minina, E. A.\nScore: 16.8, Published: 2023-11-03 DOI: 10.1101/2023.08.23.554513\nIntracellular recycling via autophagy is governed by post-translational modifications of the autophagy related (ATG) proteins. One example is ATG4-dependent delipidation of ATG8, which assumes criti-cal but distinct roles in autophagosome formation of yeast and mammals. Here, we discovered that ATG8 delipidation is dispensable for the maturation of autophagosomes, autophagic flux and related stress tolerance in Arabidopsis. In contrast, autophagy in an evolutionary distant Chlamydomonas strictly required ATG4-dependent ATG8 delipidation. Additionally, we uncovered distinct participa-tion of Arabidopsis ATG8 orthologs in autophagosome biogenesis, especially prevalent under nitro-gen depletion, thereby providing new insights into potential reasons behind ATG8 gene family expan-sion in vascular plants. Our findings point to evolutionary diversification of the molecular mechanism governing maturation of autophagosomes in eukaryotic lineages and highlight how this conserved pathway is tailored to diverse eukaryotic organisms.\nA Ralstonia solanacearum type III effector alters the actin and microtubule cytoskeleton to promote bacterial virulence in plants\nAuthors: Hiles, R.; Rogers, A.; Jaiswal, N.; Zhang, W.; Butchacas, J.; Merfa, M.; Klass, T.; Kaser, E.; Jacobs, J. M.; Staiger, C. J.; Helm, M. J.; Iyer-Pascuzzi, A. S.\nScore: 13.4, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565113\nCellular responses to biotic stress frequently involve signaling pathways that are conserved across eukaryotes. These pathways include the cytoskeleton, a proteinaceous network that senses external cues at the cell surface and signals to interior cellular components. During biotic stress, dynamic cytoskeletal rearrangements serve as a platform from which early immune-associated processes are organized and activated. Bacterial pathogens of plants and animals use proteins called type III effectors (T3Es) to interfere with host immune signaling, thereby promoting virulence. We previously found that RipU, a T3E from the soilborne phytobacterial pathogen Ralstonia solanacearum K60 (Rs K60), co-localizes with the plant cytoskeleton. Here, we show that RipU from Rs K60 (RipUK60 ) physically associates with both actin and tubulin and disrupts actin and microtubule cytoskeleton organization. We find that pharmacological disruption of the tomato (Solanum lycopersicum) cytoskeleton promotes Rs K60 colonization. RipUK60 suppresses cell surface-triggered immune responses including flg22-mediated reactive oxygen species (ROS) production and callose deposition. Importantly, tomato plants inoculated with Rs K60 lacking RipU K60 ({Delta}ripUK60) had reduced wilting symptoms and significantly reduced root colonization when compared to plants inoculated with wild-type Rs K60. Collectively, our data suggest that Rs K60 uses the type III effector RipUK60 to remodel cytoskeletal organization, thereby promoting pathogen virulence.\nGenome-wide evidence of the role of Paf1C in transcription elongation and histone H2B monoubiquitination in Arabidopsis\nAuthors: Blanco, N.; Perez-Alemany, J.; Bourbousse, C.; latrasse, D.; Ait-Mohamed, O.; Benhamed, M.; Barneche, F.; Blazquez, M. A.; Gallego-Bartolome, J.; Alabadi, D.\nScore: 12.5, Published: 2023-10-31 DOI: 10.1101/2023.10.27.564443\nThe evolutionarily conserved Paf1 complex (Paf1C) participates in transcription, and research in animals and fungi suggests that it facilitates RNAPII progression through chromatin remodeling. To obtain evidence that Paf1C acts in transcription elongation in Arabidopsis, we examined the genomic distribution of the ELF7 and VIP3 subunits of Paf1C. The occupancy of both subunits was confined to thousands of gene bodies and positively correlated with RNAPII occupancy and the level of gene expression, supporting a role as a transcription elongation factor. We found that monoubiquitinated histone H2B, which marks most transcribed genes, was strongly reduced genome-wide in elf7 seedlings. Genome-wide profiling of RNAPII revealed that in elf7 mutants, RNAPII occupancy was reduced throughout the gene body and at the transcription end site of Paf1C-targeted genes, suggesting a direct role for the complex in transcription elongation. Overall, our observations suggest that there is a direct functional link between Paf1C activity, mono-ubiquitination of histone H2B, and the transition of RNPII to productive elongation. However, for several genes, Paf1C may also act independently of H2Bub deposition or occupy these genes more stably than H2Bub marking, possibly reflecting the dynamic nature of Paf1C association and H2Bub turnover during transcription.\nPan-genome analysis of different morphotypes reveals genomic basis of Brassica oleracea domestication and differential organogenesis\nAuthors: Guo, N.; Wang, S.; Wang, T.; Duan, M.; Zong, M.; Miao, L.; Han, S.; Wang, G.; Liu, X.; Zhang, D.; Jiao, C.; Xu, H.; Chen, L.; Fei, Z.; Li, J.; Liu, F.\nScore: 11.3, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563711\nThe domestication of Brassica oleracea has resulted in diverse morphological types with distinct patterns of organ development. Here we report a graph-based pan-genome of B. oleracea constructed with high-quality genome assemblies of different morphotypes. The pan-genome harbors over 200 structural variant (SV) hotspot regions enriched with auxin and flowering-related genes. Population genomic analyses reveal that early domestication of B. oleracea focused on leaf or stem selection. Gene flows resulting from agricultural practices and variety improvement are detected among different morphotypes. Selective sweep analysis identifies an auxin-responsive SAUR gene and a CLE family gene as the crucial players in the leaf-stem differentiation during the early stage of B. oleracea domestication, and the BoKAN1 gene as instrumental in shaping the leafy heads of cabbage and Brussels sprouts. Our pan-genome and functional analyses further discover that variations in the BoFLC2 gene play key roles in the divergence of vernalization and flowering characteristics among different morphotypes, and variations in the first intron of BoFLC3 are involved in fine-tuning the flowering process in cauliflower. This study provides a comprehensive understanding of the pan-genome of B. oleracea and sheds light on the domestication and differential organ development of this globally important crop species.\n",
  "wordCount" : "2185",
  "inLanguage": "en",
  "datePublished": "2023-11-05T10:37:37Z",
  "dateModified": "2023-11-05T10:37:37Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 5, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564390">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564390" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564390">
        <p class="paperTitle">The Marchantia pangenome reveals ancient mechanisms of plant adaptation to the environment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564390" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564390" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Beaulieu, C.; Libourel, C.; Mbadinga Zamar, D.-L.; El Mahboubi, K.; Hoey, D. J.; Keller, J.; Girou, C.; San Clemente, H.; Diop, I.; Amblard, E.; Theron, A.; Cauet, S.; Rodde, N.; Sabine, Z.; Halpape, W.; Meierhenrich, A.; Laker, B.; Braeutigam, A.; Greiff, G. R. L.; The SLCU Outreach Consortium,  ; Szovenyi, P.; Cheng, S.; Tanizawa, Y.; Leebens-Mack, J. H.; Schmutz, J.; Webber, J.; Grimwood, J.; Jacquet, C.; Dunand, C.; Nelson, J.; Roux, F.; Philippe, H.; Schornack, S.; Bonhomme, M.; Delaux, P.-M.</p>
        <p class="info">Score: 62.1, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564390' target='https://doi.org/10.1101/2023.10.27.564390'> 10.1101/2023.10.27.564390</a></p>
        <p class="abstract">Plant adaptation to a terrestrial life 450 million years ago played a major role in the evolution of life on Earth. This shift from an aquatic environment has been mostly studied by focusing on flowering plants. Here, we gathered a collection of 133 accessions of the non-vascular plants Marchantia polymorpha and studied its intraspecific diversity using selection signature analyses, genome-environment association study and a gene-centered pangenome. We identified adaptive features shared with flowering plants, such as peroxidases or nucleotide-binding and leucine-rich repeat (NLR), which likely played a role in the adaptation of the first land plants to the terrestrial habitat. The M. polymorpha pangenome also harbored lineage-specific accessory genes absent from seed plants. We conclude that different land plants lineages still share many elements from the genetic toolkit evolved by their most recent common ancestor to adapt to the terrestrial habitat, refined by lineage specific polymorphisms and gene family evolutions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565078">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565078" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565078">
        <p class="paperTitle">DELLA Proteins Recruit the Mediator Complex Subunit MED15 to Co-activate Transcription in Land Plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565078" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565078" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hernandez-Garcia, J.; Serrano-Mislata, A.; Lozano-Quiles, M.; Urbez, C.; Nohales, M. A.; Blanco-Tourinan, N.; Peng, H.; Ledesma-Amaro, R.; Blazquez, M. A.</p>
        <p class="info">Score: 27.6, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565078' target='https://doi.org/10.1101/2023.11.01.565078'> 10.1101/2023.11.01.565078</a></p>
        <p class="abstract">DELLA proteins are negative regulators of the gibberellin response pathway in angiosperms, acting as central hubs that interact with hundreds of transcription factors and regulators to modulate their activities. While the mechanism of transcription factor sequestration by DELLAs to prevent DNA binding to downstream targets has been extensively documented, the mechanism that allows them to act as co-activators remains to be understood. Here, we demonstrate that DELLAs directly recruit the Mediator complex to specific loci in Arabidopsis, facilitating transcription. This recruitment involves DELLA amino-terminal domain and the conserved MED15 KIX domain. Accordingly, partial loss of MED15 function mainly disrupted processes known to rely on DELLA co-activation capacity; including cytokinin-dependent regulation of meristem function and skotomorphogenic response, gibberellin metabolism feedback, and flavonol production. We have also found that the single DELLA protein in the liverwort Marchantia polymorpha is capable of recruiting MpMED15 subunits, contributing to transcriptional co-activation. The conservation of Mediator-dependent transcriptional co-activation by DELLA between Arabidopsis and Marchantia implies that this mechanism is intrinsic to the emergence of DELLA in the last common ancestor of land plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.565032">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.565032" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.565032">
        <p class="paperTitle">Population level gene expression can repeatedly link genes to functions in maize</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.565032" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.565032" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Torres-Rodriguez, J. V.; Li, D.; Turkus, J.; Newton, L.; Davis, J.; Lopez-Corona, L.; Ali, W.; Sun, G.; Mural, R. V.; Grzybowski, M. W.; Thompson, A. M.; Schnable, J. C.</p>
        <p class="info">Score: 26.9, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.565032' target='https://doi.org/10.1101/2023.10.31.565032'> 10.1101/2023.10.31.565032</a></p>
        <p class="abstract">Transcriptome-Wide Association Studies (TWAS) can provide single gene resolution for candidate genes in plants, complementing Genome-Wide Association Studies (GWAS) but efforts in plants have been met with, at best, mixed success. We generated expression data from 693 maize genotypes, measured in a common field experiment, sampled over a two-hour period to minimize diurnal and environmental effects, using full-length RNA-seq to maximize the accurate estimation of transcript abundance. TWAS could identify roughly ten times as many genes likely to play a role in flowering time regulation as GWAS conducted data from the same experiment. TWAS using mature leaf tissue identified known true positive flowering time genes known to act in the shoot apical meristem, and trait data from new environments enabled the identification of additional flowering time genes without the need for new expression data. eQTL analysis of TWAS-tagged genes identified at least one additional known maize flowering time gene through trans-eQTL interactions. Collectively these results suggest the gene expression resource described here can link genes to functions across different plant phenotypes expressed in a range of tissues and scored in different experiments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.564550">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.564550" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.564550">
        <p class="paperTitle">Targeting editing of tomato SPEECHLESS cis-regulatory regions generates plants with altered stomatal density in response to changing climate conditions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.564550" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.564550" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nir, I.; Budrys, A.; Smoot, N. K.; Erberich, J.; Bergmann, D.</p>
        <p class="info">Score: 22.3, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.564550' target='https://doi.org/10.1101/2023.11.02.564550'> 10.1101/2023.11.02.564550</a></p>
        <p class="abstract">Flexible developmental programs enable plants to customize their organ size and cellular composition. In leaves of eudicots, the stomatal lineage produces two essential cell types, stomata and pavement cells, but the total numbers and ratio of these cell types can vary. Central to this flexibility is the stomatal lineage initiating transcription factor, SPEECHLESS (SPCH). Here we show, by multiplex CRISPR/Cas9 editing of SlSPCH cis-regulatory sequences in tomato, that we can identify variants with altered stomatal development responses to light and temperature cues. Analysis of tomato leaf development across different conditions, aided by newly-created tools for live-cell imaging and translational reporters of SlSPCH and its paralogues SlMUTE and SlFAMA, revealed the series of cellular events that lead to the environmental change-driven responses in leaf form. Plants bearing the novel SlSPCH variants generated in this study are powerful resources for fundamental and applied studies of tomato resilience in response to climate change.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564234">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564234" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564234">
        <p class="paperTitle">A fully sequenced collection of homozygous EMS mutants for forward and reverse genetic screens in Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564234" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564234" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carrere, S.; Routaboul, J.-M.; Savourat, P.; Bellenot, C.; Lopez, H.; Quiroz-Monnens, T.; Ricou, A.; Camilleri, C.; Laufs, P.; mercier, r.; Noel, L. D.</p>
        <p class="info">Score: 56.7, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564234' target='https://doi.org/10.1101/2023.10.26.564234'> 10.1101/2023.10.26.564234</a></p>
        <p class="abstract">Genetic screens are powerful tools for biological research and are one of the reasons for the success of the thale cress Arabidopsis thaliana as a model species. Here, we describe the whole-genome sequencing of 871 Arabidopsis lines from the Homozygous EMS Mutant (HEM) collection as a novel resource for forward and reverse genetics. With an average 576 high-confidence mutations per HEM line, over three independent mutations altering protein sequence are found on average per gene in the collection. Pilot reverse genetics experiments on reproductive, developmental and physiological traits confirmed the efficacy of the tool for identifying both null and knockdown alleles. The possibility of conducting subtle repeated phenotyping of HEM lines and the immediate availability of the mutations will empower forward genetic approaches. The sequence resource is searchable with the ATHEM web interface (https://lipm-browsers.toulouse.inra.fr/pub/ATHEM/), and the biological material is distributed by the Versailles Arabidopsis Stock Center.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.562893">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.562893" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.562893">
        <p class="paperTitle">Single nuclei sequencing reveals C4 photosynthesis is based on rewiring of ancestral cell identity networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.562893" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.562893" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Swift, J.; Luginbuehl, L. H.; Schreier, T.; Donald, R. M.; Lee, T.; Nery, J.; Ecker, J. R.; Hibberd, J. M.</p>
        <p class="info">Score: 54.3, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.562893' target='https://doi.org/10.1101/2023.10.26.562893'> 10.1101/2023.10.26.562893</a></p>
        <p class="abstract">In multicellular systems changes to the patterning of gene expression drive modifications in cell function and trait evolution. One striking example is found in more than sixty plant lineages where compartmentation of photosynthesis between cell types allowed the evolution of the efficient C4 pathway from the ancestral C3 state. The molecular events enabling this transition are unclear. We used single nuclei sequencing to generate atlases for C3 rice and C4 sorghum during photomorphogenesis. Our analysis revealed that initiation of photosynthesis gene expression is conditioned by cell identity. In both species a conserved cistrome defines each cell type, and photosynthesis genes switching expression from mesophyll in rice to bundle sheath in sorghum acquire hallmarks of bundle sheath identity. The sorghum bundle sheath has also acquired gene networks associated with C3 guard cells. We conclude C4 photosynthesis is based on rewiring in cis that exapts cell identity networks of C3 plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554513">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554513" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554513">
        <p class="paperTitle">ATG8 delipidation is dispensable for plant autophagy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554513" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554513" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Holla, S.; Zou, Y.; Sabljic, I.; Ohlsson, J. A.; Leong, J. X.; Ballhaus, F.; Krebs, M.; Schumacher, K.; Uestuen, S.; Dagdas, Y.; Bozhkov, P.; Minina, E. A.</p>
        <p class="info">Score: 16.8, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554513' target='https://doi.org/10.1101/2023.08.23.554513'> 10.1101/2023.08.23.554513</a></p>
        <p class="abstract">Intracellular recycling via autophagy is governed by post-translational modifications of the autophagy related (ATG) proteins. One example is ATG4-dependent delipidation of ATG8, which assumes criti-cal but distinct roles in autophagosome formation of yeast and mammals. Here, we discovered that ATG8 delipidation is dispensable for the maturation of autophagosomes, autophagic flux and related stress tolerance in Arabidopsis. In contrast, autophagy in an evolutionary distant Chlamydomonas strictly required ATG4-dependent ATG8 delipidation. Additionally, we uncovered distinct participa-tion of Arabidopsis ATG8 orthologs in autophagosome biogenesis, especially prevalent under nitro-gen depletion, thereby providing new insights into potential reasons behind ATG8 gene family expan-sion in vascular plants. Our findings point to evolutionary diversification of the molecular mechanism governing maturation of autophagosomes in eukaryotic lineages and highlight how this conserved pathway is tailored to diverse eukaryotic organisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565113">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565113" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565113">
        <p class="paperTitle">A  Ralstonia solanacearum  type III effector alters the actin and microtubule cytoskeleton to promote bacterial virulence in plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565113" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565113" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hiles, R.; Rogers, A.; Jaiswal, N.; Zhang, W.; Butchacas, J.; Merfa, M.; Klass, T.; Kaser, E.; Jacobs, J. M.; Staiger, C. J.; Helm, M. J.; Iyer-Pascuzzi, A. S.</p>
        <p class="info">Score: 13.4, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565113' target='https://doi.org/10.1101/2023.11.01.565113'> 10.1101/2023.11.01.565113</a></p>
        <p class="abstract">Cellular responses to biotic stress frequently involve signaling pathways that are conserved across eukaryotes. These pathways include the cytoskeleton, a proteinaceous network that senses external cues at the cell surface and signals to interior cellular components. During biotic stress, dynamic cytoskeletal rearrangements serve as a platform from which early immune-associated processes are organized and activated. Bacterial pathogens of plants and animals use proteins called type III effectors (T3Es) to interfere with host immune signaling, thereby promoting virulence. We previously found that RipU, a T3E from the soilborne phytobacterial pathogen Ralstonia solanacearum K60 (Rs K60), co-localizes with the plant cytoskeleton. Here, we show that RipU from Rs K60 (RipUK60 ) physically associates with both actin and tubulin and disrupts actin and microtubule cytoskeleton organization. We find that pharmacological disruption of the tomato (Solanum lycopersicum) cytoskeleton promotes Rs K60 colonization. RipUK60 suppresses cell surface-triggered immune responses including flg22-mediated reactive oxygen species (ROS) production and callose deposition. Importantly, tomato plants inoculated with Rs K60 lacking RipU K60 ({Delta}ripUK60) had reduced wilting symptoms and significantly reduced root colonization when compared to plants inoculated with wild-type Rs K60. Collectively, our data suggest that Rs K60 uses the type III effector RipUK60 to remodel cytoskeletal organization, thereby promoting pathogen virulence.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564443">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564443" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564443">
        <p class="paperTitle">Genome-wide evidence of the role of Paf1C in transcription elongation and histone H2B monoubiquitination in Arabidopsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564443" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564443" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Blanco, N.; Perez-Alemany, J.; Bourbousse, C.; latrasse, D.; Ait-Mohamed, O.; Benhamed, M.; Barneche, F.; Blazquez, M. A.; Gallego-Bartolome, J.; Alabadi, D.</p>
        <p class="info">Score: 12.5, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564443' target='https://doi.org/10.1101/2023.10.27.564443'> 10.1101/2023.10.27.564443</a></p>
        <p class="abstract">The evolutionarily conserved Paf1 complex (Paf1C) participates in transcription, and research in animals and fungi suggests that it facilitates RNAPII progression through chromatin remodeling. To obtain evidence that Paf1C acts in transcription elongation in Arabidopsis, we examined the genomic distribution of the ELF7 and VIP3 subunits of Paf1C. The occupancy of both subunits was confined to thousands of gene bodies and positively correlated with RNAPII occupancy and the level of gene expression, supporting a role as a transcription elongation factor. We found that monoubiquitinated histone H2B, which marks most transcribed genes, was strongly reduced genome-wide in elf7 seedlings. Genome-wide profiling of RNAPII revealed that in elf7 mutants, RNAPII occupancy was reduced throughout the gene body and at the transcription end site of Paf1C-targeted genes, suggesting a direct role for the complex in transcription elongation. Overall, our observations suggest that there is a direct functional link between Paf1C activity, mono-ubiquitination of histone H2B, and the transition of RNPII to productive elongation. However, for several genes, Paf1C may also act independently of H2Bub deposition or occupy these genes more stably than H2Bub marking, possibly reflecting the dynamic nature of Paf1C association and H2Bub turnover during transcription.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563711">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563711" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563711">
        <p class="paperTitle">Pan-genome analysis of different morphotypes reveals genomic basis of Brassica oleracea domestication and differential organogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563711" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563711" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guo, N.; Wang, S.; Wang, T.; Duan, M.; Zong, M.; Miao, L.; Han, S.; Wang, G.; Liu, X.; Zhang, D.; Jiao, C.; Xu, H.; Chen, L.; Fei, Z.; Li, J.; Liu, F.</p>
        <p class="info">Score: 11.3, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563711' target='https://doi.org/10.1101/2023.10.24.563711'> 10.1101/2023.10.24.563711</a></p>
        <p class="abstract">The domestication of Brassica oleracea has resulted in diverse morphological types with distinct patterns of organ development. Here we report a graph-based pan-genome of B. oleracea constructed with high-quality genome assemblies of different morphotypes. The pan-genome harbors over 200 structural variant (SV) hotspot regions enriched with auxin and flowering-related genes. Population genomic analyses reveal that early domestication of B. oleracea focused on leaf or stem selection. Gene flows resulting from agricultural practices and variety improvement are detected among different morphotypes. Selective sweep analysis identifies an auxin-responsive SAUR gene and a CLE family gene as the crucial players in the leaf-stem differentiation during the early stage of B. oleracea domestication, and the BoKAN1 gene as instrumental in shaping the leafy heads of cabbage and Brussels sprouts. Our pan-genome and functional analyses further discover that variations in the BoFLC2 gene play key roles in the divergence of vernalization and flowering characteristics among different morphotypes, and variations in the first intron of BoFLC3 are involved in fine-tuning the flowering process in cauliflower. This study provides a comprehensive understanding of the pan-genome of B. oleracea and sheds light on the domestication and differential organ development of this globally important crop species.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
