<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="FROM DUST TO SEED: A LUNAR CHICKPEA STORY
Authors: Atkin, J.; Oliveira Pedro dos Santos, S.
Score: 75.7, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576311
Food sustainability is one of the most significant barriers to long-term space travel. Providing resources from Earth is not cost-efficient, and resupply missions are not viable to meet the needs of long-term life in deep space conditions. Plants in space can provide a source of nutrition and oxygen, reducing the reliance on packaged foods, reducing resupply needs, and extending the duration of missions.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="FROM DUST TO SEED: A LUNAR CHICKPEA STORY
Authors: Atkin, J.; Oliveira Pedro dos Santos, S.
Score: 75.7, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576311
Food sustainability is one of the most significant barriers to long-term space travel. Providing resources from Earth is not cost-efficient, and resupply missions are not viable to meet the needs of long-term life in deep space conditions. Plants in space can provide a source of nutrition and oxygen, reducing the reliance on packaged foods, reducing resupply needs, and extending the duration of missions." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-28T10:37:59+00:00" />
<meta property="article:modified_time" content="2024-01-28T10:37:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="FROM DUST TO SEED: A LUNAR CHICKPEA STORY
Authors: Atkin, J.; Oliveira Pedro dos Santos, S.
Score: 75.7, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576311
Food sustainability is one of the most significant barriers to long-term space travel. Providing resources from Earth is not cost-efficient, and resupply missions are not viable to meet the needs of long-term life in deep space conditions. Plants in space can provide a source of nutrition and oxygen, reducing the reliance on packaged foods, reducing resupply needs, and extending the duration of missions."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "FROM DUST TO SEED: A LUNAR CHICKPEA STORY\nAuthors: Atkin, J.; Oliveira Pedro dos Santos, S.\nScore: 75.7, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576311\nFood sustainability is one of the most significant barriers to long-term space travel. Providing resources from Earth is not cost-efficient, and resupply missions are not viable to meet the needs of long-term life in deep space conditions. Plants in space can provide a source of nutrition and oxygen, reducing the reliance on packaged foods, reducing resupply needs, and extending the duration of missions.",
  "keywords": [
    
  ],
  "articleBody": " FROM DUST TO SEED: A LUNAR CHICKPEA STORY\nAuthors: Atkin, J.; Oliveira Pedro dos Santos, S.\nScore: 75.7, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576311\nFood sustainability is one of the most significant barriers to long-term space travel. Providing resources from Earth is not cost-efficient, and resupply missions are not viable to meet the needs of long-term life in deep space conditions. Plants in space can provide a source of nutrition and oxygen, reducing the reliance on packaged foods, reducing resupply needs, and extending the duration of missions. Using lunar regolith simulant, we employ a novel methodology to create a sustainable and productive growth medium to support the cultivation of horticultural crops on the Moon. Implementing microbial soil regeneration mechanisms derived from Earth, we leverage the interaction between Arbuscular Mycorrhizal Fungi (AMF) and Vermicompost (VC) to create a fertile LRS matrix. These amendments can sequester toxic contaminants, improve soil structure, and increase plant stress tolerance. We demonstrate the ability to produce chickpea (Cicer arietinum) in lunar regolith simulant augmented with AMF and VC under climate-controlled conditions. We cultivated chickpea to seed in a mixture containing 75% Lunar Regolith Simulant. Preliminary results suggest that higher LRS contents induce heightened stress responses. However, plants grown in 100% LRS inoculated with arbuscular mycorrhizal fungi demonstrated an average two-week survival extension compared to non-inoculated plants. This study provides, for the first time, a baseline for chickpea germination in varying mixtures of LRS and VC and will inform future studies as humanity goes back to the Moon.\nBioengineering a plant NLR immune receptor with a robust binding interface towards a conserved fungal pathogen effector\nAuthors: Zdrzalek, R.; Xi, Y.; Langner, T.; Bentham, A. R.; Petit-Houdenot, Y.; De la Concepcion, J. C.; Harant, A.; Shimizu, M.; Were, V.; Talbot, N. J.; Terauchi, R. J.; Kamoun, S.; Banfield, M. J.\nScore: 43.3, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576400\nBioengineering of plant immune receptors has emerged as a key strategy for generating novel disease resistance traits to counteract the expanding threat of plant pathogens to global food security. However, current approaches are limited by rapid evolution of plant pathogens in the field and may lack durability when deployed. Here, we show that the rice nucleotide-binding, leucine-rich repeat (NLR) immune receptor Pik-1 can be engineered to respond to a conserved family of effectors from the multihost blast fungus pathogen Magnaporthe oryzae. We switched the effector binding and response profile of the Pik NLR from its cognate rice blast effector AVR-Pik to the host-determining factor Pwl2 by installing a putative host target, OsHIPP43, in place of the native integrated HMA domain (generating Pikm-1OsHIPP43). This chimeric receptor also responded to other PWL alleles from diverse blast isolates. The crystal structure of the Pwl2/OsHIPP43 complex revealed a multifaceted, robust interface that cannot be easily disrupted by mutagenesis, and may therefore provide durable, broad resistance to blast isolates carrying PWL effectors in the field. Our findings highlight how the host targets of pathogen effectors can be used to bioengineer new recognition specificities that have more robust properties compared to naturally evolved disease resistance genes.\nSpeedy benthi: Elongated photoperiods reduce the generation time of the model plant species Nicotiana benthamiana\nAuthors: Castle, M.; Luedke, D.\nScore: 27.9, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576090\nNicotiana benthamiana is increasingly gaining prominence as a model plant species with recently published high-quality genome assemblies, which will further enable forward and reverse genetic approaches (Bally et al., 2018; Derevnina et al., 2019; Kourelis et al., 2019; Ranawaka et al., 2023; Vollheyde et al., 2023). However, the generation time of N. benthamiana poses a bottleneck in the creation of mutant and transgenic plant lines. Speed breeding (SB), by extended photoperiods and adjustments to growth parameters, is an efficient way to reduce generation times for many crop and model plant species (Ghosh et al., 2018; Watson et al., 2018; Hickey et al., 2019; Varshney et al., 2021). We hypothesized that an extended photoperiod could reduce the seed to seed generation time of N. benthamiana. We tested this hypothesis by comparing generation times under SB conditions to traditionally used photoperiods in growth chambers and green house settings. We found that a 22h photoperiod reduced the generation time of N. benthamiana by approximately 2 weeks (16-22%). Fertilization in combination with a far-red light spectrum did not yield a further reduction in generation time when combined with SB conditions. Our findings further contribute to the establishment of N. benthamiana as an important model organism for plant research.\nThe blast effector Pwl2 is a virulence factor that modifies the cellular localisation of host protein HIPP43 to suppress immunity.\nAuthors: Were, V.; Yan, X.; Foster, A.; Sklenar, J.; Langner, T.; Bentham, A.; Zdrzałek, R.; Ryder, L.; Kaimenyi, D.; Gomez De La Cruz, D.; Gentle, A.; Petit-Houdenot, Y.; Eseola, A. B.; Smoker, M.; Bautista, M. J.; Ma, W.; Kourelis, J.; Maclean, D.; Banfield, M. J.; Kamoun, S.; Menke, F. L. H.; Moscou, M. J.; Talbot, N. J.\nScore: 25.8, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576406\nThe rice blast fungus Magnaporthe oryzae secretes a battery of effector proteins during host infection to suppress plant immunity and promote disease. Among these effectors, the MAX effector-Pwl2 was first identified as a host specificity determinant for infection of weeping lovegrass (Eragrostis curvula). However, its biological activity has remained unknown. Here we show that the PWL2 gene is ubiquitous in all host-limited forms of M. oryzae and has undergone substantial copy number expansion indicating that it is a core effector of the blast fungus. We used CRISPR/Cas9-mediated gene editing to delete all three copies of PWL2 and generate a pwl2 null mutant in M. oryzae strain Guy11. This resulted in gain-of-virulence towards weeping lovegrass, but a reduction in the severity of disease symptoms on rice. To further investigate the virulence mechanism of Pwl2, we showed that transgenic rice and barley lines constitutively expressing PWL2 display suppression of reactive oxygen species generation and increased susceptibility to infection by M. oryzae. Also, we identify the barley and rice heavy metal-binding isoprenylated protein HIPP43 as a target of the Pwl2 effector. Transgenic lines overexpressing HIPP43 exhibit attenuated defense responses and increased susceptibility to blast infection, consistent with the hypothesis that Pwl2 binds HIPP43 to modulate immunity. This binding is coupled to changes in cellular localisation of these proteins. Taken together, our results provide evidence that Pwl2 is a virulence factor in M. oryzae that acts by suppressing host immunity through binding and re-localising HIPP43.\nM2WISH: an easy and efficient protocol for whole-mount mRNA in situ hybridization that allows 3D cell resolution of gene expression in Arabidopsis thaliana.\nAuthors: Chelysheva, L.; Palauqui, J.-C.; Morin, H.; Biot, E.; Nicolas, A.; Rech, P.; da Costa, M.; Barel, L.; Laufs, P.\nScore: 25.0, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576197\nGene expression analysis is essential for understanding the mechanisms involved in plant development. Here, we developed M2WISH, a protocol based on MicroWave treatment for Wholemount mRNA In Situ Hybridization in Arabidopsis. By permeabilizing tissues without damaging cellular organisation this protocol results in high and homogeneous hybridization yields that enables systematic analysis of gene expression dynamics. Moreover, when combined with cellular histochemical staining, M2WISH provides a cellular resolution of gene expression on roots, aerial meristems, leaves and embryos in the seed. We applied M2WISH to study the spatial dynamics of WUSCHEL (WUS) and CLAVATA3 (CLV3) expression during in vitro meristematic conversion of roots into shoot apical meristems. Thus, we showed that shoot apical meristems could arise from two different types of root structures that differed by their CLV3 gene expression patterns. We constructed 3D cellular representations of WUS and CLV3 gene co-expression pattern, and stressed the variability inherent to meristem conversion. Thus, this protocol generates a large amount of data on the localization of gene expression, which can be used to model complex systems.\nThe blast pathogen effector AVR-Pik binds and stabilizes rice heavy metal-associated (HMA) proteins to co-opt their function in immunity\nAuthors: Oikawa, K.; Fujisaki, K.; Shimizu, M.; Takeda, T.; Nemoto, K.; Saitoh, H.; Hirabuchi, A.; Hiraka, Y.; Miyaji, N.; Białas, A.; Langner, T.; Kellner, R.; Bozkurt, T. O.; Cesari, S.; Kroj, T.; Banfield, M. J.; Kamoun, S.; Terauchi, R.\nScore: 21.4, Published: 2024-01-25 DOI: 10.1101/2020.12.01.406389\nIntracellular nucleotide-binding domain and leucine-rich repeat-containing (NLR) receptors play crucial roles in immunity across multiple domains of life. In plants, a subset of NLRs contain noncanonical integrated domains that are thought to have evolved from host targets of pathogen effectors to serve as pathogen baits. However, the functions of host proteins with similarity to NLR integrated domains and the extent to which they are targeted by pathogen effectors remain largely unknown. Here, we show that the blast fungus effector AVR-Pik binds a subset of related rice proteins containing a heavy metal-associated (HMA) domain, one of the domains that has repeatedly integrated into plant NLR immune receptors. We find that AVR-Pik binding stabilizes the rice small HMA (sHMA) proteins OsHIPP19 and OsHIPP20. Knockout of OsHIPP20 causes enhanced disease resistance towards the blast pathogen, indicating that OsHIPP20 is a susceptibility gene (S-gene). We propose that AVR-Pik has evolved to bind HMA domain proteins and co-opt their function to suppress immunity. Yet this binding carries a trade-off, it triggers immunity in plants carrying NLR receptors with integrated HMA domains.\nGenome-informed trophic classification and functional characterization of virulence proteins from the maize tar spot pathogen Phyllachora maydis\nAuthors: Rogers, A.; Jaiswal, N.; Roggenkamp, E.; Kim, H.-S.; MaCready, J.; Chilvers, M.; Scofield, S.; Iyer-Pascuzzi, A. S.; Helm, M.\nScore: 9.8, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576543\nPhyllachora maydis is an ascomycete foliar fungal pathogen and the causal agent of tar spot in maize. Though P. maydis is considered one of the most economically important foliar pathogens of maize, our general knowledge of the trophic lifestyle and functional role of effector proteins from this fungal pathogen remains limited. Here, we utilized a genome-informed approach to predict the trophic lifestyle of P. maydis and functionally characterized a subset of candidate effectors from this fungal pathogen. Leveraging the most recent P. maydis genome annotation and the CATAStrophy pipeline, we show this fungal pathogen encodes a predicted Carbohydrate-active enzymes (CAZymes) repertoire consistent with that of biotrophs (monomertrophs). To investigate fungal pathogenicity, we selected eighteen candidate effector proteins that were previously shown to be expressed during primary disease development. We assessed whether these putative effectors share predicted structural similarity with other characterized fungal effectors and determined whether any suppress plant immune responses. Using AlphaFold2 and Foldseek, we showed one candidate effector, PM02_g1115, adopts a predicted protein structure similar to that of an effector from Verticillium dahlia. Furthermore, transient expression of candidate effector-fluorescent protein fusions in Nicotiana benthamiana revealed that most effector proteins localize to both the nucleus and the cytosol. Importantly, three candidate effectors consistently attenuated chitin-mediated reactive oxygen species production in N. benthamiana. Collectively, these results presented herein provide valuable insights into the predicted trophic lifestyle and putative functions of effectors from P. maydis and will likely stimulate continued research to elucidate the molecular mechanisms used by P. maydis to induce tar spot.\nThe endophytic fungus Serendipita indica alters auxin distribution in Arabidopsis thaliana roots through alteration of auxin transport and conjugation to promote plant growth\nAuthors: Gonzalez Ortega-Villaizan, A.; King, E.; Patel, M. K.; PEREZ ALONSO, M. M.; Scholz, S. S.; Sakakibara, H.; Kiba, T.; Kojima, M.; Takebayashi, Y.; Ramos, P.; Morales-Quintana, L.; Breitenbach, S.; Smolko, A.; Salopek-Sondi, B.; Bauer, N.; Müller-Ludwig, J.; Krapp, A.; Oelmüller, R.; Vicente-Carbajosa, J.; Pollmann, S.\nScore: 6.6, Published: 2024-01-26 DOI: 10.1101/2024.01.23.576781\nPlants share their habitats with a multitude of different microbes. This close vicinity promoted the evolution of inter-organismic interactions between plants and many different microorganisms that provide mutual growth benefits both to the plant and the microbial partner. The symbiosis of Arabidopsis thaliana with the beneficial root colonizing endophyte Serendipita indica represents a well-studied system. Co-colonization of Arabidopsis roots with S. indica significantly promotes plant growth. Due to the notable phenotypic alterations of fungus-infected root systems, the involvement of a reprogramming of plant hormone levels, especially that of indole-3-acetic acid, has been suggested earlier. However, until now, the molecular mechanism by which S. indica promotes plant growth remains largely unknown. This study used comprehensive transcriptomics, metabolomics, reverse genetics, and life cell imaging to reveal the intricacies of auxin-related processes that affect root growth in the symbiosis between A. thaliana and S. indica. Our experiments revealed the essential role of tightly controlled auxin conjugation in the plant-fungus interaction. It particularly highlighted the importance of two GRETCHEN HAGEN 3 (GH3) genes, GH3.5 and GH3.17, for the fungus infection-triggered stimulation of biomass production, thus broadening our knowledge about the function of GH3s in plants. Furthermore, we provide evidence for the transcriptional alteration of the PIN2 auxin transporter gene in roots of Arabidopsis seedlings infected with S. indica and demonstrate that this transcriptional adjustment affects auxin signaling in roots, which results in increased plant growth.\nThe molecular core of transcriptome responses to abiotic stress in plants: a machine learning-driven meta-analysis\nAuthors: Sanchez-Munoz, R.; Depaepe, T.; Samalova, M.; Hejatko, J.; Zaplana, I.; Van Der Straeten, D.\nScore: 5.7, Published: 2024-01-26 DOI: 10.1101/2024.01.24.576978\nUnderstanding how plants adapt their physiology to overcome severe stress conditions is vital in light of the current climate crisis. This remains a challenge given the complex nature of the underlying molecular mechanisms. To provide a full picture of stress mitigation mechanisms, an exhaustive analysis of publicly available stress-related transcriptomic data was conducted. We combined a meta-analysis with an unsupervised machine learning algorithm to identify a core of stress-related genes. To ensure robustness and biological significance of the output, often lacking in meta-analyses, a three-layered biovalidation was incorporated. Our results present a stress gene core, a set of key genes involved in plant tolerance to a multitude of adverse environmental conditions rather than specific ones. In addition, we provide a biologically validated database to assist in design of multi-stress resilience. Taken together, our results pave the way towards future-proof sustainable agriculture.\nThe genome of Lolium multiflorum reveals the genetic architecture of paraquat resistance\nAuthors: Brunharo, C. A.; Short, A. W.; Bobadilla, L. K.; Streisfeld, M. A.\nScore: 9.4, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573904\n- Herbicide resistance in agricultural weeds has become one of the greatest challenges for sustainable crop production. The repeated evolution of herbicide resistance provides an excellent opportunity to study the genetic and physiological basis of the resistance phenotype and the evolutionary responses to human-mediated selection pressures. Lolium multiflorum is a ubiquitous weed that has evolved herbicide resistance repeatedly around the world in various cropping systems. - We assembled and annotated a chromosome-scale genome for L. multiflorum and elucidated the genetic architecture of paraquat resistance by performing quantitative trait loci analysis, genome-wide association studies, genetic divergence analysis, and transcriptome analyses from paraquat-resistant and -susceptible L. multiflorum populations. - Results suggested that two regions of chromosome 5 were associated with paraquat resistance. The regions contain candidate genes that encode cellular transport functions, including a novel multidrug and toxin extrusion (MATE) protein, and a cation transporter previously shown to interact with polyamines. - Our results reveal the genetic architecture of paraquat resistance and identified promising candidate genes for future functional studies. Given that L. multiflorum is a weed and a cultivated crop species, the genomic resources generated will prove valuable to a wide spectrum of the plant science community.\n",
  "wordCount" : "2528",
  "inLanguage": "en",
  "datePublished": "2024-01-28T10:37:59Z",
  "dateModified": "2024-01-28T10:37:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 28, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576311">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576311" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576311">
        <p class="paperTitle">FROM DUST TO SEED: A LUNAR CHICKPEA STORY</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576311" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576311" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Atkin, J.; Oliveira Pedro dos Santos, S.</p>
        <p class="info">Score: 75.7, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576311' target='https://doi.org/10.1101/2024.01.18.576311'> 10.1101/2024.01.18.576311</a></p>
        <p class="abstract">Food sustainability is one of the most significant barriers to long-term space travel. Providing resources from Earth is not cost-efficient, and resupply missions are not viable to meet the needs of long-term life in deep space conditions. Plants in space can provide a source of nutrition and oxygen, reducing the reliance on packaged foods, reducing resupply needs, and extending the duration of missions. Using lunar regolith simulant, we employ a novel methodology to create a sustainable and productive growth medium to support the cultivation of horticultural crops on the Moon. Implementing microbial soil regeneration mechanisms derived from Earth, we leverage the interaction between Arbuscular Mycorrhizal Fungi (AMF) and Vermicompost (VC) to create a fertile LRS matrix. These amendments can sequester toxic contaminants, improve soil structure, and increase plant stress tolerance. We demonstrate the ability to produce chickpea (Cicer arietinum) in lunar regolith simulant augmented with AMF and VC under climate-controlled conditions. We cultivated chickpea to seed in a mixture containing 75% Lunar Regolith Simulant. Preliminary results suggest that higher LRS contents induce heightened stress responses. However, plants grown in 100% LRS inoculated with arbuscular mycorrhizal fungi demonstrated an average two-week survival extension compared to non-inoculated plants. This study provides, for the first time, a baseline for chickpea germination in varying mixtures of LRS and VC and will inform future studies as humanity goes back to the Moon.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576400">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576400" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576400">
        <p class="paperTitle">Bioengineering a plant NLR immune receptor with a robust binding interface towards a conserved fungal pathogen effector</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576400" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576400" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zdrzalek, R.; Xi, Y.; Langner, T.; Bentham, A. R.; Petit-Houdenot, Y.; De la Concepcion, J. C.; Harant, A.; Shimizu, M.; Were, V.; Talbot, N. J.; Terauchi, R. J.; Kamoun, S.; Banfield, M. J.</p>
        <p class="info">Score: 43.3, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576400' target='https://doi.org/10.1101/2024.01.20.576400'> 10.1101/2024.01.20.576400</a></p>
        <p class="abstract">Bioengineering of plant immune receptors has emerged as a key strategy for generating novel disease resistance traits to counteract the expanding threat of plant pathogens to global food security. However, current approaches are limited by rapid evolution of plant pathogens in the field and may lack durability when deployed. Here, we show that the rice nucleotide-binding, leucine-rich repeat (NLR) immune receptor Pik-1 can be engineered to respond to a conserved family of effectors from the multihost blast fungus pathogen Magnaporthe oryzae. We switched the effector binding and response profile of the Pik NLR from its cognate rice blast effector AVR-Pik to the host-determining factor Pwl2 by installing a putative host target, OsHIPP43, in place of the native integrated HMA domain (generating Pikm-1OsHIPP43). This chimeric receptor also responded to other PWL alleles from diverse blast isolates. The crystal structure of the Pwl2/OsHIPP43 complex revealed a multifaceted, robust interface that cannot be easily disrupted by mutagenesis, and may therefore provide durable, broad resistance to blast isolates carrying PWL effectors in the field. Our findings highlight how the host targets of pathogen effectors can be used to bioengineer new recognition specificities that have more robust properties compared to naturally evolved disease resistance genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576090">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576090" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576090">
        <p class="paperTitle">Speedy benthi: Elongated photoperiods reduce the generation time of the model plant species Nicotiana benthamiana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576090" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576090" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Castle, M.; Luedke, D.</p>
        <p class="info">Score: 27.9, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576090' target='https://doi.org/10.1101/2024.01.18.576090'> 10.1101/2024.01.18.576090</a></p>
        <p class="abstract">Nicotiana benthamiana is increasingly gaining prominence as a model plant species with recently published high-quality genome assemblies, which will further enable forward and reverse genetic approaches (Bally et al., 2018; Derevnina et al., 2019; Kourelis et al., 2019; Ranawaka et al., 2023; Vollheyde et al., 2023). However, the generation time of N. benthamiana poses a bottleneck in the creation of mutant and transgenic plant lines. Speed breeding (SB), by extended photoperiods and adjustments to growth parameters, is an efficient way to reduce generation times for many crop and model plant species (Ghosh et al., 2018; Watson et al., 2018; Hickey et al., 2019; Varshney et al., 2021). We hypothesized that an extended photoperiod could reduce the seed to seed generation time of N. benthamiana. We tested this hypothesis by comparing generation times under SB conditions to traditionally used photoperiods in growth chambers and green house settings. We found that a 22h photoperiod reduced the generation time of N. benthamiana by approximately 2 weeks (16-22%). Fertilization in combination with a far-red light spectrum did not yield a further reduction in generation time when combined with SB conditions. Our findings further contribute to the establishment of N. benthamiana as an important model organism for plant research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576406">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576406" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576406">
        <p class="paperTitle">The blast effector Pwl2 is a virulence factor that modifies the cellular localisation of host protein HIPP43 to suppress immunity.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576406" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576406" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Were, V.; Yan, X.; Foster, A.; Sklenar, J.; Langner, T.; Bentham, A.; Zdrzałek, R.; Ryder, L.; Kaimenyi, D.; Gomez De La Cruz, D.; Gentle, A.; Petit-Houdenot, Y.; Eseola, A. B.; Smoker, M.; Bautista, M. J.; Ma, W.; Kourelis, J.; Maclean, D.; Banfield, M. J.; Kamoun, S.; Menke, F. L. H.; Moscou, M. J.; Talbot, N. J.</p>
        <p class="info">Score: 25.8, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576406' target='https://doi.org/10.1101/2024.01.20.576406'> 10.1101/2024.01.20.576406</a></p>
        <p class="abstract">The rice blast fungus Magnaporthe oryzae secretes a battery of effector proteins during host infection to suppress plant immunity and promote disease. Among these effectors, the MAX effector-Pwl2 was first identified as a host specificity determinant for infection of weeping lovegrass (Eragrostis curvula). However, its biological activity has remained unknown. Here we show that the PWL2 gene is ubiquitous in all host-limited forms of M. oryzae and has undergone substantial copy number expansion indicating that it is a core effector of the blast fungus. We used CRISPR/Cas9-mediated gene editing to delete all three copies of PWL2 and generate a pwl2 null mutant in M. oryzae strain Guy11. This resulted in gain-of-virulence towards weeping lovegrass, but a reduction in the severity of disease symptoms on rice. To further investigate the virulence mechanism of Pwl2, we showed that transgenic rice and barley lines constitutively expressing PWL2 display suppression of reactive oxygen species generation and increased susceptibility to infection by M. oryzae. Also, we identify the barley and rice heavy metal-binding isoprenylated protein HIPP43 as a target of the Pwl2 effector. Transgenic lines overexpressing HIPP43 exhibit attenuated defense responses and increased susceptibility to blast infection, consistent with the hypothesis that Pwl2 binds HIPP43 to modulate immunity. This binding is coupled to changes in cellular localisation of these proteins. Taken together, our results provide evidence that Pwl2 is a virulence factor in M. oryzae that acts by suppressing host immunity through binding and re-localising HIPP43.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576197">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576197" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576197">
        <p class="paperTitle">M2WISH: an easy and efficient protocol for whole-mount mRNA in situ hybridization that allows 3D cell resolution of gene expression in Arabidopsis thaliana.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576197" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576197" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chelysheva, L.; Palauqui, J.-C.; Morin, H.; Biot, E.; Nicolas, A.; Rech, P.; da Costa, M.; Barel, L.; Laufs, P.</p>
        <p class="info">Score: 25.0, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576197' target='https://doi.org/10.1101/2024.01.18.576197'> 10.1101/2024.01.18.576197</a></p>
        <p class="abstract">Gene expression analysis is essential for understanding the mechanisms involved in plant development. Here, we developed M2WISH, a protocol based on MicroWave treatment for Wholemount mRNA In Situ Hybridization in Arabidopsis. By permeabilizing tissues without damaging cellular organisation this protocol results in high and homogeneous hybridization yields that enables systematic analysis of gene expression dynamics. Moreover, when combined with cellular histochemical staining, M2WISH provides a cellular resolution of gene expression on roots, aerial meristems, leaves and embryos in the seed. We applied M2WISH to study the spatial dynamics of WUSCHEL (WUS) and CLAVATA3 (CLV3) expression during in vitro meristematic conversion of roots into shoot apical meristems. Thus, we showed that shoot apical meristems could arise from two different types of root structures that differed by their CLV3 gene expression patterns. We constructed 3D cellular representations of WUS and CLV3 gene co-expression pattern, and stressed the variability inherent to meristem conversion. Thus, this protocol generates a large amount of data on the localization of gene expression, which can be used to model complex systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2020.12.01.406389">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2020.12.01.406389" aria-expanded="false" aria-controls="flush-collapse10.1101/2020.12.01.406389">
        <p class="paperTitle">The blast pathogen effector AVR-Pik binds and stabilizes rice heavy metal-associated (HMA) proteins to co-opt their function in immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2020.12.01.406389" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2020.12.01.406389" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oikawa, K.; Fujisaki, K.; Shimizu, M.; Takeda, T.; Nemoto, K.; Saitoh, H.; Hirabuchi, A.; Hiraka, Y.; Miyaji, N.; Białas, A.; Langner, T.; Kellner, R.; Bozkurt, T. O.; Cesari, S.; Kroj, T.; Banfield, M. J.; Kamoun, S.; Terauchi, R.</p>
        <p class="info">Score: 21.4, Published: 2024-01-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2020.12.01.406389' target='https://doi.org/10.1101/2020.12.01.406389'> 10.1101/2020.12.01.406389</a></p>
        <p class="abstract">Intracellular nucleotide-binding domain and leucine-rich repeat-containing (NLR) receptors play crucial roles in immunity across multiple domains of life. In plants, a subset of NLRs contain noncanonical integrated domains that are thought to have evolved from host targets of pathogen effectors to serve as pathogen baits. However, the functions of host proteins with similarity to NLR integrated domains and the extent to which they are targeted by pathogen effectors remain largely unknown. Here, we show that the blast fungus effector AVR-Pik binds a subset of related rice proteins containing a heavy metal-associated (HMA) domain, one of the domains that has repeatedly integrated into plant NLR immune receptors. We find that AVR-Pik binding stabilizes the rice small HMA (sHMA) proteins OsHIPP19 and OsHIPP20. Knockout of OsHIPP20 causes enhanced disease resistance towards the blast pathogen, indicating that OsHIPP20 is a susceptibility gene (S-gene). We propose that AVR-Pik has evolved to bind HMA domain proteins and co-opt their function to suppress immunity. Yet this binding carries a trade-off, it triggers immunity in plants carrying NLR receptors with integrated HMA domains.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576543">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576543" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576543">
        <p class="paperTitle">Genome-informed trophic classification and functional characterization of virulence proteins from the maize tar spot pathogen Phyllachora maydis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576543" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576543" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rogers, A.; Jaiswal, N.; Roggenkamp, E.; Kim, H.-S.; MaCready, J.; Chilvers, M.; Scofield, S.; Iyer-Pascuzzi, A. S.; Helm, M.</p>
        <p class="info">Score: 9.8, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576543' target='https://doi.org/10.1101/2024.01.22.576543'> 10.1101/2024.01.22.576543</a></p>
        <p class="abstract">Phyllachora maydis is an ascomycete foliar fungal pathogen and the causal agent of tar spot in maize. Though P. maydis is considered one of the most economically important foliar pathogens of maize, our general knowledge of the trophic lifestyle and functional role of effector proteins from this fungal pathogen remains limited. Here, we utilized a genome-informed approach to predict the trophic lifestyle of P. maydis and functionally characterized a subset of candidate effectors from this fungal pathogen. Leveraging the most recent P. maydis genome annotation and the CATAStrophy pipeline, we show this fungal pathogen encodes a predicted Carbohydrate-active enzymes (CAZymes) repertoire consistent with that of biotrophs (monomertrophs). To investigate fungal pathogenicity, we selected eighteen candidate effector proteins that were previously shown to be expressed during primary disease development. We assessed whether these putative effectors share predicted structural similarity with other characterized fungal effectors and determined whether any suppress plant immune responses. Using AlphaFold2 and Foldseek, we showed one candidate effector, PM02_g1115, adopts a predicted protein structure similar to that of an effector from Verticillium dahlia. Furthermore, transient expression of candidate effector-fluorescent protein fusions in Nicotiana benthamiana revealed that most effector proteins localize to both the nucleus and the cytosol. Importantly, three candidate effectors consistently attenuated chitin-mediated reactive oxygen species production in N. benthamiana. Collectively, these results presented herein provide valuable insights into the predicted trophic lifestyle and putative functions of effectors from P. maydis and will likely stimulate continued research to elucidate the molecular mechanisms used by P. maydis to induce tar spot.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576781">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576781" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576781">
        <p class="paperTitle">The endophytic fungus Serendipita indica alters auxin distribution in Arabidopsis thaliana roots through alteration of auxin transport and conjugation to promote plant growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576781" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576781" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gonzalez Ortega-Villaizan, A.; King, E.; Patel, M. K.; PEREZ ALONSO, M. M.; Scholz, S. S.; Sakakibara, H.; Kiba, T.; Kojima, M.; Takebayashi, Y.; Ramos, P.; Morales-Quintana, L.; Breitenbach, S.; Smolko, A.; Salopek-Sondi, B.; Bauer, N.; Müller-Ludwig, J.; Krapp, A.; Oelmüller, R.; Vicente-Carbajosa, J.; Pollmann, S.</p>
        <p class="info">Score: 6.6, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576781' target='https://doi.org/10.1101/2024.01.23.576781'> 10.1101/2024.01.23.576781</a></p>
        <p class="abstract">Plants share their habitats with a multitude of different microbes. This close vicinity promoted the evolution of inter-organismic interactions between plants and many different microorganisms that provide mutual growth benefits both to the plant and the microbial partner. The symbiosis of Arabidopsis thaliana with the beneficial root colonizing endophyte Serendipita indica represents a well-studied system. Co-colonization of Arabidopsis roots with S. indica significantly promotes plant growth. Due to the notable phenotypic alterations of fungus-infected root systems, the involvement of a reprogramming of plant hormone levels, especially that of indole-3-acetic acid, has been suggested earlier. However, until now, the molecular mechanism by which S. indica promotes plant growth remains largely unknown. This study used comprehensive transcriptomics, metabolomics, reverse genetics, and life cell imaging to reveal the intricacies of auxin-related processes that affect root growth in the symbiosis between A. thaliana and S. indica. Our experiments revealed the essential role of tightly controlled auxin conjugation in the plant-fungus interaction. It particularly highlighted the importance of two GRETCHEN HAGEN 3 (GH3) genes, GH3.5 and GH3.17, for the fungus infection-triggered stimulation of biomass production, thus broadening our knowledge about the function of GH3s in plants. Furthermore, we provide evidence for the transcriptional alteration of the PIN2 auxin transporter gene in roots of Arabidopsis seedlings infected with S. indica and demonstrate that this transcriptional adjustment affects auxin signaling in roots, which results in increased plant growth.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.576978">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.576978" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.576978">
        <p class="paperTitle">The molecular core of transcriptome responses to abiotic stress in plants: a machine learning-driven meta-analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.576978" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.576978" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanchez-Munoz, R.; Depaepe, T.; Samalova, M.; Hejatko, J.; Zaplana, I.; Van Der Straeten, D.</p>
        <p class="info">Score: 5.7, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.576978' target='https://doi.org/10.1101/2024.01.24.576978'> 10.1101/2024.01.24.576978</a></p>
        <p class="abstract">Understanding how plants adapt their physiology to overcome severe stress conditions is vital in light of the current climate crisis. This remains a challenge given the complex nature of the underlying molecular mechanisms. To provide a full picture of stress mitigation mechanisms, an exhaustive analysis of publicly available stress-related transcriptomic data was conducted. We combined a meta-analysis with an unsupervised machine learning algorithm to identify a core of stress-related genes. To ensure robustness and biological significance of the output, often lacking in meta-analyses, a three-layered biovalidation was incorporated. Our results present a stress gene core, a set of key genes involved in plant tolerance to a multitude of adverse environmental conditions rather than specific ones. In addition, we provide a biologically validated database to assist in design of multi-stress resilience. Taken together, our results pave the way towards future-proof sustainable agriculture.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573904">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573904" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573904">
        <p class="paperTitle">The genome of Lolium multiflorum reveals the genetic architecture of paraquat resistance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573904" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573904" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brunharo, C. A.; Short, A. W.; Bobadilla, L. K.; Streisfeld, M. A.</p>
        <p class="info">Score: 9.4, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573904' target='https://doi.org/10.1101/2024.01.02.573904'> 10.1101/2024.01.02.573904</a></p>
        <p class="abstract">- Herbicide resistance in agricultural weeds has become one of the greatest challenges for sustainable crop production. The repeated evolution of herbicide resistance provides an excellent opportunity to study the genetic and physiological basis of the resistance phenotype and the evolutionary responses to human-mediated selection pressures. Lolium multiflorum is a ubiquitous weed that has evolved herbicide resistance repeatedly around the world in various cropping systems.
- We assembled and annotated a chromosome-scale genome for L. multiflorum and elucidated the genetic architecture of paraquat resistance by performing quantitative trait loci analysis, genome-wide association studies, genetic divergence analysis, and transcriptome analyses from paraquat-resistant and -susceptible L. multiflorum populations.
- Results suggested that two regions of chromosome 5 were associated with paraquat resistance. The regions contain candidate genes that encode cellular transport functions, including a novel multidrug and toxin extrusion (MATE) protein, and a cation transporter previously shown to interact with polyamines.
- Our results reveal the genetic architecture of paraquat resistance and identified promising candidate genes for future functional studies. Given that L. multiflorum is a weed and a cultivated crop species, the genomic resources generated will prove valuable to a wide spectrum of the plant science community.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
