<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A Single-Nucleus Atlas of Seed-to-Seed Development in Arabidopsis
Authors: Lee, T. A.; Nobori, T.; Illouz-Eliaz, N.; Xu, J.; Jow, B.; Nery, J. R.; Ecker, J. R.
Score: 193.5, Published: 2023-09-02 DOI: 10.1101/2023.03.23.533992
Extensive studies of the reference plant Arabidopsis have enabled a deep understanding of tissues throughout development, yet a census of cell types and states throughout development is lacking. Here, we present a single-nucleus transcriptome atlas of seed-to-seed development employing over 800,000 nuclei, encompassing a diverse set of tissues across ten developmental stages, with spatial transcriptomic validation of the dynamic seed and silique.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="A Single-Nucleus Atlas of Seed-to-Seed Development in Arabidopsis
Authors: Lee, T. A.; Nobori, T.; Illouz-Eliaz, N.; Xu, J.; Jow, B.; Nery, J. R.; Ecker, J. R.
Score: 193.5, Published: 2023-09-02 DOI: 10.1101/2023.03.23.533992
Extensive studies of the reference plant Arabidopsis have enabled a deep understanding of tissues throughout development, yet a census of cell types and states throughout development is lacking. Here, we present a single-nucleus transcriptome atlas of seed-to-seed development employing over 800,000 nuclei, encompassing a diverse set of tissues across ten developmental stages, with spatial transcriptomic validation of the dynamic seed and silique." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-10T10:36:49+00:00" />
<meta property="article:modified_time" content="2023-09-10T10:36:49+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="A Single-Nucleus Atlas of Seed-to-Seed Development in Arabidopsis
Authors: Lee, T. A.; Nobori, T.; Illouz-Eliaz, N.; Xu, J.; Jow, B.; Nery, J. R.; Ecker, J. R.
Score: 193.5, Published: 2023-09-02 DOI: 10.1101/2023.03.23.533992
Extensive studies of the reference plant Arabidopsis have enabled a deep understanding of tissues throughout development, yet a census of cell types and states throughout development is lacking. Here, we present a single-nucleus transcriptome atlas of seed-to-seed development employing over 800,000 nuclei, encompassing a diverse set of tissues across ten developmental stages, with spatial transcriptomic validation of the dynamic seed and silique."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "A Single-Nucleus Atlas of Seed-to-Seed Development in Arabidopsis\nAuthors: Lee, T. A.; Nobori, T.; Illouz-Eliaz, N.; Xu, J.; Jow, B.; Nery, J. R.; Ecker, J. R.\nScore: 193.5, Published: 2023-09-02 DOI: 10.1101/2023.03.23.533992\nExtensive studies of the reference plant Arabidopsis have enabled a deep understanding of tissues throughout development, yet a census of cell types and states throughout development is lacking. Here, we present a single-nucleus transcriptome atlas of seed-to-seed development employing over 800,000 nuclei, encompassing a diverse set of tissues across ten developmental stages, with spatial transcriptomic validation of the dynamic seed and silique.",
  "keywords": [
    
  ],
  "articleBody": " A Single-Nucleus Atlas of Seed-to-Seed Development in Arabidopsis\nAuthors: Lee, T. A.; Nobori, T.; Illouz-Eliaz, N.; Xu, J.; Jow, B.; Nery, J. R.; Ecker, J. R.\nScore: 193.5, Published: 2023-09-02 DOI: 10.1101/2023.03.23.533992\nExtensive studies of the reference plant Arabidopsis have enabled a deep understanding of tissues throughout development, yet a census of cell types and states throughout development is lacking. Here, we present a single-nucleus transcriptome atlas of seed-to-seed development employing over 800,000 nuclei, encompassing a diverse set of tissues across ten developmental stages, with spatial transcriptomic validation of the dynamic seed and silique. Cross-organ analyses revealed transcriptional conservation of cell types throughout development and heterogeneity within individual cell types influenced by organ-of-origin and developmental timing, including groups of transcription factors, suggesting gatekeeping by transcription factor activation. This atlas provides a resource for the study of cell type specification throughout the development continuum and a reference for stimulus-response and genetic perturbations at the single-cell resolution.\nRetrotransposon-driven environmental regulation of FLC leads to adaptive response to herbicide\nAuthors: Raingeval, M.; Leduque, B.; Baduel, P.; Edera, A.; Roux, F.; Colot, V.; Quadrana, L.\nScore: 18.3, Published: 2023-09-07 DOI: 10.1101/2023.09.06.556499\nThe mobilization of retrotransposons yields major-effect mutations. Here, we report an adaptive retrotransposon insertion within the first intron of the Arabidopsis floral-repressor locus FLOWERING LOCUS C (FLC). The insertion-mutation augments the environmental sensitivity of FLC by affecting the balance between coding and non-coding transcript isoforms in response to environmental threads. We show that this balance is modulated epigenetically by DNA methylation and orchestrated by IBM2, a factor involved in the processing of intronic heterochromatin. The stress-sensitive allele of FLC has recently spread across populations subjected to recurrent chemical weeding, and we demonstrate that retrotransposon-driven acceleration of life cycle represents a rapid response to herbicide. Our findings illustrate how retrotransposition can create environmentally-sensitive alleles that facilitate adaptation to anthropogenic disturbances of the environment.\nAn atypical endomembrane localized CNL-type immune receptor with a conserved deletion in the N-terminal signaling domain functions in cell death and immunity\nAuthors: Sunil, S.; Beeh, S.; Stoebbe, E.; Fischer, K. M.; Wilhelm, F.; Paris, C.; Meral, A.; Teasdale, L.; Jiang, Z.; Zhang, l.; Aguilar Parras, E.; Nuernberger, T.; Weigel, D.; Lozano-Duran, R.; El Kasmi, F.\nScore: 17.7, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556214\nPlants have evolved intracellular nucleotide-binding leucine rich repeat receptors (NLRs) to induce a superior immune response. Upon activation, coiled-coil (CC) domain containing NLRs (CNLs) oligomerize to form apparent cation channels that promote calcium influx and cell death induction, with the alpha-1 helix of the individual CC domains penetrating membranes. Some members of a monophyletic subclass of CNLs, the ancient and autonomous NLRs (ANLs), are characterized by putative N- myristoylation and S-acylation sites at the N-terminus of their CCG10/GA domain, potentially mediating permanent membrane association. Whether these Potentially Membrane Localized NLRs (PMLs) mediate cell death upon activation in a similar way as reported for other CNLs has been unknown. We integrated phylogenetic, cell- biological, and functional studies to uncover the cell death function of an atypical but conserved Arabidopsis PML, PML5, which has a 113 amino acid deletion in its CCG10/GA domain. Active PML5 oligomers localize in Golgi membranes and the tonoplast, changes vacuolar morphology, and induce cell death, with the short N- terminus being sufficient for cell death. Mutant analysis supports a potential key role of PMLs in plant immunity. Similar deletions as in Arabidopsis PML5 are found in several Brassicales paralogs, pointing to the evolutionary importance of this innovation. PML5 is thus a naturally occurring CNL variant with a minimal signaling domain and its further study should help in understanding the functional importance of this minimal domain for NLR signaling.\nUpstream regulator of genomic imprinting in rice endosperm is a small RNA-associated chromatin remodeler\nAuthors: Pal, A.; Sundar, V. H.; Nambiar, A.; Shivaprasad, P. V.\nScore: 18.1, Published: 2023-09-01 DOI: 10.1101/2023.08.31.555833\nGenomic imprinting is observed in endosperm, a placenta-like seed tissue, where transposable elements (TEs) and repeat-derived small(s)RNAs mediate epigenetic changes in plants. In imprinting, uniparental gene expression arises due to parent-specific epigenetic marks on one allele but not on the other. The importance of sRNAs and their regulation in endosperm development or in imprinting is poorly understood in crops. Here we show that a previously uncharacterized CLASSY (CLSY)-family chromatin remodeler named OsCLSY3 is essential for rice endosperm development and imprinting, acting as an upstream player in sRNA pathway. Comparative transcriptome and genetic analysis indicated its endosperm-preferred expression and its paternally imprinted nature. These important features were modulated by RNA-directed DNA methylation (RdDM) of tandemly arranged TEs in its promoter. Upon perturbation of OsCLSY3 in transgenic lines we observed defects in endosperm development and loss of around 70% of all sRNAs. Interestingly, well-conserved endosperm-specific sRNAs (siren) that are vital for reproductive fitness in angiosperms were dependent on OsCLSY3. We also observed many imprinted genes and seed development-associated genes under the control of CLSY3-dependent RdDM. These results support an essential role of OsCLSY3 in rice endosperm development and imprinting, and propose similar regulatory strategies involving CLSY3 homologs among other cereals. HighlightsO_LIUnlike among dicots, in rice and maize, CLSY3 is a maternally expressed imprinted gene majorly expressed in endosperm. C_LIO_LIEndosperm-preferred expression of OsCLSY3 is regulated by RNA-directed DNA methylation at two tandem transposon elements present in its promoter. C_LIO_LIOsCLSY3 is crucial for endosperm development and grain filling. It regulates expression of key seed development and endosperm-specific imprinted genes through RNA directed DNA methylation. C_LI\nEstablishing a comprehensive web-based analysis platform for Nicotiana benthamiana genome and transcriptome\nAuthors: Kurotani, K.-i.; Hirakawa, H.; Shirasawa, K.; Tagiri, K.; Mori, M.; Ichihashi, Y.; Suzuki, T.; Tanizawa, Y.; Nakamura, Y.; Isobe, S.; Notaguchi, M.\nScore: 15.6, Published: 2023-09-05 DOI: 10.1101/2023.09.03.556139\nNicotiana benthamiana has long served as a crucial plant material extensively used in plant physiology research, particularly in the field of plant pathology, because of its high susceptibility to plant viruses. Additionally, it serves as a production platform to test vaccines and other valuable substances. Among its approximately 3.1 Gb genome, 57,583 genes have been annotated within a 61 Mb region. We created a comprehensive and easy-to-use platform to use transcriptomes for modern annotation. These tools allow to visualize gene expression profiles, draw molecular evolutionary phylogenetic trees of gene families, perform functional enrichment analyses, and facilitate output downloads. To demonstrate their utility, we analyzed the gene expression profiles of enzymes within the nicotine biosynthesis pathway, a secondary metabolic pathway characteristic of the Nicotiana genus. Using the developed tool, expression profiles of the nicotine biosynthesis pathway genes were generated. The expression patterns of eight gene groups in the pathway were strongly expressed in the roots and weakly expressed in leaves and flowers of N. benthamiana. The results were consistent with the established gene expression profiles in Nicotiana tabacum and provided insights into gene family composition and expression trends. The compilation of this database tool can facilitate genetic analysis of N. benthamiana in the future. Significance statementA tool was developed to visualize gene expression profiles, draw molecular evolutionary phylogenetic trees of gene families, perform functional enrichment analyses, and facilitate output downloads of Nicotiana benthamiana. The database developed using the tool can evolve into a comprehensive all-in-one analysis platform by continuously incorporating transcriptome data released to date, newly released RNA-seq data, and annotations in the future.\nA soybean rust effector protease suppresses host immunity and cleaves a 3-deoxy-7-phosphoheptulonate synthase\nAuthors: Chicowski, A. S.; Qi, M.; Variz, H.; Bredow, M.; Montes-Serey, C.; Caiazza, F.; Dong, H.; Margets, A. C.; Mejias, J.; Walley, J.; Craik, C.; Pedley, K. F.; Aung, K.; Innes, R. W.; Whitham, S.\nScore: 11.9, Published: 2023-09-08 DOI: 10.1101/2023.09.07.556260\nThe devastating soybean rust (SBR) pathogen, Phakopsora pachyrhizi, encodes many secreted proteins, but only two have been functionally characterized for their roles in rust virulence. Here, we demonstrate that transient expression of P. pachyrhizi effector candidate 15 (PpEC15), an aspartic protease, leads to enhanced bacterial growth in planta, suppression of callose deposition, reduced expression of plant defense-related marker genes and suppression of pathogen-associated molecular pattern (PAMP)-induced reactive oxygen species (ROS). Stable expression of PpEC15 in soybean suppresses PAMP-induced ROS production and enhances bacterial growth, indicating that, collectively, PpEC15 suppresses host and non-host innate immune responses. Yeast-two-hybrid and proximity labeling identified putative PpEC15 interacting partners including a peptide-chain release factor (PCRF), a NAC83 (NAM, ATAF, and CUC) transcription factor, and a DAHP (3-deoxy-7-phosphoheptulonate) synthase. We further show that PpEC15 is able to cleave DAHP, but does not cleave PCRF or NAC83. Virus-induced gene silencing of NAC83, PCRF and DAHP altered PAMP-induced ROS production and salicylic acid production, indicating that these proteins may be involved in immune signaling. Collectively, our data show that PpEC15 is conserved across P. pachyrhizi isolates and other economically important rust species and is involved in the suppression of plant basal defense responses. Understanding the role of PpEC15 in P. pachyrhizi virulence will provide a foundation for designing targeted intervention strategies to generate rust-resistant crops.\nWater fluxes contribute to growth patterning in shoot meristems\nAuthors: Alonso-Serra, J.; Cheddadi, I.; Kiss, A.; Cerutti, G.; Lionnet, C.; Godin, C.; Hamant, O.\nScore: 35.1, Published: 2023-08-29 DOI: 10.1101/2023.08.27.554993\nIn multicellular organisms, localized tissue outgrowth creates a new water sink thereby modifying hydraulic patterns at the organ level. These fluxes are often considered passive by-products of development and their patterning and potential contribution to morphogenesis remains largely unexplored. Here, we generated a complete map of cell volumetric growth and deformation across the shoot apex in Arabidopsis thaliana. Within the organ-meristem boundary, we found that a subpopulation of cells next to fast-growing cells experiences volumetric shrinkage. To understand this process, we used a vertex-based model integrating mechanics and hydraulics, informed by the measured growth rates. Organ outgrowth simulations revealed the emerging water fluxes and predicted water deficit with volume loss for a few cells at the boundary. Consistently, in planta, a water-soluble dye is preferentially allocated to fast-growing tissues and fails to enter the boundary domain. Analysis of intact meristems further validated our model by revealing cell shrinkage next to fast-growing cells in different contexts of tissue surface curvature and cell deformation. A molecular signature of water deficit at the boundary further confirmed our conclusion. Taken together, we propose a model where the differential sink strength of emerging organs prescribes the hydraulic patterns that define the boundary domain at the shoot apex.\nHeterochromatin re-organization associated with the transcriptional reprogramming under viral infection in Arabidopsis\nAuthors: Annacondia, M. L.; Juarez-Gonzalez, V. T.; Cheng, J.; Reig-Valiente, J. L.; Martinez, G.\nScore: 14.1, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555647\nEpigenetic mechanisms are key regulators of genomic integrity and genic expression. Emerging evidence shows that epigenetic regulation is an important component of the transcriptional reprogramming during stress. Despite this, the overall stress-induced reprogramming of the different epigenetic marks and their targets are unknown. Here, we uncovered multiple epigenetic changes taking place during viral infection in Arabidopsis thaliana and their connection with gene expression. We find that cucumber mosaic virus (CMV) infection induces an overall reorganization of the repressive epigenetic marks H3K9me2, H3K27me3, and DNA methylation, which interact between them and are dynamic during infection. Overall, these epigenetic changes are involved in the reprogramming of the transcriptional program to adapt to the biotic stress, and might ensure genome stability through the transcriptional control of transposable elements (TEs). Mechanistically, we demonstrate that the catalytic component of the Polycomb Repressive Complex 2 (PRC2) CURLY LEAF (CLF) mediates the transcriptional repression of genes gaining H3K27me3 during viral infection and that mutants on that component induce resistance against CMV. Altogether, our results provide a complete picture of the epigenetic changes that occur during biotic stress and exemplify the overall dynamism of epigenetic regulation in eukaryotic organisms.\nCell Fate Programming by Transcription Factors and Epigenetic Machinery in Stomatal Development\nAuthors: Liu, A.; Mair, A.; Matos, J.; Vollbrecht, M.; Xu, S.-L.; Bergmann, D.\nScore: 9.9, Published: 2023-08-24 DOI: 10.1101/2023.08.23.554515\nThe development of multi-cellular organisms requires coordinated changes in gene expression that are often mediated by the interaction between transcription factors (TFs) and their corresponding cis-regulatory elements (CREs). During development and differentiation, the accessibility of CREs is dynamically modulated by the epigenome. How the epigenome, CREs and TFs together exert control over cell fate commitment remains to be fully understood. In the Arabidopsis leaf epidermis, meristemoids undergo a series of stereotyped cell divisions, then switch fate to commit to stomatal differentiation. Newly created or reanalyzed scRNA-seq and ChIP-seq data confirm that stomatal development involves distinctive phases of transcriptional regulation and that differentially regulated genes are bound by the stomatal basic-helix-loop-helix (bHLH) TFs. Targets of the bHLHs often reside in repressive chromatin before activation. MNase-seq evidence further suggests that the repressive state can be overcome and remodeled upon activation by specific stomatal bHLHs. We propose that chromatin remodeling is mediated through the recruitment of a set of physical interactors that we identified through proximity labeling - the ATPase-dependent chromatin remodeling SWI/SNF complex and the histone acetyltransferase HAC1. The bHLHs and chromatin remodelers localize to overlapping genomic regions in a hierarchical order. Furthermore, plants with stage-specific knock-down of the SWI/SNF components or HAC1 fail to activate specific bHLH targets and display stomatal development defects. Together these data converge on a model for how stomatal TFs and epigenetic machinery cooperatively regulate transcription and chromatin remodeling during progressive fate specification.\nHistorical causes for the greater proportion of polyploid plants in higher latitudes\nAuthors: Hagen, E. R.; Vasconcelos, T.; Boyko, J. D.; Beaulieu, J. M.\nScore: 7.7, Published: 2023-09-05 DOI: 10.1101/2023.09.01.555981\nThe proportion of polyploid plants in a community increases with latitude, and different hypotheses have been proposed about which factors drive this pattern. Here, we aim to understand the historical causes of the latitudinal polyploidy gradient using a combination of ancestral state reconstruction methods. Specifically, we assess whether (1) polyploidization enables movement to higher latitudes (i.e., polyploidization precedes occurrences in higher latitudes) or (2) higher latitudes facilitate polyploidization (i.e., occurrence in higher latitudes precedes polyploidization). We reconstruct the ploidy states and ancestral niches of 1,032 angiosperm species at four paleoclimatic time slices ranging from 3.3 million years ago to the present, comprising taxa from four well-represented clades: Onagraceae, Primulaceae, Solanum (Solanaceae), and Pooideae (Poaceae). We use ancestral niche reconstruction models alongside a customized discrete character evolution model to allow reconstruction of states at specific time slices. Patterns of latitudinal movement are reconstructed and compared in relation to inferred ploidy shifts. We find that no single hypothesis applies equally well across all analyzed clades. While significant differences in median latitudinal occurrence were detected in the largest clade, Pooideae, no significant differences were detected in latitudinal movement in any clade. Our preliminary study is the first to attempt to connect ploidy changes to continuous latitudinal movement, but we cannot favor one hypothesis over another. Given that patterns seem to be clade-specific, a larger number of clades must be analyzed in future studies for generalities to be drawn.\n",
  "wordCount" : "2456",
  "inLanguage": "en",
  "datePublished": "2023-09-10T10:36:49Z",
  "dateModified": "2023-09-10T10:36:49Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta"><span>updated on September 10, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.03.23.533992">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.03.23.533992" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.03.23.533992">
        <p class="paperTitle">A Single-Nucleus Atlas of Seed-to-Seed Development in Arabidopsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.03.23.533992" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.03.23.533992" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, T. A.; Nobori, T.; Illouz-Eliaz, N.; Xu, J.; Jow, B.; Nery, J. R.; Ecker, J. R.</p>
        <p class="info">Score: 193.5, Published: 2023-09-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.03.23.533992' target='https://doi.org/10.1101/2023.03.23.533992'> 10.1101/2023.03.23.533992</a></p>
        <p class="abstract">Extensive studies of the reference plant Arabidopsis have enabled a deep understanding of tissues throughout development, yet a census of cell types and states throughout development is lacking. Here, we present a single-nucleus transcriptome atlas of seed-to-seed development employing over 800,000 nuclei, encompassing a diverse set of tissues across ten developmental stages, with spatial transcriptomic validation of the dynamic seed and silique. Cross-organ analyses revealed transcriptional conservation of cell types throughout development and heterogeneity within individual cell types influenced by organ-of-origin and developmental timing, including groups of transcription factors, suggesting gatekeeping by transcription factor activation. This atlas provides a resource for the study of cell type specification throughout the development continuum and a reference for stimulus-response and genetic perturbations at the single-cell resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556499">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556499" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556499">
        <p class="paperTitle">Retrotransposon-driven environmental regulation of FLC leads to adaptive response to herbicide</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556499" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556499" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Raingeval, M.; Leduque, B.; Baduel, P.; Edera, A.; Roux, F.; Colot, V.; Quadrana, L.</p>
        <p class="info">Score: 18.3, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556499' target='https://doi.org/10.1101/2023.09.06.556499'> 10.1101/2023.09.06.556499</a></p>
        <p class="abstract">The mobilization of retrotransposons yields major-effect mutations. Here, we report an adaptive retrotransposon insertion within the first intron of the Arabidopsis floral-repressor locus FLOWERING LOCUS C (FLC). The insertion-mutation augments the environmental sensitivity of FLC by affecting the balance between coding and non-coding transcript isoforms in response to environmental threads. We show that this balance is modulated epigenetically by DNA methylation and orchestrated by IBM2, a factor involved in the processing of intronic heterochromatin. The stress-sensitive allele of FLC has recently spread across populations subjected to recurrent chemical weeding, and we demonstrate that retrotransposon-driven acceleration of life cycle represents a rapid response to herbicide. Our findings illustrate how retrotransposition can create environmentally-sensitive alleles that facilitate adaptation to anthropogenic disturbances of the environment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556214">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556214" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556214">
        <p class="paperTitle">An atypical endomembrane localized CNL-type immune receptor with a conserved deletion in the N-terminal signaling domain functions in cell death and immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556214" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556214" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sunil, S.; Beeh, S.; Stoebbe, E.; Fischer, K. M.; Wilhelm, F.; Paris, C.; Meral, A.; Teasdale, L.; Jiang, Z.; Zhang, l.; Aguilar Parras, E.; Nuernberger, T.; Weigel, D.; Lozano-Duran, R.; El Kasmi, F.</p>
        <p class="info">Score: 17.7, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556214' target='https://doi.org/10.1101/2023.09.04.556214'> 10.1101/2023.09.04.556214</a></p>
        <p class="abstract">Plants have evolved intracellular nucleotide-binding leucine rich repeat receptors (NLRs) to induce a superior immune response. Upon activation, coiled-coil (CC) domain containing NLRs (CNLs) oligomerize to form apparent cation channels that promote calcium influx and cell death induction, with the alpha-1 helix of the individual CC domains penetrating membranes. Some members of a monophyletic subclass of CNLs, the ancient and autonomous NLRs (ANLs), are characterized by putative N- myristoylation and S-acylation sites at the N-terminus of their CCG10/GA domain, potentially mediating permanent membrane association. Whether these Potentially Membrane Localized NLRs (PMLs) mediate cell death upon activation in a similar way as reported for other CNLs has been unknown. We integrated phylogenetic, cell- biological, and functional studies to uncover the cell death function of an atypical but conserved Arabidopsis PML, PML5, which has a 113 amino acid deletion in its CCG10/GA domain. Active PML5 oligomers localize in Golgi membranes and the tonoplast, changes vacuolar morphology, and induce cell death, with the short N- terminus being sufficient for cell death. Mutant analysis supports a potential key role of PMLs in plant immunity. Similar deletions as in Arabidopsis PML5 are found in several Brassicales paralogs, pointing to the evolutionary importance of this innovation. PML5 is thus a naturally occurring CNL variant with a minimal signaling  domain and its further study should help in understanding the functional importance of this minimal domain for NLR signaling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555833">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555833" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555833">
        <p class="paperTitle">Upstream regulator of genomic imprinting in rice endosperm is a small RNA-associated chromatin remodeler</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555833" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555833" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pal, A.; Sundar, V. H.; Nambiar, A.; Shivaprasad, P. V.</p>
        <p class="info">Score: 18.1, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555833' target='https://doi.org/10.1101/2023.08.31.555833'> 10.1101/2023.08.31.555833</a></p>
        <p class="abstract">Genomic imprinting is observed in endosperm, a placenta-like seed tissue, where transposable elements (TEs) and repeat-derived small(s)RNAs mediate epigenetic changes in plants. In imprinting, uniparental gene expression arises due to parent-specific epigenetic marks on one allele but not on the other. The importance of sRNAs and their regulation in endosperm development or in imprinting is poorly understood in crops. Here we show that a previously uncharacterized CLASSY (CLSY)-family chromatin remodeler named OsCLSY3 is essential for rice endosperm development and imprinting, acting as an upstream player in sRNA pathway. Comparative transcriptome and genetic analysis indicated its endosperm-preferred expression and its paternally imprinted nature. These important features were modulated by RNA-directed DNA methylation (RdDM) of tandemly arranged TEs in its promoter. Upon perturbation of OsCLSY3 in transgenic lines we observed defects in endosperm development and loss of around 70% of all sRNAs. Interestingly, well-conserved endosperm-specific sRNAs (siren) that are vital for reproductive fitness in angiosperms were dependent on OsCLSY3. We also observed many imprinted genes and seed development-associated genes under the control of CLSY3-dependent RdDM. These results support an essential role of OsCLSY3 in rice endosperm development and imprinting, and propose similar regulatory strategies involving CLSY3 homologs among other cereals.

HighlightsO_LIUnlike among dicots, in rice and maize, CLSY3 is a maternally expressed imprinted gene majorly expressed in endosperm.
C_LIO_LIEndosperm-preferred expression of OsCLSY3 is regulated by RNA-directed DNA methylation at two tandem transposon elements present in its promoter.
C_LIO_LIOsCLSY3 is crucial for endosperm development and grain filling. It regulates expression of key seed development and endosperm-specific imprinted genes through RNA directed DNA methylation.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.03.556139">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.03.556139" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.03.556139">
        <p class="paperTitle">Establishing a comprehensive web-based analysis platform for Nicotiana benthamiana genome and transcriptome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.03.556139" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.03.556139" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kurotani, K.-i.; Hirakawa, H.; Shirasawa, K.; Tagiri, K.; Mori, M.; Ichihashi, Y.; Suzuki, T.; Tanizawa, Y.; Nakamura, Y.; Isobe, S.; Notaguchi, M.</p>
        <p class="info">Score: 15.6, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.03.556139' target='https://doi.org/10.1101/2023.09.03.556139'> 10.1101/2023.09.03.556139</a></p>
        <p class="abstract">Nicotiana benthamiana has long served as a crucial plant material extensively used in plant physiology research, particularly in the field of plant pathology, because of its high susceptibility to plant viruses. Additionally, it serves as a production platform to test vaccines and other valuable substances. Among its approximately 3.1 Gb genome, 57,583 genes have been annotated within a 61 Mb region. We created a comprehensive and easy-to-use platform to use transcriptomes for modern annotation. These tools allow to visualize gene expression profiles, draw molecular evolutionary phylogenetic trees of gene families, perform functional enrichment analyses, and facilitate output downloads. To demonstrate their utility, we analyzed the gene expression profiles of enzymes within the nicotine biosynthesis pathway, a secondary metabolic pathway characteristic of the Nicotiana genus. Using the developed tool, expression profiles of the nicotine biosynthesis pathway genes were generated. The expression patterns of eight gene groups in the pathway were strongly expressed in the roots and weakly expressed in leaves and flowers of N. benthamiana. The results were consistent with the established gene expression profiles in Nicotiana tabacum and provided insights into gene family composition and expression trends. The compilation of this database tool can facilitate genetic analysis of N. benthamiana in the future.

Significance statementA tool was developed to visualize gene expression profiles, draw molecular evolutionary phylogenetic trees of gene families, perform functional enrichment analyses, and facilitate output downloads of Nicotiana benthamiana. The database developed using the tool can evolve into a comprehensive all-in-one analysis platform by continuously incorporating transcriptome data released to date, newly released RNA-seq data, and annotations in the future.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556260">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556260" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556260">
        <p class="paperTitle">A soybean rust effector protease suppresses host immunity and cleaves a 3-deoxy-7-phosphoheptulonate synthase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556260" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556260" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chicowski, A. S.; Qi, M.; Variz, H.; Bredow, M.; Montes-Serey, C.; Caiazza, F.; Dong, H.; Margets, A. C.; Mejias, J.; Walley, J.; Craik, C.; Pedley, K. F.; Aung, K.; Innes, R. W.; Whitham, S.</p>
        <p class="info">Score: 11.9, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556260' target='https://doi.org/10.1101/2023.09.07.556260'> 10.1101/2023.09.07.556260</a></p>
        <p class="abstract">The devastating soybean rust (SBR) pathogen, Phakopsora pachyrhizi, encodes many secreted proteins, but only two have been functionally characterized for their roles in rust virulence. Here, we demonstrate that transient expression of P. pachyrhizi effector candidate 15 (PpEC15), an aspartic protease, leads to enhanced bacterial growth in planta, suppression of callose deposition, reduced expression of plant defense-related marker genes and suppression of pathogen-associated molecular pattern (PAMP)-induced reactive oxygen species (ROS). Stable expression of PpEC15 in soybean suppresses PAMP-induced ROS production and enhances bacterial growth, indicating that, collectively, PpEC15 suppresses host and non-host innate immune responses. Yeast-two-hybrid and proximity labeling identified putative PpEC15 interacting partners including a peptide-chain release factor (PCRF), a NAC83 (NAM, ATAF, and CUC) transcription factor, and a DAHP (3-deoxy-7-phosphoheptulonate) synthase. We further show that PpEC15 is able to cleave DAHP, but does not cleave PCRF or NAC83. Virus-induced gene silencing of NAC83, PCRF and DAHP altered PAMP-induced ROS production and salicylic acid production, indicating that these proteins may be involved in immune signaling. Collectively, our data show that PpEC15 is conserved across P. pachyrhizi isolates and other economically important rust species and is involved in the suppression of plant basal defense responses. Understanding the role of PpEC15 in P. pachyrhizi virulence will provide a foundation for designing targeted intervention strategies to generate rust-resistant crops.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.27.554993">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.27.554993" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.27.554993">
        <p class="paperTitle">Water fluxes contribute to growth patterning in shoot meristems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.27.554993" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.27.554993" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alonso-Serra, J.; Cheddadi, I.; Kiss, A.; Cerutti, G.; Lionnet, C.; Godin, C.; Hamant, O.</p>
        <p class="info">Score: 35.1, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.27.554993' target='https://doi.org/10.1101/2023.08.27.554993'> 10.1101/2023.08.27.554993</a></p>
        <p class="abstract">In multicellular organisms, localized tissue outgrowth creates a new water sink thereby modifying hydraulic patterns at the organ level. These fluxes are often considered passive by-products of development and their patterning and potential contribution to morphogenesis remains largely unexplored. Here, we generated a complete map of cell volumetric growth and deformation across the shoot apex in Arabidopsis thaliana. Within the organ-meristem boundary, we found that a subpopulation of cells next to fast-growing cells experiences volumetric shrinkage. To understand this process, we used a vertex-based model integrating mechanics and hydraulics, informed by the measured growth rates. Organ outgrowth simulations revealed the emerging water fluxes and predicted water deficit with volume loss for a few cells at the boundary. Consistently, in planta, a water-soluble dye is preferentially allocated to fast-growing tissues and fails to enter the boundary domain. Analysis of intact meristems further validated our model by revealing cell shrinkage next to fast-growing cells in different contexts of tissue surface curvature and cell deformation. A molecular signature of water deficit at the boundary further confirmed our conclusion. Taken together, we propose a model where the differential sink strength of emerging organs prescribes the hydraulic patterns that define the boundary domain at the shoot apex.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555647">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555647" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555647">
        <p class="paperTitle">Heterochromatin re-organization associated with the transcriptional reprogramming under viral infection in Arabidopsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555647" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555647" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Annacondia, M. L.; Juarez-Gonzalez, V. T.; Cheng, J.; Reig-Valiente, J. L.; Martinez, G.</p>
        <p class="info">Score: 14.1, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555647' target='https://doi.org/10.1101/2023.08.30.555647'> 10.1101/2023.08.30.555647</a></p>
        <p class="abstract">Epigenetic mechanisms are key regulators of genomic integrity and genic expression. Emerging evidence shows that epigenetic regulation is an important component of the transcriptional reprogramming during stress. Despite this, the overall stress-induced reprogramming of the different epigenetic marks and their targets are unknown. Here, we uncovered multiple epigenetic changes taking place during viral infection in Arabidopsis thaliana and their connection with gene expression. We find that cucumber mosaic virus (CMV) infection induces an overall reorganization of the repressive epigenetic marks H3K9me2, H3K27me3, and DNA methylation, which interact between them and are dynamic during infection. Overall, these epigenetic changes are involved in the reprogramming of the transcriptional program to adapt to the biotic stress, and might ensure genome stability through the transcriptional control of transposable elements (TEs). Mechanistically, we demonstrate that the catalytic component of the Polycomb Repressive Complex 2 (PRC2) CURLY LEAF (CLF) mediates the transcriptional repression of genes gaining H3K27me3 during viral infection and that mutants on that component induce resistance against CMV. Altogether, our results provide a complete picture of the epigenetic changes that occur during biotic stress and exemplify the overall dynamism of epigenetic regulation in eukaryotic organisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554515">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554515" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554515">
        <p class="paperTitle">Cell Fate Programming by Transcription Factors and Epigenetic Machinery in Stomatal Development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554515" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554515" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, A.; Mair, A.; Matos, J.; Vollbrecht, M.; Xu, S.-L.; Bergmann, D.</p>
        <p class="info">Score: 9.9, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554515' target='https://doi.org/10.1101/2023.08.23.554515'> 10.1101/2023.08.23.554515</a></p>
        <p class="abstract">The development of multi-cellular organisms requires coordinated changes in gene expression that are often mediated by the interaction between transcription factors (TFs) and their corresponding cis-regulatory elements (CREs). During development and differentiation, the accessibility of CREs is dynamically modulated by the epigenome. How the epigenome, CREs and TFs together exert control over cell fate commitment remains to be fully understood. In the Arabidopsis leaf epidermis, meristemoids undergo a series of stereotyped cell divisions, then switch fate to commit to stomatal differentiation. Newly created or reanalyzed scRNA-seq and ChIP-seq data confirm that stomatal development involves distinctive phases of transcriptional regulation and that differentially regulated genes are bound by the stomatal basic-helix-loop-helix (bHLH) TFs. Targets of the bHLHs often reside in repressive chromatin before activation. MNase-seq evidence further suggests that the repressive state can be overcome and remodeled upon activation by specific stomatal bHLHs. We propose that chromatin remodeling is mediated through the recruitment of a set of physical interactors that we identified through proximity labeling - the ATPase-dependent chromatin remodeling SWI/SNF complex and the histone acetyltransferase HAC1. The bHLHs and chromatin remodelers localize to overlapping genomic regions in a hierarchical order. Furthermore, plants with stage-specific knock-down of the SWI/SNF components or HAC1 fail to activate specific bHLH targets and display stomatal development defects. Together these data converge on a model for how stomatal TFs and epigenetic machinery cooperatively regulate transcription and chromatin remodeling during progressive fate specification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555981">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555981" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555981">
        <p class="paperTitle">Historical causes for the greater proportion of polyploid plants in higher latitudes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555981" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555981" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hagen, E. R.; Vasconcelos, T.; Boyko, J. D.; Beaulieu, J. M.</p>
        <p class="info">Score: 7.7, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555981' target='https://doi.org/10.1101/2023.09.01.555981'> 10.1101/2023.09.01.555981</a></p>
        <p class="abstract">The proportion of polyploid plants in a community increases with latitude, and different hypotheses have been proposed about which factors drive this pattern. Here, we aim to understand the historical causes of the latitudinal polyploidy gradient using a combination of ancestral state reconstruction methods. Specifically, we assess whether (1) polyploidization enables movement to higher latitudes (i.e., polyploidization precedes occurrences in higher latitudes) or (2) higher latitudes facilitate polyploidization (i.e., occurrence in higher latitudes precedes polyploidization). We reconstruct the ploidy states and ancestral niches of 1,032 angiosperm species at four paleoclimatic time slices ranging from 3.3 million years ago to the present, comprising taxa from four well-represented clades: Onagraceae, Primulaceae, Solanum (Solanaceae), and Pooideae (Poaceae). We use ancestral niche reconstruction models alongside a customized discrete character evolution model to allow reconstruction of states at specific time slices. Patterns of latitudinal movement are reconstructed and compared in relation to inferred ploidy shifts. We find that no single hypothesis applies equally well across all analyzed clades. While significant differences in median latitudinal occurrence were detected in the largest clade, Pooideae, no significant differences were detected in latitudinal movement in any clade. Our preliminary study is the first to attempt to connect ploidy changes to continuous latitudinal movement, but we cannot favor one hypothesis over another. Given that patterns seem to be clade-specific, a larger number of clades must be analyzed in future studies for generalities to be drawn.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
