<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Evolutionary Trajectory of Pattern Recognition Receptors in Plants
Authors: Ngou, B. P. M.; Wyler, M.; Schmid, M. W.; Kadota, Y.; Shirasu, K.
Score: 61.9, Published: 2023-07-05 DOI: 10.1101/2023.07.04.547604
Plants perceive pathogen-associated molecular patterns (PAMPs) via pattern recognition receptors (PRRs) to activate PRR-triggered immunity (PTI). Despite extensive research on PTI in model plant species, the evolutionary trajectory and emergence of PRRs remain elusive. Here we conducted a comparative genomic analysis of cell-surface receptors and downstream signalling components among 350 plant species.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="Evolutionary Trajectory of Pattern Recognition Receptors in Plants
Authors: Ngou, B. P. M.; Wyler, M.; Schmid, M. W.; Kadota, Y.; Shirasu, K.
Score: 61.9, Published: 2023-07-05 DOI: 10.1101/2023.07.04.547604
Plants perceive pathogen-associated molecular patterns (PAMPs) via pattern recognition receptors (PRRs) to activate PRR-triggered immunity (PTI). Despite extensive research on PTI in model plant species, the evolutionary trajectory and emergence of PRRs remain elusive. Here we conducted a comparative genomic analysis of cell-surface receptors and downstream signalling components among 350 plant species." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-12T10:41:10+00:00" />
<meta property="article:modified_time" content="2023-07-12T10:41:10+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="Evolutionary Trajectory of Pattern Recognition Receptors in Plants
Authors: Ngou, B. P. M.; Wyler, M.; Schmid, M. W.; Kadota, Y.; Shirasu, K.
Score: 61.9, Published: 2023-07-05 DOI: 10.1101/2023.07.04.547604
Plants perceive pathogen-associated molecular patterns (PAMPs) via pattern recognition receptors (PRRs) to activate PRR-triggered immunity (PTI). Despite extensive research on PTI in model plant species, the evolutionary trajectory and emergence of PRRs remain elusive. Here we conducted a comparative genomic analysis of cell-surface receptors and downstream signalling components among 350 plant species."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "Evolutionary Trajectory of Pattern Recognition Receptors in Plants\nAuthors: Ngou, B. P. M.; Wyler, M.; Schmid, M. W.; Kadota, Y.; Shirasu, K.\nScore: 61.9, Published: 2023-07-05 DOI: 10.1101/2023.07.04.547604\nPlants perceive pathogen-associated molecular patterns (PAMPs) via pattern recognition receptors (PRRs) to activate PRR-triggered immunity (PTI). Despite extensive research on PTI in model plant species, the evolutionary trajectory and emergence of PRRs remain elusive. Here we conducted a comparative genomic analysis of cell-surface receptors and downstream signalling components among 350 plant species.",
  "keywords": [
    
  ],
  "articleBody": " Evolutionary Trajectory of Pattern Recognition Receptors in Plants\nAuthors: Ngou, B. P. M.; Wyler, M.; Schmid, M. W.; Kadota, Y.; Shirasu, K.\nScore: 61.9, Published: 2023-07-05 DOI: 10.1101/2023.07.04.547604\nPlants perceive pathogen-associated molecular patterns (PAMPs) via pattern recognition receptors (PRRs) to activate PRR-triggered immunity (PTI). Despite extensive research on PTI in model plant species, the evolutionary trajectory and emergence of PRRs remain elusive. Here we conducted a comparative genomic analysis of cell-surface receptors and downstream signalling components among 350 plant species. Our findings reveal that cell-surface receptors comprise two major classes, receptor-like proteins (RLPs) and receptor-like kinases (RLKs), with RLP being more ancient whereas RLK families have undergone significant expansion. We also demonstrate that multiple downstream signalling components have an ancient origin within the plant lineage. To shed light on the immune-specificity of PRRs, we traced the evolutionary origin of immune-specific leucine-rich repeat-RLPs (LRR-RLPs) in plants. Surprisingly, we discovered that the last four LRR motifs crucial for co-receptor interaction in LRR-RLPs are closely related to those of the LRR-RLK subgroup Xb, which primarily governs growth and development. Functional characterisation further reveals that LRR-RLPs initiate immune responses through their juxtamembrane and transmembrane regions, while LRR-RLK-Xb members regulate development through their cytosolic kinase domains. Our data suggest modular evolution of cell-surface receptors in which immunity- and development-specific cell-surface receptors share a common origin. After diversification, their ectodomains, juxtamembrane, transmembrane, and cytosolic regions have either diversified or stabilised to recognize ligands that activate different downstream responses. We propose that cell-surface receptors and downstream signalling components are ancient, and likely predate the emergence of land plants, subsequently evolving to exhibit greater complexity and specificity within the land plant lineage.\nA RabGAP-Rab GTPase pair regulates plant autophagy and immunity\nAuthors: Yuen, E. L. H.; Leary, A. Y.; Clavel, M.; Tumtas, Y.; Mohseni, A.; Picchianti, L.; Jamshidiha, M.; Pandey, P.; Duggan, C.; Cota, E.; Dagdas, Y.; Bozkurt, T. O.\nScore: 52.9, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547386\nPlants rely on autophagy and membrane trafficking to tolerate stress, combat infections, and maintain cellular homeostasis. However, the molecular interplay between autophagy and membrane trafficking is poorly understood. Using an AI-assisted approach, we identified Rab3GAP-like (Rab3GAPL) as an important membrane trafficking node that controls plant autophagy negatively. Rab3GAPL suppresses autophagy by binding to ATG8, the core autophagy adaptor, and deactivating Rab8a, a small GTPase essential for autophagosome formation and defense-related secretion. Rab3GAPL from Nicotiana benthamiana, but not its mutated form deficient in ATG8 binding, reduced autophagic flux in N. benthamiana and Arabidopsis. Furthermore, Rab3GAPL-knockout mutants of the liverwort Marchantia polymorpha exhibited enhanced autophagic flux under both normal and heat stress conditions, suggesting that Rab3GAPLs negative regulatory role in autophagy is conserved in land plants. Beyond autophagy regulation, Rab3GAPL modulates focal immunity against the oomycete pathogen Phytophthora infestans by preventing defense-related secretion. Altogether, our results suggest that Rab3GAPL acts as a molecular rheostat to coordinate autophagic flux and defense-related secretion by restraining Rab8a-mediated trafficking. This unprecedented interplay between a RabGAP-Rab pair and ATG8 sheds new light on the intricate membrane transport mechanisms underlying plant autophagy and immunity.\nA fungal pathogen manipulates phytocytokine signaling for plant infection\nAuthors: Hua, C.; Zhang, L.; Stintzi, A.; Schaller, A.; Guo, H.-S.; Nuernberger, T.\nScore: 20.4, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547642\nPhytocytokines are hormone-like plant peptides that modulate immune homeostasis and development. Phytosulfokine (PSK) mediates plant growth and attenuates activation of plant pattern-triggered immunity (PTI). We show that the small cysteine-containing effector VdSCP8 from Verticillium dahliae (Vd) is a virulence-promoting protein that suppresses PTI in Arabidopsis thaliana and Nicotiana benthamiana. Apoplastic SCP8 suppresses immune activation through leucine-rich repeat ectodomain pattern recognition receptors. SCP8 virulence and immunosuppressive activities require PHYTOSULFOKINE RECEPTOR 1 (PSKR1), which binds PSK and forms a complex with co-receptor BAK1 for PTI suppression. We find that PSK, like SCP8, suppresses PTI, SCP8 stimulates PSKR1-BAK1 complex formation, and that Vd requires PSK signaling for host infection. SCP8 interacts with an apoplastic subtilase, and co-expression of SCP8 and subtilase inhibitors reduces PTI suppression. Our findings suggest that a multi-host plant pathogen manipulates PTI by enhancing immunosuppressive PSK signaling, likely through plant subtilase activity.\nComparisons of two receptor pathways in a single cell-type reveal features of signalling specificity.\nAuthors: Ma, Y.; Fluckiger, I.; Nicolet, J.; Pang, J.; Dickinson, J.; De Bellis, D.; Emonet, A.; Fujita, S.; Geldner, N.\nScore: 31.4, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547518\nTo respond appropriately to diverse stimuli, cells harbour numerous receptor pathways yet share downstream signalling components. Mitogen-Activated Protein Kinase (MPK) cascades, present in all eukaryotes, act as central hubs shared amongst diverse signalling pathways. How signalling specificity is maintained within a single plant cell-type is not well understood. We engineered a genetic background for direct comparisons of a developmental and an immunity pathway in the Arabidopsis root endodermis. We show the two pathways maintain distinct outputs despite similar MPK phosphorylation patterns. Systematic activation of all individual MPK Kinases (MKKs) in the endodermis revealed that different MKK groups can drive distinct outputs. We propose that specificity is achieved through a balance of combinatorial activation and cross-pathway inhibition within the MPK cascade. Our findings demonstrate the ability of MPK cascades to generate diverse outputs in one plant cell-type, providing new insights into the mechanisms contributing to signalling specificity.\nDataPLAN: a web-based data management plan generator for the plant sciences\nAuthors: Zhou, X.-R.; Beier, S.; Brilhaus, D.; Rodrigues, C. M.; Mühlhaus, T.; von Suchodoletz, D.; Twyman, R. M.; Usadel, B.; Kranz, A.\nScore: 11.1, Published: 2023-07-10 DOI: 10.1101/2023.07.07.548147\nResearch Data Management (RDM) is a system for the effective handling of research data that enables scientists to structure their research questions and ensure best practices throughout the data lifecycle, from acquisition, computation and annotation to data publication and re-use. Data management plans (DMPs) are documents that formally set out the RDM of a project and are required by many funding bodies. DMPs help to organize and structure RDM strategies, thus promoting data findability, accessibility, interoperability and reusability (FAIR). Although DMPs incorporate methods and standards that can be reused by different research projects, the standardization of DMP content is not as evident as the standardization of RDM practices and data/metadata. To address this issue in the plant sciences, we developed DataPLAN -- a tool that combines a questionnaire with pre-written standardized responses. We wrapped the questionnaire in a serverless single-page web application that can then generate standardized responses from DMP templates. The current templates cater to plant research grant proposals for Horizon 2020, Horizon Europe and the German Research Foundation (Deutsche Forschungsgemeinschaft, DFG). In the future the range of templates will be extended to accommodate other funding schemes, thereby enabling more users to generate their own templates. The DataPLAN web application is open-source and does not require an internet connection. By utilizing DataPLAN, the workload associated with creating, updating, and adhering to DMPs is significantly reduced.\nITPK1 regulates jasmonate-controlled root development in Arabidopsis thaliana\nAuthors: Laha, D.; Pullagurla, N. J.; Shome, S.; Yadav, R.\nScore: 16.8, Published: 2023-06-20 DOI: 10.1101/2023.06.16.545325\nJasmonic acid (JA) is a plant hormone that regulates a plethora of physiological processes including immunity and development and is perceived by the F-Box protein, Coronatine-insensitive protein 1(COI1). The discovery of inositol phosphates (InsPs) in the COI1 receptor complex highlights their role in JA perception. InsPs are phosphate-rich signaling molecules that control many aspects of plant physiology. Inositol pyrophosphates (PP-InsPs) are diphosphate containing InsP species of which InsP7 and InsP8 are the best characterized ones. Different InsP and PP-InsP species are linked with JA-related plant immunity. However, role of PP-InsP species in regulating JA-dependent developmental processes are poorly understood. Recent identification of ITPK1 kinase responsible for the production of 5-InsP7 from InsP6 in planta provides a platform to interrogate possible involvement of ITPK-derived InsP species in JA-related plant development. Herein this study, we report that ITPK1-defective plants exhibit increased root growth inhibition to bioactive JA treatment. The itpk1 plants also show increased lateral root density when treated with JA. Notably, JA treatment does not induce ITPK1 protein level. Gene expression analyses revealed that JA-biosynthetic genes are not differentially expressed in the ITPK1-deficient plants. We further demonstrate that genes encoding different JAZ repressor proteins are severely downregulated in the ITPK1-defective plants. Taken together, our study highlights the role of ITPK1 in regulating JA-dependent root architecture development through controlling expression of different JAZ repressor proteins.\nMapping the landscape of transcription factor promoter activity during vegetative development in Marchantia\nAuthors: Romani, F.; Sauret-Güeto, S.; Rebmann, M.; Annese, D.; Bonter, I.; Tomaselli, M.; Dierschke, T.; Delmans, M.; Frangedakis, E.; Silvestri, L.; Rever, J.; Bowman, J. L.; Romani, I.; Haseloff, J.\nScore: 53.4, Published: 2023-06-20 DOI: 10.1101/2023.06.17.545419\nTranscription factors (TFs) are essential for the regulation of gene expression and cell fate determination. Characterising the transcriptional activity of TF genes in space and time is a critical step towards understanding complex biological systems. The vegetative gametophyte meristems of bryophytes share some characteristics with the shoot-apical meristems of flowering plants. However, the identity and expression profiles of TFs associated with gametophyte organization are largely unknown. With only [~]450 TF genes, Marchantia polymorpha is an outstanding model system for plant systems biology. We have generated a near-complete collection of promoter elements derived from Marchantia TF genes. We experimentally tested in planta reporter fusions for all the TF promoters in the collection and systematically analysed expression patterns in Marchantia gemmae. This allowed us to build a map of precise expression domains and identify a unique set of TFs expressed in the stem-cell zone, providing new insight into the dynamic regulation of the gametophytic meristem and its evolution. In addition, we provide an online database of expression patterns for all promoters in the collection. We expect that the promoter elements characterised here will be useful for cell-type specific expression, synthetic biology applications, and functional genomics.\nRNA-directed DNA Methylation is required for seed development in outbreeding species\nAuthors: Dew-Budd, K. J.; Chow, H. T.; Kendall, T.; David, B. C.; Rozelle, J. A.; Mosher, R. A.; Beilstein, M. A.\nScore: 8.0, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547469\nIn plants, de novo DNA methylation is guided by 24-nt short interfering (si)RNAs in a process called RNA-directed DNA methylation (RdDM). Primarily targeted at transposons, RdDM causes transcriptional silencing and can indirectly influence expression of neighboring genes. During reproduction, a small number of siRNA loci are dramatically upregulation in the maternally-derived seed coat, suggesting that RdDM might have a special function during reproduction. However, the developmental consequence of RdDM has been difficult to dissect because disruption of RdDM does not result in overt phenotypes in Arabidopsis thaliana, where the pathway has been most thoroughly studied. In contrast, Brassica rapa mutants lacking RdDM have a severe seed production defect, which is determined by the maternal sporophytic genotype. To explore the factors that underlie the different phenotypes of these closely-related species, we produced RdDM mutations in three additional members of the Brassicaceae family: Camelina sativa, Capsella rubella, and Capsella grandiflora. Among these three species, only mutations in Capsella grandiflora displayed a seed production defect similar to Brassica rapa mutants, suggesting that mating system is a key determinant for reproductive phenotypes in RdDM mutants.\nSingle-cell transcriptomic analysis of pea shoot development and cell-type-specific responses to boron deficiency\nAuthors: Chen, X.; Ru, Y.; Takahashi, H.; Nakazono, M.; Shabala, S.; Smith, S. M.; Yu, M.\nScore: 9.2, Published: 2023-06-14 DOI: 10.1101/2023.06.14.545017\nUnderstanding how nutrient stress impacts plant growth is fundamentally important to the development of approaches to improve crop production under nutrient limitation. Here we applied single-cell RNA sequencing to shoot apices of Pisum sativum grown under boron (B) deficiency. We identified up to fifteen cell clusters based on the clustering of gene expression profiles and verified cell identity with cell-type-specific marker gene expression. Different cell types responded differently to B deficiency. Specifically, the expression of photosynthetic genes in mesophyll cells (MCs) was down-regulated by B deficiency, consistent with impaired photosynthetic rate. Furthermore, the down-regulation of stomatal development genes in guard cells (GCs), including homologs of MUTE and TOO MANY MOUTHS, correlated with a decrease in stomatal density under B deficiency. We also constructed the developmental trajectory of the shoot apical meristem (SAM) cells and a transcription factor (TF) interaction network. The developmental progression of SAM to MC was characterized by up-regulation of genes encoding histones and chromatin assembly and remodeling proteins including homologs of FASCIATA1 (FAS1) and SWITCH DEFECTIVE/SUCROSE NON-FERMENTABLE (SWI/SNF) complex. However, B deficiency suppressed their expression, which helps to explain impaired SAM development under B deficiency. These results represent a major advance over bulk-tissue RNA-seq analysis in which cell-type-specific responses are lost and hence important physiological responses to B deficiency are missed. The reported approach and resources have potential applications well beyond P. sativum species and could be applied to various legumes to improve their adaptability to multiple nutrient or abiotic stresses.\nRegulation of adaptive growth decisions via phosphorylation of the TRAPPII complex in Arabidopsis\nAuthors: Wiese, C.; Abele, M.; Al, B.; Altmann, M.; Steiner, A.; Kalbfuss, N.; Strohmayr, A.; Ravikumar, R.; Park, C. H.; Brunschweiger, B.; Meng, C.; Facher, E.; Ehrhardt, D. W.; Falter-Braun, P.; Wang, Z.-Y.; Ludwig, C.; Assaad, F. F.\nScore: 10.3, Published: 2023-06-22 DOI: 10.1101/2023.04.24.537966\nPlants often adapt to adverse or stress conditions via differential growth. The trans-Golgi Network (TGN) has been implicated in stress responses, but it is not clear in what capacity it mediates adaptive growth decisions. In this study, we assess the role of the TGN in stress responses by exploring the interactome of the Transport Protein Particle II (TRAPPII) complex, required for TGN structure and function. Together with yeast-two-hybrid screens, this identified shaggy-like kinases (GSK3/AtSKs) as TRAPPII interactors. Kinase assays and pharmacological inhibition provided in vitro and in vivo evidence that AtSKs target the TRAPPII-specific subunit AtTRS120. We identified three GSK3/AtSK phosphorylation sites in AtTRS120. These sites were mutated, and the resulting AtTRS120 phosphovariants subjected to a variety of single and multiple stress conditions. The non-phosphorylatable TRS120 mutant exhibited enhanced adaptation to multiple stress conditions and to osmotic stress whereas the phosphomimetic version was less resilient. This suggests that the TRAPPII phosphostatus mediates adaptive responses to abiotic stress factors. AtSKs are multitaskers that integrate a broad range of signals. Similarly, the TRAPPII interactome is vast and considerably enriched in signaling components. An AtSK-TRAPPII interaction would integrate all levels of cellular organization and instruct the TGN, a central and highly discriminate cellular hub, as to how to mobilize and allocate resources to optimize growth and survival under limiting or adverse conditions.\n",
  "wordCount" : "2388",
  "inLanguage": "en",
  "datePublished": "2023-07-12T10:41:10Z",
  "dateModified": "2023-07-12T10:41:10Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta"><span>updated on July 12, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547604">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547604" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547604">
        <p class="paperTitle">Evolutionary Trajectory of Pattern Recognition Receptors in Plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547604" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547604" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ngou, B. P. M.; Wyler, M.; Schmid, M. W.; Kadota, Y.; Shirasu, K.</p>
        <p class="info">Score: 61.9, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547604' target='https://doi.org/10.1101/2023.07.04.547604'> 10.1101/2023.07.04.547604</a></p>
        <p class="abstract">Plants perceive pathogen-associated molecular patterns (PAMPs) via pattern recognition receptors (PRRs) to activate PRR-triggered immunity (PTI). Despite extensive research on PTI in model plant species, the evolutionary trajectory and emergence of PRRs remain elusive. Here we conducted a comparative genomic analysis of cell-surface receptors and downstream signalling components among 350 plant species. Our findings reveal that cell-surface receptors comprise two major classes, receptor-like proteins (RLPs) and receptor-like kinases (RLKs), with RLP being more ancient whereas RLK families have undergone significant expansion. We also demonstrate that multiple downstream signalling components have an ancient origin within the plant lineage. To shed light on the immune-specificity of PRRs, we traced the evolutionary origin of immune-specific leucine-rich repeat-RLPs (LRR-RLPs) in plants. Surprisingly, we discovered that the last four LRR motifs crucial for co-receptor interaction in LRR-RLPs are closely related to those of the LRR-RLK subgroup Xb, which primarily governs growth and development. Functional characterisation further reveals that LRR-RLPs initiate immune responses through their juxtamembrane and transmembrane regions, while LRR-RLK-Xb members regulate development through their cytosolic kinase domains. Our data suggest modular evolution of cell-surface receptors in which immunity- and development-specific cell-surface receptors share a common origin. After diversification, their ectodomains, juxtamembrane, transmembrane, and cytosolic regions have either diversified or stabilised to recognize ligands that activate different downstream responses. We propose that cell-surface receptors and downstream signalling components are ancient, and likely predate the emergence of land plants, subsequently evolving to exhibit greater complexity and specificity within the land plant lineage.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547386">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547386" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547386">
        <p class="paperTitle">A RabGAP-Rab GTPase pair regulates plant autophagy and immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547386" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547386" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yuen, E. L. H.; Leary, A. Y.; Clavel, M.; Tumtas, Y.; Mohseni, A.; Picchianti, L.; Jamshidiha, M.; Pandey, P.; Duggan, C.; Cota, E.; Dagdas, Y.; Bozkurt, T. O.</p>
        <p class="info">Score: 52.9, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547386' target='https://doi.org/10.1101/2023.07.03.547386'> 10.1101/2023.07.03.547386</a></p>
        <p class="abstract">Plants rely on autophagy and membrane trafficking to tolerate stress, combat infections, and maintain cellular homeostasis. However, the molecular interplay between autophagy and membrane trafficking is poorly understood. Using an AI-assisted approach, we identified Rab3GAP-like (Rab3GAPL) as an important membrane trafficking node that controls plant autophagy negatively. Rab3GAPL suppresses autophagy by binding to ATG8, the core autophagy adaptor, and deactivating Rab8a, a small GTPase essential for autophagosome formation and defense-related secretion. Rab3GAPL from Nicotiana benthamiana, but not its mutated form deficient in ATG8 binding, reduced autophagic flux in N. benthamiana and Arabidopsis. Furthermore, Rab3GAPL-knockout mutants of the liverwort Marchantia polymorpha exhibited enhanced autophagic flux under both normal and heat stress conditions, suggesting that Rab3GAPLs negative regulatory role in autophagy is conserved in land plants. Beyond autophagy regulation, Rab3GAPL modulates focal immunity against the oomycete pathogen Phytophthora infestans by preventing defense-related secretion. Altogether, our results suggest that Rab3GAPL acts as a molecular rheostat to coordinate autophagic flux and defense-related secretion by restraining Rab8a-mediated trafficking. This unprecedented interplay between a RabGAP-Rab pair and ATG8 sheds new light on the intricate membrane transport mechanisms underlying plant autophagy and immunity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547642">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547642" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547642">
        <p class="paperTitle">A fungal pathogen manipulates phytocytokine signaling for plant infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547642" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547642" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hua, C.; Zhang, L.; Stintzi, A.; Schaller, A.; Guo, H.-S.; Nuernberger, T.</p>
        <p class="info">Score: 20.4, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547642' target='https://doi.org/10.1101/2023.07.04.547642'> 10.1101/2023.07.04.547642</a></p>
        <p class="abstract">Phytocytokines are hormone-like plant peptides that modulate immune homeostasis and development. Phytosulfokine (PSK) mediates plant growth and attenuates activation of plant pattern-triggered immunity (PTI). We show that the small cysteine-containing effector VdSCP8 from Verticillium dahliae (Vd) is a virulence-promoting protein that suppresses PTI in Arabidopsis thaliana and Nicotiana benthamiana. Apoplastic SCP8 suppresses immune activation through leucine-rich repeat ectodomain pattern recognition receptors. SCP8 virulence and immunosuppressive activities require PHYTOSULFOKINE RECEPTOR 1 (PSKR1), which binds PSK and forms a complex with co-receptor BAK1 for PTI suppression. We find that PSK, like SCP8, suppresses PTI, SCP8 stimulates PSKR1-BAK1 complex formation, and that Vd requires PSK signaling for host infection. SCP8 interacts with an apoplastic subtilase, and co-expression of SCP8 and subtilase inhibitors reduces PTI suppression. Our findings suggest that a multi-host plant pathogen manipulates PTI by enhancing immunosuppressive PSK signaling, likely through plant subtilase activity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547518">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547518" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547518">
        <p class="paperTitle">Comparisons of two receptor pathways in a single cell-type reveal features of signalling specificity.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547518" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547518" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ma, Y.; Fluckiger, I.; Nicolet, J.; Pang, J.; Dickinson, J.; De Bellis, D.; Emonet, A.; Fujita, S.; Geldner, N.</p>
        <p class="info">Score: 31.4, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547518' target='https://doi.org/10.1101/2023.07.03.547518'> 10.1101/2023.07.03.547518</a></p>
        <p class="abstract">To respond appropriately to diverse stimuli, cells harbour numerous receptor pathways yet share downstream signalling components. Mitogen-Activated Protein Kinase (MPK) cascades, present in all eukaryotes, act as central hubs shared amongst diverse signalling pathways. How signalling specificity is maintained within a single plant cell-type is not well understood. We engineered a genetic background for direct comparisons of a developmental and an immunity pathway in the Arabidopsis root endodermis. We show the two pathways maintain distinct outputs despite similar MPK phosphorylation patterns. Systematic activation of all individual MPK Kinases (MKKs) in the endodermis revealed that different MKK groups can drive distinct outputs. We propose that specificity is achieved through a balance of combinatorial activation and cross-pathway inhibition within the MPK cascade. Our findings demonstrate the ability of MPK cascades to generate diverse outputs in one plant cell-type, providing new insights into the mechanisms contributing to signalling specificity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548147">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548147" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548147">
        <p class="paperTitle">DataPLAN: a web-based data management plan generator for the plant sciences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548147" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548147" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, X.-R.; Beier, S.; Brilhaus, D.; Rodrigues, C. M.; Mühlhaus, T.; von Suchodoletz, D.; Twyman, R. M.; Usadel, B.; Kranz, A.</p>
        <p class="info">Score: 11.1, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548147' target='https://doi.org/10.1101/2023.07.07.548147'> 10.1101/2023.07.07.548147</a></p>
        <p class="abstract">Research Data Management (RDM) is a system for the effective handling of research data that enables scientists to structure their research questions and ensure best practices throughout the data lifecycle, from acquisition, computation and annotation to data publication and re-use. Data management plans (DMPs) are documents that formally set out the RDM of a project and are required by many funding bodies. DMPs help to organize and structure RDM strategies, thus promoting data findability, accessibility, interoperability and reusability (FAIR). Although DMPs incorporate methods and standards that can be reused by different research projects, the standardization of DMP content is not as evident as the standardization of RDM practices and data/metadata. To address this issue in the plant sciences, we developed DataPLAN -- a tool that combines a questionnaire with pre-written standardized responses. We wrapped the questionnaire in a serverless single-page web application that can then generate standardized responses from DMP templates. The current templates cater to plant research grant proposals for Horizon 2020, Horizon Europe and the German Research Foundation (Deutsche Forschungsgemeinschaft, DFG). In the future the range of templates will be extended to accommodate other funding schemes, thereby enabling more users to generate their own templates. The DataPLAN web application is open-source and does not require an internet connection. By utilizing DataPLAN, the workload associated with creating, updating, and adhering to DMPs is significantly reduced.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.16.545325">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.16.545325" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.16.545325">
        <p class="paperTitle">ITPK1 regulates jasmonate-controlled root development in Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.16.545325" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.16.545325" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Laha, D.; Pullagurla, N. J.; Shome, S.; Yadav, R.</p>
        <p class="info">Score: 16.8, Published: 2023-06-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.16.545325' target='https://doi.org/10.1101/2023.06.16.545325'> 10.1101/2023.06.16.545325</a></p>
        <p class="abstract">Jasmonic acid (JA) is a plant hormone that regulates a plethora of physiological processes including immunity and development and is perceived by the F-Box protein, Coronatine-insensitive protein 1(COI1). The discovery of inositol phosphates (InsPs) in the COI1 receptor complex highlights their role in JA perception. InsPs are phosphate-rich signaling molecules that control many aspects of plant physiology. Inositol pyrophosphates (PP-InsPs) are diphosphate containing InsP species of which InsP7 and InsP8 are the best characterized ones. Different InsP and PP-InsP species are linked with JA-related plant immunity. However, role of PP-InsP species in regulating JA-dependent developmental processes are poorly understood. Recent identification of ITPK1 kinase responsible for the production of 5-InsP7 from InsP6 in planta provides a platform to interrogate possible involvement of ITPK-derived InsP species in JA-related plant development. Herein this study, we report that ITPK1-defective plants exhibit increased root growth inhibition to bioactive JA treatment. The itpk1 plants also show increased lateral root density when treated with JA. Notably, JA treatment does not induce ITPK1 protein level. Gene expression analyses revealed that JA-biosynthetic genes are not differentially expressed in the ITPK1-deficient plants. We further demonstrate that genes encoding different JAZ repressor proteins are severely downregulated in the ITPK1-defective plants. Taken together, our study highlights the role of ITPK1 in regulating JA-dependent root architecture development through controlling expression of different JAZ repressor proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.17.545419">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.17.545419" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.17.545419">
        <p class="paperTitle">Mapping the landscape of transcription factor promoter activity during vegetative development in Marchantia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.17.545419" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.17.545419" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Romani, F.; Sauret-Güeto, S.; Rebmann, M.; Annese, D.; Bonter, I.; Tomaselli, M.; Dierschke, T.; Delmans, M.; Frangedakis, E.; Silvestri, L.; Rever, J.; Bowman, J. L.; Romani, I.; Haseloff, J.</p>
        <p class="info">Score: 53.4, Published: 2023-06-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.17.545419' target='https://doi.org/10.1101/2023.06.17.545419'> 10.1101/2023.06.17.545419</a></p>
        <p class="abstract">Transcription factors (TFs) are essential for the regulation of gene expression and cell fate determination. Characterising the transcriptional activity of TF genes in space and time is a critical step towards understanding complex biological systems. The vegetative gametophyte meristems of bryophytes share some characteristics with the shoot-apical meristems of flowering plants. However, the identity and expression profiles of TFs associated with gametophyte organization are largely unknown. With only [~]450 TF genes, Marchantia polymorpha is an outstanding model system for plant systems biology. We have generated a near-complete collection of promoter elements derived from Marchantia TF genes. We experimentally tested in planta reporter fusions for all the TF promoters in the collection and systematically analysed expression patterns in Marchantia gemmae. This allowed us to build a map of precise expression domains and identify a unique set of TFs expressed in the stem-cell zone, providing new insight into the dynamic regulation of the gametophytic meristem and its evolution. In addition, we provide an online database of expression patterns for all promoters in the collection. We expect that the promoter elements characterised here will be useful for cell-type specific expression, synthetic biology applications, and functional genomics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547469">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547469" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547469">
        <p class="paperTitle">RNA-directed DNA Methylation is required for seed development in outbreeding species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547469" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547469" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dew-Budd, K. J.; Chow, H. T.; Kendall, T.; David, B. C.; Rozelle, J. A.; Mosher, R. A.; Beilstein, M. A.</p>
        <p class="info">Score: 8.0, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547469' target='https://doi.org/10.1101/2023.07.03.547469'> 10.1101/2023.07.03.547469</a></p>
        <p class="abstract">In plants, de novo DNA methylation is guided by 24-nt short interfering (si)RNAs in a process called RNA-directed DNA methylation (RdDM). Primarily targeted at transposons, RdDM causes transcriptional silencing and can indirectly influence expression of neighboring genes. During reproduction, a small number of siRNA loci are dramatically upregulation in the maternally-derived seed coat, suggesting that RdDM might have a special function during reproduction. However, the developmental consequence of RdDM has been difficult to dissect because disruption of RdDM does not result in overt phenotypes in Arabidopsis thaliana, where the pathway has been most thoroughly studied. In contrast, Brassica rapa mutants lacking RdDM have a severe seed production defect, which is determined by the maternal sporophytic genotype. To explore the factors that underlie the different phenotypes of these closely-related species, we produced RdDM mutations in three additional members of the Brassicaceae family: Camelina sativa, Capsella rubella, and Capsella grandiflora. Among these three species, only mutations in Capsella grandiflora displayed a seed production defect similar to Brassica rapa mutants, suggesting that mating system is a key determinant for reproductive phenotypes in RdDM mutants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.14.545017">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.14.545017" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.14.545017">
        <p class="paperTitle">Single-cell transcriptomic analysis of pea shoot development and cell-type-specific responses to boron deficiency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.14.545017" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.14.545017" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, X.; Ru, Y.; Takahashi, H.; Nakazono, M.; Shabala, S.; Smith, S. M.; Yu, M.</p>
        <p class="info">Score: 9.2, Published: 2023-06-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.14.545017' target='https://doi.org/10.1101/2023.06.14.545017'> 10.1101/2023.06.14.545017</a></p>
        <p class="abstract">Understanding how nutrient stress impacts plant growth is fundamentally important to the development of approaches to improve crop production under nutrient limitation. Here we applied single-cell RNA sequencing to shoot apices of Pisum sativum grown under boron (B) deficiency. We identified up to fifteen cell clusters based on the clustering of gene expression profiles and verified cell identity with cell-type-specific marker gene expression. Different cell types responded differently to B deficiency. Specifically, the expression of photosynthetic genes in mesophyll cells (MCs) was down-regulated by B deficiency, consistent with impaired photosynthetic rate. Furthermore, the down-regulation of stomatal development genes in guard cells (GCs), including homologs of MUTE and TOO MANY MOUTHS, correlated with a decrease in stomatal density under B deficiency. We also constructed the developmental trajectory of the shoot apical meristem (SAM) cells and a transcription factor (TF) interaction network. The developmental progression of SAM to MC was characterized by up-regulation of genes encoding histones and chromatin assembly and remodeling proteins including homologs of FASCIATA1 (FAS1) and SWITCH DEFECTIVE/SUCROSE NON-FERMENTABLE (SWI/SNF) complex. However, B deficiency suppressed their expression, which helps to explain impaired SAM development under B deficiency. These results represent a major advance over bulk-tissue RNA-seq analysis in which cell-type-specific responses are lost and hence important physiological responses to B deficiency are missed. The reported approach and resources have potential applications well beyond P. sativum species and could be applied to various legumes to improve their adaptability to multiple nutrient or abiotic stresses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.04.24.537966">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.04.24.537966" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.04.24.537966">
        <p class="paperTitle">Regulation of adaptive growth decisions via phosphorylation of the TRAPPII complex in Arabidopsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.04.24.537966" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.04.24.537966" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wiese, C.; Abele, M.; Al, B.; Altmann, M.; Steiner, A.; Kalbfuss, N.; Strohmayr, A.; Ravikumar, R.; Park, C. H.; Brunschweiger, B.; Meng, C.; Facher, E.; Ehrhardt, D. W.; Falter-Braun, P.; Wang, Z.-Y.; Ludwig, C.; Assaad, F. F.</p>
        <p class="info">Score: 10.3, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.04.24.537966' target='https://doi.org/10.1101/2023.04.24.537966'> 10.1101/2023.04.24.537966</a></p>
        <p class="abstract">Plants often adapt to adverse or stress conditions via differential growth. The trans-Golgi Network (TGN) has been implicated in stress responses, but it is not clear in what capacity it mediates adaptive growth decisions. In this study, we assess the role of the TGN in stress responses by exploring the interactome of the Transport Protein Particle II (TRAPPII) complex, required for TGN structure and function. Together with yeast-two-hybrid screens, this identified shaggy-like kinases (GSK3/AtSKs) as TRAPPII interactors. Kinase assays and pharmacological inhibition provided in vitro and in vivo evidence that AtSKs target the TRAPPII-specific subunit AtTRS120. We identified three GSK3/AtSK phosphorylation sites in AtTRS120. These sites were mutated, and the resulting AtTRS120 phosphovariants subjected to a variety of single and multiple stress conditions. The non-phosphorylatable TRS120 mutant exhibited enhanced adaptation to multiple stress conditions and to osmotic stress whereas the phosphomimetic version was less resilient. This suggests that the TRAPPII phosphostatus mediates adaptive responses to abiotic stress factors. AtSKs are multitaskers that integrate a broad range of signals. Similarly, the TRAPPII interactome is vast and considerably enriched in signaling components. An AtSK-TRAPPII interaction would integrate all levels of cellular organization and instruct the TGN, a central and highly discriminate cellular hub, as to how to mobilize and allocate resources to optimize growth and survival under limiting or adverse conditions.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
