<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="GreenGate 2.0: backwards compatible addons for assembly of complex transcriptional units and their stacking with GreenGate
Authors: Piepers, M.; Erbstein, K.; Reyes-Hernandez, J.; Song, C.; Tessi, T.; Petrasiunaite, V.; Faerber, N.; Distel, K.; Maizel, A.
Score: 32.4, Published: 2023-08-04 DOI: 10.1101/2023.08.02.551719
Molecular cloning is a crucial technique in genetic engineering that enables the precise design of synthetic transcriptional units (TUs) and the manipulation of genomes. GreenGate and several other modular molecular cloning systems were developed about ten years ago and are widely used in plant research.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="GreenGate 2.0: backwards compatible addons for assembly of complex transcriptional units and their stacking with GreenGate
Authors: Piepers, M.; Erbstein, K.; Reyes-Hernandez, J.; Song, C.; Tessi, T.; Petrasiunaite, V.; Faerber, N.; Distel, K.; Maizel, A.
Score: 32.4, Published: 2023-08-04 DOI: 10.1101/2023.08.02.551719
Molecular cloning is a crucial technique in genetic engineering that enables the precise design of synthetic transcriptional units (TUs) and the manipulation of genomes. GreenGate and several other modular molecular cloning systems were developed about ten years ago and are widely used in plant research." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-06T10:36:48+00:00" />
<meta property="article:modified_time" content="2023-08-06T10:36:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="GreenGate 2.0: backwards compatible addons for assembly of complex transcriptional units and their stacking with GreenGate
Authors: Piepers, M.; Erbstein, K.; Reyes-Hernandez, J.; Song, C.; Tessi, T.; Petrasiunaite, V.; Faerber, N.; Distel, K.; Maizel, A.
Score: 32.4, Published: 2023-08-04 DOI: 10.1101/2023.08.02.551719
Molecular cloning is a crucial technique in genetic engineering that enables the precise design of synthetic transcriptional units (TUs) and the manipulation of genomes. GreenGate and several other modular molecular cloning systems were developed about ten years ago and are widely used in plant research."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "GreenGate 2.0: backwards compatible addons for assembly of complex transcriptional units and their stacking with GreenGate\nAuthors: Piepers, M.; Erbstein, K.; Reyes-Hernandez, J.; Song, C.; Tessi, T.; Petrasiunaite, V.; Faerber, N.; Distel, K.; Maizel, A.\nScore: 32.4, Published: 2023-08-04 DOI: 10.1101/2023.08.02.551719\nMolecular cloning is a crucial technique in genetic engineering that enables the precise design of synthetic transcriptional units (TUs) and the manipulation of genomes. GreenGate and several other modular molecular cloning systems were developed about ten years ago and are widely used in plant research.",
  "keywords": [
    
  ],
  "articleBody": " GreenGate 2.0: backwards compatible addons for assembly of complex transcriptional units and their stacking with GreenGate\nAuthors: Piepers, M.; Erbstein, K.; Reyes-Hernandez, J.; Song, C.; Tessi, T.; Petrasiunaite, V.; Faerber, N.; Distel, K.; Maizel, A.\nScore: 32.4, Published: 2023-08-04 DOI: 10.1101/2023.08.02.551719\nMolecular cloning is a crucial technique in genetic engineering that enables the precise design of synthetic transcriptional units (TUs) and the manipulation of genomes. GreenGate and several other modular molecular cloning systems were developed about ten years ago and are widely used in plant research. All these systems define grammars for assembling transcriptional units from building blocks, cloned as Level 0 modules flanked by four-base pair overhangs and recognition sites for a particular Type IIs endonuclease. Modules are efficiently assembled into Level 1 TUs in a hierarchical assembly process, and Level 2 multigene constructs are assembled by stacking Level 1 TUs. GreenGate is highly popular but has three main limitations. First, using ad-hoc overhangs added by PCR and classical restriction/ligation prevents the efficient use of a one-pot, one-step reaction to generate entry clones and domesticate internal sites; second, a Level 1 TU is assembled from a maximum of six modules, which may be limiting for applications such as multiplex genome editing; third, the generation of Level 2 assemblies is sequential and inefficient. GreenGate 2.0 (GG2.0) expands GreenGate features. It introduces additional overhangs, allowing for the combination of up to 12 Level 0 modules in a Level 1 TU. It includes a Universal Entry Generator plasmid (pUEG) to streamline the generation of Level 0 modules. GG2.0 introduces GreenBraid, a convenient method for stacking transcriptional units iteratively for multigene assemblies. Importantly, GG2.0 is backwards compatible with most existing GreenGate modules. Additionally, GG2.0 includes Level 0 modules for multiplex expression of guide RNAs for CRISPR/Cas9 genome editing and pre-assembled Level 1 vectors for dexamethasone-inducible gene expression and ubiquitous expression of plasma membrane and nuclear fluorescent markers. GG2.0 streamlines and increases the versatility of assembling complex transcriptional units and their combination.\nOryza genera-specific novel Histone H4 variant predisposes H4 Lysine5 Acetylation marks to modulate salt stress responses\nAuthors: G, V. H. S.; Sotelo-Parrilla, P.; Raju, S.; Jha, S.; Gireesh, A.; Gut, F.; Kutti, V. R.; Berger, F.; Jeyaprakash, A. A.; Shivaprasad, P. V.\nScore: 19.6, Published: 2023-08-01 DOI: 10.1101/2023.07.31.551207\nParalogous variants of canonical histones guide accessibility to DNA and function as additional layers of genome regulation. Across eukaryotes, the occurrence, mechanism of action and functional significance of several variants of core histones are well known except that of histone H4. Here we show that a novel variant of H4 (H4.V), expressing tissue-specifically among members of Oryza genera, mediates specific epigenetic changes contributing majorly to salt tolerance. H4.V was incorporated to specific chromosomal locations where it blocked deposition of active histone marks. Under salt stress, large scale re-distribution of H4.V enabled incorporation of stress dependent histone H4 Lysine5 Acetylation (H4K5Ac) marks. Mis-expression of H4.V led to defects at morphological level especially in reproductive tissues, and in mounting stress responses. H4.V mediated these alterations by condensing chromatin at specific genomic regions as seen with cryo-EM structure of reconstituted H4.V containing nucleosomes. These results not only uncovered the presence of a H4 variant in plants, but also a novel chromatin regulation of stress responses that might have contributed to success of semi-aquatic Oryza members under variable water-limiting conditions. One-line summaryHistone H4 variant predisposes chromatin for stress responses O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=199 SRC=\"FIGDIR/small/551207v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (53K): org.highwire.dtl.DTLVardef@1c31e03org.highwire.dtl.DTLVardef@e76e6dorg.highwire.dtl.DTLVardef@2e9dcorg.highwire.dtl.DTLVardef@14775ce_HPS_FORMAT_FIGEXP M_FIG C_FIG\nPlant TIR domains physically interact with EDS1 family proteins to propagate immune signalling.\nAuthors: Chen, J.; Zhang, X.; Bernoux, M.; Rathjen, J. P.; Dodds, P. N.\nScore: 16.4, Published: 2023-08-03 DOI: 10.1101/2023.08.02.551391\nPlant Toll/interleukin-1 receptor/resistance protein (TIR) type nucleotide-binding and leucine-rich repeat immune receptors (NLRs) require Enhanced Disease Susceptibility 1 (EDS1) family proteins and the helper NLRs NRG1 and ADR1 for immune activation. TIR signalling domains possess NADase activity, producing NAM and v-cADPR from NAD+ in vitro. However, after TIR activation different small non-cyclic signalling molecules have been detected bound to EDS1/SAG101 and EDS1/PAD4 heterodimers. These molecules have not been detected in in vitro assays or as free molecules in planta and it is not clear how they are delivered to the EDS1 complexes. Here we investigate physical and functional interactions between TIR signalling domains, EDS1 family proteins and helper NLRs to clarify these signalling transduction pathways. We show that the NbEDS1-NbSAG101b-NbNRG1 signalling pathway in N. benthamiana is necessary and sufficient for cell death signalling induced by six different TIR-containing NLRs from a range of plant species, suggesting this module is likely a universal requirement for TIR-NLR mediated cell death in N. benthamiana. We also find that TIR domains physically interact with NbEDS1, NbPAD4 and NbSAG101 in planta, independently of each other. We also find evidence for direct interaction of NbNRG1 with NbSAG101b via its C-terminal EP domain, but not with other EDS1 family members. These data suggest a model in which physical interaction between activated TIRs and EDS1 signalling complexes facilitates efficient transfer of low abundance products of TIR catalytic activity directly to EDS1 heterocomplexes. The interaction could also alter TIR catalytic activity to favor production of the ligands recognised by EDS1/SAG101 and EDS1/PAD4.\nSpatial Co-transcriptomics Reveals Discrete Stages of the Arbuscular Mycorrhizal Symbiosis\nAuthors: Serrano, K.; Bezrutczyk, M.; Goudeau, D.; Dao, T.; O'Malley, R. C.; Malmstrom, R. R.; Visel, A.; Scheller, H. V.; Cole, B.\nScore: 16.3, Published: 2023-08-02 DOI: 10.1101/2023.08.02.551648\nThe symbiotic interaction of plants with arbuscular mycorrhizal fungi (AM fungi) is ancient and widespread. Plants provide AM fungi with carbon in exchange for nutrients and water, making this interaction a prime target for crop improvement. However, plant-fungal interactions are restricted to a small subset of root cells, precluding the application of most conventional functional genomic techniques to study the molecular bases of these interactions. Here we used single-nucleus and spatial RNA sequencing to explore both M. truncatula and R. irregularis transcriptomes in AM symbiosis at cellular and spatial resolution. Integrated spatially-registered single-cell maps of interacting cells revealed major infected and uninfected plant root cell types. We observed that cortex cells exhibit distinct transcriptome profiles during different stages of colonization by AM fungi, indicating dynamic interplay between both organisms during establishment of the cellular interface enabling successful symbiosis. Our study provides insight into a symbiotic relationship of major agricultural and environmental importance and demonstrates a paradigm combining single-cell and spatial transcriptomics for the analysis of complex organismal interactions.\nBacterial pathogens deliver water/solute-permeable channels as a virulence strategy\nAuthors: Nomura, K.; Andreazza, F.; Cheng, J.; Dong, K.; Zhou, P.; He, S. Y.\nScore: 16.0, Published: 2023-07-29 DOI: 10.1101/2023.07.29.547699\nMany animal and plant pathogenic bacteria utilize a type III secretion system to deliver effector proteins into the host cell1,2. Elucidation of how these effector proteins function in the host cell is critical for understanding infectious diseases in animals and plants3-5. The widely conserved AvrE/DspE-family effectors play a central role in the pathogenesis of diverse phytopathogenic bacteria6. These conserved effectors are involved in the induction of \"water-soaking\" and host cell death that are conducive to bacterial multiplication in infected tissues. However, the exact biochemical functions of AvrE/DspE-family effectors have been recalcitrant to mechanistic understanding for three decades. Here we show that AvrE/DspE-family effectors fold into a {beta}-barrel structure that resembles bacterial porins. Expression of AvrE and DspE in Xenopus oocytes results in (i) inward and outward currents, (ii) permeability to water and (iii) osmolarity-dependent oocyte swelling and bursting. Liposome reconstitution confirmed that the DspE channel alone is sufficient to allow the passage of small molecules such as fluorescein dye. Targeted screening of chemical blockers based on the predicted pore size (15-20 [A]) of the DspE channel identified polyamidoamine (PAMAM) dendrimers as inhibitors of the DspE/AvrE channels. Remarkably, PAMAMs broadly inhibit AvrE/DspE virulence activities in Xenopus oocytes and during Erwinia amylovora and Pseudomonas syringae infections. Thus, we have unraveled the enigmatic function of a centrally important family of bacterial effectors with significant conceptual and practical implications in the study of bacterial pathogenesis.\nMetabolic arms race between a plant and a fungal pathogen\nAuthors: Liu, Y.; Mahdi, L.; Porzel, A.; Stark, P.; Esposto, D.; Scherr-Henning, A.; Bathe, U.; Acosta, I. F.; Zuccaro, A.; Balcke, G. U.; Tissier, A. F.\nScore: 15.3, Published: 2023-07-29 DOI: 10.1101/2023.07.29.551000\nIn this work, we uncover a metabolite interaction between barley and the fungal pathogen Bipolaris sorokiniana (Bs), involving hordedanes, a previously undescribed set of labdane-related diterpenoids with antimicrobial properties. Bs infection of barley roots elicits hordedane synthesis from a 600-kb gene cluster. Heterologous reconstruction of the synthesis pathway in yeast produced several hordedanes, including one of the most advanced products 19-b-hydroxy-hordetrienoic acid (19-OH-HTA). Barley mutants in the diterpene synthase genes of the cluster are unable to produce hordedanes but, unexpectedly, show reduced Bs colonization. Accordingly, 19-OH-HTA enhances both germination and growth of Bs, while it inhibits other fungi, and Bs chemically modifies 19-OH-HTA. Thus, plant and pathogen molecular interactions extend beyond protein-protein recognition and the simple detoxification of plant antimicrobial metabolites. One-Sentence SummaryA fungal pathogen uses barley diterpenoid phytoalexins to facilitate root colonization.\nMultiple host targets of Pseudomonas effector protein HopM1 form a protein complex regulating apoplastic immunity and water homeostasis\nAuthors: He, S. Y.\nScore: 13.7, Published: 2023-08-01 DOI: 10.1101/2023.07.31.551310\nBacterial type III effector proteins injected into the host cell play a critical role in mediating bacterial interactions with plant and animal hosts. Notably, some bacterial effectors are reported to target sequence-unrelated host proteins with unknown functional relationships. The Pseudomonas syringae effector HopM1 is such an example; it interacts with and/or degrades several HopM1-interacting (MIN) Arabidopsis proteins, including HopM1-interacting protein 2 (MIN2/RAD23), HopM1-interacting protein 7 (MIN7/BIG5), HopM1-interacting protein 10 (MIN10/14-3-3), and HopM1-interacting protein 13 (MIN13/BIG2). In this study, we purified the MIN7 complex formed in planta and found that it contains MIN7, MIN10, MIN13, as well as a tetratricopeptide repeat protein named HLB1. Mutational analysis showed that, like MIN7, HLB1 is required for pathogen-associated molecular pattern (PAMP)-, effector-, and benzothiadiazole (BTH)-triggered immunity. HLB1 is recruited to the trans-Golgi network (TGN)/early endosome (EE) in a MIN7-dependent manner. Both min7 and hlb1 mutant leaves contained elevated water content in the leaf apoplast and artificial water infiltration into the leaf apoplast was sufficient to phenocopy immune-suppressing phenotype of HopM1. These results suggest that multiple HopM1-targeted MIN proteins form a protein complex with a dual role in modulating water level and immunity in the apoplast, which provides an explanation for the dual phenotypes of HopM1 during bacterial pathogenesis.\nCell-specialized chloroplast signaling orchestrates photosynthetic and extracellular reactive oxygen species for stress responses\nAuthors: Tee, E. E.; Fairweather, S. J.; Vo, H. M.; Zhao, C.; Breakspear, A.; Kimura, S.; Carmody, M.; Wrzaczek, M.; Broer, S.; Faulkner, C.; Kangasjarvi, J.; Chen, Z.-H.; Pogson, B. J.; Chan, K. X.\nScore: 9.7, Published: 2023-08-03 DOI: 10.1101/2023.08.02.551742\nCellular responses to abiotic stress involve multiple secondary messengers including reactive oxygen species (ROS), Ca2+, phytohormones such as abscisic acid (ABA) and chloroplast-to-nucleus retrograde signals such as 3-phosphoadenosine 5-phosphate (PAP). Mechanism(s) by which these messengers, produced in different subcellular compartments, intersect for cell regulation remain enigmatic. Here we demonstrate a mechanistic link enabling ABA and the chloroplast retrograde signal PAP to coordinate both chloroplast and plasma membrane ROS production. In whole leaves, PAP alters various ROS-related processes including plasmodesmal permeability as well as responses to ozone and the bacterial elicitor flg22, but largely quenches ROS during oxidative stress. Conversely, in guard cells, both PAP and ABA induce a ROS burst in both chloroplasts via photosynthetic electron transport, and the apoplast via the RESPIRATORY BURST OXIDASE HOMOLOG (RBOH). Both subcellular ROS sources were necessary for ABA- and PAP-mediated stomatal closure. However, PAP signaling diverges from ABA by activating RBOHD, instead of RBOHF, for apoplastic ROS production. We identify calcium-dependent protein kinases (CPKs) transcriptionally induced by PAP as the post-translational activators of RBOHD-mediated ROS production. CPK13, CPK32, and CPK34 concurrently activate RBOHD and the slow anion channel SLAC1 by phosphorylating two Serine (S) residues, including S120 which is also targeted by the core ABA signaling kinase OPEN STOMATA 1 (OST1). Consequently, overexpressing the PAP-induced CPKs rescues stomatal closure in ost1. Our data identify chloroplasts, as sources and mediators of ROS and retrograde signals such PAP, to be critical environmental sensors and focal node in the multifaceted cellular stress response network. Significance StatementThe chloroplast is an important node to coordinate multiple plant signaling pathways in response to stresses such as drought. However, how does it function in specialized cells for which carbon fixation is secondary? Here we show that the chloroplast retrograde signal 3-phosphoadenosine 5-phosphate (PAP) plays multiple roles in reactive oxygen species (ROS) signaling and homeostasis. While PAP suppresses ROS in photosynthetic tissue, surprisingly PAP induces ROS in chloroplasts and extracellular space of guard cells to induce stomatal closure. We decipher how PAP-induced proteins activate both extracellular ROS production and anion channels for stomatal closure, thus providing a mechanism by which chloroplasts provide a strategic complement to canonical hormonal pathways in regulating plant physiological responses in specialized cells.\nZmPILS6 is an auxin efflux carrier required for maize root morphogenesis\nAuthors: Cowling, C. L.; Homayouni, A.; Callwood, J. B.; McReynolds, M. R.; Khor, J.; Ke, H.; Draves, M. A.; Dehesh, K.; Walley, J. W.; Strader, L. C.; Kelley, D. R.\nScore: 8.8, Published: 2023-08-02 DOI: 10.1101/2023.08.01.551510\nPlant root systems play a pivotal role in plant physiology and exhibit diverse phenotypic traits. Understanding the genetic mechanisms governing root growth and development in model plants like maize is crucial for enhancing crop resilience to drought and nutrient limitations. This study focused on identifying and characterizing ZmPILS6, an annotated auxin efflux carrier, as a key regulator of various crown root traits in maize. ZmPILS6-modified roots displayed reduced network area and suppressed lateral root formation, desirable traits during drought and low phosphate conditions. The research revealed that ZmPILS6 localizes to the endoplasmic reticulum and plays a vital role in controlling the spatial distribution of indole-3-acetic acid (IAA or \"auxin\") in primary roots. The study also demonstrated that ZmPILS6 can actively efflux IAA when expressed in yeast. Furthermore, the loss of ZmPILS6 resulted in significant proteome remodeling in maize roots, particularly affecting hormone signaling pathways. To identify potential interacting partners of ZmPILS6, a weighted gene co-expression analysis (WGNA) was performed. Altogether, this research contributes to the growing knowledge of essential genetic determinants governing maize root morphogenesis, which is crucial for guiding agricultural improvement strategies. Significance StatementCrop yield and stress resilience are significantly influenced by crown root architecture. A reverse genetic screen aimed at identifying novel regulators of maize root morphogenesis led to the discovery of ZmPILS6, an auxin efflux carrier. The loss of ZmPILS6 negatively impacts numerous root traits that are linked to plant physiology and function. Proteomic characterization of pils6-1 roots revealed that this evolutionarily conserved transporter is required for the proper expression of numerous phytohormone pathways, including abscisic acid, gibberellins, and jasmonic acid. Notably, ZmPILS6 appears to have a contrasting role in regulating root morphogenesis compared to its Arabidopsis ortholog, PILS6. This finding emphasizes the need for functional characterization of candidate genes directly within key crops of interest, which cannot always be correctly inferred from other model plants.\nPloidy-specific transcriptomes shed light on the heterogeneous identity and metabolism of developing pericarp cells\nAuthors: Tourdot, E.; Maza, E.; Djari, A.; Martin, P. G.; Gevaudant, F.; Chevalier, C.; Pirrello, J.; Gonzalez, N.\nScore: 7.7, Published: 2023-07-30 DOI: 10.1101/2023.07.28.550816\nEndoreduplication, during which cells increase their DNA content through successive rounds of full genome replication without cell division, is the major source of endopolyploidy in higher plants. Endoreduplication plays pivotal roles in plant growth and development and is associated with the activation of specific transcriptional programs that are characteristic to each cell type, thereby defining their identity. In plants, endoreduplication is found in numerous organs and cell types and especially in agronomically valuable ones, such as the fleshy fruit (pericarp) of tomato presenting high ploidy levels. We used the tomato pericarp tissue as a model system to explore the transcriptomes associated with endoreduplication progression during fruit growth. We confirmed that expression globally scales with ploidy level and identified sets of genes differentially expressed when comparing ploidy levels at a specific developmental stage. We found that non-endoreduplicated cells are defined by cell division state and cuticle synthesis while endoreduplicated cells are mainly defined by their metabolic activity changing rapidly over time. By combining this dataset with publicly available spatiotemporal pericarp expression data, we proposed a map describing the distribution of ploidy levels within the pericarp. These transcriptome-based predictions were validated by quantifying ploidy levels within the pericarp tissue. This in situ ploidy quantification revealed the dynamic progression of endoreduplication and its cell layer specificity during early fruit development. In summary, the study sheds light on the complex relationship between endoreduplication, cell differentiation, and gene expression patterns in the tomato pericarp. Significance statementThe progression of endoreduplication is very dynamic during early fruit development and displays cell layer specific patterns. The integration of ploidy distribution maps with ploidy-specific transcriptome data revealed that gene expression in the pericarp is controlled in a ploidy-specific manner during the early stages of tomato fruit development, resulting in the spatialization of transcriptional programs.\n",
  "wordCount" : "2841",
  "inLanguage": "en",
  "datePublished": "2023-08-06T10:36:48Z",
  "dateModified": "2023-08-06T10:36:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta"><span>updated on August 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551719">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551719" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551719">
        <p class="paperTitle">GreenGate 2.0: backwards compatible addons for assembly of complex transcriptional units and their stacking with GreenGate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551719" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551719" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Piepers, M.; Erbstein, K.; Reyes-Hernandez, J.; Song, C.; Tessi, T.; Petrasiunaite, V.; Faerber, N.; Distel, K.; Maizel, A.</p>
        <p class="info">Score: 32.4, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551719' target='https://doi.org/10.1101/2023.08.02.551719'> 10.1101/2023.08.02.551719</a></p>
        <p class="abstract">Molecular cloning is a crucial technique in genetic engineering that enables the precise design of synthetic transcriptional units (TUs) and the manipulation of genomes. GreenGate and several other modular molecular cloning systems were developed about ten years ago and are widely used in plant research. All these systems define grammars for assembling transcriptional units from building blocks, cloned as Level 0 modules flanked by four-base pair overhangs and recognition sites for a particular Type IIs endonuclease. Modules are efficiently assembled into Level 1 TUs in a hierarchical assembly process, and Level 2 multigene constructs are assembled by stacking Level 1 TUs. GreenGate is highly popular but has three main limitations. First, using ad-hoc overhangs added by PCR and classical restriction/ligation prevents the efficient use of a one-pot, one-step reaction to generate entry clones and domesticate internal sites; second, a Level 1 TU is assembled from a maximum of six modules, which may be limiting for applications such as multiplex genome editing;  third, the generation of Level 2 assemblies is sequential and inefficient. GreenGate 2.0 (GG2.0) expands GreenGate features. It introduces additional overhangs, allowing for the combination of up to 12 Level 0 modules in a Level 1 TU. It includes a Universal Entry Generator plasmid (pUEG) to streamline the generation of Level 0 modules. GG2.0 introduces GreenBraid, a convenient method for stacking transcriptional units iteratively for multigene assemblies. Importantly, GG2.0 is backwards compatible with most existing GreenGate modules. Additionally, GG2.0 includes Level 0 modules for multiplex expression of guide RNAs for CRISPR/Cas9 genome editing and pre-assembled Level 1 vectors for dexamethasone-inducible gene expression and ubiquitous expression of plasma membrane and nuclear fluorescent markers. GG2.0 streamlines and increases the versatility of assembling complex transcriptional units and their combination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551207">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551207" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551207">
        <p class="paperTitle">Oryza genera-specific novel Histone H4 variant predisposes H4 Lysine5 Acetylation marks to modulate salt stress responses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551207" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551207" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: G, V. H. S.; Sotelo-Parrilla, P.; Raju, S.; Jha, S.; Gireesh, A.; Gut, F.; Kutti, V. R.; Berger, F.; Jeyaprakash, A. A.; Shivaprasad, P. V.</p>
        <p class="info">Score: 19.6, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551207' target='https://doi.org/10.1101/2023.07.31.551207'> 10.1101/2023.07.31.551207</a></p>
        <p class="abstract">Paralogous variants of canonical histones guide accessibility to DNA and function as additional layers of genome regulation. Across eukaryotes, the occurrence, mechanism of action and functional significance of several variants of core histones are well known except that of histone H4. Here we show that a novel variant of H4 (H4.V), expressing tissue-specifically among members of Oryza genera, mediates specific epigenetic changes contributing majorly to salt tolerance. H4.V was incorporated to specific chromosomal locations where it blocked deposition of active histone marks. Under salt stress, large scale re-distribution of H4.V enabled incorporation of stress dependent histone H4 Lysine5 Acetylation (H4K5Ac) marks. Mis-expression of H4.V led to defects at morphological level especially in reproductive tissues, and in mounting stress responses. H4.V mediated these alterations by condensing chromatin at specific genomic regions as seen with cryo-EM structure of reconstituted H4.V containing nucleosomes. These results not only uncovered the presence of a H4 variant in plants, but also a novel chromatin regulation of stress responses that might have contributed to success of semi-aquatic Oryza members under variable water-limiting conditions.

One-line summaryHistone H4 variant predisposes chromatin for stress responses



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=199 SRC=&#34;FIGDIR/small/551207v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (53K):
org.highwire.dtl.DTLVardef@1c31e03org.highwire.dtl.DTLVardef@e76e6dorg.highwire.dtl.DTLVardef@2e9dcorg.highwire.dtl.DTLVardef@14775ce_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551391">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551391" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551391">
        <p class="paperTitle">Plant TIR domains physically interact with EDS1 family proteins to propagate immune signalling.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551391" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551391" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, J.; Zhang, X.; Bernoux, M.; Rathjen, J. P.; Dodds, P. N.</p>
        <p class="info">Score: 16.4, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551391' target='https://doi.org/10.1101/2023.08.02.551391'> 10.1101/2023.08.02.551391</a></p>
        <p class="abstract">Plant Toll/interleukin-1 receptor/resistance protein (TIR) type nucleotide-binding and leucine-rich repeat immune receptors (NLRs) require Enhanced Disease Susceptibility 1 (EDS1) family proteins and the helper NLRs NRG1 and ADR1 for immune activation. TIR signalling domains possess NADase activity, producing NAM and v-cADPR from NAD&#43; in vitro. However, after TIR activation different small non-cyclic signalling molecules have been detected bound to EDS1/SAG101 and EDS1/PAD4 heterodimers. These molecules have not been detected in in vitro assays or as free molecules in planta and it is not clear how they are delivered to the EDS1 complexes. Here we investigate physical and functional interactions between TIR signalling domains, EDS1 family proteins and helper NLRs to clarify these signalling transduction pathways. We show that the NbEDS1-NbSAG101b-NbNRG1 signalling pathway in N. benthamiana is necessary and sufficient for cell death signalling induced by six different TIR-containing NLRs from a range of plant species, suggesting this module is likely a universal requirement for TIR-NLR mediated cell death in N. benthamiana. We also find that TIR domains physically interact with NbEDS1, NbPAD4 and NbSAG101 in planta, independently of each other. We also find evidence for direct interaction of NbNRG1 with NbSAG101b via its C-terminal EP domain, but not with other EDS1 family members. These data suggest a model in which physical interaction between activated TIRs and EDS1 signalling complexes facilitates efficient transfer of low abundance products of TIR catalytic activity directly to EDS1 heterocomplexes. The interaction could also alter TIR catalytic activity to favor production of the ligands recognised by EDS1/SAG101 and EDS1/PAD4.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551648">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551648" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551648">
        <p class="paperTitle">Spatial Co-transcriptomics Reveals Discrete Stages of the Arbuscular Mycorrhizal Symbiosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551648" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551648" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Serrano, K.; Bezrutczyk, M.; Goudeau, D.; Dao, T.; O&#39;Malley, R. C.; Malmstrom, R. R.; Visel, A.; Scheller, H. V.; Cole, B.</p>
        <p class="info">Score: 16.3, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551648' target='https://doi.org/10.1101/2023.08.02.551648'> 10.1101/2023.08.02.551648</a></p>
        <p class="abstract">The symbiotic interaction of plants with arbuscular mycorrhizal fungi (AM fungi) is ancient and widespread. Plants provide AM fungi with carbon in exchange for nutrients and water, making this interaction a prime target for crop improvement. However, plant-fungal interactions are restricted to a small subset of root cells, precluding the application of most conventional functional genomic techniques to study the molecular bases of these interactions. Here we used single-nucleus and spatial RNA sequencing to explore both M. truncatula and R. irregularis transcriptomes in AM symbiosis at cellular and spatial resolution. Integrated spatially-registered single-cell maps of interacting cells revealed major infected and uninfected plant root cell types. We observed that cortex cells exhibit distinct transcriptome profiles during different stages of colonization by AM fungi, indicating dynamic interplay between both organisms during establishment of the cellular interface enabling successful symbiosis. Our study provides insight into a symbiotic relationship of major agricultural and environmental importance and demonstrates a paradigm combining single-cell and spatial transcriptomics for the analysis of complex organismal interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.29.547699">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.29.547699" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.29.547699">
        <p class="paperTitle">Bacterial pathogens deliver water/solute-permeable channels as a virulence strategy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.29.547699" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.29.547699" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nomura, K.; Andreazza, F.; Cheng, J.; Dong, K.; Zhou, P.; He, S. Y.</p>
        <p class="info">Score: 16.0, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.29.547699' target='https://doi.org/10.1101/2023.07.29.547699'> 10.1101/2023.07.29.547699</a></p>
        <p class="abstract">Many animal and plant pathogenic bacteria utilize a type III secretion system to deliver effector proteins into the host cell1,2. Elucidation of how these effector proteins function in the host cell is critical for understanding infectious diseases in animals and plants3-5. The widely conserved AvrE/DspE-family effectors play a central role in the pathogenesis of diverse phytopathogenic bacteria6. These conserved effectors are involved in the induction of &#34;water-soaking&#34; and host cell death that are conducive to bacterial multiplication in infected tissues. However, the exact biochemical functions of AvrE/DspE-family effectors have been recalcitrant to mechanistic understanding for three decades. Here we show that AvrE/DspE-family effectors fold into a {beta}-barrel structure that resembles bacterial porins. Expression of AvrE and DspE in Xenopus oocytes results in (i) inward and outward currents, (ii) permeability to water and (iii) osmolarity-dependent oocyte swelling and bursting. Liposome reconstitution confirmed that the DspE channel alone is sufficient to allow the passage of small molecules such as fluorescein dye. Targeted screening of chemical blockers based on the predicted pore size (15-20 [A]) of the DspE channel identified polyamidoamine (PAMAM) dendrimers as inhibitors of the DspE/AvrE channels. Remarkably, PAMAMs broadly inhibit AvrE/DspE virulence activities in Xenopus oocytes and during Erwinia amylovora and Pseudomonas syringae infections. Thus, we have unraveled the enigmatic function of a centrally important family of bacterial effectors with significant conceptual and practical implications in the study of bacterial pathogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.29.551000">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.29.551000" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.29.551000">
        <p class="paperTitle">Metabolic arms race between a plant and a fungal pathogen</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.29.551000" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.29.551000" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, Y.; Mahdi, L.; Porzel, A.; Stark, P.; Esposto, D.; Scherr-Henning, A.; Bathe, U.; Acosta, I. F.; Zuccaro, A.; Balcke, G. U.; Tissier, A. F.</p>
        <p class="info">Score: 15.3, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.29.551000' target='https://doi.org/10.1101/2023.07.29.551000'> 10.1101/2023.07.29.551000</a></p>
        <p class="abstract">In this work, we uncover a metabolite interaction between barley and the fungal pathogen Bipolaris sorokiniana (Bs), involving hordedanes, a previously undescribed set of labdane-related diterpenoids with antimicrobial properties. Bs infection of barley roots elicits hordedane synthesis from a 600-kb gene cluster. Heterologous reconstruction of the synthesis pathway in yeast produced several hordedanes, including one of the most advanced products 19-b-hydroxy-hordetrienoic acid (19-OH-HTA). Barley mutants in the diterpene synthase genes of the cluster are unable to produce hordedanes but, unexpectedly, show reduced Bs colonization. Accordingly, 19-OH-HTA enhances both germination and growth of Bs, while it inhibits other fungi, and Bs chemically modifies 19-OH-HTA. Thus, plant and pathogen molecular interactions extend beyond protein-protein recognition and the simple detoxification of plant antimicrobial metabolites.

One-Sentence SummaryA fungal pathogen uses barley diterpenoid phytoalexins to facilitate root colonization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551310">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551310" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551310">
        <p class="paperTitle">Multiple host targets of Pseudomonas effector protein HopM1 form a protein complex regulating apoplastic immunity and water homeostasis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551310" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551310" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: He, S. Y.</p>
        <p class="info">Score: 13.7, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551310' target='https://doi.org/10.1101/2023.07.31.551310'> 10.1101/2023.07.31.551310</a></p>
        <p class="abstract">Bacterial type III effector proteins injected into the host cell play a critical role in mediating bacterial interactions with plant and animal hosts. Notably, some bacterial effectors are reported to target sequence-unrelated host proteins with unknown functional relationships. The Pseudomonas syringae effector HopM1 is such an example; it interacts with and/or degrades several HopM1-interacting (MIN) Arabidopsis proteins, including HopM1-interacting protein 2 (MIN2/RAD23), HopM1-interacting protein 7 (MIN7/BIG5), HopM1-interacting protein 10 (MIN10/14-3-3), and HopM1-interacting protein 13 (MIN13/BIG2). In this study, we purified the MIN7 complex formed in planta and found that it contains MIN7, MIN10, MIN13, as well as a tetratricopeptide repeat protein named HLB1. Mutational analysis showed that, like MIN7, HLB1 is required for pathogen-associated molecular pattern (PAMP)-, effector-, and benzothiadiazole (BTH)-triggered immunity. HLB1 is recruited to the trans-Golgi network (TGN)/early endosome (EE) in a MIN7-dependent manner. Both min7 and hlb1 mutant leaves contained elevated water content in the leaf apoplast and artificial water infiltration into the leaf apoplast was sufficient to phenocopy immune-suppressing phenotype of HopM1. These results suggest that multiple HopM1-targeted MIN proteins form a protein complex with a dual role in modulating water level and immunity in the apoplast, which provides an explanation for the dual phenotypes of HopM1 during bacterial pathogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551742">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551742" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551742">
        <p class="paperTitle">Cell-specialized chloroplast signaling orchestrates photosynthetic and extracellular reactive oxygen species for stress responses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551742" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551742" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tee, E. E.; Fairweather, S. J.; Vo, H. M.; Zhao, C.; Breakspear, A.; Kimura, S.; Carmody, M.; Wrzaczek, M.; Broer, S.; Faulkner, C.; Kangasjarvi, J.; Chen, Z.-H.; Pogson, B. J.; Chan, K. X.</p>
        <p class="info">Score: 9.7, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551742' target='https://doi.org/10.1101/2023.08.02.551742'> 10.1101/2023.08.02.551742</a></p>
        <p class="abstract">Cellular responses to abiotic stress involve multiple secondary messengers including reactive oxygen species (ROS), Ca2&#43;, phytohormones such as abscisic acid (ABA) and chloroplast-to-nucleus retrograde signals such as 3-phosphoadenosine 5-phosphate (PAP). Mechanism(s) by which these messengers, produced in different subcellular compartments, intersect for cell regulation remain enigmatic. Here we demonstrate a mechanistic link enabling ABA and the chloroplast retrograde signal PAP to coordinate both chloroplast and plasma membrane ROS production. In whole leaves, PAP alters various ROS-related processes including plasmodesmal permeability as well as responses to ozone and the bacterial elicitor flg22, but largely quenches ROS during oxidative stress. Conversely, in guard cells, both PAP and ABA induce a ROS burst in both chloroplasts via photosynthetic electron transport, and the apoplast via the RESPIRATORY BURST OXIDASE HOMOLOG (RBOH). Both subcellular ROS sources were necessary for ABA- and PAP-mediated stomatal closure. However, PAP signaling diverges from ABA by activating RBOHD, instead of RBOHF, for apoplastic ROS production. We identify calcium-dependent protein kinases (CPKs) transcriptionally induced by PAP as the post-translational activators of RBOHD-mediated ROS production. CPK13, CPK32, and CPK34 concurrently activate RBOHD and the slow anion channel SLAC1 by phosphorylating two Serine (S) residues, including S120 which is also targeted by the core ABA signaling kinase OPEN STOMATA 1 (OST1). Consequently, overexpressing the PAP-induced CPKs rescues stomatal closure in ost1. Our data identify chloroplasts, as sources and mediators of ROS and retrograde signals such PAP, to be critical environmental sensors and focal node in the multifaceted cellular stress response network.

Significance StatementThe chloroplast is an important node to coordinate multiple plant signaling pathways in response to stresses such as drought. However, how does it function in specialized cells for which carbon fixation is secondary? Here we show that the chloroplast retrograde signal 3-phosphoadenosine 5-phosphate (PAP) plays multiple roles in reactive oxygen species (ROS) signaling and homeostasis. While PAP suppresses ROS in photosynthetic tissue, surprisingly PAP induces ROS in chloroplasts and extracellular space of guard cells to induce stomatal closure. We decipher how PAP-induced proteins activate both extracellular ROS production and anion channels for stomatal closure, thus providing a mechanism by which chloroplasts provide a strategic complement to canonical hormonal pathways in regulating plant physiological responses in specialized cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551510">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551510" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551510">
        <p class="paperTitle">ZmPILS6 is an auxin efflux carrier required for maize root morphogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551510" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551510" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cowling, C. L.; Homayouni, A.; Callwood, J. B.; McReynolds, M. R.; Khor, J.; Ke, H.; Draves, M. A.; Dehesh, K.; Walley, J. W.; Strader, L. C.; Kelley, D. R.</p>
        <p class="info">Score: 8.8, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551510' target='https://doi.org/10.1101/2023.08.01.551510'> 10.1101/2023.08.01.551510</a></p>
        <p class="abstract">Plant root systems play a pivotal role in plant physiology and exhibit diverse phenotypic traits. Understanding the genetic mechanisms governing root growth and development in model plants like maize is crucial for enhancing crop resilience to drought and nutrient limitations. This study focused on identifying and characterizing ZmPILS6, an annotated auxin efflux carrier, as a key regulator of various crown root traits in maize. ZmPILS6-modified roots displayed reduced network area and suppressed lateral root formation, desirable traits during drought and low phosphate conditions. The research revealed that ZmPILS6 localizes to the endoplasmic reticulum and plays a vital role in controlling the spatial distribution of indole-3-acetic acid (IAA or &#34;auxin&#34;) in primary roots. The study also demonstrated that ZmPILS6 can actively efflux IAA when expressed in yeast. Furthermore, the loss of ZmPILS6 resulted in significant proteome remodeling in maize roots, particularly affecting hormone signaling pathways. To identify potential interacting partners of ZmPILS6, a weighted gene co-expression analysis (WGNA) was performed. Altogether, this research contributes to the growing knowledge of essential genetic determinants governing maize root morphogenesis, which is crucial for guiding agricultural improvement strategies.

Significance StatementCrop yield and stress resilience are significantly influenced by crown root architecture. A reverse genetic screen aimed at identifying novel regulators of maize root morphogenesis led to the discovery of ZmPILS6, an auxin efflux carrier. The loss of ZmPILS6 negatively impacts numerous root traits that are linked to plant physiology and function. Proteomic characterization of pils6-1 roots revealed that this evolutionarily conserved transporter is required for the proper expression of numerous phytohormone pathways, including abscisic acid, gibberellins, and jasmonic acid. Notably, ZmPILS6 appears to have a contrasting role in regulating root morphogenesis compared to its Arabidopsis ortholog, PILS6. This finding emphasizes the need for functional characterization of candidate genes directly within key crops of interest, which cannot always be correctly inferred from other model plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550816">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550816" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550816">
        <p class="paperTitle">Ploidy-specific transcriptomes shed light on the heterogeneous identity and metabolism of developing pericarp cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550816" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550816" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tourdot, E.; Maza, E.; Djari, A.; Martin, P. G.; Gevaudant, F.; Chevalier, C.; Pirrello, J.; Gonzalez, N.</p>
        <p class="info">Score: 7.7, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550816' target='https://doi.org/10.1101/2023.07.28.550816'> 10.1101/2023.07.28.550816</a></p>
        <p class="abstract">Endoreduplication, during which cells increase their DNA content through successive rounds of full genome replication without cell division, is the major source of endopolyploidy in higher plants. Endoreduplication plays pivotal roles in plant growth and development and is associated with the activation of specific transcriptional programs that are characteristic to each cell type, thereby defining their identity. In plants, endoreduplication is found in numerous organs and cell types and especially in agronomically valuable ones, such as the fleshy fruit (pericarp) of tomato presenting high ploidy levels. We used the tomato pericarp tissue as a model system to explore the transcriptomes associated with endoreduplication progression during fruit growth. We confirmed that expression globally scales with ploidy level and identified sets of genes differentially expressed when comparing ploidy levels at a specific developmental stage. We found that non-endoreduplicated cells are defined by cell division state and cuticle synthesis while endoreduplicated cells are mainly defined by their metabolic activity changing rapidly over time. By combining this dataset with publicly available spatiotemporal pericarp expression data, we proposed a map describing the distribution of ploidy levels within the pericarp. These transcriptome-based predictions were validated by quantifying ploidy levels within the pericarp tissue. This in situ ploidy quantification revealed the dynamic progression of endoreduplication and its cell layer specificity during early fruit development. In summary, the study sheds light on the complex relationship between endoreduplication, cell differentiation, and gene expression patterns in the tomato pericarp.

Significance statementThe progression of endoreduplication is very dynamic during early fruit development and displays cell layer specific patterns. The integration of ploidy distribution maps with ploidy-specific transcriptome data revealed that gene expression in the pericarp is controlled in a ploidy-specific manner during the early stages of tomato fruit development, resulting in the spatialization of transcriptional programs.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
