<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Evolution of research topics and paradigms in plant sciences
Authors: Shiu, S.-H.; Lehti-Shiu, M. D.
Score: 38.1, Published: 2023-10-03 DOI: 10.1101/2023.10.02.560457
Scientific advances due to conceptual or technological innovations can be revealed by examining how research topics have evolved. But such topical evolution is difficult to uncover and quantify because of the large body of literature and the needs of expert knowledge from a wide range of areas in any field.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="Evolution of research topics and paradigms in plant sciences
Authors: Shiu, S.-H.; Lehti-Shiu, M. D.
Score: 38.1, Published: 2023-10-03 DOI: 10.1101/2023.10.02.560457
Scientific advances due to conceptual or technological innovations can be revealed by examining how research topics have evolved. But such topical evolution is difficult to uncover and quantify because of the large body of literature and the needs of expert knowledge from a wide range of areas in any field." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-08T10:37:16+00:00" />
<meta property="article:modified_time" content="2023-10-08T10:37:16+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="Evolution of research topics and paradigms in plant sciences
Authors: Shiu, S.-H.; Lehti-Shiu, M. D.
Score: 38.1, Published: 2023-10-03 DOI: 10.1101/2023.10.02.560457
Scientific advances due to conceptual or technological innovations can be revealed by examining how research topics have evolved. But such topical evolution is difficult to uncover and quantify because of the large body of literature and the needs of expert knowledge from a wide range of areas in any field."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "Evolution of research topics and paradigms in plant sciences\nAuthors: Shiu, S.-H.; Lehti-Shiu, M. D.\nScore: 38.1, Published: 2023-10-03 DOI: 10.1101/2023.10.02.560457\nScientific advances due to conceptual or technological innovations can be revealed by examining how research topics have evolved. But such topical evolution is difficult to uncover and quantify because of the large body of literature and the needs of expert knowledge from a wide range of areas in any field.",
  "keywords": [
    
  ],
  "articleBody": " Evolution of research topics and paradigms in plant sciences\nAuthors: Shiu, S.-H.; Lehti-Shiu, M. D.\nScore: 38.1, Published: 2023-10-03 DOI: 10.1101/2023.10.02.560457\nScientific advances due to conceptual or technological innovations can be revealed by examining how research topics have evolved. But such topical evolution is difficult to uncover and quantify because of the large body of literature and the needs of expert knowledge from a wide range of areas in any field. Here we used machine learning and language models to classify plant science citations into topics representing interconnected, evolving subfields. The changes in prevalence of topical records over the last 50 years reflect major research paradigm shifts and recent radiation of new topics, as well as turnovers of model species and vastly different plant science research trajectories among countries. Our approaches readily summarize the topical diversity and evolution of a scientific field with hundreds of thousands of relevant papers, and they can be applied broadly to other fields. Significance statementChanges in scientific paradigms are foundational for the advancement of science, but such changes are difficult to summarize, quantify, and illustrate. These challenges are exacerbated by the rapid, exponential growth of literature. Applying a combination of machine learning and language modeling to hundreds of thousands of published abstracts, we demonstrate that a scientific field (i.e., plant science) can be summarized as interconnected subfields evolving from one another. We also reveal insights into major research trends and the rise and decline in the use of model organisms in different countries. Our study demonstrates how artificial intelligence and language models can be broadly applied to understand scientific advances that inform science policy and funding decisions.\nPuzzle cell shape emerges from the interaction of growth with mechanical constraints\nAuthors: Trozzi, N.; Lane, B.; Perruchoud, A.; Wang, Y.; Hoermayer, L.; Ansel, M.; Mollier, C.; Malivert, A.; Clark, F.; Reichgelt, T.; Roeder, A. H. K.; Hamant, O.; Boudaoud, A.; Kwiatkowska, D.; Runions, A.; Smith, R. S.; Majda, M.\nScore: 35.1, Published: 2023-10-02 DOI: 10.1101/2023.10.01.560343\nThe puzzle-shaped cells found in the shoot epidermis of many plant species are a fascinating example of complex cell shapes. Because biological form often follows function, the unique shape of these cells suggests that they must serve some adaptive purpose for the plant. We previously proposed that these intricate shapes provide an effective strategy for reducing mechanical stress on the cell wall when epidermal cells undergo growth in more than one direction. Here we analyze a large selection of living and paleo plant species and find that the ability to make puzzle cells is a shared feature across all plant species, although their presence can be hidden as it varies depending on the organ, developmental stage, and environmental conditions. Computational modeling of Arabidopsis and maize epidermal cells revealed that presence and patterning of lobes is a dynamic process that is intricately linked to the growth history and environmental context of the plant organ. Conversely, disrupted lobeyness in mutants or with drug treatments affects plant development and leads to compensatory strategies. We propose that the mechanism underlying the formation of puzzle-shaped cells is likely conserved among higher plants and is a response to a developmental constraint driven by growth and mechanical stress.\nCellular gibberellin dynamics govern indeterminate nodule development, morphologyand function\nAuthors: Drapek, C.; Radzman-Mohd, N. A.; Rizza, A.; Schiessl, K.; Dos Santos Barbosa, F.; Wen, J.; Oldroyd, G. E. D.; Jones, A. M.\nScore: 47.9, Published: 2023-09-11 DOI: 10.1101/2023.09.09.556959\nDuring nutrient scarcity, plants can adapt their developmental strategy to maximize their chance of survival. Such plasticity in development is underpinned by hormonal regulation, which mediates the relationship between environmental cues and developmental outputs. In legumes, endosymbiosis with nitrogen fixing bacteria (rhizobia) is a key adaptation for supplying the plant with nitrogen in the form of ammonium. Rhizobia are housed in lateral root-derived organs termed nodules that maintain an environment conducive to Nitrogenase in these bacteria. Several phytohormones are important for regulating the formation of nodules, with both positive and negative roles proposed for gibberellin (GA). In this study, we determined the cellular location and function of bioactive GA during nodule organogenesis using a genetically-encoded second generation GA biosensor, GPS2. We found endogenous bioactive GA accumulates locally at the site of nodule primordia, increasing dramatically in the cortical cell layers, persisting through cell divisions and maintaining accumulation in the mature nodule meristem. We show, through mis-expression of GA catabolic enzymes that suppress GA accumulation, that GA acts as a positive regulator of nodule growth and development. Furthermore, increasing or decreasing GA through perturbation of biosynthesis gene expression can increase or decrease the size of nodules, respectively. This is unique from lateral root formation, a developmental program that shares common organogenesis regulators. We link GA to a wider gene regulatory program by showing that cytokinin as well as nodule-identity genes induce and sustain GA accumulation necessary for nodule function.\nForward Genetic Stomatal CO2 Response Screen in Grass Brachypodium distachyon Reveals Central Function of a MAP Kinase in Early Stomatal CO2 Signal Transduction\nAuthors: Lopez, B. N. K.; Ceciliato, P. H. O.; Rangel, F.; Salem, E.; Kernig, K.; Takahashi, Y.; Chow, K.; Zhang, L.; Sidhom, M.; Seitz, C.; Sibout, R.; Laudencia, D.; Woods, D. P.; McCammon, J. A.; Vogel, J.; Schroeder, J. I.\nScore: 14.6, Published: 2023-09-29 DOI: 10.1101/2023.09.28.559842\nPlants respond to increased CO2 concentrations through rapid stomatal closure which can contribute to increased water use efficiency. Grasses display faster stomatal responses than dicots due to dumbbell-shaped guard cells flanked by subsidiary cells working in opposition. However, forward genetic screening for stomatal CO2 signaling mutants in grasses has not been reported. The grass model Brachypodium distachyon is closely related to agronomically important cereal crops, sharing largely collinear genomes. To gain insights into CO2 control mechanisms of stomatal movements in grasses, we developed a forward genetics screen on an EMS-mutagenized Brachypodium distachyon M5 population using infrared imaging to identify plants with altered canopy leaf temperature at elevated CO2. A \"chill1\" mutant was isolated, exhibiting consistently cooler leaf temperatures than wildtype control plants after exposure to increased [CO2]. chill1 plants showed strongly impaired high CO2-induced stomatal closure, despite retaining a robust response to abscisic acid. Through bulked segregant whole-genome-sequencing analyses and generation and characterization of CRISPR-cas9 mutants, chill1 was mapped to a protein kinase, BdMPK5. The chill1 mutation impaired bdmpk5 protein-mediated CO2/HCO3-sensing in-vitro consistent with AlphaFold2-directed structural modeling. BdMPK5 is a key signaling component involved in CO2-induced stomatal movements, potentially functioning as a component of the CO2 sensor in grasses.\nRoot Walker: an automated pipeline for large scale quantification of early root growth responses at high spatial and temporal resolution\nAuthors: Platre, M. P.; Mehta, P.; Halvorson, Z.; zhang, L.; Matias, G. F.; Brent, L.; Faizi, K.; Goulding, C.; Busch, W.\nScore: 21.0, Published: 2023-09-25 DOI: 10.1101/2022.11.16.516796\nPlants are sessile organisms that constantly adapt to their changing environment. The root is exposed to numerous environmental signals ranging from nutrients and water to microbial molecular patterns. These signals can trigger distinct responses including the rapid increase or decrease of root growth. Consequently, using root growth as a readout for signal perception can help decipher which external cues are perceived by roots, and how these signals are integrated. To date, studies measuring root growth responses using large numbers of roots have been limited by a lack of high-throughput image acquisition, poor scalability of analytical methods, or low spatiotemporal resolution. Here, we developed the Root Walker pipeline, which uses automated microscopes to acquire time-series images of many roots exposed to controlled treatments with high-spatiotemporal-resolution, in conjunction with fast and automated image analysis software. We demonstrate the power of Root Walker by quantifying root growth rate responses at different time and throughput scales upon treatments with natural auxin, and upon treatment with two mitogen-associated protein kinase cascade inhibitors. We find a concentration-dependent root growth response to auxin and reveal the specificity of one MAPK inhibitor. We further demonstrate the ability of Root Walker for conducting genetic screens by performing a genome wide association study on 260 accessions under 2 weeks. This revealed known and unknown root growth regulators. Root Walker promises to be a useful toolkit for the plant science community, allowing large-scale screening of root growth dynamics for a variety of purposes, including genetic screens for root sensing and root growth response mechanisms.\nBrassinosteroid recruits FERONIA to safeguard cell expansion in Arabidopsis\nAuthors: Chaudhary, A.; Hsiao, Y.-C.; Yeh, F.-L. J.; Wu, H.-M.; Cheung, A. Y.; Xu, S.-L.; Wang, Z.-Y.\nScore: 9.3, Published: 2023-10-02 DOI: 10.1101/2023.10.01.560400\nPlant cell expansion is driven by turgor pressure and regulated by hormones. How plant cells avoid cell wall rupture during hormone-induced cell expansion remains a mystery. Here we show that brassinosteroid (BR), while stimulating cell elongation, promotes the plasma membrane (PM) accumulation of the receptor kinase FERONIA (FER), which monitors cell wall damage and in turn attenuates BR-induced cell elongation to prevent cell rupture. The GSK3-like kinase BIN2 phosphorylates FER, resulting in reduced FER accumulation and translocation from endoplasmic reticulum to PM. By inactivating BIN2, BR signaling promotes dephosphorylation and increases PM accumulation of FER, thereby enhancing the surveillance of cell wall integrity. Our study reveals a vital signaling circuit that coordinates hormone signaling with mechanical sensing to prevent cell bursting during hormone-induced cell expansion. One-Sentence SummaryBrassinosteroid recruits a cell wall integrity monitor to prevent growth-induced cell wall damage.\nAn updated model of shoot apical meristem regulation by ERECTA family and CLAVATA3 signaling pathways\nAuthors: Uzair, M.; Urquidi-Camacho, R. A.; Liu, Z.; Overholt, A. M.; DeGennaro, D.; Zhang, L.; Herron, B. S.; Hong, T.; Shpak, E. D.\nScore: 8.8, Published: 2023-09-30 DOI: 10.1101/2023.09.29.560237\nThe shoot apical meristem (SAM) gives rise to above-ground organs. The size of the SAM is relatively constant due to the balance of stem cell replenishment versus cell recruitment into developing organs. In angiosperms, the transcription factor WUSCHEL (WUS) promotes stem cell identity in the central zone of the SAM. WUS forms a negative feedback loop with a signaling pathway activated by CLAVATA3 (CLV3). In the periphery of the SAM, the ERECTA family (ERf) receptors promote cell differentiation and constrain the expression of WUS and CLV3. Here, we show that four ligands of ERfs redundantly inhibit CLV3 and WUS expression. Transcriptome analysis confirmed that WUS and CLV3 are the main targets of ERf signaling and uncovered several new ones. Analysis of promoter reporters indicated that in the vegetative meristem, the WUS expression domain mostly overlapped with the CLV3 domain and did not shift along the apical-basal axis in clv3. A 3D mathematical model reproduced the experimentally observed CLV3 and WUS expression patterns with fewer assumptions than earlier models. Based on these findings, we propose that CLV3 regulates cellular levels of WUS expression through autocrine signaling, while ERfs regulate WUS spatial expression, preventing its encroachment into the peripheral zone. One Sentence SummaryThrough autocrine signaling, CLV3 regulates the level of WUS expression in the vegetative SAM but not its location, while ERfs regulate the WUS spatial pattern, preventing its expansion into the peripheral zone.\nGrove-level analysis of titer and prevalence of Candidatus Liberibacter asiaticus and Wolbachia in Diaphorina citri, vector of citrus Huanglongbing\nAuthors: Mann, M.; Thompson, L.; Johnson, L.; Heck, M. L.\nScore: 8.3, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560920\nHuanglongbing (HLB, or citrus greening disease) affects all citrus varieties world-wide. In the USA, Asia, and South America the causal agent is Candidatus Liberibacter asiaticus (CLas), a phloem-limited, uncultured, alphaproteobacterium. The hemipteran insect vector, Diaphorina citri (Asian citrus psyllid) acquires and transmits CLas in a circulative, propagative manner. In addition to CLas, D. citri hosts multiple symbiotic bacteria including Wolbachia (wDi). In D. citri, wDi has been sequenced and studied but specific roles in D. citri biology are unknown. Using well established quantitative PCR methods we measured CLas titer in D. citri collected from four groves in central Florida with distinct HLB management strategies and tested whether CLas and wDi titer were correlated in a sub-set of these insects. Grove site had the largest effect on CLas titer. Sex had no effect on CLas titer, while higher wDi titer was correlated with CLas-uninfected insects. Our results suggest that more directed follow-up research is necessary and important to clarify whether field management tactics influence CLas titer in D. citri and to better understand gene-by-environment interactions among D. citri, wDi and CLas. Now that millions of trees in Florida have been treated with injectable formulations of oxytetracycline, which is likely to decrease bacterial populations in D. citri, this study may represent the last biologically meaningful snapshot of grove-level vector-pathogen ecology in the state during the HLB epidemic.\nControl of cell fate specification and patterning by an ancestral microRNA\nAuthors: Aguilar-Cruz, A.; Flores-Sandoval, E.; Gutierrez-Ramos, X.; Oltehua-Lopez, O.; Dorantes-Acosta, A. E.; Trujillo, J. T.; Kato, H.; Ishizaki, K.; Mosher, R. A.; Dolan, L.; Grimanelli, D.; Haseloff, J.; Bowman, J. L.; Arteaga-Vazquez, M. A.\nScore: 19.8, Published: 2023-09-11 DOI: 10.1101/2023.09.09.556951\nThe formation of an organized body requires the establishment and maintenance of cells with structural and functional distinctive characteristics. A central question in developmental biology is how changes in the regulation of genes drive cell specification and patterning1. microRNAs (miRNAs) are small non-coding RNAs that regulate development through mRNA cleavage and/or translational repression2. In plants, miRNAs regulate key aspects including growth, development, stem cell maintenance, vegetative phase change, leaf morphogenesis, floral organ formation and flowering time3. Biogenesis of plant miRNAs depends on the activity of DICER-LIKE 1 (DCL1), an RNase type III endonuclease that processes double stranded RNA to give rise to mature miRNAs 4. The genomes of todays flora contain at least one bona fide copy of DCL1 5,6. Using Marchantia polymorpha -a model bryophyte that allows comparative approaches to infer characteristics of the ancestral land plant-, we demonstrate that MpDCL1a is required for the biogenesis of miRNAs and uncovered a central role for miR166/Homeodomain Zipper Class III-regulated auxin synthesis in the specification of cell identity, patterning, meristem function, laminar expansion and the development of the body in the last common ancestor of extant land plants.\nSequence characterization of T, Bip, and Phbw demonstrates the role of MYB-bHLH-WD40 complexes and temperature in common bean seed color pattern formation\nAuthors: Parker, T.; Bolt, T.; Williams, T.; Penmetsa, R. V.; Mulube, M.; Palkovic, A.; Jochua, C. N.; Rubio Wilhelmi, M. d. M.; Lo, S.; Bornhorst, G.; Tian, L.; Kamfwa, K.; Hokin, S.; Farmer, A.; Diepenbrock, C. H.; Gepts, P.\nScore: 21.1, Published: 2023-09-29 DOI: 10.1101/2023.09.23.559151\nSeed colors and color patterns are critical for the survival of wild plants and the consumer appeal of crops. In common bean, a major global staple, these patterns are also critical for determining market classes, yet the genetic and environmental control of many pigmentation patterns remains unresolved. In this study, we genetically mapped variation for three important seed pattern loci, T, Bip, and phbw, which co-segregated completely with PvTTG1, PvMYC1, and PvTT8, respectively. Proteins encoded by these genes are predicted to work together in MYB-bHLH-WD40 (MBW) complexes, propagating flavonoid biosynthesis across the seed coat. Whole-genome sequencing of 37 diverse accessions identified putative mutations in each gene, including seven unique parallel mutations in T (PvTTG1) and a non-synonymous SNP in a conserved residue in bipana (PvMYC1). A 612 bp intron deletion in phbw (PvTT8) eliminated motifs conserved since the origins of the Papilionoidea and corresponded to a 20-fold reduction in transcript abundance. Mutations in MBW candidate genes for Z (PvTT2) and Sellatus (WDR) were also identified. In multi-location field trials with seven varieties with partial seed coat patterning, pigmented seed coat area was highly unstable and correlated with temperature, with up to 11-fold differences in pigmented area between the warmest and the coolest environments. In controlled growth chamber conditions, an increase of 4 {degrees}C was sufficient to cause pigmentation on an additional 21% of the seed coat area. Our results shed light on the fundamental activation of flavonoid biosynthesis in common bean and will be instrumental for maximizing consumer appeal in this nutritious staple crop. Summary- Seed colors and patterns are critical for the survival of wild plants, and are important in differentiating crop market classes, but the genetic control of these in the staple crop common bean (Phaseolus vulgaris) is largely unknown. - The genetic, transcriptional, and environmental basis of common bean seed color patterning was explored through QTL mapping, whole-genome sequencing, RT-qPCR, and automated pigmentation quantification of seed grown in multi-location field trials and growth chamber environments. - MYB-bHLH-WD40 complex-forming genes PvTTG1, PvMYC1, and PvTT8 co-segregated completely with the color patterning genes T, Bip, and phbw. Mutations were identified in each gene, including seven unique parallel mutations in T (PvTTG1), a non-synonymous SNP in a conserved residue in bipana (PvMYC1), and an intron deletion in phbw (PvTT8) eliminating highly conserved motifs and corresponding to 20-fold lower PvTT8 transcript abundance. Mutations in MBW candidate genes Z (PvTT2) and Sellatus (WDR) were also identified. In multi-location field trials, pigmented seed coat area was highly unstable and corresponded to temperature. In growth chamber conditions, an increase of 4 {degrees}C caused pigmentation on an additional 21% of the seed coat area. - Our results highlight the critical interaction between MYB-bHLH-WD40 complex components and temperature in establishing seed pattern diversity.\n",
  "wordCount" : "2841",
  "inLanguage": "en",
  "datePublished": "2023-10-08T10:37:16Z",
  "dateModified": "2023-10-08T10:37:16Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta"><span>updated on October 8, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560457">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560457" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560457">
        <p class="paperTitle">Evolution of research topics and paradigms in plant sciences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560457" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560457" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shiu, S.-H.; Lehti-Shiu, M. D.</p>
        <p class="info">Score: 38.1, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560457' target='https://doi.org/10.1101/2023.10.02.560457'> 10.1101/2023.10.02.560457</a></p>
        <p class="abstract">Scientific advances due to conceptual or technological innovations can be revealed by examining how research topics have evolved. But such topical evolution is difficult to uncover and quantify because of the large body of literature and the needs of expert knowledge from a wide range of areas in any field. Here we used machine learning and language models to classify plant science citations into topics representing interconnected, evolving subfields. The changes in prevalence of topical records over the last 50 years reflect major research paradigm shifts and recent radiation of new topics, as well as turnovers of model species and vastly different plant science research trajectories among countries. Our approaches readily summarize the topical diversity and evolution of a scientific field with hundreds of thousands of relevant papers, and they can be applied broadly to other fields.

Significance statementChanges in scientific paradigms are foundational for the advancement of science, but such changes are difficult to summarize, quantify, and illustrate. These challenges are exacerbated by the rapid, exponential growth of literature. Applying a combination of machine learning and language modeling to hundreds of thousands of published abstracts, we demonstrate that a scientific field (i.e., plant science) can be summarized as interconnected subfields evolving from one another. We also reveal insights into major research trends and the rise and decline in the use of model organisms in different countries. Our study demonstrates how artificial intelligence and language models can be broadly applied to understand scientific advances that inform science policy and funding decisions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.560343">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.560343" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.560343">
        <p class="paperTitle">Puzzle cell shape emerges from the interaction of growth with mechanical constraints</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.560343" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.560343" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Trozzi, N.; Lane, B.; Perruchoud, A.; Wang, Y.; Hoermayer, L.; Ansel, M.; Mollier, C.; Malivert, A.; Clark, F.; Reichgelt, T.; Roeder, A. H. K.; Hamant, O.; Boudaoud, A.; Kwiatkowska, D.; Runions, A.; Smith, R. S.; Majda, M.</p>
        <p class="info">Score: 35.1, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.560343' target='https://doi.org/10.1101/2023.10.01.560343'> 10.1101/2023.10.01.560343</a></p>
        <p class="abstract">The puzzle-shaped cells found in the shoot epidermis of many plant species are a fascinating example of complex cell shapes. Because biological form often follows function, the unique shape of these cells suggests that they must serve some adaptive purpose for the plant. We previously proposed that these intricate shapes provide an effective strategy for reducing mechanical stress on the cell wall when epidermal cells undergo growth in more than one direction. Here we analyze a large selection of living and paleo plant species and find that the ability to make puzzle cells is a shared feature across all plant species, although their presence can be hidden as it varies depending on the organ, developmental stage, and environmental conditions. Computational modeling of Arabidopsis and maize epidermal cells revealed that presence and patterning of lobes is a dynamic process that is intricately linked to the growth history and environmental context of the plant organ. Conversely, disrupted lobeyness in mutants or with drug treatments affects plant development and leads to compensatory strategies. We propose that the mechanism underlying the formation of puzzle-shaped cells is likely conserved among higher plants and is a response to a developmental constraint driven by growth and mechanical stress.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.556959">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.556959" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.556959">
        <p class="paperTitle">Cellular gibberellin dynamics govern indeterminate nodule development, morphologyand function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.556959" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.556959" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Drapek, C.; Radzman-Mohd, N. A.; Rizza, A.; Schiessl, K.; Dos Santos Barbosa, F.; Wen, J.; Oldroyd, G. E. D.; Jones, A. M.</p>
        <p class="info">Score: 47.9, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.556959' target='https://doi.org/10.1101/2023.09.09.556959'> 10.1101/2023.09.09.556959</a></p>
        <p class="abstract">During nutrient scarcity, plants can adapt their developmental strategy to maximize their chance of survival. Such plasticity in development is underpinned by hormonal regulation, which mediates the relationship between environmental cues and developmental outputs. In legumes, endosymbiosis with nitrogen fixing bacteria (rhizobia) is a key adaptation for supplying the plant with nitrogen in the form of ammonium. Rhizobia are housed in lateral root-derived organs termed nodules that maintain an environment conducive to Nitrogenase in these bacteria. Several phytohormones are important for regulating the formation of nodules, with both positive and negative roles proposed for gibberellin (GA). In this study, we determined the cellular location and function of bioactive GA during nodule organogenesis using a genetically-encoded second generation GA biosensor, GPS2. We found endogenous bioactive GA accumulates locally at the site of nodule primordia, increasing dramatically in the cortical cell layers, persisting through cell divisions and maintaining accumulation in the mature nodule meristem. We show, through mis-expression of GA catabolic enzymes that suppress GA accumulation, that GA acts as a positive regulator of nodule growth and development. Furthermore, increasing or decreasing GA through perturbation of biosynthesis gene expression can increase or decrease the size of nodules, respectively. This is unique from lateral root formation, a developmental program that shares common organogenesis regulators. We link GA to a wider gene regulatory program by showing that cytokinin as well as nodule-identity genes induce and sustain GA accumulation necessary for nodule function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.559842">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.559842" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.559842">
        <p class="paperTitle">Forward Genetic Stomatal CO2 Response Screen in Grass Brachypodium distachyon Reveals Central Function of a MAP Kinase in Early Stomatal CO2 Signal Transduction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.559842" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.559842" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lopez, B. N. K.; Ceciliato, P. H. O.; Rangel, F.; Salem, E.; Kernig, K.; Takahashi, Y.; Chow, K.; Zhang, L.; Sidhom, M.; Seitz, C.; Sibout, R.; Laudencia, D.; Woods, D. P.; McCammon, J. A.; Vogel, J.; Schroeder, J. I.</p>
        <p class="info">Score: 14.6, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.559842' target='https://doi.org/10.1101/2023.09.28.559842'> 10.1101/2023.09.28.559842</a></p>
        <p class="abstract">Plants respond to increased CO2 concentrations through rapid stomatal closure which can contribute to increased water use efficiency. Grasses display faster stomatal responses than dicots due to dumbbell-shaped guard cells flanked by subsidiary cells working in opposition. However, forward genetic screening for stomatal CO2 signaling mutants in grasses has not been reported. The grass model Brachypodium distachyon is closely related to agronomically important cereal crops, sharing largely collinear genomes. To gain insights into CO2 control mechanisms of stomatal movements in grasses, we developed a forward genetics screen on an EMS-mutagenized Brachypodium distachyon M5 population using infrared imaging to identify plants with altered canopy leaf temperature at elevated CO2. A &#34;chill1&#34; mutant was isolated, exhibiting consistently cooler leaf temperatures than wildtype control plants after exposure to increased [CO2]. chill1 plants showed strongly impaired high CO2-induced stomatal closure, despite retaining a robust response to abscisic acid. Through bulked segregant whole-genome-sequencing analyses and generation and characterization of CRISPR-cas9 mutants, chill1 was mapped to a protein kinase, BdMPK5. The chill1 mutation impaired bdmpk5 protein-mediated CO2/HCO3-sensing in-vitro consistent with AlphaFold2-directed structural modeling. BdMPK5 is a key signaling component involved in CO2-induced stomatal movements, potentially functioning as a component of the CO2 sensor in grasses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.11.16.516796">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.11.16.516796" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.11.16.516796">
        <p class="paperTitle">Root Walker: an automated pipeline for large scale quantification of early root growth responses at high spatial and temporal resolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.11.16.516796" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.11.16.516796" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Platre, M. P.; Mehta, P.; Halvorson, Z.; zhang, L.; Matias, G. F.; Brent, L.; Faizi, K.; Goulding, C.; Busch, W.</p>
        <p class="info">Score: 21.0, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.11.16.516796' target='https://doi.org/10.1101/2022.11.16.516796'> 10.1101/2022.11.16.516796</a></p>
        <p class="abstract">Plants are sessile organisms that constantly adapt to their changing environment. The root is exposed to numerous environmental signals ranging from nutrients and water to microbial molecular patterns. These signals can trigger distinct responses including the rapid increase or decrease of root growth. Consequently, using root growth as a readout for signal perception can help decipher which external cues are perceived by roots, and how these signals are integrated. To date, studies measuring root growth responses using large numbers of roots have been limited by a lack of high-throughput image acquisition, poor scalability of analytical methods, or low spatiotemporal resolution. Here, we developed the Root Walker pipeline, which uses automated microscopes to acquire time-series images of many roots exposed to controlled treatments with high-spatiotemporal-resolution, in conjunction with fast and automated image analysis software. We demonstrate the power of Root Walker by quantifying root growth rate responses at different time and throughput scales upon treatments with natural auxin, and upon treatment with two mitogen-associated protein kinase cascade inhibitors. We find a concentration-dependent root growth response to auxin and reveal the specificity of one MAPK inhibitor. We further demonstrate the ability of Root Walker for conducting genetic screens by performing a genome wide association study on 260 accessions under 2 weeks. This revealed known and unknown root growth regulators. Root Walker promises to be a useful toolkit for the plant science community, allowing large-scale screening of root growth dynamics for a variety of purposes, including genetic screens for root sensing and root growth response mechanisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.560400">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.560400" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.560400">
        <p class="paperTitle">Brassinosteroid recruits FERONIA to safeguard cell expansion in Arabidopsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.560400" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.560400" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chaudhary, A.; Hsiao, Y.-C.; Yeh, F.-L. J.; Wu, H.-M.; Cheung, A. Y.; Xu, S.-L.; Wang, Z.-Y.</p>
        <p class="info">Score: 9.3, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.560400' target='https://doi.org/10.1101/2023.10.01.560400'> 10.1101/2023.10.01.560400</a></p>
        <p class="abstract">Plant cell expansion is driven by turgor pressure and regulated by hormones. How plant cells avoid cell wall rupture during hormone-induced cell expansion remains a mystery. Here we show that brassinosteroid (BR), while stimulating cell elongation, promotes the plasma membrane (PM) accumulation of the receptor kinase FERONIA (FER), which monitors cell wall damage and in turn attenuates BR-induced cell elongation to prevent cell rupture. The GSK3-like kinase BIN2 phosphorylates FER, resulting in reduced FER accumulation and translocation from endoplasmic reticulum to PM. By inactivating BIN2, BR signaling promotes dephosphorylation and increases PM accumulation of FER, thereby enhancing the surveillance of cell wall integrity. Our study reveals a vital signaling circuit that coordinates hormone signaling with mechanical sensing to prevent cell bursting during hormone-induced cell expansion.

One-Sentence SummaryBrassinosteroid recruits a cell wall integrity monitor to prevent growth-induced cell wall damage.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560237">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560237" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560237">
        <p class="paperTitle">An updated model of shoot apical meristem regulation by ERECTA family and CLAVATA3 signaling pathways</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560237" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560237" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Uzair, M.; Urquidi-Camacho, R. A.; Liu, Z.; Overholt, A. M.; DeGennaro, D.; Zhang, L.; Herron, B. S.; Hong, T.; Shpak, E. D.</p>
        <p class="info">Score: 8.8, Published: 2023-09-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560237' target='https://doi.org/10.1101/2023.09.29.560237'> 10.1101/2023.09.29.560237</a></p>
        <p class="abstract">The shoot apical meristem (SAM) gives rise to above-ground organs. The size of the SAM is relatively constant due to the balance of stem cell replenishment versus cell recruitment into developing organs. In angiosperms, the transcription factor WUSCHEL (WUS) promotes stem cell identity in the central zone of the SAM. WUS forms a negative feedback loop with a signaling pathway activated by CLAVATA3 (CLV3). In the periphery of the SAM, the ERECTA family (ERf) receptors promote cell differentiation and constrain the expression of WUS and CLV3. Here, we show that four ligands of ERfs redundantly inhibit CLV3 and WUS expression. Transcriptome analysis confirmed that WUS and CLV3 are the main targets of ERf signaling and uncovered several new ones. Analysis of promoter reporters indicated that in the vegetative meristem, the WUS expression domain mostly overlapped with the CLV3 domain and did not shift along the apical-basal axis in clv3. A 3D mathematical model reproduced the experimentally observed CLV3 and WUS expression patterns with fewer assumptions than earlier models. Based on these findings, we propose that CLV3 regulates cellular levels of WUS expression through autocrine signaling, while ERfs regulate WUS spatial expression, preventing its encroachment into the peripheral zone.

One Sentence SummaryThrough autocrine signaling, CLV3 regulates the level of WUS expression in the vegetative SAM but not its location, while ERfs regulate the WUS spatial pattern, preventing its expansion into the peripheral zone.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560920">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560920" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560920">
        <p class="paperTitle">Grove-level analysis of titer and prevalence of Candidatus Liberibacter asiaticus and Wolbachia in Diaphorina citri, vector of citrus Huanglongbing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560920" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560920" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mann, M.; Thompson, L.; Johnson, L.; Heck, M. L.</p>
        <p class="info">Score: 8.3, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560920' target='https://doi.org/10.1101/2023.10.04.560920'> 10.1101/2023.10.04.560920</a></p>
        <p class="abstract">Huanglongbing (HLB, or citrus greening disease) affects all citrus varieties world-wide. In the USA, Asia, and South America the causal agent is Candidatus Liberibacter asiaticus (CLas), a phloem-limited, uncultured, alphaproteobacterium. The hemipteran insect vector, Diaphorina citri (Asian citrus psyllid) acquires and transmits CLas in a circulative, propagative manner. In addition to CLas, D. citri hosts multiple symbiotic bacteria including Wolbachia (wDi). In D. citri, wDi has been sequenced and studied but specific roles in D. citri biology are unknown. Using well established quantitative PCR methods we measured CLas titer in D. citri collected from four groves in central Florida with distinct HLB management strategies and tested whether CLas and wDi titer were correlated in a sub-set of these insects. Grove site had the largest effect on CLas titer. Sex had no effect on CLas titer, while higher wDi titer was correlated with CLas-uninfected insects. Our results suggest that more directed follow-up research is necessary and important to clarify whether field management tactics influence CLas titer in D. citri and to better understand gene-by-environment interactions among D. citri, wDi and CLas. Now that millions of trees in Florida have been treated with injectable formulations of oxytetracycline, which is likely to decrease bacterial populations in D. citri, this study may represent the last biologically meaningful snapshot of grove-level vector-pathogen ecology in the state during the HLB epidemic.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.556951">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.556951" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.556951">
        <p class="paperTitle">Control of cell fate specification and patterning by an ancestral microRNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.556951" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.556951" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aguilar-Cruz, A.; Flores-Sandoval, E.; Gutierrez-Ramos, X.; Oltehua-Lopez, O.; Dorantes-Acosta, A. E.; Trujillo, J. T.; Kato, H.; Ishizaki, K.; Mosher, R. A.; Dolan, L.; Grimanelli, D.; Haseloff, J.; Bowman, J. L.; Arteaga-Vazquez, M. A.</p>
        <p class="info">Score: 19.8, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.556951' target='https://doi.org/10.1101/2023.09.09.556951'> 10.1101/2023.09.09.556951</a></p>
        <p class="abstract">The formation of an organized body requires the establishment and maintenance of cells with structural and functional distinctive characteristics. A central question in developmental biology is how changes in the regulation of genes drive cell specification and patterning1. microRNAs (miRNAs) are small non-coding RNAs that regulate development through mRNA cleavage and/or translational repression2. In plants, miRNAs regulate key aspects including growth, development, stem cell maintenance, vegetative phase change, leaf morphogenesis, floral organ formation and flowering time3. Biogenesis of plant miRNAs depends on the activity of DICER-LIKE 1 (DCL1), an RNase type III endonuclease that processes double stranded RNA to give rise to mature miRNAs 4. The genomes of todays flora contain at least one bona fide copy of DCL1 5,6. Using Marchantia polymorpha -a model bryophyte that allows comparative approaches to infer characteristics of the ancestral land plant-, we demonstrate that MpDCL1a is required for the biogenesis of miRNAs and uncovered a central role for miR166/Homeodomain Zipper Class III-regulated auxin synthesis in the specification of cell identity, patterning, meristem function, laminar expansion and the development of the body in the last common ancestor of extant land plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.23.559151">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.23.559151" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.23.559151">
        <p class="paperTitle">Sequence characterization of T, Bip, and Phbw demonstrates the role of MYB-bHLH-WD40 complexes and temperature in common bean seed color pattern formation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.23.559151" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.23.559151" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parker, T.; Bolt, T.; Williams, T.; Penmetsa, R. V.; Mulube, M.; Palkovic, A.; Jochua, C. N.; Rubio Wilhelmi, M. d. M.; Lo, S.; Bornhorst, G.; Tian, L.; Kamfwa, K.; Hokin, S.; Farmer, A.; Diepenbrock, C. H.; Gepts, P.</p>
        <p class="info">Score: 21.1, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.23.559151' target='https://doi.org/10.1101/2023.09.23.559151'> 10.1101/2023.09.23.559151</a></p>
        <p class="abstract">Seed colors and color patterns are critical for the survival of wild plants and the consumer appeal of crops. In common bean, a major global staple, these patterns are also critical for determining market classes, yet the genetic and environmental control of many pigmentation patterns remains unresolved. In this study, we genetically mapped variation for three important seed pattern loci, T, Bip, and phbw, which co-segregated completely with PvTTG1, PvMYC1, and PvTT8, respectively. Proteins encoded by these genes are predicted to work together in MYB-bHLH-WD40 (MBW) complexes, propagating flavonoid biosynthesis across the seed coat. Whole-genome sequencing of 37 diverse accessions identified putative mutations in each gene, including seven unique parallel mutations in T (PvTTG1) and a non-synonymous SNP in a conserved residue in bipana (PvMYC1). A 612 bp intron deletion in phbw (PvTT8) eliminated motifs conserved since the origins of the Papilionoidea and corresponded to a 20-fold reduction in transcript abundance. Mutations in MBW candidate genes for Z (PvTT2) and Sellatus (WDR) were also identified. In multi-location field trials with seven varieties with partial seed coat patterning, pigmented seed coat area was highly unstable and correlated with temperature, with up to 11-fold differences in pigmented area between the warmest and the coolest environments. In controlled growth chamber conditions, an increase of 4 {degrees}C was sufficient to cause pigmentation on an additional 21% of the seed coat area. Our results shed light on the fundamental activation of flavonoid biosynthesis in common bean and will be instrumental for maximizing consumer appeal in this nutritious staple crop.

Summary- Seed colors and patterns are critical for the survival of wild plants, and are important in differentiating crop market classes, but the genetic control of these in the staple crop common bean (Phaseolus vulgaris) is largely unknown.

- The genetic, transcriptional, and environmental basis of common bean seed color patterning was explored through QTL mapping, whole-genome sequencing, RT-qPCR, and automated pigmentation quantification of seed grown in multi-location field trials and growth chamber environments.

- MYB-bHLH-WD40 complex-forming genes PvTTG1, PvMYC1, and PvTT8 co-segregated completely with the color patterning genes T, Bip, and phbw. Mutations were identified in each gene, including seven unique parallel mutations in T (PvTTG1), a non-synonymous SNP in a conserved residue in bipana (PvMYC1), and an intron deletion in phbw (PvTT8) eliminating highly conserved motifs and corresponding to 20-fold lower PvTT8 transcript abundance. Mutations in MBW candidate genes Z (PvTT2) and Sellatus (WDR) were also identified. In multi-location field trials, pigmented seed coat area was highly unstable and corresponded to temperature. In growth chamber conditions, an increase of 4 {degrees}C caused pigmentation on an additional 21% of the seed coat area.

- Our results highlight the critical interaction between MYB-bHLH-WD40 complex components and temperature in establishing seed pattern diversity.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
