<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The nucleotide binding domain of NRC-dependent disease resistance proteins is sufficient to activate downstream helper NLR oligomerization and immune signaling
Authors: Contreras, M. P.; Pai, H.; Thompson, R.; Claeys, J.; Adachi, H.; Kamoun, S.
Score: 31.7, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569466
Nucleotide-binding domain and leucine-rich repeat (NLR) proteins with pathogen sensor activities have evolved to initiate immune signaling by activating helper NLRs. However, the mechanisms underpinning helper NLR activation by sensor NLRs remain poorly understood.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="The nucleotide binding domain of NRC-dependent disease resistance proteins is sufficient to activate downstream helper NLR oligomerization and immune signaling
Authors: Contreras, M. P.; Pai, H.; Thompson, R.; Claeys, J.; Adachi, H.; Kamoun, S.
Score: 31.7, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569466
Nucleotide-binding domain and leucine-rich repeat (NLR) proteins with pathogen sensor activities have evolved to initiate immune signaling by activating helper NLRs. However, the mechanisms underpinning helper NLR activation by sensor NLRs remain poorly understood." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-06T10:40:04+00:00" />
<meta property="article:modified_time" content="2023-12-06T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="The nucleotide binding domain of NRC-dependent disease resistance proteins is sufficient to activate downstream helper NLR oligomerization and immune signaling
Authors: Contreras, M. P.; Pai, H.; Thompson, R.; Claeys, J.; Adachi, H.; Kamoun, S.
Score: 31.7, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569466
Nucleotide-binding domain and leucine-rich repeat (NLR) proteins with pathogen sensor activities have evolved to initiate immune signaling by activating helper NLRs. However, the mechanisms underpinning helper NLR activation by sensor NLRs remain poorly understood."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "The nucleotide binding domain of NRC-dependent disease resistance proteins is sufficient to activate downstream helper NLR oligomerization and immune signaling\nAuthors: Contreras, M. P.; Pai, H.; Thompson, R.; Claeys, J.; Adachi, H.; Kamoun, S.\nScore: 31.7, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569466\nNucleotide-binding domain and leucine-rich repeat (NLR) proteins with pathogen sensor activities have evolved to initiate immune signaling by activating helper NLRs. However, the mechanisms underpinning helper NLR activation by sensor NLRs remain poorly understood.",
  "keywords": [
    
  ],
  "articleBody": " The nucleotide binding domain of NRC-dependent disease resistance proteins is sufficient to activate downstream helper NLR oligomerization and immune signaling\nAuthors: Contreras, M. P.; Pai, H.; Thompson, R.; Claeys, J.; Adachi, H.; Kamoun, S.\nScore: 31.7, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569466\nNucleotide-binding domain and leucine-rich repeat (NLR) proteins with pathogen sensor activities have evolved to initiate immune signaling by activating helper NLRs. However, the mechanisms underpinning helper NLR activation by sensor NLRs remain poorly understood. Although coiled-coil (CC) type sensor NLRs such as the Potato virus X disease resistance protein Rx have been shown to activate the oligomerization of their downstream helpers NRC2 and NRC4, the domains involved in sensor-helper signaling are not known. Here, we show that the nucleotide binding (NB) domain within the NB-ARC of the Potato virus X disease resistance protein Rx is necessary and sufficient for oligomerization and immune signaling of downstream helper NLRs. In addition, the NB domains of the disease resistance proteins Gpa2 (cyst nematode resistance), Rpi-amr1, Rpi-amr3 (oomycete resistance) and Sw-5b (virus resistance) are also sufficient to activate their respective downstream NRC helpers. Moreover, the NB domain of Rx and its helper NRC2 form a minimal functional unit that can be transferred from solanaceous plants (lamiids) to the Campanulid species lettuce (Lactuca sativa). Our results challenge the prevailing paradigm that NLR proteins exclusively signal via their N-terminal domains and reveal a signaling activity for the NB domain of NRC-dependent sensor NLRs. We propose a model in which helper NLRs monitor the status of the NB domain of their upstream sensors.\nGenetically-clustered antifungal phytocytokines and receptor proteins function together to trigger plant immune signaling\nAuthors: Lintz, J.; Goto, Y.; Bender, K. W.; Bchini, R.; Dubrulle, G.; Cawston, E.; Zipfel, C.; Duplessis, S.; PETRE, B.\nScore: 31.2, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568785\nO_LIPhytocytokines regulate plant immunity via cell-surface receptors. Populus trichocarpa RUST INDUCED SECRETED PEPTIDE 1 (PtRISP1) exhibits an elicitor activity in poplar, as well as a direct antimicrobial activity against rust fungi. PtRISP1 gene directly clusters with a gene encoding a leucine-rich repeat receptor protein (LRR-RP), that we termed RISP- ASSOCIATED LRR-RP (PtRALR). C_LIO_LIIn this study, we used phylogenomics to characterize the RISP and RALR gene families, and functional assays to characterize RISP/RALR pairs. C_LIO_LIBoth RISP and RALR gene families specifically evolved in Salicaceae species (poplar and willow), and systematically cluster in the genomes. Two divergent RISPs, PtRISP1 and Salix purpurea RISP1 (SpRISP1), induced a reactive oxygen species (ROS) burst and mitogen- activated protein kinases (MAPKs) phosphorylation in Nicotiana benthamiana leaves expressing the respective clustered RALR. PtRISP1 triggers a rapid stomatal closure in poplar, and both PtRISP1 and SpRISP1 directly inhibit rust pathogen growth. C_LIO_LIAltogether, these results suggest that plants evolved phytocytokines with direct antimicrobial activities, and that the genes coding these phytocytokines co-evolved and physically cluster with their cognate receptors. C_LI\nggPlantmap: an R package for creation of informative and quantitative ggplot maps derived from plant images.\nAuthors: Jo, L.; Kajala, K.\nScore: 23.5, Published: 2023-12-03 DOI: 10.1101/2023.11.30.569429\nAs plant research generates an ever-growing volume of spatial quantitative data, the need for decentralized and user-friendly visualization tools to explore large and complex datasets tools becomes crucial. Existing resources, such as the Plant eFP (electronic Fluorescent Pictograph) browsers, have played a pivotal role on the communication of gene expression data across many plant species. However, although widely used by the plant research community, the Plant eFP browser lacks open and user-friendly tools for the creation of customized expression maps independently. Plant biologists with less coding experience can often encounter challenges when attempting to explore ways to communicate their own spatial quantitative data. We present 'ggPlantmap' an open-source R package designed to address this challenge by providing an easy and user-friendly method for the creation of ggplot representative maps from plant images. ggPlantmap is built in R, one of the most used languages in biology to empower plant scientists to create and customize eFP-like browsers tailored to their experimental data. Here, we provide an overview of the package and tutorials that are accessible even to users with minimal R programming experience. We hope that ggPlantmap can assist the plant science community, fostering innovation and improving our understanding of plant development and function.\nBait, not reward: CO2-enriched Nepenthes pitchers secrete toxic nectar\nAuthors: Lathika, C. C.; Sujatha, G. B.; Thomas, G.; Johnson, A. J.; Viswanathan, G.; Varghese, T. S.; Mohamed, S.; Shereefa, L. A.; Baby, S.\nScore: 17.0, Published: 2023-11-25 DOI: 10.1101/2023.11.25.568661\nNepenthes pitchers are leaf-evolved biological traps holding high levels of CO2 within them. Extrafloral nectar (EFN) secreted by Nepenthes pitchers has long been regarded as the major reward to the visiting arthropods, but its chemical constituents and their role in prey capture are least explored. Here we show that Nepenthes EFN is a toxic sugar mix devoid of key nitrogenous components (amino acids, proteins) and vitamin C. Fatty acids (cis-11-octadecenoic acid (C18:1), palmitic acid (C16:0), stearic acid (C18:0)) and their derivatives were detected in both N. khasiana peristome and lid EFNs. Both the nectars showed strong acetylcholinesterase (AChE) inhibition in vitro and in vivo. Bioactivity guided isolation revealed (+)-isoshinanolone, a naphthoquinone derivative, as the AChE inhibitor in Nepenthes EFNs. GC-headspace analysis showed the naphthoquinone, plumbagin, as the major volatile constituent in prey capturing regions of pitchers, along with the VOCs, (Z)-3-hexen-1-ol, 1-hexanol and acetoin. N. khasiana pitchers trap a wide spectrum of preys dominated by ants, whereas only limited herbivory was observed on its leaves and pitchers. Chemical (EFN, volatiles), gaseous (CO2), visual (colour, UV reflectance, fluorescence), physical (peristome geometry, surface microstructure, wettable surface-aquaplaning) and environmental (humidity, rain) factors effect prey capture in Nepenthes pitchers. These leaf-transformed traps with high growth rate and C:N ratio, reduced Rubisco activity, modified stomata, acidic pitcher fluid and absence of key nitrogenous metabolites are model systems displaying the effects of elevated CO2 within them. We testify that Nepenthes EFN is a toxic sugar bait which hinders the neuronal activity and flight of visiting arthropods. These unique traps adopt various deceptive strategies for prey capture, and our discovery abolishes the notion of Nepenthes EFN as a reward to the visiting ants and other arthropods.\nComparative transcriptomics of seed nourishing tissues: uncovering conserved and divergent pathways in seed plants\nAuthors: Florez-Rueda, A. M.; Miguel, C. M.; Figueiredo, D. D.\nScore: 13.4, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569347\nThe evolutionary and ecological success of Spermatophytes is intrinsically linked to the seed habit, which provides a protective environment for the pre-development of the new generation. This environment includes an ephemeral nourishing tissue that supports embryo growth. In gymnosperms this tissue originates from the asexual proliferation of the maternal megagametophyte, while in angiosperms it is a product of fertilization, and is called the endosperm. The emergence of these nourishing tissues is of profound evolutionary value, and they are also food staples for most of the worlds population. Here, using orthogroup inference, we provide a comparative transcriptomic analysis of seed nourishing tissues from representative species of all main angiosperm clades, including those of early diverging basal angiosperms, and of two gymnosperm representatives. Our results show that, although the structure and composition of seed nourishing tissues has seen significant divergence along evolution, there are signatures that are conserved throughout the phylogeny. Conversely, we identified processes that are exclusive to each clade, and thus illustrate their functional divergence. With this, we aimed to provide a foundation for future studies on the evolutionary history of seed nourishing structures, as well as a resource for gene discovery in new functional studies. Significance StatementWithin seeds a specialized structure is responsible for nourishing the embryo, and is thus analogous to the mammalian placenta. Moreover, these nourishing tissues are also important sources of staple foods and feed. Here, we provide novel gene expression datasets of nourishing tissues of early diverging angiosperms, and use this information for a meta-analysis to identify pathways conserved, or divergent, throughout evolution. Thus, we aim to provide a resource for gene discovery for seed biology studies.\nMorphometrics and phylogenomics of coca (Erythroxylum spp.) illuminate its reticulate evolution, with implications for taxonomy\nAuthors: Przelomska, N.; Diaz, R.; Avila, F. A.; Ballen, G. A.; Cortes-B, R.; Kistler, L.; Chitwood, D. H.; Charitonidou, M.; Renner, S. S.; Perez-Escobar, O. A.; Antonelli, A.\nScore: 7.5, Published: 2023-11-29 DOI: 10.1101/2023.11.28.569066\nSouth American coca (Erythroxylum coca and E. novogranatense) has been a key-stone crop for many Andean and Amazonian communities for at least 8,000 years. However, over the last half century, global demand for cocaine has placed this plant in the centre of armed conflict, deforestation, and explosive growth of illegal economies. While national and international agencies progress from a war on drugs policy model towards locally appropriate, data-informed strategies to tackle coca plantations, monitoring their expansion and composition remains essential. The principal means to identify coca plants is leaf morphology, yet the extent to which it is reflected in taxonomy is uncertain. Here, we analyse the consistency of the current naming system of coca and its four closest wild relatives (the coca clade), using morphometrics, phylogenomics, and population genomics. We include the name-bearing type specimens of cocas closest wild relatives E. gracilipes and E. cataractarum. Morphometrics of 342 digitized herbarium specimens show that leaf shape and size fail to reliably discriminate between species and varieties. However, the rounder and more obovate leaves of certain coca varieties could be associated with domestication syndrome of this crop. Our phylogenomic data indicate gene flow involving monophyletic clades of E. gracilipes and the E. coca clade. These results further clarify the evolution of coca and support a taxonomic framework wherein E. gracilipes is retained as a single species. Our findings have implications for the development of cost-effective genotyping methods to effectively discriminate varieties of cultural significance from high-yielding cultivars fuelling the lucrative cocaine market.\nConvergent evolution of desiccation tolerance in grasses\nAuthors: Marks, R. A.; Van Der Pas, L.; Schuster, J.; VanBuren, R.\nScore: 7.5, Published: 2023-12-01 DOI: 10.1101/2023.11.29.569285\nDesiccation tolerance has evolved repeatedly in plants as an adaptation to survive extreme environments. Plants use similar biophysical and cellular mechanisms to survive life without water, but convergence at the molecular, gene, and regulatory levels remains to be tested. Here, we explore the evolutionary mechanisms underlying the recurrent evolution of desiccation tolerance across grasses. We observed substantial overlap in gene duplication and expression associated with desiccation, and syntenic genes of shared origin are activated across species, indicative of parallel evolution. In other cases, similar metabolic pathways are induced, but using different gene sets, pointing towards phenotypic convergence. Species-specific mechanisms supplement these shared core mechanisms, underlining the complexity and diversity of evolutionary adaptations. Our findings provide insight into the evolutionary processes driving desiccation tolerance and highlight the roles of parallel mutation and complementary pathway adaptation in response to environmental challenges.\nSubfamily C7 Raf-like kinases MRK1, RAF26, and RAF39 regulate immune homeostasis and stomatal opening in Arabidopsis thaliana\nAuthors: Dias, M. G.; Doss, B.; Rawat, A. R.; Siegel, K. R.; Mahathanthrige, T.; Sklenar, J.; Derbyshire, P.; Dharmasena, T.; Cameron, E.; Zipfel, C.; Menke, F.; Monaghan, J.\nScore: 6.4, Published: 2023-12-01 DOI: 10.1101/2023.11.29.569073\nThe calcium-dependent protein kinase CPK28 is a regulator of immune homeostasis in multiple plant species. Here, we used a proteomics approach to uncover CPK28-associated proteins. We found that CPK28 associates with subfamily C7 Raf-like kinases MRK1, RAF26, and RAF39, and trans-phosphorylates RAF26 and RAF39. Metazoan Raf kinases function in mitogen-activated protein kinase (MAPK) cascades as MAPK kinase kinases (MKKKs). Although Raf-like kinases share some features with MKKKs, we found that MRK1, RAF26, and RAF39 are unable to trans-phosphorylate any of the 10 Arabidopsis MKKs. We show that MRK1, RAF26, and RAF39 localize to the cytosol and endomembranes, and we define redundant roles for these kinases in stomatal opening, immune-triggered reactive oxygen species (ROS) production, and resistance to a bacterial pathogen. Overall, our study suggests that C7 Raf-like kinases associate with and are phosphorylated by CPK28, function redundantly in stomatal immunity, and possess substrate specificities distinct from canonical MKKKs.\nSAGA1 and SAGA2 promote starch formation around proto-pyrenoids in Arabidopsis chloroplasts\nAuthors: Atkinson, N.; Stringer, R.; Mitchell, S. R.; Seung, D.; McCormick, A. J.\nScore: 17.3, Published: 2023-11-25 DOI: 10.1101/2023.11.25.568654\nThe pyrenoid is a chloroplastic microcompartment in which most algae and some terrestrial plants condense the primary carboxylase, Rubisco (ribulose-1,5-bisphosphate carboxylase/oxygenase) as part of a CO2-concentrating mechanism that improves the efficiency of CO2 capture. Engineering a pyrenoid-based CO2-concentrating mechanism (pCCM) into C3 crop plants is a promising strategy to enhance yield capacities and resilience to the changing climate. Many pyrenoids are characterized by a sheath of starch plates that is proposed to act as a barrier to limit CO2 diffusion. Recently, we have reconstituted a phase-separated proto-pyrenoid Rubisco matrix in the model C3 plant Arabidopsis thaliana using proteins from the alga with the most well studied pyrenoid, Chlamydomonas reinhardtii (1). Here we describe the impact of introducing the Chlamydomonas proteins StArch Granules Abnormal 1 (SAGA1) and SAGA2, which are associated with the regulation of pyrenoid starch biogenesis and morphology. We show that SAGA1 localizes to the proto-pyrenoid in engineered Arabidopsis plants, which results in the formation of atypical spherical starch granules enclosed within the proto-pyrenoid condensate and adjacent plate-like granules that partially cover the condensate, but without modifying the total amount of chloroplastic starch accrued. Additional expression of SAGA2 further increases the proportion of starch synthesised as adjacent plate-like granules that fully encircle the proto-pyrenoid. Our findings pave the way to assembling a diffusion barrier as part of a functional pCCM in vascular plants, whilst also advancing our understanding of the roles of SAGA1 and SAGA2 in starch sheath formation and opening novel avenues for engineering starch morphology.\nThe pathogenic coronatine toxin hijacks host redox signalling to suppress plant immunity\nAuthors: Frungillo, L.; Oka, N.; Lee, S.-U.; Nomoto, M.; Yun, B.-W.; Tada, Y.; Spoel, S. H.\nScore: 4.0, Published: 2023-12-01 DOI: 10.1101/2023.12.01.569535\nReciprocal antagonism between the hormones salicylic acid (SA) and jasmonic acid (JA) is particularly important during infection by the bacterial pathogen Pseudomonas syringae. P. syringae secretion of the virulence-promoting toxin, coronatine, is thought to suppress SA-induced immune responses by manipulating host JA signalling. Here, we report an unexpected JA-independent role of coronatine in promoting pathogen virulence. While JA induced resistance to P. syringae, coronatine promoted pathogen virulence by suppressing cellular accumulation of glutathione, a vital antioxidant required for immunity. Coronatine-mediated suppression of glutathione levels prevented activation of NPR1, a redox-sensitive master regulator of SA-responsive immune genes. Moreover, the accrual of nitric oxide (NO) restored virulence of coronatine-deficient P. syringae, but was counteracted by expression of the host S-nitrosothiol reductase, Thioredoxin h5. Thus, our findings indicate P. syringae utilises coronatine to suppresses host immunity by precisely manipulating glutathione- and NO-mediated redox signalling networks.\n",
  "wordCount" : "2431",
  "inLanguage": "en",
  "datePublished": "2023-12-06T10:40:04Z",
  "dateModified": "2023-12-06T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 6, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569466">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569466" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569466">
        <p class="paperTitle">The nucleotide binding domain of NRC-dependent disease resistance proteins is sufficient to activate downstream helper NLR oligomerization and immune signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569466" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569466" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Contreras, M. P.; Pai, H.; Thompson, R.; Claeys, J.; Adachi, H.; Kamoun, S.</p>
        <p class="info">Score: 31.7, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569466' target='https://doi.org/10.1101/2023.11.30.569466'> 10.1101/2023.11.30.569466</a></p>
        <p class="abstract">Nucleotide-binding domain and leucine-rich repeat (NLR) proteins with pathogen sensor activities have evolved to initiate immune signaling by activating helper NLRs. However, the mechanisms underpinning helper NLR activation by sensor NLRs remain poorly understood. Although coiled-coil (CC) type sensor NLRs such as the Potato virus X disease resistance protein Rx have been shown to activate the oligomerization of their downstream helpers NRC2 and NRC4, the domains involved in sensor-helper signaling are not known. Here, we show that the nucleotide binding (NB) domain within the NB-ARC of the Potato virus X disease resistance protein Rx is necessary and sufficient for oligomerization and immune signaling of downstream helper NLRs. In addition, the NB domains of the disease resistance proteins Gpa2 (cyst nematode resistance), Rpi-amr1, Rpi-amr3 (oomycete resistance) and Sw-5b (virus resistance) are also sufficient to activate their respective downstream NRC helpers. Moreover, the NB domain of Rx and its helper NRC2 form a minimal functional unit that can be transferred from solanaceous plants (lamiids) to the Campanulid species lettuce (Lactuca sativa). Our results challenge the prevailing paradigm that NLR proteins exclusively signal via their N-terminal domains and reveal a signaling activity for the NB domain of NRC-dependent sensor NLRs. We propose a model in which helper NLRs monitor the status of the NB domain of their upstream sensors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568785">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568785" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568785">
        <p class="paperTitle">Genetically-clustered antifungal phytocytokines and receptor proteins function together to trigger plant immune signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568785" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568785" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lintz, J.; Goto, Y.; Bender, K. W.; Bchini, R.; Dubrulle, G.; Cawston, E.; Zipfel, C.; Duplessis, S.; PETRE, B.</p>
        <p class="info">Score: 31.2, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568785' target='https://doi.org/10.1101/2023.11.27.568785'> 10.1101/2023.11.27.568785</a></p>
        <p class="abstract">O_LIPhytocytokines regulate plant immunity via cell-surface receptors. Populus trichocarpa RUST INDUCED SECRETED PEPTIDE 1 (PtRISP1) exhibits an elicitor activity in poplar, as well as a direct antimicrobial activity against rust fungi. PtRISP1 gene directly clusters with a gene encoding a leucine-rich repeat receptor protein (LRR-RP), that we termed RISP- ASSOCIATED LRR-RP (PtRALR).
C_LIO_LIIn this study, we used phylogenomics to characterize the RISP and RALR gene families, and functional assays to characterize RISP/RALR pairs.
C_LIO_LIBoth RISP and RALR gene families specifically evolved in Salicaceae species (poplar and willow), and systematically cluster in the genomes. Two divergent RISPs, PtRISP1 and Salix purpurea RISP1 (SpRISP1), induced a reactive oxygen species (ROS) burst and mitogen- activated protein kinases (MAPKs) phosphorylation in Nicotiana benthamiana leaves expressing the respective clustered RALR. PtRISP1 triggers a rapid stomatal closure in poplar, and both PtRISP1 and SpRISP1 directly inhibit rust pathogen growth.
C_LIO_LIAltogether, these results suggest that plants evolved phytocytokines with direct antimicrobial activities, and that the genes coding these phytocytokines co-evolved and physically cluster with their cognate receptors.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569429">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569429" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569429">
        <p class="paperTitle">ggPlantmap: an R package for creation of informative and quantitative ggplot maps derived from plant images.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569429" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569429" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jo, L.; Kajala, K.</p>
        <p class="info">Score: 23.5, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569429' target='https://doi.org/10.1101/2023.11.30.569429'> 10.1101/2023.11.30.569429</a></p>
        <p class="abstract">As plant research generates an ever-growing volume of spatial quantitative data, the need for decentralized and user-friendly visualization tools to explore large and complex datasets tools becomes crucial. Existing resources, such as the Plant eFP (electronic Fluorescent Pictograph) browsers, have played a pivotal role on the communication of gene expression data across many plant species. However, although widely used by the plant research community, the Plant eFP browser lacks open and user-friendly tools for the creation of customized expression maps independently. Plant biologists with less coding experience can often encounter challenges when attempting to explore ways to communicate their own spatial quantitative data. We present &#39;ggPlantmap&#39; an open-source R package designed to address this challenge by providing an easy and user-friendly method for the creation of ggplot representative maps from plant images. ggPlantmap is built in R, one of the most used languages in biology to empower plant scientists to create and customize eFP-like browsers tailored to their experimental data. Here, we provide an overview of the package and tutorials that are accessible even to users with minimal R programming experience. We hope that ggPlantmap can assist the plant science community, fostering innovation and improving our understanding of plant development and function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.25.568661">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.25.568661" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.25.568661">
        <p class="paperTitle">Bait, not reward: CO2-enriched Nepenthes pitchers secrete toxic nectar</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.25.568661" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.25.568661" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lathika, C. C.; Sujatha, G. B.; Thomas, G.; Johnson, A. J.; Viswanathan, G.; Varghese, T. S.; Mohamed, S.; Shereefa, L. A.; Baby, S.</p>
        <p class="info">Score: 17.0, Published: 2023-11-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.25.568661' target='https://doi.org/10.1101/2023.11.25.568661'> 10.1101/2023.11.25.568661</a></p>
        <p class="abstract">Nepenthes pitchers are leaf-evolved biological traps holding high levels of CO2 within them. Extrafloral nectar (EFN) secreted by Nepenthes pitchers has long been regarded as the major reward to the visiting arthropods, but its chemical constituents and their role in prey capture are least explored. Here we show that Nepenthes EFN is a toxic sugar mix devoid of key nitrogenous components (amino acids, proteins) and vitamin C. Fatty acids (cis-11-octadecenoic acid (C18:1), palmitic acid (C16:0), stearic acid (C18:0)) and their derivatives were detected in both N. khasiana peristome and lid EFNs. Both the nectars showed strong acetylcholinesterase (AChE) inhibition in vitro and in vivo. Bioactivity guided isolation revealed (&#43;)-isoshinanolone, a naphthoquinone derivative, as the AChE inhibitor in Nepenthes EFNs. GC-headspace analysis showed the naphthoquinone, plumbagin, as the major volatile constituent in prey capturing regions of pitchers, along with the VOCs, (Z)-3-hexen-1-ol, 1-hexanol and acetoin. N. khasiana pitchers trap a wide spectrum of preys dominated by ants, whereas only limited herbivory was observed on its leaves and pitchers. Chemical (EFN, volatiles), gaseous (CO2), visual (colour, UV reflectance, fluorescence), physical (peristome geometry, surface microstructure, wettable surface-aquaplaning) and environmental (humidity, rain) factors effect prey capture in Nepenthes pitchers. These leaf-transformed traps with high growth rate and C:N ratio, reduced Rubisco activity, modified stomata, acidic pitcher fluid and absence of key nitrogenous metabolites are model systems displaying the effects of elevated CO2 within them. We testify that Nepenthes EFN is  a toxic sugar bait which hinders the neuronal activity and flight of visiting arthropods. These unique traps adopt various deceptive strategies for prey capture, and our discovery abolishes the notion of Nepenthes EFN as a  reward to the visiting ants and other arthropods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569347">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569347" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569347">
        <p class="paperTitle">Comparative transcriptomics of seed nourishing tissues: uncovering conserved and divergent pathways in seed plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569347" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569347" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Florez-Rueda, A. M.; Miguel, C. M.; Figueiredo, D. D.</p>
        <p class="info">Score: 13.4, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569347' target='https://doi.org/10.1101/2023.11.30.569347'> 10.1101/2023.11.30.569347</a></p>
        <p class="abstract">The evolutionary and ecological success of Spermatophytes is intrinsically linked to the seed habit, which provides a protective environment for the pre-development of the new generation. This environment includes an ephemeral nourishing tissue that supports embryo growth. In gymnosperms this tissue originates from the asexual proliferation of the maternal megagametophyte, while in angiosperms it is a product of fertilization, and is called the endosperm. The emergence of these nourishing tissues is of profound evolutionary value, and they are also food staples for most of the worlds population. Here, using orthogroup inference, we provide a comparative transcriptomic analysis of seed nourishing tissues from representative species of all main angiosperm clades, including those of early diverging basal angiosperms, and of two gymnosperm representatives. Our results show that, although the structure and composition of seed nourishing tissues has seen significant divergence along evolution, there are signatures that are conserved throughout the phylogeny. Conversely, we identified processes that are exclusive to each clade, and thus illustrate their functional divergence. With this, we aimed to provide a foundation for future studies on the evolutionary history of seed nourishing structures, as well as a resource for gene discovery in new functional studies.

Significance StatementWithin seeds a specialized structure is responsible for nourishing the embryo, and is thus analogous to the mammalian placenta. Moreover, these nourishing tissues are also important sources of staple foods and feed. Here, we provide novel gene expression datasets of nourishing tissues of early diverging angiosperms, and use this information for a meta-analysis to identify pathways conserved, or divergent, throughout evolution. Thus, we aim to provide a resource for gene discovery for seed biology studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569066">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569066" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569066">
        <p class="paperTitle">Morphometrics and phylogenomics of coca (Erythroxylum spp.) illuminate its reticulate evolution, with implications for taxonomy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569066" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569066" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Przelomska, N.; Diaz, R.; Avila, F. A.; Ballen, G. A.; Cortes-B, R.; Kistler, L.; Chitwood, D. H.; Charitonidou, M.; Renner, S. S.; Perez-Escobar, O. A.; Antonelli, A.</p>
        <p class="info">Score: 7.5, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569066' target='https://doi.org/10.1101/2023.11.28.569066'> 10.1101/2023.11.28.569066</a></p>
        <p class="abstract">South American coca (Erythroxylum coca and E. novogranatense) has been a key-stone crop for many Andean and Amazonian communities for at least 8,000 years. However, over the last half century, global demand for cocaine has placed this plant in the centre of armed conflict, deforestation, and explosive growth of illegal economies. While national and international agencies progress from a  war on drugs policy model towards locally appropriate, data-informed strategies to tackle coca plantations, monitoring their expansion and composition remains essential. The principal means to identify coca plants is leaf morphology, yet the extent to which it is reflected in taxonomy is uncertain. Here, we analyse the consistency of the current naming system of coca and its four closest wild relatives (the  coca clade), using morphometrics, phylogenomics, and population genomics. We include the name-bearing type specimens of cocas closest wild relatives E. gracilipes and E. cataractarum. Morphometrics of 342 digitized herbarium specimens show that leaf shape and size fail to reliably discriminate between species and varieties. However, the rounder and more obovate leaves of certain coca varieties could be associated with domestication syndrome of this crop. Our phylogenomic data indicate gene flow involving monophyletic clades of E. gracilipes and the E. coca clade. These results further clarify the evolution of coca and support a taxonomic framework wherein E. gracilipes is retained as a single species. Our findings have implications for the development of cost-effective genotyping methods to effectively discriminate varieties of cultural significance from high-yielding cultivars fuelling the lucrative cocaine market.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569285">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569285" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569285">
        <p class="paperTitle">Convergent evolution of desiccation tolerance in grasses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569285" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569285" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Marks, R. A.; Van Der Pas, L.; Schuster, J.; VanBuren, R.</p>
        <p class="info">Score: 7.5, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569285' target='https://doi.org/10.1101/2023.11.29.569285'> 10.1101/2023.11.29.569285</a></p>
        <p class="abstract">Desiccation tolerance has evolved repeatedly in plants as an adaptation to survive extreme environments. Plants use similar biophysical and cellular mechanisms to survive life without water, but convergence at the molecular, gene, and regulatory levels remains to be tested. Here, we explore the evolutionary mechanisms underlying the recurrent evolution of desiccation tolerance across grasses. We observed substantial overlap in gene duplication and expression associated with desiccation, and syntenic genes of shared origin are activated across species, indicative of parallel evolution. In other cases, similar metabolic pathways are induced, but using different gene sets, pointing towards phenotypic convergence. Species-specific mechanisms supplement these shared core mechanisms, underlining the complexity and diversity of evolutionary adaptations. Our findings provide insight into the evolutionary processes driving desiccation tolerance and highlight the roles of parallel mutation and complementary pathway adaptation in response to environmental challenges.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569073">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569073" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569073">
        <p class="paperTitle">Subfamily C7 Raf-like kinases MRK1, RAF26, and RAF39 regulate immune homeostasis and stomatal opening in Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569073" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569073" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dias, M. G.; Doss, B.; Rawat, A. R.; Siegel, K. R.; Mahathanthrige, T.; Sklenar, J.; Derbyshire, P.; Dharmasena, T.; Cameron, E.; Zipfel, C.; Menke, F.; Monaghan, J.</p>
        <p class="info">Score: 6.4, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569073' target='https://doi.org/10.1101/2023.11.29.569073'> 10.1101/2023.11.29.569073</a></p>
        <p class="abstract">The calcium-dependent protein kinase CPK28 is a regulator of immune homeostasis in multiple plant species. Here, we used a proteomics approach to uncover CPK28-associated proteins. We found that CPK28 associates with subfamily C7 Raf-like kinases MRK1, RAF26, and RAF39, and trans-phosphorylates RAF26 and RAF39. Metazoan Raf kinases function in mitogen-activated protein kinase (MAPK) cascades as MAPK kinase kinases (MKKKs). Although Raf-like kinases share some features with MKKKs, we found that MRK1, RAF26, and RAF39 are unable to trans-phosphorylate any of the 10 Arabidopsis MKKs. We show that MRK1, RAF26, and RAF39 localize to the cytosol and endomembranes, and we define redundant roles for these kinases in stomatal opening, immune-triggered reactive oxygen species (ROS) production, and resistance to a bacterial pathogen. Overall, our study suggests that C7 Raf-like kinases associate with and are phosphorylated by CPK28, function redundantly in stomatal immunity, and possess substrate specificities distinct from canonical MKKKs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.25.568654">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.25.568654" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.25.568654">
        <p class="paperTitle">SAGA1 and SAGA2 promote starch formation around proto-pyrenoids in Arabidopsis chloroplasts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.25.568654" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.25.568654" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Atkinson, N.; Stringer, R.; Mitchell, S. R.; Seung, D.; McCormick, A. J.</p>
        <p class="info">Score: 17.3, Published: 2023-11-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.25.568654' target='https://doi.org/10.1101/2023.11.25.568654'> 10.1101/2023.11.25.568654</a></p>
        <p class="abstract">The pyrenoid is a chloroplastic microcompartment in which most algae and some terrestrial plants condense the primary carboxylase, Rubisco (ribulose-1,5-bisphosphate carboxylase/oxygenase) as part of a CO2-concentrating mechanism that improves the efficiency of CO2 capture. Engineering a pyrenoid-based CO2-concentrating mechanism (pCCM) into C3 crop plants is a promising strategy to enhance yield capacities and resilience to the changing climate. Many pyrenoids are characterized by a sheath of starch plates that is proposed to act as a barrier to limit CO2 diffusion. Recently, we have reconstituted a phase-separated  proto-pyrenoid Rubisco matrix in the model C3 plant Arabidopsis thaliana using proteins from the alga with the most well studied pyrenoid, Chlamydomonas reinhardtii (1). Here we describe the impact of introducing the Chlamydomonas proteins StArch Granules Abnormal 1 (SAGA1) and SAGA2, which are associated with the regulation of pyrenoid starch biogenesis and morphology. We show that SAGA1 localizes to the proto-pyrenoid in engineered Arabidopsis plants, which results in the formation of atypical spherical starch granules enclosed within the proto-pyrenoid condensate and adjacent plate-like granules that partially cover the condensate, but without modifying the total amount of chloroplastic starch accrued. Additional expression of SAGA2 further increases the proportion of starch synthesised as adjacent plate-like granules that fully encircle the proto-pyrenoid. Our findings pave the way to assembling a diffusion barrier as part of a functional pCCM in vascular plants, whilst also advancing our understanding of the roles of SAGA1 and SAGA2 in starch sheath formation and opening novel avenues for engineering starch morphology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569535">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569535" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569535">
        <p class="paperTitle">The pathogenic coronatine toxin hijacks host redox signalling to suppress plant immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569535" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569535" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Frungillo, L.; Oka, N.; Lee, S.-U.; Nomoto, M.; Yun, B.-W.; Tada, Y.; Spoel, S. H.</p>
        <p class="info">Score: 4.0, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569535' target='https://doi.org/10.1101/2023.12.01.569535'> 10.1101/2023.12.01.569535</a></p>
        <p class="abstract">Reciprocal antagonism between the hormones salicylic acid (SA) and jasmonic acid (JA) is particularly important during infection by the bacterial pathogen Pseudomonas syringae. P. syringae secretion of the virulence-promoting toxin, coronatine, is thought to suppress SA-induced immune responses by manipulating host JA signalling. Here, we report an unexpected JA-independent role of coronatine in promoting pathogen virulence. While JA induced resistance to P. syringae, coronatine promoted pathogen virulence by suppressing cellular accumulation of glutathione, a vital antioxidant required for immunity. Coronatine-mediated suppression of glutathione levels prevented activation of NPR1, a redox-sensitive master regulator of SA-responsive immune genes. Moreover, the accrual of nitric oxide (NO) restored virulence of coronatine-deficient P. syringae, but was counteracted by expression of the host S-nitrosothiol reductase, Thioredoxin h5. Thus, our findings indicate P. syringae utilises coronatine to suppresses host immunity by precisely manipulating glutathione- and NO-mediated redox signalling networks.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
