<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Synergistic control of chloroplast biogenesis by MYB-related and Golden2-like transcription factors
Authors: Frangedakis, E.; Yelina, N.; Billakurthi, K.; Schreier, T. B.; Dickinson, P. J.; Tomaselli, M.; Haseloff, J.; Hibberd, J. M.
Score: 9.2, Published: 2023-08-13 DOI: 10.1101/2023.08.11.552970
Chloroplast biogenesis is dependent on master regulators from the Golden2-like (GLK) family of transcription factors, but higher order glk mutants contain residual chlorophyll and therefore other proteins must also be involved. Here we identify MYB-related transcription factors as regulators of chloroplast biogenesis in the liverwort Marchantia polymorpha and angiosperm Arabidopsis thaliana.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="Synergistic control of chloroplast biogenesis by MYB-related and Golden2-like transcription factors
Authors: Frangedakis, E.; Yelina, N.; Billakurthi, K.; Schreier, T. B.; Dickinson, P. J.; Tomaselli, M.; Haseloff, J.; Hibberd, J. M.
Score: 9.2, Published: 2023-08-13 DOI: 10.1101/2023.08.11.552970
Chloroplast biogenesis is dependent on master regulators from the Golden2-like (GLK) family of transcription factors, but higher order glk mutants contain residual chlorophyll and therefore other proteins must also be involved. Here we identify MYB-related transcription factors as regulators of chloroplast biogenesis in the liverwort Marchantia polymorpha and angiosperm Arabidopsis thaliana." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-16T10:37:20+00:00" />
<meta property="article:modified_time" content="2023-08-16T10:37:20+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="Synergistic control of chloroplast biogenesis by MYB-related and Golden2-like transcription factors
Authors: Frangedakis, E.; Yelina, N.; Billakurthi, K.; Schreier, T. B.; Dickinson, P. J.; Tomaselli, M.; Haseloff, J.; Hibberd, J. M.
Score: 9.2, Published: 2023-08-13 DOI: 10.1101/2023.08.11.552970
Chloroplast biogenesis is dependent on master regulators from the Golden2-like (GLK) family of transcription factors, but higher order glk mutants contain residual chlorophyll and therefore other proteins must also be involved. Here we identify MYB-related transcription factors as regulators of chloroplast biogenesis in the liverwort Marchantia polymorpha and angiosperm Arabidopsis thaliana."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "Synergistic control of chloroplast biogenesis by MYB-related and Golden2-like transcription factors\nAuthors: Frangedakis, E.; Yelina, N.; Billakurthi, K.; Schreier, T. B.; Dickinson, P. J.; Tomaselli, M.; Haseloff, J.; Hibberd, J. M.\nScore: 9.2, Published: 2023-08-13 DOI: 10.1101/2023.08.11.552970\nChloroplast biogenesis is dependent on master regulators from the Golden2-like (GLK) family of transcription factors, but higher order glk mutants contain residual chlorophyll and therefore other proteins must also be involved. Here we identify MYB-related transcription factors as regulators of chloroplast biogenesis in the liverwort Marchantia polymorpha and angiosperm Arabidopsis thaliana.",
  "keywords": [
    
  ],
  "articleBody": " Synergistic control of chloroplast biogenesis by MYB-related and Golden2-like transcription factors\nAuthors: Frangedakis, E.; Yelina, N.; Billakurthi, K.; Schreier, T. B.; Dickinson, P. J.; Tomaselli, M.; Haseloff, J.; Hibberd, J. M.\nScore: 9.2, Published: 2023-08-13 DOI: 10.1101/2023.08.11.552970\nChloroplast biogenesis is dependent on master regulators from the Golden2-like (GLK) family of transcription factors, but higher order glk mutants contain residual chlorophyll and therefore other proteins must also be involved. Here we identify MYB-related transcription factors as regulators of chloroplast biogenesis in the liverwort Marchantia polymorpha and angiosperm Arabidopsis thaliana. In both species, double mutant alleles in MYB-Related genes show very limited chloroplast development and photosynthesis gene expression is perturbed to a greater extent than mutants of GLK. In M. polymorpha MYB-related genes act upstream of GLK, while in A. thaliana this relationship has been rewired. In both species, genes encoding enzymes of chlorophyll biosynthesis are controlled by both MYB-related and GLK proteins whilst those allowing CO2 fixation, photorespiration as well as photosystem assembly and repair require the MYB-related proteins. Thus, MYB-related and GLK genes have overlapping as well as distinct targets. We conclude that MYB-related transcription factors act with GLK to orchestrate chloroplast development in land plants.\nAlgal CO2 capture is powered by alternative electron pathways of photosynthesis\nAuthors: Peltier, G.; Stoffel, C.; Findinier, J.; Madireddi, S. K.; Dao, O.; Epting, V.; Grossman, A.; Li-Beisson, Y.; Burlacot, A.\nScore: 8.8, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552514\nOn Earth, microalgae contribute to about half of global net photosynthesis. During photosynthesis, sunlight is converted into chemical energy (ATP and NADPH) used by metabolism to convert CO2 into biomass. Alternative electron pathways of photosynthesis have been proposed to generate additional ATP that is required for sustaining CO2 fixation, but the relative importance of each pathway remains elusive. Here, we dissect and quantify the contribution of cyclic, pseudo-cyclic and chloroplast to mitochondria electron flows for their ability to sustain net photosynthesis in the microalga Chlamydomonas reinhardtii. We show that each pathway has the potential to energize substantial CO2 fixation, can compensate each other, and that the additional energy requirement to fix CO2 is more than 3 times higher than previous estimations. We further show that all pathways have very different efficiencies at energizing CO2 fixation, with the chloroplast-mitochondria interaction being the most efficient, thus laying bioenergetic foundations for biotechnological improvement of CO2 capture.\nMucilage produced by sorghum (Sorghum bicolor) aerial roots supports a nitrogen-fixing community\nAuthors: Venado, R. E.; Wilker, J.; Pankievicz, V.; Infante, V.; MacIntyre, A.; Wolf, E.; Vela, S.; Robbins, F. A.; Fernandes-Junior, P. I.; Vermerris, W.; Ane, J.-M.\nScore: 74.4, Published: 2023-08-06 DOI: 10.1101/2023.08.05.552127\nO_LISorghum (Sorghum bicolor) is gaining popularity as a sustainable energy crop due to its high biomass and potential for biofuel production. Some rare sorghum accessions develop many aerial roots that produce viscous carbohydrate-rich mucilage after rain. C_LIO_LIThis aerial root mucilage is strikingly similar to that observed in specific landraces of maize (Zea mays) from southern Mexico, which have been previously shown to host nitrogen-fixing bacteria (diazotroph). The landraces displaying these traits can reduce nitrogen-based fertilizer input, mitigating their negative environmental impacts. C_LIO_LIIn this study, we characterized the aerial root development of several sorghum accessions and successfully isolated more than 103 distinct diazotrophs from the sorghum mucilage. C_LIO_LIUsing acetylene reduction and 15N gas enrichment assays, we confirmed that sorghum plants acquire nitrogen from the atmosphere through the diazotrophic associations in the mucilage. C_LIO_LIThis sorghum symbiotic relationships with diazotrophs offer a promising avenue for nitrogen fixation, potentially diminishing reliance on synthetic fertilizers and promoting sustainable agricultural practices. C_LI\nOryza genera-specific novel Histone H4 variant predisposes H4 Lysine5 Acetylation marks to modulate salt stress responses\nAuthors: G, V. H. S.; Sotelo-Parrilla, P.; Raju, S.; Jha, S.; Gireesh, A.; Gut, F.; Kutti, V. R.; Berger, F.; Jeyaprakash, A. A.; Shivaprasad, P. V.\nScore: 24.0, Published: 2023-08-01 DOI: 10.1101/2023.07.31.551207\nParalogous variants of canonical histones guide accessibility to DNA and function as additional layers of genome regulation. Across eukaryotes, the occurrence, mechanism of action and functional significance of several variants of core histones are well known except that of histone H4. Here we show that a novel variant of H4 (H4.V), expressing tissue-specifically among members of Oryza genera, mediates specific epigenetic changes contributing majorly to salt tolerance. H4.V was incorporated to specific chromosomal locations where it blocked deposition of active histone marks. Under salt stress, large scale re-distribution of H4.V enabled incorporation of stress dependent histone H4 Lysine5 Acetylation (H4K5Ac) marks. Mis-expression of H4.V led to defects at morphological level especially in reproductive tissues, and in mounting stress responses. H4.V mediated these alterations by condensing chromatin at specific genomic regions as seen with cryo-EM structure of reconstituted H4.V containing nucleosomes. These results not only uncovered the presence of a H4 variant in plants, but also a novel chromatin regulation of stress responses that might have contributed to success of semi-aquatic Oryza members under variable water-limiting conditions. One-line summaryHistone H4 variant predisposes chromatin for stress responses O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=199 SRC=\"FIGDIR/small/551207v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (53K): org.highwire.dtl.DTLVardef@101fa0forg.highwire.dtl.DTLVardef@c7fda9org.highwire.dtl.DTLVardef@1de9a0aorg.highwire.dtl.DTLVardef@1532641_HPS_FORMAT_FIGEXP M_FIG C_FIG\nSMB controls decompartmentalization in Arabidopsis root cap cells to execute programmed cell death\nAuthors: Wang, J.; BOLLIER, N.; Andrade Buono, R.; Vahldick, H.; Lin, Z.; Feng, Q.; Hudecek, R.; JIANG, Q.; Mylle, E.; Van Damme, D.; Nowack, M. K.\nScore: 5.6, Published: 2023-08-11 DOI: 10.1101/2023.08.10.552584\nProgrammed cell death (PCD) is a fundamental cellular process crucial to development, homeostasis, and immunity in multicellular eukaryotes. In contrast to our knowledge on the regulation of diverse animal cell death subroutines, information on execution of PCD in plants remains fragmentary. Here we make use of the accessibility of the Arabidopsis thaliana root cap to visualize the execution process of developmentally controlled PCD. We identify a succession of selective decompartmentalization events and ion fluxes that are controlled by a gene regulatory network downstream of the NAC transcription factor SOMBRERO (SMB). Surprisingly, breakdown of the large central vacuole is a relatively late and variable event, preceded by an increase of intracellular calcium levels and acidification, release of mitochondrial matrix proteins, leakage of nuclear and endoplasmic reticulum lumina, and release of fluorescent membrane reporters into the cytosol. Elevated intracellular calcium levels and acidification are sufficient to trigger cell death execution specifically in cells that are rendered competent to undergo PCD by SMB activity, suggesting that these ion fluxes act as PCD-triggering signals.\nDiscovery of an isoflavone oxidative catabolic pathway in legume root microbiota\nAuthors: Aoki, N.; Shimasaki, T.; Yazaki, W.; Sato, T.; Nakayasu, M.; Ando, A.; Kishino, S.; Ogawa, J.; Masuda, S.; Shibata, A.; Shirasu, K.; Yazaki, K.; Sugiyama, A.\nScore: 8.6, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552369\nIsoflavones are major specialized metabolites found in legume plants, where they contribute to environmental adaptation. Isoflavones also play a role human health as promising therapeutic agents. This metabolite group is involved in interactions with soil microorganisms as initiation signals in rhizobial symbiosis and as modulators of the legume root microbiota. We previously reported that isoflavones enrich the Comamonadaceae, a predominant bacterial family in soybean roots, and that microorganisms in legume rhizosphere soil degrade isoflavones. However, the isoflavone catabolism pathway that underly the isoflavone-mediated legume-microbiota interactions have not yet been clarified. Here, we isolated Variovorax sp. strain V35, member of the Comamonadaceae that harbors isoflavone-degrading activity, from soybean roots and discovered a gene cluster responsible for isoflavone degradation named ifc. Strain V35 metabolizes isoflavones in a completely distinct oxidative manner from the reductive isoflavone metabolism pathway elucidated in the gut microbiota, in which resulting products enter the tricarboxylic acid cycle. The characterization of ifc mutants and heterologously expressed IFC enzymes revealed that isoflavones are catabolized via A-ring cleaving fission, which starts with hydroxylation at the 8-position of the A-ring. We further demonstrated that ifc genes are frequently found in bacterial strains isolated from legume plants, including mutualistic rhizobia, and contribute to detoxification of the antibacterial activity of isoflavones. Taken together, our findings reveal an oxidative catabolism pathway of isoflavone in the soybean root microbiota, providing molecular insights into isoflavone-mediated legume-microbiota interactions. SignificanceIsoflavones play pivotal roles in plant-environment interactions and in the maintenance and improvement of human health. Bacterial metabolism is a fundamental component of isoflavone-mediated interkingdom interactions. In the human gut, intestinal bacteria convert isoflavones into equol, a highly bioactive compound. However, the fate of isoflavones in the legume rhizosphere has not been elucidated, despite them being the key signaling molecules for nodule symbiosis and modulation of the legume root microbiota. Here, we discovered a novel isoflavone catabolism pathway in the soybean root microbiota and demonstrated the strong association between bacterial catabolic abilities and their interactions with host plants. Collectively, our findings provide new insights into bacterial isoflavone metabolism and a molecular understanding of legume-microbiota interactions.\nComparative Analysis, Diversification and Functional Validation of Plant Nucleotide-Binding Site Domain Genes\nAuthors: Hussain, A.; Khan, A. A.; Aslam, M. Q.; Nazar, A.; Zaman, N.; Amin, A.; Mahmood, M. A.; Mukhtar, M. S.; Rahman, H. U. U.; Farooq, M.; Saeed, M.; Amin, I.; Mansoor, S.\nScore: 4.3, Published: 2023-08-09 DOI: 10.1101/2023.08.09.552572\nNucleotide-binding site (NBS) domain genes are one of a superfamily of resistance genes involved in plant responses to pathogens. The current study identified presumably identified 12,820 NBS-containing genes across 34 species covering from mosses to monocots and dicots. These identified genes classified into 168 classes with several novel domain architectures patterns encompassing significant diversity among plant species. Several classical (NBS, NBS-LRR, TIR-NBS, TIR-NBS-LRR etc.) and species-specific structural patterns (TIR-NBS-TIR-Cupin_1-Cupin_1, TIR-NBS-Prenyltransf, Sugar_tr-NBS etc.) were discovered. We observed 603 orthogroups (OGs) with some core (most common orthogroups; OG0, OG1, OG2 etc.) and unique (highly specific to species; OG80, OG82 etc) OGs with tandem duplications. The expression profiling presented the putative upregulation of OG2, OG6, and OG15 in different tissues under various biotic and abiotic stresses in susceptible and tolerant plants to CLCuD. The genetic variation between susceptible (Coker 312) and tolerant (Mac7) G. hirsutum accessions identified several unique variants in NBS genes of Mac7 (6,583 varaints) and Coker312 (5,173 variants). The protein-ligand and proteins-protein interaction showed a strong interaction of some putative NBS proteins with ADP/ATP and different core proteins of cotton leaf curl disease virus. The silencing of GaNBS (OG2) in resistant cotton through virus-induced gene silencing (VIGS) demonstrated its putative role in virus tittering. The presented study will be further helpful to understand the plant adaptation mechanism.\nWOX11-mediated adventitious lateral root formation modulates tolerance of Arabidopsis to cyst nematode infections\nAuthors: Willig, J.-J.; Guarneri, N.; van Loon, T.; Wahyuni, S.; Astudillo-Estevez, I. E.; Xu, L.; Willemsen, V.; Goverse, A.; Sterken, M. G.; Lozano-Torres, J. L.; Bakker, J.; Smant, G.\nScore: 3.9, Published: 2023-08-15 DOI: 10.1101/2023.08.11.553027\nThe transcription factor WUSCHEL-RELATED HOMEOBOX 11 (WOX11) in Arabidopsis initiates the formation of adventitious lateral roots upon mechanical injury in primary roots. Root-invading nematodes also induce de novo root organogenesis leading to excessive root branching, but it is not known if this symptom of disease involves mediation by WOX11 and if it benefits the plant. Here, we show with targeted transcriptional repression and reporter gene analyses in Arabidopsis that the beet cyst nematode Heterodera schachtii activates WOX11-adventitious lateral rooting from primary roots close to infection sites. The activation of WOX11 in nematode-infected roots occurs downstream of jasmonic acid-dependent damage signaling via ETHYLENE RESPONSIVE FACTOR109, linking adventitious lateral root formation to nematode damage to host tissues. By measuring different root system components, we found that WOX11-mediated formation of adventitious lateral roots compensates for nematode-induced inhibition of primary root growth. Our observations further demonstrate that WOX11-mediated rooting reduces the impact of nematode infections on aboveground plant development and growth. Altogether, we conclude that the transcriptional regulation by WOX11 modulates root system plasticity under biotic stress, which is one of the key mechanisms underlying tolerance of Arabidopsis to cyst nematode infections.\nGreenMemTraX: A User-Friendly Software for Super-Resolution Analysis of Membrane Protein Dynamics and Nanoscale Organization in Plants\nAuthors: Rohr, L.; Ehinger, A.; Rausch, L.; Burmeister, N. G.; Meixner, A. J.; Harter, K.; Kemmerling, B.; zur Oven-Krockhaus, S.\nScore: 3.1, Published: 2023-08-11 DOI: 10.1101/2023.08.10.552827\nSuper-resolution microscopy provides deep insights into the dynamics of single membrane proteins and the nanoscale organization of plasma membrane proteins in living cells. However, in addition to plant-specific challenges in imaging applications, the elaborate analysis of single-molecule data complicates its widespread use in plant cell biological research. With GreenMemTraX, we provide a user-friendly, all-in-one, open-source software package that guides users through the steps of localization, tracking, mobility, and cluster analyses of plant plasma membrane proteins based on single particle tracking Photoactivated Localization Microscopy (sptPALM) data. GreenMemTraX dramatically simplifies data management following the FAIR (Findable, Accessible, Interoperable, Reusable) principles and increases the ease of use and reproducibility of the evaluated results. In addition, batch processing with detailed parameter output is available for all analysis paths, maintaining the consistency of raw data with the calculated results. The applicability of GreenMemTraX is demonstrated by evaluating sptPALM data for two representative plant plasma membrane proteins, namely the brassinosteroid receptor Brassinosteroid Insensitive 1 (BRI1) and the Receptor-Like Protein 44 (RLP44). Both were tested with the photoconvertible or photoactivatable fluorophores mEos3.2, PA-GFP, and PATagRFP and expressed in the Nicotiana benthamiana transient expression system and in stably transformed Arabidopsis thaliana plants. To facilitate plant applications, GreenMemTraX provides a number of preset parameter settings to address plant-specific obstacles.\nThe Arabidopsis SR45 splicing factor bridges the splicing machinery and the exon-exon junction complex\nAuthors: Fanara, S.; Schloesser, M.; Joris, M.; Vandevenne, M.; Hanikenne, M.; Motte, P.\nScore: 3.6, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552250\nThe Arabidopsis splicing factor serine/arginine-rich 45 (SR45) contributes to several biological processes. The sr45-1 loss-of-function mutant exhibits delayed root development, late flowering, unusual numbers of floral organs, shorter siliques with decreased seed sets, narrower leaves and petals, and altered metal distribution. Here, we studied the function of each of SR45 domains by examining their involvement in: (i) the spatial distribution of SR45, (ii) the establishment of a protein-protein interaction network including spliceosomal and exon-exon junction complex (EJC) components, and (iii) the RNA binding specificity. We report that the endogenous SR45 promoter is active during vegetative and reproductive growth, and that the SR45 protein localizes in the nucleus. We demonstrate that the C-terminal arginine/serine-rich domain is a determinant of nuclear localization. We show that the SR45 RNA recognition motif (RRM) domain specifically binds purine-rich RNA motifs via three residues (H101, H141, Y143), and is also involved in protein-protein interactions. We further show that SR45 bridges both mRNA splicing and surveillance machineries as a partner of EJC core components and peripheral factors, which requires phosphoresidues likely phosphorylated by kinases from both CLK and SRPK families. Our findings provide insights into the contribution of each SR45 domain to both spliceosome and EJC assemblies. HighlightThe nuclear localization of the Arabidopsis SR45 splicing factor and its ability to contact novel protein partners in planta and to specifically bind RNA motifs is controlled by critical residues.\n",
  "wordCount" : "2456",
  "inLanguage": "en",
  "datePublished": "2023-08-16T10:37:20Z",
  "dateModified": "2023-08-16T10:37:20Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta"><span>updated on August 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552970">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552970" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552970">
        <p class="paperTitle">Synergistic control of chloroplast biogenesis by MYB-related and Golden2-like transcription factors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552970" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552970" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Frangedakis, E.; Yelina, N.; Billakurthi, K.; Schreier, T. B.; Dickinson, P. J.; Tomaselli, M.; Haseloff, J.; Hibberd, J. M.</p>
        <p class="info">Score: 9.2, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552970' target='https://doi.org/10.1101/2023.08.11.552970'> 10.1101/2023.08.11.552970</a></p>
        <p class="abstract">Chloroplast biogenesis is dependent on master regulators from the Golden2-like (GLK) family of transcription factors, but higher order glk mutants contain residual chlorophyll and therefore other proteins must also be involved. Here we identify MYB-related transcription factors as regulators of chloroplast biogenesis in the liverwort Marchantia polymorpha and angiosperm Arabidopsis thaliana. In both species, double mutant alleles in MYB-Related genes show very limited chloroplast development and photosynthesis gene expression is perturbed to a greater extent than mutants of GLK. In M. polymorpha MYB-related genes act upstream of GLK, while in A. thaliana this relationship has been rewired. In both species, genes encoding enzymes of chlorophyll biosynthesis are controlled by both MYB-related and GLK proteins whilst those allowing CO2 fixation, photorespiration as well as photosystem assembly and repair require the MYB-related proteins. Thus, MYB-related and GLK genes have overlapping as well as distinct targets. We conclude that MYB-related transcription factors act with GLK to orchestrate chloroplast development in land plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552514">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552514" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552514">
        <p class="paperTitle">Algal CO2 capture is powered by alternative electron pathways of photosynthesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552514" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552514" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peltier, G.; Stoffel, C.; Findinier, J.; Madireddi, S. K.; Dao, O.; Epting, V.; Grossman, A.; Li-Beisson, Y.; Burlacot, A.</p>
        <p class="info">Score: 8.8, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552514' target='https://doi.org/10.1101/2023.08.08.552514'> 10.1101/2023.08.08.552514</a></p>
        <p class="abstract">On Earth, microalgae contribute to about half of global net photosynthesis. During photosynthesis, sunlight is converted into chemical energy (ATP and NADPH) used by metabolism to convert CO2 into biomass. Alternative electron pathways of photosynthesis have been proposed to generate additional ATP that is required for sustaining CO2 fixation, but the relative importance of each pathway remains elusive. Here, we dissect and quantify the contribution of cyclic, pseudo-cyclic and chloroplast to mitochondria electron flows for their ability to sustain net photosynthesis in the microalga Chlamydomonas reinhardtii. We show that each pathway has the potential to energize substantial CO2 fixation, can compensate each other, and that the additional energy requirement to fix CO2 is more than 3 times higher than previous estimations. We further show that all pathways have very different efficiencies at energizing CO2 fixation, with the chloroplast-mitochondria interaction being the most efficient, thus laying bioenergetic foundations for biotechnological improvement of CO2 capture.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.552127">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.552127" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.552127">
        <p class="paperTitle">Mucilage produced by sorghum (Sorghum bicolor) aerial roots supports a nitrogen-fixing community</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.552127" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.552127" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Venado, R. E.; Wilker, J.; Pankievicz, V.; Infante, V.; MacIntyre, A.; Wolf, E.; Vela, S.; Robbins, F. A.; Fernandes-Junior, P. I.; Vermerris, W.; Ane, J.-M.</p>
        <p class="info">Score: 74.4, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.552127' target='https://doi.org/10.1101/2023.08.05.552127'> 10.1101/2023.08.05.552127</a></p>
        <p class="abstract">O_LISorghum (Sorghum bicolor) is gaining popularity as a sustainable energy crop due to its high biomass and potential for biofuel production. Some rare sorghum accessions develop many aerial roots that produce viscous carbohydrate-rich mucilage after rain.
C_LIO_LIThis aerial root mucilage is strikingly similar to that observed in specific landraces of maize (Zea mays) from southern Mexico, which have been previously shown to host nitrogen-fixing bacteria (diazotroph). The landraces displaying these traits can reduce nitrogen-based fertilizer input, mitigating their negative environmental impacts.
C_LIO_LIIn this study, we characterized the aerial root development of several sorghum accessions and successfully isolated more than 103 distinct diazotrophs from the sorghum mucilage.
C_LIO_LIUsing acetylene reduction and 15N gas enrichment assays, we confirmed that sorghum plants acquire nitrogen from the atmosphere through the diazotrophic associations in the mucilage.
C_LIO_LIThis sorghum symbiotic relationships with diazotrophs offer a promising avenue for nitrogen fixation, potentially diminishing reliance on synthetic fertilizers and promoting sustainable agricultural practices.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551207">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551207" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551207">
        <p class="paperTitle">Oryza genera-specific novel Histone H4 variant predisposes H4 Lysine5 Acetylation marks to modulate salt stress responses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551207" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551207" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: G, V. H. S.; Sotelo-Parrilla, P.; Raju, S.; Jha, S.; Gireesh, A.; Gut, F.; Kutti, V. R.; Berger, F.; Jeyaprakash, A. A.; Shivaprasad, P. V.</p>
        <p class="info">Score: 24.0, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551207' target='https://doi.org/10.1101/2023.07.31.551207'> 10.1101/2023.07.31.551207</a></p>
        <p class="abstract">Paralogous variants of canonical histones guide accessibility to DNA and function as additional layers of genome regulation. Across eukaryotes, the occurrence, mechanism of action and functional significance of several variants of core histones are well known except that of histone H4. Here we show that a novel variant of H4 (H4.V), expressing tissue-specifically among members of Oryza genera, mediates specific epigenetic changes contributing majorly to salt tolerance. H4.V was incorporated to specific chromosomal locations where it blocked deposition of active histone marks. Under salt stress, large scale re-distribution of H4.V enabled incorporation of stress dependent histone H4 Lysine5 Acetylation (H4K5Ac) marks. Mis-expression of H4.V led to defects at morphological level especially in reproductive tissues, and in mounting stress responses. H4.V mediated these alterations by condensing chromatin at specific genomic regions as seen with cryo-EM structure of reconstituted H4.V containing nucleosomes. These results not only uncovered the presence of a H4 variant in plants, but also a novel chromatin regulation of stress responses that might have contributed to success of semi-aquatic Oryza members under variable water-limiting conditions.

One-line summaryHistone H4 variant predisposes chromatin for stress responses



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=199 SRC=&#34;FIGDIR/small/551207v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (53K):
org.highwire.dtl.DTLVardef@101fa0forg.highwire.dtl.DTLVardef@c7fda9org.highwire.dtl.DTLVardef@1de9a0aorg.highwire.dtl.DTLVardef@1532641_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552584">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552584" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552584">
        <p class="paperTitle">SMB controls decompartmentalization in Arabidopsis root cap cells to execute programmed cell death</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552584" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552584" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, J.; BOLLIER, N.; Andrade Buono, R.; Vahldick, H.; Lin, Z.; Feng, Q.; Hudecek, R.; JIANG, Q.; Mylle, E.; Van Damme, D.; Nowack, M. K.</p>
        <p class="info">Score: 5.6, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552584' target='https://doi.org/10.1101/2023.08.10.552584'> 10.1101/2023.08.10.552584</a></p>
        <p class="abstract">Programmed cell death (PCD) is a fundamental cellular process crucial to development, homeostasis, and immunity in multicellular eukaryotes. In contrast to our knowledge on the regulation of diverse animal cell death subroutines, information on execution of PCD in plants remains fragmentary. Here we make use of the accessibility of the Arabidopsis thaliana root cap to visualize the execution process of developmentally controlled PCD. We identify a succession of selective decompartmentalization events and ion fluxes that are controlled by a gene regulatory network downstream of the NAC transcription factor SOMBRERO (SMB). Surprisingly, breakdown of the large central vacuole is a relatively late and variable event, preceded by an increase of intracellular calcium levels and acidification, release of mitochondrial matrix proteins, leakage of nuclear and endoplasmic reticulum lumina, and release of fluorescent membrane reporters into the cytosol. Elevated intracellular calcium levels and acidification are sufficient to trigger cell death execution specifically in cells that are rendered competent to undergo PCD by SMB activity, suggesting that these ion fluxes act as PCD-triggering signals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552369">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552369" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552369">
        <p class="paperTitle">Discovery of an isoflavone oxidative catabolic pathway in legume root microbiota</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552369" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552369" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aoki, N.; Shimasaki, T.; Yazaki, W.; Sato, T.; Nakayasu, M.; Ando, A.; Kishino, S.; Ogawa, J.; Masuda, S.; Shibata, A.; Shirasu, K.; Yazaki, K.; Sugiyama, A.</p>
        <p class="info">Score: 8.6, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552369' target='https://doi.org/10.1101/2023.08.07.552369'> 10.1101/2023.08.07.552369</a></p>
        <p class="abstract">Isoflavones are major specialized metabolites found in legume plants, where they contribute to environmental adaptation. Isoflavones also play a role human health as promising therapeutic agents. This metabolite group is involved in interactions with soil microorganisms as initiation signals in rhizobial symbiosis and as modulators of the legume root microbiota. We previously reported that isoflavones enrich the Comamonadaceae, a predominant bacterial family in soybean roots, and that microorganisms in legume rhizosphere soil degrade isoflavones. However, the isoflavone catabolism pathway that underly the isoflavone-mediated legume-microbiota interactions have not yet been clarified. Here, we isolated Variovorax sp. strain V35, member of the Comamonadaceae that harbors isoflavone-degrading activity, from soybean roots and discovered a gene cluster responsible for isoflavone degradation named ifc. Strain V35 metabolizes isoflavones in a completely distinct oxidative manner from the reductive isoflavone metabolism pathway elucidated in the gut microbiota, in which resulting products enter the tricarboxylic acid cycle. The characterization of ifc mutants and heterologously expressed IFC enzymes revealed that isoflavones are catabolized via A-ring cleaving fission, which starts with hydroxylation at the 8-position of the A-ring. We further demonstrated that ifc genes are frequently found in bacterial strains isolated from legume plants, including mutualistic rhizobia, and contribute to detoxification of the antibacterial activity of isoflavones. Taken together, our findings reveal an oxidative catabolism pathway of isoflavone in the soybean root microbiota, providing molecular insights into isoflavone-mediated legume-microbiota interactions.

SignificanceIsoflavones play pivotal roles in plant-environment interactions and in the maintenance and improvement of human health. Bacterial metabolism is a fundamental component of isoflavone-mediated interkingdom interactions. In the human gut, intestinal bacteria convert isoflavones into equol, a highly bioactive compound. However, the fate of isoflavones in the legume rhizosphere has not been elucidated, despite them being the key signaling molecules for nodule symbiosis and modulation of the legume root microbiota. Here, we discovered a novel isoflavone catabolism pathway in the soybean root microbiota and demonstrated the strong association between bacterial catabolic abilities and their interactions with host plants. Collectively, our findings provide new insights into bacterial isoflavone metabolism and a molecular understanding of legume-microbiota interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552572">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552572" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552572">
        <p class="paperTitle">Comparative Analysis, Diversification and Functional Validation of Plant Nucleotide-Binding Site Domain Genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552572" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552572" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hussain, A.; Khan, A. A.; Aslam, M. Q.; Nazar, A.; Zaman, N.; Amin, A.; Mahmood, M. A.; Mukhtar, M. S.; Rahman, H. U. U.; Farooq, M.; Saeed, M.; Amin, I.; Mansoor, S.</p>
        <p class="info">Score: 4.3, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552572' target='https://doi.org/10.1101/2023.08.09.552572'> 10.1101/2023.08.09.552572</a></p>
        <p class="abstract">Nucleotide-binding site (NBS) domain genes are one of a superfamily of resistance genes involved in plant responses to pathogens. The current study identified presumably identified 12,820 NBS-containing genes across 34 species covering from mosses to monocots and dicots. These identified genes classified into 168 classes with several novel domain architectures patterns encompassing significant diversity among plant species. Several classical (NBS, NBS-LRR, TIR-NBS, TIR-NBS-LRR etc.) and species-specific structural patterns (TIR-NBS-TIR-Cupin_1-Cupin_1, TIR-NBS-Prenyltransf, Sugar_tr-NBS etc.) were discovered. We observed 603 orthogroups (OGs) with some core (most common orthogroups; OG0, OG1, OG2 etc.) and unique (highly specific to species; OG80, OG82 etc) OGs with tandem duplications. The expression profiling presented the putative upregulation of OG2, OG6, and OG15 in different tissues under various biotic and abiotic stresses in susceptible and tolerant plants to CLCuD. The genetic variation between susceptible (Coker 312) and tolerant (Mac7) G. hirsutum accessions identified several unique variants in NBS genes of Mac7 (6,583 varaints) and Coker312 (5,173 variants). The protein-ligand and proteins-protein interaction showed a strong interaction of some putative NBS proteins with ADP/ATP and different core proteins of cotton leaf curl disease virus. The silencing of GaNBS (OG2) in resistant cotton through virus-induced gene silencing (VIGS) demonstrated its putative role in virus tittering. The presented study will be further helpful to understand the plant adaptation mechanism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553027">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553027" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553027">
        <p class="paperTitle">WOX11-mediated adventitious lateral root formation modulates tolerance of Arabidopsis to cyst nematode infections</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553027" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553027" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Willig, J.-J.; Guarneri, N.; van Loon, T.; Wahyuni, S.; Astudillo-Estevez, I. E.; Xu, L.; Willemsen, V.; Goverse, A.; Sterken, M. G.; Lozano-Torres, J. L.; Bakker, J.; Smant, G.</p>
        <p class="info">Score: 3.9, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553027' target='https://doi.org/10.1101/2023.08.11.553027'> 10.1101/2023.08.11.553027</a></p>
        <p class="abstract">The transcription factor WUSCHEL-RELATED HOMEOBOX 11 (WOX11) in Arabidopsis initiates the formation of adventitious lateral roots upon mechanical injury in primary roots. Root-invading nematodes also induce de novo root organogenesis leading to excessive root branching, but it is not known if this symptom of disease involves mediation by WOX11 and if it benefits the plant. Here, we show with targeted transcriptional repression and reporter gene analyses in Arabidopsis that the beet cyst nematode Heterodera schachtii activates WOX11-adventitious lateral rooting from primary roots close to infection sites. The activation of WOX11 in nematode-infected roots occurs downstream of jasmonic acid-dependent damage signaling via ETHYLENE RESPONSIVE FACTOR109, linking adventitious lateral root formation to nematode damage to host tissues. By measuring different root system components, we found that WOX11-mediated formation of adventitious lateral roots compensates for nematode-induced inhibition of primary root growth. Our observations further demonstrate that WOX11-mediated rooting reduces the impact of nematode infections on aboveground plant development and growth. Altogether, we conclude that the transcriptional regulation by WOX11 modulates root system plasticity under biotic stress, which is one of the key mechanisms underlying tolerance of Arabidopsis to cyst nematode infections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552827">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552827" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552827">
        <p class="paperTitle">GreenMemTraX: A User-Friendly Software for Super-Resolution Analysis of Membrane Protein Dynamics and Nanoscale Organization in Plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552827" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552827" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rohr, L.; Ehinger, A.; Rausch, L.; Burmeister, N. G.; Meixner, A. J.; Harter, K.; Kemmerling, B.; zur Oven-Krockhaus, S.</p>
        <p class="info">Score: 3.1, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552827' target='https://doi.org/10.1101/2023.08.10.552827'> 10.1101/2023.08.10.552827</a></p>
        <p class="abstract">Super-resolution microscopy provides deep insights into the dynamics of single membrane proteins and the nanoscale organization of plasma membrane proteins in living cells. However, in addition to plant-specific challenges in imaging applications, the elaborate analysis of single-molecule data complicates its widespread use in plant cell biological research. With GreenMemTraX, we provide a user-friendly, all-in-one, open-source software package that guides users through the steps of localization, tracking, mobility, and cluster analyses of plant plasma membrane proteins based on single particle tracking Photoactivated Localization Microscopy (sptPALM) data. GreenMemTraX dramatically simplifies data management following the FAIR (Findable, Accessible, Interoperable, Reusable) principles and increases the ease of use and reproducibility of the evaluated results. In addition, batch processing with detailed parameter output is available for all analysis paths, maintaining the consistency of raw data with the calculated results. The applicability of GreenMemTraX is demonstrated by evaluating sptPALM data for two representative plant plasma membrane proteins, namely the brassinosteroid receptor Brassinosteroid Insensitive 1 (BRI1) and the Receptor-Like Protein 44 (RLP44). Both were tested with the photoconvertible or photoactivatable fluorophores mEos3.2, PA-GFP, and PATagRFP and expressed in the Nicotiana benthamiana transient expression system and in stably transformed Arabidopsis thaliana plants. To facilitate plant applications, GreenMemTraX provides a number of preset parameter settings to address plant-specific obstacles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552250">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552250" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552250">
        <p class="paperTitle">The Arabidopsis SR45 splicing factor bridges the splicing machinery and the exon-exon junction complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552250" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552250" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fanara, S.; Schloesser, M.; Joris, M.; Vandevenne, M.; Hanikenne, M.; Motte, P.</p>
        <p class="info">Score: 3.6, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552250' target='https://doi.org/10.1101/2023.08.07.552250'> 10.1101/2023.08.07.552250</a></p>
        <p class="abstract">The Arabidopsis splicing factor serine/arginine-rich 45 (SR45) contributes to several biological processes. The sr45-1 loss-of-function mutant exhibits delayed root development, late flowering, unusual numbers of floral organs, shorter siliques with decreased seed sets, narrower leaves and petals, and altered metal distribution. Here, we studied the function of each of SR45 domains by examining their involvement in: (i) the spatial distribution of SR45, (ii) the establishment of a protein-protein interaction network including spliceosomal and exon-exon junction complex (EJC) components, and (iii) the RNA binding specificity. We report that the endogenous SR45 promoter is active during vegetative and reproductive growth, and that the SR45 protein localizes in the nucleus. We demonstrate that the C-terminal arginine/serine-rich domain is a determinant of nuclear localization. We show that the SR45 RNA recognition motif (RRM) domain specifically binds purine-rich RNA motifs via three residues (H101, H141, Y143), and is also involved in protein-protein interactions. We further show that SR45 bridges both mRNA splicing and surveillance machineries as a partner of EJC core components and peripheral factors, which requires phosphoresidues likely phosphorylated by kinases from both CLK and SRPK families. Our findings provide insights into the contribution of each SR45 domain to both spliceosome and EJC assemblies.

HighlightThe nuclear localization of the Arabidopsis SR45 splicing factor and its ability to contact novel protein partners in planta and to specifically bind RNA motifs is controlled by critical residues.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
