<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="AvrSr27 is a zinc-bound effector with a modular structure important for immune recognition
Authors: Outram, M. A.; Chen, J.; Broderick, S.; Li, Z.; Aditya, S.; Tasneem, N.; Arndell, T.; Blundell, C.; Ericsson, D. J.; Figueroa, M.; Sperschneider, J.; Dodds, P.; Williams, S. J.
Score: 25.9, Published: 2023-11-21 DOI: 10.1101/2023.11.21.567997
Stem rust, caused by the fungal pathogen Puccinia graminis f. sp.tritici (Pgt) is a major threat for wheat production and global food security.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="AvrSr27 is a zinc-bound effector with a modular structure important for immune recognition
Authors: Outram, M. A.; Chen, J.; Broderick, S.; Li, Z.; Aditya, S.; Tasneem, N.; Arndell, T.; Blundell, C.; Ericsson, D. J.; Figueroa, M.; Sperschneider, J.; Dodds, P.; Williams, S. J.
Score: 25.9, Published: 2023-11-21 DOI: 10.1101/2023.11.21.567997
Stem rust, caused by the fungal pathogen Puccinia graminis f. sp.tritici (Pgt) is a major threat for wheat production and global food security." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-29T10:40:14+00:00" />
<meta property="article:modified_time" content="2023-11-29T10:40:14+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="AvrSr27 is a zinc-bound effector with a modular structure important for immune recognition
Authors: Outram, M. A.; Chen, J.; Broderick, S.; Li, Z.; Aditya, S.; Tasneem, N.; Arndell, T.; Blundell, C.; Ericsson, D. J.; Figueroa, M.; Sperschneider, J.; Dodds, P.; Williams, S. J.
Score: 25.9, Published: 2023-11-21 DOI: 10.1101/2023.11.21.567997
Stem rust, caused by the fungal pathogen Puccinia graminis f. sp.tritici (Pgt) is a major threat for wheat production and global food security."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "AvrSr27 is a zinc-bound effector with a modular structure important for immune recognition\nAuthors: Outram, M. A.; Chen, J.; Broderick, S.; Li, Z.; Aditya, S.; Tasneem, N.; Arndell, T.; Blundell, C.; Ericsson, D. J.; Figueroa, M.; Sperschneider, J.; Dodds, P.; Williams, S. J.\nScore: 25.9, Published: 2023-11-21 DOI: 10.1101/2023.11.21.567997\nStem rust, caused by the fungal pathogen Puccinia graminis f. sp.tritici (Pgt) is a major threat for wheat production and global food security.",
  "keywords": [
    
  ],
  "articleBody": " AvrSr27 is a zinc-bound effector with a modular structure important for immune recognition\nAuthors: Outram, M. A.; Chen, J.; Broderick, S.; Li, Z.; Aditya, S.; Tasneem, N.; Arndell, T.; Blundell, C.; Ericsson, D. J.; Figueroa, M.; Sperschneider, J.; Dodds, P.; Williams, S. J.\nScore: 25.9, Published: 2023-11-21 DOI: 10.1101/2023.11.21.567997\nStem rust, caused by the fungal pathogen Puccinia graminis f. sp.tritici (Pgt) is a major threat for wheat production and global food security. Central to the success of Pgt is the secretion of proteinaceous effectors that promote infection and colonisation, while immunity in wheat is driven by receptor-mediated recognition of these effectors resulting in pathogen avirulence. Here, we report the crystal structure of the cysteine-rich effector AvrSr27, the third experimentally derived structure of a Pgt effector. The AvrSr27 structure reveals a novel {beta}-strand rich modular fold consisting of two structurally similar domains and confirms the poor prediction we obtained from the AlphaFold2-derived model. The highly prevalent cysteine residues within the protein facilitate the co-ordination of 4 zinc molecules. Utilising the structure, we show that the N-terminal domain of AvrSr27 is sufficient for immune recognition and interaction by Sr27. The 7-cys motif sequence in each AvrSr27 domain, which facilitates zinc binding, was also found in two haustorially-expressed, structurally homologous Pgt proteins. Remarkably, despite relatively low sequence identity, we show that these proteins can associate with Sr27 and trigger cell death in heterologous systems and wheat protoplasts, albeit weaker than AvrSr27. Collectively, our findings have important implications for the field embarking on bespoke engineering of immunity receptors as solutions to plant disease.\nComparative mutant analyses reveal a novel mechanism of ARF regulation in land plants\nAuthors: Prigge, M. J.; Morffy, N.; De Neve, A.; Szutu, W.; Juarez, M. J. A.; Johnson, K.; Do, N.; Lavy, M.; Hake, S.; Strader, L.; Estelle, M.; Richardson, A. E.\nScore: 18.5, Published: 2023-11-13 DOI: 10.1101/2023.11.09.566459\nA major challenge in plant biology is to understand how the plant hormone auxin regulates diverse transcriptional responses throughout development, in different environments, and in different species. The answer may lie in the specific complement of auxin signaling components in each cell. The balance between activators (class-A AUXIN RESPONSE FACTORS) and repressors (class-B ARFs) is particularly important. It is unclear how this balance is achieved. Through comparative analysis of novel, dominant mutants in maize and the moss Physcomitrium patens, we have discovered a [~]500-million-year-old mechanism of class-B ARF protein level regulation, important in determining cell fate decisions across land plants. Thus, our results add a key piece to the puzzle of how auxin regulates plant development.\nCrosstalk between salicylic acid signaling and the circadian clock promotes an effective immune response\nAuthors: Fraser, O. J. P.; Cargill, S. J.; Spoel, S. H.; van Ooijen, G.\nScore: 13.8, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568095\nThe rotation of Earth creates a cycle of day and night, leading to predictable changes in environmental conditions. The circadian clock synchronizes an organism with these environmental changes and alters their physiology in anticipation. Prediction of the probable timing of pathogen infection enables plants to prime their immune system without wasting resources or sacrificing growth. Here, we explore the relationship between the immune hormone salicylic acid (SA), and the circadian clock in Arabidopsis. We found that SA altered circadian rhythmicity through the SA receptor and master transcriptional coactivator, NPR1. Reciprocally, the circadian clock gates SA-induced expression of NPR1-dependent immune genes. Furthermore, the clock gene CCA1 is essential for SA-induced immunity to the major bacterial plant pathogen Pseudomonas syringae. These results reveal new interactions between the circadian clock and SA signaling which produce an effective immune response. Understanding how and why the immune response in plants is linked to the circadian clock is crucial in working towards improved crop productivity.\nSAGA1 and SAGA2 promote starch formation around proto-pyrenoids in Arabidopsis chloroplasts\nAuthors: Atkinson, N.; Stringer, R.; Mitchell, S. R.; Seung, D.; McCormick, A. J.\nScore: 13.2, Published: 2023-11-25 DOI: 10.1101/2023.11.25.568654\nThe pyrenoid is a chloroplastic microcompartment in which most algae and some terrestrial plants condense the primary carboxylase, Rubisco (ribulose-1,5-bisphosphate carboxylase/oxygenase) as part of a CO2-concentrating mechanism that improves the efficiency of CO2 capture. Engineering a pyrenoid-based CO2-concentrating mechanism (pCCM) into C3 crop plants is a promising strategy to enhance yield capacities and resilience to the changing climate. Many pyrenoids are characterized by a sheath of starch plates that is proposed to act as a barrier to limit CO2 diffusion. Recently, we have reconstituted a phase-separated proto-pyrenoid Rubisco matrix in the model C3 plant Arabidopsis thaliana using proteins from the alga with the most well studied pyrenoid, Chlamydomonas reinhardtii (1). Here we describe the impact of introducing the Chlamydomonas proteins StArch Granules Abnormal 1 (SAGA1) and SAGA2, which are associated with the regulation of pyrenoid starch biogenesis and morphology. We show that SAGA1 localizes to the proto-pyrenoid in engineered Arabidopsis plants, which results in the formation of atypical spherical starch granules enclosed within the proto-pyrenoid condensate and adjacent plate-like granules that partially cover the condensate, but without modifying the total amount of chloroplastic starch accrued. Additional expression of SAGA2 further increases the proportion of starch synthesised as adjacent plate-like granules that fully encircle the proto-pyrenoid. Our findings pave the way to assembling a diffusion barrier as part of a functional pCCM in vascular plants, whilst also advancing our understanding of the roles of SAGA1 and SAGA2 in starch sheath formation and opening novel avenues for engineering starch morphology.\nFast and efficient root phenotyping via pose estimation\nAuthors: Berrigan, E. M.; Wang, L.; Carrillo, H.; Echegoyen, K.; Kappes, M.; Torres, J.; Ai-Perreira, A.; McCoy, E.; Shane, E.; Copeland, C. D.; Ragel, L.; Georgousakis, C.; Lee, S.; Reynolds, D.; Talgo, A.; Gonzalez, J.; Zhang, L.; Rajurkar, A. B.; Ruiz, M.; Daniels, E.; Maree, L.; Pariyar, S.; Busch, W.; Pereira, T. D.\nScore: 12.5, Published: 2023-11-21 DOI: 10.1101/2023.11.20.567949\nImage segmentation is commonly used to estimate the location and shape of plants and their external structures. Segmentation masks are then used to localize landmarks of interest and compute other geometric features that correspond to the plants phenotype. Despite its prevalence, segmentation-based approaches are laborious (requiring extensive annotation to train), and error-prone (derived geometric features are sensitive to instance mask integrity). Here we present a segmentation-free approach which leverages deep learning-based landmark detection and grouping, also known as pose estimation. We use a tool originally developed for animal motion capture called SLEAP (Social LEAP Estimates Animal Poses) to automate the detection of distinct morphological landmarks on plant roots. Using a gel cylinder imaging system across multiple species, we show that our approach can reliably and efficiently recover root system topology at high accuracy, few annotated samples, and faster speed than segmentation-based approaches. In order to make use of this landmark-based representation for root phenotyping, we developed a Python library (sleap-roots) for trait extraction directly comparable to existing segmentation-based analysis software. We show that landmark-derived root traits are highly accurate and can be used for common downstream tasks including genotype classification and unsupervised trait mapping. Altogether, this work establishes the validity and advantages of pose estimation-based plant phenotyping. To facilitate adoption of this easy-to-use tool and to encourage further development, we make sleap-roots, all training data, models, and trait extraction code available at: https://github.com/talmolab/sleap-roots and https://osf.io/k7j9g/.\nCooperativity and additivity in plant enhancers\nAuthors: Jores, T.; Tonnies, J.; Mueth, N. A.; Romanowski, A.; Fields, S.; Cuperus, J.; Queitsch, C.\nScore: 11.8, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568158\nEnhancers are cis-regulatory elements that shape gene expression in response to numerous developmental and environmental cues. In animals, several models have been proposed to explain how enhancers integrate the activity of multiple transcription factors. However, it remains largely unknown how plant enhancers integrate transcription factor activity. Here, we use Plant STARR-seq to characterize three light-responsive plant enhancers--AB80, Cab-1, and rbcS-E9--derived from genes active in photosynthesis. Saturation mutagenesis reveals mutations, many of which cluster in short regions, that strongly reduce enhancer activity in the light, in the dark or in both conditions. When tested in the light, these mutation-sensitive regions do not function on their own; rather, cooperative interactions with other such regions are required for full activity. Epistatic interactions occur between mutations in adjacent mutation-sensitive regions, and the spacing and order of mutation-sensitive regions in synthetic enhancers affects enhancer activity. In contrast, when tested in the dark, mutation-sensitive regions act independently and additively in conferring enhancer activity. Taken together, this work demonstrates that plant enhancers show evidence for both cooperative and additive interactions among their functional elements. This knowledge can be harnessed to design strong, condition-specific synthetic enhancers.\nGenetically-clustered antifungal phytocytokines and receptor proteins function together to trigger plant immune signaling\nAuthors: Lintz, J.; Goto, Y.; Bender, K. W.; Bchini, R.; Dubrulle, G.; Cawston, E.; Zipfel, C.; Duplessis, S.; PETRE, B.\nScore: 11.4, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568785\nPhytocytokines regulate plant immunity via cell-surface receptors. Populus trichocarpa RUST INDUCED SECRETED PEPTIDE 1 (PtRISP1) exhibits an elicitor activity in poplar, as well as a direct antimicrobial activity against rust fungi. PtRISP1 gene directly clusters with a gene encoding a leucine-rich repeat receptor protein (LRR-RP), that we termed RISP-ASSOCIATED LRR-RP (PtRALR). In this study, we used phylogenomics to characterize the RISP and RALR gene families, and functional assays to characterize RISP/RALR pairs. Both RISP and RALR gene families specifically evolved in Salicaceae species (poplar and willow), and systematically cluster in the genomes. Two divergent RISPs, PtRISP1 and Salix purpurea RISP1 (SpRISP1), induced a reactive oxygen species (ROS) burst and mitogen-activated protein kinases (MAPKs) phosphorylation in Nicotiana benthamiana leaves expressing the respective clustered RALR. PtRISP1 triggers a rapid stomatal closure in poplar, and both PtRISP1 and SpRISP1 directly inhibit rust pathogen growth. Altogether, these results suggest that plants evolved phytocytokines with direct antimicrobial activities, and that the genes coding these phytocytokines co-evolved and physically cluster with their cognate receptors.\nRobust organ size in Arabidopsis is primarily governed by cell growth rather than cell division patterns\nAuthors: Burda, I.; Li, C.-B.; Clark, F. K.; Roeder, A. H.\nScore: 12.7, Published: 2023-11-20 DOI: 10.1101/2023.11.11.566685\nOrgan sizes and shapes are highly reproducible, or robust, within a species and individuals. Arabidopsis thaliana sepals, which are the leaf-like organs that enclose flower buds, have consistent size and shape, which indicates robust development. Counterintuitively, variability in cell growth rate over time and between cells facilitates robust development because cumulative cell growth averages to a uniform rate. Here we investigate how sepal morphogenesis is robust to changes in cell division but not robust to changes in cell growth variability. We live image and quantitatively compare the development of sepals with increased or decreased cell division rate (lgo mutant and LGO overexpression, respectively), a mutant with altered cell growth variability (ftsh4), and double mutants combining these. We find that robustness is preserved when cell division rate changes because there is no change in the spatial pattern of growth. Meanwhile when robustness is lost in ftsh4 mutants, cell growth accumulates unevenly, and cells have disorganized growth directions. Thus, we demonstrate in vivo that both cell growth rate and direction average in robust development, preserving robustness despite changes in cell division. Summary statementRobust sepal development is preserved despite changes in cell division rate and is characterized by spatiotemporal averaging of heterogeneity in cell growth rate and direction.\nMultiplexed effector screening for recognition by endogenous resistance genes using positive defense reporters in wheat protoplasts\nAuthors: Wilson, S.; Dagvadorj, B.; Tam, R.; Murphy, L.; Schulz-Kroenert, S.; Heng, N.; Crean, E.; Greenwood, J.; Rathjen, J.; Schwessinger, B.\nScore: 39.3, Published: 2023-11-26 DOI: 10.1101/2023.04.30.538885\nO_LIPlant resistance (R) and pathogen avirulence (Avr) gene interactions play a vital role in pathogen resistance. Efficient molecular screening tools for crops lack far behind their model organism counterparts, yet they are essential to rapidly identify agriculturally important molecular interactions that trigger host resistance. C_LIO_LIHere, we have developed a novel wheat protoplast assay that enables efficient screening of Avr/R interactions at scale. Our assay allows access to the extensive gene pool of phenotypically described R genes because it does not require the overexpression of cloned R genes. It is suitable for multiplexed Avr screening, with interactions tested in pools of up to fifty Avr candidates. C_LIO_LIWe identified Avr/R-induced defense genes to create promoter-luciferase reporter. Then, we combined this with a dual-color ratiometric reporter system that normalizes read-outs accounting for experimental variability and Avr/R-induced cell-death. Moreover, we introduced a self-replicative plasmid reducing the amount of plasmid used in the assay. C_LIO_LIOur assay increases the throughput of Avr candidate screening, accelerating the study of cellular defense signaling and resistance gene identification in wheat. We anticipate that our assay will significantly accelerate Avr identification for many wheat pathogens, leading to improved genome-guided pathogen surveillance and breeding of disease-resistant crops. C_LI\nGenome sequence and cell biological toolbox of the highly regenerative, coenocytic green feather alga Bryopsis\nAuthors: Ochiai, K. K.; Hanawa, D.; Ogawa, H. A.; Tanaka, H.; Uesaka, K.; Edzuka, T.; Shirae-Kurabayashi, M.; Toyoda, A.; Itoh, T.; Goshima, G.\nScore: 7.3, Published: 2023-11-23 DOI: 10.1101/2023.11.22.568388\nGreen feather algae (Bryopsidales) undergo a unique life cycle in which a single cell repeatedly executes nuclear division without cytokinesis, resulting in the development of a thallus (\u003e 100 mm) with characteristic morphology called coenocyte. Bryopsis is a representative coenocytic alga that has exceptionally high regeneration ability: extruded cytoplasm aggregates rapidly in seawater, leading to the formation of protoplasts. However, the genetic basis of the unique cell biology of Bryopsis remains poorly understood. Here, we present a high-quality assembly and annotation of the nuclear genome of Bryopsis sp. (90.7 Mbp, 27 contigs, N50 = 6.7 Mbp, 14,034 protein-coding genes). Comparative genomic analyses indicate that the genes encoding BPL-1/Bryohealin, the aggregation-promoting lectin, are heavily duplicated in Bryopsis, whereas homologous genes are absent in other Ulvophycean algae, suggesting the basis of regeneration capability of Bryopsis. Bryopsis sp. possesses \u003e30 kinesins but only a single myosin, which differs from other green algae that have multiple types of myosin genes. Consistent with this biased motor toolkit, we observed that the bidirectional motility of chloroplasts in the cytoplasm was dependent on microtubules but not actin in Bryopsis sp. Unexpectedly, most genes required for cytokinesis in plants are present in Bryopsis, including those in the SNARE or kinesin superfamily. Nevertheless, a kinesin crucial for cytokinesis initiation in plants (NACK/Kinesin-7II) is hardly expressed in the coenocytic part of the thallus, possibly underlying the lack of cytokinesis in this portion. The present genome sequence lays the foundation for experimental biology in coenocytic macroalgae. Significance statementThe exceptionally coenocytic body and remarkable regeneration ability of Bryopsis have attracted biologists for years. However, molecular biological tools remain underdeveloped, partly due to the lack of genome information. Here, we report high-quality assembly and annotation of the genome, providing a crucial resource for experimental biology and genomics studies of Bryopsis. Furthermore, comparative genomic analysis reveals a unique gene repertoire that possibly underlies the highly regenerative coenocytic body.\n",
  "wordCount" : "2456",
  "inLanguage": "en",
  "datePublished": "2023-11-29T10:40:14Z",
  "dateModified": "2023-11-29T10:40:14Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 29, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.567997">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.567997" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.567997">
        <p class="paperTitle">AvrSr27 is a zinc-bound effector with a modular structure important for immune recognition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.567997" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.567997" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Outram, M. A.; Chen, J.; Broderick, S.; Li, Z.; Aditya, S.; Tasneem, N.; Arndell, T.; Blundell, C.; Ericsson, D. J.; Figueroa, M.; Sperschneider, J.; Dodds, P.; Williams, S. J.</p>
        <p class="info">Score: 25.9, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.567997' target='https://doi.org/10.1101/2023.11.21.567997'> 10.1101/2023.11.21.567997</a></p>
        <p class="abstract">Stem rust, caused by the fungal pathogen Puccinia graminis f. sp.tritici (Pgt) is a major threat for wheat production and global food security. Central to the success of Pgt is the secretion of proteinaceous effectors that promote infection and colonisation, while immunity in wheat is driven by receptor-mediated recognition of these effectors resulting in pathogen avirulence. Here, we report the crystal structure of the cysteine-rich effector AvrSr27, the third experimentally derived structure of a Pgt effector. The AvrSr27 structure reveals a novel {beta}-strand rich modular fold consisting of two structurally similar domains and confirms the poor prediction we obtained from the AlphaFold2-derived model. The highly prevalent cysteine residues within the protein facilitate the co-ordination of 4 zinc molecules. Utilising the structure, we show that the N-terminal domain of AvrSr27 is sufficient for immune recognition and interaction by Sr27. The 7-cys motif sequence in each AvrSr27 domain, which facilitates zinc binding, was also found in two haustorially-expressed, structurally homologous Pgt proteins. Remarkably, despite relatively low sequence identity, we show that these proteins can associate with Sr27 and trigger cell death in heterologous systems and wheat protoplasts, albeit weaker than AvrSr27. Collectively, our findings have important implications for the field embarking on bespoke engineering of immunity receptors as solutions to plant disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566459">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566459" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566459">
        <p class="paperTitle">Comparative mutant analyses reveal a novel mechanism of ARF regulation in land plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566459" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566459" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Prigge, M. J.; Morffy, N.; De Neve, A.; Szutu, W.; Juarez, M. J. A.; Johnson, K.; Do, N.; Lavy, M.; Hake, S.; Strader, L.; Estelle, M.; Richardson, A. E.</p>
        <p class="info">Score: 18.5, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566459' target='https://doi.org/10.1101/2023.11.09.566459'> 10.1101/2023.11.09.566459</a></p>
        <p class="abstract">A major challenge in plant biology is to understand how the plant hormone auxin regulates diverse transcriptional responses throughout development, in different environments, and in different species. The answer may lie in the specific complement of auxin signaling components in each cell. The balance between activators (class-A AUXIN RESPONSE FACTORS) and repressors (class-B ARFs) is particularly important. It is unclear how this balance is achieved. Through comparative analysis of novel, dominant mutants in maize and the moss Physcomitrium patens, we have discovered a [~]500-million-year-old mechanism of class-B ARF protein level regulation, important in determining cell fate decisions across land plants. Thus, our results add a key piece to the puzzle of how auxin regulates plant development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568095">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568095" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568095">
        <p class="paperTitle">Crosstalk between salicylic acid signaling and the circadian clock promotes an effective immune response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568095" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568095" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fraser, O. J. P.; Cargill, S. J.; Spoel, S. H.; van Ooijen, G.</p>
        <p class="info">Score: 13.8, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568095' target='https://doi.org/10.1101/2023.11.21.568095'> 10.1101/2023.11.21.568095</a></p>
        <p class="abstract">The rotation of Earth creates a cycle of day and night, leading to predictable changes in environmental conditions. The circadian clock synchronizes an organism with these environmental changes and alters their physiology in anticipation. Prediction of the probable timing of pathogen infection enables plants to prime their immune system without wasting resources or sacrificing growth. Here, we explore the relationship between the immune hormone salicylic acid (SA), and the circadian clock in Arabidopsis. We found that SA altered circadian rhythmicity through the SA receptor and master transcriptional coactivator, NPR1. Reciprocally, the circadian clock gates SA-induced expression of NPR1-dependent immune genes. Furthermore, the clock gene CCA1 is essential for SA-induced immunity to the major bacterial plant pathogen Pseudomonas syringae. These results reveal new interactions between the circadian clock and SA signaling which produce an effective immune response. Understanding how and why the immune response in plants is linked to the circadian clock is crucial in working towards improved crop productivity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.25.568654">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.25.568654" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.25.568654">
        <p class="paperTitle">SAGA1 and SAGA2 promote starch formation around proto-pyrenoids in Arabidopsis chloroplasts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.25.568654" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.25.568654" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Atkinson, N.; Stringer, R.; Mitchell, S. R.; Seung, D.; McCormick, A. J.</p>
        <p class="info">Score: 13.2, Published: 2023-11-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.25.568654' target='https://doi.org/10.1101/2023.11.25.568654'> 10.1101/2023.11.25.568654</a></p>
        <p class="abstract">The pyrenoid is a chloroplastic microcompartment in which most algae and some terrestrial plants condense the primary carboxylase, Rubisco (ribulose-1,5-bisphosphate carboxylase/oxygenase) as part of a CO2-concentrating mechanism that improves the efficiency of CO2 capture. Engineering a pyrenoid-based CO2-concentrating mechanism (pCCM) into C3 crop plants is a promising strategy to enhance yield capacities and resilience to the changing climate. Many pyrenoids are characterized by a sheath of starch plates that is proposed to act as a barrier to limit CO2 diffusion. Recently, we have reconstituted a phase-separated  proto-pyrenoid Rubisco matrix in the model C3 plant Arabidopsis thaliana using proteins from the alga with the most well studied pyrenoid, Chlamydomonas reinhardtii (1). Here we describe the impact of introducing the Chlamydomonas proteins StArch Granules Abnormal 1 (SAGA1) and SAGA2, which are associated with the regulation of pyrenoid starch biogenesis and morphology. We show that SAGA1 localizes to the proto-pyrenoid in engineered Arabidopsis plants, which results in the formation of atypical spherical starch granules enclosed within the proto-pyrenoid condensate and adjacent plate-like granules that partially cover the condensate, but without modifying the total amount of chloroplastic starch accrued. Additional expression of SAGA2 further increases the proportion of starch synthesised as adjacent plate-like granules that fully encircle the proto-pyrenoid. Our findings pave the way to assembling a diffusion barrier as part of a functional pCCM in vascular plants, whilst also advancing our understanding of the roles of SAGA1 and SAGA2 in starch sheath formation and opening novel avenues for engineering starch morphology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567949">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567949" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567949">
        <p class="paperTitle">Fast and efficient root phenotyping via pose estimation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567949" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567949" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Berrigan, E. M.; Wang, L.; Carrillo, H.; Echegoyen, K.; Kappes, M.; Torres, J.; Ai-Perreira, A.; McCoy, E.; Shane, E.; Copeland, C. D.; Ragel, L.; Georgousakis, C.; Lee, S.; Reynolds, D.; Talgo, A.; Gonzalez, J.; Zhang, L.; Rajurkar, A. B.; Ruiz, M.; Daniels, E.; Maree, L.; Pariyar, S.; Busch, W.; Pereira, T. D.</p>
        <p class="info">Score: 12.5, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567949' target='https://doi.org/10.1101/2023.11.20.567949'> 10.1101/2023.11.20.567949</a></p>
        <p class="abstract">Image segmentation is commonly used to estimate the location and shape of plants and their external structures. Segmentation masks are then used to localize landmarks of interest and compute other geometric features that correspond to the plants phenotype. Despite its prevalence, segmentation-based approaches are laborious (requiring extensive annotation to train), and error-prone (derived geometric features are sensitive to instance mask integrity). Here we present a segmentation-free approach which leverages deep learning-based landmark detection and grouping, also known as pose estimation. We use a tool originally developed for animal motion capture called SLEAP (Social LEAP Estimates Animal Poses) to automate the detection of distinct morphological landmarks on plant roots. Using a gel cylinder imaging system across multiple species, we show that our approach can reliably and efficiently recover root system topology at high accuracy, few annotated samples, and faster speed than segmentation-based approaches. In order to make use of this landmark-based representation for root phenotyping, we developed a Python library (sleap-roots) for trait extraction directly comparable to existing segmentation-based analysis software. We show that landmark-derived root traits are highly accurate and can be used for common downstream tasks including genotype classification and unsupervised trait mapping. Altogether, this work establishes the validity and advantages of pose estimation-based plant phenotyping. To facilitate adoption of this easy-to-use tool and to encourage further development, we make sleap-roots, all training data, models, and trait extraction code available at: https://github.com/talmolab/sleap-roots and https://osf.io/k7j9g/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568158">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568158" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568158">
        <p class="paperTitle">Cooperativity and additivity in plant enhancers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568158" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568158" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jores, T.; Tonnies, J.; Mueth, N. A.; Romanowski, A.; Fields, S.; Cuperus, J.; Queitsch, C.</p>
        <p class="info">Score: 11.8, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568158' target='https://doi.org/10.1101/2023.11.21.568158'> 10.1101/2023.11.21.568158</a></p>
        <p class="abstract">Enhancers are cis-regulatory elements that shape gene expression in response to numerous developmental and environmental cues. In animals, several models have been proposed to explain how enhancers integrate the activity of multiple transcription factors. However, it remains largely unknown how plant enhancers integrate transcription factor activity. Here, we use Plant STARR-seq to characterize three light-responsive plant enhancers--AB80, Cab-1, and rbcS-E9--derived from genes active in photosynthesis. Saturation mutagenesis reveals mutations, many of which cluster in short regions, that strongly reduce enhancer activity in the light, in the dark or in both conditions. When tested in the light, these mutation-sensitive regions do not function on their own; rather, cooperative interactions with other such regions are required for full activity. Epistatic interactions occur between mutations in adjacent mutation-sensitive regions, and the spacing and order of mutation-sensitive regions in synthetic enhancers affects enhancer activity. In contrast, when tested in the dark, mutation-sensitive regions act independently and additively in conferring enhancer activity. Taken together, this work demonstrates that plant enhancers show evidence for both cooperative and additive interactions among their functional elements. This knowledge can be harnessed to design strong, condition-specific synthetic enhancers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568785">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568785" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568785">
        <p class="paperTitle">Genetically-clustered antifungal phytocytokines and receptor proteins function together to trigger plant immune signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568785" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568785" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lintz, J.; Goto, Y.; Bender, K. W.; Bchini, R.; Dubrulle, G.; Cawston, E.; Zipfel, C.; Duplessis, S.; PETRE, B.</p>
        <p class="info">Score: 11.4, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568785' target='https://doi.org/10.1101/2023.11.27.568785'> 10.1101/2023.11.27.568785</a></p>
        <p class="abstract">Phytocytokines regulate plant immunity via cell-surface receptors. Populus trichocarpa RUST INDUCED SECRETED PEPTIDE 1 (PtRISP1) exhibits an elicitor activity in poplar, as well as a direct antimicrobial activity against rust fungi. PtRISP1 gene directly clusters with a gene encoding a leucine-rich repeat receptor protein (LRR-RP), that we termed RISP-ASSOCIATED LRR-RP (PtRALR). In this study, we used phylogenomics to characterize the RISP and RALR gene families, and functional assays to characterize RISP/RALR pairs. Both RISP and RALR gene families specifically evolved in Salicaceae species (poplar and willow), and systematically cluster in the genomes. Two divergent RISPs, PtRISP1 and Salix purpurea RISP1 (SpRISP1), induced a reactive oxygen species (ROS) burst and mitogen-activated protein kinases (MAPKs) phosphorylation in Nicotiana benthamiana leaves expressing the respective clustered RALR. PtRISP1 triggers a rapid stomatal closure in poplar, and both PtRISP1 and SpRISP1 directly inhibit rust pathogen growth. Altogether, these results suggest that plants evolved phytocytokines with direct antimicrobial activities, and that the genes coding these phytocytokines co-evolved and physically cluster with their cognate receptors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.11.566685">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.11.566685" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.11.566685">
        <p class="paperTitle">Robust organ size in Arabidopsis is primarily governed by cell growth rather than cell division patterns</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.11.566685" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.11.566685" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Burda, I.; Li, C.-B.; Clark, F. K.; Roeder, A. H.</p>
        <p class="info">Score: 12.7, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.11.566685' target='https://doi.org/10.1101/2023.11.11.566685'> 10.1101/2023.11.11.566685</a></p>
        <p class="abstract">Organ sizes and shapes are highly reproducible, or robust, within a species and individuals. Arabidopsis thaliana sepals, which are the leaf-like organs that enclose flower buds, have consistent size and shape, which indicates robust development. Counterintuitively, variability in cell growth rate over time and between cells facilitates robust development because cumulative cell growth averages to a uniform rate. Here we investigate how sepal morphogenesis is robust to changes in cell division but not robust to changes in cell growth variability. We live image and quantitatively compare the development of sepals with increased or decreased cell division rate (lgo mutant and LGO overexpression, respectively), a mutant with altered cell growth variability (ftsh4), and double mutants combining these. We find that robustness is preserved when cell division rate changes because there is no change in the spatial pattern of growth. Meanwhile when robustness is lost in ftsh4 mutants, cell growth accumulates unevenly, and cells have disorganized growth directions. Thus, we demonstrate in vivo that both cell growth rate and direction average in robust development, preserving robustness despite changes in cell division.

Summary statementRobust sepal development is preserved despite changes in cell division rate and is characterized by spatiotemporal averaging of heterogeneity in cell growth rate and direction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.04.30.538885">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.04.30.538885" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.04.30.538885">
        <p class="paperTitle">Multiplexed effector screening for recognition by endogenous resistance genes using positive defense reporters in wheat protoplasts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.04.30.538885" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.04.30.538885" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wilson, S.; Dagvadorj, B.; Tam, R.; Murphy, L.; Schulz-Kroenert, S.; Heng, N.; Crean, E.; Greenwood, J.; Rathjen, J.; Schwessinger, B.</p>
        <p class="info">Score: 39.3, Published: 2023-11-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.04.30.538885' target='https://doi.org/10.1101/2023.04.30.538885'> 10.1101/2023.04.30.538885</a></p>
        <p class="abstract">O_LIPlant resistance (R) and pathogen avirulence (Avr) gene interactions play a vital role in pathogen resistance. Efficient molecular screening tools for crops lack far behind their model organism counterparts, yet they are essential to rapidly identify agriculturally important molecular interactions that trigger host resistance.
C_LIO_LIHere, we have developed a novel wheat protoplast assay that enables efficient screening of Avr/R interactions at scale. Our assay allows access to the extensive gene pool of phenotypically described R genes because it does not require the overexpression of cloned R genes. It is suitable for multiplexed Avr screening, with interactions tested in pools of up to fifty Avr candidates.
C_LIO_LIWe identified Avr/R-induced defense genes to create promoter-luciferase reporter. Then, we combined this with a dual-color ratiometric reporter system that normalizes read-outs accounting for experimental variability and Avr/R-induced cell-death. Moreover, we introduced a self-replicative plasmid reducing the amount of plasmid used in the assay.
C_LIO_LIOur assay increases the throughput of Avr candidate screening, accelerating the study of cellular defense signaling and resistance gene identification in wheat. We anticipate that our assay will significantly accelerate Avr identification for many wheat pathogens, leading to improved genome-guided pathogen surveillance and breeding of disease-resistant crops.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568388">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568388" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568388">
        <p class="paperTitle">Genome sequence and cell biological toolbox of the highly regenerative, coenocytic green feather alga Bryopsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568388" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568388" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ochiai, K. K.; Hanawa, D.; Ogawa, H. A.; Tanaka, H.; Uesaka, K.; Edzuka, T.; Shirae-Kurabayashi, M.; Toyoda, A.; Itoh, T.; Goshima, G.</p>
        <p class="info">Score: 7.3, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568388' target='https://doi.org/10.1101/2023.11.22.568388'> 10.1101/2023.11.22.568388</a></p>
        <p class="abstract">Green feather algae (Bryopsidales) undergo a unique life cycle in which a single cell repeatedly executes nuclear division without cytokinesis, resulting in the development of a thallus (&gt; 100 mm) with characteristic morphology called coenocyte. Bryopsis is a representative coenocytic alga that has exceptionally high regeneration ability: extruded cytoplasm aggregates rapidly in seawater, leading to the formation of protoplasts. However, the genetic basis of the unique cell biology of Bryopsis remains poorly understood. Here, we present a high-quality assembly and annotation of the nuclear genome of Bryopsis sp. (90.7 Mbp, 27 contigs, N50 = 6.7 Mbp, 14,034 protein-coding genes). Comparative genomic analyses indicate that the genes encoding BPL-1/Bryohealin, the aggregation-promoting lectin, are heavily duplicated in Bryopsis, whereas homologous genes are absent in other Ulvophycean algae, suggesting the basis of regeneration capability of Bryopsis. Bryopsis sp. possesses &gt;30 kinesins but only a single myosin, which differs from other green algae that have multiple types of myosin genes. Consistent with this biased motor toolkit, we observed that the bidirectional motility of chloroplasts in the cytoplasm was dependent on microtubules but not actin in Bryopsis sp. Unexpectedly, most genes required for cytokinesis in plants are present in Bryopsis, including those in the SNARE or kinesin superfamily. Nevertheless, a kinesin crucial for cytokinesis initiation in plants (NACK/Kinesin-7II) is hardly expressed in the coenocytic part of the thallus, possibly underlying the lack of cytokinesis in this portion. The present genome sequence lays the foundation for experimental biology in coenocytic macroalgae.

Significance statementThe exceptionally coenocytic body and remarkable regeneration ability of Bryopsis have attracted biologists for years. However, molecular biological tools remain underdeveloped, partly due to the lack of genome information. Here, we report high-quality assembly and annotation of the genome, providing a crucial resource for experimental biology and genomics studies of Bryopsis. Furthermore, comparative genomic analysis reveals a unique gene repertoire that possibly underlies the highly regenerative coenocytic body.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
