<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Characterization of DNA methylation reader proteins of Arabidopsis thaliana
Authors: Cahn, J.; Lloyd, J. P. B.; Karemaker, I. D.; Jansen, P. W.; Pflueger, J.; Duncan, O.; Petereit, J.; Bogdanovic, O.; Millar, H.; Vermeulen, M.; Lister, R.
Score: 33.5, Published: 2023-12-06 DOI: 10.1101/2023.12.05.570080
In plants, cytosine DNA methylation (mC) is largely associated with transcriptional repression of transposable elements, but it can also be found in the body of expressed genes, referred to as gene body methylation (GbM).">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="Characterization of DNA methylation reader proteins of Arabidopsis thaliana
Authors: Cahn, J.; Lloyd, J. P. B.; Karemaker, I. D.; Jansen, P. W.; Pflueger, J.; Duncan, O.; Petereit, J.; Bogdanovic, O.; Millar, H.; Vermeulen, M.; Lister, R.
Score: 33.5, Published: 2023-12-06 DOI: 10.1101/2023.12.05.570080
In plants, cytosine DNA methylation (mC) is largely associated with transcriptional repression of transposable elements, but it can also be found in the body of expressed genes, referred to as gene body methylation (GbM)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-10T10:37:48+00:00" />
<meta property="article:modified_time" content="2023-12-10T10:37:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="Characterization of DNA methylation reader proteins of Arabidopsis thaliana
Authors: Cahn, J.; Lloyd, J. P. B.; Karemaker, I. D.; Jansen, P. W.; Pflueger, J.; Duncan, O.; Petereit, J.; Bogdanovic, O.; Millar, H.; Vermeulen, M.; Lister, R.
Score: 33.5, Published: 2023-12-06 DOI: 10.1101/2023.12.05.570080
In plants, cytosine DNA methylation (mC) is largely associated with transcriptional repression of transposable elements, but it can also be found in the body of expressed genes, referred to as gene body methylation (GbM)."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "Characterization of DNA methylation reader proteins of Arabidopsis thaliana\nAuthors: Cahn, J.; Lloyd, J. P. B.; Karemaker, I. D.; Jansen, P. W.; Pflueger, J.; Duncan, O.; Petereit, J.; Bogdanovic, O.; Millar, H.; Vermeulen, M.; Lister, R.\nScore: 33.5, Published: 2023-12-06 DOI: 10.1101/2023.12.05.570080\nIn plants, cytosine DNA methylation (mC) is largely associated with transcriptional repression of transposable elements, but it can also be found in the body of expressed genes, referred to as gene body methylation (GbM).",
  "keywords": [
    
  ],
  "articleBody": " Characterization of DNA methylation reader proteins of Arabidopsis thaliana\nAuthors: Cahn, J.; Lloyd, J. P. B.; Karemaker, I. D.; Jansen, P. W.; Pflueger, J.; Duncan, O.; Petereit, J.; Bogdanovic, O.; Millar, H.; Vermeulen, M.; Lister, R.\nScore: 33.5, Published: 2023-12-06 DOI: 10.1101/2023.12.05.570080\nIn plants, cytosine DNA methylation (mC) is largely associated with transcriptional repression of transposable elements, but it can also be found in the body of expressed genes, referred to as gene body methylation (GbM). GbM is correlated with ubiquitously expressed genes, however its function, or absence thereof, is highly debated. The different output that mC can have raises questions as to how it is interpreted - or read - differently in these sequence and genomic contexts. To screen for potential mC binding proteins, we performed an unbiased DNA affinity pull-down assay combined with quantitative mass spectrometry using methylated DNA probes for each DNA sequence context. All mC readers known to date were found to preferentially bind to the methylated probes, along with a range of new mC binding protein candidates. Functional characterization of these mC readers, focused on the MBD and SUVH families, was undertaken by ChIP-seq mapping of genome-wide binding sites, their protein interactors, and the impact of high-order mutations on transcriptomic and epigenomic profiles. Together, this highlighted specific context preferences for these proteins, and in particular the ability of MBD2 to bind specifically to GbM. This comprehensive analysis of Arabidopsis mC readers emphasizes the complexity and interconnectivity between DNA methylation and chromatin remodelling processes in plants.\nggPlantmap: an R package for creation of informative and quantitative ggplot maps derived from plant images.\nAuthors: Jo, L.; Kajala, K.\nScore: 23.3, Published: 2023-12-03 DOI: 10.1101/2023.11.30.569429\nAs plant research generates an ever-growing volume of spatial quantitative data, the need for decentralized and user-friendly visualization tools to explore large and complex datasets tools becomes crucial. Existing resources, such as the Plant eFP (electronic Fluorescent Pictograph) browsers, have played a pivotal role on the communication of gene expression data across many plant species. However, although widely used by the plant research community, the Plant eFP browser lacks open and user-friendly tools for the creation of customized expression maps independently. Plant biologists with less coding experience can often encounter challenges when attempting to explore ways to communicate their own spatial quantitative data. We present ggPlantmap an open-source R package designed to address this challenge by providing an easy and user-friendly method for the creation of ggplot representative maps from plant images. ggPlantmap is built in R, one of the most used languages in biology to empower plant scientists to create and customize eFP-like browsers tailored to their experimental data. Here, we provide an overview of the package and tutorials that are accessible even to users with minimal R programming experience. We hope that ggPlantmap can assist the plant science community, fostering innovation and improving our understanding of plant development and function. HighlightggPlantmap, a new addition to the plant data visualization toolbox, allows users to create graphical maps from plant images for the representation of spatial quantitative data in R.\nApoplastic barrier establishment in roots and nodules of Lotus japonicus is essential for root-shoot signaling and N-fixation\nAuthors: Shen, D.; Venado, R. E.; Neumann, U.; Dyballa-Rukes, N.; Mahiwal, S.; Metzger, S.; Nakano, R. T.; Marin, M.; Andersen, T. G.\nScore: 23.2, Published: 2023-12-08 DOI: 10.1101/2023.12.06.570432\nThe molecular framework underlying apoplastic root barrier formation has been unveiled in the model species Arabidopsis thaliana where establishment of Casparian strips occurs at an early stage of root development. In legumes, this region overlaps with the area where nitrogen-fixing bacteria can induce nodule formation, termed the susceptible zone. Moreover, while nodules themselves also contain an endodermis spanning their vascular bundles, it is current unknown if Casparian strips serve as a filter for transport across this specialized organ. Here we establish barrier mutants in the symbiosis model Lotus japonicus. We find that the while the genetic network controlling Casparian strip formation is conserved in this legume species, formation of functional barriers is crucial for establishment of N-fixing nodules. By probing this in detail, we establish a model where the Casparian strip, via its linked Schengen pathway, converge with long distance N signaling and systemic regulation of nodulation. Moreover, this also reveal that the genetic system for barrier establishment in the root endodermis is shared in nodule vascular endodermis and required for nodule function. Combined, our findings uncover a novel role of apoplastic root barriers and establishes a mutant collection suitable to probe the role of root barriers in symbiotic plant-microbe relationships.\nThe nucleotide binding domain of NRC-dependent disease resistance proteins is sufficient to activate downstream helper NLR oligomerization and immune signaling\nAuthors: Contreras, M. P.; Pai, H.; Thompson, R.; Claeys, J.; Adachi, H.; Kamoun, S.\nScore: 32.6, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569466\nNucleotide-binding domain and leucine-rich repeat (NLR) proteins with pathogen sensor activities have evolved to initiate immune signaling by activating helper NLRs. However, the mechanisms underpinning helper NLR activation by sensor NLRs remain poorly understood. Although coiled-coil (CC) type sensor NLRs such as the Potato virus X disease resistance protein Rx have been shown to activate the oligomerization of their downstream helpers NRC2 and NRC4, the domains involved in sensor-helper signaling are not known. Here, we show that the nucleotide binding (NB) domain within the NB-ARC of the Potato virus X disease resistance protein Rx is necessary and sufficient for oligomerization and immune signaling of downstream helper NLRs. In addition, the NB domains of the disease resistance proteins Gpa2 (cyst nematode resistance), Rpi-amr1, Rpi-amr3 (oomycete resistance) and Sw-5b (virus resistance) are also sufficient to activate their respective downstream NRC helpers. Moreover, the NB domain of Rx and its helper NRC2 form a minimal functional unit that can be transferred from solanaceous plants (lamiids) to the Campanulid species lettuce (Lactuca sativa). Our results challenge the prevailing paradigm that NLR proteins exclusively signal via their N-terminal domains and reveal a signaling activity for the NB domain of NRC-dependent sensor NLRs. We propose a model in which helper NLRs monitor the status of the NB domain of their upstream sensors.\nBait, not reward: CO2-enriched Nepenthes pitchers secrete toxic nectar\nAuthors: Lathika, C. C.; Sujatha, G. B.; Thomas, G.; Johnson, A. J.; Viswanathan, G.; Varghese, T. S.; Mohamed, S.; Shereefa, L. A.; Baby, S.\nScore: 17.0, Published: 2023-11-25 DOI: 10.1101/2023.11.25.568661\nNepenthes pitchers are leaf-evolved biological traps holding high levels of CO2 within them. Extrafloral nectar (EFN) secreted by Nepenthes pitchers has long been regarded as the major reward to the visiting arthropods, but its chemical constituents and their role in prey capture are least explored. Here we show that Nepenthes EFN is a toxic sugar mix devoid of key nitrogenous components (amino acids, proteins) and vitamin C. Fatty acids (cis-11-octadecenoic acid (C18:1), palmitic acid (C16:0), stearic acid (C18:0)) and their derivatives were detected in both N. khasiana peristome and lid EFNs. Both the nectars showed strong acetylcholinesterase (AChE) inhibition in vitro and in vivo. Bioactivity guided isolation revealed (+)-isoshinanolone, a naphthoquinone derivative, as the AChE inhibitor in Nepenthes EFNs. GC-headspace analysis showed the naphthoquinone, plumbagin, as the major volatile constituent in prey capturing regions of pitchers, along with the VOCs, (Z)-3-hexen-1-ol, 1-hexanol and acetoin. N. khasiana pitchers trap a wide spectrum of preys dominated by ants, whereas only limited herbivory was observed on its leaves and pitchers. Chemical (EFN, volatiles), gaseous (CO2), visual (colour, UV reflectance, fluorescence), physical (peristome geometry, surface microstructure, wettable surface-aquaplaning) and environmental (humidity, rain) factors effect prey capture in Nepenthes pitchers. These leaf-transformed traps with high growth rate and C:N ratio, reduced Rubisco activity, modified stomata, acidic pitcher fluid and absence of key nitrogenous metabolites are model systems displaying the effects of elevated CO2 within them. We testify that Nepenthes EFN is a toxic sugar bait which hinders the neuronal activity and flight of visiting arthropods. These unique traps adopt various deceptive strategies for prey capture, and our discovery abolishes the notion of Nepenthes EFN as a reward to the visiting ants and other arthropods.\nComparative transcriptomics of seed nourishing tissues: uncovering conserved and divergent pathways in seed plants\nAuthors: Florez-Rueda, A. M.; Miguel, C. M.; Figueiredo, D. D.\nScore: 13.0, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569347\nThe evolutionary and ecological success of Spermatophytes is intrinsically linked to the seed habit, which provides a protective environment for the pre-development of the new generation. This environment includes an ephemeral nourishing tissue that supports embryo growth. In gymnosperms this tissue originates from the asexual proliferation of the maternal megagametophyte, while in angiosperms it is a product of fertilization, and is called the endosperm. The emergence of these nourishing tissues is of profound evolutionary value, and they are also food staples for most of the worlds population. Here, using orthogroup inference, we provide a comparative transcriptomic analysis of seed nourishing tissues from representative species of all main angiosperm clades, including those of early diverging basal angiosperms, and of two gymnosperm representatives. Our results show that, although the structure and composition of seed nourishing tissues has seen significant divergence along evolution, there are signatures that are conserved throughout the phylogeny. Conversely, we identified processes that are exclusive to each clade, and thus illustrate their functional divergence. With this, we aimed to provide a foundation for future studies on the evolutionary history of seed nourishing structures, as well as a resource for gene discovery in new functional studies. Significance StatementWithin seeds a specialized structure is responsible for nourishing the embryo, and is thus analogous to the mammalian placenta. Moreover, these nourishing tissues are also important sources of staple foods and feed. Here, we provide novel gene expression datasets of nourishing tissues of early diverging angiosperms, and use this information for a meta-analysis to identify pathways conserved, or divergent, throughout evolution. Thus, we aim to provide a resource for gene discovery for seed biology studies.\nSeed coat formation in Arabidopsis requires a concerted action of JUMONJI histone H3K27me3 demethylases and Brassinosteroid signaling\nAuthors: Pankaj, R.; Lima, R. B.; Luo, G.-Y.; Ehlert, S.; Finger, P.; Sato, H.; Figueiredo, D. D.\nScore: 9.0, Published: 2023-12-08 DOI: 10.1101/2023.12.07.569203\nSeed development in flowering plants starts with a double fertilization process driven by the fusion of the maternal egg and central cells with two paternal sperm cells. This leads to the formation of the embryo and endosperm. These fertilization products are enveloped by the maternally-derived seed coat, the development of which is inhibited prior to fertilization by epigenetic regulator Polycomb Repressive Complex 2 (PRC2). This complex deposits the repressive histone mark H3K27me3, whose removal is necessary for seed coat formation. However, H3K27me3 marks are stable and PRC2 removal does not explain how seed coat genes become activated after fertilization. Here, we show that JUMONJI-type (JMJ) histone demethylases are expressed in the seed coats of Arabidopsis thaliana (Arabidopsis) seeds and are necessary for its formation. We further propose that JMJ activity is coupled to Brassinosteroid (BR) function, as BR effectors have been shown to physically recruit JMJ proteins to target loci. Consistent with this hypothesis, we show that loss of BR function leads to seed coat defects, which can be rescued by depletion of H3K27me3. Finally, we reveal an additional pathway through which BRs directly regulate seed coat development, independently of H3K27me3 deposition. This discovery highlights the diverse functions of BRs in coordinating seed development, beyond their known roles in plant growth and development.\nUDP-glucosyltransferase 71C4 regulates seed development by redistributing phenylpropanoid metabolism in cotton\nAuthors: Cao, Y.; Han, Z.; He, L.; Huang, C.; Chen, J.; Dai, F.; Xuan, L.; Yan, S.; Si, Z.; Hu, Y.; Zhang, T.\nScore: 8.7, Published: 2023-11-13 DOI: 10.1101/2023.11.09.566436\nSeeds are important for plant reproduction, and hence it is important to identify genes regulating seed development. Here, we focused on a member of the glycosyltransferase (GT) family, UDP-glucosyltransferase 71C4 (UGT71C4), which influences the seed width, seed length, and yield traits (lint percentage and seed index) by regulating the lignin and flavonoid pathways of phenylpropane metabolism, affects the oil and protein contents of mature seeds, and controls the seed size by regulating cell proliferation. Overexpression of UGT71C4 leads to seed enlargement by activating expression of peroxiredoxins and the flavonoid metabolism pathway; induces accumulation of ROS, which promotes cell proliferation; and significantly improves yield traits, with the seed index increasing from 10.66 to 11.91 and protein increasing by 5.34%, but oil content decreasing 8.98%. Conversely, knockout of UGT71C4 causes seeds to become smaller; the lignin metabolism pathway to be activated, especially enzymes relating to lignin synthesis leading to increased ectopic deposition of lignin in the ovule and constrained ovule growth and development; and significant improvement of yield traits, with lint percentage increasing from 39.62% to 41.94%, seed index decreasing from 10.66 to 8.60, protein content decreasing 4.28%, and no significant change in oil content. Our research provides new insights into seed size regulation through UDP-glucosyltransferase, providing potential methods for improving plant yield.\nTracing the evolutionary emergence of the temperature sensing prion-like domain in EARLY FLOWERING 3 across the plant kingdom\nAuthors: Zhu, Z.; Trenner, J.; Quint, M.\nScore: 6.3, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570556\nPlants have evolved to anticipate and adjust their growth and development in response to environmental changes. To mitigate the negative influence of global climate change on crop production, understanding the key regulators of plant performance is imperative. EARLY FLOWERING 3 (ELF3) is such a regulator involved in the circadian clock and thermomorphogenesis. Arabidopsis thaliana ELF3 contains a prion-like domain (PrD) that functions as a thermosensor, enabling its liquid-liquid phase separation at high ambient temperatures. To understand the conservation of this function across the plant kingdom, we traced the evolutionary emergence of ELF3 with a focus on the PrD, which confers liquid-liquid phase separation. We observed that the presence of the domain within ELF3, mainly contributed by the length of polyglutamine (polyQ) repeats, is largely restricted to Brassicales. This suggests that thermosensory function of ELF3 is a rather recent and secondary acquirement that was added to its main function. By analyzing 319 natural Arabidopsis thaliana accessions, we detected a wide range of polyQ length variation in ELF3. However, polyQ length is only weakly associated with geographic origin, climate conditions, and classic temperature-responsive phenotypes. Consequently, we conclude that although the emergence of PrD is not likely to be a key driver of environmental adaptation, it adds an extra layer to the role of ELF3 in thermomorphogenesis.\nDiverse 3D cellular patterns underlie the development of Cardamine hirsuta and Arabidopsis thaliana ovules\nAuthors: Mody, T. A.; Rolle, A.; Stucki, N.; Roll, F.; Bauer, U.; Schneitz, K.\nScore: 5.2, Published: 2023-12-06 DOI: 10.1101/2023.12.06.570408\nA fundamental question in biology is how organ morphogenesis comes about. The ovules of Arabidopsis thaliana have been established as a successful model to study numerous aspects of tissue morphogenesis; however, little is known regarding the relative contributions and dynamics of differential tissue and cellular growth and architecture in establishing ovule morphogenesis in different species. To address this issue, we generated a 3D digital atlas of Cardamine hirsuta ovule development with full cellular resolution. We combined quantitative comparative morphometrics and topological analysis to explore similarities and differences in the 3D cellular architectures underlying ovule development of the two species. We discovered that they show diversity in the way the three radial cell layers of the primordium contribute to its growth, in the formation of a new cell layer in the inner integument and, in certain cases, in the topological properties of the 3D cell architectures of homologous tissues despite their similar shape. Our work demonstrates the power of comparative 3D cellular morphometry and the importance of internal tissues and their cellular architecture in organ morphogenesis.\n",
  "wordCount" : "2587",
  "inLanguage": "en",
  "datePublished": "2023-12-10T10:37:48Z",
  "dateModified": "2023-12-10T10:37:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 10, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570080">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570080" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570080">
        <p class="paperTitle">Characterization of DNA methylation reader proteins of Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570080" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570080" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cahn, J.; Lloyd, J. P. B.; Karemaker, I. D.; Jansen, P. W.; Pflueger, J.; Duncan, O.; Petereit, J.; Bogdanovic, O.; Millar, H.; Vermeulen, M.; Lister, R.</p>
        <p class="info">Score: 33.5, Published: 2023-12-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570080' target='https://doi.org/10.1101/2023.12.05.570080'> 10.1101/2023.12.05.570080</a></p>
        <p class="abstract">In plants, cytosine DNA methylation (mC) is largely associated with transcriptional repression of transposable elements, but it can also be found in the body of expressed genes, referred to as gene body methylation (GbM). GbM is correlated with ubiquitously expressed genes, however its function, or absence thereof, is highly debated. The different output that mC can have raises questions as to how it is interpreted - or read - differently in these sequence and genomic contexts. To screen for potential mC binding proteins, we performed an unbiased DNA affinity pull-down assay combined with quantitative mass spectrometry using methylated DNA probes for each DNA sequence context. All mC readers known to date were found to preferentially bind to the methylated probes, along with a range of new mC binding protein candidates. Functional characterization of these mC readers, focused on the MBD and SUVH families, was undertaken by ChIP-seq mapping of genome-wide binding sites, their protein interactors, and the impact of high-order mutations on transcriptomic and epigenomic profiles. Together, this highlighted specific context preferences for these proteins, and in particular the ability of MBD2 to bind specifically to GbM. This comprehensive analysis of Arabidopsis mC readers emphasizes the complexity and interconnectivity between DNA methylation and chromatin remodelling processes in plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569429">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569429" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569429">
        <p class="paperTitle">ggPlantmap: an R package for creation of informative and quantitative ggplot maps derived from plant images.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569429" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569429" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jo, L.; Kajala, K.</p>
        <p class="info">Score: 23.3, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569429' target='https://doi.org/10.1101/2023.11.30.569429'> 10.1101/2023.11.30.569429</a></p>
        <p class="abstract">As plant research generates an ever-growing volume of spatial quantitative data, the need for decentralized and user-friendly visualization tools to explore large and complex datasets tools becomes crucial. Existing resources, such as the Plant eFP (electronic Fluorescent Pictograph) browsers, have played a pivotal role on the communication of gene expression data across many plant species. However, although widely used by the plant research community, the Plant eFP browser lacks open and user-friendly tools for the creation of customized expression maps independently. Plant biologists with less coding experience can often encounter challenges when attempting to explore ways to communicate their own spatial quantitative data. We present  ggPlantmap an open-source R package designed to address this challenge by providing an easy and user-friendly method for the creation of ggplot representative maps from plant images. ggPlantmap is built in R, one of the most used languages in biology to empower plant scientists to create and customize eFP-like browsers tailored to their experimental data. Here, we provide an overview of the package and tutorials that are accessible even to users with minimal R programming experience. We hope that ggPlantmap can assist the plant science community, fostering innovation and improving our understanding of plant development and function.

HighlightggPlantmap, a new addition to the plant data visualization toolbox, allows users to create graphical maps from plant images for the representation of spatial quantitative data in R.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.570432">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.570432" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.570432">
        <p class="paperTitle">Apoplastic barrier establishment in roots and nodules of Lotus japonicus is essential for root-shoot signaling and N-fixation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.570432" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.570432" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shen, D.; Venado, R. E.; Neumann, U.; Dyballa-Rukes, N.; Mahiwal, S.; Metzger, S.; Nakano, R. T.; Marin, M.; Andersen, T. G.</p>
        <p class="info">Score: 23.2, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.570432' target='https://doi.org/10.1101/2023.12.06.570432'> 10.1101/2023.12.06.570432</a></p>
        <p class="abstract">The molecular framework underlying apoplastic root barrier formation has been unveiled in the model species Arabidopsis thaliana where establishment of Casparian strips occurs at an early stage of root development. In legumes, this region overlaps with the area where nitrogen-fixing bacteria can induce nodule formation, termed the susceptible zone. Moreover, while nodules themselves also contain an endodermis spanning their vascular bundles, it is current unknown if Casparian strips serve as a filter for transport across this specialized organ. Here we establish barrier mutants in the symbiosis model Lotus japonicus. We find that the while the genetic network controlling Casparian strip formation is conserved in this legume species, formation of functional barriers is crucial for establishment of N-fixing nodules. By probing this in detail, we establish a model where the Casparian strip, via its linked Schengen pathway, converge with long distance N signaling and systemic regulation of nodulation. Moreover, this also reveal that the genetic system for barrier establishment in the root endodermis is shared in nodule vascular endodermis and required for nodule function. Combined, our findings uncover a novel role of apoplastic root barriers and establishes a mutant collection suitable to probe the role of root barriers in symbiotic plant-microbe relationships.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569466">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569466" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569466">
        <p class="paperTitle">The nucleotide binding domain of NRC-dependent disease resistance proteins is sufficient to activate downstream helper NLR oligomerization and immune signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569466" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569466" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Contreras, M. P.; Pai, H.; Thompson, R.; Claeys, J.; Adachi, H.; Kamoun, S.</p>
        <p class="info">Score: 32.6, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569466' target='https://doi.org/10.1101/2023.11.30.569466'> 10.1101/2023.11.30.569466</a></p>
        <p class="abstract">Nucleotide-binding domain and leucine-rich repeat (NLR) proteins with pathogen sensor activities have evolved to initiate immune signaling by activating helper NLRs. However, the mechanisms underpinning helper NLR activation by sensor NLRs remain poorly understood. Although coiled-coil (CC) type sensor NLRs such as the Potato virus X disease resistance protein Rx have been shown to activate the oligomerization of their downstream helpers NRC2 and NRC4, the domains involved in sensor-helper signaling are not known. Here, we show that the nucleotide binding (NB) domain within the NB-ARC of the Potato virus X disease resistance protein Rx is necessary and sufficient for oligomerization and immune signaling of downstream helper NLRs. In addition, the NB domains of the disease resistance proteins Gpa2 (cyst nematode resistance), Rpi-amr1, Rpi-amr3 (oomycete resistance) and Sw-5b (virus resistance) are also sufficient to activate their respective downstream NRC helpers. Moreover, the NB domain of Rx and its helper NRC2 form a minimal functional unit that can be transferred from solanaceous plants (lamiids) to the Campanulid species lettuce (Lactuca sativa). Our results challenge the prevailing paradigm that NLR proteins exclusively signal via their N-terminal domains and reveal a signaling activity for the NB domain of NRC-dependent sensor NLRs. We propose a model in which helper NLRs monitor the status of the NB domain of their upstream sensors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.25.568661">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.25.568661" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.25.568661">
        <p class="paperTitle">Bait, not reward: CO2-enriched Nepenthes pitchers secrete toxic nectar</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.25.568661" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.25.568661" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lathika, C. C.; Sujatha, G. B.; Thomas, G.; Johnson, A. J.; Viswanathan, G.; Varghese, T. S.; Mohamed, S.; Shereefa, L. A.; Baby, S.</p>
        <p class="info">Score: 17.0, Published: 2023-11-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.25.568661' target='https://doi.org/10.1101/2023.11.25.568661'> 10.1101/2023.11.25.568661</a></p>
        <p class="abstract">Nepenthes pitchers are leaf-evolved biological traps holding high levels of CO2 within them. Extrafloral nectar (EFN) secreted by Nepenthes pitchers has long been regarded as the major reward to the visiting arthropods, but its chemical constituents and their role in prey capture are least explored. Here we show that Nepenthes EFN is a toxic sugar mix devoid of key nitrogenous components (amino acids, proteins) and vitamin C. Fatty acids (cis-11-octadecenoic acid (C18:1), palmitic acid (C16:0), stearic acid (C18:0)) and their derivatives were detected in both N. khasiana peristome and lid EFNs. Both the nectars showed strong acetylcholinesterase (AChE) inhibition in vitro and in vivo. Bioactivity guided isolation revealed (&#43;)-isoshinanolone, a naphthoquinone derivative, as the AChE inhibitor in Nepenthes EFNs. GC-headspace analysis showed the naphthoquinone, plumbagin, as the major volatile constituent in prey capturing regions of pitchers, along with the VOCs, (Z)-3-hexen-1-ol, 1-hexanol and acetoin. N. khasiana pitchers trap a wide spectrum of preys dominated by ants, whereas only limited herbivory was observed on its leaves and pitchers. Chemical (EFN, volatiles), gaseous (CO2), visual (colour, UV reflectance, fluorescence), physical (peristome geometry, surface microstructure, wettable surface-aquaplaning) and environmental (humidity, rain) factors effect prey capture in Nepenthes pitchers. These leaf-transformed traps with high growth rate and C:N ratio, reduced Rubisco activity, modified stomata, acidic pitcher fluid and absence of key nitrogenous metabolites are model systems displaying the effects of elevated CO2 within them. We testify that Nepenthes EFN is  a toxic sugar bait which hinders the neuronal activity and flight of visiting arthropods. These unique traps adopt various deceptive strategies for prey capture, and our discovery abolishes the notion of Nepenthes EFN as a  reward to the visiting ants and other arthropods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569347">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569347" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569347">
        <p class="paperTitle">Comparative transcriptomics of seed nourishing tissues: uncovering conserved and divergent pathways in seed plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569347" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569347" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Florez-Rueda, A. M.; Miguel, C. M.; Figueiredo, D. D.</p>
        <p class="info">Score: 13.0, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569347' target='https://doi.org/10.1101/2023.11.30.569347'> 10.1101/2023.11.30.569347</a></p>
        <p class="abstract">The evolutionary and ecological success of Spermatophytes is intrinsically linked to the seed habit, which provides a protective environment for the pre-development of the new generation. This environment includes an ephemeral nourishing tissue that supports embryo growth. In gymnosperms this tissue originates from the asexual proliferation of the maternal megagametophyte, while in angiosperms it is a product of fertilization, and is called the endosperm. The emergence of these nourishing tissues is of profound evolutionary value, and they are also food staples for most of the worlds population. Here, using orthogroup inference, we provide a comparative transcriptomic analysis of seed nourishing tissues from representative species of all main angiosperm clades, including those of early diverging basal angiosperms, and of two gymnosperm representatives. Our results show that, although the structure and composition of seed nourishing tissues has seen significant divergence along evolution, there are signatures that are conserved throughout the phylogeny. Conversely, we identified processes that are exclusive to each clade, and thus illustrate their functional divergence. With this, we aimed to provide a foundation for future studies on the evolutionary history of seed nourishing structures, as well as a resource for gene discovery in new functional studies.

Significance StatementWithin seeds a specialized structure is responsible for nourishing the embryo, and is thus analogous to the mammalian placenta. Moreover, these nourishing tissues are also important sources of staple foods and feed. Here, we provide novel gene expression datasets of nourishing tissues of early diverging angiosperms, and use this information for a meta-analysis to identify pathways conserved, or divergent, throughout evolution. Thus, we aim to provide a resource for gene discovery for seed biology studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.569203">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.569203" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.569203">
        <p class="paperTitle">Seed coat formation in Arabidopsis requires a concerted action of JUMONJI histone H3K27me3 demethylases and Brassinosteroid signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.569203" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.569203" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pankaj, R.; Lima, R. B.; Luo, G.-Y.; Ehlert, S.; Finger, P.; Sato, H.; Figueiredo, D. D.</p>
        <p class="info">Score: 9.0, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.569203' target='https://doi.org/10.1101/2023.12.07.569203'> 10.1101/2023.12.07.569203</a></p>
        <p class="abstract">Seed development in flowering plants starts with a double fertilization process driven by the fusion of the maternal egg and central cells with two paternal sperm cells. This leads to the formation of the embryo and endosperm. These fertilization products are enveloped by the maternally-derived seed coat, the development of which is inhibited prior to fertilization by epigenetic regulator Polycomb Repressive Complex 2 (PRC2). This complex deposits the repressive histone mark H3K27me3, whose removal is necessary for seed coat formation. However, H3K27me3 marks are stable and PRC2 removal does not explain how seed coat genes become activated after fertilization. Here, we show that JUMONJI-type (JMJ) histone demethylases are expressed in the seed coats of Arabidopsis thaliana (Arabidopsis) seeds and are necessary for its formation. We further propose that JMJ activity is coupled to Brassinosteroid (BR) function, as BR effectors have been shown to physically recruit JMJ proteins to target loci. Consistent with this hypothesis, we show that loss of BR function leads to seed coat defects, which can be rescued by depletion of H3K27me3. Finally, we reveal an additional pathway through which BRs directly regulate seed coat development, independently of H3K27me3 deposition. This discovery highlights the diverse functions of BRs in coordinating seed development, beyond their known roles in plant growth and development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566436">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566436" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566436">
        <p class="paperTitle">UDP-glucosyltransferase 71C4 regulates seed development by redistributing phenylpropanoid metabolism in cotton</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566436" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566436" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cao, Y.; Han, Z.; He, L.; Huang, C.; Chen, J.; Dai, F.; Xuan, L.; Yan, S.; Si, Z.; Hu, Y.; Zhang, T.</p>
        <p class="info">Score: 8.7, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566436' target='https://doi.org/10.1101/2023.11.09.566436'> 10.1101/2023.11.09.566436</a></p>
        <p class="abstract">Seeds are important for plant reproduction, and hence it is important to identify genes regulating seed development. Here, we focused on a member of the glycosyltransferase (GT) family, UDP-glucosyltransferase 71C4 (UGT71C4), which influences the seed width, seed length, and yield traits (lint percentage and seed index) by regulating the lignin and flavonoid pathways of phenylpropane metabolism, affects the oil and protein contents of mature seeds, and controls the seed size by regulating cell proliferation. Overexpression of UGT71C4 leads to seed enlargement by activating expression of peroxiredoxins and the flavonoid metabolism pathway; induces accumulation of ROS, which promotes cell proliferation; and significantly improves yield traits, with the seed index increasing from 10.66 to 11.91 and protein increasing by 5.34%, but oil content decreasing 8.98%. Conversely, knockout of UGT71C4 causes seeds to become smaller; the lignin metabolism pathway to be activated, especially enzymes relating to lignin synthesis leading to increased ectopic deposition of lignin in the ovule and constrained ovule growth and development; and significant improvement of yield traits, with lint percentage increasing from 39.62% to 41.94%, seed index decreasing from 10.66 to 8.60, protein content decreasing 4.28%, and no significant change in oil content. Our research provides new insights into seed size regulation through UDP-glucosyltransferase, providing potential methods for improving plant yield.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570556">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570556" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570556">
        <p class="paperTitle">Tracing the evolutionary emergence of the temperature sensing prion-like domain in EARLY FLOWERING 3 across the plant kingdom</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570556" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570556" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhu, Z.; Trenner, J.; Quint, M.</p>
        <p class="info">Score: 6.3, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570556' target='https://doi.org/10.1101/2023.12.07.570556'> 10.1101/2023.12.07.570556</a></p>
        <p class="abstract">Plants have evolved to anticipate and adjust their growth and development in response to environmental changes. To mitigate the negative influence of global climate change on crop production, understanding the key regulators of plant performance is imperative. EARLY FLOWERING 3 (ELF3) is such a regulator involved in the circadian clock and thermomorphogenesis. Arabidopsis thaliana ELF3 contains a prion-like domain (PrD) that functions as a thermosensor, enabling its liquid-liquid phase separation at high ambient temperatures. To understand the conservation of this function across the plant kingdom, we traced the evolutionary emergence of ELF3 with a focus on the PrD, which confers liquid-liquid phase separation. We observed that the presence of the domain within ELF3, mainly contributed by the length of polyglutamine (polyQ) repeats, is largely restricted to Brassicales. This suggests that thermosensory function of ELF3 is a rather recent and secondary acquirement that was added to its main function. By analyzing 319 natural Arabidopsis thaliana accessions, we detected a wide range of polyQ length variation in ELF3. However, polyQ length is only weakly associated with geographic origin, climate conditions, and classic temperature-responsive phenotypes. Consequently, we conclude that although the emergence of PrD is not likely to be a key driver of environmental adaptation, it adds an extra layer to the role of ELF3 in thermomorphogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.570408">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.570408" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.570408">
        <p class="paperTitle">Diverse 3D cellular patterns underlie the development of Cardamine hirsuta and Arabidopsis thaliana ovules</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.570408" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.570408" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mody, T. A.; Rolle, A.; Stucki, N.; Roll, F.; Bauer, U.; Schneitz, K.</p>
        <p class="info">Score: 5.2, Published: 2023-12-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.570408' target='https://doi.org/10.1101/2023.12.06.570408'> 10.1101/2023.12.06.570408</a></p>
        <p class="abstract">A fundamental question in biology is how organ morphogenesis comes about. The ovules of Arabidopsis thaliana have been established as a successful model to study numerous aspects of tissue morphogenesis; however, little is known regarding the relative contributions and dynamics of differential tissue and cellular growth and architecture in establishing ovule morphogenesis in different species. To address this issue, we generated a 3D digital atlas of Cardamine hirsuta ovule development with full cellular resolution. We combined quantitative comparative morphometrics and topological analysis to explore similarities and differences in the 3D cellular architectures underlying ovule development of the two species. We discovered that they show diversity in the way the three radial cell layers of the primordium contribute to its growth, in the formation of a new cell layer in the inner integument and, in certain cases, in the topological properties of the 3D cell architectures of homologous tissues despite their similar shape. Our work demonstrates the power of comparative 3D cellular morphometry and the importance of internal tissues and their cellular architecture in organ morphogenesis.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
