<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Near-infrared imaging of phytochrome-derived autofluorescence in plant nuclei
Authors: Yoshinari, A.; Isoda, R.; Yagi, N.; Sato, Y.; Lindeboom, J. J.; Ehrhardt, D. W.; Frommer, W. B.; Nakamura, M.
Score: 26.7, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558255
Capturing images of the nuclear dynamics within live cells is an essential technique for comprehending the intricate biological processes inherent to plant cell nuclei. While various methods exist for imaging nuclei, including combining fluorescent proteins and dyes with microscopy, there is a dearth of commercially available dyes for live-cell imaging.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="Near-infrared imaging of phytochrome-derived autofluorescence in plant nuclei
Authors: Yoshinari, A.; Isoda, R.; Yagi, N.; Sato, Y.; Lindeboom, J. J.; Ehrhardt, D. W.; Frommer, W. B.; Nakamura, M.
Score: 26.7, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558255
Capturing images of the nuclear dynamics within live cells is an essential technique for comprehending the intricate biological processes inherent to plant cell nuclei. While various methods exist for imaging nuclei, including combining fluorescent proteins and dyes with microscopy, there is a dearth of commercially available dyes for live-cell imaging." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-04T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-10-04T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="Near-infrared imaging of phytochrome-derived autofluorescence in plant nuclei
Authors: Yoshinari, A.; Isoda, R.; Yagi, N.; Sato, Y.; Lindeboom, J. J.; Ehrhardt, D. W.; Frommer, W. B.; Nakamura, M.
Score: 26.7, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558255
Capturing images of the nuclear dynamics within live cells is an essential technique for comprehending the intricate biological processes inherent to plant cell nuclei. While various methods exist for imaging nuclei, including combining fluorescent proteins and dyes with microscopy, there is a dearth of commercially available dyes for live-cell imaging."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "Near-infrared imaging of phytochrome-derived autofluorescence in plant nuclei\nAuthors: Yoshinari, A.; Isoda, R.; Yagi, N.; Sato, Y.; Lindeboom, J. J.; Ehrhardt, D. W.; Frommer, W. B.; Nakamura, M.\nScore: 26.7, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558255\nCapturing images of the nuclear dynamics within live cells is an essential technique for comprehending the intricate biological processes inherent to plant cell nuclei. While various methods exist for imaging nuclei, including combining fluorescent proteins and dyes with microscopy, there is a dearth of commercially available dyes for live-cell imaging.",
  "keywords": [
    
  ],
  "articleBody": " Near-infrared imaging of phytochrome-derived autofluorescence in plant nuclei\nAuthors: Yoshinari, A.; Isoda, R.; Yagi, N.; Sato, Y.; Lindeboom, J. J.; Ehrhardt, D. W.; Frommer, W. B.; Nakamura, M.\nScore: 26.7, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558255\nCapturing images of the nuclear dynamics within live cells is an essential technique for comprehending the intricate biological processes inherent to plant cell nuclei. While various methods exist for imaging nuclei, including combining fluorescent proteins and dyes with microscopy, there is a dearth of commercially available dyes for live-cell imaging. In Arabidopsis thaliana, we discovered that nuclei emit autofluorescence in the near-infrared (NIR) range of the spectrum and devised a non-invasive technique for the visualization of live cell nuclei using this inherent NIR autofluorescence. Our studies demonstrated the capability of the NIR imaging technique to visualize the dynamic behavior of nuclei within primary roots, root hairs, and pollen tubes, which are tissues that harbor a limited number of other organelles displaying autofluorescence. We further demonstrated the applicability of NIR autofluorescence imaging in various other tissues by incorporating fluorescence lifetime imaging techniques. Nuclear autofluorescence was also detected across a wide range of plant species, enabling analyses without the need for transformation. The nuclear autofluorescence in the NIR wavelength range was not observed in animal or yeast cells. Genetic analysis revealed that this autofluorescence was caused by the phytochrome protein. Our studies demonstrated that nuclear autofluorescence imaging can be effectively employed not only in model plants but also for studying nuclei in non-model plant species.\nEvolution of wheat blast resistance gene Rmg8 accompanied by differentiation of variants recognizing the powdery mildew fungus\nAuthors: Asuke, S.; Morita, K.; Shimizu, M.; Abe, F.; Terauchi, R.; Nago, C.; Takahashi, Y.; Shibata, M.; Yoshioka, M.; Iwakawa, M.; Kishi-Kaboshi, M.; Su, Z.; Nasuda, S.; Handa, H.; Fujita, M.; Tougou, M.; Hatta, K.; Mori, N.; Matsuoka, Y.; Kato, K.; Tosa, Y.\nScore: 19.9, Published: 2023-09-28 DOI: 10.1101/2023.09.26.559445\nWheat blast, a devastating disease having spread recently from South America to Asia and Africa, is caused by Pyricularia oryzae pathotype Triticum which emerged in 1985. Rmg8 and Rmg7, genes for resistance to wheat blast found in common wheat and tetraploid wheat, respectively, recognize the same avirulence gene, AVR-Rmg8. Here, we show an evolutionary process in which resistance gene(s), which had obtained an ability to recognize AVR-Rmg8 before the differentiation of Triticum and Aegilops, has expanded its target pathogens. Molecular cloning revealed that Rmg7 was one of alleles of Pm4 (Pm4a), a gene for resistance to wheat powdery mildew on 2AL, whereas Rmg8 was its homoeolog on 2BL ineffective against wheat powdery mildew. Rmg8 variants with the ability to recognize AVR-Rmg8 were distributed not only in Triticum spp. but also in Aegilops speltoides, Ae. umbellulata, and Ae. comosa. This result suggests that the origin of resistance gene(s) recognizing AVR-Rmg8 dates back to the time before differentiation of A, B, S, U, and M genomes, that is, [~]5 million years before the emergence of its current target, the wheat blast fungus. Phylogenetic analyses suggested that, in the evolutionary process thereafter, some of their variants gained the ability to recognize the wheat powdery mildew fungus and evolved into genes for resistance to wheat powdery mildew.\nForward Genetic Stomatal CO2 Response Screen in Grass Brachypodium distachyon Reveals Central Function of a MAP Kinase in Early Stomatal CO2 Signal Transduction\nAuthors: Lopez, B. N. K.; Ceciliato, P. H. O.; Rangel, F.; Salem, E.; Kernig, K.; Takahashi, Y.; Chow, K.; Zhang, L.; Sidhom, M.; Seitz, C.; Sibout, R.; Laudencia, D.; Woods, D. P.; McCammon, J. A.; Vogel, J.; Schroeder, J. I.\nScore: 14.6, Published: 2023-09-29 DOI: 10.1101/2023.09.28.559842\nPlants respond to increased CO2 concentrations through rapid stomatal closure which can contribute to increased water use efficiency. Grasses display faster stomatal responses than dicots due to dumbbell-shaped guard cells flanked by subsidiary cells working in opposition. However, forward genetic screening for stomatal CO2 signaling mutants in grasses has not been reported. The grass model Brachypodium distachyon is closely related to agronomically important cereal crops, sharing largely collinear genomes. To gain insights into CO2 control mechanisms of stomatal movements in grasses, we developed a forward genetics screen on an EMS-mutagenized Brachypodium distachyon M5 population using infrared imaging to identify plants with altered canopy leaf temperature at elevated CO2. A chill1 mutant was isolated, exhibiting consistently cooler leaf temperatures than wildtype control plants after exposure to increased [CO2]. chill1 plants showed strongly impaired high CO2-induced stomatal closure, despite retaining a robust response to abscisic acid. Through bulked segregant whole-genome-sequencing analyses and generation and characterization of CRISPR-cas9 mutants, chill1 was mapped to a protein kinase, BdMPK5. The chill1 mutation impaired bdmpk5 protein-mediated CO2/HCO3- sensing in-vitro consistent with AlphaFold2-directed structural modeling. BdMPK5 is a key signaling component involved in CO2-induced stomatal movements, potentially functioning as a component of the CO2 sensor in grasses.\nSequence characterization of T, Bip, and Phbw demonstrates the role of MYB-bHLH-WD40 complexes and temperature in common bean seed color pattern formation\nAuthors: Parker, T.; Bolt, T.; Williams, T.; Penmetsa, R. V.; Mulube, M.; Palkovic, A.; Jochua, C. N.; Rubio Wilhelmi, M. d. M.; Lo, S.; Bornhorst, G.; Tian, L.; Kamfwa, K.; Hokin, S.; Farmer, A.; Diepenbrock, C. H.; Gepts, P.\nScore: 13.9, Published: 2023-09-29 DOI: 10.1101/2023.09.23.559151\nSeed colors and color patterns are critical for the survival of wild plants and the consumer appeal of crops. In common bean, a major global staple, these patterns are also critical for determining market classes, yet the genetic and environmental control of many pigmentation patterns remains unresolved. In this study, we genetically mapped variation for three important seed pattern loci, T, Bip, and phbw, which co-segregated completely with PvTTG1, PvMYC1, and PvTT8, respectively. Proteins encoded by these genes are predicted to work together in MYB-bHLH-WD40 (MBW) complexes, propagating flavonoid biosynthesis across the seed coat. Whole-genome sequencing of 37 diverse accessions identified putative mutations in each gene, including seven unique parallel mutations in T (PvTTG1) and a non-synonymous SNP in a conserved residue in bipana (PvMYC1). A 612 bp intron deletion in phbw (PvTT8) eliminated motifs conserved since the origins of the Papilionoidea and corresponded to a 20-fold reduction in transcript abundance. Mutations in MBW candidate genes for Z (PvTT2) and Sellatus (WDR) were also identified. In multi-location field trials with seven varieties with partial seed coat patterning, pigmented seed coat area was highly unstable and correlated with temperature, with up to 11-fold differences in pigmented area between the warmest and the coolest environments. In controlled growth chamber conditions, an increase of 4 {degrees}C was sufficient to cause pigmentation on an additional 21% of the seed coat area. Our results shed light on the fundamental activation of flavonoid biosynthesis in common bean and will be instrumental for maximizing consumer appeal in this nutritious staple crop. Summary- Seed colors and patterns are critical for the survival of wild plants, and are important in differentiating crop market classes, but the genetic control of these in the staple crop common bean (Phaseolus vulgaris) is largely unknown. - The genetic, transcriptional, and environmental basis of common bean seed color patterning was explored through QTL mapping, whole-genome sequencing, RT-qPCR, and automated pigmentation quantification of seed grown in multi-location field trials and growth chamber environments. - MYB-bHLH-WD40 complex-forming genes PvTTG1, PvMYC1, and PvTT8 co-segregated completely with the color patterning genes T, Bip, and phbw. Mutations were identified in each gene, including seven unique parallel mutations in T (PvTTG1), a non-synonymous SNP in a conserved residue in bipana (PvMYC1), and an intron deletion in phbw (PvTT8) eliminating highly conserved motifs and corresponding to 20-fold lower PvTT8 transcript abundance. Mutations in MBW candidate genes Z (PvTT2) and Sellatus (WDR) were also identified. In multi-location field trials, pigmented seed coat area was highly unstable and corresponded to temperature. In growth chamber conditions, an increase of 4 {degrees}C caused pigmentation on an additional 21% of the seed coat area. - Our results highlight the critical interaction between MYB-bHLH-WD40 complex components and temperature in establishing seed pattern diversity.\nPuzzle cell shape emerges from the interaction of growth with mechanical constraints\nAuthors: Trozzi, N.; Lane, B.; Perruchoud, A.; Wang, Y.; Hoermayer, L.; Ansel, M.; Mollier, C.; Malivert, A.; Clark, F.; Reichgelt, T.; Roeder, A. H. K.; Hamant, O.; Boudaoud, A.; Kwiatkowska, D.; Runions, A.; Smith, R. S.; Majda, M.\nScore: 9.2, Published: 2023-10-02 DOI: 10.1101/2023.10.01.560343\nThe puzzle-shaped cells found in the shoot epidermis of many plant species are a fascinating example of complex cell shapes. Because biological form often follows function, the unique shape of these cells suggests that they must serve some adaptive purpose for the plant. We previously proposed that these intricate shapes provide an effective strategy for reducing mechanical stress on the cell wall when epidermal cells undergo growth in more than one direction. Here we analyze a large selection of living and paleo plant species and find that the ability to make puzzle cells is a shared feature across all plant species, although their presence can be hidden as it varies depending on the organ, developmental stage, and environmental conditions. Computational modeling of Arabidopsis and maize epidermal cells revealed that presence and patterning of lobes is a dynamic process that is intricately linked to the growth history and environmental context of the plant organ. Conversely, disrupted lobeyness in mutants or with drug treatments affects plant development and leads to compensatory strategies. We propose that the mechanism underlying the formation of puzzle-shaped cells is likely conserved among higher plants and is a response to a developmental constraint driven by growth and mechanical stress.\nThe wheat powdery mildew resistance gene Pm4 also confers resistance to wheat blast\nAuthors: O'Hara, T.; Steed, A.; Goddard, R.; Gaurav, K.; Arora, S.; Quiroz-Chavez, J.; Ramirez-Gonzalez, R.; Badgami, R.; Gilbert, D.; Sanchez-Martin, J.; Wingen, L.; Feng, C.; Jiang, M.; Cheng, S.; Dreisigacker, S.; Keller, B.; Wulff, B.; Uauy, C.; Nicholson, P.\nScore: 8.6, Published: 2023-09-28 DOI: 10.1101/2023.09.26.559489\nWheat blast, caused by the fungus Magnaporthe oryzae, threatens global cereal production since its emergence in Brazil in 1985 and recently spread to Bangladesh and Zambia. Here we demonstrate that the AVR-Rmg8 effector, common in wheat-infecting isolates, is recognised by the gene Pm4, previously shown to confer resistance to specific races of Blumeria graminis f.sp. tritici, the cause of powdery mildew of wheat. We show that Pm4 alleles differ in their recognition of different AVR-Rmg8 alleles, and some confer resistance only in seedling leaves but not spikes making it important to select for those alleles that function in both tissues. This study has identified a gene recognising an important virulence factor present in wheat blast isolates in Bangladesh and Zambia and represents an important first step towards developing durably resistant wheat cultivars for these regions.\nAn updated model of shoot apical meristem regulation by ERECTA family and CLAVATA3 signaling pathways\nAuthors: Uzair, M.; Urquidi-Camacho, R. A.; Liu, Z.; Overholt, A. M.; DeGennaro, D.; Zhang, L.; Herron, B. S.; Hong, T.; Shpak, E. D.\nScore: 8.3, Published: 2023-09-30 DOI: 10.1101/2023.09.29.560237\nThe shoot apical meristem (SAM) gives rise to above-ground organs. The size of the SAM is relatively constant due to the balance of stem cell replenishment versus cell recruitment into developing organs. In angiosperms, the transcription factor WUSCHEL (WUS) promotes stem cell identity in the central zone of the SAM. WUS forms a negative feedback loop with a signaling pathway activated by CLAVATA3 (CLV3). In the periphery of the SAM, the ERECTA family (ERf) receptors promote cell differentiation and constrain the expression of WUS and CLV3. Here, we show that four ligands of ERfs redundantly inhibit CLV3 and WUS expression. Transcriptome analysis confirmed that WUS and CLV3 are the main targets of ERf signaling and uncovered several new ones. Analysis of promoter reporters indicated that in the vegetative meristem, the WUS expression domain mostly overlapped with the CLV3 domain and did not shift along the apical-basal axis in clv3. A 3D mathematical model reproduced the experimentally observed CLV3 and WUS expression patterns with fewer assumptions than earlier models. Based on these findings, we propose that CLV3 regulates cellular levels of WUS expression through autocrine signaling, while ERfs regulate WUS spatial expression, preventing its encroachment into the peripheral zone.\nThe FW2.2/CNR protein regulates cell-to-cell communication in tomato by modulating callose deposition at plasmodesmata\nAuthors: Beauchet, A.; Bollier, N.; Grison, M.; Rofidal, V.; Gevaudant, F.; Bayer, E. M. F.; Gonzalez, N.; Chevalier, C.\nScore: 8.0, Published: 2023-09-29 DOI: 10.1101/2023.09.27.559775\nThe FW2.2 gene is the founding member of the CELL NUMBER REGULATOR (CNR) gene family. More than 20 years ago, FW2.2 was the first cloned gene underlying a Quantitative Trait Locus (QTL) governing fruit size/weight in tomato. However, despite this discovery, the molecular mechanisms by which FW2.2 acts as a negative regulator of cell divisions during fruit growth remain undeciphered. In the present study, we confirm that FW2.2 is a transmembrane spanning protein, whose both N- and C-terminal ends are facing the apoplast. We unexpectedly found that FW2.2 is located at plasmodesmata (PD). FW2.2 participates in the spatiotemporal regulation of callose deposition at PD via an interaction with Callose Synthases, which suggests a regulatory role in cell-to-cell communication by modulating PD transport capacity and trafficking of signaling molecules during fruit development.\nCellular gibberellin dynamics govern indeterminate nodule development, morphologyand function\nAuthors: Drapek, C.; Radzman-Mohd, N. A.; Rizza, A.; Schiessl, K.; Dos Santos Barbosa, F.; Wen, J.; Oldroyd, G. E. D.; Jones, A. M.\nScore: 41.8, Published: 2023-09-11 DOI: 10.1101/2023.09.09.556959\nDuring nutrient scarcity, plants can adapt their developmental strategy to maximize their chance of survival. Such plasticity in development is underpinned by hormonal regulation, which mediates the relationship between environmental cues and developmental outputs. In legumes, endosymbiosis with nitrogen fixing bacteria (rhizobia) is a key adaptation for supplying the plant with nitrogen in the form of ammonium. Rhizobia are housed in lateral root-derived organs termed nodules that maintain an environment conducive to Nitrogenase in these bacteria. Several phytohormones are important for regulating the formation of nodules, with both positive and negative roles proposed for gibberellin (GA). In this study, we determined the cellular location and function of bioactive GA during nodule organogenesis using a genetically-encoded second generation GA biosensor, GPS2. We found endogenous bioactive GA accumulates locally at the site of nodule primordia, increasing dramatically in the cortical cell layers, persisting through cell divisions and maintaining accumulation in the mature nodule meristem. We show, through mis-expression of GA catabolic enzymes that suppress GA accumulation, that GA acts as a positive regulator of nodule growth and development. Furthermore, increasing or decreasing GA through perturbation of biosynthesis gene expression can increase or decrease the size of nodules, respectively. This is unique from lateral root formation, a developmental program that shares common organogenesis regulators. We link GA to a wider gene regulatory program by showing that cytokinin as well as nodule-identity genes induce and sustain GA accumulation necessary for nodule function.\nRhythmic lipid and gene expression responses to chilling in panicoid grasses\nAuthors: Kenchanmane Raju, S. K.; Zhang, Y.; Mahboub, S.; Ngu, D.; Qiu, Y.; Harmon, F.; Schnable, J. C.; Roston, R. L.\nScore: 6.0, Published: 2023-10-02 DOI: 10.1101/2023.09.29.560160\nChilling stress threatens plant growth and development, particularly affecting membrane fluidity and cellular integrity. Understanding plant membrane responses to chilling stress is important for unraveling the molecular mechanisms of stress tolerance. Whereas core transcriptional responses to chilling stress and stress tolerance are conserved across species, the associated changes in membrane lipids appear to be less conserved, as which lipids are affected by chilling stress varies by species. Here, we investigated changes in gene expression and membrane lipids in response to chilling stress during one diurnal cycle in sorghum (Sorghum bicolor), Urochloa (browntop signal grass, Urochloa fusca) (lipids only), and foxtail millet (Setaria italica), leveraging their evolutionary relatedness and differing levels of chilling-stress tolerance. We show that most chilling-induced lipid changes are conserved across the three species, while we observed distinct, time-specific responses in chilling-tolerant foxtail millet, indicating the presence of a finely orchestrated adaptive mechanism. We detected diurnal rhythmicity in lipid responses to chilling stress in the three grasses, which were also present in Arabidopsis (Arabidopsis thaliana), suggesting the conservation of rhythmic patterns across species and highlighting the importance of accounting for diurnal effects. When integrating lipid datasets with gene expression profiles, we identified potential candidate genes that showed corresponding transcriptional changes in response to chilling stress, providing insights into the differences in regulatory mechanisms between chilling-sensitive sorghum and chilling-tolerant foxtail millet.\n",
  "wordCount" : "2716",
  "inLanguage": "en",
  "datePublished": "2023-10-04T10:39:33Z",
  "dateModified": "2023-10-04T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta"><span>updated on October 4, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558255">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558255" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558255">
        <p class="paperTitle">Near-infrared imaging of phytochrome-derived autofluorescence in plant nuclei</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558255" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558255" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yoshinari, A.; Isoda, R.; Yagi, N.; Sato, Y.; Lindeboom, J. J.; Ehrhardt, D. W.; Frommer, W. B.; Nakamura, M.</p>
        <p class="info">Score: 26.7, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558255' target='https://doi.org/10.1101/2023.09.18.558255'> 10.1101/2023.09.18.558255</a></p>
        <p class="abstract">Capturing images of the nuclear dynamics within live cells is an essential technique for comprehending the intricate biological processes inherent to plant cell nuclei. While various methods exist for imaging nuclei, including combining fluorescent proteins and dyes with microscopy, there is a dearth of commercially available dyes for live-cell imaging. In Arabidopsis thaliana, we discovered that nuclei emit autofluorescence in the near-infrared (NIR) range of the spectrum and devised a non-invasive technique for the visualization of live cell nuclei using this inherent NIR autofluorescence. Our studies demonstrated the capability of the NIR imaging technique to visualize the dynamic behavior of nuclei within primary roots, root hairs, and pollen tubes, which are tissues that harbor a limited number of other organelles displaying autofluorescence. We further demonstrated the applicability of NIR autofluorescence imaging in various other tissues by incorporating fluorescence lifetime imaging techniques. Nuclear autofluorescence was also detected across a wide range of plant species, enabling analyses without the need for transformation. The nuclear autofluorescence in the NIR wavelength range was not observed in animal or yeast cells. Genetic analysis revealed that this autofluorescence was caused by the phytochrome protein. Our studies demonstrated that nuclear autofluorescence imaging can be effectively employed not only in model plants but also for studying nuclei in non-model plant species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559445">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559445" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559445">
        <p class="paperTitle">Evolution of wheat blast resistance gene Rmg8 accompanied by differentiation of variants recognizing the powdery mildew fungus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559445" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559445" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Asuke, S.; Morita, K.; Shimizu, M.; Abe, F.; Terauchi, R.; Nago, C.; Takahashi, Y.; Shibata, M.; Yoshioka, M.; Iwakawa, M.; Kishi-Kaboshi, M.; Su, Z.; Nasuda, S.; Handa, H.; Fujita, M.; Tougou, M.; Hatta, K.; Mori, N.; Matsuoka, Y.; Kato, K.; Tosa, Y.</p>
        <p class="info">Score: 19.9, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559445' target='https://doi.org/10.1101/2023.09.26.559445'> 10.1101/2023.09.26.559445</a></p>
        <p class="abstract">Wheat blast, a devastating disease having spread recently from South America to Asia and Africa, is caused by Pyricularia oryzae pathotype Triticum which emerged in 1985. Rmg8 and Rmg7, genes for resistance to wheat blast found in common wheat and tetraploid wheat, respectively, recognize the same avirulence gene, AVR-Rmg8. Here, we show an evolutionary process in which resistance gene(s), which had obtained an ability to recognize AVR-Rmg8 before the differentiation of Triticum and Aegilops, has expanded its target pathogens. Molecular cloning revealed that Rmg7 was one of alleles of Pm4 (Pm4a), a gene for resistance to wheat powdery mildew on 2AL, whereas Rmg8 was its homoeolog on 2BL ineffective against wheat powdery mildew. Rmg8 variants with the ability to recognize AVR-Rmg8 were distributed not only in Triticum spp. but also in Aegilops speltoides, Ae. umbellulata, and Ae. comosa. This result suggests that the origin of resistance gene(s) recognizing AVR-Rmg8 dates back to the time before differentiation of A, B, S, U, and M genomes, that is, [~]5 million years before the emergence of its current target, the wheat blast fungus. Phylogenetic analyses suggested that, in the evolutionary process thereafter, some of their variants gained the ability to recognize the wheat powdery mildew fungus and evolved into genes for resistance to wheat powdery mildew.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.559842">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.559842" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.559842">
        <p class="paperTitle">Forward Genetic Stomatal CO2 Response Screen in Grass Brachypodium distachyon Reveals Central Function of a MAP Kinase in Early Stomatal CO2 Signal Transduction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.559842" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.559842" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lopez, B. N. K.; Ceciliato, P. H. O.; Rangel, F.; Salem, E.; Kernig, K.; Takahashi, Y.; Chow, K.; Zhang, L.; Sidhom, M.; Seitz, C.; Sibout, R.; Laudencia, D.; Woods, D. P.; McCammon, J. A.; Vogel, J.; Schroeder, J. I.</p>
        <p class="info">Score: 14.6, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.559842' target='https://doi.org/10.1101/2023.09.28.559842'> 10.1101/2023.09.28.559842</a></p>
        <p class="abstract">Plants respond to increased CO2 concentrations through rapid stomatal closure which can contribute to increased water use efficiency. Grasses display faster stomatal responses than dicots due to dumbbell-shaped guard cells flanked by subsidiary cells working in opposition. However, forward genetic screening for stomatal CO2 signaling mutants in grasses has not been reported. The grass model Brachypodium distachyon is closely related to agronomically important cereal crops, sharing largely collinear genomes. To gain insights into CO2 control mechanisms of stomatal movements in grasses, we developed a forward genetics screen on an EMS-mutagenized Brachypodium distachyon M5 population using infrared imaging to identify plants with altered canopy leaf temperature at elevated CO2. A chill1 mutant was isolated, exhibiting consistently cooler leaf temperatures than wildtype control plants after exposure to increased [CO2]. chill1 plants showed strongly impaired high CO2-induced stomatal closure, despite retaining a robust response to abscisic acid. Through bulked segregant whole-genome-sequencing analyses and generation and characterization of CRISPR-cas9 mutants, chill1 was mapped to a protein kinase, BdMPK5. The chill1 mutation impaired bdmpk5 protein-mediated CO2/HCO3- sensing in-vitro consistent with AlphaFold2-directed structural modeling. BdMPK5 is a key signaling component involved in CO2-induced stomatal movements, potentially functioning as a component of the CO2 sensor in grasses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.23.559151">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.23.559151" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.23.559151">
        <p class="paperTitle">Sequence characterization of T, Bip, and Phbw demonstrates the role of MYB-bHLH-WD40 complexes and temperature in common bean seed color pattern formation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.23.559151" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.23.559151" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parker, T.; Bolt, T.; Williams, T.; Penmetsa, R. V.; Mulube, M.; Palkovic, A.; Jochua, C. N.; Rubio Wilhelmi, M. d. M.; Lo, S.; Bornhorst, G.; Tian, L.; Kamfwa, K.; Hokin, S.; Farmer, A.; Diepenbrock, C. H.; Gepts, P.</p>
        <p class="info">Score: 13.9, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.23.559151' target='https://doi.org/10.1101/2023.09.23.559151'> 10.1101/2023.09.23.559151</a></p>
        <p class="abstract">Seed colors and color patterns are critical for the survival of wild plants and the consumer appeal of crops. In common bean, a major global staple, these patterns are also critical for determining market classes, yet the genetic and environmental control of many pigmentation patterns remains unresolved. In this study, we genetically mapped variation for three important seed pattern loci, T, Bip, and phbw, which co-segregated completely with PvTTG1, PvMYC1, and PvTT8, respectively. Proteins encoded by these genes are predicted to work together in MYB-bHLH-WD40 (MBW) complexes, propagating flavonoid biosynthesis across the seed coat. Whole-genome sequencing of 37 diverse accessions identified putative mutations in each gene, including seven unique parallel mutations in T (PvTTG1) and a non-synonymous SNP in a conserved residue in bipana (PvMYC1). A 612 bp intron deletion in phbw (PvTT8) eliminated motifs conserved since the origins of the Papilionoidea and corresponded to a 20-fold reduction in transcript abundance. Mutations in MBW candidate genes for Z (PvTT2) and Sellatus (WDR) were also identified. In multi-location field trials with seven varieties with partial seed coat patterning, pigmented seed coat area was highly unstable and correlated with temperature, with up to 11-fold differences in pigmented area between the warmest and the coolest environments. In controlled growth chamber conditions, an increase of 4 {degrees}C was sufficient to cause pigmentation on an additional 21% of the seed coat area. Our results shed light on the fundamental activation of flavonoid biosynthesis in common bean and will be instrumental for maximizing consumer appeal in this nutritious staple crop.

Summary- Seed colors and patterns are critical for the survival of wild plants, and are important in differentiating crop market classes, but the genetic control of these in the staple crop common bean (Phaseolus vulgaris) is largely unknown.
- The genetic, transcriptional, and environmental basis of common bean seed color patterning was explored through QTL mapping, whole-genome sequencing, RT-qPCR, and automated pigmentation quantification of seed grown in multi-location field trials and growth chamber environments.
- MYB-bHLH-WD40 complex-forming genes PvTTG1, PvMYC1, and PvTT8 co-segregated completely with the color patterning genes T, Bip, and phbw. Mutations were identified in each gene, including seven unique parallel mutations in T (PvTTG1), a non-synonymous SNP in a conserved residue in bipana (PvMYC1), and an intron deletion in phbw (PvTT8) eliminating highly conserved motifs and corresponding to 20-fold lower PvTT8 transcript abundance. Mutations in MBW candidate genes Z (PvTT2) and Sellatus (WDR) were also identified. In multi-location field trials, pigmented seed coat area was highly unstable and corresponded to temperature. In growth chamber conditions, an increase of 4 {degrees}C caused pigmentation on an additional 21% of the seed coat area.
- Our results highlight the critical interaction between MYB-bHLH-WD40 complex components and temperature in establishing seed pattern diversity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.560343">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.560343" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.560343">
        <p class="paperTitle">Puzzle cell shape emerges from the interaction of growth with mechanical constraints</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.560343" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.560343" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Trozzi, N.; Lane, B.; Perruchoud, A.; Wang, Y.; Hoermayer, L.; Ansel, M.; Mollier, C.; Malivert, A.; Clark, F.; Reichgelt, T.; Roeder, A. H. K.; Hamant, O.; Boudaoud, A.; Kwiatkowska, D.; Runions, A.; Smith, R. S.; Majda, M.</p>
        <p class="info">Score: 9.2, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.560343' target='https://doi.org/10.1101/2023.10.01.560343'> 10.1101/2023.10.01.560343</a></p>
        <p class="abstract">The puzzle-shaped cells found in the shoot epidermis of many plant species are a fascinating example of complex cell shapes. Because biological form often follows function, the unique shape of these cells suggests that they must serve some adaptive purpose for the plant. We previously proposed that these intricate shapes provide an effective strategy for reducing mechanical stress on the cell wall when epidermal cells undergo growth in more than one direction. Here we analyze a large selection of living and paleo plant species and find that the ability to make puzzle cells is a shared feature across all plant species, although their presence can be hidden as it varies depending on the organ, developmental stage, and environmental conditions. Computational modeling of Arabidopsis and maize epidermal cells revealed that presence and patterning of lobes is a dynamic process that is intricately linked to the growth history and environmental context of the plant organ. Conversely, disrupted lobeyness in mutants or with drug treatments affects plant development and leads to compensatory strategies. We propose that the mechanism underlying the formation of puzzle-shaped cells is likely conserved among higher plants and is a response to a developmental constraint driven by growth and mechanical stress.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559489">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559489" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559489">
        <p class="paperTitle">The wheat powdery mildew resistance gene Pm4 also confers resistance to wheat blast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559489" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559489" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: O&#39;Hara, T.; Steed, A.; Goddard, R.; Gaurav, K.; Arora, S.; Quiroz-Chavez, J.; Ramirez-Gonzalez, R.; Badgami, R.; Gilbert, D.; Sanchez-Martin, J.; Wingen, L.; Feng, C.; Jiang, M.; Cheng, S.; Dreisigacker, S.; Keller, B.; Wulff, B.; Uauy, C.; Nicholson, P.</p>
        <p class="info">Score: 8.6, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559489' target='https://doi.org/10.1101/2023.09.26.559489'> 10.1101/2023.09.26.559489</a></p>
        <p class="abstract">Wheat blast, caused by the fungus Magnaporthe oryzae, threatens global cereal production since its emergence in Brazil in 1985 and recently spread to Bangladesh and Zambia. Here we demonstrate that the AVR-Rmg8 effector, common in wheat-infecting isolates, is recognised by the gene Pm4, previously shown to confer resistance to specific races of Blumeria graminis f.sp. tritici, the cause of powdery mildew of wheat. We show that Pm4 alleles differ in their recognition of different AVR-Rmg8 alleles, and some confer resistance only in seedling leaves but not spikes making it important to select for those alleles that function in both tissues. This study has identified a gene recognising an important virulence factor present in wheat blast isolates in Bangladesh and Zambia and represents an important first step towards developing durably resistant wheat cultivars for these regions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560237">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560237" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560237">
        <p class="paperTitle">An updated model of shoot apical meristem regulation by ERECTA family and CLAVATA3 signaling pathways</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560237" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560237" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Uzair, M.; Urquidi-Camacho, R. A.; Liu, Z.; Overholt, A. M.; DeGennaro, D.; Zhang, L.; Herron, B. S.; Hong, T.; Shpak, E. D.</p>
        <p class="info">Score: 8.3, Published: 2023-09-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560237' target='https://doi.org/10.1101/2023.09.29.560237'> 10.1101/2023.09.29.560237</a></p>
        <p class="abstract">The shoot apical meristem (SAM) gives rise to above-ground organs. The size of the SAM is relatively constant due to the balance of stem cell replenishment versus cell recruitment into developing organs. In angiosperms, the transcription factor WUSCHEL (WUS) promotes stem cell identity in the central zone of the SAM. WUS forms a negative feedback loop with a signaling pathway activated by CLAVATA3 (CLV3). In the periphery of the SAM, the ERECTA family (ERf) receptors promote cell differentiation and constrain the expression of WUS and CLV3. Here, we show that four ligands of ERfs redundantly inhibit CLV3 and WUS expression. Transcriptome analysis confirmed that WUS and CLV3 are the main targets of ERf signaling and uncovered several new ones. Analysis of promoter reporters indicated that in the vegetative meristem, the WUS expression domain mostly overlapped with the CLV3 domain and did not shift along the apical-basal axis in clv3. A 3D mathematical model reproduced the experimentally observed CLV3 and WUS expression patterns with fewer assumptions than earlier models. Based on these findings, we propose that CLV3 regulates cellular levels of WUS expression through autocrine signaling, while ERfs regulate WUS spatial expression, preventing its encroachment into the peripheral zone.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559775">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559775" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559775">
        <p class="paperTitle">The FW2.2/CNR protein regulates cell-to-cell communication in tomato by modulating callose deposition at plasmodesmata</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559775" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559775" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Beauchet, A.; Bollier, N.; Grison, M.; Rofidal, V.; Gevaudant, F.; Bayer, E. M. F.; Gonzalez, N.; Chevalier, C.</p>
        <p class="info">Score: 8.0, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559775' target='https://doi.org/10.1101/2023.09.27.559775'> 10.1101/2023.09.27.559775</a></p>
        <p class="abstract">The FW2.2 gene is the founding member of the CELL NUMBER REGULATOR (CNR) gene family. More than 20 years ago, FW2.2 was the first cloned gene underlying a Quantitative Trait Locus (QTL) governing fruit size/weight in tomato. However, despite this discovery, the molecular mechanisms by which FW2.2 acts as a negative regulator of cell divisions during fruit growth remain undeciphered. In the present study, we confirm that FW2.2 is a transmembrane spanning protein, whose both N- and C-terminal ends are facing the apoplast. We unexpectedly found that FW2.2 is located at plasmodesmata (PD). FW2.2 participates in the spatiotemporal regulation of callose deposition at PD via an interaction with Callose Synthases, which suggests a regulatory role in cell-to-cell communication by modulating PD transport capacity and trafficking of signaling molecules during fruit development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.556959">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.556959" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.556959">
        <p class="paperTitle">Cellular gibberellin dynamics govern indeterminate nodule development, morphologyand function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.556959" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.556959" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Drapek, C.; Radzman-Mohd, N. A.; Rizza, A.; Schiessl, K.; Dos Santos Barbosa, F.; Wen, J.; Oldroyd, G. E. D.; Jones, A. M.</p>
        <p class="info">Score: 41.8, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.556959' target='https://doi.org/10.1101/2023.09.09.556959'> 10.1101/2023.09.09.556959</a></p>
        <p class="abstract">During nutrient scarcity, plants can adapt their developmental strategy to maximize their chance of survival. Such plasticity in development is underpinned by hormonal regulation, which mediates the relationship between environmental cues and developmental outputs. In legumes, endosymbiosis with nitrogen fixing bacteria (rhizobia) is a key adaptation for supplying the plant with nitrogen in the form of ammonium. Rhizobia are housed in lateral root-derived organs termed nodules that maintain an environment conducive to Nitrogenase in these bacteria. Several phytohormones are important for regulating the formation of nodules, with both positive and negative roles proposed for gibberellin (GA). In this study, we determined the cellular location and function of bioactive GA during nodule organogenesis using a genetically-encoded second generation GA biosensor, GPS2. We found endogenous bioactive GA accumulates locally at the site of nodule primordia, increasing dramatically in the cortical cell layers, persisting through cell divisions and maintaining accumulation in the mature nodule meristem. We show, through mis-expression of GA catabolic enzymes that suppress GA accumulation, that GA acts as a positive regulator of nodule growth and development. Furthermore, increasing or decreasing GA through perturbation of biosynthesis gene expression can increase or decrease the size of nodules, respectively. This is unique from lateral root formation, a developmental program that shares common organogenesis regulators. We link GA to a wider gene regulatory program by showing that cytokinin as well as nodule-identity genes induce and sustain GA accumulation necessary for nodule function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560160">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560160" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560160">
        <p class="paperTitle">Rhythmic lipid and gene expression responses to chilling in panicoid grasses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560160" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560160" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kenchanmane Raju, S. K.; Zhang, Y.; Mahboub, S.; Ngu, D.; Qiu, Y.; Harmon, F.; Schnable, J. C.; Roston, R. L.</p>
        <p class="info">Score: 6.0, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560160' target='https://doi.org/10.1101/2023.09.29.560160'> 10.1101/2023.09.29.560160</a></p>
        <p class="abstract">Chilling stress threatens plant growth and development, particularly affecting membrane fluidity and cellular integrity. Understanding plant membrane responses to chilling stress is important for unraveling the molecular mechanisms of stress tolerance. Whereas core transcriptional responses to chilling stress and stress tolerance are conserved across species, the associated changes in membrane lipids appear to be less conserved, as which lipids are affected by chilling stress varies by species. Here, we investigated changes in gene expression and membrane lipids in response to chilling stress during one diurnal cycle in sorghum (Sorghum bicolor), Urochloa (browntop signal grass, Urochloa fusca) (lipids only), and foxtail millet (Setaria italica), leveraging their evolutionary relatedness and differing levels of chilling-stress tolerance. We show that most chilling-induced lipid changes are conserved across the three species, while we observed distinct, time-specific responses in chilling-tolerant foxtail millet, indicating the presence of a finely orchestrated adaptive mechanism. We detected diurnal rhythmicity in lipid responses to chilling stress in the three grasses, which were also present in Arabidopsis (Arabidopsis thaliana), suggesting the conservation of rhythmic patterns across species and highlighting the importance of accounting for diurnal effects. When integrating lipid datasets with gene expression profiles, we identified potential candidate genes that showed corresponding transcriptional changes in response to chilling stress, providing insights into the differences in regulatory mechanisms between chilling-sensitive sorghum and chilling-tolerant foxtail millet.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
