<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Mucilage produced by sorghum (Sorghum bicolor) aerial roots supports a nitrogen-fixing community
Authors: Venado, R. E.; Wilker, J.; Pankievicz, V.; Infante, V.; MacIntyre, A.; Wolf, E.; Vela, S.; Robbins, F. A.; Fernandes-Junior, P. I.; Vermerris, W.; Ane, J.-M.
Score: 74.6, Published: 2023-08-06 DOI: 10.1101/2023.08.05.552127
O_LISorghum (Sorghum bicolor) is gaining popularity as a sustainable energy crop due to its high biomass and potential for biofuel production. Some rare sorghum accessions develop many aerial roots that produce viscous carbohydrate-rich mucilage after rain.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="Mucilage produced by sorghum (Sorghum bicolor) aerial roots supports a nitrogen-fixing community
Authors: Venado, R. E.; Wilker, J.; Pankievicz, V.; Infante, V.; MacIntyre, A.; Wolf, E.; Vela, S.; Robbins, F. A.; Fernandes-Junior, P. I.; Vermerris, W.; Ane, J.-M.
Score: 74.6, Published: 2023-08-06 DOI: 10.1101/2023.08.05.552127
O_LISorghum (Sorghum bicolor) is gaining popularity as a sustainable energy crop due to its high biomass and potential for biofuel production. Some rare sorghum accessions develop many aerial roots that produce viscous carbohydrate-rich mucilage after rain." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-13T10:36:01+00:00" />
<meta property="article:modified_time" content="2023-08-13T10:36:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="Mucilage produced by sorghum (Sorghum bicolor) aerial roots supports a nitrogen-fixing community
Authors: Venado, R. E.; Wilker, J.; Pankievicz, V.; Infante, V.; MacIntyre, A.; Wolf, E.; Vela, S.; Robbins, F. A.; Fernandes-Junior, P. I.; Vermerris, W.; Ane, J.-M.
Score: 74.6, Published: 2023-08-06 DOI: 10.1101/2023.08.05.552127
O_LISorghum (Sorghum bicolor) is gaining popularity as a sustainable energy crop due to its high biomass and potential for biofuel production. Some rare sorghum accessions develop many aerial roots that produce viscous carbohydrate-rich mucilage after rain."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "Mucilage produced by sorghum (Sorghum bicolor) aerial roots supports a nitrogen-fixing community\nAuthors: Venado, R. E.; Wilker, J.; Pankievicz, V.; Infante, V.; MacIntyre, A.; Wolf, E.; Vela, S.; Robbins, F. A.; Fernandes-Junior, P. I.; Vermerris, W.; Ane, J.-M.\nScore: 74.6, Published: 2023-08-06 DOI: 10.1101/2023.08.05.552127\nO_LISorghum (Sorghum bicolor) is gaining popularity as a sustainable energy crop due to its high biomass and potential for biofuel production. Some rare sorghum accessions develop many aerial roots that produce viscous carbohydrate-rich mucilage after rain.",
  "keywords": [
    
  ],
  "articleBody": " Mucilage produced by sorghum (Sorghum bicolor) aerial roots supports a nitrogen-fixing community\nAuthors: Venado, R. E.; Wilker, J.; Pankievicz, V.; Infante, V.; MacIntyre, A.; Wolf, E.; Vela, S.; Robbins, F. A.; Fernandes-Junior, P. I.; Vermerris, W.; Ane, J.-M.\nScore: 74.6, Published: 2023-08-06 DOI: 10.1101/2023.08.05.552127\nO_LISorghum (Sorghum bicolor) is gaining popularity as a sustainable energy crop due to its high biomass and potential for biofuel production. Some rare sorghum accessions develop many aerial roots that produce viscous carbohydrate-rich mucilage after rain. C_LIO_LIThis aerial root mucilage is strikingly similar to that observed in specific landraces of maize (Zea mays) from southern Mexico, which have been previously shown to host nitrogen-fixing bacteria (diazotroph). The landraces displaying these traits can reduce nitrogen-based fertilizer input, mitigating their negative environmental impacts. C_LIO_LIIn this study, we characterized the aerial root development of several sorghum accessions and successfully isolated more than 103 distinct diazotrophs from the sorghum mucilage. C_LIO_LIUsing acetylene reduction and 15N gas enrichment assays, we confirmed that sorghum plants acquire nitrogen from the atmosphere through the diazotrophic associations in the mucilage. C_LIO_LIThis sorghum symbiotic relationships with diazotrophs offer a promising avenue for nitrogen fixation, potentially diminishing reliance on synthetic fertilizers and promoting sustainable agricultural practices. C_LI\nGreenGate 2.0: backwards compatible addons for assembly of complex transcriptional units and their stacking with GreenGate\nAuthors: Piepers, M.; Erbstein, K.; Reyes-Hernandez, J.; Song, C.; Tessi, T.; Petrasiunaite, V.; Faerber, N.; Distel, K.; Maizel, A.\nScore: 47.3, Published: 2023-08-04 DOI: 10.1101/2023.08.02.551719\nMolecular cloning is a crucial technique in genetic engineering that enables the precise design of synthetic transcriptional units (TUs) and the manipulation of genomes. GreenGate and several other modular molecular cloning systems were developed about ten years ago and are widely used in plant research. All these systems define grammars for assembling transcriptional units from building blocks, cloned as Level 0 modules flanked by four-base pair overhangs and recognition sites for a particular Type IIs endonuclease. Modules are efficiently assembled into Level 1 TUs in a hierarchical assembly process, and Level 2 multigene constructs are assembled by stacking Level 1 TUs. GreenGate is highly popular but has three main limitations. First, using ad-hoc overhangs added by PCR and classical restriction/ligation prevents the efficient use of a one-pot, one-step reaction to generate entry clones and domesticate internal sites; second, a Level 1 TU is assembled from a maximum of six modules, which may be limiting for applications such as multiplex genome editing; third, the generation of Level 2 assemblies is sequential and inefficient. GreenGate 2.0 (GG2.0) expands GreenGate features. It introduces additional overhangs, allowing for the combination of up to 12 Level 0 modules in a Level 1 TU. It includes a Universal Entry Generator plasmid (pUEG) to streamline the generation of Level 0 modules. GG2.0 introduces GreenBraid, a convenient method for stacking transcriptional units iteratively for multigene assemblies. Importantly, GG2.0 is backwards compatible with most existing GreenGate modules. Additionally, GG2.0 includes Level 0 modules for multiplex expression of guide RNAs for CRISPR/Cas9 genome editing and pre-assembled Level 1 vectors for dexamethasone-inducible gene expression and ubiquitous expression of plasma membrane and nuclear fluorescent markers. GG2.0 streamlines and increases the versatility of assembling complex transcriptional units and their combination.\nDiscovery of an isoflavone oxidative catabolic pathway in legume root microbiota\nAuthors: Aoki, N.; Shimasaki, T.; Yazaki, W.; Sato, T.; Nakayasu, M.; Ando, A.; Kishino, S.; Ogawa, J.; Masuda, S.; Shibata, A.; Shirasu, K.; Yazaki, K.; Sugiyama, A.\nScore: 8.6, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552369\nIsoflavones are major specialized metabolites found in legume plants, where they contribute to environmental adaptation. Isoflavones also play a role human health as promising therapeutic agents. This metabolite group is involved in interactions with soil microorganisms as initiation signals in rhizobial symbiosis and as modulators of the legume root microbiota. We previously reported that isoflavones enrich the Comamonadaceae, a predominant bacterial family in soybean roots, and that microorganisms in legume rhizosphere soil degrade isoflavones. However, the isoflavone catabolism pathway that underly the isoflavone-mediated legume-microbiota interactions have not yet been clarified. Here, we isolated Variovorax sp. strain V35, member of the Comamonadaceae that harbors isoflavone-degrading activity, from soybean roots and discovered a gene cluster responsible for isoflavone degradation named ifc. Strain V35 metabolizes isoflavones in a completely distinct oxidative manner from the reductive isoflavone metabolism pathway elucidated in the gut microbiota, in which resulting products enter the tricarboxylic acid cycle. The characterization of ifc mutants and heterologously expressed IFC enzymes revealed that isoflavones are catabolized via A-ring cleaving fission, which starts with hydroxylation at the 8-position of the A-ring. We further demonstrated that ifc genes are frequently found in bacterial strains isolated from legume plants, including mutualistic rhizobia, and contribute to detoxification of the antibacterial activity of isoflavones. Taken together, our findings reveal an oxidative catabolism pathway of isoflavone in the soybean root microbiota, providing molecular insights into isoflavone-mediated legume-microbiota interactions. SignificanceIsoflavones play pivotal roles in plant-environment interactions and in the maintenance and improvement of human health. Bacterial metabolism is a fundamental component of isoflavone-mediated interkingdom interactions. In the human gut, intestinal bacteria convert isoflavones into equol, a highly bioactive compound. However, the fate of isoflavones in the legume rhizosphere has not been elucidated, despite them being the key signaling molecules for nodule symbiosis and modulation of the legume root microbiota. Here, we discovered a novel isoflavone catabolism pathway in the soybean root microbiota and demonstrated the strong association between bacterial catabolic abilities and their interactions with host plants. Collectively, our findings provide new insights into bacterial isoflavone metabolism and a molecular understanding of legume-microbiota interactions.\nAlgal CO2 capture is powered by alternative electron pathways of photosynthesis\nAuthors: Peltier, G.; Stoffel, C.; Findinier, J.; Madireddi, S. K.; Dao, O.; Epting, V.; Grossman, A.; Li-Beisson, Y.; Burlacot, A.\nScore: 8.3, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552514\nOn Earth, microalgae contribute to about half of global net photosynthesis. During photosynthesis, sunlight is converted into chemical energy (ATP and NADPH) used by metabolism to convert CO2 into biomass. Alternative electron pathways of photosynthesis have been proposed to generate additional ATP that is required for sustaining CO2 fixation, but the relative importance of each pathway remains elusive. Here, we dissect and quantify the contribution of cyclic, pseudo-cyclic and chloroplast to mitochondria electron flows for their ability to sustain net photosynthesis in the microalga Chlamydomonas reinhardtii. We show that each pathway has the potential to energize substantial CO2 fixation, can compensate each other, and that the additional energy requirement to fix CO2 is more than 3 times higher than previous estimations. We further show that all pathways have very different efficiencies at energizing CO2 fixation, with the chloroplast-mitochondria interaction being the most efficient, thus laying bioenergetic foundations for biotechnological improvement of CO2 capture.\nInsights into stereoselective ring formation in canonical strigolactone: Discovery of a dirigent domain-containing enzyme catalyzing orobanchol synthesis\nAuthors: Homma, M.; Wakabayashi, T.; Moriwaki, Y.; Shiotani, N.; Shigeta, T.; Isobe, K.; Okazawa, A.; Ohta, D.; Terada, T.; Shimizu, K.; Mizutani, M.; Takikawa, H.; Sugimoto, Y.\nScore: 7.4, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552212\nStrigolactones (SLs) are plant apocarotenoids with diverse functions and structures. The widespread canonical SLs, with distinctive structural variations in their tricyclic lactone known as the ABC-ring, are classified into two types based on the C-ring configurations. The steric C-ring configuration arises during the BC-ring closure downstream of carlactonoic acid (CLA), a biosynthetic intermediate. Most plants stereoselectively produce either type of canonical SLs, e.g., tomato (Solanum lycopersicum) produces orobanchol with -oriented C-ring. The mechanisms governing SL structural diversification are partly understood, with limited insight into the functional implications. Moreover, the precise molecular mechanism for the stereoselective BC-ring closure reaction remains unknown. Herein, we identified an enzyme called the stereoselective BC-ring-forming factor (SRF) from the dirigent protein (DIR) family, especially the DIR-f subfamily, whose biochemical function was previously unidentified, making it a pivotal enzyme in stereoselective canonical SL biosynthesis with the -oriented C-ring. We begin by confirming the exact catalytic function of the tomato cytochrome P450 SlCYP722C, which we previously demonstrated to be involved in the orobanchol biosynthesis [Wakabayashi et al., Sci. Adv. 5, eaax9067 (2019)], to convert CLA to 18-oxocarlactonoic acid. Subsequently, we demonstrate that SRF catalyzes the stereoselective BC-ring closure reaction of 18-oxocarlactonoic acid to form orobanchol. Our approach integrates experimental and computational methods, including SRF structure prediction and molecular dynamics simulations, to propose a catalytic mechanism based on the conrotatory 4{pi}-electrocyclic reaction for stereoselective BC-ring formation in orobanchol. The present study provides insight into the molecular basis of how plants produce SLs with specific stereochemistry in a controlled manner.\nOryza genera-specific novel Histone H4 variant predisposes H4 Lysine5 Acetylation marks to modulate salt stress responses\nAuthors: G, V. H. S.; Sotelo-Parrilla, P.; Raju, S.; Jha, S.; Gireesh, A.; Gut, F.; Kutti, V. R.; Berger, F.; Jeyaprakash, A. A.; Shivaprasad, P. V.\nScore: 24.0, Published: 2023-08-01 DOI: 10.1101/2023.07.31.551207\nParalogous variants of canonical histones guide accessibility to DNA and function as additional layers of genome regulation. Across eukaryotes, the occurrence, mechanism of action and functional significance of several variants of core histones are well known except that of histone H4. Here we show that a novel variant of H4 (H4.V), expressing tissue-specifically among members of Oryza genera, mediates specific epigenetic changes contributing majorly to salt tolerance. H4.V was incorporated to specific chromosomal locations where it blocked deposition of active histone marks. Under salt stress, large scale re-distribution of H4.V enabled incorporation of stress dependent histone H4 Lysine5 Acetylation (H4K5Ac) marks. Mis-expression of H4.V led to defects at morphological level especially in reproductive tissues, and in mounting stress responses. H4.V mediated these alterations by condensing chromatin at specific genomic regions as seen with cryo-EM structure of reconstituted H4.V containing nucleosomes. These results not only uncovered the presence of a H4 variant in plants, but also a novel chromatin regulation of stress responses that might have contributed to success of semi-aquatic Oryza members under variable water-limiting conditions. One-line summaryHistone H4 variant predisposes chromatin for stress responses O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=199 SRC=\"FIGDIR/small/551207v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (53K): org.highwire.dtl.DTLVardef@101fa0forg.highwire.dtl.DTLVardef@c7fda9org.highwire.dtl.DTLVardef@1de9a0aorg.highwire.dtl.DTLVardef@1532641_HPS_FORMAT_FIGEXP M_FIG C_FIG\nThe genomes and epigenomes of aquatic plants (Lemnaceae) promote triploid hybridization and clonal reproduction\nAuthors: Ernst, E.; Abramson, B.; Acosta, K.; Hoang, P. T.; Mateo-Elizalde, C.; Schubert, V.; Pasaribu, B.; Hartwick, N.; Colt, K.; Aylward, A. J.; Ramu, U.; Birchler, J. A.; Schubert, I.; Lam, E.; Michael, T. P.; Martienssen, R. A.\nScore: 8.2, Published: 2023-08-05 DOI: 10.1101/2023.08.02.551673\nThe Lemnaceae (duckweeds) are the worlds smallest but fastest growing flowering plants, with a drastically reduced morphology and predominant clonal reproductive habit capable of continuous exponential growth. Here, we present assemblies of 10 Lemna chromosome sets by single molecule nanopore sequencing and chromosome conformation capture. Dynamics of genome evolution in the family are revealed by syntenic comparisons with Wolffia and Spirodela, and diversification of these genera was found to coincide with the \"Azolla event\", in which blooms of aquatic macrophytes reduced atmospheric CO2 from greenhouse levels found in the Eocene to those of the current ice age. Orthologous gene comparisons with other aquatic and terrestrial plants uncovered candidate genes for the unique metabolic and developmental features of the family, such as frequent hybrid polyploidy, lack of stomatal closure in high CO2, and accumulation of calcium oxalate, a promising candidate for carbon sequestration. Loss of a spermine-triggered gene network may account for drastic reduction in stature and preferentially adaxial stomata, a feature of floating aquatic plants. Strikingly, Lemnaceae genomes have selectively lost some of the genes required for RNA interference, including Argonaute genes required for post-zygotic reproductive isolation (the triploid block) and reduced gamete formation. Triploid hybrids arise commonly among Lemna, presumably by hybridization with unreduced gametes, and we have found mutations in highly-conserved ZMM crossover pathway genes that could support polyploid meiosis. Rapid but stable clonal propagation makes Lemna an ideal platform for continuous protein and starch micro-cropping, and for efficient sequestration of dissolved nutrients and atmospheric CO2. Facile regeneration of transgenic fronds from tissue culture, aided by reduced epigenetic silencing, makes Lemna a powerful biotechnological platform, as exemplified by recent engineering of high-oil Lemna that out-perform oil seed crops.\nSpatial Co-transcriptomics Reveals Discrete Stages of the Arbuscular Mycorrhizal Symbiosis\nAuthors: Serrano, K.; Bezrutczyk, M.; Goudeau, D.; Dao, T.; O'Malley, R. C.; Malmstrom, R. R.; Visel, A.; Scheller, H. V.; Cole, B.\nScore: 20.4, Published: 2023-08-07 DOI: 10.1101/2023.08.02.551648\nThe symbiotic interaction of plants with arbuscular mycorrhizal fungi (AM fungi) is ancient and widespread. Plants provide AM fungi with carbon in exchange for nutrients and water, making this interaction a prime target for crop improvement. However, plant-fungal interactions are restricted to a small subset of root cells, precluding the application of most conventional functional genomic techniques to study the molecular bases of these interactions. Here we used single-nucleus and spatial RNA sequencing to explore both M. truncatula and R. irregularis transcriptomes in AM symbiosis at cellular and spatial resolution. Integrated spatially-registered single-cell maps of interacting cells revealed major infected and uninfected plant root cell types. We observed that cortex cells exhibit distinct transcriptome profiles during different stages of colonization by AM fungi, indicating dynamic interplay between both organisms during establishment of the cellular interface enabling successful symbiosis. Our study provides insight into a symbiotic relationship of major agricultural and environmental importance and demonstrates a paradigm combining single-cell and spatial transcriptomics for the analysis of complex organismal interactions.\nComparative Analysis, Diversification and Functional Validation of Plant Nucleotide-Binding Site Domain Genes\nAuthors: Hussain, A.; Khan, A. A.; Aslam, M. Q.; Nazar, A.; Zaman, N.; Amin, A.; Mahmood, M. A.; Mukhtar, M. S.; Rahman, H. U. U.; Farooq, M.; Saeed, M.; Amin, I.; Mansoor, S.\nScore: 4.3, Published: 2023-08-09 DOI: 10.1101/2023.08.09.552572\nNucleotide-binding site (NBS) domain genes are one of a superfamily of resistance genes involved in plant responses to pathogens. The current study identified presumably identified 12,820 NBS-containing genes across 34 species covering from mosses to monocots and dicots. These identified genes classified into 168 classes with several novel domain architectures patterns encompassing significant diversity among plant species. Several classical (NBS, NBS-LRR, TIR-NBS, TIR-NBS-LRR etc.) and species-specific structural patterns (TIR-NBS-TIR-Cupin_1-Cupin_1, TIR-NBS-Prenyltransf, Sugar_tr-NBS etc.) were discovered. We observed 603 orthogroups (OGs) with some core (most common orthogroups; OG0, OG1, OG2 etc.) and unique (highly specific to species; OG80, OG82 etc) OGs with tandem duplications. The expression profiling presented the putative upregulation of OG2, OG6, and OG15 in different tissues under various biotic and abiotic stresses in susceptible and tolerant plants to CLCuD. The genetic variation between susceptible (Coker 312) and tolerant (Mac7) G. hirsutum accessions identified several unique variants in NBS genes of Mac7 (6,583 varaints) and Coker312 (5,173 variants). The protein-ligand and proteins-protein interaction showed a strong interaction of some putative NBS proteins with ADP/ATP and different core proteins of cotton leaf curl disease virus. The silencing of GaNBS (OG2) in resistant cotton through virus-induced gene silencing (VIGS) demonstrated its putative role in virus tittering. The presented study will be further helpful to understand the plant adaptation mechanism.\nCell wall integrity modulates a PHYTOCHROME-INTERACTING FACTOR (PIF) - HOOKLESS1 (HLS1) signalling module controlling apical hook formation in Arabidopsis.\nAuthors: Lorrai, R.; Erguvan, O.; Raggi, S.; Jonsson, K.; Siroka, J.; Tarkowska, D.; Novak, O.; Verger, S.; Robert, S.; Ferrari, S.\nScore: 4.2, Published: 2023-08-06 DOI: 10.1101/2023.08.05.551077\nEtiolated seedlings of dicots form an apical hook to protect the meristems during soil emergence. Hook formation is the result of differential growth on both sides of the hypocotyl apex and is tightly controlled by environmental cues and hormones, among which auxin and gibberellins (GAs) are the main contributors. Cell expansion is tightly regulated by the cell wall, but whether and how feedback from this structure contributes to hook development is still unclear. Here we show that etiolated seedlings of the Arabidopsis thaliana quasimodo2-1 (qua2) mutant, defective in pectin biosynthesis, display severe defects in apical hook formation and maintenance, accompanied by loss of asymmetric auxin maxima and differential cell expansion. Moreover, qua2 seedlings show reduced expression of HOOKLESS1 (HLS1) and PHYTOCHROME-INTERACTING FACTOR 4 and 5 (PIF4/5), positive regulators of hook formation, and accumulate reduced levels of the active gibberellin GA4. Treatment of wild-type seedlings with the cellulose inhibitor isoxaben (isx) also prevents hook development and represses HLS1 expression and PIF4 accumulation. Moreover, isx stabilizes the DELLA protein REPRESSOR OF ga1-3 (RGA), which inhibits HLS1 expression and hook formation. Exogenous GAs or HLS1 overexpression partially restore hook development in isx-treated seedlings. Notably, agar concentration in the medium restores, both in qua2 and isx-treated seedlings, hook development and WT-like levels of PIFs and HLS1. We propose that turgor-dependent signals link changes in cell wall integrity to the PIF4/5-HLS1 signalling module to repress differential cell elongation during hook formation. Significance statementCell wall integrity modulates apical hook development through poorly understood mechanisms. We show here that, in Arabidopsis, repression of hook formation by either mutations in pectin biosynthesis or by isoxaben treatment is at least partially mediated by the downregulation of a gibberellin-controlled signalling module that comprises PIF4/5 and HLS1. Our results indicate that the signals derived from changes in the cell wall can modulate hormone-mediated pathways to control asymmetric growth during plant development.\n",
  "wordCount" : "2822",
  "inLanguage": "en",
  "datePublished": "2023-08-13T10:36:01Z",
  "dateModified": "2023-08-13T10:36:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta"><span>updated on August 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.552127">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.552127" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.552127">
        <p class="paperTitle">Mucilage produced by sorghum (Sorghum bicolor) aerial roots supports a nitrogen-fixing community</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.552127" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.552127" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Venado, R. E.; Wilker, J.; Pankievicz, V.; Infante, V.; MacIntyre, A.; Wolf, E.; Vela, S.; Robbins, F. A.; Fernandes-Junior, P. I.; Vermerris, W.; Ane, J.-M.</p>
        <p class="info">Score: 74.6, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.552127' target='https://doi.org/10.1101/2023.08.05.552127'> 10.1101/2023.08.05.552127</a></p>
        <p class="abstract">O_LISorghum (Sorghum bicolor) is gaining popularity as a sustainable energy crop due to its high biomass and potential for biofuel production. Some rare sorghum accessions develop many aerial roots that produce viscous carbohydrate-rich mucilage after rain.
C_LIO_LIThis aerial root mucilage is strikingly similar to that observed in specific landraces of maize (Zea mays) from southern Mexico, which have been previously shown to host nitrogen-fixing bacteria (diazotroph). The landraces displaying these traits can reduce nitrogen-based fertilizer input, mitigating their negative environmental impacts.
C_LIO_LIIn this study, we characterized the aerial root development of several sorghum accessions and successfully isolated more than 103 distinct diazotrophs from the sorghum mucilage.
C_LIO_LIUsing acetylene reduction and 15N gas enrichment assays, we confirmed that sorghum plants acquire nitrogen from the atmosphere through the diazotrophic associations in the mucilage.
C_LIO_LIThis sorghum symbiotic relationships with diazotrophs offer a promising avenue for nitrogen fixation, potentially diminishing reliance on synthetic fertilizers and promoting sustainable agricultural practices.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551719">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551719" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551719">
        <p class="paperTitle">GreenGate 2.0: backwards compatible addons for assembly of complex transcriptional units and their stacking with GreenGate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551719" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551719" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Piepers, M.; Erbstein, K.; Reyes-Hernandez, J.; Song, C.; Tessi, T.; Petrasiunaite, V.; Faerber, N.; Distel, K.; Maizel, A.</p>
        <p class="info">Score: 47.3, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551719' target='https://doi.org/10.1101/2023.08.02.551719'> 10.1101/2023.08.02.551719</a></p>
        <p class="abstract">Molecular cloning is a crucial technique in genetic engineering that enables the precise design of synthetic transcriptional units (TUs) and the manipulation of genomes. GreenGate and several other modular molecular cloning systems were developed about ten years ago and are widely used in plant research. All these systems define grammars for assembling transcriptional units from building blocks, cloned as Level 0 modules flanked by four-base pair overhangs and recognition sites for a particular Type IIs endonuclease. Modules are efficiently assembled into Level 1 TUs in a hierarchical assembly process, and Level 2 multigene constructs are assembled by stacking Level 1 TUs. GreenGate is highly popular but has three main limitations. First, using ad-hoc overhangs added by PCR and classical restriction/ligation prevents the efficient use of a one-pot, one-step reaction to generate entry clones and domesticate internal sites; second, a Level 1 TU is assembled from a maximum of six modules, which may be limiting for applications such as multiplex genome editing; third, the generation of Level 2 assemblies is sequential and inefficient. GreenGate 2.0 (GG2.0) expands GreenGate features. It introduces additional overhangs, allowing for the combination of up to 12 Level 0 modules in a Level 1 TU. It includes a Universal Entry Generator plasmid (pUEG) to streamline the generation of Level 0 modules. GG2.0 introduces GreenBraid, a convenient method for stacking transcriptional units iteratively for multigene assemblies. Importantly, GG2.0 is backwards compatible with most existing GreenGate modules. Additionally, GG2.0 includes Level 0 modules for multiplex expression of guide RNAs for CRISPR/Cas9 genome editing and pre-assembled Level 1 vectors for dexamethasone-inducible gene expression and ubiquitous expression of plasma membrane and nuclear fluorescent markers. GG2.0 streamlines and increases the versatility of assembling complex transcriptional units and their combination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552369">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552369" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552369">
        <p class="paperTitle">Discovery of an isoflavone oxidative catabolic pathway in legume root microbiota</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552369" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552369" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aoki, N.; Shimasaki, T.; Yazaki, W.; Sato, T.; Nakayasu, M.; Ando, A.; Kishino, S.; Ogawa, J.; Masuda, S.; Shibata, A.; Shirasu, K.; Yazaki, K.; Sugiyama, A.</p>
        <p class="info">Score: 8.6, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552369' target='https://doi.org/10.1101/2023.08.07.552369'> 10.1101/2023.08.07.552369</a></p>
        <p class="abstract">Isoflavones are major specialized metabolites found in legume plants, where they contribute to environmental adaptation. Isoflavones also play a role human health as promising therapeutic agents. This metabolite group is involved in interactions with soil microorganisms as initiation signals in rhizobial symbiosis and as modulators of the legume root microbiota. We previously reported that isoflavones enrich the Comamonadaceae, a predominant bacterial family in soybean roots, and that microorganisms in legume rhizosphere soil degrade isoflavones. However, the isoflavone catabolism pathway that underly the isoflavone-mediated legume-microbiota interactions have not yet been clarified. Here, we isolated Variovorax sp. strain V35, member of the Comamonadaceae that harbors isoflavone-degrading activity, from soybean roots and discovered a gene cluster responsible for isoflavone degradation named ifc. Strain V35 metabolizes isoflavones in a completely distinct oxidative manner from the reductive isoflavone metabolism pathway elucidated in the gut microbiota, in which resulting products enter the tricarboxylic acid cycle. The characterization of ifc mutants and heterologously expressed IFC enzymes revealed that isoflavones are catabolized via A-ring cleaving fission, which starts with hydroxylation at the 8-position of the A-ring. We further demonstrated that ifc genes are frequently found in bacterial strains isolated from legume plants, including mutualistic rhizobia, and contribute to detoxification of the antibacterial activity of isoflavones. Taken together, our findings reveal an oxidative catabolism pathway of isoflavone in the soybean root microbiota, providing molecular insights into isoflavone-mediated legume-microbiota interactions.

SignificanceIsoflavones play pivotal roles in plant-environment interactions and in the maintenance and improvement of human health. Bacterial metabolism is a fundamental component of isoflavone-mediated interkingdom interactions. In the human gut, intestinal bacteria convert isoflavones into equol, a highly bioactive compound. However, the fate of isoflavones in the legume rhizosphere has not been elucidated, despite them being the key signaling molecules for nodule symbiosis and modulation of the legume root microbiota. Here, we discovered a novel isoflavone catabolism pathway in the soybean root microbiota and demonstrated the strong association between bacterial catabolic abilities and their interactions with host plants. Collectively, our findings provide new insights into bacterial isoflavone metabolism and a molecular understanding of legume-microbiota interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552514">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552514" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552514">
        <p class="paperTitle">Algal CO2 capture is powered by alternative electron pathways of photosynthesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552514" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552514" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peltier, G.; Stoffel, C.; Findinier, J.; Madireddi, S. K.; Dao, O.; Epting, V.; Grossman, A.; Li-Beisson, Y.; Burlacot, A.</p>
        <p class="info">Score: 8.3, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552514' target='https://doi.org/10.1101/2023.08.08.552514'> 10.1101/2023.08.08.552514</a></p>
        <p class="abstract">On Earth, microalgae contribute to about half of global net photosynthesis. During photosynthesis, sunlight is converted into chemical energy (ATP and NADPH) used by metabolism to convert CO2 into biomass. Alternative electron pathways of photosynthesis have been proposed to generate additional ATP that is required for sustaining CO2 fixation, but the relative importance of each pathway remains elusive. Here, we dissect and quantify the contribution of cyclic, pseudo-cyclic and chloroplast to mitochondria electron flows for their ability to sustain net photosynthesis in the microalga Chlamydomonas reinhardtii. We show that each pathway has the potential to energize substantial CO2 fixation, can compensate each other, and that the additional energy requirement to fix CO2 is more than 3 times higher than previous estimations. We further show that all pathways have very different efficiencies at energizing CO2 fixation, with the chloroplast-mitochondria interaction being the most efficient, thus laying bioenergetic foundations for biotechnological improvement of CO2 capture.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552212">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552212" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552212">
        <p class="paperTitle">Insights into stereoselective ring formation in canonical strigolactone: Discovery of a dirigent domain-containing enzyme catalyzing orobanchol synthesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552212" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552212" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Homma, M.; Wakabayashi, T.; Moriwaki, Y.; Shiotani, N.; Shigeta, T.; Isobe, K.; Okazawa, A.; Ohta, D.; Terada, T.; Shimizu, K.; Mizutani, M.; Takikawa, H.; Sugimoto, Y.</p>
        <p class="info">Score: 7.4, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552212' target='https://doi.org/10.1101/2023.08.07.552212'> 10.1101/2023.08.07.552212</a></p>
        <p class="abstract">Strigolactones (SLs) are plant apocarotenoids with diverse functions and structures. The widespread canonical SLs, with distinctive structural variations in their tricyclic lactone known as the ABC-ring, are classified into two types based on the C-ring configurations. The steric C-ring configuration arises during the BC-ring closure downstream of carlactonoic acid (CLA), a biosynthetic intermediate. Most plants stereoselectively produce either type of canonical SLs, e.g., tomato (Solanum lycopersicum) produces orobanchol with -oriented C-ring. The mechanisms governing SL structural diversification are partly understood, with limited insight into the functional implications. Moreover, the precise molecular mechanism for the stereoselective BC-ring closure reaction remains unknown. Herein, we identified an enzyme called the stereoselective BC-ring-forming factor (SRF) from the dirigent protein (DIR) family, especially the DIR-f subfamily, whose biochemical function was previously unidentified, making it a pivotal enzyme in stereoselective canonical SL biosynthesis with the -oriented C-ring. We begin by confirming the exact catalytic function of the tomato cytochrome P450 SlCYP722C, which we previously demonstrated to be involved in the orobanchol biosynthesis [Wakabayashi et al., Sci. Adv. 5, eaax9067 (2019)], to convert CLA to 18-oxocarlactonoic acid. Subsequently, we demonstrate that SRF catalyzes the stereoselective BC-ring closure reaction of 18-oxocarlactonoic acid to form orobanchol. Our approach integrates experimental and computational methods, including SRF structure prediction and molecular dynamics simulations, to propose a catalytic mechanism based on the conrotatory 4{pi}-electrocyclic reaction for stereoselective BC-ring formation in orobanchol. The present study provides insight into the molecular basis of how plants produce SLs with specific stereochemistry in a controlled manner.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551207">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551207" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551207">
        <p class="paperTitle">Oryza genera-specific novel Histone H4 variant predisposes H4 Lysine5 Acetylation marks to modulate salt stress responses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551207" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551207" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: G, V. H. S.; Sotelo-Parrilla, P.; Raju, S.; Jha, S.; Gireesh, A.; Gut, F.; Kutti, V. R.; Berger, F.; Jeyaprakash, A. A.; Shivaprasad, P. V.</p>
        <p class="info">Score: 24.0, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551207' target='https://doi.org/10.1101/2023.07.31.551207'> 10.1101/2023.07.31.551207</a></p>
        <p class="abstract">Paralogous variants of canonical histones guide accessibility to DNA and function as additional layers of genome regulation. Across eukaryotes, the occurrence, mechanism of action and functional significance of several variants of core histones are well known except that of histone H4. Here we show that a novel variant of H4 (H4.V), expressing tissue-specifically among members of Oryza genera, mediates specific epigenetic changes contributing majorly to salt tolerance. H4.V was incorporated to specific chromosomal locations where it blocked deposition of active histone marks. Under salt stress, large scale re-distribution of H4.V enabled incorporation of stress dependent histone H4 Lysine5 Acetylation (H4K5Ac) marks. Mis-expression of H4.V led to defects at morphological level especially in reproductive tissues, and in mounting stress responses. H4.V mediated these alterations by condensing chromatin at specific genomic regions as seen with cryo-EM structure of reconstituted H4.V containing nucleosomes. These results not only uncovered the presence of a H4 variant in plants, but also a novel chromatin regulation of stress responses that might have contributed to success of semi-aquatic Oryza members under variable water-limiting conditions.

One-line summaryHistone H4 variant predisposes chromatin for stress responses



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=199 SRC=&#34;FIGDIR/small/551207v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (53K):
org.highwire.dtl.DTLVardef@101fa0forg.highwire.dtl.DTLVardef@c7fda9org.highwire.dtl.DTLVardef@1de9a0aorg.highwire.dtl.DTLVardef@1532641_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551673">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551673" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551673">
        <p class="paperTitle">The genomes and epigenomes of aquatic plants (Lemnaceae) promote triploid hybridization and clonal reproduction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551673" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551673" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ernst, E.; Abramson, B.; Acosta, K.; Hoang, P. T.; Mateo-Elizalde, C.; Schubert, V.; Pasaribu, B.; Hartwick, N.; Colt, K.; Aylward, A. J.; Ramu, U.; Birchler, J. A.; Schubert, I.; Lam, E.; Michael, T. P.; Martienssen, R. A.</p>
        <p class="info">Score: 8.2, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551673' target='https://doi.org/10.1101/2023.08.02.551673'> 10.1101/2023.08.02.551673</a></p>
        <p class="abstract">The Lemnaceae (duckweeds) are the worlds smallest but fastest growing flowering plants, with a drastically reduced morphology and predominant clonal reproductive habit capable of continuous exponential growth. Here, we present assemblies of 10 Lemna chromosome sets by single molecule nanopore sequencing and chromosome conformation capture. Dynamics of genome evolution in the family are revealed by syntenic comparisons with Wolffia and Spirodela, and diversification of these genera was found to coincide with the &#34;Azolla event&#34;, in which blooms of aquatic macrophytes reduced atmospheric CO2 from greenhouse levels found in the Eocene to those of the current ice age. Orthologous gene comparisons with other aquatic and terrestrial plants uncovered candidate genes for the unique metabolic and developmental features of the family, such as frequent hybrid polyploidy, lack of stomatal closure in high CO2, and accumulation of calcium oxalate, a promising candidate for carbon sequestration. Loss of a spermine-triggered gene network may account for drastic reduction in stature and preferentially adaxial stomata, a feature of floating aquatic plants. Strikingly, Lemnaceae genomes have selectively lost some of the genes required for RNA interference, including Argonaute genes required for post-zygotic reproductive isolation (the triploid block) and reduced gamete formation. Triploid hybrids arise commonly among Lemna, presumably by hybridization with unreduced gametes, and we have found mutations in highly-conserved ZMM crossover pathway genes that could support polyploid meiosis. Rapid but stable clonal propagation makes Lemna an ideal platform for continuous protein and starch micro-cropping, and for efficient sequestration of dissolved nutrients and atmospheric CO2. Facile regeneration of transgenic fronds from tissue culture, aided by reduced epigenetic silencing, makes Lemna a powerful biotechnological platform, as exemplified by recent engineering of high-oil Lemna that out-perform oil seed crops.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551648">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551648" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551648">
        <p class="paperTitle">Spatial Co-transcriptomics Reveals Discrete Stages of the Arbuscular Mycorrhizal Symbiosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551648" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551648" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Serrano, K.; Bezrutczyk, M.; Goudeau, D.; Dao, T.; O&#39;Malley, R. C.; Malmstrom, R. R.; Visel, A.; Scheller, H. V.; Cole, B.</p>
        <p class="info">Score: 20.4, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551648' target='https://doi.org/10.1101/2023.08.02.551648'> 10.1101/2023.08.02.551648</a></p>
        <p class="abstract">The symbiotic interaction of plants with arbuscular mycorrhizal fungi (AM fungi) is ancient and widespread. Plants provide AM fungi with carbon in exchange for nutrients and water, making this interaction a prime target for crop improvement. However, plant-fungal interactions are restricted to a small subset of root cells, precluding the application of most conventional functional genomic techniques to study the molecular bases of these interactions. Here we used single-nucleus and spatial RNA sequencing to explore both M. truncatula and R. irregularis transcriptomes in AM symbiosis at cellular and spatial resolution. Integrated spatially-registered single-cell maps of interacting cells revealed major infected and uninfected plant root cell types. We observed that cortex cells exhibit distinct transcriptome profiles during different stages of colonization by AM fungi, indicating dynamic interplay between both organisms during establishment of the cellular interface enabling successful symbiosis. Our study provides insight into a symbiotic relationship of major agricultural and environmental importance and demonstrates a paradigm combining single-cell and spatial transcriptomics for the analysis of complex organismal interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552572">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552572" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552572">
        <p class="paperTitle">Comparative Analysis, Diversification and Functional Validation of Plant Nucleotide-Binding Site Domain Genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552572" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552572" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hussain, A.; Khan, A. A.; Aslam, M. Q.; Nazar, A.; Zaman, N.; Amin, A.; Mahmood, M. A.; Mukhtar, M. S.; Rahman, H. U. U.; Farooq, M.; Saeed, M.; Amin, I.; Mansoor, S.</p>
        <p class="info">Score: 4.3, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552572' target='https://doi.org/10.1101/2023.08.09.552572'> 10.1101/2023.08.09.552572</a></p>
        <p class="abstract">Nucleotide-binding site (NBS) domain genes are one of a superfamily of resistance genes involved in plant responses to pathogens. The current study identified presumably identified 12,820 NBS-containing genes across 34 species covering from mosses to monocots and dicots. These identified genes classified into 168 classes with several novel domain architectures patterns encompassing significant diversity among plant species. Several classical (NBS, NBS-LRR, TIR-NBS, TIR-NBS-LRR etc.) and species-specific structural patterns (TIR-NBS-TIR-Cupin_1-Cupin_1, TIR-NBS-Prenyltransf, Sugar_tr-NBS etc.) were discovered. We observed 603 orthogroups (OGs) with some core (most common orthogroups; OG0, OG1, OG2 etc.) and unique (highly specific to species; OG80, OG82 etc) OGs with tandem duplications. The expression profiling presented the putative upregulation of OG2, OG6, and OG15 in different tissues under various biotic and abiotic stresses in susceptible and tolerant plants to CLCuD. The genetic variation between susceptible (Coker 312) and tolerant (Mac7) G. hirsutum accessions identified several unique variants in NBS genes of Mac7 (6,583 varaints) and Coker312 (5,173 variants). The protein-ligand and proteins-protein interaction showed a strong interaction of some putative NBS proteins with ADP/ATP and different core proteins of cotton leaf curl disease virus. The silencing of GaNBS (OG2) in resistant cotton through virus-induced gene silencing (VIGS) demonstrated its putative role in virus tittering. The presented study will be further helpful to understand the plant adaptation mechanism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.551077">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.551077" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.551077">
        <p class="paperTitle">Cell wall integrity modulates a PHYTOCHROME-INTERACTING FACTOR (PIF) - HOOKLESS1 (HLS1) signalling module controlling apical hook formation in Arabidopsis.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.551077" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.551077" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lorrai, R.; Erguvan, O.; Raggi, S.; Jonsson, K.; Siroka, J.; Tarkowska, D.; Novak, O.; Verger, S.; Robert, S.; Ferrari, S.</p>
        <p class="info">Score: 4.2, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.551077' target='https://doi.org/10.1101/2023.08.05.551077'> 10.1101/2023.08.05.551077</a></p>
        <p class="abstract">Etiolated seedlings of dicots form an apical hook to protect the meristems during soil emergence. Hook formation is the result of differential growth on both sides of the hypocotyl apex and is tightly controlled by environmental cues and hormones, among which auxin and gibberellins (GAs) are the main contributors. Cell expansion is tightly regulated by the cell wall, but whether and how feedback from this structure contributes to hook development is still unclear. Here we show that etiolated seedlings of the Arabidopsis thaliana quasimodo2-1 (qua2) mutant, defective in pectin biosynthesis, display severe defects in apical hook formation and maintenance, accompanied by loss of asymmetric auxin maxima and differential cell expansion. Moreover, qua2 seedlings show reduced expression of HOOKLESS1 (HLS1) and PHYTOCHROME-INTERACTING FACTOR 4 and 5 (PIF4/5), positive regulators of hook formation, and accumulate reduced levels of the active gibberellin GA4. Treatment of wild-type seedlings with the cellulose inhibitor isoxaben (isx) also prevents hook development and represses HLS1 expression and PIF4 accumulation. Moreover, isx stabilizes the DELLA protein REPRESSOR OF ga1-3 (RGA), which inhibits HLS1 expression and hook formation. Exogenous GAs or HLS1 overexpression partially restore hook development in isx-treated seedlings. Notably, agar concentration in the medium restores, both in qua2 and isx-treated seedlings, hook development and WT-like levels of PIFs and HLS1. We propose that turgor-dependent signals link changes in cell wall integrity to the PIF4/5-HLS1 signalling module to repress differential cell elongation during hook formation.

Significance statementCell wall integrity modulates apical hook development through poorly understood mechanisms. We show here that, in Arabidopsis, repression of hook formation by either mutations in pectin biosynthesis or by isoxaben treatment is at least partially mediated by the downregulation of a gibberellin-controlled signalling module that comprises PIF4/5 and HLS1. Our results indicate that the signals derived from changes in the cell wall can modulate hormone-mediated pathways to control asymmetric growth during plant development.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
