<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Plant-based production of diverse human milk oligosaccharides
Authors: Barnum, C. R.; Paviani, B.; Couture, G.; Masarweh, C.; Chen, Y.; Huang, Y.-P.; Mills, D. A.; Lebrilla, C. B.; Barile, D.; Yang, M.; Shih, P.
Score: 13.9, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558286
Human milk oligosaccharides (HMOs) are a diverse class of carbohydrates that aid in the health and development of infants. The vast health benefits of HMOs have made them a commercial target for microbial production; however, producing the ~130 structurally diverse HMOs at scale has proven difficult.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="Plant-based production of diverse human milk oligosaccharides
Authors: Barnum, C. R.; Paviani, B.; Couture, G.; Masarweh, C.; Chen, Y.; Huang, Y.-P.; Mills, D. A.; Lebrilla, C. B.; Barile, D.; Yang, M.; Shih, P.
Score: 13.9, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558286
Human milk oligosaccharides (HMOs) are a diverse class of carbohydrates that aid in the health and development of infants. The vast health benefits of HMOs have made them a commercial target for microbial production; however, producing the ~130 structurally diverse HMOs at scale has proven difficult." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-24T10:37:06+00:00" />
<meta property="article:modified_time" content="2023-09-24T10:37:06+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="Plant-based production of diverse human milk oligosaccharides
Authors: Barnum, C. R.; Paviani, B.; Couture, G.; Masarweh, C.; Chen, Y.; Huang, Y.-P.; Mills, D. A.; Lebrilla, C. B.; Barile, D.; Yang, M.; Shih, P.
Score: 13.9, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558286
Human milk oligosaccharides (HMOs) are a diverse class of carbohydrates that aid in the health and development of infants. The vast health benefits of HMOs have made them a commercial target for microbial production; however, producing the ~130 structurally diverse HMOs at scale has proven difficult."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "Plant-based production of diverse human milk oligosaccharides\nAuthors: Barnum, C. R.; Paviani, B.; Couture, G.; Masarweh, C.; Chen, Y.; Huang, Y.-P.; Mills, D. A.; Lebrilla, C. B.; Barile, D.; Yang, M.; Shih, P.\nScore: 13.9, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558286\nHuman milk oligosaccharides (HMOs) are a diverse class of carbohydrates that aid in the health and development of infants. The vast health benefits of HMOs have made them a commercial target for microbial production; however, producing the ~130 structurally diverse HMOs at scale has proven difficult.",
  "keywords": [
    
  ],
  "articleBody": " Plant-based production of diverse human milk oligosaccharides\nAuthors: Barnum, C. R.; Paviani, B.; Couture, G.; Masarweh, C.; Chen, Y.; Huang, Y.-P.; Mills, D. A.; Lebrilla, C. B.; Barile, D.; Yang, M.; Shih, P.\nScore: 13.9, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558286\nHuman milk oligosaccharides (HMOs) are a diverse class of carbohydrates that aid in the health and development of infants. The vast health benefits of HMOs have made them a commercial target for microbial production; however, producing the ~130 structurally diverse HMOs at scale has proven difficult. Here, we produce a vast diversity of HMOs by leveraging the robust carbohydrate anabolism of plants. This diversity includes high value HMOs, such as lacto-N-fucopentaose I, that have not yet been commercially produced using state-of-the-art microbial fermentative processes. HMOs produced in transgenic plants provided strong bifidogenic properties, indicating their ability to serve as a prebiotic supplement. Technoeconomic analyses demonstrate that producing HMOs in plants provides a path to the large-scale production of specific HMOs at lower prices than microbial production platforms. Our work demonstrates the promise in leveraging plants for the cheap and sustainable production of HMOs.\nEctopic assembly of an auxin efflux control machinery shifts developmental trajectories\nAuthors: Aliaga Fandino, A. C.; Jelinkova, A.; Marhava, P.; Petrasek, J.; Hardtke, C. S.\nScore: 13.2, Published: 2023-09-17 DOI: 10.1101/2023.09.16.558043\nPolar auxin transport in the Arabidopsis root tip maintains high auxin levels around the stem cell niche that gradually decrease in dividing cells but increase again once they transition towards differentiation. Protophloem differentiates earlier than other proximal tissues and employs a unique auxin canalization machinery that is thought to balance auxin efflux with retention. It consists of a proposed activator of PIN auxin efflux carriers, the AGC kinase PAX; its inhibitor, BRX; and PIP5K enzymes, which promote polar PAX and BRX localization. Because of dynamic PAX-BRX-PIP5K interplay, the net cellular output of this machinery remains unclear. Here we deciphered the dosage-sensitive regulatory interactions between PAX, BRX and PIP5K by their ectopic expression in developing xylem vessels. The data suggest that the dominant collective output of the PAX-BRX-PIP5K module is a localized reduction in PIN abundance. This requires PAX-stimulated clathrin-mediated PIN endocytosis by site-specific phosphorylation, which distinguishes PAX from other AGC kinases. Importantly, ectopic assembly of the PAX-BRX-PIP5K module is sufficient to cause cellular auxin retention and affects root growth vigor by accelerating the trajectory of xylem vessel development. Our data thus provide direct evidence that local manipulation of auxin efflux alters the timing of cellular differentiation in the root.\nMechanotransduction in Lateral Root Initiation: A Model Integrating Growth Mechanics and Auxin Signaling\nAuthors: Ramos, J. R. D.; Reyes-Hernandez, B. J.; Alim, K.; Maizel, A.\nScore: 12.6, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558377\nPlant development relies on the precise coordination of cell growth, which is influenced by the mechanical constraints imposed by rigid cell walls. The hormone auxin plays a crucial role in regulating this growth by altering the mechanical properties of cell walls. During the post-embryonic formation of lateral roots, pericycle cells deep within the main root are triggered by auxin to resume growth and divide to form a new root. This growth involves a complex interplay between auxin, growth, and the resolution of mechanical conflicts. However, the exact mechanisms by which this coordination is achieved are still unknown. Here, we propose a model that integrates tissue mechanics and auxin transport, revealing a connection between the auxin-induced relaxation of mechanical stress in the pericycle and auxin signalling in the endodermis. We show that the growth of pericycle cells is initially limited by the endodermis, resulting in a modest initial growth. However, this modest growth is sufficient to redirect auxin to the overlying endodermis, which then actively accommodates the growth, allowing for the subsequent development of the lateral root. Our model uncovers the mechanical parameters that underlie endodermal accommodation and how the structure and shape of the endodermis influence the formation of the new root. These findings highlight the interconnected relationship between mechanics and auxin flow during lateral root initiation, emphasizing the vital role of the endodermis in shaping root development through mechanotransduction and auxin signalling.\nThe Arabidopsis U1 snRNP regulates mRNA 3'-end processing\nAuthors: Mangilet, A. F.; Weber, J.; Schueler, S.; Droste-Borel, I.; Streicher, S.; Schmutzer, T.; Rot, G.; Macek, B.; Laubinger, S.\nScore: 10.6, Published: 2023-09-20 DOI: 10.1101/2023.09.19.558503\nThe removal of introns by the spliceosome is a key gene regulatory mechanism in eukaryotes, with the U1 snRNP subunit of the spliceosome playing a crucial role in the early stages of splicing. Studies in metazoans show that the U1 snRNP also conducts splicing-independent functions, but the lack of genetic tools and knowledge about U1 snRNP-associated proteins have limited the study of such splicing-independent functions in plants. Here, we describe an RNA-centric approach that identified more than 200 proteins associated with the Arabidopsis U1 snRNP, among them mRNA cleavage and polyadenylation factors. The loss of U1 core components is linked to premature cleavage and polyadenylation within gene bodies and alternative polyadenylation site selection in 3-UTRs. Overall, our work provides a comprehensive view of U1 snRNP interactors and reveals novel functions in regulating mRNA 3-end processing in Arabidopsis, thus establishing the groundwork for a better understanding of non-canonical functions of plant U1 snRNPs.\nPGR5 is required to avoid photosynthetic oscillations during light transitions\nAuthors: Johnson, M. P.; Degen, G. E.; Pastorelli, F.\nScore: 7.7, Published: 2023-09-21 DOI: 10.1101/2023.09.21.558795\nThe production of ATP and NADPH by the light reactions of photosynthesis and their consumption by the Calvin-Benson-Bassham (CBB) cycle and other downstream metabolic reactions requires careful regulation. Environmental shifts perturb this careful balance, leading to photo-oxidative stress and losses in CO2 assimilation. Imbalances in the production and consumption of ATP and NADPH manifest themselves as transient instability in the chlorophyll fluorescence, P700, electrochromic shift and CO2 uptake signals recorded on leaves. These oscillations can be induced in wild-type plants by sudden shifts in CO2 concentration or light intensity, however mutants exhibiting increased oscillatory behaviour have yet to be reported. This has precluded an understanding of the regulatory mechanisms employed by plants to suppress oscillations. Here we show that the Arabidopsis pgr5 mutant, which is deficient in PGR5-dependent cyclic electron transfer (CET), exhibits increased oscillatory behaviour. In contrast, mutants lacking the NDH-dependent CET are largely unaffected. The absence of oscillations in the hope2 mutant, which like pgr5, lacks photosynthetic control and exhibits high ATP synthase conductivity, ruled out loss of these photoprotective mechanisms as causes. Instead, we observed slower formation of proton motive force and by inference ATP synthesis in pgr5 following environmental perturbation, leading to the transient reduction of the electron transfer chain and photosynthetic oscillations. PGR5-dependent CET therefore plays a major role in damping the effect of environmental perturbations on photosynthesis to avoid losses in CO2 fixation.\nAlternative splicing regulation in plants by effectors of symbiotic arbuscular mycorrhizal fungi\nAuthors: Requena, N.; Betz, R.; Heidt, S.; Figueira-Galan, D.; Langner, T.\nScore: 7.3, Published: 2023-09-20 DOI: 10.1101/2023.09.20.558436\nMost plants in natural ecosystems live in association with AM (arbuscular mycorrhizal) fungi to survive under poor nutrient conditions and to cope with other abiotic and biotic stresses. To engage in symbiosis, AM fungi secrete effector molecules that, similar to pathogenic effectors, reprogram plant cells. Despite numerous effectors being predicted in the genome of AM fungi, only a few have been functionally characterized. Here we show that the SP7-like family, a Glomeromycotina-specific effector family, impacts on the alternative splicing program of their hosts. SP7-like effectors localize at nuclear condensates and interact with the plant mRNA processing machinery, most prominently with the splicing factor SR45 and the core splicing proteins U170K and U2AF35. Ectopic expression of two of these effectors in the crop plant potato changed the alternative splicing pattern of a specific subset of genes. Unravelling the communication mechanisms between symbiotic fungi and their host plants will help to identify targets to improve plant nutrition.\nGENOMES UNCOUPLED1: an RNA-binding protein required for early PSII biogenesis\nAuthors: Cui, C.; Sun, S.; Zhang, S.; Zhang, Y.; Kim, C.\nScore: 6.1, Published: 2023-09-22 DOI: 10.1101/2023.09.21.558905\nGenomes Uncoupled1 (GUN1), a nuclear-encoded chloroplast pentatricopeptide repeat (PPR) protein, serves as a master integrator of diverse retrograde signals, mediating chloroplast-to-nucleus retrograde signaling. Although PPR proteins primarily function in organelle RNA metabolism, a target RNA of GUN1 remained unknown. This study reveals that GUN1 recognizes psbD transcripts derived from a blue light-responsive promoter (BLRP), transcripts referred to as psbD BLRP. Overexpression of GUN1 significantly reduces the levels of psbD BLRP, whereas the loss of GUN1 leads to the accumulation of the putative target RNA. The in vitro RNA and in vivo genetic studies further demonstrate the critical role of the C-terminal small MutS-related (SMR) domain in stimulating psbD BLRP processing and PsbD (D2, a PSII core protein) synthesis. The GUN1-dependent psbD BLRP processing promotes PSII biogenesis during the early seedling development and de-etiolation phase. This finding underscores the role of GUN1 as an RNA-binding protein, highlighting the essential function of the SMR domain in processing the target RNA.\nExpression of cell-wall related genes is highly variable and correlates with sepal morphology\nAuthors: Hartasanchez, D. A.; Kiss, A. K.; Battu, V.; Soraru, C.; Delgado Vaquera, A.; Massinon, F.; Braso-Vives, M.; Mollier, C.; Martin-Magnette, M.-L.; Boudaoud, A.; Moneger, F.\nScore: 14.1, Published: 2023-09-15 DOI: 10.1101/2022.04.26.489498\nControl of organ morphology is a fundamental feature of living organisms. There is, however, observable variation in organ size and shape within a given genotype. Taking the sepal of Arabidopsis as a model, we investigated whether we can use variability of gene expression alongside variability of organ morphology to identify gene regulatory networks potentially involved in organ size and shape determination. We produced a dataset composed of morphological parameters and genome-wide transcriptome obtained from 27 individual sepals from wild-type plants with nearly identical genetic backgrounds, environment, and developmental stage. Sepals exhibited appreciable variability in both morphology and transcriptome, with response to stimulus genes and cell-wall related genes displaying high variability in expression. We additionally identified five modules of co-expressed genes which correlated significantly with morphology, revealing biologically relevant gene regulatory networks. Interestingly, cell-wall related genes were overrepresented in two of the top three modules. Overall, our work highlights the benefit of using coupled variation in gene expression and phenotype in wild-type plants to shed light on the mechanisms underlying organ size and shape determination. Although causality between gene expression and sepal morphology has not been established, our approach opens the way to informed analysis for mutant characterization and functional studies.\nViable mutants of essential genes in Physcomitrium patens as tools for studying primary metabolic processes\nAuthors: Haslam, T. M.; Herrfurth, C.; Feussner, I.\nScore: 4.8, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558677\nSphingolipids are essential components of plant cells, which have been notoriously difficult to study in part due to pleiotropic or lethal knock-out mutant phenotypes. By relying on alternative end-joining of double stranded breaks, we successfully used CRISPR/Cas9 mutagenesis to generate a population of diverse, viable mutant alleles of genes required for sphingolipid assembly in totipotent protoplasts of the moss Physcomitrium patens. We targeted the INOSITOL PHOSPHORYLCERAMIDE SYNTHASE (IPCS) gene family, which catalyzes the committed step in the synthesis of glycosyl inositol phosphorylceramides (GIPCs), the most abundant class of sphingolipids found in plants. We isolated knock-out single mutants and knock-down higher-order mutants showing a spectrum of deficiencies in GIPC content. Remarkably, we also identified two mutant alleles accumulating inositol phosphorylceramides, the direct products of IPCS activity, and provide our best explanation for this unexpected phenotype. Our approach is broadly applicable for studying essential genes and gene families, and for obtaining unusual lesions within a gene of interest.\nHigh-resolution mapping of Ryd4Hb, a major resistance gene to Barley yellow dwarf virus from Hordeum bulbosum\nAuthors: Pidon, H.; Ruge-Wehling, B.; Will, T.; Habekuss, A.; Wendler, N.; Oldach, K.; Maasberg-Prelle, A.; Korzun, V.; Stein, N.\nScore: 4.6, Published: 2023-09-20 DOI: 10.1101/2023.09.19.558385\nVirus diseases are causing high yield losses in crops worldwide. The Barley yellow dwarf virus (BYDV) complex is responsible for one of the most widespread and economically important viral diseases of cereals. While no complete resistance gene has been uncovered in the primary genepool of barley, sources of resistance were identified in the wild relative Hordeum bulbosum, representing the secondary genepool of barley. One such locus, Ryd4Hb, has been previously introgressed into barley, and was allocated to chromosome 3H, but is tightly linked to a sublethality factor that prevents the incorporation and utilization of Ryd4Hb in barley varieties. To solve this problem, we fine-mapped Ryd4Hb and separated it from this negative factor. We narrowed the Ryd4Hb locus to a 66.5 kbp physical interval in the barley Morex reference genome. The region comprises one complete and one partial gene from the nucleotide-binding and leucine-rich repeat immune receptor family, typical of dominant virus resistance genes. The closest homolog to these two Ryd4Hb candidate genes is the wheat Sr35 stem rust resistance gene. In addition to the fine mapping, we reduced the sublethality factor interval to 600 kbp in barley. Aphid feeding experiments demonstrated that Ryd4Hb provides a direct resistance to BYDV rather than a resistance to its vector. The presented results, including the high-throughput molecular markers, will permit a more targeted selection of the resistance in breeding, enabling the use of Ryd4Hb in barley varieties. Key messageWe mapped Ryd4Hb in a 66.5 kpb interval in barley and dissociated it from a sublethality factor. These results will enable a targeted selection of the resistance in barley breeding.\n",
  "wordCount" : "2220",
  "inLanguage": "en",
  "datePublished": "2023-09-24T10:37:06Z",
  "dateModified": "2023-09-24T10:37:06Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta"><span>updated on September 24, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558286">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558286" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558286">
        <p class="paperTitle">Plant-based production of diverse human milk oligosaccharides</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558286" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558286" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Barnum, C. R.; Paviani, B.; Couture, G.; Masarweh, C.; Chen, Y.; Huang, Y.-P.; Mills, D. A.; Lebrilla, C. B.; Barile, D.; Yang, M.; Shih, P.</p>
        <p class="info">Score: 13.9, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558286' target='https://doi.org/10.1101/2023.09.18.558286'> 10.1101/2023.09.18.558286</a></p>
        <p class="abstract">Human milk oligosaccharides (HMOs) are a diverse class of carbohydrates that aid in the health and development of infants. The vast health benefits of HMOs have made them a commercial target for microbial production; however, producing the ~130 structurally diverse HMOs at scale has proven difficult. Here, we produce a vast diversity of HMOs by leveraging the robust carbohydrate anabolism of plants. This diversity includes high value HMOs, such as lacto-N-fucopentaose I, that have not yet been commercially produced using state-of-the-art microbial fermentative processes. HMOs produced in transgenic plants provided strong bifidogenic properties, indicating their ability to serve as a prebiotic supplement. Technoeconomic analyses demonstrate that producing HMOs in plants provides a path to the large-scale production of specific HMOs at lower prices than microbial production platforms. Our work demonstrates the promise in leveraging plants for the cheap and sustainable production of HMOs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.16.558043">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.16.558043" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.16.558043">
        <p class="paperTitle">Ectopic assembly of an auxin efflux control machinery shifts developmental trajectories</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.16.558043" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.16.558043" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aliaga Fandino, A. C.; Jelinkova, A.; Marhava, P.; Petrasek, J.; Hardtke, C. S.</p>
        <p class="info">Score: 13.2, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.16.558043' target='https://doi.org/10.1101/2023.09.16.558043'> 10.1101/2023.09.16.558043</a></p>
        <p class="abstract">Polar auxin transport in the Arabidopsis root tip maintains high auxin levels around the stem cell niche that gradually decrease in dividing cells but increase again once they transition towards differentiation. Protophloem differentiates earlier than other proximal tissues and employs a unique auxin  canalization machinery that is thought to balance auxin efflux with retention. It consists of a proposed activator of PIN auxin efflux carriers, the AGC kinase PAX; its inhibitor, BRX; and PIP5K enzymes, which promote polar PAX and BRX localization. Because of dynamic PAX-BRX-PIP5K interplay, the net cellular output of this machinery remains unclear. Here we deciphered the dosage-sensitive regulatory interactions between PAX, BRX and PIP5K by their ectopic expression in developing xylem vessels. The data suggest that the dominant collective output of the PAX-BRX-PIP5K module is a localized reduction in PIN abundance. This requires PAX-stimulated clathrin-mediated PIN endocytosis by site-specific phosphorylation, which distinguishes PAX from other AGC kinases. Importantly, ectopic assembly of the PAX-BRX-PIP5K module is sufficient to cause cellular auxin retention and affects root growth vigor by accelerating the trajectory of xylem vessel development. Our data thus provide direct evidence that local manipulation of auxin efflux alters the timing of cellular differentiation in the root.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558377">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558377" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558377">
        <p class="paperTitle">Mechanotransduction in Lateral Root Initiation: A Model Integrating Growth Mechanics and Auxin Signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558377" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558377" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ramos, J. R. D.; Reyes-Hernandez, B. J.; Alim, K.; Maizel, A.</p>
        <p class="info">Score: 12.6, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558377' target='https://doi.org/10.1101/2023.09.18.558377'> 10.1101/2023.09.18.558377</a></p>
        <p class="abstract">Plant development relies on the precise coordination of cell growth, which is influenced by the mechanical constraints imposed by rigid cell walls. The hormone auxin plays a crucial role in regulating this growth by altering the mechanical properties of cell walls. During the post-embryonic formation of lateral roots, pericycle cells deep within the main root are triggered by auxin to resume growth and divide to form a new root. This growth involves a complex interplay between auxin, growth, and the resolution of mechanical conflicts. However, the exact mechanisms by which this coordination is achieved are still unknown. Here, we propose a model that integrates tissue mechanics and auxin transport, revealing a connection between the auxin-induced relaxation of mechanical stress in the pericycle and auxin signalling in the endodermis. We show that the growth of pericycle cells is initially limited by the endodermis, resulting in a modest initial growth. However, this modest growth is sufficient to redirect auxin to the overlying endodermis, which then actively accommodates the growth, allowing for the subsequent development of the lateral root. Our model uncovers the mechanical parameters that underlie endodermal accommodation and how the structure and shape of the endodermis influence the formation of the new root. These findings highlight the interconnected relationship between mechanics and auxin flow during lateral root initiation, emphasizing the vital role of the endodermis in shaping root development through mechanotransduction and auxin signalling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558503">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558503" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558503">
        <p class="paperTitle">The Arabidopsis U1 snRNP regulates mRNA 3&#39;-end processing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558503" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558503" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mangilet, A. F.; Weber, J.; Schueler, S.; Droste-Borel, I.; Streicher, S.; Schmutzer, T.; Rot, G.; Macek, B.; Laubinger, S.</p>
        <p class="info">Score: 10.6, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558503' target='https://doi.org/10.1101/2023.09.19.558503'> 10.1101/2023.09.19.558503</a></p>
        <p class="abstract">The removal of introns by the spliceosome is a key gene regulatory mechanism in eukaryotes, with the U1 snRNP subunit of the spliceosome playing a crucial role in the early stages of splicing. Studies in metazoans show that the U1 snRNP also conducts splicing-independent functions, but the lack of genetic tools and knowledge about U1 snRNP-associated proteins have limited the study of such splicing-independent functions in plants. Here, we describe an RNA-centric approach that identified more than 200 proteins associated with the Arabidopsis U1 snRNP, among them mRNA cleavage and polyadenylation factors. The loss of U1 core components is linked to premature cleavage and polyadenylation within gene bodies and alternative polyadenylation site selection in 3-UTRs. Overall, our work provides a comprehensive view of U1 snRNP interactors and reveals novel functions in regulating mRNA 3-end processing in Arabidopsis, thus establishing the groundwork for a better understanding of non-canonical functions of plant U1 snRNPs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558795">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558795" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558795">
        <p class="paperTitle">PGR5 is required to avoid photosynthetic oscillations during light transitions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558795" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558795" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnson, M. P.; Degen, G. E.; Pastorelli, F.</p>
        <p class="info">Score: 7.7, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558795' target='https://doi.org/10.1101/2023.09.21.558795'> 10.1101/2023.09.21.558795</a></p>
        <p class="abstract">The production of ATP and NADPH by the light reactions of photosynthesis and their consumption by the Calvin-Benson-Bassham (CBB) cycle and other downstream metabolic reactions requires careful regulation. Environmental shifts perturb this careful balance, leading to photo-oxidative stress and losses in CO2 assimilation. Imbalances in the production and consumption of ATP and NADPH manifest themselves as transient instability in the chlorophyll fluorescence, P700, electrochromic shift and CO2 uptake signals recorded on leaves. These oscillations can be induced in wild-type plants by sudden shifts in CO2 concentration or light intensity, however mutants exhibiting increased oscillatory behaviour have yet to be reported. This has precluded an understanding of the regulatory mechanisms employed by plants to suppress oscillations. Here we show that the Arabidopsis pgr5 mutant, which is deficient in PGR5-dependent cyclic electron transfer (CET), exhibits increased oscillatory behaviour. In contrast, mutants lacking the NDH-dependent CET are largely unaffected. The absence of oscillations in the hope2 mutant, which like pgr5, lacks photosynthetic control and exhibits high ATP synthase conductivity, ruled out loss of these photoprotective mechanisms as causes. Instead, we observed slower formation of proton motive force and by inference ATP synthesis in pgr5 following environmental perturbation, leading to the transient reduction of the electron transfer chain and photosynthetic oscillations. PGR5-dependent CET therefore plays a major role in damping the effect of environmental perturbations on photosynthesis to avoid losses in CO2 fixation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558436">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558436" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558436">
        <p class="paperTitle">Alternative splicing regulation in plants by effectors of symbiotic arbuscular mycorrhizal fungi</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558436" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558436" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Requena, N.; Betz, R.; Heidt, S.; Figueira-Galan, D.; Langner, T.</p>
        <p class="info">Score: 7.3, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558436' target='https://doi.org/10.1101/2023.09.20.558436'> 10.1101/2023.09.20.558436</a></p>
        <p class="abstract">Most plants in natural ecosystems live in association with AM (arbuscular mycorrhizal) fungi to survive under poor nutrient conditions and to cope with other abiotic and biotic stresses. To engage in symbiosis, AM fungi secrete effector molecules that, similar to pathogenic effectors, reprogram plant cells. Despite numerous effectors being predicted in the genome of AM fungi, only a few have been functionally characterized. Here we show that the SP7-like family, a Glomeromycotina-specific effector family, impacts on the alternative splicing program of their hosts. SP7-like effectors localize at nuclear condensates and interact with the plant mRNA processing machinery, most prominently with the splicing factor SR45 and the core splicing proteins U170K and U2AF35. Ectopic expression of two of these effectors in the crop plant potato changed the alternative splicing pattern of a specific subset of genes. Unravelling the communication mechanisms between symbiotic fungi and their host plants will help to identify targets to improve plant nutrition.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558905">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558905" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558905">
        <p class="paperTitle">GENOMES UNCOUPLED1: an RNA-binding protein required for early PSII biogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558905" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558905" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cui, C.; Sun, S.; Zhang, S.; Zhang, Y.; Kim, C.</p>
        <p class="info">Score: 6.1, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558905' target='https://doi.org/10.1101/2023.09.21.558905'> 10.1101/2023.09.21.558905</a></p>
        <p class="abstract">Genomes Uncoupled1 (GUN1), a nuclear-encoded chloroplast pentatricopeptide repeat (PPR) protein, serves as a master integrator of diverse retrograde signals, mediating chloroplast-to-nucleus retrograde signaling. Although PPR proteins primarily function in organelle RNA metabolism, a target RNA of GUN1 remained unknown. This study reveals that GUN1 recognizes psbD transcripts derived from a blue light-responsive promoter (BLRP), transcripts referred to as psbD BLRP. Overexpression of GUN1 significantly reduces the levels of psbD BLRP, whereas the loss of GUN1 leads to the accumulation of the putative target RNA. The in vitro RNA and in vivo genetic studies further demonstrate the critical role of the C-terminal small MutS-related (SMR) domain in stimulating psbD BLRP processing and PsbD (D2, a PSII core protein) synthesis. The GUN1-dependent psbD BLRP processing promotes PSII biogenesis during the early seedling development and de-etiolation phase. This finding underscores the role of GUN1 as an RNA-binding protein, highlighting the essential function of the SMR domain in processing the target RNA.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.04.26.489498">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.04.26.489498" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.04.26.489498">
        <p class="paperTitle">Expression of cell-wall related genes is highly variable and correlates with sepal morphology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.04.26.489498" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.04.26.489498" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hartasanchez, D. A.; Kiss, A. K.; Battu, V.; Soraru, C.; Delgado Vaquera, A.; Massinon, F.; Braso-Vives, M.; Mollier, C.; Martin-Magnette, M.-L.; Boudaoud, A.; Moneger, F.</p>
        <p class="info">Score: 14.1, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.04.26.489498' target='https://doi.org/10.1101/2022.04.26.489498'> 10.1101/2022.04.26.489498</a></p>
        <p class="abstract">Control of organ morphology is a fundamental feature of living organisms. There is, however, observable variation in organ size and shape within a given genotype. Taking the sepal of Arabidopsis as a model, we investigated whether we can use variability of gene expression alongside variability of organ morphology to identify gene regulatory networks potentially involved in organ size and shape determination. We produced a dataset composed of morphological parameters and genome-wide transcriptome obtained from 27 individual sepals from wild-type plants with nearly identical genetic backgrounds, environment, and developmental stage. Sepals exhibited appreciable variability in both morphology and transcriptome, with response to stimulus genes and cell-wall related genes displaying high variability in expression. We additionally identified five modules of co-expressed genes which correlated significantly with morphology, revealing biologically relevant gene regulatory networks. Interestingly, cell-wall related genes were overrepresented in two of the top three modules. Overall, our work highlights the benefit of using coupled variation in gene expression and phenotype in wild-type plants to shed light on the mechanisms underlying organ size and shape determination. Although causality between gene expression and sepal morphology has not been established, our approach opens the way to informed analysis for mutant characterization and functional studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558677">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558677" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558677">
        <p class="paperTitle">Viable mutants of essential genes in Physcomitrium patens as tools for studying primary metabolic processes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558677" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558677" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Haslam, T. M.; Herrfurth, C.; Feussner, I.</p>
        <p class="info">Score: 4.8, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558677' target='https://doi.org/10.1101/2023.09.20.558677'> 10.1101/2023.09.20.558677</a></p>
        <p class="abstract">Sphingolipids are essential components of plant cells, which have been notoriously difficult to study in part due to pleiotropic or lethal knock-out mutant phenotypes. By relying on alternative end-joining of double stranded breaks, we successfully used CRISPR/Cas9 mutagenesis to generate a population of diverse, viable mutant alleles of genes required for sphingolipid assembly in totipotent protoplasts of the moss Physcomitrium patens. We targeted the INOSITOL PHOSPHORYLCERAMIDE SYNTHASE (IPCS) gene family, which catalyzes the committed step in the synthesis of glycosyl inositol phosphorylceramides (GIPCs), the most abundant class of sphingolipids found in plants. We isolated knock-out single mutants and knock-down higher-order mutants showing a spectrum of deficiencies in GIPC content. Remarkably, we also identified two mutant alleles accumulating inositol phosphorylceramides, the direct products of IPCS activity, and provide our best explanation for this unexpected phenotype. Our approach is broadly applicable for studying essential genes and gene families, and for obtaining unusual lesions within a gene of interest.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558385">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558385" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558385">
        <p class="paperTitle">High-resolution mapping of Ryd4Hb, a major resistance gene to Barley yellow dwarf virus from Hordeum bulbosum</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558385" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558385" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pidon, H.; Ruge-Wehling, B.; Will, T.; Habekuss, A.; Wendler, N.; Oldach, K.; Maasberg-Prelle, A.; Korzun, V.; Stein, N.</p>
        <p class="info">Score: 4.6, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558385' target='https://doi.org/10.1101/2023.09.19.558385'> 10.1101/2023.09.19.558385</a></p>
        <p class="abstract">Virus diseases are causing high yield losses in crops worldwide. The Barley yellow dwarf virus (BYDV) complex is responsible for one of the most widespread and economically important viral diseases of cereals. While no complete resistance gene has been uncovered in the primary genepool of barley, sources of resistance were identified in the wild relative Hordeum bulbosum, representing the secondary genepool of barley. One such locus, Ryd4Hb, has been previously introgressed into barley, and was allocated to chromosome 3H, but is tightly linked to a sublethality factor that prevents the incorporation and utilization of Ryd4Hb in barley varieties. To solve this problem, we fine-mapped Ryd4Hb and separated it from this negative factor. We narrowed the Ryd4Hb locus to a 66.5 kbp physical interval in the barley  Morex reference genome. The region comprises one complete and one partial gene from the nucleotide-binding and leucine-rich repeat immune receptor family, typical of dominant virus resistance genes. The closest homolog to these two Ryd4Hb candidate genes is the wheat Sr35 stem rust resistance gene. In addition to the fine mapping, we reduced the sublethality factor interval to 600 kbp in barley. Aphid feeding experiments demonstrated that Ryd4Hb provides a direct resistance to BYDV rather than a resistance to its vector. The presented results, including the high-throughput molecular markers, will permit a more targeted selection of the resistance in breeding, enabling the use of Ryd4Hb in barley varieties.

Key messageWe mapped Ryd4Hb in a 66.5 kpb interval in barley and dissociated it from a sublethality factor. These results will enable a targeted selection of the resistance in barley breeding.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
