<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The origin, deployment, and evolution of a plant-parasitic nematode effectorome
Authors: Molloy, B.; Shin, D. S.; Long, J.; Pellegrin, C.; Senatori, B.; Vieira, P.; Thorpe, P. J.; Damm, A.; Ahmad, M.; Vermeulen, K.; Derevnina, L.; Wei, S.; Sperling, A.; Reyes Estevez, E.; Bruty, S.; de Souza, V. H. M.; Kranse, O. P.; Maier, T.; Baum, T.; Eves-van den Akker, S.
Score: 39.8, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574317
Plant-parasitic nematodes constrain global food security.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="The origin, deployment, and evolution of a plant-parasitic nematode effectorome
Authors: Molloy, B.; Shin, D. S.; Long, J.; Pellegrin, C.; Senatori, B.; Vieira, P.; Thorpe, P. J.; Damm, A.; Ahmad, M.; Vermeulen, K.; Derevnina, L.; Wei, S.; Sperling, A.; Reyes Estevez, E.; Bruty, S.; de Souza, V. H. M.; Kranse, O. P.; Maier, T.; Baum, T.; Eves-van den Akker, S.
Score: 39.8, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574317
Plant-parasitic nematodes constrain global food security." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-10T10:40:05+00:00" />
<meta property="article:modified_time" content="2024-01-10T10:40:05+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="The origin, deployment, and evolution of a plant-parasitic nematode effectorome
Authors: Molloy, B.; Shin, D. S.; Long, J.; Pellegrin, C.; Senatori, B.; Vieira, P.; Thorpe, P. J.; Damm, A.; Ahmad, M.; Vermeulen, K.; Derevnina, L.; Wei, S.; Sperling, A.; Reyes Estevez, E.; Bruty, S.; de Souza, V. H. M.; Kranse, O. P.; Maier, T.; Baum, T.; Eves-van den Akker, S.
Score: 39.8, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574317
Plant-parasitic nematodes constrain global food security."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "The origin, deployment, and evolution of a plant-parasitic nematode effectorome\nAuthors: Molloy, B.; Shin, D. S.; Long, J.; Pellegrin, C.; Senatori, B.; Vieira, P.; Thorpe, P. J.; Damm, A.; Ahmad, M.; Vermeulen, K.; Derevnina, L.; Wei, S.; Sperling, A.; Reyes Estevez, E.; Bruty, S.; de Souza, V. H. M.; Kranse, O. P.; Maier, T.; Baum, T.; Eves-van den Akker, S.\nScore: 39.8, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574317\nPlant-parasitic nematodes constrain global food security.",
  "keywords": [
    
  ],
  "articleBody": " The origin, deployment, and evolution of a plant-parasitic nematode effectorome\nAuthors: Molloy, B.; Shin, D. S.; Long, J.; Pellegrin, C.; Senatori, B.; Vieira, P.; Thorpe, P. J.; Damm, A.; Ahmad, M.; Vermeulen, K.; Derevnina, L.; Wei, S.; Sperling, A.; Reyes Estevez, E.; Bruty, S.; de Souza, V. H. M.; Kranse, O. P.; Maier, T.; Baum, T.; Eves-van den Akker, S.\nScore: 39.8, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574317\nPlant-parasitic nematodes constrain global food security. During parasitism, they secrete effectors into the host plant from two types of pharyngeal gland cells. These effectors elicit profound changes in host biology to suppress immunity and establish a unique feeding organ from which the nematode draws nutrition. Despite the importance of effectors in nematode parasitism, there has been no comprehensive identification and characterisation of the effector repertoire of any plant-parasitic nematode. To address this, we advance techniques for gland cell isolation and transcriptional analysis to define a stringent annotation of putative effectors for the cyst nematode Heterodera schachtii at three key life-stages. We define 659 effector gene loci: 293 \"known\" high-confidence homologs of plant-parasitic nematode effectors, and 366 \"novel\" effectors with high gland cell expression. In doing so we define a comprehensive \"effectorome\" of a plant-parasitic nematode. Using this effector definition, we provide the first systems-level understanding of the origin, deployment and evolution of a plant-parasitic nematode effectorome. The robust identification of the comprehensive effector repertoire of a plant-parasitic nematode will underpin our understanding of nematode pathology, and hence, inform strategies for crop protection.\nNext-generation mapping of the salicylic acid signaling hub and transcriptional cascade\nAuthors: Powers, J.; Zhang, X.; Reyes, A. V.; Zavaliev, R.; Xu, S.-L.; Dong, X.\nScore: 33.4, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574047\nFor over 60 years, salicylic acid (SA) has been known as a plant immune signal required for both basal and systemic acquired resistance (SAR). SA activates these immune responses by reprogramming up to 20% of the transcriptome through the function of NPR1. However, components in the NPR1-signaling hub, which appears as nuclear condensates, and the NPR1-signaling cascade remained elusive due to difficulties in studying transcriptional cofactors whose chromatin associations are often indirect and transient. To overcome this challenge, we applied TurboID to divulge the NPR1-proxiome, which detected almost all known NPR1-interactors as well as new components of transcription-related complexes. Testing of new components showed that chromatin remodeling and histone demethylation contribute to SA-induced resistance. Globally, NPR1-proxiome shares a striking similarity to GBPL3-proxiome involved in SA synthesis, except associated transcription factors (TFs), suggesting that common regulatory modules are recruited to reprogram specific transcriptomes by transcriptional cofactors, like NPR1, through binding to unique TFs. Stepwise greenCUT\u0026RUN analyses showed that, upon SA-induction, NPR1 initiates the transcriptional cascade primarily through association with TGA TFs to induce expression of secondary TFs, predominantly WRKYs. WRKY54 and WRKY70 then play a major role in inducing immune-output genes without interacting with NPR1 at the chromatin. Moreover, a loss of NPR1 condensate formation decreases its chromatin-association and transcriptional activity, indicating the importance of condensates in organizing the NPR1-signaling hub and initiating the transcriptional cascade. This study demonstrates how combinatorial applications of TurboID and stepwise greenCUT\u0026RUN transcend traditional genetic methods to globally map signaling hubs and transcriptional cascades.\nInvestigating the cis-Regulatory Basis of C3 and C4 Photosynthesis in Grasses at Single-Cell Resolution\nAuthors: Mendieta, J. P.; Tu, X.; Yan, H.; Zhang, X.; Marrand, A. P.; Zhong, S.; Schmitz, R. J.\nScore: 31.9, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574340\nWhile considerable knowledge exists about the enzymes pivotal for C4 photosynthesis, much less is known about cis-regulation important for specifying their expression in distinct cell types. Here, we use single-cell-indexed ATAC-seq to identify cell-type-specific accessible chromatin regions (ACRs) associated with C4 enzymes for five different grass species. This study spans four C4 species, covering three distinct photosynthetic subtypes: Zea mays and Sorghum bicolor (NADP-ME), Panicum miliaceum (NAD-ME), Urochloa fusca (PEPCK), along with the C3 outgroup Oryza sativa. We studied the cis-regulatory landscape of enzymes essential across all C4 species and those unique to C4 subtypes, measuring cell-type-specific biases for C4 enzymes using chromatin accessibility data. Integrating these data with phylogenetic trees revealed diverse co-option of gene family members between species, showcasing the various paths of C4 evolution. We mapped cell-type-specific ACRs surrounding each C4 gene, discovering that, on average, these genes have two to three cell-type-specific ACRs located beyond the core promoter. These results provide a detailed genomic map of potential regulatory sequences operating on key C4 genes. Examining the evolutionary history of these cell-type-specific ACRs revealed a spectrum of conserved and novel ACRs, even among closely related species, indicating ongoing evolution of cis-regulation at these C4 loci. This work illuminates the dynamic and complex nature of CRE evolution in C4 photosynthesis, particularly highlighting the intricate cis-regulatory evolution of key loci. Our findings offer a valuable resource for future investigations, potentially aiding in the optimization of C3 crop performance under changing climatic conditions.\nEvolution of cell-type-specific accessible chromatin regions and the cis-regulatory elements that drive lineage-specific innovation\nAuthors: Yan, H.; Mendieta, J. P.; Zhang, X.; Marand, A. P.; Liang, Y.; Luo, Z.; Roule, T.; Wagner, D.; Tu, X.; Wang, Y.; Zhong, S.; Wessler, S. R.; Schmitz, R. J.\nScore: 15.6, Published: 2024-01-09 DOI: 10.1101/2024.01.08.574753\nCis-regulatory elements (CREs) are critical in coordinating the regulation of gene expression. Understanding how CREs evolve represents a challenging endeavor. Here, we constructed a comprehensive single-cell atlas of CRE activity in Oryza sativa, integrating data from 104,029 nuclei representing 128 discrete cell states across nine distinct organs. We used a comparative genomic approach to analyze cell-type-specific CRE activity between Oryza sativa and four additional grass species (Zea mays, Sorghum bicolor, Panicum miliaceum, and Urochloa fusca). We revealed a complex interplay whereby accessible chromatin regions (ACRs) had different levels of conservation dependent on if these regions were broadly accessible, or cell-type specific. We found that on average epidermal ACRs were less conserved in regions of synteny compared to other cell types, potentially indicating that large amounts of regulatory evolution has taken place in the epidermal layers of these species. We found a significant overlap between cell-type-specific ACRs and conserved non-coding sequences. Finally, we identified a subset of ACRs that were overlapping the repressive histone modification H3k27me3, which were conserved across evolutionary time, implicating them as potentially critical silencer CREs. Collectively, these results highlight the dynamic evolution of cell-type-specific CRE activity using comparative genomics.\nDynamic evolution of the heterochromatin sensing histone demethylase IBM1\nAuthors: Zhang, Y.; Jang, H.; Luo, Z.; Schmitz, R. J.\nScore: 11.7, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574644\nHeterochromatin constitutes a fundamental aspect of genomes that is crucial for maintaining genome stability. In flowering plants, maintenance of heterochromatin relies on a positive feedback loop involving the histone 3 lysine nine methyltransferase (H3K9), KRYPTONITE (KYP), and the DNA methyltransferase, CHROMOMETHYLASE3 (CMT3). An H3K9 demethylase, INCREASED IN BONSAI METHYLATION 1 (IBM1), has evolved to modulate the activity of KYP-CMT3 within transcribed genes. The absence of IBM1 activity results in aberrant methylation of gene bodies, which is deleterious. This study demonstrates extensive genetic and gene expression variations in KYP, CMT3, and IBM1 within and between flowering plant species. IBM1 activity in Arabidopsis thaliana is uniquely regulated by the abundance of H3K9me2 in a repetitive sequence within an intron preceding the histone demethylase domain. This mechanism enables IBM1 to monitor global levels of H3K9me2. We discovered that the methylated intron is prevalent across flowering plants, however, its underlying sequence exhibits dynamic evolution. Its absence in species lacking gene body DNA methylation suggests its primary role in sensing H3K9me2 and preventing its integration into these constitutively expressed genes. Furthermore, our investigation uncovered Arabidopsis thaliana accessions resembling weak ibm1 mutants, several Brassicaceae species with reduced IBM1 expression, and an IBM1 deletion. Evolution towards reduced IBM1 activity in some flowering plants could explain the frequent natural occurrence of diminished or lost CMT3 activity, as cmt3 mutants in A. thaliana mitigate the deleterious effects of IBM1.\nGenetically encoded Boolean logic operators to sense and integrate phenylpropanoid metabolite levels in plants\nAuthors: Ferreira, S. S.; Antunes, M. S.\nScore: 9.6, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574427\nO_LISynthetic biology has the potential to revolutionize biotechnology, public health and agriculture. Recent studies have shown the enormous potential of plants as chassis for synthetic biology applications. However, tools to precisely manipulate metabolic pathways for bioproduction in plants are still needed. C_LIO_LIWe have adapted bacterial allosteric transcription factors (aTFs) to control gene expression in plants in a ligand-specific manner. The aTFs used here function as transcription repressors of semi-synthetic promoters, and aTF activity is regulated by specific plant metabolites, especially phenylpropanoid-related molecules. Using these aTFs, we also designed synthetic genetic circuits capable of computing Boolean logic operations. C_LIO_LIThree aTFs, CouR, FapR and TtgR, were able to achieve [~]95% repression of their respective target promoters. For TtgR, a 6-fold de-repression could be triggered by inducing its ligand (naringenin) accumulation, showing its use as biosensor. Moreover, we designed synthetic genetic circuits that use AND, NAND, IMPLY and NIMPLY Boolean logic operations and integrate metabolite levels as input to the circuit. C_LIO_LIWe showed that biosensors can be implemented in plants to detect phenylpropanoid-related metabolites and activate a genetic circuit that follows a pre-defined logic, demonstrating their potential as tools for exerting control over plant metabolic pathways and facilitating the bioproduction of natural products. C_LI\nER-anchored protein sorting controls the fate of two proteasome activators for intracellular organelle communication during proteotoxic stress\nAuthors: Langin, G.; Raffeiner, M.; Biermann, D.; Franz-Wachtel, M.; Spinti, D.; Börnke, F.; Macek, B.; Üstün, S.\nScore: 40.7, Published: 2023-12-12 DOI: 10.1101/2023.12.11.571118\nProteotoxic stress, characterized by the accumulation of damaged proteins, poses a significant challenge to cellular homeostasis. To mitigate proteotoxicity eukaryotes employ the proteasome that is regulated by proteasome activators, e.g. transcription factors that promote gene expression of proteasome subunits. As proteotoxicity originates in different compartments, cells need to perceive signals from various locations. Understanding which components integrate signals to address proteotoxicity is essential to develop strategies to cope with proteotoxicity but remain elusive. Here, we identify that the proteasome autoregulatory feedback loop acts as a gatekeeper to facilitate the communication between nucleus and chloroplast. We reveal that the ER-anchored protein sorting system (ERAPS) controls the proteasomal degradation or nuclear translocation of proteasome activators NAC53 and NAC78. While both transcription factors activate the proteasome gene expression, they repress photosynthesis-associated nuclear genes during proteotoxicity. Collectively, our discoveries provide a novel conceptual framework in which the proteasome autoregulatory feedback loop coordinates subcellular proteostasis.\nMesophyll-Specific Circadian Dynamics of CAM Induction in the Ice Plant Unveiled by Single-Cell Transcriptomics\nAuthors: Perron, N.; Dervinis, C.; Pereira, W. J.; Barbazuk, W. B.; Kirst, M.\nScore: 8.1, Published: 2024-01-08 DOI: 10.1101/2024.01.05.574430\nCrassulacean acid metabolism (CAM) is an evolutionary modification of the C3 photosynthetic carbon dioxide fixation pathway used by approximately 7% of terrestrial plants to live in drought-prone environments. Facultative CAM species, such as Mesembryanthemum crystallinum (common ice plant), possess the unique ability to switch from C3 to CAM photosynthesis in response to high-salinity and water-deficit stress. Here we characterized the environmentally-triggered transition from C3 to CAM in the ice plant using single nucleus RNA sequencing (snRNA-seq) to identify its putative regulators, supported by a novel high-quality assembled and annotated genome. Analysis of snRNA-seq datasets from ice plant leaves transitioning between C3 and CAM collected at dawn and dusk revealed substantial transcriptional changes in mesophyll cells at the onset of CAM induction. Notably, our findings identify mesophyll sub-cell types engaged in either CAM or C3 photosynthesis at dusk. Cell trajectory inference analysis reconstructed both 24-hour CAM and C3 cycles, enabling a direct comparison of gene expression profiles in these pathways. This comparative study uncovered divergent expression patterns of key circadian clock genes in CAM and C3 cell trajectories, pointing to a connection between circadian regulation and CAM induction.\nThe genome of Lolium multiflorum reveals the genetic architecture of paraquat resistance\nAuthors: Brunharo, C. A.; Short, A. W.; Bobadilla, L. K.; Streisfeld, M. A.\nScore: 7.1, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573904\n- Herbicide resistance in agricultural weeds has become one of the greatest challenges for sustainable crop production. The repeated evolution of herbicide resistance provides an excellent opportunity to study the genetic and physiological basis of the resistance phenotype and the evolutionary responses to human-mediated selection pressures. Lolium multiflorum is a ubiquitous weed that has evolved herbicide resistance repeatedly around the world in various cropping systems. - We assembled and annotated a chromosome-scale genome for L. multiflorum and elucidated the genetic architecture of paraquat resistance by performing quantitative trait loci analysis, genome-wide association studies, genetic divergence analysis, and transcriptome analyses from paraquat-resistant and -susceptible L. multiflorum populations. - Results suggested that two regions of chromosome 5 were associated with paraquat resistance. The regions contain candidate genes that encode cellular transport functions, including a novel multidrug and toxin extrusion (MATE) protein, and a cation transporter previously shown to interact with polyamines. - Our results reveal the genetic architecture of paraquat resistance and identified promising candidate genes for future functional studies. Given that L. multiflorum is a weed and a cultivated crop species, the genomic resources generated will prove valuable to a wide spectrum of the plant science community.\nHigh-resolution metabolite imaging: luteolinidin accumulates at the host-cyanobiont interface during cold-acclimation in Azolla symbioses\nAuthors: Güngör, E.; Bartels, B.; Bolchi, G.; Heeren, R. M. A.; Ellis, S.; Schluepmann, H.\nScore: 5.8, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574342\nAquatic ferns of the genus Azolla (Azolla) form highly productive symbioses with filamentous cyanobacteria fixing N2 in their leaf cavities, Nostoc azollae. Stressed symbioses characteristically turn red due to 3-deoxyanthocyanin (DA) accumulation, rare in angiosperms and of unknown function. To reveal DA functions upon cold acclimation and recovery, we integrated laser-desorption-ionization mass-spectrometry-imaging (LDI-MSI), a new A. filiculoides genome-assembly and annotation, and dual RNA-sequencing into phenotypic analyses of the symbioses. Azolla sp. Anzali recovered even when cold-induced DA-accumulation was inhibited by abscisic acid. Cyanobacterial filaments generally disappeared upon cold acclimation, and N. azollae transcript profiles were unlike those of resting stages formed in cold-resistant sporocarps, yet filaments re-appeared in leaf cavities of newly formed green fronds upon cold-recovery. The high transcript accumulation upon cold acclimation of AfDFR1 encoding a flavanone 4-reductase active in vitro suggested that the enzyme of the first step in the DA-pathway may regulate accumulation of DAs in different tissues. However, LDI-MSI highlighted the necessity to describe metabolite accumulation beyond class assignments as individual DA and caffeoylquinic acid metabolites accumulated differentially. For example, luteolinidin accumulated in epithelial cells, including those lining the leaf cavity, supporting a role for the former in the symbiotic interaction during cold acclimation. Summary statementDuring cold acclimation in Azolla symbioses, individual compounds from the same phenolic class accumulated in different host tissues: luteolinidin associated with biotic interactions at the symbiosis interface whilst apigenidin with photooxidative stress mitigation in the mesophyll.\n",
  "wordCount" : "2442",
  "inLanguage": "en",
  "datePublished": "2024-01-10T10:40:05Z",
  "dateModified": "2024-01-10T10:40:05Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 10, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574317">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574317" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574317">
        <p class="paperTitle">The origin, deployment, and evolution of a plant-parasitic nematode effectorome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574317" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574317" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Molloy, B.; Shin, D. S.; Long, J.; Pellegrin, C.; Senatori, B.; Vieira, P.; Thorpe, P. J.; Damm, A.; Ahmad, M.; Vermeulen, K.; Derevnina, L.; Wei, S.; Sperling, A.; Reyes Estevez, E.; Bruty, S.; de Souza, V. H. M.; Kranse, O. P.; Maier, T.; Baum, T.; Eves-van den Akker, S.</p>
        <p class="info">Score: 39.8, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574317' target='https://doi.org/10.1101/2024.01.05.574317'> 10.1101/2024.01.05.574317</a></p>
        <p class="abstract">Plant-parasitic nematodes constrain global food security. During parasitism, they secrete effectors into the host plant from two types of pharyngeal gland cells. These effectors elicit profound changes in host biology to suppress immunity and establish a unique feeding organ from which the nematode draws nutrition. Despite the importance of effectors in nematode parasitism, there has been no comprehensive identification and characterisation of the effector repertoire of any plant-parasitic nematode.

To address this, we advance techniques for gland cell isolation and transcriptional analysis to define a stringent annotation of putative effectors for the cyst nematode Heterodera schachtii at three key life-stages. We define 659 effector gene loci: 293 &#34;known&#34; high-confidence homologs of plant-parasitic nematode effectors, and 366 &#34;novel&#34; effectors with high gland cell expression. In doing so we define a comprehensive &#34;effectorome&#34; of a plant-parasitic nematode.

Using this effector definition, we provide the first systems-level understanding of the origin, deployment and evolution of a plant-parasitic nematode effectorome. The robust identification of the comprehensive effector repertoire of a plant-parasitic nematode will underpin our understanding of nematode pathology, and hence, inform strategies for crop protection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574047">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574047" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574047">
        <p class="paperTitle">Next-generation mapping of the salicylic acid signaling hub and transcriptional cascade</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574047" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574047" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Powers, J.; Zhang, X.; Reyes, A. V.; Zavaliev, R.; Xu, S.-L.; Dong, X.</p>
        <p class="info">Score: 33.4, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574047' target='https://doi.org/10.1101/2024.01.03.574047'> 10.1101/2024.01.03.574047</a></p>
        <p class="abstract">For over 60 years, salicylic acid (SA) has been known as a plant immune signal required for both basal and systemic acquired resistance (SAR). SA activates these immune responses by reprogramming up to 20% of the transcriptome through the function of NPR1. However, components in the NPR1-signaling hub, which appears as nuclear condensates, and the NPR1-signaling cascade remained elusive due to difficulties in studying transcriptional cofactors whose chromatin associations are often indirect and transient. To overcome this challenge, we applied TurboID to divulge the NPR1-proxiome, which detected almost all known NPR1-interactors as well as new components of transcription-related complexes. Testing of new components showed that chromatin remodeling and histone demethylation contribute to SA-induced resistance. Globally, NPR1-proxiome shares a striking similarity to GBPL3-proxiome involved in SA synthesis, except associated transcription factors (TFs), suggesting that common regulatory modules are recruited to reprogram specific transcriptomes by transcriptional cofactors, like NPR1, through binding to unique TFs. Stepwise greenCUT&amp;RUN analyses showed that, upon SA-induction, NPR1 initiates the transcriptional cascade primarily through association with TGA TFs to induce expression of secondary TFs, predominantly WRKYs. WRKY54 and WRKY70 then play a major role in inducing immune-output genes without interacting with NPR1 at the chromatin. Moreover, a loss of NPR1 condensate formation decreases its chromatin-association and transcriptional activity, indicating the importance of condensates in organizing the NPR1-signaling hub and initiating the transcriptional cascade. This study demonstrates how combinatorial applications of TurboID and stepwise greenCUT&amp;RUN transcend traditional genetic methods to globally map signaling hubs and transcriptional cascades.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574340">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574340" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574340">
        <p class="paperTitle">Investigating the cis-Regulatory Basis of C3 and C4 Photosynthesis in Grasses at Single-Cell Resolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574340" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574340" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mendieta, J. P.; Tu, X.; Yan, H.; Zhang, X.; Marrand, A. P.; Zhong, S.; Schmitz, R. J.</p>
        <p class="info">Score: 31.9, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574340' target='https://doi.org/10.1101/2024.01.05.574340'> 10.1101/2024.01.05.574340</a></p>
        <p class="abstract">While considerable knowledge exists about the enzymes pivotal for C4 photosynthesis, much less is known about cis-regulation important for specifying their expression in distinct cell types. Here, we use single-cell-indexed ATAC-seq to identify cell-type-specific accessible chromatin regions (ACRs) associated with C4 enzymes for five different grass species. This study spans four C4 species, covering three distinct photosynthetic subtypes: Zea mays and Sorghum bicolor (NADP-ME), Panicum miliaceum (NAD-ME), Urochloa fusca (PEPCK), along with the C3 outgroup Oryza sativa. We studied the cis-regulatory landscape of enzymes essential across all C4 species and those unique to C4 subtypes, measuring cell-type-specific biases for C4 enzymes using chromatin accessibility data. Integrating these data with phylogenetic trees revealed diverse co-option of gene family members between species, showcasing the various paths of C4 evolution. We mapped cell-type-specific ACRs surrounding each C4 gene, discovering that, on average, these genes have two to three cell-type-specific ACRs located beyond the core promoter. These results provide a detailed genomic map of potential regulatory sequences operating on key C4 genes. Examining the evolutionary history of these cell-type-specific ACRs revealed a spectrum of conserved and novel ACRs, even among closely related species, indicating ongoing evolution of cis-regulation at these C4 loci. This work illuminates the dynamic and complex nature of CRE evolution in C4 photosynthesis, particularly highlighting the intricate cis-regulatory evolution of key loci. Our findings offer a valuable resource for future investigations, potentially aiding in the optimization of C3 crop performance under changing climatic conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574753">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574753" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574753">
        <p class="paperTitle">Evolution of cell-type-specific accessible chromatin regions and the cis-regulatory elements that drive lineage-specific innovation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574753" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574753" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yan, H.; Mendieta, J. P.; Zhang, X.; Marand, A. P.; Liang, Y.; Luo, Z.; Roule, T.; Wagner, D.; Tu, X.; Wang, Y.; Zhong, S.; Wessler, S. R.; Schmitz, R. J.</p>
        <p class="info">Score: 15.6, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574753' target='https://doi.org/10.1101/2024.01.08.574753'> 10.1101/2024.01.08.574753</a></p>
        <p class="abstract">Cis-regulatory elements (CREs) are critical in coordinating the regulation of gene expression. Understanding how CREs evolve represents a challenging endeavor. Here, we constructed a comprehensive single-cell atlas of CRE activity in Oryza sativa, integrating data from 104,029 nuclei representing 128 discrete cell states across nine distinct organs. We used a comparative genomic approach to analyze cell-type-specific CRE activity between Oryza sativa and four additional grass species (Zea mays, Sorghum bicolor, Panicum miliaceum, and Urochloa fusca). We revealed a complex interplay whereby accessible chromatin regions (ACRs) had different levels of conservation dependent on if these regions were broadly accessible, or cell-type specific. We found that on average epidermal ACRs were less conserved in regions of synteny compared to other cell types, potentially indicating that large amounts of regulatory evolution has taken place in the epidermal layers of these species. We found a significant overlap between cell-type-specific ACRs and conserved non-coding sequences. Finally, we identified a subset of ACRs that were overlapping the repressive histone modification H3k27me3, which were conserved across evolutionary time, implicating them as potentially critical silencer CREs. Collectively, these results highlight the dynamic evolution of cell-type-specific CRE activity using comparative genomics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574644">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574644" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574644">
        <p class="paperTitle">Dynamic evolution of the heterochromatin sensing histone demethylase IBM1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574644" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574644" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Y.; Jang, H.; Luo, Z.; Schmitz, R. J.</p>
        <p class="info">Score: 11.7, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574644' target='https://doi.org/10.1101/2024.01.08.574644'> 10.1101/2024.01.08.574644</a></p>
        <p class="abstract">Heterochromatin constitutes a fundamental aspect of genomes that is crucial for maintaining genome stability. In flowering plants, maintenance of heterochromatin relies on a positive feedback loop involving the histone 3 lysine nine methyltransferase (H3K9), KRYPTONITE (KYP), and the DNA methyltransferase, CHROMOMETHYLASE3 (CMT3). An H3K9 demethylase, INCREASED IN BONSAI METHYLATION 1 (IBM1), has evolved to modulate the activity of KYP-CMT3 within transcribed genes. The absence of IBM1 activity results in aberrant methylation of gene bodies, which is deleterious. This study demonstrates extensive genetic and gene expression variations in KYP, CMT3, and IBM1 within and between flowering plant species. IBM1 activity in Arabidopsis thaliana is uniquely regulated by the abundance of H3K9me2 in a repetitive sequence within an intron preceding the histone demethylase domain. This mechanism enables IBM1 to monitor global levels of H3K9me2. We discovered that the methylated intron is prevalent across flowering plants, however, its underlying sequence exhibits dynamic evolution. Its absence in species lacking gene body DNA methylation suggests its primary role in sensing H3K9me2 and preventing its integration into these constitutively expressed genes. Furthermore, our investigation uncovered Arabidopsis thaliana accessions resembling weak ibm1 mutants, several Brassicaceae species with reduced IBM1 expression, and an IBM1 deletion. Evolution towards reduced IBM1 activity in some flowering plants could explain the frequent natural occurrence of diminished or lost CMT3 activity, as cmt3 mutants in A. thaliana mitigate the deleterious effects of IBM1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574427">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574427" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574427">
        <p class="paperTitle">Genetically encoded Boolean logic operators to sense and integrate phenylpropanoid metabolite levels in plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574427" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574427" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ferreira, S. S.; Antunes, M. S.</p>
        <p class="info">Score: 9.6, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574427' target='https://doi.org/10.1101/2024.01.05.574427'> 10.1101/2024.01.05.574427</a></p>
        <p class="abstract">O_LISynthetic biology has the potential to revolutionize biotechnology, public health and agriculture. Recent studies have shown the enormous potential of plants as chassis for synthetic biology applications. However, tools to precisely manipulate metabolic pathways for bioproduction in plants are still needed.
C_LIO_LIWe have adapted bacterial allosteric transcription factors (aTFs) to control gene expression in plants in a ligand-specific manner. The aTFs used here function as transcription repressors of semi-synthetic promoters, and aTF activity is regulated by specific plant metabolites, especially phenylpropanoid-related molecules. Using these aTFs, we also designed synthetic genetic circuits capable of computing Boolean logic operations.
C_LIO_LIThree aTFs, CouR, FapR and TtgR, were able to achieve [~]95% repression of their respective target promoters. For TtgR, a 6-fold de-repression could be triggered by inducing its ligand (naringenin) accumulation, showing its use as biosensor. Moreover, we designed synthetic genetic circuits that use AND, NAND, IMPLY and NIMPLY Boolean logic operations and integrate metabolite levels as input to the circuit.
C_LIO_LIWe showed that biosensors can be implemented in plants to detect phenylpropanoid-related metabolites and activate a genetic circuit that follows a pre-defined logic, demonstrating their potential as tools for exerting control over plant metabolic pathways and facilitating the bioproduction of natural products.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.571118">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.571118" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.571118">
        <p class="paperTitle">ER-anchored protein sorting controls the fate of two proteasome activators for intracellular organelle communication during proteotoxic stress</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.571118" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.571118" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Langin, G.; Raffeiner, M.; Biermann, D.; Franz-Wachtel, M.; Spinti, D.; Börnke, F.; Macek, B.; Üstün, S.</p>
        <p class="info">Score: 40.7, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.571118' target='https://doi.org/10.1101/2023.12.11.571118'> 10.1101/2023.12.11.571118</a></p>
        <p class="abstract">Proteotoxic stress, characterized by the accumulation of damaged proteins, poses a significant challenge to cellular homeostasis. To mitigate proteotoxicity eukaryotes employ the proteasome that is regulated by proteasome activators, e.g. transcription factors that promote gene expression of proteasome subunits. As proteotoxicity originates in different compartments, cells need to perceive signals from various locations. Understanding which components integrate signals to address proteotoxicity is essential to develop strategies to cope with proteotoxicity but remain elusive. Here, we identify that the proteasome autoregulatory feedback loop acts as a gatekeeper to facilitate the communication between nucleus and chloroplast. We reveal that the ER-anchored protein sorting system (ERAPS) controls the proteasomal degradation or nuclear translocation of proteasome activators NAC53 and NAC78. While both transcription factors activate the proteasome gene expression, they repress photosynthesis-associated nuclear genes during proteotoxicity. Collectively, our discoveries provide a novel conceptual framework in which the proteasome autoregulatory feedback loop coordinates subcellular proteostasis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574430">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574430" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574430">
        <p class="paperTitle">Mesophyll-Specific Circadian Dynamics of CAM Induction in the Ice Plant Unveiled by Single-Cell Transcriptomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574430" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574430" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Perron, N.; Dervinis, C.; Pereira, W. J.; Barbazuk, W. B.; Kirst, M.</p>
        <p class="info">Score: 8.1, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574430' target='https://doi.org/10.1101/2024.01.05.574430'> 10.1101/2024.01.05.574430</a></p>
        <p class="abstract">Crassulacean acid metabolism (CAM) is an evolutionary modification of the C3 photosynthetic carbon dioxide fixation pathway used by approximately 7% of terrestrial plants to live in drought-prone environments. Facultative CAM species, such as Mesembryanthemum crystallinum (common ice plant), possess the unique ability to switch from C3 to CAM photosynthesis in response to high-salinity and water-deficit stress. Here we characterized the environmentally-triggered transition from C3 to CAM in the ice plant using single nucleus RNA sequencing (snRNA-seq) to identify its putative regulators, supported by a novel high-quality assembled and annotated genome. Analysis of snRNA-seq datasets from ice plant leaves transitioning between C3 and CAM collected at dawn and dusk revealed substantial transcriptional changes in mesophyll cells at the onset of CAM induction. Notably, our findings identify mesophyll sub-cell types engaged in either CAM or C3 photosynthesis at dusk. Cell trajectory inference analysis reconstructed both 24-hour CAM and C3 cycles, enabling a direct comparison of gene expression profiles in these pathways. This comparative study uncovered divergent expression patterns of key circadian clock genes in CAM and C3 cell trajectories, pointing to a connection between circadian regulation and CAM induction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573904">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573904" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573904">
        <p class="paperTitle">The genome of Lolium multiflorum reveals the genetic architecture of paraquat resistance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573904" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573904" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brunharo, C. A.; Short, A. W.; Bobadilla, L. K.; Streisfeld, M. A.</p>
        <p class="info">Score: 7.1, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573904' target='https://doi.org/10.1101/2024.01.02.573904'> 10.1101/2024.01.02.573904</a></p>
        <p class="abstract">- Herbicide resistance in agricultural weeds has become one of the greatest challenges for sustainable crop production. The repeated evolution of herbicide resistance provides an excellent opportunity to study the genetic and physiological basis of the resistance phenotype and the evolutionary responses to human-mediated selection pressures. Lolium multiflorum is a ubiquitous weed that has evolved herbicide resistance repeatedly around the world in various cropping systems.
- We assembled and annotated a chromosome-scale genome for L. multiflorum and elucidated the genetic architecture of paraquat resistance by performing quantitative trait loci analysis, genome-wide association studies, genetic divergence analysis, and transcriptome analyses from paraquat-resistant and -susceptible L. multiflorum populations.
- Results suggested that two regions of chromosome 5 were associated with paraquat resistance. The regions contain candidate genes that encode cellular transport functions, including a novel multidrug and toxin extrusion (MATE) protein, and a cation transporter previously shown to interact with polyamines.
- Our results reveal the genetic architecture of paraquat resistance and identified promising candidate genes for future functional studies. Given that L. multiflorum is a weed and a cultivated crop species, the genomic resources generated will prove valuable to a wide spectrum of the plant science community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574342">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574342" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574342">
        <p class="paperTitle">High-resolution metabolite imaging: luteolinidin accumulates at the host-cyanobiont interface during cold-acclimation in Azolla symbioses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574342" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574342" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Güngör, E.; Bartels, B.; Bolchi, G.; Heeren, R. M. A.; Ellis, S.; Schluepmann, H.</p>
        <p class="info">Score: 5.8, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574342' target='https://doi.org/10.1101/2024.01.05.574342'> 10.1101/2024.01.05.574342</a></p>
        <p class="abstract">Aquatic ferns of the genus Azolla (Azolla) form highly productive symbioses with filamentous cyanobacteria fixing N2 in their leaf cavities, Nostoc azollae. Stressed symbioses characteristically turn red due to 3-deoxyanthocyanin (DA) accumulation, rare in angiosperms and of unknown function. To reveal DA functions upon cold acclimation and recovery, we integrated laser-desorption-ionization mass-spectrometry-imaging (LDI-MSI), a new A. filiculoides genome-assembly and annotation, and dual RNA-sequencing into phenotypic analyses of the symbioses.

Azolla sp. Anzali recovered even when cold-induced DA-accumulation was inhibited by abscisic acid. Cyanobacterial filaments generally disappeared upon cold acclimation, and N. azollae transcript profiles were unlike those of resting stages formed in cold-resistant sporocarps, yet filaments re-appeared in leaf cavities of newly formed green fronds upon cold-recovery.

The high transcript accumulation upon cold acclimation of AfDFR1 encoding a flavanone 4-reductase active in vitro suggested that the enzyme of the first step in the DA-pathway may regulate accumulation of DAs in different tissues. However, LDI-MSI highlighted the necessity to describe metabolite accumulation beyond class assignments as individual DA and caffeoylquinic acid metabolites accumulated differentially. For example, luteolinidin accumulated in epithelial cells, including those lining the leaf cavity, supporting a role for the former in the symbiotic interaction during cold acclimation.

Summary statementDuring cold acclimation in Azolla symbioses, individual compounds from the same phenolic class accumulated in different host tissues: luteolinidin associated with biotic interactions at the symbiosis interface whilst apigenidin with photooxidative stress mitigation in the mesophyll.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
