<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Cellular gibberellin dynamics govern indeterminate nodule development, morphologyand function
Authors: Drapek, C.; Radzman-Mohd, N. A.; Rizza, A.; Schiessl, K.; Dos Santos Barbosa, F.; Wen, J.; Oldroyd, G. E. D.; Jones, A. M.
Score: 34.0, Published: 2023-09-11 DOI: 10.1101/2023.09.09.556959
During nutrient scarcity, plants can adapt their developmental strategy to maximize their chance of survival. Such plasticity in development is underpinned by hormonal regulation, which mediates the relationship between environmental cues and developmental outputs.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="Cellular gibberellin dynamics govern indeterminate nodule development, morphologyand function
Authors: Drapek, C.; Radzman-Mohd, N. A.; Rizza, A.; Schiessl, K.; Dos Santos Barbosa, F.; Wen, J.; Oldroyd, G. E. D.; Jones, A. M.
Score: 34.0, Published: 2023-09-11 DOI: 10.1101/2023.09.09.556959
During nutrient scarcity, plants can adapt their developmental strategy to maximize their chance of survival. Such plasticity in development is underpinned by hormonal regulation, which mediates the relationship between environmental cues and developmental outputs." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-17T10:36:50+00:00" />
<meta property="article:modified_time" content="2023-09-17T10:36:50+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="Cellular gibberellin dynamics govern indeterminate nodule development, morphologyand function
Authors: Drapek, C.; Radzman-Mohd, N. A.; Rizza, A.; Schiessl, K.; Dos Santos Barbosa, F.; Wen, J.; Oldroyd, G. E. D.; Jones, A. M.
Score: 34.0, Published: 2023-09-11 DOI: 10.1101/2023.09.09.556959
During nutrient scarcity, plants can adapt their developmental strategy to maximize their chance of survival. Such plasticity in development is underpinned by hormonal regulation, which mediates the relationship between environmental cues and developmental outputs."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "Cellular gibberellin dynamics govern indeterminate nodule development, morphologyand function\nAuthors: Drapek, C.; Radzman-Mohd, N. A.; Rizza, A.; Schiessl, K.; Dos Santos Barbosa, F.; Wen, J.; Oldroyd, G. E. D.; Jones, A. M.\nScore: 34.0, Published: 2023-09-11 DOI: 10.1101/2023.09.09.556959\nDuring nutrient scarcity, plants can adapt their developmental strategy to maximize their chance of survival. Such plasticity in development is underpinned by hormonal regulation, which mediates the relationship between environmental cues and developmental outputs.",
  "keywords": [
    
  ],
  "articleBody": " Cellular gibberellin dynamics govern indeterminate nodule development, morphologyand function\nAuthors: Drapek, C.; Radzman-Mohd, N. A.; Rizza, A.; Schiessl, K.; Dos Santos Barbosa, F.; Wen, J.; Oldroyd, G. E. D.; Jones, A. M.\nScore: 34.0, Published: 2023-09-11 DOI: 10.1101/2023.09.09.556959\nDuring nutrient scarcity, plants can adapt their developmental strategy to maximize their chance of survival. Such plasticity in development is underpinned by hormonal regulation, which mediates the relationship between environmental cues and developmental outputs. In legumes, endosymbiosis with nitrogen fixing bacteria (rhizobia) is a key adaptation for supplying the plant with nitrogen in the form of ammonium. Rhizobia are housed in lateral root-derived organs termed nodules that maintain an environment conducive to Nitrogenase in these bacteria. Several phytohormones are important for regulating the formation of nodules, with both positive and negative roles proposed for gibberellin (GA). In this study, we determined the cellular location and function of bioactive GA during nodule organogenesis using a genetically-encoded second generation GA biosensor, GPS2. We found endogenous bioactive GA accumulates locally at the site of nodule primordia, increasing dramatically in the cortical cell layers, persisting through cell divisions and maintaining accumulation in the mature nodule meristem. We show, through mis-expression of GA catabolic enzymes that suppress GA accumulation, that GA acts as a positive regulator of nodule growth and development. Furthermore, increasing or decreasing GA through perturbation of biosynthesis gene expression can increase or decrease the size of nodules, respectively. This is unique from lateral root formation, a developmental program that shares common organogenesis regulators. We link GA to a wider gene regulatory program by showing that cytokinin as well as nodule-identity genes induce and sustain GA accumulation necessary for nodule function.\nRetrotransposon-driven environmental regulation of FLC leads to adaptive response to herbicide\nAuthors: Raingeval, M.; Leduque, B.; Baduel, P.; Edera, A.; Roux, F.; Colot, V.; Quadrana, L.\nScore: 35.8, Published: 2023-09-07 DOI: 10.1101/2023.09.06.556499\nThe mobilization of retrotransposons yields major-effect mutations. Here, we report an adaptive retrotransposon insertion within the first intron of the Arabidopsis floral-repressor locus FLOWERING LOCUS C (FLC). The insertion-mutation augments the environmental sensitivity of FLC by affecting the balance between coding and non-coding transcript isoforms in response to environmental threads. We show that this balance is modulated epigenetically by DNA methylation and orchestrated by IBM2, a factor involved in the processing of intronic heterochromatin. The stress-sensitive allele of FLC has recently spread across populations subjected to recurrent chemical weeding, and we demonstrate that retrotransposon-driven acceleration of life cycle represents a rapid response to herbicide. Our findings illustrate how retrotransposition can create environmentally-sensitive alleles that facilitate adaptation to anthropogenic disturbances of the environment.\nRubisco is evolving for improved catalytic efficiency and CO2 assimilation in plants\nAuthors: Bouvier, J.; Emms, D.; Kelly, S.\nScore: 35.4, Published: 2023-09-12 DOI: 10.1101/2022.07.06.498985\nRubisco is the primary entry point for carbon into the biosphere. However, rubisco is widely regarded as inefficient leading many to question whether the enzyme can adapt to become a better catalyst. Through a phylogenetic investigation of the molecular and kinetic evolution of Form I rubisco we demonstrate that rubisco is not stagnant. Instead, we demonstrate rbcL is among the 1% of slowest evolving genes and enzymes on Earth, accumulating one nucleotide substitution every 0.9 million years and one amino acid mutation every 7.2 million years. Despite this, we demonstrate that rubisco catalysis is continuing to evolve toward improved CO2/O2 specificity, carboxylase turnover, and carboxylation efficiency. Consistent with this kinetic adaptation, we reveal that increased rubisco evolution leads to a concomitant improvement in leaf-level CO2 assimilation. Thus, rubisco is continually evolving toward improved catalytic efficiency and CO2 assimilation in plants.\nSingle-plant-omics reveals the cascade of transcriptional changes during the vegetative-to-reproductive transition\nAuthors: Redmond, E. J.; Ronald, J.; Davis, S. J.; Ezer, D.\nScore: 17.1, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557157\nPlants undergo rapid developmental transitions, as well as gradual developmental processes. Moreover, individual plants within a population will undergo the developmental transitions asynchronously, so it is difficult to assemble a time series to resolve the sequence of transcriptional changes that take place during these rapid transitions. Single-plant-omics has the potential to distinguish between transcriptional events that are associated with these binary and continuous processes. Furthermore, we can utilise single-plant-omics to exploit this developmental asynchrony to order individual plants by their developmental trajectory, revealing a detailed cascade of transcriptional events. Here, we utilise single-plant-transcriptomics to resolve the transcriptional events that coincide with the onset of bolting. We performed RNA-seq on the leaves of individual plants from a large population of wild type Arabidopsis thaliana replicated at one time point during the vegetative-to-reproductive transition. Even though more than half of transcripts were differentially expressed between bolted and unbolted plants, we were able to find a subset of regulators that were more closely associated with gradual developmental traits like leaf size and biomass. Using a novel pseudotime inference algorithm, we determined that some senescence-associated processes, such as the reduction in ribosome biogenesis, are evident in the transcriptome before a bolt is visible. These results show the potential of single-plant-omics to reveal the detailed sequence of events that occur during rapid developmental transitions. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=179 SRC=\"FIGDIR/small/557157v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (28K): org.highwire.dtl.DTLVardef@15fc366org.highwire.dtl.DTLVardef@159188org.highwire.dtl.DTLVardef@5b23borg.highwire.dtl.DTLVardef@1f873d0_HPS_FORMAT_FIGEXP M_FIG Graphical Abstract: Physiological changes around bolting can be categorised into: binary processes, which appear to have either occurred or not occurred at any given timepoint; or continuous processes, which can be observed quantitatively. For binary processes, expression of strongly correlated genes can appear to follow a step change dynamic over time. However, when considered over a shorter timescale, the dynamics appear much smoother. For continuous processes, the shorter timescale should also capture smooth changes in gene expression. C_FIG\nStacking of PRRs in potato to achieve enhanced resistance against Phytophthora infestans\nAuthors: Ascurra, Y. C. T.; Wouters, D.; Visser, R.; Nuernberger, T.; Vleeshouwers, V.\nScore: 15.7, Published: 2023-09-11 DOI: 10.1101/2023.09.07.556738\nPlants employ pattern recognition receptors (PRRs) to sense pathogen-associated molecular patterns (PAMPs) or apoplastic effectors at the plant cell surface, as well as nucleotide-binding domain leucine-rich-repeat-containing receptors (NLRs) to sense effectors inside the plant cell. Breeding for potato resistance to P. infestans has focused on the use of NLRs, however, these genes are typically quickly overcome since the matching avirulence genes evolve exceptionally quickly. Here, we stacked two PRRs, PERU and RLP23, that recognize the rather conserved Phytophthora PAMPs Pep-13/25 and nlp20, respectively, in the potato cultivar Atlantic, and evaluated their effect on P. infestans resistance. We found that PERU and RLP23 cooperate for the early immune responses like the accumulation of reactive oxygen species (ROS) and production of ethylene by recognizing their corresponding PAMPs. Furthermore, we show that potato plants overexpressing these two PRRs are slightly less affected by P. infestans compared to the single transformants. Together, our data suggest that pyramiding of surface receptors can provide additional enhanced resistance against pathogens, however, more effective or synergistic combinations that may include intracellular NLR receptors should be explored.\nControl of cell fate specification and patterning by an ancestral microRNA\nAuthors: Aguilar-Cruz, A.; Flores-Sandoval, E.; Gutierrez-Ramos, X.; Oltehua-Lopez, O.; Dorantes-Acosta, A. E.; Trujillo, J. T.; Kato, H.; Ishizaki, K.; Mosher, R. A.; Dolan, L.; Grimanelli, D.; Haseloff, J.; Bowman, J. L.; Arteaga-Vazquez, M. A.\nScore: 11.0, Published: 2023-09-11 DOI: 10.1101/2023.09.09.556951\nThe formation of an organized body requires the establishment and maintenance of cells with structural and functional distinctive characteristics. A central question in developmental biology is how changes in the regulation of genes drive cell specification and patterning1. microRNAs (miRNAs) are small non-coding RNAs that regulate development through mRNA cleavage and/or translational repression2. In plants, miRNAs regulate key aspects including growth, development, stem cell maintenance, vegetative phase change, leaf morphogenesis, floral organ formation and flowering time3. Biogenesis of plant miRNAs depends on the activity of DICER-LIKE 1 (DCL1), an RNase type III endonuclease that processes double stranded RNA to give rise to mature miRNAs 4. The genomes of todays flora contain at least one bona fide copy of DCL1 5,6. Using Marchantia polymorpha -a model bryophyte that allows comparative approaches to infer characteristics of the ancestral land plant-, we demonstrate that MpDCL1a is required for the biogenesis of miRNAs and uncovered a central role for miR166/Homeodomain Zipper Class III-regulated auxin synthesis in the specification of cell identity, patterning, meristem function, laminar expansion and the development of the body in the last common ancestor of extant land plants.\nA soybean rust effector protease suppresses host immunity and cleaves a 3-deoxy-7-phosphoheptulonate synthase\nAuthors: Chicowski, A. S.; Qi, M.; Variz, H.; Bredow, M.; Montes-Serey, C.; Caiazza, F.; Dong, H.; Margets, A. C.; Mejias, J.; Walley, J.; Craik, C.; Pedley, K. F.; Aung, K.; Innes, R. W.; Whitham, S.\nScore: 17.4, Published: 2023-09-08 DOI: 10.1101/2023.09.07.556260\nThe devastating soybean rust (SBR) pathogen, Phakopsora pachyrhizi, encodes many secreted proteins, but only two have been functionally characterized for their roles in rust virulence. Here, we demonstrate that transient expression of P. pachyrhizi effector candidate 15 (PpEC15), an aspartic protease, leads to enhanced bacterial growth in planta, suppression of callose deposition, reduced expression of plant defense-related marker genes and suppression of pathogen-associated molecular pattern (PAMP)-induced reactive oxygen species (ROS). Stable expression of PpEC15 in soybean suppresses PAMP-induced ROS production and enhances bacterial growth, indicating that, collectively, PpEC15 suppresses host and non-host innate immune responses. Yeast-two-hybrid and proximity labeling identified putative PpEC15 interacting partners including a peptide-chain release factor (PCRF), a NAC83 (NAM, ATAF, and CUC) transcription factor, and a DAHP (3-deoxy-7-phosphoheptulonate) synthase. We further show that PpEC15 can cleave DAHP but does not cleave PCRF or NAC83. Virus-induced gene silencing of NAC83, PCRF and DAHP altered PAMP-induced ROS production and salicylic acid production, indicating that these proteins may be involved in immune signaling. Collectively, our data show that PpEC15 is conserved across P. pachyrhizi isolates and other economically important rust species and is involved in the suppression of plant basal defense responses. Understanding the role of PpEC15 in P. pachyrhizi virulence will provide a foundation for designing targeted intervention strategies to generate rust-resistant crops.\nYerba mate (Ilex paraguariensis) genome provides new insights into convergent evolution of caffeine biosynthesis\nAuthors: Vignale, F. A.; Hernandez Garcia, A.; Modenutti, C. P.; Sosa, E. J.; Defelipe, L. A.; Oliveira, R. R. M.; Nunes, G. L.; Acevedo, R. M.; Burguener, G. F.; Rossi, M.; Zapata, P. D.; Marti, D. A.; Oliveira, G.; Smith, M. N.; Dubs, N. M.; Nair, S.; Barkman, T. J.; Turjanski, A. G.\nScore: 8.8, Published: 2023-09-10 DOI: 10.1101/2023.09.08.556846\nYerba mate (Ilex paraguariensis) is an economically important crop marketed for the elaboration of mate, the third-most widely consumed caffeine-containing infusion worldwide. Here we report the first genome assembly of this species, which has a total length of 1.06 Gb and contains 53,390 protein-coding genes. Comparative analyses revealed that the large yerba mate genome size is partly due to a whole-genome duplication (Ip-) during the early evolutionary history of Ilex, in addition to the hexaploidization event ({gamma}) shared by core eudicots. Characterization of the genome allowed us to clone the genes encoding methyltransferase enzymes that catalyse multiple reactions required for caffeine production. To our surprise, this species has converged upon a different biochemical pathway compared to that of its relatives, coffee and tea. In order to gain insight into the structural basis for the convergent enzyme activities, we obtained a crystal structure for the terminal enzyme in the pathway that forms caffeine. The structure reveals that convergent solutions have evolved for substrate positioning because different amino acid residues facilitate a different substrate orientation such that efficient methylation occurs in the independently evolved enzymes in yerba mate and coffee. While our results show phylogenomic constraint limits the genes coopted for convergence of caffeine biosynthesis, the x-ray diffraction data suggests structural constraints are minimal for the convergent evolution of individual reactions.\nSoybean RIN4 represents a mechanistic link between plant immune and symbiotic signaling\nAuthors: Toth, K.; Kim, D.; Cho, S.-H.; Nguyen, C. T.; Hartanto, C.; Michno, J.-M.; Stec, A. O.; Stupar, R. M.; Stacey, G.\nScore: 7.0, Published: 2023-09-15 DOI: 10.1101/2023.09.12.557450\nThe legume-rhizobium symbiosis represents a unique and beneficial interaction between legumes and nitrogen-fixing soil bacteria, called rhizobia. The initiation and development of this symbiosis is complex and begins with recognition of key molecular signals, produced by the plant and its symbiont, which determine symbiotic compatibility. Current data suggest that the invading symbiont initially triggers plant immune responses that are subsequently suppressed. Hence, there is growing evidence that features of plant immunity may be relevant to symbiotic establishment. RIN4 is a key immune regulator in plants, regulating basal immunity and it is also targeted by pathogen effector proteins that either confer susceptibility or resistance, depending on the presence of the appropriate resistance protein. Surprisingly, we found that RIN4 was rapidly phosphorylated upon rhizobial inoculation of soybean root hairs. RNAi silencing and mutant studies indicate that RIN4 expression is essential for effective nodulation of soybean. RIN4 phosphorylation occurs within a fifteen amino acid motif, which is highly conserved within the Fabales (legumes) and Rosales orders, that comprise species capable of nitrogen-fixing endosymbiosis with rhizobia. RIN4 proteins mutated in this conserved phosphorylation site failed to support efficient soybean nodulation. Phosphorylation of this site is mediated by the symbiotic receptor-like kinase, SymRK, a well-studied member of the symbiotic signaling pathway. The data implicate RIN4 phosphorylation as a key mediator of rhizobial compatibility, interconnecting symbiotic and immune signaling pathways.\nThe HOMEODOMAIN-Like protein HDL mediates chromatin organization and rewires leaf epidermal patterning\nAuthors: Ali, A.; Kuan, C.; Hung, F.-Y.; Chen, T.-C.; Lee, H.-C.; Yang, S.-L.; Feng, Y.-R.; Wu, K.; Ho, C.-M. K.\nScore: 4.5, Published: 2023-09-15 DOI: 10.1101/2023.09.13.557516\nThe Arabidopsis leaf epidermis is comprised of trichomes, pavement cells, and stomata, originating from versatile precursor cells capable of dividing or differentiating to create unique epidermal patterns. The mechanism governing these transitions and the maintenance of cell heterogeneity remains unclear. In this study, we identified a novel homeodomain-like superfamily protein, HDL, localized in chromocenters and playing a role in chromatin organization. HDL interacts with histone deacetylase 6 (HDA6) and methyltransferases, indicating its function in modulating chromatin accessibility. In hdl mutants, we observed moderately increased chromatin accessibility in the promoter region of protein-coding genes, along with reduced stomatal density and elevated trichome numbers on leaf surfaces. Corresponding to these phenotypes, stomatal-related gene expression decreased, while a transcriptional reporter for GLABRA2, a trichome initiation gene, exhibited higher and more variable expression levels. These findings highlight how HDL-mediated chromatin organization influences epidermal cell fate by modulating gene expression and enhancing cell heterogeneity within the leaf epidermis.\n",
  "wordCount" : "2398",
  "inLanguage": "en",
  "datePublished": "2023-09-17T10:36:50Z",
  "dateModified": "2023-09-17T10:36:50Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta"><span>updated on September 17, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.556959">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.556959" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.556959">
        <p class="paperTitle">Cellular gibberellin dynamics govern indeterminate nodule development, morphologyand function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.556959" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.556959" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Drapek, C.; Radzman-Mohd, N. A.; Rizza, A.; Schiessl, K.; Dos Santos Barbosa, F.; Wen, J.; Oldroyd, G. E. D.; Jones, A. M.</p>
        <p class="info">Score: 34.0, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.556959' target='https://doi.org/10.1101/2023.09.09.556959'> 10.1101/2023.09.09.556959</a></p>
        <p class="abstract">During nutrient scarcity, plants can adapt their developmental strategy to maximize their chance of survival. Such plasticity in development is underpinned by hormonal regulation, which mediates the relationship between environmental cues and developmental outputs. In legumes, endosymbiosis with nitrogen fixing bacteria (rhizobia) is a key adaptation for supplying the plant with nitrogen in the form of ammonium. Rhizobia are housed in lateral root-derived organs termed nodules that maintain an environment conducive to Nitrogenase in these bacteria. Several phytohormones are important for regulating the formation of nodules, with both positive and negative roles proposed for gibberellin (GA). In this study, we determined the cellular location and function of bioactive GA during nodule organogenesis using a genetically-encoded second generation GA biosensor, GPS2. We found endogenous bioactive GA accumulates locally at the site of nodule primordia, increasing dramatically in the cortical cell layers, persisting through cell divisions and maintaining accumulation in the mature nodule meristem. We show, through mis-expression of GA catabolic enzymes that suppress GA accumulation, that GA acts as a positive regulator of nodule growth and development. Furthermore, increasing or decreasing GA through perturbation of biosynthesis gene expression can increase or decrease the size of nodules, respectively. This is unique from lateral root formation, a developmental program that shares common organogenesis regulators. We link GA to a wider gene regulatory program by showing that cytokinin as well as nodule-identity genes induce and sustain GA accumulation necessary for nodule function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556499">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556499" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556499">
        <p class="paperTitle">Retrotransposon-driven environmental regulation of FLC leads to adaptive response to herbicide</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556499" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556499" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Raingeval, M.; Leduque, B.; Baduel, P.; Edera, A.; Roux, F.; Colot, V.; Quadrana, L.</p>
        <p class="info">Score: 35.8, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556499' target='https://doi.org/10.1101/2023.09.06.556499'> 10.1101/2023.09.06.556499</a></p>
        <p class="abstract">The mobilization of retrotransposons yields major-effect mutations. Here, we report an adaptive retrotransposon insertion within the first intron of the Arabidopsis floral-repressor locus FLOWERING LOCUS C (FLC). The insertion-mutation augments the environmental sensitivity of FLC by affecting the balance between coding and non-coding transcript isoforms in response to environmental threads. We show that this balance is modulated epigenetically by DNA methylation and orchestrated by IBM2, a factor involved in the processing of intronic heterochromatin. The stress-sensitive allele of FLC has recently spread across populations subjected to recurrent chemical weeding, and we demonstrate that retrotransposon-driven acceleration of life cycle represents a rapid response to herbicide. Our findings illustrate how retrotransposition can create environmentally-sensitive alleles that facilitate adaptation to anthropogenic disturbances of the environment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.07.06.498985">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.07.06.498985" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.07.06.498985">
        <p class="paperTitle">Rubisco is evolving for improved catalytic efficiency and CO2 assimilation in plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.07.06.498985" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.07.06.498985" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bouvier, J.; Emms, D.; Kelly, S.</p>
        <p class="info">Score: 35.4, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.07.06.498985' target='https://doi.org/10.1101/2022.07.06.498985'> 10.1101/2022.07.06.498985</a></p>
        <p class="abstract">Rubisco is the primary entry point for carbon into the biosphere. However, rubisco is widely regarded as inefficient leading many to question whether the enzyme can adapt to become a better catalyst. Through a phylogenetic investigation of the molecular and kinetic evolution of Form I rubisco we demonstrate that rubisco is not stagnant. Instead, we demonstrate rbcL is among the 1% of slowest evolving genes and enzymes on Earth, accumulating one nucleotide substitution every 0.9 million years and one amino acid mutation every 7.2 million years. Despite this, we demonstrate that rubisco catalysis is continuing to evolve toward improved CO2/O2 specificity, carboxylase turnover, and carboxylation efficiency. Consistent with this kinetic adaptation, we reveal that increased rubisco evolution leads to a concomitant improvement in leaf-level CO2 assimilation. Thus, rubisco is continually evolving toward improved catalytic efficiency and CO2 assimilation in plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557157">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557157" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557157">
        <p class="paperTitle">Single-plant-omics reveals the cascade of transcriptional changes during the vegetative-to-reproductive transition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557157" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557157" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Redmond, E. J.; Ronald, J.; Davis, S. J.; Ezer, D.</p>
        <p class="info">Score: 17.1, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557157' target='https://doi.org/10.1101/2023.09.11.557157'> 10.1101/2023.09.11.557157</a></p>
        <p class="abstract">Plants undergo rapid developmental transitions, as well as gradual developmental processes. Moreover, individual plants within a population will undergo the developmental transitions asynchronously, so it is difficult to assemble a time series to resolve the sequence of transcriptional changes that take place during these rapid transitions. Single-plant-omics has the potential to distinguish between transcriptional events that are associated with these binary and continuous processes. Furthermore, we can utilise single-plant-omics to exploit this developmental asynchrony to order individual plants by their developmental trajectory, revealing a detailed cascade of transcriptional events.

Here, we utilise single-plant-transcriptomics to resolve the transcriptional events that coincide with the onset of bolting. We performed RNA-seq on the leaves of individual plants from a large population of wild type Arabidopsis thaliana replicated at one time point during the vegetative-to-reproductive transition. Even though more than half of transcripts were differentially expressed between bolted and unbolted plants, we were able to find a subset of regulators that were more closely associated with gradual developmental traits like leaf size and biomass. Using a novel pseudotime inference algorithm, we determined that some senescence-associated processes, such as the reduction in ribosome biogenesis, are evident in the transcriptome before a bolt is visible.

These results show the potential of single-plant-omics to reveal the detailed sequence of events that occur during rapid developmental transitions.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=179 SRC=&#34;FIGDIR/small/557157v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (28K):
org.highwire.dtl.DTLVardef@15fc366org.highwire.dtl.DTLVardef@159188org.highwire.dtl.DTLVardef@5b23borg.highwire.dtl.DTLVardef@1f873d0_HPS_FORMAT_FIGEXP  M_FIG Graphical Abstract: Physiological changes around bolting can be categorised into: binary processes, which appear to have either occurred or not occurred at any given timepoint; or continuous processes, which can be observed quantitatively. For binary processes, expression of strongly correlated genes can appear to follow a step change dynamic over time. However, when considered over a shorter timescale, the dynamics appear much smoother. For continuous processes, the shorter timescale should also capture smooth changes in gene expression.

C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556738">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556738" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556738">
        <p class="paperTitle">Stacking of PRRs in potato to achieve enhanced resistance against Phytophthora infestans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556738" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556738" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ascurra, Y. C. T.; Wouters, D.; Visser, R.; Nuernberger, T.; Vleeshouwers, V.</p>
        <p class="info">Score: 15.7, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556738' target='https://doi.org/10.1101/2023.09.07.556738'> 10.1101/2023.09.07.556738</a></p>
        <p class="abstract">Plants employ pattern recognition receptors (PRRs) to sense pathogen-associated molecular patterns (PAMPs) or apoplastic effectors at the plant cell surface, as well as nucleotide-binding domain leucine-rich-repeat-containing receptors (NLRs) to sense effectors inside the plant cell. Breeding for potato resistance to P. infestans has focused on the use of NLRs, however, these genes are typically quickly overcome since the matching avirulence genes evolve exceptionally quickly. Here, we stacked two PRRs, PERU and RLP23, that recognize the rather conserved Phytophthora PAMPs Pep-13/25 and nlp20, respectively, in the potato cultivar Atlantic, and evaluated their effect on P. infestans resistance. We found that PERU and RLP23 cooperate for the early immune responses like the accumulation of reactive oxygen species (ROS) and production of ethylene by recognizing their corresponding PAMPs. Furthermore, we show that potato plants overexpressing these two PRRs are slightly less affected by P. infestans compared to the single transformants. Together, our data suggest that pyramiding of surface receptors can provide additional enhanced resistance against pathogens, however, more effective or synergistic combinations that may include intracellular NLR receptors should be explored.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.556951">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.556951" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.556951">
        <p class="paperTitle">Control of cell fate specification and patterning by an ancestral microRNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.556951" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.556951" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aguilar-Cruz, A.; Flores-Sandoval, E.; Gutierrez-Ramos, X.; Oltehua-Lopez, O.; Dorantes-Acosta, A. E.; Trujillo, J. T.; Kato, H.; Ishizaki, K.; Mosher, R. A.; Dolan, L.; Grimanelli, D.; Haseloff, J.; Bowman, J. L.; Arteaga-Vazquez, M. A.</p>
        <p class="info">Score: 11.0, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.556951' target='https://doi.org/10.1101/2023.09.09.556951'> 10.1101/2023.09.09.556951</a></p>
        <p class="abstract">The formation of an organized body requires the establishment and maintenance of cells with structural and functional distinctive characteristics. A central question in developmental biology is how changes in the regulation of genes drive cell specification and patterning1. microRNAs (miRNAs) are small non-coding RNAs that regulate development through mRNA cleavage and/or translational repression2. In plants, miRNAs regulate key aspects including growth, development, stem cell maintenance, vegetative phase change, leaf morphogenesis, floral organ formation and flowering time3. Biogenesis of plant miRNAs depends on the activity of DICER-LIKE 1 (DCL1), an RNase type III endonuclease that processes double stranded RNA to give rise to mature miRNAs 4. The genomes of todays flora contain at least one bona fide copy of DCL1 5,6. Using Marchantia polymorpha -a model bryophyte that allows comparative approaches to infer characteristics of the ancestral land plant-, we demonstrate that MpDCL1a is required for the biogenesis of miRNAs and uncovered a central role for miR166/Homeodomain Zipper Class III-regulated auxin synthesis in the specification of cell identity, patterning, meristem function, laminar expansion and the development of the body in the last common ancestor of extant land plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556260">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556260" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556260">
        <p class="paperTitle">A soybean rust effector protease suppresses host immunity and cleaves a 3-deoxy-7-phosphoheptulonate synthase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556260" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556260" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chicowski, A. S.; Qi, M.; Variz, H.; Bredow, M.; Montes-Serey, C.; Caiazza, F.; Dong, H.; Margets, A. C.; Mejias, J.; Walley, J.; Craik, C.; Pedley, K. F.; Aung, K.; Innes, R. W.; Whitham, S.</p>
        <p class="info">Score: 17.4, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556260' target='https://doi.org/10.1101/2023.09.07.556260'> 10.1101/2023.09.07.556260</a></p>
        <p class="abstract">The devastating soybean rust (SBR) pathogen, Phakopsora pachyrhizi, encodes many secreted proteins, but only two have been functionally characterized for their roles in rust virulence. Here, we demonstrate that transient expression of P. pachyrhizi effector candidate 15 (PpEC15), an aspartic protease, leads to enhanced bacterial growth in planta, suppression of callose deposition, reduced expression of plant defense-related marker genes and suppression of pathogen-associated molecular pattern (PAMP)-induced reactive oxygen species (ROS). Stable expression of PpEC15 in soybean suppresses PAMP-induced ROS production and enhances bacterial growth, indicating that, collectively, PpEC15 suppresses host and non-host innate immune responses. Yeast-two-hybrid and proximity labeling identified putative PpEC15 interacting partners including a peptide-chain release factor (PCRF), a NAC83 (NAM, ATAF, and CUC) transcription factor, and a DAHP (3-deoxy-7-phosphoheptulonate) synthase. We further show that PpEC15 can cleave DAHP but does not cleave PCRF or NAC83. Virus-induced gene silencing of NAC83, PCRF and DAHP altered PAMP-induced ROS production and salicylic acid production, indicating that these proteins may be involved in immune signaling. Collectively, our data show that PpEC15 is conserved across P. pachyrhizi isolates and other economically important rust species and is involved in the suppression of plant basal defense responses. Understanding the role of PpEC15 in P. pachyrhizi virulence will provide a foundation for designing targeted intervention strategies to generate rust-resistant crops.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556846">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556846" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556846">
        <p class="paperTitle">Yerba mate (Ilex paraguariensis) genome provides new insights into convergent evolution of caffeine biosynthesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556846" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556846" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vignale, F. A.; Hernandez Garcia, A.; Modenutti, C. P.; Sosa, E. J.; Defelipe, L. A.; Oliveira, R. R. M.; Nunes, G. L.; Acevedo, R. M.; Burguener, G. F.; Rossi, M.; Zapata, P. D.; Marti, D. A.; Oliveira, G.; Smith, M. N.; Dubs, N. M.; Nair, S.; Barkman, T. J.; Turjanski, A. G.</p>
        <p class="info">Score: 8.8, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556846' target='https://doi.org/10.1101/2023.09.08.556846'> 10.1101/2023.09.08.556846</a></p>
        <p class="abstract">Yerba mate (Ilex paraguariensis) is an economically important crop marketed for the elaboration of mate, the third-most widely consumed caffeine-containing infusion worldwide. Here we report the first genome assembly of this species, which has a total length of 1.06 Gb and contains 53,390 protein-coding genes. Comparative analyses revealed that the large yerba mate genome size is partly due to a whole-genome duplication (Ip-) during the early evolutionary history of Ilex, in addition to the hexaploidization event ({gamma}) shared by core eudicots. Characterization of the genome allowed us to clone the genes encoding methyltransferase enzymes that catalyse multiple reactions required for caffeine production. To our surprise, this species has converged upon a different biochemical pathway compared to that of its relatives, coffee and tea. In order to gain insight into the structural basis for the convergent enzyme activities, we obtained a crystal structure for the terminal enzyme in the pathway that forms caffeine. The structure reveals that convergent solutions have evolved for substrate positioning because different amino acid residues facilitate a different substrate orientation such that efficient methylation occurs in the independently evolved enzymes in yerba mate and coffee. While our results show phylogenomic constraint limits the genes coopted for convergence of caffeine biosynthesis, the x-ray diffraction data suggests structural constraints are minimal for the convergent evolution of individual reactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557450">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557450" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557450">
        <p class="paperTitle">Soybean RIN4 represents a mechanistic link between plant immune and symbiotic signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557450" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557450" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Toth, K.; Kim, D.; Cho, S.-H.; Nguyen, C. T.; Hartanto, C.; Michno, J.-M.; Stec, A. O.; Stupar, R. M.; Stacey, G.</p>
        <p class="info">Score: 7.0, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557450' target='https://doi.org/10.1101/2023.09.12.557450'> 10.1101/2023.09.12.557450</a></p>
        <p class="abstract">The legume-rhizobium symbiosis represents a unique and beneficial interaction between legumes and nitrogen-fixing soil bacteria, called rhizobia. The initiation and development of this symbiosis is complex and begins with recognition of key molecular signals, produced by the plant and its symbiont, which determine symbiotic compatibility. Current data suggest that the invading symbiont initially triggers plant immune responses that are subsequently suppressed. Hence, there is growing evidence that features of plant immunity may be relevant to symbiotic establishment. RIN4 is a key immune regulator in plants, regulating basal immunity and it is also targeted by pathogen effector proteins that either confer susceptibility or resistance, depending on the presence of the appropriate resistance protein. Surprisingly, we found that RIN4 was rapidly phosphorylated upon rhizobial inoculation of soybean root hairs. RNAi silencing and mutant studies indicate that RIN4 expression is essential for effective nodulation of soybean. RIN4 phosphorylation occurs within a fifteen amino acid motif, which is highly conserved within the Fabales (legumes) and Rosales orders, that comprise species capable of nitrogen-fixing endosymbiosis with rhizobia. RIN4 proteins mutated in this conserved phosphorylation site failed to support efficient soybean nodulation. Phosphorylation of this site is mediated by the symbiotic receptor-like kinase, SymRK, a well-studied member of the symbiotic signaling pathway. The data implicate RIN4 phosphorylation as a key mediator of rhizobial compatibility, interconnecting symbiotic and immune signaling pathways.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557516">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557516" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557516">
        <p class="paperTitle">The HOMEODOMAIN-Like protein HDL mediates chromatin organization and rewires leaf epidermal patterning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557516" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557516" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ali, A.; Kuan, C.; Hung, F.-Y.; Chen, T.-C.; Lee, H.-C.; Yang, S.-L.; Feng, Y.-R.; Wu, K.; Ho, C.-M. K.</p>
        <p class="info">Score: 4.5, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557516' target='https://doi.org/10.1101/2023.09.13.557516'> 10.1101/2023.09.13.557516</a></p>
        <p class="abstract">The Arabidopsis leaf epidermis is comprised of trichomes, pavement cells, and stomata, originating from versatile precursor cells capable of dividing or differentiating to create unique epidermal patterns. The mechanism governing these transitions and the maintenance of cell heterogeneity remains unclear. In this study, we identified a novel homeodomain-like superfamily protein, HDL, localized in chromocenters and playing a role in chromatin organization. HDL interacts with histone deacetylase 6 (HDA6) and methyltransferases, indicating its function in modulating chromatin accessibility. In hdl mutants, we observed moderately increased chromatin accessibility in the promoter region of protein-coding genes, along with reduced stomatal density and elevated trichome numbers on leaf surfaces. Corresponding to these phenotypes, stomatal-related gene expression decreased, while a transcriptional reporter for GLABRA2, a trichome initiation gene, exhibited higher and more variable expression levels. These findings highlight how HDL-mediated chromatin organization influences epidermal cell fate by modulating gene expression and enhancing cell heterogeneity within the leaf epidermis.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
