<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Chemical inhibition of stomatal differentiation by perturbation of the master-regulatory bHLH heterodimer via an ACT-Like domain
Authors: Nakagawa, A.; Sepuru, K. M.; Yip, S. J. A.; Seo, H.; Coffin, C. M.; Segawa, Y.; Iwasaki, R.; Kato, H.; Kim, S.; Aihara, Y.; Kinoshita, T.; Itami, K.; Han, S.-K.; Murakami, K.; Torii, K. U.
Score: 28.2, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565226
Selective perturbation of protein interactions with chemical compounds enables dissection and control of developmental processes.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="Chemical inhibition of stomatal differentiation by perturbation of the master-regulatory bHLH heterodimer via an ACT-Like domain
Authors: Nakagawa, A.; Sepuru, K. M.; Yip, S. J. A.; Seo, H.; Coffin, C. M.; Segawa, Y.; Iwasaki, R.; Kato, H.; Kim, S.; Aihara, Y.; Kinoshita, T.; Itami, K.; Han, S.-K.; Murakami, K.; Torii, K. U.
Score: 28.2, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565226
Selective perturbation of protein interactions with chemical compounds enables dissection and control of developmental processes." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-12T10:37:43+00:00" />
<meta property="article:modified_time" content="2023-11-12T10:37:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="Chemical inhibition of stomatal differentiation by perturbation of the master-regulatory bHLH heterodimer via an ACT-Like domain
Authors: Nakagawa, A.; Sepuru, K. M.; Yip, S. J. A.; Seo, H.; Coffin, C. M.; Segawa, Y.; Iwasaki, R.; Kato, H.; Kim, S.; Aihara, Y.; Kinoshita, T.; Itami, K.; Han, S.-K.; Murakami, K.; Torii, K. U.
Score: 28.2, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565226
Selective perturbation of protein interactions with chemical compounds enables dissection and control of developmental processes."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "Chemical inhibition of stomatal differentiation by perturbation of the master-regulatory bHLH heterodimer via an ACT-Like domain\nAuthors: Nakagawa, A.; Sepuru, K. M.; Yip, S. J. A.; Seo, H.; Coffin, C. M.; Segawa, Y.; Iwasaki, R.; Kato, H.; Kim, S.; Aihara, Y.; Kinoshita, T.; Itami, K.; Han, S.-K.; Murakami, K.; Torii, K. U.\nScore: 28.2, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565226\nSelective perturbation of protein interactions with chemical compounds enables dissection and control of developmental processes.",
  "keywords": [
    
  ],
  "articleBody": " Chemical inhibition of stomatal differentiation by perturbation of the master-regulatory bHLH heterodimer via an ACT-Like domain\nAuthors: Nakagawa, A.; Sepuru, K. M.; Yip, S. J. A.; Seo, H.; Coffin, C. M.; Segawa, Y.; Iwasaki, R.; Kato, H.; Kim, S.; Aihara, Y.; Kinoshita, T.; Itami, K.; Han, S.-K.; Murakami, K.; Torii, K. U.\nScore: 28.2, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565226\nSelective perturbation of protein interactions with chemical compounds enables dissection and control of developmental processes. Differentiation of stomata, cellular valves vital for plant growth and survival, is specified by the basic-helix-loop-helix (bHLH) heterodimers. Harnessing a new amination reaction, we here report a synthesis, target identification, and mode of action of an atypical doubly-sulfonylated imidazolone, Stomidazolone, which triggers stomatal stem cell arrest. Our forward chemical genetics followed by biophysical analyses elucidated that Stomidazolone directly binds to the C-terminal ACT-Like (ACTL) domain of MUTE, a master regulator of stomatal differentiation, and perturbs its heterodimerization with a partner bHLH, SCREAM. Guided by structural docking modeling, we rationally designed MUTE with reduced Stomidazolone binding. These engineered MUTE proteins are fully functional and confer Stomidazolone resistance in vivo. Our study identifies doubly-sulfonylated imidazolone as a direct inhibitor of the stomatal master regulator, further expanding the chemical space for perturbing bHLH-ACTL proteins to manipulate plant development.\nConservation of molecular responses upon viral infection in the non-vascular plant Marchantia polymorpha\nAuthors: Moner-Ross, E.; Jimenez-Gongora, T.; Villar-Martin, L.; Gonzalez-Miguel, V. M.; kutnjak, D.; Rubio-Somoza, I.\nScore: 24.1, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565823\nAfter their transition from water to land around 450 million years ago, plants colonized new habitats facing unprecedented pathogenic microbes. That expansion was mostly supported by a growing anatomical complexity based on the acquisition of developmental innovations, such as roots, stomata and vascular tissue. Despite several of those innovations became central for the interaction between plants and their associated microbes, little is known about their impact on plant immune programs and on the diversification of infection strategies of their pathogens. A paradigmatic case is the close relationship between plant vasculature and viruses. Vascular tissues provide a unique cellular environment for viral replication and existence, besides constituting a fast track for viral systemic spread throughout the plant. Since most of our knowledge about plant-virus interactions come from studies in vascular plants, we here present a comparative study to contribute to the understanding of the evolution of plant-virus interactions by molecularly characterizing the interplay between the bryophyte Marchantia polymorpha and viruses. Virome analysis of Marchantia plants shows that they are primarily associated with RNA viruses in natural settings. Additional molecular characterization of the interaction between Marchantia and tobacco mosaic virus (TMV) show conserved basic processes with vascular plants and divergent features. Viral infection triggers an extensive transcriptional reprogramming in Marchantia encompassing broad range defence responses, inhibition of cell cycle and photosynthesis and a sustained wound response that prevents further viral movement. Additionally, infected plants show premature aging and organ maturation. Notably, we found that some core responses that occur in infected areas in Marchantia, were described to be restricted to vascular tissues in Nicotiana, suggesting that evolutionary appearance of developmental innovations that became central in plants-virus interactions resulted in re-routing of defence responses. Finally, we uncover the conserved role of a transcription factor interacting with the TMV silencing suppressor p126 in specifically abrogating TMV infection.\nRepression of pattern-triggered immune responses by hypoxia\nAuthors: Mooney, B. C.; Doorly, C. M.; Mantz, M.; Garcia, P.; Huesgen, P. F.; Graciet, E.\nScore: 11.8, Published: 2023-11-07 DOI: 10.1101/2023.11.07.565979\nCombined abiotic/biotic stresses frequently occur in nature. Yet, relatively little is known about how plants regulate the crosstalk between stress response pathways to trigger a coordinated response to combined stresses. Protein degradation by the ubiquitin/proteasome system (UPS) is central to the regulation of plant responses to multiple stresses, including the crosstalk between pathways. The Arg/N-degron pathway, a subset of the UPS, targets proteins based on their N-termini. In plants, it has been implicated in the responses to both biotic and abiotic stresses, while being a key regulator of hypoxia response through the degradation of ERF-VII transcription factors, which orchestrate the onset of the hypoxia response program. Because of its central position in regulating multiple (a)biotic stresses, we hypothesized that the Arg/N-degron pathway could act in the crosstalk between abiotic/biotic stresses. In exploring this possibility using the model pathogen-associated molecular pattern flg22 to elicit pattern-triggered immunity (PTI), we uncovered a link between the transcriptional response programs to hypoxia and to flg22. Combined hypoxia/flg22 treatments further showed that hypoxia represses the flg22 transcriptional program, as well as the expression of pattern recognition receptors, MAPK signalling and callose deposition during PTI, through mechanisms that are at least partly dependent on the ERF-VIIs. These findings are of relevance to understanding the trade-offs between plant responses to combined abiotic/biotic stresses in the context of our efforts to increase crop resilience to global climate change. Our results also show that the well-known repressive effect of hypoxia on innate immunity in animals also applies to plants. Significance statementUnderstanding how plants regulate the crosstalk between stress response pathways is key to our efforts to increase crop resilience and mitigate yield losses caused by global climate change. Despite the urgency to do so, relatively little remains known about how plants respond to combined stresses, which frequently occur in nature. Here, we show that the hypoxia response program and the basal layer of plant immunity (pattern-triggered immunity or PTI) share components. Our data also show that hypoxia represses several key aspects of the plants innate immune response, a situation akin to that discovered in animals decades ago. These findings have implications for our ability to develop resilient crops by limiting the negative trade-offs that exist between hypoxia response and immunity.\nGibberellin Perception Sensors 1 and 2 reveal cellular GA dynamics articulated by COP1 and GA20ox1 that are necessary but not sufficient to pattern hypocotyl cell elongation\nAuthors: Griffiths, J.; Rizza, A.; Tang, B.; Frommer, W. B.; Jones, A. M.\nScore: 9.9, Published: 2023-11-08 DOI: 10.1101/2023.11.06.565859\nThe phytohormone gibberellin (GA) is critical for environmentally sensitive plant development including germination, skotomorphogenesis and flowering. The FRET biosensor GIBBERELLIN PERCEPTION SENSOR1, which permits single-cell GA measurements in vivo, was previously used to observe a GA gradient correlated with cell length in dark-grown but not light-grown hypocotyls. We sought to understand how light signalling integrates into cellular GA regulation. Here we show how the E3 ligase COP1 and transcription factor HY5 play central roles in directing cellular GA distribution in skoto- and photomorphogenic hypocotyls, respectively. We demonstrate that the expression pattern of biosynthetic enzyme GA20ox1 is the key determinant of the GA gradient in dark-grown hypocotyls and is a target of COP1 signalling. We engineered a second generation GPS2 biosensor with improved orthogonality and reversibility to show the cellular pattern of GA depletion during the transition to growth in the light. This GA depletion partly explains the resetting of hypocotyl growth dynamics during photomorphogenesis. Achieving cell-level resolution has revealed how GA distributions link environmental conditions with morphology and morphological plasticity and the GPS2 biosensor is an ideal tool for GA studies in further conditions, organs and plant species.\nJAGGER localization and function is dependent on GPI anchor addition\nAuthors: Figueiredo, R.; Costa, M.; Moreira, D.; Moreira, M.; Noble, J.; Pereira, L. G.; Melo, P.; Palanivelu, R.; Coimbra, S.; Pereira, A. M.\nScore: 8.2, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565777\nIn flowering plants, successful double fertilization requires the correct delivery of two sperm cells to the female gametophyte inside the ovule. The delivery of a single pair of sperm cells is achieved by the entrance of a single pollen tube into one female gametophyte. To prevent polyspermy, Arabidopsis ovules avoid the attraction of multiples pollen tubes to one ovule - polytubey block. In Arabidopsis jagger mutants, a significant number of ovules attract more than one pollen tube to an ovule due to an impairment in synergid degeneration. JAGGER encodes a putative arabinogalactan protein (AGP) which is predicted to be anchored to the plasma membrane by a glycosylphosphatidylinositol (GPI) anchor. Here, we show that JAGGER fused to citrine yellow fluorescent protein (JAGGER-cYFP) is functional and localizes mostly to the periphery of ovule integuments and transmitting tract cells. We further investigated the importance of GPI-anchor addition domains for JAGGER localization and function. Different JAGGER proteins with deletions in predicted {omega}-site regions and GAS (GPI attachment signal) domain, expected to compromise the addition of the GPI anchor, led to disruption of JAGGER localization in the cell periphery. All JAGGER proteins with disrupted localization were also not able to rescue the polytubey phenotype, pointing to the importance of GPI-anchor addition to in vivo function of the JAGGER protein.\nThe Arabidopsis Information Resource in 2024\nAuthors: Reiser, L.; Bakker, E.; Subramaniam, S.; Chen, X.; Sawant, S.; Khosa, K.; Prithvi, T.; Berardini, T. Z.\nScore: 8.1, Published: 2023-11-07 DOI: 10.1101/2023.11.06.565838\nSince 1999, The Arabidopsis Information Resource (www.arabidopsis.org) has been curating data about the Arabidopsis thaliana genome. Its primary focus is integrating experimental gene function information from the peer-reviewed literature and codifying it as controlled vocabulary annotations. Our goal is to produce a gold standard functional annotation set that reflects the current state of knowledge about the Arabidopsis genome. At the same time, the resource serves as a nexus for community-based collaborations aimed at improving data quality, access and reuse. For the past decade, our work has been made possible by subscriptions from our global user base. This update covers our ongoing biocuration work, some of our modernization efforts that contribute to the first major infrastructure overhaul since 2011, the introduction of JBrowse2, and the resources role in community activities such as organizing the structural reannotation of the genome. For gene function assessment, we used Gene Ontology annotations as a metric to evaluate: (1) what is currently known about Arabidopsis gene function, and (2) the set of unknown genes. Currently, 74% of the proteome has been annotated to at least one Gene Ontology term. Of those loci, half have experimental support for at least one of the following aspects: molecular function, biological process, or cellular component. Our work sheds light on the genes for which we have not yet identified any published experimental data and have no functional annotation. Drawing attention to these unknown genes highlights knowledge gaps and potential sources of novel discoveries. Article SummaryThe Arabidopsis Information Resource (TAIR, www.arabidopsis.org) is a comprehensive website about Arabidopsis thaliana, a small plant thats very easy to grow and analyze in the laboratory and is used to understand how many other plants function. We share our progress in data collection and organization, website and tool improvement, and our involvement in community projects.\nProtein migration and topography mapping reveals endogenous protein processing and homomers in Nicotiana benthamiana.\nAuthors: Zheng, K.; Lyu, J.; Thomas, E.; Schuster, M.; Sanguankiattichai, N.; Kaschani, F.; Kaiser, M.; van der Hoorn, R. A. L.\nScore: 6.3, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565301\nProcessing by proteases irreversibly regulates the fate of plant proteins, yet only few processing events have been described in plants. Here, we used in-gel digests and mass spectrometry to monitor the migration and topography of plant proteins within a protein gel. We detected 5,040 proteins from agroinfiltrated N. benthamiana leaves and found that 60% of these proteins have multiple proteoforms that migrate at different molecular weights. Transient expression of double-tagged candidate proteins confirmed several processing events in vivo. This analysis confirms the proteolytic removal and degradation of autoinhibitory prodomains of most but not all proteases, and revealed differential processing of three large hydrolase families. This analysis also uncovered intricate processing of glycosidases and putative ectodomain shedding of a diverse range of receptor-like kinases. Remarkably, we also discovered the likely homomerisation of \u003e160 plant proteins, causing the detection of multiple peptides at twice the MW of the majority signal. The PROTOMAP dataset contains a rich resource for the further study of proteoforms in plants.\nSingle nuclei sequencing reveals C4 photosynthesis is based on rewiring of ancestral cell identity networks\nAuthors: Swift, J.; Luginbuehl, L. H.; Schreier, T.; Donald, R. M.; Lee, T.; Nery, J.; Ecker, J. R.; Hibberd, J. M.\nScore: 61.0, Published: 2023-10-27 DOI: 10.1101/2023.10.26.562893\nIn multicellular systems changes to the patterning of gene expression drive modifications in cell function and trait evolution. One striking example is found in more than sixty plant lineages where compartmentation of photosynthesis between cell types allowed the evolution of the efficient C4 pathway from the ancestral C3 state. The molecular events enabling this transition are unclear. We used single nuclei sequencing to generate atlases for C3 rice and C4 sorghum during photomorphogenesis. Our analysis revealed that initiation of photosynthesis gene expression is conditioned by cell identity. In both species a conserved cistrome defines each cell type, and photosynthesis genes switching expression from mesophyll in rice to bundle sheath in sorghum acquire hallmarks of bundle sheath identity. The sorghum bundle sheath has also acquired gene networks associated with C3 guard cells. We conclude C4 photosynthesis is based on rewiring in cis that exapts cell identity networks of C3 plants.\nA fully sequenced collection of homozygous EMS mutants for forward and reverse genetic screens in Arabidopsis thaliana\nAuthors: Carrere, S.; Routaboul, J.-M.; Savourat, P.; Bellenot, C.; Lopez, H.; Quiroz-Monnens, T.; Ricou, A.; Camilleri, C.; Laufs, P.; mercier, r.; Noel, L. D.\nScore: 59.9, Published: 2023-10-27 DOI: 10.1101/2023.10.26.564234\nGenetic screens are powerful tools for biological research and are one of the reasons for the success of the thale cress Arabidopsis thaliana as a model species. Here, we describe the whole-genome sequencing of 871 Arabidopsis lines from the Homozygous EMS Mutant (HEM) collection as a novel resource for forward and reverse genetics. With an average 576 high-confidence mutations per HEM line, over three independent mutations altering protein sequence are found on average per gene in the collection. Pilot reverse genetics experiments on reproductive, developmental and physiological traits confirmed the efficacy of the tool for identifying both null and knockdown alleles. The possibility of conducting subtle repeated phenotyping of HEM lines and the immediate availability of the mutations will empower forward genetic approaches. The sequence resource is searchable with the ATHEM web interface (https://lipm-browsers.toulouse.inra.fr/pub/ATHEM/), and the biological material is distributed by the Versailles Arabidopsis Stock Center.\nA fully sequenced collection of homozygous EMS mutants for forward and reverse genetic screens in Arabidopsis thaliana\nAuthors: Carrere, S.; Routaboul, J.-M.; Savourat, P.; Bellenot, C.; Lopez, H.; Quiroz-Monnens, T.; Ricou, A.; Camilleri, C.; Laufs, P.; mercier, r.; Noel, L. D.\nScore: 59.9, Published: 2023-10-27 DOI: 10.1101/2023.10.26.564234\nGenetic screens are powerful tools for biological research and are one of the reasons for the success of the thale cress Arabidopsis thaliana as a model species. Here, we describe the whole-genome sequencing of 871 Arabidopsis lines from the Homozygous EMS Mutant (HEM) collection as a novel resource for forward and reverse genetics. With an average 576 high-confidence mutations per HEM line, over three independent mutations altering protein sequence are found on average per gene in the collection. Pilot reverse genetics experiments on reproductive, developmental and physiological traits confirmed the efficacy of the tool for identifying both null and knockdown alleles. The possibility of conducting subtle repeated phenotyping of HEM lines and the immediate availability of the mutations will empower forward genetic approaches. The sequence resource is searchable with the ATHEM web interface (https://lipm-browsers.toulouse.inra.fr/pub/ATHEM/), and the biological material is distributed by the Versailles Arabidopsis Stock Center.\n",
  "wordCount" : "2539",
  "inLanguage": "en",
  "datePublished": "2023-11-12T10:37:43Z",
  "dateModified": "2023-11-12T10:37:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 12, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565226">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565226" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565226">
        <p class="paperTitle">Chemical inhibition of stomatal differentiation by perturbation of the master-regulatory bHLH heterodimer via an ACT-Like domain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565226" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565226" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nakagawa, A.; Sepuru, K. M.; Yip, S. J. A.; Seo, H.; Coffin, C. M.; Segawa, Y.; Iwasaki, R.; Kato, H.; Kim, S.; Aihara, Y.; Kinoshita, T.; Itami, K.; Han, S.-K.; Murakami, K.; Torii, K. U.</p>
        <p class="info">Score: 28.2, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565226' target='https://doi.org/10.1101/2023.11.02.565226'> 10.1101/2023.11.02.565226</a></p>
        <p class="abstract">Selective perturbation of protein interactions with chemical compounds enables dissection and control of developmental processes. Differentiation of stomata, cellular valves vital for plant growth and survival, is specified by the basic-helix-loop-helix (bHLH) heterodimers. Harnessing a new amination reaction, we here report a synthesis, target identification, and mode of action of an atypical doubly-sulfonylated imidazolone, Stomidazolone, which triggers stomatal stem cell arrest. Our forward chemical genetics followed by biophysical analyses elucidated that Stomidazolone directly binds to the C-terminal ACT-Like (ACTL) domain of MUTE, a master regulator of stomatal differentiation, and perturbs its heterodimerization with a partner bHLH, SCREAM. Guided by structural docking modeling, we rationally designed MUTE with reduced Stomidazolone binding. These engineered MUTE proteins are fully functional and confer Stomidazolone resistance in vivo. Our study identifies doubly-sulfonylated imidazolone as a direct inhibitor of the stomatal master regulator, further expanding the chemical space for perturbing bHLH-ACTL proteins to manipulate plant development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565823">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565823" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565823">
        <p class="paperTitle">Conservation of molecular responses upon viral infection in the non-vascular plant Marchantia polymorpha</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565823" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565823" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moner-Ross, E.; Jimenez-Gongora, T.; Villar-Martin, L.; Gonzalez-Miguel, V. M.; kutnjak, D.; Rubio-Somoza, I.</p>
        <p class="info">Score: 24.1, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565823' target='https://doi.org/10.1101/2023.11.06.565823'> 10.1101/2023.11.06.565823</a></p>
        <p class="abstract">After their transition from water to land around 450 million years ago, plants colonized new habitats facing unprecedented pathogenic microbes. That expansion was mostly supported by a growing anatomical complexity based on the acquisition of developmental innovations, such as roots, stomata and vascular tissue. Despite several of those innovations became central for the interaction between plants and their associated microbes, little is known about their impact on plant immune programs and on the diversification of infection strategies of their pathogens. A paradigmatic case is the close relationship between plant vasculature and viruses. Vascular tissues provide a unique cellular environment for viral replication and existence, besides constituting a fast track for viral systemic spread throughout the plant. Since most of our knowledge about plant-virus interactions come from studies in vascular plants, we here present a comparative study to contribute to the understanding of the evolution of plant-virus interactions by molecularly characterizing the interplay between the bryophyte Marchantia polymorpha and viruses. Virome analysis of Marchantia plants shows that they are primarily associated with RNA viruses in natural settings. Additional molecular characterization of the interaction between Marchantia and tobacco mosaic virus (TMV) show conserved basic processes with vascular plants and divergent features. Viral infection triggers an extensive transcriptional reprogramming in Marchantia encompassing broad range defence responses, inhibition of cell cycle and photosynthesis and a sustained wound response that prevents further viral movement. Additionally, infected plants show premature aging and organ maturation. Notably, we found that some core responses that occur in infected areas in Marchantia, were described to be restricted to vascular tissues in Nicotiana, suggesting that evolutionary appearance of developmental innovations that became central in plants-virus interactions resulted in re-routing of defence responses. Finally, we uncover the conserved role of a transcription factor interacting with the TMV silencing suppressor p126 in specifically abrogating TMV infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.565979">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.565979" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.565979">
        <p class="paperTitle">Repression of pattern-triggered immune responses by hypoxia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.565979" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.565979" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mooney, B. C.; Doorly, C. M.; Mantz, M.; Garcia, P.; Huesgen, P. F.; Graciet, E.</p>
        <p class="info">Score: 11.8, Published: 2023-11-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.565979' target='https://doi.org/10.1101/2023.11.07.565979'> 10.1101/2023.11.07.565979</a></p>
        <p class="abstract">Combined abiotic/biotic stresses frequently occur in nature. Yet, relatively little is known about how plants regulate the crosstalk between stress response pathways to trigger a coordinated response to combined stresses. Protein degradation by the ubiquitin/proteasome system (UPS) is central to the regulation of plant responses to multiple stresses, including the crosstalk between pathways. The Arg/N-degron pathway, a subset of the UPS, targets proteins based on their N-termini. In plants, it has been implicated in the responses to both biotic and abiotic stresses, while being a key regulator of hypoxia response through the degradation of ERF-VII transcription factors, which orchestrate the onset of the hypoxia response program. Because of its central position in regulating multiple (a)biotic stresses, we hypothesized that the Arg/N-degron pathway could act in the crosstalk between abiotic/biotic stresses. In exploring this possibility using the model pathogen-associated molecular pattern flg22 to elicit pattern-triggered immunity (PTI), we uncovered a link between the transcriptional response programs to hypoxia and to flg22. Combined hypoxia/flg22 treatments further showed that hypoxia represses the flg22 transcriptional program, as well as the expression of pattern recognition receptors, MAPK signalling and callose deposition during PTI, through mechanisms that are at least partly dependent on the ERF-VIIs. These findings are of relevance to understanding the trade-offs between plant responses to combined abiotic/biotic stresses in the context of our efforts to increase crop resilience to global climate change. Our results also show that the well-known repressive effect of hypoxia on innate immunity in animals also applies to plants.

Significance statementUnderstanding how plants regulate the crosstalk between stress response pathways is key to our efforts to increase crop resilience and mitigate yield losses caused by global climate change. Despite the urgency to do so, relatively little remains known about how plants respond to combined stresses, which frequently occur in nature. Here, we show that the hypoxia response program and the basal layer of plant immunity (pattern-triggered immunity or PTI) share components. Our data also show that hypoxia represses several key aspects of the plants innate immune response, a situation akin to that discovered in animals decades ago. These findings have implications for our ability to develop resilient crops by limiting the negative trade-offs that exist between hypoxia response and immunity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565859">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565859" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565859">
        <p class="paperTitle">Gibberellin Perception Sensors 1 and 2 reveal cellular GA dynamics articulated by COP1 and GA20ox1 that are necessary but not sufficient to pattern hypocotyl cell elongation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565859" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565859" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Griffiths, J.; Rizza, A.; Tang, B.; Frommer, W. B.; Jones, A. M.</p>
        <p class="info">Score: 9.9, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565859' target='https://doi.org/10.1101/2023.11.06.565859'> 10.1101/2023.11.06.565859</a></p>
        <p class="abstract">The phytohormone gibberellin (GA) is critical for environmentally sensitive plant development including germination, skotomorphogenesis and flowering. The FRET biosensor GIBBERELLIN PERCEPTION SENSOR1, which permits single-cell GA measurements in vivo, was previously used to observe a GA gradient correlated with cell length in dark-grown but not light-grown hypocotyls. We sought to understand how light signalling integrates into cellular GA regulation. Here we show how the E3 ligase COP1 and transcription factor HY5 play central roles in directing cellular GA distribution in skoto- and photomorphogenic hypocotyls, respectively. We demonstrate that the expression pattern of biosynthetic enzyme GA20ox1 is the key determinant of the GA gradient in dark-grown hypocotyls and is a target of COP1 signalling. We engineered a second generation GPS2 biosensor with improved orthogonality and reversibility to show the cellular pattern of GA depletion during the transition to growth in the light. This GA depletion partly explains the resetting of hypocotyl growth dynamics during photomorphogenesis. Achieving cell-level resolution has revealed how GA distributions link environmental conditions with morphology and morphological plasticity and the GPS2 biosensor is an ideal tool for GA studies in further conditions, organs and plant species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565777">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565777" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565777">
        <p class="paperTitle">JAGGER localization and function is dependent on GPI anchor addition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565777" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565777" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Figueiredo, R.; Costa, M.; Moreira, D.; Moreira, M.; Noble, J.; Pereira, L. G.; Melo, P.; Palanivelu, R.; Coimbra, S.; Pereira, A. M.</p>
        <p class="info">Score: 8.2, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565777' target='https://doi.org/10.1101/2023.11.06.565777'> 10.1101/2023.11.06.565777</a></p>
        <p class="abstract">In flowering plants, successful double fertilization requires the correct delivery of two sperm cells to the female gametophyte inside the ovule. The delivery of a single pair of sperm cells is achieved by the entrance of a single pollen tube into one female gametophyte. To prevent polyspermy, Arabidopsis ovules avoid the attraction of multiples pollen tubes to one ovule - polytubey block. In Arabidopsis jagger mutants, a significant number of ovules attract more than one pollen tube to an ovule due to an impairment in synergid degeneration. JAGGER encodes a putative arabinogalactan protein (AGP) which is predicted to be anchored to the plasma membrane by a glycosylphosphatidylinositol (GPI) anchor. Here, we show that JAGGER fused to citrine yellow fluorescent protein (JAGGER-cYFP) is functional and localizes mostly to the periphery of ovule integuments and transmitting tract cells. We further investigated the importance of GPI-anchor addition domains for JAGGER localization and function. Different JAGGER proteins with deletions in predicted {omega}-site regions and GAS (GPI attachment signal) domain, expected to compromise the addition of the GPI anchor, led to disruption of JAGGER localization in the cell periphery. All JAGGER proteins with disrupted localization were also not able to rescue the polytubey phenotype, pointing to the importance of GPI-anchor addition to in vivo function of the JAGGER protein.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565838">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565838" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565838">
        <p class="paperTitle">The Arabidopsis Information Resource in 2024</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565838" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565838" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reiser, L.; Bakker, E.; Subramaniam, S.; Chen, X.; Sawant, S.; Khosa, K.; Prithvi, T.; Berardini, T. Z.</p>
        <p class="info">Score: 8.1, Published: 2023-11-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565838' target='https://doi.org/10.1101/2023.11.06.565838'> 10.1101/2023.11.06.565838</a></p>
        <p class="abstract">Since 1999, The Arabidopsis Information Resource (www.arabidopsis.org) has been curating data about the Arabidopsis thaliana genome. Its primary focus is integrating experimental gene function information from the peer-reviewed literature and codifying it as controlled vocabulary annotations. Our goal is to produce a  gold standard functional annotation set that reflects the current state of knowledge about the Arabidopsis genome. At the same time, the resource serves as a nexus for community-based collaborations aimed at improving data quality, access and reuse. For the past decade, our work has been made possible by subscriptions from our global user base. This update covers our ongoing biocuration work, some of our modernization efforts that contribute to the first major infrastructure overhaul since 2011, the introduction of JBrowse2, and the resources role in community activities such as organizing the structural reannotation of the genome. For gene function assessment, we used Gene Ontology annotations as a metric to evaluate: (1) what is currently known about Arabidopsis gene function, and (2) the set of  unknown genes. Currently, 74% of the proteome has been annotated to at least one Gene Ontology term. Of those loci, half have experimental support for at least one of the following aspects: molecular function, biological process, or cellular component. Our work sheds light on the genes for which we have not yet identified any published experimental data and have no functional annotation. Drawing attention to these unknown genes highlights knowledge gaps and potential sources of novel discoveries.

Article SummaryThe Arabidopsis Information Resource (TAIR, www.arabidopsis.org) is a comprehensive website about Arabidopsis thaliana, a small plant thats very easy to grow and analyze in the laboratory and is used to understand how many other plants function. We share our progress in data collection and organization, website and tool improvement, and our involvement in community projects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565301">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565301" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565301">
        <p class="paperTitle">Protein migration and topography mapping reveals endogenous protein processing and homomers in Nicotiana benthamiana.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565301" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565301" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zheng, K.; Lyu, J.; Thomas, E.; Schuster, M.; Sanguankiattichai, N.; Kaschani, F.; Kaiser, M.; van der Hoorn, R. A. L.</p>
        <p class="info">Score: 6.3, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565301' target='https://doi.org/10.1101/2023.11.02.565301'> 10.1101/2023.11.02.565301</a></p>
        <p class="abstract">Processing by proteases irreversibly regulates the fate of plant proteins, yet only few processing events have been described in plants. Here, we used in-gel digests and mass spectrometry to monitor the migration and topography of plant proteins within a protein gel. We detected 5,040 proteins from agroinfiltrated N. benthamiana leaves and found that 60% of these proteins have multiple proteoforms that migrate at different molecular weights. Transient expression of double-tagged candidate proteins confirmed several processing events in vivo. This analysis confirms the proteolytic removal and degradation of autoinhibitory prodomains of most but not all proteases, and revealed differential processing of three large hydrolase families. This analysis also uncovered intricate processing of glycosidases and putative ectodomain shedding of a diverse range of receptor-like kinases. Remarkably, we also discovered the likely homomerisation of &gt;160 plant proteins, causing the detection of multiple peptides at twice the MW of the majority signal. The PROTOMAP dataset contains a rich resource for the further study of proteoforms in plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.562893">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.562893" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.562893">
        <p class="paperTitle">Single nuclei sequencing reveals C4 photosynthesis is based on rewiring of ancestral cell identity networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.562893" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.562893" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Swift, J.; Luginbuehl, L. H.; Schreier, T.; Donald, R. M.; Lee, T.; Nery, J.; Ecker, J. R.; Hibberd, J. M.</p>
        <p class="info">Score: 61.0, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.562893' target='https://doi.org/10.1101/2023.10.26.562893'> 10.1101/2023.10.26.562893</a></p>
        <p class="abstract">In multicellular systems changes to the patterning of gene expression drive modifications in cell function and trait evolution. One striking example is found in more than sixty plant lineages where compartmentation of photosynthesis between cell types allowed the evolution of the efficient C4 pathway from the ancestral C3 state. The molecular events enabling this transition are unclear. We used single nuclei sequencing to generate atlases for C3 rice and C4 sorghum during photomorphogenesis. Our analysis revealed that initiation of photosynthesis gene expression is conditioned by cell identity. In both species a conserved cistrome defines each cell type, and photosynthesis genes switching expression from mesophyll in rice to bundle sheath in sorghum acquire hallmarks of bundle sheath identity. The sorghum bundle sheath has also acquired gene networks associated with C3 guard cells. We conclude C4 photosynthesis is based on rewiring in cis that exapts cell identity networks of C3 plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564234">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564234" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564234">
        <p class="paperTitle">A fully sequenced collection of homozygous EMS mutants for forward and reverse genetic screens in Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564234" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564234" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carrere, S.; Routaboul, J.-M.; Savourat, P.; Bellenot, C.; Lopez, H.; Quiroz-Monnens, T.; Ricou, A.; Camilleri, C.; Laufs, P.; mercier, r.; Noel, L. D.</p>
        <p class="info">Score: 59.9, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564234' target='https://doi.org/10.1101/2023.10.26.564234'> 10.1101/2023.10.26.564234</a></p>
        <p class="abstract">Genetic screens are powerful tools for biological research and are one of the reasons for the success of the thale cress Arabidopsis thaliana as a model species. Here, we describe the whole-genome sequencing of 871 Arabidopsis lines from the Homozygous EMS Mutant (HEM) collection as a novel resource for forward and reverse genetics. With an average 576 high-confidence mutations per HEM line, over three independent mutations altering protein sequence are found on average per gene in the collection. Pilot reverse genetics experiments on reproductive, developmental and physiological traits confirmed the efficacy of the tool for identifying both null and knockdown alleles. The possibility of conducting subtle repeated phenotyping of HEM lines and the immediate availability of the mutations will empower forward genetic approaches. The sequence resource is searchable with the ATHEM web interface (https://lipm-browsers.toulouse.inra.fr/pub/ATHEM/), and the biological material is distributed by the Versailles Arabidopsis Stock Center.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564234">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564234" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564234">
        <p class="paperTitle">A fully sequenced collection of homozygous EMS mutants for forward and reverse genetic screens in Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564234" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564234" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carrere, S.; Routaboul, J.-M.; Savourat, P.; Bellenot, C.; Lopez, H.; Quiroz-Monnens, T.; Ricou, A.; Camilleri, C.; Laufs, P.; mercier, r.; Noel, L. D.</p>
        <p class="info">Score: 59.9, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564234' target='https://doi.org/10.1101/2023.10.26.564234'> 10.1101/2023.10.26.564234</a></p>
        <p class="abstract">Genetic screens are powerful tools for biological research and are one of the reasons for the success of the thale cress Arabidopsis thaliana as a model species. Here, we describe the whole-genome sequencing of 871 Arabidopsis lines from the Homozygous EMS Mutant (HEM) collection as a novel resource for forward and reverse genetics. With an average 576 high-confidence mutations per HEM line, over three independent mutations altering protein sequence are found on average per gene in the collection. Pilot reverse genetics experiments on reproductive, developmental and physiological traits confirmed the efficacy of the tool for identifying both null and knockdown alleles. The possibility of conducting subtle repeated phenotyping of HEM lines and the immediate availability of the mutations will empower forward genetic approaches. The sequence resource is searchable with the ATHEM web interface (https://lipm-browsers.toulouse.inra.fr/pub/ATHEM/), and the biological material is distributed by the Versailles Arabidopsis Stock Center.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
