<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Uncovering the Infection Strategy of Phyllachora maydis during Maize Colonization: A Comprehensive Analysis
Authors: Caldwell, D. L.; Da Silva, C. R.; McCoy, A. G.; Avila, H.; Bonkowski, J. C.; Chilvers, M.; Helm, M.; Telenko, D. E.; Iyer-Pascuzzi, A. S.
Score: 21.2, Published: 2023-08-27 DOI: 10.1101/2023.08.26.554799
Tar spot, a disease caused by the ascomycete fungal pathogen Phyllachora maydis, is considered one of the most significant yield-limiting diseases of maize (Zea mays L.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="Uncovering the Infection Strategy of Phyllachora maydis during Maize Colonization: A Comprehensive Analysis
Authors: Caldwell, D. L.; Da Silva, C. R.; McCoy, A. G.; Avila, H.; Bonkowski, J. C.; Chilvers, M.; Helm, M.; Telenko, D. E.; Iyer-Pascuzzi, A. S.
Score: 21.2, Published: 2023-08-27 DOI: 10.1101/2023.08.26.554799
Tar spot, a disease caused by the ascomycete fungal pathogen Phyllachora maydis, is considered one of the most significant yield-limiting diseases of maize (Zea mays L." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-30T10:38:23+00:00" />
<meta property="article:modified_time" content="2023-08-30T10:38:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="Uncovering the Infection Strategy of Phyllachora maydis during Maize Colonization: A Comprehensive Analysis
Authors: Caldwell, D. L.; Da Silva, C. R.; McCoy, A. G.; Avila, H.; Bonkowski, J. C.; Chilvers, M.; Helm, M.; Telenko, D. E.; Iyer-Pascuzzi, A. S.
Score: 21.2, Published: 2023-08-27 DOI: 10.1101/2023.08.26.554799
Tar spot, a disease caused by the ascomycete fungal pathogen Phyllachora maydis, is considered one of the most significant yield-limiting diseases of maize (Zea mays L."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "Uncovering the Infection Strategy of Phyllachora maydis during Maize Colonization: A Comprehensive Analysis\nAuthors: Caldwell, D. L.; Da Silva, C. R.; McCoy, A. G.; Avila, H.; Bonkowski, J. C.; Chilvers, M.; Helm, M.; Telenko, D. E.; Iyer-Pascuzzi, A. S.\nScore: 21.2, Published: 2023-08-27 DOI: 10.1101/2023.08.26.554799\nTar spot, a disease caused by the ascomycete fungal pathogen Phyllachora maydis, is considered one of the most significant yield-limiting diseases of maize (Zea mays L.",
  "keywords": [
    
  ],
  "articleBody": " Uncovering the Infection Strategy of Phyllachora maydis during Maize Colonization: A Comprehensive Analysis\nAuthors: Caldwell, D. L.; Da Silva, C. R.; McCoy, A. G.; Avila, H.; Bonkowski, J. C.; Chilvers, M.; Helm, M.; Telenko, D. E.; Iyer-Pascuzzi, A. S.\nScore: 21.2, Published: 2023-08-27 DOI: 10.1101/2023.08.26.554799\nTar spot, a disease caused by the ascomycete fungal pathogen Phyllachora maydis, is considered one of the most significant yield-limiting diseases of maize (Zea mays L.) within the United States. P. maydis may also be found in association with other fungi, forming a disease complex with characteristic fish eye lesions. Understanding how P. maydis colonizes maize leaf cells is essential for developing effective disease control strategies. Here, we used histological approaches to elucidate how P. maydis infects and multiplies within susceptible maize leaves. We collected tar spot-infected maize leaf samples from four different fields in northern Indiana at three different time points during the growing season. Samples were chemically fixed and paraffin-embedded for high-resolution light and scanning electron microscopy. We observed a consistent pattern of disease progression in independent leaf samples collected across different geographical regions. Each stromata contained a central pycnidium that produced asexual spores. Perithecia with sexual spores developed in the stomatal chambers adjacent to the pycnidia, and a cap of spores formed over the stromata. P. maydis reproductive structures formed around but not within the vasculature. In our samples containing fish eye lesions, P. maydis is associated with two additional fungi, one of which is likely a member of the Paraphaeospheria genus; the other is an unknown fungi. Our data provide fundamental insights into how this pathogen colonizes and spreads within maize leaves. This knowledge can inform new approaches to managing tar spot, which could help mitigate the significant economic losses caused by this disease.\nWater fluxes contribute to growth patterning in shoot meristems\nAuthors: Alonso-Serra, J.; Cheddadi, I.; Kiss, A.; Cerutti, G.; Lionnet, C.; Godin, C.; Hamant, O.\nScore: 18.6, Published: 2023-08-29 DOI: 10.1101/2023.08.27.554993\nIn multicellular organisms, localized tissue outgrowth creates a new water sink thereby modifying hydraulic patterns at the organ level. These fluxes are often considered passive by-products of development and their patterning and potential contribution to morphogenesis remains largely unexplored. Here, we generated a complete map of cell volumetric growth and deformation across the shoot apex in Arabidopsis thaliana. Within the organ-meristem boundary, we found that a subpopulation of cells next to fast-growing cells experiences volumetric shrinkage. To understand this process, we used a vertex-based model integrating mechanics and hydraulics, informed by the measured growth rates. Organ outgrowth simulations revealed the emerging water fluxes and predicted water deficit with volume loss for a few cells at the boundary. Consistently, in planta, a water-soluble dye is preferentially allocated to fast-growing tissues and fails to enter the boundary domain. Analysis of intact meristems further validated our model by revealing cell shrinkage next to fast-growing cells in different contexts of tissue surface curvature and cell deformation. A molecular signature of water deficit at the boundary further confirmed our conclusion. Taken together, we propose a model where the differential sink strength of emerging organs prescribes the hydraulic patterns that define the boundary domain at the shoot apex.\nThe phosphorylation landscape of infection-related development by the rice blast fungus\nAuthors: Cruz-Mireles, N.; Oses-Ruiz, M.; Derbyshire, P.; Jegousse, C.; Ryder, L. S.; Bautista, M.-J.; Eseola, A. B.; Tang, B.; Yan, X.; Ma, W.; Findlay, K. C.; Were, V.; MacLean, D.; Talbot, N. J.; Menke, F. L. H.\nScore: 68.6, Published: 2023-08-20 DOI: 10.1101/2023.08.19.553964\nMany of the worlds most devastating crop diseases are caused by fungal pathogens which elaborate specialized infection structures to invade plant tissue. Here we present a quantitative mass spectrometry-based phosphoproteomic analysis of infection-related development by the rice blast fungus Magnaporthe oryzae, which threatens global food security. We mapped 8,005 phosphosites on 2,062 fungal proteins, revealing major re-wiring of phosphorylation-based signaling cascades during fungal infection. Comparing phosphosite conservation across 41 fungal species reveals phosphorylation signatures specifically associated with biotrophic and hemibiotrophic fungal infection. We then used parallel reaction monitoring to identify phosphoproteins directly regulated by the Pmk1 MAP kinase that controls plant infection by M. oryzae. We define 33 substrates of Pmk1 and show that Pmk1-dependent phosphorylation of a newly identified regulator, Vts1, is required for rice blast disease. Defining the phosphorylation landscape of infection therefore identifies potential therapeutic interventions for control of plant diseases.\nGreen Index: a widely accessible method to quantify greenness of photosynthetic organisms\nAuthors: Signorelli, S.; Casaretto, E.; Millar, A. H.\nScore: 11.1, Published: 2023-08-24 DOI: 10.1101/2023.08.23.554481\nPlant phenotyping involves the quantitative determination of complex plant traits using image analysis. One important parameter is how green plant tissues appear to the observer, which is indicative of their health and developmental stage. Various formulas have been developed to quantify this by calculating leaf greenness scores. We have developed a revised formula called \"Green Index\" (GI) devised out of the need to quantitatively assess how the apparent greenness of seedlings changes during de-etiolation. The GI calculation is simple, uses widely available RGB values of pixels in images as input, and does not require commercial software platforms or advanced computational skills. In this study we describe the conception of the GI formula, compare it with other widely used greenness formulas, and test its wider application in plant phenotyping using the open source free software platform RawTherapee. We demonstrate the utility of the GI in addressing common issues encountered in assessing plant biology experiments, underscoring its potential as a reliable and accessible tool. Finally, we explore the correlation between GI and chlorophyll content, assess its reliance on different types of photography, and summarizes the key steps for its effective utilization.\nSpatiotemporally distinct responses to mechanical forces shape the developing seed of Arabidopsis\nAuthors: Bauer, A.; Bied, C.; Delattre, A.; Ingram, G.; Golz, J. F.; Landrein, B.\nScore: 18.4, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554152\nOrgan morphogenesis depends on mechanical interactions between cells and tissues. These interactions generate forces that can be sensed by cells and affect key cellular processes. However, how mechanical forces contribute, together with biochemical signals, to the shaping of complex organs is still unclear. We address this question using the seed of Arabidopsis as a model system. We show that seeds first experience a phase of high anisotropic growth that is dependent on the response of cortical microtubule (CMT) to forces, which guide cellulose deposition according to shape-driven stresses in the outermost layer of the seed coat. However, at later stages of development, we show that seed growth is isotropic and depend on the properties of an inner layer of the seed coat that stiffens its walls in response to tension but has isotropic material properties. Finally, we show that the transition from anisotropic to isotropic growth is due to dampening of CMT responses to shape-driven stresses. Altogether, our work support that spatiotemporally distinct mechanical responses control the shape of developing seeds in Arabidopsis.\nA pectin-binding peptide with a structural and signaling role in the assembly of the plant cell wall\nAuthors: Schoenaers, S.; Lee, H. K.; Gonneau, M.; Faucher, E.; Levasseur, T.; Akary, E.; Claeijs, N.; Moussu, S.; Broyart, C.; Balcerowicz, D.; AbdElgawad, H.; Bassi, A.; Santa Cruz Damineli, D.; Costa, A.; Feijo, J. A.; Moreau, C.; Bonnin, E.; Cathala, B.; Santiago, J.; Hofte, H.; Vissenberg, K.\nScore: 9.6, Published: 2023-08-23 DOI: 10.1101/2023.08.23.554416\nPressurized cells with strong walls make up the hydrostatic skeleton of plants. Assembly and expansion of such stressed walls depend on a family of secreted RAPID ALKALINIZATION FACTOR (RALF) peptides which, curiously, bind both a membrane receptor complex and wall-localized LEUCINE-RICH REPEAT EXTENSINs (LRXs) in a mutually exclusive way. Here we show that, in root hairs, the RALF22 peptide has a dual structural and signaling role in cell expansion. Together with LRX1, it directs the compaction of charged pectin polymers at the root hair tip into periodic circumferential rings. Free RALF22 induces the formation of a complex with LORELEI-LIKE-GPI-ANCHORED PROTEIN 1 (LLG1) and FERONIA (FER), triggering adaptive cellular responses. These findings show how a peptide simultaneously functions as a structural component organizing cell wall architecture and as a signaling molecule that regulates this process. This mechanism may also underlie wall assembly and expansion in other plant cell types.\nChitosan stimulates root hair callose deposition and inhibits root hair growth\nAuthors: Drs, M.; Haluska, S.; Skrabalkova, E.; Krupar, P.; Potocka, A.; Brejskova, L.; Muller, K.; Serrano, N.; Voxeur, A.; Vernhettes, S.; Ortmannova, J.; Caldarescu, G.; Fendrych, M.; Potocky, M.; Zarsky, V.; Pecenkova, T.\nScore: 16.7, Published: 2023-08-21 DOI: 10.1101/2023.07.30.551171\nAlthough angiosperm plants have a general capacity to react after the immunity elicitor chitin or chitosan treatment by the cell wall callose deposition, this response in particular cell types and its evolutionary conservation is not understood. Here we show that also the growing root hairs (RHs) of Arabidopsis can respond to a mild (0.001%) chitosan treatment by the callose deposition and by a deceleration of the RH growth. We demonstrate that the glucan synthase-like 5 (GSL5)/PMR4 is vital for chitosan-induced callose deposition but not for RH growth inhibition. Upon the higher chitosan concentration (0.01%) treatment, RHs do not deposit callose, while growth inhibition is prominent. To understand the specificities of the low and high concentration chitosan treatments, we analysed the corresponding PTI signalling components, gene expression, and RH cellular endomembrane and cytoskeleton modifications. Importantly, chitosan-induced callose deposition is also present in the functionally analogous and evolutionarily only distantly related RH-like structures rhizophores (lycophytes) and rhizoids (bryophytes). Our results point to the RH callose deposition as a conserved strategy of soil-anchoring plant cells (rhizoids/rhizophores/RHs) to deal with mild biotic stress. At the same time, high chitosan concentration prominently disturbs intracellular dynamics, tip-localised endomembrane compartments and RH growth, precluding callose deposition.\nA low-cost and open-source imaging platform reveals spatiotemporal insight into Arabidopsis leaf elongation and movement\nAuthors: Oskam, L.; Snoek, B. L.; Pantazopoulou, C. K.; van Veen, H.; Matton, S. E. A.; Dijkhuizen, R. F.; Pierik, R.\nScore: 9.0, Published: 2023-08-27 DOI: 10.1101/2023.08.25.554752\nPlant organs move throughout the diurnal cycle, changing leaf and petiole positions to balance light capture, leaf temperature and water loss under dynamic environmental conditions. Upward movement of the petiole, called hyponasty, is one of several traits of the shade avoidance syndrome (SAS). SAS traits are elicited upon perception of vegetation shade signals such as far-red light (FR) and improve light capture in dense vegetation. Monitoring plant movement at a high temporal resolution allows studying functionality, as well as molecular regulation of hyponasty. However, high temporal resolution imaging solutions are often very expensive, making this unavailable to many researchers. Here, we present a modular and low-cost imaging set-up, based on small Raspberry Pi computers, that can track leaf movements and elongation growth with high temporal resolution. We also developed an open-source, semi-automated image analysis pipeline. Using this setup we followed responses to FR enrichment, light intensity and their interactions. Tracking both elongation and angle of petiole, lamina and entire leaf revealed insight into R:FR sensitivities of leaf growth and movement dynamics, and its interactions with background light intensity. We also identified spatial separation in hyponastic response regulation for the petiole and the lamina of the leaf, depending on the light conditions.\nIsoform-resolved genome annotation enables mapping of tissue-specific betalain regulation and anthocyanin regulator co-option in amaranth\nAuthors: Winkler, T. S.; Vollmer, S. K.; Stetter, M. G.\nScore: 9.0, Published: 2023-08-25 DOI: 10.1101/2023.08.24.554588\nO_LIBetalains are coloring pigments produced only in some families of the order Caryophyllales, where they have functionally replaced anthocyanins as coloring pigments. While the betalain pathway itself is known, the tissue-specific regulation of the pathway remains mostly unclear. C_LIO_LIWe first improve the high-quality Amaranthus hypochondriacus reference genome and create a highly complete genome annotation by combining ab initio prediction with long-read isoform sequencing data. We accurately annotate the betalain and anthocyanin pathway genes and their regulators in the genome of the betalain producing amaranth and further map the genetic basis of tissue color and tissue-specific regulation of the betalain pathway through a previously unknown MYB transcription factor, AhMYB2. C_LIO_LIDetailed haplotype analysis enabled us to identify two functional mutations that lead to a knock-out of the betalain pathway in natural accessions of amaranth. In addition, the down-regulation of AhMYB2 in the flower leads to reduced expression of key betalain enzyme genes and loss of red flower color. C_LIO_LIOur improved amaranth reference genome represents the most complete genome of amaranth to date and is a valuable resource for betalain and amaranth research. The high similarity of the betalain flower regulator AhMYB2 to anthocyanin regulators and a partially conserved interaction motif support the co-option of anthocyanin regulators for the betalain pathway as possible reason for the mutually exclusive nature of the two pigments. C_LI\nThe HSF-DREB-MYB regulatory module regulates flavonol biosynthesis and flavonoid B-ring hydroxylation in banana\nAuthors: Naik, J.; Rajput, R.; Stracke, R.; Pandey, A.\nScore: 6.7, Published: 2023-08-24 DOI: 10.1101/2023.08.23.554507\nPlant flavonols act primarily as ultraviolet radiation absorbers, reactive oxygen species scavengers, and phytoalexins, and they contribute to biotic and abiotic stress tolerance in plants. Banana (Musa acuminata), an herbaceous monocot and important fruit crop, accumulates flavonol derivatives in different organs, including the edible fruit pulp. Although flavonol content varies greatly in different organs, the molecular mechanisms involving transcriptional regulation of flavonol synthesis in banana are not known. Here, we characterized three SG7-R2R3 MYB transcription factors MaMYBFA1, MaMYBFA2, and MaMYBFA3) and their upstream regulators, heat shock transcription factor (MaHSF11) and dehydration responsive element binding factor (MaDREB1), to elucidate the molecular mechanism involved in transcriptional regulation of flavonol biosynthesis in banana. MaMYBFA positively regulate flavonol synthase 2 (MaFLS2) and downregulates MaFLS1. We show these transcription factors to be weak regulators of flavonol synthesis. Overexpression of MaHSF11 enhances flavonol contents, particularly that of myricetin, and promotes flavonol B-ring hydroxylation, which contributes to the diversity of flavonol derivatives. MaHSF11 directly interacts with the MaFLS1 and flavonoid 3', 5'-hydroxylase1 (MaF3'5'H1) promoters, both in vitro and in vivo. MaHSF11 activates the expression of MaDREB1 directly, which in turn regulates the expression of MaMYBFA3. Overall, our study elucidates a key regulatory mechanism for flavonol synthesis in banana and suggests possible targets for genetic optimization to enhance nutritional value and stress responses in this globally important fruit crop.\n",
  "wordCount" : "2354",
  "inLanguage": "en",
  "datePublished": "2023-08-30T10:38:23Z",
  "dateModified": "2023-08-30T10:38:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta"><span>updated on August 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.26.554799">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.26.554799" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.26.554799">
        <p class="paperTitle">Uncovering the Infection Strategy of Phyllachora maydis during Maize Colonization: A Comprehensive Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.26.554799" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.26.554799" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Caldwell, D. L.; Da Silva, C. R.; McCoy, A. G.; Avila, H.; Bonkowski, J. C.; Chilvers, M.; Helm, M.; Telenko, D. E.; Iyer-Pascuzzi, A. S.</p>
        <p class="info">Score: 21.2, Published: 2023-08-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.26.554799' target='https://doi.org/10.1101/2023.08.26.554799'> 10.1101/2023.08.26.554799</a></p>
        <p class="abstract">Tar spot, a disease caused by the ascomycete fungal pathogen Phyllachora maydis, is considered one of the most significant yield-limiting diseases of maize (Zea mays L.) within the United States. P. maydis may also be found in association with other fungi, forming a disease complex with characteristic fish eye lesions. Understanding how P. maydis colonizes maize leaf cells is essential for developing effective disease control strategies. Here, we used histological approaches to elucidate how P. maydis infects and multiplies within susceptible maize leaves. We collected tar spot-infected maize leaf samples from four different fields in northern Indiana at three different time points during the growing season. Samples were chemically fixed and paraffin-embedded for high-resolution light and scanning electron microscopy. We observed a consistent pattern of disease progression in independent leaf samples collected across different geographical regions. Each stromata contained a central pycnidium that produced asexual spores. Perithecia with sexual spores developed in the stomatal chambers adjacent to the pycnidia, and a cap of spores formed over the stromata. P. maydis reproductive structures formed around but not within the vasculature. In our samples containing fish eye lesions, P. maydis is associated with two additional fungi, one of which is likely a member of the Paraphaeospheria genus; the other is an unknown fungi. Our data provide fundamental insights into how this pathogen colonizes and spreads within maize leaves. This knowledge can inform new approaches to managing tar spot, which could help mitigate the significant economic losses caused by this disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.27.554993">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.27.554993" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.27.554993">
        <p class="paperTitle">Water fluxes contribute to growth patterning in shoot meristems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.27.554993" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.27.554993" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alonso-Serra, J.; Cheddadi, I.; Kiss, A.; Cerutti, G.; Lionnet, C.; Godin, C.; Hamant, O.</p>
        <p class="info">Score: 18.6, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.27.554993' target='https://doi.org/10.1101/2023.08.27.554993'> 10.1101/2023.08.27.554993</a></p>
        <p class="abstract">In multicellular organisms, localized tissue outgrowth creates a new water sink thereby modifying hydraulic patterns at the organ level. These fluxes are often considered passive by-products of development and their patterning and potential contribution to morphogenesis remains largely unexplored. Here, we generated a complete map of cell volumetric growth and deformation across the shoot apex in Arabidopsis thaliana. Within the organ-meristem boundary, we found that a subpopulation of cells next to fast-growing cells experiences volumetric shrinkage. To understand this process, we used a vertex-based model integrating mechanics and hydraulics, informed by the measured growth rates. Organ outgrowth simulations revealed the emerging water fluxes and predicted water deficit with volume loss for a few cells at the boundary. Consistently, in planta, a water-soluble dye is preferentially allocated to fast-growing tissues and fails to enter the boundary domain. Analysis of intact meristems further validated our model by revealing cell shrinkage next to fast-growing cells in different contexts of tissue surface curvature and cell deformation. A molecular signature of water deficit at the boundary further confirmed our conclusion. Taken together, we propose a model where the differential sink strength of emerging organs prescribes the hydraulic patterns that define the boundary domain at the shoot apex.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.19.553964">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.19.553964" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.19.553964">
        <p class="paperTitle">The phosphorylation landscape of infection-related development by the rice blast fungus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.19.553964" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.19.553964" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cruz-Mireles, N.; Oses-Ruiz, M.; Derbyshire, P.; Jegousse, C.; Ryder, L. S.; Bautista, M.-J.; Eseola, A. B.; Tang, B.; Yan, X.; Ma, W.; Findlay, K. C.; Were, V.; MacLean, D.; Talbot, N. J.; Menke, F. L. H.</p>
        <p class="info">Score: 68.6, Published: 2023-08-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.19.553964' target='https://doi.org/10.1101/2023.08.19.553964'> 10.1101/2023.08.19.553964</a></p>
        <p class="abstract">Many of the worlds most devastating crop diseases are caused by fungal pathogens which elaborate specialized infection structures to invade plant tissue. Here we present a quantitative mass spectrometry-based phosphoproteomic analysis of infection-related development by the rice blast fungus Magnaporthe oryzae, which threatens global food security. We mapped 8,005 phosphosites on 2,062 fungal proteins, revealing major re-wiring of phosphorylation-based signaling cascades during fungal infection. Comparing phosphosite conservation across 41 fungal species reveals phosphorylation signatures specifically associated with biotrophic and hemibiotrophic fungal infection. We then used parallel reaction monitoring to identify phosphoproteins directly regulated by the Pmk1 MAP kinase that controls plant infection by M. oryzae. We define 33 substrates of Pmk1 and show that Pmk1-dependent phosphorylation of a newly identified regulator, Vts1, is required for rice blast disease. Defining the phosphorylation landscape of infection therefore identifies potential therapeutic interventions for control of plant diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554481">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554481" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554481">
        <p class="paperTitle">Green Index: a widely accessible method to quantify greenness of photosynthetic organisms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554481" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554481" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Signorelli, S.; Casaretto, E.; Millar, A. H.</p>
        <p class="info">Score: 11.1, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554481' target='https://doi.org/10.1101/2023.08.23.554481'> 10.1101/2023.08.23.554481</a></p>
        <p class="abstract">Plant phenotyping involves the quantitative determination of complex plant traits using image analysis. One important parameter is how green plant tissues appear to the observer, which is indicative of their health and developmental stage. Various formulas have been developed to quantify this by calculating leaf greenness scores. We have developed a revised formula called &#34;Green Index&#34; (GI) devised out of the need to quantitatively assess how the apparent greenness of seedlings changes during de-etiolation. The GI calculation is simple, uses widely available RGB values of pixels in images as input, and does not require commercial software platforms or advanced computational skills. In this study we describe the conception of the GI formula, compare it with other widely used greenness formulas, and test its wider application in plant phenotyping using the open source free software platform RawTherapee. We demonstrate the utility of the GI in addressing common issues encountered in assessing plant biology experiments, underscoring its potential as a reliable and accessible tool. Finally, we explore the correlation between GI and chlorophyll content, assess its reliance on different types of photography, and summarizes the key steps for its effective utilization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554152">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554152" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554152">
        <p class="paperTitle">Spatiotemporally distinct responses to mechanical forces shape the developing seed of Arabidopsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554152" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554152" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bauer, A.; Bied, C.; Delattre, A.; Ingram, G.; Golz, J. F.; Landrein, B.</p>
        <p class="info">Score: 18.4, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554152' target='https://doi.org/10.1101/2023.08.21.554152'> 10.1101/2023.08.21.554152</a></p>
        <p class="abstract">Organ morphogenesis depends on mechanical interactions between cells and tissues. These interactions generate forces that can be sensed by cells and affect key cellular processes. However, how mechanical forces contribute, together with biochemical signals, to the shaping of complex organs is still unclear. We address this question using the seed of Arabidopsis as a model system. We show that seeds first experience a phase of high anisotropic growth that is dependent on the response of cortical microtubule (CMT) to forces, which guide cellulose deposition according to shape-driven stresses in the outermost layer of the seed coat. However, at later stages of development, we show that seed growth is isotropic and depend on the properties of an inner layer of the seed coat that stiffens its walls in response to tension but has isotropic material properties. Finally, we show that the transition from anisotropic to isotropic growth is due to dampening of CMT responses to shape-driven stresses. Altogether, our work support that spatiotemporally distinct mechanical responses control the shape of developing seeds in Arabidopsis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554416">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554416" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554416">
        <p class="paperTitle">A pectin-binding peptide with a structural and signaling role in the assembly of the plant cell wall</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554416" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554416" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schoenaers, S.; Lee, H. K.; Gonneau, M.; Faucher, E.; Levasseur, T.; Akary, E.; Claeijs, N.; Moussu, S.; Broyart, C.; Balcerowicz, D.; AbdElgawad, H.; Bassi, A.; Santa Cruz Damineli, D.; Costa, A.; Feijo, J. A.; Moreau, C.; Bonnin, E.; Cathala, B.; Santiago, J.; Hofte, H.; Vissenberg, K.</p>
        <p class="info">Score: 9.6, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554416' target='https://doi.org/10.1101/2023.08.23.554416'> 10.1101/2023.08.23.554416</a></p>
        <p class="abstract">Pressurized cells with strong walls make up the hydrostatic skeleton of plants. Assembly and expansion of such stressed walls depend on a family of secreted RAPID ALKALINIZATION FACTOR (RALF) peptides which, curiously, bind both a membrane receptor complex and wall-localized LEUCINE-RICH REPEAT EXTENSINs (LRXs) in a mutually exclusive way. Here we show that, in root hairs, the RALF22 peptide has a dual structural and signaling role in cell expansion. Together with LRX1, it directs the compaction of charged pectin polymers at the root hair tip into periodic circumferential rings. Free RALF22 induces the formation of a complex with LORELEI-LIKE-GPI-ANCHORED PROTEIN 1 (LLG1) and FERONIA (FER), triggering adaptive cellular responses. These findings show how a peptide simultaneously functions as a structural component organizing cell wall architecture and as a signaling molecule that regulates this process. This mechanism may also underlie wall assembly and expansion in other plant cell types.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.30.551171">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.30.551171" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.30.551171">
        <p class="paperTitle">Chitosan stimulates root hair callose deposition and inhibits root hair growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.30.551171" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.30.551171" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Drs, M.; Haluska, S.; Skrabalkova, E.; Krupar, P.; Potocka, A.; Brejskova, L.; Muller, K.; Serrano, N.; Voxeur, A.; Vernhettes, S.; Ortmannova, J.; Caldarescu, G.; Fendrych, M.; Potocky, M.; Zarsky, V.; Pecenkova, T.</p>
        <p class="info">Score: 16.7, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.30.551171' target='https://doi.org/10.1101/2023.07.30.551171'> 10.1101/2023.07.30.551171</a></p>
        <p class="abstract">Although angiosperm plants have a general capacity to react after the immunity elicitor chitin or chitosan treatment by the cell wall callose deposition, this response in particular cell types and its evolutionary conservation is not understood. Here we show that also the growing root hairs (RHs) of Arabidopsis can respond to a mild (0.001%) chitosan treatment by the callose deposition and by a deceleration of the RH growth. We demonstrate that the glucan synthase-like 5 (GSL5)/PMR4 is vital for chitosan-induced callose deposition but not for RH growth inhibition. Upon the higher chitosan concentration (0.01%) treatment, RHs do not deposit callose, while growth inhibition is prominent. To understand the specificities of the low and high concentration chitosan treatments, we analysed the corresponding PTI signalling components, gene expression, and RH cellular endomembrane and cytoskeleton modifications. Importantly, chitosan-induced callose deposition is also present in the functionally analogous and evolutionarily only distantly related RH-like structures rhizophores (lycophytes) and rhizoids (bryophytes). Our results point to the RH callose deposition as a conserved strategy of soil-anchoring plant cells (rhizoids/rhizophores/RHs) to deal with mild biotic stress. At the same time, high chitosan concentration prominently disturbs intracellular dynamics, tip-localised endomembrane compartments and RH growth, precluding callose deposition.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554752">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554752" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554752">
        <p class="paperTitle">A low-cost and open-source imaging platform reveals spatiotemporal insight into Arabidopsis leaf elongation and movement</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554752" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554752" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oskam, L.; Snoek, B. L.; Pantazopoulou, C. K.; van Veen, H.; Matton, S. E. A.; Dijkhuizen, R. F.; Pierik, R.</p>
        <p class="info">Score: 9.0, Published: 2023-08-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554752' target='https://doi.org/10.1101/2023.08.25.554752'> 10.1101/2023.08.25.554752</a></p>
        <p class="abstract">Plant organs move throughout the diurnal cycle, changing leaf and petiole positions to balance light capture, leaf temperature and water loss under dynamic environmental conditions. Upward movement of the petiole, called hyponasty, is one of several traits of the shade avoidance syndrome (SAS). SAS traits are elicited upon perception of vegetation shade signals such as far-red light (FR) and improve light capture in dense vegetation. Monitoring plant movement at a high temporal resolution allows studying functionality, as well as molecular regulation of hyponasty. However, high temporal resolution imaging solutions are often very expensive, making this unavailable to many researchers. Here, we present a modular and low-cost imaging set-up, based on small Raspberry Pi computers, that can track leaf movements and elongation growth with high temporal resolution. We also developed an open-source, semi-automated image analysis pipeline. Using this setup we followed responses to FR enrichment, light intensity and their interactions. Tracking both elongation and angle of petiole, lamina and entire leaf revealed insight into R:FR sensitivities of leaf growth and movement dynamics, and its interactions with background light intensity. We also identified spatial separation in hyponastic response regulation for the petiole and the lamina of the leaf, depending on the light conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554588">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554588" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554588">
        <p class="paperTitle">Isoform-resolved genome annotation enables mapping of tissue-specific betalain regulation and anthocyanin regulator co-option in amaranth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554588" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554588" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Winkler, T. S.; Vollmer, S. K.; Stetter, M. G.</p>
        <p class="info">Score: 9.0, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554588' target='https://doi.org/10.1101/2023.08.24.554588'> 10.1101/2023.08.24.554588</a></p>
        <p class="abstract">O_LIBetalains are coloring pigments produced only in some families of the order Caryophyllales, where they have functionally replaced anthocyanins as coloring pigments. While the betalain pathway itself is known, the tissue-specific regulation of the pathway remains mostly unclear.
C_LIO_LIWe first improve the high-quality Amaranthus hypochondriacus reference genome and create a highly complete genome annotation by combining ab initio prediction with long-read isoform sequencing data. We accurately annotate the betalain and anthocyanin pathway genes and their regulators in the genome of the betalain producing amaranth and further map the genetic basis of tissue color and tissue-specific regulation of the betalain pathway through a previously unknown MYB transcription factor, AhMYB2.
C_LIO_LIDetailed haplotype analysis enabled us to identify two functional mutations that lead to a knock-out of the betalain pathway in natural accessions of amaranth. In addition, the down-regulation of AhMYB2 in the flower leads to reduced expression of key betalain enzyme genes and loss of red flower color.
C_LIO_LIOur improved amaranth reference genome represents the most complete genome of amaranth to date and is a valuable resource for betalain and amaranth research. The high similarity of the betalain flower regulator AhMYB2 to anthocyanin regulators and a partially conserved interaction motif support the co-option of anthocyanin regulators for the betalain pathway as possible reason for the mutually exclusive nature of the two pigments.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554507">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554507" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554507">
        <p class="paperTitle">The HSF-DREB-MYB regulatory module regulates flavonol biosynthesis and flavonoid B-ring hydroxylation in banana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554507" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554507" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Naik, J.; Rajput, R.; Stracke, R.; Pandey, A.</p>
        <p class="info">Score: 6.7, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554507' target='https://doi.org/10.1101/2023.08.23.554507'> 10.1101/2023.08.23.554507</a></p>
        <p class="abstract">Plant flavonols act primarily as ultraviolet radiation absorbers, reactive oxygen species scavengers, and phytoalexins, and they contribute to biotic and abiotic stress tolerance in plants. Banana (Musa acuminata), an herbaceous monocot and important fruit crop, accumulates flavonol derivatives in different organs, including the edible fruit pulp. Although flavonol content varies greatly in different organs, the molecular mechanisms involving transcriptional regulation of flavonol synthesis in banana are not known. Here, we characterized three SG7-R2R3 MYB transcription factors MaMYBFA1, MaMYBFA2, and MaMYBFA3) and their upstream regulators, heat shock transcription factor (MaHSF11) and dehydration responsive element binding factor (MaDREB1), to elucidate the molecular mechanism involved in transcriptional regulation of flavonol biosynthesis in banana. MaMYBFA positively regulate flavonol synthase 2 (MaFLS2) and downregulates MaFLS1. We show these transcription factors to be weak regulators of flavonol synthesis. Overexpression of MaHSF11 enhances flavonol contents, particularly that of myricetin, and promotes flavonol B-ring hydroxylation, which contributes to the diversity of flavonol derivatives. MaHSF11 directly interacts with the MaFLS1 and flavonoid 3&#39;, 5&#39;-hydroxylase1 (MaF3&#39;5&#39;H1) promoters, both in vitro and in vivo. MaHSF11 activates the expression of MaDREB1 directly, which in turn regulates the expression of MaMYBFA3. Overall, our study elucidates a key regulatory mechanism for flavonol synthesis in banana and suggests possible targets for genetic optimization to enhance nutritional value and stress responses in this globally important fruit crop.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
