<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Characterization of DNA methylation reader proteins of Arabidopsis thaliana
Authors: Cahn, J.; Lloyd, J. P. B.; Karemaker, I. D.; Jansen, P. W.; Pflueger, J.; Duncan, O.; Petereit, J.; Bogdanovic, O.; Millar, H.; Vermeulen, M.; Lister, R.
Score: 33.5, Published: 2023-12-06 DOI: 10.1101/2023.12.05.570080
In plants, cytosine DNA methylation (mC) is largely associated with transcriptional repression of transposable elements, but it can also be found in the body of expressed genes, referred to as gene body methylation (GbM).">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="Characterization of DNA methylation reader proteins of Arabidopsis thaliana
Authors: Cahn, J.; Lloyd, J. P. B.; Karemaker, I. D.; Jansen, P. W.; Pflueger, J.; Duncan, O.; Petereit, J.; Bogdanovic, O.; Millar, H.; Vermeulen, M.; Lister, R.
Score: 33.5, Published: 2023-12-06 DOI: 10.1101/2023.12.05.570080
In plants, cytosine DNA methylation (mC) is largely associated with transcriptional repression of transposable elements, but it can also be found in the body of expressed genes, referred to as gene body methylation (GbM)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-13T10:39:57+00:00" />
<meta property="article:modified_time" content="2023-12-13T10:39:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="Characterization of DNA methylation reader proteins of Arabidopsis thaliana
Authors: Cahn, J.; Lloyd, J. P. B.; Karemaker, I. D.; Jansen, P. W.; Pflueger, J.; Duncan, O.; Petereit, J.; Bogdanovic, O.; Millar, H.; Vermeulen, M.; Lister, R.
Score: 33.5, Published: 2023-12-06 DOI: 10.1101/2023.12.05.570080
In plants, cytosine DNA methylation (mC) is largely associated with transcriptional repression of transposable elements, but it can also be found in the body of expressed genes, referred to as gene body methylation (GbM)."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "Characterization of DNA methylation reader proteins of Arabidopsis thaliana\nAuthors: Cahn, J.; Lloyd, J. P. B.; Karemaker, I. D.; Jansen, P. W.; Pflueger, J.; Duncan, O.; Petereit, J.; Bogdanovic, O.; Millar, H.; Vermeulen, M.; Lister, R.\nScore: 33.5, Published: 2023-12-06 DOI: 10.1101/2023.12.05.570080\nIn plants, cytosine DNA methylation (mC) is largely associated with transcriptional repression of transposable elements, but it can also be found in the body of expressed genes, referred to as gene body methylation (GbM).",
  "keywords": [
    
  ],
  "articleBody": " Characterization of DNA methylation reader proteins of Arabidopsis thaliana\nAuthors: Cahn, J.; Lloyd, J. P. B.; Karemaker, I. D.; Jansen, P. W.; Pflueger, J.; Duncan, O.; Petereit, J.; Bogdanovic, O.; Millar, H.; Vermeulen, M.; Lister, R.\nScore: 33.5, Published: 2023-12-06 DOI: 10.1101/2023.12.05.570080\nIn plants, cytosine DNA methylation (mC) is largely associated with transcriptional repression of transposable elements, but it can also be found in the body of expressed genes, referred to as gene body methylation (GbM). GbM is correlated with ubiquitously expressed genes, however its function, or absence thereof, is highly debated. The different output that mC can have raises questions as to how it is interpreted - or read - differently in these sequence and genomic contexts. To screen for potential mC binding proteins, we performed an unbiased DNA affinity pull-down assay combined with quantitative mass spectrometry using methylated DNA probes for each DNA sequence context. All mC readers known to date were found to preferentially bind to the methylated probes, along with a range of new mC binding protein candidates. Functional characterization of these mC readers, focused on the MBD and SUVH families, was undertaken by ChIP-seq mapping of genome-wide binding sites, their protein interactors, and the impact of high-order mutations on transcriptomic and epigenomic profiles. Together, this highlighted specific context preferences for these proteins, and in particular the ability of MBD2 to bind specifically to GbM. This comprehensive analysis of Arabidopsis mC readers emphasizes the complexity and interconnectivity between DNA methylation and chromatin remodelling processes in plants.\nApoplastic barrier establishment in roots and nodules of Lotus japonicus is essential for root-shoot signaling and N-fixation\nAuthors: Shen, D.; Venado, R. E.; Neumann, U.; Dyballa-Rukes, N.; Mahiwal, S.; Metzger, S.; Nakano, R. T.; Marin, M.; Andersen, T. G.\nScore: 29.3, Published: 2023-12-08 DOI: 10.1101/2023.12.06.570432\nThe molecular framework underlying apoplastic root barrier formation has been unveiled in the model species Arabidopsis thaliana where establishment of Casparian strips occurs at an early stage of root development. In legumes, this region overlaps with the area where nitrogen-fixing bacteria can induce nodule formation, termed the susceptible zone. Moreover, while nodules themselves also contain an endodermis spanning their vascular bundles, it is current unknown if Casparian strips serve as a filter for transport across this specialized organ. Here we establish barrier mutants in the symbiosis model Lotus japonicus. We find that the while genetic network controlling Casparian strip formation is conserved in this legume species, formation of functional barriers is crucial for establishment of N-fixing nodules. By probing this in detail, we establish a model where the Casparian strip, via its linked Schengen pathway, converge with long distance N signaling and systemic regulation of nodulation. Moreover, this also reveal that the genetic system for barrier establishment in the root endodermis is shared in nodule vascular endodermis and required for nodule function. Combined, our findings uncover a novel role of apoplastic root barriers and establishes a mutant collection suitable to probe the role of root barriers in symbiotic plant-microbe relationships.\nWarm temperature and mild water stress cooperatively promote root elongation\nAuthors: Hayes, S.; Leong, C. K.; Zhang, W.; Lamain, M.; Lamers, J.; de Zeeuw, T.; Verstappen, F.; Hiltbrunner, A.; Testerink, C.\nScore: 28.1, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569400\nWarm temperatures have a dramatic effect on plant development. In shoots, stems elongate, and leaves are raised in a developmental programme called thermomorphogenesis. This results in enhanced leaf cooling capacity1. Thermomorphogenesis is tightly intertwined with light signalling pathways. The level of integration is so high that it has been proposed that shoot temperature sensing may have evolved from the co-option of an existing light signalling pathway during the colonisation of land by plants2. Roots also undergo thermomorphogenesis, but the mechanism by which this occurs is less well understood. Main root elongation is enhanced at warm temperatures, and this response is independent of many of the light and temperature signalling components of the shoot3. Roots develop in darkness and so it is a reasonable assumption that root temperature signalling is not through modulation of light signalling. It was recently speculated that due to the close correlation between warm temperature and soil moisture content, root temperature signalling could feasibly be related to water availability signals2. In this study we tested the interaction between temperature and water availability signalling in plant roots. We found that these environmental factors co-operatively enhance main root elongation. This interaction effect was dependent on SUCROSE NON-FERMENTING RELATED KINASE 2.2 (SnRK2.2) and SnRK2.3 and the E3 ubiquitin ligase CONSTITUTIVELY PHOTOMORPHOGENIC 1 (COP1). We found that SnRK2.2 / 2.3 and COP1 have opposite effects on the stability of the transcription factor ELONGATED HYPOCOTYL 5 (HY5) in elongation zone hair cells. The stability of HY5 in these cell types generally corresponded to the degree of root elongation seen in each mutant background. Our study reveals several molecular components of root thermomorphogenesis and highlights the importance of an integrative approach to plant environmental signalling. Our results may have direct implications for agricultural land management, especially as global climates become more unpredictable.\nRestriction of intercellular communication is required for organ regeneration\nAuthors: Cohen, I.; Hak, H.; Perez-Sancho, J.; Spiegelman, Z.; Efroni, I.\nScore: 18.4, Published: 2023-12-10 DOI: 10.1101/2023.12.10.570601\nThe excision of the root tip, including the stem cell niche, triggers rapid regeneration from remnant cells in the stump. In plants, cell identity relies on positional information transported through cytoplasmatic bridges known as plasmodesmata. However, how such information is reset to allow the regeneration of lost identities is unknown. Here, we show that the movement of mobile signals is temporarily restricted near the incision site and that regeneration-induced members of the LATERAL ORGAN DOMAIN (LBD) plant-specific developmental regulators are necessary and sufficient for this restriction. Regeneration was disrupted in high-order lbd mutants but was restored by transient localized plasmodesmata closure. We propose that LBD-mediated modulation of intercellular connectivity is crucial for plant regeneration and may have widespread roles in de novo pattern formation.\nA Day in the Life of Arabidopsis: 24-Hour Time-lapse Single-nucleus Transcriptomics Reveal Cell-type specific Circadian Rhythms\nAuthors: Qin, Y.; Liu, Z.; Gao, S.; Long, Y.; Zhu, X.; Liu, B.; Gao, Y.; Xie, Q.; Nohales, M. A.; Xu, X.; Zhai, J.\nScore: 17.6, Published: 2023-12-10 DOI: 10.1101/2023.12.09.570919\nFunctional circadian clock is fundamental to the adaptation and survival of organisms. In land plants, the comprehensive profiling of circadian gene expression at the single-cell level is largely unknown partly due to the challenges in obtaining precisely-timed single cells from plant cells embedded within cell walls. To bridge this gap, we employed single-nucleus RNA sequencing (snRNA-seq) on twelve seedling samples collected at 2-hour intervals over a 24-hour day in Arabidopsis, yielding a total of over 130,000 nuclei. From this data, we found that three cell clusters in the shoot share a coherent rhythm, while more than 3,000 genes display cell-type specific rhythmic expression. Only 19 genes are oscillating in more than ten different cell types, and the majority of them are well-documented core oscillators, suggesting the snRNA-seq circadian data could be used to identify key circadian regulators in a broad range of plant systems. Our data provides the first comprehensive resource for plant circadian rhythms at the single-cell level (hosted at https://zhailab.bio.sustech.edu.cn/sc_circadian).\nSeed coat formation in Arabidopsis requires a concerted action of JUMONJI histone H3K27me3 demethylases and Brassinosteroid signaling\nAuthors: Pankaj, R.; Lima, R. B.; Luo, G.-Y.; Ehlert, S.; Finger, P.; Sato, H.; Figueiredo, D. D.\nScore: 17.1, Published: 2023-12-08 DOI: 10.1101/2023.12.07.569203\nSeed development in flowering plants starts with a double fertilization event where two paternal gametes, the sperm cells, fertilize their maternal counterparts, egg cell and central cell. This leads to the formation of the embryo and the endosperm. These fertilization products are enveloped by the maternally-derived seed coat, the development of which is inhibited prior to fertilization by epigenetic regulator Polycomb Repressive Complex 2 (PRC2). This complex deposits the repressive histone mark H3K27me3, whose removal is necessary for seed coat formation. However, H3K27me3 marks are stable and PRC2 removal does not explain how seed coat genes become activated after fertilization. Here, we show that JUMONJI-type (JMJ) histone demethylases are expressed in the seed coats of Arabidopsis thaliana (Arabidopsis) and are necessary for its formation. We further propose that JMJ activity is coupled to Brassinosteroid (BR) function, as BR effectors have been shown to physically recruit JMJ proteins to target loci. Consistent with this hypothesis, we show that loss of BR function leads to seed coat defects, which can be rescued by depletion of H3K27me3. Finally, we reveal an additional pathway through which BRs directly regulate seed coat development, independently of H3K27me3 deposition. This discovery highlights the diverse functions of BRs in coordinating seed development, beyond their known roles in plant growth and development.\nER-anchored protein sorting controls the fate of two proteasome activators for intracellular organelle communication during proteotoxic stress\nAuthors: Langin, G.; Raffeiner, M.; Biermann, D.; Franz-Wachtel, M.; Spinti, D.; Börnke, F.; Macek, B.; Üstün, S.\nScore: 13.2, Published: 2023-12-12 DOI: 10.1101/2023.12.11.571118\nProteotoxic stress, characterized by the accumulation of damaged proteins, poses a significant challenge to cellular homeostasis. To mitigate proteotoxicity eukaryotes employ the proteasome that is regulated by proteasome activators, e.g. transcription factors that promote gene expression of proteasome subunits. As proteotoxicity originates in different compartments, cells need to perceive signals from various locations. Understanding which components integrate signals to address proteotoxicity is essential to develop strategies to cope with proteotoxicity but remain elusive. Here, we identify that the proteasome autoregulatory feedback loop acts as a gatekeeper to facilitate the communication between nucleus and chloroplast. We reveal that the ER-anchored protein sorting system (ERAPS) controls the proteasomal degradation or nuclear translocation of proteasome activators NAC53 and NAC78. While both transcription factors activate the proteasome gene expression, they repress photosynthesis-associated nuclear genes during proteotoxicity. Collectively, our discoveries provide a novel conceptual framework in which the proteasome autoregulatory feedback loop coordinates subcellular proteostasis.\nBait, not reward: CO2-enriched Nepenthes pitchers secrete toxic nectar\nAuthors: Lathika, C. C.; Sujatha, G. B.; Thomas, G.; Johnson, A. J.; Viswanathan, G.; Varghese, T. S.; Mohamed, S.; Shereefa, L. A.; Baby, S.\nScore: 30.1, Published: 2023-11-25 DOI: 10.1101/2023.11.25.568661\nNepenthes pitchers are leaf-evolved biological traps holding high levels of CO2 within them. Extrafloral nectar (EFN) secreted by Nepenthes pitchers has long been regarded as the major reward to the visiting arthropods, but its chemical constituents and their role in prey capture are least explored. Here we show that Nepenthes EFN is a toxic sugar mix devoid of key nitrogenous components (amino acids, proteins) and vitamin C. Fatty acids (cis-11-octadecenoic acid (C18:1), palmitic acid (C16:0), stearic acid (C18:0)) and their derivatives were detected in both N. khasiana peristome and lid EFNs. Both the nectars showed strong acetylcholinesterase (AChE) inhibition in vitro and in vivo. Bioactivity guided isolation revealed (+)-isoshinanolone, a naphthoquinone derivative, as the AChE inhibitor in Nepenthes EFNs. GC-headspace analysis showed the naphthoquinone, plumbagin, as the major volatile constituent in prey capturing regions of pitchers, along with the VOCs, (Z)-3-hexen-1-ol, 1-hexanol and acetoin. N. khasiana pitchers trap a wide spectrum of preys dominated by ants, whereas only limited herbivory was observed on its leaves and pitchers. Chemical (EFN, volatiles), gaseous (CO2), visual (colour, UV reflectance, fluorescence), physical (peristome geometry, surface microstructure, wettable surface-aquaplaning) and environmental (humidity, rain) factors effect prey capture in Nepenthes pitchers. These leaf-transformed traps with high growth rate and C:N ratio, reduced Rubisco activity, modified stomata, acidic pitcher fluid and absence of key nitrogenous metabolites are model systems displaying the effects of elevated CO2 within them. We testify that Nepenthes EFN is a toxic sugar bait which hinders the neuronal activity and flight of visiting arthropods. These unique traps adopt various deceptive strategies for prey capture, and our discovery abolishes the notion of Nepenthes EFN as a reward to the visiting ants and other arthropods.\nThe nucleotide binding domain of NRC-dependent disease resistance proteins is sufficient to activate downstream helper NLR oligomerization and immune signaling\nAuthors: Contreras, M. P.; Pai, H.; Thompson, R.; Claeys, J.; Adachi, H.; Kamoun, S.\nScore: 37.7, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569466\nNucleotide-binding domain and leucine-rich repeat (NLR) proteins with pathogen sensor activities have evolved to initiate immune signaling by activating helper NLRs. However, the mechanisms underpinning helper NLR activation by sensor NLRs remain poorly understood. Although coiled-coil (CC) type sensor NLRs such as the Potato virus X disease resistance protein Rx have been shown to activate the oligomerization of their downstream helpers NRC2 and NRC4, the domains involved in sensor-helper signaling are not known. Here, we show that the nucleotide binding (NB) domain within the NB-ARC of the Potato virus X disease resistance protein Rx is necessary and sufficient for oligomerization and immune signaling of downstream helper NLRs. In addition, the NB domains of the disease resistance proteins Gpa2 (cyst nematode resistance), Rpi-amr1, Rpi-amr3 (oomycete resistance) and Sw-5b (virus resistance) are also sufficient to activate their respective downstream NRC helpers. Moreover, the NB domain of Rx and its helper NRC2 form a minimal functional unit that can be transferred from solanaceous plants (lamiids) to the Campanulid species lettuce (Lactuca sativa). Our results challenge the prevailing paradigm that NLR proteins exclusively signal via their N-terminal domains and reveal a signaling activity for the NB domain of NRC-dependent sensor NLRs. We propose a model in which helper NLRs monitor the status of the NB domain of their upstream sensors.\nTracing the evolutionary emergence of the temperature sensing prion-like domain in EARLY FLOWERING 3 across the plant kingdom\nAuthors: Zhu, Z.; Trenner, J.; Quint, M.\nScore: 6.8, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570556\nPlants have evolved to anticipate and adjust their growth and development in response to environmental changes. To mitigate the negative influence of global climate change on crop production, understanding the key regulators of plant performance is imperative. EARLY FLOWERING 3 (ELF3) is such a regulator involved in the circadian clock and thermomorphogenesis. Arabidopsis thaliana ELF3 contains a prion-like domain (PrD) that functions as a thermosensor, enabling its liquid-liquid phase separation at high ambient temperatures. To understand the conservation of this function across the plant kingdom, we traced the evolutionary emergence of ELF3 with a focus on the PrD, which confers liquid-liquid phase separation. We observed that the presence of the domain within ELF3, mainly contributed by the length of polyglutamine (polyQ) repeats, is largely restricted to Brassicales. This suggests that ELF3s thermosensory function is a rather recent and secondary acquirement that was added to its main function. By analyzing 319 natural Arabidopsis thaliana accessions, we detected a wide range of polyQ length variation in ELF3. However, polyQ length is only weakly associated with geographic origin, climate conditions, and classic temperature-responsive phenotypes. Consequently, we conclude that although the emergence of PrD is not likely to be a key driver of environmental adaptation, it adds an extra layer to ELF3s role in thermomorphogenesis.\n",
  "wordCount" : "2448",
  "inLanguage": "en",
  "datePublished": "2023-12-13T10:39:57Z",
  "dateModified": "2023-12-13T10:39:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 13, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570080">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570080" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570080">
        <p class="paperTitle">Characterization of DNA methylation reader proteins of Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570080" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570080" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cahn, J.; Lloyd, J. P. B.; Karemaker, I. D.; Jansen, P. W.; Pflueger, J.; Duncan, O.; Petereit, J.; Bogdanovic, O.; Millar, H.; Vermeulen, M.; Lister, R.</p>
        <p class="info">Score: 33.5, Published: 2023-12-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570080' target='https://doi.org/10.1101/2023.12.05.570080'> 10.1101/2023.12.05.570080</a></p>
        <p class="abstract">In plants, cytosine DNA methylation (mC) is largely associated with transcriptional repression of transposable elements, but it can also be found in the body of expressed genes, referred to as gene body methylation (GbM). GbM is correlated with ubiquitously expressed genes, however its function, or absence thereof, is highly debated. The different output that mC can have raises questions as to how it is interpreted - or read - differently in these sequence and genomic contexts. To screen for potential mC binding proteins, we performed an unbiased DNA affinity pull-down assay combined with quantitative mass spectrometry using methylated DNA probes for each DNA sequence context. All mC readers known to date were found to preferentially bind to the methylated probes, along with a range of new mC binding protein candidates. Functional characterization of these mC readers, focused on the MBD and SUVH families, was undertaken by ChIP-seq mapping of genome-wide binding sites, their protein interactors, and the impact of high-order mutations on transcriptomic and epigenomic profiles. Together, this highlighted specific context preferences for these proteins, and in particular the ability of MBD2 to bind specifically to GbM. This comprehensive analysis of Arabidopsis mC readers emphasizes the complexity and interconnectivity between DNA methylation and chromatin remodelling processes in plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.570432">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.570432" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.570432">
        <p class="paperTitle">Apoplastic barrier establishment in roots and nodules of Lotus japonicus is essential for root-shoot signaling and N-fixation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.570432" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.570432" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shen, D.; Venado, R. E.; Neumann, U.; Dyballa-Rukes, N.; Mahiwal, S.; Metzger, S.; Nakano, R. T.; Marin, M.; Andersen, T. G.</p>
        <p class="info">Score: 29.3, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.570432' target='https://doi.org/10.1101/2023.12.06.570432'> 10.1101/2023.12.06.570432</a></p>
        <p class="abstract">The molecular framework underlying apoplastic root barrier formation has been unveiled in the model species Arabidopsis thaliana where establishment of Casparian strips occurs at an early stage of root development. In legumes, this region overlaps with the area where nitrogen-fixing bacteria can induce nodule formation, termed the susceptible zone. Moreover, while nodules themselves also contain an endodermis spanning their vascular bundles, it is current unknown if Casparian strips serve as a filter for transport across this specialized organ. Here we establish barrier mutants in the symbiosis model Lotus japonicus. We find that the while genetic network controlling Casparian strip formation is conserved in this legume species, formation of functional barriers is crucial for establishment of N-fixing nodules. By probing this in detail, we establish a model where the Casparian strip, via its linked Schengen pathway, converge with long distance N signaling and systemic regulation of nodulation. Moreover, this also reveal that the genetic system for barrier establishment in the root endodermis is shared in nodule vascular endodermis and required for nodule function. Combined, our findings uncover a novel role of apoplastic root barriers and establishes a mutant collection suitable to probe the role of root barriers in symbiotic plant-microbe relationships.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569400">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569400" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569400">
        <p class="paperTitle">Warm temperature and mild water stress cooperatively promote root elongation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569400" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569400" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hayes, S.; Leong, C. K.; Zhang, W.; Lamain, M.; Lamers, J.; de Zeeuw, T.; Verstappen, F.; Hiltbrunner, A.; Testerink, C.</p>
        <p class="info">Score: 28.1, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569400' target='https://doi.org/10.1101/2023.11.30.569400'> 10.1101/2023.11.30.569400</a></p>
        <p class="abstract">Warm temperatures have a dramatic effect on plant development. In shoots, stems elongate, and leaves are raised in a developmental programme called thermomorphogenesis. This results in enhanced leaf cooling capacity1. Thermomorphogenesis is tightly intertwined with light signalling pathways. The level of integration is so high that it has been proposed that shoot temperature sensing may have evolved from the co-option of an existing light signalling pathway during the colonisation of land by plants2. Roots also undergo thermomorphogenesis, but the mechanism by which this occurs is less well understood. Main root elongation is enhanced at warm temperatures, and this response is independent of many of the light and temperature signalling components of the shoot3. Roots develop in darkness and so it is a reasonable assumption that root temperature signalling is not through modulation of light signalling. It was recently speculated that due to the close correlation between warm temperature and soil moisture content, root temperature signalling could feasibly be related to water availability signals2. In this study we tested the interaction between temperature and water availability signalling in plant roots. We found that these environmental factors co-operatively enhance main root elongation. This interaction effect was dependent on SUCROSE NON-FERMENTING RELATED KINASE 2.2 (SnRK2.2) and SnRK2.3 and the E3 ubiquitin ligase CONSTITUTIVELY PHOTOMORPHOGENIC 1 (COP1). We found that SnRK2.2 / 2.3 and COP1 have opposite effects on the stability of the transcription factor ELONGATED HYPOCOTYL 5 (HY5) in elongation zone hair cells. The stability of HY5 in these cell types generally corresponded to the degree of root elongation seen in each mutant background. Our study reveals several molecular components of root thermomorphogenesis and highlights the importance of an integrative approach to plant environmental signalling. Our results may have direct implications for agricultural land management, especially as global climates become more unpredictable.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.10.570601">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.10.570601" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.10.570601">
        <p class="paperTitle">Restriction of intercellular communication is required for organ regeneration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.10.570601" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.10.570601" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cohen, I.; Hak, H.; Perez-Sancho, J.; Spiegelman, Z.; Efroni, I.</p>
        <p class="info">Score: 18.4, Published: 2023-12-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.10.570601' target='https://doi.org/10.1101/2023.12.10.570601'> 10.1101/2023.12.10.570601</a></p>
        <p class="abstract">The excision of the root tip, including the stem cell niche, triggers rapid regeneration from remnant cells in the stump. In plants, cell identity relies on positional information transported through cytoplasmatic bridges known as plasmodesmata. However, how such information is reset to allow the regeneration of lost identities is unknown. Here, we show that the movement of mobile signals is temporarily restricted near the incision site and that regeneration-induced members of the LATERAL ORGAN DOMAIN (LBD) plant-specific developmental regulators are necessary and sufficient for this restriction. Regeneration was disrupted in high-order lbd mutants but was restored by transient localized plasmodesmata closure. We propose that LBD-mediated modulation of intercellular connectivity is crucial for plant regeneration and may have widespread roles in de novo pattern formation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.09.570919">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.09.570919" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.09.570919">
        <p class="paperTitle">A Day in the Life of Arabidopsis: 24-Hour Time-lapse Single-nucleus Transcriptomics Reveal Cell-type specific Circadian Rhythms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.09.570919" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.09.570919" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Qin, Y.; Liu, Z.; Gao, S.; Long, Y.; Zhu, X.; Liu, B.; Gao, Y.; Xie, Q.; Nohales, M. A.; Xu, X.; Zhai, J.</p>
        <p class="info">Score: 17.6, Published: 2023-12-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.09.570919' target='https://doi.org/10.1101/2023.12.09.570919'> 10.1101/2023.12.09.570919</a></p>
        <p class="abstract">Functional circadian clock is fundamental to the adaptation and survival of organisms. In land plants, the comprehensive profiling of circadian gene expression at the single-cell level is largely unknown partly due to the challenges in obtaining precisely-timed single cells from plant cells embedded within cell walls. To bridge this gap, we employed single-nucleus RNA sequencing (snRNA-seq) on twelve seedling samples collected at 2-hour intervals over a 24-hour day in Arabidopsis, yielding a total of over 130,000 nuclei. From this data, we found that three cell clusters in the shoot share a coherent rhythm, while more than 3,000 genes display cell-type specific rhythmic expression. Only 19 genes are oscillating in more than ten different cell types, and the majority of them are well-documented core oscillators, suggesting the snRNA-seq circadian data could be used to identify key circadian regulators in a broad range of plant systems. Our data provides the first comprehensive resource for plant circadian rhythms at the single-cell level (hosted at https://zhailab.bio.sustech.edu.cn/sc_circadian).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.569203">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.569203" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.569203">
        <p class="paperTitle">Seed coat formation in Arabidopsis requires a concerted action of JUMONJI histone H3K27me3 demethylases and Brassinosteroid signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.569203" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.569203" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pankaj, R.; Lima, R. B.; Luo, G.-Y.; Ehlert, S.; Finger, P.; Sato, H.; Figueiredo, D. D.</p>
        <p class="info">Score: 17.1, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.569203' target='https://doi.org/10.1101/2023.12.07.569203'> 10.1101/2023.12.07.569203</a></p>
        <p class="abstract">Seed development in flowering plants starts with a double fertilization event where two paternal gametes, the sperm cells, fertilize their maternal counterparts, egg cell and central cell. This leads to the formation of the embryo and the endosperm. These fertilization products are enveloped by the maternally-derived seed coat, the development of which is inhibited prior to fertilization by epigenetic regulator Polycomb Repressive Complex 2 (PRC2). This complex deposits the repressive histone mark H3K27me3, whose removal is necessary for seed coat formation. However, H3K27me3 marks are stable and PRC2 removal does not explain how seed coat genes become activated after fertilization. Here, we show that JUMONJI-type (JMJ) histone demethylases are expressed in the seed coats of Arabidopsis thaliana (Arabidopsis) and are necessary for its formation. We further propose that JMJ activity is coupled to Brassinosteroid (BR) function, as BR effectors have been shown to physically recruit JMJ proteins to target loci. Consistent with this hypothesis, we show that loss of BR function leads to seed coat defects, which can be rescued by depletion of H3K27me3. Finally, we reveal an additional pathway through which BRs directly regulate seed coat development, independently of H3K27me3 deposition. This discovery highlights the diverse functions of BRs in coordinating seed development, beyond their known roles in plant growth and development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.571118">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.571118" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.571118">
        <p class="paperTitle">ER-anchored protein sorting controls the fate of two proteasome activators for intracellular organelle communication during proteotoxic stress</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.571118" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.571118" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Langin, G.; Raffeiner, M.; Biermann, D.; Franz-Wachtel, M.; Spinti, D.; Börnke, F.; Macek, B.; Üstün, S.</p>
        <p class="info">Score: 13.2, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.571118' target='https://doi.org/10.1101/2023.12.11.571118'> 10.1101/2023.12.11.571118</a></p>
        <p class="abstract">Proteotoxic stress, characterized by the accumulation of damaged proteins, poses a significant challenge to cellular homeostasis. To mitigate proteotoxicity eukaryotes employ the proteasome that is regulated by proteasome activators, e.g. transcription factors that promote gene expression of proteasome subunits. As proteotoxicity originates in different compartments, cells need to perceive signals from various locations. Understanding which components integrate signals to address proteotoxicity is essential to develop strategies to cope with proteotoxicity but remain elusive. Here, we identify that the proteasome autoregulatory feedback loop acts as a gatekeeper to facilitate the communication between nucleus and chloroplast. We reveal that the ER-anchored protein sorting system (ERAPS) controls the proteasomal degradation or nuclear translocation of proteasome activators NAC53 and NAC78. While both transcription factors activate the proteasome gene expression, they repress photosynthesis-associated nuclear genes during proteotoxicity. Collectively, our discoveries provide a novel conceptual framework in which the proteasome autoregulatory feedback loop coordinates subcellular proteostasis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.25.568661">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.25.568661" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.25.568661">
        <p class="paperTitle">Bait, not reward: CO2-enriched Nepenthes pitchers secrete toxic nectar</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.25.568661" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.25.568661" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lathika, C. C.; Sujatha, G. B.; Thomas, G.; Johnson, A. J.; Viswanathan, G.; Varghese, T. S.; Mohamed, S.; Shereefa, L. A.; Baby, S.</p>
        <p class="info">Score: 30.1, Published: 2023-11-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.25.568661' target='https://doi.org/10.1101/2023.11.25.568661'> 10.1101/2023.11.25.568661</a></p>
        <p class="abstract">Nepenthes pitchers are leaf-evolved biological traps holding high levels of CO2 within them. Extrafloral nectar (EFN) secreted by Nepenthes pitchers has long been regarded as the major reward to the visiting arthropods, but its chemical constituents and their role in prey capture are least explored. Here we show that Nepenthes EFN is a toxic sugar mix devoid of key nitrogenous components (amino acids, proteins) and vitamin C. Fatty acids (cis-11-octadecenoic acid (C18:1), palmitic acid (C16:0), stearic acid (C18:0)) and their derivatives were detected in both N. khasiana peristome and lid EFNs. Both the nectars showed strong acetylcholinesterase (AChE) inhibition in vitro and in vivo. Bioactivity guided isolation revealed (&#43;)-isoshinanolone, a naphthoquinone derivative, as the AChE inhibitor in Nepenthes EFNs. GC-headspace analysis showed the naphthoquinone, plumbagin, as the major volatile constituent in prey capturing regions of pitchers, along with the VOCs, (Z)-3-hexen-1-ol, 1-hexanol and acetoin. N. khasiana pitchers trap a wide spectrum of preys dominated by ants, whereas only limited herbivory was observed on its leaves and pitchers. Chemical (EFN, volatiles), gaseous (CO2), visual (colour, UV reflectance, fluorescence), physical (peristome geometry, surface microstructure, wettable surface-aquaplaning) and environmental (humidity, rain) factors effect prey capture in Nepenthes pitchers. These leaf-transformed traps with high growth rate and C:N ratio, reduced Rubisco activity, modified stomata, acidic pitcher fluid and absence of key nitrogenous metabolites are model systems displaying the effects of elevated CO2 within them. We testify that Nepenthes EFN is  a toxic sugar bait which hinders the neuronal activity and flight of visiting arthropods. These unique traps adopt various deceptive strategies for prey capture, and our discovery abolishes the notion of Nepenthes EFN as a  reward to the visiting ants and other arthropods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569466">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569466" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569466">
        <p class="paperTitle">The nucleotide binding domain of NRC-dependent disease resistance proteins is sufficient to activate downstream helper NLR oligomerization and immune signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569466" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569466" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Contreras, M. P.; Pai, H.; Thompson, R.; Claeys, J.; Adachi, H.; Kamoun, S.</p>
        <p class="info">Score: 37.7, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569466' target='https://doi.org/10.1101/2023.11.30.569466'> 10.1101/2023.11.30.569466</a></p>
        <p class="abstract">Nucleotide-binding domain and leucine-rich repeat (NLR) proteins with pathogen sensor activities have evolved to initiate immune signaling by activating helper NLRs. However, the mechanisms underpinning helper NLR activation by sensor NLRs remain poorly understood. Although coiled-coil (CC) type sensor NLRs such as the Potato virus X disease resistance protein Rx have been shown to activate the oligomerization of their downstream helpers NRC2 and NRC4, the domains involved in sensor-helper signaling are not known. Here, we show that the nucleotide binding (NB) domain within the NB-ARC of the Potato virus X disease resistance protein Rx is necessary and sufficient for oligomerization and immune signaling of downstream helper NLRs. In addition, the NB domains of the disease resistance proteins Gpa2 (cyst nematode resistance), Rpi-amr1, Rpi-amr3 (oomycete resistance) and Sw-5b (virus resistance) are also sufficient to activate their respective downstream NRC helpers. Moreover, the NB domain of Rx and its helper NRC2 form a minimal functional unit that can be transferred from solanaceous plants (lamiids) to the Campanulid species lettuce (Lactuca sativa). Our results challenge the prevailing paradigm that NLR proteins exclusively signal via their N-terminal domains and reveal a signaling activity for the NB domain of NRC-dependent sensor NLRs. We propose a model in which helper NLRs monitor the status of the NB domain of their upstream sensors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570556">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570556" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570556">
        <p class="paperTitle">Tracing the evolutionary emergence of the temperature sensing prion-like domain in EARLY FLOWERING 3 across the plant kingdom</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570556" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570556" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhu, Z.; Trenner, J.; Quint, M.</p>
        <p class="info">Score: 6.8, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570556' target='https://doi.org/10.1101/2023.12.07.570556'> 10.1101/2023.12.07.570556</a></p>
        <p class="abstract">Plants have evolved to anticipate and adjust their growth and development in response to environmental changes. To mitigate the negative influence of global climate change on crop production, understanding the key regulators of plant performance is imperative. EARLY FLOWERING 3 (ELF3) is such a regulator involved in the circadian clock and thermomorphogenesis. Arabidopsis thaliana ELF3 contains a prion-like domain (PrD) that functions as a thermosensor, enabling its liquid-liquid phase separation at high ambient temperatures. To understand the conservation of this function across the plant kingdom, we traced the evolutionary emergence of ELF3 with a focus on the PrD, which confers liquid-liquid phase separation. We observed that the presence of the domain within ELF3, mainly contributed by the length of polyglutamine (polyQ) repeats, is largely restricted to Brassicales. This suggests that ELF3s thermosensory function is a rather recent and secondary acquirement that was added to its main function. By analyzing 319 natural Arabidopsis thaliana accessions, we detected a wide range of polyQ length variation in ELF3. However, polyQ length is only weakly associated with geographic origin, climate conditions, and classic temperature-responsive phenotypes. Consequently, we conclude that although the emergence of PrD is not likely to be a key driver of environmental adaptation, it adds an extra layer to ELF3s role in thermomorphogenesis.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
