<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="FROM DUST TO SEED: A LUNAR CHICKPEA STORY
Authors: Atkin, J.; Oliveira Pedro dos Santos, S.
Score: 114.2, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576311
Food sustainability is one of the most significant barriers to long-term space travel. Providing resources from Earth is not cost-efficient, and resupply missions are not viable to meet the needs of long-term life in deep space conditions. Plants in space can provide a source of nutrition and oxygen, reducing the reliance on packaged foods, reducing resupply needs, and extending the duration of missions.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="FROM DUST TO SEED: A LUNAR CHICKPEA STORY
Authors: Atkin, J.; Oliveira Pedro dos Santos, S.
Score: 114.2, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576311
Food sustainability is one of the most significant barriers to long-term space travel. Providing resources from Earth is not cost-efficient, and resupply missions are not viable to meet the needs of long-term life in deep space conditions. Plants in space can provide a source of nutrition and oxygen, reducing the reliance on packaged foods, reducing resupply needs, and extending the duration of missions." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-04T10:38:39+00:00" />
<meta property="article:modified_time" content="2024-02-04T10:38:39+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="FROM DUST TO SEED: A LUNAR CHICKPEA STORY
Authors: Atkin, J.; Oliveira Pedro dos Santos, S.
Score: 114.2, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576311
Food sustainability is one of the most significant barriers to long-term space travel. Providing resources from Earth is not cost-efficient, and resupply missions are not viable to meet the needs of long-term life in deep space conditions. Plants in space can provide a source of nutrition and oxygen, reducing the reliance on packaged foods, reducing resupply needs, and extending the duration of missions."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "FROM DUST TO SEED: A LUNAR CHICKPEA STORY\nAuthors: Atkin, J.; Oliveira Pedro dos Santos, S.\nScore: 114.2, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576311\nFood sustainability is one of the most significant barriers to long-term space travel. Providing resources from Earth is not cost-efficient, and resupply missions are not viable to meet the needs of long-term life in deep space conditions. Plants in space can provide a source of nutrition and oxygen, reducing the reliance on packaged foods, reducing resupply needs, and extending the duration of missions.",
  "keywords": [
    
  ],
  "articleBody": " FROM DUST TO SEED: A LUNAR CHICKPEA STORY\nAuthors: Atkin, J.; Oliveira Pedro dos Santos, S.\nScore: 114.2, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576311\nFood sustainability is one of the most significant barriers to long-term space travel. Providing resources from Earth is not cost-efficient, and resupply missions are not viable to meet the needs of long-term life in deep space conditions. Plants in space can provide a source of nutrition and oxygen, reducing the reliance on packaged foods, reducing resupply needs, and extending the duration of missions. Using lunar regolith simulant, we employ a novel methodology to create a sustainable and productive growth medium to support the cultivation of horticultural crops on the Moon. Implementing microbial soil regeneration mechanisms derived from Earth, we leverage the interaction between Arbuscular Mycorrhizal Fungi (AMF) and Vermicompost (VC) to create a fertile LRS matrix. These amendments can sequester toxic contaminants, improve soil structure, and increase plant stress tolerance. We demonstrate the ability to produce chickpea (Cicer arietinum) in lunar regolith simulant augmented with AMF and VC under climate-controlled conditions. We cultivated chickpea to seed in a mixture containing 75% Lunar Regolith Simulant. Preliminary results suggest that higher LRS contents induce heightened stress responses. However, plants grown in 100% LRS inoculated with arbuscular mycorrhizal fungi demonstrated an average two-week survival extension compared to non-inoculated plants. This study provides, for the first time, a baseline for chickpea germination in varying mixtures of LRS and VC and will inform future studies as humanity goes back to the Moon.\nMEDIATOR SUBUNIT 25 modulates ERFVII-controlled hypoxia responses in Arabidopsis\nAuthors: Schippers, J.; von Bongartz, K.; Laritzki, L.; Frohn, S.; Frings, S.; Renziehausen, T.; Augstein, F.; Winkels, K.; Sprangers, K.; Sasidharan, R.; Vertommen, D.; van Breusegem, F.; Hartman, S.; Beemster, G. T. S.; Mhamdi, A.; van Dongen, J. T.; Schmidt-Schippers, R.\nScore: 18.2, Published: 2024-01-29 DOI: 10.1101/2024.01.26.577166\nFlooding impairs plant growth through oxygen deprivation, which activates plant survival and acclimation responses. Low-oxygen responses are generally associated with activation of group VII ETHYLENE-RESPONSE FACTOR (ERFVII) transcription factors. However, mechanism and molecular components by which ERFVII factors initiate gene expression are not fully elucidated. Here, we show that the Mediator complex subunit AtMED25 is recruited by RELATED TO APETALA 2.2 (RAP2.2) and RAP2.12 to coordinate gene expression during hypoxia in Arabidopsis thaliana.. The med25 mutants display reduced low-oxygen stress tolerance. AtMED25 associates with several ERFVII-controlled hypoxia core genes and its loss impairs transcription under hypoxia due to decreasing RNA polymerase II recruitment. Protein complex pulldown assays demonstrate that the Mediator complex built around AtMED25 is adjusted under low-oxygen conditions. Moreover, during hypoxia, no functional cooperation between AtMED25 and the two subunits AtMED8 and AtMED16 occurs, contrasting previous observations made for other conditions. In addition, AtMED25 function under hypoxia is independent from ethylene signalling. Finally, a functional conservation at the molecular level was found for the MED25-ERFVII module between Arabidopsis thaliana and the monocot Oryza sativa, pointing to a potentially universal role of MED25 in enabling ERFVII-dependent transcript responses to hypoxia in plants.\nThe molecular core of transcriptome responses to abiotic stress in plants: a machine learning-driven meta-analysis\nAuthors: Sanchez-Munoz, R.; Depaepe, T.; Samalova, M.; Hejatko, J.; Zaplana, I.; Van Der Straeten, D.\nScore: 18.8, Published: 2024-01-26 DOI: 10.1101/2024.01.24.576978\nUnderstanding how plants adapt their physiology to overcome severe stress conditions is vital in light of the current climate crisis. This remains a challenge given the complex nature of the underlying molecular mechanisms. To provide a full picture of stress mitigation mechanisms, an exhaustive analysis of publicly available stress-related transcriptomic data was conducted. We combined a meta-analysis with an unsupervised machine learning algorithm to identify a core of stress-related genes. To ensure robustness and biological significance of the output, often lacking in meta-analyses, a three-layered biovalidation was incorporated. Our results present a stress gene core, a set of key genes involved in plant tolerance to a multitude of adverse environmental conditions rather than specific ones. In addition, we provide a biologically validated database to assist in design of multi-stress resilience. Taken together, our results pave the way towards future-proof sustainable agriculture. TeaserUsing a machine learning-driven meta-analysis, a plant stress gene core was identified as a hub mediating multi-stress regulation\nModular mechanisms of immune priming and growth inhibition mediated by plant effector-triggered immunity\nAuthors: Chhillar, H.; Yeh, P.-M.; Nguyen, H. H.; Jones, J. D.; Ding, P.\nScore: 13.8, Published: 2024-02-01 DOI: 10.1101/2024.02.01.578334\nExcessive activation of effector-triggered immunity (ETI) in plants inhibits plant growth and activates cell death. ETI mediated by intracellular Toll/Interleukin-1 receptor/Resistance protein (TIR) nucleotide-binding leucine-rich-repeat receptors (NLRs) involves two partially redundant signalling nodes in Arabidopsis, EDS1-PAD4-ADR1 and EDS1-SAG101-NRG1. Genetic and transcriptomic analyses show that EDS1-PAD4-ADR1 primarily enhances the immune component abundance and is critical for limiting pathogen growth, whereas EDS1-SAG101-NRG1 mainly activates the hypersensitive cell death response (HR) but is dispensable for immune priming. This study enhances our understanding of the distinct contributions of these two signalling modules to ETI and suggests potential strategies for improving disease resistance in crops without compromising yield.\nA shade-responsive microProtein in the Arabidopsis ATHB2 gene regulates elongation growth and root development\nAuthors: Edwards, A. K.; Chiurazzi, M. J.; Blaakmeer, A.; Vittozzi, Y.; Sharma, A.; Matton, S.; Kruusvee, V.; Straub, D.; Sessa, G.; Carabelli, M.; Morelli, G.; Wenkel, S.\nScore: 12.5, Published: 2024-02-01 DOI: 10.1101/2024.02.01.578400\nThe ability of plants to thrive under suboptimal light conditions, such as shade, is crucial for their overall survival and reproductive success. Here, we show that Arabidopsis seedlings produce a large number of alternative transcripts when exposed to shade. Notably, one of the identified transcript candidates, which was upregulated in shade conditions, was found to be an alternative transcript of the ATHB2 gene. ATHB2 belongs to the HD-ZIPII class of transcription factors and is a well-established regulator of the shade avoidance response. The function of the alternative transcript and the small leucine zipper protein encoded by it, ATHB2miP, was investigated. We found that ATHB2miP is primarily expressed in the shoot meristem and interacts with full-length ATHB2 protein to inhibit its activity through a negative feedback mechanism. Deletion of the genomic region encoding the leucine zipper domain of the ATHB2 gene using CRISPR, resulted in plants exhibiting altered shade avoidance responses and root development. We show that the leucine zipper domain is required for dimerising and localising to nuclear photobodies. There is a significant overlap in deregulated genes between plants ectopically expressing ATHB2miP and athb2 mutant plants. The analysis of gene ontology and clustering revealed that the most affected processes are auxin synthesis and signaling, root development, and iron homeostasis. Shade growth experiments at different iron concentrations revealed a role for ATHB2 in regulating iron uptake and showed that iron availability affects shade growth in an ATHB2-dependent manner. This study identifies ATHB2miP as a novel regulator of shade avoidance responses in Arabidopsis, highlighting the intricate transcriptional regulation underlying these processes.\nSingle-Cell RNA-seq based elucidation of the antidepressant hyperforin biosynthesis de novo in St. John's wort.\nAuthors: Wu, S.; Morotti, A. L. M.; Yang, J.; Wang, E.; Tatsis, E. C.\nScore: 12.1, Published: 2024-01-31 DOI: 10.1101/2024.01.24.577018\nHyperforin is the compound responsible for the effectiveness of St. Johns wort (Hypericum perforatum) as an antidepressant, but its biosynthesis remains unknown. Gene discovery based on co-expression analysis of bulk RNA-sequencing data or genome mining failed to discover the missing steps in hyperforin biosynthesis. Here we sequenced the tetraploid H. perforatum genome. By single-cell RNA-seq, we identified a distinct type of cells, Hyper cells, wherein hyperforin biosynthesis de novo takes place. Through pathway reconstitution in yeast and tobacco, we identify and characterize four transmembrane prenyltransferases to resolve hyperforin biosynthesis. The hyperforin polycyclic scaffold is created by a reaction cascade involving an irregular isoprenoid coupling and a tandem cyclization. Our findings reveal how and where hyperforin is biosynthesized that enables synthetic-biology reconstitution of the complete pathway. These results deepen our comprehension of specialized metabolism at the cellular level, and we anticipate acceleration of pathway elucidation in plant metabolism.\nMorphological and genetic analysis of inflorescence and flower development in hemp (Cannabis sativa L.)\nAuthors: Shi, J.; Schilling, S.; Melzer, R.\nScore: 7.6, Published: 2024-01-29 DOI: 10.1101/2024.01.25.577276\nHemp (Cannabis sativa) is one of the oldest cultivated crops, and its myriad of uses have fascinated humans for millennia. In contemporary agriculture, the seeds are used for high-quality oil, human food, and nutritional supplements. From female inflorescences, secondary compounds (cannabinoids, terpenes, and flavonoids) are isolated that are implicated to have a wide range of medicinal applications. These elements provide hemp with significant market and research value and emphasise the need to thoroughly understand reproductive development in hemp. Here, we present a detailed morphological and molecular analysis of hemp inflorescence and flower development of the cultivar FINOLA. Hemp is unusual among flowering plants as it is dioecious, i.e. develops male and female plants. We define eight landmark events in male and female flower development and show that developmental differences between male and female plants extend beyond floral morphology, additionally comprising inflorescence structure as well as flowering time. Further, we demonstrate that the time of activation of key reproductive developmental regulators is significantly different in male and female hemp plants. Our comparison of male and female hemp plants shows that developmental pathways diverge very early, already at the two-leaf stage, laying a basis for further exploration into the genetics of sex determination of C. sativa.\nGenome-Wide Profiling of Soybean WRINKLED1 Transcription Factor Binding Sites Provides Insight into the Regulation of Fatty Acid and Triacylglycerol Biosynthesis Program in Seeds\nAuthors: Jo, L.; Pelletier, J.; Harada, J. J.\nScore: 8.2, Published: 2024-01-26 DOI: 10.1101/2024.01.23.576967\nUnderstanding the regulatory mechanisms controlling seed lipid accumulation provides opportunities for the development of new crops with enhanced oil levels. The WRINKLED1 transcription factor (WRI1 TF) is a central regulator of lipid biosynthesis in many plant species. We characterized the genome-wide binding profile of soybean (Gm)WRI1 and showed that the TF directly regulates many genes required for the fatty acid and triacylglycerol biosynthesis pathway. We showed that GmWRI1 binds primarily to regions downstream of target gene transcription start sites. Regions bound by GmWRI1 were enriched for its canonical DNA binding element, the AW Box (CNTNGNNNNNNNCG), and another DNA element, the CNC Box (CNCCNCC), that shows some similarity to other characterized DNA motifs. Functional assays showed that both DNA elements are required to mediate transcriptional activation by GmWRI1. Comparison of genes targeted directly by GmWRI1 and by GmLEC1, a central regulator of the maturation phase of seed development, revealed that the two TFs act in a positive feedback subcircuit to control fatty acid and triacylglycerol biosynthesis. Our results provide new insights into the genetic circuitry by which GmWRI1 regulates fatty acid and triacylglycerol biosynthetic genes, and provides evidence that GmWRI1 contributes to establishing the regulatory state of the soybean embryo to promote oil accumulation.\nLeveraging genomics and temporal high-throughput phenotyping to enhance association mapping and yield prediction in sesame\nAuthors: Sabag, I.; Bi, Y.; sahoo, M. M.; Herrmann, I.; Morota, G.; Peleg, Z.\nScore: 6.6, Published: 2024-02-01 DOI: 10.1101/2024.02.01.578346\nSesame (Sesamum indicum) is an important oilseed crop with rising demand due to its high oil quality. To meet these future demands, there is an urgent need to develop and integrate new breeding strategies. While genomic resources have advanced genetic research in sesame, implementation of high-throughput phenotyping and genetic analysis of longitudinal traits remains limited. Here, we combined high-throughput phenotyping and random regression models to investigate the dynamics of plant height, leaf area index, and five spectral vegetation indices throughout the sesame growing seasons in a diversity panel. Modeling the temporal phenotypic and additive genetic trajectories revealed distinct patterns corresponding to the sesame growth cycle. We also conducted longitudinal genomic prediction and association mapping of plant height using various models and cross-validation schemes. Moderate prediction accuracy was obtained when predicting new genotypes at each time point, and moderate to high values were obtained when forecasting future phenotypes. Association mapping revealed three genomic regions in linkage groups 6, 8, and 11 conferring trait variation over time and growth rate. Furthermore, we leveraged correlations between the temporal trait and seed-yield and applied multi-trait genomic prediction. We obtained an improvement over single-trait analysis, especially when phenotypes from earlier time points were used, highlighting the potential of using a high-throughput phenotyping platform as a selection tool. Our results shed light on the genetic control of longitudinal traits in sesame and underscore the potential of high-throughput phenotyping to detect a wide range of traits and genotypes that can inform sesame breeding efforts to enhance yield.\nThe endophytic fungus Serendipita indica alters auxin distribution in Arabidopsis thaliana roots through alteration of auxin transport and conjugation to promote plant growth\nAuthors: Gonzalez Ortega-Villaizan, A.; King, E.; Patel, M. K.; PEREZ ALONSO, M. M.; Scholz, S. S.; Sakakibara, H.; Kiba, T.; Kojima, M.; Takebayashi, Y.; Ramos, P.; Morales-Quintana, L.; Breitenbach, S.; Smolko, A.; Salopek-Sondi, B.; Bauer, N.; Müller-Ludwig, J.; Krapp, A.; Oelmüller, R.; Vicente-Carbajosa, J.; Pollmann, S.\nScore: 10.7, Published: 2024-01-26 DOI: 10.1101/2024.01.23.576781\nPlants share their habitats with a multitude of different microbes. This close vicinity promoted the evolution of inter-organismic interactions between plants and many different microorganisms that provide mutual growth benefits both to the plant and the microbial partner. The symbiosis of Arabidopsis thaliana with the beneficial root colonizing endophyte Serendipita indica represents a well-studied system. Co-colonization of Arabidopsis roots with S. indica significantly promotes plant growth. Due to the notable phenotypic alterations of fungus-infected root systems, the involvement of a reprogramming of plant hormone levels, especially that of indole-3-acetic acid, has been suggested earlier. However, until now, the molecular mechanism by which S. indica promotes plant growth remains largely unknown. This study used comprehensive transcriptomics, metabolomics, reverse genetics, and life cell imaging to reveal the intricacies of auxin-related processes that affect root growth in the symbiosis between A. thaliana and S. indica. Our experiments revealed the essential role of tightly controlled auxin conjugation in the plant-fungus interaction. It particularly highlighted the importance of two GRETCHEN HAGEN 3 (GH3) genes, GH3.5 and GH3.17, for the fungus infection-triggered stimulation of biomass production, thus broadening our knowledge about the function of GH3s in plants. Furthermore, we provide evidence for the transcriptional alteration of the PIN2 auxin transporter gene in roots of Arabidopsis seedlings infected with S. indica and demonstrate that this transcriptional adjustment affects auxin signaling in roots, which results in increased plant growth.\n",
  "wordCount" : "2378",
  "inLanguage": "en",
  "datePublished": "2024-02-04T10:38:39Z",
  "dateModified": "2024-02-04T10:38:39Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 4, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576311">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576311" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576311">
        <p class="paperTitle">FROM DUST TO SEED: A LUNAR CHICKPEA STORY</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576311" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576311" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Atkin, J.; Oliveira Pedro dos Santos, S.</p>
        <p class="info">Score: 114.2, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576311' target='https://doi.org/10.1101/2024.01.18.576311'> 10.1101/2024.01.18.576311</a></p>
        <p class="abstract">Food sustainability is one of the most significant barriers to long-term space travel. Providing resources from Earth is not cost-efficient, and resupply missions are not viable to meet the needs of long-term life in deep space conditions. Plants in space can provide a source of nutrition and oxygen, reducing the reliance on packaged foods, reducing resupply needs, and extending the duration of missions. Using lunar regolith simulant, we employ a novel methodology to create a sustainable and productive growth medium to support the cultivation of horticultural crops on the Moon. Implementing microbial soil regeneration mechanisms derived from Earth, we leverage the interaction between Arbuscular Mycorrhizal Fungi (AMF) and Vermicompost (VC) to create a fertile LRS matrix. These amendments can sequester toxic contaminants, improve soil structure, and increase plant stress tolerance. We demonstrate the ability to produce chickpea (Cicer arietinum) in lunar regolith simulant augmented with AMF and VC under climate-controlled conditions. We cultivated chickpea to seed in a mixture containing 75% Lunar Regolith Simulant. Preliminary results suggest that higher LRS contents induce heightened stress responses. However, plants grown in 100% LRS inoculated with arbuscular mycorrhizal fungi demonstrated an average two-week survival extension compared to non-inoculated plants. This study provides, for the first time, a baseline for chickpea germination in varying mixtures of LRS and VC and will inform future studies as humanity goes back to the Moon.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577166">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577166" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577166">
        <p class="paperTitle">MEDIATOR SUBUNIT 25 modulates ERFVII-controlled hypoxia responses in Arabidopsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577166" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577166" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schippers, J.; von Bongartz, K.; Laritzki, L.; Frohn, S.; Frings, S.; Renziehausen, T.; Augstein, F.; Winkels, K.; Sprangers, K.; Sasidharan, R.; Vertommen, D.; van Breusegem, F.; Hartman, S.; Beemster, G. T. S.; Mhamdi, A.; van Dongen, J. T.; Schmidt-Schippers, R.</p>
        <p class="info">Score: 18.2, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577166' target='https://doi.org/10.1101/2024.01.26.577166'> 10.1101/2024.01.26.577166</a></p>
        <p class="abstract">Flooding impairs plant growth through oxygen deprivation, which activates plant survival and acclimation responses. Low-oxygen responses are generally associated with activation of group VII ETHYLENE-RESPONSE FACTOR (ERFVII) transcription factors. However, mechanism and molecular components by which ERFVII factors initiate gene expression are not fully elucidated. Here, we show that the Mediator complex subunit AtMED25 is recruited by RELATED TO APETALA 2.2 (RAP2.2) and RAP2.12 to coordinate gene expression during hypoxia in Arabidopsis thaliana.. The med25 mutants display reduced low-oxygen stress tolerance. AtMED25 associates with several ERFVII-controlled hypoxia core genes and its loss impairs transcription under hypoxia due to decreasing RNA polymerase II recruitment. Protein complex pulldown assays demonstrate that the Mediator complex built around AtMED25 is adjusted under low-oxygen conditions. Moreover, during hypoxia, no functional cooperation between AtMED25 and the two subunits AtMED8 and AtMED16 occurs, contrasting previous observations made for other conditions. In addition, AtMED25 function under hypoxia is independent from ethylene signalling. Finally, a functional conservation at the molecular level was found for the MED25-ERFVII module between Arabidopsis thaliana and the monocot Oryza sativa, pointing to a potentially universal role of MED25 in enabling ERFVII-dependent transcript responses to hypoxia in plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.576978">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.576978" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.576978">
        <p class="paperTitle">The molecular core of transcriptome responses to abiotic stress in plants: a machine learning-driven meta-analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.576978" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.576978" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanchez-Munoz, R.; Depaepe, T.; Samalova, M.; Hejatko, J.; Zaplana, I.; Van Der Straeten, D.</p>
        <p class="info">Score: 18.8, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.576978' target='https://doi.org/10.1101/2024.01.24.576978'> 10.1101/2024.01.24.576978</a></p>
        <p class="abstract">Understanding how plants adapt their physiology to overcome severe stress conditions is vital in light of the current climate crisis. This remains a challenge given the complex nature of the underlying molecular mechanisms. To provide a full picture of stress mitigation mechanisms, an exhaustive analysis of publicly available stress-related transcriptomic data was conducted. We combined a meta-analysis with an unsupervised machine learning algorithm to identify a core of stress-related genes. To ensure robustness and biological significance of the output, often lacking in meta-analyses, a three-layered biovalidation was incorporated. Our results present a  stress gene core, a set of key genes involved in plant tolerance to a multitude of adverse environmental conditions rather than specific ones. In addition, we provide a biologically validated database to assist in design of multi-stress resilience. Taken together, our results pave the way towards future-proof sustainable agriculture.

TeaserUsing a machine learning-driven meta-analysis, a plant  stress gene core was identified as a hub mediating multi-stress regulation</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578334">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578334" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578334">
        <p class="paperTitle">Modular mechanisms of immune priming and growth inhibition mediated by plant effector-triggered immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578334" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578334" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chhillar, H.; Yeh, P.-M.; Nguyen, H. H.; Jones, J. D.; Ding, P.</p>
        <p class="info">Score: 13.8, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578334' target='https://doi.org/10.1101/2024.02.01.578334'> 10.1101/2024.02.01.578334</a></p>
        <p class="abstract">Excessive activation of effector-triggered immunity (ETI) in plants inhibits plant growth and activates cell death. ETI mediated by intracellular Toll/Interleukin-1 receptor/Resistance protein (TIR) nucleotide-binding leucine-rich-repeat receptors (NLRs) involves two partially redundant signalling nodes in Arabidopsis, EDS1-PAD4-ADR1 and EDS1-SAG101-NRG1. Genetic and transcriptomic analyses show that EDS1-PAD4-ADR1 primarily enhances the immune component abundance and is critical for limiting pathogen growth, whereas EDS1-SAG101-NRG1 mainly activates the hypersensitive cell death response (HR) but is dispensable for immune priming. This study enhances our understanding of the distinct contributions of these two signalling modules to ETI and suggests potential strategies for improving disease resistance in crops without compromising yield.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578400">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578400" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578400">
        <p class="paperTitle">A shade-responsive microProtein in the Arabidopsis ATHB2 gene regulates elongation growth and root development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578400" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578400" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Edwards, A. K.; Chiurazzi, M. J.; Blaakmeer, A.; Vittozzi, Y.; Sharma, A.; Matton, S.; Kruusvee, V.; Straub, D.; Sessa, G.; Carabelli, M.; Morelli, G.; Wenkel, S.</p>
        <p class="info">Score: 12.5, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578400' target='https://doi.org/10.1101/2024.02.01.578400'> 10.1101/2024.02.01.578400</a></p>
        <p class="abstract">The ability of plants to thrive under suboptimal light conditions, such as shade, is crucial for their overall survival and reproductive success. Here, we show that Arabidopsis seedlings produce a large number of alternative transcripts when exposed to shade. Notably, one of the identified transcript candidates, which was upregulated in shade conditions, was found to be an alternative transcript of the ATHB2 gene. ATHB2 belongs to the HD-ZIPII class of transcription factors and is a well-established regulator of the shade avoidance response. The function of the alternative transcript and the small leucine zipper protein encoded by it, ATHB2miP, was investigated. We found that ATHB2miP is primarily expressed in the shoot meristem and interacts with full-length ATHB2 protein to inhibit its activity through a negative feedback mechanism. Deletion of the genomic region encoding the leucine zipper domain of the ATHB2 gene using CRISPR, resulted in plants exhibiting altered shade avoidance responses and root development. We show that the leucine zipper domain is required for dimerising and localising to nuclear photobodies. There is a significant overlap in deregulated genes between plants ectopically expressing ATHB2miP and athb2 mutant plants. The analysis of gene ontology and clustering revealed that the most affected processes are auxin synthesis and signaling, root development, and iron homeostasis. Shade growth experiments at different iron concentrations revealed a role for ATHB2 in regulating iron uptake and showed that iron availability affects shade growth in an ATHB2-dependent manner. This study identifies ATHB2miP as a novel regulator of shade avoidance responses in Arabidopsis, highlighting the intricate transcriptional regulation underlying these processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.577018">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.577018" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.577018">
        <p class="paperTitle">Single-Cell RNA-seq based elucidation of the antidepressant hyperforin biosynthesis de novo in St. John&#39;s wort.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.577018" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.577018" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wu, S.; Morotti, A. L. M.; Yang, J.; Wang, E.; Tatsis, E. C.</p>
        <p class="info">Score: 12.1, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.577018' target='https://doi.org/10.1101/2024.01.24.577018'> 10.1101/2024.01.24.577018</a></p>
        <p class="abstract">Hyperforin is the compound responsible for the effectiveness of St. Johns wort (Hypericum perforatum) as an antidepressant, but its biosynthesis remains unknown. Gene discovery based on co-expression analysis of bulk RNA-sequencing data or genome mining failed to discover the missing steps in hyperforin biosynthesis. Here we sequenced the tetraploid H. perforatum genome. By single-cell RNA-seq, we identified a distinct type of cells, Hyper cells, wherein hyperforin biosynthesis de novo takes place. Through pathway reconstitution in yeast and tobacco, we identify and characterize four transmembrane prenyltransferases to resolve hyperforin biosynthesis. The hyperforin polycyclic scaffold is created by a reaction cascade involving an irregular isoprenoid coupling and a tandem cyclization. Our findings reveal how and where hyperforin is biosynthesized that enables synthetic-biology reconstitution of the complete pathway. These results deepen our comprehension of specialized metabolism at the cellular level, and we anticipate acceleration of pathway elucidation in plant metabolism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.577276">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.577276" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.577276">
        <p class="paperTitle">Morphological and genetic analysis of inflorescence and flower development in hemp (Cannabis sativa L.)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.577276" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.577276" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shi, J.; Schilling, S.; Melzer, R.</p>
        <p class="info">Score: 7.6, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.577276' target='https://doi.org/10.1101/2024.01.25.577276'> 10.1101/2024.01.25.577276</a></p>
        <p class="abstract">Hemp (Cannabis sativa) is one of the oldest cultivated crops, and its myriad of uses have fascinated humans for millennia. In contemporary agriculture, the seeds are used for high-quality oil, human food, and nutritional supplements. From female inflorescences, secondary compounds (cannabinoids, terpenes, and flavonoids) are isolated that are implicated to have a wide range of medicinal applications. These elements provide hemp with significant market and research value and emphasise the need to thoroughly understand reproductive development in hemp.

Here, we present a detailed morphological and molecular analysis of hemp inflorescence and flower development of the cultivar  FINOLA. Hemp is unusual among flowering plants as it is dioecious, i.e. develops male and female plants. We define eight landmark events in male and female flower development and show that developmental differences between male and female plants extend beyond floral morphology, additionally comprising inflorescence structure as well as flowering time. Further, we demonstrate that the time of activation of key reproductive developmental regulators is significantly different in male and female hemp plants. Our comparison of male and female hemp plants shows that developmental pathways diverge very early, already at the two-leaf stage, laying a basis for further exploration into the genetics of sex determination of C. sativa.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576967">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576967" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576967">
        <p class="paperTitle">Genome-Wide Profiling of Soybean WRINKLED1 Transcription Factor Binding Sites Provides Insight into the Regulation of Fatty Acid and Triacylglycerol Biosynthesis Program in Seeds</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576967" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576967" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jo, L.; Pelletier, J.; Harada, J. J.</p>
        <p class="info">Score: 8.2, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576967' target='https://doi.org/10.1101/2024.01.23.576967'> 10.1101/2024.01.23.576967</a></p>
        <p class="abstract">Understanding the regulatory mechanisms controlling seed lipid accumulation provides opportunities for the development of new crops with enhanced oil levels. The WRINKLED1 transcription factor (WRI1 TF) is a central regulator of lipid biosynthesis in many plant species. We characterized the genome-wide binding profile of soybean (Gm)WRI1 and showed that the TF directly regulates many genes required for the fatty acid and triacylglycerol biosynthesis pathway. We showed that GmWRI1 binds primarily to regions downstream of target gene transcription start sites. Regions bound by GmWRI1 were enriched for its canonical DNA binding element, the AW Box (CNTNGNNNNNNNCG), and another DNA element, the CNC Box (CNCCNCC), that shows some similarity to other characterized DNA motifs. Functional assays showed that both DNA elements are required to mediate transcriptional activation by GmWRI1. Comparison of genes targeted directly by GmWRI1 and by GmLEC1, a central regulator of the maturation phase of seed development, revealed that the two TFs act in a positive feedback subcircuit to control fatty acid and triacylglycerol biosynthesis. Our results provide new insights into the genetic circuitry by which GmWRI1 regulates fatty acid and triacylglycerol biosynthetic genes, and provides evidence that GmWRI1 contributes to establishing the regulatory state of the soybean embryo to promote oil accumulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578346">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578346" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578346">
        <p class="paperTitle">Leveraging genomics and temporal high-throughput phenotyping to enhance association mapping and yield prediction in sesame</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578346" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578346" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sabag, I.; Bi, Y.; sahoo, M. M.; Herrmann, I.; Morota, G.; Peleg, Z.</p>
        <p class="info">Score: 6.6, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578346' target='https://doi.org/10.1101/2024.02.01.578346'> 10.1101/2024.02.01.578346</a></p>
        <p class="abstract">Sesame (Sesamum indicum) is an important oilseed crop with rising demand due to its high oil quality. To meet these future demands, there is an urgent need to develop and integrate new breeding strategies. While genomic resources have advanced genetic research in sesame, implementation of high-throughput phenotyping and genetic analysis of longitudinal traits remains limited. Here, we combined high-throughput phenotyping and random regression models to investigate the dynamics of plant height, leaf area index, and five spectral vegetation indices throughout the sesame growing seasons in a diversity panel. Modeling the temporal phenotypic and additive genetic trajectories revealed distinct patterns corresponding to the sesame growth cycle. We also conducted longitudinal genomic prediction and association mapping of plant height using various models and cross-validation schemes. Moderate prediction accuracy was obtained when predicting new genotypes at each time point, and moderate to high values were obtained when forecasting future phenotypes. Association mapping revealed three genomic regions in linkage groups 6, 8, and 11 conferring trait variation over time and growth rate. Furthermore, we leveraged correlations between the temporal trait and seed-yield and applied multi-trait genomic prediction. We obtained an improvement over single-trait analysis, especially when phenotypes from earlier time points were used, highlighting the potential of using a high-throughput phenotyping platform as a selection tool. Our results shed light on the genetic control of longitudinal traits in sesame and underscore the potential of high-throughput phenotyping to detect a wide range of traits and genotypes that can inform sesame breeding efforts to enhance yield.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576781">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576781" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576781">
        <p class="paperTitle">The endophytic fungus Serendipita indica alters auxin distribution in Arabidopsis thaliana roots through alteration of auxin transport and conjugation to promote plant growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576781" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576781" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gonzalez Ortega-Villaizan, A.; King, E.; Patel, M. K.; PEREZ ALONSO, M. M.; Scholz, S. S.; Sakakibara, H.; Kiba, T.; Kojima, M.; Takebayashi, Y.; Ramos, P.; Morales-Quintana, L.; Breitenbach, S.; Smolko, A.; Salopek-Sondi, B.; Bauer, N.; Müller-Ludwig, J.; Krapp, A.; Oelmüller, R.; Vicente-Carbajosa, J.; Pollmann, S.</p>
        <p class="info">Score: 10.7, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576781' target='https://doi.org/10.1101/2024.01.23.576781'> 10.1101/2024.01.23.576781</a></p>
        <p class="abstract">Plants share their habitats with a multitude of different microbes. This close vicinity promoted the evolution of inter-organismic interactions between plants and many different microorganisms that provide mutual growth benefits both to the plant and the microbial partner. The symbiosis of Arabidopsis thaliana with the beneficial root colonizing endophyte Serendipita indica represents a well-studied system. Co-colonization of Arabidopsis roots with S. indica significantly promotes plant growth. Due to the notable phenotypic alterations of fungus-infected root systems, the involvement of a reprogramming of plant hormone levels, especially that of indole-3-acetic acid, has been suggested earlier. However, until now, the molecular mechanism by which S. indica promotes plant growth remains largely unknown. This study used comprehensive transcriptomics, metabolomics, reverse genetics, and life cell imaging to reveal the intricacies of auxin-related processes that affect root growth in the symbiosis between A. thaliana and S. indica. Our experiments revealed the essential role of tightly controlled auxin conjugation in the plant-fungus interaction. It particularly highlighted the importance of two GRETCHEN HAGEN 3 (GH3) genes, GH3.5 and GH3.17, for the fungus infection-triggered stimulation of biomass production, thus broadening our knowledge about the function of GH3s in plants. Furthermore, we provide evidence for the transcriptional alteration of the PIN2 auxin transporter gene in roots of Arabidopsis seedlings infected with S. indica and demonstrate that this transcriptional adjustment affects auxin signaling in roots, which results in increased plant growth.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
