<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Near-infrared imaging of phytochrome-derived autofluorescence in plant nuclei
Authors: Yoshinari, A.; Isoda, R.; Yagi, N.; Sato, Y.; Lindeboom, J. J.; Ehrhardt, D. W.; Frommer, W. B.; Nakamura, M.
Score: 25.0, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558255
Capturing images of the nuclear dynamics within live cells is an essential technique for comprehending the intricate biological processes inherent to plant cell nuclei. While various methods exist for imaging nuclei, including combining fluorescent proteins and dyes with microscopy, there is a dearth of commercially available dyes for live-cell imaging.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="Near-infrared imaging of phytochrome-derived autofluorescence in plant nuclei
Authors: Yoshinari, A.; Isoda, R.; Yagi, N.; Sato, Y.; Lindeboom, J. J.; Ehrhardt, D. W.; Frommer, W. B.; Nakamura, M.
Score: 25.0, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558255
Capturing images of the nuclear dynamics within live cells is an essential technique for comprehending the intricate biological processes inherent to plant cell nuclei. While various methods exist for imaging nuclei, including combining fluorescent proteins and dyes with microscopy, there is a dearth of commercially available dyes for live-cell imaging." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-01T10:37:47+00:00" />
<meta property="article:modified_time" content="2023-10-01T10:37:47+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="Near-infrared imaging of phytochrome-derived autofluorescence in plant nuclei
Authors: Yoshinari, A.; Isoda, R.; Yagi, N.; Sato, Y.; Lindeboom, J. J.; Ehrhardt, D. W.; Frommer, W. B.; Nakamura, M.
Score: 25.0, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558255
Capturing images of the nuclear dynamics within live cells is an essential technique for comprehending the intricate biological processes inherent to plant cell nuclei. While various methods exist for imaging nuclei, including combining fluorescent proteins and dyes with microscopy, there is a dearth of commercially available dyes for live-cell imaging."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "Near-infrared imaging of phytochrome-derived autofluorescence in plant nuclei\nAuthors: Yoshinari, A.; Isoda, R.; Yagi, N.; Sato, Y.; Lindeboom, J. J.; Ehrhardt, D. W.; Frommer, W. B.; Nakamura, M.\nScore: 25.0, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558255\nCapturing images of the nuclear dynamics within live cells is an essential technique for comprehending the intricate biological processes inherent to plant cell nuclei. While various methods exist for imaging nuclei, including combining fluorescent proteins and dyes with microscopy, there is a dearth of commercially available dyes for live-cell imaging.",
  "keywords": [
    
  ],
  "articleBody": " Near-infrared imaging of phytochrome-derived autofluorescence in plant nuclei\nAuthors: Yoshinari, A.; Isoda, R.; Yagi, N.; Sato, Y.; Lindeboom, J. J.; Ehrhardt, D. W.; Frommer, W. B.; Nakamura, M.\nScore: 25.0, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558255\nCapturing images of the nuclear dynamics within live cells is an essential technique for comprehending the intricate biological processes inherent to plant cell nuclei. While various methods exist for imaging nuclei, including combining fluorescent proteins and dyes with microscopy, there is a dearth of commercially available dyes for live-cell imaging. In Arabidopsis thaliana, we discovered that nuclei emit autofluorescence in the near-infrared (NIR) range of the spectrum and devised a non-invasive technique for the visualization of live cell nuclei using this inherent NIR autofluorescence. Our studies demonstrated the capability of the NIR imaging technique to visualize the dynamic behavior of nuclei within primary roots, root hairs, and pollen tubes, which are tissues that harbor a limited number of other organelles displaying autofluorescence. We further demonstrated the applicability of NIR autofluorescence imaging in various other tissues by incorporating fluorescence lifetime imaging techniques. Nuclear autofluorescence was also detected across a wide range of plant species, enabling analyses without the need for transformation. The nuclear autofluorescence in the NIR wavelength range was not observed in animal or yeast cells. Genetic analysis revealed that this autofluorescence was caused by the phytochrome protein. Our studies demonstrated that nuclear autofluorescence imaging can be effectively employed not only in model plants but also for studying nuclei in non-model plant species.\nWhole mount multiplexed visualization of DNA, mRNA, and protein in plant-parasitic nematodes\nAuthors: Sperling, A. L.; Eves-van den Akker, S.\nScore: 19.7, Published: 2023-09-24 DOI: 10.1101/2023.09.23.559107\nBackgroundPlant-parasitic nematodes compromise the agriculture of a wide variety of the most common crops worldwide. Obtaining information on the fundamental biology of these organisms and how they infect the plant has been restricted by the ability to visualize intact nematodes using small molecule stains, antibodies, or in situ hybridization. Consequently, there is limited information available about the internal composition of the nematodes or the biology of the effector molecules they use to reprogram their host plant. ResultsWe present the Sperling prep -a whole mount method for nematode preparation that enables staining with small molecules, antibodies, or in situ hybridization chain reaction. This method does not require specialized apparatus and utilizes typical laboratory equipment and materials. By dissociating the strong cuticle and interior muscle layers, we enabled entry of the small molecule stains into the tissue. After permeabilization, small molecule stains can be used to visualize the nuclei with the DNA stain DAPI and the internal structures of the digestive tract and longitudinal musculature with the filamentous actin stain phalloidin. The permeabilization even allows entry of larger antibodies, albeit with lower efficiency. Finally, this method works exceptionally well with in situ HCR. Using this method, we have visualized effector transcripts specific to the dorsal gland and the subventral grand of the sugar beet cyst nematode, Heterodera schachtii, multiplexed in the same nematode. ConclusionWe were able to visualize the internal structures of the nematode as well as key effector transcripts that are used during plant infection and parasitism. Therefore, this method provides an important toolkit for studying the biology of plant-parasitic nematodes.\nEvolution of wheat blast resistance gene Rmg8 accompanied by differentiation of variants recognizing the powdery mildew fungus\nAuthors: Asuke, S.; Morita, K.; Shimizu, M.; Abe, F.; Terauchi, R.; Nago, C.; Takahashi, Y.; Shibata, M.; Yoshioka, M.; Iwakawa, M.; Kishi-Kaboshi, M.; Su, Z.; Nasuda, S.; Handa, H.; Fujita, M.; Tougou, M.; Hatta, K.; Mori, N.; Matsuoka, Y.; Kato, K.; Tosa, Y.\nScore: 18.6, Published: 2023-09-28 DOI: 10.1101/2023.09.26.559445\nWheat blast, a devastating disease having spread recently from South America to Asia and Africa, is caused by Pyricularia oryzae pathotype Triticum which emerged in 1985. Rmg8 and Rmg7, genes for resistance to wheat blast found in common wheat and tetraploid wheat, respectively, recognize the same avirulence gene, AVR-Rmg8. Here, we show an evolutionary process in which resistance gene(s), which had obtained an ability to recognize AVR-Rmg8 before the differentiation of Triticum and Aegilops, has expanded its target pathogens. Molecular cloning revealed that Rmg7 was one of alleles of Pm4 (Pm4a), a gene for resistance to wheat powdery mildew on 2AL, whereas Rmg8 was its homoeolog on 2BL ineffective against wheat powdery mildew. Rmg8 variants with the ability to recognize AVR-Rmg8 were distributed not only in Triticum spp. but also in Aegilops speltoides Ae. umbellulata, and Ae. comosa. This result suggests that the origin of resistance gene(s) recognizing AVR-Rmg8 dates back to the time before differentiation of A, B, S, U, and M genomes, that is, ~5 million years before the emergence of its current target, the wheat blast fungus. Phylogenetic analyses suggested that, in the evolutionary process thereafter, some of their variants gained the ability to recognize the wheat powdery mildew fungus and evolved into genes for resistance to wheat powdery mildew.\nSequence characterization of T, Bip, and Phbw demonstrates the role of MYB-bHLH-WD40 complexes and temperature in common bean seed color pattern formation\nAuthors: Parker, T.; Bolt, T.; Williams, T.; Penmetsa, R. V.; Mulube, M.; Palkovic, A.; Jochua, C. N.; Rubio Wilhelmi, M. d. M.; Lo, S.; Bornhorst, G.; Tian, L.; Kamfwa, K.; Hokin, S.; Farmer, A.; Diepenbrock, C. H.; Gepts, P.\nScore: 13.9, Published: 2023-09-29 DOI: 10.1101/2023.09.23.559151\nSeed colors and color patterns are critical for the survival of wild plants and the consumer appeal of crops. In common bean, a major global staple, these patterns are also critical for determining market classes, yet the genetic and environmental control of many pigmentation patterns remains unresolved. In this study, we genetically mapped variation for three important seed pattern loci, T, Bip, and phbw, which co-segregated completely with PvTTG1, PvMYC1, and PvTT8, respectively. Proteins encoded by these genes are predicted to work together in MYB-bHLH-WD40 (MBW) complexes, propagating flavonoid biosynthesis across the seed coat. Whole-genome sequencing of 37 diverse accessions identified putative mutations in each gene, including seven unique parallel mutations in T (PvTTG1) and a non-synonymous SNP in a conserved residue in bipana (PvMYC1). A 612 bp intron deletion in phbw (PvTT8) eliminated motifs conserved since the origins of the Papilionoidea and corresponded to a 20-fold reduction in transcript abundance. Mutations in MBW candidate genes for Z (PvTT2) and Sellatus (WDR) were also identified. In multi-location field trials with seven varieties with partial seed coat patterning, pigmented seed coat area was highly unstable and correlated with temperature, with up to 11-fold differences in pigmented area between the warmest and the coolest environments. In controlled growth chamber conditions, an increase of 4 {degrees}C was sufficient to cause pigmentation on an additional 21% of the seed coat area. Our results shed light on the fundamental activation of flavonoid biosynthesis in common bean and will be instrumental for maximizing consumer appeal in this nutritious staple crop. Summary- Seed colors and patterns are critical for the survival of wild plants, and are important in differentiating crop market classes, but the genetic control of these in the staple crop common bean (Phaseolus vulgaris) is largely unknown. - The genetic, transcriptional, and environmental basis of common bean seed color patterning was explored through QTL mapping, whole-genome sequencing, RT-qPCR, and automated pigmentation quantification of seed grown in multi-location field trials and growth chamber environments. - MYB-bHLH-WD40 complex-forming genes PvTTG1, PvMYC1, and PvTT8 co-segregated completely with the color patterning genes T, Bip, and phbw. Mutations were identified in each gene, including seven unique parallel mutations in T (PvTTG1), a non-synonymous SNP in a conserved residue in bipana (PvMYC1), and an intron deletion in phbw (PvTT8) eliminating highly conserved motifs and corresponding to 20-fold lower PvTT8 transcript abundance. Mutations in MBW candidate genes Z (PvTT2) and Sellatus (WDR) were also identified. In multi-location field trials, pigmented seed coat area was highly unstable and corresponded to temperature. In growth chamber conditions, an increase of 4 {degrees}C caused pigmentation on an additional 21% of the seed coat area. - Our results highlight the critical interaction between MYB-bHLH-WD40 complex components and temperature in establishing seed pattern diversity.\nThe wheat powdery mildew resistance gene Pm4 also confers resistance to wheat blast\nAuthors: O'Hara, T.; Steed, A.; Goddard, R.; Gaurav, K.; Arora, S.; Quiroz-Chavez, J.; Ramirez-Gonzalez, R.; Badgami, R.; Gilbert, D.; Sanchez-Martin, J.; Wingen, L.; Feng, C.; Jiang, M.; Cheng, S.; Dreisigacker, S.; Keller, B.; Wulff, B.; Uauy, C.; Nicholson, P.\nScore: 5.5, Published: 2023-09-28 DOI: 10.1101/2023.09.26.559489\nWheat blast, caused by the fungus Magnaporthe oryzae, threatens global cereal production since its emergence in Brazil in 1985 and recently spread to Bangladesh and Zambia. Here we demonstrate that the AVR-Rmg8 effector, common in wheat-infecting isolates, is recognised by the gene Pm4, previously shown to confer resistance to specific races of Blumeria graminis f.sp. tritici, the cause of powdery mildew of wheat. We show that Pm4 alleles differ in their recognition of different AVR-Rmg8 alleles, and some confer resistance only in seedling leaves but not spikes making it important to select for those alleles that function in both tissues. This study has identified a gene recognising an important virulence factor present in wheat blast isolates in Bangladesh and Zambia and represents an important first step towards developing durably resistant wheat cultivars for these regions.\nVirulence patterns of oat crown rust in Australia - season 2022\nAuthors: Henningsen, E. C.; Lewis, D. C.; Nguyen, D. T.; Sperschneider, J.; Kianian, S. F.; Stone, E.; Dodds, P.; Figueroa, M.\nScore: 5.2, Published: 2023-09-24 DOI: 10.1101/2023.09.24.558528\nPuccinia coronata f. sp. avenae (Pca) is an important foliar pathogen of oat which causes crown rust disease. The virulence profile of 48 Pca isolates derived from different locations in Australia was characterised using a collection of oat lines often utilised in rust surveys in the USA and Australia. This analysis indicates that Pca populations in Eastern Australia are broadly virulent, in contrast to the population in Western Australia (WA). Several oat lines/Pc genes are effective against all rust samples collected from WA, suggesting they may provide useful resistance in this region if deployed in combination. We identified 19 lines from the US oat differential set that display disease resistance to Pca in WA, some in agreement with previous rust survey reports. We adopted the 10-letter nomenclature system to define oat crown rust races in Australia and compare the frequency of those virulence traits to published data from the USA. Based on this nomenclature, 42 unique races were detected among the 48 isolates, reflecting the high diversity of virulence phenotypes for Pca in Australia. Nevertheless, the Pca population in the USA is substantially more broadly virulent than that of Australia. Close examination of resistance profiles for the oat differential set lines after infection with Pca supports hypotheses of allelism or redundancy among Pc genes or the presence of several R genes in some oat differential lines. These findings illustrate the need to deconvolute the oat differential set using molecular tools.\nA Functional Exodermal Suberin is Key for Plant Nutrition and Growth in Potato\nAuthors: Company-Arumi, D.; Montells, C.; Iglesias, M.; Margui, E.; Verdaguer, D.; Vogel-Miku, K.; Kelemen, M.; Figueras, M.; Antico, E.; Serra, O.\nScore: 5.3, Published: 2023-09-18 DOI: 10.1101/2023.09.14.557788\nAngiosperm roots, except in Arabidopsis, have both endodermis and exodermis, which regulate radial water and solute movement through lignin and suberin deposition. While endodermal suberin in Arabidopsis acts as a barrier to water and solute uptake and backflow, its implications in other angiosperms with both layers and the role of exodermal suberin remain unclear. We examined potato roots (Solanum tuberosum) and found that exodermis lacks the typical Casparian strip but forms an outer lignin cap, and quickly suberizes near the root tip. In contrast, a few endodermal cells, with Casparian strip, start suberizing much later. The continuous early exodermal suberization covering the root underlines its potential role in mineral nutrient radial movement. To demonstrate it, we used plants downregulating the suberin biosynthetic gene CYP86A33, which had the root suberin reduced in a 61%. Phenotypic analyses of the suberin-deficient mutant showed altered mineral nutrient concentration, slightly reduced water content and compromised growth. Micro-PIXE analyses identified the distribution of elements within the roots and highlighted anatomical compartments defined by apoplastic barriers. These findings advance our understanding of nutrient radial transport, demonstrate exodermal suberin as a bidirectional and selective barrier to element movement, and underscore its importance in nutrient homeostasis and plant growth.\nDifferential mutation rates in plant meristematic layers\nAuthors: Amundson, K. R.; Marimuthu, M. P. A.; Nguyen, O.; Sarika, K.; DeMarco, I. J.; Phan, A.; Henry, I. M.; Comai, L.\nScore: 4.6, Published: 2023-09-26 DOI: 10.1101/2023.09.25.559363\nThe upper plant body is formed by the continued growth of shoot apical meristems. In angiosperms, meristems are organized in three cell layers that tend to remain clonally isolated. Somatic variants emerge when mutant cells overtake part or all of a meristem. During sexual reproduction, only one layer, the L2, contributes to the next generation by forming gametes. The L2 is known to frequently invade and replace the L3, while L1-L2 separation is persistent. The role of different meristem layers in mutation accumulation is unknown. We discovered a potato periclinal chimera in which the L2 and L3, but not the L1, are marked by a chromosomal translocation. This enabled the identification of plants regenerated from leaf protoplasts originating from either the L1 or L2+L3. Leveraging these layer-specific clones, we identified and compared mutations that accumulated in the layers in the clonal parent for several decades. Here we show that the L1 accumulates mutations at 1.9 times the rate of the L2, indicating that plants might protect the germline by mechanisms that reduce the rate of mutation accumulation in the L2. In contrast to these layer-specific mutations, we found no evidence of somatic mutation fixation in all three meristem layers. Our findings highlight how periclinal chimeras are formed by independent mutational processes in which a mutation-prone epidermal layer could increase clonal variation.\nGENOMES UNCOUPLED1: an RNA-binding protein required for early PSII biogenesis\nAuthors: Cui, C.; Sun, S.; Zhang, S.; Zhang, Y.; Kim, C.\nScore: 9.0, Published: 2023-09-22 DOI: 10.1101/2023.09.21.558905\nGenomes Uncoupled1 (GUN1), a nuclear-encoded chloroplast pentatricopeptide repeat (PPR) protein, serves as a master integrator of diverse retrograde signals, mediating chloroplast-to-nucleus retrograde signaling. Although PPR proteins primarily function in organelle RNA metabolism, a target RNA of GUN1 remained unknown. This study reveals that GUN1 recognizes psbD transcripts derived from a blue light-responsive promoter (BLRP), transcripts referred to as psbD BLRP. Overexpression of GUN1 significantly reduces the levels of psbD BLRP, whereas the loss of GUN1 leads to the accumulation of the putative target RNA. The in vitro RNA and in vivo genetic studies further demonstrate the critical role of the C-terminal small MutS-related (SMR) domain in stimulating psbD BLRP processing and PsbD (D2, a PSII core protein) synthesis. The GUN1-dependent psbD BLRP processing promotes PSII biogenesis during the early seedling development and de-etiolation phase. This finding underscores the role of GUN1 as an RNA-binding protein, highlighting the essential function of the SMR domain in processing the target RNA. SignificanceBiogenic retrograde signaling is essential in regulating chloroplast biogenesis, and it entails a nuclear-encoded chloroplast protein, Genomes Uncoupled1 (GUN1). Since its initial discovery in 1993, the gun1 mutant has been widely used to reveal the precise function of GUN1 in plastids as well as its downstream signaling components. Although GUN1 contains pentatricopeptide repeat motifs, it has been considered a non-canonical PPR protein because no one could demonstrate GUN1-RNA interaction in planta. However, through our computational analysis of PPR codes and subsequent biochemical studies, we were able to unravel its putative target RNA, namely psbD BLRP. Therefore, our finding may lead to a reevaluation of GUN1 research in the context of RNA metabolism and a revision of our current understanding of GUN1.\nStomata, a vulnerability in the plant defence against phytophagous mites that ABA can overcome\nAuthors: Rosa-Diaz, I.; Rowe, J.; Cayuela-Lopez, A.; Arbona, V.; Diaz, I.; Jones, A. M.\nScore: 3.1, Published: 2023-09-29 DOI: 10.1101/2023.09.29.555308\nArthropod herbivory possess a significant threat to crop yield, prompting plants to employ intricate defense mechanisms against pest feeding. The generalist pest, Tetranychus urticae, inflicts rapid damage and remains a challenge due to its broad target range. In this study, we explored Arabidopsis thaliana's response to T. urticae infestation, revealing the induction of abscisic acid (ABA), a hormone typically associated with abiotic stress adaptation, including stomatal closure during water stress. Leveraging a FRET-based ABA biosensor (nlsABACUS2-400n), we observed elevated ABA levels in various leaf cell types post-mite feeding. While ABA's role in pest resistance or susceptibility has been debated, an ABA-deficient mutant exhibited increased mite infestation, alongside intact canonical biotic stress signalling, indicating an independent function of ABA in mite defense. Through genetic and pharmacological interventions targeting ABA levels, ABA signalling, stomatal aperture, and density, we established that ABA-triggered stomatal closure effectively hinders mite feeding and minimizes leaf cell damage. This study underscores the critical interplay between biotic and abiotic stresses in plants, highlighting how the vulnerability to mite infestation arising from open stomata, crucial for transpiration and photosynthesis, underscores the intricate relationship between these two stress types.\n",
  "wordCount" : "2830",
  "inLanguage": "en",
  "datePublished": "2023-10-01T10:37:47Z",
  "dateModified": "2023-10-01T10:37:47Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta"><span>updated on October 1, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558255">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558255" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558255">
        <p class="paperTitle">Near-infrared imaging of phytochrome-derived autofluorescence in plant nuclei</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558255" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558255" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yoshinari, A.; Isoda, R.; Yagi, N.; Sato, Y.; Lindeboom, J. J.; Ehrhardt, D. W.; Frommer, W. B.; Nakamura, M.</p>
        <p class="info">Score: 25.0, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558255' target='https://doi.org/10.1101/2023.09.18.558255'> 10.1101/2023.09.18.558255</a></p>
        <p class="abstract">Capturing images of the nuclear dynamics within live cells is an essential technique for comprehending the intricate biological processes inherent to plant cell nuclei. While various methods exist for imaging nuclei, including combining fluorescent proteins and dyes with microscopy, there is a dearth of commercially available dyes for live-cell imaging. In Arabidopsis thaliana, we discovered that nuclei emit autofluorescence in the near-infrared (NIR) range of the spectrum and devised a non-invasive technique for the visualization of live cell nuclei using this inherent NIR autofluorescence. Our studies demonstrated the capability of the NIR imaging technique to visualize the dynamic behavior of nuclei within primary roots, root hairs, and pollen tubes, which are tissues that harbor a limited number of other organelles displaying autofluorescence. We further demonstrated the applicability of NIR autofluorescence imaging in various other tissues by incorporating fluorescence lifetime imaging techniques. Nuclear autofluorescence was also detected across a wide range of plant species, enabling analyses without the need for transformation. The nuclear autofluorescence in the NIR wavelength range was not observed in animal or yeast cells. Genetic analysis revealed that this autofluorescence was caused by the phytochrome protein. Our studies demonstrated that nuclear autofluorescence imaging can be effectively employed not only in model plants but also for studying nuclei in non-model plant species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.23.559107">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.23.559107" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.23.559107">
        <p class="paperTitle">Whole mount multiplexed visualization of DNA, mRNA, and protein in plant-parasitic nematodes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.23.559107" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.23.559107" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sperling, A. L.; Eves-van den Akker, S.</p>
        <p class="info">Score: 19.7, Published: 2023-09-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.23.559107' target='https://doi.org/10.1101/2023.09.23.559107'> 10.1101/2023.09.23.559107</a></p>
        <p class="abstract">BackgroundPlant-parasitic nematodes compromise the agriculture of a wide variety of the most common crops worldwide. Obtaining information on the fundamental biology of these organisms and how they infect the plant has been restricted by the ability to visualize intact nematodes using small molecule stains, antibodies, or in situ hybridization. Consequently, there is limited information available about the internal composition of the nematodes or the biology of the effector molecules they use to reprogram their host plant.

ResultsWe present the Sperling prep -a whole mount method for nematode preparation that enables staining with small molecules, antibodies, or in situ hybridization chain reaction. This method does not require specialized apparatus and utilizes typical laboratory equipment and materials. By dissociating the strong cuticle and interior muscle layers, we enabled entry of the small molecule stains into the tissue. After permeabilization, small molecule stains can be used to visualize the nuclei with the DNA stain DAPI and the internal structures of the digestive tract and longitudinal musculature with the filamentous actin stain phalloidin. The permeabilization even allows entry of larger antibodies, albeit with lower efficiency. Finally, this method works exceptionally well with in situ HCR. Using this method, we have visualized effector transcripts specific to the dorsal gland and the subventral grand of the sugar beet cyst nematode, Heterodera schachtii, multiplexed in the same nematode.

ConclusionWe were able to visualize the internal structures of the nematode as well as key effector transcripts that are used during plant infection and parasitism. Therefore, this method provides an important toolkit for studying the biology of plant-parasitic nematodes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559445">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559445" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559445">
        <p class="paperTitle">Evolution of wheat blast resistance gene Rmg8 accompanied by differentiation of variants recognizing the powdery mildew fungus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559445" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559445" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Asuke, S.; Morita, K.; Shimizu, M.; Abe, F.; Terauchi, R.; Nago, C.; Takahashi, Y.; Shibata, M.; Yoshioka, M.; Iwakawa, M.; Kishi-Kaboshi, M.; Su, Z.; Nasuda, S.; Handa, H.; Fujita, M.; Tougou, M.; Hatta, K.; Mori, N.; Matsuoka, Y.; Kato, K.; Tosa, Y.</p>
        <p class="info">Score: 18.6, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559445' target='https://doi.org/10.1101/2023.09.26.559445'> 10.1101/2023.09.26.559445</a></p>
        <p class="abstract">Wheat blast, a devastating disease having spread recently from South America to Asia and Africa, is caused by Pyricularia oryzae pathotype Triticum which emerged in 1985. Rmg8 and Rmg7, genes for resistance to wheat blast found in common wheat and tetraploid wheat, respectively, recognize the same avirulence gene, AVR-Rmg8. Here, we show an evolutionary process in which resistance gene(s), which had obtained an ability to recognize AVR-Rmg8 before the differentiation of Triticum and Aegilops, has expanded its target pathogens. Molecular cloning revealed that Rmg7 was one of alleles of Pm4 (Pm4a), a gene for resistance to wheat powdery mildew on 2AL, whereas Rmg8 was its homoeolog on 2BL ineffective against wheat powdery mildew. Rmg8 variants with the ability to recognize AVR-Rmg8 were distributed not only in Triticum spp. but also in Aegilops speltoides Ae. umbellulata, and Ae. comosa. This result suggests that the origin of resistance gene(s) recognizing AVR-Rmg8 dates back to the time before differentiation of A, B, S, U, and M genomes, that is, ~5 million years before the emergence of its current target, the wheat blast fungus. Phylogenetic analyses suggested that, in the evolutionary process thereafter, some of their variants gained the ability to recognize the wheat powdery mildew fungus and evolved into genes for resistance to wheat powdery mildew.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.23.559151">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.23.559151" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.23.559151">
        <p class="paperTitle">Sequence characterization of T, Bip, and Phbw demonstrates the role of MYB-bHLH-WD40 complexes and temperature in common bean seed color pattern formation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.23.559151" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.23.559151" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parker, T.; Bolt, T.; Williams, T.; Penmetsa, R. V.; Mulube, M.; Palkovic, A.; Jochua, C. N.; Rubio Wilhelmi, M. d. M.; Lo, S.; Bornhorst, G.; Tian, L.; Kamfwa, K.; Hokin, S.; Farmer, A.; Diepenbrock, C. H.; Gepts, P.</p>
        <p class="info">Score: 13.9, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.23.559151' target='https://doi.org/10.1101/2023.09.23.559151'> 10.1101/2023.09.23.559151</a></p>
        <p class="abstract">Seed colors and color patterns are critical for the survival of wild plants and the consumer appeal of crops. In common bean, a major global staple, these patterns are also critical for determining market classes, yet the genetic and environmental control of many pigmentation patterns remains unresolved. In this study, we genetically mapped variation for three important seed pattern loci, T, Bip, and phbw, which co-segregated completely with PvTTG1, PvMYC1, and PvTT8, respectively. Proteins encoded by these genes are predicted to work together in MYB-bHLH-WD40 (MBW) complexes, propagating flavonoid biosynthesis across the seed coat. Whole-genome sequencing of 37 diverse accessions identified putative mutations in each gene, including seven unique parallel mutations in T (PvTTG1) and a non-synonymous SNP in a conserved residue in bipana (PvMYC1). A 612 bp intron deletion in phbw (PvTT8) eliminated motifs conserved since the origins of the Papilionoidea and corresponded to a 20-fold reduction in transcript abundance. Mutations in MBW candidate genes for Z (PvTT2) and Sellatus (WDR) were also identified. In multi-location field trials with seven varieties with partial seed coat patterning, pigmented seed coat area was highly unstable and correlated with temperature, with up to 11-fold differences in pigmented area between the warmest and the coolest environments. In controlled growth chamber conditions, an increase of 4 {degrees}C was sufficient to cause pigmentation on an additional 21% of the seed coat area. Our results shed light on the fundamental activation of flavonoid biosynthesis in common bean and will be instrumental for maximizing consumer appeal in this nutritious staple crop.

Summary- Seed colors and patterns are critical for the survival of wild plants, and are important in differentiating crop market classes, but the genetic control of these in the staple crop common bean (Phaseolus vulgaris) is largely unknown.
- The genetic, transcriptional, and environmental basis of common bean seed color patterning was explored through QTL mapping, whole-genome sequencing, RT-qPCR, and automated pigmentation quantification of seed grown in multi-location field trials and growth chamber environments.
- MYB-bHLH-WD40 complex-forming genes PvTTG1, PvMYC1, and PvTT8 co-segregated completely with the color patterning genes T, Bip, and phbw. Mutations were identified in each gene, including seven unique parallel mutations in T (PvTTG1), a non-synonymous SNP in a conserved residue in bipana (PvMYC1), and an intron deletion in phbw (PvTT8) eliminating highly conserved motifs and corresponding to 20-fold lower PvTT8 transcript abundance. Mutations in MBW candidate genes Z (PvTT2) and Sellatus (WDR) were also identified. In multi-location field trials, pigmented seed coat area was highly unstable and corresponded to temperature. In growth chamber conditions, an increase of 4 {degrees}C caused pigmentation on an additional 21% of the seed coat area.
- Our results highlight the critical interaction between MYB-bHLH-WD40 complex components and temperature in establishing seed pattern diversity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559489">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559489" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559489">
        <p class="paperTitle">The wheat powdery mildew resistance gene Pm4 also confers resistance to wheat blast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559489" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559489" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: O&#39;Hara, T.; Steed, A.; Goddard, R.; Gaurav, K.; Arora, S.; Quiroz-Chavez, J.; Ramirez-Gonzalez, R.; Badgami, R.; Gilbert, D.; Sanchez-Martin, J.; Wingen, L.; Feng, C.; Jiang, M.; Cheng, S.; Dreisigacker, S.; Keller, B.; Wulff, B.; Uauy, C.; Nicholson, P.</p>
        <p class="info">Score: 5.5, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559489' target='https://doi.org/10.1101/2023.09.26.559489'> 10.1101/2023.09.26.559489</a></p>
        <p class="abstract">Wheat blast, caused by the fungus Magnaporthe oryzae, threatens global cereal production since its emergence in Brazil in 1985 and recently spread to Bangladesh and Zambia. Here we demonstrate that the AVR-Rmg8 effector, common in wheat-infecting isolates, is recognised by the gene Pm4, previously shown to confer resistance to specific races of Blumeria graminis f.sp. tritici, the cause of powdery mildew of wheat. We show that Pm4 alleles differ in their recognition of different AVR-Rmg8 alleles, and some confer resistance only in seedling leaves but not spikes making it important to select for those alleles that function in both tissues. This study has identified a gene recognising an important virulence factor present in wheat blast isolates in Bangladesh and Zambia and represents an important first step towards developing durably resistant wheat cultivars for these regions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.24.558528">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.24.558528" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.24.558528">
        <p class="paperTitle">Virulence patterns of oat crown rust in Australia - season 2022</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.24.558528" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.24.558528" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Henningsen, E. C.; Lewis, D. C.; Nguyen, D. T.; Sperschneider, J.; Kianian, S. F.; Stone, E.; Dodds, P.; Figueroa, M.</p>
        <p class="info">Score: 5.2, Published: 2023-09-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.24.558528' target='https://doi.org/10.1101/2023.09.24.558528'> 10.1101/2023.09.24.558528</a></p>
        <p class="abstract">Puccinia coronata f. sp. avenae (Pca) is an important foliar pathogen of oat which causes crown rust disease. The virulence profile of 48 Pca isolates derived from different locations in Australia was characterised using a collection of oat lines often utilised in rust surveys in the USA and Australia. This analysis indicates that Pca populations in Eastern Australia are broadly virulent, in contrast to the population in Western Australia (WA). Several oat lines/Pc genes are effective against all rust samples collected from WA, suggesting they may provide useful resistance in this region if deployed in combination. We identified 19 lines from the US oat differential set that display disease resistance to Pca in WA, some in agreement with previous rust survey reports. We adopted the 10-letter nomenclature system to define oat crown rust races in Australia and compare the frequency of those virulence traits to published data from the USA. Based on this nomenclature, 42 unique races were detected among the 48 isolates, reflecting the high diversity of virulence phenotypes for Pca in Australia. Nevertheless, the Pca population in the USA is substantially more broadly virulent than that of Australia. Close examination of resistance profiles for the oat differential set lines after infection with Pca supports hypotheses of allelism or redundancy among Pc genes or the presence of several R genes in some oat differential lines. These findings illustrate the need to deconvolute the oat differential set using molecular tools.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557788">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557788" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557788">
        <p class="paperTitle">A Functional Exodermal Suberin is Key for Plant Nutrition and Growth in Potato</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557788" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557788" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Company-Arumi, D.; Montells, C.; Iglesias, M.; Margui, E.; Verdaguer, D.; Vogel-Miku, K.; Kelemen, M.; Figueras, M.; Antico, E.; Serra, O.</p>
        <p class="info">Score: 5.3, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557788' target='https://doi.org/10.1101/2023.09.14.557788'> 10.1101/2023.09.14.557788</a></p>
        <p class="abstract">Angiosperm roots, except in Arabidopsis, have both endodermis and exodermis, which regulate radial water and solute movement through lignin and suberin deposition. While endodermal suberin in Arabidopsis acts as a barrier to water and solute uptake and backflow, its implications in other angiosperms with both layers and the role of exodermal suberin remain unclear. We examined potato roots (Solanum tuberosum) and found that exodermis lacks the typical Casparian strip but forms an outer lignin cap, and quickly suberizes near the root tip. In contrast, a few endodermal cells, with Casparian strip, start suberizing much later. The continuous early exodermal suberization covering the root underlines its potential role in mineral nutrient radial movement. To demonstrate it, we used plants downregulating the suberin biosynthetic gene CYP86A33, which had the root suberin reduced in a 61%. Phenotypic analyses of the suberin-deficient mutant showed altered mineral nutrient concentration, slightly reduced water content and compromised growth. Micro-PIXE analyses identified the distribution of elements within the roots and highlighted anatomical compartments defined by apoplastic barriers. These findings advance our understanding of nutrient radial transport, demonstrate exodermal suberin as a bidirectional and selective barrier to element movement, and underscore its importance in nutrient homeostasis and plant growth.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559363">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559363" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559363">
        <p class="paperTitle">Differential mutation rates in plant meristematic layers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559363" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559363" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Amundson, K. R.; Marimuthu, M. P. A.; Nguyen, O.; Sarika, K.; DeMarco, I. J.; Phan, A.; Henry, I. M.; Comai, L.</p>
        <p class="info">Score: 4.6, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559363' target='https://doi.org/10.1101/2023.09.25.559363'> 10.1101/2023.09.25.559363</a></p>
        <p class="abstract">The upper plant body is formed by the continued growth of shoot apical meristems. In angiosperms, meristems are organized in three cell layers that tend to remain clonally isolated. Somatic variants emerge when mutant cells overtake part or all of a meristem. During sexual reproduction, only one layer, the L2, contributes to the next generation by forming gametes. The L2 is known to frequently invade and replace the L3, while L1-L2 separation is persistent. The role of different meristem layers in mutation accumulation is unknown. We discovered a potato periclinal chimera in which the L2 and L3, but not the L1, are marked by a chromosomal translocation. This enabled the identification of plants regenerated from leaf protoplasts originating from either the L1 or L2&#43;L3. Leveraging these layer-specific clones, we identified and compared mutations that accumulated in the layers in the clonal parent for several decades. Here we show that the L1 accumulates mutations at 1.9 times the rate of the L2, indicating that plants might protect the germline by mechanisms that reduce the rate of mutation accumulation in the L2. In contrast to these layer-specific mutations, we found no evidence of somatic mutation fixation in all three meristem layers. Our findings highlight how periclinal chimeras are formed by independent mutational processes in which a mutation-prone epidermal layer could increase clonal variation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558905">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558905" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558905">
        <p class="paperTitle">GENOMES UNCOUPLED1: an RNA-binding protein required for early PSII biogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558905" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558905" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cui, C.; Sun, S.; Zhang, S.; Zhang, Y.; Kim, C.</p>
        <p class="info">Score: 9.0, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558905' target='https://doi.org/10.1101/2023.09.21.558905'> 10.1101/2023.09.21.558905</a></p>
        <p class="abstract">Genomes Uncoupled1 (GUN1), a nuclear-encoded chloroplast pentatricopeptide repeat (PPR) protein, serves as a master integrator of diverse retrograde signals, mediating chloroplast-to-nucleus retrograde signaling. Although PPR proteins primarily function in organelle RNA metabolism, a target RNA of GUN1 remained unknown. This study reveals that GUN1 recognizes psbD transcripts derived from a blue light-responsive promoter (BLRP), transcripts referred to as psbD BLRP. Overexpression of GUN1 significantly reduces the levels of psbD BLRP, whereas the loss of GUN1 leads to the accumulation of the putative target RNA. The in vitro RNA and in vivo genetic studies further demonstrate the critical role of the C-terminal small MutS-related (SMR) domain in stimulating psbD BLRP processing and PsbD (D2, a PSII core protein) synthesis. The GUN1-dependent psbD BLRP processing promotes PSII biogenesis during the early seedling development and de-etiolation phase. This finding underscores the role of GUN1 as an RNA-binding protein, highlighting the essential function of the SMR domain in processing the target RNA.

SignificanceBiogenic retrograde signaling is essential in regulating chloroplast biogenesis, and it entails a nuclear-encoded chloroplast protein, Genomes Uncoupled1 (GUN1). Since its initial discovery in 1993, the gun1 mutant has been widely used to reveal the precise function of GUN1 in plastids as well as its downstream signaling components. Although GUN1 contains pentatricopeptide repeat motifs, it has been considered a non-canonical PPR protein because no one could demonstrate GUN1-RNA interaction in planta. However, through our computational analysis of PPR codes and subsequent biochemical studies, we were able to unravel its putative target RNA, namely psbD BLRP. Therefore, our finding may lead to a reevaluation of GUN1 research in the context of RNA metabolism and a revision of our current understanding of GUN1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.555308">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.555308" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.555308">
        <p class="paperTitle">Stomata, a vulnerability in the plant defence against phytophagous mites that ABA can overcome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.555308" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.555308" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rosa-Diaz, I.; Rowe, J.; Cayuela-Lopez, A.; Arbona, V.; Diaz, I.; Jones, A. M.</p>
        <p class="info">Score: 3.1, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.555308' target='https://doi.org/10.1101/2023.09.29.555308'> 10.1101/2023.09.29.555308</a></p>
        <p class="abstract">Arthropod herbivory possess a significant threat to crop yield, prompting plants to employ intricate defense mechanisms against pest feeding. The generalist pest, Tetranychus urticae, inflicts rapid damage and remains a challenge due to its broad target range. In this study, we explored Arabidopsis thaliana&#39;s response to T. urticae infestation, revealing the induction of abscisic acid (ABA), a hormone typically associated with abiotic stress adaptation, including stomatal closure during water stress. Leveraging a FRET-based ABA biosensor (nlsABACUS2-400n), we observed elevated ABA levels in various leaf cell types post-mite feeding. While ABA&#39;s role in pest resistance or susceptibility has been debated, an ABA-deficient mutant exhibited increased mite infestation, alongside intact canonical biotic stress signalling, indicating an independent function of ABA in mite defense. Through genetic and pharmacological interventions targeting ABA levels, ABA signalling, stomatal aperture, and density, we established that ABA-triggered stomatal closure effectively hinders mite feeding and minimizes leaf cell damage. This study underscores the critical interplay between biotic and abiotic stresses in plants, highlighting how the vulnerability to mite infestation arising from open stomata, crucial for transpiration and photosynthesis, underscores the intricate relationship between these two stress types.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
