<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>plant biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>plant biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Identification of cambium stem cell factors and their positioning mechanism
Authors: Eswaran, G.; Rutten, J. P.; Han, J.; Iida, H.; Lopez Ortiz, J.; Makila, R.; Wybouw, B.; Vainio, L.; Porcher, A.; Leal Gavarron, M.; Zhang, J.; Blomster, T.; Wang, X.; Dolan, D. W. P.; Smetana, O.; Brady, S.; Kucukoglu Topcu, M.; ten Tusscher, K. H.; Etchells, P.; Mahonen, A. P.
Score: 17.9, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549889
Wood constitutes the majority of terrestrial biomass.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/plant-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="plant biology" />
<meta property="og:description" content="Identification of cambium stem cell factors and their positioning mechanism
Authors: Eswaran, G.; Rutten, J. P.; Han, J.; Iida, H.; Lopez Ortiz, J.; Makila, R.; Wybouw, B.; Vainio, L.; Porcher, A.; Leal Gavarron, M.; Zhang, J.; Blomster, T.; Wang, X.; Dolan, D. W. P.; Smetana, O.; Brady, S.; Kucukoglu Topcu, M.; ten Tusscher, K. H.; Etchells, P.; Mahonen, A. P.
Score: 17.9, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549889
Wood constitutes the majority of terrestrial biomass." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/plant-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-23T10:36:34+00:00" />
<meta property="article:modified_time" content="2023-07-23T10:36:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="plant biology"/>
<meta name="twitter:description" content="Identification of cambium stem cell factors and their positioning mechanism
Authors: Eswaran, G.; Rutten, J. P.; Han, J.; Iida, H.; Lopez Ortiz, J.; Makila, R.; Wybouw, B.; Vainio, L.; Porcher, A.; Leal Gavarron, M.; Zhang, J.; Blomster, T.; Wang, X.; Dolan, D. W. P.; Smetana, O.; Brady, S.; Kucukoglu Topcu, M.; ten Tusscher, K. H.; Etchells, P.; Mahonen, A. P.
Score: 17.9, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549889
Wood constitutes the majority of terrestrial biomass."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "plant biology",
      "item": "https://trxiv.yorks0n.com/posts/plant-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "plant biology",
  "name": "plant biology",
  "description": "Identification of cambium stem cell factors and their positioning mechanism\nAuthors: Eswaran, G.; Rutten, J. P.; Han, J.; Iida, H.; Lopez Ortiz, J.; Makila, R.; Wybouw, B.; Vainio, L.; Porcher, A.; Leal Gavarron, M.; Zhang, J.; Blomster, T.; Wang, X.; Dolan, D. W. P.; Smetana, O.; Brady, S.; Kucukoglu Topcu, M.; ten Tusscher, K. H.; Etchells, P.; Mahonen, A. P.\nScore: 17.9, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549889\nWood constitutes the majority of terrestrial biomass.",
  "keywords": [
    
  ],
  "articleBody": " Identification of cambium stem cell factors and their positioning mechanism\nAuthors: Eswaran, G.; Rutten, J. P.; Han, J.; Iida, H.; Lopez Ortiz, J.; Makila, R.; Wybouw, B.; Vainio, L.; Porcher, A.; Leal Gavarron, M.; Zhang, J.; Blomster, T.; Wang, X.; Dolan, D. W. P.; Smetana, O.; Brady, S.; Kucukoglu Topcu, M.; ten Tusscher, K. H.; Etchells, P.; Mahonen, A. P.\nScore: 17.9, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549889\nWood constitutes the majority of terrestrial biomass. Composed of xylem, it arises from one side of the vascular cambium, a bifacial stem cell niche that also produces phloem on the opposing side. It is currently unknown which molecular factors endow cambium stem cell identity. Here we show that TDIF ligand-activated PXY receptors promote the expression of CAMBIUM AINTEGUMENTA-LIKE (CAIL) transcription factors to define cambium stem cell identity in the Arabidopsis root. By sequestrating the phloem-originated TDIF, xylem-dependent PXY confines the TDIF signaling front, resulting in the activation of CAIL expression and stem cell identity in only a narrow domain. Our findings show how signals emanating from cells on opposing sides ensure robust yet dynamically adjustable positioning of a bifacial stem cell.\nBasal expression of immune receptor genes requires low levels of the phytohormone salicylic acid\nAuthors: van Butselaar, T.; Silva, S.; Lapin, D.; Banales, I.; Tonn, S.; van Schie, C.; Van den Ackerveken, G.\nScore: 13.9, Published: 2023-07-16 DOI: 10.1101/2023.07.14.548351\nThe hormone salicylic acid (SA) plays a crucial role in plant immunity by activating responses that arrest pathogen ingress. Since SA accumulation also penalizes growth, the question remains why healthy plants synthesize this hormone. By overexpressing SA-inactivating hydroxylases in Arabidopsis thaliana, we reveal that basal SA levels in unchallenged plants are needed for expression of selected immune receptor and signaling genes, thereby enabling early pathogen detection and activation of immunity.\nRoot barrier surveillance mechanisms convey soil nitrogen status to shoots\nAuthors: Shen, D.; Wippel, K.; Remmel, S.; Zhang, P.; Kuertoes, N.; Neumann, U.; Kopriva, S.; Andersen, T. G.\nScore: 42.6, Published: 2023-07-18 DOI: 10.1101/2023.04.19.537470\nIn roots, formation of the Casparian strip in the endodermal cell walls provides a mechanism for selective uptake of nutrients and water. Establishment of this filter is under spatial surveillance by a receptor-ligand mechanism termed the Schengen pathway. This system provides a mechanism to initiate downstream signaling responses in case of dysfunctional barrier establishment. Due to this interconnected nature, the endogenous physiological role of this survaillance mechanism remain difficult to untangle from the direct barrier function. This is in particularly evident in complex growth setups where multiple inputs are integrated into general whole-plant responses. In this work, we address this by rewiring the genetic network that underlies root barrier formation. We create plants with enhanced, Schengen-independent Casparian strip formation that are not only suitable to probe the role of stronger barrier establishment, but also to evaluate the associated signaling output independently. With focus on the latter, we subjected the rewired plants, as well as a number of established barrier mutants, to multifaceted growth conditions including nitrogen fertilized agricultural soil conditions. By profiling their above- and belowground (a)biotic responses our work reveals that, while increased Casparian strip formation mainly provides the plant with an improved stress resistance, the Schengen pathway is necessary for establishment of a growth-promoting root microbiome and serves to convey information of soil nitrogen status to the shoot. This identifies the Schengen pathway as an essential receptor-based signaling hub for adaptive integration of barrier status, nutritional responses and (a)biotic signaling between above- and belowground tissues.\nCompositae-ParaLoss-1272: Complementary sunflower specific probe-set reduces issues with paralogs in complex systems\nAuthors: Moore-Pollard, E. R.; Jones, D. S.; Mandel, J. R.\nScore: 5.9, Published: 2023-07-21 DOI: 10.1101/2023.07.19.549085\nPremise: The sunflower family specific probe set, Compositae-1061, has enabled family-wide phylogenomic studies and investigations at lower-taxonomic levels by targeting 1,000+ genes. However, it generally lacks resolution at the genus to species level, especially in groups with complex evolutionary histories including polyploidy and hybridization. Methods: In this study, we developed a new HybSeq probe set, Compositae-ParaLoss-1272, designed to target orthologous loci in Asteraceae family members. We tested its efficiency across the family by simulating target-enrichment sequencing in silico. Additionally, we tested its effectiveness at lower taxonomic levels in genus Packera which has a complex evolutionary and taxonomic history. We performed HybSeq with Compositae-ParaLoss-1272 for 19 taxa which were previously studied using the Compositae-1061 probe set. Sequences from both probe sets were used to generate phylogenies, compare topologies, and assess node support. Results: We report that Compositae-ParaLoss-1272 captured loci across all tested Asteraceae members. Additionally, Compositae-ParaLoss-1272 had less gene tree discordance, recovered considerably fewer paralogous sequences, and retained longer loci than Compositae-1061. Discussion: Given the complexity of plant evolutionary histories, assigning orthology for phylogenomic analyses will continue to be challenging. However, we anticipate this new probe set will provide improved resolution and utility for studies at lower-taxonomic levels and complex groups in the sunflower family.\nPhenomic vs Genomic Prediction - A Comparison of Prediction Accuracies for Grain Yield in Hard Winter Wheat Lines\nAuthors: Winn, Z. J.; Amsberry, A.; Haley, S. D.; DeWitt, N. D.; Mason, E.\nScore: 5.6, Published: 2023-07-20 DOI: 10.1101/2023.07.19.549748\nCommon bread wheat (Triticum aestivum L.) is a key component of global diets, but the genetic improvement of wheat is not keeping pace with the growing demands of the world's population. To increase efficiency and reduce costs, breeding programs are rapidly adopting the use of unoccupied aerial vehicles (UAVs) to conduct high-throughput spectral analyses. This study examined the effectiveness of multispectral indices in predicting grain yield compared to genomic prediction. Multispectral data were collected on advanced generation yield nursery trials during the 2019-2021 growing seasons in the Colorado State University Wheat Breeding Program. Genome-wide genotyping was performed on these advanced generations and all plots were harvested to measure grain yield. Two methods were used to predict grain yield: genomic estimated breeding values (GEBVs) generated by a genomic best linear unbiased prediction (gBLUP) model and phenomic phenotypic estimates (PPEs) using only spectral indices via multiple linear regression (MLR), k-nearest neighbors (KNN), and random forest (RF) models. In cross-validation, PPEs produced by MLR, KNN, and RF models had higher prediction accuracy (r: 0.41 [\u0026le;] r [\u0026le;] 0.48) than GEBVs produced by gBLUP (r = 0.35). In leave-one-year-out forward validation using only multispectral data for 2020 and 2021, PPEs from MLR and KNN models had higher prediction accuracy of grain yield than GEBVs of those same lines. These findings suggest that a limited number of spectra may produce PPEs that are more accurate than or equivalently accurate as GEBVs derived from gBLUP, and this method should be evaluated in earlier development material where sequencing is not feasible.\nResolving evolutionary relationships in the groundsels: phylogenomics, divergence time estimates, and biogeography of Packera (Asteraceae: Senecioneae)\nAuthors: Moore-Pollard, E. R.; Mandel, J. R.\nScore: 5.0, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549592\nThe genus Packera belongs to the sunflower family and contains an estimated 64 species and varieties endemic to North America. Some Packera are known to hybridize or exhibit polyploidy, making it difficult to reconstruct evolutionary relationships within the group. Previous molecular phylogenetic studies of Packera employing ITS data recovered low resolution trees, providing little information on the evolutionary relationships within this complex genus. Therefore, we used next-generation sequencing data to infer nuclear and plastid phylogenies of Packera and related Senecioneae taxa. The nuclear phylogeny was calibrated to produce a timetree, then used to reconstruct the macroevolutionary history of Packera, including its historical biogeography. We then compared the reconstructed evolutionary history to previously published scenarios based on phylogenetic and geohistorical data. We found that the nuclear and plastid phylogenies were highly incongruent, with the nuclear tree presenting higher resolution than the plastid tree which had an apparent lack of plastid diversity. The nuclear tree indicated that geography may have played a major role in the evolution and taxonomic diversification of Packera. The estimated origin of Packera at approximately 19.2MY - 25.9MY (late Oligocene to early Miocene) is older than in most other studies. Nonetheless, it aligns well with previous geohistorical predictions, which suggest that speciation and diversification events in Packera were driven by changes in geography and climate in North America. Moreover, Packera likely originated in the western United States or Mexico, and subsequently diversified north and east into the rest of North America and Russia, in agreement with other studies.\nFrom Asymptomatic to Symptomatic: Multiomics profiling of the temporal response of grapevine viral-mixed infection\nAuthors: Lamine Fall, M.; Poursalavati, A.; Sidibe, A.; Xu, D.; Lemoyne, P.; Silva Martins, G.; Jalali Javaran, V.; Moffett, P.; Odile, C.\nScore: 3.5, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549167\nMixed viral infections are common in grapevines. However, our understanding of the factors and signaling pathways that influence the expression of viral symptoms in mixed infections is still incomplete. In a previous study, we revealed that the presence of grapevine leafroll-associated virus species in mixed infections was randomly associated with the devel-opment of virus-like symptoms. To understand what drives the timing of these virus-like symptoms in mixed infections, we used dsRNA and total RNA sequencing and metabolomic analysis to profile the viromes, metabolites, and transcripts of grapevine leaves collected at two different times of the year (summer and autumn). We demonstrated that neither viral titre nor virome composition changes were associated with symptom expression in autumn. The total phenolic content and antioxidant capacity increased in most plants except for those with early onset symptoms. According to the results of differential gene expression analysis, cell wall biosynthesis pathways were significantly downregulated in all grapevine plants infected with grapevine leafroll-associated virus 3, grapevine asteroid mosaic-associated virus, and grape-vine Pinot gris virus. In addition, polyketide pathways were significantly upregulated in all cultivars, while flavonoid precursor (e.g. abscisic acid) production was significantly reduced in plants that expressed strong virus-like symptoms. In the Vidal cultivar, an uncharacterized double-stranded RNA-binding protein (DRB) appears to play a critical role in the plants an-tiviral defences, supporting the recent hypothesis that DRBs make an important contribution to dominant antiviral responses in plants. The seasonality of the expression virus-like symptoms appears to be a consequence of the dynamic interactions between antiviral factors and viral counter-defences that occur at different developmental stages of grapevine.\nDevelopment of a mobile, high-throughput, and low-cost image-based plant growth phenotyping system\nAuthors: Yu, L.; Sussman, H.; Khmelnitsky, O.; Ishka, M. R.; Srinivasan, A.; Nelson, A.; Julkowska, M. M.\nScore: 3.4, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549560\nNondestructive plant phenotyping is fundamental for unraveling molecular processes underlying plant development and response to the environment. While the emergence of high-through phenotyping facilities can further our understanding of plant development and stress responses, their high costs significantly hinder scientific progress. To democratize high-throughput plant phenotyping, we developed sets of low-cost image- and weight-based devices to monitor plant growth and evapotranspiration. We paired these devices with a suite of computational pipelines for integrated and straightforward data analysis. We validated the suitability of our system for large screens by evaluating a cowpea diversity panel for responses to drought stress. The observed natural variation was subsequently used for Genome-Wide Association Study, where we identified nine genetic loci that putatively contribute to cowpea drought resilience during early vegetative development. We validated the homologs of the identified candidate genes in Arabidopsis using available mutant lines. These results demonstrate the varied applicability of this low-cost phenotyping system. In the future, we foresee these setups facilitating identification of genetic components of growth, plant architecture, and stress tolerance across a wide variety of species.\nAnnual dynamics and distribution of Xylella fastidiosa in infected almond trees\nAuthors: Zecharia, N.; Vanunu, M.; Dror, O.; Hatib, K.; Holland, D.; Shtienberg, D.; Bahar, O.\nScore: 2.0, Published: 2023-07-19 DOI: 10.1101/2023.07.17.549336\nThis research focused on studying the dynamics of the bacterial pathogen Xylella fastidiosa in almond trees at different developmental stages and in various tree parts. The objective was to understand the annual distribution and concentration of X. fastidiosa within almond trees. Different tree parts, including leaf petioles, annual and perennial shoots, fruit parts, flowers, and roots, from ten X. fastidiosa-infected almond trees were sampled over two years. The distribution and concentration of X. fastidiosa were determined using qPCR and serial dilution plating. Throughout the study, X. fastidiosa was never found in the fruit, flowers, and roots of almond trees, but it was present in leaves and annual and perennial shoots. We show that the inability of X. fastidiosa to colonize roots is likely due to incompatibility with the GF677 rootstock. The presence of X. fastidiosa in shoots remained consistent throughout the year, while in leaf petioles it varied across developmental stages, with lower detection during early and late stages of the season. Similarly, viable X. fastidiosa cells could be isolated from shoots at all developmental stages, while in leaf petioles no successful isolations were achieved during the vegetative and nut growth stage. Examining the development of almond leaf scorch symptoms over time in trees with preliminary infections revealed that once symptoms have appeared on a single branch, other asymptomatic limbs were likely already colonized by the bacterium, hence, selective pruning of symptomatic branches is unlikely to cure the tree. Overall, this study enhances our understanding of X. fastidiosa dynamics in almonds and may have practical applications for its detection and control in almond orchards.\nOrigin and diversity of Capsella bursa-pastoris from the genomic point view.\nAuthors: Penin, A. A.; Kasianov, A. S.; Klepikova, A. V.; Omelchenko, D. O.; Makarenko, M. S.; Logacheva, M. D.\nScore: 2.0, Published: 2023-07-15 DOI: 10.1101/2023.07.13.548917\nBackgroundCapsella bursa-pastoris, a cosmopolitan weed of hybrid origin, is an emerging model object for the study of early consequences of polyploidy, being a fast growing annual and a close relative of Arabidopsis thaliana. The development of this model is hampered by the absence of a reference genome sequence. Resultswe present here a subgenome-resolved chromosome-scale assembly and a genetic map of the genome of Capsella bursa-pastoris. It shows that the subgenomes are mostly colinear, with no massive deletions, insertions or rearrangements in any of them. A subgenome-aware annotation reveals the lack of genome dominance - both subgenomes carry similar number of genes. While most chromosomes can be unambiguously recognized as derived from either paternal or maternal parent, we also found homeologous exchange between two chromosomes. It led to an emergence of two hybrid chromosomes; this event is shared between distant populations of C. bursa-pastoris. The whole-genome analysis of 119 samples belonging to C. bursa-pastoris and its parental species C. grandiflora/rubella and C. orientalis reveals introgression from C. orientalis but not from C. grandiflora/rubella. ConclusionsC. bursa-pastoris do not show genome dominance. In the earliest stages of evolution of this species a homeologous exchange occurred; its presence in all present-day populations of C. bursa-pastoris indicates on a single origin of this species. The evidence coming from whole-genome analysis challenges the current view that C. grandiflora/rubella was a direct progenitor of C. bursa-pastoris; we hypothesize that it was an extinct (or undiscovered) species sister to C. grandiflora/rubella.\n",
  "wordCount" : "2472",
  "inLanguage": "en",
  "datePublished": "2023-07-23T10:36:34Z",
  "dateModified": "2023-07-23T10:36:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/plant-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      plant biology
    </h1>
    <div class="post-meta"><span>updated on July 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549889">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549889" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549889">
        <p class="paperTitle">Identification of cambium stem cell factors and their positioning mechanism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549889" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549889" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eswaran, G.; Rutten, J. P.; Han, J.; Iida, H.; Lopez Ortiz, J.; Makila, R.; Wybouw, B.; Vainio, L.; Porcher, A.; Leal Gavarron, M.; Zhang, J.; Blomster, T.; Wang, X.; Dolan, D. W. P.; Smetana, O.; Brady, S.; Kucukoglu Topcu, M.; ten Tusscher, K. H.; Etchells, P.; Mahonen, A. P.</p>
        <p class="info">Score: 17.9, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549889' target='https://doi.org/10.1101/2023.07.20.549889'> 10.1101/2023.07.20.549889</a></p>
        <p class="abstract">Wood constitutes the majority of terrestrial biomass. Composed of xylem, it arises from one side of the vascular cambium, a bifacial stem cell niche that also produces phloem on the opposing side. It is currently unknown which molecular factors endow cambium stem cell identity. Here we show that TDIF ligand-activated PXY receptors promote the expression of CAMBIUM AINTEGUMENTA-LIKE (CAIL) transcription factors to define cambium stem cell identity in the Arabidopsis root. By sequestrating the phloem-originated TDIF, xylem-dependent PXY confines the TDIF signaling front, resulting in the activation of CAIL expression and stem cell identity in only a narrow domain. Our findings show how signals emanating from cells on opposing sides ensure robust yet dynamically adjustable positioning of a bifacial stem cell.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.548351">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.548351" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.548351">
        <p class="paperTitle">Basal expression of immune receptor genes requires low levels of the phytohormone salicylic acid</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.548351" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.548351" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: van Butselaar, T.; Silva, S.; Lapin, D.; Banales, I.; Tonn, S.; van Schie, C.; Van den Ackerveken, G.</p>
        <p class="info">Score: 13.9, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.548351' target='https://doi.org/10.1101/2023.07.14.548351'> 10.1101/2023.07.14.548351</a></p>
        <p class="abstract">The hormone salicylic acid (SA) plays a crucial role in plant immunity by activating responses that arrest pathogen ingress. Since SA accumulation also penalizes growth, the question remains why healthy plants synthesize this hormone. By overexpressing SA-inactivating hydroxylases in Arabidopsis thaliana, we reveal that basal SA levels in unchallenged plants are needed for expression of selected immune receptor and signaling genes, thereby enabling early pathogen detection and activation of immunity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.04.19.537470">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.04.19.537470" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.04.19.537470">
        <p class="paperTitle">Root barrier surveillance mechanisms convey soil nitrogen status to shoots</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.04.19.537470" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.04.19.537470" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shen, D.; Wippel, K.; Remmel, S.; Zhang, P.; Kuertoes, N.; Neumann, U.; Kopriva, S.; Andersen, T. G.</p>
        <p class="info">Score: 42.6, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.04.19.537470' target='https://doi.org/10.1101/2023.04.19.537470'> 10.1101/2023.04.19.537470</a></p>
        <p class="abstract">In roots, formation of the Casparian strip in the endodermal cell walls provides a mechanism for selective uptake of nutrients and water. Establishment of this filter is under spatial surveillance by a receptor-ligand mechanism termed the Schengen pathway. This system provides a mechanism to initiate downstream signaling responses in case of dysfunctional barrier establishment. Due to this interconnected nature, the endogenous physiological role of this survaillance mechanism remain difficult to untangle from the direct barrier function. This is in particularly evident in complex growth setups where multiple inputs are integrated into general whole-plant responses. In this work, we address this by rewiring the genetic network that underlies root barrier formation. We create plants with enhanced, Schengen-independent Casparian strip formation that are not only suitable to probe the role of stronger barrier establishment, but also to evaluate the associated signaling output independently. With focus on the latter, we subjected the rewired plants, as well as a number of established barrier mutants, to multifaceted growth conditions including nitrogen fertilized agricultural soil conditions. By profiling their above- and belowground (a)biotic responses our work reveals that, while increased Casparian strip formation mainly provides the plant with an improved stress resistance, the Schengen pathway is necessary for establishment of a growth-promoting root microbiome and serves to convey information of soil nitrogen status to the shoot. This identifies the Schengen pathway as an essential receptor-based signaling hub for adaptive integration of barrier status, nutritional responses and (a)biotic signaling between above- and belowground tissues.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549085">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549085" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549085">
        <p class="paperTitle">Compositae-ParaLoss-1272: Complementary sunflower specific probe-set reduces issues with paralogs in complex systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549085" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549085" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moore-Pollard, E. R.; Jones, D. S.; Mandel, J. R.</p>
        <p class="info">Score: 5.9, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549085' target='https://doi.org/10.1101/2023.07.19.549085'> 10.1101/2023.07.19.549085</a></p>
        <p class="abstract">Premise: The sunflower family specific probe set, Compositae-1061, has enabled family-wide phylogenomic studies and investigations at lower-taxonomic levels by targeting 1,000&#43; genes. However, it generally lacks resolution at the genus to species level, especially in groups with complex evolutionary histories including polyploidy and hybridization. Methods: In this study, we developed a new HybSeq probe set, Compositae-ParaLoss-1272, designed to target orthologous loci in Asteraceae family members. We tested its efficiency across the family by simulating target-enrichment sequencing in silico. Additionally, we tested its effectiveness at lower taxonomic levels in genus Packera which has a complex evolutionary and taxonomic history. We performed HybSeq with Compositae-ParaLoss-1272 for 19 taxa which were previously studied using the Compositae-1061 probe set. Sequences from both probe sets were used to generate phylogenies, compare topologies, and assess node support. Results: We report that Compositae-ParaLoss-1272 captured loci across all tested Asteraceae members. Additionally, Compositae-ParaLoss-1272 had less gene tree discordance, recovered considerably fewer paralogous sequences, and retained longer loci than Compositae-1061. Discussion: Given the complexity of plant evolutionary histories, assigning orthology for phylogenomic analyses will continue to be challenging. However, we anticipate this new probe set will provide improved resolution and utility for studies at lower-taxonomic levels and complex groups in the sunflower family.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549748">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549748" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549748">
        <p class="paperTitle">Phenomic vs Genomic Prediction - A Comparison of Prediction Accuracies for Grain Yield in Hard Winter Wheat Lines</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549748" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549748" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Winn, Z. J.; Amsberry, A.; Haley, S. D.; DeWitt, N. D.; Mason, E.</p>
        <p class="info">Score: 5.6, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549748' target='https://doi.org/10.1101/2023.07.19.549748'> 10.1101/2023.07.19.549748</a></p>
        <p class="abstract">Common bread wheat (Triticum aestivum L.) is a key component of global diets, but the genetic improvement of wheat is not keeping pace with the growing demands of the world&#39;s population. To increase efficiency and reduce costs, breeding programs are rapidly adopting the use of unoccupied aerial vehicles (UAVs) to conduct high-throughput spectral analyses. This study examined the effectiveness of multispectral indices in predicting grain yield compared to genomic prediction. Multispectral data were collected on advanced generation yield nursery trials during the 2019-2021 growing seasons in the Colorado State University Wheat Breeding Program. Genome-wide genotyping was performed on these advanced generations and all plots were harvested to measure grain yield. Two methods were used to predict grain yield: genomic estimated breeding values (GEBVs) generated by a genomic best linear unbiased prediction (gBLUP) model and phenomic phenotypic estimates (PPEs) using only spectral indices via multiple linear regression (MLR), k-nearest neighbors (KNN), and random forest (RF) models. In cross-validation, PPEs produced by MLR, KNN, and RF models had higher prediction accuracy (r: 0.41 [&amp;le;] r [&amp;le;] 0.48) than GEBVs produced by gBLUP (r = 0.35). In leave-one-year-out forward validation using only multispectral data for 2020 and 2021, PPEs from MLR and KNN models had higher prediction accuracy of grain yield than GEBVs of those same lines. These findings suggest that a limited number of spectra may produce PPEs that are more accurate than or equivalently accurate as GEBVs derived from gBLUP, and this method should be evaluated in earlier development material where sequencing is not feasible.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549592">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549592" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549592">
        <p class="paperTitle">Resolving evolutionary relationships in the groundsels: phylogenomics, divergence time estimates, and biogeography of Packera (Asteraceae: Senecioneae)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549592" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549592" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moore-Pollard, E. R.; Mandel, J. R.</p>
        <p class="info">Score: 5.0, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549592' target='https://doi.org/10.1101/2023.07.18.549592'> 10.1101/2023.07.18.549592</a></p>
        <p class="abstract">The genus Packera belongs to the sunflower family and contains an estimated 64 species and varieties endemic to North America. Some Packera are known to hybridize or exhibit polyploidy, making it difficult to reconstruct evolutionary relationships within the group. Previous molecular phylogenetic studies of Packera employing ITS data recovered low resolution trees, providing little information on the evolutionary relationships within this complex genus. Therefore, we used next-generation sequencing data to infer nuclear and plastid phylogenies of Packera and related Senecioneae taxa. The nuclear phylogeny was calibrated to produce a timetree, then used to reconstruct the macroevolutionary history of Packera, including its historical biogeography. We then compared the reconstructed evolutionary history to previously published scenarios based on phylogenetic and geohistorical data. We found that the nuclear and plastid phylogenies were highly incongruent, with the nuclear tree presenting higher resolution than the plastid tree which had an apparent lack of plastid diversity. The nuclear tree indicated that geography may have played a major role in the evolution and taxonomic diversification of Packera. The estimated origin of Packera at approximately 19.2MY - 25.9MY (late Oligocene to early Miocene) is older than in most other studies. Nonetheless, it aligns well with previous geohistorical predictions, which suggest that speciation and diversification events in Packera were driven by changes in geography and climate in North America. Moreover, Packera likely originated in the western United States or Mexico, and subsequently diversified north and east into the rest of North America and Russia, in agreement with other studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549167">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549167" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549167">
        <p class="paperTitle">From Asymptomatic to Symptomatic: Multiomics profiling of the temporal response of grapevine viral-mixed infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549167" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549167" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lamine Fall, M.; Poursalavati, A.; Sidibe, A.; Xu, D.; Lemoyne, P.; Silva Martins, G.; Jalali Javaran, V.; Moffett, P.; Odile, C.</p>
        <p class="info">Score: 3.5, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549167' target='https://doi.org/10.1101/2023.07.17.549167'> 10.1101/2023.07.17.549167</a></p>
        <p class="abstract">Mixed viral infections are common in grapevines. However, our understanding of the factors and signaling pathways that influence the expression of viral symptoms in mixed infections is still incomplete. In a previous study, we revealed that the presence of grapevine leafroll-associated virus species in mixed infections was randomly associated with the devel-opment of virus-like symptoms. To understand what drives the timing of these virus-like symptoms in mixed infections, we used dsRNA and total RNA sequencing and metabolomic analysis to profile the viromes, metabolites, and transcripts of grapevine leaves collected at two different times of the year (summer and autumn). We demonstrated that neither viral titre nor virome composition changes were associated with symptom expression in autumn. The total phenolic content and antioxidant capacity increased in most plants except for those with early onset symptoms. According to the results of differential gene expression analysis, cell wall biosynthesis pathways were significantly downregulated in all grapevine plants infected with grapevine leafroll-associated virus 3, grapevine asteroid mosaic-associated virus, and grape-vine Pinot gris virus. In addition, polyketide pathways were significantly upregulated in all cultivars, while flavonoid precursor (e.g. abscisic acid) production was significantly reduced in plants that expressed strong virus-like symptoms. In the  Vidal cultivar, an uncharacterized double-stranded RNA-binding protein (DRB) appears to play a critical role in the plants an-tiviral defences, supporting the recent hypothesis that DRBs make an important contribution to dominant antiviral responses in plants. The seasonality of the expression virus-like symptoms appears to be a consequence of the dynamic interactions between antiviral factors and viral counter-defences that occur at different developmental stages of grapevine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549560">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549560" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549560">
        <p class="paperTitle">Development of a mobile, high-throughput, and low-cost image-based plant growth phenotyping system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549560" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549560" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yu, L.; Sussman, H.; Khmelnitsky, O.; Ishka, M. R.; Srinivasan, A.; Nelson, A.; Julkowska, M. M.</p>
        <p class="info">Score: 3.4, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549560' target='https://doi.org/10.1101/2023.07.18.549560'> 10.1101/2023.07.18.549560</a></p>
        <p class="abstract">Nondestructive plant phenotyping is fundamental for unraveling molecular processes underlying plant development and response to the environment. While the emergence of high-through phenotyping facilities can further our understanding of plant development and stress responses, their high costs significantly hinder scientific progress. To democratize high-throughput plant phenotyping, we developed sets of low-cost image- and weight-based devices to monitor plant growth and evapotranspiration. We paired these devices with a suite of computational pipelines for integrated and straightforward data analysis. We validated the suitability of our system for large screens by evaluating a cowpea diversity panel for responses to drought stress. The observed natural variation was subsequently used for Genome-Wide Association Study, where we identified nine genetic loci that putatively contribute to cowpea drought resilience during early vegetative development. We validated the homologs of the identified candidate genes in Arabidopsis using available mutant lines. These results demonstrate the varied applicability of this low-cost phenotyping system. In the future, we foresee these setups facilitating identification of genetic components of growth, plant architecture, and stress tolerance across a wide variety of species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549336">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549336" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549336">
        <p class="paperTitle">Annual dynamics and distribution of Xylella fastidiosa in infected almond trees</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549336" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549336" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zecharia, N.; Vanunu, M.; Dror, O.; Hatib, K.; Holland, D.; Shtienberg, D.; Bahar, O.</p>
        <p class="info">Score: 2.0, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549336' target='https://doi.org/10.1101/2023.07.17.549336'> 10.1101/2023.07.17.549336</a></p>
        <p class="abstract">This research focused on studying the dynamics of the bacterial pathogen Xylella fastidiosa in almond trees at different developmental stages and in various tree parts. The objective was to understand the annual distribution and concentration of X. fastidiosa within almond trees. Different tree parts, including leaf petioles, annual and perennial shoots, fruit parts, flowers, and roots, from ten X. fastidiosa-infected almond trees were sampled over two years. The distribution and concentration of X. fastidiosa were determined using qPCR and serial dilution plating. Throughout the study, X. fastidiosa was never found in the fruit, flowers, and roots of almond trees, but it was present in leaves and annual and perennial shoots. We show that the inability of X. fastidiosa to colonize roots is likely due to incompatibility with the GF677 rootstock. The presence of X. fastidiosa in shoots remained consistent throughout the year, while in leaf petioles it varied across developmental stages, with lower detection during early and late stages of the season. Similarly, viable X. fastidiosa cells could be isolated from shoots at all developmental stages, while in leaf petioles no successful isolations were achieved during the vegetative and nut growth stage. Examining the development of almond leaf scorch symptoms over time in trees with preliminary infections revealed that once symptoms have appeared on a single branch, other asymptomatic limbs were likely already colonized by the bacterium, hence, selective pruning of symptomatic branches is unlikely to cure the tree. Overall, this study enhances our understanding of X. fastidiosa dynamics in almonds and may have practical applications for its detection and control in almond orchards.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548917">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548917" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548917">
        <p class="paperTitle">Origin and diversity of Capsella bursa-pastoris from the genomic point view.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548917" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548917" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Penin, A. A.; Kasianov, A. S.; Klepikova, A. V.; Omelchenko, D. O.; Makarenko, M. S.; Logacheva, M. D.</p>
        <p class="info">Score: 2.0, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548917' target='https://doi.org/10.1101/2023.07.13.548917'> 10.1101/2023.07.13.548917</a></p>
        <p class="abstract">BackgroundCapsella bursa-pastoris, a cosmopolitan weed of hybrid origin, is an emerging model object for the study of early consequences of polyploidy, being a fast growing annual and a close relative of Arabidopsis thaliana. The development of this model is hampered by the absence of a reference genome sequence.

Resultswe present here a subgenome-resolved chromosome-scale assembly and a genetic map of the genome of Capsella bursa-pastoris. It shows that the subgenomes are mostly colinear, with no massive deletions, insertions or rearrangements in any of them. A subgenome-aware annotation reveals the lack of genome dominance - both subgenomes carry similar number of genes. While most chromosomes can be unambiguously recognized as derived from either paternal or maternal parent, we also found homeologous exchange between two chromosomes. It led to an emergence of two hybrid chromosomes; this event is shared between distant populations of C. bursa-pastoris. The whole-genome analysis of 119 samples belonging to C. bursa-pastoris and its parental species C. grandiflora/rubella and C. orientalis reveals introgression from C. orientalis but not from C. grandiflora/rubella.

ConclusionsC. bursa-pastoris do not show genome dominance. In the earliest stages of evolution of this species a homeologous exchange occurred; its presence in all present-day populations of C. bursa-pastoris indicates on a single origin of this species. The evidence coming from whole-genome analysis challenges the current view that C. grandiflora/rubella was a direct progenitor of C. bursa-pastoris; we hypothesize that it was an extinct (or undiscovered) species sister to C. grandiflora/rubella.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
