<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Turn-On Protein Switches for Controlling Actin Binding in Cells
Authors: Effiong, U. M.; Khairandish, H.; Ramirez-Velez, I.; Wang, Y.; Belardi, B.
Score: 9.9, Published: 2023-10-26 DOI: 10.1101/2023.10.26.561921
Within a shared cytoplasm, filamentous actin (F-actin) plays numerous and critical roles across the cell body. Cells rely on actin-binding proteins (ABPs) to organize F-actin and to integrate its polymeric characteristics into diverse cellular processes. Yet, the multitude of ABPs that engage with and shape F-actin make studying a single ABPs influence on cellular activities a significant challenge.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Turn-On Protein Switches for Controlling Actin Binding in Cells
Authors: Effiong, U. M.; Khairandish, H.; Ramirez-Velez, I.; Wang, Y.; Belardi, B.
Score: 9.9, Published: 2023-10-26 DOI: 10.1101/2023.10.26.561921
Within a shared cytoplasm, filamentous actin (F-actin) plays numerous and critical roles across the cell body. Cells rely on actin-binding proteins (ABPs) to organize F-actin and to integrate its polymeric characteristics into diverse cellular processes. Yet, the multitude of ABPs that engage with and shape F-actin make studying a single ABPs influence on cellular activities a significant challenge." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-08T10:39:23+00:00" />
<meta property="article:modified_time" content="2023-11-08T10:39:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Turn-On Protein Switches for Controlling Actin Binding in Cells
Authors: Effiong, U. M.; Khairandish, H.; Ramirez-Velez, I.; Wang, Y.; Belardi, B.
Score: 9.9, Published: 2023-10-26 DOI: 10.1101/2023.10.26.561921
Within a shared cytoplasm, filamentous actin (F-actin) plays numerous and critical roles across the cell body. Cells rely on actin-binding proteins (ABPs) to organize F-actin and to integrate its polymeric characteristics into diverse cellular processes. Yet, the multitude of ABPs that engage with and shape F-actin make studying a single ABPs influence on cellular activities a significant challenge."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Turn-On Protein Switches for Controlling Actin Binding in Cells\nAuthors: Effiong, U. M.; Khairandish, H.; Ramirez-Velez, I.; Wang, Y.; Belardi, B.\nScore: 9.9, Published: 2023-10-26 DOI: 10.1101/2023.10.26.561921\nWithin a shared cytoplasm, filamentous actin (F-actin) plays numerous and critical roles across the cell body. Cells rely on actin-binding proteins (ABPs) to organize F-actin and to integrate its polymeric characteristics into diverse cellular processes. Yet, the multitude of ABPs that engage with and shape F-actin make studying a single ABPs influence on cellular activities a significant challenge.",
  "keywords": [
    
  ],
  "articleBody": " Turn-On Protein Switches for Controlling Actin Binding in Cells\nAuthors: Effiong, U. M.; Khairandish, H.; Ramirez-Velez, I.; Wang, Y.; Belardi, B.\nScore: 9.9, Published: 2023-10-26 DOI: 10.1101/2023.10.26.561921\nWithin a shared cytoplasm, filamentous actin (F-actin) plays numerous and critical roles across the cell body. Cells rely on actin-binding proteins (ABPs) to organize F-actin and to integrate its polymeric characteristics into diverse cellular processes. Yet, the multitude of ABPs that engage with and shape F-actin make studying a single ABPs influence on cellular activities a significant challenge. Moreover, without a means of manipulating actin-binding subcellularly, harnessing the F-actin cytoskeleton for synthetic biology purposes remains elusive. Here, we describe a suite of designed proteins, Controllable Actin-binding Switch Tools (CASTs), whose actin-binding behavior can be controlled with external stimuli. CASTs were developed that respond to different external inputs, providing options for turn-on kinetics and enabling orthogonality. Being genetically encoded, we show that CASTs can be inserted into native protein sequences to control F-actin association locally and engineered into new structures to control cell and tissue shape and behavior.\nA modular toolbox for the optogenetic deactivation of transcription\nAuthors: Muench, P.; Fiumara, M.; Southern, N.; Coda, D.; Aschenbrenner, S.; Correia, B.; Graeff, J.; Niopek, D.; Mathony, J.\nScore: 9.4, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565805\nLight-controlled transcriptional activation is a commonly used optogenetic strategy that allows researchers to regulate gene expression with high spatiotemporal precision. The vast majority of existing tools are, however, limited to light-triggered induction of gene expression. Here, we inverted this mode of action and created two complementary optogenetic systems capable of efficiently terminating transcriptional activation in response to blue light. First, we designed highly compact regulators, by photo-controlling VP16 transactivation peptide exposure. Then, applying a two-hybrid strategy, we engineered LOOMINA (light off-operated modular inductor of transcriptional activation), a versatile transcriptional control platform for mammalian cells that is highly adaptable and compatible with various effector proteins. Leveraging the flexibility of CRISPR systems, we integrated LOOMINA with Cas9 as a DNA-binding domain to control transcription from various endogenous promoters with exceptionally high dynamic ranges in multiple cell lines, including neuron-like cells. Both functionally and mechanistically, LOOMINA represents a valuable addition to the optogenetic repertoire for transcriptional regulation.\nSimplified post-assembly plasmid library amplification for increased transformation yields in E. coli and S. cerevisiae\nAuthors: Fryer, T.; Wolff, D. S.; Overath, M. D.; Schaefer, E.; Laustsen, A. H.; Jenkins, T. P.; Andersen, C.\nScore: 5.1, Published: 2023-11-02 DOI: 10.1101/2023.11.02.565362\nMany biological disciplines rely upon the transformation of host cells with heterologous DNA, the limited efficiency of which can significantly hinder a researchers work. Directed evolution in particular typically requires the screening of large (thousand-billion member) libraries to identify sequences of interest, and the creation of these libraries at large enough scales to overcome transformation inefficiencies is a cost-and time-intensive process. We simplify this process by using Rolling Circle Amplification (RCA) to amplify in vitro plasmid DNA assembly reactions, followed by facile resolution of the concatomeric products to monomers through treatment with specific endonucleases and subsequent efficient transformation of the linear DNA products into host cells for in vivo circularisation. We demonstrate that use of a nicking endonuclease to generate homologous single-stranded ends increases the efficiency of E. coli chemical transformation versus both linear DNA with double-stranded homologous ends, and circular golden-gate assembly products, whilst use of a restriction endonuclease to generate linear DNA with double-stranded homologous ends increases the efficiency of chemical and electrotransformation of S. cerevisiae. Importantly, we also optimise the process such that both RCA and endonuclease treatment occur efficiently in the same buffer, streamlining the workflow and reducing product loss through purification steps. We expect our approach to have utility beyond directed evolution in E. coli and S. cerevisiae, to areas such as genome engineering and the manipulation of alternative organisms with even poorer transformation efficiencies.\nBatCRISPRi: Bacillus titratable CRISPRi for dynamic control in Bacillus subtilis\nAuthors: Florez, A. F.; Castillo-Hair, S.; Gutierrez-Lopez, L.; Eaton, D.; Paulsson, J.; Garner, E. C.\nScore: 4.7, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565046\nThe discovery of new genes regulating essential biological processes has become increasingly important, and CRISPRi has emerged as a powerful tool for achieving this goal. This method has been used in many model organisms to decrease the expression of specific genes and assess their impact on phenotype. Pooled CRISPRi libraries in bacteria have been particularly useful in discovering new regulators of growth, division, and other biological processes. However, these libraries rely on the induction of dCas9 via an inducible promoter, which can be problematic due to promoter leakiness. This is a widespread phenomenon of any inducible promoter that can result in the unwanted downregulation of genes and the emergence of genetic suppressors when essential genes are knocked down. To overcome this issue, we have developed a novel strategy that eliminates dCas9 leakiness and enables reversible knockdown control using the rapamycin-dependent degron system in Bacillus subtilis. This degron system causes rapid degradation of dCas9, resulting in an almost instant reset of the system. Our results demonstrate that it is possible to achieve zero CRISPRi activity in the uninduced state and full activity in the induced state. This improved CRISPRi system will enable researchers to investigate phenotypic changes more effectively while reducing the undesirable effects of leaky expression and noise in their phenotypic data. Moreover, a rapid degradation system could serve as a tool for dynamic perturbation before compensation mechanisms or stress responses kick in. Finally, this approach can be adapted to other organisms and other promoter-inducible systems, potentially opening up strategies for tighter control of gene expression.\nDecellularized Banana Leaves: Eco-Friendly Scaffolds for Cell-Based Seafood\nAuthors: Sarkarat, R.; Banavar, A.; Amirvaresi, A.; Li, X.; Nguyen, C.; Kaplan, D. L.; Nitin, N.; Ovissipour, R.\nScore: 1.8, Published: 2023-11-03 DOI: 10.1101/2023.11.03.565310\nCellular agriculture, as an emerging food production system, holds potential to address sustainability, food security, and agricultural resilience. Within the cell-based meat supply chain, one of the key steps is scaffolding. In this study, we assessed decellularized banana leaves, various coating materials, and different cell seeding strategies to determine their effects on cell viability, cell growth, cell alignment, and the response of the materials to thermal processing. The efficiency of decellularization was verified through DNA quantification, which decreased from 445 ng/mg in fresh banana leaves to non-detectable levels in the decellularized samples. This was further confirmed by FTIR and PCA modeling. Cell viability exceeded 98% on uncoated, soy-coated, and gelatin-coated samples of the decellularized banana leaves. Alignment of cells on gelatin-coated samples was the highest among the samples, with a dominant orientation of 65.8{degrees}, compared to soy-coated and uncoated samples with dominant orientations of 9.2{degrees} and -6.3{degrees}, respectively. In terms of quality attributes, the kinetics of shrinkage indicated that coating with soy and the presence of cells increased the activation energy due to the higher energy required for protein denaturation. Moreover, the kinetics of area changes in plain scaffolds without cells followed a first-order pattern, while with seeded cells a second-order pattern was followed. In summary, decellularized banana leaves present a sustainable and suitable biomaterial to support cells towards future needs related to meat production.\nProtein interaction kinetics delimit the performance of phosphorylation-driven protein switches.\nAuthors: Winter, D. L.; Wairara, A. R.; Bennett, J. L.; Donald, W. A.; Glover, D. J.\nScore: 1.2, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565761\nPost-translational modifications (PTMs) such as phosphorylation and dephosphorylation can rapidly alter protein surface chemistry and structural conformation which can, in turn, switch protein-protein interactions (PPIs) within signaling networks. Recently, de novo designed phosphorylation-responsive protein switches have been created that harness kinase- and phosphatase-mediated phosphorylation that modulate PPIs. PTM-driven protein switches could be useful for investigating PTM dynamics in living cells, developing biocompatible nanodevices, and engineering signaling pathways to program cell behavior. However, little is known about the physical and kinetic constraints of PTM-driven protein switches, which limits their practical application. In this study, we present a theoretical framework to evaluate two-component PTM-driven protein switches based on four performance metrics: effective concentration, dynamic range, response time, and reversibility. Our computational models reveal an intricate relationship between the binding kinetics, phosphorylation kinetics, and switch concentration that governs the sensitivity and reversibility of PTM-driven protein switches. Building upon the insights of our theoretical investigation, we built and evaluated two novel phosphorylation-driven protein switches consisting of phosphorylation-sensitive coiled coils as sensor domains fused to fluorescent proteins as actuator domains. By modulating the phosphorylation state of the switches with a specific protein kinase and phosphatase, we demonstrate fast, reversible transitions between easily differentiated on and off states. The response of the switches linearly correlated to the concentration of the kinase, demonstrating its potential as a biosensor for kinase measurements in real time. As intended, both switches responded to specific kinase activity with an increase in fluorescence signal and our model could be used to distinguish between two mechanisms of switch activation: dimerization or a structural rearrangement. In summary, the protein switch kinetics model presented here should be useful to guide the design of PTM-driven switches and tune their performance towards concrete applications.\n",
  "wordCount" : "1495",
  "inLanguage": "en",
  "datePublished": "2023-11-08T10:39:23Z",
  "dateModified": "2023-11-08T10:39:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 8, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.561921">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.561921" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.561921">
        <p class="paperTitle">Turn-On Protein Switches for Controlling Actin Binding in Cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.561921" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.561921" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Effiong, U. M.; Khairandish, H.; Ramirez-Velez, I.; Wang, Y.; Belardi, B.</p>
        <p class="info">Score: 9.9, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.561921' target='https://doi.org/10.1101/2023.10.26.561921'> 10.1101/2023.10.26.561921</a></p>
        <p class="abstract">Within a shared cytoplasm, filamentous actin (F-actin) plays numerous and critical roles across the cell body. Cells rely on actin-binding proteins (ABPs) to organize F-actin and to integrate its polymeric characteristics into diverse cellular processes. Yet, the multitude of ABPs that engage with and shape F-actin make studying a single ABPs influence on cellular activities a significant challenge. Moreover, without a means of manipulating actin-binding subcellularly, harnessing the F-actin cytoskeleton for synthetic biology purposes remains elusive. Here, we describe a suite of designed proteins, Controllable Actin-binding Switch Tools (CASTs), whose actin-binding behavior can be controlled with external stimuli. CASTs were developed that respond to different external inputs, providing options for turn-on kinetics and enabling orthogonality. Being genetically encoded, we show that CASTs can be inserted into native protein sequences to control F-actin association locally and engineered into new structures to control cell and tissue shape and behavior.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565805">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565805" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565805">
        <p class="paperTitle">A modular toolbox for the optogenetic deactivation of transcription</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565805" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565805" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Muench, P.; Fiumara, M.; Southern, N.; Coda, D.; Aschenbrenner, S.; Correia, B.; Graeff, J.; Niopek, D.; Mathony, J.</p>
        <p class="info">Score: 9.4, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565805' target='https://doi.org/10.1101/2023.11.06.565805'> 10.1101/2023.11.06.565805</a></p>
        <p class="abstract">Light-controlled transcriptional activation is a commonly used optogenetic strategy that allows researchers to regulate gene expression with high spatiotemporal precision. The vast majority of existing tools are, however, limited to light-triggered induction of gene expression. Here, we inverted this mode of action and created two complementary optogenetic systems capable of efficiently terminating transcriptional activation in response to blue light. First, we designed highly compact regulators, by photo-controlling VP16 transactivation peptide exposure. Then, applying a two-hybrid strategy, we engineered LOOMINA (light off-operated modular inductor of transcriptional activation), a versatile transcriptional control platform for mammalian cells that is highly adaptable and compatible with various effector proteins. Leveraging the flexibility of CRISPR systems, we integrated LOOMINA with Cas9 as a DNA-binding domain to control transcription from various endogenous promoters with exceptionally high dynamic ranges in multiple cell lines, including neuron-like cells. Both functionally and mechanistically, LOOMINA represents a valuable addition to the optogenetic repertoire for transcriptional regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565362">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565362" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565362">
        <p class="paperTitle">Simplified post-assembly plasmid library amplification for increased transformation yields in E. coli and S. cerevisiae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565362" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565362" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fryer, T.; Wolff, D. S.; Overath, M. D.; Schaefer, E.; Laustsen, A. H.; Jenkins, T. P.; Andersen, C.</p>
        <p class="info">Score: 5.1, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565362' target='https://doi.org/10.1101/2023.11.02.565362'> 10.1101/2023.11.02.565362</a></p>
        <p class="abstract">Many biological disciplines rely upon the transformation of host cells with heterologous DNA, the limited efficiency of which can significantly hinder a researchers work. Directed evolution in particular typically requires the screening of large (thousand-billion member) libraries to identify sequences of interest, and the creation of these libraries at large enough scales to overcome transformation inefficiencies is a cost-and time-intensive process. We simplify this process by using Rolling Circle Amplification (RCA) to amplify in vitro plasmid DNA assembly reactions, followed by facile resolution of the concatomeric products to monomers through treatment with specific endonucleases and subsequent efficient transformation of the linear DNA products into host cells for in vivo circularisation. We demonstrate that use of a nicking endonuclease to generate homologous single-stranded ends increases the efficiency of E. coli chemical transformation versus both linear DNA with double-stranded homologous ends, and circular golden-gate assembly products, whilst use of a restriction endonuclease to generate linear DNA with double-stranded homologous ends increases the efficiency of chemical and electrotransformation of S. cerevisiae. Importantly, we also optimise the process such that both RCA and endonuclease treatment occur efficiently in the same buffer, streamlining the workflow and reducing product loss through purification steps. We expect our approach to have utility beyond directed evolution in E. coli and S. cerevisiae, to areas such as genome engineering and the manipulation of alternative organisms with even poorer transformation efficiencies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565046">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565046" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565046">
        <p class="paperTitle">BatCRISPRi: Bacillus titratable CRISPRi for dynamic control in Bacillus subtilis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565046" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565046" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Florez, A. F.; Castillo-Hair, S.; Gutierrez-Lopez, L.; Eaton, D.; Paulsson, J.; Garner, E. C.</p>
        <p class="info">Score: 4.7, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565046' target='https://doi.org/10.1101/2023.11.01.565046'> 10.1101/2023.11.01.565046</a></p>
        <p class="abstract">The discovery of new genes regulating essential biological processes has become increasingly important, and CRISPRi has emerged as a powerful tool for achieving this goal. This method has been used in many model organisms to decrease the expression of specific genes and assess their impact on phenotype. Pooled CRISPRi libraries in bacteria have been particularly useful in discovering new regulators of growth, division, and other biological processes. However, these libraries rely on the induction of dCas9 via an inducible promoter, which can be problematic due to promoter leakiness. This is a widespread phenomenon of any inducible promoter that can result in the unwanted downregulation of genes and the emergence of genetic suppressors when essential genes are knocked down. To overcome this issue, we have developed a novel strategy that eliminates dCas9 leakiness and enables reversible knockdown control using the rapamycin-dependent degron system in Bacillus subtilis. This degron system causes rapid degradation of dCas9, resulting in an almost instant reset of the system. Our results demonstrate that it is possible to achieve zero CRISPRi activity in the uninduced state and full activity in the induced state. This improved CRISPRi system will enable researchers to investigate phenotypic changes more effectively while reducing the undesirable effects of leaky expression and noise in their phenotypic data. Moreover, a rapid degradation system could serve as a tool for dynamic perturbation before compensation mechanisms or stress responses kick in. Finally, this approach can be adapted to other organisms and other promoter-inducible systems, potentially opening up strategies for tighter control of gene expression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565310">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565310" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565310">
        <p class="paperTitle">Decellularized Banana Leaves: Eco-Friendly Scaffolds for Cell-Based Seafood</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565310" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565310" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sarkarat, R.; Banavar, A.; Amirvaresi, A.; Li, X.; Nguyen, C.; Kaplan, D. L.; Nitin, N.; Ovissipour, R.</p>
        <p class="info">Score: 1.8, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565310' target='https://doi.org/10.1101/2023.11.03.565310'> 10.1101/2023.11.03.565310</a></p>
        <p class="abstract">Cellular agriculture, as an emerging food production system, holds potential to address sustainability, food security, and agricultural resilience. Within the cell-based meat supply chain, one of the key steps is scaffolding. In this study, we assessed decellularized banana leaves, various coating materials, and different cell seeding strategies to determine their effects on cell viability, cell growth, cell alignment, and the response of the materials to thermal processing. The efficiency of decellularization was verified through DNA quantification, which decreased from 445 ng/mg in fresh banana leaves to non-detectable levels in the decellularized samples. This was further confirmed by FTIR and PCA modeling. Cell viability exceeded 98% on uncoated, soy-coated, and gelatin-coated samples of the decellularized banana leaves. Alignment of cells on gelatin-coated samples was the highest among the samples, with a dominant orientation of 65.8{degrees}, compared to soy-coated and uncoated samples with dominant orientations of 9.2{degrees} and -6.3{degrees}, respectively. In terms of quality attributes, the kinetics of shrinkage indicated that coating with soy and the presence of cells increased the activation energy due to the higher energy required for protein denaturation. Moreover, the kinetics of area changes in plain scaffolds without cells followed a first-order pattern, while with seeded cells a second-order pattern was followed. In summary, decellularized banana leaves present a sustainable and suitable biomaterial to support cells towards future needs related to meat production.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565761">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565761" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565761">
        <p class="paperTitle">Protein interaction kinetics delimit the performance of phosphorylation-driven protein switches.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565761" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565761" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Winter, D. L.; Wairara, A. R.; Bennett, J. L.; Donald, W. A.; Glover, D. J.</p>
        <p class="info">Score: 1.2, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565761' target='https://doi.org/10.1101/2023.11.06.565761'> 10.1101/2023.11.06.565761</a></p>
        <p class="abstract">Post-translational modifications (PTMs) such as phosphorylation and dephosphorylation can rapidly alter protein surface chemistry and structural conformation which can, in turn, switch protein-protein interactions (PPIs) within signaling networks. Recently, de novo designed phosphorylation-responsive protein switches have been created that harness kinase- and phosphatase-mediated phosphorylation that modulate PPIs. PTM-driven protein switches could be useful for investigating PTM dynamics in living cells, developing biocompatible nanodevices, and engineering signaling pathways to program cell behavior. However, little is known about the physical and kinetic constraints of PTM-driven protein switches, which limits their practical application. In this study, we present a theoretical framework to evaluate two-component PTM-driven protein switches based on four performance metrics: effective concentration, dynamic range, response time, and reversibility. Our computational models reveal an intricate relationship between the binding kinetics, phosphorylation kinetics, and switch concentration that governs the sensitivity and reversibility of PTM-driven protein switches. Building upon the insights of our theoretical investigation, we built and evaluated two novel phosphorylation-driven protein switches consisting of phosphorylation-sensitive coiled coils as sensor domains fused to fluorescent proteins as actuator domains. By modulating the phosphorylation state of the switches with a specific protein kinase and phosphatase, we demonstrate fast, reversible transitions between easily differentiated on and off states. The response of the switches linearly correlated to the concentration of the kinase, demonstrating its potential as a biosensor for kinase measurements in real time. As intended, both switches responded to specific kinase activity with an increase in fluorescence signal and our model could be used to distinguish between two mechanisms of switch activation: dimerization or a structural rearrangement. In summary, the protein switch kinetics model presented here should be useful to guide the design of PTM-driven switches and tune their performance towards concrete applications.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
