<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Extreme miniaturization in plasmid design: generation of the 903 bp cloning vector pICOt2
Authors: Staal, J.; Beyaert, R.
Score: 37.5, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569326
Inspired by the size optimizations in the recently published pUdO series of plasmids, we set out to break our previous world record (pICOz, 1185 bp) in designing the smallest possible self-replicating high copy cloning vector. By replacing the zeocin resistance marker with trimethoprim resistance, deleting the parts upstream of -35 in the ampicilin resistance promoter and replacing the extended multiple cloning site with a single SpeI site, we managed to reduce the plasmid size to 903 bp, almost a 24% reduction in size compared to our previous world record.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Extreme miniaturization in plasmid design: generation of the 903 bp cloning vector pICOt2
Authors: Staal, J.; Beyaert, R.
Score: 37.5, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569326
Inspired by the size optimizations in the recently published pUdO series of plasmids, we set out to break our previous world record (pICOz, 1185 bp) in designing the smallest possible self-replicating high copy cloning vector. By replacing the zeocin resistance marker with trimethoprim resistance, deleting the parts upstream of -35 in the ampicilin resistance promoter and replacing the extended multiple cloning site with a single SpeI site, we managed to reduce the plasmid size to 903 bp, almost a 24% reduction in size compared to our previous world record." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-03T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-12-03T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Extreme miniaturization in plasmid design: generation of the 903 bp cloning vector pICOt2
Authors: Staal, J.; Beyaert, R.
Score: 37.5, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569326
Inspired by the size optimizations in the recently published pUdO series of plasmids, we set out to break our previous world record (pICOz, 1185 bp) in designing the smallest possible self-replicating high copy cloning vector. By replacing the zeocin resistance marker with trimethoprim resistance, deleting the parts upstream of -35 in the ampicilin resistance promoter and replacing the extended multiple cloning site with a single SpeI site, we managed to reduce the plasmid size to 903 bp, almost a 24% reduction in size compared to our previous world record."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Extreme miniaturization in plasmid design: generation of the 903 bp cloning vector pICOt2\nAuthors: Staal, J.; Beyaert, R.\nScore: 37.5, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569326\nInspired by the size optimizations in the recently published pUdO series of plasmids, we set out to break our previous world record (pICOz, 1185 bp) in designing the smallest possible self-replicating high copy cloning vector. By replacing the zeocin resistance marker with trimethoprim resistance, deleting the parts upstream of -35 in the ampicilin resistance promoter and replacing the extended multiple cloning site with a single SpeI site, we managed to reduce the plasmid size to 903 bp, almost a 24% reduction in size compared to our previous world record.",
  "keywords": [
    
  ],
  "articleBody": " Extreme miniaturization in plasmid design: generation of the 903 bp cloning vector pICOt2\nAuthors: Staal, J.; Beyaert, R.\nScore: 37.5, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569326\nInspired by the size optimizations in the recently published pUdO series of plasmids, we set out to break our previous world record (pICOz, 1185 bp) in designing the smallest possible self-replicating high copy cloning vector. By replacing the zeocin resistance marker with trimethoprim resistance, deleting the parts upstream of -35 in the ampicilin resistance promoter and replacing the extended multiple cloning site with a single SpeI site, we managed to reduce the plasmid size to 903 bp, almost a 24% reduction in size compared to our previous world record.\nAn interdependent Metabolic and Genetic Network shows emergent properties in vitro\nAuthors: Giaveri, S.; Bohra, N.; Diehl, C.; Ballinger, M.; Paczia, N.; Glatter, T.; Erb, T. J.\nScore: 25.2, Published: 2023-11-26 DOI: 10.1101/2023.11.26.568713\nA hallmark of all living organisms is their ability for self-regeneration which requires a tight integration of metabolic and genetic networks. Here we constructed a metabolic and genetic linked in vitro network (MGLN) that shows life-like behavior outside of a cellular context and generates its own building blocks from non-living matter. To this end, we integrated the metabolism of the crotonyl-CoA/ethyl-malonyl-CoA/hydroxybutyryl-CoA (CETCH) cycle with cell-free protein synthesis using recombinant elements (PURE). We demonstrate that the MGLN produces the essential amino acid glycine from inorganic carbon (CO2), and incorporates it into target proteins following DNA-encoded instructions. By programming genetically encoded response into metabolic networks our work opens new avenues for the development of advanced biomimetic systems with emergent properties, including decision-making, self-regeneration and evolution.\nA library of reporters of the global regulators of gene expression of Escherichia coli\nAuthors: Dash, S.; Jagadeesan, R.; Baptista, I. S. C.; Chauhan, V.; Kandavalli, V.; Oliveira, S. M. D.; Ribeiro, A. S.\nScore: 16.9, Published: 2023-11-28 DOI: 10.1101/2023.11.28.568972\nThe topology of the transcription factor network (TFN) of E. coli is far from uniform, with 22 global regulator (GR) proteins controlling one-third of all genes. So far, their production rates cannot be tracked by comparable fluorescent proteins. We developed a library of fluorescent reporters for 16 GRs for this purpose. Each consists of a single-copy plasmid coding for GFP fused to the full-length copy of the native promoter. We tracked their activity in exponential and stationary growth, as well as under weak and strong stresses. We show that the reporters have high sensitivity and specificity to all stresses tested and detect single-cell variability in transcription rates. Given the influence of GRs on the TFN, we expect that the new library will contribute to dissecting global transcriptional stress-response programs of E. coli. Moreover, the library can be invaluable in bioindustrial applications that tune those programs to, instead of cell growth, favor productivity while reducing energy consumption. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=200 SRC=\"FIGDIR/small/568972v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (46K): org.highwire.dtl.DTLVardef@f63b62org.highwire.dtl.DTLVardef@1b4d0a9org.highwire.dtl.DTLVardef@1ef2afforg.highwire.dtl.DTLVardef@effd94_HPS_FORMAT_FIGEXP M_FIG C_FIG\nAutotrophic growth of E. coli is achieved by a small number of genetic changes\nAuthors: Ben Nissan, R.; Milshtein, E.; Pahl, V.; de Pins, B.; Jona, G.; Levi, D.; Yung, H.; Nir, N.; Ezra, D.; Gleizer, S.; Link, H.; Noor, E.; Milo, R.\nScore: 56.9, Published: 2023-11-26 DOI: 10.1101/2023.06.03.543535\nSynthetic autotrophy is a promising avenue to sustainable bioproduction from CO2. Here, we use iterative laboratory evolution to generate several distinct autotrophic strains. Utilising this genetic diversity, we identify that just three mutations are sufficient for E. coli to grow autotrophically, when introduced alongside non-native energy (formate dehydrogenase) and carbon-fixing (RuBisCO, phosphoribulokinase, carbonic anhydrase) modules. The mutated genes are involved in glycolysis (pgi), central-carbon regulation (crp), and RNA transcription (rpoB). The pgi mutation reduces the enzymes activity, thereby stabilising the carbon-fixing cycle by capping a major branching flux. For the other two mutations, we observe down-regulation of several metabolic pathways and increased expression of native genes associated with the carbon-fixing module (rpiB) and the energy module (fdoGH), as well as an increased ratio of NADH/NAD+ - the cycles electron-donor. This study demonstrates the malleability of metabolism and its capacity to switch trophic modes using only a small number of genetic changes and could facilitate transforming other heterotrophic organisms into autotrophs.\nHigh-yield enzymatic synthesis of mono- and trifluorinated alanine enantiomers\nAuthors: Nieto-Dominguez, M.; Sako, A.; Enemark-Rasmussen, K.; Held Gotfredsen, C.; Rago, D.; Nikel, P. I.\nScore: 7.4, Published: 2023-11-28 DOI: 10.1101/2023.11.28.569005\nFluorinated amino acids are a promising entry point for incorporating new-to-Nature chemistries in biological systems. Hence, novel methods are needed for the selective synthesis of these building blocks. In this study, we focused on the enzymatic synthesis of fluorinated alanine enantiomers. To this end, the alanine dehydrogenase from Vibrio proteolyticus and the diaminopimelate dehydrogenase from Symbiobacterium thermophilum were applied to the in vitro production of (R)-3-fluoroalanine and (S)-3-fluoroalanine, respectively, using 3-fluoropyruvate as the substrate. Additionally, an alanine racemase from Streptomyces lavendulae, originally selected for setting an alternative enzymatic cascade leading to the production of these non-canonical amino acids, had an unprecedented catalytic efficiency in the {beta}-elimination of fluorine from the monosubstituted fluoroalanine. The in vitro enzymatic cascade based on the dehydrogenases of V. proteolyticus and S. thermophilum included a cofactor recycling system, whereby a formate dehydrogenase from Pseudomonas sp. 101 (either native or engineered) coupled formate oxidation to NAD(P)H formation. Under these conditions, the reaction yields for (R)-3-fluoroalanine and (S)-3-fluoroalanine reached \u003e85% on the fluorinated substrate and proceeded with complete enantiomeric excess. Moreover, the selected dehydrogenases were also able to catalyze the conversion of trifluoropyruvate into trifluorinated alanine, as a first-case example of biocatalysis with amino acids carrying a trifluoromethyl group.\nBacteriophage-host interactions in microgravity onboard the International Space Station\nAuthors: Huss, P.; Chitboonthavisuk, C.; Meger, A.; Nishikawa, K.; Oates, R. P.; Mills, H.; Holzhaus, O.; Raman, S.\nScore: 21.4, Published: 2023-11-13 DOI: 10.1101/2023.10.10.561409\nBacteriophage-host interactions play a fundamental role in shaping microbial ecosystems. Although researchers have extensively studied phage-host interactions in terrestrial ecosystems, the impact of microgravity on phage-host interactions is yet to be investigated. Here, we report the dynamics of interactions between T7 bacteriophage and E. coli in microgravity onboard the International Space Station (ISS). We found phage activity was delayed but ultimately successful in microgravity. We identified several de novo mutations in phage and bacteria that improved fitness in microgravity. Deep mutational scanning of the receptor binding domain revealed substantial differences in the number, position, and mutational preferences between gravity and microgravity, reflecting underlying differences in the bacterial adaptations. Combinatorial libraries informed by microgravity selections gave T7 mutants with 100-10,000-fold higher activity on uropathogenic E. coli under terrestrial conditions than wildtype T7. Our findings lay the foundation for future research on the impact of microgravity on phage-host interactions and microbial communities.\nDirected Evolution of Genetically Encoded LYTACs for Cell-Mediated Delivery\nAuthors: Yang, J. L.; Yamada-Hunter, S. A.; Labanieh, L.; Sotillo, E.; Cheah, J. S.; Roberts, D. S.; Mackall, C.; Ting, A. Y.; Bertozzi, C.\nScore: 25.7, Published: 2023-11-15 DOI: 10.1101/2023.11.14.567117\nLysosome-targeting chimeras (LYTACs) are a promising therapeutic modality to drive the degradation of extracellular proteins. However, early versions of LYTAC contain synthetic glycopeptides that cannot be genetically encoded. Here we present our designs for a fully genetically encodable LYTAC (GELYTAC), making our tool compatible with integration into therapeutic cells for targeted delivery at diseased sites. To achieve this, we replaced the glycopeptide portion of LYTACs with the protein insulin like growth factor 2 (IGF2). After showing initial efficacy with wild type IGF2, we increased the potency of GELYTAC using directed evolution. Subsequently, we demonstrated that our engineered GELYTAC construct not only secretes from HEK293T cells but also from human primary T-cells to drive the uptake of various targets into receiver cells. Immune cells engineered to secrete GELYTAC thus represent a promising avenue for spatially-selective targeted protein degradation. Significance StatementBetter therapeutic windows can be achieved by targeting therapeutics to their desired sites of action. For protein therapeutics, this might be achieved by engineering cell therapies that home to a tissue of interest and secrete the biologic drug locally. Here, we demonstrate that human primary T cells can be engineered to produce genetically encoded lysosome targeting chimeras (GELYTACs). These GELYTACs mediate the degradation of extracellular proteins associated with cancer progression. Thus, cells engineered to produce GELYTACs represent a potential new class of cancer therapeutics.\nBiofoundry-assisted Golden Gate cloning with AssemblyTron\nAuthors: Bryant, J. A.; Wright, R. C.\nScore: 4.2, Published: 2023-11-29 DOI: 10.1101/2023.11.28.569037\nGolden Gate assembly is a requisite method in synthetic biology that facilitates critical conventions such as genetic part abstraction and rapid prototyping. However, compared to robotic implementation, manual Golden Gate implementation is cumbersome, error-prone, and inconsistent for complex assembly designs. AssemblyTron is an open-source python package that provides an affordable automation solution using open-source Opentrons OT-2 lab robots. Automating Golden Gate assembly with AssemblyTron can reduce failure-rate, resource consumption, and training requirements for building complex DNA constructs, as well as indexed and combinatorial libraries. Here, we dissect a panel of upgrades to AssemblyTrons Golden Gate assembly capabilities, which include Golden Gate assembly into modular cloning part vectors, error-prone PCR combinatorial mutant library assembly, and modular cloning indexed plasmid library assembly. These upgrades enable a broad pool of users with varying levels of experience to readily implement advanced Golden Gate applications using low-cost, open-source lab robotics.\nProfiling Expression Strategies for a Type III Polyketide Synthase in a Lysate-Based, Cell-free System\nAuthors: Sword, T. T.; Dinglasan, J. L. N.; Abbas, G. S. K.; Barker, J. W.; Spradley, M. E.; Greene, E. R.; Gooden, D. S.; Emrich, S. J.; Gilchrist, M. A.; Doktycz, M. J.; Bailey, C. B.\nScore: 2.9, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569483\nSome of the most metabolically diverse species of bacteria (e.g., Actinobacteria) have higher GC content in their DNA, differ substantially in codon usage, and have distinct protein folding environments compared to tractable expression hosts like Escherichia coli. Consequentially, expressing biosynthetic gene clusters (BGCs) from these bacteria in E. coli frequently results in a myriad of unpredictable issues with protein expression and folding, delaying the biochemical characterization of new natural products. Current strategies to achieve soluble, active expression of these enzymes in tractable hosts, such as BGC refactoring, can be a lengthy trial-and-error process. Cell-free expression (CFE) has emerged as 1) a valuable expression platform for enzymes that are challenging to synthesize in vivo, and as 2) a testbed for rapid prototyping that can improve cellular expression. Here, we use a type III polyketide synthase from Streptomyces griseus, RppA, which catalyzes the formation of the red pigment flaviolin, as a reporter to investigate BGC refactoring techniques. We synergistically tune promoter and codon usage to improve flaviolin production from cell-free expressed RppA. We then assess the utility of cell-free systems for prototyping these refactoring tactics prior to their implementation in cells. Overall, codon harmonization improves natural product synthesis more than traditional codon optimization across cell-free and cellular environments. Refactoring promoters and/or coding sequences via CFE can be a valuable strategy to rapidly screen for catalytically functional production of enzymes from BCGs. By showing the coordinators between CFE versus in vivo expression, this work advances CFE as a tool for natural product research.\nGlycodiversification of gentamicins through in vivo glycosyltransferase swapping enabled the creation of novel hybrid aminoglycoside antibiotics with potent activity and low ototoxicity\nAuthors: Jian, X.; Wang, C.; Wu, S.; Sun, G.; Huang, C.; Qiu, C.; Liu, Y.; Leadlay, P. F.; Liu, D.; Deng, Z.; Zhou, F.; Sun, Y.\nScore: 2.5, Published: 2023-11-29 DOI: 10.1101/2023.11.29.569180\nAminoglycosides (AGs) are a class of potent antibiotics with a broad spectrum of activity. However, their use is limited by safety concerns associated with nephrotoxicity and ototoxicity, as well as drug resistance. To address these issues, semi-synthetic approaches for modifying natural AGs have successfully generated new generations of AGs, however, with limited types of modification due to significant challenges in synthesis. This study explores a novel approach that harness the bacterial biosynthetic machinery of gentamicins and kanamycins to create hybrid AGs, installing extensive natural modifications from gentamicins onto kanamycins. This was achieved by glycodiversification of gentamicins via swapping the glycosyltransferase (GT) in their producer with the GT from kanamycins biosynthetic pathway and resulted in the creation of a series of novel AGs with combined structural features of two, therefore referred to as genkamicins (GKs). The manipulation of the hybrid metabolic pathway enabled the target accumulation of different GK species and the successful isolation and characterization of six GK components. These compounds display retained antimicrobial activity against a panel of World Health Organization (WHO) critical priority pathogens, and GK-C2a, in particular, demonstrates low ototoxicity compared to clinical drugs in zebrafish embryos. This study provides a new strategy for diversifying the structure of AGs and a potential avenue for developing less toxic AG drugs to combat infectious diseases.\n",
  "wordCount" : "2103",
  "inLanguage": "en",
  "datePublished": "2023-12-03T10:38:01Z",
  "dateModified": "2023-12-03T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 3, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569326">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569326" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569326">
        <p class="paperTitle">Extreme miniaturization in plasmid design: generation of the 903 bp cloning vector pICOt2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569326" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569326" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Staal, J.; Beyaert, R.</p>
        <p class="info">Score: 37.5, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569326' target='https://doi.org/10.1101/2023.11.29.569326'> 10.1101/2023.11.29.569326</a></p>
        <p class="abstract">Inspired by the size optimizations in the recently published pUdO series of plasmids, we set out to break our previous world record (pICOz, 1185 bp) in designing the smallest possible self-replicating high copy cloning vector. By replacing the zeocin resistance marker with trimethoprim resistance, deleting the parts upstream of -35 in the ampicilin resistance promoter and replacing the extended multiple cloning site with a single SpeI site, we managed to reduce the plasmid size to 903 bp, almost a 24% reduction in size compared to our previous world record.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.26.568713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.26.568713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.26.568713">
        <p class="paperTitle">An interdependent Metabolic and Genetic Network shows emergent properties in vitro</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.26.568713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.26.568713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Giaveri, S.; Bohra, N.; Diehl, C.; Ballinger, M.; Paczia, N.; Glatter, T.; Erb, T. J.</p>
        <p class="info">Score: 25.2, Published: 2023-11-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.26.568713' target='https://doi.org/10.1101/2023.11.26.568713'> 10.1101/2023.11.26.568713</a></p>
        <p class="abstract">A hallmark of all living organisms is their ability for self-regeneration which requires a tight integration of metabolic and genetic networks. Here we constructed a metabolic and genetic linked in vitro network (MGLN) that shows life-like behavior outside of a cellular context and generates its own building blocks from non-living matter. To this end, we integrated the metabolism of the crotonyl-CoA/ethyl-malonyl-CoA/hydroxybutyryl-CoA (CETCH) cycle with cell-free protein synthesis using recombinant elements (PURE). We demonstrate that the MGLN produces the essential amino acid glycine from inorganic carbon (CO2), and incorporates it into target proteins following DNA-encoded instructions. By programming genetically encoded response into metabolic networks our work opens new avenues for the development of advanced biomimetic systems with emergent properties, including decision-making, self-regeneration and evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.568972">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.568972" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.568972">
        <p class="paperTitle">A library of reporters of the global regulators of gene expression of Escherichia coli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.568972" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.568972" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dash, S.; Jagadeesan, R.; Baptista, I. S. C.; Chauhan, V.; Kandavalli, V.; Oliveira, S. M. D.; Ribeiro, A. S.</p>
        <p class="info">Score: 16.9, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.568972' target='https://doi.org/10.1101/2023.11.28.568972'> 10.1101/2023.11.28.568972</a></p>
        <p class="abstract">The topology of the transcription factor network (TFN) of E. coli is far from uniform, with 22 global regulator (GR) proteins controlling one-third of all genes. So far, their production rates cannot be tracked by comparable fluorescent proteins. We developed a library of fluorescent reporters for 16 GRs for this purpose. Each consists of a single-copy plasmid coding for GFP fused to the full-length copy of the native promoter. We tracked their activity in exponential and stationary growth, as well as under weak and strong stresses. We show that the reporters have high sensitivity and specificity to all stresses tested and detect single-cell variability in transcription rates. Given the influence of GRs on the TFN, we expect that the new library will contribute to dissecting global transcriptional stress-response programs of E. coli. Moreover, the library can be invaluable in bioindustrial applications that tune those programs to, instead of cell growth, favor productivity while reducing energy consumption.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=200 SRC=&#34;FIGDIR/small/568972v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (46K):
org.highwire.dtl.DTLVardef@f63b62org.highwire.dtl.DTLVardef@1b4d0a9org.highwire.dtl.DTLVardef@1ef2afforg.highwire.dtl.DTLVardef@effd94_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.03.543535">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.03.543535" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.03.543535">
        <p class="paperTitle">Autotrophic growth of E. coli is achieved by a small number of genetic changes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.03.543535" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.03.543535" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ben Nissan, R.; Milshtein, E.; Pahl, V.; de Pins, B.; Jona, G.; Levi, D.; Yung, H.; Nir, N.; Ezra, D.; Gleizer, S.; Link, H.; Noor, E.; Milo, R.</p>
        <p class="info">Score: 56.9, Published: 2023-11-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.03.543535' target='https://doi.org/10.1101/2023.06.03.543535'> 10.1101/2023.06.03.543535</a></p>
        <p class="abstract">Synthetic autotrophy is a promising avenue to sustainable bioproduction from CO2. Here, we use iterative laboratory evolution to generate several distinct autotrophic strains. Utilising this genetic diversity, we identify that just three mutations are sufficient for E. coli to grow autotrophically, when introduced alongside non-native energy (formate dehydrogenase) and carbon-fixing (RuBisCO, phosphoribulokinase, carbonic anhydrase) modules. The mutated genes are involved in glycolysis (pgi), central-carbon regulation (crp), and RNA transcription (rpoB). The pgi mutation reduces the enzymes activity, thereby stabilising the carbon-fixing cycle by capping a major branching flux. For the other two mutations, we observe down-regulation of several metabolic pathways and increased expression of native genes associated with the carbon-fixing module (rpiB) and the energy module (fdoGH), as well as an increased ratio of NADH/NAD&#43; - the cycles electron-donor. This study demonstrates the malleability of metabolism and its capacity to switch trophic modes using only a small number of genetic changes and could facilitate transforming other heterotrophic organisms into autotrophs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569005">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569005" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569005">
        <p class="paperTitle">High-yield enzymatic synthesis of mono- and trifluorinated alanine enantiomers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569005" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569005" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nieto-Dominguez, M.; Sako, A.; Enemark-Rasmussen, K.; Held Gotfredsen, C.; Rago, D.; Nikel, P. I.</p>
        <p class="info">Score: 7.4, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569005' target='https://doi.org/10.1101/2023.11.28.569005'> 10.1101/2023.11.28.569005</a></p>
        <p class="abstract">Fluorinated amino acids are a promising entry point for incorporating new-to-Nature chemistries in biological systems. Hence, novel methods are needed for the selective synthesis of these building blocks. In this study, we focused on the enzymatic synthesis of fluorinated alanine enantiomers. To this end, the alanine dehydrogenase from Vibrio proteolyticus and the diaminopimelate dehydrogenase from Symbiobacterium thermophilum were applied to the in vitro production of (R)-3-fluoroalanine and (S)-3-fluoroalanine, respectively, using 3-fluoropyruvate as the substrate. Additionally, an alanine racemase from Streptomyces lavendulae, originally selected for setting an alternative enzymatic cascade leading to the production of these non-canonical amino acids, had an unprecedented catalytic efficiency in the {beta}-elimination of fluorine from the monosubstituted fluoroalanine. The in vitro enzymatic cascade based on the dehydrogenases of V. proteolyticus and S. thermophilum included a cofactor recycling system, whereby a formate dehydrogenase from Pseudomonas sp. 101 (either native or engineered) coupled formate oxidation to NAD(P)H formation. Under these conditions, the reaction yields for (R)-3-fluoroalanine and (S)-3-fluoroalanine reached &gt;85% on the fluorinated substrate and proceeded with complete enantiomeric excess. Moreover, the selected dehydrogenases were also able to catalyze the conversion of trifluoropyruvate into trifluorinated alanine, as a first-case example of biocatalysis with amino acids carrying a trifluoromethyl group.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561409">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561409" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561409">
        <p class="paperTitle">Bacteriophage-host interactions in microgravity onboard the International Space Station</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561409" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561409" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huss, P.; Chitboonthavisuk, C.; Meger, A.; Nishikawa, K.; Oates, R. P.; Mills, H.; Holzhaus, O.; Raman, S.</p>
        <p class="info">Score: 21.4, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561409' target='https://doi.org/10.1101/2023.10.10.561409'> 10.1101/2023.10.10.561409</a></p>
        <p class="abstract">Bacteriophage-host interactions play a fundamental role in shaping microbial ecosystems. Although researchers have extensively studied phage-host interactions in terrestrial ecosystems, the impact of microgravity on phage-host interactions is yet to be investigated. Here, we report the dynamics of interactions between T7 bacteriophage and E. coli in microgravity onboard the International Space Station (ISS). We found phage activity was delayed but ultimately successful in microgravity. We identified several de novo mutations in phage and bacteria that improved fitness in microgravity. Deep mutational scanning of the receptor binding domain revealed substantial differences in the number, position, and mutational preferences between gravity and microgravity, reflecting underlying differences in the bacterial adaptations. Combinatorial libraries informed by microgravity selections gave T7 mutants with 100-10,000-fold higher activity on uropathogenic E. coli under terrestrial conditions than wildtype T7. Our findings lay the foundation for future research on the impact of microgravity on phage-host interactions and microbial communities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567117">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567117" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567117">
        <p class="paperTitle">Directed Evolution of Genetically Encoded LYTACs for Cell-Mediated Delivery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567117" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567117" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, J. L.; Yamada-Hunter, S. A.; Labanieh, L.; Sotillo, E.; Cheah, J. S.; Roberts, D. S.; Mackall, C.; Ting, A. Y.; Bertozzi, C.</p>
        <p class="info">Score: 25.7, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567117' target='https://doi.org/10.1101/2023.11.14.567117'> 10.1101/2023.11.14.567117</a></p>
        <p class="abstract">Lysosome-targeting chimeras (LYTACs) are a promising therapeutic modality to drive the degradation of extracellular proteins. However, early versions of LYTAC contain synthetic glycopeptides that cannot be genetically encoded. Here we present our designs for a fully genetically encodable LYTAC (GELYTAC), making our tool compatible with integration into therapeutic cells for targeted delivery at diseased sites. To achieve this, we replaced the glycopeptide portion of LYTACs with the protein insulin like growth factor 2 (IGF2). After showing initial efficacy with wild type IGF2, we increased the potency of GELYTAC using directed evolution. Subsequently, we demonstrated that our engineered GELYTAC construct not only secretes from HEK293T cells but also from human primary T-cells to drive the uptake of various targets into receiver cells. Immune cells engineered to secrete GELYTAC thus represent a promising avenue for spatially-selective targeted protein degradation.

Significance StatementBetter therapeutic windows can be achieved by targeting therapeutics to their desired sites of action. For protein therapeutics, this might be achieved by engineering cell therapies that home to a tissue of interest and secrete the biologic drug locally. Here, we demonstrate that human primary T cells can be engineered to produce genetically encoded lysosome targeting chimeras (GELYTACs). These GELYTACs mediate the degradation of extracellular proteins associated with cancer progression. Thus, cells engineered to produce GELYTACs represent a potential new class of cancer therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569037">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569037" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569037">
        <p class="paperTitle">Biofoundry-assisted Golden Gate cloning with AssemblyTron</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569037" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569037" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bryant, J. A.; Wright, R. C.</p>
        <p class="info">Score: 4.2, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569037' target='https://doi.org/10.1101/2023.11.28.569037'> 10.1101/2023.11.28.569037</a></p>
        <p class="abstract">Golden Gate assembly is a requisite method in synthetic biology that facilitates critical conventions such as genetic part abstraction and rapid prototyping. However, compared to robotic implementation, manual Golden Gate implementation is cumbersome, error-prone, and inconsistent for complex assembly designs. AssemblyTron is an open-source python package that provides an affordable automation solution using open-source Opentrons OT-2 lab robots. Automating Golden Gate assembly with AssemblyTron can reduce failure-rate, resource consumption, and training requirements for building complex DNA constructs, as well as indexed and combinatorial libraries. Here, we dissect a panel of upgrades to AssemblyTrons Golden Gate assembly capabilities, which include Golden Gate assembly into modular cloning part vectors, error-prone PCR combinatorial mutant library assembly, and modular cloning indexed plasmid library assembly. These upgrades enable a broad pool of users with varying levels of experience to readily implement advanced Golden Gate applications using low-cost, open-source lab robotics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569483">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569483" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569483">
        <p class="paperTitle">Profiling Expression Strategies for a Type III Polyketide Synthase in a Lysate-Based, Cell-free System</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569483" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569483" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sword, T. T.; Dinglasan, J. L. N.; Abbas, G. S. K.; Barker, J. W.; Spradley, M. E.; Greene, E. R.; Gooden, D. S.; Emrich, S. J.; Gilchrist, M. A.; Doktycz, M. J.; Bailey, C. B.</p>
        <p class="info">Score: 2.9, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569483' target='https://doi.org/10.1101/2023.11.30.569483'> 10.1101/2023.11.30.569483</a></p>
        <p class="abstract">Some of the most metabolically diverse species of bacteria (e.g., Actinobacteria) have higher GC content in their DNA, differ substantially in codon usage, and have distinct protein folding environments compared to tractable expression hosts like Escherichia coli. Consequentially, expressing biosynthetic gene clusters (BGCs) from these bacteria in E. coli frequently results in a myriad of unpredictable issues with protein expression and folding, delaying the biochemical characterization of new natural products. Current strategies to achieve soluble, active expression of these enzymes in tractable hosts, such as BGC refactoring, can be a lengthy trial-and-error process. Cell-free expression (CFE) has emerged as 1) a valuable expression platform for enzymes that are challenging to synthesize in vivo, and as 2) a testbed for rapid prototyping that can improve cellular expression. Here, we use a type III polyketide synthase from Streptomyces griseus, RppA, which catalyzes the formation of the red pigment flaviolin, as a reporter to investigate BGC refactoring techniques. We synergistically tune promoter and codon usage to improve flaviolin production from cell-free expressed RppA. We then assess the utility of cell-free systems for prototyping these refactoring tactics prior to their implementation in cells. Overall, codon harmonization improves natural product synthesis more than traditional codon optimization across cell-free and cellular environments. Refactoring promoters and/or coding sequences via CFE can be a valuable strategy to rapidly screen for catalytically functional production of enzymes from BCGs. By showing the coordinators between CFE versus in vivo expression, this work advances CFE as a tool for natural product research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569180">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569180" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569180">
        <p class="paperTitle">Glycodiversification of gentamicins through in vivo glycosyltransferase swapping enabled the creation of novel hybrid aminoglycoside antibiotics with potent activity and low ototoxicity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569180" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569180" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jian, X.; Wang, C.; Wu, S.; Sun, G.; Huang, C.; Qiu, C.; Liu, Y.; Leadlay, P. F.; Liu, D.; Deng, Z.; Zhou, F.; Sun, Y.</p>
        <p class="info">Score: 2.5, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569180' target='https://doi.org/10.1101/2023.11.29.569180'> 10.1101/2023.11.29.569180</a></p>
        <p class="abstract">Aminoglycosides (AGs) are a class of potent antibiotics with a broad spectrum of activity. However, their use is limited by safety concerns associated with nephrotoxicity and ototoxicity, as well as drug resistance. To address these issues, semi-synthetic approaches for modifying natural AGs have successfully generated new generations of AGs, however, with limited types of modification due to significant challenges in synthesis. This study explores a novel approach that harness the bacterial biosynthetic machinery of gentamicins and kanamycins to create hybrid AGs, installing extensive natural modifications from gentamicins onto kanamycins. This was achieved by glycodiversification of gentamicins via swapping the glycosyltransferase (GT) in their producer with the GT from kanamycins biosynthetic pathway and resulted in the creation of a series of novel AGs with combined structural features of two, therefore referred to as genkamicins (GKs). The manipulation of the hybrid metabolic pathway enabled the target accumulation of different GK species and the successful isolation and characterization of six GK components. These compounds display retained antimicrobial activity against a panel of World Health Organization (WHO) critical priority pathogens, and GK-C2a, in particular, demonstrates low ototoxicity compared to clinical drugs in zebrafish embryos. This study provides a new strategy for diversifying the structure of AGs and a potential avenue for developing less toxic AG drugs to combat infectious diseases.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
