<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Modular RNA motifs for orthogonal phase separated compartments
Authors: Stewart, J. M.; Li, S.; Tang, A.; Klocke, M. A. K. A.; Gobry, M. V.; Fabrini, G.; Di Michele, L.; Rothemund, P. W. K.; Franco, E.
Score: 36.9, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561123
Recent discoveries in biology have highlighted the importance of protein and RNA-based condensates as an alternative to classical membrane-bound organelles for the task of compartmentalizing molecules and biochemical reactions.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Modular RNA motifs for orthogonal phase separated compartments
Authors: Stewart, J. M.; Li, S.; Tang, A.; Klocke, M. A. K. A.; Gobry, M. V.; Fabrini, G.; Di Michele, L.; Rothemund, P. W. K.; Franco, E.
Score: 36.9, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561123
Recent discoveries in biology have highlighted the importance of protein and RNA-based condensates as an alternative to classical membrane-bound organelles for the task of compartmentalizing molecules and biochemical reactions." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-11T10:39:46+00:00" />
<meta property="article:modified_time" content="2023-10-11T10:39:46+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Modular RNA motifs for orthogonal phase separated compartments
Authors: Stewart, J. M.; Li, S.; Tang, A.; Klocke, M. A. K. A.; Gobry, M. V.; Fabrini, G.; Di Michele, L.; Rothemund, P. W. K.; Franco, E.
Score: 36.9, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561123
Recent discoveries in biology have highlighted the importance of protein and RNA-based condensates as an alternative to classical membrane-bound organelles for the task of compartmentalizing molecules and biochemical reactions."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Modular RNA motifs for orthogonal phase separated compartments\nAuthors: Stewart, J. M.; Li, S.; Tang, A.; Klocke, M. A. K. A.; Gobry, M. V.; Fabrini, G.; Di Michele, L.; Rothemund, P. W. K.; Franco, E.\nScore: 36.9, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561123\nRecent discoveries in biology have highlighted the importance of protein and RNA-based condensates as an alternative to classical membrane-bound organelles for the task of compartmentalizing molecules and biochemical reactions.",
  "keywords": [
    
  ],
  "articleBody": " Modular RNA motifs for orthogonal phase separated compartments\nAuthors: Stewart, J. M.; Li, S.; Tang, A.; Klocke, M. A. K. A.; Gobry, M. V.; Fabrini, G.; Di Michele, L.; Rothemund, P. W. K.; Franco, E.\nScore: 36.9, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561123\nRecent discoveries in biology have highlighted the importance of protein and RNA-based condensates as an alternative to classical membrane-bound organelles for the task of compartmentalizing molecules and biochemical reactions. Here, we demonstrate the rational design of pure RNA condensates from star-shaped RNA motifs. We generate condensates using two different RNA nanostar architectures: multi-stranded nanostars whose binding interactions are programmed via single-stranded overhangs, and single-stranded nanostars whose interactions are programmed via kissing loops. Through rational design of the nanostar interaction sequences, we demonstrate that both architectures can produce orthogonal (distinct and immiscible) condensates, which can be individually tracked via fluorogenic aptamers. We also show that aptamers make it possible to recruit peptides and proteins to the condensates with high specificity. Successful cotranscriptional formation of condensates from single-stranded nanostars suggests that they may be genetically encoded and produced in living cells. We provide a library of orthogonal RNA condensates that can be modularly customized and offer a route toward creating systems of functional artificial organelles.\nA Modular Design for Synthetic Membraneless Organelles Enables Compositional and Functional Control\nAuthors: Walls, M. T.; Xu, K.; Brangwynne, C. P.; Avalos, J. L.\nScore: 22.7, Published: 2023-10-04 DOI: 10.1101/2023.10.03.560789\nLiving cells organize a wide variety of processes through compartmentalization into membraneless organelles, known as biomolecular condensates. Given their ubiquitous presence across a wide spectrum of different organisms and cell types, biomolecular condensates are increasingly considered to offer great potential for biotechnological applications. However, native condensates remain difficult to harness for engineering applications, both due to their intertwined mechanisms of assembly and compositional control, and potential disruptions to native cellular processes. Here, we demonstrate a modular framework for the formation of synthetic condensates designed to decouple cluster formation and protein recruitment. Synthetic condensates are built through constitutive oligomerization of intrinsically-disordered regions (IDRs), which drive the formation of condensates whose composition can be independently defined through fused interaction domains. The composition of the proteins driven to partition into the condensate can be quantitatively described using a binding equilibrium model, demonstrating predictive control of how component expression levels and interaction affinity determine the degree of protein recruitment. Finally, the engineered system is utilized to regulate protein interactions and metabolic flux by harnessing the systems compositional tunability.\nBackbone extension acyl rearrangements enable cellular synthesis of proteins with internal β2-peptide linkages\nAuthors: Roe, L.; Schissel, C. K.; Dover, T. L.; Shah, B.; Hamlish, N. X.; Zheng, S.; Dilworth, D. A.; Wong, N.; Zhang, Z.; Chatterjee, A.; Francis, M. B.; Miller, S. J.; Schepartz, A.\nScore: 15.7, Published: 2023-10-04 DOI: 10.1101/2023.10.03.560714\nProteins and polypeptides containing extended backbone monomers embody highly desirable structures and functions, but they cannot yet be biosynthesized in cells. There are two challenges at work. First is the ribosome, whose ability to promote rapid bond-forming reactions to and from anything other than an -amino acid or -hydroxy acid is unknown. The second challenge is the absence of orthogonal enzymes that acylate tRNA with extended backbone monomers. Here we describe a general approach to the programmed cellular synthesis of proteins containing extended backbone monomers that circumvents both of these challenges. Rather than relying on direct and uncharacterized reactions of non--amino acid monomers within the ribosomal PTC, we develop a proximity-guided intramolecular rearrangement that effectively edits the protein backbone post-translationally. The method relies on the ability of PylRS-like aminoacyl-tRNA synthetase enzymes to accept diverse -hydroxy acid monomers, including those whose side chains contain masked nucleophiles. Introduction of such an -hydroxy acid monomer into a protein translated in vivo, followed by nucleophile unmasking, sets up a thermodynamically favored and quantitative intramolecular Backbone Extension Acyl Rearrangement (BEAR) reaction that edits the protein backbone to install an extended backbone monomer. In the examples described here, the intramolecular rearrangement converts an -peptide backbone directly into a {beta}-backbone. As far as we know, this report represents the first example in which a much-desired expanded backbone {beta}-amino acid linkage has been introduced site selectively into a protein in a cell.\nCompact Programmable Control of Protein Secretion in Mammalian Cells\nAuthors: Vlahos, A. E.; Call, C. C.; Kadaba, S. E.; Guo, S.; Gao, X. J.\nScore: 13.9, Published: 2023-10-04 DOI: 10.1101/2023.10.04.560774\nSynthetic biology currently holds immense potential to engineer the spatiotemporal control of intercellular signals for biomedicine. Programming behaviors using protein-based circuits has advantages over traditional gene circuits such as compact delivery and direct interactions with signaling proteins. Previously, we described a generalizable platform called RELEASE to enable the control of intercellular signaling through the proteolytic removal of ER-retention motifs compatible with pre-existing protease-based circuits. However, these tools lacked the ability to reliably program complex expression profiles and required numerous proteases, limiting delivery options. Here, we harness the recruitment and antagonistic behavior of endogenous 14-3-3 proteins to create RELEASE-NOT to turn off protein secretion in response to protease activity. By combining RELEASE and RELEASE-NOT, we establish a suite of protein-level processing and output modules called Compact RELEASE (compRELEASE). This innovation enables functions such as logic processing and analog signal filtering using a single input protease. Furthermore, we demonstrate the compactness of the post-translational design by using polycistronic single transcripts to engineer cells to control protein secretion via lentiviral integration and leverage mRNA delivery to selectively express cell surface proteins only in engineered cells harboring inducible proteases. CompRELEASE enables complex control of protein secretion and enhances the potential of synthetic protein circuits for therapeutic applications, while minimizing the overall genetic payload.\nCo-transcriptional production of programmable RNA condensates and synthetic organelles\nAuthors: Fabrini, G.; Nuccio, S. P.; Stewart, J. M.; Lin, S.; Tang, A.; Rothemund, P. W. K.; Franco, E.; Di Antonio, M.; Di Michele, L.\nScore: 13.3, Published: 2023-10-08 DOI: 10.1101/2023.10.06.561174\nCondensation of RNA and proteins is central to cellular functions, and the ability to program it would be valuable in synthetic biology and synthetic cell science. Here we introduce a modular platform for engineering synthetic RNA condensates from tailor-made, branched RNA nanostructures that fold and assemble co-transcriptionally. Up to three orthogonal condensates can form simultaneously and selectively accumulate guest molecules. The RNA condensates can be expressed within synthetic cells to produce membrane-less organelles with controlled number, size, morphology and compositions, and that display the ability to selectively capture proteins. The in situ expression of programmable RNA condensates could underpin spatial organisation of functionalities in both biological and synthetic cells.\nA green alternative to fragrant agarwood sesquiterpenoid production\nAuthors: Gutierrez, S.; Overmans, S.; Wellman, G. B.; Samaras, V. G.; Oviedo, C.; Gede, M.; Szekely, G.; Lauersen, K. J.\nScore: 9.9, Published: 2023-10-07 DOI: 10.1101/2023.10.06.561217\nCertain endangered Thymelaeaceous trees are major sources of the fragrant and highly valued resinous agarwood, comprised of hundreds of oxygenated sesquiterpenoids (STPs). Despite growing pressure on natural agarwood sources, the chemical complexity of STPs severely limits synthetic production. Here, we catalogued the chemical diversity in 58 agarwood samples by two--dimensional gas chromatography--mass spectrometry and partially recreated complex STP mixtures through synthetic biology. We improved STP yields in the unicellular alga Chlamydomonas reinhardtii by combinatorial engineering to biosynthesise nine macrocyclic STP backbones found in agarwood. A bioprocess following green-chemistry principles was developed that exploits {per thousand}milking{per thousand} of STPs without cell lysis, solvent--solvent STP extraction, solvent--STP nanofiltration, and bulk STP oxy--functionalisation to obtain terpene mixtures like those of agarwood. This process occurs with total solvent recycling and enables continuous production. Our synthetic-biology approach offers a sustainable alternative to harvesting agarwood trees to obtain mixtures of complex, fragrant, oxygenated STPs.\nAn antimicrobial peptide expression platform for targeting pathogenic bacterial species\nAuthors: Rutter, J. W.; Dekker, L.; Clare, C.; Fedorec, A. J.; Barnes, C. P.\nScore: 7.5, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561505\nBacteriocins are antimicrobial peptides that are naturally produced by many bacteria. They hold great potential in the fight against antibiotic resistant bacteria, including ESKAPE pathogens. However, they often have low stability in vivo and therefore, may not be effective when administered orally. Engineered live biotherapeutic products (eLBPs) that secrete bacteriocins can be created to deliver bacteriocins directly at the site of infection. Here we develop a modular bacteriocin secretion platform that can be used to express and secrete multiple bacteriocins from non-pathogenic Escherichia coli host strains. As a proof of concept we create Enterocin A and Enterocin B secreting strains that show strong antimicrobial activity against Enterococcus faecalis and Enterococcus faecium, and characterise this activity in both solid culture and liquid co-culture. We then develop a Lokta-Volterra model that can be used to capture the interactions of these competitor strains. We show that simultaneous exposure to EntA and EntB can increase the level of Enterococcus killing and delay the development of resistance. Our system has the potential to be used as an eLBP to secrete additional bacteriocins for the targeted killing of other pathogenic bacteria.\nOptoProfilin: A Single Component Biosensor of Applied Cellular Stress\nAuthors: Mann, N.; Hill, J. T.; Wang, K.; Hughes, R. M.\nScore: 5.2, Published: 2023-10-04 DOI: 10.1101/2023.10.04.560945\nThe actin cytoskeleton is a biosensor of cellular stress and a potential prognosticator of human disease. In particular, aberrant cytoskeletal structures such as cofilin-actin rods and stress granules formed in response to energetic and oxidative stress are closely linked to neurodegenerative diseases such as Alzheimers, Parkinsons, and ALS. Whether these cytoskeletal phenomena can be harnessed for the development of biosensors for cytoskeletal dysfunction and, by extension, neurodegenerative disease progression, remains an open question. In this work, we describe the design and development of an optogenetic iteration of profilin, an actin monomer binding protein with critical functions in cytoskeletal dynamics. We demonstrate that this optically activated profilin ( OptoProfilin) can act as an optically triggered biosensor of applied cellular stress in select immortalized cell lines. Notably, OptoProfilin is a single component biosensor, likely increasing its utility for experimentalists. While a large body of work closely links profilin activity with cellular stress and neurodegenerative disease, this, to our knowledge, is the first example of profilin as an optogenetic biosensor of stress-induced changes in the cytoskeleton.\nEngineered bacteria launch and control an oncolytic virus\nAuthors: Singer, Z. S.; Pabon, J.; Huang, H.; Rice, C. M.; Danino, T.\nScore: 19.2, Published: 2023-09-28 DOI: 10.1101/2023.09.28.559873\nThe ability of bacteria and viruses to selectively replicate in tumors has led to synthetic engineering of new microbial therapies. Here we design a cooperative strategy whereby S. typhimurium bacteria transcribe and deliver the Senecavirus A RNA genome inside host cells, launching a potent oncolytic viral infection. Then, we engineer the virus to require a bacterially delivered protease in order to achieve virion maturation, demonstrating bacterial control over the virus. This work extends bacterially delivered therapeutics to viral genomes, and the governing of a viral population through engineered microbial interactions. One-Sentence SummaryBacteria are engineered to act as a synthetic \"capsid\" delivering Senecavirus A genome and controlling its spread.\nA robust life-or-death selection platform for enzyme evolution\nAuthors: Jansen, S. C.; Mayer, C.\nScore: 4.3, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561342\nLife-or-death selections evaluate the fitness of individual organisms on a population level. In enzyme engineering, such growth selections allow the rapid and straightforward identification of highly efficient biocatalysts from extensive libraries. However, selection-based improvement of (industrially-relevant) biocatalysts is challenging, as they require highly dependable strategies that artificially link their activities to host survival. Here, we showcase a robust and scalable life-or-death selection platform centered around the complementation of non-canonical amino acid-dependent bacteria. Specifically, we demonstrate how serial passaging of populations featuring millions of carbamoylase variants autonomously selects biocatalysts with up to 90,000-fold higher initial rates. Notably, selection of replicate populations enriched diverse biocatalysts, which feature distinct amino-acid motifs that drastically boost carbamoylase activity. As beneficial substitutions also originated from unintended copying errors during library preparation or cell division, we anticipate that our life-or-death selection platform will be applicable to the continuous, autonomous evolution of diverse biocatalysts in the future.\n",
  "wordCount" : "1982",
  "inLanguage": "en",
  "datePublished": "2023-10-11T10:39:46Z",
  "dateModified": "2023-10-11T10:39:46Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on October 11, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561123">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561123" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561123">
        <p class="paperTitle">Modular RNA motifs for orthogonal phase separated compartments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561123" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561123" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stewart, J. M.; Li, S.; Tang, A.; Klocke, M. A. K. A.; Gobry, M. V.; Fabrini, G.; Di Michele, L.; Rothemund, P. W. K.; Franco, E.</p>
        <p class="info">Score: 36.9, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561123' target='https://doi.org/10.1101/2023.10.06.561123'> 10.1101/2023.10.06.561123</a></p>
        <p class="abstract">Recent discoveries in biology have highlighted the importance of protein and RNA-based condensates as an alternative to classical membrane-bound organelles for the task of compartmentalizing molecules and biochemical reactions. Here, we demonstrate the rational design of pure RNA condensates from star-shaped RNA motifs. We generate condensates using two different RNA nanostar architectures: multi-stranded nanostars whose binding interactions are programmed via single-stranded overhangs, and single-stranded nanostars whose interactions are programmed via kissing loops. Through rational design of the nanostar interaction sequences, we demonstrate that both architectures can produce orthogonal (distinct and immiscible) condensates, which can be individually tracked via fluorogenic aptamers. We also show that aptamers make it possible to recruit peptides and proteins to the condensates with high specificity. Successful cotranscriptional formation of condensates from single-stranded nanostars suggests that they may be genetically encoded and produced in living cells. We provide a library of orthogonal RNA condensates that can be modularly customized and offer a route toward creating systems of functional artificial organelles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560789">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560789" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560789">
        <p class="paperTitle">A Modular Design for Synthetic Membraneless Organelles Enables Compositional and Functional Control</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560789" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560789" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Walls, M. T.; Xu, K.; Brangwynne, C. P.; Avalos, J. L.</p>
        <p class="info">Score: 22.7, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560789' target='https://doi.org/10.1101/2023.10.03.560789'> 10.1101/2023.10.03.560789</a></p>
        <p class="abstract">Living cells organize a wide variety of processes through compartmentalization into membraneless organelles, known as biomolecular condensates. Given their ubiquitous presence across a wide spectrum of different organisms and cell types, biomolecular condensates are increasingly considered to offer great potential for biotechnological applications. However, native condensates remain difficult to harness for engineering applications, both due to their intertwined mechanisms of assembly and compositional control, and potential disruptions to native cellular processes. Here, we demonstrate a modular framework for the formation of synthetic condensates designed to decouple cluster formation and protein recruitment. Synthetic condensates are built through constitutive oligomerization of intrinsically-disordered regions (IDRs), which drive the formation of condensates whose composition can be independently defined through fused interaction domains. The composition of the proteins driven to partition into the condensate can be quantitatively described using a binding equilibrium model, demonstrating predictive control of how component expression levels and interaction affinity determine the degree of protein recruitment. Finally, the engineered system is utilized to regulate protein interactions and metabolic flux by harnessing the systems compositional tunability.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560714">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560714" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560714">
        <p class="paperTitle">Backbone extension acyl rearrangements enable cellular synthesis of proteins with internal β2-peptide linkages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560714" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560714" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roe, L.; Schissel, C. K.; Dover, T. L.; Shah, B.; Hamlish, N. X.; Zheng, S.; Dilworth, D. A.; Wong, N.; Zhang, Z.; Chatterjee, A.; Francis, M. B.; Miller, S. J.; Schepartz, A.</p>
        <p class="info">Score: 15.7, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560714' target='https://doi.org/10.1101/2023.10.03.560714'> 10.1101/2023.10.03.560714</a></p>
        <p class="abstract">Proteins and polypeptides containing extended backbone monomers embody highly desirable structures and functions, but they cannot yet be biosynthesized in cells. There are two challenges at work. First is the ribosome, whose ability to promote rapid bond-forming reactions to and from anything other than an -amino acid or -hydroxy acid is unknown. The second challenge is the absence of orthogonal enzymes that acylate tRNA with extended backbone monomers. Here we describe a general approach to the programmed cellular synthesis of proteins containing extended backbone monomers that circumvents both of these challenges. Rather than relying on direct and uncharacterized reactions of non--amino acid monomers within the ribosomal PTC, we develop a proximity-guided intramolecular rearrangement that effectively edits the protein backbone post-translationally. The method relies on the ability of PylRS-like aminoacyl-tRNA synthetase enzymes to accept diverse -hydroxy acid monomers, including those whose side chains contain masked nucleophiles. Introduction of such an -hydroxy acid monomer into a protein translated in vivo, followed by nucleophile unmasking, sets up a thermodynamically favored and quantitative intramolecular Backbone Extension Acyl Rearrangement (BEAR) reaction that edits the protein backbone to install an extended backbone monomer. In the examples described here, the intramolecular rearrangement converts an -peptide backbone directly into a {beta}-backbone. As far as we know, this report represents the first example in which a much-desired expanded backbone {beta}-amino acid linkage has been introduced site selectively into a protein in a cell.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560774">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560774" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560774">
        <p class="paperTitle">Compact Programmable Control of Protein Secretion in Mammalian Cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560774" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560774" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vlahos, A. E.; Call, C. C.; Kadaba, S. E.; Guo, S.; Gao, X. J.</p>
        <p class="info">Score: 13.9, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560774' target='https://doi.org/10.1101/2023.10.04.560774'> 10.1101/2023.10.04.560774</a></p>
        <p class="abstract">Synthetic biology currently holds immense potential to engineer the spatiotemporal control of intercellular signals for biomedicine. Programming behaviors using protein-based circuits has advantages over traditional gene circuits such as compact delivery and direct interactions with signaling proteins. Previously, we described a generalizable platform called RELEASE to enable the control of intercellular signaling through the proteolytic removal of ER-retention motifs compatible with pre-existing protease-based circuits. However, these tools lacked the ability to reliably program complex expression profiles and required numerous proteases, limiting delivery options. Here, we harness the recruitment and antagonistic behavior of endogenous 14-3-3 proteins to create RELEASE-NOT to turn off protein secretion in response to protease activity. By combining RELEASE and RELEASE-NOT, we establish a suite of protein-level processing and output modules called Compact RELEASE (compRELEASE). This innovation enables functions such as logic processing and analog signal filtering using a single input protease. Furthermore, we demonstrate the compactness of the post-translational design by using polycistronic single transcripts to engineer cells to control protein secretion via lentiviral integration and leverage mRNA delivery to selectively express cell surface proteins only in engineered cells harboring inducible proteases. CompRELEASE enables complex control of protein secretion and enhances the potential of synthetic protein circuits for therapeutic applications, while minimizing the overall genetic payload.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561174">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561174" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561174">
        <p class="paperTitle">Co-transcriptional production of programmable RNA condensates and synthetic organelles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561174" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561174" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fabrini, G.; Nuccio, S. P.; Stewart, J. M.; Lin, S.; Tang, A.; Rothemund, P. W. K.; Franco, E.; Di Antonio, M.; Di Michele, L.</p>
        <p class="info">Score: 13.3, Published: 2023-10-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561174' target='https://doi.org/10.1101/2023.10.06.561174'> 10.1101/2023.10.06.561174</a></p>
        <p class="abstract">Condensation of RNA and proteins is central to cellular functions, and the ability to program it would be valuable in synthetic biology and synthetic cell science. Here we introduce a modular platform for engineering synthetic RNA condensates from tailor-made, branched RNA nanostructures that fold and assemble co-transcriptionally. Up to three orthogonal condensates can form simultaneously and selectively accumulate guest molecules. The RNA condensates can be expressed within synthetic cells to produce membrane-less organelles with controlled number, size, morphology and compositions, and that display the ability to selectively capture proteins. The in situ expression of programmable RNA condensates could underpin spatial organisation of functionalities in both biological and synthetic cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561217">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561217" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561217">
        <p class="paperTitle">A green alternative to fragrant agarwood sesquiterpenoid production</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561217" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561217" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gutierrez, S.; Overmans, S.; Wellman, G. B.; Samaras, V. G.; Oviedo, C.; Gede, M.; Szekely, G.; Lauersen, K. J.</p>
        <p class="info">Score: 9.9, Published: 2023-10-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561217' target='https://doi.org/10.1101/2023.10.06.561217'> 10.1101/2023.10.06.561217</a></p>
        <p class="abstract">Certain endangered Thymelaeaceous trees are major sources of the fragrant and highly valued resinous agarwood, comprised of hundreds of oxygenated sesquiterpenoids (STPs). Despite growing pressure on natural agarwood sources, the chemical complexity of STPs severely limits synthetic production. Here, we catalogued the chemical diversity in 58 agarwood samples by two--dimensional gas chromatography--mass spectrometry and partially recreated complex STP mixtures through synthetic biology. We improved STP yields in the unicellular alga Chlamydomonas reinhardtii by combinatorial engineering to biosynthesise nine macrocyclic STP backbones found in agarwood. A bioprocess following green-chemistry principles was developed that exploits {per thousand}milking{per thousand} of STPs without cell lysis, solvent--solvent STP extraction, solvent--STP nanofiltration, and bulk STP oxy--functionalisation to obtain terpene mixtures like those of agarwood. This process occurs with total solvent recycling and enables continuous production. Our synthetic-biology approach offers a sustainable alternative to harvesting agarwood trees to obtain mixtures of complex, fragrant, oxygenated STPs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561505">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561505" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561505">
        <p class="paperTitle">An antimicrobial peptide expression platform for targeting pathogenic bacterial species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561505" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561505" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rutter, J. W.; Dekker, L.; Clare, C.; Fedorec, A. J.; Barnes, C. P.</p>
        <p class="info">Score: 7.5, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561505' target='https://doi.org/10.1101/2023.10.09.561505'> 10.1101/2023.10.09.561505</a></p>
        <p class="abstract">Bacteriocins are antimicrobial peptides that are naturally produced by many bacteria. They hold great potential in the fight against antibiotic resistant bacteria, including ESKAPE pathogens. However, they often have low stability in vivo and therefore, may not be effective when administered orally. Engineered live biotherapeutic products (eLBPs) that secrete bacteriocins can be created to deliver bacteriocins directly at the site of infection. Here we develop a modular bacteriocin secretion platform that can be used to express and secrete multiple bacteriocins from non-pathogenic Escherichia coli host strains. As a proof of concept we create Enterocin A and Enterocin B secreting strains that show strong antimicrobial activity against Enterococcus faecalis and Enterococcus faecium, and characterise this activity in both solid culture and liquid co-culture. We then develop a Lokta-Volterra model that can be used to capture the interactions of these competitor strains. We show that simultaneous exposure to EntA and EntB can increase the level of Enterococcus killing and delay the development of resistance. Our system has the potential to be used as an eLBP to secrete additional bacteriocins for the targeted killing of other pathogenic bacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560945">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560945" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560945">
        <p class="paperTitle">OptoProfilin: A Single Component Biosensor of Applied Cellular Stress</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560945" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560945" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mann, N.; Hill, J. T.; Wang, K.; Hughes, R. M.</p>
        <p class="info">Score: 5.2, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560945' target='https://doi.org/10.1101/2023.10.04.560945'> 10.1101/2023.10.04.560945</a></p>
        <p class="abstract">The actin cytoskeleton is a biosensor of cellular stress and a potential prognosticator of human disease. In particular, aberrant cytoskeletal structures such as cofilin-actin rods and stress granules formed in response to energetic and oxidative stress are closely linked to neurodegenerative diseases such as Alzheimers, Parkinsons, and ALS. Whether these cytoskeletal phenomena can be harnessed for the development of biosensors for cytoskeletal dysfunction and, by extension, neurodegenerative disease progression, remains an open question. In this work, we describe the design and development of an optogenetic iteration of profilin, an actin monomer binding protein with critical functions in cytoskeletal dynamics. We demonstrate that this optically activated profilin ( OptoProfilin) can act as an optically triggered biosensor of applied cellular stress in select immortalized cell lines. Notably, OptoProfilin is a single component biosensor, likely increasing its utility for experimentalists. While a large body of work closely links profilin activity with cellular stress and neurodegenerative disease, this, to our knowledge, is the first example of profilin as an optogenetic biosensor of stress-induced changes in the cytoskeleton.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.559873">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.559873" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.559873">
        <p class="paperTitle">Engineered bacteria launch and control an oncolytic virus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.559873" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.559873" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Singer, Z. S.; Pabon, J.; Huang, H.; Rice, C. M.; Danino, T.</p>
        <p class="info">Score: 19.2, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.559873' target='https://doi.org/10.1101/2023.09.28.559873'> 10.1101/2023.09.28.559873</a></p>
        <p class="abstract">The ability of bacteria and viruses to selectively replicate in tumors has led to synthetic engineering of new microbial therapies. Here we design a cooperative strategy whereby S. typhimurium bacteria transcribe and deliver the Senecavirus A RNA genome inside host cells, launching a potent oncolytic viral infection. Then, we engineer the virus to require a bacterially delivered protease in order to achieve virion maturation, demonstrating bacterial control over the virus. This work extends bacterially delivered therapeutics to viral genomes, and the governing of a viral population through engineered microbial interactions.

One-Sentence SummaryBacteria are engineered to act as a synthetic &#34;capsid&#34; delivering Senecavirus A genome and controlling its spread.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561342">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561342" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561342">
        <p class="paperTitle">A robust life-or-death selection platform for enzyme evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561342" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561342" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jansen, S. C.; Mayer, C.</p>
        <p class="info">Score: 4.3, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561342' target='https://doi.org/10.1101/2023.10.09.561342'> 10.1101/2023.10.09.561342</a></p>
        <p class="abstract">Life-or-death selections evaluate the fitness of individual organisms on a population level. In enzyme engineering, such growth selections allow the rapid and straightforward identification of highly efficient biocatalysts from extensive libraries. However, selection-based improvement of (industrially-relevant) biocatalysts is challenging, as they require highly dependable strategies that artificially link their activities to host survival. Here, we showcase a robust and scalable life-or-death selection platform centered around the complementation of non-canonical amino acid-dependent bacteria. Specifically, we demonstrate how serial passaging of populations featuring millions of carbamoylase variants autonomously selects biocatalysts with up to 90,000-fold higher initial rates. Notably, selection of replicate populations enriched diverse biocatalysts, which feature distinct amino-acid motifs that drastically boost carbamoylase activity. As beneficial substitutions also originated from unintended copying errors during library preparation or cell division, we anticipate that our life-or-death selection platform will be applicable to the continuous, autonomous evolution of diverse biocatalysts in the future.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
