<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Breakthrough in Dicot Prime Editing: Enabling Heritable Desired Edits in Tomato and Arabidopsis
Authors: Vu, T. V.; Nguyen, N. T.; Kim, J.; Song, Y. J.; Nguyen, T. H.; Kim, J.-Y.
Score: 12.4, Published: 2024-02-11 DOI: 10.1101/2024.02.11.579803
Prime editing (PE) enables almost all types of precise genome editing in animals and plants. It has been successfully adapted to edit several plants at variable efficiency and versatility. However, this technique is inefficient for dicots for unknown reasons.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Breakthrough in Dicot Prime Editing: Enabling Heritable Desired Edits in Tomato and Arabidopsis
Authors: Vu, T. V.; Nguyen, N. T.; Kim, J.; Song, Y. J.; Nguyen, T. H.; Kim, J.-Y.
Score: 12.4, Published: 2024-02-11 DOI: 10.1101/2024.02.11.579803
Prime editing (PE) enables almost all types of precise genome editing in animals and plants. It has been successfully adapted to edit several plants at variable efficiency and versatility. However, this technique is inefficient for dicots for unknown reasons." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-18T10:38:00+00:00" />
<meta property="article:modified_time" content="2024-02-18T10:38:00+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Breakthrough in Dicot Prime Editing: Enabling Heritable Desired Edits in Tomato and Arabidopsis
Authors: Vu, T. V.; Nguyen, N. T.; Kim, J.; Song, Y. J.; Nguyen, T. H.; Kim, J.-Y.
Score: 12.4, Published: 2024-02-11 DOI: 10.1101/2024.02.11.579803
Prime editing (PE) enables almost all types of precise genome editing in animals and plants. It has been successfully adapted to edit several plants at variable efficiency and versatility. However, this technique is inefficient for dicots for unknown reasons."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Breakthrough in Dicot Prime Editing: Enabling Heritable Desired Edits in Tomato and Arabidopsis\nAuthors: Vu, T. V.; Nguyen, N. T.; Kim, J.; Song, Y. J.; Nguyen, T. H.; Kim, J.-Y.\nScore: 12.4, Published: 2024-02-11 DOI: 10.1101/2024.02.11.579803\nPrime editing (PE) enables almost all types of precise genome editing in animals and plants. It has been successfully adapted to edit several plants at variable efficiency and versatility. However, this technique is inefficient for dicots for unknown reasons.",
  "keywords": [
    
  ],
  "articleBody": " Breakthrough in Dicot Prime Editing: Enabling Heritable Desired Edits in Tomato and Arabidopsis\nAuthors: Vu, T. V.; Nguyen, N. T.; Kim, J.; Song, Y. J.; Nguyen, T. H.; Kim, J.-Y.\nScore: 12.4, Published: 2024-02-11 DOI: 10.1101/2024.02.11.579803\nPrime editing (PE) enables almost all types of precise genome editing in animals and plants. It has been successfully adapted to edit several plants at variable efficiency and versatility. However, this technique is inefficient for dicots for unknown reasons. Here, by employing novel combinations of PE components, including an RNA chaperone and modified epegRNAs driven by a PolII-PolIII composite promoter and a viral replicon system, we obtained up to 9.7% of the desired PE efficiency at the callus stage assessed by targeted deep sequencing. Subsequently, we identified that up to 38.2% of transformants contained desired PE alleles in tomatoes and Arabidopsis, marking the first successful heritable PE transmission in dicots. Our PE tools also showed high accuracy, specificity, and multiplexing capability, which unlocked the potential for practical PE applications in dicots, paving the way for transformative advancements in plant sciences.\nUltra-high efficiency T cell reprogramming at multiple loci with SEED-Selection\nAuthors: Chang, C. R.; Vykunta, V. S.; Goodman, D. B.; Muldoon, J. J.; Nyberg, W. A.; Liu, C.; Allain, V.; Rothrock, A.; Wang, C. H.; Marson, A.; Shy, B. R.; Eyquem, J.\nScore: 14.5, Published: 2024-02-07 DOI: 10.1101/2024.02.06.576175\nMultiplexed reprogramming of T cell specificity and function can generate powerful next-generation cellular therapies. However, current manufacturing methods produce heterogenous mixtures of partially engineered cells. Here, we develop a one-step process to enrich for unlabeled cells with knock-ins at multiple target loci using a family of repair templates named Synthetic Exon/Expression Disruptors (SEEDs). SEED engineering associates transgene integration with the disruption of a paired endogenous surface protein, allowing non-modified and partially edited cells to be immunomagnetically depleted (SEED-Selection). We design SEEDs to fully reprogram three critical loci encoding T cell specificity, co-receptor expression, and MHC expression, with up to 98% purity after selection for individual modifications and up to 90% purity for six simultaneous edits (three knock-ins and three knockouts). These methods are simple, compatible with existing clinical manufacturing workflows, and can be readily adapted to other loci to facilitate production of complex gene-edited cell therapies.\nSelf-assembled cell-scale containers made from DNA origami membranes\nAuthors: Karfusehr, C.; Eder, M.; Simmel, F. C.\nScore: 5.7, Published: 2024-02-09 DOI: 10.1101/2024.02.09.579479\nDNA origami provides a methodology for the sequence-programmable generation of precisely defined molecular nanostructures with sizes of order 100 nm. A new frontier for the field is the generation of superstructures made from DNA origami subunits, which requires other self-assembly strategies than those used for DNA origami itself. Challenges faced by current approaches include the increasing complexity, cost and development time for the structures and off-target assembly. Here, we demonstrate how radially symmetric origami subunits that are inspired by the structure and interactions of lipids organize into giant DNA origami monolayer membranes that can be readily programmed to form vesicles or hollow tubes with diameters ranging from 100 nm to over 1 {micro}m. DNA origami membranes are an unprecedented approach for compartmentalization that opens up new possibilities for bottom-up biology and cell-scale soft robotics.\nA spring-loaded and leakage-tolerant synthetic gene switch for in-vitro detection of DNA and RNA\nAuthors: Gupta, K.; Krieg, E.\nScore: 4.8, Published: 2024-02-14 DOI: 10.1101/2024.02.12.579921\nNucleic acid tests (NATs) are essential for biomedical diagnostics. Traditional NATs, often complex and expensive, have prompted the exploration of Toehold-Mediated Strand Displacement (TMSD) circuits as an economical alternative. However, the wide application of TMSD-based reactions is limited by leakage--the spurious activation of the reaction leading to high background signals and false positives. Here we introduce a new TMSD cascade that recognizes a custom nucleic acid input and generates an amplified output. The system is based on a pair of thermodynamically spring-loaded DNA modules. The binding of a predefined nucleic acid target triggers an intermolecular reaction that activates a T7 promoter, leading to the perpetual transcription of a fluorescent aptamer that can be detected by a smartphone camera. The system is designed to permit the selective depletion of leakage byproducts to achieve high sensitivity and zero-background signal in the absence of the correct trigger. Using Zika virus (ZIKV)- and severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)-derived nucleic acid sequences, we show that the assay generates a reliable target-specific readout. Native RNA can be directly detected under isothermal conditions, without requiring reverse transcription, with a sensitivity as low as 200 attomole. The modularity of the assay allows easy re-programming for the detection of other targets by exchanging a single sequence domain. This work provides a low-complexity and high-fidelity synthetic biology tool for point-of-care diagnostics and for the construction of more complex biomolecular computations.\nPangenomic landscapes shape the genetic circuit performance in a Stutzerimonas biodesign toolkit\nAuthors: Chan, D. T. C.; Bernstein, H. C.\nScore: 2.5, Published: 2024-02-15 DOI: 10.1101/2024.02.15.580380\nImplementation of identical biodesign strategies into different species often results in different performance, a process called the chassis effect. Our current understanding of how cellular host context underpins its ability to be engineered is lacking and closing this knowledge gap will greatly improve the rational design of microorganisms. Here, we combined global differential gene expression analysis and pangenomics to uncover how genome structure and function relates to the observed chassis effect of an engineered genetic inverter device operating in six closely related Stutzerimonas hosts. The differential expression of the core genome, gene clusters shared between all hosts, was found to be the main source of significant concordance to the observed device performance, whereas specialty genes from respective accessory genomes were not significant. A data-driven investigation revealed that genes related to denitrification and efflux pumps were among the most differentially expressed gene clusters in response to the engineered device. This study establishes that the effectiveness of synthetic gene circuits can be traced along differences in closely related microbial hosts that each represent unique hardware options for biodesign.\nDegron-based bioPROTACs for controlling signaling in CAR T cells\nAuthors: Kim, M. S.; Bhargava, H. K.; Shavey, G. E.; Lim, W. A.; El-Samad, H.; Ng, A. H.\nScore: 1.8, Published: 2024-02-17 DOI: 10.1101/2024.02.16.580396\nChimeric antigen receptor (CAR) T cells have made a tremendous impact in the clinic, but potent signaling through the CAR can be detrimental to treatment safety and efficacy. The use of protein degradation to control CAR signaling can address these issues in pre-clinical models. Existing strategies for regulating CAR stability rely on small molecules to induce systemic degradation. In contrast to small molecule regulation, genetic circuits offer a more precise method to control CAR signaling in an autonomous, cell-by-cell fashion. Here, we describe a programmable protein degradation tool that adopts the framework of bioPROTACs, heterobifunctional proteins that contain a target recognition domain fused to a domain that recruits the endogenous ubiquitin proteasome system. We develop novel bioPROTACs that utilize a compact four residue degron and demonstrate degradation of cytosolic and membrane protein targets using either a nanobody or synthetic leucine zipper as a protein binder. Our bioPROTACs exhibit potent degradation of CARs and can inhibit CAR signaling in primary human T cells. We demonstrate the utility of our bioPROTACs by constructing a genetic circuit to degrade the tyrosine kinase ZAP70 in response to recognition of a specific membrane-bound antigen. This circuit is able to disrupt CAR T cell signaling only in the presence of a specific cell population. These results suggest that bioPROTACs are a powerful tool for expanding the cell engineering toolbox for CAR T cells.\nMechanism Study on the Effect of Kidney Tonifying and Liver Soothing Methods on Mouse Oocyte Quality by Regulating OSFs and Smads Pathways\nAuthors: Zhang, R.; Jia, R.; Bai, J.; Du, H.; Yang, L.\nScore: 1.0, Published: 2024-02-14 DOI: 10.1101/2024.02.12.579988\nObjective: To observe the effects of kidney tonifying and liver soothing methods on the secretion of factor BMP-6, related receptors ALK-2/6, and downstream Smads pathway Smad1/5/844 in oocytes, and to explore the targeted mechanisms of their effects on mouse oocyte quality; Method: Healthy female mice aged 6-7 weeks were randomly divided into 6 groups, namely high and low dose kidney tonifying groups; High - and low-dose liver soothing groups; Control group and normal group. The high-dose and low-dose groups of tonifying the kidney were given oral solution of tonifying the kidney and regulating the meridian at 5.4g/ml and 2.7g/ml, respectively. The high-dose and low-dose groups of soothing the liver were given suspension of Xiaoyao Pill at 0.6g/ml and 0.3g/ml, respectively. The control group and normal group were given distilled water by gavage. Immunohistochemistry and Western blot were used to detect the expression of BMP-6, ALK-2/6, and Smad1/5/8/4 proteins in oocytes, while PCR was used to detect the mRNA expression of these indicators in oocytes; Result: Both methods can increase the expression of BMP-6 in oocytes of mice in the treatment group, activate ALK-2/6, and phosphorylate Smad1/5/8 to bind with Smad4, initiating signal transduction; The high-dose group of kidney tonifying is superior to other groups in regulating BMP-6, ALK-2/6, and Smad1/5/4. Conclusion: Kidney tonifying and liver soothing methods can regulate BMP-6 and its Smads pathway through different mechanisms to improve mouse oocyte quality.\nInfection-on-Chip: an in vitro human vessel to study Neisseria meningitidis colonization and vascular damages\nAuthors: Pinon, L.; Chabaud, M.; Nivoit, P.; Wong-Ng, J.; Nguyen, T.-T.; Paul, V.; Goussard, S.; Frachon, E.; Obino, D.; Gobaa, S.; Dumenil, G.\nScore: 1.0, Published: 2024-02-10 DOI: 10.1101/2024.02.09.579276\nBloodstream infections leading to sepsis are a life-threatening condition and remain difficult to treat, however, in vitro experimental models that reflect their key features are still lacking. We here developed a photoablation-based 3-dimensional, microfluidic model of meningococcal vascular colonization, which allows to study cardinal features of the bacteria-blood vessel interaction within controllable vascular geometries. Meningococci are Gram-negative human-specific bacteria responsible for meningitis and a severe form of sepsis that is associated with vascular damages, referred to as purpura fulminans. The infection-on-chip device is used to quantitatively assess bacterial adhesion and proliferation at high spatio-temporal resolution in a physiologically relevant microenvironment. In addition, we here show that vascular colonization by meningococci in our Infection-on-Chip device recapitulates key features of disease progression, including vascular leakage and the recruitment of neutrophils to sites of infections, mirroring results obtained using our previously described human skin xenograft mouse model. As a result, our Infection-on-chip platform provides a robust alternative approach to the use of animal and 2D cellular models, opening the path to the better understanding of disease progression and testing innovative therapeutics in an in vitro but physiologically relevant environment.\nEnhancing Plant Photosynthesis using Carbon Dots as Light Converter and Photosensitizer\nAuthors: Hu, H.; Cheng, W.; Wang, X.; Yang, Y.; Yu, X.; Ding, J.; Lin, Y.; Zhao, W.; Zhao, Q.; Ledesma-Amaro, R.; Chen, X.; Liu, J.; Yang, C.; Gao, X.\nScore: 2.0, Published: 2024-02-07 DOI: 10.1101/2024.02.06.579025\nImproving photosynthetic efficiency is pivotal for CO2-based biomanufacturing and agriculture purposes. Despite the progress on photosynthetic biohybrids integrating biocatalysts with synthetic materials, nanomaterials with improved optical and photoelectrochemical properties are still needed to increase the energy-conversion efficiency. Here, we present a novel approach using carbon dots (CDs) as both intracellular photosensitizers and light converters for enhancing solar energy utilization in photosynthetic organisms. The CDs were produced from cyanobacterial biomass and used to convert a broad spectrum of solar irradiation to red light. We demonstrated that the nanosized CDs were incorporated into cyanobacterial cells and transferred light-excited electrons into the photosynthetic electron transfer chain. The biohybrids consisting of the CDs and Synechococcus elongatus exhibited increased growth rates, enhanced activities of both photosystems, and accelerated linear electron transport, compared with the cyanobacterial cells only. The supplementation of the CDs increased CO2-fixation rate and CO2-to-glycerol production by 2.4-fold and 2.2-fold, respectively. Furthermore, the CDs were shown to enhance photosynthesis and promote growth of Arabidopsis thaliana. The fresh weight of plant was increased 1.8-fold by CDs addition. These results reveal that simultaneous photosensitization and spectral modification could substantially improve the efficiency of natural photosynthesis. This study presents CDs as an attractive nanomaterial with great application potential in agriculture and solar-powered biomanufacturing.\nDNA representations and generalization performance of sequence-to-expression models\nAuthors: Shen, Y.; Kudla, G.; Oyarzun, D. A.\nScore: 10.0, Published: 2024-02-07 DOI: 10.1101/2024.02.06.579067\nThe increasing demand for biological products drives many efforts to engineer cells that produce heterologous proteins at maximal yield. Recent advances in massively parallel reporter assays can deliver data suitable for training machine learning models and sup-port the design of microbial strains with optimized protein expression phenotypes. The best performing sequence- to-expression models have been trained on one-hot encodings, a mechanism-agnostic representation of nucleotide sequences. Despite their excellent local pre-dictive power, however, such models suffer from a limited ability to generalize predictions far away from the training data. Here, we show that libraries of genetic constructs can have substantially different cluster structure depending on the chosen sequence representation, and demonstrate that such differences can be leveraged to improve generalization perfor-mance. Using a large sequence- to-expression dataset from Escherichia coli, we show that non-deep regressors and convolutional neural networks trained on one-hot encodings fail to generalize predictions, and that learned representations using state-of-the-art large language models also struggle with out-of-domain accuracy. In contrast, we show that despite their poorer local performance, mechanistic sequence features such as codon bias, nucleotide con-tent or mRNA stability, provide promising gains on model generalization. We explore several strategies to integrate different feature sets into a single predictive model, including feature stacking, ensemble model stacking, and geometric stacking, a novel architecture based on graph convolutional neural networks. Our work suggests that integration of domain-agnostic and domain-aware sequence features offers an unexplored route for improving the quality of sequence- to-expression models and facilitate their adoption in the biotechnology and phar-maceutical sectors.\n",
  "wordCount" : "2273",
  "inLanguage": "en",
  "datePublished": "2024-02-18T10:38:00Z",
  "dateModified": "2024-02-18T10:38:00Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 18, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.11.579803">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.11.579803" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.11.579803">
        <p class="paperTitle">Breakthrough in Dicot Prime Editing: Enabling Heritable Desired Edits in Tomato and Arabidopsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.11.579803" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.11.579803" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vu, T. V.; Nguyen, N. T.; Kim, J.; Song, Y. J.; Nguyen, T. H.; Kim, J.-Y.</p>
        <p class="info">Score: 12.4, Published: 2024-02-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.11.579803' target='https://doi.org/10.1101/2024.02.11.579803'> 10.1101/2024.02.11.579803</a></p>
        <p class="abstract">Prime editing (PE) enables almost all types of precise genome editing in animals and plants. It has been successfully adapted to edit several plants at variable efficiency and versatility. However, this technique is inefficient for dicots for unknown reasons. Here, by employing novel combinations of PE components, including an RNA chaperone and modified epegRNAs driven by a PolII-PolIII composite promoter and a viral replicon system, we obtained up to 9.7% of the desired PE efficiency at the callus stage assessed by targeted deep sequencing. Subsequently, we identified that up to 38.2% of transformants contained desired PE alleles in tomatoes and Arabidopsis, marking the first successful heritable PE transmission in dicots. Our PE tools also showed high accuracy, specificity, and multiplexing capability, which unlocked the potential for practical PE applications in dicots, paving the way for transformative advancements in plant sciences.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.576175">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.576175" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.576175">
        <p class="paperTitle">Ultra-high efficiency T cell reprogramming at multiple loci with SEED-Selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.576175" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.576175" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chang, C. R.; Vykunta, V. S.; Goodman, D. B.; Muldoon, J. J.; Nyberg, W. A.; Liu, C.; Allain, V.; Rothrock, A.; Wang, C. H.; Marson, A.; Shy, B. R.; Eyquem, J.</p>
        <p class="info">Score: 14.5, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.576175' target='https://doi.org/10.1101/2024.02.06.576175'> 10.1101/2024.02.06.576175</a></p>
        <p class="abstract">Multiplexed reprogramming of T cell specificity and function can generate powerful next-generation cellular therapies. However, current manufacturing methods produce heterogenous mixtures of partially engineered cells. Here, we develop a one-step process to enrich for unlabeled cells with knock-ins at multiple target loci using a family of repair templates named Synthetic Exon/Expression Disruptors (SEEDs). SEED engineering associates transgene integration with the disruption of a paired endogenous surface protein, allowing non-modified and partially edited cells to be immunomagnetically depleted (SEED-Selection). We design SEEDs to fully reprogram three critical loci encoding T cell specificity, co-receptor expression, and MHC expression, with up to 98% purity after selection for individual modifications and up to 90% purity for six simultaneous edits (three knock-ins and three knockouts). These methods are simple, compatible with existing clinical manufacturing workflows, and can be readily adapted to other loci to facilitate production of complex gene-edited cell therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579479">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579479" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579479">
        <p class="paperTitle">Self-assembled cell-scale containers made from DNA origami membranes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579479" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579479" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Karfusehr, C.; Eder, M.; Simmel, F. C.</p>
        <p class="info">Score: 5.7, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579479' target='https://doi.org/10.1101/2024.02.09.579479'> 10.1101/2024.02.09.579479</a></p>
        <p class="abstract">DNA origami provides a methodology for the sequence-programmable generation of precisely defined molecular nanostructures with sizes of order 100 nm. A new frontier for the field is the generation of superstructures made from DNA origami subunits, which requires other self-assembly strategies than those used for DNA origami itself. Challenges faced by current approaches include the increasing complexity, cost and development time for the structures and off-target assembly. Here, we demonstrate how radially symmetric origami subunits that are inspired by the structure and interactions of lipids organize into giant DNA origami monolayer membranes that can be readily programmed to form vesicles or hollow tubes with diameters ranging from 100 nm to over 1 {micro}m. DNA origami membranes are an unprecedented approach for compartmentalization that opens up new possibilities for bottom-up biology and cell-scale soft robotics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.12.579921">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.12.579921" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.12.579921">
        <p class="paperTitle">A spring-loaded and leakage-tolerant synthetic gene switch for in-vitro detection of DNA and RNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.12.579921" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.12.579921" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gupta, K.; Krieg, E.</p>
        <p class="info">Score: 4.8, Published: 2024-02-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.12.579921' target='https://doi.org/10.1101/2024.02.12.579921'> 10.1101/2024.02.12.579921</a></p>
        <p class="abstract">Nucleic acid tests (NATs) are essential for biomedical diagnostics. Traditional NATs, often complex and expensive, have prompted the exploration of Toehold-Mediated Strand Displacement (TMSD) circuits as an economical alternative. However, the wide application of TMSD-based reactions is limited by  leakage--the spurious activation of the reaction leading to high background signals and false positives. Here we introduce a new TMSD cascade that recognizes a custom nucleic acid input and generates an amplified output. The system is based on a pair of thermodynamically spring-loaded DNA modules. The binding of a predefined nucleic acid target triggers an intermolecular reaction that activates a T7 promoter, leading to the perpetual transcription of a fluorescent aptamer that can be detected by a smartphone camera. The system is designed to permit the selective depletion of leakage byproducts to achieve high sensitivity and zero-background signal in the absence of the correct trigger. Using Zika virus (ZIKV)- and severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)-derived nucleic acid sequences, we show that the assay generates a reliable target-specific readout. Native RNA can be directly detected under isothermal conditions, without requiring reverse transcription, with a sensitivity as low as 200 attomole. The modularity of the assay allows easy re-programming for the detection of other targets by exchanging a single sequence domain. This work provides a low-complexity and high-fidelity synthetic biology tool for point-of-care diagnostics and for the construction of more complex biomolecular computations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.15.580380">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.15.580380" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.15.580380">
        <p class="paperTitle">Pangenomic landscapes shape the genetic circuit performance in a Stutzerimonas biodesign toolkit</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.15.580380" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.15.580380" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chan, D. T. C.; Bernstein, H. C.</p>
        <p class="info">Score: 2.5, Published: 2024-02-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.15.580380' target='https://doi.org/10.1101/2024.02.15.580380'> 10.1101/2024.02.15.580380</a></p>
        <p class="abstract">Implementation of identical biodesign strategies into different species often results in different performance, a process called the chassis effect. Our current understanding of how cellular host context underpins its ability to be engineered is lacking and closing this knowledge gap will greatly improve the rational design of microorganisms. Here, we combined global differential gene expression analysis and pangenomics to uncover how genome structure and function relates to the observed chassis effect of an engineered genetic inverter device operating in six closely related Stutzerimonas hosts. The differential expression of the core genome, gene clusters shared between all hosts, was found to be the main source of significant concordance to the observed device performance, whereas specialty genes from respective accessory genomes were not significant. A data-driven investigation revealed that genes related to denitrification and efflux pumps were among the most differentially expressed gene clusters in response to the engineered device. This study establishes that the effectiveness of synthetic gene circuits can be traced along differences in closely related microbial hosts that each represent unique hardware options for biodesign.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.16.580396">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.16.580396" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.16.580396">
        <p class="paperTitle">Degron-based bioPROTACs for controlling signaling in CAR T cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.16.580396" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.16.580396" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, M. S.; Bhargava, H. K.; Shavey, G. E.; Lim, W. A.; El-Samad, H.; Ng, A. H.</p>
        <p class="info">Score: 1.8, Published: 2024-02-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.16.580396' target='https://doi.org/10.1101/2024.02.16.580396'> 10.1101/2024.02.16.580396</a></p>
        <p class="abstract">Chimeric antigen receptor (CAR) T cells have made a tremendous impact in the clinic, but potent signaling through the CAR can be detrimental to treatment safety and efficacy. The use of protein degradation to control CAR signaling can address these issues in pre-clinical models. Existing strategies for regulating CAR stability rely on small molecules to induce systemic degradation. In contrast to small molecule regulation, genetic circuits offer a more precise method to control CAR signaling in an autonomous, cell-by-cell fashion. Here, we describe a programmable protein degradation tool that adopts the framework of bioPROTACs, heterobifunctional proteins that contain a target recognition domain fused to a domain that recruits the endogenous ubiquitin proteasome system. We develop novel bioPROTACs that utilize a compact four residue degron and demonstrate degradation of cytosolic and membrane protein targets using either a nanobody or synthetic leucine zipper as a protein binder. Our bioPROTACs exhibit potent degradation of CARs and can inhibit CAR signaling in primary human T cells. We demonstrate the utility of our bioPROTACs by constructing a genetic circuit to degrade the tyrosine kinase ZAP70 in response to recognition of a specific membrane-bound antigen. This circuit is able to disrupt CAR T cell signaling only in the presence of a specific cell population. These results suggest that bioPROTACs are a powerful tool for expanding the cell engineering toolbox for CAR T cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.12.579988">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.12.579988" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.12.579988">
        <p class="paperTitle">Mechanism Study on the Effect of Kidney Tonifying and Liver Soothing Methods on Mouse Oocyte Quality by Regulating OSFs and Smads Pathways</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.12.579988" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.12.579988" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, R.; Jia, R.; Bai, J.; Du, H.; Yang, L.</p>
        <p class="info">Score: 1.0, Published: 2024-02-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.12.579988' target='https://doi.org/10.1101/2024.02.12.579988'> 10.1101/2024.02.12.579988</a></p>
        <p class="abstract">Objective: To observe the effects of kidney tonifying and liver soothing methods on the secretion of factor BMP-6, related receptors ALK-2/6, and downstream Smads pathway Smad1/5/844 in oocytes, and to explore the targeted mechanisms of their effects on mouse oocyte quality; Method: Healthy female mice aged 6-7 weeks were randomly divided into 6 groups, namely high and low dose kidney tonifying groups; High - and low-dose liver soothing groups; Control group and normal group. The high-dose and low-dose groups of tonifying the kidney were given oral solution of tonifying the kidney and regulating the meridian at 5.4g/ml and 2.7g/ml, respectively. The high-dose and low-dose groups of soothing the liver were given suspension of Xiaoyao Pill at 0.6g/ml and 0.3g/ml, respectively. The control group and normal group were given distilled water by gavage. Immunohistochemistry and Western blot were used to detect the expression of BMP-6, ALK-2/6, and Smad1/5/8/4 proteins in oocytes, while PCR was used to detect the mRNA expression of these indicators in oocytes; Result: Both methods can increase the expression of BMP-6 in oocytes of mice in the treatment group, activate ALK-2/6, and phosphorylate Smad1/5/8 to bind with Smad4, initiating signal transduction; The high-dose group of kidney tonifying is superior to other groups in regulating BMP-6, ALK-2/6, and Smad1/5/4. Conclusion: Kidney tonifying and liver soothing methods can regulate BMP-6 and its Smads pathway through different mechanisms to improve mouse oocyte quality.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579276">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579276" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579276">
        <p class="paperTitle">Infection-on-Chip: an in vitro human vessel to study Neisseria meningitidis colonization and vascular damages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579276" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579276" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pinon, L.; Chabaud, M.; Nivoit, P.; Wong-Ng, J.; Nguyen, T.-T.; Paul, V.; Goussard, S.; Frachon, E.; Obino, D.; Gobaa, S.; Dumenil, G.</p>
        <p class="info">Score: 1.0, Published: 2024-02-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579276' target='https://doi.org/10.1101/2024.02.09.579276'> 10.1101/2024.02.09.579276</a></p>
        <p class="abstract">Bloodstream infections leading to sepsis are a life-threatening condition and remain difficult to treat, however, in vitro experimental models that reflect their key features are still lacking. We here developed a photoablation-based 3-dimensional, microfluidic model of meningococcal vascular colonization, which allows to study cardinal features of the bacteria-blood vessel interaction within controllable vascular geometries. Meningococci are Gram-negative human-specific bacteria responsible for meningitis and a severe form of sepsis that is associated with vascular damages, referred to as purpura fulminans. The infection-on-chip device is used to quantitatively assess bacterial adhesion and proliferation at high spatio-temporal resolution in a physiologically relevant microenvironment. In addition, we here show that vascular colonization by meningococci in our Infection-on-Chip device recapitulates key features of disease progression, including vascular leakage and the recruitment of neutrophils to sites of infections, mirroring results obtained using our previously described human skin xenograft mouse model. As a result, our Infection-on-chip platform provides a robust alternative approach to the use of animal and 2D cellular models, opening the path to the better understanding of disease progression and testing innovative therapeutics in an in vitro but physiologically relevant environment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579025">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579025" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579025">
        <p class="paperTitle">Enhancing Plant Photosynthesis using Carbon Dots as Light Converter and Photosensitizer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579025" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579025" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, H.; Cheng, W.; Wang, X.; Yang, Y.; Yu, X.; Ding, J.; Lin, Y.; Zhao, W.; Zhao, Q.; Ledesma-Amaro, R.; Chen, X.; Liu, J.; Yang, C.; Gao, X.</p>
        <p class="info">Score: 2.0, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579025' target='https://doi.org/10.1101/2024.02.06.579025'> 10.1101/2024.02.06.579025</a></p>
        <p class="abstract">Improving photosynthetic efficiency is pivotal for CO2-based biomanufacturing and agriculture purposes. Despite the progress on photosynthetic biohybrids integrating biocatalysts with synthetic materials, nanomaterials with improved optical and photoelectrochemical properties are still needed to increase the energy-conversion efficiency. Here, we present a novel approach using carbon dots (CDs) as both intracellular photosensitizers and light converters for enhancing solar energy utilization in photosynthetic organisms. The CDs were produced from cyanobacterial biomass and used to convert a broad spectrum of solar irradiation to red light. We demonstrated that the nanosized CDs were incorporated into cyanobacterial cells and transferred light-excited electrons into the photosynthetic electron transfer chain. The biohybrids consisting of the CDs and Synechococcus elongatus exhibited increased growth rates, enhanced activities of both photosystems, and accelerated linear electron transport, compared with the cyanobacterial cells only. The supplementation of the CDs increased CO2-fixation rate and CO2-to-glycerol production by 2.4-fold and 2.2-fold, respectively. Furthermore, the CDs were shown to enhance photosynthesis and promote growth of Arabidopsis thaliana. The fresh weight of plant was increased 1.8-fold by CDs addition. These results reveal that simultaneous photosensitization and spectral modification could substantially improve the efficiency of natural photosynthesis. This study presents CDs as an attractive nanomaterial with great application potential in agriculture and solar-powered biomanufacturing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579067">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579067" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579067">
        <p class="paperTitle">DNA representations and generalization performance of sequence-to-expression models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579067" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579067" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shen, Y.; Kudla, G.; Oyarzun, D. A.</p>
        <p class="info">Score: 10.0, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579067' target='https://doi.org/10.1101/2024.02.06.579067'> 10.1101/2024.02.06.579067</a></p>
        <p class="abstract">The increasing demand for biological products drives many efforts to engineer cells that produce heterologous proteins at maximal yield. Recent advances in massively parallel reporter assays can deliver data suitable for training machine learning models and sup-port the design of microbial strains with optimized protein expression phenotypes. The best performing sequence- to-expression models have been trained on one-hot encodings, a mechanism-agnostic representation of nucleotide sequences. Despite their excellent local pre-dictive power, however, such models suffer from a limited ability to generalize predictions far away from the training data. Here, we show that libraries of genetic constructs can have substantially different cluster structure depending on the chosen sequence representation, and demonstrate that such differences can be leveraged to improve generalization perfor-mance. Using a large sequence- to-expression dataset from Escherichia coli, we show that non-deep regressors and convolutional neural networks trained on one-hot encodings fail to generalize predictions, and that learned representations using state-of-the-art large language models also struggle with out-of-domain accuracy. In contrast, we show that despite their poorer local performance, mechanistic sequence features such as codon bias, nucleotide con-tent or mRNA stability, provide promising gains on model generalization. We explore several strategies to integrate different feature sets into a single predictive model, including feature stacking, ensemble model stacking, and geometric stacking, a novel architecture based on graph convolutional neural networks. Our work suggests that integration of domain-agnostic and domain-aware sequence features offers an unexplored route for improving the quality of sequence- to-expression models and facilitate their adoption in the biotechnology and phar-maceutical sectors.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
