<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Emergent digital bio-computation through spatial diffusion and engineered bacteria
Authors: Fedorec, A. J. H.; Treloar, N. J.; Wen, K. Y.; Dekker, L.; Ong, Q. H.; Jurkeviciute, G.; Lyu, E.; Rutter, J.; Rosa, L.; Zaikin, A.; Barnes, C. P.
Score: 24.2, Published: 2023-07-11 DOI: 10.1101/2023.07.07.548081
Building computationally capable biological systems has long been an aim of synthetic biology. The potential utility of biocomputing devices ranges from biosafety and environmental applications to diagnosis and personalised medicine.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Emergent digital bio-computation through spatial diffusion and engineered bacteria
Authors: Fedorec, A. J. H.; Treloar, N. J.; Wen, K. Y.; Dekker, L.; Ong, Q. H.; Jurkeviciute, G.; Lyu, E.; Rutter, J.; Rosa, L.; Zaikin, A.; Barnes, C. P.
Score: 24.2, Published: 2023-07-11 DOI: 10.1101/2023.07.07.548081
Building computationally capable biological systems has long been an aim of synthetic biology. The potential utility of biocomputing devices ranges from biosafety and environmental applications to diagnosis and personalised medicine." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-16T10:39:40+00:00" />
<meta property="article:modified_time" content="2023-07-16T10:39:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Emergent digital bio-computation through spatial diffusion and engineered bacteria
Authors: Fedorec, A. J. H.; Treloar, N. J.; Wen, K. Y.; Dekker, L.; Ong, Q. H.; Jurkeviciute, G.; Lyu, E.; Rutter, J.; Rosa, L.; Zaikin, A.; Barnes, C. P.
Score: 24.2, Published: 2023-07-11 DOI: 10.1101/2023.07.07.548081
Building computationally capable biological systems has long been an aim of synthetic biology. The potential utility of biocomputing devices ranges from biosafety and environmental applications to diagnosis and personalised medicine."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Emergent digital bio-computation through spatial diffusion and engineered bacteria\nAuthors: Fedorec, A. J. H.; Treloar, N. J.; Wen, K. Y.; Dekker, L.; Ong, Q. H.; Jurkeviciute, G.; Lyu, E.; Rutter, J.; Rosa, L.; Zaikin, A.; Barnes, C. P.\nScore: 24.2, Published: 2023-07-11 DOI: 10.1101/2023.07.07.548081\nBuilding computationally capable biological systems has long been an aim of synthetic biology. The potential utility of biocomputing devices ranges from biosafety and environmental applications to diagnosis and personalised medicine.",
  "keywords": [
    
  ],
  "articleBody": " Emergent digital bio-computation through spatial diffusion and engineered bacteria\nAuthors: Fedorec, A. J. H.; Treloar, N. J.; Wen, K. Y.; Dekker, L.; Ong, Q. H.; Jurkeviciute, G.; Lyu, E.; Rutter, J.; Rosa, L.; Zaikin, A.; Barnes, C. P.\nScore: 24.2, Published: 2023-07-11 DOI: 10.1101/2023.07.07.548081\nBuilding computationally capable biological systems has long been an aim of synthetic biology. The potential utility of biocomputing devices ranges from biosafety and environmental applications to diagnosis and personalised medicine. Here we present work for the design of bacterial computers which use spatial patterning to process information. Our computers are composed of bacterial colonies which, inspired by patterning in embryo development, receive information in the form of diffusible morphogen-like signals. A computation is encoded by the physical locations of the input sources of morphogen and the output receiver colonies. We demonstrate, mathematically and with engineered Escherichia coli, the simple digital logic capability of single bacterial colonies and show how additional colonies are required to build complex functions. Inspired by electronic design automation, an algorithm for designing optimal spatial circuits computing two-level digital functions is presented. This enhances the capability of our system to complex digital functions without increasing the biological complexity. We extend our experimental system to incorporate biosensing colonies as morphogen sources, demonstrating how a diagnostic device might be constructed. Our approach will open up new ways to perform biological computation, with applications in bioengineering, biomaterials and biosensing. Ultimately, these computational bacterial communities will help us explore information processing in natural biological systems.\nxTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein\nAuthors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Gong, J.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.\nScore: 84.2, Published: 2023-07-14 DOI: 10.1101/2023.07.05.547496\nProtein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently. This paper proposes a unified protein language model, xTrimoPGLM, to address these two types of tasks simultaneously through an innovative pre-training framework. Our key technical contribution is an exploration of the compatibility and the potential for joint optimization of the two types of objectives, which has led to a strategy for training xTrimoPGLM at an unprecedented scale of 100 billion parameters and 1 trillion training tokens. Our extensive experiments reveal that xTrimoPGLM significantly outperforms other advanced baselines in diverse protein understanding tasks (13 out of 15 tasks across four categories) and generates novel protein sequences which are structurally similar to natural ones. Furthermore, using the same xTrimoPGLM framework, we train an antibody-specific model (xTrimoPGLM-Ab) using 1 billion parameters. This model set a new record in predicting antibody naturalness and structures, both essential to the field of antibody-based drug design, and demonstrated a significantly faster inference speed than AlphaFold2. These results highlight the substantial capability and versatility of xTrimoPGLM in understanding and generating protein sequences.\nHighly efficient cellular expression of circular mRNA enables prolonged protein expression\nAuthors: Unti, M. J.; Jaffrey, S. R.\nScore: 11.6, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548538\nA major problem with mRNA therapeutics is the limited duration of protein expression due to the short half-life of mRNA. New approaches for generating highly stable circular mRNA in vitro have allowed increased duration of protein expression. However, it remains difficult to genetically encode circular mRNAs in mammalian cells, which limits the use of circular mRNA in cell-derived therapeutics. Here we describe the adaptation of the Tornado (Twister-optimized RNA for durable overexpression) system to achieve in-cell synthesis of circular mRNAs. We identify the promoter and internal ribosomal entry site (IRES) that result in high levels of protein expression in cells. We then show that these circular mRNAs can be packaged into virus-like particles (VLPs) thus enabling prolonged protein expression. Overall, these data describe a platform for synthesis of circular mRNAs and how these circular mRNAs can markedly enhance the ability of VLPs to function as a mRNA delivery tool.\nGenerative and predictive neural networks for the design of functional RNA molecules\nAuthors: Riley, A. T.; Robson, J. M.; Green, A. A.\nScore: 8.6, Published: 2023-07-14 DOI: 10.1101/2023.07.14.549043\nRNA is a remarkably versatile molecule that has been engineered for applications in therapeutics, diagnostics, and in vivo information-processing systems. However, the complex relationship between the sequence and structural properties of an RNA molecule and its ability to perform specific functions often necessitates extensive experimental screening of candidate sequences. Here we present a generalized neural network architecture that utilizes the sequence and structure of RNA molecules (SANDSTORM) to inform functional predictions. We demonstrate that this approach achieves state-of-the-art performance across several distinct RNA prediction tasks, while learning interpretable abstractions of RNA secondary structure. We paired these predictive models with generative adversarial RNA design networks (GARDN), allowing the generative modelling of novel mRNA 5-prime untranslated regions and toehold switch riboregulators exhibiting a predetermined fitness. This approach enabled the design of novel toehold switches with a 43-fold increase in experimentally characterized dynamic range compared to those designed using classic thermodynamic algorithms. SANDSTORM and GARDN thus represent powerful new predictive and generative tools for the development of diagnostic and therapeutic RNA molecules with improved function.\nAn expanded genetic toolbox to accelerate the creation of Acholeplasma laidlawii driven by synthetic genomes\nAuthors: Nucifora, D. P.; Mehta, N. D.; Giguere, D. J.; Karas, B. J.\nScore: 29.4, Published: 2023-07-12 DOI: 10.1101/2022.09.21.508766\nAssembling synthetic bacterial genomes in yeast and genome transplantation has enabled an unmatched level of bacterial strain engineering, giving rise to cells with minimal and chemically synthetic genomes. However, this technology is currently limited to members of the Spiroplasma phylogenetic group, mostly Mycoplasmas, within the Mollicute class. Here, we propose new genetic tools for developing these technologies for Acholeplasma laidlawii, which is phylogenetically distant from Mycoplasmas and, unlike most Mollicutes, uses a standard genetic code. We first investigated a donor-recipient relationship between two A. laidlawii strains through whole-genome sequencing. We then created multi-host shuttle plasmids and used them to optimize an electroporation protocol. We also demonstrated the use of evolution to create superior strains for DNA uptake via electroporation. For genome transplantation, we selected A. laidlawii 8195 as the recipient strain and created a PG-8A donor strain by inserting a Tn5 transposon carrying a tetracycline resistance gene. The tools presented here will improve Acholeplasma research and accelerate the effort toward creating A. laidlawii strains driven by synthetic genomes.\nA Platform for Controlled Template-Independent Enzymatic Synthesis of RNA Oligonucleotides and Therapeutics\nAuthors: Wiegand, D. J.; Rittichier, J.; Meyer, E.; Lee, H.; Conway, N. J.; Ahlstedt, D.; Yurtsever, Z.; Rainone, D.; Kuru, E.; Church, G.\nScore: 14.9, Published: 2023-06-29 DOI: 10.1101/2023.06.29.547106\nTherapeutic RNA oligonucleotides have shown tremendous potential to manage and treat disease, yet current manufacturing methods cannot deliver on this promise. Here, we report the development and optimization of a novel, aqueous-based, template-independent enzymatic RNA oligonucleotide synthesis platform as an alternative to traditional chemical methodologies. Our platform is made possible by reversible terminator nucleoside triphosphates and an enzyme capable of their incorporation. We show that many common therapeutic RNA modifications are compatible with our process and demonstrate the enzymatic synthesis of natural and modified oligonucleotides in both liquid and solid phases. Our platform offers many unique advantages over chemical synthesis, including the realization of a more sustainable process to produce therapeutic RNA oligonucleotides. One-Sentence SummaryAn enzyme and novel nucleotide building blocks are used to synthesize RNA oligonucleotides template independently under aqueous conditions.\nTruly tiny acoustic biomolecules for ultrasound imaging and therapy\nAuthors: Ling, B.; Gungoren, B.; Yao, Y.; Dutka, P.; Smith, C. A. B.; Lee, J.; Swift, M. B.; Shapiro, M. G.\nScore: 25.5, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546773\nNanotechnology offers significant advantages for medical imaging and therapy, including enhanced contrast and precision targeting. However, integrating these benefits into ultrasonography has been challenging due to the size and stability constraints of conventional bubble-based agents. Here we describe bicones, truly tiny acoustic contrast agents based on gas vesicles, a unique class of air-filled protein nanostructures naturally produced in buoyant microbes. We show that these sub-80 nm particles can be effectively detected both in vitro and in vivo, infiltrate tumors via leaky vasculature, deliver potent mechanical effects through ultrasound-induced inertial cavitation, and are easily engineered for molecular targeting, prolonged circulation time, and payload conjugation.\nModular and integrative activity reporters enhance biochemical studies in the yeast ER.\nAuthors: Martinusen, S. G.; Slaton, E. W.; Nelson, S. E.; Pulgar, M. A.; Besu, J. T.; Simas, C. F.; Denard, C. A.\nScore: 4.5, Published: 2023-07-12 DOI: 10.1101/2023.07.12.548713\nThe yeast endoplasmic reticulum sequestration and screening (YESS) system is a generalizable platform that has become highly useful to investigate post-translational modification enzymes (PTM-enzymes). This system enables researchers to profile and engineer the activity and substrate specificity of PTM-enzymes and to discover inhibitor-resistant enzyme mutants. In this study, we expand the capabilities of YESS by transferring its functional components to integrative plasmids. The YESS integrative system yields uniform protein expression and protease activities in various configurations, allows one to integrate activity reporters at two independent loci and to split the system between integrative and centromeric plasmids. We characterize these integrative reporters with two viral proteases, Tobacco etch virus (TEVp) and 3-chymotrypsin like protease (3CLpro), in terms of coefficient of variance, signal-to-noise ratio and fold-activation. Overall, we provide a framework for chromosomal-based studies that is modular, enabling rigorous high-throughput assays of PTM-enzymes in yeast.\nEngineered sensor bacteria evolve master-level gameplay through accelerated adaptation\nAuthors: Prakash, S.; Racovita, A.; Varela, C.; Walsh, M.; Galizi, R.; Isalan, M.; Jaramillo, A.\nScore: 260.6, Published: 2023-07-11 DOI: 10.1101/2022.04.22.489191\nGene circuits enable cells to make decisions by controlling the expression of genes in reaction to specific environmental factors1. These circuits can be designed to encode logical operations2-7, but implementation of more complex algorithms has proved more challenging. Directed evolution optimizes gene circuits8 without the need for design knowledge9, but adjusting multiple genes and conditions10 in genotype searches poses challenges11. Here we show a multicellular sensor system, AdaptoCells, in Escherichia coli, that can evolve complex behavior through an accelerated adaptation to chemical environments. AdaptoCells recognize chemical patterns and act as a decision-making system. Using an iterative improvement method, we demonstrate that the AdaptoCells can evolve to achieve mastery in the game of tic-tac-toe, demonstrating an unprecedented level of complexity for engineered living cells. We provide an effective and straightforward way to encode complexity in gene circuits, allowing for fast adaptation in response to dynamic environments and leading to optimal decisions.\nEfficient PCR-based gene targeting in isolates of the non-conventional yeast Debaryomyces hansenii\nAuthors: Alhajouj, S.; Turkolmez, S.; Abalkhail, T.; Alwan, Z. H. O.; Gilmour, D. J.; Mitchell, P. J.; Hettema, E. H.\nScore: 1.2, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548570\nDebaryomyces hansenii is a yeast with considerable biotechnological potential as an osmotolerant, stress tolerant oleaginous microbe. However, targeted genome modification tools are limited and require a strain with auxotrophic markers. Gene targeting by homologous recombination has been reported to be inefficient but here we describe a set of reagents and a method that allows gene targeting at high efficiency in wild type isolates. It uses a simple PCR-based amplification that extends a completely heterologous selectable marker with 50 bp flanks identical to the target site in the genome. Transformants integrate the PCR product through homologous recombination at high frequency (\u003e75%). We illustrate the potential of this method by disrupting genes at high efficiency and by expressing a heterologous protein from a safe chromosomal harbour site. These methods should stimulate and facilitate further analysis of D. hansenii strains and open the way to engineer strains for biotechnology.\n",
  "wordCount" : "1927",
  "inLanguage": "en",
  "datePublished": "2023-07-16T10:39:40Z",
  "dateModified": "2023-07-16T10:39:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on July 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548081">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548081" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548081">
        <p class="paperTitle">Emergent digital bio-computation through spatial diffusion and engineered bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548081" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548081" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fedorec, A. J. H.; Treloar, N. J.; Wen, K. Y.; Dekker, L.; Ong, Q. H.; Jurkeviciute, G.; Lyu, E.; Rutter, J.; Rosa, L.; Zaikin, A.; Barnes, C. P.</p>
        <p class="info">Score: 24.2, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548081' target='https://doi.org/10.1101/2023.07.07.548081'> 10.1101/2023.07.07.548081</a></p>
        <p class="abstract">Building computationally capable biological systems has long been an aim of synthetic biology. The potential utility of biocomputing devices ranges from biosafety and environmental applications to diagnosis and personalised medicine. Here we present work for the design of bacterial computers which use spatial patterning to process information. Our computers are composed of bacterial colonies which, inspired by patterning in embryo development, receive information in the form of diffusible morphogen-like signals. A computation is encoded by the physical locations of the input sources of morphogen and the output receiver colonies. We demonstrate, mathematically and with engineered Escherichia coli, the simple digital logic capability of single bacterial colonies and show how additional colonies are required to build complex functions. Inspired by electronic design automation, an algorithm for designing optimal spatial circuits computing two-level digital functions is presented. This enhances the capability of our system to complex digital functions without increasing the biological complexity. We extend our experimental system to incorporate biosensing colonies as morphogen sources, demonstrating how a diagnostic device might be constructed. Our approach will open up new ways to perform biological computation, with applications in bioengineering, biomaterials and biosensing. Ultimately, these computational bacterial communities will help us explore information processing in natural biological systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547496">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547496" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547496">
        <p class="paperTitle">xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547496" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547496" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Gong, J.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.</p>
        <p class="info">Score: 84.2, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547496' target='https://doi.org/10.1101/2023.07.05.547496'> 10.1101/2023.07.05.547496</a></p>
        <p class="abstract">Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently. This paper proposes a unified protein language model, xTrimoPGLM, to address these two types of tasks simultaneously through an innovative pre-training framework. Our key technical contribution is an exploration of the compatibility and the potential for joint optimization of the two types of objectives, which has led to a strategy for training xTrimoPGLM at an unprecedented scale of 100 billion parameters and 1 trillion training tokens. Our extensive experiments reveal that xTrimoPGLM significantly outperforms other advanced baselines in diverse protein understanding tasks (13 out of 15 tasks across four categories) and generates novel protein sequences which are structurally similar to natural ones. Furthermore, using the same xTrimoPGLM framework, we train an antibody-specific model (xTrimoPGLM-Ab) using 1 billion parameters. This model set a new record in predicting antibody naturalness and structures, both essential to the field of antibody-based drug design, and demonstrated a significantly faster inference speed than AlphaFold2. These results highlight the substantial capability and versatility of xTrimoPGLM in understanding and generating protein sequences.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548538">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548538" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548538">
        <p class="paperTitle">Highly efficient cellular expression of circular mRNA enables prolonged protein expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548538" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548538" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Unti, M. J.; Jaffrey, S. R.</p>
        <p class="info">Score: 11.6, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548538' target='https://doi.org/10.1101/2023.07.11.548538'> 10.1101/2023.07.11.548538</a></p>
        <p class="abstract">A major problem with mRNA therapeutics is the limited duration of protein expression due to the short half-life of mRNA. New approaches for generating highly stable circular mRNA in vitro have allowed increased duration of protein expression. However, it remains difficult to genetically encode circular mRNAs in mammalian cells, which limits the use of circular mRNA in cell-derived therapeutics. Here we describe the adaptation of the Tornado (Twister-optimized RNA for durable overexpression) system to achieve in-cell synthesis of circular mRNAs. We identify the promoter and internal ribosomal entry site (IRES) that result in high levels of protein expression in cells. We then show that these circular mRNAs can be packaged into virus-like particles (VLPs) thus enabling prolonged protein expression. Overall, these data describe a platform for synthesis of circular mRNAs and how these circular mRNAs can markedly enhance the ability of VLPs to function as a mRNA delivery tool.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.549043">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.549043" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.549043">
        <p class="paperTitle">Generative and predictive neural networks for the design of functional RNA molecules</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.549043" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.549043" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Riley, A. T.; Robson, J. M.; Green, A. A.</p>
        <p class="info">Score: 8.6, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.549043' target='https://doi.org/10.1101/2023.07.14.549043'> 10.1101/2023.07.14.549043</a></p>
        <p class="abstract">RNA is a remarkably versatile molecule that has been engineered for applications in therapeutics, diagnostics, and in vivo information-processing systems. However, the complex relationship between the sequence and structural properties of an RNA molecule and its ability to perform specific functions often necessitates extensive experimental screening of candidate sequences. Here we present a generalized neural network architecture that utilizes the sequence and structure of RNA molecules (SANDSTORM) to inform functional predictions. We demonstrate that this approach achieves state-of-the-art performance across several distinct RNA prediction tasks, while learning interpretable abstractions of RNA secondary structure. We paired these predictive models with generative adversarial RNA design networks (GARDN), allowing the generative modelling of novel mRNA 5-prime untranslated regions and toehold switch riboregulators exhibiting a predetermined fitness. This approach enabled the design of novel toehold switches with a 43-fold increase in experimentally characterized dynamic range compared to those designed using classic thermodynamic algorithms. SANDSTORM and GARDN thus represent powerful new predictive and generative tools for the development of diagnostic and therapeutic RNA molecules with improved function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.09.21.508766">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.09.21.508766" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.09.21.508766">
        <p class="paperTitle">An expanded genetic toolbox to accelerate the creation of Acholeplasma laidlawii driven by synthetic genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.09.21.508766" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.09.21.508766" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nucifora, D. P.; Mehta, N. D.; Giguere, D. J.; Karas, B. J.</p>
        <p class="info">Score: 29.4, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.09.21.508766' target='https://doi.org/10.1101/2022.09.21.508766'> 10.1101/2022.09.21.508766</a></p>
        <p class="abstract">Assembling synthetic bacterial genomes in yeast and genome transplantation has enabled an unmatched level of bacterial strain engineering, giving rise to cells with minimal and chemically synthetic genomes. However, this technology is currently limited to members of the Spiroplasma phylogenetic group, mostly Mycoplasmas, within the Mollicute class. Here, we propose new genetic tools for developing these technologies for Acholeplasma laidlawii, which is phylogenetically distant from Mycoplasmas and, unlike most Mollicutes, uses a standard genetic code. We first investigated a donor-recipient relationship between two A. laidlawii strains through whole-genome sequencing. We then created multi-host shuttle plasmids and used them to optimize an electroporation protocol. We also demonstrated the use of evolution to create superior strains for DNA uptake via electroporation. For genome transplantation, we selected A. laidlawii 8195 as the recipient strain and created a PG-8A donor strain by inserting a Tn5 transposon carrying a tetracycline resistance gene. The tools presented here will improve Acholeplasma research and accelerate the effort toward creating A. laidlawii strains driven by synthetic genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547106">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547106" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547106">
        <p class="paperTitle">A Platform for Controlled Template-Independent Enzymatic Synthesis of RNA Oligonucleotides and Therapeutics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547106" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547106" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wiegand, D. J.; Rittichier, J.; Meyer, E.; Lee, H.; Conway, N. J.; Ahlstedt, D.; Yurtsever, Z.; Rainone, D.; Kuru, E.; Church, G.</p>
        <p class="info">Score: 14.9, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547106' target='https://doi.org/10.1101/2023.06.29.547106'> 10.1101/2023.06.29.547106</a></p>
        <p class="abstract">Therapeutic RNA oligonucleotides have shown tremendous potential to manage and treat disease, yet current manufacturing methods cannot deliver on this promise. Here, we report the development and optimization of a novel, aqueous-based, template-independent enzymatic RNA oligonucleotide synthesis platform as an alternative to traditional chemical methodologies. Our platform is made possible by reversible terminator nucleoside triphosphates and an enzyme capable of their incorporation. We show that many common therapeutic RNA modifications are compatible with our process and demonstrate the enzymatic synthesis of natural and modified oligonucleotides in both liquid and solid phases. Our platform offers many unique advantages over chemical synthesis, including the realization of a more sustainable process to produce therapeutic RNA oligonucleotides.

One-Sentence SummaryAn enzyme and novel nucleotide building blocks are used to synthesize RNA oligonucleotides template independently under aqueous conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546773">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546773" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546773">
        <p class="paperTitle">Truly tiny acoustic biomolecules for ultrasound imaging and therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546773" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546773" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ling, B.; Gungoren, B.; Yao, Y.; Dutka, P.; Smith, C. A. B.; Lee, J.; Swift, M. B.; Shapiro, M. G.</p>
        <p class="info">Score: 25.5, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546773' target='https://doi.org/10.1101/2023.06.27.546773'> 10.1101/2023.06.27.546773</a></p>
        <p class="abstract">Nanotechnology offers significant advantages for medical imaging and therapy, including enhanced contrast and precision targeting. However, integrating these benefits into ultrasonography has been challenging due to the size and stability constraints of conventional bubble-based agents. Here we describe bicones, truly tiny acoustic contrast agents based on gas vesicles, a unique class of air-filled protein nanostructures naturally produced in buoyant microbes. We show that these sub-80 nm particles can be effectively detected both in vitro and in vivo, infiltrate tumors via leaky vasculature, deliver potent mechanical effects through ultrasound-induced inertial cavitation, and are easily engineered for molecular targeting, prolonged circulation time, and payload conjugation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548713">
        <p class="paperTitle">Modular and integrative activity reporters enhance biochemical studies in the yeast ER.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinusen, S. G.; Slaton, E. W.; Nelson, S. E.; Pulgar, M. A.; Besu, J. T.; Simas, C. F.; Denard, C. A.</p>
        <p class="info">Score: 4.5, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548713' target='https://doi.org/10.1101/2023.07.12.548713'> 10.1101/2023.07.12.548713</a></p>
        <p class="abstract">The yeast endoplasmic reticulum sequestration and screening (YESS) system is a generalizable platform that has become highly useful to investigate post-translational modification enzymes (PTM-enzymes). This system enables researchers to profile and engineer the activity and substrate specificity of PTM-enzymes and to discover inhibitor-resistant enzyme mutants. In this study, we expand the capabilities of YESS by transferring its functional components to integrative plasmids. The YESS integrative system yields uniform protein expression and protease activities in various configurations, allows one to integrate activity reporters at two independent loci and to split the system between integrative and centromeric plasmids. We characterize these integrative reporters with two viral proteases, Tobacco etch virus (TEVp) and 3-chymotrypsin like protease (3CLpro), in terms of coefficient of variance, signal-to-noise ratio and fold-activation. Overall, we provide a framework for chromosomal-based studies that is modular, enabling rigorous high-throughput assays of PTM-enzymes in yeast.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.04.22.489191">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.04.22.489191" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.04.22.489191">
        <p class="paperTitle">Engineered sensor bacteria evolve master-level gameplay through accelerated adaptation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.04.22.489191" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.04.22.489191" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Prakash, S.; Racovita, A.; Varela, C.; Walsh, M.; Galizi, R.; Isalan, M.; Jaramillo, A.</p>
        <p class="info">Score: 260.6, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.04.22.489191' target='https://doi.org/10.1101/2022.04.22.489191'> 10.1101/2022.04.22.489191</a></p>
        <p class="abstract">Gene circuits enable cells to make decisions by controlling the expression of genes in reaction to specific environmental factors1. These circuits can be designed to encode logical operations2-7, but implementation of more complex algorithms has proved more challenging. Directed evolution optimizes gene circuits8 without the need for design knowledge9, but adjusting multiple genes and conditions10 in genotype searches poses challenges11. Here we show a multicellular sensor system, AdaptoCells, in Escherichia coli, that can evolve complex behavior through an accelerated adaptation to chemical environments. AdaptoCells recognize chemical patterns and act as a decision-making system. Using an iterative improvement method, we demonstrate that the AdaptoCells can evolve to achieve mastery in the game of tic-tac-toe, demonstrating an unprecedented level of complexity for engineered living cells. We provide an effective and straightforward way to encode complexity in gene circuits, allowing for fast adaptation in response to dynamic environments and leading to optimal decisions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548570">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548570" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548570">
        <p class="paperTitle">Efficient PCR-based gene targeting in isolates of the non-conventional yeast Debaryomyces hansenii</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548570" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548570" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alhajouj, S.; Turkolmez, S.; Abalkhail, T.; Alwan, Z. H. O.; Gilmour, D. J.; Mitchell, P. J.; Hettema, E. H.</p>
        <p class="info">Score: 1.2, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548570' target='https://doi.org/10.1101/2023.07.11.548570'> 10.1101/2023.07.11.548570</a></p>
        <p class="abstract">Debaryomyces hansenii is a yeast with considerable biotechnological potential as an osmotolerant, stress tolerant oleaginous microbe. However, targeted genome modification tools are limited and require a strain with auxotrophic markers. Gene targeting by homologous recombination has been reported to be inefficient but here we describe a set of reagents and a method that allows gene targeting at high efficiency in wild type isolates. It uses a simple PCR-based amplification that extends a completely heterologous selectable marker with 50 bp flanks identical to the target site in the genome. Transformants integrate the PCR product through homologous recombination at high frequency (&gt;75%). We illustrate the potential of this method by disrupting genes at high efficiency and by expressing a heterologous protein from a safe chromosomal harbour site. These methods should stimulate and facilitate further analysis of D. hansenii strains and open the way to engineer strains for biotechnology.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
