<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Species-specific design of artificial promoters by transfer-learning based generative deep-learning model
Authors: Xia, Y.; Du, X.; Liu, B.; Guo, S.; Huo, Y.-X.
Score: 9.8, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573416
Native prokaryotic promoters share common sequence patterns, but are species dependent. For understudied species with limited data, it is challenging to predict the strength of existing promoters and generate novel promoters. Here, we developed PromoGen, a collection of nucleotide language models to generate species-specific functional promoters, across dozens of species in a data and parameter efficient way.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Species-specific design of artificial promoters by transfer-learning based generative deep-learning model
Authors: Xia, Y.; Du, X.; Liu, B.; Guo, S.; Huo, Y.-X.
Score: 9.8, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573416
Native prokaryotic promoters share common sequence patterns, but are species dependent. For understudied species with limited data, it is challenging to predict the strength of existing promoters and generate novel promoters. Here, we developed PromoGen, a collection of nucleotide language models to generate species-specific functional promoters, across dozens of species in a data and parameter efficient way." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-03T10:39:43+00:00" />
<meta property="article:modified_time" content="2024-01-03T10:39:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Species-specific design of artificial promoters by transfer-learning based generative deep-learning model
Authors: Xia, Y.; Du, X.; Liu, B.; Guo, S.; Huo, Y.-X.
Score: 9.8, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573416
Native prokaryotic promoters share common sequence patterns, but are species dependent. For understudied species with limited data, it is challenging to predict the strength of existing promoters and generate novel promoters. Here, we developed PromoGen, a collection of nucleotide language models to generate species-specific functional promoters, across dozens of species in a data and parameter efficient way."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Species-specific design of artificial promoters by transfer-learning based generative deep-learning model\nAuthors: Xia, Y.; Du, X.; Liu, B.; Guo, S.; Huo, Y.-X.\nScore: 9.8, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573416\nNative prokaryotic promoters share common sequence patterns, but are species dependent. For understudied species with limited data, it is challenging to predict the strength of existing promoters and generate novel promoters. Here, we developed PromoGen, a collection of nucleotide language models to generate species-specific functional promoters, across dozens of species in a data and parameter efficient way.",
  "keywords": [
    
  ],
  "articleBody": " Species-specific design of artificial promoters by transfer-learning based generative deep-learning model\nAuthors: Xia, Y.; Du, X.; Liu, B.; Guo, S.; Huo, Y.-X.\nScore: 9.8, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573416\nNative prokaryotic promoters share common sequence patterns, but are species dependent. For understudied species with limited data, it is challenging to predict the strength of existing promoters and generate novel promoters. Here, we developed PromoGen, a collection of nucleotide language models to generate species-specific functional promoters, across dozens of species in a data and parameter efficient way. Twenty-seven species-specific models in this collection were finetuned from the pretrained model which was trained on multi-species promoters. When systematically compared with native promoters, the Escherichia coli- and Bacillus subtilis-specific artificial PromoGen-generated promoters (PGPs) were demonstrated to hold all essential features and distribution patterns of native promoters. A regression model was developed to score generated either by PromoGen or by another competitive neural network, and the overall score of PGPs is higher. Encouraged by in silico analysis, we further experimentally characterized twenty-two B. subtilis PGPs in vivo, results showed that four of tested PGPs reached the strong promoter level while all were active. Furthermore, we developed a user-friendly website to generate species-specific promoters for 27 different species by PromoGen. This work presented an efficient deep-learning strategy for de novo species-specific promoter generation even with limited datasets, providing valuable promoter toolboxes especially for the metabolic engineering of understudied microorganisms. O_FIG O_LINKSMALLFIG WIDTH=138 HEIGHT=200 SRC=\"FIGDIR/small/573416v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (33K): org.highwire.dtl.DTLVardef@612fbdorg.highwire.dtl.DTLVardef@852490org.highwire.dtl.DTLVardef@1239650org.highwire.dtl.DTLVardef@1617e92_HPS_FORMAT_FIGEXP M_FIG C_FIG\nContinual improvement of multiplex mutagenesis in Arabidopsis\nAuthors: Develtere, W.; Decaestecker, W.; Rombaut, D.; Anders, C.; Clicque, E.; Vuylsteke, M.; Jacobs, T. B.\nScore: 21.4, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572307\nCRISPR/Cas9 is currently the most powerful tool to generate mutations in plant genomes and more efficient tools are needed as the scale of experiments increases. In the model plant Arabidopsis, the choice of promoter driving Cas9 expression is critical to generate germline mutations. Several optimal promoters have been reported. However, it is unclear which promoter is ideal as they have not been thoroughly tested side-by-side. Furthermore, most plant vectors still use one of the two Cas9 nuclear localization sequence (NLS) configurations initially reported and can still be optimized. We genotyped more than 6,000 Arabidopsis T2 plants to test seven promoters and eleven NLS architectures across 14 targets to systematically improve the generation of single and multiplex inheritable mutations. We find that the RPS5A promoter and double-BP NLS architecture were individually the most efficient components. When combined, 99% of T2 plant contained at least one knockout mutation and 84% contained 4-7-plex knock-outs. These optimizations will be useful to generate higher-order knockouts in the germline of Arabidopsis and likely be applicable to other CRISPR systems as well.\nP-NADs: PUX-based NAnobody Degraders for Ubiquitin-Independent Degradation of Target Proteins\nAuthors: Wang, J.; Chistov, G.; Zhang, J.; Huntington, B.; Salem, I.; Sandholu, A.; Arold, S. T.\nScore: 2.1, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572486\nTargeted protein degradation (TPD) allows cells to maintain a functional proteome and to rapidly adapt to changing conditions. Methods that repurpose TPD for the deactivation of specific proteins have demonstrated significant potential in therapeutic and research applications. Most of these methods are based on proteolysis targeting chimaera (PROTAC) which link the protein target to an E3 ubiquitin ligase binding moiety, resulting in the ubiquitin-based degradation of the target protein. In this study, we introduce a method for ubiquitin-independent TPD based on nanobody-conjugated plant ubiquitin regulatory X domain-containing (PUX) adaptor proteins. We show that the PUX-based NAnobody Degraders (P-NADs) can unfold a target protein through the Arabidopsis and human orthologues of the CDC48 unfoldase without the need for ubiquitination or initiating motifs. Despite originating from plants, P-NAD plasmids can be transfected into a human cell line, where produced proteins use the endogenous CDC48 machinery for ubiquitin-independent TPD. Thus, P-NADs pave the road for ubiquitin-independent therapeutic TPD approaches. The P-NAD design combined with established in vitro and cellular assays make this system also a versatile platform for elucidating functional aspects of CDC48-based TPD in plants and animals.\nBuilding synthetic biosensors using red blood cell proteins\nAuthors: Dolberg, T. B.; Gunnels, T. F.; Ling, T.; Sarnese, K. A.; Crispino, J. D.; Leonard, J. N.\nScore: 19.9, Published: 2023-12-16 DOI: 10.1101/2023.12.16.571988\nAs the use of engineered cell therapies expands from pioneering efforts in cancer immunotherapy to other applications, an attractive but less explored approach is the use of engineered red blood cells (RBCs). Compared to other cells, RBCs have a very long circulation time and reside in the blood compartment, so they could be ideally suited for applications as sentinel cells that enable in situ sensing and diagnostics. However, we largely lack tools for converting RBCs into biosensors. A unique challenge is that RBCs remodel their membranes during maturation, shedding many membrane components, suggesting that an RBC-specific approach may be needed. Towards addressing this need, here we develop a biosensing architecture built on RBC membrane proteins that are retained through erythropoiesis. This biosensor employs a mechanism in which extracellular ligand binding is transduced into intracellular reconstitution of a split output protein (including either a fluorophore or an enzyme). By comparatively evaluating a range of biosensor architectures, linker types, scaffold choices, and output signals, we identify biosensor designs and design features that confer substantial ligand-induced signal in vitro. Finally, we demonstrate that erythroid precursor cells engineered with our RBC protein biosensors function in vivo. This study establishes a foundation for developing RBC-based biosensors that could ultimately address unmet needs including non-invasive monitoring of physiological signals for a range of diagnostic applications.\nA long-acting prolactin to combat lactation insufficiency\nAuthors: Kready, K.; Doiron, K.; Redfield Chan, K.; Way, J. C.; Justman, Q.; Powe, C. E.; Silver, P. A.\nScore: 26.7, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571886\nHuman infants are born to breastfeed. While 50% of lactating persons struggle to make enough milk, there are no governmentally-approved drugs to enhance lactation1. Here, we engineer a variant of the naturally-occurring driver of lactation, the hormone Prolactin, to increase its serum half-life and produce a viable drug candidate. Our engineered variant, Prolactin-eXtra Long-acting (Prolactin-XL), is comprised of endogenously active human prolactin fused to an engineered human IgG Fc domain designed to overcome the unique drug development challenges specific to the lactating person-infant dyad. Our Prolactin-XL has a serum half-life of 70.9h in mice, 2,625-fold longer than endogenously active prolactin alone (70.9h v. 0.027h). We demonstrate that Prolactin-XL increases milk production and restores growth of pups fed by dams with pharmacologically-ablated lactation. We show that Prolactin-XL-enhanced lactation is accompanied by reversible, lactocyte-driven changes in mammary gland morphology. This work establishes long-acting prolactins as a potentially powerful pharmacologic means to combat insufficient lactation.\nVirus-like DNA tracers for characterizing wastewater-based pathogen monitoring systems\nAuthors: Gopal, A.; Machtinger, A. N.; Wang, B.; Rice, D. P.; McLaren, M. R.; Esvelt, K. M.\nScore: 2.2, Published: 2023-12-25 DOI: 10.1101/2023.12.24.573197\nWastewater-based pathogen monitoring has recently become a ubiquitous method for tracking the abundance and evolution of pathogens in a community or port of entry. But accurate inference of disease incidence from wastewater remains challenging, in large part due to noise and bias arising from wastewater transport and measurement. We developed a new system of tracer particles designed to enhance wastewater monitoring of viral pathogens. These tracers, which consist of synthetic DNA barcodes packaged into non-infectious viral capsids, can be deposited at known amounts and times into the sewer system and measured in wastewater samples to assess features such as assay sensitivity, and the decay and delay in viral signal arising from a shedding event. Here we describe the design, construction, and initial characterization of a 25-plex tracer library. Our tracer library supports simultaneous measurement of up to 5 distinct tracers by qPCR and up to 25 distinct tracers by amplicon or metagenomic sequencing. We demonstrate experimentally that the tracers are biologically inert and exhibit good stability and qPCR-assay performance in wastewater. These features collectively suggest that barcoded, virus-like tracers have the required properties for high-accuracy, high-resolution characterization of wastewater monitoring systems.\nSculpting conducting nanopore size and shape through de novo protein design\nAuthors: Berhanu, S.; Majumder, S.; Muntener, T.; Whitehouse, J.; Berner, C.; Bera, A. K.; Kang, A.; Liang, B.; Khan, G. N.; Sankaran, B.; Tamm, L.; Brockwel, D. J.; Hiller, S.; Radford, S.; Baker, D.; Vorobieva, A. A.\nScore: 31.1, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572500\nTransmembrane {beta}-barrels (TMBs) are widely used for single molecule DNA and RNA sequencing and have considerable potential for a broad range of sensing and sequencing applications. Current engineering approaches for nanopore sensors are limited to naturally occurring channels such as CsgG, which have evolved to carry out functions very different from sensing, and hence provide sub-optimal starting points. In contrast, de novo protein design can in principle create an unlimited number of new nanopores with any desired properties. Here we describe a general approach to the design of transmembrane {beta}-barrel pores with different diameter and pore geometry. NMR and crystallographic characterization shows that the designs are stably folded with structures close to the design models. We report the first examples of de novo designed TMBs with 10, 12 and 14 stranded {beta}-barrels. The designs have distinct conductances that correlate with their pore diameter, ranging from 110 pS ([~]0.5 nm pore diameter) to 430 pS ([~]1.1 nm pore diameter), and can be converted into sensitive small-molecule sensors with high signal to noise ratio. The capability to generate on demand {beta}-barrel pores of defined geometry opens up fundamentally new opportunities for custom engineering of sequencing and sensing technologies. One sentence summaryDe novo design enables the generation of stable and quite transmembrane beta-barrel nanopores with tailored sizes, shapes and properties.\nStructure-Infused Protein Language Models\nAuthors: Penaherrera, D.; Koes, D. R.\nScore: 1.2, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571525\nEmbeddings from protein language models (PLMs) capture intricate patterns for protein sequences, enabling more accurate and efficient prediction of protein properties. Incorporating protein structure information as direct input into PLMs results in an improvement on the predictive ability of protein embeddings on downstream tasks. In this work we demonstrate that indirectly infusing structure information into PLMs also leads to performance gains on structure related tasks. The key difference between this framework and others is that at inference time the model does not require access to structure to produce its embeddings.\nPHEIGES, all-cell-free phage synthesis and selection from engineered genomes\nAuthors: Levrier, A.; Karpathakis, I.; Nash, B.; Bowden, S. D.; Lindner, A. B.; Noireaux, V.\nScore: 11.3, Published: 2023-12-19 DOI: 10.1101/2023.12.07.570578\nBacteriophages constitute an invaluable biological reservoir for biotechnology and medicine. The ability to exploit such vast resources is hampered by the lack of methods to rapidly engineer, assemble, package genomes, and select phages. Cell-free transcription-translation (TXTL) offers experimental settings to address such a limitation. Here, we describe PHage Engineering by In vitro Gene Expression and Selection (PHEIGES) using T7 phage genome and Escherichia coli TXTL. Phage genomes are assembled in vitro from PCR-amplified fragments and directly expressed in batch TXTL reactions to produce up to 1011 PFU/ml engineered phages within one day. We further demonstrate an important genotype-phenotype linkage of phage assembly in bulk TXTL. This enables rapid selection of phages with altered rough lipopolysaccharides specificity from phage genomes incorporating tail fiber mutant libraries. We establish the scalability of PHEIGES by one-pot assembly of such mutants with fluorescent gene integration and 10% length-reduced genome.\nThermogenetics for cardiac pacing\nAuthors: Balatskiy, A. V.; Nesterenko, A. M.; Lanin, A. A.; Ovechkina, V. S.; Sabinin, S. S.; Fetisova, E. S.; Moshchenko, A. A.; Jappy, D.; Sokolov, R. A.; Biglova, D. Z.; Solius, G. M.; Solius, E. M.; Korolev, S. V.; Podgorny, O. V.; Kelmanson, I. V.; Rozov, A. V.; Fedotov, A. B.; Bruegmann, T.; Zheltikov, A. M.; Mozhaev, A. A.; Belousov, V. V.\nScore: 0.2, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573885\nCardiac arrhythmias are common disorders that can be fatal. Modern methods of treating bradyarrhythmias include the implantation of pacemakers and cardioverters - defibrillators. However, the implantable devices can cause various complications including infectious ones, related to the electrodes installed inside the heart. Less invasive heart rhythm modulation could be beneficial for some cohorts of patients. We present an alternative approach to heart pacing based on thermogenetics. We used adeno-associated viruses to deliver genetic human transient receptor potential subfamily V member 1 (TRPV1), a heat-sensitive cation channel, into isolated cardiomyocytes and the mouse heart. This allowed us to induce action potentials and control contractility using short heat pulses delivered by infrared laser illumination. Using this approach, we demonstrated the thermogenetic pacing of isolated cardiomyocytes in vitro and in the mouse heart in vivo. Our results demonstrate the unique potential of thermogenetics for developing novel therapeutic strategies for heart rhythm modulation.\n",
  "wordCount" : "2098",
  "inLanguage": "en",
  "datePublished": "2024-01-03T10:39:43Z",
  "dateModified": "2024-01-03T10:39:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 3, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573416">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573416" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573416">
        <p class="paperTitle">Species-specific design of artificial promoters by transfer-learning based generative deep-learning model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573416" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573416" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xia, Y.; Du, X.; Liu, B.; Guo, S.; Huo, Y.-X.</p>
        <p class="info">Score: 9.8, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573416' target='https://doi.org/10.1101/2023.12.27.573416'> 10.1101/2023.12.27.573416</a></p>
        <p class="abstract">Native prokaryotic promoters share common sequence patterns, but are species dependent. For understudied species with limited data, it is challenging to predict the strength of existing promoters and generate novel promoters. Here, we developed PromoGen, a collection of nucleotide language models to generate species-specific functional promoters, across dozens of species in a data and parameter efficient way. Twenty-seven species-specific models in this collection were finetuned from the pretrained model which was trained on multi-species promoters. When systematically compared with native promoters, the Escherichia coli- and Bacillus subtilis-specific artificial PromoGen-generated promoters (PGPs) were demonstrated to hold all essential features and distribution patterns of native promoters. A regression model was developed to score generated either by PromoGen or by another competitive neural network, and the overall score of PGPs is higher. Encouraged by in silico analysis, we further experimentally characterized twenty-two B. subtilis PGPs in vivo, results showed that four of tested PGPs reached the strong promoter level while all were active. Furthermore, we developed a user-friendly website to generate species-specific promoters for 27 different species by PromoGen. This work presented an efficient deep-learning strategy for de novo species-specific promoter generation even with limited datasets, providing valuable promoter toolboxes especially for the metabolic engineering of understudied microorganisms.



O_FIG O_LINKSMALLFIG WIDTH=138 HEIGHT=200 SRC=&#34;FIGDIR/small/573416v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (33K):
org.highwire.dtl.DTLVardef@612fbdorg.highwire.dtl.DTLVardef@852490org.highwire.dtl.DTLVardef@1239650org.highwire.dtl.DTLVardef@1617e92_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572307">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572307" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572307">
        <p class="paperTitle">Continual improvement of multiplex mutagenesis in Arabidopsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572307" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572307" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Develtere, W.; Decaestecker, W.; Rombaut, D.; Anders, C.; Clicque, E.; Vuylsteke, M.; Jacobs, T. B.</p>
        <p class="info">Score: 21.4, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572307' target='https://doi.org/10.1101/2023.12.19.572307'> 10.1101/2023.12.19.572307</a></p>
        <p class="abstract">CRISPR/Cas9 is currently the most powerful tool to generate mutations in plant genomes and more efficient tools are needed as the scale of experiments increases. In the model plant Arabidopsis, the choice of promoter driving Cas9 expression is critical to generate germline mutations. Several optimal promoters have been reported. However, it is unclear which promoter is ideal as they have not been thoroughly tested side-by-side. Furthermore, most plant vectors still use one of the two Cas9 nuclear localization sequence (NLS) configurations initially reported and can still be optimized. We genotyped more than 6,000 Arabidopsis T2 plants to test seven promoters and eleven NLS architectures across 14 targets to systematically improve the generation of single and multiplex inheritable mutations. We find that the RPS5A promoter and double-BP NLS architecture were individually the most efficient components. When combined, 99% of T2 plant contained at least one knockout mutation and 84% contained 4-7-plex knock-outs. These optimizations will be useful to generate higher-order knockouts in the germline of Arabidopsis and likely be applicable to other CRISPR systems as well.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572486">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572486" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572486">
        <p class="paperTitle">P-NADs: PUX-based NAnobody Degraders for Ubiquitin-Independent Degradation of Target Proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572486" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572486" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, J.; Chistov, G.; Zhang, J.; Huntington, B.; Salem, I.; Sandholu, A.; Arold, S. T.</p>
        <p class="info">Score: 2.1, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572486' target='https://doi.org/10.1101/2023.12.20.572486'> 10.1101/2023.12.20.572486</a></p>
        <p class="abstract">Targeted protein degradation (TPD) allows cells to maintain a functional proteome and to rapidly adapt to changing conditions. Methods that repurpose TPD for the deactivation of specific proteins have demonstrated significant potential in therapeutic and research applications. Most of these methods are based on proteolysis targeting chimaera (PROTAC) which link the protein target to an E3 ubiquitin ligase binding moiety, resulting in the ubiquitin-based degradation of the target protein. In this study, we introduce a method for ubiquitin-independent TPD based on nanobody-conjugated plant ubiquitin regulatory X domain-containing (PUX) adaptor proteins. We show that the PUX-based NAnobody Degraders (P-NADs) can unfold a target protein through the Arabidopsis and human orthologues of the CDC48 unfoldase without the need for ubiquitination or initiating motifs. Despite originating from plants, P-NAD plasmids can be transfected into a human cell line, where produced proteins use the endogenous CDC48 machinery for ubiquitin-independent TPD. Thus, P-NADs pave the road for ubiquitin-independent therapeutic TPD approaches. The P-NAD design combined with established in vitro and cellular assays make this system also a versatile platform for elucidating functional aspects of CDC48-based TPD in plants and animals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.16.571988">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.16.571988" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.16.571988">
        <p class="paperTitle">Building synthetic biosensors using red blood cell proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.16.571988" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.16.571988" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dolberg, T. B.; Gunnels, T. F.; Ling, T.; Sarnese, K. A.; Crispino, J. D.; Leonard, J. N.</p>
        <p class="info">Score: 19.9, Published: 2023-12-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.16.571988' target='https://doi.org/10.1101/2023.12.16.571988'> 10.1101/2023.12.16.571988</a></p>
        <p class="abstract">As the use of engineered cell therapies expands from pioneering efforts in cancer immunotherapy to other applications, an attractive but less explored approach is the use of engineered red blood cells (RBCs). Compared to other cells, RBCs have a very long circulation time and reside in the blood compartment, so they could be ideally suited for applications as sentinel cells that enable in situ sensing and diagnostics. However, we largely lack tools for converting RBCs into biosensors. A unique challenge is that RBCs remodel their membranes during maturation, shedding many membrane components, suggesting that an RBC-specific approach may be needed. Towards addressing this need, here we develop a biosensing architecture built on RBC membrane proteins that are retained through erythropoiesis. This biosensor employs a mechanism in which extracellular ligand binding is transduced into intracellular reconstitution of a split output protein (including either a fluorophore or an enzyme). By comparatively evaluating a range of biosensor architectures, linker types, scaffold choices, and output signals, we identify biosensor designs and design features that confer substantial ligand-induced signal in vitro. Finally, we demonstrate that erythroid precursor cells engineered with our RBC protein biosensors function in vivo. This study establishes a foundation for developing RBC-based biosensors that could ultimately address unmet needs including non-invasive monitoring of physiological signals for a range of diagnostic applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571886">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571886" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571886">
        <p class="paperTitle">A long-acting prolactin to combat lactation insufficiency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571886" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571886" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kready, K.; Doiron, K.; Redfield Chan, K.; Way, J. C.; Justman, Q.; Powe, C. E.; Silver, P. A.</p>
        <p class="info">Score: 26.7, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571886' target='https://doi.org/10.1101/2023.12.15.571886'> 10.1101/2023.12.15.571886</a></p>
        <p class="abstract">Human infants are born to breastfeed. While 50% of lactating persons struggle to make enough milk, there are no governmentally-approved drugs to enhance lactation1. Here, we engineer a variant of the naturally-occurring driver of lactation, the hormone Prolactin, to increase its serum half-life and produce a viable drug candidate. Our engineered variant, Prolactin-eXtra Long-acting (Prolactin-XL), is comprised of endogenously active human prolactin fused to an engineered human IgG Fc domain designed to overcome the unique drug development challenges specific to the lactating person-infant dyad. Our Prolactin-XL has a serum half-life of 70.9h in mice, 2,625-fold longer than endogenously active prolactin alone (70.9h v. 0.027h). We demonstrate that Prolactin-XL increases milk production and restores growth of pups fed by dams with pharmacologically-ablated lactation. We show that Prolactin-XL-enhanced lactation is accompanied by reversible, lactocyte-driven changes in mammary gland morphology. This work establishes long-acting prolactins as a potentially powerful pharmacologic means to combat insufficient lactation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.24.573197">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.24.573197" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.24.573197">
        <p class="paperTitle">Virus-like DNA tracers for characterizing wastewater-based pathogen monitoring systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.24.573197" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.24.573197" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gopal, A.; Machtinger, A. N.; Wang, B.; Rice, D. P.; McLaren, M. R.; Esvelt, K. M.</p>
        <p class="info">Score: 2.2, Published: 2023-12-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.24.573197' target='https://doi.org/10.1101/2023.12.24.573197'> 10.1101/2023.12.24.573197</a></p>
        <p class="abstract">Wastewater-based pathogen monitoring has recently become a ubiquitous method for tracking the abundance and evolution of pathogens in a community or port of entry. But accurate inference of disease incidence from wastewater remains challenging, in large part due to noise and bias arising from wastewater transport and measurement. We developed a new system of tracer particles designed to enhance wastewater monitoring of viral pathogens. These tracers, which consist of synthetic DNA barcodes packaged into non-infectious viral capsids, can be deposited at known amounts and times into the sewer system and measured in wastewater samples to assess features such as assay sensitivity, and the decay and delay in viral signal arising from a shedding event. Here we describe the design, construction, and initial characterization of a 25-plex tracer library. Our tracer library supports simultaneous measurement of up to 5 distinct tracers by qPCR and up to 25 distinct tracers by amplicon or metagenomic sequencing. We demonstrate experimentally that the tracers are biologically inert and exhibit good stability and qPCR-assay performance in wastewater. These features collectively suggest that barcoded, virus-like tracers have the required properties for high-accuracy, high-resolution characterization of wastewater monitoring systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572500">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572500" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572500">
        <p class="paperTitle">Sculpting conducting nanopore size and shape through de novo protein design</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572500" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572500" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Berhanu, S.; Majumder, S.; Muntener, T.; Whitehouse, J.; Berner, C.; Bera, A. K.; Kang, A.; Liang, B.; Khan, G. N.; Sankaran, B.; Tamm, L.; Brockwel, D. J.; Hiller, S.; Radford, S.; Baker, D.; Vorobieva, A. A.</p>
        <p class="info">Score: 31.1, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572500' target='https://doi.org/10.1101/2023.12.20.572500'> 10.1101/2023.12.20.572500</a></p>
        <p class="abstract">Transmembrane {beta}-barrels (TMBs) are widely used for single molecule DNA and RNA sequencing and have considerable potential for a broad range of sensing and sequencing applications. Current engineering approaches for nanopore sensors are limited to naturally occurring channels such as CsgG, which have evolved to carry out functions very different from sensing, and hence provide sub-optimal starting points. In contrast, de novo protein design can in principle create an unlimited number of new nanopores with any desired properties. Here we describe a general approach to the design of transmembrane {beta}-barrel pores with different diameter and pore geometry. NMR and crystallographic characterization shows that the designs are stably folded with structures close to the design models. We report the first examples of de novo designed TMBs with 10, 12 and 14 stranded {beta}-barrels. The designs have distinct conductances that correlate with their pore diameter, ranging from 110 pS ([~]0.5 nm pore diameter) to 430 pS ([~]1.1 nm pore diameter), and can be converted into sensitive small-molecule sensors with high signal to noise ratio. The capability to generate on demand {beta}-barrel pores of defined geometry opens up fundamentally new opportunities for custom engineering of sequencing and sensing technologies.

One sentence summaryDe novo design enables the generation of stable and quite transmembrane beta-barrel nanopores with tailored sizes, shapes and properties.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571525">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571525" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571525">
        <p class="paperTitle">Structure-Infused Protein Language Models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571525" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571525" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Penaherrera, D.; Koes, D. R.</p>
        <p class="info">Score: 1.2, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571525' target='https://doi.org/10.1101/2023.12.13.571525'> 10.1101/2023.12.13.571525</a></p>
        <p class="abstract">Embeddings from protein language models (PLMs) capture intricate patterns for protein sequences, enabling more accurate and efficient prediction of protein properties. Incorporating protein structure information as direct input into PLMs results in an improvement on the predictive ability of protein embeddings on downstream tasks. In this work we demonstrate that indirectly infusing structure information into PLMs also leads to performance gains on structure related tasks. The key difference between this framework and others is that at inference time the model does not require access to structure to produce its embeddings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570578">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570578" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570578">
        <p class="paperTitle">PHEIGES, all-cell-free phage synthesis and selection from engineered genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570578" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570578" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Levrier, A.; Karpathakis, I.; Nash, B.; Bowden, S. D.; Lindner, A. B.; Noireaux, V.</p>
        <p class="info">Score: 11.3, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570578' target='https://doi.org/10.1101/2023.12.07.570578'> 10.1101/2023.12.07.570578</a></p>
        <p class="abstract">Bacteriophages constitute an invaluable biological reservoir for biotechnology and medicine. The ability to exploit such vast resources is hampered by the lack of methods to rapidly engineer, assemble, package genomes, and select phages. Cell-free transcription-translation (TXTL) offers experimental settings to address such a limitation. Here, we describe PHage Engineering by In vitro Gene Expression and Selection (PHEIGES) using T7 phage genome and Escherichia coli TXTL. Phage genomes are assembled in vitro from PCR-amplified fragments and directly expressed in batch TXTL reactions to produce up to 1011 PFU/ml engineered phages within one day. We further demonstrate an important genotype-phenotype linkage of phage assembly in bulk TXTL. This enables rapid selection of phages with altered rough lipopolysaccharides specificity from phage genomes incorporating tail fiber mutant libraries. We establish the scalability of PHEIGES by one-pot assembly of such mutants with fluorescent gene integration and 10% length-reduced genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573885">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573885" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573885">
        <p class="paperTitle">Thermogenetics for cardiac pacing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573885" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573885" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Balatskiy, A. V.; Nesterenko, A. M.; Lanin, A. A.; Ovechkina, V. S.; Sabinin, S. S.; Fetisova, E. S.; Moshchenko, A. A.; Jappy, D.; Sokolov, R. A.; Biglova, D. Z.; Solius, G. M.; Solius, E. M.; Korolev, S. V.; Podgorny, O. V.; Kelmanson, I. V.; Rozov, A. V.; Fedotov, A. B.; Bruegmann, T.; Zheltikov, A. M.; Mozhaev, A. A.; Belousov, V. V.</p>
        <p class="info">Score: 0.2, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573885' target='https://doi.org/10.1101/2024.01.02.573885'> 10.1101/2024.01.02.573885</a></p>
        <p class="abstract">Cardiac arrhythmias are common disorders that can be fatal. Modern methods of treating bradyarrhythmias include the implantation of pacemakers and cardioverters - defibrillators. However, the implantable devices can cause various complications including infectious ones, related to the electrodes installed inside the heart. Less invasive heart rhythm modulation could be beneficial for some cohorts of patients. We present an alternative approach to heart pacing based on thermogenetics. We used adeno-associated viruses to deliver genetic human transient receptor potential subfamily V member 1 (TRPV1), a heat-sensitive cation channel, into isolated cardiomyocytes and the mouse heart. This allowed us to induce action potentials and control contractility using short heat pulses delivered by infrared laser illumination. Using this approach, we demonstrated the thermogenetic pacing of isolated cardiomyocytes in vitro and in the mouse heart in vivo. Our results demonstrate the unique potential of thermogenetics for developing novel therapeutic strategies for heart rhythm modulation.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
