<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="DNA-Diffusion: Leveraging Generative Models for Controlling Chromatin Accessibility and Gene Expression via Synthetic Regulatory Elements
Authors: Ferreira DaSilva, L.; Senan, S.; Patel, Z. M.; Reddy, A. J.; Gabbita, S.; Nussbaum, Z.; Cordova, C. M. V.; Wenteler, A.; Weber, N.; Tunjic, T. M.; Khan, T. A.; Li, Z.; Smith, C. R.; Bejan, M.; Louis, L. K.; Cornejo, P.; Connell, W.; Wong, E. S.; Meuleman, W.; Pinello, L.
Score: 63.5, Published: 2024-02-01 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="DNA-Diffusion: Leveraging Generative Models for Controlling Chromatin Accessibility and Gene Expression via Synthetic Regulatory Elements
Authors: Ferreira DaSilva, L.; Senan, S.; Patel, Z. M.; Reddy, A. J.; Gabbita, S.; Nussbaum, Z.; Cordova, C. M. V.; Wenteler, A.; Weber, N.; Tunjic, T. M.; Khan, T. A.; Li, Z.; Smith, C. R.; Bejan, M.; Louis, L. K.; Cornejo, P.; Connell, W.; Wong, E. S.; Meuleman, W.; Pinello, L.
Score: 63.5, Published: 2024-02-01 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-11T10:38:55+00:00" />
<meta property="article:modified_time" content="2024-02-11T10:38:55+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="DNA-Diffusion: Leveraging Generative Models for Controlling Chromatin Accessibility and Gene Expression via Synthetic Regulatory Elements
Authors: Ferreira DaSilva, L.; Senan, S.; Patel, Z. M.; Reddy, A. J.; Gabbita, S.; Nussbaum, Z.; Cordova, C. M. V.; Wenteler, A.; Weber, N.; Tunjic, T. M.; Khan, T. A.; Li, Z.; Smith, C. R.; Bejan, M.; Louis, L. K.; Cornejo, P.; Connell, W.; Wong, E. S.; Meuleman, W.; Pinello, L.
Score: 63.5, Published: 2024-02-01 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "DNA-Diffusion: Leveraging Generative Models for Controlling Chromatin Accessibility and Gene Expression via Synthetic Regulatory Elements\nAuthors: Ferreira DaSilva, L.; Senan, S.; Patel, Z. M.; Reddy, A. J.; Gabbita, S.; Nussbaum, Z.; Cordova, C. M. V.; Wenteler, A.; Weber, N.; Tunjic, T. M.; Khan, T. A.; Li, Z.; Smith, C. R.; Bejan, M.; Louis, L. K.; Cornejo, P.; Connell, W.; Wong, E. S.; Meuleman, W.; Pinello, L.\nScore: 63.5, Published: 2024-02-01 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " DNA-Diffusion: Leveraging Generative Models for Controlling Chromatin Accessibility and Gene Expression via Synthetic Regulatory Elements\nAuthors: Ferreira DaSilva, L.; Senan, S.; Patel, Z. M.; Reddy, A. J.; Gabbita, S.; Nussbaum, Z.; Cordova, C. M. V.; Wenteler, A.; Weber, N.; Tunjic, T. M.; Khan, T. A.; Li, Z.; Smith, C. R.; Bejan, M.; Louis, L. K.; Cornejo, P.; Connell, W.; Wong, E. S.; Meuleman, W.; Pinello, L.\nScore: 63.5, Published: 2024-02-01 DOI: 10.1101/2024.02.01.578352\nThe challenge of systematically modifying and optimizing regulatory elements for precise gene expression control is central to modern genomics and synthetic biology. Advancements in generative AI have paved the way for designing synthetic sequences with the aim of safely and accurately modulating gene expression. We leverage diffusion models to design context-specific DNA regulatory sequences, which hold significant potential toward enabling novel therapeutic applications requiring precise modulation of gene expression. Our framework uses a cell type-specific diffusion model to generate synthetic 200 bp regulatory elements based on chromatin accessibility across different cell types. We evaluate the generated sequences based on key metrics to ensure they retain properties of endogenous sequences: transcription factor binding site composition, potential for cell type-specific chromatin accessibility, and capacity for sequences generated by DNA diffusion to activate gene expression in different cell contexts using state-of-the-art prediction models. Our results demonstrate the ability to robustly generate DNA sequences with cell type-specific regulatory potential. DNA-Diffusion paves the way for revolutionizing a regulatory modulation approach to mammalian synthetic biology and precision gene therapy.\nDNA representations and generalization performance of sequence-to-expression models\nAuthors: Shen, Y.; Kudla, G.; Oyarzun, D. A.\nScore: 9.2, Published: 2024-02-07 DOI: 10.1101/2024.02.06.579067\nThe increasing demand for biological products drives many efforts to engineer cells that produce heterologous proteins at maximal yield. Recent advances in massively parallel reporter assays can deliver data suitable for training machine learning models and sup-port the design of microbial strains with optimized protein expression phenotypes. The best performing sequence- to-expression models have been trained on one-hot encodings, a mechanism-agnostic representation of nucleotide sequences. Despite their excellent local pre-dictive power, however, such models suffer from a limited ability to generalize predictions far away from the training data. Here, we show that libraries of genetic constructs can have substantially different cluster structure depending on the chosen sequence representation, and demonstrate that such differences can be leveraged to improve generalization perfor-mance. Using a large sequence- to-expression dataset from Escherichia coli, we show that non-deep regressors and convolutional neural networks trained on one-hot encodings fail to generalize predictions, and that learned representations using state-of-the-art large language models also struggle with out-of-domain accuracy. In contrast, we show that despite their poorer local performance, mechanistic sequence features such as codon bias, nucleotide con-tent or mRNA stability, provide promising gains on model generalization. We explore several strategies to integrate different feature sets into a single predictive model, including feature stacking, ensemble model stacking, and geometric stacking, a novel architecture based on graph convolutional neural networks. Our work suggests that integration of domain-agnostic and domain-aware sequence features offers an unexplored route for improving the quality of sequence- to-expression models and facilitate their adoption in the biotechnology and phar-maceutical sectors.\nEnhanced Sequence-Activity Mapping and Evolution of Artificial Metalloenzymes by Active Learning\nAuthors: Vornholt, T.; Mutny, M.; Schmidt, G. W.; Schellhaas, C.; Tachibana, R.; Panke, S.; Ward, T. R.; Krause, A.; Jeschek, M.\nScore: 5.0, Published: 2024-02-06 DOI: 10.1101/2024.02.06.579157\nTailored enzymes hold great potential to accelerate the transition to a sustainable bioeconomy. Yet, enzyme engineering remains challenging as it relies largely on serendipity and is, therefore, highly laborious and prone to failure. The efficiency and success rates of engineering campaigns may be improved substantially by applying machine learning to construct a comprehensive representation of the sequence-activity landscape from small sets of experimental data. However, it often proves challenging to reliably model a large protein sequence space while keeping the experimental effort tractable. To address this challenge, we present an integrated pipeline combining large-scale screening with active machine learning and model-guided library design. We applied this strategy to efficiently engineer an artificial metalloenzyme (ArM) catalysing a new-to-nature hydroamination reaction. By combining lab automation and next-generation sequencing, we acquired sequence-activity data for several thousand ArM variants. We then used Gaussian process regression to model the activity landscape and guide further screening rounds according to user-defined objectives. Crucial characteristics of our enhanced enzyme engineering pipeline include i) the cost-effective generation of information-rich experimental data sets, ii) the integration of an explorative round to improve the performance of the model, as well as iii) the consideration of experimental noise during modelling. Our approach led to an order-of-magnitude boost in the hit rate of screening while making efficient use of experimental resources. Smart search strategies like this should find broad utility in enzyme engineering and accelerate the development of novel biocatalysts.\nUltra-high efficiency T cell reprogramming at multiple loci with SEED-Selection\nAuthors: Chang, C. R.; Vykunta, V. S.; Goodman, D. B.; Muldoon, J. J.; Nyberg, W. A.; Liu, C.; Allain, V.; Rothrock, A.; Wang, C. H.; Marson, A.; Shy, B. R.; Eyquem, J.\nScore: 3.6, Published: 2024-02-07 DOI: 10.1101/2024.02.06.576175\nMultiplexed reprogramming of T cell specificity and function can generate powerful next-generation cellular therapies. However, current manufacturing methods produce heterogenous mixtures of partially engineered cells. Here, we develop a one-step process to enrich for unlabeled cells with knock-ins at multiple target loci using a family of repair templates named Synthetic Exon/Expression Disruptors (SEEDs). SEED engineering associates transgene integration with the disruption of a paired endogenous surface protein, allowing non-modified and partially edited cells to be immunomagnetically depleted (SEED-Selection). We design SEEDs to fully reprogram three critical loci encoding T cell specificity, co-receptor expression, and MHC expression, with up to 98% purity after selection for individual modifications and up to 90% purity for six simultaneous edits (three knock-ins and three knockouts). These methods are simple, compatible with existing clinical manufacturing workflows, and can be readily adapted to other loci to facilitate production of complex gene-edited cell therapies.\nFunctional Modification of Cyanobacterial Phycobiliprotein and Phycobilisomes through Bilin Metabolism Control\nAuthors: Sato, M.; Kawaguchi, T.; Maeda, K.; Watanabe, M.; Ikeuchi, M.; Narikawa, R.; Watanabe, S.\nScore: 3.2, Published: 2024-02-04 DOI: 10.1101/2024.02.04.578791\nThe light-harvesting antenna complexes in cyanobacteria are known as phycobilisomes (PBSs), and they can adapt to a diverse range of light environments owing to the deployment of different phycobiliproteins within the PBS structures. Freshwater cyanobacteria such as Synechococcus elongatus PCC 7942 thrive under red light due to phycocyanin (PC), along with phycocyanobilin (PCB), in PBS. Conversely, cyanobacteria in shorter-wavelength light environments, such as green light, employ phycoerythrin paired with phycoerythrobilin (PEB) alongside PC in PBS. Synthetic biology studies have shown that PEB production can be enhanced by additional expression of the heterologous PEB synthases, namely 15,16-dihydrobiliverdin:ferredoxin oxidoreductase and PEB:ferredoxin oxidoreductase (PebAB), leading to cellular browning due to PEB accumulation. This approach is genetically unstable, and the properties of the resulting PEB-bound PBS complexes remain uncharacterized. Herein, we engineered a novel strain of Synechococcus 7942 PEB1, with finely tuned control of PEB metabolism. PEB1 exhibited a remarkable and reversible color shift from green to brown and pink, based on the PebAB induction levels. High induction led to a complete PCB-to-PEB substitution, causing the disassembly of the PBS rod complex. In contrast, low PebAB levels resulted in the formation of a stable chimeric PBS complex with partial PCB-to-PEB substitution. This adaptation enabled efficient light harvesting in the green spectrum and energy transfer to the photosynthetic reaction center. These findings, which improve our understanding of PBS and highlight the structural importance of the bilin composition, lay the foundation for future PBS adaptation studies in bioengineering, synthetic biology, and renewable energy.\nUltra-sensitive water contaminant detection with transcription factor interfaced microcantilevers\nAuthors: Agarwal, D. K.; Lucci, T. J.; Jung, J. K.; Shekhawat, G. S.; Lucks, J. B.; Dravid, V. P. B.\nScore: 8.7, Published: 2024-02-01 DOI: 10.1101/2024.02.01.578376\nWater contamination is a growing global concern, creating a need to develop technologies that can detect a range of target compounds at the required thresholds. Here, we address this need by merging biological allosteric transcription factors with DNA coated nanomechanical microcantilevers to detect chemicals in water with digital readout. After proof-of-concept demonstration and optimization to detect tetracycline with the TetR transcription factor, we use the CadC transcription factor to detect Pb2+ and Cd2+ in water at concentrations down to 2 ppb and 1 ppb, respectively, in less than fifteen minutes. A computational model suggests this improvement in sensitivity could be achieved by the DNA coated microcantilever surface changing transcription factor binding properties. Our findings demonstrate a promising new approach for water quality monitoring with fast, highly sensitive, digital readouts.\nCRISPR-Enabled Autonomous Transposable Element (CREATE) for RNA-based gene editing and delivery\nAuthors: Wang, Y.; Lin, R.-Z.; Harris, M.; Lavayen, B.; Mccreedy, B.; Getts, D.\nScore: 11.0, Published: 2024-01-30 DOI: 10.1101/2024.01.29.577809\nTo realize the potential of genome editing for broad therapeutic applications, new tools are needed for delivery of multi-kilobase (kb) payloads at desired genomic sites with high precision. Here we introduce the CRISPR-Enabled Autonomous Transposable Element (CREATE), an RNA-based genome editing system that merges CRISPR/Cas9 with the human L1 retrotransposon to insert gene-sized payloads without DNA donors or double strand breaks. CREATE enables L1-mediated reverse transcription and integration of an RNA-encoded payload gene specifically at two Cas9-induced nick sites. CREATE is delivered using mRNA components and achieves integration of \u003e1 kb payload in mammalian cells, opening the door to mRNA mediated therapeutic genome editing in vivo. One-Sentence SummaryAn mRNA-based, large-scale genome editing technology integrating CRISPR/Cas9 and L1 retroelement.\nSelf-assembled cell-scale containers made from DNA origami membranes\nAuthors: Karfusehr, C.; Eder, M.; Simmel, F. C.\nScore: 2.4, Published: 2024-02-09 DOI: 10.1101/2024.02.09.579479\nDNA origami provides a methodology for the sequence-programmable generation of precisely defined molecular nanostructures with sizes of order 100 nm. A new frontier for the field is the generation of superstructures made from DNA origami subunits, which requires other self-assembly strategies than those used for DNA origami itself. Challenges faced by current approaches include the increasing complexity, cost and development time for the structures and off-target assembly. Here, we demonstrate how radially symmetric origami subunits that are inspired by the structure and interactions of lipids organize into giant DNA origami monolayer membranes that can be readily programmed to form vesicles or hollow tubes with diameters ranging from 100 nm to over 1 um. DNA origami membranes are an unprecedented approach for compartmentalization that opens up new possibilities for bottom-up biology and cell-scale soft robotics.\nModel-guided metabolic engineering of curcuminoid Production in Pseudomonas putida\nAuthors: Martin-Pascual, M.; Moreno-Paz, S.; Van Rosmalen, R. P.; Dorigo, J.; Demaria, F.; Van Kranenburg, R.; Martins dos Santos, V.; Suarez-Diez, M. S.\nScore: 1.5, Published: 2024-02-08 DOI: 10.1101/2024.02.08.579459\nProduction of value-added, plant-derived compounds in microbes increasingly attracts commercially interest in food and pharmaceutical industries. However, plant metabolic pathways are complex, require a robust balance of enzymes, cofactors, ATP and other metabolites, and often result in low production when transplanted to bacteria. This is exemplified by the biosynthesis of curcuminoids from the Curcuma longa plant. Here, we combine dynamic pathway modeling, systematic testing of isoenzymes, and the optimization of gene expression levels and substrate concentrations for the biosynthesis of curcuminoids in Pseudomonas putida, leading to unprecedented conversion rates of caffeic acid and tyrosine to curcumin. The development of kinetic ensemble models guided the design of production strains, emphasizing the necessity of high relative expression of c3h, curs2 and dcs and, the low relative expression of tal, comt, ccoaomt, and 4cl4. This optimization resulted in a strain that achieved a 10.8 {+/-} 1.8% of the maximum theoretical yield of curcumin from tyrosine. This represents a 4.1-fold increase in production efficiency and the highest yield reported to date, demonstrating the potential of P. putida as a promising platform for curcuminoid production. Our findings highlight the effectiveness of our strategy not only in the advances in the production of curcuminoids but also in setting a framework for the biosynthesis of other complex compounds.\nEngineering an anti-CD206-synNotch receptor: insightsinto the development of novel synthetic receptors\nAuthors: Semeniuk, S.; Qian, B.; Cachat, E.\nScore: 1.5, Published: 2024-02-07 DOI: 10.1101/2024.02.07.579082\nImmune cells play a pivotal role in the establishment, growth and progression of tumors at primary and metastatic sites. Macrophages, in particular, play a critical role in suppressing immune responses and promoting an anti-inflammatory environment through both direct and indirect cell-cell interactions. However, our understanding of the mechanisms underlying such interactions is limited due to a lack of reliable tools for studying transient interactions between cancer cells and macrophages within the tumor microenvironment. Recent advances in mammalian synthetic biology have introduced a wide range of synthetic receptors that have been used in diverse biosensing applications. One such synthetic receptor is the synNotch receptor, which can be tailored to sense specific ligands displayed on the surface of target cells. With this study, we aimed at developing a novel CD206-synNotch receptor, targeting CD206+ macrophages, a population of macrophages that play a crucial role in promoting metastatic seeding and persistent growth. Engineered in cancer cells and used in mouse metastasis models, such tool could help monitor and understand the effects cell-cell interactions between macrophages and cancer cells have on metastasis establishment. Here, we report the development of cancer landing pad cells for versatile applications, the engineering of CD206-synNotch cells, report the measurements of their activity and specificity, and discuss the unexpected caveats when considering their in vivo applications.\n",
  "wordCount" : "2224",
  "inLanguage": "en",
  "datePublished": "2024-02-11T10:38:55Z",
  "dateModified": "2024-02-11T10:38:55Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 11, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578352">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578352" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578352">
        <p class="paperTitle">DNA-Diffusion: Leveraging Generative Models for Controlling Chromatin Accessibility and Gene Expression via Synthetic Regulatory Elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578352" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578352" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ferreira DaSilva, L.; Senan, S.; Patel, Z. M.; Reddy, A. J.; Gabbita, S.; Nussbaum, Z.; Cordova, C. M. V.; Wenteler, A.; Weber, N.; Tunjic, T. M.; Khan, T. A.; Li, Z.; Smith, C. R.; Bejan, M.; Louis, L. K.; Cornejo, P.; Connell, W.; Wong, E. S.; Meuleman, W.; Pinello, L.</p>
        <p class="info">Score: 63.5, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578352' target='https://doi.org/10.1101/2024.02.01.578352'> 10.1101/2024.02.01.578352</a></p>
        <p class="abstract">The challenge of systematically modifying and optimizing regulatory elements for precise gene expression control is central to modern genomics and synthetic biology. Advancements in generative AI have paved the way for designing synthetic sequences with the aim of safely and accurately modulating gene expression. We leverage diffusion models to design context-specific DNA regulatory sequences, which hold significant potential toward enabling novel therapeutic applications requiring precise modulation of gene expression. Our framework uses a cell type-specific diffusion model to generate synthetic 200 bp regulatory elements based on chromatin accessibility across different cell types. We evaluate the generated sequences based on key metrics to ensure they retain properties of endogenous sequences: transcription factor binding site composition, potential for cell type-specific chromatin accessibility, and capacity for sequences generated by DNA diffusion to activate gene expression in different cell contexts using state-of-the-art prediction models. Our results demonstrate the ability to robustly generate DNA sequences with cell type-specific regulatory potential. DNA-Diffusion paves the way for revolutionizing a regulatory modulation approach to mammalian synthetic biology and precision gene therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579067">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579067" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579067">
        <p class="paperTitle">DNA representations and generalization performance of sequence-to-expression models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579067" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579067" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shen, Y.; Kudla, G.; Oyarzun, D. A.</p>
        <p class="info">Score: 9.2, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579067' target='https://doi.org/10.1101/2024.02.06.579067'> 10.1101/2024.02.06.579067</a></p>
        <p class="abstract">The increasing demand for biological products drives many efforts to engineer cells that produce heterologous proteins at maximal yield. Recent advances in massively parallel reporter assays can deliver data suitable for training machine learning models and sup-port the design of microbial strains with optimized protein expression phenotypes. The best performing sequence- to-expression models have been trained on one-hot encodings, a mechanism-agnostic representation of nucleotide sequences. Despite their excellent local pre-dictive power, however, such models suffer from a limited ability to generalize predictions far away from the training data. Here, we show that libraries of genetic constructs can have substantially different cluster structure depending on the chosen sequence representation, and demonstrate that such differences can be leveraged to improve generalization perfor-mance. Using a large sequence- to-expression dataset from Escherichia coli, we show that non-deep regressors and convolutional neural networks trained on one-hot encodings fail to generalize predictions, and that learned representations using state-of-the-art large language models also struggle with out-of-domain accuracy. In contrast, we show that despite their poorer local performance, mechanistic sequence features such as codon bias, nucleotide con-tent or mRNA stability, provide promising gains on model generalization. We explore several strategies to integrate different feature sets into a single predictive model, including feature stacking, ensemble model stacking, and geometric stacking, a novel architecture based on graph convolutional neural networks. Our work suggests that integration of domain-agnostic and domain-aware sequence features offers an unexplored route for improving the quality of sequence- to-expression models and facilitate their adoption in the biotechnology and phar-maceutical sectors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579157">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579157" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579157">
        <p class="paperTitle">Enhanced Sequence-Activity Mapping and Evolution of Artificial Metalloenzymes by Active Learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579157" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579157" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vornholt, T.; Mutny, M.; Schmidt, G. W.; Schellhaas, C.; Tachibana, R.; Panke, S.; Ward, T. R.; Krause, A.; Jeschek, M.</p>
        <p class="info">Score: 5.0, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579157' target='https://doi.org/10.1101/2024.02.06.579157'> 10.1101/2024.02.06.579157</a></p>
        <p class="abstract">Tailored enzymes hold great potential to accelerate the transition to a sustainable bioeconomy. Yet, enzyme engineering remains challenging as it relies largely on serendipity and is, therefore, highly laborious and prone to failure. The efficiency and success rates of engineering campaigns may be improved substantially by applying machine learning to construct a comprehensive representation of the sequence-activity landscape from small sets of experimental data. However, it often proves challenging to reliably model a large protein sequence space while keeping the experimental effort tractable. To address this challenge, we present an integrated pipeline combining large-scale screening with active machine learning and model-guided library design. We applied this strategy to efficiently engineer an artificial metalloenzyme (ArM) catalysing a new-to-nature hydroamination reaction. By combining lab automation and next-generation sequencing, we acquired sequence-activity data for several thousand ArM variants. We then used Gaussian process regression to model the activity landscape and guide further screening rounds according to user-defined objectives. Crucial characteristics of our enhanced enzyme engineering pipeline include i) the cost-effective generation of information-rich experimental data sets, ii) the integration of an explorative round to improve the performance of the model, as well as iii) the consideration of experimental noise during modelling. Our approach led to an order-of-magnitude boost in the hit rate of screening while making efficient use of experimental resources. Smart search strategies like this should find broad utility in enzyme engineering and accelerate the development of novel biocatalysts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.576175">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.576175" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.576175">
        <p class="paperTitle">Ultra-high efficiency T cell reprogramming at multiple loci with SEED-Selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.576175" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.576175" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chang, C. R.; Vykunta, V. S.; Goodman, D. B.; Muldoon, J. J.; Nyberg, W. A.; Liu, C.; Allain, V.; Rothrock, A.; Wang, C. H.; Marson, A.; Shy, B. R.; Eyquem, J.</p>
        <p class="info">Score: 3.6, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.576175' target='https://doi.org/10.1101/2024.02.06.576175'> 10.1101/2024.02.06.576175</a></p>
        <p class="abstract">Multiplexed reprogramming of T cell specificity and function can generate powerful next-generation cellular therapies. However, current manufacturing methods produce heterogenous mixtures of partially engineered cells. Here, we develop a one-step process to enrich for unlabeled cells with knock-ins at multiple target loci using a family of repair templates named Synthetic Exon/Expression Disruptors (SEEDs). SEED engineering associates transgene integration with the disruption of a paired endogenous surface protein, allowing non-modified and partially edited cells to be immunomagnetically depleted (SEED-Selection). We design SEEDs to fully reprogram three critical loci encoding T cell specificity, co-receptor expression, and MHC expression, with up to 98% purity after selection for individual modifications and up to 90% purity for six simultaneous edits (three knock-ins and three knockouts). These methods are simple, compatible with existing clinical manufacturing workflows, and can be readily adapted to other loci to facilitate production of complex gene-edited cell therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.04.578791">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.04.578791" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.04.578791">
        <p class="paperTitle">Functional Modification of Cyanobacterial Phycobiliprotein and Phycobilisomes through Bilin Metabolism Control</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.04.578791" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.04.578791" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sato, M.; Kawaguchi, T.; Maeda, K.; Watanabe, M.; Ikeuchi, M.; Narikawa, R.; Watanabe, S.</p>
        <p class="info">Score: 3.2, Published: 2024-02-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.04.578791' target='https://doi.org/10.1101/2024.02.04.578791'> 10.1101/2024.02.04.578791</a></p>
        <p class="abstract">The light-harvesting antenna complexes in cyanobacteria are known as phycobilisomes (PBSs), and they can adapt to a diverse range of light environments owing to the deployment of different phycobiliproteins within the PBS structures. Freshwater cyanobacteria such as Synechococcus elongatus PCC 7942 thrive under red light due to phycocyanin (PC), along with phycocyanobilin (PCB), in PBS. Conversely, cyanobacteria in shorter-wavelength light environments, such as green light, employ phycoerythrin paired with phycoerythrobilin (PEB) alongside PC in PBS. Synthetic biology studies have shown that PEB production can be enhanced by additional expression of the heterologous PEB synthases, namely 15,16-dihydrobiliverdin:ferredoxin oxidoreductase and PEB:ferredoxin oxidoreductase (PebAB), leading to cellular browning due to PEB accumulation. This approach is genetically unstable, and the properties of the resulting PEB-bound PBS complexes remain uncharacterized. Herein, we engineered a novel strain of Synechococcus 7942 PEB1, with finely tuned control of PEB metabolism. PEB1 exhibited a remarkable and reversible color shift from green to brown and pink, based on the PebAB induction levels. High induction led to a complete PCB-to-PEB substitution, causing the disassembly of the PBS rod complex. In contrast, low PebAB levels resulted in the formation of a stable chimeric PBS complex with partial PCB-to-PEB substitution. This adaptation enabled efficient light harvesting in the green spectrum and energy transfer to the photosynthetic reaction center. These findings, which improve our understanding of PBS and highlight the structural importance of the bilin composition, lay the foundation for future PBS adaptation studies in bioengineering, synthetic biology, and renewable energy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578376">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578376" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578376">
        <p class="paperTitle">Ultra-sensitive water contaminant detection with transcription factor interfaced microcantilevers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578376" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578376" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Agarwal, D. K.; Lucci, T. J.; Jung, J. K.; Shekhawat, G. S.; Lucks, J. B.; Dravid, V. P. B.</p>
        <p class="info">Score: 8.7, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578376' target='https://doi.org/10.1101/2024.02.01.578376'> 10.1101/2024.02.01.578376</a></p>
        <p class="abstract">Water contamination is a growing global concern, creating a need to develop technologies that can detect a range of target compounds at the required thresholds. Here, we address this need by merging biological allosteric transcription factors with DNA coated nanomechanical microcantilevers to detect chemicals in water with digital readout. After proof-of-concept demonstration and optimization to detect tetracycline with the TetR transcription factor, we use the CadC transcription factor to detect Pb2&#43; and Cd2&#43; in water at concentrations down to 2 ppb and 1 ppb, respectively, in less than fifteen minutes. A computational model suggests this improvement in sensitivity could be achieved by the DNA coated microcantilever surface changing transcription factor binding properties. Our findings demonstrate a promising new approach for water quality monitoring with fast, highly sensitive, digital readouts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577809">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577809" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577809">
        <p class="paperTitle">CRISPR-Enabled Autonomous Transposable Element (CREATE) for RNA-based gene editing and delivery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577809" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577809" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Y.; Lin, R.-Z.; Harris, M.; Lavayen, B.; Mccreedy, B.; Getts, D.</p>
        <p class="info">Score: 11.0, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577809' target='https://doi.org/10.1101/2024.01.29.577809'> 10.1101/2024.01.29.577809</a></p>
        <p class="abstract">To realize the potential of genome editing for broad therapeutic applications, new tools are needed for delivery of multi-kilobase (kb) payloads at desired genomic sites with high precision. Here we introduce the CRISPR-Enabled Autonomous Transposable Element (CREATE), an RNA-based genome editing system that merges CRISPR/Cas9 with the human L1 retrotransposon to insert gene-sized payloads without DNA donors or double strand breaks. CREATE enables L1-mediated reverse transcription and integration of an RNA-encoded payload gene specifically at two Cas9-induced nick sites. CREATE is delivered using mRNA components and achieves integration of &gt;1 kb payload in mammalian cells, opening the door to mRNA mediated therapeutic genome editing in vivo.

One-Sentence SummaryAn mRNA-based, large-scale genome editing technology integrating CRISPR/Cas9 and L1 retroelement.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579479">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579479" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579479">
        <p class="paperTitle">Self-assembled cell-scale containers made from DNA origami membranes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579479" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579479" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Karfusehr, C.; Eder, M.; Simmel, F. C.</p>
        <p class="info">Score: 2.4, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579479' target='https://doi.org/10.1101/2024.02.09.579479'> 10.1101/2024.02.09.579479</a></p>
        <p class="abstract">DNA origami provides a methodology for the sequence-programmable generation of precisely defined molecular nanostructures with sizes of order 100 nm. A new frontier for the field is the generation of superstructures made from DNA origami subunits, which requires other self-assembly strategies than those used for DNA origami itself. Challenges faced by current approaches include the increasing complexity, cost and development time for the structures and off-target assembly. Here, we demonstrate how radially symmetric origami subunits that are inspired by the structure and interactions of lipids organize into giant DNA origami monolayer membranes that can be readily programmed to form vesicles or hollow tubes with diameters ranging from 100 nm to over 1 um. DNA origami membranes are an unprecedented approach for compartmentalization that opens up new possibilities for bottom-up biology and cell-scale soft robotics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.08.579459">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.08.579459" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.08.579459">
        <p class="paperTitle">Model-guided metabolic engineering of curcuminoid Production in Pseudomonas putida</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.08.579459" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.08.579459" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martin-Pascual, M.; Moreno-Paz, S.; Van Rosmalen, R. P.; Dorigo, J.; Demaria, F.; Van Kranenburg, R.; Martins dos Santos, V.; Suarez-Diez, M. S.</p>
        <p class="info">Score: 1.5, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.08.579459' target='https://doi.org/10.1101/2024.02.08.579459'> 10.1101/2024.02.08.579459</a></p>
        <p class="abstract">Production of value-added, plant-derived compounds in microbes increasingly attracts commercially interest in food and pharmaceutical industries. However, plant metabolic pathways are complex, require a robust balance of enzymes, cofactors, ATP and other metabolites, and often result in low production when transplanted to bacteria. This is exemplified by the biosynthesis of curcuminoids from the Curcuma longa plant. Here, we combine dynamic pathway modeling, systematic testing of isoenzymes, and the optimization of gene expression levels and substrate concentrations for the biosynthesis of curcuminoids in Pseudomonas putida, leading to unprecedented conversion rates of caffeic acid and tyrosine to curcumin. The development of kinetic ensemble models guided the design of production strains, emphasizing the necessity of high relative expression of c3h, curs2 and dcs and, the low relative expression of tal, comt, ccoaomt, and 4cl4. This optimization resulted in a strain that achieved a 10.8 {&#43;/-} 1.8% of the maximum theoretical yield of curcumin from tyrosine. This represents a 4.1-fold increase in production efficiency and the highest yield reported to date, demonstrating the potential of P. putida as a promising platform for curcuminoid production. Our findings highlight the effectiveness of our strategy not only in the advances in the production of curcuminoids but also in setting a framework for the biosynthesis of other complex compounds.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.579082">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.579082" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.579082">
        <p class="paperTitle">Engineering an anti-CD206-synNotch receptor: insightsinto the development of novel synthetic receptors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.579082" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.579082" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Semeniuk, S.; Qian, B.; Cachat, E.</p>
        <p class="info">Score: 1.5, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.579082' target='https://doi.org/10.1101/2024.02.07.579082'> 10.1101/2024.02.07.579082</a></p>
        <p class="abstract">Immune cells play a pivotal role in the establishment, growth and progression of tumors at primary and metastatic sites. Macrophages, in particular, play a critical role in suppressing immune responses and promoting an anti-inflammatory environment through both direct and indirect cell-cell interactions. However, our understanding of the mechanisms underlying such interactions is limited due to a lack of reliable tools for studying transient interactions between cancer cells and macrophages within the tumor microenvironment. Recent advances in mammalian synthetic biology have introduced a wide range of synthetic receptors that have been used in diverse biosensing applications. One such synthetic receptor is the synNotch receptor, which can be tailored to sense specific ligands displayed on the surface of target cells. With this study, we aimed at developing a novel CD206-synNotch receptor, targeting CD206&#43; macrophages, a population of macrophages that play a crucial role in promoting metastatic seeding and persistent growth. Engineered in cancer cells and used in mouse metastasis models, such tool could help monitor and understand the effects cell-cell interactions between macrophages and cancer cells have on metastasis establishment. Here, we report the development of cancer landing pad cells for versatile applications, the engineering of CD206-synNotch cells, report the measurements of their activity and specificity, and discuss the unexpected caveats when considering their in vivo applications.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
