<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Machine-guided design of synthetic cell type-specific cis-regulatory elements
Authors: Gosai, S. J.; Castro, R. I.; Fuentes, N.; Butts, J. C.; Kales, S.; Noche, R. R.; Mouri, K.; Sabeti, P. C.; Reilly, S. K.; Tewhey, R.
Score: 55.9, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552077
Cis-regulatory elements (CREs) control gene expression, orchestrating tissue identity, developmental timing, and stimulus responses, which collectively define the thousands of unique cell types in the body. While there is great potential for strategically incorporating CREs in therapeutic or biotechnology applications that require tissue specificity, there is no guarantee that an optimal CRE for an intended purpose has arisen naturally through evolution.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Machine-guided design of synthetic cell type-specific cis-regulatory elements
Authors: Gosai, S. J.; Castro, R. I.; Fuentes, N.; Butts, J. C.; Kales, S.; Noche, R. R.; Mouri, K.; Sabeti, P. C.; Reilly, S. K.; Tewhey, R.
Score: 55.9, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552077
Cis-regulatory elements (CREs) control gene expression, orchestrating tissue identity, developmental timing, and stimulus responses, which collectively define the thousands of unique cell types in the body. While there is great potential for strategically incorporating CREs in therapeutic or biotechnology applications that require tissue specificity, there is no guarantee that an optimal CRE for an intended purpose has arisen naturally through evolution." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-13T10:36:01+00:00" />
<meta property="article:modified_time" content="2023-08-13T10:36:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Machine-guided design of synthetic cell type-specific cis-regulatory elements
Authors: Gosai, S. J.; Castro, R. I.; Fuentes, N.; Butts, J. C.; Kales, S.; Noche, R. R.; Mouri, K.; Sabeti, P. C.; Reilly, S. K.; Tewhey, R.
Score: 55.9, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552077
Cis-regulatory elements (CREs) control gene expression, orchestrating tissue identity, developmental timing, and stimulus responses, which collectively define the thousands of unique cell types in the body. While there is great potential for strategically incorporating CREs in therapeutic or biotechnology applications that require tissue specificity, there is no guarantee that an optimal CRE for an intended purpose has arisen naturally through evolution."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Machine-guided design of synthetic cell type-specific cis-regulatory elements\nAuthors: Gosai, S. J.; Castro, R. I.; Fuentes, N.; Butts, J. C.; Kales, S.; Noche, R. R.; Mouri, K.; Sabeti, P. C.; Reilly, S. K.; Tewhey, R.\nScore: 55.9, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552077\nCis-regulatory elements (CREs) control gene expression, orchestrating tissue identity, developmental timing, and stimulus responses, which collectively define the thousands of unique cell types in the body. While there is great potential for strategically incorporating CREs in therapeutic or biotechnology applications that require tissue specificity, there is no guarantee that an optimal CRE for an intended purpose has arisen naturally through evolution.",
  "keywords": [
    
  ],
  "articleBody": " Machine-guided design of synthetic cell type-specific cis-regulatory elements\nAuthors: Gosai, S. J.; Castro, R. I.; Fuentes, N.; Butts, J. C.; Kales, S.; Noche, R. R.; Mouri, K.; Sabeti, P. C.; Reilly, S. K.; Tewhey, R.\nScore: 55.9, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552077\nCis-regulatory elements (CREs) control gene expression, orchestrating tissue identity, developmental timing, and stimulus responses, which collectively define the thousands of unique cell types in the body. While there is great potential for strategically incorporating CREs in therapeutic or biotechnology applications that require tissue specificity, there is no guarantee that an optimal CRE for an intended purpose has arisen naturally through evolution. Here, we present a platform to engineer and validate synthetic CREs capable of driving gene expression with programmed cell type specificity. We leverage innovations in deep neural network modeling of CRE activity across three cell types, efficient in silico optimization, and massively parallel reporter assays (MPRAs) to design and empirically test thousands of CREs. Through in vitro and in vivo validation, we show that synthetic sequences outperform natural sequences from the human genome in driving cell type-specific expression. Synthetic sequences leverage unique sequence syntax to promote activity in the on-target cell type and simultaneously reduce activity in off-target cells. Together, we provide a generalizable framework to prospectively engineer CREs and demonstrate the required literacy to write regulatory code that is fit-for-purpose in vivo across vertebrates.\nEngineering the highly productive cyanobacterium Synechococcus sp. PCC 11901\nAuthors: Victoria, A. J.; Selao, T. T.; Moreno-Cabezuelo, J. A.; Mills, L. A.; Gale, G. A.; Lea-Smith, D. J.; McCormick, A. J.\nScore: 11.5, Published: 2023-08-07 DOI: 10.1101/2023.08.04.552009\nSynechococcus sp. PCC 11901 (PCC 11901) is a fast-growing marine cyanobacterial strain that has a capacity for sustained biomass accumulation to very high cell densities, comparable to that achieved by commercially relevant heterotrophic organisms. However, genetic tools to engineer PCC 11901 for biotechnology applications are limited. Here we describe a suite of tools based on the CyanoGate MoClo system to unlock the engineering potential of PCC 11901. First, we characterised neutral sites suitable for stable genomic integration that do not affect growth even at high cell densities. Second, we tested a suite of constitutive promoters, terminators, and inducible promoters including a 2,4-diacetylphloroglucinol (DAPG)-inducible PhlF repressor system, which has not previously been demonstrated in cyanobacteria and showed tight regulation and a 228-fold dynamic range of induction. Lastly, we developed a DAPG-inducible dCas9-based CRISPR interference (CRISPRi) system and a modular method to generate markerless mutants using CRISPR-Cas12a. Based on our findings, PCC 11901 is highly responsive to CRISPRi-based repression and showed high efficiencies for single insertion (31-81%) and multiplex double insertion (25%) genome editing with Cas12a. We envision that these tools will lay the foundations for the adoption of PCC 11901 as a robust model strain for engineering biology and green biotechnology. On sentence summaryGenetic parts were characterised in Synechococcus sp. PCC 11901, including a tightly regulated inducible promoter system, efficient CRISPRi and a novel markerless Cas12a genome editing approach.\nDevelopment of a Quorum-Sensing Based Bacterial GMO for Targeted Treatment of Bovine Mastitis.\nAuthors: Dutta, A.; Chatterjee, S.; Borkar, S. P.; Das, S.; Ghosh, U.; Adhikary, A.\nScore: 3.4, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552207\nAO_SCPLOWBSTRACTC_SCPLOWBovine mastitis, a prevalent and costly disease affecting dairy cows, severely impacts milk production, quality, and cow health and poses significant challenges to the dairy industry worldwide. Traditional treatment strategies relying on antibiotics face concerns of antibiotic resistance and residual antibiotic residues in dairy products. Therefore, there is an urgent need for innovative and sustainable approaches. Our study explores the efficacy of a GMO-based antibiotic-free treatment in fighting Bovine mastitis. We use the bioengineered defensin Nisin PV as the antimicrobial to specifically target the causative pathogen Staphylococcus aureus. To disrupt the biofilm formation, we have leveraged the potency of DNaseI. By hacking the quorum sensing technique of Staphylococcus aureus, our Genetically modified bacteria sense the presence of pathogens and initiate their eradication by producing Nisin PV and DNaseI. Additionally, a lysis-based kill switch is incorporated to ensure the release of the therapeutic components. The findings of this research have the potential to revolutionize bovine mastitis management, offering a sustainable and effective alternative to antibiotic-based treatments. Implementation of bioengineered defensins could reduce disease incidence, minimize economic losses, and promote responsible antimicrobial stewardship in the dairy industry. This innovative approach contributes to the development of sustainable solutions for safeguarding animal health, enhancing milk quality, and preserving human health.\nMapping immunological and host receptor binding determinants of SARS-CoV spike protein utilizing the Qubevirus platform\nAuthors: Bopda Waffo, A.\nScore: 1.5, Published: 2023-07-28 DOI: 10.1101/2023.07.27.550841\nThe motifs involved in tropism and immunological interactions of SARS-CoV spike (S) protein were investigated utilizing the Qubevirus platform. We showed that separately, 14 overlapping peptide fragments representing the S protein (F1-14 of 100 residues each) could be inserted into the C-terminus of A1 on recombinant Qubevirus without affecting its viability. Additionally, recombinant phage expression resulted in the surface exposure of different engineered fragments in an accessible manner. The F6 from S425-525, was found to contain the binding determinant of the recombinant human angiotensin converting enzyme 2 (rhACE2), with the shortest active binding motif situated between residues S437-492. Upstream, another fragment, F7, containing an overlapping portion of F6 would not bind to rhACE2, confirming not just only that residues were linear but equally also the appropriate structural orientation of F6 upon the Qubevirus. The F6 (S441-460) and other inserts, including F7/F8 (S601-620) and F10 (S781-800), were demonstrated to contain important immunological determinants through recognition and binding of S protein specific (anti-S) antibodies. An engineered chimeric insert bearing the fusion of all three anti-S reactive epitopes, improved substantially the recognition and binding to their cognate antibodies. These results provide insights into humoral immune relevant epitopes and tropism characteristics of the S protein with implications for the development of subunit vaccines or other biologics against SARS-CoV. SignificanceMapping epitopes within the receptor binding domains of viruses which are essential for viral tropism is critical for developing antiviral agents and subunit vaccines. In this study we have engineered the surface of Qubevirus to display a peptide library derived from the SARS-CoV S protein. In biopanning with S protein antibodies, we have identified three peptide fragments (EP1, EP2 and EP3) which reacted selectively with antibodies specific to the S protein. We demonstrated that all recombinant phage displayed peptide fragments both individually and as chimera exposed important immunological epitopes to their cognate antibodies. A peptide fragment F6 situated at S425-525, was found containing the binding determinant of the recombinant human angiotensin converting enzyme 2 (rhACE2), with the shortest active binding motif situated between residues S437-492. The platform is rapidly to identify epitopes and receptor binding sites within viral receptors found in target host cell. Thus, this platform holds great significance.\n",
  "wordCount" : "1120",
  "inLanguage": "en",
  "datePublished": "2023-08-13T10:36:01Z",
  "dateModified": "2023-08-13T10:36:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on August 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552077">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552077" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552077">
        <p class="paperTitle">Machine-guided design of synthetic cell type-specific cis-regulatory elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552077" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552077" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gosai, S. J.; Castro, R. I.; Fuentes, N.; Butts, J. C.; Kales, S.; Noche, R. R.; Mouri, K.; Sabeti, P. C.; Reilly, S. K.; Tewhey, R.</p>
        <p class="info">Score: 55.9, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552077' target='https://doi.org/10.1101/2023.08.08.552077'> 10.1101/2023.08.08.552077</a></p>
        <p class="abstract">Cis-regulatory elements (CREs) control gene expression, orchestrating tissue identity, developmental timing, and stimulus responses, which collectively define the thousands of unique cell types in the body. While there is great potential for strategically incorporating CREs in therapeutic or biotechnology applications that require tissue specificity, there is no guarantee that an optimal CRE for an intended purpose has arisen naturally through evolution. Here, we present a platform to engineer and validate synthetic CREs capable of driving gene expression with programmed cell type specificity. We leverage innovations in deep neural network modeling of CRE activity across three cell types, efficient in silico optimization, and massively parallel reporter assays (MPRAs) to design and empirically test thousands of CREs. Through in vitro and in vivo validation, we show that synthetic sequences outperform natural sequences from the human genome in driving cell type-specific expression. Synthetic sequences leverage unique sequence syntax to promote activity in the on-target cell type and simultaneously reduce activity in off-target cells. Together, we provide a generalizable framework to prospectively engineer CREs and demonstrate the required literacy to write regulatory code that is fit-for-purpose in vivo across vertebrates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552009">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552009" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552009">
        <p class="paperTitle">Engineering the highly productive cyanobacterium Synechococcus sp. PCC 11901</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552009" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552009" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Victoria, A. J.; Selao, T. T.; Moreno-Cabezuelo, J. A.; Mills, L. A.; Gale, G. A.; Lea-Smith, D. J.; McCormick, A. J.</p>
        <p class="info">Score: 11.5, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552009' target='https://doi.org/10.1101/2023.08.04.552009'> 10.1101/2023.08.04.552009</a></p>
        <p class="abstract">Synechococcus sp. PCC 11901 (PCC 11901) is a fast-growing marine cyanobacterial strain that has a capacity for sustained biomass accumulation to very high cell densities, comparable to that achieved by commercially relevant heterotrophic organisms. However, genetic tools to engineer PCC 11901 for biotechnology applications are limited. Here we describe a suite of tools based on the CyanoGate MoClo system to unlock the engineering potential of PCC 11901. First, we characterised neutral sites suitable for stable genomic integration that do not affect growth even at high cell densities. Second, we tested a suite of constitutive promoters, terminators, and inducible promoters including a 2,4-diacetylphloroglucinol (DAPG)-inducible PhlF repressor system, which has not previously been demonstrated in cyanobacteria and showed tight regulation and a 228-fold dynamic range of induction. Lastly, we developed a DAPG-inducible dCas9-based CRISPR interference (CRISPRi) system and a modular method to generate markerless mutants using CRISPR-Cas12a. Based on our findings, PCC 11901 is highly responsive to CRISPRi-based repression and showed high efficiencies for single insertion (31-81%) and multiplex double insertion (25%) genome editing with Cas12a. We envision that these tools will lay the foundations for the adoption of PCC 11901 as a robust model strain for engineering biology and green biotechnology.

On sentence summaryGenetic parts were characterised in Synechococcus sp. PCC 11901, including a tightly regulated inducible promoter system, efficient CRISPRi and a novel markerless Cas12a genome editing approach.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552207">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552207" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552207">
        <p class="paperTitle">Development of a Quorum-Sensing Based Bacterial GMO for Targeted Treatment of Bovine Mastitis.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552207" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552207" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dutta, A.; Chatterjee, S.; Borkar, S. P.; Das, S.; Ghosh, U.; Adhikary, A.</p>
        <p class="info">Score: 3.4, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552207' target='https://doi.org/10.1101/2023.08.07.552207'> 10.1101/2023.08.07.552207</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWBovine mastitis, a prevalent and costly disease affecting dairy cows, severely impacts milk production, quality, and cow health and poses significant challenges to the dairy industry worldwide. Traditional treatment strategies relying on antibiotics face concerns of antibiotic resistance and residual antibiotic residues in dairy products. Therefore, there is an urgent need for innovative and sustainable approaches. Our study explores the efficacy of a GMO-based antibiotic-free treatment in fighting Bovine mastitis. We use the bioengineered defensin Nisin PV as the antimicrobial to specifically target the causative pathogen Staphylococcus aureus. To disrupt the biofilm formation, we have leveraged the potency of DNaseI. By hacking the quorum sensing technique of Staphylococcus aureus, our Genetically modified bacteria sense the presence of pathogens and initiate their eradication by producing Nisin PV and DNaseI. Additionally, a lysis-based kill switch is incorporated to ensure the release of the therapeutic components. The findings of this research have the potential to revolutionize bovine mastitis management, offering a sustainable and effective alternative to antibiotic-based treatments. Implementation of bioengineered defensins could reduce disease incidence, minimize economic losses, and promote responsible antimicrobial stewardship in the dairy industry. This innovative approach contributes to the development of sustainable solutions for safeguarding animal health, enhancing milk quality, and preserving human health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550841">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550841" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550841">
        <p class="paperTitle">Mapping immunological and host receptor binding determinants of SARS-CoV spike protein utilizing the Qubevirus platform</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550841" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550841" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bopda Waffo, A.</p>
        <p class="info">Score: 1.5, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550841' target='https://doi.org/10.1101/2023.07.27.550841'> 10.1101/2023.07.27.550841</a></p>
        <p class="abstract">The motifs involved in tropism and immunological interactions of SARS-CoV spike (S) protein were investigated utilizing the Qubevirus platform. We showed that separately, 14 overlapping peptide fragments representing the S protein (F1-14 of 100 residues each) could be inserted into the C-terminus of A1 on recombinant Qubevirus without affecting its viability. Additionally, recombinant phage expression resulted in the surface exposure of different engineered fragments in an accessible manner. The F6 from S425-525, was found to contain the binding determinant of the recombinant human angiotensin converting enzyme 2 (rhACE2), with the shortest active binding motif situated between residues S437-492. Upstream, another fragment, F7, containing an overlapping portion of F6 would not bind to rhACE2, confirming not just only that residues were linear but equally also the appropriate structural orientation of F6 upon the Qubevirus. The F6 (S441-460) and other inserts, including F7/F8 (S601-620) and F10 (S781-800), were demonstrated to contain important immunological determinants through recognition and binding of S protein specific (anti-S) antibodies. An engineered chimeric insert bearing the fusion of all three anti-S reactive epitopes, improved substantially the recognition and binding to their cognate antibodies. These results provide insights into humoral immune relevant epitopes and tropism characteristics of the S protein with implications for the development of subunit vaccines or other biologics against SARS-CoV.

SignificanceMapping epitopes within the receptor binding domains of viruses which are essential for viral tropism is critical for developing antiviral agents and subunit vaccines. In this study we have engineered the surface of Qubevirus to display a peptide library derived from the SARS-CoV S protein. In biopanning with S protein antibodies, we have identified three peptide fragments (EP1, EP2 and EP3) which reacted selectively with antibodies specific to the S protein. We demonstrated that all recombinant phage displayed peptide fragments both individually and as chimera exposed important immunological epitopes to their cognate antibodies. A peptide fragment F6 situated at S425-525, was found containing the binding determinant of the recombinant human angiotensin converting enzyme 2 (rhACE2), with the shortest active binding motif situated between residues S437-492. The platform is rapidly to identify epitopes and receptor binding sites within viral receptors found in target host cell. Thus, this platform holds great significance.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
