<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein
Authors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Jing, G.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.
Score: 74.0, Published: 2023-07-06 DOI: 10.1101/2023.07.05.547496
Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein
Authors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Jing, G.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.
Score: 74.0, Published: 2023-07-06 DOI: 10.1101/2023.07.05.547496
Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-09T10:40:17+00:00" />
<meta property="article:modified_time" content="2023-07-09T10:40:17+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein
Authors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Jing, G.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.
Score: 74.0, Published: 2023-07-06 DOI: 10.1101/2023.07.05.547496
Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein\nAuthors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Jing, G.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.\nScore: 74.0, Published: 2023-07-06 DOI: 10.1101/2023.07.05.547496\nProtein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently.",
  "keywords": [
    
  ],
  "articleBody": " xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein\nAuthors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Jing, G.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.\nScore: 74.0, Published: 2023-07-06 DOI: 10.1101/2023.07.05.547496\nProtein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently. This paper proposes a unified protein language model, xTrimoPGLM, to address these two types of tasks simultaneously through an innovative pre-training framework. Our key technical contribution is an exploration of the compatibility and the potential for joint optimization of the two types of objectives, which has led to a strategy for training xTrimoPGLM at an unprecedented scale of 100 billion parameters and 1 trillion training tokens. Our extensive experiments reveal that xTrimoPGLM significantly outperforms other advanced baselines in diverse protein understanding tasks (13 out of 15 tasks across four categories) and generates novel protein sequences which are structurally similar to natural ones. Furthermore, using the same xTrimoPGLM framework, we train an antibody-specific model (xTrimoPGLM-Ab) using 1 billion parameters. This model set a new record in predicting antibody naturalness and structures, both essential to the field of antibody-based drug design, and demonstrated a significantly faster inference speed than AlphaFold2. These results highlight the substantial capability and versatility of xTrimoPGLM in understanding and generating protein sequences.\nMapping interactions between disordered regions reveals promiscuity in biomolecular condensate formation\nAuthors: Gilat, A.; Dubrueil, B.; Levy, E. D.\nScore: 13.9, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547715\nIntrinsically-disordered regions (IDRs) promote intracellular phase separation and the formation of biomolecular condensates through interactions encoded in their primary sequence. While these condensates form spatially distinct assemblies in cells, it is unclear whether such specificity can be conferred by IDRs alone. Indeed, IDRs exhibit high conformational flexibility whereas specificity in protein recognition is generally associated with well-defined 3D structures. To characterize IDR-IDR interactions and assess their ability to mediate self-specific partitioning, we developed a synthetic system of Multivalent IDRs forming Constitutive DROPlets (micDROP). We investigated ten natural IDRs that underwent phase separation in micDROP. These IDRs exhibited a wide range of saturation concentrations in vivo, which correlated well with their total sequence stickiness. We then probed IDR-IDR specificity by co-expressing pairs of IDRs fused to homologous scaffolds that did not co-assemble. We observed a high degree of promiscuity, particularly among IDRs from the proteins Ddx4, DYRK3, ER, FUS, hnRNPA1, HspB8, RBM14 and TAF15, whereas TDP43 and UBQ2 formed spatially distinct condensates regardless of their partner. Further investigation revealed the short and conserved -helical segment of TDP43's IDR was governing its specific self-recognition. Our findings imply that IDRs can tune their phase separation propensity through sequence composition, while their formation of discrete condensates likely requires additional cellular or structural determinants.\nTruly tiny acoustic biomolecules for ultrasound imaging and therapy\nAuthors: Ling, B.; Gungoren, B.; Yao, Y.; Dutka, P.; Smith, C. A. B.; Lee, J.; Swift, M. B.; Shapiro, M. G.\nScore: 25.5, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546773\nNanotechnology offers significant advantages for medical imaging and therapy, including enhanced contrast and precision targeting. However, integrating these benefits into ultrasonography has been challenging due to the size and stability constraints of conventional bubble-based agents. Here we describe bicones, truly tiny acoustic contrast agents based on gas vesicles, a unique class of air-filled protein nanostructures naturally produced in buoyant microbes. We show that these sub-80 nm particles can be effectively detected both in vitro and in vivo, infiltrate tumors via leaky vasculature, deliver potent mechanical effects through ultrasound-induced inertial cavitation, and are easily engineered for molecular targeting, prolonged circulation time, and payload conjugation.\npUdOs: concise plasmids for bacterial and mammalian cells\nAuthors: Ourida Manigat, F.; Connell, L. B.; Stewart, B. N.; LePabic, A.-R.; Tessier, C. J. G.; Emlaw, J. R.; Calvert, N. D.; Rossl, A.; Shuhendler, A. J.; daCosta, C. J. B.; Campbell-Valois, F.-X.\nScore: 6.5, Published: 2023-07-06 DOI: 10.1101/2023.07.05.547852\nThe pUdOs are 28 plasmids of small size combining four different origins of replication and seven selection markers, which together afford flexible use in Escherichia coli and several related gram-negative bacteria. The promoterless multicloning site is insulated from upstream spurious promoters by strong transcription terminators, and contains type IIP or IIS restriction sites for conventional or Golden-gate cloning. pUdOs can be converted into efficient expression vectors through the insertion of a promoter at the user's discretion. For example, we demonstrate the utility of pUdOs as the backbone for an improved version of a Type III Secretion System reporter in Shigella. In addition, we derive a series of pUdO-based mammalian expression vectors affording distinct levels of expression and transfection efficiencies comparable to commonly used mammalian expression plasmids. Thus, pUdOs could advantageously replace traditional plasmids in a wide variety of cell types and applications.\nA Platform for Controlled Template-Independent Enzymatic Synthesis of RNA Oligonucleotides and Therapeutics\nAuthors: Wiegand, D. J.; Rittichier, J.; Meyer, E.; Lee, H.; Conway, N. J.; Ahlstedt, D.; Yurtsever, Z.; Rainone, D.; Kuru, E.; Church, G.\nScore: 4.9, Published: 2023-06-29 DOI: 10.1101/2023.06.29.547106\nTherapeutic RNA oligonucleotides have shown tremendous potential to manage and treat disease, yet current manufacturing methods cannot deliver on this promise. Here, we report the development and optimization of a novel, aqueous-based, template-independent enzymatic RNA oligonucleotide synthesis platform as an alternative to traditional chemical methodologies. Our platform is made possible by reversible terminator nucleoside triphosphates and an enzyme capable of their incorporation. We show that many common therapeutic RNA modifications are compatible with our process and demonstrate the enzymatic synthesis of natural and modified oligonucleotides in both liquid and solid phases. Our platform offers many unique advantages over chemical synthesis, including the realization of a more sustainable process to produce therapeutic RNA oligonucleotides. One-Sentence SummaryAn enzyme and novel nucleotide building blocks are used to synthesize RNA oligonucleotides template independently under aqueous conditions.\nCRISPR-mediated germline mutagenesis for genetic sterilization of Anopheles gambiae males.\nAuthors: Smidler, A. L.; Paton, D. G.; Church, G. M.; Shaw, W. R.; Catteruccia, F.\nScore: 9.7, Published: 2023-06-13 DOI: 10.1101/2023.06.13.544841\nRapid spread of insecticide resistance among anopheline mosquitoes threatens malaria elimination efforts, necessitating development of alternative vector control technologies. Sterile Insect Technique (SIT) has been successfully implemented in multiple insect pests to suppress field populations by the release of large numbers of sterile males, yet it has proven difficult to adapt to Anopheles vectors. Here we outline adaptation of a CRISPR-based genetic sterilization system to selectively ablate male sperm cells in the malaria mosquito Anopheles gambiae. We achieve robust mosaic biallelic mutagenesis of zero population growth (zpg, a gene essential for differentiation of germ cells) in F1 individuals after intercrossing a germline-expressing Cas9 transgenic line to a line expressing zpg-targeting gRNAs. Approximately 95% of mutagenized males display complete genetic sterilization, and cause similarly high levels of infertility in their female mates. Using a fluorescence reporter that allows detection of the germline leads to a 100% accurate selection of spermless males, improving the system. These males cause a striking reduction in mosquito population size when released at field-like frequencies in competition cages against wild type males. These findings demonstrate that such a genetic system could be adopted for SIT against important malaria vectors.\nT7 phage-assisted evolution of riboswitches using error-prone replication and dual selection\nAuthors: Goicoechea, E.; Jaramillo, A.\nScore: 19.1, Published: 2023-07-04 DOI: 10.1101/2023.01.11.523572\nLeveraging riboswitches, non-coding mRNA fragments pivotal to gene regulation, poses a challenge in effectively selecting and enriching these functional genetic sensors, which can toggle between ON and OFF states in response to their cognate inducers. Here, we show our engineered phage T7, enabling the evolution of a theophylline riboswitch. We have replaced T7's DNA polymerase with a transcription factor controlled by a theophylline riboswitch and have created two types of host environments to propagate the engineered phage. Both types host an error-prone T7 DNA polymerase regulated by a T7 promoter along with another critical gene - either cmk or pifA, depending on the host type. The cmk gene is necessary for T7 replication and is used in the first host type for selection in the riboswitch's ON state. Conversely, the second host type incorporates the pifA gene, leading to abortive T7 infections and used for selection in the riboswitch's OFF state. This dual-selection system, termed T7AE, was then applied to a library of 65,536 engineered T7 phages, each carrying randomized riboswitch variants. Through successive passage in both host types with and without theophylline, we observed an enrichment of phages encoding functional riboswitches that conferred a fitness advantage to the phage in both hosts. The T7AE technique thereby opens new pathways for the evolution and advancement of gene switches, including non-coding RNA-based switches, setting the stage for significant strides in synthetic biology.\nCRISPR-based gene drives generate super-Mendelian inheritance in the disease vector Culex quinquefasciatus.\nAuthors: Harvey-Samuel, T.; Feng, X.; Okamoto, E. M.; Purusothaman, D.-K.; Leftwich, P. T.; Alphey, L.; Gantz, V. M.\nScore: 25.8, Published: 2023-06-15 DOI: 10.1101/2023.06.12.544656\nCulex mosquitoes pose a significant public health threat as vectors for a variety of diseases including West Nile virus and lymphatic filariasis, and transmit pathogens threatening livestock, companion animals, and endangered birds. Rampant insecticide resistance makes controlling these mosquitoes challenging and necessitates the development of new control strategies. Gene drive technologies have made significant progress in other mosquito species, although similar advances have been lagging in Culex. Here we test the first CRISPR-based homing gene drive for Culex quinquefasciatus, demonstrating the possibility of using this technology to control Culex mosquitoes. Our results show that the inheritance of two split-gene-drive transgenes, targeting different loci, are biased in the presence of a Cas9-expressing transgene although with modest efficiencies. Our findings extend the list of disease vectors where engineered homing gene drives have been demonstrated to include Culex alongside Anopheles and Aedes, and pave the way for future development of these technologies to control Culex mosquitoes.\nExploring alternative quorum sensing model structures and quorum quenching strategies\nAuthors: Cimolato, C.; Bellato, M.; Selvaggio, G.; Marchetti, L.; Giordano, G.; Schenato, L.\nScore: 1.0, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548074\nBacterial quorum sensing (QS) is a cell-to-cell communication mechanism through which bacteria share information about cell density, and tune gene expression accordingly. Pathogens exploit QS to orchestrate virulence and regulate the expression of genes related to antimicrobial resistance. Despite the vast literature on QS, the properties of the underlying molecular network are not entirely clear. We compare two synthetic QS circuit architectures: in the first, a single positive feedback loop autoinduces the synthesis of the signal molecule; the second includes an additional positive feedback loop enhancing the synthesis of the signal molecule receptor. Our comprehensive analysis of the two systems and their equilibria highlights the differences in the bistable and hysteretic behaviors of the alternative QS structures. Finally, we investigate three different QS inhibition approaches; numerical analysis predicts their effect on the steady-state behavior of the two different QS models, revealing critical parameter thresholds that guarantee an effective QS suppression.\nA digital twin for DNA data storage based on comprehensive quantification of errors and biases\nAuthors: Gimpel, A. L.; Stark, W. J.; Heckel, R.; Grass, R. N.\nScore: 1.0, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547683\nArchiving data in synthetic DNA offers unprecedented storage density and longevity. Handling and storage introduce errors and biases into DNA-based storage systems, necessitating the use of Error Correction Coding (ECC) which comes at the cost of added redundancy. However, insufficient data on these errors and biases, as well as a lack of modelling tools, limit data-driven ECC development and experimental design. In this study, we present a comprehensive characterisation of the error sources and biases present in the most common DNA data storage workflows, including commercial DNA synthesis, PCR, decay by accelerated aging, and sequencing-by-synthesis. Using the data from 40 sequencing experiments, we build a digital twin of the DNA data storage process, capable of simulating state-of-the-art workflows and reproducing their experimental results. We showcase the digital twins ability to replace experiments and rationalize the design of redundancy in two case studies, highlighting opportunities for tangible cost savings and data-driven ECC development.\n",
  "wordCount" : "2009",
  "inLanguage": "en",
  "datePublished": "2023-07-09T10:40:17Z",
  "dateModified": "2023-07-09T10:40:17Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on July 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547496">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547496" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547496">
        <p class="paperTitle">xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547496" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547496" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Jing, G.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.</p>
        <p class="info">Score: 74.0, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547496' target='https://doi.org/10.1101/2023.07.05.547496'> 10.1101/2023.07.05.547496</a></p>
        <p class="abstract">Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently. This paper proposes a unified protein language model, xTrimoPGLM, to address these two types of tasks simultaneously through an innovative pre-training framework. Our key technical contribution is an exploration of the compatibility and the potential for joint optimization of the two types of objectives, which has led to a strategy for training xTrimoPGLM at an unprecedented scale of 100 billion parameters and 1 trillion training tokens. Our extensive experiments reveal that xTrimoPGLM significantly outperforms other advanced baselines in diverse protein understanding tasks (13 out of 15 tasks across four categories) and generates novel protein sequences which are structurally similar to natural ones. Furthermore, using the same xTrimoPGLM framework, we train an antibody-specific model (xTrimoPGLM-Ab) using 1 billion parameters. This model set a new record in predicting antibody naturalness and structures, both essential to the field of antibody-based drug design, and demonstrated a significantly faster inference speed than AlphaFold2. These results highlight the substantial capability and versatility of xTrimoPGLM in understanding and generating protein sequences.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547715">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547715" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547715">
        <p class="paperTitle">Mapping interactions between disordered regions reveals promiscuity in biomolecular condensate formation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547715" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547715" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gilat, A.; Dubrueil, B.; Levy, E. D.</p>
        <p class="info">Score: 13.9, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547715' target='https://doi.org/10.1101/2023.07.04.547715'> 10.1101/2023.07.04.547715</a></p>
        <p class="abstract">Intrinsically-disordered regions (IDRs) promote intracellular phase separation and the formation of biomolecular condensates through interactions encoded in their primary sequence. While these condensates form spatially distinct assemblies in cells, it is unclear whether such specificity can be conferred by IDRs alone. Indeed, IDRs exhibit high conformational flexibility whereas specificity in protein recognition is generally associated with well-defined 3D structures. To characterize IDR-IDR interactions and assess their ability to mediate self-specific partitioning, we developed a synthetic system of Multivalent IDRs forming Constitutive DROPlets (micDROP). We investigated ten natural IDRs that underwent phase separation in micDROP. These IDRs exhibited a wide range of saturation concentrations in vivo, which correlated well with their total sequence stickiness. We then probed IDR-IDR specificity by co-expressing pairs of IDRs fused to homologous scaffolds that did not co-assemble. We observed a high degree of promiscuity, particularly among IDRs from the proteins Ddx4, DYRK3, ER, FUS, hnRNPA1, HspB8, RBM14 and TAF15, whereas TDP43 and UBQ2 formed spatially distinct condensates regardless of their partner. Further investigation revealed the short and conserved -helical segment of TDP43&#39;s IDR was governing its specific self-recognition. Our findings imply that IDRs can tune their phase separation propensity through sequence composition, while their formation of discrete condensates likely requires additional cellular or structural determinants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546773">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546773" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546773">
        <p class="paperTitle">Truly tiny acoustic biomolecules for ultrasound imaging and therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546773" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546773" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ling, B.; Gungoren, B.; Yao, Y.; Dutka, P.; Smith, C. A. B.; Lee, J.; Swift, M. B.; Shapiro, M. G.</p>
        <p class="info">Score: 25.5, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546773' target='https://doi.org/10.1101/2023.06.27.546773'> 10.1101/2023.06.27.546773</a></p>
        <p class="abstract">Nanotechnology offers significant advantages for medical imaging and therapy, including enhanced contrast and precision targeting. However, integrating these benefits into ultrasonography has been challenging due to the size and stability constraints of conventional bubble-based agents. Here we describe bicones, truly tiny acoustic contrast agents based on gas vesicles, a unique class of air-filled protein nanostructures naturally produced in buoyant microbes. We show that these sub-80 nm particles can be effectively detected both in vitro and in vivo, infiltrate tumors via leaky vasculature, deliver potent mechanical effects through ultrasound-induced inertial cavitation, and are easily engineered for molecular targeting, prolonged circulation time, and payload conjugation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547852">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547852" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547852">
        <p class="paperTitle">pUdOs: concise plasmids for bacterial and mammalian cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547852" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547852" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ourida Manigat, F.; Connell, L. B.; Stewart, B. N.; LePabic, A.-R.; Tessier, C. J. G.; Emlaw, J. R.; Calvert, N. D.; Rossl, A.; Shuhendler, A. J.; daCosta, C. J. B.; Campbell-Valois, F.-X.</p>
        <p class="info">Score: 6.5, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547852' target='https://doi.org/10.1101/2023.07.05.547852'> 10.1101/2023.07.05.547852</a></p>
        <p class="abstract">The pUdOs are 28 plasmids of small size combining four different origins of replication and seven selection markers, which together afford flexible use in Escherichia coli and several related gram-negative bacteria. The promoterless multicloning site is insulated from upstream spurious promoters by strong transcription terminators, and contains type IIP or IIS restriction sites for conventional or Golden-gate cloning. pUdOs can be converted into efficient expression vectors through the insertion of a promoter at the user&#39;s discretion. For example, we demonstrate the utility of pUdOs as the backbone for an improved version of a Type III Secretion System reporter in Shigella. In addition, we derive a series of pUdO-based mammalian expression vectors affording distinct levels of expression and transfection efficiencies comparable to commonly used mammalian expression plasmids. Thus, pUdOs could advantageously replace traditional plasmids in a wide variety of cell types and applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547106">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547106" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547106">
        <p class="paperTitle">A Platform for Controlled Template-Independent Enzymatic Synthesis of RNA Oligonucleotides and Therapeutics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547106" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547106" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wiegand, D. J.; Rittichier, J.; Meyer, E.; Lee, H.; Conway, N. J.; Ahlstedt, D.; Yurtsever, Z.; Rainone, D.; Kuru, E.; Church, G.</p>
        <p class="info">Score: 4.9, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547106' target='https://doi.org/10.1101/2023.06.29.547106'> 10.1101/2023.06.29.547106</a></p>
        <p class="abstract">Therapeutic RNA oligonucleotides have shown tremendous potential to manage and treat disease, yet current manufacturing methods cannot deliver on this promise. Here, we report the development and optimization of a novel, aqueous-based, template-independent enzymatic RNA oligonucleotide synthesis platform as an alternative to traditional chemical methodologies. Our platform is made possible by reversible terminator nucleoside triphosphates and an enzyme capable of their incorporation. We show that many common therapeutic RNA modifications are compatible with our process and demonstrate the enzymatic synthesis of natural and modified oligonucleotides in both liquid and solid phases. Our platform offers many unique advantages over chemical synthesis, including the realization of a more sustainable process to produce therapeutic RNA oligonucleotides.

One-Sentence SummaryAn enzyme and novel nucleotide building blocks are used to synthesize RNA oligonucleotides template independently under aqueous conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.13.544841">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.13.544841" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.13.544841">
        <p class="paperTitle">CRISPR-mediated germline mutagenesis for genetic sterilization of Anopheles gambiae males.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.13.544841" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.13.544841" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smidler, A. L.; Paton, D. G.; Church, G. M.; Shaw, W. R.; Catteruccia, F.</p>
        <p class="info">Score: 9.7, Published: 2023-06-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.13.544841' target='https://doi.org/10.1101/2023.06.13.544841'> 10.1101/2023.06.13.544841</a></p>
        <p class="abstract">Rapid spread of insecticide resistance among anopheline mosquitoes threatens malaria elimination efforts, necessitating development of alternative vector control technologies. Sterile Insect Technique (SIT) has been successfully implemented in multiple insect pests to suppress field populations by the release of large numbers of sterile males, yet it has proven difficult to adapt to Anopheles vectors. Here we outline adaptation of a CRISPR-based genetic sterilization system to selectively ablate male sperm cells in the malaria mosquito Anopheles gambiae. We achieve robust mosaic biallelic mutagenesis of zero population growth (zpg, a gene essential for differentiation of germ cells) in F1 individuals after intercrossing a germline-expressing Cas9 transgenic line to a line expressing zpg-targeting gRNAs. Approximately 95% of mutagenized males display complete genetic sterilization, and cause similarly high levels of infertility in their female mates. Using a fluorescence reporter that allows detection of the germline leads to a 100% accurate selection of spermless males, improving the system. These males cause a striking reduction in mosquito population size when released at field-like frequencies in competition cages against wild type males. These findings demonstrate that such a genetic system could be adopted for SIT against important malaria vectors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.11.523572">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.11.523572" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.11.523572">
        <p class="paperTitle">T7 phage-assisted evolution of riboswitches using error-prone replication and dual selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.11.523572" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.11.523572" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goicoechea, E.; Jaramillo, A.</p>
        <p class="info">Score: 19.1, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.11.523572' target='https://doi.org/10.1101/2023.01.11.523572'> 10.1101/2023.01.11.523572</a></p>
        <p class="abstract">Leveraging riboswitches, non-coding mRNA fragments pivotal to gene regulation, poses a challenge in effectively selecting and enriching these functional genetic sensors, which can toggle between ON and OFF states in response to their cognate inducers. Here, we show our engineered phage T7, enabling the evolution of a theophylline riboswitch. We have replaced T7&#39;s DNA polymerase with a transcription factor controlled by a theophylline riboswitch and have created two types of host environments to propagate the engineered phage. Both types host an error-prone T7 DNA polymerase regulated by a T7 promoter along with another critical gene - either cmk or pifA, depending on the host type. The cmk gene is necessary for T7 replication and is used in the first host type for selection in the riboswitch&#39;s ON state. Conversely, the second host type incorporates the pifA gene, leading to abortive T7 infections and used for selection in the riboswitch&#39;s OFF state. This dual-selection system, termed T7AE, was then applied to a library of 65,536 engineered T7 phages, each carrying randomized riboswitch variants. Through successive passage in both host types with and without theophylline, we observed an enrichment of phages encoding functional riboswitches that conferred a fitness advantage to the phage in both hosts. The T7AE technique thereby opens new pathways for the evolution and advancement of gene switches, including non-coding RNA-based switches, setting the stage for significant strides in synthetic biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.12.544656">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.12.544656" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.12.544656">
        <p class="paperTitle">CRISPR-based gene drives generate super-Mendelian inheritance in the disease vector Culex quinquefasciatus.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.12.544656" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.12.544656" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harvey-Samuel, T.; Feng, X.; Okamoto, E. M.; Purusothaman, D.-K.; Leftwich, P. T.; Alphey, L.; Gantz, V. M.</p>
        <p class="info">Score: 25.8, Published: 2023-06-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.12.544656' target='https://doi.org/10.1101/2023.06.12.544656'> 10.1101/2023.06.12.544656</a></p>
        <p class="abstract">Culex mosquitoes pose a significant public health threat as vectors for a variety of diseases including West Nile virus and lymphatic filariasis, and transmit pathogens threatening livestock, companion animals, and endangered birds. Rampant insecticide resistance makes controlling these mosquitoes challenging and necessitates the development of new control strategies. Gene drive technologies have made significant progress in other mosquito species, although similar advances have been lagging in Culex. Here we test the first CRISPR-based homing gene drive for Culex quinquefasciatus, demonstrating the possibility of using this technology to control Culex mosquitoes. Our results show that the inheritance of two split-gene-drive transgenes, targeting different loci, are biased in the presence of a Cas9-expressing transgene although with modest efficiencies. Our findings extend the list of disease vectors where engineered homing gene drives have been demonstrated to include Culex alongside Anopheles and Aedes, and pave the way for future development of these technologies to control Culex mosquitoes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548074" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548074">
        <p class="paperTitle">Exploring alternative quorum sensing model structures and quorum quenching strategies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cimolato, C.; Bellato, M.; Selvaggio, G.; Marchetti, L.; Giordano, G.; Schenato, L.</p>
        <p class="info">Score: 1.0, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548074' target='https://doi.org/10.1101/2023.07.07.548074'> 10.1101/2023.07.07.548074</a></p>
        <p class="abstract">Bacterial quorum sensing (QS) is a cell-to-cell communication mechanism through which bacteria share information about cell density, and tune gene expression accordingly. Pathogens exploit QS to orchestrate virulence and regulate the expression of genes related to antimicrobial resistance. Despite the vast literature on QS, the properties of the underlying molecular network are not entirely clear. We compare two synthetic QS circuit architectures: in the first, a single positive feedback loop autoinduces the synthesis of the signal molecule; the second includes an additional positive feedback loop enhancing the synthesis of the signal molecule receptor. Our comprehensive analysis of the two systems and their equilibria highlights the differences in the bistable and hysteretic behaviors of the alternative QS structures. Finally, we investigate three different QS inhibition approaches; numerical analysis predicts their effect on the steady-state behavior of the two different QS models, revealing critical parameter thresholds that guarantee an effective QS suppression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547683">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547683" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547683">
        <p class="paperTitle">A digital twin for DNA data storage based on comprehensive quantification of errors and biases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547683" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547683" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gimpel, A. L.; Stark, W. J.; Heckel, R.; Grass, R. N.</p>
        <p class="info">Score: 1.0, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547683' target='https://doi.org/10.1101/2023.07.04.547683'> 10.1101/2023.07.04.547683</a></p>
        <p class="abstract">Archiving data in synthetic DNA offers unprecedented storage density and longevity. Handling and storage introduce errors and biases into DNA-based storage systems, necessitating the use of Error Correction Coding (ECC) which comes at the cost of added redundancy. However, insufficient data on these errors and biases, as well as a lack of modelling tools, limit data-driven ECC development and experimental design. In this study, we present a comprehensive characterisation of the error sources and biases present in the most common DNA data storage workflows, including commercial DNA synthesis, PCR, decay by accelerated aging, and sequencing-by-synthesis. Using the data from 40 sequencing experiments, we build a digital twin of the DNA data storage process, capable of simulating state-of-the-art workflows and reproducing their experimental results. We showcase the digital twins ability to replace experiments and rationalize the design of redundancy in two case studies, highlighting opportunities for tangible cost savings and data-driven ECC development.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
