<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Continuous evolution of user-defined genes at 1-million-times the genomic mutation rate
Authors: Rix, G.; Williams, R. L.; Spinner, H.; Hu, V. J.; Marks, D. S.; Liu, C. C.
Score: 54.5, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566922
When nature maintains or evolves a genes function over millions of years at scale, it produces a diversity of homologous sequences whose patterns of conservation and change contain rich structural, functional, and historical information about the gene.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Continuous evolution of user-defined genes at 1-million-times the genomic mutation rate
Authors: Rix, G.; Williams, R. L.; Spinner, H.; Hu, V. J.; Marks, D. S.; Liu, C. C.
Score: 54.5, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566922
When nature maintains or evolves a genes function over millions of years at scale, it produces a diversity of homologous sequences whose patterns of conservation and change contain rich structural, functional, and historical information about the gene." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-19T10:38:34+00:00" />
<meta property="article:modified_time" content="2023-11-19T10:38:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Continuous evolution of user-defined genes at 1-million-times the genomic mutation rate
Authors: Rix, G.; Williams, R. L.; Spinner, H.; Hu, V. J.; Marks, D. S.; Liu, C. C.
Score: 54.5, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566922
When nature maintains or evolves a genes function over millions of years at scale, it produces a diversity of homologous sequences whose patterns of conservation and change contain rich structural, functional, and historical information about the gene."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Continuous evolution of user-defined genes at 1-million-times the genomic mutation rate\nAuthors: Rix, G.; Williams, R. L.; Spinner, H.; Hu, V. J.; Marks, D. S.; Liu, C. C.\nScore: 54.5, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566922\nWhen nature maintains or evolves a genes function over millions of years at scale, it produces a diversity of homologous sequences whose patterns of conservation and change contain rich structural, functional, and historical information about the gene.",
  "keywords": [
    
  ],
  "articleBody": " Continuous evolution of user-defined genes at 1-million-times the genomic mutation rate\nAuthors: Rix, G.; Williams, R. L.; Spinner, H.; Hu, V. J.; Marks, D. S.; Liu, C. C.\nScore: 54.5, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566922\nWhen nature maintains or evolves a genes function over millions of years at scale, it produces a diversity of homologous sequences whose patterns of conservation and change contain rich structural, functional, and historical information about the gene. However, natural gene diversity likely excludes vast regions of functional sequence space and includes phylogenetic and evolutionary eccentricities, limiting what information we can extract. We introduce an accessible experimental approach for compressing long-term gene evolution to laboratory timescales, allowing for the direct observation of extensive adaptation and divergence followed by inference of structural, functional, and environmental constraints for any selectable gene. To enable this approach, we developed a new orthogonal DNA replication (OrthoRep) system that durably hypermutates chosen genes at a rate of \u003e10-4 substitutions per base in vivo. When OrthoRep was used to evolve a conditionally essential maladapted enzyme, we obtained thousands of unique multi-mutation sequences with many pairs \u003e60 amino acids apart (\u003e15% divergence), revealing known and new factors influencing enzyme adaptation. The fitness of evolved sequences was not predictable by advanced machine learning models trained on natural variation. We suggest that OrthoRep supports the prospective and systematic discovery of constraints shaping gene evolution, uncovering of new regions in fitness landscapes, and general applications in biomolecular engineering.\nDirected Evolution of Genetically Encoded LYTACs for Cell-Mediated Delivery\nAuthors: Yang, J. L.; Yamada-Hunter, S. A.; Labanieh, L.; Sotillo, E.; Cheah, J. S.; Roberts, D. S.; Mackall, C.; Ting, A. Y.; Bertozzi, C.\nScore: 18.5, Published: 2023-11-15 DOI: 10.1101/2023.11.14.567117\nLysosome-targeting chimeras (LYTACs) are a promising therapeutic modality to drive the degradation of extracellular proteins. However, early versions of LYTAC contain synthetic glycopeptides that cannot be genetically encoded. Here we present our designs for a fully genetically encodable LYTAC (GELYTAC), making our tool compatible with integration into therapeutic cells for targeted delivery at diseased sites. To achieve this, we replaced the glycopeptide portion of LYTACs with the protein insulin like growth factor 2 (IGF2). After showing initial efficacy with wild type IGF2, we increased the potency of GELYTAC using directed evolution. Subsequently, we demonstrated that our engineered GELYTAC construct not only secretes from HEK293T cells but also from human primary T-cells to drive the uptake of various targets into receiver cells. Immune cells engineered to secrete GELYTAC thus represent a promising avenue for spatially-selective targeted protein degradation. Significance StatementBetter therapeutic windows can be achieved by targeting therapeutics to their desired sites of action. For protein therapeutics, this might be achieved by engineering cell therapies that home to a tissue of interest and secrete the biologic drug locally. Here, we demonstrate that human primary T cells can be engineered to produce genetically encoded lysosome targeting chimeras (GELYTACs). These GELYTACs mediate the degradation of extracellular proteins associated with cancer progression. Thus, cells engineered to produce GELYTACs represent a potential new class of cancer therapeutics.\nCryptography in the DNA of living cells enabled by multi-site base editing\nAuthors: Volf, V.; Zhang, S.; Song, K. M.; Qian, S.; Chen, F.; Church, G.\nScore: 14.3, Published: 2023-11-15 DOI: 10.1101/2023.11.15.567131\nAlthough DNA is increasingly being adopted as a generalizable medium for information storage and transfer, reliable methods for ensuring information security remain to be addressed. In this study, we developed and validated a cryptographic encoding scheme, Genomic Sequence Encryption (GSE), to address the challenge of information confidentiality and integrity in biological substrates. GSE enables genomic information encoding that is readable only with a cryptographic key. We show that GSE can be used for cell signatures that enable the recipient of a cell line to authenticate its origin and validate if the cell line has been modified in the interim. We implement GSE through multi-site base editing and encode information through editing across \u003e100 genomic sites in mammalian cells. We further present an enrichment step to obtain individual stem cells with more than two dozen edits across a single genome with minimal screening. This capability can be used to introduce encrypted signatures in living animals. As an encryption scheme, GSE is falsification-proof and enables secure information transfer in biological substrates.\nMinimization of the E. coli ribosome, aided and optimized by community science\nAuthors: Tangpradabkul, T.; Palo, M. Z.; Townley, J.; Hsu, K. B.; Eterna Participants, ; Smaga, S.; Das, R.; Schepartz, A.\nScore: 20.4, Published: 2023-11-10 DOI: 10.1101/2023.11.09.566451\nThe ribosome is a ribonucleoprotein complex found in all domains of life. Its role is to catalyze protein synthesis, the messenger RNA (mRNA)-templated formation of amide bonds between -amino acid monomers. Amide bond formation occurs within a highly conserved region of the large ribosomal subunit known as the peptidyl transferase center (PTC). Here we describe the stepwise design and characterization of mini-PTC 1.1, a 284-nucleotide RNA that recapitulates many essential features of the Escherichia coli PTC. Mini-PTC 1.1 folds into a PTC-like structure under physiological conditions, even in the absence of r-proteins, and engages small molecule analogs of A- and P-site tRNAs. The sequence of mini-PTC 1.1 differs from the wild type E. coli ribosome at 12 nucleotides that were installed by a cohort of citizen scientists using the on-line video game Eterna. These base changes improve both the secondary structure and tertiary folding of mini-PTC 1.1 as well as its ability to bind small molecule substrate analogs. Here, the combined input from Eterna citizen-scientists and RNA structural analysis provides a robust workflow for the design of a minimal PTC that recapitulates many features of an intact ribosome.\nA tuneable minimal cell membrane reveals that two lipids suffice for life\nAuthors: Justice, I. G.; Saenz, J. P.\nScore: 36.9, Published: 2023-11-08 DOI: 10.1101/2023.10.24.563757\nAll cells are encapsulated by a lipid membrane which facilitates the interaction between life and its environment. The property and function of the cell membrane is critically dependent on its lipidome composition which can range from 10s to 100s of unique lipid structures. How life has evolved to exploit such diverse mixtures of lipids has been experimentally challenging to address. In this study, we introduce a simple approach to minimize and tune lipidome composition in Mycoplasma mycoides and the Minimal Cell (JCVI-Syn3A). By varying the lipid composition of the growth medium, and introducing unmodifiable diether phospholipids, we demonstrate that a lipidome composed of two lipids can support life. By systematically reintroducing phospholipid features, we show that acyl chain diversity is more critical for supporting growth than head group diversity. We also reveal significant effects of altering phospholipid chirality on membrane permeability, cellular robustness, and growth, demonstrating how subtle changes in lipid stereochemistry can have profound effects on cell fitness. Our approach to tune mycoplasma lipidome composition introduces a minimal living model membrane system to understand the design principles and the role of lipid complexity in bioengineering of cell membranes.\nGenetically stable kill-switch using ''demon and angel'' expression construct of essential genes\nAuthors: Kato, Y.; Mori, H.\nScore: 5.9, Published: 2023-11-13 DOI: 10.1101/2023.11.12.566782\nGenetic instability of synthetic genetic devices is a key obstacle for practical use. This problem is particularly critical in kill-switches for conditional host killing. Here, we propose a genetically stable kill-switch based on a \"demon and angel\" expression construct of a toxic essential gene. The kill-switch conditionally overexpresses the toxic essential gene. Additionally, the identical essential gene is deleted in the genome. The essential gene is expressed at a low level to maintain host survival in the OFF state and kills the host by the overexpression in the ON state. The single expression construct is responsible for both killing the hosts and maintaining viability, reducing the emergence of loss-of-function mutants. We constructed the kill-switch using the toxic essential gene encoding tyrosyl-tRNA synthetase, tyrS, in Escherichia coli. The bacteria harboring the kill-switch were conditionally suicidal over 300 generations. Toxic overexpression of essential genes has also been found in other organisms, suggesting that the \"demon and angel\" kill switch is scalable to various organisms. GRAPHICAL ABSTRACT O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=141 SRC=\"FIGDIR/small/566782v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (27K): org.highwire.dtl.DTLVardef@3ffe8forg.highwire.dtl.DTLVardef@16191b4org.highwire.dtl.DTLVardef@a4d398org.highwire.dtl.DTLVardef@13fb500_HPS_FORMAT_FIGEXP M_FIG C_FIG\nEvolution is All You Need in Promoter Design and Optimization\nAuthors: Ren, R.; Yu, H.; Teng, J.; Mao, S.; Bian, Z.; Tao, Y.; Yau, S. S.-T.\nScore: 5.9, Published: 2023-11-18 DOI: 10.1101/2023.11.18.567645\nPredicting the strength of promoters and guiding their directed evolution is a crucial task in synthetic biology. This approach significantly reduces the experimental costs in conventional promoter engineering. Previous studies employing machine learning or deep learning methods have shown some success in this task, but their outcomes were not satisfactory enough, primarily due to the neglect of evolutionary information. In this paper, we introduce the Chaos-Attention net for Promoter Evolution (CAPE) to address the limitations of existing methods. We comprehensively extract evolutionary information within promoters using chaos game representation and process the overall information with DenseNet and Transformer. Our model achieves state-of-the-art results on two kinds of distinct tasks. The incorporation of evolutionary information enhances the model's accuracy, with transfer learning further extending its adaptability. Furthermore, experimental results confirm CAPE's efficacy in simulating in silico directed evolution of promoters, marking a significant advancement in predictive modeling for prokaryotic promoter strength. Our paper also presents a user-friendly website for the practical implementation of in silico directed evolution on promoters.\nThermodynamically-consistent, reduced models of gene regulatory networks\nAuthors: Pan, M.; Gawthrop, P. J.; Faria, M.; Johnston, S. T.\nScore: 4.1, Published: 2023-11-13 DOI: 10.1101/2023.11.13.566770\nSynthetic biology aims to engineer novel functionalities into biological systems. While the approach has been widely applied to single cells, the scale of synthetic circuits designed in this way is limited by factors such as resource competition and retroactivity. Synthetic biology of cell populations has the potential to overcome some of these limitations by physically isolating synthetic genes from each other. To rationally design cell populations, we require mathematical models that link between intracellular biochemistry and intercellular interactions. The interfacing of agent-based models with systems biology models is particularly important in understanding the effects of cell heterogeneity and cell-to-cell interactions. In this study, we develop a model of gene expression that is suitable for incorporation into agent-based models of cell populations. To be scalable to large cell populations, models of gene expression should be both computationally efficient and compliant with the laws of physics. We satisfy the first requirement by applying a model reduction scheme to translation, and the second requirement by formulating models using bond graphs. We show that the reduced model of translation faithfully reproduces the behaviour of the full model at steady state. The reduced model is benchmarked against the full model, and we find a sub-stantial speedup at realistic protein lengths. Using the modularity of the bond graph approach, we couple separate models of gene expression to build models of the toggle switch and repressilator. With these models, we explore the effects of resource availability and cell-to-cell heterogeneity on circuit behaviour. The modelling approaches developed in this study are a bridge towards rationally designing collective cell behaviours such as synchronisation and division of labour.\nComputational Design of Periplasmic Binding Protein Biosensors Guided by Molecular Dynamics\nAuthors: O'Shea, J. M.; Richardson, A.; Doerner, P. W.; Wood, C. W.\nScore: 5.6, Published: 2023-11-10 DOI: 10.1101/2023.11.10.566541\nPeriplasmic binding proteins (PBPs) evolved to scavenge materials from the environment by binding substrate and delivering it to the host bacteria. PBPs are common scaffolds for biosensors: effector proteins, such as circularly permuted green fluorescent protein (cpGFP), can be inserted into a PBP sequence such that the effector proteins output is changed by conformational changes upon PBP substate binding. The site of insertion for maximum output change is often determined by comparison of PBP apo/holo crystal structures, but random insertion libraries have shown that this method can miss the best sites. Here, we present a computational method for identifying insertion sites for cpGFP in the maltose binding PBP that creates functional biosensors. Our method, which uses protein contact analysis, identifies the best-known insertion regions, as well as additional regions that have not been identified previously. We experimentally characterised biosensors with insertions at these regions and found that 75% of designs were functional. This method is generalisable beyond this system and PBPs more generally, and so could be applied to other binding proteins or effector domains.\nStrong sequence dependence in RNA/DNA hybrid strand displacement kinetics\nAuthors: Smith, F. G.; Goertz, J. P.; Stevens, M. M.; Ouldridge, T. E.\nScore: 3.4, Published: 2023-11-14 DOI: 10.1101/2023.11.14.567030\nStrand displacement reactions underlie dynamic nucleic acid nanotechnology. The kinetic and thermodynamic features of DNA-based displacement reactions are well understood and well predicted by current computational models. By contrast, understanding of RNA/DNA hybrid strand displacement kinetics is limited, restricting the design of increasingly complex RNA/DNA hybrid reaction networks with more tightly regulated dynamics. Given the importance of RNA as a diagnostic biomarker, and its critical role in intracellular processes, this shortfall is particularly limiting for the development of strand displacement-based therapeutics and diagnostics. Herein, we characterise 22 RNA/DNA hybrid strand displacement systems, systematically varying several common design parameters including toehold length and branch migration domain length. We observe the differences in stability between RNA-DNA hybrids and DNA-DNA duplexes have large effects on strand displacement rates, with rates for equivalent sequences differing by up to 3 orders of magnitude. Crucially, however, this effect is strongly sequence-dependent, with RNA invaders strongly favoured in a system with RNA strands of high purine content, and disfavoured in a system when the RNA strands have low purine content. These results lay the groundwork for more general design principles, allowing for creation of de novo reaction networks with novel complexity while maintaining predictable reaction kinetics.\n",
  "wordCount" : "2266",
  "inLanguage": "en",
  "datePublished": "2023-11-19T10:38:34Z",
  "dateModified": "2023-11-19T10:38:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 19, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566922">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566922" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566922">
        <p class="paperTitle">Continuous evolution of user-defined genes at 1-million-times the genomic mutation rate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566922" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566922" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rix, G.; Williams, R. L.; Spinner, H.; Hu, V. J.; Marks, D. S.; Liu, C. C.</p>
        <p class="info">Score: 54.5, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566922' target='https://doi.org/10.1101/2023.11.13.566922'> 10.1101/2023.11.13.566922</a></p>
        <p class="abstract">When nature maintains or evolves a genes function over millions of years at scale, it produces a diversity of homologous sequences whose patterns of conservation and change contain rich structural, functional, and historical information about the gene. However, natural gene diversity likely excludes vast regions of functional sequence space and includes phylogenetic and evolutionary eccentricities, limiting what information we can extract. We introduce an accessible experimental approach for compressing long-term gene evolution to laboratory timescales, allowing for the direct observation of extensive adaptation and divergence followed by inference of structural, functional, and environmental constraints for any selectable gene. To enable this approach, we developed a new orthogonal DNA replication (OrthoRep) system that durably hypermutates chosen genes at a rate of &gt;10-4 substitutions per base in vivo. When OrthoRep was used to evolve a conditionally essential maladapted enzyme, we obtained thousands of unique multi-mutation sequences with many pairs &gt;60 amino acids apart (&gt;15% divergence), revealing known and new factors influencing enzyme adaptation. The fitness of evolved sequences was not predictable by advanced machine learning models trained on natural variation. We suggest that OrthoRep supports the prospective and systematic discovery of constraints shaping gene evolution, uncovering of new regions in fitness landscapes, and general applications in biomolecular engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567117">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567117" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567117">
        <p class="paperTitle">Directed Evolution of Genetically Encoded LYTACs for Cell-Mediated Delivery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567117" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567117" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, J. L.; Yamada-Hunter, S. A.; Labanieh, L.; Sotillo, E.; Cheah, J. S.; Roberts, D. S.; Mackall, C.; Ting, A. Y.; Bertozzi, C.</p>
        <p class="info">Score: 18.5, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567117' target='https://doi.org/10.1101/2023.11.14.567117'> 10.1101/2023.11.14.567117</a></p>
        <p class="abstract">Lysosome-targeting chimeras (LYTACs) are a promising therapeutic modality to drive the degradation of extracellular proteins. However, early versions of LYTAC contain synthetic glycopeptides that cannot be genetically encoded. Here we present our designs for a fully genetically encodable LYTAC (GELYTAC), making our tool compatible with integration into therapeutic cells for targeted delivery at diseased sites. To achieve this, we replaced the glycopeptide portion of LYTACs with the protein insulin like growth factor 2 (IGF2). After showing initial efficacy with wild type IGF2, we increased the potency of GELYTAC using directed evolution. Subsequently, we demonstrated that our engineered GELYTAC construct not only secretes from HEK293T cells but also from human primary T-cells to drive the uptake of various targets into receiver cells. Immune cells engineered to secrete GELYTAC thus represent a promising avenue for spatially-selective targeted protein degradation.

Significance StatementBetter therapeutic windows can be achieved by targeting therapeutics to their desired sites of action. For protein therapeutics, this might be achieved by engineering cell therapies that home to a tissue of interest and secrete the biologic drug locally. Here, we demonstrate that human primary T cells can be engineered to produce genetically encoded lysosome targeting chimeras (GELYTACs). These GELYTACs mediate the degradation of extracellular proteins associated with cancer progression. Thus, cells engineered to produce GELYTACs represent a potential new class of cancer therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567131">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567131" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567131">
        <p class="paperTitle">Cryptography in the DNA of living cells enabled by multi-site base editing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567131" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567131" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Volf, V.; Zhang, S.; Song, K. M.; Qian, S.; Chen, F.; Church, G.</p>
        <p class="info">Score: 14.3, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567131' target='https://doi.org/10.1101/2023.11.15.567131'> 10.1101/2023.11.15.567131</a></p>
        <p class="abstract">Although DNA is increasingly being adopted as a generalizable medium for information storage and transfer, reliable methods for ensuring information security remain to be addressed. In this study, we developed and validated a cryptographic encoding scheme, Genomic Sequence Encryption (GSE), to address the challenge of information confidentiality and integrity in biological substrates. GSE enables genomic information encoding that is readable only with a cryptographic key. We show that GSE can be used for cell signatures that enable the recipient of a cell line to authenticate its origin and validate if the cell line has been modified in the interim. We implement GSE through multi-site base editing and encode information through editing across &gt;100 genomic sites in mammalian cells. We further present an enrichment step to obtain individual stem cells with more than two dozen edits across a single genome with minimal screening. This capability can be used to introduce encrypted signatures in living animals. As an encryption scheme, GSE is falsification-proof and enables secure information transfer in biological substrates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566451">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566451" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566451">
        <p class="paperTitle">Minimization of the E. coli ribosome, aided and optimized by community science</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566451" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566451" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tangpradabkul, T.; Palo, M. Z.; Townley, J.; Hsu, K. B.; Eterna Participants,  ; Smaga, S.; Das, R.; Schepartz, A.</p>
        <p class="info">Score: 20.4, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566451' target='https://doi.org/10.1101/2023.11.09.566451'> 10.1101/2023.11.09.566451</a></p>
        <p class="abstract">The ribosome is a ribonucleoprotein complex found in all domains of life. Its role is to catalyze protein synthesis, the messenger RNA (mRNA)-templated formation of amide bonds between -amino acid monomers. Amide bond formation occurs within a highly conserved region of the large ribosomal subunit known as the peptidyl transferase center (PTC). Here we describe the stepwise design and characterization of mini-PTC 1.1, a 284-nucleotide RNA that recapitulates many essential features of the Escherichia coli PTC. Mini-PTC 1.1 folds into a PTC-like structure under physiological conditions, even in the absence of r-proteins, and engages small molecule analogs of A- and P-site tRNAs. The sequence of mini-PTC 1.1 differs from the wild type E. coli ribosome at 12 nucleotides that were installed by a cohort of citizen scientists using the on-line video game Eterna. These base changes improve both the secondary structure and tertiary folding of mini-PTC 1.1 as well as its ability to bind small molecule substrate analogs. Here, the combined input from Eterna citizen-scientists and RNA structural analysis provides a robust workflow for the design of a minimal PTC that recapitulates many features of an intact ribosome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563757">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563757" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563757">
        <p class="paperTitle">A tuneable minimal cell membrane reveals that two lipids suffice for life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563757" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563757" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Justice, I. G.; Saenz, J. P.</p>
        <p class="info">Score: 36.9, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563757' target='https://doi.org/10.1101/2023.10.24.563757'> 10.1101/2023.10.24.563757</a></p>
        <p class="abstract">All cells are encapsulated by a lipid membrane which facilitates the interaction between life and its environment. The property and function of the cell membrane is critically dependent on its lipidome composition which can range from 10s to 100s of unique lipid structures. How life has evolved to exploit such diverse mixtures of lipids has been experimentally challenging to address. In this study, we introduce a simple approach to minimize and tune lipidome composition in Mycoplasma mycoides and the Minimal Cell (JCVI-Syn3A). By varying the lipid composition of the growth medium, and introducing unmodifiable diether phospholipids, we demonstrate that a lipidome composed of two lipids can support life. By systematically reintroducing phospholipid features, we show that acyl chain diversity is more critical for supporting growth than head group diversity. We also reveal significant effects of altering phospholipid chirality on membrane permeability, cellular robustness, and growth, demonstrating how subtle changes in lipid stereochemistry can have profound effects on cell fitness. Our approach to tune mycoplasma lipidome composition introduces a minimal living model membrane system to understand the design principles and the role of lipid complexity in bioengineering of cell membranes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.12.566782">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.12.566782" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.12.566782">
        <p class="paperTitle">Genetically stable kill-switch using &#39;&#39;demon and angel&#39;&#39; expression construct of essential genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.12.566782" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.12.566782" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kato, Y.; Mori, H.</p>
        <p class="info">Score: 5.9, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.12.566782' target='https://doi.org/10.1101/2023.11.12.566782'> 10.1101/2023.11.12.566782</a></p>
        <p class="abstract">Genetic instability of synthetic genetic devices is a key obstacle for practical use. This problem is particularly critical in kill-switches for conditional host killing. Here, we propose a genetically stable kill-switch based on a &#34;demon and angel&#34; expression construct of a toxic essential gene. The kill-switch conditionally overexpresses the toxic essential gene. Additionally, the identical essential gene is deleted in the genome. The essential gene is expressed at a low level to maintain host survival in the OFF state and kills the host by the overexpression in the ON state. The single expression construct is responsible for both killing the hosts and maintaining viability, reducing the emergence of loss-of-function mutants. We constructed the kill-switch using the toxic essential gene encoding tyrosyl-tRNA synthetase, tyrS, in Escherichia coli. The bacteria harboring the kill-switch were conditionally suicidal over 300 generations. Toxic overexpression of essential genes has also been found in other organisms, suggesting that the &#34;demon and angel&#34; kill switch is scalable to various organisms.

GRAPHICAL ABSTRACT

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=141 SRC=&#34;FIGDIR/small/566782v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (27K):
org.highwire.dtl.DTLVardef@3ffe8forg.highwire.dtl.DTLVardef@16191b4org.highwire.dtl.DTLVardef@a4d398org.highwire.dtl.DTLVardef@13fb500_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.18.567645">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.18.567645" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.18.567645">
        <p class="paperTitle">Evolution is All You Need in Promoter Design and Optimization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.18.567645" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.18.567645" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ren, R.; Yu, H.; Teng, J.; Mao, S.; Bian, Z.; Tao, Y.; Yau, S. S.-T.</p>
        <p class="info">Score: 5.9, Published: 2023-11-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.18.567645' target='https://doi.org/10.1101/2023.11.18.567645'> 10.1101/2023.11.18.567645</a></p>
        <p class="abstract">Predicting the strength of promoters and guiding their directed evolution is a crucial task in synthetic biology. This approach significantly reduces the experimental costs in conventional promoter engineering. Previous studies employing machine learning or deep learning methods have shown some success in this task, but their outcomes were not satisfactory enough, primarily due to the neglect of evolutionary information. In this paper, we introduce the Chaos-Attention net for Promoter Evolution (CAPE) to address the limitations of existing methods. We comprehensively extract evolutionary information within promoters using chaos game representation and process the overall information with DenseNet and Transformer. Our model achieves state-of-the-art results on two kinds of distinct tasks. The incorporation of evolutionary information enhances the model&#39;s accuracy, with transfer learning further extending its adaptability. Furthermore, experimental results confirm CAPE&#39;s efficacy in simulating in silico directed evolution of promoters, marking a significant advancement in predictive modeling for prokaryotic promoter strength. Our paper also presents a user-friendly website for the practical implementation of in silico directed evolution on promoters.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566770">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566770" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566770">
        <p class="paperTitle">Thermodynamically-consistent, reduced models of gene regulatory networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566770" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566770" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pan, M.; Gawthrop, P. J.; Faria, M.; Johnston, S. T.</p>
        <p class="info">Score: 4.1, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566770' target='https://doi.org/10.1101/2023.11.13.566770'> 10.1101/2023.11.13.566770</a></p>
        <p class="abstract">Synthetic biology aims to engineer novel functionalities into biological systems. While the approach has been widely applied to single cells, the scale of synthetic circuits designed in this way is limited by factors such as resource competition and retroactivity. Synthetic biology of cell populations has the potential to overcome some of these limitations by physically isolating synthetic genes from each other. To rationally design cell populations, we require mathematical models that link between intracellular biochemistry and intercellular interactions. The interfacing of agent-based models with systems biology models is particularly important in understanding the effects of cell heterogeneity and cell-to-cell interactions. In this study, we develop a model of gene expression that is suitable for incorporation into agent-based models of cell populations. To be scalable to large cell populations, models of gene expression should be both computationally efficient and compliant with the laws of physics. We satisfy the first requirement by applying a model reduction scheme to translation, and the second requirement by formulating models using bond graphs. We show that the reduced model of translation faithfully reproduces the behaviour of the full model at steady state. The reduced model is benchmarked against the full model, and we find a sub-stantial speedup at realistic protein lengths. Using the modularity of the bond graph approach, we couple separate models of gene expression to build models of the toggle switch and repressilator. With these models, we explore the effects of resource availability and cell-to-cell heterogeneity on circuit behaviour. The modelling approaches developed in this study are a bridge towards rationally designing collective cell behaviours such as synchronisation and division of labour.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566541">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566541" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566541">
        <p class="paperTitle">Computational Design of Periplasmic Binding Protein Biosensors Guided by Molecular Dynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566541" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566541" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: O&#39;Shea, J. M.; Richardson, A.; Doerner, P. W.; Wood, C. W.</p>
        <p class="info">Score: 5.6, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566541' target='https://doi.org/10.1101/2023.11.10.566541'> 10.1101/2023.11.10.566541</a></p>
        <p class="abstract">Periplasmic binding proteins (PBPs) evolved to scavenge materials from the environment by binding substrate and delivering it to the host bacteria. PBPs are common scaffolds for biosensors: effector proteins, such as circularly permuted green fluorescent protein (cpGFP), can be inserted into a PBP sequence such that the effector proteins output is changed by conformational changes upon PBP substate binding. The site of insertion for maximum output change is often determined by comparison of PBP apo/holo crystal structures, but random insertion libraries have shown that this method can miss the best sites. Here, we present a computational method for identifying insertion sites for cpGFP in the maltose binding PBP that creates functional biosensors. Our method, which uses protein contact analysis, identifies the best-known insertion regions, as well as additional regions that have not been identified previously. We experimentally characterised biosensors with insertions at these regions and found that 75% of designs were functional. This method is generalisable beyond this system and PBPs more generally, and so could be applied to other binding proteins or effector domains.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567030">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567030" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567030">
        <p class="paperTitle">Strong sequence dependence in RNA/DNA hybrid strand displacement kinetics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567030" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567030" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smith, F. G.; Goertz, J. P.; Stevens, M. M.; Ouldridge, T. E.</p>
        <p class="info">Score: 3.4, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567030' target='https://doi.org/10.1101/2023.11.14.567030'> 10.1101/2023.11.14.567030</a></p>
        <p class="abstract">Strand displacement reactions underlie dynamic nucleic acid nanotechnology. The kinetic and thermodynamic features of DNA-based displacement reactions are well understood and well predicted by current computational models. By contrast, understanding of RNA/DNA hybrid strand displacement kinetics is limited, restricting the design of increasingly complex RNA/DNA hybrid reaction networks with more tightly regulated dynamics. Given the importance of RNA as a diagnostic biomarker, and its critical role in intracellular processes, this shortfall is particularly limiting for the development of strand displacement-based therapeutics and diagnostics. Herein, we characterise 22 RNA/DNA hybrid strand displacement systems, systematically varying several common design parameters including toehold length and branch migration domain length. We observe the differences in stability between RNA-DNA hybrids and DNA-DNA duplexes have large effects on strand displacement rates, with rates for equivalent sequences differing by up to 3 orders of magnitude. Crucially, however, this effect is strongly sequence-dependent, with RNA invaders strongly favoured in a system with RNA strands of high purine content, and disfavoured in a system when the RNA strands have low purine content. These results lay the groundwork for more general design principles, allowing for creation of de novo reaction networks with novel complexity while maintaining predictable reaction kinetics.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
