<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Truly tiny acoustic biomolecules for ultrasound imaging and therapy
Authors: Ling, B.; Gungoren, B.; Yao, Y.; Dutka, P.; Smith, C. A. B.; Lee, J.; Swift, M. B.; Shapiro, M. G.
Score: 15.5, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546773
Nanotechnology offers significant advantages for medical imaging and therapy, including enhanced contrast and precision targeting. However, integrating these benefits into ultrasonography has been challenging due to the size and stability constraints of conventional bubble-based agents.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Truly tiny acoustic biomolecules for ultrasound imaging and therapy
Authors: Ling, B.; Gungoren, B.; Yao, Y.; Dutka, P.; Smith, C. A. B.; Lee, J.; Swift, M. B.; Shapiro, M. G.
Score: 15.5, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546773
Nanotechnology offers significant advantages for medical imaging and therapy, including enhanced contrast and precision targeting. However, integrating these benefits into ultrasonography has been challenging due to the size and stability constraints of conventional bubble-based agents." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-05T10:42:43+00:00" />
<meta property="article:modified_time" content="2023-07-05T10:42:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Truly tiny acoustic biomolecules for ultrasound imaging and therapy
Authors: Ling, B.; Gungoren, B.; Yao, Y.; Dutka, P.; Smith, C. A. B.; Lee, J.; Swift, M. B.; Shapiro, M. G.
Score: 15.5, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546773
Nanotechnology offers significant advantages for medical imaging and therapy, including enhanced contrast and precision targeting. However, integrating these benefits into ultrasonography has been challenging due to the size and stability constraints of conventional bubble-based agents."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Truly tiny acoustic biomolecules for ultrasound imaging and therapy\nAuthors: Ling, B.; Gungoren, B.; Yao, Y.; Dutka, P.; Smith, C. A. B.; Lee, J.; Swift, M. B.; Shapiro, M. G.\nScore: 15.5, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546773\nNanotechnology offers significant advantages for medical imaging and therapy, including enhanced contrast and precision targeting. However, integrating these benefits into ultrasonography has been challenging due to the size and stability constraints of conventional bubble-based agents.",
  "keywords": [
    
  ],
  "articleBody": " Truly tiny acoustic biomolecules for ultrasound imaging and therapy\nAuthors: Ling, B.; Gungoren, B.; Yao, Y.; Dutka, P.; Smith, C. A. B.; Lee, J.; Swift, M. B.; Shapiro, M. G.\nScore: 15.5, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546773\nNanotechnology offers significant advantages for medical imaging and therapy, including enhanced contrast and precision targeting. However, integrating these benefits into ultrasonography has been challenging due to the size and stability constraints of conventional bubble-based agents. Here we describe bicones, truly tiny acoustic contrast agents based on gas vesicles, a unique class of air-filled protein nanostructures naturally produced in buoyant microbes. We show that these sub-80 nm particles can be effectively detected both in vitro and in vivo, infiltrate tumors via leaky vasculature, deliver potent mechanical effects through ultrasound-induced inertial cavitation, and are easily engineered for molecular targeting, prolonged circulation time, and payload conjugation.\nMapping interactions between disordered regions reveals promiscuity in biomolecular condensate formation\nAuthors: Gilat, A.; Dubrueil, B.; Levy, E. D.\nScore: 7.2, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547715\nIntrinsically-disordered regions (IDRs) promote intracellular phase separation and the formation of biomolecular condensates through interactions encoded in their primary sequence. While these condensates form spatially distinct assemblies in cells, it is unclear whether such specificity can be conferred by IDRs alone. Indeed, IDRs exhibit high conformational flexibility whereas specificity in protein recognition is generally associated with well-defined 3D structures. To characterize IDR-IDR interactions and assess their ability to mediate self-specific partitioning, we developed a synthetic system of Multivalent IDRs forming Constitutive DROPlets (micDROP). We investigated ten natural IDRs that underwent phase separation in micDROP. These IDRs exhibited a wide range of saturation concentrations in vivo, which correlated well with their total sequence stickiness. We then probed IDR-IDR specificity by co-expressing pairs of IDRs fused to homologous scaffolds that did not co-assemble. We observed a high degree of promiscuity, particularly among IDRs from the proteins Ddx4, DYRK3, ER, FUS, hnRNPA1, HspB8, RBM14 and TAF15, whereas TDP43 and UBQ2 formed spatially distinct condensates regardless of their partner. Further investigation revealed the short and conserved -helical segment of TDP43's IDR was governing its specific self-recognition. Our findings imply that IDRs can tune their phase separation propensity through sequence composition, while their formation of discrete condensates likely requires additional cellular or structural determinants.\n50-nm gas-filled protein nanostructures to enable the access of lymphatic cells by ultrasound technologies\nAuthors: Shen, Q.; Li, Z.; Meyer, M. D.; De Guzman, M. T.; Lim, J. C.; Bouchard, R. R.; Lu, G. J.\nScore: 6.5, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546433\nUltrasound imaging and ultrasound-mediated gene and drug delivery are rapidly advancing diagnostic and therapeutic methods; however, their use is often limited by the need of microbubbles, which cannot transverse many biological barriers due to their large size. Here we introduce 50-nm gas-filled protein nanostructures derived from genetically engineered gas vesicles that we referred to as 50nmGVs. These diamond-shaped nanostructures have hydrodynamic diameters smaller than commercially available 50-nm gold nanoparticles and are, to our knowledge, the smallest stable, free-floating bubbles made to date. 50nmGVs can be produced in bacteria, purified through centrifugation, and remain stable for months. Interstitially injected 50nmGVs can extravasate into lymphatic tissues and gain access to critical immune cell populations, and electron microscopy images of lymph node tissues reveal their subcellular location in antigen-presenting cells adjacent to lymphocytes. We anticipate that 50nmGVs can substantially broaden the range of cells accessible to current ultrasound technologies and may generate applications beyond biomedicine as ultrasmall stable gas-filled nanomaterials.\nA Platform for Controlled Template-Independent Enzymatic Synthesis of RNA Oligonucleotides and Therapeutics\nAuthors: Wiegand, D. J.; Rittichier, J.; Meyer, E.; Lee, H.; Conway, N. J.; Ahlstedt, D.; Yurtsever, Z.; Rainone, D.; Kuru, E.; Church, G.\nScore: 3.9, Published: 2023-06-29 DOI: 10.1101/2023.06.29.547106\nTherapeutic RNA oligonucleotides have shown tremendous potential to manage and treat disease, yet current manufacturing methods cannot deliver on this promise. Here, we report the development and optimization of a novel, aqueous-based, template-independent enzymatic RNA oligonucleotide synthesis platform as an alternative to traditional chemical methodologies. Our platform is made possible by reversible terminator nucleoside triphosphates and an enzyme capable of their incorporation. We show that many common therapeutic RNA modifications are compatible with our process and demonstrate the enzymatic synthesis of natural and modified oligonucleotides in both liquid and solid phases. Our platform offers many unique advantages over chemical synthesis, including the realization of a more sustainable process to produce therapeutic RNA oligonucleotides. One-Sentence SummaryAn enzyme and novel nucleotide building blocks are used to synthesize RNA oligonucleotides template independently under aqueous conditions.\nEnabling AI in Synthetic Biology through Construction File Specification\nAuthors: Ataii, N.; Bakshi, S.; Chen, Y.; Fernandez, M.; Scheftel, Z.; Shao, Z.; Tou, C. J.; Vega, M.; Wang, Y.; Zhang, H.; Zhao, Z.; Anderson, J. C.\nScore: 2.8, Published: 2023-06-28 DOI: 10.1101/2023.06.28.546630\nThe Construction File (CF) specification establishes a standardized interface for molecular biology operations, laying a foundation for automation and enhanced efficiency in experiment design. It is implemented across three distinct software projects: PyDNA_CF_Simulator, a Python project featuring a ChatGPT plugin for interactive parsing and simulating experiments; ConstructionFileSimulator, a field-tested Java project that showcases Experiment objects expressed as flat files; and C6-Tools, a JavaScript project integrated with Google Sheets via Apps Script, providing a user-friendly interface for authoring and simulation of CF. The CF specification not only standardizes and modularizes molecular biology operations but also promotes collaboration, automation, and reuse, significantly reducing potential errors. The potential integration of CF with artificial intelligence, particularly GPT-4, suggests innovative automation strategies for synthetic biology. While challenges such as token limits, data storage, and biosecurity remain, proposed solutions promise a way forward in harnessing AI for experiment design. This shift from human-driven design to AI-assisted workflows, steered by high-level objectives, charts a potential future path in synthetic biology, envisioning an environment where complexities are managed more effectively.\nAutomated Microbial Classification System based on Deep Convolutional Neural Networks using Images from Colony Picker\nAuthors: Park, S.; Yeoh, J. W.; Choo, C. T.; Lim, C. K.; Dao, V. L.; Poh, C. L.\nScore: 1.5, Published: 2023-06-30 DOI: 10.1101/2023.06.29.547155\nColony screening in single and multi-species environments is an essential step for microbiome studies. However, it possesses a high possibility of inaccurately classifying the species of interest and demands a high degree of manpower and time. Thus, automating the classification of microbes is beneficial to minimize the time and inaccuracy in the colony screening/picking step. Here, we developed an automated microbial classification system for five target species, based on deep Convolutional Neural Networks (CNN) using images captured by an automated robotic colony picker. Multiple possible scenarios of colony culture and diverse morphologies of colonies were examined in building the training and test datasets to train and validate the model and performance on real-life implementations. The final model trained using 60,000 training images, with 12,000 images per species and 3-fold cross-validation, achieved a test accuracy of 94.2% and a test loss of 0.154. Upon testing using a deployment dataset of 4,500 images (900 images per species) with different methods of applying cells onto the agar plate, high accuracy of up to 96.6% was obtained. Five evaluation metrics were implemented to evaluate diverse scenarios of the test data to justify the validity of the model in real-life applications. This model forms a foundation for the classification of more species through transfer learning in the future.\nRepurposing the dark genome. IV - noncoding proteins\nAuthors: Nayak, S.; Dhar, P. K.\nScore: 1.5, Published: 2023-06-29 DOI: 10.1101/2023.06.29.547021\nThe dark genome comprising of non-expressing, non-translating, and extinct DNA sequences has remained a largely unexplored genomic space. Using computational and experimental approaches, novel insights into the dark matter genome have recently been gained, revealing the presence of a vast and unexplored resource. Non-coding RNA (ncRNA) refers to a class of RNA molecules that do not encode proteins but play important regulatory roles in the cell. We asked if it was possible to make functional peptides and proteins from ncRNA leading to a new biological insight and applications? Here we present initial computational data in support of making functional noncoding proteins (NCP) from ncRNA sequences. Different types of non-coding genomic sequences originating from Caenorhabditis elegans, Drosophila melanogaster, Arabidopsis thaliana, and Homo sapiens were studied to understand sequence composition, secondary structure, and physiochemical properties of NCPs. This work builds the foundation for experimentally characterizing the first-in-the-class non-coding proteins leading to a novel insights and applications.\nCRISPR-mediated germline mutagenesis for genetic sterilization of Anopheles gambiae males.\nAuthors: Smidler, A. L.; Paton, D. G.; Church, G. M.; Shaw, W. R.; Catteruccia, F.\nScore: 9.7, Published: 2023-06-13 DOI: 10.1101/2023.06.13.544841\nRapid spread of insecticide resistance among anopheline mosquitoes threatens malaria elimination efforts, necessitating development of alternative vector control technologies. Sterile Insect Technique (SIT) has been successfully implemented in multiple insect pests to suppress field populations by the release of large numbers of sterile males, yet it has proven difficult to adapt to Anopheles vectors. Here we outline adaptation of a CRISPR-based genetic sterilization system to selectively ablate male sperm cells in the malaria mosquito Anopheles gambiae. We achieve robust mosaic biallelic mutagenesis of zero population growth (zpg, a gene essential for differentiation of germ cells) in F1 individuals after intercrossing a germline-expressing Cas9 transgenic line to a line expressing zpg-targeting gRNAs. Approximately 95% of mutagenized males display complete genetic sterilization, and cause similarly high levels of infertility in their female mates. Using a fluorescence reporter that allows detection of the germline leads to a 100% accurate selection of spermless males, improving the system. These males cause a striking reduction in mosquito population size when released at field-like frequencies in competition cages against wild type males. These findings demonstrate that such a genetic system could be adopted for SIT against important malaria vectors.\nA SEVA-based, CRISPR-Cas3-assisted genome engineering approach for Pseudomonas with efficient vector curing\nAuthors: Lammens, E.-M.; Volke, D. C.; Schroven, K.; Voet, M.; Kerremans, A.; Lavigne, R.; Hendrix, H.\nScore: 1.2, Published: 2023-06-29 DOI: 10.1101/2023.06.29.547033\nThe development of CRISPR-Cas-based engineering technologies has revolutionized the microbial biotechnology field. Over the years, the Class II Type II CRISPR-Cas9 system has become the gold standard for genome editing in many bacterial hosts. However, the Cas9 system does not allow efficient genomic integration in Pseudomonas putida, an emerging Synthetic Biology host, without the assistance of lambda-Red recombineering. In this work, we utilize the alternative Class I Type I-C CRISPR-Cas3 system from Pseudomonas aeruginosa as a highly-efficient genome editing tool for P. putida and P. aeruginosa. This system consists of two vectors, one encoding the Cas genes, CRISPR array and targeting spacer, and a second SEVA-vector, containing the homologous repair template. Both vectors are Golden Gate compatible for rapid cloning and are available with multiple antibiotic markers, for application in various Gram-negative hosts and different designs. By employing this Cas3 system, we successfully integrated an 820-bp cassette in the genome of P. putida and performed several genomic deletions in P. aeruginosa within four days, with an efficiency of \u003e83% for both hosts. Moreover, by introducing a universal self-targeting spacer, the Cas3 system rapidly cures all helper vectors, including itself, from the host strain in a matter of days. As such, this system constitutes a valuable engineering tool for Pseudomonas, to complement the existing range of Cas9-based editing methods and facilitates genomic engineering efforts of this important genus. ImportanceThe CRISPR-Cas3 editing system as presented here facilitates the creation of genomic alterations in P. putida and P. aeruginosa in a straightforward manner. By providing the Cas3 system as a vector set with Golden Gate compatibility and different antibiotic markers, as well as by employing the established SEVA vector set to provide the homology repair template, this system is flexible and can readily be ported to a multitude of Gram-negative hosts. Besides genome editing, the Cas3 system can also be used as an effective and universal tool for vector curing. This is achieved by introducing a spacer that targets the oriT, present on the majority of established (SEVA) vectors. Based on this, the Cas3 system efficiently removes up to three vectors in only a few days. As such, this curing approach may also benefit other genomic engineering methods or remove naturally-occurring plasmids from bacteria.\nCRISPR-based gene drives generate super-Mendelian inheritance in the disease vector Culex quinquefasciatus.\nAuthors: Harvey-Samuel, T.; Feng, X.; Okamoto, E. M.; Purusothaman, D.-K.; Leftwich, P. T.; Alphey, L.; Gantz, V. M.\nScore: 25.8, Published: 2023-06-15 DOI: 10.1101/2023.06.12.544656\nCulex mosquitoes pose a significant public health threat as vectors for a variety of diseases including West Nile virus and lymphatic filariasis, and transmit pathogens threatening livestock, companion animals, and endangered birds. Rampant insecticide resistance makes controlling these mosquitoes challenging and necessitates the development of new control strategies. Gene drive technologies have made significant progress in other mosquito species, although similar advances have been lagging in Culex. Here we test the first CRISPR-based homing gene drive for Culex quinquefasciatus, demonstrating the possibility of using this technology to control Culex mosquitoes. Our results show that the inheritance of two split-gene-drive transgenes, targeting different loci, are biased in the presence of a Cas9-expressing transgene although with modest efficiencies. Our findings extend the list of disease vectors where engineered homing gene drives have been demonstrated to include Culex alongside Anopheles and Aedes, and pave the way for future development of these technologies to control Culex mosquitoes.\n",
  "wordCount" : "2201",
  "inLanguage": "en",
  "datePublished": "2023-07-05T10:42:43Z",
  "dateModified": "2023-07-05T10:42:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on July 5, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546773">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546773" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546773">
        <p class="paperTitle">Truly tiny acoustic biomolecules for ultrasound imaging and therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546773" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546773" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ling, B.; Gungoren, B.; Yao, Y.; Dutka, P.; Smith, C. A. B.; Lee, J.; Swift, M. B.; Shapiro, M. G.</p>
        <p class="info">Score: 15.5, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546773' target='https://doi.org/10.1101/2023.06.27.546773'> 10.1101/2023.06.27.546773</a></p>
        <p class="abstract">Nanotechnology offers significant advantages for medical imaging and therapy, including enhanced contrast and precision targeting. However, integrating these benefits into ultrasonography has been challenging due to the size and stability constraints of conventional bubble-based agents. Here we describe bicones, truly tiny acoustic contrast agents based on gas vesicles, a unique class of air-filled protein nanostructures naturally produced in buoyant microbes. We show that these sub-80 nm particles can be effectively detected both in vitro and in vivo, infiltrate tumors via leaky vasculature, deliver potent mechanical effects through ultrasound-induced inertial cavitation, and are easily engineered for molecular targeting, prolonged circulation time, and payload conjugation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547715">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547715" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547715">
        <p class="paperTitle">Mapping interactions between disordered regions reveals promiscuity in biomolecular condensate formation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547715" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547715" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gilat, A.; Dubrueil, B.; Levy, E. D.</p>
        <p class="info">Score: 7.2, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547715' target='https://doi.org/10.1101/2023.07.04.547715'> 10.1101/2023.07.04.547715</a></p>
        <p class="abstract">Intrinsically-disordered regions (IDRs) promote intracellular phase separation and the formation of biomolecular condensates through interactions encoded in their primary sequence. While these condensates form spatially distinct assemblies in cells, it is unclear whether such specificity can be conferred by IDRs alone. Indeed, IDRs exhibit high conformational flexibility whereas specificity in protein recognition is generally associated with well-defined 3D structures. To characterize IDR-IDR interactions and assess their ability to mediate self-specific partitioning, we developed a synthetic system of Multivalent IDRs forming Constitutive DROPlets (micDROP). We investigated ten natural IDRs that underwent phase separation in micDROP. These IDRs exhibited a wide range of saturation concentrations in vivo, which correlated well with their total sequence stickiness. We then probed IDR-IDR specificity by co-expressing pairs of IDRs fused to homologous scaffolds that did not co-assemble. We observed a high degree of promiscuity, particularly among IDRs from the proteins Ddx4, DYRK3, ER, FUS, hnRNPA1, HspB8, RBM14 and TAF15, whereas TDP43 and UBQ2 formed spatially distinct condensates regardless of their partner. Further investigation revealed the short and conserved -helical segment of TDP43&#39;s IDR was governing its specific self-recognition. Our findings imply that IDRs can tune their phase separation propensity through sequence composition, while their formation of discrete condensates likely requires additional cellular or structural determinants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546433">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546433" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546433">
        <p class="paperTitle">50-nm gas-filled protein nanostructures to enable the access of lymphatic cells by ultrasound technologies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546433" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546433" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shen, Q.; Li, Z.; Meyer, M. D.; De Guzman, M. T.; Lim, J. C.; Bouchard, R. R.; Lu, G. J.</p>
        <p class="info">Score: 6.5, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546433' target='https://doi.org/10.1101/2023.06.27.546433'> 10.1101/2023.06.27.546433</a></p>
        <p class="abstract">Ultrasound imaging and ultrasound-mediated gene and drug delivery are rapidly advancing diagnostic and therapeutic methods; however, their use is often limited by the need of microbubbles, which cannot transverse many biological barriers due to their large size. Here we introduce 50-nm gas-filled protein nanostructures derived from genetically engineered gas vesicles that we referred to as 50nmGVs. These diamond-shaped nanostructures have hydrodynamic diameters smaller than commercially available 50-nm gold nanoparticles and are, to our knowledge, the smallest stable, free-floating bubbles made to date. 50nmGVs can be produced in bacteria, purified through centrifugation, and remain stable for months. Interstitially injected 50nmGVs can extravasate into lymphatic tissues and gain access to critical immune cell populations, and electron microscopy images of lymph node tissues reveal their subcellular location in antigen-presenting cells adjacent to lymphocytes. We anticipate that 50nmGVs can substantially broaden the range of cells accessible to current ultrasound technologies and may generate applications beyond biomedicine as ultrasmall stable gas-filled nanomaterials.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547106">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547106" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547106">
        <p class="paperTitle">A Platform for Controlled Template-Independent Enzymatic Synthesis of RNA Oligonucleotides and Therapeutics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547106" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547106" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wiegand, D. J.; Rittichier, J.; Meyer, E.; Lee, H.; Conway, N. J.; Ahlstedt, D.; Yurtsever, Z.; Rainone, D.; Kuru, E.; Church, G.</p>
        <p class="info">Score: 3.9, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547106' target='https://doi.org/10.1101/2023.06.29.547106'> 10.1101/2023.06.29.547106</a></p>
        <p class="abstract">Therapeutic RNA oligonucleotides have shown tremendous potential to manage and treat disease, yet current manufacturing methods cannot deliver on this promise. Here, we report the development and optimization of a novel, aqueous-based, template-independent enzymatic RNA oligonucleotide synthesis platform as an alternative to traditional chemical methodologies. Our platform is made possible by reversible terminator nucleoside triphosphates and an enzyme capable of their incorporation. We show that many common therapeutic RNA modifications are compatible with our process and demonstrate the enzymatic synthesis of natural and modified oligonucleotides in both liquid and solid phases. Our platform offers many unique advantages over chemical synthesis, including the realization of a more sustainable process to produce therapeutic RNA oligonucleotides.

One-Sentence SummaryAn enzyme and novel nucleotide building blocks are used to synthesize RNA oligonucleotides template independently under aqueous conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546630">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546630" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546630">
        <p class="paperTitle">Enabling AI in Synthetic Biology through Construction File Specification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546630" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546630" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ataii, N.; Bakshi, S.; Chen, Y.; Fernandez, M.; Scheftel, Z.; Shao, Z.; Tou, C. J.; Vega, M.; Wang, Y.; Zhang, H.; Zhao, Z.; Anderson, J. C.</p>
        <p class="info">Score: 2.8, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546630' target='https://doi.org/10.1101/2023.06.28.546630'> 10.1101/2023.06.28.546630</a></p>
        <p class="abstract">The Construction File (CF) specification establishes a standardized interface for molecular biology operations, laying a foundation for automation and enhanced efficiency in experiment design. It is implemented across three distinct software projects: PyDNA_CF_Simulator, a Python project featuring a ChatGPT plugin for interactive parsing and simulating experiments; ConstructionFileSimulator, a field-tested Java project that showcases  Experiment objects expressed as flat files; and C6-Tools, a JavaScript project integrated with Google Sheets via Apps Script, providing a user-friendly interface for authoring and simulation of CF. The CF specification not only standardizes and modularizes molecular biology operations but also promotes collaboration, automation, and reuse, significantly reducing potential errors. The potential integration of CF with artificial intelligence, particularly GPT-4, suggests innovative automation strategies for synthetic biology. While challenges such as token limits, data storage, and biosecurity remain, proposed solutions promise a way forward in harnessing AI for experiment design. This shift from human-driven design to AI-assisted workflows, steered by high-level objectives, charts a potential future path in synthetic biology, envisioning an environment where complexities are managed more effectively.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547155">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547155" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547155">
        <p class="paperTitle">Automated Microbial Classification System based on Deep Convolutional Neural Networks using Images from Colony Picker</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547155" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547155" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Park, S.; Yeoh, J. W.; Choo, C. T.; Lim, C. K.; Dao, V. L.; Poh, C. L.</p>
        <p class="info">Score: 1.5, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547155' target='https://doi.org/10.1101/2023.06.29.547155'> 10.1101/2023.06.29.547155</a></p>
        <p class="abstract">Colony screening in single and multi-species environments is an essential step for microbiome studies. However, it possesses a high possibility of inaccurately classifying the species of interest and demands a high degree of manpower and time. Thus, automating the classification of microbes is beneficial to minimize the time and inaccuracy in the colony screening/picking step. Here, we developed an automated microbial classification system for five target species, based on deep Convolutional Neural Networks (CNN) using images captured by an automated robotic colony picker. Multiple possible scenarios of colony culture and diverse morphologies of colonies were examined in building the training and test datasets to train and validate the model and performance on real-life implementations. The final model trained using 60,000 training images, with 12,000 images per species and 3-fold cross-validation, achieved a test accuracy of 94.2% and a test loss of 0.154. Upon testing using a deployment dataset of 4,500 images (900 images per species) with different methods of applying cells onto the agar plate, high accuracy of up to 96.6% was obtained. Five evaluation metrics were implemented to evaluate diverse scenarios of the test data to justify the validity of the model in real-life applications. This model forms a foundation for the classification of more species through transfer learning in the future.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547021">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547021" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547021">
        <p class="paperTitle">Repurposing the dark genome. IV - noncoding proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547021" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547021" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nayak, S.; Dhar, P. K.</p>
        <p class="info">Score: 1.5, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547021' target='https://doi.org/10.1101/2023.06.29.547021'> 10.1101/2023.06.29.547021</a></p>
        <p class="abstract">The dark genome comprising of non-expressing, non-translating, and extinct DNA sequences has remained a largely unexplored genomic space. Using computational and experimental approaches, novel insights into the dark matter genome have recently been gained, revealing the presence of a vast and unexplored resource. Non-coding RNA (ncRNA) refers to a class of RNA molecules that do not encode proteins but play important regulatory roles in the cell. We asked if it was possible to make functional peptides and proteins from ncRNA leading to a new biological insight and applications? Here we present initial computational data in support of making functional noncoding proteins (NCP) from ncRNA sequences. Different types of non-coding genomic sequences originating from Caenorhabditis elegans, Drosophila melanogaster, Arabidopsis thaliana, and Homo sapiens were studied to understand sequence composition, secondary structure, and physiochemical properties of NCPs. This work builds the foundation for experimentally characterizing the first-in-the-class non-coding proteins leading to a novel insights and applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.13.544841">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.13.544841" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.13.544841">
        <p class="paperTitle">CRISPR-mediated germline mutagenesis for genetic sterilization of Anopheles gambiae males.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.13.544841" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.13.544841" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smidler, A. L.; Paton, D. G.; Church, G. M.; Shaw, W. R.; Catteruccia, F.</p>
        <p class="info">Score: 9.7, Published: 2023-06-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.13.544841' target='https://doi.org/10.1101/2023.06.13.544841'> 10.1101/2023.06.13.544841</a></p>
        <p class="abstract">Rapid spread of insecticide resistance among anopheline mosquitoes threatens malaria elimination efforts, necessitating development of alternative vector control technologies. Sterile Insect Technique (SIT) has been successfully implemented in multiple insect pests to suppress field populations by the release of large numbers of sterile males, yet it has proven difficult to adapt to Anopheles vectors. Here we outline adaptation of a CRISPR-based genetic sterilization system to selectively ablate male sperm cells in the malaria mosquito Anopheles gambiae. We achieve robust mosaic biallelic mutagenesis of zero population growth (zpg, a gene essential for differentiation of germ cells) in F1 individuals after intercrossing a germline-expressing Cas9 transgenic line to a line expressing zpg-targeting gRNAs. Approximately 95% of mutagenized males display complete genetic sterilization, and cause similarly high levels of infertility in their female mates. Using a fluorescence reporter that allows detection of the germline leads to a 100% accurate selection of spermless males, improving the system. These males cause a striking reduction in mosquito population size when released at field-like frequencies in competition cages against wild type males. These findings demonstrate that such a genetic system could be adopted for SIT against important malaria vectors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547033">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547033" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547033">
        <p class="paperTitle">A SEVA-based, CRISPR-Cas3-assisted genome engineering approach for Pseudomonas with efficient vector curing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547033" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547033" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lammens, E.-M.; Volke, D. C.; Schroven, K.; Voet, M.; Kerremans, A.; Lavigne, R.; Hendrix, H.</p>
        <p class="info">Score: 1.2, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547033' target='https://doi.org/10.1101/2023.06.29.547033'> 10.1101/2023.06.29.547033</a></p>
        <p class="abstract">The development of CRISPR-Cas-based engineering technologies has revolutionized the microbial biotechnology field. Over the years, the Class II Type II CRISPR-Cas9 system has become the gold standard for genome editing in many bacterial hosts. However, the Cas9 system does not allow efficient genomic integration in Pseudomonas putida, an emerging Synthetic Biology host, without the assistance of lambda-Red recombineering. In this work, we utilize the alternative Class I Type I-C CRISPR-Cas3 system from Pseudomonas aeruginosa as a highly-efficient genome editing tool for P. putida and P. aeruginosa. This system consists of two vectors, one encoding the Cas genes, CRISPR array and targeting spacer, and a second SEVA-vector, containing the homologous repair template. Both vectors are Golden Gate compatible for rapid cloning and are available with multiple antibiotic markers, for application in various Gram-negative hosts and different designs. By employing this Cas3 system, we successfully integrated an 820-bp cassette in the genome of P. putida and performed several genomic deletions in P. aeruginosa within four days, with an efficiency of &gt;83% for both hosts. Moreover, by introducing a universal self-targeting spacer, the Cas3 system rapidly cures all helper vectors, including itself, from the host strain in a matter of days. As such, this system constitutes a valuable engineering tool for Pseudomonas, to complement the existing range of Cas9-based editing methods and facilitates genomic engineering efforts of this important genus.

ImportanceThe CRISPR-Cas3 editing system as presented here facilitates the creation of genomic alterations in P. putida and P. aeruginosa in a straightforward manner. By providing the Cas3 system as a vector set with Golden Gate compatibility and different antibiotic markers, as well as by employing the established SEVA vector set to provide the homology repair template, this system is flexible and can readily be ported to a multitude of Gram-negative hosts. Besides genome editing, the Cas3 system can also be used as an effective and universal tool for vector curing. This is achieved by introducing a spacer that targets the oriT, present on the majority of established (SEVA) vectors. Based on this, the Cas3 system efficiently removes up to three vectors in only a few days. As such, this curing approach may also benefit other genomic engineering methods or remove naturally-occurring plasmids from bacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.12.544656">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.12.544656" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.12.544656">
        <p class="paperTitle">CRISPR-based gene drives generate super-Mendelian inheritance in the disease vector Culex quinquefasciatus.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.12.544656" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.12.544656" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harvey-Samuel, T.; Feng, X.; Okamoto, E. M.; Purusothaman, D.-K.; Leftwich, P. T.; Alphey, L.; Gantz, V. M.</p>
        <p class="info">Score: 25.8, Published: 2023-06-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.12.544656' target='https://doi.org/10.1101/2023.06.12.544656'> 10.1101/2023.06.12.544656</a></p>
        <p class="abstract">Culex mosquitoes pose a significant public health threat as vectors for a variety of diseases including West Nile virus and lymphatic filariasis, and transmit pathogens threatening livestock, companion animals, and endangered birds. Rampant insecticide resistance makes controlling these mosquitoes challenging and necessitates the development of new control strategies. Gene drive technologies have made significant progress in other mosquito species, although similar advances have been lagging in Culex. Here we test the first CRISPR-based homing gene drive for Culex quinquefasciatus, demonstrating the possibility of using this technology to control Culex mosquitoes. Our results show that the inheritance of two split-gene-drive transgenes, targeting different loci, are biased in the presence of a Cas9-expressing transgene although with modest efficiencies. Our findings extend the list of disease vectors where engineered homing gene drives have been demonstrated to include Culex alongside Anopheles and Aedes, and pave the way for future development of these technologies to control Culex mosquitoes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
