<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein
Authors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Gong, J.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.
Score: 98.1, Published: 2023-07-14 DOI: 10.1101/2023.07.05.547496
Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein
Authors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Gong, J.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.
Score: 98.1, Published: 2023-07-14 DOI: 10.1101/2023.07.05.547496
Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-26T10:37:30+00:00" />
<meta property="article:modified_time" content="2023-07-26T10:37:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein
Authors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Gong, J.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.
Score: 98.1, Published: 2023-07-14 DOI: 10.1101/2023.07.05.547496
Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein\nAuthors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Gong, J.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.\nScore: 98.1, Published: 2023-07-14 DOI: 10.1101/2023.07.05.547496\nProtein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently.",
  "keywords": [
    
  ],
  "articleBody": " xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein\nAuthors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Gong, J.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.\nScore: 98.1, Published: 2023-07-14 DOI: 10.1101/2023.07.05.547496\nProtein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently. This paper proposes a unified protein language model, xTrimoPGLM, to address these two types of tasks simultaneously through an innovative pre-training framework. Our key technical contribution is an exploration of the compatibility and the potential for joint optimization of the two types of objectives, which has led to a strategy for training xTrimoPGLM at an unprecedented scale of 100 billion parameters and 1 trillion training tokens. Our extensive experiments reveal that xTrimoPGLM significantly outperforms other advanced baselines in diverse protein understanding tasks (13 out of 15 tasks across four categories) and generates novel protein sequences which are structurally similar to natural ones. Furthermore, using the same xTrimoPGLM framework, we train an antibody-specific model (xTrimoPGLM-Ab) using 1 billion parameters. This model set a new record in predicting antibody naturalness and structures, both essential to the field of antibody-based drug design, and demonstrated a significantly faster inference speed than AlphaFold2. These results highlight the substantial capability and versatility of xTrimoPGLM in understanding and generating protein sequences.\nMicrobial biosensor for sensing and treatment of intestinal inflammation\nAuthors: Zhu, D.; Galley, J. D.; Pizzini, J.; Musteata, E.; Tabor, J. J.; Britton, R. A.\nScore: 11.5, Published: 2023-07-22 DOI: 10.1101/2023.07.21.550106\nSubstantial synthetic biology efforts have been made to engineer biosensors to detect intestinal inflammation, however none target the most clinically accepted biomarker, calprotectin. To develop an in situ biosensor for calprotectin, we optimized a zinc uptake regulator (Zur) regulated promoter coupled with a memory circuit that can detect and record intestinal inflammation in vivo. The level of activation strongly correlates with calprotectin levels in the colon of two independent mouse models of colitis. Coupling of the biosensor with the production of the anti-inflammatory cytokine IL-10 allowed for the resolution of chemically induced colitis, demonstrating the ability of the biosensor to sense and respond to disease. This work highlights the utility of developing synthetic organisms for the diagnosis and treatment of intestinal disease using clinically validated biomarkers. One sentence summaryWe have optimized a microbial biosensor to detect and respond to the clinically relevant intestinal inflammation biomarker calprotectin.\nElucidating the assembly of gas vesicles by systematic protein-protein interaction analysis\nAuthors: Iburg, M.; Anderson, A. P.; Wong, V. T.; Anton, E. D.; He, A.; Lu, G. J.\nScore: 6.8, Published: 2023-07-23 DOI: 10.1101/2023.07.23.550228\nGas vesicles (GVs) are gas-filled microbial organelles formed by unique 3-nm thick, amphipathic, force-bearing protein shells, which can withstand multiple atmospheric pressures and maintain a physically stable air bubble with megapascal surface tension. However, the molecular process to assemble this shell remains elusive: while 6-8 assembly factor proteins were identified as essential, none of them have a defined function. As one of the first steps to elucidate the assembly mechanism, we devise a high-throughput in vivo assay to determine the interactions of all 11 proteins in a GV operon. Complete or partial deletions of the operon establish the interdependence relationship of the interaction on the background GV proteins with additional information on assembly tolerance and cellular burden. Clusters of GV protein interactions are revealed, which establishes the plausible protein complexes important for the assembly process of these protein organelles. We anticipate our findings will set the stage for solving the molecular mechanism of GV assembly and designing GVs that efficiently assemble in heterologous hosts during biomedical applications.\nReductive amination cascades in cell-free and resting whole cell formats for valorization of lignin deconstruction products\nAuthors: Nain, P.; Dickey, R.; Somasundaram, V.; Sulzbach, M.; Kunjapur, A. M.\nScore: 4.5, Published: 2023-07-21 DOI: 10.1101/2023.07.21.550087\nThe selective introduction of amine groups within deconstruction products of lignin could provide an avenue for valorizing waste biomass while achieving a green synthesis of industrially relevant building blocks from sustainable sources. Here, we built and characterized enzyme cascades that create aldehydes and subsequently primary amines from diverse lignin-derived carboxylic acids using a carboxylic acid reductase (CAR) and an {omega}-transaminase (TA). We report a detailed characterization of two reaction formats - cell-free biocatalysis and resting whole-cell biocatalysis - using the conversion of vanillate to vanillyl amine as model chemistry. We showed that resting whole-cell reactions could be competitive with cell-free reactions (97% and 70% yields, respectively) while avoiding steps of cell lysis, purification, and cofactor supplementation. We also demonstrated the recyclability of resting whole cells for multiple batches of catalysis without significant loss in productivity. Finally, we used the knowledge gained from this study to produce several amines from carboxylic acid precursors using one-pot biocatalytic reactions. These results expand our knowledge of these industrially relevant enzyme families to new substrates and contexts for environmentally friendly and potentially low-cost synthesis of diverse aryl aldehydes and amines.\nA SEVA-based, CRISPR-Cas3-assisted genome engineering approach for Pseudomonas with efficient vector curing\nAuthors: Lammens, E.-M.; Volke, D. C.; Schroven, K.; Voet, M.; Kerremans, A.; Lavigne, R.; Hendrix, H.\nScore: 4.5, Published: 2023-06-29 DOI: 10.1101/2023.06.29.547033\nThe development of CRISPR-Cas-based engineering technologies has revolutionized the microbial biotechnology field. Over the years, the Class II Type II CRISPR-Cas9 system has become the gold standard for genome editing in many bacterial hosts. However, the Cas9 system does not allow efficient genomic integration in Pseudomonas putida, an emerging Synthetic Biology host, without the assistance of lambda-Red recombineering. In this work, we utilize the alternative Class I Type I-C CRISPR-Cas3 system from Pseudomonas aeruginosa as a highly-efficient genome editing tool for P. putida and P. aeruginosa. This system consists of two vectors, one encoding the Cas genes, CRISPR array and targeting spacer, and a second SEVA-vector, containing the homologous repair template. Both vectors are Golden Gate compatible for rapid cloning and are available with multiple antibiotic markers, for application in various Gram-negative hosts and different designs. By employing this Cas3 system, we successfully integrated an 820-bp cassette in the genome of P. putida and performed several genomic deletions in P. aeruginosa within four days, with an efficiency of \u003e83% for both hosts. Moreover, by introducing a universal self-targeting spacer, the Cas3 system rapidly cures all helper vectors, including itself, from the host strain in a matter of days. As such, this system constitutes a valuable engineering tool for Pseudomonas, to complement the existing range of Cas9-based editing methods and facilitates genomic engineering efforts of this important genus. ImportanceThe CRISPR-Cas3 editing system as presented here facilitates the creation of genomic alterations in P. putida and P. aeruginosa in a straightforward manner. By providing the Cas3 system as a vector set with Golden Gate compatibility and different antibiotic markers, as well as by employing the established SEVA vector set to provide the homology repair template, this system is flexible and can readily be ported to a multitude of Gram-negative hosts. Besides genome editing, the Cas3 system can also be used as an effective and universal tool for vector curing. This is achieved by introducing a spacer that targets the oriT, present on the majority of established (SEVA) vectors. Based on this, the Cas3 system efficiently removes up to three vectors in only a few days. As such, this curing approach may also benefit other genomic engineering methods or remove naturally-occurring plasmids from bacteria.\nGenerative and predictive neural networks for the design of functional RNA molecules\nAuthors: Riley, A. T.; Robson, J. M.; Green, A. A.\nScore: 14.2, Published: 2023-07-14 DOI: 10.1101/2023.07.14.549043\nRNA is a remarkably versatile molecule that has been engineered for applications in therapeutics, diagnostics, and in vivo information-processing systems. However, the complex relationship between the sequence and structural properties of an RNA molecule and its ability to perform specific functions often necessitates extensive experimental screening of candidate sequences. Here we present a generalized neural network architecture that utilizes the sequence and structure of RNA molecules (SANDSTORM) to inform functional predictions. We demonstrate that this approach achieves state-of-the-art performance across several distinct RNA prediction tasks, while learning interpretable abstractions of RNA secondary structure. We paired these predictive models with generative adversarial RNA design networks (GARDN), allowing the generative modelling of novel mRNA 5 untranslated regions and toehold switch riboregulators exhibiting a predetermined fitness. This approach enabled the design of novel toehold switches with a 43-fold increase in experimentally characterized dynamic range compared to those designed using classic thermodynamic algorithms. SANDSTORM and GARDN thus represent powerful new predictive and generative tools for the development of diagnostic and therapeutic RNA molecules with improved function.\nMapping interactions between disordered regions reveals promiscuity in biomolecular condensate formation\nAuthors: Gilat, A.; Dubrueil, B.; Levy, E. D.\nScore: 16.0, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547715\nIntrinsically-disordered regions (IDRs) promote intracellular phase separation and the formation of biomolecular condensates through interactions encoded in their primary sequence. While these condensates form spatially distinct assemblies in cells, it is unclear whether such specificity can be conferred by IDRs alone. Indeed, IDRs exhibit high conformational flexibility whereas specificity in protein recognition is generally associated with well-defined 3D structures. To characterize IDR-IDR interactions and assess their ability to mediate self-specific partitioning, we developed a synthetic system of Multivalent IDRs forming Constitutive DROPlets (micDROP). We investigated ten natural IDRs that underwent phase separation in micDROP. These IDRs exhibited a wide range of saturation concentrations in vivo, which correlated well with their total sequence stickiness. We then probed IDR-IDR specificity by co-expressing pairs of IDRs fused to homologous scaffolds that did not co-assemble. We observed a high degree of promiscuity, particularly among IDRs from the proteins Ddx4, DYRK3, ER, FUS, hnRNPA1, HspB8, RBM14 and TAF15, whereas TDP43 and UBQ2 formed spatially distinct condensates regardless of their partner. Further investigation revealed the short and conserved -helical segment of TDP43s IDR was governing its specific self-recognition. Our findings imply that IDRs can tune their phase separation propensity through sequence composition, while their formation of discrete condensates likely requires additional cellular or structural determinants. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=200 SRC=\"FIGDIR/small/547715v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (60K): org.highwire.dtl.DTLVardef@b72917org.highwire.dtl.DTLVardef@eaafa1org.highwire.dtl.DTLVardef@1b0758corg.highwire.dtl.DTLVardef@1f56126_HPS_FORMAT_FIGEXP M_FIG C_FIG\nBRAIDing receptors for cell specific targeting\nAuthors: Chen, H.; Lee, S.-J.; Li, R.; Sura, A.; Suen, N.; Dilip, A.; Pomogov, Y.; Vuppalapaty, M.; Lu, C.; Post, Y.; Li, Y.\nScore: 1.8, Published: 2023-07-18 DOI: 10.1101/2023.07.18.549566\nSystemic toxicity is a major challenge to therapeutic development, consequently, cell-type specific targeting is highly desirable to improve on-target cell efficacy while reducing off-target toxicity. Here, we describe a cell targeting system we called BRAID (BRidged Activation by Intra/intermolecular Division) where an active molecule is divided into two inactive or less active parts that are subsequently brought together via a bridging receptor on target cell. We tested this hypothesis using WNT/{beta}-catenin signaling system and demonstrated that a multivalent WNT agonist molecule divided into two inactive components induced signaling specifically on hepatocytes assembled via two different epitopes on a hepatocyte receptor, {beta}Klotho. These data provided proof-of-concept for a cell specific targeting approach and also demonstrated the feasibility of combining different signaling pathways where desirable. This approach has broad applications to other receptor systems.\nA Platform for Controlled Template-Independent Enzymatic Synthesis of RNA Oligonucleotides and Therapeutics\nAuthors: Wiegand, D. J.; Rittichier, J.; Meyer, E.; Lee, H.; Conway, N. J.; Ahlstedt, D.; Yurtsever, Z.; Rainone, D.; Kuru, E.; Church, G.\nScore: 16.3, Published: 2023-06-29 DOI: 10.1101/2023.06.29.547106\nTherapeutic RNA oligonucleotides have shown tremendous potential to manage and treat disease, yet current manufacturing methods cannot deliver on this promise. Here, we report the development and optimization of a novel, aqueous-based, template-independent enzymatic RNA oligonucleotide synthesis platform as an alternative to traditional chemical methodologies. Our platform is made possible by reversible terminator nucleoside triphosphates and an enzyme capable of their incorporation. We show that many common therapeutic RNA modifications are compatible with our process and demonstrate the enzymatic synthesis of natural and modified oligonucleotides in both liquid and solid phases. Our platform offers many unique advantages over chemical synthesis, including the realization of a more sustainable process to produce therapeutic RNA oligonucleotides. One-Sentence SummaryAn enzyme and novel nucleotide building blocks are used to synthesize RNA oligonucleotides template independently under aqueous conditions.\nCharacterization and diversification of AraC/XylS family regulators guided by transposon sequencing\nAuthors: Pearson, A. N.; Incha, M. R.; Ho, C.; Schmidt, M.; Roberts, J. B.; Nava, A. A.; Keasling, J. D.\nScore: 1.0, Published: 2023-07-22 DOI: 10.1101/2023.07.21.550116\nIn this study, we explored the development of engineered inducible systems. Publicly available data from previous transposon sequencing assays were used to identify regulators of metabolism in Pseudomonas putida KT2440. For the AraC-family regulators (AFRs) represented in this data, we posited AFR/promoter/inducer groupings. Eleven promoters were characterized for a response to their proposed inducers in P. putida, and the resultant data were used to create and test nine two-plasmid sensor systems in E. coli. Several of these were further developed into a palette of single-plasmid inducible systems. From these experiments, we observed an unreported inducer response from a previously characterized AFR, demonstrated that the addition of a P. putida transporter improved the sensor dynamics of an AFR in E. coli, and identified an uncharacterized AFR with a novel potential inducer specificity. Finally, targeted mutations in an AFR, informed by structural predictions, enabled further diversification of these inducible plasmids. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=105 SRC=\"FIGDIR/small/550116v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (35K): org.highwire.dtl.DTLVardef@1ffc892org.highwire.dtl.DTLVardef@1c91a7borg.highwire.dtl.DTLVardef@1e98a5corg.highwire.dtl.DTLVardef@17ab383_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "2236",
  "inLanguage": "en",
  "datePublished": "2023-07-26T10:37:30Z",
  "dateModified": "2023-07-26T10:37:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on July 26, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547496">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547496" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547496">
        <p class="paperTitle">xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547496" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547496" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Gong, J.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.</p>
        <p class="info">Score: 98.1, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547496' target='https://doi.org/10.1101/2023.07.05.547496'> 10.1101/2023.07.05.547496</a></p>
        <p class="abstract">Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently. This paper proposes a unified protein language model, xTrimoPGLM, to address these two types of tasks simultaneously through an innovative pre-training framework. Our key technical contribution is an exploration of the compatibility and the potential for joint optimization of the two types of objectives, which has led to a strategy for training xTrimoPGLM at an unprecedented scale of 100 billion parameters and 1 trillion training tokens. Our extensive experiments reveal that xTrimoPGLM significantly outperforms other advanced baselines in diverse protein understanding tasks (13 out of 15 tasks across four categories) and generates novel protein sequences which are structurally similar to natural ones. Furthermore, using the same xTrimoPGLM framework, we train an antibody-specific model (xTrimoPGLM-Ab) using 1 billion parameters. This model set a new record in predicting antibody naturalness and structures, both essential to the field of antibody-based drug design, and demonstrated a significantly faster inference speed than AlphaFold2. These results highlight the substantial capability and versatility of xTrimoPGLM in understanding and generating protein sequences.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.550106">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.550106" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.550106">
        <p class="paperTitle">Microbial biosensor for sensing and treatment of intestinal inflammation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.550106" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.550106" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhu, D.; Galley, J. D.; Pizzini, J.; Musteata, E.; Tabor, J. J.; Britton, R. A.</p>
        <p class="info">Score: 11.5, Published: 2023-07-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.550106' target='https://doi.org/10.1101/2023.07.21.550106'> 10.1101/2023.07.21.550106</a></p>
        <p class="abstract">Substantial synthetic biology efforts have been made to engineer biosensors to detect intestinal inflammation, however none target the most clinically accepted biomarker, calprotectin. To develop an in situ biosensor for calprotectin, we optimized a zinc uptake regulator (Zur) regulated promoter coupled with a memory circuit that can detect and record intestinal inflammation in vivo. The level of activation strongly correlates with calprotectin levels in the colon of two independent mouse models of colitis. Coupling of the biosensor with the production of the anti-inflammatory cytokine IL-10 allowed for the resolution of chemically induced colitis, demonstrating the ability of the biosensor to sense and respond to disease. This work highlights the utility of developing synthetic organisms for the diagnosis and treatment of intestinal disease using clinically validated biomarkers.

One sentence summaryWe have optimized a microbial biosensor to detect and respond to the clinically relevant intestinal inflammation biomarker calprotectin.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.23.550228">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.23.550228" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.23.550228">
        <p class="paperTitle">Elucidating the assembly of gas vesicles by systematic protein-protein interaction analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.23.550228" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.23.550228" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Iburg, M.; Anderson, A. P.; Wong, V. T.; Anton, E. D.; He, A.; Lu, G. J.</p>
        <p class="info">Score: 6.8, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.23.550228' target='https://doi.org/10.1101/2023.07.23.550228'> 10.1101/2023.07.23.550228</a></p>
        <p class="abstract">Gas vesicles (GVs) are gas-filled microbial organelles formed by unique 3-nm thick, amphipathic, force-bearing protein shells, which can withstand multiple atmospheric pressures and maintain a physically stable air bubble with megapascal surface tension. However, the molecular process to assemble this shell remains elusive: while 6-8 assembly factor proteins were identified as essential, none of them have a defined function. As one of the first steps to elucidate the assembly mechanism, we devise a high-throughput in vivo assay to determine the interactions of all 11 proteins in a GV operon. Complete or partial deletions of the operon establish the interdependence relationship of the interaction on the background GV proteins with additional information on assembly tolerance and cellular burden. Clusters of GV protein interactions are revealed, which establishes the plausible protein complexes important for the assembly process of these protein organelles. We anticipate our findings will set the stage for solving the molecular mechanism of GV assembly and designing GVs that efficiently assemble in heterologous hosts during biomedical applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.550087">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.550087" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.550087">
        <p class="paperTitle">Reductive amination cascades in cell-free and resting whole cell formats for valorization of lignin deconstruction products</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.550087" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.550087" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nain, P.; Dickey, R.; Somasundaram, V.; Sulzbach, M.; Kunjapur, A. M.</p>
        <p class="info">Score: 4.5, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.550087' target='https://doi.org/10.1101/2023.07.21.550087'> 10.1101/2023.07.21.550087</a></p>
        <p class="abstract">The selective introduction of amine groups within deconstruction products of lignin could provide an avenue for valorizing waste biomass while achieving a green synthesis of industrially relevant building blocks from sustainable sources. Here, we built and characterized enzyme cascades that create aldehydes and subsequently primary amines from diverse lignin-derived carboxylic acids using a carboxylic acid reductase (CAR) and an {omega}-transaminase (TA). We report a detailed characterization of two reaction formats - cell-free biocatalysis and resting whole-cell biocatalysis - using the conversion of vanillate to vanillyl amine as model chemistry. We showed that resting whole-cell reactions could be competitive with cell-free reactions (97% and 70% yields, respectively) while avoiding steps of cell lysis, purification, and cofactor supplementation. We also demonstrated the recyclability of resting whole cells for multiple batches of catalysis without significant loss in productivity. Finally, we used the knowledge gained from this study to produce several amines from carboxylic acid precursors using one-pot biocatalytic reactions. These results expand our knowledge of these industrially relevant enzyme families to new substrates and contexts for environmentally friendly and potentially low-cost synthesis of diverse aryl aldehydes and amines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547033">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547033" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547033">
        <p class="paperTitle">A SEVA-based, CRISPR-Cas3-assisted genome engineering approach for Pseudomonas with efficient vector curing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547033" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547033" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lammens, E.-M.; Volke, D. C.; Schroven, K.; Voet, M.; Kerremans, A.; Lavigne, R.; Hendrix, H.</p>
        <p class="info">Score: 4.5, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547033' target='https://doi.org/10.1101/2023.06.29.547033'> 10.1101/2023.06.29.547033</a></p>
        <p class="abstract">The development of CRISPR-Cas-based engineering technologies has revolutionized the microbial biotechnology field. Over the years, the Class II Type II CRISPR-Cas9 system has become the gold standard for genome editing in many bacterial hosts. However, the Cas9 system does not allow efficient genomic integration in Pseudomonas putida, an emerging Synthetic Biology host, without the assistance of lambda-Red recombineering. In this work, we utilize the alternative Class I Type I-C CRISPR-Cas3 system from Pseudomonas aeruginosa as a highly-efficient genome editing tool for P. putida and P. aeruginosa. This system consists of two vectors, one encoding the Cas genes, CRISPR array and targeting spacer, and a second SEVA-vector, containing the homologous repair template. Both vectors are Golden Gate compatible for rapid cloning and are available with multiple antibiotic markers, for application in various Gram-negative hosts and different designs. By employing this Cas3 system, we successfully integrated an 820-bp cassette in the genome of P. putida and performed several genomic deletions in P. aeruginosa within four days, with an efficiency of &gt;83% for both hosts. Moreover, by introducing a universal self-targeting spacer, the Cas3 system rapidly cures all helper vectors, including itself, from the host strain in a matter of days. As such, this system constitutes a valuable engineering tool for Pseudomonas, to complement the existing range of Cas9-based editing methods and facilitates genomic engineering efforts of this important genus.

ImportanceThe CRISPR-Cas3 editing system as presented here facilitates the creation of genomic alterations in P. putida and P. aeruginosa in a straightforward manner. By providing the Cas3 system as a vector set with Golden Gate compatibility and different antibiotic markers, as well as by employing the established SEVA vector set to provide the homology repair template, this system is flexible and can readily be ported to a multitude of Gram-negative hosts. Besides genome editing, the Cas3 system can also be used as an effective and universal tool for vector curing. This is achieved by introducing a spacer that targets the oriT, present on the majority of established (SEVA) vectors. Based on this, the Cas3 system efficiently removes up to three vectors in only a few days. As such, this curing approach may also benefit other genomic engineering methods or remove naturally-occurring plasmids from bacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.549043">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.549043" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.549043">
        <p class="paperTitle">Generative and predictive neural networks for the design of functional RNA molecules</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.549043" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.549043" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Riley, A. T.; Robson, J. M.; Green, A. A.</p>
        <p class="info">Score: 14.2, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.549043' target='https://doi.org/10.1101/2023.07.14.549043'> 10.1101/2023.07.14.549043</a></p>
        <p class="abstract">RNA is a remarkably versatile molecule that has been engineered for applications in therapeutics, diagnostics, and in vivo information-processing systems. However, the complex relationship between the sequence and structural properties of an RNA molecule and its ability to perform specific functions often necessitates extensive experimental screening of candidate sequences. Here we present a generalized neural network architecture that utilizes the sequence and structure of RNA molecules (SANDSTORM) to inform functional predictions. We demonstrate that this approach achieves state-of-the-art performance across several distinct RNA prediction tasks, while learning interpretable abstractions of RNA secondary structure. We paired these predictive models with generative adversarial RNA design networks (GARDN), allowing the generative modelling of novel mRNA 5 untranslated regions and toehold switch riboregulators exhibiting a predetermined fitness. This approach enabled the design of novel toehold switches with a 43-fold increase in experimentally characterized dynamic range compared to those designed using classic thermodynamic algorithms. SANDSTORM and GARDN thus represent powerful new predictive and generative tools for the development of diagnostic and therapeutic RNA molecules with improved function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547715">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547715" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547715">
        <p class="paperTitle">Mapping interactions between disordered regions reveals promiscuity in biomolecular condensate formation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547715" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547715" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gilat, A.; Dubrueil, B.; Levy, E. D.</p>
        <p class="info">Score: 16.0, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547715' target='https://doi.org/10.1101/2023.07.04.547715'> 10.1101/2023.07.04.547715</a></p>
        <p class="abstract">Intrinsically-disordered regions (IDRs) promote intracellular phase separation and the formation of biomolecular condensates through interactions encoded in their primary sequence. While these condensates form spatially distinct assemblies in cells, it is unclear whether such specificity can be conferred by IDRs alone. Indeed, IDRs exhibit high conformational flexibility whereas specificity in protein recognition is generally associated with well-defined 3D structures. To characterize IDR-IDR interactions and assess their ability to mediate self-specific partitioning, we developed a synthetic system of Multivalent IDRs forming Constitutive DROPlets (micDROP). We investigated ten natural IDRs that underwent phase separation in micDROP. These IDRs exhibited a wide range of saturation concentrations in vivo, which correlated well with their total sequence stickiness. We then probed IDR-IDR specificity by co-expressing pairs of IDRs fused to homologous scaffolds that did not co-assemble. We observed a high degree of promiscuity, particularly among IDRs from the proteins Ddx4, DYRK3, ER, FUS, hnRNPA1, HspB8, RBM14 and TAF15, whereas TDP43 and UBQ2 formed spatially distinct condensates regardless of their partner. Further investigation revealed the short and conserved -helical segment of TDP43s IDR was governing its specific self-recognition. Our findings imply that IDRs can tune their phase separation propensity through sequence composition, while their formation of discrete condensates likely requires additional cellular or structural determinants.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=200 SRC=&#34;FIGDIR/small/547715v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (60K):
org.highwire.dtl.DTLVardef@b72917org.highwire.dtl.DTLVardef@eaafa1org.highwire.dtl.DTLVardef@1b0758corg.highwire.dtl.DTLVardef@1f56126_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549566">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549566" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549566">
        <p class="paperTitle">BRAIDing receptors for cell specific targeting</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549566" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549566" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, H.; Lee, S.-J.; Li, R.; Sura, A.; Suen, N.; Dilip, A.; Pomogov, Y.; Vuppalapaty, M.; Lu, C.; Post, Y.; Li, Y.</p>
        <p class="info">Score: 1.8, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549566' target='https://doi.org/10.1101/2023.07.18.549566'> 10.1101/2023.07.18.549566</a></p>
        <p class="abstract">Systemic toxicity is a major challenge to therapeutic development, consequently, cell-type specific targeting is highly desirable to improve on-target cell efficacy while reducing off-target toxicity. Here, we describe a cell targeting system we called BRAID (BRidged Activation by Intra/intermolecular Division) where an active molecule is divided into two inactive or less active parts that are subsequently brought together via a bridging receptor on target cell. We tested this hypothesis using WNT/{beta}-catenin signaling system and demonstrated that a multivalent WNT agonist molecule divided into two inactive components induced signaling specifically on hepatocytes assembled via two different epitopes on a hepatocyte receptor, {beta}Klotho. These data provided proof-of-concept for a cell specific targeting approach and also demonstrated the feasibility of combining different signaling pathways where desirable. This approach has broad applications to other receptor systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547106">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547106" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547106">
        <p class="paperTitle">A Platform for Controlled Template-Independent Enzymatic Synthesis of RNA Oligonucleotides and Therapeutics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547106" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547106" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wiegand, D. J.; Rittichier, J.; Meyer, E.; Lee, H.; Conway, N. J.; Ahlstedt, D.; Yurtsever, Z.; Rainone, D.; Kuru, E.; Church, G.</p>
        <p class="info">Score: 16.3, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547106' target='https://doi.org/10.1101/2023.06.29.547106'> 10.1101/2023.06.29.547106</a></p>
        <p class="abstract">Therapeutic RNA oligonucleotides have shown tremendous potential to manage and treat disease, yet current manufacturing methods cannot deliver on this promise. Here, we report the development and optimization of a novel, aqueous-based, template-independent enzymatic RNA oligonucleotide synthesis platform as an alternative to traditional chemical methodologies. Our platform is made possible by reversible terminator nucleoside triphosphates and an enzyme capable of their incorporation. We show that many common therapeutic RNA modifications are compatible with our process and demonstrate the enzymatic synthesis of natural and modified oligonucleotides in both liquid and solid phases. Our platform offers many unique advantages over chemical synthesis, including the realization of a more sustainable process to produce therapeutic RNA oligonucleotides.

One-Sentence SummaryAn enzyme and novel nucleotide building blocks are used to synthesize RNA oligonucleotides template independently under aqueous conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.550116">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.550116" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.550116">
        <p class="paperTitle">Characterization and diversification of AraC/XylS family regulators guided by transposon sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.550116" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.550116" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pearson, A. N.; Incha, M. R.; Ho, C.; Schmidt, M.; Roberts, J. B.; Nava, A. A.; Keasling, J. D.</p>
        <p class="info">Score: 1.0, Published: 2023-07-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.550116' target='https://doi.org/10.1101/2023.07.21.550116'> 10.1101/2023.07.21.550116</a></p>
        <p class="abstract">In this study, we explored the development of engineered inducible systems. Publicly available data from previous transposon sequencing assays were used to identify regulators of metabolism in Pseudomonas putida KT2440. For the AraC-family regulators (AFRs) represented in this data, we posited AFR/promoter/inducer groupings. Eleven promoters were characterized for a response to their proposed inducers in P. putida, and the resultant data were used to create and test nine two-plasmid sensor systems in E. coli. Several of these were further developed into a palette of single-plasmid inducible systems. From these experiments, we observed an unreported inducer response from a previously characterized AFR, demonstrated that the addition of a P. putida transporter improved the sensor dynamics of an AFR in E. coli, and identified an uncharacterized AFR with a novel potential inducer specificity. Finally, targeted mutations in an AFR, informed by structural predictions, enabled further diversification of these inducible plasmids.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=105 SRC=&#34;FIGDIR/small/550116v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (35K):
org.highwire.dtl.DTLVardef@1ffc892org.highwire.dtl.DTLVardef@1c91a7borg.highwire.dtl.DTLVardef@1e98a5corg.highwire.dtl.DTLVardef@17ab383_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
