<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Reconstructing cell histories in space with image-readable base editor recording
Authors: Chadly, D. M.; Frieda, K. L.; Gui, C.; Klock, L.; Tran, M.; Sui, M. Y.; Takei, Y.; Bouckaert, R.; Lois, C.; Cai, L.; Elowitz, M. B.
Score: 102.4, Published: 2024-01-04 DOI: 10.1101/2024.01.03.573434
Knowing the ancestral states and lineage relationships of individual cells could unravel the dynamic programs underlying development. Engineering cells to actively record information within their own genomic DNA could reveal these histories, but existing recording systems have limited information capacity or disrupt spatial context.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Reconstructing cell histories in space with image-readable base editor recording
Authors: Chadly, D. M.; Frieda, K. L.; Gui, C.; Klock, L.; Tran, M.; Sui, M. Y.; Takei, Y.; Bouckaert, R.; Lois, C.; Cai, L.; Elowitz, M. B.
Score: 102.4, Published: 2024-01-04 DOI: 10.1101/2024.01.03.573434
Knowing the ancestral states and lineage relationships of individual cells could unravel the dynamic programs underlying development. Engineering cells to actively record information within their own genomic DNA could reveal these histories, but existing recording systems have limited information capacity or disrupt spatial context." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-10T10:40:05+00:00" />
<meta property="article:modified_time" content="2024-01-10T10:40:05+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Reconstructing cell histories in space with image-readable base editor recording
Authors: Chadly, D. M.; Frieda, K. L.; Gui, C.; Klock, L.; Tran, M.; Sui, M. Y.; Takei, Y.; Bouckaert, R.; Lois, C.; Cai, L.; Elowitz, M. B.
Score: 102.4, Published: 2024-01-04 DOI: 10.1101/2024.01.03.573434
Knowing the ancestral states and lineage relationships of individual cells could unravel the dynamic programs underlying development. Engineering cells to actively record information within their own genomic DNA could reveal these histories, but existing recording systems have limited information capacity or disrupt spatial context."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Reconstructing cell histories in space with image-readable base editor recording\nAuthors: Chadly, D. M.; Frieda, K. L.; Gui, C.; Klock, L.; Tran, M.; Sui, M. Y.; Takei, Y.; Bouckaert, R.; Lois, C.; Cai, L.; Elowitz, M. B.\nScore: 102.4, Published: 2024-01-04 DOI: 10.1101/2024.01.03.573434\nKnowing the ancestral states and lineage relationships of individual cells could unravel the dynamic programs underlying development. Engineering cells to actively record information within their own genomic DNA could reveal these histories, but existing recording systems have limited information capacity or disrupt spatial context.",
  "keywords": [
    
  ],
  "articleBody": " Reconstructing cell histories in space with image-readable base editor recording\nAuthors: Chadly, D. M.; Frieda, K. L.; Gui, C.; Klock, L.; Tran, M.; Sui, M. Y.; Takei, Y.; Bouckaert, R.; Lois, C.; Cai, L.; Elowitz, M. B.\nScore: 102.4, Published: 2024-01-04 DOI: 10.1101/2024.01.03.573434\nKnowing the ancestral states and lineage relationships of individual cells could unravel the dynamic programs underlying development. Engineering cells to actively record information within their own genomic DNA could reveal these histories, but existing recording systems have limited information capacity or disrupt spatial context. Here, we introduce baseMEMOIR, which combines base editing, sequential hybridization imaging, and Bayesian inference to allow reconstruction of high-resolution cell lineage trees and cell state dynamics while preserving spatial organization. BaseMEMOIR stochastically and irreversibly edits engineered dinucleotides to one of three alternative image-readable states. By genomically integrating arrays of editable dinucleotides, we constructed an embryonic stem cell line with 792 bits of recordable, image-readable memory, a 50-fold increase over the state of the art. Simulations showed that this memory size was sufficient for accurate reconstruction of deep lineage trees. Experimentally, baseMEMOIR allowed precise reconstruction of lineage trees 6 or more generations deep in embryonic stem cell colonies. Further, it also allowed inference of ancestral cell states and their quantitative cell state transition rates, all from endpoint images. baseMEMOIR thus provides a scalable framework for reconstructing single cell histories in spatially organized multicellular systems.\nUnlocking de novo antibody design with generative artificial intelligence\nAuthors: Shanehsazzadeh, A.; McPartlon, M.; Kasun, G.; Steiger, A. K.; Sutton, J. M.; Yassine, E.; McCloskey, C.; Haile, R.; Shuai, R.; Alverio, J.; Rakocevic, G.; Levine, S.; Cejovic, J.; Gutierrez, J. M.; Morehead, A.; Dubrovskyi, O.; Chung, C.; Luton, B. K.; Diaz, N.; Kohnert, C.; Consbruck, R.; Carter, H.; LaCombe, C.; Bist, I.; Vilaychack, P.; Anderson, Z.; Xiu, L.; Bringas, P.; Alarcon, K.; Knight, B.; Radach, M.; Bateman, K.; Kopec-Belliveau, G.; Chapman, D.; Bennett, J.; Ventura, A. B.; Canales, G. M.; Gowda, M.; Jackson, K. A.; Caguiat, R.; Brown, A.; Ganini da Silva, D.; Guo, Z.; Abdulhaqq, S\nScore: 352.3, Published: 2024-01-07 DOI: 10.1101/2023.01.08.523187\nGenerative AI has the potential to redefine the process of therapeutic antibody discovery. In this report, we describe and validate deep generative models for the de novo design of antibodies against human epidermal growth factor receptor (HER2) without additional optimization. The models enabled an efficient workflow that combined in silico design methods with high-throughput experimental techniques to rapidly identify binders from a library of [~]106 heavy chain complementarity-determining region (HCDR) variants. We demonstrated that the workflow achieves binding rates of 10.6% for HCDR3 and 1.8% for HCDR123 designs and is statistically superior to baselines. We further characterized 421 diverse binders using surface plasmon resonance (SPR), finding 71 with low nanomolar affinity similar to the therapeutic anti-HER2 antibody trastuzumab. A selected subset of 11 diverse high-affinity binders were functionally equivalent or superior to trastuzumab, with most demonstrating suitable developability features. We designed one binder with [~]3x higher cell-based potency compared to trastuzumab and another with improved cross-species reactivity1. Our generative AI approach unlocks an accelerated path to designing therapeutic antibodies against diverse targets.\nCompact RNA sensors for increasingly complex functions of multiple inputs\nAuthors: Choe, C. A.; Andreasson, J. O. L.; Melaine, F.; Kladwang, W.; Wu, M. J.; Portela, F.; Wellington-Oguri, R.; Nicol, J. J.; Wayment-Steele, H. K.; Gotrik, M.; Eterna Participants, ; Kahtri, P.; Greenleaf, W. J.; Das, R.\nScore: 5.2, Published: 2024-01-04 DOI: 10.1101/2024.01.04.572289\nDesigning single molecules that compute general functions of input molecular partners represents a major unsolved challenge in molecular design. Here, we demonstrate that high-throughput, iterative experimental testing of diverse RNA designs crowdsourced from Eterna yields sensors of increasingly complex functions of input oligonucleotide concentrations. After designing single-input RNA sensors with activation ratios beyond our detection limits, we created logic gates, including challenging XOR and XNOR gates, and sensors that respond to the ratio of two inputs. Finally, we describe the OpenTB challenge, which elicited 85-nucleotide sensors that compute a score for diagnosing active tuberculosis, based on the ratio of products of three gene segments. Building on OpenTB design strategies, we created an algorithm Nucleologic that produces similarly compact sensors for the three-gene score based on RNA and DNA. These results open new avenues for diverse applications of compact, single molecule sensors previously limited by design complexity.\nLight-based juxtacrine signaling between synthetic cells\nAuthors: Moghimianavval, H.; Loi, K. J.; Hwang, S.-W.; Bashirzadeh, Y.; Liu, A. P.\nScore: 2.9, Published: 2024-01-06 DOI: 10.1101/2024.01.05.574425\nCell signaling through direct physical cell-cell contacts plays vital roles in biology during development, angiogenesis, and immune response. Intercellular communication mechanisms between synthetic cells constructed from the bottom up are majorly reliant on diffusible chemical signals, thus limiting the range of responses in receiver cells. Engineering contact-dependent signaling between synthetic cells promises to unlock more complicated signaling schemes with different types of responses. Here, we design and demonstrate a light-activated contact-dependent communication tool for synthetic cells. We utilize a split bioluminescent protein to limit signal generation exclusively to contact interfaces of synthetic cells, driving the recruitment of a photoswitchable protein in receiver cells, akin to juxtacrine signaling in living cells. Our modular design not only demonstrates contact-dependent communication between synthetic cells but also provides a platform for engineering orthogonal contact-dependent signaling mechanisms.\nContinual improvement of multiplex mutagenesis in Arabidopsis\nAuthors: Develtere, W.; Decaestecker, W.; Rombaut, D.; Anders, C.; Clicque, E.; Vuylsteke, M.; Jacobs, T. B.\nScore: 23.1, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572307\nCRISPR/Cas9 is currently the most powerful tool to generate mutations in plant genomes and more efficient tools are needed as the scale of experiments increases. In the model plant Arabidopsis, the choice of promoter driving Cas9 expression is critical to generate germline mutations. Several optimal promoters have been reported. However, it is unclear which promoter is ideal as they have not been thoroughly tested side-by-side. Furthermore, most plant vectors still use one of the two Cas9 nuclear localization sequence (NLS) configurations initially reported and can still be optimized. We genotyped more than 6,000 Arabidopsis T2 plants to test seven promoters and eleven NLS architectures across 14 targets to systematically improve the generation of single and multiplex inheritable mutations. We find that the RPS5A promoter and double-BP NLS architecture were individually the most efficient components. When combined, 99% of T2 plant contained at least one knockout mutation and 84% contained 4-7-plex knock-outs. These optimizations will be useful to generate higher-order knockouts in the germline of Arabidopsis and likely be applicable to other CRISPR systems as well.\nVirus-like DNA tracers for characterizing wastewater-based pathogen monitoring systems\nAuthors: Gopal, A.; Machtinger, A. N.; Wang, B.; Rice, D. P.; McLaren, M. R.; Esvelt, K. M.\nScore: 4.1, Published: 2023-12-25 DOI: 10.1101/2023.12.24.573197\nWastewater-based pathogen monitoring has recently become a ubiquitous method for tracking the abundance and evolution of pathogens in a community or port of entry. But accurate inference of disease incidence from wastewater remains challenging, in large part due to noise and bias arising from wastewater transport and measurement. We developed a new system of tracer particles designed to enhance wastewater monitoring of viral pathogens. These tracers, which consist of synthetic DNA barcodes packaged into non-infectious viral capsids, can be deposited at known amounts and times into the sewer system and measured in wastewater samples to assess features such as assay sensitivity, and the decay and delay in viral signal arising from a shedding event. Here we describe the design, construction, and initial characterization of a 25-plex tracer library. Our tracer library supports simultaneous measurement of up to 5 distinct tracers by qPCR and up to 25 distinct tracers by amplicon or metagenomic sequencing. We demonstrate experimentally that the tracers are biologically inert and exhibit good stability and qPCR-assay performance in wastewater. These features collectively suggest that barcoded, virus-like tracers have the required properties for high-accuracy, high-resolution characterization of wastewater monitoring systems.\nMaturation and conformational switching of a de novo designed phase-separating polypeptide\nAuthors: Hilditch, A. T.; Romanyuk, A.; Hodgson, L. R.; Mantell, J.; Neal, C. R.; Verkade, P.; Obexer, R.; Serpell, L. C.; McManus, J. J.; Woolfson, D. N.\nScore: 1.5, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574599\nCellular compartments formed by biomolecular condensation are a widespread feature of cell biology. These organelle-like assemblies compartmentalize macromolecules dynamically within the crowded intracellular environment. However, the intermolecular interactions that produce condensed droplets may also create arrested states and potentially pathological assemblies such as fibers, aggregates, and gels, through droplet maturation. Protein liquid-liquid phase separation is a metastable process, so maturation may be an intrinsic property of phase-separating proteins, where nucleation of different phases or states arise in supersaturated condensates. Here, we describe the formation of both phase-separated droplets and proteinaceous fibers driven by a de novo designed polypeptide. We characterize the formation of supramolecular fibers in vitro and in bacterial cells. We show that client proteins can be targeted to the fibers in cells using the droplet-forming construct. Finally, we explore the interplay between phase separation and fiber formation of the de novo polypeptide, showing that the droplets mature with a post-translational switch to largely {beta} conformations, analogous to models of pathological phase separation.\nDevelopment of an optogenetic gene expression system in Lactococcus lactis using a split photoactivatable T7 RNA polymerase\nAuthors: Rodrigo-Navarro, A.; Salmeron-Sanchez, M.\nScore: 1.5, Published: 2024-01-06 DOI: 10.1101/2024.01.05.574370\nCellular processes can be modulated by physical means, such as light, which offers advantages over chemically inducible systems with respect to spatiotemporal control. Here we introduce an optogenetic gene expression system for Lactococcus lactis that utilizes a split T7 RNA polymerase linked to two variants of the Vivid regulators. Depending on the chosen photoreceptor variant, either Magnets or enhanced Magnets, this system can achieve either high protein expression levels or low basal activity in the absence of light, exhibiting a fold induction close to 30, rapid expression kinetics, and heightened light sensitivity. This system functions effectively in liquid cultures and within cells embedded in hydrogel matrices, highlighting its potential in the development of novel engineered living materials capable of responding to physical stimuli such as light. The optogenetic component of this system is highly customizable, allowing for the adjustment of expression patterns through modifications to the promoters and/or engineered T7 RNA polymerase variants. We anticipate that this system can be broadly adapted to other Gram-positive hosts with minimal modifications required.\nLeveraging engineered Pseudomonas putida minicells for bioconversion of organic acids into short-chain methyl ketones\nAuthors: Kozaeva, E.; Nieto-Dominguez, M.; Tang, K. K. Y.; Nikel, P. I.\nScore: 1.2, Published: 2024-01-06 DOI: 10.1101/2024.01.06.574483\nMethyl ketones, key building-blocks widely used in diverse industrial applications, largely depend on oil-derived chemical methods for their production. Here, we investigated bio-based production alternatives for short-chain ketones, adapting the solvent-tolerant soil bacterium Pseudomonas putida as a host for ketone biosynthesis either by whole-cell biocatalysis or using engineered minicells, chromosome-free bacterial vesicles. Organic acids (acetate, propanoate and butyrate) were selected as the main carbon substrate to drive the biosynthesis of acetone, 2-butanone and 2-pentanone. Pathway optimization identified efficient enzyme variants from Clostridium acetobutylicum and Escherichia coli, which were tested under both constitutive and inducible expression of the cognate genes. By implementing these optimized pathways in P. putida minicells, which can be prepared through a simple 3-step purification protocol, the feedstock was converted into the target short-chain methyl ketones, remaining catalytically functional for \u003e4 months. These results highlight the value of combining morphology and pathway engineering of non-canonical bacterial hosts to establish alternative bioprocesses for toxic chemicals that are difficult to produce by conventional approaches. GRAPHICAL ABSTRACT O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=168 SRC=\"FIGDIR/small/574483v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (39K): org.highwire.dtl.DTLVardef@37acf6org.highwire.dtl.DTLVardef@13c708borg.highwire.dtl.DTLVardef@100bc76org.highwire.dtl.DTLVardef@1552610_HPS_FORMAT_FIGEXP M_FIG C_FIG\nSpecies-specific design of artificial promoters by transfer-learning based generative deep-learning model\nAuthors: Xia, Y.; Du, X.; Liu, B.; Guo, S.; Huo, Y.-X.\nScore: 9.8, Published: 2024-01-07 DOI: 10.1101/2023.12.27.573416\nNative prokaryotic promoters share common sequence patterns, but are species dependent. For understudied species with limited data, it is challenging to predict the strength of existing promoters and generate novel promoters. Here, we developed PromoGen, a collection of nucleotide language models to generate species-specific functional promoters, across dozens of species in a data and parameter efficient way. Twenty-seven species-specific models in this collection were finetuned from the pretrained model which was trained on multi-species promoters. When systematically compared with native promoters, the Escherichia coli- and Bacillus subtilis-specific artificial PromoGen-generated promoters (PGPs) were demonstrated to hold all essential features and distribution patterns of native promoters. A regression model was developed to score generated either by PromoGen or by another competitive neural network, and the overall score of PGPs is higher. Encouraged by in silico analysis, we further experimentally characterized twenty-two B. subtilis PGPs in vivo, results showed that four of tested PGPs reached the strong promoter level while all were active. Furthermore, we developed a user-friendly website to generate species-specific promoters for 27 different species by PromoGen. This work presented an efficient deep-learning strategy for de novo species-specific promoter generation even with limited datasets, providing valuable promoter toolboxes especially for the metabolic engineering of understudied microorganisms. O_FIG O_LINKSMALLFIG WIDTH=138 HEIGHT=200 SRC=\"FIGDIR/small/573416v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (33K): org.highwire.dtl.DTLVardef@da053forg.highwire.dtl.DTLVardef@18e8e69org.highwire.dtl.DTLVardef@27b32corg.highwire.dtl.DTLVardef@641703_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "2163",
  "inLanguage": "en",
  "datePublished": "2024-01-10T10:40:05Z",
  "dateModified": "2024-01-10T10:40:05Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 10, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.573434">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.573434" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.573434">
        <p class="paperTitle">Reconstructing cell histories in space with image-readable base editor recording</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.573434" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.573434" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chadly, D. M.; Frieda, K. L.; Gui, C.; Klock, L.; Tran, M.; Sui, M. Y.; Takei, Y.; Bouckaert, R.; Lois, C.; Cai, L.; Elowitz, M. B.</p>
        <p class="info">Score: 102.4, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.573434' target='https://doi.org/10.1101/2024.01.03.573434'> 10.1101/2024.01.03.573434</a></p>
        <p class="abstract">Knowing the ancestral states and lineage relationships of individual cells could unravel the dynamic programs underlying development. Engineering cells to actively record information within their own genomic DNA could reveal these histories, but existing recording systems have limited information capacity or disrupt spatial context. Here, we introduce baseMEMOIR, which combines base editing, sequential hybridization imaging, and Bayesian inference to allow reconstruction of high-resolution cell lineage trees and cell state dynamics while preserving spatial organization. BaseMEMOIR stochastically and irreversibly edits engineered dinucleotides to one of three alternative image-readable states. By genomically integrating arrays of editable dinucleotides, we constructed an embryonic stem cell line with 792 bits of recordable, image-readable memory, a 50-fold increase over the state of the art. Simulations showed that this memory size was sufficient for accurate reconstruction of deep lineage trees. Experimentally, baseMEMOIR allowed precise reconstruction of lineage trees 6 or more generations deep in embryonic stem cell colonies. Further, it also allowed inference of ancestral cell states and their quantitative cell state transition rates, all from endpoint images. baseMEMOIR thus provides a scalable framework for reconstructing single cell histories in spatially organized multicellular systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.08.523187">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.08.523187" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.08.523187">
        <p class="paperTitle">Unlocking de novo antibody design with generative artificial intelligence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.08.523187" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.08.523187" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shanehsazzadeh, A.; McPartlon, M.; Kasun, G.; Steiger, A. K.; Sutton, J. M.; Yassine, E.; McCloskey, C.; Haile, R.; Shuai, R.; Alverio, J.; Rakocevic, G.; Levine, S.; Cejovic, J.; Gutierrez, J. M.; Morehead, A.; Dubrovskyi, O.; Chung, C.; Luton, B. K.; Diaz, N.; Kohnert, C.; Consbruck, R.; Carter, H.; LaCombe, C.; Bist, I.; Vilaychack, P.; Anderson, Z.; Xiu, L.; Bringas, P.; Alarcon, K.; Knight, B.; Radach, M.; Bateman, K.; Kopec-Belliveau, G.; Chapman, D.; Bennett, J.; Ventura, A. B.; Canales, G. M.; Gowda, M.; Jackson, K. A.; Caguiat, R.; Brown, A.; Ganini da Silva, D.; Guo, Z.; Abdulhaqq, S</p>
        <p class="info">Score: 352.3, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.08.523187' target='https://doi.org/10.1101/2023.01.08.523187'> 10.1101/2023.01.08.523187</a></p>
        <p class="abstract">Generative AI has the potential to redefine the process of therapeutic antibody discovery. In this report, we describe and validate deep generative models for the de novo design of antibodies against human epidermal growth factor receptor (HER2) without additional optimization. The models enabled an efficient workflow that combined in silico design methods with high-throughput experimental techniques to rapidly identify binders from a library of [~]106 heavy chain complementarity-determining region (HCDR) variants. We demonstrated that the workflow achieves binding rates of 10.6% for HCDR3 and 1.8% for HCDR123 designs and is statistically superior to baselines. We further characterized 421 diverse binders using surface plasmon resonance (SPR), finding 71 with low nanomolar affinity similar to the therapeutic anti-HER2 antibody trastuzumab. A selected subset of 11 diverse high-affinity binders were functionally equivalent or superior to trastuzumab, with most demonstrating suitable developability features. We designed one binder with [~]3x higher cell-based potency compared to trastuzumab and another with improved cross-species reactivity1. Our generative AI approach unlocks an accelerated path to designing therapeutic antibodies against diverse targets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.572289">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.572289" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.572289">
        <p class="paperTitle">Compact RNA sensors for increasingly complex functions of multiple inputs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.572289" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.572289" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Choe, C. A.; Andreasson, J. O. L.; Melaine, F.; Kladwang, W.; Wu, M. J.; Portela, F.; Wellington-Oguri, R.; Nicol, J. J.; Wayment-Steele, H. K.; Gotrik, M.; Eterna Participants,  ; Kahtri, P.; Greenleaf, W. J.; Das, R.</p>
        <p class="info">Score: 5.2, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.572289' target='https://doi.org/10.1101/2024.01.04.572289'> 10.1101/2024.01.04.572289</a></p>
        <p class="abstract">Designing single molecules that compute general functions of input molecular partners represents a major unsolved challenge in molecular design. Here, we demonstrate that high-throughput, iterative experimental testing of diverse RNA designs crowdsourced from Eterna yields sensors of increasingly complex functions of input oligonucleotide concentrations. After designing single-input RNA sensors with activation ratios beyond our detection limits, we created logic gates, including challenging XOR and XNOR gates, and sensors that respond to the ratio of two inputs. Finally, we describe the OpenTB challenge, which elicited 85-nucleotide sensors that compute a score for diagnosing active tuberculosis, based on the ratio of products of three gene segments. Building on OpenTB design strategies, we created an algorithm Nucleologic that produces similarly compact sensors for the three-gene score based on RNA and DNA. These results open new avenues for diverse applications of compact, single molecule sensors previously limited by design complexity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574425">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574425" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574425">
        <p class="paperTitle">Light-based juxtacrine signaling between synthetic cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574425" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574425" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moghimianavval, H.; Loi, K. J.; Hwang, S.-W.; Bashirzadeh, Y.; Liu, A. P.</p>
        <p class="info">Score: 2.9, Published: 2024-01-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574425' target='https://doi.org/10.1101/2024.01.05.574425'> 10.1101/2024.01.05.574425</a></p>
        <p class="abstract">Cell signaling through direct physical cell-cell contacts plays vital roles in biology during development, angiogenesis, and immune response. Intercellular communication mechanisms between synthetic cells constructed from the bottom up are majorly reliant on diffusible chemical signals, thus limiting the range of responses in receiver cells. Engineering contact-dependent signaling between synthetic cells promises to unlock more complicated signaling schemes with different types of responses. Here, we design and demonstrate a light-activated contact-dependent communication tool for synthetic cells. We utilize a split bioluminescent protein to limit signal generation exclusively to contact interfaces of synthetic cells, driving the recruitment of a photoswitchable protein in receiver cells, akin to juxtacrine signaling in living cells. Our modular design not only demonstrates contact-dependent communication between synthetic cells but also provides a platform for engineering orthogonal contact-dependent signaling mechanisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572307">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572307" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572307">
        <p class="paperTitle">Continual improvement of multiplex mutagenesis in Arabidopsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572307" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572307" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Develtere, W.; Decaestecker, W.; Rombaut, D.; Anders, C.; Clicque, E.; Vuylsteke, M.; Jacobs, T. B.</p>
        <p class="info">Score: 23.1, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572307' target='https://doi.org/10.1101/2023.12.19.572307'> 10.1101/2023.12.19.572307</a></p>
        <p class="abstract">CRISPR/Cas9 is currently the most powerful tool to generate mutations in plant genomes and more efficient tools are needed as the scale of experiments increases. In the model plant Arabidopsis, the choice of promoter driving Cas9 expression is critical to generate germline mutations. Several optimal promoters have been reported. However, it is unclear which promoter is ideal as they have not been thoroughly tested side-by-side. Furthermore, most plant vectors still use one of the two Cas9 nuclear localization sequence (NLS) configurations initially reported and can still be optimized. We genotyped more than 6,000 Arabidopsis T2 plants to test seven promoters and eleven NLS architectures across 14 targets to systematically improve the generation of single and multiplex inheritable mutations. We find that the RPS5A promoter and double-BP NLS architecture were individually the most efficient components. When combined, 99% of T2 plant contained at least one knockout mutation and 84% contained 4-7-plex knock-outs. These optimizations will be useful to generate higher-order knockouts in the germline of Arabidopsis and likely be applicable to other CRISPR systems as well.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.24.573197">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.24.573197" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.24.573197">
        <p class="paperTitle">Virus-like DNA tracers for characterizing wastewater-based pathogen monitoring systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.24.573197" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.24.573197" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gopal, A.; Machtinger, A. N.; Wang, B.; Rice, D. P.; McLaren, M. R.; Esvelt, K. M.</p>
        <p class="info">Score: 4.1, Published: 2023-12-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.24.573197' target='https://doi.org/10.1101/2023.12.24.573197'> 10.1101/2023.12.24.573197</a></p>
        <p class="abstract">Wastewater-based pathogen monitoring has recently become a ubiquitous method for tracking the abundance and evolution of pathogens in a community or port of entry. But accurate inference of disease incidence from wastewater remains challenging, in large part due to noise and bias arising from wastewater transport and measurement. We developed a new system of tracer particles designed to enhance wastewater monitoring of viral pathogens. These tracers, which consist of synthetic DNA barcodes packaged into non-infectious viral capsids, can be deposited at known amounts and times into the sewer system and measured in wastewater samples to assess features such as assay sensitivity, and the decay and delay in viral signal arising from a shedding event. Here we describe the design, construction, and initial characterization of a 25-plex tracer library. Our tracer library supports simultaneous measurement of up to 5 distinct tracers by qPCR and up to 25 distinct tracers by amplicon or metagenomic sequencing. We demonstrate experimentally that the tracers are biologically inert and exhibit good stability and qPCR-assay performance in wastewater. These features collectively suggest that barcoded, virus-like tracers have the required properties for high-accuracy, high-resolution characterization of wastewater monitoring systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574599">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574599" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574599">
        <p class="paperTitle">Maturation and conformational switching of a de novo designed phase-separating polypeptide</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574599" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574599" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hilditch, A. T.; Romanyuk, A.; Hodgson, L. R.; Mantell, J.; Neal, C. R.; Verkade, P.; Obexer, R.; Serpell, L. C.; McManus, J. J.; Woolfson, D. N.</p>
        <p class="info">Score: 1.5, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574599' target='https://doi.org/10.1101/2024.01.08.574599'> 10.1101/2024.01.08.574599</a></p>
        <p class="abstract">Cellular compartments formed by biomolecular condensation are a widespread feature of cell biology. These organelle-like assemblies compartmentalize macromolecules dynamically within the crowded intracellular environment. However, the intermolecular interactions that produce condensed droplets may also create arrested states and potentially pathological assemblies such as fibers, aggregates, and gels, through droplet maturation. Protein liquid-liquid phase separation is a metastable process, so maturation may be an intrinsic property of phase-separating proteins, where nucleation of different phases or states arise in supersaturated condensates. Here, we describe the formation of both phase-separated droplets and proteinaceous fibers driven by a de novo designed polypeptide. We characterize the formation of supramolecular fibers in vitro and in bacterial cells. We show that client proteins can be targeted to the fibers in cells using the droplet-forming construct. Finally, we explore the interplay between phase separation and fiber formation of the de novo polypeptide, showing that the droplets mature with a post-translational switch to largely {beta} conformations, analogous to models of pathological phase separation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574370">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574370" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574370">
        <p class="paperTitle">Development of an optogenetic gene expression system in Lactococcus lactis using a split photoactivatable T7 RNA polymerase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574370" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574370" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rodrigo-Navarro, A.; Salmeron-Sanchez, M.</p>
        <p class="info">Score: 1.5, Published: 2024-01-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574370' target='https://doi.org/10.1101/2024.01.05.574370'> 10.1101/2024.01.05.574370</a></p>
        <p class="abstract">Cellular processes can be modulated by physical means, such as light, which offers advantages over chemically inducible systems with respect to spatiotemporal control. Here we introduce an optogenetic gene expression system for Lactococcus lactis that utilizes a split T7 RNA polymerase linked to two variants of the Vivid regulators. Depending on the chosen photoreceptor variant, either  Magnets or  enhanced Magnets, this system can achieve either high protein expression levels or low basal activity in the absence of light, exhibiting a fold induction close to 30, rapid expression kinetics, and heightened light sensitivity. This system functions effectively in liquid cultures and within cells embedded in hydrogel matrices, highlighting its potential in the development of novel engineered living materials capable of responding to physical stimuli such as light. The optogenetic component of this system is highly customizable, allowing for the adjustment of expression patterns through modifications to the promoters and/or engineered T7 RNA polymerase variants. We anticipate that this system can be broadly adapted to other Gram-positive hosts with minimal modifications required.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.06.574483">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.06.574483" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.06.574483">
        <p class="paperTitle">Leveraging engineered Pseudomonas putida minicells for bioconversion of organic acids into short-chain methyl ketones</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.06.574483" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.06.574483" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kozaeva, E.; Nieto-Dominguez, M.; Tang, K. K. Y.; Nikel, P. I.</p>
        <p class="info">Score: 1.2, Published: 2024-01-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.06.574483' target='https://doi.org/10.1101/2024.01.06.574483'> 10.1101/2024.01.06.574483</a></p>
        <p class="abstract">Methyl ketones, key building-blocks widely used in diverse industrial applications, largely depend on oil-derived chemical methods for their production. Here, we investigated bio-based production alternatives for short-chain ketones, adapting the solvent-tolerant soil bacterium Pseudomonas putida as a host for ketone biosynthesis either by whole-cell biocatalysis or using engineered minicells, chromosome-free bacterial vesicles. Organic acids (acetate, propanoate and butyrate) were selected as the main carbon substrate to drive the biosynthesis of acetone, 2-butanone and 2-pentanone. Pathway optimization identified efficient enzyme variants from Clostridium acetobutylicum and Escherichia coli, which were tested under both constitutive and inducible expression of the cognate genes. By implementing these optimized pathways in P. putida minicells, which can be prepared through a simple 3-step purification protocol, the feedstock was converted into the target short-chain methyl ketones, remaining catalytically functional for &gt;4 months. These results highlight the value of combining morphology and pathway engineering of non-canonical bacterial hosts to establish alternative bioprocesses for toxic chemicals that are difficult to produce by conventional approaches.

GRAPHICAL ABSTRACT

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=168 SRC=&#34;FIGDIR/small/574483v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (39K):
org.highwire.dtl.DTLVardef@37acf6org.highwire.dtl.DTLVardef@13c708borg.highwire.dtl.DTLVardef@100bc76org.highwire.dtl.DTLVardef@1552610_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573416">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573416" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573416">
        <p class="paperTitle">Species-specific design of artificial promoters by transfer-learning based generative deep-learning model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573416" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573416" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xia, Y.; Du, X.; Liu, B.; Guo, S.; Huo, Y.-X.</p>
        <p class="info">Score: 9.8, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573416' target='https://doi.org/10.1101/2023.12.27.573416'> 10.1101/2023.12.27.573416</a></p>
        <p class="abstract">Native prokaryotic promoters share common sequence patterns, but are species dependent. For understudied species with limited data, it is challenging to predict the strength of existing promoters and generate novel promoters. Here, we developed PromoGen, a collection of nucleotide language models to generate species-specific functional promoters, across dozens of species in a data and parameter efficient way. Twenty-seven species-specific models in this collection were finetuned from the pretrained model which was trained on multi-species promoters. When systematically compared with native promoters, the Escherichia coli- and Bacillus subtilis-specific artificial PromoGen-generated promoters (PGPs) were demonstrated to hold all essential features and distribution patterns of native promoters. A regression model was developed to score generated either by PromoGen or by another competitive neural network, and the overall score of PGPs is higher. Encouraged by in silico analysis, we further experimentally characterized twenty-two B. subtilis PGPs in vivo, results showed that four of tested PGPs reached the strong promoter level while all were active. Furthermore, we developed a user-friendly website to generate species-specific promoters for 27 different species by PromoGen. This work presented an efficient deep-learning strategy for de novo species-specific promoter generation even with limited datasets, providing valuable promoter toolboxes especially for the metabolic engineering of understudied microorganisms.



O_FIG O_LINKSMALLFIG WIDTH=138 HEIGHT=200 SRC=&#34;FIGDIR/small/573416v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (33K):
org.highwire.dtl.DTLVardef@da053forg.highwire.dtl.DTLVardef@18e8e69org.highwire.dtl.DTLVardef@27b32corg.highwire.dtl.DTLVardef@641703_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
