<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="ProteinNPT: Improving Protein Property Predictionand Design with Non-Parametric Transformers
Authors: Notin, P.; Weitzman, R.; Marks, D. S.; Gal, Y.
Score: 23.3, Published: 2023-12-07 DOI: 10.1101/2023.12.06.570473
Protein design holds immense potential for optimizing naturally occurring proteins, with broad applications in drug discovery, material design, and sustainability. How-ever, computational methods for protein engineering are confronted with significant challenges, such as an expansive design space, sparse functional regions, and a scarcity of available labels.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="ProteinNPT: Improving Protein Property Predictionand Design with Non-Parametric Transformers
Authors: Notin, P.; Weitzman, R.; Marks, D. S.; Gal, Y.
Score: 23.3, Published: 2023-12-07 DOI: 10.1101/2023.12.06.570473
Protein design holds immense potential for optimizing naturally occurring proteins, with broad applications in drug discovery, material design, and sustainability. How-ever, computational methods for protein engineering are confronted with significant challenges, such as an expansive design space, sparse functional regions, and a scarcity of available labels." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-13T10:39:57+00:00" />
<meta property="article:modified_time" content="2023-12-13T10:39:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="ProteinNPT: Improving Protein Property Predictionand Design with Non-Parametric Transformers
Authors: Notin, P.; Weitzman, R.; Marks, D. S.; Gal, Y.
Score: 23.3, Published: 2023-12-07 DOI: 10.1101/2023.12.06.570473
Protein design holds immense potential for optimizing naturally occurring proteins, with broad applications in drug discovery, material design, and sustainability. How-ever, computational methods for protein engineering are confronted with significant challenges, such as an expansive design space, sparse functional regions, and a scarcity of available labels."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "ProteinNPT: Improving Protein Property Predictionand Design with Non-Parametric Transformers\nAuthors: Notin, P.; Weitzman, R.; Marks, D. S.; Gal, Y.\nScore: 23.3, Published: 2023-12-07 DOI: 10.1101/2023.12.06.570473\nProtein design holds immense potential for optimizing naturally occurring proteins, with broad applications in drug discovery, material design, and sustainability. How-ever, computational methods for protein engineering are confronted with significant challenges, such as an expansive design space, sparse functional regions, and a scarcity of available labels.",
  "keywords": [
    
  ],
  "articleBody": " ProteinNPT: Improving Protein Property Predictionand Design with Non-Parametric Transformers\nAuthors: Notin, P.; Weitzman, R.; Marks, D. S.; Gal, Y.\nScore: 23.3, Published: 2023-12-07 DOI: 10.1101/2023.12.06.570473\nProtein design holds immense potential for optimizing naturally occurring proteins, with broad applications in drug discovery, material design, and sustainability. How-ever, computational methods for protein engineering are confronted with significant challenges, such as an expansive design space, sparse functional regions, and a scarcity of available labels. These issues are further exacerbated in practice by the fact most real-life design scenarios necessitate the simultaneous optimization of multiple properties. In this work, we introduce ProteinNPT, a non-parametric trans-former variant tailored to protein sequences and particularly suited to label-scarce and multi-task learning settings. We first focus on the supervised fitness prediction setting and develop several cross-validation schemes which support robust perfor-mance assessment. We subsequently reimplement prior top-performing baselines, introduce several extensions of these baselines by integrating diverse branches of the protein engineering literature, and demonstrate that ProteinNPT consistently outperforms all of them across a diverse set of protein property prediction tasks. Finally, we demonstrate the value of our approach for iterative protein design across extensive in silico Bayesian optimization and conditional sampling experiments.\nProteinGym: Large-Scale Benchmarks for Protein Design and Fitness Prediction\nAuthors: Notin, P.; Kollasch, A. W.; Ritter, D.; van Niekerk, L.; Paul, S.; Spinner, H.; Rollins, N.; Shaw, A.; Weitzman, R.; Frazer, J.; Dias, M.; Franceschi, D.; Orenbuch, R.; Gal, Y.; Marks, D. S.\nScore: 23.0, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570727\nPredicting the effects of mutations in proteins is critical to many applications, from understanding genetic disease to designing novel proteins that can address our most pressing challenges in climate, agriculture and healthcare. Despite the surge in machine learning-based protein models to tackle these questions, an assessment of their respective benefits is challenging due to the use of distinct, often contrived, experimental datasets, and the variable performance of models across different protein families. Addressing these challenges requires scale. To that end we introduce ProteinGym, a large-scale and holistic set of benchmarks specifically designed for protein fitness prediction and design. It encompasses both a broad collection of over 250 standardized deep mutational scanning assays, spanning millions of mutated sequences, as well as curated clinical datasets providing high-quality expert annotations about mutation effects. We devise a robust evaluation framework that combines metrics for both fitness prediction and design, factors in known limitations of the underlying experimental methods, and covers both zero-shot and supervised settings. We report the performance of a diverse set of over 70 high-performing models from various subfields (eg., alignment-based, inverse folding) into a unified benchmark suite. We open source the corresponding codebase, datasets, MSAs, structures, model predictions and develop a user-friendly website that facilitates data access and analysis.\nIn vitro validated antibody design against multiple therapeutic antigens using generative inverse folding\nAuthors: Shanehsazzadeh, A.; Alverio, J.; Kasun, G.; Levine, S.; Khan, J. A.; Chung, C.; Diaz, N.; Luton, B. K.; Tarter, Y.; McCloskey, C.; Bateman, K. B.; Carter, H.; Chapman, D.; Consbruck, R.; Jaeger, A.; Kohnert, C.; Kopec-Belliveau, G.; Sutton, J. M.; Guo, Z.; Canales, G.; Ejan, K.; Marsh, E.; Ruelos, A.; Ripley, R.; Stoddard, B.; Caguiat, R.; Chapman, K.; Saunders, M.; Sharp, J.; Ganini da Silva, D.; Feltner, A.; Ripley, J.; Bryant, M. E.; Castillo, D.; Meier, J.; Stegmann, C. M.; Moran, K.; Lemke, C.; Abdulhaqq, S.; Klug, L. R.; Bachas, S.\nScore: 22.1, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570889\nDeep learning approaches have demonstrated the ability to design protein sequences given backbone structures [1, 2, 3, 4, 5]. While these approaches have been applied in silico to designing antibody complementarity-determining regions (CDRs), they have yet to be validated in vitro for designing antibody binders, which is the true measure of success for antibody design. Here we describe IgDesign, a deep learning method for antibody CDR design, and demonstrate its robustness with successful binder design for 8 therapeutic antigens. The model is tasked with designing heavy chain CDR3 (HCDR3) or all three heavy chain CDRs (HCDR123) using native backbone structures of antibody-antigen complexes, along with the antigen and antibody framework (FWR) sequences as context. For each of the 8 antigens, we design 100 HCDR3s and 100 HCDR123s, scaffold them into the native antibody's variable region, and screen them for binding against the antigen using surface plasmon resonance (SPR). As a baseline, we screen 100 HCDR3s taken from the model's training set and paired with the native HCDR1 and HCDR2. We observe that both HCDR3 design and HCDR123 design outperform this HCDR3- only baseline. IgDesign is the first experimentally validated antibody inverse folding model. It can design antibody binders to multiple therapeutic antigens with high success rates and, in some cases, improved affinities over clinically validated reference antibodies. Antibody inverse folding has applications to both de novo antibody design and lead optimization, making IgDesign a valuable tool for accelerating drug development and enabling therapeutic design.\nValidation of cell-free protein synthesis aboard the International Space Station\nAuthors: Kocalar, S.; Miller, B. M.; Huang, A.; Gleason, E.; Martin, K.; Foley, K.; Copeland, D. S.; Jewett, M.; Saavedra, E. A.; Kraves, S.\nScore: 18.8, Published: 2023-12-06 DOI: 10.1101/2023.12.06.570403\nCell-free protein synthesis (CFPS) is a rapidly maturing in vitro gene expression platform that can be used to transcribe and translate nucleic acids at the point of need, enabling on-demand synthesis of peptide-based vaccines and biotherapeutics, as well as the development of diagnostic tests for environmental contaminants and infectious agents. Unlike traditional cell-based systems, CFPS platforms do not require the maintenance of living cells and can be deployed with minimal equipment; therefore, they hold promise for applications in low-resource contexts, including spaceflight. Here we evaluate the performance of cell-free BioBits(R) platform aboard the International Space Station by expressing RNA-based aptamers and fluorescent proteins that can serve as biological indicators. We validate two classes of biological sensors that detect either the small molecule DFHBI or a specific RNA sequence. Upon detection of their respective analytes, both biological sensors produce fluorescent readouts that are visually confirmed using a handheld fluorescence viewer and imaged for quantitative analysis. Our findings provide insight into the kinetics of cell-free transcription and translation in a microgravity environment and reveal that both biosensors perform robustly in space. Our findings lay the groundwork for portable, low-cost applications ranging from point-of-care health monitoring to on-demand detection of environmental hazards in low-resource communities both on Earth and beyond. Visual graphic O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=123 SRC=\"FIGDIR/small/570403v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (47K): org.highwire.dtl.DTLVardef@40ed76org.highwire.dtl.DTLVardef@a60a37org.highwire.dtl.DTLVardef@e8c209org.highwire.dtl.DTLVardef@11b086b_HPS_FORMAT_FIGEXP M_FIG C_FIG\nDegenerate DropSynth for Simultaneous Assembly of Diverse Gene Libraries and Local Designed Mutants\nAuthors: Holston, A. S.; Hinton, S. R.; Lindley, K. A.; Kearns, N. C.; Plesa, C.\nScore: 15.2, Published: 2023-12-12 DOI: 10.1101/2023.12.11.569291\nProtein engineering efforts often involve the creation of hybrid or chimeric proteins, where functionality critically hinges on the precise design of linkers and fusion points. Traditional methods have been constrained by a focus on single genes or the random selection of fusion points. Here we introduce an approach which enables the creation of large gene libraries where each library comprises a multitude of diverse, specifically designed genes, each with a corresponding set of programmatically designed fusion points or linkers. When combined with multiplex functional assays, these libraries facilitate the derivation of generalized engineering principles applicable across whole protein families or domain types. Degenerate DropSynth is a multiplex gene synthesis technique which allows for the assembly of up to eight distinct variants for each of the 1,536 designed parent genes in a single reaction. We assemble chimeric sensor histidine kinases and demonstrate the assembly of genes up to 1 kbp in length with an 8% rate of perfect assemblies per gene. Our findings indicate that incorporating an increased number of variants in droplets containing barcoded beads does not significantly affect the rate of perfect assemblies. However, maintaining a consistent level of degeneracy across the library is important to ensure good coverage and reduce inequality. The results suggest the potential for scaling this process to assemble at least 8,000 distinct variants in a single reaction. Degenerate DropSynth enables the systematic exploration of protein families through large-scale, programmable assembly of chimeric proteins, moving beyond the limitations of individual protein studies.\nPHEIGES, all-cell-free phage synthesis and selection from engineered genomes\nAuthors: Levrier, A.; Karpathakis, I.; Nash, B.; Bowden, S. D.; Lindner, A. B.; Noireaux, V.\nScore: 11.4, Published: 2023-12-07 DOI: 10.1101/2023.12.07.570578\nBacteriophages constitute an invaluable biological reservoir for biotechnology and medicine. The ability to exploit such vast resources is hampered by the lack of methods to rapidly engineer, assemble, package genomes, and select phages. Cell-free transcription-translation (TXTL) offers experimental settings to address such a limitation. Here, we describe PHage Engineering by In vitro Gene Expression and Selection (PHEIGES) using T7 phage genome and Escherichia coli TXTL. Phage genomes are assembled in vitro from PCR-amplified fragments and directly expressed in batch TXTL reactions to produce up to 1011 PFU/ml engineered phages within one day. We further demonstrate an important genotype-phenotype linkage of phage assembly in bulk TXTL. This enables rapid selection of phages with altered rough lipopolysaccharides specificity from phage genomes incorporating tail fiber mutant libraries. We establish the scalability of PHEIGES by one-pot assembly of such mutants with fluorescent gene integration and 10% length-reduced genome.\nOligonucleotide-based CRISPR-Cas9 toolbox for efficient engineering of Komagataella phaffii\nAuthors: Strucko, T.; Gadar-Lopez, A. E.; Frohling, F. B.; Frost, E. T.; Olsson, H.; Jarczynska, Z. D.; Mortensen, U. H.\nScore: 11.2, Published: 2023-12-09 DOI: 10.1101/2023.12.09.570913\nKomagataella phaffii (Pichia pastoris) is a methylotrophic yeast that is favored by industry and academia mainly for expression of heterologous proteins. However, its full potential as a host for bio-production of valuable compounds is not yet fully exploited. The emergence of CRISPR-Cas9 technology has significantly improved the efficiency of gene manipulations of non-conventional species including K. phaffii. Yet, improvements in gene-editing methods are desirable to further accelerate engineering of industrially and scientifically relevant K. phaffii strains. In this study, we have developed a versatile one vector-based CRISPR-Cas9 method and showed that it works efficiently at different genetic loci using linear DNA fragments with very short targeting sequences. Importantly, we show that by using our setup it is possible to catalyze single-stranded oligonucleotide-mediated mutagenesis and marker-free gene integrations. Notably, we performed site-specific point mutations and full gene deletions using single stranded 90-mers at very high efficiencies. Lastly, we present a strategy for transient inactivation of non-homologous end-joining (NHEJ) pathway, where KU70 gene is disrupted by a visual marker (uidA gene). The latter system enables precise CRISPR-Cas9 based editing (including multiplexing) and accelerates selection of the mutants that have simultaneously undergone a desired genetic modification(s) and restored NHEJ-proficient genotype. In conclusion, the tools presented in this study can be applied for easy and efficient engineering of K. phaffii strains and could potentially be coupled with high-throughput automated workflows.\nDiscovery of a high-performance phage-derived promoter/repressor system for probiotic lactobacillus engineering\nAuthors: Blanch-Asensio, M.; Tadimarri, V. S.; Wilk, A.; Sankaran, S.\nScore: 9.4, Published: 2023-12-05 DOI: 10.1101/2023.12.05.570180\nBackgroundThe Lactobacillus family comprises many species of great importance for the food and healthcare industries, with numerous strains identified as beneficial for humans and used as probiotics. Hence, there is a growing interest in engineering these probiotic bacteria as live biotherapeutics for animals and humans. However, the genetic parts needed to regulate gene expression in these bacteria remain limited compared to model bacteria like E. coli or B. subtilis. To address this deficit, in this study, we selected and tested several bacteriophage-derived genetic parts with the potential to regulate transcription in lactobacilli. ResultsWe screened genetic parts from 6 different lactobacilli-infecting phages and identified one promoter/repressor system with unprecedented functionality in L. plantarum WCFS1. The phage-derived promoter was found to achieve expression levels nearly 9-fold higher than the previously reported strongest promoter in this strain and the repressor was able to almost completely repress this expression by reducing it nearly 500-fold. ConclusionsThe new parts and insights gained from their engineering will enhance the genetic programmability of lactobacilli for healthcare and industrial applications.\nMethylmercury demethylation and volatilization by animals expressing microbial enzymes.\nAuthors: Tepper, K.; King, J.; Cholan, P. M.; Pfitzner, C.; Morsch, M.; Apte, S.; Maselko, M.\nScore: 7.5, Published: 2023-12-11 DOI: 10.1101/2023.12.11.571038\nMethylmercury is a highly toxic pollutant that accumulates in food webs where it is inaccessible to current remediation technologies. We demonstrate that animals can be engineered to express the microbial enzymes, organomercurial lyase (MerB) and mercuric reductase (MerA), to bioremediate methylmercury. MerA and MerB from Escherichia coli were functional in invertebrate (Drosophila melanogaster) and vertebrate (Danio rerio) model systems and converted methylmercury into volatile Hg0. The engineered animals tolerated higher exposures to methylmercury and accumulated less than half as much mercury relative to their wild-type counterparts. The outcomes of this research could be applied to reduce mercury contamination in farmed and recreationally caught fish, for species conservation, and to restore value to organic wastes contaminated with mercury.\nPopulation suppression with dominant female-lethal alleles is boosted by homing gene drive\nAuthors: Zhu, J.; Chen, J.; Liu, Y.; Xu, X.; Champer, J.\nScore: 3.4, Published: 2023-12-05 DOI: 10.1101/2023.12.05.570109\nMethods to suppress pest insect populations using genetic constructs and repeated releases of male homozygotes have recently been shown to be an attractive alternative to older sterile insect technique based on radiation. Female-specific lethal alleles have substantially increased power, but still require large, sustained transgenic insect releases. Gene drive alleles bias their own inheritance to spread throughout populations, potentially allowing population suppression with a single, small-size release. However, suppression drives often suffer from efficiency issues, and the most well-studied type, homing drives, tend to spread without limit. In this study, we show that coupling female-specific lethal alleles with homing gene drive allowed substantial improvement in efficiency while still retaining the self-limiting nature (and thus confinement) of a lethal allele strategy. Using a mosquito model, we show the required releases sizes for population elimination in a variety of scenarios, including different density growth curves, with comparisons to other systems. Resistance alleles reduced the power of this method, but these could be overcome by targeting an essential gene with the drive while also providing rescue. A proof-of-principle demonstration of this system in Drosophila melanogaster was effective in both basing its inheritance and achieving high lethality among females that inherit the construct in the absence of antibiotic. Overall, our study shows that substantial improvements can be achieved in female-specific lethal systems for population suppression by combining them with a gene drive.\n",
  "wordCount" : "2389",
  "inLanguage": "en",
  "datePublished": "2023-12-13T10:39:57Z",
  "dateModified": "2023-12-13T10:39:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 13, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.570473">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.570473" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.570473">
        <p class="paperTitle">ProteinNPT: Improving Protein Property Predictionand Design with Non-Parametric Transformers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.570473" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.570473" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Notin, P.; Weitzman, R.; Marks, D. S.; Gal, Y.</p>
        <p class="info">Score: 23.3, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.570473' target='https://doi.org/10.1101/2023.12.06.570473'> 10.1101/2023.12.06.570473</a></p>
        <p class="abstract">Protein design holds immense potential for optimizing naturally occurring proteins, with broad applications in drug discovery, material design, and sustainability. How-ever, computational methods for protein engineering are confronted with significant challenges, such as an expansive design space, sparse functional regions, and a scarcity of available labels. These issues are further exacerbated in practice by the fact most real-life design scenarios necessitate the simultaneous optimization of multiple properties. In this work, we introduce ProteinNPT, a non-parametric trans-former variant tailored to protein sequences and particularly suited to label-scarce and multi-task learning settings. We first focus on the supervised fitness prediction setting and develop several cross-validation schemes which support robust perfor-mance assessment. We subsequently reimplement prior top-performing baselines, introduce several extensions of these baselines by integrating diverse branches of the protein engineering literature, and demonstrate that ProteinNPT consistently outperforms all of them across a diverse set of protein property prediction tasks. Finally, we demonstrate the value of our approach for iterative protein design across extensive in silico Bayesian optimization and conditional sampling experiments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570727">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570727" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570727">
        <p class="paperTitle">ProteinGym: Large-Scale Benchmarks for Protein Design and Fitness Prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570727" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570727" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Notin, P.; Kollasch, A. W.; Ritter, D.; van Niekerk, L.; Paul, S.; Spinner, H.; Rollins, N.; Shaw, A.; Weitzman, R.; Frazer, J.; Dias, M.; Franceschi, D.; Orenbuch, R.; Gal, Y.; Marks, D. S.</p>
        <p class="info">Score: 23.0, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570727' target='https://doi.org/10.1101/2023.12.07.570727'> 10.1101/2023.12.07.570727</a></p>
        <p class="abstract">Predicting the effects of mutations in proteins is critical to many applications, from understanding genetic disease to designing novel proteins that can address our most pressing challenges in climate, agriculture and healthcare. Despite the surge in machine learning-based protein models to tackle these questions, an assessment of their respective benefits is challenging due to the use of distinct, often contrived, experimental datasets, and the variable performance of models across different protein families. Addressing these challenges requires scale. To that end we introduce ProteinGym, a large-scale and holistic set of benchmarks specifically designed for protein fitness prediction and design. It encompasses both a broad collection of over 250 standardized deep mutational scanning assays, spanning millions of mutated sequences, as well as curated clinical datasets providing high-quality expert annotations about mutation effects. We devise a robust evaluation framework that combines metrics for both fitness prediction and design, factors in known limitations of the underlying experimental methods, and covers both zero-shot and supervised settings. We report the performance of a diverse set of over 70 high-performing models from various subfields (eg., alignment-based, inverse folding) into a unified benchmark suite. We open source the corresponding codebase, datasets, MSAs, structures, model predictions and develop a user-friendly website that facilitates data access and analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570889">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570889" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570889">
        <p class="paperTitle">In vitro validated antibody design against multiple therapeutic antigens using generative inverse folding</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570889" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570889" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shanehsazzadeh, A.; Alverio, J.; Kasun, G.; Levine, S.; Khan, J. A.; Chung, C.; Diaz, N.; Luton, B. K.; Tarter, Y.; McCloskey, C.; Bateman, K. B.; Carter, H.; Chapman, D.; Consbruck, R.; Jaeger, A.; Kohnert, C.; Kopec-Belliveau, G.; Sutton, J. M.; Guo, Z.; Canales, G.; Ejan, K.; Marsh, E.; Ruelos, A.; Ripley, R.; Stoddard, B.; Caguiat, R.; Chapman, K.; Saunders, M.; Sharp, J.; Ganini da Silva, D.; Feltner, A.; Ripley, J.; Bryant, M. E.; Castillo, D.; Meier, J.; Stegmann, C. M.; Moran, K.; Lemke, C.; Abdulhaqq, S.; Klug, L. R.; Bachas, S.</p>
        <p class="info">Score: 22.1, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570889' target='https://doi.org/10.1101/2023.12.08.570889'> 10.1101/2023.12.08.570889</a></p>
        <p class="abstract">Deep learning approaches have demonstrated the ability to design protein sequences given backbone structures [1, 2, 3, 4, 5]. While these approaches have been applied in silico to designing antibody complementarity-determining regions (CDRs), they have yet to be validated in vitro for designing antibody binders, which is the true measure of success for antibody design. Here we describe IgDesign, a deep learning method for antibody CDR design, and demonstrate its robustness with successful binder design for 8 therapeutic antigens. The model is tasked with designing heavy chain CDR3 (HCDR3) or all three heavy chain CDRs (HCDR123) using native backbone structures of antibody-antigen complexes, along with the antigen and antibody framework (FWR) sequences as context. For each of the 8 antigens, we design 100 HCDR3s and 100 HCDR123s, scaffold them into the native antibody&#39;s variable region, and screen them for binding against the antigen using surface plasmon resonance (SPR). As a baseline, we screen 100 HCDR3s taken from the model&#39;s training set and paired with the native HCDR1 and HCDR2. We observe that both HCDR3 design and HCDR123 design outperform this HCDR3- only baseline. IgDesign is the first experimentally validated antibody inverse folding model. It can design antibody binders to multiple therapeutic antigens with high success rates and, in some cases, improved affinities over clinically validated reference antibodies. Antibody inverse folding has applications to both de novo antibody design and lead optimization, making IgDesign a valuable tool for accelerating drug development and enabling therapeutic design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.570403">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.570403" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.570403">
        <p class="paperTitle">Validation of cell-free protein synthesis aboard the International Space Station</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.570403" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.570403" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kocalar, S.; Miller, B. M.; Huang, A.; Gleason, E.; Martin, K.; Foley, K.; Copeland, D. S.; Jewett, M.; Saavedra, E. A.; Kraves, S.</p>
        <p class="info">Score: 18.8, Published: 2023-12-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.570403' target='https://doi.org/10.1101/2023.12.06.570403'> 10.1101/2023.12.06.570403</a></p>
        <p class="abstract">Cell-free protein synthesis (CFPS) is a rapidly maturing in vitro gene expression platform that can be used to transcribe and translate nucleic acids at the point of need, enabling on-demand synthesis of peptide-based vaccines and biotherapeutics, as well as the development of diagnostic tests for environmental contaminants and infectious agents. Unlike traditional cell-based systems, CFPS platforms do not require the maintenance of living cells and can be deployed with minimal equipment; therefore, they hold promise for applications in low-resource contexts, including spaceflight. Here we evaluate the performance of cell-free BioBits(R) platform aboard the International Space Station by expressing RNA-based aptamers and fluorescent proteins that can serve as biological indicators. We validate two classes of biological sensors that detect either the small molecule DFHBI or a specific RNA sequence. Upon detection of their respective analytes, both biological sensors produce fluorescent readouts that are visually confirmed using a handheld fluorescence viewer and imaged for quantitative analysis. Our findings provide insight into the kinetics of cell-free transcription and translation in a microgravity environment and reveal that both biosensors perform robustly in space. Our findings lay the groundwork for portable, low-cost applications ranging from point-of-care health monitoring to on-demand detection of environmental hazards in low-resource communities both on Earth and beyond.

Visual graphic

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=123 SRC=&#34;FIGDIR/small/570403v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (47K):
org.highwire.dtl.DTLVardef@40ed76org.highwire.dtl.DTLVardef@a60a37org.highwire.dtl.DTLVardef@e8c209org.highwire.dtl.DTLVardef@11b086b_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.569291">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.569291" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.569291">
        <p class="paperTitle">Degenerate DropSynth for Simultaneous Assembly of Diverse Gene Libraries and Local Designed Mutants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.569291" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.569291" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Holston, A. S.; Hinton, S. R.; Lindley, K. A.; Kearns, N. C.; Plesa, C.</p>
        <p class="info">Score: 15.2, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.569291' target='https://doi.org/10.1101/2023.12.11.569291'> 10.1101/2023.12.11.569291</a></p>
        <p class="abstract">Protein engineering efforts often involve the creation of hybrid or chimeric proteins, where functionality critically hinges on the precise design of linkers and fusion points. Traditional methods have been constrained by a focus on single genes or the random selection of fusion points. Here we introduce an approach which enables the creation of large gene libraries where each library comprises a multitude of diverse, specifically designed genes, each with a corresponding set of programmatically designed fusion points or linkers. When combined with multiplex functional assays, these libraries facilitate the derivation of generalized engineering principles applicable across whole protein families or domain types. Degenerate DropSynth is a multiplex gene synthesis technique which allows for the assembly of up to eight distinct variants for each of the 1,536 designed parent genes in a single reaction. We assemble chimeric sensor histidine kinases and demonstrate the assembly of genes up to 1 kbp in length with an 8% rate of perfect assemblies per gene. Our findings indicate that incorporating an increased number of variants in droplets containing barcoded beads does not significantly affect the rate of perfect assemblies. However, maintaining a consistent level of degeneracy across the library is important to ensure good coverage and reduce inequality. The results suggest the potential for scaling this process to assemble at least 8,000 distinct variants in a single reaction. Degenerate DropSynth enables the systematic exploration of protein families through large-scale, programmable assembly of chimeric proteins, moving beyond the limitations of individual protein studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570578">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570578" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570578">
        <p class="paperTitle">PHEIGES, all-cell-free phage synthesis and selection from engineered genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570578" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570578" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Levrier, A.; Karpathakis, I.; Nash, B.; Bowden, S. D.; Lindner, A. B.; Noireaux, V.</p>
        <p class="info">Score: 11.4, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570578' target='https://doi.org/10.1101/2023.12.07.570578'> 10.1101/2023.12.07.570578</a></p>
        <p class="abstract">Bacteriophages constitute an invaluable biological reservoir for biotechnology and medicine. The ability to exploit such vast resources is hampered by the lack of methods to rapidly engineer, assemble, package genomes, and select phages. Cell-free transcription-translation (TXTL) offers experimental settings to address such a limitation. Here, we describe PHage Engineering by In vitro Gene Expression and Selection (PHEIGES) using T7 phage genome and Escherichia coli TXTL. Phage genomes are assembled in vitro from PCR-amplified fragments and directly expressed in batch TXTL reactions to produce up to 1011 PFU/ml engineered phages within one day. We further demonstrate an important genotype-phenotype linkage of phage assembly in bulk TXTL. This enables rapid selection of phages with altered rough lipopolysaccharides specificity from phage genomes incorporating tail fiber mutant libraries. We establish the scalability of PHEIGES by one-pot assembly of such mutants with fluorescent gene integration and 10% length-reduced genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.09.570913">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.09.570913" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.09.570913">
        <p class="paperTitle">Oligonucleotide-based CRISPR-Cas9 toolbox for efficient engineering of Komagataella phaffii</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.09.570913" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.09.570913" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Strucko, T.; Gadar-Lopez, A. E.; Frohling, F. B.; Frost, E. T.; Olsson, H.; Jarczynska, Z. D.; Mortensen, U. H.</p>
        <p class="info">Score: 11.2, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.09.570913' target='https://doi.org/10.1101/2023.12.09.570913'> 10.1101/2023.12.09.570913</a></p>
        <p class="abstract">Komagataella phaffii (Pichia pastoris) is a methylotrophic yeast that is favored by industry and academia mainly for expression of heterologous proteins. However, its full potential as a host for bio-production of valuable compounds is not yet fully exploited. The emergence of CRISPR-Cas9 technology has significantly improved the efficiency of gene manipulations of non-conventional species including K. phaffii. Yet, improvements in gene-editing methods are desirable to further accelerate engineering of industrially and scientifically relevant K. phaffii strains. In this study, we have developed a versatile one vector-based CRISPR-Cas9 method and showed that it works efficiently at different genetic loci using linear DNA fragments with very short targeting sequences. Importantly, we show that by using our setup it is possible to catalyze single-stranded oligonucleotide-mediated mutagenesis and marker-free gene integrations. Notably, we performed site-specific point mutations and full gene deletions using single stranded 90-mers at very high efficiencies. Lastly, we present a strategy for transient inactivation of non-homologous end-joining (NHEJ) pathway, where KU70 gene is disrupted by a visual marker (uidA gene). The latter system enables precise CRISPR-Cas9 based editing (including multiplexing) and accelerates selection of the mutants that have simultaneously undergone a desired genetic modification(s) and restored NHEJ-proficient genotype. In conclusion, the tools presented in this study can be applied for easy and efficient engineering of K. phaffii strains and could potentially be coupled with high-throughput automated workflows.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570180">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570180" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570180">
        <p class="paperTitle">Discovery of a high-performance phage-derived promoter/repressor system for probiotic lactobacillus engineering</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570180" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570180" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Blanch-Asensio, M.; Tadimarri, V. S.; Wilk, A.; Sankaran, S.</p>
        <p class="info">Score: 9.4, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570180' target='https://doi.org/10.1101/2023.12.05.570180'> 10.1101/2023.12.05.570180</a></p>
        <p class="abstract">BackgroundThe Lactobacillus family comprises many species of great importance for the food and healthcare industries, with numerous strains identified as beneficial for humans and used as probiotics. Hence, there is a growing interest in engineering these probiotic bacteria as live biotherapeutics for animals and humans. However, the genetic parts needed to regulate gene expression in these bacteria remain limited compared to model bacteria like E. coli or B. subtilis. To address this deficit, in this study, we selected and tested several bacteriophage-derived genetic parts with the potential to regulate transcription in lactobacilli.

ResultsWe screened genetic parts from 6 different lactobacilli-infecting phages and identified one promoter/repressor system with unprecedented functionality in L. plantarum WCFS1. The phage-derived promoter was found to achieve expression levels nearly 9-fold higher than the previously reported strongest promoter in this strain and the repressor was able to almost completely repress this expression by reducing it nearly 500-fold.

ConclusionsThe new parts and insights gained from their engineering will enhance the genetic programmability of lactobacilli for healthcare and industrial applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.571038">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.571038" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.571038">
        <p class="paperTitle">Methylmercury demethylation and volatilization by animals expressing microbial enzymes.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.571038" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.571038" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tepper, K.; King, J.; Cholan, P. M.; Pfitzner, C.; Morsch, M.; Apte, S.; Maselko, M.</p>
        <p class="info">Score: 7.5, Published: 2023-12-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.571038' target='https://doi.org/10.1101/2023.12.11.571038'> 10.1101/2023.12.11.571038</a></p>
        <p class="abstract">Methylmercury is a highly toxic pollutant that accumulates in food webs where it is inaccessible to current remediation technologies. We demonstrate that animals can be engineered to express the microbial enzymes, organomercurial lyase (MerB) and mercuric reductase (MerA), to bioremediate methylmercury. MerA and MerB from Escherichia coli were functional in invertebrate (Drosophila melanogaster) and vertebrate (Danio rerio) model systems and converted methylmercury into volatile Hg0. The engineered animals tolerated higher exposures to methylmercury and accumulated less than half as much mercury relative to their wild-type counterparts. The outcomes of this research could be applied to reduce mercury contamination in farmed and recreationally caught fish, for species conservation, and to restore value to organic wastes contaminated with mercury.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570109">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570109" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570109">
        <p class="paperTitle">Population suppression with dominant female-lethal alleles is boosted by homing gene drive</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570109" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570109" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhu, J.; Chen, J.; Liu, Y.; Xu, X.; Champer, J.</p>
        <p class="info">Score: 3.4, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570109' target='https://doi.org/10.1101/2023.12.05.570109'> 10.1101/2023.12.05.570109</a></p>
        <p class="abstract">Methods to suppress pest insect populations using genetic constructs and repeated releases of male homozygotes have recently been shown to be an attractive alternative to older sterile insect technique based on radiation. Female-specific lethal alleles have substantially increased power, but still require large, sustained transgenic insect releases. Gene drive alleles bias their own inheritance to spread throughout populations, potentially allowing population suppression with a single, small-size release. However, suppression drives often suffer from efficiency issues, and the most well-studied type, homing drives, tend to spread without limit. In this study, we show that coupling female-specific lethal alleles with homing gene drive allowed substantial improvement in efficiency while still retaining the self-limiting nature (and thus confinement) of a lethal allele strategy. Using a mosquito model, we show the required releases sizes for population elimination in a variety of scenarios, including different density growth curves, with comparisons to other systems. Resistance alleles reduced the power of this method, but these could be overcome by targeting an essential gene with the drive while also providing rescue. A proof-of-principle demonstration of this system in Drosophila melanogaster was effective in both basing its inheritance and achieving high lethality among females that inherit the construct in the absence of antibiotic. Overall, our study shows that substantial improvements can be achieved in female-specific lethal systems for population suppression by combining them with a gene drive.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
