<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein
Authors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Jing, G.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.
Score: 77.8, Published: 2023-07-11 DOI: 10.1101/2023.07.05.547496
Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein
Authors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Jing, G.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.
Score: 77.8, Published: 2023-07-11 DOI: 10.1101/2023.07.05.547496
Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-12T10:41:10+00:00" />
<meta property="article:modified_time" content="2023-07-12T10:41:10+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein
Authors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Jing, G.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.
Score: 77.8, Published: 2023-07-11 DOI: 10.1101/2023.07.05.547496
Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein\nAuthors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Jing, G.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.\nScore: 77.8, Published: 2023-07-11 DOI: 10.1101/2023.07.05.547496\nProtein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently.",
  "keywords": [
    
  ],
  "articleBody": " xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein\nAuthors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Jing, G.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.\nScore: 77.8, Published: 2023-07-11 DOI: 10.1101/2023.07.05.547496\nProtein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently. This paper proposes a unified protein language model, xTrimoPGLM, to address these two types of tasks simultaneously through an innovative pre-training framework. Our key technical contribution is an exploration of the compatibility and the potential for joint optimization of the two types of objectives, which has led to a strategy for training xTrimoPGLM at an unprecedented scale of 100 billion parameters and 1 trillion training tokens. Our extensive experiments reveal that xTrimoPGLM significantly outperforms other advanced baselines in diverse protein understanding tasks (13 out of 15 tasks across four categories) and generates novel protein sequences which are structurally similar to natural ones. Furthermore, using the same xTrimoPGLM framework, we train an antibody-specific model (xTrimoPGLM-Ab) using 1 billion parameters. This model set a new record in predicting antibody naturalness and structures, both essential to the field of antibody-based drug design, and demonstrated a significantly faster inference speed than AlphaFold2. These results highlight the substantial capability and versatility of xTrimoPGLM in understanding and generating protein sequences.\nMapping interactions between disordered regions reveals promiscuity in biomolecular condensate formation\nAuthors: Gilat, A.; Dubrueil, B.; Levy, E. D.\nScore: 14.4, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547715\nIntrinsically-disordered regions (IDRs) promote intracellular phase separation and the formation of biomolecular condensates through interactions encoded in their primary sequence. While these condensates form spatially distinct assemblies in cells, it is unclear whether such specificity can be conferred by IDRs alone. Indeed, IDRs exhibit high conformational flexibility whereas specificity in protein recognition is generally associated with well-defined 3D structures. To characterize IDR-IDR interactions and assess their ability to mediate self-specific partitioning, we developed a synthetic system of Multivalent IDRs forming Constitutive DROPlets (micDROP). We investigated ten natural IDRs that underwent phase separation in micDROP. These IDRs exhibited a wide range of saturation concentrations in vivo, which correlated well with their total sequence stickiness. We then probed IDR-IDR specificity by co-expressing pairs of IDRs fused to homologous scaffolds that did not co-assemble. We observed a high degree of promiscuity, particularly among IDRs from the proteins Ddx4, DYRK3, ER, FUS, hnRNPA1, HspB8, RBM14 and TAF15, whereas TDP43 and UBQ2 formed spatially distinct condensates regardless of their partner. Further investigation revealed the short and conserved -helical segment of TDP43s IDR was governing its specific self-recognition. Our findings imply that IDRs can tune their phase separation propensity through sequence composition, while their formation of discrete condensates likely requires additional cellular or structural determinants. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=200 SRC=\"FIGDIR/small/547715v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (60K): org.highwire.dtl.DTLVardef@1b86475org.highwire.dtl.DTLVardef@cb424forg.highwire.dtl.DTLVardef@6e6575org.highwire.dtl.DTLVardef@1c70144_HPS_FORMAT_FIGEXP M_FIG C_FIG\nEmergent digital bio-computation through spatial diffusion and engineered bacteria\nAuthors: Fedorec, A. J. H.; Treloar, N. J.; Wen, K. Y.; Dekker, L.; Ong, Q. H.; Jurkeviciute, G.; Lyu, E.; Rutter, J.; Rosa, L.; Zaikin, A.; Barnes, C. P.\nScore: 10.7, Published: 2023-07-11 DOI: 10.1101/2023.07.07.548081\nBuilding computationally capable biological systems has long been an aim of synthetic biology. The potential utility of biocomputing devices ranges from biosafety and environmental applications to diagnosis and personalised medicine. Here we present work for the design of bacterial computers which use spatial patterning to process information. Our computers are composed of bacterial colonies which, inspired by patterning in embryo development, receive information in the form of diffusible morphogen-like signals. A computation is encoded by the physical locations of the input sources of morphogen and the output receiver colonies. We demonstrate, mathematically and with engineered Escherichia coli, the simple digital logic capability of single bacterial colonies and show how additional colonies are required to build complex functions. Inspired by electronic design automation, an algorithm for designing optimal spatial circuits computing two-level digital functions is presented. This enhances the capability of our system to complex digital functions without increasing the biological complexity. We extend our experimental system to incorporate biosensing colonies as morphogen sources, demonstrating how a diagnostic device might be constructed. Our approach will open up new ways to perform biological computation, with applications in bioengineering, biomaterials and biosensing. Ultimately, these computational bacterial communities will help us explore information processing in natural biological systems.\nTruly tiny acoustic biomolecules for ultrasound imaging and therapy\nAuthors: Ling, B.; Gungoren, B.; Yao, Y.; Dutka, P.; Smith, C. A. B.; Lee, J.; Swift, M. B.; Shapiro, M. G.\nScore: 25.5, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546773\nNanotechnology offers significant advantages for medical imaging and therapy, including enhanced contrast and precision targeting. However, integrating these benefits into ultrasonography has been challenging due to the size and stability constraints of conventional bubble-based agents. Here we describe bicones, truly tiny acoustic contrast agents based on gas vesicles, a unique class of air-filled protein nanostructures naturally produced in buoyant microbes. We show that these sub-80 nm particles can be effectively detected both in vitro and in vivo, infiltrate tumors via leaky vasculature, deliver potent mechanical effects through ultrasound-induced inertial cavitation, and are easily engineered for molecular targeting, prolonged circulation time, and payload conjugation.\npUdOs: concise plasmids for bacterial and mammalian cells\nAuthors: Ourida Manigat, F.; Connell, L. B.; Stewart, B. N.; LePabic, A.-R.; Tessier, C. J. G.; Emlaw, J. R.; Calvert, N. D.; Rossl, A.; Shuhendler, A. J.; daCosta, C. J. B.; Campbell-Valois, F.-X.\nScore: 6.5, Published: 2023-07-06 DOI: 10.1101/2023.07.05.547852\nThe pUdOs are 28 plasmids of small size combining four different origins of replication and seven selection markers, which together afford flexible use in Escherichia coli and several related gram-negative bacteria. The promoterless multicloning site is insulated from upstream spurious promoters by strong transcription terminators, and contains type IIP or IIS restriction sites for conventional or Golden-gate cloning. pUdOs can be converted into efficient expression vectors through the insertion of a promoter at the user's discretion. For example, we demonstrate the utility of pUdOs as the backbone for an improved version of a Type III Secretion System reporter in Shigella. In addition, we derive a series of pUdO-based mammalian expression vectors affording distinct levels of expression and transfection efficiencies comparable to commonly used mammalian expression plasmids. Thus, pUdOs could advantageously replace traditional plasmids in a wide variety of cell types and applications.\nHighly efficient cellular expression of circular mRNA enables prolonged protein expression\nAuthors: Unti, M. J.; Jaffrey, S. R.\nScore: 2.0, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548538\nA major problem with mRNA therapeutics is that the mRNA is usually degraded within a few hours after entering the cytosol. This limits the duration and total amount of protein expression. New approaches for in vitro synthesis of circular mRNA have allowed increased levels and duration of protein synthesis from mRNA therapeutics due to the long half-life of circular mRNA. However, it remains difficult to genetically encode circular mRNAs in mammalian cells, which makes it difficult to use circular mRNA in cell-derived therapeutics. Here we describe the adaptation of the Tornado (Twister-optimized RNA for durable overexpression) system to achieve in-cell synthesis of circular mRNAs. We screen different promoters and internal ribosomal entry sites (IRESs), and identify combinations that result in high levels of circular mRNA and protein expression in cells. We show that these circular mRNAs can be packaged into virus-like particles (VLPs) thus enabling prolonged protein expression. Overall, these data describe a platform for synthesis of circular mRNAs and how these circular mRNAs can markedly enhance the ability of VLPs to function as a mRNA delivery tool.\nT7 phage-assisted evolution of riboswitches using error-prone replication and dual selection\nAuthors: Goicoechea, E.; Jaramillo, A.\nScore: 19.1, Published: 2023-07-04 DOI: 10.1101/2023.01.11.523572\nLeveraging riboswitches, non-coding mRNA fragments pivotal to gene regulation, poses a challenge in effectively selecting and enriching these functional genetic sensors, which can toggle between ON and OFF states in response to their cognate inducers. Here, we show our engineered phage T7, enabling the evolution of a theophylline riboswitch. We have replaced T7s DNA polymerase with a transcription factor controlled by a theophylline riboswitch and have created two types of host environments to propagate the engineered phage. Both types host an error-prone T7 DNA polymerase regulated by a T7 promoter along with another critical gene - either cmk or pifA, depending on the host type. The cmk gene is necessary for T7 replication and is used in the first host type for selection in the riboswitchs ON state. Conversely, the second host type incorporates the pifA gene, leading to abortive T7 infections and used for selection in the riboswitchs OFF state. This dual-selection system, termed T7AE, was then applied to a library of 65,536 engineered T7 phages, each carrying randomized riboswitch variants. Through successive passage in both host types with and without theophylline, we observed an enrichment of phages encoding functional riboswitches that conferred a fitness advantage to the phage in both hosts. The T7AE technique thereby opens new pathways for the evolution and advancement of gene switches, including non-coding RNA-based switches, setting the stage for significant strides in synthetic biology.\nEfficient PCR-based gene targeting in isolates of the non-conventional yeast Debaryomyces hansenii\nAuthors: Alhajouj, S.; Turkolmez, S.; Abalkhail, T.; Alwan, Z. H. O.; Gilmour, D. J.; Mitchell, P. J.; Hettema, E. H.\nScore: 1.2, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548570\nDebaryomyces hansenii is a yeast with considerable biotechnological potential as an osmotolerant, stress tolerant oleaginous microbe. However, targeted genome modification tools are limited and require a strain with auxotrophic markers. Gene targeting by homologous recombination has been reported to be inefficient but here we describe a set of reagents and a method that allows gene targeting at high efficiency in wild type isolates. It uses a simple PCR-based amplification that extends a completely heterologous selectable marker with 50 bp flanks identical to the target site in the genome. Transformants integrate the PCR product through homologous recombination at high frequency (\u003e75%). We illustrate the potential of this method by disrupting genes at high efficiency and by expressing a heterologous protein from a safe chromosomal harbour site. These methods should stimulate and facilitate further analysis of D. hansenii strains and open the way to engineer strains for biotechnology.\nA Platform for Controlled Template-Independent Enzymatic Synthesis of RNA Oligonucleotides and Therapeutics\nAuthors: Wiegand, D. J.; Rittichier, J.; Meyer, E.; Lee, H.; Conway, N. J.; Ahlstedt, D.; Yurtsever, Z.; Rainone, D.; Kuru, E.; Church, G.\nScore: 4.9, Published: 2023-06-29 DOI: 10.1101/2023.06.29.547106\nTherapeutic RNA oligonucleotides have shown tremendous potential to manage and treat disease, yet current manufacturing methods cannot deliver on this promise. Here, we report the development and optimization of a novel, aqueous-based, template-independent enzymatic RNA oligonucleotide synthesis platform as an alternative to traditional chemical methodologies. Our platform is made possible by reversible terminator nucleoside triphosphates and an enzyme capable of their incorporation. We show that many common therapeutic RNA modifications are compatible with our process and demonstrate the enzymatic synthesis of natural and modified oligonucleotides in both liquid and solid phases. Our platform offers many unique advantages over chemical synthesis, including the realization of a more sustainable process to produce therapeutic RNA oligonucleotides. One-Sentence SummaryAn enzyme and novel nucleotide building blocks are used to synthesize RNA oligonucleotides template independently under aqueous conditions.\nExploring alternative quorum sensing model structures and quorum quenching strategies\nAuthors: Cimolato, C.; Bellato, M.; Selvaggio, G.; Marchetti, L.; Giordano, G.; Schenato, L.\nScore: 1.0, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548074\nBacterial quorum sensing (QS) is a cell-to-cell communication mechanism through which bacteria share information about cell density, and tune gene expression accordingly. Pathogens exploit QS to orchestrate virulence and regulate the expression of genes related to antimicrobial resistance. Despite the vast literature on QS, the properties of the underlying molecular network are not entirely clear. We compare two synthetic QS circuit architectures: in the first, a single positive feedback loop autoinduces the synthesis of the signal molecule; the second includes an additional positive feedback loop enhancing the synthesis of the signal molecule receptor. Our comprehensive analysis of the two systems and their equilibria highlights the differences in the bistable and hysteretic behaviors of the alternative QS structures. Finally, we investigate three different QS inhibition approaches; numerical analysis predicts their effect on the steady-state behavior of the two different QS models, revealing critical parameter thresholds that guarantee an effective QS suppression.\n",
  "wordCount" : "2060",
  "inLanguage": "en",
  "datePublished": "2023-07-12T10:41:10Z",
  "dateModified": "2023-07-12T10:41:10Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on July 12, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547496">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547496" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547496">
        <p class="paperTitle">xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547496" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547496" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Jing, G.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.</p>
        <p class="info">Score: 77.8, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547496' target='https://doi.org/10.1101/2023.07.05.547496'> 10.1101/2023.07.05.547496</a></p>
        <p class="abstract">Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently. This paper proposes a unified protein language model, xTrimoPGLM, to address these two types of tasks simultaneously through an innovative pre-training framework. Our key technical contribution is an exploration of the compatibility and the potential for joint optimization of the two types of objectives, which has led to a strategy for training xTrimoPGLM at an unprecedented scale of 100 billion parameters and 1 trillion training tokens. Our extensive experiments reveal that xTrimoPGLM significantly outperforms other advanced baselines in diverse protein understanding tasks (13 out of 15 tasks across four categories) and generates novel protein sequences which are structurally similar to natural ones. Furthermore, using the same xTrimoPGLM framework, we train an antibody-specific model (xTrimoPGLM-Ab) using 1 billion parameters. This model set a new record in predicting antibody naturalness and structures, both essential to the field of antibody-based drug design, and demonstrated a significantly faster inference speed than AlphaFold2. These results highlight the substantial capability and versatility of xTrimoPGLM in understanding and generating protein sequences.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547715">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547715" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547715">
        <p class="paperTitle">Mapping interactions between disordered regions reveals promiscuity in biomolecular condensate formation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547715" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547715" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gilat, A.; Dubrueil, B.; Levy, E. D.</p>
        <p class="info">Score: 14.4, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547715' target='https://doi.org/10.1101/2023.07.04.547715'> 10.1101/2023.07.04.547715</a></p>
        <p class="abstract">Intrinsically-disordered regions (IDRs) promote intracellular phase separation and the formation of biomolecular condensates through interactions encoded in their primary sequence. While these condensates form spatially distinct assemblies in cells, it is unclear whether such specificity can be conferred by IDRs alone. Indeed, IDRs exhibit high conformational flexibility whereas specificity in protein recognition is generally associated with well-defined 3D structures. To characterize IDR-IDR interactions and assess their ability to mediate self-specific partitioning, we developed a synthetic system of Multivalent IDRs forming Constitutive DROPlets (micDROP). We investigated ten natural IDRs that underwent phase separation in micDROP. These IDRs exhibited a wide range of saturation concentrations in vivo, which correlated well with their total sequence stickiness. We then probed IDR-IDR specificity by co-expressing pairs of IDRs fused to homologous scaffolds that did not co-assemble. We observed a high degree of promiscuity, particularly among IDRs from the proteins Ddx4, DYRK3, ER, FUS, hnRNPA1, HspB8, RBM14 and TAF15, whereas TDP43 and UBQ2 formed spatially distinct condensates regardless of their partner. Further investigation revealed the short and conserved -helical segment of TDP43s IDR was governing its specific self-recognition. Our findings imply that IDRs can tune their phase separation propensity through sequence composition, while their formation of discrete condensates likely requires additional cellular or structural determinants.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=200 SRC=&#34;FIGDIR/small/547715v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (60K):
org.highwire.dtl.DTLVardef@1b86475org.highwire.dtl.DTLVardef@cb424forg.highwire.dtl.DTLVardef@6e6575org.highwire.dtl.DTLVardef@1c70144_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548081">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548081" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548081">
        <p class="paperTitle">Emergent digital bio-computation through spatial diffusion and engineered bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548081" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548081" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fedorec, A. J. H.; Treloar, N. J.; Wen, K. Y.; Dekker, L.; Ong, Q. H.; Jurkeviciute, G.; Lyu, E.; Rutter, J.; Rosa, L.; Zaikin, A.; Barnes, C. P.</p>
        <p class="info">Score: 10.7, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548081' target='https://doi.org/10.1101/2023.07.07.548081'> 10.1101/2023.07.07.548081</a></p>
        <p class="abstract">Building computationally capable biological systems has long been an aim of synthetic biology. The potential utility of biocomputing devices ranges from biosafety and environmental applications to diagnosis and personalised medicine. Here we present work for the design of bacterial computers which use spatial patterning to process information. Our computers are composed of bacterial colonies which, inspired by patterning in embryo development, receive information in the form of diffusible morphogen-like signals. A computation is encoded by the physical locations of the input sources of morphogen and the output receiver colonies. We demonstrate, mathematically and with engineered Escherichia coli, the simple digital logic capability of single bacterial colonies and show how additional colonies are required to build complex functions. Inspired by electronic design automation, an algorithm for designing optimal spatial circuits computing two-level digital functions is presented. This enhances the capability of our system to complex digital functions without increasing the biological complexity. We extend our experimental system to incorporate biosensing colonies as morphogen sources, demonstrating how a diagnostic device might be constructed. Our approach will open up new ways to perform biological computation, with applications in bioengineering, biomaterials and biosensing. Ultimately, these computational bacterial communities will help us explore information processing in natural biological systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546773">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546773" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546773">
        <p class="paperTitle">Truly tiny acoustic biomolecules for ultrasound imaging and therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546773" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546773" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ling, B.; Gungoren, B.; Yao, Y.; Dutka, P.; Smith, C. A. B.; Lee, J.; Swift, M. B.; Shapiro, M. G.</p>
        <p class="info">Score: 25.5, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546773' target='https://doi.org/10.1101/2023.06.27.546773'> 10.1101/2023.06.27.546773</a></p>
        <p class="abstract">Nanotechnology offers significant advantages for medical imaging and therapy, including enhanced contrast and precision targeting. However, integrating these benefits into ultrasonography has been challenging due to the size and stability constraints of conventional bubble-based agents. Here we describe bicones, truly tiny acoustic contrast agents based on gas vesicles, a unique class of air-filled protein nanostructures naturally produced in buoyant microbes. We show that these sub-80 nm particles can be effectively detected both in vitro and in vivo, infiltrate tumors via leaky vasculature, deliver potent mechanical effects through ultrasound-induced inertial cavitation, and are easily engineered for molecular targeting, prolonged circulation time, and payload conjugation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547852">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547852" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547852">
        <p class="paperTitle">pUdOs: concise plasmids for bacterial and mammalian cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547852" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547852" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ourida Manigat, F.; Connell, L. B.; Stewart, B. N.; LePabic, A.-R.; Tessier, C. J. G.; Emlaw, J. R.; Calvert, N. D.; Rossl, A.; Shuhendler, A. J.; daCosta, C. J. B.; Campbell-Valois, F.-X.</p>
        <p class="info">Score: 6.5, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547852' target='https://doi.org/10.1101/2023.07.05.547852'> 10.1101/2023.07.05.547852</a></p>
        <p class="abstract">The pUdOs are 28 plasmids of small size combining four different origins of replication and seven selection markers, which together afford flexible use in Escherichia coli and several related gram-negative bacteria. The promoterless multicloning site is insulated from upstream spurious promoters by strong transcription terminators, and contains type IIP or IIS restriction sites for conventional or Golden-gate cloning. pUdOs can be converted into efficient expression vectors through the insertion of a promoter at the user&#39;s discretion. For example, we demonstrate the utility of pUdOs as the backbone for an improved version of a Type III Secretion System reporter in Shigella. In addition, we derive a series of pUdO-based mammalian expression vectors affording distinct levels of expression and transfection efficiencies comparable to commonly used mammalian expression plasmids. Thus, pUdOs could advantageously replace traditional plasmids in a wide variety of cell types and applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548538">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548538" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548538">
        <p class="paperTitle">Highly efficient cellular expression of circular mRNA enables prolonged protein expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548538" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548538" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Unti, M. J.; Jaffrey, S. R.</p>
        <p class="info">Score: 2.0, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548538' target='https://doi.org/10.1101/2023.07.11.548538'> 10.1101/2023.07.11.548538</a></p>
        <p class="abstract">A major problem with mRNA therapeutics is that the mRNA is usually degraded within a few hours after entering the cytosol. This limits the duration and total amount of protein expression. New approaches for in vitro synthesis of circular mRNA have allowed increased levels and duration of protein synthesis from mRNA therapeutics due to the long half-life of circular mRNA. However, it remains difficult to genetically encode circular mRNAs in mammalian cells, which makes it difficult to use circular mRNA in cell-derived therapeutics. Here we describe the adaptation of the Tornado (Twister-optimized RNA for durable overexpression) system to achieve in-cell synthesis of circular mRNAs. We screen different promoters and internal ribosomal entry sites (IRESs), and identify combinations that result in high levels of circular mRNA and protein expression in cells. We show that these circular mRNAs can be packaged into virus-like particles (VLPs) thus enabling prolonged protein expression. Overall, these data describe a platform for synthesis of circular mRNAs and how these circular mRNAs can markedly enhance the ability of VLPs to function as a mRNA delivery tool.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.11.523572">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.11.523572" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.11.523572">
        <p class="paperTitle">T7 phage-assisted evolution of riboswitches using error-prone replication and dual selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.11.523572" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.11.523572" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goicoechea, E.; Jaramillo, A.</p>
        <p class="info">Score: 19.1, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.11.523572' target='https://doi.org/10.1101/2023.01.11.523572'> 10.1101/2023.01.11.523572</a></p>
        <p class="abstract">Leveraging riboswitches, non-coding mRNA fragments pivotal to gene regulation, poses a challenge in effectively selecting and enriching these functional genetic sensors, which can toggle between ON and OFF states in response to their cognate inducers. Here, we show our engineered phage T7, enabling the evolution of a theophylline riboswitch. We have replaced T7s DNA polymerase with a transcription factor controlled by a theophylline riboswitch and have created two types of host environments to propagate the engineered phage. Both types host an error-prone T7 DNA polymerase regulated by a T7 promoter along with another critical gene - either cmk or pifA, depending on the host type. The cmk gene is necessary for T7 replication and is used in the first host type for selection in the riboswitchs ON state. Conversely, the second host type incorporates the pifA gene, leading to abortive T7 infections and used for selection in the riboswitchs OFF state. This dual-selection system, termed T7AE, was then applied to a library of 65,536 engineered T7 phages, each carrying randomized riboswitch variants. Through successive passage in both host types with and without theophylline, we observed an enrichment of phages encoding functional riboswitches that conferred a fitness advantage to the phage in both hosts. The T7AE technique thereby opens new pathways for the evolution and advancement of gene switches, including non-coding RNA-based switches, setting the stage for significant strides in synthetic biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548570">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548570" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548570">
        <p class="paperTitle">Efficient PCR-based gene targeting in isolates of the non-conventional yeast Debaryomyces hansenii</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548570" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548570" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alhajouj, S.; Turkolmez, S.; Abalkhail, T.; Alwan, Z. H. O.; Gilmour, D. J.; Mitchell, P. J.; Hettema, E. H.</p>
        <p class="info">Score: 1.2, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548570' target='https://doi.org/10.1101/2023.07.11.548570'> 10.1101/2023.07.11.548570</a></p>
        <p class="abstract">Debaryomyces hansenii is a yeast with considerable biotechnological potential as an osmotolerant, stress tolerant oleaginous microbe. However, targeted genome modification tools are limited and require a strain with auxotrophic markers. Gene targeting by homologous recombination has been reported to be inefficient but here we describe a set of reagents and a method that allows gene targeting at high efficiency in wild type isolates. It uses a simple PCR-based amplification that extends a completely heterologous selectable marker with 50 bp flanks identical to the target site in the genome. Transformants integrate the PCR product through homologous recombination at high frequency (&gt;75%). We illustrate the potential of this method by disrupting genes at high efficiency and by expressing a heterologous protein from a safe chromosomal harbour site. These methods should stimulate and facilitate further analysis of D. hansenii strains and open the way to engineer strains for biotechnology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547106">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547106" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547106">
        <p class="paperTitle">A Platform for Controlled Template-Independent Enzymatic Synthesis of RNA Oligonucleotides and Therapeutics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547106" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547106" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wiegand, D. J.; Rittichier, J.; Meyer, E.; Lee, H.; Conway, N. J.; Ahlstedt, D.; Yurtsever, Z.; Rainone, D.; Kuru, E.; Church, G.</p>
        <p class="info">Score: 4.9, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547106' target='https://doi.org/10.1101/2023.06.29.547106'> 10.1101/2023.06.29.547106</a></p>
        <p class="abstract">Therapeutic RNA oligonucleotides have shown tremendous potential to manage and treat disease, yet current manufacturing methods cannot deliver on this promise. Here, we report the development and optimization of a novel, aqueous-based, template-independent enzymatic RNA oligonucleotide synthesis platform as an alternative to traditional chemical methodologies. Our platform is made possible by reversible terminator nucleoside triphosphates and an enzyme capable of their incorporation. We show that many common therapeutic RNA modifications are compatible with our process and demonstrate the enzymatic synthesis of natural and modified oligonucleotides in both liquid and solid phases. Our platform offers many unique advantages over chemical synthesis, including the realization of a more sustainable process to produce therapeutic RNA oligonucleotides.

One-Sentence SummaryAn enzyme and novel nucleotide building blocks are used to synthesize RNA oligonucleotides template independently under aqueous conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548074" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548074">
        <p class="paperTitle">Exploring alternative quorum sensing model structures and quorum quenching strategies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cimolato, C.; Bellato, M.; Selvaggio, G.; Marchetti, L.; Giordano, G.; Schenato, L.</p>
        <p class="info">Score: 1.0, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548074' target='https://doi.org/10.1101/2023.07.07.548074'> 10.1101/2023.07.07.548074</a></p>
        <p class="abstract">Bacterial quorum sensing (QS) is a cell-to-cell communication mechanism through which bacteria share information about cell density, and tune gene expression accordingly. Pathogens exploit QS to orchestrate virulence and regulate the expression of genes related to antimicrobial resistance. Despite the vast literature on QS, the properties of the underlying molecular network are not entirely clear. We compare two synthetic QS circuit architectures: in the first, a single positive feedback loop autoinduces the synthesis of the signal molecule; the second includes an additional positive feedback loop enhancing the synthesis of the signal molecule receptor. Our comprehensive analysis of the two systems and their equilibria highlights the differences in the bistable and hysteretic behaviors of the alternative QS structures. Finally, we investigate three different QS inhibition approaches; numerical analysis predicts their effect on the steady-state behavior of the two different QS models, revealing critical parameter thresholds that guarantee an effective QS suppression.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
