<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A three-node Turing gene circuit forms periodic spatial patterns in bacteria
Authors: Tica, J.; Oliver Huidobro, M.; Zhu, T.; Wachter, G. K. A.; H. Pazuki, R.; Tonello, E.; Siebert, H.; Stumpf, M. P. H.; Endres, R. G.; Isalan, M.
Score: 28.7, Published: 2023-10-19 DOI: 10.1101/2023.10.19.563112
Turing patterns1 are well-known self-organising systems that can form spots, stripes, or labyrinths. They represent a major theory of patterning in tissue organisation, due to their remarkable similarity to some natural patterns, such as skin pigmentation in zebrafish2, digit spacing3,4, and many others.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="A three-node Turing gene circuit forms periodic spatial patterns in bacteria
Authors: Tica, J.; Oliver Huidobro, M.; Zhu, T.; Wachter, G. K. A.; H. Pazuki, R.; Tonello, E.; Siebert, H.; Stumpf, M. P. H.; Endres, R. G.; Isalan, M.
Score: 28.7, Published: 2023-10-19 DOI: 10.1101/2023.10.19.563112
Turing patterns1 are well-known self-organising systems that can form spots, stripes, or labyrinths. They represent a major theory of patterning in tissue organisation, due to their remarkable similarity to some natural patterns, such as skin pigmentation in zebrafish2, digit spacing3,4, and many others." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-25T10:39:04+00:00" />
<meta property="article:modified_time" content="2023-10-25T10:39:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="A three-node Turing gene circuit forms periodic spatial patterns in bacteria
Authors: Tica, J.; Oliver Huidobro, M.; Zhu, T.; Wachter, G. K. A.; H. Pazuki, R.; Tonello, E.; Siebert, H.; Stumpf, M. P. H.; Endres, R. G.; Isalan, M.
Score: 28.7, Published: 2023-10-19 DOI: 10.1101/2023.10.19.563112
Turing patterns1 are well-known self-organising systems that can form spots, stripes, or labyrinths. They represent a major theory of patterning in tissue organisation, due to their remarkable similarity to some natural patterns, such as skin pigmentation in zebrafish2, digit spacing3,4, and many others."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "A three-node Turing gene circuit forms periodic spatial patterns in bacteria\nAuthors: Tica, J.; Oliver Huidobro, M.; Zhu, T.; Wachter, G. K. A.; H. Pazuki, R.; Tonello, E.; Siebert, H.; Stumpf, M. P. H.; Endres, R. G.; Isalan, M.\nScore: 28.7, Published: 2023-10-19 DOI: 10.1101/2023.10.19.563112\nTuring patterns1 are well-known self-organising systems that can form spots, stripes, or labyrinths. They represent a major theory of patterning in tissue organisation, due to their remarkable similarity to some natural patterns, such as skin pigmentation in zebrafish2, digit spacing3,4, and many others.",
  "keywords": [
    
  ],
  "articleBody": " A three-node Turing gene circuit forms periodic spatial patterns in bacteria\nAuthors: Tica, J.; Oliver Huidobro, M.; Zhu, T.; Wachter, G. K. A.; H. Pazuki, R.; Tonello, E.; Siebert, H.; Stumpf, M. P. H.; Endres, R. G.; Isalan, M.\nScore: 28.7, Published: 2023-10-19 DOI: 10.1101/2023.10.19.563112\nTuring patterns1 are well-known self-organising systems that can form spots, stripes, or labyrinths. They represent a major theory of patterning in tissue organisation, due to their remarkable similarity to some natural patterns, such as skin pigmentation in zebrafish2, digit spacing3,4, and many others. The involvement of Turing patterns in biology has been debated because of their stringent fine-tuning requirements, where patterns only occur within a small subset of parameters5,6. This has complicated the engineering of a synthetic gene circuit for Turing patterns from first principles, even though natural genetic Turing networks have been successfully identified4,7. Here, we engineered a synthetic genetic reaction-diffusion system where three nodes interact according to a non-classical Turing network with improved parametric robustness6. The system was optimised in E. coli and reproducibly generated stationary, periodic, concentric stripe patterns in growing colonies. The patterns were successfully reproduced with a partial differential equation model, in a parameter regime obtained by fitting to experimental data. Our synthetic Turing system can contribute to novel nanotechnologies, such as patterned biomaterial deposition8,9, and provide insights into developmental patterning programs10.\nTowards AI-designed genomes using a variational autoencoder\nAuthors: Dudek, N. K.; Precup, D.\nScore: 22.1, Published: 2023-10-22 DOI: 10.1101/2023.10.22.563484\nSynthetic biology holds great promise for bioengineering applications such as environmental bioremediation, probiotic formulation, and production of renewable biofuels. Humans' capacity to design biological systems from scratch is limited by their sheer size and complexity. We introduce a framework for training a machine learning model to learn the basic genetic principles underlying the gene composition of bacterial genomes. Our variational autoencoder model, DeepGenomeVector, was trained to take as input corrupted bacterial genetic blueprints (i.e. complete gene sets, henceforth \"genome vectors\") in which most genes had been \"removed\", and re-create the original. The resulting model effectively captures the complex dependencies in genomic networks, as evaluated by both qualitative and quantitative metrics. An in-depth functional analysis of a generated gene vector shows that its encoded pathways are interconnected and nearly complete. On the test set, where the model's ability to re-generate the original, uncorrupted genome vector was evaluated, an AUC score of 0.98 and an F1 score of 0.82 provide support for the model's ability to generate diverse, high-quality genome vectors. This work showcases the power of machine learning approaches for synthetic biology and highlights the possibility that just as humans can design an AI that animates a robot, AIs may one day be able to design a genomic blueprint that animates a carbon-based cell.\nDNA nanostructure-templated antibody complexes provide insights into the geometric requirements of human complement cascade activation\nAuthors: Abendstein, L.; Noteborn, W.; Veenman, L.; Dijkstra, D.; Bovenkamp, F. v. d.; Trouw, L.; Sharp, T.\nScore: 19.4, Published: 2023-10-20 DOI: 10.1101/2023.10.20.562751\nThe classical complement pathway is activated by antigen-bound IgG antibodies. Monomeric IgG must oligomerize to activate complement via the hexameric C1q complex, and hexamerizing mutants of IgG appear as promising therapeutic candidates. However, structural data have shown that it is not necessary to bind all six C1q arms to initiate complement, revealing a symmetry mismatch between C1 and the hexameric IgG complex, which has not been adequately explained. Here we use DNA nanotechnology to produce specific nanostructures to template antigens, and thereby control IgG valency. These DNA nano templated IgG complexes can activate complement on cell-mimetic lipid membranes, which enabled us to determine the effect of IgG valency on complement activation without the requirement to mutate antibodies. We investigated this using biophysical assays together with 3D cryo-electron tomography. Our data revealed that the cleavage of complement component C4 by the C1 complex is proportional to the number of antigens. Increased IgG valency also translated to better terminal pathway activation and membrane attack complex formation. Together, these data provide insights into how nanopatterning antigen-antibody complexes influence the activation of the C1 complex and suggest routes to modulate complement activation by antibody engineering. Furthermore, to our knowledge this is the first time DNA nanotechnology has been used to study the activation of the complement system.\nTuning and minimizing lipidome composition in Mycoplasma mycoides and the Minimal Cell JCVI-Syn3A\nAuthors: Justice, I. G.; Saenz, J. P.\nScore: 15.6, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563757\nAll cells are encapsulated by a lipid membrane which facilitates the interaction between life and its environment. The property and function of the cell membrane is critically dependent on its lipidome composition which can range from 10s to 100s of unique lipid structures. How life has evolved to exploit such diverse mixtures of lipids has been experimentally challenging to address. In this study, we introduce a simple approach to minimize and tune lipidome composition in Mycoplasma mycoides and the Minimal Cell (JCVI-Syn3A). By varying the lipid composition of the growth medium, and introducing unmodifiable diether phospholipids, we demonstrate that a lipidome composed of two lipids can support life. By systematically reintroducing phospholipid features, we show that acyl chain diversity is more critical for supporting growth than head group diversity. We also reveal significant effects of altering phospholipid chirality on membrane permeability, cellular robustness, and growth, demonstrating how subtle changes in lipid stereochemistry can have profound effects on cell fitness. Our approach to tune mycoplasma lipidome composition introduces a minimal living model membrane system to understand the design principles and the role of lipid complexity in bioengineering of cell membranes.\nAn Antibiotic Degrading Engineered Living Material Platform to Combat Environmental Antibiotic Resistance\nAuthors: Ozkul, G.; Kehribar, E. S.; Seker, U. O. S.\nScore: 14.1, Published: 2023-10-19 DOI: 10.1101/2023.10.19.563050\nPresence of antibiotics in natural water bodies is a growing problem regarding the occurrence of antibiotic resistance among various species. This is mainly caused due to excessive use of medical and veterinary antibiotics as well as the lack of effective treatment processes for eliminating residual antibiotic from wastewaters. In this study, we introduce a genetically engineered biomaterial as a solution for effective degradation of one of the dominantly found antibiotics in natural water bodies. Our biomaterial harnesses laccase type enzymes which are known to attack specific type of antibiotics, i.e. fluoroquinolone type synthetic antibiotics, and as a result degradation occurs. The engineered biomaterial is built using Escherichia coli biofilm protein CsgA as a scaffold, which is fused separately to two different laccase enzymes with the SpyTag - SpyCatcher peptide-protein duo. The final material is implemented as a part of a living material system. Our proposed system can be integrated as living antibiotic degrading materials to help the combat with antibiotic resistance.\nAllosteric regulation of a molecular motor through de novo protein design\nAuthors: Cross, J. A.; Dawson, W. M.; Shulka, S. R.; Weijman, J. F.; Mantell, J.; Dodding, M. P.; Woolfson, D. N.\nScore: 6.8, Published: 2023-10-17 DOI: 10.1101/2023.10.17.562760\nMany enzymes are allosterically regulated. Our ability to manipulate these structural changes is limited. Here we install an allosteric switch into the kinesin-1 microtubule motor in vitro and in cells. Kinesin-1 is a heterotetramer that accesses open active and closed auto-inhibited states. The equilibrium between these centres on a flexible elbow within a complex coiled-coil architecture. We target the elbow to engineer a closed state that can be opened with a de novo designed peptide. The alternative states are modelled computationally and confirmed by biophysical measurements and electron microscopy. In cells, peptide-driven activation increases kinesin transport, demonstrating a primary role for conformational switching in regulating motor activity. The designs are enabled by our understanding of ubiquitous coiled-coil structures, opening possibilities for controlling other protein activities. One Sentence SummaryDe novo peptide and protein design are used to engineer an allosteric switch into kinesin-1 motors in vitro and directly in cells.\nConstraints on Lanthanide Separation by Selective Biosorption\nAuthors: Anderson, C.; Medin, S.; Adair, J.; Demopoulos, B.; Eneli, E.; Kuelbs, C.; Lee, J. J.; Sheppard, T. J.; Sinar, D.; Thurston, Z.; Xu, M.; Zhang, K.; Barstow, B.\nScore: 6.6, Published: 2023-10-20 DOI: 10.1101/2023.10.18.562985\nRare Earth Elements (REE) are essential ingredients of sustainable energy technologies, but separation of lanthanides is considered one of the hardest problems in chemistry today. Biosorption, where molecules adsorb to the surface of biological materials, offers a sustainable alternative to environmentally harmful solvent extractions currently used for REE separations. The REE-biosorption capabilities of some microorganisms already allow for REE separations that under specialized conditions are competitive with solvent extractions. Our recent work has discovered the genetic basis for REE biosorption selectivity. However, it is unclear if the small changes to selectivity produced by changes to single genetic loci could allow biosorption to leapfrog existing REE separation technologies. In this article we present three models of lanthanide separation by biosorption and desorption. The first model shows that if a biosorbing microbe behaves as it if has a single type of binding site with small preference for one lanthanide, then the small changes in selectivity produced by changes to a single genetic locus could reduce the length of a separation process by {approx} 25%. Large multi-locus gene edits could reduce a separation process length by almost 90%. On the other hand, if the microbe contains multiple sites each with a distinct preference for an individual lanthanide, then separations become challenging and larger genetic edits would be needed to enable high purity. However, even if these large genetic edits are not possible, high purity separations could still be possible by combining multiple microbes, each with small increases in selectivity, with the first designed to enrich for the target lanthanide, while subsequent microbes are designed to remove contaminants.\nBacteriocin peer selection for the production of antibiotic selection free biotherapeutic pDNA\nAuthors: El Bakkoury, M.; P. Gomez de Cadinanos, L.; Gabant, P.\nScore: 4.3, Published: 2023-10-23 DOI: 10.1101/2023.10.23.563565\nPlasmid vectors are well established tools used to genetically engineer bacteria both in the laboratory and at industrial scale. The past few decades have seen a rising interest in the use of plasmid DNA (pDNA) for biotherapeutic applications. This interest is a strong driver for the development of technologies to increase pDNA production at biopharmaceutical scale in terms of decreasing production costs and meeting regulatory requirements. Although cell free technologies are emerging, pDNA vectors are still produced by fermentation in Escherichia coli strains. As plasmids are extra-chromosomic molecules there is a probability of losing a certain ratio within the E. coli population during the fermentation process leading to a decrease of DNA production efficiency. Maintaining pDNA in the population is thus a key element to reach efficient and robust production. Traditionally, antibiotic resistance genes and antibiotics have been used to generate a selective pressure to ensure pDNA stability in the microbial population during the production process. Nowadays, having an antibiotic resistance gene in the pDNA coding sequence represents a limitation both for safety and legal requirements and in terms of production yield. For this reason, we have developed a pDNA antibiotic-free bacteriocin-based selection system, based on the genes involved in the production, processing, secretion and immunity of the bacteriocin microcin V. Our approach is based on the peer pressure exerted by the bacteriocin and does not rely on the addition of any selective agent in the medium to limit population drift and ensure plasmid stability. This novel antibiotic-free approach may be applied to any pDNA vector in different E. coli strains and expands their potential applications in both animal and human health as delivery vectors for biotherapeutics.\nLight-driven synchronization of optogenetic clocks\nAuthors: Cannarsa, M. C.; Liguori, F.; Pellicciotta, N.; Frangipane, G.; Di Leonardo, R.\nScore: 2.8, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563722\nSynthetic genetic oscillators can serve as internal clocks within engineered cells to program periodic expression. However, cell-to-cell variability introduces a dispersion in the characteristics of these clocks that drives the population to complete desynchronization. Here we introduce the optorepressilator, an optically controllable genetic clock that combines the repressilator, a three-node synthetic network in E. coli, with an optogenetic module enabling to reset, delay, or advance its phase using optical inputs. We demonstrate that a population of optorepressilators can be synchronized by transient green light exposure or entrained to oscillate indefinitely by a train of short pulses, through a mechanism reminiscent of natural circadian clocks. Furthermore, we investigate the system's response to detuned external stimuli observing multiple regimes of global synchronization. Integrating experiments and mathematical modeling, we show that the entrainment mechanism is robust and can be understood quantitatively from single cell to population level.\nA discovery platform to identify inducible synthetic circuitry from varied microbial sources\nAuthors: Robinson, C. M.; Carreno, D.; Weber, T.; Chen, Y.; Riglar, D. T.\nScore: 17.8, Published: 2023-10-13 DOI: 10.1101/2023.10.13.562223\nGut-resident microbes encode a variety of systems capable of sophisticated sensing and conditional gene expression control within the mammalian gut during health and disease. Bacterial synthetic biology approaches, such as whole-cell biosensing and sense-and-respond therapeutics, aim to re-purpose such systems for research and clinical applications; however, the toolbox of well characterised in vivo and disease-related inducible systems remains limited. Here, we dramatically expand the flexibility and power of a high-throughput approach to biosensor evaluation using bacterial memory circuits encoded in a gut commensal E. coli species. We construct a novel library of bacterial two-component system (TCS) based biosensors sourced from gut pathogens, tagged with unique DNA barcodes using a single-pot construction method. Using our pipeline, we evaluate sensor activity and performance heterogeneity across in vitro and in vivo conditions leading to novel sensor discovery. In the process, we optimise library construction, sample quality thresholds, and parameters for engineered bacterial biosensor screening in the murine gut. This approach can be applied to transcriptionally activated sensing elements of any type and will allow rapid development of new biosensors that can advance synthetic biology approaches for complex environments, such as the gut.\n",
  "wordCount" : "2296",
  "inLanguage": "en",
  "datePublished": "2023-10-25T10:39:04Z",
  "dateModified": "2023-10-25T10:39:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on October 25, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.563112">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.563112" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.563112">
        <p class="paperTitle">A three-node Turing gene circuit forms periodic spatial patterns in bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.563112" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.563112" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tica, J.; Oliver Huidobro, M.; Zhu, T.; Wachter, G. K. A.; H. Pazuki, R.; Tonello, E.; Siebert, H.; Stumpf, M. P. H.; Endres, R. G.; Isalan, M.</p>
        <p class="info">Score: 28.7, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.563112' target='https://doi.org/10.1101/2023.10.19.563112'> 10.1101/2023.10.19.563112</a></p>
        <p class="abstract">Turing patterns1 are well-known self-organising systems that can form spots, stripes, or labyrinths. They represent a major theory of patterning in tissue organisation, due to their remarkable similarity to some natural patterns, such as skin pigmentation in zebrafish2, digit spacing3,4, and many others. The involvement of Turing patterns in biology has been debated because of their stringent fine-tuning requirements, where patterns only occur within a small subset of parameters5,6. This has complicated the engineering of a synthetic gene circuit for Turing patterns from first principles, even though natural genetic Turing networks have been successfully identified4,7. Here, we engineered a synthetic genetic reaction-diffusion system where three nodes interact according to a non-classical Turing network with improved parametric robustness6. The system was optimised in E. coli and reproducibly generated stationary, periodic, concentric stripe patterns in growing colonies. The patterns were successfully reproduced with a partial differential equation model, in a parameter regime obtained by fitting to experimental data. Our synthetic Turing system can contribute to novel nanotechnologies, such as patterned biomaterial deposition8,9, and provide insights into developmental patterning programs10.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.22.563484">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.22.563484" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.22.563484">
        <p class="paperTitle">Towards AI-designed genomes using a variational autoencoder</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.22.563484" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.22.563484" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dudek, N. K.; Precup, D.</p>
        <p class="info">Score: 22.1, Published: 2023-10-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.22.563484' target='https://doi.org/10.1101/2023.10.22.563484'> 10.1101/2023.10.22.563484</a></p>
        <p class="abstract">Synthetic biology holds great promise for bioengineering applications such as environmental bioremediation, probiotic formulation, and production of renewable biofuels. Humans&#39; capacity to design biological systems from scratch is limited by their sheer size and complexity. We introduce a framework for training a machine learning model to learn the basic genetic principles underlying the gene composition of bacterial genomes. Our variational autoencoder model, DeepGenomeVector, was trained to take as input corrupted bacterial genetic blueprints (i.e. complete gene sets, henceforth &#34;genome vectors&#34;) in which most genes had been &#34;removed&#34;, and re-create the original. The resulting model effectively captures the complex dependencies in genomic networks, as evaluated by both qualitative and quantitative metrics. An in-depth functional analysis of a generated gene vector shows that its encoded pathways are interconnected and nearly complete. On the test set, where the model&#39;s ability to re-generate the original, uncorrupted genome vector was evaluated, an AUC score of 0.98 and an F1 score of 0.82 provide support for the model&#39;s ability to generate diverse, high-quality genome vectors. This work showcases the power of machine learning approaches for synthetic biology and highlights the possibility that just as humans can design an AI that animates a robot, AIs may one day be able to design a genomic blueprint that animates a carbon-based cell.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.562751">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.562751" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.562751">
        <p class="paperTitle">DNA nanostructure-templated antibody complexes provide insights into the geometric requirements of human complement cascade activation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.562751" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.562751" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abendstein, L.; Noteborn, W.; Veenman, L.; Dijkstra, D.; Bovenkamp, F. v. d.; Trouw, L.; Sharp, T.</p>
        <p class="info">Score: 19.4, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.562751' target='https://doi.org/10.1101/2023.10.20.562751'> 10.1101/2023.10.20.562751</a></p>
        <p class="abstract">The classical complement pathway is activated by antigen-bound IgG antibodies. Monomeric IgG must oligomerize to activate complement via the hexameric C1q complex, and hexamerizing mutants of IgG appear as promising therapeutic candidates. However, structural data have shown that it is not necessary to bind all six C1q arms to initiate complement, revealing a symmetry mismatch between C1 and the hexameric IgG complex, which has not been adequately explained. Here we use DNA nanotechnology to produce specific nanostructures to template antigens, and thereby control IgG valency. These DNA nano templated IgG complexes can activate complement on cell-mimetic lipid membranes, which enabled us to determine the effect of IgG valency on complement activation without the requirement to mutate antibodies. We investigated this using biophysical assays together with 3D cryo-electron tomography. Our data revealed that the cleavage of complement component C4 by the C1 complex is proportional to the number of antigens. Increased IgG valency also translated to better terminal pathway activation and membrane attack complex formation. Together, these data provide insights into how nanopatterning antigen-antibody complexes influence the activation of the C1 complex and suggest routes to modulate complement activation by antibody engineering. Furthermore, to our knowledge this is the first time DNA nanotechnology has been used to study the activation of the complement system.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563757">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563757" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563757">
        <p class="paperTitle">Tuning and minimizing lipidome composition in Mycoplasma mycoides and the Minimal Cell JCVI-Syn3A</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563757" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563757" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Justice, I. G.; Saenz, J. P.</p>
        <p class="info">Score: 15.6, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563757' target='https://doi.org/10.1101/2023.10.24.563757'> 10.1101/2023.10.24.563757</a></p>
        <p class="abstract">All cells are encapsulated by a lipid membrane which facilitates the interaction between life and its environment. The property and function of the cell membrane is critically dependent on its lipidome composition which can range from 10s to 100s of unique lipid structures. How life has evolved to exploit such diverse mixtures of lipids has been experimentally challenging to address. In this study, we introduce a simple approach to minimize and tune lipidome composition in Mycoplasma mycoides and the Minimal Cell (JCVI-Syn3A). By varying the lipid composition of the growth medium, and introducing unmodifiable diether phospholipids, we demonstrate that a lipidome composed of two lipids can support life. By systematically reintroducing phospholipid features, we show that acyl chain diversity is more critical for supporting growth than head group diversity. We also reveal significant effects of altering phospholipid chirality on membrane permeability, cellular robustness, and growth, demonstrating how subtle changes in lipid stereochemistry can have profound effects on cell fitness. Our approach to tune mycoplasma lipidome composition introduces a minimal living model membrane system to understand the design principles and the role of lipid complexity in bioengineering of cell membranes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.563050">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.563050" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.563050">
        <p class="paperTitle">An Antibiotic Degrading Engineered Living Material Platform to Combat Environmental Antibiotic Resistance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.563050" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.563050" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ozkul, G.; Kehribar, E. S.; Seker, U. O. S.</p>
        <p class="info">Score: 14.1, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.563050' target='https://doi.org/10.1101/2023.10.19.563050'> 10.1101/2023.10.19.563050</a></p>
        <p class="abstract">Presence of antibiotics in natural water bodies is a growing problem regarding the occurrence of antibiotic resistance among various species. This is mainly caused due to excessive use of medical and veterinary antibiotics as well as the lack of effective treatment processes for eliminating residual antibiotic from wastewaters. In this study, we introduce a genetically engineered biomaterial as a solution for effective degradation of one of the dominantly found antibiotics in natural water bodies. Our biomaterial harnesses laccase type enzymes which are known to attack specific type of antibiotics, i.e. fluoroquinolone type synthetic antibiotics, and as a result degradation occurs. The engineered biomaterial is built using Escherichia coli biofilm protein CsgA as a scaffold, which is fused separately to two different laccase enzymes with the SpyTag - SpyCatcher peptide-protein duo. The final material is implemented as a part of a living material system. Our proposed system can be integrated as living antibiotic degrading materials to help the combat with antibiotic resistance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562760">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562760" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562760">
        <p class="paperTitle">Allosteric regulation of a molecular motor through de novo protein design</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562760" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562760" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cross, J. A.; Dawson, W. M.; Shulka, S. R.; Weijman, J. F.; Mantell, J.; Dodding, M. P.; Woolfson, D. N.</p>
        <p class="info">Score: 6.8, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562760' target='https://doi.org/10.1101/2023.10.17.562760'> 10.1101/2023.10.17.562760</a></p>
        <p class="abstract">Many enzymes are allosterically regulated. Our ability to manipulate these structural changes is limited. Here we install an allosteric switch into the kinesin-1 microtubule motor in vitro and in cells. Kinesin-1 is a heterotetramer that accesses open active and closed auto-inhibited states. The equilibrium between these centres on a flexible elbow within a complex coiled-coil architecture. We target the elbow to engineer a closed state that can be opened with a de novo designed peptide. The alternative states are modelled computationally and confirmed by biophysical measurements and electron microscopy. In cells, peptide-driven activation increases kinesin transport, demonstrating a primary role for conformational switching in regulating motor activity. The designs are enabled by our understanding of ubiquitous coiled-coil structures, opening possibilities for controlling other protein activities.

One Sentence SummaryDe novo peptide and protein design are used to engineer an allosteric switch into kinesin-1 motors in vitro and directly in cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.562985">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.562985" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.562985">
        <p class="paperTitle">Constraints on Lanthanide Separation by Selective Biosorption</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.562985" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.562985" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Anderson, C.; Medin, S.; Adair, J.; Demopoulos, B.; Eneli, E.; Kuelbs, C.; Lee, J. J.; Sheppard, T. J.; Sinar, D.; Thurston, Z.; Xu, M.; Zhang, K.; Barstow, B.</p>
        <p class="info">Score: 6.6, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.562985' target='https://doi.org/10.1101/2023.10.18.562985'> 10.1101/2023.10.18.562985</a></p>
        <p class="abstract">Rare Earth Elements (REE) are essential ingredients of sustainable energy technologies, but separation of lanthanides is considered one of the hardest problems in chemistry today. Biosorption, where molecules adsorb to the surface of biological materials, offers a sustainable alternative to environmentally harmful solvent extractions currently used for REE separations. The REE-biosorption capabilities of some microorganisms already allow for REE separations that under specialized conditions are competitive with solvent extractions. Our recent work has discovered the genetic basis for REE biosorption selectivity. However, it is unclear if the small changes to selectivity produced by changes to single genetic loci could allow biosorption to leapfrog existing REE separation technologies. In this article we present three models of lanthanide separation by biosorption and desorption. The first model shows that if a biosorbing microbe behaves as it if has a single type of binding site with small preference for one lanthanide, then the small changes in selectivity produced by changes to a single genetic locus could reduce the length of a separation process by {approx} 25%. Large multi-locus gene edits could reduce a separation process length by almost 90%. On the other hand, if the microbe contains multiple sites each with a distinct preference for an individual lanthanide, then separations become challenging and larger genetic edits would be needed to enable high purity. However, even if these large genetic edits are not possible, high purity separations could still be possible by combining multiple microbes, each with small increases in selectivity, with the first designed to enrich for the target lanthanide, while subsequent microbes are designed to remove contaminants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563565">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563565" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563565">
        <p class="paperTitle">Bacteriocin peer selection for the production of antibiotic selection free biotherapeutic pDNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563565" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563565" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: El Bakkoury, M.; P. Gomez de Cadinanos, L.; Gabant, P.</p>
        <p class="info">Score: 4.3, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563565' target='https://doi.org/10.1101/2023.10.23.563565'> 10.1101/2023.10.23.563565</a></p>
        <p class="abstract">Plasmid vectors are well established tools used to genetically engineer bacteria both in the laboratory and at industrial scale. The past few decades have seen a rising interest in the use of plasmid DNA (pDNA) for biotherapeutic applications. This interest is a strong driver for the development of technologies to increase pDNA production at biopharmaceutical scale in terms of decreasing production costs and meeting regulatory requirements. Although cell free technologies are emerging, pDNA vectors are still produced by fermentation in Escherichia coli strains. As plasmids are extra-chromosomic molecules there is a probability of losing a certain ratio within the E. coli population during the fermentation process leading to a decrease of DNA production efficiency. Maintaining pDNA in the population is thus a key element to reach efficient and robust production. Traditionally, antibiotic resistance genes and antibiotics have been used to generate a selective pressure to ensure pDNA stability in the microbial population during the production process. Nowadays, having an antibiotic resistance gene in the pDNA coding sequence represents a limitation both for safety and legal requirements and in terms of production yield. For this reason, we have developed a pDNA antibiotic-free bacteriocin-based selection system, based on the genes involved in the production, processing, secretion and immunity of the bacteriocin microcin V. Our approach is based on the peer pressure exerted by the bacteriocin and does not rely on the addition of any selective agent in the medium to limit population drift and ensure plasmid stability. This novel antibiotic-free approach may be applied to any pDNA vector in different E. coli strains and expands their potential applications in both animal and human health as delivery vectors for biotherapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563722">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563722" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563722">
        <p class="paperTitle">Light-driven synchronization of optogenetic clocks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563722" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563722" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cannarsa, M. C.; Liguori, F.; Pellicciotta, N.; Frangipane, G.; Di Leonardo, R.</p>
        <p class="info">Score: 2.8, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563722' target='https://doi.org/10.1101/2023.10.24.563722'> 10.1101/2023.10.24.563722</a></p>
        <p class="abstract">Synthetic genetic oscillators can serve as internal clocks within engineered cells to program periodic expression. However, cell-to-cell variability introduces a dispersion in the characteristics of these clocks that drives the population to complete desynchronization. Here we introduce the optorepressilator, an optically controllable genetic clock that combines the repressilator, a three-node synthetic network in E. coli, with an optogenetic module enabling to reset, delay, or advance its phase using optical inputs. We demonstrate that a population of optorepressilators can be synchronized by transient green light exposure or entrained to oscillate indefinitely by a train of short pulses, through a mechanism reminiscent of natural circadian clocks. Furthermore, we investigate the system&#39;s response to detuned external stimuli observing multiple regimes of global synchronization. Integrating experiments and mathematical modeling, we show that the entrainment mechanism is robust and can be understood quantitatively from single cell to population level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562223">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562223" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562223">
        <p class="paperTitle">A discovery platform to identify inducible synthetic circuitry from varied microbial sources</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562223" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562223" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Robinson, C. M.; Carreno, D.; Weber, T.; Chen, Y.; Riglar, D. T.</p>
        <p class="info">Score: 17.8, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562223' target='https://doi.org/10.1101/2023.10.13.562223'> 10.1101/2023.10.13.562223</a></p>
        <p class="abstract">Gut-resident microbes encode a variety of systems capable of sophisticated sensing and conditional gene expression control within the mammalian gut during health and disease. Bacterial synthetic biology approaches, such as whole-cell biosensing and  sense-and-respond therapeutics, aim to re-purpose such systems for research and clinical applications; however, the toolbox of well characterised in vivo and disease-related inducible systems remains limited. Here, we dramatically expand the flexibility and power of a high-throughput approach to biosensor evaluation using bacterial memory circuits encoded in a gut commensal E. coli species. We construct a novel library of bacterial two-component system (TCS) based biosensors sourced from gut pathogens, tagged with unique DNA barcodes using a single-pot construction method. Using our pipeline, we evaluate sensor activity and performance heterogeneity across in vitro and in vivo conditions leading to novel sensor discovery. In the process, we optimise library construction, sample quality thresholds, and parameters for engineered bacterial biosensor screening in the murine gut. This approach can be applied to transcriptionally activated sensing elements of any type and will allow rapid development of new biosensors that can advance synthetic biology approaches for complex environments, such as the gut.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
