<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Sculpting conducting nanopore size and shape through de novo protein design
Authors: Berhanu, S.; Majumder, S.; Muntener, T.; Whitehouse, J.; Berner, C.; Bera, A. K.; Kang, A.; Liang, B.; Khan, G. N.; Sankaran, B.; Tamm, L.; Brockwel, D. J.; Hiller, S.; Radford, S.; Baker, D.; Vorobieva, A. A.
Score: 31.1, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572500
Transmembrane {beta}-barrels (TMBs) are widely used for single molecule DNA and RNA sequencing and have considerable potential for a broad range of sensing and sequencing applications.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Sculpting conducting nanopore size and shape through de novo protein design
Authors: Berhanu, S.; Majumder, S.; Muntener, T.; Whitehouse, J.; Berner, C.; Bera, A. K.; Kang, A.; Liang, B.; Khan, G. N.; Sankaran, B.; Tamm, L.; Brockwel, D. J.; Hiller, S.; Radford, S.; Baker, D.; Vorobieva, A. A.
Score: 31.1, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572500
Transmembrane {beta}-barrels (TMBs) are widely used for single molecule DNA and RNA sequencing and have considerable potential for a broad range of sensing and sequencing applications." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-31T10:38:19+00:00" />
<meta property="article:modified_time" content="2023-12-31T10:38:19+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Sculpting conducting nanopore size and shape through de novo protein design
Authors: Berhanu, S.; Majumder, S.; Muntener, T.; Whitehouse, J.; Berner, C.; Bera, A. K.; Kang, A.; Liang, B.; Khan, G. N.; Sankaran, B.; Tamm, L.; Brockwel, D. J.; Hiller, S.; Radford, S.; Baker, D.; Vorobieva, A. A.
Score: 31.1, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572500
Transmembrane {beta}-barrels (TMBs) are widely used for single molecule DNA and RNA sequencing and have considerable potential for a broad range of sensing and sequencing applications."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Sculpting conducting nanopore size and shape through de novo protein design\nAuthors: Berhanu, S.; Majumder, S.; Muntener, T.; Whitehouse, J.; Berner, C.; Bera, A. K.; Kang, A.; Liang, B.; Khan, G. N.; Sankaran, B.; Tamm, L.; Brockwel, D. J.; Hiller, S.; Radford, S.; Baker, D.; Vorobieva, A. A.\nScore: 31.1, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572500\nTransmembrane {beta}-barrels (TMBs) are widely used for single molecule DNA and RNA sequencing and have considerable potential for a broad range of sensing and sequencing applications.",
  "keywords": [
    
  ],
  "articleBody": " Sculpting conducting nanopore size and shape through de novo protein design\nAuthors: Berhanu, S.; Majumder, S.; Muntener, T.; Whitehouse, J.; Berner, C.; Bera, A. K.; Kang, A.; Liang, B.; Khan, G. N.; Sankaran, B.; Tamm, L.; Brockwel, D. J.; Hiller, S.; Radford, S.; Baker, D.; Vorobieva, A. A.\nScore: 31.1, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572500\nTransmembrane {beta}-barrels (TMBs) are widely used for single molecule DNA and RNA sequencing and have considerable potential for a broad range of sensing and sequencing applications. Current engineering approaches for nanopore sensors are limited to naturally occurring channels such as CsgG, which have evolved to carry out functions very different from sensing, and hence provide sub-optimal starting points. In contrast, de novo protein design can in principle create an unlimited number of new nanopores with any desired properties. Here we describe a general approach to the design of transmembrane {beta}-barrel pores with different diameter and pore geometry. NMR and crystallographic characterization shows that the designs are stably folded with structures close to the design models. We report the first examples of de novo designed TMBs with 10, 12 and 14 stranded {beta}-barrels. The designs have distinct conductances that correlate with their pore diameter, ranging from 110 pS ([~]0.5 nm pore diameter) to 430 pS ([~]1.1 nm pore diameter), and can be converted into sensitive small-molecule sensors with high signal to noise ratio. The capability to generate on demand {beta}-barrel pores of defined geometry opens up fundamentally new opportunities for custom engineering of sequencing and sensing technologies. One sentence summaryDe novo design enables the generation of stable and quite transmembrane beta-barrel nanopores with tailored sizes, shapes and properties.\nSpecies-specific design of artificial promoters by transfer-learning based generative deep-learning model\nAuthors: Xia, Y.; Du, X.; Liu, B.; Guo, S.; Huo, Y.-X.\nScore: 9.6, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573416\nNative prokaryotic promoters share common sequence patterns, but are species dependent. For understudied species with limited data, it is challenging to predict the strength of existing promoters and generate novel promoters. Here, we developed PromoGen, a collection of nucleotide language models to generate species-specific functional promoters, across dozens of species in a data and parameter efficient way. Twenty-seven species-specific models in this collection were finetuned from the pretrained model which was trained on multi-species promoters. When systematically compared with native promoters, the Escherichia coli- and Bacillus subtilis-specific artificial PromoGen-generated promoters (PGPs) were demonstrated to hold all essential features and distribution patterns of native promoters. A regression model was developed to score generated either by PromoGen or by another competitive neural network, and the overall score of PGPs is higher. Encouraged by in silico analysis, we further experimentally characterized twenty-two B. subtilis PGPs in vivo, results showed that four of tested PGPs reached the strong promoter level while all were active. Furthermore, we developed a user-friendly website to generate species-specific promoters for 27 different species by PromoGen. This work presented an efficient deep-learning strategy for de novo species-specific promoter generation even with limited datasets, providing valuable promoter toolboxes especially for the metabolic engineering of understudied microorganisms. O_FIG O_LINKSMALLFIG WIDTH=138 HEIGHT=200 SRC=\"FIGDIR/small/573416v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (33K): org.highwire.dtl.DTLVardef@222d8aorg.highwire.dtl.DTLVardef@1dbd39borg.highwire.dtl.DTLVardef@121943aorg.highwire.dtl.DTLVardef@18a8b30_HPS_FORMAT_FIGEXP M_FIG C_FIG\nP-NADs: PUX-based NAnobody Degraders for Ubiquitin-Independent Degradation of Target Proteins\nAuthors: Wang, J.; Chistov, G.; Zhang, J.; Huntington, B.; Salem, I.; Sandholu, A.; Arold, S. T.\nScore: 2.1, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572486\nTargeted protein degradation (TPD) allows cells to maintain a functional proteome and to rapidly adapt to changing conditions. Methods that repurpose TPD for the deactivation of specific proteins have demonstrated significant potential in therapeutic and research applications. Most of these methods are based on proteolysis targeting chimaera (PROTAC) which link the protein target to an E3 ubiquitin ligase binding moiety, resulting in the ubiquitin-based degradation of the target protein. In this study, we introduce a method for ubiquitin-independent TPD based on nanobody-conjugated plant ubiquitin regulatory X domain-containing (PUX) adaptor proteins. We show that the PUX-based NAnobody Degraders (P-NADs) can unfold a target protein through the Arabidopsis and human orthologues of the CDC48 unfoldase without the need for ubiquitination or initiating motifs. Despite originating from plants, P-NAD plasmids can be transfected into a human cell line, where produced proteins use the endogenous CDC48 machinery for ubiquitin-independent TPD. Thus, P-NADs pave the road for ubiquitin-independent therapeutic TPD approaches. The P-NAD design combined with established in vitro and cellular assays make this system also a versatile platform for elucidating functional aspects of CDC48-based TPD in plants and animals.\nA translation-independent directed evolution strategy to engineer aminoacyl-tRNA synthetases\nAuthors: Soni, C.; Prywes, N.; Hall, M.; Savage, D. F.; Schepartz, A.; Chatterjee, A.\nScore: 17.9, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571473\nUsing directed evolution, engineered aminoacyl-tRNA synthetases (aaRS) have been developed that enable co-translational incorporation of numerous noncanonical amino acids (ncAAs) into proteins in living cells. Until now, the selection of such novel aaRS mutants has relied on coupling their activity to the expression of a reporter protein with a selectable phenotype. However, such translation-dependent selection schemes are incompatible with exotic monomers that diverge structurally from canonical -amino acids and are suboptimal substrates for the ribosome. To enable the ribosomal incorporation of such exotic monomers, a two-step solution is needed: A) Engineering an aaRS to acylate its cognate tRNA with the exotic monomer, without relying on ribosomal translation as a readout, and B) Subsequent engineering of the ribosome to accept the resulting acylated tRNA for translation. Here, we report a platform for aaRS engineering that directly selects for tRNA-acylation without ribosomal translation (START). In START, each distinct aaRS mutant is correlated to a cognate tRNA containing a unique sequence barcode. Acylation by an active aaRS mutant protects the associated barcode-containing tRNAs from an oxidative treatment designed to damage the 3'-terminus of the uncharged tRNAs. Sequencing of these surviving barcode-containing tRNAs is then used to reveal the identity of aaRS mutants that acylated the correlated tRNA sequences. The efficacy of START was demonstrated by identifying novel mutants of the M. alvus pyrrolysyl-tRNA synthetase from a naive library that charge noncanonical amino acids.\nVirus-like DNA tracers for characterizing wastewater-based pathogen monitoring systems\nAuthors: Gopal, A.; Machtinger, A. N.; Wang, B.; Rice, D. P.; McLaren, M. R.; Esvelt, K. M.\nScore: 1.8, Published: 2023-12-25 DOI: 10.1101/2023.12.24.573197\nWastewater-based pathogen monitoring has recently become a ubiquitous method for tracking the abundance and evolution of pathogens in a community or port of entry. But accurate inference of disease incidence from wastewater remains challenging, in large part due to noise and bias arising from wastewater transport and measurement. We developed a new system of tracer particles designed to enhance wastewater monitoring of viral pathogens. These tracers, which consist of synthetic DNA barcodes packaged into non-infectious viral capsids, can be deposited at known amounts and times into the sewer system and measured in wastewater samples to assess features such as assay sensitivity, and the decay and delay in viral signal arising from a shedding event. Here we describe the design, construction, and initial characterization of a 25-plex tracer library. Our tracer library supports simultaneous measurement of up to 5 distinct tracers by qPCR and up to 25 distinct tracers by amplicon or metagenomic sequencing. We demonstrate experimentally that the tracers are biologically inert and exhibit good stability and qPCR-assay performance in wastewater. These features collectively suggest that barcoded, virus-like tracers have the required properties for high-accuracy, high-resolution characterization of wastewater monitoring systems.\nBuilding synthetic biosensors using red blood cell proteins\nAuthors: Dolberg, T. B.; Gunnels, T. F.; Ling, T.; Sarnese, K. A.; Crispino, J. D.; Leonard, J. N.\nScore: 19.9, Published: 2023-12-16 DOI: 10.1101/2023.12.16.571988\nAs the use of engineered cell therapies expands from pioneering efforts in cancer immunotherapy to other applications, an attractive but less explored approach is the use of engineered red blood cells (RBCs). Compared to other cells, RBCs have a very long circulation time and reside in the blood compartment, so they could be ideally suited for applications as sentinel cells that enable in situ sensing and diagnostics. However, we largely lack tools for converting RBCs into biosensors. A unique challenge is that RBCs remodel their membranes during maturation, shedding many membrane components, suggesting that an RBC-specific approach may be needed. Towards addressing this need, here we develop a biosensing architecture built on RBC membrane proteins that are retained through erythropoiesis. This biosensor employs a mechanism in which extracellular ligand binding is transduced into intracellular reconstitution of a split output protein (including either a fluorophore or an enzyme). By comparatively evaluating a range of biosensor architectures, linker types, scaffold choices, and output signals, we identify biosensor designs and design features that confer substantial ligand-induced signal in vitro. Finally, we demonstrate that erythroid precursor cells engineered with our RBC protein biosensors function in vivo. This study establishes a foundation for developing RBC-based biosensors that could ultimately address unmet needs including non-invasive monitoring of physiological signals for a range of diagnostic applications.\nA long-acting prolactin to combat lactation insufficiency\nAuthors: Kready, K.; Doiron, K.; Redfield Chan, K.; Way, J. C.; Justman, Q.; Powe, C. E.; Silver, P. A.\nScore: 26.2, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571886\nHuman infants are born to breastfeed. While 50% of lactating persons struggle to make enough milk, there are no governmentally-approved drugs to enhance lactation1. Here, we engineer a variant of the naturally-occurring driver of lactation, the hormone Prolactin, to increase its serum half-life and produce a viable drug candidate. Our engineered variant, Prolactin-eXtra Long-acting (Prolactin-XL), is comprised of endogenously active human prolactin fused to an engineered human IgG Fc domain designed to overcome the unique drug development challenges specific to the lactating person-infant dyad. Our Prolactin-XL has a serum half-life of 70.9h in mice, 2,625-fold longer than endogenously active prolactin alone (70.9h v. 0.027h). We demonstrate that Prolactin-XL increases milk production and restores growth of pups fed by dams with pharmacologically-ablated lactation. We show that Prolactin-XL-enhanced lactation is accompanied by reversible, lactocyte-driven changes in mammary gland morphology. This work establishes long-acting prolactins as a potentially powerful pharmacologic means to combat insufficient lactation.\nStructure-Infused Protein Language Models\nAuthors: Penaherrera, D.; Koes, D. R.\nScore: 1.2, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571525\nEmbeddings from protein language models (PLMs) capture intricate patterns for protein sequences, enabling more accurate and efficient prediction of protein properties. Incorporating protein structure information as direct input into PLMs results in an improvement on the predictive ability of protein embeddings on downstream tasks. In this work we demonstrate that indirectly infusing structure information into PLMs also leads to performance gains on structure related tasks. The key difference between this framework and others is that at inference time the model does not require access to structure to produce its embeddings.\nIn vitro validated antibody design against multiple therapeutic antigens using generative inverse folding\nAuthors: Shanehsazzadeh, A.; Alverio, J.; Kasun, G.; Levine, S.; Khan, J. A.; Chung, C.; Diaz, N.; Luton, B. K.; Tarter, Y.; McCloskey, C.; Bateman, K. B.; Carter, H.; Chapman, D.; Consbruck, R.; Jaeger, A.; Kohnert, C.; Kopec-Belliveau, G.; Sutton, J. M.; Guo, Z.; Canales, G.; Ejan, K.; Marsh, E.; Ruelos, A.; Ripley, R.; Stoddard, B.; Caguiat, R.; Chapman, K.; Saunders, M.; Sharp, J.; Ganini da Silva, D.; Feltner, A.; Ripley, J.; Bryant, M. E.; Castillo, D.; Meier, J.; Stegmann, C. M.; Moran, K.; Lemke, C.; Abdulhaqq, S.; Klug, L. R.; Bachas, S.\nScore: 32.3, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570889\nDeep learning approaches have demonstrated the ability to design protein sequences given backbone structures [1, 2, 3, 4, 5]. While these approaches have been applied in silico to designing antibody complementarity-determining regions (CDRs), they have yet to be validated in vitro for designing antibody binders, which is the true measure of success for antibody design. Here we describe IgDesign, a deep learning method for antibody CDR design, and demonstrate its robustness with successful binder design for 8 therapeutic antigens. The model is tasked with designing heavy chain CDR3 (HCDR3) or all three heavy chain CDRs (HCDR123) using native backbone structures of antibody-antigen complexes, along with the antigen and antibody framework (FWR) sequences as context. For each of the 8 antigens, we design 100 HCDR3s and 100 HCDR123s, scaffold them into the native antibodys variable region, and screen them for binding against the antigen using surface plasmon resonance (SPR). As a baseline, we screen 100 HCDR3s taken from the models training set and paired with the native HCDR1 and HCDR2. We observe that both HCDR3 design and HCDR123 design outperform this HCDR3-only baseline. IgDesign is the first experimentally validated antibody inverse folding model. It can design antibody binders to multiple therapeutic antigens with high success rates and, in some cases, improved affinities over clinically validated reference antibodies. Antibody inverse folding has applications to both de novo antibody design and lead optimization, making IgDesign a valuable tool for accelerating drug development and enabling therapeutic design.\n",
  "wordCount" : "2112",
  "inLanguage": "en",
  "datePublished": "2023-12-31T10:38:19Z",
  "dateModified": "2023-12-31T10:38:19Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 31, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572500">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572500" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572500">
        <p class="paperTitle">Sculpting conducting nanopore size and shape through de novo protein design</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572500" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572500" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Berhanu, S.; Majumder, S.; Muntener, T.; Whitehouse, J.; Berner, C.; Bera, A. K.; Kang, A.; Liang, B.; Khan, G. N.; Sankaran, B.; Tamm, L.; Brockwel, D. J.; Hiller, S.; Radford, S.; Baker, D.; Vorobieva, A. A.</p>
        <p class="info">Score: 31.1, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572500' target='https://doi.org/10.1101/2023.12.20.572500'> 10.1101/2023.12.20.572500</a></p>
        <p class="abstract">Transmembrane {beta}-barrels (TMBs) are widely used for single molecule DNA and RNA sequencing and have considerable potential for a broad range of sensing and sequencing applications. Current engineering approaches for nanopore sensors are limited to naturally occurring channels such as CsgG, which have evolved to carry out functions very different from sensing, and hence provide sub-optimal starting points. In contrast, de novo protein design can in principle create an unlimited number of new nanopores with any desired properties. Here we describe a general approach to the design of transmembrane {beta}-barrel pores with different diameter and pore geometry. NMR and crystallographic characterization shows that the designs are stably folded with structures close to the design models. We report the first examples of de novo designed TMBs with 10, 12 and 14 stranded {beta}-barrels. The designs have distinct conductances that correlate with their pore diameter, ranging from 110 pS ([~]0.5 nm pore diameter) to 430 pS ([~]1.1 nm pore diameter), and can be converted into sensitive small-molecule sensors with high signal to noise ratio. The capability to generate on demand {beta}-barrel pores of defined geometry opens up fundamentally new opportunities for custom engineering of sequencing and sensing technologies.

One sentence summaryDe novo design enables the generation of stable and quite transmembrane beta-barrel nanopores with tailored sizes, shapes and properties.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573416">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573416" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573416">
        <p class="paperTitle">Species-specific design of artificial promoters by transfer-learning based generative deep-learning model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573416" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573416" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xia, Y.; Du, X.; Liu, B.; Guo, S.; Huo, Y.-X.</p>
        <p class="info">Score: 9.6, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573416' target='https://doi.org/10.1101/2023.12.27.573416'> 10.1101/2023.12.27.573416</a></p>
        <p class="abstract">Native prokaryotic promoters share common sequence patterns, but are species dependent. For understudied species with limited data, it is challenging to predict the strength of existing promoters and generate novel promoters. Here, we developed PromoGen, a collection of nucleotide language models to generate species-specific functional promoters, across dozens of species in a data and parameter efficient way. Twenty-seven species-specific models in this collection were finetuned from the pretrained model which was trained on multi-species promoters. When systematically compared with native promoters, the Escherichia coli- and Bacillus subtilis-specific artificial PromoGen-generated promoters (PGPs) were demonstrated to hold all essential features and distribution patterns of native promoters. A regression model was developed to score generated either by PromoGen or by another competitive neural network, and the overall score of PGPs is higher. Encouraged by in silico analysis, we further experimentally characterized twenty-two B. subtilis PGPs in vivo, results showed that four of tested PGPs reached the strong promoter level while all were active. Furthermore, we developed a user-friendly website to generate species-specific promoters for 27 different species by PromoGen. This work presented an efficient deep-learning strategy for de novo species-specific promoter generation even with limited datasets, providing valuable promoter toolboxes especially for the metabolic engineering of understudied microorganisms.



O_FIG O_LINKSMALLFIG WIDTH=138 HEIGHT=200 SRC=&#34;FIGDIR/small/573416v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (33K):
org.highwire.dtl.DTLVardef@222d8aorg.highwire.dtl.DTLVardef@1dbd39borg.highwire.dtl.DTLVardef@121943aorg.highwire.dtl.DTLVardef@18a8b30_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572486">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572486" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572486">
        <p class="paperTitle">P-NADs: PUX-based NAnobody Degraders for Ubiquitin-Independent Degradation of Target Proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572486" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572486" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, J.; Chistov, G.; Zhang, J.; Huntington, B.; Salem, I.; Sandholu, A.; Arold, S. T.</p>
        <p class="info">Score: 2.1, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572486' target='https://doi.org/10.1101/2023.12.20.572486'> 10.1101/2023.12.20.572486</a></p>
        <p class="abstract">Targeted protein degradation (TPD) allows cells to maintain a functional proteome and to rapidly adapt to changing conditions. Methods that repurpose TPD for the deactivation of specific proteins have demonstrated significant potential in therapeutic and research applications. Most of these methods are based on proteolysis targeting chimaera (PROTAC) which link the protein target to an E3 ubiquitin ligase binding moiety, resulting in the ubiquitin-based degradation of the target protein. In this study, we introduce a method for ubiquitin-independent TPD based on nanobody-conjugated plant ubiquitin regulatory X domain-containing (PUX) adaptor proteins. We show that the PUX-based NAnobody Degraders (P-NADs) can unfold a target protein through the Arabidopsis and human orthologues of the CDC48 unfoldase without the need for ubiquitination or initiating motifs. Despite originating from plants, P-NAD plasmids can be transfected into a human cell line, where produced proteins use the endogenous CDC48 machinery for ubiquitin-independent TPD. Thus, P-NADs pave the road for ubiquitin-independent therapeutic TPD approaches. The P-NAD design combined with established in vitro and cellular assays make this system also a versatile platform for elucidating functional aspects of CDC48-based TPD in plants and animals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571473">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571473" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571473">
        <p class="paperTitle">A translation-independent directed evolution strategy to engineer aminoacyl-tRNA synthetases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571473" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571473" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Soni, C.; Prywes, N.; Hall, M.; Savage, D. F.; Schepartz, A.; Chatterjee, A.</p>
        <p class="info">Score: 17.9, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571473' target='https://doi.org/10.1101/2023.12.13.571473'> 10.1101/2023.12.13.571473</a></p>
        <p class="abstract">Using directed evolution, engineered aminoacyl-tRNA synthetases (aaRS) have been developed that enable co-translational incorporation of numerous noncanonical amino acids (ncAAs) into proteins in living cells. Until now, the selection of such novel aaRS mutants has relied on coupling their activity to the expression of a reporter protein with a selectable phenotype. However, such translation-dependent selection schemes are incompatible with exotic monomers that diverge structurally from canonical -amino acids and are suboptimal substrates for the ribosome. To enable the ribosomal incorporation of such exotic monomers, a two-step solution is needed: A) Engineering an aaRS to acylate its cognate tRNA with the exotic monomer, without relying on ribosomal translation as a readout, and B) Subsequent engineering of the ribosome to accept the resulting acylated tRNA for translation. Here, we report a platform for aaRS engineering that directly selects for tRNA-acylation without ribosomal translation (START). In START, each distinct aaRS mutant is correlated to a cognate tRNA containing a unique sequence barcode. Acylation by an active aaRS mutant protects the associated barcode-containing tRNAs from an oxidative treatment designed to damage the 3&#39;-terminus of the uncharged tRNAs. Sequencing of these surviving barcode-containing tRNAs is then used to reveal the identity of aaRS mutants that acylated the correlated tRNA sequences. The efficacy of START was demonstrated by identifying novel mutants of the M. alvus pyrrolysyl-tRNA synthetase from a naive library that charge noncanonical amino acids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.24.573197">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.24.573197" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.24.573197">
        <p class="paperTitle">Virus-like DNA tracers for characterizing wastewater-based pathogen monitoring systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.24.573197" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.24.573197" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gopal, A.; Machtinger, A. N.; Wang, B.; Rice, D. P.; McLaren, M. R.; Esvelt, K. M.</p>
        <p class="info">Score: 1.8, Published: 2023-12-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.24.573197' target='https://doi.org/10.1101/2023.12.24.573197'> 10.1101/2023.12.24.573197</a></p>
        <p class="abstract">Wastewater-based pathogen monitoring has recently become a ubiquitous method for tracking the abundance and evolution of pathogens in a community or port of entry. But accurate inference of disease incidence from wastewater remains challenging, in large part due to noise and bias arising from wastewater transport and measurement. We developed a new system of tracer particles designed to enhance wastewater monitoring of viral pathogens. These tracers, which consist of synthetic DNA barcodes packaged into non-infectious viral capsids, can be deposited at known amounts and times into the sewer system and measured in wastewater samples to assess features such as assay sensitivity, and the decay and delay in viral signal arising from a shedding event. Here we describe the design, construction, and initial characterization of a 25-plex tracer library. Our tracer library supports simultaneous measurement of up to 5 distinct tracers by qPCR and up to 25 distinct tracers by amplicon or metagenomic sequencing. We demonstrate experimentally that the tracers are biologically inert and exhibit good stability and qPCR-assay performance in wastewater. These features collectively suggest that barcoded, virus-like tracers have the required properties for high-accuracy, high-resolution characterization of wastewater monitoring systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.16.571988">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.16.571988" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.16.571988">
        <p class="paperTitle">Building synthetic biosensors using red blood cell proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.16.571988" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.16.571988" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dolberg, T. B.; Gunnels, T. F.; Ling, T.; Sarnese, K. A.; Crispino, J. D.; Leonard, J. N.</p>
        <p class="info">Score: 19.9, Published: 2023-12-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.16.571988' target='https://doi.org/10.1101/2023.12.16.571988'> 10.1101/2023.12.16.571988</a></p>
        <p class="abstract">As the use of engineered cell therapies expands from pioneering efforts in cancer immunotherapy to other applications, an attractive but less explored approach is the use of engineered red blood cells (RBCs). Compared to other cells, RBCs have a very long circulation time and reside in the blood compartment, so they could be ideally suited for applications as sentinel cells that enable in situ sensing and diagnostics. However, we largely lack tools for converting RBCs into biosensors. A unique challenge is that RBCs remodel their membranes during maturation, shedding many membrane components, suggesting that an RBC-specific approach may be needed. Towards addressing this need, here we develop a biosensing architecture built on RBC membrane proteins that are retained through erythropoiesis. This biosensor employs a mechanism in which extracellular ligand binding is transduced into intracellular reconstitution of a split output protein (including either a fluorophore or an enzyme). By comparatively evaluating a range of biosensor architectures, linker types, scaffold choices, and output signals, we identify biosensor designs and design features that confer substantial ligand-induced signal in vitro. Finally, we demonstrate that erythroid precursor cells engineered with our RBC protein biosensors function in vivo. This study establishes a foundation for developing RBC-based biosensors that could ultimately address unmet needs including non-invasive monitoring of physiological signals for a range of diagnostic applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571886">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571886" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571886">
        <p class="paperTitle">A long-acting prolactin to combat lactation insufficiency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571886" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571886" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kready, K.; Doiron, K.; Redfield Chan, K.; Way, J. C.; Justman, Q.; Powe, C. E.; Silver, P. A.</p>
        <p class="info">Score: 26.2, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571886' target='https://doi.org/10.1101/2023.12.15.571886'> 10.1101/2023.12.15.571886</a></p>
        <p class="abstract">Human infants are born to breastfeed. While 50% of lactating persons struggle to make enough milk, there are no governmentally-approved drugs to enhance lactation1. Here, we engineer a variant of the naturally-occurring driver of lactation, the hormone Prolactin, to increase its serum half-life and produce a viable drug candidate. Our engineered variant, Prolactin-eXtra Long-acting (Prolactin-XL), is comprised of endogenously active human prolactin fused to an engineered human IgG Fc domain designed to overcome the unique drug development challenges specific to the lactating person-infant dyad. Our Prolactin-XL has a serum half-life of 70.9h in mice, 2,625-fold longer than endogenously active prolactin alone (70.9h v. 0.027h). We demonstrate that Prolactin-XL increases milk production and restores growth of pups fed by dams with pharmacologically-ablated lactation. We show that Prolactin-XL-enhanced lactation is accompanied by reversible, lactocyte-driven changes in mammary gland morphology. This work establishes long-acting prolactins as a potentially powerful pharmacologic means to combat insufficient lactation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571525">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571525" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571525">
        <p class="paperTitle">Structure-Infused Protein Language Models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571525" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571525" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Penaherrera, D.; Koes, D. R.</p>
        <p class="info">Score: 1.2, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571525' target='https://doi.org/10.1101/2023.12.13.571525'> 10.1101/2023.12.13.571525</a></p>
        <p class="abstract">Embeddings from protein language models (PLMs) capture intricate patterns for protein sequences, enabling more accurate and efficient prediction of protein properties. Incorporating protein structure information as direct input into PLMs results in an improvement on the predictive ability of protein embeddings on downstream tasks. In this work we demonstrate that indirectly infusing structure information into PLMs also leads to performance gains on structure related tasks. The key difference between this framework and others is that at inference time the model does not require access to structure to produce its embeddings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570889">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570889" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570889">
        <p class="paperTitle">In vitro validated antibody design against multiple therapeutic antigens using generative inverse folding</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570889" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570889" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shanehsazzadeh, A.; Alverio, J.; Kasun, G.; Levine, S.; Khan, J. A.; Chung, C.; Diaz, N.; Luton, B. K.; Tarter, Y.; McCloskey, C.; Bateman, K. B.; Carter, H.; Chapman, D.; Consbruck, R.; Jaeger, A.; Kohnert, C.; Kopec-Belliveau, G.; Sutton, J. M.; Guo, Z.; Canales, G.; Ejan, K.; Marsh, E.; Ruelos, A.; Ripley, R.; Stoddard, B.; Caguiat, R.; Chapman, K.; Saunders, M.; Sharp, J.; Ganini da Silva, D.; Feltner, A.; Ripley, J.; Bryant, M. E.; Castillo, D.; Meier, J.; Stegmann, C. M.; Moran, K.; Lemke, C.; Abdulhaqq, S.; Klug, L. R.; Bachas, S.</p>
        <p class="info">Score: 32.3, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570889' target='https://doi.org/10.1101/2023.12.08.570889'> 10.1101/2023.12.08.570889</a></p>
        <p class="abstract">Deep learning approaches have demonstrated the ability to design protein sequences given backbone structures [1, 2, 3, 4, 5]. While these approaches have been applied in silico to designing antibody complementarity-determining regions (CDRs), they have yet to be validated in vitro for designing antibody binders, which is the true measure of success for antibody design. Here we describe IgDesign, a deep learning method for antibody CDR design, and demonstrate its robustness with successful binder design for 8 therapeutic antigens. The model is tasked with designing heavy chain CDR3 (HCDR3) or all three heavy chain CDRs (HCDR123) using native backbone structures of antibody-antigen complexes, along with the antigen and antibody framework (FWR) sequences as context. For each of the 8 antigens, we design 100 HCDR3s and 100 HCDR123s, scaffold them into the native antibodys variable region, and screen them for binding against the antigen using surface plasmon resonance (SPR). As a baseline, we screen 100 HCDR3s taken from the models training set and paired with the native HCDR1 and HCDR2. We observe that both HCDR3 design and HCDR123 design outperform this HCDR3-only baseline. IgDesign is the first experimentally validated antibody inverse folding model. It can design antibody binders to multiple therapeutic antigens with high success rates and, in some cases, improved affinities over clinically validated reference antibodies. Antibody inverse folding has applications to both de novo antibody design and lead optimization, making IgDesign a valuable tool for accelerating drug development and enabling therapeutic design.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
