<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Neural network extrapolation to distant regions of the protein fitness landscape
Authors: Fahlberg, S. A.; Freschlin, C. R.; Heinzelman, P.; Romero, P. A.
Score: 19.4, Published: 2023-11-09 DOI: 10.1101/2023.11.08.566287
Machine learning (ML) has transformed protein engineering by constructing models of the underlying sequence-function landscape to accelerate the discovery of new biomolecules. ML-guided protein design requires models, trained on local sequence-function information, to accurately predict distant fitness peaks. In this work, we evaluate neural networks&#39; capacity to extrapolate beyond their training data.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Neural network extrapolation to distant regions of the protein fitness landscape
Authors: Fahlberg, S. A.; Freschlin, C. R.; Heinzelman, P.; Romero, P. A.
Score: 19.4, Published: 2023-11-09 DOI: 10.1101/2023.11.08.566287
Machine learning (ML) has transformed protein engineering by constructing models of the underlying sequence-function landscape to accelerate the discovery of new biomolecules. ML-guided protein design requires models, trained on local sequence-function information, to accurately predict distant fitness peaks. In this work, we evaluate neural networks&#39; capacity to extrapolate beyond their training data." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-12T10:37:43+00:00" />
<meta property="article:modified_time" content="2023-11-12T10:37:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Neural network extrapolation to distant regions of the protein fitness landscape
Authors: Fahlberg, S. A.; Freschlin, C. R.; Heinzelman, P.; Romero, P. A.
Score: 19.4, Published: 2023-11-09 DOI: 10.1101/2023.11.08.566287
Machine learning (ML) has transformed protein engineering by constructing models of the underlying sequence-function landscape to accelerate the discovery of new biomolecules. ML-guided protein design requires models, trained on local sequence-function information, to accurately predict distant fitness peaks. In this work, we evaluate neural networks&#39; capacity to extrapolate beyond their training data."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Neural network extrapolation to distant regions of the protein fitness landscape\nAuthors: Fahlberg, S. A.; Freschlin, C. R.; Heinzelman, P.; Romero, P. A.\nScore: 19.4, Published: 2023-11-09 DOI: 10.1101/2023.11.08.566287\nMachine learning (ML) has transformed protein engineering by constructing models of the underlying sequence-function landscape to accelerate the discovery of new biomolecules. ML-guided protein design requires models, trained on local sequence-function information, to accurately predict distant fitness peaks. In this work, we evaluate neural networks\u0026#39; capacity to extrapolate beyond their training data.",
  "keywords": [
    
  ],
  "articleBody": " Neural network extrapolation to distant regions of the protein fitness landscape\nAuthors: Fahlberg, S. A.; Freschlin, C. R.; Heinzelman, P.; Romero, P. A.\nScore: 19.4, Published: 2023-11-09 DOI: 10.1101/2023.11.08.566287\nMachine learning (ML) has transformed protein engineering by constructing models of the underlying sequence-function landscape to accelerate the discovery of new biomolecules. ML-guided protein design requires models, trained on local sequence-function information, to accurately predict distant fitness peaks. In this work, we evaluate neural networks' capacity to extrapolate beyond their training data. We perform model-guided design using a panel of neural network architectures trained on protein G (GB1)-Immunoglobulin G (IgG) binding data and experimentally test thousands of GB1 designs to systematically evaluate the models' extrapolation. We find each model architecture infers markedly different landscapes from the same data, which give rise to unique design preferences. We find simpler models excel in local extrapolation to design high fitness proteins, while more sophisticated convolutional models can venture deep into sequence space to design proteins that fold but are no longer functional. Our findings highlight how each architecture's inductive biases prime them to learn different aspects of the protein fitness landscape.\nMinimization of the E. coli ribosome, aided and optimized by community science\nAuthors: Tangpradabkul, T.; Palo, M. Z.; Townley, J.; Hsu, K. B.; Eterna Participants, ; Smaga, S.; Das, R.; Schepartz, A.\nScore: 15.2, Published: 2023-11-10 DOI: 10.1101/2023.11.09.566451\nThe ribosome is a ribonucleoprotein complex found in all domains of life. Its role is to catalyze protein synthesis, the messenger RNA (mRNA)-templated formation of amide bonds between -amino acid monomers. Amide bond formation occurs within a highly conserved region of the large ribosomal subunit known as the peptidyl transferase center (PTC). Here we describe the stepwise design and characterization of mini-PTC 1.1, a 284-nucleotide RNA that recapitulates many essential features of the Escherichia coli PTC. Mini-PTC 1.1 folds into a PTC-like structure under physiological conditions, even in the absence of r-proteins, and engages small molecule analogs of A- and P-site tRNAs. The sequence of mini-PTC 1.1 differs from the wild type E. coli ribosome at 12 nucleotides that were installed by a cohort of citizen scientists using the on-line video game Eterna. These base changes improve both the secondary structure and tertiary folding of mini-PTC 1.1 as well as its ability to bind small molecule substrate analogs. Here, the combined input from Eterna citizen-scientists and RNA structural analysis provides a robust workflow for the design of a minimal PTC that recapitulates many features of an intact ribosome.\nIncorporation of multiple beta 2-backbones into a protein invivo using an orthogonal aminoacyl-tRNA synthetase\nAuthors: Hamlish, N.; Abramyan, A.; Schepartz, A.\nScore: 14.8, Published: 2023-11-08 DOI: 10.1101/2023.11.07.565973\nSynthesis of sequence-defined biomaterials whose monomer backbones diverge from canonical -amino acids represents the next frontier in protein and biomaterial evolution with the potential to yield better biological therapeutics, bioremediation tools, and biodegradable plastic-like materials. One monomer family of particular interest for biomaterials are {beta}-hydroxy acids. Many natural products contain isolated {beta}-esters, and polymeric {beta}-esters are found in polyhydroxyalkanoate (PHA) polyesters under development as bioplastics and drug encapsulation/delivery systems. Here we report that {beta}2-hydroxy acids possessing both (R) and (S) absolute configuration are excellent substrates for pyrrolysyl-tRNA synthetase (PylRS) enzymes in vitro, and that (S)-{beta}2-hydroxy acids are substrates in cellulo. Using the MaPylRS/MatRNAPyl pair, in conjunction with wild-type E. coli ribosomes and EF-Tu, we report the cellular synthesis of model proteins containing two (S)-{beta}2-hydroxy acid residues at internal positions. Metadynamics simulations provide a rationale for the observed enantioselective preference of the ribosome for the (S)-{beta}2-hydroxy acid backbone and mechanistic insights that inform future ribosomal engineering efforts. As far as we know, this finding represents the first example of an orthogonal synthetase that accepts a {beta}-backbone substrate and the first example of a protein hetero-oligomer containing multiple expanded-backbone monomers produced in cellulo.\nA modular toolbox for the optogenetic deactivation of transcription\nAuthors: Muench, P.; Fiumara, M.; Southern, N.; Coda, D.; Aschenbrenner, S.; Correia, B.; Graeff, J.; Niopek, D.; Mathony, J.\nScore: 9.9, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565805\nLight-controlled transcriptional activation is a commonly used optogenetic strategy that allows researchers to regulate gene expression with high spatiotemporal precision. The vast majority of existing tools are, however, limited to light-triggered induction of gene expression. Here, we inverted this mode of action and created two complementary optogenetic systems capable of efficiently terminating transcriptional activation in response to blue light. First, we designed highly compact regulators, by photo-controlling VP16 transactivation peptide exposure. Then, applying a two-hybrid strategy, we engineered LOOMINA (light off-operated modular inductor of transcriptional activation), a versatile transcriptional control platform for mammalian cells that is highly adaptable and compatible with various effector proteins. Leveraging the flexibility of CRISPR systems, we integrated LOOMINA with Cas9 as a DNA-binding domain to control transcription from various endogenous promoters with exceptionally high dynamic ranges in multiple cell lines, including neuron-like cells. Both functionally and mechanistically, LOOMINA represents a valuable addition to the optogenetic repertoire for transcriptional regulation.\nBatCRISPRi: Bacillus titratable CRISPRi for dynamic control in Bacillus subtilis\nAuthors: Florez, A. F.; Castillo-Hair, S.; Gutierrez-Lopez, L.; Eaton, D.; Paulsson, J.; Garner, E. C.\nScore: 11.7, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565046\nThe discovery of new genes regulating essential biological processes has become increasingly important, and CRISPRi has emerged as a powerful tool for achieving this goal. This method has been used in many model organisms to decrease the expression of specific genes and assess their impact on phenotype. Pooled CRISPRi libraries in bacteria have been particularly useful in discovering new regulators of growth, division, and other biological processes. However, these libraries rely on the induction of dCas9 via an inducible promoter, which can be problematic due to promoter leakiness. This is a widespread phenomenon of any inducible promoter that can result in the unwanted downregulation of genes and the emergence of genetic suppressors when essential genes are knocked down. To overcome this issue, we have developed a novel strategy that eliminates dCas9 leakiness and enables reversible knockdown control using the rapamycin-dependent degron system in Bacillus subtilis. This degron system causes rapid degradation of dCas9, resulting in an almost instant reset of the system. Our results demonstrate that it is possible to achieve zero CRISPRi activity in the uninduced state and full activity in the induced state. This improved CRISPRi system will enable researchers to investigate phenotypic changes more effectively while reducing the undesirable effects of leaky expression and noise in their phenotypic data. Moreover, a rapid degradation system could serve as a tool for dynamic perturbation before compensation mechanisms or stress responses kick in. Finally, this approach can be adapted to other organisms and other promoter-inducible systems, potentially opening up strategies for tighter control of gene expression.\nGADY algorithm: Towards an evolutionary protocol for bottom-up design of synthetic bacteria\nAuthors: Sanchez Iniguez, U.; Lledo Villaescusa, S.; Lahoz-Beltra, R.\nScore: 2.0, Published: 2023-11-08 DOI: 10.1101/2023.11.07.566064\nSynthetic biology applications are currently based on the programming of bacteria with tailor-made circuits designed ad hoc by applying a top-down strategy. We introduce a novel algorithm oriented to design synthetic bacteria according to a bottom-up approach, i.e. via an evolutionary programming algorithm. The proposed algorithm has been referred to as GADY: an acronym for Get signal, Antidote, Die when a killer gene is expressed, emits Yellow fluorescence. We include in this technical note a script of the algorithm in Gro cellular programming language, programming a colony of synthetic bacteria and illustrating its usefulness in a case of bioremediation or elimination of a strain of pathogenic bacteria.\nComputational Design of Periplasmic Binding Protein Biosensors Guided by Molecular Dynamics\nAuthors: O'Shea, J. M.; Richardson, A.; Doerner, P. W.; Wood, C. W.\nScore: 1.5, Published: 2023-11-10 DOI: 10.1101/2023.11.10.566541\nPeriplasmic binding proteins (PBPs) evolved to scavenge materials from the environment by binding substrate and delivering it to the host bacteria. PBPs are common scaffolds for biosensors: effector proteins, such as circularly permuted green fluorescent protein (cpGFP), can be inserted into a PBP sequence such that the effector protein's output is changed by conformational changes upon PBP substate binding. The site of insertion for maximum output change is often determined by comparison of PBP apo/holo crystal structures, but random insertion libraries have shown that this method can miss the best sites. Here, we present a computational method for identifying insertion sites for cpGFP in the maltose binding PBP that creates functional biosensors. Our method, which uses protein contact analysis, identifies the best-known insertion regions, as well as additional regions that have not been identified previously. We experimentally characterised biosensors with insertions at these regions and found that 75% of designs were functional. This method is generalisable beyond this system and PBPs more generally, and so could be applied to other binding proteins or effector domains.\nScreening and characterization of aging regulators using synthesized yeast chromosome XIII\nAuthors: Huang, W.\nScore: 1.2, Published: 2023-11-08 DOI: 10.1101/2023.11.07.566118\nYeast, a single eukaryotic cell model organism, demonstrates a progressive aging process. In the era of synthetic biology, study of the impact of synthetic chromosomes and aging is urgent and intriguing. Herein, we successfully constructed the 884 Kb synXIII of Saccharomyces cerevisiae and conducted replicative aging studies using the synthetic strains. We verified that the rRNA-related transcriptional factor RRN9 is a major positive controller of replicative lifespan. Using SCRaMbLE and an HSP104 reporter as a biomarker for mutant discovery, we screened 135 SCRaMbLEd synXIII strains with extended lifespan and identified 10 genes on synXIII that potentially serve as aging regulators. In addition, the genome-scale regression analysis of long-replicative lifespan SCRaMbLEd strains revealed distinct dysregulation of nucleus, ribosome, and mitochondrion function networks. Our findings suggest that Sc2.0 yeast has potential for unveiling new aging-related genes and gene-gene interactions underlying replicative lifespan.\nProtein interaction kinetics delimit the performance of phosphorylation-driven protein switches.\nAuthors: Winter, D. L.; Wairara, A. R.; Bennett, J. L.; Donald, W. A.; Glover, D. J.\nScore: 1.2, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565761\nPost-translational modifications (PTMs) such as phosphorylation and dephosphorylation can rapidly alter protein surface chemistry and structural conformation which can, in turn, switch protein-protein interactions (PPIs) within signaling networks. Recently, de novo designed phosphorylation-responsive protein switches have been created that harness kinase- and phosphatase-mediated phosphorylation that modulate PPIs. PTM-driven protein switches could be useful for investigating PTM dynamics in living cells, developing biocompatible nanodevices, and engineering signaling pathways to program cell behavior. However, little is known about the physical and kinetic constraints of PTM-driven protein switches, which limits their practical application. In this study, we present a theoretical framework to evaluate two-component PTM-driven protein switches based on four performance metrics: effective concentration, dynamic range, response time, and reversibility. Our computational models reveal an intricate relationship between the binding kinetics, phosphorylation kinetics, and switch concentration that governs the sensitivity and reversibility of PTM-driven protein switches. Building upon the insights of our theoretical investigation, we built and evaluated two novel phosphorylation-driven protein switches consisting of phosphorylation-sensitive coiled coils as sensor domains fused to fluorescent proteins as actuator domains. By modulating the phosphorylation state of the switches with a specific protein kinase and phosphatase, we demonstrate fast, reversible transitions between easily differentiated \"on\" and \"off\" states. The response of the switches linearly correlated to the concentration of the kinase, demonstrating its potential as a biosensor for kinase measurements in real time. As intended, both switches responded to specific kinase activity with an increase in fluorescence signal and our model could be used to distinguish between two mechanisms of switch activation: dimerization or a structural rearrangement. In summary, the protein switch kinetics model presented here should be useful to guide the design of PTM-driven switches and tune their performance towards concrete applications.\nDegron-controlled protein degradation in Escherichia coli: New Approaches and Parameters\nAuthors: Cronan, G. E.; Kuzminov, A.\nScore: 1.0, Published: 2023-11-09 DOI: 10.1101/2023.11.08.566101\nProtein degron tags have proven uniquely useful for characterization of gene function. Degrons mediate quick depletion, usually within minutes, of a protein of interest - allowing researchers to characterize cellular responses to the loss of function. To develop a general purpose degron tool in E. coli, we sought to build upon a previously characterized system of SspB-dependent inducible protein degradation. For this, we created a family of expression vectors containing a destabilized allele of SspB, capable of a rapid and nearly perfect \"off-to-on\" induction response. Using this system, we demonstrated control over several enzymes of DNA metabolism, but also found with other substates apparent limitations of a SspB-dependent system. Several degron target proteins were degraded too slowly to affect their complete depletion during active growth, whereas others appeared completely refractory to degron-promoted degradation. We demonstrated that a model substrate, beta-galactosidase, was positively recognized as a degron substrate, but failed to be degraded by the ClpXP protease - demonstrating an apparently unknown mechanism of protease resistance. Thus, only a minority of our, admittedly biased, selection of degron substates proved amenable to rapid SspB-catalyzed degradation. We conclude that substrate-dependence of the SspB system remains a critical factor for the success of this degron system. For substrates that prove degradable, we provide a series of titratable SspB-expression vehicles.\n",
  "wordCount" : "2160",
  "inLanguage": "en",
  "datePublished": "2023-11-12T10:37:43Z",
  "dateModified": "2023-11-12T10:37:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 12, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566287">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566287" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566287">
        <p class="paperTitle">Neural network extrapolation to distant regions of the protein fitness landscape</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566287" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566287" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fahlberg, S. A.; Freschlin, C. R.; Heinzelman, P.; Romero, P. A.</p>
        <p class="info">Score: 19.4, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566287' target='https://doi.org/10.1101/2023.11.08.566287'> 10.1101/2023.11.08.566287</a></p>
        <p class="abstract">Machine learning (ML) has transformed protein engineering by constructing models of the underlying sequence-function landscape to accelerate the discovery of new biomolecules. ML-guided protein design requires models, trained on local sequence-function information, to accurately predict distant fitness peaks. In this work, we evaluate neural networks&#39; capacity to extrapolate beyond their training data. We perform model-guided design using a panel of neural network architectures trained on protein G (GB1)-Immunoglobulin G (IgG) binding data and experimentally test thousands of GB1 designs to systematically evaluate the models&#39; extrapolation. We find each model architecture infers markedly different landscapes from the same data, which give rise to unique design preferences. We find simpler models excel in local extrapolation to design high fitness proteins, while more sophisticated convolutional models can venture deep into sequence space to design proteins that fold but are no longer functional. Our findings highlight how each architecture&#39;s inductive biases prime them to learn different aspects of the protein fitness landscape.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566451">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566451" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566451">
        <p class="paperTitle">Minimization of the E. coli ribosome, aided and optimized by community science</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566451" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566451" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tangpradabkul, T.; Palo, M. Z.; Townley, J.; Hsu, K. B.; Eterna Participants,  ; Smaga, S.; Das, R.; Schepartz, A.</p>
        <p class="info">Score: 15.2, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566451' target='https://doi.org/10.1101/2023.11.09.566451'> 10.1101/2023.11.09.566451</a></p>
        <p class="abstract">The ribosome is a ribonucleoprotein complex found in all domains of life. Its role is to catalyze protein synthesis, the messenger RNA (mRNA)-templated formation of amide bonds between -amino acid monomers. Amide bond formation occurs within a highly conserved region of the large ribosomal subunit known as the peptidyl transferase center (PTC). Here we describe the stepwise design and characterization of mini-PTC 1.1, a 284-nucleotide RNA that recapitulates many essential features of the Escherichia coli PTC. Mini-PTC 1.1 folds into a PTC-like structure under physiological conditions, even in the absence of r-proteins, and engages small molecule analogs of A- and P-site tRNAs. The sequence of mini-PTC 1.1 differs from the wild type E. coli ribosome at 12 nucleotides that were installed by a cohort of citizen scientists using the on-line video game Eterna. These base changes improve both the secondary structure and tertiary folding of mini-PTC 1.1 as well as its ability to bind small molecule substrate analogs. Here, the combined input from Eterna citizen-scientists and RNA structural analysis provides a robust workflow for the design of a minimal PTC that recapitulates many features of an intact ribosome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.565973">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.565973" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.565973">
        <p class="paperTitle">Incorporation of multiple beta 2-backbones into a protein invivo using an orthogonal aminoacyl-tRNA synthetase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.565973" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.565973" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hamlish, N.; Abramyan, A.; Schepartz, A.</p>
        <p class="info">Score: 14.8, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.565973' target='https://doi.org/10.1101/2023.11.07.565973'> 10.1101/2023.11.07.565973</a></p>
        <p class="abstract">Synthesis of sequence-defined biomaterials whose monomer backbones diverge from canonical -amino acids represents the next frontier in protein and biomaterial evolution with the potential to yield better biological therapeutics, bioremediation tools, and biodegradable plastic-like materials. One monomer family of particular interest for biomaterials are {beta}-hydroxy acids. Many natural products contain isolated {beta}-esters, and polymeric {beta}-esters are found in polyhydroxyalkanoate (PHA) polyesters under development as bioplastics and drug encapsulation/delivery systems. Here we report that {beta}2-hydroxy acids possessing both (R) and (S) absolute configuration are excellent substrates for pyrrolysyl-tRNA synthetase (PylRS) enzymes in vitro, and that (S)-{beta}2-hydroxy acids are substrates in cellulo. Using the MaPylRS/MatRNAPyl pair, in conjunction with wild-type E. coli ribosomes and EF-Tu, we report the cellular synthesis of model proteins containing two (S)-{beta}2-hydroxy acid residues at internal positions. Metadynamics simulations provide a rationale for the observed enantioselective preference of the ribosome for the (S)-{beta}2-hydroxy acid backbone and mechanistic insights that inform future ribosomal engineering efforts. As far as we know, this finding represents the first example of an orthogonal synthetase that accepts a {beta}-backbone substrate and the first example of a protein hetero-oligomer containing multiple expanded-backbone monomers produced in cellulo.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565805">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565805" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565805">
        <p class="paperTitle">A modular toolbox for the optogenetic deactivation of transcription</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565805" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565805" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Muench, P.; Fiumara, M.; Southern, N.; Coda, D.; Aschenbrenner, S.; Correia, B.; Graeff, J.; Niopek, D.; Mathony, J.</p>
        <p class="info">Score: 9.9, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565805' target='https://doi.org/10.1101/2023.11.06.565805'> 10.1101/2023.11.06.565805</a></p>
        <p class="abstract">Light-controlled transcriptional activation is a commonly used optogenetic strategy that allows researchers to regulate gene expression with high spatiotemporal precision. The vast majority of existing tools are, however, limited to light-triggered induction of gene expression. Here, we inverted this mode of action and created two complementary optogenetic systems capable of efficiently terminating transcriptional activation in response to blue light. First, we designed highly compact regulators, by photo-controlling VP16 transactivation peptide exposure. Then, applying a two-hybrid strategy, we engineered LOOMINA (light off-operated modular inductor of transcriptional activation), a versatile transcriptional control platform for mammalian cells that is highly adaptable and compatible with various effector proteins. Leveraging the flexibility of CRISPR systems, we integrated LOOMINA with Cas9 as a DNA-binding domain to control transcription from various endogenous promoters with exceptionally high dynamic ranges in multiple cell lines, including neuron-like cells. Both functionally and mechanistically, LOOMINA represents a valuable addition to the optogenetic repertoire for transcriptional regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565046">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565046" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565046">
        <p class="paperTitle">BatCRISPRi: Bacillus titratable CRISPRi for dynamic control in Bacillus subtilis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565046" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565046" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Florez, A. F.; Castillo-Hair, S.; Gutierrez-Lopez, L.; Eaton, D.; Paulsson, J.; Garner, E. C.</p>
        <p class="info">Score: 11.7, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565046' target='https://doi.org/10.1101/2023.11.01.565046'> 10.1101/2023.11.01.565046</a></p>
        <p class="abstract">The discovery of new genes regulating essential biological processes has become increasingly important, and CRISPRi has emerged as a powerful tool for achieving this goal. This method has been used in many model organisms to decrease the expression of specific genes and assess their impact on phenotype. Pooled CRISPRi libraries in bacteria have been particularly useful in discovering new regulators of growth, division, and other biological processes. However, these libraries rely on the induction of dCas9 via an inducible promoter, which can be problematic due to promoter leakiness. This is a widespread phenomenon of any inducible promoter that can result in the unwanted downregulation of genes and the emergence of genetic suppressors when essential genes are knocked down. To overcome this issue, we have developed a novel strategy that eliminates dCas9 leakiness and enables reversible knockdown control using the rapamycin-dependent degron system in Bacillus subtilis. This degron system causes rapid degradation of dCas9, resulting in an almost instant reset of the system. Our results demonstrate that it is possible to achieve zero CRISPRi activity in the uninduced state and full activity in the induced state. This improved CRISPRi system will enable researchers to investigate phenotypic changes more effectively while reducing the undesirable effects of leaky expression and noise in their phenotypic data. Moreover, a rapid degradation system could serve as a tool for dynamic perturbation before compensation mechanisms or stress responses kick in. Finally, this approach can be adapted to other organisms and other promoter-inducible systems, potentially opening up strategies for tighter control of gene expression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.566064">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.566064" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.566064">
        <p class="paperTitle">GADY algorithm: Towards an evolutionary protocol for bottom-up design of synthetic bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.566064" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.566064" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanchez Iniguez, U.; Lledo Villaescusa, S.; Lahoz-Beltra, R.</p>
        <p class="info">Score: 2.0, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.566064' target='https://doi.org/10.1101/2023.11.07.566064'> 10.1101/2023.11.07.566064</a></p>
        <p class="abstract">Synthetic biology applications are currently based on the programming of bacteria with tailor-made circuits designed ad hoc by applying a top-down strategy. We introduce a novel algorithm oriented to design synthetic bacteria according to a bottom-up approach, i.e. via an  evolutionary programming algorithm. The proposed algorithm has been referred to as GADY: an acronym for Get signal, Antidote, Die when a killer gene is expressed, emits Yellow fluorescence. We include in this technical note a script of the algorithm in Gro cellular programming language, programming a colony of synthetic bacteria and illustrating its usefulness in a case of bioremediation or elimination of a strain of pathogenic bacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566541">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566541" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566541">
        <p class="paperTitle">Computational Design of Periplasmic Binding Protein Biosensors Guided by Molecular Dynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566541" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566541" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: O&#39;Shea, J. M.; Richardson, A.; Doerner, P. W.; Wood, C. W.</p>
        <p class="info">Score: 1.5, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566541' target='https://doi.org/10.1101/2023.11.10.566541'> 10.1101/2023.11.10.566541</a></p>
        <p class="abstract">Periplasmic binding proteins (PBPs) evolved to scavenge materials from the environment by binding substrate and delivering it to the host bacteria. PBPs are common scaffolds for biosensors: effector proteins, such as circularly permuted green fluorescent protein (cpGFP), can be inserted into a PBP sequence such that the effector protein&#39;s output is changed by conformational changes upon PBP substate binding. The site of insertion for maximum output change is often determined by comparison of PBP apo/holo crystal structures, but random insertion libraries have shown that this method can miss the best sites. Here, we present a computational method for identifying insertion sites for cpGFP in the maltose binding PBP that creates functional biosensors. Our method, which uses protein contact analysis, identifies the best-known insertion regions, as well as additional regions that have not been identified previously. We experimentally characterised biosensors with insertions at these regions and found that 75% of designs were functional. This method is generalisable beyond this system and PBPs more generally, and so could be applied to other binding proteins or effector domains.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.566118">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.566118" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.566118">
        <p class="paperTitle">Screening and characterization of aging regulators using synthesized yeast chromosome XIII</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.566118" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.566118" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, W.</p>
        <p class="info">Score: 1.2, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.566118' target='https://doi.org/10.1101/2023.11.07.566118'> 10.1101/2023.11.07.566118</a></p>
        <p class="abstract">Yeast, a single eukaryotic cell model organism, demonstrates a progressive aging process. In the era of synthetic biology, study of the impact of synthetic chromosomes and aging is urgent and intriguing. Herein, we successfully constructed the 884 Kb synXIII of Saccharomyces cerevisiae and conducted replicative aging studies using the synthetic strains. We verified that the rRNA-related transcriptional factor RRN9 is a major positive controller of replicative lifespan. Using SCRaMbLE and an HSP104 reporter as a biomarker for mutant discovery, we screened 135 SCRaMbLEd synXIII strains with extended lifespan and identified 10 genes on synXIII that potentially serve as aging regulators. In addition, the genome-scale regression analysis of long-replicative lifespan SCRaMbLEd strains revealed distinct dysregulation of nucleus, ribosome, and mitochondrion function networks. Our findings suggest that Sc2.0 yeast has potential for unveiling new aging-related genes and gene-gene interactions underlying replicative lifespan.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565761">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565761" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565761">
        <p class="paperTitle">Protein interaction kinetics delimit the performance of phosphorylation-driven protein switches.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565761" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565761" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Winter, D. L.; Wairara, A. R.; Bennett, J. L.; Donald, W. A.; Glover, D. J.</p>
        <p class="info">Score: 1.2, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565761' target='https://doi.org/10.1101/2023.11.06.565761'> 10.1101/2023.11.06.565761</a></p>
        <p class="abstract">Post-translational modifications (PTMs) such as phosphorylation and dephosphorylation can rapidly alter protein surface chemistry and structural conformation which can, in turn, switch protein-protein interactions (PPIs) within signaling networks. Recently, de novo designed phosphorylation-responsive protein switches have been created that harness kinase- and phosphatase-mediated phosphorylation that modulate PPIs. PTM-driven protein switches could be useful for investigating PTM dynamics in living cells, developing biocompatible nanodevices, and engineering signaling pathways to program cell behavior. However, little is known about the physical and kinetic constraints of PTM-driven protein switches, which limits their practical application. In this study, we present a theoretical framework to evaluate two-component PTM-driven protein switches based on four performance metrics: effective concentration, dynamic range, response time, and reversibility. Our computational models reveal an intricate relationship between the binding kinetics, phosphorylation kinetics, and switch concentration that governs the sensitivity and reversibility of PTM-driven protein switches. Building upon the insights of our theoretical investigation, we built and evaluated two novel phosphorylation-driven protein switches consisting of phosphorylation-sensitive coiled coils as sensor domains fused to fluorescent proteins as actuator domains. By modulating the phosphorylation state of the switches with a specific protein kinase and phosphatase, we demonstrate fast, reversible transitions between easily differentiated &#34;on&#34; and &#34;off&#34; states. The response of the switches linearly correlated to the concentration of the kinase, demonstrating its potential as a biosensor for kinase measurements in real time. As intended, both switches responded to specific kinase activity with an increase in fluorescence signal and our model could be used to distinguish between two mechanisms of switch activation: dimerization or a structural rearrangement. In summary, the protein switch kinetics model presented here should be useful to guide the design of PTM-driven switches and tune their performance towards concrete applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566101">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566101" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566101">
        <p class="paperTitle">Degron-controlled protein degradation in Escherichia coli: New Approaches and Parameters</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566101" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566101" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cronan, G. E.; Kuzminov, A.</p>
        <p class="info">Score: 1.0, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566101' target='https://doi.org/10.1101/2023.11.08.566101'> 10.1101/2023.11.08.566101</a></p>
        <p class="abstract">Protein degron tags have proven uniquely useful for characterization of gene function. Degrons mediate quick depletion, usually within minutes, of a protein of interest - allowing researchers to characterize cellular responses to the loss of function. To develop a general purpose degron tool in E. coli, we sought to build upon a previously characterized system of SspB-dependent inducible protein degradation. For this, we created a family of expression vectors containing a destabilized allele of SspB, capable of a rapid and nearly perfect &#34;off-to-on&#34; induction response. Using this system, we demonstrated control over several enzymes of DNA metabolism, but also found with other substates apparent limitations of a SspB-dependent system. Several degron target proteins were degraded too slowly to affect their complete depletion during active growth, whereas others appeared completely refractory to degron-promoted degradation. We demonstrated that a model substrate, beta-galactosidase, was positively recognized as a degron substrate, but failed to be degraded by the ClpXP protease - demonstrating an apparently unknown mechanism of protease resistance. Thus, only a minority of our, admittedly biased, selection of degron substates proved amenable to rapid SspB-catalyzed degradation. We conclude that substrate-dependence of the SspB system remains a critical factor for the success of this degron system. For substrates that prove degradable, we provide a series of titratable SspB-expression vehicles.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
