<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein
Authors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Gong, J.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.
Score: 98.1, Published: 2023-07-14 DOI: 10.1101/2023.07.05.547496
Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein
Authors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Gong, J.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.
Score: 98.1, Published: 2023-07-14 DOI: 10.1101/2023.07.05.547496
Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-30T10:36:57+00:00" />
<meta property="article:modified_time" content="2023-07-30T10:36:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein
Authors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Gong, J.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.
Score: 98.1, Published: 2023-07-14 DOI: 10.1101/2023.07.05.547496
Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein\nAuthors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Gong, J.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.\nScore: 98.1, Published: 2023-07-14 DOI: 10.1101/2023.07.05.547496\nProtein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently.",
  "keywords": [
    
  ],
  "articleBody": " xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein\nAuthors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Gong, J.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.\nScore: 98.1, Published: 2023-07-14 DOI: 10.1101/2023.07.05.547496\nProtein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently. This paper proposes a unified protein language model, xTrimoPGLM, to address these two types of tasks simultaneously through an innovative pre-training framework. Our key technical contribution is an exploration of the compatibility and the potential for joint optimization of the two types of objectives, which has led to a strategy for training xTrimoPGLM at an unprecedented scale of 100 billion parameters and 1 trillion training tokens. Our extensive experiments reveal that xTrimoPGLM significantly outperforms other advanced baselines in diverse protein understanding tasks (13 out of 15 tasks across four categories) and generates novel protein sequences which are structurally similar to natural ones. Furthermore, using the same xTrimoPGLM framework, we train an antibody-specific model (xTrimoPGLM-Ab) using 1 billion parameters. This model set a new record in predicting antibody naturalness and structures, both essential to the field of antibody-based drug design, and demonstrated a significantly faster inference speed than AlphaFold2. These results highlight the substantial capability and versatility of xTrimoPGLM in understanding and generating protein sequences.\nMicrobial biosensor for sensing and treatment of intestinal inflammation\nAuthors: Zhu, D.; Galley, J. D.; Pizzini, J.; Musteata, E.; Tabor, J. J.; Britton, R. A.\nScore: 11.5, Published: 2023-07-22 DOI: 10.1101/2023.07.21.550106\nSubstantial synthetic biology efforts have been made to engineer biosensors to detect intestinal inflammation, however none target the most clinically accepted biomarker, calprotectin. To develop an in situ biosensor for calprotectin, we optimized a zinc uptake regulator (Zur) regulated promoter coupled with a memory circuit that can detect and record intestinal inflammation in vivo. The level of activation strongly correlates with calprotectin levels in the colon of two independent mouse models of colitis. Coupling of the biosensor with the production of the anti-inflammatory cytokine IL-10 allowed for the resolution of chemically induced colitis, demonstrating the ability of the biosensor to sense and respond to disease. This work highlights the utility of developing synthetic organisms for the diagnosis and treatment of intestinal disease using clinically validated biomarkers. One sentence summaryWe have optimized a microbial biosensor to detect and respond to the clinically relevant intestinal inflammation biomarker calprotectin.\nGas vesicle-blood interactions enhance ultrasound imaging contrast\nAuthors: Ling, B.; Ko, J. H.; Stordy, B.; Zhang, Y.; Didden, T. F.; Malounda, D.; Swift, M. B.; Chan, W. C. W.; Shapiro, M. G.\nScore: 9.5, Published: 2023-07-25 DOI: 10.1101/2023.07.24.550434\nGas vesicles (GVs) are genetically encoded, air-filled protein nanostructures of broad interest for biomedical research and clinical applications, acting as imaging and therapeutic agents for ultrasound, magnetic resonance, and optical techniques. However, the biomedical applications of GVs as a systemically injectable nanomaterial have been hindered by a lack of understanding of GVs interactions with blood components, which can significantly impact in vivo performance. Here, we investigate the dynamics of GVs in the bloodstream using a combination of ultrasound and optical imaging, surface functionalization, flow cytometry, and mass spectrometry. We find that erythrocytes and serum proteins bind to GVs and shape their acoustic response, circulation time, and immunogenicity. We show that by modifying the GV surface, we can alter these interactions and thereby modify GVs in vivo performance. These results provide critical insights for the development of GVs as agents for nanomedicine.\nElucidating the assembly of gas vesicles by systematic protein-protein interaction analysis\nAuthors: Iburg, M.; Anderson, A. P.; Wong, V. T.; Anton, E. D.; He, A.; Lu, G. J.\nScore: 6.8, Published: 2023-07-23 DOI: 10.1101/2023.07.23.550228\nGas vesicles (GVs) are gas-filled microbial organelles formed by unique 3-nm thick, amphipathic, force-bearing protein shells, which can withstand multiple atmospheric pressures and maintain a physically stable air bubble with megapascal surface tension. However, the molecular process to assemble this shell remains elusive: while 6-8 assembly factor proteins were identified as essential, none of them have a defined function. As one of the first steps to elucidate the assembly mechanism, we devise a high-throughput in vivo assay to determine the interactions of all 11 proteins in a GV operon. Complete or partial deletions of the operon establish the interdependence relationship of the interaction on the background GV proteins with additional information on assembly tolerance and cellular burden. Clusters of GV protein interactions are revealed, which establishes the plausible protein complexes important for the assembly process of these protein organelles. We anticipate our findings will set the stage for solving the molecular mechanism of GV assembly and designing GVs that efficiently assemble in heterologous hosts during biomedical applications.\nReductive amination cascades in cell-free and resting whole cell formats for valorization of lignin deconstruction products\nAuthors: Nain, P.; Dickey, R.; Somasundaram, V.; Sulzbach, M.; Kunjapur, A. M.\nScore: 5.6, Published: 2023-07-21 DOI: 10.1101/2023.07.21.550087\nThe selective introduction of amine groups within deconstruction products of lignin could provide an avenue for valorizing waste biomass while achieving a green synthesis of industrially relevant building blocks from sustainable sources. Here, we built and characterized enzyme cascades that create aldehydes and subsequently primary amines from diverse lignin-derived carboxylic acids using a carboxylic acid reductase (CAR) and an {omega}-transaminase (TA). We report a detailed characterization of two reaction formats - cell-free biocatalysis and resting whole-cell biocatalysis - using the conversion of vanillate to vanillyl amine as model chemistry. We showed that resting whole-cell reactions could be competitive with cell-free reactions (97% and 70% yields, respectively) while avoiding steps of cell lysis, purification, and cofactor supplementation. We also demonstrated the recyclability of resting whole cells for multiple batches of catalysis without significant loss in productivity. Finally, we used the knowledge gained from this study to produce several amines from carboxylic acid precursors using one-pot biocatalytic reactions. These results expand our knowledge of these industrially relevant enzyme families to new substrates and contexts for environmentally friendly and potentially low-cost synthesis of diverse aryl aldehydes and amines.\nVisible light-induced specific protein reaction delineates early stages of cell adhesion\nAuthors: Rahikainen, R.; Vester, S. K.; Turkki, P.; Janosko, C.; Deiters, A.; Hytonen, V. P.; Howarth, M.\nScore: 5.2, Published: 2023-07-22 DOI: 10.1101/2023.07.21.549850\nLight is well established for control of bond breakage, but not for control of specific bond formation in complex environments. We previously engineered diffusion-limited reactivity of SpyTag003 peptide with its protein partner SpyCatcher003 through spontaneous transamidation. This system enables precise and irreversible assembly of biological building blocks, with applications from biomaterials to vaccines. Here, we establish a system for rapid control of this amide bond formation with visible light. We have generated a caged SpyCatcher003, which allows light triggering of covalent bond formation to SpyTag003 in mammalian cells. Photocaging is achieved through site-specific incorporation of an unnatural coumarin-lysine at the reactive site of SpyCatcher003. We showed uniform specific reaction in cell lysate upon light activation. We then used the spatiotemporal precision of a 405 nm confocal laser for uncaging in seconds, probing the earliest events in mechanotransduction by talin, the key force sensor between the cytoskeleton and extracellular matrix. Reconstituting talin induced rapid biphasic extension of lamellipodia, revealing the kinetics of talin-regulated cell spreading and polarization. Thereafter we determined the hierarchy of recruitment of key components for cell adhesion. Precise control over site-specific protein reaction with visible light creates diverse opportunities for cell biology and nanoassembly.\nGenerative and predictive neural networks for the design of functional RNA molecules\nAuthors: Riley, A. T.; Robson, J. M.; Green, A. A.\nScore: 14.2, Published: 2023-07-14 DOI: 10.1101/2023.07.14.549043\nRNA is a remarkably versatile molecule that has been engineered for applications in therapeutics, diagnostics, and in vivo information-processing systems. However, the complex relationship between the sequence and structural properties of an RNA molecule and its ability to perform specific functions often necessitates extensive experimental screening of candidate sequences. Here we present a generalized neural network architecture that utilizes the sequence and structure of RNA molecules (SANDSTORM) to inform functional predictions. We demonstrate that this approach achieves state-of-the-art performance across several distinct RNA prediction tasks, while learning interpretable abstractions of RNA secondary structure. We paired these predictive models with generative adversarial RNA design networks (GARDN), allowing the generative modelling of novel mRNA 5 untranslated regions and toehold switch riboregulators exhibiting a predetermined fitness. This approach enabled the design of novel toehold switches with a 43-fold increase in experimentally characterized dynamic range compared to those designed using classic thermodynamic algorithms. SANDSTORM and GARDN thus represent powerful new predictive and generative tools for the development of diagnostic and therapeutic RNA molecules with improved function.\nMapping immunological and host receptor binding determinants of SARS-CoV spike protein utilizing the Qubevirus platform\nAuthors: Bopda Waffo, A.\nScore: 1.5, Published: 2023-07-28 DOI: 10.1101/2023.07.27.550841\nThe motifs involved in tropism and immunological interactions of SARS-CoV spike (S) protein were investigated utilizing the Qubevirus platform. We showed that separately, 14 overlapping peptide fragments representing the S protein (F1-14 of 100 residues each) could be inserted into the C-terminus of A1 on recombinant Qubevirus without affecting its viability. Additionally, recombinant phage expression resulted in the surface exposure of different engineered fragments in an accessible manner. The F6 from S425-525, was found to contain the binding determinant of the recombinant human angiotensin converting enzyme 2 (rhACE2), with the shortest active binding motif situated between residues S437-492. Upstream, another fragment, F7, containing an overlapping portion of F6 would not bind to rhACE2, confirming not just only that residues were linear but equally also the appropriate structural orientation of F6 upon the Qubevirus. The F6 (S441-460) and other inserts, including F7/F8 (S601-620) and F10 (S781-800), were demonstrated to contain important immunological determinants through recognition and binding of S protein specific (anti-S) antibodies. An engineered chimeric insert bearing the fusion of all three anti-S reactive epitopes, improved substantially the recognition and binding to their cognate antibodies. These results provide insights into humoral immune relevant epitopes and tropism characteristics of the S protein with implications for the development of subunit vaccines or other biologics against SARS-CoV.\nModular and integrative activity reporters enhance biochemical studies in the yeast ER.\nAuthors: Martinusen, S. G.; Slaton, E. W.; Nelson, S. E.; Pulgar, M. A.; Besu, J. T.; Simas, C. F.; Denard, C. A.\nScore: 5.5, Published: 2023-07-12 DOI: 10.1101/2023.07.12.548713\nThe yeast endoplasmic reticulum sequestration and screening (YESS) system is a generalizable platform that has become highly useful to investigate post-translational modification enzymes (PTM-enzymes). This system enables researchers to profile and engineer the activity and substrate specificity of PTM-enzymes and to discover inhibitor-resistant enzyme mutants. In this study, we expand the capabilities of YESS by transferring its functional components to integrative plasmids. The YESS integrative system yields uniform protein expression and protease activities in various configurations, allows one to integrate activity reporters at two independent loci and to split the system between integrative and centromeric plasmids. We characterize these integrative reporters with two viral proteases, Tobacco etch virus (TEVp) and 3-chymotrypsin like protease (3CLpro), in terms of coefficient of variance, signal-to-noise ratio and fold-activation. Overall, we provide a framework for chromosomal-based studies that is modular, enabling rigorous high-throughput assays of PTM-enzymes in yeast.\nHighly efficient cellular expression of circular mRNA enables prolonged protein expression\nAuthors: Unti, M. J.; Jaffrey, S. R.\nScore: 12.6, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548538\nA major problem with mRNA therapeutics is the limited duration of protein expression due to the short half-life of mRNA. New approaches for generating highly stable circular mRNA in vitro have allowed increased duration of protein expression. However, it remains difficult to genetically encode circular mRNAs in mammalian cells, which limits the use of circular mRNA in cell-derived therapeutics. Here we describe the adaptation of the Tornado (Twister-optimized RNA for durable overexpression) system to achieve in-cell synthesis of circular mRNAs. We identify the promoter and internal ribosomal entry site (IRES) that result in high levels of protein expression in cells. We then show that these circular mRNAs can be packaged into virus-like particles (VLPs) thus enabling prolonged protein expression. Overall, these data describe a platform for synthesis of circular mRNAs and how these circular mRNAs can markedly enhance the ability of VLPs to function as a mRNA delivery tool.\n",
  "wordCount" : "2046",
  "inLanguage": "en",
  "datePublished": "2023-07-30T10:36:57Z",
  "dateModified": "2023-07-30T10:36:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on July 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547496">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547496" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547496">
        <p class="paperTitle">xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547496" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547496" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Gong, J.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.</p>
        <p class="info">Score: 98.1, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547496' target='https://doi.org/10.1101/2023.07.05.547496'> 10.1101/2023.07.05.547496</a></p>
        <p class="abstract">Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently. This paper proposes a unified protein language model, xTrimoPGLM, to address these two types of tasks simultaneously through an innovative pre-training framework. Our key technical contribution is an exploration of the compatibility and the potential for joint optimization of the two types of objectives, which has led to a strategy for training xTrimoPGLM at an unprecedented scale of 100 billion parameters and 1 trillion training tokens. Our extensive experiments reveal that xTrimoPGLM significantly outperforms other advanced baselines in diverse protein understanding tasks (13 out of 15 tasks across four categories) and generates novel protein sequences which are structurally similar to natural ones. Furthermore, using the same xTrimoPGLM framework, we train an antibody-specific model (xTrimoPGLM-Ab) using 1 billion parameters. This model set a new record in predicting antibody naturalness and structures, both essential to the field of antibody-based drug design, and demonstrated a significantly faster inference speed than AlphaFold2. These results highlight the substantial capability and versatility of xTrimoPGLM in understanding and generating protein sequences.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.550106">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.550106" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.550106">
        <p class="paperTitle">Microbial biosensor for sensing and treatment of intestinal inflammation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.550106" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.550106" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhu, D.; Galley, J. D.; Pizzini, J.; Musteata, E.; Tabor, J. J.; Britton, R. A.</p>
        <p class="info">Score: 11.5, Published: 2023-07-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.550106' target='https://doi.org/10.1101/2023.07.21.550106'> 10.1101/2023.07.21.550106</a></p>
        <p class="abstract">Substantial synthetic biology efforts have been made to engineer biosensors to detect intestinal inflammation, however none target the most clinically accepted biomarker, calprotectin. To develop an in situ biosensor for calprotectin, we optimized a zinc uptake regulator (Zur) regulated promoter coupled with a memory circuit that can detect and record intestinal inflammation in vivo. The level of activation strongly correlates with calprotectin levels in the colon of two independent mouse models of colitis. Coupling of the biosensor with the production of the anti-inflammatory cytokine IL-10 allowed for the resolution of chemically induced colitis, demonstrating the ability of the biosensor to sense and respond to disease. This work highlights the utility of developing synthetic organisms for the diagnosis and treatment of intestinal disease using clinically validated biomarkers.

One sentence summaryWe have optimized a microbial biosensor to detect and respond to the clinically relevant intestinal inflammation biomarker calprotectin.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550434">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550434" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550434">
        <p class="paperTitle">Gas vesicle-blood interactions enhance ultrasound imaging contrast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550434" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550434" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ling, B.; Ko, J. H.; Stordy, B.; Zhang, Y.; Didden, T. F.; Malounda, D.; Swift, M. B.; Chan, W. C. W.; Shapiro, M. G.</p>
        <p class="info">Score: 9.5, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550434' target='https://doi.org/10.1101/2023.07.24.550434'> 10.1101/2023.07.24.550434</a></p>
        <p class="abstract">Gas vesicles (GVs) are genetically encoded, air-filled protein nanostructures of broad interest for biomedical research and clinical applications, acting as imaging and therapeutic agents for ultrasound, magnetic resonance, and optical techniques. However, the biomedical applications of GVs as a systemically injectable nanomaterial have been hindered by a lack of understanding of GVs interactions with blood components, which can significantly impact in vivo performance. Here, we investigate the dynamics of GVs in the bloodstream using a combination of ultrasound and optical imaging, surface functionalization, flow cytometry, and mass spectrometry. We find that erythrocytes and serum proteins bind to GVs and shape their acoustic response, circulation time, and immunogenicity. We show that by modifying the GV surface, we can alter these interactions and thereby modify GVs in vivo performance. These results provide critical insights for the development of GVs as agents for nanomedicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.23.550228">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.23.550228" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.23.550228">
        <p class="paperTitle">Elucidating the assembly of gas vesicles by systematic protein-protein interaction analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.23.550228" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.23.550228" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Iburg, M.; Anderson, A. P.; Wong, V. T.; Anton, E. D.; He, A.; Lu, G. J.</p>
        <p class="info">Score: 6.8, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.23.550228' target='https://doi.org/10.1101/2023.07.23.550228'> 10.1101/2023.07.23.550228</a></p>
        <p class="abstract">Gas vesicles (GVs) are gas-filled microbial organelles formed by unique 3-nm thick, amphipathic, force-bearing protein shells, which can withstand multiple atmospheric pressures and maintain a physically stable air bubble with megapascal surface tension. However, the molecular process to assemble this shell remains elusive: while 6-8 assembly factor proteins were identified as essential, none of them have a defined function. As one of the first steps to elucidate the assembly mechanism, we devise a high-throughput in vivo assay to determine the interactions of all 11 proteins in a GV operon. Complete or partial deletions of the operon establish the interdependence relationship of the interaction on the background GV proteins with additional information on assembly tolerance and cellular burden. Clusters of GV protein interactions are revealed, which establishes the plausible protein complexes important for the assembly process of these protein organelles. We anticipate our findings will set the stage for solving the molecular mechanism of GV assembly and designing GVs that efficiently assemble in heterologous hosts during biomedical applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.550087">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.550087" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.550087">
        <p class="paperTitle">Reductive amination cascades in cell-free and resting whole cell formats for valorization of lignin deconstruction products</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.550087" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.550087" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nain, P.; Dickey, R.; Somasundaram, V.; Sulzbach, M.; Kunjapur, A. M.</p>
        <p class="info">Score: 5.6, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.550087' target='https://doi.org/10.1101/2023.07.21.550087'> 10.1101/2023.07.21.550087</a></p>
        <p class="abstract">The selective introduction of amine groups within deconstruction products of lignin could provide an avenue for valorizing waste biomass while achieving a green synthesis of industrially relevant building blocks from sustainable sources. Here, we built and characterized enzyme cascades that create aldehydes and subsequently primary amines from diverse lignin-derived carboxylic acids using a carboxylic acid reductase (CAR) and an {omega}-transaminase (TA). We report a detailed characterization of two reaction formats - cell-free biocatalysis and resting whole-cell biocatalysis - using the conversion of vanillate to vanillyl amine as model chemistry. We showed that resting whole-cell reactions could be competitive with cell-free reactions (97% and 70% yields, respectively) while avoiding steps of cell lysis, purification, and cofactor supplementation. We also demonstrated the recyclability of resting whole cells for multiple batches of catalysis without significant loss in productivity. Finally, we used the knowledge gained from this study to produce several amines from carboxylic acid precursors using one-pot biocatalytic reactions. These results expand our knowledge of these industrially relevant enzyme families to new substrates and contexts for environmentally friendly and potentially low-cost synthesis of diverse aryl aldehydes and amines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.549850">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.549850" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.549850">
        <p class="paperTitle">Visible light-induced specific protein reaction delineates early stages of cell adhesion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.549850" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.549850" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rahikainen, R.; Vester, S. K.; Turkki, P.; Janosko, C.; Deiters, A.; Hytonen, V. P.; Howarth, M.</p>
        <p class="info">Score: 5.2, Published: 2023-07-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.549850' target='https://doi.org/10.1101/2023.07.21.549850'> 10.1101/2023.07.21.549850</a></p>
        <p class="abstract">Light is well established for control of bond breakage, but not for control of specific bond formation in complex environments. We previously engineered diffusion-limited reactivity of SpyTag003 peptide with its protein partner SpyCatcher003 through spontaneous transamidation. This system enables precise and irreversible assembly of biological building blocks, with applications from biomaterials to vaccines. Here, we establish a system for rapid control of this amide bond formation with visible light. We have generated a caged SpyCatcher003, which allows light triggering of covalent bond formation to SpyTag003 in mammalian cells. Photocaging is achieved through site-specific incorporation of an unnatural coumarin-lysine at the reactive site of SpyCatcher003. We showed uniform specific reaction in cell lysate upon light activation. We then used the spatiotemporal precision of a 405 nm confocal laser for uncaging in seconds, probing the earliest events in mechanotransduction by talin, the key force sensor between the cytoskeleton and extracellular matrix. Reconstituting talin induced rapid biphasic extension of lamellipodia, revealing the kinetics of talin-regulated cell spreading and polarization. Thereafter we determined the hierarchy of recruitment of key components for cell adhesion. Precise control over site-specific protein reaction with visible light creates diverse opportunities for cell biology and nanoassembly.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.549043">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.549043" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.549043">
        <p class="paperTitle">Generative and predictive neural networks for the design of functional RNA molecules</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.549043" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.549043" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Riley, A. T.; Robson, J. M.; Green, A. A.</p>
        <p class="info">Score: 14.2, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.549043' target='https://doi.org/10.1101/2023.07.14.549043'> 10.1101/2023.07.14.549043</a></p>
        <p class="abstract">RNA is a remarkably versatile molecule that has been engineered for applications in therapeutics, diagnostics, and in vivo information-processing systems. However, the complex relationship between the sequence and structural properties of an RNA molecule and its ability to perform specific functions often necessitates extensive experimental screening of candidate sequences. Here we present a generalized neural network architecture that utilizes the sequence and structure of RNA molecules (SANDSTORM) to inform functional predictions. We demonstrate that this approach achieves state-of-the-art performance across several distinct RNA prediction tasks, while learning interpretable abstractions of RNA secondary structure. We paired these predictive models with generative adversarial RNA design networks (GARDN), allowing the generative modelling of novel mRNA 5 untranslated regions and toehold switch riboregulators exhibiting a predetermined fitness. This approach enabled the design of novel toehold switches with a 43-fold increase in experimentally characterized dynamic range compared to those designed using classic thermodynamic algorithms. SANDSTORM and GARDN thus represent powerful new predictive and generative tools for the development of diagnostic and therapeutic RNA molecules with improved function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550841">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550841" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550841">
        <p class="paperTitle">Mapping immunological and host receptor binding determinants of SARS-CoV spike protein utilizing the Qubevirus platform</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550841" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550841" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bopda Waffo, A.</p>
        <p class="info">Score: 1.5, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550841' target='https://doi.org/10.1101/2023.07.27.550841'> 10.1101/2023.07.27.550841</a></p>
        <p class="abstract">The motifs involved in tropism and immunological interactions of SARS-CoV spike (S) protein were investigated utilizing the Qubevirus platform. We showed that separately, 14 overlapping peptide fragments representing the S protein (F1-14 of 100 residues each) could be inserted into the C-terminus of A1 on recombinant Qubevirus without affecting its viability. Additionally, recombinant phage expression resulted in the surface exposure of different engineered fragments in an accessible manner. The F6 from S425-525, was found to contain the binding determinant of the recombinant human angiotensin converting enzyme 2 (rhACE2), with the shortest active binding motif situated between residues S437-492. Upstream, another fragment, F7, containing an overlapping portion of F6 would not bind to rhACE2, confirming not just only that residues were linear but equally also the appropriate structural orientation of F6 upon the Qubevirus. The F6 (S441-460) and other inserts, including F7/F8 (S601-620) and F10 (S781-800), were demonstrated to contain important immunological determinants through recognition and binding of S protein specific (anti-S) antibodies. An engineered chimeric insert bearing the fusion of all three anti-S reactive epitopes, improved substantially the recognition and binding to their cognate antibodies. These results provide insights into humoral immune relevant epitopes and tropism characteristics of the S protein with implications for the development of subunit vaccines or other biologics against SARS-CoV.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548713">
        <p class="paperTitle">Modular and integrative activity reporters enhance biochemical studies in the yeast ER.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinusen, S. G.; Slaton, E. W.; Nelson, S. E.; Pulgar, M. A.; Besu, J. T.; Simas, C. F.; Denard, C. A.</p>
        <p class="info">Score: 5.5, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548713' target='https://doi.org/10.1101/2023.07.12.548713'> 10.1101/2023.07.12.548713</a></p>
        <p class="abstract">The yeast endoplasmic reticulum sequestration and screening (YESS) system is a generalizable platform that has become highly useful to investigate post-translational modification enzymes (PTM-enzymes). This system enables researchers to profile and engineer the activity and substrate specificity of PTM-enzymes and to discover inhibitor-resistant enzyme mutants. In this study, we expand the capabilities of YESS by transferring its functional components to integrative plasmids. The YESS integrative system yields uniform protein expression and protease activities in various configurations, allows one to integrate activity reporters at two independent loci and to split the system between integrative and centromeric plasmids. We characterize these integrative reporters with two viral proteases, Tobacco etch virus (TEVp) and 3-chymotrypsin like protease (3CLpro), in terms of coefficient of variance, signal-to-noise ratio and fold-activation. Overall, we provide a framework for chromosomal-based studies that is modular, enabling rigorous high-throughput assays of PTM-enzymes in yeast.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548538">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548538" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548538">
        <p class="paperTitle">Highly efficient cellular expression of circular mRNA enables prolonged protein expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548538" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548538" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Unti, M. J.; Jaffrey, S. R.</p>
        <p class="info">Score: 12.6, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548538' target='https://doi.org/10.1101/2023.07.11.548538'> 10.1101/2023.07.11.548538</a></p>
        <p class="abstract">A major problem with mRNA therapeutics is the limited duration of protein expression due to the short half-life of mRNA. New approaches for generating highly stable circular mRNA in vitro have allowed increased duration of protein expression. However, it remains difficult to genetically encode circular mRNAs in mammalian cells, which limits the use of circular mRNA in cell-derived therapeutics. Here we describe the adaptation of the Tornado (Twister-optimized RNA for durable overexpression) system to achieve in-cell synthesis of circular mRNAs. We identify the promoter and internal ribosomal entry site (IRES) that result in high levels of protein expression in cells. We then show that these circular mRNAs can be packaged into virus-like particles (VLPs) thus enabling prolonged protein expression. Overall, these data describe a platform for synthesis of circular mRNAs and how these circular mRNAs can markedly enhance the ability of VLPs to function as a mRNA delivery tool.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
