<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Gas vesicle-blood interactions enhance ultrasound imaging contrast
Authors: Ling, B.; Ko, J. H.; Stordy, B.; Zhang, Y.; Didden, T. F.; Malounda, D.; Swift, M. B.; Chan, W. C. W.; Shapiro, M. G.
Score: 10.4, Published: 2023-07-25 DOI: 10.1101/2023.07.24.550434
Gas vesicles (GVs) are genetically encoded, air-filled protein nanostructures of broad interest for biomedical research and clinical applications, acting as imaging and therapeutic agents for ultrasound, magnetic resonance, and optical techniques. However, the biomedical applications of GVs as a systemically injectable nanomaterial have been hindered by a lack of understanding of GVs interactions with blood components, which can significantly impact in vivo performance.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Gas vesicle-blood interactions enhance ultrasound imaging contrast
Authors: Ling, B.; Ko, J. H.; Stordy, B.; Zhang, Y.; Didden, T. F.; Malounda, D.; Swift, M. B.; Chan, W. C. W.; Shapiro, M. G.
Score: 10.4, Published: 2023-07-25 DOI: 10.1101/2023.07.24.550434
Gas vesicles (GVs) are genetically encoded, air-filled protein nanostructures of broad interest for biomedical research and clinical applications, acting as imaging and therapeutic agents for ultrasound, magnetic resonance, and optical techniques. However, the biomedical applications of GVs as a systemically injectable nanomaterial have been hindered by a lack of understanding of GVs interactions with blood components, which can significantly impact in vivo performance." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-02T10:38:09+00:00" />
<meta property="article:modified_time" content="2023-08-02T10:38:09+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Gas vesicle-blood interactions enhance ultrasound imaging contrast
Authors: Ling, B.; Ko, J. H.; Stordy, B.; Zhang, Y.; Didden, T. F.; Malounda, D.; Swift, M. B.; Chan, W. C. W.; Shapiro, M. G.
Score: 10.4, Published: 2023-07-25 DOI: 10.1101/2023.07.24.550434
Gas vesicles (GVs) are genetically encoded, air-filled protein nanostructures of broad interest for biomedical research and clinical applications, acting as imaging and therapeutic agents for ultrasound, magnetic resonance, and optical techniques. However, the biomedical applications of GVs as a systemically injectable nanomaterial have been hindered by a lack of understanding of GVs interactions with blood components, which can significantly impact in vivo performance."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Gas vesicle-blood interactions enhance ultrasound imaging contrast\nAuthors: Ling, B.; Ko, J. H.; Stordy, B.; Zhang, Y.; Didden, T. F.; Malounda, D.; Swift, M. B.; Chan, W. C. W.; Shapiro, M. G.\nScore: 10.4, Published: 2023-07-25 DOI: 10.1101/2023.07.24.550434\nGas vesicles (GVs) are genetically encoded, air-filled protein nanostructures of broad interest for biomedical research and clinical applications, acting as imaging and therapeutic agents for ultrasound, magnetic resonance, and optical techniques. However, the biomedical applications of GVs as a systemically injectable nanomaterial have been hindered by a lack of understanding of GVs interactions with blood components, which can significantly impact in vivo performance.",
  "keywords": [
    
  ],
  "articleBody": " Gas vesicle-blood interactions enhance ultrasound imaging contrast\nAuthors: Ling, B.; Ko, J. H.; Stordy, B.; Zhang, Y.; Didden, T. F.; Malounda, D.; Swift, M. B.; Chan, W. C. W.; Shapiro, M. G.\nScore: 10.4, Published: 2023-07-25 DOI: 10.1101/2023.07.24.550434\nGas vesicles (GVs) are genetically encoded, air-filled protein nanostructures of broad interest for biomedical research and clinical applications, acting as imaging and therapeutic agents for ultrasound, magnetic resonance, and optical techniques. However, the biomedical applications of GVs as a systemically injectable nanomaterial have been hindered by a lack of understanding of GVs interactions with blood components, which can significantly impact in vivo performance. Here, we investigate the dynamics of GVs in the bloodstream using a combination of ultrasound and optical imaging, surface functionalization, flow cytometry, and mass spectrometry. We find that erythrocytes and serum proteins bind to GVs and shape their acoustic response, circulation time, and immunogenicity. We show that by modifying the GV surface, we can alter these interactions and thereby modify GVs in vivo performance. These results provide critical insights for the development of GVs as agents for nanomedicine.\nChemogenetic manipulation of endogenous proteins in fission yeast using a self-localizing ligand-induced protein translocation (SLIPT) system\nAuthors: Nakamura, A.; Goto, Y.; Sugiyama, H.; Tsukiji, S.; Aoki, K.\nScore: 6.1, Published: 2023-07-29 DOI: 10.1101/2023.07.27.550911\nCells sense extracellular stimuli through membrane receptors and process the information through an intracellular signaling network. Protein translocation triggers intracellular signaling, and techniques such as chemically induced dimerization (CID) have been used to manipulate signaling pathways by altering the subcellular localization of signaling molecules. However, in the fission yeast Schizosaccharomyces pombe, the commonly used FKBP-FRB system has technical limitations, and therefore perturbation tools with low cytotoxicity and high temporal resolution are needed. We here applied our recently developed self-localizing ligand-induced protein translocation (SLIPT) system to S. pombe and successfully perturbed several cell cycle-related proteins. The SLIPT system utilizes self-localizing ligands to recruit binding partners to specific subcellular compartments, such as the plasma membrane or nucleus. We optimzed the self-localizing ligands to maintain long-term recruitment of target molecules to the plasma membrane. By knocking in genes encoding the binding partners for self-localizing ligands, we observed changes in the localization of several endogenous molecules and found perturbations in the cell cycle and associated phenotypes. This study demonstrates the effectiveness of the SLIPT system as a chemogenetic tool for rapid perturbation of endogenous molecules in S. pombe, providing a valuable approach for studying intracellular signaling and cell cycle regulation with improved temporal resolution.\nVisible light-induced specific protein reaction delineates early stages of cell adhesion\nAuthors: Rahikainen, R.; Vester, S. K.; Turkki, P.; Janosko, C.; Deiters, A.; Hytonen, V. P.; Howarth, M.\nScore: 5.2, Published: 2023-07-22 DOI: 10.1101/2023.07.21.549850\nLight is well established for control of bond breakage, but not for control of specific bond formation in complex environments. We previously engineered diffusion-limited reactivity of SpyTag003 peptide with its protein partner SpyCatcher003 through spontaneous transamidation. This system enables precise and irreversible assembly of biological building blocks, with applications from biomaterials to vaccines. Here, we establish a system for rapid control of this amide bond formation with visible light. We have generated a caged SpyCatcher003, which allows light triggering of covalent bond formation to SpyTag003 in mammalian cells. Photocaging is achieved through site-specific incorporation of an unnatural coumarin-lysine at the reactive site of SpyCatcher003. We showed uniform specific reaction in cell lysate upon light activation. We then used the spatiotemporal precision of a 405 nm confocal laser for uncaging in seconds, probing the earliest events in mechanotransduction by talin, the key force sensor between the cytoskeleton and extracellular matrix. Reconstituting talin induced rapid biphasic extension of lamellipodia, revealing the kinetics of talin-regulated cell spreading and polarization. Thereafter we determined the hierarchy of recruitment of key components for cell adhesion. Precise control over site-specific protein reaction with visible light creates diverse opportunities for cell biology and nanoassembly.\nChanges in coding and efficiency through modular modifications to a One Pot PURE system for in vitro transcription \u0026 translation\nAuthors: Ngo, P. H. T.; Ishida, S.; Busogi, B. B.; Ledesma, M.; Do, H.; Kar, S.; Ellington, A. H. T.\nScore: 3.1, Published: 2023-07-28 DOI: 10.1101/2023.07.28.550900\nThe incorporation of unnatural amino acids are attractive methods for improving or bringing new and novel functions in peptides and proteins. Cell-free protein synthesis using the Protein Synthesis Using Recombinant Elements (PURE) system is an attractive platform for efficient unnatural amino acid incorporation. In this work, we further adapted and modified the One Pot PURE for a robust and modular system of enzymatic single site-specific of unnatural amino acid. We demonstrated the flexibility of this system through the introduction of two orthogonal aminoacyl tRNA synthetases and the suppression of two distinctive stop codons.\nCharge-Patterned Disordered Peptides Tune Intracellular Phase Separation in Bacteria\nAuthors: Liao, J.; Yeong, V.; Obermeyer, A. C.\nScore: 2.0, Published: 2023-08-01 DOI: 10.1101/2023.08.01.551457\nSubcellular phase separated compartments, known as biomolecular condensates, play an important role in the spatiotemporal organization of cells. To understand the sequence-determinants of phase separation in bacteria, we engineered protein-based condensates in Escherichia coli by utilizing electrostatic interactions as the main driving force. Minimal cationic disordered peptides were used to supercharge negative, neutral, and positive globular model proteins, enabling their phase separation with anionic biomacromolecules in the cell. The phase behavior was governed by the interaction strength between the cationic proteins and anionic biopolymers in addition to the protein concentration. The interaction strength primarily depended on the overall net charge of the protein, but the distribution of charge between the globular and disordered domains also had an impact. Notably, the protein charge distribution between domains could tune mesoscale attributes such as the size, number, and subcellular localization of condensates within E. coli cells. The length and charge density of the disordered peptides had significant effects on protein expression levels, ultimately influencing the formation of condensates. Taken together, charge-patterned disordered peptides provide a platform for understanding the molecular grammar underlying phase separation in bacteria.\nTranscriptomics elucidates metabolic regulation and functional promoters in the basidiomycete red yeast Xanthophyllomyces dendrorhous CBS 6938\nAuthors: Tobin, E. E.; Collins, J. H.; Marsan, C. B.; Nadeau, G. T.; Mori, K.; Lipzen, A.; Mondo, S.; Grigoriev, I. V.; Young, E. M.\nScore: 1.8, Published: 2023-07-31 DOI: 10.1101/2023.07.31.551333\nGenomics has become the primary way to explore microbial diversity, because genetic tools are currently difficult to develop in non-model organisms. Here, we demonstrate that -omics can be leveraged to accelerate genetic tool development for the basidiomycete yeast Xanthophyllomyces dendrorhous CBS 6938, the sole biotechnologically relevant organism in the Tremellomycete family. First, we sequence the genome. Then, we perform transcriptomics under a variety of conditions, focusing on light and oxidative stress. This data not only reveals novel photobiology and metabolic regulation, it also allows derivation of constitutive and regulated gene expression parts. Our analysis of X. dendrorhous photobiology shows for the first time that a complex system of white-collar and cryptochrome homologs mediate response to ultraviolet light (UV). Our analysis of metabolic regulation shows that UV activates DNA repair, aromatic amino acid and carotenoid biosynthesis and represses central carbon metabolism and the fungal-like apoptotic pathway. Thus, X. dendrorhous shows a dynamic response toward biosynthetic pathways for light-absorbing compounds and survival and away from energy production. We then define a modular cloning system, including antibiotic selections, integration sites, and reporter genes, and use the transcriptomics to derive strong constitutive and regulated promoters. Notably, we discover a novel promoter from a hypothetical gene that has 9-fold activation upon UV exposure. Thus, -omics-to-parts workflows can simultaneously provide useful genomic data and advance genetic tools for non-model microbes, particularly those without a closely related model organism. This approach will be broadly useful in current efforts to engineer diverse microbes.\nFunctional and Structural Characterization of an IclR Family Transcription Factor for the Development of Dicarboxylic Acid Biosensors\nAuthors: Pham, C.; Nasr, M.; Skarina, T.; Di Leo, R.; Kwan, D. H.; Marting, V. J. J.; Stogios, P. J.; Mahadevan, R.; Savchenko, A.\nScore: 1.6, Published: 2023-07-27 DOI: 10.1101/2023.07.27.550818\nProkaryotic transcription factors (TFs) regulate gene expression in response to small molecules, thus representing promising candidates as versatile small molecule-detecting biosensors valuable for synthetic biology applications. The engineering of such biosensors requires thorough in vitro and in vivo characterization of TF ligand response as well as detailed molecular structure information. In this work we characterize the PcaR TF belonging to the IclR family. We present in vitro functional analysis of PcaRs ligand profile and construction of genetic circuits for the characterization of PcaR as an in vivo biosensor in the model eukaryote Saccharomyces cerevisiae. We report the crystal structures of PcaR in the apo state and in complex with one of its ligands, succinate, which suggests the mechanism of dicarboxylic acid recognition by this TF. This work provides key structural and functional insights enabling the engineering of PcaR for dicarboxylic acid biosensors. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=50 SRC=\"FIGDIR/small/550818v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (15K): org.highwire.dtl.DTLVardef@d4999aorg.highwire.dtl.DTLVardef@1b0812dorg.highwire.dtl.DTLVardef@f0d30corg.highwire.dtl.DTLVardef@afceda_HPS_FORMAT_FIGEXP M_FIG C_FIG HighlightsO_LIPcaR is an IclR family transcription regulator responsive to dicarboxylic acids C_LIO_LIPcaR was established as an in vivo biosensor in yeast C_LIO_LICrystal structure of PcaR in the apo form was solved C_LIO_LICrystal structure with PcaR in complex with succinate was solved C_LIO_LISequence alignments unveil ligand-binding positions in the IclR family C_LI\nMapping immunological and host receptor binding determinants of SARS-CoV spike protein utilizing the Qubevirus platform\nAuthors: Bopda Waffo, A.\nScore: 1.5, Published: 2023-07-28 DOI: 10.1101/2023.07.27.550841\nThe motifs involved in tropism and immunological interactions of SARS-CoV spike (S) protein were investigated utilizing the Qubevirus platform. We showed that separately, 14 overlapping peptide fragments representing the S protein (F1-14 of 100 residues each) could be inserted into the C-terminus of A1 on recombinant Qubevirus without affecting its viability. Additionally, recombinant phage expression resulted in the surface exposure of different engineered fragments in an accessible manner. The F6 from S425-525, was found to contain the binding determinant of the recombinant human angiotensin converting enzyme 2 (rhACE2), with the shortest active binding motif situated between residues S437-492. Upstream, another fragment, F7, containing an overlapping portion of F6 would not bind to rhACE2, confirming not just only that residues were linear but equally also the appropriate structural orientation of F6 upon the Qubevirus. The F6 (S441-460) and other inserts, including F7/F8 (S601-620) and F10 (S781-800), were demonstrated to contain important immunological determinants through recognition and binding of S protein specific (anti-S) antibodies. An engineered chimeric insert bearing the fusion of all three anti-S reactive epitopes, improved substantially the recognition and binding to their cognate antibodies. These results provide insights into humoral immune relevant epitopes and tropism characteristics of the S protein with implications for the development of subunit vaccines or other biologics against SARS-CoV. SignificanceMapping epitopes within the receptor binding domains of viruses which are essential for viral tropism is critical for developing antiviral agents and subunit vaccines. In this study we have engineered the surface of Qubevirus to display a peptide library derived from the SARS-CoV S protein. In biopanning with S protein antibodies, we have identified three peptide fragments (EP1, EP2 and EP3) which reacted selectively with antibodies specific to the S protein. We demonstrated that all recombinant phage displayed peptide fragments both individually and as chimera exposed important immunological epitopes to their cognate antibodies. A peptide fragment F6 situated at S425-525, was found containing the binding determinant of the recombinant human angiotensin converting enzyme 2 (rhACE2), with the shortest active binding motif situated between residues S437-492. The platform is rapidly to identify epitopes and receptor binding sites within viral receptors found in target host cell. Thus, this platform holds great significance.\nModular and integrative activity reporters enhance biochemical studies in the yeast ER.\nAuthors: Martinusen, S. G.; Slaton, E. W.; Nelson, S. E.; Pulgar, M. A.; Besu, J. T.; Simas, C. F.; Denard, C. A.\nScore: 5.5, Published: 2023-07-12 DOI: 10.1101/2023.07.12.548713\nThe yeast endoplasmic reticulum sequestration and screening (YESS) system is a generalizable platform that has become highly useful to investigate post-translational modification enzymes (PTM-enzymes). This system enables researchers to profile and engineer the activity and substrate specificity of PTM-enzymes and to discover inhibitor-resistant enzyme mutants. In this study, we expand the capabilities of YESS by transferring its functional components to integrative plasmids. The YESS integrative system yields uniform protein expression and protease activities in various configurations, allows one to integrate activity reporters at two independent loci and to split the system between integrative and centromeric plasmids. We characterize these integrative reporters with two viral proteases, Tobacco etch virus (TEVp) and 3-chymotrypsin like protease (3CLpro), in terms of coefficient of variance, signal-to-noise ratio and fold-activation. Overall, we provide a framework for chromosomal-based studies that is modular, enabling rigorous high-throughput assays of PTM-enzymes in yeast.\nHighly efficient cellular expression of circular mRNA enables prolonged protein expression\nAuthors: Unti, M. J.; Jaffrey, S. R.\nScore: 12.8, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548538\nA major problem with mRNA therapeutics is the limited duration of protein expression due to the short half-life of mRNA. New approaches for generating highly stable circular mRNA in vitro have allowed increased duration of protein expression. However, it remains difficult to genetically encode circular mRNAs in mammalian cells, which limits the use of circular mRNA in cell-derived therapeutics. Here we describe the adaptation of the Tornado (Twister-optimized RNA for durable overexpression) system to achieve in-cell synthesis of circular mRNAs. We identify the promoter and internal ribosomal entry site (IRES) that result in high levels of protein expression in cells. We then show that these circular mRNAs can be packaged into virus-like particles (VLPs) thus enabling prolonged protein expression. Overall, these data describe a platform for synthesis of circular mRNAs and how these circular mRNAs can markedly enhance the ability of VLPs to function as a mRNA delivery tool.\n",
  "wordCount" : "2280",
  "inLanguage": "en",
  "datePublished": "2023-08-02T10:38:09Z",
  "dateModified": "2023-08-02T10:38:09Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on August 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550434">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550434" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550434">
        <p class="paperTitle">Gas vesicle-blood interactions enhance ultrasound imaging contrast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550434" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550434" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ling, B.; Ko, J. H.; Stordy, B.; Zhang, Y.; Didden, T. F.; Malounda, D.; Swift, M. B.; Chan, W. C. W.; Shapiro, M. G.</p>
        <p class="info">Score: 10.4, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550434' target='https://doi.org/10.1101/2023.07.24.550434'> 10.1101/2023.07.24.550434</a></p>
        <p class="abstract">Gas vesicles (GVs) are genetically encoded, air-filled protein nanostructures of broad interest for biomedical research and clinical applications, acting as imaging and therapeutic agents for ultrasound, magnetic resonance, and optical techniques. However, the biomedical applications of GVs as a systemically injectable nanomaterial have been hindered by a lack of understanding of GVs interactions with blood components, which can significantly impact in vivo performance. Here, we investigate the dynamics of GVs in the bloodstream using a combination of ultrasound and optical imaging, surface functionalization, flow cytometry, and mass spectrometry. We find that erythrocytes and serum proteins bind to GVs and shape their acoustic response, circulation time, and immunogenicity. We show that by modifying the GV surface, we can alter these interactions and thereby modify GVs in vivo performance. These results provide critical insights for the development of GVs as agents for nanomedicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550911">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550911" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550911">
        <p class="paperTitle">Chemogenetic manipulation of endogenous proteins in fission yeast using a self-localizing ligand-induced protein translocation (SLIPT) system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550911" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550911" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nakamura, A.; Goto, Y.; Sugiyama, H.; Tsukiji, S.; Aoki, K.</p>
        <p class="info">Score: 6.1, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550911' target='https://doi.org/10.1101/2023.07.27.550911'> 10.1101/2023.07.27.550911</a></p>
        <p class="abstract">Cells sense extracellular stimuli through membrane receptors and process the information through an intracellular signaling network. Protein translocation triggers intracellular signaling, and techniques such as chemically induced dimerization (CID) have been used to manipulate signaling pathways by altering the subcellular localization of signaling molecules. However, in the fission yeast Schizosaccharomyces pombe, the commonly used FKBP-FRB system has technical limitations, and therefore perturbation tools with low cytotoxicity and high temporal resolution are needed. We here applied our recently developed self-localizing ligand-induced protein translocation (SLIPT) system to S. pombe and successfully perturbed several cell cycle-related proteins. The SLIPT system utilizes self-localizing ligands to recruit binding partners to specific subcellular compartments, such as the plasma membrane or nucleus. We optimzed the self-localizing ligands to maintain long-term recruitment of target molecules to the plasma membrane. By knocking in genes encoding the binding partners for self-localizing ligands, we observed changes in the localization of several endogenous molecules and found perturbations in the cell cycle and associated phenotypes. This study demonstrates the effectiveness of the SLIPT system as a chemogenetic tool for rapid perturbation of endogenous molecules in S. pombe, providing a valuable approach for studying intracellular signaling and cell cycle regulation with improved temporal resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.549850">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.549850" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.549850">
        <p class="paperTitle">Visible light-induced specific protein reaction delineates early stages of cell adhesion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.549850" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.549850" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rahikainen, R.; Vester, S. K.; Turkki, P.; Janosko, C.; Deiters, A.; Hytonen, V. P.; Howarth, M.</p>
        <p class="info">Score: 5.2, Published: 2023-07-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.549850' target='https://doi.org/10.1101/2023.07.21.549850'> 10.1101/2023.07.21.549850</a></p>
        <p class="abstract">Light is well established for control of bond breakage, but not for control of specific bond formation in complex environments. We previously engineered diffusion-limited reactivity of SpyTag003 peptide with its protein partner SpyCatcher003 through spontaneous transamidation. This system enables precise and irreversible assembly of biological building blocks, with applications from biomaterials to vaccines. Here, we establish a system for rapid control of this amide bond formation with visible light. We have generated a caged SpyCatcher003, which allows light triggering of covalent bond formation to SpyTag003 in mammalian cells. Photocaging is achieved through site-specific incorporation of an unnatural coumarin-lysine at the reactive site of SpyCatcher003. We showed uniform specific reaction in cell lysate upon light activation. We then used the spatiotemporal precision of a 405 nm confocal laser for uncaging in seconds, probing the earliest events in mechanotransduction by talin, the key force sensor between the cytoskeleton and extracellular matrix. Reconstituting talin induced rapid biphasic extension of lamellipodia, revealing the kinetics of talin-regulated cell spreading and polarization. Thereafter we determined the hierarchy of recruitment of key components for cell adhesion. Precise control over site-specific protein reaction with visible light creates diverse opportunities for cell biology and nanoassembly.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550900">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550900" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550900">
        <p class="paperTitle">Changes in coding and efficiency through modular modifications to a One Pot PURE system for in vitro transcription &amp; translation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550900" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550900" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ngo, P. H. T.; Ishida, S.; Busogi, B. B.; Ledesma, M.; Do, H.; Kar, S.; Ellington, A. H. T.</p>
        <p class="info">Score: 3.1, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550900' target='https://doi.org/10.1101/2023.07.28.550900'> 10.1101/2023.07.28.550900</a></p>
        <p class="abstract">The incorporation of unnatural amino acids are attractive methods for improving or bringing new and novel functions in peptides and proteins. Cell-free protein synthesis using the Protein Synthesis Using Recombinant Elements (PURE) system is an attractive platform for efficient unnatural amino acid incorporation. In this work, we further adapted and modified the One Pot PURE for a robust and modular system of enzymatic single site-specific of unnatural amino acid. We demonstrated the flexibility of this system through the introduction of two orthogonal aminoacyl tRNA synthetases and the suppression of two distinctive stop codons.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551457">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551457" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551457">
        <p class="paperTitle">Charge-Patterned Disordered Peptides Tune Intracellular Phase Separation in Bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551457" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551457" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liao, J.; Yeong, V.; Obermeyer, A. C.</p>
        <p class="info">Score: 2.0, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551457' target='https://doi.org/10.1101/2023.08.01.551457'> 10.1101/2023.08.01.551457</a></p>
        <p class="abstract">Subcellular phase separated compartments, known as biomolecular condensates, play an important role in the spatiotemporal organization of cells. To understand the sequence-determinants of phase separation in bacteria, we engineered protein-based condensates in Escherichia coli by utilizing electrostatic interactions as the main driving force. Minimal cationic disordered peptides were used to supercharge negative, neutral, and positive globular model proteins, enabling their phase separation with anionic biomacromolecules in the cell. The phase behavior was governed by the interaction strength between the cationic proteins and anionic biopolymers in addition to the protein concentration. The interaction strength primarily depended on the overall net charge of the protein, but the distribution of charge between the globular and disordered domains also had an impact. Notably, the protein charge distribution between domains could tune mesoscale attributes such as the size, number, and subcellular localization of condensates within E. coli cells. The length and charge density of the disordered peptides had significant effects on protein expression levels, ultimately influencing the formation of condensates. Taken together, charge-patterned disordered peptides provide a platform for understanding the molecular grammar underlying phase separation in bacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551333">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551333" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551333">
        <p class="paperTitle">Transcriptomics elucidates metabolic regulation and functional promoters in the basidiomycete red yeast 	Xanthophyllomyces dendrorhous CBS 6938</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551333" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551333" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tobin, E. E.; Collins, J. H.; Marsan, C. B.; Nadeau, G. T.; Mori, K.; Lipzen, A.; Mondo, S.; Grigoriev, I. V.; Young, E. M.</p>
        <p class="info">Score: 1.8, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551333' target='https://doi.org/10.1101/2023.07.31.551333'> 10.1101/2023.07.31.551333</a></p>
        <p class="abstract">Genomics has become the primary way to explore microbial diversity, because genetic tools are currently difficult to develop in non-model organisms. Here, we demonstrate that -omics can be leveraged to accelerate genetic tool development for the basidiomycete yeast Xanthophyllomyces dendrorhous CBS 6938, the sole biotechnologically relevant organism in the Tremellomycete family. First, we sequence the genome. Then, we perform transcriptomics under a variety of conditions, focusing on light and oxidative stress. This data not only reveals novel photobiology and metabolic regulation, it also allows derivation of constitutive and regulated gene expression parts. Our analysis of X. dendrorhous photobiology shows for the first time that a complex system of white-collar and cryptochrome homologs mediate response to ultraviolet light (UV). Our analysis of metabolic regulation shows that UV activates DNA repair, aromatic amino acid and carotenoid biosynthesis and represses central carbon metabolism and the fungal-like apoptotic pathway. Thus, X. dendrorhous shows a dynamic response toward biosynthetic pathways for light-absorbing compounds and survival and away from energy production. We then define a modular cloning system, including antibiotic selections, integration sites, and reporter genes, and use the transcriptomics to derive strong constitutive and regulated promoters. Notably, we discover a novel promoter from a hypothetical gene that has 9-fold activation upon UV exposure. Thus, -omics-to-parts workflows can simultaneously provide useful genomic data and advance genetic tools for non-model microbes, particularly those without a closely related model organism. This approach will be broadly useful in current efforts to engineer diverse microbes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550818">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550818" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550818">
        <p class="paperTitle">Functional and Structural Characterization of an IclR Family Transcription Factor for the Development of Dicarboxylic Acid Biosensors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550818" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550818" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pham, C.; Nasr, M.; Skarina, T.; Di Leo, R.; Kwan, D. H.; Marting, V. J. J.; Stogios, P. J.; Mahadevan, R.; Savchenko, A.</p>
        <p class="info">Score: 1.6, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550818' target='https://doi.org/10.1101/2023.07.27.550818'> 10.1101/2023.07.27.550818</a></p>
        <p class="abstract">Prokaryotic transcription factors (TFs) regulate gene expression in response to small molecules, thus representing promising candidates as versatile small molecule-detecting biosensors valuable for synthetic biology applications. The engineering of such biosensors requires thorough in vitro and in vivo characterization of TF ligand response as well as detailed molecular structure information. In this work we characterize the PcaR TF belonging to the IclR family. We present in vitro functional analysis of PcaRs ligand profile and construction of genetic circuits for the characterization of PcaR as an in vivo biosensor in the model eukaryote Saccharomyces cerevisiae. We report the crystal structures of PcaR in the apo state and in complex with one of its ligands, succinate, which suggests the mechanism of dicarboxylic acid recognition by this TF. This work provides key structural and functional insights enabling the engineering of PcaR for dicarboxylic acid biosensors.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=50 SRC=&#34;FIGDIR/small/550818v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (15K):
org.highwire.dtl.DTLVardef@d4999aorg.highwire.dtl.DTLVardef@1b0812dorg.highwire.dtl.DTLVardef@f0d30corg.highwire.dtl.DTLVardef@afceda_HPS_FORMAT_FIGEXP  M_FIG C_FIG HighlightsO_LIPcaR is an IclR family transcription regulator responsive to dicarboxylic acids
C_LIO_LIPcaR was established as an in vivo biosensor in yeast
C_LIO_LICrystal structure of PcaR in the apo form was solved
C_LIO_LICrystal structure with PcaR in complex with succinate was solved
C_LIO_LISequence alignments unveil ligand-binding positions in the IclR family
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550841">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550841" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550841">
        <p class="paperTitle">Mapping immunological and host receptor binding determinants of SARS-CoV spike protein utilizing the Qubevirus platform</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550841" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550841" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bopda Waffo, A.</p>
        <p class="info">Score: 1.5, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550841' target='https://doi.org/10.1101/2023.07.27.550841'> 10.1101/2023.07.27.550841</a></p>
        <p class="abstract">The motifs involved in tropism and immunological interactions of SARS-CoV spike (S) protein were investigated utilizing the Qubevirus platform. We showed that separately, 14 overlapping peptide fragments representing the S protein (F1-14 of 100 residues each) could be inserted into the C-terminus of A1 on recombinant Qubevirus without affecting its viability. Additionally, recombinant phage expression resulted in the surface exposure of different engineered fragments in an accessible manner. The F6 from S425-525, was found to contain the binding determinant of the recombinant human angiotensin converting enzyme 2 (rhACE2), with the shortest active binding motif situated between residues S437-492. Upstream, another fragment, F7, containing an overlapping portion of F6 would not bind to rhACE2, confirming not just only that residues were linear but equally also the appropriate structural orientation of F6 upon the Qubevirus. The F6 (S441-460) and other inserts, including F7/F8 (S601-620) and F10 (S781-800), were demonstrated to contain important immunological determinants through recognition and binding of S protein specific (anti-S) antibodies. An engineered chimeric insert bearing the fusion of all three anti-S reactive epitopes, improved substantially the recognition and binding to their cognate antibodies. These results provide insights into humoral immune relevant epitopes and tropism characteristics of the S protein with implications for the development of subunit vaccines or other biologics against SARS-CoV.

SignificanceMapping epitopes within the receptor binding domains of viruses which are essential for viral tropism is critical for developing antiviral agents and subunit vaccines. In this study we have engineered the surface of Qubevirus to display a peptide library derived from the SARS-CoV S protein. In biopanning with S protein antibodies, we have identified three peptide fragments (EP1, EP2 and EP3) which reacted selectively with antibodies specific to the S protein. We demonstrated that all recombinant phage displayed peptide fragments both individually and as chimera exposed important immunological epitopes to their cognate antibodies. A peptide fragment F6 situated at S425-525, was found containing the binding determinant of the recombinant human angiotensin converting enzyme 2 (rhACE2), with the shortest active binding motif situated between residues S437-492. The platform is rapidly to identify epitopes and receptor binding sites within viral receptors found in target host cell. Thus, this platform holds great significance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548713">
        <p class="paperTitle">Modular and integrative activity reporters enhance biochemical studies in the yeast ER.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinusen, S. G.; Slaton, E. W.; Nelson, S. E.; Pulgar, M. A.; Besu, J. T.; Simas, C. F.; Denard, C. A.</p>
        <p class="info">Score: 5.5, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548713' target='https://doi.org/10.1101/2023.07.12.548713'> 10.1101/2023.07.12.548713</a></p>
        <p class="abstract">The yeast endoplasmic reticulum sequestration and screening (YESS) system is a generalizable platform that has become highly useful to investigate post-translational modification enzymes (PTM-enzymes). This system enables researchers to profile and engineer the activity and substrate specificity of PTM-enzymes and to discover inhibitor-resistant enzyme mutants. In this study, we expand the capabilities of YESS by transferring its functional components to integrative plasmids. The YESS integrative system yields uniform protein expression and protease activities in various configurations, allows one to integrate activity reporters at two independent loci and to split the system between integrative and centromeric plasmids. We characterize these integrative reporters with two viral proteases, Tobacco etch virus (TEVp) and 3-chymotrypsin like protease (3CLpro), in terms of coefficient of variance, signal-to-noise ratio and fold-activation. Overall, we provide a framework for chromosomal-based studies that is modular, enabling rigorous high-throughput assays of PTM-enzymes in yeast.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548538">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548538" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548538">
        <p class="paperTitle">Highly efficient cellular expression of circular mRNA enables prolonged protein expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548538" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548538" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Unti, M. J.; Jaffrey, S. R.</p>
        <p class="info">Score: 12.8, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548538' target='https://doi.org/10.1101/2023.07.11.548538'> 10.1101/2023.07.11.548538</a></p>
        <p class="abstract">A major problem with mRNA therapeutics is the limited duration of protein expression due to the short half-life of mRNA. New approaches for generating highly stable circular mRNA in vitro have allowed increased duration of protein expression. However, it remains difficult to genetically encode circular mRNAs in mammalian cells, which limits the use of circular mRNA in cell-derived therapeutics. Here we describe the adaptation of the Tornado (Twister-optimized RNA for durable overexpression) system to achieve in-cell synthesis of circular mRNAs. We identify the promoter and internal ribosomal entry site (IRES) that result in high levels of protein expression in cells. We then show that these circular mRNAs can be packaged into virus-like particles (VLPs) thus enabling prolonged protein expression. Overall, these data describe a platform for synthesis of circular mRNAs and how these circular mRNAs can markedly enhance the ability of VLPs to function as a mRNA delivery tool.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
