<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Sculpting conducting nanopore size and shape through de novo protein design
Authors: Berhanu, S.; Majumder, S.; Muntener, T.; Whitehouse, J.; Berner, C.; Bera, A. K.; Kang, A.; Liang, B.; Khan, G. N.; Sankaran, B.; Tamm, L.; Brockwel, D. J.; Hiller, S.; Radford, S.; Baker, D.; Vorobieva, A. A.
Score: 30.6, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572500
Transmembrane {beta}-barrels (TMBs) are widely used for single molecule DNA and RNA sequencing and have considerable potential for a broad range of sensing and sequencing applications.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Sculpting conducting nanopore size and shape through de novo protein design
Authors: Berhanu, S.; Majumder, S.; Muntener, T.; Whitehouse, J.; Berner, C.; Bera, A. K.; Kang, A.; Liang, B.; Khan, G. N.; Sankaran, B.; Tamm, L.; Brockwel, D. J.; Hiller, S.; Radford, S.; Baker, D.; Vorobieva, A. A.
Score: 30.6, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572500
Transmembrane {beta}-barrels (TMBs) are widely used for single molecule DNA and RNA sequencing and have considerable potential for a broad range of sensing and sequencing applications." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-27T10:39:30+00:00" />
<meta property="article:modified_time" content="2023-12-27T10:39:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Sculpting conducting nanopore size and shape through de novo protein design
Authors: Berhanu, S.; Majumder, S.; Muntener, T.; Whitehouse, J.; Berner, C.; Bera, A. K.; Kang, A.; Liang, B.; Khan, G. N.; Sankaran, B.; Tamm, L.; Brockwel, D. J.; Hiller, S.; Radford, S.; Baker, D.; Vorobieva, A. A.
Score: 30.6, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572500
Transmembrane {beta}-barrels (TMBs) are widely used for single molecule DNA and RNA sequencing and have considerable potential for a broad range of sensing and sequencing applications."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Sculpting conducting nanopore size and shape through de novo protein design\nAuthors: Berhanu, S.; Majumder, S.; Muntener, T.; Whitehouse, J.; Berner, C.; Bera, A. K.; Kang, A.; Liang, B.; Khan, G. N.; Sankaran, B.; Tamm, L.; Brockwel, D. J.; Hiller, S.; Radford, S.; Baker, D.; Vorobieva, A. A.\nScore: 30.6, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572500\nTransmembrane {beta}-barrels (TMBs) are widely used for single molecule DNA and RNA sequencing and have considerable potential for a broad range of sensing and sequencing applications.",
  "keywords": [
    
  ],
  "articleBody": " Sculpting conducting nanopore size and shape through de novo protein design\nAuthors: Berhanu, S.; Majumder, S.; Muntener, T.; Whitehouse, J.; Berner, C.; Bera, A. K.; Kang, A.; Liang, B.; Khan, G. N.; Sankaran, B.; Tamm, L.; Brockwel, D. J.; Hiller, S.; Radford, S.; Baker, D.; Vorobieva, A. A.\nScore: 30.6, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572500\nTransmembrane {beta}-barrels (TMBs) are widely used for single molecule DNA and RNA sequencing and have considerable potential for a broad range of sensing and sequencing applications. Current engineering approaches for nanopore sensors are limited to naturally occurring channels such as CsgG, which have evolved to carry out functions very different from sensing, and hence provide sub-optimal starting points. In contrast, de novo protein design can in principle create an unlimited number of new nanopores with any desired properties. Here we describe a general approach to the design of transmembrane {beta}-barrel pores with different diameter and pore geometry. NMR and crystallographic characterization shows that the designs are stably folded with structures close to the design models. We report the first examples of de novo designed TMBs with 10, 12 and 14 stranded {beta}-barrels. The designs have distinct conductances that correlate with their pore diameter, ranging from 110 pS ([~]0.5 nm pore diameter) to 430 pS ([~]1.1 nm pore diameter), and can be converted into sensitive small-molecule sensors with high signal to noise ratio. The capability to generate on demand {beta}-barrel pores of defined geometry opens up fundamentally new opportunities for custom engineering of sequencing and sensing technologies. One sentence summaryDe novo design enables the generation of stable and quite transmembrane beta-barrel nanopores with tailored sizes, shapes and properties.\nInverse folding of protein complexes with a structure-informed language model enables unsupervised antibody evolution\nAuthors: Shanker, V. R.; Bruun, T. U. J.; Hie, B. L.; Kim, P. S.\nScore: 17.5, Published: 2023-12-21 DOI: 10.1101/2023.12.19.572475\nLarge language models trained on sequence information alone are capable of learning high level principles of protein design. However, beyond sequence, the three-dimensional structures of proteins determine their specific function, activity, and evolvability. Here we show that a general protein language model augmented with protein structure backbone coordinates and trained on the inverse folding problem can guide evolution for diverse proteins without needing to explicitly model individual functional tasks. We demonstrate inverse folding to be an effective unsupervised, structure-based sequence optimization strategy that also generalizes to multimeric complexes by implicitly learning features of binding and amino acid epistasis. Using this approach, we screened [~]30 variants of two therapeutic clinical antibodies used to treat SARS- CoV-2 infection and achieved up to 26-fold improvement in neutralization and 37-fold improvement in affinity against antibody-escaped viral variants-of-concern BQ.1.1 and XBB.1.5, respectively. In addition to substantial overall improvements in protein function, we find inverse folding performs with leading experimental success rates among other reported machine learning-guided directed evolution methods, without requiring any task-specific training data.\nContinual improvement of multiplex mutagenesis in Arabidopsis\nAuthors: Develtere, W.; Decaestecker, W.; Rombaut, D.; Anders, C.; Clicque, E.; Vuylsteke, M.; Jacobs, T. B.\nScore: 16.8, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572307\nCRISPR/Cas9 is currently the most powerful tool to generate mutations in plant genomes and more efficient tools are needed as the scale of experiments increases. In the model plant Arabidopsis, the choice of promoter driving Cas9 expression is critical to generate germline mutations. Several optimal promoters have been reported. However, it is unclear which promoter is ideal as they have not been thoroughly tested side-by-side. Furthermore, most plant vectors still use one of the two Cas9 nuclear localization sequence (NLS) configurations initially reported and can still be optimized. We genotyped more than 6,000 Arabidopsis T2 plants to test seven promoters and eleven NLS architectures across 14 targets to systematically improve the generation of single and multiplex inheritable mutations. We find that the RPS5A promoter and double-BP NLS architecture were individually the most efficient components. When combined, 99% of T2 plant contained at least one knockout mutation and 84% contained 4-7-plex knock-outs. These optimizations will be useful to generate higher-order knockouts in the germline of Arabidopsis and likely be applicable to other CRISPR systems as well.\nIdentifying functional multi-host shuttle plasmids to advance synthetic biology applications in Mesorhizobium and Bradyrhizobium\nAuthors: Meaney, J. S.; Panchal, A. K.; Wilcox, A. J.; diCenzo, G. C.; Karas, B. J.\nScore: 5.5, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572671\nAmmonia availability has a crucial role in agriculture as it ensures healthy plant growth and increased crop yields. Since diazotrophs are the only organisms capable of reducing dinitrogen to ammonium, they have a great ecological importance and potential to mitigate the environmental and economic costs of synthetic fertilizer use. Rhizobia are especially valuable being that they can engage in nitrogen-fixing symbiotic relationships with legumes, and they demonstrate great diversity and plasticity in genomic and phenotypic traits. However, few rhizobial species have improved genetic tractability for synthetic biology applications. This study established a basic genetic toolbox with antibiotic resistance markers, multi-host shuttle plasmids and a streamlined protocol for biparental conjugation with Mesorhizobium and Bradyrhizobium species. We identified two repABC origins of replication from Sinorhizobium meliloti (pSymB) and Rhizobium etli (p42d) that were stable across all three strains of interest. Furthermore, the NZP2235 genome was sequenced and phylogenetic analysis determined its reclassification to Mesorhizobium huakuii. These tools will enable the use of plasmid-based strategies for more advanced genetic engineering projects and ultimately contribute towards the development of more sustainable agriculture practices by means of novel nitrogen-fixing organelles, elite bioinoculants or symbiotic association with non-legumes. GRAPHICAL ABSTRACT O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=94 SRC=\"FIGDIR/small/572671v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (43K): org.highwire.dtl.DTLVardef@1f8040org.highwire.dtl.DTLVardef@1d2969org.highwire.dtl.DTLVardef@89ea1aorg.highwire.dtl.DTLVardef@687316_HPS_FORMAT_FIGEXP M_FIG C_FIG\nBuilding synthetic biosensors using red blood cell proteins\nAuthors: Dolberg, T. B.; Gunnels, T. F.; Ling, T.; Sarnese, K. A.; Crispino, J. D.; Leonard, J. N.\nScore: 19.6, Published: 2023-12-16 DOI: 10.1101/2023.12.16.571988\nAs the use of engineered cell therapies expands from pioneering efforts in cancer immunotherapy to other applications, an attractive but less explored approach is the use of engineered red blood cells (RBCs). Compared to other cells, RBCs have a very long circulation time and reside in the blood compartment, so they could be ideally suited for applications as sentinel cells that enable in situ sensing and diagnostics. However, we largely lack tools for converting RBCs into biosensors. A unique challenge is that RBCs remodel their membranes during maturation, shedding many membrane components, suggesting that an RBC-specific approach may be needed. Towards addressing this need, here we develop a biosensing architecture built on RBC membrane proteins that are retained through erythropoiesis. This biosensor employs a mechanism in which extracellular ligand binding is transduced into intracellular reconstitution of a split output protein (including either a fluorophore or an enzyme). By comparatively evaluating a range of biosensor architectures, linker types, scaffold choices, and output signals, we identify biosensor designs and design features that confer substantial ligand-induced signal in vitro. Finally, we demonstrate that erythroid precursor cells engineered with our RBC protein biosensors function in vivo. This study establishes a foundation for developing RBC-based biosensors that could ultimately address unmet needs including non-invasive monitoring of physiological signals for a range of diagnostic applications.\nA translation-independent directed evolution strategy to engineer aminoacyl-tRNA synthetases\nAuthors: Soni, C.; Prywes, N.; Hall, M.; Savage, D. F.; Schepartz, A.; Chatterjee, A.\nScore: 17.9, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571473\nUsing directed evolution, engineered aminoacyl-tRNA synthetases (aaRS) have been developed that enable co-translational incorporation of numerous noncanonical amino acids (ncAAs) into proteins in living cells. Until now, the selection of such novel aaRS mutants has relied on coupling their activity to the expression of a reporter protein with a selectable phenotype. However, such translation-dependent selection schemes are incompatible with exotic monomers that diverge structurally from canonical -amino acids and are suboptimal substrates for the ribosome. To enable the ribosomal incorporation of such exotic monomers, a two-step solution is needed: A) Engineering an aaRS to acylate its cognate tRNA with the exotic monomer, without relying on ribosomal translation as a readout, and B) Subsequent engineering of the ribosome to accept the resulting acylated tRNA for translation. Here, we report a platform for aaRS engineering that directly selects for tRNA-acylation without ribosomal translation (START). In START, each distinct aaRS mutant is correlated to a cognate tRNA containing a unique sequence barcode. Acylation by an active aaRS mutant protects the associated barcode-containing tRNAs from an oxidative treatment designed to damage the 3'-terminus of the uncharged tRNAs. Sequencing of these surviving barcode-containing tRNAs is then used to reveal the identity of aaRS mutants that acylated the correlated tRNA sequences. The efficacy of START was demonstrated by identifying novel mutants of the M. alvus pyrrolysyl-tRNA synthetase from a naive library that charge noncanonical amino acids.\nVirus-like DNA tracers for characterizing wastewater-based pathogen monitoring systems\nAuthors: Gopal, A.; Machtinger, A. N.; Wang, B.; Rice, D. P.; McLaren, M. R.; Esvelt, K. M.\nScore: 1.8, Published: 2023-12-25 DOI: 10.1101/2023.12.24.573197\nWastewater-based pathogen monitoring has recently become a ubiquitous method for tracking the abundance and evolution of pathogens in a community or port of entry. But accurate inference of disease incidence from wastewater remains challenging, in large part due to noise and bias arising from wastewater transport and measurement. We developed a new system of tracer particles designed to enhance wastewater monitoring of viral pathogens. These tracers, which consist of synthetic DNA barcodes packaged into non-infectious viral capsids, can be deposited at known amounts and times into the sewer system and measured in wastewater samples to assess features such as assay sensitivity, and the decay and delay in viral signal arising from a shedding event. Here we describe the design, construction, and initial characterization of a 25-plex tracer library. Our tracer library supports simultaneous measurement of up to 5 distinct tracers by qPCR and up to 25 distinct tracers by amplicon or metagenomic sequencing. We demonstrate experimentally that the tracers are biologically inert and exhibit good stability and qPCR-assay performance in wastewater. These features collectively suggest that barcoded, virus-like tracers have the required properties for high-accuracy, high-resolution characterization of wastewater monitoring systems.\nGenetic modification strategies for electroporation and CRISPR-Cas-based technologies in the non-competent Gram-negative bacterium Acinetobacter sp. Tol 5\nAuthors: Ishikawa, M.; Hori, K.\nScore: 1.2, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572688\nEnvironmental isolates are promising candidates for new chassis of synthetic biology because of their inherent conversion capabilities and resilience to environmental stresses; however, many remain genetically intractable and unamenable to established genetic tools tailored for model bacteria. Acinetobacter sp. Tol 5 possesses intriguing properties for use in synthetic biology applications. However, genetic manipulation via electroporation is hindered by its low transformation efficiency. This study demonstrated the genetic refinement of the Tol 5 strain, achieving efficient transformation via electroporation. We deleted two genes encoding restriction enzymes. The resulting mutant strain not only exhibited marked efficiency of electrotransformation but also proved receptive to both in vitro and in vivo DNA assembly technologies, thereby facilitating the construction of recombinant DNA. In addition, we successfully adapted a CRISPR-Cas9-based base-editing platform developed for other Acinetobacter species. Our genetic modification strategy allows for the domestication of non-model bacteria, streamlining their utilization in synthetic biology applications.\nA cell-free system for target discovery of small membrane proteins\nAuthors: Jiang, S.; Celen, G.; Niederholtmeyer, H.; Yuan, J.\nScore: 0.8, Published: 2023-12-22 DOI: 10.1101/2023.12.22.573026\nNumerous small proteins have been discovered across all domains of life, among which many are hydrophobic and predicted to localize to the cell membrane. Based on a few that are well-studied, small membrane proteins are regulators involved in various biological processes, such as cell signaling, nutrient transport, drug resistance, and stress response. However, the function of most recently identified small membrane proteins remains elusive. Their small size and hydrophobicity make protein production challenging, hindering function discovery. Here, we combined a cell-free system with lipid sponge droplets and synthesized small membrane proteins in vitro. Lipid sponge droplets contain a dense network of lipid bilayers, which accommodates and extracts newly synthesized small membrane proteins from the aqueous surroundings. Using small bacterial membrane proteins MgrB and SafA as proof of principle, we showed that the in vitro produced membrane proteins reached sufficient concentrations for downstream analysis, and more importantly, they are functionally active. The cell-free system was also suitable for synthesizing other small membrane proteins, including one from human, indicating its high level of versatility. Furthermore, we showed that small proteins produced in this system can be used for in vitro pull-down assays to identify interaction partners. This work presents a robust alternative approach for producing small membrane proteins, which may aid their function discovery in all domains of life.\n",
  "wordCount" : "2075",
  "inLanguage": "en",
  "datePublished": "2023-12-27T10:39:30Z",
  "dateModified": "2023-12-27T10:39:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 27, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572500">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572500" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572500">
        <p class="paperTitle">Sculpting conducting nanopore size and shape through de novo protein design</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572500" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572500" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Berhanu, S.; Majumder, S.; Muntener, T.; Whitehouse, J.; Berner, C.; Bera, A. K.; Kang, A.; Liang, B.; Khan, G. N.; Sankaran, B.; Tamm, L.; Brockwel, D. J.; Hiller, S.; Radford, S.; Baker, D.; Vorobieva, A. A.</p>
        <p class="info">Score: 30.6, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572500' target='https://doi.org/10.1101/2023.12.20.572500'> 10.1101/2023.12.20.572500</a></p>
        <p class="abstract">Transmembrane {beta}-barrels (TMBs) are widely used for single molecule DNA and RNA sequencing and have considerable potential for a broad range of sensing and sequencing applications. Current engineering approaches for nanopore sensors are limited to naturally occurring channels such as CsgG, which have evolved to carry out functions very different from sensing, and hence provide sub-optimal starting points. In contrast, de novo protein design can in principle create an unlimited number of new nanopores with any desired properties. Here we describe a general approach to the design of transmembrane {beta}-barrel pores with different diameter and pore geometry. NMR and crystallographic characterization shows that the designs are stably folded with structures close to the design models. We report the first examples of de novo designed TMBs with 10, 12 and 14 stranded {beta}-barrels. The designs have distinct conductances that correlate with their pore diameter, ranging from 110 pS ([~]0.5 nm pore diameter) to 430 pS ([~]1.1 nm pore diameter), and can be converted into sensitive small-molecule sensors with high signal to noise ratio. The capability to generate on demand {beta}-barrel pores of defined geometry opens up fundamentally new opportunities for custom engineering of sequencing and sensing technologies.

One sentence summaryDe novo design enables the generation of stable and quite transmembrane beta-barrel nanopores with tailored sizes, shapes and properties.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572475">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572475" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572475">
        <p class="paperTitle">Inverse folding of protein complexes with a structure-informed language model enables unsupervised antibody evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572475" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572475" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shanker, V. R.; Bruun, T. U. J.; Hie, B. L.; Kim, P. S.</p>
        <p class="info">Score: 17.5, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572475' target='https://doi.org/10.1101/2023.12.19.572475'> 10.1101/2023.12.19.572475</a></p>
        <p class="abstract">Large language models trained on sequence information alone are capable of learning high level principles of protein design. However, beyond sequence, the three-dimensional structures of proteins determine their specific function, activity, and evolvability. Here we show that a general protein language model augmented with protein structure backbone coordinates and trained on the inverse folding problem can guide evolution for diverse proteins without needing to explicitly model individual functional tasks. We demonstrate inverse folding to be an effective unsupervised, structure-based sequence optimization strategy that also generalizes to multimeric complexes by implicitly learning features of binding and amino acid epistasis. Using this approach, we screened [~]30 variants of two therapeutic clinical antibodies used to treat SARS- CoV-2 infection and achieved up to 26-fold improvement in neutralization and 37-fold improvement in affinity against antibody-escaped viral variants-of-concern BQ.1.1 and XBB.1.5, respectively. In addition to substantial overall improvements in protein function, we find inverse folding performs with leading experimental success rates among other reported machine learning-guided directed evolution methods, without requiring any task-specific training data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572307">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572307" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572307">
        <p class="paperTitle">Continual improvement of multiplex mutagenesis in Arabidopsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572307" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572307" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Develtere, W.; Decaestecker, W.; Rombaut, D.; Anders, C.; Clicque, E.; Vuylsteke, M.; Jacobs, T. B.</p>
        <p class="info">Score: 16.8, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572307' target='https://doi.org/10.1101/2023.12.19.572307'> 10.1101/2023.12.19.572307</a></p>
        <p class="abstract">CRISPR/Cas9 is currently the most powerful tool to generate mutations in plant genomes and more efficient tools are needed as the scale of experiments increases. In the model plant Arabidopsis, the choice of promoter driving Cas9 expression is critical to generate germline mutations. Several optimal promoters have been reported. However, it is unclear which promoter is ideal as they have not been thoroughly tested side-by-side. Furthermore, most plant vectors still use one of the two Cas9 nuclear localization sequence (NLS) configurations initially reported and can still be optimized. We genotyped more than 6,000 Arabidopsis T2 plants to test seven promoters and eleven NLS architectures across 14 targets to systematically improve the generation of single and multiplex inheritable mutations. We find that the RPS5A promoter and double-BP NLS architecture were individually the most efficient components. When combined, 99% of T2 plant contained at least one knockout mutation and 84% contained 4-7-plex knock-outs. These optimizations will be useful to generate higher-order knockouts in the germline of Arabidopsis and likely be applicable to other CRISPR systems as well.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572671">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572671" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572671">
        <p class="paperTitle">Identifying functional multi-host shuttle plasmids to advance synthetic biology applications in Mesorhizobium and Bradyrhizobium</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572671" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572671" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Meaney, J. S.; Panchal, A. K.; Wilcox, A. J.; diCenzo, G. C.; Karas, B. J.</p>
        <p class="info">Score: 5.5, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572671' target='https://doi.org/10.1101/2023.12.20.572671'> 10.1101/2023.12.20.572671</a></p>
        <p class="abstract">Ammonia availability has a crucial role in agriculture as it ensures healthy plant growth and increased crop yields. Since diazotrophs are the only organisms capable of reducing dinitrogen to ammonium, they have a great ecological importance and potential to mitigate the environmental and economic costs of synthetic fertilizer use. Rhizobia are especially valuable being that they can engage in nitrogen-fixing symbiotic relationships with legumes, and they demonstrate great diversity and plasticity in genomic and phenotypic traits. However, few rhizobial species have improved genetic tractability for synthetic biology applications. This study established a basic genetic toolbox with antibiotic resistance markers, multi-host shuttle plasmids and a streamlined protocol for biparental conjugation with Mesorhizobium and Bradyrhizobium species. We identified two repABC origins of replication from Sinorhizobium meliloti (pSymB) and Rhizobium etli (p42d) that were stable across all three strains of interest. Furthermore, the NZP2235 genome was sequenced and phylogenetic analysis determined its reclassification to Mesorhizobium huakuii. These tools will enable the use of plasmid-based strategies for more advanced genetic engineering projects and ultimately contribute towards the development of more sustainable agriculture practices by means of novel nitrogen-fixing organelles, elite bioinoculants or symbiotic association with non-legumes.

GRAPHICAL ABSTRACT

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=94 SRC=&#34;FIGDIR/small/572671v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (43K):
org.highwire.dtl.DTLVardef@1f8040org.highwire.dtl.DTLVardef@1d2969org.highwire.dtl.DTLVardef@89ea1aorg.highwire.dtl.DTLVardef@687316_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.16.571988">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.16.571988" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.16.571988">
        <p class="paperTitle">Building synthetic biosensors using red blood cell proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.16.571988" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.16.571988" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dolberg, T. B.; Gunnels, T. F.; Ling, T.; Sarnese, K. A.; Crispino, J. D.; Leonard, J. N.</p>
        <p class="info">Score: 19.6, Published: 2023-12-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.16.571988' target='https://doi.org/10.1101/2023.12.16.571988'> 10.1101/2023.12.16.571988</a></p>
        <p class="abstract">As the use of engineered cell therapies expands from pioneering efforts in cancer immunotherapy to other applications, an attractive but less explored approach is the use of engineered red blood cells (RBCs). Compared to other cells, RBCs have a very long circulation time and reside in the blood compartment, so they could be ideally suited for applications as sentinel cells that enable in situ sensing and diagnostics. However, we largely lack tools for converting RBCs into biosensors. A unique challenge is that RBCs remodel their membranes during maturation, shedding many membrane components, suggesting that an RBC-specific approach may be needed. Towards addressing this need, here we develop a biosensing architecture built on RBC membrane proteins that are retained through erythropoiesis. This biosensor employs a mechanism in which extracellular ligand binding is transduced into intracellular reconstitution of a split output protein (including either a fluorophore or an enzyme). By comparatively evaluating a range of biosensor architectures, linker types, scaffold choices, and output signals, we identify biosensor designs and design features that confer substantial ligand-induced signal in vitro. Finally, we demonstrate that erythroid precursor cells engineered with our RBC protein biosensors function in vivo. This study establishes a foundation for developing RBC-based biosensors that could ultimately address unmet needs including non-invasive monitoring of physiological signals for a range of diagnostic applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571473">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571473" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571473">
        <p class="paperTitle">A translation-independent directed evolution strategy to engineer aminoacyl-tRNA synthetases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571473" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571473" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Soni, C.; Prywes, N.; Hall, M.; Savage, D. F.; Schepartz, A.; Chatterjee, A.</p>
        <p class="info">Score: 17.9, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571473' target='https://doi.org/10.1101/2023.12.13.571473'> 10.1101/2023.12.13.571473</a></p>
        <p class="abstract">Using directed evolution, engineered aminoacyl-tRNA synthetases (aaRS) have been developed that enable co-translational incorporation of numerous noncanonical amino acids (ncAAs) into proteins in living cells. Until now, the selection of such novel aaRS mutants has relied on coupling their activity to the expression of a reporter protein with a selectable phenotype. However, such translation-dependent selection schemes are incompatible with exotic monomers that diverge structurally from canonical -amino acids and are suboptimal substrates for the ribosome. To enable the ribosomal incorporation of such exotic monomers, a two-step solution is needed: A) Engineering an aaRS to acylate its cognate tRNA with the exotic monomer, without relying on ribosomal translation as a readout, and B) Subsequent engineering of the ribosome to accept the resulting acylated tRNA for translation. Here, we report a platform for aaRS engineering that directly selects for tRNA-acylation without ribosomal translation (START). In START, each distinct aaRS mutant is correlated to a cognate tRNA containing a unique sequence barcode. Acylation by an active aaRS mutant protects the associated barcode-containing tRNAs from an oxidative treatment designed to damage the 3&#39;-terminus of the uncharged tRNAs. Sequencing of these surviving barcode-containing tRNAs is then used to reveal the identity of aaRS mutants that acylated the correlated tRNA sequences. The efficacy of START was demonstrated by identifying novel mutants of the M. alvus pyrrolysyl-tRNA synthetase from a naive library that charge noncanonical amino acids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.24.573197">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.24.573197" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.24.573197">
        <p class="paperTitle">Virus-like DNA tracers for characterizing wastewater-based pathogen monitoring systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.24.573197" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.24.573197" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gopal, A.; Machtinger, A. N.; Wang, B.; Rice, D. P.; McLaren, M. R.; Esvelt, K. M.</p>
        <p class="info">Score: 1.8, Published: 2023-12-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.24.573197' target='https://doi.org/10.1101/2023.12.24.573197'> 10.1101/2023.12.24.573197</a></p>
        <p class="abstract">Wastewater-based pathogen monitoring has recently become a ubiquitous method for tracking the abundance and evolution of pathogens in a community or port of entry. But accurate inference of disease incidence from wastewater remains challenging, in large part due to noise and bias arising from wastewater transport and measurement. We developed a new system of tracer particles designed to enhance wastewater monitoring of viral pathogens. These tracers, which consist of synthetic DNA barcodes packaged into non-infectious viral capsids, can be deposited at known amounts and times into the sewer system and measured in wastewater samples to assess features such as assay sensitivity, and the decay and delay in viral signal arising from a shedding event. Here we describe the design, construction, and initial characterization of a 25-plex tracer library. Our tracer library supports simultaneous measurement of up to 5 distinct tracers by qPCR and up to 25 distinct tracers by amplicon or metagenomic sequencing. We demonstrate experimentally that the tracers are biologically inert and exhibit good stability and qPCR-assay performance in wastewater. These features collectively suggest that barcoded, virus-like tracers have the required properties for high-accuracy, high-resolution characterization of wastewater monitoring systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572688">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572688" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572688">
        <p class="paperTitle">Genetic modification strategies for electroporation and CRISPR-Cas-based technologies in the non-competent Gram-negative bacterium Acinetobacter sp. Tol 5</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572688" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572688" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ishikawa, M.; Hori, K.</p>
        <p class="info">Score: 1.2, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572688' target='https://doi.org/10.1101/2023.12.20.572688'> 10.1101/2023.12.20.572688</a></p>
        <p class="abstract">Environmental isolates are promising candidates for new chassis of synthetic biology because of their inherent conversion capabilities and resilience to environmental stresses; however, many remain genetically intractable and unamenable to established genetic tools tailored for model bacteria. Acinetobacter sp. Tol 5 possesses intriguing properties for use in synthetic biology applications. However, genetic manipulation via electroporation is hindered by its low transformation efficiency. This study demonstrated the genetic refinement of the Tol 5 strain, achieving efficient transformation via electroporation. We deleted two genes encoding restriction enzymes. The resulting mutant strain not only exhibited marked efficiency of electrotransformation but also proved receptive to both in vitro and in vivo DNA assembly technologies, thereby facilitating the construction of recombinant DNA. In addition, we successfully adapted a CRISPR-Cas9-based base-editing platform developed for other Acinetobacter species. Our genetic modification strategy allows for the domestication of non-model bacteria, streamlining their utilization in synthetic biology applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573026">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573026" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573026">
        <p class="paperTitle">A cell-free system for target discovery of small membrane proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573026" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573026" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, S.; Celen, G.; Niederholtmeyer, H.; Yuan, J.</p>
        <p class="info">Score: 0.8, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573026' target='https://doi.org/10.1101/2023.12.22.573026'> 10.1101/2023.12.22.573026</a></p>
        <p class="abstract">Numerous small proteins have been discovered across all domains of life, among which many are hydrophobic and predicted to localize to the cell membrane. Based on a few that are well-studied, small membrane proteins are regulators involved in various biological processes, such as cell signaling, nutrient transport, drug resistance, and stress response. However, the function of most recently identified small membrane proteins remains elusive. Their small size and hydrophobicity make protein production challenging, hindering function discovery. Here, we combined a cell-free system with lipid sponge droplets and synthesized small membrane proteins in vitro. Lipid sponge droplets contain a dense network of lipid bilayers, which accommodates and extracts newly synthesized small membrane proteins from the aqueous surroundings. Using small bacterial membrane proteins MgrB and SafA as proof of principle, we showed that the in vitro produced membrane proteins reached sufficient concentrations for downstream analysis, and more importantly, they are functionally active. The cell-free system was also suitable for synthesizing other small membrane proteins, including one from human, indicating its high level of versatility. Furthermore, we showed that small proteins produced in this system can be used for in vitro pull-down assays to identify interaction partners. This work presents a robust alternative approach for producing small membrane proteins, which may aid their function discovery in all domains of life.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
