<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Engineering orthogonal ribosomes for real-time monitoring using fluorescence
Authors: Csibra, E.; Klopprogge, B.; Hau Sorensen, G.; Gorochowski, T. E.
Score: 22.2, Published: 2023-11-19 DOI: 10.1101/2023.11.19.567736
A promising route to tackle the trade-off in cellular resources between synthetic protein production and cellular growth is to use a separate dedicated pool of orthogonal ribosomes to produce synthetic proteins. However, the optimisation of strains containing two ribosomal pools - native for the host cells proteome and orthogonal for synthetic proteins - has yet to be thoroughly explored.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Engineering orthogonal ribosomes for real-time monitoring using fluorescence
Authors: Csibra, E.; Klopprogge, B.; Hau Sorensen, G.; Gorochowski, T. E.
Score: 22.2, Published: 2023-11-19 DOI: 10.1101/2023.11.19.567736
A promising route to tackle the trade-off in cellular resources between synthetic protein production and cellular growth is to use a separate dedicated pool of orthogonal ribosomes to produce synthetic proteins. However, the optimisation of strains containing two ribosomal pools - native for the host cells proteome and orthogonal for synthetic proteins - has yet to be thoroughly explored." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-26T10:37:33+00:00" />
<meta property="article:modified_time" content="2023-11-26T10:37:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Engineering orthogonal ribosomes for real-time monitoring using fluorescence
Authors: Csibra, E.; Klopprogge, B.; Hau Sorensen, G.; Gorochowski, T. E.
Score: 22.2, Published: 2023-11-19 DOI: 10.1101/2023.11.19.567736
A promising route to tackle the trade-off in cellular resources between synthetic protein production and cellular growth is to use a separate dedicated pool of orthogonal ribosomes to produce synthetic proteins. However, the optimisation of strains containing two ribosomal pools - native for the host cells proteome and orthogonal for synthetic proteins - has yet to be thoroughly explored."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Engineering orthogonal ribosomes for real-time monitoring using fluorescence\nAuthors: Csibra, E.; Klopprogge, B.; Hau Sorensen, G.; Gorochowski, T. E.\nScore: 22.2, Published: 2023-11-19 DOI: 10.1101/2023.11.19.567736\nA promising route to tackle the trade-off in cellular resources between synthetic protein production and cellular growth is to use a separate dedicated pool of orthogonal ribosomes to produce synthetic proteins. However, the optimisation of strains containing two ribosomal pools - native for the host cells proteome and orthogonal for synthetic proteins - has yet to be thoroughly explored.",
  "keywords": [
    
  ],
  "articleBody": " Engineering orthogonal ribosomes for real-time monitoring using fluorescence\nAuthors: Csibra, E.; Klopprogge, B.; Hau Sorensen, G.; Gorochowski, T. E.\nScore: 22.2, Published: 2023-11-19 DOI: 10.1101/2023.11.19.567736\nA promising route to tackle the trade-off in cellular resources between synthetic protein production and cellular growth is to use a separate dedicated pool of orthogonal ribosomes to produce synthetic proteins. However, the optimisation of strains containing two ribosomal pools - native for the host cells proteome and orthogonal for synthetic proteins - has yet to be thoroughly explored. Here, we address this by creating orthogonal ribosomes that fluoresce by inserting fluorescent RNA aptamers into tethered orthogonal ribosomal RNA (TO-rRNA). To study the tolerance of the engineered ribosomes to aptamer insertion, we assembled and screened a library of candidate insertion sites, identifying several sites in both the 16S and 23S TO-rRNA that enables ribosome labelling with minimal effect on translation activity. Serendipitously, we identify one site in 23S TO-rRNA, where insertion appears to not only be tolerated but to enhance orthogonal ribosome activity, across multiple bacterial strains and RNA insertions. Using bulk and single cell assays, we demonstrate that this variant allows us to label orthogonal ribosomes for dynamic tracking and across populations, making it a promising tool for optimising orthogonal translation in engineered cells. Ribosome engineering offers great potential, both for the development of next-generation microbial cell factories, as well as a tool to expand our understanding of ribosome function in living cells. This work provides a window into the assembly, localisation and function of these molecular machines to meet these aims.\nHighly parallelized construction of DNA from low-cost oligonucleotide mixtures using Data-optimized Assembly Design and Golden Gate\nAuthors: Lund, S.; Potapov, V.; Johnson, S. R.; Buss, J.; Tanner, N. A.\nScore: 16.1, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567888\nCommercially synthesized genes are typically made using variations of homology-based cloning techniques, including polymerase cycling assembly from chemically synthesized microarray-derived oligonucleotides. Here we apply Data-optimized Assembly Design to the synthesis of hundreds of codon-optimized genes in both constitutive and inducible vectors using Golden Gate Assembly. Starting from oligonucleotide pools, we synthesize genes in three simple steps: 1) Amplification of parts belonging to individual assemblies in parallel from a single pool; 2) Golden Gate Assembly of parts for each construct; and 3) Transformation. We construct genes from receiving DNA to sequence confirmed isolates in as little as 4 days. By leveraging the ligation fidelity afforded by T4 DNA ligase, we expect to be able to construct a larger breadth of sequences not currently supported by homology-based methods which require stability of extensive single-stranded DNA overhangs.\nEvolution is All You Need in Promoter Design and Optimization\nAuthors: Ren, R.; Yu, H.; Teng, J.; Mao, S.; Bian, Z.; Tao, Y.; Yau, S. S.-T.\nScore: 8.4, Published: 2023-11-18 DOI: 10.1101/2023.11.18.567645\nPredicting the strength of promoters and guiding their directed evolution is a crucial task in synthetic biology. This approach significantly reduces the experimental costs in conventional promoter engineering. Previous studies employing machine learning or deep learning methods have shown some success in this task, but their outcomes were not satisfactory enough, primarily due to the neglect of evolutionary information. In this paper, we introduce the Chaos-Attention net for Promoter Evolution (CAPE) to address the limitations of existing methods. We comprehensively extract evolutionary information within promoters using chaos game representation and process the overall information with DenseNet and Transformer. Our model achieves state-of-the-art results on two kinds of distinct tasks. The incorporation of evolutionary information enhances the models accuracy, with transfer learning further extending its adaptability. Furthermore, experimental results confirm CAPEs efficacy in simulating in silico directed evolution of promoters, marking a significant advancement in predictive modeling for prokaryotic promoter strength. Our paper also presents a user-friendly website for the practical implementation of in silico directed evolution on promoters.\nYeastIT: Reducing mutational bias for in vivo directed evolution using a novel yeast mutator strain based on dual adenine-/cytosine-targeting and error-prone DNA repair\nAuthors: Napiorkowska, M.; Fischer, K.; Penner, M.; Knyphausen, P.; Hollfelder, F.\nScore: 7.2, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567881\nEngineering proteins with new functions and properties often requires navigating large sequence spaces through rounds of iterative improvement. However, a disparity exists between the gradual pace of natural long-term evolution and a typical laboratory evolution workflow that relies on enriching functional variants from highly diverse in vitro generated libraries through very few screening rounds. Laboratory experiments often eschew presumed natural strategies such as neutral/non-adaptive and multi-phase evolution trajectories, and therefore mutagenesis technologies suitable for long nature-like timescales are needed. Here, we introduce YeastIT, a novel in vivo mutagenesis tool for protein engineering that leverages an S. cerevisiae strain engineered to exhibit mutagenic activity directed to the gene of interest, allowing its continuous diversification. Mutagenesis is achieved by generating DNA damage through nucleoside deamination, followed by introduction of mutations by harnessing the process of error-prone DNA translesion synthesis. By eliminating the transformation step, YeastIT allows multiple rounds of screening or selection without interruptions for library diversification, thereby enabling long-term and continuous evolution campaigns. Our characterization of the mutational spectrum and frequency of the YeastIT-generated libraries, and its comparison to other methods (error-prone PCR, PACE, MutaT7, eMutaT7, OrthoRep, TRIDENT, EvolVR) demonstrates comparable mutation rates combined with a significant reduction in mutagenic bias relative to most of the alternatives. To validate YeastIT, we carried out directed evolution of a DARPin binding protein to achieve a 15-fold improved affinity. YeastIT thus provides a tool for exploring different evolutionary trajectories which overcomes previous limitations of variant availability (due to bias and low mutation rates) and emulates the way proteins emerge in Nature.\nContinuous evolution of user-defined genes at 1-million-times the genomic mutation rate\nAuthors: Rix, G.; Williams, R. L.; Spinner, H.; Hu, V. J.; Marks, D. S.; Liu, C. C.\nScore: 57.7, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566922\nWhen nature maintains or evolves a genes function over millions of years at scale, it produces a diversity of homologous sequences whose patterns of conservation and change contain rich structural, functional, and historical information about the gene. However, natural gene diversity likely excludes vast regions of functional sequence space and includes phylogenetic and evolutionary eccentricities, limiting what information we can extract. We introduce an accessible experimental approach for compressing long-term gene evolution to laboratory timescales, allowing for the direct observation of extensive adaptation and divergence followed by inference of structural, functional, and environmental constraints for any selectable gene. To enable this approach, we developed a new orthogonal DNA replication (OrthoRep) system that durably hypermutates chosen genes at a rate of \u003e10-4 substitutions per base in vivo. When OrthoRep was used to evolve a conditionally essential maladapted enzyme, we obtained thousands of unique multi-mutation sequences with many pairs \u003e60 amino acids apart (\u003e15% divergence), revealing known and new factors influencing enzyme adaptation. The fitness of evolved sequences was not predictable by advanced machine learning models trained on natural variation. We suggest that OrthoRep supports the prospective and systematic discovery of constraints shaping gene evolution, uncovering of new regions in fitness landscapes, and general applications in biomolecular engineering.\ndFLASH; dual FLuorescent transcription factor Activity Sensor for Histone integrated live-cell reporting and high-content screening\nAuthors: Allen, T. P.; Roennfeldt, A. E.; Reckdharajkumar, M.; Liu, M.; Quinn, R.; Russell, D. L.; Peet, D. J.; Whitelaw, M. L.; Bersten, D. C.\nScore: 4.5, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568191\nLive-cell reporting of regulated transcription factor (TF) activity has a wide variety of applications in synthetic biology, drug discovery, and functional genomics. As a result, there is high value in the generation of versatile, sensitive, robust systems that can function across a range of cell types and be adapted toward diverse TF classes. Here we present the dual FLuorescent transcription factor Activity Sensor for Histone integrated live-cell reporting (dFLASH), a modular sensor for TF activity that can be readily integrated into cellular genomes. We demonstrate readily modified dFLASH platforms that homogenously, robustly, and specifically sense regulation of endogenous Hypoxia Inducible Factor (HIF) and Progesterone receptor (PGR) activities, as well as regulated coactivator recruitment to a synthetic DNA-Binding Domain- Activator Domain fusion proteins. The dual-colour nuclear fluorescence produced normalised dynamic live-cell TF activity sensing with facile generation of high-content screening lines with strong signal:noise ratios and reproducible screening capabilities (Z-prime = 0.68-0.74). Finally, we demonstrate the utility of this platform for functional genomics applications by using CRISPRoff to modulate the HIF regulatory pathway, and utility for drug screening by using high content imaging in a bimodal design to isolate activators and inhibitors of the HIF pathway from a ~1600 natural product library.\nGenomically integrated orthogonal translation in Escherichia coli, a new synthetic auxotrophic chassis with altered genetic code, genetic firewall, and enhanced protein expression\nAuthors: Karbalaei-Heidari, H. R.; Budisa, N.\nScore: 3.8, Published: 2023-11-19 DOI: 10.1101/2023.11.18.567690\nIn the last three decades, genetic code engineering has expanded protein biosynthesis options from the natural set of 20 canonical amino acids to over 250 non-canonical amino acids (ncAAs). This progress involves rewiring of protein translation by establishing Orthogonal Translation Systems (OTS) through orthogonal pairs. Traditionally encoded on plasmid vectors, these systems are often unstable and burdensome in large-scale fermentations. To moving forward from academia to reliable technology, it is crucial to integrate OTS genetic modules stably into a molecular chassis with a defined genome background. Here, we demonstrate genomically integrated OTS in Escherichia coli, creating a synthetic auxotrophic chassis with an altered genetic code. Using CRISPR-associated transposase tool (CASTs), we targeted multiple genome sites, inserting OTS components (enzymes, tRNA genes) non-disruptively. Our OTS system demonstrated site-specific incorporation of m-oNB-Dopa through in-frame amber stop codon readthrough, enabling the expression of smart underwater bioglues. Simple metabolic labelling, introducing fluoroproline analogs enhancing conformational stability during orthogonal translation, further bolstered system robustness. These chassis, equipped also with synthetic auxotrophy for m-oNB-Dopa, serve as a built-in genetic barrier (genetic firewall), ensuring safe bioproduction in genetically isolated settings. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=120 SRC=\"FIGDIR/small/567690v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (37K): org.highwire.dtl.DTLVardef@496f34org.highwire.dtl.DTLVardef@123cc67org.highwire.dtl.DTLVardef@140a4bborg.highwire.dtl.DTLVardef@fefef7_HPS_FORMAT_FIGEXP M_FIG C_FIG One-sentence AbstractCRISPR-assisted transposition has enabled the development of a robust and biosafe Escherichia coli-based chassis with genomically integrated orthogonal translation components, enhancing synthetic protein production and laying the foundation for the transition of this research field from academia to reliable technology.\nDirected Evolution of Genetically Encoded LYTACs for Cell-Mediated Delivery\nAuthors: Yang, J. L.; Yamada-Hunter, S. A.; Labanieh, L.; Sotillo, E.; Cheah, J. S.; Roberts, D. S.; Mackall, C.; Ting, A. Y.; Bertozzi, C.\nScore: 21.0, Published: 2023-11-15 DOI: 10.1101/2023.11.14.567117\nLysosome-targeting chimeras (LYTACs) are a promising therapeutic modality to drive the degradation of extracellular proteins. However, early versions of LYTAC contain synthetic glycopeptides that cannot be genetically encoded. Here we present our designs for a fully genetically encodable LYTAC (GELYTAC), making our tool compatible with integration into therapeutic cells for targeted delivery at diseased sites. To achieve this, we replaced the glycopeptide portion of LYTACs with the protein insulin like growth factor 2 (IGF2). After showing initial efficacy with wild type IGF2, we increased the potency of GELYTAC using directed evolution. Subsequently, we demonstrated that our engineered GELYTAC construct not only secretes from HEK293T cells but also from human primary T-cells to drive the uptake of various targets into receiver cells. Immune cells engineered to secrete GELYTAC thus represent a promising avenue for spatially-selective targeted protein degradation. Significance StatementBetter therapeutic windows can be achieved by targeting therapeutics to their desired sites of action. For protein therapeutics, this might be achieved by engineering cell therapies that home to a tissue of interest and secrete the biologic drug locally. Here, we demonstrate that human primary T cells can be engineered to produce genetically encoded lysosome targeting chimeras (GELYTACs). These GELYTACs mediate the degradation of extracellular proteins associated with cancer progression. Thus, cells engineered to produce GELYTACs represent a potential new class of cancer therapeutics.\nCryptography in the DNA of living cells enabled by multi-site base editing\nAuthors: Volf, V.; Zhang, S.; Song, K. M.; Qian, S.; Chen, F.; Church, G.\nScore: 15.9, Published: 2023-11-15 DOI: 10.1101/2023.11.15.567131\nAlthough DNA is increasingly being adopted as a generalizable medium for information storage and transfer, reliable methods for ensuring information security remain to be addressed. In this study, we developed and validated a cryptographic encoding scheme, Genomic Sequence Encryption (GSE), to address the challenge of information confidentiality and integrity in biological substrates. GSE enables genomic information encoding that is readable only with a cryptographic key. We show that GSE can be used for cell signatures that enable the recipient of a cell line to authenticate its origin and validate if the cell line has been modified in the interim. We implement GSE through multi-site base editing and encode information through editing across \u003e100 genomic sites in mammalian cells. We further present an enrichment step to obtain individual stem cells with more than two dozen edits across a single genome with minimal screening. This capability can be used to introduce encrypted signatures in living animals. As an encryption scheme, GSE is falsification-proof and enables secure information transfer in biological substrates.\nAn experimental target-based platform in yeast for screening Plasmodium vivax deoxyhypusine synthase inhibitors\nAuthors: Fernandes Silva, S.; Klippel, A. H.; Sigurdardottir, S.; Mahdizadeh, S. J.; Tiukova, I.; Bourgard, C.; Monteiro do Amaral Prado, H.; de Araujo, R. V.; Bilsland, E.; King, R.; Massirer, K. B.; Eriksson, L.; Bengtson, M. H.; Sunnerhagen, P.; Zanelli, C. F.\nScore: 1.2, Published: 2023-11-24 DOI: 10.1101/2023.11.23.568514\nThe enzyme deoxyhypusine synthase (DHS) catalyzes the first step in the post-translational modification of the eukaryotic translation factor 5A (eIF5A). This is the only protein known to contain the amino acid hypusine, which results from this modification. Both eIF5A and DHS are essential for cell viability in eukaryotes, and inhibiting DHS can be a promising strategy for the development of new therapeutic alternatives. The human and parasitic orthologous proteins are different enough to render selective targeting against infectious diseases; however, no DHS inhibitor selective for the parasite ortholog has previously been reported. Here, we established a yeast surrogate genetics platform to identify inhibitors of DHS from Plasmodium vivax, one of the major causative agents of malaria. We constructed genetically modified Saccharomyces cerevisiae strains expressing DHS genes from Homo sapiens (HsDHS) or P. vivax (PvDHS) in place of the endogenous DHS gene from S. cerevisiae. This new strain background was ~60-fold more sensitive to an inhibitor of human DHS than the one previously used. Initially, a virtual screen using datasets from the ChEMBL-NTD database was performed. Candidate ligands were tested in growth assays using the newly generated yeast strains expressing heterologous DHS genes. Among these, two showed promise by preferentially reducing the growth of the PvDHS-expressing strain. Further, in a robotized assay, we screened 400 compounds from the Pathogen Box library using the same S. cerevisiae strains, and one compound preferentially reduced the growth of the PvDHS-expressing yeast strain. Western blot revealed that these compounds significantly reduced eIF5A hypusination in yeast. Our study demonstrates that this yeast-based platform is suitable for identifying and verifying candidate small molecule DHS inhibitors, selective for the parasite over the human ortholog.\n",
  "wordCount" : "2484",
  "inLanguage": "en",
  "datePublished": "2023-11-26T10:37:33Z",
  "dateModified": "2023-11-26T10:37:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 26, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.19.567736">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.19.567736" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.19.567736">
        <p class="paperTitle">Engineering orthogonal ribosomes for real-time monitoring using fluorescence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.19.567736" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.19.567736" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Csibra, E.; Klopprogge, B.; Hau Sorensen, G.; Gorochowski, T. E.</p>
        <p class="info">Score: 22.2, Published: 2023-11-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.19.567736' target='https://doi.org/10.1101/2023.11.19.567736'> 10.1101/2023.11.19.567736</a></p>
        <p class="abstract">A promising route to tackle the trade-off in cellular resources between synthetic protein production and cellular growth is to use a separate dedicated pool of orthogonal ribosomes to produce synthetic proteins. However, the optimisation of strains containing two ribosomal pools - native for the host cells proteome and orthogonal for synthetic proteins - has yet to be thoroughly explored. Here, we address this by creating orthogonal ribosomes that fluoresce by inserting fluorescent RNA aptamers into tethered orthogonal ribosomal RNA (TO-rRNA). To study the tolerance of the engineered ribosomes to aptamer insertion, we assembled and screened a library of candidate insertion sites, identifying several sites in both the 16S and 23S TO-rRNA that enables ribosome labelling with minimal effect on translation activity. Serendipitously, we identify one site in 23S TO-rRNA, where insertion appears to not only be tolerated but to enhance orthogonal ribosome activity, across multiple bacterial strains and RNA insertions. Using bulk and single cell assays, we demonstrate that this variant allows us to label orthogonal ribosomes for dynamic tracking and across populations, making it a promising tool for optimising orthogonal translation in engineered cells. Ribosome engineering offers great potential, both for the development of next-generation microbial cell factories, as well as a tool to expand our understanding of ribosome function in living cells. This work provides a window into the assembly, localisation and function of these molecular machines to meet these aims.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567888">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567888" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567888">
        <p class="paperTitle">Highly parallelized construction of DNA from low-cost oligonucleotide mixtures using Data-optimized Assembly Design and Golden Gate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567888" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567888" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lund, S.; Potapov, V.; Johnson, S. R.; Buss, J.; Tanner, N. A.</p>
        <p class="info">Score: 16.1, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567888' target='https://doi.org/10.1101/2023.11.20.567888'> 10.1101/2023.11.20.567888</a></p>
        <p class="abstract">Commercially synthesized genes are typically made using variations of homology-based cloning techniques, including polymerase cycling assembly from chemically synthesized microarray-derived oligonucleotides. Here we apply Data-optimized Assembly Design to the synthesis of hundreds of codon-optimized genes in both constitutive and inducible vectors using Golden Gate Assembly. Starting from oligonucleotide pools, we synthesize genes in three simple steps: 1) Amplification of parts belonging to individual assemblies in parallel from a single pool; 2) Golden Gate Assembly of parts for each construct; and 3) Transformation. We construct genes from receiving DNA to sequence confirmed isolates in as little as 4 days. By leveraging the ligation fidelity afforded by T4 DNA ligase, we expect to be able to construct a larger breadth of sequences not currently supported by homology-based methods which require stability of extensive single-stranded DNA overhangs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.18.567645">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.18.567645" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.18.567645">
        <p class="paperTitle">Evolution is All You Need in Promoter Design and Optimization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.18.567645" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.18.567645" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ren, R.; Yu, H.; Teng, J.; Mao, S.; Bian, Z.; Tao, Y.; Yau, S. S.-T.</p>
        <p class="info">Score: 8.4, Published: 2023-11-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.18.567645' target='https://doi.org/10.1101/2023.11.18.567645'> 10.1101/2023.11.18.567645</a></p>
        <p class="abstract">Predicting the strength of promoters and guiding their directed evolution is a crucial task in synthetic biology. This approach significantly reduces the experimental costs in conventional promoter engineering. Previous studies employing machine learning or deep learning methods have shown some success in this task, but their outcomes were not satisfactory enough, primarily due to the neglect of evolutionary information. In this paper, we introduce the Chaos-Attention net for Promoter Evolution (CAPE) to address the limitations of existing methods. We comprehensively extract evolutionary information within promoters using chaos game representation and process the overall information with DenseNet and Transformer. Our model achieves state-of-the-art results on two kinds of distinct tasks. The incorporation of evolutionary information enhances the models accuracy, with transfer learning further extending its adaptability. Furthermore, experimental results confirm CAPEs efficacy in simulating in silico directed evolution of promoters, marking a significant advancement in predictive modeling for prokaryotic promoter strength. Our paper also presents a user-friendly website for the practical implementation of in silico directed evolution on promoters.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567881">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567881" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567881">
        <p class="paperTitle">YeastIT: Reducing mutational bias for in vivo directed evolution using a novel yeast mutator strain based on dual adenine-/cytosine-targeting and error-prone DNA repair</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567881" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567881" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Napiorkowska, M.; Fischer, K.; Penner, M.; Knyphausen, P.; Hollfelder, F.</p>
        <p class="info">Score: 7.2, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567881' target='https://doi.org/10.1101/2023.11.20.567881'> 10.1101/2023.11.20.567881</a></p>
        <p class="abstract">Engineering proteins with new functions and properties often requires navigating large sequence spaces through rounds of iterative improvement. However, a disparity exists between the gradual pace of natural long-term evolution and a typical laboratory evolution workflow that relies on enriching functional variants from highly diverse in vitro generated libraries through very few screening rounds. Laboratory experiments often eschew presumed natural strategies such as neutral/non-adaptive and multi-phase evolution trajectories, and therefore mutagenesis technologies suitable for long  nature-like timescales are needed. Here, we introduce YeastIT, a novel in vivo mutagenesis tool for protein engineering that leverages an S. cerevisiae strain engineered to exhibit mutagenic activity directed to the gene of interest, allowing its continuous diversification. Mutagenesis is achieved by generating DNA damage through nucleoside deamination, followed by introduction of mutations by harnessing the process of error-prone DNA translesion synthesis. By eliminating the transformation step, YeastIT allows multiple rounds of screening or selection without interruptions for library diversification, thereby enabling long-term and continuous evolution campaigns. Our characterization of the mutational spectrum and frequency of the YeastIT-generated libraries, and its comparison to other methods (error-prone PCR, PACE, MutaT7, eMutaT7, OrthoRep, TRIDENT, EvolVR) demonstrates comparable mutation rates combined with a significant reduction in mutagenic bias relative to most of the alternatives. To validate YeastIT, we carried out directed evolution of a DARPin binding protein to achieve a 15-fold improved affinity. YeastIT thus provides a tool for exploring different evolutionary trajectories which overcomes previous limitations of variant availability (due to bias and low mutation rates) and emulates the way proteins emerge in Nature.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566922">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566922" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566922">
        <p class="paperTitle">Continuous evolution of user-defined genes at 1-million-times the genomic mutation rate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566922" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566922" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rix, G.; Williams, R. L.; Spinner, H.; Hu, V. J.; Marks, D. S.; Liu, C. C.</p>
        <p class="info">Score: 57.7, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566922' target='https://doi.org/10.1101/2023.11.13.566922'> 10.1101/2023.11.13.566922</a></p>
        <p class="abstract">When nature maintains or evolves a genes function over millions of years at scale, it produces a diversity of homologous sequences whose patterns of conservation and change contain rich structural, functional, and historical information about the gene. However, natural gene diversity likely excludes vast regions of functional sequence space and includes phylogenetic and evolutionary eccentricities, limiting what information we can extract. We introduce an accessible experimental approach for compressing long-term gene evolution to laboratory timescales, allowing for the direct observation of extensive adaptation and divergence followed by inference of structural, functional, and environmental constraints for any selectable gene. To enable this approach, we developed a new orthogonal DNA replication (OrthoRep) system that durably hypermutates chosen genes at a rate of &gt;10-4 substitutions per base in vivo. When OrthoRep was used to evolve a conditionally essential maladapted enzyme, we obtained thousands of unique multi-mutation sequences with many pairs &gt;60 amino acids apart (&gt;15% divergence), revealing known and new factors influencing enzyme adaptation. The fitness of evolved sequences was not predictable by advanced machine learning models trained on natural variation. We suggest that OrthoRep supports the prospective and systematic discovery of constraints shaping gene evolution, uncovering of new regions in fitness landscapes, and general applications in biomolecular engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568191">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568191" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568191">
        <p class="paperTitle">dFLASH; dual FLuorescent transcription factor Activity Sensor for Histone integrated live-cell reporting and high-content screening</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568191" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568191" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Allen, T. P.; Roennfeldt, A. E.; Reckdharajkumar, M.; Liu, M.; Quinn, R.; Russell, D. L.; Peet, D. J.; Whitelaw, M. L.; Bersten, D. C.</p>
        <p class="info">Score: 4.5, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568191' target='https://doi.org/10.1101/2023.11.21.568191'> 10.1101/2023.11.21.568191</a></p>
        <p class="abstract">Live-cell reporting of regulated transcription factor (TF) activity has a wide variety of applications in synthetic biology, drug discovery, and functional genomics. As a result, there is high value in the generation of versatile, sensitive, robust systems that can function across a range of cell types and be adapted toward diverse TF classes. Here we present the dual FLuorescent transcription factor Activity Sensor for Histone integrated live-cell reporting (dFLASH), a modular sensor for TF activity that can be readily integrated into cellular genomes. We demonstrate readily modified dFLASH platforms that homogenously, robustly, and specifically sense regulation of endogenous Hypoxia Inducible Factor (HIF) and Progesterone receptor (PGR) activities, as well as regulated coactivator recruitment to a synthetic DNA-Binding Domain- Activator Domain fusion proteins. The dual-colour nuclear fluorescence produced normalised dynamic live-cell TF activity sensing with facile generation of high-content screening lines with strong signal:noise ratios and reproducible screening capabilities (Z-prime = 0.68-0.74). Finally, we demonstrate the utility of this platform for functional genomics applications by using CRISPRoff to modulate the HIF regulatory pathway, and utility for drug screening by using high content imaging in a bimodal design to isolate activators and inhibitors of the HIF pathway from a ~1600 natural product library.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.18.567690">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.18.567690" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.18.567690">
        <p class="paperTitle">Genomically integrated orthogonal translation in Escherichia coli, a new synthetic auxotrophic chassis with altered genetic code, genetic firewall, and enhanced protein expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.18.567690" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.18.567690" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Karbalaei-Heidari, H. R.; Budisa, N.</p>
        <p class="info">Score: 3.8, Published: 2023-11-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.18.567690' target='https://doi.org/10.1101/2023.11.18.567690'> 10.1101/2023.11.18.567690</a></p>
        <p class="abstract">In the last three decades, genetic code engineering has expanded protein biosynthesis options from the natural set of 20 canonical amino acids to over 250 non-canonical amino acids (ncAAs). This progress involves rewiring of protein translation by establishing Orthogonal Translation Systems (OTS) through orthogonal pairs. Traditionally encoded on plasmid vectors, these systems are often unstable and burdensome in large-scale fermentations. To moving forward from academia to reliable technology, it is crucial to integrate OTS genetic modules stably into a molecular chassis with a defined genome background. Here, we demonstrate genomically integrated OTS in Escherichia coli, creating a synthetic auxotrophic chassis with an altered genetic code. Using CRISPR-associated transposase tool (CASTs), we targeted multiple genome sites, inserting OTS components (enzymes, tRNA genes) non-disruptively. Our OTS system demonstrated site-specific incorporation of m-oNB-Dopa through in-frame amber stop codon readthrough, enabling the expression of smart underwater bioglues. Simple metabolic labelling, introducing fluoroproline analogs enhancing conformational stability during orthogonal translation, further bolstered system robustness. These chassis, equipped also with synthetic auxotrophy for m-oNB-Dopa, serve as a built-in genetic barrier (genetic firewall), ensuring safe bioproduction in genetically isolated settings.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=120 SRC=&#34;FIGDIR/small/567690v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (37K):
org.highwire.dtl.DTLVardef@496f34org.highwire.dtl.DTLVardef@123cc67org.highwire.dtl.DTLVardef@140a4bborg.highwire.dtl.DTLVardef@fefef7_HPS_FORMAT_FIGEXP  M_FIG C_FIG One-sentence AbstractCRISPR-assisted transposition has enabled the development of a robust and biosafe Escherichia coli-based chassis with genomically integrated orthogonal translation components, enhancing synthetic protein production and laying the foundation for the transition of this research field from academia to reliable technology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567117">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567117" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567117">
        <p class="paperTitle">Directed Evolution of Genetically Encoded LYTACs for Cell-Mediated Delivery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567117" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567117" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, J. L.; Yamada-Hunter, S. A.; Labanieh, L.; Sotillo, E.; Cheah, J. S.; Roberts, D. S.; Mackall, C.; Ting, A. Y.; Bertozzi, C.</p>
        <p class="info">Score: 21.0, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567117' target='https://doi.org/10.1101/2023.11.14.567117'> 10.1101/2023.11.14.567117</a></p>
        <p class="abstract">Lysosome-targeting chimeras (LYTACs) are a promising therapeutic modality to drive the degradation of extracellular proteins. However, early versions of LYTAC contain synthetic glycopeptides that cannot be genetically encoded. Here we present our designs for a fully genetically encodable LYTAC (GELYTAC), making our tool compatible with integration into therapeutic cells for targeted delivery at diseased sites. To achieve this, we replaced the glycopeptide portion of LYTACs with the protein insulin like growth factor 2 (IGF2). After showing initial efficacy with wild type IGF2, we increased the potency of GELYTAC using directed evolution. Subsequently, we demonstrated that our engineered GELYTAC construct not only secretes from HEK293T cells but also from human primary T-cells to drive the uptake of various targets into receiver cells. Immune cells engineered to secrete GELYTAC thus represent a promising avenue for spatially-selective targeted protein degradation.

Significance StatementBetter therapeutic windows can be achieved by targeting therapeutics to their desired sites of action. For protein therapeutics, this might be achieved by engineering cell therapies that home to a tissue of interest and secrete the biologic drug locally. Here, we demonstrate that human primary T cells can be engineered to produce genetically encoded lysosome targeting chimeras (GELYTACs). These GELYTACs mediate the degradation of extracellular proteins associated with cancer progression. Thus, cells engineered to produce GELYTACs represent a potential new class of cancer therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567131">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567131" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567131">
        <p class="paperTitle">Cryptography in the DNA of living cells enabled by multi-site base editing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567131" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567131" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Volf, V.; Zhang, S.; Song, K. M.; Qian, S.; Chen, F.; Church, G.</p>
        <p class="info">Score: 15.9, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567131' target='https://doi.org/10.1101/2023.11.15.567131'> 10.1101/2023.11.15.567131</a></p>
        <p class="abstract">Although DNA is increasingly being adopted as a generalizable medium for information storage and transfer, reliable methods for ensuring information security remain to be addressed. In this study, we developed and validated a cryptographic encoding scheme, Genomic Sequence Encryption (GSE), to address the challenge of information confidentiality and integrity in biological substrates. GSE enables genomic information encoding that is readable only with a cryptographic key. We show that GSE can be used for cell signatures that enable the recipient of a cell line to authenticate its origin and validate if the cell line has been modified in the interim. We implement GSE through multi-site base editing and encode information through editing across &gt;100 genomic sites in mammalian cells. We further present an enrichment step to obtain individual stem cells with more than two dozen edits across a single genome with minimal screening. This capability can be used to introduce encrypted signatures in living animals. As an encryption scheme, GSE is falsification-proof and enables secure information transfer in biological substrates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568514">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568514" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568514">
        <p class="paperTitle">An experimental target-based platform in yeast for screening Plasmodium vivax deoxyhypusine synthase inhibitors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568514" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568514" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fernandes Silva, S.; Klippel, A. H.; Sigurdardottir, S.; Mahdizadeh, S. J.; Tiukova, I.; Bourgard, C.; Monteiro do Amaral Prado, H.; de Araujo, R. V.; Bilsland, E.; King, R.; Massirer, K. B.; Eriksson, L.; Bengtson, M. H.; Sunnerhagen, P.; Zanelli, C. F.</p>
        <p class="info">Score: 1.2, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568514' target='https://doi.org/10.1101/2023.11.23.568514'> 10.1101/2023.11.23.568514</a></p>
        <p class="abstract">The enzyme deoxyhypusine synthase (DHS) catalyzes the first step in the post-translational modification of the eukaryotic translation factor 5A (eIF5A). This is the only protein known to contain the amino acid hypusine, which results from this modification. Both eIF5A and DHS are essential for cell viability in eukaryotes, and inhibiting DHS can be a promising strategy for the development of new therapeutic alternatives. The human and parasitic orthologous proteins are different enough to render selective targeting against infectious diseases; however, no DHS inhibitor selective for the parasite ortholog has previously been reported. Here, we established a yeast surrogate genetics platform to identify inhibitors of DHS from Plasmodium vivax, one of the major causative agents of malaria. We constructed genetically modified Saccharomyces cerevisiae strains expressing DHS genes from Homo sapiens (HsDHS) or P. vivax (PvDHS) in place of the endogenous DHS gene from S. cerevisiae. This new strain background was ~60-fold more sensitive to an inhibitor of human DHS than the one previously used. Initially, a virtual screen using datasets from the ChEMBL-NTD database was performed. Candidate ligands were tested in growth assays using the newly generated yeast strains expressing heterologous DHS genes. Among these, two showed promise by preferentially reducing the growth of the PvDHS-expressing strain. Further, in a robotized assay, we screened 400 compounds from the Pathogen Box library using the same S. cerevisiae strains, and one compound preferentially reduced the growth of the PvDHS-expressing yeast strain. Western blot revealed that these compounds significantly reduced eIF5A hypusination in yeast. Our study demonstrates that this yeast-based platform is suitable for identifying and verifying candidate small molecule DHS inhibitors, selective for the parasite over the human ortholog.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
