<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="DNA-Diffusion: Leveraging Generative Models for Controlling Chromatin Accessibility and Gene Expression via Synthetic Regulatory Elements
Authors: Ferreira DaSilva, L.; Senan, S.; Patel, Z. M.; Reddy, A. J.; Gabbita, S.; Nussbaum, Z.; Cordova, C. M. V.; Wenteler, A.; Weber, N.; Tunjic, T. M.; Khan, T. A.; Li, Z.; Smith, C. R.; Bejan, M.; Louis, L. K.; Cornejo, P.; Connell, W.; Wong, E. S.; Meuleman, W.; Pinello, L.
Score: 54.8, Published: 2024-02-01 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="DNA-Diffusion: Leveraging Generative Models for Controlling Chromatin Accessibility and Gene Expression via Synthetic Regulatory Elements
Authors: Ferreira DaSilva, L.; Senan, S.; Patel, Z. M.; Reddy, A. J.; Gabbita, S.; Nussbaum, Z.; Cordova, C. M. V.; Wenteler, A.; Weber, N.; Tunjic, T. M.; Khan, T. A.; Li, Z.; Smith, C. R.; Bejan, M.; Louis, L. K.; Cornejo, P.; Connell, W.; Wong, E. S.; Meuleman, W.; Pinello, L.
Score: 54.8, Published: 2024-02-01 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-04T10:38:39+00:00" />
<meta property="article:modified_time" content="2024-02-04T10:38:39+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="DNA-Diffusion: Leveraging Generative Models for Controlling Chromatin Accessibility and Gene Expression via Synthetic Regulatory Elements
Authors: Ferreira DaSilva, L.; Senan, S.; Patel, Z. M.; Reddy, A. J.; Gabbita, S.; Nussbaum, Z.; Cordova, C. M. V.; Wenteler, A.; Weber, N.; Tunjic, T. M.; Khan, T. A.; Li, Z.; Smith, C. R.; Bejan, M.; Louis, L. K.; Cornejo, P.; Connell, W.; Wong, E. S.; Meuleman, W.; Pinello, L.
Score: 54.8, Published: 2024-02-01 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "DNA-Diffusion: Leveraging Generative Models for Controlling Chromatin Accessibility and Gene Expression via Synthetic Regulatory Elements\nAuthors: Ferreira DaSilva, L.; Senan, S.; Patel, Z. M.; Reddy, A. J.; Gabbita, S.; Nussbaum, Z.; Cordova, C. M. V.; Wenteler, A.; Weber, N.; Tunjic, T. M.; Khan, T. A.; Li, Z.; Smith, C. R.; Bejan, M.; Louis, L. K.; Cornejo, P.; Connell, W.; Wong, E. S.; Meuleman, W.; Pinello, L.\nScore: 54.8, Published: 2024-02-01 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " DNA-Diffusion: Leveraging Generative Models for Controlling Chromatin Accessibility and Gene Expression via Synthetic Regulatory Elements\nAuthors: Ferreira DaSilva, L.; Senan, S.; Patel, Z. M.; Reddy, A. J.; Gabbita, S.; Nussbaum, Z.; Cordova, C. M. V.; Wenteler, A.; Weber, N.; Tunjic, T. M.; Khan, T. A.; Li, Z.; Smith, C. R.; Bejan, M.; Louis, L. K.; Cornejo, P.; Connell, W.; Wong, E. S.; Meuleman, W.; Pinello, L.\nScore: 54.8, Published: 2024-02-01 DOI: 10.1101/2024.02.01.578352\nThe challenge of systematically modifying and optimizing regulatory elements for precise gene expression control is central to modern genomics and synthetic biology. Advancements in generative AI have paved the way for designing synthetic sequences with the aim of safely and accurately modulating gene expression. We leverage diffusion models to design context-specific DNA regulatory sequences, which hold significant potential toward enabling novel therapeutic applications requiring precise modulation of gene expression. Our framework uses a cell type-specific diffusion model to generate synthetic 200 bp regulatory elements based on chromatin accessibility across different cell types. We evaluate the generated sequences based on key metrics to ensure they retain properties of endogenous sequences: transcription factor binding site composition, potential for cell type-specific chromatin accessibility, and capacity for sequences generated by DNA diffusion to activate gene expression in different cell contexts using state-of-the-art prediction models. Our results demonstrate the ability to robustly generate DNA sequences with cell type-specific regulatory potential. DNA-Diffusion paves the way for revolutionizing a regulatory modulation approach to mammalian synthetic biology and precision gene therapy.\nCRISPR-Enabled Autonomous Transposable Element (CREATE) for RNA-based gene editing and delivery\nAuthors: Wang, Y.; Lin, R.-Z.; Harris, M.; Lavayen, B.; Mccreedy, B.; Getts, D.\nScore: 8.4, Published: 2024-01-30 DOI: 10.1101/2024.01.29.577809\nTo realize the potential of genome editing for broad therapeutic applications, new tools are needed for delivery of multi-kilobase (kb) payloads at desired genomic sites with high precision. Here we introduce the CRISPR-Enabled Autonomous Transposable Element (CREATE), an RNA-based genome editing system that merges CRISPR/Cas9 with the human L1 retrotransposon to insert gene-sized payloads without DNA donors or double strand breaks. CREATE enables L1-mediated reverse transcription and integration of an RNA-encoded payload gene specifically at two Cas9-induced nick sites. CREATE is delivered using mRNA components and achieves integration of \u003e1 kb payload in mammalian cells, opening the door to mRNA mediated therapeutic genome editing in vivo. One-Sentence SummaryAn mRNA-based, large-scale genome editing technology integrating CRISPR/Cas9 and L1 retroelement.\nUltra-sensitive water contaminant detection with transcription factor interfaced microcantilevers\nAuthors: Agarwal, D. K.; Lucci, T. J.; Jung, J. K.; Shekhawat, G. S.; Lucks, J. B.; Dravid, V. P. B.\nScore: 6.8, Published: 2024-02-01 DOI: 10.1101/2024.02.01.578376\nWater contamination is a growing global concern, creating a need to develop technologies that can detect a range of target compounds at the required thresholds. Here, we address this need by merging biological allosteric transcription factors with DNA coated nanomechanical microcantilevers to detect chemicals in water with digital readout. After proof-of-concept demonstration and optimization to detect tetracycline with the TetR transcription factor, we use the CadC transcription factor to detect Pb2+ and Cd2+ in water at concentrations down to 2 ppb and 1 ppb, respectively, in less than fifteen minutes. A computational model suggests this improvement in sensitivity could be achieved by the DNA coated microcantilever surface changing transcription factor binding properties. Our findings demonstrate a promising new approach for water quality monitoring with fast, highly sensitive, digital readouts.\nDIB-BOT: An open-source hardware approach for high throughput droplet interface bilayer deposition\nAuthors: Mason, A. F.; Wickham, S. F.; Baker, M. A.\nScore: 6.8, Published: 2024-01-26 DOI: 10.1101/2024.01.26.577347\nDroplet interface bilayers (DIBs) provide a controlled lipid environment for the single-molecule investigation of a range of biologically relevant membrane-bound processes and have garnered attention for their potential applications in bottom-up artificial cells, biosensing, and biophysics. However, the fabrication of DIBs is currently hindered by time-consuming processes and specialized equipment. These fabrication limitations prevent the scale-up of DIB assays, making it difficult to generate the large data sets required to achieve statistically significant conclusions in single-molecule biological assays where heterogeneous behaviour is often observed. This research describes an open-source solution, dubbed \"DIB-BOT,\" constructed by coupling a nanoinjector with an entry-level 3D printer. We present DIB-BOT as a platform to achieve rapid, reproducible, and reliable fabrication of large numbers of DIBs, addressing the limitations of manual methods. Leveraging commercially available off-the-shelf components, DIB-BOT exhibits high spatial reproducibility, minimal user input, and the ability to scale experiments rapidly. Here we demonstrate the utility of the system by integrating pairwise droplet assembly with a fluorescence plate-reader to execute a biologically relevant assay. When compared with manual DIB fabrication, the DIB-BOT had a tenfold reduction in droplet volume error, a threefold reduction in positional error, and 100% droplet yield. Overall, this method has potential to reduce entry barriers to the use of DIB methods, broadening the applications of DIB research, and generating higher quality data sets.\ngraded-CRISPRi, a novel tool for tuning the strengths of CRISPRi-mediated knockdowns in Vibrio natriegens using gRNA libraries\nAuthors: Stukenberg, D.; Faber, A.; Becker, A.\nScore: 5.1, Published: 2024-01-29 DOI: 10.1101/2024.01.29.577714\nIn recent years, the fast-growing bacterium Vibrio natriegens has gained increasing attention as it has the potential to become a next-generation chassis for Synthetic Biology. A wide range of genetic parts and genome engineering methods have already been developed. However, there is still a need for a well-characterized tool to effectively and gradually reduce the expression level of native genes. To bridge this gap, we created graded-CRISPRi, a system utilizing gRNA variants that lead to varying levels of repression strength. By incorporating multiple gRNA sequences into our design, we successfully extended this concept to simultaneously repress four distinct reporter genes. Furthermore, we demonstrated the capability of using graded-CRISPRi to target native genes, thereby examining the effect of various knockdown levels on growth.\nThe integration of tandem gene repeats via a bacterial type-II toxin-antitoxin-mediated gene amplification (ToxAmp) system and stability visualisation in Saccharomyces cerevisiae\nAuthors: Evans, S.; Lu, Z.; McDonnell, L.; Anderson, W.; Peralta, F.; Watkins, T.; Ahmed, H.; Luna-Flores, C. H.; Loan, T. D.; Navone, L.; Trau, M.; Scott, C.; Speight, R.; Vickers, C.; Peng, B.\nScore: 2.0, Published: 2024-01-30 DOI: 10.1101/2024.01.30.578080\nTandem gene repeats naturally occur as important genomic features and determine many traits in living organisms, like human diseases and microbial productivities of target bioproducts. Here, we develop a bacterial type-II toxin-antitoxin-mediated method to manipulate genomic integration of tandem gene repeats in Saccharomyces cerevisiae and further visualise the evolutionary trajectories of gene repeats. We designed a tri-vector system to introduce toxin-antitoxin-driven gene amplification (ToxAmp) modules, and accidentally re-visited the high-level capacity of multi-fragment co-transformation in S. cerevisiae. This system delivered the multi-copy gene integration in the form of tandem gene repeats spontaneously and independently from toxin-antitoxin-mediated selection. Inducing the toxin (RelE) expressing via a copper (II)-inducible CUP1 promoter successfully drove the in-situ gene amplification of the antitoxin (RelB) module, resulting in [~]40 copies of a green fluorescence reporter (GFP) gene per copy of genome. The copy-number changes, increasing and decreasing, and stable maintenance were visualised using the GFP and blue chromoprotein AeBlue as reporters. Copy-number increasing happened spontaneously not depending on a selection pressure and was quickly enriched through toxin-antitoxin-mediated selection. In summary, the bacterial toxin-antitoxin systems provide a flexible mechanism to manipulate gene copy number in eukaryotic cells and can be exploited for synthetic biology and metabolic engineering applications. Table of Contents Graphic O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=146 SRC=\"FIGDIR/small/578080v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (36K): org.highwire.dtl.DTLVardef@1411376org.highwire.dtl.DTLVardef@164c036org.highwire.dtl.DTLVardef@1874713org.highwire.dtl.DTLVardef@4b6478_HPS_FORMAT_FIGEXP M_FIG C_FIG\nxTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein\nAuthors: Chen, B.; Cheng, X.; Li, P.; Gengyang, L.-a.; Gong, J.; Li, S.; Bei, Z.; Tan, X.; Wang, B.; Zeng, X.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.\nScore: 97.6, Published: 2024-01-11 DOI: 10.1101/2023.07.05.547496\nProtein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently. We propose a unified protein language model, xTrimoPGLM, to address these two types of tasks simultaneously through an innovative pre-training framework. Our key technical contribution is an exploration of the compatibility and the potential for joint optimization of the two types of objectives, which has led to a strategy for training xTrimoPGLM at an unprecedented scale of 100 billion parameters and 1 trillion training tokens. Our extensive experiments reveal that 1) xTrimoPGLM significantly outperforms other advanced baselines in 18 protein understanding benchmarks across four categories. The model also facilitates an atomic-resolution view of protein structures, leading to an advanced 3D structural prediction model that surpasses existing language model-based tools. 2) xTrimoPGLM not only can generate de novo protein sequences following the principles of natural ones, but also can perform programmable generation after supervised fine-tuning (SFT) on curated sequences. These results highlight the substantial capability and versatility of xTrimoPGLM in understanding and generating protein sequences, contributing to the evolving landscape of foundation models in protein science.\nSequence-function mapping of proline-rich antimicrobial peptides\nAuthors: Collins, J.; McConnell, A.; Schmitz, Z.; Hackel, B. J.\nScore: 1.2, Published: 2024-01-29 DOI: 10.1101/2024.01.28.577586\nAntimicrobial peptides (AMPs) are essential elements of natural cellular combat and candidates as antibiotic therapy. Elevated function may be needed for robust physiological performance. Yet, both pure protein design and combinatorial library discovery are hindered by the complexity of antimicrobial activity. We applied a recently developed high-throughput technique, sequence-activity mapping of AMPs via depletion (SAMP-Dep), to proline-rich AMPs. Robust self-inhibition was achieved for metalnikowin 1 (Met) and apidaecin 1b (Api). SAMP-Dep exhibited high reproducibility with correlation coefficients 0.90 and 0.92, for Met and Api, respectively, between replicates and 0.99 and 0.96 for synonymous genetic variants. Sequence-activity maps were obtained via characterization of 26,000 and 34,000 mutants of Met and Api, respectively. Both AMPs exhibit similar mutational profiles including beneficial mutations at one terminus, the C-terminus for Met and N-terminus for Api, which is consistent with their opposite binding orientations in the ribosome. While Met and Api reside with the family of proline-rich AMPs, different proline sites exhibit substantially different mutational tolerance. Within the PRP motif, proline mutation eliminates activity, whereas non-PRP prolines readily tolerate mutation. Homologous mutations are more tolerated, particularly at alternating sites on one face of the peptide. Important and consistent epistasis was observed following the PRP domain within the segment that extends into the ribosomal exit tunnel for both peptides. Variants identified from the SAMP-Dep platform were produced and exposed toward Gram-negative species exogenously, showing either increased potency or specificity for strains tested. In addition to mapping sequence-activity space for fundamental insight and therapeutic engineering, the results advance the robustness of the SAMP-Dep platform for activity characterization.\n",
  "wordCount" : "1771",
  "inLanguage": "en",
  "datePublished": "2024-02-04T10:38:39Z",
  "dateModified": "2024-02-04T10:38:39Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 4, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578352">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578352" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578352">
        <p class="paperTitle">DNA-Diffusion: Leveraging Generative Models for Controlling Chromatin Accessibility and Gene Expression via Synthetic Regulatory Elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578352" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578352" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ferreira DaSilva, L.; Senan, S.; Patel, Z. M.; Reddy, A. J.; Gabbita, S.; Nussbaum, Z.; Cordova, C. M. V.; Wenteler, A.; Weber, N.; Tunjic, T. M.; Khan, T. A.; Li, Z.; Smith, C. R.; Bejan, M.; Louis, L. K.; Cornejo, P.; Connell, W.; Wong, E. S.; Meuleman, W.; Pinello, L.</p>
        <p class="info">Score: 54.8, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578352' target='https://doi.org/10.1101/2024.02.01.578352'> 10.1101/2024.02.01.578352</a></p>
        <p class="abstract">The challenge of systematically modifying and optimizing regulatory elements for precise gene expression control is central to modern genomics and synthetic biology. Advancements in generative AI have paved the way for designing synthetic sequences with the aim of safely and accurately modulating gene expression. We leverage diffusion models to design context-specific DNA regulatory sequences, which hold significant potential toward enabling novel therapeutic applications requiring precise modulation of gene expression. Our framework uses a cell type-specific diffusion model to generate synthetic 200 bp regulatory elements based on chromatin accessibility across different cell types. We evaluate the generated sequences based on key metrics to ensure they retain properties of endogenous sequences: transcription factor binding site composition, potential for cell type-specific chromatin accessibility, and capacity for sequences generated by DNA diffusion to activate gene expression in different cell contexts using state-of-the-art prediction models. Our results demonstrate the ability to robustly generate DNA sequences with cell type-specific regulatory potential. DNA-Diffusion paves the way for revolutionizing a regulatory modulation approach to mammalian synthetic biology and precision gene therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577809">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577809" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577809">
        <p class="paperTitle">CRISPR-Enabled Autonomous Transposable Element (CREATE) for RNA-based gene editing and delivery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577809" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577809" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Y.; Lin, R.-Z.; Harris, M.; Lavayen, B.; Mccreedy, B.; Getts, D.</p>
        <p class="info">Score: 8.4, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577809' target='https://doi.org/10.1101/2024.01.29.577809'> 10.1101/2024.01.29.577809</a></p>
        <p class="abstract">To realize the potential of genome editing for broad therapeutic applications, new tools are needed for delivery of multi-kilobase (kb) payloads at desired genomic sites with high precision. Here we introduce the CRISPR-Enabled Autonomous Transposable Element (CREATE), an RNA-based genome editing system that merges CRISPR/Cas9 with the human L1 retrotransposon to insert gene-sized payloads without DNA donors or double strand breaks. CREATE enables L1-mediated reverse transcription and integration of an RNA-encoded payload gene specifically at two Cas9-induced nick sites. CREATE is delivered using mRNA components and achieves integration of &gt;1 kb payload in mammalian cells, opening the door to mRNA mediated therapeutic genome editing in vivo.

One-Sentence SummaryAn mRNA-based, large-scale genome editing technology integrating CRISPR/Cas9 and L1 retroelement.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578376">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578376" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578376">
        <p class="paperTitle">Ultra-sensitive water contaminant detection with transcription factor interfaced microcantilevers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578376" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578376" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Agarwal, D. K.; Lucci, T. J.; Jung, J. K.; Shekhawat, G. S.; Lucks, J. B.; Dravid, V. P. B.</p>
        <p class="info">Score: 6.8, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578376' target='https://doi.org/10.1101/2024.02.01.578376'> 10.1101/2024.02.01.578376</a></p>
        <p class="abstract">Water contamination is a growing global concern, creating a need to develop technologies that can detect a range of target compounds at the required thresholds. Here, we address this need by merging biological allosteric transcription factors with DNA coated nanomechanical microcantilevers to detect chemicals in water with digital readout. After proof-of-concept demonstration and optimization to detect tetracycline with the TetR transcription factor, we use the CadC transcription factor to detect Pb2&#43; and Cd2&#43; in water at concentrations down to 2 ppb and 1 ppb, respectively, in less than fifteen minutes. A computational model suggests this improvement in sensitivity could be achieved by the DNA coated microcantilever surface changing transcription factor binding properties. Our findings demonstrate a promising new approach for water quality monitoring with fast, highly sensitive, digital readouts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577347">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577347" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577347">
        <p class="paperTitle">DIB-BOT: An open-source hardware approach for high throughput droplet interface bilayer deposition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577347" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577347" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mason, A. F.; Wickham, S. F.; Baker, M. A.</p>
        <p class="info">Score: 6.8, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577347' target='https://doi.org/10.1101/2024.01.26.577347'> 10.1101/2024.01.26.577347</a></p>
        <p class="abstract">Droplet interface bilayers (DIBs) provide a controlled lipid environment for the single-molecule investigation of a range of biologically relevant membrane-bound processes and have garnered attention for their potential applications in bottom-up artificial cells, biosensing, and biophysics. However, the fabrication of DIBs is currently hindered by time-consuming processes and specialized equipment. These fabrication limitations prevent the scale-up of DIB assays, making it difficult to generate the large data sets required to achieve statistically significant conclusions in single-molecule biological assays where heterogeneous behaviour is often observed. This research describes an open-source solution, dubbed &#34;DIB-BOT,&#34; constructed by coupling a nanoinjector with an entry-level 3D printer. We present DIB-BOT as a platform to achieve rapid, reproducible, and reliable fabrication of large numbers of DIBs, addressing the limitations of manual methods. Leveraging commercially available off-the-shelf components, DIB-BOT exhibits high spatial reproducibility, minimal user input, and the ability to scale experiments rapidly. Here we demonstrate the utility of the system by integrating pairwise droplet assembly with a fluorescence plate-reader to execute a biologically relevant assay. When compared with manual DIB fabrication, the DIB-BOT had a tenfold reduction in droplet volume error, a threefold reduction in positional error, and 100% droplet yield. Overall, this method has potential to reduce entry barriers to the use of DIB methods, broadening the applications of DIB research, and generating higher quality data sets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577714">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577714" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577714">
        <p class="paperTitle">graded-CRISPRi, a novel tool for tuning the strengths of CRISPRi-mediated knockdowns in Vibrio natriegens using gRNA libraries</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577714" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577714" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stukenberg, D.; Faber, A.; Becker, A.</p>
        <p class="info">Score: 5.1, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577714' target='https://doi.org/10.1101/2024.01.29.577714'> 10.1101/2024.01.29.577714</a></p>
        <p class="abstract">In recent years, the fast-growing bacterium Vibrio natriegens has gained increasing attention as it has the potential to become a next-generation chassis for Synthetic Biology. A wide range of genetic parts and genome engineering methods have already been developed. However, there is still a need for a well-characterized tool to effectively and gradually reduce the expression level of native genes. To bridge this gap, we created graded-CRISPRi, a system utilizing gRNA variants that lead to varying levels of repression strength. By incorporating multiple gRNA sequences into our design, we successfully extended this concept to simultaneously repress four distinct reporter genes. Furthermore, we demonstrated the capability of using graded-CRISPRi to target native genes, thereby examining the effect of various knockdown levels on growth.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.578080">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.578080" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.578080">
        <p class="paperTitle">The integration of tandem gene repeats via a bacterial type-II toxin-antitoxin-mediated gene amplification (ToxAmp) system and stability visualisation in Saccharomyces cerevisiae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.578080" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.578080" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Evans, S.; Lu, Z.; McDonnell, L.; Anderson, W.; Peralta, F.; Watkins, T.; Ahmed, H.; Luna-Flores, C. H.; Loan, T. D.; Navone, L.; Trau, M.; Scott, C.; Speight, R.; Vickers, C.; Peng, B.</p>
        <p class="info">Score: 2.0, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.578080' target='https://doi.org/10.1101/2024.01.30.578080'> 10.1101/2024.01.30.578080</a></p>
        <p class="abstract">Tandem gene repeats naturally occur as important genomic features and determine many traits in living organisms, like human diseases and microbial productivities of target bioproducts. Here, we develop a bacterial type-II toxin-antitoxin-mediated method to manipulate genomic integration of tandem gene repeats in Saccharomyces cerevisiae and further visualise the evolutionary trajectories of gene repeats. We designed a tri-vector system to introduce toxin-antitoxin-driven gene amplification (ToxAmp) modules, and accidentally re-visited the high-level capacity of multi-fragment co-transformation in S. cerevisiae. This system delivered the multi-copy gene integration in the form of tandem gene repeats spontaneously and independently from toxin-antitoxin-mediated selection. Inducing the toxin (RelE) expressing via a copper (II)-inducible CUP1 promoter successfully drove the in-situ gene amplification of the antitoxin (RelB) module, resulting in [~]40 copies of a green fluorescence reporter (GFP) gene per copy of genome. The copy-number changes, increasing and decreasing, and stable maintenance were visualised using the GFP and blue chromoprotein AeBlue as reporters. Copy-number increasing happened spontaneously not depending on a selection pressure and was quickly enriched through toxin-antitoxin-mediated selection. In summary, the bacterial toxin-antitoxin systems provide a flexible mechanism to manipulate gene copy number in eukaryotic cells and can be exploited for synthetic biology and metabolic engineering applications.

Table of Contents Graphic

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=146 SRC=&#34;FIGDIR/small/578080v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (36K):
org.highwire.dtl.DTLVardef@1411376org.highwire.dtl.DTLVardef@164c036org.highwire.dtl.DTLVardef@1874713org.highwire.dtl.DTLVardef@4b6478_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547496">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547496" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547496">
        <p class="paperTitle">xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547496" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547496" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, B.; Cheng, X.; Li, P.; Gengyang, L.-a.; Gong, J.; Li, S.; Bei, Z.; Tan, X.; Wang, B.; Zeng, X.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.</p>
        <p class="info">Score: 97.6, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547496' target='https://doi.org/10.1101/2023.07.05.547496'> 10.1101/2023.07.05.547496</a></p>
        <p class="abstract">Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently. We propose a unified protein language model, xTrimoPGLM, to address these two types of tasks simultaneously through an innovative pre-training framework. Our key technical contribution is an exploration of the compatibility and the potential for joint optimization of the two types of objectives, which has led to a strategy for training xTrimoPGLM at an unprecedented scale of 100 billion parameters and 1 trillion training tokens. Our extensive experiments reveal that 1) xTrimoPGLM significantly outperforms other advanced baselines in 18 protein understanding benchmarks across four categories. The model also facilitates an atomic-resolution view of protein structures, leading to an advanced 3D structural prediction model that surpasses existing language model-based tools. 2) xTrimoPGLM not only can generate de novo protein sequences following the principles of natural ones, but also can perform programmable generation after supervised fine-tuning (SFT) on curated sequences. These results highlight the substantial capability and versatility of xTrimoPGLM in understanding and generating protein sequences, contributing to the evolving landscape of foundation models in protein science.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.28.577586">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.28.577586" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.28.577586">
        <p class="paperTitle">Sequence-function mapping of proline-rich antimicrobial peptides</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.28.577586" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.28.577586" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Collins, J.; McConnell, A.; Schmitz, Z.; Hackel, B. J.</p>
        <p class="info">Score: 1.2, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.28.577586' target='https://doi.org/10.1101/2024.01.28.577586'> 10.1101/2024.01.28.577586</a></p>
        <p class="abstract">Antimicrobial peptides (AMPs) are essential elements of natural cellular combat and candidates as antibiotic therapy. Elevated function may be needed for robust physiological performance. Yet, both pure protein design and combinatorial library discovery are hindered by the complexity of antimicrobial activity. We applied a recently developed high-throughput technique, sequence-activity mapping of AMPs via depletion (SAMP-Dep), to proline-rich AMPs. Robust self-inhibition was achieved for metalnikowin 1 (Met) and apidaecin 1b (Api). SAMP-Dep exhibited high reproducibility with correlation coefficients 0.90 and 0.92, for Met and Api, respectively, between replicates and 0.99 and 0.96 for synonymous genetic variants. Sequence-activity maps were obtained via characterization of 26,000 and 34,000 mutants of Met and Api, respectively. Both AMPs exhibit similar mutational profiles including beneficial mutations at one terminus, the C-terminus for Met and N-terminus for Api, which is consistent with their opposite binding orientations in the ribosome. While Met and Api reside with the family of proline-rich AMPs, different proline sites exhibit substantially different mutational tolerance. Within the PRP motif, proline mutation eliminates activity, whereas non-PRP prolines readily tolerate mutation. Homologous mutations are more tolerated, particularly at alternating sites on one  face of the peptide. Important and consistent epistasis was observed following the PRP domain within the segment that extends into the ribosomal exit tunnel for both peptides. Variants identified from the SAMP-Dep platform were produced and exposed toward Gram-negative species exogenously, showing either increased potency or specificity for strains tested. In addition to mapping sequence-activity space for fundamental insight and therapeutic engineering, the results advance the robustness of the SAMP-Dep platform for activity characterization.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
