<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Modular RNA motifs for orthogonal phase separated compartments
Authors: Stewart, J. M.; Li, S.; Tang, A.; Klocke, M. A. K. A.; Gobry, M. V.; Fabrini, G.; Di Michele, L.; Rothemund, P. W. K.; Franco, E.
Score: 37.7, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561123
Recent discoveries in biology have highlighted the importance of protein and RNA-based condensates as an alternative to classical membrane-bound organelles for the task of compartmentalizing molecules and biochemical reactions.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Modular RNA motifs for orthogonal phase separated compartments
Authors: Stewart, J. M.; Li, S.; Tang, A.; Klocke, M. A. K. A.; Gobry, M. V.; Fabrini, G.; Di Michele, L.; Rothemund, P. W. K.; Franco, E.
Score: 37.7, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561123
Recent discoveries in biology have highlighted the importance of protein and RNA-based condensates as an alternative to classical membrane-bound organelles for the task of compartmentalizing molecules and biochemical reactions." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-15T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-10-15T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Modular RNA motifs for orthogonal phase separated compartments
Authors: Stewart, J. M.; Li, S.; Tang, A.; Klocke, M. A. K. A.; Gobry, M. V.; Fabrini, G.; Di Michele, L.; Rothemund, P. W. K.; Franco, E.
Score: 37.7, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561123
Recent discoveries in biology have highlighted the importance of protein and RNA-based condensates as an alternative to classical membrane-bound organelles for the task of compartmentalizing molecules and biochemical reactions."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Modular RNA motifs for orthogonal phase separated compartments\nAuthors: Stewart, J. M.; Li, S.; Tang, A.; Klocke, M. A. K. A.; Gobry, M. V.; Fabrini, G.; Di Michele, L.; Rothemund, P. W. K.; Franco, E.\nScore: 37.7, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561123\nRecent discoveries in biology have highlighted the importance of protein and RNA-based condensates as an alternative to classical membrane-bound organelles for the task of compartmentalizing molecules and biochemical reactions.",
  "keywords": [
    
  ],
  "articleBody": " Modular RNA motifs for orthogonal phase separated compartments\nAuthors: Stewart, J. M.; Li, S.; Tang, A.; Klocke, M. A. K. A.; Gobry, M. V.; Fabrini, G.; Di Michele, L.; Rothemund, P. W. K.; Franco, E.\nScore: 37.7, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561123\nRecent discoveries in biology have highlighted the importance of protein and RNA-based condensates as an alternative to classical membrane-bound organelles for the task of compartmentalizing molecules and biochemical reactions. Here, we demonstrate the rational design of pure RNA condensates from star-shaped RNA motifs. We generate condensates using two different RNA nanostar architectures: multi-stranded nanostars whose binding interactions are programmed via single-stranded overhangs, and single-stranded nanostars whose interactions are programmed via kissing loops. Through rational design of the nanostar interaction sequences, we demonstrate that both architectures can produce orthogonal (distinct and immiscible) condensates, which can be individually tracked via fluorogenic aptamers. We also show that aptamers make it possible to recruit peptides and proteins to the condensates with high specificity. Successful cotranscriptional formation of condensates from single-stranded nanostars suggests that they may be genetically encoded and produced in living cells. We provide a library of orthogonal RNA condensates that can be modularly customized and offer a route toward creating systems of functional artificial organelles.\nAn antimicrobial peptide expression platform for targeting pathogenic bacterial species\nAuthors: Rutter, J. W.; Dekker, L.; Clare, C.; Fedorec, A. J.; Barnes, C. P.\nScore: 12.7, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561505\nBacteriocins are antimicrobial peptides that are naturally produced by many bacteria. They hold great potential in the fight against antibiotic resistant bacteria, including ESKAPE pathogens. However, they often have low stability in vivo and therefore, may not be effective when administered orally. Engineered live biotherapeutic products (eLBPs) that secrete bacteriocins can be created to deliver bacteriocins directly at the site of infection. Here we develop a modular bacteriocin secretion platform that can be used to express and secrete multiple bacteriocins from non-pathogenic Escherichia coli host strains. As a proof of concept we create Enterocin A and Enterocin B secreting strains that show strong antimicrobial activity against Enterococcus faecalis and Enterococcus faecium, and characterise this activity in both solid culture and liquid co-culture. We then develop a Lokta-Volterra model that can be used to capture the interactions of these competitor strains. We show that simultaneous exposure to EntA and EntB can increase the level of Enterococcus killing and delay the development of resistance. Our system has the potential to be used as an eLBP to secrete additional bacteriocins for the targeted killing of other pathogenic bacteria.\nBacteriophage-host interactions in microgravity onboard the International Space Station\nAuthors: Huss, P.; Chitboonthavisuk, C.; Meger, A.; Nishikawa, K.; Oates, R. P.; Mills, H.; Holzhaus, O.; Raman, S.\nScore: 12.3, Published: 2023-10-10 DOI: 10.1101/2023.10.10.561409\nBacteriophage-host interactions play a fundamental role in shaping microbial ecosystems. Although phage-host interactions have been extensively studied in terrestrial ecosystems, the impact of microgravity on phage-host interactions is yet to be investigated. Here, we report the dynamics of interactions between T7 bacteriophage and E. coli in microgravity onboard the International Space Station (ISS). We found phage activity was delayed but ultimately successful in microgravity. We identified several de novo mutations in phage and bacteria that improved fitness in microgravity. Deep mutational scanning of the receptor binding domain revealed substantial differences in the number, position, and mutational preferences between gravity and microgravity, reflecting underlying differences in the bacterial adaptations. Combinatorial libraries informed by microgravity selections gave T7 mutants with 100-10,000-fold higher activity on uropathogenic E. coli under terrestrial conditions than wildtype T7. Our findings lay the foundation for future research on the impact of microgravity on phage-host interactions and microbial communities.\nCo-transcriptional production of programmable RNA condensates and synthetic organelles\nAuthors: Fabrini, G.; Nuccio, S. P.; Stewart, J. M.; Lin, S.; Tang, A.; Rothemund, P. W. K.; Franco, E.; Di Antonio, M.; Di Michele, L.\nScore: 12.9, Published: 2023-10-08 DOI: 10.1101/2023.10.06.561174\nCondensation of RNA and proteins is central to cellular functions, and the ability to program it would be valuable in synthetic biology and synthetic cell science. Here we introduce a modular platform for engineering synthetic RNA condensates from tailor-made, branched RNA nanostructures that fold and assemble co-transcriptionally. Up to three orthogonal condensates can form simultaneously and selectively accumulate guest molecules. The RNA condensates can be expressed within synthetic cells to produce membrane-less organelles with controlled number, size, morphology and compositions, and that display the ability to selectively capture proteins. The in situ expression of programmable RNA condensates could underpin spatial organisation of functionalities in both biological and synthetic cells.\nCRISPR-Cas tools for simultaneous transcription \u0026 translation control in bacteria\nAuthors: Cardiff, R.; Faulkner, I.; Beall, J.; Carothers, J. M.; Zalatan, J. G.\nScore: 7.9, Published: 2023-10-12 DOI: 10.1101/2023.10.11.561958\nRobust control over gene translation at arbitrary mRNA targets is an outstanding challenge in microbial synthetic biology. The development of tools that can regulate translation will greatly expand our ability to precisely control genes across the genome. In E. coli, most genes are contained in multi-gene operons, which are subject to polar effects where targeting one gene for repression leads to silencing of both genes. These effects pose a challenge for independently regulating individual genes in multi-gene operons. Here, we use CRISPR-dCas13 to address this challenge. We find that dCas13-mediated repression exhibits up to 6-fold lower polar effects compared to dCas9. We then show that we can selectively activate single genes in a synthetic multi-gene operon by coupling dCas9 transcriptional activation of an operon with dCas13 translational repression of individual genes within the operon. We also show that dCas13 and dCas9 can be multiplexed for improved biosynthesis of a medically-relevant human milk oligosaccharide. Taken together, our findings suggest that combining transcriptional and translational control can access effects that are difficult to achieve with either mode independently. These combined tools for gene regulation will expand our abilities to precisely engineer bacteria for biotechnology and perform systematic genetic screens.\nArrayed in vivo barcoding for multiplexed sequence verification of plasmid DNA and demultiplexing of pooled libraries\nAuthors: Li, W.; Miller, D.; Liu, X.; Tosi, L.; Chkaiban, L.; Mei, H.; Hung, P.-H.; Parekkadan, B.; Sherlock, G.; Levy, S. F.\nScore: 7.2, Published: 2023-10-13 DOI: 10.1101/2023.10.13.562064\nSequence verification of plasmid DNA is critical for many cloning and molecular biology workflows. To leverage high-throughput sequencing, several methods have been developed that add a unique DNA barcode to individual samples prior to pooling and sequencing. However, these methods require an individual plasmid extraction and/or in vitro barcoding reaction for each sample processed, limiting throughput and adding cost. Here, we develop an arrayed in vivo plasmid barcoding platform that enables pooled plasmid extraction and library preparation for Oxford Nanopore sequencing. This method has a high accuracy and recovery rate, and greatly increases throughput and reduces cost relative to other plasmid barcoding methods or Sanger sequencing. We use in vivo barcoding to sequence verify \u003e45,000 plasmids and show that the method can be used to transform error-containing dispersed plasmid pools into sequence-perfect arrays or well-balanced pools. In vivo barcoding does not require any specialized equipment beyond a low-overhead Oxford Nanopore sequencer, enabling most labs to flexibly process hundreds to thousands of plasmids in parallel.\nA green alternative to fragrant agarwood sesquiterpenoid production\nAuthors: Gutierrez, S.; Overmans, S.; Wellman, G. B.; Samaras, V. G.; Oviedo, C.; Gede, M.; Szekely, G.; Lauersen, K. J.\nScore: 9.0, Published: 2023-10-07 DOI: 10.1101/2023.10.06.561217\nCertain endangered Thymelaeaceous trees are major sources of the fragrant and highly valued resinous agarwood, comprised of hundreds of oxygenated sesquiterpenoids (STPs). Despite growing pressure on natural agarwood sources, the chemical complexity of STPs severely limits synthetic production. Here, we catalogued the chemical diversity in 58 agarwood samples by two--dimensional gas chromatography--mass spectrometry and partially recreated complex STP mixtures through synthetic biology. We improved STP yields in the unicellular alga Chlamydomonas reinhardtii by combinatorial engineering to biosynthesise nine macrocyclic STP backbones found in agarwood. A bioprocess following green-chemistry principles was developed that exploits {per thousand}milking{per thousand} of STPs without cell lysis, solvent--solvent STP extraction, solvent--STP nanofiltration, and bulk STP oxy--functionalisation to obtain terpene mixtures like those of agarwood. This process occurs with total solvent recycling and enables continuous production. Our synthetic-biology approach offers a sustainable alternative to harvesting agarwood trees to obtain mixtures of complex, fragrant, oxygenated STPs.\nA robust life-or-death selection platform for enzyme evolution\nAuthors: Jansen, S. C.; Mayer, C.\nScore: 5.2, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561342\nLife-or-death selections evaluate the fitness of individual organisms on a population level. In enzyme engineering, such growth selections allow the rapid and straightforward identification of highly efficient biocatalysts from extensive libraries. However, selection-based improvement of (industrially-relevant) biocatalysts is challenging, as they require highly dependable strategies that artificially link their activities to host survival. Here, we showcase a robust and scalable life-or-death selection platform centered around the complementation of non-canonical amino acid-dependent bacteria. Specifically, we demonstrate how serial passaging of populations featuring millions of carbamoylase variants autonomously selects biocatalysts with up to 90,000-fold higher initial rates. Notably, selection of replicate populations enriched diverse biocatalysts, which feature distinct amino-acid motifs that drastically boost carbamoylase activity. As beneficial substitutions also originated from unintended copying errors during library preparation or cell division, we anticipate that our life-or-death selection platform will be applicable to the continuous, autonomous evolution of diverse biocatalysts in the future.\nFluorescent minihelix assay for translation quantification in high-throughput cell-free systems\nAuthors: Willi, J. A.; Jewett, M. C.\nScore: 3.8, Published: 2023-10-13 DOI: 10.1101/2023.10.13.562088\nHere we report the use of an optimized tetra-cysteine minihelix both as a fusion protein and as a standalone reporter in vitro with the Flash dye to study cell-free protein expression dynamics and engineered ribosome activity. The fluorescent product can be detected and quantified via its characteristic emission spectrum in a standard 96/384-well plate reader, RT-qPCR system, or gel electrophoresis. The fluorescent reporter helix is short enough to be encoded on a primer pair and can be added to a protein of interest via a simple PCR. The thus tagged protein can be detected in real time during or at the end of cell-free protein syntesis reactions, or in gel, without the need for staining. The fluorescent signal is stable and linearly correlates with protein concentration, thus suitable to quantify products of interest. Finally, we demonstrate how this reporter can be used for future efforts in engineering in vitro translation systems.\nProbiotic neoantigen delivery vectors for precision cancer immunotherapy\nAuthors: Redenti, A.; Im, J.; Redenti, B.; Li, F.; Rouanne, M.; Sheng, Z.; Sun, W.; Gurbatri, C. R.; Huang, S.; Komaranchath, M.; Jang, Y.; Hahn, J.; Ballister, E. R.; Vincent, R. L.; Vardoshivilli, A.; Danino, T.; Arpaia, N.\nScore: 19.1, Published: 2023-09-30 DOI: 10.1101/2023.09.29.560228\nMicrobial systems have been synthetically engineered to deploy therapeutic payloads in vivo1-4. With emerging evidence that bacteria naturally home to tumors5-7 and modulate anti-tumor immunity8,9, one promising application is the development of bacterial vectors as precision cancer vaccines10-12. In this study, we engineered probiotic E. coli Nissle 1917 (EcN) as an anti-tumor vaccination platform optimized for enhanced production and cytosolic delivery of neoepitope-containing peptide arrays, with increased susceptibility to blood clearance and phagocytosis. These features enhance both safety and immunogenicity, achieving a system which drives potent and specific T cell-mediated anti-cancer immunity that effectively controls or eliminates tumor growth and extends survival in advanced murine primary and metastatic solid tumors. We demonstrate that the elicited anti-tumor immune response involves extensive priming and activation of neoantigen-specific CD4+ and CD8+ T cells, broader activation of both T and NK cells, and a reduction of tumor-infiltrating immunosuppressive myeloid and regulatory T and B cell populations. Taken together, this work leverages the advantages of living medicines to deliver arrays of tumor-specific neoantigen-derived epitopes within the optimal context to induce specific, effective, and durable systemic anti-tumor immunity.\n",
  "wordCount" : "1944",
  "inLanguage": "en",
  "datePublished": "2023-10-15T10:38:01Z",
  "dateModified": "2023-10-15T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on October 15, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561123">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561123" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561123">
        <p class="paperTitle">Modular RNA motifs for orthogonal phase separated compartments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561123" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561123" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stewart, J. M.; Li, S.; Tang, A.; Klocke, M. A. K. A.; Gobry, M. V.; Fabrini, G.; Di Michele, L.; Rothemund, P. W. K.; Franco, E.</p>
        <p class="info">Score: 37.7, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561123' target='https://doi.org/10.1101/2023.10.06.561123'> 10.1101/2023.10.06.561123</a></p>
        <p class="abstract">Recent discoveries in biology have highlighted the importance of protein and RNA-based condensates as an alternative to classical membrane-bound organelles for the task of compartmentalizing molecules and biochemical reactions. Here, we demonstrate the rational design of pure RNA condensates from star-shaped RNA motifs. We generate condensates using two different RNA nanostar architectures: multi-stranded nanostars whose binding interactions are programmed via single-stranded overhangs, and single-stranded nanostars whose interactions are programmed via kissing loops. Through rational design of the nanostar interaction sequences, we demonstrate that both architectures can produce orthogonal (distinct and immiscible) condensates, which can be individually tracked via fluorogenic aptamers. We also show that aptamers make it possible to recruit peptides and proteins to the condensates with high specificity. Successful cotranscriptional formation of condensates from single-stranded nanostars suggests that they may be genetically encoded and produced in living cells. We provide a library of orthogonal RNA condensates that can be modularly customized and offer a route toward creating systems of functional artificial organelles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561505">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561505" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561505">
        <p class="paperTitle">An antimicrobial peptide expression platform for targeting pathogenic bacterial species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561505" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561505" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rutter, J. W.; Dekker, L.; Clare, C.; Fedorec, A. J.; Barnes, C. P.</p>
        <p class="info">Score: 12.7, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561505' target='https://doi.org/10.1101/2023.10.09.561505'> 10.1101/2023.10.09.561505</a></p>
        <p class="abstract">Bacteriocins are antimicrobial peptides that are naturally produced by many bacteria. They hold great potential in the fight against antibiotic resistant bacteria, including ESKAPE pathogens. However, they often have low stability in vivo and therefore, may not be effective when administered orally. Engineered live biotherapeutic products (eLBPs) that secrete bacteriocins can be created to deliver bacteriocins directly at the site of infection. Here we develop a modular bacteriocin secretion platform that can be used to express and secrete multiple bacteriocins from non-pathogenic Escherichia coli host strains. As a proof of concept we create Enterocin A and Enterocin B secreting strains that show strong antimicrobial activity against Enterococcus faecalis and Enterococcus faecium, and characterise this activity in both solid culture and liquid co-culture. We then develop a Lokta-Volterra model that can be used to capture the interactions of these competitor strains. We show that simultaneous exposure to EntA and EntB can increase the level of Enterococcus killing and delay the development of resistance. Our system has the potential to be used as an eLBP to secrete additional bacteriocins for the targeted killing of other pathogenic bacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561409">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561409" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561409">
        <p class="paperTitle">Bacteriophage-host interactions in microgravity onboard the International Space Station</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561409" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561409" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huss, P.; Chitboonthavisuk, C.; Meger, A.; Nishikawa, K.; Oates, R. P.; Mills, H.; Holzhaus, O.; Raman, S.</p>
        <p class="info">Score: 12.3, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561409' target='https://doi.org/10.1101/2023.10.10.561409'> 10.1101/2023.10.10.561409</a></p>
        <p class="abstract">Bacteriophage-host interactions play a fundamental role in shaping microbial ecosystems. Although phage-host interactions have been extensively studied in terrestrial ecosystems, the impact of microgravity on phage-host interactions is yet to be investigated. Here, we report the dynamics of interactions between T7 bacteriophage and E. coli in microgravity onboard the International Space Station (ISS). We found phage activity was delayed but ultimately successful in microgravity. We identified several de novo mutations in phage and bacteria that improved fitness in microgravity. Deep mutational scanning of the receptor binding domain revealed substantial differences in the number, position, and mutational preferences between gravity and microgravity, reflecting underlying differences in the bacterial adaptations. Combinatorial libraries informed by microgravity selections gave T7 mutants with 100-10,000-fold higher activity on uropathogenic E. coli under terrestrial conditions than wildtype T7. Our findings lay the foundation for future research on the impact of microgravity on phage-host interactions and microbial communities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561174">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561174" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561174">
        <p class="paperTitle">Co-transcriptional production of programmable RNA condensates and synthetic organelles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561174" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561174" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fabrini, G.; Nuccio, S. P.; Stewart, J. M.; Lin, S.; Tang, A.; Rothemund, P. W. K.; Franco, E.; Di Antonio, M.; Di Michele, L.</p>
        <p class="info">Score: 12.9, Published: 2023-10-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561174' target='https://doi.org/10.1101/2023.10.06.561174'> 10.1101/2023.10.06.561174</a></p>
        <p class="abstract">Condensation of RNA and proteins is central to cellular functions, and the ability to program it would be valuable in synthetic biology and synthetic cell science. Here we introduce a modular platform for engineering synthetic RNA condensates from tailor-made, branched RNA nanostructures that fold and assemble co-transcriptionally. Up to three orthogonal condensates can form simultaneously and selectively accumulate guest molecules. The RNA condensates can be expressed within synthetic cells to produce membrane-less organelles with controlled number, size, morphology and compositions, and that display the ability to selectively capture proteins. The in situ expression of programmable RNA condensates could underpin spatial organisation of functionalities in both biological and synthetic cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561958">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561958" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561958">
        <p class="paperTitle">CRISPR-Cas tools for simultaneous transcription &amp; translation control in bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561958" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561958" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cardiff, R.; Faulkner, I.; Beall, J.; Carothers, J. M.; Zalatan, J. G.</p>
        <p class="info">Score: 7.9, Published: 2023-10-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561958' target='https://doi.org/10.1101/2023.10.11.561958'> 10.1101/2023.10.11.561958</a></p>
        <p class="abstract">Robust control over gene translation at arbitrary mRNA targets is an outstanding challenge in microbial synthetic biology. The development of tools that can regulate translation will greatly expand our ability to precisely control genes across the genome. In E. coli, most genes are contained in multi-gene operons, which are subject to polar effects where targeting one gene for repression leads to silencing of both genes. These effects pose a challenge for independently regulating individual genes in multi-gene operons. Here, we use CRISPR-dCas13 to address this challenge. We find that dCas13-mediated repression exhibits up to 6-fold lower polar effects compared to dCas9. We then show that we can selectively activate single genes in a synthetic multi-gene operon by coupling dCas9 transcriptional activation of an operon with dCas13 translational repression of individual genes within the operon. We also show that dCas13 and dCas9 can be multiplexed for improved biosynthesis of a medically-relevant human milk oligosaccharide. Taken together, our findings suggest that combining transcriptional and translational control can access effects that are difficult to achieve with either mode independently. These combined tools for gene regulation will expand our abilities to precisely engineer bacteria for biotechnology and perform systematic genetic screens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562064">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562064" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562064">
        <p class="paperTitle">Arrayed in vivo barcoding for multiplexed sequence verification of plasmid DNA and demultiplexing of pooled libraries</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562064" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562064" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, W.; Miller, D.; Liu, X.; Tosi, L.; Chkaiban, L.; Mei, H.; Hung, P.-H.; Parekkadan, B.; Sherlock, G.; Levy, S. F.</p>
        <p class="info">Score: 7.2, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562064' target='https://doi.org/10.1101/2023.10.13.562064'> 10.1101/2023.10.13.562064</a></p>
        <p class="abstract">Sequence verification of plasmid DNA is critical for many cloning and molecular biology workflows. To leverage high-throughput sequencing, several methods have been developed that add a unique DNA barcode to individual samples prior to pooling and sequencing. However, these methods require an individual plasmid extraction and/or in vitro barcoding reaction for each sample processed, limiting throughput and adding cost. Here, we develop an arrayed in vivo plasmid barcoding platform that enables pooled plasmid extraction and library preparation for Oxford Nanopore sequencing. This method has a high accuracy and recovery rate, and greatly increases throughput and reduces cost relative to other plasmid barcoding methods or Sanger sequencing. We use in vivo barcoding to sequence verify &gt;45,000 plasmids and show that the method can be used to transform error-containing dispersed plasmid pools into sequence-perfect arrays or well-balanced pools. In vivo barcoding does not require any specialized equipment beyond a low-overhead Oxford Nanopore sequencer, enabling most labs to flexibly process hundreds to thousands of plasmids in parallel.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561217">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561217" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561217">
        <p class="paperTitle">A green alternative to fragrant agarwood sesquiterpenoid production</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561217" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561217" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gutierrez, S.; Overmans, S.; Wellman, G. B.; Samaras, V. G.; Oviedo, C.; Gede, M.; Szekely, G.; Lauersen, K. J.</p>
        <p class="info">Score: 9.0, Published: 2023-10-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561217' target='https://doi.org/10.1101/2023.10.06.561217'> 10.1101/2023.10.06.561217</a></p>
        <p class="abstract">Certain endangered Thymelaeaceous trees are major sources of the fragrant and highly valued resinous agarwood, comprised of hundreds of oxygenated sesquiterpenoids (STPs). Despite growing pressure on natural agarwood sources, the chemical complexity of STPs severely limits synthetic production. Here, we catalogued the chemical diversity in 58 agarwood samples by two--dimensional gas chromatography--mass spectrometry and partially recreated complex STP mixtures through synthetic biology. We improved STP yields in the unicellular alga Chlamydomonas reinhardtii by combinatorial engineering to biosynthesise nine macrocyclic STP backbones found in agarwood. A bioprocess following green-chemistry principles was developed that exploits {per thousand}milking{per thousand} of STPs without cell lysis, solvent--solvent STP extraction, solvent--STP nanofiltration, and bulk STP oxy--functionalisation to obtain terpene mixtures like those of agarwood. This process occurs with total solvent recycling and enables continuous production. Our synthetic-biology approach offers a sustainable alternative to harvesting agarwood trees to obtain mixtures of complex, fragrant, oxygenated STPs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561342">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561342" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561342">
        <p class="paperTitle">A robust life-or-death selection platform for enzyme evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561342" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561342" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jansen, S. C.; Mayer, C.</p>
        <p class="info">Score: 5.2, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561342' target='https://doi.org/10.1101/2023.10.09.561342'> 10.1101/2023.10.09.561342</a></p>
        <p class="abstract">Life-or-death selections evaluate the fitness of individual organisms on a population level. In enzyme engineering, such growth selections allow the rapid and straightforward identification of highly efficient biocatalysts from extensive libraries. However, selection-based improvement of (industrially-relevant) biocatalysts is challenging, as they require highly dependable strategies that artificially link their activities to host survival. Here, we showcase a robust and scalable life-or-death selection platform centered around the complementation of non-canonical amino acid-dependent bacteria. Specifically, we demonstrate how serial passaging of populations featuring millions of carbamoylase variants autonomously selects biocatalysts with up to 90,000-fold higher initial rates. Notably, selection of replicate populations enriched diverse biocatalysts, which feature distinct amino-acid motifs that drastically boost carbamoylase activity. As beneficial substitutions also originated from unintended copying errors during library preparation or cell division, we anticipate that our life-or-death selection platform will be applicable to the continuous, autonomous evolution of diverse biocatalysts in the future.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562088">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562088" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562088">
        <p class="paperTitle">Fluorescent minihelix assay for translation quantification in high-throughput cell-free systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562088" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562088" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Willi, J. A.; Jewett, M. C.</p>
        <p class="info">Score: 3.8, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562088' target='https://doi.org/10.1101/2023.10.13.562088'> 10.1101/2023.10.13.562088</a></p>
        <p class="abstract">Here we report the use of an optimized tetra-cysteine minihelix both as a fusion protein and as a standalone reporter in vitro with the Flash dye to study cell-free protein expression dynamics and engineered ribosome activity. The fluorescent product can be detected and quantified via its characteristic emission spectrum in a standard 96/384-well plate reader, RT-qPCR system, or gel electrophoresis. The fluorescent reporter helix is short enough to be encoded on a primer pair and can be added to a protein of interest via a simple PCR. The thus tagged protein can be detected in real time during or at the end of cell-free protein syntesis reactions, or in gel, without the need for staining. The fluorescent signal is stable and linearly correlates with protein concentration, thus suitable to quantify products of interest. Finally, we demonstrate how this reporter can be used for future efforts in engineering in vitro translation systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560228">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560228" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560228">
        <p class="paperTitle">Probiotic neoantigen delivery vectors for precision cancer immunotherapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560228" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560228" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Redenti, A.; Im, J.; Redenti, B.; Li, F.; Rouanne, M.; Sheng, Z.; Sun, W.; Gurbatri, C. R.; Huang, S.; Komaranchath, M.; Jang, Y.; Hahn, J.; Ballister, E. R.; Vincent, R. L.; Vardoshivilli, A.; Danino, T.; Arpaia, N.</p>
        <p class="info">Score: 19.1, Published: 2023-09-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560228' target='https://doi.org/10.1101/2023.09.29.560228'> 10.1101/2023.09.29.560228</a></p>
        <p class="abstract">Microbial systems have been synthetically engineered to deploy therapeutic payloads in vivo1-4. With emerging evidence that bacteria naturally home to tumors5-7 and modulate anti-tumor immunity8,9, one promising application is the development of bacterial vectors as precision cancer vaccines10-12. In this study, we engineered probiotic E. coli Nissle 1917 (EcN) as an anti-tumor vaccination platform optimized for enhanced production and cytosolic delivery of neoepitope-containing peptide arrays, with increased susceptibility to blood clearance and phagocytosis. These features enhance both safety and immunogenicity, achieving a system which drives potent and specific T cell-mediated anti-cancer immunity that effectively controls or eliminates tumor growth and extends survival in advanced murine primary and metastatic solid tumors. We demonstrate that the elicited anti-tumor immune response involves extensive priming and activation of neoantigen-specific CD4&#43; and CD8&#43; T cells, broader activation of both T and NK cells, and a reduction of tumor-infiltrating immunosuppressive myeloid and regulatory T and B cell populations. Taken together, this work leverages the advantages of living medicines to deliver arrays of tumor-specific neoantigen-derived epitopes within the optimal context to induce specific, effective, and durable systemic anti-tumor immunity.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
