<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="IMPROVING ANTIBODY AFFINITY USING LABORATORYDATA WITH LANGUAGE MODEL GUIDED DESIGN
Authors: Krause, B.; Subramanian, S.; Yuan, T.; Yang, M.; Sato, A.; Naik, N.
Score: 17.4, Published: 2023-09-13 DOI: 10.1101/2023.09.13.557505
AO_SCPLOWBSTRACTC_SCPLOWProtein design requires searching exponentially large spaces of possible sequences to find candidates with desired properties. Language models (LMs) pretrained on universal protein datasets have shown potential to help make this search space tractable. However, LMs trained on natural sequences alone have limitations in designing proteins with novel functions, which is especially important for many pharmaceutical applications.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="IMPROVING ANTIBODY AFFINITY USING LABORATORYDATA WITH LANGUAGE MODEL GUIDED DESIGN
Authors: Krause, B.; Subramanian, S.; Yuan, T.; Yang, M.; Sato, A.; Naik, N.
Score: 17.4, Published: 2023-09-13 DOI: 10.1101/2023.09.13.557505
AO_SCPLOWBSTRACTC_SCPLOWProtein design requires searching exponentially large spaces of possible sequences to find candidates with desired properties. Language models (LMs) pretrained on universal protein datasets have shown potential to help make this search space tractable. However, LMs trained on natural sequences alone have limitations in designing proteins with novel functions, which is especially important for many pharmaceutical applications." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-17T10:36:50+00:00" />
<meta property="article:modified_time" content="2023-09-17T10:36:50+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="IMPROVING ANTIBODY AFFINITY USING LABORATORYDATA WITH LANGUAGE MODEL GUIDED DESIGN
Authors: Krause, B.; Subramanian, S.; Yuan, T.; Yang, M.; Sato, A.; Naik, N.
Score: 17.4, Published: 2023-09-13 DOI: 10.1101/2023.09.13.557505
AO_SCPLOWBSTRACTC_SCPLOWProtein design requires searching exponentially large spaces of possible sequences to find candidates with desired properties. Language models (LMs) pretrained on universal protein datasets have shown potential to help make this search space tractable. However, LMs trained on natural sequences alone have limitations in designing proteins with novel functions, which is especially important for many pharmaceutical applications."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "IMPROVING ANTIBODY AFFINITY USING LABORATORYDATA WITH LANGUAGE MODEL GUIDED DESIGN\nAuthors: Krause, B.; Subramanian, S.; Yuan, T.; Yang, M.; Sato, A.; Naik, N.\nScore: 17.4, Published: 2023-09-13 DOI: 10.1101/2023.09.13.557505\nAO_SCPLOWBSTRACTC_SCPLOWProtein design requires searching exponentially large spaces of possible sequences to find candidates with desired properties. Language models (LMs) pretrained on universal protein datasets have shown potential to help make this search space tractable. However, LMs trained on natural sequences alone have limitations in designing proteins with novel functions, which is especially important for many pharmaceutical applications.",
  "keywords": [
    
  ],
  "articleBody": " IMPROVING ANTIBODY AFFINITY USING LABORATORYDATA WITH LANGUAGE MODEL GUIDED DESIGN\nAuthors: Krause, B.; Subramanian, S.; Yuan, T.; Yang, M.; Sato, A.; Naik, N.\nScore: 17.4, Published: 2023-09-13 DOI: 10.1101/2023.09.13.557505\nAO_SCPLOWBSTRACTC_SCPLOWProtein design requires searching exponentially large spaces of possible sequences to find candidates with desired properties. Language models (LMs) pretrained on universal protein datasets have shown potential to help make this search space tractable. However, LMs trained on natural sequences alone have limitations in designing proteins with novel functions, which is especially important for many pharmaceutical applications. In this work, we used a combination of methods to finetune pretrained LMs on laboratory data collected in an anti-CD40L antibody library campaign to develop an ensemble scoring function to model the fitness landscape and guide the design of new antibodies. Laboratory testing showed that the designed antibodies had improved affinity to CD40L. Notably, the designs improved the affinities of four antibodies, originally ranging from 1 nanomolar to 100 picomolar, all to below 25 picomolar, approaching the limit of detection. This work is a promising step towards realizing the potential of LMs to leverage laboratory data to develop improved treatments for diseases.\nTargeted Transcriptional Activation using a CRISPR-Associated Transposon System\nAuthors: Garza Elizondo, A. M.; Chappell, J.\nScore: 13.5, Published: 2023-09-12 DOI: 10.1101/2023.09.12.557412\nSynthetic perturbation of gene expression is central to our ability to reliably uncover genotype-phenotype relationships in microbes. Here, we present a novel gene transcription activation strategy that uses the Vibrio cholerae CRISPR-Associated Transposon (CAST) system to selectively insert promoter elements upstream of genes of interest. Through this strategy, we show robust activation of both recombinant and endogenous genes across the E. coli chromosome. We then demonstrate precise tuning of expression levels by exchanging the promoter elements being inserted. Finally, we demonstrate that CAST activation can be used to synthetically induce ampicillin-resistant phenotypes in E. coli.\nEngineering synthetic phosphorylation signaling networks in human cells\nAuthors: Yang, X.; Rocks, J. W.; Jiang, K.; Walters, A. J.; Rai, K.; Liu, J.; Nguyen, J.; Olson, S. D.; Mehta, P.; Collins, J. J.; Daringer, N. M.; Bashor, C. J.\nScore: 9.2, Published: 2023-09-11 DOI: 10.1101/2023.09.11.557100\nProtein phosphorylation signaling networks play a central role in how cells sense and respond to their environment. Here, we describe the engineering of artificial phosphorylation networks in which \"push-pull\" motifs--reversible enzymatic phosphorylation cycles consisting of opposing kinase and phosphatase activities--are assembled from modular protein domain parts and then wired together to create synthetic phosphorylation circuits in human cells. We demonstrate that the composability of our design scheme permits model-guided tuning of circuit function and the ability to make diverse network connections; synthetic phosphorylation circuits can be coupled to upstream cell surface receptors to enable fast-timescale sensing of extracellular ligands, while downstream connections can regulate gene expression. We leverage these capabilities to program cells to act as controllers that dynamically regulate immune cell cytokine production. Our work introduces a generalizable approach for designing and budling phosphorylation signaling circuits that enable user-defined sense-and-respond function for diverse biosensing and therapeutic applications.\nPrecise manipulation of site and stoichiometry of capsid modification enables optimization of functional adeno-associated virus conjugates\nAuthors: Erickson, S. B.; Pham, Q.; Cao, X.; Glicksman, J.; Kelemen, R. E.; Shahraeini, S. S.; Bodkin, S.; Kiyam, Z.; Chatterjee, A.\nScore: 5.8, Published: 2023-09-07 DOI: 10.1101/2023.09.07.556719\nThe ability to engineer adeno-associated virus (AAV) vectors for targeted infection of specific cell types is critically important to fully harness its potential of human gene therapy. A promising approach to achieve this objective involves chemically attaching retargeting ligands onto the virus capsid. Site-specific incorporation of a bioorthogonal noncanonical amino acid (ncAA) into the AAV capsid proteins provides a particularly attractive strategy to introduce such modifications with exquisite precision. In this study, we show that using ncAA mutagenesis, it is possible to systematically alter the attachment site of a retargeting ligand (cyclic-RGD) on the AAV capsid to create diverse conjugate architectures, and that the site of attachment heavily impacts the retargeting efficiency. We further demonstrate that the performance of these AAV conjugates is highly sensitive to the stoichiometry of capsid labeling (labels per capsid), with an intermediate labeling density ([~]12 per capsid) providing optimal activity. Finally, we developed technology to precisely control the number of attachment sites per AAV capsid, by selectively incorporating a ncAA into the minor capsid proteins with high fidelity and efficiency, such that AAV-conjugates with varying stoichiometry can be synthesized in a homogeneous manner. Together, this platform provides unparalleled control over site and stoichiometry of capsid modification, which will enable the development of next-generation AAV vectors tailored with desirable attributes.\nA cell-free transcription-translation pipeline for recreating methylation patterns boosts DNA transformation in bacteria\nAuthors: Vento, J. M.; Durmusoglu, D.; Li, T.; Sullivan, S. M.; Ttofali, F.; van Schaik, J.; Yu, Y.; Barquist, L.; Crook, N.; Beisel, C. L.\nScore: 2.1, Published: 2023-09-16 DOI: 10.1101/2023.09.16.557782\nThe bacterial world offers diverse strains for understanding medical and environmental processes and for engineering synthetic-biology chasses. However, genetically manipulating these strains has faced a long-standing bottleneck: how to efficiently transform DNA. Here we report IMPRINT, a generalized, rapid and scalable approach based on cell-free transcription-translation (TXTL) systems to overcome DNA restriction, a prominent barrier to transformation. IMPRINT utilizes TXTL to express DNA methyltransferases from the bacterial host's restriction-modification systems. The expressed methyltransferases then methylate DNA in vitro to match the host DNA's methylation pattern, circumventing restriction and enhancing transformation. With IMPRINT, we efficiently multiplex methylation by diverse DNA methyltransferases and enhance plasmid transformation in gram-negative and gram-positive bacteria. We also developed a high-throughput pipeline that identifies the most consequential methyltransferases, and we apply IMPRINT to facilitate a library screen for translational rules in a hard-to-transform Bifidobacterium. Overall, IMPRINT can enhance DNA transformation, enabling use of increasingly sophisticated genetic manipulation tools across the bacterial world.\nManufacturing DNA in E. coli yields higher fidelity DNA than in vitro enzymatic synthesis\nAuthors: Hersch, S. J.; Chandrasekaran, S.; Lam, J.; Nafissi, N.; Slavcev, R.\nScore: 1.5, Published: 2023-09-12 DOI: 10.1101/2023.09.12.557453\nThe rise of biotechnologies such as gene therapy have brought DNA vectors to the forefront of pharmaceutical development. The quality of the genetic starting material plays a pivotal role in determining the quality of the final product. In this study we examined the fidelity of DNA replication using enzymatic methods (in vitro) compared to plasmid DNA produced in vivo in E. coli. Next-generation sequencing approaches predominantly rely on in vitro polymerases, which have inherent limitations in sensitivity. To address this challenge, we introduce a novel assay based on loss-of-function (LOF) mutations in the conditionally toxic sacB gene. Our findings show that DNA production in E. coli results in significantly fewer LOF mutations (approximately 80-to 3000-fold less) compared to various enzymatic DNA synthesis methods. This includes the most accurate PCR polymerase (Q5) and a commonly employed rolling circle amplification (RCA) DNA polymerase (Phi29). These results suggest that using low-fidelity starting material DNA synthesized in vitro by PCR or RCA may introduce a substantial number of impurities, potentially affecting the quality and yield of final pharmaceutical products. In summary, our study underscores that DNA synthesized in vitro has a significantly higher mutation rate than DNA produced traditionally in E. coli. Therefore, utilizing in vitro enzymatically-produced DNA in biotechnology and biomanufacturing may entail considerable fidelity-related risks, while DNA starting material derived from E. coli substantially mitigates this risk, enhancing overall quality in the production processes.\nEngineering the ADDobody protein scaffold for generation of high-avidity ADDomer super-binders\nAuthors: Buzas, D.; Sun, H.; Toelzer, C.; Yadav, S. K. N.; Borucu, U.; Gautam, G.; Gupta, K.; Bufton, J.; Capin, J.; Sessions, R. B.; Garzoni, F.; Berger, I.; Schaffitzel, C.\nScore: 1.2, Published: 2023-09-09 DOI: 10.1101/2023.09.09.556966\nAdenovirus-derived dodecamer (ADDomer) nanoparticles comprise 60 copies of Adenovirus penton base protein (PBP). ADDomer is thermostable, rendering the storage, transport and deployment of ADDomer-based therapeutics independent of a cold-chain. To expand the scope of ADDomer nanoparticles for new applications, we engineered ADDobodies. ADDobodies represent the crown domain of the PBP, genetically separated from its multimerization domain. We inserted heterologous sequences into hyper-variable loops in the crown domain. The resulting ADDobodies were expressed at high yields in Escherichia coli, are monomeric and maintain thermostability. We solved the X-ray structure of an ADDobody prototype validating our design. We demonstrated that ADDobodies can be used to select a specific binder against a target in in vitro selection experiments using ribosome display, with an enrichment factor of [~]104-fold in one selection round. We show that ADDobodies can be converted back into ADDomers by genetically reconnecting the selected ADDobody with the PBP multimerization domain from a different species, giving rise to a multivalent nanoparticle, called Chimera, confirmed by a 2.2 [A] structure determined by cryogenic electron microscopy (cryo-EM). Chimera comprises 60 binding sites, resulting in ultra-high, picomolar avidity to the target.\n",
  "wordCount" : "1451",
  "inLanguage": "en",
  "datePublished": "2023-09-17T10:36:50Z",
  "dateModified": "2023-09-17T10:36:50Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on September 17, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557505">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557505" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557505">
        <p class="paperTitle">IMPROVING ANTIBODY AFFINITY USING LABORATORYDATA WITH LANGUAGE MODEL GUIDED DESIGN</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557505" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557505" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krause, B.; Subramanian, S.; Yuan, T.; Yang, M.; Sato, A.; Naik, N.</p>
        <p class="info">Score: 17.4, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557505' target='https://doi.org/10.1101/2023.09.13.557505'> 10.1101/2023.09.13.557505</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWProtein design requires searching exponentially large spaces of possible sequences to find candidates with desired properties. Language models (LMs) pretrained on universal protein datasets have shown potential to help make this search space tractable. However, LMs trained on natural sequences alone have limitations in designing proteins with novel functions, which is especially important for many pharmaceutical applications. In this work, we used a combination of methods to finetune pretrained LMs on laboratory data collected in an anti-CD40L antibody library campaign to develop an ensemble scoring function to model the fitness landscape and guide the design of new antibodies. Laboratory testing showed that the designed antibodies had improved affinity to CD40L. Notably, the designs improved the affinities of four antibodies, originally ranging from 1 nanomolar to 100 picomolar, all to below 25 picomolar, approaching the limit of detection. This work is a promising step towards realizing the potential of LMs to leverage laboratory data to develop improved treatments for diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557412">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557412" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557412">
        <p class="paperTitle">Targeted Transcriptional Activation using a CRISPR-Associated Transposon System</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557412" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557412" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garza Elizondo, A. M.; Chappell, J.</p>
        <p class="info">Score: 13.5, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557412' target='https://doi.org/10.1101/2023.09.12.557412'> 10.1101/2023.09.12.557412</a></p>
        <p class="abstract">Synthetic perturbation of gene expression is central to our ability to reliably uncover genotype-phenotype relationships in microbes. Here, we present a novel gene transcription activation strategy that uses the Vibrio cholerae CRISPR-Associated Transposon (CAST) system to selectively insert promoter elements upstream of genes of interest. Through this strategy, we show robust activation of both recombinant and endogenous genes across the E. coli chromosome. We then demonstrate precise tuning of expression levels by exchanging the promoter elements being inserted. Finally, we demonstrate that CAST activation can be used to synthetically induce ampicillin-resistant phenotypes in E. coli.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557100">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557100" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557100">
        <p class="paperTitle">Engineering synthetic phosphorylation signaling networks in human cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557100" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557100" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, X.; Rocks, J. W.; Jiang, K.; Walters, A. J.; Rai, K.; Liu, J.; Nguyen, J.; Olson, S. D.; Mehta, P.; Collins, J. J.; Daringer, N. M.; Bashor, C. J.</p>
        <p class="info">Score: 9.2, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557100' target='https://doi.org/10.1101/2023.09.11.557100'> 10.1101/2023.09.11.557100</a></p>
        <p class="abstract">Protein phosphorylation signaling networks play a central role in how cells sense and respond to their environment. Here, we describe the engineering of artificial phosphorylation networks in which &#34;push-pull&#34; motifs--reversible enzymatic phosphorylation cycles consisting of opposing kinase and phosphatase activities--are assembled from modular protein domain parts and then wired together to create synthetic phosphorylation circuits in human cells. We demonstrate that the composability of our design scheme permits model-guided tuning of circuit function and the ability to make diverse network connections; synthetic phosphorylation circuits can be coupled to upstream cell surface receptors to enable fast-timescale sensing of extracellular ligands, while downstream connections can regulate gene expression. We leverage these capabilities to program cells to act as controllers that dynamically regulate immune cell cytokine production. Our work introduces a generalizable approach for designing and budling phosphorylation signaling circuits that enable user-defined sense-and-respond function for diverse biosensing and therapeutic applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556719">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556719" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556719">
        <p class="paperTitle">Precise manipulation of site and stoichiometry of capsid modification enables optimization of functional adeno-associated virus conjugates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556719" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556719" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Erickson, S. B.; Pham, Q.; Cao, X.; Glicksman, J.; Kelemen, R. E.; Shahraeini, S. S.; Bodkin, S.; Kiyam, Z.; Chatterjee, A.</p>
        <p class="info">Score: 5.8, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556719' target='https://doi.org/10.1101/2023.09.07.556719'> 10.1101/2023.09.07.556719</a></p>
        <p class="abstract">The ability to engineer adeno-associated virus (AAV) vectors for targeted infection of specific cell types is critically important to fully harness its potential of human gene therapy. A promising approach to achieve this objective involves chemically attaching retargeting ligands onto the virus capsid. Site-specific incorporation of a bioorthogonal noncanonical amino acid (ncAA) into the AAV capsid proteins provides a particularly attractive strategy to introduce such modifications with exquisite precision. In this study, we show that using ncAA mutagenesis, it is possible to systematically alter the attachment site of a retargeting ligand (cyclic-RGD) on the AAV capsid to create diverse conjugate architectures, and that the site of attachment heavily impacts the retargeting efficiency. We further demonstrate that the performance of these AAV conjugates is highly sensitive to the stoichiometry of capsid labeling (labels per capsid), with an intermediate labeling density ([~]12 per capsid) providing optimal activity. Finally, we developed technology to precisely control the number of attachment sites per AAV capsid, by selectively incorporating a ncAA into the minor capsid proteins with high fidelity and efficiency, such that AAV-conjugates with varying stoichiometry can be synthesized in a homogeneous manner. Together, this platform provides unparalleled control over site and stoichiometry of capsid modification, which will enable the development of next-generation AAV vectors tailored with desirable attributes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.16.557782">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.16.557782" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.16.557782">
        <p class="paperTitle">A cell-free transcription-translation pipeline for recreating methylation patterns boosts DNA transformation in bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.16.557782" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.16.557782" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vento, J. M.; Durmusoglu, D.; Li, T.; Sullivan, S. M.; Ttofali, F.; van Schaik, J.; Yu, Y.; Barquist, L.; Crook, N.; Beisel, C. L.</p>
        <p class="info">Score: 2.1, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.16.557782' target='https://doi.org/10.1101/2023.09.16.557782'> 10.1101/2023.09.16.557782</a></p>
        <p class="abstract">The bacterial world offers diverse strains for understanding medical and environmental processes and for engineering synthetic-biology chasses. However, genetically manipulating these strains has faced a long-standing bottleneck: how to efficiently transform DNA. Here we report IMPRINT, a generalized, rapid and scalable approach based on cell-free transcription-translation (TXTL) systems to overcome DNA restriction, a prominent barrier to transformation. IMPRINT utilizes TXTL to express DNA methyltransferases from the bacterial host&#39;s restriction-modification systems. The expressed methyltransferases then methylate DNA in vitro to match the host DNA&#39;s methylation pattern, circumventing restriction and enhancing transformation. With IMPRINT, we efficiently multiplex methylation by diverse DNA methyltransferases and enhance plasmid transformation in gram-negative and gram-positive bacteria. We also developed a high-throughput pipeline that identifies the most consequential methyltransferases, and we apply IMPRINT to facilitate a library screen for translational rules in a hard-to-transform Bifidobacterium. Overall, IMPRINT can enhance DNA transformation, enabling use of increasingly sophisticated genetic manipulation tools across the bacterial world.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557453">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557453" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557453">
        <p class="paperTitle">Manufacturing DNA in E. coli yields higher fidelity DNA than in vitro enzymatic synthesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557453" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557453" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hersch, S. J.; Chandrasekaran, S.; Lam, J.; Nafissi, N.; Slavcev, R.</p>
        <p class="info">Score: 1.5, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557453' target='https://doi.org/10.1101/2023.09.12.557453'> 10.1101/2023.09.12.557453</a></p>
        <p class="abstract">The rise of biotechnologies such as gene therapy have brought DNA vectors to the forefront of pharmaceutical development. The quality of the genetic starting material plays a pivotal role in determining the quality of the final product. In this study we examined the fidelity of DNA replication using enzymatic methods (in vitro) compared to plasmid DNA produced in vivo in E. coli. Next-generation sequencing approaches predominantly rely on in vitro polymerases, which have inherent limitations in sensitivity. To address this challenge, we introduce a novel assay based on loss-of-function (LOF) mutations in the conditionally toxic sacB gene. Our findings show that DNA production in E. coli results in significantly fewer LOF mutations (approximately 80-to 3000-fold less) compared to various enzymatic DNA synthesis methods. This includes the most accurate PCR polymerase (Q5) and a commonly employed rolling circle amplification (RCA) DNA polymerase (Phi29). These results suggest that using low-fidelity starting material DNA synthesized in vitro by PCR or RCA may introduce a substantial number of impurities, potentially affecting the quality and yield of final pharmaceutical products. In summary, our study underscores that DNA synthesized in vitro has a significantly higher mutation rate than DNA produced traditionally in E. coli. Therefore, utilizing in vitro enzymatically-produced DNA in biotechnology and biomanufacturing may entail considerable fidelity-related risks, while DNA starting material derived from E. coli substantially mitigates this risk, enhancing overall quality in the production processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.556966">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.556966" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.556966">
        <p class="paperTitle">Engineering the ADDobody protein scaffold for generation of high-avidity ADDomer super-binders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.556966" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.556966" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Buzas, D.; Sun, H.; Toelzer, C.; Yadav, S. K. N.; Borucu, U.; Gautam, G.; Gupta, K.; Bufton, J.; Capin, J.; Sessions, R. B.; Garzoni, F.; Berger, I.; Schaffitzel, C.</p>
        <p class="info">Score: 1.2, Published: 2023-09-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.556966' target='https://doi.org/10.1101/2023.09.09.556966'> 10.1101/2023.09.09.556966</a></p>
        <p class="abstract">Adenovirus-derived dodecamer (ADDomer) nanoparticles comprise 60 copies of Adenovirus penton base protein (PBP). ADDomer is thermostable, rendering the storage, transport and deployment of ADDomer-based therapeutics independent of a cold-chain. To expand the scope of ADDomer nanoparticles for new applications, we engineered ADDobodies. ADDobodies represent the crown domain of the PBP, genetically separated from its multimerization domain. We inserted heterologous sequences into hyper-variable loops in the crown domain. The resulting ADDobodies were expressed at high yields in Escherichia coli, are monomeric and maintain thermostability. We solved the X-ray structure of an ADDobody prototype validating our design. We demonstrated that ADDobodies can be used to select a specific binder against a target in in vitro selection experiments using ribosome display, with an enrichment factor of [~]104-fold in one selection round. We show that ADDobodies can be converted back into ADDomers by genetically reconnecting the selected ADDobody with the PBP multimerization domain from a different species, giving rise to a multivalent nanoparticle, called Chimera, confirmed by a 2.2 [A] structure determined by cryogenic electron microscopy (cryo-EM). Chimera comprises 60 binding sites, resulting in ultra-high, picomolar avidity to the target.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
