<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="In vitro validated antibody design against multiple therapeutic antigens using generative inverse folding
Authors: Shanehsazzadeh, A.; Alverio, J.; Kasun, G.; Levine, S.; Khan, J. A.; Chung, C.; Diaz, N.; Luton, B. K.; Tarter, Y.; McCloskey, C.; Bateman, K. B.; Carter, H.; Chapman, D.; Consbruck, R.; Jaeger, A.; Kohnert, C.; Kopec-Belliveau, G.; Sutton, J. M.; Guo, Z.; Canales, G.; Ejan, K.; Marsh, E.; Ruelos, A.; Ripley, R.; Stoddard, B.; Caguiat, R.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="In vitro validated antibody design against multiple therapeutic antigens using generative inverse folding
Authors: Shanehsazzadeh, A.; Alverio, J.; Kasun, G.; Levine, S.; Khan, J. A.; Chung, C.; Diaz, N.; Luton, B. K.; Tarter, Y.; McCloskey, C.; Bateman, K. B.; Carter, H.; Chapman, D.; Consbruck, R.; Jaeger, A.; Kohnert, C.; Kopec-Belliveau, G.; Sutton, J. M.; Guo, Z.; Canales, G.; Ejan, K.; Marsh, E.; Ruelos, A.; Ripley, R.; Stoddard, B.; Caguiat, R." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-20T10:36:27+00:00" />
<meta property="article:modified_time" content="2023-12-20T10:36:27+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="In vitro validated antibody design against multiple therapeutic antigens using generative inverse folding
Authors: Shanehsazzadeh, A.; Alverio, J.; Kasun, G.; Levine, S.; Khan, J. A.; Chung, C.; Diaz, N.; Luton, B. K.; Tarter, Y.; McCloskey, C.; Bateman, K. B.; Carter, H.; Chapman, D.; Consbruck, R.; Jaeger, A.; Kohnert, C.; Kopec-Belliveau, G.; Sutton, J. M.; Guo, Z.; Canales, G.; Ejan, K.; Marsh, E.; Ruelos, A.; Ripley, R.; Stoddard, B.; Caguiat, R."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "In vitro validated antibody design against multiple therapeutic antigens using generative inverse folding\nAuthors: Shanehsazzadeh, A.; Alverio, J.; Kasun, G.; Levine, S.; Khan, J. A.; Chung, C.; Diaz, N.; Luton, B. K.; Tarter, Y.; McCloskey, C.; Bateman, K. B.; Carter, H.; Chapman, D.; Consbruck, R.; Jaeger, A.; Kohnert, C.; Kopec-Belliveau, G.; Sutton, J. M.; Guo, Z.; Canales, G.; Ejan, K.; Marsh, E.; Ruelos, A.; Ripley, R.; Stoddard, B.; Caguiat, R.",
  "keywords": [
    
  ],
  "articleBody": " In vitro validated antibody design against multiple therapeutic antigens using generative inverse folding\nAuthors: Shanehsazzadeh, A.; Alverio, J.; Kasun, G.; Levine, S.; Khan, J. A.; Chung, C.; Diaz, N.; Luton, B. K.; Tarter, Y.; McCloskey, C.; Bateman, K. B.; Carter, H.; Chapman, D.; Consbruck, R.; Jaeger, A.; Kohnert, C.; Kopec-Belliveau, G.; Sutton, J. M.; Guo, Z.; Canales, G.; Ejan, K.; Marsh, E.; Ruelos, A.; Ripley, R.; Stoddard, B.; Caguiat, R.; Chapman, K.; Saunders, M.; Sharp, J.; Ganini da Silva, D.; Feltner, A.; Ripley, J.; Bryant, M. E.; Castillo, D.; Meier, J.; Stegmann, C. M.; Moran, K.; Lemke, C.; Abdulhaqq, S.; Klug, L. R.; Bachas, S.\nScore: 32.1, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570889\nDeep learning approaches have demonstrated the ability to design protein sequences given backbone structures [1, 2, 3, 4, 5]. While these approaches have been applied in silico to designing antibody complementarity-determining regions (CDRs), they have yet to be validated in vitro for designing antibody binders, which is the true measure of success for antibody design. Here we describe IgDesign, a deep learning method for antibody CDR design, and demonstrate its robustness with successful binder design for 8 therapeutic antigens. The model is tasked with designing heavy chain CDR3 (HCDR3) or all three heavy chain CDRs (HCDR123) using native backbone structures of antibody-antigen complexes, along with the antigen and antibody framework (FWR) sequences as context. For each of the 8 antigens, we design 100 HCDR3s and 100 HCDR123s, scaffold them into the native antibodys variable region, and screen them for binding against the antigen using surface plasmon resonance (SPR). As a baseline, we screen 100 HCDR3s taken from the models training set and paired with the native HCDR1 and HCDR2. We observe that both HCDR3 design and HCDR123 design outperform this HCDR3-only baseline. IgDesign is the first experimentally validated antibody inverse folding model. It can design antibody binders to multiple therapeutic antigens with high success rates and, in some cases, improved affinities over clinically validated reference antibodies. Antibody inverse folding has applications to both de novo antibody design and lead optimization, making IgDesign a valuable tool for accelerating drug development and enabling therapeutic design.\nA long-acting prolactin to combat lactation insufficiency\nAuthors: Kready, K.; Doiron, K.; Redfield Chan, K.; Way, J. C.; Justman, Q.; Powe, C. E.; Silver, P. A.\nScore: 26.2, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571886\nHuman infants are born to breastfeed. While 50% of lactating persons struggle to make enough milk, there are no governmentally-approved drugs to enhance lactation1. Here, we engineer a variant of the naturally-occurring driver of lactation, the hormone Prolactin, to increase its serum half-life and produce a viable drug candidate. Our engineered variant, Prolactin-eXtra Long-acting (Prolactin-XL), is comprised of endogenously active human prolactin fused to an engineered human IgG Fc domain designed to overcome the unique drug development challenges specific to the lactating person-infant dyad. Our Prolactin-XL has a serum half-life of 70.9h in mice, 2,625-fold longer than endogenously active prolactin alone (70.9h v. 0.027h). We demonstrate that Prolactin-XL increases milk production and restores growth of pups fed by dams with pharmacologically-ablated lactation. We show that Prolactin-XL-enhanced lactation is accompanied by reversible, lactocyte-driven changes in mammary gland morphology. This work establishes long-acting prolactins as a potentially powerful pharmacologic means to combat insufficient lactation.\nBuilding synthetic biosensors using red blood cell proteins\nAuthors: Dolberg, T. B.; Gunnels, T. F.; Ling, T.; Sarnese, K. A.; Crispino, J. D.; Leonard, J. N.\nScore: 17.8, Published: 2023-12-16 DOI: 10.1101/2023.12.16.571988\nAs the use of engineered cell therapies expands from pioneering efforts in cancer immunotherapy to other applications, an attractive but less explored approach is the use of engineered red blood cells (RBCs). Compared to other cells, RBCs have a very long circulation time and reside in the blood compartment, so they could be ideally suited for applications as sentinel cells that enable in situ sensing and diagnostics. However, we largely lack tools for converting RBCs into biosensors. A unique challenge is that RBCs remodel their membranes during maturation, shedding many membrane components, suggesting that an RBC-specific approach may be needed. Towards addressing this need, here we develop a biosensing architecture built on RBC membrane proteins that are retained through erythropoiesis. This biosensor employs a mechanism in which extracellular ligand binding is transduced into intracellular reconstitution of a split output protein (including either a fluorophore or an enzyme). By comparatively evaluating a range of biosensor architectures, linker types, scaffold choices, and output signals, we identify biosensor designs and design features that confer substantial ligand-induced signal in vitro. Finally, we demonstrate that erythroid precursor cells engineered with our RBC protein biosensors function in vivo. This study establishes a foundation for developing RBC-based biosensors that could ultimately address unmet needs including non-invasive monitoring of physiological signals for a range of diagnostic applications.\nA translation-independent directed evolution strategy to engineer aminoacyl-tRNA synthetases\nAuthors: Soni, C.; Prywes, N.; Hall, M.; Savage, D. F.; Schepartz, A.; Chatterjee, A.\nScore: 16.0, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571473\nUsing directed evolution, engineered aminoacyl-tRNA synthetases (aaRS) have been developed that enable co-translational incorporation of numerous noncanonical amino acids (ncAAs) into proteins in living cells. Until now, the selection of such novel aaRS mutants has relied on coupling their activity to the expression of a reporter protein with a selectable phenotype. However, such translation-dependent selection schemes are incompatible with exotic monomers that diverge structurally from canonical -amino acids and are suboptimal substrates for the ribosome. To enable the ribosomal incorporation of such exotic monomers, a two-step solution is needed: A) Engineering an aaRS to acylate its cognate tRNA with the exotic monomer, without relying on ribosomal translation as a readout, and B) Subsequent engineering of the ribosome to accept the resulting acylated tRNA for translation. Here, we report a platform for aaRS engineering that directly selects for tRNA-acylation without ribosomal translation (START). In START, each distinct aaRS mutant is correlated to a cognate tRNA containing a unique sequence barcode. Acylation by an active aaRS mutant protects the associated barcode-containing tRNAs from an oxidative treatment designed to damage the 3'-terminus of the uncharged tRNAs. Sequencing of these surviving barcode-containing tRNAs is then used to reveal the identity of aaRS mutants that acylated the correlated tRNA sequences. The efficacy of START was demonstrated by identifying novel mutants of the M. alvus pyrrolysyl-tRNA synthetase from a naive library that charge noncanonical amino acids.\nProteinGym: Large-Scale Benchmarks for Protein Design and Fitness Prediction\nAuthors: Notin, P.; Kollasch, A. W.; Ritter, D.; van Niekerk, L.; Paul, S.; Spinner, H.; Rollins, N.; Shaw, A.; Weitzman, R.; Frazer, J.; Dias, M.; Franceschi, D.; Orenbuch, R.; Gal, Y.; Marks, D. S.\nScore: 31.4, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570727\nPredicting the effects of mutations in proteins is critical to many applications, from understanding genetic disease to designing novel proteins that can address our most pressing challenges in climate, agriculture and healthcare. Despite a surge in machine learning-based protein models to tackle these questions, an assessment of their respective benefits is challenging due to the use of distinct, often contrived, experimental datasets, and the variable performance of models across different protein families. Addressing these challenges requires scale. To that end we introduce ProteinGym, a large-scale and holistic set of benchmarks specifically designed for protein fitness prediction and design. It encompasses both a broad collection of over 250 standardized deep mutational scanning assays, spanning millions of mutated sequences, as well as curated clinical datasets providing high-quality expert annotations about mutation effects. We devise a robust evaluation framework that combines metrics for both fitness prediction and design, factors in known limitations of the underlying experimental methods, and covers both zero-shot and supervised settings. We report the performance of a diverse set of over 70 high-performing models from various subfields (eg., alignment-based, inverse folding) into a unified benchmark suite. We open source the corresponding codebase, datasets, MSAs, structures, model predictions and develop a user-friendly website that facilitates data access and analysis.\nProtein language model-guided engineering of an anti-CRISPR protein for precise genome editing in human cells\nAuthors: Marsiglia, J.; Vaalavirta, K.; Knight, E.; Nakamura, M.; Cong, L.; Hughes, N. W.\nScore: 8.3, Published: 2023-12-13 DOI: 10.1101/2023.12.13.571376\nPromiscuous editing by CRISPR/Cas systems within the human genome is a major challenge that must be addressed prior to applying these systems therapeutically. In bacteria, CRISPR/Cas systems have evolved in a co-evolutionary arms race with infectious phage viruses that contain inhibitory anti-CRISPR proteins within their genomes. Here, we harness the outcome of this co-evolutionary arms race to engineer an AcrIIA4 anti-CRISPR protein to increase the precision of CRISPR/Cas-based genome targeting. We developed an approach that specifically leveraged (1) protein language models, (2) deep mutational scanning, and (3) highly parallel DNA repair measurements within human cells. In a single experiment, [~]10,000 AcrIIA4 variants were tested to identify lead AcrIIA4 variants that eliminated detectable off-target editing events while retaining on-target activity. The candidates were further tested in a focused round of screening that included a high-fidelity version of Cas9 as a benchmark. Finally, arrayed experiments using Cas9 delivered as ribonucleoprotein were conducted that demonstrated an increase in gene editing precision across two independent genomic loci and a reduction in the frequency of translocation events between an on-target and off-target site. Thus, language-model-guided high-throughput screening is an effective way to efficiently engineer AcrIIA4 to increase gene editing precision, which could be used to improve the fidelity of gene editing-based therapeutics and to reduce genotoxicity.\nDegenerate DropSynth for Simultaneous Assembly of Diverse Gene Libraries and Local Designed Mutants\nAuthors: Holston, A. S.; Hinton, S. R.; Lindley, K. A.; Kearns, N. C.; Plesa, C.\nScore: 15.7, Published: 2023-12-12 DOI: 10.1101/2023.12.11.569291\nProtein engineering efforts often involve the creation of hybrid or chimeric proteins, where functionality critically hinges on the precise design of linkers and fusion points. Traditional methods have been constrained by a focus on single genes or the random selection of fusion points. Here we introduce an approach which enables the creation of large gene libraries where each library comprises a multitude of diverse, specifically designed genes, each with a corresponding set of programmatically designed fusion points or linkers. When combined with multiplex functional assays, these libraries facilitate the derivation of generalized engineering principles applicable across whole protein families or domain types. Degenerate DropSynth is a multiplex gene synthesis technique which allows for the assembly of up to eight distinct variants for each of the 1,536 designed parent genes in a single reaction. We assemble chimeric sensor histidine kinases and demonstrate the assembly of genes up to 1 kbp in length with an 8% rate of perfect assemblies per gene. Our findings indicate that incorporating an increased number of variants in droplets containing barcoded beads does not significantly affect the rate of perfect assemblies. However, maintaining a consistent level of degeneracy across the library is important to ensure good coverage and reduce inequality. The results suggest the potential for scaling this process to assemble at least 8,000 distinct variants in a single reaction. Degenerate DropSynth enables the systematic exploration of protein families through large-scale, programmable assembly of chimeric proteins, moving beyond the limitations of individual protein studies.\nCell surface sculpting using logic-gated protein actuators\nAuthors: Kofoed, C.; Tay, N. E. S.; Ye, X.; Erkalo, G.; Muir, T. W.\nScore: 4.8, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572113\nCell differentiation and tissue specialization lead to unique cellular surface landscapes and exacerbated or loss of expression patterns can result in further heterogenicity distinctive of pathological phenotypes [1-3]. Immunotherapies and emerging protein therapeutics seek to exploit such differences by engaging cell populations selectively based on their surface markers. Since a single surface antigen rarely defines a specific cell type [4,5], the development of programmable molecular systems that integrate multiple cell surface features to convert on-target inputs to user-defined outputs is highly desirable. Here, we describe an autonomous decision-making protein device driven by proximity-gated protein trans-splicing that allows local generation of an active protein from two otherwise inactive fragments. We show that this protein actuator platform can perform various Boolean logic operations on cell surfaces, allowing highly selective recruitment of enzymatic and cytotoxic activities to specific cells within mixed populations. Due to its intrinsic modularity and tunability, this technology is expected to be compatible with different types of inputs, targeting modalities and functional outputs, and as such will have broad application in the synthetic biology and biotechnology areas.\nContinual improvement of multiplex mutagenesis in Arabidopsis\nAuthors: Develtere, W.; Decaestecker, W.; Rombaut, D.; Anders, C.; Clicque, E.; Vuylsteke, M.; Jacobs, T. B.\nScore: 2.8, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572307\nCRISPR/Cas9 is currently the most powerful tool to generate mutations in plant genomes and more efficient tools are needed as the scale of experiments increases. In the model plant Arabidopsis, the choice of promoter driving Cas9 expression is critical to generate germline mutations. Several optimal promoters have been reported. However, it is unclear which promoter is ideal as they have not been thoroughly tested side-by-side. Furthermore, most plant vectors still use one of the two Cas9 nuclear localization sequence (NLS) configurations initially reported and can still be optimized. We genotyped more than 6,000 Arabidopsis T2 plants to test seven promoters and eleven NLS architectures across 14 targets to systematically improve the generation of single and multiplex inheritable mutations. We find that the RPS5A promoter and double-BP NLS architecture were individually the most efficient components. When combined, 99% of T2 plant contained at least one knockout mutation and 84% contained 4-7-plex knock-outs. These optimizations will be useful to generate higher-order knockouts in the germline of Arabidopsis and likely be applicable to other CRISPR systems as well.\nMethylmercury demethylation and volatilization by animals expressing microbial enzymes.\nAuthors: Tepper, K.; King, J.; Cholan, P. M.; Pfitzner, C.; Morsch, M.; Apte, S.; Maselko, M.\nScore: 8.8, Published: 2023-12-18 DOI: 10.1101/2023.12.11.571038\nMethylmercury is a highly toxic pollutant that accumulates in food webs where it is inaccessible to current remediation technologies. We demonstrate that animals can be engineered to express the microbial enzymes, organomercurial lyase (MerB) and mercuric reductase (MerA), to bioremediate methylmercury. MerA and MerB from Escherichia coli were functional in invertebrate (Drosophila melanogaster) and vertebrate (Danio rerio) model systems and converted methylmercury into volatile Hg0. The engineered animals tolerated higher exposures to methylmercury and accumulated less than half as much mercury relative to their wild-type counterparts. The outcomes of this research could be applied to reduce mercury contamination in farmed and recreationally caught fish, for species conservation, and to restore value to organic wastes contaminated with mercury.\n",
  "wordCount" : "2388",
  "inLanguage": "en",
  "datePublished": "2023-12-20T10:36:27Z",
  "dateModified": "2023-12-20T10:36:27Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 20, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570889">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570889" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570889">
        <p class="paperTitle">In vitro validated antibody design against multiple therapeutic antigens using generative inverse folding</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570889" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570889" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shanehsazzadeh, A.; Alverio, J.; Kasun, G.; Levine, S.; Khan, J. A.; Chung, C.; Diaz, N.; Luton, B. K.; Tarter, Y.; McCloskey, C.; Bateman, K. B.; Carter, H.; Chapman, D.; Consbruck, R.; Jaeger, A.; Kohnert, C.; Kopec-Belliveau, G.; Sutton, J. M.; Guo, Z.; Canales, G.; Ejan, K.; Marsh, E.; Ruelos, A.; Ripley, R.; Stoddard, B.; Caguiat, R.; Chapman, K.; Saunders, M.; Sharp, J.; Ganini da Silva, D.; Feltner, A.; Ripley, J.; Bryant, M. E.; Castillo, D.; Meier, J.; Stegmann, C. M.; Moran, K.; Lemke, C.; Abdulhaqq, S.; Klug, L. R.; Bachas, S.</p>
        <p class="info">Score: 32.1, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570889' target='https://doi.org/10.1101/2023.12.08.570889'> 10.1101/2023.12.08.570889</a></p>
        <p class="abstract">Deep learning approaches have demonstrated the ability to design protein sequences given backbone structures [1, 2, 3, 4, 5]. While these approaches have been applied in silico to designing antibody complementarity-determining regions (CDRs), they have yet to be validated in vitro for designing antibody binders, which is the true measure of success for antibody design. Here we describe IgDesign, a deep learning method for antibody CDR design, and demonstrate its robustness with successful binder design for 8 therapeutic antigens. The model is tasked with designing heavy chain CDR3 (HCDR3) or all three heavy chain CDRs (HCDR123) using native backbone structures of antibody-antigen complexes, along with the antigen and antibody framework (FWR) sequences as context. For each of the 8 antigens, we design 100 HCDR3s and 100 HCDR123s, scaffold them into the native antibodys variable region, and screen them for binding against the antigen using surface plasmon resonance (SPR). As a baseline, we screen 100 HCDR3s taken from the models training set and paired with the native HCDR1 and HCDR2. We observe that both HCDR3 design and HCDR123 design outperform this HCDR3-only baseline. IgDesign is the first experimentally validated antibody inverse folding model. It can design antibody binders to multiple therapeutic antigens with high success rates and, in some cases, improved affinities over clinically validated reference antibodies. Antibody inverse folding has applications to both de novo antibody design and lead optimization, making IgDesign a valuable tool for accelerating drug development and enabling therapeutic design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571886">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571886" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571886">
        <p class="paperTitle">A long-acting prolactin to combat lactation insufficiency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571886" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571886" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kready, K.; Doiron, K.; Redfield Chan, K.; Way, J. C.; Justman, Q.; Powe, C. E.; Silver, P. A.</p>
        <p class="info">Score: 26.2, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571886' target='https://doi.org/10.1101/2023.12.15.571886'> 10.1101/2023.12.15.571886</a></p>
        <p class="abstract">Human infants are born to breastfeed. While 50% of lactating persons struggle to make enough milk, there are no governmentally-approved drugs to enhance lactation1. Here, we engineer a variant of the naturally-occurring driver of lactation, the hormone Prolactin, to increase its serum half-life and produce a viable drug candidate. Our engineered variant, Prolactin-eXtra Long-acting (Prolactin-XL), is comprised of endogenously active human prolactin fused to an engineered human IgG Fc domain designed to overcome the unique drug development challenges specific to the lactating person-infant dyad. Our Prolactin-XL has a serum half-life of 70.9h in mice, 2,625-fold longer than endogenously active prolactin alone (70.9h v. 0.027h). We demonstrate that Prolactin-XL increases milk production and restores growth of pups fed by dams with pharmacologically-ablated lactation. We show that Prolactin-XL-enhanced lactation is accompanied by reversible, lactocyte-driven changes in mammary gland morphology. This work establishes long-acting prolactins as a potentially powerful pharmacologic means to combat insufficient lactation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.16.571988">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.16.571988" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.16.571988">
        <p class="paperTitle">Building synthetic biosensors using red blood cell proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.16.571988" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.16.571988" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dolberg, T. B.; Gunnels, T. F.; Ling, T.; Sarnese, K. A.; Crispino, J. D.; Leonard, J. N.</p>
        <p class="info">Score: 17.8, Published: 2023-12-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.16.571988' target='https://doi.org/10.1101/2023.12.16.571988'> 10.1101/2023.12.16.571988</a></p>
        <p class="abstract">As the use of engineered cell therapies expands from pioneering efforts in cancer immunotherapy to other applications, an attractive but less explored approach is the use of engineered red blood cells (RBCs). Compared to other cells, RBCs have a very long circulation time and reside in the blood compartment, so they could be ideally suited for applications as sentinel cells that enable in situ sensing and diagnostics. However, we largely lack tools for converting RBCs into biosensors. A unique challenge is that RBCs remodel their membranes during maturation, shedding many membrane components, suggesting that an RBC-specific approach may be needed. Towards addressing this need, here we develop a biosensing architecture built on RBC membrane proteins that are retained through erythropoiesis. This biosensor employs a mechanism in which extracellular ligand binding is transduced into intracellular reconstitution of a split output protein (including either a fluorophore or an enzyme). By comparatively evaluating a range of biosensor architectures, linker types, scaffold choices, and output signals, we identify biosensor designs and design features that confer substantial ligand-induced signal in vitro. Finally, we demonstrate that erythroid precursor cells engineered with our RBC protein biosensors function in vivo. This study establishes a foundation for developing RBC-based biosensors that could ultimately address unmet needs including non-invasive monitoring of physiological signals for a range of diagnostic applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571473">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571473" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571473">
        <p class="paperTitle">A translation-independent directed evolution strategy to engineer aminoacyl-tRNA synthetases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571473" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571473" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Soni, C.; Prywes, N.; Hall, M.; Savage, D. F.; Schepartz, A.; Chatterjee, A.</p>
        <p class="info">Score: 16.0, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571473' target='https://doi.org/10.1101/2023.12.13.571473'> 10.1101/2023.12.13.571473</a></p>
        <p class="abstract">Using directed evolution, engineered aminoacyl-tRNA synthetases (aaRS) have been developed that enable co-translational incorporation of numerous noncanonical amino acids (ncAAs) into proteins in living cells. Until now, the selection of such novel aaRS mutants has relied on coupling their activity to the expression of a reporter protein with a selectable phenotype. However, such translation-dependent selection schemes are incompatible with exotic monomers that diverge structurally from canonical -amino acids and are suboptimal substrates for the ribosome. To enable the ribosomal incorporation of such exotic monomers, a two-step solution is needed: A) Engineering an aaRS to acylate its cognate tRNA with the exotic monomer, without relying on ribosomal translation as a readout, and B) Subsequent engineering of the ribosome to accept the resulting acylated tRNA for translation. Here, we report a platform for aaRS engineering that directly selects for tRNA-acylation without ribosomal translation (START). In START, each distinct aaRS mutant is correlated to a cognate tRNA containing a unique sequence barcode. Acylation by an active aaRS mutant protects the associated barcode-containing tRNAs from an oxidative treatment designed to damage the 3&#39;-terminus of the uncharged tRNAs. Sequencing of these surviving barcode-containing tRNAs is then used to reveal the identity of aaRS mutants that acylated the correlated tRNA sequences. The efficacy of START was demonstrated by identifying novel mutants of the M. alvus pyrrolysyl-tRNA synthetase from a naive library that charge noncanonical amino acids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570727">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570727" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570727">
        <p class="paperTitle">ProteinGym: Large-Scale Benchmarks for Protein Design and Fitness Prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570727" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570727" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Notin, P.; Kollasch, A. W.; Ritter, D.; van Niekerk, L.; Paul, S.; Spinner, H.; Rollins, N.; Shaw, A.; Weitzman, R.; Frazer, J.; Dias, M.; Franceschi, D.; Orenbuch, R.; Gal, Y.; Marks, D. S.</p>
        <p class="info">Score: 31.4, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570727' target='https://doi.org/10.1101/2023.12.07.570727'> 10.1101/2023.12.07.570727</a></p>
        <p class="abstract">Predicting the effects of mutations in proteins is critical to many applications, from understanding genetic disease to designing novel proteins that can address our most pressing challenges in climate, agriculture and healthcare. Despite a surge in machine learning-based protein models to tackle these questions, an assessment of their respective benefits is challenging due to the use of distinct, often contrived, experimental datasets, and the variable performance of models across different protein families. Addressing these challenges requires scale. To that end we introduce ProteinGym, a large-scale and holistic set of benchmarks specifically designed for protein fitness prediction and design. It encompasses both a broad collection of over 250 standardized deep mutational scanning assays, spanning millions of mutated sequences, as well as curated clinical datasets providing high-quality expert annotations about mutation effects. We devise a robust evaluation framework that combines metrics for both fitness prediction and design, factors in known limitations of the underlying experimental methods, and covers both zero-shot and supervised settings. We report the performance of a diverse set of over 70 high-performing models from various subfields (eg., alignment-based, inverse folding) into a unified benchmark suite. We open source the corresponding codebase, datasets, MSAs, structures, model predictions and develop a user-friendly website that facilitates data access and analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571376">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571376" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571376">
        <p class="paperTitle">Protein language model-guided engineering of an anti-CRISPR protein for precise genome editing in human cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571376" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571376" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Marsiglia, J.; Vaalavirta, K.; Knight, E.; Nakamura, M.; Cong, L.; Hughes, N. W.</p>
        <p class="info">Score: 8.3, Published: 2023-12-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571376' target='https://doi.org/10.1101/2023.12.13.571376'> 10.1101/2023.12.13.571376</a></p>
        <p class="abstract">Promiscuous editing by CRISPR/Cas systems within the human genome is a major challenge that must be addressed prior to applying these systems therapeutically. In bacteria, CRISPR/Cas systems have evolved in a co-evolutionary arms race with infectious phage viruses that contain inhibitory anti-CRISPR proteins within their genomes. Here, we harness the outcome of this co-evolutionary arms race to engineer an AcrIIA4 anti-CRISPR protein to increase the precision of CRISPR/Cas-based genome targeting. We developed an approach that specifically leveraged (1) protein language models, (2) deep mutational scanning, and (3) highly parallel DNA repair measurements within human cells. In a single experiment, [~]10,000 AcrIIA4 variants were tested to identify lead AcrIIA4 variants that eliminated detectable off-target editing events while retaining on-target activity. The candidates were further tested in a focused round of screening that included a high-fidelity version of Cas9 as a benchmark. Finally, arrayed experiments using Cas9 delivered as ribonucleoprotein were conducted that demonstrated an increase in gene editing precision across two independent genomic loci and a reduction in the frequency of translocation events between an on-target and off-target site. Thus, language-model-guided high-throughput screening is an effective way to efficiently engineer AcrIIA4 to increase gene editing precision, which could be used to improve the fidelity of gene editing-based therapeutics and to reduce genotoxicity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.569291">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.569291" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.569291">
        <p class="paperTitle">Degenerate DropSynth for Simultaneous Assembly of Diverse Gene Libraries and Local Designed Mutants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.569291" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.569291" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Holston, A. S.; Hinton, S. R.; Lindley, K. A.; Kearns, N. C.; Plesa, C.</p>
        <p class="info">Score: 15.7, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.569291' target='https://doi.org/10.1101/2023.12.11.569291'> 10.1101/2023.12.11.569291</a></p>
        <p class="abstract">Protein engineering efforts often involve the creation of hybrid or chimeric proteins, where functionality critically hinges on the precise design of linkers and fusion points. Traditional methods have been constrained by a focus on single genes or the random selection of fusion points. Here we introduce an approach which enables the creation of large gene libraries where each library comprises a multitude of diverse, specifically designed genes, each with a corresponding set of programmatically designed fusion points or linkers. When combined with multiplex functional assays, these libraries facilitate the derivation of generalized engineering principles applicable across whole protein families or domain types. Degenerate DropSynth is a multiplex gene synthesis technique which allows for the assembly of up to eight distinct variants for each of the 1,536 designed parent genes in a single reaction. We assemble chimeric sensor histidine kinases and demonstrate the assembly of genes up to 1 kbp in length with an 8% rate of perfect assemblies per gene. Our findings indicate that incorporating an increased number of variants in droplets containing barcoded beads does not significantly affect the rate of perfect assemblies. However, maintaining a consistent level of degeneracy across the library is important to ensure good coverage and reduce inequality. The results suggest the potential for scaling this process to assemble at least 8,000 distinct variants in a single reaction. Degenerate DropSynth enables the systematic exploration of protein families through large-scale, programmable assembly of chimeric proteins, moving beyond the limitations of individual protein studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572113">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572113" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572113">
        <p class="paperTitle">Cell surface sculpting using logic-gated protein actuators</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572113" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572113" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kofoed, C.; Tay, N. E. S.; Ye, X.; Erkalo, G.; Muir, T. W.</p>
        <p class="info">Score: 4.8, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572113' target='https://doi.org/10.1101/2023.12.18.572113'> 10.1101/2023.12.18.572113</a></p>
        <p class="abstract">Cell differentiation and tissue specialization lead to unique cellular surface landscapes and exacerbated or loss of expression patterns can result in further heterogenicity distinctive of pathological phenotypes [1-3]. Immunotherapies and emerging protein therapeutics seek to exploit such differences by engaging cell populations selectively based on their surface markers. Since a single surface antigen rarely defines a specific cell type [4,5], the development of programmable molecular systems that integrate multiple cell surface features to convert on-target inputs to user-defined outputs is highly desirable. Here, we describe an autonomous decision-making protein device driven by proximity-gated protein trans-splicing that allows local generation of an active protein from two otherwise inactive fragments. We show that this protein actuator platform can perform various Boolean logic operations on cell surfaces, allowing highly selective recruitment of enzymatic and cytotoxic activities to specific cells within mixed populations. Due to its intrinsic modularity and tunability, this technology is expected to be compatible with different types of inputs, targeting modalities and functional outputs, and as such will have broad application in the synthetic biology and biotechnology areas.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572307">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572307" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572307">
        <p class="paperTitle">Continual improvement of multiplex mutagenesis in Arabidopsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572307" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572307" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Develtere, W.; Decaestecker, W.; Rombaut, D.; Anders, C.; Clicque, E.; Vuylsteke, M.; Jacobs, T. B.</p>
        <p class="info">Score: 2.8, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572307' target='https://doi.org/10.1101/2023.12.19.572307'> 10.1101/2023.12.19.572307</a></p>
        <p class="abstract">CRISPR/Cas9 is currently the most powerful tool to generate mutations in plant genomes and more efficient tools are needed as the scale of experiments increases. In the model plant Arabidopsis, the choice of promoter driving Cas9 expression is critical to generate germline mutations. Several optimal promoters have been reported. However, it is unclear which promoter is ideal as they have not been thoroughly tested side-by-side. Furthermore, most plant vectors still use one of the two Cas9 nuclear localization sequence (NLS) configurations initially reported and can still be optimized. We genotyped more than 6,000 Arabidopsis T2 plants to test seven promoters and eleven NLS architectures across 14 targets to systematically improve the generation of single and multiplex inheritable mutations. We find that the RPS5A promoter and double-BP NLS architecture were individually the most efficient components. When combined, 99% of T2 plant contained at least one knockout mutation and 84% contained 4-7-plex knock-outs. These optimizations will be useful to generate higher-order knockouts in the germline of Arabidopsis and likely be applicable to other CRISPR systems as well.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.571038">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.571038" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.571038">
        <p class="paperTitle">Methylmercury demethylation and volatilization by animals expressing microbial enzymes.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.571038" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.571038" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tepper, K.; King, J.; Cholan, P. M.; Pfitzner, C.; Morsch, M.; Apte, S.; Maselko, M.</p>
        <p class="info">Score: 8.8, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.571038' target='https://doi.org/10.1101/2023.12.11.571038'> 10.1101/2023.12.11.571038</a></p>
        <p class="abstract">Methylmercury is a highly toxic pollutant that accumulates in food webs where it is inaccessible to current remediation technologies. We demonstrate that animals can be engineered to express the microbial enzymes, organomercurial lyase (MerB) and mercuric reductase (MerA), to bioremediate methylmercury. MerA and MerB from Escherichia coli were functional in invertebrate (Drosophila melanogaster) and vertebrate (Danio rerio) model systems and converted methylmercury into volatile Hg0. The engineered animals tolerated higher exposures to methylmercury and accumulated less than half as much mercury relative to their wild-type counterparts. The outcomes of this research could be applied to reduce mercury contamination in farmed and recreationally caught fish, for species conservation, and to restore value to organic wastes contaminated with mercury.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
