<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Tuning and minimizing lipidome composition in Mycoplasma mycoides and the Minimal Cell JCVI-Syn3A
Authors: Justice, I. G.; Saenz, J. P.
Score: 28.9, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563757
All cells are encapsulated by a lipid membrane which facilitates the interaction between life and its environment. The property and function of the cell membrane is critically dependent on its lipidome composition which can range from 10s to 100s of unique lipid structures. How life has evolved to exploit such diverse mixtures of lipids has been experimentally challenging to address.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Tuning and minimizing lipidome composition in Mycoplasma mycoides and the Minimal Cell JCVI-Syn3A
Authors: Justice, I. G.; Saenz, J. P.
Score: 28.9, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563757
All cells are encapsulated by a lipid membrane which facilitates the interaction between life and its environment. The property and function of the cell membrane is critically dependent on its lipidome composition which can range from 10s to 100s of unique lipid structures. How life has evolved to exploit such diverse mixtures of lipids has been experimentally challenging to address." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-01T10:38:40+00:00" />
<meta property="article:modified_time" content="2023-11-01T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Tuning and minimizing lipidome composition in Mycoplasma mycoides and the Minimal Cell JCVI-Syn3A
Authors: Justice, I. G.; Saenz, J. P.
Score: 28.9, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563757
All cells are encapsulated by a lipid membrane which facilitates the interaction between life and its environment. The property and function of the cell membrane is critically dependent on its lipidome composition which can range from 10s to 100s of unique lipid structures. How life has evolved to exploit such diverse mixtures of lipids has been experimentally challenging to address."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Tuning and minimizing lipidome composition in Mycoplasma mycoides and the Minimal Cell JCVI-Syn3A\nAuthors: Justice, I. G.; Saenz, J. P.\nScore: 28.9, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563757\nAll cells are encapsulated by a lipid membrane which facilitates the interaction between life and its environment. The property and function of the cell membrane is critically dependent on its lipidome composition which can range from 10s to 100s of unique lipid structures. How life has evolved to exploit such diverse mixtures of lipids has been experimentally challenging to address.",
  "keywords": [
    
  ],
  "articleBody": " Tuning and minimizing lipidome composition in Mycoplasma mycoides and the Minimal Cell JCVI-Syn3A\nAuthors: Justice, I. G.; Saenz, J. P.\nScore: 28.9, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563757\nAll cells are encapsulated by a lipid membrane which facilitates the interaction between life and its environment. The property and function of the cell membrane is critically dependent on its lipidome composition which can range from 10s to 100s of unique lipid structures. How life has evolved to exploit such diverse mixtures of lipids has been experimentally challenging to address. In this study, we introduce a simple approach to minimize and tune lipidome composition in Mycoplasma mycoides and the Minimal Cell (JCVI-Syn3A). By varying the lipid composition of the growth medium, and introducing unmodifiable diether phospholipids, we demonstrate that a lipidome composed of two lipids can support life. By systematically reintroducing phospholipid features, we show that acyl chain diversity is more critical for supporting growth than head group diversity. We also reveal significant effects of altering phospholipid chirality on membrane permeability, cellular robustness, and growth, demonstrating how subtle changes in lipid stereochemistry can have profound effects on cell fitness. Our approach to tune mycoplasma lipidome composition introduces a minimal living model membrane system to understand the design principles and the role of lipid complexity in bioengineering of cell membranes.\nEngineered yeast cells simulating CD19+ cancers to control CAR T cell activation\nAuthors: Deichmann, M.; Schiesaro, G.; Ramanathan, K.; Zeeberg, K. V. W.; Koefoed, N. M. T.; Ormhoj, M.; Friis, R. U. W.; Gill, R. T.; Hadrup, S. R.; Jensen, E. D.; Jensen, M. K.\nScore: 13.1, Published: 2023-10-26 DOI: 10.1101/2023.10.25.563929\nChimeric antigen receptor (CAR) T cells have become an established form of immunotherapy and continue to show promising results for the treatment of hematological cancers in most patients. While this immunotherapy continues to generate excitement in the clinic, antigen modulation in cancer cells affects the quality and safety of CAR T cell therapy. Here we present the engineering of yeast to robustly and dynamically modulate antigen densities, hence establishing a novel tool for assessing antigen-density thresholds in CAR T cell responses. Using a synthetic biology approach, we successfully engineer yeast to simulate cancer cells and demonstrate i) controllable antigen densities of CD19 through the use of G protein-coupled receptors (GPCRs), ii) a customizable system for tuning antigen densities through the employment of heterologous GPCRs for defining signal input types and signal pathway engineering for adapting intensities, and iii) activational control of clinically-derived CAR T cells within a physiologically relevant range compared to the activation elicited by a NALM6 cancer cell line. Based on this yeast-based antigen-presenting cell system, we envision an efficient assessment of how varying antigen densities in cancer cells from patients affect CAR T cell responses and ultimately support safer and better quality of personalized cancer therapies.\nLight-driven synchronization of optogenetic clocks\nAuthors: Cannarsa, M. C.; Liguori, F.; Pellicciotta, N.; Frangipane, G.; Di Leonardo, R.\nScore: 3.5, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563722\nSynthetic genetic oscillators can serve as internal clocks within engineered cells to program periodic expression. However, cell-to-cell variability introduces a dispersion in the characteristics of these clocks that drives the population to complete desynchronization. Here we introduce the optorepressilator, an optically controllable genetic clock that combines the repressilator, a three-node synthetic network in E. coli, with an optogenetic module enabling to reset, delay, or advance its phase using optical inputs. We demonstrate that a population of optorepressilators can be synchronized by transient green light exposure or entrained to oscillate indefinitely by a train of short pulses, through a mechanism reminiscent of natural circadian clocks. Furthermore, we investigate the systems response to detuned external stimuli observing multiple regimes of global synchronization. Integrating experiments and mathematical modeling, we show that the entrainment mechanism is robust and can be understood quantitatively from single cell to population level.\nAllosteric regulation of a molecular motor through de novo protein design\nAuthors: Cross, J. A.; Dawson, W. M.; Shulka, S. R.; Weijman, J. F.; Mantell, J.; Dodding, M. P.; Woolfson, D. N.\nScore: 8.9, Published: 2023-10-17 DOI: 10.1101/2023.10.17.562760\nMany enzymes are allosterically regulated. Our ability to manipulate these structural changes is limited. Here we install an allosteric switch into the kinesin-1 microtubule motor in vitro and in cells. Kinesin-1 is a heterotetramer that accesses open active and closed auto-inhibited states. The equilibrium between these centres on a flexible elbow within a complex coiled-coil architecture. We target the elbow to engineer a closed state that can be opened with a de novo designed peptide. The alternative states are modelled computationally and confirmed by biophysical measurements and electron microscopy. In cells, peptide-driven activation increases kinesin transport, demonstrating a primary role for conformational switching in regulating motor activity. The designs are enabled by our understanding of ubiquitous coiled-coil structures, opening possibilities for controlling other protein activities. One Sentence SummaryDe novo peptide and protein design are used to engineer an allosteric switch into kinesin-1 motors in vitro and directly in cells.\nAllosteric regulation of a molecular motor through de novo protein design\nAuthors: Cross, J. A.; Dawson, W. M.; Shulka, S. R.; Weijman, J. F.; Mantell, J.; Dodding, M. P.; Woolfson, D. N.\nScore: 8.9, Published: 2023-10-17 DOI: 10.1101/2023.10.17.562760\nMany enzymes are allosterically regulated. Our ability to manipulate these structural changes is limited. Here we install an allosteric switch into the kinesin-1 microtubule motor in vitro and in cells. Kinesin-1 is a heterotetramer that accesses open active and closed auto-inhibited states. The equilibrium between these centres on a flexible elbow within a complex coiled-coil architecture. We target the elbow to engineer a closed state that can be opened with a de novo designed peptide. The alternative states are modelled computationally and confirmed by biophysical measurements and electron microscopy. In cells, peptide-driven activation increases kinesin transport, demonstrating a primary role for conformational switching in regulating motor activity. The designs are enabled by our understanding of ubiquitous coiled-coil structures, opening possibilities for controlling other protein activities. One Sentence SummaryDe novo peptide and protein design are used to engineer an allosteric switch into kinesin-1 motors in vitro and directly in cells.\nExpression of Filaments of the Geobacter Extracellular Cytochrome OmcS in Shewanella oneidensis\nAuthors: Lin, T.; Ding, W.; Zhang, D.; Yang, Y.; Li, F.; Xu, D.; Lovley, D. R.; Song, H.\nScore: 2.5, Published: 2023-10-26 DOI: 10.1101/2023.10.26.564108\nThe physiological role of Geobacter sulfurreducens extracellular cytochrome filaments is a matter of debate and the development of proposed electronic device applications of cytochrome filaments awaits methods for large-scale cytochrome nanowire production. Functional studies in Geobacter sulfurreducens are stymied by the broad diversity of redox-active proteins on the outer cell surface and the redundancy and plasticity of extracellular electron transport routes. G. sulfurreducens is a poor chassis for producing cytochrome nanowires for electronics because of its slow, anaerobic growth. Here we report that filaments of the G. sulfurreducens cytochrome OmcS can be heterologously expressed in Shewanella oneidensis. Multiple lines of evidence demonstrated that a strain of S. oneidensis, expressing the G. sulfurreducens OmcS gene on a plasmid, localized OmcS on the outer cell surface. Atomic force microscopy revealed filaments with the unique morphology of OmcS filaments emanating from cells. Electron transfer to OmcS appeared to require a functional outer-membrane porin-cytochrome conduit. The results suggest that S. oneidensis, which grows rapidly to high culture densities under aerobic conditions, may be a suitable for development of a chassis for producing cytochrome nanowires for electronics applications and may also be good model microbe for elucidating cytochrome filament function in anaerobic extracellular electron transfer.\nAutomated, self-resistance gene-guided, and high-throughput genome mining of bioactive natural products from Streptomyces\nAuthors: Yuan, Y.; Huang, C.; Singh, N.; Xun, G.; Zhao, H.\nScore: 1.5, Published: 2023-10-26 DOI: 10.1101/2023.10.26.564101\nNatural products (NPs) produced by bacteria, fungi and plants are a major source of drug leads. Streptomyces species are particularly important in this regard as they produce numerous natural products with prominent bioactivities. Here we report a fully automated, scalable and high-throughput platform for discovery of bioactive natural products in Streptomyces (FAST-NPS). This platform comprises computational prediction and prioritization of target biosynthetic gene clusters (BGCs) guided by self-resistance genes, highly efficient and automated direct cloning and heterologous expression of BGCs, followed by high-throughput fermentation and product extraction from Streptomyces strains. As a proof of concept, we applied this platform to clone 105 BGCs ranging from 10 to 100 kb that contain potential self-resistance genes from 11 Streptomyces strains with a success rate of 95%. Heterologous expression of all successfully cloned BGCs in Streptomyces lividans TK24 led to the discovery of 23 natural products from 12 BGCs. We selected 5 of these 12 BGCs for further characterization and found each of them could produce at least one natural product with antibacterial and/or anti-tumor activity, which resulted in a total of 8 bioactive natural products. Overall, this work would greatly accelerate the discovery of bioactive natural products for biomedical and biotechnological applications. Graphic Abstracts O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=184 SRC=\"FIGDIR/small/564101v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (41K): org.highwire.dtl.DTLVardef@1f1dabborg.highwire.dtl.DTLVardef@14f3537org.highwire.dtl.DTLVardef@12ee4eborg.highwire.dtl.DTLVardef@1b70f44_HPS_FORMAT_FIGEXP M_FIG C_FIG\nModular RNA motifs for orthogonal phase separated compartments\nAuthors: Stewart, J. M.; Li, S.; Tang, A.; Klocke, M. A. K. A.; Gobry, M. V.; Fabrini, G.; Di Michele, L.; Rothemund, P. W. K.; Franco, E.\nScore: 38.5, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561123\nRecent discoveries in biology have highlighted the importance of protein and RNA-based condensates as an alternative to classical membrane-bound organelles for the task of compartmentalizing molecules and biochemical reactions. Here, we demonstrate the rational design of pure RNA condensates from star-shaped RNA motifs. We generate condensates using two different RNA nanostar architectures: multi-stranded nanostars whose binding interactions are programmed via single-stranded overhangs, and single-stranded nanostars whose interactions are programmed via kissing loops. Through rational design of the nanostar interaction sequences, we demonstrate that both architectures can produce orthogonal (distinct and immiscible) condensates, which can be individually tracked via fluorogenic aptamers. We also show that aptamers make it possible to recruit peptides and proteins to the condensates with high specificity. Successful cotranscriptional formation of condensates from single-stranded nanostars suggests that they may be genetically encoded and produced in living cells. We provide a library of orthogonal RNA condensates that can be modularly customized and offer a route toward creating systems of functional artificial organelles.\nFast-track adaptive laboratory evolution of Cupriavidus necator H16 with divalent metal cations\nAuthors: Sitompul, S. N.; Price, J.; Tee, K. L.; Wong, T. S.\nScore: 1.2, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563758\nMicrobial strain improvement through adaptive laboratory evolution (ALE) has been a key strategy in biotechnology for enhancing desired phenotypic traits. Here, we present an accelerated ALE (aALE) workflow and its successful implementation in evolving C. necator H16 for enhanced tolerance towards elevated glycerol concentrations. The method involves the deliberate induction of genetic diversity through controlled exposure to divalent metal cations, enabling the rapid identification of improved variants. Through this approach, we observed the emergence of robust variants capable of growing in high glycerol concentration environments, demonstrating the efficacy of our aALE workflow. Our method offers several advantages over traditional ALE approaches, including its independence from genetically modified strains, specialized genetic tools, and potentially hazardous DNA-modifying agents. By utilizing divalent metal cations as mutagens, we offer a safer, more efficient, and cost-effective alternative for expansion of genetic diversity. With its ability to foster rapid microbial evolution, aALE serves as a valuable addition to the ALE toolbox, holding significant promise for the advancement of microbial strain engineering and bioprocess optimization.\nA three-node Turing gene circuit forms periodic spatial patterns in bacteria\nAuthors: Tica, J.; Oliver Huidobro, M.; Zhu, T.; Wachter, G. K. A.; H. Pazuki, R.; Tonello, E.; Siebert, H.; Stumpf, M. P. H.; Endres, R. G.; Isalan, M.\nScore: 28.7, Published: 2023-10-19 DOI: 10.1101/2023.10.19.563112\nTuring patterns1 are well-known self-organising systems that can form spots, stripes, or labyrinths. They represent a major theory of patterning in tissue organisation, due to their remarkable similarity to some natural patterns, such as skin pigmentation in zebrafish2, digit spacing3,4, and many others. The involvement of Turing patterns in biology has been debated because of their stringent fine-tuning requirements, where patterns only occur within a small subset of parameters5,6. This has complicated the engineering of a synthetic gene circuit for Turing patterns from first principles, even though natural genetic Turing networks have been successfully identified4,7. Here, we engineered a synthetic genetic reaction-diffusion system where three nodes interact according to a non-classical Turing network with improved parametric robustness6. The system was optimised in E. coli and reproducibly generated stationary, periodic, concentric stripe patterns in growing colonies. The patterns were successfully reproduced with a partial differential equation model, in a parameter regime obtained by fitting to experimental data. Our synthetic Turing system can contribute to novel nanotechnologies, such as patterned biomaterial deposition8,9, and provide insights into developmental patterning programs10.\n",
  "wordCount" : "2113",
  "inLanguage": "en",
  "datePublished": "2023-11-01T10:38:40Z",
  "dateModified": "2023-11-01T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 1, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563757">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563757" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563757">
        <p class="paperTitle">Tuning and minimizing lipidome composition in Mycoplasma mycoides and the Minimal Cell JCVI-Syn3A</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563757" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563757" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Justice, I. G.; Saenz, J. P.</p>
        <p class="info">Score: 28.9, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563757' target='https://doi.org/10.1101/2023.10.24.563757'> 10.1101/2023.10.24.563757</a></p>
        <p class="abstract">All cells are encapsulated by a lipid membrane which facilitates the interaction between life and its environment. The property and function of the cell membrane is critically dependent on its lipidome composition which can range from 10s to 100s of unique lipid structures. How life has evolved to exploit such diverse mixtures of lipids has been experimentally challenging to address. In this study, we introduce a simple approach to minimize and tune lipidome composition in Mycoplasma mycoides and the Minimal Cell (JCVI-Syn3A). By varying the lipid composition of the growth medium, and introducing unmodifiable diether phospholipids, we demonstrate that a lipidome composed of two lipids can support life. By systematically reintroducing phospholipid features, we show that acyl chain diversity is more critical for supporting growth than head group diversity. We also reveal significant effects of altering phospholipid chirality on membrane permeability, cellular robustness, and growth, demonstrating how subtle changes in lipid stereochemistry can have profound effects on cell fitness. Our approach to tune mycoplasma lipidome composition introduces a minimal living model membrane system to understand the design principles and the role of lipid complexity in bioengineering of cell membranes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.563929">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.563929" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.563929">
        <p class="paperTitle">Engineered yeast cells simulating CD19&#43; cancers to control CAR T cell activation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.563929" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.563929" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Deichmann, M.; Schiesaro, G.; Ramanathan, K.; Zeeberg, K. V. W.; Koefoed, N. M. T.; Ormhoj, M.; Friis, R. U. W.; Gill, R. T.; Hadrup, S. R.; Jensen, E. D.; Jensen, M. K.</p>
        <p class="info">Score: 13.1, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.563929' target='https://doi.org/10.1101/2023.10.25.563929'> 10.1101/2023.10.25.563929</a></p>
        <p class="abstract">Chimeric antigen receptor (CAR) T cells have become an established form of immunotherapy and continue to show promising results for the treatment of hematological cancers in most patients. While this immunotherapy continues to generate excitement in the clinic, antigen modulation in cancer cells affects the quality and safety of CAR T cell therapy. Here we present the engineering of yeast to robustly and dynamically modulate antigen densities, hence establishing a novel tool for assessing antigen-density thresholds in CAR T cell responses. Using a synthetic biology approach, we successfully engineer yeast to simulate cancer cells and demonstrate i) controllable antigen densities of CD19 through the use of G protein-coupled receptors (GPCRs), ii) a customizable system for tuning antigen densities through the employment of heterologous GPCRs for defining signal input types and signal pathway engineering for adapting intensities, and iii) activational control of clinically-derived CAR T cells within a physiologically relevant range compared to the activation elicited by a NALM6 cancer cell line. Based on this yeast-based antigen-presenting cell system, we envision an efficient assessment of how varying antigen densities in cancer cells from patients affect CAR T cell responses and ultimately support safer and better quality of personalized cancer therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563722">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563722" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563722">
        <p class="paperTitle">Light-driven synchronization of optogenetic clocks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563722" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563722" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cannarsa, M. C.; Liguori, F.; Pellicciotta, N.; Frangipane, G.; Di Leonardo, R.</p>
        <p class="info">Score: 3.5, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563722' target='https://doi.org/10.1101/2023.10.24.563722'> 10.1101/2023.10.24.563722</a></p>
        <p class="abstract">Synthetic genetic oscillators can serve as internal clocks within engineered cells to program periodic expression. However, cell-to-cell variability introduces a dispersion in the characteristics of these clocks that drives the population to complete desynchronization. Here we introduce the optorepressilator, an optically controllable genetic clock that combines the repressilator, a three-node synthetic network in E. coli, with an optogenetic module enabling to reset, delay, or advance its phase using optical inputs. We demonstrate that a population of optorepressilators can be synchronized by transient green light exposure or entrained to oscillate indefinitely by a train of short pulses, through a mechanism reminiscent of natural circadian clocks. Furthermore, we investigate the systems response to detuned external stimuli observing multiple regimes of global synchronization. Integrating experiments and mathematical modeling, we show that the entrainment mechanism is robust and can be understood quantitatively from single cell to population level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562760">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562760" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562760">
        <p class="paperTitle">Allosteric regulation of a molecular motor through de novo protein design</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562760" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562760" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cross, J. A.; Dawson, W. M.; Shulka, S. R.; Weijman, J. F.; Mantell, J.; Dodding, M. P.; Woolfson, D. N.</p>
        <p class="info">Score: 8.9, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562760' target='https://doi.org/10.1101/2023.10.17.562760'> 10.1101/2023.10.17.562760</a></p>
        <p class="abstract">Many enzymes are allosterically regulated. Our ability to manipulate these structural changes is limited. Here we install an allosteric switch into the kinesin-1 microtubule motor in vitro and in cells. Kinesin-1 is a heterotetramer that accesses open active and closed auto-inhibited states. The equilibrium between these centres on a flexible elbow within a complex coiled-coil architecture. We target the elbow to engineer a closed state that can be opened with a de novo designed peptide. The alternative states are modelled computationally and confirmed by biophysical measurements and electron microscopy. In cells, peptide-driven activation increases kinesin transport, demonstrating a primary role for conformational switching in regulating motor activity. The designs are enabled by our understanding of ubiquitous coiled-coil structures, opening possibilities for controlling other protein activities.

One Sentence SummaryDe novo peptide and protein design are used to engineer an allosteric switch into kinesin-1 motors in vitro and directly in cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562760">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562760" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562760">
        <p class="paperTitle">Allosteric regulation of a molecular motor through de novo protein design</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562760" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562760" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cross, J. A.; Dawson, W. M.; Shulka, S. R.; Weijman, J. F.; Mantell, J.; Dodding, M. P.; Woolfson, D. N.</p>
        <p class="info">Score: 8.9, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562760' target='https://doi.org/10.1101/2023.10.17.562760'> 10.1101/2023.10.17.562760</a></p>
        <p class="abstract">Many enzymes are allosterically regulated. Our ability to manipulate these structural changes is limited. Here we install an allosteric switch into the kinesin-1 microtubule motor in vitro and in cells. Kinesin-1 is a heterotetramer that accesses open active and closed auto-inhibited states. The equilibrium between these centres on a flexible elbow within a complex coiled-coil architecture. We target the elbow to engineer a closed state that can be opened with a de novo designed peptide. The alternative states are modelled computationally and confirmed by biophysical measurements and electron microscopy. In cells, peptide-driven activation increases kinesin transport, demonstrating a primary role for conformational switching in regulating motor activity. The designs are enabled by our understanding of ubiquitous coiled-coil structures, opening possibilities for controlling other protein activities.

One Sentence SummaryDe novo peptide and protein design are used to engineer an allosteric switch into kinesin-1 motors in vitro and directly in cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564108">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564108" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564108">
        <p class="paperTitle">Expression of Filaments of the Geobacter Extracellular Cytochrome OmcS in Shewanella oneidensis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564108" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564108" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lin, T.; Ding, W.; Zhang, D.; Yang, Y.; Li, F.; Xu, D.; Lovley, D. R.; Song, H.</p>
        <p class="info">Score: 2.5, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564108' target='https://doi.org/10.1101/2023.10.26.564108'> 10.1101/2023.10.26.564108</a></p>
        <p class="abstract">The physiological role of Geobacter sulfurreducens extracellular cytochrome filaments is a matter of debate and the development of proposed electronic device applications of cytochrome filaments awaits methods for large-scale cytochrome nanowire production. Functional studies in Geobacter sulfurreducens are stymied by the broad diversity of redox-active proteins on the outer cell surface and the redundancy and plasticity of extracellular electron transport routes. G. sulfurreducens is a poor chassis for producing cytochrome nanowires for electronics because of its slow, anaerobic growth. Here we report that filaments of the G. sulfurreducens cytochrome OmcS can be heterologously expressed in Shewanella oneidensis. Multiple lines of evidence demonstrated that a strain of S. oneidensis, expressing the G. sulfurreducens OmcS gene on a plasmid, localized OmcS on the outer cell surface. Atomic force microscopy revealed filaments with the unique morphology of OmcS filaments emanating from cells. Electron transfer to OmcS appeared to require a functional outer-membrane porin-cytochrome conduit. The results suggest that S. oneidensis, which grows rapidly to high culture densities under aerobic conditions, may be a suitable for development of a chassis for producing cytochrome nanowires for electronics applications and may also be good model microbe for elucidating cytochrome filament function in anaerobic extracellular electron transfer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564101">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564101" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564101">
        <p class="paperTitle">Automated, self-resistance gene-guided, and high-throughput genome mining of bioactive natural products from Streptomyces</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564101" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564101" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yuan, Y.; Huang, C.; Singh, N.; Xun, G.; Zhao, H.</p>
        <p class="info">Score: 1.5, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564101' target='https://doi.org/10.1101/2023.10.26.564101'> 10.1101/2023.10.26.564101</a></p>
        <p class="abstract">Natural products (NPs) produced by bacteria, fungi and plants are a major source of drug leads. Streptomyces species are particularly important in this regard as they produce numerous natural products with prominent bioactivities. Here we report a fully automated, scalable and high-throughput platform for discovery of bioactive natural products in Streptomyces (FAST-NPS). This platform comprises computational prediction and prioritization of target biosynthetic gene clusters (BGCs) guided by self-resistance genes, highly efficient and automated direct cloning and heterologous expression of BGCs, followed by high-throughput fermentation and product extraction from Streptomyces strains. As a proof of concept, we applied this platform to clone 105 BGCs ranging from 10 to 100 kb that contain potential self-resistance genes from 11 Streptomyces strains with a success rate of 95%. Heterologous expression of all successfully cloned BGCs in Streptomyces lividans TK24 led to the discovery of 23 natural products from 12 BGCs. We selected 5 of these 12 BGCs for further characterization and found each of them could produce at least one natural product with antibacterial and/or anti-tumor activity, which resulted in a total of 8 bioactive natural products. Overall, this work would greatly accelerate the discovery of bioactive natural products for biomedical and biotechnological applications.

Graphic Abstracts

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=184 SRC=&#34;FIGDIR/small/564101v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (41K):
org.highwire.dtl.DTLVardef@1f1dabborg.highwire.dtl.DTLVardef@14f3537org.highwire.dtl.DTLVardef@12ee4eborg.highwire.dtl.DTLVardef@1b70f44_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561123">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561123" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561123">
        <p class="paperTitle">Modular RNA motifs for orthogonal phase separated compartments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561123" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561123" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stewart, J. M.; Li, S.; Tang, A.; Klocke, M. A. K. A.; Gobry, M. V.; Fabrini, G.; Di Michele, L.; Rothemund, P. W. K.; Franco, E.</p>
        <p class="info">Score: 38.5, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561123' target='https://doi.org/10.1101/2023.10.06.561123'> 10.1101/2023.10.06.561123</a></p>
        <p class="abstract">Recent discoveries in biology have highlighted the importance of protein and RNA-based condensates as an alternative to classical membrane-bound organelles for the task of compartmentalizing molecules and biochemical reactions. Here, we demonstrate the rational design of pure RNA condensates from star-shaped RNA motifs. We generate condensates using two different RNA nanostar architectures: multi-stranded nanostars whose binding interactions are programmed via single-stranded overhangs, and single-stranded nanostars whose interactions are programmed via kissing loops. Through rational design of the nanostar interaction sequences, we demonstrate that both architectures can produce orthogonal (distinct and immiscible) condensates, which can be individually tracked via fluorogenic aptamers. We also show that aptamers make it possible to recruit peptides and proteins to the condensates with high specificity. Successful cotranscriptional formation of condensates from single-stranded nanostars suggests that they may be genetically encoded and produced in living cells. We provide a library of orthogonal RNA condensates that can be modularly customized and offer a route toward creating systems of functional artificial organelles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563758">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563758" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563758">
        <p class="paperTitle">Fast-track adaptive laboratory evolution of Cupriavidus necator H16 with divalent metal cations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563758" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563758" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sitompul, S. N.; Price, J.; Tee, K. L.; Wong, T. S.</p>
        <p class="info">Score: 1.2, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563758' target='https://doi.org/10.1101/2023.10.24.563758'> 10.1101/2023.10.24.563758</a></p>
        <p class="abstract">Microbial strain improvement through adaptive laboratory evolution (ALE) has been a key strategy in biotechnology for enhancing desired phenotypic traits. Here, we present an accelerated ALE (aALE) workflow and its successful implementation in evolving C. necator H16 for enhanced tolerance towards elevated glycerol concentrations. The method involves the deliberate induction of genetic diversity through controlled exposure to divalent metal cations, enabling the rapid identification of improved variants. Through this approach, we observed the emergence of robust variants capable of growing in high glycerol concentration environments, demonstrating the efficacy of our aALE workflow. Our method offers several advantages over traditional ALE approaches, including its independence from genetically modified strains, specialized genetic tools, and potentially hazardous DNA-modifying agents. By utilizing divalent metal cations as mutagens, we offer a safer, more efficient, and cost-effective alternative for expansion of genetic diversity. With its ability to foster rapid microbial evolution, aALE serves as a valuable addition to the ALE toolbox, holding significant promise for the advancement of microbial strain engineering and bioprocess optimization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.563112">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.563112" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.563112">
        <p class="paperTitle">A three-node Turing gene circuit forms periodic spatial patterns in bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.563112" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.563112" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tica, J.; Oliver Huidobro, M.; Zhu, T.; Wachter, G. K. A.; H. Pazuki, R.; Tonello, E.; Siebert, H.; Stumpf, M. P. H.; Endres, R. G.; Isalan, M.</p>
        <p class="info">Score: 28.7, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.563112' target='https://doi.org/10.1101/2023.10.19.563112'> 10.1101/2023.10.19.563112</a></p>
        <p class="abstract">Turing patterns1 are well-known self-organising systems that can form spots, stripes, or labyrinths. They represent a major theory of patterning in tissue organisation, due to their remarkable similarity to some natural patterns, such as skin pigmentation in zebrafish2, digit spacing3,4, and many others. The involvement of Turing patterns in biology has been debated because of their stringent fine-tuning requirements, where patterns only occur within a small subset of parameters5,6. This has complicated the engineering of a synthetic gene circuit for Turing patterns from first principles, even though natural genetic Turing networks have been successfully identified4,7. Here, we engineered a synthetic genetic reaction-diffusion system where three nodes interact according to a non-classical Turing network with improved parametric robustness6. The system was optimised in E. coli and reproducibly generated stationary, periodic, concentric stripe patterns in growing colonies. The patterns were successfully reproduced with a partial differential equation model, in a parameter regime obtained by fitting to experimental data. Our synthetic Turing system can contribute to novel nanotechnologies, such as patterned biomaterial deposition8,9, and provide insights into developmental patterning programs10.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
