<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="An interdependent Metabolic and Genetic Network shows emergent properties in vitro
Authors: Giaveri, S.; Bohra, N.; Diehl, C.; Ballinger, M.; Paczia, N.; Glatter, T.; Erb, T. J.
Score: 24.4, Published: 2023-11-26 DOI: 10.1101/2023.11.26.568713
A hallmark of all living organisms is their ability for self-regeneration which requires a tight integration of metabolic and genetic networks. Here we constructed a metabolic and genetic linked in vitro network (MGLN) that shows life-like behavior outside of a cellular context and generates its own building blocks from non-living matter.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="An interdependent Metabolic and Genetic Network shows emergent properties in vitro
Authors: Giaveri, S.; Bohra, N.; Diehl, C.; Ballinger, M.; Paczia, N.; Glatter, T.; Erb, T. J.
Score: 24.4, Published: 2023-11-26 DOI: 10.1101/2023.11.26.568713
A hallmark of all living organisms is their ability for self-regeneration which requires a tight integration of metabolic and genetic networks. Here we constructed a metabolic and genetic linked in vitro network (MGLN) that shows life-like behavior outside of a cellular context and generates its own building blocks from non-living matter." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-29T10:40:14+00:00" />
<meta property="article:modified_time" content="2023-11-29T10:40:14+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="An interdependent Metabolic and Genetic Network shows emergent properties in vitro
Authors: Giaveri, S.; Bohra, N.; Diehl, C.; Ballinger, M.; Paczia, N.; Glatter, T.; Erb, T. J.
Score: 24.4, Published: 2023-11-26 DOI: 10.1101/2023.11.26.568713
A hallmark of all living organisms is their ability for self-regeneration which requires a tight integration of metabolic and genetic networks. Here we constructed a metabolic and genetic linked in vitro network (MGLN) that shows life-like behavior outside of a cellular context and generates its own building blocks from non-living matter."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "An interdependent Metabolic and Genetic Network shows emergent properties in vitro\nAuthors: Giaveri, S.; Bohra, N.; Diehl, C.; Ballinger, M.; Paczia, N.; Glatter, T.; Erb, T. J.\nScore: 24.4, Published: 2023-11-26 DOI: 10.1101/2023.11.26.568713\nA hallmark of all living organisms is their ability for self-regeneration which requires a tight integration of metabolic and genetic networks. Here we constructed a metabolic and genetic linked in vitro network (MGLN) that shows life-like behavior outside of a cellular context and generates its own building blocks from non-living matter.",
  "keywords": [
    
  ],
  "articleBody": " An interdependent Metabolic and Genetic Network shows emergent properties in vitro\nAuthors: Giaveri, S.; Bohra, N.; Diehl, C.; Ballinger, M.; Paczia, N.; Glatter, T.; Erb, T. J.\nScore: 24.4, Published: 2023-11-26 DOI: 10.1101/2023.11.26.568713\nA hallmark of all living organisms is their ability for self-regeneration which requires a tight integration of metabolic and genetic networks. Here we constructed a metabolic and genetic linked in vitro network (MGLN) that shows life-like behavior outside of a cellular context and generates its own building blocks from non-living matter. To this end, we integrated the metabolism of the crotonyl-CoA/ethyl-malonyl-CoA/hydroxybutyryl-CoA (CETCH) cycle with cell-free protein synthesis using recombinant elements (PURE). We demonstrate that the MGLN produces the essential amino acid glycine from inorganic carbon (CO2), and incorporates it into target proteins following DNA-encoded instructions. By programming genetically encoded response into metabolic networks our work opens new avenues for the development of advanced biomimetic systems with emergent properties, including decision-making, self-regeneration and evolution.\nHighly parallelized construction of DNA from low-cost oligonucleotide mixtures using Data-optimized Assembly Design and Golden Gate\nAuthors: Lund, S.; Potapov, V.; Johnson, S. R.; Buss, J.; Tanner, N. A.\nScore: 16.6, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567888\nCommercially synthesized genes are typically made using variations of homology-based cloning techniques, including polymerase cycling assembly from chemically synthesized microarray-derived oligonucleotides. Here we apply Data-optimized Assembly Design to the synthesis of hundreds of codon-optimized genes in both constitutive and inducible vectors using Golden Gate Assembly. Starting from oligonucleotide pools, we synthesize genes in three simple steps: 1) Amplification of parts belonging to individual assemblies in parallel from a single pool; 2) Golden Gate Assembly of parts for each construct; and 3) Transformation. We construct genes from receiving DNA to sequence confirmed isolates in as little as 4 days. By leveraging the ligation fidelity afforded by T4 DNA ligase, we expect to be able to construct a larger breadth of sequences not currently supported by homology-based methods which require stability of extensive single-stranded DNA overhangs.\nAutotrophic growth of E. coli is achieved by a small number of genetic changes\nAuthors: Ben Nissan, R.; Milshtein, E.; Pahl, V.; de Pins, B.; Jona, G.; Levi, D.; Yung, H.; Nir, N.; Ezra, D.; Gleizer, S.; Link, H.; Noor, E.; Milo, R.\nScore: 56.9, Published: 2023-11-26 DOI: 10.1101/2023.06.03.543535\nSynthetic autotrophy is a promising avenue to sustainable bioproduction from CO2. Here, we use iterative laboratory evolution to generate several distinct autotrophic strains. Utilising this genetic diversity, we identify that just three mutations are sufficient for E. coli to grow autotrophically, when introduced alongside non-native energy (formate dehydrogenase) and carbon-fixing (RuBisCO, phosphoribulokinase, carbonic anhydrase) modules. The mutated genes are involved in glycolysis (pgi), central-carbon regulation (crp), and RNA transcription (rpoB). The pgi mutation reduces the enzymes activity, thereby stabilising the carbon-fixing cycle by capping a major branching flux. For the other two mutations, we observe down-regulation of several metabolic pathways and increased expression of native genes associated with the carbon-fixing module (rpiB) and the energy module (fdoGH), as well as an increased ratio of NADH/NAD+ - the cycles electron-donor. This study demonstrates the malleability of metabolism and its capacity to switch trophic modes using only a small number of genetic changes and could facilitate transforming other heterotrophic organisms into autotrophs.\nBacteriophage-host interactions in microgravity onboard the International Space Station\nAuthors: Huss, P.; Chitboonthavisuk, C.; Meger, A.; Nishikawa, K.; Oates, R. P.; Mills, H.; Holzhaus, O.; Raman, S.\nScore: 21.4, Published: 2023-11-13 DOI: 10.1101/2023.10.10.561409\nBacteriophage-host interactions play a fundamental role in shaping microbial ecosystems. Although researchers have extensively studied phage-host interactions in terrestrial ecosystems, the impact of microgravity on phage-host interactions is yet to be investigated. Here, we report the dynamics of interactions between T7 bacteriophage and E. coli in microgravity onboard the International Space Station (ISS). We found phage activity was delayed but ultimately successful in microgravity. We identified several de novo mutations in phage and bacteria that improved fitness in microgravity. Deep mutational scanning of the receptor binding domain revealed substantial differences in the number, position, and mutational preferences between gravity and microgravity, reflecting underlying differences in the bacterial adaptations. Combinatorial libraries informed by microgravity selections gave T7 mutants with 100-10,000-fold higher activity on uropathogenic E. coli under terrestrial conditions than wildtype T7. Our findings lay the foundation for future research on the impact of microgravity on phage-host interactions and microbial communities.\nYeastIT: Reducing mutational bias for in vivo directed evolution using a novel yeast mutator strain based on dual adenine-/cytosine-targeting and error-prone DNA repair\nAuthors: Napiorkowska, M.; Fischer, K.; Penner, M.; Knyphausen, P.; Hollfelder, F.\nScore: 7.7, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567881\nEngineering proteins with new functions and properties often requires navigating large sequence spaces through rounds of iterative improvement. However, a disparity exists between the gradual pace of natural long-term evolution and a typical laboratory evolution workflow that relies on enriching functional variants from highly diverse in vitro generated libraries through very few screening rounds. Laboratory experiments often eschew presumed natural strategies such as neutral/non-adaptive and multi-phase evolution trajectories, and therefore mutagenesis technologies suitable for long nature-like timescales are needed. Here, we introduce YeastIT, a novel in vivo mutagenesis tool for protein engineering that leverages an S. cerevisiae strain engineered to exhibit mutagenic activity directed to the gene of interest, allowing its continuous diversification. Mutagenesis is achieved by generating DNA damage through nucleoside deamination, followed by introduction of mutations by harnessing the process of error-prone DNA translesion synthesis. By eliminating the transformation step, YeastIT allows multiple rounds of screening or selection without interruptions for library diversification, thereby enabling long-term and continuous evolution campaigns. Our characterization of the mutational spectrum and frequency of the YeastIT-generated libraries, and its comparison to other methods (error-prone PCR, PACE, MutaT7, eMutaT7, OrthoRep, TRIDENT, EvolVR) demonstrates comparable mutation rates combined with a significant reduction in mutagenic bias relative to most of the alternatives. To validate YeastIT, we carried out directed evolution of a DARPin binding protein to achieve a 15-fold improved affinity. YeastIT thus provides a tool for exploring different evolutionary trajectories which overcomes previous limitations of variant availability (due to bias and low mutation rates) and emulates the way proteins emerge in Nature.\ndFLASH; dual FLuorescent transcription factor Activity Sensor for Histone integrated live-cell reporting and high-content screening\nAuthors: Allen, T. P.; Roennfeldt, A. E.; Reckdharajkumar, M.; Liu, M.; Quinn, R.; Russell, D. L.; Peet, D. J.; Whitelaw, M. L.; Bersten, D. C.\nScore: 5.8, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568191\nLive-cell reporting of regulated transcription factor (TF) activity has a wide variety of applications in synthetic biology, drug discovery, and functional genomics. As a result, there is high value in the generation of versatile, sensitive, robust systems that can function across a range of cell types and be adapted toward diverse TF classes. Here we present the dual FLuorescent transcription factor Activity Sensor for Histone integrated live-cell reporting (dFLASH), a modular sensor for TF activity that can be readily integrated into cellular genomes. We demonstrate readily modified dFLASH platforms that homogenously, robustly, and specifically sense regulation of endogenous Hypoxia Inducible Factor (HIF) and Progesterone receptor (PGR) activities, as well as regulated coactivator recruitment to a synthetic DNA-Binding Domain-Activator Domain fusion proteins. The dual-colour nuclear fluorescence produced normalised dynamic live-cell TF activity sensing with facile generation of high-content screening lines with strong signal:noise ratios and reproducible screening capabilities (Z = 0.68-0.74). Finally, we demonstrate the utility of this platform for functional genomics applications by using CRISPRoff to modulate the HIF regulatory pathway, and utility for drug screening by using high content imaging in a bimodal design to isolate activators and inhibitors of the HIF pathway from a [~]1600 natural product library.\nContinuous evolution of user-defined genes at 1-million-times the genomic mutation rate\nAuthors: Rix, G.; Williams, R. L.; Spinner, H.; Hu, V. J.; Marks, D. S.; Liu, C. C.\nScore: 58.5, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566922\nWhen nature maintains or evolves a genes function over millions of years at scale, it produces a diversity of homologous sequences whose patterns of conservation and change contain rich structural, functional, and historical information about the gene. However, natural gene diversity likely excludes vast regions of functional sequence space and includes phylogenetic and evolutionary eccentricities, limiting what information we can extract. We introduce an accessible experimental approach for compressing long-term gene evolution to laboratory timescales, allowing for the direct observation of extensive adaptation and divergence followed by inference of structural, functional, and environmental constraints for any selectable gene. To enable this approach, we developed a new orthogonal DNA replication (OrthoRep) system that durably hypermutates chosen genes at a rate of \u003e10-4 substitutions per base in vivo. When OrthoRep was used to evolve a conditionally essential maladapted enzyme, we obtained thousands of unique multi-mutation sequences with many pairs \u003e60 amino acids apart (\u003e15% divergence), revealing known and new factors influencing enzyme adaptation. The fitness of evolved sequences was not predictable by advanced machine learning models trained on natural variation. We suggest that OrthoRep supports the prospective and systematic discovery of constraints shaping gene evolution, uncovering of new regions in fitness landscapes, and general applications in biomolecular engineering.\nDirected Evolution of Genetically Encoded LYTACs for Cell-Mediated Delivery\nAuthors: Yang, J. L.; Yamada-Hunter, S. A.; Labanieh, L.; Sotillo, E.; Cheah, J. S.; Roberts, D. S.; Mackall, C.; Ting, A. Y.; Bertozzi, C.\nScore: 21.9, Published: 2023-11-15 DOI: 10.1101/2023.11.14.567117\nLysosome-targeting chimeras (LYTACs) are a promising therapeutic modality to drive the degradation of extracellular proteins. However, early versions of LYTAC contain synthetic glycopeptides that cannot be genetically encoded. Here we present our designs for a fully genetically encodable LYTAC (GELYTAC), making our tool compatible with integration into therapeutic cells for targeted delivery at diseased sites. To achieve this, we replaced the glycopeptide portion of LYTACs with the protein insulin like growth factor 2 (IGF2). After showing initial efficacy with wild type IGF2, we increased the potency of GELYTAC using directed evolution. Subsequently, we demonstrated that our engineered GELYTAC construct not only secretes from HEK293T cells but also from human primary T-cells to drive the uptake of various targets into receiver cells. Immune cells engineered to secrete GELYTAC thus represent a promising avenue for spatially-selective targeted protein degradation. Significance StatementBetter therapeutic windows can be achieved by targeting therapeutics to their desired sites of action. For protein therapeutics, this might be achieved by engineering cell therapies that home to a tissue of interest and secrete the biologic drug locally. Here, we demonstrate that human primary T cells can be engineered to produce genetically encoded lysosome targeting chimeras (GELYTACs). These GELYTACs mediate the degradation of extracellular proteins associated with cancer progression. Thus, cells engineered to produce GELYTACs represent a potential new class of cancer therapeutics.\nHigh-yield enzymatic synthesis of mono- and trifluorinated alanine enantiomers\nAuthors: Nieto-Dominguez, M.; Sako, A.; Enemark-Rasmussen, K.; Held Gotfredsen, C.; Rago, D.; Nikel, P. I.\nScore: 2.0, Published: 2023-11-28 DOI: 10.1101/2023.11.28.569005\nFluorinated amino acids are a promising entry point for incorporating new-to-Nature chemistries in biological systems. Hence, novel methods are needed for the selective synthesis of these building blocks. In this study, we focused on the enzymatic synthesis of fluorinated alanine enantiomers. To this end, the alanine dehydrogenase from Vibrio proteolyticus and the diaminopimelate dehydrogenase from Symbiobacterium thermophilum were applied to the in vitro production of (R)-3-fluoroalanine and (S)-3-fluoroalanine, respectively, using 3-fluoropyruvate as the substrate. Additionally, an alanine racemase from Streptomyces lavendulae, originally selected for setting an alternative enzymatic cascade leading to the production of these non-canonical amino acids, had an unprecedented catalytic efficiency in the beta-elimination of fluorine from the monosubstituted fluoroalanine. The in vitro enzymatic cascade based on the dehydrogenases of V. proteolyticus and S. thermophilum included a cofactor recycling system, whereby a formate dehydrogenase from Pseudomonas sp. 101 (either native or engineered) coupled formate oxidation to NAD(P)H formation. Under these conditions, the reaction yields for (R)-3-fluoroalanine and (S)-3-fluoroalanine reached \u003e85% on the fluorinated substrate and proceeded with complete enantiomeric excess. Moreover, the selected dehydrogenases were also able to catalyze the conversion of trifluoropyruvate into trifluorinated alanine, as a first-case example of biocatalysis with amino acids carrying a trifluoromethyl group.\nA library of reporters of the global regulators of gene expression of Escherichia coli\nAuthors: Dash, S.; Jagadeesan, R.; Baptista, I. S. C.; Chauhan, V.; Kandavalli, V.; Oliveira, S. M. D.; Ribeiro, A. S.\nScore: 1.2, Published: 2023-11-28 DOI: 10.1101/2023.11.28.568972\nThe topology of the transcription factor network (TFN) of E. coli is far from uniform, with 22 global regulator (GR) proteins controlling one-third of all genes. So far, their production rates cannot be tracked by comparable fluorescent proteins. We developed a library of fluorescent reporters for 16 GRs for this purpose. Each consists of a single-copy plasmid coding for GFP fused to the full-length copy of the native promoter. We tracked their activity in exponential and stationary growth, as well as under weak and strong stresses. We show that the reporters have high sensitivity and specificity to all stresses tested and detect single-cell variability in transcription rates. Given the influence of GRs on the TFN, we expect that the new library will contribute to dissecting global transcriptional stress-response programs of E. coli. Moreover, the library can be invaluable in bioindustrial applications that tune those programs to, instead of cell growth, favor productivity while reducing energy consumption.\n",
  "wordCount" : "2188",
  "inLanguage": "en",
  "datePublished": "2023-11-29T10:40:14Z",
  "dateModified": "2023-11-29T10:40:14Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 29, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.26.568713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.26.568713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.26.568713">
        <p class="paperTitle">An interdependent Metabolic and Genetic Network shows emergent properties in vitro</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.26.568713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.26.568713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Giaveri, S.; Bohra, N.; Diehl, C.; Ballinger, M.; Paczia, N.; Glatter, T.; Erb, T. J.</p>
        <p class="info">Score: 24.4, Published: 2023-11-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.26.568713' target='https://doi.org/10.1101/2023.11.26.568713'> 10.1101/2023.11.26.568713</a></p>
        <p class="abstract">A hallmark of all living organisms is their ability for self-regeneration which requires a tight integration of metabolic and genetic networks. Here we constructed a metabolic and genetic linked in vitro network (MGLN) that shows life-like behavior outside of a cellular context and generates its own building blocks from non-living matter. To this end, we integrated the metabolism of the crotonyl-CoA/ethyl-malonyl-CoA/hydroxybutyryl-CoA (CETCH) cycle with cell-free protein synthesis using recombinant elements (PURE). We demonstrate that the MGLN produces the essential amino acid glycine from inorganic carbon (CO2), and incorporates it into target proteins following DNA-encoded instructions. By programming genetically encoded response into metabolic networks our work opens new avenues for the development of advanced biomimetic systems with emergent properties, including decision-making, self-regeneration and evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567888">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567888" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567888">
        <p class="paperTitle">Highly parallelized construction of DNA from low-cost oligonucleotide mixtures using Data-optimized Assembly Design and Golden Gate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567888" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567888" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lund, S.; Potapov, V.; Johnson, S. R.; Buss, J.; Tanner, N. A.</p>
        <p class="info">Score: 16.6, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567888' target='https://doi.org/10.1101/2023.11.20.567888'> 10.1101/2023.11.20.567888</a></p>
        <p class="abstract">Commercially synthesized genes are typically made using variations of homology-based cloning techniques, including polymerase cycling assembly from chemically synthesized microarray-derived oligonucleotides. Here we apply Data-optimized Assembly Design to the synthesis of hundreds of codon-optimized genes in both constitutive and inducible vectors using Golden Gate Assembly. Starting from oligonucleotide pools, we synthesize genes in three simple steps: 1) Amplification of parts belonging to individual assemblies in parallel from a single pool; 2) Golden Gate Assembly of parts for each construct; and 3) Transformation. We construct genes from receiving DNA to sequence confirmed isolates in as little as 4 days. By leveraging the ligation fidelity afforded by T4 DNA ligase, we expect to be able to construct a larger breadth of sequences not currently supported by homology-based methods which require stability of extensive single-stranded DNA overhangs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.03.543535">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.03.543535" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.03.543535">
        <p class="paperTitle">Autotrophic growth of E. coli is achieved by a small number of genetic changes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.03.543535" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.03.543535" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ben Nissan, R.; Milshtein, E.; Pahl, V.; de Pins, B.; Jona, G.; Levi, D.; Yung, H.; Nir, N.; Ezra, D.; Gleizer, S.; Link, H.; Noor, E.; Milo, R.</p>
        <p class="info">Score: 56.9, Published: 2023-11-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.03.543535' target='https://doi.org/10.1101/2023.06.03.543535'> 10.1101/2023.06.03.543535</a></p>
        <p class="abstract">Synthetic autotrophy is a promising avenue to sustainable bioproduction from CO2. Here, we use iterative laboratory evolution to generate several distinct autotrophic strains. Utilising this genetic diversity, we identify that just three mutations are sufficient for E. coli to grow autotrophically, when introduced alongside non-native energy (formate dehydrogenase) and carbon-fixing (RuBisCO, phosphoribulokinase, carbonic anhydrase) modules. The mutated genes are involved in glycolysis (pgi), central-carbon regulation (crp), and RNA transcription (rpoB). The pgi mutation reduces the enzymes activity, thereby stabilising the carbon-fixing cycle by capping a major branching flux. For the other two mutations, we observe down-regulation of several metabolic pathways and increased expression of native genes associated with the carbon-fixing module (rpiB) and the energy module (fdoGH), as well as an increased ratio of NADH/NAD&#43; - the cycles electron-donor. This study demonstrates the malleability of metabolism and its capacity to switch trophic modes using only a small number of genetic changes and could facilitate transforming other heterotrophic organisms into autotrophs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561409">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561409" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561409">
        <p class="paperTitle">Bacteriophage-host interactions in microgravity onboard the International Space Station</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561409" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561409" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huss, P.; Chitboonthavisuk, C.; Meger, A.; Nishikawa, K.; Oates, R. P.; Mills, H.; Holzhaus, O.; Raman, S.</p>
        <p class="info">Score: 21.4, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561409' target='https://doi.org/10.1101/2023.10.10.561409'> 10.1101/2023.10.10.561409</a></p>
        <p class="abstract">Bacteriophage-host interactions play a fundamental role in shaping microbial ecosystems. Although researchers have extensively studied phage-host interactions in terrestrial ecosystems, the impact of microgravity on phage-host interactions is yet to be investigated. Here, we report the dynamics of interactions between T7 bacteriophage and E. coli in microgravity onboard the International Space Station (ISS). We found phage activity was delayed but ultimately successful in microgravity. We identified several de novo mutations in phage and bacteria that improved fitness in microgravity. Deep mutational scanning of the receptor binding domain revealed substantial differences in the number, position, and mutational preferences between gravity and microgravity, reflecting underlying differences in the bacterial adaptations. Combinatorial libraries informed by microgravity selections gave T7 mutants with 100-10,000-fold higher activity on uropathogenic E. coli under terrestrial conditions than wildtype T7. Our findings lay the foundation for future research on the impact of microgravity on phage-host interactions and microbial communities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567881">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567881" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567881">
        <p class="paperTitle">YeastIT: Reducing mutational bias for in vivo directed evolution using a novel yeast mutator strain based on dual adenine-/cytosine-targeting and error-prone DNA repair</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567881" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567881" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Napiorkowska, M.; Fischer, K.; Penner, M.; Knyphausen, P.; Hollfelder, F.</p>
        <p class="info">Score: 7.7, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567881' target='https://doi.org/10.1101/2023.11.20.567881'> 10.1101/2023.11.20.567881</a></p>
        <p class="abstract">Engineering proteins with new functions and properties often requires navigating large sequence spaces through rounds of iterative improvement. However, a disparity exists between the gradual pace of natural long-term evolution and a typical laboratory evolution workflow that relies on enriching functional variants from highly diverse in vitro generated libraries through very few screening rounds. Laboratory experiments often eschew presumed natural strategies such as neutral/non-adaptive and multi-phase evolution trajectories, and therefore mutagenesis technologies suitable for long  nature-like timescales are needed. Here, we introduce YeastIT, a novel in vivo mutagenesis tool for protein engineering that leverages an S. cerevisiae strain engineered to exhibit mutagenic activity directed to the gene of interest, allowing its continuous diversification. Mutagenesis is achieved by generating DNA damage through nucleoside deamination, followed by introduction of mutations by harnessing the process of error-prone DNA translesion synthesis. By eliminating the transformation step, YeastIT allows multiple rounds of screening or selection without interruptions for library diversification, thereby enabling long-term and continuous evolution campaigns. Our characterization of the mutational spectrum and frequency of the YeastIT-generated libraries, and its comparison to other methods (error-prone PCR, PACE, MutaT7, eMutaT7, OrthoRep, TRIDENT, EvolVR) demonstrates comparable mutation rates combined with a significant reduction in mutagenic bias relative to most of the alternatives. To validate YeastIT, we carried out directed evolution of a DARPin binding protein to achieve a 15-fold improved affinity. YeastIT thus provides a tool for exploring different evolutionary trajectories which overcomes previous limitations of variant availability (due to bias and low mutation rates) and emulates the way proteins emerge in Nature.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568191">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568191" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568191">
        <p class="paperTitle">dFLASH; dual FLuorescent transcription factor Activity Sensor for Histone integrated live-cell reporting and high-content screening</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568191" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568191" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Allen, T. P.; Roennfeldt, A. E.; Reckdharajkumar, M.; Liu, M.; Quinn, R.; Russell, D. L.; Peet, D. J.; Whitelaw, M. L.; Bersten, D. C.</p>
        <p class="info">Score: 5.8, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568191' target='https://doi.org/10.1101/2023.11.21.568191'> 10.1101/2023.11.21.568191</a></p>
        <p class="abstract">Live-cell reporting of regulated transcription factor (TF) activity has a wide variety of applications in synthetic biology, drug discovery, and functional genomics. As a result, there is high value in the generation of versatile, sensitive, robust systems that can function across a range of cell types and be adapted toward diverse TF classes. Here we present the dual FLuorescent transcription factor Activity Sensor for Histone integrated live-cell reporting (dFLASH), a modular sensor for TF activity that can be readily integrated into cellular genomes. We demonstrate readily modified dFLASH platforms that homogenously, robustly, and specifically sense regulation of endogenous Hypoxia Inducible Factor (HIF) and Progesterone receptor (PGR) activities, as well as regulated coactivator recruitment to a synthetic DNA-Binding Domain-Activator Domain fusion proteins. The dual-colour nuclear fluorescence produced normalised dynamic live-cell TF activity sensing with facile generation of high-content screening lines with strong signal:noise ratios and reproducible screening capabilities (Z = 0.68-0.74). Finally, we demonstrate the utility of this platform for functional genomics applications by using CRISPRoff to modulate the HIF regulatory pathway, and utility for drug screening by using high content imaging in a bimodal design to isolate activators and inhibitors of the HIF pathway from a [~]1600 natural product library.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566922">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566922" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566922">
        <p class="paperTitle">Continuous evolution of user-defined genes at 1-million-times the genomic mutation rate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566922" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566922" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rix, G.; Williams, R. L.; Spinner, H.; Hu, V. J.; Marks, D. S.; Liu, C. C.</p>
        <p class="info">Score: 58.5, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566922' target='https://doi.org/10.1101/2023.11.13.566922'> 10.1101/2023.11.13.566922</a></p>
        <p class="abstract">When nature maintains or evolves a genes function over millions of years at scale, it produces a diversity of homologous sequences whose patterns of conservation and change contain rich structural, functional, and historical information about the gene. However, natural gene diversity likely excludes vast regions of functional sequence space and includes phylogenetic and evolutionary eccentricities, limiting what information we can extract. We introduce an accessible experimental approach for compressing long-term gene evolution to laboratory timescales, allowing for the direct observation of extensive adaptation and divergence followed by inference of structural, functional, and environmental constraints for any selectable gene. To enable this approach, we developed a new orthogonal DNA replication (OrthoRep) system that durably hypermutates chosen genes at a rate of &gt;10-4 substitutions per base in vivo. When OrthoRep was used to evolve a conditionally essential maladapted enzyme, we obtained thousands of unique multi-mutation sequences with many pairs &gt;60 amino acids apart (&gt;15% divergence), revealing known and new factors influencing enzyme adaptation. The fitness of evolved sequences was not predictable by advanced machine learning models trained on natural variation. We suggest that OrthoRep supports the prospective and systematic discovery of constraints shaping gene evolution, uncovering of new regions in fitness landscapes, and general applications in biomolecular engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567117">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567117" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567117">
        <p class="paperTitle">Directed Evolution of Genetically Encoded LYTACs for Cell-Mediated Delivery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567117" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567117" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, J. L.; Yamada-Hunter, S. A.; Labanieh, L.; Sotillo, E.; Cheah, J. S.; Roberts, D. S.; Mackall, C.; Ting, A. Y.; Bertozzi, C.</p>
        <p class="info">Score: 21.9, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567117' target='https://doi.org/10.1101/2023.11.14.567117'> 10.1101/2023.11.14.567117</a></p>
        <p class="abstract">Lysosome-targeting chimeras (LYTACs) are a promising therapeutic modality to drive the degradation of extracellular proteins. However, early versions of LYTAC contain synthetic glycopeptides that cannot be genetically encoded. Here we present our designs for a fully genetically encodable LYTAC (GELYTAC), making our tool compatible with integration into therapeutic cells for targeted delivery at diseased sites. To achieve this, we replaced the glycopeptide portion of LYTACs with the protein insulin like growth factor 2 (IGF2). After showing initial efficacy with wild type IGF2, we increased the potency of GELYTAC using directed evolution. Subsequently, we demonstrated that our engineered GELYTAC construct not only secretes from HEK293T cells but also from human primary T-cells to drive the uptake of various targets into receiver cells. Immune cells engineered to secrete GELYTAC thus represent a promising avenue for spatially-selective targeted protein degradation.

Significance StatementBetter therapeutic windows can be achieved by targeting therapeutics to their desired sites of action. For protein therapeutics, this might be achieved by engineering cell therapies that home to a tissue of interest and secrete the biologic drug locally. Here, we demonstrate that human primary T cells can be engineered to produce genetically encoded lysosome targeting chimeras (GELYTACs). These GELYTACs mediate the degradation of extracellular proteins associated with cancer progression. Thus, cells engineered to produce GELYTACs represent a potential new class of cancer therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569005">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569005" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569005">
        <p class="paperTitle">High-yield enzymatic synthesis of mono- and trifluorinated alanine enantiomers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569005" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569005" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nieto-Dominguez, M.; Sako, A.; Enemark-Rasmussen, K.; Held Gotfredsen, C.; Rago, D.; Nikel, P. I.</p>
        <p class="info">Score: 2.0, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569005' target='https://doi.org/10.1101/2023.11.28.569005'> 10.1101/2023.11.28.569005</a></p>
        <p class="abstract">Fluorinated amino acids are a promising entry point for incorporating new-to-Nature chemistries in biological systems. Hence, novel methods are needed for the selective synthesis of these building blocks. In this study, we focused on the enzymatic synthesis of fluorinated alanine enantiomers. To this end, the alanine dehydrogenase from Vibrio proteolyticus and the diaminopimelate dehydrogenase from Symbiobacterium thermophilum were applied to the in vitro production of (R)-3-fluoroalanine and (S)-3-fluoroalanine, respectively, using 3-fluoropyruvate as the substrate. Additionally, an alanine racemase from Streptomyces lavendulae, originally selected for setting an alternative enzymatic cascade leading to the production of these non-canonical amino acids, had an unprecedented catalytic efficiency in the beta-elimination of fluorine from the monosubstituted fluoroalanine. The in vitro enzymatic cascade based on the dehydrogenases of V. proteolyticus and S. thermophilum included a cofactor recycling system, whereby a formate dehydrogenase from Pseudomonas sp. 101 (either native or engineered) coupled formate oxidation to NAD(P)H formation. Under these conditions, the reaction yields for (R)-3-fluoroalanine and (S)-3-fluoroalanine reached &gt;85% on the fluorinated substrate and proceeded with complete enantiomeric excess. Moreover, the selected dehydrogenases were also able to catalyze the conversion of trifluoropyruvate into trifluorinated alanine, as a first-case example of biocatalysis with amino acids carrying a trifluoromethyl group.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.568972">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.568972" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.568972">
        <p class="paperTitle">A library of reporters of the global regulators of gene expression of Escherichia coli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.568972" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.568972" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dash, S.; Jagadeesan, R.; Baptista, I. S. C.; Chauhan, V.; Kandavalli, V.; Oliveira, S. M. D.; Ribeiro, A. S.</p>
        <p class="info">Score: 1.2, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.568972' target='https://doi.org/10.1101/2023.11.28.568972'> 10.1101/2023.11.28.568972</a></p>
        <p class="abstract">The topology of the transcription factor network (TFN) of E. coli is far from uniform, with 22 global regulator (GR) proteins controlling one-third of all genes. So far, their production rates cannot be tracked by comparable fluorescent proteins. We developed a library of fluorescent reporters for 16 GRs for this purpose. Each consists of a single-copy plasmid coding for GFP fused to the full-length copy of the native promoter. We tracked their activity in exponential and stationary growth, as well as under weak and strong stresses. We show that the reporters have high sensitivity and specificity to all stresses tested and detect single-cell variability in transcription rates. Given the influence of GRs on the TFN, we expect that the new library will contribute to dissecting global transcriptional stress-response programs of E. coli. Moreover, the library can be invaluable in bioindustrial applications that tune those programs to, instead of cell growth, favor productivity while reducing energy consumption.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
