<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="De novo-designed minibinders expand the synthetic biology sensing repertoire
Authors: Weinberg, Z. Y.; Soliman, S. S.; Kim, M. S.; Chen, I. P.; Ott, M.; El-Samad, H.
Score: 32.0, Published: 2024-01-15 DOI: 10.1101/2024.01.12.575267
Synthetic and chimeric receptors capable of recognizing and responding to user-defined antigens have enabled &#34;smart&#34; therapeutics based on engineered cells. These cell engineering tools depend on antigen sensors which are most often derived from antibodies. Advances in the de novo design of proteins have enabled the design of protein binders with the potential to target epitopes with unique properties and faster production timelines compared to antibodies.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="De novo-designed minibinders expand the synthetic biology sensing repertoire
Authors: Weinberg, Z. Y.; Soliman, S. S.; Kim, M. S.; Chen, I. P.; Ott, M.; El-Samad, H.
Score: 32.0, Published: 2024-01-15 DOI: 10.1101/2024.01.12.575267
Synthetic and chimeric receptors capable of recognizing and responding to user-defined antigens have enabled &#34;smart&#34; therapeutics based on engineered cells. These cell engineering tools depend on antigen sensors which are most often derived from antibodies. Advances in the de novo design of proteins have enabled the design of protein binders with the potential to target epitopes with unique properties and faster production timelines compared to antibodies." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-17T10:40:22+00:00" />
<meta property="article:modified_time" content="2024-01-17T10:40:22+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="De novo-designed minibinders expand the synthetic biology sensing repertoire
Authors: Weinberg, Z. Y.; Soliman, S. S.; Kim, M. S.; Chen, I. P.; Ott, M.; El-Samad, H.
Score: 32.0, Published: 2024-01-15 DOI: 10.1101/2024.01.12.575267
Synthetic and chimeric receptors capable of recognizing and responding to user-defined antigens have enabled &#34;smart&#34; therapeutics based on engineered cells. These cell engineering tools depend on antigen sensors which are most often derived from antibodies. Advances in the de novo design of proteins have enabled the design of protein binders with the potential to target epitopes with unique properties and faster production timelines compared to antibodies."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "De novo-designed minibinders expand the synthetic biology sensing repertoire\nAuthors: Weinberg, Z. Y.; Soliman, S. S.; Kim, M. S.; Chen, I. P.; Ott, M.; El-Samad, H.\nScore: 32.0, Published: 2024-01-15 DOI: 10.1101/2024.01.12.575267\nSynthetic and chimeric receptors capable of recognizing and responding to user-defined antigens have enabled \u0026#34;smart\u0026#34; therapeutics based on engineered cells. These cell engineering tools depend on antigen sensors which are most often derived from antibodies. Advances in the de novo design of proteins have enabled the design of protein binders with the potential to target epitopes with unique properties and faster production timelines compared to antibodies.",
  "keywords": [
    
  ],
  "articleBody": " De novo-designed minibinders expand the synthetic biology sensing repertoire\nAuthors: Weinberg, Z. Y.; Soliman, S. S.; Kim, M. S.; Chen, I. P.; Ott, M.; El-Samad, H.\nScore: 32.0, Published: 2024-01-15 DOI: 10.1101/2024.01.12.575267\nSynthetic and chimeric receptors capable of recognizing and responding to user-defined antigens have enabled \"smart\" therapeutics based on engineered cells. These cell engineering tools depend on antigen sensors which are most often derived from antibodies. Advances in the de novo design of proteins have enabled the design of protein binders with the potential to target epitopes with unique properties and faster production timelines compared to antibodies. Building upon our previous work combining a de novo-designed minibinder of the Spike protein of SARS-CoV-2 with the synthetic receptor synNotch (SARSNotch), we investigated whether minibinders can be readily adapted to a diversity of cell engineering tools. We show that the Spike minibinder LCB1 easily generalizes to a next-generation proteolytic receptor SNIPR that performs similarly to our previously reported SARSNotch. LCB1-SNIPR successfully enables the detection of live SARS-CoV-2, an improvement over SARSNotch which can only detect cell-expressed Spike. To test the generalizability of minibinders to diverse applications, we tested LCB1 as an antigen sensor for a chimeric antigen receptor (CAR). LCB1-CAR enabled CD8+ T cells to cytotoxically target Spike-expressing cells. Our findings suggest that minibinders represent a novel class of antigen sensors that have the potential to dramatically expand the sensing repertoire of cell engineering tools.\nSecreted Particle Information Transfer (SPIT) - A Cellular Platform For In Vivo Genetic Engineering\nAuthors: Charlesworth, C. T.; Homma, S.; Suchy, F.; Wang, S.; Bhadhury, J.; Amaya, A. K.; Camarena, J.; Zhang, J.; Tan, T. K.; Igarishi, K. J.; Nakauchi, H.\nScore: 10.0, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575257\nA multitude of tools now exist that allow us to precisely manipulate the human genome in a myriad of different ways. However, successful delivery of these tools to the cells of human patients remains a major barrier to their clinical implementation. Here we introduce a new cellular approach for in vivo genetic engineering, Secreted Particle Information Transfer (SPIT) that utilizes human cells as delivery vectors for in vivo genetic engineering. We demonstrate the application of SPIT for cell-cell delivery of Cre recombinase and CRISPR-Cas9 enzymes, we show that genetic logic can be incorporated into SPIT and present the first demonstration of human cells as a delivery platform for in vivo genetic engineering in immunocompetent mice. We successfully applied SPIT to genetically modify multiple organs and tissue stem cells in vivo including the liver, spleen, intestines, peripheral blood, and bone marrow. We anticipate that by harnessing the large packaging capacity of a human cells nucleus, the ability of human cells to engraft into patients long term and the capacity of human cells for complex genetic programming, that SPIT will become a paradigm shifting approach for in vivo genetic engineering.\nRecombinant venom proteins in insect seminal fluid reduces female lifespan\nAuthors: Beach, S. J.; Maselko, M.\nScore: 6.3, Published: 2024-01-15 DOI: 10.1101/2024.01.14.575309\nThe emergence of insecticide resistance has increased the need for alternative pest management tools. Numerous genetic biocontrol approaches, which involve the release of genetically modified organisms to control pest populations, are in various stages of development to provide highly targeted pest control. However, all current mating-based genetic biocontrol technologies function by releasing engineered males which skew sex-ratios or reduce offspring viability in subsequent generations. This allows mated females continue to cause harm (e.g. transmit disease). Here, we demonstrate the first example of intragenerational genetic biocontrol, wherein mating with engineered males reduces female lifespan. The toxic male technique (TMT) involves the heterologous expression of insecticidal proteins within the male reproductive tract that are transferred to females via mating. We demonstrate TMT in Drosophila melanogaster males, which reduce the median lifespan of mated females by 37 - 59% compared to controls mated to wild type males. Agent-based models of Aedes aegypti predict that compared to existing genetic biocontrol technologies, even modest levels of mated female mortality could allow TMT to suppress a female population substantially faster, which is likely to result in reduced disease burdens. TMT presents a novel approach for combatting outbreaks of disease vectors and agricultural pests.\nGenome streamlining to improve performance of a fast-growing cyanobacterium Synechococcus elongatus UTEX 2973\nAuthors: Sengupta, A.; Bandyopadhyay, A.; Sarkar, D.; Hendry, J. I.; Schubert, M. G.; Liu, D.; Church, G.; Maranas, C. D.; Pakrasi, H. B.\nScore: 3.8, Published: 2024-01-16 DOI: 10.1101/2024.01.16.575707\nCyanobacteria are photosynthetic organisms that have garnered significant recognition as potential hosts for sustainable bioproduction. However, their complex regulatory networks pose significant challenges to major metabolic engineering efforts, thereby limiting their feasibility as production hosts. Genome streamlining has been demonstrated to be a successful approach for improving productivity and fitness in heterotrophs but is yet to be explored to its full potential in phototrophs. Here we present the systematic reduction of the genome of the cyanobacterium exhibiting the fastest exponential growth, Synechococcus elongatus UTEX 2973. This work, the first of its kind in a photoautotroph, involved an iterative process using state-of-the-art genome-editing technology guided by experimental analysis and computational tools. CRISPR/Cas3 enabled large, progressive deletions of predicted dispensable regions and aided in the identification of essential genes. The large deletions were combined to obtain a strain with 55 kb genome reduction. The strains with streamlined genome showed improvement in growth (up to 23%) and productivity (by 22.7%) as compared to the WT. This streamlining strategy not only has the potential to develop cyanobacterial strains with improved growth and productivity traits but can also facilitate a better understanding of their genome to phenome relationships.\nDevelopment of Int-Plex@ binary memory switch system: plant genome modulation driven by large serine-integrases.\nAuthors: de Oliveira, M. A.; Lima, R. N.; Florentino, L. H.; Severo de Almeida, M. M.; Melo, F. L.; Bonnet, R. V.; Lacorte, C.; Rech, E.\nScore: 1.8, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575089\nThe comprehension of virus-host interactions has allowed numerous advances in developing biotechnological methodologies for plant genome editions, constituting a promising path for plant genetic engineering. Among these advancements, phage- encoded large serine-integrases have emerged as noteworthy tools to modulate plant metabolic pathways by inserting, excising, or inverting DNA stretches in a reversible and specific way. The present work shows the foundation of the Int-Plex@ (INTegrase PLant EXpression) binary memory switch system, which consists of the application of four distinct orthogonal prophage large serine-integrases (Int) (BxB1, phiC31, Int13, and Int9) as an input trigger mechanism for the inversion or excision of genomic DNA. The memory genetic switch is divided into the excision module and the inversion module. The excision module is activated by BxB1 or phiC31 enzymes (input). In this case, the DNA sequence flanked by its attachment sites is excised from the genome (output). The inversion module is activated by Int9 or Int13 (input). The inverted mgf gene sequence is flipped to its functional coding sequence, and the switch output is mGFP. Moreover, prokaryotic-based cell-free in vitro transcription-translation reactions (TxTl) were used as a fast platform for testing Ints attB/P in tandem site activity. Furthermore different plasmid delivery strategies for plant cell Int heterologous expression were tested: leaf tissue agroinfiltration of Agrobacterium tumefaciens transformed with binary plasmids and a biolistic system. After each treatment, the edited genomic DNA sequences were amplified and verified by Sanger and Nanopore sequencing. Despite the challenges of using Ints, the potential benefits are significant and deserve deeper exploration and development. The Int-Plex@ binary genome memory switch system can be applied to produce genetic circuits combined with omics tools and sgRNAs to engineer and modulate plant metabolic pathways temporally and reversibly.\nReconstructing cell histories in space with image-readable base editor recording\nAuthors: Chadly, D. M.; Frieda, K. L.; Gui, C.; Klock, L.; Tran, M.; Sui, M. Y.; Takei, Y.; Bouckaert, R.; Lois, C.; Cai, L.; Elowitz, M. B.\nScore: 103.6, Published: 2024-01-04 DOI: 10.1101/2024.01.03.573434\nKnowing the ancestral states and lineage relationships of individual cells could unravel the dynamic programs underlying development. Engineering cells to actively record information within their own genomic DNA could reveal these histories, but existing recording systems have limited information capacity or disrupt spatial context. Here, we introduce baseMEMOIR, which combines base editing, sequential hybridization imaging, and Bayesian inference to allow reconstruction of high-resolution cell lineage trees and cell state dynamics while preserving spatial organization. BaseMEMOIR stochastically and irreversibly edits engineered dinucleotides to one of three alternative image-readable states. By genomically integrating arrays of editable dinucleotides, we constructed an embryonic stem cell line with 792 bits of recordable, image-readable memory, a 50-fold increase over the state of the art. Simulations showed that this memory size was sufficient for accurate reconstruction of deep lineage trees. Experimentally, baseMEMOIR allowed precise reconstruction of lineage trees 6 or more generations deep in embryonic stem cell colonies. Further, it also allowed inference of ancestral cell states and their quantitative cell state transition rates, all from endpoint images. baseMEMOIR thus provides a scalable framework for reconstructing single cell histories in spatially organized multicellular systems.\nBiology System Description Language (BiSDL): a modeling language for the design of multicellular synthetic biological systems\nAuthors: Giannantoni, L.; Bardini, R.; Savino, A.; Di Carlo, S.\nScore: 1.4, Published: 2024-01-14 DOI: 10.1101/2024.01.13.575499\nBackground: The Biology System Description Language (BiSDL) is an accessible, easy-to-use computational language for multicellular synthetic biology. It allows synthetic biologists to represent spatiality and multi-level cellular dynamics inherent to multicellular designs, filling a gap in the state of the art. Developed for designing and simulating spatial, multicellular synthetic biological systems, BiSDL integrates high-level conceptual design with detailed low-level modeling, fostering collaboration in the Design-Build-Test-Learn cycle. BiSDL descriptions directly compile into Nets-Within-Nets (NWNs) models, offering a unique approach to spatial and hierarchical modeling in biological systems. Results: BiSDL's effectiveness is showcased through two case studies on complex multicellular systems: a bacterial consortium and a synthetic morphogen system. These studies highlight the BiSDL proficiency in representing spatial interactions and multi-level cellular dynamics. The language facilitates the compilation of conceptual designs into detailed, simulatable models, leveraging the NWNs formalism. This enables intuitive modeling of complex biological systems, making advanced computational tools more accessible to a broader range of researchers. Conclusions: BiSDL represents a significant step forward in computational languages for synthetic biology, providing a sophisticated yet user-friendly tool for the design and simulation of complex biological systems with an emphasis on spatiality and cellular dynamics. Its introduction has the potential to transform research and development in synthetic biology, allowing for deeper insights and novel applications in understanding and manipulating multicellular systems.\nUnlocking de novo antibody design with generative artificial intelligence\nAuthors: Shanehsazzadeh, A.; McPartlon, M.; Kasun, G.; Steiger, A. K.; Sutton, J. M.; Yassine, E.; McCloskey, C.; Haile, R.; Shuai, R.; Alverio, J.; Rakocevic, G.; Levine, S.; Cejovic, J.; Gutierrez, J. M.; Morehead, A.; Dubrovskyi, O.; Chung, C.; Luton, B. K.; Diaz, N.; Kohnert, C.; Consbruck, R.; Carter, H.; LaCombe, C.; Bist, I.; Vilaychack, P.; Anderson, Z.; Xiu, L.; Bringas, P.; Alarcon, K.; Knight, B.; Radach, M.; Bateman, K.; Kopec-Belliveau, G.; Chapman, D.; Bennett, J.; Ventura, A. B.; Canales, G. M.; Gowda, M.; Jackson, K. A.; Caguiat, R.; Brown, A.; Ganini da Silva, D.; Guo, Z.; Abdulhaqq, S\nScore: 352.8, Published: 2024-01-07 DOI: 10.1101/2023.01.08.523187\nGenerative AI has the potential to redefine the process of therapeutic antibody discovery. In this report, we describe and validate deep generative models for the de novo design of antibodies against human epidermal growth factor receptor (HER2) without additional optimization. The models enabled an efficient workflow that combined in silico design methods with high-throughput experimental techniques to rapidly identify binders from a library of [~]106 heavy chain complementarity-determining region (HCDR) variants. We demonstrated that the workflow achieves binding rates of 10.6% for HCDR3 and 1.8% for HCDR123 designs and is statistically superior to baselines. We further characterized 421 diverse binders using surface plasmon resonance (SPR), finding 71 with low nanomolar affinity similar to the therapeutic anti-HER2 antibody trastuzumab. A selected subset of 11 diverse high-affinity binders were functionally equivalent or superior to trastuzumab, with most demonstrating suitable developability features. We designed one binder with [~]3x higher cell-based potency compared to trastuzumab and another with improved cross-species reactivity1. Our generative AI approach unlocks an accelerated path to designing therapeutic antibodies against diverse targets.\nSequencing coverage analysis for combinatorialDNA-based storage systems\nAuthors: Preuss, I.; Galili, B.; Yakhini, Z.; anavy, l.\nScore: 0.8, Published: 2024-01-10 DOI: 10.1101/2024.01.10.574966\nThis study introduces a novel model for analyzing and determining the required sequencing coverage in DNA-based data storage, focusing on combinatorial DNA encoding. We explore the application of the coupon collector model for combinatorial-letter reconstruction, post-sequencing, which ensure efficient data retrieval and error reduction. We use a Markov Chain model to compute the probability of error-free reconstruction. We develop theoretical bounds on the decoding probability and use empirical simulations to validate these bounds. The work contributes to the understanding of sequencing coverage in DNA-based data storage, offering insights into decoding complexity, error correction, and sequence reconstruction. We provide a Python package that takes the code design and other message parameters as input, and then computes the required read coverage to guarantee reconstruction at a given desired confidence.\nStacking effects on mutation detection by T4 DNA ligation within dimeric DNA origami triangle barcodes for single-molecule nanopore analysis\nAuthors: Aboagye-Mensah, D.; Confederat, S.; Alammar, F.; Kailas, L.; Adedeji-Olulana, A. F.; Stopar, A.; Nicholson, A. W.; Thomson, N. H.; Actis, P.; Castronovo, M.\nScore: 0.8, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574918\nSolid-state nanopores represent an emerging technology for the highly sensitive detection of biomolecular markers, but the detection of DNA point mutations is challenged by the high noise levels associated with solid-state nanopore reading. In contrast, barcoded DNA origami nanostructures can provide unique single-molecule nanopore fingerprints. In this work, we have integrated nanopore-barcoded DNA nanostructures with enzymatic DNA ligation, the latter of which is routinely involved in clinical protocols for DNA mutation detection. We designed two triangular DNA origami variants containing three elongated staples that provide strands extensions on one side that are complementary to a target sequence. Addition of the latter in solution promotes the formation of a DNA triangle dimer. Since T4 DNA ligase repairs a nick in a dsDNA segment only if there is Watson-Crick base-pairing at the nick, the two DNA triangles can be covalently linked only if the DNA sequence bridging the two triangles carries the targeted mutation. We have found striking differences between ligation detection by gel electrophoresis, AFM, and quartz capillary-based nanopores. The stacking interaction between DNA triangles is enhanced by the formation of dimers, and promote the formation of higher order nanostructure, which serve as molecular weight amplification for DNA ligation in gels. The triangle-triangle stacking dynamics presumably involves a clam-like folding mechanism, which is detectable by quartz nanopore analysis, and which hinders ligation by T4 DNA ligase. The results provide the basis for development of rapid, highly sensitive, and affordable high-throughput approaches for profiling genetic variations in point-of-care settings.\n",
  "wordCount" : "2465",
  "inLanguage": "en",
  "datePublished": "2024-01-17T10:40:22Z",
  "dateModified": "2024-01-17T10:40:22Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 17, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575267">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575267" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575267">
        <p class="paperTitle">De novo-designed minibinders expand the synthetic biology sensing repertoire</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575267" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575267" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weinberg, Z. Y.; Soliman, S. S.; Kim, M. S.; Chen, I. P.; Ott, M.; El-Samad, H.</p>
        <p class="info">Score: 32.0, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575267' target='https://doi.org/10.1101/2024.01.12.575267'> 10.1101/2024.01.12.575267</a></p>
        <p class="abstract">Synthetic and chimeric receptors capable of recognizing and responding to user-defined antigens have enabled &#34;smart&#34; therapeutics based on engineered cells. These cell engineering tools depend on antigen sensors which are most often derived from antibodies. Advances in the de novo design of proteins have enabled the design of protein binders with the potential to target epitopes with unique properties and faster production timelines compared to antibodies. Building upon our previous work combining a de novo-designed minibinder of the Spike protein of SARS-CoV-2 with the synthetic receptor synNotch (SARSNotch), we investigated whether minibinders can be readily adapted to a diversity of cell engineering tools. We show that the Spike minibinder LCB1 easily generalizes to a next-generation proteolytic receptor SNIPR that performs similarly to our previously reported SARSNotch. LCB1-SNIPR successfully enables the detection of live SARS-CoV-2, an improvement over SARSNotch which can only detect cell-expressed Spike. To test the generalizability of minibinders to diverse applications, we tested LCB1 as an antigen sensor for a chimeric antigen receptor (CAR). LCB1-CAR enabled CD8&#43; T cells to cytotoxically target Spike-expressing cells. Our findings suggest that minibinders represent a novel class of antigen sensors that have the potential to dramatically expand the sensing repertoire of cell engineering tools.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575257">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575257" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575257">
        <p class="paperTitle">Secreted Particle Information Transfer (SPIT) - A Cellular Platform For In Vivo Genetic Engineering</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575257" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575257" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Charlesworth, C. T.; Homma, S.; Suchy, F.; Wang, S.; Bhadhury, J.; Amaya, A. K.; Camarena, J.; Zhang, J.; Tan, T. K.; Igarishi, K. J.; Nakauchi, H.</p>
        <p class="info">Score: 10.0, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575257' target='https://doi.org/10.1101/2024.01.11.575257'> 10.1101/2024.01.11.575257</a></p>
        <p class="abstract">A multitude of tools now exist that allow us to precisely manipulate the human genome in a myriad of different ways. However, successful delivery of these tools to the cells of human patients remains a major barrier to their clinical implementation. Here we introduce a new cellular approach for in vivo genetic engineering, Secreted Particle Information Transfer (SPIT) that utilizes human cells as delivery vectors for in vivo genetic engineering. We demonstrate the application of SPIT for cell-cell delivery of Cre recombinase and CRISPR-Cas9 enzymes, we show that genetic logic can be incorporated into SPIT and present the first demonstration of human cells as a delivery platform for in vivo genetic engineering in immunocompetent mice. We successfully applied SPIT to genetically modify multiple organs and tissue stem cells in vivo including the liver, spleen, intestines, peripheral blood, and bone marrow. We anticipate that by harnessing the large packaging capacity of a human cells nucleus, the ability of human cells to engraft into patients long term and the capacity of human cells for complex genetic programming, that SPIT will become a paradigm shifting approach for in vivo genetic engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.575309">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.575309" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.575309">
        <p class="paperTitle">Recombinant venom proteins in insect seminal fluid reduces female lifespan</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.575309" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.575309" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Beach, S. J.; Maselko, M.</p>
        <p class="info">Score: 6.3, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.575309' target='https://doi.org/10.1101/2024.01.14.575309'> 10.1101/2024.01.14.575309</a></p>
        <p class="abstract">The emergence of insecticide resistance has increased the need for alternative pest management tools. Numerous genetic biocontrol approaches, which involve the release of genetically modified organisms to control pest populations, are in various stages of development to provide highly targeted pest control. However, all current mating-based genetic biocontrol technologies function by releasing engineered males which skew sex-ratios or reduce offspring viability in subsequent generations. This allows mated females continue to cause harm (e.g. transmit disease). Here, we demonstrate the first example of intragenerational genetic biocontrol, wherein mating with engineered males reduces female lifespan. The toxic male technique (TMT) involves the heterologous expression of insecticidal proteins within the male reproductive tract that are transferred to females via mating. We demonstrate TMT in Drosophila melanogaster males, which reduce the median lifespan of mated females by 37 - 59% compared to controls mated to wild type males. Agent-based models of Aedes aegypti predict that compared to existing genetic biocontrol technologies, even modest levels of mated female mortality could allow TMT to suppress a female population substantially faster, which is likely to result in reduced disease burdens. TMT presents a novel approach for combatting outbreaks of disease vectors and agricultural pests.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575707">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575707" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575707">
        <p class="paperTitle">Genome streamlining to improve performance of a fast-growing cyanobacterium Synechococcus elongatus UTEX 2973</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575707" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575707" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sengupta, A.; Bandyopadhyay, A.; Sarkar, D.; Hendry, J. I.; Schubert, M. G.; Liu, D.; Church, G.; Maranas, C. D.; Pakrasi, H. B.</p>
        <p class="info">Score: 3.8, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575707' target='https://doi.org/10.1101/2024.01.16.575707'> 10.1101/2024.01.16.575707</a></p>
        <p class="abstract">Cyanobacteria are photosynthetic organisms that have garnered significant recognition as potential hosts for sustainable bioproduction. However, their complex regulatory networks pose significant challenges to major metabolic engineering efforts, thereby limiting their feasibility as production hosts. Genome streamlining has been demonstrated to be a successful approach for improving productivity and fitness in heterotrophs but is yet to be explored to its full potential in phototrophs. Here we present the systematic reduction of the genome of the cyanobacterium exhibiting the fastest exponential growth, Synechococcus elongatus UTEX 2973. This work, the first of its kind in a photoautotroph, involved an iterative process using state-of-the-art genome-editing technology guided by experimental analysis and computational tools. CRISPR/Cas3 enabled large, progressive deletions of predicted dispensable regions and aided in the identification of essential genes. The large deletions were combined to obtain a strain with 55 kb genome reduction. The strains with streamlined genome showed improvement in growth (up to 23%) and productivity (by 22.7%) as compared to the WT. This streamlining strategy not only has the potential to develop cyanobacterial strains with improved growth and productivity traits but can also facilitate a better understanding of their genome to phenome relationships.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575089">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575089" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575089">
        <p class="paperTitle">Development of Int-Plex@ binary memory switch system: plant genome modulation driven by large serine-integrases.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575089" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575089" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de Oliveira, M. A.; Lima, R. N.; Florentino, L. H.; Severo de Almeida, M. M.; Melo, F. L.; Bonnet, R. V.; Lacorte, C.; Rech, E.</p>
        <p class="info">Score: 1.8, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575089' target='https://doi.org/10.1101/2024.01.11.575089'> 10.1101/2024.01.11.575089</a></p>
        <p class="abstract">The comprehension of virus-host interactions has allowed numerous advances in developing biotechnological methodologies for plant genome editions, constituting a promising path for plant genetic engineering. Among these advancements, phage- encoded large serine-integrases have emerged as noteworthy tools to modulate plant metabolic pathways by inserting, excising, or inverting DNA stretches in a reversible and specific way. The present work shows the foundation of the Int-Plex@ (INTegrase PLant EXpression) binary memory switch system, which consists of the application of four distinct orthogonal prophage large serine-integrases (Int) (BxB1, phiC31, Int13, and Int9) as an input trigger mechanism for the inversion or excision of genomic DNA. The memory genetic switch is divided into the excision module and the inversion module. The excision module is activated by BxB1 or phiC31 enzymes (input). In this case, the DNA sequence flanked by its attachment sites is excised from the genome (output). The inversion module is activated by Int9 or Int13 (input). The inverted mgf gene sequence is flipped to its functional coding sequence, and the switch output is mGFP. Moreover, prokaryotic-based cell-free in vitro transcription-translation reactions (TxTl) were used as a fast platform for testing Ints attB/P in tandem site activity. Furthermore different plasmid delivery strategies for plant cell Int heterologous expression were tested: leaf tissue agroinfiltration of Agrobacterium tumefaciens transformed with binary plasmids and a biolistic system. After each treatment, the edited genomic DNA sequences were amplified and verified by Sanger and Nanopore sequencing. Despite the challenges of using Ints, the potential benefits are significant and deserve deeper exploration and development. The Int-Plex@ binary genome memory switch system can be applied to produce genetic circuits combined with omics tools and sgRNAs to engineer and modulate plant metabolic pathways temporally and reversibly.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.573434">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.573434" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.573434">
        <p class="paperTitle">Reconstructing cell histories in space with image-readable base editor recording</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.573434" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.573434" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chadly, D. M.; Frieda, K. L.; Gui, C.; Klock, L.; Tran, M.; Sui, M. Y.; Takei, Y.; Bouckaert, R.; Lois, C.; Cai, L.; Elowitz, M. B.</p>
        <p class="info">Score: 103.6, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.573434' target='https://doi.org/10.1101/2024.01.03.573434'> 10.1101/2024.01.03.573434</a></p>
        <p class="abstract">Knowing the ancestral states and lineage relationships of individual cells could unravel the dynamic programs underlying development. Engineering cells to actively record information within their own genomic DNA could reveal these histories, but existing recording systems have limited information capacity or disrupt spatial context. Here, we introduce baseMEMOIR, which combines base editing, sequential hybridization imaging, and Bayesian inference to allow reconstruction of high-resolution cell lineage trees and cell state dynamics while preserving spatial organization. BaseMEMOIR stochastically and irreversibly edits engineered dinucleotides to one of three alternative image-readable states. By genomically integrating arrays of editable dinucleotides, we constructed an embryonic stem cell line with 792 bits of recordable, image-readable memory, a 50-fold increase over the state of the art. Simulations showed that this memory size was sufficient for accurate reconstruction of deep lineage trees. Experimentally, baseMEMOIR allowed precise reconstruction of lineage trees 6 or more generations deep in embryonic stem cell colonies. Further, it also allowed inference of ancestral cell states and their quantitative cell state transition rates, all from endpoint images. baseMEMOIR thus provides a scalable framework for reconstructing single cell histories in spatially organized multicellular systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.13.575499">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.13.575499" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.13.575499">
        <p class="paperTitle">Biology System Description Language (BiSDL): a modeling language for the design of multicellular synthetic biological systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.13.575499" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.13.575499" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Giannantoni, L.; Bardini, R.; Savino, A.; Di Carlo, S.</p>
        <p class="info">Score: 1.4, Published: 2024-01-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.13.575499' target='https://doi.org/10.1101/2024.01.13.575499'> 10.1101/2024.01.13.575499</a></p>
        <p class="abstract">Background: The Biology System Description Language (BiSDL) is an accessible, easy-to-use computational language for multicellular synthetic biology. It allows synthetic biologists to represent spatiality and multi-level cellular dynamics inherent to multicellular designs, filling a gap in the state of the art. Developed for designing and simulating spatial, multicellular synthetic biological systems, BiSDL integrates high-level conceptual design with detailed low-level modeling, fostering collaboration in the Design-Build-Test-Learn cycle. BiSDL descriptions directly compile into Nets-Within-Nets (NWNs) models, offering a unique approach to spatial and hierarchical modeling in biological systems. Results: BiSDL&#39;s effectiveness is showcased through two case studies on complex multicellular systems: a bacterial consortium and a synthetic morphogen system. These studies highlight the BiSDL proficiency in representing spatial interactions and multi-level cellular dynamics. The language facilitates the compilation of conceptual designs into detailed, simulatable models, leveraging the NWNs formalism. This enables intuitive modeling of complex biological systems, making advanced computational tools more accessible to a broader range of researchers. Conclusions: BiSDL represents a significant step forward in computational languages for synthetic biology, providing a sophisticated yet user-friendly tool for the design and simulation of complex biological systems with an emphasis on spatiality and cellular dynamics. Its introduction has the potential to transform research and development in synthetic biology, allowing for deeper insights and novel applications in understanding and manipulating multicellular systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.08.523187">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.08.523187" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.08.523187">
        <p class="paperTitle">Unlocking de novo antibody design with generative artificial intelligence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.08.523187" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.08.523187" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shanehsazzadeh, A.; McPartlon, M.; Kasun, G.; Steiger, A. K.; Sutton, J. M.; Yassine, E.; McCloskey, C.; Haile, R.; Shuai, R.; Alverio, J.; Rakocevic, G.; Levine, S.; Cejovic, J.; Gutierrez, J. M.; Morehead, A.; Dubrovskyi, O.; Chung, C.; Luton, B. K.; Diaz, N.; Kohnert, C.; Consbruck, R.; Carter, H.; LaCombe, C.; Bist, I.; Vilaychack, P.; Anderson, Z.; Xiu, L.; Bringas, P.; Alarcon, K.; Knight, B.; Radach, M.; Bateman, K.; Kopec-Belliveau, G.; Chapman, D.; Bennett, J.; Ventura, A. B.; Canales, G. M.; Gowda, M.; Jackson, K. A.; Caguiat, R.; Brown, A.; Ganini da Silva, D.; Guo, Z.; Abdulhaqq, S</p>
        <p class="info">Score: 352.8, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.08.523187' target='https://doi.org/10.1101/2023.01.08.523187'> 10.1101/2023.01.08.523187</a></p>
        <p class="abstract">Generative AI has the potential to redefine the process of therapeutic antibody discovery. In this report, we describe and validate deep generative models for the de novo design of antibodies against human epidermal growth factor receptor (HER2) without additional optimization. The models enabled an efficient workflow that combined in silico design methods with high-throughput experimental techniques to rapidly identify binders from a library of [~]106 heavy chain complementarity-determining region (HCDR) variants. We demonstrated that the workflow achieves binding rates of 10.6% for HCDR3 and 1.8% for HCDR123 designs and is statistically superior to baselines. We further characterized 421 diverse binders using surface plasmon resonance (SPR), finding 71 with low nanomolar affinity similar to the therapeutic anti-HER2 antibody trastuzumab. A selected subset of 11 diverse high-affinity binders were functionally equivalent or superior to trastuzumab, with most demonstrating suitable developability features. We designed one binder with [~]3x higher cell-based potency compared to trastuzumab and another with improved cross-species reactivity1. Our generative AI approach unlocks an accelerated path to designing therapeutic antibodies against diverse targets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574966">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574966" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574966">
        <p class="paperTitle">Sequencing coverage analysis for combinatorialDNA-based storage systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574966" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574966" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Preuss, I.; Galili, B.; Yakhini, Z.; anavy, l.</p>
        <p class="info">Score: 0.8, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574966' target='https://doi.org/10.1101/2024.01.10.574966'> 10.1101/2024.01.10.574966</a></p>
        <p class="abstract">This study introduces a novel model for analyzing and determining the required sequencing coverage in DNA-based data storage, focusing on combinatorial DNA encoding. We explore the application of the coupon collector model for combinatorial-letter reconstruction, post-sequencing, which ensure efficient data retrieval and error reduction. We use a Markov Chain model to compute the probability of error-free reconstruction. We develop theoretical bounds on the decoding probability and use empirical simulations to validate these bounds. The work contributes to the understanding of sequencing coverage in DNA-based data storage, offering insights into decoding complexity, error correction, and sequence reconstruction. We provide a Python package that takes the code design and other message parameters as input, and then computes the required read coverage to guarantee reconstruction at a given desired confidence.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574918">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574918" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574918">
        <p class="paperTitle">Stacking effects on mutation detection by T4 DNA ligation within dimeric DNA origami triangle barcodes for single-molecule nanopore analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574918" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574918" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aboagye-Mensah, D.; Confederat, S.; Alammar, F.; Kailas, L.; Adedeji-Olulana, A. F.; Stopar, A.; Nicholson, A. W.; Thomson, N. H.; Actis, P.; Castronovo, M.</p>
        <p class="info">Score: 0.8, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574918' target='https://doi.org/10.1101/2024.01.09.574918'> 10.1101/2024.01.09.574918</a></p>
        <p class="abstract">Solid-state nanopores represent an emerging technology for the highly sensitive detection of biomolecular markers, but the detection of DNA point mutations is challenged by the high noise levels associated with solid-state nanopore reading. In contrast, barcoded DNA origami nanostructures can provide unique single-molecule nanopore fingerprints. In this work, we have integrated nanopore-barcoded DNA nanostructures with enzymatic DNA ligation, the latter of which is routinely involved in clinical protocols for DNA mutation detection. We designed two triangular DNA origami variants containing three elongated staples that provide strands extensions on one side that are complementary to a target sequence. Addition of the latter in solution promotes the formation of a DNA triangle dimer. Since T4 DNA ligase repairs a nick in a dsDNA segment only if there is Watson-Crick base-pairing at the nick, the two DNA triangles can be covalently linked only if the DNA sequence bridging the two triangles carries the targeted mutation. We have found striking differences between ligation detection by gel electrophoresis, AFM, and quartz capillary-based nanopores. The stacking interaction between DNA triangles is enhanced by the formation of dimers, and promote the formation of higher order nanostructure, which serve as molecular weight amplification for DNA ligation in gels. The triangle-triangle stacking dynamics presumably involves a clam-like folding mechanism, which is detectable by quartz nanopore analysis, and which hinders ligation by T4 DNA ligase. The results provide the basis for development of rapid, highly sensitive, and affordable high-throughput approaches for profiling genetic variations in point-of-care settings.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
