<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Efficient Natural Plasmid Transformation of Vibrio natriegens Enables Zero-capital Molecular Biology
Authors: Specht, D. A.; Sheppard, T. J.; Kennedy, F.; Li, S.; Gadikota, G.; Barstow, B.
Score: 40.1, Published: 2023-08-13 DOI: 10.1101/2023.08.11.553013
The fast-growing microbe Vibrio natriegens is capable of natural transformation where it draws DNA in from media via an active process under physiological conditions. Using an engineered strain with a genomic copy of the master competence regulator tfoX from Vibrio cholera in combination with a new minimal competence media (MCM) that uses acetate as an energy source, we demonstrate naturally competent cells which are created, transformed, and recovered entirely in the same media, without exchange or addition of new media.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Efficient Natural Plasmid Transformation of Vibrio natriegens Enables Zero-capital Molecular Biology
Authors: Specht, D. A.; Sheppard, T. J.; Kennedy, F.; Li, S.; Gadikota, G.; Barstow, B.
Score: 40.1, Published: 2023-08-13 DOI: 10.1101/2023.08.11.553013
The fast-growing microbe Vibrio natriegens is capable of natural transformation where it draws DNA in from media via an active process under physiological conditions. Using an engineered strain with a genomic copy of the master competence regulator tfoX from Vibrio cholera in combination with a new minimal competence media (MCM) that uses acetate as an energy source, we demonstrate naturally competent cells which are created, transformed, and recovered entirely in the same media, without exchange or addition of new media." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-23T10:37:56+00:00" />
<meta property="article:modified_time" content="2023-08-23T10:37:56+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Efficient Natural Plasmid Transformation of Vibrio natriegens Enables Zero-capital Molecular Biology
Authors: Specht, D. A.; Sheppard, T. J.; Kennedy, F.; Li, S.; Gadikota, G.; Barstow, B.
Score: 40.1, Published: 2023-08-13 DOI: 10.1101/2023.08.11.553013
The fast-growing microbe Vibrio natriegens is capable of natural transformation where it draws DNA in from media via an active process under physiological conditions. Using an engineered strain with a genomic copy of the master competence regulator tfoX from Vibrio cholera in combination with a new minimal competence media (MCM) that uses acetate as an energy source, we demonstrate naturally competent cells which are created, transformed, and recovered entirely in the same media, without exchange or addition of new media."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Efficient Natural Plasmid Transformation of Vibrio natriegens Enables Zero-capital Molecular Biology\nAuthors: Specht, D. A.; Sheppard, T. J.; Kennedy, F.; Li, S.; Gadikota, G.; Barstow, B.\nScore: 40.1, Published: 2023-08-13 DOI: 10.1101/2023.08.11.553013\nThe fast-growing microbe Vibrio natriegens is capable of natural transformation where it draws DNA in from media via an active process under physiological conditions. Using an engineered strain with a genomic copy of the master competence regulator tfoX from Vibrio cholera in combination with a new minimal competence media (MCM) that uses acetate as an energy source, we demonstrate naturally competent cells which are created, transformed, and recovered entirely in the same media, without exchange or addition of new media.",
  "keywords": [
    
  ],
  "articleBody": " Efficient Natural Plasmid Transformation of Vibrio natriegens Enables Zero-capital Molecular Biology\nAuthors: Specht, D. A.; Sheppard, T. J.; Kennedy, F.; Li, S.; Gadikota, G.; Barstow, B.\nScore: 40.1, Published: 2023-08-13 DOI: 10.1101/2023.08.11.553013\nThe fast-growing microbe Vibrio natriegens is capable of natural transformation where it draws DNA in from media via an active process under physiological conditions. Using an engineered strain with a genomic copy of the master competence regulator tfoX from Vibrio cholera in combination with a new minimal competence media (MCM) that uses acetate as an energy source, we demonstrate naturally competent cells which are created, transformed, and recovered entirely in the same media, without exchange or addition of new media. Cells are naturally competent to plasmids, recombination with linear DNA, and co-transformation of both to select for scarless and markerless genomic edits. The entire process is simple and inexpensive, requiring no capital equipment for an entirely room temperature process (Zero Capital protocol, 104 cfu/{micro}g), or just an incubator (High Efficiency protocol, 105-6 cfu/{micro}g). These cells retain their naturally competent state when frozen and are transformable immediately upon thawing like a typical chemical or electrochemical competent cell. Since the optimized transformation protocol requires only 50 minutes of hands-on time, and V. natriegens grows quickly even on plates, a transformation started at 9 AM yields abundant culturable single colonies by 5 PM. Further, because all stages of transformation occur in the same media, and the process can be arbitrarily scaled in volume, this natural competence strain and media could be ideal for automated directed evolution applications. As a result, naturally competent V. natriegens could compete with E. coli as an excellent chassis for low-cost and highly scalable synthetic biology.\nGene drive and genetic sex conversion in the global agricultural pest Ceratitis capitata\nAuthors: Meccariello, A.; Hou, S.; Davydova, S.; Fawcett, J.; Siddall, A.; Leftwich, P. T.; Krsticevic, F.; Papathanos, P. A.; Windbichler, N.\nScore: 15.4, Published: 2023-08-17 DOI: 10.1101/2023.08.16.553191\nHoming-based gene drives are novel interventions promising the area-wide, species-specific genetic control of harmful insect populations. Here we characterise a first set of gene drives in a tephritid agricultural pest species, the Mediterranean fruit fly Ceratitis capitata (medfly). Our results show that the medfly is highly amenable to homing-based gene drive strategies. By targeting the medfly transformer gene, we also demonstrate two different mechanisms by which CRISPR-Cas9 gene drive can be coupled to sex conversion, whereby genetic females are transformed into fertile and harmless XX males. Given this unique malleability of sex determination, we modelled gene drive interventions that couple sex conversion and female sterility and found that such approaches could be effective and tolerant of resistant allele selection in the target population. Our results open the door for developing gene drive strains for the population suppression of the medfly and related tephritid pests by co-targeting female reproduction and shifting the reproductive sex ratio towards males. They demonstrate the untapped potential for gene drives to tackle agricultural pests in an environmentally friendly and economical way.\nA molecular proximity sensor based on an engineered, dual-component guide RNA\nAuthors: Choi, J.; Chen, W.; Liao, H.; Li, X.; Shendure, J.\nScore: 36.8, Published: 2023-08-14 DOI: 10.1101/2023.08.14.553235\nOne of the goals of synthetic biology is to enable the design of arbitrary molecular circuits with programmable inputs and outputs. Such circuits bridge the properties of electronic and natural circuits, processing information in a predictable manner within living cells. Genome editing is a potentially powerful component of synthetic molecular circuits, whether for modulating the expression of a target gene or for stably recording information to genomic DNA. However, programming molecular events such as protein-protein interactions or induced proximity as triggers for genome editing remains challenging. Here we demonstrate a strategy termed P3 editing, which links protein-protein proximity to the formation of a functional CRISPR-Cas9 dual-component guide RNA. By engineering the crRNA:tracrRNA interaction, we demonstrate that various known protein-protein interactions, as well as the chemically-induced dimerization of protein domains, can be used to activate prime editing or base editing in human cells. Additionally, we explore how P3 editing can incorporate outputs from ADAR-based RNA sensors, potentially allowing specific RNAs to induce specific genome edits within a larger circuit. Our strategy enhances the controllability of CRISPR-based genome editing, facilitating its use in synthetic molecular circuits deployed in living cells.\nTargeting mosquito X-chromosomes reveals complex transmission dynamics of sex ratio distorting gene drives\nAuthors: Haber, D. A.; Arien, Y.; Lamdan, L. B.; Alcalay, Y.; Zecharia, C.; Krsticevic, F.; Yonah, E. S.; Avraham, R. D.; Windbichler, N.; Papathanos, P. A.\nScore: 11.1, Published: 2023-08-20 DOI: 10.1101/2023.08.20.554004\nEngineered sex ratio distorters (SRDs) have been proposed as a powerful component of both self-limiting and self-sustaining genetic control strategies designed to suppress harmful insect pest or disease vector populations such as malaria mosquitoes. Two different CRISPR-based SRD mechanisms, X-shredding which eliminates X-bearing sperm in males and X-poisoning which causes postzygotic mortality of daughters inheriting disrupted X-linked genes, have been proposed. The nature of the sex distortion mechanism is predicted to affect the population dynamics of released Y-chromosome SRDs: A Y-linked X-shredder is invasive constituting a classical meiotic Y-drive, whereas Y-linked X-poisoning is self-limiting, unable to invade but also insulated from selection and disappearance. Here, we established the first X-poisoning SRD system in the malaria vector Anopheles gambiae targeting three X-linked ribosomal protein (RP) genes during spermatogenesis. Surprisingly, when we characterized transgenic strains showing a significant male bias, we found that sex distortion was primarily driven by a loss of X-bearing sperm with limited evidence for postzygotic lethality of daughters. By leveraging the Drosophila model, we show unambiguously that engineered SRD traits can operate differently in these two insects. Unlike X-shredding, X-poisoning could theoretically operate at early stages of spermatogenesis. We therefore explored premeiotic Cas9 expression to target the mosquito X-chromosome. We find that, by pre-empting the onset of meiotic sex chromosome inactivation, this approach may enable the development of Y-linked SRDs, once trade-offs between sex chromosome transmission distortion and male fertility are functionally balanced.\nOne-pot chemo-enzymatic synthesis and one-step recovery of homogeneous long-chain polyphosphates from microalgal biomass\nAuthors: Lin, Y.-H.; Nishikawa, S.; Jia, T. Z.; Yeh, F.-I.; Khusnutdinova, A.; Yakunin, A. F.; Fujishima, K.; Wang, P.-H.\nScore: 8.1, Published: 2023-08-20 DOI: 10.1101/2023.08.19.553819\nPhosphate, an essential component of life, fertilizers, and detergents, is a finite resource that could be depleted within 70 years, while improper phosphate waste disposal in aquatic environments results in eutrophication. Despite some chemical-based methods, biological phosphorus removal using polyphosphate-accumulating organisms, such as microalgae, serves as a sustainable alternative to reclaim phosphate from wastewater. Polyphosphates have profound biological functions and biomedical applications, serving as energy stock, coagulation factors, and antiviral agents depending on their length, showing inherent value in polyphosphate recovery. Here, we leveraged the power of thermodynamic coupling and phase transitions to establish a one-pot, two-step multi-enzyme cascade to convert polydisperse polyphosphate in microalgae biomass into high-molecular-weight insoluble long-chain polyphosphates, allowing for one-step purification. We then optimzed a thermo-digestion approach to transform the 1,300-mers into shorter polyphosphates. Altogether, the processes established here enable the establishment of a sustainable P bioeconomy platform to refine microalgal biomass for biotechnological uses.\nDesigned Endocytosis-Triggering Proteins mediate Targeted Degradation and amplify signaling\nAuthors: Huang, B.; Abedi, M.; Ahn, G.; Coventry, B.; Sappington, I.; Wang, R.; Schlichthaerle, T.; Zhang, J. Z.; Wang, Y.; Goreshnik, I.; Chiu, C. W.; Chazin-Gray, A.; Chan, S.; Gerben, S.; Murray, A.; Wang, S.; O'Neill, J.; Yeh, R.; Misquith, A.; Wolf, A.; Tomasovic, L. M.; Piraner, D. I.; Gonzalez, M. J. D.; Bennett, N. R.; Venkatesh, P.; Ahlrichs, M.; Dobbins, C.; Yang, W.; Wang, X.; Vafeados, D.; Mout, R.; Shivaei, S.; Cao, L.; Carter, L.; Stewart, L.; Spangler, J. B.; Bernardes, G. J. L.; Roybal, K. T.; Greisen, P. J.; Li, X.; Bertozzi, C.; Baker, D.\nScore: 6.8, Published: 2023-08-21 DOI: 10.1101/2023.08.19.553321\nEndocytosis and lysosomal trafficking of cell surface receptors can be triggered by interaction with endogenous ligands. Therapeutic approaches such as LYTAC1,2 and KineTAC3, have taken advantage of this to target specific proteins for degradation by fusing modified native ligands to target binding proteins. While powerful, these approaches can be limited by possible competition with the endogenous ligand(s), the requirement in some cases for chemical modification that limits genetic encodability and can complicate manufacturing, and more generally, there may not be natural ligands which stimulate endocytosis through a given receptor. Here we describe general protein design approaches for designing endocytosis triggering binding proteins (EndoTags) that overcome these challenges. We present EndoTags for the IGF-2R, ASGPR, and Transferrin receptors, and show that fusing these tags to proteins which bind to soluble or transmembrane protein leads to lysosomal trafficking and target degradation; as these receptors have different tissue distributions, the different EndoTags could enable targeting of degradation to different tissues. The modularity and genetic encodability of EndoTags enables AND gate control for higher specificity targeted degradation, and the localized secretion of degraders from engineered cells. The tunability and modularity of our genetically encodable EndoTags should contribute to deciphering the relationship between receptor engagement and cellular trafficking, and they have considerable therapeutic potential as targeted degradation inducers, signaling activators for endocytosis-dependent pathways, and cellular uptake inducers for targeted antibody drug and RNA conjugates.\nA chemical reaction network model of PURE\nAuthors: Jurado, Z.; Pandey, A.; Murray, R. M.\nScore: 5.0, Published: 2023-08-14 DOI: 10.1101/2023.08.14.553301\nCell-free expression systems provide a method for rapid DNA circuit prototyping and functional protein synthesis. While crude extracts remain a black box with many components carrying out unknown reactions, the PURE system contains only the required transcription and translation components for protein production. All proteins and small molecules are at known concentrations, opening up the possibility of detailed modeling for reliable computational predictions. However, there is little to no experimental data supporting the expression of target proteins for detailed protein models PURE models. In this work, we build a chemical reaction network transcription model for PURE protein synthesis. We compare the transcription models using DNA encoding for the malachite-green aptamer (MGapt) to measure mRNA production. Furthermore, we expand the PURE detailed translation model for an arbitrary set of amino acids and length. Lastly, we combine the transcription and the expanded translation models to create a PURE protein synthesis model built purely from mass-action reactions. We use the combined model to capture the translation of a plasmid encoding MGapt and deGFP under a T7-promoter and a strong RBS. The model accurately predicts the MGapt mRNA production for the first two hours, the dynamics of deGFP expression, and the total protein production with an accuracy within 10 %.\nMachine-guided design of synthetic cell type-specific cis-regulatory elements\nAuthors: Gosai, S. J.; Castro, R. I.; Fuentes, N.; Butts, J. C.; Kales, S.; Noche, R. R.; Mouri, K.; Sabeti, P. C.; Reilly, S. K.; Tewhey, R.\nScore: 60.9, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552077\nCis-regulatory elements (CREs) control gene expression, orchestrating tissue identity, developmental timing, and stimulus responses, which collectively define the thousands of unique cell types in the body. While there is great potential for strategically incorporating CREs in therapeutic or biotechnology applications that require tissue specificity, there is no guarantee that an optimal CRE for an intended purpose has arisen naturally through evolution. Here, we present a platform to engineer and validate synthetic CREs capable of driving gene expression with programmed cell type specificity. We leverage innovations in deep neural network modeling of CRE activity across three cell types, efficient in silico optimization, and massively parallel reporter assays (MPRAs) to design and empirically test thousands of CREs. Through in vitro and in vivo validation, we show that synthetic sequences outperform natural sequences from the human genome in driving cell type-specific expression. Synthetic sequences leverage unique sequence syntax to promote activity in the on-target cell type and simultaneously reduce activity in off-target cells. Together, we provide a generalizable framework to prospectively engineer CREs and demonstrate the required literacy to write regulatory code that is fit-for-purpose in vivo across vertebrates.\nCell-Free Protein Expression in Polymer Materials\nAuthors: Lee, M. F. S.; Lee, J. A.; Biondo, J. R.; Lux, J. E.; Raig, R. M.; Berger, P. N.; Bernhards, C. B.; Kuhn, D. L.; Gupta, M. K.; Lux, M. W.\nScore: 4.0, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553881\nWhile synthetic biology has advanced complex capabilities like sensing and molecule synthesis in aqueous solutions, important applications may also be pursued for biological systems in solid materials. Harsh processing conditions used to produce many synthetic materials such as plastics makes incorporation of biological functionality challenging. One technology that shows promise in circumventing these issues is cell-free protein synthesis (CFPS), where core cellular functionality is reconstituted outside the cell. CFPS enables genetic functions to be implemented without the complications of membrane transport or concerns over cellular viability or release of genetically modified organisms. Here we demonstrate that dried CFPS reactions have remarkable tolerance to heat and organic solvent exposure during the casting processes for polymer materials. We demonstrate the utility of this observation by creating plastics that have spatially patterned genetic functionality, produce antimicrobials in situ, and perform sensing reactions. The resulting materials unlock the potential to deliver DNA-programmable bio-functionality in a ubiquitous class of synthetic materials.\nManipulating the 3D organization of the largest synthetic yeast chromosome\nAuthors: Zhang, W.; Stefanita, L. L.; Yamashita, H.; Shen, M. J.; Mitchell, L. A.; Kurasawa, H.; Lobzaev, E.; Fanfani, V.; Haase, M. A. B.; Sun, X.; Jiang, Q.; Goldberg, G. W.; Ichikawa, D.; Lauer, S. L.; McCulloch, L. H.; Easo, N.; Lin, S. J.; Camellato, B. R.; Zhu, Y.; Cai, J.; Xu, Z.; Zhao, Y.; Sacasa, M.; Accardo, R.; Ahn, J. Y.; Annadanam, S.; Brammer Basta, L. A.; Bello, N.; Cai, L.; Cerritos, S.; Cornwell, M.; D'Amato, A.; Hacker, M.; Hersey, K.; Kennedy, E.; Kianercy, A.; Kim, D.; Lim, H. S.; McCutcheon, G.; McGirr, K.; Meaney, N.; Meyer, L.; Moyer, A.; Nimer, M.; Sabbatini, C.; Shores, L.; Sil\nScore: 13.3, Published: 2023-08-12 DOI: 10.1101/2022.04.09.487066\nWhether synthetic genomes can power life has attracted broad interest in the synthetic biology field, especially when the synthetic genomes are extensively modified with thousands of designer features. Here we report de novo synthesis of the largest eukaryotic chromosome thus far, synIV, a 1,454,621-bp Saccharomyces cerevisiae chromosome resulting from extensive genome streamlining and modification. During the construction of synIV, we developed megachunk assembly combined with a hierarchical integration strategy, which significantly increased the accuracy and flexibility of synthetic chromosome construction and facilitated chromosome debugging. In addition to the drastic sequence changes made to synIV by rewriting it, we further manipulated the three-dimensional structure of synIV in the yeast nucleus to explore spatial gene regulation within the nuclear space. Surprisingly, we found few gene expression changes, suggesting that positioning inside the yeast nucleoplasm plays a minor role in gene regulation. Lastly, we tethered synIV to the inner nuclear membrane via its hundreds of loxPsym sites and observed transcriptional repression of the entire chromosome, demonstrating chromosome-wide transcription manipulation without changing the DNA sequences. Our manipulation of the spatial structure of the largest synthetic yeast chromosome shed light on higher-order architectural design of the synthetic genomes. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=199 SRC=\"FIGDIR/small/487066v2_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (64K): org.highwire.dtl.DTLVardef@1fa2433org.highwire.dtl.DTLVardef@15ea9e7org.highwire.dtl.DTLVardef@1e48bc1org.highwire.dtl.DTLVardef@193457e_HPS_FORMAT_FIGEXP M_FIG C_FIG HighlightsO_LIDe novo synthesis of the largest eukaryotic chromosome, synIV C_LIO_LISynIV shows similar 3D structure to wild-type IV, despite thousands of changes made to it C_LIO_LI\"Inside-out\" repositioning of synIV in nucleus shows minor transcriptional changes C_LIO_LIMultipoint tethering synIV to inner nuclear membrane represses transcription of whole chromosome C_LI\n",
  "wordCount" : "2552",
  "inLanguage": "en",
  "datePublished": "2023-08-23T10:37:56Z",
  "dateModified": "2023-08-23T10:37:56Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on August 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553013">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553013" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553013">
        <p class="paperTitle">Efficient Natural Plasmid Transformation of Vibrio natriegens Enables Zero-capital Molecular Biology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553013" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553013" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Specht, D. A.; Sheppard, T. J.; Kennedy, F.; Li, S.; Gadikota, G.; Barstow, B.</p>
        <p class="info">Score: 40.1, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553013' target='https://doi.org/10.1101/2023.08.11.553013'> 10.1101/2023.08.11.553013</a></p>
        <p class="abstract">The fast-growing microbe Vibrio natriegens is capable of natural transformation where it draws DNA in from media via an active process under physiological conditions. Using an engineered strain with a genomic copy of the master competence regulator tfoX from Vibrio cholera in combination with a new minimal competence media (MCM) that uses acetate as an energy source, we demonstrate naturally competent cells which are created, transformed, and recovered entirely in the same media, without exchange or addition of new media. Cells are naturally competent to plasmids, recombination with linear DNA, and co-transformation of both to select for scarless and markerless genomic edits. The entire process is simple and inexpensive, requiring no capital equipment for an entirely room temperature process (Zero Capital protocol, 104 cfu/{micro}g), or just an incubator (High Efficiency protocol, 105-6 cfu/{micro}g). These cells retain their naturally competent state when frozen and are transformable immediately upon thawing like a typical chemical or electrochemical competent cell. Since the optimized transformation protocol requires only 50 minutes of hands-on time, and V. natriegens grows quickly even on plates, a transformation started at 9 AM yields abundant culturable single colonies by 5 PM. Further, because all stages of transformation occur in the same media, and the process can be arbitrarily scaled in volume, this natural competence strain and media could be ideal for automated directed evolution applications. As a result, naturally competent V. natriegens could compete with E. coli as an excellent chassis for low-cost and highly scalable synthetic biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553191">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553191" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553191">
        <p class="paperTitle">Gene drive and genetic sex conversion in the global agricultural pest Ceratitis capitata</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553191" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553191" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Meccariello, A.; Hou, S.; Davydova, S.; Fawcett, J.; Siddall, A.; Leftwich, P. T.; Krsticevic, F.; Papathanos, P. A.; Windbichler, N.</p>
        <p class="info">Score: 15.4, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553191' target='https://doi.org/10.1101/2023.08.16.553191'> 10.1101/2023.08.16.553191</a></p>
        <p class="abstract">Homing-based gene drives are novel interventions promising the area-wide, species-specific genetic control of harmful insect populations. Here we characterise a first set of gene drives in a tephritid agricultural pest species, the Mediterranean fruit fly Ceratitis capitata (medfly). Our results show that the medfly is highly amenable to homing-based gene drive strategies. By targeting the medfly transformer gene, we also demonstrate two different mechanisms by which CRISPR-Cas9 gene drive can be coupled to sex conversion, whereby genetic females are transformed into fertile and harmless XX males. Given this unique malleability of sex determination, we modelled gene drive interventions that couple sex conversion and female sterility and found that such approaches could be effective and tolerant of resistant allele selection in the target population. Our results open the door for developing gene drive strains for the population suppression of the medfly and related tephritid pests by co-targeting female reproduction and shifting the reproductive sex ratio towards males. They demonstrate the untapped potential for gene drives to tackle agricultural pests in an environmentally friendly and economical way.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.553235">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.553235" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.553235">
        <p class="paperTitle">A molecular proximity sensor based on an engineered, dual-component guide RNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.553235" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.553235" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Choi, J.; Chen, W.; Liao, H.; Li, X.; Shendure, J.</p>
        <p class="info">Score: 36.8, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.553235' target='https://doi.org/10.1101/2023.08.14.553235'> 10.1101/2023.08.14.553235</a></p>
        <p class="abstract">One of the goals of synthetic biology is to enable the design of arbitrary molecular circuits with programmable inputs and outputs. Such circuits bridge the properties of electronic and natural circuits, processing information in a predictable manner within living cells. Genome editing is a potentially powerful component of synthetic molecular circuits, whether for modulating the expression of a target gene or for stably recording information to genomic DNA. However, programming molecular events such as protein-protein interactions or induced proximity as triggers for genome editing remains challenging. Here we demonstrate a strategy termed P3 editing, which links protein-protein proximity to the formation of a functional CRISPR-Cas9 dual-component guide RNA. By engineering the crRNA:tracrRNA interaction, we demonstrate that various known protein-protein interactions, as well as the chemically-induced dimerization of protein domains, can be used to activate prime editing or base editing in human cells. Additionally, we explore how P3 editing can incorporate outputs from ADAR-based RNA sensors, potentially allowing specific RNAs to induce specific genome edits within a larger circuit. Our strategy enhances the controllability of CRISPR-based genome editing, facilitating its use in synthetic molecular circuits deployed in living cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.20.554004">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.20.554004" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.20.554004">
        <p class="paperTitle">Targeting mosquito X-chromosomes reveals complex transmission dynamics of sex ratio distorting gene drives</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.20.554004" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.20.554004" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Haber, D. A.; Arien, Y.; Lamdan, L. B.; Alcalay, Y.; Zecharia, C.; Krsticevic, F.; Yonah, E. S.; Avraham, R. D.; Windbichler, N.; Papathanos, P. A.</p>
        <p class="info">Score: 11.1, Published: 2023-08-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.20.554004' target='https://doi.org/10.1101/2023.08.20.554004'> 10.1101/2023.08.20.554004</a></p>
        <p class="abstract">Engineered sex ratio distorters (SRDs) have been proposed as a powerful component of both self-limiting and self-sustaining genetic control strategies designed to suppress harmful insect pest or disease vector populations such as malaria mosquitoes. Two different CRISPR-based SRD mechanisms, X-shredding which eliminates X-bearing sperm in males and X-poisoning which causes postzygotic mortality of daughters inheriting disrupted X-linked genes, have been proposed. The nature of the sex distortion mechanism is predicted to affect the population dynamics of released Y-chromosome SRDs: A Y-linked X-shredder is invasive constituting a classical meiotic Y-drive, whereas Y-linked X-poisoning is self-limiting, unable to invade but also insulated from selection and disappearance. Here, we established the first X-poisoning SRD system in the malaria vector Anopheles gambiae targeting three X-linked ribosomal protein (RP) genes during spermatogenesis. Surprisingly, when we characterized transgenic strains showing a significant male bias, we found that sex distortion was primarily driven by a loss of X-bearing sperm with limited evidence for postzygotic lethality of daughters. By leveraging the Drosophila model, we show unambiguously that engineered SRD traits can operate differently in these two insects. Unlike X-shredding, X-poisoning could theoretically operate at early stages of spermatogenesis. We therefore explored premeiotic Cas9 expression to target the mosquito X-chromosome. We find that, by pre-empting the onset of meiotic sex chromosome inactivation, this approach may enable the development of Y-linked SRDs, once trade-offs between sex chromosome transmission distortion and male fertility are functionally balanced.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.19.553819">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.19.553819" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.19.553819">
        <p class="paperTitle">One-pot chemo-enzymatic synthesis and one-step recovery of homogeneous long-chain polyphosphates from microalgal biomass</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.19.553819" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.19.553819" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lin, Y.-H.; Nishikawa, S.; Jia, T. Z.; Yeh, F.-I.; Khusnutdinova, A.; Yakunin, A. F.; Fujishima, K.; Wang, P.-H.</p>
        <p class="info">Score: 8.1, Published: 2023-08-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.19.553819' target='https://doi.org/10.1101/2023.08.19.553819'> 10.1101/2023.08.19.553819</a></p>
        <p class="abstract">Phosphate, an essential component of life, fertilizers, and detergents, is a finite resource that could be depleted within 70 years, while improper phosphate waste disposal in aquatic environments results in eutrophication. Despite some chemical-based methods, biological phosphorus removal using polyphosphate-accumulating organisms, such as microalgae, serves as a sustainable alternative to reclaim phosphate from wastewater. Polyphosphates have profound biological functions and biomedical applications, serving as energy stock, coagulation factors, and antiviral agents depending on their length, showing inherent value in polyphosphate recovery. Here, we leveraged the power of thermodynamic coupling and phase transitions to establish a one-pot, two-step multi-enzyme cascade to convert polydisperse polyphosphate in microalgae biomass into high-molecular-weight insoluble long-chain polyphosphates, allowing for one-step purification. We then optimzed a thermo-digestion approach to transform the 1,300-mers into shorter polyphosphates. Altogether, the processes established here enable the establishment of a sustainable P bioeconomy platform to refine microalgal biomass for biotechnological uses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.19.553321">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.19.553321" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.19.553321">
        <p class="paperTitle">Designed Endocytosis-Triggering Proteins mediate Targeted Degradation and amplify signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.19.553321" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.19.553321" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, B.; Abedi, M.; Ahn, G.; Coventry, B.; Sappington, I.; Wang, R.; Schlichthaerle, T.; Zhang, J. Z.; Wang, Y.; Goreshnik, I.; Chiu, C. W.; Chazin-Gray, A.; Chan, S.; Gerben, S.; Murray, A.; Wang, S.; O&#39;Neill, J.; Yeh, R.; Misquith, A.; Wolf, A.; Tomasovic, L. M.; Piraner, D. I.; Gonzalez, M. J. D.; Bennett, N. R.; Venkatesh, P.; Ahlrichs, M.; Dobbins, C.; Yang, W.; Wang, X.; Vafeados, D.; Mout, R.; Shivaei, S.; Cao, L.; Carter, L.; Stewart, L.; Spangler, J. B.; Bernardes, G. J. L.; Roybal, K. T.; Greisen, P. J.; Li, X.; Bertozzi, C.; Baker, D.</p>
        <p class="info">Score: 6.8, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.19.553321' target='https://doi.org/10.1101/2023.08.19.553321'> 10.1101/2023.08.19.553321</a></p>
        <p class="abstract">Endocytosis and lysosomal trafficking of cell surface receptors can be triggered by interaction with endogenous ligands. Therapeutic approaches such as LYTAC1,2 and KineTAC3, have taken advantage of this to target specific proteins for degradation by fusing modified native ligands to target binding proteins. While powerful, these approaches can be limited by possible competition with the endogenous ligand(s), the requirement in some cases for chemical modification that limits genetic encodability and can complicate manufacturing, and more generally, there may not be natural ligands which stimulate endocytosis through a given receptor. Here we describe general protein design approaches for designing endocytosis triggering binding proteins (EndoTags) that overcome these challenges. We present EndoTags for the IGF-2R, ASGPR, and Transferrin receptors, and show that fusing these tags to proteins which bind to soluble or transmembrane protein leads to lysosomal trafficking and target degradation; as these receptors have different tissue distributions, the different EndoTags could enable targeting of degradation to different tissues. The modularity and genetic encodability of EndoTags enables AND gate control for higher specificity targeted degradation, and the localized secretion of degraders from engineered cells. The tunability and modularity of our genetically encodable EndoTags should contribute to deciphering the relationship between receptor engagement and cellular trafficking, and they have considerable therapeutic potential as targeted degradation inducers, signaling activators for endocytosis-dependent pathways, and cellular uptake inducers for targeted antibody drug and RNA conjugates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.553301">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.553301" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.553301">
        <p class="paperTitle">A chemical reaction network model of PURE</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.553301" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.553301" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jurado, Z.; Pandey, A.; Murray, R. M.</p>
        <p class="info">Score: 5.0, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.553301' target='https://doi.org/10.1101/2023.08.14.553301'> 10.1101/2023.08.14.553301</a></p>
        <p class="abstract">Cell-free expression systems provide a method for rapid DNA circuit prototyping and functional protein synthesis. While crude extracts remain a black box with many components carrying out unknown reactions, the PURE system contains only the required transcription and translation components for protein production. All proteins and small molecules are at known concentrations, opening up the possibility of detailed modeling for reliable computational predictions. However, there is little to no experimental data supporting the expression of target proteins for detailed protein models PURE models. In this work, we build a chemical reaction network transcription model for PURE protein synthesis. We compare the transcription models using DNA encoding for the malachite-green aptamer (MGapt) to measure mRNA production. Furthermore, we expand the PURE detailed translation model for an arbitrary set of amino acids and length. Lastly, we combine the transcription and the expanded translation models to create a PURE protein synthesis model built purely from mass-action reactions. We use the combined model to capture the translation of a plasmid encoding MGapt and deGFP under a T7-promoter and a strong RBS. The model accurately predicts the MGapt mRNA production for the first two hours, the dynamics of deGFP expression, and the total protein production with an accuracy within 10 %.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552077">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552077" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552077">
        <p class="paperTitle">Machine-guided design of synthetic cell type-specific cis-regulatory elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552077" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552077" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gosai, S. J.; Castro, R. I.; Fuentes, N.; Butts, J. C.; Kales, S.; Noche, R. R.; Mouri, K.; Sabeti, P. C.; Reilly, S. K.; Tewhey, R.</p>
        <p class="info">Score: 60.9, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552077' target='https://doi.org/10.1101/2023.08.08.552077'> 10.1101/2023.08.08.552077</a></p>
        <p class="abstract">Cis-regulatory elements (CREs) control gene expression, orchestrating tissue identity, developmental timing, and stimulus responses, which collectively define the thousands of unique cell types in the body. While there is great potential for strategically incorporating CREs in therapeutic or biotechnology applications that require tissue specificity, there is no guarantee that an optimal CRE for an intended purpose has arisen naturally through evolution. Here, we present a platform to engineer and validate synthetic CREs capable of driving gene expression with programmed cell type specificity. We leverage innovations in deep neural network modeling of CRE activity across three cell types, efficient in silico optimization, and massively parallel reporter assays (MPRAs) to design and empirically test thousands of CREs. Through in vitro and in vivo validation, we show that synthetic sequences outperform natural sequences from the human genome in driving cell type-specific expression. Synthetic sequences leverage unique sequence syntax to promote activity in the on-target cell type and simultaneously reduce activity in off-target cells. Together, we provide a generalizable framework to prospectively engineer CREs and demonstrate the required literacy to write regulatory code that is fit-for-purpose in vivo across vertebrates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553881">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553881" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553881">
        <p class="paperTitle">Cell-Free Protein Expression in Polymer Materials</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553881" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553881" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, M. F. S.; Lee, J. A.; Biondo, J. R.; Lux, J. E.; Raig, R. M.; Berger, P. N.; Bernhards, C. B.; Kuhn, D. L.; Gupta, M. K.; Lux, M. W.</p>
        <p class="info">Score: 4.0, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553881' target='https://doi.org/10.1101/2023.08.18.553881'> 10.1101/2023.08.18.553881</a></p>
        <p class="abstract">While synthetic biology has advanced complex capabilities like sensing and molecule synthesis in aqueous solutions, important applications may also be pursued for biological systems in solid materials. Harsh processing conditions used to produce many synthetic materials such as plastics makes incorporation of biological functionality challenging. One technology that shows promise in circumventing these issues is cell-free protein synthesis (CFPS), where core cellular functionality is reconstituted outside the cell. CFPS enables genetic functions to be implemented without the complications of membrane transport or concerns over cellular viability or release of genetically modified organisms. Here we demonstrate that dried CFPS reactions have remarkable tolerance to heat and organic solvent exposure during the casting processes for polymer materials. We demonstrate the utility of this observation by creating plastics that have spatially patterned genetic functionality, produce antimicrobials in situ, and perform sensing reactions. The resulting materials unlock the potential to deliver DNA-programmable bio-functionality in a ubiquitous class of synthetic materials.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.04.09.487066">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.04.09.487066" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.04.09.487066">
        <p class="paperTitle">Manipulating the 3D organization of the largest synthetic yeast chromosome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.04.09.487066" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.04.09.487066" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, W.; Stefanita, L. L.; Yamashita, H.; Shen, M. J.; Mitchell, L. A.; Kurasawa, H.; Lobzaev, E.; Fanfani, V.; Haase, M. A. B.; Sun, X.; Jiang, Q.; Goldberg, G. W.; Ichikawa, D.; Lauer, S. L.; McCulloch, L. H.; Easo, N.; Lin, S. J.; Camellato, B. R.; Zhu, Y.; Cai, J.; Xu, Z.; Zhao, Y.; Sacasa, M.; Accardo, R.; Ahn, J. Y.; Annadanam, S.; Brammer Basta, L. A.; Bello, N.; Cai, L.; Cerritos, S.; Cornwell, M.; D&#39;Amato, A.; Hacker, M.; Hersey, K.; Kennedy, E.; Kianercy, A.; Kim, D.; Lim, H. S.; McCutcheon, G.; McGirr, K.; Meaney, N.; Meyer, L.; Moyer, A.; Nimer, M.; Sabbatini, C.; Shores, L.; Sil</p>
        <p class="info">Score: 13.3, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.04.09.487066' target='https://doi.org/10.1101/2022.04.09.487066'> 10.1101/2022.04.09.487066</a></p>
        <p class="abstract">Whether synthetic genomes can power life has attracted broad interest in the synthetic biology field, especially when the synthetic genomes are extensively modified with thousands of designer features. Here we report de novo synthesis of the largest eukaryotic chromosome thus far, synIV, a 1,454,621-bp Saccharomyces cerevisiae chromosome resulting from extensive genome streamlining and modification. During the construction of synIV, we developed megachunk assembly combined with a hierarchical integration strategy, which significantly increased the accuracy and flexibility of synthetic chromosome construction and facilitated chromosome debugging. In addition to the drastic sequence changes made to synIV by rewriting it, we further manipulated the three-dimensional structure of synIV in the yeast nucleus to explore spatial gene regulation within the nuclear space. Surprisingly, we found few gene expression changes, suggesting that positioning inside the yeast nucleoplasm plays a minor role in gene regulation. Lastly, we tethered synIV to the inner nuclear membrane via its hundreds of loxPsym sites and observed transcriptional repression of the entire chromosome, demonstrating chromosome-wide transcription manipulation without changing the DNA sequences. Our manipulation of the spatial structure of the largest synthetic yeast chromosome shed light on higher-order architectural design of the synthetic genomes.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=199 SRC=&#34;FIGDIR/small/487066v2_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (64K):
org.highwire.dtl.DTLVardef@1fa2433org.highwire.dtl.DTLVardef@15ea9e7org.highwire.dtl.DTLVardef@1e48bc1org.highwire.dtl.DTLVardef@193457e_HPS_FORMAT_FIGEXP  M_FIG C_FIG HighlightsO_LIDe novo synthesis of the largest eukaryotic chromosome, synIV
C_LIO_LISynIV shows similar 3D structure to wild-type IV, despite thousands of changes made to it
C_LIO_LI&#34;Inside-out&#34; repositioning of synIV in nucleus shows minor transcriptional changes
C_LIO_LIMultipoint tethering synIV to inner nuclear membrane represses transcription of whole chromosome
C_LI</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
