<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="DNA representations and generalization performance of sequence-to-expression models
Authors: Shen, Y.; Kudla, G.; Oyarzun, D. A.
Score: 10.0, Published: 2024-02-07 DOI: 10.1101/2024.02.06.579067
The increasing demand for biological products drives many efforts to engineer cells that produce heterologous proteins at maximal yield. Recent advances in massively parallel reporter assays can deliver data suitable for training machine learning models and sup-port the design of microbial strains with optimized protein expression phenotypes. The best performing sequence- to-expression models have been trained on one-hot encodings, a mechanism-agnostic representation of nucleotide sequences.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="DNA representations and generalization performance of sequence-to-expression models
Authors: Shen, Y.; Kudla, G.; Oyarzun, D. A.
Score: 10.0, Published: 2024-02-07 DOI: 10.1101/2024.02.06.579067
The increasing demand for biological products drives many efforts to engineer cells that produce heterologous proteins at maximal yield. Recent advances in massively parallel reporter assays can deliver data suitable for training machine learning models and sup-port the design of microbial strains with optimized protein expression phenotypes. The best performing sequence- to-expression models have been trained on one-hot encodings, a mechanism-agnostic representation of nucleotide sequences." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-14T10:38:48+00:00" />
<meta property="article:modified_time" content="2024-02-14T10:38:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="DNA representations and generalization performance of sequence-to-expression models
Authors: Shen, Y.; Kudla, G.; Oyarzun, D. A.
Score: 10.0, Published: 2024-02-07 DOI: 10.1101/2024.02.06.579067
The increasing demand for biological products drives many efforts to engineer cells that produce heterologous proteins at maximal yield. Recent advances in massively parallel reporter assays can deliver data suitable for training machine learning models and sup-port the design of microbial strains with optimized protein expression phenotypes. The best performing sequence- to-expression models have been trained on one-hot encodings, a mechanism-agnostic representation of nucleotide sequences."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "DNA representations and generalization performance of sequence-to-expression models\nAuthors: Shen, Y.; Kudla, G.; Oyarzun, D. A.\nScore: 10.0, Published: 2024-02-07 DOI: 10.1101/2024.02.06.579067\nThe increasing demand for biological products drives many efforts to engineer cells that produce heterologous proteins at maximal yield. Recent advances in massively parallel reporter assays can deliver data suitable for training machine learning models and sup-port the design of microbial strains with optimized protein expression phenotypes. The best performing sequence- to-expression models have been trained on one-hot encodings, a mechanism-agnostic representation of nucleotide sequences.",
  "keywords": [
    
  ],
  "articleBody": " DNA representations and generalization performance of sequence-to-expression models\nAuthors: Shen, Y.; Kudla, G.; Oyarzun, D. A.\nScore: 10.0, Published: 2024-02-07 DOI: 10.1101/2024.02.06.579067\nThe increasing demand for biological products drives many efforts to engineer cells that produce heterologous proteins at maximal yield. Recent advances in massively parallel reporter assays can deliver data suitable for training machine learning models and sup-port the design of microbial strains with optimized protein expression phenotypes. The best performing sequence- to-expression models have been trained on one-hot encodings, a mechanism-agnostic representation of nucleotide sequences. Despite their excellent local pre-dictive power, however, such models suffer from a limited ability to generalize predictions far away from the training data. Here, we show that libraries of genetic constructs can have substantially different cluster structure depending on the chosen sequence representation, and demonstrate that such differences can be leveraged to improve generalization perfor-mance. Using a large sequence- to-expression dataset from Escherichia coli, we show that non-deep regressors and convolutional neural networks trained on one-hot encodings fail to generalize predictions, and that learned representations using state-of-the-art large language models also struggle with out-of-domain accuracy. In contrast, we show that despite their poorer local performance, mechanistic sequence features such as codon bias, nucleotide con-tent or mRNA stability, provide promising gains on model generalization. We explore several strategies to integrate different feature sets into a single predictive model, including feature stacking, ensemble model stacking, and geometric stacking, a novel architecture based on graph convolutional neural networks. Our work suggests that integration of domain-agnostic and domain-aware sequence features offers an unexplored route for improving the quality of sequence- to-expression models and facilitate their adoption in the biotechnology and phar-maceutical sectors.\nEnhanced Sequence-Activity Mapping and Evolution of Artificial Metalloenzymes by Active Learning\nAuthors: Vornholt, T.; Mutny, M.; Schmidt, G. W.; Schellhaas, C.; Tachibana, R.; Panke, S.; Ward, T. R.; Krause, A.; Jeschek, M.\nScore: 5.0, Published: 2024-02-06 DOI: 10.1101/2024.02.06.579157\nTailored enzymes hold great potential to accelerate the transition to a sustainable bioeconomy. Yet, enzyme engineering remains challenging as it relies largely on serendipity and is, therefore, highly laborious and prone to failure. The efficiency and success rates of engineering campaigns may be improved substantially by applying machine learning to construct a comprehensive representation of the sequence-activity landscape from small sets of experimental data. However, it often proves challenging to reliably model a large protein sequence space while keeping the experimental effort tractable. To address this challenge, we present an integrated pipeline combining large-scale screening with active machine learning and model-guided library design. We applied this strategy to efficiently engineer an artificial metalloenzyme (ArM) catalysing a new-to-nature hydroamination reaction. By combining lab automation and next-generation sequencing, we acquired sequence-activity data for several thousand ArM variants. We then used Gaussian process regression to model the activity landscape and guide further screening rounds according to user-defined objectives. Crucial characteristics of our enhanced enzyme engineering pipeline include i) the cost-effective generation of information-rich experimental data sets, ii) the integration of an explorative round to improve the performance of the model, as well as iii) the consideration of experimental noise during modelling. Our approach led to an order-of-magnitude boost in the hit rate of screening while making efficient use of experimental resources. Smart search strategies like this should find broad utility in enzyme engineering and accelerate the development of novel biocatalysts.\nBreakthrough in Dicot Prime Editing: Enabling Heritable Desired Edits in Tomato and Arabidopsis\nAuthors: Vu, T. V.; Nguyen, N. T.; Kim, J.; Song, Y. J.; Nguyen, T. H.; Kim, J.-Y.\nScore: 4.2, Published: 2024-02-11 DOI: 10.1101/2024.02.11.579803\nPrime editing (PE) enables almost all types of precise genome editing in animals and plants. It has been successfully adapted to edit several plants at variable efficiency and versatility. However, this technique is inefficient for dicots for unknown reasons. Here, by employing novel combinations of PE components, including an RNA chaperone and modified epegRNAs driven by a PolII-PolIII composite promoter and a viral replicon system, we obtained up to 9.7% of the desired PE efficiency at the callus stage assessed by targeted deep sequencing. Subsequently, we identified that up to 38.2% of transformants contained desired PE alleles in tomatoes and Arabidopsis, marking the first successful heritable PE transmission in dicots. Our PE tools also showed high accuracy, specificity, and multiplexing capability, which unlocked the potential for practical PE applications in dicots, paving the way for transformative advancements in plant sciences.\nUltra-high efficiency T cell reprogramming at multiple loci with SEED-Selection\nAuthors: Chang, C. R.; Vykunta, V. S.; Goodman, D. B.; Muldoon, J. J.; Nyberg, W. A.; Liu, C.; Allain, V.; Rothrock, A.; Wang, C. H.; Marson, A.; Shy, B. R.; Eyquem, J.\nScore: 3.6, Published: 2024-02-07 DOI: 10.1101/2024.02.06.576175\nMultiplexed reprogramming of T cell specificity and function can generate powerful next-generation cellular therapies. However, current manufacturing methods produce heterogenous mixtures of partially engineered cells. Here, we develop a one-step process to enrich for unlabeled cells with knock-ins at multiple target loci using a family of repair templates named Synthetic Exon/Expression Disruptors (SEEDs). SEED engineering associates transgene integration with the disruption of a paired endogenous surface protein, allowing non-modified and partially edited cells to be immunomagnetically depleted (SEED-Selection). We design SEEDs to fully reprogram three critical loci encoding T cell specificity, co-receptor expression, and MHC expression, with up to 98% purity after selection for individual modifications and up to 90% purity for six simultaneous edits (three knock-ins and three knockouts). These methods are simple, compatible with existing clinical manufacturing workflows, and can be readily adapted to other loci to facilitate production of complex gene-edited cell therapies.\nSelf-assembled cell-scale containers made from DNA origami membranes\nAuthors: Karfusehr, C.; Eder, M.; Simmel, F. C.\nScore: 3.4, Published: 2024-02-09 DOI: 10.1101/2024.02.09.579479\nDNA origami provides a methodology for the sequence-programmable generation of precisely defined molecular nanostructures with sizes of order 100 nm. A new frontier for the field is the generation of superstructures made from DNA origami subunits, which requires other self-assembly strategies than those used for DNA origami itself. Challenges faced by current approaches include the increasing complexity, cost and development time for the structures and off-target assembly. Here, we demonstrate how radially symmetric origami subunits that are inspired by the structure and interactions of lipids organize into giant DNA origami monolayer membranes that can be readily programmed to form vesicles or hollow tubes with diameters ranging from 100 nm to over 1 {micro}m. DNA origami membranes are an unprecedented approach for compartmentalization that opens up new possibilities for bottom-up biology and cell-scale soft robotics.\nModel-guided metabolic engineering of curcuminoid Production in Pseudomonas putida\nAuthors: Martin-Pascual, M.; Moreno-Paz, S.; Van Rosmalen, R. P.; Dorigo, J.; Demaria, F.; Van Kranenburg, R.; Martins dos Santos, V.; Suarez-Diez, M. S.\nScore: 1.5, Published: 2024-02-08 DOI: 10.1101/2024.02.08.579459\nProduction of value-added, plant-derived compounds in microbes increasingly attracts commercially interest in food and pharmaceutical industries. However, plant metabolic pathways are complex, require a robust balance of enzymes, cofactors, ATP and other metabolites, and often result in low production when transplanted to bacteria. This is exemplified by the biosynthesis of curcuminoids from the Curcuma longa plant. Here, we combine dynamic pathway modeling, systematic testing of isoenzymes, and the optimization of gene expression levels and substrate concentrations for the biosynthesis of curcuminoids in Pseudomonas putida, leading to unprecedented conversion rates of caffeic acid and tyrosine to curcumin. The development of kinetic ensemble models guided the design of production strains, emphasizing the necessity of high relative expression of c3h, curs2 and dcs and, the low relative expression of tal, comt, ccoaomt, and 4cl4. This optimization resulted in a strain that achieved a 10.8 {+/-}1.8% of the maximum theoretical yield of curcumin from tyrosine. This represents a 4.1-fold increase in production efficiency and the highest yield reported to date, demonstrating the potential of P. putida as a promising platform for curcuminoid production. Our findings highlight the effectiveness of our strategy not only in the advances in the production of curcuminoids but also in setting a framework for the biosynthesis of other complex compounds.\nEngineering an anti-CD206-synNotch receptor: insightsinto the development of novel synthetic receptors\nAuthors: Semeniuk, S.; Qian, B.; Cachat, E.\nScore: 1.5, Published: 2024-02-07 DOI: 10.1101/2024.02.07.579082\nImmune cells play a pivotal role in the establishment, growth and progression of tumors at primary and metastatic sites. Macrophages, in particular, play a critical role in suppressing immune responses and promoting an anti-inflammatory environment through both direct and indirect cell-cell interactions. However, our understanding of the mechanisms underlying such interactions is limited due to a lack of reliable tools for studying transient interactions between cancer cells and macrophages within the tumor microenvironment. Recent advances in mammalian synthetic biology have introduced a wide range of synthetic receptors that have been used in diverse biosensing applications. One such synthetic receptor is the synNotch receptor, which can be tailored to sense specific ligands displayed on the surface of target cells. With this study, we aimed at developing a novel CD206-synNotch receptor, targeting CD206+ macrophages, a population of macrophages that play a crucial role in promoting metastatic seeding and persistent growth. Engineered in cancer cells and used in mouse metastasis models, such tool could help monitor and understand the effects cell-cell interactions between macrophages and cancer cells have on metastasis establishment. Here, we report the development of cancer landing pad cells for versatile applications, the engineering of CD206-synNotch cells, report the measurements of their activity and specificity, and discuss the unexpected caveats when considering their in vivo applications.\nInfection-on-Chip: an in vitro human vessel to study Neisseria meningitidis colonization and vascular damages\nAuthors: Pinon, L.; Chabaud, M.; Nivoit, P.; Wong-Ng, J.; Nguyen, T.-T.; Paul, V.; Goussard, S.; Frachon, E.; Obino, D.; Gobaa, S.; Dumenil, G.\nScore: 1.0, Published: 2024-02-10 DOI: 10.1101/2024.02.09.579276\nBloodstream infections leading to sepsis are a life-threatening condition and remain difficult to treat, however, in vitro experimental models that reflect their key features are still lacking. We here developed a photoablation-based 3-dimensional, microfluidic model of meningococcal vascular colonization, which allows to study cardinal features of the bacteria-blood vessel interaction within controllable vascular geometries. Meningococci are Gram-negative human-specific bacteria responsible for meningitis and a severe form of sepsis that is associated with vascular damages, referred to as purpura fulminans. The infection-on-chip device is used to quantitatively assess bacterial adhesion and proliferation at high spatio-temporal resolution in a physiologically relevant microenvironment. In addition, we here show that vascular colonization by meningococci in our Infection-on-Chip device recapitulates key features of disease progression, including vascular leakage and the recruitment of neutrophils to sites of infections, mirroring results obtained using our previously described human skin xenograft mouse model. As a result, our Infection-on-chip platform provides a robust alternative approach to the use of animal and 2D cellular models, opening the path to the better understanding of disease progression and testing innovative therapeutics in an in vitro but physiologically relevant environment.\nEnhancing Plant Photosynthesis using Carbon Dots as Light Converter and Photosensitizer\nAuthors: Hu, H.; Cheng, W.; Wang, X.; Yang, Y.; Yu, X.; Ding, J.; Lin, Y.; Zhao, W.; Zhao, Q.; Ledesma-Amaro, R.; Chen, X.; Liu, J.; Yang, C.; Gao, X.\nScore: 1.0, Published: 2024-02-07 DOI: 10.1101/2024.02.06.579025\nImproving photosynthetic efficiency is pivotal for CO2-based biomanufacturing and agriculture purposes. Despite the progress on photosynthetic biohybrids integrating biocatalysts with synthetic materials, nanomaterials with improved optical and photoelectrochemical properties are still needed to increase the energy-conversion efficiency. Here, we present a novel approach using carbon dots (CDs) as both intracellular photosensitizers and light converters for enhancing solar energy utilization in photosynthetic organisms. The CDs were produced from cyanobacterial biomass and used to convert a broad spectrum of solar irradiation to red light. We demonstrated that the nanosized CDs were incorporated into cyanobacterial cells and transferred light-excited electrons into the photosynthetic electron transfer chain. The biohybrids consisting of the CDs and Synechococcus elongatus exhibited increased growth rates, enhanced activities of both photosystems, and accelerated linear electron transport, compared with the cyanobacterial cells only. The supplementation of the CDs increased CO2-fixation rate and CO2-to-glycerol production by 2.4-fold and 2.2-fold, respectively. Furthermore, the CDs were shown to enhance photosynthesis and promote growth of Arabidopsis thaliana. The fresh weight of plant was increased 1.8-fold by CDs addition. These results reveal that simultaneous photosensitization and spectral modification could substantially improve the efficiency of natural photosynthesis. This study presents CDs as an attractive nanomaterial with great application potential in agriculture and solar-powered biomanufacturing.\n",
  "wordCount" : "2045",
  "inLanguage": "en",
  "datePublished": "2024-02-14T10:38:48Z",
  "dateModified": "2024-02-14T10:38:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579067">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579067" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579067">
        <p class="paperTitle">DNA representations and generalization performance of sequence-to-expression models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579067" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579067" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shen, Y.; Kudla, G.; Oyarzun, D. A.</p>
        <p class="info">Score: 10.0, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579067' target='https://doi.org/10.1101/2024.02.06.579067'> 10.1101/2024.02.06.579067</a></p>
        <p class="abstract">The increasing demand for biological products drives many efforts to engineer cells that produce heterologous proteins at maximal yield. Recent advances in massively parallel reporter assays can deliver data suitable for training machine learning models and sup-port the design of microbial strains with optimized protein expression phenotypes. The best performing sequence- to-expression models have been trained on one-hot encodings, a mechanism-agnostic representation of nucleotide sequences. Despite their excellent local pre-dictive power, however, such models suffer from a limited ability to generalize predictions far away from the training data. Here, we show that libraries of genetic constructs can have substantially different cluster structure depending on the chosen sequence representation, and demonstrate that such differences can be leveraged to improve generalization perfor-mance. Using a large sequence- to-expression dataset from Escherichia coli, we show that non-deep regressors and convolutional neural networks trained on one-hot encodings fail to generalize predictions, and that learned representations using state-of-the-art large language models also struggle with out-of-domain accuracy. In contrast, we show that despite their poorer local performance, mechanistic sequence features such as codon bias, nucleotide con-tent or mRNA stability, provide promising gains on model generalization. We explore several strategies to integrate different feature sets into a single predictive model, including feature stacking, ensemble model stacking, and geometric stacking, a novel architecture based on graph convolutional neural networks. Our work suggests that integration of domain-agnostic and domain-aware sequence features offers an unexplored route for improving the quality of sequence- to-expression models and facilitate their adoption in the biotechnology and phar-maceutical sectors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579157">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579157" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579157">
        <p class="paperTitle">Enhanced Sequence-Activity Mapping and Evolution of Artificial Metalloenzymes by Active Learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579157" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579157" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vornholt, T.; Mutny, M.; Schmidt, G. W.; Schellhaas, C.; Tachibana, R.; Panke, S.; Ward, T. R.; Krause, A.; Jeschek, M.</p>
        <p class="info">Score: 5.0, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579157' target='https://doi.org/10.1101/2024.02.06.579157'> 10.1101/2024.02.06.579157</a></p>
        <p class="abstract">Tailored enzymes hold great potential to accelerate the transition to a sustainable bioeconomy. Yet, enzyme engineering remains challenging as it relies largely on serendipity and is, therefore, highly laborious and prone to failure. The efficiency and success rates of engineering campaigns may be improved substantially by applying machine learning to construct a comprehensive representation of the sequence-activity landscape from small sets of experimental data. However, it often proves challenging to reliably model a large protein sequence space while keeping the experimental effort tractable. To address this challenge, we present an integrated pipeline combining large-scale screening with active machine learning and model-guided library design. We applied this strategy to efficiently engineer an artificial metalloenzyme (ArM) catalysing a new-to-nature hydroamination reaction. By combining lab automation and next-generation sequencing, we acquired sequence-activity data for several thousand ArM variants. We then used Gaussian process regression to model the activity landscape and guide further screening rounds according to user-defined objectives. Crucial characteristics of our enhanced enzyme engineering pipeline include i) the cost-effective generation of information-rich experimental data sets, ii) the integration of an explorative round to improve the performance of the model, as well as iii) the consideration of experimental noise during modelling. Our approach led to an order-of-magnitude boost in the hit rate of screening while making efficient use of experimental resources. Smart search strategies like this should find broad utility in enzyme engineering and accelerate the development of novel biocatalysts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.11.579803">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.11.579803" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.11.579803">
        <p class="paperTitle">Breakthrough in Dicot Prime Editing: Enabling Heritable Desired Edits in Tomato and Arabidopsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.11.579803" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.11.579803" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vu, T. V.; Nguyen, N. T.; Kim, J.; Song, Y. J.; Nguyen, T. H.; Kim, J.-Y.</p>
        <p class="info">Score: 4.2, Published: 2024-02-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.11.579803' target='https://doi.org/10.1101/2024.02.11.579803'> 10.1101/2024.02.11.579803</a></p>
        <p class="abstract">Prime editing (PE) enables almost all types of precise genome editing in animals and plants. It has been successfully adapted to edit several plants at variable efficiency and versatility. However, this technique is inefficient for dicots for unknown reasons. Here, by employing novel combinations of PE components, including an RNA chaperone and modified epegRNAs driven by a PolII-PolIII composite promoter and a viral replicon system, we obtained up to 9.7% of the desired PE efficiency at the callus stage assessed by targeted deep sequencing. Subsequently, we identified that up to 38.2% of transformants contained desired PE alleles in tomatoes and Arabidopsis, marking the first successful heritable PE transmission in dicots. Our PE tools also showed high accuracy, specificity, and multiplexing capability, which unlocked the potential for practical PE applications in dicots, paving the way for transformative advancements in plant sciences.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.576175">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.576175" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.576175">
        <p class="paperTitle">Ultra-high efficiency T cell reprogramming at multiple loci with SEED-Selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.576175" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.576175" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chang, C. R.; Vykunta, V. S.; Goodman, D. B.; Muldoon, J. J.; Nyberg, W. A.; Liu, C.; Allain, V.; Rothrock, A.; Wang, C. H.; Marson, A.; Shy, B. R.; Eyquem, J.</p>
        <p class="info">Score: 3.6, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.576175' target='https://doi.org/10.1101/2024.02.06.576175'> 10.1101/2024.02.06.576175</a></p>
        <p class="abstract">Multiplexed reprogramming of T cell specificity and function can generate powerful next-generation cellular therapies. However, current manufacturing methods produce heterogenous mixtures of partially engineered cells. Here, we develop a one-step process to enrich for unlabeled cells with knock-ins at multiple target loci using a family of repair templates named Synthetic Exon/Expression Disruptors (SEEDs). SEED engineering associates transgene integration with the disruption of a paired endogenous surface protein, allowing non-modified and partially edited cells to be immunomagnetically depleted (SEED-Selection). We design SEEDs to fully reprogram three critical loci encoding T cell specificity, co-receptor expression, and MHC expression, with up to 98% purity after selection for individual modifications and up to 90% purity for six simultaneous edits (three knock-ins and three knockouts). These methods are simple, compatible with existing clinical manufacturing workflows, and can be readily adapted to other loci to facilitate production of complex gene-edited cell therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579479">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579479" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579479">
        <p class="paperTitle">Self-assembled cell-scale containers made from DNA origami membranes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579479" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579479" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Karfusehr, C.; Eder, M.; Simmel, F. C.</p>
        <p class="info">Score: 3.4, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579479' target='https://doi.org/10.1101/2024.02.09.579479'> 10.1101/2024.02.09.579479</a></p>
        <p class="abstract">DNA origami provides a methodology for the sequence-programmable generation of precisely defined molecular nanostructures with sizes of order 100 nm. A new frontier for the field is the generation of superstructures made from DNA origami subunits, which requires other self-assembly strategies than those used for DNA origami itself. Challenges faced by current approaches include the increasing complexity, cost and development time for the structures and off-target assembly. Here, we demonstrate how radially symmetric origami subunits that are inspired by the structure and interactions of lipids organize into giant DNA origami monolayer membranes that can be readily programmed to form vesicles or hollow tubes with diameters ranging from 100 nm to over 1 {micro}m. DNA origami membranes are an unprecedented approach for compartmentalization that opens up new possibilities for bottom-up biology and cell-scale soft robotics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.08.579459">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.08.579459" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.08.579459">
        <p class="paperTitle">Model-guided metabolic engineering of curcuminoid Production in Pseudomonas putida</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.08.579459" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.08.579459" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martin-Pascual, M.; Moreno-Paz, S.; Van Rosmalen, R. P.; Dorigo, J.; Demaria, F.; Van Kranenburg, R.; Martins dos Santos, V.; Suarez-Diez, M. S.</p>
        <p class="info">Score: 1.5, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.08.579459' target='https://doi.org/10.1101/2024.02.08.579459'> 10.1101/2024.02.08.579459</a></p>
        <p class="abstract">Production of value-added, plant-derived compounds in microbes increasingly attracts commercially interest in food and pharmaceutical industries. However, plant metabolic pathways are complex, require a robust balance of enzymes, cofactors, ATP and other metabolites, and often result in low production when transplanted to bacteria. This is exemplified by the biosynthesis of curcuminoids from the Curcuma longa plant. Here, we combine dynamic pathway modeling, systematic testing of isoenzymes, and the optimization of gene expression levels and substrate concentrations for the biosynthesis of curcuminoids in Pseudomonas putida, leading to unprecedented conversion rates of caffeic acid and tyrosine to curcumin. The development of kinetic ensemble models guided the design of production strains, emphasizing the necessity of high relative expression of c3h, curs2 and dcs and, the low relative expression of tal, comt, ccoaomt, and 4cl4. This optimization resulted in a strain that achieved a 10.8 {&#43;/-}1.8% of the maximum theoretical yield of curcumin from tyrosine. This represents a 4.1-fold increase in production efficiency and the highest yield reported to date, demonstrating the potential of P. putida as a promising platform for curcuminoid production. Our findings highlight the effectiveness of our strategy not only in the advances in the production of curcuminoids but also in setting a framework for the biosynthesis of other complex compounds.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.579082">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.579082" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.579082">
        <p class="paperTitle">Engineering an anti-CD206-synNotch receptor: insightsinto the development of novel synthetic receptors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.579082" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.579082" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Semeniuk, S.; Qian, B.; Cachat, E.</p>
        <p class="info">Score: 1.5, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.579082' target='https://doi.org/10.1101/2024.02.07.579082'> 10.1101/2024.02.07.579082</a></p>
        <p class="abstract">Immune cells play a pivotal role in the establishment, growth and progression of tumors at primary and metastatic sites. Macrophages, in particular, play a critical role in suppressing immune responses and promoting an anti-inflammatory environment through both direct and indirect cell-cell interactions. However, our understanding of the mechanisms underlying such interactions is limited due to a lack of reliable tools for studying transient interactions between cancer cells and macrophages within the tumor microenvironment. Recent advances in mammalian synthetic biology have introduced a wide range of synthetic receptors that have been used in diverse biosensing applications. One such synthetic receptor is the synNotch receptor, which can be tailored to sense specific ligands displayed on the surface of target cells. With this study, we aimed at developing a novel CD206-synNotch receptor, targeting CD206&#43; macrophages, a population of macrophages that play a crucial role in promoting metastatic seeding and persistent growth. Engineered in cancer cells and used in mouse metastasis models, such tool could help monitor and understand the effects cell-cell interactions between macrophages and cancer cells have on metastasis establishment. Here, we report the development of cancer landing pad cells for versatile applications, the engineering of CD206-synNotch cells, report the measurements of their activity and specificity, and discuss the unexpected caveats when considering their in vivo applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579276">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579276" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579276">
        <p class="paperTitle">Infection-on-Chip: an in vitro human vessel to study Neisseria meningitidis colonization and vascular damages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579276" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579276" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pinon, L.; Chabaud, M.; Nivoit, P.; Wong-Ng, J.; Nguyen, T.-T.; Paul, V.; Goussard, S.; Frachon, E.; Obino, D.; Gobaa, S.; Dumenil, G.</p>
        <p class="info">Score: 1.0, Published: 2024-02-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579276' target='https://doi.org/10.1101/2024.02.09.579276'> 10.1101/2024.02.09.579276</a></p>
        <p class="abstract">Bloodstream infections leading to sepsis are a life-threatening condition and remain difficult to treat, however, in vitro experimental models that reflect their key features are still lacking. We here developed a photoablation-based 3-dimensional, microfluidic model of meningococcal vascular colonization, which allows to study cardinal features of the bacteria-blood vessel interaction within controllable vascular geometries. Meningococci are Gram-negative human-specific bacteria responsible for meningitis and a severe form of sepsis that is associated with vascular damages, referred to as purpura fulminans. The infection-on-chip device is used to quantitatively assess bacterial adhesion and proliferation at high spatio-temporal resolution in a physiologically relevant microenvironment. In addition, we here show that vascular colonization by meningococci in our Infection-on-Chip device recapitulates key features of disease progression, including vascular leakage and the recruitment of neutrophils to sites of infections, mirroring results obtained using our previously described human skin xenograft mouse model. As a result, our Infection-on-chip platform provides a robust alternative approach to the use of animal and 2D cellular models, opening the path to the better understanding of disease progression and testing innovative therapeutics in an in vitro but physiologically relevant environment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579025">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579025" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579025">
        <p class="paperTitle">Enhancing Plant Photosynthesis using Carbon Dots as Light Converter and Photosensitizer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579025" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579025" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, H.; Cheng, W.; Wang, X.; Yang, Y.; Yu, X.; Ding, J.; Lin, Y.; Zhao, W.; Zhao, Q.; Ledesma-Amaro, R.; Chen, X.; Liu, J.; Yang, C.; Gao, X.</p>
        <p class="info">Score: 1.0, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579025' target='https://doi.org/10.1101/2024.02.06.579025'> 10.1101/2024.02.06.579025</a></p>
        <p class="abstract">Improving photosynthetic efficiency is pivotal for CO2-based biomanufacturing and agriculture purposes. Despite the progress on photosynthetic biohybrids integrating biocatalysts with synthetic materials, nanomaterials with improved optical and photoelectrochemical properties are still needed to increase the energy-conversion efficiency. Here, we present a novel approach using carbon dots (CDs) as both intracellular photosensitizers and light converters for enhancing solar energy utilization in photosynthetic organisms. The CDs were produced from cyanobacterial biomass and used to convert a broad spectrum of solar irradiation to red light. We demonstrated that the nanosized CDs were incorporated into cyanobacterial cells and transferred light-excited electrons into the photosynthetic electron transfer chain. The biohybrids consisting of the CDs and Synechococcus elongatus exhibited increased growth rates, enhanced activities of both photosystems, and accelerated linear electron transport, compared with the cyanobacterial cells only. The supplementation of the CDs increased CO2-fixation rate and CO2-to-glycerol production by 2.4-fold and 2.2-fold, respectively. Furthermore, the CDs were shown to enhance photosynthesis and promote growth of Arabidopsis thaliana. The fresh weight of plant was increased 1.8-fold by CDs addition. These results reveal that simultaneous photosensitization and spectral modification could substantially improve the efficiency of natural photosynthesis. This study presents CDs as an attractive nanomaterial with great application potential in agriculture and solar-powered biomanufacturing.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
