<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Continuous evolution of user-defined genes at 1-million-times the genomic mutation rate
Authors: Rix, G.; Williams, R. L.; Spinner, H.; Hu, V. J.; Marks, D. S.; Liu, C. C.
Score: 56.5, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566922
When nature maintains or evolves a genes function over millions of years at scale, it produces a diversity of homologous sequences whose patterns of conservation and change contain rich structural, functional, and historical information about the gene.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Continuous evolution of user-defined genes at 1-million-times the genomic mutation rate
Authors: Rix, G.; Williams, R. L.; Spinner, H.; Hu, V. J.; Marks, D. S.; Liu, C. C.
Score: 56.5, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566922
When nature maintains or evolves a genes function over millions of years at scale, it produces a diversity of homologous sequences whose patterns of conservation and change contain rich structural, functional, and historical information about the gene." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-22T10:41:31+00:00" />
<meta property="article:modified_time" content="2023-11-22T10:41:31+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Continuous evolution of user-defined genes at 1-million-times the genomic mutation rate
Authors: Rix, G.; Williams, R. L.; Spinner, H.; Hu, V. J.; Marks, D. S.; Liu, C. C.
Score: 56.5, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566922
When nature maintains or evolves a genes function over millions of years at scale, it produces a diversity of homologous sequences whose patterns of conservation and change contain rich structural, functional, and historical information about the gene."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Continuous evolution of user-defined genes at 1-million-times the genomic mutation rate\nAuthors: Rix, G.; Williams, R. L.; Spinner, H.; Hu, V. J.; Marks, D. S.; Liu, C. C.\nScore: 56.5, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566922\nWhen nature maintains or evolves a genes function over millions of years at scale, it produces a diversity of homologous sequences whose patterns of conservation and change contain rich structural, functional, and historical information about the gene.",
  "keywords": [
    
  ],
  "articleBody": " Continuous evolution of user-defined genes at 1-million-times the genomic mutation rate\nAuthors: Rix, G.; Williams, R. L.; Spinner, H.; Hu, V. J.; Marks, D. S.; Liu, C. C.\nScore: 56.5, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566922\nWhen nature maintains or evolves a genes function over millions of years at scale, it produces a diversity of homologous sequences whose patterns of conservation and change contain rich structural, functional, and historical information about the gene. However, natural gene diversity likely excludes vast regions of functional sequence space and includes phylogenetic and evolutionary eccentricities, limiting what information we can extract. We introduce an accessible experimental approach for compressing long-term gene evolution to laboratory timescales, allowing for the direct observation of extensive adaptation and divergence followed by inference of structural, functional, and environmental constraints for any selectable gene. To enable this approach, we developed a new orthogonal DNA replication (OrthoRep) system that durably hypermutates chosen genes at a rate of \u003e10-4 substitutions per base in vivo. When OrthoRep was used to evolve a conditionally essential maladapted enzyme, we obtained thousands of unique multi-mutation sequences with many pairs \u003e60 amino acids apart (\u003e15% divergence), revealing known and new factors influencing enzyme adaptation. The fitness of evolved sequences was not predictable by advanced machine learning models trained on natural variation. We suggest that OrthoRep supports the prospective and systematic discovery of constraints shaping gene evolution, uncovering of new regions in fitness landscapes, and general applications in biomolecular engineering.\nEngineering orthogonal ribosomes for real-time monitoring using fluorescence\nAuthors: Csibra, E.; Klopprogge, B.; Hau Sorensen, G.; Gorochowski, T. E.\nScore: 22.2, Published: 2023-11-19 DOI: 10.1101/2023.11.19.567736\nA promising route to tackle the trade-off in cellular resources between synthetic protein production and cellular growth is to use a separate dedicated pool of orthogonal ribosomes to produce synthetic proteins. However, the optimisation of strains containing two ribosomal pools - native for the host cell's proteome and orthogonal for synthetic proteins - has yet to be thoroughly explored. Here, we address this by creating orthogonal ribosomes that fluoresce by inserting fluorescent RNA aptamers into tethered orthogonal ribosomal RNA (TO-rRNA). To study the tolerance of the engineered ribosomes to aptamer insertion, we assembled and screened a library of candidate insertion sites, identifying several sites in both the 16S and 23S TO-rRNA that enables ribosome labelling with minimal effect on translation activity. Serendipitously, we identify one site in 23S TO-rRNA, where insertion appears to not only be tolerated but to enhance orthogonal ribosome activity, across multiple bacterial strains and RNA insertions. Using bulk and single cell assays, we demonstrate that this variant allows us to label orthogonal ribosomes for dynamic tracking and across populations, making it a promising tool for optimising orthogonal translation in engineered cells. Ribosome engineering offers great potential, both for the development of next-generation microbial cell factories, as well as a tool to expand our understanding of ribosome function in living cells. This work provides a window into the assembly, localisation and function of these molecular machines to meet these aims.\nDirected Evolution of Genetically Encoded LYTACs for Cell-Mediated Delivery\nAuthors: Yang, J. L.; Yamada-Hunter, S. A.; Labanieh, L.; Sotillo, E.; Cheah, J. S.; Roberts, D. S.; Mackall, C.; Ting, A. Y.; Bertozzi, C.\nScore: 20.2, Published: 2023-11-15 DOI: 10.1101/2023.11.14.567117\nLysosome-targeting chimeras (LYTACs) are a promising therapeutic modality to drive the degradation of extracellular proteins. However, early versions of LYTAC contain synthetic glycopeptides that cannot be genetically encoded. Here we present our designs for a fully genetically encodable LYTAC (GELYTAC), making our tool compatible with integration into therapeutic cells for targeted delivery at diseased sites. To achieve this, we replaced the glycopeptide portion of LYTACs with the protein insulin like growth factor 2 (IGF2). After showing initial efficacy with wild type IGF2, we increased the potency of GELYTAC using directed evolution. Subsequently, we demonstrated that our engineered GELYTAC construct not only secretes from HEK293T cells but also from human primary T-cells to drive the uptake of various targets into receiver cells. Immune cells engineered to secrete GELYTAC thus represent a promising avenue for spatially-selective targeted protein degradation. Significance StatementBetter therapeutic windows can be achieved by targeting therapeutics to their desired sites of action. For protein therapeutics, this might be achieved by engineering cell therapies that home to a tissue of interest and secrete the biologic drug locally. Here, we demonstrate that human primary T cells can be engineered to produce genetically encoded lysosome targeting chimeras (GELYTACs). These GELYTACs mediate the degradation of extracellular proteins associated with cancer progression. Thus, cells engineered to produce GELYTACs represent a potential new class of cancer therapeutics.\nCryptography in the DNA of living cells enabled by multi-site base editing\nAuthors: Volf, V.; Zhang, S.; Song, K. M.; Qian, S.; Chen, F.; Church, G.\nScore: 15.9, Published: 2023-11-15 DOI: 10.1101/2023.11.15.567131\nAlthough DNA is increasingly being adopted as a generalizable medium for information storage and transfer, reliable methods for ensuring information security remain to be addressed. In this study, we developed and validated a cryptographic encoding scheme, Genomic Sequence Encryption (GSE), to address the challenge of information confidentiality and integrity in biological substrates. GSE enables genomic information encoding that is readable only with a cryptographic key. We show that GSE can be used for cell signatures that enable the recipient of a cell line to authenticate its origin and validate if the cell line has been modified in the interim. We implement GSE through multi-site base editing and encode information through editing across \u003e100 genomic sites in mammalian cells. We further present an enrichment step to obtain individual stem cells with more than two dozen edits across a single genome with minimal screening. This capability can be used to introduce encrypted signatures in living animals. As an encryption scheme, GSE is falsification-proof and enables secure information transfer in biological substrates.\nA tuneable minimal cell membrane reveals that two lipids suffice for life\nAuthors: Justice, I. G.; Saenz, J. P.\nScore: 36.9, Published: 2023-11-08 DOI: 10.1101/2023.10.24.563757\nAll cells are encapsulated by a lipid membrane which facilitates the interaction between life and its environment. The property and function of the cell membrane is critically dependent on its lipidome composition which can range from 10s to 100s of unique lipid structures. How life has evolved to exploit such diverse mixtures of lipids has been experimentally challenging to address. In this study, we introduce a simple approach to minimize and tune lipidome composition in Mycoplasma mycoides and the Minimal Cell (JCVI-Syn3A). By varying the lipid composition of the growth medium, and introducing unmodifiable diether phospholipids, we demonstrate that a lipidome composed of two lipids can support life. By systematically reintroducing phospholipid features, we show that acyl chain diversity is more critical for supporting growth than head group diversity. We also reveal significant effects of altering phospholipid chirality on membrane permeability, cellular robustness, and growth, demonstrating how subtle changes in lipid stereochemistry can have profound effects on cell fitness. Our approach to tune mycoplasma lipidome composition introduces a minimal living model membrane system to understand the design principles and the role of lipid complexity in bioengineering of cell membranes.\nHighly parallelized construction of DNA from low-cost oligonucleotide mixtures using Data-optimized Assembly Design and Golden Gate\nAuthors: Lund, S.; Potapov, V.; Johnson, S. R.; Buss, J.; Tanner, N. A.\nScore: 8.7, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567888\nCommercially synthesized genes are typically made using variations of homology-based cloning techniques, including polymerase cycling assembly from chemically synthesized microarray-derived oligonucleotides. Here we apply Data-optimized Assembly Design to the synthesis of hundreds of codon-optimized genes in both constitutive and inducible vectors using Golden Gate Assembly. Starting from oligonucleotide pools, we synthesize genes in three simple steps: 1) Amplification of parts belonging to individual assemblies in parallel from a single pool; 2) Golden Gate Assembly of parts for each construct; and 3) Transformation. We construct genes from receiving DNA to sequence confirmed isolates in as little as 4 days. By leveraging the ligation fidelity afforded by T4 DNA ligase, we expect to be able to construct a larger breadth of sequences not currently supported by homology-based methods which require stability of extensive single-stranded DNA overhangs.\nEvolution is All You Need in Promoter Design and Optimization\nAuthors: Ren, R.; Yu, H.; Teng, J.; Mao, S.; Bian, Z.; Tao, Y.; Yau, S. S.-T.\nScore: 8.6, Published: 2023-11-18 DOI: 10.1101/2023.11.18.567645\nPredicting the strength of promoters and guiding their directed evolution is a crucial task in synthetic biology. This approach significantly reduces the experimental costs in conventional promoter engineering. Previous studies employing machine learning or deep learning methods have shown some success in this task, but their outcomes were not satisfactory enough, primarily due to the neglect of evolutionary information. In this paper, we introduce the Chaos-Attention net for Promoter Evolution (CAPE) to address the limitations of existing methods. We comprehensively extract evolutionary information within promoters using chaos game representation and process the overall information with DenseNet and Transformer. Our model achieves state-of-the-art results on two kinds of distinct tasks. The incorporation of evolutionary information enhances the models accuracy, with transfer learning further extending its adaptability. Furthermore, experimental results confirm CAPEs efficacy in simulating in silico directed evolution of promoters, marking a significant advancement in predictive modeling for prokaryotic promoter strength. Our paper also presents a user-friendly website for the practical implementation of in silico directed evolution on promoters.\nStrong sequence dependence in RNA/DNA hybrid strand displacement kinetics\nAuthors: Smith, F. G.; Goertz, J. P.; Stevens, M. M.; Ouldridge, T. E.\nScore: 3.9, Published: 2023-11-14 DOI: 10.1101/2023.11.14.567030\nStrand displacement reactions underlie dynamic nucleic acid nanotechnology. The kinetic and thermodynamic features of DNA-based displacement reactions are well understood and well predicted by current computational models. By contrast, understanding of RNA/DNA hybrid strand displacement kinetics is limited, restricting the design of increasingly complex RNA/DNA hybrid reaction networks with more tightly regulated dynamics. Given the importance of RNA as a diagnostic biomarker, and its critical role in intracellular processes, this shortfall is particularly limiting for the development of strand displacement-based therapeutics and diagnostics. Herein, we characterise 22 RNA/DNA hybrid strand displacement systems, systematically varying several common design parameters including toehold length and branch migration domain length. We observe the differences in stability between RNA-DNA hybrids and DNA-DNA duplexes have large effects on strand displacement rates, with rates for equivalent sequences differing by up to 3 orders of magnitude. Crucially, however, this effect is strongly sequence-dependent, with RNA invaders strongly favoured in a system with RNA strands of high purine content, and disfavoured in a system when the RNA strands have low purine content. These results lay the groundwork for more general design principles, allowing for creation of de novo reaction networks with novel complexity while maintaining predictable reaction kinetics.\nGenomically integrated orthogonal translation in Escherichia coli, a new synthetic auxotrophic chassis with altered genetic code, genetic firewall, and enhanced protein expression\nAuthors: Karbalaei-Heidari, H. R.; Budisa, N.\nScore: 3.8, Published: 2023-11-19 DOI: 10.1101/2023.11.18.567690\nIn the last three decades, genetic code engineering has expanded protein biosynthesis options from the natural set of 20 canonical amino acids to over 250 non-canonical amino acids (ncAAs). This progress involves rewiring of protein translation by establishing Orthogonal Translation Systems (OTS) through orthogonal pairs. Traditionally encoded on plasmid vectors, these systems are often unstable and burdensome in large-scale fermentations. To moving forward from academia to reliable technology, it is crucial to integrate OTS genetic modules stably into a molecular chassis with a defined genome background. Here, we demonstrate genomically integrated OTS in Escherichia coli, creating a synthetic auxotrophic chassis with an altered genetic code. Using CRISPR-associated transposase tool (CASTs), we targeted multiple genome sites, inserting OTS components (enzymes, tRNA genes) non-disruptively. Our OTS system demonstrated site-specific incorporation of m-oNB-Dopa through in-frame amber stop codon readthrough, enabling the expression of smart underwater bioglues. Simple metabolic labelling, introducing fluoroproline analogs enhancing conformational stability during orthogonal translation, further bolstered system robustness. These chassis, equipped also with synthetic auxotrophy for m-oNB-Dopa, serve as a built-in genetic barrier (genetic firewall), ensuring safe bioproduction in genetically isolated settings.\nThermodynamically-consistent, reduced models of gene regulatory networks\nAuthors: Pan, M.; Gawthrop, P. J.; Faria, M.; Johnston, S. T.\nScore: 4.1, Published: 2023-11-13 DOI: 10.1101/2023.11.13.566770\nSynthetic biology aims to engineer novel functionalities into biological systems. While the approach has been widely applied to single cells, the scale of synthetic circuits designed in this way is limited by factors such as resource competition and retroactivity. Synthetic biology of cell populations has the potential to overcome some of these limitations by physically isolating synthetic genes from each other. To rationally design cell populations, we require mathematical models that link between intracellular biochemistry and intercellular interactions. The interfacing of agent-based models with systems biology models is particularly important in understanding the effects of cell heterogeneity and cell-to-cell interactions. In this study, we develop a model of gene expression that is suitable for incorporation into agent-based models of cell populations. To be scalable to large cell populations, models of gene expression should be both computationally efficient and compliant with the laws of physics. We satisfy the first requirement by applying a model reduction scheme to translation, and the second requirement by formulating models using bond graphs. We show that the reduced model of translation faithfully reproduces the behaviour of the full model at steady state. The reduced model is benchmarked against the full model, and we find a sub-stantial speedup at realistic protein lengths. Using the modularity of the bond graph approach, we couple separate models of gene expression to build models of the toggle switch and repressilator. With these models, we explore the effects of resource availability and cell-to-cell heterogeneity on circuit behaviour. The modelling approaches developed in this study are a bridge towards rationally designing collective cell behaviours such as synchronisation and division of labour.\n",
  "wordCount" : "2279",
  "inLanguage": "en",
  "datePublished": "2023-11-22T10:41:31Z",
  "dateModified": "2023-11-22T10:41:31Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 22, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566922">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566922" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566922">
        <p class="paperTitle">Continuous evolution of user-defined genes at 1-million-times the genomic mutation rate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566922" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566922" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rix, G.; Williams, R. L.; Spinner, H.; Hu, V. J.; Marks, D. S.; Liu, C. C.</p>
        <p class="info">Score: 56.5, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566922' target='https://doi.org/10.1101/2023.11.13.566922'> 10.1101/2023.11.13.566922</a></p>
        <p class="abstract">When nature maintains or evolves a genes function over millions of years at scale, it produces a diversity of homologous sequences whose patterns of conservation and change contain rich structural, functional, and historical information about the gene. However, natural gene diversity likely excludes vast regions of functional sequence space and includes phylogenetic and evolutionary eccentricities, limiting what information we can extract. We introduce an accessible experimental approach for compressing long-term gene evolution to laboratory timescales, allowing for the direct observation of extensive adaptation and divergence followed by inference of structural, functional, and environmental constraints for any selectable gene. To enable this approach, we developed a new orthogonal DNA replication (OrthoRep) system that durably hypermutates chosen genes at a rate of &gt;10-4 substitutions per base in vivo. When OrthoRep was used to evolve a conditionally essential maladapted enzyme, we obtained thousands of unique multi-mutation sequences with many pairs &gt;60 amino acids apart (&gt;15% divergence), revealing known and new factors influencing enzyme adaptation. The fitness of evolved sequences was not predictable by advanced machine learning models trained on natural variation. We suggest that OrthoRep supports the prospective and systematic discovery of constraints shaping gene evolution, uncovering of new regions in fitness landscapes, and general applications in biomolecular engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.19.567736">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.19.567736" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.19.567736">
        <p class="paperTitle">Engineering orthogonal ribosomes for real-time monitoring using fluorescence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.19.567736" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.19.567736" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Csibra, E.; Klopprogge, B.; Hau Sorensen, G.; Gorochowski, T. E.</p>
        <p class="info">Score: 22.2, Published: 2023-11-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.19.567736' target='https://doi.org/10.1101/2023.11.19.567736'> 10.1101/2023.11.19.567736</a></p>
        <p class="abstract">A promising route to tackle the trade-off in cellular resources between synthetic protein production and cellular growth is to use a separate dedicated pool of orthogonal ribosomes to produce synthetic proteins. However, the optimisation of strains containing two ribosomal pools - native for the host cell&#39;s proteome and orthogonal for synthetic proteins - has yet to be thoroughly explored. Here, we address this by creating orthogonal ribosomes that fluoresce by inserting fluorescent RNA aptamers into tethered orthogonal ribosomal RNA (TO-rRNA). To study the tolerance of the engineered ribosomes to aptamer insertion, we assembled and screened a library of candidate insertion sites, identifying several sites in both the 16S and 23S TO-rRNA that enables ribosome labelling with minimal effect on translation activity. Serendipitously, we identify one site in 23S TO-rRNA, where insertion appears to not only be tolerated but to enhance orthogonal ribosome activity, across multiple bacterial strains and RNA insertions. Using bulk and single cell assays, we demonstrate that this variant allows us to label orthogonal ribosomes for dynamic tracking and across populations, making it a promising tool for optimising orthogonal translation in engineered cells. Ribosome engineering offers great potential, both for the development of next-generation microbial cell factories, as well as a tool to expand our understanding of ribosome function in living cells. This work provides a window into the assembly, localisation and function of these molecular machines to meet these aims.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567117">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567117" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567117">
        <p class="paperTitle">Directed Evolution of Genetically Encoded LYTACs for Cell-Mediated Delivery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567117" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567117" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, J. L.; Yamada-Hunter, S. A.; Labanieh, L.; Sotillo, E.; Cheah, J. S.; Roberts, D. S.; Mackall, C.; Ting, A. Y.; Bertozzi, C.</p>
        <p class="info">Score: 20.2, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567117' target='https://doi.org/10.1101/2023.11.14.567117'> 10.1101/2023.11.14.567117</a></p>
        <p class="abstract">Lysosome-targeting chimeras (LYTACs) are a promising therapeutic modality to drive the degradation of extracellular proteins. However, early versions of LYTAC contain synthetic glycopeptides that cannot be genetically encoded. Here we present our designs for a fully genetically encodable LYTAC (GELYTAC), making our tool compatible with integration into therapeutic cells for targeted delivery at diseased sites. To achieve this, we replaced the glycopeptide portion of LYTACs with the protein insulin like growth factor 2 (IGF2). After showing initial efficacy with wild type IGF2, we increased the potency of GELYTAC using directed evolution. Subsequently, we demonstrated that our engineered GELYTAC construct not only secretes from HEK293T cells but also from human primary T-cells to drive the uptake of various targets into receiver cells. Immune cells engineered to secrete GELYTAC thus represent a promising avenue for spatially-selective targeted protein degradation.

Significance StatementBetter therapeutic windows can be achieved by targeting therapeutics to their desired sites of action. For protein therapeutics, this might be achieved by engineering cell therapies that home to a tissue of interest and secrete the biologic drug locally. Here, we demonstrate that human primary T cells can be engineered to produce genetically encoded lysosome targeting chimeras (GELYTACs). These GELYTACs mediate the degradation of extracellular proteins associated with cancer progression. Thus, cells engineered to produce GELYTACs represent a potential new class of cancer therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567131">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567131" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567131">
        <p class="paperTitle">Cryptography in the DNA of living cells enabled by multi-site base editing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567131" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567131" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Volf, V.; Zhang, S.; Song, K. M.; Qian, S.; Chen, F.; Church, G.</p>
        <p class="info">Score: 15.9, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567131' target='https://doi.org/10.1101/2023.11.15.567131'> 10.1101/2023.11.15.567131</a></p>
        <p class="abstract">Although DNA is increasingly being adopted as a generalizable medium for information storage and transfer, reliable methods for ensuring information security remain to be addressed. In this study, we developed and validated a cryptographic encoding scheme, Genomic Sequence Encryption (GSE), to address the challenge of information confidentiality and integrity in biological substrates. GSE enables genomic information encoding that is readable only with a cryptographic key. We show that GSE can be used for cell signatures that enable the recipient of a cell line to authenticate its origin and validate if the cell line has been modified in the interim. We implement GSE through multi-site base editing and encode information through editing across &gt;100 genomic sites in mammalian cells. We further present an enrichment step to obtain individual stem cells with more than two dozen edits across a single genome with minimal screening. This capability can be used to introduce encrypted signatures in living animals. As an encryption scheme, GSE is falsification-proof and enables secure information transfer in biological substrates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563757">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563757" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563757">
        <p class="paperTitle">A tuneable minimal cell membrane reveals that two lipids suffice for life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563757" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563757" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Justice, I. G.; Saenz, J. P.</p>
        <p class="info">Score: 36.9, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563757' target='https://doi.org/10.1101/2023.10.24.563757'> 10.1101/2023.10.24.563757</a></p>
        <p class="abstract">All cells are encapsulated by a lipid membrane which facilitates the interaction between life and its environment. The property and function of the cell membrane is critically dependent on its lipidome composition which can range from 10s to 100s of unique lipid structures. How life has evolved to exploit such diverse mixtures of lipids has been experimentally challenging to address. In this study, we introduce a simple approach to minimize and tune lipidome composition in Mycoplasma mycoides and the Minimal Cell (JCVI-Syn3A). By varying the lipid composition of the growth medium, and introducing unmodifiable diether phospholipids, we demonstrate that a lipidome composed of two lipids can support life. By systematically reintroducing phospholipid features, we show that acyl chain diversity is more critical for supporting growth than head group diversity. We also reveal significant effects of altering phospholipid chirality on membrane permeability, cellular robustness, and growth, demonstrating how subtle changes in lipid stereochemistry can have profound effects on cell fitness. Our approach to tune mycoplasma lipidome composition introduces a minimal living model membrane system to understand the design principles and the role of lipid complexity in bioengineering of cell membranes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567888">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567888" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567888">
        <p class="paperTitle">Highly parallelized construction of DNA from low-cost oligonucleotide mixtures using Data-optimized Assembly Design and Golden Gate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567888" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567888" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lund, S.; Potapov, V.; Johnson, S. R.; Buss, J.; Tanner, N. A.</p>
        <p class="info">Score: 8.7, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567888' target='https://doi.org/10.1101/2023.11.20.567888'> 10.1101/2023.11.20.567888</a></p>
        <p class="abstract">Commercially synthesized genes are typically made using variations of homology-based cloning techniques, including polymerase cycling assembly from chemically synthesized microarray-derived oligonucleotides. Here we apply Data-optimized Assembly Design to the synthesis of hundreds of codon-optimized genes in both constitutive and inducible vectors using Golden Gate Assembly. Starting from oligonucleotide pools, we synthesize genes in three simple steps: 1) Amplification of parts belonging to individual assemblies in parallel from a single pool; 2) Golden Gate Assembly of parts for each construct; and 3) Transformation. We construct genes from receiving DNA to sequence confirmed isolates in as little as 4 days. By leveraging the ligation fidelity afforded by T4 DNA ligase, we expect to be able to construct a larger breadth of sequences not currently supported by homology-based methods which require stability of extensive single-stranded DNA overhangs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.18.567645">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.18.567645" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.18.567645">
        <p class="paperTitle">Evolution is All You Need in Promoter Design and Optimization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.18.567645" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.18.567645" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ren, R.; Yu, H.; Teng, J.; Mao, S.; Bian, Z.; Tao, Y.; Yau, S. S.-T.</p>
        <p class="info">Score: 8.6, Published: 2023-11-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.18.567645' target='https://doi.org/10.1101/2023.11.18.567645'> 10.1101/2023.11.18.567645</a></p>
        <p class="abstract">Predicting the strength of promoters and guiding their directed evolution is a crucial task in synthetic biology. This approach significantly reduces the experimental costs in conventional promoter engineering. Previous studies employing machine learning or deep learning methods have shown some success in this task, but their outcomes were not satisfactory enough, primarily due to the neglect of evolutionary information. In this paper, we introduce the Chaos-Attention net for Promoter Evolution (CAPE) to address the limitations of existing methods. We comprehensively extract evolutionary information within promoters using chaos game representation and process the overall information with DenseNet and Transformer. Our model achieves state-of-the-art results on two kinds of distinct tasks. The incorporation of evolutionary information enhances the models accuracy, with transfer learning further extending its adaptability. Furthermore, experimental results confirm CAPEs efficacy in simulating in silico directed evolution of promoters, marking a significant advancement in predictive modeling for prokaryotic promoter strength. Our paper also presents a user-friendly website for the practical implementation of in silico directed evolution on promoters.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567030">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567030" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567030">
        <p class="paperTitle">Strong sequence dependence in RNA/DNA hybrid strand displacement kinetics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567030" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567030" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smith, F. G.; Goertz, J. P.; Stevens, M. M.; Ouldridge, T. E.</p>
        <p class="info">Score: 3.9, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567030' target='https://doi.org/10.1101/2023.11.14.567030'> 10.1101/2023.11.14.567030</a></p>
        <p class="abstract">Strand displacement reactions underlie dynamic nucleic acid nanotechnology. The kinetic and thermodynamic features of DNA-based displacement reactions are well understood and well predicted by current computational models. By contrast, understanding of RNA/DNA hybrid strand displacement kinetics is limited, restricting the design of increasingly complex RNA/DNA hybrid reaction networks with more tightly regulated dynamics. Given the importance of RNA as a diagnostic biomarker, and its critical role in intracellular processes, this shortfall is particularly limiting for the development of strand displacement-based therapeutics and diagnostics. Herein, we characterise 22 RNA/DNA hybrid strand displacement systems, systematically varying several common design parameters including toehold length and branch migration domain length. We observe the differences in stability between RNA-DNA hybrids and DNA-DNA duplexes have large effects on strand displacement rates, with rates for equivalent sequences differing by up to 3 orders of magnitude. Crucially, however, this effect is strongly sequence-dependent, with RNA invaders strongly favoured in a system with RNA strands of high purine content, and disfavoured in a system when the RNA strands have low purine content. These results lay the groundwork for more general design principles, allowing for creation of de novo reaction networks with novel complexity while maintaining predictable reaction kinetics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.18.567690">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.18.567690" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.18.567690">
        <p class="paperTitle">Genomically integrated orthogonal translation in Escherichia coli, a new synthetic auxotrophic chassis with altered genetic code, genetic firewall, and enhanced protein expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.18.567690" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.18.567690" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Karbalaei-Heidari, H. R.; Budisa, N.</p>
        <p class="info">Score: 3.8, Published: 2023-11-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.18.567690' target='https://doi.org/10.1101/2023.11.18.567690'> 10.1101/2023.11.18.567690</a></p>
        <p class="abstract">In the last three decades, genetic code engineering has expanded protein biosynthesis options from the natural set of 20 canonical amino acids to over 250 non-canonical amino acids (ncAAs). This progress involves rewiring of protein translation by establishing Orthogonal Translation Systems (OTS) through orthogonal pairs. Traditionally encoded on plasmid vectors, these systems are often unstable and burdensome in large-scale fermentations. To moving forward from academia to reliable technology, it is crucial to integrate OTS genetic modules stably into a molecular chassis with a defined genome background. Here, we demonstrate genomically integrated OTS in Escherichia coli, creating a synthetic auxotrophic chassis with an altered genetic code. Using CRISPR-associated transposase tool (CASTs), we targeted multiple genome sites, inserting OTS components (enzymes, tRNA genes) non-disruptively. Our OTS system demonstrated site-specific incorporation of m-oNB-Dopa through in-frame amber stop codon readthrough, enabling the expression of smart underwater bioglues. Simple metabolic labelling, introducing fluoroproline analogs enhancing conformational stability during orthogonal translation, further bolstered system robustness. These chassis, equipped also with synthetic auxotrophy for m-oNB-Dopa, serve as a built-in genetic barrier (genetic firewall), ensuring safe bioproduction in genetically isolated settings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566770">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566770" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566770">
        <p class="paperTitle">Thermodynamically-consistent, reduced models of gene regulatory networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566770" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566770" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pan, M.; Gawthrop, P. J.; Faria, M.; Johnston, S. T.</p>
        <p class="info">Score: 4.1, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566770' target='https://doi.org/10.1101/2023.11.13.566770'> 10.1101/2023.11.13.566770</a></p>
        <p class="abstract">Synthetic biology aims to engineer novel functionalities into biological systems. While the approach has been widely applied to single cells, the scale of synthetic circuits designed in this way is limited by factors such as resource competition and retroactivity. Synthetic biology of cell populations has the potential to overcome some of these limitations by physically isolating synthetic genes from each other. To rationally design cell populations, we require mathematical models that link between intracellular biochemistry and intercellular interactions. The interfacing of agent-based models with systems biology models is particularly important in understanding the effects of cell heterogeneity and cell-to-cell interactions. In this study, we develop a model of gene expression that is suitable for incorporation into agent-based models of cell populations. To be scalable to large cell populations, models of gene expression should be both computationally efficient and compliant with the laws of physics. We satisfy the first requirement by applying a model reduction scheme to translation, and the second requirement by formulating models using bond graphs. We show that the reduced model of translation faithfully reproduces the behaviour of the full model at steady state. The reduced model is benchmarked against the full model, and we find a sub-stantial speedup at realistic protein lengths. Using the modularity of the bond graph approach, we couple separate models of gene expression to build models of the toggle switch and repressilator. With these models, we explore the effects of resource availability and cell-to-cell heterogeneity on circuit behaviour. The modelling approaches developed in this study are a bridge towards rationally designing collective cell behaviours such as synchronisation and division of labour.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
