<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Truly tiny acoustic biomolecules for ultrasound imaging and therapy
Authors: Ling, B.; Gungoren, B.; Yao, Y.; Dutka, P.; Smith, C. A. B.; Lee, J.; Swift, M. B.; Shapiro, M. G.
Score: 15.2, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546773
Nanotechnology offers significant advantages for medical imaging and therapy, including enhanced contrast and precision targeting. However, integrating these benefits into ultrasonography has been challenging due to the size and stability constraints of conventional bubble-based agents.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Truly tiny acoustic biomolecules for ultrasound imaging and therapy
Authors: Ling, B.; Gungoren, B.; Yao, Y.; Dutka, P.; Smith, C. A. B.; Lee, J.; Swift, M. B.; Shapiro, M. G.
Score: 15.2, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546773
Nanotechnology offers significant advantages for medical imaging and therapy, including enhanced contrast and precision targeting. However, integrating these benefits into ultrasonography has been challenging due to the size and stability constraints of conventional bubble-based agents." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-02T10:39:32+00:00" />
<meta property="article:modified_time" content="2023-07-02T10:39:32+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Truly tiny acoustic biomolecules for ultrasound imaging and therapy
Authors: Ling, B.; Gungoren, B.; Yao, Y.; Dutka, P.; Smith, C. A. B.; Lee, J.; Swift, M. B.; Shapiro, M. G.
Score: 15.2, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546773
Nanotechnology offers significant advantages for medical imaging and therapy, including enhanced contrast and precision targeting. However, integrating these benefits into ultrasonography has been challenging due to the size and stability constraints of conventional bubble-based agents."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Truly tiny acoustic biomolecules for ultrasound imaging and therapy\nAuthors: Ling, B.; Gungoren, B.; Yao, Y.; Dutka, P.; Smith, C. A. B.; Lee, J.; Swift, M. B.; Shapiro, M. G.\nScore: 15.2, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546773\nNanotechnology offers significant advantages for medical imaging and therapy, including enhanced contrast and precision targeting. However, integrating these benefits into ultrasonography has been challenging due to the size and stability constraints of conventional bubble-based agents.",
  "keywords": [
    
  ],
  "articleBody": " Truly tiny acoustic biomolecules for ultrasound imaging and therapy\nAuthors: Ling, B.; Gungoren, B.; Yao, Y.; Dutka, P.; Smith, C. A. B.; Lee, J.; Swift, M. B.; Shapiro, M. G.\nScore: 15.2, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546773\nNanotechnology offers significant advantages for medical imaging and therapy, including enhanced contrast and precision targeting. However, integrating these benefits into ultrasonography has been challenging due to the size and stability constraints of conventional bubble-based agents. Here we describe bicones, truly tiny acoustic contrast agents based on gas vesicles, a unique class of air-filled protein nanostructures naturally produced in buoyant microbes. We show that these sub-80 nm particles can be effectively detected both in vitro and in vivo, infiltrate tumors via leaky vasculature, deliver potent mechanical effects through ultrasound-induced inertial cavitation, and are easily engineered for molecular targeting, prolonged circulation time, and payload conjugation.\nA synthetic biology approach to assemble and reboot clinically-relevant Pseudomonas aeruginosa tailed phages\nAuthors: Ipoutcha, T.; Racharaks, R.; Huttelmaier, S.; Wilson, C. J.; Ozer, E. A.; Hartmann, E. M.\nScore: 12.1, Published: 2023-06-24 DOI: 10.1101/2023.06.23.546310\nThe rise in frequency of antibiotic resistance has made bacterial infections, specifically Pseudomonas aeruginosa, a cause for greater concern. Phage therapy is a promising solution that uses naturally isolated phages to treat bacterial infections. Ecological limitations, which stipulate a discrete host range and the inevitable evolution of resistance, may be overcome through a better understanding of phage biology and the utilization of engineered phages. In this study, we developed a synthetic biology approach to construct tailed phages that naturally target clinically-relevant strains of Pseudomonas aeruginosa. As proof of concept, we successfully cloned and assembled the JG024 and DMS3 phage genomes in yeast using transformation-associated recombination (TAR) cloning and rebooted these two phage genomes in two different strains of P. aeruginosa. We identified factors that affected phage reboot efficiency like the phage species or the presence of antiviral defense systems in the bacterial strain. We have successfully extended this method to two other phage species and observed that the method enables the reboot of phages that are naturally unable to infect the strain used for reboot. This research represents a critical step towards the construction of clinically-relevant, engineered P. aeruginosa phages. ImportancePseudomonas aeruginosa is a bacterium responsible for severe infections and a common major complication in cystic fibrosis. The use of antibiotics to treat bacterial infections has become increasingly difficult as antibiotic resistance has become more prevalent. Phage therapy is an alternative solution that is already being used in some European countries, but its use is limited by narrow host range due to the phage receptor specificity, the presence of antiviral defense systems in the bacterial strain, and the possible emergence of phage resistance. In this study, we demonstrate the use of a synthetic biology approach to construct and reboot clinically-relevant P. aeruginosa tailed phages. This method enables a significant expansion of possibilities through the construction of engineered phages for therapy applications.\nC2CAplus: a one-pot isothermal circle-to-circle DNA amplification system\nAuthors: Grasemann, L.; Pizarro, P. T.; Maerkl, S. J.\nScore: 7.7, Published: 2023-06-26 DOI: 10.1101/2023.06.26.546530\nRolling circle amplification (RCA) is a widely used DNA amplification method that uses circular template DNA as input and produces multimeric, linear single or double stranded DNA. Circle-to-circle amplification (C2CA) has further expanded this method by implementing product re-circularization using restriction and ligation, leading to a higher amplification yield, and enabling the generation of circular products. However, C2CA is a multistep, non-isothermal method, requiring multiple fluid manipulations and thereby compromises several advantages of RCA. Here, we improved C2CA to implement a one-pot, single step, isothermal reaction at temperatures ranging from 25 to 37{degrees}C. Our C2CAplus method is simple, robust, and produces large quantities of product DNA that can be seen with the naked eye.\n50-nm gas-filled protein nanostructures to enable the access of lymphatic cells by ultrasound technologies\nAuthors: Shen, Q.; Li, Z.; Meyer, M. D.; De Guzman, M. T.; Lim, J. C.; Bouchard, R. R.; Lu, G. J.\nScore: 6.5, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546433\nUltrasound imaging and ultrasound-mediated gene and drug delivery are rapidly advancing diagnostic and therapeutic methods; however, their use is often limited by the need of microbubbles, which cannot transverse many biological barriers due to their large size. Here we introduce 50-nm gas-filled protein nanostructures derived from genetically engineered gas vesicles that we referred to as 50nmGVs. These diamond-shaped nanostructures have hydrodynamic diameters smaller than commercially available 50-nm gold nanoparticles and are, to our knowledge, the smallest stable, free-floating bubbles made to date. 50nmGVs can be produced in bacteria, purified through centrifugation, and remain stable for months. Interstitially injected 50nmGVs can extravasate into lymphatic tissues and gain access to critical immune cell populations, and electron microscopy images of lymph node tissues reveal their subcellular location in antigen-presenting cells adjacent to lymphocytes. We anticipate that 50nmGVs can substantially broaden the range of cells accessible to current ultrasound technologies and may generate applications beyond biomedicine as ultrasmall stable gas-filled nanomaterials.\nImpact of random 50-base sequences inserted into an intron on splicing in Saccharomyces cerevisiae\nAuthors: Perchlik, M.; Sasse, A.; Mostafavi, S.; Fields, S.; Cuperus, J.\nScore: 6.5, Published: 2023-06-24 DOI: 10.1101/2023.06.21.545966\nIntron splicing is a key regulatory step in gene expression in eukaryotes. Three sequence elements required for splicing - 5 and 3 splice sites and a branch point - are especially well- characterized in Saccharomyces cerevisiae, but our understanding of additional intron features that impact splicing in this organism is incomplete, due largely to its small number of introns. To overcome this limitation, we constructed a library in S. cerevisiae of random 50-nucleotide elements (N50) individually inserted into the intron of a reporter gene and quantified canonical splicing and the use of cryptic splice sites by sequencing analysis. More than 70% of approximately 140,000 N50 elements reduced splicing by at least 20% compared to the intron control. N50 features, including higher GC content, presence of GU repeats and stronger predicted secondary structure of its pre-mRNA, correlated with reduced splicing efficiency. A likely basis for the reduced splicing of such a large proportion of variants is the formation of RNA structures that pair N50 bases - such as the GU repeats - with other bases specifically within the reporter pre-mRNA analyzed. However, neither convolutional neural network nor linear models were able to explain more than a small fraction of the variance in splicing efficiency across the library, suggesting that complex non-linear interactions in RNA structures are not accurately captured by RNA structure prediction methods given the limited number of variants. Our results imply that the specific context of a pre-mRNA may determine the bases allowable in an intron to prevent secondary structures that reduce splicing.\nA Platform for Controlled Template-Independent Enzymatic Synthesis of RNA Oligonucleotides and Therapeutics\nAuthors: Wiegand, D. J.; Rittichier, J.; Meyer, E.; Lee, H.; Conway, N. J.; Ahlstedt, D.; Yurtsever, Z.; Rainone, D.; Kuru, E.; Church, G.\nScore: 3.9, Published: 2023-06-29 DOI: 10.1101/2023.06.29.547106\nTherapeutic RNA oligonucleotides have shown tremendous potential to manage and treat disease, yet current manufacturing methods cannot deliver on this promise. Here, we report the development and optimization of a novel, aqueous-based, template-independent enzymatic RNA oligonucleotide synthesis platform as an alternative to traditional chemical methodologies. Our platform is made possible by reversible terminator nucleoside triphosphates and an enzyme capable of their incorporation. We show that many common therapeutic RNA modifications are compatible with our process and demonstrate the enzymatic synthesis of natural and modified oligonucleotides in both liquid and solid phases. Our platform offers many unique advantages over chemical synthesis, including the realization of a more sustainable process to produce therapeutic RNA oligonucleotides.\nHarnessing Escherichia coli Dark Genome to Produce Anti-Alzheimer Peptides\nAuthors: Verma, N.; Manvati, S.; Dhar, P.\nScore: 3.4, Published: 2023-06-24 DOI: 10.1101/2023.06.23.546343\nAlzheimers disease (AD) is characterized by progressive neurodegeneration. The critical molecular trigger is believed to be the accumulation of A{beta} neurotoxic oligomers. Given the proteolytic processing of Amyloid Precursor Protein (APP) by {beta}-secretase (beta-site APP cleaving enzyme 1, BACE1) as the key step in the building up of A{beta} oligomers, BACE inhibitors come with therapeutic prospects of preventing or delaying the onset of Alzheimers. To find inhibitory peptides against BACE1, a library of dark peptides was constructed from 4400 intergenic DNA sequences of Escherichia coli. The sequence level analysis was followed by protein structure predictions, molecular docking, and simulation. Based on bioinformatics analysis, 5 potential peptides were screened for experimental validation. Out of these two peptides were identified as lead molecules based on BACE1 inhibitory activity, followed by FRET inhibitory assay, western blot, and RT-PCR. An 86.7 % drop in BACE1 level was observed in the presence of the ECOI2 peptide. Though encouraging results were obtained from in-silico and in-vitro studies, more work is required to study the efficacy of these peptides in suitable animal models.\nIn vitro construction and long read sequencing analysis of a 24 kb long artificial DNA sequence encoding the Universal Declaration of the Rights of Man and of the Citizen.\nAuthors: Leblanc, J.; Boulle, O.; Roux, E.; Nicolas, J.; Lavenier, D.; Audic, Y.\nScore: 3.1, Published: 2023-06-26 DOI: 10.1101/2023.06.26.546242\nIn absence of DNA template, the ab initio production of long double-stranded DNA molecules of predefined sequences is particularly challenging. The DNA synthesis step remains a bottleneck for many applications such as functional assessment of ancestral genes, analysis of alternative splicing or DNA-based data storage. We propose in this report a fully in vitro protocol to generate very long double-stranded DNA molecule starting from commercially available short DNA blocks in less than 3 days. This innovative application of Golden Gate assembly allowed us to streamline the assembly process to produce a 24 kb long DNA molecule storing part of the Universal Declaration of Human rights and citizens. The DNA molecule produced can be readily cloned into suitable host/vector system for amplification and selection.\nEnabling AI in Synthetic Biology through Construction File Specification\nAuthors: Ataii, N.; Bakshi, S.; Chen, Y.; Fernandez, M.; Scheftel, Z.; Shao, Z.; Tou, C. J.; Vega, M.; Wang, Y.; Zhang, H.; Zhao, Z.; Anderson, J. C.\nScore: 2.8, Published: 2023-06-28 DOI: 10.1101/2023.06.28.546630\nThe Construction File (CF) specification establishes a standardized interface for molecular biology operations, laying a foundation for automation and enhanced efficiency in experiment design. It is implemented across three distinct software projects: PyDNA_CF_Simulator, a Python project featuring a ChatGPT plugin for interactive parsing and simulating experiments; ConstructionFileSimulator, a field-tested Java project that showcases Experiment objects expressed as flat files; and C6-Tools, a JavaScript project integrated with Google Sheets via Apps Script, providing a user-friendly interface for authoring and simulation of CF. The CF specification not only standardizes and modularizes molecular biology operations but also promotes collaboration, automation, and reuse, significantly reducing potential errors. The potential integration of CF with artificial intelligence, particularly GPT-4, suggests innovative automation strategies for synthetic biology. While challenges such as token limits, data storage, and biosecurity remain, proposed solutions promise a way forward in harnessing AI for experiment design. This shift from human-driven design to AI-assisted workflows, steered by high-level objectives, charts a potential future path in synthetic biology, envisioning an environment where complexities are managed more effectively.\nAutomated Microbial Classification System based on Deep Convolutional Neural Networks using Images from Colony Picker\nAuthors: Park, S.; Yeoh, J. W.; Choo, C. T.; Lim, C. K.; Dao, V. L.; Poh, C. L.\nScore: 1.5, Published: 2023-06-30 DOI: 10.1101/2023.06.29.547155\nColony screening in single and multi-species environments is an essential step for microbiome studies. However, it possesses a high possibility of inaccurately classifying the species of interest and demands a high degree of manpower and time. Thus, automating the classification of microbes is beneficial to minimize the time and inaccuracy in the colony screening/picking step. Here, we developed an automated microbial classification system for five target species, based on deep Convolutional Neural Networks (CNN) using images captured by an automated robotic colony picker. Multiple possible scenarios of colony culture and diverse morphologies of colonies were examined in building the training and test datasets to train and validate the model and performance on real-life implementations. The final model trained using 60,000 training images, with 12,000 images per species and 3-fold cross-validation, achieved a test accuracy of 94.2% and a test loss of 0.154. Upon testing using a deployment dataset of 4,500 images (900 images per species) with different methods of applying cells onto the agar plate, high accuracy of up to 96.6% was obtained. Five evaluation metrics were implemented to evaluate diverse scenarios of the test data to justify the validity of the model in real-life applications. This model forms a foundation for the classification of more species through transfer learning in the future.\n",
  "wordCount" : "2094",
  "inLanguage": "en",
  "datePublished": "2023-07-02T10:39:32Z",
  "dateModified": "2023-07-02T10:39:32Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on July 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546773">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546773" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546773">
        <p class="paperTitle">Truly tiny acoustic biomolecules for ultrasound imaging and therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546773" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546773" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ling, B.; Gungoren, B.; Yao, Y.; Dutka, P.; Smith, C. A. B.; Lee, J.; Swift, M. B.; Shapiro, M. G.</p>
        <p class="info">Score: 15.2, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546773' target='https://doi.org/10.1101/2023.06.27.546773'> 10.1101/2023.06.27.546773</a></p>
        <p class="abstract">Nanotechnology offers significant advantages for medical imaging and therapy, including enhanced contrast and precision targeting. However, integrating these benefits into ultrasonography has been challenging due to the size and stability constraints of conventional bubble-based agents. Here we describe bicones, truly tiny acoustic contrast agents based on gas vesicles, a unique class of air-filled protein nanostructures naturally produced in buoyant microbes. We show that these sub-80 nm particles can be effectively detected both in vitro and in vivo, infiltrate tumors via leaky vasculature, deliver potent mechanical effects through ultrasound-induced inertial cavitation, and are easily engineered for molecular targeting, prolonged circulation time, and payload conjugation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546310">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546310" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546310">
        <p class="paperTitle">A synthetic biology approach to assemble and reboot clinically-relevant Pseudomonas aeruginosa tailed phages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546310" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546310" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ipoutcha, T.; Racharaks, R.; Huttelmaier, S.; Wilson, C. J.; Ozer, E. A.; Hartmann, E. M.</p>
        <p class="info">Score: 12.1, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546310' target='https://doi.org/10.1101/2023.06.23.546310'> 10.1101/2023.06.23.546310</a></p>
        <p class="abstract">The rise in frequency of antibiotic resistance has made bacterial infections, specifically Pseudomonas aeruginosa, a cause for greater concern. Phage therapy is a promising solution that uses naturally isolated phages to treat bacterial infections. Ecological limitations, which stipulate a discrete host range and the inevitable evolution of resistance, may be overcome through a better understanding of phage biology and the utilization of engineered phages. In this study, we developed a synthetic biology approach to construct tailed phages that naturally target clinically-relevant strains of Pseudomonas aeruginosa. As proof of concept, we successfully cloned and assembled the JG024 and DMS3 phage genomes in yeast using transformation-associated recombination (TAR) cloning and rebooted these two phage genomes in two different strains of P. aeruginosa. We identified factors that affected phage reboot efficiency like the phage species or the presence of antiviral defense systems in the bacterial strain. We have successfully extended this method to two other phage species and observed that the method enables the reboot of phages that are naturally unable to infect the strain used for reboot. This research represents a critical step towards the construction of clinically-relevant, engineered P. aeruginosa phages.

ImportancePseudomonas aeruginosa is a bacterium responsible for severe infections and a common major complication in cystic fibrosis. The use of antibiotics to treat bacterial infections has become increasingly difficult as antibiotic resistance has become more prevalent. Phage therapy is an alternative solution that is already being used in some European countries, but its use is limited by narrow host range due to the phage receptor specificity, the presence of antiviral defense systems in the bacterial strain, and the possible emergence of phage resistance. In this study, we demonstrate the use of a synthetic biology approach to construct and reboot clinically-relevant P. aeruginosa tailed phages. This method enables a significant expansion of possibilities through the construction of engineered phages for therapy applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546530">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546530" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546530">
        <p class="paperTitle">C2CAplus: a one-pot isothermal circle-to-circle DNA amplification system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546530" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546530" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grasemann, L.; Pizarro, P. T.; Maerkl, S. J.</p>
        <p class="info">Score: 7.7, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546530' target='https://doi.org/10.1101/2023.06.26.546530'> 10.1101/2023.06.26.546530</a></p>
        <p class="abstract">Rolling circle amplification (RCA) is a widely used DNA amplification method that uses circular template DNA as input and produces multimeric, linear single or double stranded DNA. Circle-to-circle amplification (C2CA) has further expanded this method by implementing product re-circularization using restriction and ligation, leading to a higher amplification yield, and enabling the generation of circular products. However, C2CA is a multistep, non-isothermal method, requiring multiple fluid manipulations and thereby compromises several advantages of RCA. Here, we improved C2CA to implement a one-pot, single step, isothermal reaction at temperatures ranging from 25 to 37{degrees}C. Our C2CAplus method is simple, robust, and produces large quantities of product DNA that can be seen with the naked eye.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546433">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546433" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546433">
        <p class="paperTitle">50-nm gas-filled protein nanostructures to enable the access of lymphatic cells by ultrasound technologies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546433" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546433" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shen, Q.; Li, Z.; Meyer, M. D.; De Guzman, M. T.; Lim, J. C.; Bouchard, R. R.; Lu, G. J.</p>
        <p class="info">Score: 6.5, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546433' target='https://doi.org/10.1101/2023.06.27.546433'> 10.1101/2023.06.27.546433</a></p>
        <p class="abstract">Ultrasound imaging and ultrasound-mediated gene and drug delivery are rapidly advancing diagnostic and therapeutic methods; however, their use is often limited by the need of microbubbles, which cannot transverse many biological barriers due to their large size. Here we introduce 50-nm gas-filled protein nanostructures derived from genetically engineered gas vesicles that we referred to as 50nmGVs. These diamond-shaped nanostructures have hydrodynamic diameters smaller than commercially available 50-nm gold nanoparticles and are, to our knowledge, the smallest stable, free-floating bubbles made to date. 50nmGVs can be produced in bacteria, purified through centrifugation, and remain stable for months. Interstitially injected 50nmGVs can extravasate into lymphatic tissues and gain access to critical immune cell populations, and electron microscopy images of lymph node tissues reveal their subcellular location in antigen-presenting cells adjacent to lymphocytes. We anticipate that 50nmGVs can substantially broaden the range of cells accessible to current ultrasound technologies and may generate applications beyond biomedicine as ultrasmall stable gas-filled nanomaterials.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545966">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545966" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545966">
        <p class="paperTitle">Impact of random 50-base sequences inserted into an intron on splicing in Saccharomyces cerevisiae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545966" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545966" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Perchlik, M.; Sasse, A.; Mostafavi, S.; Fields, S.; Cuperus, J.</p>
        <p class="info">Score: 6.5, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545966' target='https://doi.org/10.1101/2023.06.21.545966'> 10.1101/2023.06.21.545966</a></p>
        <p class="abstract">Intron splicing is a key regulatory step in gene expression in eukaryotes. Three sequence elements required for splicing - 5 and 3 splice sites and a branch point - are especially well- characterized in Saccharomyces cerevisiae, but our understanding of additional intron features that impact splicing in this organism is incomplete, due largely to its small number of introns. To overcome this limitation, we constructed a library in S. cerevisiae of random 50-nucleotide elements (N50) individually inserted into the intron of a reporter gene and quantified canonical splicing and the use of cryptic splice sites by sequencing analysis. More than 70% of approximately 140,000 N50 elements reduced splicing by at least 20% compared to the intron control. N50 features, including higher GC content, presence of GU repeats and stronger predicted secondary structure of its pre-mRNA, correlated with reduced splicing efficiency. A likely basis for the reduced splicing of such a large proportion of variants is the formation of RNA structures that pair N50 bases - such as the GU repeats - with other bases specifically within the reporter pre-mRNA analyzed. However, neither convolutional neural network nor linear models were able to explain more than a small fraction of the variance in splicing efficiency across the library, suggesting that complex non-linear interactions in RNA structures are not accurately captured by RNA structure prediction methods given the limited number of variants. Our results imply that the specific context of a pre-mRNA may determine the bases allowable in an intron to prevent secondary structures that reduce splicing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547106">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547106" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547106">
        <p class="paperTitle">A Platform for Controlled Template-Independent Enzymatic Synthesis of RNA Oligonucleotides and Therapeutics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547106" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547106" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wiegand, D. J.; Rittichier, J.; Meyer, E.; Lee, H.; Conway, N. J.; Ahlstedt, D.; Yurtsever, Z.; Rainone, D.; Kuru, E.; Church, G.</p>
        <p class="info">Score: 3.9, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547106' target='https://doi.org/10.1101/2023.06.29.547106'> 10.1101/2023.06.29.547106</a></p>
        <p class="abstract">Therapeutic RNA oligonucleotides have shown tremendous potential to manage and treat disease, yet current manufacturing methods cannot deliver on this promise. Here, we report the development and optimization of a novel, aqueous-based, template-independent enzymatic RNA oligonucleotide synthesis platform as an alternative to traditional chemical methodologies. Our platform is made possible by reversible terminator nucleoside triphosphates and an enzyme capable of their incorporation. We show that many common therapeutic RNA modifications are compatible with our process and demonstrate the enzymatic synthesis of natural and modified oligonucleotides in both liquid and solid phases. Our platform offers many unique advantages over chemical synthesis, including the realization of a more sustainable process to produce therapeutic RNA oligonucleotides.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546343">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546343" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546343">
        <p class="paperTitle">Harnessing Escherichia coli Dark Genome to Produce Anti-Alzheimer Peptides</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546343" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546343" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Verma, N.; Manvati, S.; Dhar, P.</p>
        <p class="info">Score: 3.4, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546343' target='https://doi.org/10.1101/2023.06.23.546343'> 10.1101/2023.06.23.546343</a></p>
        <p class="abstract">Alzheimers disease (AD) is characterized by progressive neurodegeneration. The critical molecular trigger is believed to be the accumulation of A{beta} neurotoxic oligomers. Given the proteolytic processing of Amyloid Precursor Protein (APP) by {beta}-secretase (beta-site APP cleaving enzyme 1, BACE1) as the key step in the building up of A{beta} oligomers, BACE inhibitors come with therapeutic prospects of preventing or delaying the onset of Alzheimers. To find inhibitory peptides against BACE1, a library of  dark peptides was constructed from 4400 intergenic DNA sequences of Escherichia coli. The sequence level analysis was followed by protein structure predictions, molecular docking, and simulation. Based on bioinformatics analysis, 5 potential peptides were screened for experimental validation. Out of these two peptides were identified as lead molecules based on BACE1 inhibitory activity, followed by FRET inhibitory assay, western blot, and RT-PCR. An 86.7 % drop in BACE1 level was observed in the presence of the ECOI2 peptide. Though encouraging results were obtained from in-silico and in-vitro studies, more work is required to study the efficacy of these peptides in suitable animal models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546242">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546242" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546242">
        <p class="paperTitle">In vitro construction and long read sequencing analysis of a 24 kb long artificial DNA sequence encoding the Universal Declaration of the Rights of Man and of the Citizen.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546242" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546242" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Leblanc, J.; Boulle, O.; Roux, E.; Nicolas, J.; Lavenier, D.; Audic, Y.</p>
        <p class="info">Score: 3.1, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546242' target='https://doi.org/10.1101/2023.06.26.546242'> 10.1101/2023.06.26.546242</a></p>
        <p class="abstract">In absence of DNA template, the ab initio production of long double-stranded DNA molecules of predefined sequences is particularly challenging. The DNA synthesis step remains a bottleneck for many applications such as functional assessment of ancestral genes, analysis of alternative splicing or DNA-based data storage. We propose in this report a fully in vitro protocol to generate very long double-stranded DNA molecule starting from commercially available short DNA blocks in less than 3 days. This innovative application of Golden Gate assembly allowed us to streamline the assembly process to produce a 24 kb long DNA molecule storing part of the Universal Declaration of Human rights and citizens. The DNA molecule produced can be readily cloned into suitable host/vector system for amplification and selection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546630">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546630" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546630">
        <p class="paperTitle">Enabling AI in Synthetic Biology through Construction File Specification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546630" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546630" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ataii, N.; Bakshi, S.; Chen, Y.; Fernandez, M.; Scheftel, Z.; Shao, Z.; Tou, C. J.; Vega, M.; Wang, Y.; Zhang, H.; Zhao, Z.; Anderson, J. C.</p>
        <p class="info">Score: 2.8, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546630' target='https://doi.org/10.1101/2023.06.28.546630'> 10.1101/2023.06.28.546630</a></p>
        <p class="abstract">The Construction File (CF) specification establishes a standardized interface for molecular biology operations, laying a foundation for automation and enhanced efficiency in experiment design. It is implemented across three distinct software projects: PyDNA_CF_Simulator, a Python project featuring a ChatGPT plugin for interactive parsing and simulating experiments; ConstructionFileSimulator, a field-tested Java project that showcases  Experiment objects expressed as flat files; and C6-Tools, a JavaScript project integrated with Google Sheets via Apps Script, providing a user-friendly interface for authoring and simulation of CF. The CF specification not only standardizes and modularizes molecular biology operations but also promotes collaboration, automation, and reuse, significantly reducing potential errors. The potential integration of CF with artificial intelligence, particularly GPT-4, suggests innovative automation strategies for synthetic biology. While challenges such as token limits, data storage, and biosecurity remain, proposed solutions promise a way forward in harnessing AI for experiment design. This shift from human-driven design to AI-assisted workflows, steered by high-level objectives, charts a potential future path in synthetic biology, envisioning an environment where complexities are managed more effectively.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547155">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547155" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547155">
        <p class="paperTitle">Automated Microbial Classification System based on Deep Convolutional Neural Networks using Images from Colony Picker</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547155" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547155" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Park, S.; Yeoh, J. W.; Choo, C. T.; Lim, C. K.; Dao, V. L.; Poh, C. L.</p>
        <p class="info">Score: 1.5, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547155' target='https://doi.org/10.1101/2023.06.29.547155'> 10.1101/2023.06.29.547155</a></p>
        <p class="abstract">Colony screening in single and multi-species environments is an essential step for microbiome studies. However, it possesses a high possibility of inaccurately classifying the species of interest and demands a high degree of manpower and time. Thus, automating the classification of microbes is beneficial to minimize the time and inaccuracy in the colony screening/picking step. Here, we developed an automated microbial classification system for five target species, based on deep Convolutional Neural Networks (CNN) using images captured by an automated robotic colony picker. Multiple possible scenarios of colony culture and diverse morphologies of colonies were examined in building the training and test datasets to train and validate the model and performance on real-life implementations. The final model trained using 60,000 training images, with 12,000 images per species and 3-fold cross-validation, achieved a test accuracy of 94.2% and a test loss of 0.154. Upon testing using a deployment dataset of 4,500 images (900 images per species) with different methods of applying cells onto the agar plate, high accuracy of up to 96.6% was obtained. Five evaluation metrics were implemented to evaluate diverse scenarios of the test data to justify the validity of the model in real-life applications. This model forms a foundation for the classification of more species through transfer learning in the future.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
