<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Validation of cell-free protein synthesis aboard the International Space Station
Authors: Kocalar, S.; Miller, B. M.; Huang, A.; Gleason, E.; Martin, K.; Foley, K.; Copeland, D. S.; Jewett, M.; Saavedra, E. A.; Kraves, S.
Score: 18.8, Published: 2023-12-06 DOI: 10.1101/2023.12.06.570403
Cell-free protein synthesis (CFPS) is a rapidly maturing in vitro gene expression platform that can be used to transcribe and translate nucleic acids at the point of need, enabling on-demand synthesis of peptide-based vaccines and biotherapeutics, as well as the development of diagnostic tests for environmental contaminants and infectious agents.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Validation of cell-free protein synthesis aboard the International Space Station
Authors: Kocalar, S.; Miller, B. M.; Huang, A.; Gleason, E.; Martin, K.; Foley, K.; Copeland, D. S.; Jewett, M.; Saavedra, E. A.; Kraves, S.
Score: 18.8, Published: 2023-12-06 DOI: 10.1101/2023.12.06.570403
Cell-free protein synthesis (CFPS) is a rapidly maturing in vitro gene expression platform that can be used to transcribe and translate nucleic acids at the point of need, enabling on-demand synthesis of peptide-based vaccines and biotherapeutics, as well as the development of diagnostic tests for environmental contaminants and infectious agents." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-10T10:37:48+00:00" />
<meta property="article:modified_time" content="2023-12-10T10:37:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Validation of cell-free protein synthesis aboard the International Space Station
Authors: Kocalar, S.; Miller, B. M.; Huang, A.; Gleason, E.; Martin, K.; Foley, K.; Copeland, D. S.; Jewett, M.; Saavedra, E. A.; Kraves, S.
Score: 18.8, Published: 2023-12-06 DOI: 10.1101/2023.12.06.570403
Cell-free protein synthesis (CFPS) is a rapidly maturing in vitro gene expression platform that can be used to transcribe and translate nucleic acids at the point of need, enabling on-demand synthesis of peptide-based vaccines and biotherapeutics, as well as the development of diagnostic tests for environmental contaminants and infectious agents."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Validation of cell-free protein synthesis aboard the International Space Station\nAuthors: Kocalar, S.; Miller, B. M.; Huang, A.; Gleason, E.; Martin, K.; Foley, K.; Copeland, D. S.; Jewett, M.; Saavedra, E. A.; Kraves, S.\nScore: 18.8, Published: 2023-12-06 DOI: 10.1101/2023.12.06.570403\nCell-free protein synthesis (CFPS) is a rapidly maturing in vitro gene expression platform that can be used to transcribe and translate nucleic acids at the point of need, enabling on-demand synthesis of peptide-based vaccines and biotherapeutics, as well as the development of diagnostic tests for environmental contaminants and infectious agents.",
  "keywords": [
    
  ],
  "articleBody": " Validation of cell-free protein synthesis aboard the International Space Station\nAuthors: Kocalar, S.; Miller, B. M.; Huang, A.; Gleason, E.; Martin, K.; Foley, K.; Copeland, D. S.; Jewett, M.; Saavedra, E. A.; Kraves, S.\nScore: 18.8, Published: 2023-12-06 DOI: 10.1101/2023.12.06.570403\nCell-free protein synthesis (CFPS) is a rapidly maturing in vitro gene expression platform that can be used to transcribe and translate nucleic acids at the point of need, enabling on-demand synthesis of peptide-based vaccines and biotherapeutics, as well as the development of diagnostic tests for environmental contaminants and infectious agents. Unlike traditional cell-based systems, CFPS platforms do not require the maintenance of living cells and can be deployed with minimal equipment; therefore, they hold promise for applications in low-resource contexts, including spaceflight. Here we evaluate the performance of cell-free BioBits(R) platform aboard the International Space Station by expressing RNA-based aptamers and fluorescent proteins that can serve as biological indicators. We validate two classes of biological sensors that detect either the small molecule DFHBI or a specific RNA sequence. Upon detection of their respective analytes, both biological sensors produce fluorescent readouts that are visually confirmed using a handheld fluorescence viewer and imaged for quantitative analysis. Our findings provide insight into the kinetics of cell-free transcription and translation in a microgravity environment and reveal that both biosensors perform robustly in space. Our findings lay the groundwork for portable, low-cost applications ranging from point-of-care health monitoring to on-demand detection of environmental hazards in low-resource communities both on Earth and beyond. Visual graphic O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=123 SRC=\"FIGDIR/small/570403v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (47K): org.highwire.dtl.DTLVardef@172406aorg.highwire.dtl.DTLVardef@1e315d8org.highwire.dtl.DTLVardef@1af3598org.highwire.dtl.DTLVardef@1e7c563_HPS_FORMAT_FIGEXP M_FIG C_FIG\nDemonstration of Bioplastic Production from CO2 and Formate using the Reductive Glycine Pathway in E. coli\nAuthors: Fedorova, D.; Ben-Nissan, R.; Milshtein, E.; Jona, G.; Dezorella, N.; Feiguelman, G.; Fedorov, R.; Gomaa, A.; Lindner, A. B.; Noor, E.; Milo, R.\nScore: 9.6, Published: 2023-12-03 DOI: 10.1101/2023.12.02.569694\nThere is a strong need to develop technologies that reduce greenhouse gas emissions. Recently, progress was made in engineering the model organism E. coli to grow using CO2 and formate as its only carbon and energy sources using the reductive glycine pathway (rGlyP). Here, we use this engineered strain of E. coli as a host system for the production of polyhydroxybutyrate (PHB), a biologically derived and biodegradable plastic. We confirmed the production of PHB in this strain using Nile red fluorescent microscopy, transmission electron microscopy and GC measurements. Since formate can be efficiently generated from CO2 via electricity, this study serves as a proof of concept for the emerging field of electro-bioproduction and opens a new avenue for the production of carbon-neutral chemicals.\nBioorthogonal labeling enables in situ fluorescence imaging of expressed gas vesicle nanostructures\nAuthors: Schrunk, E.; Dutka, P.; Hurt, R. C.; Wu, D.; Shapiro, M. G.\nScore: 10.4, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569486\nGas vesicles (GVs) are proteinaceous nanostructures that, along with virus-like particles, encapsulins, nano-cages, and other macromolecular assemblies are being developed for potential biomedical applications. To facilitate such development, it would be valuable to characterize these nanostructures sub-cellular assembly and localization. However, traditional fluorescent protein fusions are not tolerated by GVs primary constituent protein, making optical microscopy a challenge. Here, we introduce a method for fluorescently visualizing intracellular GVs using the bioorthogonal label FlAsH, which becomes fluorescent upon binding the six-amino acid tetracysteine (TC) tag. We engineered the GV subunit protein, GvpA, to display the TC tag, and showed that GVs bearing TC-tagged GvpA can be successfully assembled and fluorescently visualized in HEK 293T cells. We used fluorescence images of the tagged GVs to study GV size and distance distributions within these cells. This bioorthogonal labeling approach will enable research to provide a greater understanding of GVs and could be adapted to similar proteinaceous nanostructures.\nDiscovery of a high-performance phage-derived promoter/repressor system for probiotic lactobacillus engineering\nAuthors: Blanch-Asensio, M.; Tadimarri, V. S.; Wilk, A.; Sankaran, S.\nScore: 9.4, Published: 2023-12-05 DOI: 10.1101/2023.12.05.570180\nBackgroundThe Lactobacillus family comprises many species of great importance for the food and healthcare industries, with numerous strains identified as beneficial for humans and used as probiotics. Hence, there is a growing interest in engineering these probiotic bacteria as live biotherapeutics for animals and humans. However, the genetic parts needed to regulate gene expression in these bacteria remain limited compared to model bacteria like E. coli or B. subtilis. To address this deficit, in this study, we selected and tested several bacteriophage-derived genetic parts with the potential to regulate transcription in lactobacilli. ResultsWe screened genetic parts from 6 different lactobacilli-infecting phages and identified one promoter/repressor system with unprecedented functionality in L. plantarum WCFS1. The phage-derived promoter was found to achieve expression levels nearly 9-fold higher than the previously reported strongest promoter in this strain and the repressor was able to almost completely repress this expression by reducing it nearly 500-fold. ConclusionsThe new parts and insights gained from their engineering will enhance the genetic programmability of lactobacilli for healthcare and industrial applications.\nPHEIGES, all-cell-free phage synthesis and selection from engineered genomes\nAuthors: Levrier, A.; Karpathakis, I.; Nash, B.; Bowden, S. D.; Lindner, A. B.; Noireaux, V.\nScore: 9.2, Published: 2023-12-07 DOI: 10.1101/2023.12.07.570578\nBacteriophages constitute an invaluable biological reservoir for biotechnology and medicine. The ability to exploit such vast resources is hampered by the lack of methods to rapidly engineer, assemble, package genomes, and select phages. Cell-free transcription-translation(TXTL) offers experimental settings to address such a limitation. Here, we describe PHage Engineering by In vitro Gene Expression and Selection (PHEIGES) using T7 phage genome and Escherichia coli TXTL. Phage genomes are assembled in vitro from PCR-amplified fragments and directly expressed in batch TXTL reactions to produce up to 1011 PFU/ml engineered phages within one day. We further demonstrate an important genotype-phenotype linkage of phage assembly in bulk TXTL. This enables rapid selection of phages with altered rough lipopolysaccharides specificity from phage genomes incorporating tail fiber mutant libraries. We establish the scalability of PHEIGES by one-pot assembly of such mutants with fluorescent gene integration and 10% length-reduced genome.\nHigh yield, low magnesium flexizyme reactions in a water-ice eutectic phase\nAuthors: Davisson, J.; Alejo, J.; Blank, M.; Kalb, E.; Prasad, A.; Knudson, I.; Schepartz, A.; Engelhart, A. E.; Adamala, K. P.\nScore: 4.2, Published: 2023-12-03 DOI: 10.1101/2023.12.03.569792\nFlexizymes enable the stoichiometric acylation of tRNAs with a variety of compounds, enabling the in vitro translation of peptides with both non-natural backbones and side chains. However, flexizyme reactions have several drawbacks, including single-turnover kinetics, high Mg(II) carryover inhibiting in vitro translation, and rapid product hydrolysis. Here we present flexizyme reactions utilizing an ice-eutectic phase, with high yields, 30X lower Mg(II), and long-term product stability. The eutectic flexizyme reactions increase the ease of use and flexibility of flexizyme aminoacylation, and increase the in vitro protein production.\nPopulation suppression with dominant female-lethal alleles is boosted by homing gene drive\nAuthors: Zhu, J.; Chen, J.; Liu, Y.; Xu, X.; Champer, J.\nScore: 3.4, Published: 2023-12-05 DOI: 10.1101/2023.12.05.570109\nMethods to suppress pest insect populations using genetic constructs and repeated releases of male homozygotes have recently been shown to be an attractive alternative to older sterile insect technique based on radiation. Female-specific lethal alleles have substantially increased power, but still require large, sustained transgenic insect releases. Gene drive alleles bias their own inheritance to spread throughout populations, potentially allowing population suppression with a single, small-size release. However, suppression drives often suffer from efficiency issues, and the most well-studied type, homing drives, tend to spread without limit. In this study, we show that coupling female-specific lethal alleles with homing gene drive allowed substantial improvement in efficiency while still retaining the self-limiting nature (and thus confinement) of a lethal allele strategy. Using a mosquito model, we show the required releases sizes for population elimination in a variety of scenarios, including different density growth curves, with comparisons to other systems. Resistance alleles reduced the power of this method, but these could be overcome by targeting an essential gene with the drive while also providing rescue. A proof-of-principle demonstration of this system in Drosophila melanogaster was effective in both basing its inheritance and achieving high lethality among females that inherit the construct in the absence of antibiotic. Overall, our study shows that substantial improvements can be achieved in female-specific lethal systems for population suppression by combining them with a gene drive.\nExtreme miniaturization in plasmid design: generation of the 903 bp cloning vector pICOt2\nAuthors: Staal, J.; Beyaert, R.\nScore: 41.2, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569326\nInspired by the size optimizations in the recently published pUdO series of plasmids, we set out to break our previous world record (pICOz, 1185 bp) in designing the smallest possible self-replicating high copy cloning vector. By replacing the zeocin resistance marker with trimethoprim resistance, deleting the parts upstream of -35 in the ampicilin resistance promoter and replacing the extended multiple cloning site with a single SpeI site, we managed to reduce the plasmid size to 903 bp, almost a 24% reduction in size compared to our previous world record.\nA homing rescue gene drive with multiplexed gRNAs reaches high frequency in cage populations but generates functional resistance\nAuthors: Chen, J.; Hou, S.; Feng, R.; Xu, X.; Liang, N.; Champer, J.\nScore: 4.2, Published: 2023-11-29 DOI: 10.1101/2023.11.29.569221\nCRISPR homing gene drive is a potent technology with considerable potential for managing populations of medically and agriculturally significant insects. It induces a bias in the inheritance of the drive allele in progeny, rapidly spreading desired genes throughout the population. Homing drives operate by Cas9 cleavage followed by homology-directed repair, copying the drive allele to the wild-type chromosome. However, resistance alleles formed by end-joining repair pose a significant obstacle to the spread of the drive. To address this challenge, we created a homing drive targeting the essential but haplosufficient hairy gene. Our strategy involves spreading the drive construct through the homing process, eliminating nonfunctional resistance, which are recessive lethal, while rescuing drive-carrying individuals with a recoded version of hairy. This strategy eliminates resistance more slowly than a previous strategy targeting haplolethal genes, but it may be easier to construct in non-model organisms. Our drive inheritance rate was moderate, and multigenerational cage studies showed quick drive spread to 96-97% of the population. However, the drive failed to reach the whole population due to the formation of functional resistance alleles, despite use of four gRNAs, a strategy that previously was successful at preventing functional resistance. Sequencing showed that these alleles had a large deletion and must have utilized an alternate start codon. The resistance allele had a modest fitness advantage over the drive in a cage study, which could prevent long-term persistence of the drive, especially if cargo genes had an additional fitness cost. Thus, revised design strategies targeting more essential regions of a target gene may often be necessary to avoid such functional resistance, even when using multiplexed gRNAs.\nA red light-induced genetic system for control of extracellular electron transfer\nAuthors: Zhao, F.; Niman, C. M.; Ostovar, G.; Chavez, M. S.; Atkinson, J. T.; Bonis, B. M.; Gralnick, J. A.; El-Naggar, M. Y.; Boedicker, J. Q.\nScore: 2.5, Published: 2023-12-02 DOI: 10.1101/2023.12.02.569691\nOptogenetics is a powerful tool for spatiotemporal control of gene expression. Several light-inducible gene regulators have been developed to function in bacteria, and these regulatory circuits have been ported into new host strains. Here, we developed and adapted a red light-inducible transcription factor for Shewanella oneidensis. This regulatory circuit is based on the iLight optogenetic system, which controls gene expression using red light. Promoter engineering and a thermodynamic model were used to adapt this system to achieve differential gene expression in light and dark conditions within a S. oneidensis host strain. We further improved the iLight optogenetic system by adding a repressor to invert the genetic circuit and activate gene expression under red light illumination. The inverted iLight genetic circuit was used to control extracellular electron transfer (EET) within S. oneidensis. The ability to use both red and blue light-induced optogenetic circuits simultaneously was demonstrated. Our work expands the synthetic biology toolbox of Shewanella, which could facilitate future advances in applications with electrogenic bacteria.\n",
  "wordCount" : "1994",
  "inLanguage": "en",
  "datePublished": "2023-12-10T10:37:48Z",
  "dateModified": "2023-12-10T10:37:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 10, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.570403">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.570403" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.570403">
        <p class="paperTitle">Validation of cell-free protein synthesis aboard the International Space Station</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.570403" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.570403" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kocalar, S.; Miller, B. M.; Huang, A.; Gleason, E.; Martin, K.; Foley, K.; Copeland, D. S.; Jewett, M.; Saavedra, E. A.; Kraves, S.</p>
        <p class="info">Score: 18.8, Published: 2023-12-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.570403' target='https://doi.org/10.1101/2023.12.06.570403'> 10.1101/2023.12.06.570403</a></p>
        <p class="abstract">Cell-free protein synthesis (CFPS) is a rapidly maturing in vitro gene expression platform that can be used to transcribe and translate nucleic acids at the point of need, enabling on-demand synthesis of peptide-based vaccines and biotherapeutics, as well as the development of diagnostic tests for environmental contaminants and infectious agents. Unlike traditional cell-based systems, CFPS platforms do not require the maintenance of living cells and can be deployed with minimal equipment; therefore, they hold promise for applications in low-resource contexts, including spaceflight. Here we evaluate the performance of cell-free BioBits(R) platform aboard the International Space Station by expressing RNA-based aptamers and fluorescent proteins that can serve as biological indicators. We validate two classes of biological sensors that detect either the small molecule DFHBI or a specific RNA sequence. Upon detection of their respective analytes, both biological sensors produce fluorescent readouts that are visually confirmed using a handheld fluorescence viewer and imaged for quantitative analysis. Our findings provide insight into the kinetics of cell-free transcription and translation in a microgravity environment and reveal that both biosensors perform robustly in space. Our findings lay the groundwork for portable, low-cost applications ranging from point-of-care health monitoring to on-demand detection of environmental hazards in low-resource communities both on Earth and beyond.

Visual graphic

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=123 SRC=&#34;FIGDIR/small/570403v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (47K):
org.highwire.dtl.DTLVardef@172406aorg.highwire.dtl.DTLVardef@1e315d8org.highwire.dtl.DTLVardef@1af3598org.highwire.dtl.DTLVardef@1e7c563_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.02.569694">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.02.569694" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.02.569694">
        <p class="paperTitle">Demonstration of Bioplastic Production from CO2 and Formate using the Reductive Glycine Pathway in E. coli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.02.569694" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.02.569694" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fedorova, D.; Ben-Nissan, R.; Milshtein, E.; Jona, G.; Dezorella, N.; Feiguelman, G.; Fedorov, R.; Gomaa, A.; Lindner, A. B.; Noor, E.; Milo, R.</p>
        <p class="info">Score: 9.6, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.02.569694' target='https://doi.org/10.1101/2023.12.02.569694'> 10.1101/2023.12.02.569694</a></p>
        <p class="abstract">There is a strong need to develop technologies that reduce greenhouse gas emissions. Recently, progress was made in engineering the model organism E. coli to grow using CO2 and formate as its only carbon and energy sources using the reductive glycine pathway (rGlyP). Here, we use this engineered strain of E. coli as a host system for the production of polyhydroxybutyrate (PHB), a biologically derived and biodegradable plastic. We confirmed the production of PHB in this strain using Nile red fluorescent microscopy, transmission electron microscopy and GC measurements. Since formate can be efficiently generated from CO2 via electricity, this study serves as a proof of concept for the emerging field of electro-bioproduction and opens a new avenue for the production of carbon-neutral chemicals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569486">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569486" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569486">
        <p class="paperTitle">Bioorthogonal labeling enables in situ fluorescence imaging of expressed gas vesicle nanostructures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569486" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569486" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schrunk, E.; Dutka, P.; Hurt, R. C.; Wu, D.; Shapiro, M. G.</p>
        <p class="info">Score: 10.4, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569486' target='https://doi.org/10.1101/2023.11.30.569486'> 10.1101/2023.11.30.569486</a></p>
        <p class="abstract">Gas vesicles (GVs) are proteinaceous nanostructures that, along with virus-like particles, encapsulins, nano-cages, and other macromolecular assemblies are being developed for potential biomedical applications. To facilitate such development, it would be valuable to characterize these nanostructures sub-cellular assembly and localization. However, traditional fluorescent protein fusions are not tolerated by GVs primary constituent protein, making optical microscopy a challenge. Here, we introduce a method for fluorescently visualizing intracellular GVs using the bioorthogonal label FlAsH, which becomes fluorescent upon binding the six-amino acid tetracysteine (TC) tag. We engineered the GV subunit protein, GvpA, to display the TC tag, and showed that GVs bearing TC-tagged GvpA can be successfully assembled and fluorescently visualized in HEK 293T cells. We used fluorescence images of the tagged GVs to study GV size and distance distributions within these cells. This bioorthogonal labeling approach will enable research to provide a greater understanding of GVs and could be adapted to similar proteinaceous nanostructures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570180">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570180" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570180">
        <p class="paperTitle">Discovery of a high-performance phage-derived promoter/repressor system for probiotic lactobacillus engineering</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570180" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570180" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Blanch-Asensio, M.; Tadimarri, V. S.; Wilk, A.; Sankaran, S.</p>
        <p class="info">Score: 9.4, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570180' target='https://doi.org/10.1101/2023.12.05.570180'> 10.1101/2023.12.05.570180</a></p>
        <p class="abstract">BackgroundThe Lactobacillus family comprises many species of great importance for the food and healthcare industries, with numerous strains identified as beneficial for humans and used as probiotics. Hence, there is a growing interest in engineering these probiotic bacteria as live biotherapeutics for animals and humans. However, the genetic parts needed to regulate gene expression in these bacteria remain limited compared to model bacteria like E. coli or B. subtilis. To address this deficit, in this study, we selected and tested several bacteriophage-derived genetic parts with the potential to regulate transcription in lactobacilli.

ResultsWe screened genetic parts from 6 different lactobacilli-infecting phages and identified one promoter/repressor system with unprecedented functionality in L. plantarum WCFS1. The phage-derived promoter was found to achieve expression levels nearly 9-fold higher than the previously reported strongest promoter in this strain and the repressor was able to almost completely repress this expression by reducing it nearly 500-fold.

ConclusionsThe new parts and insights gained from their engineering will enhance the genetic programmability of lactobacilli for healthcare and industrial applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570578">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570578" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570578">
        <p class="paperTitle">PHEIGES, all-cell-free phage synthesis and selection from engineered genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570578" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570578" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Levrier, A.; Karpathakis, I.; Nash, B.; Bowden, S. D.; Lindner, A. B.; Noireaux, V.</p>
        <p class="info">Score: 9.2, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570578' target='https://doi.org/10.1101/2023.12.07.570578'> 10.1101/2023.12.07.570578</a></p>
        <p class="abstract">Bacteriophages constitute an invaluable biological reservoir for biotechnology and medicine. The ability to exploit such vast resources is hampered by the lack of methods to rapidly engineer, assemble, package genomes, and select phages. Cell-free transcription-translation(TXTL) offers experimental settings to address such a limitation. Here, we describe PHage Engineering by In vitro Gene Expression and Selection (PHEIGES) using T7 phage genome and Escherichia coli TXTL. Phage genomes are assembled in vitro from PCR-amplified fragments and directly expressed in batch TXTL reactions to produce up to 1011 PFU/ml engineered phages within one day. We further demonstrate an important genotype-phenotype linkage of phage assembly in bulk TXTL. This enables rapid selection of phages with altered rough lipopolysaccharides specificity from phage genomes incorporating tail fiber mutant libraries. We establish the scalability of PHEIGES by one-pot assembly of such mutants with fluorescent gene integration and 10% length-reduced genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.569792">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.569792" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.569792">
        <p class="paperTitle">High yield, low magnesium flexizyme reactions in a water-ice eutectic phase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.569792" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.569792" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davisson, J.; Alejo, J.; Blank, M.; Kalb, E.; Prasad, A.; Knudson, I.; Schepartz, A.; Engelhart, A. E.; Adamala, K. P.</p>
        <p class="info">Score: 4.2, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.569792' target='https://doi.org/10.1101/2023.12.03.569792'> 10.1101/2023.12.03.569792</a></p>
        <p class="abstract">Flexizymes enable the stoichiometric acylation of tRNAs with a variety of compounds, enabling the in vitro translation of peptides with both non-natural backbones and side chains. However, flexizyme reactions have several drawbacks, including single-turnover kinetics, high Mg(II) carryover inhibiting in vitro translation, and rapid product hydrolysis. Here we present flexizyme reactions utilizing an ice-eutectic phase, with high yields, 30X lower Mg(II), and long-term product stability. The eutectic flexizyme reactions increase the ease of use and flexibility of flexizyme aminoacylation, and increase the in vitro protein production.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570109">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570109" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570109">
        <p class="paperTitle">Population suppression with dominant female-lethal alleles is boosted by homing gene drive</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570109" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570109" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhu, J.; Chen, J.; Liu, Y.; Xu, X.; Champer, J.</p>
        <p class="info">Score: 3.4, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570109' target='https://doi.org/10.1101/2023.12.05.570109'> 10.1101/2023.12.05.570109</a></p>
        <p class="abstract">Methods to suppress pest insect populations using genetic constructs and repeated releases of male homozygotes have recently been shown to be an attractive alternative to older sterile insect technique based on radiation. Female-specific lethal alleles have substantially increased power, but still require large, sustained transgenic insect releases. Gene drive alleles bias their own inheritance to spread throughout populations, potentially allowing population suppression with a single, small-size release. However, suppression drives often suffer from efficiency issues, and the most well-studied type, homing drives, tend to spread without limit. In this study, we show that coupling female-specific lethal alleles with homing gene drive allowed substantial improvement in efficiency while still retaining the self-limiting nature (and thus confinement) of a lethal allele strategy. Using a mosquito model, we show the required releases sizes for population elimination in a variety of scenarios, including different density growth curves, with comparisons to other systems. Resistance alleles reduced the power of this method, but these could be overcome by targeting an essential gene with the drive while also providing rescue. A proof-of-principle demonstration of this system in Drosophila melanogaster was effective in both basing its inheritance and achieving high lethality among females that inherit the construct in the absence of antibiotic. Overall, our study shows that substantial improvements can be achieved in female-specific lethal systems for population suppression by combining them with a gene drive.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569326">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569326" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569326">
        <p class="paperTitle">Extreme miniaturization in plasmid design: generation of the 903 bp cloning vector pICOt2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569326" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569326" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Staal, J.; Beyaert, R.</p>
        <p class="info">Score: 41.2, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569326' target='https://doi.org/10.1101/2023.11.29.569326'> 10.1101/2023.11.29.569326</a></p>
        <p class="abstract">Inspired by the size optimizations in the recently published pUdO series of plasmids, we set out to break our previous world record (pICOz, 1185 bp) in designing the smallest possible self-replicating high copy cloning vector. By replacing the zeocin resistance marker with trimethoprim resistance, deleting the parts upstream of -35 in the ampicilin resistance promoter and replacing the extended multiple cloning site with a single SpeI site, we managed to reduce the plasmid size to 903 bp, almost a 24% reduction in size compared to our previous world record.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569221">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569221" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569221">
        <p class="paperTitle">A homing rescue gene drive with multiplexed gRNAs reaches high frequency in cage populations but generates functional resistance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569221" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569221" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, J.; Hou, S.; Feng, R.; Xu, X.; Liang, N.; Champer, J.</p>
        <p class="info">Score: 4.2, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569221' target='https://doi.org/10.1101/2023.11.29.569221'> 10.1101/2023.11.29.569221</a></p>
        <p class="abstract">CRISPR homing gene drive is a potent technology with considerable potential for managing populations of medically and agriculturally significant insects. It induces a bias in the inheritance of the drive allele in progeny, rapidly spreading desired genes throughout the population. Homing drives operate by Cas9 cleavage followed by homology-directed repair, copying the drive allele to the wild-type chromosome. However, resistance alleles formed by end-joining repair pose a significant obstacle to the spread of the drive. To address this challenge, we created a homing drive targeting the essential but haplosufficient hairy gene. Our strategy involves spreading the drive construct through the homing process, eliminating nonfunctional resistance, which are recessive lethal, while rescuing drive-carrying individuals with a recoded version of hairy. This strategy eliminates resistance more slowly than a previous strategy targeting haplolethal genes, but it may be easier to construct in non-model organisms. Our drive inheritance rate was moderate, and multigenerational cage studies showed quick drive spread to 96-97% of the population. However, the drive failed to reach the whole population due to the formation of functional resistance alleles, despite use of four gRNAs, a strategy that previously was successful at preventing functional resistance. Sequencing showed that these alleles had a large deletion and must have utilized an alternate start codon. The resistance allele had a modest fitness advantage over the drive in a cage study, which could prevent long-term persistence of the drive, especially if cargo genes had an additional fitness cost. Thus, revised design strategies targeting more essential regions of a target gene may often be necessary to avoid such functional resistance, even when using multiplexed gRNAs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.02.569691">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.02.569691" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.02.569691">
        <p class="paperTitle">A red light-induced genetic system for control of extracellular electron transfer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.02.569691" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.02.569691" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhao, F.; Niman, C. M.; Ostovar, G.; Chavez, M. S.; Atkinson, J. T.; Bonis, B. M.; Gralnick, J. A.; El-Naggar, M. Y.; Boedicker, J. Q.</p>
        <p class="info">Score: 2.5, Published: 2023-12-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.02.569691' target='https://doi.org/10.1101/2023.12.02.569691'> 10.1101/2023.12.02.569691</a></p>
        <p class="abstract">Optogenetics is a powerful tool for spatiotemporal control of gene expression. Several light-inducible gene regulators have been developed to function in bacteria, and these regulatory circuits have been ported into new host strains. Here, we developed and adapted a red light-inducible transcription factor for Shewanella oneidensis. This regulatory circuit is based on the iLight optogenetic system, which controls gene expression using red light. Promoter engineering and a thermodynamic model were used to adapt this system to achieve differential gene expression in light and dark conditions within a S. oneidensis host strain. We further improved the iLight optogenetic system by adding a repressor to invert the genetic circuit and activate gene expression under red light illumination. The inverted iLight genetic circuit was used to control extracellular electron transfer (EET) within S. oneidensis. The ability to use both red and blue light-induced optogenetic circuits simultaneously was demonstrated. Our work expands the synthetic biology toolbox of Shewanella, which could facilitate future advances in applications with electrogenic bacteria.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
