<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Engineering synthetic phosphorylation signaling networks in human cells
Authors: Yang, X.; Rocks, J. W.; Jiang, K.; Walters, A. J.; Rai, K.; Liu, J.; Nguyen, J.; Olson, S. D.; Mehta, P.; Collins, J. J.; Daringer, N. M.; Bashor, C. J.
Score: 6.8, Published: 2023-09-11 DOI: 10.1101/2023.09.11.557100
Protein phosphorylation signaling networks play a central role in how cells sense and respond to their environment. Here, we describe the engineering of artificial phosphorylation networks in which &#34;push-pull&#34; motifs-reversible enzymatic phosphorylation cycles consisting of opposing kinase and phosphatase activities-are assembled from modular protein domain parts and then wired together to create synthetic phosphorylation circuits in human cells.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Engineering synthetic phosphorylation signaling networks in human cells
Authors: Yang, X.; Rocks, J. W.; Jiang, K.; Walters, A. J.; Rai, K.; Liu, J.; Nguyen, J.; Olson, S. D.; Mehta, P.; Collins, J. J.; Daringer, N. M.; Bashor, C. J.
Score: 6.8, Published: 2023-09-11 DOI: 10.1101/2023.09.11.557100
Protein phosphorylation signaling networks play a central role in how cells sense and respond to their environment. Here, we describe the engineering of artificial phosphorylation networks in which &#34;push-pull&#34; motifs-reversible enzymatic phosphorylation cycles consisting of opposing kinase and phosphatase activities-are assembled from modular protein domain parts and then wired together to create synthetic phosphorylation circuits in human cells." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-13T10:39:03+00:00" />
<meta property="article:modified_time" content="2023-09-13T10:39:03+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Engineering synthetic phosphorylation signaling networks in human cells
Authors: Yang, X.; Rocks, J. W.; Jiang, K.; Walters, A. J.; Rai, K.; Liu, J.; Nguyen, J.; Olson, S. D.; Mehta, P.; Collins, J. J.; Daringer, N. M.; Bashor, C. J.
Score: 6.8, Published: 2023-09-11 DOI: 10.1101/2023.09.11.557100
Protein phosphorylation signaling networks play a central role in how cells sense and respond to their environment. Here, we describe the engineering of artificial phosphorylation networks in which &#34;push-pull&#34; motifs-reversible enzymatic phosphorylation cycles consisting of opposing kinase and phosphatase activities-are assembled from modular protein domain parts and then wired together to create synthetic phosphorylation circuits in human cells."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Engineering synthetic phosphorylation signaling networks in human cells\nAuthors: Yang, X.; Rocks, J. W.; Jiang, K.; Walters, A. J.; Rai, K.; Liu, J.; Nguyen, J.; Olson, S. D.; Mehta, P.; Collins, J. J.; Daringer, N. M.; Bashor, C. J.\nScore: 6.8, Published: 2023-09-11 DOI: 10.1101/2023.09.11.557100\nProtein phosphorylation signaling networks play a central role in how cells sense and respond to their environment. Here, we describe the engineering of artificial phosphorylation networks in which \u0026#34;push-pull\u0026#34; motifs-reversible enzymatic phosphorylation cycles consisting of opposing kinase and phosphatase activities-are assembled from modular protein domain parts and then wired together to create synthetic phosphorylation circuits in human cells.",
  "keywords": [
    
  ],
  "articleBody": " Engineering synthetic phosphorylation signaling networks in human cells\nAuthors: Yang, X.; Rocks, J. W.; Jiang, K.; Walters, A. J.; Rai, K.; Liu, J.; Nguyen, J.; Olson, S. D.; Mehta, P.; Collins, J. J.; Daringer, N. M.; Bashor, C. J.\nScore: 6.8, Published: 2023-09-11 DOI: 10.1101/2023.09.11.557100\nProtein phosphorylation signaling networks play a central role in how cells sense and respond to their environment. Here, we describe the engineering of artificial phosphorylation networks in which \"push-pull\" motifs-reversible enzymatic phosphorylation cycles consisting of opposing kinase and phosphatase activities-are assembled from modular protein domain parts and then wired together to create synthetic phosphorylation circuits in human cells. We demonstrate that the composability of our design scheme permits model-guided tuning of circuit function and the ability to make diverse network connections; synthetic phosphorylation circuits can be coupled to upstream cell surface receptors to enable fast-timescale sensing of extracellular ligands, while downstream connections can regulate gene expression. We leverage these capabilities to program cells to act as controllers that dynamically regulate immune cell cytokine production. Our work introduces a generalizable approach for designing and budling phosphorylation signaling circuits that enable user-defined sense-and-respond function for diverse biosensing and therapeutic applications.\nPrecise manipulation of site and stoichiometry of capsid modification enables optimization of functional adeno-associated virus conjugates\nAuthors: Erickson, S. B.; Pham, Q.; Cao, X.; Glicksman, J.; Kelemen, R. E.; Shahraeini, S. S.; Bodkin, S.; Kiyam, Z.; Chatterjee, A.\nScore: 5.3, Published: 2023-09-07 DOI: 10.1101/2023.09.07.556719\nThe ability to engineer adeno-associated virus (AAV) vectors for targeted infection of specific cell types is critically important to fully harness its potential of human gene therapy. A promising approach to achieve this objective involves chemically attaching retargeting ligands onto the virus capsid. Site-specific incorporation of a bioorthogonal noncanonical amino acid (ncAA) into the AAV capsid proteins provides a particularly attractive strategy to introduce such modifications with exquisite precision. In this study, we show that using ncAA mutagenesis, it is possible to systematically alter the attachment site of a retargeting ligand (cyclic-RGD) on the AAV capsid to create diverse conjugate architectures, and that the site of attachment heavily impacts the retargeting efficiency. We further demonstrate that the performance of these AAV conjugates is highly sensitive to the stoichiometry of capsid labeling (labels per capsid), with an intermediate labeling density ([~]12 per capsid) providing optimal activity. Finally, we developed technology to precisely control the number of attachment sites per AAV capsid, by selectively incorporating a ncAA into the minor capsid proteins with high fidelity and efficiency, such that AAV-conjugates with varying stoichiometry can be synthesized in a homogeneous manner. Together, this platform provides unparalleled control over site and stoichiometry of capsid modification, which will enable the development of next-generation AAV vectors tailored with desirable attributes.\nTargeted Transcriptional Activation using a CRISPR-Associated Transposon System\nAuthors: Garza Elizondo, A. M.; Chappell, J.\nScore: 3.6, Published: 2023-09-12 DOI: 10.1101/2023.09.12.557412\nSynthetic perturbation of gene expression is central to our ability to reliably uncover genotype-phenotype relationships in microbes. Here, we present a novel gene transcription activation strategy that uses the Vibrio cholerae CRISPR-Associated Transposon (CAST) system to selectively insert promoter elements upstream of genes of interest. Through this strategy, we show robust activation of both recombinant and endogenous genes across the E. coli chromosome. We then demonstrate precise tuning of expression levels by exchanging the promoter elements being inserted. Finally, we demonstrate that CAST activation can be used to synthetically induce ampicillin-resistant phenotypes in E. coli.\nA hybrid combination of in vitro cultured buccal mucosal cells using two different methodologies, complementing each other in successfully repairing a stricture-inflicted human male urethral epithelium\nAuthors: Horiguchi, A.; Kushibiki, T.; Yoshine, M.; Shinchi, M.; Ojima, K.; Hirano, Y.; Katoh, S.; Iwasaki, M.; Surya Prakash, V.; Ichiyama, K.; Senthilkumar, R.; Preethy, S.; Abraham, S. J.\nScore: 2.6, Published: 2023-08-30 DOI: 10.1101/2023.08.29.555240\nBackgroundAutologous buccal mucosal tissue derived cell transplantation techniques in repairing a stricture inflicted male urethral epithelium have been evolving. There was not much of clarity on the cell type, in vitro culture methods and the mode of transplantation, until we reported our buccal epithelium expanded and encapsulated in scaffold-hybrid approach to urethral stricture (BEES-HAUS) clinical study yielding a successful engraftment and repair with a long-term patency. We herein report with technical clarity on the advantages of mixing two-dimensional (2D) monolayer cultured fibroblast like cells and three dimensional (3D) thermo-reversible gelation polymer (TGP) cultured cells; the former secreting IGF-1, a cytokine known for its healing effects and the latter expressing epithelial surface markers in flow cytometry, both sourced from human buccal tissue, together transplanted using TGP as a carrier. MethodsHuman buccal tissues (n=22) redundant after urethroplasty surgery was used after informed consent and IEC approval. They were enzyme digested, divided into two portions; one was cultured as monolayer method (2D) and the other in 3D in TGP. Flowcytometry and quantification of IGF-1 in cell culture supernatant through the culture period were undertaken. ResultsIn flowcytometry, the cells on day 0, lacked AE1/AE3 - pancytokeratin expression indicative of epithelial phenotype of culture, which progressively increased in the 3D-TGP group, during invitro culture of up to 21 days. The 2D showed expression of only fibroblasts like cells that were negative for AE1/AE3 but positive for CD140b. IGF-1 secretion was significantly higher in 2D cultures than in 3D-TGP (p-value \u003c 0.05). ConclusionThe 3D- TGP cultured cells of epithelial nature and the 2D cultured fibroblast like cells secreting IGF-1, together when transplanted using TGP scaffold as a carrier, adapted to a hostile in vivo milieu after releasing the fibrous strands with urethrotomy, successfully engrafted and repaired a stricture-inflicted male urethral epithelium in the BEES-HAUS procedure. While this hybrid combination of cells are considered to have potential in managing other epithelial damages, further research of such hybrid combination and their behaviour in disease affected environments may help to expand this solution in regenerating and repairing other tissues and organs as well. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=123 SRC=\"FIGDIR/small/555240v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (70K): org.highwire.dtl.DTLVardef@64a3aeorg.highwire.dtl.DTLVardef@136020borg.highwire.dtl.DTLVardef@c043acorg.highwire.dtl.DTLVardef@9b00c6_HPS_FORMAT_FIGEXP M_FIG C_FIG Illustration of pathogenesis of urethral stricture and the contribution of the hybrid combination of two-dimensional (2D) and three dimensional, 3D-TGP (Thermo-reversible gelation polymer) cultured cells to the successful repair of the stricture-inflicted male urethral epithelium in the buccal epithelium expanded and encapsulated in scaffold-hybrid approach to urethral stricture (BEES-HAUS) technique.\nCRISPR-COPIES: An in silico platform for discovery of neutral integration sites for CRISPR/Cas-facilitated gene integration\nAuthors: Boob, A. G.; Zhu, Z.; Intasian, P.; Jain, M.; Petrov, V. A.; Tan, S.-I.; Xun, G.; Zhao, H.\nScore: 1.8, Published: 2023-09-06 DOI: 10.1101/2023.09.06.556564\nThe CRISPR/Cas system has emerged as a powerful tool for genome editing in metabolic engineering and human gene therapy. However, locating the optimal site on the chromosome to integrate heterologous genes using the CRISPR/Cas system remains an open question. Selecting a suitable site for gene integration involves considering multiple complex criteria, including factors related to CRISPR/Cas-mediated integration, genetic stability, and gene expression. Consequently, identifying such sites on specific or different chromosomal locations typically requires extensive characterization efforts. To address these challenges, we have developed CRISPR-COPIES, a COmputational Pipeline for the Identification of CRISPR/Cas-facilitated intEgration Sites. This tool leverages ScaNN, a state-of-the-art model on the embedding-based nearest neighbor search for fast and accurate off-target search and can identify genome-wide intergenic sites for most bacterial and fungal genomes within minutes. As a proof of concept, we utilized CRISPR-COPIES to characterize neutral integration sites in three diverse species: Saccharomyces cerevisiae, Cupriavidus necator, and a human cell line. In addition, we developed a user-friendly web interface for CRISPR-COPIES (https://biofoundry.web.illinois.edu/copies/). We anticipate that CRISPR-COPIES will serve as a valuable tool for targeted DNA integration and aid in the characterization of synthetic biology toolkits, enable rapid strain construction to produce valuable biochemicals and support human gene and cell therapy applications. Graphical abstractOverview and application of CRISPR-COPIES in the field of biotechnology. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=93 SRC=\"FIGDIR/small/556564v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (20K): org.highwire.dtl.DTLVardef@13bbf9aorg.highwire.dtl.DTLVardef@1a035d7org.highwire.dtl.DTLVardef@12edf18org.highwire.dtl.DTLVardef@38b0a6_HPS_FORMAT_FIGEXP M_FIG C_FIG\nEngineering the ADDobody protein scaffold for generation of high-avidity ADDomer super-binders\nAuthors: Buzas, D.; Sun, H.; Toelzer, C.; Yadav, S. K. N.; Borucu, U.; Gautam, G.; Gupta, K.; Bufton, J.; Capin, J.; Sessions, R. B.; Garzoni, F.; Berger, I.; Schaffitzel, C.\nScore: 1.2, Published: 2023-09-09 DOI: 10.1101/2023.09.09.556966\nAdenovirus-derived dodecamer (ADDomer) nanoparticles comprise 60 copies of Adenovirus penton base protein (PBP). ADDomer is thermostable, rendering the storage, transport and deployment of ADDomer-based therapeutics independent of a cold-chain. To expand the scope of ADDomer nanoparticles for new applications, we engineered ADDobodies. ADDobodies represent the crown domain of the PBP, genetically separated from its multimerization domain. We inserted heterologous sequences into hyper-variable loops in the crown domain. The resulting ADDobodies were expressed at high yields in Escherichia coli, are monomeric and maintain thermostability. We solved the X-ray structure of an ADDobody prototype validating our design. We demonstrated that ADDobodies can be used to select a specific binder against a target in in vitro selection experiments using ribosome display, with an enrichment factor of ~104-fold in one selection round. We show that ADDobodies can be converted back into ADDomers by genetically reconnecting the selected ADDobody with the PBP multimerization domain from a different species, giving rise to a multivalent nanoparticle, called Chimera, confirmed by a 2.2 A structure determined by cryogenic electron microscopy (cryo-EM). Chimera comprises 60 binding sites, resulting in ultra-high, picomolar avidity to the target.\nAn emergent biosynthetic pathway to essential amino acids by metabolic metathesis\nAuthors: Rivollier, J.; Gosling, S.; Pezo, V.; Heck, M.-P.; MARLIERE, P.\nScore: 1.2, Published: 2023-09-07 DOI: 10.1101/2023.09.06.556532\nAn experimental approach to implanting foreign chemical reactions into living cells is to select for the catalytic production of a vital building-block such as an amino acid. Alkene metathesis is unknown in extant biochemistry, but is emerging as a new type of reaction to be catalyzed by protein enzymes. Here we show how the alkenic amino acid vinylglycine can be generated in a biocompatible reaction from 5-allyloxy-2-amino-pent-3-enoate (APE) by ring-closing metathesis catalyzed by a standard Hoveyda-Grubbs catalyst. The vinylglycine produced in situ is then used as a precursor of isoleucine and methionine, thus allowing the growth of strains of Escherichia coli requiring these essential amino acids. The robust nutritional screen we have developed paves the way for the directed evolution of genetically encoded metathesis enzymes and the chemical elaboration of metathesis coenzymes.\nManufacturing DNA in E. coli yields higher fidelity DNA than in vitro enzymatic synthesis\nAuthors: Hersch, S. J.; Chandrasekaran, S.; Lam, J.; Nafissi, N.; Slavcev, R.\nScore: 1.0, Published: 2023-09-12 DOI: 10.1101/2023.09.12.557453\nThe rise of biotechnologies such as gene therapy have brought DNA vectors to the forefront of pharmaceutical development. The quality of the genetic starting material plays a pivotal role in determining the quality of the final product. In this study we examined the fidelity of DNA replication using enzymatic methods (in vitro) compared to plasmid DNA produced in vivo in E. coli. Next-generation sequencing approaches predominantly rely on in vitro polymerases, which have inherent limitations in sensitivity. To address this challenge, we introduce a novel assay based on loss-of-function (LOF) mutations in the conditionally toxic sacB gene. Our findings show that DNA production in E. coli results in significantly fewer LOF mutations (approximately 80- to 3000-fold less) compared to various enzymatic DNA synthesis methods. This includes the most accurate PCR polymerase (Q5) and a commonly employed rolling circle amplification (RCA) DNA polymerase (Phi29). These results suggest that using low-fidelity starting material DNA synthesized in vitro by PCR or RCA may introduce a substantial number of impurities, potentially affecting the quality and yield of final pharmaceutical products. In summary, our study underscores that DNA synthesized in vitro has a significantly higher mutation rate than DNA produced traditionally in E. coli. Therefore, utilizing in vitro enzymatically-produced DNA in biotechnology and biomanufacturing may entail considerable fidelity-related risks, while DNA starting material derived from E. coli substantially mitigates this risk, enhancing overall quality in the production processes.\n",
  "wordCount" : "1965",
  "inLanguage": "en",
  "datePublished": "2023-09-13T10:39:03Z",
  "dateModified": "2023-09-13T10:39:03Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on September 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557100">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557100" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557100">
        <p class="paperTitle">Engineering synthetic phosphorylation signaling networks in human cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557100" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557100" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, X.; Rocks, J. W.; Jiang, K.; Walters, A. J.; Rai, K.; Liu, J.; Nguyen, J.; Olson, S. D.; Mehta, P.; Collins, J. J.; Daringer, N. M.; Bashor, C. J.</p>
        <p class="info">Score: 6.8, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557100' target='https://doi.org/10.1101/2023.09.11.557100'> 10.1101/2023.09.11.557100</a></p>
        <p class="abstract">Protein phosphorylation signaling networks play a central role in how cells sense and respond to their environment. Here, we describe the engineering of artificial phosphorylation networks in which &#34;push-pull&#34; motifs-reversible enzymatic phosphorylation cycles consisting of opposing kinase and phosphatase activities-are assembled from modular protein domain parts and then wired together to create synthetic phosphorylation circuits in human cells. We demonstrate that the composability of our design scheme permits model-guided tuning of circuit function and the ability to make diverse network connections; synthetic phosphorylation circuits can be coupled to upstream cell surface receptors to enable fast-timescale sensing of extracellular ligands, while downstream connections can regulate gene expression. We leverage these capabilities to program cells to act as controllers that dynamically regulate immune cell cytokine production. Our work introduces a generalizable approach for designing and budling phosphorylation signaling circuits that enable user-defined sense-and-respond function for diverse biosensing and therapeutic applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556719">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556719" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556719">
        <p class="paperTitle">Precise manipulation of site and stoichiometry of capsid modification enables optimization of functional adeno-associated virus conjugates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556719" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556719" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Erickson, S. B.; Pham, Q.; Cao, X.; Glicksman, J.; Kelemen, R. E.; Shahraeini, S. S.; Bodkin, S.; Kiyam, Z.; Chatterjee, A.</p>
        <p class="info">Score: 5.3, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556719' target='https://doi.org/10.1101/2023.09.07.556719'> 10.1101/2023.09.07.556719</a></p>
        <p class="abstract">The ability to engineer adeno-associated virus (AAV) vectors for targeted infection of specific cell types is critically important to fully harness its potential of human gene therapy. A promising approach to achieve this objective involves chemically attaching retargeting ligands onto the virus capsid. Site-specific incorporation of a bioorthogonal noncanonical amino acid (ncAA) into the AAV capsid proteins provides a particularly attractive strategy to introduce such modifications with exquisite precision. In this study, we show that using ncAA mutagenesis, it is possible to systematically alter the attachment site of a retargeting ligand (cyclic-RGD) on the AAV capsid to create diverse conjugate architectures, and that the site of attachment heavily impacts the retargeting efficiency. We further demonstrate that the performance of these AAV conjugates is highly sensitive to the stoichiometry of capsid labeling (labels per capsid), with an intermediate labeling density ([~]12 per capsid) providing optimal activity. Finally, we developed technology to precisely control the number of attachment sites per AAV capsid, by selectively incorporating a ncAA into the minor capsid proteins with high fidelity and efficiency, such that AAV-conjugates with varying stoichiometry can be synthesized in a homogeneous manner. Together, this platform provides unparalleled control over site and stoichiometry of capsid modification, which will enable the development of next-generation AAV vectors tailored with desirable attributes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557412">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557412" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557412">
        <p class="paperTitle">Targeted Transcriptional Activation using a CRISPR-Associated Transposon System</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557412" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557412" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garza Elizondo, A. M.; Chappell, J.</p>
        <p class="info">Score: 3.6, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557412' target='https://doi.org/10.1101/2023.09.12.557412'> 10.1101/2023.09.12.557412</a></p>
        <p class="abstract">Synthetic perturbation of gene expression is central to our ability to reliably uncover genotype-phenotype relationships in microbes. Here, we present a novel gene transcription activation strategy that uses the Vibrio cholerae CRISPR-Associated Transposon (CAST) system to selectively insert promoter elements upstream of genes of interest. Through this strategy, we show robust activation of both recombinant and endogenous genes across the E. coli chromosome. We then demonstrate precise tuning of expression levels by exchanging the promoter elements being inserted. Finally, we demonstrate that CAST activation can be used to synthetically induce ampicillin-resistant phenotypes in E. coli.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555240">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555240" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555240">
        <p class="paperTitle">A hybrid combination of in vitro cultured buccal mucosal cells using two different methodologies, complementing each other in successfully repairing a stricture-inflicted human male urethral epithelium</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555240" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555240" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Horiguchi, A.; Kushibiki, T.; Yoshine, M.; Shinchi, M.; Ojima, K.; Hirano, Y.; Katoh, S.; Iwasaki, M.; Surya Prakash, V.; Ichiyama, K.; Senthilkumar, R.; Preethy, S.; Abraham, S. J.</p>
        <p class="info">Score: 2.6, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555240' target='https://doi.org/10.1101/2023.08.29.555240'> 10.1101/2023.08.29.555240</a></p>
        <p class="abstract">BackgroundAutologous buccal mucosal tissue derived cell transplantation techniques in repairing a stricture inflicted male urethral epithelium have been evolving. There was not much of clarity on the cell type, in vitro culture methods and the mode of transplantation, until we reported our buccal epithelium expanded and encapsulated in scaffold-hybrid approach to urethral stricture (BEES-HAUS) clinical study yielding a successful engraftment and repair with a long-term patency. We herein report with technical clarity on the advantages of mixing two-dimensional (2D) monolayer cultured fibroblast like cells and three dimensional (3D) thermo-reversible gelation polymer (TGP) cultured cells; the former secreting IGF-1, a cytokine known for its healing effects and the latter expressing epithelial surface markers in flow cytometry, both sourced from human buccal tissue, together transplanted using TGP as a carrier.

MethodsHuman buccal tissues (n=22) redundant after urethroplasty surgery was used after informed consent and IEC approval. They were enzyme digested, divided into two portions; one was cultured as monolayer method (2D) and the other in 3D in TGP. Flowcytometry and quantification of IGF-1 in cell culture supernatant through the culture period were undertaken.

ResultsIn flowcytometry, the cells on day 0, lacked AE1/AE3 - pancytokeratin expression indicative of epithelial phenotype of culture, which progressively increased in the 3D-TGP group, during invitro culture of up to 21 days. The 2D showed expression of only fibroblasts like cells that were negative for AE1/AE3 but positive for CD140b. IGF-1 secretion was significantly higher in 2D cultures than in 3D-TGP (p-value &lt; 0.05).

ConclusionThe 3D- TGP cultured cells of epithelial nature and the 2D cultured fibroblast like cells secreting IGF-1, together when transplanted using TGP scaffold as a carrier, adapted to a hostile in vivo milieu after releasing the fibrous strands with urethrotomy, successfully engrafted and repaired a stricture-inflicted male urethral epithelium in the BEES-HAUS procedure. While this hybrid combination of cells are considered to have potential in managing other epithelial damages, further research of such hybrid combination and their behaviour in disease affected environments may help to expand this solution in regenerating and repairing other tissues and organs as well.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=123 SRC=&#34;FIGDIR/small/555240v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (70K):
org.highwire.dtl.DTLVardef@64a3aeorg.highwire.dtl.DTLVardef@136020borg.highwire.dtl.DTLVardef@c043acorg.highwire.dtl.DTLVardef@9b00c6_HPS_FORMAT_FIGEXP  M_FIG C_FIG Illustration of pathogenesis of urethral stricture and the contribution of the hybrid combination of two-dimensional (2D) and three dimensional, 3D-TGP (Thermo-reversible gelation polymer) cultured cells to the successful repair of the stricture-inflicted male urethral epithelium in the buccal epithelium expanded and encapsulated in scaffold-hybrid approach to urethral stricture (BEES-HAUS) technique.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556564">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556564" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556564">
        <p class="paperTitle">CRISPR-COPIES: An in silico platform for discovery of neutral integration sites for CRISPR/Cas-facilitated gene integration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556564" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556564" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boob, A. G.; Zhu, Z.; Intasian, P.; Jain, M.; Petrov, V. A.; Tan, S.-I.; Xun, G.; Zhao, H.</p>
        <p class="info">Score: 1.8, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556564' target='https://doi.org/10.1101/2023.09.06.556564'> 10.1101/2023.09.06.556564</a></p>
        <p class="abstract">The CRISPR/Cas system has emerged as a powerful tool for genome editing in metabolic engineering and human gene therapy. However, locating the optimal site on the chromosome to integrate heterologous genes using the CRISPR/Cas system remains an open question. Selecting a suitable site for gene integration involves considering multiple complex criteria, including factors related to CRISPR/Cas-mediated integration, genetic stability, and gene expression. Consequently, identifying such sites on specific or different chromosomal locations typically requires extensive characterization efforts. To address these challenges, we have developed CRISPR-COPIES, a COmputational Pipeline for the Identification of CRISPR/Cas-facilitated intEgration Sites. This tool leverages ScaNN, a state-of-the-art model on the embedding-based nearest neighbor search for fast and accurate off-target search and can identify genome-wide intergenic sites for most bacterial and fungal genomes within minutes. As a proof of concept, we utilized CRISPR-COPIES to characterize neutral integration sites in three diverse species: Saccharomyces cerevisiae, Cupriavidus necator, and a human cell line. In addition, we developed a user-friendly web interface for CRISPR-COPIES (https://biofoundry.web.illinois.edu/copies/). We anticipate that CRISPR-COPIES will serve as a valuable tool for targeted DNA integration and aid in the characterization of synthetic biology toolkits, enable rapid strain construction to produce valuable biochemicals and support human gene and cell therapy applications.

Graphical abstractOverview and application of CRISPR-COPIES in the field of biotechnology.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=93 SRC=&#34;FIGDIR/small/556564v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (20K):
org.highwire.dtl.DTLVardef@13bbf9aorg.highwire.dtl.DTLVardef@1a035d7org.highwire.dtl.DTLVardef@12edf18org.highwire.dtl.DTLVardef@38b0a6_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.556966">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.556966" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.556966">
        <p class="paperTitle">Engineering the ADDobody protein scaffold for generation of high-avidity ADDomer super-binders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.556966" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.556966" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Buzas, D.; Sun, H.; Toelzer, C.; Yadav, S. K. N.; Borucu, U.; Gautam, G.; Gupta, K.; Bufton, J.; Capin, J.; Sessions, R. B.; Garzoni, F.; Berger, I.; Schaffitzel, C.</p>
        <p class="info">Score: 1.2, Published: 2023-09-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.556966' target='https://doi.org/10.1101/2023.09.09.556966'> 10.1101/2023.09.09.556966</a></p>
        <p class="abstract">Adenovirus-derived dodecamer (ADDomer) nanoparticles comprise 60 copies of Adenovirus penton base protein (PBP). ADDomer is thermostable, rendering the storage, transport and deployment of ADDomer-based therapeutics independent of a cold-chain. To expand the scope of ADDomer nanoparticles for new applications, we engineered ADDobodies. ADDobodies represent the crown domain of the PBP, genetically separated from its multimerization domain. We inserted heterologous sequences into hyper-variable loops in the crown domain. The resulting ADDobodies were expressed at high yields in Escherichia coli, are monomeric and maintain thermostability. We solved the X-ray structure of an ADDobody prototype validating our design. We demonstrated that ADDobodies can be used to select a specific binder against a target in in vitro selection experiments using ribosome display, with an enrichment factor of ~104-fold in one selection round. We show that ADDobodies can be converted back into ADDomers by genetically reconnecting the selected ADDobody with the PBP multimerization domain from a different species, giving rise to a multivalent nanoparticle, called Chimera, confirmed by a 2.2 A structure determined by cryogenic electron microscopy (cryo-EM). Chimera comprises 60 binding sites, resulting in ultra-high, picomolar avidity to the target.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556532">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556532" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556532">
        <p class="paperTitle">An emergent biosynthetic pathway to essential amino acids by metabolic metathesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556532" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556532" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rivollier, J.; Gosling, S.; Pezo, V.; Heck, M.-P.; MARLIERE, P.</p>
        <p class="info">Score: 1.2, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556532' target='https://doi.org/10.1101/2023.09.06.556532'> 10.1101/2023.09.06.556532</a></p>
        <p class="abstract">An experimental approach to implanting foreign chemical reactions into living cells is to select for the catalytic production of a vital building-block such as an amino acid. Alkene metathesis is unknown in extant biochemistry, but is emerging as a new type of reaction to be catalyzed by protein enzymes. Here we show how the alkenic amino acid vinylglycine can be generated in a biocompatible reaction from 5-allyloxy-2-amino-pent-3-enoate (APE) by ring-closing metathesis catalyzed by a standard Hoveyda-Grubbs catalyst. The vinylglycine produced in situ is then used as a precursor of isoleucine and methionine, thus allowing the growth of strains of Escherichia coli requiring these essential amino acids. The robust nutritional screen we have developed paves the way for the directed evolution of genetically encoded metathesis enzymes and the chemical elaboration of metathesis coenzymes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557453">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557453" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557453">
        <p class="paperTitle">Manufacturing DNA in E. coli yields higher fidelity DNA than in vitro enzymatic synthesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557453" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557453" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hersch, S. J.; Chandrasekaran, S.; Lam, J.; Nafissi, N.; Slavcev, R.</p>
        <p class="info">Score: 1.0, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557453' target='https://doi.org/10.1101/2023.09.12.557453'> 10.1101/2023.09.12.557453</a></p>
        <p class="abstract">The rise of biotechnologies such as gene therapy have brought DNA vectors to the forefront of pharmaceutical development. The quality of the genetic starting material plays a pivotal role in determining the quality of the final product. In this study we examined the fidelity of DNA replication using enzymatic methods (in vitro) compared to plasmid DNA produced in vivo in E. coli. Next-generation sequencing approaches predominantly rely on in vitro polymerases, which have inherent limitations in sensitivity. To address this challenge, we introduce a novel assay based on loss-of-function (LOF) mutations in the conditionally toxic sacB gene. Our findings show that DNA production in E. coli results in significantly fewer LOF mutations (approximately 80- to 3000-fold less) compared to various enzymatic DNA synthesis methods. This includes the most accurate PCR polymerase (Q5) and a commonly employed rolling circle amplification (RCA) DNA polymerase (Phi29). These results suggest that using low-fidelity starting material DNA synthesized in vitro by PCR or RCA may introduce a substantial number of impurities, potentially affecting the quality and yield of final pharmaceutical products. In summary, our study underscores that DNA synthesized in vitro has a significantly higher mutation rate than DNA produced traditionally in E. coli. Therefore, utilizing in vitro enzymatically-produced DNA in biotechnology and biomanufacturing may entail considerable fidelity-related risks, while DNA starting material derived from E. coli substantially mitigates this risk, enhancing overall quality in the production processes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
