<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Computational design of sequence-specific DNA-binding proteins
Authors: Glasscock, C. J.; Pecoraro, R.; McHugh, R.; Doyle, L. A.; Chen, W.; Boivin, O.; Lonnquist, B.; Na, E.; Politanska, Y.; Haddox, H. K.; Cox, D.; Norn, C.; Coventry, B.; Goreshnik, I.; Vafeados, D.; Lee, G. R.; Gordan, R.; Stoddard, B. L.; DiMaio, F.; Baker, D.
Score: 189.9, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558720
Sequence-specific DNA-binding proteins (DBPs) play critical roles in biology and biotechnology, and there has been considerable interest in the engineering of DBPs with new or altered specificities for genome editing and other applications.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Computational design of sequence-specific DNA-binding proteins
Authors: Glasscock, C. J.; Pecoraro, R.; McHugh, R.; Doyle, L. A.; Chen, W.; Boivin, O.; Lonnquist, B.; Na, E.; Politanska, Y.; Haddox, H. K.; Cox, D.; Norn, C.; Coventry, B.; Goreshnik, I.; Vafeados, D.; Lee, G. R.; Gordan, R.; Stoddard, B. L.; DiMaio, F.; Baker, D.
Score: 189.9, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558720
Sequence-specific DNA-binding proteins (DBPs) play critical roles in biology and biotechnology, and there has been considerable interest in the engineering of DBPs with new or altered specificities for genome editing and other applications." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-01T10:37:47+00:00" />
<meta property="article:modified_time" content="2023-10-01T10:37:47+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Computational design of sequence-specific DNA-binding proteins
Authors: Glasscock, C. J.; Pecoraro, R.; McHugh, R.; Doyle, L. A.; Chen, W.; Boivin, O.; Lonnquist, B.; Na, E.; Politanska, Y.; Haddox, H. K.; Cox, D.; Norn, C.; Coventry, B.; Goreshnik, I.; Vafeados, D.; Lee, G. R.; Gordan, R.; Stoddard, B. L.; DiMaio, F.; Baker, D.
Score: 189.9, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558720
Sequence-specific DNA-binding proteins (DBPs) play critical roles in biology and biotechnology, and there has been considerable interest in the engineering of DBPs with new or altered specificities for genome editing and other applications."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Computational design of sequence-specific DNA-binding proteins\nAuthors: Glasscock, C. J.; Pecoraro, R.; McHugh, R.; Doyle, L. A.; Chen, W.; Boivin, O.; Lonnquist, B.; Na, E.; Politanska, Y.; Haddox, H. K.; Cox, D.; Norn, C.; Coventry, B.; Goreshnik, I.; Vafeados, D.; Lee, G. R.; Gordan, R.; Stoddard, B. L.; DiMaio, F.; Baker, D.\nScore: 189.9, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558720\nSequence-specific DNA-binding proteins (DBPs) play critical roles in biology and biotechnology, and there has been considerable interest in the engineering of DBPs with new or altered specificities for genome editing and other applications.",
  "keywords": [
    
  ],
  "articleBody": " Computational design of sequence-specific DNA-binding proteins\nAuthors: Glasscock, C. J.; Pecoraro, R.; McHugh, R.; Doyle, L. A.; Chen, W.; Boivin, O.; Lonnquist, B.; Na, E.; Politanska, Y.; Haddox, H. K.; Cox, D.; Norn, C.; Coventry, B.; Goreshnik, I.; Vafeados, D.; Lee, G. R.; Gordan, R.; Stoddard, B. L.; DiMaio, F.; Baker, D.\nScore: 189.9, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558720\nSequence-specific DNA-binding proteins (DBPs) play critical roles in biology and biotechnology, and there has been considerable interest in the engineering of DBPs with new or altered specificities for genome editing and other applications. While there has been some success in reprogramming naturally occurring DBPs using selection methods, the computational design of new DBPs that recognize arbitrary target sites remains an outstanding challenge. We describe a computational method for the design of small DBPs that recognize specific target sequences through interactions with bases in the major groove, and employ this method in conjunction with experimental screening to generate binders for 5 distinct DNA targets. These binders exhibit specificity closely matching the computational models for the target DNA sequences at as many as 6 base positions and affinities as low as 30-100 nM. The crystal structure of a designed DBP-target site complex is in close agreement with the design model, highlighting the accuracy of the design method. The designed DBPs function in both Escherichia coli and mammalian cells to repress and activate transcription of neighboring genes. Our method is a substantial step towards a general route to small and hence readily deliverable sequence-specific DBPs for gene regulation and editing.\nEngineered bacteria launch and control an oncolytic virus\nAuthors: Singer, Z. S.; Pabon, J.; Huang, H.; Rice, C. M.; Danino, T.\nScore: 14.3, Published: 2023-09-28 DOI: 10.1101/2023.09.28.559873\nThe ability of bacteria and viruses to selectively replicate in tumors has led to synthetic engineering of new microbial therapies. Here we design a cooperative strategy whereby S. typhimurium bacteria transcribe and deliver the Senecavirus A RNA genome inside host cells, launching a potent oncolytic viral infection. Then, we engineer the virus to require a bacterially delivered protease in order to achieve virion maturation, demonstrating bacterial control over the virus. This work extends bacterially delivered therapeutics to viral genomes, and the governing of a viral population through engineered microbial interactions.\nLeucine zipper-based sorting system enables generation of multi-functional CAR T cells\nAuthors: James, S. E.; Chen, S.; Ng, B. D.; Fischman, J. S.; Jahn, L.; Boardman, A. P.; Rajagopalan, A.; Elias, H. K.; Massa, A.; Manuele, D.; Nichols, K. B.; Lazrak, A.; Lee, N.; Fei, T.; DeWolf, S.; Peled, J.; Vardhana, S. A.; Klebanoff, C. A.; van den Brink, M. R. M.\nScore: 7.5, Published: 2023-09-13 DOI: 10.1101/2023.09.13.557232\nResistance to chimeric antigen receptor (CAR) T cell therapy develops through multiple mechanisms including antigen-loss escape and tumor-induced immune suppression. Expression of multiple CARs may overcome multi-antigen-loss escape. Similarly, expression of switch receptors that convert inhibitory immune checkpoint signals into positive costimulatory signals may enhance CAR T cell activity in the tumor microenvironment. Engineering multiple features into one cell product, however, is limited by transgene packaging constraints of current vector systems. Here, we describe a leucine zipper-based cell sorting methodology that enables selective single-step immunomagnetic purification of cells co-transduced with two vectors, designed to potentially double the number of incorporated transgenes. This \"Zip-sorting\" system facilitated generation of T cells simultaneously expressing up to four CARs and co-expressing up to three switch receptors. These multi-CAR multi-Switch receptor arrays enabled T cells to eliminate antigenically heterogeneous syngeneic leukemia populations co-expressing multiple inhibitory ligands. Zip-sorted multi-CAR multi-Switch receptor T cells represent a potent therapeutic strategy to overcome multiple mechanisms of CAR T cell resistance.\nEngineered retrons generate genome-independent protein-binding DNA for cellular control\nAuthors: Lee, G.; Kim, J.\nScore: 5.1, Published: 2023-09-28 DOI: 10.1101/2023.09.27.556556\nDNA-protein interactions are core components of myriad natural and synthetic gene networks. Despite the potential new design space, DNA-protein interactions remain underexploited in vivo due to challenges in controlling specific DNA segments, including protein-binding sequences, independently of the genome. Here we engineer retrons, prokaryotic retroelements, to intracellularly generate genome-independent programmable small DNA for sequence-specific protein-binding. Using reprogrammed retron-derived DNA for allosteric transcription factor, we demonstrated dynamic regulation of synthetic gene networks and construction of automated feedback circuits for signal amplification, adaptation, and memory. Furthermore, we developed a new class of stimuli-responsive molecular 'bait and prey' that enable modular, rapid, and post-translational control of protein subcellular localization. This work substantially expands possible application area of DNA-protein interactions, laying the foundation for technical advances in synthetic biology.\nHigh plasticity of ribosomal DNA organization in budding yeast\nAuthors: Jiang, S.; Cai, Z.; Wang, Y.; Zeng, C.; Zhang, J.; Yu, W.; Su, C.; Zhao, S.; Chen, Y.; Shen, Y.; Ma, Y.; Cai, Y.; Dai, J.\nScore: 3.8, Published: 2023-09-26 DOI: 10.1101/2023.09.26.559284\nIn eukaryotic genomes, ribosomal DNA (rDNA) generally resides as a highly repetitive and dynamic structure, making it difficult to study. Here, a synthetic rDNA array on chromosome III in budding yeast was constructed to serve as the sole source of rRNA. Utilizing the loxPsym site within each rDNA repeat and the Cre recombinase, we were able to reduce the copy number to as few as eight copies. Additionally, we constructed strains with two or three rDNA arrays, and found that the presence of multiple arrays did not affect the formation of a single nucleolus. Although alteration on the position and number of rDNA arrays did impact three-dimensional genome structure, the additional rDNA arrays had no deleterious influence on cell growth or transcriptomes. Together, this study sheds light on the high plasticity of rDNA organization and opens up opportunities for future rDNA engineering. HighlightsO_LIA method was established for efficient construction of synthetic rDNA arrays in budding yeast C_LIO_LIThe rDNA repeats in a haploid yeast can be reduced to as few as eight copies to support cell viability C_LIO_LIYeast cells with two or three DNA arrays on distinct chromosomes form a single nucleolus. C_LIO_LIDispersed rDNA arrays result in no deleterious influence on cell growth or transcriptomes. C_LI\nA genetic circuit on a single DNA molecule as autonomous dissipative nanodevice\nAuthors: Greiss, F.; Lardon, N.; Schutz, L.; Barak, Y.; Daube, S. S.; Weinhold, E.; Noireaux, V.; Bar-Ziv, R.\nScore: 7.0, Published: 2023-09-21 DOI: 10.1101/2023.09.21.558768\nRealizing genetic circuits on single DNA molecules as self-encoded dissipative nanodevices is a major step toward miniaturization of autonomous biological systems. A circuit operating on a single DNA implies that genetically encoded proteins localize during coupled transcriptiontranslation to DNA, but a single-molecule measurement demonstrating this has remained a challenge. Here, we used a genetically encoded fluorescent reporter system with improved spatiotemporal resolution and observed the synthesis of individual proteins tethered to a DNA molecule by transient complexes of RNA polymerase, messenger RNA, and ribosome. Against expectations in dilute cell-free conditions where equilibrium considerations favor dispersion, these nascent proteins linger long enough to regulate cascaded reactions on the same DNA. We rationally designed a pulsatile genetic circuit by encoding an activator and repressor in feedback on the same DNA molecule. Driven by the local synthesis of only several proteins per hour and gene, the circuit dynamics exhibited enhanced variability between individual DNA molecules, and fluctuations with a broad power spectrum. Our results demonstrate that coexpressional localization, as a nonequilibrium process, facilitates single-DNA genetic circuits as dissipative nanodevices, with implications for nanobiotechnology applications and artificial cell design.\nSynthetic control of actin polymerization and symmetry breaking in activeprotocells\nAuthors: Razavi, S.; Wong, F.; Abubaker-Sharif, B.; Matsubayashi, H. T.; Nakamura, H.; Sandoval, E.; Robinson, D. N.; Chen, B.; Liu, J.; Iglesias, P. A.; Inoue, T.\nScore: 3.2, Published: 2023-09-23 DOI: 10.1101/2023.09.22.559060\nNon-linear biomolecular interactions on the membranes drive membrane remodeling that underlies fundamental biological processes including chemotaxis, cytokinesis, and endocytosis. The multitude of biomolecules, the redundancy in their interactions, and the importance of spatiotemporal context in membrane organization hampers understanding the physical principles governing membrane mechanics. A minimal, in vitro system that models the functional interactions between molecular signaling and membrane remodeling, while remaining faithful to cellular physiology and geometry is powerful yet remains unachieved. Here, inspired by the biophysical processes underpinning chemotaxis, we reconstituted externally-controlled actin polymerization inside giant unilamellar vesicles, guiding self-organization on the membrane. We show that applying undirected external chemical inputs to this system results in directed actin polymerization and membrane deformation that are uncorrelated with upstream biochemical cues, indicating symmetry breaking. A biophysical model of the dynamics and mechanics of both actin polymerization and membrane shape suggests that inhomogeneous distributions of actin generate membrane shape deformations in a non-linear fashion, a prediction consistent with experimental measurements and subsequent local perturbations. The active protocellular system demonstrates the interplay between actin dynamics and membrane shape in a symmetry breaking context that is relevant to chemotaxis and a suite of other biological processes.\nThe CASwitch: a synthetic biology solution for high-performance inducible gene expression systems in biotechnology\nAuthors: De Carluccio, G.; Fusco, V.; di Bernardo, D.\nScore: 11.7, Published: 2023-09-20 DOI: 10.1101/2023.09.20.558637\nInducible gene expression systems can be used to control the expression of a gene of interest by means of a small-molecule. One of the most common designs involves engineering a small-molecule responsive transcription factor (TF) and its cognate promoter, which often results in a compromise between minimal uninduced background expression (leakiness) and maximal induced expression. Here, we focussed on an alternative strategy using quantitative synthetic biology to mitigate leakiness while maintaining high expression, without modifying neither the TF nor the promoter. Through mathematical modelling and experimental validations, we designed the CASwitch, a mammalian synthetic gene circuit based on combining two well-known network motifs: the Coherent Feed-Forward Loop (CFFL) and the Mutual Inhibition (MI). The CASwitch combines the CRISPR-Cas endoribonuclease CasRx with the state-of-the-art Tet-On3G inducible gene system to achieve high performances. To demonstrate the potentialities of the CASwitch, we applied it to three different scenarios: enhancing a whole-cell biosensor, controlling expression of a toxic gene and inducible production of Adeno-Associated Virus (AAV) vectors.\nPassivating blunt-ended helices to control monodispersity and multi-subunit assembly of DNA origami structures\nAuthors: Berengut, J. F.; Rizzuto, F. J.; Berg, W. R.; Lee, L. K.\nScore: 3.2, Published: 2023-09-11 DOI: 10.1101/2023.09.09.557003\nDNA origami facilitates the synthesis of bespoke nanoscale structures suitable for a wide range of applications. Effective design requires prevention of uncontrolled aggregation, while still permitting directed assembly of multi-subunit superstructures. Uncontrolled aggregation can be caused by base-stacking interactions between arrays of blunt-ended helices on different structures, which are routinely passivated by incorporating disordered regions as either scaffold loops or poly-nucleotide brushes (usually poly-thymine) at the end of DNA helices. Such disordered regions are ubiquitous in DNA origami structures yet their exact design requirements in different chemical environments are ill defined. In this study, we systematically examine the use of scaffold loops and poly-nucleotide brushes for passivation and for controlling multi-subunit assembly. We assess the dependence of length and sequence for preventing aggregation amidst a titration of MgCl2 concentrations and the suitability of each strategy for enabling controlled multi-subunit assembly. We then introduce a novel strategy where double-stranded DNA helices run orthogonal to arrays of blunt-ended DNA helices forming a steric shield that prevents base stacking. The results define the limitations of each method and important design considerations for achieving monodispersity. For example, poly-thymine brushes are most effective for achieving monodispersity in the broadest conditions whereas scaffold loops can facilitate directed multi-subunit assembly. Finally, orthogonal DNA helices remove the need for disordered regions altogether, prevent aggregation in a broad range of MgCl2 concentrations and facilitate directed multi-subunit assembly. This study expands the design tools available and enables a more informed approach for achieving control of monodispersity and multi-subunit assembly in DNA origami structures.\nA synthetic Notch (synNotch) system linking intratumoral immune-cancer cell communication to a synthetic blood biomarker assay\nAuthors: Fu, Y.; Wang, T.; Ronald, J. A.\nScore: 2.2, Published: 2023-09-26 DOI: 10.1101/2023.09.26.559329\nIntroductionCellular immunotherapy has greatly improved cancer treatment in recent years. For instance, chimeric antigen receptor (CAR) T-cell therapy has been proven highly effective in treating hematological malignancies, and many CAR cell designs are being explored for solid tumors. However, many questions remain why responses differ across patients and some tumor types are resistant. Improved and relatively inexpensive ways to monitor these cells could provide some answers. Clinically, blood tests are regularly used to monitor these therapies, but blood signals often do not reflect the activity of immune cells within the tumor(s). Here, using the synthetic Notch (synNotch) receptor that tethers antigen binding to customized transgene expression, we linked intratumoral immune-cancer cell communication to a simple secreted reporter blood test. Specifically, we engineered immune cells with a CD19-targeted synNotch receptor and demonstrate that binding to CD19 on cancer cells in vivo resulted in the production of secreted embryonic alkaline phosphatase (SEAP) at levels that are readily detected in the blood. Methods and ResultsJurkat T cells were engineered via sequential lentiviral transduction of two components: an anti-CD19 synNotch receptor and asynNotchresponse element encoding SEAP. Co-culture of engineered cells with CD19+, but not CD19-, Nalm6 cells, resulted in significantly elevated SEAP in media. Nod-scid-gamma (NSG) mice were subcutaneously injected with either CD19+ or CD19- Nalm6 cells. Intratumoral injection of engineered T cells (1x107) resulted significantly elevated blood SEAP activity in mice bearing CD19+ tumors (n=7), but not CD19- tumors (n=5). DiscussionOur synNotch reporter system allows for the monitoring of antigen-dependent intratumoral immune-cancer cell interactions through a simple and convenient blood test. Continued development of this system for different target antigens of interest should provide a broadly applicable platform for improved monitoring of many cell-based immunotherapies during their initial development and clinical translation, ultimately improving our understanding of design considerations and patient-specific responses.\n",
  "wordCount" : "2256",
  "inLanguage": "en",
  "datePublished": "2023-10-01T10:37:47Z",
  "dateModified": "2023-10-01T10:37:47Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on October 1, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558720">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558720" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558720">
        <p class="paperTitle">Computational design of sequence-specific DNA-binding proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558720" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558720" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Glasscock, C. J.; Pecoraro, R.; McHugh, R.; Doyle, L. A.; Chen, W.; Boivin, O.; Lonnquist, B.; Na, E.; Politanska, Y.; Haddox, H. K.; Cox, D.; Norn, C.; Coventry, B.; Goreshnik, I.; Vafeados, D.; Lee, G. R.; Gordan, R.; Stoddard, B. L.; DiMaio, F.; Baker, D.</p>
        <p class="info">Score: 189.9, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558720' target='https://doi.org/10.1101/2023.09.20.558720'> 10.1101/2023.09.20.558720</a></p>
        <p class="abstract">Sequence-specific DNA-binding proteins (DBPs) play critical roles in biology and biotechnology, and there has been considerable interest in the engineering of DBPs with new or altered specificities for genome editing and other applications. While there has been some success in reprogramming naturally occurring DBPs using selection methods, the computational design of new DBPs that recognize arbitrary target sites remains an outstanding challenge. We describe a computational method for the design of small DBPs that recognize specific target sequences through interactions with bases in the major groove, and employ this method in conjunction with experimental screening to generate binders for 5 distinct DNA targets. These binders exhibit specificity closely matching the computational models for the target DNA sequences at as many as 6 base positions and affinities as low as 30-100 nM. The crystal structure of a designed DBP-target site complex is in close agreement with the design model, highlighting the accuracy of the design method. The designed DBPs function in both Escherichia coli and mammalian cells to repress and activate transcription of neighboring genes. Our method is a substantial step towards a general route to small and hence readily deliverable sequence-specific DBPs for gene regulation and editing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.559873">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.559873" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.559873">
        <p class="paperTitle">Engineered bacteria launch and control an oncolytic virus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.559873" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.559873" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Singer, Z. S.; Pabon, J.; Huang, H.; Rice, C. M.; Danino, T.</p>
        <p class="info">Score: 14.3, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.559873' target='https://doi.org/10.1101/2023.09.28.559873'> 10.1101/2023.09.28.559873</a></p>
        <p class="abstract">The ability of bacteria and viruses to selectively replicate in tumors has led to synthetic engineering of new microbial therapies. Here we design a cooperative strategy whereby S. typhimurium bacteria transcribe and deliver the Senecavirus A RNA genome inside host cells, launching a potent oncolytic viral infection. Then, we engineer the virus to require a bacterially delivered protease in order to achieve virion maturation, demonstrating bacterial control over the virus. This work extends bacterially delivered therapeutics to viral genomes, and the governing of a viral population through engineered microbial interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557232">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557232" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557232">
        <p class="paperTitle">Leucine zipper-based sorting system enables generation of multi-functional CAR T cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557232" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557232" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: James, S. E.; Chen, S.; Ng, B. D.; Fischman, J. S.; Jahn, L.; Boardman, A. P.; Rajagopalan, A.; Elias, H. K.; Massa, A.; Manuele, D.; Nichols, K. B.; Lazrak, A.; Lee, N.; Fei, T.; DeWolf, S.; Peled, J.; Vardhana, S. A.; Klebanoff, C. A.; van den Brink, M. R. M.</p>
        <p class="info">Score: 7.5, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557232' target='https://doi.org/10.1101/2023.09.13.557232'> 10.1101/2023.09.13.557232</a></p>
        <p class="abstract">Resistance to chimeric antigen receptor (CAR) T cell therapy develops through multiple mechanisms including antigen-loss escape and tumor-induced immune suppression. Expression of multiple CARs may overcome multi-antigen-loss escape. Similarly, expression of switch receptors that convert inhibitory immune checkpoint signals into positive costimulatory signals may enhance CAR T cell activity in the tumor microenvironment. Engineering multiple features into one cell product, however, is limited by transgene packaging constraints of current vector systems. Here, we describe a leucine zipper-based cell sorting methodology that enables selective single-step immunomagnetic purification of cells co-transduced with two vectors, designed to potentially double the number of incorporated transgenes. This &#34;Zip-sorting&#34; system facilitated generation of T cells simultaneously expressing up to four CARs and co-expressing up to three switch receptors. These multi-CAR multi-Switch receptor arrays enabled T cells to eliminate antigenically heterogeneous syngeneic leukemia populations co-expressing multiple inhibitory ligands. Zip-sorted multi-CAR multi-Switch receptor T cells represent a potent therapeutic strategy to overcome multiple mechanisms of CAR T cell resistance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.556556">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.556556" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.556556">
        <p class="paperTitle">Engineered retrons generate genome-independent protein-binding DNA for cellular control</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.556556" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.556556" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, G.; Kim, J.</p>
        <p class="info">Score: 5.1, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.556556' target='https://doi.org/10.1101/2023.09.27.556556'> 10.1101/2023.09.27.556556</a></p>
        <p class="abstract">DNA-protein interactions are core components of myriad natural and synthetic gene networks. Despite the potential new design space, DNA-protein interactions remain underexploited in vivo due to challenges in controlling specific DNA segments, including protein-binding sequences, independently of the genome. Here we engineer retrons, prokaryotic retroelements, to intracellularly generate genome-independent programmable small DNA for sequence-specific protein-binding. Using reprogrammed retron-derived DNA for allosteric transcription factor, we demonstrated dynamic regulation of synthetic gene networks and construction of automated feedback circuits for signal amplification, adaptation, and memory. Furthermore, we developed a new class of stimuli-responsive molecular &#39;bait and prey&#39; that enable modular, rapid, and post-translational control of protein subcellular localization. This work substantially expands possible application area of DNA-protein interactions, laying the foundation for technical advances in synthetic biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559284">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559284" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559284">
        <p class="paperTitle">High plasticity of ribosomal DNA organization in budding yeast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559284" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559284" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, S.; Cai, Z.; Wang, Y.; Zeng, C.; Zhang, J.; Yu, W.; Su, C.; Zhao, S.; Chen, Y.; Shen, Y.; Ma, Y.; Cai, Y.; Dai, J.</p>
        <p class="info">Score: 3.8, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559284' target='https://doi.org/10.1101/2023.09.26.559284'> 10.1101/2023.09.26.559284</a></p>
        <p class="abstract">In eukaryotic genomes, ribosomal DNA (rDNA) generally resides as a highly repetitive and dynamic structure, making it difficult to study. Here, a synthetic rDNA array on chromosome III in budding yeast was constructed to serve as the sole source of rRNA. Utilizing the loxPsym site within each rDNA repeat and the Cre recombinase, we were able to reduce the copy number to as few as eight copies. Additionally, we constructed strains with two or three rDNA arrays, and found that the presence of multiple arrays did not affect the formation of a single nucleolus. Although alteration on the position and number of rDNA arrays did impact three-dimensional genome structure, the additional rDNA arrays had no deleterious influence on cell growth or transcriptomes. Together, this study sheds light on the high plasticity of rDNA organization and opens up opportunities for future rDNA engineering.

HighlightsO_LIA method was established for efficient construction of synthetic rDNA arrays in budding yeast
C_LIO_LIThe rDNA repeats in a haploid yeast can be reduced to as few as eight copies to support cell viability
C_LIO_LIYeast cells with two or three DNA arrays on distinct chromosomes form a single nucleolus.
C_LIO_LIDispersed rDNA arrays result in no deleterious influence on cell growth or transcriptomes.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558768">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558768" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558768">
        <p class="paperTitle">A genetic circuit on a single DNA molecule as autonomous dissipative nanodevice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558768" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558768" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Greiss, F.; Lardon, N.; Schutz, L.; Barak, Y.; Daube, S. S.; Weinhold, E.; Noireaux, V.; Bar-Ziv, R.</p>
        <p class="info">Score: 7.0, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558768' target='https://doi.org/10.1101/2023.09.21.558768'> 10.1101/2023.09.21.558768</a></p>
        <p class="abstract">Realizing genetic circuits on single DNA molecules as self-encoded dissipative nanodevices is a major step toward miniaturization of autonomous biological systems. A circuit operating on a single DNA implies that genetically encoded proteins localize during coupled transcriptiontranslation to DNA, but a single-molecule measurement demonstrating this has remained a challenge. Here, we used a genetically encoded fluorescent reporter system with improved spatiotemporal resolution and observed the synthesis of individual proteins tethered to a DNA molecule by transient complexes of RNA polymerase, messenger RNA, and ribosome. Against expectations in dilute cell-free conditions where equilibrium considerations favor dispersion, these nascent proteins linger long enough to regulate cascaded reactions on the same DNA. We rationally designed a pulsatile genetic circuit by encoding an activator and repressor in feedback on the same DNA molecule. Driven by the local synthesis of only several proteins per hour and gene, the circuit dynamics exhibited enhanced variability between individual DNA molecules, and fluctuations with a broad power spectrum. Our results demonstrate that coexpressional localization, as a nonequilibrium process, facilitates single-DNA genetic circuits as dissipative nanodevices, with implications for nanobiotechnology applications and artificial cell design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.559060">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.559060" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.559060">
        <p class="paperTitle">Synthetic control of actin polymerization and symmetry breaking in activeprotocells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.559060" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.559060" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Razavi, S.; Wong, F.; Abubaker-Sharif, B.; Matsubayashi, H. T.; Nakamura, H.; Sandoval, E.; Robinson, D. N.; Chen, B.; Liu, J.; Iglesias, P. A.; Inoue, T.</p>
        <p class="info">Score: 3.2, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.559060' target='https://doi.org/10.1101/2023.09.22.559060'> 10.1101/2023.09.22.559060</a></p>
        <p class="abstract">Non-linear biomolecular interactions on the membranes drive membrane remodeling that underlies fundamental biological processes including chemotaxis, cytokinesis, and endocytosis. The multitude of biomolecules, the redundancy in their interactions, and the importance of spatiotemporal context in membrane organization hampers understanding the physical principles governing membrane mechanics. A minimal, in vitro system that models the functional interactions between molecular signaling and membrane remodeling, while remaining faithful to cellular physiology and geometry is powerful yet remains unachieved. Here, inspired by the biophysical processes underpinning chemotaxis, we reconstituted externally-controlled actin polymerization inside giant unilamellar vesicles, guiding self-organization on the membrane. We show that applying undirected external chemical inputs to this system results in directed actin polymerization and membrane deformation that are uncorrelated with upstream biochemical cues, indicating symmetry breaking. A biophysical model of the dynamics and mechanics of both actin polymerization and membrane shape suggests that inhomogeneous distributions of actin generate membrane shape deformations in a non-linear fashion, a prediction consistent with experimental measurements and subsequent local perturbations. The active protocellular system demonstrates the interplay between actin dynamics and membrane shape in a symmetry breaking context that is relevant to chemotaxis and a suite of other biological processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558637">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558637" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558637">
        <p class="paperTitle">The CASwitch: a synthetic biology solution for high-performance inducible gene expression systems in biotechnology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558637" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558637" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: De Carluccio, G.; Fusco, V.; di Bernardo, D.</p>
        <p class="info">Score: 11.7, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558637' target='https://doi.org/10.1101/2023.09.20.558637'> 10.1101/2023.09.20.558637</a></p>
        <p class="abstract">Inducible gene expression systems can be used to control the expression of a gene of interest by means of a small-molecule. One of the most common designs involves engineering a small-molecule responsive transcription factor (TF) and its cognate promoter, which often results in a compromise between minimal uninduced background expression (leakiness) and maximal induced expression. Here, we focussed on an alternative strategy using quantitative synthetic biology to mitigate leakiness while maintaining high expression, without modifying neither the TF nor the promoter. Through mathematical modelling and experimental validations, we designed the CASwitch, a mammalian synthetic gene circuit based on combining two well-known network motifs: the Coherent Feed-Forward Loop (CFFL) and the Mutual Inhibition (MI). The CASwitch combines the CRISPR-Cas endoribonuclease CasRx with the state-of-the-art Tet-On3G inducible gene system to achieve high performances. To demonstrate the potentialities of the CASwitch, we applied it to three different scenarios: enhancing a whole-cell biosensor, controlling expression of a toxic gene and inducible production of Adeno-Associated Virus (AAV) vectors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.557003">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.557003" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.557003">
        <p class="paperTitle">Passivating blunt-ended helices to control monodispersity and multi-subunit assembly of DNA origami structures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.557003" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.557003" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Berengut, J. F.; Rizzuto, F. J.; Berg, W. R.; Lee, L. K.</p>
        <p class="info">Score: 3.2, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.557003' target='https://doi.org/10.1101/2023.09.09.557003'> 10.1101/2023.09.09.557003</a></p>
        <p class="abstract">DNA origami facilitates the synthesis of bespoke nanoscale structures suitable for a wide range of applications. Effective design requires prevention of uncontrolled aggregation, while still permitting directed assembly of multi-subunit superstructures. Uncontrolled aggregation can be caused by base-stacking interactions between arrays of blunt-ended helices on different structures, which are routinely passivated by incorporating disordered regions as either scaffold loops or poly-nucleotide brushes (usually poly-thymine) at the end of DNA helices. Such disordered regions are ubiquitous in DNA origami structures yet their exact design requirements in different chemical environments are ill defined. In this study, we systematically examine the use of scaffold loops and poly-nucleotide brushes for passivation and for controlling multi-subunit assembly. We assess the dependence of length and sequence for preventing aggregation amidst a titration of MgCl2 concentrations and the suitability of each strategy for enabling controlled multi-subunit assembly. We then introduce a novel strategy where double-stranded DNA helices run orthogonal to arrays of blunt-ended DNA helices forming a steric shield that prevents base stacking. The results define the limitations of each method and important design considerations for achieving monodispersity. For example, poly-thymine brushes are most effective for achieving monodispersity in the broadest conditions whereas scaffold loops can facilitate directed multi-subunit assembly. Finally, orthogonal DNA helices remove the need for disordered regions altogether, prevent aggregation in a broad range of MgCl2 concentrations and facilitate directed multi-subunit assembly. This study expands the design tools available and enables a more informed approach for achieving control of monodispersity and multi-subunit assembly in DNA origami structures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559329">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559329" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559329">
        <p class="paperTitle">A synthetic Notch (synNotch) system linking intratumoral immune-cancer cell communication to a synthetic blood biomarker assay</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559329" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559329" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fu, Y.; Wang, T.; Ronald, J. A.</p>
        <p class="info">Score: 2.2, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559329' target='https://doi.org/10.1101/2023.09.26.559329'> 10.1101/2023.09.26.559329</a></p>
        <p class="abstract">IntroductionCellular immunotherapy has greatly improved cancer treatment in recent years. For instance, chimeric antigen receptor (CAR) T-cell therapy has been proven highly effective in treating hematological malignancies, and many CAR cell designs are being explored for solid tumors. However, many questions remain why responses differ across patients and some tumor types are resistant. Improved and relatively inexpensive ways to monitor these cells could provide some answers. Clinically, blood tests are regularly used to monitor these therapies, but blood signals often do not reflect the activity of immune cells within the tumor(s). Here, using the synthetic Notch (synNotch) receptor that tethers antigen binding to customized transgene expression, we linked intratumoral immune-cancer cell communication to a simple secreted reporter blood test. Specifically, we engineered immune cells with a CD19-targeted synNotch receptor and demonstrate that binding to CD19 on cancer cells in vivo resulted in the production of secreted embryonic alkaline phosphatase (SEAP) at levels that are readily detected in the blood.

Methods and ResultsJurkat T cells were engineered via sequential lentiviral transduction of two components: an anti-CD19 synNotch receptor and asynNotchresponse element encoding SEAP. Co-culture of engineered cells with CD19&#43;, but not CD19-, Nalm6 cells, resulted in significantly elevated SEAP in media. Nod-scid-gamma (NSG) mice were subcutaneously injected with either CD19&#43; or CD19- Nalm6 cells. Intratumoral injection of engineered T cells (1x107) resulted significantly elevated blood SEAP activity in mice bearing CD19&#43; tumors (n=7), but not CD19- tumors (n=5).

DiscussionOur synNotch reporter system allows for the monitoring of antigen-dependent intratumoral immune-cancer cell interactions through a simple and convenient blood test. Continued development of this system for different target antigens of interest should provide a broadly applicable platform for improved monitoring of many cell-based immunotherapies during their initial development and clinical translation, ultimately improving our understanding of design considerations and patient-specific responses.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
