<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Engineering the highly productive cyanobacterium Synechococcus sp. PCC 11901
Authors: Victoria, A. J.; Selao, T. T.; Moreno-Cabezuelo, J. A.; Mills, L. A.; Gale, G. A.; Lea-Smith, D. J.; McCormick, A. J.
Score: 11.0, Published: 2023-08-07 DOI: 10.1101/2023.08.04.552009
Synechococcus sp. PCC 11901 (PCC 11901) is a fast-growing marine cyanobacterial strain that has a capacity for sustained biomass accumulation to very high cell densities, comparable to that achieved by commercially relevant heterotrophic organisms.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Engineering the highly productive cyanobacterium Synechococcus sp. PCC 11901
Authors: Victoria, A. J.; Selao, T. T.; Moreno-Cabezuelo, J. A.; Mills, L. A.; Gale, G. A.; Lea-Smith, D. J.; McCormick, A. J.
Score: 11.0, Published: 2023-08-07 DOI: 10.1101/2023.08.04.552009
Synechococcus sp. PCC 11901 (PCC 11901) is a fast-growing marine cyanobacterial strain that has a capacity for sustained biomass accumulation to very high cell densities, comparable to that achieved by commercially relevant heterotrophic organisms." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-09T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-08-09T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Engineering the highly productive cyanobacterium Synechococcus sp. PCC 11901
Authors: Victoria, A. J.; Selao, T. T.; Moreno-Cabezuelo, J. A.; Mills, L. A.; Gale, G. A.; Lea-Smith, D. J.; McCormick, A. J.
Score: 11.0, Published: 2023-08-07 DOI: 10.1101/2023.08.04.552009
Synechococcus sp. PCC 11901 (PCC 11901) is a fast-growing marine cyanobacterial strain that has a capacity for sustained biomass accumulation to very high cell densities, comparable to that achieved by commercially relevant heterotrophic organisms."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Engineering the highly productive cyanobacterium Synechococcus sp. PCC 11901\nAuthors: Victoria, A. J.; Selao, T. T.; Moreno-Cabezuelo, J. A.; Mills, L. A.; Gale, G. A.; Lea-Smith, D. J.; McCormick, A. J.\nScore: 11.0, Published: 2023-08-07 DOI: 10.1101/2023.08.04.552009\nSynechococcus sp. PCC 11901 (PCC 11901) is a fast-growing marine cyanobacterial strain that has a capacity for sustained biomass accumulation to very high cell densities, comparable to that achieved by commercially relevant heterotrophic organisms.",
  "keywords": [
    
  ],
  "articleBody": " Engineering the highly productive cyanobacterium Synechococcus sp. PCC 11901\nAuthors: Victoria, A. J.; Selao, T. T.; Moreno-Cabezuelo, J. A.; Mills, L. A.; Gale, G. A.; Lea-Smith, D. J.; McCormick, A. J.\nScore: 11.0, Published: 2023-08-07 DOI: 10.1101/2023.08.04.552009\nSynechococcus sp. PCC 11901 (PCC 11901) is a fast-growing marine cyanobacterial strain that has a capacity for sustained biomass accumulation to very high cell densities, comparable to that achieved by commercially relevant heterotrophic organisms. However, genetic tools to engineer PCC 11901 for biotechnology applications are limited. Here we describe a suite of tools based on the CyanoGate MoClo system to unlock the engineering potential of PCC 11901. First, we characterised neutral sites suitable for stable genomic integration that do not affect growth even at high cell densities. Second, we tested a suite of constitutive promoters, terminators, and inducible promoters including a 2,4-diacetylphloroglucinol (DAPG)-inducible PhlF repressor system, which has not previously been demonstrated in cyanobacteria and showed tight regulation and a 228-fold dynamic range of induction. Lastly, we developed a DAPG-inducible dCas9-based CRISPR interference (CRISPRi) system and a modular method to generate markerless mutants using CRISPR-Cas12a. Based on our findings, PCC 11901 is highly responsive to CRISPRi-based repression and showed high efficiencies for single insertion (31-81%) and multiplex double insertion (25%) genome editing with Cas12a. We envision that these tools will lay the foundations for the adoption of PCC 11901 as a robust model strain for engineering biology and green biotechnology.\nCharge-Patterned Disordered Peptides Tune Intracellular Phase Separation in Bacteria\nAuthors: Liao, J.; Yeong, V.; Obermeyer, A. C.\nScore: 4.3, Published: 2023-08-01 DOI: 10.1101/2023.08.01.551457\nSubcellular phase separated compartments, known as biomolecular condensates, play an important role in the spatiotemporal organization of cells. To understand the sequence-determinants of phase separation in bacteria, we engineered protein-based condensates in Escherichia coli by utilizing electrostatic interactions as the main driving force. Minimal cationic disordered peptides were used to supercharge negative, neutral, and positive globular model proteins, enabling their phase separation with anionic biomacromolecules in the cell. The phase behavior was governed by the interaction strength between the cationic proteins and anionic biopolymers in addition to the protein concentration. The interaction strength primarily depended on the overall net charge of the protein, but the distribution of charge between the globular and disordered domains also had an impact. Notably, the protein charge distribution between domains could tune mesoscale attributes such as the size, number, and subcellular localization of condensates within E. coli cells. The length and charge density of the disordered peptides had significant effects on protein expression levels, ultimately influencing the formation of condensates. Taken together, charge-patterned disordered peptides provide a platform for understanding the molecular grammar underlying phase separation in bacteria. HighlightsO_LIMinimal disordered cationic peptides of varying charge densities can promote protein phase separation in bacterial cells. C_LIO_LIProtein net charge and charge-patterning are distinct determinants of phase behavior. C_LIO_LIProtein charge distribution can be used to tune the size, number, position, and reversibility of condensates. C_LIO_LIMultiple proteins can be selectively recruited to synthetic condensates with disordered cationic peptides. C_LI\nDevelopment of a Quorum-Sensing Based Bacterial GMO for Targeted Treatment of Bovine Mastitis.\nAuthors: Dutta, A.; Chatterjee, S.; Borkar, S. P.; Das, S.; Ghosh, U.; Adhikary, A.\nScore: 3.4, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552207\nBovine mastitis, a prevalent and costly disease affecting dairy cows, severely impacts milk production, quality, and cow health and poses significant challenges to the dairy industry worldwide. Traditional treatment strategies relying on antibiotics face concerns of antibiotic resistance and residual antibiotic residues in dairy products. Therefore, there is an urgent need for innovative and sustainable approaches. Our study explores the efficacy of a GMO-based antibiotic-free treatment in fighting Bovine mastitis. We use the bioengineered defensin Nisin PV as the antimicrobial to specifically target the causative pathogen Staphylococcus aureus. To disrupt the biofilm formation, we have leveraged the potency of DNaseI. By hacking the quorum sensing technique of Staphylococcus aureus, our Genetically modified bacteria sense the presence of pathogens and initiate their eradication by producing Nisin PV and DNaseI. Additionally, a lysis-based kill switch is incorporated to ensure the release of the therapeutic components. The findings of this research have the potential to revolutionize bovine mastitis management, offering a sustainable and effective alternative to antibiotic-based treatments. Implementation of bioengineered defensins could reduce disease incidence, minimize economic losses, and promote responsible antimicrobial stewardship in the dairy industry. This innovative approach contributes to the development of sustainable solutions for safeguarding animal health, enhancing milk quality, and preserving human health.\nVisible light-induced specific protein reaction delineates early stages of cell adhesion\nAuthors: Rahikainen, R.; Vester, S. K.; Turkki, P.; Janosko, C.; Deiters, A.; Hytonen, V. P.; Howarth, M.\nScore: 7.7, Published: 2023-07-22 DOI: 10.1101/2023.07.21.549850\nLight is well established for control of bond breakage, but not for control of specific bond formation in complex environments. We previously engineered diffusion-limited reactivity of SpyTag003 peptide with its protein partner SpyCatcher003 through spontaneous transamidation. This system enables precise and irreversible assembly of biological building blocks, with applications from biomaterials to vaccines. Here, we establish a system for rapid control of this amide bond formation with visible light. We have generated a caged SpyCatcher003, which allows light triggering of covalent bond formation to SpyTag003 in mammalian cells. Photocaging is achieved through site-specific incorporation of an unnatural coumarin-lysine at the reactive site of SpyCatcher003. We showed uniform specific reaction in cell lysate upon light activation. We then used the spatiotemporal precision of a 405 nm confocal laser for uncaging in seconds, probing the earliest events in mechanotransduction by talin, the key force sensor between the cytoskeleton and extracellular matrix. Reconstituting talin induced rapid biphasic extension of lamellipodia, revealing the kinetics of talin-regulated cell spreading and polarization. Thereafter we determined the hierarchy of recruitment of key components for cell adhesion. Precise control over site-specific protein reaction with visible light creates diverse opportunities for cell biology and nanoassembly.\nChanges in coding and efficiency through modular modifications to a One Pot PURE system for in vitro transcription \u0026 translation\nAuthors: Ngo, P. H. T.; Ishida, S.; Busogi, B. B.; Ledesma, M.; Do, H.; Kar, S.; Ellington, A. H. T.\nScore: 4.5, Published: 2023-07-28 DOI: 10.1101/2023.07.28.550900\nThe incorporation of unnatural amino acids are attractive methods for improving or bringing new and novel functions in peptides and proteins. Cell-free protein synthesis using the Protein Synthesis Using Recombinant Elements (PURE) system is an attractive platform for efficient unnatural amino acid incorporation. In this work, we further adapted and modified the One Pot PURE for a robust and modular system of enzymatic single site-specific of unnatural amino acid. We demonstrated the flexibility of this system through the introduction of two orthogonal aminoacyl tRNA synthetases and the suppression of two distinctive stop codons.\nMapping immunological and host receptor binding determinants of SARS-CoV spike protein utilizing the Qubevirus platform\nAuthors: Bopda Waffo, A.\nScore: 1.5, Published: 2023-07-28 DOI: 10.1101/2023.07.27.550841\nThe motifs involved in tropism and immunological interactions of SARS-CoV spike (S) protein were investigated utilizing the Qubevirus platform. We showed that separately, 14 overlapping peptide fragments representing the S protein (F1-14 of 100 residues each) could be inserted into the C-terminus of A1 on recombinant Qubevirus without affecting its viability. Additionally, recombinant phage expression resulted in the surface exposure of different engineered fragments in an accessible manner. The F6 from S425-525, was found to contain the binding determinant of the recombinant human angiotensin converting enzyme 2 (rhACE2), with the shortest active binding motif situated between residues S437-492. Upstream, another fragment, F7, containing an overlapping portion of F6 would not bind to rhACE2, confirming not just only that residues were linear but equally also the appropriate structural orientation of F6 upon the Qubevirus. The F6 (S441-460) and other inserts, including F7/F8 (S601-620) and F10 (S781-800), were demonstrated to contain important immunological determinants through recognition and binding of S protein specific (anti-S) antibodies. An engineered chimeric insert bearing the fusion of all three anti-S reactive epitopes, improved substantially the recognition and binding to their cognate antibodies. These results provide insights into humoral immune relevant epitopes and tropism characteristics of the S protein with implications for the development of subunit vaccines or other biologics against SARS-CoV. SignificanceMapping epitopes within the receptor binding domains of viruses which are essential for viral tropism is critical for developing antiviral agents and subunit vaccines. In this study we have engineered the surface of Qubevirus to display a peptide library derived from the SARS-CoV S protein. In biopanning with S protein antibodies, we have identified three peptide fragments (EP1, EP2 and EP3) which reacted selectively with antibodies specific to the S protein. We demonstrated that all recombinant phage displayed peptide fragments both individually and as chimera exposed important immunological epitopes to their cognate antibodies. A peptide fragment F6 situated at S425-525, was found containing the binding determinant of the recombinant human angiotensin converting enzyme 2 (rhACE2), with the shortest active binding motif situated between residues S437-492. The platform is rapidly to identify epitopes and receptor binding sites within viral receptors found in target host cell. Thus, this platform holds great significance.\n",
  "wordCount" : "1497",
  "inLanguage": "en",
  "datePublished": "2023-08-09T10:39:33Z",
  "dateModified": "2023-08-09T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on August 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552009">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552009" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552009">
        <p class="paperTitle">Engineering the highly productive cyanobacterium Synechococcus sp. PCC 11901</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552009" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552009" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Victoria, A. J.; Selao, T. T.; Moreno-Cabezuelo, J. A.; Mills, L. A.; Gale, G. A.; Lea-Smith, D. J.; McCormick, A. J.</p>
        <p class="info">Score: 11.0, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552009' target='https://doi.org/10.1101/2023.08.04.552009'> 10.1101/2023.08.04.552009</a></p>
        <p class="abstract">Synechococcus sp. PCC 11901 (PCC 11901) is a fast-growing marine cyanobacterial strain that has a capacity for sustained biomass accumulation to very high cell densities, comparable to that achieved by commercially relevant heterotrophic organisms. However, genetic tools to engineer PCC 11901 for biotechnology applications are limited. Here we describe a suite of tools based on the CyanoGate MoClo system to unlock the engineering potential of PCC 11901. First, we characterised neutral sites suitable for stable genomic integration that do not affect growth even at high cell densities. Second, we tested a suite of constitutive promoters, terminators, and inducible promoters including a 2,4-diacetylphloroglucinol (DAPG)-inducible PhlF repressor system, which has not previously been demonstrated in cyanobacteria and showed tight regulation and a 228-fold dynamic range of induction. Lastly, we developed a DAPG-inducible dCas9-based CRISPR interference (CRISPRi) system and a modular method to generate markerless mutants using CRISPR-Cas12a. Based on our findings, PCC 11901 is highly responsive to CRISPRi-based repression and showed high efficiencies for single insertion (31-81%) and multiplex double insertion (25%) genome editing with Cas12a. We envision that these tools will lay the foundations for the adoption of PCC 11901 as a robust model strain for engineering biology and green biotechnology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551457">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551457" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551457">
        <p class="paperTitle">Charge-Patterned Disordered Peptides Tune Intracellular Phase Separation in Bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551457" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551457" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liao, J.; Yeong, V.; Obermeyer, A. C.</p>
        <p class="info">Score: 4.3, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551457' target='https://doi.org/10.1101/2023.08.01.551457'> 10.1101/2023.08.01.551457</a></p>
        <p class="abstract">Subcellular phase separated compartments, known as biomolecular condensates, play an important role in the spatiotemporal organization of cells. To understand the sequence-determinants of phase separation in bacteria, we engineered protein-based condensates in Escherichia coli by utilizing electrostatic interactions as the main driving force. Minimal cationic disordered peptides were used to supercharge negative, neutral, and positive globular model proteins, enabling their phase separation with anionic biomacromolecules in the cell. The phase behavior was governed by the interaction strength between the cationic proteins and anionic biopolymers in addition to the protein concentration. The interaction strength primarily depended on the overall net charge of the protein, but the distribution of charge between the globular and disordered domains also had an impact. Notably, the protein charge distribution between domains could tune mesoscale attributes such as the size, number, and subcellular localization of condensates within E. coli cells. The length and charge density of the disordered peptides had significant effects on protein expression levels, ultimately influencing the formation of condensates. Taken together, charge-patterned disordered peptides provide a platform for understanding the molecular grammar underlying phase separation in bacteria.

HighlightsO_LIMinimal disordered cationic peptides of varying charge densities can promote protein phase separation in bacterial cells.
C_LIO_LIProtein net charge and charge-patterning are distinct determinants of phase behavior.
C_LIO_LIProtein charge distribution can be used to tune the size, number, position, and reversibility of condensates.
C_LIO_LIMultiple proteins can be selectively recruited to synthetic condensates with disordered cationic peptides.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552207">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552207" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552207">
        <p class="paperTitle">Development of a Quorum-Sensing Based Bacterial GMO for Targeted Treatment of Bovine Mastitis.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552207" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552207" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dutta, A.; Chatterjee, S.; Borkar, S. P.; Das, S.; Ghosh, U.; Adhikary, A.</p>
        <p class="info">Score: 3.4, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552207' target='https://doi.org/10.1101/2023.08.07.552207'> 10.1101/2023.08.07.552207</a></p>
        <p class="abstract">Bovine mastitis, a prevalent and costly disease affecting dairy cows, severely impacts milk production, quality, and cow health and poses significant challenges to the dairy industry worldwide. Traditional treatment strategies relying on antibiotics face concerns of antibiotic resistance and residual antibiotic residues in dairy products. Therefore, there is an urgent need for innovative and sustainable approaches. Our study explores the efficacy of a GMO-based antibiotic-free treatment in fighting Bovine mastitis. We use the bioengineered defensin Nisin PV as the antimicrobial to specifically target the causative pathogen Staphylococcus aureus. To disrupt the biofilm formation, we have leveraged the potency of DNaseI. By hacking the quorum sensing technique of Staphylococcus aureus, our Genetically modified bacteria sense the presence of pathogens and initiate their eradication by producing Nisin PV and DNaseI. Additionally, a lysis-based kill switch is incorporated to ensure the release of the therapeutic components. The findings of this research have the potential to revolutionize bovine mastitis management, offering a sustainable and effective alternative to antibiotic-based treatments. Implementation of bioengineered defensins could reduce disease incidence, minimize economic losses, and promote responsible antimicrobial stewardship in the dairy industry. This innovative approach contributes to the development of sustainable solutions for safeguarding animal health, enhancing milk quality, and preserving human health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.549850">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.549850" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.549850">
        <p class="paperTitle">Visible light-induced specific protein reaction delineates early stages of cell adhesion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.549850" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.549850" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rahikainen, R.; Vester, S. K.; Turkki, P.; Janosko, C.; Deiters, A.; Hytonen, V. P.; Howarth, M.</p>
        <p class="info">Score: 7.7, Published: 2023-07-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.549850' target='https://doi.org/10.1101/2023.07.21.549850'> 10.1101/2023.07.21.549850</a></p>
        <p class="abstract">Light is well established for control of bond breakage, but not for control of specific bond formation in complex environments. We previously engineered diffusion-limited reactivity of SpyTag003 peptide with its protein partner SpyCatcher003 through spontaneous transamidation. This system enables precise and irreversible assembly of biological building blocks, with applications from biomaterials to vaccines. Here, we establish a system for rapid control of this amide bond formation with visible light. We have generated a caged SpyCatcher003, which allows light triggering of covalent bond formation to SpyTag003 in mammalian cells. Photocaging is achieved through site-specific incorporation of an unnatural coumarin-lysine at the reactive site of SpyCatcher003. We showed uniform specific reaction in cell lysate upon light activation. We then used the spatiotemporal precision of a 405 nm confocal laser for uncaging in seconds, probing the earliest events in mechanotransduction by talin, the key force sensor between the cytoskeleton and extracellular matrix. Reconstituting talin induced rapid biphasic extension of lamellipodia, revealing the kinetics of talin-regulated cell spreading and polarization. Thereafter we determined the hierarchy of recruitment of key components for cell adhesion. Precise control over site-specific protein reaction with visible light creates diverse opportunities for cell biology and nanoassembly.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550900">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550900" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550900">
        <p class="paperTitle">Changes in coding and efficiency through modular modifications to a One Pot PURE system for in vitro transcription &amp; translation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550900" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550900" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ngo, P. H. T.; Ishida, S.; Busogi, B. B.; Ledesma, M.; Do, H.; Kar, S.; Ellington, A. H. T.</p>
        <p class="info">Score: 4.5, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550900' target='https://doi.org/10.1101/2023.07.28.550900'> 10.1101/2023.07.28.550900</a></p>
        <p class="abstract">The incorporation of unnatural amino acids are attractive methods for improving or bringing new and novel functions in peptides and proteins. Cell-free protein synthesis using the Protein Synthesis Using Recombinant Elements (PURE) system is an attractive platform for efficient unnatural amino acid incorporation. In this work, we further adapted and modified the One Pot PURE for a robust and modular system of enzymatic single site-specific of unnatural amino acid. We demonstrated the flexibility of this system through the introduction of two orthogonal aminoacyl tRNA synthetases and the suppression of two distinctive stop codons.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550841">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550841" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550841">
        <p class="paperTitle">Mapping immunological and host receptor binding determinants of SARS-CoV spike protein utilizing the Qubevirus platform</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550841" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550841" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bopda Waffo, A.</p>
        <p class="info">Score: 1.5, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550841' target='https://doi.org/10.1101/2023.07.27.550841'> 10.1101/2023.07.27.550841</a></p>
        <p class="abstract">The motifs involved in tropism and immunological interactions of SARS-CoV spike (S) protein were investigated utilizing the Qubevirus platform. We showed that separately, 14 overlapping peptide fragments representing the S protein (F1-14 of 100 residues each) could be inserted into the C-terminus of A1 on recombinant Qubevirus without affecting its viability. Additionally, recombinant phage expression resulted in the surface exposure of different engineered fragments in an accessible manner. The F6 from S425-525, was found to contain the binding determinant of the recombinant human angiotensin converting enzyme 2 (rhACE2), with the shortest active binding motif situated between residues S437-492. Upstream, another fragment, F7, containing an overlapping portion of F6 would not bind to rhACE2, confirming not just only that residues were linear but equally also the appropriate structural orientation of F6 upon the Qubevirus. The F6 (S441-460) and other inserts, including F7/F8 (S601-620) and F10 (S781-800), were demonstrated to contain important immunological determinants through recognition and binding of S protein specific (anti-S) antibodies. An engineered chimeric insert bearing the fusion of all three anti-S reactive epitopes, improved substantially the recognition and binding to their cognate antibodies. These results provide insights into humoral immune relevant epitopes and tropism characteristics of the S protein with implications for the development of subunit vaccines or other biologics against SARS-CoV.

SignificanceMapping epitopes within the receptor binding domains of viruses which are essential for viral tropism is critical for developing antiviral agents and subunit vaccines. In this study we have engineered the surface of Qubevirus to display a peptide library derived from the SARS-CoV S protein. In biopanning with S protein antibodies, we have identified three peptide fragments (EP1, EP2 and EP3) which reacted selectively with antibodies specific to the S protein. We demonstrated that all recombinant phage displayed peptide fragments both individually and as chimera exposed important immunological epitopes to their cognate antibodies. A peptide fragment F6 situated at S425-525, was found containing the binding determinant of the recombinant human angiotensin converting enzyme 2 (rhACE2), with the shortest active binding motif situated between residues S437-492. The platform is rapidly to identify epitopes and receptor binding sites within viral receptors found in target host cell. Thus, this platform holds great significance.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
