<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Emergent digital bio-computation through spatial diffusion and engineered bacteria
Authors: Fedorec, A. J. H.; Treloar, N. J.; Wen, K. Y.; Dekker, L.; Ong, Q. H.; Jurkeviciute, G.; Lyu, E.; Rutter, J.; Rosa, L.; Zaikin, A.; Barnes, C. P.
Score: 24.2, Published: 2023-07-11 DOI: 10.1101/2023.07.07.548081
Building computationally capable biological systems has long been an aim of synthetic biology. The potential utility of biocomputing devices ranges from biosafety and environmental applications to diagnosis and personalised medicine.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Emergent digital bio-computation through spatial diffusion and engineered bacteria
Authors: Fedorec, A. J. H.; Treloar, N. J.; Wen, K. Y.; Dekker, L.; Ong, Q. H.; Jurkeviciute, G.; Lyu, E.; Rutter, J.; Rosa, L.; Zaikin, A.; Barnes, C. P.
Score: 24.2, Published: 2023-07-11 DOI: 10.1101/2023.07.07.548081
Building computationally capable biological systems has long been an aim of synthetic biology. The potential utility of biocomputing devices ranges from biosafety and environmental applications to diagnosis and personalised medicine." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-19T10:38:24+00:00" />
<meta property="article:modified_time" content="2023-07-19T10:38:24+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Emergent digital bio-computation through spatial diffusion and engineered bacteria
Authors: Fedorec, A. J. H.; Treloar, N. J.; Wen, K. Y.; Dekker, L.; Ong, Q. H.; Jurkeviciute, G.; Lyu, E.; Rutter, J.; Rosa, L.; Zaikin, A.; Barnes, C. P.
Score: 24.2, Published: 2023-07-11 DOI: 10.1101/2023.07.07.548081
Building computationally capable biological systems has long been an aim of synthetic biology. The potential utility of biocomputing devices ranges from biosafety and environmental applications to diagnosis and personalised medicine."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Emergent digital bio-computation through spatial diffusion and engineered bacteria\nAuthors: Fedorec, A. J. H.; Treloar, N. J.; Wen, K. Y.; Dekker, L.; Ong, Q. H.; Jurkeviciute, G.; Lyu, E.; Rutter, J.; Rosa, L.; Zaikin, A.; Barnes, C. P.\nScore: 24.2, Published: 2023-07-11 DOI: 10.1101/2023.07.07.548081\nBuilding computationally capable biological systems has long been an aim of synthetic biology. The potential utility of biocomputing devices ranges from biosafety and environmental applications to diagnosis and personalised medicine.",
  "keywords": [
    
  ],
  "articleBody": " Emergent digital bio-computation through spatial diffusion and engineered bacteria\nAuthors: Fedorec, A. J. H.; Treloar, N. J.; Wen, K. Y.; Dekker, L.; Ong, Q. H.; Jurkeviciute, G.; Lyu, E.; Rutter, J.; Rosa, L.; Zaikin, A.; Barnes, C. P.\nScore: 24.2, Published: 2023-07-11 DOI: 10.1101/2023.07.07.548081\nBuilding computationally capable biological systems has long been an aim of synthetic biology. The potential utility of biocomputing devices ranges from biosafety and environmental applications to diagnosis and personalised medicine. Here we present work for the design of bacterial computers which use spatial patterning to process information. Our computers are composed of bacterial colonies which, inspired by patterning in embryo development, receive information in the form of diffusible morphogen-like signals. A computation is encoded by the physical locations of the input sources of morphogen and the output receiver colonies. We demonstrate, mathematically and with engineered Escherichia coli, the simple digital logic capability of single bacterial colonies and show how additional colonies are required to build complex functions. Inspired by electronic design automation, an algorithm for designing optimal spatial circuits computing two-level digital functions is presented. This enhances the capability of our system to complex digital functions without increasing the biological complexity. We extend our experimental system to incorporate biosensing colonies as morphogen sources, demonstrating how a diagnostic device might be constructed. Our approach will open up new ways to perform biological computation, with applications in bioengineering, biomaterials and biosensing. Ultimately, these computational bacterial communities will help us explore information processing in natural biological systems.\nGenerative and predictive neural networks for the design of functional RNA molecules\nAuthors: Riley, A. T.; Robson, J. M.; Green, A. A.\nScore: 13.4, Published: 2023-07-14 DOI: 10.1101/2023.07.14.549043\nRNA is a remarkably versatile molecule that has been engineered for applications in therapeutics, diagnostics, and in vivo information-processing systems. However, the complex relationship between the sequence and structural properties of an RNA molecule and its ability to perform specific functions often necessitates extensive experimental screening of candidate sequences. Here we present a generalized neural network architecture that utilizes the sequence and structure of RNA molecules (SANDSTORM) to inform functional predictions. We demonstrate that this approach achieves state-of-the-art performance across several distinct RNA prediction tasks, while learning interpretable abstractions of RNA secondary structure. We paired these predictive models with generative adversarial RNA design networks (GARDN), allowing the generative modelling of novel mRNA 5 untranslated regions and toehold switch riboregulators exhibiting a predetermined fitness. This approach enabled the design of novel toehold switches with a 43-fold increase in experimentally characterized dynamic range compared to those designed using classic thermodynamic algorithms. SANDSTORM and GARDN thus represent powerful new predictive and generative tools for the development of diagnostic and therapeutic RNA molecules with improved function.\nHighly efficient cellular expression of circular mRNA enables prolonged protein expression\nAuthors: Unti, M. J.; Jaffrey, S. R.\nScore: 11.6, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548538\nA major problem with mRNA therapeutics is the limited duration of protein expression due to the short half-life of mRNA. New approaches for generating highly stable circular mRNA in vitro have allowed increased duration of protein expression. However, it remains difficult to genetically encode circular mRNAs in mammalian cells, which limits the use of circular mRNA in cell-derived therapeutics. Here we describe the adaptation of the Tornado (Twister-optimized RNA for durable overexpression) system to achieve in-cell synthesis of circular mRNAs. We identify the promoter and internal ribosomal entry site (IRES) that result in high levels of protein expression in cells. We then show that these circular mRNAs can be packaged into virus-like particles (VLPs) thus enabling prolonged protein expression. Overall, these data describe a platform for synthesis of circular mRNAs and how these circular mRNAs can markedly enhance the ability of VLPs to function as a mRNA delivery tool.\nAn expanded genetic toolbox to accelerate the creation of Acholeplasma laidlawii driven by synthetic genomes\nAuthors: Nucifora, D. P.; Mehta, N. D.; Giguere, D. J.; Karas, B. J.\nScore: 29.4, Published: 2023-07-12 DOI: 10.1101/2022.09.21.508766\nAssembling synthetic bacterial genomes in yeast and genome transplantation has enabled an unmatched level of bacterial strain engineering, giving rise to cells with minimal and chemically synthetic genomes. However, this technology is currently limited to members of the Spiroplasma phylogenetic group, mostly Mycoplasmas, within the Mollicute class. Here, we propose new genetic tools for developing these technologies for Acholeplasma laidlawii, which is phylogenetically distant from Mycoplasmas and, unlike most Mollicutes, uses a standard genetic code. We first investigated a donor-recipient relationship between two A. laidlawii strains through whole-genome sequencing. We then created multi-host shuttle plasmids and used them to optimize an electroporation protocol. We also demonstrated the use of evolution to create superior strains for DNA uptake via electroporation. For genome transplantation, we selected A. laidlawii 8195 as the recipient strain and created a PG-8A donor strain by inserting a Tn5 transposon carrying a tetracycline resistance gene. The tools presented here will improve Acholeplasma research and accelerate the effort toward creating A. laidlawii strains driven by synthetic genomes. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=102 SRC=\"FIGDIR/small/508766v2_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (31K): org.highwire.dtl.DTLVardef@1653cfeorg.highwire.dtl.DTLVardef@1874b7dorg.highwire.dtl.DTLVardef@16cff67org.highwire.dtl.DTLVardef@f3e5c6_HPS_FORMAT_FIGEXP M_FIG C_FIG\nA Platform for Controlled Template-Independent Enzymatic Synthesis of RNA Oligonucleotides and Therapeutics\nAuthors: Wiegand, D. J.; Rittichier, J.; Meyer, E.; Lee, H.; Conway, N. J.; Ahlstedt, D.; Yurtsever, Z.; Rainone, D.; Kuru, E.; Church, G.\nScore: 14.9, Published: 2023-06-29 DOI: 10.1101/2023.06.29.547106\nTherapeutic RNA oligonucleotides have shown tremendous potential to manage and treat disease, yet current manufacturing methods cannot deliver on this promise. Here, we report the development and optimization of a novel, aqueous-based, template-independent enzymatic RNA oligonucleotide synthesis platform as an alternative to traditional chemical methodologies. Our platform is made possible by reversible terminator nucleoside triphosphates and an enzyme capable of their incorporation. We show that many common therapeutic RNA modifications are compatible with our process and demonstrate the enzymatic synthesis of natural and modified oligonucleotides in both liquid and solid phases. Our platform offers many unique advantages over chemical synthesis, including the realization of a more sustainable process to produce therapeutic RNA oligonucleotides. One-Sentence SummaryAn enzyme and novel nucleotide building blocks are used to synthesize RNA oligonucleotides template independently under aqueous conditions.\nxTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein\nAuthors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Gong, J.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.\nScore: 83.1, Published: 2023-07-14 DOI: 10.1101/2023.07.05.547496\nProtein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently. This paper proposes a unified protein language model, xTrimoPGLM, to address these two types of tasks simultaneously through an innovative pre-training framework. Our key technical contribution is an exploration of the compatibility and the potential for joint optimization of the two types of objectives, which has led to a strategy for training xTrimoPGLM at an unprecedented scale of 100 billion parameters and 1 trillion training tokens. Our extensive experiments reveal that xTrimoPGLM significantly outperforms other advanced baselines in diverse protein understanding tasks (13 out of 15 tasks across four categories) and generates novel protein sequences which are structurally similar to natural ones. Furthermore, using the same xTrimoPGLM framework, we train an antibody-specific model (xTrimoPGLM-Ab) using 1 billion parameters. This model set a new record in predicting antibody naturalness and structures, both essential to the field of antibody-based drug design, and demonstrated a significantly faster inference speed than AlphaFold2. These results highlight the substantial capability and versatility of xTrimoPGLM in understanding and generating protein sequences.\nEngineered sensor bacteria evolve master-level gameplay through accelerated adaptation\nAuthors: Prakash, S.; Racovita, A.; Varela, C.; Walsh, M.; Galizi, R.; Isalan, M.; Jaramillo, A.\nScore: 259.8, Published: 2023-07-11 DOI: 10.1101/2022.04.22.489191\nGene circuits enable cells to make decisions by controlling the expression of genes in reaction to specific environmental factors1. These circuits can be designed to encode logical operations2-7, but implementation of more complex algorithms has proved more challenging. Directed evolution optimizes gene circuits8 without the need for design knowledge9, but adjusting multiple genes and conditions10 in genotype searches poses challenges11. Here we show a multicellular sensor system, AdaptoCells, in Escherichia coli, that can evolve complex behavior through an accelerated adaptation to chemical environments. AdaptoCells recognize chemical patterns and act as a decision-making system. Using an iterative improvement method, we demonstrate that the AdaptoCells can evolve to achieve mastery in the game of tic-tac-toe, demonstrating an unprecedented level of complexity for engineered living cells. We provide an effective and straightforward way to encode complexity in gene circuits, allowing for fast adaptation in response to dynamic environments and leading to optimal decisions.\nModular and integrative activity reporters enhance biochemical studies in the yeast ER.\nAuthors: Martinusen, S. G.; Slaton, E. W.; Nelson, S. E.; Pulgar, M. A.; Besu, J. T.; Simas, C. F.; Denard, C. A.\nScore: 4.5, Published: 2023-07-12 DOI: 10.1101/2023.07.12.548713\nThe yeast endoplasmic reticulum sequestration and screening (YESS) system is a generalizable platform that has become highly useful to investigate post-translational modification enzymes (PTM-enzymes). This system enables researchers to profile and engineer the activity and substrate specificity of PTM-enzymes and to discover inhibitor-resistant enzyme mutants. In this study, we expand the capabilities of YESS by transferring its functional components to integrative plasmids. The YESS integrative system yields uniform protein expression and protease activities in various configurations, allows one to integrate activity reporters at two independent loci and to split the system between integrative and centromeric plasmids. We characterize these integrative reporters with two viral proteases, Tobacco etch virus (TEVp) and 3-chymotrypsin like protease (3CLpro), in terms of coefficient of variance, signal-to-noise ratio and fold-activation. Overall, we provide a framework for chromosomal-based studies that is modular, enabling rigorous high-throughput assays of PTM-enzymes in yeast.\nLactoSpanks: a collection of IPTG inducible promoters for the commensal lactic acid bacteria Lactobacillus gasseri.\nAuthors: Fristot, E.; Cambray, G.; Bonnet, J.\nScore: 3.7, Published: 2023-07-13 DOI: 10.1101/2023.07.13.548755\nLactic acid bacteria (LAB) are important for many biotechnological applications, such as bioproduction and engineered probiotics for therapy. Inducible promoters are key gene expression control elements, yet those available in LAB are mainly based on bacteriocin systems and have many drawbacks, including large gene clusters, costly inducer peptides and little portability to in vivo settings. Using Lactobacillus gasseri, a model commensal bacteria from the human gut, we report the engineering of Lactospanks promoters (Pls), a collection of variable strength inducible promoters controlled by the LacI repressor from B. subtilis and induced by isopropyl {beta}-D-1-thiogalactopyranoside (IPTG). We first show that the Phyper-spank promoter from Bacillus subtilis is functional in L. gasseri, albeit with substantial leakage. We then construct and screen a semi-rational library of Phyper-spank variants to select a set of four IPTG-inducible promoters that span a range of expression levels and exhibit reduced leakages and operational dynamic ranges (from ca. 9 to 28 fold-change). With their low genetic footprint and simplicity of use, Lactospanks will support many applications in L. gasseri, and potentially other lactic acid and gram-positive bacteria.\nA SEVA-based, CRISPR-Cas3-assisted genome engineering approach for Pseudomonas with efficient vector curing\nAuthors: Lammens, E.-M.; Volke, D. C.; Schroven, K.; Voet, M.; Kerremans, A.; Lavigne, R.; Hendrix, H.\nScore: 2.8, Published: 2023-06-29 DOI: 10.1101/2023.06.29.547033\nThe development of CRISPR-Cas-based engineering technologies has revolutionized the microbial biotechnology field. Over the years, the Class II Type II CRISPR-Cas9 system has become the gold standard for genome editing in many bacterial hosts. However, the Cas9 system does not allow efficient genomic integration in Pseudomonas putida, an emerging Synthetic Biology host, without the assistance of lambda-Red recombineering. In this work, we utilize the alternative Class I Type I-C CRISPR-Cas3 system from Pseudomonas aeruginosa as a highly-efficient genome editing tool for P. putida and P. aeruginosa. This system consists of two vectors, one encoding the Cas genes, CRISPR array and targeting spacer, and a second SEVA-vector, containing the homologous repair template. Both vectors are Golden Gate compatible for rapid cloning and are available with multiple antibiotic markers, for application in various Gram-negative hosts and different designs. By employing this Cas3 system, we successfully integrated an 820-bp cassette in the genome of P. putida and performed several genomic deletions in P. aeruginosa within four days, with an efficiency of \u003e83% for both hosts. Moreover, by introducing a universal self-targeting spacer, the Cas3 system rapidly cures all helper vectors, including itself, from the host strain in a matter of days. As such, this system constitutes a valuable engineering tool for Pseudomonas, to complement the existing range of Cas9-based editing methods and facilitates genomic engineering efforts of this important genus. ImportanceThe CRISPR-Cas3 editing system as presented here facilitates the creation of genomic alterations in P. putida and P. aeruginosa in a straightforward manner. By providing the Cas3 system as a vector set with Golden Gate compatibility and different antibiotic markers, as well as by employing the established SEVA vector set to provide the homology repair template, this system is flexible and can readily be ported to a multitude of Gram-negative hosts. Besides genome editing, the Cas3 system can also be used as an effective and universal tool for vector curing. This is achieved by introducing a spacer that targets the oriT, present on the majority of established (SEVA) vectors. Based on this, the Cas3 system efficiently removes up to three vectors in only a few days. As such, this curing approach may also benefit other genomic engineering methods or remove naturally-occurring plasmids from bacteria.\n",
  "wordCount" : "2232",
  "inLanguage": "en",
  "datePublished": "2023-07-19T10:38:24Z",
  "dateModified": "2023-07-19T10:38:24Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on July 19, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548081">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548081" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548081">
        <p class="paperTitle">Emergent digital bio-computation through spatial diffusion and engineered bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548081" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548081" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fedorec, A. J. H.; Treloar, N. J.; Wen, K. Y.; Dekker, L.; Ong, Q. H.; Jurkeviciute, G.; Lyu, E.; Rutter, J.; Rosa, L.; Zaikin, A.; Barnes, C. P.</p>
        <p class="info">Score: 24.2, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548081' target='https://doi.org/10.1101/2023.07.07.548081'> 10.1101/2023.07.07.548081</a></p>
        <p class="abstract">Building computationally capable biological systems has long been an aim of synthetic biology. The potential utility of biocomputing devices ranges from biosafety and environmental applications to diagnosis and personalised medicine. Here we present work for the design of bacterial computers which use spatial patterning to process information. Our computers are composed of bacterial colonies which, inspired by patterning in embryo development, receive information in the form of diffusible morphogen-like signals. A computation is encoded by the physical locations of the input sources of morphogen and the output receiver colonies. We demonstrate, mathematically and with engineered Escherichia coli, the simple digital logic capability of single bacterial colonies and show how additional colonies are required to build complex functions. Inspired by electronic design automation, an algorithm for designing optimal spatial circuits computing two-level digital functions is presented. This enhances the capability of our system to complex digital functions without increasing the biological complexity. We extend our experimental system to incorporate biosensing colonies as morphogen sources, demonstrating how a diagnostic device might be constructed. Our approach will open up new ways to perform biological computation, with applications in bioengineering, biomaterials and biosensing. Ultimately, these computational bacterial communities will help us explore information processing in natural biological systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.549043">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.549043" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.549043">
        <p class="paperTitle">Generative and predictive neural networks for the design of functional RNA molecules</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.549043" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.549043" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Riley, A. T.; Robson, J. M.; Green, A. A.</p>
        <p class="info">Score: 13.4, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.549043' target='https://doi.org/10.1101/2023.07.14.549043'> 10.1101/2023.07.14.549043</a></p>
        <p class="abstract">RNA is a remarkably versatile molecule that has been engineered for applications in therapeutics, diagnostics, and in vivo information-processing systems. However, the complex relationship between the sequence and structural properties of an RNA molecule and its ability to perform specific functions often necessitates extensive experimental screening of candidate sequences. Here we present a generalized neural network architecture that utilizes the sequence and structure of RNA molecules (SANDSTORM) to inform functional predictions. We demonstrate that this approach achieves state-of-the-art performance across several distinct RNA prediction tasks, while learning interpretable abstractions of RNA secondary structure. We paired these predictive models with generative adversarial RNA design networks (GARDN), allowing the generative modelling of novel mRNA 5 untranslated regions and toehold switch riboregulators exhibiting a predetermined fitness. This approach enabled the design of novel toehold switches with a 43-fold increase in experimentally characterized dynamic range compared to those designed using classic thermodynamic algorithms. SANDSTORM and GARDN thus represent powerful new predictive and generative tools for the development of diagnostic and therapeutic RNA molecules with improved function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548538">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548538" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548538">
        <p class="paperTitle">Highly efficient cellular expression of circular mRNA enables prolonged protein expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548538" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548538" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Unti, M. J.; Jaffrey, S. R.</p>
        <p class="info">Score: 11.6, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548538' target='https://doi.org/10.1101/2023.07.11.548538'> 10.1101/2023.07.11.548538</a></p>
        <p class="abstract">A major problem with mRNA therapeutics is the limited duration of protein expression due to the short half-life of mRNA. New approaches for generating highly stable circular mRNA in vitro have allowed increased duration of protein expression. However, it remains difficult to genetically encode circular mRNAs in mammalian cells, which limits the use of circular mRNA in cell-derived therapeutics. Here we describe the adaptation of the Tornado (Twister-optimized RNA for durable overexpression) system to achieve in-cell synthesis of circular mRNAs. We identify the promoter and internal ribosomal entry site (IRES) that result in high levels of protein expression in cells. We then show that these circular mRNAs can be packaged into virus-like particles (VLPs) thus enabling prolonged protein expression. Overall, these data describe a platform for synthesis of circular mRNAs and how these circular mRNAs can markedly enhance the ability of VLPs to function as a mRNA delivery tool.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.09.21.508766">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.09.21.508766" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.09.21.508766">
        <p class="paperTitle">An expanded genetic toolbox to accelerate the creation of Acholeplasma laidlawii driven by synthetic genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.09.21.508766" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.09.21.508766" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nucifora, D. P.; Mehta, N. D.; Giguere, D. J.; Karas, B. J.</p>
        <p class="info">Score: 29.4, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.09.21.508766' target='https://doi.org/10.1101/2022.09.21.508766'> 10.1101/2022.09.21.508766</a></p>
        <p class="abstract">Assembling synthetic bacterial genomes in yeast and genome transplantation has enabled an unmatched level of bacterial strain engineering, giving rise to cells with minimal and chemically synthetic genomes. However, this technology is currently limited to members of the Spiroplasma phylogenetic group, mostly Mycoplasmas, within the Mollicute class. Here, we propose new genetic tools for developing these technologies for Acholeplasma laidlawii, which is phylogenetically distant from Mycoplasmas and, unlike most Mollicutes, uses a standard genetic code. We first investigated a donor-recipient relationship between two A. laidlawii strains through whole-genome sequencing. We then created multi-host shuttle plasmids and used them to optimize an electroporation protocol. We also demonstrated the use of evolution to create superior strains for DNA uptake via electroporation. For genome transplantation, we selected A. laidlawii 8195 as the recipient strain and created a PG-8A donor strain by inserting a Tn5 transposon carrying a tetracycline resistance gene. The tools presented here will improve Acholeplasma research and accelerate the effort toward creating A. laidlawii strains driven by synthetic genomes.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=102 SRC=&#34;FIGDIR/small/508766v2_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (31K):
org.highwire.dtl.DTLVardef@1653cfeorg.highwire.dtl.DTLVardef@1874b7dorg.highwire.dtl.DTLVardef@16cff67org.highwire.dtl.DTLVardef@f3e5c6_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547106">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547106" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547106">
        <p class="paperTitle">A Platform for Controlled Template-Independent Enzymatic Synthesis of RNA Oligonucleotides and Therapeutics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547106" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547106" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wiegand, D. J.; Rittichier, J.; Meyer, E.; Lee, H.; Conway, N. J.; Ahlstedt, D.; Yurtsever, Z.; Rainone, D.; Kuru, E.; Church, G.</p>
        <p class="info">Score: 14.9, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547106' target='https://doi.org/10.1101/2023.06.29.547106'> 10.1101/2023.06.29.547106</a></p>
        <p class="abstract">Therapeutic RNA oligonucleotides have shown tremendous potential to manage and treat disease, yet current manufacturing methods cannot deliver on this promise. Here, we report the development and optimization of a novel, aqueous-based, template-independent enzymatic RNA oligonucleotide synthesis platform as an alternative to traditional chemical methodologies. Our platform is made possible by reversible terminator nucleoside triphosphates and an enzyme capable of their incorporation. We show that many common therapeutic RNA modifications are compatible with our process and demonstrate the enzymatic synthesis of natural and modified oligonucleotides in both liquid and solid phases. Our platform offers many unique advantages over chemical synthesis, including the realization of a more sustainable process to produce therapeutic RNA oligonucleotides.

One-Sentence SummaryAn enzyme and novel nucleotide building blocks are used to synthesize RNA oligonucleotides template independently under aqueous conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547496">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547496" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547496">
        <p class="paperTitle">xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547496" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547496" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, B.; Cheng, X.; Gengyang, L.-a.; Li, S.; Zeng, X.; Wang, B.; Gong, J.; Liu, C.; Zeng, A.; Dong, Y.; Tang, J.; Song, L.</p>
        <p class="info">Score: 83.1, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547496' target='https://doi.org/10.1101/2023.07.05.547496'> 10.1101/2023.07.05.547496</a></p>
        <p class="abstract">Protein language models have shown remarkable success in learning biological information from protein sequences. However, most existing models are limited by either autoencoding or autoregressive pre-training objectives, which makes them struggle to handle protein understanding and generation tasks concurrently. This paper proposes a unified protein language model, xTrimoPGLM, to address these two types of tasks simultaneously through an innovative pre-training framework. Our key technical contribution is an exploration of the compatibility and the potential for joint optimization of the two types of objectives, which has led to a strategy for training xTrimoPGLM at an unprecedented scale of 100 billion parameters and 1 trillion training tokens. Our extensive experiments reveal that xTrimoPGLM significantly outperforms other advanced baselines in diverse protein understanding tasks (13 out of 15 tasks across four categories) and generates novel protein sequences which are structurally similar to natural ones. Furthermore, using the same xTrimoPGLM framework, we train an antibody-specific model (xTrimoPGLM-Ab) using 1 billion parameters. This model set a new record in predicting antibody naturalness and structures, both essential to the field of antibody-based drug design, and demonstrated a significantly faster inference speed than AlphaFold2. These results highlight the substantial capability and versatility of xTrimoPGLM in understanding and generating protein sequences.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.04.22.489191">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.04.22.489191" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.04.22.489191">
        <p class="paperTitle">Engineered sensor bacteria evolve master-level gameplay through accelerated adaptation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.04.22.489191" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.04.22.489191" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Prakash, S.; Racovita, A.; Varela, C.; Walsh, M.; Galizi, R.; Isalan, M.; Jaramillo, A.</p>
        <p class="info">Score: 259.8, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.04.22.489191' target='https://doi.org/10.1101/2022.04.22.489191'> 10.1101/2022.04.22.489191</a></p>
        <p class="abstract">Gene circuits enable cells to make decisions by controlling the expression of genes in reaction to specific environmental factors1. These circuits can be designed to encode logical operations2-7, but implementation of more complex algorithms has proved more challenging. Directed evolution optimizes gene circuits8 without the need for design knowledge9, but adjusting multiple genes and conditions10 in genotype searches poses challenges11. Here we show a multicellular sensor system, AdaptoCells, in Escherichia coli, that can evolve complex behavior through an accelerated adaptation to chemical environments. AdaptoCells recognize chemical patterns and act as a decision-making system. Using an iterative improvement method, we demonstrate that the AdaptoCells can evolve to achieve mastery in the game of tic-tac-toe, demonstrating an unprecedented level of complexity for engineered living cells. We provide an effective and straightforward way to encode complexity in gene circuits, allowing for fast adaptation in response to dynamic environments and leading to optimal decisions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548713">
        <p class="paperTitle">Modular and integrative activity reporters enhance biochemical studies in the yeast ER.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinusen, S. G.; Slaton, E. W.; Nelson, S. E.; Pulgar, M. A.; Besu, J. T.; Simas, C. F.; Denard, C. A.</p>
        <p class="info">Score: 4.5, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548713' target='https://doi.org/10.1101/2023.07.12.548713'> 10.1101/2023.07.12.548713</a></p>
        <p class="abstract">The yeast endoplasmic reticulum sequestration and screening (YESS) system is a generalizable platform that has become highly useful to investigate post-translational modification enzymes (PTM-enzymes). This system enables researchers to profile and engineer the activity and substrate specificity of PTM-enzymes and to discover inhibitor-resistant enzyme mutants. In this study, we expand the capabilities of YESS by transferring its functional components to integrative plasmids. The YESS integrative system yields uniform protein expression and protease activities in various configurations, allows one to integrate activity reporters at two independent loci and to split the system between integrative and centromeric plasmids. We characterize these integrative reporters with two viral proteases, Tobacco etch virus (TEVp) and 3-chymotrypsin like protease (3CLpro), in terms of coefficient of variance, signal-to-noise ratio and fold-activation. Overall, we provide a framework for chromosomal-based studies that is modular, enabling rigorous high-throughput assays of PTM-enzymes in yeast.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548755">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548755" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548755">
        <p class="paperTitle">LactoSpanks: a collection of IPTG inducible promoters for the commensal lactic acid bacteria Lactobacillus gasseri.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548755" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548755" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fristot, E.; Cambray, G.; Bonnet, J.</p>
        <p class="info">Score: 3.7, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548755' target='https://doi.org/10.1101/2023.07.13.548755'> 10.1101/2023.07.13.548755</a></p>
        <p class="abstract">Lactic acid bacteria (LAB) are important for many biotechnological applications, such as bioproduction and engineered probiotics for therapy. Inducible promoters are key gene expression control elements, yet those available in LAB are mainly based on bacteriocin systems and have many drawbacks, including large gene clusters, costly inducer peptides and little portability to in vivo settings. Using Lactobacillus gasseri, a model commensal bacteria from the human gut, we report the engineering of Lactospanks promoters (Pls), a collection of variable strength inducible promoters controlled by the LacI repressor from B. subtilis and induced by isopropyl {beta}-D-1-thiogalactopyranoside (IPTG). We first show that the Phyper-spank promoter from Bacillus subtilis is functional in L. gasseri, albeit with substantial leakage. We then construct and screen a semi-rational library of Phyper-spank variants to select a set of four IPTG-inducible promoters that span a range of expression levels and exhibit reduced leakages and operational dynamic ranges (from ca. 9 to 28 fold-change). With their low genetic footprint and simplicity of use, Lactospanks will support many applications in L. gasseri, and potentially other lactic acid and gram-positive bacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547033">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547033" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547033">
        <p class="paperTitle">A SEVA-based, CRISPR-Cas3-assisted genome engineering approach for Pseudomonas with efficient vector curing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547033" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547033" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lammens, E.-M.; Volke, D. C.; Schroven, K.; Voet, M.; Kerremans, A.; Lavigne, R.; Hendrix, H.</p>
        <p class="info">Score: 2.8, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547033' target='https://doi.org/10.1101/2023.06.29.547033'> 10.1101/2023.06.29.547033</a></p>
        <p class="abstract">The development of CRISPR-Cas-based engineering technologies has revolutionized the microbial biotechnology field. Over the years, the Class II Type II CRISPR-Cas9 system has become the gold standard for genome editing in many bacterial hosts. However, the Cas9 system does not allow efficient genomic integration in Pseudomonas putida, an emerging Synthetic Biology host, without the assistance of lambda-Red recombineering. In this work, we utilize the alternative Class I Type I-C CRISPR-Cas3 system from Pseudomonas aeruginosa as a highly-efficient genome editing tool for P. putida and P. aeruginosa. This system consists of two vectors, one encoding the Cas genes, CRISPR array and targeting spacer, and a second SEVA-vector, containing the homologous repair template. Both vectors are Golden Gate compatible for rapid cloning and are available with multiple antibiotic markers, for application in various Gram-negative hosts and different designs. By employing this Cas3 system, we successfully integrated an 820-bp cassette in the genome of P. putida and performed several genomic deletions in P. aeruginosa within four days, with an efficiency of &gt;83% for both hosts. Moreover, by introducing a universal self-targeting spacer, the Cas3 system rapidly cures all helper vectors, including itself, from the host strain in a matter of days. As such, this system constitutes a valuable engineering tool for Pseudomonas, to complement the existing range of Cas9-based editing methods and facilitates genomic engineering efforts of this important genus.

ImportanceThe CRISPR-Cas3 editing system as presented here facilitates the creation of genomic alterations in P. putida and P. aeruginosa in a straightforward manner. By providing the Cas3 system as a vector set with Golden Gate compatibility and different antibiotic markers, as well as by employing the established SEVA vector set to provide the homology repair template, this system is flexible and can readily be ported to a multitude of Gram-negative hosts. Besides genome editing, the Cas3 system can also be used as an effective and universal tool for vector curing. This is achieved by introducing a spacer that targets the oriT, present on the majority of established (SEVA) vectors. Based on this, the Cas3 system efficiently removes up to three vectors in only a few days. As such, this curing approach may also benefit other genomic engineering methods or remove naturally-occurring plasmids from bacteria.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
