<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="DIB-BOT: An open-source hardware approach for high throughput droplet interface bilayer deposition
Authors: Mason, A. F.; Wickham, S. F.; Baker, M. A.
Score: 6.8, Published: 2024-01-26 DOI: 10.1101/2024.01.26.577347
Droplet interface bilayers (DIBs) provide a controlled lipid environment for the single-molecule investigation of a range of biologically relevant membrane-bound processes and have garnered attention for their potential applications in bottom-up artificial cells, biosensing, and biophysics. However, the fabrication of DIBs is currently hindered by time-consuming processes and specialized equipment.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="DIB-BOT: An open-source hardware approach for high throughput droplet interface bilayer deposition
Authors: Mason, A. F.; Wickham, S. F.; Baker, M. A.
Score: 6.8, Published: 2024-01-26 DOI: 10.1101/2024.01.26.577347
Droplet interface bilayers (DIBs) provide a controlled lipid environment for the single-molecule investigation of a range of biologically relevant membrane-bound processes and have garnered attention for their potential applications in bottom-up artificial cells, biosensing, and biophysics. However, the fabrication of DIBs is currently hindered by time-consuming processes and specialized equipment." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-31T10:38:28+00:00" />
<meta property="article:modified_time" content="2024-01-31T10:38:28+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="DIB-BOT: An open-source hardware approach for high throughput droplet interface bilayer deposition
Authors: Mason, A. F.; Wickham, S. F.; Baker, M. A.
Score: 6.8, Published: 2024-01-26 DOI: 10.1101/2024.01.26.577347
Droplet interface bilayers (DIBs) provide a controlled lipid environment for the single-molecule investigation of a range of biologically relevant membrane-bound processes and have garnered attention for their potential applications in bottom-up artificial cells, biosensing, and biophysics. However, the fabrication of DIBs is currently hindered by time-consuming processes and specialized equipment."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "DIB-BOT: An open-source hardware approach for high throughput droplet interface bilayer deposition\nAuthors: Mason, A. F.; Wickham, S. F.; Baker, M. A.\nScore: 6.8, Published: 2024-01-26 DOI: 10.1101/2024.01.26.577347\nDroplet interface bilayers (DIBs) provide a controlled lipid environment for the single-molecule investigation of a range of biologically relevant membrane-bound processes and have garnered attention for their potential applications in bottom-up artificial cells, biosensing, and biophysics. However, the fabrication of DIBs is currently hindered by time-consuming processes and specialized equipment.",
  "keywords": [
    
  ],
  "articleBody": " DIB-BOT: An open-source hardware approach for high throughput droplet interface bilayer deposition\nAuthors: Mason, A. F.; Wickham, S. F.; Baker, M. A.\nScore: 6.8, Published: 2024-01-26 DOI: 10.1101/2024.01.26.577347\nDroplet interface bilayers (DIBs) provide a controlled lipid environment for the single-molecule investigation of a range of biologically relevant membrane-bound processes and have garnered attention for their potential applications in bottom-up artificial cells, biosensing, and biophysics. However, the fabrication of DIBs is currently hindered by time-consuming processes and specialized equipment. These fabrication limitations prevent the scale-up of DIB assays, making it difficult to generate the large data sets required to achieve statistically significant conclusions in single-molecule biological assays where heterogeneous behaviour is often observed. This research describes an open-source solution, dubbed \"DIB-BOT,\" constructed by coupling a nanoinjector with an entry-level 3D printer. We present DIB-BOT as a platform to achieve rapid, reproducible, and reliable fabrication of large numbers of DIBs, addressing the limitations of manual methods. Leveraging commercially available off-the-shelf components, DIB-BOT exhibits high spatial reproducibility, minimal user input, and the ability to scale experiments rapidly. Here we demonstrate the utility of the system by integrating pairwise droplet assembly with a fluorescence plate-reader to execute a biologically relevant assay. When compared with manual DIB fabrication, the DIB-BOT had a tenfold reduction in droplet volume error, a threefold reduction in positional error, and 100% droplet yield. Overall, this method has potential to reduce entry barriers to the use of DIB methods, broadening the applications of DIB research, and generating higher quality data sets.\ngraded-CRISPRi, a novel tool for tuning the strengths of CRISPRi-mediated knockdowns in Vibrio natriegens using gRNA libraries\nAuthors: Stukenberg, D.; Faber, A.; Becker, A.\nScore: 5.1, Published: 2024-01-29 DOI: 10.1101/2024.01.29.577714\nIn recent years, the fast-growing bacterium Vibrio natriegens has gained increasing attention as it has the potential to become a next-generation chassis for Synthetic Biology. A wide range of genetic parts and genome engineering methods have already been developed. However, there is still a need for a well characterized tool to effectively and gradually reduce the expression level of native genes. To bridge this gap, we created graded-CRISPRi, a system utilizing gRNA variants that lead to varying levels of repression strength. By incorporating multiple gRNA sequences into our design, we successfully extended this concept to simultaneously repress four distinct reporter genes. Furthermore, we demonstrated the capability of using graded-CRISPRi to target native genes, thereby examining the effect of various knockdown levels on growth.\nOrthogonalized human protease control of secreted signals\nAuthors: Aldrete, C. A.; Vlahos, A. E.; Pei, J.; Cong, Q.; Gao, X. J.\nScore: 12.1, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576308\nSynthetic circuits that regulate protein secretion in human cells could support cell-based therapies by enabling control over local environments. While protein-level circuits enable such potential clinical applications, featuring orthogonality and compactness, their non-human origin poses a potential immunogenic risk. Here, we developed Humanized Drug Induced Regulation of Engineered CyTokines (hDIRECT) as a platform to control cytokine activity exclusively using human-derived proteins. We sourced a specific human protease and its FDA-approved inhibitor. We engineered cytokines whose activities can be activated and abrogated by proteolytic cleavage. We utilized species specificity and re-localization strategies to orthogonalize the cytokines and protease from the human context that they would be deployed in. hDIRECT should enable local cytokine activation to support a variety of cell-based therapies such as muscle regeneration and cancer immunotherapy. Our work offers a proof of concept for the emerging appreciation of humanization in synthetic biology for human health.\nDe novo-designed minibinders expand the synthetic biology sensing repertoire\nAuthors: Weinberg, Z. Y.; Soliman, S. S.; Kim, M. S.; Chen, I. P.; Ott, M.; El-Samad, H.\nScore: 36.0, Published: 2024-01-15 DOI: 10.1101/2024.01.12.575267\nSynthetic and chimeric receptors capable of recognizing and responding to user-defined antigens have enabled \"smart\" therapeutics based on engineered cells. These cell engineering tools depend on antigen sensors which are most often derived from antibodies. Advances in the de novo design of proteins have enabled the design of protein binders with the potential to target epitopes with unique properties and faster production timelines compared to antibodies. Building upon our previous work combining a de novo-designed minibinder of the Spike protein of SARS-CoV-2 with the synthetic receptor synNotch (SARSNotch), we investigated whether minibinders can be readily adapted to a diversity of cell engineering tools. We show that the Spike minibinder LCB1 easily generalizes to a next-generation proteolytic receptor SNIPR that performs similarly to our previously reported SARSNotch. LCB1-SNIPR successfully enables the detection of live SARS-CoV-2, an improvement over SARSNotch which can only detect cell-expressed Spike. To test the generalizability of minibinders to diverse applications, we tested LCB1 as an antigen sensor for a chimeric antigen receptor (CAR). LCB1-CAR enabled CD8+ T cells to cytotoxically target Spike-expressing cells. Our findings suggest that minibinders represent a novel class of antigen sensors that have the potential to dramatically expand the sensing repertoire of cell engineering tools.\nSequence-function mapping of proline-rich antimicrobial peptides\nAuthors: Collins, J.; McConnell, A.; Schmitz, Z.; Hackel, B. J.\nScore: 1.2, Published: 2024-01-29 DOI: 10.1101/2024.01.28.577586\nAntimicrobial peptides (AMPs) are essential elements of natural cellular combat and candidates as antibiotic therapy. Elevated function may be needed for robust physiological performance. Yet, both pure protein design and combinatorial library discovery are hindered by the complexity of antimicrobial activity. We applied a recently developed high-throughput technique, sequence-activity mapping of AMPs via depletion (SAMP-Dep), to proline-rich AMPs. Robust self-inhibition was achieved for metalnikowin 1 (Met) and apidaecin 1b (Api). SAMP-Dep exhibited high reproducibility with correlation coefficients 0.90 and 0.92, for Met and Api, respectively, between replicates and 0.99 and 0.96 for synonymous genetic variants. Sequence-activity maps were obtained via characterization of 26,000 and 34,000 mutants of Met and Api, respectively. Both AMPs exhibit similar mutational profiles including beneficial mutations at one terminus, the C-terminus for Met and N-terminus for Api, which is consistent with their opposite binding orientations in the ribosome. While Met and Api reside with the family of proline-rich AMPs, different proline sites exhibit substantially different mutational tolerance. Within the PRP motif, proline mutation eliminates activity, whereas non-PRP prolines readily tolerate mutation. Homologous mutations are more tolerated, particularly at alternating sites on one 'face' of the peptide. Important and consistent epistasis was observed following the PRP domain within the segment that extends into the ribosomal exit tunnel for both peptides. Variants identified from the SAMP-Dep platform were produced and exposed toward Gram-negative species exogenously, showing either increased potency or specificity for strains tested. In addition to mapping sequence-activity space for fundamental insight and therapeutic engineering, the results advance the robustness of the SAMP-Dep platform for activity characterization.\nThe integration of tandem gene repeats via a bacterial type-II toxin-antitoxin-mediated gene amplification (ToxAmp) system and stability visualisation in Saccharomyces cerevisiae\nAuthors: Evans, S.; Lu, Z.; McDonnell, L.; Anderson, W.; Peralta, F.; Watkins, T.; Ahmed, H.; Luna-Flores, C. H.; Loan, T. D.; Navone, L.; Trau, M.; Scott, C.; Speight, R.; Vickers, C.; Peng, B.\nScore: 1.0, Published: 2024-01-30 DOI: 10.1101/2024.01.30.578080\nTandem gene repeats naturally occur as important genomic features and determine many traits in living organisms, like human diseases and microbial productivities of target bioproducts. Here, we develop a bacterial type-II toxin-antitoxin-mediated method to manipulate genomic integration of tandem gene repeats in Saccharomyces cerevisiae and further visualise the evolutionary trajectories of gene repeats. We designed a tri-vector system to introduce toxin-antitoxin-driven gene amplification (ToxAmp) modules, and accidentally re-visited the high-level capacity of multi-fragment co-transformation in S. cerevisiae. This system delivered the multi-copy gene integration in the form of tandem gene repeats spontaneously and independently from toxin-antitoxin-mediated selection. Inducing the toxin (RelE) expressing via a copper (II)-inducible CUP1 promoter successfully drove the in-situ gene amplification of the antitoxin (RelB) module, resulting in ~40 copies of a green fluorescence reporter (GFP) gene per copy of genome. The copy-number changes, increasing and decreasing, and stable maintenance were visualised using the GFP and blue chromoprotein AeBlue as reporters. Copy-number increasing happened spontaneously not depending on a selection pressure and was quickly enriched through toxin-antitoxin-mediated selection. In summary, the bacterial toxin-antitoxin systems provide a flexible mechanism to manipulate gene copy number in eukaryotic cells and can be exploited for synthetic biology and metabolic engineering applications.\n",
  "wordCount" : "1352",
  "inLanguage": "en",
  "datePublished": "2024-01-31T10:38:28Z",
  "dateModified": "2024-01-31T10:38:28Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 31, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577347">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577347" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577347">
        <p class="paperTitle">DIB-BOT: An open-source hardware approach for high throughput droplet interface bilayer deposition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577347" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577347" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mason, A. F.; Wickham, S. F.; Baker, M. A.</p>
        <p class="info">Score: 6.8, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577347' target='https://doi.org/10.1101/2024.01.26.577347'> 10.1101/2024.01.26.577347</a></p>
        <p class="abstract">Droplet interface bilayers (DIBs) provide a controlled lipid environment for the single-molecule investigation of a range of biologically relevant membrane-bound processes and have garnered attention for their potential applications in bottom-up artificial cells, biosensing, and biophysics. However, the fabrication of DIBs is currently hindered by time-consuming processes and specialized equipment. These fabrication limitations prevent the scale-up of DIB assays, making it difficult to generate the large data sets required to achieve statistically significant conclusions in single-molecule biological assays where heterogeneous behaviour is often observed. This research describes an open-source solution, dubbed &#34;DIB-BOT,&#34; constructed by coupling a nanoinjector with an entry-level 3D printer. We present DIB-BOT as a platform to achieve rapid, reproducible, and reliable fabrication of large numbers of DIBs, addressing the limitations of manual methods. Leveraging commercially available off-the-shelf components, DIB-BOT exhibits high spatial reproducibility, minimal user input, and the ability to scale experiments rapidly. Here we demonstrate the utility of the system by integrating pairwise droplet assembly with a fluorescence plate-reader to execute a biologically relevant assay. When compared with manual DIB fabrication, the DIB-BOT had a tenfold reduction in droplet volume error, a threefold reduction in positional error, and 100% droplet yield. Overall, this method has potential to reduce entry barriers to the use of DIB methods, broadening the applications of DIB research, and generating higher quality data sets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577714">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577714" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577714">
        <p class="paperTitle">graded-CRISPRi, a novel tool for tuning the strengths of CRISPRi-mediated knockdowns in Vibrio natriegens using gRNA libraries</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577714" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577714" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stukenberg, D.; Faber, A.; Becker, A.</p>
        <p class="info">Score: 5.1, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577714' target='https://doi.org/10.1101/2024.01.29.577714'> 10.1101/2024.01.29.577714</a></p>
        <p class="abstract">In recent years, the fast-growing bacterium Vibrio natriegens has gained increasing attention as it has the potential to become a next-generation chassis for Synthetic Biology. A wide range of genetic parts and genome engineering methods have already been developed. However, there is still a need for a well characterized tool to effectively and gradually reduce the expression level of native genes. To bridge this gap, we created graded-CRISPRi, a system utilizing gRNA variants that lead to varying levels of repression strength. By incorporating multiple gRNA sequences into our design, we successfully extended this concept to simultaneously repress four distinct reporter genes. Furthermore, we demonstrated the capability of using graded-CRISPRi to target native genes, thereby examining the effect of various knockdown levels on growth.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576308">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576308" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576308">
        <p class="paperTitle">Orthogonalized human protease control of secreted signals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576308" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576308" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aldrete, C. A.; Vlahos, A. E.; Pei, J.; Cong, Q.; Gao, X. J.</p>
        <p class="info">Score: 12.1, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576308' target='https://doi.org/10.1101/2024.01.18.576308'> 10.1101/2024.01.18.576308</a></p>
        <p class="abstract">Synthetic circuits that regulate protein secretion in human cells could support cell-based therapies by enabling control over local environments. While protein-level circuits enable such potential clinical applications, featuring orthogonality and compactness, their non-human origin poses a potential immunogenic risk. Here, we developed Humanized Drug Induced Regulation of Engineered CyTokines (hDIRECT) as a platform to control cytokine activity exclusively using human-derived proteins. We sourced a specific human protease and its FDA-approved inhibitor. We engineered cytokines whose activities can be activated and abrogated by proteolytic cleavage. We utilized species specificity and re-localization strategies to orthogonalize the cytokines and protease from the human context that they would be deployed in. hDIRECT should enable local cytokine activation to support a variety of cell-based therapies such as muscle regeneration and cancer immunotherapy. Our work offers a proof of concept for the emerging appreciation of humanization in synthetic biology for human health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575267">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575267" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575267">
        <p class="paperTitle">De novo-designed minibinders expand the synthetic biology sensing repertoire</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575267" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575267" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weinberg, Z. Y.; Soliman, S. S.; Kim, M. S.; Chen, I. P.; Ott, M.; El-Samad, H.</p>
        <p class="info">Score: 36.0, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575267' target='https://doi.org/10.1101/2024.01.12.575267'> 10.1101/2024.01.12.575267</a></p>
        <p class="abstract">Synthetic and chimeric receptors capable of recognizing and responding to user-defined antigens have enabled &#34;smart&#34; therapeutics based on engineered cells. These cell engineering tools depend on antigen sensors which are most often derived from antibodies. Advances in the de novo design of proteins have enabled the design of protein binders with the potential to target epitopes with unique properties and faster production timelines compared to antibodies. Building upon our previous work combining a de novo-designed minibinder of the Spike protein of SARS-CoV-2 with the synthetic receptor synNotch (SARSNotch), we investigated whether minibinders can be readily adapted to a diversity of cell engineering tools. We show that the Spike minibinder LCB1 easily generalizes to a next-generation proteolytic receptor SNIPR that performs similarly to our previously reported SARSNotch. LCB1-SNIPR successfully enables the detection of live SARS-CoV-2, an improvement over SARSNotch which can only detect cell-expressed Spike. To test the generalizability of minibinders to diverse applications, we tested LCB1 as an antigen sensor for a chimeric antigen receptor (CAR). LCB1-CAR enabled CD8&#43; T cells to cytotoxically target Spike-expressing cells. Our findings suggest that minibinders represent a novel class of antigen sensors that have the potential to dramatically expand the sensing repertoire of cell engineering tools.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.28.577586">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.28.577586" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.28.577586">
        <p class="paperTitle">Sequence-function mapping of proline-rich antimicrobial peptides</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.28.577586" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.28.577586" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Collins, J.; McConnell, A.; Schmitz, Z.; Hackel, B. J.</p>
        <p class="info">Score: 1.2, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.28.577586' target='https://doi.org/10.1101/2024.01.28.577586'> 10.1101/2024.01.28.577586</a></p>
        <p class="abstract">Antimicrobial peptides (AMPs) are essential elements of natural cellular combat and candidates as antibiotic therapy. Elevated function may be needed for robust physiological performance. Yet, both pure protein design and combinatorial library discovery are hindered by the complexity of antimicrobial activity. We applied a recently developed high-throughput technique, sequence-activity mapping of AMPs via depletion (SAMP-Dep), to proline-rich AMPs. Robust self-inhibition was achieved for metalnikowin 1 (Met) and apidaecin 1b (Api). SAMP-Dep exhibited high reproducibility with correlation coefficients 0.90 and 0.92, for Met and Api, respectively, between replicates and 0.99 and 0.96 for synonymous genetic variants. Sequence-activity maps were obtained via characterization of 26,000 and 34,000 mutants of Met and Api, respectively. Both AMPs exhibit similar mutational profiles including beneficial mutations at one terminus, the C-terminus for Met and N-terminus for Api, which is consistent with their opposite binding orientations in the ribosome. While Met and Api reside with the family of proline-rich AMPs, different proline sites exhibit substantially different mutational tolerance. Within the PRP motif, proline mutation eliminates activity, whereas non-PRP prolines readily tolerate mutation. Homologous mutations are more tolerated, particularly at alternating sites on one &#39;face&#39; of the peptide. Important and consistent epistasis was observed following the PRP domain within the segment that extends into the ribosomal exit tunnel for both peptides. Variants identified from the SAMP-Dep platform were produced and exposed toward Gram-negative species exogenously, showing either increased potency or specificity for strains tested. In addition to mapping sequence-activity space for fundamental insight and therapeutic engineering, the results advance the robustness of the SAMP-Dep platform for activity characterization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.578080">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.578080" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.578080">
        <p class="paperTitle">The integration of tandem gene repeats via a bacterial type-II toxin-antitoxin-mediated gene amplification (ToxAmp) system and stability visualisation in Saccharomyces cerevisiae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.578080" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.578080" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Evans, S.; Lu, Z.; McDonnell, L.; Anderson, W.; Peralta, F.; Watkins, T.; Ahmed, H.; Luna-Flores, C. H.; Loan, T. D.; Navone, L.; Trau, M.; Scott, C.; Speight, R.; Vickers, C.; Peng, B.</p>
        <p class="info">Score: 1.0, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.578080' target='https://doi.org/10.1101/2024.01.30.578080'> 10.1101/2024.01.30.578080</a></p>
        <p class="abstract">Tandem gene repeats naturally occur as important genomic features and determine many traits in living organisms, like human diseases and microbial productivities of target bioproducts. Here, we develop a bacterial type-II toxin-antitoxin-mediated method to manipulate genomic integration of tandem gene repeats in Saccharomyces cerevisiae and further visualise the evolutionary trajectories of gene repeats. We designed a tri-vector system to introduce toxin-antitoxin-driven gene amplification (ToxAmp) modules, and accidentally re-visited the high-level capacity of multi-fragment co-transformation in S. cerevisiae. This system delivered the multi-copy gene integration in the form of tandem gene repeats spontaneously and independently from toxin-antitoxin-mediated selection. Inducing the toxin (RelE) expressing via a copper (II)-inducible CUP1 promoter successfully drove the in-situ gene amplification of the antitoxin (RelB) module, resulting in ~40 copies of a green fluorescence reporter (GFP) gene per copy of genome. The copy-number changes, increasing and decreasing, and stable maintenance were visualised using the GFP and blue chromoprotein AeBlue as reporters. Copy-number increasing happened spontaneously not depending on a selection pressure and was quickly enriched through toxin-antitoxin-mediated selection. In summary, the bacterial toxin-antitoxin systems provide a flexible mechanism to manipulate gene copy number in eukaryotic cells and can be exploited for synthetic biology and metabolic engineering applications.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
