<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://yorks0n.github.io/TRxiv2" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/TRxiv2/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A synthetic biology approach to assemble and reboot clinically-relevant Pseudomonas aeruginosa tailed phages
Authors: Ipoutcha, T.; Racharaks, R.; Huttelmaier, S.; Wilson, C. J.; Ozer, E. A.; Hartmann, E. M.
Score: 12.1, Published: 2023-06-24 DOI: 10.1101/2023.06.23.546310
The rise in frequency of antibiotic resistance has made bacterial infections, specifically Pseudomonas aeruginosa, a cause for greater concern. Phage therapy is a promising solution that uses naturally isolated phages to treat bacterial infections. Ecological limitations, which stipulate a discrete host range and the inevitable evolution of resistance, may be overcome through a better understanding of phage biology and the utilization of engineered phages.">
<meta name="author" content="">
<link rel="canonical" href="https://yorks0n.github.io/TRxiv2/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/TRxiv2/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/TRxiv2/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://yorks0n.github.io/TRxiv2/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://yorks0n.github.io/TRxiv2/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://yorks0n.github.io/TRxiv2/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://yorks0n.github.io/TRxiv2/apple-touch-icon.png">
<link rel="mask-icon" href="https://yorks0n.github.io/TRxiv2/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="A synthetic biology approach to assemble and reboot clinically-relevant Pseudomonas aeruginosa tailed phages
Authors: Ipoutcha, T.; Racharaks, R.; Huttelmaier, S.; Wilson, C. J.; Ozer, E. A.; Hartmann, E. M.
Score: 12.1, Published: 2023-06-24 DOI: 10.1101/2023.06.23.546310
The rise in frequency of antibiotic resistance has made bacterial infections, specifically Pseudomonas aeruginosa, a cause for greater concern. Phage therapy is a promising solution that uses naturally isolated phages to treat bacterial infections. Ecological limitations, which stipulate a discrete host range and the inevitable evolution of resistance, may be overcome through a better understanding of phage biology and the utilization of engineered phages." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://yorks0n.github.io/TRxiv2/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-06-28T09:51:07+00:00" />
<meta property="article:modified_time" content="2023-06-28T09:51:07+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="A synthetic biology approach to assemble and reboot clinically-relevant Pseudomonas aeruginosa tailed phages
Authors: Ipoutcha, T.; Racharaks, R.; Huttelmaier, S.; Wilson, C. J.; Ozer, E. A.; Hartmann, E. M.
Score: 12.1, Published: 2023-06-24 DOI: 10.1101/2023.06.23.546310
The rise in frequency of antibiotic resistance has made bacterial infections, specifically Pseudomonas aeruginosa, a cause for greater concern. Phage therapy is a promising solution that uses naturally isolated phages to treat bacterial infections. Ecological limitations, which stipulate a discrete host range and the inevitable evolution of resistance, may be overcome through a better understanding of phage biology and the utilization of engineered phages."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://yorks0n.github.io/TRxiv2/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://yorks0n.github.io/TRxiv2/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "A synthetic biology approach to assemble and reboot clinically-relevant Pseudomonas aeruginosa tailed phages\nAuthors: Ipoutcha, T.; Racharaks, R.; Huttelmaier, S.; Wilson, C. J.; Ozer, E. A.; Hartmann, E. M.\nScore: 12.1, Published: 2023-06-24 DOI: 10.1101/2023.06.23.546310\nThe rise in frequency of antibiotic resistance has made bacterial infections, specifically Pseudomonas aeruginosa, a cause for greater concern. Phage therapy is a promising solution that uses naturally isolated phages to treat bacterial infections. Ecological limitations, which stipulate a discrete host range and the inevitable evolution of resistance, may be overcome through a better understanding of phage biology and the utilization of engineered phages.",
  "keywords": [
    
  ],
  "articleBody": " A synthetic biology approach to assemble and reboot clinically-relevant Pseudomonas aeruginosa tailed phages\nAuthors: Ipoutcha, T.; Racharaks, R.; Huttelmaier, S.; Wilson, C. J.; Ozer, E. A.; Hartmann, E. M.\nScore: 12.1, Published: 2023-06-24 DOI: 10.1101/2023.06.23.546310\nThe rise in frequency of antibiotic resistance has made bacterial infections, specifically Pseudomonas aeruginosa, a cause for greater concern. Phage therapy is a promising solution that uses naturally isolated phages to treat bacterial infections. Ecological limitations, which stipulate a discrete host range and the inevitable evolution of resistance, may be overcome through a better understanding of phage biology and the utilization of engineered phages. In this study, we developed a synthetic biology approach to construct tailed phages that naturally target clinically-relevant strains of Pseudomonas aeruginosa. As proof of concept, we successfully cloned and assembled the JG024 and DMS3 phage genomes in yeast using transformation-associated recombination (TAR) cloning and rebooted these two phage genomes in two different strains of P. aeruginosa. We identified factors that affected phage reboot efficiency like the phage species or the presence of antiviral defense systems in the bacterial strain. We have successfully extended this method to two other phage species and observed that the method enables the reboot of phages that are naturally unable to infect the strain used for reboot. This research represents a critical step towards the construction of clinically-relevant, engineered P. aeruginosa phages.\nC2CAplus: a one-pot isothermal circle-to-circle DNA amplification system\nAuthors: Grasemann, L.; Pizarro, P. T.; Maerkl, S. J.\nScore: 7.7, Published: 2023-06-26 DOI: 10.1101/2023.06.26.546530\nRolling circle amplification (RCA) is a widely used DNA amplification method that uses circular template DNA as input and produces multimeric, linear single or double stranded DNA. Circle-to-circle amplification (C2CA) has further expanded this method by implementing product re-circularization using restriction and ligation, leading to a higher amplification yield, and enabling the generation of circular products. However, C2CA is a multistep, non- isothermal method, requiring multiple fluid manipulations and thereby compromises several advantages of RCA. Here, we improved C2CA to implement a one-pot, single step, isothermal reaction at temperatures ranging from 25 to 37C. Our C2CAplus method is simple, robust, and produces large quantities of product DNA that can be seen with the naked eye.\nImpact of random 50-base sequences inserted into an intron on splicing in Saccharomyces cerevisiae\nAuthors: Perchlik, M.; Sasse, A.; Mostafavi, S.; Fields, S.; Cuperus, J.\nScore: 6.5, Published: 2023-06-24 DOI: 10.1101/2023.06.21.545966\nIntron splicing is a key regulatory step in gene expression in eukaryotes. Three sequence elements required for splicing - 5' and 3' splice sites and a branch point - are especially well-characterized in Saccharomyces cerevisiae, but our understanding of additional intron features that impact splicing in this organism is incomplete, due largely to its small number of introns. To overcome this limitation, we constructed a library in S. cerevisiae of random 50-nucleotide elements (N50) individually inserted into the intron of a reporter gene and quantified canonical splicing and the use of cryptic splice sites by sequencing analysis. More than 70% of approximately 140,000 N50 elements reduced splicing by at least 20% compared to the intron control. N50 features, including higher GC content, presence of GU repeats and stronger predicted secondary structure of its pre-mRNA, correlated with reduced splicing efficiency. A likely basis for the reduced splicing of such a large proportion of variants is the formation of RNA structures that pair N50 bases - such as the GU repeats - with other bases specifically within the reporter pre-mRNA analyzed. However, neither convolutional neural network nor linear models were able to explain more than a small fraction of the variance in splicing efficiency across the library, suggesting that complex non-linear interactions in RNA structures are not accurately captured by RNA structure prediction methods given the limited number of variants. Our results imply that the specific context of a pre-mRNA may determine the bases allowable in an intron to prevent secondary structures that reduce splicing.\nIn vitro construction and long read sequencing analysis of a 24 kb long artificial DNA sequence encoding the Universal Declaration of the Rights of Man and of the Citizen.\nAuthors: Leblanc, J.; Boulle, O.; Roux, E.; Nicolas, J.; Lavenier, D.; Audic, Y.\nScore: 4.1, Published: 2023-06-26 DOI: 10.1101/2023.06.26.546242\nIn absence of DNA template, the ab initio production of long double-stranded DNA molecules of predefined sequences is particularly challenging. The DNA synthesis step remains a bottleneck for many applications such as functional assessment of ancestral genes, analysis of alternative splicing or DNA-based data storage. We propose in this report a fully in vitro protocol to generate very long double-stranded DNA molecule starting from commercially available short DNA blocks in less than 3 days. This innovative application of Golden Gate assembly allowed us to streamline the assembly process to produce a 24 kb long DNA molecule storing part of the Universal Declaration of Human rights and citizens. The DNA molecule produced can be readily cloned into suitable host/vector system for amplification and selection.\nHarnessing Escherichia coli Dark Genome to Produce Anti-Alzheimer Peptides\nAuthors: Verma, N.; Manvati, S.; Dhar, P.\nScore: 2.5, Published: 2023-06-24 DOI: 10.1101/2023.06.23.546343\nAlzheimer disease (AD) is characterised by progressive neurodegeneration. The critical molecular trigger is believed to be the accumulation of alpha-beta neurotoxic oligomers. Given the proteolytic processing of Amyloid Precursor Protein (APP) by beta-secretase (beta-site APP cleaving enzyme 1, BACE1) as the key step in the building up of alpha-beta oligomers, BACE inhibitors come with therapeutic prospects of preventing or delaying the onset of Alzheimers. To find inhibitory peptides against BACE1, a library of dark peptides was constructed from 4400 intergenic DNA sequences of Escherichia coli. The sequence level analysis was followed by protein structure predictions, molecular docking, and simulation. Based on bioinformatics analysis, 5 potential peptides were screened for experimental validation. Out of these 2 peptides were identified as lead molecules based on BACE1 inhibitory activity, followed by FRET inhibitory assay, western blot, and RT-PCR. An 86.7 % drop in BACE1 level was observed in the presence of the ECOI2 peptide. Though encouraging results were obtained from in-silico and in-vitro studies, more work is required to study the efficacy of these peptides in suitable animal models.\nOptimizing 5'UTRs for mRNA-delivered gene editing using deep learning\nAuthors: Castillo Hair, S.; Fedak, S.; Wang, B.; Linder, J.; Havens, K.; Certo, M.; Seelig, G.\nScore: 5.8, Published: 2023-06-16 DOI: 10.1101/2023.06.15.545194\nmRNA therapeutics are revolutionizing the pharmaceutical industry, but methods to optimize the primary sequence for increased expression are still lacking. Here, we design 5UTRs for efficient mRNA translation using deep learning. We perform polysome profiling of fully or partially randomized 5UTR libraries in three cell types and find that UTR performance is highly correlated across cell types. We train models on all our datasets and use them to guide the design of high-performing 5UTRs using gradient descent and generative neural networks. We experimentally test designed 5UTRs with mRNA encoding megaTALTM gene editing enzymes for two different gene targets and in two different cell lines. We find that the designed 5UTRs support strong gene editing activity. Editing efficiency is correlated between cell types and gene targets, although the best performing UTR was specific to one cargo and cell type. Our results highlight the potential of model-based sequence design for mRNA therapeutics.\nSynthetic Frizzled agonist and LRP antagonist for high-efficiency Wnt/β-catenin signaling manipulation in organoid cultures and in vivo\nAuthors: Dai, Q.; Wang, J.; Lin, Z.; Yu, D.; Yang, H.; Zhang, J.; Li, X.; Zeng, J.; Hu, H.; Ni, C.; Zhao, B.\nScore: 1.0, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545860\nWnt/{beta}-catenin signaling and its dysregulation play critical roles in the fate determination of stem cells and the pathology of various diseases. However, the application of translated Wnt ligand in regenerative medicine is hampered by its hydrophobicity and cross-reactivity with Frizzled (FZD) receptors. Here, we generate an engineered water-soluble, FZD subtype-specific agonist, RRP-pbFn, for high-efficiency Wnt/{beta}-catenin signaling activation. In the absence of direct binding to LRP5/6, RRP-pbFn stimulates Wnt/{beta}-catenin signaling more potently than surrogate Wnt. RRP-pbFn supports the growth of a variety of mouse and human organoids, and induces the expansion of liver and intestine progenitors in vivo. Meanwhile, we develop a synthetic LRP antagonist, RRP-Dkk1c, which exhibits heightened effectiveness in attenuating Wnt/{beta}-catenin signaling activity compared to Dkk1, thereby abolishing the formation of CT26-derived colon cancer xenograft in vivo. Together, these two paired Wnt/{beta}-catenin signaling manipulators hold great promise for biomedical research and potential therapeutics.\nManipulation of encapsulated artificial phospholipid membranes using sub-micellar lysolipid concentrations.\nAuthors: Dimitriou, P.; Li, J.; Jamieson, W. D.; Schneider, J. J.; Castell, O.; Barrow, D.\nScore: 0.8, Published: 2023-06-25 DOI: 10.1101/2023.06.25.546396\nEngineered artificial cells often involve phospholipid membranes in the form of vesicles or membrane mimics. Droplet interface bilayers (DIBs) constitute a commonly used membrane mimic within synthetic biology. However, these model membranes have limited accessibility due to their requirement to be surrounded by an oil environment. Here, we demonstrate in-situ bilayer manipulation of submillimeter, free-standing, encapsulated droplet interface bilayers (eDIBs) in hydrogel capsules formed by ready-to-use 3D-printed microfluidic devices. The eDIB capsules were exposed to various concentrations of membrane tension-altering lysophosphatidylcholine (LPC), in order to investigate the interaction of lysolipids with three-dimensional, droplet bilayer networks. Micellar LPC concentrations trigger the bursting of the eDIB droplets, while at concentrations below the critical micelle concentration (CMC), the encapsulated aqueous inner droplet networks endure structural changes, precisely affecting the DIB contact angles and bilayer area. Manipulation of these enclosed, 3D-orchestrated membrane mimics facilitates the exploration of readily accessible compartmentalized artificial cellular machinery. Collectively, the multi-compartmentalized capsules and the lysolipid-mediated membrane modulation introduce a chemical approach to control the properties and mechanics of artificial cellular membranes, as well as the functionalities of artificial cells, toward responsive soft material developments and drug delivery applications.\nDeep neural networks for predicting single cell responses and probability landscapes\nAuthors: Klumpe, H.; Lugagne, J.-B.; Khalil, A. S.; Dunlop, M.\nScore: 0.8, Published: 2023-06-24 DOI: 10.1101/2023.06.24.546373\nEngineering biology relies on the accurate prediction of cell responses. However, making these predictions is challenging for a variety of reasons, including the stochasticity of biochemical reactions, variability between cells, and incomplete information about underlying biological processes. Machine learning methods, which can model diverse input-output relationships without requiring a priori mechanistic knowledge, are an ideal tool for this task. For example, such approaches can be used to predict gene expression dynamics given time-series data of past expression history. To explore this application, we computationally simulated single cell responses, incorporating different sources of noise and alternative genetic circuit designs. We showed that deep neural networks trained on these simulated data were able to correctly infer the underlying dynamics of a cell response even in the presence of measurement noise and stochasticity in the biochemical reactions. The training set size and the amount of past data provided as inputs both affected prediction quality, with cascaded genetic circuits that introduce delays requiring more past data. We also tested prediction performance on a bistable auto-activation circuit, finding that our initial method for predicting a single trajectory was fundamentally ill-suited for multimodal dynamics. To address this, we updated the network architecture to predict the entire distribution of future states, showing it could accurately predict bimodal expression distributions. Overall, these methods can be readily applied to the diverse prediction tasks necessary to predict and control a variety of biological circuits, a key aspect of many synthetic biology applications.\nPolysulfide Nanoparticles Ameliorate Ischaemia Reperfusion Injury in Renal Transplantation and Improve Kidney Function Post-Transplantation\nAuthors: Stone, J. P.; D'Arcy, R.; Geraghty, A.; Amin, K.; Montero-Fernandez, A.; Dabare, D.; Ghibaudi, M.; Francini, N.; Coradello, G.; Bramhall, J.; Galwey, N. W.; Ruseva, M.; Tirelli, N.; Fildes, J.\nScore: 0.8, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545864\nIntroductionIschaemia-reperfusion injury (IRI) results in the pathophysiological generation of reactive oxygen species (ROS) with concurrent activation of the complement cascade. This manifests as delayed graft function, which has a significant impact on the longevity of the graft. We aimed to evaluate if ROS scavenging nanoparticles can be used to mitigate IRI related injury during normothermic machine perfusion (NMP). MethodsA randomised, two-stage, preclinical trial was used to assess the impact of poly(propylene sulfide) (polysulfide) nanoparticles (PPS-NPs) on parameters associated with IRI in a renal NMP system (experiment 1, n=6 vs 6). Paired porcine kidneys were randomised to receive either an NP-preservation flush followed by 6 hours of NMP with NP-perfusate, or control preservation flush and standard NMP. Following this, an allogeneic transplant-reperfusion model was used to evaluate if treatment with PPS-NPs improved renal haemodynamics post-transplantation (experiment 2, n=6 vs 6). Kidneys were perfused for 3 hours with or without NP, before being reperfused on a circuit primed with matched blood from genetically different donor pigs for 6 hours, without immunosuppression. ResultsIn experiment 1, all kidneys perfused well for 6 hours with physiological renal haemodynamics and biochemistry. Kidneys perfused with PPS-NPs had improved regional tissue perfusion on infra-red imaging. In experiment 2, renal haemodynamics were significantly improved during allogeneic reperfusion (post-transplant) after treatment with NP. Complement activation remained significantly lower in treated kidneys with a diminished TNF- response. This translated into an improvement in tissue integrity. ConclusionIRI was ameliorated following treatment with NPs during preservation and NMP. This was evidenced by an improvement in renal haemodynamics and diminished inflammatory markers upon reperfusion with allogeneic blood.\n",
  "wordCount" : "2190",
  "inLanguage": "en",
  "datePublished": "2023-06-28T09:51:07Z",
  "dateModified": "2023-06-28T09:51:07Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://yorks0n.github.io/TRxiv2/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://yorks0n.github.io/TRxiv2/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://yorks0n.github.io/TRxiv2" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on June 28, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546310">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546310" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546310">
        <p class="paperTitle">A synthetic biology approach to assemble and reboot clinically-relevant Pseudomonas aeruginosa tailed phages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546310" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546310" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ipoutcha, T.; Racharaks, R.; Huttelmaier, S.; Wilson, C. J.; Ozer, E. A.; Hartmann, E. M.</p>
        <p class="info">Score: 12.1, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546310' target='https://doi.org/10.1101/2023.06.23.546310'> 10.1101/2023.06.23.546310</a></p>
        <p class="abstract">The rise in frequency of antibiotic resistance has made bacterial infections, specifically Pseudomonas aeruginosa, a cause for greater concern. Phage therapy is a promising solution that uses naturally isolated phages to treat bacterial infections. Ecological limitations, which stipulate a discrete host range and the inevitable evolution of resistance, may be overcome through a better understanding of phage biology and the utilization of engineered phages. In this study, we developed a synthetic biology approach to construct tailed phages that naturally target clinically-relevant strains of Pseudomonas aeruginosa. As proof of concept, we successfully cloned and assembled the JG024 and DMS3 phage genomes in yeast using transformation-associated recombination (TAR) cloning and rebooted these two phage genomes in two different strains of P. aeruginosa. We identified factors that affected phage reboot efficiency like the phage species or the presence of antiviral defense systems in the bacterial strain. We have successfully extended this method to two other phage species and observed that the method enables the reboot of phages that are naturally unable to infect the strain used for reboot. This research represents a critical step towards the construction of clinically-relevant, engineered P. aeruginosa phages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546530">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546530" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546530">
        <p class="paperTitle">C2CAplus: a one-pot isothermal circle-to-circle DNA amplification system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546530" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546530" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grasemann, L.; Pizarro, P. T.; Maerkl, S. J.</p>
        <p class="info">Score: 7.7, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546530' target='https://doi.org/10.1101/2023.06.26.546530'> 10.1101/2023.06.26.546530</a></p>
        <p class="abstract">Rolling circle amplification (RCA) is a widely used DNA amplification method that uses circular template DNA as input and produces multimeric, linear single or double stranded DNA. Circle-to-circle amplification (C2CA) has further expanded this method by implementing product re-circularization using restriction and ligation, leading to a higher amplification yield, and enabling the generation of circular products. However, C2CA is a multistep, non- isothermal method, requiring multiple fluid manipulations and thereby compromises several advantages of RCA. Here, we improved C2CA to implement a one-pot, single step, isothermal reaction at temperatures ranging from 25 to 37C. Our C2CAplus method is simple, robust, and produces large quantities of product DNA that can be seen with the naked eye.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545966">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545966" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545966">
        <p class="paperTitle">Impact of random 50-base sequences inserted into an intron on splicing in Saccharomyces cerevisiae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545966" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545966" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Perchlik, M.; Sasse, A.; Mostafavi, S.; Fields, S.; Cuperus, J.</p>
        <p class="info">Score: 6.5, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545966' target='https://doi.org/10.1101/2023.06.21.545966'> 10.1101/2023.06.21.545966</a></p>
        <p class="abstract">Intron splicing is a key regulatory step in gene expression in eukaryotes. Three sequence elements required for splicing - 5&#39; and 3&#39; splice sites and a branch point - are especially well-characterized in Saccharomyces cerevisiae, but our understanding of additional intron features that impact splicing in this organism is incomplete, due largely to its small number of introns. To overcome this limitation, we constructed a library in S. cerevisiae of random 50-nucleotide elements (N50) individually inserted into the intron of a reporter gene and quantified canonical splicing and the use of cryptic splice sites by sequencing analysis. More than 70% of approximately 140,000 N50 elements reduced splicing by at least 20% compared to the intron control. N50 features, including higher GC content, presence of GU repeats and stronger predicted secondary structure of its pre-mRNA, correlated with reduced splicing efficiency. A likely basis for the reduced splicing of such a large proportion of variants is the formation of RNA structures that pair N50 bases - such as the GU repeats - with other bases specifically within the reporter pre-mRNA analyzed. However, neither convolutional neural network nor linear models were able to explain more than a small fraction of the variance in splicing efficiency across the library, suggesting that complex non-linear interactions in RNA structures are not accurately captured by RNA structure prediction methods given the limited number of variants. Our results imply that the specific context of a pre-mRNA may determine the bases allowable in an intron to prevent secondary structures that reduce splicing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546242">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546242" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546242">
        <p class="paperTitle">In vitro construction and long read sequencing analysis of a 24 kb long artificial DNA sequence encoding the Universal Declaration of the Rights of Man and of the Citizen.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546242" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546242" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Leblanc, J.; Boulle, O.; Roux, E.; Nicolas, J.; Lavenier, D.; Audic, Y.</p>
        <p class="info">Score: 4.1, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546242' target='https://doi.org/10.1101/2023.06.26.546242'> 10.1101/2023.06.26.546242</a></p>
        <p class="abstract">In absence of DNA template, the ab initio production of long double-stranded DNA molecules of predefined sequences is particularly challenging. The DNA synthesis step remains a bottleneck for many applications such as functional assessment of ancestral genes, analysis of alternative splicing or DNA-based data storage. We propose in this report a fully in vitro protocol to generate very long double-stranded DNA molecule starting from commercially available short DNA blocks in less than 3 days. This innovative application of Golden Gate assembly allowed us to streamline the assembly process to produce a 24 kb long DNA molecule storing part of the Universal Declaration of Human rights and citizens. The DNA molecule produced can be readily cloned into suitable host/vector system for amplification and selection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546343">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546343" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546343">
        <p class="paperTitle">Harnessing Escherichia coli Dark Genome to Produce Anti-Alzheimer Peptides</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546343" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546343" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Verma, N.; Manvati, S.; Dhar, P.</p>
        <p class="info">Score: 2.5, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546343' target='https://doi.org/10.1101/2023.06.23.546343'> 10.1101/2023.06.23.546343</a></p>
        <p class="abstract">Alzheimer disease (AD) is characterised by progressive neurodegeneration. The critical molecular trigger is believed to be the accumulation of alpha-beta neurotoxic oligomers. Given the proteolytic processing of Amyloid Precursor Protein (APP) by beta-secretase (beta-site APP cleaving enzyme 1, BACE1) as the key step in the building up of alpha-beta oligomers, BACE inhibitors come with therapeutic prospects of preventing or delaying the onset of Alzheimers. To find inhibitory peptides against BACE1, a library of dark peptides was constructed from 4400 intergenic DNA sequences of Escherichia coli. The sequence level analysis was followed by protein structure predictions, molecular docking, and simulation. Based on bioinformatics analysis, 5 potential peptides were screened for experimental validation. Out of these 2 peptides were identified as lead molecules based on BACE1 inhibitory activity, followed by FRET inhibitory assay, western blot, and RT-PCR. An 86.7 % drop in BACE1 level was observed in the presence of the ECOI2 peptide. Though encouraging results were obtained from in-silico and in-vitro studies, more work is required to study the efficacy of these peptides in suitable animal models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.15.545194">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.15.545194" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.15.545194">
        <p class="paperTitle">Optimizing 5&#39;UTRs for mRNA-delivered gene editing using deep learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.15.545194" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.15.545194" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Castillo Hair, S.; Fedak, S.; Wang, B.; Linder, J.; Havens, K.; Certo, M.; Seelig, G.</p>
        <p class="info">Score: 5.8, Published: 2023-06-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.15.545194' target='https://doi.org/10.1101/2023.06.15.545194'> 10.1101/2023.06.15.545194</a></p>
        <p class="abstract">mRNA therapeutics are revolutionizing the pharmaceutical industry, but methods to optimize the primary sequence for increased expression are still lacking. Here, we design 5UTRs for efficient mRNA translation using deep learning. We perform polysome profiling of fully or partially randomized 5UTR libraries in three cell types and find that UTR performance is highly correlated across cell types. We train models on all our datasets and use them to guide the design of high-performing 5UTRs using gradient descent and generative neural networks. We experimentally test designed 5UTRs with mRNA encoding megaTALTM gene editing enzymes for two different gene targets and in two different cell lines. We find that the designed 5UTRs support strong gene editing activity. Editing efficiency is correlated between cell types and gene targets, although the best performing UTR was specific to one cargo and cell type. Our results highlight the potential of model-based sequence design for mRNA therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545860">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545860" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545860">
        <p class="paperTitle">Synthetic Frizzled agonist and LRP antagonist for high-efficiency Wnt/β-catenin signaling manipulation in organoid cultures and in vivo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545860" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545860" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dai, Q.; Wang, J.; Lin, Z.; Yu, D.; Yang, H.; Zhang, J.; Li, X.; Zeng, J.; Hu, H.; Ni, C.; Zhao, B.</p>
        <p class="info">Score: 1.0, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545860' target='https://doi.org/10.1101/2023.06.21.545860'> 10.1101/2023.06.21.545860</a></p>
        <p class="abstract">Wnt/{beta}-catenin signaling and its dysregulation play critical roles in the fate determination of stem cells and the pathology of various diseases. However, the application of translated Wnt ligand in regenerative medicine is hampered by its hydrophobicity and cross-reactivity with Frizzled (FZD) receptors. Here, we generate an engineered water-soluble, FZD subtype-specific agonist, RRP-pbFn, for high-efficiency Wnt/{beta}-catenin signaling activation. In the absence of direct binding to LRP5/6, RRP-pbFn stimulates Wnt/{beta}-catenin signaling more potently than surrogate Wnt. RRP-pbFn supports the growth of a variety of mouse and human organoids, and induces the expansion of liver and intestine progenitors in vivo. Meanwhile, we develop a synthetic LRP antagonist, RRP-Dkk1c, which exhibits heightened effectiveness in attenuating Wnt/{beta}-catenin signaling activity compared to Dkk1, thereby abolishing the formation of CT26-derived colon cancer xenograft in vivo. Together, these two paired Wnt/{beta}-catenin signaling manipulators hold great promise for biomedical research and potential therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.25.546396">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.25.546396" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.25.546396">
        <p class="paperTitle">Manipulation of encapsulated artificial phospholipid membranes using sub-micellar lysolipid concentrations.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.25.546396" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.25.546396" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dimitriou, P.; Li, J.; Jamieson, W. D.; Schneider, J. J.; Castell, O.; Barrow, D.</p>
        <p class="info">Score: 0.8, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.25.546396' target='https://doi.org/10.1101/2023.06.25.546396'> 10.1101/2023.06.25.546396</a></p>
        <p class="abstract">Engineered artificial cells often involve phospholipid membranes in the form of vesicles or membrane mimics. Droplet interface bilayers (DIBs) constitute a commonly used membrane mimic within synthetic biology. However, these model membranes have limited accessibility due to their requirement to be surrounded by an oil environment. Here, we demonstrate in-situ bilayer manipulation of submillimeter, free-standing, encapsulated droplet interface bilayers (eDIBs) in hydrogel capsules formed by ready-to-use 3D-printed microfluidic devices. The eDIB capsules were exposed to various concentrations of membrane tension-altering lysophosphatidylcholine (LPC), in order to investigate the interaction of lysolipids with three-dimensional, droplet bilayer networks. Micellar LPC concentrations trigger the bursting of the eDIB droplets, while at concentrations below the critical micelle concentration (CMC), the encapsulated aqueous inner droplet networks endure structural changes, precisely affecting the DIB contact angles and bilayer area. Manipulation of these enclosed, 3D-orchestrated membrane mimics facilitates the exploration of readily accessible compartmentalized artificial cellular machinery. Collectively, the multi-compartmentalized capsules and the lysolipid-mediated membrane modulation introduce a chemical approach to control the properties and mechanics of artificial cellular membranes, as well as the functionalities of artificial cells, toward responsive soft material developments and drug delivery applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.24.546373">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.24.546373" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.24.546373">
        <p class="paperTitle">Deep neural networks for predicting single cell responses and probability landscapes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.24.546373" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.24.546373" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Klumpe, H.; Lugagne, J.-B.; Khalil, A. S.; Dunlop, M.</p>
        <p class="info">Score: 0.8, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.24.546373' target='https://doi.org/10.1101/2023.06.24.546373'> 10.1101/2023.06.24.546373</a></p>
        <p class="abstract">Engineering biology relies on the accurate prediction of cell responses. However, making these predictions is challenging for a variety of reasons, including the stochasticity of biochemical reactions, variability between cells, and incomplete information about underlying biological processes. Machine learning methods, which can model diverse input-output relationships without requiring a priori mechanistic knowledge, are an ideal tool for this task. For example, such approaches can be used to predict gene expression dynamics given time-series data of past expression history. To explore this application, we computationally simulated single cell responses, incorporating different sources of noise and alternative genetic circuit designs. We showed that deep neural networks trained on these simulated data were able to correctly infer the underlying dynamics of a cell response even in the presence of measurement noise and stochasticity in the biochemical reactions. The training set size and the amount of past data provided as inputs both affected prediction quality, with cascaded genetic circuits that introduce delays requiring more past data. We also tested prediction performance on a bistable auto-activation circuit, finding that our initial method for predicting a single trajectory was fundamentally ill-suited for multimodal dynamics. To address this, we updated the network architecture to predict the entire distribution of future states, showing it could accurately predict bimodal expression distributions. Overall, these methods can be readily applied to the diverse prediction tasks necessary to predict and control a variety of biological circuits, a key aspect of many synthetic biology applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545864">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545864" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545864">
        <p class="paperTitle">Polysulfide Nanoparticles Ameliorate Ischaemia Reperfusion Injury in Renal Transplantation and Improve Kidney Function Post-Transplantation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545864" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545864" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stone, J. P.; D&#39;Arcy, R.; Geraghty, A.; Amin, K.; Montero-Fernandez, A.; Dabare, D.; Ghibaudi, M.; Francini, N.; Coradello, G.; Bramhall, J.; Galwey, N. W.; Ruseva, M.; Tirelli, N.; Fildes, J.</p>
        <p class="info">Score: 0.8, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545864' target='https://doi.org/10.1101/2023.06.21.545864'> 10.1101/2023.06.21.545864</a></p>
        <p class="abstract">IntroductionIschaemia-reperfusion injury (IRI) results in the pathophysiological generation of reactive oxygen species (ROS) with concurrent activation of the complement cascade. This manifests as delayed graft function, which has a significant impact on the longevity of the graft. We aimed to evaluate if ROS scavenging nanoparticles can be used to mitigate IRI related injury during normothermic machine perfusion (NMP).

MethodsA randomised, two-stage, preclinical trial was used to assess the impact of poly(propylene sulfide) (polysulfide) nanoparticles (PPS-NPs) on parameters associated with IRI in a renal NMP system (experiment 1, n=6 vs 6). Paired porcine kidneys were randomised to receive either an NP-preservation flush followed by 6 hours of NMP with NP-perfusate, or control preservation flush and standard NMP. Following this, an allogeneic transplant-reperfusion model was used to evaluate if treatment with PPS-NPs improved renal haemodynamics post-transplantation (experiment 2, n=6 vs 6). Kidneys were perfused for 3 hours with or without NP, before being reperfused on a circuit primed with matched blood from genetically different donor pigs for 6 hours, without immunosuppression.

ResultsIn experiment 1, all kidneys perfused well for 6 hours with physiological renal haemodynamics and biochemistry. Kidneys perfused with PPS-NPs had improved regional tissue perfusion on infra-red imaging. In experiment 2, renal haemodynamics were significantly improved during allogeneic reperfusion (post-transplant) after treatment with NP. Complement activation remained significantly lower in treated kidneys with a diminished TNF- response. This translated into an improvement in tissue integrity.

ConclusionIRI was ameliorated following treatment with NPs during preservation and NMP. This was evidenced by an improvement in renal haemodynamics and diminished inflammatory markers upon reperfusion with allogeneic blood.</p>
      </div>
    </div>
  </div>
</div>









<script src="/TRxiv2/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://yorks0n.github.io/TRxiv2">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
