<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Orthogonalized human protease control of secreted signals
Authors: Aldrete, C. A.; Vlahos, A. E.; Pei, J.; Cong, Q.; Gao, X. J.
Score: 8.4, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576308
Synthetic circuits that regulate protein secretion in human cells could support cell-based therapies by enabling control over local environments. While protein-level circuits enable such potential clinical applications, featuring orthogonality and compactness, their non-human origin poses a potential immunogenic risk. Here, we developed Humanized Drug Induced Regulation of Engineered CyTokines (hDIRECT) as a platform to control cytokine activity exclusively using human-derived proteins.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Orthogonalized human protease control of secreted signals
Authors: Aldrete, C. A.; Vlahos, A. E.; Pei, J.; Cong, Q.; Gao, X. J.
Score: 8.4, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576308
Synthetic circuits that regulate protein secretion in human cells could support cell-based therapies by enabling control over local environments. While protein-level circuits enable such potential clinical applications, featuring orthogonality and compactness, their non-human origin poses a potential immunogenic risk. Here, we developed Humanized Drug Induced Regulation of Engineered CyTokines (hDIRECT) as a platform to control cytokine activity exclusively using human-derived proteins." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-24T10:40:04+00:00" />
<meta property="article:modified_time" content="2024-01-24T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Orthogonalized human protease control of secreted signals
Authors: Aldrete, C. A.; Vlahos, A. E.; Pei, J.; Cong, Q.; Gao, X. J.
Score: 8.4, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576308
Synthetic circuits that regulate protein secretion in human cells could support cell-based therapies by enabling control over local environments. While protein-level circuits enable such potential clinical applications, featuring orthogonality and compactness, their non-human origin poses a potential immunogenic risk. Here, we developed Humanized Drug Induced Regulation of Engineered CyTokines (hDIRECT) as a platform to control cytokine activity exclusively using human-derived proteins."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Orthogonalized human protease control of secreted signals\nAuthors: Aldrete, C. A.; Vlahos, A. E.; Pei, J.; Cong, Q.; Gao, X. J.\nScore: 8.4, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576308\nSynthetic circuits that regulate protein secretion in human cells could support cell-based therapies by enabling control over local environments. While protein-level circuits enable such potential clinical applications, featuring orthogonality and compactness, their non-human origin poses a potential immunogenic risk. Here, we developed Humanized Drug Induced Regulation of Engineered CyTokines (hDIRECT) as a platform to control cytokine activity exclusively using human-derived proteins.",
  "keywords": [
    
  ],
  "articleBody": " Orthogonalized human protease control of secreted signals\nAuthors: Aldrete, C. A.; Vlahos, A. E.; Pei, J.; Cong, Q.; Gao, X. J.\nScore: 8.4, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576308\nSynthetic circuits that regulate protein secretion in human cells could support cell-based therapies by enabling control over local environments. While protein-level circuits enable such potential clinical applications, featuring orthogonality and compactness, their non-human origin poses a potential immunogenic risk. Here, we developed Humanized Drug Induced Regulation of Engineered CyTokines (hDIRECT) as a platform to control cytokine activity exclusively using human-derived proteins. We sourced a specific human protease and its FDA-approved inhibitor. We engineered cytokines whose activities can be activated and abrogated by proteolytic cleavage. We utilized species specificity and re-localization strategies to orthogonalize the cytokines and protease from the human context that they would be deployed in. hDIRECT should enable local cytokine activation to support a variety of cell-based therapies such as muscle regeneration and cancer immunotherapy. Our work offers a proof of concept for the emerging appreciation of humanization in synthetic biology for human health.\nGenome streamlining to improve performance of a fast-growing cyanobacterium Synechococcus elongatus UTEX 2973\nAuthors: Sengupta, A.; Bandyopadhyay, A.; Sarkar, D.; Hendry, J. I.; Schubert, M. G.; Liu, D.; Church, G.; Maranas, C. D.; Pakrasi, H. B.\nScore: 7.1, Published: 2024-01-16 DOI: 10.1101/2024.01.16.575707\nCyanobacteria are photosynthetic organisms that have garnered significant recognition as potential hosts for sustainable bioproduction. However, their complex regulatory networks pose significant challenges to major metabolic engineering efforts, thereby limiting their feasibility as production hosts. Genome streamlining has been demonstrated to be a successful approach for improving productivity and fitness in heterotrophs but is yet to be explored to its full potential in phototrophs. Here we present the systematic reduction of the genome of the cyanobacterium exhibiting the fastest exponential growth, Synechococcus elongatus UTEX 2973. This work, the first of its kind in a photoautotroph, involved an iterative process using state-of-the-art genome-editing technology guided by experimental analysis and computational tools. CRISPR/Cas3 enabled large, progressive deletions of predicted dispensable regions and aided in the identification of essential genes. The large deletions were combined to obtain a strain with 55 kb genome reduction. The strains with streamlined genome showed improvement in growth (up to 23%) and productivity (by 22.7%) as compared to the WT. This streamlining strategy not only has the potential to develop cyanobacterial strains with improved growth and productivity traits but can also facilitate a better understanding of their genome to phenome relationships. ImportanceGenome streamlining is an evolutionary strategy used by natural living systems to dispense unnecessary genes from their genome as a mechanism to adapt and evolve. While this strategy has been successfully borrowed to develop synthetic heterotrophic microbial systems with desired phenotype, it has not been extensively explored in photoautotrophs. Genome streamlining strategy incorporates both computational predictions to identify the dispensable regions and experimental validation using genome editing tool and in this study we have employed a modified strategy with the goal to minimize the genome size to an extent that allows optimal cellular fitness under specified conditions. Our strategy has explored a novel genome-editing tool in photoautotrophs which, unlike other existing tools, enables large, spontaneous optimal deletions from the genome. Our findings demonstrate the effectiveness of this modified strategy in obtaining strains with streamlined genome, exhibiting improved fitness and productivity.\nDe novo-designed minibinders expand the synthetic biology sensing repertoire\nAuthors: Weinberg, Z. Y.; Soliman, S. S.; Kim, M. S.; Chen, I. P.; Ott, M.; El-Samad, H.\nScore: 34.2, Published: 2024-01-15 DOI: 10.1101/2024.01.12.575267\nSynthetic and chimeric receptors capable of recognizing and responding to user-defined antigens have enabled \"smart\" therapeutics based on engineered cells. These cell engineering tools depend on antigen sensors which are most often derived from antibodies. Advances in the de novo design of proteins have enabled the design of protein binders with the potential to target epitopes with unique properties and faster production timelines compared to antibodies. Building upon our previous work combining a de novo-designed minibinder of the Spike protein of SARS-CoV-2 with the synthetic receptor synNotch (SARSNotch), we investigated whether minibinders can be readily adapted to a diversity of cell engineering tools. We show that the Spike minibinder LCB1 easily generalizes to a next-generation proteolytic receptor SNIPR that performs similarly to our previously reported SARSNotch. LCB1-SNIPR successfully enables the detection of live SARS-CoV-2, an improvement over SARSNotch which can only detect cell-expressed Spike. To test the generalizability of minibinders to diverse applications, we tested LCB1 as an antigen sensor for a chimeric antigen receptor (CAR). LCB1-CAR enabled CD8+ T cells to cytotoxically target Spike-expressing cells. Our findings suggest that minibinders represent a novel class of antigen sensors that have the potential to dramatically expand the sensing repertoire of cell engineering tools.\nUnlocking de novo antibody design with generative artificial intelligence\nAuthors: Shanehsazzadeh, A.; McPartlon, M.; Kasun, G.; Steiger, A. K.; Sutton, J. M.; Yassine, E.; McCloskey, C.; Haile, R.; Shuai, R.; Alverio, J.; Rakocevic, G.; Levine, S.; Cejovic, J.; Gutierrez, J. M.; Morehead, A.; Dubrovskyi, O.; Chung, C.; Luton, B. K.; Diaz, N.; Kohnert, C.; Consbruck, R.; Carter, H.; LaCombe, C.; Bist, I.; Vilaychack, P.; Anderson, Z.; Xiu, L.; Bringas, P.; Alarcon, K.; Knight, B.; Radach, M.; Bateman, K.; Kopec-Belliveau, G.; Chapman, D.; Bennett, J.; Ventura, A. B.; Canales, G. M.; Gowda, M.; Jackson, K. A.; Caguiat, R.; Brown, A.; Ganini da Silva, D.; Guo, Z.; Abdulhaqq, S\nScore: 351.7, Published: 2024-01-07 DOI: 10.1101/2023.01.08.523187\nGenerative AI has the potential to redefine the process of therapeutic antibody discovery. In this report, we describe and validate deep generative models for the de novo design of antibodies against human epidermal growth factor receptor (HER2) without additional optimization. The models enabled an efficient workflow that combined in silico design methods with high-throughput experimental techniques to rapidly identify binders from a library of [~]106 heavy chain complementarity-determining region (HCDR) variants. We demonstrated that the workflow achieves binding rates of 10.6% for HCDR3 and 1.8% for HCDR123 designs and is statistically superior to baselines. We further characterized 421 diverse binders using surface plasmon resonance (SPR), finding 71 with low nanomolar affinity similar to the therapeutic anti-HER2 antibody trastuzumab. A selected subset of 11 diverse high-affinity binders were functionally equivalent or superior to trastuzumab, with most demonstrating suitable developability features. We designed one binder with [~]3x higher cell-based potency compared to trastuzumab and another with improved cross-species reactivity1. Our generative AI approach unlocks an accelerated path to designing therapeutic antibodies against diverse targets.\nRecombinant venom proteins in insect seminal fluid reduces female lifespan\nAuthors: Beach, S. J.; Maselko, M.\nScore: 7.7, Published: 2024-01-15 DOI: 10.1101/2024.01.14.575309\nThe emergence of insecticide resistance has increased the need for alternative pest management tools1,2 Numerous genetic biocontrol approaches, which involve the release of genetically modified organisms to control pest populations, are in various stages of development to provide highly targeted pest control3-7. However, all current mating-based genetic biocontrol technologies function by releasing engineered males which skew sex-ratios or reduce offspring viability in subsequent generations. This allows mated females continue to cause harm (e.g. transmit disease). Here, we demonstrate the first example of intragenerational genetic biocontrol, wherein mating with engineered males reduces female lifespan. The toxic male technique (TMT) involves the heterologous expression of insecticidal proteins within the male reproductive tract that are transferred to females via mating. We demonstrate TMT in Drosophila melanogaster males, which reduce the median lifespan of mated females by 37 - 59% compared to controls mated to wild type males. Agent-based models of Aedes aegypti predict that compared to existing genetic biocontrol technologies, even modest levels of mated female mortality could allow TMT to suppress a female population substantially faster, which is likely to result in reduced disease burdens. TMT presents a novel approach for combatting outbreaks of disease vectors and agricultural pests. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=132 SRC=\"FIGDIR/small/575309v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (30K): org.highwire.dtl.DTLVardef@1529ed5org.highwire.dtl.DTLVardef@139c517org.highwire.dtl.DTLVardef@1ba93b0org.highwire.dtl.DTLVardef@364432_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "1302",
  "inLanguage": "en",
  "datePublished": "2024-01-24T10:40:04Z",
  "dateModified": "2024-01-24T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 24, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576308">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576308" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576308">
        <p class="paperTitle">Orthogonalized human protease control of secreted signals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576308" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576308" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aldrete, C. A.; Vlahos, A. E.; Pei, J.; Cong, Q.; Gao, X. J.</p>
        <p class="info">Score: 8.4, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576308' target='https://doi.org/10.1101/2024.01.18.576308'> 10.1101/2024.01.18.576308</a></p>
        <p class="abstract">Synthetic circuits that regulate protein secretion in human cells could support cell-based therapies by enabling control over local environments. While protein-level circuits enable such potential clinical applications, featuring orthogonality and compactness, their non-human origin poses a potential immunogenic risk. Here, we developed Humanized Drug Induced Regulation of Engineered CyTokines (hDIRECT) as a platform to control cytokine activity exclusively using human-derived proteins. We sourced a specific human protease and its FDA-approved inhibitor. We engineered cytokines whose activities can be activated and abrogated by proteolytic cleavage. We utilized species specificity and re-localization strategies to orthogonalize the cytokines and protease from the human context that they would be deployed in. hDIRECT should enable local cytokine activation to support a variety of cell-based therapies such as muscle regeneration and cancer immunotherapy. Our work offers a proof of concept for the emerging appreciation of humanization in synthetic biology for human health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575707">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575707" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575707">
        <p class="paperTitle">Genome streamlining to improve performance of a fast-growing cyanobacterium Synechococcus elongatus UTEX 2973</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575707" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575707" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sengupta, A.; Bandyopadhyay, A.; Sarkar, D.; Hendry, J. I.; Schubert, M. G.; Liu, D.; Church, G.; Maranas, C. D.; Pakrasi, H. B.</p>
        <p class="info">Score: 7.1, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575707' target='https://doi.org/10.1101/2024.01.16.575707'> 10.1101/2024.01.16.575707</a></p>
        <p class="abstract">Cyanobacteria are photosynthetic organisms that have garnered significant recognition as potential hosts for sustainable bioproduction. However, their complex regulatory networks pose significant challenges to major metabolic engineering efforts, thereby limiting their feasibility as production hosts. Genome streamlining has been demonstrated to be a successful approach for improving productivity and fitness in heterotrophs but is yet to be explored to its full potential in phototrophs. Here we present the systematic reduction of the genome of the cyanobacterium exhibiting the fastest exponential growth, Synechococcus elongatus UTEX 2973. This work, the first of its kind in a photoautotroph, involved an iterative process using state-of-the-art genome-editing technology guided by experimental analysis and computational tools. CRISPR/Cas3 enabled large, progressive deletions of predicted dispensable regions and aided in the identification of essential genes. The large deletions were combined to obtain a strain with 55 kb genome reduction. The strains with streamlined genome showed improvement in growth (up to 23%) and productivity (by 22.7%) as compared to the WT. This streamlining strategy not only has the potential to develop cyanobacterial strains with improved growth and productivity traits but can also facilitate a better understanding of their genome to phenome relationships.

ImportanceGenome streamlining is an evolutionary strategy used by natural living systems to dispense unnecessary genes from their genome as a mechanism to adapt and evolve. While this strategy has been successfully borrowed to develop synthetic heterotrophic microbial systems with desired phenotype, it has not been extensively explored in photoautotrophs. Genome streamlining strategy incorporates both computational predictions to identify the dispensable regions and experimental validation using genome editing tool and in this study we have employed a modified strategy with the goal to minimize the genome size to an extent that allows optimal cellular fitness under specified conditions. Our strategy has explored a novel genome-editing tool in photoautotrophs which, unlike other existing tools, enables large, spontaneous optimal deletions from the genome. Our findings demonstrate the effectiveness of this modified strategy in obtaining strains with streamlined genome, exhibiting improved fitness and productivity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575267">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575267" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575267">
        <p class="paperTitle">De novo-designed minibinders expand the synthetic biology sensing repertoire</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575267" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575267" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weinberg, Z. Y.; Soliman, S. S.; Kim, M. S.; Chen, I. P.; Ott, M.; El-Samad, H.</p>
        <p class="info">Score: 34.2, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575267' target='https://doi.org/10.1101/2024.01.12.575267'> 10.1101/2024.01.12.575267</a></p>
        <p class="abstract">Synthetic and chimeric receptors capable of recognizing and responding to user-defined antigens have enabled &#34;smart&#34; therapeutics based on engineered cells. These cell engineering tools depend on antigen sensors which are most often derived from antibodies. Advances in the de novo design of proteins have enabled the design of protein binders with the potential to target epitopes with unique properties and faster production timelines compared to antibodies. Building upon our previous work combining a de novo-designed minibinder of the Spike protein of SARS-CoV-2 with the synthetic receptor synNotch (SARSNotch), we investigated whether minibinders can be readily adapted to a diversity of cell engineering tools. We show that the Spike minibinder LCB1 easily generalizes to a next-generation proteolytic receptor SNIPR that performs similarly to our previously reported SARSNotch. LCB1-SNIPR successfully enables the detection of live SARS-CoV-2, an improvement over SARSNotch which can only detect cell-expressed Spike. To test the generalizability of minibinders to diverse applications, we tested LCB1 as an antigen sensor for a chimeric antigen receptor (CAR). LCB1-CAR enabled CD8&#43; T cells to cytotoxically target Spike-expressing cells. Our findings suggest that minibinders represent a novel class of antigen sensors that have the potential to dramatically expand the sensing repertoire of cell engineering tools.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.08.523187">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.08.523187" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.08.523187">
        <p class="paperTitle">Unlocking de novo antibody design with generative artificial intelligence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.08.523187" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.08.523187" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shanehsazzadeh, A.; McPartlon, M.; Kasun, G.; Steiger, A. K.; Sutton, J. M.; Yassine, E.; McCloskey, C.; Haile, R.; Shuai, R.; Alverio, J.; Rakocevic, G.; Levine, S.; Cejovic, J.; Gutierrez, J. M.; Morehead, A.; Dubrovskyi, O.; Chung, C.; Luton, B. K.; Diaz, N.; Kohnert, C.; Consbruck, R.; Carter, H.; LaCombe, C.; Bist, I.; Vilaychack, P.; Anderson, Z.; Xiu, L.; Bringas, P.; Alarcon, K.; Knight, B.; Radach, M.; Bateman, K.; Kopec-Belliveau, G.; Chapman, D.; Bennett, J.; Ventura, A. B.; Canales, G. M.; Gowda, M.; Jackson, K. A.; Caguiat, R.; Brown, A.; Ganini da Silva, D.; Guo, Z.; Abdulhaqq, S</p>
        <p class="info">Score: 351.7, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.08.523187' target='https://doi.org/10.1101/2023.01.08.523187'> 10.1101/2023.01.08.523187</a></p>
        <p class="abstract">Generative AI has the potential to redefine the process of therapeutic antibody discovery. In this report, we describe and validate deep generative models for the de novo design of antibodies against human epidermal growth factor receptor (HER2) without additional optimization. The models enabled an efficient workflow that combined in silico design methods with high-throughput experimental techniques to rapidly identify binders from a library of [~]106 heavy chain complementarity-determining region (HCDR) variants. We demonstrated that the workflow achieves binding rates of 10.6% for HCDR3 and 1.8% for HCDR123 designs and is statistically superior to baselines. We further characterized 421 diverse binders using surface plasmon resonance (SPR), finding 71 with low nanomolar affinity similar to the therapeutic anti-HER2 antibody trastuzumab. A selected subset of 11 diverse high-affinity binders were functionally equivalent or superior to trastuzumab, with most demonstrating suitable developability features. We designed one binder with [~]3x higher cell-based potency compared to trastuzumab and another with improved cross-species reactivity1. Our generative AI approach unlocks an accelerated path to designing therapeutic antibodies against diverse targets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.575309">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.575309" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.575309">
        <p class="paperTitle">Recombinant venom proteins in insect seminal fluid reduces female lifespan</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.575309" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.575309" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Beach, S. J.; Maselko, M.</p>
        <p class="info">Score: 7.7, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.575309' target='https://doi.org/10.1101/2024.01.14.575309'> 10.1101/2024.01.14.575309</a></p>
        <p class="abstract">The emergence of insecticide resistance has increased the need for alternative pest management tools1,2 Numerous genetic biocontrol approaches, which involve the release of genetically modified organisms to control pest populations, are in various stages of development to provide highly targeted pest control3-7. However, all current mating-based genetic biocontrol technologies function by releasing engineered males which skew sex-ratios or reduce offspring viability in subsequent generations. This allows mated females continue to cause harm (e.g. transmit disease). Here, we demonstrate the first example of intragenerational genetic biocontrol, wherein mating with engineered males reduces female lifespan. The toxic male technique (TMT) involves the heterologous expression of insecticidal proteins within the male reproductive tract that are transferred to females via mating. We demonstrate TMT in Drosophila melanogaster males, which reduce the median lifespan of mated females by 37 - 59% compared to controls mated to wild type males. Agent-based models of Aedes aegypti predict that compared to existing genetic biocontrol technologies, even modest levels of mated female mortality could allow TMT to suppress a female population substantially faster, which is likely to result in reduced disease burdens. TMT presents a novel approach for combatting outbreaks of disease vectors and agricultural pests.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=132 SRC=&#34;FIGDIR/small/575309v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (30K):
org.highwire.dtl.DTLVardef@1529ed5org.highwire.dtl.DTLVardef@139c517org.highwire.dtl.DTLVardef@1ba93b0org.highwire.dtl.DTLVardef@364432_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
