<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Computational design of sequence-specific DNA-binding proteins
Authors: Glasscock, C. J.; Pecoraro, R.; McHugh, R.; Doyle, L. A.; Chen, W.; Boivin, O.; Lonnquist, B.; Na, E.; Politanska, Y.; Haddox, H. K.; Cox, D.; Norn, C.; Coventry, B.; Goreshnik, I.; Vafeados, D.; Lee, G. R.; Gordan, R.; Stoddard, B. L.; DiMaio, F.; Baker, D.
Score: 170.1, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558720
Sequence-specific DNA-binding proteins (DBPs) play critical roles in biology and biotechnology, and there has been considerable interest in the engineering of DBPs with new or altered specificities for genome editing and other applications.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Computational design of sequence-specific DNA-binding proteins
Authors: Glasscock, C. J.; Pecoraro, R.; McHugh, R.; Doyle, L. A.; Chen, W.; Boivin, O.; Lonnquist, B.; Na, E.; Politanska, Y.; Haddox, H. K.; Cox, D.; Norn, C.; Coventry, B.; Goreshnik, I.; Vafeados, D.; Lee, G. R.; Gordan, R.; Stoddard, B. L.; DiMaio, F.; Baker, D.
Score: 170.1, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558720
Sequence-specific DNA-binding proteins (DBPs) play critical roles in biology and biotechnology, and there has been considerable interest in the engineering of DBPs with new or altered specificities for genome editing and other applications." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-24T10:37:06+00:00" />
<meta property="article:modified_time" content="2023-09-24T10:37:06+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Computational design of sequence-specific DNA-binding proteins
Authors: Glasscock, C. J.; Pecoraro, R.; McHugh, R.; Doyle, L. A.; Chen, W.; Boivin, O.; Lonnquist, B.; Na, E.; Politanska, Y.; Haddox, H. K.; Cox, D.; Norn, C.; Coventry, B.; Goreshnik, I.; Vafeados, D.; Lee, G. R.; Gordan, R.; Stoddard, B. L.; DiMaio, F.; Baker, D.
Score: 170.1, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558720
Sequence-specific DNA-binding proteins (DBPs) play critical roles in biology and biotechnology, and there has been considerable interest in the engineering of DBPs with new or altered specificities for genome editing and other applications."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Computational design of sequence-specific DNA-binding proteins\nAuthors: Glasscock, C. J.; Pecoraro, R.; McHugh, R.; Doyle, L. A.; Chen, W.; Boivin, O.; Lonnquist, B.; Na, E.; Politanska, Y.; Haddox, H. K.; Cox, D.; Norn, C.; Coventry, B.; Goreshnik, I.; Vafeados, D.; Lee, G. R.; Gordan, R.; Stoddard, B. L.; DiMaio, F.; Baker, D.\nScore: 170.1, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558720\nSequence-specific DNA-binding proteins (DBPs) play critical roles in biology and biotechnology, and there has been considerable interest in the engineering of DBPs with new or altered specificities for genome editing and other applications.",
  "keywords": [
    
  ],
  "articleBody": " Computational design of sequence-specific DNA-binding proteins\nAuthors: Glasscock, C. J.; Pecoraro, R.; McHugh, R.; Doyle, L. A.; Chen, W.; Boivin, O.; Lonnquist, B.; Na, E.; Politanska, Y.; Haddox, H. K.; Cox, D.; Norn, C.; Coventry, B.; Goreshnik, I.; Vafeados, D.; Lee, G. R.; Gordan, R.; Stoddard, B. L.; DiMaio, F.; Baker, D.\nScore: 170.1, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558720\nSequence-specific DNA-binding proteins (DBPs) play critical roles in biology and biotechnology, and there has been considerable interest in the engineering of DBPs with new or altered specificities for genome editing and other applications. While there has been some success in reprogramming naturally occurring DBPs using selection methods, the computational design of new DBPs that recognize arbitrary target sites remains an outstanding challenge. We describe a computational method for the design of small DBPs that recognize specific target sequences through interactions with bases in the major groove, and employ this method in conjunction with experimental screening to generate binders for 5 distinct DNA targets. These binders exhibit specificity closely matching the computational models for the target DNA sequences at as many as 6 base positions and affinities as low as 30 to 100 nM. The crystal structure of a designed DBP-target site complex is in close agreement with the design model, highlighting the accuracy of the design method. The designed DBPs function in both Escherichia coli and mammalian cells to repress and activate transcription of neighboring genes. Our method is a substantial step towards a general route to small and hence readily deliverable sequence-specific DBPs for gene regulation and editing.\nA cell-free transcription-translation pipeline for recreating methylation patterns boosts DNA transformation in bacteria\nAuthors: Vento, J. M.; Durmusoglu, D.; Li, T.; Sullivan, S. M.; Ttofali, F.; van Schaik, J.; Yu, Y.; Barquist, L.; Crook, N.; Beisel, C. L.\nScore: 17.5, Published: 2023-09-16 DOI: 10.1101/2023.09.16.557782\nThe bacterial world offers diverse strains for understanding medical and environmental processes and for engineering synthetic-biology chasses. However, genetically manipulating these strains has faced a long-standing bottleneck: how to efficiently transform DNA. Here we report IMPRINT, a generalized, rapid and scalable approach based on cell-free transcription-translation (TXTL) systems to overcome DNA restriction, a prominent barrier to transformation. IMPRINT utilizes TXTL to express DNA methyltransferases from the bacterial hosts restriction-modification systems. The expressed methyltransferases then methylate DNA in vitro to match the host DNAs methylation pattern, circumventing restriction and enhancing transformation. With IMPRINT, we efficiently multiplex methylation by diverse DNA methyltransferases and enhance plasmid transformation in gram-negative and gram-positive bacteria. We also developed a high-throughput pipeline that identifies the most consequential methyltransferases, and we apply IMPRINT to facilitate a library screen for translational rules in a hard-to-transform Bifidobacterium. Overall, IMPRINT can enhance DNA transformation, enabling use of increasingly sophisticated genetic manipulation tools across the bacterial world.\nThe CASwitch: a synthetic biology solution for high-performance inducible gene expression systems in biotechnology\nAuthors: De Carluccio, G.; Fusco, V.; di Bernardo, D.\nScore: 9.8, Published: 2023-09-20 DOI: 10.1101/2023.09.20.558637\nInducible gene expression systems can be used to control the expression of a gene of interest by means of a small-molecule. One of the most common designs involves engineering a small-molecule responsive transcription factor (TF) and its cognate promoter, which often results in a compromise between minimal uninduced background expression (leakiness) and maximal induced expression. Here, we focussed on an alternative strategy using quantitative synthetic biology to mitigate leakiness while maintaining high expression, without modifying neither the TF nor the promoter. Through mathematical modelling and experimental validations, we designed the CASwitch, a mammalian synthetic gene circuit based on combining two well-known network motifs: the Coherent Feed-Forward Loop (CFFL) and the Mutual Inhibition (MI). The CASwitch combines the CRISPR-Cas endoribonuclease CasRx with the state-of-the-art Tet-On3G inducible gene system to achieve high performances. To demonstrate the potentialities of the CASwitch, we applied it to three different scenarios: enhancing a whole-cell biosensor, controlling expression of a toxic gene and inducible production of Adeno-Associated Virus (AAV) vectors.\nBiofoundry-scale DNA assembly validation using cost-effective high-throughput long read sequencing\nAuthors: Vegh, P.; Donovan, S.; Rosser, S.; Stracquadanio, G.; Fragkoudis, R.\nScore: 7.9, Published: 2023-09-20 DOI: 10.1101/2023.09.19.558498\nBiofoundries are automated high-throughput facilities specialising in the design, construction and testing of engineered/synthetic DNA constructs (plasmids), often from genetic parts. A critical step of this process is assessing the fidelity of the assembled DNA construct to the desired design. Current methods utilised for this purpose are restriction digest or PCR followed by fragment analysis, and sequencing. The Edinburgh Genome Foundry (EGF) has recently established a single-molecule sequencing quality control step using the Oxford Nanopore sequencing technology, along with a companion Nextflow pipeline and a Python package to perform in-depth analysis and generate a detailed report. Our software enables biofoundry scientists and end-users to rapidly analyse sequencing data, without specialised bioinformatics knowledge. In conclusion, we have created a laboratory and software protocol that validates assembled, cloned or edited plasmids, using Nanopore long reads, which can serve as a useful resource for the genetics, synthetic biology and sequencing communities. Author informationAll authors contributed to the design of the sequencing quality control step and pipeline, and the preparation of the manuscript. P.V. wrote the manuscript, designed and implemented the bioinformatics pipeline and interpreted results. S.D. wrote the manuscript, implemented the laboratory protocol and interpreted results. G.S. designed the bioinformatics pipeline. R.F. wrote the manuscript and contributed to the design of the laboratory protocol and pipeline. Address: Edinburgh Genome Foundry (University of Edinburgh), Michael Swann Building, Max Born Crescent, Edinburgh, EH9 3BF, United Kingdom Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=108 SRC=\"FIGDIR/small/558498v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (21K): org.highwire.dtl.DTLVardef@deaec5org.highwire.dtl.DTLVardef@1a422a1org.highwire.dtl.DTLVardef@dd7bf2org.highwire.dtl.DTLVardef@f461d3_HPS_FORMAT_FIGEXP M_FIG C_FIG\nA genetic circuit on a single DNA molecule as autonomous dissipative nanodevice\nAuthors: Greiss, F.; Lardon, N.; Schutz, L.; Barak, Y.; Daube, S. S.; Weinhold, E.; Noireaux, V.; Bar-Ziv, R.\nScore: 5.0, Published: 2023-09-21 DOI: 10.1101/2023.09.21.558768\nRealizing genetic circuits on single DNA molecules as self-encoded dissipative nanodevices is a major step toward miniaturization of autonomous biological systems. A circuit operating on a single DNA implies that genetically encoded proteins localize during coupled transcription-translation to DNA, but a single-molecule measurement demonstrating this has remained a challenge. Here, we used a genetically encoded fluorescent reporter system with improved spatiotemporal resolution and observed the synthesis of individual proteins tethered to a DNA molecule by transient complexes of RNA polymerase, messenger RNA, and ribosome. Against expectations in dilute cell-free conditions where equilibrium considerations favor dispersion, these nascent proteins linger long enough to regulate cascaded reactions on the same DNA. We rationally designed a pulsatile genetic circuit by encoding an activator and repressor in feedback on the same DNA molecule. Driven by the local synthesis of only several proteins per hour and gene, the circuit dynamics exhibited enhanced variability between individual DNA molecules, and fluctuations with a broad power spectrum. Our results demonstrate that co-expressional localization, as a nonequilibrium process, facilitates single-DNA genetic circuits as dissipative nanodevices, with implications for nanobiotechnology applications and artificial cell design.\nExpanding the toolbox for phycobiliprotein assembly: phycoerythrobilin biosynthesis in Synechocystis\nAuthors: Heck, S.; Sommer, F. K.; Zehner, S.; Schroda, M.; Gehringer, M. M.; Frankenberg-Dinkel, N.\nScore: 4.7, Published: 2023-09-19 DOI: 10.1101/2023.09.18.558311\nPhycobiliproteins (PBPs) play a vital role in light harvesting by cyanobacteria, which enables efficient utilization of photon energy for oxygenic photosynthesis. The PBPs carry phycobilins, open-chain tetrapyrrole chromophores derived from heme. The structure and chromophore composition of PBPs is dependent on the organisms ecological niche. In cyanobacteria, these holo-proteins typically form large, macromolecular antenna complexes called phycobilisomes (PBSs). The PBS of Synechocystis sp. PCC 6803 (hereafter Synechocystis) consists of allophycocyanin (APC) and phycocyanin (PC), which exclusively harbor phycocyanobilin (PCB) as a chromophore. Investigations into heterologous PBP biosynthesis in E. coli have proven limiting with respect to PBP assembly and their functional characterization. Consequently, we wanted to engineer a platform for the investigation of heterologously produced PBPs, focusing on unusual, phycoerythrobilin (PEB)-containing light-harvesting proteins called phycoerythrins (PEs) in Synechocystis. As a first step, a gene encoding for the synthesis of the natural cyanobacterial chromophore, PEB, was introduced into Synechocystis. We provide spectroscopic evidence for heterologous PEB formation and show covalent attachment of PEB to the -subunit of PC, CpcA, by HPLC and LC-MS/MS analyses. Fluorescence microscopy and PBS isolation demonstrate a cellular dispersal of PBPs with modified phycobilin content. However, these modifications have minor effects on physiological responses, as demonstrated by growth rates, oxygen evolution, nutrient accumulation, and PBP content analyses. As a result, Synechocystis demonstrates the capacity to efficiently manage PEB biosynthesis and therefore reflects a promising platform for both biochemical and physiological investigations of foreign and unusual PEs.\nUsing the E. coli Alleleome in Strain Design\nAuthors: Phaneuf, P. V.; Jarczynska, Z. D.; Kandasamy, V.; Chauhan, S. M.; Feist, A. M.; Palsson, B. O.\nScore: 4.3, Published: 2023-09-17 DOI: 10.1101/2023.09.17.558058\nLeveraging observed variants in strain design is a promising technique for creating strains with specific properties. Adaptive laboratory evolution (ALE) experiments generate variants that enhance fitness under specific conditions and can contribute to application-specific strain designs. Further, the wild-type (WT) coding alleleome of an organism, the complete set of its genes WT alleles, can provide an additional amount and diversity of variants not yet accessible from the aggregation of ALE experiment results. This study used both an ALE mutation database (3093 genomes) and a large set of WT genomes (12,661 genomes) to explore the solution space of genes involved in tolerance to 10 conditions of industrial importance. To accomplish this, ALE variants for 22 genes previously identified as potentially important for industrial chemical tolerance were collected and supplemented with all available variants from the WT coding alleleome. A total of 4879 variants were reintroduced and used in 10 selection experiments. Although ALE variants were generally found to be more advantageous, specific WT variants were more highly enriched than ALE variants in certain conditions. Additionally, ALE and WT variants rarely overlapped on AA positions, but their clustering did coincide with where highly enriched variants were ultimately located. For genes primarily hosting potential gain-of-function variations, substitutions predicted to have a conservative impact frequently outperformed more radical substitutions. Case studies demonstrated that maximizing the amount of variants enabled easier identification of variant trends, which in turn can be used to better understand areas and characteristics of genes that can be feasibly varied, resulting in what can be thought of as a genome design variable. The combination of ALE and WT variants is a promising approach for use in future projects to better constrain and ultimately achieve practical coverage in the exploration of feasible sequence solution space. Visual Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=64 SRC=\"FIGDIR/small/558058v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (23K): org.highwire.dtl.DTLVardef@17dd15dorg.highwire.dtl.DTLVardef@13c3cb7org.highwire.dtl.DTLVardef@1b635e3org.highwire.dtl.DTLVardef@18c0675_HPS_FORMAT_FIGEXP M_FIG C_FIG\nIMPROVING ANTIBODY AFFINITY USING LABORATORYDATA WITH LANGUAGE MODEL GUIDED DESIGN\nAuthors: Krause, B.; Subramanian, S.; Yuan, T.; Yang, M.; Sato, A.; Naik, N.\nScore: 20.1, Published: 2023-09-13 DOI: 10.1101/2023.09.13.557505\nAO_SCPLOWBSTRACTC_SCPLOWProtein design requires searching exponentially large spaces of possible sequences to find candidates with desired properties. Language models (LMs) pretrained on universal protein datasets have shown potential to help make this search space tractable. However, LMs trained on natural sequences alone have limitations in designing proteins with novel functions, which is especially important for many pharmaceutical applications. In this work, we used a combination of methods to finetune pretrained LMs on laboratory data collected in an anti-CD40L antibody library campaign to develop an ensemble scoring function to model the fitness landscape and guide the design of new antibodies. Laboratory testing showed that the designed antibodies had improved affinity to CD40L. Notably, the designs improved the affinities of four antibodies, originally ranging from 1 nanomolar to 100 picomolar, all to below 25 picomolar, approaching the limit of detection. This work is a promising step towards realizing the potential of LMs to leverage laboratory data to develop improved treatments for diseases.\nHigh-throughput feedback-enabled optogenetic stimulation and spectroscopy in microwell plates\nAuthors: Benman, W.; Datta, S.; Gonzalez-Martinez, D.; Lee, G.; Hooper, J.; Qian, G.; Leavitt, G.; Salloum, L.; Ho, G.; Mhatre, S.; Magaraci, M. S.; Patterson, M.; Mannickarottu, S. G.; Malani, S.; Avalos, J. L.; Chow, B. Y.; Bugaj, L. J.\nScore: 25.0, Published: 2023-09-13 DOI: 10.1101/2022.07.13.499906\nThe ability to perform sophisticated, high-throughput optogenetic experiments has been greatly enhanced by recent open-source illumination devices that allow independent programming of light patterns in single wells of microwell plates. However, there is currently a lack of instrumentation to monitor such experiments in real time, necessitating repeated transfers of the samples to stand-alone analytical instruments, thus limiting the types of experiments that could be performed. Here we address this gap with the development of the optoPlateReader (oPR), an open-source, solid-state, compact device that allows automated optogenetic stimulation and spectroscopy in each well of a 96-well plate. The oPR integrates an optoPlate illumination module with a module called the optoReader, an array of 96 photodiodes and LEDs that allows 96 parallel light measurements. The oPR was optimized for stimulation with blue light and for measurements of optical density and fluorescence. After calibration of all device components, we used the oPR to measure growth and to induce and measure fluorescent protein expression in E. coli. We further demonstrated how the optical read/write capabilities of the oPR permit computer-in-the-loop feedback control, where the current state of the sample can be used to adjust the optical stimulation parameters of the sample according to pre-defined feedback algorithms. The oPR will thus help realize an untapped potential for optogenetic experiments by enabling automated reading, writing, and feedback in microwell plates through open-source hardware that is accessible, customizable, and inexpensive.\nEscherichia coli Microcin B17 as a chassis for the development of novel antimicrobial peptides.\nAuthors: Withanage, S.; Masschelein, J.; Pinheiro, V. B.\nScore: 8.6, Published: 2023-08-30 DOI: 10.1101/2023.08.29.555279\nAntimicrobial resistance remains an urgent and growing global health problem. Peptides with antimicrobial activity are widespread in nature but are rarely harnessed for application, making them an underexplored and untapped resource. Many harbor post-translational modifications, which are essential for function and have evolved alongside specialized export and immunity machineries - leading to the current paradigm that natural sequences cannot be changed, and curtailing their development as antimicrobial agents. Nonetheless, we found sufficient ambiguity in literature to question if that remained true for E. coli MccB17. We demonstrate that its natural machinery for synthesis, export and immunity, can be co-opted as a platform for novel azole-modified antimicrobial peptide discovery. We show that the sequence space available for exploration is large and densely populated with functional sequences. One-Sentence SummaryE. coli can be harnessed as an in vivo platform for the isolation of novel antimicrobial azole-modified peptides.\n",
  "wordCount" : "2402",
  "inLanguage": "en",
  "datePublished": "2023-09-24T10:37:06Z",
  "dateModified": "2023-09-24T10:37:06Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on September 24, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558720">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558720" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558720">
        <p class="paperTitle">Computational design of sequence-specific DNA-binding proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558720" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558720" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Glasscock, C. J.; Pecoraro, R.; McHugh, R.; Doyle, L. A.; Chen, W.; Boivin, O.; Lonnquist, B.; Na, E.; Politanska, Y.; Haddox, H. K.; Cox, D.; Norn, C.; Coventry, B.; Goreshnik, I.; Vafeados, D.; Lee, G. R.; Gordan, R.; Stoddard, B. L.; DiMaio, F.; Baker, D.</p>
        <p class="info">Score: 170.1, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558720' target='https://doi.org/10.1101/2023.09.20.558720'> 10.1101/2023.09.20.558720</a></p>
        <p class="abstract">Sequence-specific DNA-binding proteins (DBPs) play critical roles in biology and biotechnology, and there has been considerable interest in the engineering of DBPs with new or altered specificities for genome editing and other applications. While there has been some success in reprogramming naturally occurring DBPs using selection methods, the computational design of new DBPs that recognize arbitrary target sites remains an outstanding challenge. We describe a computational method for the design of small DBPs that recognize specific target sequences through interactions with bases in the major groove, and employ this method in conjunction with experimental screening to generate binders for 5 distinct DNA targets. These binders exhibit specificity closely matching the computational models for the target DNA sequences at as many as 6 base positions and affinities as low as 30 to 100 nM. The crystal structure of a designed DBP-target site complex is in close agreement with the design model, highlighting the accuracy of the design method. The designed DBPs function in both Escherichia coli and mammalian cells to repress and activate transcription of neighboring genes. Our method is a substantial step towards a general route to small and hence readily deliverable sequence-specific DBPs for gene regulation and editing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.16.557782">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.16.557782" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.16.557782">
        <p class="paperTitle">A cell-free transcription-translation pipeline for recreating methylation patterns boosts DNA transformation in bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.16.557782" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.16.557782" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vento, J. M.; Durmusoglu, D.; Li, T.; Sullivan, S. M.; Ttofali, F.; van Schaik, J.; Yu, Y.; Barquist, L.; Crook, N.; Beisel, C. L.</p>
        <p class="info">Score: 17.5, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.16.557782' target='https://doi.org/10.1101/2023.09.16.557782'> 10.1101/2023.09.16.557782</a></p>
        <p class="abstract">The bacterial world offers diverse strains for understanding medical and environmental processes and for engineering synthetic-biology chasses. However, genetically manipulating these strains has faced a long-standing bottleneck: how to efficiently transform DNA. Here we report IMPRINT, a generalized, rapid and scalable approach based on cell-free transcription-translation (TXTL) systems to overcome DNA restriction, a prominent barrier to transformation. IMPRINT utilizes TXTL to express DNA methyltransferases from the bacterial hosts restriction-modification systems. The expressed methyltransferases then methylate DNA in vitro to match the host DNAs methylation pattern, circumventing restriction and enhancing transformation. With IMPRINT, we efficiently multiplex methylation by diverse DNA methyltransferases and enhance plasmid transformation in gram-negative and gram-positive bacteria. We also developed a high-throughput pipeline that identifies the most consequential methyltransferases, and we apply IMPRINT to facilitate a library screen for translational rules in a hard-to-transform Bifidobacterium. Overall, IMPRINT can enhance DNA transformation, enabling use of increasingly sophisticated genetic manipulation tools across the bacterial world.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558637">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558637" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558637">
        <p class="paperTitle">The CASwitch: a synthetic biology solution for high-performance inducible gene expression systems in biotechnology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558637" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558637" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: De Carluccio, G.; Fusco, V.; di Bernardo, D.</p>
        <p class="info">Score: 9.8, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558637' target='https://doi.org/10.1101/2023.09.20.558637'> 10.1101/2023.09.20.558637</a></p>
        <p class="abstract">Inducible gene expression systems can be used to control the expression of a gene of interest by means of a small-molecule. One of the most common designs involves engineering a small-molecule responsive transcription factor (TF) and its cognate promoter, which often results in a compromise between minimal uninduced background expression (leakiness) and maximal induced expression. Here, we focussed on an alternative strategy using quantitative synthetic biology to mitigate leakiness while maintaining high expression, without modifying neither the TF nor the promoter. Through mathematical modelling and experimental validations, we designed the CASwitch, a mammalian synthetic gene circuit based on combining two well-known network motifs: the Coherent Feed-Forward Loop (CFFL) and the Mutual Inhibition (MI). The CASwitch combines the CRISPR-Cas endoribonuclease CasRx with the state-of-the-art Tet-On3G inducible gene system to achieve high performances. To demonstrate the potentialities of the CASwitch, we applied it to three different scenarios: enhancing a whole-cell biosensor, controlling expression of a toxic gene and inducible production of Adeno-Associated Virus (AAV) vectors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558498">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558498" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558498">
        <p class="paperTitle">Biofoundry-scale DNA assembly validation using cost-effective high-throughput long read sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558498" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558498" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vegh, P.; Donovan, S.; Rosser, S.; Stracquadanio, G.; Fragkoudis, R.</p>
        <p class="info">Score: 7.9, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558498' target='https://doi.org/10.1101/2023.09.19.558498'> 10.1101/2023.09.19.558498</a></p>
        <p class="abstract">Biofoundries are automated high-throughput facilities specialising in the design, construction and testing of engineered/synthetic DNA constructs (plasmids), often from genetic parts. A critical step of this process is assessing the fidelity of the assembled DNA construct to the desired design. Current methods utilised for this purpose are restriction digest or PCR followed by fragment analysis, and sequencing. The Edinburgh Genome Foundry (EGF) has recently established a single-molecule sequencing quality control step using the Oxford Nanopore sequencing technology, along with a companion Nextflow pipeline and a Python package to perform in-depth analysis and generate a detailed report. Our software enables biofoundry scientists and end-users to rapidly analyse sequencing data, without specialised bioinformatics knowledge. In conclusion, we have created a laboratory and software protocol that validates assembled, cloned or edited plasmids, using Nanopore long reads, which can serve as a useful resource for the genetics, synthetic biology and sequencing communities.

Author informationAll authors contributed to the design of the sequencing quality control step and pipeline, and the preparation of the manuscript. P.V. wrote the manuscript, designed and implemented the bioinformatics pipeline and interpreted results. S.D. wrote the manuscript, implemented the laboratory protocol and interpreted results. G.S. designed the bioinformatics pipeline. R.F. wrote the manuscript and contributed to the design of the laboratory protocol and pipeline.

Address: Edinburgh Genome Foundry (University of Edinburgh), Michael Swann Building, Max Born Crescent, Edinburgh, EH9 3BF, United Kingdom

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=108 SRC=&#34;FIGDIR/small/558498v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (21K):
org.highwire.dtl.DTLVardef@deaec5org.highwire.dtl.DTLVardef@1a422a1org.highwire.dtl.DTLVardef@dd7bf2org.highwire.dtl.DTLVardef@f461d3_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558768">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558768" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558768">
        <p class="paperTitle">A genetic circuit on a single DNA molecule as autonomous dissipative nanodevice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558768" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558768" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Greiss, F.; Lardon, N.; Schutz, L.; Barak, Y.; Daube, S. S.; Weinhold, E.; Noireaux, V.; Bar-Ziv, R.</p>
        <p class="info">Score: 5.0, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558768' target='https://doi.org/10.1101/2023.09.21.558768'> 10.1101/2023.09.21.558768</a></p>
        <p class="abstract">Realizing genetic circuits on single DNA molecules as self-encoded dissipative nanodevices is a major step toward miniaturization of autonomous biological systems. A circuit operating on a single DNA implies that genetically encoded proteins localize during coupled transcription-translation to DNA, but a single-molecule measurement demonstrating this has remained a challenge. Here, we used a genetically encoded fluorescent reporter system with improved spatiotemporal resolution and observed the synthesis of individual proteins tethered to a DNA molecule by transient complexes of RNA polymerase, messenger RNA, and ribosome. Against expectations in dilute cell-free conditions where equilibrium considerations favor dispersion, these nascent proteins linger long enough to regulate cascaded reactions on the same DNA. We rationally designed a pulsatile genetic circuit by encoding an activator and repressor in feedback on the same DNA molecule. Driven by the local synthesis of only several proteins per hour and gene, the circuit dynamics exhibited enhanced variability between individual DNA molecules, and fluctuations with a broad power spectrum. Our results demonstrate that co-expressional localization, as a nonequilibrium process, facilitates single-DNA genetic circuits as dissipative nanodevices, with implications for nanobiotechnology applications and artificial cell design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558311">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558311" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558311">
        <p class="paperTitle">Expanding the toolbox for phycobiliprotein assembly: phycoerythrobilin biosynthesis in Synechocystis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558311" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558311" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heck, S.; Sommer, F. K.; Zehner, S.; Schroda, M.; Gehringer, M. M.; Frankenberg-Dinkel, N.</p>
        <p class="info">Score: 4.7, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558311' target='https://doi.org/10.1101/2023.09.18.558311'> 10.1101/2023.09.18.558311</a></p>
        <p class="abstract">Phycobiliproteins (PBPs) play a vital role in light harvesting by cyanobacteria, which enables efficient utilization of photon energy for oxygenic photosynthesis. The PBPs carry phycobilins, open-chain tetrapyrrole chromophores derived from heme. The structure and chromophore composition of PBPs is dependent on the organisms ecological niche. In cyanobacteria, these holo-proteins typically form large, macromolecular antenna complexes called phycobilisomes (PBSs). The PBS of Synechocystis sp. PCC 6803 (hereafter Synechocystis) consists of allophycocyanin (APC) and phycocyanin (PC), which exclusively harbor phycocyanobilin (PCB) as a chromophore. Investigations into heterologous PBP biosynthesis in E. coli have proven limiting with respect to PBP assembly and their functional characterization. Consequently, we wanted to engineer a platform for the investigation of heterologously produced PBPs, focusing on unusual, phycoerythrobilin (PEB)-containing light-harvesting proteins called phycoerythrins (PEs) in Synechocystis. As a first step, a gene encoding for the synthesis of the natural cyanobacterial chromophore, PEB, was introduced into Synechocystis. We provide spectroscopic evidence for heterologous PEB formation and show covalent attachment of PEB to the -subunit of PC, CpcA, by HPLC and LC-MS/MS analyses. Fluorescence microscopy and PBS isolation demonstrate a cellular dispersal of PBPs with modified phycobilin content. However, these modifications have minor effects on physiological responses, as demonstrated by growth rates, oxygen evolution, nutrient accumulation, and PBP content analyses. As a result, Synechocystis demonstrates the capacity to efficiently manage PEB biosynthesis and therefore reflects a promising platform for both biochemical and physiological investigations of foreign and unusual PEs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.17.558058">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.17.558058" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.17.558058">
        <p class="paperTitle">Using the E. coli Alleleome in Strain Design</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.17.558058" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.17.558058" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Phaneuf, P. V.; Jarczynska, Z. D.; Kandasamy, V.; Chauhan, S. M.; Feist, A. M.; Palsson, B. O.</p>
        <p class="info">Score: 4.3, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.17.558058' target='https://doi.org/10.1101/2023.09.17.558058'> 10.1101/2023.09.17.558058</a></p>
        <p class="abstract">Leveraging observed variants in strain design is a promising technique for creating strains with specific properties. Adaptive laboratory evolution (ALE) experiments generate variants that enhance fitness under specific conditions and can contribute to application-specific strain designs. Further, the wild-type (WT) coding alleleome of an organism, the complete set of its genes WT alleles, can provide an additional amount and diversity of variants not yet accessible from the aggregation of ALE experiment results. This study used both an ALE mutation database (3093 genomes) and a large set of WT genomes (12,661 genomes) to explore the solution space of genes involved in tolerance to 10 conditions of industrial importance. To accomplish this, ALE variants for 22 genes previously identified as potentially important for industrial chemical tolerance were collected and supplemented with all available variants from the WT coding alleleome. A total of 4879 variants were reintroduced and used in 10 selection experiments. Although ALE variants were generally found to be more advantageous, specific WT variants were more highly enriched than ALE variants in certain conditions. Additionally, ALE and WT variants rarely overlapped on AA positions, but their clustering did coincide with where highly enriched variants were ultimately located. For genes primarily hosting potential gain-of-function variations, substitutions predicted to have a conservative impact frequently outperformed more radical substitutions. Case studies demonstrated that maximizing the amount of variants enabled easier identification of variant trends, which in turn can be used to better understand areas and characteristics of genes that can be feasibly varied, resulting in what can be thought of as a genome design variable. The combination of ALE and WT variants is a promising approach for use in future projects to better constrain and ultimately achieve practical coverage in the exploration of feasible sequence solution space.

Visual Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=64 SRC=&#34;FIGDIR/small/558058v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (23K):
org.highwire.dtl.DTLVardef@17dd15dorg.highwire.dtl.DTLVardef@13c3cb7org.highwire.dtl.DTLVardef@1b635e3org.highwire.dtl.DTLVardef@18c0675_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557505">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557505" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557505">
        <p class="paperTitle">IMPROVING ANTIBODY AFFINITY USING LABORATORYDATA WITH LANGUAGE MODEL GUIDED DESIGN</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557505" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557505" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krause, B.; Subramanian, S.; Yuan, T.; Yang, M.; Sato, A.; Naik, N.</p>
        <p class="info">Score: 20.1, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557505' target='https://doi.org/10.1101/2023.09.13.557505'> 10.1101/2023.09.13.557505</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWProtein design requires searching exponentially large spaces of possible sequences to find candidates with desired properties. Language models (LMs) pretrained on universal protein datasets have shown potential to help make this search space tractable. However, LMs trained on natural sequences alone have limitations in designing proteins with novel functions, which is especially important for many pharmaceutical applications. In this work, we used a combination of methods to finetune pretrained LMs on laboratory data collected in an anti-CD40L antibody library campaign to develop an ensemble scoring function to model the fitness landscape and guide the design of new antibodies. Laboratory testing showed that the designed antibodies had improved affinity to CD40L. Notably, the designs improved the affinities of four antibodies, originally ranging from 1 nanomolar to 100 picomolar, all to below 25 picomolar, approaching the limit of detection. This work is a promising step towards realizing the potential of LMs to leverage laboratory data to develop improved treatments for diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.07.13.499906">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.07.13.499906" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.07.13.499906">
        <p class="paperTitle">High-throughput feedback-enabled optogenetic stimulation and spectroscopy in microwell plates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.07.13.499906" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.07.13.499906" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benman, W.; Datta, S.; Gonzalez-Martinez, D.; Lee, G.; Hooper, J.; Qian, G.; Leavitt, G.; Salloum, L.; Ho, G.; Mhatre, S.; Magaraci, M. S.; Patterson, M.; Mannickarottu, S. G.; Malani, S.; Avalos, J. L.; Chow, B. Y.; Bugaj, L. J.</p>
        <p class="info">Score: 25.0, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.07.13.499906' target='https://doi.org/10.1101/2022.07.13.499906'> 10.1101/2022.07.13.499906</a></p>
        <p class="abstract">The ability to perform sophisticated, high-throughput optogenetic experiments has been greatly enhanced by recent open-source illumination devices that allow independent programming of light patterns in single wells of microwell plates. However, there is currently a lack of instrumentation to monitor such experiments in real time, necessitating repeated transfers of the samples to stand-alone analytical instruments, thus limiting the types of experiments that could be performed. Here we address this gap with the development of the optoPlateReader (oPR), an open-source, solid-state, compact device that allows automated optogenetic stimulation and spectroscopy in each well of a 96-well plate. The oPR integrates an optoPlate illumination module with a module called the optoReader, an array of 96 photodiodes and LEDs that allows 96 parallel light measurements. The oPR was optimized for stimulation with blue light and for measurements of optical density and fluorescence. After calibration of all device components, we used the oPR to measure growth and to induce and measure fluorescent protein expression in E. coli. We further demonstrated how the optical read/write capabilities of the oPR permit computer-in-the-loop feedback control, where the current state of the sample can be used to adjust the optical stimulation parameters of the sample according to pre-defined feedback algorithms. The oPR will thus help realize an untapped potential for optogenetic experiments by enabling automated reading, writing, and feedback in microwell plates through open-source hardware that is accessible, customizable, and inexpensive.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555279">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555279" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555279">
        <p class="paperTitle">Escherichia coli Microcin B17 as a chassis for the development of novel antimicrobial peptides.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555279" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555279" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Withanage, S.; Masschelein, J.; Pinheiro, V. B.</p>
        <p class="info">Score: 8.6, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555279' target='https://doi.org/10.1101/2023.08.29.555279'> 10.1101/2023.08.29.555279</a></p>
        <p class="abstract">Antimicrobial resistance remains an urgent and growing global health problem. Peptides with antimicrobial activity are widespread in nature but are rarely harnessed for application, making them an underexplored and untapped resource. Many harbor post-translational modifications, which are essential for function and have evolved alongside specialized export and immunity machineries - leading to the current paradigm that natural sequences cannot be changed, and curtailing their development as antimicrobial agents. Nonetheless, we found sufficient ambiguity in literature to question if that remained true for E. coli MccB17. We demonstrate that its natural machinery for synthesis, export and immunity, can be co-opted as a platform for novel azole-modified antimicrobial peptide discovery. We show that the sequence space available for exploration is large and densely populated with functional sequences.

One-Sentence SummaryE. coli can be harnessed as an in vivo platform for the isolation of novel antimicrobial azole-modified peptides.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
