<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A discovery platform to identify inducible synthetic circuitry from varied microbial sources
Authors: Robinson, C. M.; Carreno, D.; Weber, T.; Chen, Y.; Riglar, D. T.
Score: 15.8, Published: 2023-10-13 DOI: 10.1101/2023.10.13.562223
Gut-resident microbes encode a variety of systems capable of sophisticated sensing and conditional gene expression control within the mammalian gut during health and disease. Bacterial synthetic biology approaches, such as whole-cell biosensing and &#39;sense-and-respond&#39; therapeutics, aim to re-purpose such systems for research and clinical applications; however, the toolbox of well characterised in vivo and disease-related inducible systems remains limited.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="A discovery platform to identify inducible synthetic circuitry from varied microbial sources
Authors: Robinson, C. M.; Carreno, D.; Weber, T.; Chen, Y.; Riglar, D. T.
Score: 15.8, Published: 2023-10-13 DOI: 10.1101/2023.10.13.562223
Gut-resident microbes encode a variety of systems capable of sophisticated sensing and conditional gene expression control within the mammalian gut during health and disease. Bacterial synthetic biology approaches, such as whole-cell biosensing and &#39;sense-and-respond&#39; therapeutics, aim to re-purpose such systems for research and clinical applications; however, the toolbox of well characterised in vivo and disease-related inducible systems remains limited." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-18T10:38:41+00:00" />
<meta property="article:modified_time" content="2023-10-18T10:38:41+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="A discovery platform to identify inducible synthetic circuitry from varied microbial sources
Authors: Robinson, C. M.; Carreno, D.; Weber, T.; Chen, Y.; Riglar, D. T.
Score: 15.8, Published: 2023-10-13 DOI: 10.1101/2023.10.13.562223
Gut-resident microbes encode a variety of systems capable of sophisticated sensing and conditional gene expression control within the mammalian gut during health and disease. Bacterial synthetic biology approaches, such as whole-cell biosensing and &#39;sense-and-respond&#39; therapeutics, aim to re-purpose such systems for research and clinical applications; however, the toolbox of well characterised in vivo and disease-related inducible systems remains limited."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "A discovery platform to identify inducible synthetic circuitry from varied microbial sources\nAuthors: Robinson, C. M.; Carreno, D.; Weber, T.; Chen, Y.; Riglar, D. T.\nScore: 15.8, Published: 2023-10-13 DOI: 10.1101/2023.10.13.562223\nGut-resident microbes encode a variety of systems capable of sophisticated sensing and conditional gene expression control within the mammalian gut during health and disease. Bacterial synthetic biology approaches, such as whole-cell biosensing and \u0026#39;sense-and-respond\u0026#39; therapeutics, aim to re-purpose such systems for research and clinical applications; however, the toolbox of well characterised in vivo and disease-related inducible systems remains limited.",
  "keywords": [
    
  ],
  "articleBody": " A discovery platform to identify inducible synthetic circuitry from varied microbial sources\nAuthors: Robinson, C. M.; Carreno, D.; Weber, T.; Chen, Y.; Riglar, D. T.\nScore: 15.8, Published: 2023-10-13 DOI: 10.1101/2023.10.13.562223\nGut-resident microbes encode a variety of systems capable of sophisticated sensing and conditional gene expression control within the mammalian gut during health and disease. Bacterial synthetic biology approaches, such as whole-cell biosensing and 'sense-and-respond' therapeutics, aim to re-purpose such systems for research and clinical applications; however, the toolbox of well characterised in vivo and disease-related inducible systems remains limited. Here, we dramatically expand the flexibility and power of a high-throughput approach to biosensor evaluation using bacterial memory circuits encoded in a gut commensal E. coli species. We construct a novel library of bacterial two-component system (TCS) based biosensors sourced from gut pathogens, tagged with unique DNA barcodes using a single-pot construction method. Using our pipeline, we evaluate sensor activity and performance heterogeneity across in vitro and in vivo conditions leading to novel sensor discovery. In the process, we optimise library construction, sample quality thresholds, and parameters for engineered bacterial biosensor screening in the murine gut. This approach can be applied to transcriptionally activated sensing elements of any type and will allow rapid development of new biosensors that can advance synthetic biology approaches for complex environments, such as the gut.\nBacteriophage-host interactions in microgravity onboard the International Space Station\nAuthors: Huss, P.; Chitboonthavisuk, C.; Meger, A.; Nishikawa, K.; Oates, R. P.; Mills, H.; Holzhaus, O.; Raman, S.\nScore: 12.3, Published: 2023-10-10 DOI: 10.1101/2023.10.10.561409\nBacteriophage-host interactions play a fundamental role in shaping microbial ecosystems. Although phage-host interactions have been extensively studied in terrestrial ecosystems, the impact of microgravity on phage-host interactions is yet to be investigated. Here, we report the dynamics of interactions between T7 bacteriophage and E. coli in microgravity onboard the International Space Station (ISS). We found phage activity was delayed but ultimately successful in microgravity. We identified several de novo mutations in phage and bacteria that improved fitness in microgravity. Deep mutational scanning of the receptor binding domain revealed substantial differences in the number, position, and mutational preferences between gravity and microgravity, reflecting underlying differences in the bacterial adaptations. Combinatorial libraries informed by microgravity selections gave T7 mutants with 100-10,000-fold higher activity on uropathogenic E. coli under terrestrial conditions than wildtype T7. Our findings lay the foundation for future research on the impact of microgravity on phage-host interactions and microbial communities.\nAn antimicrobial peptide expression platform for targeting pathogenic bacterial species\nAuthors: Rutter, J. W.; Dekker, L.; Clare, C.; Fedorec, A. J.; Barnes, C. P.\nScore: 13.5, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561505\nBacteriocins are antimicrobial peptides that are naturally produced by many bacteria. They hold great potential in the fight against antibiotic resistant bacteria, including ESKAPE pathogens. However, they often have low stability in vivo and therefore, may not be effective when administered orally. Engineered live biotherapeutic products (eLBPs) that secrete bacteriocins can be created to deliver bacteriocins directly at the site of infection. Here we develop a modular bacteriocin secretion platform that can be used to express and secrete multiple bacteriocins from non-pathogenic Escherichia coli host strains. As a proof of concept we create Enterocin A and Enterocin B secreting strains that show strong antimicrobial activity against Enterococcus faecalis and Enterococcus faecium, and characterise this activity in both solid culture and liquid co-culture. We then develop a Lokta-Volterra model that can be used to capture the interactions of these competitor strains. We show that simultaneous exposure to EntA and EntB can increase the level of Enterococcus killing and delay the development of resistance. Our system has the potential to be used as an eLBP to secrete additional bacteriocins for the targeted killing of other pathogenic bacteria.\nArrayed in vivo barcoding for multiplexed sequence verification of plasmid DNA and demultiplexing of pooled libraries\nAuthors: Li, W.; Miller, D.; Liu, X.; Tosi, L.; Chkaiban, L.; Mei, H.; Hung, P.-H.; Parekkadan, B.; Sherlock, G.; Levy, S. F.\nScore: 10.4, Published: 2023-10-13 DOI: 10.1101/2023.10.13.562064\nSequence verification of plasmid DNA is critical for many cloning and molecular biology workflows. To leverage high-throughput sequencing, several methods have been developed that add a unique DNA barcode to individual samples prior to pooling and sequencing. However, these methods require an individual plasmid extraction and/or in vitro barcoding reaction for each sample processed, limiting throughput and adding cost. Here, we develop an arrayed in vivo plasmid barcoding platform that enables pooled plasmid extraction and library preparation for Oxford Nanopore sequencing. This method has a high accuracy and recovery rate, and greatly increases throughput and reduces cost relative to other plasmid barcoding methods or Sanger sequencing. We use in vivo barcoding to sequence verify \u003e45,000 plasmids and show that the method can be used to transform error-containing dispersed plasmid pools into sequence-perfect arrays or well-balanced pools. In vivo barcoding does not require any specialized equipment beyond a low-overhead Oxford Nanopore sequencer, enabling most labs to flexibly process hundreds to thousands of plasmids in parallel.\nCRISPR-Cas tools for simultaneous transcription \u0026 translation control in bacteria\nAuthors: Cardiff, R.; Faulkner, I.; Beall, J.; Carothers, J. M.; Zalatan, J. G.\nScore: 10.4, Published: 2023-10-12 DOI: 10.1101/2023.10.11.561958\nRobust control over gene translation at arbitrary mRNA targets is an outstanding challenge in microbial synthetic biology. The development of tools that can regulate translation will greatly expand our ability to precisely control genes across the genome. In E. coli, most genes are contained in multi-gene operons, which are subject to polar effects where targeting one gene for repression leads to silencing of both genes. These effects pose a challenge for independently regulating individual genes in multi-gene operons. Here, we use CRISPR-dCas13 to address this challenge. We find that dCas13-mediated repression exhibits up to 6-fold lower polar effects compared to dCas9. We then show that we can selectively activate single genes in a synthetic multi-gene operon by coupling dCas9 transcriptional activation of an operon with dCas13 translational repression of individual genes within the operon. We also show that dCas13 and dCas9 can be multiplexed for improved biosynthesis of a medically-relevant human milk oligosaccharide. Taken together, our findings suggest that combining transcriptional and translational control can access effects that are difficult to achieve with either mode independently. These combined tools for gene regulation will expand our abilities to precisely engineer bacteria for biotechnology and perform systematic genetic screens. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=102 SRC=\"FIGDIR/small/561958v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (26K): org.highwire.dtl.DTLVardef@f407baorg.highwire.dtl.DTLVardef@1275b64org.highwire.dtl.DTLVardef@e127c1org.highwire.dtl.DTLVardef@1f90c0c_HPS_FORMAT_FIGEXP M_FIG C_FIG\nFluorescent minihelix assay for translation quantification in high-throughput cell-free systems\nAuthors: Willi, J. A.; Jewett, M. C.\nScore: 6.3, Published: 2023-10-13 DOI: 10.1101/2023.10.13.562088\nHere we report the use of an optimized tetra-cysteine minihelix both as a fusion protein and as a standalone reporter in vitro with the Flash dye to study cell-free protein expression dynamics and engineered ribosome activity. The fluorescent product can be detected and quantified via its characteristic emission spectrum in a standard 96/384-well plate reader, RT-qPCR system, or gel electrophoresis. The fluorescent reporter helix is short enough to be encoded on a primer pair and can be added to a protein of interest via a simple PCR. The thus tagged protein can be detected in real time during or at the end of cell-free protein syntesis reactions, or in gel, without the need for staining. The fluorescent signal is stable and linearly correlates with protein concentration, thus suitable to quantify products of interest. Finally, we demonstrate how this reporter can be used for future efforts in engineering in vitro translation systems.\nAllosteric regulation of a molecular motor through de novo protein design\nAuthors: Cross, J. A.; Dawson, W. M.; Shulka, S. R.; Weijman, J. F.; Mantell, J.; Dodding, M. P.; Woolfson, D. N.\nScore: 4.7, Published: 2023-10-17 DOI: 10.1101/2023.10.17.562760\nMany enzymes are allosterically regulated. Our ability to manipulate these structural changes is limited. Here we install an allosteric switch into the kinesin-1 microtubule motor in vitro and in cells. Kinesin-1 is a heterotetramer that accesses open active and closed auto-inhibited states. The equilibrium between these centres on a flexible elbow within a complex coiled-coil architecture. We target the elbow to engineer a closed state that can be opened with a de novo designed peptide. The alternative states are modelled computationally and confirmed by biophysical measurements and electron microscopy. In cells, peptide-driven activation increases kinesin transport, demonstrating a primary role for conformational switching in regulating motor activity. The designs are enabled by our understanding of ubiquitous coiled-coil structures, opening possibilities for controlling other protein activities.\nReprogramming megakaryocytes to engineer platelets as delivery vehicles\nAuthors: Javdan, S. B.; Islam, F.; Deans, T. L.\nScore: 4.2, Published: 2023-10-14 DOI: 10.1101/2023.10.13.562311\nWe developed a method for the efficient generation of engineered platelets that can be filled with any recombinant therapeutic protein during the differentiation process by reprogramming megakaryocytes, the progenitor cells of platelets. To demonstrate the versatility of this approach, we loaded cytoplasmic and secreted proteins that can be delivered as active enzymes to recipient cells, be released upon platelet activation, or be continuously secreted by platelets over time.\nFrom Context to Code: Rational De Novo DNA Design and Predicting Cross-Species DNA Functionality Using Deep Learning Transformer Models\nAuthors: Dahiya, G. S.; Bakken, T. I.; Fages-Lartaud, M.; Lale, R.\nScore: 4.0, Published: 2023-10-15 DOI: 10.1101/2023.10.15.562386\nSynthetic biology currently operates under a framework dominated by trial-and-error approaches, which hinders the effective engineering of organisms and the expansion of large-scale biomanufacturing. Motivated by the success of computational designs in areas like architecture and aeronautics, we aspire to transition to a more efficient and predictive methodology in synthetic biology. In this study, we report a DNA Design Platform that relies on the predictive power of Transformer-based deep learning architectures. The platform transforms the conventional paradigms in synthetic biology by enabling the context-sensitive and host-specific engineering of 5' regulatory elements--promoters and 5' untranslated regions (UTRs)--along with an array of codon-optimised coding sequence (CDS) variants. This allows us to generate context-sensitive 5' regulatory sequences and CDSs, achieving an unparalleled level of specificity and adaptability in different target hosts. With context-aware design, we significantly broaden the range of possible gene expression profiles and phenotypic outcomes, substantially reducing the need for laborious high-throughput screening efforts. Our context-aware, AI-driven design strategy marks a significant advancement in synthetic biology, offering a scalable and refined approach for gene expression optimisation across a diverse range of expression hosts. In summary, this study represents a substantial leap forward in the field, utilising deep learning models to transform the conventional design, build, test, learn-cycle into a more efficient and predictive framework.\nA green alternative to fragrant agarwood sesquiterpenoid production\nAuthors: Gutierrez, S.; Overmans, S.; Wellman, G. B.; Samaras, V. G.; Oviedo, C.; Gede, M.; Szekely, G.; Lauersen, K. J.\nScore: 9.0, Published: 2023-10-07 DOI: 10.1101/2023.10.06.561217\nCertain endangered Thymelaeaceous trees are major sources of the fragrant and highly valued resinous agarwood, comprised of hundreds of oxygenated sesquiterpenoids (STPs). Despite growing pressure on natural agarwood sources, the chemical complexity of STPs severely limits synthetic production. Here, we catalogued the chemical diversity in 58 agarwood samples by two--dimensional gas chromatography--mass spectrometry and partially recreated complex STP mixtures through synthetic biology. We improved STP yields in the unicellular alga Chlamydomonas reinhardtii by combinatorial engineering to biosynthesise nine macrocyclic STP backbones found in agarwood. A bioprocess following green-chemistry principles was developed that exploits {per thousand}milking{per thousand} of STPs without cell lysis, solvent--solvent STP extraction, solvent--STP nanofiltration, and bulk STP oxy--functionalisation to obtain terpene mixtures like those of agarwood. This process occurs with total solvent recycling and enables continuous production. Our synthetic-biology approach offers a sustainable alternative to harvesting agarwood trees to obtain mixtures of complex, fragrant, oxygenated STPs.\n",
  "wordCount" : "1927",
  "inLanguage": "en",
  "datePublished": "2023-10-18T10:38:41Z",
  "dateModified": "2023-10-18T10:38:41Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on October 18, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562223">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562223" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562223">
        <p class="paperTitle">A discovery platform to identify inducible synthetic circuitry from varied microbial sources</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562223" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562223" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Robinson, C. M.; Carreno, D.; Weber, T.; Chen, Y.; Riglar, D. T.</p>
        <p class="info">Score: 15.8, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562223' target='https://doi.org/10.1101/2023.10.13.562223'> 10.1101/2023.10.13.562223</a></p>
        <p class="abstract">Gut-resident microbes encode a variety of systems capable of sophisticated sensing and conditional gene expression control within the mammalian gut during health and disease. Bacterial synthetic biology approaches, such as whole-cell biosensing and &#39;sense-and-respond&#39; therapeutics, aim to re-purpose such systems for research and clinical applications; however, the toolbox of well characterised in vivo and disease-related inducible systems remains limited. Here, we dramatically expand the flexibility and power of a high-throughput approach to biosensor evaluation using bacterial memory circuits encoded in a gut commensal E. coli species. We construct a novel library of bacterial two-component system (TCS) based biosensors sourced from gut pathogens, tagged with unique DNA barcodes using a single-pot construction method. Using our pipeline, we evaluate sensor activity and performance heterogeneity across in vitro and in vivo conditions leading to novel sensor discovery. In the process, we optimise library construction, sample quality thresholds, and parameters for engineered bacterial biosensor screening in the murine gut. This approach can be applied to transcriptionally activated sensing elements of any type and will allow rapid development of new biosensors that can advance synthetic biology approaches for complex environments, such as the gut.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561409">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561409" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561409">
        <p class="paperTitle">Bacteriophage-host interactions in microgravity onboard the International Space Station</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561409" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561409" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huss, P.; Chitboonthavisuk, C.; Meger, A.; Nishikawa, K.; Oates, R. P.; Mills, H.; Holzhaus, O.; Raman, S.</p>
        <p class="info">Score: 12.3, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561409' target='https://doi.org/10.1101/2023.10.10.561409'> 10.1101/2023.10.10.561409</a></p>
        <p class="abstract">Bacteriophage-host interactions play a fundamental role in shaping microbial ecosystems. Although phage-host interactions have been extensively studied in terrestrial ecosystems, the impact of microgravity on phage-host interactions is yet to be investigated. Here, we report the dynamics of interactions between T7 bacteriophage and E. coli in microgravity onboard the International Space Station (ISS). We found phage activity was delayed but ultimately successful in microgravity. We identified several de novo mutations in phage and bacteria that improved fitness in microgravity. Deep mutational scanning of the receptor binding domain revealed substantial differences in the number, position, and mutational preferences between gravity and microgravity, reflecting underlying differences in the bacterial adaptations. Combinatorial libraries informed by microgravity selections gave T7 mutants with 100-10,000-fold higher activity on uropathogenic E. coli under terrestrial conditions than wildtype T7. Our findings lay the foundation for future research on the impact of microgravity on phage-host interactions and microbial communities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561505">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561505" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561505">
        <p class="paperTitle">An antimicrobial peptide expression platform for targeting pathogenic bacterial species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561505" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561505" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rutter, J. W.; Dekker, L.; Clare, C.; Fedorec, A. J.; Barnes, C. P.</p>
        <p class="info">Score: 13.5, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561505' target='https://doi.org/10.1101/2023.10.09.561505'> 10.1101/2023.10.09.561505</a></p>
        <p class="abstract">Bacteriocins are antimicrobial peptides that are naturally produced by many bacteria. They hold great potential in the fight against antibiotic resistant bacteria, including ESKAPE pathogens. However, they often have low stability in vivo and therefore, may not be effective when administered orally. Engineered live biotherapeutic products (eLBPs) that secrete bacteriocins can be created to deliver bacteriocins directly at the site of infection. Here we develop a modular bacteriocin secretion platform that can be used to express and secrete multiple bacteriocins from non-pathogenic Escherichia coli host strains. As a proof of concept we create Enterocin A and Enterocin B secreting strains that show strong antimicrobial activity against Enterococcus faecalis and Enterococcus faecium, and characterise this activity in both solid culture and liquid co-culture. We then develop a Lokta-Volterra model that can be used to capture the interactions of these competitor strains. We show that simultaneous exposure to EntA and EntB can increase the level of Enterococcus killing and delay the development of resistance. Our system has the potential to be used as an eLBP to secrete additional bacteriocins for the targeted killing of other pathogenic bacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562064">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562064" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562064">
        <p class="paperTitle">Arrayed in vivo barcoding for multiplexed sequence verification of plasmid DNA and demultiplexing of pooled libraries</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562064" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562064" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, W.; Miller, D.; Liu, X.; Tosi, L.; Chkaiban, L.; Mei, H.; Hung, P.-H.; Parekkadan, B.; Sherlock, G.; Levy, S. F.</p>
        <p class="info">Score: 10.4, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562064' target='https://doi.org/10.1101/2023.10.13.562064'> 10.1101/2023.10.13.562064</a></p>
        <p class="abstract">Sequence verification of plasmid DNA is critical for many cloning and molecular biology workflows. To leverage high-throughput sequencing, several methods have been developed that add a unique DNA barcode to individual samples prior to pooling and sequencing. However, these methods require an individual plasmid extraction and/or in vitro barcoding reaction for each sample processed, limiting throughput and adding cost. Here, we develop an arrayed in vivo plasmid barcoding platform that enables pooled plasmid extraction and library preparation for Oxford Nanopore sequencing. This method has a high accuracy and recovery rate, and greatly increases throughput and reduces cost relative to other plasmid barcoding methods or Sanger sequencing. We use in vivo barcoding to sequence verify &gt;45,000 plasmids and show that the method can be used to transform error-containing dispersed plasmid pools into sequence-perfect arrays or well-balanced pools. In vivo barcoding does not require any specialized equipment beyond a low-overhead Oxford Nanopore sequencer, enabling most labs to flexibly process hundreds to thousands of plasmids in parallel.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561958">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561958" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561958">
        <p class="paperTitle">CRISPR-Cas tools for simultaneous transcription &amp; translation control in bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561958" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561958" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cardiff, R.; Faulkner, I.; Beall, J.; Carothers, J. M.; Zalatan, J. G.</p>
        <p class="info">Score: 10.4, Published: 2023-10-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561958' target='https://doi.org/10.1101/2023.10.11.561958'> 10.1101/2023.10.11.561958</a></p>
        <p class="abstract">Robust control over gene translation at arbitrary mRNA targets is an outstanding challenge in microbial synthetic biology. The development of tools that can regulate translation will greatly expand our ability to precisely control genes across the genome. In E. coli, most genes are contained in multi-gene operons, which are subject to polar effects where targeting one gene for repression leads to silencing of both genes. These effects pose a challenge for independently regulating individual genes in multi-gene operons. Here, we use CRISPR-dCas13 to address this challenge. We find that dCas13-mediated repression exhibits up to 6-fold lower polar effects compared to dCas9. We then show that we can selectively activate single genes in a synthetic multi-gene operon by coupling dCas9 transcriptional activation of an operon with dCas13 translational repression of individual genes within the operon. We also show that dCas13 and dCas9 can be multiplexed for improved biosynthesis of a medically-relevant human milk oligosaccharide. Taken together, our findings suggest that combining transcriptional and translational control can access effects that are difficult to achieve with either mode independently. These combined tools for gene regulation will expand our abilities to precisely engineer bacteria for biotechnology and perform systematic genetic screens.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=102 SRC=&#34;FIGDIR/small/561958v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (26K):
org.highwire.dtl.DTLVardef@f407baorg.highwire.dtl.DTLVardef@1275b64org.highwire.dtl.DTLVardef@e127c1org.highwire.dtl.DTLVardef@1f90c0c_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562088">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562088" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562088">
        <p class="paperTitle">Fluorescent minihelix assay for translation quantification in high-throughput cell-free systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562088" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562088" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Willi, J. A.; Jewett, M. C.</p>
        <p class="info">Score: 6.3, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562088' target='https://doi.org/10.1101/2023.10.13.562088'> 10.1101/2023.10.13.562088</a></p>
        <p class="abstract">Here we report the use of an optimized tetra-cysteine minihelix both as a fusion protein and as a standalone reporter in vitro with the Flash dye to study cell-free protein expression dynamics and engineered ribosome activity. The fluorescent product can be detected and quantified via its characteristic emission spectrum in a standard 96/384-well plate reader, RT-qPCR system, or gel electrophoresis. The fluorescent reporter helix is short enough to be encoded on a primer pair and can be added to a protein of interest via a simple PCR. The thus tagged protein can be detected in real time during or at the end of cell-free protein syntesis reactions, or in gel, without the need for staining. The fluorescent signal is stable and linearly correlates with protein concentration, thus suitable to quantify products of interest. Finally, we demonstrate how this reporter can be used for future efforts in engineering in vitro translation systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562760">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562760" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562760">
        <p class="paperTitle">Allosteric regulation of a molecular motor through de novo protein design</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562760" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562760" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cross, J. A.; Dawson, W. M.; Shulka, S. R.; Weijman, J. F.; Mantell, J.; Dodding, M. P.; Woolfson, D. N.</p>
        <p class="info">Score: 4.7, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562760' target='https://doi.org/10.1101/2023.10.17.562760'> 10.1101/2023.10.17.562760</a></p>
        <p class="abstract">Many enzymes are allosterically regulated. Our ability to manipulate these structural changes is limited. Here we install an allosteric switch into the kinesin-1 microtubule motor in vitro and in cells. Kinesin-1 is a heterotetramer that accesses open active and closed auto-inhibited states. The equilibrium between these centres on a flexible elbow within a complex coiled-coil architecture. We target the elbow to engineer a closed state that can be opened with a de novo designed peptide. The alternative states are modelled computationally and confirmed by biophysical measurements and electron microscopy. In cells, peptide-driven activation increases kinesin transport, demonstrating a primary role for conformational switching in regulating motor activity. The designs are enabled by our understanding of ubiquitous coiled-coil structures, opening possibilities for controlling other protein activities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562311">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562311" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562311">
        <p class="paperTitle">Reprogramming megakaryocytes to engineer platelets as delivery vehicles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562311" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562311" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Javdan, S. B.; Islam, F.; Deans, T. L.</p>
        <p class="info">Score: 4.2, Published: 2023-10-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562311' target='https://doi.org/10.1101/2023.10.13.562311'> 10.1101/2023.10.13.562311</a></p>
        <p class="abstract">We developed a method for the efficient generation of engineered platelets that can be filled with any recombinant therapeutic protein during the differentiation process by reprogramming megakaryocytes, the progenitor cells of platelets. To demonstrate the versatility of this approach, we loaded cytoplasmic and secreted proteins that can be delivered as active enzymes to recipient cells, be released upon platelet activation, or be continuously secreted by platelets over time.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.15.562386">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.15.562386" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.15.562386">
        <p class="paperTitle">From Context to Code: Rational De Novo DNA Design and Predicting Cross-Species DNA Functionality Using Deep Learning Transformer Models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.15.562386" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.15.562386" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dahiya, G. S.; Bakken, T. I.; Fages-Lartaud, M.; Lale, R.</p>
        <p class="info">Score: 4.0, Published: 2023-10-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.15.562386' target='https://doi.org/10.1101/2023.10.15.562386'> 10.1101/2023.10.15.562386</a></p>
        <p class="abstract">Synthetic biology currently operates under a framework dominated by trial-and-error approaches, which hinders the effective engineering of organisms and the expansion of large-scale biomanufacturing. Motivated by the success of computational designs in areas like architecture and aeronautics, we aspire to transition to a more efficient and predictive methodology in synthetic biology. In this study, we report a DNA Design Platform that relies on the predictive power of Transformer-based deep learning architectures. The platform transforms the conventional paradigms in synthetic biology by enabling the context-sensitive and host-specific engineering of 5&#39; regulatory elements--promoters and 5&#39; untranslated regions (UTRs)--along with an array of codon-optimised coding sequence (CDS) variants. This allows us to generate context-sensitive 5&#39; regulatory sequences and CDSs, achieving an unparalleled level of specificity and adaptability in different target hosts. With context-aware design, we significantly broaden the range of possible gene expression profiles and phenotypic outcomes, substantially reducing the need for laborious high-throughput screening efforts. Our context-aware, AI-driven design strategy marks a significant advancement in synthetic biology, offering a scalable and refined approach for gene expression optimisation across a diverse range of expression hosts. In summary, this study represents a substantial leap forward in the field, utilising deep learning models to transform the conventional design, build, test, learn-cycle into a more efficient and predictive framework.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561217">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561217" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561217">
        <p class="paperTitle">A green alternative to fragrant agarwood sesquiterpenoid production</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561217" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561217" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gutierrez, S.; Overmans, S.; Wellman, G. B.; Samaras, V. G.; Oviedo, C.; Gede, M.; Szekely, G.; Lauersen, K. J.</p>
        <p class="info">Score: 9.0, Published: 2023-10-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561217' target='https://doi.org/10.1101/2023.10.06.561217'> 10.1101/2023.10.06.561217</a></p>
        <p class="abstract">Certain endangered Thymelaeaceous trees are major sources of the fragrant and highly valued resinous agarwood, comprised of hundreds of oxygenated sesquiterpenoids (STPs). Despite growing pressure on natural agarwood sources, the chemical complexity of STPs severely limits synthetic production. Here, we catalogued the chemical diversity in 58 agarwood samples by two--dimensional gas chromatography--mass spectrometry and partially recreated complex STP mixtures through synthetic biology. We improved STP yields in the unicellular alga Chlamydomonas reinhardtii by combinatorial engineering to biosynthesise nine macrocyclic STP backbones found in agarwood. A bioprocess following green-chemistry principles was developed that exploits {per thousand}milking{per thousand} of STPs without cell lysis, solvent--solvent STP extraction, solvent--STP nanofiltration, and bulk STP oxy--functionalisation to obtain terpene mixtures like those of agarwood. This process occurs with total solvent recycling and enables continuous production. Our synthetic-biology approach offers a sustainable alternative to harvesting agarwood trees to obtain mixtures of complex, fragrant, oxygenated STPs.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
