<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Machine-guided design of synthetic cell type-specific cis-regulatory elements
Authors: Gosai, S. J.; Castro, R. I.; Fuentes, N.; Butts, J. C.; Kales, S.; Noche, R. R.; Mouri, K.; Sabeti, P. C.; Reilly, S. K.; Tewhey, R.
Score: 57.7, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552077
Cis-regulatory elements (CREs) control gene expression, orchestrating tissue identity, developmental timing, and stimulus responses, which collectively define the thousands of unique cell types in the body. While there is great potential for strategically incorporating CREs in therapeutic or biotechnology applications that require tissue specificity, there is no guarantee that an optimal CRE for an intended purpose has arisen naturally through evolution.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Machine-guided design of synthetic cell type-specific cis-regulatory elements
Authors: Gosai, S. J.; Castro, R. I.; Fuentes, N.; Butts, J. C.; Kales, S.; Noche, R. R.; Mouri, K.; Sabeti, P. C.; Reilly, S. K.; Tewhey, R.
Score: 57.7, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552077
Cis-regulatory elements (CREs) control gene expression, orchestrating tissue identity, developmental timing, and stimulus responses, which collectively define the thousands of unique cell types in the body. While there is great potential for strategically incorporating CREs in therapeutic or biotechnology applications that require tissue specificity, there is no guarantee that an optimal CRE for an intended purpose has arisen naturally through evolution." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-16T10:37:20+00:00" />
<meta property="article:modified_time" content="2023-08-16T10:37:20+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Machine-guided design of synthetic cell type-specific cis-regulatory elements
Authors: Gosai, S. J.; Castro, R. I.; Fuentes, N.; Butts, J. C.; Kales, S.; Noche, R. R.; Mouri, K.; Sabeti, P. C.; Reilly, S. K.; Tewhey, R.
Score: 57.7, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552077
Cis-regulatory elements (CREs) control gene expression, orchestrating tissue identity, developmental timing, and stimulus responses, which collectively define the thousands of unique cell types in the body. While there is great potential for strategically incorporating CREs in therapeutic or biotechnology applications that require tissue specificity, there is no guarantee that an optimal CRE for an intended purpose has arisen naturally through evolution."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Machine-guided design of synthetic cell type-specific cis-regulatory elements\nAuthors: Gosai, S. J.; Castro, R. I.; Fuentes, N.; Butts, J. C.; Kales, S.; Noche, R. R.; Mouri, K.; Sabeti, P. C.; Reilly, S. K.; Tewhey, R.\nScore: 57.7, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552077\nCis-regulatory elements (CREs) control gene expression, orchestrating tissue identity, developmental timing, and stimulus responses, which collectively define the thousands of unique cell types in the body. While there is great potential for strategically incorporating CREs in therapeutic or biotechnology applications that require tissue specificity, there is no guarantee that an optimal CRE for an intended purpose has arisen naturally through evolution.",
  "keywords": [
    
  ],
  "articleBody": " Machine-guided design of synthetic cell type-specific cis-regulatory elements\nAuthors: Gosai, S. J.; Castro, R. I.; Fuentes, N.; Butts, J. C.; Kales, S.; Noche, R. R.; Mouri, K.; Sabeti, P. C.; Reilly, S. K.; Tewhey, R.\nScore: 57.7, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552077\nCis-regulatory elements (CREs) control gene expression, orchestrating tissue identity, developmental timing, and stimulus responses, which collectively define the thousands of unique cell types in the body. While there is great potential for strategically incorporating CREs in therapeutic or biotechnology applications that require tissue specificity, there is no guarantee that an optimal CRE for an intended purpose has arisen naturally through evolution. Here, we present a platform to engineer and validate synthetic CREs capable of driving gene expression with programmed cell type specificity. We leverage innovations in deep neural network modeling of CRE activity across three cell types, efficient in silico optimization, and massively parallel reporter assays (MPRAs) to design and empirically test thousands of CREs. Through in vitro and in vivo validation, we show that synthetic sequences outperform natural sequences from the human genome in driving cell type-specific expression. Synthetic sequences leverage unique sequence syntax to promote activity in the on-target cell type and simultaneously reduce activity in off-target cells. Together, we provide a generalizable framework to prospectively engineer CREs and demonstrate the required literacy to write regulatory code that is fit-for-purpose in vivo across vertebrates.\nEfficient Natural Plasmid Transformation of Vibrio natriegens Enables Zero-capital Molecular Biology\nAuthors: Specht, D. A.; Sheppard, T. J.; Kennedy, F.; Li, S.; Gadikota, G.; Barstow, B.\nScore: 35.6, Published: 2023-08-13 DOI: 10.1101/2023.08.11.553013\nThe fast-growing microbe Vibrio natriegens is capable of natural transformation where it draws DNA in from media via an active process under physiological conditions. Using an engineered strain with a genomic copy of the master competence regulator tfoX from Vibrio cholera in combination with a new minimal competence media (MCM) that uses acetate as an energy source, we demonstrate naturally competent cells which are created, transformed, and recovered entirely in the same media, without exchange or addition of new media. Cells are naturally competent to plasmids, recombination with linear DNA, and co-transformation of both to select for scarless and markerless genomic edits. The entire process is simple and inexpensive, requiring no capital equipment for an entirely room temperature process (Zero Capital protocol, 10^4 cfu/g), or just an incubator (High Efficiency protocol, 10^5-6 cfu/g). These cells retain their naturally competent state when frozen and are transformable immediately upon thawing like a typical chemical or electrochemical competent cell. Since the optimized transformation protocol requires only 50 minutes of hands-on time, and V. natriegens grows quickly even on plates, a transformation started at 9 AM yields abundant culturable single colonies by 5 PM. Further, because all stages of transformation occur in the same media, and the process can be arbitrarily scaled in volume, this natural competence strain and media could be ideal for automated directed evolution applications. As a result, naturally competent V. natriegens could compete with E. coli as an excellent chassis for low-cost and highly scalable synthetic biology.\nA molecular proximity sensor based on an engineered, dual-component guide RNA\nAuthors: Choi, J.; Chen, W.; Liao, H.; Li, X.; Shendure, J.\nScore: 30.2, Published: 2023-08-14 DOI: 10.1101/2023.08.14.553235\nOne of the goals of synthetic biology is to enable the design of arbitrary molecular circuits with programmable inputs and outputs. Such circuits bridge the properties of electronic and natural circuits, processing information in a predictable manner within living cells. Genome editing is a potentially powerful component of synthetic molecular circuits, whether for modulating the expression of a target gene or for stably recording information to genomic DNA. However, programming molecular events such as protein-protein interactions or induced proximity as triggers for genome editing remains challenging. Here we demonstrate a strategy termed P3 editing, which links protein-protein proximity to the formation of a functional CRISPR-Cas9 dual-component guide RNA. By engineering the crRNA:tracrRNA interaction, we demonstrate that various known protein-protein interactions, as well as the chemically-induced dimerization of protein domains, can be used to activate prime editing or base editing in human cells. Additionally, we explore how P3 editing can incorporate outputs from ADAR-based RNA sensors, potentially allowing specific RNAs to induce specific genome edits within a larger circuit. Our strategy enhances the controllability of CRISPR-based genome editing, facilitating its use in synthetic molecular circuits deployed in living cells.\nIntern-based thermoregulated meganucleases for biocontainment of genetic material\nAuthors: Foo, G. W.; Leichthammer, C. D.; Saita, I. M.; Lukas, N. D.; Batko, I. Z.; Heinrichs, D. E.; Edgell, D.\nScore: 1.5, Published: 2023-08-14 DOI: 10.1101/2023.08.14.553307\nLimiting the spread of synthetic genetic information outside of the intended use is essential for applications where biocontainment is critical. In particular, biocontainment of engineered probiotics and plasmids that are excreted from the mammalian gastrointestinal tract is needed to prevent escape and acquisition of genetic material that could confer a selective advantage to microbial communities. Here, we built a simple and lightweight biocontainment system that post-translationally activates a site-specific DNA endonuclease to degrade DNA at 18{degrees}C and not at higher temperatures. We constructed an orthogonal set of temperature sensitive-meganucleases, or TSMs, by inserting the yeast VMA1 L212P temperature-sensitive intein into the coding regions of LAGLIDADG homing endonucleases. We showed that the TSMs eliminated plasmids carrying the cognate TSM target site from laboratory strains of Escherichia coli at the permissive 18{degrees}C but not at higher restrictive temperatures. Plasmid elimination is dependent on both TSM endonuclease activity and intein splicing. We demonstrated that TSMs eliminated plasmids from the E. coli Nissle 1917 strain after passage through the mouse gut when fecal resuspensions were incubated at 18{degrees}C but not at 37{degrees}C. Collectively, our data demonstrates the potential of thermoregulated meganucleases as a means of restricting engineered plasmids and probiotics to the mammalian gut.\nDesign of a Biohybrid Materials Circuit with Binary Decoder Functionality\nAuthors: Mohsenin, H.; Wagner, H.; Rosenblatt, M.; Kemmer, S.; Timmer, J.; Weber, W.\nScore: 1.2, Published: 2023-08-13 DOI: 10.1101/2023.08.10.552766\nSynthetic biology applies concepts from electrical engineering and information processing to endow cells with computational functionality. Transferring the underlying molecular components into materials and wiring them according to topologies inspired by electronic circuit boards has yielded materials systems that perform selected computational operations. However, the limited functionality of available building blocks is restricting the implementation of advanced information-processing circuits into materials. Here, we engineer a set of protease-based biohybrid modules the bioactivity of which can either be induced or inhibited. Guided by a quantitative mathematical model and following a design-build-test-learn cycle, we wire the modules according to circuit topologies inspired by electronic signal decoders, a fundamental motif in information processing. We design a 2-input/4-output binary decoder for the detection of two small molecules in a material framework that could perform regulated outputs in form of distinct protease activities. The here demonstrated smart material system is strongly modular and could be used for biomolecular information processing for example in advanced biosensing or drug delivery applications.\nEngineering a Synthetic Escherichia coli Coculture for Compartmentalized de novo Biosynthesis of Isobutyl Butyrate from Mixed Sugars\nAuthors: Seo, H.; Castro, G. I.; Trinh, C. T.\nScore: 1.0, Published: 2023-08-13 DOI: 10.1101/2023.08.11.553049\nShort-chain esters are versatile chemicals with use as flavors, fragrances, solvents, and fuels. The de novo ester biosynthesis consists of diverging and converging pathway submodules, which is challenging to engineer to achieve optimal metabolic fluxes and selective product synthesis. Compartmentalizing the pathway submodules into specialist cells that facilitate pathway modularization and labor division can present a promising solution. Here, we engineered a synthetic Escherichia coli coculture with the compartmentalized sugar utilization and ester biosynthesis pathways to produce isobutyl butyrate from a mixture of glucose and xylose. To compartmentalize the sugar-utilizing pathway submodules, we engineered a xylose-utilizing E. coli specialist that selectively consumes xylose over glucose and bypasses the carbon catabolite repression (CCR) while leveraging the native CCR machinery to activate a glucose-utilizing E. coli specialist. Upon compartmentalizing the isobutyl butyrate pathway submodules into these sugar-utilizing specialist cells, a robust synthetic coculture could be engineered to selectively produce isobutyl butyrate at a level of 392 mg/L, about 31-fold higher than the monoculture.\nEngineering the highly productive cyanobacterium Synechococcus sp. PCC 11901\nAuthors: Victoria, A. J.; Selao, T. T.; Moreno-Cabezuelo, J. A.; Mills, L. A.; Gale, G. A.; Lea-Smith, D. J.; McCormick, A. J.\nScore: 12.0, Published: 2023-08-07 DOI: 10.1101/2023.08.04.552009\nSynechococcus sp. PCC 11901 (PCC 11901) is a fast-growing marine cyanobacterial strain that has a capacity for sustained biomass accumulation to very high cell densities, comparable to that achieved by commercially relevant heterotrophic organisms. However, genetic tools to engineer PCC 11901 for biotechnology applications are limited. Here we describe a suite of tools based on the CyanoGate MoClo system to unlock the engineering potential of PCC 11901. First, we characterised neutral sites suitable for stable genomic integration that do not affect growth even at high cell densities. Second, we tested a suite of constitutive promoters, terminators, and inducible promoters including a 2,4-diacetylphloroglucinol (DAPG)-inducible PhlF repressor system, which has not previously been demonstrated in cyanobacteria and showed tight regulation and a 228-fold dynamic range of induction. Lastly, we developed a DAPG-inducible dCas9-based CRISPR interference (CRISPRi) system and a modular method to generate markerless mutants using CRISPR-Cas12a. Based on our findings, PCC 11901 is highly responsive to CRISPRi-based repression and showed high efficiencies for single insertion (31-81%) and multiplex double insertion (25%) genome editing with Cas12a. We envision that these tools will lay the foundations for the adoption of PCC 11901 as a robust model strain for engineering biology and green biotechnology. On sentence summaryGenetic parts were characterised in Synechococcus sp. PCC 11901, including a tightly regulated inducible promoter system, efficient CRISPRi and a novel markerless Cas12a genome editing approach.\nGas vesicle-blood interactions enhance ultrasound imaging contrast\nAuthors: Ling, B.; Ko, J. H.; Stordy, B.; Zhang, Y.; Didden, T. F.; Malounda, D.; Swift, M. B.; Chan, W. C. W.; Shapiro, M. G.\nScore: 11.9, Published: 2023-07-25 DOI: 10.1101/2023.07.24.550434\nGas vesicles (GVs) are genetically encoded, air-filled protein nanostructures of broad interest for biomedical research and clinical applications, acting as imaging and therapeutic agents for ultrasound, magnetic resonance, and optical techniques. However, the biomedical applications of GVs as a systemically injectable nanomaterial have been hindered by a lack of understanding of GVs interactions with blood components, which can significantly impact in vivo performance. Here, we investigate the dynamics of GVs in the bloodstream using a combination of ultrasound and optical imaging, surface functionalization, flow cytometry, and mass spectrometry. We find that erythrocytes and serum proteins bind to GVs and shape their acoustic response, circulation time, and immunogenicity. We show that by modifying the GV surface, we can alter these interactions and thereby modify GVs in vivo performance. These results provide critical insights for the development of GVs as agents for nanomedicine.\nLong duration environmental biosensing by recording analyte detection in DNA using recombinase memory\nAuthors: Kalvapalle, P.; Sridhar, S.; Silberg, J. J.; Stadler, L. B.\nScore: 0.8, Published: 2023-08-13 DOI: 10.1101/2023.08.10.552812\nMicrobial biosensors that convert environmental information into real-time visual outputs are limited in their sensing abilities in complex environments, such as soil and wastewater. Alternative reporter outputs are needed that stably record the presence of analytes. Here, we test the performance of recombinase-memory biosensors that sense a sugar (arabinose) and a microbial communication molecule (3-oxo-C12- homoserine lactone) over 8 days (~70 generations) following analyte exposure. These biosensors use analyte sensing to trigger the expression of a recombinase which flips a segment of DNA, creating a genetic memory, and initiates fluorescent protein expression. The initial designs failed over time due to unintended DNA flipping in the absence of the analyte and loss of the flipped state after exposure to the analyte. Biosensor performance was improved by decreasing recombinase expression, removing the fluorescent protein output, and using qPCR to read out stored information. Application of memory biosensors in wastewater isolates achieved memory of analyte exposure in an uncharacterized Pseudomonas isolate. By returning these engineered isolates to their native environments, recombinase-memory systems are expected to enable longer duration and in situ investigation of microbial signaling, community shifts, and gene transfer beyond the reach of traditional environmental biosensors.\n",
  "wordCount" : "2011",
  "inLanguage": "en",
  "datePublished": "2023-08-16T10:37:20Z",
  "dateModified": "2023-08-16T10:37:20Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on August 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552077">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552077" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552077">
        <p class="paperTitle">Machine-guided design of synthetic cell type-specific cis-regulatory elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552077" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552077" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gosai, S. J.; Castro, R. I.; Fuentes, N.; Butts, J. C.; Kales, S.; Noche, R. R.; Mouri, K.; Sabeti, P. C.; Reilly, S. K.; Tewhey, R.</p>
        <p class="info">Score: 57.7, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552077' target='https://doi.org/10.1101/2023.08.08.552077'> 10.1101/2023.08.08.552077</a></p>
        <p class="abstract">Cis-regulatory elements (CREs) control gene expression, orchestrating tissue identity, developmental timing, and stimulus responses, which collectively define the thousands of unique cell types in the body. While there is great potential for strategically incorporating CREs in therapeutic or biotechnology applications that require tissue specificity, there is no guarantee that an optimal CRE for an intended purpose has arisen naturally through evolution. Here, we present a platform to engineer and validate synthetic CREs capable of driving gene expression with programmed cell type specificity. We leverage innovations in deep neural network modeling of CRE activity across three cell types, efficient in silico optimization, and massively parallel reporter assays (MPRAs) to design and empirically test thousands of CREs. Through in vitro and in vivo validation, we show that synthetic sequences outperform natural sequences from the human genome in driving cell type-specific expression. Synthetic sequences leverage unique sequence syntax to promote activity in the on-target cell type and simultaneously reduce activity in off-target cells. Together, we provide a generalizable framework to prospectively engineer CREs and demonstrate the required literacy to write regulatory code that is fit-for-purpose in vivo across vertebrates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553013">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553013" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553013">
        <p class="paperTitle">Efficient Natural Plasmid Transformation of Vibrio natriegens Enables Zero-capital Molecular Biology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553013" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553013" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Specht, D. A.; Sheppard, T. J.; Kennedy, F.; Li, S.; Gadikota, G.; Barstow, B.</p>
        <p class="info">Score: 35.6, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553013' target='https://doi.org/10.1101/2023.08.11.553013'> 10.1101/2023.08.11.553013</a></p>
        <p class="abstract">The fast-growing microbe Vibrio natriegens is capable of natural transformation where it draws DNA in from media via an active process under physiological conditions. Using an engineered strain with a genomic copy of the master competence regulator tfoX from Vibrio cholera in combination with a new minimal competence media (MCM) that uses acetate as an energy source, we demonstrate naturally competent cells which are created, transformed, and recovered entirely in the same media, without exchange or addition of new media. Cells are naturally competent to plasmids, recombination with linear DNA, and co-transformation of both to select for scarless and markerless genomic edits. The entire process is simple and inexpensive, requiring no capital equipment for an entirely room temperature process (Zero Capital protocol, 10^4 cfu/g), or just an incubator (High Efficiency protocol, 10^5-6 cfu/g). These cells retain their naturally competent state when frozen and are transformable immediately upon thawing like a typical chemical or electrochemical competent cell. Since the optimized transformation protocol requires only 50 minutes of hands-on time, and V. natriegens grows quickly even on plates, a transformation started at 9 AM yields abundant culturable single colonies by 5 PM. Further, because all stages of transformation occur in the same media, and the process can be arbitrarily scaled in volume, this natural competence strain and media could be ideal for automated directed evolution applications. As a result, naturally competent V. natriegens could compete with E. coli as an excellent chassis for low-cost and highly scalable synthetic biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.553235">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.553235" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.553235">
        <p class="paperTitle">A molecular proximity sensor based on an engineered, dual-component guide RNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.553235" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.553235" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Choi, J.; Chen, W.; Liao, H.; Li, X.; Shendure, J.</p>
        <p class="info">Score: 30.2, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.553235' target='https://doi.org/10.1101/2023.08.14.553235'> 10.1101/2023.08.14.553235</a></p>
        <p class="abstract">One of the goals of synthetic biology is to enable the design of arbitrary molecular circuits with programmable inputs and outputs. Such circuits bridge the properties of electronic and natural circuits, processing information in a predictable manner within living cells. Genome editing is a potentially powerful component of synthetic molecular circuits, whether for modulating the expression of a target gene or for stably recording information to genomic DNA. However, programming molecular events such as protein-protein interactions or induced proximity as triggers for genome editing remains challenging. Here we demonstrate a strategy termed P3 editing, which links protein-protein proximity to the formation of a functional CRISPR-Cas9 dual-component guide RNA. By engineering the crRNA:tracrRNA interaction, we demonstrate that various known protein-protein interactions, as well as the chemically-induced dimerization of protein domains, can be used to activate prime editing or base editing in human cells. Additionally, we explore how P3 editing can incorporate outputs from ADAR-based RNA sensors, potentially allowing specific RNAs to induce specific genome edits within a larger circuit. Our strategy enhances the controllability of CRISPR-based genome editing, facilitating its use in synthetic molecular circuits deployed in living cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.553307">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.553307" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.553307">
        <p class="paperTitle">Intern-based thermoregulated meganucleases for biocontainment of genetic material</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.553307" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.553307" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Foo, G. W.; Leichthammer, C. D.; Saita, I. M.; Lukas, N. D.; Batko, I. Z.; Heinrichs, D. E.; Edgell, D.</p>
        <p class="info">Score: 1.5, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.553307' target='https://doi.org/10.1101/2023.08.14.553307'> 10.1101/2023.08.14.553307</a></p>
        <p class="abstract">Limiting the spread of synthetic genetic information outside of the intended use is essential for applications where biocontainment is critical. In particular, biocontainment of engineered probiotics and plasmids that are excreted from the mammalian gastrointestinal tract is needed to prevent escape and acquisition of genetic material that could confer a selective advantage to microbial communities. Here, we built a simple and lightweight biocontainment system that post-translationally activates a site-specific DNA endonuclease to degrade DNA at 18{degrees}C and not at higher temperatures. We constructed an orthogonal set of temperature sensitive-meganucleases, or TSMs, by inserting the yeast VMA1 L212P temperature-sensitive intein into the coding regions of LAGLIDADG homing endonucleases. We showed that the TSMs eliminated plasmids carrying the cognate TSM target site from laboratory strains of Escherichia coli at the permissive 18{degrees}C but not at higher restrictive temperatures. Plasmid elimination is dependent on both TSM endonuclease activity and intein splicing. We demonstrated that TSMs eliminated plasmids from the E. coli Nissle 1917 strain after passage through the mouse gut when fecal resuspensions were incubated at 18{degrees}C but not at 37{degrees}C. Collectively, our data demonstrates the potential of thermoregulated meganucleases as a means of restricting engineered plasmids and probiotics to the mammalian gut.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552766">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552766" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552766">
        <p class="paperTitle">Design of a Biohybrid Materials Circuit with Binary Decoder Functionality</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552766" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552766" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mohsenin, H.; Wagner, H.; Rosenblatt, M.; Kemmer, S.; Timmer, J.; Weber, W.</p>
        <p class="info">Score: 1.2, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552766' target='https://doi.org/10.1101/2023.08.10.552766'> 10.1101/2023.08.10.552766</a></p>
        <p class="abstract">Synthetic biology applies concepts from electrical engineering and information processing to endow cells with computational functionality. Transferring the underlying molecular components into materials and wiring them according to topologies inspired by electronic circuit boards has yielded materials systems that perform selected computational operations. However, the limited functionality of available building blocks is restricting the implementation of advanced information-processing circuits into materials. Here, we engineer a set of protease-based biohybrid modules the bioactivity of which can either be induced or inhibited. Guided by a quantitative mathematical model and following a design-build-test-learn cycle, we wire the modules according to circuit topologies inspired by electronic signal decoders, a fundamental motif in information processing. We design a 2-input/4-output binary decoder for the detection of two small molecules in a material framework that could perform regulated outputs in form of distinct protease activities. The here demonstrated smart material system is strongly modular and could be used for biomolecular information processing for example in advanced biosensing or drug delivery applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553049">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553049" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553049">
        <p class="paperTitle">Engineering a Synthetic Escherichia coli Coculture for Compartmentalized de novo Biosynthesis of Isobutyl Butyrate from Mixed Sugars</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553049" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553049" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Seo, H.; Castro, G. I.; Trinh, C. T.</p>
        <p class="info">Score: 1.0, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553049' target='https://doi.org/10.1101/2023.08.11.553049'> 10.1101/2023.08.11.553049</a></p>
        <p class="abstract">Short-chain esters are versatile chemicals with use as flavors, fragrances, solvents, and fuels. The de novo ester biosynthesis consists of diverging and converging pathway submodules, which is challenging to engineer to achieve optimal metabolic fluxes and selective product synthesis. Compartmentalizing the pathway submodules into specialist cells that facilitate pathway modularization and labor division can present a promising solution. Here, we engineered a synthetic Escherichia coli coculture with the compartmentalized sugar utilization and ester biosynthesis pathways to produce isobutyl butyrate from a mixture of glucose and xylose. To compartmentalize the sugar-utilizing pathway submodules, we engineered a xylose-utilizing E. coli specialist that selectively consumes xylose over glucose and bypasses the carbon catabolite repression (CCR) while leveraging the native CCR machinery to activate a glucose-utilizing E. coli specialist. Upon compartmentalizing the isobutyl butyrate pathway submodules into these sugar-utilizing specialist cells, a robust synthetic coculture could be engineered to selectively produce isobutyl butyrate at a level of 392 mg/L, about 31-fold higher than the monoculture.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552009">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552009" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552009">
        <p class="paperTitle">Engineering the highly productive cyanobacterium Synechococcus sp. PCC 11901</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552009" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552009" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Victoria, A. J.; Selao, T. T.; Moreno-Cabezuelo, J. A.; Mills, L. A.; Gale, G. A.; Lea-Smith, D. J.; McCormick, A. J.</p>
        <p class="info">Score: 12.0, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552009' target='https://doi.org/10.1101/2023.08.04.552009'> 10.1101/2023.08.04.552009</a></p>
        <p class="abstract">Synechococcus sp. PCC 11901 (PCC 11901) is a fast-growing marine cyanobacterial strain that has a capacity for sustained biomass accumulation to very high cell densities, comparable to that achieved by commercially relevant heterotrophic organisms. However, genetic tools to engineer PCC 11901 for biotechnology applications are limited. Here we describe a suite of tools based on the CyanoGate MoClo system to unlock the engineering potential of PCC 11901. First, we characterised neutral sites suitable for stable genomic integration that do not affect growth even at high cell densities. Second, we tested a suite of constitutive promoters, terminators, and inducible promoters including a 2,4-diacetylphloroglucinol (DAPG)-inducible PhlF repressor system, which has not previously been demonstrated in cyanobacteria and showed tight regulation and a 228-fold dynamic range of induction. Lastly, we developed a DAPG-inducible dCas9-based CRISPR interference (CRISPRi) system and a modular method to generate markerless mutants using CRISPR-Cas12a. Based on our findings, PCC 11901 is highly responsive to CRISPRi-based repression and showed high efficiencies for single insertion (31-81%) and multiplex double insertion (25%) genome editing with Cas12a. We envision that these tools will lay the foundations for the adoption of PCC 11901 as a robust model strain for engineering biology and green biotechnology.

On sentence summaryGenetic parts were characterised in Synechococcus sp. PCC 11901, including a tightly regulated inducible promoter system, efficient CRISPRi and a novel markerless Cas12a genome editing approach.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550434">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550434" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550434">
        <p class="paperTitle">Gas vesicle-blood interactions enhance ultrasound imaging contrast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550434" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550434" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ling, B.; Ko, J. H.; Stordy, B.; Zhang, Y.; Didden, T. F.; Malounda, D.; Swift, M. B.; Chan, W. C. W.; Shapiro, M. G.</p>
        <p class="info">Score: 11.9, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550434' target='https://doi.org/10.1101/2023.07.24.550434'> 10.1101/2023.07.24.550434</a></p>
        <p class="abstract">Gas vesicles (GVs) are genetically encoded, air-filled protein nanostructures of broad interest for biomedical research and clinical applications, acting as imaging and therapeutic agents for ultrasound, magnetic resonance, and optical techniques. However, the biomedical applications of GVs as a systemically injectable nanomaterial have been hindered by a lack of understanding of GVs interactions with blood components, which can significantly impact in vivo performance. Here, we investigate the dynamics of GVs in the bloodstream using a combination of ultrasound and optical imaging, surface functionalization, flow cytometry, and mass spectrometry. We find that erythrocytes and serum proteins bind to GVs and shape their acoustic response, circulation time, and immunogenicity. We show that by modifying the GV surface, we can alter these interactions and thereby modify GVs in vivo performance. These results provide critical insights for the development of GVs as agents for nanomedicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552812">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552812" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552812">
        <p class="paperTitle">Long duration environmental biosensing by recording analyte detection in DNA using recombinase memory</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552812" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552812" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kalvapalle, P.; Sridhar, S.; Silberg, J. J.; Stadler, L. B.</p>
        <p class="info">Score: 0.8, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552812' target='https://doi.org/10.1101/2023.08.10.552812'> 10.1101/2023.08.10.552812</a></p>
        <p class="abstract">Microbial biosensors that convert environmental information into real-time visual outputs are limited in their sensing abilities in complex environments, such as soil and wastewater. Alternative reporter outputs are needed that stably record the presence of analytes. Here, we test the performance of recombinase-memory biosensors that sense a sugar (arabinose) and a microbial communication molecule (3-oxo-C12- homoserine lactone) over 8 days (~70 generations) following analyte exposure. These biosensors use analyte sensing to trigger the expression of a recombinase which flips a segment of DNA, creating a genetic memory, and initiates fluorescent protein expression. The initial designs failed over time due to unintended DNA flipping in the absence of the analyte and loss of the flipped state after exposure to the analyte. Biosensor performance was improved by decreasing recombinase expression, removing the fluorescent protein output, and using qPCR to read out stored information. Application of memory biosensors in wastewater isolates achieved memory of analyte exposure in an uncharacterized Pseudomonas isolate. By returning these engineered isolates to their native environments, recombinase-memory systems are expected to enable longer duration and in situ investigation of microbial signaling, community shifts, and gene transfer beyond the reach of traditional environmental biosensors.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
