<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>synthetic biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>synthetic biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Ultralow background membrane editors for spatiotemporal control of lipid metabolism and signaling
Authors: Li, X.-L.; Tei, R.; Uematsu, M.; Baskin, J. M.
Score: 20.6, Published: 2023-08-31 DOI: 10.1101/2023.08.31.555787
Phosphatidic acid (PA) is a multifunctional lipid with important metabolic and signaling functions, and efforts to dissect its pleiotropy demand strategies for perturbing its levels with spatiotemporal precision. Previous membrane editing approaches for generating local PA pools used light-mediated induced proximity to recruit a PA-synthesizing enzyme, phospholipase D (PLD), from the cytosol to the target organelle membrane.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/synthetic-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="synthetic biology" />
<meta property="og:description" content="Ultralow background membrane editors for spatiotemporal control of lipid metabolism and signaling
Authors: Li, X.-L.; Tei, R.; Uematsu, M.; Baskin, J. M.
Score: 20.6, Published: 2023-08-31 DOI: 10.1101/2023.08.31.555787
Phosphatidic acid (PA) is a multifunctional lipid with important metabolic and signaling functions, and efforts to dissect its pleiotropy demand strategies for perturbing its levels with spatiotemporal precision. Previous membrane editing approaches for generating local PA pools used light-mediated induced proximity to recruit a PA-synthesizing enzyme, phospholipase D (PLD), from the cytosol to the target organelle membrane." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/synthetic-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-06T10:39:07+00:00" />
<meta property="article:modified_time" content="2023-09-06T10:39:07+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="synthetic biology"/>
<meta name="twitter:description" content="Ultralow background membrane editors for spatiotemporal control of lipid metabolism and signaling
Authors: Li, X.-L.; Tei, R.; Uematsu, M.; Baskin, J. M.
Score: 20.6, Published: 2023-08-31 DOI: 10.1101/2023.08.31.555787
Phosphatidic acid (PA) is a multifunctional lipid with important metabolic and signaling functions, and efforts to dissect its pleiotropy demand strategies for perturbing its levels with spatiotemporal precision. Previous membrane editing approaches for generating local PA pools used light-mediated induced proximity to recruit a PA-synthesizing enzyme, phospholipase D (PLD), from the cytosol to the target organelle membrane."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "synthetic biology",
      "item": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "synthetic biology",
  "name": "synthetic biology",
  "description": "Ultralow background membrane editors for spatiotemporal control of lipid metabolism and signaling\nAuthors: Li, X.-L.; Tei, R.; Uematsu, M.; Baskin, J. M.\nScore: 20.6, Published: 2023-08-31 DOI: 10.1101/2023.08.31.555787\nPhosphatidic acid (PA) is a multifunctional lipid with important metabolic and signaling functions, and efforts to dissect its pleiotropy demand strategies for perturbing its levels with spatiotemporal precision. Previous membrane editing approaches for generating local PA pools used light-mediated induced proximity to recruit a PA-synthesizing enzyme, phospholipase D (PLD), from the cytosol to the target organelle membrane.",
  "keywords": [
    
  ],
  "articleBody": " Ultralow background membrane editors for spatiotemporal control of lipid metabolism and signaling\nAuthors: Li, X.-L.; Tei, R.; Uematsu, M.; Baskin, J. M.\nScore: 20.6, Published: 2023-08-31 DOI: 10.1101/2023.08.31.555787\nPhosphatidic acid (PA) is a multifunctional lipid with important metabolic and signaling functions, and efforts to dissect its pleiotropy demand strategies for perturbing its levels with spatiotemporal precision. Previous membrane editing approaches for generating local PA pools used light-mediated induced proximity to recruit a PA-synthesizing enzyme, phospholipase D (PLD), from the cytosol to the target organelle membrane. Whereas these optogenetic PLDs exhibited high activity, their residual activity in the dark led to undesired chronic lipid production. Here, we report ultralow background membrane editors for PA wherein light directly controls PLD catalytic activity, as opposed to localization and access to substrates, exploiting a LOV domain-based conformational photoswitch inserted into the PLD sequence and enabling their stable and non-perturbative targeting to multiple organelle membranes. By coupling organelle-targeted LOVPLD activation to lipidomics analysis, we discovered different rates of metabolism for PA and its downstream products depending on the subcellular location of PA production. We also elucidated signaling roles for PA pools on different membranes in conferring local activation of AMP-activated protein kinase signaling. This work illustrates how membrane editors featuring acute, optogenetic conformational switches can provide new insights into organelle-selective lipid metabolic and signaling pathways. TOC Graphic O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=129 SRC=\"FIGDIR/small/555787v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (28K): org.highwire.dtl.DTLVardef@1beba57org.highwire.dtl.DTLVardef@bba8b3org.highwire.dtl.DTLVardef@4096b6org.highwire.dtl.DTLVardef@66b1f0_HPS_FORMAT_FIGEXP M_FIG C_FIG\nEscherichia coli Microcin B17 as a chassis for the development of novel antimicrobial peptides.\nAuthors: Withanage, S.; Masschelein, J.; Pinheiro, V. B.\nScore: 8.1, Published: 2023-08-30 DOI: 10.1101/2023.08.29.555279\nAntimicrobial resistance remains an urgent and growing global health problem. Peptides with antimicrobial activity are widespread in nature but are rarely harnessed for application, making them an underexplored and untapped resource. Many harbor post-translational modifications, which are essential for function and have evolved alongside specialized export and immunity machineries - leading to the current paradigm that natural sequences cannot be changed, and curtailing their development as antimicrobial agents. Nonetheless, we found sufficient ambiguity in literature to question if that remained true for E. coli MccB17. We demonstrate that its natural machinery for synthesis, export and immunity, can be co-opted as a platform for novel azole-modified antimicrobial peptide discovery. We show that the sequence space available for exploration is large and densely populated with functional sequences. One-Sentence SummaryE. coli can be harnessed as an in vivo platform for the isolation of novel antimicrobial azole-modified peptides.\nMicrobial cell factory optimisation using genome-wide host-pathway interaction screens\nAuthors: Cachera, P. P.-Y. J.; Kurt, N. C.; Ropke, A.; Strucko, T.; Mortensen, U. H.; Jensen, M. K.\nScore: 7.1, Published: 2023-08-30 DOI: 10.1101/2023.08.30.555557\nThe ubiquity of genetic interactions in living cells challenges the concept of parts orthogonality, which is a cornerstone of synthetic biology. Parts, such as heterologously expressed genes, draw from shared pools of limited cellular resources and interactions between parts themselves and their host are inevitable. Instead of trying to eliminate or disregard these interactions, we propose to leverage them to promote desirable phenotypes. We recently described CRI-SPA, a method for high-throughput genome-wide gene delivery and screening of host:pathway interactions in Saccharomyces cerevisiae. In this study, we combine this method with biosensor-based high-throughput screening and high-density colony image analysis to identify lead engineering targets for optimising cis-cis-muconic acid (CCM) production in yeast cell factories. Using the biosensor screen, we phenotype \u003e9,700 genotypes for their interaction with the heterologously expressed CCM biosynthesis pathway, including both gene knock-out and overexpression, and identify novel metabolic targets belonging to sulphur assimilation and methionine synthesis, as well as cellular redox homeostasis, positively impacting CCM biosynthesis by up to 280%. Our genome-wide exploration of host pathway interaction opens novel strategies for the metabolic engineering of yeast cell factories.\nA synthetic biology approach to transgene expression\nAuthors: Leftwich, P. T.; Purcell, J. C.; Anderson, M. A. E.; Fragkoudis, R.; Basu, S.; Lycett, G. T.; Alphey, L.\nScore: 6.8, Published: 2023-08-31 DOI: 10.1101/2023.08.31.555539\nThe ability to control gene expression is pivotal in genetic engineering and synthetic biology. However, in most non-model and pest insect species, empirical evidence for predictable modulation of gene expression levels is lacking. This knowledge gap is critical for genetic control systems, particularly in mosquitoes, where transgenic methods offer novel routes for pest control. Commonly, the choice of RNA polymerase II promoter (Pol II) is the primary method for controlling gene expression, but the options are limited. To address this, we developed a systematic approach to characterize modifications in translation initiation sequences (TIS) and 3 untranslated regions (UTR) of transgenes, enabling the creation of a toolbox for gene expression modulation in mosquitoes and potentially other insects. The approach demonstrated highly predictable gene expression changes across various cell lines and promoter sequences, representing a significant advancement in mosquito synthetic biology gene expression. tools. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=100 SRC=\"FIGDIR/small/555539v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (31K): org.highwire.dtl.DTLVardef@c10b0eorg.highwire.dtl.DTLVardef@1bcf4bborg.highwire.dtl.DTLVardef@674bf0org.highwire.dtl.DTLVardef@1aa871a_HPS_FORMAT_FIGEXP M_FIG C_FIG\nBuilding the bioeconomy: A targeted assessment approach to identifying biobased technologies, challenges and opportunities in the UK\nAuthors: Holland, C.; Shapira, P.\nScore: 6.5, Published: 2023-08-29 DOI: 10.1101/2023.08.28.554388\nWe explore opportunities, challenges, and strategies to translate and responsibly scale innovative biobased technologies to build more sustainable bioeconomies. The pandemic and other recent disruptions have increased exposure to issues of resilience and regional imbalance and raised attention to pathways that could shift production and consumption regimes based more on local biobased resources and dispersed production. The paper reviews potential biobased technologies strategies and then identifies promising and feasible options with a focus on the United Kingdom. Initial landscape and bibliometric analyses identified 50 potential existing and emerging potential biobased technologies. These technologies were assessed for their ability to fulfil requirements related to biobased production, national applicability, and economic, societal, and environmental benefits, leading to identification of 18 promising biobased production technologies. Through further analysis and focus group discussion with industrial, governmental, academic, agricultural, and social stakeholders, three technology clusters were identified for targeted assessment, drawing on cellulose-, lignin-, and seaweed-feedstocks. Case studies for each of these clusters were developed, addressing conversations around sustainable management and the use of biomass feedstocks, and associated environmental, social, and economic challenges. These cases are presented with discussion of insights and implications for policy. The approach presented in the paper is put forward as a scalable assessment method which can be useful in prompting, informing, and advancing discussion and deliberation on opportunities and challenges for biobased transformations.\nCell-Free Protein Expression in Polymer Materials\nAuthors: Lee, M. F. S.; Lee, J. A.; Biondo, J. R.; Lux, J. E.; Raig, R. M.; Berger, P. N.; Bernhards, C. B.; Kuhn, D. L.; Gupta, M. K.; Lux, M. W.\nScore: 6.5, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553881\nWhile synthetic biology has advanced complex capabilities like sensing and molecule synthesis in aqueous solutions, important applications may also be pursued for biological systems in solid materials. Harsh processing conditions used to produce many synthetic materials such as plastics makes incorporation of biological functionality challenging. One technology that shows promise in circumventing these issues is cell-free protein synthesis (CFPS), where core cellular functionality is reconstituted outside the cell. CFPS enables genetic functions to be implemented without the complications of membrane transport or concerns over cellular viability or release of genetically modified organisms. Here we demonstrate that dried CFPS reactions have remarkable tolerance to heat and organic solvent exposure during the casting processes for polymer materials. We demonstrate the utility of this observation by creating plastics that have spatially patterned genetic functionality, produce antimicrobials in situ, and perform sensing reactions. The resulting materials unlock the potential to deliver DNA-programmable bio-functionality in a ubiquitous class of synthetic materials.\nA molecular proximity sensor based on an engineered, dual-component guide RNA\nAuthors: Choi, J.; Chen, W.; Liao, H.; Li, X.; Shendure, J.\nScore: 39.8, Published: 2023-08-14 DOI: 10.1101/2023.08.14.553235\nOne of the goals of synthetic biology is to enable the design of arbitrary molecular circuits with programmable inputs and outputs. Such circuits bridge the properties of electronic and natural circuits, processing information in a predictable manner within living cells. Genome editing is a potentially powerful component of synthetic molecular circuits, whether for modulating the expression of a target gene or for stably recording information to genomic DNA. However, programming molecular events such as protein-protein interactions or induced proximity as triggers for genome editing remains challenging. Here we demonstrate a strategy termed P3 editing, which links protein-protein proximity to the formation of a functional CRISPR-Cas9 dual-component guide RNA. By engineering the crRNA:tracrRNA interaction, we demonstrate that various known protein-protein interactions, as well as the chemically-induced dimerization of protein domains, can be used to activate prime editing or base editing in human cells. Additionally, we explore how P3 editing can incorporate outputs from ADAR-based RNA sensors, potentially allowing specific RNAs to induce specific genome edits within a larger circuit. Our strategy enhances the controllability of CRISPR-based genome editing, facilitating its use in synthetic molecular circuits deployed in living cells.\nShifting Redox Reaction Equilibria on Demand Using an Orthogonal Redox Cofactor\nAuthors: Aspacio, D.; Zhang, Y.; Cui, Y.; King, E.; Black, W. B.; Perea, S.; Luu, E.; Siegel, J. B.; Li, H.\nScore: 1.5, Published: 2023-08-30 DOI: 10.1101/2023.08.29.555398\nNatural metabolism relies on chemical compartmentalization of two redox cofactors, NAD+ and NADP+, to orchestrate life-essential redox reaction directions. However, in whole cells the reliance on these canonical cofactors limits flexible control of redox reaction direction as these reactions are permanently tied to catabolism or anabolism. In cell-free systems, NADP+ is too expensive in large scale. We have previously reported the use of nicotinamide mononucleotide, (NMN+) as a low-cost, noncanonical redox cofactor capable of specific electron delivery to diverse chemistries. Here, we present Nox Ortho, an NMNH-specific water-forming oxidase, that completes the toolkit to modulate NMNH/NMN+ ratio. This work uncovers an enzyme design principle that succeeds in parallel engineering of six butanediol dehydrogenases as NMN(H)-orthogonal biocatalysts consistently with a 103 - 106 -fold cofactor specificity switch from NAD(P)+ to NMN+. We combine these to produce chiral-pure 2,3-butanediol (Bdo) isomers without interference from NAD(H) or NADP(H) in vitro and in E. coli cells. We establish that NMN(H) can be held at a distinct redox ratio on demand, decoupled from both NAD(H) and NADP(H) redox ratios in vitro and in vivo.\nDesigned Endocytosis-Triggering Proteins mediate Targeted Degradation and amplify signaling\nAuthors: Huang, B.; Abedi, M.; Ahn, G.; Coventry, B.; Sappington, I.; Wang, R.; Schlichthaerle, T.; Zhang, J. Z.; Wang, Y.; Goreshnik, I.; Chiu, C. W.; Chazin-Gray, A.; Chan, S.; Gerben, S.; Murray, A.; Wang, S.; O'Neill, J.; Yeh, R.; Misquith, A.; Wolf, A.; Tomasovic, L. M.; Piraner, D. I.; Gonzalez, M. J. D.; Bennett, N. R.; Venkatesh, P.; Ahlrichs, M.; Dobbins, C.; Yang, W.; Wang, X.; Vafeados, D.; Mout, R.; Shivaei, S.; Cao, L.; Carter, L.; Stewart, L.; Spangler, J. B.; Bernardes, G. J. L.; Roybal, K. T.; Greisen, P. J.; Li, X.; Bertozzi, C.; Baker, D.\nScore: 13.3, Published: 2023-08-21 DOI: 10.1101/2023.08.19.553321\nEndocytosis and lysosomal trafficking of cell surface receptors can be triggered by interaction with endogenous ligands. Therapeutic approaches such as LYTAC1,2 and KineTAC3, have taken advantage of this to target specific proteins for degradation by fusing modified native ligands to target binding proteins. While powerful, these approaches can be limited by possible competition with the endogenous ligand(s), the requirement in some cases for chemical modification that limits genetic encodability and can complicate manufacturing, and more generally, there may not be natural ligands which stimulate endocytosis through a given receptor. Here we describe general protein design approaches for designing endocytosis triggering binding proteins (EndoTags) that overcome these challenges. We present EndoTags for the IGF-2R, ASGPR, and Transferrin receptors, and show that fusing these tags to proteins which bind to soluble or transmembrane protein leads to lysosomal trafficking and target degradation; as these receptors have different tissue distributions, the different EndoTags could enable targeting of degradation to different tissues. The modularity and genetic encodability of EndoTags enables AND gate control for higher specificity targeted degradation, and the localized secretion of degraders from engineered cells. The tunability and modularity of our genetically encodable EndoTags should contribute to deciphering the relationship between receptor engagement and cellular trafficking, and they have considerable therapeutic potential as targeted degradation inducers, signaling activators for endocytosis-dependent pathways, and cellular uptake inducers for targeted antibody drug and RNA conjugates.\n",
  "wordCount" : "2036",
  "inLanguage": "en",
  "datePublished": "2023-09-06T10:39:07Z",
  "dateModified": "2023-09-06T10:39:07Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/synthetic-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      synthetic biology
    </h1>
    <div class="post-meta"><span>updated on September 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555787">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555787" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555787">
        <p class="paperTitle">Ultralow background membrane editors for spatiotemporal control of lipid metabolism and signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555787" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555787" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, X.-L.; Tei, R.; Uematsu, M.; Baskin, J. M.</p>
        <p class="info">Score: 20.6, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555787' target='https://doi.org/10.1101/2023.08.31.555787'> 10.1101/2023.08.31.555787</a></p>
        <p class="abstract">Phosphatidic acid (PA) is a multifunctional lipid with important metabolic and signaling functions, and efforts to dissect its pleiotropy demand strategies for perturbing its levels with spatiotemporal precision. Previous membrane editing approaches for generating local PA pools used light-mediated induced proximity to recruit a PA-synthesizing enzyme, phospholipase D (PLD), from the cytosol to the target organelle membrane. Whereas these optogenetic PLDs exhibited high activity, their residual activity in the dark led to undesired chronic lipid production. Here, we report ultralow background membrane editors for PA wherein light directly controls PLD catalytic activity, as opposed to localization and access to substrates, exploiting a LOV domain-based conformational photoswitch inserted into the PLD sequence and enabling their stable and non-perturbative targeting to multiple organelle membranes. By coupling organelle-targeted LOVPLD activation to lipidomics analysis, we discovered different rates of metabolism for PA and its downstream products depending on the subcellular location of PA production. We also elucidated signaling roles for PA pools on different membranes in conferring local activation of AMP-activated protein kinase signaling. This work illustrates how membrane editors featuring acute, optogenetic conformational switches can provide new insights into organelle-selective lipid metabolic and signaling pathways.

TOC Graphic

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=129 SRC=&#34;FIGDIR/small/555787v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (28K):
org.highwire.dtl.DTLVardef@1beba57org.highwire.dtl.DTLVardef@bba8b3org.highwire.dtl.DTLVardef@4096b6org.highwire.dtl.DTLVardef@66b1f0_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555279">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555279" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555279">
        <p class="paperTitle">Escherichia coli Microcin B17 as a chassis for the development of novel antimicrobial peptides.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555279" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555279" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Withanage, S.; Masschelein, J.; Pinheiro, V. B.</p>
        <p class="info">Score: 8.1, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555279' target='https://doi.org/10.1101/2023.08.29.555279'> 10.1101/2023.08.29.555279</a></p>
        <p class="abstract">Antimicrobial resistance remains an urgent and growing global health problem. Peptides with antimicrobial activity are widespread in nature but are rarely harnessed for application, making them an underexplored and untapped resource. Many harbor post-translational modifications, which are essential for function and have evolved alongside specialized export and immunity machineries - leading to the current paradigm that natural sequences cannot be changed, and curtailing their development as antimicrobial agents. Nonetheless, we found sufficient ambiguity in literature to question if that remained true for E. coli MccB17. We demonstrate that its natural machinery for synthesis, export and immunity, can be co-opted as a platform for novel azole-modified antimicrobial peptide discovery. We show that the sequence space available for exploration is large and densely populated with functional sequences.

One-Sentence SummaryE. coli can be harnessed as an in vivo platform for the isolation of novel antimicrobial azole-modified peptides.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555557">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555557" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555557">
        <p class="paperTitle">Microbial cell factory optimisation using genome-wide host-pathway interaction screens</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555557" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555557" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cachera, P. P.-Y. J.; Kurt, N. C.; Ropke, A.; Strucko, T.; Mortensen, U. H.; Jensen, M. K.</p>
        <p class="info">Score: 7.1, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555557' target='https://doi.org/10.1101/2023.08.30.555557'> 10.1101/2023.08.30.555557</a></p>
        <p class="abstract">The ubiquity of genetic interactions in living cells challenges the concept of parts orthogonality, which is a cornerstone of synthetic biology. Parts, such as heterologously expressed genes, draw from shared pools of limited cellular resources and interactions between parts themselves and their host are inevitable. Instead of trying to eliminate or disregard these interactions, we propose to leverage them to promote desirable phenotypes. We recently described CRI-SPA, a method for high-throughput genome-wide gene delivery and screening of host:pathway interactions in Saccharomyces cerevisiae. In this study, we combine this method with biosensor-based high-throughput screening and high-density colony image analysis to identify lead engineering targets for optimising cis-cis-muconic acid (CCM) production in yeast cell factories. Using the biosensor screen, we phenotype &gt;9,700 genotypes for their interaction with the heterologously expressed CCM biosynthesis pathway, including both gene knock-out and overexpression, and identify novel metabolic targets belonging to sulphur assimilation and methionine synthesis, as well as cellular redox homeostasis, positively impacting CCM biosynthesis by up to 280%. Our genome-wide exploration of host pathway interaction opens novel strategies for the metabolic engineering of yeast cell factories.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555539">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555539" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555539">
        <p class="paperTitle">A synthetic biology approach to transgene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555539" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555539" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Leftwich, P. T.; Purcell, J. C.; Anderson, M. A. E.; Fragkoudis, R.; Basu, S.; Lycett, G. T.; Alphey, L.</p>
        <p class="info">Score: 6.8, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555539' target='https://doi.org/10.1101/2023.08.31.555539'> 10.1101/2023.08.31.555539</a></p>
        <p class="abstract">The ability to control gene expression is pivotal in genetic engineering and synthetic biology. However, in most non-model and pest insect species, empirical evidence for predictable modulation of gene expression levels is lacking. This knowledge gap is critical for genetic control systems, particularly in mosquitoes, where transgenic methods offer novel routes for pest control. Commonly, the choice of RNA polymerase II promoter (Pol II) is the primary method for controlling gene expression, but the options are limited.

To address this, we developed a systematic approach to characterize modifications in translation initiation sequences (TIS) and 3 untranslated regions (UTR) of transgenes, enabling the creation of a toolbox for gene expression modulation in mosquitoes and potentially other insects. The approach demonstrated highly predictable gene expression changes across various cell lines and promoter sequences, representing a significant advancement in mosquito synthetic biology gene expression. tools.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=100 SRC=&#34;FIGDIR/small/555539v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (31K):
org.highwire.dtl.DTLVardef@c10b0eorg.highwire.dtl.DTLVardef@1bcf4bborg.highwire.dtl.DTLVardef@674bf0org.highwire.dtl.DTLVardef@1aa871a_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.554388">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.554388" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.554388">
        <p class="paperTitle">Building the bioeconomy: A targeted assessment approach to identifying biobased technologies, challenges and opportunities in the UK</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.554388" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.554388" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Holland, C.; Shapira, P.</p>
        <p class="info">Score: 6.5, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.554388' target='https://doi.org/10.1101/2023.08.28.554388'> 10.1101/2023.08.28.554388</a></p>
        <p class="abstract">We explore opportunities, challenges, and strategies to translate and responsibly scale innovative biobased technologies to build more sustainable bioeconomies. The pandemic and other recent disruptions have increased exposure to issues of resilience and regional imbalance and raised attention to pathways that could shift production and consumption regimes based more on local biobased resources and dispersed production. The paper reviews potential biobased technologies strategies and then identifies promising and feasible options with a focus on the United Kingdom. Initial landscape and bibliometric analyses identified 50 potential existing and emerging potential biobased technologies. These technologies were assessed for their ability to fulfil requirements related to biobased production, national applicability, and economic, societal, and environmental benefits, leading to identification of 18 promising biobased production technologies. Through further analysis and focus group discussion with industrial, governmental, academic, agricultural, and social stakeholders, three technology clusters were identified for targeted assessment, drawing on cellulose-, lignin-, and seaweed-feedstocks. Case studies for each of these clusters were developed, addressing conversations around sustainable management and the use of biomass feedstocks, and associated environmental, social, and economic challenges. These cases are presented with discussion of insights and implications for policy. The approach presented in the paper is put forward as a scalable assessment method which can be useful in prompting, informing, and advancing discussion and deliberation on opportunities and challenges for biobased transformations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553881">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553881" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553881">
        <p class="paperTitle">Cell-Free Protein Expression in Polymer Materials</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553881" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553881" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, M. F. S.; Lee, J. A.; Biondo, J. R.; Lux, J. E.; Raig, R. M.; Berger, P. N.; Bernhards, C. B.; Kuhn, D. L.; Gupta, M. K.; Lux, M. W.</p>
        <p class="info">Score: 6.5, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553881' target='https://doi.org/10.1101/2023.08.18.553881'> 10.1101/2023.08.18.553881</a></p>
        <p class="abstract">While synthetic biology has advanced complex capabilities like sensing and molecule synthesis in aqueous solutions, important applications may also be pursued for biological systems in solid materials. Harsh processing conditions used to produce many synthetic materials such as plastics makes incorporation of biological functionality challenging. One technology that shows promise in circumventing these issues is cell-free protein synthesis (CFPS), where core cellular functionality is reconstituted outside the cell. CFPS enables genetic functions to be implemented without the complications of membrane transport or concerns over cellular viability or release of genetically modified organisms. Here we demonstrate that dried CFPS reactions have remarkable tolerance to heat and organic solvent exposure during the casting processes for polymer materials. We demonstrate the utility of this observation by creating plastics that have spatially patterned genetic functionality, produce antimicrobials in situ, and perform sensing reactions. The resulting materials unlock the potential to deliver DNA-programmable bio-functionality in a ubiquitous class of synthetic materials.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.553235">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.553235" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.553235">
        <p class="paperTitle">A molecular proximity sensor based on an engineered, dual-component guide RNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.553235" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.553235" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Choi, J.; Chen, W.; Liao, H.; Li, X.; Shendure, J.</p>
        <p class="info">Score: 39.8, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.553235' target='https://doi.org/10.1101/2023.08.14.553235'> 10.1101/2023.08.14.553235</a></p>
        <p class="abstract">One of the goals of synthetic biology is to enable the design of arbitrary molecular circuits with programmable inputs and outputs. Such circuits bridge the properties of electronic and natural circuits, processing information in a predictable manner within living cells. Genome editing is a potentially powerful component of synthetic molecular circuits, whether for modulating the expression of a target gene or for stably recording information to genomic DNA. However, programming molecular events such as protein-protein interactions or induced proximity as triggers for genome editing remains challenging. Here we demonstrate a strategy termed P3 editing, which links protein-protein proximity to the formation of a functional CRISPR-Cas9 dual-component guide RNA. By engineering the crRNA:tracrRNA interaction, we demonstrate that various known protein-protein interactions, as well as the chemically-induced dimerization of protein domains, can be used to activate prime editing or base editing in human cells. Additionally, we explore how P3 editing can incorporate outputs from ADAR-based RNA sensors, potentially allowing specific RNAs to induce specific genome edits within a larger circuit. Our strategy enhances the controllability of CRISPR-based genome editing, facilitating its use in synthetic molecular circuits deployed in living cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555398">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555398" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555398">
        <p class="paperTitle">Shifting Redox Reaction Equilibria on Demand Using an Orthogonal Redox Cofactor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555398" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555398" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aspacio, D.; Zhang, Y.; Cui, Y.; King, E.; Black, W. B.; Perea, S.; Luu, E.; Siegel, J. B.; Li, H.</p>
        <p class="info">Score: 1.5, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555398' target='https://doi.org/10.1101/2023.08.29.555398'> 10.1101/2023.08.29.555398</a></p>
        <p class="abstract">Natural metabolism relies on chemical compartmentalization of two redox cofactors, NAD&#43; and NADP&#43;, to orchestrate life-essential redox reaction directions. However, in whole cells the reliance on these canonical cofactors limits flexible control of redox reaction direction as these reactions are permanently tied to catabolism or anabolism. In cell-free systems, NADP&#43; is too expensive in large scale. We have previously reported the use of nicotinamide mononucleotide, (NMN&#43;) as a low-cost, noncanonical redox cofactor capable of specific electron delivery to diverse chemistries.

Here, we present Nox Ortho, an NMNH-specific water-forming oxidase, that completes the toolkit to modulate NMNH/NMN&#43; ratio. This work uncovers an enzyme design principle that succeeds in parallel engineering of six butanediol dehydrogenases as NMN(H)-orthogonal biocatalysts consistently with a 103 - 106 -fold cofactor specificity switch from NAD(P)&#43; to NMN&#43;. We combine these to produce chiral-pure 2,3-butanediol (Bdo) isomers without interference from NAD(H) or NADP(H) in vitro and in E. coli cells. We establish that NMN(H) can be held at a distinct redox ratio on demand, decoupled from both NAD(H) and NADP(H) redox ratios in vitro and in vivo.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.19.553321">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.19.553321" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.19.553321">
        <p class="paperTitle">Designed Endocytosis-Triggering Proteins mediate Targeted Degradation and amplify signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.19.553321" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.19.553321" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, B.; Abedi, M.; Ahn, G.; Coventry, B.; Sappington, I.; Wang, R.; Schlichthaerle, T.; Zhang, J. Z.; Wang, Y.; Goreshnik, I.; Chiu, C. W.; Chazin-Gray, A.; Chan, S.; Gerben, S.; Murray, A.; Wang, S.; O&#39;Neill, J.; Yeh, R.; Misquith, A.; Wolf, A.; Tomasovic, L. M.; Piraner, D. I.; Gonzalez, M. J. D.; Bennett, N. R.; Venkatesh, P.; Ahlrichs, M.; Dobbins, C.; Yang, W.; Wang, X.; Vafeados, D.; Mout, R.; Shivaei, S.; Cao, L.; Carter, L.; Stewart, L.; Spangler, J. B.; Bernardes, G. J. L.; Roybal, K. T.; Greisen, P. J.; Li, X.; Bertozzi, C.; Baker, D.</p>
        <p class="info">Score: 13.3, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.19.553321' target='https://doi.org/10.1101/2023.08.19.553321'> 10.1101/2023.08.19.553321</a></p>
        <p class="abstract">Endocytosis and lysosomal trafficking of cell surface receptors can be triggered by interaction with endogenous ligands. Therapeutic approaches such as LYTAC1,2 and KineTAC3, have taken advantage of this to target specific proteins for degradation by fusing modified native ligands to target binding proteins. While powerful, these approaches can be limited by possible competition with the endogenous ligand(s), the requirement in some cases for chemical modification that limits genetic encodability and can complicate manufacturing, and more generally, there may not be natural ligands which stimulate endocytosis through a given receptor. Here we describe general protein design approaches for designing endocytosis triggering binding proteins (EndoTags) that overcome these challenges. We present EndoTags for the IGF-2R, ASGPR, and Transferrin receptors, and show that fusing these tags to proteins which bind to soluble or transmembrane protein leads to lysosomal trafficking and target degradation; as these receptors have different tissue distributions, the different EndoTags could enable targeting of degradation to different tissues. The modularity and genetic encodability of EndoTags enables AND gate control for higher specificity targeted degradation, and the localized secretion of degraders from engineered cells. The tunability and modularity of our genetically encodable EndoTags should contribute to deciphering the relationship between receptor engagement and cellular trafficking, and they have considerable therapeutic potential as targeted degradation inducers, signaling activators for endocytosis-dependent pathways, and cellular uptake inducers for targeted antibody drug and RNA conjugates.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
