<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Plasma-based antigen persistence in the post-acute phase of SARS-CoV-2 infection
Authors: Peluso, M. J.; Swank, Z. N.; Goldberg, S. A.; Lu, S.; Dalhuisen, T.; Borberg, E.; Senussi, Y.; Luna, M. A.; Chang Song, C.; Clark, A.; Zamora, A.; Lew, M.; Viswanathan, B.; Huang, B.; Anglin, K.; Hoh, R.; Hsue, P. Y.; Durstenfeld, M. S.; Spinelli, M. A.; Glidden, D. V.; Henrich, T. J.; Kelly, J. D.; Deeks, S. G.; Walt, D.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Plasma-based antigen persistence in the post-acute phase of SARS-CoV-2 infection
Authors: Peluso, M. J.; Swank, Z. N.; Goldberg, S. A.; Lu, S.; Dalhuisen, T.; Borberg, E.; Senussi, Y.; Luna, M. A.; Chang Song, C.; Clark, A.; Zamora, A.; Lew, M.; Viswanathan, B.; Huang, B.; Anglin, K.; Hoh, R.; Hsue, P. Y.; Durstenfeld, M. S.; Spinelli, M. A.; Glidden, D. V.; Henrich, T. J.; Kelly, J. D.; Deeks, S. G.; Walt, D." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-05T10:37:37+00:00" />
<meta property="article:modified_time" content="2023-11-05T10:37:37+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Plasma-based antigen persistence in the post-acute phase of SARS-CoV-2 infection
Authors: Peluso, M. J.; Swank, Z. N.; Goldberg, S. A.; Lu, S.; Dalhuisen, T.; Borberg, E.; Senussi, Y.; Luna, M. A.; Chang Song, C.; Clark, A.; Zamora, A.; Lew, M.; Viswanathan, B.; Huang, B.; Anglin, K.; Hoh, R.; Hsue, P. Y.; Durstenfeld, M. S.; Spinelli, M. A.; Glidden, D. V.; Henrich, T. J.; Kelly, J. D.; Deeks, S. G.; Walt, D."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Plasma-based antigen persistence in the post-acute phase of SARS-CoV-2 infection\nAuthors: Peluso, M. J.; Swank, Z. N.; Goldberg, S. A.; Lu, S.; Dalhuisen, T.; Borberg, E.; Senussi, Y.; Luna, M. A.; Chang Song, C.; Clark, A.; Zamora, A.; Lew, M.; Viswanathan, B.; Huang, B.; Anglin, K.; Hoh, R.; Hsue, P. Y.; Durstenfeld, M. S.; Spinelli, M. A.; Glidden, D. V.; Henrich, T. J.; Kelly, J. D.; Deeks, S. G.; Walt, D.",
  "keywords": [
    
  ],
  "articleBody": " Plasma-based antigen persistence in the post-acute phase of SARS-CoV-2 infection\nAuthors: Peluso, M. J.; Swank, Z. N.; Goldberg, S. A.; Lu, S.; Dalhuisen, T.; Borberg, E.; Senussi, Y.; Luna, M. A.; Chang Song, C.; Clark, A.; Zamora, A.; Lew, M.; Viswanathan, B.; Huang, B.; Anglin, K.; Hoh, R.; Hsue, P. Y.; Durstenfeld, M. S.; Spinelli, M. A.; Glidden, D. V.; Henrich, T. J.; Kelly, J. D.; Deeks, S. G.; Walt, D. R.; Martin, J. N.\nScore: 516.5, Published: 2023-10-26 DOI: 10.1101/2023.10.24.23297114\nBACKGROUNDAlthough RNA viruses like SARS-CoV-2 are generally thought to be transient, the persistence of viral components beyond the acute phase can be driven by a variety of virologic and immunologic factors. Recent studies have suggested that SARS-CoV-2 antigens may persist following COVID-19 but were limited by a lack of comparison to a large number of true negative control samples. METHODSUsing single molecule array (Simoa) assays for SARS-CoV-2 spike, S1, and nucleocapsid antigen in plasma from 171 pandemic-era individuals in the post-acute phase of SARS-CoV-2 infection and 250 pre-pandemic control samples, we compared prevalence of antigen detection. We used logistic regression models and prevalence ratios (PRs) to assess the relationship between demographic and disease factors and antigen persistence. RESULTSCompared to the proportion of antigen positivity in the pre-pandemic controls (2%), detection of any SARS-CoV-2 antigen was more frequent across all post-acute COVID-19 time bins (3-6 months: 12.6%, p\u003c0.001; 6-10 months, 10.7%, p=0.0002; 10-14 months, 7.5%, p=0.017). These differences were driven by spike protein for up to 14 months and nucleocapsid in the first 6 months after infection. The co-occurrence of multiple antigens at a single timepoint was uncommon. Hospitalization for acute COVID-19 (versus not hospitalized) and worse self-reported health during acute COVID-19 among those not hospitalized (versus more benign illness) were associated with higher prevalence of post-acute antigen detection (PR 1.86, p=0.03; PR 3.5, p=0.07, respectively) in the pandemic era. CONCLUSIONSOur findings provide strong evidence that SARS-CoV-2 antigens can persist beyond the period of acute illness. The observation that more than 10% of plasma samples for over a year following initial SARS-CoV-2 infection contain detectable viral antigen, which are potentially immunogenic, has significant implications given the sheer number of people infected with SARS-CoV-2 to date. More work will be needed to determine whether these antigens have a causal role in post-acute sequelae of SARS-CoV-2 infection (PASC).\nComplement dysregulation is a predictive and therapeutically amenable feature of long COVID\nAuthors: Baillie, K.; Davies, H. E.; Keat, S. B. K.; Ladell, K.; Miners, K. L.; Jones, S. A.; Mellou, E.; Toonen, E. J. M.; Price, D. A.; Morgan, B. P.; Zelek, W. M.\nScore: 132.3, Published: 2023-10-28 DOI: 10.1101/2023.10.26.23297597\nBackgroundLong COVID encompasses a heterogeneous set of ongoing symptoms that affect many individuals after recovery from infection with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). The underlying biological mechanisms nonetheless remain obscure, precluding accurate diagnosis and effective intervention. Complement dysregulation is a hallmark of acute COVID-19 but has not been investigated as a potential determinant of long COVID. MethodsWe quantified a series of complement proteins, including markers of activation and regulation, in plasma samples from healthy convalescent individuals with a confirmed history of infection with SARS-CoV-2 and age/ethnicity/gender/infection/vaccine-matched patients with long COVID. FindingsMarkers of classical (C1s-C1INH complex), alternative (Ba, iC3b), and terminal pathway (C5a, TCC) activation were significantly elevated in patients with long COVID. These markers in combination had a receiver operating characteristic predictive power of 0.794. Other complement proteins and regulators were also quantitatively different between healthy convalescent individuals and patients with long COVID. Generalized linear modeling further revealed that a clinically tractable combination of just four of these markers, namely the activation fragments iC3b, TCC, Ba, and C5a, had a predictive power of 0.785. ConclusionsThese findings suggest that complement biomarkers could facilitate the diagnosis of long COVID and further suggest that currently available inhibitors of complement activation could be used to treat long COVID. FundingThis work was funded by the National Institute for Health Research (COV-LT2-0041), the PolyBio Research Foundation, and the UK Dementia Research Institute.\nPublished benefits of ivermectin use in Itajai, Brazil for COVID-19 infection, hospitalisation, and mortality are entirely explained by statistical artefacts\nAuthors: Mills, R.; Pecanha Antonio, A. C.; Tucker-Kellogg, G.\nScore: 419.3, Published: 2023-10-30 DOI: 10.1101/2023.08.10.23293924\nBackgroundTwo recent publications by Kerr et al. (Cureus 14(1):e21272; Cureus 14(8): e28624) reported dramatic effects of prophylactic ivermectin use for both prevention of COVID-19 and reduction of COVID-19-related hospitalisation and mortality, including a dose-dependent effect of ivermectin prophylaxis. These papers have gained an unusually large public influence: they were incorporated into debates around COVID-19 policies and may have contributed to decreased trust in vaccine efficacy and public health authorities more broadly. Both studies were based on retrospective observational analysis of city-wide registry data from the city of Itajai, Brazil from July-December 2020. MethodsStarting with initially identified sources of error, we conducted a revised statistical analysis of available data, including data made available with the original papers and public data from the Brazil Ministry of Health. We identified additional uncorrected sources of bias and errors from the original analysis, including incorrect subject exclusion and missing subjects, analysis of longitudinal data with cross-sectional design, an enrolment time bias, and multiple sources of immortal time bias. In models assuming no actual effect from ivermectin use, we conducted Monte Carlo simulations to estimate the contribution of these biases to any observed effect. ResultsUntreated statistical artefacts and methodological errors alone lead to dramatic apparent risk reduction associated with ivermectin use in both studies. The magnitude of apparent risk reduction from these artefacts is comparable to the results reported by the studies themselves, including apparent protection from infection, hospitalisation, and death, and including the reported apparent dose-response relationship. ConclusionsThe inference of ivermectin effect reported in both papers is unsupported, as the observed effects are entirely explained by untreated statistical artefacts and methodological errors. Our re-analysis calls for caution in interpreting highly publicised observational studies and highlights the importance of common sources of bias in clinical research.\nViral respiratory epidemic modelling of societal segregation based on vaccination status\nAuthors: Hickey, J.; Rancourt, D. G.\nScore: 60.3, Published: 2023-10-31 DOI: 10.1101/2022.08.21.22279035\nBackgroundSocietal segregation of unvaccinated people from public spaces has been a novel and controversial COVID-era public health practice in many countries. Models exploring potential consequences of vaccination-status-based segregation have not considered how segregation influences the contact frequencies in the segregated groups. We systematically investigate implementing effects of segregation on population-specific contact frequencies and show this critically determines the predicted epidemiological outcomes, focusing on the attack rates in the vaccinated and unvaccinated populations and the share of infections among vaccinated people that were due to contacts with infectious unvaccinated people. MethodsWe describe a susceptible-infectious-recovered (SIR) two-population model for vaccinated and unvaccinated groups of individuals that transmit an infectious disease by person-to-person contact. The degree of segregation of the two groups, ranging from zero to complete segregation, is implemented using the like-to-like mixing approach developed for sexually-transmitted diseases, adapted for presumed SARS-CoV-2 transmission. We allow the contact frequencies for individuals in the two groups to be different and depend, with variable strength, on the degree of segregation. ResultsSegregation can either increase or decrease the attack rate among the vaccinated, depending on the type of segregation (isolating or compounding), and the contagiousness of the disease. For diseases with low contagiousness, segregation can cause an attack rate in the vaccinated, which does not occur without segregation. InterpretationThere is no predicted blanket epidemiological advantage to segregation, either for the vaccinated or the unvaccinated. Negative epidemiological consequences can occur for both groups.\nPostacute Sequelae SARS-CoV-2 Infection by Vaccination Status: A Six-Month Latent Class Analysis\nAuthors: Sun, X.; DeShazo, J. P.; Anatale-Tardiff, L.; Di Fusco, M.; Allen, K. E.; Porter, T. M.; Coetzer, H.; Lopez, S. M. C.; Puzniak, L.; Cappelleri, J. C.\nScore: 60.7, Published: 2023-10-21 DOI: 10.1101/2023.10.20.23297332\nSymptoms post-SARS-CoV-2 infection may persist for months and cause significant impairment and impact to quality of life. Acute symptoms of SARS-CoV-2 infection are well studied, yet data on clusters of symptoms over time, or postacute sequelae of SARS-CoV-2 infection (PASC), are limited. We aim to characterize PASC phenotypes by identifying symptom clusters over a six-month period following infection in individuals vaccinated (boosted and not) and those unvaccinated. Subjects with [\u0026ge;]1 self-reported symptom and positive RT-PCR for SARS-CoV-2 at CVS Health US test sites were recruited between January and April 2022. Patient-reported outcomes symptoms, heath-related quality of life (QoL), work productivity and activity impairment (WPAI) were captured at 1 month, 3 months, and 6 months post-acute infection. Logistic regression and latent class analysis (LCA) were performed on 20 symptoms using baseline socio-demographic, clinical characteristics, and vaccination status as well as EQ-5and WPAI results as covariables. Subjects with more symptoms were associated with lower health-related quality of life, and worse WPAI scores. LCA identified three phenotypes that are primarily differentiated by number of symptoms. These three phenotypes remained consistent across time periods. Vaccinated individuals were more likely to be in the low symptom burden latent classes at all time points compared to unvaccinated individuals.\nSystematic SARS-CoV-2 S Gene Sequencing in Wastewater Samples Enables Early Lineage Detection and Uncovers Rare Mutations in Portugal\nAuthors: Reis, A. C.; Pinto, D.; Monteiro, S.; Santos, R.; Martins, J. V.; Sousa, A.; Pascoa, R.; Lourinho, R.; Cunha, M. V.\nScore: 11.7, Published: 2023-10-31 DOI: 10.1101/2023.10.30.23297774\nAs the COVID-19 pandemic reached its peak, many countries implemented genomic surveillance systems to track the evolution and transmission of SARS-CoV-2. Transition from the pandemic to the endemic phase prioritized alternative testing strategies to maintain effective epidemic surveillance at the population level, with less intensive sequencing efforts. One such promising approach was Wastewater-Based Surveillance (WBS), which offers non-invasive, cost-effective means for analysing virus trends at the sewershed level. From 2020 onwards, wastewater has been recognized as an instrumental source of information for public health, with national and international authorities exploring options to implement national wastewater surveillance systems and increasingly relying on WBS as early warning of potential pathogen outbreaks. In Portugal, several pioneer projects joined the academia, water utilities and Public Administration around WBS. To validate WBS as an effective genomic surveillance strategy, it is crucial to collect long term performance data. In this work, we present one year of systematic SARS-CoV-2 wastewater surveillance in Portugal, representing 35% of the mainland population. We employed two complementary methods for lineage determination - allelic discrimination by RT-PCR and S gene sequencing. This combination allowed us to monitor variant evolution in near-real-time and identify low-frequency mutations. Over the course of this year-long study, spanning from May 2022 to April 2023, we successfully tracked the dominant Omicron sub-lineages, their progression and evolution, which aligned with concurrent clinical surveillance data. Our results underscore the effectiveness of WBS as a tracking system for virus variants, with the ability to unveil mutations undetected via massive sequencing of clinical samples from Portugal, demonstrating the ability of WBS to uncover new mutations and detect rare genetic variants. Our findings emphasize that knowledge of the genetic diversity of SARS-CoV-2 at the population level can be extended far beyond via the combination of routine clinical genomic surveillance with wastewater sequencing and genotyping.\nPost-pandemic memory T-cell response to SARS-CoV-2 is durable, broadly targeted and cross-reactive to hypermutated BA.2.86\nAuthors: Nesamari, R.; Omondi, M. A.; Hoft, M. A.; Ngomti, A.; Baguma, R.; Nkayi, A. A.; Besethi, A. S.; Magugu, S. F.; Mosala, P.; Walters, A.; Clark, G. M.; Mennen, M.; Skelem, S.; Adriaanse, M.; Grifoni, A.; Sette, A.; Keeton, R. S.; Ntusi, N. A.; Riou, C.; Burgers, W. A.\nScore: 7.1, Published: 2023-10-30 DOI: 10.1101/2023.10.28.23297714\nThe COVID-19 post-pandemic period is characterised by infection waves of uncertain size (due to low rates of SARS-CoV-2 testing and notification), as well as limited uptake or global access to updated variant vaccines. Ongoing SARS-CoV-2 evolution has given rise to recombinant Omicron lineages that dominate globally (XBB.1), as well as the emergence of hypermutated variants (BA.2.86). In this context, durable and cross-reactive T-cell immune memory is critical for continued protection against severe COVID-19. We examined T-cell responses to SARS-CoV-2 approximately 1.5 years since Omicron first emerged. We describe sustained CD4+ and CD8+ spike-specific T-cell memory responses in healthcare workers in South Africa (n=39), most of whom had received 2 doses of Ad26.CoV2.S (Johnson \u0026 Johnson/Janssen) vaccine and experienced at least one SARS-CoV-2 infection. Spike-specific T cells were highly cross-reactive with all Omicron variants tested, including BA.2.86. Abundant non-spike (nucleocapsid and membrane)-specific T cells were detectable in most participants, augmenting the total T-cell resources available for protection. The bulk of SARS-CoV-2-specific T-cell responses had an early-differentiated phenotype, explaining their persistent nature. Thus, hybrid immunity leads to the accumulation of spike and non-spike T cells evident 3.5 years after the start of the pandemic, with preserved recognition of highly mutated SARS-CoV-2 variants. Long-term T-cell immune memory is likely to provide continued protection against severe outcomes of COVID-19. In BriefNesamari et al. investigate T-cell responses in the context of hybrid immunity 3.5 years after the start of the COVID-19 pandemic. They show that T-cell memory is highly durable and cross-reactive with recombinant variants XBB.1 and hypermutated BA.2.86. Abundant non-spike responses augment the overall T-cell response.\nWastewater detection of emerging arbovirus infections: Case study of Dengue in the United States\nAuthors: Wolfe, M. K.; Paulos, A.; Zulli, A.; Duong, D.; Shelden, B.; White, B.; Boehm, A.\nScore: 6.2, Published: 2023-10-28 DOI: 10.1101/2023.10.27.23297694\nClimate change and urbanization are increasing the distribution of insect vectors of infectious diseases. Dengue virus is an arbovirus that causes nearly 100 million symptomatic infections per year and is endemic in 124 countries, and the range of its mosquito vectors continues to increase. Surveillance of dengue virus infections is complicated by the fact that infections can be asymptomatic and symptoms may not be readily recognizable to clinicians. Here we show that wastewater monitoring can be used to detect dengue virus RNA to yield information about circulation of dengue infections in a community. We collected three samples of wastewater solids per week from three different wastewater treatment plants in Miami-Dade County, Florida where dengue infections have been locally acquired. Using molecular methods, we tested wastewater solids for RNA from the 4 dengue virus serotypes and consistently detected dengue virus 3 RNA at all three wastewater plants, and did not detect the other 3 serotypes. According to publicly available data on dengue infections, the vast majority of infections were caused by serotype 3. Wastewater detection of dengue virus RNA is possible with as few as 4.23 laboratory confirmed dengue cases per 1 million people, based on publicly available infection data. SynopsisDengue virus RNA was detected in wastewater solids in a location with local- and travel-associated dengue infections.\nThe impact of cirrhosis on the inflammatory milieu before and long-term after hepatitis C virus elimination by direct-acting antiviral therapy\nAuthors: Witte, M.; Oltmanns, C.; Tauwaldt, J.; Schmaus, H.; Mischke, J.; Grabert, G.; Bretthauer, M.; Deterding, K.; Maasoumy, B.; Wedemeyer, H.; Kacprowski, T.; Kraft, A. R. M.; Cornberg, M.\nScore: 4.7, Published: 2023-10-31 DOI: 10.1101/2023.10.31.23297828\nBackground and AimsChronic hepatitis C virus (HCV) infection can lead to cirrhosis, development of hepatocellular carcinoma (HCC) and several extrahepatic manifestations. A sustained virological response (SVR) is achieved with direct-acting antivirals (DAA) in over 95% of the patients, but sequelae do not improve in all patients, suggesting permanent biological alterations induced by HCV infection. Therefore, we investigated the influence of chronic HCV infection, viral elimination and cirrhosis on inflammatory immune mediators. Approach and ResultsIn 102 chronic HCV patients, 46 with and 56 without cirrhosis, 92 soluble immune mediators (SIM) were measured in plasma samples at therapy start, end of treatment and long-term follow-up (median 96 weeks). 39 HBsAg positive persons with HBeAg negative infection served as controls. At baseline, 42 SIM were altered in chronic HCV patients (adj.p \u003c0.05). Notably, patients with cirrhosis displayed a higher frequency and severity of alterations. At long-term follow-up, the SIM profile of the non-cirrhotic patients recovered to the level of the control group, while 41 SIM remained altered in cirrhotic patients. 33 of these SIM correlated with elastography, among them SIM linked to carcinogenesis as e.g. HGF, IL8 and IL6 (KEGG Pathways hsa05202, hsa05200). ConclusionsHCV-related changes in the inflammatory milieu can persist even after HCV elimination, specifically in cirrhotic patients. These changes are closely associated with liver damage and carcinogenesis. Our findings underscore the need for HCV elimination before extensive liver injury occurs and suggest further investigation of the relationship between persistent inflammatory milieu changes and long-term sequelae after HCV elimination. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=120 SRC=\"FIGDIR/small/23297828v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (39K): org.highwire.dtl.DTLVardef@2589a5org.highwire.dtl.DTLVardef@6fb4d7org.highwire.dtl.DTLVardef@1c9a2f7org.highwire.dtl.DTLVardef@ea3f77_HPS_FORMAT_FIGEXP M_FIG C_FIG\nSpatiotemporal dynamics and epidemiological impact of SARS-CoV-2 XBB lineages dissemination in Brazil in 2023\nAuthors: Arantes, I.; Gomes, M. F. C.; Ito, K.; Sarafim, S.; Miyajima, F.; Khouri, R.; Graf, T.; Siqueira, M.; Resende, P.; Naveca, F.; Bello, G.; COVID-19 FIOCRUZ Genomic Surveillance Network, Score: 4.2, Published: 2023-10-31 DOI: 10.1101/2023.10.30.23297466\nThe SARS-CoV-2 XBB is a group of highly immune-evasive lineages of the Omicron VOC that emerged by recombining BA.2-descendent lineages and spread worldwide during 2023. In this study, we combine SARS-CoV-2 genomic data (n = 11,065 sequences) with epidemiological data of Severe Acute Respiratory Infection (SARI) cases collected in Brazil between October 2022 and July 2023 to reconstruct the space-time dynamics and epidemiologic impact of XBB dissemination in the country. Our analyses revealed that the introduction and local emergence of lineages carrying convergent mutations within the Spike protein, especially F486P, F456L, and L455F, propelled the spread of XBB* lineages in Brazil. The average relative instantaneous reproduction numbers of XBB*+F486P, XBB*+F486P+F456L, and XBB*+F486P+ F456L+L455F lineages in Brazil were estimated to be 1.24, 1.33, and 1.48 higher than that of other co-circulating lineages (mainly BQ.1*/BE*), respectively. Despite such a growth advantage, the dissemination of these XBB* lineages had a reduced impact on Brazils epidemiological scenario concerning previous Omicron subvariants. The peak number of SARI cases from SARS-CoV-2 during the XBB wave was approximately 90%, 80%, and 70% lower than that observed during the previous BA.1*, BA.5*, and BQ.1* waves, respectively. These findings revealed the emergence of multiple XBB lineages with progressively increasing growth advantage, yet with relatively limited epidemiological impact in Brazil throughout 2023. The XBB*+F486P+F456L+L455F lineages stand out for their heightened transmissibility, warranting close monitoring in the months ahead.\n",
  "wordCount" : "3025",
  "inLanguage": "en",
  "datePublished": "2023-11-05T10:37:37Z",
  "dateModified": "2023-11-05T10:37:37Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 5, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.23297114">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.23297114" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.23297114">
        <p class="paperTitle">Plasma-based antigen persistence in the post-acute phase of SARS-CoV-2 infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.23297114" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.23297114" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peluso, M. J.; Swank, Z. N.; Goldberg, S. A.; Lu, S.; Dalhuisen, T.; Borberg, E.; Senussi, Y.; Luna, M. A.; Chang Song, C.; Clark, A.; Zamora, A.; Lew, M.; Viswanathan, B.; Huang, B.; Anglin, K.; Hoh, R.; Hsue, P. Y.; Durstenfeld, M. S.; Spinelli, M. A.; Glidden, D. V.; Henrich, T. J.; Kelly, J. D.; Deeks, S. G.; Walt, D. R.; Martin, J. N.</p>
        <p class="info">Score: 516.5, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.23297114' target='https://doi.org/10.1101/2023.10.24.23297114'> 10.1101/2023.10.24.23297114</a></p>
        <p class="abstract">BACKGROUNDAlthough RNA viruses like SARS-CoV-2 are generally thought to be transient, the persistence of viral components beyond the acute phase can be driven by a variety of virologic and immunologic factors. Recent studies have suggested that SARS-CoV-2 antigens may persist following COVID-19 but were limited by a lack of comparison to a large number of true negative control samples.

METHODSUsing single molecule array (Simoa) assays for SARS-CoV-2 spike, S1, and nucleocapsid antigen in plasma from 171 pandemic-era individuals in the post-acute phase of SARS-CoV-2 infection and 250 pre-pandemic control samples, we compared prevalence of antigen detection. We used logistic regression models and prevalence ratios (PRs) to assess the relationship between demographic and disease factors and antigen persistence.

RESULTSCompared to the proportion of antigen positivity in the pre-pandemic controls (2%), detection of any SARS-CoV-2 antigen was more frequent across all post-acute COVID-19 time bins (3-6 months: 12.6%, p&lt;0.001; 6-10 months, 10.7%, p=0.0002; 10-14 months, 7.5%, p=0.017). These differences were driven by spike protein for up to 14 months and nucleocapsid in the first 6 months after infection. The co-occurrence of multiple antigens at a single timepoint was uncommon. Hospitalization for acute COVID-19 (versus not hospitalized) and worse self-reported health during acute COVID-19 among those not hospitalized (versus more benign illness) were associated with higher prevalence of post-acute antigen detection (PR 1.86, p=0.03; PR 3.5, p=0.07, respectively) in the pandemic era.

CONCLUSIONSOur findings provide strong evidence that SARS-CoV-2 antigens can persist beyond the period of acute illness. The observation that more than 10% of plasma samples for over a year following initial SARS-CoV-2 infection contain detectable viral antigen, which are potentially immunogenic, has significant implications given the sheer number of people infected with SARS-CoV-2 to date. More work will be needed to determine whether these antigens have a causal role in post-acute sequelae of SARS-CoV-2 infection (PASC).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.23297597">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.23297597" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.23297597">
        <p class="paperTitle">Complement dysregulation is a predictive and therapeutically amenable feature of long COVID</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.23297597" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.23297597" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baillie, K.; Davies, H. E.; Keat, S. B. K.; Ladell, K.; Miners, K. L.; Jones, S. A.; Mellou, E.; Toonen, E. J. M.; Price, D. A.; Morgan, B. P.; Zelek, W. M.</p>
        <p class="info">Score: 132.3, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.23297597' target='https://doi.org/10.1101/2023.10.26.23297597'> 10.1101/2023.10.26.23297597</a></p>
        <p class="abstract">BackgroundLong COVID encompasses a heterogeneous set of ongoing symptoms that affect many individuals after recovery from infection with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). The underlying biological mechanisms nonetheless remain obscure, precluding accurate diagnosis and effective intervention. Complement dysregulation is a hallmark of acute COVID-19 but has not been investigated as a potential determinant of long COVID.

MethodsWe quantified a series of complement proteins, including markers of activation and regulation, in plasma samples from healthy convalescent individuals with a confirmed history of infection with SARS-CoV-2 and age/ethnicity/gender/infection/vaccine-matched patients with long COVID.

FindingsMarkers of classical (C1s-C1INH complex), alternative (Ba, iC3b), and terminal pathway (C5a, TCC) activation were significantly elevated in patients with long COVID. These markers in combination had a receiver operating characteristic predictive power of 0.794. Other complement proteins and regulators were also quantitatively different between healthy convalescent individuals and patients with long COVID. Generalized linear modeling further revealed that a clinically tractable combination of just four of these markers, namely the activation fragments iC3b, TCC, Ba, and C5a, had a predictive power of 0.785.

ConclusionsThese findings suggest that complement biomarkers could facilitate the diagnosis of long COVID and further suggest that currently available inhibitors of complement activation could be used to treat long COVID.

FundingThis work was funded by the National Institute for Health Research (COV-LT2-0041), the PolyBio Research Foundation, and the UK Dementia Research Institute.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.23293924">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.23293924" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.23293924">
        <p class="paperTitle">Published benefits of ivermectin use in Itajai, Brazil for COVID-19 infection, hospitalisation, and mortality are entirely explained by statistical artefacts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.23293924" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.23293924" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mills, R.; Pecanha Antonio, A. C.; Tucker-Kellogg, G.</p>
        <p class="info">Score: 419.3, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.23293924' target='https://doi.org/10.1101/2023.08.10.23293924'> 10.1101/2023.08.10.23293924</a></p>
        <p class="abstract">BackgroundTwo recent publications by Kerr et al. (Cureus 14(1):e21272; Cureus 14(8): e28624) reported dramatic effects of prophylactic ivermectin use for both prevention of COVID-19 and reduction of COVID-19-related hospitalisation and mortality, including a dose-dependent effect of ivermectin prophylaxis. These papers have gained an unusually large public influence: they were incorporated into debates around COVID-19 policies and may have contributed to decreased trust in vaccine efficacy and public health authorities more broadly. Both studies were based on retrospective observational analysis of city-wide registry data from the city of Itajai, Brazil from July-December 2020.

MethodsStarting with initially identified sources of error, we conducted a revised statistical analysis of available data, including data made available with the original papers and public data from the Brazil Ministry of Health. We identified additional uncorrected sources of bias and errors from the original analysis, including incorrect subject exclusion and missing subjects, analysis of longitudinal data with cross-sectional design, an enrolment time bias, and multiple sources of immortal time bias. In models assuming no actual effect from ivermectin use, we conducted Monte Carlo simulations to estimate the contribution of these biases to any observed effect.

ResultsUntreated statistical artefacts and methodological errors alone lead to dramatic apparent risk reduction associated with ivermectin use in both studies. The magnitude of apparent risk reduction from these artefacts is comparable to the results reported by the studies themselves, including apparent protection from infection, hospitalisation, and death, and including the reported apparent dose-response relationship.

ConclusionsThe inference of ivermectin effect reported in both papers is unsupported, as the observed effects are entirely explained by untreated statistical artefacts and methodological errors. Our re-analysis calls for caution in interpreting highly publicised observational studies and highlights the importance of common sources of bias in clinical research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.08.21.22279035">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.08.21.22279035" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.08.21.22279035">
        <p class="paperTitle">Viral respiratory epidemic modelling of societal segregation based on vaccination status</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.08.21.22279035" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.08.21.22279035" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hickey, J.; Rancourt, D. G.</p>
        <p class="info">Score: 60.3, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.08.21.22279035' target='https://doi.org/10.1101/2022.08.21.22279035'> 10.1101/2022.08.21.22279035</a></p>
        <p class="abstract">BackgroundSocietal segregation of unvaccinated people from public spaces has been a novel and controversial COVID-era public health practice in many countries. Models exploring potential consequences of vaccination-status-based segregation have not considered how segregation influences the contact frequencies in the segregated groups. We systematically investigate implementing effects of segregation on population-specific contact frequencies and show this critically determines the predicted epidemiological outcomes, focusing on the attack rates in the vaccinated and unvaccinated populations and the share of infections among vaccinated people that were due to contacts with infectious unvaccinated people.

MethodsWe describe a susceptible-infectious-recovered (SIR) two-population model for vaccinated and unvaccinated groups of individuals that transmit an infectious disease by person-to-person contact. The degree of segregation of the two groups, ranging from zero to complete segregation, is implemented using the like-to-like mixing approach developed for sexually-transmitted diseases, adapted for presumed SARS-CoV-2 transmission. We allow the contact frequencies for individuals in the two groups to be different and depend, with variable strength, on the degree of segregation.

ResultsSegregation can either increase or decrease the attack rate among the vaccinated, depending on the type of segregation (isolating or compounding), and the contagiousness of the disease. For diseases with low contagiousness, segregation can cause an attack rate in the vaccinated, which does not occur without segregation.

InterpretationThere is no predicted blanket epidemiological advantage to segregation, either for the vaccinated or the unvaccinated. Negative epidemiological consequences can occur for both groups.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.23297332">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.23297332" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.23297332">
        <p class="paperTitle">Postacute Sequelae SARS-CoV-2 Infection by Vaccination Status: A Six-Month Latent Class Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.23297332" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.23297332" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sun, X.; DeShazo, J. P.; Anatale-Tardiff, L.; Di Fusco, M.; Allen, K. E.; Porter, T. M.; Coetzer, H.; Lopez, S. M. C.; Puzniak, L.; Cappelleri, J. C.</p>
        <p class="info">Score: 60.7, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.23297332' target='https://doi.org/10.1101/2023.10.20.23297332'> 10.1101/2023.10.20.23297332</a></p>
        <p class="abstract">Symptoms post-SARS-CoV-2 infection may persist for months and cause significant impairment and impact to quality of life. Acute symptoms of SARS-CoV-2 infection are well studied, yet data on clusters of symptoms over time, or postacute sequelae of SARS-CoV-2 infection (PASC), are limited. We aim to characterize PASC phenotypes by identifying symptom clusters over a six-month period following infection in individuals vaccinated (boosted and not) and those unvaccinated.

Subjects with [&amp;ge;]1 self-reported symptom and positive RT-PCR for SARS-CoV-2 at CVS Health US test sites were recruited between January and April 2022. Patient-reported outcomes symptoms, heath-related quality of life (QoL), work productivity and activity impairment (WPAI) were captured at 1 month, 3 months, and 6 months post-acute infection.

Logistic regression and latent class analysis (LCA) were performed on 20 symptoms using baseline socio-demographic, clinical characteristics, and vaccination status as well as EQ-5and WPAI results as covariables. Subjects with more symptoms were associated with lower health-related quality of life, and worse WPAI scores.

LCA identified three phenotypes that are primarily differentiated by number of symptoms. These three phenotypes remained consistent across time periods. Vaccinated individuals were more likely to be in the low symptom burden latent classes at all time points compared to unvaccinated individuals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.23297774">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.23297774" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.23297774">
        <p class="paperTitle">Systematic SARS-CoV-2 S Gene Sequencing in Wastewater Samples Enables Early Lineage Detection and Uncovers Rare Mutations in Portugal</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.23297774" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.23297774" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reis, A. C.; Pinto, D.; Monteiro, S.; Santos, R.; Martins, J. V.; Sousa, A.; Pascoa, R.; Lourinho, R.; Cunha, M. V.</p>
        <p class="info">Score: 11.7, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.23297774' target='https://doi.org/10.1101/2023.10.30.23297774'> 10.1101/2023.10.30.23297774</a></p>
        <p class="abstract">As the COVID-19 pandemic reached its peak, many countries implemented genomic surveillance systems to track the evolution and transmission of SARS-CoV-2. Transition from the pandemic to the endemic phase prioritized alternative testing strategies to maintain effective epidemic surveillance at the population level, with less intensive sequencing efforts. One such promising approach was Wastewater-Based Surveillance (WBS), which offers non-invasive, cost-effective means for analysing virus trends at the sewershed level. From 2020 onwards, wastewater has been recognized as an instrumental source of information for public health, with national and international authorities exploring options to implement national wastewater surveillance systems and increasingly relying on WBS as early warning of potential pathogen outbreaks. In Portugal, several pioneer projects joined the academia, water utilities and Public Administration around WBS.

To validate WBS as an effective genomic surveillance strategy, it is crucial to collect long term performance data. In this work, we present one year of systematic SARS-CoV-2 wastewater surveillance in Portugal, representing 35% of the mainland population. We employed two complementary methods for lineage determination - allelic discrimination by RT-PCR and S gene sequencing. This combination allowed us to monitor variant evolution in near-real-time and identify low-frequency mutations.

Over the course of this year-long study, spanning from May 2022 to April 2023, we successfully tracked the dominant Omicron sub-lineages, their progression and evolution, which aligned with concurrent clinical surveillance data. Our results underscore the effectiveness of WBS as a tracking system for virus variants, with the ability to unveil mutations undetected via massive sequencing of clinical samples from Portugal, demonstrating the ability of WBS to uncover new mutations and detect rare genetic variants.

Our findings emphasize that knowledge of the genetic diversity of SARS-CoV-2 at the population level can be extended far beyond via the combination of routine clinical genomic surveillance with wastewater sequencing and genotyping.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.28.23297714">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.28.23297714" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.28.23297714">
        <p class="paperTitle">Post-pandemic memory T-cell response to SARS-CoV-2 is durable, broadly targeted and cross-reactive to hypermutated BA.2.86</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.28.23297714" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.28.23297714" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nesamari, R.; Omondi, M. A.; Hoft, M. A.; Ngomti, A.; Baguma, R.; Nkayi, A. A.; Besethi, A. S.; Magugu, S. F.; Mosala, P.; Walters, A.; Clark, G. M.; Mennen, M.; Skelem, S.; Adriaanse, M.; Grifoni, A.; Sette, A.; Keeton, R. S.; Ntusi, N. A.; Riou, C.; Burgers, W. A.</p>
        <p class="info">Score: 7.1, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.28.23297714' target='https://doi.org/10.1101/2023.10.28.23297714'> 10.1101/2023.10.28.23297714</a></p>
        <p class="abstract">The COVID-19 post-pandemic period is characterised by infection waves of uncertain size (due to low rates of SARS-CoV-2 testing and notification), as well as limited uptake or global access to updated variant vaccines. Ongoing SARS-CoV-2 evolution has given rise to recombinant Omicron lineages that dominate globally (XBB.1), as well as the emergence of hypermutated variants (BA.2.86). In this context, durable and cross-reactive T-cell immune memory is critical for continued protection against severe COVID-19. We examined T-cell responses to SARS-CoV-2 approximately 1.5 years since Omicron first emerged. We describe sustained CD4&#43; and CD8&#43; spike-specific T-cell memory responses in healthcare workers in South Africa (n=39), most of whom had received 2 doses of Ad26.CoV2.S (Johnson &amp; Johnson/Janssen) vaccine and experienced at least one SARS-CoV-2 infection. Spike-specific T cells were highly cross-reactive with all Omicron variants tested, including BA.2.86. Abundant non-spike (nucleocapsid and membrane)-specific T cells were detectable in most participants, augmenting the total T-cell resources available for protection. The bulk of SARS-CoV-2-specific T-cell responses had an early-differentiated phenotype, explaining their persistent nature. Thus, hybrid immunity leads to the accumulation of spike and non-spike T cells evident 3.5 years after the start of the pandemic, with preserved recognition of highly mutated SARS-CoV-2 variants. Long-term T-cell immune memory is likely to provide continued protection against severe outcomes of COVID-19.

In BriefNesamari et al. investigate T-cell responses in the context of hybrid immunity 3.5 years after the start of the COVID-19 pandemic. They show that T-cell memory is highly durable and cross-reactive with recombinant variants XBB.1 and hypermutated BA.2.86. Abundant non-spike responses augment the overall T-cell response.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.23297694">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.23297694" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.23297694">
        <p class="paperTitle">Wastewater detection of emerging arbovirus infections: Case study of Dengue in the United States</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.23297694" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.23297694" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wolfe, M. K.; Paulos, A.; Zulli, A.; Duong, D.; Shelden, B.; White, B.; Boehm, A.</p>
        <p class="info">Score: 6.2, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.23297694' target='https://doi.org/10.1101/2023.10.27.23297694'> 10.1101/2023.10.27.23297694</a></p>
        <p class="abstract">Climate change and urbanization are increasing the distribution of insect vectors of infectious diseases. Dengue virus is an arbovirus that causes nearly 100 million symptomatic infections per year and is endemic in 124 countries, and the range of its mosquito vectors continues to increase. Surveillance of dengue virus infections is complicated by the fact that infections can be asymptomatic and symptoms may not be readily recognizable to clinicians. Here we show that wastewater monitoring can be used to detect dengue virus RNA to yield information about circulation of dengue infections in a community. We collected three samples of wastewater solids per week from three different wastewater treatment plants in Miami-Dade County, Florida where dengue infections have been locally acquired. Using molecular methods, we tested wastewater solids for RNA from the 4 dengue virus serotypes and consistently detected dengue virus 3 RNA at all three wastewater plants, and did not detect the other 3 serotypes. According to publicly available data on dengue infections, the vast majority of infections were caused by serotype 3. Wastewater detection of dengue virus RNA is possible with as few as 4.23 laboratory confirmed dengue cases per 1 million people, based on publicly available infection data.

SynopsisDengue virus RNA was detected in wastewater solids in a location with local- and travel-associated dengue infections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.23297828">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.23297828" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.23297828">
        <p class="paperTitle">The impact of cirrhosis on the inflammatory milieu before and long-term after hepatitis C virus elimination by direct-acting antiviral therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.23297828" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.23297828" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Witte, M.; Oltmanns, C.; Tauwaldt, J.; Schmaus, H.; Mischke, J.; Grabert, G.; Bretthauer, M.; Deterding, K.; Maasoumy, B.; Wedemeyer, H.; Kacprowski, T.; Kraft, A. R. M.; Cornberg, M.</p>
        <p class="info">Score: 4.7, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.23297828' target='https://doi.org/10.1101/2023.10.31.23297828'> 10.1101/2023.10.31.23297828</a></p>
        <p class="abstract">Background and AimsChronic hepatitis C virus (HCV) infection can lead to cirrhosis, development of hepatocellular carcinoma (HCC) and several extrahepatic manifestations. A sustained virological response (SVR) is achieved with direct-acting antivirals (DAA) in over 95% of the patients, but sequelae do not improve in all patients, suggesting permanent biological alterations induced by HCV infection. Therefore, we investigated the influence of chronic HCV infection, viral elimination and cirrhosis on inflammatory immune mediators.

Approach and ResultsIn 102 chronic HCV patients, 46 with and 56 without cirrhosis, 92 soluble immune mediators (SIM) were measured in plasma samples at therapy start, end of treatment and long-term follow-up (median 96 weeks). 39 HBsAg positive persons with HBeAg negative infection served as controls.

At baseline, 42 SIM were altered in chronic HCV patients (adj.p &lt;0.05). Notably, patients with cirrhosis displayed a higher frequency and severity of alterations. At long-term follow-up, the SIM profile of the non-cirrhotic patients recovered to the level of the control group, while 41 SIM remained altered in cirrhotic patients. 33 of these SIM correlated with elastography, among them SIM linked to carcinogenesis as e.g. HGF, IL8 and IL6 (KEGG Pathways hsa05202, hsa05200).

ConclusionsHCV-related changes in the inflammatory milieu can persist even after HCV elimination, specifically in cirrhotic patients. These changes are closely associated with liver damage and carcinogenesis. Our findings underscore the need for HCV elimination before extensive liver injury occurs and suggest further investigation of the relationship between persistent inflammatory milieu changes and long-term sequelae after HCV elimination.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=120 SRC=&#34;FIGDIR/small/23297828v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (39K):
org.highwire.dtl.DTLVardef@2589a5org.highwire.dtl.DTLVardef@6fb4d7org.highwire.dtl.DTLVardef@1c9a2f7org.highwire.dtl.DTLVardef@ea3f77_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.23297466">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.23297466" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.23297466">
        <p class="paperTitle">Spatiotemporal dynamics and epidemiological impact of SARS-CoV-2 XBB lineages dissemination in Brazil in 2023</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.23297466" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.23297466" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arantes, I.; Gomes, M. F. C.; Ito, K.; Sarafim, S.; Miyajima, F.; Khouri, R.; Graf, T.; Siqueira, M.; Resende, P.; Naveca, F.; Bello, G.; COVID-19 FIOCRUZ Genomic Surveillance Network,  </p>
        <p class="info">Score: 4.2, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.23297466' target='https://doi.org/10.1101/2023.10.30.23297466'> 10.1101/2023.10.30.23297466</a></p>
        <p class="abstract">The SARS-CoV-2 XBB is a group of highly immune-evasive lineages of the Omicron VOC that emerged by recombining BA.2-descendent lineages and spread worldwide during 2023. In this study, we combine SARS-CoV-2 genomic data (n = 11,065 sequences) with epidemiological data of Severe Acute Respiratory Infection (SARI) cases collected in Brazil between October 2022 and July 2023 to reconstruct the space-time dynamics and epidemiologic impact of XBB dissemination in the country. Our analyses revealed that the introduction and local emergence of lineages carrying convergent mutations within the Spike protein, especially F486P, F456L, and L455F, propelled the spread of XBB* lineages in Brazil. The average relative instantaneous reproduction numbers of XBB*&#43;F486P, XBB*&#43;F486P&#43;F456L, and XBB*&#43;F486P&#43; F456L&#43;L455F lineages in Brazil were estimated to be 1.24, 1.33, and 1.48 higher than that of other co-circulating lineages (mainly BQ.1*/BE*), respectively. Despite such a growth advantage, the dissemination of these XBB* lineages had a reduced impact on Brazils epidemiological scenario concerning previous Omicron subvariants. The peak number of SARI cases from SARS-CoV-2 during the XBB wave was approximately 90%, 80%, and 70% lower than that observed during the previous BA.1*, BA.5*, and BQ.1* waves, respectively. These findings revealed the emergence of multiple XBB lineages with progressively increasing growth advantage, yet with relatively limited epidemiological impact in Brazil throughout 2023. The XBB*&#43;F486P&#43;F456L&#43;L455F lineages stand out for their heightened transmissibility, warranting close monitoring in the months ahead.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
