<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Safety and Immunogenicity of XBB.1.5-Containing mRNA Vaccines
Authors: Chalkias, S.; McGhee, N.; Whatley, J. L.; Essink, B.; Brosz, A.; Tomassini, J. E.; Girard, B.; Wu, K.; Edwards, D. K.; Nasir, A.; Lee, D.; Avena, L. E.; Feng, J.; Deng, W.; Montefiori, D. C.; Baden, L. R.; Miller, J. M.; Das, R.
Score: 3355.8, Published: 2023-09-07 DOI: 10.1101/2023.08.22.23293434
BackgroundSubvariants of the severe acute respiratory syndrome coronavirus (SARS-CoV-2) omicron XBB-lineage have the potential to escape immunity provided by prior infection or vaccination.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Safety and Immunogenicity of XBB.1.5-Containing mRNA Vaccines
Authors: Chalkias, S.; McGhee, N.; Whatley, J. L.; Essink, B.; Brosz, A.; Tomassini, J. E.; Girard, B.; Wu, K.; Edwards, D. K.; Nasir, A.; Lee, D.; Avena, L. E.; Feng, J.; Deng, W.; Montefiori, D. C.; Baden, L. R.; Miller, J. M.; Das, R.
Score: 3355.8, Published: 2023-09-07 DOI: 10.1101/2023.08.22.23293434
BackgroundSubvariants of the severe acute respiratory syndrome coronavirus (SARS-CoV-2) omicron XBB-lineage have the potential to escape immunity provided by prior infection or vaccination." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-27T10:39:11+00:00" />
<meta property="article:modified_time" content="2023-09-27T10:39:11+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Safety and Immunogenicity of XBB.1.5-Containing mRNA Vaccines
Authors: Chalkias, S.; McGhee, N.; Whatley, J. L.; Essink, B.; Brosz, A.; Tomassini, J. E.; Girard, B.; Wu, K.; Edwards, D. K.; Nasir, A.; Lee, D.; Avena, L. E.; Feng, J.; Deng, W.; Montefiori, D. C.; Baden, L. R.; Miller, J. M.; Das, R.
Score: 3355.8, Published: 2023-09-07 DOI: 10.1101/2023.08.22.23293434
BackgroundSubvariants of the severe acute respiratory syndrome coronavirus (SARS-CoV-2) omicron XBB-lineage have the potential to escape immunity provided by prior infection or vaccination."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Safety and Immunogenicity of XBB.1.5-Containing mRNA Vaccines\nAuthors: Chalkias, S.; McGhee, N.; Whatley, J. L.; Essink, B.; Brosz, A.; Tomassini, J. E.; Girard, B.; Wu, K.; Edwards, D. K.; Nasir, A.; Lee, D.; Avena, L. E.; Feng, J.; Deng, W.; Montefiori, D. C.; Baden, L. R.; Miller, J. M.; Das, R.\nScore: 3355.8, Published: 2023-09-07 DOI: 10.1101/2023.08.22.23293434\nBackgroundSubvariants of the severe acute respiratory syndrome coronavirus (SARS-CoV-2) omicron XBB-lineage have the potential to escape immunity provided by prior infection or vaccination.",
  "keywords": [
    
  ],
  "articleBody": " Safety and Immunogenicity of XBB.1.5-Containing mRNA Vaccines\nAuthors: Chalkias, S.; McGhee, N.; Whatley, J. L.; Essink, B.; Brosz, A.; Tomassini, J. E.; Girard, B.; Wu, K.; Edwards, D. K.; Nasir, A.; Lee, D.; Avena, L. E.; Feng, J.; Deng, W.; Montefiori, D. C.; Baden, L. R.; Miller, J. M.; Das, R.\nScore: 3355.8, Published: 2023-09-07 DOI: 10.1101/2023.08.22.23293434\nBackgroundSubvariants of the severe acute respiratory syndrome coronavirus (SARS-CoV-2) omicron XBB-lineage have the potential to escape immunity provided by prior infection or vaccination. For Covid-19 immunizations beginning in the Fall 2023, the U.S. FDA has recommended updating to a monovalent omicron XBB.1.5-containing vaccine. MethodsIn this ongoing, phase 2/3 study participants were randomized 1:1 to receive 50-{micro}g doses of mRNA-1273.815 monovalent (50-{micro}g omicron XBB.1.5 spike mRNA) or mRNA-1273.231 bivalent (25-{micro}g omicron XBB.1.5 and 25-{micro}g omicron BA.4/BA.5 spike mRNAs) vaccines, administered as 5th doses, to adults who previously received a primary series and 3rd dose of an original mRNA coronavirus disease 2019 (Covid-19) vaccine, and a 4th dose of a bivalent (omicron BA.4/BA.5 and original SARS-CoV-2) vaccine. Interim safety and immunogenicity data 15 days post-vaccination are presented. ResultsIn April 2023, participants received mRNA-1273.815 (n=50) and mRNA-1273.231 (n=51). The median intervals from the prior dose of BA.4/BA.5-containing bivalent vaccine were 8.2 and 8.3 months for the mRNA-1273.815 and mRNA-1273.231 groups, respectively. Both vaccines increased neutralizing antibody (nAb) geometric mean titers against all variants tested at day 15 post-booster nAb compared to pre-booster levels. Geometric mean fold-rises from pre-booster titers after the monovalent booster were numerically higher against XBB.1.5, XBB.1.16 and SARS-CoV-2 (D614G) than those of the bivalent booster and were comparable against BA.4/BA.5 and BQ1.1 variants for both vaccines. The monovalent vaccine also elicited nAb responses against omicron XBB.2.3.2, EG.5.1, FL.1.5.1 and BA.2.86 that were similar to those against XBB.1.5 in a subset (n=20) of participants. The occurrence of solicited adverse reactions and unsolicited adverse events were overall similar to those previously reported for the original mRNA-1273 50-{micro}g and omicron BA.4/BA.5-containing bivalent mRNA-1273 vaccines. ConclusionIn this interim analysis, XBB.1.5-containing monovalent and bivalent vaccines elicited potent neutralizing responses against variants of the omicron XBB-lineage (XBB.1.5, XBB.1.6, XBB.2.3.2, EG.5.1, and FL.1.5.1) as well as the recently emerged BA.2.86 variant. The safety profile of the XBB.1.5-containing vaccine was consistent with those of prior vaccines. These results overall indicate that the XBB.1.5-containing mRNA-1273.815 vaccine has the potential to provide protection against these emerging variants and support the Covid-19 vaccine update in 2023-2024 to a monovalent XBB.1.5-containing vaccine.\nLiver abnormalities following SARS-CoV-2 infection in children under 10 years of age\nAuthors: Terebuh, P.; Olaker, V. R.; Kendall, E. K.; Kaelber, D. C.; Xu, R.; Davis, P. B.\nScore: 53.8, Published: 2023-09-23 DOI: 10.1101/2023.09.21.23295905\nObjectiveBeginning in October 2021 in the US and elsewhere, cases of severe pediatric hepatitis of unknown etiology were identified in young children. While the adenovirus and adenovirus-associated virus have emerged as leading etiologic suspects, we attempted to investigate a potential role for SARS-CoV-2 in the development of subsequent liver abnormalities. DesignWe conducted a study utilizing retrospective cohorts of de-identified, aggregated data from the electronic health records of over 100 million patients contributed by US health care organizations. ResultsCompared to propensity-score-matched children with other respiratory infections, children aged 1-10 years with COVID-19 had a higher risk of elevated transaminases (Hazard ratio (HR) (95% Confidence interval (CI)) 2.16 (1.74-2.69)) or total bilirubin (HR (CI) 3.02 (1.91-4.78)), or new diagnoses of liver diseases (HR (CI) 1.67 (1.21-2.30)) from one to six months after infection. Patients with pre-existing liver abnormalities, liver abnormalities surrounding acute infection, younger age (1-4 years), or illness requiring hospitalization all had similarly elevated risk. Children who developed liver abnormalities following COVID-19 had more pre-existing conditions than those who developed abnormalities following other infections. ConclusionThese results indicate that SARS-CoV-2 may prime the patient for subsequent development of liver infections or non-infectious liver diseases. While rare ([~]1 in 1,000), SARS-CoV-2 is a risk for subsequent abnormalities in liver function or the diagnosis of diseases of the liver. What is already known on this topicClusters of severe hepatitis in children in 2022 coincident with the increase in COVID-19 infections in children raised the question of the contribution of SARS-CoV-2 to the hepatitis outbreak, though it was soon determined that SARS-CoV-2 was not the primary etiologic agent. What this study addsSARS-CoV-2 may prime the patient for subsequent development of liver infections or non-infectious liver diseases. How this study might affect research, practice or policyDespite the mild initial disease in children, there may be longer term consequences of COVID-19, such as liver abnormalities, that warrants further investigation.\nRisk of Coronavirus Disease 2019 (COVID-19) among Those Up-to-Date and Not Up-to-Date on COVID-19 Vaccination by US CDC Criteria\nAuthors: Shrestha, N. K.; Burke, P. C.; Nowacki, A. S.; Gordon, S. M.\nScore: 5158.2, Published: 2023-09-14 DOI: 10.1101/2023.06.09.23290893\nBackgroundThe CDC recently defined being \"up-to-date\" on COVID-19 vaccination as having received at least one dose of a COVID-19 bivalent vaccine. The purpose of this study was to compare the risk of COVID-19 among those \"up-to-date\" and \"not up-to-date\". MethodsEmployees of Cleveland Clinic in Ohio, USA, in employment when the COVID-19 bivalent vaccine first became available, and still employed when the XBB lineages became dominant, were included. Cumulative incidence of COVID-19 since the XBB lineages became dominant was compared across the \"up-to-date\" and \"not up-to-date\" states, by treating COVID-19 bivalent vaccination as a time- dependent covariate whose value changed on receipt of the vaccine. Risk of COVID-19 by vaccination status was also evaluated using multivariable Cox proportional hazards regression adjusting for propensity to get tested for COVID-19, age, sex, most recent prior SARS-CoV-2 infection, and number of prior vaccine doses. ResultsCOVID-19 occurred in 1475 (3%) of 48 344 employees during the 100-day study period. The cumulative incidence of COVID-19 was lower in the \"not up-to-date\" than the \"up-to-date\" state. On multivariable analysis, being \"up-to-date\" was not associated with lower risk of COVID-19 (HR, 1.05; 95% C.I., 0.88-1.25; P-value, 0.58). Results were very similar when those 65 years and older were only considered \"up-to-date\" after 2 doses of the bivalent vaccine. ConclusionsSince the XBB lineages became dominant, adults \"up-to-date\" on COVID-19 vaccination by the CDC definition do not have a lower risk of COVID-19 than those \"not up-to-date\", bringing into question the value of this risk classification definition. SummaryAmong 48 344 working-aged Cleveland Clinic employees, those \"up-to-date\" on COVID-19 vaccination did not have a lower risk of COVID-19 than those not \"up-to-date\". The current CDC definition provides a meaningless classification of risk of COVID-19 in the adult population.\nThe impact of climate change and natural climate variability on the global distribution of Aedes aegypti\nAuthors: Kaye, A. R.; Obolski, U.; Sun, L.; Hurrell, J. W.; Tildesley, M. J.; Thompson, R. N.\nScore: 35.5, Published: 2023-08-31 DOI: 10.1101/2023.08.31.23294902\nAedes aegypti spread pathogens affecting humans, including the dengue, Zika and yellow fever viruses. Anthropogenic climate change is altering the spatial distribution of Ae. aegypti and therefore the locations at risk of vector-borne disease. In addition to climate change, natural climate variability, resulting from internal atmospheric processes and interactions between climate system components (e.g. atmosphere-land, atmosphere-ocean) determines climate outcomes. However, the combined effects of climate change and natural climate variability on future Ae. aegypti spread have not been assessed fully. We developed an ecological model in which Ae. aegypti population dynamics depend on climate variables (temperature and rainfall). We used 100 projections from the Community Earth System Model, a comprehensive climate model that simulates natural climate variability as well as anthropogenic climate change, in combination with our ecological model to generate a range of equally plausible scenarios describing the global distribution of suitable conditions for Ae. aegypti up to 2100. Like other studies, we project the poleward expansion of Ae. aegypti under climate change. However, the extent of spread varies considerably between projections, each under the same Shared Socioeconomic Pathway scenario (SSP3-7.0). For example, by 2100, climatic conditions in London may be suitable for Ae. aegypti for between one and five months in the year, depending on natural climate variability. Our results demonstrate that natural climate variability yields different possible future Ae. aegypti spread scenarios. This affects vector-borne disease risks, including the potential for some regions to experience outbreaks earlier than expected under climate change alone.\nQuantity of SARS-CoV-2 RNA copies exhaled per minute during natural breathing over the course of COVID-19 infection\nAuthors: Lane, G.; Zhou, G.; Hultquist, J. F.; Simons, L. M.; Lorenzo-Redondo, R.; Ozer, E. A.; McCarthy, D. M.; Ison, M. G.; Achenbach, C. J.; Wang, X.; Wai, C. M.; Wyatt, E.; Aalsburg, A.; Yang, Q.; Noto, T.; Alisoltani, A.; Ysselstein, D.; Awatramani, R.; Murphy, R.; Theron, G.; Zelano, C.\nScore: 599.4, Published: 2023-09-08 DOI: 10.1101/2023.09.06.23295138\nSARS-CoV-2 is spread through exhaled breath of infected individuals. A fundamental question in understanding transmission of SARS-CoV-2 is how much virus an individual is exhaling into the environment while they breathe, over the course of their infection. Research on viral load dynamics during COVID-19 infection has focused on internal swab specimens, which provide a measure of viral loads inside the respiratory tract, but not on breath. Therefore, the dynamics of viral shedding on exhaled breath over the course of infection are poorly understood. Here, we collected exhaled breath specimens from COVID-19 patients and used RTq-PCR to show that numbers of exhaled SARS-CoV-2 RNA copies during COVID-19 infection do not decrease significantly until day 8 from symptom-onset. COVID-19-positive participants exhaled an average of 80 SARS-CoV-2 viral RNA copies per minute during the first 8 days of infection, with significant variability both between and within individuals, including spikes over 800 copies a minute in some patients. After day 8, there was a steep drop to levels nearing the limit of detection, persisting for up to 20 days. We further found that levels of exhaled viral RNA increased with self-rated symptom-severity, though individual variation was high. Levels of exhaled viral RNA did not differ across age, sex, time of day, vaccination status or viral variant. Our data provide a fine-grained, direct measure of the number of SARS-CoV-2 viral copies exhaled per minute during natural breathing--including 312 breath specimens collected multiple times daily over the course of infection--in order to fill an important gap in our understanding of the time course of exhaled viral loads in COVID-19.\nContinued selection on cryptic SARS-CoV-2 observed in Missouri wastewater\nAuthors: Gregory, D.; Rushford, C.; Lin, C.-H.; Darby, C.; Niehues, N.; Semkiw, E.; Reynolds, M.; Wenzel, J.; Johnson, M. C.\nScore: 22.6, Published: 2023-09-20 DOI: 10.1101/2023.09.18.23295717\nDeep sequencing of wastewater to detect SARS-Cov-2 has been used during the COVID-19 pandemic to monitor viral variants as they appear and circulate in communities. SARS-CoV-2 lineages of an unknown source that have not been detected in clinical samples, referred to as cryptic lineages, are sometimes repeatedly detected in specific locations. We have continued to detect one such lineage previously seen in a Missouri site. This cryptic lineage has continued to evolve, indicating continued selective pressure similar to that observed in Omicron lineages. Author SummaryMonitoring sewage for SARS-CoV-2 has been an important part of understanding the dynamics of the viruss spread and persistence within and across communities during the pandemic. We and others have also observed variants appearing in wastewater that do not appear in clinical sampling. Many of these variants not only possess genomic changes identical to or at the same position as those that have been observed in variants of concern, particularly currently circulating Omicron variants, but often acquire the changes before they have been observed in clinical samples. We report here the continued observation of a variant in Missouri wastewater, but not in clinical sampling, that has continued to evolve, gaining genomic changes that often are the same and predate changes seen in clinical samples. These observation add to our understanding of the selective pressures driving the evolution of SAR-CoV-2.\nEffect of Higher-Dose Fluvoxamine vs Placebo on Time to Sustained Recovery in Outpatients with Mild to Moderate COVID-19: A Randomized Clinical Trial\nAuthors: Accelerating COVID-19 Therapeutic Interventions and Vaccines (ACTIV)-6 Study Group and Investigators, ; Naggie, S.\nScore: 133.6, Published: 2023-09-13 DOI: 10.1101/2023.09.12.23295424\nBackgroundThe impact of fluvoxamine in reducing symptom duration among outpatients with mild to moderate coronavirus disease 2019 (COVID-19) remains uncertain. Our objective was to assess the effectiveness of fluvoxamine 100 mg twice daily, compared with placebo, for treating mild to moderate COVID-19. MethodsThe ACTIV-6 platform randomized clinical trial aims to evaluate repurposed medications for mild to moderate COVID-19. Between August 25, 2022, and January 20, 2023, 1175 participants were enrolled at 103 US sites for evaluating fluvoxamine; participants were age [\u0026ge;]30 years with confirmed SARS-CoV-2 infection and [\u0026ge;]2 acute COVID-19 symptoms for [\u0026le;]7 days. Participants were randomized to receive fluvoxamine 50 mg twice daily on day 1 followed by 100 mg twice daily for 12 additional days or to placebo. The primary outcome was time to sustained recovery (defined as at least 3 consecutive days without symptoms). Secondary outcomes included time to death; time to hospitalization or death; a composite of hospitalization, urgent care visit, emergency department visit, or death; COVID clinical progression scale; and difference in mean time unwell. ResultsAmong participants who were randomized and received study drug, the median age was 50 years (IQR 40-60), 66% were female, 45% identified as Hispanic/Latino, and 77% reported [\u0026ge;]2 doses of a SARS-CoV-2 vaccine. Among 589 participants who received fluvoxamine and 586 who received placebo, differences in time to sustained recovery were not observed (adjusted hazard ratio [HR], 0.99 [95% credible interval, 0.89-1.09; P(efficacy) = 0.4]). Additionally, unadjusted, median time to sustained recovery was 10 days (95% CI 10-11) in both the intervention and placebo group. No deaths were reported. Thirty-five participants reported healthcare utilization events (a priori defined as death, hospitalization, emergency department/urgent care visit); 14 in the fluvoxamine group compared with 21 in the placebo group (HR 0.69; 95% CrI 0.27-1.21; P(efficacy)=0.86) There were 7 serious adverse events in 6 participants (2 with fluvoxamine and 4 with placebo). ConclusionsAmong outpatients with mild to moderate COVID-19, treatment with fluvoxamine does not reduce duration of COVID-19 symptoms. Trial RegistrationClinicalTrials.gov (NCT04885530).\nHuman origin ascertained for SARS-CoV-2 Omicron-like spike sequences detected in wastewater: a targeted surveillance study of a cryptic lineage in an urban sewershed\nAuthors: Shafer, M. M.; Bobholz, M. J.; Vuyk, W. C.; Gregory, D. A.; Roguet, A.; Haddock Soto, L. A.; Rushford, C.; Janssen, K. H.; Emmen, I. E.; Ries, H. J.; Pilch, H. E.; Mullen, P. A.; Fahney, R. B.; Wei, W.; Lambert, M.; Wenzel, J.; Halfmann, P.; Kawaoka, Y.; Wilson, N. A.; Friedrich, T. C.; Pray, I. W.; Westergaard, R.; O'Connor, D. H.; Johnson, M.\nScore: 1110.7, Published: 2023-09-22 DOI: 10.1101/2022.10.28.22281553\nBackgroundThe origin of novel SARS-CoV-2 spike sequences found in wastewater, without corresponding detection in clinical specimens, remains unclear. We sought to determine the origin of one such \"cryptic\" wastewater lineage by tracking and characterizing its persistence and genomic evolution over time. MethodsWe first detected a cryptic lineage in Wisconsin municipal wastewater in January 2022. By systematically sampling wastewater from targeted sub-sewershed lines and maintenance holes using compositing autosamplers, we traced this lineage (labeled WI-CL-001) to its source at a single commercial building. There we detected WI-CL-001 at concentrations as high as 2.7 x 109 genome copies per liter (gc/L) via RT-dPCR. In addition to using metagenomic 12s rRNA sequencing to determine the viruss host species, we also sequenced SARS-CoV-2 spike receptor binding domains (RBDs), and where possible, whole viral genomes to identify and characterize the evolution of this lineage over the 13 consecutive months that it was detectable. FindingsThe vast majority of 12s rRNAs sequenced from wastewater leaving the identified source building were human. Additionally, we generated over 100 viral RBD and whole genome sequences from wastewater samples containing the cryptic lineage collected between January 2022 and January 2023. These sequences contained a combination of fixed nucleotide substitutions characteristic of Pango lineage B.1.234, which circulated in humans in Wisconsin at low levels from October 2020 to February 2021. Despite this, mutations in the spike gene, and elsewhere, resembled those subsequently found in Omicron variants. InterpretationWe propose that prolonged detection of WI-CL-001 in wastewater represents persistent shedding of SARS-CoV-2 from a single human initially infected by an ancestral B.1.234 virus. The accumulation of convergent \"Omicron-like\" mutations in WI-CL-001s ancestral B.1.234 genome likely reflects persistent infection and extensive within-host evolution. FundingThe Rockefeller Foundation, Wisconsin Department of Health Services, Centers for Disease Control and Prevention (CDC), National Institute on Drug Abuse (NIDA), and the Center for Research on Influenza Pathogenesis and Transmission. Research in contextO_ST_ABSEvidence before this studyC_ST_ABSTo identify other studies that characterized unusual wastewater-specific SARS-CoV-2 lineages, we conducted a PubMed search using the keywords \"cryptic SARS-CoV-2 lineages\" or \"novel SARS-CoV-2 lineages\" in addition to \"wastewater\" on May 9, 2023. From the 18 papers retrieved, only two reported wastewater-specific cryptic lineages. These lineages were identified by members of our author team in wastewater from California, Missouri, and New York City. None of these could be definitively traced to a specific source. A third study in Nevada identified a unique recombinant variant (designated Pango lineage XL) in wastewater, which was also discovered in two clinical specimens from the same community. However, it was unclear whether the clinical specimens collected were from the same individual(s) responsible for the virus detected in the wastewater. To our knowledge, no prior study has successfully traced novel SARS-CoV-2 lineages detected in wastewater back to a specific location. How and where cryptic lineages are introduced into wastewater is not known. The added value of this studyThis study documents the presence and likely source of a novel and highly divergent cryptic SARS-CoV-2 lineage detected in Wisconsin wastewater for 13 months. In contrast to previously reported cryptic lineages, we successfully traced the lineage (WI-CL-001) to a single commercial building with approximately 30 employees. The exceptionally high viral RNA concentrations at the source building facilitated the tracing effort and allowed for the sequencing of WI-CL-001s whole genome, expanding our view of the lineages mutational landscape beyond the spike gene. Implications of all the available evidenceWI-CL-001s persistence in wastewater, its heavily mutated Omicron-like genotype, and its identified point source at a human-occupied commercial building all support the hypothesis that cryptic wastewater lineages can arise from persistently infected humans. Because cryptic wastewater lineages have some amino acid changes that subsequently emerge in circulating viruses, increased global monitoring of such lineages could help forecast variants that may arise in the future.\nA highly divergent SARS-CoV-2 lineage B.1.1 sample in a patient with long-term COVID-19\nAuthors: Nabieva, E.; Komissarov, A. B.; Klink, G. V.; Zaitsev, S. V.; Sergeeva, M.; Fadeev, A. V.; Komissarova, K.; Ivanova, A.; Pisareva, M.; Kudrya, K.; Danilenko, D.; Lioznov, D.; Hisner, R.; Gueli, F.; Peacock, T. P.; Roemer, C.; Bazykin, G. A.\nScore: 75.7, Published: 2023-09-17 DOI: 10.1101/2023.09.14.23295379\nWe report the genomic analysis of a highly divergent SARS-CoV-2 sample obtained in October 2022 from an HIV+ patient with presumably long-term COVID-19 infection. Phylogenetic analysis indicates that the sample is characterized by a gain of 89 mutations since divergence from its nearest sequenced neighbor, which had been collected in September 2020 and belongs to the B.1.1 lineage, largely extinct in 2022. 33 of these mutations were coding and occurred in the Spike protein. Of these, 17 are lineage-defining in some of the variants of concern (VOCs) or are in sites where another mutation is lineage-defining in a variant of concern, and/or shown to be involved in antibody evasion, and/or detected in other cases of persistent COVID-19; these include some \"usual suspects,\" such as Spike:L452R, E484Q, K417T, Y453F, and N460K. Molecular clock analysis indicates that mutations in this lineage accumulated at an increased rate compared to the ancestral B.1.1 strain. This increase is driven by the accumulation of nonsynonymous mutations, for an average dN/dS value of 2.2, indicating strong positive selection during within-patient evolution. Additionally, there is reason to believe that the virus had persisted for at least some time in the gastrointestinal tract, as evidenced by the presence of mutations that are rare in the general population samples but common in samples from wastewater. Our analysis adds to the growing body of research on evolution of SARS-CoV-2 in chronically infected patients and its relationship to the emergence of variants of concern.\nGenomic Epidemiologic Investigation of a Multispecies Hospital Outbreak of NDM-5-Producing Enterobacterales Infections\nAuthors: Raabe, N. J.; Valek, A. L.; Griffith, M. P.; Mills, E.; Waggle, K.; Rangachar Srinivasa, V.; Ayres, A. M.; Bradford, C.; Creager, H.; Pless, L. L.; Sundermann, A. J.; Van Tyne, D.; Snyder, G. M.; Harrison, L. H.\nScore: 11.8, Published: 2023-09-01 DOI: 10.1101/2023.08.31.23294545\nBackgroundNew Delhi metallo-{beta}-lactamase (NDM) represents an emergent mechanism of carbapenem resistance associated with high mortality and limited antimicrobial treatment options. Because the blaNDM resistance gene is often carried on plasmids, traditional infection prevention and control (IP\u0026C) surveillance methods like speciation, antimicrobial resistance testing, and reactive whole genome sequencing (WGS) may not detect plasmid transfer in multispecies outbreaks. MethodsInitial outbreak detection of NDM-producing Enterobacterales identified at an acute care hospital occurred via traditional IP\u0026C methods and was supplemented by real-time WGS surveillance, which was performed weekly using the Illumina platform. To resolve NDM-encoding plasmids, we performed long-read Oxford Nanopore sequencing and constructed hybrid assemblies using Illumina and Nanopore sequencing data. Reports of relatedness between NDM-producing organisms and reactive WGS for suspected outbreaks were shared with the IP\u0026C team for assessment and intervention. FindingsWe observed a multispecies outbreak of NDM-5-producing Enterobacterales isolated from 15 patients between February 2021 and February 2023. The 19 clinical and surveillance isolates sequenced included seven bacterial species and each encoded the same NDM-5 plasmid, which showed high homology to NDM plasmids previously observed in Asia. WGS surveillance and epidemiologic investigation characterized ten horizontal plasmid transfer events and six bacterial transmission events between patients housed in varying hospital units. Transmission prevention focused on enhanced observation and adherence to basic infection prevention measures. InterpretationOur investigation revealed a complex, multispecies outbreak of NDM that involved multiple plasmid transfer and bacterial transmission events, increasing the complexity of outbreak identification and transmission prevention. Our investigation highlights the utility of combining traditional IP\u0026C and prospective genomic methods in identifying and containing plasmid-associated outbreaks. FundingThis work was funded in part by the National Institute of Allergy and Infectious Diseases, National Institutes of Health (NIH) (R01AI127472) (R21AI1783691). 5. SummaryWe investigated a multispecies outbreak of Enterobacterales harboring the same New-Delhi metallo-{beta}-lactamase-encoding plasmid using both traditional infection prevention and genomic approaches. Our investigation revealed a complex outbreak involving 7 bacterial species, including both bacterial transmission and plasmid transfer events.\n",
  "wordCount" : "3730",
  "inLanguage": "en",
  "datePublished": "2023-09-27T10:39:11Z",
  "dateModified": "2023-09-27T10:39:11Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta"><span>updated on September 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.23293434">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.23293434" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.23293434">
        <p class="paperTitle">Safety and Immunogenicity of XBB.1.5-Containing mRNA Vaccines</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.23293434" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.23293434" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chalkias, S.; McGhee, N.; Whatley, J. L.; Essink, B.; Brosz, A.; Tomassini, J. E.; Girard, B.; Wu, K.; Edwards, D. K.; Nasir, A.; Lee, D.; Avena, L. E.; Feng, J.; Deng, W.; Montefiori, D. C.; Baden, L. R.; Miller, J. M.; Das, R.</p>
        <p class="info">Score: 3355.8, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.23293434' target='https://doi.org/10.1101/2023.08.22.23293434'> 10.1101/2023.08.22.23293434</a></p>
        <p class="abstract">BackgroundSubvariants of the severe acute respiratory syndrome coronavirus (SARS-CoV-2) omicron XBB-lineage have the potential to escape immunity provided by prior infection or vaccination. For Covid-19 immunizations beginning in the Fall 2023, the U.S. FDA has recommended updating to a monovalent omicron XBB.1.5-containing vaccine.

MethodsIn this ongoing, phase 2/3 study participants were randomized 1:1 to receive 50-{micro}g doses of mRNA-1273.815 monovalent (50-{micro}g omicron XBB.1.5 spike mRNA) or mRNA-1273.231 bivalent (25-{micro}g omicron XBB.1.5 and 25-{micro}g omicron BA.4/BA.5 spike mRNAs) vaccines, administered as 5th doses, to adults who previously received a primary series and 3rd dose of an original mRNA coronavirus disease 2019 (Covid-19) vaccine, and a 4th dose of a bivalent (omicron BA.4/BA.5 and original SARS-CoV-2) vaccine. Interim safety and immunogenicity data 15 days post-vaccination are presented.

ResultsIn April 2023, participants received mRNA-1273.815 (n=50) and mRNA-1273.231 (n=51). The median intervals from the prior dose of BA.4/BA.5-containing bivalent vaccine were 8.2 and 8.3 months for the mRNA-1273.815 and mRNA-1273.231 groups, respectively. Both vaccines increased neutralizing antibody (nAb) geometric mean titers against all variants tested at day 15 post-booster nAb compared to pre-booster levels. Geometric mean fold-rises from pre-booster titers after the monovalent booster were numerically higher against XBB.1.5, XBB.1.16 and SARS-CoV-2 (D614G) than those of the bivalent booster and were comparable against BA.4/BA.5 and BQ1.1 variants for both vaccines. The monovalent vaccine also elicited nAb responses against omicron XBB.2.3.2, EG.5.1, FL.1.5.1 and BA.2.86 that were similar to those against XBB.1.5 in a subset (n=20) of participants. The occurrence of solicited adverse reactions and unsolicited adverse events were overall similar to those previously reported for the original mRNA-1273 50-{micro}g and omicron BA.4/BA.5-containing bivalent mRNA-1273 vaccines.

ConclusionIn this interim analysis, XBB.1.5-containing monovalent and bivalent vaccines elicited potent neutralizing responses against variants of the omicron XBB-lineage (XBB.1.5, XBB.1.6, XBB.2.3.2, EG.5.1, and FL.1.5.1) as well as the recently emerged BA.2.86 variant. The safety profile of the XBB.1.5-containing vaccine was consistent with those of prior vaccines. These results overall indicate that the XBB.1.5-containing mRNA-1273.815 vaccine has the potential to provide protection against these emerging variants and support the Covid-19 vaccine update in 2023-2024 to a monovalent XBB.1.5-containing vaccine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.23295905">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.23295905" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.23295905">
        <p class="paperTitle">Liver abnormalities following SARS-CoV-2 infection in children under 10 years of age</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.23295905" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.23295905" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Terebuh, P.; Olaker, V. R.; Kendall, E. K.; Kaelber, D. C.; Xu, R.; Davis, P. B.</p>
        <p class="info">Score: 53.8, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.23295905' target='https://doi.org/10.1101/2023.09.21.23295905'> 10.1101/2023.09.21.23295905</a></p>
        <p class="abstract">ObjectiveBeginning in October 2021 in the US and elsewhere, cases of severe pediatric hepatitis of unknown etiology were identified in young children. While the adenovirus and adenovirus-associated virus have emerged as leading etiologic suspects, we attempted to investigate a potential role for SARS-CoV-2 in the development of subsequent liver abnormalities.

DesignWe conducted a study utilizing retrospective cohorts of de-identified, aggregated data from the electronic health records of over 100 million patients contributed by US health care organizations.

ResultsCompared to propensity-score-matched children with other respiratory infections, children aged 1-10 years with COVID-19 had a higher risk of elevated transaminases (Hazard ratio (HR) (95% Confidence interval (CI)) 2.16 (1.74-2.69)) or total bilirubin (HR (CI) 3.02 (1.91-4.78)), or new diagnoses of liver diseases (HR (CI) 1.67 (1.21-2.30)) from one to six months after infection. Patients with pre-existing liver abnormalities, liver abnormalities surrounding acute infection, younger age (1-4 years), or illness requiring hospitalization all had similarly elevated risk. Children who developed liver abnormalities following COVID-19 had more pre-existing conditions than those who developed abnormalities following other infections.

ConclusionThese results indicate that SARS-CoV-2 may prime the patient for subsequent development of liver infections or non-infectious liver diseases. While rare ([~]1 in 1,000), SARS-CoV-2 is a risk for subsequent abnormalities in liver function or the diagnosis of diseases of the liver.

What is already known on this topicClusters of severe hepatitis in children in 2022 coincident with the increase in COVID-19 infections in children raised the question of the contribution of SARS-CoV-2 to the hepatitis outbreak, though it was soon determined that SARS-CoV-2 was not the primary etiologic agent.

What this study addsSARS-CoV-2 may prime the patient for subsequent development of liver infections or non-infectious liver diseases.

How this study might affect research, practice or policyDespite the mild initial disease in children, there may be longer term consequences of COVID-19, such as liver abnormalities, that warrants further investigation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.09.23290893">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.09.23290893" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.09.23290893">
        <p class="paperTitle">Risk of Coronavirus Disease 2019 (COVID-19) among Those Up-to-Date and Not Up-to-Date on COVID-19 Vaccination by US CDC Criteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.09.23290893" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.09.23290893" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shrestha, N. K.; Burke, P. C.; Nowacki, A. S.; Gordon, S. M.</p>
        <p class="info">Score: 5158.2, Published: 2023-09-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.09.23290893' target='https://doi.org/10.1101/2023.06.09.23290893'> 10.1101/2023.06.09.23290893</a></p>
        <p class="abstract">BackgroundThe CDC recently defined being &#34;up-to-date&#34; on COVID-19 vaccination as having received at least one dose of a COVID-19 bivalent vaccine. The purpose of this study was to compare the risk of COVID-19 among those &#34;up-to-date&#34; and &#34;not up-to-date&#34;.

MethodsEmployees of Cleveland Clinic in Ohio, USA, in employment when the COVID-19 bivalent vaccine first became available, and still employed when the XBB lineages became dominant, were included. Cumulative incidence of COVID-19 since the XBB lineages became dominant was compared across the &#34;up-to-date&#34; and &#34;not up-to-date&#34; states, by treating COVID-19 bivalent vaccination as a time- dependent covariate whose value changed on receipt of the vaccine. Risk of COVID-19 by vaccination status was also evaluated using multivariable Cox proportional hazards regression adjusting for propensity to get tested for COVID-19, age, sex, most recent prior SARS-CoV-2 infection, and number of prior vaccine doses.

ResultsCOVID-19 occurred in 1475 (3%) of 48 344 employees during the 100-day study period. The cumulative incidence of COVID-19 was lower in the &#34;not up-to-date&#34; than the &#34;up-to-date&#34; state. On multivariable analysis, being &#34;up-to-date&#34; was not associated with lower risk of COVID-19 (HR, 1.05; 95% C.I., 0.88-1.25; P-value, 0.58). Results were very similar when those 65 years and older were only considered &#34;up-to-date&#34; after 2 doses of the bivalent vaccine.

ConclusionsSince the XBB lineages became dominant, adults &#34;up-to-date&#34; on COVID-19 vaccination by the CDC definition do not have a lower risk of COVID-19 than those &#34;not up-to-date&#34;, bringing into question the value of this risk classification definition.

SummaryAmong 48 344 working-aged Cleveland Clinic employees, those &#34;up-to-date&#34; on COVID-19 vaccination did not have a lower risk of COVID-19 than those not &#34;up-to-date&#34;. The current CDC definition provides a meaningless classification of risk of COVID-19 in the adult population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.23294902">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.23294902" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.23294902">
        <p class="paperTitle">The impact of climate change and natural climate variability on the global distribution of Aedes aegypti</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.23294902" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.23294902" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaye, A. R.; Obolski, U.; Sun, L.; Hurrell, J. W.; Tildesley, M. J.; Thompson, R. N.</p>
        <p class="info">Score: 35.5, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.23294902' target='https://doi.org/10.1101/2023.08.31.23294902'> 10.1101/2023.08.31.23294902</a></p>
        <p class="abstract">Aedes aegypti spread pathogens affecting humans, including the dengue, Zika and yellow fever viruses. Anthropogenic climate change is altering the spatial distribution of Ae. aegypti and therefore the locations at risk of vector-borne disease. In addition to climate change, natural climate variability, resulting from internal atmospheric processes and interactions between climate system components (e.g. atmosphere-land, atmosphere-ocean) determines climate outcomes. However, the combined effects of climate change and natural climate variability on future Ae. aegypti spread have not been assessed fully. We developed an ecological model in which Ae. aegypti population dynamics depend on climate variables (temperature and rainfall). We used 100 projections from the Community Earth System Model, a comprehensive climate model that simulates natural climate variability as well as anthropogenic climate change, in combination with our ecological model to generate a range of equally plausible scenarios describing the global distribution of suitable conditions for Ae. aegypti up to 2100. Like other studies, we project the poleward expansion of Ae. aegypti under climate change. However, the extent of spread varies considerably between projections, each under the same Shared Socioeconomic Pathway scenario (SSP3-7.0). For example, by 2100, climatic conditions in London may be suitable for Ae. aegypti for between one and five months in the year, depending on natural climate variability. Our results demonstrate that natural climate variability yields different possible future Ae. aegypti spread scenarios. This affects vector-borne disease risks, including the potential for some regions to experience outbreaks earlier than expected under climate change alone.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.23295138">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.23295138" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.23295138">
        <p class="paperTitle">Quantity of SARS-CoV-2 RNA copies exhaled per minute during natural breathing over the course of COVID-19 infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.23295138" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.23295138" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lane, G.; Zhou, G.; Hultquist, J. F.; Simons, L. M.; Lorenzo-Redondo, R.; Ozer, E. A.; McCarthy, D. M.; Ison, M. G.; Achenbach, C. J.; Wang, X.; Wai, C. M.; Wyatt, E.; Aalsburg, A.; Yang, Q.; Noto, T.; Alisoltani, A.; Ysselstein, D.; Awatramani, R.; Murphy, R.; Theron, G.; Zelano, C.</p>
        <p class="info">Score: 599.4, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.23295138' target='https://doi.org/10.1101/2023.09.06.23295138'> 10.1101/2023.09.06.23295138</a></p>
        <p class="abstract">SARS-CoV-2 is spread through exhaled breath of infected individuals. A fundamental question in understanding transmission of SARS-CoV-2 is how much virus an individual is exhaling into the environment while they breathe, over the course of their infection. Research on viral load dynamics during COVID-19 infection has focused on internal swab specimens, which provide a measure of viral loads inside the respiratory tract, but not on breath. Therefore, the dynamics of viral shedding on exhaled breath over the course of infection are poorly understood. Here, we collected exhaled breath specimens from COVID-19 patients and used RTq-PCR to show that numbers of exhaled SARS-CoV-2 RNA copies during COVID-19 infection do not decrease significantly until day 8 from symptom-onset. COVID-19-positive participants exhaled an average of 80 SARS-CoV-2 viral RNA copies per minute during the first 8 days of infection, with significant variability both between and within individuals, including spikes over 800 copies a minute in some patients. After day 8, there was a steep drop to levels nearing the limit of detection, persisting for up to 20 days. We further found that levels of exhaled viral RNA increased with self-rated symptom-severity, though individual variation was high. Levels of exhaled viral RNA did not differ across age, sex, time of day, vaccination status or viral variant. Our data provide a fine-grained, direct measure of the number of SARS-CoV-2 viral copies exhaled per minute during natural breathing--including 312 breath specimens collected multiple times daily over the course of infection--in order to fill an important gap in our understanding of the time course of exhaled viral loads in COVID-19.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.23295717">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.23295717" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.23295717">
        <p class="paperTitle">Continued selection on cryptic SARS-CoV-2 observed in Missouri wastewater</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.23295717" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.23295717" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gregory, D.; Rushford, C.; Lin, C.-H.; Darby, C.; Niehues, N.; Semkiw, E.; Reynolds, M.; Wenzel, J.; Johnson, M. C.</p>
        <p class="info">Score: 22.6, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.23295717' target='https://doi.org/10.1101/2023.09.18.23295717'> 10.1101/2023.09.18.23295717</a></p>
        <p class="abstract">Deep sequencing of wastewater to detect SARS-Cov-2 has been used during the COVID-19 pandemic to monitor viral variants as they appear and circulate in communities. SARS-CoV-2 lineages of an unknown source that have not been detected in clinical samples, referred to as cryptic lineages, are sometimes repeatedly detected in specific locations. We have continued to detect one such lineage previously seen in a Missouri site. This cryptic lineage has continued to evolve, indicating continued selective pressure similar to that observed in Omicron lineages.

Author SummaryMonitoring sewage for SARS-CoV-2 has been an important part of understanding the dynamics of the viruss spread and persistence within and across communities during the pandemic. We and others have also observed variants appearing in wastewater that do not appear in clinical sampling. Many of these variants not only possess genomic changes identical to or at the same position as those that have been observed in variants of concern, particularly currently circulating Omicron variants, but often acquire the changes before they have been observed in clinical samples. We report here the continued observation of a variant in Missouri wastewater, but not in clinical sampling, that has continued to evolve, gaining genomic changes that often are the same and predate changes seen in clinical samples. These observation add to our understanding of the selective pressures driving the evolution of SAR-CoV-2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.23295424">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.23295424" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.23295424">
        <p class="paperTitle">Effect of Higher-Dose Fluvoxamine vs Placebo on Time to Sustained Recovery in Outpatients with Mild to Moderate COVID-19: A Randomized Clinical Trial</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.23295424" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.23295424" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Accelerating COVID-19 Therapeutic Interventions and Vaccines (ACTIV)-6 Study Group and Investigators,  ; Naggie, S.</p>
        <p class="info">Score: 133.6, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.23295424' target='https://doi.org/10.1101/2023.09.12.23295424'> 10.1101/2023.09.12.23295424</a></p>
        <p class="abstract">BackgroundThe impact of fluvoxamine in reducing symptom duration among outpatients with mild to moderate coronavirus disease 2019 (COVID-19) remains uncertain. Our objective was to assess the effectiveness of fluvoxamine 100 mg twice daily, compared with placebo, for treating mild to moderate COVID-19.

MethodsThe ACTIV-6 platform randomized clinical trial aims to evaluate repurposed medications for mild to moderate COVID-19. Between August 25, 2022, and January 20, 2023, 1175 participants were enrolled at 103 US sites for evaluating fluvoxamine; participants were age [&amp;ge;]30 years with confirmed SARS-CoV-2 infection and [&amp;ge;]2 acute COVID-19 symptoms for [&amp;le;]7 days. Participants were randomized to receive fluvoxamine 50 mg twice daily on day 1 followed by 100 mg twice daily for 12 additional days or to placebo. The primary outcome was time to sustained recovery (defined as at least 3 consecutive days without symptoms). Secondary outcomes included time to death; time to hospitalization or death; a composite of hospitalization, urgent care visit, emergency department visit, or death; COVID clinical progression scale; and difference in mean time unwell.

ResultsAmong participants who were randomized and received study drug, the median age was 50 years (IQR 40-60), 66% were female, 45% identified as Hispanic/Latino, and 77% reported [&amp;ge;]2 doses of a SARS-CoV-2 vaccine. Among 589 participants who received fluvoxamine and 586 who received placebo, differences in time to sustained recovery were not observed (adjusted hazard ratio [HR], 0.99 [95% credible interval, 0.89-1.09; P(efficacy) = 0.4]). Additionally, unadjusted, median time to sustained recovery was 10 days (95% CI 10-11) in both the intervention and placebo group. No deaths were reported. Thirty-five participants reported healthcare utilization events (a priori defined as death, hospitalization, emergency department/urgent care visit); 14 in the fluvoxamine group compared with 21 in the placebo group (HR 0.69; 95% CrI 0.27-1.21; P(efficacy)=0.86) There were 7 serious adverse events in 6 participants (2 with fluvoxamine and 4 with placebo).

ConclusionsAmong outpatients with mild to moderate COVID-19, treatment with fluvoxamine does not reduce duration of COVID-19 symptoms.

Trial RegistrationClinicalTrials.gov (NCT04885530).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.10.28.22281553">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.10.28.22281553" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.10.28.22281553">
        <p class="paperTitle">Human origin ascertained for SARS-CoV-2 Omicron-like spike sequences detected in wastewater: a targeted surveillance study of a cryptic lineage in an urban sewershed</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.10.28.22281553" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.10.28.22281553" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shafer, M. M.; Bobholz, M. J.; Vuyk, W. C.; Gregory, D. A.; Roguet, A.; Haddock Soto, L. A.; Rushford, C.; Janssen, K. H.; Emmen, I. E.; Ries, H. J.; Pilch, H. E.; Mullen, P. A.; Fahney, R. B.; Wei, W.; Lambert, M.; Wenzel, J.; Halfmann, P.; Kawaoka, Y.; Wilson, N. A.; Friedrich, T. C.; Pray, I. W.; Westergaard, R.; O&#39;Connor, D. H.; Johnson, M.</p>
        <p class="info">Score: 1110.7, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.10.28.22281553' target='https://doi.org/10.1101/2022.10.28.22281553'> 10.1101/2022.10.28.22281553</a></p>
        <p class="abstract">BackgroundThe origin of novel SARS-CoV-2 spike sequences found in wastewater, without corresponding detection in clinical specimens, remains unclear. We sought to determine the origin of one such &#34;cryptic&#34; wastewater lineage by tracking and characterizing its persistence and genomic evolution over time.

MethodsWe first detected a cryptic lineage in Wisconsin municipal wastewater in January 2022. By systematically sampling wastewater from targeted sub-sewershed lines and maintenance holes using compositing autosamplers, we traced this lineage (labeled WI-CL-001) to its source at a single commercial building. There we detected WI-CL-001 at concentrations as high as 2.7 x 109 genome copies per liter (gc/L) via RT-dPCR. In addition to using metagenomic 12s rRNA sequencing to determine the viruss host species, we also sequenced SARS-CoV-2 spike receptor binding domains (RBDs), and where possible, whole viral genomes to identify and characterize the evolution of this lineage over the 13 consecutive months that it was detectable.

FindingsThe vast majority of 12s rRNAs sequenced from wastewater leaving the identified source building were human. Additionally, we generated over 100 viral RBD and whole genome sequences from wastewater samples containing the cryptic lineage collected between January 2022 and January 2023. These sequences contained a combination of fixed nucleotide substitutions characteristic of Pango lineage B.1.234, which circulated in humans in Wisconsin at low levels from October 2020 to February 2021. Despite this, mutations in the spike gene, and elsewhere, resembled those subsequently found in Omicron variants.

InterpretationWe propose that prolonged detection of WI-CL-001 in wastewater represents persistent shedding of SARS-CoV-2 from a single human initially infected by an ancestral B.1.234 virus. The accumulation of convergent &#34;Omicron-like&#34; mutations in WI-CL-001s ancestral B.1.234 genome likely reflects persistent infection and extensive within-host evolution.

FundingThe Rockefeller Foundation, Wisconsin Department of Health Services, Centers for Disease Control and Prevention (CDC), National Institute on Drug Abuse (NIDA), and the Center for Research on Influenza Pathogenesis and Transmission.

Research in contextO_ST_ABSEvidence before this studyC_ST_ABSTo identify other studies that characterized unusual wastewater-specific SARS-CoV-2 lineages, we conducted a PubMed search using the keywords &#34;cryptic SARS-CoV-2 lineages&#34; or &#34;novel SARS-CoV-2 lineages&#34; in addition to &#34;wastewater&#34; on May 9, 2023. From the 18 papers retrieved, only two reported wastewater-specific cryptic lineages. These lineages were identified by members of our author team in wastewater from California, Missouri, and New York City. None of these could be definitively traced to a specific source.

A third study in Nevada identified a unique recombinant variant (designated Pango lineage XL) in wastewater, which was also discovered in two clinical specimens from the same community. However, it was unclear whether the clinical specimens collected were from the same individual(s) responsible for the virus detected in the wastewater. To our knowledge, no prior study has successfully traced novel SARS-CoV-2 lineages detected in wastewater back to a specific location. How and where cryptic lineages are introduced into wastewater is not known.

The added value of this studyThis study documents the presence and likely source of a novel and highly divergent cryptic SARS-CoV-2 lineage detected in Wisconsin wastewater for 13 months. In contrast to previously reported cryptic lineages, we successfully traced the lineage (WI-CL-001) to a single commercial building with approximately 30 employees. The exceptionally high viral RNA concentrations at the source building facilitated the tracing effort and allowed for the sequencing of WI-CL-001s whole genome, expanding our view of the lineages mutational landscape beyond the spike gene.

Implications of all the available evidenceWI-CL-001s persistence in wastewater, its heavily mutated Omicron-like genotype, and its identified point source at a human-occupied commercial building all support the hypothesis that cryptic wastewater lineages can arise from persistently infected humans. Because cryptic wastewater lineages have some amino acid changes that subsequently emerge in circulating viruses, increased global monitoring of such lineages could help forecast variants that may arise in the future.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.23295379">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.23295379" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.23295379">
        <p class="paperTitle">A highly divergent SARS-CoV-2 lineage B.1.1 sample in a patient with long-term COVID-19</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.23295379" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.23295379" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nabieva, E.; Komissarov, A. B.; Klink, G. V.; Zaitsev, S. V.; Sergeeva, M.; Fadeev, A. V.; Komissarova, K.; Ivanova, A.; Pisareva, M.; Kudrya, K.; Danilenko, D.; Lioznov, D.; Hisner, R.; Gueli, F.; Peacock, T. P.; Roemer, C.; Bazykin, G. A.</p>
        <p class="info">Score: 75.7, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.23295379' target='https://doi.org/10.1101/2023.09.14.23295379'> 10.1101/2023.09.14.23295379</a></p>
        <p class="abstract">We report the genomic analysis of a highly divergent SARS-CoV-2 sample obtained in October 2022 from an HIV&#43; patient with presumably long-term COVID-19 infection. Phylogenetic analysis indicates that the sample is characterized by a gain of 89 mutations since divergence from its nearest sequenced neighbor, which had been collected in September 2020 and belongs to the B.1.1 lineage, largely extinct in 2022. 33 of these mutations were coding and occurred in the Spike protein. Of these, 17 are lineage-defining in some of the variants of concern (VOCs) or are in sites where another mutation is lineage-defining in a variant of concern, and/or shown to be involved in antibody evasion, and/or detected in other cases of persistent COVID-19; these include some &#34;usual suspects,&#34; such as Spike:L452R, E484Q, K417T, Y453F, and N460K. Molecular clock analysis indicates that mutations in this lineage accumulated at an increased rate compared to the ancestral B.1.1 strain. This increase is driven by the accumulation of nonsynonymous mutations, for an average dN/dS value of 2.2, indicating strong positive selection during within-patient evolution. Additionally, there is reason to believe that the virus had persisted for at least some time in the gastrointestinal tract, as evidenced by the presence of mutations that are rare in the general population samples but common in samples from wastewater. Our analysis adds to the growing body of research on evolution of SARS-CoV-2 in chronically infected patients and its relationship to the emergence of variants of concern.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.23294545">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.23294545" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.23294545">
        <p class="paperTitle">Genomic Epidemiologic Investigation of a Multispecies Hospital Outbreak of NDM-5-Producing Enterobacterales Infections</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.23294545" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.23294545" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Raabe, N. J.; Valek, A. L.; Griffith, M. P.; Mills, E.; Waggle, K.; Rangachar Srinivasa, V.; Ayres, A. M.; Bradford, C.; Creager, H.; Pless, L. L.; Sundermann, A. J.; Van Tyne, D.; Snyder, G. M.; Harrison, L. H.</p>
        <p class="info">Score: 11.8, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.23294545' target='https://doi.org/10.1101/2023.08.31.23294545'> 10.1101/2023.08.31.23294545</a></p>
        <p class="abstract">BackgroundNew Delhi metallo-{beta}-lactamase (NDM) represents an emergent mechanism of carbapenem resistance associated with high mortality and limited antimicrobial treatment options. Because the blaNDM resistance gene is often carried on plasmids, traditional infection prevention and control (IP&amp;C) surveillance methods like speciation, antimicrobial resistance testing, and reactive whole genome sequencing (WGS) may not detect plasmid transfer in multispecies outbreaks.

MethodsInitial outbreak detection of NDM-producing Enterobacterales identified at an acute care hospital occurred via traditional IP&amp;C methods and was supplemented by real-time WGS surveillance, which was performed weekly using the Illumina platform. To resolve NDM-encoding plasmids, we performed long-read Oxford Nanopore sequencing and constructed hybrid assemblies using Illumina and Nanopore sequencing data. Reports of relatedness between NDM-producing organisms and reactive WGS for suspected outbreaks were shared with the IP&amp;C team for assessment and intervention.

FindingsWe observed a multispecies outbreak of NDM-5-producing Enterobacterales isolated from 15 patients between February 2021 and February 2023. The 19 clinical and surveillance isolates sequenced included seven bacterial species and each encoded the same NDM-5 plasmid, which showed high homology to NDM plasmids previously observed in Asia. WGS surveillance and epidemiologic investigation characterized ten horizontal plasmid transfer events and six bacterial transmission events between patients housed in varying hospital units. Transmission prevention focused on enhanced observation and adherence to basic infection prevention measures.

InterpretationOur investigation revealed a complex, multispecies outbreak of NDM that involved multiple plasmid transfer and bacterial transmission events, increasing the complexity of outbreak identification and transmission prevention. Our investigation highlights the utility of combining traditional IP&amp;C and prospective genomic methods in identifying and containing plasmid-associated outbreaks.

FundingThis work was funded in part by the National Institute of Allergy and Infectious Diseases, National Institutes of Health (NIH) (R01AI127472) (R21AI1783691).

5. SummaryWe investigated a multispecies outbreak of Enterobacterales harboring the same New-Delhi metallo-{beta}-lactamase-encoding plasmid using both traditional infection prevention and genomic approaches. Our investigation revealed a complex outbreak involving 7 bacterial species, including both bacterial transmission and plasmid transfer events.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
