<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19
Authors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19
Authors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-23T10:37:56+00:00" />
<meta property="article:modified_time" content="2023-08-23T10:37:56+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19
Authors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19\nAuthors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A.",
  "keywords": [
    
  ],
  "articleBody": " Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19\nAuthors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A.; Dalhuisen, T.; Durstenfeld, M. S.; Hsue, P. Y.; Kelly, J. D.; Kumar, N.; Martin, J. N.; Gambhir, A.; Somsouk, M.; Seo, Y.; Deeks, S. G.; Laszik, Z. G.; VanBrocklin, H. F.; Henrich, T. J.\nScore: 2036.6, Published: 2023-07-31 DOI: 10.1101/2023.07.27.23293177\nThe etiologic mechanisms of post-acute medical morbidities and unexplained symptoms (Long COVID) following SARS-CoV-2 infection are incompletely understood. There is growing evidence that viral persistence and immune dysregulation may play a major role. We performed whole-body positron emission tomography (PET) imaging in a cohort of 24 participants at time points ranging from 27 to 910 days following acute SARS-CoV-2 infection using a novel radiopharmaceutical agent, [18F]F-AraG, a highly selective tracer that allows for anatomical quantitation of activated T lymphocytes. Tracer uptake in the post-acute COVID group, which included those with and without Long COVID symptoms, was significantly higher compared to pre-pandemic controls in many anatomical regions, including the brain stem, spinal cord, bone marrow, nasopharyngeal and hilar lymphoid tissue, cardiopulmonary tissues, and gut wall. Although T cell activation tended to be higher in participants imaged closer to the time of the acute illness, tracer uptake was increased in participants imaged up to 2.5 years following SARS-CoV-2 infection. We observed that T cell activation in spinal cord and gut wall was associated with the presence of Long COVID symptoms. In addition, tracer uptake in lung tissue was higher in those with persistent pulmonary symptoms. Notably, increased T cell activation in these tissues was also observed in many individuals without Long COVID. Given the high [18F]F-AraG uptake detected in the gut, we obtained colorectal tissue for in situ hybridization SARS-CoV-2 RNA and immunohistochemical studies in a subset of participants with Long COVID symptoms. We identified cellular SARS-CoV-2 RNA in rectosigmoid lamina propria tissue in all these participants, ranging from 158 to 676 days following initial COVID-19 illness, suggesting that tissue viral persistence could be associated with long-term immunological perturbations.\nPublished benefits of ivermectin use in Itajai, Brazil for COVID-19 infection, hospitalisation, and mortality are entirely explained by statistical artefacts\nAuthors: Mills, R.; Pecanha Antonio, A. C.; Tucker-Kellogg, G.\nScore: 269.8, Published: 2023-08-15 DOI: 10.1101/2023.08.10.23293924\nBackgroundTwo recent publications by Kerr et al. (Cureus 14(1):e21272; Cureus 14(8): e28624) reported dramatic effects of prophylactic ivermectin use for both prevention of COVID-19 and reduction of COVID-19-related hospitalisation and mortality, including a dose-dependent effect of ivermectin prophylaxis. These papers have gained an unusually large public influence: they were incorporated into debates around COVID-19 policies and may have contributed to decreased trust in vaccine efficacy and public health authorities more broadly. Both studies were based on retrospective observational analysis of city-wide registry data from the city of Itajai, Brazil from July-December 2020. MethodsStarting with initially identified sources of error, we conducted a revised statistical analysis of available data, including data made available with the original papers and public data from the Brazil Ministry of Health. We identified additional uncorrected sources of bias and errors from the original analysis, including incorrect subject exclusion and missing subjects, an enrolment time bias, and multiple sources of immortal time bias. In models assuming no actual effect from ivermectin use, we conducted Monte Carlo simulations to estimate the contribution of these biases to any observed effect. ResultsUntreated statistical artefacts and methodological errors alone lead to dramatic apparent risk reduction associated with Ivermectin use in both studies. The magnitude of apparent risk reduction from these artefacts is comparable to the results reported by the studies themselves, including apparent protection from infection, hospitalisation, and death, and including the reported apparent dose-response relationship. ConclusionsThe inference of ivermectin efficacy reported in both papers is unsupported, as the observed effects are entirely explained by untreated statistical artefacts and methodological errors. Our re-analysis calls for caution in interpreting highly publicised observational studies and highlights the importance of common sources of bias in clinical research.\nA molnupiravir-associated mutational signature in global SARS-CoV-2 genomes\nAuthors: Sanderson, T.; Hisner, R.; Donovan-Banfield, I.; Hartman, H.; Lochen, A.; Peacock, T. P.; Ruis, C.\nScore: 2428.8, Published: 2023-08-18 DOI: 10.1101/2023.01.26.23284998\nMolnupiravir, an antiviral medication that has been widely used against SARS-CoV-2, acts by inducing mutations in the virus genome during replication. Most random mutations are likely to be deleterious to the virus, and many will be lethal, and so molnupiravir-induced elevated mutation rates reduce viral load2,3. However, if some patients treated with molnupiravir do not fully clear SARS-CoV-2 infections, there could be the potential for onward transmission of molnupiravir-mutated viruses. Here we show that SARS-CoV-2 sequencing databases contain extensive evidence of molnupiravir mutagenesis. Using a systematic approach, we find that a specific class of long phylogenetic branches, distinguished by a high proportion of G-to-A and C-to-T mutations, appear almost exclusively in sequences from 2022, after the introduction of molnupiravir treatment, and in countries and age-groups with widespread usage of the drug. We identify a mutational spectrum, with preferred nucleotide contexts, from viruses in patients known to have been treated with molnupiravir and show that its signature matches that seen in these long branches, in some cases with onwards transmission of molnupiravir-derived lineages. Finally, we analyse treatment records to confirm a direct association between these high G-to-A branches and the use of molnupiravir.\nAnalysis of SARS-CoV-2 Variant-Specific Serum Antibodies Post-Vaccination Utilizing Immortalized Human Hepatocyte-like Cells (HLC) to Assess Development of Protective Immunity\nAuthors: Collins, D. P.; Steer, C. J.\nScore: 11.3, Published: 2023-08-13 DOI: 10.1101/2023.08.08.23293863\nBackgroundOur previous studies demonstrated that SARS-CoV-2 spike proteins could bind to hepatocytes via the asialoglycoprotein receptor-1 (ASGR-1) facilitating direct infection by the SARS-CoV-2 virus. Immortalized E12-HLC expressed the phenotypic and biological properties of primary human hepatocytes, including their ability to bind spike proteins via ASGR-1 with exception of the spike 1 protein. This binding could be inhibited by spike protein-specific monoclonal antibodies. We used the same spike-blocking analysis to determine if post-vaccination serum was capable of blocking spike protein binding to HLC. Samples collected from subjects prior to, and post-vaccination were quantified for anti-variant-specific antibody (original wild type, alpha (), beta ({beta}), gamma ({gamma}) and delta ({delta}) variants) by a flow cytometry based immunofluorescent assay. Inhibition of variant spike protein binding to HLC and AT-2 (as a known model for spike 1 binding to the ACE-2 receptor) was analyzed by confocal microscopy. This study was designed to investigate the ability of post-vaccination antibodies to mediate immunity to spike S2, and to validate the utility of the E12-HLC in analyzing that immunity. MethodsSerum was collected from 10 individuals pre- and post-vaccination with the J\u0026J, Moderna or Pfizer vaccines. The serum samples were quantified for variant-specific antibodies in a flow cytometry-based immunofluorescent assay utilizing beads coated with biotinylated variant spike proteins (, {beta}, {gamma}, {delta}). Presence of variant-specific antibodies was visualized by anti-human IgG-Alexa 488. Inhibition of spike protein binding to cells was analyzed by immunofluorescent confocal analysis. Biotinylated variant spike proteins were preincubated with serum samples and then tested for binding to target cells. Binding was visualized by Streptavidin-Alexa 594. Results were compared to binding of unblocked spike variants. ResultsAll variant spike proteins tested bound to both the HLC and AT-2 cells. Pre-vaccination serum samples had no detectable reactivity to any of the variant spike proteins and were unable to inhibit binding of the variant spike proteins to either target cell. Post-vaccination serum samples demonstrated a progression of SARS-CoV-2 antibody levels from low early post-vaccination levels to higher levels at 2.5 months after vaccination. Concurrently, serum samples taken at those different timeframes demonstrated that serum obtained from shortly after vaccination were not as effective in blocking spike protein as serum obtained after 2.5 months post-vaccination. Antibody concentrations were not necessarily associated with better blocking of spike protein binding as spike variant-specific serum antibody concentrations varied significantly between subjects and within each subject. It was also demonstrated that vaccination with all the various available vaccines stimulated antibodies that inhibited binding of the available variant spike proteins to both HLC and AT-2 cells. ConclusionHLC, along with AT-2 cells, provides a useful platform to study the development of protective antibodies that prevent the binding SARS-CoV-2 spike proteins to target cells. It was shown that vaccination with the three available vaccines all elicited serum antibodies that were protective against binding of each of the variant spike proteins to both AT-2 and HLC cells. This study suggests that analysis of immune serum to block spike binding to target cells may be a more useful technique to assess protective immunity than quantitation of gross antibody alone.\nWaning of post-vaccination neutralizing antibody responses against SARS-CoV-2, a systematic literature review and meta-analysis\nAuthors: Jacobsen, H.; Sitaras, I.; Katzmarzyk, M.; Cobos Jimenez, V.; Naughton, R.; Higdon, M. M.; Deloria Knoll, M.\nScore: 71.9, Published: 2023-08-10 DOI: 10.1101/2023.08.08.23293864\nBackgroundMass COVID-19 vaccination and the continuous introduction of new viral variants of SARS-CoV-2, especially of Omicron subvariants, has resulted in an increase in the proportion of the population with hybrid immunity at various stages of waning protection. We systematically reviewed waning of post-vaccination neutralizing antibody titers in different immunological settings to investigate potential differences. MethodsWe searched for studies providing data for post-vaccination neutralizing antibody responses against SARS-CoV-2 in PubMed, bioRxiv, and medRxiv from Dec 15, 2021, to Jan 31, 2023, using keywords related to COVID-19, vaccination, and antibody neutralization. We used random effects meta-regression to estimate the average fold-reduction in post-vaccination neutralizing antibody titers against the Index strain or Omicron BA.1. from month 1 to month 6 post last dose, stratified by vaccination regimen (primary or booster) and infection-naive vs hybrid-immune status. FindingsIn total, 26 studies reporting longitudinal post-vaccination neutralizing antibody titers were included. Neutralization titers against the Index variant were available from all studies for infection-naive participants, and from nine for hybrid-immune participants. Against Omicron BA.1, nine and eight studies were available for infection-naive and hybrid-immune cohorts, respectively. In infection-naive cohorts, post-vaccination neutralization titers against the Index strain waned 5.1-fold (95% CI 3.4-7.8) from month 1 to month 6 following primary regimen and 3.8-fold (95% CI 2.4-5.9) following the booster. Titers against Omicron BA.1 waned 5.9-fold (95% CI 3.8-9.0) in infection-naive, post-booster cohorts. In hybrid-immune, post-primary vaccination cohorts, titers waned 3.7-fold (95% CI 1.7-7.9) against the Index strain and 5.0-fold (95% CI 1.1-21.8) against Omicron BA.1. InterpretationNo obvious differences in waning between post-primary or post-boost vaccination were observed for vaccines used widely to date, nor between infection-naive and hybrid-immune participants. Titers against Omicron BA.1 may wane faster compared to Index titers, which may worsen for more recent Omicron sub-variants and should be monitored. Relatively small datasets limit the precision of our current analysis; further investigation is needed when more data become available. However, based on our current findings, striking differences in waning for the analyzed and future comparisons are unlikely.\nTemporal trends of dengue seroprevalence among children in coastal Kenya, 1998-2018: a longitudinal cohort study\nAuthors: Karanja, H. K.; Orindi, B.; Gitonga, J. N.; Mugo, D.; Mwai, K.; Nyamwaya, D. K.; Omuoyo, D. O.; Kitsao, B. S.; Musyoki, J. N.; Wambua, J.; Otieno, E.; Ochola-Oyier, L. I.; Bejon, P.; Warimwe, G. M.\nScore: 8.1, Published: 2023-08-16 DOI: 10.1101/2023.08.13.23294039\nSerosurveys suggest widespread dengue virus (DENV) transmission in Africa but there is limited information on the temporal patterns of exposure. Here, we estimated the prevalence and incidence of DENV infections in coastal Kenya over a 20-year period (1998-2018). Sera (n=8038) obtained annually from a longitudinal cohort of 1847 children aged 15 years and below were screened for anti-DENV IgG antibodies. Anti-DENV IgG seroprevalence increased with age and peaked during outbreak years. Among 1354 children who were seronegative at recruitment, we observed an overall incidence (seroconversion) rate of 129.5 (95% CI 118.7-141.4) DENV infections per 1000 person-years. The highest incidence was observed in 2013 at 520 infections per 1000 person-years (95% CI 443.6-610.2) coinciding with a large DENV outbreak in coastal Kenya. Our data suggest long-term DENV exposure among children in coastal Kenya highlighting an urgent need for clinical surveillance to determine the associated health burden in this setting.\nCohort Profile: Prospective Cohort to Study the COVID-19 Immune Response in Retail Workers in Quebec, Canada (CISACOV)\nAuthors: Santerre, K.; Theriault, M.; Brousseau, N.; Rochette, S.; Pelletier, J.; Gilbert, C.; Masson, J.-F.; Baz, M.; Boudreau, D.; Trottier, S.\nScore: 4.8, Published: 2023-08-21 DOI: 10.1101/2023.08.18.23294172\nPurpose: Retail workers are an understudied occupational group that may have been at increased risk of contracting SARS-CoV-2 during the COVID-19 pandemic. Therefore, we set up a longitudinal cohort of participants working in this sector to better document the incidence of SARS-CoV-2 infection and the immune response to infection and/or vaccination in this group. Participants: A total of 304 participants were recruited between April 20, 2021 and October 22, 2021. They were invited to attend three visits (each separated by ~12 weeks) during which they provided blood samples and information on participant characteristics, COVID-19 symptoms, and vaccination. An extension phase of two additional visits was carried out between March 15th, 2022 and October 3rd, 2022 to document the impact of the Omicron variant among the 198 participants who were still eligible for recruitment. Participants were aged 18 to 75 and worked in grocery stores, hardware stores, bars or restaurants within the Quebec City metropolitan area (Canada). Findings to date: This article describes participants' demographic, socioeconomic, behavioral, clinical and occupational characteristics, and their COVID-19 symptoms (where applicable). It also describes SARS-CoV-2 vaccination status and any SARS-CoV-2 diagnostic test (i.e., PCR or rapid antigen) performed from the beginning of the pandemic until the last visit. Future plans: The incidence of SARS-CoV-2 infections will be assessed. The immune response (innate and acquired) to SARS-CoV-2 infection or vaccination will be studied using a variety of techniques, including reference and experimental enzyme-linked immunosorbent assays, microneutralization assays with live viruses, experimental pseudoneutralization with an angiotensin-converting enzyme 2-spike assay, peripheral blood mononuclear cells and neutrophil stimulation, and a proliferation assay based on carboxyfluorescein diacetate succinimidyl ester.\nGenomic and eco-epidemiological investigations in Uruguay reveal local Chikungunya virus transmission dynamics during its expansion across the Americas in 2023.\nAuthors: Burgueno, A.; Giovanetti, M.; Fonseca, V.; Morel, M. N.; Lima, M.; Castro, E.; Guimaraes, N. R.; Iani, F. C. M.; Bormida, V.; Cortinas, M. N.; Ramas, V.; Coppola, L.; Bento, A. I.; Franco, L.; Mendez Rico, J.; Lourenco, J.; Alcantara, L. C. J.; Chiparelli, H.\nScore: 3.9, Published: 2023-08-20 DOI: 10.1101/2023.08.17.23294156\nUruguay experienced its first Chikungunya virus outbreak in 2023, resulting in a significant burden to its healthcare system. We conducted analysis based on real-time genomic surveillance (30 novel whole genomes) to offer timely insights into recent local transmission dynamics and eco-epidemiological factors behind its emergence and spread in the country.\nSARS-CoV-2 Viral Clearance and Evolution Varies by Extent of Immunodeficiency\nAuthors: Li, Y.; Choudhary, M. C.; Regan, J.; Boucau, J.; Nathan, A.; Speidel, T.; Liew, M. Y.; Edelstein, G. E.; Kawano, Y.; Uddin, R.; Deo, R.; Marino, C.; Getz, M. A.; Reynold, Z.; Barry, M.; Gilbert, R. F.; Tien, D.; Sagar, S.; Vyas, T. D.; Flynn, J. P.; Hammond, S. P.; Novack, L. A.; Choi, B.; Cernadas, M.; Wallace, Z. S.; Sparks, J. A.; Vyas, J. M.; Seaman, M. S.; Gaiha, G. D.; Siedner, M. J.; Barczak, A. K.; Lemieux, J. E.; Li, J. Z.\nScore: 53.6, Published: 2023-08-07 DOI: 10.1101/2023.07.31.23293441\nDespite vaccination and antiviral therapies, immunocompromised individuals are at risk for prolonged SARS-CoV-2 infection, but the immune defects that predispose to persistent COVID-19 remain incompletely understood. In this study, we performed detailed viro-immunologic analyses of a prospective cohort of participants with COVID-19. The median time to nasal viral RNA and culture clearance in the severe hematologic malignancy/transplant group (S-HT) were 72 and 40 days, respectively, which were significantly longer than clearance rates in the severe autoimmune/B-cell deficient (S-A), non-severe, and non-immunocompromised groups (P\u003c0.001). Participants who were severely immunocompromised had greater SARS-CoV-2 evolution and a higher risk of developing antiviral treatment resistance. Both S-HT and S-A participants had diminished SARS-CoV-2-specific humoral, while only the S-HT group had reduced T cell-mediated responses. This highlights the varied risk of persistent COVID-19 across immunosuppressive conditions and suggests that suppression of both B and T cell responses results in the highest contributing risk of persistent infection.\nCoccidioidomycosis Emergence in South America: Exploring Northeastern Brazil's Epidemiological, Clinical, and Genomic Landscape\nAuthors: Eulalio, K. D.; Kollath, D. R.; Martins, L. M. S.; Filho, A. d. D.; Cavalcanti, M. d. A. S.; Moreira, L. M.; Tenorio, B. G.; Alves, L. G. d. B.; Yamauchi, D.; Benard, G.; Thompson, G. R.; nacher, m.; Stajich, J. S. E.; Bagagli, E.; Felipe, M. S. S.; Barker, B.; Trilles, L.; Teixeira, M. d. M.\nScore: 3.1, Published: 2023-08-17 DOI: 10.1101/2023.08.14.23294078\nCoccidioidomycosis is an invasive mycosis included in WHOs priority list. It is endemic and notifiable in the United States but neglected in Central and South America. We used a multi-institutional approach to assess whether disease characteristics, genetic variation in the pathogen or environmental factors affects the epidemiology of coccidioidomycosis and disease outcomes throughout the American continent. We identified 292 patients with coccidioidomycosis between 1978 and 2021 in the Piaui and Maranhao states of Brazil; the largest cases series reported outside the US/Mexico epidemic range. The male-to-female ratio was 57.4:1 and the main risk factor was armadillo hunting (91.1%) 4 to 30 days before symptom onset. Forty-two outbreaks involving two to six patients were observed. Most patients (92.8%) presented typical acute pulmonary disease, followed by disseminated (3.4%), chronic pulmonary (2.4%) and regressive pulmonary (1.4%). The most frequent clinical symptoms were cough (93%), fever (90%) and chest pain (77%). Mortality was observed in 8% of the patients. In 2004, and between 2015 and 2017, we observed a spike in coccidioidomycosis in Brazil, particularly in the state of Piaui. Unlike other main hotspots, the soil is acidic in this region and precipitation (p=0.015) and precipitation one-year prior (p=0.001) were predictors of higher coccidioidomycosis rates. The Brazilian strains are genotypically divergent from other described C. posadasii within the Texas/Mexico/South America clade. Coccidioidomycosis in Northeastern Brazil has a specific infection profile and armadillo hunters are at higher risk. Low pluviosity and extensive drought appear key to increasing the number of cases in Brazil. A unique C. posadasii lineage exists in Brazil; therefore, environmental, virulence, and/or pathogenesis traits may differ from other Coccidioides genotypes.\n",
  "wordCount" : "3075",
  "inLanguage": "en",
  "datePublished": "2023-08-23T10:37:56Z",
  "dateModified": "2023-08-23T10:37:56Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta"><span>updated on August 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.23293177">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.23293177" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.23293177">
        <p class="paperTitle">Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.23293177" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.23293177" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A.; Dalhuisen, T.; Durstenfeld, M. S.; Hsue, P. Y.; Kelly, J. D.; Kumar, N.; Martin, J. N.; Gambhir, A.; Somsouk, M.; Seo, Y.; Deeks, S. G.; Laszik, Z. G.; VanBrocklin, H. F.; Henrich, T. J.</p>
        <p class="info">Score: 2036.6, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.23293177' target='https://doi.org/10.1101/2023.07.27.23293177'> 10.1101/2023.07.27.23293177</a></p>
        <p class="abstract">The etiologic mechanisms of post-acute medical morbidities and unexplained symptoms (Long COVID) following SARS-CoV-2 infection are incompletely understood. There is growing evidence that viral persistence and immune dysregulation may play a major role. We performed whole-body positron emission tomography (PET) imaging in a cohort of 24 participants at time points ranging from 27 to 910 days following acute SARS-CoV-2 infection using a novel radiopharmaceutical agent, [18F]F-AraG, a highly selective tracer that allows for anatomical quantitation of activated T lymphocytes. Tracer uptake in the post-acute COVID group, which included those with and without Long COVID symptoms, was significantly higher compared to pre-pandemic controls in many anatomical regions, including the brain stem, spinal cord, bone marrow, nasopharyngeal and hilar lymphoid tissue, cardiopulmonary tissues, and gut wall. Although T cell activation tended to be higher in participants imaged closer to the time of the acute illness, tracer uptake was increased in participants imaged up to 2.5 years following SARS-CoV-2 infection. We observed that T cell activation in spinal cord and gut wall was associated with the presence of Long COVID symptoms. In addition, tracer uptake in lung tissue was higher in those with persistent pulmonary symptoms. Notably, increased T cell activation in these tissues was also observed in many individuals without Long COVID. Given the high [18F]F-AraG uptake detected in the gut, we obtained colorectal tissue for in situ hybridization SARS-CoV-2 RNA and immunohistochemical studies in a subset of participants with Long COVID symptoms. We identified cellular SARS-CoV-2 RNA in rectosigmoid lamina propria tissue in all these participants, ranging from 158 to 676 days following initial COVID-19 illness, suggesting that tissue viral persistence could be associated with long-term immunological perturbations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.23293924">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.23293924" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.23293924">
        <p class="paperTitle">Published benefits of ivermectin use in Itajai, Brazil for COVID-19 infection, hospitalisation, and mortality are entirely explained by statistical artefacts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.23293924" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.23293924" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mills, R.; Pecanha Antonio, A. C.; Tucker-Kellogg, G.</p>
        <p class="info">Score: 269.8, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.23293924' target='https://doi.org/10.1101/2023.08.10.23293924'> 10.1101/2023.08.10.23293924</a></p>
        <p class="abstract">BackgroundTwo recent publications by Kerr et al. (Cureus 14(1):e21272; Cureus 14(8): e28624) reported dramatic effects of prophylactic ivermectin use for both prevention of COVID-19 and reduction of COVID-19-related hospitalisation and mortality, including a dose-dependent effect of ivermectin prophylaxis. These papers have gained an unusually large public influence: they were incorporated into debates around COVID-19 policies and may have contributed to decreased trust in vaccine efficacy and public health authorities more broadly. Both studies were based on retrospective observational analysis of city-wide registry data from the city of Itajai, Brazil from July-December 2020.

MethodsStarting with initially identified sources of error, we conducted a revised statistical analysis of available data, including data made available with the original papers and public data from the Brazil Ministry of Health. We identified additional uncorrected sources of bias and errors from the original analysis, including incorrect subject exclusion and missing subjects, an enrolment time bias, and multiple sources of immortal time bias. In models assuming no actual effect from ivermectin use, we conducted Monte Carlo simulations to estimate the contribution of these biases to any observed effect.

ResultsUntreated statistical artefacts and methodological errors alone lead to dramatic apparent risk reduction associated with Ivermectin use in both studies. The magnitude of apparent risk reduction from these artefacts is comparable to the results reported by the studies themselves, including apparent protection from infection, hospitalisation, and death, and including the reported apparent dose-response relationship.

ConclusionsThe inference of ivermectin efficacy reported in both papers is unsupported, as the observed effects are entirely explained by untreated statistical artefacts and methodological errors. Our re-analysis calls for caution in interpreting highly publicised observational studies and highlights the importance of common sources of bias in clinical research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.26.23284998">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.26.23284998" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.26.23284998">
        <p class="paperTitle">A molnupiravir-associated mutational signature in global SARS-CoV-2 genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.26.23284998" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.26.23284998" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanderson, T.; Hisner, R.; Donovan-Banfield, I.; Hartman, H.; Lochen, A.; Peacock, T. P.; Ruis, C.</p>
        <p class="info">Score: 2428.8, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.26.23284998' target='https://doi.org/10.1101/2023.01.26.23284998'> 10.1101/2023.01.26.23284998</a></p>
        <p class="abstract">Molnupiravir, an antiviral medication that has been widely used against SARS-CoV-2, acts by inducing mutations in the virus genome during replication. Most random mutations are likely to be deleterious to the virus, and many will be lethal, and so molnupiravir-induced elevated mutation rates reduce viral load2,3. However, if some patients treated with molnupiravir do not fully clear SARS-CoV-2 infections, there could be the potential for onward transmission of molnupiravir-mutated viruses. Here we show that SARS-CoV-2 sequencing databases contain extensive evidence of molnupiravir mutagenesis. Using a systematic approach, we find that a specific class of long phylogenetic branches, distinguished by a high proportion of G-to-A and C-to-T mutations, appear almost exclusively in sequences from 2022, after the introduction of molnupiravir treatment, and in countries and age-groups with widespread usage of the drug. We identify a mutational spectrum, with preferred nucleotide contexts, from viruses in patients known to have been treated with molnupiravir and show that its signature matches that seen in these long branches, in some cases with onwards transmission of molnupiravir-derived lineages. Finally, we analyse treatment records to confirm a direct association between these high G-to-A branches and the use of molnupiravir.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.23293863">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.23293863" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.23293863">
        <p class="paperTitle">Analysis of SARS-CoV-2 Variant-Specific Serum Antibodies Post-Vaccination Utilizing Immortalized Human Hepatocyte-like Cells (HLC) to Assess Development of Protective Immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.23293863" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.23293863" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Collins, D. P.; Steer, C. J.</p>
        <p class="info">Score: 11.3, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.23293863' target='https://doi.org/10.1101/2023.08.08.23293863'> 10.1101/2023.08.08.23293863</a></p>
        <p class="abstract">BackgroundOur previous studies demonstrated that SARS-CoV-2 spike proteins could bind to hepatocytes via the asialoglycoprotein receptor-1 (ASGR-1) facilitating direct infection by the SARS-CoV-2 virus. Immortalized E12-HLC expressed the phenotypic and biological properties of primary human hepatocytes, including their ability to bind spike proteins via ASGR-1 with exception of the spike 1 protein. This binding could be inhibited by spike protein-specific monoclonal antibodies. We used the same spike-blocking analysis to determine if post-vaccination serum was capable of blocking spike protein binding to HLC. Samples collected from subjects prior to, and post-vaccination were quantified for anti-variant-specific antibody (original wild type, alpha (), beta ({beta}), gamma ({gamma}) and delta ({delta}) variants) by a flow cytometry based immunofluorescent assay. Inhibition of variant spike protein binding to HLC and AT-2 (as a known model for spike 1 binding to the ACE-2 receptor) was analyzed by confocal microscopy. This study was designed to investigate the ability of post-vaccination antibodies to mediate immunity to spike S2, and to validate the utility of the E12-HLC in analyzing that immunity.

MethodsSerum was collected from 10 individuals pre- and post-vaccination with the J&amp;J, Moderna or Pfizer vaccines. The serum samples were quantified for variant-specific antibodies in a flow cytometry-based immunofluorescent assay utilizing beads coated with biotinylated variant spike proteins (, {beta}, {gamma}, {delta}). Presence of variant-specific antibodies was visualized by anti-human IgG-Alexa 488. Inhibition of spike protein binding to cells was analyzed by immunofluorescent confocal analysis. Biotinylated variant spike proteins were preincubated with serum samples and then tested for binding to target cells. Binding was visualized by Streptavidin-Alexa 594. Results were compared to binding of unblocked spike variants.

ResultsAll variant spike proteins tested bound to both the HLC and AT-2 cells. Pre-vaccination serum samples had no detectable reactivity to any of the variant spike proteins and were unable to inhibit binding of the variant spike proteins to either target cell. Post-vaccination serum samples demonstrated a progression of SARS-CoV-2 antibody levels from low early post-vaccination levels to higher levels at 2.5 months after vaccination. Concurrently, serum samples taken at those different timeframes demonstrated that serum obtained from shortly after vaccination were not as effective in blocking spike protein as serum obtained after 2.5 months post-vaccination. Antibody concentrations were not necessarily associated with better blocking of spike protein binding as spike variant-specific serum antibody concentrations varied significantly between subjects and within each subject. It was also demonstrated that vaccination with all the various available vaccines stimulated antibodies that inhibited binding of the available variant spike proteins to both HLC and AT-2 cells.

ConclusionHLC, along with AT-2 cells, provides a useful platform to study the development of protective antibodies that prevent the binding SARS-CoV-2 spike proteins to target cells. It was shown that vaccination with the three available vaccines all elicited serum antibodies that were protective against binding of each of the variant spike proteins to both AT-2 and HLC cells. This study suggests that analysis of immune serum to block spike binding to target cells may be a more useful technique to assess protective immunity than quantitation of gross antibody alone.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.23293864">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.23293864" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.23293864">
        <p class="paperTitle">Waning of post-vaccination neutralizing antibody responses against SARS-CoV-2, a systematic literature review and meta-analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.23293864" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.23293864" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jacobsen, H.; Sitaras, I.; Katzmarzyk, M.; Cobos Jimenez, V.; Naughton, R.; Higdon, M. M.; Deloria Knoll, M.</p>
        <p class="info">Score: 71.9, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.23293864' target='https://doi.org/10.1101/2023.08.08.23293864'> 10.1101/2023.08.08.23293864</a></p>
        <p class="abstract">BackgroundMass COVID-19 vaccination and the continuous introduction of new viral variants of SARS-CoV-2, especially of Omicron subvariants, has resulted in an increase in the proportion of the population with hybrid immunity at various stages of waning protection. We systematically reviewed waning of post-vaccination neutralizing antibody titers in different immunological settings to investigate potential differences.

MethodsWe searched for studies providing data for post-vaccination neutralizing antibody responses against SARS-CoV-2 in PubMed, bioRxiv, and medRxiv from Dec 15, 2021, to Jan 31, 2023, using keywords related to COVID-19, vaccination, and antibody neutralization. We used random effects meta-regression to estimate the average fold-reduction in post-vaccination neutralizing antibody titers against the Index strain or Omicron BA.1. from month 1 to month 6 post last dose, stratified by vaccination regimen (primary or booster) and infection-naive vs hybrid-immune status.

FindingsIn total, 26 studies reporting longitudinal post-vaccination neutralizing antibody titers were included. Neutralization titers against the Index variant were available from all studies for infection-naive participants, and from nine for hybrid-immune participants. Against Omicron BA.1, nine and eight studies were available for infection-naive and hybrid-immune cohorts, respectively. In infection-naive cohorts, post-vaccination neutralization titers against the Index strain waned 5.1-fold (95% CI 3.4-7.8) from month 1 to month 6 following primary regimen and 3.8-fold (95% CI 2.4-5.9) following the booster. Titers against Omicron BA.1 waned 5.9-fold (95% CI 3.8-9.0) in infection-naive, post-booster cohorts. In hybrid-immune, post-primary vaccination cohorts, titers waned 3.7-fold (95% CI 1.7-7.9) against the Index strain and 5.0-fold (95% CI 1.1-21.8) against Omicron BA.1.

InterpretationNo obvious differences in waning between post-primary or post-boost vaccination were observed for vaccines used widely to date, nor between infection-naive and hybrid-immune participants. Titers against Omicron BA.1 may wane faster compared to Index titers, which may worsen for more recent Omicron sub-variants and should be monitored. Relatively small datasets limit the precision of our current analysis; further investigation is needed when more data become available. However, based on our current findings, striking differences in waning for the analyzed and future comparisons are unlikely.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.13.23294039">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.13.23294039" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.13.23294039">
        <p class="paperTitle">Temporal trends of dengue seroprevalence among children in coastal Kenya, 1998-2018: a longitudinal cohort study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.13.23294039" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.13.23294039" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Karanja, H. K.; Orindi, B.; Gitonga, J. N.; Mugo, D.; Mwai, K.; Nyamwaya, D. K.; Omuoyo, D. O.; Kitsao, B. S.; Musyoki, J. N.; Wambua, J.; Otieno, E.; Ochola-Oyier, L. I.; Bejon, P.; Warimwe, G. M.</p>
        <p class="info">Score: 8.1, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.13.23294039' target='https://doi.org/10.1101/2023.08.13.23294039'> 10.1101/2023.08.13.23294039</a></p>
        <p class="abstract">Serosurveys suggest widespread dengue virus (DENV) transmission in Africa but there is limited information on the temporal patterns of exposure. Here, we estimated the prevalence and incidence of DENV infections in coastal Kenya over a 20-year period (1998-2018). Sera (n=8038) obtained annually from a longitudinal cohort of 1847 children aged 15 years and below were screened for anti-DENV IgG antibodies. Anti-DENV IgG seroprevalence increased with age and peaked during outbreak years. Among 1354 children who were seronegative at recruitment, we observed an overall incidence (seroconversion) rate of 129.5 (95% CI 118.7-141.4) DENV infections per 1000 person-years. The highest incidence was observed in 2013 at 520 infections per 1000 person-years (95% CI 443.6-610.2) coinciding with a large DENV outbreak in coastal Kenya. Our data suggest long-term DENV exposure among children in coastal Kenya highlighting an urgent need for clinical surveillance to determine the associated health burden in this setting.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.23294172">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.23294172" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.23294172">
        <p class="paperTitle">Cohort Profile: Prospective Cohort to Study the COVID-19 Immune Response in Retail Workers in Quebec, Canada (CISACOV)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.23294172" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.23294172" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Santerre, K.; Theriault, M.; Brousseau, N.; Rochette, S.; Pelletier, J.; Gilbert, C.; Masson, J.-F.; Baz, M.; Boudreau, D.; Trottier, S.</p>
        <p class="info">Score: 4.8, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.23294172' target='https://doi.org/10.1101/2023.08.18.23294172'> 10.1101/2023.08.18.23294172</a></p>
        <p class="abstract">Purpose: Retail workers are an understudied occupational group that may have been at increased risk of contracting SARS-CoV-2 during the COVID-19 pandemic. Therefore, we set up a longitudinal cohort of participants working in this sector to better document the incidence of SARS-CoV-2 infection and the immune response to infection and/or vaccination in this group. Participants: A total of 304 participants were recruited between April 20, 2021 and October 22, 2021. They were invited to attend three visits (each separated by ~12 weeks) during which they provided blood samples and information on participant characteristics, COVID-19 symptoms, and vaccination. An extension phase of two additional visits was carried out between March 15th, 2022 and October 3rd, 2022 to document the impact of the Omicron variant among the 198 participants who were still eligible for recruitment. Participants were aged 18 to 75 and worked in grocery stores, hardware stores, bars or restaurants within the Quebec City metropolitan area (Canada). Findings to date: This article describes participants&#39; demographic, socioeconomic, behavioral, clinical and occupational characteristics, and their COVID-19 symptoms (where applicable). It also describes SARS-CoV-2 vaccination status and any SARS-CoV-2 diagnostic test (i.e., PCR or rapid antigen) performed from the beginning of the pandemic until the last visit. Future plans: The incidence of SARS-CoV-2 infections will be assessed. The immune response (innate and acquired) to SARS-CoV-2 infection or vaccination will be studied using a variety of techniques, including reference and experimental enzyme-linked immunosorbent assays, microneutralization assays with live viruses, experimental pseudoneutralization with an angiotensin-converting enzyme 2-spike assay, peripheral blood mononuclear cells and neutrophil stimulation, and a proliferation assay based on carboxyfluorescein diacetate succinimidyl ester.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.23294156">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.23294156" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.23294156">
        <p class="paperTitle">Genomic and eco-epidemiological investigations in Uruguay reveal local Chikungunya virus transmission dynamics during its expansion across the Americas in 2023.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.23294156" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.23294156" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Burgueno, A.; Giovanetti, M.; Fonseca, V.; Morel, M. N.; Lima, M.; Castro, E.; Guimaraes, N. R.; Iani, F. C. M.; Bormida, V.; Cortinas, M. N.; Ramas, V.; Coppola, L.; Bento, A. I.; Franco, L.; Mendez Rico, J.; Lourenco, J.; Alcantara, L. C. J.; Chiparelli, H.</p>
        <p class="info">Score: 3.9, Published: 2023-08-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.23294156' target='https://doi.org/10.1101/2023.08.17.23294156'> 10.1101/2023.08.17.23294156</a></p>
        <p class="abstract">Uruguay experienced its first Chikungunya virus outbreak in 2023, resulting in a significant burden to its healthcare system. We conducted analysis based on real-time genomic surveillance (30 novel whole genomes) to offer timely insights into recent local transmission dynamics and eco-epidemiological factors behind its emergence and spread in the country.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.23293441">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.23293441" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.23293441">
        <p class="paperTitle">SARS-CoV-2 Viral Clearance and Evolution Varies by Extent of Immunodeficiency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.23293441" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.23293441" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Y.; Choudhary, M. C.; Regan, J.; Boucau, J.; Nathan, A.; Speidel, T.; Liew, M. Y.; Edelstein, G. E.; Kawano, Y.; Uddin, R.; Deo, R.; Marino, C.; Getz, M. A.; Reynold, Z.; Barry, M.; Gilbert, R. F.; Tien, D.; Sagar, S.; Vyas, T. D.; Flynn, J. P.; Hammond, S. P.; Novack, L. A.; Choi, B.; Cernadas, M.; Wallace, Z. S.; Sparks, J. A.; Vyas, J. M.; Seaman, M. S.; Gaiha, G. D.; Siedner, M. J.; Barczak, A. K.; Lemieux, J. E.; Li, J. Z.</p>
        <p class="info">Score: 53.6, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.23293441' target='https://doi.org/10.1101/2023.07.31.23293441'> 10.1101/2023.07.31.23293441</a></p>
        <p class="abstract">Despite vaccination and antiviral therapies, immunocompromised individuals are at risk for prolonged SARS-CoV-2 infection, but the immune defects that predispose to persistent COVID-19 remain incompletely understood. In this study, we performed detailed viro-immunologic analyses of a prospective cohort of participants with COVID-19. The median time to nasal viral RNA and culture clearance in the severe hematologic malignancy/transplant group (S-HT) were 72 and 40 days, respectively, which were significantly longer than clearance rates in the severe autoimmune/B-cell deficient (S-A), non-severe, and non-immunocompromised groups (P&lt;0.001). Participants who were severely immunocompromised had greater SARS-CoV-2 evolution and a higher risk of developing antiviral treatment resistance. Both S-HT and S-A participants had diminished SARS-CoV-2-specific humoral, while only the S-HT group had reduced T cell-mediated responses. This highlights the varied risk of persistent COVID-19 across immunosuppressive conditions and suggests that suppression of both B and T cell responses results in the highest contributing risk of persistent infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.23294078">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.23294078" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.23294078">
        <p class="paperTitle">Coccidioidomycosis Emergence in South America: Exploring Northeastern Brazil&#39;s Epidemiological, Clinical, and Genomic Landscape</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.23294078" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.23294078" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eulalio, K. D.; Kollath, D. R.; Martins, L. M. S.; Filho, A. d. D.; Cavalcanti, M. d. A. S.; Moreira, L. M.; Tenorio, B. G.; Alves, L. G. d. B.; Yamauchi, D.; Benard, G.; Thompson, G. R.; nacher, m.; Stajich, J. S. E.; Bagagli, E.; Felipe, M. S. S.; Barker, B.; Trilles, L.; Teixeira, M. d. M.</p>
        <p class="info">Score: 3.1, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.23294078' target='https://doi.org/10.1101/2023.08.14.23294078'> 10.1101/2023.08.14.23294078</a></p>
        <p class="abstract">Coccidioidomycosis is an invasive mycosis included in WHOs priority list. It is endemic and notifiable in the United States but neglected in Central and South America. We used a multi-institutional approach to assess whether disease characteristics, genetic variation in the pathogen or environmental factors affects the epidemiology of coccidioidomycosis and disease outcomes throughout the American continent. We identified 292 patients with coccidioidomycosis between 1978 and 2021 in the Piaui and Maranhao states of Brazil; the largest cases series reported outside the US/Mexico epidemic range. The male-to-female ratio was 57.4:1 and the main risk factor was armadillo hunting (91.1%) 4 to 30 days before symptom onset. Forty-two outbreaks involving two to six patients were observed. Most patients (92.8%) presented typical acute pulmonary disease, followed by disseminated (3.4%), chronic pulmonary (2.4%) and regressive pulmonary (1.4%). The most frequent clinical symptoms were cough (93%), fever (90%) and chest pain (77%). Mortality was observed in 8% of the patients. In 2004, and between 2015 and 2017, we observed a spike in coccidioidomycosis in Brazil, particularly in the state of Piaui. Unlike other main hotspots, the soil is acidic in this region and precipitation (p=0.015) and precipitation one-year prior (p=0.001) were predictors of higher coccidioidomycosis rates. The Brazilian strains are genotypically divergent from other described C. posadasii within the Texas/Mexico/South America clade. Coccidioidomycosis in Northeastern Brazil has a specific infection profile and armadillo hunters are at higher risk. Low pluviosity and extensive drought appear key to increasing the number of cases in Brazil. A unique C. posadasii lineage exists in Brazil; therefore, environmental, virulence, and/or pathogenesis traits may differ from other Coccidioides genotypes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
