<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Possible roles of phytochemicals with bioactive properties in the prevention of and recovery from COVID-19
Authors: Koyama, S.; Shields, V. D.; Heinbockel, T.; Joseph, P. V.; Adhikari, P.; Kaur, R.; Kumar, R.; Alizadeh, R.; Bhutani, S.; Calcinoni, O.; Mucignat-Caretta, C.; Chen, J.; Cooper, K. W.; Das, S.; Dominguez, P. R.; Guardia, M. D.; Klyuchnikova, M. A.; Laktionova, T. K.; MOri, E.; Namjoo, Z.; Nguyen, H.; Ozdener, M. H.; Parsa, S.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Possible roles of phytochemicals with bioactive properties in the prevention of and recovery from COVID-19
Authors: Koyama, S.; Shields, V. D.; Heinbockel, T.; Joseph, P. V.; Adhikari, P.; Kaur, R.; Kumar, R.; Alizadeh, R.; Bhutani, S.; Calcinoni, O.; Mucignat-Caretta, C.; Chen, J.; Cooper, K. W.; Das, S.; Dominguez, P. R.; Guardia, M. D.; Klyuchnikova, M. A.; Laktionova, T. K.; MOri, E.; Namjoo, Z.; Nguyen, H.; Ozdener, M. H.; Parsa, S." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-04T10:38:39+00:00" />
<meta property="article:modified_time" content="2024-02-04T10:38:39+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Possible roles of phytochemicals with bioactive properties in the prevention of and recovery from COVID-19
Authors: Koyama, S.; Shields, V. D.; Heinbockel, T.; Joseph, P. V.; Adhikari, P.; Kaur, R.; Kumar, R.; Alizadeh, R.; Bhutani, S.; Calcinoni, O.; Mucignat-Caretta, C.; Chen, J.; Cooper, K. W.; Das, S.; Dominguez, P. R.; Guardia, M. D.; Klyuchnikova, M. A.; Laktionova, T. K.; MOri, E.; Namjoo, Z.; Nguyen, H.; Ozdener, M. H.; Parsa, S."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Possible roles of phytochemicals with bioactive properties in the prevention of and recovery from COVID-19\nAuthors: Koyama, S.; Shields, V. D.; Heinbockel, T.; Joseph, P. V.; Adhikari, P.; Kaur, R.; Kumar, R.; Alizadeh, R.; Bhutani, S.; Calcinoni, O.; Mucignat-Caretta, C.; Chen, J.; Cooper, K. W.; Das, S.; Dominguez, P. R.; Guardia, M. D.; Klyuchnikova, M. A.; Laktionova, T. K.; MOri, E.; Namjoo, Z.; Nguyen, H.; Ozdener, M. H.; Parsa, S.",
  "keywords": [
    
  ],
  "articleBody": " Possible roles of phytochemicals with bioactive properties in the prevention of and recovery from COVID-19\nAuthors: Koyama, S.; Shields, V. D.; Heinbockel, T.; Joseph, P. V.; Adhikari, P.; Kaur, R.; Kumar, R.; Alizadeh, R.; Bhutani, S.; Calcinoni, O.; Mucignat-Caretta, C.; Chen, J.; Cooper, K. W.; Das, S.; Dominguez, P. R.; Guardia, M. D.; Klyuchnikova, M. A.; Laktionova, T. K.; MOri, E.; Namjoo, Z.; Nguyen, H.; Ozdener, M. H.; Parsa, S.; Poyraz, E. O.; Strub, D. J.; Taghizadeh-Hesary, F.; Ueha, R.; Voznessenskaya, V. V.\nScore: 66.6, Published: 2024-01-30 DOI: 10.1101/2024.01.29.24301882\nPurposeThere have been large geographical differences in the infection and death rates of COVID-19. Foods and beverages containing high amounts of phytochemicals with bioactive properties were suggested to prevent contracting, to limit the severity of, and to facilitate recovery from COVID-19. The goal of our study was to determine the correlation of the type of foods/beverages people consumed and the risk reduction of contracting COVID-19 and the recovery from COVID-19. MethodsWe developed an online survey that asked the participants whether they contracted COVID-19, their symptoms, time to recover, and their frequency of eating various types of foods/beverages. The survey was first developed in English and then translated into 10 different languages. ResultsThe participants who did not contract COVID-19 consumed vegetables, herbs/spices, and fermented foods/beverages significantly more than the participants who contracted COVID-19 and those who were not tested but became sick most likely from COVID-19. The geographic location of participants corresponded with the language of the survey, except for the English version, thus, nine out of the 10 language versions represented a country. Among the six countries (India, Iran, Italy, Japan, Russia, Spain) with over one hundred participants, we found that in India and Japan the people who contracted COVID-19 showed significantly shorter recovery time, and greater daily intake of vegetables, herbs/spices, and fermented foods/beverages was associated with faster recovery. ConclusionOur results suggest that phytochemical compounds included in the vegetables may have contributed in not only preventing contraction of COVID-19, but also accelerating their recovery. (249 words; EJN limit is 250 words)\nAltered IgG4 Antibody Response to Repeated mRNA versus Protein COVID Vaccines\nAuthors: Kalkeri, R.; Zhu, M.; Cloney-Clark, S.; Plested, J. S.; Parekh, A.; Gorinson, D.; Cai, R.; Mahato, S.; Ramanathan, P.; Aurelia, L. C.; John Selva, K.; Marchese, A. M.; Fries, L.; Chung, A. W.; Dunkle, L. M.\nScore: 329.5, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301374\nRepeated mRNA SARS-CoV-2 vaccination has been associated with increases in the proportion of IgG4 in spike-specific antibody responses and concurrent reductions in Fc{gamma}-mediated effector functions that may limit control of viral infection. Here, we assessed anti-Spike total IgG, IgG1, IgG2, IgG3 and IgG4, and surrogate markers for antibody-dependent cellular phagocytosis (ADCP, Fc{gamma}RIIa binding), antibody-dependent cellular cytotoxicity (ADCC, Fc{gamma}RIIIa binding), and antibody-dependent complement deposition (ADCD, C1q binding) associated with repeated SARS-CoV-2 vaccination with NVX-CoV2373 (Novavax Inc., Gaithersburg, MD). The NVX-CoV2373 protein vaccine did not induce notable increases in spike-specific IgG4 or negatively impact surrogates for Fc{gamma} effector responses. Conversely, repeated NVX-CoV2373 vaccination uniquely enhanced IgG3 responses which are known to exhibit strong affinity for Fc{gamma}RIIIa and have previously been linked to potent neutralization of SARS-CoV-2. Subsequent investigations will help to understand the immunological diversity generated by different SARS-CoV-2 vaccine types and have the potential to reshape public health strategies.\nPredicting COVID-19 cases across a large university campus using complementary built environment and wastewater surveillance approaches\nAuthors: Hinz, A.; Moggridge, J. A.; Ke, H.; Hicks, A. M. A.; Doukhanine, E.; Fralick, M.; Hug, L.; MacFadden, D.; Mejbel, H.; Nott, C.; Raudanskis, A.; Thampi, N.; Wong, A.; Kassen, R.\nScore: 34.7, Published: 2024-01-26 DOI: 10.1101/2024.01.24.23300025\nBackgroundEnvironmental surveillance of SARS-CoV-2 via wastewater has become an invaluable tool for population-level surveillance. Built environment sampling may provide complementary spatially-refined detection for viral surveillance in congregate settings such as universities. MethodsWe conducted a prospective environmental surveillance study at the University of Ottawa between September 2021 and April 2022. Floor surface samples were collected twice weekly from six university buildings. Samples were analyzed for the presence of SARS-CoV-2 using RT-qPCR. A Poisson regression was used to model the campus-wide COVID-19 cases predicted from the fraction of floor swabs positive for SARS-CoV-2 RNA, building CO2 levels, Wi-Fi usage, and SARS-CoV-2 RNA levels in regional wastewater. We used a mixed-effects Poisson regression analysis to model building-level cases using viral copies detected in floor samples as a predictor. A random intercepts logistic regression model tested whether floor samples collected in high-traffic areas were more likely to have SARS-CoV-2 present than low-traffic areas. ResultsOver the 32-week study period, we collected 554 floor swabs at six university buildings. Overall, 13% of swabs were PCR-positive for SARS-CoV-2, with positivity ranging between 4.8% and 32.7% among university buildings. Both floor swab positivity (Spearman r = 0.74, 95% CI: 0.53-0.87) and regional wastewater signal (Spearman r = 0.50, 95% CI: 0.18-0.73) were positively correlated with on-campus COVID-19 cases. In addition, built environment detection was a predictor of cases linked to individual university buildings (IRlog10(copies) + 1 = 17, 95% CI: 7-44). There was no significant difference in detection between floors sampled in high-traffic versus low-traffic areas (OR = 1.3, 95% CI: 0.8-2.1). ConclusionsDetection of SARS-CoV-2 RNA on floors and viral RNA levels found in wastewater were strongly associated with the incidence of COVID-19 cases on a university campus. These data suggest a potential role for institutional built environment sampling, used together with wastewater surveillance, for predicting COVID-19 cases at both campus-wide and building level scales.\nBlood transcriptomics reveal persistent SARS-CoV-2 RNA and candidate biomarkers in Long COVID patients\nAuthors: MENEZES, S. M.; JAMOULLE, M.; Carletto, M. P.; Van Holm, B.; Moens, L.; Meyts, I.; Maes, P.; Van Weyenbergh, J.\nScore: 336.9, Published: 2024-01-17 DOI: 10.1101/2024.01.14.24301293\nWith an estimated 65 million individuals suffering from Long COVID, validated therapeutic strategies as well as non-invasive biomarkers are direly needed to guide clinical management. We used blood digital transcriptomics in search of viral persistence and Long COVID diagnostic biomarkers in a real-world, general practice-based setting with a long clinical follow-up. We demonstrate systemic SARS-CoV-2 persistence for more than 2 years after acute COVID-19 infection. A 2-gene biomarker, including FYN and SARS-CoV-2 antisense RNA, correctly classifies Long COVID with 93.8% sensitivity and 91.7% specificity. Specific immune transcripts and immunometabolism score correlate to systemic viral load and patient-reported anxiety/depression, providing mechanistic links as well as therapeutic targets to tackle Long COVID.\nEvidence of artemisinin partial resistance in North-western Tanzania: clinical and drug resistance markers study\nAuthors: Ishengoma, D. S.; Mandara, C. I.; Bakari, C.; Fola, A. A.; Madebe, R. A.; Seth, M.; Francis, F.; Buguzi, C.; Moshi, R.; Garimo, I.; Lazaro, S.; Aaron, S.; Lusasi, A.; Chacky, F.; Mohamed, A.; Njau, R.; Kitau, J.; Rasmussen, C.; Bailey, J. A.; Juliano, J. J.; Warsame, M.\nScore: 15.8, Published: 2024-02-01 DOI: 10.1101/2024.01.31.24301954\nBackgroundArtemisinin-based combination therapies (ACTs) are the recommended antimalarial drugs for the treatment of uncomplicated malaria. The recent emergence of artemisinin partial resistance (ART-R) in Rwanda, Uganda and Eritrea is of great concern. In Tanzania, a nationwide molecular malaria surveillance in 2021 showed a high prevalence of the Kelch13 (K13) 561H mutation in Plasmodium falciparum from the north-western region, close to the border with Rwanda and Uganda. This study was conducted in 2022 to evaluate the efficacy of artemether-lumefantrine (AL) and artesunate-amodiaquine (ASAQ) for the treatment of uncomplicated falciparum malaria and to confirm the presence of ART-R in Tanzania. MethodsThis single-arm study evaluated the efficacy of AL and ASAQ in eligible children aged six months to 10 years at Bukangara Dispensary in Karagwe District, Kagera Region. Clinical and parasitological responses were monitored for 28 days according to standard WHO protocol. Mutations in K13 gene and extended haplotypes with these mutations were analysed using Sanger and whole genome sequencing data, respectively. Findings176 children (88 in each AL and ASAQ group) were enrolled and all achieved the defined outcomes. PCR-corrected adequate clinical and parasitological response (ACPR) was 98.3% (95% CI: 90.8-100) and 100.0% (95% CI: 95.8-100) for AL and ASAQ, respectively. Parasitaemia on day 3 was observed in 11/88 (12.5%) and 17/88 (19.3%) in the AL and ASAQ groups, respectively. The half-life of parasitaemia was significantly higher (\u003e6.5 hrs) in patients with parasitaemia on day 3 and/or mutations in K13 gene at enrolment. Most patients with parasitaemia on day 3 (8/11 = 72.7% in the AL group and 10/17 = 58.8% in the ASAQ group) had 561H mutation at enrolment. The parasites with K13 mutations were not similar to those from south-east Asia and Rwanda, but had the same core haplotype of a new 561H haplotype reported in Kagera in 2021. InterpretationThese findings confirm the presence of ART-R in Tanzania. A context-specific strategy to respond to artemisinin partial resistance is urgently needed. Although both AL and ASAQ showed high efficacy, increased vigilance for reduced efficacy of these ACTs and detection of ART-R in other parts of the country is critical. FundingBill and Melinda Gates Foundation to the World Health Organization (WHO, OPP 1209843) and the National Institute for Medical Research (NIMR, Inv. No. 002202), and US National Institute for Health (R01AI156267 to JAB, DSI and JJJ, and K24AI134990 to JJJ). Research in contextO_ST_ABSEvidence before this studyC_ST_ABSArtemisinin partial resistance (ART-R) is defined as delayed clearance after treatment with an artemisinin combination therapy (ACT) or artesunate monotherapy of a parasite strain carrying a validated marker of ART-R. At present, 13 different Kelch13 (K13) mutations have been validated as markers of ART-R. ART-R is confirmed in an area if a quality-controlled study using an ACT or artesunate monotherapy, finds more than 5% of patients have parasites with validated K13 mutations and delayed clearance as evidenced by either persistent parasitemia detected by microscopy on day 3 or a parasite clearance half-life of [\u0026ge;]5 hours. ART-R was first reported from Cambodia in 2008 and later from several countries in Southeast Asia. Published articles up to December 2023 were searched in PubMed with the terms; \"artemisini n\", \"artemisinin partial resistance\", \"artemisinin-based combination therapies\", \"Kelch 13\" in combination with \"Africa\" or \"Tanzania\". The publications confirmed the emergence of ART-R associated with mutations in K13: 561H in Rwanda, A675V and C469Y in Uganda and R622I in Eritrea. All these studies showed a high cure rate of the tested ACTs. The R622I mutant was not reported from Southeast Asia but is circulating in the Horn of Africa (Eritrea, Ethiopia, Sudan and Somalia). In Tanzania, a nationwide malaria molecular surveilla nce launched in January 2021 showed a high prevalence of 561H mutation in the north-western region of Kagera, close to the border with Rwanda and Uganda. Added value of this studyThe study documented delayed parasite clearance associated with pre-treatment validated K13 561H mutation. It confirms and provides evidence for the first-time of ART-R in Kagera region, north-western Tanzania, an area close to the border with Rwanda and Uganda. This makes Tanzania the fourth country in Africa with confirmed ART-R. The study documents presence of K13 mutation associated with ART-R suggesting that partial resistance to artemisinins is rapidly evolving and can still be found in more areas of Africa. Parasites with K13 mutations were not similar to those from south-east Asia and Rwanda, but had the same core haplotype of a new 561H haplotype reported in Kagera in 2021.The findings of this study furthermore show that both AL and ASAQ are highly effective. Implications of all the available evidenceThe emergence of confirmed ART-R in Africa, so far in four countries (Rwanda, Uganda, Eritrea and Tanzania), poses a serious threat to malaria control in Africa, which accounts for more than 95% of the global malaria burden. The current evidence of ART-R in Kagera region calls for an urgent response, including the development of a context-specific strategy based on the recently launched WHO strategy to respond to antimalarial drug resistance in Africa. The fact that ART-R has been confirmed in Kagera region, an area bordering Rwanda and Uganda, where resistance also has been reported, also calls for cross-border collaboration to harmonize strategies to combat this threat in the Great Lakes region of Africa. Nationwide studies on molecular markers in Tanzania, which revealed a high prevalence of K13 validated mutatio ns in the Kagera region, guided where to conduct the current study. This suggests that molecular marker surveillance could play an important role in conducting targeted antimalarial drug efficacy studies and confirming ART-R in other parts of Tanzania and beyond.\nImpact of COVID-19 vaccination on symptoms and immune phenotypes in vaccine-naIÌˆve individuals with Long COVID\nAuthors: Grady, C. B.; Bhattacharjee, B.; Silva, J.; Jaycox, J.; Lee, L. W.; Monteiro, V. S.; Sawano, M.; Massey, D.; Caraballo, C.; Gehlhausen, J.; Tabachnikova, A.; Mao, T.; Lucas, C.; Pena-Hernandez, M. A.; Xu, L.; Tzeng, T. J.; Takahashi, T.; Herrin, J.; Guthe, D. B.; Akrami, A.; Assaf, G.; Davis, H.; Harris, K.; McCorkell, L.; Schulz, W.; Griffin, D.; Wei, H.; Ring, A. M.; Guan, L.; Dela Cruz, C.; Iwasaki, A.; Krumholz, H.\nScore: 79.6, Published: 2024-01-12 DOI: 10.1101/2024.01.11.24300929\nBackgroundLong COVID contributes to the global burden of disease. Proposed root cause hypotheses include the persistence of SARS-CoV-2 viral reservoir, autoimmunity, and reactivation of latent herpesviruses. Patients have reported various changes in Long COVID symptoms after COVID-19 vaccinations, leaving uncertainty about whether vaccine-induced immune responses may alleviate or worsen disease pathology. MethodsIn this prospective study, we evaluated changes in symptoms and immune responses after COVID-19 vaccination in 16 vaccine-naive individuals with Long COVID. Surveys were administered before vaccination and then at 2, 6, and 12 weeks after receiving the first vaccine dose of the primary series. Simultaneously, SARS-CoV-2-reactive TCR enrichment, SARS-CoV-2-specific antibody responses, antibody responses to other viral and self-antigens, and circulating cytokines were quantified before vaccination and at 6 and 12 weeks after vaccination. ResultsSelf-report at 12 weeks post-vaccination indicated 10 out of 16 participants had improved health, 3 had no change, 1 had worse health, and 2 reported marginal changes. Significant elevation in SARS-CoV-2-specific TCRs and Spike protein-specific IgG were observed 6 and 12 weeks after vaccination. No changes in reactivities were observed against herpes viruses and self-antigens. Within this dataset, higher baseline sIL-6R was associated with symptom improvement, and the two top features associated with non-improvement were high IFN-{beta} and CNTF, among soluble analytes. ConclusionsOur study showed that in this small sample, vaccination improved the health or resulted in no change to the health of most participants, though few experienced worsening. Vaccination was associated with increased SARS-CoV-2 Spike protein-specific IgG and T cell expansion in most individuals with Long COVID. Symptom improvement was observed in those with baseline elevated sIL-6R, while elevated interferon and neuropeptide levels were associated with a lack of improvement. Plain language summaryThe impact of the COVID-19 vaccine on vaccine-naive individuals suffering from Long COVID is uncertain. This study assessed the experience and immune signatures of 16 unvaccinated participants with Long COVID. A total of 10 participants had improved health status after vaccination, and one person reported only worsening health. As expected, vaccination increased immune cells and antibodies against the viral spike protein. Immune signatures may prove to be predictors of health status after vaccination. However, given the small number of participants, these initial findings need further validation.\nRelative contribution of COVID-19 vaccination and SARS-CoV-2 infection to population-level seroprevalence of SARS-CoV-2 spike antibodies in a large integrated health system\nAuthors: Chervo, T. C.; Elkin, E. P.; Nugent, J. R.; Valice, E.; Amsden, L. B.; Ergas, I. J.; Munneke, J. R.; Flores, M.; Saelee, G. N.; Hsiao, C. A.; Schapiro, J. M.; Quesenberry, C. P.; Corley, D. A.; Habel, L. A.; Kushi, L. H.; Skarbinski, J.\nScore: 5.6, Published: 2024-02-02 DOI: 10.1101/2024.01.31.24301674\nBackground: Understanding the relative contributions of SARS-CoV-2 infection-induced and vaccine-induced seroprevalence is key to measuring overall population-level seroprevalence and help guide policy decisions. Methods: Using a series of six population-based cross-sectional surveys conducted among persons aged [\u0026ge;]7 years in a large health system with over 4.5 million members between May 2021 and April 2022, we combined data from the electronic health record (EHR), an electronic survey and SARS-CoV-2 spike antibody binding assay, to assess the relative contributions of infection and vaccination to population-level SARS-CoV-2 seroprevalence. EHR and survey data were incorporated to determine spike antibody positivity due to SARS-CoV-2 infection and COVID-19 vaccination. We used sampling and non-response weighting to create population-level estimates. Results: We enrolled 4,319 persons over six recruitment waves. SARS-CoV-2 spike antibody seroprevalence increased from 83.3% (CI 77.0-88.9) in May 2021 to 93.5% (CI 89.5-97.5) in April 2022. By April 2022, 68.5% (CI 61.9-74.3) of the population was seropositive from COVID-19 vaccination only, 13.9% (10.7-17.9) from COVID-19 vaccination and prior diagnosed SARS-CoV-2 infection, 8.2% (CI 4.5-14.5) from prior diagnosed SARS-CoV-2 infection only and 2.9% (CI 1.1-7.6) from prior undiagnosed SARS-CoV-2 infection only. We found high agreement ([\u0026ge;]97%) between EHR and survey data for ascertaining COVID-19 vaccination and SARS-CoV-2 infection status. Conclusions: By April 2022, 93.5% of persons had detectable SARS-CoV-2 spike antibody, predominantly from COVID-19 vaccination. In this highly vaccinated population and over 18 months into the pandemic, SARS-CoV-2 infection without COVID-19 vaccination was a small contributor to overall population-level seroprevalence.\nSARS-CoV-2 sequencing with cloud-based analysis illustrates expedient co-ordinated surveillance of viral genomic epidemiology across six continents\nAuthors: Amoako, D.; Anh, N. T.; Bastable, J.; Brouard, M.; Campano Romero, C.; Castillo Ramirez, A.; Constantinides, B.; Crook, D.; Cuong, P. M.; Diagne, M. M.; Diallo, A.; Dung, N. T.; Dunn, L.; Duyet, L. V.; Everatt, J.; Fletcher, K.; Fowler, P.; Gall, M.; Gentry, J.; Gharbia, S.; Hospital for Tropical Diseases SARS-CoV-2 testing team, ; Hong, N. T. T.; Hunt, M.; Iqbal, Z.; Jeffery, K.; Kekana, D.; Kesteman, T.; Knaggs, J.; Lopes Alves, M.; Man, D. N. H.; Mathers, A.; Ngoc, N. M.; Oakley, S.; Parikh, H.; Peto, T.; Rojas Herrera, M.; Sanderson, N.; Sintchenko, V.; Swann, J.; Tam, N. T.; Tan, L. V.; Score: 4.7, Published: 2024-01-31 DOI: 10.1101/2023.11.27.23298986\nViral sequencing has been critical in the COVID-19 pandemic response, but sequencing and bioinformatics capacity remain inconsistent. To examine the utility of a cloud-based sequencing analysis platform for SARS-CoV-2 sequencing, we conducted a cross-sectional study incorporating seven countries in July 2022. Sites submitted sequential SARS-CoV-2 sequences over two weeks to the Global Pathogen Analysis Service (GPAS). The GPAS bioinformatics cloud platform performs sequence assembly plus lineage and related sample identification. Users can share information with collaborators while retaining data ownership. Seven sites contributed sequencing reads from 5,346 clinical samples, of which 4,799/5,346 (89.8%) had a lineage identified. Omicron lineages dominated, with the vast majority being BA.5, BA.4 and BA.2, commensurate with contemporary genomic epidemiological observations. Phylogenetic analysis demonstrated low within-lineage diversity, and highly similar sequences present in globally disparate sites. A cloud-based analysis platform like GPAS addresses bioinformatics bottlenecks and facilitates collaboration in pathogen surveillance, enhancing epidemic and pandemic preparedness.\nPatients with end-stage kidney disease and COVID-19 are commonly hospitalized early during COVID-19 illness: an opportunity for early intervention\nAuthors: Seethapathy, R.; Wang, Q.; Katz-Agranov, N.; Strohbehn, I. A.; Moreno, D.; Harden, D.; Bhattacharyya, R. P.; Sise, M. E.\nScore: 2.1, Published: 2024-01-23 DOI: 10.1101/2024.01.23.24301661\nAntiviral medications such as remdesivir, molnupiravir, and nirmatrelvir/ritonavir are most effective when used early in the course of coronavirus disease 2019 (COVID-19). These medications are mainly authorized for outpatient use in high risk populations. End-stage kidney disease (ESKD) is among the strongest risk factors for mortality from COVID-19, however, therapeutic options have been lacking in this patient population given exclusion of ESKD in the registrational trials of antiviral therapy leading to limited FDA approval. In our retrospective study of patients with ESKD on dialysis admitted for symptomatic COVID-19 from March 2020 to January 2020, we found that majority of patients (\u003e80%) were admitted to the hospital early during their disease course (within 5 days of symptom onset). Despite this pattern of early admission, there was a high risk of respiratory failure within 90 days since admission (30%) among this population. We argue that this unique pattern of early presentation and high risk of progression to respiratory failure of the ESKD patients suggests an opportunity for further research to determine if outpatient antiviral therapies should be expanded to patients with ESKD to address the huge unmet need of therapeutic intervention in this vulnerable population.\nImmunological imprinting shapes the specificity of human antibody responses against SARS-CoV-2 variants\nAuthors: Johnston, T. S.; Li, S. H.; Painter, M. M.; Atkinson, R. K.; Douek, N. R.; Reeg, D. B.; Doueck, D. C.; Wherry, E. J.; Hensley, S. E.\nScore: 85.3, Published: 2024-01-09 DOI: 10.1101/2024.01.08.24301002\nThe spike glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) continues to accumulate substitutions, leading to breakthrough infections of vaccinated individuals and prompting the development of updated booster vaccines. Here, we determined the specificity and functionality of antibody and B cell responses following exposure to BA.5 and XBB variants in individuals who received ancestral SARS-CoV-2 mRNA vaccines. BA.5 exposures elicited antibody responses that primarily targeted epitopes conserved between the BA.5 and ancestral spike, with poor reactivity to the XBB.1.5 variant. XBB exposures also elicited antibody responses that targeted epitopes conserved between the XBB.1.5 and ancestral spike. However, unlike BA.5, a single XBB exposure elicited low levels of XBB.1.5-specific antibodies and B cells in some individuals. Pre-existing cross-reactive B cells and antibodies were correlated with stronger overall responses to XBB but weaker XBB-specific responses, suggesting that baseline immunity influences the activation of variant-specific SARS-CoV-2 responses. HighlightsO_LIVariant breakthrough infections boost ancestral cross-reactive antibodies and B cells C_LIO_LIFirst and second BA.5 exposures fail to elicit variant-specific antibodies and B cells C_LIO_LIXBB infections and monovalent vaccinations elicit XBB.1.5-specific responses in some individuals C_LIO_LIXBB.1.5-specific responses correlate with low levels of pre-existing humoral immunity C_LI\n",
  "wordCount" : "3571",
  "inLanguage": "en",
  "datePublished": "2024-02-04T10:38:39Z",
  "dateModified": "2024-02-04T10:38:39Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 4, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.24301882">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.24301882" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.24301882">
        <p class="paperTitle">Possible roles of phytochemicals with bioactive properties in the prevention of and recovery from COVID-19</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.24301882" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.24301882" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koyama, S.; Shields, V. D.; Heinbockel, T.; Joseph, P. V.; Adhikari, P.; Kaur, R.; Kumar, R.; Alizadeh, R.; Bhutani, S.; Calcinoni, O.; Mucignat-Caretta, C.; Chen, J.; Cooper, K. W.; Das, S.; Dominguez, P. R.; Guardia, M. D.; Klyuchnikova, M. A.; Laktionova, T. K.; MOri, E.; Namjoo, Z.; Nguyen, H.; Ozdener, M. H.; Parsa, S.; Poyraz, E. O.; Strub, D. J.; Taghizadeh-Hesary, F.; Ueha, R.; Voznessenskaya, V. V.</p>
        <p class="info">Score: 66.6, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.24301882' target='https://doi.org/10.1101/2024.01.29.24301882'> 10.1101/2024.01.29.24301882</a></p>
        <p class="abstract">PurposeThere have been large geographical differences in the infection and death rates of COVID-19. Foods and beverages containing high amounts of phytochemicals with bioactive properties were suggested to prevent contracting, to limit the severity of, and to facilitate recovery from COVID-19. The goal of our study was to determine the correlation of the type of foods/beverages people consumed and the risk reduction of contracting COVID-19 and the recovery from COVID-19.

MethodsWe developed an online survey that asked the participants whether they contracted COVID-19, their symptoms, time to recover, and their frequency of eating various types of foods/beverages. The survey was first developed in English and then translated into 10 different languages.

ResultsThe participants who did not contract COVID-19 consumed vegetables, herbs/spices, and fermented foods/beverages significantly more than the participants who contracted COVID-19 and those who were not tested but became sick most likely from COVID-19. The geographic location of participants corresponded with the language of the survey, except for the English version, thus, nine out of the 10 language versions represented a country. Among the six countries (India, Iran, Italy, Japan, Russia, Spain) with over one hundred participants, we found that in India and Japan the people who contracted COVID-19 showed significantly shorter recovery time, and greater daily intake of vegetables, herbs/spices, and fermented foods/beverages was associated with faster recovery.

ConclusionOur results suggest that phytochemical compounds included in the vegetables may have contributed in not only preventing contraction of COVID-19, but also accelerating their recovery. (249 words; EJN limit is 250 words)</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.24301374">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.24301374" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.24301374">
        <p class="paperTitle">Altered IgG4 Antibody Response to Repeated mRNA versus Protein COVID Vaccines</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.24301374" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.24301374" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kalkeri, R.; Zhu, M.; Cloney-Clark, S.; Plested, J. S.; Parekh, A.; Gorinson, D.; Cai, R.; Mahato, S.; Ramanathan, P.; Aurelia, L. C.; John Selva, K.; Marchese, A. M.; Fries, L.; Chung, A. W.; Dunkle, L. M.</p>
        <p class="info">Score: 329.5, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.24301374' target='https://doi.org/10.1101/2024.01.17.24301374'> 10.1101/2024.01.17.24301374</a></p>
        <p class="abstract">Repeated mRNA SARS-CoV-2 vaccination has been associated with increases in the proportion of IgG4 in spike-specific antibody responses and concurrent reductions in Fc{gamma}-mediated effector functions that may limit control of viral infection. Here, we assessed anti-Spike total IgG, IgG1, IgG2, IgG3 and IgG4, and surrogate markers for antibody-dependent cellular phagocytosis (ADCP, Fc{gamma}RIIa binding), antibody-dependent cellular cytotoxicity (ADCC, Fc{gamma}RIIIa binding), and antibody-dependent complement deposition (ADCD, C1q binding) associated with repeated SARS-CoV-2 vaccination with NVX-CoV2373 (Novavax Inc., Gaithersburg, MD). The NVX-CoV2373 protein vaccine did not induce notable increases in spike-specific IgG4 or negatively impact surrogates for Fc{gamma} effector responses. Conversely, repeated NVX-CoV2373 vaccination uniquely enhanced IgG3 responses which are known to exhibit strong affinity for Fc{gamma}RIIIa and have previously been linked to potent neutralization of SARS-CoV-2. Subsequent investigations will help to understand the immunological diversity generated by different SARS-CoV-2 vaccine types and have the potential to reshape public health strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.23300025">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.23300025" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.23300025">
        <p class="paperTitle">Predicting COVID-19 cases across a large university campus using complementary built environment and wastewater surveillance approaches</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.23300025" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.23300025" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hinz, A.; Moggridge, J. A.; Ke, H.; Hicks, A. M. A.; Doukhanine, E.; Fralick, M.; Hug, L.; MacFadden, D.; Mejbel, H.; Nott, C.; Raudanskis, A.; Thampi, N.; Wong, A.; Kassen, R.</p>
        <p class="info">Score: 34.7, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.23300025' target='https://doi.org/10.1101/2024.01.24.23300025'> 10.1101/2024.01.24.23300025</a></p>
        <p class="abstract">BackgroundEnvironmental surveillance of SARS-CoV-2 via wastewater has become an invaluable tool for population-level surveillance. Built environment sampling may provide complementary spatially-refined detection for viral surveillance in congregate settings such as universities.

MethodsWe conducted a prospective environmental surveillance study at the University of Ottawa between September 2021 and April 2022. Floor surface samples were collected twice weekly from six university buildings. Samples were analyzed for the presence of SARS-CoV-2 using RT-qPCR. A Poisson regression was used to model the campus-wide COVID-19 cases predicted from the fraction of floor swabs positive for SARS-CoV-2 RNA, building CO2 levels, Wi-Fi usage, and SARS-CoV-2 RNA levels in regional wastewater. We used a mixed-effects Poisson regression analysis to model building-level cases using viral copies detected in floor samples as a predictor. A random intercepts logistic regression model tested whether floor samples collected in high-traffic areas were more likely to have SARS-CoV-2 present than low-traffic areas.

ResultsOver the 32-week study period, we collected 554 floor swabs at six university buildings. Overall, 13% of swabs were PCR-positive for SARS-CoV-2, with positivity ranging between 4.8% and 32.7% among university buildings. Both floor swab positivity (Spearman r = 0.74, 95% CI: 0.53-0.87) and regional wastewater signal (Spearman r = 0.50, 95% CI: 0.18-0.73) were positively correlated with on-campus COVID-19 cases. In addition, built environment detection was a predictor of cases linked to individual university buildings (IRlog10(copies) &#43; 1 = 17, 95% CI: 7-44). There was no significant difference in detection between floors sampled in high-traffic versus low-traffic areas (OR = 1.3, 95% CI: 0.8-2.1).

ConclusionsDetection of SARS-CoV-2 RNA on floors and viral RNA levels found in wastewater were strongly associated with the incidence of COVID-19 cases on a university campus. These data suggest a potential role for institutional built environment sampling, used together with wastewater surveillance, for predicting COVID-19 cases at both campus-wide and building level scales.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.24301293">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.24301293" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.24301293">
        <p class="paperTitle">Blood transcriptomics reveal persistent SARS-CoV-2 RNA and candidate biomarkers in Long COVID patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.24301293" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.24301293" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: MENEZES, S. M.; JAMOULLE, M.; Carletto, M. P.; Van Holm, B.; Moens, L.; Meyts, I.; Maes, P.; Van Weyenbergh, J.</p>
        <p class="info">Score: 336.9, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.24301293' target='https://doi.org/10.1101/2024.01.14.24301293'> 10.1101/2024.01.14.24301293</a></p>
        <p class="abstract">With an estimated 65 million individuals suffering from Long COVID, validated therapeutic strategies as well as non-invasive biomarkers are direly needed to guide clinical management. We used blood digital transcriptomics in search of viral persistence and Long COVID diagnostic biomarkers in a real-world, general practice-based setting with a long clinical follow-up. We demonstrate systemic SARS-CoV-2 persistence for more than 2 years after acute COVID-19 infection. A 2-gene biomarker, including FYN and SARS-CoV-2 antisense RNA, correctly classifies Long COVID with 93.8% sensitivity and 91.7% specificity. Specific immune transcripts and immunometabolism score correlate to systemic viral load and patient-reported anxiety/depression, providing mechanistic links as well as therapeutic targets to tackle Long COVID.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.24301954">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.24301954" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.24301954">
        <p class="paperTitle">Evidence of artemisinin partial resistance in North-western Tanzania: clinical and drug resistance markers study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.24301954" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.24301954" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ishengoma, D. S.; Mandara, C. I.; Bakari, C.; Fola, A. A.; Madebe, R. A.; Seth, M.; Francis, F.; Buguzi, C.; Moshi, R.; Garimo, I.; Lazaro, S.; Aaron, S.; Lusasi, A.; Chacky, F.; Mohamed, A.; Njau, R.; Kitau, J.; Rasmussen, C.; Bailey, J. A.; Juliano, J. J.; Warsame, M.</p>
        <p class="info">Score: 15.8, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.24301954' target='https://doi.org/10.1101/2024.01.31.24301954'> 10.1101/2024.01.31.24301954</a></p>
        <p class="abstract">BackgroundArtemisinin-based combination therapies (ACTs) are the recommended antimalarial drugs for the treatment of uncomplicated malaria. The recent emergence of artemisinin partial resistance (ART-R) in Rwanda, Uganda and Eritrea is of great concern. In Tanzania, a nationwide molecular malaria surveillance in 2021 showed a high prevalence of the Kelch13 (K13) 561H mutation in Plasmodium falciparum from the north-western region, close to the border with Rwanda and Uganda. This study was conducted in 2022 to evaluate the efficacy of artemether-lumefantrine (AL) and artesunate-amodiaquine (ASAQ) for the treatment of uncomplicated falciparum malaria and to confirm the presence of ART-R in Tanzania.

MethodsThis single-arm study evaluated the efficacy of AL and ASAQ in eligible children aged six months to 10 years at Bukangara Dispensary in Karagwe District, Kagera Region. Clinical and parasitological responses were monitored for 28 days according to standard WHO protocol. Mutations in K13 gene and extended haplotypes with these mutations were analysed using Sanger and whole genome sequencing data, respectively.

Findings176 children (88 in each AL and ASAQ group) were enrolled and all achieved the defined outcomes. PCR-corrected adequate clinical and parasitological response (ACPR) was 98.3% (95% CI: 90.8-100) and 100.0% (95% CI: 95.8-100) for AL and ASAQ, respectively. Parasitaemia on day 3 was observed in 11/88 (12.5%) and 17/88 (19.3%) in the AL and ASAQ groups, respectively. The half-life of parasitaemia was significantly higher (&gt;6.5 hrs) in patients with parasitaemia on day 3 and/or mutations in K13 gene at enrolment. Most patients with parasitaemia on day 3 (8/11 = 72.7% in the AL group and 10/17 = 58.8% in the ASAQ group) had 561H mutation at enrolment. The parasites with K13 mutations were not similar to those from south-east Asia and Rwanda, but had the same core haplotype of a new 561H haplotype reported in Kagera in 2021.

InterpretationThese findings confirm the presence of ART-R in Tanzania. A context-specific strategy to respond to artemisinin partial resistance is urgently needed. Although both AL and ASAQ showed high efficacy, increased vigilance for reduced efficacy of these ACTs and detection of ART-R in other parts of the country is critical.

FundingBill and Melinda Gates Foundation to the World Health Organization (WHO, OPP 1209843) and the National Institute for Medical Research (NIMR, Inv. No. 002202), and US National Institute for Health (R01AI156267 to JAB, DSI and JJJ, and K24AI134990 to JJJ).

Research in contextO_ST_ABSEvidence before this studyC_ST_ABSArtemisinin partial resistance (ART-R) is defined as delayed clearance after treatment with an artemisinin combination therapy (ACT) or artesunate monotherapy of a parasite strain carrying a validated marker of ART-R. At present, 13 different Kelch13 (K13) mutations have been validated as markers of ART-R. ART-R is confirmed in an area if a quality-controlled study using an ACT or artesunate monotherapy, finds more than 5% of patients have parasites with validated K13 mutations and delayed clearance as evidenced by either persistent parasitemia detected by microscopy on day 3 or a parasite clearance half-life of [&amp;ge;]5 hours. ART-R was first reported from Cambodia in 2008 and later from several countries in Southeast Asia. Published articles up to December 2023 were searched in PubMed with the terms; &#34;artemisini n&#34;, &#34;artemisinin partial resistance&#34;, &#34;artemisinin-based combination therapies&#34;, &#34;Kelch 13&#34; in combination with &#34;Africa&#34; or &#34;Tanzania&#34;. The publications confirmed the emergence of ART-R associated with mutations in K13: 561H in Rwanda, A675V and C469Y in Uganda and R622I in Eritrea. All these studies showed a high cure rate of the tested ACTs. The R622I mutant was not reported from Southeast Asia but is circulating in the Horn of Africa (Eritrea, Ethiopia, Sudan and Somalia). In Tanzania, a nationwide malaria molecular surveilla nce launched in January 2021 showed a high prevalence of 561H mutation in the north-western region of Kagera, close to the border with Rwanda and Uganda.

Added value of this studyThe study documented delayed parasite clearance associated with pre-treatment validated K13 561H mutation. It confirms and provides evidence for the first-time of ART-R in Kagera region, north-western Tanzania, an area close to the border with Rwanda and Uganda. This makes Tanzania the fourth country in Africa with confirmed ART-R. The study documents presence of K13 mutation associated with ART-R suggesting that partial resistance to artemisinins is rapidly evolving and can still be found in more areas of Africa. Parasites with K13 mutations were not similar to those from south-east Asia and Rwanda, but had the same core haplotype of a new 561H haplotype reported in Kagera in 2021.The findings of this study furthermore show that both AL and ASAQ are highly effective.

Implications of all the available evidenceThe emergence of confirmed ART-R in Africa, so far in four countries (Rwanda, Uganda, Eritrea and Tanzania), poses a serious threat to malaria control in Africa, which accounts for more than 95% of the global malaria burden. The current evidence of ART-R in Kagera region calls for an urgent response, including the development of a context-specific strategy based on the recently launched WHO strategy to respond to antimalarial drug resistance in Africa. The fact that ART-R has been confirmed in Kagera region, an area bordering Rwanda and Uganda, where resistance also has been reported, also calls for cross-border collaboration to harmonize strategies to combat this threat in the Great Lakes region of Africa. Nationwide studies on molecular markers in Tanzania, which revealed a high prevalence of K13 validated mutatio ns in the Kagera region, guided where to conduct the current study. This suggests that molecular marker surveillance could play an important role in conducting targeted antimalarial drug efficacy studies and confirming ART-R in other parts of Tanzania and beyond.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.24300929">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.24300929" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.24300929">
        <p class="paperTitle">Impact of COVID-19 vaccination on symptoms and immune phenotypes in vaccine-naIÌˆve individuals with Long COVID</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.24300929" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.24300929" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grady, C. B.; Bhattacharjee, B.; Silva, J.; Jaycox, J.; Lee, L. W.; Monteiro, V. S.; Sawano, M.; Massey, D.; Caraballo, C.; Gehlhausen, J.; Tabachnikova, A.; Mao, T.; Lucas, C.; Pena-Hernandez, M. A.; Xu, L.; Tzeng, T. J.; Takahashi, T.; Herrin, J.; Guthe, D. B.; Akrami, A.; Assaf, G.; Davis, H.; Harris, K.; McCorkell, L.; Schulz, W.; Griffin, D.; Wei, H.; Ring, A. M.; Guan, L.; Dela Cruz, C.; Iwasaki, A.; Krumholz, H.</p>
        <p class="info">Score: 79.6, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.24300929' target='https://doi.org/10.1101/2024.01.11.24300929'> 10.1101/2024.01.11.24300929</a></p>
        <p class="abstract">BackgroundLong COVID contributes to the global burden of disease. Proposed root cause hypotheses include the persistence of SARS-CoV-2 viral reservoir, autoimmunity, and reactivation of latent herpesviruses. Patients have reported various changes in Long COVID symptoms after COVID-19 vaccinations, leaving uncertainty about whether vaccine-induced immune responses may alleviate or worsen disease pathology.

MethodsIn this prospective study, we evaluated changes in symptoms and immune responses after COVID-19 vaccination in 16 vaccine-naive individuals with Long COVID. Surveys were administered before vaccination and then at 2, 6, and 12 weeks after receiving the first vaccine dose of the primary series. Simultaneously, SARS-CoV-2-reactive TCR enrichment, SARS-CoV-2-specific antibody responses, antibody responses to other viral and self-antigens, and circulating cytokines were quantified before vaccination and at 6 and 12 weeks after vaccination.

ResultsSelf-report at 12 weeks post-vaccination indicated 10 out of 16 participants had improved health, 3 had no change, 1 had worse health, and 2 reported marginal changes. Significant elevation in SARS-CoV-2-specific TCRs and Spike protein-specific IgG were observed 6 and 12 weeks after vaccination. No changes in reactivities were observed against herpes viruses and self-antigens. Within this dataset, higher baseline sIL-6R was associated with symptom improvement, and the two top features associated with non-improvement were high IFN-{beta} and CNTF, among soluble analytes.

ConclusionsOur study showed that in this small sample, vaccination improved the health or resulted in no change to the health of most participants, though few experienced worsening. Vaccination was associated with increased SARS-CoV-2 Spike protein-specific IgG and T cell expansion in most individuals with Long COVID. Symptom improvement was observed in those with baseline elevated sIL-6R, while elevated interferon and neuropeptide levels were associated with a lack of improvement.

Plain language summaryThe impact of the COVID-19 vaccine on vaccine-naive individuals suffering from Long COVID is uncertain. This study assessed the experience and immune signatures of 16 unvaccinated participants with Long COVID. A total of 10 participants had improved health status after vaccination, and one person reported only worsening health. As expected, vaccination increased immune cells and antibodies against the viral spike protein. Immune signatures may prove to be predictors of health status after vaccination. However, given the small number of participants, these initial findings need further validation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.24301674">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.24301674" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.24301674">
        <p class="paperTitle">Relative contribution of COVID-19 vaccination and SARS-CoV-2 infection to population-level seroprevalence of SARS-CoV-2 spike antibodies in a large integrated health system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.24301674" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.24301674" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chervo, T. C.; Elkin, E. P.; Nugent, J. R.; Valice, E.; Amsden, L. B.; Ergas, I. J.; Munneke, J. R.; Flores, M.; Saelee, G. N.; Hsiao, C. A.; Schapiro, J. M.; Quesenberry, C. P.; Corley, D. A.; Habel, L. A.; Kushi, L. H.; Skarbinski, J.</p>
        <p class="info">Score: 5.6, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.24301674' target='https://doi.org/10.1101/2024.01.31.24301674'> 10.1101/2024.01.31.24301674</a></p>
        <p class="abstract">Background: Understanding the relative contributions of SARS-CoV-2 infection-induced and vaccine-induced seroprevalence is key to measuring overall population-level seroprevalence and help guide policy decisions. Methods: Using a series of six population-based cross-sectional surveys conducted among persons aged [&amp;ge;]7 years in a large health system with over 4.5 million members between May 2021 and April 2022, we combined data from the electronic health record (EHR), an electronic survey and SARS-CoV-2 spike antibody binding assay, to assess the relative contributions of infection and vaccination to population-level SARS-CoV-2 seroprevalence. EHR and survey data were incorporated to determine spike antibody positivity due to SARS-CoV-2 infection and COVID-19 vaccination. We used sampling and non-response weighting to create population-level estimates. Results: We enrolled 4,319 persons over six recruitment waves. SARS-CoV-2 spike antibody seroprevalence increased from 83.3% (CI 77.0-88.9) in May 2021 to 93.5% (CI 89.5-97.5) in April 2022. By April 2022, 68.5% (CI 61.9-74.3) of the population was seropositive from COVID-19 vaccination only, 13.9% (10.7-17.9) from COVID-19 vaccination and prior diagnosed SARS-CoV-2 infection, 8.2% (CI 4.5-14.5) from prior diagnosed SARS-CoV-2 infection only and 2.9% (CI 1.1-7.6) from prior undiagnosed SARS-CoV-2 infection only. We found high agreement ([&amp;ge;]97%) between EHR and survey data for ascertaining COVID-19 vaccination and SARS-CoV-2 infection status. Conclusions: By April 2022, 93.5% of persons had detectable SARS-CoV-2 spike antibody, predominantly from COVID-19 vaccination. In this highly vaccinated population and over 18 months into the pandemic, SARS-CoV-2 infection without COVID-19 vaccination was a small contributor to overall population-level seroprevalence.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.23298986">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.23298986" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.23298986">
        <p class="paperTitle">SARS-CoV-2 sequencing with cloud-based analysis illustrates expedient co-ordinated surveillance of viral genomic epidemiology across six continents</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.23298986" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.23298986" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Amoako, D.; Anh, N. T.; Bastable, J.; Brouard, M.; Campano Romero, C.; Castillo Ramirez, A.; Constantinides, B.; Crook, D.; Cuong, P. M.; Diagne, M. M.; Diallo, A.; Dung, N. T.; Dunn, L.; Duyet, L. V.; Everatt, J.; Fletcher, K.; Fowler, P.; Gall, M.; Gentry, J.; Gharbia, S.; Hospital for Tropical Diseases SARS-CoV-2 testing team,  ; Hong, N. T. T.; Hunt, M.; Iqbal, Z.; Jeffery, K.; Kekana, D.; Kesteman, T.; Knaggs, J.; Lopes Alves, M.; Man, D. N. H.; Mathers, A.; Ngoc, N. M.; Oakley, S.; Parikh, H.; Peto, T.; Rojas Herrera, M.; Sanderson, N.; Sintchenko, V.; Swann, J.; Tam, N. T.; Tan, L. V.; </p>
        <p class="info">Score: 4.7, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.23298986' target='https://doi.org/10.1101/2023.11.27.23298986'> 10.1101/2023.11.27.23298986</a></p>
        <p class="abstract">Viral sequencing has been critical in the COVID-19 pandemic response, but sequencing and bioinformatics capacity remain inconsistent. To examine the utility of a cloud-based sequencing analysis platform for SARS-CoV-2 sequencing, we conducted a cross-sectional study incorporating seven countries in July 2022. Sites submitted sequential SARS-CoV-2 sequences over two weeks to the Global Pathogen Analysis Service (GPAS). The GPAS bioinformatics cloud platform performs sequence assembly plus lineage and related sample identification. Users can share information with collaborators while retaining data ownership. Seven sites contributed sequencing reads from 5,346 clinical samples, of which 4,799/5,346 (89.8%) had a lineage identified. Omicron lineages dominated, with the vast majority being BA.5, BA.4 and BA.2, commensurate with contemporary genomic epidemiological observations. Phylogenetic analysis demonstrated low within-lineage diversity, and highly similar sequences present in globally disparate sites. A cloud-based analysis platform like GPAS addresses bioinformatics bottlenecks and facilitates collaboration in pathogen surveillance, enhancing epidemic and pandemic preparedness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.24301661">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.24301661" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.24301661">
        <p class="paperTitle">Patients with end-stage kidney disease and COVID-19 are commonly hospitalized early during COVID-19 illness: an opportunity for early intervention</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.24301661" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.24301661" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Seethapathy, R.; Wang, Q.; Katz-Agranov, N.; Strohbehn, I. A.; Moreno, D.; Harden, D.; Bhattacharyya, R. P.; Sise, M. E.</p>
        <p class="info">Score: 2.1, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.24301661' target='https://doi.org/10.1101/2024.01.23.24301661'> 10.1101/2024.01.23.24301661</a></p>
        <p class="abstract">Antiviral medications such as remdesivir, molnupiravir, and nirmatrelvir/ritonavir are most effective when used early in the course of coronavirus disease 2019 (COVID-19). These medications are mainly authorized for outpatient use in high risk populations. End-stage kidney disease (ESKD) is among the strongest risk factors for mortality from COVID-19, however, therapeutic options have been lacking in this patient population given exclusion of ESKD in the registrational trials of antiviral therapy leading to limited FDA approval. In our retrospective study of patients with ESKD on dialysis admitted for symptomatic COVID-19 from March 2020 to January 2020, we found that majority of patients (&gt;80%) were admitted to the hospital early during their disease course (within 5 days of symptom onset). Despite this pattern of early admission, there was a high risk of respiratory failure within 90 days since admission (30%) among this population. We argue that this unique pattern of early presentation and high risk of progression to respiratory failure of the ESKD patients suggests an opportunity for further research to determine if outpatient antiviral therapies should be expanded to patients with ESKD to address the huge unmet need of therapeutic intervention in this vulnerable population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.24301002">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.24301002" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.24301002">
        <p class="paperTitle">Immunological imprinting shapes the specificity of human antibody responses against SARS-CoV-2 variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.24301002" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.24301002" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnston, T. S.; Li, S. H.; Painter, M. M.; Atkinson, R. K.; Douek, N. R.; Reeg, D. B.; Doueck, D. C.; Wherry, E. J.; Hensley, S. E.</p>
        <p class="info">Score: 85.3, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.24301002' target='https://doi.org/10.1101/2024.01.08.24301002'> 10.1101/2024.01.08.24301002</a></p>
        <p class="abstract">The spike glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) continues to accumulate substitutions, leading to breakthrough infections of vaccinated individuals and prompting the development of updated booster vaccines. Here, we determined the specificity and functionality of antibody and B cell responses following exposure to BA.5 and XBB variants in individuals who received ancestral SARS-CoV-2 mRNA vaccines. BA.5 exposures elicited antibody responses that primarily targeted epitopes conserved between the BA.5 and ancestral spike, with poor reactivity to the XBB.1.5 variant. XBB exposures also elicited antibody responses that targeted epitopes conserved between the XBB.1.5 and ancestral spike. However, unlike BA.5, a single XBB exposure elicited low levels of XBB.1.5-specific antibodies and B cells in some individuals. Pre-existing cross-reactive B cells and antibodies were correlated with stronger overall responses to XBB but weaker XBB-specific responses, suggesting that baseline immunity influences the activation of variant-specific SARS-CoV-2 responses.

HighlightsO_LIVariant breakthrough infections boost ancestral cross-reactive antibodies and B cells
C_LIO_LIFirst and second BA.5 exposures fail to elicit variant-specific antibodies and B cells
C_LIO_LIXBB infections and monovalent vaccinations elicit XBB.1.5-specific responses in some individuals
C_LIO_LIXBB.1.5-specific responses correlate with low levels of pre-existing humoral immunity
C_LI</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
