<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19
Authors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19
Authors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-13T10:36:01+00:00" />
<meta property="article:modified_time" content="2023-08-13T10:36:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19
Authors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19\nAuthors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A.",
  "keywords": [
    
  ],
  "articleBody": " Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19\nAuthors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A.; Dalhuisen, T.; Durstenfeld, M. S.; Hsue, P. Y.; Kelly, J. D.; Kumar, N.; Martin, J. N.; Gambhir, A.; Somsouk, M.; Seo, Y.; Deeks, S. G.; Laszik, Z. G.; VanBrocklin, H. F.; Henrich, T. J.\nScore: 1759.6, Published: 2023-07-31 DOI: 10.1101/2023.07.27.23293177\nThe etiologic mechanisms of post-acute medical morbidities and unexplained symptoms (Long COVID) following SARS-CoV-2 infection are incompletely understood. There is growing evidence that viral persistence and immune dysregulation may play a major role. We performed whole-body positron emission tomography (PET) imaging in a cohort of 24 participants at time points ranging from 27 to 910 days following acute SARS-CoV-2 infection using a novel radiopharmaceutical agent, [18F]F-AraG, a highly selective tracer that allows for anatomical quantitation of activated T lymphocytes. Tracer uptake in the post-acute COVID group, which included those with and without Long COVID symptoms, was significantly higher compared to pre-pandemic controls in many anatomical regions, including the brain stem, spinal cord, bone marrow, nasopharyngeal and hilar lymphoid tissue, cardiopulmonary tissues, and gut wall. Although T cell activation tended to be higher in participants imaged closer to the time of the acute illness, tracer uptake was increased in participants imaged up to 2.5 years following SARS-CoV-2 infection. We observed that T cell activation in spinal cord and gut wall was associated with the presence of Long COVID symptoms. In addition, tracer uptake in lung tissue was higher in those with persistent pulmonary symptoms. Notably, increased T cell activation in these tissues was also observed in many individuals without Long COVID. Given the high [18F]F-AraG uptake detected in the gut, we obtained colorectal tissue for in situ hybridization SARS-CoV-2 RNA and immunohistochemical studies in a subset of participants with Long COVID symptoms. We identified cellular SARS-CoV-2 RNA in rectosigmoid lamina propria tissue in all these participants, ranging from 158 to 676 days following initial COVID-19 illness, suggesting that tissue viral persistence could be associated with long-term immunological perturbations.\nThe post-COVID-19 population has a high prevalence of crossreactive antibodies to spikes from all Orthocoronavirinae genera\nAuthors: Singh, G.; Abbad, A.; Kleiner, G.; Srivastava, K.; Gleason, C.; Carreno Quiroz, J. M.; Simon, V.; Krammer, F.\nScore: 47.2, Published: 2023-08-06 DOI: 10.1101/2023.08.01.23293522\nThe Orthocoronaviridae subfamily is large comprising four highly divergent genera. Four seasonal coronaviruses were circulating in humans prior to the coronavirus disease 2019 (COVID-19) pandemic. Infection with these viruses induced antibody responses that are relatively narrow with little cross-reactivity to spike proteins of other coronaviruses. Here, we report that infection with and vaccination against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) induces broadly crossreactive binding antibodies to spikes from a wide range of coronaviruses including members of the sarbecovirus subgenus, betacoronaviruses including Middle Eastern respiratory syndrome coronavirus (MERS CoV), and extending to alpha-, gamma- and delta-coronavirus spikes. These data show that the coronavirus spike antibody landscape in humans has profoundly been changed and broadened as a result of the SARS-CoV-2 pandemic. While we do not understand the functionality of these crossreactive antibodies, they may lead to enhanced resistance of the population to infection with newly emerging coronaviruses with pandemic potential.\nWaning of post-vaccination neutralizing antibody responses against SARS-CoV-2, a systematic literature review and meta-analysis\nAuthors: Jacobsen, H.; Sitaras, I.; Katzmarzyk, M.; Cobos Jimenez, V.; Naughton, R.; Higdon, M. M.; Deloria Knoll, M.\nScore: 26.1, Published: 2023-08-10 DOI: 10.1101/2023.08.08.23293864\nBackgroundMass COVID-19 vaccination and the continuous introduction of new viral variants of SARS-CoV-2, especially of Omicron subvariants, has resulted in an increase in the proportion of the population with hybrid immunity at various stages of waning protection. We systematically reviewed waning of post-vaccination neutralizing antibody titers in different immunological settings to investigate potential differences. MethodsWe searched for studies providing data for post-vaccination neutralizing antibody responses against SARS-CoV-2 in PubMed, bioRxiv, and medRxiv from Dec 15, 2021, to Jan 31, 2023, using keywords related to COVID-19, vaccination, and antibody neutralization. We used random effects meta-regression to estimate the average fold-reduction in post-vaccination neutralizing antibody titers against the Index strain or Omicron BA.1. from month 1 to month 6 post last dose, stratified by vaccination regimen (primary or booster) and infection-naive vs hybrid-immune status. FindingsIn total, 26 studies reporting longitudinal post-vaccination neutralizing antibody titers were included. Neutralization titers against the Index variant were available from all studies for infection-naive participants, and from nine for hybrid-immune participants. Against Omicron BA.1, nine and eight studies were available for infection-naive and hybrid-immune cohorts, respectively. In infection-naive cohorts, post-vaccination neutralization titers against the Index strain waned 5.1-fold (95% CI 3.4-7.8) from month 1 to month 6 following primary regimen and 3.8-fold (95% CI 2.4-5.9) following the booster. Titers against Omicron BA.1 waned 5.9-fold (95% CI 3.8-9.0) in infection-naive, post-booster cohorts. In hybrid-immune, post-primary vaccination cohorts, titers waned 3.7-fold (95% CI 1.7-7.9) against the Index strain and 5.0-fold (95% CI 1.1-21.8) against Omicron BA.1. InterpretationNo obvious differences in waning between post-primary or post-boost vaccination were observed for vaccines used widely to date, nor between infection-naive and hybrid-immune participants. Titers against Omicron BA.1 may wane faster compared to Index titers, which may worsen for more recent Omicron sub-variants and should be monitored. Relatively small datasets limit the precision of our current analysis; further investigation is needed when more data become available. However, based on our current findings, striking differences in waning for the analyzed and future comparisons are unlikely.\nSARS-CoV-2 Viral Clearance and Evolution Varies by Extent of Immunodeficiency\nAuthors: Li, Y.; Choudhary, M. C.; Regan, J.; Boucau, J.; Nathan, A.; Speidel, T.; Liew, M. Y.; Edelstein, G. E.; Kawano, Y.; Uddin, R.; Deo, R.; Marino, C.; Getz, M. A.; Reynold, Z.; Barry, M.; Gilbert, R. F.; Tien, D.; Sagar, S.; Vyas, T. D.; Flynn, J. P.; Hammond, S. P.; Novack, L. A.; Choi, B.; Cernadas, M.; Wallace, Z. S.; Sparks, J. A.; Vyas, J. M.; Seaman, M. S.; Gaiha, G. D.; Siedner, M. J.; Barczak, A. K.; Lemieux, J. E.; Li, J. Z.\nScore: 49.2, Published: 2023-08-07 DOI: 10.1101/2023.07.31.23293441\nDespite vaccination and antiviral therapies, immunocompromised individuals are at risk for prolonged SARS-CoV-2 infection, but the immune defects that predispose to persistent COVID- 19 remain incompletely understood. In this study, we performed detailed viro-immunologic analyses of a prospective cohort of participants with COVID-19. The median time to nasal viral RNA and culture clearance in the severe hematologic malignancy/transplant group (S-HT) were 72 and 21 days, respectively, which were significantly longer than clearance rates in the severe autoimmune/B-cell deficient (S-A), non-severe, and non-immunocompromised groups (P\u003c0.001). Participants who were severely immunocompromised had greater SARS-CoV-2 evolution and higher risk of developing antiviral treatment resistance. Both S-HT and S-A participants had diminished SARS-CoV-2-specific humoral, while only the S-HT group had reduced T cell-mediated responses. This highlights the varied risk of persistent COVID-19 across immunosuppressive conditions and suggests that suppression of both B and T cell responses results in the highest contributing risk of persistent infection.\nDiscovery of the sixth Candida auris clade in Singapore\nAuthors: Suphavilai, C.; Ko, K. K. K.; Lim, K. M.; Tan, M. G.; Boonsimma, P.; Chu, J. J. K.; Goh, S. S.; Rajandran, P.; Lee, L. C.; Tan, K. Y.; Shaik Ismail, B.; Aung, M. K.; Yang, Y.; Sim, J. X. Y.; Venkatachalam, I.; Cherng, B. P. Z.; Spruijtenburg, B.; Chan, K. S.; Oon, L. L. E.; Tan, A. L.; Tan, Y. E.; Wijaya, L.; Tan, B. H.; Ling, M. L.; Koh, T. H.; Meis, J. F.; Tsui, C. K. M.; Nagarajan, N.\nScore: 24.6, Published: 2023-08-06 DOI: 10.1101/2023.08.01.23293435\nBackgroundThe emerging fungal pathogen Candida auris poses a serious threat to global public health due to its worldwide distribution, multidrug-resistance, high transmissibility, propensity to cause outbreaks and high mortality rates. We report three C. auris isolates detected in Singapore, which are genetically distinct from all known clades (I-V) and represent a new clade (Clade VI). MethodsThree epidemiologically unlinked clinical isolates belonging to the potential new C. auris clade were whole-genome sequenced and phenotypically characterized. The complete genomes of these isolates were compared to representative genomes of all known clades. To provide a global context, 3,651 international whole-genome sequences (WGS) from the NCBI database were included in the high-resolution single nucleotide polymorphism (SNP) analysis. Antifungal resistance genes, mating type locus, and chromosomal rearrangements were characterized from the WGS data of the Clade VI isolates. We further implemented Bayesian logistic regression models to simulate the automatic detection of Clade V and VI as their WGS data became available. FindingsThe three Clade VI isolates were separated by \u003e36,000 SNPs from all existing C. auris clades. These isolates had opposite mating type allele and different chromosomal rearrangements when compared to their closest Clade IV relatives. As a proof-of-concept, our classification model was able to flag these outlier genomes as a potential new clade. Furthermore, an independent WGS submission from Bangladesh was found to belong to this new clade. InterpretationThe discovery of a new C. auris clade in Singapore and Bangladesh, showing close relationship to Clade IV members in South America, highlights the unknown genetic diversity and origin of C. auris, particularly in under-resourced regions. Active surveillance in clinical settings, along with effective sequencing strategies and downstream analysis, will be essential in the identification of novel strains, tracking of transmission, and containment of adverse clinical impacts caused by C. auris infections. FundingThis work was supported by the Singapore National Medical Research Council (NMRC) research training fellowship (MOH-FLWSHP19may-0005), the NCRS Duke-NUS Academic Medical Center Academic Clinical Program grant (09/FY2022/P1/17-A32, GRDUKP003401), and the Genedant-GIS Innovation Program grant. Research in contextO_ST_ABSEvidence before this studyC_ST_ABSWe searched PubMed using the search terms \"Candida auris\" AND \"clade\", for papers published between Jan 1, 2009, and July 1, 2023. This search retrieved 115 publications. 60 relevant publications were identified. 28 studies analyzed and discussed the molecular epidemiology of C. auris, including the description of C. auris clades, either in outbreak or surveillance settings. There were 11 case reports of C. auris clinical cases that included clade determination. Two studies focused on the detection and clade determination of C. auris from non-healthcare environments. Clade-specific characteristics were described or analyzed in 14 studies. One study applied machine learning to C. auris drug resistance analysis, but not for clade determination. Four studies focused on the description of potentially new C. auris lineages, subclades, or clades. All publications described isolates that belong to one of the five known C. auris clades (I-V). All publications found that strains from different clades differed by more than 35,000 SNPs, and that there are clade-specific differences in geographical distribution, phenotypic characteristics, antifungal susceptibility profile, outbreak potential, and clinical manifestations. The NCBI Pathogen Detection system contained 4,506 C. auris genomes on July 1, 2023. There were ten (0{middle dot}22%) submissions from Southeast Asian countries and 92 (2{middle dot}04%) submissions from South Asia and the Indian subcontinent, which are parts of the Indomalayan biogeographic realm. Added value of this studyTo the best of our knowledge, we are the first group to perform hybrid assemblies on three representative isolates in a new C. auris clade, which is separated from all other existing clades (I-V) by \u003e36,000 SNPs. Whole-genome SNP analysis and phenotypic characterization of these epidemiologically unlinked isolates detected in Singapore suggest that they represent a previously unreported sixth major clade. High-resolution SNP analysis of 3,651 international whole-genome sequences from the NCBI database, which generated the final dataset consisting of more than 6.6 million genome pairs, revealed six distinct genetic clusters representing the five known clades and the new sixth clade (Indomalayan). In addition, we demonstrate that a machine learning approach can be used to flag these outlier genomes for further investigations as soon as they become available, thus providing the earliest possible alert for potential new public health threats. Implications of all the available evidenceDespite the high antimicrobial resistance burden in Southeast Asia and South Asia, these regions are disproportionately underrepresented in terms of genomic surveillance of C. auris, a multidrug-resistant fungal pathogen. The detection of three epidemiologically unlinked C. auris isolates in Singapore belonging to a new C. auris clade suggests that yet-to-be-reported strains may be circulating in the region. Given the propensity for multidrug resistance, healthcare-associated infection outbreaks, and the associated high mortality, active surveillance and continued vigilance is necessary.\nEarly underdetected dissemination followed by extensive local transmission propelled the 2022 mpox epidemic and limited impact of vaccination\nAuthors: Paredes, M. I.; Ahmed, N.; Figgins, M.; Colizza, V.; Lemey, P.; McCrone, J. T.; Müller, N. F.; Tran-Kiem, C.; Bedford, T.\nScore: 9.0, Published: 2023-08-02 DOI: 10.1101/2023.07.27.23293266\nThe World Health Organization (WHO) declared mpox a public health emergency of international concern in July 2022. It is still unclear to what extent international travel contributed to the explosive spread of mpox and the degree to which national vaccination campaigns were responsible for controlling the epidemic. We built phylogeographic and phylodynamic models to analyze MPXV genomes sampled between March 2022 and January 2023 from five global regions together with air traffic and epidemiological data to analyze the global spread of mpox. Our models reveal community transmission prior to detection by local surveillance, changes in case-reporting throughout the epidemic, and a large degree of transmission heterogeneity. Additionally, we find that viral introductions played a limited role in prolonging spread after initial dissemination, suggesting that travel bans would have had only a minor impact. We find that the time-varying effective reproductive number in North America declines below one before more than 10% of individuals at high risk individuals in the USA had vaccine-induced immunity, suggesting little impact of vaccination in controlling the epidemic. Given that cases quickly declined after detection most likely due to behavioral modifications, our findings highlight the importance of broader routine specimen screening surveillance for emerging infectious diseases.\nEffectiveness of one dose of killed oral cholera vaccine in an endemic community in the Democratic Republic of Congo: A matched case-control study\nAuthors: Malembaka, E. B.; Bugeme, P. M.; Hutchins, C.; Xu, H.; Dent, J. D.; Demby, M. N.; Gallandat, K.; Saidi, J. M.; Rumedeka, B. B.; Itongwa, M.; Tshiwedi-Tsilabia, E.; Kitoga, F.; Bodisa-Matamu, T.; Kavunga-Membo, H.; Kulondwa, J.-C.; Bengehya, J.; Debes, A. K.; Taty, N.; Lee, E. C.; Lessler, J.; Leung, D. T.; Cumming, O.; Okitayemba, P. W.; Mukadi-Bamuleka, D.; Knee, J.; Azman, A. S.\nScore: 6.8, Published: 2023-08-09 DOI: 10.1101/2023.08.07.23293369\nBackgroundA global shortage of cholera vaccines has increased use of single-dose regimens, rather than the standard two-dose regimen. There is limited evidence on single-dose protection, particularly in children. In late 2020 a mass vaccination campaign was conducted in Uvira, a cholera endemic urban setting in eastern Democratic Republic of the Congo. Coverage surveys showed that the majority of vaccinated people received only one dose. We examined the effectiveness of a single-dose of the oral cholera vaccine Euvichol-plus in this high-burden setting. MethodsIn this case-control study, we recruited medically-attended confirmed cholera cases and age-, sex-, and neighborhood-matched community controls during two distinct periods after mass vaccination, October 2021 to March 2022 (12-17 months post-vaccination) and October 2022 to May 2023 (24-31 months post-vaccination). The odds of vaccination in cases and controls were contrasted in conditional logistic regression models to estimate unadjusted and adjusted vaccine effectiveness. FindingsWe enrolled 432 confirmed cases and 1554 matched controls during the two study periods with 21.5% of cases being under five years old. The adjusted single-dose VE was 52% (95% CI: 32{middle dot}3-66) 12-17 months post-vaccination and 40{middle dot}8% (95% CI: 8{middle dot}6-61{middle dot}7) 24-31 months post-vaccination. Combining the two study periods led to an adjusted single-dose VE of 45% (95% CI 28{middle dot}2-57{middle dot}8), with no significant differences among children under five years old and older individuals. InterpretationA single-dose of Euvichol-Plus provided substantial protection against medically-attended cholera for at least 31 months post-vaccination in this cholera endemic setting. FundingThis work was supported by the Wellcome Trust and Gavi (GAVI-RFP-2019-062).\nSARS-CoV-2 Pandemic Non-Pharmacologic Interventions Temporally Associated with Reduced Pediatric Infections Due to Mycoplasma pneumoniae and Co-Infecting Respiratory Viruses in Arkansas\nAuthors: Boyanton, B. L.; Frenner, R. A.; Ingold, A.; Ambroggio, L.; Kennedy, J. L.\nScore: 4.8, Published: 2023-08-09 DOI: 10.1101/2023.08.05.23293566\nIntroductionNon-pharmacologic interventions (NPIs), such as universal masking, implemented during the SARS-CoV-2 pandemic have reduced respiratory infections among children. This study focuses on evaluating the impact of NPIs on Mycoplasma pneumoniae infections in children, analyzing data from two hospitals in Arkansas, and examining age-related differences and coinfections with other viruses. MethodsThe study was approved by the Institutional Review Board and included patients aged [\u0026le;]18 years with upper respiratory tract symptoms. Data from the FilmArray(R) Respiratory Panel (FARP) were collected and divided into pre-NPI and NPI periods for analysis. Total test positivity rate and interval change in the positivity rate were evaluated. Statistical differences were determined by Chi-square ({chi}2-independence) analysis. ResultsA total of 68,949 tests were performed with a statistical increase in testing during the NPI period. The overall test positivity rate for M. pneumoniae decreased by 74% (0.86% to 0.03%) during the NPI period, and the preschool age group had the highest number of positive tests in the pre- and NPI periods (Pre-NPI: n=40, NPI: n=12 positive tests, p=\u003c0.001). The reduction in M. pneumoniae infections was consistent across age groups. Coinfections with other respiratory viruses, particularly human rhinovirus/enterovirus, were observed at much lower levels. ConclusionsNPIs effectively reduced M. pneumoniae in pediatric patients in Arkansas, and coinfections with specific viruses still occurred, albeit at lower levels during the SARS-CoV-2 pandemic. As NPIs are relaxed and the pandemic ends, we expect M. pneumoniae infections to return to pre-pandemic levels within the next 1-2 years.\nEarly Biological Markers of Post-Acute Sequelae of SARS-CoV-2 Infection\nAuthors: Lu, S.; Peluso, M. J.; Glidden, D. V.; Davidson, M.; Lugtu, K.; Pineda-Ramirez, J.; Tassetto, M.; Garcia-Knight, M.; Zhang, A.; Goldberg, S. A.; Chen, J.; Forbes, M.; Park, S.; Martinez, A.; So, M.; Donovan, A.; Viswanathan, B.; Hoh, R.; Donohue, K.; McIlwain, D.; Gaudilliere, B.; Anglin, K.; Yee, B.; Chenna, A. A.; Winslow, J.; Petropoulos, C.; Briggs-Hagen, M.; Andino, R.; Midgley, C.; Martin, J. N.; Saydeh, S.; Kelly, J. D.\nScore: 306.3, Published: 2023-07-16 DOI: 10.1101/2023.07.14.23292649\nTo understand the roles of acute phase viral dynamics and host immune responses in PASC, we enrolled 136 participants within 5 days of their first positive SARS-CoV-2 real-time PCR. Participants self-collected nasal specimens up to 21 times within the first 28 days after symptom onset; Interviewer-administered clinical questionnaires and blood samples were collected at enrollment and days 9, 14, 21, 28, and month 4 and 8 post-symptom. Defining PASC as the presence of any symptom new or worse since infection reported at their 4-month visit, we compared viral markers (quantity and duration of viral RNA load, infectious viral load, and plasma N-antigen level) and host immune markers (IL-6, IL-10, TNF-, IFN-, IFN-{gamma}, MCP, IP-10, and Spike IgG) over the acute period. In comparison to those who fully recovered, those who developed PASC demonstrated significantly higher maximum levels of SARS-CoV-2 RNA, infectious virus, and N-antigen, longer duration of viral shedding, and lower Spike-specific IgG levels within the first 10 days of the acute phase of illness. No significant differences were identified among a panel of host immune markers, though there was a trend toward higher initial levels of certain markers (e.g., MCP-1, IFN-, and IFN-{gamma}) in those who went on to develop PASC. Early viral dynamics and the associated host immune responses play a role in the pathogenesis of PASC. These findings highlight the importance of understanding the early biological markers from acute SARS-CoV-2 infection in the natural history of PASC. Onset Sentence SummaryEarly viral dynamics and the associated host immune responses play a role in the pathogenesis of PASC.\nChronic Shedding of a SARS-CoV-2 Alpha Variant Lineage Q.3/Q.4 in Wastewater\nAuthors: Conway, M. J.; Yang, H.; Revord, L. A.; Ward, A. S.; Abel, J. D.; Williams, M. R.; Uzarski, R. L.; Alm, E. W.\nScore: 4.2, Published: 2023-07-27 DOI: 10.1101/2023.07.26.23293191\nCentral Michigan University (CMU) participated in a state-wide SARS-CoV-2 wastewater monitoring program since 2021. Wastewater samples were collected from on-campus sites and nine off-campus wastewater treatment plants servicing small metropolitan and rural communities. SARS-CoV-2 genome copies were quantified using droplet digital PCR and results were reported to the health department. One rural, off-campus site consistently produced higher concentrations of SARS-CoV-2 genome copies. Samples from this site were sequenced and initially contained predominately Alpha variant lineage Q.3, which transitioned to lineage Q.4. Alpha variant lineage Q.3/Q.4 was detected at this site beginning in fall 2021 and continued until summer 2023. Mutational analysis of reconstructed genes revealed divergence from the Alpha variant lineage Q.3 clinical sequence over time, including numerous mutations in the surface glycoprotein RBD and NTD. We discuss the possibility that a chronic SARS-CoV-2 infection accumulated adaptive mutations that promoted long-term infection. This study reveals that small wastewater treatment plants can enhance resolution of rare events and facilitate reconstruction of viral genomes due to the relative lack of contaminating sequences.\n",
  "wordCount" : "3507",
  "inLanguage": "en",
  "datePublished": "2023-08-13T10:36:01Z",
  "dateModified": "2023-08-13T10:36:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta"><span>updated on August 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.23293177">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.23293177" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.23293177">
        <p class="paperTitle">Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.23293177" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.23293177" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A.; Dalhuisen, T.; Durstenfeld, M. S.; Hsue, P. Y.; Kelly, J. D.; Kumar, N.; Martin, J. N.; Gambhir, A.; Somsouk, M.; Seo, Y.; Deeks, S. G.; Laszik, Z. G.; VanBrocklin, H. F.; Henrich, T. J.</p>
        <p class="info">Score: 1759.6, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.23293177' target='https://doi.org/10.1101/2023.07.27.23293177'> 10.1101/2023.07.27.23293177</a></p>
        <p class="abstract">The etiologic mechanisms of post-acute medical morbidities and unexplained symptoms (Long COVID) following SARS-CoV-2 infection are incompletely understood. There is growing evidence that viral persistence and immune dysregulation may play a major role. We performed whole-body positron emission tomography (PET) imaging in a cohort of 24 participants at time points ranging from 27 to 910 days following acute SARS-CoV-2 infection using a novel radiopharmaceutical agent, [18F]F-AraG, a highly selective tracer that allows for anatomical quantitation of activated T lymphocytes. Tracer uptake in the post-acute COVID group, which included those with and without Long COVID symptoms, was significantly higher compared to pre-pandemic controls in many anatomical regions, including the brain stem, spinal cord, bone marrow, nasopharyngeal and hilar lymphoid tissue, cardiopulmonary tissues, and gut wall. Although T cell activation tended to be higher in participants imaged closer to the time of the acute illness, tracer uptake was increased in participants imaged up to 2.5 years following SARS-CoV-2 infection. We observed that T cell activation in spinal cord and gut wall was associated with the presence of Long COVID symptoms. In addition, tracer uptake in lung tissue was higher in those with persistent pulmonary symptoms. Notably, increased T cell activation in these tissues was also observed in many individuals without Long COVID. Given the high [18F]F-AraG uptake detected in the gut, we obtained colorectal tissue for in situ hybridization SARS-CoV-2 RNA and immunohistochemical studies in a subset of participants with Long COVID symptoms. We identified cellular SARS-CoV-2 RNA in rectosigmoid lamina propria tissue in all these participants, ranging from 158 to 676 days following initial COVID-19 illness, suggesting that tissue viral persistence could be associated with long-term immunological perturbations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.23293522">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.23293522" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.23293522">
        <p class="paperTitle">The post-COVID-19 population has a high prevalence of crossreactive antibodies to spikes from all Orthocoronavirinae genera</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.23293522" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.23293522" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Singh, G.; Abbad, A.; Kleiner, G.; Srivastava, K.; Gleason, C.; Carreno Quiroz, J. M.; Simon, V.; Krammer, F.</p>
        <p class="info">Score: 47.2, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.23293522' target='https://doi.org/10.1101/2023.08.01.23293522'> 10.1101/2023.08.01.23293522</a></p>
        <p class="abstract">The Orthocoronaviridae subfamily is large comprising four highly divergent genera. Four seasonal coronaviruses were circulating in humans prior to the coronavirus disease 2019 (COVID-19) pandemic. Infection with these viruses induced antibody responses that are relatively narrow with little cross-reactivity to spike proteins of other coronaviruses. Here, we report that infection with and vaccination against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) induces broadly crossreactive binding antibodies to spikes from a wide range of coronaviruses including members of the sarbecovirus subgenus, betacoronaviruses including Middle Eastern respiratory syndrome coronavirus (MERS CoV), and extending to alpha-, gamma- and delta-coronavirus spikes. These data show that the coronavirus spike antibody landscape in humans has profoundly been changed and broadened as a result of the SARS-CoV-2 pandemic. While we do not understand the functionality of these crossreactive antibodies, they may lead to enhanced resistance of the population to infection with newly emerging coronaviruses with pandemic potential.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.23293864">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.23293864" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.23293864">
        <p class="paperTitle">Waning of post-vaccination neutralizing antibody responses against SARS-CoV-2, a systematic literature review and meta-analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.23293864" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.23293864" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jacobsen, H.; Sitaras, I.; Katzmarzyk, M.; Cobos Jimenez, V.; Naughton, R.; Higdon, M. M.; Deloria Knoll, M.</p>
        <p class="info">Score: 26.1, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.23293864' target='https://doi.org/10.1101/2023.08.08.23293864'> 10.1101/2023.08.08.23293864</a></p>
        <p class="abstract">BackgroundMass COVID-19 vaccination and the continuous introduction of new viral variants of SARS-CoV-2, especially of Omicron subvariants, has resulted in an increase in the proportion of the population with hybrid immunity at various stages of waning protection. We systematically reviewed waning of post-vaccination neutralizing antibody titers in different immunological settings to investigate potential differences.

MethodsWe searched for studies providing data for post-vaccination neutralizing antibody responses against SARS-CoV-2 in PubMed, bioRxiv, and medRxiv from Dec 15, 2021, to Jan 31, 2023, using keywords related to COVID-19, vaccination, and antibody neutralization. We used random effects meta-regression to estimate the average fold-reduction in post-vaccination neutralizing antibody titers against the Index strain or Omicron BA.1. from month 1 to month 6 post last dose, stratified by vaccination regimen (primary or booster) and infection-naive vs hybrid-immune status.

FindingsIn total, 26 studies reporting longitudinal post-vaccination neutralizing antibody titers were included. Neutralization titers against the Index variant were available from all studies for infection-naive participants, and from nine for hybrid-immune participants. Against Omicron BA.1, nine and eight studies were available for infection-naive and hybrid-immune cohorts, respectively. In infection-naive cohorts, post-vaccination neutralization titers against the Index strain waned 5.1-fold (95% CI 3.4-7.8) from month 1 to month 6 following primary regimen and 3.8-fold (95% CI 2.4-5.9) following the booster. Titers against Omicron BA.1 waned 5.9-fold (95% CI 3.8-9.0) in infection-naive, post-booster cohorts. In hybrid-immune, post-primary vaccination cohorts, titers waned 3.7-fold (95% CI 1.7-7.9) against the Index strain and 5.0-fold (95% CI 1.1-21.8) against Omicron BA.1.

InterpretationNo obvious differences in waning between post-primary or post-boost vaccination were observed for vaccines used widely to date, nor between infection-naive and hybrid-immune participants. Titers against Omicron BA.1 may wane faster compared to Index titers, which may worsen for more recent Omicron sub-variants and should be monitored. Relatively small datasets limit the precision of our current analysis; further investigation is needed when more data become available. However, based on our current findings, striking differences in waning for the analyzed and future comparisons are unlikely.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.23293441">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.23293441" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.23293441">
        <p class="paperTitle">SARS-CoV-2 Viral Clearance and Evolution Varies by Extent of Immunodeficiency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.23293441" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.23293441" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Y.; Choudhary, M. C.; Regan, J.; Boucau, J.; Nathan, A.; Speidel, T.; Liew, M. Y.; Edelstein, G. E.; Kawano, Y.; Uddin, R.; Deo, R.; Marino, C.; Getz, M. A.; Reynold, Z.; Barry, M.; Gilbert, R. F.; Tien, D.; Sagar, S.; Vyas, T. D.; Flynn, J. P.; Hammond, S. P.; Novack, L. A.; Choi, B.; Cernadas, M.; Wallace, Z. S.; Sparks, J. A.; Vyas, J. M.; Seaman, M. S.; Gaiha, G. D.; Siedner, M. J.; Barczak, A. K.; Lemieux, J. E.; Li, J. Z.</p>
        <p class="info">Score: 49.2, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.23293441' target='https://doi.org/10.1101/2023.07.31.23293441'> 10.1101/2023.07.31.23293441</a></p>
        <p class="abstract">Despite vaccination and antiviral therapies, immunocompromised individuals are at risk for prolonged SARS-CoV-2 infection, but the immune defects that predispose to persistent COVID- 19 remain incompletely understood. In this study, we performed detailed viro-immunologic analyses of a prospective cohort of participants with COVID-19. The median time to nasal viral RNA and culture clearance in the severe hematologic malignancy/transplant group (S-HT) were 72 and 21 days, respectively, which were significantly longer than clearance rates in the severe autoimmune/B-cell deficient (S-A), non-severe, and non-immunocompromised groups (P&lt;0.001). Participants who were severely immunocompromised had greater SARS-CoV-2 evolution and higher risk of developing antiviral treatment resistance. Both S-HT and S-A participants had diminished SARS-CoV-2-specific humoral, while only the S-HT group had reduced T cell-mediated responses. This highlights the varied risk of persistent COVID-19 across immunosuppressive conditions and suggests that suppression of both B and T cell responses results in the highest contributing risk of persistent infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.23293435">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.23293435" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.23293435">
        <p class="paperTitle">Discovery of the sixth Candida auris clade in Singapore</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.23293435" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.23293435" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Suphavilai, C.; Ko, K. K. K.; Lim, K. M.; Tan, M. G.; Boonsimma, P.; Chu, J. J. K.; Goh, S. S.; Rajandran, P.; Lee, L. C.; Tan, K. Y.; Shaik Ismail, B.; Aung, M. K.; Yang, Y.; Sim, J. X. Y.; Venkatachalam, I.; Cherng, B. P. Z.; Spruijtenburg, B.; Chan, K. S.; Oon, L. L. E.; Tan, A. L.; Tan, Y. E.; Wijaya, L.; Tan, B. H.; Ling, M. L.; Koh, T. H.; Meis, J. F.; Tsui, C. K. M.; Nagarajan, N.</p>
        <p class="info">Score: 24.6, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.23293435' target='https://doi.org/10.1101/2023.08.01.23293435'> 10.1101/2023.08.01.23293435</a></p>
        <p class="abstract">BackgroundThe emerging fungal pathogen Candida auris poses a serious threat to global public health due to its worldwide distribution, multidrug-resistance, high transmissibility, propensity to cause outbreaks and high mortality rates. We report three C. auris isolates detected in Singapore, which are genetically distinct from all known clades (I-V) and represent a new clade (Clade VI).

MethodsThree epidemiologically unlinked clinical isolates belonging to the potential new C. auris clade were whole-genome sequenced and phenotypically characterized. The complete genomes of these isolates were compared to representative genomes of all known clades. To provide a global context, 3,651 international whole-genome sequences (WGS) from the NCBI database were included in the high-resolution single nucleotide polymorphism (SNP) analysis. Antifungal resistance genes, mating type locus, and chromosomal rearrangements were characterized from the WGS data of the Clade VI isolates. We further implemented Bayesian logistic regression models to simulate the automatic detection of Clade V and VI as their WGS data became available.

FindingsThe three Clade VI isolates were separated by &gt;36,000 SNPs from all existing C. auris clades. These isolates had opposite mating type allele and different chromosomal rearrangements when compared to their closest Clade IV relatives. As a proof-of-concept, our classification model was able to flag these outlier genomes as a potential new clade. Furthermore, an independent WGS submission from Bangladesh was found to belong to this new clade.

InterpretationThe discovery of a new C. auris clade in Singapore and Bangladesh, showing close relationship to Clade IV members in South America, highlights the unknown genetic diversity and origin of C. auris, particularly in under-resourced regions. Active surveillance in clinical settings, along with effective sequencing strategies and downstream analysis, will be essential in the identification of novel strains, tracking of transmission, and containment of adverse clinical impacts caused by C. auris infections.

FundingThis work was supported by the Singapore National Medical Research Council (NMRC) research training fellowship (MOH-FLWSHP19may-0005), the NCRS Duke-NUS Academic Medical Center Academic Clinical Program grant (09/FY2022/P1/17-A32, GRDUKP003401), and the Genedant-GIS Innovation Program grant.

Research in contextO_ST_ABSEvidence before this studyC_ST_ABSWe searched PubMed using the search terms &#34;Candida auris&#34; AND &#34;clade&#34;, for papers published between Jan 1, 2009, and July 1, 2023. This search retrieved 115 publications. 60 relevant publications were identified. 28 studies analyzed and discussed the molecular epidemiology of C. auris, including the description of C. auris clades, either in outbreak or surveillance settings. There were 11 case reports of C. auris clinical cases that included clade determination. Two studies focused on the detection and clade determination of C. auris from non-healthcare environments. Clade-specific characteristics were described or analyzed in 14 studies. One study applied machine learning to C. auris drug resistance analysis, but not for clade determination. Four studies focused on the description of potentially new C. auris lineages, subclades, or clades. All publications described isolates that belong to one of the five known C. auris clades (I-V). All publications found that strains from different clades differed by more than 35,000 SNPs, and that there are clade-specific differences in geographical distribution, phenotypic characteristics, antifungal susceptibility profile, outbreak potential, and clinical manifestations.

The NCBI Pathogen Detection system contained 4,506 C. auris genomes on July 1, 2023. There were ten (0{middle dot}22%) submissions from Southeast Asian countries and 92 (2{middle dot}04%) submissions from South Asia and the Indian subcontinent, which are parts of the Indomalayan biogeographic realm.

Added value of this studyTo the best of our knowledge, we are the first group to perform hybrid assemblies on three representative isolates in a new C. auris clade, which is separated from all other existing clades (I-V) by &gt;36,000 SNPs. Whole-genome SNP analysis and phenotypic characterization of these epidemiologically unlinked isolates detected in Singapore suggest that they represent a previously unreported sixth major clade. High-resolution SNP analysis of 3,651 international whole-genome sequences from the NCBI database, which generated the final dataset consisting of more than 6.6 million genome pairs, revealed six distinct genetic clusters representing the five known clades and the new sixth clade (Indomalayan). In addition, we demonstrate that a machine learning approach can be used to flag these outlier genomes for further investigations as soon as they become available, thus providing the earliest possible alert for potential new public health threats.

Implications of all the available evidenceDespite the high antimicrobial resistance burden in Southeast Asia and South Asia, these regions are disproportionately underrepresented in terms of genomic surveillance of C. auris, a multidrug-resistant fungal pathogen. The detection of three epidemiologically unlinked C. auris isolates in Singapore belonging to a new C. auris clade suggests that yet-to-be-reported strains may be circulating in the region. Given the propensity for multidrug resistance, healthcare-associated infection outbreaks, and the associated high mortality, active surveillance and continued vigilance is necessary.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.23293266">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.23293266" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.23293266">
        <p class="paperTitle">Early underdetected dissemination followed by extensive local transmission propelled the 2022 mpox epidemic and limited impact of vaccination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.23293266" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.23293266" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Paredes, M. I.; Ahmed, N.; Figgins, M.; Colizza, V.; Lemey, P.; McCrone, J. T.; Müller, N. F.; Tran-Kiem, C.; Bedford, T.</p>
        <p class="info">Score: 9.0, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.23293266' target='https://doi.org/10.1101/2023.07.27.23293266'> 10.1101/2023.07.27.23293266</a></p>
        <p class="abstract">The World Health Organization (WHO) declared mpox a public health emergency of international concern in July 2022. It is still unclear to what extent international travel contributed to the explosive spread of mpox and the degree to which national vaccination campaigns were responsible for controlling the epidemic. We built phylogeographic and phylodynamic models to analyze MPXV genomes sampled between March 2022 and January 2023 from five global regions together with air traffic and epidemiological data to analyze the global spread of mpox. Our models reveal community transmission prior to detection by local surveillance, changes in case-reporting throughout the epidemic, and a large degree of transmission heterogeneity. Additionally, we find that viral introductions played a limited role in prolonging spread after initial dissemination, suggesting that travel bans would have had only a minor impact. We find that the time-varying effective reproductive number in North America declines below one before more than 10% of individuals at high risk individuals in the USA had vaccine-induced immunity, suggesting little impact of vaccination in controlling the epidemic. Given that cases quickly declined after detection most likely due to behavioral modifications, our findings highlight the importance of broader routine specimen screening surveillance for emerging infectious diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.23293369">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.23293369" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.23293369">
        <p class="paperTitle">Effectiveness of one dose of killed oral cholera vaccine in an endemic community in the Democratic Republic of Congo: A matched case-control study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.23293369" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.23293369" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Malembaka, E. B.; Bugeme, P. M.; Hutchins, C.; Xu, H.; Dent, J. D.; Demby, M. N.; Gallandat, K.; Saidi, J. M.; Rumedeka, B. B.; Itongwa, M.; Tshiwedi-Tsilabia, E.; Kitoga, F.; Bodisa-Matamu, T.; Kavunga-Membo, H.; Kulondwa, J.-C.; Bengehya, J.; Debes, A. K.; Taty, N.; Lee, E. C.; Lessler, J.; Leung, D. T.; Cumming, O.; Okitayemba, P. W.; Mukadi-Bamuleka, D.; Knee, J.; Azman, A. S.</p>
        <p class="info">Score: 6.8, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.23293369' target='https://doi.org/10.1101/2023.08.07.23293369'> 10.1101/2023.08.07.23293369</a></p>
        <p class="abstract">BackgroundA global shortage of cholera vaccines has increased use of single-dose regimens, rather than the standard two-dose regimen. There is limited evidence on single-dose protection, particularly in children. In late 2020 a mass vaccination campaign was conducted in Uvira, a cholera endemic urban setting in eastern Democratic Republic of the Congo. Coverage surveys showed that the majority of vaccinated people received only one dose. We examined the effectiveness of a single-dose of the oral cholera vaccine Euvichol-plus in this high-burden setting.

MethodsIn this case-control study, we recruited medically-attended confirmed cholera cases and age-, sex-, and neighborhood-matched community controls during two distinct periods after mass vaccination, October 2021 to March 2022 (12-17 months post-vaccination) and October 2022 to May 2023 (24-31 months post-vaccination). The odds of vaccination in cases and controls were contrasted in conditional logistic regression models to estimate unadjusted and adjusted vaccine effectiveness.

FindingsWe enrolled 432 confirmed cases and 1554 matched controls during the two study periods with 21.5% of cases being under five years old. The adjusted single-dose VE was 52% (95% CI: 32{middle dot}3-66) 12-17 months post-vaccination and 40{middle dot}8% (95% CI: 8{middle dot}6-61{middle dot}7) 24-31 months post-vaccination. Combining the two study periods led to an adjusted single-dose VE of 45% (95% CI 28{middle dot}2-57{middle dot}8), with no significant differences among children under five years old and older individuals.

InterpretationA single-dose of Euvichol-Plus provided substantial protection against medically-attended cholera for at least 31 months post-vaccination in this cholera endemic setting.

FundingThis work was supported by the Wellcome Trust and Gavi (GAVI-RFP-2019-062).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.23293566">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.23293566" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.23293566">
        <p class="paperTitle">SARS-CoV-2 Pandemic Non-Pharmacologic Interventions Temporally Associated with Reduced Pediatric Infections Due to Mycoplasma pneumoniae and Co-Infecting Respiratory Viruses in Arkansas</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.23293566" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.23293566" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boyanton, B. L.; Frenner, R. A.; Ingold, A.; Ambroggio, L.; Kennedy, J. L.</p>
        <p class="info">Score: 4.8, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.23293566' target='https://doi.org/10.1101/2023.08.05.23293566'> 10.1101/2023.08.05.23293566</a></p>
        <p class="abstract">IntroductionNon-pharmacologic interventions (NPIs), such as universal masking, implemented during the SARS-CoV-2 pandemic have reduced respiratory infections among children. This study focuses on evaluating the impact of NPIs on Mycoplasma pneumoniae infections in children, analyzing data from two hospitals in Arkansas, and examining age-related differences and coinfections with other viruses.

MethodsThe study was approved by the Institutional Review Board and included patients aged [&amp;le;]18 years with upper respiratory tract symptoms. Data from the FilmArray(R) Respiratory Panel (FARP) were collected and divided into pre-NPI and NPI periods for analysis. Total test positivity rate and interval change in the positivity rate were evaluated. Statistical differences were determined by Chi-square ({chi}2-independence) analysis.

ResultsA total of 68,949 tests were performed with a statistical increase in testing during the NPI period. The overall test positivity rate for M. pneumoniae decreased by 74% (0.86% to 0.03%) during the NPI period, and the preschool age group had the highest number of positive tests in the pre- and NPI periods (Pre-NPI: n=40, NPI: n=12 positive tests, p=&lt;0.001). The reduction in M. pneumoniae infections was consistent across age groups. Coinfections with other respiratory viruses, particularly human rhinovirus/enterovirus, were observed at much lower levels.

ConclusionsNPIs effectively reduced M. pneumoniae in pediatric patients in Arkansas, and coinfections with specific viruses still occurred, albeit at lower levels during the SARS-CoV-2 pandemic. As NPIs are relaxed and the pandemic ends, we expect M. pneumoniae infections to return to pre-pandemic levels within the next 1-2 years.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.23292649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.23292649" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.23292649">
        <p class="paperTitle">Early Biological Markers of Post-Acute Sequelae of SARS-CoV-2 Infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.23292649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.23292649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lu, S.; Peluso, M. J.; Glidden, D. V.; Davidson, M.; Lugtu, K.; Pineda-Ramirez, J.; Tassetto, M.; Garcia-Knight, M.; Zhang, A.; Goldberg, S. A.; Chen, J.; Forbes, M.; Park, S.; Martinez, A.; So, M.; Donovan, A.; Viswanathan, B.; Hoh, R.; Donohue, K.; McIlwain, D.; Gaudilliere, B.; Anglin, K.; Yee, B.; Chenna, A. A.; Winslow, J.; Petropoulos, C.; Briggs-Hagen, M.; Andino, R.; Midgley, C.; Martin, J. N.; Saydeh, S.; Kelly, J. D.</p>
        <p class="info">Score: 306.3, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.23292649' target='https://doi.org/10.1101/2023.07.14.23292649'> 10.1101/2023.07.14.23292649</a></p>
        <p class="abstract">To understand the roles of acute phase viral dynamics and host immune responses in PASC, we enrolled 136 participants within 5 days of their first positive SARS-CoV-2 real-time PCR. Participants self-collected nasal specimens up to 21 times within the first 28 days after symptom onset; Interviewer-administered clinical questionnaires and blood samples were collected at enrollment and days 9, 14, 21, 28, and month 4 and 8 post-symptom. Defining PASC as the presence of any symptom new or worse since infection reported at their 4-month visit, we compared viral markers (quantity and duration of viral RNA load, infectious viral load, and plasma N-antigen level) and host immune markers (IL-6, IL-10, TNF-, IFN-, IFN-{gamma}, MCP, IP-10, and Spike IgG) over the acute period. In comparison to those who fully recovered, those who developed PASC demonstrated significantly higher maximum levels of SARS-CoV-2 RNA, infectious virus, and N-antigen, longer duration of viral shedding, and lower Spike-specific IgG levels within the first 10 days of the acute phase of illness. No significant differences were identified among a panel of host immune markers, though there was a trend toward higher initial levels of certain markers (e.g., MCP-1, IFN-, and IFN-{gamma}) in those who went on to develop PASC. Early viral dynamics and the associated host immune responses play a role in the pathogenesis of PASC. These findings highlight the importance of understanding the early biological markers from acute SARS-CoV-2 infection in the natural history of PASC.

Onset Sentence SummaryEarly viral dynamics and the associated host immune responses play a role in the pathogenesis of PASC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.23293191">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.23293191" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.23293191">
        <p class="paperTitle">Chronic Shedding of a SARS-CoV-2 Alpha Variant Lineage Q.3/Q.4 in Wastewater</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.23293191" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.23293191" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Conway, M. J.; Yang, H.; Revord, L. A.; Ward, A. S.; Abel, J. D.; Williams, M. R.; Uzarski, R. L.; Alm, E. W.</p>
        <p class="info">Score: 4.2, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.23293191' target='https://doi.org/10.1101/2023.07.26.23293191'> 10.1101/2023.07.26.23293191</a></p>
        <p class="abstract">Central Michigan University (CMU) participated in a state-wide SARS-CoV-2 wastewater monitoring program since 2021. Wastewater samples were collected from on-campus sites and nine off-campus wastewater treatment plants servicing small metropolitan and rural communities. SARS-CoV-2 genome copies were quantified using droplet digital PCR and results were reported to the health department. One rural, off-campus site consistently produced higher concentrations of SARS-CoV-2 genome copies. Samples from this site were sequenced and initially contained predominately Alpha variant lineage Q.3, which transitioned to lineage Q.4. Alpha variant lineage Q.3/Q.4 was detected at this site beginning in fall 2021 and continued until summer 2023. Mutational analysis of reconstructed genes revealed divergence from the Alpha variant lineage Q.3 clinical sequence over time, including numerous mutations in the surface glycoprotein RBD and NTD. We discuss the possibility that a chronic SARS-CoV-2 infection accumulated adaptive mutations that promoted long-term infection. This study reveals that small wastewater treatment plants can enhance resolution of rare events and facilitate reconstruction of viral genomes due to the relative lack of contaminating sequences.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
