<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19
Authors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19
Authors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-27T10:37:02+00:00" />
<meta property="article:modified_time" content="2023-08-27T10:37:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19
Authors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19\nAuthors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A.",
  "keywords": [
    
  ],
  "articleBody": " Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19\nAuthors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A.; Dalhuisen, T.; Durstenfeld, M. S.; Hsue, P. Y.; Kelly, J. D.; Kumar, N.; Martin, J. N.; Gambhir, A.; Somsouk, M.; Seo, Y.; Deeks, S. G.; Laszik, Z. G.; VanBrocklin, H. F.; Henrich, T. J.\nScore: 2071.4, Published: 2023-07-31 DOI: 10.1101/2023.07.27.23293177\nThe etiologic mechanisms of post-acute medical morbidities and unexplained symptoms (Long COVID) following SARS-CoV-2 infection are incompletely understood. There is growing evidence that viral persistence and immune dysregulation may play a major role. We performed whole-body positron emission tomography (PET) imaging in a cohort of 24 participants at time points ranging from 27 to 910 days following acute SARS-CoV-2 infection using a novel radiopharmaceutical agent, [18F]F-AraG, a highly selective tracer that allows for anatomical quantitation of activated T lymphocytes. Tracer uptake in the post-acute COVID group, which included those with and without Long COVID symptoms, was significantly higher compared to pre-pandemic controls in many anatomical regions, including the brain stem, spinal cord, bone marrow, nasopharyngeal and hilar lymphoid tissue, cardiopulmonary tissues, and gut wall. Although T cell activation tended to be higher in participants imaged closer to the time of the acute illness, tracer uptake was increased in participants imaged up to 2.5 years following SARS-CoV-2 infection. We observed that T cell activation in spinal cord and gut wall was associated with the presence of Long COVID symptoms. In addition, tracer uptake in lung tissue was higher in those with persistent pulmonary symptoms. Notably, increased T cell activation in these tissues was also observed in many individuals without Long COVID. Given the high [18F]F-AraG uptake detected in the gut, we obtained colorectal tissue for in situ hybridization SARS-CoV-2 RNA and immunohistochemical studies in a subset of participants with Long COVID symptoms. We identified cellular SARS-CoV-2 RNA in rectosigmoid lamina propria tissue in all these participants, ranging from 158 to 676 days following initial COVID-19 illness, suggesting that tissue viral persistence could be associated with long-term immunological perturbations.\nA molnupiravir-associated mutational signature in global SARS-CoV-2 genomes\nAuthors: Sanderson, T.; Hisner, R.; Donovan-Banfield, I.; Hartman, H.; Lochen, A.; Peacock, T. P.; Ruis, C.\nScore: 2427.4, Published: 2023-08-18 DOI: 10.1101/2023.01.26.23284998\nMolnupiravir, an antiviral medication that has been widely used against SARS-CoV-2, acts by inducing mutations in the virus genome during replication. Most random mutations are likely to be deleterious to the virus, and many will be lethal, and so molnupiravir-induced elevated mutation rates reduce viral load2,3. However, if some patients treated with molnupiravir do not fully clear SARS-CoV-2 infections, there could be the potential for onward transmission of molnupiravir-mutated viruses. Here we show that SARS-CoV-2 sequencing databases contain extensive evidence of molnupiravir mutagenesis. Using a systematic approach, we find that a specific class of long phylogenetic branches, distinguished by a high proportion of G-to-A and C-to-T mutations, appear almost exclusively in sequences from 2022, after the introduction of molnupiravir treatment, and in countries and age-groups with widespread usage of the drug. We identify a mutational spectrum, with preferred nucleotide contexts, from viruses in patients known to have been treated with molnupiravir and show that its signature matches that seen in these long branches, in some cases with onwards transmission of molnupiravir-derived lineages. Finally, we analyse treatment records to confirm a direct association between these high G-to-A branches and the use of molnupiravir.\nSafety and Immunogenicity of XBB.1.5-Containing mRNA Vaccines\nAuthors: Chalkias, S.; McGhee, N.; Whatley, J. L.; Essink, B.; Brosz, A.; Tomassini, J. E.; Girard, B.; Edwards, D. K.; Nasir, A.; Lee, D.; Avena, L. E.; Feng, J.; Deng, W.; Montefiori, D. C.; Baden, L. R.; Miller, J. M.; Das, R.\nScore: 47.9, Published: 2023-08-24 DOI: 10.1101/2023.08.22.23293434\nBackground: Subvariants of the severe acute respiratory syndrome coronavirus (SARS-CoV-2) omicron XBB-lineage have the potential to escape immunity provided by prior infection or vaccination. For Covid-19 immunizations beginning in the Fall 2023, the U.S. FDA has recommended updating to a monovalent omicron XBB.1.5-containing vaccine. Methods: In this ongoing, phase 2/3 study participants were randomized 1:1 to receive 50-{micro}g doses of mRNA-1273.815 monovalent (50-{micro}g omicron XBB.1.5 spike mRNA) or mRNA-1273.231 bivalent (25-{micro}g omicron XBB.1.5 and 25-{micro}g omicron BA.4/BA.5 spike mRNAs)vaccines, administered as 5th doses, to adults who previously received a primary series and 3rd dose of an original mRNA coronavirus disease 2019 (Covid-19) vaccine, and a 4th dose of a bivalent (omicron BA.4/BA.5 and original SARS-CoV-2) vaccine. Interim safety and immunogenicity data 15 days post-vaccination are presented. Results: In April 2023, participants received mRNA-1273.815 (n=50) and mRNA-1273.231 (n=51). The median intervals from the prior dose of BA.4/BA.5-containing bivalent vaccine were 8.2 and 8.3 months for the mRNA-1273.815 and mRNA-1273.231 groups, respectively. Both vaccines increased neutralizing antibody (nAb) geometric mean titers against all variants tested at day 15 post-booster nAb compared to pre-booster levels. Geometric mean fold-rises from pre-booster titers after the monovalent booster were higher against XBB.1.5, XBB.1.16 and SARS-CoV-2 (D614G) than those of the bivalent booster and were comparable against BA.4/BA.5 and BQ1.1 variants for both vaccines. The monovalent vaccine also elicited nAb responses against omicron XBB.2.3.2, EG.5.1, and FL.1.5.1 that were similar to those seen for XBB.1.5 in a subset (n=20) of participants. The occurrence of solicited adverse reactions and unsolicited adverse events were overall similar to those previously reported for the original mRNA-1273 50-{micro}g and omicron BA.4/BA.5-containing bivalent mRNA-1273 vaccines. Conclusion: In this interim analysis, XBB.1.5-containing monovalent and bivalent vaccines elicited potent neutralizing responses against variants of the omicron XBB-lineage (XBB.1.5, XBB.1.6, XBB.2.3.2) as well as more recent (EG.5.1, FL.1.5.1) variants. The safety profile of the XBB.1.5-containing vaccine was consistent with those of prior vaccines. These results support the Covid-19 vaccine update in 2023-2024 to a monovalent XBB.1.5-containing vaccine.\nSARS-CoV-2 shedding and evolution in immunocompromised hosts during the Omicron period: a multicenter prospective analysis\nAuthors: Raglow, Z.; Surie, D.; Chappell, J. D.; Zhu, Y.; Martin, E. T.; Kwon, J. H.; Frosch, A. E.; Mohamed, A.; Gilbert, J.; Bendall, E. E.; Bahr, A.; Halasa, N.; Talbot, H. K.; Grijalva, C. G.; Baughman, A.; Womack, K. N.; Johnson, C.; Swan, S. A.; Koumans, E.; McMorrow, M. L.; Harcourt, J. L.; Atherton, L. J.; Burroughs, A.; Thornburg, N. J.; Self, W. H.; Lauring, A. S.\nScore: 24.1, Published: 2023-08-24 DOI: 10.1101/2023.08.22.23294416\nBackground: Prolonged SARS-CoV-2 infections in immunocompromised hosts may predict or source the emergence of highly mutated variants. The types of immunosuppression placing patients at highest risk for prolonged infection and associated intrahost viral evolution remain unclear. Methods: Adults aged \u003e18 years were enrolled at 5 hospitals and followed from 4/11/2022-2/1/2023. Eligible patients were SARS-CoV-2 positive in the previous 14 days and had a moderate or severely immunocompromising condition or treatment. Nasal specimens were tested by rRT-PCR every 2-4 weeks until negative in consecutive specimens. Positive specimens underwent viral culture and whole genome sequencing. A Cox proportional hazards model was used to assess factors associated with duration of infection. Results: We enrolled 150 patients with: B cell malignancy or anti-B cell therapy (n=18), solid organ or hematopoietic stem cell transplant (SOT/HSCT) (n=59), AIDS (n=5), non-B cell malignancy (n=23), and autoimmune/autoinflammatory conditions (n=45). Thirty-eight (25%) were rRT-PCR positive and 12 (8%) were culture-positive [\u0026ge;]21 days after initial SARS-CoV-2 detection or illness onset. Patients with B cell dysfunction had longer duration of rRT-PCR positivity compared to those with autoimmune/autoinflammatory conditions (aHR 0.32, 95% CI 0.15-0.64). Consensus (\u003e50% frequency) spike mutations were identified in 5 individuals who were rRT-PCR positive \u003e56 days; 61% were in the receptor-binding domain (RBD). Mutations shared by multiple individuals were rare (\u003c5%) in global circulation.\nPublished benefits of ivermectin use in Itajai, Brazil for COVID-19 infection, hospitalisation, and mortality are entirely explained by statistical artefacts\nAuthors: Mills, R.; Pecanha Antonio, A. C.; Tucker-Kellogg, G.\nScore: 286.6, Published: 2023-08-15 DOI: 10.1101/2023.08.10.23293924\nBackgroundTwo recent publications by Kerr et al. (Cureus 14(1):e21272; Cureus 14(8): e28624) reported dramatic effects of prophylactic ivermectin use for both prevention of COVID-19 and reduction of COVID-19-related hospitalisation and mortality, including a dose-dependent effect of ivermectin prophylaxis. These papers have gained an unusually large public influence: they were incorporated into debates around COVID-19 policies and may have contributed to decreased trust in vaccine efficacy and public health authorities more broadly. Both studies were based on retrospective observational analysis of city-wide registry data from the city of Itajai, Brazil from July-December 2020. MethodsStarting with initially identified sources of error, we conducted a revised statistical analysis of available data, including data made available with the original papers and public data from the Brazil Ministry of Health. We identified additional uncorrected sources of bias and errors from the original analysis, including incorrect subject exclusion and missing subjects, an enrolment time bias, and multiple sources of immortal time bias. In models assuming no actual effect from ivermectin use, we conducted Monte Carlo simulations to estimate the contribution of these biases to any observed effect. ResultsUntreated statistical artefacts and methodological errors alone lead to dramatic apparent risk reduction associated with Ivermectin use in both studies. The magnitude of apparent risk reduction from these artefacts is comparable to the results reported by the studies themselves, including apparent protection from infection, hospitalisation, and death, and including the reported apparent dose-response relationship. ConclusionsThe inference of ivermectin efficacy reported in both papers is unsupported, as the observed effects are entirely explained by untreated statistical artefacts and methodological errors. Our re-analysis calls for caution in interpreting highly publicised observational studies and highlights the importance of common sources of bias in clinical research.\nHeterogeneous SARS-CoV-2 kinetics due to variable timing and intensity of immune responses\nAuthors: Owens, K.; Esmaeili-Wellman, S.; Schiffer, J. T.\nScore: 10.8, Published: 2023-08-21 DOI: 10.1101/2023.08.20.23294350\nThe viral kinetics of documented SARS-CoV-2 infections exhibit a high degree of inter-individual variability. We identified six distinct viral shedding patterns, which differed according to peak viral load, duration, expansion rate and clearance rate, by clustering data from 810 infections in the National Basketball Association cohort. Omicron variant infections in previously vaccinated individuals generally led to lower cumulative shedding levels of SARS-CoV-2 than other scenarios. We then developed a mechanistic mathematical model that recapitulated 1510 observed viral trajectories, including viral rebound and cases of reinfection. Lower peak viral loads were explained by a more rapid and sustained transition of susceptible cells to a refractory state during infection, as well as an earlier and more potent late, cytolytic immune response. Our results suggest that viral elimination occurs more rapidly during omicron infection, following vaccination, and following re-infection due to enhanced innate and acquired immune responses. Because viral load has been linked with COVID-19 severity and transmission risk, our model provides a framework for understanding the wide range of observed SARS-CoV-2 infection outcomes.\nThe post-COVID-19 population has a high prevalence of crossreactive antibodies to spikes from all Orthocoronavirinae genera\nAuthors: Singh, G.; Abbad, A.; Kleiner, G.; Srivastava, K.; Gleason, C.; Carreno Quiroz, J. M.; Simon, V.; Krammer, F.\nScore: 61.9, Published: 2023-08-06 DOI: 10.1101/2023.08.01.23293522\nThe Orthocoronaviridae subfamily is large comprising four highly divergent genera. Four seasonal coronaviruses were circulating in humans prior to the coronavirus disease 2019 (COVID-19) pandemic. Infection with these viruses induced antibody responses that are relatively narrow with little cross-reactivity to spike proteins of other coronaviruses. Here, we report that infection with and vaccination against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) induces broadly crossreactive binding antibodies to spikes from a wide range of coronaviruses including members of the sarbecovirus subgenus, betacoronaviruses including Middle Eastern respiratory syndrome coronavirus (MERS CoV), and extending to alpha-, gamma- and delta-coronavirus spikes. These data show that the coronavirus spike antibody landscape in humans has profoundly been changed and broadened as a result of the SARS-CoV-2 pandemic. While we do not understand the functionality of these crossreactive antibodies, they may lead to enhanced resistance of the population to infection with newly emerging coronaviruses with pandemic potential.\nProtection conferred by COVID-19 vaccination, prior SARS-CoV-2 infection, or hybrid immunity against Omicron-associated severe outcomes among community-dwelling adults\nAuthors: Lee, N.; Nguyen, L.; Austin, P. C.; Brown, K. A.; Grewal, R.; Buchan, S. A.; Nasreen, S.; Gubbay, J.; Schwartz, K. L.; Tadrous, M.; Wilson, K.; Wilson, S. E.; Kwong, J. C.\nScore: 5.7, Published: 2023-08-25 DOI: 10.1101/2023.08.24.23294503\nIntroduction: We assessed protection conferred by COVID-19 vaccines and/or prior SARS-CoV-2 infection against Omicron-associated severe outcomes during successive sublineage-predominant periods. Methods: We used a test-negative design to estimate protection by vaccines and/or prior infection against hospitalization/death among community-dwelling, PCR-tested adults aged \u003e50 years in Ontario, Canada between January 2, 2022 and June 30, 2023. Multivariable logistic regression was used to estimate the relative change in the odds of hospitalization/death with each vaccine dose (2-5) and/or prior PCR-confirmed SARS-CoV-2 infection (compared with unvaccinated, uninfected subjects) up to 15 months since the last vaccination or infection. Results: We included 18,526 cases with Omicron-associated severe outcomes and 90,778 test-negative controls. Vaccine protection was high during BA.1/BA.2 predominance, but was generally \u003c50% during periods of BA.4/BA.5 and BQ/XBB predominance without boosters. A third/fourth dose transiently increased protection during BA.4/BA.5 predominance (third-dose, 6-month: 68%, 95%CI 63%-72%; fourth-dose, 6-month: 80%, 95%CI 77%-83%), but was lower and waned quickly during BQ/XBB predominance (third-dose, 6-month: 59%, 95%CI 48%-67%; 12-month: 49%, 95%CI 41%-56%; fourth-dose, 6-month: 62%, 95%CI 56%-68%, 12-months: 51%, 95%CI 41%-56%). Hybrid immunity conferred nearly 90% protection throughout BA.1/BA.2 and BA.4/BA.5 predominance, but was reduced during BQ/XBB predominance (third-dose, 6-month: 60%, 95%CI 36%-75%; fourth-dose, 6-month: 63%, 95%CI 42%-76%). Protection was restored with a fifth dose (bivalent; 6-month: 91%, 95%CI 79%-96%). Prior infection alone did not confer lasting protection. Conclusion: Protection from COVID-19 vaccines and/or prior SARS-CoV-2 infections against severe outcomes is reduced when immune-evasive variants/subvariants emerge and may also wane over time. Our findings support a variant-adapted booster vaccination strategy with periodic review.\nCohort Profile: Prospective Cohort to Study the COVID-19 Immune Response in Retail Workers in Quebec, Canada (CISACOV)\nAuthors: Santerre, K.; Theriault, M.; Brousseau, N.; Rochette, S.; Pelletier, J.; Gilbert, C.; Masson, J.-F.; Baz, M.; Boudreau, D.; Trottier, S.\nScore: 5.3, Published: 2023-08-21 DOI: 10.1101/2023.08.18.23294172\nPurposeRetail workers are an understudied occupational group that may have been at increased risk of contracting SARS-CoV-2 during the COVID-19 pandemic. Therefore, we set up a longitudinal cohort of participants working in this sector to better document the incidence of SARS-CoV-2 infection and the immune response to infection and/or vaccination in this group. ParticipantsA total of 304 participants were recruited between April 20, 2021 and October 22, 2021. They were invited to attend three visits (each separated by [~]12 weeks) during which they provided blood samples and information on participant characteristics, COVID-19 symptoms, and vaccination. An extension phase of two additional visits was carried out between March 15th, 2022 and October 3rd, 2022 to document the impact of the Omicron variant among the 198 participants who were still eligible for recruitment. Participants were aged 18 to 75 and worked in grocery stores, hardware stores, bars or restaurants within the Quebec City metropolitan area (Canada). Findings to date: This article describes participants demographic, socioeconomic, behavioral, clinical and occupational characteristics, and their COVID-19 symptoms (where applicable). It also describes SARS-CoV-2 vaccination status and any SARS-CoV-2 diagnostic test (i.e., PCR or rapid antigen) performed from the beginning of the pandemic until the last visit. Future plansThe incidence of SARS-CoV-2 infections will be assessed. The immune response (innate and acquired) to SARS-CoV-2 infection or vaccination will be studied using a variety of techniques, including reference and experimental enzyme-linked immunosorbent assays, microneutralization assays with live viruses, experimental pseudoneutralization with an angiotensin-converting enzyme 2-spike assay, peripheral blood mononuclear cells and neutrophil stimulation, and a proliferation assay based on carboxyfluorescein diacetate succinimidyl ester. RegistrationNot applicable. Strengths and limitationsO_LIThis cohort offers a comprehensive dataset to study the immune response to SARS-CoV-2 infection or vaccination (alone), or hybrid immunity, as participants provided information on a wide range of demographic, socioeconomic, behavioral, clinical, and occupational variables. However, the low proportion of racial minorities (i.e., 3.0%) limits the use of this cohort to study racial determinants of immunity to SARS-CoV-2. C_LIO_LIThis cohort focuses on workers in the food and retail service sector, an understudied population at high risk of occupational exposure to infectious agents. C_LIO_LIThis study covered seven pandemic waves and thus captured a large number of confirmed infections from different variants. C_LIO_LISample collection was initiated immediately prior to COVID-19 vaccine availability for this population and thus captured successive vaccination campaigns over 17 months. C_LIO_LIIn keeping with the study design, none of the participants had severe COVID-19 disease requiring hospitalization at baseline, and none of the COVID-19 illnesses that occurred during the study required hospitalization; this cohort may not, therefore, be used to study the immune response leading to severe health outcomes, but is appropriate to study the immune response to mild SARS-CoV-2 infections. C_LIO_LINearly 1300 blood samples were collected; furthermore, only 13 out of 304 (4.3%) participants withdrew before attending all three initial visits, and 4 out of 198 (2.0%) participants who remained eligible in the extension phase withdrew before attending the fifth visit. A series of at least 5 blood samples drawn over 48 weeks is therefore available for most participants. C_LI\nSARS-CoV-2 Viral Clearance and Evolution Varies by Extent of Immunodeficiency\nAuthors: Li, Y.; Choudhary, M. C.; Regan, J.; Boucau, J.; Nathan, A.; Speidel, T.; Liew, M. Y.; Edelstein, G. E.; Kawano, Y.; Uddin, R.; Deo, R.; Marino, C.; Getz, M. A.; Reynold, Z.; Barry, M.; Gilbert, R. F.; Tien, D.; Sagar, S.; Vyas, T. D.; Flynn, J. P.; Hammond, S. P.; Novack, L. A.; Choi, B.; Cernadas, M.; Wallace, Z. S.; Sparks, J. A.; Vyas, J. M.; Seaman, M. S.; Gaiha, G. D.; Siedner, M. J.; Barczak, A. K.; Lemieux, J. E.; Li, J. Z.\nScore: 53.1, Published: 2023-08-07 DOI: 10.1101/2023.07.31.23293441\nDespite vaccination and antiviral therapies, immunocompromised individuals are at risk for prolonged SARS-CoV-2 infection, but the immune defects that predispose to persistent COVID-19 remain incompletely understood. In this study, we performed detailed viro-immunologic analyses of a prospective cohort of participants with COVID-19. The median time to nasal viral RNA and culture clearance in the severe hematologic malignancy/transplant group (S-HT) were 72 and 40 days, respectively, which were significantly longer than clearance rates in the severe autoimmune/B-cell deficient (S-A), non-severe, and non-immunocompromised groups (P\u003c0.001). Participants who were severely immunocompromised had greater SARS-CoV-2 evolution and a higher risk of developing antiviral treatment resistance. Both S-HT and S-A participants had diminished SARS-CoV-2-specific humoral, while only the S-HT group had reduced T cell-mediated responses. This highlights the varied risk of persistent COVID-19 across immunosuppressive conditions and suggests that suppression of both B and T cell responses results in the highest contributing risk of persistent infection.\n",
  "wordCount" : "3127",
  "inLanguage": "en",
  "datePublished": "2023-08-27T10:37:02Z",
  "dateModified": "2023-08-27T10:37:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta"><span>updated on August 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.23293177">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.23293177" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.23293177">
        <p class="paperTitle">Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.23293177" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.23293177" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A.; Dalhuisen, T.; Durstenfeld, M. S.; Hsue, P. Y.; Kelly, J. D.; Kumar, N.; Martin, J. N.; Gambhir, A.; Somsouk, M.; Seo, Y.; Deeks, S. G.; Laszik, Z. G.; VanBrocklin, H. F.; Henrich, T. J.</p>
        <p class="info">Score: 2071.4, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.23293177' target='https://doi.org/10.1101/2023.07.27.23293177'> 10.1101/2023.07.27.23293177</a></p>
        <p class="abstract">The etiologic mechanisms of post-acute medical morbidities and unexplained symptoms (Long COVID) following SARS-CoV-2 infection are incompletely understood. There is growing evidence that viral persistence and immune dysregulation may play a major role. We performed whole-body positron emission tomography (PET) imaging in a cohort of 24 participants at time points ranging from 27 to 910 days following acute SARS-CoV-2 infection using a novel radiopharmaceutical agent, [18F]F-AraG, a highly selective tracer that allows for anatomical quantitation of activated T lymphocytes. Tracer uptake in the post-acute COVID group, which included those with and without Long COVID symptoms, was significantly higher compared to pre-pandemic controls in many anatomical regions, including the brain stem, spinal cord, bone marrow, nasopharyngeal and hilar lymphoid tissue, cardiopulmonary tissues, and gut wall. Although T cell activation tended to be higher in participants imaged closer to the time of the acute illness, tracer uptake was increased in participants imaged up to 2.5 years following SARS-CoV-2 infection. We observed that T cell activation in spinal cord and gut wall was associated with the presence of Long COVID symptoms. In addition, tracer uptake in lung tissue was higher in those with persistent pulmonary symptoms. Notably, increased T cell activation in these tissues was also observed in many individuals without Long COVID. Given the high [18F]F-AraG uptake detected in the gut, we obtained colorectal tissue for in situ hybridization SARS-CoV-2 RNA and immunohistochemical studies in a subset of participants with Long COVID symptoms. We identified cellular SARS-CoV-2 RNA in rectosigmoid lamina propria tissue in all these participants, ranging from 158 to 676 days following initial COVID-19 illness, suggesting that tissue viral persistence could be associated with long-term immunological perturbations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.26.23284998">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.26.23284998" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.26.23284998">
        <p class="paperTitle">A molnupiravir-associated mutational signature in global SARS-CoV-2 genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.26.23284998" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.26.23284998" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanderson, T.; Hisner, R.; Donovan-Banfield, I.; Hartman, H.; Lochen, A.; Peacock, T. P.; Ruis, C.</p>
        <p class="info">Score: 2427.4, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.26.23284998' target='https://doi.org/10.1101/2023.01.26.23284998'> 10.1101/2023.01.26.23284998</a></p>
        <p class="abstract">Molnupiravir, an antiviral medication that has been widely used against SARS-CoV-2, acts by inducing mutations in the virus genome during replication. Most random mutations are likely to be deleterious to the virus, and many will be lethal, and so molnupiravir-induced elevated mutation rates reduce viral load2,3. However, if some patients treated with molnupiravir do not fully clear SARS-CoV-2 infections, there could be the potential for onward transmission of molnupiravir-mutated viruses. Here we show that SARS-CoV-2 sequencing databases contain extensive evidence of molnupiravir mutagenesis. Using a systematic approach, we find that a specific class of long phylogenetic branches, distinguished by a high proportion of G-to-A and C-to-T mutations, appear almost exclusively in sequences from 2022, after the introduction of molnupiravir treatment, and in countries and age-groups with widespread usage of the drug. We identify a mutational spectrum, with preferred nucleotide contexts, from viruses in patients known to have been treated with molnupiravir and show that its signature matches that seen in these long branches, in some cases with onwards transmission of molnupiravir-derived lineages. Finally, we analyse treatment records to confirm a direct association between these high G-to-A branches and the use of molnupiravir.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.23293434">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.23293434" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.23293434">
        <p class="paperTitle">Safety and Immunogenicity of XBB.1.5-Containing mRNA Vaccines</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.23293434" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.23293434" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chalkias, S.; McGhee, N.; Whatley, J. L.; Essink, B.; Brosz, A.; Tomassini, J. E.; Girard, B.; Edwards, D. K.; Nasir, A.; Lee, D.; Avena, L. E.; Feng, J.; Deng, W.; Montefiori, D. C.; Baden, L. R.; Miller, J. M.; Das, R.</p>
        <p class="info">Score: 47.9, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.23293434' target='https://doi.org/10.1101/2023.08.22.23293434'> 10.1101/2023.08.22.23293434</a></p>
        <p class="abstract">Background: Subvariants of the severe acute respiratory syndrome coronavirus (SARS-CoV-2) omicron XBB-lineage have the potential to escape immunity provided by prior infection or vaccination. For Covid-19 immunizations beginning in the Fall 2023, the U.S. FDA has recommended updating to a monovalent omicron XBB.1.5-containing vaccine. Methods: In this ongoing, phase 2/3 study participants were randomized 1:1 to receive 50-{micro}g doses of mRNA-1273.815 monovalent (50-{micro}g omicron XBB.1.5 spike mRNA) or mRNA-1273.231 bivalent (25-{micro}g omicron XBB.1.5 and 25-{micro}g omicron BA.4/BA.5 spike mRNAs)vaccines, administered as 5th doses, to adults who previously received a primary series and 3rd dose of an original mRNA coronavirus disease 2019 (Covid-19) vaccine, and a 4th dose of a bivalent (omicron BA.4/BA.5 and original SARS-CoV-2) vaccine. Interim safety and immunogenicity data 15 days post-vaccination are presented. Results: In April 2023, participants received mRNA-1273.815 (n=50) and mRNA-1273.231 (n=51). The median intervals from the prior dose of BA.4/BA.5-containing bivalent vaccine were 8.2 and 8.3 months for the mRNA-1273.815 and mRNA-1273.231 groups, respectively. Both vaccines increased neutralizing antibody (nAb) geometric mean titers against all variants tested at day 15 post-booster nAb compared to pre-booster levels. Geometric mean fold-rises from pre-booster titers after the monovalent booster were higher against XBB.1.5, XBB.1.16 and SARS-CoV-2 (D614G) than those of the bivalent booster and were comparable against BA.4/BA.5 and BQ1.1 variants for both vaccines. The monovalent vaccine also elicited nAb responses against omicron XBB.2.3.2, EG.5.1, and FL.1.5.1 that were similar to those seen for XBB.1.5 in a subset (n=20) of participants. The occurrence of solicited adverse reactions and unsolicited adverse events were overall similar to those previously reported for the original mRNA-1273 50-{micro}g and omicron BA.4/BA.5-containing bivalent mRNA-1273 vaccines. Conclusion: In this interim analysis, XBB.1.5-containing monovalent and bivalent vaccines elicited potent neutralizing responses against variants of the omicron XBB-lineage (XBB.1.5, XBB.1.6, XBB.2.3.2) as well as more recent (EG.5.1, FL.1.5.1) variants. The safety profile of the XBB.1.5-containing vaccine was consistent with those of prior vaccines. These results support the Covid-19 vaccine update in 2023-2024 to a monovalent XBB.1.5-containing vaccine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.23294416">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.23294416" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.23294416">
        <p class="paperTitle">SARS-CoV-2 shedding and evolution in immunocompromised hosts during the Omicron period: a multicenter prospective analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.23294416" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.23294416" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Raglow, Z.; Surie, D.; Chappell, J. D.; Zhu, Y.; Martin, E. T.; Kwon, J. H.; Frosch, A. E.; Mohamed, A.; Gilbert, J.; Bendall, E. E.; Bahr, A.; Halasa, N.; Talbot, H. K.; Grijalva, C. G.; Baughman, A.; Womack, K. N.; Johnson, C.; Swan, S. A.; Koumans, E.; McMorrow, M. L.; Harcourt, J. L.; Atherton, L. J.; Burroughs, A.; Thornburg, N. J.; Self, W. H.; Lauring, A. S.</p>
        <p class="info">Score: 24.1, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.23294416' target='https://doi.org/10.1101/2023.08.22.23294416'> 10.1101/2023.08.22.23294416</a></p>
        <p class="abstract">Background: Prolonged SARS-CoV-2 infections in immunocompromised hosts may predict or source the emergence of highly mutated variants. The types of immunosuppression placing patients at highest risk for prolonged infection and associated intrahost viral evolution remain unclear. Methods: Adults aged &gt;18 years were enrolled at 5 hospitals and followed from 4/11/2022-2/1/2023. Eligible patients were SARS-CoV-2 positive in the previous 14 days and had a moderate or severely immunocompromising condition or treatment. Nasal specimens were tested by rRT-PCR every 2-4 weeks until negative in consecutive specimens. Positive specimens underwent viral culture and whole genome sequencing. A Cox proportional hazards model was used to assess factors associated with duration of infection. Results: We enrolled 150 patients with: B cell malignancy or anti-B cell therapy (n=18), solid organ or hematopoietic stem cell transplant (SOT/HSCT) (n=59), AIDS (n=5), non-B cell malignancy (n=23), and autoimmune/autoinflammatory conditions (n=45). Thirty-eight (25%) were rRT-PCR positive and 12 (8%) were culture-positive [&amp;ge;]21 days after initial SARS-CoV-2 detection or illness onset. Patients with B cell dysfunction had longer duration of rRT-PCR positivity compared to those with autoimmune/autoinflammatory conditions (aHR 0.32, 95% CI 0.15-0.64). Consensus (&gt;50% frequency) spike mutations were identified in 5 individuals who were rRT-PCR positive &gt;56 days; 61% were in the receptor-binding domain (RBD). Mutations shared by multiple individuals were rare (&lt;5%) in global circulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.23293924">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.23293924" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.23293924">
        <p class="paperTitle">Published benefits of ivermectin use in Itajai, Brazil for COVID-19 infection, hospitalisation, and mortality are entirely explained by statistical artefacts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.23293924" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.23293924" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mills, R.; Pecanha Antonio, A. C.; Tucker-Kellogg, G.</p>
        <p class="info">Score: 286.6, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.23293924' target='https://doi.org/10.1101/2023.08.10.23293924'> 10.1101/2023.08.10.23293924</a></p>
        <p class="abstract">BackgroundTwo recent publications by Kerr et al. (Cureus 14(1):e21272; Cureus 14(8): e28624) reported dramatic effects of prophylactic ivermectin use for both prevention of COVID-19 and reduction of COVID-19-related hospitalisation and mortality, including a dose-dependent effect of ivermectin prophylaxis. These papers have gained an unusually large public influence: they were incorporated into debates around COVID-19 policies and may have contributed to decreased trust in vaccine efficacy and public health authorities more broadly. Both studies were based on retrospective observational analysis of city-wide registry data from the city of Itajai, Brazil from July-December 2020.

MethodsStarting with initially identified sources of error, we conducted a revised statistical analysis of available data, including data made available with the original papers and public data from the Brazil Ministry of Health. We identified additional uncorrected sources of bias and errors from the original analysis, including incorrect subject exclusion and missing subjects, an enrolment time bias, and multiple sources of immortal time bias. In models assuming no actual effect from ivermectin use, we conducted Monte Carlo simulations to estimate the contribution of these biases to any observed effect.

ResultsUntreated statistical artefacts and methodological errors alone lead to dramatic apparent risk reduction associated with Ivermectin use in both studies. The magnitude of apparent risk reduction from these artefacts is comparable to the results reported by the studies themselves, including apparent protection from infection, hospitalisation, and death, and including the reported apparent dose-response relationship.

ConclusionsThe inference of ivermectin efficacy reported in both papers is unsupported, as the observed effects are entirely explained by untreated statistical artefacts and methodological errors. Our re-analysis calls for caution in interpreting highly publicised observational studies and highlights the importance of common sources of bias in clinical research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.20.23294350">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.20.23294350" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.20.23294350">
        <p class="paperTitle">Heterogeneous SARS-CoV-2 kinetics due to variable timing and intensity of immune responses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.20.23294350" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.20.23294350" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Owens, K.; Esmaeili-Wellman, S.; Schiffer, J. T.</p>
        <p class="info">Score: 10.8, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.20.23294350' target='https://doi.org/10.1101/2023.08.20.23294350'> 10.1101/2023.08.20.23294350</a></p>
        <p class="abstract">The viral kinetics of documented SARS-CoV-2 infections exhibit a high degree of inter-individual variability. We identified six distinct viral shedding patterns, which differed according to peak viral load, duration, expansion rate and clearance rate, by clustering data from 810 infections in the National Basketball Association cohort. Omicron variant infections in previously vaccinated individuals generally led to lower cumulative shedding levels of SARS-CoV-2 than other scenarios. We then developed a mechanistic mathematical model that recapitulated 1510 observed viral trajectories, including viral rebound and cases of reinfection. Lower peak viral loads were explained by a more rapid and sustained transition of susceptible cells to a refractory state during infection, as well as an earlier and more potent late, cytolytic immune response. Our results suggest that viral elimination occurs more rapidly during omicron infection, following vaccination, and following re-infection due to enhanced innate and acquired immune responses. Because viral load has been linked with COVID-19 severity and transmission risk, our model provides a framework for understanding the wide range of observed SARS-CoV-2 infection outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.23293522">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.23293522" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.23293522">
        <p class="paperTitle">The post-COVID-19 population has a high prevalence of crossreactive antibodies to spikes from all Orthocoronavirinae genera</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.23293522" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.23293522" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Singh, G.; Abbad, A.; Kleiner, G.; Srivastava, K.; Gleason, C.; Carreno Quiroz, J. M.; Simon, V.; Krammer, F.</p>
        <p class="info">Score: 61.9, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.23293522' target='https://doi.org/10.1101/2023.08.01.23293522'> 10.1101/2023.08.01.23293522</a></p>
        <p class="abstract">The Orthocoronaviridae subfamily is large comprising four highly divergent genera. Four seasonal coronaviruses were circulating in humans prior to the coronavirus disease 2019 (COVID-19) pandemic. Infection with these viruses induced antibody responses that are relatively narrow with little cross-reactivity to spike proteins of other coronaviruses. Here, we report that infection with and vaccination against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) induces broadly crossreactive binding antibodies to spikes from a wide range of coronaviruses including members of the sarbecovirus subgenus, betacoronaviruses including Middle Eastern respiratory syndrome coronavirus (MERS CoV), and extending to alpha-, gamma- and delta-coronavirus spikes. These data show that the coronavirus spike antibody landscape in humans has profoundly been changed and broadened as a result of the SARS-CoV-2 pandemic. While we do not understand the functionality of these crossreactive antibodies, they may lead to enhanced resistance of the population to infection with newly emerging coronaviruses with pandemic potential.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.23294503">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.23294503" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.23294503">
        <p class="paperTitle">Protection conferred by COVID-19 vaccination, prior SARS-CoV-2 infection, or hybrid immunity against Omicron-associated severe outcomes among community-dwelling adults</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.23294503" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.23294503" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, N.; Nguyen, L.; Austin, P. C.; Brown, K. A.; Grewal, R.; Buchan, S. A.; Nasreen, S.; Gubbay, J.; Schwartz, K. L.; Tadrous, M.; Wilson, K.; Wilson, S. E.; Kwong, J. C.</p>
        <p class="info">Score: 5.7, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.23294503' target='https://doi.org/10.1101/2023.08.24.23294503'> 10.1101/2023.08.24.23294503</a></p>
        <p class="abstract">Introduction: We assessed protection conferred by COVID-19 vaccines and/or prior SARS-CoV-2 infection against Omicron-associated severe outcomes during successive sublineage-predominant periods. Methods: We used a test-negative design to estimate protection by vaccines and/or prior infection against hospitalization/death among community-dwelling, PCR-tested adults aged &gt;50 years in Ontario, Canada between January 2, 2022 and June 30, 2023. Multivariable logistic regression was used to estimate the relative change in the odds of hospitalization/death with each vaccine dose (2-5) and/or prior PCR-confirmed SARS-CoV-2 infection (compared with unvaccinated, uninfected subjects) up to 15 months since the last vaccination or infection. Results: We included 18,526 cases with Omicron-associated severe outcomes and 90,778 test-negative controls. Vaccine protection was high during BA.1/BA.2 predominance, but was generally &lt;50% during periods of BA.4/BA.5 and BQ/XBB predominance without boosters. A third/fourth dose transiently increased protection during BA.4/BA.5 predominance (third-dose, 6-month: 68%, 95%CI 63%-72%; fourth-dose, 6-month: 80%, 95%CI 77%-83%), but was lower and waned quickly during BQ/XBB predominance (third-dose, 6-month: 59%, 95%CI 48%-67%; 12-month: 49%, 95%CI 41%-56%; fourth-dose, 6-month: 62%, 95%CI 56%-68%, 12-months: 51%, 95%CI 41%-56%). Hybrid immunity conferred nearly 90% protection throughout BA.1/BA.2 and BA.4/BA.5 predominance, but was reduced during BQ/XBB predominance (third-dose, 6-month: 60%, 95%CI 36%-75%; fourth-dose, 6-month: 63%, 95%CI 42%-76%). Protection was restored with a fifth dose (bivalent; 6-month: 91%, 95%CI 79%-96%). Prior infection alone did not confer lasting protection. Conclusion: Protection from COVID-19 vaccines and/or prior SARS-CoV-2 infections against severe outcomes is reduced when immune-evasive variants/subvariants emerge and may also wane over time. Our findings support a variant-adapted booster vaccination strategy with periodic review.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.23294172">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.23294172" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.23294172">
        <p class="paperTitle">Cohort Profile: Prospective Cohort to Study the COVID-19 Immune Response in Retail Workers in Quebec, Canada (CISACOV)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.23294172" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.23294172" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Santerre, K.; Theriault, M.; Brousseau, N.; Rochette, S.; Pelletier, J.; Gilbert, C.; Masson, J.-F.; Baz, M.; Boudreau, D.; Trottier, S.</p>
        <p class="info">Score: 5.3, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.23294172' target='https://doi.org/10.1101/2023.08.18.23294172'> 10.1101/2023.08.18.23294172</a></p>
        <p class="abstract">PurposeRetail workers are an understudied occupational group that may have been at increased risk of contracting SARS-CoV-2 during the COVID-19 pandemic. Therefore, we set up a longitudinal cohort of participants working in this sector to better document the incidence of SARS-CoV-2 infection and the immune response to infection and/or vaccination in this group.

ParticipantsA total of 304 participants were recruited between April 20, 2021 and October 22, 2021. They were invited to attend three visits (each separated by [~]12 weeks) during which they provided blood samples and information on participant characteristics, COVID-19 symptoms, and vaccination. An extension phase of two additional visits was carried out between March 15th, 2022 and October 3rd, 2022 to document the impact of the Omicron variant among the 198 participants who were still eligible for recruitment. Participants were aged 18 to 75 and worked in grocery stores, hardware stores, bars or restaurants within the Quebec City metropolitan area (Canada). Findings to date: This article describes participants demographic, socioeconomic, behavioral, clinical and occupational characteristics, and their COVID-19 symptoms (where applicable). It also describes SARS-CoV-2 vaccination status and any SARS-CoV-2 diagnostic test (i.e., PCR or rapid antigen) performed from the beginning of the pandemic until the last visit.

Future plansThe incidence of SARS-CoV-2 infections will be assessed. The immune response (innate and acquired) to SARS-CoV-2 infection or vaccination will be studied using a variety of techniques, including reference and experimental enzyme-linked immunosorbent assays, microneutralization assays with live viruses, experimental pseudoneutralization with an angiotensin-converting enzyme 2-spike assay, peripheral blood mononuclear cells and neutrophil stimulation, and a proliferation assay based on carboxyfluorescein diacetate succinimidyl ester.

RegistrationNot applicable.

Strengths and limitationsO_LIThis cohort offers a comprehensive dataset to study the immune response to SARS-CoV-2 infection or vaccination (alone), or hybrid immunity, as participants provided information on a wide range of demographic, socioeconomic, behavioral, clinical, and occupational variables. However, the low proportion of racial minorities (i.e., 3.0%) limits the use of this cohort to study racial determinants of immunity to SARS-CoV-2.
C_LIO_LIThis cohort focuses on workers in the food and retail service sector, an understudied population at high risk of occupational exposure to infectious agents.
C_LIO_LIThis study covered seven pandemic waves and thus captured a large number of confirmed infections from different variants.
C_LIO_LISample collection was initiated immediately prior to COVID-19 vaccine availability for this population and thus captured successive vaccination campaigns over 17 months.
C_LIO_LIIn keeping with the study design, none of the participants had severe COVID-19 disease requiring hospitalization at baseline, and none of the COVID-19 illnesses that occurred during the study required hospitalization; this cohort may not, therefore, be used to study the immune response leading to severe health outcomes, but is appropriate to study the immune response to mild SARS-CoV-2 infections.
C_LIO_LINearly 1300 blood samples were collected; furthermore, only 13 out of 304 (4.3%) participants withdrew before attending all three initial visits, and 4 out of 198 (2.0%) participants who remained eligible in the extension phase withdrew before attending the fifth visit. A series of at least 5 blood samples drawn over 48 weeks is therefore available for most participants.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.23293441">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.23293441" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.23293441">
        <p class="paperTitle">SARS-CoV-2 Viral Clearance and Evolution Varies by Extent of Immunodeficiency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.23293441" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.23293441" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Y.; Choudhary, M. C.; Regan, J.; Boucau, J.; Nathan, A.; Speidel, T.; Liew, M. Y.; Edelstein, G. E.; Kawano, Y.; Uddin, R.; Deo, R.; Marino, C.; Getz, M. A.; Reynold, Z.; Barry, M.; Gilbert, R. F.; Tien, D.; Sagar, S.; Vyas, T. D.; Flynn, J. P.; Hammond, S. P.; Novack, L. A.; Choi, B.; Cernadas, M.; Wallace, Z. S.; Sparks, J. A.; Vyas, J. M.; Seaman, M. S.; Gaiha, G. D.; Siedner, M. J.; Barczak, A. K.; Lemieux, J. E.; Li, J. Z.</p>
        <p class="info">Score: 53.1, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.23293441' target='https://doi.org/10.1101/2023.07.31.23293441'> 10.1101/2023.07.31.23293441</a></p>
        <p class="abstract">Despite vaccination and antiviral therapies, immunocompromised individuals are at risk for prolonged SARS-CoV-2 infection, but the immune defects that predispose to persistent COVID-19 remain incompletely understood. In this study, we performed detailed viro-immunologic analyses of a prospective cohort of participants with COVID-19. The median time to nasal viral RNA and culture clearance in the severe hematologic malignancy/transplant group (S-HT) were 72 and 40 days, respectively, which were significantly longer than clearance rates in the severe autoimmune/B-cell deficient (S-A), non-severe, and non-immunocompromised groups (P&lt;0.001). Participants who were severely immunocompromised had greater SARS-CoV-2 evolution and a higher risk of developing antiviral treatment resistance. Both S-HT and S-A participants had diminished SARS-CoV-2-specific humoral, while only the S-HT group had reduced T cell-mediated responses. This highlights the varied risk of persistent COVID-19 across immunosuppressive conditions and suggests that suppression of both B and T cell responses results in the highest contributing risk of persistent infection.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
