<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Blood transcriptomics reveal persistent SARS-CoV-2 RNA and candidate biomarkers in Long COVID patients
Authors: MENEZES, S. M.; JAMOULLE, M.; Carletto, M. P.; Van Holm, B.; Moens, L.; Meyts, I.; Maes, P.; Van Weyenbergh, J.
Score: 265.1, Published: 2024-01-17 DOI: 10.1101/2024.01.14.24301293
With an estimated 65 million individuals suffering from Long COVID, validated therapeutic strategies as well as non-invasive biomarkers are direly needed to guide clinical management. We used blood digital transcriptomics in search of viral persistence and Long COVID diagnostic biomarkers in a real-world, general practice-based setting with a long clinical follow-up.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Blood transcriptomics reveal persistent SARS-CoV-2 RNA and candidate biomarkers in Long COVID patients
Authors: MENEZES, S. M.; JAMOULLE, M.; Carletto, M. P.; Van Holm, B.; Moens, L.; Meyts, I.; Maes, P.; Van Weyenbergh, J.
Score: 265.1, Published: 2024-01-17 DOI: 10.1101/2024.01.14.24301293
With an estimated 65 million individuals suffering from Long COVID, validated therapeutic strategies as well as non-invasive biomarkers are direly needed to guide clinical management. We used blood digital transcriptomics in search of viral persistence and Long COVID diagnostic biomarkers in a real-world, general practice-based setting with a long clinical follow-up." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-21T10:39:08+00:00" />
<meta property="article:modified_time" content="2024-01-21T10:39:08+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Blood transcriptomics reveal persistent SARS-CoV-2 RNA and candidate biomarkers in Long COVID patients
Authors: MENEZES, S. M.; JAMOULLE, M.; Carletto, M. P.; Van Holm, B.; Moens, L.; Meyts, I.; Maes, P.; Van Weyenbergh, J.
Score: 265.1, Published: 2024-01-17 DOI: 10.1101/2024.01.14.24301293
With an estimated 65 million individuals suffering from Long COVID, validated therapeutic strategies as well as non-invasive biomarkers are direly needed to guide clinical management. We used blood digital transcriptomics in search of viral persistence and Long COVID diagnostic biomarkers in a real-world, general practice-based setting with a long clinical follow-up."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Blood transcriptomics reveal persistent SARS-CoV-2 RNA and candidate biomarkers in Long COVID patients\nAuthors: MENEZES, S. M.; JAMOULLE, M.; Carletto, M. P.; Van Holm, B.; Moens, L.; Meyts, I.; Maes, P.; Van Weyenbergh, J.\nScore: 265.1, Published: 2024-01-17 DOI: 10.1101/2024.01.14.24301293\nWith an estimated 65 million individuals suffering from Long COVID, validated therapeutic strategies as well as non-invasive biomarkers are direly needed to guide clinical management. We used blood digital transcriptomics in search of viral persistence and Long COVID diagnostic biomarkers in a real-world, general practice-based setting with a long clinical follow-up.",
  "keywords": [
    
  ],
  "articleBody": " Blood transcriptomics reveal persistent SARS-CoV-2 RNA and candidate biomarkers in Long COVID patients\nAuthors: MENEZES, S. M.; JAMOULLE, M.; Carletto, M. P.; Van Holm, B.; Moens, L.; Meyts, I.; Maes, P.; Van Weyenbergh, J.\nScore: 265.1, Published: 2024-01-17 DOI: 10.1101/2024.01.14.24301293\nWith an estimated 65 million individuals suffering from Long COVID, validated therapeutic strategies as well as non-invasive biomarkers are direly needed to guide clinical management. We used blood digital transcriptomics in search of viral persistence and Long COVID diagnostic biomarkers in a real-world, general practice-based setting with a long clinical follow-up. We demonstrate systemic SARS-CoV-2 persistence for more than 2 years after acute COVID-19 infection. A 2-gene biomarker, including FYN and SARS-CoV-2 antisense RNA, correctly classifies Long COVID with 93.8% sensitivity and 91.7% specificity. Specific immune transcripts and immunometabolism score correlate to systemic viral load and patient-reported anxiety/depression, providing mechanistic links as well as therapeutic targets to tackle Long COVID.\nAltered IgG4 Antibody Response to Repeated mRNA versus Protein COVID Vaccines\nAuthors: Kalkeri, R.; Zhu, M.; Cloney-Clark, S.; Plested, J. S.; Parekh, A.; Gorinson, D.; Cai, R.; Mahato, S.; Ramanathan, P.; Aurelia, L. C.; John Selva, K.; Marchese, A. M.; Fries, L.; Chung, A. W.; Dunkle, L. M.\nScore: 84.2, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301374\nRepeated mRNA SARS-CoV-2 vaccination has been associated with increases in the proportion of IgG4 in spike-specific antibody responses and concurrent reductions in Fc{gamma}-mediated effector functions that may limit control of viral infection. Here, we assessed anti-Spike total IgG, IgG1, IgG2, IgG3 and IgG4, and surrogate markers for antibody-dependent cellular phagocytosis (ADCP, Fc{gamma}RIIa binding), antibody-dependent cellular cytotoxicity (ADCC, Fc{gamma}RIIIa binding), and antibody-dependent complement deposition (ADCD, C1q binding) associated with repeated SARS-CoV-2 vaccination with NVX-CoV2373 (Novavax Inc., Gaithersburg, MD). The NVX-CoV2373 protein vaccine did not induce notable increases in spike-specific IgG4 or negatively impact surrogates for Fc{gamma} effector responses. Conversely, repeated NVX-CoV2373 vaccination uniquely enhanced IgG3 responses which are known to exhibit strong affinity for Fc{gamma}RIIIa and have previously been linked to potent neutralization of SARS-CoV-2. Subsequent investigations will help to understand the immunological diversity generated by different SARS-CoV-2 vaccine types and have the potential to reshape public health strategies.\nImpact of COVID-19 vaccination on symptoms and immune phenotypes in vaccine-naIÌˆve individuals with Long COVID\nAuthors: Grady, C. B.; Bhattacharjee, B.; Silva, J.; Jaycox, J.; Lee, L. W.; Monteiro, V. S.; Sawano, M.; Massey, D.; Caraballo, C.; Gehlhausen, J.; Tabachnikova, A.; Mao, T.; Lucas, C.; Pena-Hernandez, M. A.; Xu, L.; Tzeng, T. J.; Takahashi, T.; Herrin, J.; Guthe, D. B.; Akrami, A.; Assaf, G.; Davis, H.; Harris, K.; McCorkell, L.; Schulz, W.; Griffin, D.; Wei, H.; Ring, A. M.; Guan, L.; Dela Cruz, C.; Iwasaki, A.; Krumholz, H.\nScore: 63.2, Published: 2024-01-12 DOI: 10.1101/2024.01.11.24300929\nBackgroundLong COVID contributes to the global burden of disease. Proposed root cause hypotheses include the persistence of SARS-CoV-2 viral reservoir, autoimmunity, and reactivation of latent herpesviruses. Patients have reported various changes in Long COVID symptoms after COVID-19 vaccinations, leaving uncertainty about whether vaccine-induced immune responses may alleviate or worsen disease pathology. MethodsIn this prospective study, we evaluated changes in symptoms and immune responses after COVID-19 vaccination in 16 vaccine-naive individuals with Long COVID. Surveys were administered before vaccination and then at 2, 6, and 12 weeks after receiving the first vaccine dose of the primary series. Simultaneously, SARS-CoV-2-reactive TCR enrichment, SARS-CoV-2-specific antibody responses, antibody responses to other viral and self-antigens, and circulating cytokines were quantified before vaccination and at 6 and 12 weeks after vaccination. ResultsSelf-report at 12 weeks post-vaccination indicated 10 out of 16 participants had improved health, 3 had no change, 1 had worse health, and 2 reported marginal changes. Significant elevation in SARS-CoV-2-specific TCRs and Spike protein-specific IgG were observed 6 and 12 weeks after vaccination. No changes in reactivities were observed against herpes viruses and self-antigens. Within this dataset, higher baseline sIL-6R was associated with symptom improvement, and the two top features associated with non-improvement were high IFN-{beta} and CNTF, among soluble analytes. ConclusionsOur study showed that in this small sample, vaccination improved the health or resulted in no change to the health of most participants, though few experienced worsening. Vaccination was associated with increased SARS-CoV-2 Spike protein-specific IgG and T cell expansion in most individuals with Long COVID. Symptom improvement was observed in those with baseline elevated sIL-6R, while elevated interferon and neuropeptide levels were associated with a lack of improvement. Plain language summaryThe impact of the COVID-19 vaccine on vaccine-naive individuals suffering from Long COVID is uncertain. This study assessed the experience and immune signatures of 16 unvaccinated participants with Long COVID. A total of 10 participants had improved health status after vaccination, and one person reported only worsening health. As expected, vaccination increased immune cells and antibodies against the viral spike protein. Immune signatures may prove to be predictors of health status after vaccination. However, given the small number of participants, these initial findings need further validation.\nA high-resolution genomic and phenotypic analysis of resistance evolution of an Escherichia coli strain from a critical care patient treated with piperacillin/tazobactam.\nAuthors: Fraser, A. J.; Ball, R.; Cantillon, D.; Brettell, L.; Graf, F. E.; Lewis, J. M.; van Aartsen, J. J.; Parry, C. M.; Heinz, E.; Edwards, T.\nScore: 6.7, Published: 2024-01-16 DOI: 10.1101/2024.01.15.24301199\nResistance to the {beta}-lactam/{beta}-lactamase inhibitor (BL/BLI) combination antibiotic piperacillin/tazobactam (TZP) predominantly occurs via {beta}-lactamase enzymes also leading to resistance to third-generation cephalosporins (3GCs). However, if {beta}-lactamases inactive against 3GCs and inhibited by tazobactam are expressed at high levels leading to enzyme hyperproduction, the surplus enzyme escapes inhibition by tazobactam and inactivates the antibiotic piperacillin. Understanding this mechanism is clinically relevant as enzyme hyperproduction can emerge upon antibiotic administration, resulting in treatment failure despite initial resistance profiles supporting TZP use. We report the identification of an Escherichia coli isolate that developed resistance to TZP during patient treatment. Our whole genome sequencing (WGS) analyses show that TZP resistance evolved via IS26-mediated duplication of a blaTEM-1 containing gene cassette on a plasmid, resulting in hyperproduction of TEM-1 {beta}-lactamase. We demonstrate that ten copies of blaTEM-1 induce resistance greater than 32-times the MIC and exposure to TZP further increases amplification of blaTEM-1. Furthermore, in the absence of TZP, gene copy number of IS26 and blaTEM-1 remains stable over five days, despite a 48,205 bp genome size increase compared to the pre-amplification isolate. We additionally detect phenotypic changes that might indicate host adaptation potentially linked to the additional genes in the amplified cassette. Our analysis advances the understanding of infections caused by isolates evolving {beta}-lactamase hyperproduction, which represent a complex problem in both detection and treatment. As 40% of antibiotics active against WHO priority pathogens in the pre-clinical pipeline are BL/BLI combinations further investigations are of urgent concern. ImportanceWe investigated an Escherichia coli strain obtained from the bloodstream of a hospitalised patient, that evolved resistance against the antimicrobials initially used as empirical treatment. Comparing the whole-genome sequences of the susceptible isolate with the evolved, resistant isolate showed duplications of the only encoded {beta}-lactamase gene, blaTEM-1, resulting in increased enzyme production and resistance to TZP, a commonly prescribed BL/BLI combination antimicrobial. Despite the additional energy needed for increased enzyme production and retaining the additional copies of duplicated genes, we did not find growth differences under standard laboratory conditions. We furthermore identify phenotypic changes that indicate host adaptation and mirror phenotypic changes observed in other species of opportunistic bacterial pathogens. Our findings highlight that BL/BLI combinations can lead to rapid within-patient evolution of antimicrobial resistance, which is of high relevance when considering the implementation of newly developed drugs, many of which belong to the BL/BLI class.\nTransient anti-interferon autoantibodies in the airways are associated with efficient recovery from COVID-19\nAuthors: Babcock, B. R.; Kosters, A.; Eddins, D. J.; Donaire, M. S. B.; Sarvadhavabhatla, S.; Pae, V.; Beltran, F.; Murray, V. W.; Gill, G.; Xie, G.; Dobosh, B. S.; Giacalone, V. D.; Tirouvanziam, R. M.; Ramonell, R. P.; Jenks, S. A.; Sanz, I.; Lee, F. E.-H.; Roan, N. R.; Lee, S. A.; Ghosn, E. E. B.\nScore: 19.0, Published: 2024-01-11 DOI: 10.1101/2024.01.11.24301000\nPre-existing anti-interferon alpha (anti-IFN-) autoantibodies in blood are associated with susceptibility to life-threatening COVID-19. However, it is unclear whether anti-IFN- autoantibodies in the airways - the initial site of infection - can also determine disease outcomes. In this study, we developed a new multiparameter technology, flowBEAT, to quantify and profile the isotypes of anti-IFN- and anti-SARS-CoV-2 antibodies in longitudinal samples collected over 20 months from the airway and matching blood of 129 donors with mild, moderate, and severe COVID-19. We found unexpectedly that nasal anti-IFN- autoantibodies were induced post-infection onset in more than 70% of mild to moderate COVID-19 cases and associated with robust anti-SARS-CoV-2 immunity, fewer symptoms, and efficient recovery. Nasal anti-IFN- autoantibodies followed the peak of host IFN- production and waned with disease recovery, revealing a regulated balance between IFN- and anti-IFN- response. Notably, only a subset of mild to moderate patients progressed to develop systemic anti-IFN-, which correlated with systemic inflammation and worsened symptoms. In contrast, patients with life-threatening COVID-19 sustained elevated anti-IFN- in both airways and blood, coupled with uncontrolled viral load and IFN- production. Our studies thereby reveal a novel protective role for nasal anti-IFN- autoantibodies in the immunopathology of COVID-19 and, more broadly, suggest that anti-IFN- may serve an important regulatory function to restore homeostasis following viral invasion of the respiratory mucosa.\nRisk Factors for Pediatric Critical COVID-19: A Systematic Review and Meta-Analysis\nAuthors: Aparicio, C.; Willis, Z. I.; Nakamura, M. M.; Wolf, J.; Little, C.; Maron, G. M.; Sue, P. K.; Anosike, B. I.; Miller, C.; Bio, L. L.; Singh, P.; James, S. H.; Oliveira, C. R.; PIDS Pediatric COVID-19 Therapies Task Force, Score: 2.9, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301452\nBackground: Risk stratification is a cornerstone of the Pediatric Infectious Diseases Society COVID-19 treatment guidance. This systematic review and meta-analysis aimed to define the clinical characteristics and comorbidities associated with critical COVID-19 in children and adolescents. Methods: Two independent reviewers screened the literature (Medline and EMBASE) for studies published through August 2023 that reported outcome data on patients aged 21 years or younger with COVID-19. Critical disease was defined as an invasive mechanical ventilation requirement, intensive care unit admission, or death. Random effects models were used to estimate pooled odds ratios (OR) with 95% confidence intervals (CI), and heterogeneity was explored through subgroup analyses. Results: Among 10,178 articles, 136 studies met the inclusion criteria for review. Data from 70 studies, which collectively examined 172,165 children and adolescents with COVID-19, were pooled for meta-analysis. In previously healthy children, the absolute risk of critical disease from COVID-19 was 4% (95% CI, 1%-10%). Compared with no comorbidities, the pooled OR for critical disease was 3.95 (95% CI, 2.78-5.63) for presence of one comorbidity and 9.51 (95% CI, 5.62-16.06) for [\u0026ge;]2 comorbidities. Key risk factors included cardiovascular and neurological disorders, chronic pulmonary conditions (excluding asthma), diabetes, obesity, and immunocompromise, all with statistically significant ORs \u003e2.00. Conclusions: While the absolute risk for critical COVID-19 in children and adolescents without underlying health conditions is relatively low, the presence of one or more comorbidities was associated with markedly increased risk. These findings support the importance of risk stratification in tailoring pediatric COVID-19 management.\nImmunological imprinting shapes the specificity of human antibody responses against SARS-CoV-2 variants\nAuthors: Johnston, T. S.; Li, S. H.; Painter, M. M.; Atkinson, R. K.; Douek, N. R.; Reeg, D. B.; Doueck, D. C.; Wherry, E. J.; Hensley, S. E.\nScore: 82.4, Published: 2024-01-09 DOI: 10.1101/2024.01.08.24301002\nThe spike glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) continues to accumulate substitutions, leading to breakthrough infections of vaccinated individuals and prompting the development of updated booster vaccines. Here, we determined the specificity and functionality of antibody and B cell responses following exposure to BA.5 and XBB variants in individuals who received ancestral SARS-CoV-2 mRNA vaccines. BA.5 exposures elicited antibody responses that primarily targeted epitopes conserved between the BA.5 and ancestral spike, with poor reactivity to the XBB.1.5 variant. XBB exposures also elicited antibody responses that targeted epitopes conserved between the XBB.1.5 and ancestral spike. However, unlike BA.5, a single XBB exposure elicited low levels of XBB.1.5-specific antibodies and B cells in some individuals. Pre-existing cross-reactive B cells and antibodies were correlated with stronger overall responses to XBB but weaker XBB-specific responses, suggesting that baseline immunity influences the activation of variant-specific SARS-CoV-2 responses. HighlightsO_LIVariant breakthrough infections boost ancestral cross-reactive antibodies and B cells C_LIO_LIFirst and second BA.5 exposures fail to elicit variant-specific antibodies and B cells C_LIO_LIXBB infections and monovalent vaccinations elicit XBB.1.5-specific responses in some individuals C_LIO_LIXBB.1.5-specific responses correlate with low levels of pre-existing humoral immunity C_LI\nA putative hepatitis B virus sequence motif associated with hepatocellular carcinoma in South African adults\nAuthors: Maponga, T. G.; McNaughton, A. L.; Campbell, C.; DeCesare, M.; Mokaya, J.; Lumley, S. F.; Bonsall, D.; Ip, C. L.; Chai, H.; Van Rensburg, C.; Glashoff, R.; Waddilove, E.; Preiser, W.; Blackard, J. T.; Ansari, M. A.; Kramvis, A.; Andersson, M. I.; Matthews, P. C.\nScore: 1.0, Published: 2024-01-15 DOI: 10.1101/2024.01.13.24301263\nAimChronic hepatitis B virus (HBV) infection is a major risk factor for hepatocellular carcinoma (HCC) particularly in African populations, in whom malignancy frequently presents at an advanced stage with poor outcomes. We derived HBV whole genome sequences (WGS) from individuals with HCC and compared them to sequences from individuals without HCC. Methods: We identified adults with HBV infection, with and without complicating HCC, in Cape Town, South Africa and utilized pan-genotypic probe-based enrichment followed by Illumina sequencing to derive HBV WGS. ResultsCompared to the non-HCC group, HCC patients were more likely to be male (p \u003c 0.0001), older (p = 0.01), HIV-negative (p = 0.006), and to have higher HBV viral loads (p \u003c 0.0001). Among 19 HCC and 12 non-HCC patients, genotype A dominated (74%), of which 96% were subtype A1. PreS2 deletions ({Delta}38-55) were enriched in HBV sequences from HCC patients (n = 7). The sequence motif most strongly associated with HCC comprised either deletion or polymorphism at site T53 in PreS2 - collectively coined non-T53 - together with a basal core promoter (BCP) mutation G1764A (AUROC 0.79). ConclusionsIn this setting, HBV sequence polymorphisms and deletions are associated with HCC, and non-T53 + G1764A represents a putative signature motif for HCC. Additional investigations are needed to disaggregate the impact of age, sex and HIV status, to ascertain the extent to which viral polymorphisms contribute to oncogenesis, and to determine whether HBV sequence is a useful biomarker for risk stratification.\nClimate warming is expanding dengue burden in the Americas and Asia\nAuthors: Childs, M. L.; Lyberger, K.; Harris, M.; Burke, M.; Mordecai, E. A.\nScore: 1.5, Published: 2024-01-09 DOI: 10.1101/2024.01.08.24301015\nClimate change poses significant threats to public health, with dengue representing a growing concern due to its high existing burden and sensitivity to climatic conditions. Yet, the quantitative impacts of temperature warming on dengue, both in the past and in the future, remain poorly understood. In this study, we quantify how dengue responds to climatic fluctuations, and use this inferred temperature response to estimate the impacts of historical warming and forecast trends under future climate change scenarios. To estimate the causal impact of temperature on the spread of dengue in the Americas and Asia, we assembled a dataset encompassing nearly 1.5 million dengue incidence records from 21 countries. Our analysis revealed a nonlinear relationship between temperature and dengue incidence with the largest marginal effects at lower temperatures (around 15{degrees}C), peak incidence at 27.8{degrees}C (95% CI: 27.3 - 28.2{degrees}C), and subsequent declines at higher temperatures. Our findings indicate that historical climate change has already increased dengue incidence 18% (12 - 25%) in the study region, and projections suggest a potential increase of 40% (17 - 76) to 57% (33 - 107%) by mid-century depending on the climate scenario, with some areas seeing up to 200% increases. Notably, our models suggest that lower emissions scenarios would substantially reduce the warming-driven increase in dengue burden. Together, these insights contribute to the broader understanding of how long-term climate patterns influence dengue, providing a valuable foundation for public health planning and the development of strategies to mitigate future risks due to climate change.\nAir cleaners and respiratory infections in schools: A modeling study using epidemiological, environmental, and molecular data\nAuthors: Banholzer, N.; Jent, P.; Bittel, P.; Zurcher, K.; Furrer, L.; Bertschinger, S.; Weingartner, E.; Ramette, A.; Egger, M.; Hascher, T.; Fenner, L.\nScore: 2.5, Published: 2023-12-30 DOI: 10.1101/2023.12.29.23300635\nBackgroundUsing a multiple-measurement approach, we examined the real-world effectiveness of portable HEPA-air filtration devices (air cleaners) in a school setting. MethodsWe collected environmental (CO2, particle concentrations), epidemiological (absences related to respiratory infections), audio (coughing), and molecular data (bioaerosol and saliva samples) over seven weeks during winter 2022/2023 in two Swiss secondary school classes. Using a cross-over study design, we compared particle concentrations, coughing, and the risk of infection with vs without air cleaners. ResultsAll 38 students (age 13-15 years) participated. With air cleaners, mean particle con-centration decreased by 77% (95% credible interval 63%-86%). There were no differences in CO2 levels. Absences related to respiratory infections were 22 without vs 13 with air cleaners. Bayesian modeling suggested a reduced risk of infection, with a posterior probability of 91% and a relative risk of 0.73 (95% credible interval 0.44-1.18). Coughing also tended to be less frequent (posterior probability 93%). Molecular analysis detected mainly non-SARS-CoV-2 viruses in saliva (50/448 positive), but not in bioaerosols (2/105 positive) or HEPA-filters (4/160). The detection rate was similar with vs without air cleaners. Spatiotemporal analysis of positive saliva samples identified several likely transmissions. ConclusionsAir cleaners improved air quality, showed a potential benefit in reducing respiratory infections, and were associated with less coughing. Airborne detection of non-SARS-CoV-2 viruses was rare, suggesting that these viruses may be more difficult to detect in the air. Future studies should examine the importance of close contact and long-range transmission, and the cost-effectiveness of using air cleaners.\n",
  "wordCount" : "2961",
  "inLanguage": "en",
  "datePublished": "2024-01-21T10:39:08Z",
  "dateModified": "2024-01-21T10:39:08Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 21, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.24301293">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.24301293" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.24301293">
        <p class="paperTitle">Blood transcriptomics reveal persistent SARS-CoV-2 RNA and candidate biomarkers in Long COVID patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.24301293" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.24301293" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: MENEZES, S. M.; JAMOULLE, M.; Carletto, M. P.; Van Holm, B.; Moens, L.; Meyts, I.; Maes, P.; Van Weyenbergh, J.</p>
        <p class="info">Score: 265.1, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.24301293' target='https://doi.org/10.1101/2024.01.14.24301293'> 10.1101/2024.01.14.24301293</a></p>
        <p class="abstract">With an estimated 65 million individuals suffering from Long COVID, validated therapeutic strategies as well as non-invasive biomarkers are direly needed to guide clinical management. We used blood digital transcriptomics in search of viral persistence and Long COVID diagnostic biomarkers in a real-world, general practice-based setting with a long clinical follow-up. We demonstrate systemic SARS-CoV-2 persistence for more than 2 years after acute COVID-19 infection. A 2-gene biomarker, including FYN and SARS-CoV-2 antisense RNA, correctly classifies Long COVID with 93.8% sensitivity and 91.7% specificity. Specific immune transcripts and immunometabolism score correlate to systemic viral load and patient-reported anxiety/depression, providing mechanistic links as well as therapeutic targets to tackle Long COVID.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.24301374">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.24301374" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.24301374">
        <p class="paperTitle">Altered IgG4 Antibody Response to Repeated mRNA versus Protein COVID Vaccines</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.24301374" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.24301374" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kalkeri, R.; Zhu, M.; Cloney-Clark, S.; Plested, J. S.; Parekh, A.; Gorinson, D.; Cai, R.; Mahato, S.; Ramanathan, P.; Aurelia, L. C.; John Selva, K.; Marchese, A. M.; Fries, L.; Chung, A. W.; Dunkle, L. M.</p>
        <p class="info">Score: 84.2, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.24301374' target='https://doi.org/10.1101/2024.01.17.24301374'> 10.1101/2024.01.17.24301374</a></p>
        <p class="abstract">Repeated mRNA SARS-CoV-2 vaccination has been associated with increases in the proportion of IgG4 in spike-specific antibody responses and concurrent reductions in Fc{gamma}-mediated effector functions that may limit control of viral infection. Here, we assessed anti-Spike total IgG, IgG1, IgG2, IgG3 and IgG4, and surrogate markers for antibody-dependent cellular phagocytosis (ADCP, Fc{gamma}RIIa binding), antibody-dependent cellular cytotoxicity (ADCC, Fc{gamma}RIIIa binding), and antibody-dependent complement deposition (ADCD, C1q binding) associated with repeated SARS-CoV-2 vaccination with NVX-CoV2373 (Novavax Inc., Gaithersburg, MD). The NVX-CoV2373 protein vaccine did not induce notable increases in spike-specific IgG4 or negatively impact surrogates for Fc{gamma} effector responses. Conversely, repeated NVX-CoV2373 vaccination uniquely enhanced IgG3 responses which are known to exhibit strong affinity for Fc{gamma}RIIIa and have previously been linked to potent neutralization of SARS-CoV-2. Subsequent investigations will help to understand the immunological diversity generated by different SARS-CoV-2 vaccine types and have the potential to reshape public health strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.24300929">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.24300929" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.24300929">
        <p class="paperTitle">Impact of COVID-19 vaccination on symptoms and immune phenotypes in vaccine-naIÌˆve individuals with Long COVID</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.24300929" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.24300929" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grady, C. B.; Bhattacharjee, B.; Silva, J.; Jaycox, J.; Lee, L. W.; Monteiro, V. S.; Sawano, M.; Massey, D.; Caraballo, C.; Gehlhausen, J.; Tabachnikova, A.; Mao, T.; Lucas, C.; Pena-Hernandez, M. A.; Xu, L.; Tzeng, T. J.; Takahashi, T.; Herrin, J.; Guthe, D. B.; Akrami, A.; Assaf, G.; Davis, H.; Harris, K.; McCorkell, L.; Schulz, W.; Griffin, D.; Wei, H.; Ring, A. M.; Guan, L.; Dela Cruz, C.; Iwasaki, A.; Krumholz, H.</p>
        <p class="info">Score: 63.2, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.24300929' target='https://doi.org/10.1101/2024.01.11.24300929'> 10.1101/2024.01.11.24300929</a></p>
        <p class="abstract">BackgroundLong COVID contributes to the global burden of disease. Proposed root cause hypotheses include the persistence of SARS-CoV-2 viral reservoir, autoimmunity, and reactivation of latent herpesviruses. Patients have reported various changes in Long COVID symptoms after COVID-19 vaccinations, leaving uncertainty about whether vaccine-induced immune responses may alleviate or worsen disease pathology.

MethodsIn this prospective study, we evaluated changes in symptoms and immune responses after COVID-19 vaccination in 16 vaccine-naive individuals with Long COVID. Surveys were administered before vaccination and then at 2, 6, and 12 weeks after receiving the first vaccine dose of the primary series. Simultaneously, SARS-CoV-2-reactive TCR enrichment, SARS-CoV-2-specific antibody responses, antibody responses to other viral and self-antigens, and circulating cytokines were quantified before vaccination and at 6 and 12 weeks after vaccination.

ResultsSelf-report at 12 weeks post-vaccination indicated 10 out of 16 participants had improved health, 3 had no change, 1 had worse health, and 2 reported marginal changes. Significant elevation in SARS-CoV-2-specific TCRs and Spike protein-specific IgG were observed 6 and 12 weeks after vaccination. No changes in reactivities were observed against herpes viruses and self-antigens. Within this dataset, higher baseline sIL-6R was associated with symptom improvement, and the two top features associated with non-improvement were high IFN-{beta} and CNTF, among soluble analytes.

ConclusionsOur study showed that in this small sample, vaccination improved the health or resulted in no change to the health of most participants, though few experienced worsening. Vaccination was associated with increased SARS-CoV-2 Spike protein-specific IgG and T cell expansion in most individuals with Long COVID. Symptom improvement was observed in those with baseline elevated sIL-6R, while elevated interferon and neuropeptide levels were associated with a lack of improvement.

Plain language summaryThe impact of the COVID-19 vaccine on vaccine-naive individuals suffering from Long COVID is uncertain. This study assessed the experience and immune signatures of 16 unvaccinated participants with Long COVID. A total of 10 participants had improved health status after vaccination, and one person reported only worsening health. As expected, vaccination increased immune cells and antibodies against the viral spike protein. Immune signatures may prove to be predictors of health status after vaccination. However, given the small number of participants, these initial findings need further validation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.24301199">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.24301199" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.24301199">
        <p class="paperTitle">A high-resolution genomic and phenotypic analysis of resistance evolution of an Escherichia coli strain from a critical care patient treated with piperacillin/tazobactam.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.24301199" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.24301199" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fraser, A. J.; Ball, R.; Cantillon, D.; Brettell, L.; Graf, F. E.; Lewis, J. M.; van Aartsen, J. J.; Parry, C. M.; Heinz, E.; Edwards, T.</p>
        <p class="info">Score: 6.7, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.24301199' target='https://doi.org/10.1101/2024.01.15.24301199'> 10.1101/2024.01.15.24301199</a></p>
        <p class="abstract">Resistance to the {beta}-lactam/{beta}-lactamase inhibitor (BL/BLI) combination antibiotic piperacillin/tazobactam (TZP) predominantly occurs via {beta}-lactamase enzymes also leading to resistance to third-generation cephalosporins (3GCs). However, if {beta}-lactamases inactive against 3GCs and inhibited by tazobactam are expressed at high levels leading to enzyme hyperproduction, the surplus enzyme escapes inhibition by tazobactam and inactivates the antibiotic piperacillin. Understanding this mechanism is clinically relevant as enzyme hyperproduction can emerge upon antibiotic administration, resulting in treatment failure despite initial resistance profiles supporting TZP use.

We report the identification of an Escherichia coli isolate that developed resistance to TZP during patient treatment. Our whole genome sequencing (WGS) analyses show that TZP resistance evolved via IS26-mediated duplication of a blaTEM-1 containing gene cassette on a plasmid, resulting in hyperproduction of TEM-1 {beta}-lactamase. We demonstrate that ten copies of blaTEM-1 induce resistance greater than 32-times the MIC and exposure to TZP further increases amplification of blaTEM-1. Furthermore, in the absence of TZP, gene copy number of IS26 and blaTEM-1 remains stable over five days, despite a 48,205 bp genome size increase compared to the pre-amplification isolate. We additionally detect phenotypic changes that might indicate host adaptation potentially linked to the additional genes in the amplified cassette.

Our analysis advances the understanding of infections caused by isolates evolving {beta}-lactamase hyperproduction, which represent a complex problem in both detection and treatment. As 40% of antibiotics active against WHO priority pathogens in the pre-clinical pipeline are BL/BLI combinations further investigations are of urgent concern.

ImportanceWe investigated an Escherichia coli strain obtained from the bloodstream of a hospitalised patient, that evolved resistance against the antimicrobials initially used as empirical treatment. Comparing the whole-genome sequences of the susceptible isolate with the evolved, resistant isolate showed duplications of the only encoded {beta}-lactamase gene, blaTEM-1, resulting in increased enzyme production and resistance to TZP, a commonly prescribed BL/BLI combination antimicrobial. Despite the additional energy needed for increased enzyme production and retaining the additional copies of duplicated genes, we did not find growth differences under standard laboratory conditions. We furthermore identify phenotypic changes that indicate host adaptation and mirror phenotypic changes observed in other species of opportunistic bacterial pathogens. Our findings highlight that BL/BLI combinations can lead to rapid within-patient evolution of antimicrobial resistance, which is of high relevance when considering the implementation of newly developed drugs, many of which belong to the BL/BLI class.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.24301000">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.24301000" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.24301000">
        <p class="paperTitle">Transient anti-interferon autoantibodies in the airways are associated with efficient recovery from COVID-19</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.24301000" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.24301000" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Babcock, B. R.; Kosters, A.; Eddins, D. J.; Donaire, M. S. B.; Sarvadhavabhatla, S.; Pae, V.; Beltran, F.; Murray, V. W.; Gill, G.; Xie, G.; Dobosh, B. S.; Giacalone, V. D.; Tirouvanziam, R. M.; Ramonell, R. P.; Jenks, S. A.; Sanz, I.; Lee, F. E.-H.; Roan, N. R.; Lee, S. A.; Ghosn, E. E. B.</p>
        <p class="info">Score: 19.0, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.24301000' target='https://doi.org/10.1101/2024.01.11.24301000'> 10.1101/2024.01.11.24301000</a></p>
        <p class="abstract">Pre-existing anti-interferon alpha (anti-IFN-) autoantibodies in blood are associated with susceptibility to life-threatening COVID-19. However, it is unclear whether anti-IFN- autoantibodies in the airways - the initial site of infection - can also determine disease outcomes. In this study, we developed a new multiparameter technology, flowBEAT, to quantify and profile the isotypes of anti-IFN- and anti-SARS-CoV-2 antibodies in longitudinal samples collected over 20 months from the airway and matching blood of 129 donors with mild, moderate, and severe COVID-19. We found unexpectedly that nasal anti-IFN- autoantibodies were induced post-infection onset in more than 70% of mild to moderate COVID-19 cases and associated with robust anti-SARS-CoV-2 immunity, fewer symptoms, and efficient recovery. Nasal anti-IFN- autoantibodies followed the peak of host IFN- production and waned with disease recovery, revealing a regulated balance between IFN- and anti-IFN- response. Notably, only a subset of mild to moderate patients progressed to develop systemic anti-IFN-, which correlated with systemic inflammation and worsened symptoms. In contrast, patients with life-threatening COVID-19 sustained elevated anti-IFN- in both airways and blood, coupled with uncontrolled viral load and IFN- production. Our studies thereby reveal a novel protective role for nasal anti-IFN- autoantibodies in the immunopathology of COVID-19 and, more broadly, suggest that anti-IFN- may serve an important regulatory function to restore homeostasis following viral invasion of the respiratory mucosa.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.24301452">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.24301452" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.24301452">
        <p class="paperTitle">Risk Factors for Pediatric Critical COVID-19: A Systematic Review and Meta-Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.24301452" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.24301452" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aparicio, C.; Willis, Z. I.; Nakamura, M. M.; Wolf, J.; Little, C.; Maron, G. M.; Sue, P. K.; Anosike, B. I.; Miller, C.; Bio, L. L.; Singh, P.; James, S. H.; Oliveira, C. R.; PIDS Pediatric COVID-19 Therapies Task Force,  </p>
        <p class="info">Score: 2.9, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.24301452' target='https://doi.org/10.1101/2024.01.17.24301452'> 10.1101/2024.01.17.24301452</a></p>
        <p class="abstract">Background: Risk stratification is a cornerstone of the Pediatric Infectious Diseases Society COVID-19 treatment guidance. This systematic review and meta-analysis aimed to define the clinical characteristics and comorbidities associated with critical COVID-19 in children and adolescents. Methods: Two independent reviewers screened the literature (Medline and EMBASE) for studies published through August 2023 that reported outcome data on patients aged 21 years or younger with COVID-19. Critical disease was defined as an invasive mechanical ventilation requirement, intensive care unit admission, or death. Random effects models were used to estimate pooled odds ratios (OR) with 95% confidence intervals (CI), and heterogeneity was explored through subgroup analyses. Results: Among 10,178 articles, 136 studies met the inclusion criteria for review. Data from 70 studies, which collectively examined 172,165 children and adolescents with COVID-19, were pooled for meta-analysis. In previously healthy children, the absolute risk of critical disease from COVID-19 was 4% (95% CI, 1%-10%). Compared with no comorbidities, the pooled OR for critical disease was 3.95 (95% CI, 2.78-5.63) for presence of one comorbidity and 9.51 (95% CI, 5.62-16.06) for [&amp;ge;]2 comorbidities. Key risk factors included cardiovascular and neurological disorders, chronic pulmonary conditions (excluding asthma), diabetes, obesity, and immunocompromise, all with statistically significant ORs &gt;2.00. Conclusions: While the absolute risk for critical COVID-19 in children and adolescents without underlying health conditions is relatively low, the presence of one or more comorbidities was associated with markedly increased risk. These findings support the importance of risk stratification in tailoring pediatric COVID-19 management.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.24301002">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.24301002" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.24301002">
        <p class="paperTitle">Immunological imprinting shapes the specificity of human antibody responses against SARS-CoV-2 variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.24301002" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.24301002" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnston, T. S.; Li, S. H.; Painter, M. M.; Atkinson, R. K.; Douek, N. R.; Reeg, D. B.; Doueck, D. C.; Wherry, E. J.; Hensley, S. E.</p>
        <p class="info">Score: 82.4, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.24301002' target='https://doi.org/10.1101/2024.01.08.24301002'> 10.1101/2024.01.08.24301002</a></p>
        <p class="abstract">The spike glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) continues to accumulate substitutions, leading to breakthrough infections of vaccinated individuals and prompting the development of updated booster vaccines. Here, we determined the specificity and functionality of antibody and B cell responses following exposure to BA.5 and XBB variants in individuals who received ancestral SARS-CoV-2 mRNA vaccines. BA.5 exposures elicited antibody responses that primarily targeted epitopes conserved between the BA.5 and ancestral spike, with poor reactivity to the XBB.1.5 variant. XBB exposures also elicited antibody responses that targeted epitopes conserved between the XBB.1.5 and ancestral spike. However, unlike BA.5, a single XBB exposure elicited low levels of XBB.1.5-specific antibodies and B cells in some individuals. Pre-existing cross-reactive B cells and antibodies were correlated with stronger overall responses to XBB but weaker XBB-specific responses, suggesting that baseline immunity influences the activation of variant-specific SARS-CoV-2 responses.

HighlightsO_LIVariant breakthrough infections boost ancestral cross-reactive antibodies and B cells
C_LIO_LIFirst and second BA.5 exposures fail to elicit variant-specific antibodies and B cells
C_LIO_LIXBB infections and monovalent vaccinations elicit XBB.1.5-specific responses in some individuals
C_LIO_LIXBB.1.5-specific responses correlate with low levels of pre-existing humoral immunity
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.13.24301263">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.13.24301263" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.13.24301263">
        <p class="paperTitle">A putative hepatitis B virus sequence motif associated with hepatocellular carcinoma in South African adults</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.13.24301263" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.13.24301263" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Maponga, T. G.; McNaughton, A. L.; Campbell, C.; DeCesare, M.; Mokaya, J.; Lumley, S. F.; Bonsall, D.; Ip, C. L.; Chai, H.; Van Rensburg, C.; Glashoff, R.; Waddilove, E.; Preiser, W.; Blackard, J. T.; Ansari, M. A.; Kramvis, A.; Andersson, M. I.; Matthews, P. C.</p>
        <p class="info">Score: 1.0, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.13.24301263' target='https://doi.org/10.1101/2024.01.13.24301263'> 10.1101/2024.01.13.24301263</a></p>
        <p class="abstract">AimChronic hepatitis B virus (HBV) infection is a major risk factor for hepatocellular carcinoma (HCC) particularly in African populations, in whom malignancy frequently presents at an advanced stage with poor outcomes. We derived HBV whole genome sequences (WGS) from individuals with HCC and compared them to sequences from individuals without HCC. Methods: We identified adults with HBV infection, with and without complicating HCC, in Cape Town, South Africa and utilized pan-genotypic probe-based enrichment followed by Illumina sequencing to derive HBV WGS.

ResultsCompared to the non-HCC group, HCC patients were more likely to be male (p &lt; 0.0001), older (p = 0.01), HIV-negative (p = 0.006), and to have higher HBV viral loads (p &lt; 0.0001). Among 19 HCC and 12 non-HCC patients, genotype A dominated (74%), of which 96% were subtype A1. PreS2 deletions ({Delta}38-55) were enriched in HBV sequences from HCC patients (n = 7). The sequence motif most strongly associated with HCC comprised either deletion or polymorphism at site T53 in PreS2 - collectively coined  non-T53 - together with a basal core promoter (BCP) mutation G1764A (AUROC 0.79).

ConclusionsIn this setting, HBV sequence polymorphisms and deletions are associated with HCC, and  non-T53 &#43; G1764A represents a putative signature motif for HCC. Additional investigations are needed to disaggregate the impact of age, sex and HIV status, to ascertain the extent to which viral polymorphisms contribute to oncogenesis, and to determine whether HBV sequence is a useful biomarker for risk stratification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.24301015">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.24301015" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.24301015">
        <p class="paperTitle">Climate warming is expanding dengue burden in the Americas and Asia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.24301015" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.24301015" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Childs, M. L.; Lyberger, K.; Harris, M.; Burke, M.; Mordecai, E. A.</p>
        <p class="info">Score: 1.5, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.24301015' target='https://doi.org/10.1101/2024.01.08.24301015'> 10.1101/2024.01.08.24301015</a></p>
        <p class="abstract">Climate change poses significant threats to public health, with dengue representing a growing concern due to its high existing burden and sensitivity to climatic conditions. Yet, the quantitative impacts of temperature warming on dengue, both in the past and in the future, remain poorly understood. In this study, we quantify how dengue responds to climatic fluctuations, and use this inferred temperature response to estimate the impacts of historical warming and forecast trends under future climate change scenarios. To estimate the causal impact of temperature on the spread of dengue in the Americas and Asia, we assembled a dataset encompassing nearly 1.5 million dengue incidence records from 21 countries. Our analysis revealed a nonlinear relationship between temperature and dengue incidence with the largest marginal effects at lower temperatures (around 15{degrees}C), peak incidence at 27.8{degrees}C (95% CI: 27.3 - 28.2{degrees}C), and subsequent declines at higher temperatures. Our findings indicate that historical climate change has already increased dengue incidence 18% (12 - 25%) in the study region, and projections suggest a potential increase of 40% (17 - 76) to 57% (33 - 107%) by mid-century depending on the climate scenario, with some areas seeing up to 200% increases. Notably, our models suggest that lower emissions scenarios would substantially reduce the warming-driven increase in dengue burden. Together, these insights contribute to the broader understanding of how long-term climate patterns influence dengue, providing a valuable foundation for public health planning and the development of strategies to mitigate future risks due to climate change.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.23300635">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.23300635" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.23300635">
        <p class="paperTitle">Air cleaners and respiratory infections in schools: A modeling study using epidemiological, environmental, and molecular data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.23300635" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.23300635" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Banholzer, N.; Jent, P.; Bittel, P.; Zurcher, K.; Furrer, L.; Bertschinger, S.; Weingartner, E.; Ramette, A.; Egger, M.; Hascher, T.; Fenner, L.</p>
        <p class="info">Score: 2.5, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.23300635' target='https://doi.org/10.1101/2023.12.29.23300635'> 10.1101/2023.12.29.23300635</a></p>
        <p class="abstract">BackgroundUsing a multiple-measurement approach, we examined the real-world effectiveness of portable HEPA-air filtration devices (air cleaners) in a school setting.

MethodsWe collected environmental (CO2, particle concentrations), epidemiological (absences related to respiratory infections), audio (coughing), and molecular data (bioaerosol and saliva samples) over seven weeks during winter 2022/2023 in two Swiss secondary school classes. Using a cross-over study design, we compared particle concentrations, coughing, and the risk of infection with vs without air cleaners.

ResultsAll 38 students (age 13-15 years) participated. With air cleaners, mean particle con-centration decreased by 77% (95% credible interval 63%-86%). There were no differences in CO2 levels. Absences related to respiratory infections were 22 without vs 13 with air cleaners. Bayesian modeling suggested a reduced risk of infection, with a posterior probability of 91% and a relative risk of 0.73 (95% credible interval 0.44-1.18). Coughing also tended to be less frequent (posterior probability 93%). Molecular analysis detected mainly non-SARS-CoV-2 viruses in saliva (50/448 positive), but not in bioaerosols (2/105 positive) or HEPA-filters (4/160). The detection rate was similar with vs without air cleaners. Spatiotemporal analysis of positive saliva samples identified several likely transmissions.

ConclusionsAir cleaners improved air quality, showed a potential benefit in reducing respiratory infections, and were associated with less coughing. Airborne detection of non-SARS-CoV-2 viruses was rare, suggesting that these viruses may be more difficult to detect in the air. Future studies should examine the importance of close contact and long-range transmission, and the cost-effectiveness of using air cleaners.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
