<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Risk of SARS-CoV-2 reinfection during multiple Omicron variant waves in the UK general population
Authors: Wei, J.; Stoesser, N.; Matthews, P.; Khera, T.; Gethings, O.; Diamond, I.; Studley, R.; Taylor, N.; peto, t. E.; Walker, A. S.; Pouwels, K.; Eyre, D. W.
Score: 78.2, Published: 2023-06-30 DOI: 10.1101/2023.06.29.23292043
SARS-CoV-2 reinfections increased substantially after Omicron variants emerged. Large-scale community-based comparisons across multiple Omicron waves of reinfection characteristics, risk factors, and protection afforded by previous infection and vaccination, are limited, especially after widespread national testing stopped.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Risk of SARS-CoV-2 reinfection during multiple Omicron variant waves in the UK general population
Authors: Wei, J.; Stoesser, N.; Matthews, P.; Khera, T.; Gethings, O.; Diamond, I.; Studley, R.; Taylor, N.; peto, t. E.; Walker, A. S.; Pouwels, K.; Eyre, D. W.
Score: 78.2, Published: 2023-06-30 DOI: 10.1101/2023.06.29.23292043
SARS-CoV-2 reinfections increased substantially after Omicron variants emerged. Large-scale community-based comparisons across multiple Omicron waves of reinfection characteristics, risk factors, and protection afforded by previous infection and vaccination, are limited, especially after widespread national testing stopped." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-16T10:39:40+00:00" />
<meta property="article:modified_time" content="2023-07-16T10:39:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Risk of SARS-CoV-2 reinfection during multiple Omicron variant waves in the UK general population
Authors: Wei, J.; Stoesser, N.; Matthews, P.; Khera, T.; Gethings, O.; Diamond, I.; Studley, R.; Taylor, N.; peto, t. E.; Walker, A. S.; Pouwels, K.; Eyre, D. W.
Score: 78.2, Published: 2023-06-30 DOI: 10.1101/2023.06.29.23292043
SARS-CoV-2 reinfections increased substantially after Omicron variants emerged. Large-scale community-based comparisons across multiple Omicron waves of reinfection characteristics, risk factors, and protection afforded by previous infection and vaccination, are limited, especially after widespread national testing stopped."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Risk of SARS-CoV-2 reinfection during multiple Omicron variant waves in the UK general population\nAuthors: Wei, J.; Stoesser, N.; Matthews, P.; Khera, T.; Gethings, O.; Diamond, I.; Studley, R.; Taylor, N.; peto, t. E.; Walker, A. S.; Pouwels, K.; Eyre, D. W.\nScore: 78.2, Published: 2023-06-30 DOI: 10.1101/2023.06.29.23292043\nSARS-CoV-2 reinfections increased substantially after Omicron variants emerged. Large-scale community-based comparisons across multiple Omicron waves of reinfection characteristics, risk factors, and protection afforded by previous infection and vaccination, are limited, especially after widespread national testing stopped.",
  "keywords": [
    
  ],
  "articleBody": " Risk of SARS-CoV-2 reinfection during multiple Omicron variant waves in the UK general population\nAuthors: Wei, J.; Stoesser, N.; Matthews, P.; Khera, T.; Gethings, O.; Diamond, I.; Studley, R.; Taylor, N.; peto, t. E.; Walker, A. S.; Pouwels, K.; Eyre, D. W.\nScore: 78.2, Published: 2023-06-30 DOI: 10.1101/2023.06.29.23292043\nSARS-CoV-2 reinfections increased substantially after Omicron variants emerged. Large-scale community-based comparisons across multiple Omicron waves of reinfection characteristics, risk factors, and protection afforded by previous infection and vaccination, are limited, especially after widespread national testing stopped. We studied 245,895 adults [\u0026ge;]18y in the UKs national COVID-19 Infection Survey with at least one infection (identified from positive swab tests done within the study, linked from national testing programmes, or self-reported by participants, up to their last study assessment). We quantified the risk of reinfection in multiple infection waves, including those driven by BA.1, BA.2, BA.4/5, and most recently BQ.1/CH.1.1/XBB.1.5 variants, in which most reinfections occurred. Reinfections had higher cycle threshold (Ct) values (lower viral load) and lower percentages self-reporting symptoms compared with first infections. Across multiple Omicron waves, protection against reinfection was significantly higher in those previously infected with more recent than earlier variants, even at the same time from previous infection. Protection against Omicron reinfections decreased over time from the most recent infection if this was the previous or penultimate variant (generally within the preceding year), but did not change or even slightly increased over time if this was with an even earlier variant (generally \u003e1 year previously). Those 14-180 days after receiving their most recent vaccination had a lower risk of reinfection with all Omicron variants except BA.2 than those \u003e180 days from their most recent vaccination. Reinfection risk was independently higher in those aged 30-45 years, and with either low or high Ct values in their most recent previous infection. Overall, the risk of Omicron reinfection is high, but with lower severity than first infections; reinfection risk is likely driven as much by viral evolution as waning immunity.\nBooster dose of self-amplifying SARS-CoV-2 RNA vaccine vs. mRNA vaccine: a phase 3 comparison of ARCT-154 with Comirnaty\nAuthors: Oda, Y.; Kumagai, Y.; Kanai, M.; Iwama, Y.; Okura, I.; Minamida, T.; Yagi, Y.; Kurosawa, T.; Greener, B.; Zhang, Y.; Walson, J. L.\nScore: 34.8, Published: 2023-07-13 DOI: 10.1101/2023.07.13.23292597\nBackground Licensed mRNA vaccines demonstrated initial effectiveness against COVID-19 but require booster doses to broaden the anti-SARS-CoV-2 response. There is an unmet need for novel highly immunogenic and broadly protective vaccines. We compared immunogenicity and tolerability of ARCT-154, a novel self-amplifying mRNA vaccine with the mRNA vaccine, Comirnaty. Methods We compared immune responses to ARCT-154 and Comirnaty booster doses in healthy 18-77-year-old Japanese adults initially immunised with two doses of mRNA COVID-19 vaccine (Comirnaty or Spikevax) then a third dose of Comirnaty at least 3 months previously. Neutralising antibodies were measured before and 28 days after booster vaccination. The primary objective was to demonstrate non-inferiority of the immune response against Wuhan-Hu-1 SARS-CoV-2 virus as geometric mean titre (GMT) ratios and seroresponse rates (SRR) of neutralising antibodies; key secondary endpoints included the immune response against the Omicron BA.4/5 variant and vaccine tolerability assessed using participant-completed electronic diaries. Findings Between December 13, 2022 and February 25, 2023 we enrolled 828 participants randomised 1:1 to receive ARCT-154 (n = 420) or Comirnaty (n = 408) booster doses. Four weeks after boosting, ARCT-154 induced higher Wuhan-Hu-1 neutralising antibodies GMTs than Comirnaty (5641 [95% CI: 4321-7363] and 3934 [2993-5169], respectively), a GMT ratio of 1.43 (95% CI: 1.26-1.63), with SRR of 65.2% (60.2-69.9) and 51.6% (46.4-56.8) meeting the non-inferiority criteria. Respective anti-Omicron BA.4/5 GMTs were 2551 (1687-3859) and 1958 (1281-2993), a GMT ratio of 1.30 (95% CI: 1.07-1.58), with SRR of 69.9% (65.0-74.4) and 58.0% (52.8-63.1), meeting the superiority criteria for ARCT-154 over Comirnaty. Booster doses of either ARCT-154 or Comirnaty were equally well-tolerated with no causally-associated severe or serious adverse events; 94.8% and 96.8% of ARCT-154 and Comirnaty vaccinees reported local reactions and 65.7% and 62.5% had solicited systemic adverse events. Events were mainly mild in severity, occurring and resolving within 3-4 days of vaccination. Interpretation Immune responses four weeks after an ARCT-154 booster dose in mRNA-immunised adults were higher than after a Comirnaty booster, meeting non-inferiority criteria against the prototype Wuhan-Hu-1 virus, and superiority criteria against the Omicron BA.4/5 variant.\nEarly oral switch in low-risk Staphylococcus aureus bloodstream infection\nAuthors: Kaasch, A. J.; Lopez-Cortes, L. E.; Rodriguez-Bano, J.; Cisneros, J. M.; Navarro, D.; Faetkenheuer, G.; Jung, N.; Rieg, S.; Lepeule, R.; Coutte, L.; Bernard, L.; Lemaignen, A.; Koesters, K.; MacKenzie, C. R.; Soriano, A.; Hagel, S.; Fantin, B.; Lafaurie, M.; Talarmin, J.-P.; Dinh, A.; Guimard, T.; Boutoille, D.; Welte, T.; Reuter, S.; Kluytmans, J.; Martin, M. L.; Forestier, E.; Stocker, H.; Vitrat, V.; Tattevin, P.; Rommerskirchen, A.; Noret, M.; Adams, A.; Kern, W. V.; Hellmich, M.; Seifert, H.; SABATO group, Score: 103.7, Published: 2023-07-05 DOI: 10.1101/2023.07.03.23291932\nBackgroundStaphylococcus aureus bloodstream infection (SAB) is treated with at least 14 days of intravenously administered antimicrobials. We assessed the efficacy and safety of an early oral switch therapy in patients at low risk for SAB-related complications. MethodsIn an international non-inferiority trial, we randomized patients with SAB after 5 to 7 days of intravenous antimicrobial therapy to either switch to an oral antimicrobial or to continue with intravenous standard therapy. Main exclusion criteria were signs and symptoms of complicated SAB, non-removable foreign devices, and severe comorbidity. Composite primary endpoint was the occurrence of any SAB-related complication (relapsing SAB, deep-seated infection, and mortality attributable to SAB) within 90 days. Results213 patients were randomized into the intention-to-treat population. In the oral switch group, 14/108 (13%) participants reached the primary endpoint versus 13/105 (12%) in the standard therapy group (adjusted difference 0.7%, 95% confidence interval [CI] -7.8% to 9.1%). Participants in the oral switch group were discharged earlier (median hospital stay from SAB onset of 12 days versus 16 days; adjusted difference -3.1 days [95% CI -7.5 to 1.4]). There was no statistical difference in 30-day survival and complications of intravenous administration. More participants in the oral group experienced at least one serious adverse event (34% versus 26%, p=0.292). ConclusionOral switch was non-inferior to intravenous standard therapy in participants with low-risk SAB. However, a careful assessment of patients for signs and symptoms of complicated SAB at time of presentation and thereafter is necessary before considering early oral switch therapy. The trial was registered as NCT01792804 in ClinicalTrials.gov, as DRKS00004741 in the German Clinical trials register, and as EudraCT 2013-000577-77.\nSARS-CoV-2 virologic rebound with nirmatrelvir-ritonavir therapy\nAuthors: Edelstein, G. E.; Boucau, J.; Uddin, R.; Marino, C.; Liew, M. Y.; Barry, M.; Choudhary, M. C.; Gilbert, R. F.; Reynolds, Z.; Li, Y.; Tien, D.; Sagar, S.; Vyas, T. D.; Kawano, Y.; Sparks, J. A.; Hammond, S. P.; Wallace, Z.; Vyas, J. M.; Barczak, A. K.; Lemieux, J. E.; Li, J. Z.; Siedner, M. J.\nScore: 174.1, Published: 2023-06-27 DOI: 10.1101/2023.06.23.23288598\nObjectiveTo compare the frequency of replication-competent virologic rebound with and without nirmatrelvir-ritonavir treatment for acute COVID-19. Secondary aims were to estimate the validity of symptoms to detect rebound and the incidence of emergent nirmatrelvir-resistance mutations after rebound. DesignObservational cohort study. SettingMulticenter healthcare system in Boston, Massachusetts. ParticipantsWe enrolled ambulatory adults with a positive COVID-19 test and/or a prescription for nirmatrelvir-ritonavir. ExposuresReceipt of 5 days of nirmatrelvir-ritonavir treatment versus no COVID-19 therapy. Main Outcome and MeasuresThe primary outcome was COVID-19 virologic rebound, defined as either (1) a positive SARS-CoV-2 viral culture following a prior negative culture or (2) two consecutive viral loads [\u0026ge;]4.0 log10 copies/milliliter after a prior reduction in viral load to \u003c4.0 log10 copies/milliliter. ResultsCompared with untreated individuals (n=55), those taking nirmatrelvir-ritonavir (n=72) were older, received more COVID-19 vaccinations, and were more commonly immunosuppressed. Fifteen individuals (20.8%) taking nirmatrelvir-ritonavir experienced virologic rebound versus one (1.8%) of the untreated (absolute difference 19.0% [95%CI 9.0-29.0%], P=0.001). In multivariable models, only N-R was associated with VR (AOR 10.02, 95%CI 1.13-88.74). VR occurred more commonly among those with earlier nirmatrelvirritonavir initiation (29.0%, 16.7% and 0% when initiated days 0, 1, and [\u0026ge;]2 after diagnosis, respectively, P=0.089). Among participants on N-R, those experiencing rebound had prolonged shedding of replication-competent virus compared to those that did not rebound (median: 14 vs 3 days). Only 8/16 with virologic rebound reported worsening symptoms (50%, 95%CI 25%-75%); 2 were completely asymptomatic. We detected no post-rebound nirmatrelvir-resistance mutations in the NSP5 protease gene. Conclusions and RelevanceVirologic rebound occurred in approximately one in five people taking nirmatrelvir-ritonavir and often occurred without worsening symptoms. Because it is associated with replication-competent viral shedding, close monitoring and potential isolation of those who rebound should be considered.\nNationally representative prevalence and determinants of post-acute sequelae of SARS-CoV-2 infection (Long COVID) amongst Mexican adults in 2022\nAuthors: Bello-Chavolla, O. Y.; Fermin-Martinez, C. A.; Fernandez-Chirino, L.; Ramirez-Garcia, D.; Vargas-Vazquez, A.; Basile-Alvarez, M. R.; Sanchez-Castro, P.; Nunez-Luna, A.; Antonio-Villa, N. E.\nScore: 14.0, Published: 2023-07-12 DOI: 10.1101/2023.07.10.23292475\nOBJECTIVETo characterize the epidemiology of post-acute sequelae after SARS-CoV-2 infection (PASC) in Mexico during 2022 and identify potential predictors of PASC prevalence using nationally representative data. METHODSWe analyzed data from the 2022 Mexican National Health and Nutrition Survey (ENSANUT) totaling 24,434 participants, representing 85,521,661 adults [\u0026ge;]20 years. PASC was defined using both the World Health Organization definition and a PASC score [\u0026ge;]12. Estimates of PASC prevalence were stratified by age, sex, rural vs. urban setting, social lag quartiles, number of reinfections, vaccination status and by periods of predominance of SARS-CoV-2 circulating variants. Predictors of PASC were assessed using logistic regression models adjusted by survey weights. RESULTSPersistent symptoms after SARS-CoV-2 infection were reported by 12.44% (95%CI 11.89-12.99) of adults [\u0026ge;]20 years in Mexico during 2022. The most common persistent symptoms were musculoskeletal pain, headache, cough, loss of smell or taste, fever, post-exertional malaise, brain fog, anxiety, chest pain, and sleep disorders. PASC was present in 21.21% (95%CI 7.71-9.65) subjects with previously diagnosed COVID-19. Over 28.6% patients with PASC reported symptoms persistence [\u0026ge;]6 months and 14.05% reported incapacitating symptoms. Higher PASC prevalence was associated with SARS-CoV-2 reinfections, depressive symptoms and living in states with high social lag. PASC prevalence, particularly its more severe forms, decreased with COVID-19 vaccination and for infections during periods of Omicron variant predominance. CONCLUSIONSPASC implies a significant public health burden in Mexico as the COVID-19 pandemic transitions into endemicity. Promoting reinfection prevention and booster vaccination may be useful to reduce PASC burden.\nIncreased circulating fibronectin, depletion of natural IgM and heightened EBV, HSV-1 reactivation in ME/CFS and long COVID\nAuthors: Liu, Z.; Hollmann, C.; Kalanidhi, S.; Grothey, A.; Keating, S.; Mena-Palomo, I.; Lamer, S.; Schlosser, A.; Kaiping, A.; Scheller, C.; Sotzny, F.; Horn, A.; Nuernberger, C.; Cejka, V.; Afshar, B.; Bahmer, T.; Schreiber, S.; Janne Vehreschild, J.; Milljukov, O.; Schaefer, C.; Kretzler, L.; Keil, T.; Reese, J.-P.; Eichner, F. A.; Schmidbauer, L.; Heuschmann, P. U.; Stoerk, S.; Morbach, C.; Riemekasten, G.; Beyersdorf, N.; Scheibenbogen, C.; Naviaux, R. K.; Williams, M.; Ariza, M. E.; Prusty, B. K.\nScore: 470.3, Published: 2023-06-29 DOI: 10.1101/2023.06.23.23291827\nMyalgic Encephalomyelitis/ Chronic Fatigue syndrome (ME/CFS) is a complex, debilitating, long-term illness without a diagnostic biomarker. ME/CFS patients share overlapping symptoms with long COVID patients, an observation which has strengthened the infectious origin hypothesis of ME/CFS. However, the exact sequence of events leading to disease development is largely unknown for both clinical conditions. Here we show antibody response to herpesvirus dUTPases, particularly to that of Epstein-Barr virus (EBV) and HSV-1, increased circulating fibronectin (FN1) levels in serum and depletion of natural IgM against fibronectin ((n)IgM-FN1) are common factors for both severe ME/CFS and long COVID. We provide evidence for herpesvirus dUTPases-mediated alterations in host cell cytoskeleton, mitochondrial dysfunction and OXPHOS. Our data show altered active immune complexes, immunoglobulin-mediated mitochondrial fragmentation as well as adaptive IgM production in ME/CFS patients. Our findings provide mechanistic insight into both ME/CFS and long COVID development. Finding of increased circulating FN1 and depletion of (n)IgM-FN1 as a biomarker for the severity of both ME/CFS and long COVID has an immediate implication in diagnostics and development of treatment modalities.\nHealthy adults possess cross-reactive neuraminidase inhibition antibodies to an A(H5N1) clade 2.3.4.4b avian influenza virus A/Black Faced Spoonbill/Hong Kong/AFCD-HKU-22-21429-01012/2022\nAuthors: Daulagala, P.; Cheng, S. M. S.; Chin, A. W. H.; Luk, L. L. H.; Leung, K.; Wu, J. T.; Poon, L. L. M.; Peiris, J. S. M.; Yen, H.-L.\nScore: 50.0, Published: 2023-06-27 DOI: 10.1101/2023.06.23.23291839\nHemagglutination inhibition (HI) and neuraminidase inhibition (NI) antibodies to a clade 2.3.4.4b A(H5N1) highly pathogenic avian influenza virus were measured in 63 age-stratified healthy adults in Hong Kong. No HI antibody was detected; 61 subjects had detectable NI antibodies to A(H5N1). NI titers to A(H5N1) and A(H1N1)pdm09 viruses were correlated.\nEnvironmental Surface Monitoring as a Noninvasive Method for SARS-CoV-2 Surveillance in Community Settings: Lessons from a University Campus Study\nAuthors: Ali, S.; Cella, E.; Johnston, C.; Deichen, M.; Azarian, T.\nScore: 7.4, Published: 2023-07-14 DOI: 10.1101/2023.07.13.23292575\nEnvironmental testing of high-touch objects is a potential noninvasive approach for monitoring population-level trends of SARS-CoV-2 and other respiratory viruses within a defined setting. We aimed to determine the association between SARS-CoV-2 contamination on high-touch environmental surfaces, community level case incidence, and university student health data. Environmental swabs were collected from January 2022 to November 2022 from high-touch objects and surfaces from five locations on a large university campus in Florida, USA. RT-qPCR was used to detect and quantify viral RNA, and a subset of positive samples was analyzed by viral genome sequencing to identify circulating lineages. During the study period, we detected SARS-CoV-2 viral RNA on 90.7% of 162 tested samples. Levels of environmental viral RNA correlated with trends in community-level activity and case reports from the student health center. A significant positive correlation was observed between the estimated viral gene copy number in environmental samples and the weekly confirmed cases at the university. Viral sequencing data from environmental samples identified lineages contemporaneously circulating in the local community and state based on genomic surveillance data. Further, we detected emerging variants in environmental samples prior to their identification by clinical genomic surveillance. Our results demonstrate the utility of viral monitoring on high-touch environmental surfaces for SARS-CoV-2 surveillance at a community level. In communities with delayed or limited testing facilities, immediate environmental surface testing may considerably inform epidemic dynamics.\nExamination of Influenza A Infection Rate, Its Determinants, and Seasonal Influenza Vaccine Effectiveness in the Post-COVID-19 Pandemic Era\nAuthors: Wagenhäuser, I.; Mees, J.; Reusch, J.; Lam, T.-T.; Schubert-Unkmeir, A.; Krone, L. B.; Frey, A.; Kurzai, O.; Frantz, S.; Dölken, L.; Liese, J.; Gabel, A.; Petri, N.; Krone, M.\nScore: 4.3, Published: 2023-07-12 DOI: 10.1101/2023.07.08.23292128\nBackgroundIn the context of the COVID-19 pandemic, a pronounced wave of Influenza A occurred in the 2022/23 winter season under generally relaxed post-pandemic non-pharmaceutical preventive measures. AimThis study aimed to investigate the Influenza A infection rate, factors influencing its occurrence and seasonal Influenza vaccine effectiveness on seroconversion in the post-COVID-19 pandemic era. MethodsThe seroconversion of Anti-Influenza-A-Nucleoprotein/Matrix IgG was investigated in 402 healthcare workers (HCWs) during the winter season of 2022/2023 (23 May 2022 to 11 May 2023). The participants provided a serum sample and completed a study questionnaire both before and after the seasonal Influenza A wave (24 October 2022 to 8 January 2023). The levels of a vaccine-independent Anti-Influenza-A-Nucleoprotein/Matrix IgG were measured using the SERION ELISA classic Influenza A IgG assay, with a 2-fold increase as indicative of seroconversion after asymptomatic or symptomatic influenza infection. ResultsAmong the participants, 20.6% (95% CI 17.0-24.9%; 83/402) showed seroconversion. The multivariate logistic regression analysis revealed that the age category of [\u0026ge;] 45 years (p=0.03) and regular patient contact (p=0.02) significantly influenced seroconversion. However, the factors male gender, BMI, smoking, household size, seasonal Influenza vaccination, and SARS-CoV-2 infection during the Influenza A season were not significantly associated with seroconversion. The effectiveness of the 2022/23 seasonal Influenza vaccine on seroconversion induced by Influenza infection was 22.6% (95% CI -17.1-50.6%). ConclusionDuring the initial Influenza A season following the COVID-19 pandemic, approximately 20% of HCWs contracted an Influenza A infection. This highlights a potential risk and a significant asymptomatic or symptomatic infection rate posing a theoretical risk for intrahospital transmission chains and nosocomial infections.\nThe distribution of antimicrobial resistance genes across phylogroup, host species and geography in 16,000 publicly-available E. coli genomes\nAuthors: Pursey, E.; Dimitriu, T.; Gaze, W. H.; Westra, E. R.; van Houte, S.\nScore: 29.0, Published: 2023-07-12 DOI: 10.1101/2022.08.05.22278465\nE. coli is a highly diverse bacterial species that generates a huge global burden of antimicrobial-resistant infections. A wealth of whole genome sequence data is available on public databases for this species, presenting new opportunities to analyse the distribution of antimicrobial resistance (AMR) genes across its genetic and ecological diversity. We extracted and categorised metadata on host species and geographic location and combined this with in silico phylogrouping to describe the characteristics of [~]16,000 assembled E. coli genomes from the NCBI RefSeq database. We estimated AMR carriage using various metrics: counts of overall genes, multidrug- and extensively drug-resistant categories, and selected {beta}-lactamases of current global concern - blaCTX-M and carbapenemase genes. We present estimates of AMR carriage for these metrics by species type (human, agricultural/domestic animal, wild birds and other wild animals), geographic subregion, and across phylogroups. In addition, we describe the distribution of phylogroups within host types and geographic subregions. Our findings show high AMR carriage in commensal-associated phylogroups, agricultural and wild animal hosts and in many subregions. However, we also quantify large biases in sequencing data, the substantial gaps in our knowledge of AMR in many hosts, regions and environmental settings, and the need for systematic sampling to gain a more accurate picture.\n",
  "wordCount" : "2913",
  "inLanguage": "en",
  "datePublished": "2023-07-16T10:39:40Z",
  "dateModified": "2023-07-16T10:39:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta"><span>updated on July 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.23292043">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.23292043" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.23292043">
        <p class="paperTitle">Risk of SARS-CoV-2 reinfection during multiple Omicron variant waves in the UK general population</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.23292043" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.23292043" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wei, J.; Stoesser, N.; Matthews, P.; Khera, T.; Gethings, O.; Diamond, I.; Studley, R.; Taylor, N.; peto, t. E.; Walker, A. S.; Pouwels, K.; Eyre, D. W.</p>
        <p class="info">Score: 78.2, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.23292043' target='https://doi.org/10.1101/2023.06.29.23292043'> 10.1101/2023.06.29.23292043</a></p>
        <p class="abstract">SARS-CoV-2 reinfections increased substantially after Omicron variants emerged. Large-scale community-based comparisons across multiple Omicron waves of reinfection characteristics, risk factors, and protection afforded by previous infection and vaccination, are limited, especially after widespread national testing stopped. We studied 245,895 adults [&amp;ge;]18y in the UKs national COVID-19 Infection Survey with at least one infection (identified from positive swab tests done within the study, linked from national testing programmes, or self-reported by participants, up to their last study assessment). We quantified the risk of reinfection in multiple infection waves, including those driven by BA.1, BA.2, BA.4/5, and most recently BQ.1/CH.1.1/XBB.1.5 variants, in which most reinfections occurred. Reinfections had higher cycle threshold (Ct) values (lower viral load) and lower percentages self-reporting symptoms compared with first infections. Across multiple Omicron waves, protection against reinfection was significantly higher in those previously infected with more recent than earlier variants, even at the same time from previous infection. Protection against Omicron reinfections decreased over time from the most recent infection if this was the previous or penultimate variant (generally within the preceding year), but did not change or even slightly increased over time if this was with an even earlier variant (generally &gt;1 year previously). Those 14-180 days after receiving their most recent vaccination had a lower risk of reinfection with all Omicron variants except BA.2 than those &gt;180 days from their most recent vaccination. Reinfection risk was independently higher in those aged 30-45 years, and with either low or high Ct values in their most recent previous infection. Overall, the risk of Omicron reinfection is high, but with lower severity than first infections; reinfection risk is likely driven as much by viral evolution as waning immunity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.23292597">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.23292597" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.23292597">
        <p class="paperTitle">Booster dose of self-amplifying SARS-CoV-2 RNA vaccine vs. mRNA vaccine: a phase 3 comparison of ARCT-154 with Comirnaty</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.23292597" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.23292597" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oda, Y.; Kumagai, Y.; Kanai, M.; Iwama, Y.; Okura, I.; Minamida, T.; Yagi, Y.; Kurosawa, T.; Greener, B.; Zhang, Y.; Walson, J. L.</p>
        <p class="info">Score: 34.8, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.23292597' target='https://doi.org/10.1101/2023.07.13.23292597'> 10.1101/2023.07.13.23292597</a></p>
        <p class="abstract">Background Licensed mRNA vaccines demonstrated initial effectiveness against COVID-19 but require booster doses to broaden the anti-SARS-CoV-2 response. There is an unmet need for novel highly immunogenic and broadly protective vaccines. We compared immunogenicity and tolerability of ARCT-154, a novel self-amplifying mRNA vaccine with the mRNA vaccine, Comirnaty. Methods We compared immune responses to ARCT-154 and Comirnaty booster doses in healthy 18-77-year-old Japanese adults initially immunised with two doses of mRNA COVID-19 vaccine (Comirnaty or Spikevax) then a third dose of Comirnaty at least 3 months previously. Neutralising antibodies were measured before and 28 days after booster vaccination. The primary objective was to demonstrate non-inferiority of the immune response against Wuhan-Hu-1 SARS-CoV-2 virus as geometric mean titre (GMT) ratios and seroresponse rates (SRR) of neutralising antibodies; key secondary endpoints included the immune response against the Omicron BA.4/5 variant and vaccine tolerability assessed using participant-completed electronic diaries. Findings Between December 13, 2022 and February 25, 2023 we enrolled 828 participants randomised 1:1 to receive ARCT-154 (n = 420) or Comirnaty (n = 408) booster doses. Four weeks after boosting, ARCT-154 induced higher Wuhan-Hu-1 neutralising antibodies GMTs than Comirnaty (5641 [95% CI: 4321-7363] and 3934 [2993-5169], respectively), a GMT ratio of 1.43 (95% CI: 1.26-1.63), with SRR of 65.2% (60.2-69.9) and 51.6% (46.4-56.8) meeting the non-inferiority criteria. Respective anti-Omicron BA.4/5 GMTs were 2551 (1687-3859) and 1958 (1281-2993), a GMT ratio of 1.30 (95% CI: 1.07-1.58), with SRR of 69.9% (65.0-74.4) and 58.0% (52.8-63.1), meeting the superiority criteria for ARCT-154 over Comirnaty. Booster doses of either ARCT-154 or Comirnaty were equally well-tolerated with no causally-associated severe or serious adverse events; 94.8% and 96.8% of ARCT-154 and Comirnaty vaccinees reported local reactions and 65.7% and 62.5% had solicited systemic adverse events. Events were mainly mild in severity, occurring and resolving within 3-4 days of vaccination. Interpretation Immune responses four weeks after an ARCT-154 booster dose in mRNA-immunised adults were higher than after a Comirnaty booster, meeting non-inferiority criteria against the prototype Wuhan-Hu-1 virus, and superiority criteria against the Omicron BA.4/5 variant.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.23291932">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.23291932" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.23291932">
        <p class="paperTitle">Early oral switch in low-risk Staphylococcus aureus bloodstream infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.23291932" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.23291932" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaasch, A. J.; Lopez-Cortes, L. E.; Rodriguez-Bano, J.; Cisneros, J. M.; Navarro, D.; Faetkenheuer, G.; Jung, N.; Rieg, S.; Lepeule, R.; Coutte, L.; Bernard, L.; Lemaignen, A.; Koesters, K.; MacKenzie, C. R.; Soriano, A.; Hagel, S.; Fantin, B.; Lafaurie, M.; Talarmin, J.-P.; Dinh, A.; Guimard, T.; Boutoille, D.; Welte, T.; Reuter, S.; Kluytmans, J.; Martin, M. L.; Forestier, E.; Stocker, H.; Vitrat, V.; Tattevin, P.; Rommerskirchen, A.; Noret, M.; Adams, A.; Kern, W. V.; Hellmich, M.; Seifert, H.; SABATO group,  </p>
        <p class="info">Score: 103.7, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.23291932' target='https://doi.org/10.1101/2023.07.03.23291932'> 10.1101/2023.07.03.23291932</a></p>
        <p class="abstract">BackgroundStaphylococcus aureus bloodstream infection (SAB) is treated with at least 14 days of intravenously administered antimicrobials. We assessed the efficacy and safety of an early oral switch therapy in patients at low risk for SAB-related complications.

MethodsIn an international non-inferiority trial, we randomized patients with SAB after 5 to 7 days of intravenous antimicrobial therapy to either switch to an oral antimicrobial or to continue with intravenous standard therapy. Main exclusion criteria were signs and symptoms of complicated SAB, non-removable foreign devices, and severe comorbidity. Composite primary endpoint was the occurrence of any SAB-related complication (relapsing SAB, deep-seated infection, and mortality attributable to SAB) within 90 days.

Results213 patients were randomized into the intention-to-treat population. In the oral switch group, 14/108 (13%) participants reached the primary endpoint versus 13/105 (12%) in the standard therapy group (adjusted difference 0.7%, 95% confidence interval [CI] -7.8% to 9.1%). Participants in the oral switch group were discharged earlier (median hospital stay from SAB onset of 12 days versus 16 days; adjusted difference -3.1 days [95% CI -7.5 to 1.4]). There was no statistical difference in 30-day survival and complications of intravenous administration. More participants in the oral group experienced at least one serious adverse event (34% versus 26%, p=0.292).

ConclusionOral switch was non-inferior to intravenous standard therapy in participants with low-risk SAB. However, a careful assessment of patients for signs and symptoms of complicated SAB at time of presentation and thereafter is necessary before considering early oral switch therapy.

The trial was registered as NCT01792804 in ClinicalTrials.gov, as DRKS00004741 in the German Clinical trials register, and as EudraCT 2013-000577-77.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.23288598">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.23288598" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.23288598">
        <p class="paperTitle">SARS-CoV-2 virologic rebound with nirmatrelvir-ritonavir therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.23288598" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.23288598" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Edelstein, G. E.; Boucau, J.; Uddin, R.; Marino, C.; Liew, M. Y.; Barry, M.; Choudhary, M. C.; Gilbert, R. F.; Reynolds, Z.; Li, Y.; Tien, D.; Sagar, S.; Vyas, T. D.; Kawano, Y.; Sparks, J. A.; Hammond, S. P.; Wallace, Z.; Vyas, J. M.; Barczak, A. K.; Lemieux, J. E.; Li, J. Z.; Siedner, M. J.</p>
        <p class="info">Score: 174.1, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.23288598' target='https://doi.org/10.1101/2023.06.23.23288598'> 10.1101/2023.06.23.23288598</a></p>
        <p class="abstract">ObjectiveTo compare the frequency of replication-competent virologic rebound with and without nirmatrelvir-ritonavir treatment for acute COVID-19. Secondary aims were to estimate the validity of symptoms to detect rebound and the incidence of emergent nirmatrelvir-resistance mutations after rebound.

DesignObservational cohort study.

SettingMulticenter healthcare system in Boston, Massachusetts.

ParticipantsWe enrolled ambulatory adults with a positive COVID-19 test and/or a prescription for nirmatrelvir-ritonavir.

ExposuresReceipt of 5 days of nirmatrelvir-ritonavir treatment versus no COVID-19 therapy.

Main Outcome and MeasuresThe primary outcome was COVID-19 virologic rebound, defined as either (1) a positive SARS-CoV-2 viral culture following a prior negative culture or (2) two consecutive viral loads [&amp;ge;]4.0 log10 copies/milliliter after a prior reduction in viral load to &lt;4.0 log10 copies/milliliter.

ResultsCompared with untreated individuals (n=55), those taking nirmatrelvir-ritonavir (n=72) were older, received more COVID-19 vaccinations, and were more commonly immunosuppressed. Fifteen individuals (20.8%) taking nirmatrelvir-ritonavir experienced virologic rebound versus one (1.8%) of the untreated (absolute difference 19.0% [95%CI 9.0-29.0%], P=0.001). In multivariable models, only N-R was associated with VR (AOR 10.02, 95%CI 1.13-88.74). VR occurred more commonly among those with earlier nirmatrelvirritonavir initiation (29.0%, 16.7% and 0% when initiated days 0, 1, and [&amp;ge;]2 after diagnosis, respectively, P=0.089). Among participants on N-R, those experiencing rebound had prolonged shedding of replication-competent virus compared to those that did not rebound (median: 14 vs 3 days). Only 8/16 with virologic rebound reported worsening symptoms (50%, 95%CI 25%-75%); 2 were completely asymptomatic. We detected no post-rebound nirmatrelvir-resistance mutations in the NSP5 protease gene.

Conclusions and RelevanceVirologic rebound occurred in approximately one in five people taking nirmatrelvir-ritonavir and often occurred without worsening symptoms. Because it is associated with replication-competent viral shedding, close monitoring and potential isolation of those who rebound should be considered.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.23292475">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.23292475" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.23292475">
        <p class="paperTitle">Nationally representative prevalence and determinants of post-acute sequelae of SARS-CoV-2 infection (Long COVID) amongst Mexican adults in 2022</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.23292475" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.23292475" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bello-Chavolla, O. Y.; Fermin-Martinez, C. A.; Fernandez-Chirino, L.; Ramirez-Garcia, D.; Vargas-Vazquez, A.; Basile-Alvarez, M. R.; Sanchez-Castro, P.; Nunez-Luna, A.; Antonio-Villa, N. E.</p>
        <p class="info">Score: 14.0, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.23292475' target='https://doi.org/10.1101/2023.07.10.23292475'> 10.1101/2023.07.10.23292475</a></p>
        <p class="abstract">OBJECTIVETo characterize the epidemiology of post-acute sequelae after SARS-CoV-2 infection (PASC) in Mexico during 2022 and identify potential predictors of PASC prevalence using nationally representative data.

METHODSWe analyzed data from the 2022 Mexican National Health and Nutrition Survey (ENSANUT) totaling 24,434 participants, representing 85,521,661 adults [&amp;ge;]20 years. PASC was defined using both the World Health Organization definition and a PASC score [&amp;ge;]12. Estimates of PASC prevalence were stratified by age, sex, rural vs. urban setting, social lag quartiles, number of reinfections, vaccination status and by periods of predominance of SARS-CoV-2 circulating variants. Predictors of PASC were assessed using logistic regression models adjusted by survey weights.

RESULTSPersistent symptoms after SARS-CoV-2 infection were reported by 12.44% (95%CI 11.89-12.99) of adults [&amp;ge;]20 years in Mexico during 2022. The most common persistent symptoms were musculoskeletal pain, headache, cough, loss of smell or taste, fever, post-exertional malaise, brain fog, anxiety, chest pain, and sleep disorders. PASC was present in 21.21% (95%CI 7.71-9.65) subjects with previously diagnosed COVID-19. Over 28.6% patients with PASC reported symptoms persistence [&amp;ge;]6 months and 14.05% reported incapacitating symptoms. Higher PASC prevalence was associated with SARS-CoV-2 reinfections, depressive symptoms and living in states with high social lag. PASC prevalence, particularly its more severe forms, decreased with COVID-19 vaccination and for infections during periods of Omicron variant predominance.

CONCLUSIONSPASC implies a significant public health burden in Mexico as the COVID-19 pandemic transitions into endemicity. Promoting reinfection prevention and booster vaccination may be useful to reduce PASC burden.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.23291827">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.23291827" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.23291827">
        <p class="paperTitle">Increased circulating fibronectin, depletion of natural IgM and heightened EBV, HSV-1 reactivation in ME/CFS and long COVID</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.23291827" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.23291827" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, Z.; Hollmann, C.; Kalanidhi, S.; Grothey, A.; Keating, S.; Mena-Palomo, I.; Lamer, S.; Schlosser, A.; Kaiping, A.; Scheller, C.; Sotzny, F.; Horn, A.; Nuernberger, C.; Cejka, V.; Afshar, B.; Bahmer, T.; Schreiber, S.; Janne Vehreschild, J.; Milljukov, O.; Schaefer, C.; Kretzler, L.; Keil, T.; Reese, J.-P.; Eichner, F. A.; Schmidbauer, L.; Heuschmann, P. U.; Stoerk, S.; Morbach, C.; Riemekasten, G.; Beyersdorf, N.; Scheibenbogen, C.; Naviaux, R. K.; Williams, M.; Ariza, M. E.; Prusty, B. K.</p>
        <p class="info">Score: 470.3, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.23291827' target='https://doi.org/10.1101/2023.06.23.23291827'> 10.1101/2023.06.23.23291827</a></p>
        <p class="abstract">Myalgic Encephalomyelitis/ Chronic Fatigue syndrome (ME/CFS) is a complex, debilitating, long-term illness without a diagnostic biomarker. ME/CFS patients share overlapping symptoms with long COVID patients, an observation which has strengthened the infectious origin hypothesis of ME/CFS. However, the exact sequence of events leading to disease development is largely unknown for both clinical conditions. Here we show antibody response to herpesvirus dUTPases, particularly to that of Epstein-Barr virus (EBV) and HSV-1, increased circulating fibronectin (FN1) levels in serum and depletion of natural IgM against fibronectin ((n)IgM-FN1) are common factors for both severe ME/CFS and long COVID. We provide evidence for herpesvirus dUTPases-mediated alterations in host cell cytoskeleton, mitochondrial dysfunction and OXPHOS. Our data show altered active immune complexes, immunoglobulin-mediated mitochondrial fragmentation as well as adaptive IgM production in ME/CFS patients. Our findings provide mechanistic insight into both ME/CFS and long COVID development. Finding of increased circulating FN1 and depletion of (n)IgM-FN1 as a biomarker for the severity of both ME/CFS and long COVID has an immediate implication in diagnostics and development of treatment modalities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.23291839">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.23291839" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.23291839">
        <p class="paperTitle">Healthy adults possess cross-reactive neuraminidase inhibition antibodies to an A(H5N1) clade 2.3.4.4b avian influenza virus A/Black Faced Spoonbill/Hong Kong/AFCD-HKU-22-21429-01012/2022</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.23291839" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.23291839" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Daulagala, P.; Cheng, S. M. S.; Chin, A. W. H.; Luk, L. L. H.; Leung, K.; Wu, J. T.; Poon, L. L. M.; Peiris, J. S. M.; Yen, H.-L.</p>
        <p class="info">Score: 50.0, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.23291839' target='https://doi.org/10.1101/2023.06.23.23291839'> 10.1101/2023.06.23.23291839</a></p>
        <p class="abstract">Hemagglutination inhibition (HI) and neuraminidase inhibition (NI) antibodies to a clade 2.3.4.4b A(H5N1) highly pathogenic avian influenza virus were measured in 63 age-stratified healthy adults in Hong Kong. No HI antibody was detected; 61 subjects had detectable NI antibodies to A(H5N1). NI titers to A(H5N1) and A(H1N1)pdm09 viruses were correlated.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.23292575">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.23292575" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.23292575">
        <p class="paperTitle">Environmental Surface Monitoring as a Noninvasive Method for SARS-CoV-2 Surveillance in Community Settings: Lessons from a University Campus Study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.23292575" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.23292575" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ali, S.; Cella, E.; Johnston, C.; Deichen, M.; Azarian, T.</p>
        <p class="info">Score: 7.4, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.23292575' target='https://doi.org/10.1101/2023.07.13.23292575'> 10.1101/2023.07.13.23292575</a></p>
        <p class="abstract">Environmental testing of high-touch objects is a potential noninvasive approach for monitoring population-level trends of SARS-CoV-2 and other respiratory viruses within a defined setting. We aimed to determine the association between SARS-CoV-2 contamination on high-touch environmental surfaces, community level case incidence, and university student health data. Environmental swabs were collected from January 2022 to November 2022 from high-touch objects and surfaces from five locations on a large university campus in Florida, USA. RT-qPCR was used to detect and quantify viral RNA, and a subset of positive samples was analyzed by viral genome sequencing to identify circulating lineages. During the study period, we detected SARS-CoV-2 viral RNA on 90.7% of 162 tested samples. Levels of environmental viral RNA correlated with trends in community-level activity and case reports from the student health center. A significant positive correlation was observed between the estimated viral gene copy number in environmental samples and the weekly confirmed cases at the university. Viral sequencing data from environmental samples identified lineages contemporaneously circulating in the local community and state based on genomic surveillance data. Further, we detected emerging variants in environmental samples prior to their identification by clinical genomic surveillance. Our results demonstrate the utility of viral monitoring on high-touch environmental surfaces for SARS-CoV-2 surveillance at a community level. In communities with delayed or limited testing facilities, immediate environmental surface testing may considerably inform epidemic dynamics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.08.23292128">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.08.23292128" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.08.23292128">
        <p class="paperTitle">Examination of Influenza A Infection Rate, Its Determinants, and Seasonal Influenza Vaccine Effectiveness in the Post-COVID-19 Pandemic Era</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.08.23292128" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.08.23292128" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wagenhäuser, I.; Mees, J.; Reusch, J.; Lam, T.-T.; Schubert-Unkmeir, A.; Krone, L. B.; Frey, A.; Kurzai, O.; Frantz, S.; Dölken, L.; Liese, J.; Gabel, A.; Petri, N.; Krone, M.</p>
        <p class="info">Score: 4.3, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.08.23292128' target='https://doi.org/10.1101/2023.07.08.23292128'> 10.1101/2023.07.08.23292128</a></p>
        <p class="abstract">BackgroundIn the context of the COVID-19 pandemic, a pronounced wave of Influenza A occurred in the 2022/23 winter season under generally relaxed post-pandemic non-pharmaceutical preventive measures.

AimThis study aimed to investigate the Influenza A infection rate, factors influencing its occurrence and seasonal Influenza vaccine effectiveness on seroconversion in the post-COVID-19 pandemic era.

MethodsThe seroconversion of Anti-Influenza-A-Nucleoprotein/Matrix IgG was investigated in 402 healthcare workers (HCWs) during the winter season of 2022/2023 (23 May 2022 to 11 May 2023). The participants provided a serum sample and completed a study questionnaire both before and after the seasonal Influenza A wave (24 October 2022 to 8 January 2023). The levels of a vaccine-independent Anti-Influenza-A-Nucleoprotein/Matrix IgG were measured using the SERION ELISA classic Influenza A IgG assay, with a 2-fold increase as indicative of seroconversion after asymptomatic or symptomatic influenza infection.

ResultsAmong the participants, 20.6% (95% CI 17.0-24.9%; 83/402) showed seroconversion. The multivariate logistic regression analysis revealed that the age category of [&amp;ge;] 45 years (p=0.03) and regular patient contact (p=0.02) significantly influenced seroconversion. However, the factors male gender, BMI, smoking, household size, seasonal Influenza vaccination, and SARS-CoV-2 infection during the Influenza A season were not significantly associated with seroconversion. The effectiveness of the 2022/23 seasonal Influenza vaccine on seroconversion induced by Influenza infection was 22.6% (95% CI -17.1-50.6%).

ConclusionDuring the initial Influenza A season following the COVID-19 pandemic, approximately 20% of HCWs contracted an Influenza A infection. This highlights a potential risk and a significant asymptomatic or symptomatic infection rate posing a theoretical risk for intrahospital transmission chains and nosocomial infections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.08.05.22278465">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.08.05.22278465" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.08.05.22278465">
        <p class="paperTitle">The distribution of antimicrobial resistance genes across phylogroup, host species and geography in 16,000 publicly-available E. coli genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.08.05.22278465" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.08.05.22278465" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pursey, E.; Dimitriu, T.; Gaze, W. H.; Westra, E. R.; van Houte, S.</p>
        <p class="info">Score: 29.0, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.08.05.22278465' target='https://doi.org/10.1101/2022.08.05.22278465'> 10.1101/2022.08.05.22278465</a></p>
        <p class="abstract">E. coli is a highly diverse bacterial species that generates a huge global burden of antimicrobial-resistant infections. A wealth of whole genome sequence data is available on public databases for this species, presenting new opportunities to analyse the distribution of antimicrobial resistance (AMR) genes across its genetic and ecological diversity. We extracted and categorised metadata on host species and geographic location and combined this with in silico phylogrouping to describe the characteristics of [~]16,000 assembled E. coli genomes from the NCBI RefSeq database. We estimated AMR carriage using various metrics: counts of overall genes, multidrug- and extensively drug-resistant categories, and selected {beta}-lactamases of current global concern - blaCTX-M and carbapenemase genes. We present estimates of AMR carriage for these metrics by species type (human, agricultural/domestic animal, wild birds and other wild animals), geographic subregion, and across phylogroups. In addition, we describe the distribution of phylogroups within host types and geographic subregions. Our findings show high AMR carriage in commensal-associated phylogroups, agricultural and wild animal hosts and in many subregions. However, we also quantify large biases in sequencing data, the substantial gaps in our knowledge of AMR in many hosts, regions and environmental settings, and the need for systematic sampling to gain a more accurate picture.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
