<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Safety and Immunogenicity of XBB.1.5-Containing mRNA Vaccines
Authors: Chalkias, S.; McGhee, N.; Whatley, J. L.; Essink, B.; Brosz, A.; Tomassini, J. E.; Girard, B.; Edwards, D. K.; Nasir, A.; Lee, D.; Avena, L. E.; Feng, J.; Deng, W.; Montefiori, D. C.; Baden, L. R.; Miller, J. M.; Das, R.
Score: 493.0, Published: 2023-08-24 DOI: 10.1101/2023.08.22.23293434
BackgroundSubvariants of the severe acute respiratory syndrome coronavirus (SARS-CoV-2) omicron XBB-lineage have the potential to escape immunity provided by prior infection or vaccination.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Safety and Immunogenicity of XBB.1.5-Containing mRNA Vaccines
Authors: Chalkias, S.; McGhee, N.; Whatley, J. L.; Essink, B.; Brosz, A.; Tomassini, J. E.; Girard, B.; Edwards, D. K.; Nasir, A.; Lee, D.; Avena, L. E.; Feng, J.; Deng, W.; Montefiori, D. C.; Baden, L. R.; Miller, J. M.; Das, R.
Score: 493.0, Published: 2023-08-24 DOI: 10.1101/2023.08.22.23293434
BackgroundSubvariants of the severe acute respiratory syndrome coronavirus (SARS-CoV-2) omicron XBB-lineage have the potential to escape immunity provided by prior infection or vaccination." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-30T10:38:23+00:00" />
<meta property="article:modified_time" content="2023-08-30T10:38:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Safety and Immunogenicity of XBB.1.5-Containing mRNA Vaccines
Authors: Chalkias, S.; McGhee, N.; Whatley, J. L.; Essink, B.; Brosz, A.; Tomassini, J. E.; Girard, B.; Edwards, D. K.; Nasir, A.; Lee, D.; Avena, L. E.; Feng, J.; Deng, W.; Montefiori, D. C.; Baden, L. R.; Miller, J. M.; Das, R.
Score: 493.0, Published: 2023-08-24 DOI: 10.1101/2023.08.22.23293434
BackgroundSubvariants of the severe acute respiratory syndrome coronavirus (SARS-CoV-2) omicron XBB-lineage have the potential to escape immunity provided by prior infection or vaccination."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Safety and Immunogenicity of XBB.1.5-Containing mRNA Vaccines\nAuthors: Chalkias, S.; McGhee, N.; Whatley, J. L.; Essink, B.; Brosz, A.; Tomassini, J. E.; Girard, B.; Edwards, D. K.; Nasir, A.; Lee, D.; Avena, L. E.; Feng, J.; Deng, W.; Montefiori, D. C.; Baden, L. R.; Miller, J. M.; Das, R.\nScore: 493.0, Published: 2023-08-24 DOI: 10.1101/2023.08.22.23293434\nBackgroundSubvariants of the severe acute respiratory syndrome coronavirus (SARS-CoV-2) omicron XBB-lineage have the potential to escape immunity provided by prior infection or vaccination.",
  "keywords": [
    
  ],
  "articleBody": " Safety and Immunogenicity of XBB.1.5-Containing mRNA Vaccines\nAuthors: Chalkias, S.; McGhee, N.; Whatley, J. L.; Essink, B.; Brosz, A.; Tomassini, J. E.; Girard, B.; Edwards, D. K.; Nasir, A.; Lee, D.; Avena, L. E.; Feng, J.; Deng, W.; Montefiori, D. C.; Baden, L. R.; Miller, J. M.; Das, R.\nScore: 493.0, Published: 2023-08-24 DOI: 10.1101/2023.08.22.23293434\nBackgroundSubvariants of the severe acute respiratory syndrome coronavirus (SARS-CoV-2) omicron XBB-lineage have the potential to escape immunity provided by prior infection or vaccination. For Covid-19 immunizations beginning in the Fall 2023, the U.S. FDA has recommended updating to a monovalent omicron XBB.1.5-containing vaccine. MethodsIn this ongoing, phase 2/3 study participants were randomized 1:1 to receive 50-{micro}g doses of mRNA-1273.815 monovalent (50-{micro}g omicron XBB.1.5 spike mRNA) or mRNA-1273.231 bivalent (25-{micro}g omicron XBB.1.5 and 25-{micro}g omicron BA.4/BA.5 spike mRNAs vaccines, administered as 5th doses, to adults who previously received a primary series and 3rd dose of an original mRNA coronavirus disease 2019 (Covid-19) vaccine, and a 4th dose of a bivalent (omicron BA.4/BA.5 and original SARS-CoV-2) vaccine. Interim safety and immunogenicity data 15 days post-vaccination are presented. ResultsIn April 2023, participants received mRNA-1273.815 (n=50) and mRNA-1273.231 (n=51). The median intervals from the prior dose of BA.4/BA.5-containing bivalent vaccine were 8.2 and 8.3 months for the mRNA-1273.815 and mRNA-1273.231 groups, respectively. Both vaccines increased neutralizing antibody (nAb) geometric mean titers against all variants tested at day 15 post-booster nAb compared to pre-booster levels. Geometric mean fold-rises from pre-booster titers after the monovalent booster were higher against XBB.1.5, XBB.1.16 and SARS-CoV-2 (D614G) than those of the bivalent booster and were comparable against BA.4/BA.5 and BQ1.1 variants for both vaccines. The monovalent vaccine also elicited nAb responses against omicron XBB.2.3.2, EG.5.1, and FL.1.5.1 that were similar to those seen for XBB.1.5 in a subset (n=20) of participants. The occurrence of solicited adverse reactions and unsolicited adverse events were overall similar to those previously reported for the original mRNA-1273 50-{micro}g and omicron BA.4/BA.5-containing bivalent mRNA-1273 vaccines. ConclusionIn this interim analysis, XBB.1.5-containing monovalent and bivalent vaccines elicited potent neutralizing responses against variants of the omicron XBB-lineage (XBB.1.5, XBB.1.6, XBB.2.3.2) as well as more recent (EG.5.1, FL.1.5.1) variants. The safety profile of the XBB.1.5-containing vaccine was consistent with those of prior vaccines. These results support the Covid-19 vaccine update in 2023-2024 to a monovalent XBB.1.5-containing vaccine.\nKinetics and durability of humoral responses to SARS-CoV-2 infection and vaccination\nAuthors: Srivastava, K.; Carreno Quiroz, J. M.; Gleason, C.; Monahan, B.; Singh, G.; Abbad, A.; Tcheou, J.; Raskin, A.; Kleiner, G.; van Bakel, H.; Sordillo, E. M.; PARIS Study Group, ; Krammer, F. M.; Simon, V.\nScore: 327.3, Published: 2023-08-28 DOI: 10.1101/2023.08.26.23294679\nWe analyzed the kinetics and durability of the humoral responses to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection and vaccination using \u003e8,000 longitudinal samples collected over a three-year period (April 2020 to April 2023) in the New York City metropolitan area. Upon primary immunization, participants with pre-existing immunity mounted higher antibody responses faster and achieved higher steady-state levels compared to naive individuals. Antibody durability was characterized by two phases: an initial rapid decay, followed by a phase of stabilization with very slow decay resulting in an individual spike binding antibody steady state. Booster vaccination equalized the differences in antibody levels between participants with and without hybrid immunity, but the antibody titers reached decreased with each successive antigen exposure. Break-through infections increased antibody titers to similar levels as an additional vaccine dose in naive individuals. Our study provides strong evidence for the fact that SARS-CoV-2 antibody responses are long lasting, with an initial waning phase followed by a stabilization phase.\nPost-COVID conditions during Delta and early-Omicron SARS-CoV-2 variant periods among adults in the United States\nAuthors: Edwards, D. L.; Logan, P.; Feldstein, L.; Accorsi, E.; Fairlie, T.; Saydah, S.\nScore: 212.3, Published: 2023-08-13 DOI: 10.1101/2023.08.09.23293776\nBackgroundPost-COVID conditions after infection with new SARS-CoV-2 variants have been incompletely described. We compared the prevalence and risk factors for ongoing symptoms lasting 4 weeks or longer (often referred to as post-COVID Conditions) among adults who had tested positive vs. negative during the Delta and early-Omicron periods. MethodsSelf-reported survey data regarding symptoms and previous SARS-CoV 2 test results were collected from May 31 - July 6, 2022, from a probability sampling of United States adults. Respondents were classified according to their test result, predominant circulating variant when respondents first tested positive (Delta vs early-Omicron), and demographic risk factors. ResultsAmong 2,421 respondents, 256 tested positive during Delta, 460 during early-Omicron, and 1,705 always tested negative. Nearly one-fourth (22.3%) of negative respondents reported [\u0026ge;]1symptom that lasted [\u0026ge;]4 weeks, compared to 60.6% (p\u003c0.05) of respondents who tested positive during the Delta period and 47.8% (p\u003c0.05) during the early-Omicron period. Fatigue, change in smell/taste, and cough were commonly reported by respondents who tested positive. Demographic risk factors associated with ongoing symptoms were being female and unemployed (aOR 1.28, 95% CI 1.06-1.55; aOR 1.48, 95% CI: 1.17-1.87). ConclusionThe reported occurrence of ongoing symptoms associated with post-COVID conditions was reduced during the early-Omicron period, compared with Delta.\nMultimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19\nAuthors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A.; Dalhuisen, T.; Durstenfeld, M. S.; Hsue, P. Y.; Kelly, J. D.; Kumar, N.; Martin, J. N.; Gambhir, A.; Somsouk, M.; Seo, Y.; Deeks, S. G.; Laszik, Z. G.; VanBrocklin, H. F.; Henrich, T. J.\nScore: 2075.7, Published: 2023-07-31 DOI: 10.1101/2023.07.27.23293177\nThe etiologic mechanisms of post-acute medical morbidities and unexplained symptoms (Long COVID) following SARS-CoV-2 infection are incompletely understood. There is growing evidence that viral persistence and immune dysregulation may play a major role. We performed whole-body positron emission tomography (PET) imaging in a cohort of 24 participants at time points ranging from 27 to 910 days following acute SARS-CoV-2 infection using a novel radiopharmaceutical agent, [18F]F-AraG, a highly selective tracer that allows for anatomical quantitation of activated T lymphocytes. Tracer uptake in the post-acute COVID group, which included those with and without Long COVID symptoms, was significantly higher compared to pre-pandemic controls in many anatomical regions, including the brain stem, spinal cord, bone marrow, nasopharyngeal and hilar lymphoid tissue, cardiopulmonary tissues, and gut wall. Although T cell activation tended to be higher in participants imaged closer to the time of the acute illness, tracer uptake was increased in participants imaged up to 2.5 years following SARS-CoV-2 infection. We observed that T cell activation in spinal cord and gut wall was associated with the presence of Long COVID symptoms. In addition, tracer uptake in lung tissue was higher in those with persistent pulmonary symptoms. Notably, increased T cell activation in these tissues was also observed in many individuals without Long COVID. Given the high [18F]F-AraG uptake detected in the gut, we obtained colorectal tissue for in situ hybridization SARS-CoV-2 RNA and immunohistochemical studies in a subset of participants with Long COVID symptoms. We identified cellular SARS-CoV-2 RNA in rectosigmoid lamina propria tissue in all these participants, ranging from 158 to 676 days following initial COVID-19 illness, suggesting that tissue viral persistence could be associated with long-term immunological perturbations.\nRetrospective, observational analysis of the first one hundred consecutive cases of personalized bacteriophage therapy of difficult-to-treat infections facilitated by a Belgian consortium\nAuthors: Pirnay, J.-P.; Djebara, S.; Steurs, G.; Griselain, J.; Cochez, C.; De Soir, S.; Glonti, T.; Spiessens, A.; Vanden Berghe, E.; Green, S.; Wagemans, J.; Lood, C.; Schrevens, E.; Chanishvili, N.; Kutateladze, M.; de Jode, M.; Ceyssens, P.-J.; Draye, J.-P.; Verbeken, G.; De Vos, D.; Rose, T.; Onsea, J.; Van Nieuwenhuyse, B.; Bacteriophage Therapy Providers, ; Bacteriophage Donors, ; Soentjens, P.; Lavigne, R.; Merabishvili, M.\nScore: 46.3, Published: 2023-08-28 DOI: 10.1101/2023.08.28.23294728\nIn contrast to the many reports of successful cases of personalized bacteriophage therapy, randomized controlled trials of non-personalized bacteriophage products did not bring the expected results. Here, we present the outcomes of a retrospective, observational analysis of the first 100 consecutive cases of personalized bacteriophage therapy of difficult-to-treat infections facilitated by a Belgian consortium. The most common indications were lower respiratory tract, skin \u0026 soft tissue, and bone infections, and involved combinations of 26 bacteriophages, individually selected and sometimes pre-adapted to target the causative bacterial pathogens. Clinical improvement and eradication of the targeted bacteria were reported for 77.2% and 61.3% of infections, respectively. Eradication was 70% less probable when no concomitant antibiotics were used (odds-ratio = 0.3; 95% confidence interval = 0.127 - 0.749). In vivo selection of bacteriophage resistance and in vitro bacteriophage-antibiotic synergy were documented in 43.8% (7/16 patients) and 90% (9/10) of evaluated patients, respectively. Bacteriophage immune neutralization was observed in 38.5% (5/13) of screened patients. (BT100 study, ClinicalTrials.gov registration: NCT05498363.)\nA molnupiravir-associated mutational signature in global SARS-CoV-2 genomes\nAuthors: Sanderson, T.; Hisner, R.; Donovan-Banfield, I.; Hartman, H.; Lochen, A.; Peacock, T. P.; Ruis, C.\nScore: 2447.5, Published: 2023-08-18 DOI: 10.1101/2023.01.26.23284998\nMolnupiravir, an antiviral medication that has been widely used against SARS-CoV-2, acts by inducing mutations in the virus genome during replication. Most random mutations are likely to be deleterious to the virus, and many will be lethal, and so molnupiravir-induced elevated mutation rates reduce viral load2,3. However, if some patients treated with molnupiravir do not fully clear SARS-CoV-2 infections, there could be the potential for onward transmission of molnupiravir-mutated viruses. Here we show that SARS-CoV-2 sequencing databases contain extensive evidence of molnupiravir mutagenesis. Using a systematic approach, we find that a specific class of long phylogenetic branches, distinguished by a high proportion of G-to-A and C-to-T mutations, appear almost exclusively in sequences from 2022, after the introduction of molnupiravir treatment, and in countries and age-groups with widespread usage of the drug. We identify a mutational spectrum, with preferred nucleotide contexts, from viruses in patients known to have been treated with molnupiravir and show that its signature matches that seen in these long branches, in some cases with onwards transmission of molnupiravir-derived lineages. Finally, we analyse treatment records to confirm a direct association between these high G-to-A branches and the use of molnupiravir.\nSARS-CoV-2 shedding and evolution in immunocompromised hosts during the Omicron period: a multicenter prospective analysis\nAuthors: Raglow, Z.; Surie, D.; Chappell, J. D.; Zhu, Y.; Martin, E. T.; Kwon, J. H.; Frosch, A. E.; Mohamed, A.; Gilbert, J.; Bendall, E. E.; Bahr, A.; Halasa, N.; Talbot, H. K.; Grijalva, C. G.; Baughman, A.; Womack, K. N.; Johnson, C.; Swan, S. A.; Koumans, E.; McMorrow, M. L.; Harcourt, J. L.; Atherton, L. J.; Burroughs, A.; Thornburg, N. J.; Self, W. H.; Lauring, A. S.\nScore: 26.0, Published: 2023-08-24 DOI: 10.1101/2023.08.22.23294416\nBackgroundProlonged SARS-CoV-2 infections in immunocompromised hosts may predict or source the emergence of highly mutated variants. The types of immunosuppression placing patients at highest risk for prolonged infection and associated intrahost viral evolution remain unclear. MethodsAdults aged [\u0026ge;]18 years were enrolled at 5 hospitals and followed from 4/11/2022 - 2/1/2023. Eligible patients were SARS-CoV-2-positive in the previous 14 days and had a moderate or severely immunocompromising condition or treatment. Nasal specimens were tested by rRT-PCR every 2-4 weeks until negative in consecutive specimens. Positive specimens underwent viral culture and whole genome sequencing. A Cox proportional hazards model was used to assess factors associated with duration of infection. ResultsWe enrolled 150 patients with: B cell malignancy or anti-B cell therapy (n=18), solid organ or hematopoietic stem cell transplant (SOT/HSCT) (n=59), AIDS (n=5), non-B cell malignancy (n=23), and autoimmune/autoinflammatory conditions (n=45). Thirty-eight (25%) were rRT-PCR-positive and 12 (8%) were culture-positive [\u0026ge;]21 days after initial SARS-CoV-2 detection or illness onset. Patients with B cell dysfunction had longer duration of rRT-PCR- positivity compared to those with autoimmune/autoinflammatory conditions (aHR 0.32, 95% CI 0.15-0.64). Consensus (\u003e50% frequency) spike mutations were identified in 5 individuals who were rRT-PCR-positive \u003e56 days; 61% were in the receptor-binding domain (RBD). Mutations shared by multiple individuals were rare (\u003c5%) in global circulation. ConclusionsIn this cohort, prolonged replication-competent Omicron SARS-CoV-2 infections were uncommon. Within-host evolutionary rates were similar across patients, but individuals with infections lasting \u003e56 days accumulated spike mutations, which were distinct from those seen globally.\nPublished benefits of ivermectin use in Itajai, Brazil for COVID-19 infection, hospitalisation, and mortality are entirely explained by statistical artefacts\nAuthors: Mills, R.; Pecanha Antonio, A. C.; Tucker-Kellogg, G.\nScore: 285.6, Published: 2023-08-15 DOI: 10.1101/2023.08.10.23293924\nBackgroundTwo recent publications by Kerr et al. (Cureus 14(1):e21272; Cureus 14(8): e28624) reported dramatic effects of prophylactic ivermectin use for both prevention of COVID-19 and reduction of COVID-19-related hospitalisation and mortality, including a dose-dependent effect of ivermectin prophylaxis. These papers have gained an unusually large public influence: they were incorporated into debates around COVID-19 policies and may have contributed to decreased trust in vaccine efficacy and public health authorities more broadly. Both studies were based on retrospective observational analysis of city-wide registry data from the city of Itajai, Brazil from July-December 2020. MethodsStarting with initially identified sources of error, we conducted a revised statistical analysis of available data, including data made available with the original papers and public data from the Brazil Ministry of Health. We identified additional uncorrected sources of bias and errors from the original analysis, including incorrect subject exclusion and missing subjects, an enrolment time bias, and multiple sources of immortal time bias. In models assuming no actual effect from ivermectin use, we conducted Monte Carlo simulations to estimate the contribution of these biases to any observed effect. ResultsUntreated statistical artefacts and methodological errors alone lead to dramatic apparent risk reduction associated with Ivermectin use in both studies. The magnitude of apparent risk reduction from these artefacts is comparable to the results reported by the studies themselves, including apparent protection from infection, hospitalisation, and death, and including the reported apparent dose-response relationship. ConclusionsThe inference of ivermectin efficacy reported in both papers is unsupported, as the observed effects are entirely explained by untreated statistical artefacts and methodological errors. Our re-analysis calls for caution in interpreting highly publicised observational studies and highlights the importance of common sources of bias in clinical research.\nSARS-CoV-2 Infection Biomarkers Reveal an Extended RSAD2 Dependant Metabolic Pathway\nAuthors: Sala, S.; Nitschke, P.; Masuda, R.; Gray, N.; Lawler, N.; Wood, J. M.; Butler, J. N.; Berezhnoy, G.; Bolanos, A.; Boughton, B. A.; Lonati, C.; Roessler, T.; Singh, Y.; Wilson, I. D.; Lodge, S.; Morillon, A.-C.; Loo, R. L.; Hall, D.; Whiley, L.; Evans, G. B.; Grove, T. L.; Almo, S. C.; Harris, L. D.; Holmes, E.; Merle, U.; Trautwein, C.; Nicholson, J. K.; Wist, J.\nScore: 27.4, Published: 2023-08-23 DOI: 10.1101/2023.05.08.23289637\nWe present compelling evidence for the existence of an extended innate viperin dependent pathway which provides crucial evidence for an adaptive response to viral agents like SARS-CoV-2. We show the in vivo biosynthesis of a family of endogenous cytosine metabolites with potential antiviral activity. Two dimensional Nuclear magnetic resonance (NMR) spectroscopy revealed a characteristic spin-system motif indicating the presence of an extended panel of urinary metabolites during the acute viral replication phase. Mass spectrometry additionally allowed the characterization and quantification of the most abundant serum metabolites showing potential diagnostic value of the compounds for viral infections. In total, we unveiled ten nucleoside (cytosine and uracil based) analogue structures, eight of which were previously unknown in humans. The molecular structures of the nucleoside analogues and their correlation with an array of serum cytokines, including IFN-2, IFN-{gamma} and IL-10, suggest an association with the viperin enzyme contributing to an endogenous innate immune defence mechanism against viral infection.\nDistinct COVID-19 vaccine combinations result in divergent immune responses\nAuthors: Zaeck, L. M.; Tan, N. H.; Rietdijk, W. J. R.; Geers, D.; Sablerolles, R. S. G.; Bogers, S.; van Dijk, L. L. A.; Gommers, L.; van Leeuwen, L. P. M.; Rugebregt, S.; Goorhuis, A.; Postma, D. F.; Visser, L. G.; Dalm, V. A. S. H.; Lafeber, M.; Kootstra, N. A.; Huckriede, A. L. W.; Haagmans, B. L.; van Baarle, D.; Koopmans, M. P. G.; van der Kuy, P. H. M.; GeurtsvanKessel, C. H.; de Vries, R. D.; SWITCH-ON Research Group, Score: 16.5, Published: 2023-08-25 DOI: 10.1101/2023.08.25.23294606\nWaning antibody responses after COVID-19 vaccination combined with the emergence of the SARS-CoV-2 Omicron lineage led to reduced vaccine effectiveness. As a countermeasure, bivalent mRNA-based booster vaccines encoding the ancestral spike protein in combination with that of Omicron BA.1 or BA.5 were introduced. Since then, BA.2-descendent lineages have become dominant, such as XBB.1.5 or BA.2.86. Here, we assessed how different COVID-19 priming regimens affect the immunogenicity of the recently used bivalent booster vaccinations and breakthrough infections. BA.1 and BA.5 bivalent vaccines boosted neutralizing antibodies and T-cells up to 3 months after boost; however, cross-neutralization of XBB.1.5 was poor. Interestingly, different combinations of prime-boost regimens induced divergent responses: participants primed with Ad26.COV2.S developed lower binding antibody levels after bivalent boost while neutralization and T-cell responses were similar to mRNA-based primed participants. In contrast, the breadth of neutralization was higher in mRNA-primed and bivalent BA.5 boosted participants. Combined, we highlight important lessons learned from the employed COVID-19 vaccination strategies. Our data further support the use of monovalent vaccines based on circulating strains when vaccinating risk groups, as recently recommended by the WHO. We emphasize the importance of the continuous assessment of immune responses targeting circulating variants to guide future COVID-19 vaccination policies.\n",
  "wordCount" : "2885",
  "inLanguage": "en",
  "datePublished": "2023-08-30T10:38:23Z",
  "dateModified": "2023-08-30T10:38:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta"><span>updated on August 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.23293434">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.23293434" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.23293434">
        <p class="paperTitle">Safety and Immunogenicity of XBB.1.5-Containing mRNA Vaccines</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.23293434" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.23293434" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chalkias, S.; McGhee, N.; Whatley, J. L.; Essink, B.; Brosz, A.; Tomassini, J. E.; Girard, B.; Edwards, D. K.; Nasir, A.; Lee, D.; Avena, L. E.; Feng, J.; Deng, W.; Montefiori, D. C.; Baden, L. R.; Miller, J. M.; Das, R.</p>
        <p class="info">Score: 493.0, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.23293434' target='https://doi.org/10.1101/2023.08.22.23293434'> 10.1101/2023.08.22.23293434</a></p>
        <p class="abstract">BackgroundSubvariants of the severe acute respiratory syndrome coronavirus (SARS-CoV-2) omicron XBB-lineage have the potential to escape immunity provided by prior infection or vaccination. For Covid-19 immunizations beginning in the Fall 2023, the U.S. FDA has recommended updating to a monovalent omicron XBB.1.5-containing vaccine.

MethodsIn this ongoing, phase 2/3 study participants were randomized 1:1 to receive 50-{micro}g doses of mRNA-1273.815 monovalent (50-{micro}g omicron XBB.1.5 spike mRNA) or mRNA-1273.231 bivalent (25-{micro}g omicron XBB.1.5 and 25-{micro}g omicron BA.4/BA.5 spike mRNAs vaccines, administered as 5th doses, to adults who previously received a primary series and 3rd dose of an original mRNA coronavirus disease 2019 (Covid-19) vaccine, and a 4th dose of a bivalent (omicron BA.4/BA.5 and original SARS-CoV-2) vaccine. Interim safety and immunogenicity data 15 days post-vaccination are presented.

ResultsIn April 2023, participants received mRNA-1273.815 (n=50) and mRNA-1273.231 (n=51). The median intervals from the prior dose of BA.4/BA.5-containing bivalent vaccine were 8.2 and 8.3 months for the mRNA-1273.815 and mRNA-1273.231 groups, respectively. Both vaccines increased neutralizing antibody (nAb) geometric mean titers against all variants tested at day 15 post-booster nAb compared to pre-booster levels. Geometric mean fold-rises from pre-booster titers after the monovalent booster were higher against XBB.1.5, XBB.1.16 and SARS-CoV-2 (D614G) than those of the bivalent booster and were comparable against BA.4/BA.5 and BQ1.1 variants for both vaccines. The monovalent vaccine also elicited nAb responses against omicron XBB.2.3.2, EG.5.1, and FL.1.5.1 that were similar to those seen for XBB.1.5 in a subset (n=20) of participants. The occurrence of solicited adverse reactions and unsolicited adverse events were overall similar to those previously reported for the original mRNA-1273 50-{micro}g and omicron BA.4/BA.5-containing bivalent mRNA-1273 vaccines.

ConclusionIn this interim analysis, XBB.1.5-containing monovalent and bivalent vaccines elicited potent neutralizing responses against variants of the omicron XBB-lineage (XBB.1.5, XBB.1.6, XBB.2.3.2) as well as more recent (EG.5.1, FL.1.5.1) variants. The safety profile of the XBB.1.5-containing vaccine was consistent with those of prior vaccines. These results support the Covid-19 vaccine update in 2023-2024 to a monovalent XBB.1.5-containing vaccine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.26.23294679">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.26.23294679" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.26.23294679">
        <p class="paperTitle">Kinetics and durability of humoral responses to SARS-CoV-2 infection and vaccination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.26.23294679" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.26.23294679" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Srivastava, K.; Carreno Quiroz, J. M.; Gleason, C.; Monahan, B.; Singh, G.; Abbad, A.; Tcheou, J.; Raskin, A.; Kleiner, G.; van Bakel, H.; Sordillo, E. M.; PARIS Study Group,  ; Krammer, F. M.; Simon, V.</p>
        <p class="info">Score: 327.3, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.26.23294679' target='https://doi.org/10.1101/2023.08.26.23294679'> 10.1101/2023.08.26.23294679</a></p>
        <p class="abstract">We analyzed the kinetics and durability of the humoral responses to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection and vaccination using &gt;8,000 longitudinal samples collected over a three-year period (April 2020 to April 2023) in the New York City metropolitan area. Upon primary immunization, participants with pre-existing immunity mounted higher antibody responses faster and achieved higher steady-state levels compared to naive individuals. Antibody durability was characterized by two phases: an initial rapid decay, followed by a phase of stabilization with very slow decay resulting in an individual spike binding antibody steady state. Booster vaccination equalized the differences in antibody levels between participants with and without hybrid immunity, but the antibody titers reached decreased with each successive antigen exposure. Break-through infections increased antibody titers to similar levels as an additional vaccine dose in naive individuals. Our study provides strong evidence for the fact that SARS-CoV-2 antibody responses are long lasting, with an initial waning phase followed by a stabilization phase.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.23293776">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.23293776" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.23293776">
        <p class="paperTitle">Post-COVID conditions during Delta and early-Omicron SARS-CoV-2 variant periods among adults in the United States</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.23293776" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.23293776" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Edwards, D. L.; Logan, P.; Feldstein, L.; Accorsi, E.; Fairlie, T.; Saydah, S.</p>
        <p class="info">Score: 212.3, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.23293776' target='https://doi.org/10.1101/2023.08.09.23293776'> 10.1101/2023.08.09.23293776</a></p>
        <p class="abstract">BackgroundPost-COVID conditions after infection with new SARS-CoV-2 variants have been incompletely described. We compared the prevalence and risk factors for ongoing symptoms lasting 4 weeks or longer (often referred to as post-COVID Conditions) among adults who had tested positive vs. negative during the Delta and early-Omicron periods.

MethodsSelf-reported survey data regarding symptoms and previous SARS-CoV 2 test results were collected from May 31 - July 6, 2022, from a probability sampling of United States adults. Respondents were classified according to their test result, predominant circulating variant when respondents first tested positive (Delta vs early-Omicron), and demographic risk factors.

ResultsAmong 2,421 respondents, 256 tested positive during Delta, 460 during early-Omicron, and 1,705 always tested negative. Nearly one-fourth (22.3%) of negative respondents reported [&amp;ge;]1symptom that lasted [&amp;ge;]4 weeks, compared to 60.6% (p&lt;0.05) of respondents who tested positive during the Delta period and 47.8% (p&lt;0.05) during the early-Omicron period. Fatigue, change in smell/taste, and cough were commonly reported by respondents who tested positive. Demographic risk factors associated with ongoing symptoms were being female and unemployed (aOR 1.28, 95% CI 1.06-1.55; aOR 1.48, 95% CI: 1.17-1.87).

ConclusionThe reported occurrence of ongoing symptoms associated with post-COVID conditions was reduced during the early-Omicron period, compared with Delta.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.23293177">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.23293177" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.23293177">
        <p class="paperTitle">Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.23293177" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.23293177" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A.; Dalhuisen, T.; Durstenfeld, M. S.; Hsue, P. Y.; Kelly, J. D.; Kumar, N.; Martin, J. N.; Gambhir, A.; Somsouk, M.; Seo, Y.; Deeks, S. G.; Laszik, Z. G.; VanBrocklin, H. F.; Henrich, T. J.</p>
        <p class="info">Score: 2075.7, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.23293177' target='https://doi.org/10.1101/2023.07.27.23293177'> 10.1101/2023.07.27.23293177</a></p>
        <p class="abstract">The etiologic mechanisms of post-acute medical morbidities and unexplained symptoms (Long COVID) following SARS-CoV-2 infection are incompletely understood. There is growing evidence that viral persistence and immune dysregulation may play a major role. We performed whole-body positron emission tomography (PET) imaging in a cohort of 24 participants at time points ranging from 27 to 910 days following acute SARS-CoV-2 infection using a novel radiopharmaceutical agent, [18F]F-AraG, a highly selective tracer that allows for anatomical quantitation of activated T lymphocytes. Tracer uptake in the post-acute COVID group, which included those with and without Long COVID symptoms, was significantly higher compared to pre-pandemic controls in many anatomical regions, including the brain stem, spinal cord, bone marrow, nasopharyngeal and hilar lymphoid tissue, cardiopulmonary tissues, and gut wall. Although T cell activation tended to be higher in participants imaged closer to the time of the acute illness, tracer uptake was increased in participants imaged up to 2.5 years following SARS-CoV-2 infection. We observed that T cell activation in spinal cord and gut wall was associated with the presence of Long COVID symptoms. In addition, tracer uptake in lung tissue was higher in those with persistent pulmonary symptoms. Notably, increased T cell activation in these tissues was also observed in many individuals without Long COVID. Given the high [18F]F-AraG uptake detected in the gut, we obtained colorectal tissue for in situ hybridization SARS-CoV-2 RNA and immunohistochemical studies in a subset of participants with Long COVID symptoms. We identified cellular SARS-CoV-2 RNA in rectosigmoid lamina propria tissue in all these participants, ranging from 158 to 676 days following initial COVID-19 illness, suggesting that tissue viral persistence could be associated with long-term immunological perturbations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.23294728">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.23294728" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.23294728">
        <p class="paperTitle">Retrospective, observational analysis of the first one hundred consecutive cases of personalized bacteriophage therapy of difficult-to-treat infections facilitated by a Belgian consortium</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.23294728" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.23294728" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pirnay, J.-P.; Djebara, S.; Steurs, G.; Griselain, J.; Cochez, C.; De Soir, S.; Glonti, T.; Spiessens, A.; Vanden Berghe, E.; Green, S.; Wagemans, J.; Lood, C.; Schrevens, E.; Chanishvili, N.; Kutateladze, M.; de Jode, M.; Ceyssens, P.-J.; Draye, J.-P.; Verbeken, G.; De Vos, D.; Rose, T.; Onsea, J.; Van Nieuwenhuyse, B.; Bacteriophage Therapy Providers,  ; Bacteriophage Donors,  ; Soentjens, P.; Lavigne, R.; Merabishvili, M.</p>
        <p class="info">Score: 46.3, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.23294728' target='https://doi.org/10.1101/2023.08.28.23294728'> 10.1101/2023.08.28.23294728</a></p>
        <p class="abstract">In contrast to the many reports of successful cases of personalized bacteriophage therapy, randomized controlled trials of non-personalized bacteriophage products did not bring the expected results. Here, we present the outcomes of a retrospective, observational analysis of the first 100 consecutive cases of personalized bacteriophage therapy of difficult-to-treat infections facilitated by a Belgian consortium. The most common indications were lower respiratory tract, skin &amp; soft tissue, and bone infections, and involved combinations of 26 bacteriophages, individually selected and sometimes pre-adapted to target the causative bacterial pathogens. Clinical improvement and eradication of the targeted bacteria were reported for 77.2% and 61.3% of infections, respectively. Eradication was 70% less probable when no concomitant antibiotics were used (odds-ratio = 0.3; 95% confidence interval = 0.127 - 0.749). In vivo selection of bacteriophage resistance and in vitro bacteriophage-antibiotic synergy were documented in 43.8% (7/16 patients) and 90% (9/10) of evaluated patients, respectively. Bacteriophage immune neutralization was observed in 38.5% (5/13) of screened patients. (BT100 study, ClinicalTrials.gov registration: NCT05498363.)</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.26.23284998">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.26.23284998" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.26.23284998">
        <p class="paperTitle">A molnupiravir-associated mutational signature in global SARS-CoV-2 genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.26.23284998" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.26.23284998" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanderson, T.; Hisner, R.; Donovan-Banfield, I.; Hartman, H.; Lochen, A.; Peacock, T. P.; Ruis, C.</p>
        <p class="info">Score: 2447.5, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.26.23284998' target='https://doi.org/10.1101/2023.01.26.23284998'> 10.1101/2023.01.26.23284998</a></p>
        <p class="abstract">Molnupiravir, an antiviral medication that has been widely used against SARS-CoV-2, acts by inducing mutations in the virus genome during replication. Most random mutations are likely to be deleterious to the virus, and many will be lethal, and so molnupiravir-induced elevated mutation rates reduce viral load2,3. However, if some patients treated with molnupiravir do not fully clear SARS-CoV-2 infections, there could be the potential for onward transmission of molnupiravir-mutated viruses. Here we show that SARS-CoV-2 sequencing databases contain extensive evidence of molnupiravir mutagenesis. Using a systematic approach, we find that a specific class of long phylogenetic branches, distinguished by a high proportion of G-to-A and C-to-T mutations, appear almost exclusively in sequences from 2022, after the introduction of molnupiravir treatment, and in countries and age-groups with widespread usage of the drug. We identify a mutational spectrum, with preferred nucleotide contexts, from viruses in patients known to have been treated with molnupiravir and show that its signature matches that seen in these long branches, in some cases with onwards transmission of molnupiravir-derived lineages. Finally, we analyse treatment records to confirm a direct association between these high G-to-A branches and the use of molnupiravir.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.23294416">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.23294416" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.23294416">
        <p class="paperTitle">SARS-CoV-2 shedding and evolution in immunocompromised hosts during the Omicron period: a multicenter prospective analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.23294416" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.23294416" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Raglow, Z.; Surie, D.; Chappell, J. D.; Zhu, Y.; Martin, E. T.; Kwon, J. H.; Frosch, A. E.; Mohamed, A.; Gilbert, J.; Bendall, E. E.; Bahr, A.; Halasa, N.; Talbot, H. K.; Grijalva, C. G.; Baughman, A.; Womack, K. N.; Johnson, C.; Swan, S. A.; Koumans, E.; McMorrow, M. L.; Harcourt, J. L.; Atherton, L. J.; Burroughs, A.; Thornburg, N. J.; Self, W. H.; Lauring, A. S.</p>
        <p class="info">Score: 26.0, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.23294416' target='https://doi.org/10.1101/2023.08.22.23294416'> 10.1101/2023.08.22.23294416</a></p>
        <p class="abstract">BackgroundProlonged SARS-CoV-2 infections in immunocompromised hosts may predict or source the emergence of highly mutated variants. The types of immunosuppression placing patients at highest risk for prolonged infection and associated intrahost viral evolution remain unclear.

MethodsAdults aged [&amp;ge;]18 years were enrolled at 5 hospitals and followed from 4/11/2022 - 2/1/2023. Eligible patients were SARS-CoV-2-positive in the previous 14 days and had a moderate or severely immunocompromising condition or treatment. Nasal specimens were tested by rRT-PCR every 2-4 weeks until negative in consecutive specimens. Positive specimens underwent viral culture and whole genome sequencing. A Cox proportional hazards model was used to assess factors associated with duration of infection.

ResultsWe enrolled 150 patients with: B cell malignancy or anti-B cell therapy (n=18), solid organ or hematopoietic stem cell transplant (SOT/HSCT) (n=59), AIDS (n=5), non-B cell malignancy (n=23), and autoimmune/autoinflammatory conditions (n=45). Thirty-eight (25%) were rRT-PCR-positive and 12 (8%) were culture-positive [&amp;ge;]21 days after initial SARS-CoV-2 detection or illness onset. Patients with B cell dysfunction had longer duration of rRT-PCR- positivity compared to those with autoimmune/autoinflammatory conditions (aHR 0.32, 95% CI 0.15-0.64). Consensus (&gt;50% frequency) spike mutations were identified in 5 individuals who were rRT-PCR-positive &gt;56 days; 61% were in the receptor-binding domain (RBD). Mutations shared by multiple individuals were rare (&lt;5%) in global circulation.

ConclusionsIn this cohort, prolonged replication-competent Omicron SARS-CoV-2 infections were uncommon. Within-host evolutionary rates were similar across patients, but individuals with infections lasting &gt;56 days accumulated spike mutations, which were distinct from those seen globally.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.23293924">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.23293924" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.23293924">
        <p class="paperTitle">Published benefits of ivermectin use in Itajai, Brazil for COVID-19 infection, hospitalisation, and mortality are entirely explained by statistical artefacts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.23293924" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.23293924" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mills, R.; Pecanha Antonio, A. C.; Tucker-Kellogg, G.</p>
        <p class="info">Score: 285.6, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.23293924' target='https://doi.org/10.1101/2023.08.10.23293924'> 10.1101/2023.08.10.23293924</a></p>
        <p class="abstract">BackgroundTwo recent publications by Kerr et al. (Cureus 14(1):e21272; Cureus 14(8): e28624) reported dramatic effects of prophylactic ivermectin use for both prevention of COVID-19 and reduction of COVID-19-related hospitalisation and mortality, including a dose-dependent effect of ivermectin prophylaxis. These papers have gained an unusually large public influence: they were incorporated into debates around COVID-19 policies and may have contributed to decreased trust in vaccine efficacy and public health authorities more broadly. Both studies were based on retrospective observational analysis of city-wide registry data from the city of Itajai, Brazil from July-December 2020.

MethodsStarting with initially identified sources of error, we conducted a revised statistical analysis of available data, including data made available with the original papers and public data from the Brazil Ministry of Health. We identified additional uncorrected sources of bias and errors from the original analysis, including incorrect subject exclusion and missing subjects, an enrolment time bias, and multiple sources of immortal time bias. In models assuming no actual effect from ivermectin use, we conducted Monte Carlo simulations to estimate the contribution of these biases to any observed effect.

ResultsUntreated statistical artefacts and methodological errors alone lead to dramatic apparent risk reduction associated with Ivermectin use in both studies. The magnitude of apparent risk reduction from these artefacts is comparable to the results reported by the studies themselves, including apparent protection from infection, hospitalisation, and death, and including the reported apparent dose-response relationship.

ConclusionsThe inference of ivermectin efficacy reported in both papers is unsupported, as the observed effects are entirely explained by untreated statistical artefacts and methodological errors. Our re-analysis calls for caution in interpreting highly publicised observational studies and highlights the importance of common sources of bias in clinical research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.08.23289637">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.08.23289637" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.08.23289637">
        <p class="paperTitle">SARS-CoV-2 Infection Biomarkers Reveal an Extended RSAD2 Dependant Metabolic Pathway</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.08.23289637" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.08.23289637" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sala, S.; Nitschke, P.; Masuda, R.; Gray, N.; Lawler, N.; Wood, J. M.; Butler, J. N.; Berezhnoy, G.; Bolanos, A.; Boughton, B. A.; Lonati, C.; Roessler, T.; Singh, Y.; Wilson, I. D.; Lodge, S.; Morillon, A.-C.; Loo, R. L.; Hall, D.; Whiley, L.; Evans, G. B.; Grove, T. L.; Almo, S. C.; Harris, L. D.; Holmes, E.; Merle, U.; Trautwein, C.; Nicholson, J. K.; Wist, J.</p>
        <p class="info">Score: 27.4, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.08.23289637' target='https://doi.org/10.1101/2023.05.08.23289637'> 10.1101/2023.05.08.23289637</a></p>
        <p class="abstract">We present compelling evidence for the existence of an extended innate viperin dependent pathway which provides crucial evidence for an adaptive response to viral agents like SARS-CoV-2. We show the in vivo biosynthesis of a family of endogenous cytosine metabolites with potential antiviral activity. Two dimensional Nuclear magnetic resonance (NMR) spectroscopy revealed a characteristic spin-system motif indicating the presence of an extended panel of urinary metabolites during the acute viral replication phase. Mass spectrometry additionally allowed the characterization and quantification of the most abundant serum metabolites showing potential diagnostic value of the compounds for viral infections. In total, we unveiled ten nucleoside (cytosine and uracil based) analogue structures, eight of which were previously unknown in humans. The molecular structures of the nucleoside analogues and their correlation with an array of serum cytokines, including IFN-2, IFN-{gamma} and IL-10, suggest an association with the viperin enzyme contributing to an endogenous innate immune defence mechanism against viral infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.23294606">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.23294606" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.23294606">
        <p class="paperTitle">Distinct COVID-19 vaccine combinations result in divergent immune responses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.23294606" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.23294606" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zaeck, L. M.; Tan, N. H.; Rietdijk, W. J. R.; Geers, D.; Sablerolles, R. S. G.; Bogers, S.; van Dijk, L. L. A.; Gommers, L.; van Leeuwen, L. P. M.; Rugebregt, S.; Goorhuis, A.; Postma, D. F.; Visser, L. G.; Dalm, V. A. S. H.; Lafeber, M.; Kootstra, N. A.; Huckriede, A. L. W.; Haagmans, B. L.; van Baarle, D.; Koopmans, M. P. G.; van der Kuy, P. H. M.; GeurtsvanKessel, C. H.; de Vries, R. D.; SWITCH-ON Research Group,  </p>
        <p class="info">Score: 16.5, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.23294606' target='https://doi.org/10.1101/2023.08.25.23294606'> 10.1101/2023.08.25.23294606</a></p>
        <p class="abstract">Waning antibody responses after COVID-19 vaccination combined with the emergence of the SARS-CoV-2 Omicron lineage led to reduced vaccine effectiveness. As a countermeasure, bivalent mRNA-based booster vaccines encoding the ancestral spike protein in combination with that of Omicron BA.1 or BA.5 were introduced. Since then, BA.2-descendent lineages have become dominant, such as XBB.1.5 or BA.2.86. Here, we assessed how different COVID-19 priming regimens affect the immunogenicity of the recently used bivalent booster vaccinations and breakthrough infections. BA.1 and BA.5 bivalent vaccines boosted neutralizing antibodies and T-cells up to 3 months after boost; however, cross-neutralization of XBB.1.5 was poor. Interestingly, different combinations of prime-boost regimens induced divergent responses: participants primed with Ad26.COV2.S developed lower binding antibody levels after bivalent boost while neutralization and T-cell responses were similar to mRNA-based primed participants. In contrast, the breadth of neutralization was higher in mRNA-primed and bivalent BA.5 boosted participants. Combined, we highlight important  lessons learned from the employed COVID-19 vaccination strategies. Our data further support the use of monovalent vaccines based on circulating strains when vaccinating risk groups, as recently recommended by the WHO. We emphasize the importance of the continuous assessment of immune responses targeting circulating variants to guide future COVID-19 vaccination policies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
