<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Published benefits of ivermectin use in Itajai, Brazil for COVID-19 infection, hospitalisation, and mortality are entirely explained by statistical artefacts
Authors: Mills, R.; Pecanha Antonio, A. C.; Tucker-Kellogg, G.
Score: 265.2, Published: 2023-08-15 DOI: 10.1101/2023.08.10.23293924
BackgroundTwo recent publications by Kerr et al. (Cureus 14(1):e21272; Cureus 14(8): e28624) reported dramatic effects of prophylactic ivermectin use for both prevention of COVID-19 and reduction of COVID-19-related hospitalisation and mortality, including a dose-dependent effect of ivermectin prophylaxis.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Published benefits of ivermectin use in Itajai, Brazil for COVID-19 infection, hospitalisation, and mortality are entirely explained by statistical artefacts
Authors: Mills, R.; Pecanha Antonio, A. C.; Tucker-Kellogg, G.
Score: 265.2, Published: 2023-08-15 DOI: 10.1101/2023.08.10.23293924
BackgroundTwo recent publications by Kerr et al. (Cureus 14(1):e21272; Cureus 14(8): e28624) reported dramatic effects of prophylactic ivermectin use for both prevention of COVID-19 and reduction of COVID-19-related hospitalisation and mortality, including a dose-dependent effect of ivermectin prophylaxis." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-20T10:35:54+00:00" />
<meta property="article:modified_time" content="2023-08-20T10:35:54+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Published benefits of ivermectin use in Itajai, Brazil for COVID-19 infection, hospitalisation, and mortality are entirely explained by statistical artefacts
Authors: Mills, R.; Pecanha Antonio, A. C.; Tucker-Kellogg, G.
Score: 265.2, Published: 2023-08-15 DOI: 10.1101/2023.08.10.23293924
BackgroundTwo recent publications by Kerr et al. (Cureus 14(1):e21272; Cureus 14(8): e28624) reported dramatic effects of prophylactic ivermectin use for both prevention of COVID-19 and reduction of COVID-19-related hospitalisation and mortality, including a dose-dependent effect of ivermectin prophylaxis."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Published benefits of ivermectin use in Itajai, Brazil for COVID-19 infection, hospitalisation, and mortality are entirely explained by statistical artefacts\nAuthors: Mills, R.; Pecanha Antonio, A. C.; Tucker-Kellogg, G.\nScore: 265.2, Published: 2023-08-15 DOI: 10.1101/2023.08.10.23293924\nBackgroundTwo recent publications by Kerr et al. (Cureus 14(1):e21272; Cureus 14(8): e28624) reported dramatic effects of prophylactic ivermectin use for both prevention of COVID-19 and reduction of COVID-19-related hospitalisation and mortality, including a dose-dependent effect of ivermectin prophylaxis.",
  "keywords": [
    
  ],
  "articleBody": " Published benefits of ivermectin use in Itajai, Brazil for COVID-19 infection, hospitalisation, and mortality are entirely explained by statistical artefacts\nAuthors: Mills, R.; Pecanha Antonio, A. C.; Tucker-Kellogg, G.\nScore: 265.2, Published: 2023-08-15 DOI: 10.1101/2023.08.10.23293924\nBackgroundTwo recent publications by Kerr et al. (Cureus 14(1):e21272; Cureus 14(8): e28624) reported dramatic effects of prophylactic ivermectin use for both prevention of COVID-19 and reduction of COVID-19-related hospitalisation and mortality, including a dose-dependent effect of ivermectin prophylaxis. These papers have gained an unusually large public influence: they were incorporated into debates around COVID-19 policies and may have contributed to decreased trust in vaccine efficacy and public health authorities more broadly. Both studies were based on retrospective observational analysis of city-wide registry data from the city of Itajai, Brazil from July-December 2020. MethodsStarting with initially identified sources of error, we conducted a revised statistical analysis of available data, including data made available with the original papers and public data from the Brazil Ministry of Health. We identified additional uncorrected sources of bias and errors from the original analysis, including incorrect subject exclusion and missing subjects, an enrolment time bias, and multiple sources of immortal time bias. In models assuming no actual effect from ivermectin use, we conducted Monte Carlo simulations to estimate the contribution of these biases to any observed effect. ResultsUntreated statistical artefacts and methodological errors alone lead to dramatic apparent risk reduction associated with Ivermectin use in both studies. The magnitude of apparent risk reduction from these artefacts is comparable to the results reported by the studies themselves, including apparent protection from infection, hospitalisation, and death, and including the reported apparent dose-response relationship. ConclusionsThe inference of ivermectin efficacy reported in both papers is unsupported, as the observed effects are entirely explained by untreated statistical artefacts and methodological errors. Our re-analysis calls for caution in interpreting highly publicised observational studies and highlights the importance of common sources of bias in clinical research.\nA molnupiravir-associated mutational signature in global SARS-CoV-2 genomes\nAuthors: Sanderson, T.; Hisner, R.; Donovan-Banfield, I.; Hartman, H.; Lochen, A.; Peacock, T. P.; Ruis, C.\nScore: 2360.4, Published: 2023-08-18 DOI: 10.1101/2023.01.26.23284998\nMolnupiravir, an antiviral medication that has been widely used against SARS-CoV-2, acts by inducing mutations in the virus genome during replication. Most random mutations are likely to be deleterious to the virus, and many will be lethal, and so molnupiravir-induced elevated mutation rates reduce viral load. However, if some patients treated with molnupiravir do not fully clear SARS-CoV-2 infections, there could be the potential for onward transmission of molnupiravir-mutated viruses. Here we show that SARS-CoV-2 sequencing databases contain extensive evidence of molnupiravir mutagenesis. Using a systematic approach, we find that a specific class of long phylogenetic branches, distinguished by a high proportion of G-to-A and C-to-T mutations, appear almost exclusively in sequences from 2022, after the introduction of molnupiravir treatment, and in countries and age-groups with widespread usage of the drug. We identify a mutational spectrum, with preferred nucleotide contexts, from viruses in patients known to have been treated with molnupiravir and show that its signature matches that seen in these long branches, in some cases with onwards transmission of molnupiravir-derived lineages. Finally, we analyse treatment records to confirm a direct association between these high G-to-A branches and the use of molnupiravir.\nWaning of post-vaccination neutralizing antibody responses against SARS-CoV-2, a systematic literature review and meta-analysis\nAuthors: Jacobsen, H.; Sitaras, I.; Katzmarzyk, M.; Cobos Jimenez, V.; Naughton, R.; Higdon, M. M.; Deloria Knoll, M.\nScore: 71.8, Published: 2023-08-10 DOI: 10.1101/2023.08.08.23293864\nBackgroundMass COVID-19 vaccination and the continuous introduction of new viral variants of SARS-CoV-2, especially of Omicron subvariants, has resulted in an increase in the proportion of the population with hybrid immunity at various stages of waning protection. We systematically reviewed waning of post-vaccination neutralizing antibody titers in different immunological settings to investigate potential differences. MethodsWe searched for studies providing data for post-vaccination neutralizing antibody responses against SARS-CoV-2 in PubMed, bioRxiv, and medRxiv from Dec 15, 2021, to Jan 31, 2023, using keywords related to COVID-19, vaccination, and antibody neutralization. We used random effects meta-regression to estimate the average fold-reduction in post-vaccination neutralizing antibody titers against the Index strain or Omicron BA.1. from month 1 to month 6 post last dose, stratified by vaccination regimen (primary or booster) and infection-naive vs hybrid-immune status. FindingsIn total, 26 studies reporting longitudinal post-vaccination neutralizing antibody titers were included. Neutralization titers against the Index variant were available from all studies for infection-naive participants, and from nine for hybrid-immune participants. Against Omicron BA.1, nine and eight studies were available for infection-naive and hybrid-immune cohorts, respectively. In infection-naive cohorts, post-vaccination neutralization titers against the Index strain waned 5.1-fold (95% CI 3.4-7.8) from month 1 to month 6 following primary regimen and 3.8-fold (95% CI 2.4-5.9) following the booster. Titers against Omicron BA.1 waned 5.9-fold (95% CI 3.8-9.0) in infection-naive, post-booster cohorts. In hybrid-immune, post-primary vaccination cohorts, titers waned 3.7-fold (95% CI 1.7-7.9) against the Index strain and 5.0-fold (95% CI 1.1-21.8) against Omicron BA.1. InterpretationNo obvious differences in waning between post-primary or post-boost vaccination were observed for vaccines used widely to date, nor between infection-naive and hybrid-immune participants. Titers against Omicron BA.1 may wane faster compared to Index titers, which may worsen for more recent Omicron sub-variants and should be monitored. Relatively small datasets limit the precision of our current analysis; further investigation is needed when more data become available. However, based on our current findings, striking differences in waning for the analyzed and future comparisons are unlikely.\nMultimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19\nAuthors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A.; Dalhuisen, T.; Durstenfeld, M. S.; Hsue, P. Y.; Kelly, J. D.; Kumar, N.; Martin, J. N.; Gambhir, A.; Somsouk, M.; Seo, Y.; Deeks, S. G.; Laszik, Z. G.; VanBrocklin, H. F.; Henrich, T. J.\nScore: 1760.8, Published: 2023-07-31 DOI: 10.1101/2023.07.27.23293177\nThe etiologic mechanisms of post-acute medical morbidities and unexplained symptoms (Long COVID) following SARS-CoV-2 infection are incompletely understood. There is growing evidence that viral persistence and immune dysregulation may play a major role. We performed whole-body positron emission tomography (PET) imaging in a cohort of 24 participants at time points ranging from 27 to 910 days following acute SARS-CoV-2 infection using a novel radiopharmaceutical agent, [18F]F-AraG, a highly selective tracer that allows for anatomical quantitation of activated T lymphocytes. Tracer uptake in the post-acute COVID group, which included those with and without Long COVID symptoms, was significantly higher compared to pre-pandemic controls in many anatomical regions, including the brain stem, spinal cord, bone marrow, nasopharyngeal and hilar lymphoid tissue, cardiopulmonary tissues, and gut wall. Although T cell activation tended to be higher in participants imaged closer to the time of the acute illness, tracer uptake was increased in participants imaged up to 2.5 years following SARS-CoV-2 infection. We observed that T cell activation in spinal cord and gut wall was associated with the presence of Long COVID symptoms. In addition, tracer uptake in lung tissue was higher in those with persistent pulmonary symptoms. Notably, increased T cell activation in these tissues was also observed in many individuals without Long COVID. Given the high [18F]F-AraG uptake detected in the gut, we obtained colorectal tissue for in situ hybridization SARS-CoV-2 RNA and immunohistochemical studies in a subset of participants with Long COVID symptoms. We identified cellular SARS-CoV-2 RNA in rectosigmoid lamina propria tissue in all these participants, ranging from 158 to 676 days following initial COVID-19 illness, suggesting that tissue viral persistence could be associated with long-term immunological perturbations.\nPost-COVID conditions during Delta and early-Omicron SARS-CoV-2 variant periods among adults in the United States\nAuthors: Edwards, D. L.; Logan, P.; Feldstein, L.; Accorsi, E.; Fairlie, T.; Saydah, S.\nScore: 16.1, Published: 2023-08-13 DOI: 10.1101/2023.08.09.23293776\nBackgroundPost-COVID conditions after infection with new SARS-CoV-2 variants have been incompletely described. We compared the prevalence and risk factors for ongoing symptoms lasting 4 weeks or longer (often referred to as post-COVID Conditions) among adults who had tested positive vs. negative during the Delta and early-Omicron periods. MethodsSelf-reported survey data regarding symptoms and previous SARS-CoV 2 test results were collected from May 31 - July 6, 2022, from a probability sampling of United States adults. Respondents were classified according to their test result, predominant circulating variant when respondents first tested positive (Delta vs early-Omicron), and demographic risk factors. ResultsAmong 2,421 respondents, 256 tested positive during Delta, 460 during early-Omicron, and 1,705 always tested negative. Nearly one-fourth (22.3%) of negative respondents reported [\u0026ge;]1symptom that lasted [\u0026ge;]4 weeks, compared to 60.6% (p\u003c0.05) of respondents who tested positive during the Delta period and 47.8% (p\u003c0.05) during the early-Omicron period. Fatigue, change in smell/taste, and cough were commonly reported by respondents who tested positive. Demographic risk factors associated with ongoing symptoms were being female and unemployed (aOR 1.28, 95% CI 1.06-1.55; aOR 1.48, 95% CI: 1.17-1.87). ConclusionThe reported occurrence of ongoing symptoms associated with post-COVID conditions was reduced during the early-Omicron period, compared with Delta.\nSelf-tests for COVID-19: what is the evidence? A living systematic review and meta-analysis (2020-2023)\nAuthors: Anand, A.; Vialard, F.; Esmail, A.; Ahmad Khan, F.; O'Byrne, P.; Routy, J.-P.; Dheda, K.; Pant Pai, N.\nScore: 15.8, Published: 2023-08-10 DOI: 10.1101/2023.08.09.23293885\nCOVID-19 self-testing strategy (COVIDST) can rapidly identify symptomatic and asymptomatic SARS-CoV-2-infected individuals and their contacts, potentially reducing transmission. In this living systematic review, we evaluated the evidence for real-world COVIDST performance. Two independent reviewers searched six databases (PubMed, Embase, Web of Science, World Health Organization database, Cochrane COVID-19 registry, Europe PMC) for the period April 1st, 2020, to January 18th, 2023. Data on studies evaluating COVIDST against laboratory-based conventional testing and reported on diagnostic accuracy, feasibility, acceptability, impact, and qualitative outcomes were abstracted. Bivariate random effects meta-analyses of COVIDST accuracy were performed (n=14). Subgroup analyses (by sampling site, symptomatic/asymptomatic infection, supervised/unsupervised strategy, with/without digital supports) were conducted. Data from 70 included studies, conducted across 25 countries with a median sample size of 817 (range: 28-784,707) were pooled. Specificity was high overall, irrespective of subgroups (98.37-99.71%). Highest sensitivities were reported for: a) symptomatic individuals (73.91%, 95%CI: 68.41-78.75%; n=9), b) mid-turbinate nasal samples (77.79%, 95%CI: 56.03-90.59%; n=14), c) supervised strategy (86.67%, 95%CI: 59.64-96.62%; n=13), and d) presence of digital interventions (70.15%, 95%CI: 50.18-84.63%; n=14). Sensitivity was lower in asymptomatic populations (40.18%, 95% CI: 21.52-62.20%; n=4), due to errors in test conduct and absence of supervision or a digital support. We found no difference in COVIDST sensitivity between delta and omicron pre-dominant period. Digital supports increased confidence in COVIDST reporting and interpretation (n=16). Overall acceptability was 91.0-98.7% (n=2) with lower acceptability reported for daily self-testing (39.5-51.1%). Feasibility was 69.0-100.0% (n=5) with lower feasibility (35.9-64.6%) for serial self-testing. COVIDST decreased closures in school, workplace, and social events (n=4). COVIDST is an effective rapid screening strategy for home-, workplace- or school-based screening, for symptomatic persons, and for preventing transmission during outbreaks. This data is useful for updating COVIDST policy. Our review demonstrates that COVIDST has paved the way for the introduction of self-tests, worldwide.\nAnalysis of SARS-CoV-2 Variant-Specific Serum Antibodies Post-Vaccination Utilizing Immortalized Human Hepatocyte-like Cells (HLC) to Assess Development of Protective Immunity\nAuthors: Collins, D. P.; Steer, C. J.\nScore: 11.3, Published: 2023-08-13 DOI: 10.1101/2023.08.08.23293863\nBackgroundOur previous studies demonstrated that SARS-CoV-2 spike proteins could bind to hepatocytes via the asialoglycoprotein receptor-1 (ASGR-1) facilitating direct infection by the SARS-CoV-2 virus. Immortalized E12-HLC expressed the phenotypic and biological properties of primary human hepatocytes, including their ability to bind spike proteins via ASGR-1 with exception of the spike 1 protein. This binding could be inhibited by spike protein-specific monoclonal antibodies. We used the same spike-blocking analysis to determine if post-vaccination serum was capable of blocking spike protein binding to HLC. Samples collected from subjects prior to, and post-vaccination were quantified for anti-variant-specific antibody (original wild type, alpha (), beta ({beta}), gamma ({gamma}) and delta ({delta}) variants) by a flow cytometry based immunofluorescent assay. Inhibition of variant spike protein binding to HLC and AT-2 (as a known model for spike 1 binding to the ACE-2 receptor) was analyzed by confocal microscopy. This study was designed to investigate the ability of post-vaccination antibodies to mediate immunity to spike S2, and to validate the utility of the E12-HLC in analyzing that immunity. MethodsSerum was collected from 10 individuals pre- and post-vaccination with the J\u0026J, Moderna or Pfizer vaccines. The serum samples were quantified for variant-specific antibodies in a flow cytometry-based immunofluorescent assay utilizing beads coated with biotinylated variant spike proteins (, {beta}, {gamma}, {delta}). Presence of variant-specific antibodies was visualized by anti-human IgG-Alexa 488. Inhibition of spike protein binding to cells was analyzed by immunofluorescent confocal analysis. Biotinylated variant spike proteins were preincubated with serum samples and then tested for binding to target cells. Binding was visualized by Streptavidin-Alexa 594. Results were compared to binding of unblocked spike variants. ResultsAll variant spike proteins tested bound to both the HLC and AT-2 cells. Pre-vaccination serum samples had no detectable reactivity to any of the variant spike proteins and were unable to inhibit binding of the variant spike proteins to either target cell. Post-vaccination serum samples demonstrated a progression of SARS-CoV-2 antibody levels from low early post-vaccination levels to higher levels at 2.5 months after vaccination. Concurrently, serum samples taken at those different timeframes demonstrated that serum obtained from shortly after vaccination were not as effective in blocking spike protein as serum obtained after 2.5 months post-vaccination. Antibody concentrations were not necessarily associated with better blocking of spike protein binding as spike variant-specific serum antibody concentrations varied significantly between subjects and within each subject. It was also demonstrated that vaccination with all the various available vaccines stimulated antibodies that inhibited binding of the available variant spike proteins to both HLC and AT-2 cells. ConclusionHLC, along with AT-2 cells, provides a useful platform to study the development of protective antibodies that prevent the binding SARS-CoV-2 spike proteins to target cells. It was shown that vaccination with the three available vaccines all elicited serum antibodies that were protective against binding of each of the variant spike proteins to both AT-2 and HLC cells. This study suggests that analysis of immune serum to block spike binding to target cells may be a more useful technique to assess protective immunity than quantitation of gross antibody alone.\nTemporal trends of dengue seroprevalence among children in coastal Kenya, 1998-2018: a longitudinal cohort study\nAuthors: Karanja, H. K.; Orindi, B.; Gitonga, J. N.; Mugo, D.; Mwai, K.; Nyamwaya, D. K.; Omuoyo, D. O.; Kitsao, B. S.; Musyoki, J. N.; Wambua, J.; Otieno, E.; Ochola-Oyier, L. I.; Bejon, P.; Warimwe, G. M.\nScore: 8.1, Published: 2023-08-16 DOI: 10.1101/2023.08.13.23294039\nSerosurveys suggest widespread dengue virus (DENV) transmission in Africa but there is limited information on the temporal patterns of exposure. Here, we estimated the prevalence and incidence of DENV infections in coastal Kenya over a 20-year period (1998-2018). Sera (n=8038) obtained annually from a longitudinal cohort of 1847 children aged 15 years and below were screened for anti-DENV IgG antibodies. Anti-DENV IgG seroprevalence increased with age and peaked during outbreak years. Among 1354 children who were seronegative at recruitment, we observed an overall incidence (seroconversion) rate of 129.5 (95% CI 118.7-141.4) DENV infections per 1000 person-years. The highest incidence was observed in 2013 at 520 infections per 1000 person-years (95% CI 443.6-610.2) coinciding with a large DENV outbreak in coastal Kenya. Our data suggest long-term DENV exposure among children in coastal Kenya highlighting an urgent need for clinical surveillance to determine the associated health burden in this setting.\nThe post-COVID-19 population has a high prevalence of crossreactive antibodies to spikes from all Orthocoronavirinae genera\nAuthors: Singh, G.; Abbad, A.; Kleiner, G.; Srivastava, K.; Gleason, C.; Carreno Quiroz, J. M.; Simon, V.; Krammer, F.\nScore: 52.2, Published: 2023-08-06 DOI: 10.1101/2023.08.01.23293522\nThe Orthocoronaviridae subfamily is large comprising four highly divergent genera. Four seasonal coronaviruses were circulating in humans prior to the coronavirus disease 2019 (COVID-19) pandemic. Infection with these viruses induced antibody responses that are relatively narrow with little cross-reactivity to spike proteins of other coronaviruses. Here, we report that infection with and vaccination against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) induces broadly crossreactive binding antibodies to spikes from a wide range of coronaviruses including members of the sarbecovirus subgenus, betacoronaviruses including Middle Eastern respiratory syndrome coronavirus (MERS CoV), and extending to alpha-, gamma- and delta-coronavirus spikes. These data show that the coronavirus spike antibody landscape in humans has profoundly been changed and broadened as a result of the SARS-CoV-2 pandemic. While we do not understand the functionality of these crossreactive antibodies, they may lead to enhanced resistance of the population to infection with newly emerging coronaviruses with pandemic potential.\nGenetic Variant rs1205 is Associated with COVID-19 Outcomes: The Strong Heart Study and Strong Heart Family Study.\nAuthors: Best, L. G.; Erdei, E.; Haack, K.; Kent, J. W.; Malloy, K.; Newman, D. E.; O'Leary, M.; O'Leary, R.; Sun, Q.; Navas-Acien, A.; Franceschini, N.; Cole, S. A.\nScore: 10.5, Published: 2023-08-08 DOI: 10.1101/2023.08.04.23293551\nBackgroundAlthough COVID-19 infection has been associated with a number of clinical and environmental risk factors, host genetic variation has also been associated with the incidence and morbidity of infection. The CRP gene codes for a critical component of the innate immune system and CRP variants have been reported associated with infectious disease and vaccination outcomes. We investigated possible associations between COVID-19 outcome and a limited number of candidate gene variants including rs1205. Methodology/Principal FindingsThe Strong Heart and Strong Heart Family studies have accumulated detailed genetic, cardiovascular risk and event data in geographically dispersed American Indian communities since 1988. Chi-square tests, logistic regression and generalized linear mixed models (implemented in SOLAR) were used in analysis. Genotypic data and 91 COVID-19 adjudicated deaths or hospitalizations from 2/1/20 through 3/1/23 were identified among 3,780 participants in two subsets. Among 21 candidate variants including genes in the interferon response pathway, APOE, TMPRSS2, TLR3, the HLA complex and the ABO blood group, only rs1205, a 3 untranslated region variant in the CRP gene, showed nominally significant association in T-dominant model analyses (odds ratio 1.859, 95%CI 1.001-3.453, p=0.049) after adjustment for age, sex, center, body mass index, and a history of cardiovascular disease. Within the younger subset, association with the rs1205 T-Dom genotype was stronger, both in the same adjusted logistic model and in the SOLAR analysis also adjusting for other genetic relatedness. ConclusionA T-dominant genotype of rs1205 in the CRP gene is associated with COVID-19 death or hospitalization, even after adjustment for relevant clinical factors and potential participant relatedness. Additional study of other populations and genetic variants of this gene are warranted. Author summaryConsiderable inter-individual variability in COVID-19 clinical outcome has been noted from the onset of this pandemic. Some risk factors, such as age, diabetes, obesity, prior cardiovascular disease (CVD) are well established. The possibility of inherited, host genetic risk factors has also been examined and a number validated in very large, population based datasets. The present study leveraged on-going clinical surveillance of CVD and available genetic information in an American Indian population to investigate potential host genetic contributors to COVID-19 morbidity and mortality. Although the number of cases ascertained was relatively small, a novel variant in the C-reactive protein gene was found to be associated with COVID-19 outcome. This protein constitutes a critical component of the innate immune system and CRP variants have been reported associated with infectious disease and vaccination outcomes. Improved understanding of the pathophysiology of this infection may allow more targeted therapy.\n",
  "wordCount" : "3272",
  "inLanguage": "en",
  "datePublished": "2023-08-20T10:35:54Z",
  "dateModified": "2023-08-20T10:35:54Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta"><span>updated on August 20, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.23293924">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.23293924" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.23293924">
        <p class="paperTitle">Published benefits of ivermectin use in Itajai, Brazil for COVID-19 infection, hospitalisation, and mortality are entirely explained by statistical artefacts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.23293924" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.23293924" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mills, R.; Pecanha Antonio, A. C.; Tucker-Kellogg, G.</p>
        <p class="info">Score: 265.2, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.23293924' target='https://doi.org/10.1101/2023.08.10.23293924'> 10.1101/2023.08.10.23293924</a></p>
        <p class="abstract">BackgroundTwo recent publications by Kerr et al. (Cureus 14(1):e21272; Cureus 14(8): e28624) reported dramatic effects of prophylactic ivermectin use for both prevention of COVID-19 and reduction of COVID-19-related hospitalisation and mortality, including a dose-dependent effect of ivermectin prophylaxis. These papers have gained an unusually large public influence: they were incorporated into debates around COVID-19 policies and may have contributed to decreased trust in vaccine efficacy and public health authorities more broadly. Both studies were based on retrospective observational analysis of city-wide registry data from the city of Itajai, Brazil from July-December 2020.

MethodsStarting with initially identified sources of error, we conducted a revised statistical analysis of available data, including data made available with the original papers and public data from the Brazil Ministry of Health. We identified additional uncorrected sources of bias and errors from the original analysis, including incorrect subject exclusion and missing subjects, an enrolment time bias, and multiple sources of immortal time bias. In models assuming no actual effect from ivermectin use, we conducted Monte Carlo simulations to estimate the contribution of these biases to any observed effect.

ResultsUntreated statistical artefacts and methodological errors alone lead to dramatic apparent risk reduction associated with Ivermectin use in both studies. The magnitude of apparent risk reduction from these artefacts is comparable to the results reported by the studies themselves, including apparent protection from infection, hospitalisation, and death, and including the reported apparent dose-response relationship.

ConclusionsThe inference of ivermectin efficacy reported in both papers is unsupported, as the observed effects are entirely explained by untreated statistical artefacts and methodological errors. Our re-analysis calls for caution in interpreting highly publicised observational studies and highlights the importance of common sources of bias in clinical research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.26.23284998">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.26.23284998" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.26.23284998">
        <p class="paperTitle">A molnupiravir-associated mutational signature in global SARS-CoV-2 genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.26.23284998" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.26.23284998" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanderson, T.; Hisner, R.; Donovan-Banfield, I.; Hartman, H.; Lochen, A.; Peacock, T. P.; Ruis, C.</p>
        <p class="info">Score: 2360.4, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.26.23284998' target='https://doi.org/10.1101/2023.01.26.23284998'> 10.1101/2023.01.26.23284998</a></p>
        <p class="abstract">Molnupiravir, an antiviral medication that has been widely used against SARS-CoV-2, acts by inducing mutations in the virus genome during replication. Most random mutations are likely to be deleterious to the virus, and many will be lethal, and so molnupiravir-induced elevated mutation rates reduce viral load. However, if some patients treated with molnupiravir do not fully clear SARS-CoV-2 infections, there could be the potential for onward transmission of molnupiravir-mutated viruses. Here we show that SARS-CoV-2 sequencing databases contain extensive evidence of molnupiravir mutagenesis. Using a systematic approach, we find that a specific class of long phylogenetic branches, distinguished by a high proportion of G-to-A and C-to-T mutations, appear almost exclusively in sequences from 2022, after the introduction of molnupiravir treatment, and in countries and age-groups with widespread usage of the drug. We identify a mutational spectrum, with preferred nucleotide contexts, from viruses in patients known to have been treated with molnupiravir and show that its signature matches that seen in these long branches, in some cases with onwards transmission of molnupiravir-derived lineages. Finally, we analyse treatment records to confirm a direct association between these high G-to-A branches and the use of molnupiravir.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.23293864">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.23293864" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.23293864">
        <p class="paperTitle">Waning of post-vaccination neutralizing antibody responses against SARS-CoV-2, a systematic literature review and meta-analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.23293864" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.23293864" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jacobsen, H.; Sitaras, I.; Katzmarzyk, M.; Cobos Jimenez, V.; Naughton, R.; Higdon, M. M.; Deloria Knoll, M.</p>
        <p class="info">Score: 71.8, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.23293864' target='https://doi.org/10.1101/2023.08.08.23293864'> 10.1101/2023.08.08.23293864</a></p>
        <p class="abstract">BackgroundMass COVID-19 vaccination and the continuous introduction of new viral variants of SARS-CoV-2, especially of Omicron subvariants, has resulted in an increase in the proportion of the population with hybrid immunity at various stages of waning protection. We systematically reviewed waning of post-vaccination neutralizing antibody titers in different immunological settings to investigate potential differences.

MethodsWe searched for studies providing data for post-vaccination neutralizing antibody responses against SARS-CoV-2 in PubMed, bioRxiv, and medRxiv from Dec 15, 2021, to Jan 31, 2023, using keywords related to COVID-19, vaccination, and antibody neutralization. We used random effects meta-regression to estimate the average fold-reduction in post-vaccination neutralizing antibody titers against the Index strain or Omicron BA.1. from month 1 to month 6 post last dose, stratified by vaccination regimen (primary or booster) and infection-naive vs hybrid-immune status.

FindingsIn total, 26 studies reporting longitudinal post-vaccination neutralizing antibody titers were included. Neutralization titers against the Index variant were available from all studies for infection-naive participants, and from nine for hybrid-immune participants. Against Omicron BA.1, nine and eight studies were available for infection-naive and hybrid-immune cohorts, respectively. In infection-naive cohorts, post-vaccination neutralization titers against the Index strain waned 5.1-fold (95% CI 3.4-7.8) from month 1 to month 6 following primary regimen and 3.8-fold (95% CI 2.4-5.9) following the booster. Titers against Omicron BA.1 waned 5.9-fold (95% CI 3.8-9.0) in infection-naive, post-booster cohorts. In hybrid-immune, post-primary vaccination cohorts, titers waned 3.7-fold (95% CI 1.7-7.9) against the Index strain and 5.0-fold (95% CI 1.1-21.8) against Omicron BA.1.

InterpretationNo obvious differences in waning between post-primary or post-boost vaccination were observed for vaccines used widely to date, nor between infection-naive and hybrid-immune participants. Titers against Omicron BA.1 may wane faster compared to Index titers, which may worsen for more recent Omicron sub-variants and should be monitored. Relatively small datasets limit the precision of our current analysis; further investigation is needed when more data become available. However, based on our current findings, striking differences in waning for the analyzed and future comparisons are unlikely.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.23293177">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.23293177" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.23293177">
        <p class="paperTitle">Multimodal Molecular Imaging Reveals Tissue-Based T Cell Activation and Viral RNA Persistence for Up to Two Years Following COVID-19</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.23293177" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.23293177" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peluso, M. J.; Ryder, D. M.; Flavell, R.; Wang, Y.; Levi, J.; LaFranchi, B. H.; Deveau, T.-M. M.; Buck, A. M.; Munter, S. E.; Asare, K. A.; Aslam, M.; Koch, W.; Szabo, G.; Hoh, R.; Deswal, M.; Rodriguez, A.; Buitrago, M.; Tai, V.; Shrestha, U.; Lu, S.; Goldberg, S. A.; Dalhuisen, T.; Durstenfeld, M. S.; Hsue, P. Y.; Kelly, J. D.; Kumar, N.; Martin, J. N.; Gambhir, A.; Somsouk, M.; Seo, Y.; Deeks, S. G.; Laszik, Z. G.; VanBrocklin, H. F.; Henrich, T. J.</p>
        <p class="info">Score: 1760.8, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.23293177' target='https://doi.org/10.1101/2023.07.27.23293177'> 10.1101/2023.07.27.23293177</a></p>
        <p class="abstract">The etiologic mechanisms of post-acute medical morbidities and unexplained symptoms (Long COVID) following SARS-CoV-2 infection are incompletely understood. There is growing evidence that viral persistence and immune dysregulation may play a major role. We performed whole-body positron emission tomography (PET) imaging in a cohort of 24 participants at time points ranging from 27 to 910 days following acute SARS-CoV-2 infection using a novel radiopharmaceutical agent, [18F]F-AraG, a highly selective tracer that allows for anatomical quantitation of activated T lymphocytes. Tracer uptake in the post-acute COVID group, which included those with and without Long COVID symptoms, was significantly higher compared to pre-pandemic controls in many anatomical regions, including the brain stem, spinal cord, bone marrow, nasopharyngeal and hilar lymphoid tissue, cardiopulmonary tissues, and gut wall. Although T cell activation tended to be higher in participants imaged closer to the time of the acute illness, tracer uptake was increased in participants imaged up to 2.5 years following SARS-CoV-2 infection. We observed that T cell activation in spinal cord and gut wall was associated with the presence of Long COVID symptoms. In addition, tracer uptake in lung tissue was higher in those with persistent pulmonary symptoms. Notably, increased T cell activation in these tissues was also observed in many individuals without Long COVID. Given the high [18F]F-AraG uptake detected in the gut, we obtained colorectal tissue for in situ hybridization SARS-CoV-2 RNA and immunohistochemical studies in a subset of participants with Long COVID symptoms. We identified cellular SARS-CoV-2 RNA in rectosigmoid lamina propria tissue in all these participants, ranging from 158 to 676 days following initial COVID-19 illness, suggesting that tissue viral persistence could be associated with long-term immunological perturbations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.23293776">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.23293776" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.23293776">
        <p class="paperTitle">Post-COVID conditions during Delta and early-Omicron SARS-CoV-2 variant periods among adults in the United States</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.23293776" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.23293776" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Edwards, D. L.; Logan, P.; Feldstein, L.; Accorsi, E.; Fairlie, T.; Saydah, S.</p>
        <p class="info">Score: 16.1, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.23293776' target='https://doi.org/10.1101/2023.08.09.23293776'> 10.1101/2023.08.09.23293776</a></p>
        <p class="abstract">BackgroundPost-COVID conditions after infection with new SARS-CoV-2 variants have been incompletely described. We compared the prevalence and risk factors for ongoing symptoms lasting 4 weeks or longer (often referred to as post-COVID Conditions) among adults who had tested positive vs. negative during the Delta and early-Omicron periods.

MethodsSelf-reported survey data regarding symptoms and previous SARS-CoV 2 test results were collected from May 31 - July 6, 2022, from a probability sampling of United States adults. Respondents were classified according to their test result, predominant circulating variant when respondents first tested positive (Delta vs early-Omicron), and demographic risk factors.

ResultsAmong 2,421 respondents, 256 tested positive during Delta, 460 during early-Omicron, and 1,705 always tested negative. Nearly one-fourth (22.3%) of negative respondents reported [&amp;ge;]1symptom that lasted [&amp;ge;]4 weeks, compared to 60.6% (p&lt;0.05) of respondents who tested positive during the Delta period and 47.8% (p&lt;0.05) during the early-Omicron period. Fatigue, change in smell/taste, and cough were commonly reported by respondents who tested positive. Demographic risk factors associated with ongoing symptoms were being female and unemployed (aOR 1.28, 95% CI 1.06-1.55; aOR 1.48, 95% CI: 1.17-1.87).

ConclusionThe reported occurrence of ongoing symptoms associated with post-COVID conditions was reduced during the early-Omicron period, compared with Delta.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.23293885">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.23293885" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.23293885">
        <p class="paperTitle">Self-tests for COVID-19: what is the evidence? A living systematic review and meta-analysis (2020-2023)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.23293885" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.23293885" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Anand, A.; Vialard, F.; Esmail, A.; Ahmad Khan, F.; O&#39;Byrne, P.; Routy, J.-P.; Dheda, K.; Pant Pai, N.</p>
        <p class="info">Score: 15.8, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.23293885' target='https://doi.org/10.1101/2023.08.09.23293885'> 10.1101/2023.08.09.23293885</a></p>
        <p class="abstract">COVID-19 self-testing strategy (COVIDST) can rapidly identify symptomatic and asymptomatic SARS-CoV-2-infected individuals and their contacts, potentially reducing transmission. In this living systematic review, we evaluated the evidence for real-world COVIDST performance. Two independent reviewers searched six databases (PubMed, Embase, Web of Science, World Health Organization database, Cochrane COVID-19 registry, Europe PMC) for the period April 1st, 2020, to January 18th, 2023. Data on studies evaluating COVIDST against laboratory-based conventional testing and reported on diagnostic accuracy, feasibility, acceptability, impact, and qualitative outcomes were abstracted. Bivariate random effects meta-analyses of COVIDST accuracy were performed (n=14). Subgroup analyses (by sampling site, symptomatic/asymptomatic infection, supervised/unsupervised strategy, with/without digital supports) were conducted. Data from 70 included studies, conducted across 25 countries with a median sample size of 817 (range: 28-784,707) were pooled. Specificity was high overall, irrespective of subgroups (98.37-99.71%). Highest sensitivities were reported for: a) symptomatic individuals (73.91%, 95%CI: 68.41-78.75%; n=9), b) mid-turbinate nasal samples (77.79%, 95%CI: 56.03-90.59%; n=14), c) supervised strategy (86.67%, 95%CI: 59.64-96.62%; n=13), and d) presence of digital interventions (70.15%, 95%CI: 50.18-84.63%; n=14). Sensitivity was lower in asymptomatic populations (40.18%, 95% CI: 21.52-62.20%; n=4), due to errors in test conduct and absence of supervision or a digital support. We found no difference in COVIDST sensitivity between delta and omicron pre-dominant period. Digital supports increased confidence in COVIDST reporting and interpretation (n=16). Overall acceptability was 91.0-98.7% (n=2) with lower acceptability reported for daily self-testing (39.5-51.1%). Feasibility was 69.0-100.0% (n=5) with lower feasibility (35.9-64.6%) for serial self-testing. COVIDST decreased closures in school, workplace, and social events (n=4). COVIDST is an effective rapid screening strategy for home-, workplace- or school-based screening, for symptomatic persons, and for preventing transmission during outbreaks. This data is useful for updating COVIDST policy. Our review demonstrates that COVIDST has paved the way for the introduction of self-tests, worldwide.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.23293863">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.23293863" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.23293863">
        <p class="paperTitle">Analysis of SARS-CoV-2 Variant-Specific Serum Antibodies Post-Vaccination Utilizing Immortalized Human Hepatocyte-like Cells (HLC) to Assess Development of Protective Immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.23293863" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.23293863" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Collins, D. P.; Steer, C. J.</p>
        <p class="info">Score: 11.3, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.23293863' target='https://doi.org/10.1101/2023.08.08.23293863'> 10.1101/2023.08.08.23293863</a></p>
        <p class="abstract">BackgroundOur previous studies demonstrated that SARS-CoV-2 spike proteins could bind to hepatocytes via the asialoglycoprotein receptor-1 (ASGR-1) facilitating direct infection by the SARS-CoV-2 virus. Immortalized E12-HLC expressed the phenotypic and biological properties of primary human hepatocytes, including their ability to bind spike proteins via ASGR-1 with exception of the spike 1 protein. This binding could be inhibited by spike protein-specific monoclonal antibodies. We used the same spike-blocking analysis to determine if post-vaccination serum was capable of blocking spike protein binding to HLC. Samples collected from subjects prior to, and post-vaccination were quantified for anti-variant-specific antibody (original wild type, alpha (), beta ({beta}), gamma ({gamma}) and delta ({delta}) variants) by a flow cytometry based immunofluorescent assay. Inhibition of variant spike protein binding to HLC and AT-2 (as a known model for spike 1 binding to the ACE-2 receptor) was analyzed by confocal microscopy. This study was designed to investigate the ability of post-vaccination antibodies to mediate immunity to spike S2, and to validate the utility of the E12-HLC in analyzing that immunity.

MethodsSerum was collected from 10 individuals pre- and post-vaccination with the J&amp;J, Moderna or Pfizer vaccines. The serum samples were quantified for variant-specific antibodies in a flow cytometry-based immunofluorescent assay utilizing beads coated with biotinylated variant spike proteins (, {beta}, {gamma}, {delta}). Presence of variant-specific antibodies was visualized by anti-human IgG-Alexa 488. Inhibition of spike protein binding to cells was analyzed by immunofluorescent confocal analysis. Biotinylated variant spike proteins were preincubated with serum samples and then tested for binding to target cells. Binding was visualized by Streptavidin-Alexa 594. Results were compared to binding of unblocked spike variants.

ResultsAll variant spike proteins tested bound to both the HLC and AT-2 cells. Pre-vaccination serum samples had no detectable reactivity to any of the variant spike proteins and were unable to inhibit binding of the variant spike proteins to either target cell. Post-vaccination serum samples demonstrated a progression of SARS-CoV-2 antibody levels from low early post-vaccination levels to higher levels at 2.5 months after vaccination. Concurrently, serum samples taken at those different timeframes demonstrated that serum obtained from shortly after vaccination were not as effective in blocking spike protein as serum obtained after 2.5 months post-vaccination. Antibody concentrations were not necessarily associated with better blocking of spike protein binding as spike variant-specific serum antibody concentrations varied significantly between subjects and within each subject. It was also demonstrated that vaccination with all the various available vaccines stimulated antibodies that inhibited binding of the available variant spike proteins to both HLC and AT-2 cells.

ConclusionHLC, along with AT-2 cells, provides a useful platform to study the development of protective antibodies that prevent the binding SARS-CoV-2 spike proteins to target cells. It was shown that vaccination with the three available vaccines all elicited serum antibodies that were protective against binding of each of the variant spike proteins to both AT-2 and HLC cells. This study suggests that analysis of immune serum to block spike binding to target cells may be a more useful technique to assess protective immunity than quantitation of gross antibody alone.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.13.23294039">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.13.23294039" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.13.23294039">
        <p class="paperTitle">Temporal trends of dengue seroprevalence among children in coastal Kenya, 1998-2018: a longitudinal cohort study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.13.23294039" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.13.23294039" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Karanja, H. K.; Orindi, B.; Gitonga, J. N.; Mugo, D.; Mwai, K.; Nyamwaya, D. K.; Omuoyo, D. O.; Kitsao, B. S.; Musyoki, J. N.; Wambua, J.; Otieno, E.; Ochola-Oyier, L. I.; Bejon, P.; Warimwe, G. M.</p>
        <p class="info">Score: 8.1, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.13.23294039' target='https://doi.org/10.1101/2023.08.13.23294039'> 10.1101/2023.08.13.23294039</a></p>
        <p class="abstract">Serosurveys suggest widespread dengue virus (DENV) transmission in Africa but there is limited information on the temporal patterns of exposure. Here, we estimated the prevalence and incidence of DENV infections in coastal Kenya over a 20-year period (1998-2018). Sera (n=8038) obtained annually from a longitudinal cohort of 1847 children aged 15 years and below were screened for anti-DENV IgG antibodies. Anti-DENV IgG seroprevalence increased with age and peaked during outbreak years. Among 1354 children who were seronegative at recruitment, we observed an overall incidence (seroconversion) rate of 129.5 (95% CI 118.7-141.4) DENV infections per 1000 person-years. The highest incidence was observed in 2013 at 520 infections per 1000 person-years (95% CI 443.6-610.2) coinciding with a large DENV outbreak in coastal Kenya. Our data suggest long-term DENV exposure among children in coastal Kenya highlighting an urgent need for clinical surveillance to determine the associated health burden in this setting.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.23293522">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.23293522" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.23293522">
        <p class="paperTitle">The post-COVID-19 population has a high prevalence of crossreactive antibodies to spikes from all Orthocoronavirinae genera</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.23293522" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.23293522" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Singh, G.; Abbad, A.; Kleiner, G.; Srivastava, K.; Gleason, C.; Carreno Quiroz, J. M.; Simon, V.; Krammer, F.</p>
        <p class="info">Score: 52.2, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.23293522' target='https://doi.org/10.1101/2023.08.01.23293522'> 10.1101/2023.08.01.23293522</a></p>
        <p class="abstract">The Orthocoronaviridae subfamily is large comprising four highly divergent genera. Four seasonal coronaviruses were circulating in humans prior to the coronavirus disease 2019 (COVID-19) pandemic. Infection with these viruses induced antibody responses that are relatively narrow with little cross-reactivity to spike proteins of other coronaviruses. Here, we report that infection with and vaccination against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) induces broadly crossreactive binding antibodies to spikes from a wide range of coronaviruses including members of the sarbecovirus subgenus, betacoronaviruses including Middle Eastern respiratory syndrome coronavirus (MERS CoV), and extending to alpha-, gamma- and delta-coronavirus spikes. These data show that the coronavirus spike antibody landscape in humans has profoundly been changed and broadened as a result of the SARS-CoV-2 pandemic. While we do not understand the functionality of these crossreactive antibodies, they may lead to enhanced resistance of the population to infection with newly emerging coronaviruses with pandemic potential.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.23293551">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.23293551" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.23293551">
        <p class="paperTitle">Genetic Variant rs1205 is Associated with COVID-19 Outcomes: The Strong Heart Study and Strong Heart Family Study.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.23293551" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.23293551" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Best, L. G.; Erdei, E.; Haack, K.; Kent, J. W.; Malloy, K.; Newman, D. E.; O&#39;Leary, M.; O&#39;Leary, R.; Sun, Q.; Navas-Acien, A.; Franceschini, N.; Cole, S. A.</p>
        <p class="info">Score: 10.5, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.23293551' target='https://doi.org/10.1101/2023.08.04.23293551'> 10.1101/2023.08.04.23293551</a></p>
        <p class="abstract">BackgroundAlthough COVID-19 infection has been associated with a number of clinical and environmental risk factors, host genetic variation has also been associated with the incidence and morbidity of infection. The CRP gene codes for a critical component of the innate immune system and CRP variants have been reported associated with infectious disease and vaccination outcomes. We investigated possible associations between COVID-19 outcome and a limited number of candidate gene variants including rs1205.

Methodology/Principal FindingsThe Strong Heart and Strong Heart Family studies have accumulated detailed genetic, cardiovascular risk and event data in geographically dispersed American Indian communities since 1988. Chi-square tests, logistic regression and generalized linear mixed models (implemented in SOLAR) were used in analysis. Genotypic data and 91 COVID-19 adjudicated deaths or hospitalizations from 2/1/20 through 3/1/23 were identified among 3,780 participants in two subsets. Among 21 candidate variants including genes in the interferon response pathway, APOE, TMPRSS2, TLR3, the HLA complex and the ABO blood group, only rs1205, a 3 untranslated region variant in the CRP gene, showed nominally significant association in T-dominant model analyses (odds ratio 1.859, 95%CI 1.001-3.453, p=0.049) after adjustment for age, sex, center, body mass index, and a history of cardiovascular disease. Within the younger subset, association with the rs1205 T-Dom genotype was stronger, both in the same adjusted logistic model and in the SOLAR analysis also adjusting for other genetic relatedness.

ConclusionA T-dominant genotype of rs1205 in the CRP gene is associated with COVID-19 death or hospitalization, even after adjustment for relevant clinical factors and potential participant relatedness. Additional study of other populations and genetic variants of this gene are warranted.

Author summaryConsiderable inter-individual variability in COVID-19 clinical outcome has been noted from the onset of this pandemic. Some risk factors, such as age, diabetes, obesity, prior cardiovascular disease (CVD) are well established. The possibility of inherited, host genetic risk factors has also been examined and a number validated in very large, population based datasets. The present study leveraged on-going clinical surveillance of CVD and available genetic information in an American Indian population to investigate potential host genetic contributors to COVID-19 morbidity and mortality. Although the number of cases ascertained was relatively small, a novel variant in the C-reactive protein gene was found to be associated with COVID-19 outcome. This protein constitutes a critical component of the innate immune system and CRP variants have been reported associated with infectious disease and vaccination outcomes. Improved understanding of the pathophysiology of this infection may allow more targeted therapy.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
