<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Reverse mutational scanning of spike BA.2.86 identifies the epitopes contributing to immune escape from polyclonal sera
Authors: Bdeir, N.; Lüddecke, T.; Maass, H.; Schmelz, S.; Jacobsen, H.; Metzdorf, K.; Cossmann, A.; Stankov, M.; Hoffmann, M.; Pöhlmann, S.; Blankenfeldt, W.; Jablonka, A.; Behrens, G. M. N.; Cicin-Sain, L.
Score: 21.1, Published: 2024-01-03 DOI: 10.1101/2024.01.03.23300575
The recently detected Omicron BA.2.86 lineage contains more than 30 amino acid mutations relative to BA.2. Here, we identify the epitopes driving immune escape of BA.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Reverse mutational scanning of spike BA.2.86 identifies the epitopes contributing to immune escape from polyclonal sera
Authors: Bdeir, N.; Lüddecke, T.; Maass, H.; Schmelz, S.; Jacobsen, H.; Metzdorf, K.; Cossmann, A.; Stankov, M.; Hoffmann, M.; Pöhlmann, S.; Blankenfeldt, W.; Jablonka, A.; Behrens, G. M. N.; Cicin-Sain, L.
Score: 21.1, Published: 2024-01-03 DOI: 10.1101/2024.01.03.23300575
The recently detected Omicron BA.2.86 lineage contains more than 30 amino acid mutations relative to BA.2. Here, we identify the epitopes driving immune escape of BA." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-07T10:38:59+00:00" />
<meta property="article:modified_time" content="2024-01-07T10:38:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Reverse mutational scanning of spike BA.2.86 identifies the epitopes contributing to immune escape from polyclonal sera
Authors: Bdeir, N.; Lüddecke, T.; Maass, H.; Schmelz, S.; Jacobsen, H.; Metzdorf, K.; Cossmann, A.; Stankov, M.; Hoffmann, M.; Pöhlmann, S.; Blankenfeldt, W.; Jablonka, A.; Behrens, G. M. N.; Cicin-Sain, L.
Score: 21.1, Published: 2024-01-03 DOI: 10.1101/2024.01.03.23300575
The recently detected Omicron BA.2.86 lineage contains more than 30 amino acid mutations relative to BA.2. Here, we identify the epitopes driving immune escape of BA."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Reverse mutational scanning of spike BA.2.86 identifies the epitopes contributing to immune escape from polyclonal sera\nAuthors: Bdeir, N.; Lüddecke, T.; Maass, H.; Schmelz, S.; Jacobsen, H.; Metzdorf, K.; Cossmann, A.; Stankov, M.; Hoffmann, M.; Pöhlmann, S.; Blankenfeldt, W.; Jablonka, A.; Behrens, G. M. N.; Cicin-Sain, L.\nScore: 21.1, Published: 2024-01-03 DOI: 10.1101/2024.01.03.23300575\nThe recently detected Omicron BA.2.86 lineage contains more than 30 amino acid mutations relative to BA.2. Here, we identify the epitopes driving immune escape of BA.",
  "keywords": [
    
  ],
  "articleBody": " Reverse mutational scanning of spike BA.2.86 identifies the epitopes contributing to immune escape from polyclonal sera\nAuthors: Bdeir, N.; Lüddecke, T.; Maass, H.; Schmelz, S.; Jacobsen, H.; Metzdorf, K.; Cossmann, A.; Stankov, M.; Hoffmann, M.; Pöhlmann, S.; Blankenfeldt, W.; Jablonka, A.; Behrens, G. M. N.; Cicin-Sain, L.\nScore: 21.1, Published: 2024-01-03 DOI: 10.1101/2024.01.03.23300575\nThe recently detected Omicron BA.2.86 lineage contains more than 30 amino acid mutations relative to BA.2. Here, we identify the epitopes driving immune escape of BA.2.86 and its derivative JN.1 (BA.2.86 + S455L) lineage. We investigated the cross-reactive humoral immunity within a cohort of health care workers against Omicron BA.2.86 and JN.1 by employing pseudo-viral mutants. We demonstrate that BA.2.86 and especially JN.1 evaded neutralization by serum antibodies of fully vaccinated individuals. To discern the contribution of individual epitope mutations to immune escape, we constructed a library of 33 BA.2.86 mutants, each of which harbored a single revertant mutation going back to BA.2. This library was used in a reverse mutational scanning approach to define serum neutralization titers against each epitope separately. The mutations within the receptor binding domain (RBD) at position K356T and to a lesser extent the mutations N460K, V483{Delta}, A484K, and F486P enhanced the immune escape. More surprisingly, the mutation 16insMPLF within the spike N-terminal domain (NTD) and the mutation P621S in S1/S2 significantly contributed to antibody escape of BA.2.86. Upon XBB.1.5 booster vaccination, neutralization titers against JN.1 and BA.2.86 improved relative to all ancestral strains, and the residual immune escape was driven by mutations at positions 16insMPLF, {Delta}144Y, E544K, P621S, and A484K.\nEscalating combinations of enhanced infectivity and immune escape define SARS-CoV-2 Omicron lineage replacement\nAuthors: Chen, N. F. G.; Pham, K.; Chaguza, C.; Lopes, R.; Klaassen, F.; Weinberger, D. M.; Pitzer, V. E.; Warren, J. L.; Grubaugh, N. D.; Hahn, A. M.\nScore: 7.3, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300790\nIn 2022, consecutive sweeps of the highly transmissible SARS-CoV-2 Omicron-family maintained high viral transmission levels despite extensive antigen exposure on the population level resulting from both vaccinations and infections. To better understand variant fitness in the context of the highly dynamic immunity landscape of 2022, we aimed to dissect the interplay between immunity and fitness advantages of emerging SARS-CoV-2 Omicron lineages on the population-level. We evaluated the relative contribution of higher intrinsic transmissibility or immune escape on the fitness of emerging lineages by analyzing data collected through our local genomic surveillance program from Connecticut, USA. We compared growth rates, estimated infections, effective reproductive rates, average viral copy numbers, and likelihood for causing vaccine break-through infections. Using these population-level data, we find that newly emerging Omicron lineages reach dominance through a specific combination of enhanced intrinsic transmissibility and immune escape that varies over time depending on the state of the host-population. Using similar frameworks that integrate whole genome sequencing together with clinical, laboratory, and epidemiological data can advance our knowledge on host-pathogen dynamics in the post-emergence phase that can be applied to other communicable diseases beyond SARS-CoV-2.\nDiverse and weakly immunogenic var gene expression facilitates malaria infection\nAuthors: Bhardwaj, I.; Nyarko, P. B.; Ashn, A. B.; Cohen, C.; Ceesay, S.; Achan, J.; Dabira, E.; Nakajima, R.; Jain, A.; Taghavian, O.; Jasinskas, A.; Felgner, P. L.; D'Alessandro, U.; Bousema, T.; Travassos, M.; Radulescu, O.; Claessens, A.\nScore: 5.2, Published: 2024-01-02 DOI: 10.1101/2023.12.27.23300577\nPlasmodium falciparum is believed to escape immunity via antigenic variation, mediated in part by 60 var genes. These genes undergo mutually exclusive expression and encode the PfEMP1 surface antigen. The frequency of var switching and the immunogenicity of each expressed PfEMP1 remain unclear. To this end, we carried out a Controlled Human Malaria Infection (CHMI) study with 19 adult African volunteers in The Gambia to gain insight into the effect of naturally acquired immunity on the expressed var gene repertoire during early phase of an infection. Our findings demonstrated a strong correlation between the diversity of var expression, quantified through entropy, and infection outcome. Low-immunity individuals were characterised by high var entropy profiles, higher parasitaemia, and lower sero-recognised PfEMP1 domains compared to high-immunity individuals. For the first time we recorded the probability of var gene switching in vitro and of turnover in vivo, enabling us to estimate both intrinsic switching and negative-selection effects. These processes are rapid, resulting in estimated turnover/switching probabilities of 69% - 97% and 7% - 57% per generation, in vivo and in vitro, respectively. Var (PfEMP1) expression triggered time-dependent humoral immune responses in low immunity individuals, with many PfEMP1 domains remaining weakly immunogenic. We conclude that the role of intrinsic var switching is to reset and maintain a diverse var repertoire. The high var switching rates and weak PfEMP1 immunogenicity benefit parasite survival during the CHMI.\nPhylogeographic Inference of SARS-CoV-2 Delta Wave in Texas, USA using a Novel Spatial Transmission Count Statistic\nAuthors: Lyu, L.; Veytsel, G. E.; Stott, G.; Fox, S. J.; Dailey, C. A.; Damodaran, L.; Fujimoto, K.; Kuo, J.; Brown, P.; Sealy, R.; Brown, A.; Alabady, M.; Bahl, J.\nScore: 4.2, Published: 2023-12-30 DOI: 10.1101/2023.12.28.23300535\nViral genomes contain records of geographic movements and cross-scale transmission dynamics. However, the impact of population heterogeneity, particularly among rural and urban areas, on viral spread and epidemic trajectory has been less explored due to limited data availability. Intensive and widespread efforts to collect and sequence SARS-CoV-2 viral samples have enabled the development of comparative genomic approaches to reconstruct spatial transmission history and understand viral transmission across different scales. Large genomic datasets with few mutations present challenges for traditional phylodynamic approaches. To address this issue, we propose a novel spatial transmission count statistic that efficiently summarizes the geographic transmission patterns imprinted on viral phylogenies. Our analysis pipeline reconstructs a time-scaled phylogeny with ancestral trait states and identifies spatial transmission linkages, categorized as imports, local transmission, and exports. These linkages are summarized to represent the epidemic profile of the focal area. We demonstrate the utility of this approach for near real-time outbreak analysis using over 12,000 full genomes and linked epidemiological data to investigate the spread of the SARS-CoV-2 Delta variant in Texas. Our goal is to trace the Delta variants origin, timing and to understand the role of urban and rural areas in the spatial diffusion patterns observed in Texas. Our study shows (1) highly populated urban centers were the main sources of the epidemic in Texas; (2) the outbreaks in urban centers were connected to the global epidemic; and (3) outbreaks in urban centers were locally maintained, while epidemics in rural areas were driven by repeated introductions. Significance StatementWe developed a novel phylogeographic approach that analyzes transmission patterns at low computational cost. This method not only facilitates the inference of spatial scales of transmission but also enables exploration of how specific demographic characteristics influence transmission patterns among heterogenous populations. The rural population in the US, comprising approximately 60 million individuals, has been significantly impacted by COVID-19. Applying our new method, we examined the variations in epidemic patterns between urban centers (e.g., Houston) and rural areas in Texas. We found that urban centers are the primary source for SARS-CoV-2 in rural areas. This analysis lays the groundwork for designing effective public health interventions specifically tailored to the needs of affected areas.\nThoughts and perspectives of metagenome sequencing as a diagnostic tool for infectious disease: an interpretive phenomenological study\nAuthors: Trivett, H.; Darby, A. C.; Oyebode, O.\nScore: 3.2, Published: 2024-01-03 DOI: 10.1101/2024.01.02.24300703\nBackgroundEffective infectious disease diagnostics (IDD) are vital for informing clinical decision-making regarding the treatment and patient management of disease and infections. Conventional clinical methods rely upon culture-dependent techniques, and there has been little shift in the acceptance and integration of culture-independent sequencing methods into routine clinical IDD. This study explored stakeholders experiences within IDD, including those working in clinical settings and those conducting research at the forefront of microbial genomics. We aimed to identify factors driving the development and implementation of metagenome sequencing as a routine diagnostic. MethodsVirtual semi-structured interviews were conducted with purposively selected individuals involved in IDD. The interviews explored the experiences of implementing metagenome sequencing as a diagnostic tool and decisions about which diagnostics are used for identifying bacteria-causing infections. Thematic analysis was used to analyse the data, and an Interpretive Phenomenological approach was used throughout. ResultsTen individuals were interviewed between July 2021 and October 2021, including Clinical scientists, consultants, and professors in academia. Their experience ranged from no knowledge of metagenome sequencing to an expert understanding of the phenomenon. Five themes emerged: Diagnostic Choice, Infrastructure, Open Data Sharing, COVID-19, and Communication. Participants recognised the need for new diagnostics to be implemented to overcome the limitations of current diagnostic approaches but highlighted the barriers to integrating new diagnostics into clinical settings, such as the impact on clinical decision-making, accreditation, and cost. However, participants felt that lessons could be learnt from using metagenomics in COVID-19 and how other diagnostic platforms have been integrated into clinical settings over the last 20 years. ConclusionsThe study provided clear evidence to address the knowledge gap in current literature and practice for developing and implementing metagenome sequencing as a potential IDD. The knowledge of new and upcoming genomic diagnostic testing is not equally distributed throughout the UK, impacting the understanding and drive to integrate metagenome sequencing into routine clinical diagnostics. Improvements in access to new diagnostics could improve patient treatment and management and positively impact public health.\nCOVID-19 pandemic re-shaped the global dispersal of seasonal influenza viruses\nAuthors: Chen, Z.; Tsui, J. L.- H.; Gutierrez, B.; Moreno, S. B.; Plessis, L. d.; Deng, X.; Cai, J.; Bajaj, S.; Suchard, M. A.; Pybus, O. G.; Lemey, P.; Kraemer, M. U. G.; Yu, H.\nScore: 33.5, Published: 2023-12-24 DOI: 10.1101/2023.12.20.23300299\nUnderstanding how the global dispersal patterns of seasonal influenza viruses were perturbed during and after the COVID-19 pandemic is needed to inform influenza intervention and vaccination strategies in the post-pandemic period. Although global human mobility has been identified as a key driver of influenza dispersal1, alongside climatic and evolutionary factors2,3, the impact of international travel restrictions on global influenza transmission and recovery remains unknown. Here we combine molecular, epidemiological, climatic, and international travel data within a phylodynamic framework to show that, despite human mobility remaining the principal driver of global influenza virus dissemination, the pandemics onset led to a shift in the international population structure and migration network of seasonal influenza lineages. We find that South Asia and Africa played important roles as exporters and phylogenetic trunk locations of influenza in 2020 and 2021, and we highlight the association between population movement, antigenic drift and persistence during the intensive non-pharmaceutical interventions (NPIs) phase. The influenza B/Yamagata lineage disappeared in a context of reduced relative genetic diversity, moderate lineage turnover, and lower positive selection pressure. Our results demonstrate that mobility perturbations reshaped the global dispersal dynamics of influenza viruses, with potential implications for vaccine design and genomic surveillance programmes. As the risk of future pandemics persists, our study provides an opportunity to assess the impact of NPIs during the pandemic on respiratory infectious diseases beyond the interplay between SARS-CoV-2 and influenza viruses.\nTrends in SARS-CoV-2 seroprevalence among pregnant women attending first antenatal care visits in Zambia: a repeated cross-sectional survey, 2021-2022\nAuthors: Heilmann, E.; Tembo, T.; Fwoloshi, S.; Kabamba, B.; Chilambe, F.; Kalenga, K.; Siwingwa, M.; Mulube, C.; Seffren, V.; Bolton-Moore, C.; Simwanza, J.; Yingst, S.; Yadav, R.; Rogier, E.; Auld, A. F.; Agolory, S.; Kapina, M.; Gutman, J. R.; Savory, T.; Kangale, C.; Mulenga, L. B.; Sikazwe, I.; Hines, J. Z.\nScore: 2.5, Published: 2024-01-03 DOI: 10.1101/2024.01.02.24300729\nBackgroundSARS-CoV-2 serosurveys help estimate the extent of transmission and guide allocation of COVID-19 vaccines. We measured SARS-CoV-2 seroprevalence among women attending ANC clinics to assess exposure trends over time in Zambia. MethodsWe conducted repeated cross-sectional surveys among pregnant women aged 15-49 years attending their first ANC visits in four districts of Zambia (two urban and two rural) during September 2021-September 2022. Serologic testing was done using a multiplex bead assay which detects IgG antibodies to the nucleocapsid protein and the spike protein receptor-binding domain (RBD). We calculated monthly SARS-CoV-2 seroprevalence by district. We also categorized seropositive results as infection alone, infection and vaccination, or vaccination alone based on COVID-19 vaccination status and anti-RBD and anti-nucleocapsid test results. FindingsAmong 8,304 participants, 5,296 (63.8%) were cumulatively seropositive for SARS-CoV-2 antibodies. SARS-CoV-2 seroprevalence primarily increased from September 2021 to September 2022 in three districts (Lusaka: 61.8-100.0%, Chongwe: 39.6-94.7%, Chipata: 56.5-95.0%), but in Chadiza, seroprevalence increased from 27.8% in September 2021 to 77.2% in April 2022 before gradually dropping to 56.6% in July 2022. Among 5,906 participants with a valid COVID-19 vaccination status, infection alone accounted for antibody responses in 77.7% (4,590) of participants. InterpretationMost women attending ANC had evidence of prior SARS-CoV-2 infection and most SARS-CoV-2 seropositivity was infection-induced. Capturing COVID-19 vaccination status and using a multiplex bead assay with anti-nucleocapsid and anti-RBD targets facilitated distinguishing infection-induced versus vaccine-induced antibody responses during a period of increasing COVID-19 vaccine coverage in Zambia. Declining seroprevalence in Chadiza may indicate waning antibodies and a need for booster vaccines. ANC clinics have a potential role in ongoing SARS-CoV-2 serosurveillance and can continue to provide insights into SARS-CoV-2 antibody dynamics to inform near real-time public health responses.\nPrediction of post-PCV13 pneumococcal evolution using invasive disease data enhanced by inverse-invasiveness weighting\nAuthors: Qiu, X.; McGee, L.; Hammitt, L. L.; Grant, L. R.; O'Brien, K. L.; Hanage, W. P.; Lipsitch, M.\nScore: 1.5, Published: 2023-12-11 DOI: 10.1101/2023.12.10.23299786\nBackgroundAfter introduction of pneumococcal conjugate vaccines (PCVs), serotype replacement occurred in the population of Streptococcus pneumoniae. Predicting which pneumococcal clones and serotypes will become more common in carriage after vaccination can enhance vaccine design and public health interventions, while also improving our understanding of pneumococcal evolution. We sought to use invasive disease data to assess how well negative frequency-dependent selection (NFDS) models could explain pneumococcal carriage population evolution in the post-PCV13 epoch by weighting invasive data to approximate strain proportions in the carriage population. MethodsInvasive pneumococcal isolates were collected and sequenced during 1998-2018 by the Active Bacterial Core surveillance (ABCs) from the Centers for Disease Control and Prevention (CDC). To predict the post-PCV13 population dynamics in the carriage population using a NFDS model, all genomic data were processed under a bioinformatic pipeline of assembly, annotation, and pangenome analysis to define genetically similar sequence clusters (i.e., strains) and a set of accessory genes present in 5% to 95% of the isolates. The NFDS model predicted the strain proportion by calculating the post-vaccine strain composition in the weighted invasive disease population that would best match pre-vaccine accessory gene frequencies. To overcome the biases of invasive disease data, serotype-specific inverse-invasiveness weights were defined as the ratio of the proportion of the serotype in the carriage data to the proportion in the invasive data, using data from 1998-2001 in the United States, before conjugate vaccine introduction. The weights were applied to adjust both the observed strain proportion and the accessory gene frequencies. ResultsInverse-invasiveness weighting increased the correlation of accessory gene frequencies between invasive and carriage data with reduced residuals in linear or logit scale for pre-vaccine, post-PCV7, and post-PCV13. Similarly, weighting increased the correlation of accessory gene frequencies between different time periods in the invasive data. By weighting the invasive data, we were able to use the NFDS model to predict strain proportions in the carriage population in the post-PCV13 epoch, with the adjusted R-squared between predicted and observed strain proportions increasing from 0.176 to 0.544 after weighting. ConclusionsThe weighting system adjusted the invasive disease surveillance data to better represent the carriage population of S. pneumoniae. The NFDS mechanism predicted the strain proportions in the projected carriage population as estimated from the weighted invasive disease frequencies in the post-PCV13 epoch. Our methods enrich the value of genomic sequences from invasive disease surveillance, which is readily available, easy to collect, and of direct interest to public health. IMPORTANCEStreptococcus pneumoniae, a common colonizer in the human nasopharynx, can cause invasive diseases including pneumonia, bacteremia, and meningitis mostly in children under 5 years or older adults. The PCV7 was introduced in 2000 in the United States within the pediatric population to prevent disease and reduce deaths, followed by PCV13 in 2010, PCV15 in 2022, and PCV20 in 2023. After the removal of vaccine serotypes, the prevalence of carriage remained stable as the vacated pediatric ecological niche was filled with certain non-vaccine serotypes. Predicting which pneumococcal clones, and which serotypes, will be most successful in colonization after vaccination can enhance vaccine design and public health interventions, while also improving our understanding of pneumococcal evolution. While carriage data, which are collected from the pneumococcal population that is competing to colonize and transmit, are most directly relevant to evolutionary studies, invasive disease data are often more plentiful. Previously, evolutionary models based on negative frequency-dependent selection (NFDS) on the accessory genome were shown to predict which non-vaccine strains and serotypes were most successful in colonization following the introduction of PCV7. Here, we show that an inverse-invasiveness weighting system applied to invasive disease surveillance data allows the NFDS model to predict strain proportions in the projected carriage population in the post-PCV13/pre-PCV15 and -PCV20 epoch. The significance of our research lies in using a sample of invasive disease surveillance data to extend the use of NFDS as an evolutionary mechanism to predict post-PCV13 population dynamics. This has shown that we can correct for biased sampling that arises from differences in virulence and can enrich the value of genomic data from disease surveillance and advances our understanding of how NFDS impacts carriage population dynamics after both PCV7 and PCV13 vaccination.\nLack of association between HLA and asymptomatic SARS-CoV-2 infection\nAuthors: Marchal, A.; Cirulli, E. T.; Neveux, I.; Bellos, E.; Thwaites, R. S.; Schiabor Barrett, K. M.; Zhang, Y.; Nemes-Bokun, I.; Kalinova, M.; Catchpole, A.; Tangye, S. G.; Spaan, A. N.; Lack, J. B.; Ghosn, J.; Burdet, C.; Gorochov, G.; Tubach, F.; Hausfater, P.; COVID Human Genetic Effort, ; COVIDeF Study Group, ; French COVID Cohort Study Group, ; CoV-Contact Cohort, ; COVID-STORM Clinicians, ; COVID Clinicians, ; Orchestra Working Group, ; Amsterdam UMC Covid-19 Biobank, ; NIAID-USUHS COVID Study Group, ; Dalgard, C. L.; Zhang, S.-Y.; Zhang, Q.; Chiu, C.; Fellay, J.; Grzymski, J. J.; San\nScore: 49.6, Published: 2023-12-08 DOI: 10.1101/2023.12.06.23299623\nHuman genetic studies of critical COVID-19 pneumonia have revealed the essential role of type I interferon-dependent innate immunity to SARS-CoV-2 infection. Conversely, an association between the HLA-B*15:01 allele and asymptomatic SARS-CoV-2 infection in unvaccinated individuals was recently reported, suggesting a contribution of pre-existing T cell-dependent adaptive immunity. We report a lack of association of classical HLA alleles, including HLA-B*15:01, with pre-omicron asymptomatic SARS-CoV-2 infection in unvaccinated participants in a prospective population-based study in the US (191 asymptomatic vs. 945 symptomatic COVID-19 cases). Moreover, we found no such association in the international COVID Human Genetic Effort cohort (206 asymptomatic vs. 574 mild or moderate COVID-19 cases and 1,625 severe or critical COVID-19 cases). Finally, in the Human Challenge Characterisation study, the three HLA-B*15:01 individuals infected with SARS-CoV-2 developed symptoms. As with other acute primary infections, no classical HLA alleles favoring an asymptomatic course of SARS-CoV-2 infection were identified. These findings suggest that memory T-cell immunity to seasonal coronaviruses does not strongly influence the outcome of SARS-CoV-2 infection in unvaccinated individuals.\nProportion of Hospitalizations Preventable with Increased Oral SARS-CoV-2 Antiviral Treatment\nAuthors: Levy, M. E.; Chilunda, V.; Heaton, P. R.; Grzymski, J.; Pawloski, P. A.; Goldman, J. D.; Luo, S.\nScore: 2.9, Published: 2023-12-21 DOI: 10.1101/2023.12.19.23300241\nWe estimated the proportion of hospitalizations that could have been averted had all eligible high-risk adults with SARS-CoV-2 infection in a clinical cohort been treated with an oral SARS-CoV-2 antiviral agent early in infection. Among 3,037 patients with risk factors for progressing to severe COVID-19, 946 (31.1%) received an oral antiviral prescription (834 nirmatrelvir and 112 molnupiravir). Only 3.0% of treated patients vs 9.1% of untreated patients were hospitalized (adjusted risk ratio [RR]=0.27; 95% confidence interval [CI]: 0.18-0.41). If all patients had been treated, an estimated 63.3% (95% CI: 50.4-75.1) of hospitalizations within 30 days could have been prevented. This finding is attributed to large gaps in treatment and a strong protective association of antiviral treatment with subsequent hospitalization.\n",
  "wordCount" : "3340",
  "inLanguage": "en",
  "datePublished": "2024-01-07T10:38:59Z",
  "dateModified": "2024-01-07T10:38:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 7, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.23300575">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.23300575" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.23300575">
        <p class="paperTitle">Reverse mutational scanning of spike BA.2.86 identifies the epitopes contributing to immune escape from polyclonal sera</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.23300575" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.23300575" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bdeir, N.; Lüddecke, T.; Maass, H.; Schmelz, S.; Jacobsen, H.; Metzdorf, K.; Cossmann, A.; Stankov, M.; Hoffmann, M.; Pöhlmann, S.; Blankenfeldt, W.; Jablonka, A.; Behrens, G. M. N.; Cicin-Sain, L.</p>
        <p class="info">Score: 21.1, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.23300575' target='https://doi.org/10.1101/2024.01.03.23300575'> 10.1101/2024.01.03.23300575</a></p>
        <p class="abstract">The recently detected Omicron BA.2.86 lineage contains more than 30 amino acid mutations relative to BA.2. Here, we identify the epitopes driving immune escape of BA.2.86 and its derivative JN.1 (BA.2.86 &#43; S455L) lineage. We investigated the cross-reactive humoral immunity within a cohort of health care workers against Omicron BA.2.86 and JN.1 by employing pseudo-viral mutants. We demonstrate that BA.2.86 and especially JN.1 evaded neutralization by serum antibodies of fully vaccinated individuals. To discern the contribution of individual epitope mutations to immune escape, we constructed a library of 33 BA.2.86 mutants, each of which harbored a single revertant mutation going back to BA.2. This library was used in a reverse mutational scanning approach to define serum neutralization titers against each epitope separately. The mutations within the receptor binding domain (RBD) at position K356T and to a lesser extent the mutations N460K, V483{Delta}, A484K, and F486P enhanced the immune escape. More surprisingly, the mutation 16insMPLF within the spike N-terminal domain (NTD) and the mutation P621S in S1/S2 significantly contributed to antibody escape of BA.2.86. Upon XBB.1.5 booster vaccination, neutralization titers against JN.1 and BA.2.86 improved relative to all ancestral strains, and the residual immune escape was driven by mutations at positions 16insMPLF, {Delta}144Y, E544K, P621S, and A484K.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.24300790">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.24300790" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.24300790">
        <p class="paperTitle">Escalating combinations of enhanced infectivity and immune escape define SARS-CoV-2 Omicron lineage replacement</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.24300790" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.24300790" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, N. F. G.; Pham, K.; Chaguza, C.; Lopes, R.; Klaassen, F.; Weinberger, D. M.; Pitzer, V. E.; Warren, J. L.; Grubaugh, N. D.; Hahn, A. M.</p>
        <p class="info">Score: 7.3, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.24300790' target='https://doi.org/10.1101/2024.01.03.24300790'> 10.1101/2024.01.03.24300790</a></p>
        <p class="abstract">In 2022, consecutive sweeps of the highly transmissible SARS-CoV-2 Omicron-family maintained high viral transmission levels despite extensive antigen exposure on the population level resulting from both vaccinations and infections. To better understand variant fitness in the context of the highly dynamic immunity landscape of 2022, we aimed to dissect the interplay between immunity and fitness advantages of emerging SARS-CoV-2 Omicron lineages on the population-level. We evaluated the relative contribution of higher intrinsic transmissibility or immune escape on the fitness of emerging lineages by analyzing data collected through our local genomic surveillance program from Connecticut, USA. We compared growth rates, estimated infections, effective reproductive rates, average viral copy numbers, and likelihood for causing vaccine break-through infections. Using these population-level data, we find that newly emerging Omicron lineages reach dominance through a specific combination of enhanced intrinsic transmissibility and immune escape that varies over time depending on the state of the host-population. Using similar frameworks that integrate whole genome sequencing together with clinical, laboratory, and epidemiological data can advance our knowledge on host-pathogen dynamics in the post-emergence phase that can be applied to other communicable diseases beyond SARS-CoV-2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.23300577">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.23300577" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.23300577">
        <p class="paperTitle">Diverse and weakly immunogenic var gene expression facilitates malaria infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.23300577" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.23300577" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bhardwaj, I.; Nyarko, P. B.; Ashn, A. B.; Cohen, C.; Ceesay, S.; Achan, J.; Dabira, E.; Nakajima, R.; Jain, A.; Taghavian, O.; Jasinskas, A.; Felgner, P. L.; D&#39;Alessandro, U.; Bousema, T.; Travassos, M.; Radulescu, O.; Claessens, A.</p>
        <p class="info">Score: 5.2, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.23300577' target='https://doi.org/10.1101/2023.12.27.23300577'> 10.1101/2023.12.27.23300577</a></p>
        <p class="abstract">Plasmodium falciparum is believed to escape immunity via antigenic variation, mediated in part by 60 var genes. These genes undergo mutually exclusive expression and encode the PfEMP1 surface antigen. The frequency of var switching and the immunogenicity of each expressed PfEMP1 remain unclear. To this end, we carried out a Controlled Human Malaria Infection (CHMI) study with 19 adult African volunteers in The Gambia to gain insight into the effect of naturally acquired immunity on the expressed var gene repertoire during early phase of an infection. Our findings demonstrated a strong correlation between the diversity of var expression, quantified through entropy, and infection outcome. Low-immunity individuals were characterised by high var entropy profiles, higher parasitaemia, and lower sero-recognised PfEMP1 domains compared to high-immunity individuals. For the first time we recorded the probability of var gene switching in vitro and of turnover in vivo, enabling us to estimate both intrinsic switching and negative-selection effects. These processes are rapid, resulting in estimated turnover/switching probabilities of 69% - 97% and 7% - 57% per generation, in vivo and in vitro, respectively. Var (PfEMP1) expression triggered time-dependent humoral immune responses in low immunity individuals, with many PfEMP1 domains remaining weakly immunogenic. We conclude that the role of intrinsic var switching is to reset and maintain a diverse var repertoire. The high var switching rates and weak PfEMP1 immunogenicity benefit parasite survival during the CHMI.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.23300535">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.23300535" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.23300535">
        <p class="paperTitle">Phylogeographic Inference of SARS-CoV-2 Delta Wave in Texas, USA using a Novel Spatial Transmission Count Statistic</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.23300535" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.23300535" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lyu, L.; Veytsel, G. E.; Stott, G.; Fox, S. J.; Dailey, C. A.; Damodaran, L.; Fujimoto, K.; Kuo, J.; Brown, P.; Sealy, R.; Brown, A.; Alabady, M.; Bahl, J.</p>
        <p class="info">Score: 4.2, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.23300535' target='https://doi.org/10.1101/2023.12.28.23300535'> 10.1101/2023.12.28.23300535</a></p>
        <p class="abstract">Viral genomes contain records of geographic movements and cross-scale transmission dynamics. However, the impact of population heterogeneity, particularly among rural and urban areas, on viral spread and epidemic trajectory has been less explored due to limited data availability. Intensive and widespread efforts to collect and sequence SARS-CoV-2 viral samples have enabled the development of comparative genomic approaches to reconstruct spatial transmission history and understand viral transmission across different scales. Large genomic datasets with few mutations present challenges for traditional phylodynamic approaches. To address this issue, we propose a novel spatial transmission count statistic that efficiently summarizes the geographic transmission patterns imprinted on viral phylogenies. Our analysis pipeline reconstructs a time-scaled phylogeny with ancestral trait states and identifies spatial transmission linkages, categorized as imports, local transmission, and exports. These linkages are summarized to represent the epidemic profile of the focal area. We demonstrate the utility of this approach for near real-time outbreak analysis using over 12,000 full genomes and linked epidemiological data to investigate the spread of the SARS-CoV-2 Delta variant in Texas. Our goal is to trace the Delta variants origin, timing and to understand the role of urban and rural areas in the spatial diffusion patterns observed in Texas. Our study shows (1) highly populated urban centers were the main sources of the epidemic in Texas; (2) the outbreaks in urban centers were connected to the global epidemic; and (3) outbreaks in urban centers were locally maintained, while epidemics in rural areas were driven by repeated introductions.

Significance StatementWe developed a novel phylogeographic approach that analyzes transmission patterns at low computational cost. This method not only facilitates the inference of spatial scales of transmission but also enables exploration of how specific demographic characteristics influence transmission patterns among heterogenous populations. The rural population in the US, comprising approximately 60 million individuals, has been significantly impacted by COVID-19. Applying our new method, we examined the variations in epidemic patterns between urban centers (e.g., Houston) and rural areas in Texas. We found that urban centers are the primary source for SARS-CoV-2 in rural areas. This analysis lays the groundwork for designing effective public health interventions specifically tailored to the needs of affected areas.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.24300703">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.24300703" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.24300703">
        <p class="paperTitle">Thoughts and perspectives of metagenome sequencing as a diagnostic tool for infectious disease: an interpretive phenomenological study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.24300703" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.24300703" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Trivett, H.; Darby, A. C.; Oyebode, O.</p>
        <p class="info">Score: 3.2, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.24300703' target='https://doi.org/10.1101/2024.01.02.24300703'> 10.1101/2024.01.02.24300703</a></p>
        <p class="abstract">BackgroundEffective infectious disease diagnostics (IDD) are vital for informing clinical decision-making regarding the treatment and patient management of disease and infections. Conventional clinical methods rely upon culture-dependent techniques, and there has been little shift in the acceptance and integration of culture-independent sequencing methods into routine clinical IDD. This study explored stakeholders experiences within IDD, including those working in clinical settings and those conducting research at the forefront of microbial genomics. We aimed to identify factors driving the development and implementation of metagenome sequencing as a routine diagnostic.

MethodsVirtual semi-structured interviews were conducted with purposively selected individuals involved in IDD. The interviews explored the experiences of implementing metagenome sequencing as a diagnostic tool and decisions about which diagnostics are used for identifying bacteria-causing infections. Thematic analysis was used to analyse the data, and an Interpretive Phenomenological approach was used throughout.

ResultsTen individuals were interviewed between July 2021 and October 2021, including Clinical scientists, consultants, and professors in academia. Their experience ranged from no knowledge of metagenome sequencing to an expert understanding of the phenomenon. Five themes emerged: Diagnostic Choice, Infrastructure, Open Data Sharing, COVID-19, and Communication. Participants recognised the need for new diagnostics to be implemented to overcome the limitations of current diagnostic approaches but highlighted the barriers to integrating new diagnostics into clinical settings, such as the impact on clinical decision-making, accreditation, and cost. However, participants felt that lessons could be learnt from using metagenomics in COVID-19 and how other diagnostic platforms have been integrated into clinical settings over the last 20 years.

ConclusionsThe study provided clear evidence to address the knowledge gap in current literature and practice for developing and implementing metagenome sequencing as a potential IDD. The knowledge of new and upcoming genomic diagnostic testing is not equally distributed throughout the UK, impacting the understanding and drive to integrate metagenome sequencing into routine clinical diagnostics. Improvements in access to new diagnostics could improve patient treatment and management and positively impact public health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.23300299">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.23300299" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.23300299">
        <p class="paperTitle">COVID-19 pandemic re-shaped the global dispersal of seasonal influenza viruses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.23300299" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.23300299" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, Z.; Tsui, J. L.- H.; Gutierrez, B.; Moreno, S. B.; Plessis, L. d.; Deng, X.; Cai, J.; Bajaj, S.; Suchard, M. A.; Pybus, O. G.; Lemey, P.; Kraemer, M. U. G.; Yu, H.</p>
        <p class="info">Score: 33.5, Published: 2023-12-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.23300299' target='https://doi.org/10.1101/2023.12.20.23300299'> 10.1101/2023.12.20.23300299</a></p>
        <p class="abstract">Understanding how the global dispersal patterns of seasonal influenza viruses were perturbed during and after the COVID-19 pandemic is needed to inform influenza intervention and vaccination strategies in the post-pandemic period. Although global human mobility has been identified as a key driver of influenza dispersal1, alongside climatic and evolutionary factors2,3, the impact of international travel restrictions on global influenza transmission and recovery remains unknown. Here we combine molecular, epidemiological, climatic, and international travel data within a phylodynamic framework to show that, despite human mobility remaining the principal driver of global influenza virus dissemination, the pandemics onset led to a shift in the international population structure and migration network of seasonal influenza lineages. We find that South Asia and Africa played important roles as exporters and phylogenetic trunk locations of influenza in 2020 and 2021, and we highlight the association between population movement, antigenic drift and persistence during the intensive non-pharmaceutical interventions (NPIs) phase. The influenza B/Yamagata lineage disappeared in a context of reduced relative genetic diversity, moderate lineage turnover, and lower positive selection pressure. Our results demonstrate that mobility perturbations reshaped the global dispersal dynamics of influenza viruses, with potential implications for vaccine design and genomic surveillance programmes. As the risk of future pandemics persists, our study provides an opportunity to assess the impact of NPIs during the pandemic on respiratory infectious diseases beyond the interplay between SARS-CoV-2 and influenza viruses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.24300729">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.24300729" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.24300729">
        <p class="paperTitle">Trends in SARS-CoV-2 seroprevalence among pregnant women attending first antenatal care visits in Zambia: a repeated cross-sectional survey, 2021-2022</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.24300729" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.24300729" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heilmann, E.; Tembo, T.; Fwoloshi, S.; Kabamba, B.; Chilambe, F.; Kalenga, K.; Siwingwa, M.; Mulube, C.; Seffren, V.; Bolton-Moore, C.; Simwanza, J.; Yingst, S.; Yadav, R.; Rogier, E.; Auld, A. F.; Agolory, S.; Kapina, M.; Gutman, J. R.; Savory, T.; Kangale, C.; Mulenga, L. B.; Sikazwe, I.; Hines, J. Z.</p>
        <p class="info">Score: 2.5, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.24300729' target='https://doi.org/10.1101/2024.01.02.24300729'> 10.1101/2024.01.02.24300729</a></p>
        <p class="abstract">BackgroundSARS-CoV-2 serosurveys help estimate the extent of transmission and guide allocation of COVID-19 vaccines. We measured SARS-CoV-2 seroprevalence among women attending ANC clinics to assess exposure trends over time in Zambia.

MethodsWe conducted repeated cross-sectional surveys among pregnant women aged 15-49 years attending their first ANC visits in four districts of Zambia (two urban and two rural) during September 2021-September 2022. Serologic testing was done using a multiplex bead assay which detects IgG antibodies to the nucleocapsid protein and the spike protein receptor-binding domain (RBD). We calculated monthly SARS-CoV-2 seroprevalence by district. We also categorized seropositive results as infection alone, infection and vaccination, or vaccination alone based on COVID-19 vaccination status and anti-RBD and anti-nucleocapsid test results.

FindingsAmong 8,304 participants, 5,296 (63.8%) were cumulatively seropositive for SARS-CoV-2 antibodies. SARS-CoV-2 seroprevalence primarily increased from September 2021 to September 2022 in three districts (Lusaka: 61.8-100.0%, Chongwe: 39.6-94.7%, Chipata: 56.5-95.0%), but in Chadiza, seroprevalence increased from 27.8% in September 2021 to 77.2% in April 2022 before gradually dropping to 56.6% in July 2022. Among 5,906 participants with a valid COVID-19 vaccination status, infection alone accounted for antibody responses in 77.7% (4,590) of participants.

InterpretationMost women attending ANC had evidence of prior SARS-CoV-2 infection and most SARS-CoV-2 seropositivity was infection-induced. Capturing COVID-19 vaccination status and using a multiplex bead assay with anti-nucleocapsid and anti-RBD targets facilitated distinguishing infection-induced versus vaccine-induced antibody responses during a period of increasing COVID-19 vaccine coverage in Zambia. Declining seroprevalence in Chadiza may indicate waning antibodies and a need for booster vaccines. ANC clinics have a potential role in ongoing SARS-CoV-2 serosurveillance and can continue to provide insights into SARS-CoV-2 antibody dynamics to inform near real-time public health responses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.10.23299786">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.10.23299786" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.10.23299786">
        <p class="paperTitle">Prediction of post-PCV13 pneumococcal evolution using invasive disease data enhanced by inverse-invasiveness weighting</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.10.23299786" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.10.23299786" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Qiu, X.; McGee, L.; Hammitt, L. L.; Grant, L. R.; O&#39;Brien, K. L.; Hanage, W. P.; Lipsitch, M.</p>
        <p class="info">Score: 1.5, Published: 2023-12-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.10.23299786' target='https://doi.org/10.1101/2023.12.10.23299786'> 10.1101/2023.12.10.23299786</a></p>
        <p class="abstract">BackgroundAfter introduction of pneumococcal conjugate vaccines (PCVs), serotype replacement occurred in the population of Streptococcus pneumoniae. Predicting which pneumococcal clones and serotypes will become more common in carriage after vaccination can enhance vaccine design and public health interventions, while also improving our understanding of pneumococcal evolution. We sought to use invasive disease data to assess how well negative frequency-dependent selection (NFDS) models could explain pneumococcal carriage population evolution in the post-PCV13 epoch by weighting invasive data to approximate strain proportions in the carriage population.

MethodsInvasive pneumococcal isolates were collected and sequenced during 1998-2018 by the Active Bacterial Core surveillance (ABCs) from the Centers for Disease Control and Prevention (CDC). To predict the post-PCV13 population dynamics in the carriage population using a NFDS model, all genomic data were processed under a bioinformatic pipeline of assembly, annotation, and pangenome analysis to define genetically similar sequence clusters (i.e., strains) and a set of accessory genes present in 5% to 95% of the isolates. The NFDS model predicted the strain proportion by calculating the post-vaccine strain composition in the weighted invasive disease population that would best match pre-vaccine accessory gene frequencies. To overcome the biases of invasive disease data, serotype-specific inverse-invasiveness weights were defined as the ratio of the proportion of the serotype in the carriage data to the proportion in the invasive data, using data from 1998-2001 in the United States, before conjugate vaccine introduction. The weights were applied to adjust both the observed strain proportion and the accessory gene frequencies.

ResultsInverse-invasiveness weighting increased the correlation of accessory gene frequencies between invasive and carriage data with reduced residuals in linear or logit scale for pre-vaccine, post-PCV7, and post-PCV13. Similarly, weighting increased the correlation of accessory gene frequencies between different time periods in the invasive data. By weighting the invasive data, we were able to use the NFDS model to predict strain proportions in the carriage population in the post-PCV13 epoch, with the adjusted R-squared between predicted and observed strain proportions increasing from 0.176 to 0.544 after weighting.

ConclusionsThe weighting system adjusted the invasive disease surveillance data to better represent the carriage population of S. pneumoniae. The NFDS mechanism predicted the strain proportions in the projected carriage population as estimated from the weighted invasive disease frequencies in the post-PCV13 epoch. Our methods enrich the value of genomic sequences from invasive disease surveillance, which is readily available, easy to collect, and of direct interest to public health.

IMPORTANCEStreptococcus pneumoniae, a common colonizer in the human nasopharynx, can cause invasive diseases including pneumonia, bacteremia, and meningitis mostly in children under 5 years or older adults. The PCV7 was introduced in 2000 in the United States within the pediatric population to prevent disease and reduce deaths, followed by PCV13 in 2010, PCV15 in 2022, and PCV20 in 2023. After the removal of vaccine serotypes, the prevalence of carriage remained stable as the vacated pediatric ecological niche was filled with certain non-vaccine serotypes. Predicting which pneumococcal clones, and which serotypes, will be most successful in colonization after vaccination can enhance vaccine design and public health interventions, while also improving our understanding of pneumococcal evolution. While carriage data, which are collected from the pneumococcal population that is competing to colonize and transmit, are most directly relevant to evolutionary studies, invasive disease data are often more plentiful. Previously, evolutionary models based on negative frequency-dependent selection (NFDS) on the accessory genome were shown to predict which non-vaccine strains and serotypes were most successful in colonization following the introduction of PCV7. Here, we show that an inverse-invasiveness weighting system applied to invasive disease surveillance data allows the NFDS model to predict strain proportions in the projected carriage population in the post-PCV13/pre-PCV15 and -PCV20 epoch. The significance of our research lies in using a sample of invasive disease surveillance data to extend the use of NFDS as an evolutionary mechanism to predict post-PCV13 population dynamics. This has shown that we can correct for biased sampling that arises from differences in virulence and can enrich the value of genomic data from disease surveillance and advances our understanding of how NFDS impacts carriage population dynamics after both PCV7 and PCV13 vaccination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.23299623">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.23299623" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.23299623">
        <p class="paperTitle">Lack of association between HLA and asymptomatic SARS-CoV-2 infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.23299623" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.23299623" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Marchal, A.; Cirulli, E. T.; Neveux, I.; Bellos, E.; Thwaites, R. S.; Schiabor Barrett, K. M.; Zhang, Y.; Nemes-Bokun, I.; Kalinova, M.; Catchpole, A.; Tangye, S. G.; Spaan, A. N.; Lack, J. B.; Ghosn, J.; Burdet, C.; Gorochov, G.; Tubach, F.; Hausfater, P.; COVID Human Genetic Effort,  ; COVIDeF Study Group,  ; French COVID Cohort Study Group,  ; CoV-Contact Cohort,  ; COVID-STORM Clinicians,  ; COVID Clinicians,  ; Orchestra Working Group,  ; Amsterdam UMC Covid-19 Biobank,  ; NIAID-USUHS COVID Study Group,  ; Dalgard, C. L.; Zhang, S.-Y.; Zhang, Q.; Chiu, C.; Fellay, J.; Grzymski, J. J.; San</p>
        <p class="info">Score: 49.6, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.23299623' target='https://doi.org/10.1101/2023.12.06.23299623'> 10.1101/2023.12.06.23299623</a></p>
        <p class="abstract">Human genetic studies of critical COVID-19 pneumonia have revealed the essential role of type I interferon-dependent innate immunity to SARS-CoV-2 infection. Conversely, an association between the HLA-B*15:01 allele and asymptomatic SARS-CoV-2 infection in unvaccinated individuals was recently reported, suggesting a contribution of pre-existing T cell-dependent adaptive immunity. We report a lack of association of classical HLA alleles, including HLA-B*15:01, with pre-omicron asymptomatic SARS-CoV-2 infection in unvaccinated participants in a prospective population-based study in the US (191 asymptomatic vs. 945 symptomatic COVID-19 cases). Moreover, we found no such association in the international COVID Human Genetic Effort cohort (206 asymptomatic vs. 574 mild or moderate COVID-19 cases and 1,625 severe or critical COVID-19 cases). Finally, in the Human Challenge Characterisation study, the three HLA-B*15:01 individuals infected with SARS-CoV-2 developed symptoms. As with other acute primary infections, no classical HLA alleles favoring an asymptomatic course of SARS-CoV-2 infection were identified. These findings suggest that memory T-cell immunity to seasonal coronaviruses does not strongly influence the outcome of SARS-CoV-2 infection in unvaccinated individuals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.23300241">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.23300241" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.23300241">
        <p class="paperTitle">Proportion of Hospitalizations Preventable with Increased Oral SARS-CoV-2 Antiviral Treatment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.23300241" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.23300241" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Levy, M. E.; Chilunda, V.; Heaton, P. R.; Grzymski, J.; Pawloski, P. A.; Goldman, J. D.; Luo, S.</p>
        <p class="info">Score: 2.9, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.23300241' target='https://doi.org/10.1101/2023.12.19.23300241'> 10.1101/2023.12.19.23300241</a></p>
        <p class="abstract">We estimated the proportion of hospitalizations that could have been averted had all eligible high-risk adults with SARS-CoV-2 infection in a clinical cohort been treated with an oral SARS-CoV-2 antiviral agent early in infection. Among 3,037 patients with risk factors for progressing to severe COVID-19, 946 (31.1%) received an oral antiviral prescription (834 nirmatrelvir and 112 molnupiravir). Only 3.0% of treated patients vs 9.1% of untreated patients were hospitalized (adjusted risk ratio [RR]=0.27; 95% confidence interval [CI]: 0.18-0.41). If all patients had been treated, an estimated 63.3% (95% CI: 50.4-75.1) of hospitalizations within 30 days could have been prevented. This finding is attributed to large gaps in treatment and a strong protective association of antiviral treatment with subsequent hospitalization.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
