<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Hematologic abnormalities after COVID-19 vaccination: A large Korean population-based cohort study
Authors: CHOI, H. S.; KIM, M.-H.; CHOI, M. G.; PARK, J. H.; CHUN, E. M.
Score: 1629.5, Published: 2023-11-22 DOI: 10.1101/2023.11.15.23298565
Adverse hematologic events have been reported after COVID-19 vaccination. The objective of this study was to investigate whether hematologic abnormalities develop after COVID-19 vaccination. Retrospective cohort analyses of data from the Korean National Health Insurance Service (KNHIS) database were conducted from July 2022 to August 2023.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Hematologic abnormalities after COVID-19 vaccination: A large Korean population-based cohort study
Authors: CHOI, H. S.; KIM, M.-H.; CHOI, M. G.; PARK, J. H.; CHUN, E. M.
Score: 1629.5, Published: 2023-11-22 DOI: 10.1101/2023.11.15.23298565
Adverse hematologic events have been reported after COVID-19 vaccination. The objective of this study was to investigate whether hematologic abnormalities develop after COVID-19 vaccination. Retrospective cohort analyses of data from the Korean National Health Insurance Service (KNHIS) database were conducted from July 2022 to August 2023." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-10T10:37:48+00:00" />
<meta property="article:modified_time" content="2023-12-10T10:37:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Hematologic abnormalities after COVID-19 vaccination: A large Korean population-based cohort study
Authors: CHOI, H. S.; KIM, M.-H.; CHOI, M. G.; PARK, J. H.; CHUN, E. M.
Score: 1629.5, Published: 2023-11-22 DOI: 10.1101/2023.11.15.23298565
Adverse hematologic events have been reported after COVID-19 vaccination. The objective of this study was to investigate whether hematologic abnormalities develop after COVID-19 vaccination. Retrospective cohort analyses of data from the Korean National Health Insurance Service (KNHIS) database were conducted from July 2022 to August 2023."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Hematologic abnormalities after COVID-19 vaccination: A large Korean population-based cohort study\nAuthors: CHOI, H. S.; KIM, M.-H.; CHOI, M. G.; PARK, J. H.; CHUN, E. M.\nScore: 1629.5, Published: 2023-11-22 DOI: 10.1101/2023.11.15.23298565\nAdverse hematologic events have been reported after COVID-19 vaccination. The objective of this study was to investigate whether hematologic abnormalities develop after COVID-19 vaccination. Retrospective cohort analyses of data from the Korean National Health Insurance Service (KNHIS) database were conducted from July 2022 to August 2023.",
  "keywords": [
    
  ],
  "articleBody": " Hematologic abnormalities after COVID-19 vaccination: A large Korean population-based cohort study\nAuthors: CHOI, H. S.; KIM, M.-H.; CHOI, M. G.; PARK, J. H.; CHUN, E. M.\nScore: 1629.5, Published: 2023-11-22 DOI: 10.1101/2023.11.15.23298565\nAdverse hematologic events have been reported after COVID-19 vaccination. The objective of this study was to investigate whether hematologic abnormalities develop after COVID-19 vaccination. Retrospective cohort analyses of data from the Korean National Health Insurance Service (KNHIS) database were conducted from July 2022 to August 2023. We randomly selected data of half of those living in Seoul City as of January 1, 2021 with their diagnostic records up to December 31, 2021. The included participants were vaccinated and nonvaccinated persons aged 20 years or older (n= 4,203,887). Hematologic abnormalities after COVID-19 vaccination were identified as nutritional anemia, hemolytic anemia, aplastic anemia, coagulation defects, and neutropenia using International Classification of Diseases, Tenth Revision codes after index date. Incidence rates of hematologic abnormalities in the vaccination group 3 months after vaccination were significantly higher than those in the nonvaccinated group: 14.79 vs. 9.59 (P\u003c.001) for nutritional anemia, 7.83 vs. 5.00 (P\u003c.001) for aplastic anemia, and 4.85 vs. 1.85 (P\u003c.001) for coagulation defects. COVID-19 mRNA vaccine was associated with higher development of nutritional anemia (odds ratio [OR], 1.230 [95% CI, 1.129-1.339], P\u003c.001) and aplastic anemia (OR, 1.242 [95% CI, 1.110-1.390], P\u003c.001) than the viral vector vaccine. The risk of coagulation defects was increased (OR, 1.986 [95% CI, 1.523-2.589], P\u003c.001) after vaccination, and there was no risk difference between mRNA vaccine and viral vector vaccine (OR, 1.075 [95% CI, 0.936-1.233], P=.306). In conclusions, COVID-19 vaccination increased the risk of hematologic abnormalities. When administering the COVID-19 vaccine, careful observation will be necessary after vaccination.\nThe spectrum of non-fatal immune-related adverse events following COVID-19 vaccination: The population-based cohort study in Seoul, South Korea\nAuthors: Suh, J. H.; Kim, H. J.; Kim, M.-H.; Choi, M. G.; Chun, E. M.\nScore: 503.1, Published: 2023-11-22 DOI: 10.1101/2023.11.15.23298566\nObjectiveTo examine the incidence rate and risk of non-fatal irAEs, including gynecological, hematological, dermatological, ophthalmological, otologic, and dental problems following the COVID-19 vaccination. MethodsWe conducted a population-based cohort study from the National Health Insurance Service (NHIS) database in Seoul, South Korea. The non-fatal irAEs included gynecological, hematological, dermatological, ophthalmological, ear, and periodontal problems as reported by the Vaccine Adverse Event Reporting Center. The cumulative incidence rate per 10,000 population, Odds ratio, and Hazard ratio (HR) with 95% Confidence Interval (CI) were measured to assess the non-fatal irAEs after COVID-19 vaccination. ResultsThe cIR of non-fatal irAEs for three months was significantly higher in vaccinated subjects than in non-vaccinated subjects, except for endometriosis. The vaccination significantly increased the risks of all the non-fatal irAEs except for visual impairment. The risk of inner ear disease showed the highest HRs (HR [95% CI] = 2.368 [2.153-2.604]) among the non-fatal irAEs following COVID-19 vaccination. Among the vaccinated subjects, heterologous vaccination was associated with the increased risk of most of the non-fatal irAEs. ConclusionsThe three-month risks of incidental non-fatal irAEs are substantially higher in the COVID-19 vaccinated subjects than in non-vaccinated controls. Our findings suggested that vaccinated subjects with predisposition are potentially vulnerable to the occurrence of diverse irAEs although the COVID-19 vaccines may not be fatal.\nLack of association between HLA and asymptomatic SARS-CoV-2 infection\nAuthors: Marchal, A.; Cirulli, E. T.; Neveux, I.; Bellos, E.; Thwaites, R. S.; Schiabor Barrett, K. M.; Zhang, Y.; Nemes-Bokun, I.; Kalinova, M.; Catchpole, A.; Tangye, S. G.; Spaan, A. N.; Lack, J. B.; Ghosn, J.; Burdet, C.; Gorochov, G.; Tubach, F.; Hausfater, P.; COVID Human Genetic Effort, ; COVIDeF Study Group, ; French COVID Cohort Study Group, ; CoV-Contact Cohort, ; COVID-STORM Clinicians, ; COVID Clinicians, ; Orchestra Working Group, ; Amsterdam UMC Covid-19 Biobank, ; NIAID-USUHS COVID Study Group, ; Dalgard, C. L.; Zhang, S.-Y.; Zhang, Q.; Chiu, C.; Fellay, J.; Grzymski, J. J.; San\nScore: 41.8, Published: 2023-12-08 DOI: 10.1101/2023.12.06.23299623\nHuman genetic studies of critical COVID-19 pneumonia have revealed the essential role of type I interferon-dependent innate immunity to SARS-CoV-2 infection. Conversely, an association between the HLA-B*15:01 allele and asymptomatic SARS-CoV-2 infection in unvaccinated individuals was recently reported, suggesting a contribution of pre-existing T cell-dependent adaptive immunity. We report a lack of association of classical HLA alleles, including HLA-B*15:01, with pre-omicron asymptomatic SARS-CoV-2 infection in unvaccinated participants in a prospective population-based study in the US (191 asymptomatic vs. 945 symptomatic COVID-19 cases). Moreover, we found no such association in the international COVID Human Genetic Effort cohort (206 asymptomatic vs. 574 mild or moderate COVID-19 cases and 1,625 severe or critical COVID-19 cases). Finally, in the Human Challenge Characterisation study, the three HLA-B*15:01 individuals infected with SARS-CoV-2 developed symptoms. As with other acute primary infections, no classical HLA alleles favoring an asymptomatic course of SARS-CoV-2 infection were identified. These findings suggest that memory T-cell immunity to seasonal coronaviruses does not strongly influence the outcome of SARS-CoV-2 infection in unvaccinated individuals.\nNanopore sequencing of influenza A and B in Oxfordshire and the United Kingdom, 2022-23\nAuthors: Cane, J.; Sanderson, N.; Barnett, S.; Vaughan, A.; Pott, M.; Kapel, N.; Morgan, M.; Jesuthasan, G.; Samuel, R.; Ehsaan, M.; Boothe, H.; Haduli, E.; Studley, R.; Rourke, E.; Diamond, I.; Fowler, T.; Watson, C.; Stoesser, N.; Walker, A. S.; Street, T.; Eyre, D. W.\nScore: 9.6, Published: 2023-11-22 DOI: 10.1101/2023.11.21.23298840\nObjectivesWe evaluated Nanopore sequencing for influenza surveillance. MethodsInfluenza A and B PCR-positive samples from hospital patients in Oxfordshire, UK, and a UK-wide population survey from winter 2022-23 underwent Nanopore sequencing following targeted rt-PCR amplification. ResultsFrom 941 infections, successful sequencing was achieved in 292/388(75%) available Oxfordshire samples: 231(79%) A/H3N2, 53(18%) A/H1N1, and 8(3%) B/Victoria and in 53/113(47%) UK-wide samples. Sequencing was more successful at lower Ct values. Most same-sample replicate sequences had identical haemagglutinin segments (124/141;88%); a subset of samples also Illumina sequenced were very similar to Nanopore sequences. Comparison of Oxfordshire and UK-wide sequences showed frequent inter-regional transmission. Infections were closely-related to 2022-23 vaccine strains. Only one sample had a neuraminidase inhibitor resistance mutation. 849/941(90%) Oxfordshire infections were community-acquired. 63/88(72%) potentially healthcare-associated cases shared a hospital ward with [\u0026ge;]1 known infectious case. 33 epidemiologically-plausible transmission links had sequencing data for both source and recipient: 8 were within [\u0026le;]5 SNPs, of these, 5(63%) involved potential sources that were also hospital-acquired. ConclusionsNanopore influenza sequencing was reproducible and antiviral resistance rare. Inter-regional transmission was common; most infections were genomically similar. Hospital-acquired infections are likely an important source of nosocomial transmission and should be prioritised for infection prevention and control. HighlightsO_LINanopore sequencing is a reproducible tool for influenza surveillance C_LIO_LIInter-regional transmission of influenza was common across the UK C_LIO_LIInfluenza anti-viral resistance was rare C_LIO_LIIn 1 year most infections were genetically similar, hindering transmission studies C_LIO_LIHospital-acquired infections are likely a key source of nosocomial transmission C_LI\nPrimary exposure to Zika virus increases risk of symptomatic dengue virus infection with serotypes 2, 3, and 4 but not serotype 1\nAuthors: Zambrana, J. V.; Hasund, C. M.; Aogo, R. A.; Bos, S.; Gonzalez, K.; Collado, D.; Miranda, T.; Kuan, G.; Arguello, S.; Balmaseda, A.; Katzelnick, L.; Harris, E.\nScore: 7.8, Published: 2023-11-30 DOI: 10.1101/2023.11.29.23299187\nInfection with any of the four dengue virus serotypes (DENV1-4) can protect against or enhance subsequent dengue depending on pre-existing antibodies and the subsequent infecting serotype. Additionally, primary infection with the related flavivirus Zika virus (ZIKV) has been shown to increase DENV2 disease. Here, we measured how prior DENV and ZIKV immunity influenced risk of disease caused by all four serotypes in a pediatric Nicaraguan cohort. Of 3,412 participants in 2022, 10.6% experienced symptomatic DENV infections caused by DENV1 (n=139), DENV4 (n=133), DENV3 (n=54), DENV2 (n=9), or an undetermined serotype (n=39). Longitudinal clinical and serological data were used to define infection histories, and generalized linear and additive models adjusted for age, sex, time since the last infection, cohort year, and repeat measurements were used to predict disease risk. Compared to flavivirus-naive participants, primary ZIKV infection increased disease risk of DENV4 (relative risk = 2.62, 95% confidence interval: 1.48-4.63) and DENV3 (2.90, 1.34-6.27) but not DENV1 (1.20, 0.72-1.99). Primary DENV infection or a DENV followed by ZIKV infection also increased DENV4 risk. We re-analyzed 19 years of cohort data and demonstrated that prior flavivirus-immunity and pre- existing antibody titer differentially affected disease risk for incoming serotypes, increasing risk of DENV2 and DENV4, protecting against DENV1, and protecting at high titers but enhancing at low titers against DENV3. We thus find that prior ZIKV infection, like prior DENV infection, increases risk of certain DENV serotypes. Cross-reactivity among flaviviruses should be carefully considered when assessing vaccine safety and efficacy. One-Sentence SummaryDengue disease risk is differentially modulated depending on pre- existing immunity to dengue and Zika virus infections and the secondary infecting serotype.\nDiagnostic Performance of Rapid Antigen Testing for SARS-CoV-2: The COVid-19 AntiGen (COVAG) Extension study\nAuthors: Wertenauer, C.; Dressel, A.; Wieland, E.; Wertenauer, H.-J.; Braitmaier, H.; Straub, A.; Luetzner, N.; Maerz, W.\nScore: 13.6, Published: 2023-11-29 DOI: 10.1101/2023.11.29.23299183\nBackgroundRapid antigen tests (RATs) for SARS-CoV-2 have been used to combat the still ongoing Covid-19 pandemic. This study is the extension of the COVAG study originally performed from February 1 to March 31, 2021. We compared two RATs, the Panbio COVID-19 Ag Rapid Test (Abbott) and the SD Biosensor Q SARS-CoV-2 Rapid Antigen Test (Roche), against RT-PCR on the foil of new variants. MethodsWe included 888 all-comers at a diagnostic center between October 20, 2021, and March 18, 2022. RT-PCR-positive samples with a Ct value [\u0026le;] 32 were examined for SARS-CoV-2 variants. FindingsThe sensitivity of the Abbott-RAT and Roche-RAT were 65% and 67%, respectively. For both RATs, lower Ct values were significantly correlated with higher sensitivity. For samples with Ct values [\u0026le;] 25, the sensitivities of the Roche-RAT and of the Abbott-RAT were 96% and 95%, for Ct values 25-30 both were 19%, and for Ct values [\u0026ge;] 30 they were 6% and 2%, respectively. The RATs had substantially higher sensitivities in symptomatic than asymptomatic participants (76, 77%, vs. 29, 31%, for Abbott-RAT, Roche-RAT, respectively) and in participants referred to testing by their primary care physician (84%, 85%) compared to participants who sought testing due to referral by the health department (55%, 58%) or a warning by the Corona-Warn-App (49%, 49%). In persons with self-reported previous Covid-19 sensitivities were markedly lower than in patients without previous Covid-19: 27% vs. 75% for Roche-RAT and 27% vs. 73% for Abbott-RAT. Depending on the vaccination status, the sensitivity of the RATs is 67.6%, 61.5% and 70.6% for non-vaccinated, vaccinated and boostered participants, respectively. For the considered subpopulation of 888 participants, we find no significant correlation between vaccination status and sensitivity. The Omicron variant was detected with a sensitivity of 94% and 92%, the delta variant with a sensitivity of 80% and 80% for Abbott-RAT and Roche-RAT, respectively. This difference is attributable to the lower Ct values of the Omicron samples compared to the Delta samples. When adjusted for the Ct value, a multivariate logistic regression did not show a significant difference between Omicron and Delta. In terms of sensitivity, we found no significant difference between the wild-type and the Omicron and Delta variants, but a significantly lower sensitivity to the alpha variant compared to the other variants. For a Ct value [\u0026le;] 25 the sensitivities were 95.2% and 96.0% for the Abbott-RAT and the Roche-RAT, respectively (Table 4). For a Ct value of 25-30 both RATs had a sensitivity of 18.8%. For a Ct value of 30-32, the sensitivities were 0.0% and 7.1% respectively, for Ct values [\u0026ge;]32 the sensitivities were 3.0% and 6.0% for Abbott-RAT and Roche-RAT, respectively. The specificities were \u003e99% overall. Interpretation: The sensitivity of the RATs for asymptomatic carriers is unsatisfactory questioning their use for screening. When used in symptomatic patients or when requested by a primary care physician the sensitivities were higher. Our study does not suggest that the vaccination status influences the sensitivity of RATs.\nStrain Differences in Bloodstream and Skin Infection MRSA isolated between 2019-2021 in a Single Health System\nAuthors: Hofstetter, K. S.; Jacko, N. F.; Shumaker, M. J.; Talbot, B. M.; Petit, R. A.; Read, T. D.; David, M. Z.\nScore: 2.9, Published: 2023-12-03 DOI: 10.1101/2023.12.02.23299293\nS. aureus is a common cause of skin and soft tissue infections (SSTIs) and has recently become the most common cause of bloodstream infections (BSIs), but whether the strains causing these two clinical syndromes overlap has not been studied adequately. USA300/500 (clonal complex [CC] 8-ST8) and USA100 (CC5-ST5) have dominated among methicillin-resistant S. aureus (MRSA) strains in the U.S. since the early 2000s. We compared the genomes of unselected MRSA isolates from 131 SSTIs with those from 145 BSIs at a single U.S. center in overlapping periods in 2018-2021. CC8 MRSA was more common among SSTIs and CC5 was more common among BSIs, consistent with prior literature. Based on clustering genomes with a threshold of 15 single nucleotide polymorphisms (SNPs), we identified clusters limited to SSTI patients and separate clusters exclusively comprising BSI patients. However, we additionally identified 8 outbreak clusters that included at least one SSTI and one BSI isolate. This suggests that virulent MRSA strains are transmitted from person-to-person locally in the healthcare setting and the community and that single lineages are often capable of causing both SSTIs and BSIs.\nAn amplicon-based Illumina and nanopore sequencing workflow for Chikungunya virus West Africa genotype.\nAuthors: Dieng, I.; Ndiaye, M.; Kane, M.; Balde, D.; Mbanne, M.; Diop, S. M. B. S.; Sankhe, S.; Dia, M.; Dieng, M.; Doukanda, S. F. M.; Faye, O.; Sall, A. A.; Dia, N.; Fall, G.; Faye, O.; Diagne, M. M.\nScore: 2.5, Published: 2023-12-09 DOI: 10.1101/2023.12.07.23299611\nThe Chikungunya virus, a global arbovirus, is currently causing a major outbreak in the Western African region, with the highest cases reported in Senegal and Burkina Faso. Recent molecular evolution analyses reveal that the strain responsible for the epidemic belongs to the West African genotype, with new mutations potentially impacting viral replication, antigenicity, and host adaptation. Real-time genomic monitoring is needed to track the virus's spread in new regions. A scalable West African genotype amplicon-based Whole Genome Sequencing for multiple Next Generation Sequencing platforms has been developed to support genomic investigations and identify epidemiological links during the virus's ongoing spread. This technology will help identify potential threats and support real-time genomic investigations in the ongoing spread of the virus.\nEffectiveness of Bivalent Boosters Over Nine and Half Months in Nebraska\nAuthors: Paritala, S.; Xu, Y.; Du, Y.; Donahue, M.; Maloney, P.; Lin, D.-Y.\nScore: 2.1, Published: 2023-12-04 DOI: 10.1101/2023.12.03.23299338\nOn September 1, 2022, Moderna and Pfizer-BioNTech bivalent vaccines replaced existing monovalent vaccines as booster doses against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) for persons aged 12 and older in the United States. We assessed the effectiveness of these bivalent boosters against Omicron infection and severe outcomes (COVID-19 hospitalization and death) over a 9.5-month period using line-level data from the state of Nebraska. We found that the relative effectiveness of bivalent boosters, compared with one fewer vaccine dose, against Omicron infection and subsequent death was 39.0% (95% confidence interval [CI], 35.7 to 42.2) and 70.0% (95% CI, 38.8 to 85.3), respectively, at four weeks post administration and gradually waned afterward. Vaccine effectiveness was broadly similar against different Omicron subvariants.\nThe Antiviral Mechanism of Action of Molnupiravir in Humans with COVID-19\nAuthors: Sheahan, T. P.; Stevens, L. J.; Narowski, T. M.; Krajewski, T. J.; Lee, C.; Mollan, K. R.; Gribble, J.; Moreria, F. R.; Castillo, I. N.; Cuadra, E.; Alabanza, P.; James Loftis, A.; Coombs, R. W.; Goecker, E. A.; Greninger, A. L.; Chappell, J. D.; Brown, A. J.; Won, J.; Lipansky, F.; Holman, W.; Szewczyk, L. J.; Baric, R. S.; Painter, W. P.; Eron, J. J.; Premkumar, L.; Denison, M. R.; Fischer, W. A.\nScore: 3.5, Published: 2023-11-27 DOI: 10.1101/2023.11.21.23298766\nMeaningful metrics of antiviral activity are essential for determining the efficacy of therapeutics in human clinical trials. Molnupiravir (MOV) is a broadly acting antiviral nucleoside analog prodrug that acts as a competitive alternative substrate for the SARS-CoV-2 RNA-dependent RNA polymerase (RdRp). We developed an assay, Culture-PCR, to better understand the impact of MOV therapy on infectious SARS-CoV-2. Culture-PCR revealed MOV eliminated infectious virus within 48 hours in the nasopharyngeal compartment, the upper airway location with the greatest levels of infectious virus. MOV therapy was associated with increases in mutations across the viral genome but select regions were completely unaffected, thus identifying regions where mutation likely abrogates infectivity. MOV therapy did not alter the magnitude or neutralization capacity of the humoral immune response, a documented correlate of protection. Thus, we provide holistic insights into the function of MOV in adults with COVID-19.\n",
  "wordCount" : "2754",
  "inLanguage": "en",
  "datePublished": "2023-12-10T10:37:48Z",
  "dateModified": "2023-12-10T10:37:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 10, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.23298565">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.23298565" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.23298565">
        <p class="paperTitle">Hematologic abnormalities after COVID-19 vaccination: A large Korean population-based cohort study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.23298565" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.23298565" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: CHOI, H. S.; KIM, M.-H.; CHOI, M. G.; PARK, J. H.; CHUN, E. M.</p>
        <p class="info">Score: 1629.5, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.23298565' target='https://doi.org/10.1101/2023.11.15.23298565'> 10.1101/2023.11.15.23298565</a></p>
        <p class="abstract">Adverse hematologic events have been reported after COVID-19 vaccination. The objective of this study was to investigate whether hematologic abnormalities develop after COVID-19 vaccination. Retrospective cohort analyses of data from the Korean National Health Insurance Service (KNHIS) database were conducted from July 2022 to August 2023. We randomly selected data of half of those living in Seoul City as of January 1, 2021 with their diagnostic records up to December 31, 2021. The included participants were vaccinated and nonvaccinated persons aged 20 years or older (n= 4,203,887). Hematologic abnormalities after COVID-19 vaccination were identified as nutritional anemia, hemolytic anemia, aplastic anemia, coagulation defects, and neutropenia using International Classification of Diseases, Tenth Revision codes after index date. Incidence rates of hematologic abnormalities in the vaccination group 3 months after vaccination were significantly higher than those in the nonvaccinated group: 14.79 vs. 9.59 (P&lt;.001) for nutritional anemia, 7.83 vs. 5.00 (P&lt;.001) for aplastic anemia, and 4.85 vs. 1.85 (P&lt;.001) for coagulation defects. COVID-19 mRNA vaccine was associated with higher development of nutritional anemia (odds ratio [OR], 1.230 [95% CI, 1.129-1.339], P&lt;.001) and aplastic anemia (OR, 1.242 [95% CI, 1.110-1.390], P&lt;.001) than the viral vector vaccine. The risk of coagulation defects was increased (OR, 1.986 [95% CI, 1.523-2.589], P&lt;.001) after vaccination, and there was no risk difference between mRNA vaccine and viral vector vaccine (OR, 1.075 [95% CI, 0.936-1.233], P=.306). In conclusions, COVID-19 vaccination increased the risk of hematologic abnormalities. When administering the COVID-19 vaccine, careful observation will be necessary after vaccination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.23298566">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.23298566" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.23298566">
        <p class="paperTitle">The spectrum of non-fatal immune-related adverse events following COVID-19 vaccination: The population-based cohort study in Seoul, South Korea</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.23298566" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.23298566" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Suh, J. H.; Kim, H. J.; Kim, M.-H.; Choi, M. G.; Chun, E. M.</p>
        <p class="info">Score: 503.1, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.23298566' target='https://doi.org/10.1101/2023.11.15.23298566'> 10.1101/2023.11.15.23298566</a></p>
        <p class="abstract">ObjectiveTo examine the incidence rate and risk of non-fatal irAEs, including gynecological, hematological, dermatological, ophthalmological, otologic, and dental problems following the COVID-19 vaccination.

MethodsWe conducted a population-based cohort study from the National Health Insurance Service (NHIS) database in Seoul, South Korea. The non-fatal irAEs included gynecological, hematological, dermatological, ophthalmological, ear, and periodontal problems as reported by the Vaccine Adverse Event Reporting Center. The cumulative incidence rate per 10,000 population, Odds ratio, and Hazard ratio (HR) with 95% Confidence Interval (CI) were measured to assess the non-fatal irAEs after COVID-19 vaccination.

ResultsThe cIR of non-fatal irAEs for three months was significantly higher in vaccinated subjects than in non-vaccinated subjects, except for endometriosis. The vaccination significantly increased the risks of all the non-fatal irAEs except for visual impairment. The risk of inner ear disease showed the highest HRs (HR [95% CI] = 2.368 [2.153-2.604]) among the non-fatal irAEs following COVID-19 vaccination. Among the vaccinated subjects, heterologous vaccination was associated with the increased risk of most of the non-fatal irAEs.

ConclusionsThe three-month risks of incidental non-fatal irAEs are substantially higher in the COVID-19 vaccinated subjects than in non-vaccinated controls. Our findings suggested that vaccinated subjects with predisposition are potentially vulnerable to the occurrence of diverse irAEs although the COVID-19 vaccines may not be fatal.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.23299623">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.23299623" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.23299623">
        <p class="paperTitle">Lack of association between HLA and asymptomatic SARS-CoV-2 infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.23299623" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.23299623" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Marchal, A.; Cirulli, E. T.; Neveux, I.; Bellos, E.; Thwaites, R. S.; Schiabor Barrett, K. M.; Zhang, Y.; Nemes-Bokun, I.; Kalinova, M.; Catchpole, A.; Tangye, S. G.; Spaan, A. N.; Lack, J. B.; Ghosn, J.; Burdet, C.; Gorochov, G.; Tubach, F.; Hausfater, P.; COVID Human Genetic Effort,  ; COVIDeF Study Group,  ; French COVID Cohort Study Group,  ; CoV-Contact Cohort,  ; COVID-STORM Clinicians,  ; COVID Clinicians,  ; Orchestra Working Group,  ; Amsterdam UMC Covid-19 Biobank,  ; NIAID-USUHS COVID Study Group,  ; Dalgard, C. L.; Zhang, S.-Y.; Zhang, Q.; Chiu, C.; Fellay, J.; Grzymski, J. J.; San</p>
        <p class="info">Score: 41.8, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.23299623' target='https://doi.org/10.1101/2023.12.06.23299623'> 10.1101/2023.12.06.23299623</a></p>
        <p class="abstract">Human genetic studies of critical COVID-19 pneumonia have revealed the essential role of type I interferon-dependent innate immunity to SARS-CoV-2 infection. Conversely, an association between the HLA-B*15:01 allele and asymptomatic SARS-CoV-2 infection in unvaccinated individuals was recently reported, suggesting a contribution of pre-existing T cell-dependent adaptive immunity. We report a lack of association of classical HLA alleles, including HLA-B*15:01, with pre-omicron asymptomatic SARS-CoV-2 infection in unvaccinated participants in a prospective population-based study in the US (191 asymptomatic vs. 945 symptomatic COVID-19 cases). Moreover, we found no such association in the international COVID Human Genetic Effort cohort (206 asymptomatic vs. 574 mild or moderate COVID-19 cases and 1,625 severe or critical COVID-19 cases). Finally, in the Human Challenge Characterisation study, the three HLA-B*15:01 individuals infected with SARS-CoV-2 developed symptoms. As with other acute primary infections, no classical HLA alleles favoring an asymptomatic course of SARS-CoV-2 infection were identified. These findings suggest that memory T-cell immunity to seasonal coronaviruses does not strongly influence the outcome of SARS-CoV-2 infection in unvaccinated individuals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.23298840">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.23298840" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.23298840">
        <p class="paperTitle">Nanopore sequencing of influenza A and B in Oxfordshire and the United Kingdom, 2022-23</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.23298840" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.23298840" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cane, J.; Sanderson, N.; Barnett, S.; Vaughan, A.; Pott, M.; Kapel, N.; Morgan, M.; Jesuthasan, G.; Samuel, R.; Ehsaan, M.; Boothe, H.; Haduli, E.; Studley, R.; Rourke, E.; Diamond, I.; Fowler, T.; Watson, C.; Stoesser, N.; Walker, A. S.; Street, T.; Eyre, D. W.</p>
        <p class="info">Score: 9.6, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.23298840' target='https://doi.org/10.1101/2023.11.21.23298840'> 10.1101/2023.11.21.23298840</a></p>
        <p class="abstract">ObjectivesWe evaluated Nanopore sequencing for influenza surveillance.

MethodsInfluenza A and B PCR-positive samples from hospital patients in Oxfordshire, UK, and a UK-wide population survey from winter 2022-23 underwent Nanopore sequencing following targeted rt-PCR amplification.

ResultsFrom 941 infections, successful sequencing was achieved in 292/388(75%) available Oxfordshire samples: 231(79%) A/H3N2, 53(18%) A/H1N1, and 8(3%) B/Victoria and in 53/113(47%) UK-wide samples. Sequencing was more successful at lower Ct values. Most same-sample replicate sequences had identical haemagglutinin segments (124/141;88%); a subset of samples also Illumina sequenced were very similar to Nanopore sequences.

Comparison of Oxfordshire and UK-wide sequences showed frequent inter-regional transmission. Infections were closely-related to 2022-23 vaccine strains. Only one sample had a neuraminidase inhibitor resistance mutation.

849/941(90%) Oxfordshire infections were community-acquired. 63/88(72%) potentially healthcare-associated cases shared a hospital ward with [&amp;ge;]1 known infectious case. 33 epidemiologically-plausible transmission links had sequencing data for both source and recipient: 8 were within [&amp;le;]5 SNPs, of these, 5(63%) involved potential sources that were also hospital-acquired.

ConclusionsNanopore influenza sequencing was reproducible and antiviral resistance rare. Inter-regional transmission was common; most infections were genomically similar. Hospital-acquired infections are likely an important source of nosocomial transmission and should be prioritised for infection prevention and control.

HighlightsO_LINanopore sequencing is a reproducible tool for influenza surveillance
C_LIO_LIInter-regional transmission of influenza was common across the UK
C_LIO_LIInfluenza anti-viral resistance was rare
C_LIO_LIIn 1 year most infections were genetically similar, hindering transmission studies
C_LIO_LIHospital-acquired infections are likely a key source of nosocomial transmission
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.23299187">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.23299187" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.23299187">
        <p class="paperTitle">Primary exposure to Zika virus increases risk of symptomatic dengue virus infection with serotypes 2, 3, and 4 but not serotype 1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.23299187" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.23299187" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zambrana, J. V.; Hasund, C. M.; Aogo, R. A.; Bos, S.; Gonzalez, K.; Collado, D.; Miranda, T.; Kuan, G.; Arguello, S.; Balmaseda, A.; Katzelnick, L.; Harris, E.</p>
        <p class="info">Score: 7.8, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.23299187' target='https://doi.org/10.1101/2023.11.29.23299187'> 10.1101/2023.11.29.23299187</a></p>
        <p class="abstract">Infection with any of the four dengue virus serotypes (DENV1-4) can protect against or enhance subsequent dengue depending on pre-existing antibodies and the subsequent infecting serotype. Additionally, primary infection with the related flavivirus Zika virus (ZIKV) has been shown to increase DENV2 disease. Here, we measured how prior DENV and ZIKV immunity influenced risk of disease caused by all four serotypes in a pediatric Nicaraguan cohort. Of 3,412 participants in 2022, 10.6% experienced symptomatic DENV infections caused by DENV1 (n=139), DENV4 (n=133), DENV3 (n=54), DENV2 (n=9), or an undetermined serotype (n=39). Longitudinal clinical and serological data were used to define infection histories, and generalized linear and additive models adjusted for age, sex, time since the last infection, cohort year, and repeat measurements were used to predict disease risk. Compared to flavivirus-naive participants, primary ZIKV infection increased disease risk of DENV4 (relative risk = 2.62, 95% confidence interval: 1.48-4.63) and DENV3 (2.90, 1.34-6.27) but not DENV1 (1.20, 0.72-1.99). Primary DENV infection or a DENV followed by ZIKV infection also increased DENV4 risk. We re-analyzed 19 years of cohort data and demonstrated that prior flavivirus-immunity and pre- existing antibody titer differentially affected disease risk for incoming serotypes, increasing risk of DENV2 and DENV4, protecting against DENV1, and protecting at high titers but enhancing at low titers against DENV3. We thus find that prior ZIKV infection, like prior DENV infection, increases risk of certain DENV serotypes. Cross-reactivity among flaviviruses should be carefully considered when assessing vaccine safety and efficacy.

One-Sentence SummaryDengue disease risk is differentially modulated depending on pre- existing immunity to dengue and Zika virus infections and the secondary infecting serotype.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.23299183">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.23299183" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.23299183">
        <p class="paperTitle">Diagnostic Performance of Rapid Antigen Testing for SARS-CoV-2: The COVid-19 AntiGen (COVAG) Extension study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.23299183" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.23299183" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wertenauer, C.; Dressel, A.; Wieland, E.; Wertenauer, H.-J.; Braitmaier, H.; Straub, A.; Luetzner, N.; Maerz, W.</p>
        <p class="info">Score: 13.6, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.23299183' target='https://doi.org/10.1101/2023.11.29.23299183'> 10.1101/2023.11.29.23299183</a></p>
        <p class="abstract">BackgroundRapid antigen tests (RATs) for SARS-CoV-2 have been used to combat the still ongoing Covid-19 pandemic. This study is the extension of the COVAG study originally performed from February 1 to March 31, 2021. We compared two RATs, the Panbio COVID-19 Ag Rapid Test (Abbott) and the SD Biosensor Q SARS-CoV-2 Rapid Antigen Test (Roche), against RT-PCR on the foil of new variants.

MethodsWe included 888 all-comers at a diagnostic center between October 20, 2021, and March 18, 2022. RT-PCR-positive samples with a Ct value [&amp;le;] 32 were examined for SARS-CoV-2 variants.

FindingsThe sensitivity of the Abbott-RAT and Roche-RAT were 65% and 67%, respectively. For both RATs, lower Ct values were significantly correlated with higher sensitivity. For samples with Ct values [&amp;le;] 25, the sensitivities of the Roche-RAT and of the Abbott-RAT were 96% and 95%, for Ct values 25-30 both were 19%, and for Ct values [&amp;ge;] 30 they were 6% and 2%, respectively. The RATs had substantially higher sensitivities in symptomatic than asymptomatic participants (76, 77%, vs. 29, 31%, for Abbott-RAT, Roche-RAT, respectively) and in participants referred to testing by their primary care physician (84%, 85%) compared to participants who sought testing due to referral by the health department (55%, 58%) or a warning by the Corona-Warn-App (49%, 49%). In persons with self-reported previous Covid-19 sensitivities were markedly lower than in patients without previous Covid-19: 27% vs. 75% for Roche-RAT and 27% vs. 73% for Abbott-RAT. Depending on the vaccination status, the sensitivity of the RATs is 67.6%, 61.5% and 70.6% for non-vaccinated, vaccinated and boostered participants, respectively. For the considered subpopulation of 888 participants, we find no significant correlation between vaccination status and sensitivity.

The Omicron variant was detected with a sensitivity of 94% and 92%, the delta variant with a sensitivity of 80% and 80% for Abbott-RAT and Roche-RAT, respectively. This difference is attributable to the lower Ct values of the Omicron samples compared to the Delta samples. When adjusted for the Ct value, a multivariate logistic regression did not show a significant difference between Omicron and Delta. In terms of sensitivity, we found no significant difference between the wild-type and the Omicron and Delta variants, but a significantly lower sensitivity to the alpha variant compared to the other variants.

For a Ct value [&amp;le;] 25 the sensitivities were 95.2% and 96.0% for the Abbott-RAT and the Roche-RAT, respectively (Table 4). For a Ct value of 25-30 both RATs had a sensitivity of 18.8%. For a Ct value of 30-32, the sensitivities were 0.0% and 7.1% respectively, for Ct values [&amp;ge;]32 the sensitivities were 3.0% and 6.0% for Abbott-RAT and Roche-RAT, respectively.

The specificities were &gt;99% overall.

Interpretation: The sensitivity of the RATs for asymptomatic carriers is unsatisfactory questioning their use for screening. When used in symptomatic patients or when requested by a primary care physician the sensitivities were higher. Our study does not suggest that the vaccination status influences the sensitivity of RATs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.02.23299293">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.02.23299293" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.02.23299293">
        <p class="paperTitle">Strain Differences in Bloodstream and Skin Infection MRSA isolated between 2019-2021 in a Single Health System</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.02.23299293" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.02.23299293" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hofstetter, K. S.; Jacko, N. F.; Shumaker, M. J.; Talbot, B. M.; Petit, R. A.; Read, T. D.; David, M. Z.</p>
        <p class="info">Score: 2.9, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.02.23299293' target='https://doi.org/10.1101/2023.12.02.23299293'> 10.1101/2023.12.02.23299293</a></p>
        <p class="abstract">S. aureus is a common cause of skin and soft tissue infections (SSTIs) and has recently become the most common cause of bloodstream infections (BSIs), but whether the strains causing these two clinical syndromes overlap has not been studied adequately. USA300/500 (clonal complex [CC] 8-ST8) and USA100 (CC5-ST5) have dominated among methicillin-resistant S. aureus (MRSA) strains in the U.S. since the early 2000s. We compared the genomes of unselected MRSA isolates from 131 SSTIs with those from 145 BSIs at a single U.S. center in overlapping periods in 2018-2021. CC8 MRSA was more common among SSTIs and CC5 was more common among BSIs, consistent with prior literature. Based on clustering genomes with a threshold of 15 single nucleotide polymorphisms (SNPs), we identified clusters limited to SSTI patients and separate clusters exclusively comprising BSI patients. However, we additionally identified 8 outbreak clusters that included at least one SSTI and one BSI isolate. This suggests that virulent MRSA strains are transmitted from person-to-person locally in the healthcare setting and the community and that single lineages are often capable of causing both SSTIs and BSIs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.23299611">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.23299611" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.23299611">
        <p class="paperTitle">An amplicon-based Illumina and nanopore sequencing workflow for Chikungunya virus West Africa genotype.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.23299611" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.23299611" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dieng, I.; Ndiaye, M.; Kane, M.; Balde, D.; Mbanne, M.; Diop, S. M. B. S.; Sankhe, S.; Dia, M.; Dieng, M.; Doukanda, S. F. M.; Faye, O.; Sall, A. A.; Dia, N.; Fall, G.; Faye, O.; Diagne, M. M.</p>
        <p class="info">Score: 2.5, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.23299611' target='https://doi.org/10.1101/2023.12.07.23299611'> 10.1101/2023.12.07.23299611</a></p>
        <p class="abstract">The Chikungunya virus, a global arbovirus, is currently causing a major outbreak in the Western African region, with the highest cases reported in Senegal and Burkina Faso. Recent molecular evolution analyses reveal that the strain responsible for the epidemic belongs to the West African genotype, with new mutations potentially impacting viral replication, antigenicity, and host adaptation. Real-time genomic monitoring is needed to track the virus&#39;s spread in new regions. A scalable West African genotype amplicon-based Whole Genome Sequencing for multiple Next Generation Sequencing platforms has been developed to support genomic investigations and identify epidemiological links during the virus&#39;s ongoing spread. This technology will help identify potential threats and support real-time genomic investigations in the ongoing spread of the virus.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.23299338">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.23299338" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.23299338">
        <p class="paperTitle">Effectiveness of Bivalent Boosters Over Nine and Half Months in Nebraska</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.23299338" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.23299338" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Paritala, S.; Xu, Y.; Du, Y.; Donahue, M.; Maloney, P.; Lin, D.-Y.</p>
        <p class="info">Score: 2.1, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.23299338' target='https://doi.org/10.1101/2023.12.03.23299338'> 10.1101/2023.12.03.23299338</a></p>
        <p class="abstract">On September 1, 2022, Moderna and Pfizer-BioNTech bivalent vaccines replaced existing monovalent vaccines as booster doses against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) for persons aged 12 and older in the United States. We assessed the effectiveness of these bivalent boosters against Omicron infection and severe outcomes (COVID-19 hospitalization and death) over a 9.5-month period using line-level data from the state of Nebraska. We found that the relative effectiveness of bivalent boosters, compared with one fewer vaccine dose, against Omicron infection and subsequent death was 39.0% (95% confidence interval [CI], 35.7 to 42.2) and 70.0% (95% CI, 38.8 to 85.3), respectively, at four weeks post administration and gradually waned afterward. Vaccine effectiveness was broadly similar against different Omicron subvariants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.23298766">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.23298766" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.23298766">
        <p class="paperTitle">The Antiviral Mechanism of Action of Molnupiravir in Humans with COVID-19</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.23298766" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.23298766" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sheahan, T. P.; Stevens, L. J.; Narowski, T. M.; Krajewski, T. J.; Lee, C.; Mollan, K. R.; Gribble, J.; Moreria, F. R.; Castillo, I. N.; Cuadra, E.; Alabanza, P.; James Loftis, A.; Coombs, R. W.; Goecker, E. A.; Greninger, A. L.; Chappell, J. D.; Brown, A. J.; Won, J.; Lipansky, F.; Holman, W.; Szewczyk, L. J.; Baric, R. S.; Painter, W. P.; Eron, J. J.; Premkumar, L.; Denison, M. R.; Fischer, W. A.</p>
        <p class="info">Score: 3.5, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.23298766' target='https://doi.org/10.1101/2023.11.21.23298766'> 10.1101/2023.11.21.23298766</a></p>
        <p class="abstract">Meaningful metrics of antiviral activity are essential for determining the efficacy of therapeutics in human clinical trials. Molnupiravir (MOV) is a broadly acting antiviral nucleoside analog prodrug that acts as a competitive alternative substrate for the SARS-CoV-2 RNA-dependent RNA polymerase (RdRp). We developed an assay, Culture-PCR, to better understand the impact of MOV therapy on infectious SARS-CoV-2. Culture-PCR revealed MOV eliminated infectious virus within 48 hours in the nasopharyngeal compartment, the upper airway location with the greatest levels of infectious virus. MOV therapy was associated with increases in mutations across the viral genome but select regions were completely unaffected, thus identifying regions where mutation likely abrogates infectivity. MOV therapy did not alter the magnitude or neutralization capacity of the humoral immune response, a documented correlate of protection. Thus, we provide holistic insights into the function of MOV in adults with COVID-19.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
