<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Immunological imprinting shapes the specificity of human antibody responses against SARS-CoV-2 variants
Authors: Johnston, T. S.; Li, S. H.; Painter, M. M.; Atkinson, R. K.; Douek, N. R.; Reeg, D. B.; Doueck, D. C.; Wherry, E. J.; Hensley, S. E.
Score: 84.2, Published: 2024-01-09 DOI: 10.1101/2024.01.08.24301002
The spike glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) continues to accumulate substitutions, leading to breakthrough infections of vaccinated individuals and prompting the development of updated booster vaccines.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Immunological imprinting shapes the specificity of human antibody responses against SARS-CoV-2 variants
Authors: Johnston, T. S.; Li, S. H.; Painter, M. M.; Atkinson, R. K.; Douek, N. R.; Reeg, D. B.; Doueck, D. C.; Wherry, E. J.; Hensley, S. E.
Score: 84.2, Published: 2024-01-09 DOI: 10.1101/2024.01.08.24301002
The spike glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) continues to accumulate substitutions, leading to breakthrough infections of vaccinated individuals and prompting the development of updated booster vaccines." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-17T10:40:22+00:00" />
<meta property="article:modified_time" content="2024-01-17T10:40:22+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Immunological imprinting shapes the specificity of human antibody responses against SARS-CoV-2 variants
Authors: Johnston, T. S.; Li, S. H.; Painter, M. M.; Atkinson, R. K.; Douek, N. R.; Reeg, D. B.; Doueck, D. C.; Wherry, E. J.; Hensley, S. E.
Score: 84.2, Published: 2024-01-09 DOI: 10.1101/2024.01.08.24301002
The spike glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) continues to accumulate substitutions, leading to breakthrough infections of vaccinated individuals and prompting the development of updated booster vaccines."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Immunological imprinting shapes the specificity of human antibody responses against SARS-CoV-2 variants\nAuthors: Johnston, T. S.; Li, S. H.; Painter, M. M.; Atkinson, R. K.; Douek, N. R.; Reeg, D. B.; Doueck, D. C.; Wherry, E. J.; Hensley, S. E.\nScore: 84.2, Published: 2024-01-09 DOI: 10.1101/2024.01.08.24301002\nThe spike glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) continues to accumulate substitutions, leading to breakthrough infections of vaccinated individuals and prompting the development of updated booster vaccines.",
  "keywords": [
    
  ],
  "articleBody": " Immunological imprinting shapes the specificity of human antibody responses against SARS-CoV-2 variants\nAuthors: Johnston, T. S.; Li, S. H.; Painter, M. M.; Atkinson, R. K.; Douek, N. R.; Reeg, D. B.; Doueck, D. C.; Wherry, E. J.; Hensley, S. E.\nScore: 84.2, Published: 2024-01-09 DOI: 10.1101/2024.01.08.24301002\nThe spike glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) continues to accumulate substitutions, leading to breakthrough infections of vaccinated individuals and prompting the development of updated booster vaccines. Here, we determined the specificity and functionality of antibody and B cell responses following exposure to BA.5 and XBB variants in individuals who received ancestral SARS-CoV-2 mRNA vaccines. BA.5 exposures elicited antibody responses that primarily targeted epitopes conserved between the BA.5 and ancestral spike, with poor reactivity to the XBB.1.5 variant. XBB exposures also elicited antibody responses that targeted epitopes conserved between the XBB.1.5 and ancestral spike. However, unlike BA.5, a single XBB exposure elicited low levels of XBB.1.5-specific antibodies and B cells in some individuals. Pre-existing cross-reactive B cells and antibodies were correlated with stronger overall responses to XBB but weaker XBB-specific responses, suggesting that baseline immunity influences the activation of variant-specific SARS-CoV-2 responses. HighlightsO_LIVariant breakthrough infections boost ancestral cross-reactive antibodies and B cells C_LIO_LIFirst and second BA.5 exposures fail to elicit variant-specific antibodies and B cells C_LIO_LIXBB infections and monovalent vaccinations elicit XBB.1.5-specific responses in some individuals C_LIO_LIXBB.1.5-specific responses correlate with low levels of pre-existing humoral immunity C_LI\nImpact of COVID-19 vaccination on symptoms and immune phenotypes in vaccine-naIÌˆve individuals with Long COVID\nAuthors: Grady, C. B.; Bhattacharjee, B.; Silva, J.; Jaycox, J.; Lee, L. W.; Monteiro, V. S.; Sawano, M.; Massey, D.; Caraballo, C.; Gehlhausen, J.; Tabachnikova, A.; Mao, T.; Lucas, C.; Pena-Hernandez, M. A.; Xu, L.; Tzeng, T. J.; Takahashi, T.; Herrin, J.; Guthe, D. B.; Akrami, A.; Assaf, G.; Davis, H.; Harris, K.; McCorkell, L.; Schulz, W.; Griffin, D.; Wei, H.; Ring, A. M.; Guan, L.; Dela Cruz, C.; Iwasaki, A.; Krumholz, H.\nScore: 48.6, Published: 2024-01-12 DOI: 10.1101/2024.01.11.24300929\nBackgroundLong COVID contributes to the global burden of disease. Proposed root cause hypotheses include the persistence of SARS-CoV-2 viral reservoir, autoimmunity, and reactivation of latent herpesviruses. Patients have reported various changes in Long COVID symptoms after COVID-19 vaccinations, leaving uncertainty about whether vaccine-induced immune responses may alleviate or worsen disease pathology. MethodsIn this prospective study, we evaluated changes in symptoms and immune responses after COVID-19 vaccination in 16 vaccine-naive individuals with Long COVID. Surveys were administered before vaccination and then at 2, 6, and 12 weeks after receiving the first vaccine dose of the primary series. Simultaneously, SARS-CoV-2-reactive TCR enrichment, SARS-CoV-2-specific antibody responses, antibody responses to other viral and self-antigens, and circulating cytokines were quantified before vaccination and at 6 and 12 weeks after vaccination. ResultsSelf-report at 12 weeks post-vaccination indicated 10 out of 16 participants had improved health, 3 had no change, 1 had worse health, and 2 reported marginal changes. Significant elevation in SARS-CoV-2-specific TCRs and Spike protein-specific IgG were observed 6 and 12 weeks after vaccination. No changes in reactivities were observed against herpes viruses and self-antigens. Within this dataset, higher baseline sIL-6R was associated with symptom improvement, and the two top features associated with non-improvement were high IFN-{beta} and CNTF, among soluble analytes. ConclusionsOur study showed that in this small sample, vaccination improved the health or resulted in no change to the health of most participants, though few experienced worsening. Vaccination was associated with increased SARS-CoV-2 Spike protein-specific IgG and T cell expansion in most individuals with Long COVID. Symptom improvement was observed in those with baseline elevated sIL-6R, while elevated interferon and neuropeptide levels were associated with a lack of improvement. Plain language summaryThe impact of the COVID-19 vaccine on vaccine-naive individuals suffering from Long COVID is uncertain. This study assessed the experience and immune signatures of 16 unvaccinated participants with Long COVID. A total of 10 participants had improved health status after vaccination, and one person reported only worsening health. As expected, vaccination increased immune cells and antibodies against the viral spike protein. Immune signatures may prove to be predictors of health status after vaccination. However, given the small number of participants, these initial findings need further validation.\nTransient anti-interferon autoantibodies in the airways are associated with efficient recovery from COVID-19\nAuthors: Babcock, B. R.; Kosters, A.; Eddins, D. J.; Donaire, M. S. B.; Sarvadhavabhatla, S.; Pae, V.; Beltran, F.; Murray, V. W.; Gill, G.; Xie, G.; Dobosh, B. S.; Giacalone, V. D.; Tirouvanziam, R. M.; Ramonell, R. P.; Jenks, S. A.; Sanz, I.; Lee, F. E.-H.; Roan, N. R.; Lee, S. A.; Ghosn, E. E. B.\nScore: 18.4, Published: 2024-01-11 DOI: 10.1101/2024.01.11.24301000\nPre-existing anti-interferon alpha (anti-IFN-) autoantibodies in blood are associated with susceptibility to life-threatening COVID-19. However, it is unclear whether anti-IFN- autoantibodies in the airways - the initial site of infection - can also determine disease outcomes. In this study, we developed a new multiparameter technology, flowBEAT, to quantify and profile the isotypes of anti-IFN- and anti-SARS-CoV-2 antibodies in longitudinal samples collected over 20 months from the airway and matching blood of 129 donors with mild, moderate, and severe COVID-19. We found unexpectedly that nasal anti-IFN- autoantibodies were induced post-infection onset in more than 70% of mild to moderate COVID-19 cases and associated with robust anti-SARS-CoV-2 immunity, fewer symptoms, and efficient recovery. Nasal anti-IFN- autoantibodies followed the peak of host IFN- production and waned with disease recovery, revealing a regulated balance between IFN- and anti-IFN- response. Notably, only a subset of mild to moderate patients progressed to develop systemic anti-IFN-, which correlated with systemic inflammation and worsened symptoms. In contrast, patients with life-threatening COVID-19 sustained elevated anti-IFN- in both airways and blood, coupled with uncontrolled viral load and IFN- production. Our studies thereby reveal a novel protective role for nasal anti-IFN- autoantibodies in the immunopathology of COVID-19 and, more broadly, suggest that anti-IFN- may serve an important regulatory function to restore homeostasis following viral invasion of the respiratory mucosa.\nProportion of Hospitalizations Preventable with Increased Oral SARS-CoV-2 Antiviral Treatment\nAuthors: Levy, M. E.; Chilunda, V.; Heaton, P. R.; Grzymski, J.; Pawloski, P. A.; Goldman, J. D.; Luo, S.\nScore: 12.9, Published: 2023-12-21 DOI: 10.1101/2023.12.19.23300241\nWe estimated the proportion of hospitalizations that could have been averted had all eligible high-risk adults with SARS-CoV-2 infection in a clinical cohort been treated with an oral SARS-CoV-2 antiviral agent early in infection. Among 3,037 patients with risk factors for progressing to severe COVID-19, 946 (31.1%) received an oral antiviral prescription (834 nirmatrelvir and 112 molnupiravir). Only 3.0% of treated patients vs 9.1% of untreated patients were hospitalized (adjusted risk ratio [RR]=0.27; 95% confidence interval [CI]: 0.18-0.41). If all patients had been treated, an estimated 63.3% (95% CI: 50.4-75.1) of hospitalizations within 30 days could have been prevented. This finding is attributed to large gaps in treatment and a strong protective association of antiviral treatment with subsequent hospitalization.\nRetrospective Analysis of Blood Biomarkers of Neurological Injury in Human Cases of Viral Infection and Bacterial Sepsis\nAuthors: Bartlett, M. L.; Goux, H.; Johnson, L.; Schully, K.; Gregory, M.; Brandsma, J.; Chenoweth, J. G.; Clark, D. V.; Rivera, L. F.; Lezcano-Coba, C.; Vittor, A. Y.; Hayes, R.; Galue, J.; Carrera, J.-P.; Smith, D. R.\nScore: 7.8, Published: 2024-01-07 DOI: 10.1101/2024.01.05.24300906\nBackgroundBlood biomarkers of neurological injury could provide a rapid diagnosis of central nervous system (CNS) injury caused by infections. An FDA-approved assay for mild traumatic brain injury (TBI) measures glial fibrillary acidic protein (GFAP) and ubiquitin carboxy-terminal hydrolase L1 (UCH-L1), which signal astrocyte and neuronal injury, respectively. Here, we assessed the applicability of this biomarker assay for determining infection-induced brain injury. MethodsWe measured serum levels of GFAP and UCH-L1 retrospectively in serum samples from three study populations: 1) human cases infected with Venezuelan equine encephalitis virus (VEEV) and Madariaga virus (MADV) (n = 73), 2) human sepsis patients who were severely ill or diagnosed with encephalitis (n = 66), and 3) sepsis cases that were subsequently evaluated for cognitive impairment (n = 64). ResultsIn the virus infection group, we found elevated GFAP for VEEV (p = 0.014) and MADV (p = 0.011) infections, which correlated with seizures (p = 0.006). In the bacterial sepsis group, GFAP was elevated in cases diagnosed with encephalitis (p = 0.0007) and correlated with headaches (p = 0.0002). In the bacterial sepsis cases with a later cognitive assessment, elevated GFAP (p = 0.0057) at study enrollment was associated with cognitive impairment six months later with a positive prognostic capacity of 79% (CI: 66-95%; p = 0.0068). ConclusionsGFAP and UCH-L1 levels measured using an FDA-approved assay for TBI may indicate brain injury resulting from viral or bacterial infections and could predict the development of neurological sequelae.\nA high-resolution genomic and phenotypic analysis of resistance evolution of an Escherichia coli strain from a critical care patient treated with piperacillin/tazobactam.\nAuthors: Fraser, A. J.; Ball, R.; Cantillon, D.; Brettell, L.; Graf, F. E.; Lewis, J. M.; van Aartsen, J. J.; Parry, C. M.; Heinz, E.; Edwards, T.\nScore: 6.2, Published: 2024-01-16 DOI: 10.1101/2024.01.15.24301199\nResistance to the {beta}-lactam/{beta}-lactamase inhibitor (BL/BLI) combination antibiotic piperacillin/tazobactam (TZP) predominantly occurs via {beta}-lactamase enzymes also leading to resistance to third-generation cephalosporins (3GCs). However, if {beta}-lactamases inactive against 3GCs and inhibited by tazobactam are expressed at high levels leading to enzyme hyperproduction, the surplus enzyme escapes inhibition by tazobactam and inactivates the antibiotic piperacillin. Understanding this mechanism is clinically relevant as enzyme hyperproduction can emerge upon antibiotic administration, resulting in treatment failure despite initial resistance profiles supporting TZP use. We report the identification of an Escherichia coli isolate that developed resistance to TZP during patient treatment. Our whole genome sequencing (WGS) analyses show that TZP resistance evolved via IS26-mediated duplication of a blaTEM-1 containing gene cassette on a plasmid, resulting in hyperproduction of TEM-1 {beta}-lactamase. We demonstrate that ten copies of blaTEM-1 induce resistance greater than 32-times the MIC and exposure to TZP further increases amplification of blaTEM-1. Furthermore, in the absence of TZP, gene copy number of IS26 and blaTEM-1 remains stable over five days, despite a 48,205 bp genome size increase compared to the pre-amplification isolate. We additionally detect phenotypic changes that might indicate host adaptation potentially linked to the additional genes in the amplified cassette. Our analysis advances the understanding of infections caused by isolates evolving {beta}-lactamase hyperproduction, which represent a complex problem in both detection and treatment. As 40% of antibiotics active against WHO priority pathogens in the pre-clinical pipeline are BL/BLI combinations further investigations are of urgent concern.\nEstimating SARS-CoV-2 transmission parameters between coinciding outbreaks in a university population and the surrounding community\nAuthors: Mietchen, M. S.; Clancey, E.; McMichael, C.; Lofgren, E.\nScore: 6.1, Published: 2024-01-11 DOI: 10.1101/2024.01.10.24301116\nPrior studies suggest that population heterogeneity in SARS-CoV-2 (COVID-19) transmission plays an important role in epidemic dynamics. During the fall of 2020, many US universities and the surrounding communities experienced an increase in reported incidence of SARS-CoV-2 infections, with a high disease burden among students. We explore the transmission dynamics of an outbreak of SARS-CoV-2 among university students, how it impacted the non-student population via cross-transmission, and how it could influence pandemic planning and response. Using surveillance data of reported SARS-CoV-2 cases, we developed a two-population SEIR model to estimate transmission parameters and evaluate how these subpopulations interacted during the 2020 Fall semester. We estimated the transmission rate among the university students ({beta}U) and community residents ({beta}C), as well as the rate of cross-transmission between the two subpopulations ({beta}M) using particle Markov Chain Monte Carlo (pMCMC) simulation-based methods. We found that both populations were more likely to interact with others in their population and that cross-transmission was minimal. The cross-transmission estimate ({beta}M) was considerably smaller [0.04 x 10-5 (95% CI: 0.00 x 10-5, 0.15 x 10-5)] compared to the community estimate ({beta}C) at 2.09 x 10-5 (95% CI: 1.12 x 10-5, 2.90 x 10-5) and university estimate ({beta}U) at 27.92 x 10-5 (95% CI: 19.97 x 10-5, 39.15 x 10-5). The higher within population transmission rates among the university and the community (698 and 52 times higher, respectively) when compared to the cross-transmission rate, suggests that these two populations did not transmit between each other heavily, despite their geographic overlap. During the first wave of the pandemic, two distinct epidemics occurred among two subpopulations within a relatively small US county population where university students accounted for roughly 41% of the total population. Transmission parameter estimates varied substantially with minimal or no cross-transmission between the subpopulations. Assumptions that county-level and other small populations are well-mixed during a respiratory viral pandemic should be reconsidered. More granular models reflecting overlapping subpopulations may assist with better-targeted interventions for local public health and healthcare facilities.\nEstimation of SARS-CoV-2 fitness gains from genomic surveillance data without prior lineage classification\nAuthors: Donker, T.; Papathanassopoulos, A.; Ghosh, H.; Kociurzynski, R.; Felder, M.; Grundmann, H.; Reuter, S.\nScore: 3.5, Published: 2024-01-09 DOI: 10.1101/2024.01.08.24300976\nThe emergence of SARS-CoV-2 variants with increased fitness has had a strong impact on the epidemiology of COVID-19, with the higher effective reproduction number of the viral variants leading to new epidemic waves. Tracking such variants and their genetic signatures, using data collected through genomic surveillance, is therefore crucial for forecasting likely surges in incidence. Current methods of estimating fitness advantages of variants rely on tracking the changing proportion of a particular lineage over time, but describing successful lineages in a rapidly evolving viral population is a difficult task. We propose a new method of estimating fitness gains directly from nucleotide information generated by genomic surveillance, without a-priori assigning isolates to lineages from phylogenies, based solely on the abundance of Single Nucleotide Polymorphisms (SNPs). The method is based on mapping changes in the genetic population structure over time. Changes in the abundance of SNPs associated with periods of increasing fitness allow for the unbiased discovery of new variants, and thereby obviating a deliberate lineage assignment and phylogenetic inference. We conclude that the method provides a fast and reliable way to estimate fitness advantages of variants without the need for a-priori assigning isolates to lineages.\nVariant-specific humoral immune response to SARS-CoV-2 escape mutants arising in clinically severe, prolonged infection\nAuthors: Guenther, T.; Schoefbaenker, M.; Lorentzen, E. U.; Romberg, M.-L.; Hennies, M. T.; Neddermeyer, R.; Mueller, M. M.; Mellmann, A.; Lenz, G.; Stelljes, M.; Hrincius, E. R.; Vollenberg, R.; Ludwig, S.; Tepasse, P.-R.; Kuehn, J. E.\nScore: 24.1, Published: 2024-01-07 DOI: 10.1101/2024.01.06.24300890\nNeutralising antibodies against the SARS-CoV-2 spike (S) protein are major determinants of protective immunity, though insufficient antibody responses may cause the emergence of escape mutants. We studied the intra-host evolution and the humoral immune response in a B-cell depleted, haemato-oncologic patient experiencing clinically severe, prolonged SARS-CoV-2 infection with a virus of lineage B.1.177.81. Bamlanivimab treatment early in infection was associated with the emergence of the bamlanivimab escape mutation S:S494P. Ten days before virus elimination, additional mutations within the N-terminal domain (NTD) and the receptor binding domain (RBD) of S were observed, of which the triple mutant S:Delta141-4 E484K S494P became dominant. Routine serology revealed no evidence of an antibody response in the patient. A detailed analysis of the variant-specific immune response by pseudotyped virus neutralisation test (pVNT), surrogate VNT (sVNT), and immunoglobulin-capture EIA showed that the onset of an IgM-dominated antibody response coincided with the appearance of escape mutations. The formation of neutralising antibodies against S:Delta141-4 E484K S494P correlated with virus elimination. One year later, the patient experienced clinically mild re-infection with Omicron BA.1.18, which was treated with sotrovimab and resulted in a massive increase of Omicron-reactive antibodies. In conclusion, the onset of an IgM-dominated endogenous immune response in an immunocompromised patient coincided with the appearance of additional mutations in the NTD and RBD of S in a bamlanivimab-resistant virus. Although virus elimination was ultimately achieved, this humoral immune response escaped detection by routine diagnosis and created a situation temporarily favouring the emergence of escape variants with known epidemiological relevance.\nCritically-ill COVID-19 susceptibility gene CCR3 shows natural selection in sub-Saharan Africans\nAuthors: Sun, Z.; Pan, L.; Tian, A.; Chen, P.\nScore: 2.0, Published: 2024-01-13 DOI: 10.1101/2024.01.12.24301202\nThe prevalence of COVID-19 critical illness varies across ethnicities, with recent studies suggesting that genetic factors may contribute to this variation. The aim of this study was to investigate natural selection signals of genes associated with critically-ill COVID-19 in sub-Saharan Africans. Severe COVID-19 SNPs were obtained from the HGI website. Selection signals were assessed in 661 sub-Sahara Africans from 1000 Genomes Project using integrated haplotype score (iHS), cross-population extended haplotype homozygosity (xpEHH), and fixation index (Fst). Allele frequency trajectory analysis of ancient DNA samples were used to validate the existing of selection in sub-Sahara Africans. We also used Mendelian randomization to decipher the correlation between natural selection and critically-ill COVID-19. We identified that CCR3 exhibited significant natural selection signals in sub-Sahara Africans. Within the CCR3 gene, rs17217831-A showed both high iHS (Standardized iHS = 2) and high XP-EHH (Standardized XP-EHH = 2.5) in sub-Sahara Africans. Allele frequency trajectory of CCR3 rs17217831-A revealed natural selection occurring in the recent 1,500 years. Natural selection resulted in increased CCR3 expression in sub-Sahara Africans. Mendelian Randomization provided evidence that increased blood CCR3 expression and eosinophil counts lowered the risk of critically ill COVID-19. Our findings suggest that sub-Saharan Africans are less vulnerable to critically ill COVID-19 due to natural selection and identify CCR3 as a potential novel therapeutic target.\n",
  "wordCount" : "2874",
  "inLanguage": "en",
  "datePublished": "2024-01-17T10:40:22Z",
  "dateModified": "2024-01-17T10:40:22Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 17, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.24301002">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.24301002" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.24301002">
        <p class="paperTitle">Immunological imprinting shapes the specificity of human antibody responses against SARS-CoV-2 variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.24301002" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.24301002" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnston, T. S.; Li, S. H.; Painter, M. M.; Atkinson, R. K.; Douek, N. R.; Reeg, D. B.; Doueck, D. C.; Wherry, E. J.; Hensley, S. E.</p>
        <p class="info">Score: 84.2, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.24301002' target='https://doi.org/10.1101/2024.01.08.24301002'> 10.1101/2024.01.08.24301002</a></p>
        <p class="abstract">The spike glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) continues to accumulate substitutions, leading to breakthrough infections of vaccinated individuals and prompting the development of updated booster vaccines. Here, we determined the specificity and functionality of antibody and B cell responses following exposure to BA.5 and XBB variants in individuals who received ancestral SARS-CoV-2 mRNA vaccines. BA.5 exposures elicited antibody responses that primarily targeted epitopes conserved between the BA.5 and ancestral spike, with poor reactivity to the XBB.1.5 variant. XBB exposures also elicited antibody responses that targeted epitopes conserved between the XBB.1.5 and ancestral spike. However, unlike BA.5, a single XBB exposure elicited low levels of XBB.1.5-specific antibodies and B cells in some individuals. Pre-existing cross-reactive B cells and antibodies were correlated with stronger overall responses to XBB but weaker XBB-specific responses, suggesting that baseline immunity influences the activation of variant-specific SARS-CoV-2 responses.

HighlightsO_LIVariant breakthrough infections boost ancestral cross-reactive antibodies and B cells
C_LIO_LIFirst and second BA.5 exposures fail to elicit variant-specific antibodies and B cells
C_LIO_LIXBB infections and monovalent vaccinations elicit XBB.1.5-specific responses in some individuals
C_LIO_LIXBB.1.5-specific responses correlate with low levels of pre-existing humoral immunity
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.24300929">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.24300929" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.24300929">
        <p class="paperTitle">Impact of COVID-19 vaccination on symptoms and immune phenotypes in vaccine-naIÌˆve individuals with Long COVID</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.24300929" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.24300929" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grady, C. B.; Bhattacharjee, B.; Silva, J.; Jaycox, J.; Lee, L. W.; Monteiro, V. S.; Sawano, M.; Massey, D.; Caraballo, C.; Gehlhausen, J.; Tabachnikova, A.; Mao, T.; Lucas, C.; Pena-Hernandez, M. A.; Xu, L.; Tzeng, T. J.; Takahashi, T.; Herrin, J.; Guthe, D. B.; Akrami, A.; Assaf, G.; Davis, H.; Harris, K.; McCorkell, L.; Schulz, W.; Griffin, D.; Wei, H.; Ring, A. M.; Guan, L.; Dela Cruz, C.; Iwasaki, A.; Krumholz, H.</p>
        <p class="info">Score: 48.6, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.24300929' target='https://doi.org/10.1101/2024.01.11.24300929'> 10.1101/2024.01.11.24300929</a></p>
        <p class="abstract">BackgroundLong COVID contributes to the global burden of disease. Proposed root cause hypotheses include the persistence of SARS-CoV-2 viral reservoir, autoimmunity, and reactivation of latent herpesviruses. Patients have reported various changes in Long COVID symptoms after COVID-19 vaccinations, leaving uncertainty about whether vaccine-induced immune responses may alleviate or worsen disease pathology.

MethodsIn this prospective study, we evaluated changes in symptoms and immune responses after COVID-19 vaccination in 16 vaccine-naive individuals with Long COVID. Surveys were administered before vaccination and then at 2, 6, and 12 weeks after receiving the first vaccine dose of the primary series. Simultaneously, SARS-CoV-2-reactive TCR enrichment, SARS-CoV-2-specific antibody responses, antibody responses to other viral and self-antigens, and circulating cytokines were quantified before vaccination and at 6 and 12 weeks after vaccination.

ResultsSelf-report at 12 weeks post-vaccination indicated 10 out of 16 participants had improved health, 3 had no change, 1 had worse health, and 2 reported marginal changes. Significant elevation in SARS-CoV-2-specific TCRs and Spike protein-specific IgG were observed 6 and 12 weeks after vaccination. No changes in reactivities were observed against herpes viruses and self-antigens. Within this dataset, higher baseline sIL-6R was associated with symptom improvement, and the two top features associated with non-improvement were high IFN-{beta} and CNTF, among soluble analytes.

ConclusionsOur study showed that in this small sample, vaccination improved the health or resulted in no change to the health of most participants, though few experienced worsening. Vaccination was associated with increased SARS-CoV-2 Spike protein-specific IgG and T cell expansion in most individuals with Long COVID. Symptom improvement was observed in those with baseline elevated sIL-6R, while elevated interferon and neuropeptide levels were associated with a lack of improvement.

Plain language summaryThe impact of the COVID-19 vaccine on vaccine-naive individuals suffering from Long COVID is uncertain. This study assessed the experience and immune signatures of 16 unvaccinated participants with Long COVID. A total of 10 participants had improved health status after vaccination, and one person reported only worsening health. As expected, vaccination increased immune cells and antibodies against the viral spike protein. Immune signatures may prove to be predictors of health status after vaccination. However, given the small number of participants, these initial findings need further validation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.24301000">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.24301000" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.24301000">
        <p class="paperTitle">Transient anti-interferon autoantibodies in the airways are associated with efficient recovery from COVID-19</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.24301000" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.24301000" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Babcock, B. R.; Kosters, A.; Eddins, D. J.; Donaire, M. S. B.; Sarvadhavabhatla, S.; Pae, V.; Beltran, F.; Murray, V. W.; Gill, G.; Xie, G.; Dobosh, B. S.; Giacalone, V. D.; Tirouvanziam, R. M.; Ramonell, R. P.; Jenks, S. A.; Sanz, I.; Lee, F. E.-H.; Roan, N. R.; Lee, S. A.; Ghosn, E. E. B.</p>
        <p class="info">Score: 18.4, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.24301000' target='https://doi.org/10.1101/2024.01.11.24301000'> 10.1101/2024.01.11.24301000</a></p>
        <p class="abstract">Pre-existing anti-interferon alpha (anti-IFN-) autoantibodies in blood are associated with susceptibility to life-threatening COVID-19. However, it is unclear whether anti-IFN- autoantibodies in the airways - the initial site of infection - can also determine disease outcomes. In this study, we developed a new multiparameter technology, flowBEAT, to quantify and profile the isotypes of anti-IFN- and anti-SARS-CoV-2 antibodies in longitudinal samples collected over 20 months from the airway and matching blood of 129 donors with mild, moderate, and severe COVID-19. We found unexpectedly that nasal anti-IFN- autoantibodies were induced post-infection onset in more than 70% of mild to moderate COVID-19 cases and associated with robust anti-SARS-CoV-2 immunity, fewer symptoms, and efficient recovery. Nasal anti-IFN- autoantibodies followed the peak of host IFN- production and waned with disease recovery, revealing a regulated balance between IFN- and anti-IFN- response. Notably, only a subset of mild to moderate patients progressed to develop systemic anti-IFN-, which correlated with systemic inflammation and worsened symptoms. In contrast, patients with life-threatening COVID-19 sustained elevated anti-IFN- in both airways and blood, coupled with uncontrolled viral load and IFN- production. Our studies thereby reveal a novel protective role for nasal anti-IFN- autoantibodies in the immunopathology of COVID-19 and, more broadly, suggest that anti-IFN- may serve an important regulatory function to restore homeostasis following viral invasion of the respiratory mucosa.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.23300241">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.23300241" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.23300241">
        <p class="paperTitle">Proportion of Hospitalizations Preventable with Increased Oral SARS-CoV-2 Antiviral Treatment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.23300241" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.23300241" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Levy, M. E.; Chilunda, V.; Heaton, P. R.; Grzymski, J.; Pawloski, P. A.; Goldman, J. D.; Luo, S.</p>
        <p class="info">Score: 12.9, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.23300241' target='https://doi.org/10.1101/2023.12.19.23300241'> 10.1101/2023.12.19.23300241</a></p>
        <p class="abstract">We estimated the proportion of hospitalizations that could have been averted had all eligible high-risk adults with SARS-CoV-2 infection in a clinical cohort been treated with an oral SARS-CoV-2 antiviral agent early in infection. Among 3,037 patients with risk factors for progressing to severe COVID-19, 946 (31.1%) received an oral antiviral prescription (834 nirmatrelvir and 112 molnupiravir). Only 3.0% of treated patients vs 9.1% of untreated patients were hospitalized (adjusted risk ratio [RR]=0.27; 95% confidence interval [CI]: 0.18-0.41). If all patients had been treated, an estimated 63.3% (95% CI: 50.4-75.1) of hospitalizations within 30 days could have been prevented. This finding is attributed to large gaps in treatment and a strong protective association of antiviral treatment with subsequent hospitalization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.24300906">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.24300906" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.24300906">
        <p class="paperTitle">Retrospective Analysis of Blood Biomarkers of Neurological Injury in Human Cases of Viral Infection and Bacterial Sepsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.24300906" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.24300906" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bartlett, M. L.; Goux, H.; Johnson, L.; Schully, K.; Gregory, M.; Brandsma, J.; Chenoweth, J. G.; Clark, D. V.; Rivera, L. F.; Lezcano-Coba, C.; Vittor, A. Y.; Hayes, R.; Galue, J.; Carrera, J.-P.; Smith, D. R.</p>
        <p class="info">Score: 7.8, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.24300906' target='https://doi.org/10.1101/2024.01.05.24300906'> 10.1101/2024.01.05.24300906</a></p>
        <p class="abstract">BackgroundBlood biomarkers of neurological injury could provide a rapid diagnosis of central nervous system (CNS) injury caused by infections. An FDA-approved assay for mild traumatic brain injury (TBI) measures glial fibrillary acidic protein (GFAP) and ubiquitin carboxy-terminal hydrolase L1 (UCH-L1), which signal astrocyte and neuronal injury, respectively. Here, we assessed the applicability of this biomarker assay for determining infection-induced brain injury.

MethodsWe measured serum levels of GFAP and UCH-L1 retrospectively in serum samples from three study populations: 1) human cases infected with Venezuelan equine encephalitis virus (VEEV) and Madariaga virus (MADV) (n = 73), 2) human sepsis patients who were severely ill or diagnosed with encephalitis (n = 66), and 3) sepsis cases that were subsequently evaluated for cognitive impairment (n = 64).

ResultsIn the virus infection group, we found elevated GFAP for VEEV (p = 0.014) and MADV (p = 0.011) infections, which correlated with seizures (p = 0.006). In the bacterial sepsis group, GFAP was elevated in cases diagnosed with encephalitis (p = 0.0007) and correlated with headaches (p = 0.0002). In the bacterial sepsis cases with a later cognitive assessment, elevated GFAP (p = 0.0057) at study enrollment was associated with cognitive impairment six months later with a positive prognostic capacity of 79% (CI: 66-95%; p = 0.0068).

ConclusionsGFAP and UCH-L1 levels measured using an FDA-approved assay for TBI may indicate brain injury resulting from viral or bacterial infections and could predict the development of neurological sequelae.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.24301199">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.24301199" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.24301199">
        <p class="paperTitle">A high-resolution genomic and phenotypic analysis of resistance evolution of an Escherichia coli strain from a critical care patient treated with piperacillin/tazobactam.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.24301199" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.24301199" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fraser, A. J.; Ball, R.; Cantillon, D.; Brettell, L.; Graf, F. E.; Lewis, J. M.; van Aartsen, J. J.; Parry, C. M.; Heinz, E.; Edwards, T.</p>
        <p class="info">Score: 6.2, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.24301199' target='https://doi.org/10.1101/2024.01.15.24301199'> 10.1101/2024.01.15.24301199</a></p>
        <p class="abstract">Resistance to the {beta}-lactam/{beta}-lactamase inhibitor (BL/BLI) combination antibiotic piperacillin/tazobactam (TZP) predominantly occurs via {beta}-lactamase enzymes also leading to resistance to third-generation cephalosporins (3GCs). However, if {beta}-lactamases inactive against 3GCs and inhibited by tazobactam are expressed at high levels leading to enzyme hyperproduction, the surplus enzyme escapes inhibition by tazobactam and inactivates the antibiotic piperacillin. Understanding this mechanism is clinically relevant as enzyme hyperproduction can emerge upon antibiotic administration, resulting in treatment failure despite initial resistance profiles supporting TZP use. We report the identification of an Escherichia coli isolate that developed resistance to TZP during patient treatment. Our whole genome sequencing (WGS) analyses show that TZP resistance evolved via IS26-mediated duplication of a blaTEM-1 containing gene cassette on a plasmid, resulting in hyperproduction of TEM-1 {beta}-lactamase. We demonstrate that ten copies of blaTEM-1 induce resistance greater than 32-times the MIC and exposure to TZP further increases amplification of blaTEM-1. Furthermore, in the absence of TZP, gene copy number of IS26 and blaTEM-1 remains stable over five days, despite a 48,205 bp genome size increase compared to the pre-amplification isolate. We additionally detect phenotypic changes that might indicate host adaptation potentially linked to the additional genes in the amplified cassette. Our analysis advances the understanding of infections caused by isolates evolving {beta}-lactamase hyperproduction, which represent a complex problem in both detection and treatment. As 40% of antibiotics active against WHO priority pathogens in the pre-clinical pipeline are BL/BLI combinations further investigations are of urgent concern.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.24301116">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.24301116" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.24301116">
        <p class="paperTitle">Estimating SARS-CoV-2 transmission parameters between coinciding outbreaks in a university population and the surrounding community</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.24301116" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.24301116" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mietchen, M. S.; Clancey, E.; McMichael, C.; Lofgren, E.</p>
        <p class="info">Score: 6.1, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.24301116' target='https://doi.org/10.1101/2024.01.10.24301116'> 10.1101/2024.01.10.24301116</a></p>
        <p class="abstract">Prior studies suggest that population heterogeneity in SARS-CoV-2 (COVID-19) transmission plays an important role in epidemic dynamics. During the fall of 2020, many US universities and the surrounding communities experienced an increase in reported incidence of SARS-CoV-2 infections, with a high disease burden among students. We explore the transmission dynamics of an outbreak of SARS-CoV-2 among university students, how it impacted the non-student population via cross-transmission, and how it could influence pandemic planning and response.

Using surveillance data of reported SARS-CoV-2 cases, we developed a two-population SEIR model to estimate transmission parameters and evaluate how these subpopulations interacted during the 2020 Fall semester. We estimated the transmission rate among the university students ({beta}U) and community residents ({beta}C), as well as the rate of cross-transmission between the two subpopulations ({beta}M) using particle Markov Chain Monte Carlo (pMCMC) simulation-based methods.

We found that both populations were more likely to interact with others in their population and that cross-transmission was minimal. The cross-transmission estimate ({beta}M) was considerably smaller [0.04 x 10-5 (95% CI: 0.00 x 10-5, 0.15 x 10-5)] compared to the community estimate ({beta}C) at 2.09 x 10-5 (95% CI: 1.12 x 10-5, 2.90 x 10-5) and university estimate ({beta}U) at 27.92 x 10-5 (95% CI: 19.97 x 10-5, 39.15 x 10-5). The higher within population transmission rates among the university and the community (698 and 52 times higher, respectively) when compared to the cross-transmission rate, suggests that these two populations did not transmit between each other heavily, despite their geographic overlap.

During the first wave of the pandemic, two distinct epidemics occurred among two subpopulations within a relatively small US county population where university students accounted for roughly 41% of the total population. Transmission parameter estimates varied substantially with minimal or no cross-transmission between the subpopulations. Assumptions that county-level and other small populations are well-mixed during a respiratory viral pandemic should be reconsidered. More granular models reflecting overlapping subpopulations may assist with better-targeted interventions for local public health and healthcare facilities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.24300976">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.24300976" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.24300976">
        <p class="paperTitle">Estimation of SARS-CoV-2 fitness gains from genomic surveillance data without prior lineage classification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.24300976" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.24300976" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Donker, T.; Papathanassopoulos, A.; Ghosh, H.; Kociurzynski, R.; Felder, M.; Grundmann, H.; Reuter, S.</p>
        <p class="info">Score: 3.5, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.24300976' target='https://doi.org/10.1101/2024.01.08.24300976'> 10.1101/2024.01.08.24300976</a></p>
        <p class="abstract">The emergence of SARS-CoV-2 variants with increased fitness has had a strong impact on the epidemiology of COVID-19, with the higher effective reproduction number of the viral variants leading to new epidemic waves. Tracking such variants and their genetic signatures, using data collected through genomic surveillance, is therefore crucial for forecasting likely surges in incidence. Current methods of estimating fitness advantages of variants rely on tracking the changing proportion of a particular lineage over time, but describing successful lineages in a rapidly evolving viral population is a difficult task. We propose a new method of estimating fitness gains directly from nucleotide information generated by genomic surveillance, without a-priori assigning isolates to lineages from phylogenies, based solely on the abundance of Single Nucleotide Polymorphisms (SNPs). The method is based on mapping changes in the genetic population structure over time. Changes in the abundance of SNPs associated with periods of increasing fitness allow for the unbiased discovery of new variants, and thereby obviating a deliberate lineage assignment and phylogenetic inference. We conclude that the method provides a fast and reliable way to estimate fitness advantages of variants without the need for a-priori assigning isolates to lineages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.06.24300890">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.06.24300890" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.06.24300890">
        <p class="paperTitle">Variant-specific humoral immune response to SARS-CoV-2 escape mutants arising in clinically severe, prolonged infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.06.24300890" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.06.24300890" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guenther, T.; Schoefbaenker, M.; Lorentzen, E. U.; Romberg, M.-L.; Hennies, M. T.; Neddermeyer, R.; Mueller, M. M.; Mellmann, A.; Lenz, G.; Stelljes, M.; Hrincius, E. R.; Vollenberg, R.; Ludwig, S.; Tepasse, P.-R.; Kuehn, J. E.</p>
        <p class="info">Score: 24.1, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.06.24300890' target='https://doi.org/10.1101/2024.01.06.24300890'> 10.1101/2024.01.06.24300890</a></p>
        <p class="abstract">Neutralising antibodies against the SARS-CoV-2 spike (S) protein are major determinants of protective immunity, though insufficient antibody responses may cause the emergence of escape mutants. We studied the intra-host evolution and the humoral immune response in a B-cell depleted, haemato-oncologic patient experiencing clinically severe, prolonged SARS-CoV-2 infection with a virus of lineage B.1.177.81.

Bamlanivimab treatment early in infection was associated with the emergence of the bamlanivimab escape mutation S:S494P. Ten days before virus elimination, additional mutations within the N-terminal domain (NTD) and the receptor binding domain (RBD) of S were observed, of which the triple mutant S:Delta141-4 E484K S494P became dominant. Routine serology revealed no evidence of an antibody response in the patient. A detailed analysis of the variant-specific immune response by pseudotyped virus neutralisation test (pVNT), surrogate VNT (sVNT), and immunoglobulin-capture EIA showed that the onset of an IgM-dominated antibody response coincided with the appearance of escape mutations. The formation of neutralising antibodies against S:Delta141-4 E484K S494P correlated with virus elimination. One year later, the patient experienced clinically mild re-infection with Omicron BA.1.18, which was treated with sotrovimab and resulted in a massive increase of Omicron-reactive antibodies.

In conclusion, the onset of an IgM-dominated endogenous immune response in an immunocompromised patient coincided with the appearance of additional mutations in the NTD and RBD of S in a bamlanivimab-resistant virus. Although virus elimination was ultimately achieved, this humoral immune response escaped detection by routine diagnosis and created a situation temporarily favouring the emergence of escape variants with known epidemiological relevance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.24301202">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.24301202" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.24301202">
        <p class="paperTitle">Critically-ill COVID-19 susceptibility gene CCR3 shows natural selection in sub-Saharan Africans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.24301202" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.24301202" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sun, Z.; Pan, L.; Tian, A.; Chen, P.</p>
        <p class="info">Score: 2.0, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.24301202' target='https://doi.org/10.1101/2024.01.12.24301202'> 10.1101/2024.01.12.24301202</a></p>
        <p class="abstract">The prevalence of COVID-19 critical illness varies across ethnicities, with recent studies suggesting that genetic factors may contribute to this variation. The aim of this study was to investigate natural selection signals of genes associated with critically-ill COVID-19 in sub-Saharan Africans. Severe COVID-19 SNPs were obtained from the HGI website. Selection signals were assessed in 661 sub-Sahara Africans from 1000 Genomes Project using integrated haplotype score (iHS), cross-population extended haplotype homozygosity (xpEHH), and fixation index (Fst). Allele frequency trajectory analysis of ancient DNA samples were used to validate the existing of selection in sub-Sahara Africans. We also used Mendelian randomization to decipher the correlation between natural selection and critically-ill COVID-19. We identified that CCR3 exhibited significant natural selection signals in sub-Sahara Africans. Within the CCR3 gene, rs17217831-A showed both high iHS (Standardized iHS = 2) and high XP-EHH (Standardized XP-EHH = 2.5) in sub-Sahara Africans. Allele frequency trajectory of CCR3 rs17217831-A revealed natural selection occurring in the recent 1,500 years. Natural selection resulted in increased CCR3 expression in sub-Sahara Africans. Mendelian Randomization provided evidence that increased blood CCR3 expression and eosinophil counts lowered the risk of critically ill COVID-19. Our findings suggest that sub-Saharan Africans are less vulnerable to critically ill COVID-19 due to natural selection and identify CCR3 as a potential novel therapeutic target.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
