<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Altered IgG4 Antibody Response to Repeated mRNA versus Protein COVID Vaccines
Authors: Kalkeri, R.; Zhu, M.; Cloney-Clark, S.; Plested, J. S.; Parekh, A.; Gorinson, D.; Cai, R.; Mahato, S.; Ramanathan, P.; Aurelia, L. C.; John Selva, K.; Marchese, A. M.; Fries, L.; Chung, A. W.; Dunkle, L. M.
Score: 283.3, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301374
Repeated mRNA SARS-CoV-2 vaccination has been associated with increases in the proportion of IgG4 in spike-specific antibody responses and concurrent reductions in Fc{gamma}-mediated effector functions that may limit control of viral infection.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Altered IgG4 Antibody Response to Repeated mRNA versus Protein COVID Vaccines
Authors: Kalkeri, R.; Zhu, M.; Cloney-Clark, S.; Plested, J. S.; Parekh, A.; Gorinson, D.; Cai, R.; Mahato, S.; Ramanathan, P.; Aurelia, L. C.; John Selva, K.; Marchese, A. M.; Fries, L.; Chung, A. W.; Dunkle, L. M.
Score: 283.3, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301374
Repeated mRNA SARS-CoV-2 vaccination has been associated with increases in the proportion of IgG4 in spike-specific antibody responses and concurrent reductions in Fc{gamma}-mediated effector functions that may limit control of viral infection." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-28T10:37:59+00:00" />
<meta property="article:modified_time" content="2024-01-28T10:37:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Altered IgG4 Antibody Response to Repeated mRNA versus Protein COVID Vaccines
Authors: Kalkeri, R.; Zhu, M.; Cloney-Clark, S.; Plested, J. S.; Parekh, A.; Gorinson, D.; Cai, R.; Mahato, S.; Ramanathan, P.; Aurelia, L. C.; John Selva, K.; Marchese, A. M.; Fries, L.; Chung, A. W.; Dunkle, L. M.
Score: 283.3, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301374
Repeated mRNA SARS-CoV-2 vaccination has been associated with increases in the proportion of IgG4 in spike-specific antibody responses and concurrent reductions in Fc{gamma}-mediated effector functions that may limit control of viral infection."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Altered IgG4 Antibody Response to Repeated mRNA versus Protein COVID Vaccines\nAuthors: Kalkeri, R.; Zhu, M.; Cloney-Clark, S.; Plested, J. S.; Parekh, A.; Gorinson, D.; Cai, R.; Mahato, S.; Ramanathan, P.; Aurelia, L. C.; John Selva, K.; Marchese, A. M.; Fries, L.; Chung, A. W.; Dunkle, L. M.\nScore: 283.3, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301374\nRepeated mRNA SARS-CoV-2 vaccination has been associated with increases in the proportion of IgG4 in spike-specific antibody responses and concurrent reductions in Fc{gamma}-mediated effector functions that may limit control of viral infection.",
  "keywords": [
    
  ],
  "articleBody": " Altered IgG4 Antibody Response to Repeated mRNA versus Protein COVID Vaccines\nAuthors: Kalkeri, R.; Zhu, M.; Cloney-Clark, S.; Plested, J. S.; Parekh, A.; Gorinson, D.; Cai, R.; Mahato, S.; Ramanathan, P.; Aurelia, L. C.; John Selva, K.; Marchese, A. M.; Fries, L.; Chung, A. W.; Dunkle, L. M.\nScore: 283.3, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301374\nRepeated mRNA SARS-CoV-2 vaccination has been associated with increases in the proportion of IgG4 in spike-specific antibody responses and concurrent reductions in Fc{gamma}-mediated effector functions that may limit control of viral infection. Here, we assessed anti-Spike total IgG, IgG1, IgG2, IgG3 and IgG4, and surrogate markers for antibody-dependent cellular phagocytosis (ADCP, Fc{gamma}RIIa binding), antibody-dependent cellular cytotoxicity (ADCC, Fc{gamma}RIIIa binding), and antibody-dependent complement deposition (ADCD, C1q binding) associated with repeated SARS-CoV-2 vaccination with NVX-CoV2373 (Novavax Inc., Gaithersburg, MD). The NVX-CoV2373 protein vaccine did not induce notable increases in spike-specific IgG4 or negatively impact surrogates for Fc{gamma} effector responses. Conversely, repeated NVX-CoV2373 vaccination uniquely enhanced IgG3 responses which are known to exhibit strong affinity for Fc{gamma}RIIIa and have previously been linked to potent neutralization of SARS-CoV-2. Subsequent investigations will help to understand the immunological diversity generated by different SARS-CoV-2 vaccine types and have the potential to reshape public health strategies.\nBlood transcriptomics reveal persistent SARS-CoV-2 RNA and candidate biomarkers in Long COVID patients\nAuthors: MENEZES, S. M.; JAMOULLE, M.; Carletto, M. P.; Van Holm, B.; Moens, L.; Meyts, I.; Maes, P.; Van Weyenbergh, J.\nScore: 330.8, Published: 2024-01-17 DOI: 10.1101/2024.01.14.24301293\nWith an estimated 65 million individuals suffering from Long COVID, validated therapeutic strategies as well as non-invasive biomarkers are direly needed to guide clinical management. We used blood digital transcriptomics in search of viral persistence and Long COVID diagnostic biomarkers in a real-world, general practice-based setting with a long clinical follow-up. We demonstrate systemic SARS-CoV-2 persistence for more than 2 years after acute COVID-19 infection. A 2-gene biomarker, including FYN and SARS-CoV-2 antisense RNA, correctly classifies Long COVID with 93.8% sensitivity and 91.7% specificity. Specific immune transcripts and immunometabolism score correlate to systemic viral load and patient-reported anxiety/depression, providing mechanistic links as well as therapeutic targets to tackle Long COVID.\nRoutes of importation and spatial dynamics of SARS-CoV-2 variants during localised interventions in Chile\nAuthors: Gutierrez, B.; Tsui, J. L.- H.; Pullano, G.; Mazzoli, M.; Gangavarapu, K.; Inward, R.; Bajaj, S.; Evans Pena, R.; Busch Moreno, S.; Suchard, M. A.; Pybus, O. G.; Dunner, A.; Puentes, R.; Ayala, S.; Fernandez, J.; Araos, R.; Ferres, L.; Colizza, V.; Kraemer, M. U. G.\nScore: 47.5, Published: 2024-01-22 DOI: 10.1101/2024.01.18.24301504\nSouth America suffered large SARS-CoV-2 epidemics between 2020 and 2022 caused by multiple variants of interest and concern, some causing substantial morbidity and mortality. However, their transmission dynamics are poorly characterised. The epidemic situation in Chile enables us to investigate differences in the distribution and spread of variants Alpha, Gamma, Lambda, Mu and Delta. Chile implemented non-pharmaceutical interventions and an integrated genomic and epidemiological surveillance system that included airport and community surveillance to track SARS-CoV-2 variants. Here we combine viral genomic data and anonymised human mobility data from mobile phones to characterise the routes of importation of different variants into Chile, the relative contributions of airport-based importations to viral diversity versus land border crossings and test the impact of the mobility network on the diffusion of viral lineages within the country. We find that Alpha, Lambda and Mu were identified in Chile via airport surveillance six, four and five weeks ahead of their detection via community surveillance, respectively. Further, some variants that originated in South America were imported into Chile via land rather than international air travel, most notably Gamma. Different variants exhibited similar trends of viral dissemination throughout the country following their importation, and we show that the mobility network predicts the time of arrival of imported lineages to different Chilean comunas. Higher stringency of local NPIs was also associated with fewer domestic viral importations. Our results show how genomic surveillance combined with high resolution mobility data can help predict the multi-scale geographic expansion of emerging infectious diseases. Significance statementGlobal preparedness for pandemic threats requires an understanding of the global variations of spatiotemporal transmission dynamics. Regional differences are important because the local context sets the conditions for the unfolding of local epidemics, which in turn affect transmission dynamics at a broader scale. Knowledge gaps from the SARS-CoV-2 pandemic remain for regions like South America, where distinct sets of viral variants emerged and spread from late 2020 onwards, and where changes in human behaviour resulted in epidemics which differed from those observed in other regions. Our interdisciplinary analysis of the SARS-CoV-2 epidemic in Chile provides insights into the spatiotemporal trends of viral diffusion in the region which shed light on the drivers that can influence future epidemic waves and pandemics.\nMonitoring Report: Respiratory Viruses - December 2023 Data\nAuthors: Gratzl, S.; Cartwright, B. M. G.; Rodriguez, P. J.; Baker, C.; Stucky, N.\nScore: 43.1, Published: 2024-01-22 DOI: 10.1101/2024.01.20.24301528\nBackgroundFew sources regularly monitor hospitalizations associated with respiratory viruses. This study provides current hospitalization trends associated with six common respiratory viruses: COVID-19, influenza, human metapneumovirus (HMPV), parainfluenza virus, respiratory syncytial virus (RSV), and rhinovirus. ObjectiveThis study aims to supplement the surveillance data provided by the CDC by describing trends overall and for each respiratory virus. This study also provides valuable insight into two at-risk populations: infants and children (age 0-4) and older adults (age 65 and over). MethodsUsing a subset of real-world electronic health record (EHR) data from Truveta, a growing collective of health systems that provide more than 18% of all daily clinical care in the USA consortium of US healthcare systems, we identified people who were hospitalized between October 01, 2019 and December 31, 2023. We identified people who tested positive for any of the six respiratory virus within 14 days of the hospitalization. We report weekly trends in the rate of hospitalizations associated with each virus per all hospitalizations for the overall population and the two high-risk sub populations, infants and children and older adults. ResultsWe included 502,484 hospitalizations of 471,401 unique patients who tested positive for a respiratory virus between October 01, 2019 and December 31, 2023. Overall, the rate of hospitalizations associated with respiratory viruses continue to increase through the end of December 2023 (92.8% increase since November 2023). The largest increases over the past month have been seen in influenza- (251.7% increase), HMPV- (115.0% increase), and COVID-associated (88.6% increase) hospitalizations. For infants and children, RSV accounted for the most respiratory virus-associated hospitalizations, despite decreases throughout December 2023. In this population, COVID-associated hospitalizations had the biggest increase in the last month. For the older adult population, respiratory virus-associated hospitalizations increased to 10.3% per all hospitalizations. COVID remained the largest contributor; 5.3% of all hospitalizations were associated with COVID in the older adult population. DiscussionRespiratory virus-associated hospitalizations continued to increase overall, including for the two high risk populations we studied. RSV-associated hospitalizations continue to be the main contributor to illness in infants and children, while COVID-associated hospitalizations are the largest contributor for the older adult population. We will continue to monitor trends in all respiratory viruses. Trends in Surveillance Overall population O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=101 SRC=\"FIGDIR/small/24301528v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (23K): org.highwire.dtl.DTLVardef@169dc4aorg.highwire.dtl.DTLVardef@fd8a48org.highwire.dtl.DTLVardef@18190f7org.highwire.dtl.DTLVardef@15470e8_HPS_FORMAT_FIGEXP M_FIG C_FIG Overall, the rate of hospitalizations associated with respiratory viruses continue to increase through the end of December 2023 (92.8% increase since November 2023). Specifically, COVID- (88.6%), influenza- (251.7%), and HMPV- (115.0%) associated hospitalization trends increased. COVID and influenza-associated hospitalizations are at 3.3% and 3.0% of all hospitalizations, respectively. Other viruses are each below 1% of hospitalizations. Nevertheless, at the end of December 2023, respiratory virus-associated hospitalizations accounted for 7.7% of all hospitalizations. Infants and children (age 0-4) O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=70 SRC=\"FIGDIR/small/24301528v1_ufig2.gif\" ALT=\"Figure 2\"\u003e O_LINKSMALLFIG WIDTH=200 HEIGHT=31 SRC=\"FIGDIR/small/24301528v1_ufig2a.gif\" ALT=\"Figure 2\"\u003e View larger version (31K): org.highwire.dtl.DTLVardef@c0b1e0org.highwire.dtl.DTLVardef@de18b2org.highwire.dtl.DTLVardef@12786fborg.highwire.dtl.DTLVardef@ee507_HPS_FORMAT_FIGEXP M_FIG C_FIG For the population between 0-4 years old, in December 2023, RSV accounted for the most respiratory virus-associated hospitalizations, despite decreasing rates since November 2023 (24.7% decrease). COVID- (183.5% increase), influenza- (52.5% increase), and rhino-associated (39.5% increase) hospitalizations are increasing in this population. In this age group, 5.1% of all hospitalizations were associated with respiratory viruses in December 2023, a 10.5% increase from November 2023. Older adults (age 65 and over) O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=105 SRC=\"FIGDIR/small/24301528v1_ufig3.gif\" ALT=\"Figure 3\"\u003e View larger version (25K): org.highwire.dtl.DTLVardef@1de30a5org.highwire.dtl.DTLVardef@4bd8fdorg.highwire.dtl.DTLVardef@192d57aorg.highwire.dtl.DTLVardef@fd645f_HPS_FORMAT_FIGEXP M_FIG C_FIG In the older adults population (individuals over 65 years of age), there was a 93.7% increase in hospitalizations associated with respiratory viruses; 10.3% of all hospitalizations are associated with a respiratory virus. Influenza-associated hospitalizations had the largest percent increase between November and December 2023 (267.8%). COVID-associated hospitalizations continue to make up the largest percentage of hospitalizations in this age group (5.3% in December).\nPredicting COVID-19 cases across a large university campus using complementary built environment and wastewater surveillance approaches\nAuthors: Hinz, A.; Moggridge, J. A.; Ke, H.; Hicks, A. M. A.; Doukhanine, E.; Fralick, M.; Hug, L.; MacFadden, D.; Mejbel, H.; Nott, C.; Raudanskis, A.; Thampi, N.; Wong, A.; Kassen, R.\nScore: 22.6, Published: 2024-01-26 DOI: 10.1101/2024.01.24.23300025\nBackground Environmental surveillance of SARS-CoV-2 via wastewater has become an invaluable tool for population-level surveillance. Built environment sampling may provide complementary spatially-refined detection for viral surveillance in congregate settings such as universities. Methods We conducted a prospective environmental surveillance study at the University of Ottawa between September 2021 and April 2022. Floor surface samples were collected twice weekly from six university buildings. Samples were analyzed for the presence of SARS-CoV-2 using RT-qPCR. A Poisson regression was used to model the campus-wide COVID-19 cases predicted from the fraction of floor swabs positive for SARS-CoV-2 RNA, building CO2 levels, Wi-Fi usage, and SARS-CoV-2 RNA levels in regional wastewater. We used a mixed-effects Poisson regression analysis to model building-level cases using viral copies detected in floor samples as a predictor. A random intercepts logistic regression model tested whether floor samples collected in high-traffic areas were more likely to have SARS-CoV-2 present than low-traffic areas. Results Over the 32-week study period, we collected 554 floor swabs at six university buildings. Overall, 13% of swabs were PCR-positive for SARS-CoV-2, with positivity ranging between 4.8% and 32.7% among university buildings. Both floor swab positivity (Spearman r = 0.74, 95% CI: 0.53-0.87) and regional wastewater signal (Spearman r = 0.50, 95% CI: 0.18-0.73) were positively correlated with on-campus COVID-19 cases. In addition, built environment detection was a predictor of cases linked to individual university buildings (IR log10(copies) + 1 = 17, 95% CI: 7-44). There was no significant difference in detection between floors sampled in high-traffic versus low-traffic areas (OR = 1.3, 95% CI: 0.8-2.1). Conclusions Detection of SARS-CoV-2 RNA on floors and viral RNA levels found in wastewater were strongly associated with the incidence of COVID-19 cases on a university campus. These data suggest a potential role for institutional built environment sampling, used together with wastewater surveillance, for predicting COVID-19 cases at both campus-wide and building level scales.\nImpact of COVID-19 vaccination on symptoms and immune phenotypes in vaccine-naIÌˆve individuals with Long COVID\nAuthors: Grady, C. B.; Bhattacharjee, B.; Silva, J.; Jaycox, J.; Lee, L. W.; Monteiro, V. S.; Sawano, M.; Massey, D.; Caraballo, C.; Gehlhausen, J.; Tabachnikova, A.; Mao, T.; Lucas, C.; Pena-Hernandez, M. A.; Xu, L.; Tzeng, T. J.; Takahashi, T.; Herrin, J.; Guthe, D. B.; Akrami, A.; Assaf, G.; Davis, H.; Harris, K.; McCorkell, L.; Schulz, W.; Griffin, D.; Wei, H.; Ring, A. M.; Guan, L.; Dela Cruz, C.; Iwasaki, A.; Krumholz, H.\nScore: 72.6, Published: 2024-01-12 DOI: 10.1101/2024.01.11.24300929\nBackgroundLong COVID contributes to the global burden of disease. Proposed root cause hypotheses include the persistence of SARS-CoV-2 viral reservoir, autoimmunity, and reactivation of latent herpesviruses. Patients have reported various changes in Long COVID symptoms after COVID-19 vaccinations, leaving uncertainty about whether vaccine-induced immune responses may alleviate or worsen disease pathology. MethodsIn this prospective study, we evaluated changes in symptoms and immune responses after COVID-19 vaccination in 16 vaccine-naive individuals with Long COVID. Surveys were administered before vaccination and then at 2, 6, and 12 weeks after receiving the first vaccine dose of the primary series. Simultaneously, SARS-CoV-2-reactive TCR enrichment, SARS-CoV-2-specific antibody responses, antibody responses to other viral and self-antigens, and circulating cytokines were quantified before vaccination and at 6 and 12 weeks after vaccination. ResultsSelf-report at 12 weeks post-vaccination indicated 10 out of 16 participants had improved health, 3 had no change, 1 had worse health, and 2 reported marginal changes. Significant elevation in SARS-CoV-2-specific TCRs and Spike protein-specific IgG were observed 6 and 12 weeks after vaccination. No changes in reactivities were observed against herpes viruses and self-antigens. Within this dataset, higher baseline sIL-6R was associated with symptom improvement, and the two top features associated with non-improvement were high IFN-{beta} and CNTF, among soluble analytes. ConclusionsOur study showed that in this small sample, vaccination improved the health or resulted in no change to the health of most participants, though few experienced worsening. Vaccination was associated with increased SARS-CoV-2 Spike protein-specific IgG and T cell expansion in most individuals with Long COVID. Symptom improvement was observed in those with baseline elevated sIL-6R, while elevated interferon and neuropeptide levels were associated with a lack of improvement. Plain language summaryThe impact of the COVID-19 vaccine on vaccine-naive individuals suffering from Long COVID is uncertain. This study assessed the experience and immune signatures of 16 unvaccinated participants with Long COVID. A total of 10 participants had improved health status after vaccination, and one person reported only worsening health. As expected, vaccination increased immune cells and antibodies against the viral spike protein. Immune signatures may prove to be predictors of health status after vaccination. However, given the small number of participants, these initial findings need further validation.\nStrategic vaccine stockpiles for regional epidemics of emerging viruses: a geospatial modeling framework\nAuthors: Carlson, C. J.; Garnier, R.; Tiu, A. J.; Luby, S. P.; Bansal, S.\nScore: 6.1, Published: 2024-01-20 DOI: 10.1101/2024.01.19.24301505\nMultinational epidemics of emerging infectious diseases are increasingly common, due to anthropogenic pressure on ecosystems and the growing connectivity of human populations. Early and efficient vaccination can contain outbreaks and prevent mass mortality, but optimal vaccine stockpiling strategies are dependent on pathogen characteristics, reservoir ecology, and epidemic dynamics. Here, we model major regional outbreaks of Nipah virus and Middle East respiratory syndrome, and use these to develop a generalized framework for estimating vaccine stockpile needs based on spillover geography, spatially-heterogeneous healthcare capacity and spatially-distributed human mobility networks. Because outbreak sizes were highly skewed, we found that most outbreaks were readily contained (median stockpile estimate for MERS-CoV: 2,089 doses; Nipah: 1,882 doses), but the maximum estimated stockpile need in a highly unlikely large outbreak scenario was 2-3 orders of magnitude higher (MERS-CoV: [~]87,000 doses; Nipah [~]1.1 million doses). Sensitivity analysis revealed that stockpile needs were more dependent on basic epidemiological parameters (i.e., death and recovery rate) and healthcare availability than any uncertainty related to vaccine efficacy or deployment strategy. Our results highlight the value of descriptive epidemiology for real-world modeling applications, and suggest that stockpile allocation should consider ecological, epidemiological, and social dimensions of risk.\nSARS-CoV-2 vaccination in the first year after hematopoietic cell transplant or chimeric antigen receptor T cell therapy: A prospective, multicenter, observational study (BMT CTN 2101)\nAuthors: Hill, J. A.; Martens, M. J.; Young, J.-A. H.; Bhavsar, K.; Kou, J.; Chen, M.; Lee, L. W.; Baluch, A.; Dhodapkar, M. V.; Nakamura, R.; Peyton, K.; Howard, D. S.; Ibrahim, U.; Shahid, Z.; Armistead, P.; Westervelt, P.; McCarty, J.; McGuirk, J.; Hamadani, M.; DeWolf, S.; Hosszu, K.; Sharon, E.; Spahn, A.; Toor, A. A.; Waldvogel, S.; Greenberger, L. M.; Auletta, J. J.; Horowitz, M. M.; Riches, M. L.; Perales, M.-A.\nScore: 4.7, Published: 2024-01-25 DOI: 10.1101/2024.01.24.24301058\nBackground The optimal timing of vaccination with SARS-CoV-2 vaccines after cellular therapy is incompletely understood. Objective To describe humoral and cellular responses after SARS-CoV-2 vaccination initiated \u003c4 months versus 4-12 months after cellular therapy. Design Multicenter prospective observational study. Setting 34 centers in the United States. Participants 466 allogeneic hematopoietic cell transplant (HCT; n=231), autologous HCT (n=170), or chimeric antigen receptor T cell (CAR-T cell) therapy (n=65) recipients enrolled between April 2021 and June 2022. Interventions SARS-CoV-2 vaccination as part of routine care. Measurements We obtained blood prior to and after vaccinations at up to five time points and tested for SARS-CoV-2 spike (anti-S) IgG in all participants and neutralizing antibodies for Wuhan D614G, Delta B.1.617.2, and Omicron B.1.1.529 strains, as well as SARS-CoV-2-specific T cell receptors (TCRs), in a subgroup. Results Anti-S IgG and neutralizing antibody responses increased with vaccination in HCT recipients irrespective of vaccine initiation timing but were unchanged in CAR-T cell therapy recipients initiating vaccines within 4 months. Anti-S IgG [\u0026ge;]2,500 U/mL was correlated with high neutralizing antibody titers and attained by the last time point in 70%, 69%, and 34% of allogeneic HCT, autologous HCT, and CAR-T cell therapy recipients, respectively. SARS-CoV-2-specific T cell responses were attained in 57%, 83%, and 58%, respectively. Humoral and cellular responses did not significantly differ among participants initiating vaccinations \u003c4 months vs 4-12 months after cellular therapy. Pre-cellular therapy SARS-CoV-2 infection or vaccination were key predictors of post-cellular therapy anti-S IgG levels. Limitations The majority of participants were adults and received mRNA vaccines. Conclusions These data support starting mRNA SARS-CoV-2 vaccination three to four months after allogeneic HCT, autologous HCT, and CAR-T cell therapy. Funding National Marrow Donor Program, Leukemia and Lymphoma Society, Multiple Myeloma Research Foundation, Novartis, LabCorp, American Society for Transplantation and Cellular Therapy, Adaptive Biotechnologies, and the National Institutes of Health\nAntiviral efficacy of fluoxetine in early symptomatic COVID-19: an open-label, randomised, controlled, adaptive platform trial (PLATCOV)\nAuthors: Jittamala, P.; Boyd, S.; Schilling, W. H. K.; Watson, J. A.; Ngamprasertchai, T.; Siripoon, T.; Luvira, V.; Batty, E. M.; Wongnak, P.; Esper, L. M.; Almeida, P. J.; Cruz, C.; Ascencao, F. R.; Aguiar, R. S.; Ghanchi, N. K.; Callery, J. J.; Singh, S.; Kruabkontho, V.; Ngernseng, T.; Tubprasert, J.; Madmanee, W.; Suwannasin, K.; Promsongsil, A.; Hanboonkunupakarn, B.; Poovorawan, K.; Potaporn, M.; Srisubat, A.; Loharjun, B.; Taylor, W. R.; Qamar, F. N.; Kazi, A. M.; Beg, M. A.; Chommanam, D.; Vidhamaly, S.; Chotivanich, K.; Imwong, M.; Pukrittayakamee, S.; Dondorp, A. M.; Day, N. P.; Teixeira, M.\nScore: 2.8, Published: 2024-01-18 DOI: 10.1101/2024.01.16.24301337\nBackgroundThe selective serotonin reuptake inhibitors (SSRIs) fluoxetine and fluvoxamine were repurposed for the treatment of early COVID-19 based on their antiviral activity in vitro, and observational and clinical trial evidence suggesting they prevented progression to severe disease. However, these SSRIs have not been recommended in guidelines and their antiviral activity in vivo has not been characterised. MethodsPLATCOV is an open-label, multicentre, phase 2, randomised, controlled, adaptive pharmacometric platform trial running in Thailand, Brazil, Pakistan, and Laos. We recruited low-risk adult outpatients aged 18-50 with early symptomatic COVID-19 (symptoms \u003c4 days). Patients were assigned using block randomisation to one of eleven treatment arms including oral fluoxetine (40mg/day for 7 days), or no study drug. Uniform randomisation ratios were applied across the active treatment groups while the no study drug group comprised [\u0026ge;]20% of patients at all times. The primary endpoint was the rate of oropharyngeal viral clearance assessed in a modified intention-to-treat population (\u003e2 days follow-up). The viral clearance rate was estimated under a Bayesian hierarchical linear model fitted to the log10 viral densities in standardised duplicate oropharyngeal swab eluates taken daily over one week (18 measurements per patient). This ongoing trial is registered at ClinicalTrials.gov (NCT05041907). FindingsBetween 5 April 2022 and 8 May 2023 271 patients were concurrently randomised to either fluoxetine (n=120) or no study drug (n=151). Fluoxetine was well tolerated and accelerated the rate of viral clearance relative to the no study drug arm by 15% (95% credible interval (CrI): 2% to 34%). In a pooled meta-analysis including all unblinded patients the antiviral activity of fluoxetine was substantially less than ritonavir-boosted nirmatrelvir-85% increase in rate of viral clearance (95% CrI: 61 to 112%); and less than remdesivir 35% (14 to 59%), molnupiravir 37% (18 to 60%), and casirivimab/imdevimab 29% (10 to 48%). InterpretationFluoxetine has in vivo antiviral activity against SARS-CoV-2. Although the level of antiviral efficacy is substantially less than with other currently available antiviral drugs, fluoxetine might still be useful in prophylaxis where less antiviral effect is required. FundingWellcome Trust Grant ref: 223195/Z/21/Z through the COVID-19 Therapeutics Accelerator. Evidence before this studyThe SSRIs fluoxetine and fluvoxamine have been proposed as COVID-19 therapeutics based initially on observational, randomised trial and in vitro evidence. The observational reports suggested that patients taking SSRIs had a reduced probability of developing severe COVID-19 and dying. We searched PubMed and EMBASE for studies in English up until the 30th November 2023 using the search terms \"fluoxetine\", \"fluvoxamine\" and \"COVID-19\" with the search restricted to randomised controlled trials (RCTs). Eight outpatient fluvoxamine RCTs were identified. There were no fluoxetine RCTs in outpatients. A meta-analysis of available RCTs is compatible with a moderate reduction in hospitalisation and death in COVID-19 patients with an estimated risk ratio of 0.80 (95% CI: 0.62,1.01). Added value of the studyWe showed that in early COVID-19 illness the SSRI fluoxetine has weak antiviral activity in vivo. This activity is substantially less than other available antivirals such as ritonavir-boosted nirmatrelvir and molnupiravir. The pharmacometric approach described here provides a quantitative measure of in vivo antiviral effects with tractable sample sizes. Implications of available evidenceFluoxetine has weak in vivo antiviral activity in early COVID-19. This is insufficient for treatment but, as less antiviral activity is required to prevent an infection, fluoxetine could still be beneficial in prophylaxis.\nDODGE: Automated point source bacterial outbreak detection using cumulative long term genomic surveillance.\nAuthors: Payne, M.; Hu, D.; Wang, Q.; Sullivan, G.; Graham, R. M.; Rathnayake, I. U.; Jennison, A.; Sintchenko, V.; Lan, R.\nScore: 2.0, Published: 2024-01-22 DOI: 10.1101/2024.01.21.24301506\nSummaryThe reliable and timely recognition of outbreaks is a key component of public health surveillance for foodborne diseases. Whole genome sequencing (WGS) offers high resolution typing of foodborne bacterial pathogens and facilitates the accurate detection of outbreaks. This detection relies on grouping WGS data into clusters at an appropriate genetic threshold, however, methods and tools for selecting and adjusting such thresholds according to the required resolution of surveillance and epidemiological context are lacking. Here we present DODGE (Dynamic Outbreak Detection for Genomic Epidemiology), an algorithm to dynamically select and compare these genetic thresholds. DODGE can analyse expanding datasets over time and clusters that are predicted to correspond to outbreaks (or investigation clusters) can be named with the established genomic nomenclature systems to facilitate integrated analysis across jurisdictions. DODGE was tested in two real-world genomic surveillance datasets of different duration, two months from Australia and nine years from the UK. In both cases only a minority of isolates were identified as investigation clusters. Two known outbreaks in the UK dataset were detected by DODGE and were recognised at an earlier timepoint than the outbreaks were reported. These findings demonstrated the potential of the DODGE approach to improve the effectiveness and timeliness of genomic surveillance for foodborne diseases and the effectiveness of the algorithm developed. Availability and implementationDODGE is freely available at https://github.com/LanLab/dodge and can easily be installed using Conda. Supplementary informationSupplementary Tables, Results, Figure 1 and Figure 2 O_FIG O_LINKSMALLFIG WIDTH=193 HEIGHT=200 SRC=\"FIGDIR/small/24301506v1_fig1.gif\" ALT=\"Figure 1\"\u003e View larger version (48K): org.highwire.dtl.DTLVardef@4b93edorg.highwire.dtl.DTLVardef@1dcbf38org.highwire.dtl.DTLVardef@18e25acorg.highwire.dtl.DTLVardef@14b63e7_HPS_FORMAT_FIGEXP M_FIG O_FLOATNOFigure 1.C_FLOATNO The DODGE pipeline, algorithm and Australian dataset investigation clusters. A. Flowchart describing 6 stages of the DODGE algorithm. B. Example investigation cluster detection with the same 6 stages marked. Blue circles represent isolates, red numbered lines are genetic distances. At each genetic threshold isolates within the grey shaded area are the cluster being evaluated. C. High level schematic of the DODGE pipeline including the DODGE algorithm. Genetic data in the form of allele profiles (Enterobase or MGTdb) or SNPs (output by snippy) for isolates from a given temporal window (a week or month) are combined with previous time periods to generate a combined distance matrix. Distances between isolate pairs that are not in an optional input distance matrix (Blue arrow) are calculated and added. Clusters are identified using single linkage clustering from the distance matrix. These clusters are compared to existing investigation and non-investigation clusters from previous time periods (blue arrow) to identify expanded or unchanged investigation clusters. Remaining non investigation clusters are then used to identify novel investigation clusters using the DODGE algorithm detailed in B and C. Green boxes are input files, red outlined boxes are output files, blue arrows represent outputs from one time period used as inputs in the next. D. Investigation clusters identified from the Australian dataset over time. X axis is date of collection by week. Y axis is investigation cluster with MGT ST based ID. The area of circles is proportional to number of isolates in that investigation cluster in that week. Colour represents the genetic threshold used for that investigation cluster. Red outline indicates the week in which the cluster was identified as an investigation cluster by the DODGE algorithm. C_FIG\n",
  "wordCount" : "4134",
  "inLanguage": "en",
  "datePublished": "2024-01-28T10:37:59Z",
  "dateModified": "2024-01-28T10:37:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 28, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.24301374">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.24301374" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.24301374">
        <p class="paperTitle">Altered IgG4 Antibody Response to Repeated mRNA versus Protein COVID Vaccines</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.24301374" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.24301374" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kalkeri, R.; Zhu, M.; Cloney-Clark, S.; Plested, J. S.; Parekh, A.; Gorinson, D.; Cai, R.; Mahato, S.; Ramanathan, P.; Aurelia, L. C.; John Selva, K.; Marchese, A. M.; Fries, L.; Chung, A. W.; Dunkle, L. M.</p>
        <p class="info">Score: 283.3, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.24301374' target='https://doi.org/10.1101/2024.01.17.24301374'> 10.1101/2024.01.17.24301374</a></p>
        <p class="abstract">Repeated mRNA SARS-CoV-2 vaccination has been associated with increases in the proportion of IgG4 in spike-specific antibody responses and concurrent reductions in Fc{gamma}-mediated effector functions that may limit control of viral infection. Here, we assessed anti-Spike total IgG, IgG1, IgG2, IgG3 and IgG4, and surrogate markers for antibody-dependent cellular phagocytosis (ADCP, Fc{gamma}RIIa binding), antibody-dependent cellular cytotoxicity (ADCC, Fc{gamma}RIIIa binding), and antibody-dependent complement deposition (ADCD, C1q binding) associated with repeated SARS-CoV-2 vaccination with NVX-CoV2373 (Novavax Inc., Gaithersburg, MD). The NVX-CoV2373 protein vaccine did not induce notable increases in spike-specific IgG4 or negatively impact surrogates for Fc{gamma} effector responses. Conversely, repeated NVX-CoV2373 vaccination uniquely enhanced IgG3 responses which are known to exhibit strong affinity for Fc{gamma}RIIIa and have previously been linked to potent neutralization of SARS-CoV-2. Subsequent investigations will help to understand the immunological diversity generated by different SARS-CoV-2 vaccine types and have the potential to reshape public health strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.24301293">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.24301293" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.24301293">
        <p class="paperTitle">Blood transcriptomics reveal persistent SARS-CoV-2 RNA and candidate biomarkers in Long COVID patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.24301293" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.24301293" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: MENEZES, S. M.; JAMOULLE, M.; Carletto, M. P.; Van Holm, B.; Moens, L.; Meyts, I.; Maes, P.; Van Weyenbergh, J.</p>
        <p class="info">Score: 330.8, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.24301293' target='https://doi.org/10.1101/2024.01.14.24301293'> 10.1101/2024.01.14.24301293</a></p>
        <p class="abstract">With an estimated 65 million individuals suffering from Long COVID, validated therapeutic strategies as well as non-invasive biomarkers are direly needed to guide clinical management. We used blood digital transcriptomics in search of viral persistence and Long COVID diagnostic biomarkers in a real-world, general practice-based setting with a long clinical follow-up. We demonstrate systemic SARS-CoV-2 persistence for more than 2 years after acute COVID-19 infection. A 2-gene biomarker, including FYN and SARS-CoV-2 antisense RNA, correctly classifies Long COVID with 93.8% sensitivity and 91.7% specificity. Specific immune transcripts and immunometabolism score correlate to systemic viral load and patient-reported anxiety/depression, providing mechanistic links as well as therapeutic targets to tackle Long COVID.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.24301504">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.24301504" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.24301504">
        <p class="paperTitle">Routes of importation and spatial dynamics of SARS-CoV-2 variants during localised interventions in Chile</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.24301504" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.24301504" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gutierrez, B.; Tsui, J. L.- H.; Pullano, G.; Mazzoli, M.; Gangavarapu, K.; Inward, R.; Bajaj, S.; Evans Pena, R.; Busch Moreno, S.; Suchard, M. A.; Pybus, O. G.; Dunner, A.; Puentes, R.; Ayala, S.; Fernandez, J.; Araos, R.; Ferres, L.; Colizza, V.; Kraemer, M. U. G.</p>
        <p class="info">Score: 47.5, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.24301504' target='https://doi.org/10.1101/2024.01.18.24301504'> 10.1101/2024.01.18.24301504</a></p>
        <p class="abstract">South America suffered large SARS-CoV-2 epidemics between 2020 and 2022 caused by multiple variants of interest and concern, some causing substantial morbidity and mortality. However, their transmission dynamics are poorly characterised. The epidemic situation in Chile enables us to investigate differences in the distribution and spread of variants Alpha, Gamma, Lambda, Mu and Delta. Chile implemented non-pharmaceutical interventions and an integrated genomic and epidemiological surveillance system that included airport and community surveillance to track SARS-CoV-2 variants. Here we combine viral genomic data and anonymised human mobility data from mobile phones to characterise the routes of importation of different variants into Chile, the relative contributions of airport-based importations to viral diversity versus land border crossings and test the impact of the mobility network on the diffusion of viral lineages within the country. We find that Alpha, Lambda and Mu were identified in Chile via airport surveillance six, four and five weeks ahead of their detection via community surveillance, respectively. Further, some variants that originated in South America were imported into Chile via land rather than international air travel, most notably Gamma. Different variants exhibited similar trends of viral dissemination throughout the country following their importation, and we show that the mobility network predicts the time of arrival of imported lineages to different Chilean comunas. Higher stringency of local NPIs was also associated with fewer domestic viral importations. Our results show how genomic surveillance combined with high resolution mobility data can help predict the multi-scale geographic expansion of emerging infectious diseases.

Significance statementGlobal preparedness for pandemic threats requires an understanding of the global variations of spatiotemporal transmission dynamics. Regional differences are important because the local context sets the conditions for the unfolding of local epidemics, which in turn affect transmission dynamics at a broader scale. Knowledge gaps from the SARS-CoV-2 pandemic remain for regions like South America, where distinct sets of viral variants emerged and spread from late 2020 onwards, and where changes in human behaviour resulted in epidemics which differed from those observed in other regions. Our interdisciplinary analysis of the SARS-CoV-2 epidemic in Chile provides insights into the spatiotemporal trends of viral diffusion in the region which shed light on the drivers that can influence future epidemic waves and pandemics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.24301528">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.24301528" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.24301528">
        <p class="paperTitle">Monitoring Report: Respiratory Viruses - December 2023 Data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.24301528" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.24301528" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gratzl, S.; Cartwright, B. M. G.; Rodriguez, P. J.; Baker, C.; Stucky, N.</p>
        <p class="info">Score: 43.1, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.24301528' target='https://doi.org/10.1101/2024.01.20.24301528'> 10.1101/2024.01.20.24301528</a></p>
        <p class="abstract">BackgroundFew sources regularly monitor hospitalizations associated with respiratory viruses. This study provides current hospitalization trends associated with six common respiratory viruses: COVID-19, influenza, human metapneumovirus (HMPV), parainfluenza virus, respiratory syncytial virus (RSV), and rhinovirus.

ObjectiveThis study aims to supplement the surveillance data provided by the CDC by describing trends overall and for each respiratory virus. This study also provides valuable insight into two at-risk populations: infants and children (age 0-4) and older adults (age 65 and over).

MethodsUsing a subset of real-world electronic health record (EHR) data from Truveta, a growing collective of health systems that provide more than 18% of all daily clinical care in the USA consortium of US healthcare systems, we identified people who were hospitalized between October 01, 2019 and December 31, 2023. We identified people who tested positive for any of the six respiratory virus within 14 days of the hospitalization. We report weekly trends in the rate of hospitalizations associated with each virus per all hospitalizations for the overall population and the two high-risk sub populations, infants and children and older adults.

ResultsWe included 502,484 hospitalizations of 471,401 unique patients who tested positive for a respiratory virus between October 01, 2019 and December 31, 2023.

Overall, the rate of hospitalizations associated with respiratory viruses continue to increase through the end of December 2023 (92.8% increase since November 2023). The largest increases over the past month have been seen in influenza- (251.7% increase), HMPV- (115.0% increase), and COVID-associated (88.6% increase) hospitalizations. For infants and children, RSV accounted for the most respiratory virus-associated hospitalizations, despite decreases throughout December 2023. In this population, COVID-associated hospitalizations had the biggest increase in the last month. For the older adult population, respiratory virus-associated hospitalizations increased to 10.3% per all hospitalizations. COVID remained the largest contributor; 5.3% of all hospitalizations were associated with COVID in the older adult population.

DiscussionRespiratory virus-associated hospitalizations continued to increase overall, including for the two high risk populations we studied. RSV-associated hospitalizations continue to be the main contributor to illness in infants and children, while COVID-associated hospitalizations are the largest contributor for the older adult population. We will continue to monitor trends in all respiratory viruses.

Trends in Surveillance Overall population

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=101 SRC=&#34;FIGDIR/small/24301528v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (23K):
org.highwire.dtl.DTLVardef@169dc4aorg.highwire.dtl.DTLVardef@fd8a48org.highwire.dtl.DTLVardef@18190f7org.highwire.dtl.DTLVardef@15470e8_HPS_FORMAT_FIGEXP  M_FIG C_FIG Overall, the rate of hospitalizations associated with respiratory viruses continue to increase through the end of December 2023 (92.8% increase since November 2023). Specifically, COVID- (88.6%), influenza- (251.7%), and HMPV- (115.0%) associated hospitalization trends increased. COVID and influenza-associated hospitalizations are at 3.3% and 3.0% of all hospitalizations, respectively. Other viruses are each below 1% of hospitalizations. Nevertheless, at the end of December 2023, respiratory virus-associated hospitalizations accounted for 7.7% of all hospitalizations.

Infants and children (age 0-4)

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=70 SRC=&#34;FIGDIR/small/24301528v1_ufig2.gif&#34; ALT=&#34;Figure 2&#34;&gt;
O_LINKSMALLFIG WIDTH=200 HEIGHT=31 SRC=&#34;FIGDIR/small/24301528v1_ufig2a.gif&#34; ALT=&#34;Figure 2&#34;&gt;
View larger version (31K):
org.highwire.dtl.DTLVardef@c0b1e0org.highwire.dtl.DTLVardef@de18b2org.highwire.dtl.DTLVardef@12786fborg.highwire.dtl.DTLVardef@ee507_HPS_FORMAT_FIGEXP  M_FIG C_FIG For the population between 0-4 years old, in December 2023, RSV accounted for the most respiratory virus-associated hospitalizations, despite decreasing rates since November 2023 (24.7% decrease). COVID- (183.5% increase), influenza- (52.5% increase), and rhino-associated (39.5% increase) hospitalizations are increasing in this population. In this age group, 5.1% of all hospitalizations were associated with respiratory viruses in December 2023, a 10.5% increase from November 2023.

Older adults (age 65 and over)

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=105 SRC=&#34;FIGDIR/small/24301528v1_ufig3.gif&#34; ALT=&#34;Figure 3&#34;&gt;
View larger version (25K):
org.highwire.dtl.DTLVardef@1de30a5org.highwire.dtl.DTLVardef@4bd8fdorg.highwire.dtl.DTLVardef@192d57aorg.highwire.dtl.DTLVardef@fd645f_HPS_FORMAT_FIGEXP  M_FIG C_FIG In the older adults population (individuals over 65 years of age), there was a 93.7% increase in hospitalizations associated with respiratory viruses; 10.3% of all hospitalizations are associated with a respiratory virus. Influenza-associated hospitalizations had the largest percent increase between November and December 2023 (267.8%). COVID-associated hospitalizations continue to make up the largest percentage of hospitalizations in this age group (5.3% in December).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.23300025">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.23300025" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.23300025">
        <p class="paperTitle">Predicting COVID-19 cases across a large university campus using complementary built environment and wastewater surveillance approaches</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.23300025" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.23300025" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hinz, A.; Moggridge, J. A.; Ke, H.; Hicks, A. M. A.; Doukhanine, E.; Fralick, M.; Hug, L.; MacFadden, D.; Mejbel, H.; Nott, C.; Raudanskis, A.; Thampi, N.; Wong, A.; Kassen, R.</p>
        <p class="info">Score: 22.6, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.23300025' target='https://doi.org/10.1101/2024.01.24.23300025'> 10.1101/2024.01.24.23300025</a></p>
        <p class="abstract">Background Environmental surveillance of SARS-CoV-2 via wastewater has become an invaluable tool for population-level surveillance. Built environment sampling may provide complementary spatially-refined detection for viral surveillance in congregate settings such as universities. Methods We conducted a prospective environmental surveillance study at the University of Ottawa between September 2021 and April 2022. Floor surface samples were collected twice weekly from six university buildings. Samples were analyzed for the presence of SARS-CoV-2 using RT-qPCR. A Poisson regression was used to model the campus-wide COVID-19 cases predicted from the fraction of floor swabs positive for SARS-CoV-2 RNA, building CO2 levels, Wi-Fi usage, and SARS-CoV-2 RNA levels in regional wastewater. We used a mixed-effects Poisson regression analysis to model building-level cases using viral copies detected in floor samples as a predictor. A random intercepts logistic regression model tested whether floor samples collected in high-traffic areas were more likely to have SARS-CoV-2 present than low-traffic areas. Results Over the 32-week study period, we collected 554 floor swabs at six university buildings. Overall, 13% of swabs were PCR-positive for SARS-CoV-2, with positivity ranging between 4.8% and 32.7% among university buildings. Both floor swab positivity (Spearman r = 0.74, 95% CI: 0.53-0.87) and regional wastewater signal (Spearman r = 0.50, 95% CI: 0.18-0.73) were positively correlated with on-campus COVID-19 cases. In addition, built environment detection was a predictor of cases linked to individual university buildings (IR log10(copies) &#43; 1 = 17, 95% CI: 7-44). There was no significant difference in detection between floors sampled in high-traffic versus low-traffic areas (OR = 1.3, 95% CI: 0.8-2.1). Conclusions Detection of SARS-CoV-2 RNA on floors and viral RNA levels found in wastewater were strongly associated with the incidence of COVID-19 cases on a university campus. These data suggest a potential role for institutional built environment sampling, used together with wastewater surveillance, for predicting COVID-19 cases at both campus-wide and building level scales.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.24300929">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.24300929" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.24300929">
        <p class="paperTitle">Impact of COVID-19 vaccination on symptoms and immune phenotypes in vaccine-naIÌˆve individuals with Long COVID</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.24300929" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.24300929" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grady, C. B.; Bhattacharjee, B.; Silva, J.; Jaycox, J.; Lee, L. W.; Monteiro, V. S.; Sawano, M.; Massey, D.; Caraballo, C.; Gehlhausen, J.; Tabachnikova, A.; Mao, T.; Lucas, C.; Pena-Hernandez, M. A.; Xu, L.; Tzeng, T. J.; Takahashi, T.; Herrin, J.; Guthe, D. B.; Akrami, A.; Assaf, G.; Davis, H.; Harris, K.; McCorkell, L.; Schulz, W.; Griffin, D.; Wei, H.; Ring, A. M.; Guan, L.; Dela Cruz, C.; Iwasaki, A.; Krumholz, H.</p>
        <p class="info">Score: 72.6, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.24300929' target='https://doi.org/10.1101/2024.01.11.24300929'> 10.1101/2024.01.11.24300929</a></p>
        <p class="abstract">BackgroundLong COVID contributes to the global burden of disease. Proposed root cause hypotheses include the persistence of SARS-CoV-2 viral reservoir, autoimmunity, and reactivation of latent herpesviruses. Patients have reported various changes in Long COVID symptoms after COVID-19 vaccinations, leaving uncertainty about whether vaccine-induced immune responses may alleviate or worsen disease pathology.

MethodsIn this prospective study, we evaluated changes in symptoms and immune responses after COVID-19 vaccination in 16 vaccine-naive individuals with Long COVID. Surveys were administered before vaccination and then at 2, 6, and 12 weeks after receiving the first vaccine dose of the primary series. Simultaneously, SARS-CoV-2-reactive TCR enrichment, SARS-CoV-2-specific antibody responses, antibody responses to other viral and self-antigens, and circulating cytokines were quantified before vaccination and at 6 and 12 weeks after vaccination.

ResultsSelf-report at 12 weeks post-vaccination indicated 10 out of 16 participants had improved health, 3 had no change, 1 had worse health, and 2 reported marginal changes. Significant elevation in SARS-CoV-2-specific TCRs and Spike protein-specific IgG were observed 6 and 12 weeks after vaccination. No changes in reactivities were observed against herpes viruses and self-antigens. Within this dataset, higher baseline sIL-6R was associated with symptom improvement, and the two top features associated with non-improvement were high IFN-{beta} and CNTF, among soluble analytes.

ConclusionsOur study showed that in this small sample, vaccination improved the health or resulted in no change to the health of most participants, though few experienced worsening. Vaccination was associated with increased SARS-CoV-2 Spike protein-specific IgG and T cell expansion in most individuals with Long COVID. Symptom improvement was observed in those with baseline elevated sIL-6R, while elevated interferon and neuropeptide levels were associated with a lack of improvement.

Plain language summaryThe impact of the COVID-19 vaccine on vaccine-naive individuals suffering from Long COVID is uncertain. This study assessed the experience and immune signatures of 16 unvaccinated participants with Long COVID. A total of 10 participants had improved health status after vaccination, and one person reported only worsening health. As expected, vaccination increased immune cells and antibodies against the viral spike protein. Immune signatures may prove to be predictors of health status after vaccination. However, given the small number of participants, these initial findings need further validation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.19.24301505">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.19.24301505" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.19.24301505">
        <p class="paperTitle">Strategic vaccine stockpiles for regional epidemics of emerging viruses: a geospatial modeling framework</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.19.24301505" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.19.24301505" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carlson, C. J.; Garnier, R.; Tiu, A. J.; Luby, S. P.; Bansal, S.</p>
        <p class="info">Score: 6.1, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.19.24301505' target='https://doi.org/10.1101/2024.01.19.24301505'> 10.1101/2024.01.19.24301505</a></p>
        <p class="abstract">Multinational epidemics of emerging infectious diseases are increasingly common, due to anthropogenic pressure on ecosystems and the growing connectivity of human populations. Early and efficient vaccination can contain outbreaks and prevent mass mortality, but optimal vaccine stockpiling strategies are dependent on pathogen characteristics, reservoir ecology, and epidemic dynamics. Here, we model major regional outbreaks of Nipah virus and Middle East respiratory syndrome, and use these to develop a generalized framework for estimating vaccine stockpile needs based on spillover geography, spatially-heterogeneous healthcare capacity and spatially-distributed human mobility networks. Because outbreak sizes were highly skewed, we found that most outbreaks were readily contained (median stockpile estimate for MERS-CoV: 2,089 doses; Nipah: 1,882 doses), but the maximum estimated stockpile need in a highly unlikely large outbreak scenario was 2-3 orders of magnitude higher (MERS-CoV: [~]87,000 doses; Nipah [~]1.1 million doses). Sensitivity analysis revealed that stockpile needs were more dependent on basic epidemiological parameters (i.e., death and recovery rate) and healthcare availability than any uncertainty related to vaccine efficacy or deployment strategy. Our results highlight the value of descriptive epidemiology for real-world modeling applications, and suggest that stockpile allocation should consider ecological, epidemiological, and social dimensions of risk.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.24301058">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.24301058" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.24301058">
        <p class="paperTitle">SARS-CoV-2 vaccination in the first year after hematopoietic cell transplant or chimeric antigen receptor T cell therapy: A prospective, multicenter, observational study (BMT CTN 2101)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.24301058" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.24301058" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hill, J. A.; Martens, M. J.; Young, J.-A. H.; Bhavsar, K.; Kou, J.; Chen, M.; Lee, L. W.; Baluch, A.; Dhodapkar, M. V.; Nakamura, R.; Peyton, K.; Howard, D. S.; Ibrahim, U.; Shahid, Z.; Armistead, P.; Westervelt, P.; McCarty, J.; McGuirk, J.; Hamadani, M.; DeWolf, S.; Hosszu, K.; Sharon, E.; Spahn, A.; Toor, A. A.; Waldvogel, S.; Greenberger, L. M.; Auletta, J. J.; Horowitz, M. M.; Riches, M. L.; Perales, M.-A.</p>
        <p class="info">Score: 4.7, Published: 2024-01-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.24301058' target='https://doi.org/10.1101/2024.01.24.24301058'> 10.1101/2024.01.24.24301058</a></p>
        <p class="abstract">Background The optimal timing of vaccination with SARS-CoV-2 vaccines after cellular therapy is incompletely understood. Objective To describe humoral and cellular responses after SARS-CoV-2 vaccination initiated &lt;4 months versus 4-12 months after cellular therapy. Design Multicenter prospective observational study. Setting 34 centers in the United States. Participants 466 allogeneic hematopoietic cell transplant (HCT; n=231), autologous HCT (n=170), or chimeric antigen receptor T cell (CAR-T cell) therapy (n=65) recipients enrolled between April 2021 and June 2022. Interventions SARS-CoV-2 vaccination as part of routine care. Measurements We obtained blood prior to and after vaccinations at up to five time points and tested for SARS-CoV-2 spike (anti-S) IgG in all participants and neutralizing antibodies for Wuhan D614G, Delta B.1.617.2, and Omicron B.1.1.529 strains, as well as SARS-CoV-2-specific T cell receptors (TCRs), in a subgroup. Results Anti-S IgG and neutralizing antibody responses increased with vaccination in HCT recipients irrespective of vaccine initiation timing but were unchanged in CAR-T cell therapy recipients initiating vaccines within 4 months. Anti-S IgG [&amp;ge;]2,500 U/mL was correlated with high neutralizing antibody titers and attained by the last time point in 70%, 69%, and 34% of allogeneic HCT, autologous HCT, and CAR-T cell therapy recipients, respectively. SARS-CoV-2-specific T cell responses were attained in 57%, 83%, and 58%, respectively. Humoral and cellular responses did not significantly differ among participants initiating vaccinations &lt;4 months vs 4-12 months after cellular therapy. Pre-cellular therapy SARS-CoV-2 infection or vaccination were key predictors of post-cellular therapy anti-S IgG levels. Limitations The majority of participants were adults and received mRNA vaccines. Conclusions These data support starting mRNA SARS-CoV-2 vaccination three to four months after allogeneic HCT, autologous HCT, and CAR-T cell therapy. Funding National Marrow Donor Program, Leukemia and Lymphoma Society, Multiple Myeloma Research Foundation, Novartis, LabCorp, American Society for Transplantation and Cellular Therapy, Adaptive Biotechnologies, and the National Institutes of Health</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.24301337">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.24301337" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.24301337">
        <p class="paperTitle">Antiviral efficacy of fluoxetine in early symptomatic COVID-19: an open-label, randomised, controlled, adaptive platform trial (PLATCOV)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.24301337" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.24301337" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jittamala, P.; Boyd, S.; Schilling, W. H. K.; Watson, J. A.; Ngamprasertchai, T.; Siripoon, T.; Luvira, V.; Batty, E. M.; Wongnak, P.; Esper, L. M.; Almeida, P. J.; Cruz, C.; Ascencao, F. R.; Aguiar, R. S.; Ghanchi, N. K.; Callery, J. J.; Singh, S.; Kruabkontho, V.; Ngernseng, T.; Tubprasert, J.; Madmanee, W.; Suwannasin, K.; Promsongsil, A.; Hanboonkunupakarn, B.; Poovorawan, K.; Potaporn, M.; Srisubat, A.; Loharjun, B.; Taylor, W. R.; Qamar, F. N.; Kazi, A. M.; Beg, M. A.; Chommanam, D.; Vidhamaly, S.; Chotivanich, K.; Imwong, M.; Pukrittayakamee, S.; Dondorp, A. M.; Day, N. P.; Teixeira, M.</p>
        <p class="info">Score: 2.8, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.24301337' target='https://doi.org/10.1101/2024.01.16.24301337'> 10.1101/2024.01.16.24301337</a></p>
        <p class="abstract">BackgroundThe selective serotonin reuptake inhibitors (SSRIs) fluoxetine and fluvoxamine were repurposed for the treatment of early COVID-19 based on their antiviral activity in vitro, and observational and clinical trial evidence suggesting they prevented progression to severe disease. However, these SSRIs have not been recommended in guidelines and their antiviral activity in vivo has not been characterised.

MethodsPLATCOV is an open-label, multicentre, phase 2, randomised, controlled, adaptive pharmacometric platform trial running in Thailand, Brazil, Pakistan, and Laos. We recruited low-risk adult outpatients aged 18-50 with early symptomatic COVID-19 (symptoms &lt;4 days). Patients were assigned using block randomisation to one of eleven treatment arms including oral fluoxetine (40mg/day for 7 days), or no study drug. Uniform randomisation ratios were applied across the active treatment groups while the no study drug group comprised [&amp;ge;]20% of patients at all times.

The primary endpoint was the rate of oropharyngeal viral clearance assessed in a modified intention-to-treat population (&gt;2 days follow-up). The viral clearance rate was estimated under a Bayesian hierarchical linear model fitted to the log10 viral densities in standardised duplicate oropharyngeal swab eluates taken daily over one week (18 measurements per patient). This ongoing trial is registered at ClinicalTrials.gov (NCT05041907).

FindingsBetween 5 April 2022 and 8 May 2023 271 patients were concurrently randomised to either fluoxetine (n=120) or no study drug (n=151). Fluoxetine was well tolerated and accelerated the rate of viral clearance relative to the no study drug arm by 15% (95% credible interval (CrI): 2% to 34%). In a pooled meta-analysis including all unblinded patients the antiviral activity of fluoxetine was substantially less than ritonavir-boosted nirmatrelvir-85% increase in rate of viral clearance (95% CrI: 61 to 112%); and less than remdesivir 35% (14 to 59%), molnupiravir 37% (18 to 60%), and casirivimab/imdevimab 29% (10 to 48%).

InterpretationFluoxetine has in vivo antiviral activity against SARS-CoV-2. Although the level of antiviral efficacy is substantially less than with other currently available antiviral drugs, fluoxetine might still be useful in prophylaxis where less antiviral effect is required.

FundingWellcome Trust Grant ref: 223195/Z/21/Z through the COVID-19 Therapeutics Accelerator.

Evidence before this studyThe SSRIs fluoxetine and fluvoxamine have been proposed as COVID-19 therapeutics based initially on observational, randomised trial and in vitro evidence. The observational reports suggested that patients taking SSRIs had a reduced probability of developing severe COVID-19 and dying. We searched PubMed and EMBASE for studies in English up until the 30th November 2023 using the search terms &#34;fluoxetine&#34;, &#34;fluvoxamine&#34; and &#34;COVID-19&#34; with the search restricted to randomised controlled trials (RCTs). Eight outpatient fluvoxamine RCTs were identified. There were no fluoxetine RCTs in outpatients. A meta-analysis of available RCTs is compatible with a moderate reduction in hospitalisation and death in COVID-19 patients with an estimated risk ratio of 0.80 (95% CI: 0.62,1.01).

Added value of the studyWe showed that in early COVID-19 illness the SSRI fluoxetine has weak antiviral activity in vivo. This activity is substantially less than other available antivirals such as ritonavir-boosted nirmatrelvir and molnupiravir. The pharmacometric approach described here provides a quantitative measure of in vivo antiviral effects with tractable sample sizes.

Implications of available evidenceFluoxetine has weak in vivo antiviral activity in early COVID-19. This is insufficient for treatment but, as less antiviral activity is required to prevent an infection, fluoxetine could still be beneficial in prophylaxis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.21.24301506">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.21.24301506" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.21.24301506">
        <p class="paperTitle">DODGE: Automated point source bacterial outbreak detection using cumulative long term genomic surveillance.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.21.24301506" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.21.24301506" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Payne, M.; Hu, D.; Wang, Q.; Sullivan, G.; Graham, R. M.; Rathnayake, I. U.; Jennison, A.; Sintchenko, V.; Lan, R.</p>
        <p class="info">Score: 2.0, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.21.24301506' target='https://doi.org/10.1101/2024.01.21.24301506'> 10.1101/2024.01.21.24301506</a></p>
        <p class="abstract">SummaryThe reliable and timely recognition of outbreaks is a key component of public health surveillance for foodborne diseases. Whole genome sequencing (WGS) offers high resolution typing of foodborne bacterial pathogens and facilitates the accurate detection of outbreaks. This detection relies on grouping WGS data into clusters at an appropriate genetic threshold, however, methods and tools for selecting and adjusting such thresholds according to the required resolution of surveillance and epidemiological context are lacking. Here we present DODGE (Dynamic Outbreak Detection for Genomic Epidemiology), an algorithm to dynamically select and compare these genetic thresholds. DODGE can analyse expanding datasets over time and clusters that are predicted to correspond to outbreaks (or  investigation clusters) can be named with the established genomic nomenclature systems to facilitate integrated analysis across jurisdictions. DODGE was tested in two real-world genomic surveillance datasets of different duration, two months from Australia and nine years from the UK. In both cases only a minority of isolates were identified as investigation clusters. Two known outbreaks in the UK dataset were detected by DODGE and were recognised at an earlier timepoint than the outbreaks were reported. These findings demonstrated the potential of the DODGE approach to improve the effectiveness and timeliness of genomic surveillance for foodborne diseases and the effectiveness of the algorithm developed.

Availability and implementationDODGE is freely available at https://github.com/LanLab/dodge and can easily be installed using Conda.

Supplementary informationSupplementary Tables, Results, Figure 1 and Figure 2

O_FIG O_LINKSMALLFIG WIDTH=193 HEIGHT=200 SRC=&#34;FIGDIR/small/24301506v1_fig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (48K):
org.highwire.dtl.DTLVardef@4b93edorg.highwire.dtl.DTLVardef@1dcbf38org.highwire.dtl.DTLVardef@18e25acorg.highwire.dtl.DTLVardef@14b63e7_HPS_FORMAT_FIGEXP  M_FIG O_FLOATNOFigure 1.C_FLOATNO The DODGE pipeline, algorithm and Australian dataset investigation clusters. A. Flowchart describing 6 stages of the DODGE algorithm. B. Example investigation cluster detection with the same 6 stages marked. Blue circles represent isolates, red numbered lines are genetic distances. At each genetic threshold isolates within the grey shaded area are the cluster being evaluated. C. High level schematic of the DODGE pipeline including the DODGE algorithm. Genetic data in the form of allele profiles (Enterobase or MGTdb) or SNPs (output by snippy) for isolates from a given temporal window (a week or month) are combined with previous time periods to generate a combined distance matrix. Distances between isolate pairs that are not in an optional input distance matrix (Blue arrow) are calculated and added. Clusters are identified using single linkage clustering from the distance matrix. These clusters are compared to existing investigation and non-investigation clusters from previous time periods (blue arrow) to identify expanded or unchanged investigation clusters. Remaining non investigation clusters are then used to identify novel investigation clusters using the DODGE algorithm detailed in B and C. Green boxes are input files, red outlined boxes are output files, blue arrows represent outputs from one time period used as inputs in the next. D. Investigation clusters identified from the Australian dataset over time. X axis is date of collection by week. Y axis is investigation cluster with MGT ST based ID. The area of circles is proportional to number of isolates in that investigation cluster in that week. Colour represents the genetic threshold used for that investigation cluster. Red outline indicates the week in which the cluster was identified as an investigation cluster by the DODGE algorithm.

C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
