<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Humoral and cellular immune responses following BNT162b2 XBB.1.5 vaccination
Authors: Stankov, M. V.; Hoffmann, M.; Gutierrez, R.; Cossmann, A.; Morillas Ramos, G.; Graalmann, T.; Friedrichsen, M.; Ravens, I.; Tamara, I.; Ristenpart, J.; Schimrock, A.; Willenzon, S.; Ahrenstorff, G.; Witte, T.; Foerster, R.; Kempf, A.; Poehlmann, S.; Hammerschmidt, S. I.; Dopfer-Jablonka, A.; Behrens, G. M.
Score: 78.2, Published: 2023-10-05 DOI: 10.1101/2023.10.04.23296545
SARS-CoV-2 Omicron XBB subvariants efficiently evade immunity from prior infection or vaccination, requiring vaccine adaptation.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Humoral and cellular immune responses following BNT162b2 XBB.1.5 vaccination
Authors: Stankov, M. V.; Hoffmann, M.; Gutierrez, R.; Cossmann, A.; Morillas Ramos, G.; Graalmann, T.; Friedrichsen, M.; Ravens, I.; Tamara, I.; Ristenpart, J.; Schimrock, A.; Willenzon, S.; Ahrenstorff, G.; Witte, T.; Foerster, R.; Kempf, A.; Poehlmann, S.; Hammerschmidt, S. I.; Dopfer-Jablonka, A.; Behrens, G. M.
Score: 78.2, Published: 2023-10-05 DOI: 10.1101/2023.10.04.23296545
SARS-CoV-2 Omicron XBB subvariants efficiently evade immunity from prior infection or vaccination, requiring vaccine adaptation." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-08T10:37:16+00:00" />
<meta property="article:modified_time" content="2023-10-08T10:37:16+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Humoral and cellular immune responses following BNT162b2 XBB.1.5 vaccination
Authors: Stankov, M. V.; Hoffmann, M.; Gutierrez, R.; Cossmann, A.; Morillas Ramos, G.; Graalmann, T.; Friedrichsen, M.; Ravens, I.; Tamara, I.; Ristenpart, J.; Schimrock, A.; Willenzon, S.; Ahrenstorff, G.; Witte, T.; Foerster, R.; Kempf, A.; Poehlmann, S.; Hammerschmidt, S. I.; Dopfer-Jablonka, A.; Behrens, G. M.
Score: 78.2, Published: 2023-10-05 DOI: 10.1101/2023.10.04.23296545
SARS-CoV-2 Omicron XBB subvariants efficiently evade immunity from prior infection or vaccination, requiring vaccine adaptation."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Humoral and cellular immune responses following BNT162b2 XBB.1.5 vaccination\nAuthors: Stankov, M. V.; Hoffmann, M.; Gutierrez, R.; Cossmann, A.; Morillas Ramos, G.; Graalmann, T.; Friedrichsen, M.; Ravens, I.; Tamara, I.; Ristenpart, J.; Schimrock, A.; Willenzon, S.; Ahrenstorff, G.; Witte, T.; Foerster, R.; Kempf, A.; Poehlmann, S.; Hammerschmidt, S. I.; Dopfer-Jablonka, A.; Behrens, G. M.\nScore: 78.2, Published: 2023-10-05 DOI: 10.1101/2023.10.04.23296545\nSARS-CoV-2 Omicron XBB subvariants efficiently evade immunity from prior infection or vaccination, requiring vaccine adaptation.",
  "keywords": [
    
  ],
  "articleBody": " Humoral and cellular immune responses following BNT162b2 XBB.1.5 vaccination\nAuthors: Stankov, M. V.; Hoffmann, M.; Gutierrez, R.; Cossmann, A.; Morillas Ramos, G.; Graalmann, T.; Friedrichsen, M.; Ravens, I.; Tamara, I.; Ristenpart, J.; Schimrock, A.; Willenzon, S.; Ahrenstorff, G.; Witte, T.; Foerster, R.; Kempf, A.; Poehlmann, S.; Hammerschmidt, S. I.; Dopfer-Jablonka, A.; Behrens, G. M.\nScore: 78.2, Published: 2023-10-05 DOI: 10.1101/2023.10.04.23296545\nSARS-CoV-2 Omicron XBB subvariants efficiently evade immunity from prior infection or vaccination, requiring vaccine adaptation. Here, we analyzed immunogenicity of an adapted vaccine, BNT162b2 Omicron XBB.1.5, which is currently used for booster vaccination. Booster vaccination significantly increased anti-Spike IgG, accompanied by expansion of cross-reactive memory B cells recognizing Wuhan and Omicron XBB.1.5 spike variants. Geometric mean neutralizing titers against XBB.1.5, XBB.1.16 and XBB.2.3, as well as cross-reactive responses against EG.5.1 and BA.2.86 increased significantly relative to pre-booster titers. Finally, the number of Wuhan and XBB.1.5 spike reactive IFN-{gamma}-producing T cells significantly increased after booster vaccination. In summary, BNT162b2 Omicron XBB.1.5 vaccination resulted in potent neutralizing antibody responses against Omicron XBB variants, including the recent Omicron variants EG.5.1 (Eris) and BA.2.86 (Pirola), as well as XBB.1.5 reactive T cell responses, suggesting that booster vaccination will augment protection against these emerging variants.\nThe more symptoms the better? Covid-19 vaccine side effects and long-term neutralizing antibody response\nAuthors: Dutcher, E. G.; Epel, E. S.; Mason, A. E.; Hecht, F. M.; Robinson, J. E.; Drury, S. S.; Prather, A. A.\nScore: 64.6, Published: 2023-10-06 DOI: 10.1101/2023.09.26.23296186\nProtection against SARS-CoV-2 wanes over time, and booster uptake has been low. This study explores the link between post-vaccination symptoms, biometric changes, and neutralizing antibodies (nAB) after mRNA vaccination. Data were collected from adults (n = 363) who received two doses of either BNT162b2 or mRNA-1273, with serum nAB concentration measured at 1 and 6 months post-vaccination. Daily symptom surveys were completed for six days starting on the day of each dose. Concurrently, objective biometric measurements, including skin temperature, heart rate, heart rate variability, and respiratory rate, were collected. We found that certain symptoms (chills, tiredness, feeling unwell, and headache) after the second dose were associated with increases in nAB at 1 and 6 months post-vaccination, to roughly 140-160% the level of individuals without each symptom. Each additional symptom predicted a 1.1-fold nAB increase. Greater changes in skin temperature and heart rate after the second dose predicted higher nAB levels. Skin temperature had a stronger predictive relationship for 6-month than 1-month nAB level. In the context of low ongoing vaccine uptake, our findings suggest that public health messaging could seek to reframe systemic symptoms after vaccination as desirable.\nSome principles for using epidemiologic study results to parameterize transmission models\nAuthors: Joshi, K.; Kahn, R.; Boyer, C.; Lipsitch, M.\nScore: 46.2, Published: 2023-10-03 DOI: 10.1101/2023.10.03.23296455\nBackgroundInfectious disease models, including individual based models (IBMs), can be used to inform public health response. For these models to be effective, accurate estimates of key parameters describing the natural history of infection and disease are needed. However, obtaining these parameter estimates from epidemiological studies is not always straightforward. We aim to 1) outline challenges to parameter estimation that arise due to common biases found in epidemiologic studies and 2) describe the conditions under which careful consideration in the design and analysis of the study could allow us to obtain a causal estimate of the parameter of interest. In this discussion we do not focus on issues of generalizability and transportability. MethodsUsing examples from the COVID-19 pandemic, we first identify different ways of parameterizing IBMs and describe ideal study designs to estimate these parameters. Given real-world limitations, we describe challenges in parameter estimation due to confounding and conditioning on a post-exposure observation. We then describe ideal study designs that can lead to unbiased parameter estimates. We finally discuss additional challenges in estimating progression probabilities and the consequences of these challenges. ResultsCausal estimation can only occur if we are able to accurately measure and control for all confounding variables that create non-causal associations between the exposure and outcome of interest, which is sometimes challenging given the nature of the variables we need to measure. In the absence of perfect control, non-causal parameter estimates should still be used, as sometimes they are the best available information we have. ConclusionsIdentifying which estimates from epidemiologic studies correspond to the quantities needed to parameterize disease models, and determining whether these parameters have causal interpretations, can inform future study designs and improve inferences from infectious disease models. Understanding the way in which biases can arise in parameter estimation can inform sensitivity analyses or help with interpretation of results if the magnitude and direction of the bias is understood.\nHuman origin ascertained for SARS-CoV-2 Omicron-like spike sequences detected in wastewater: a targeted surveillance study of a cryptic lineage in an urban sewershed\nAuthors: Shafer, M. M.; Bobholz, M. J.; Vuyk, W. C.; Gregory, D. A.; Roguet, A.; Haddock Soto, L. A.; Rushford, C.; Janssen, K. H.; Emmen, I. E.; Ries, H. J.; Pilch, H. E.; Mullen, P. A.; Fahney, R. B.; Wei, W.; Lambert, M.; Wenzel, J.; Halfmann, P.; Kawaoka, Y.; Wilson, N. A.; Friedrich, T. C.; Pray, I. W.; Westergaard, R.; O'Connor, D. H.; Johnson, M.\nScore: 1143.8, Published: 2023-09-22 DOI: 10.1101/2022.10.28.22281553\nBackgroundThe origin of novel SARS-CoV-2 spike sequences found in wastewater, without corresponding detection in clinical specimens, remains unclear. We sought to determine the origin of one such \"cryptic\" wastewater lineage by tracking and characterizing its persistence and genomic evolution over time. MethodsWe first detected a cryptic lineage in Wisconsin municipal wastewater in January 2022. By systematically sampling wastewater from targeted sub-sewershed lines and maintenance holes using compositing autosamplers, we traced this lineage (labeled WI-CL-001) to its source at a single commercial building. There we detected WI-CL-001 at concentrations as high as 2.7 x 109 genome copies per liter (gc/L) via RT-dPCR. In addition to using metagenomic 12s rRNA sequencing to determine the viruss host species, we also sequenced SARS-CoV-2 spike receptor binding domains (RBDs), and where possible, whole viral genomes to identify and characterize the evolution of this lineage over the 13 consecutive months that it was detectable. FindingsThe vast majority of 12s rRNAs sequenced from wastewater leaving the identified source building were human. Additionally, we generated over 100 viral RBD and whole genome sequences from wastewater samples containing the cryptic lineage collected between January 2022 and January 2023. These sequences contained a combination of fixed nucleotide substitutions characteristic of Pango lineage B.1.234, which circulated in humans in Wisconsin at low levels from October 2020 to February 2021. Despite this, mutations in the spike gene, and elsewhere, resembled those subsequently found in Omicron variants. InterpretationWe propose that prolonged detection of WI-CL-001 in wastewater represents persistent shedding of SARS-CoV-2 from a single human initially infected by an ancestral B.1.234 virus. The accumulation of convergent \"Omicron-like\" mutations in WI-CL-001s ancestral B.1.234 genome likely reflects persistent infection and extensive within-host evolution. FundingThe Rockefeller Foundation, Wisconsin Department of Health Services, Centers for Disease Control and Prevention (CDC), National Institute on Drug Abuse (NIDA), and the Center for Research on Influenza Pathogenesis and Transmission. Research in contextO_ST_ABSEvidence before this studyC_ST_ABSTo identify other studies that characterized unusual wastewater-specific SARS-CoV-2 lineages, we conducted a PubMed search using the keywords \"cryptic SARS-CoV-2 lineages\" or \"novel SARS-CoV-2 lineages\" in addition to \"wastewater\" on May 9, 2023. From the 18 papers retrieved, only two reported wastewater-specific cryptic lineages. These lineages were identified by members of our author team in wastewater from California, Missouri, and New York City. None of these could be definitively traced to a specific source. A third study in Nevada identified a unique recombinant variant (designated Pango lineage XL) in wastewater, which was also discovered in two clinical specimens from the same community. However, it was unclear whether the clinical specimens collected were from the same individual(s) responsible for the virus detected in the wastewater. To our knowledge, no prior study has successfully traced novel SARS-CoV-2 lineages detected in wastewater back to a specific location. How and where cryptic lineages are introduced into wastewater is not known. The added value of this studyThis study documents the presence and likely source of a novel and highly divergent cryptic SARS-CoV-2 lineage detected in Wisconsin wastewater for 13 months. In contrast to previously reported cryptic lineages, we successfully traced the lineage (WI-CL-001) to a single commercial building with approximately 30 employees. The exceptionally high viral RNA concentrations at the source building facilitated the tracing effort and allowed for the sequencing of WI-CL-001s whole genome, expanding our view of the lineages mutational landscape beyond the spike gene. Implications of all the available evidenceWI-CL-001s persistence in wastewater, its heavily mutated Omicron-like genotype, and its identified point source at a human-occupied commercial building all support the hypothesis that cryptic wastewater lineages can arise from persistently infected humans. Because cryptic wastewater lineages have some amino acid changes that subsequently emerge in circulating viruses, increased global monitoring of such lineages could help forecast variants that may arise in the future.\nVaccine Effectiveness Against Long COVID in Children: A Report from the RECOVER EHR Cohort\nAuthors: Razzaghi, H.; Forrest, C.; Hirabayshi, K.; Wu, Q.; Allen, A.; Rao, S.; Chen, Y.; Bunnell, T.; Chrischilles, E. A.; Cowell, L. G.; Cummins, M. R.; Hanauer, D. A.; Higginbotham, M. J.; Horne, B.; Horowitz, C. R.; Jhaveri, R.; Kim, S.; Mishkin, A.; Muszynski, J.; Naggie, S.; Pajor, N. M.; Paranjape, A.; Schwenk, H. T.; Sills, M. R.; Tedla, Y. G.; Williams, D. A.; Bailey, C.\nScore: 128.0, Published: 2023-09-28 DOI: 10.1101/2023.09.27.23296100\nObjectiveVaccination reduces the risk of acute COVID-19 in children, but it is less clear whether it protects against long COVID. We estimated vaccine effectiveness (VE) against long COVID in children aged 5-17 years. MethodsThis retrospective cohort study used data from 17 health systems in the RECOVER PCORnet electronic health record (EHR) Program for visits between vaccine availability, and October 29, 2022. Conditional logistic regression was used to estimate VE against long COVID with matching on age group (5-11, 12-17) and time period and adjustment for sex, ethnicity, health system, comorbidity burden, and pre-exposure health care utilization. We examined both probable (symptom-based) and diagnosed long COVID in the year following vaccination. ResultsThe vaccination rate was 56% in the cohort of 1,037,936 children. The incidence of probable long COVID was 4.5% among patients with COVID-19, while diagnosed long COVID was 0.7%. Adjusted vaccine effectiveness within 12 months was 35.4% (95 CI 24.5 - 44.5) against probable long COVID and 41.7% (15.0 - 60.0) against diagnosed long COVID. VE was higher for adolescents 50.3% [36.3 - 61.0]) than children aged 5-11 (23.8% [4.9 - 39.0]). VE was higher at 6 months (61.4% [51.0 - 69.6]) but decreased to 10.6% (-26.8 - 37.0%) at 18-months. DiscussionThis large retrospective study shows a moderate protective effect of SARS-CoV-2 vaccination against long COVID. The effect is stronger in adolescents, who have higher risk of long COVID, and wanes over time. Understanding VE mechanism against long COVID requires more study, including EHR sources and prospective data. Article SummaryVaccination against COVID-19 has a protective effect against long COVID in children and adolescents. The effect wanes over time but remains significant at 12 months. Whats Known on This SubjectVaccines reduce the risk and severity of COVID-19 in children. There is evidence for reduced long COVID risk in adults who are vaccinated, but little information about similar effects for children and adolescents, who have distinct forms of long COVID. What This Study AddsUsing electronic health records from US health systems, we examined large cohorts of vaccinated and unvaccinated patients \u003c18 years old and show that vaccination against COVID-19 is associated with reduced risk of long COVID for at least 12 months. Contributors StatementDrs. Hanieh Razzaghi and Charles Bailey conceptualized and designed the study, supervised analyses, drafted the initial manuscript, and critically reviewed and revised the manuscript. Drs. Christopher Forrest and Yong Chen designed the study and critically reviewed and revised the manuscript. Ms. Kathryn Hirabayashi, Ms. Andrea Allen, and Dr. Qiong Wu conducted analyses, and critically reviewed and revised the manuscript. Drs. Suchitra Rao, H Timothy Bunnell, Elizabeth A. Chrischilles, Lindsay G. Cowell, Mollie R. Cummins, David A. Hanauer, Benjamin D. Horne, Carol R. Horowitz, Ravi Jhaveri, Susan Kim, Aaron Mishkin, Jennifer A. Muszynski, Susanna Nagie, Nathan M. Pajor, Anuradha Paranjape, Hayden T. Schwenk, Marion R. Sills, Yacob G. Tedla, David A. Williams, and Ms. Miranda Higginbotham critically reviewed and revised the manuscript. All authors approved the final manuscript as submitted and agree to be accountable for all aspects of the work. Authorship statementAuthorship has been determined according to ICMJE recommendations.\nSeroincidence of SARS-CoV-2 infection prior to and during the rollout of vaccines in a community-based prospective cohort of U.S. adults\nAuthors: Nash, D.; Srivastava, A.; Shen, Y.; Penrose, K.; Kulkarni, S.; Zimba, R.; You, W.; Berry, A.; Mirzayi, C.; Maroko, A. R.; Parcesepe, A.; Grov, C.; Robertson, M.\nScore: 16.0, Published: 2023-10-02 DOI: 10.1101/2023.09.29.23296142\nLONG ABSTRACTO_ST_ABSBackgroundC_ST_ABSInfectious disease surveillance systems, which largely rely on diagnosed cases, underestimate the true incidence of SARS-CoV-2 infection, due to under-ascertainment and underreporting. We used repeat serologic testing to measure N-protein seroconversion in a well-characterized cohort of U.S. adults with no serologic evidence of SARS-CoV-2 infection to estimate the incidence of SARS-CoV-2 infection and characterize risk factors, with comparisons before and after the start of the SARS-CoV-2 vaccine and variant eras. MethodsWe assessed the incidence rate of infection and risk factors in two sub-groups (cohorts) that were SARS-CoV-2 N-protein seronegative at the start of each follow-up period: 1) the pre-vaccine/wild-type era cohort (n=3,421), followed from April to November 2020; and 2) the vaccine/variant era cohort (n=2,735), followed from November 2020 to June 2022. Both cohorts underwent repeat serologic testing with an assay for antibodies to the SARS-CoV-2 N protein (Bio-Rad Platelia SARS-CoV-2 total Ab). We estimated crude incidence and sociodemographic/epidemiologic risk factors in both cohorts. We used multivariate Poisson models to compare the risk of SARS-CoV-2 infection in the pre-vaccine/wild-type era cohort (referent group) to that in the vaccine/variant era cohort, within strata of vaccination status and epidemiologic risk factors (essential worker status, child in the household, case in the household, social distancing). FindingsIn the pre-vaccine/wild-type era cohort, only 18 of the 3,421 participants (0.53%) had \u003e1 vaccine dose by the end of follow-up, compared with 2,497/2,735 (91.3%) in the vaccine/variant era cohort. We observed 323 and 815 seroconversions in the pre-vaccine/wild-type era and the vaccine/variant era and cohorts, respectively, with corresponding incidence rates of 9.6 (95% CI: 8.3-11.5) and 25.7 (95% CI: 24.2-27.3) per 100 person-years. Associations of sociodemographic and epidemiologic risk factors with SARS-CoV-2 incidence were largely similar in the pre-vaccine/wild-type and vaccine/variant era cohorts. However, some new epidemiologic risk factors emerged in the vaccine/variant era cohort, including having a child in the household, and never wearing a mask while using public transit. Adjusted incidence rate ratios (aIRR), with the entire pre-vaccine/wild-type era cohort as the referent group, showed markedly higher incidence in the vaccine/variant era cohort, but with more vaccine doses associated with lower incidence: aIRRun/undervaccinated=5.3 (95% CI: 4.2-6.7); aIRRprimary series only=5.1 (95% CI: 4.2-7.3); aIRRboosted once=2.5 (95% CI: 2.1-3.0), and aIRRboosted twice=1.65 (95% CI: 1.3-2.1). These associations were essentially unchanged in risk factor-stratified models. InterpretationIn SARS-CoV-2 N protein seronegative individuals, large increases in incidence and newly emerging epidemiologic risk factors in the vaccine/variant era likely resulted from multiple co-occurring factors, including policy changes, behavior changes, surges in transmission, and changes in SARS-CoV-2 variant properties. While SARS-CoV-2 incidence increased markedly in most groups in the vaccine/variant era, being up to date on vaccines and the use of non-pharmaceutical interventions (NPIs), such as masking and social distancing, remained reliable strategies to mitigate the risk of SARS-CoV-2 infection, even through major surges due to immune evasive variants. Repeat serologic testing in cohort studies is a useful and complementary strategy to characterize SARS-CoV-2 incidence and risk factors. SHORT ABSTRACTThis study used repeat serologic testing to estimate infection rates and risk factors in two overlapping cohorts of SARS-CoV-2 N protein seronegative U.S. adults. One mostly unvaccinated sub-cohort was tracked from April to November 2020 (pre-vaccine/wild-type era, n=3,421), and the other, mostly vaccinated cohort, from November 2020 to June 2022 (vaccine/variant era, n=2,735). Vaccine uptake was from 0.53% and 91.3% in the pre-vaccine and vaccine/variant cohorts, respectively. Corresponding seroconversion rates were 9.6 and 25.7 per 100 person-years. In both cohorts, sociodemographic and epidemiologic risk factors for infection were similar, though new risks emerged in the vaccine/variant era, such as having a child in the household. Despite higher incidence rates in the vaccine/variant cohort, vaccine boosters, masking, and distancing likely reduced infection risk, even through major variant surges. Repeat serologic testing in cohorts is a useful and complementary strategy to characterize incidence and risk factors. FundingThe work was supported by the CUNY Institute for Implementation Science in Population Health, the U.S. National Institutes of Allergy and Infectious Diseases (NIAID), Pfizer, Inc., and the U.S. National Institute of Mental Health (NIMH).\nRisk of Coronavirus Disease 2019 (COVID-19) among Those Up-to-Date and Not Up-to-Date on COVID-19 Vaccination by US CDC Criteria\nAuthors: Shrestha, N. K.; Burke, P. C.; Nowacki, A. S.; Gordon, S. M.\nScore: 5132.1, Published: 2023-09-14 DOI: 10.1101/2023.06.09.23290893\nBackgroundThe CDC recently defined being \"up-to-date\" on COVID-19 vaccination as having received at least one dose of a COVID-19 bivalent vaccine. The purpose of this study was to compare the risk of COVID-19 among those \"up-to-date\" and \"not up-to-date\". MethodsEmployees of Cleveland Clinic in Ohio, USA, in employment when the COVID-19 bivalent vaccine first became available, and still employed when the XBB lineages became dominant, were included. Cumulative incidence of COVID-19 since the XBB lineages became dominant was compared across the \"up-to-date\" and \"not up-to-date\" states, by treating COVID-19 bivalent vaccination as a time- dependent covariate whose value changed on receipt of the vaccine. Risk of COVID-19 by vaccination status was also evaluated using multivariable Cox proportional hazards regression adjusting for propensity to get tested for COVID-19, age, sex, most recent prior SARS-CoV-2 infection, and number of prior vaccine doses. ResultsCOVID-19 occurred in 1475 (3%) of 48 344 employees during the 100-day study period. The cumulative incidence of COVID-19 was lower in the \"not up-to-date\" than the \"up-to-date\" state. On multivariable analysis, being \"up-to-date\" was not associated with lower risk of COVID-19 (HR, 1.05; 95% C.I., 0.88-1.25; P-value, 0.58). Results were very similar when those 65 years and older were only considered \"up-to-date\" after 2 doses of the bivalent vaccine. ConclusionsSince the XBB lineages became dominant, adults \"up-to-date\" on COVID-19 vaccination by the CDC definition do not have a lower risk of COVID-19 than those \"not up-to-date\", bringing into question the value of this risk classification definition. SummaryAmong 48 344 working-aged Cleveland Clinic employees, those \"up-to-date\" on COVID-19 vaccination did not have a lower risk of COVID-19 than those not \"up-to-date\". The current CDC definition provides a meaningless classification of risk of COVID-19 in the adult population.\nHigh prevalence and risk of malaria among asymptomatic individuals from villages with high rates of artemisinin partial resistance in Kyerwa district, North-western Tanzania.\nAuthors: Mandai, S. S.; Francis, F.; Challe, D. P.; Seth, M. D.; Madebe, R. A.; Petro, D. A.; Budodo, R.; Kisambale, A. J.; Chacha, G. A.; Moshi, R.; Mbwambo, R. B.; Pereus, D.; Bakari, C.; Aaron, S.; Mbwambo, D.; Lusasi, A.; Kajange, S.; Lazaro, S.; Kapologwe, N.; Mandara, C. I.; Ishengoma, D. S.\nScore: 6.2, Published: 2023-10-05 DOI: 10.1101/2023.10.05.23296564\nAbstract Background: Tanzania adopted and has been implementing the World Health Organization (WHO) recommended interventions to control and eventually eliminate malaria. However, malaria is still a leading public health problem and the country experiences heterogeneous transmission; but the drivers of these patterns are not clearly known. This study assessed the prevalence and risk of malaria infections among asymptomatic individuals living in a hyperendemic area which has high prevalence of artemisinin partial resistant parasites in Kyerwa District of Kagera region, North-western Tanzania. Methods This was a community-based cross-sectional survey that recruited participants from five villages of Kyerwa district in Kagera region. Demographic, anthropometric, clinical, parasitological, types of houses inhabited and socio-economic status (SES) data were collected using electronic capture tools running on Open Data Kit (ODK). Risk factors associated with malaria infections were determined by univariate and multivariate logistic regression and the results were presented as crude (cOR) and adjusted Odds Ratio (aOR), with 95% confidence intervals (CI). Results: A total of 4,454 individuals were tested using malaria rapid diagnostic tests (RDTs) and 1,979 (44.4%) had a positive test. The prevalence of malaria varied from 14.4% to 68.5% with significant differences among the villages (p\u003c0.001). The prevalence and risk of malaria infections were significantly higher in males (aOR =1.25, 95% CI: 1.06-1.48, p=0.04), school children ((aged 5-10 years, aOR =4.09, 95% CI: 3.39-5.10, p\u003c0.001) and (10-15 years, aOR=4.40, 95% CI: 3.46-5.59, p\u003c0.001)) and among individuals who were not using bed nets (aOR =1.29, 95% CI: 1.10-1.42, p=0.002). Other risks of malaria infections included lower SES (aOR=1.27, 95% CI:1.04-1.55, p\u003c0.001) and living in houses with open windows (aOR=1.26, 95% CI: 1.03-1.54, p=0.024). Conclusion: This study showed high prevalence of malaria infections and high heterogeneity at micro-geographic levels. The risk of malaria infections was higher in school children, males, individuals who did not use bed nets, and among participants with low SES or living in poorly constructed houses. These findings provide important baseline data in an area with a high prevalence of artemisinin partial-resistant parasites and will be utilized in future studies to monitor the trends and potential spread of such parasites. Keywords: Malaria, asymptomatic infections, risk factors, partial artemisinin resistance, Tanzania.\nBioinformatic Analysis of Defective Viral Genomes in SARS-CoV-2 and Its Impact on Population Infection Characteristics\nAuthors: Xu, Z.; Peng, Q.; Song, J.; Zhang, H.; Wei, D.; Demongeot, J.\nScore: 3.8, Published: 2023-10-05 DOI: 10.1101/2023.10.05.23296580\nDVGs (Defective Viral Genomes) and SIP (Semi-Infectious Particle) are commonly present in RNA virus infections. In this study, we analyzed high-throughput sequencing data and found that DVGs or SIPs are also widely present in SARS-CoV-2. Comparison of SARS-CoV-2 with various DNA viruses revealed that the SARS-CoV-2 genome is more susceptible to damage and has greater sequencing sample heterogeneity. Variability analysis at the whole-genome sequencing depth showed a higher coefficient of variation for SARS-CoV-2, and DVG analysis indicated a high proportion of splicing sites, suggesting significant genome heterogeneity and implying that most virus particles assembled are enveloped with incomplete RNA sequences. We further analyzed the characteristics of different strains in terms of sequencing depth and DVG content differences and found that as the virus evolves, the proportion of intact genomes in virus particles increases, which can be significantly reflected in third-generation sequencing data, while the proportion of DVG gradually decreases. Specifically, the proportion of intact genome of Omicron was greater than that of Delta and Alpha strains. This can well explain why Omicron strain is more infectious than Delta and Alpha strains. We also speculate that this improvement in completeness is due to the enhancement of virus assembly ability, as the Omicron strain can quickly realize the binding of RNA and capsid protein, thereby shortening the exposure time of exposed virus RNA in the host environment and greatly reducing its degradation level. Finally, by using mathematical modeling, we simulated how DVG effects under different environmental factors affect the infection characteristics and evolution of the population. We can explain well why the severity of symptoms is closely related to the amount of virus invasion and why the same strain causes huge differences in population infection characteristics under different environmental conditions. Our study provides a new approach for future virus research and vaccine development.\nLiver abnormalities following SARS-CoV-2 infection in children under 10 years of age\nAuthors: Terebuh, P.; Olaker, V. R.; Kendall, E. K.; Kaelber, D. C.; Xu, R.; Davis, P. B.\nScore: 60.9, Published: 2023-09-23 DOI: 10.1101/2023.09.21.23295905\nObjectiveBeginning in October 2021 in the US and elsewhere, cases of severe pediatric hepatitis of unknown etiology were identified in young children. While the adenovirus and adenovirus-associated virus have emerged as leading etiologic suspects, we attempted to investigate a potential role for SARS-CoV-2 in the development of subsequent liver abnormalities. DesignWe conducted a study utilizing retrospective cohorts of de-identified, aggregated data from the electronic health records of over 100 million patients contributed by US health care organizations. ResultsCompared to propensity-score-matched children with other respiratory infections, children aged 1-10 years with COVID-19 had a higher risk of elevated transaminases (Hazard ratio (HR) (95% Confidence interval (CI)) 2.16 (1.74-2.69)) or total bilirubin (HR (CI) 3.02 (1.91-4.78)), or new diagnoses of liver diseases (HR (CI) 1.67 (1.21-2.30)) from one to six months after infection. Patients with pre-existing liver abnormalities, liver abnormalities surrounding acute infection, younger age (1-4 years), or illness requiring hospitalization all had similarly elevated risk. Children who developed liver abnormalities following COVID-19 had more pre-existing conditions than those who developed abnormalities following other infections. ConclusionThese results indicate that SARS-CoV-2 may prime the patient for subsequent development of liver infections or non-infectious liver diseases. While rare ([~]1 in 1,000), SARS-CoV-2 is a risk for subsequent abnormalities in liver function or the diagnosis of diseases of the liver. What is already known on this topicClusters of severe hepatitis in children in 2022 coincident with the increase in COVID-19 infections in children raised the question of the contribution of SARS-CoV-2 to the hepatitis outbreak, though it was soon determined that SARS-CoV-2 was not the primary etiologic agent. What this study addsSARS-CoV-2 may prime the patient for subsequent development of liver infections or non-infectious liver diseases. How this study might affect research, practice or policyDespite the mild initial disease in children, there may be longer term consequences of COVID-19, such as liver abnormalities, that warrants further investigation.\n",
  "wordCount" : "4252",
  "inLanguage": "en",
  "datePublished": "2023-10-08T10:37:16Z",
  "dateModified": "2023-10-08T10:37:16Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta"><span>updated on October 8, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.23296545">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.23296545" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.23296545">
        <p class="paperTitle">Humoral and cellular immune responses following BNT162b2 XBB.1.5 vaccination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.23296545" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.23296545" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stankov, M. V.; Hoffmann, M.; Gutierrez, R.; Cossmann, A.; Morillas Ramos, G.; Graalmann, T.; Friedrichsen, M.; Ravens, I.; Tamara, I.; Ristenpart, J.; Schimrock, A.; Willenzon, S.; Ahrenstorff, G.; Witte, T.; Foerster, R.; Kempf, A.; Poehlmann, S.; Hammerschmidt, S. I.; Dopfer-Jablonka, A.; Behrens, G. M.</p>
        <p class="info">Score: 78.2, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.23296545' target='https://doi.org/10.1101/2023.10.04.23296545'> 10.1101/2023.10.04.23296545</a></p>
        <p class="abstract">SARS-CoV-2 Omicron XBB subvariants efficiently evade immunity from prior infection or vaccination, requiring vaccine adaptation. Here, we analyzed immunogenicity of an adapted vaccine, BNT162b2 Omicron XBB.1.5, which is currently used for booster vaccination. Booster vaccination significantly increased anti-Spike IgG, accompanied by expansion of cross-reactive memory B cells recognizing Wuhan and Omicron XBB.1.5 spike variants. Geometric mean neutralizing titers against XBB.1.5, XBB.1.16 and XBB.2.3, as well as cross-reactive responses against EG.5.1 and BA.2.86 increased significantly relative to pre-booster titers. Finally, the number of Wuhan and XBB.1.5 spike reactive IFN-{gamma}-producing T cells significantly increased after booster vaccination. In summary, BNT162b2 Omicron XBB.1.5 vaccination resulted in potent neutralizing antibody responses against Omicron XBB variants, including the recent Omicron variants EG.5.1 (Eris) and BA.2.86 (Pirola), as well as XBB.1.5 reactive T cell responses, suggesting that booster vaccination will augment protection against these emerging variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.23296186">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.23296186" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.23296186">
        <p class="paperTitle">The more symptoms the better? Covid-19 vaccine side effects and long-term neutralizing antibody response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.23296186" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.23296186" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dutcher, E. G.; Epel, E. S.; Mason, A. E.; Hecht, F. M.; Robinson, J. E.; Drury, S. S.; Prather, A. A.</p>
        <p class="info">Score: 64.6, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.23296186' target='https://doi.org/10.1101/2023.09.26.23296186'> 10.1101/2023.09.26.23296186</a></p>
        <p class="abstract">Protection against SARS-CoV-2 wanes over time, and booster uptake has been low. This study explores the link between post-vaccination symptoms, biometric changes, and neutralizing antibodies (nAB) after mRNA vaccination. Data were collected from adults (n = 363) who received two doses of either BNT162b2 or mRNA-1273, with serum nAB concentration measured at 1 and 6 months post-vaccination. Daily symptom surveys were completed for six days starting on the day of each dose. Concurrently, objective biometric measurements, including skin temperature, heart rate, heart rate variability, and respiratory rate, were collected. We found that certain symptoms (chills, tiredness, feeling unwell, and headache) after the second dose were associated with increases in nAB at 1 and 6 months post-vaccination, to roughly 140-160% the level of individuals without each symptom. Each additional symptom predicted a 1.1-fold nAB increase. Greater changes in skin temperature and heart rate after the second dose predicted higher nAB levels. Skin temperature had a stronger predictive relationship for 6-month than 1-month nAB level. In the context of low ongoing vaccine uptake, our findings suggest that public health messaging could seek to reframe systemic symptoms after vaccination as desirable.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.23296455">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.23296455" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.23296455">
        <p class="paperTitle">Some principles for using epidemiologic study results to parameterize transmission models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.23296455" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.23296455" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Joshi, K.; Kahn, R.; Boyer, C.; Lipsitch, M.</p>
        <p class="info">Score: 46.2, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.23296455' target='https://doi.org/10.1101/2023.10.03.23296455'> 10.1101/2023.10.03.23296455</a></p>
        <p class="abstract">BackgroundInfectious disease models, including individual based models (IBMs), can be used to inform public health response. For these models to be effective, accurate estimates of key parameters describing the natural history of infection and disease are needed. However, obtaining these parameter estimates from epidemiological studies is not always straightforward. We aim to 1) outline challenges to parameter estimation that arise due to common biases found in epidemiologic studies and 2) describe the conditions under which careful consideration in the design and analysis of the study could allow us to obtain a causal estimate of the parameter of interest. In this discussion we do not focus on issues of generalizability and transportability.

MethodsUsing examples from the COVID-19 pandemic, we first identify different ways of parameterizing IBMs and describe ideal study designs to estimate these parameters. Given real-world limitations, we describe challenges in parameter estimation due to confounding and conditioning on a post-exposure observation. We then describe ideal study designs that can lead to unbiased parameter estimates. We finally discuss additional challenges in estimating progression probabilities and the consequences of these challenges.

ResultsCausal estimation can only occur if we are able to accurately measure and control for all confounding variables that create non-causal associations between the exposure and outcome of interest, which is sometimes challenging given the nature of the variables we need to measure. In the absence of perfect control, non-causal parameter estimates should still be used, as sometimes they are the best available information we have.

ConclusionsIdentifying which estimates from epidemiologic studies correspond to the quantities needed to parameterize disease models, and determining whether these parameters have causal interpretations, can inform future study designs and improve inferences from infectious disease models. Understanding the way in which biases can arise in parameter estimation can inform sensitivity analyses or help with interpretation of results if the magnitude and direction of the bias is understood.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.10.28.22281553">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.10.28.22281553" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.10.28.22281553">
        <p class="paperTitle">Human origin ascertained for SARS-CoV-2 Omicron-like spike sequences detected in wastewater: a targeted surveillance study of a cryptic lineage in an urban sewershed</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.10.28.22281553" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.10.28.22281553" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shafer, M. M.; Bobholz, M. J.; Vuyk, W. C.; Gregory, D. A.; Roguet, A.; Haddock Soto, L. A.; Rushford, C.; Janssen, K. H.; Emmen, I. E.; Ries, H. J.; Pilch, H. E.; Mullen, P. A.; Fahney, R. B.; Wei, W.; Lambert, M.; Wenzel, J.; Halfmann, P.; Kawaoka, Y.; Wilson, N. A.; Friedrich, T. C.; Pray, I. W.; Westergaard, R.; O&#39;Connor, D. H.; Johnson, M.</p>
        <p class="info">Score: 1143.8, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.10.28.22281553' target='https://doi.org/10.1101/2022.10.28.22281553'> 10.1101/2022.10.28.22281553</a></p>
        <p class="abstract">BackgroundThe origin of novel SARS-CoV-2 spike sequences found in wastewater, without corresponding detection in clinical specimens, remains unclear. We sought to determine the origin of one such &#34;cryptic&#34; wastewater lineage by tracking and characterizing its persistence and genomic evolution over time.

MethodsWe first detected a cryptic lineage in Wisconsin municipal wastewater in January 2022. By systematically sampling wastewater from targeted sub-sewershed lines and maintenance holes using compositing autosamplers, we traced this lineage (labeled WI-CL-001) to its source at a single commercial building. There we detected WI-CL-001 at concentrations as high as 2.7 x 109 genome copies per liter (gc/L) via RT-dPCR. In addition to using metagenomic 12s rRNA sequencing to determine the viruss host species, we also sequenced SARS-CoV-2 spike receptor binding domains (RBDs), and where possible, whole viral genomes to identify and characterize the evolution of this lineage over the 13 consecutive months that it was detectable.

FindingsThe vast majority of 12s rRNAs sequenced from wastewater leaving the identified source building were human. Additionally, we generated over 100 viral RBD and whole genome sequences from wastewater samples containing the cryptic lineage collected between January 2022 and January 2023. These sequences contained a combination of fixed nucleotide substitutions characteristic of Pango lineage B.1.234, which circulated in humans in Wisconsin at low levels from October 2020 to February 2021. Despite this, mutations in the spike gene, and elsewhere, resembled those subsequently found in Omicron variants.

InterpretationWe propose that prolonged detection of WI-CL-001 in wastewater represents persistent shedding of SARS-CoV-2 from a single human initially infected by an ancestral B.1.234 virus. The accumulation of convergent &#34;Omicron-like&#34; mutations in WI-CL-001s ancestral B.1.234 genome likely reflects persistent infection and extensive within-host evolution.

FundingThe Rockefeller Foundation, Wisconsin Department of Health Services, Centers for Disease Control and Prevention (CDC), National Institute on Drug Abuse (NIDA), and the Center for Research on Influenza Pathogenesis and Transmission.

Research in contextO_ST_ABSEvidence before this studyC_ST_ABSTo identify other studies that characterized unusual wastewater-specific SARS-CoV-2 lineages, we conducted a PubMed search using the keywords &#34;cryptic SARS-CoV-2 lineages&#34; or &#34;novel SARS-CoV-2 lineages&#34; in addition to &#34;wastewater&#34; on May 9, 2023. From the 18 papers retrieved, only two reported wastewater-specific cryptic lineages. These lineages were identified by members of our author team in wastewater from California, Missouri, and New York City. None of these could be definitively traced to a specific source.

A third study in Nevada identified a unique recombinant variant (designated Pango lineage XL) in wastewater, which was also discovered in two clinical specimens from the same community. However, it was unclear whether the clinical specimens collected were from the same individual(s) responsible for the virus detected in the wastewater. To our knowledge, no prior study has successfully traced novel SARS-CoV-2 lineages detected in wastewater back to a specific location. How and where cryptic lineages are introduced into wastewater is not known.

The added value of this studyThis study documents the presence and likely source of a novel and highly divergent cryptic SARS-CoV-2 lineage detected in Wisconsin wastewater for 13 months. In contrast to previously reported cryptic lineages, we successfully traced the lineage (WI-CL-001) to a single commercial building with approximately 30 employees. The exceptionally high viral RNA concentrations at the source building facilitated the tracing effort and allowed for the sequencing of WI-CL-001s whole genome, expanding our view of the lineages mutational landscape beyond the spike gene.

Implications of all the available evidenceWI-CL-001s persistence in wastewater, its heavily mutated Omicron-like genotype, and its identified point source at a human-occupied commercial building all support the hypothesis that cryptic wastewater lineages can arise from persistently infected humans. Because cryptic wastewater lineages have some amino acid changes that subsequently emerge in circulating viruses, increased global monitoring of such lineages could help forecast variants that may arise in the future.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.23296100">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.23296100" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.23296100">
        <p class="paperTitle">Vaccine Effectiveness Against Long COVID in Children: A Report from the RECOVER EHR Cohort</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.23296100" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.23296100" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Razzaghi, H.; Forrest, C.; Hirabayshi, K.; Wu, Q.; Allen, A.; Rao, S.; Chen, Y.; Bunnell, T.; Chrischilles, E. A.; Cowell, L. G.; Cummins, M. R.; Hanauer, D. A.; Higginbotham, M. J.; Horne, B.; Horowitz, C. R.; Jhaveri, R.; Kim, S.; Mishkin, A.; Muszynski, J.; Naggie, S.; Pajor, N. M.; Paranjape, A.; Schwenk, H. T.; Sills, M. R.; Tedla, Y. G.; Williams, D. A.; Bailey, C.</p>
        <p class="info">Score: 128.0, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.23296100' target='https://doi.org/10.1101/2023.09.27.23296100'> 10.1101/2023.09.27.23296100</a></p>
        <p class="abstract">ObjectiveVaccination reduces the risk of acute COVID-19 in children, but it is less clear whether it protects against long COVID. We estimated vaccine effectiveness (VE) against long COVID in children aged 5-17 years.

MethodsThis retrospective cohort study used data from 17 health systems in the RECOVER PCORnet electronic health record (EHR) Program for visits between vaccine availability, and October 29, 2022. Conditional logistic regression was used to estimate VE against long COVID with matching on age group (5-11, 12-17) and time period and adjustment for sex, ethnicity, health system, comorbidity burden, and pre-exposure health care utilization. We examined both probable (symptom-based) and diagnosed long COVID in the year following vaccination.

ResultsThe vaccination rate was 56% in the cohort of 1,037,936 children. The incidence of probable long COVID was 4.5% among patients with COVID-19, while diagnosed long COVID was 0.7%. Adjusted vaccine effectiveness within 12 months was 35.4% (95 CI 24.5 - 44.5) against probable long COVID and 41.7% (15.0 - 60.0) against diagnosed long COVID. VE was higher for adolescents 50.3% [36.3 - 61.0]) than children aged 5-11 (23.8% [4.9 - 39.0]). VE was higher at 6 months (61.4% [51.0 - 69.6]) but decreased to 10.6% (-26.8 - 37.0%) at 18-months.

DiscussionThis large retrospective study shows a moderate protective effect of SARS-CoV-2 vaccination against long COVID. The effect is stronger in adolescents, who have higher risk of long COVID, and wanes over time. Understanding VE mechanism against long COVID requires more study, including EHR sources and prospective data.

Article SummaryVaccination against COVID-19 has a protective effect against long COVID in children and adolescents. The effect wanes over time but remains significant at 12 months.

Whats Known on This SubjectVaccines reduce the risk and severity of COVID-19 in children. There is evidence for reduced long COVID risk in adults who are vaccinated, but little information about similar effects for children and adolescents, who have distinct forms of long COVID.

What This Study AddsUsing electronic health records from US health systems, we examined large cohorts of vaccinated and unvaccinated patients &lt;18 years old and show that vaccination against COVID-19 is associated with reduced risk of long COVID for at least 12 months.

Contributors StatementDrs. Hanieh Razzaghi and Charles Bailey conceptualized and designed the study, supervised analyses, drafted the initial manuscript, and critically reviewed and revised the manuscript.

Drs. Christopher Forrest and Yong Chen designed the study and critically reviewed and revised the manuscript.

Ms. Kathryn Hirabayashi, Ms. Andrea Allen, and Dr. Qiong Wu conducted analyses, and critically reviewed and revised the manuscript.

Drs. Suchitra Rao, H Timothy Bunnell, Elizabeth A. Chrischilles, Lindsay G. Cowell, Mollie R. Cummins, David A. Hanauer, Benjamin D. Horne, Carol R. Horowitz, Ravi Jhaveri, Susan Kim, Aaron Mishkin, Jennifer A. Muszynski, Susanna Nagie, Nathan M. Pajor, Anuradha Paranjape, Hayden T. Schwenk, Marion R. Sills, Yacob G. Tedla, David A. Williams, and Ms. Miranda Higginbotham critically reviewed and revised the manuscript.

All authors approved the final manuscript as submitted and agree to be accountable for all aspects of the work.

Authorship statementAuthorship has been determined according to ICMJE recommendations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.23296142">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.23296142" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.23296142">
        <p class="paperTitle">Seroincidence of SARS-CoV-2 infection prior to and during the rollout of vaccines in a community-based prospective cohort of U.S. adults</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.23296142" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.23296142" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nash, D.; Srivastava, A.; Shen, Y.; Penrose, K.; Kulkarni, S.; Zimba, R.; You, W.; Berry, A.; Mirzayi, C.; Maroko, A. R.; Parcesepe, A.; Grov, C.; Robertson, M.</p>
        <p class="info">Score: 16.0, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.23296142' target='https://doi.org/10.1101/2023.09.29.23296142'> 10.1101/2023.09.29.23296142</a></p>
        <p class="abstract">LONG ABSTRACTO_ST_ABSBackgroundC_ST_ABSInfectious disease surveillance systems, which largely rely on diagnosed cases, underestimate the true incidence of SARS-CoV-2 infection, due to under-ascertainment and underreporting. We used repeat serologic testing to measure N-protein seroconversion in a well-characterized cohort of U.S. adults with no serologic evidence of SARS-CoV-2 infection to estimate the incidence of SARS-CoV-2 infection and characterize risk factors, with comparisons before and after the start of the SARS-CoV-2 vaccine and variant eras.

MethodsWe assessed the incidence rate of infection and risk factors in two sub-groups (cohorts) that were SARS-CoV-2 N-protein seronegative at the start of each follow-up period: 1) the pre-vaccine/wild-type era cohort (n=3,421), followed from April to November 2020; and 2) the vaccine/variant era cohort (n=2,735), followed from November 2020 to June 2022. Both cohorts underwent repeat serologic testing with an assay for antibodies to the SARS-CoV-2 N protein (Bio-Rad Platelia SARS-CoV-2 total Ab). We estimated crude incidence and sociodemographic/epidemiologic risk factors in both cohorts. We used multivariate Poisson models to compare the risk of SARS-CoV-2 infection in the pre-vaccine/wild-type era cohort (referent group) to that in the vaccine/variant era cohort, within strata of vaccination status and epidemiologic risk factors (essential worker status, child in the household, case in the household, social distancing).

FindingsIn the pre-vaccine/wild-type era cohort, only 18 of the 3,421 participants (0.53%) had &gt;1 vaccine dose by the end of follow-up, compared with 2,497/2,735 (91.3%) in the vaccine/variant era cohort. We observed 323 and 815 seroconversions in the pre-vaccine/wild-type era and the vaccine/variant era and cohorts, respectively, with corresponding incidence rates of 9.6 (95% CI: 8.3-11.5) and 25.7 (95% CI: 24.2-27.3) per 100 person-years. Associations of sociodemographic and epidemiologic risk factors with SARS-CoV-2 incidence were largely similar in the pre-vaccine/wild-type and vaccine/variant era cohorts. However, some new epidemiologic risk factors emerged in the vaccine/variant era cohort, including having a child in the household, and never wearing a mask while using public transit. Adjusted incidence rate ratios (aIRR), with the entire pre-vaccine/wild-type era cohort as the referent group, showed markedly higher incidence in the vaccine/variant era cohort, but with more vaccine doses associated with lower incidence: aIRRun/undervaccinated=5.3 (95% CI: 4.2-6.7); aIRRprimary series only=5.1 (95% CI: 4.2-7.3); aIRRboosted once=2.5 (95% CI: 2.1-3.0), and aIRRboosted twice=1.65 (95% CI: 1.3-2.1). These associations were essentially unchanged in risk factor-stratified models.

InterpretationIn SARS-CoV-2 N protein seronegative individuals, large increases in incidence and newly emerging epidemiologic risk factors in the vaccine/variant era likely resulted from multiple co-occurring factors, including policy changes, behavior changes, surges in transmission, and changes in SARS-CoV-2 variant properties. While SARS-CoV-2 incidence increased markedly in most groups in the vaccine/variant era, being up to date on vaccines and the use of non-pharmaceutical interventions (NPIs), such as masking and social distancing, remained reliable strategies to mitigate the risk of SARS-CoV-2 infection, even through major surges due to immune evasive variants. Repeat serologic testing in cohort studies is a useful and complementary strategy to characterize SARS-CoV-2 incidence and risk factors.

SHORT ABSTRACTThis study used repeat serologic testing to estimate infection rates and risk factors in two overlapping cohorts of SARS-CoV-2 N protein seronegative U.S. adults. One mostly unvaccinated sub-cohort was tracked from April to November 2020 (pre-vaccine/wild-type era, n=3,421), and the other, mostly vaccinated cohort, from November 2020 to June 2022 (vaccine/variant era, n=2,735). Vaccine uptake was from 0.53% and 91.3% in the pre-vaccine and vaccine/variant cohorts, respectively. Corresponding seroconversion rates were 9.6 and 25.7 per 100 person-years. In both cohorts, sociodemographic and epidemiologic risk factors for infection were similar, though new risks emerged in the vaccine/variant era, such as having a child in the household. Despite higher incidence rates in the vaccine/variant cohort, vaccine boosters, masking, and distancing likely reduced infection risk, even through major variant surges. Repeat serologic testing in cohorts is a useful and complementary strategy to characterize incidence and risk factors.

FundingThe work was supported by the CUNY Institute for Implementation Science in Population Health, the U.S. National Institutes of Allergy and Infectious Diseases (NIAID), Pfizer, Inc., and the U.S. National Institute of Mental Health (NIMH).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.09.23290893">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.09.23290893" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.09.23290893">
        <p class="paperTitle">Risk of Coronavirus Disease 2019 (COVID-19) among Those Up-to-Date and Not Up-to-Date on COVID-19 Vaccination by US CDC Criteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.09.23290893" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.09.23290893" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shrestha, N. K.; Burke, P. C.; Nowacki, A. S.; Gordon, S. M.</p>
        <p class="info">Score: 5132.1, Published: 2023-09-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.09.23290893' target='https://doi.org/10.1101/2023.06.09.23290893'> 10.1101/2023.06.09.23290893</a></p>
        <p class="abstract">BackgroundThe CDC recently defined being &#34;up-to-date&#34; on COVID-19 vaccination as having received at least one dose of a COVID-19 bivalent vaccine. The purpose of this study was to compare the risk of COVID-19 among those &#34;up-to-date&#34; and &#34;not up-to-date&#34;.

MethodsEmployees of Cleveland Clinic in Ohio, USA, in employment when the COVID-19 bivalent vaccine first became available, and still employed when the XBB lineages became dominant, were included. Cumulative incidence of COVID-19 since the XBB lineages became dominant was compared across the &#34;up-to-date&#34; and &#34;not up-to-date&#34; states, by treating COVID-19 bivalent vaccination as a time- dependent covariate whose value changed on receipt of the vaccine. Risk of COVID-19 by vaccination status was also evaluated using multivariable Cox proportional hazards regression adjusting for propensity to get tested for COVID-19, age, sex, most recent prior SARS-CoV-2 infection, and number of prior vaccine doses.

ResultsCOVID-19 occurred in 1475 (3%) of 48 344 employees during the 100-day study period. The cumulative incidence of COVID-19 was lower in the &#34;not up-to-date&#34; than the &#34;up-to-date&#34; state. On multivariable analysis, being &#34;up-to-date&#34; was not associated with lower risk of COVID-19 (HR, 1.05; 95% C.I., 0.88-1.25; P-value, 0.58). Results were very similar when those 65 years and older were only considered &#34;up-to-date&#34; after 2 doses of the bivalent vaccine.

ConclusionsSince the XBB lineages became dominant, adults &#34;up-to-date&#34; on COVID-19 vaccination by the CDC definition do not have a lower risk of COVID-19 than those &#34;not up-to-date&#34;, bringing into question the value of this risk classification definition.

SummaryAmong 48 344 working-aged Cleveland Clinic employees, those &#34;up-to-date&#34; on COVID-19 vaccination did not have a lower risk of COVID-19 than those not &#34;up-to-date&#34;. The current CDC definition provides a meaningless classification of risk of COVID-19 in the adult population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.23296564">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.23296564" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.23296564">
        <p class="paperTitle">High prevalence and risk of malaria among asymptomatic individuals from villages with high rates of artemisinin partial resistance in Kyerwa district, North-western Tanzania.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.23296564" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.23296564" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mandai, S. S.; Francis, F.; Challe, D. P.; Seth, M. D.; Madebe, R. A.; Petro, D. A.; Budodo, R.; Kisambale, A. J.; Chacha, G. A.; Moshi, R.; Mbwambo, R. B.; Pereus, D.; Bakari, C.; Aaron, S.; Mbwambo, D.; Lusasi, A.; Kajange, S.; Lazaro, S.; Kapologwe, N.; Mandara, C. I.; Ishengoma, D. S.</p>
        <p class="info">Score: 6.2, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.23296564' target='https://doi.org/10.1101/2023.10.05.23296564'> 10.1101/2023.10.05.23296564</a></p>
        <p class="abstract">Abstract Background: Tanzania adopted and has been implementing the World Health Organization (WHO) recommended interventions to control and eventually eliminate malaria. However, malaria is still a leading public health problem and the country experiences heterogeneous transmission; but the drivers of these patterns are not clearly known. This study assessed the prevalence and risk of malaria infections among asymptomatic individuals living in a hyperendemic area which has high prevalence of artemisinin partial resistant parasites in Kyerwa District of Kagera region, North-western Tanzania. Methods This was a community-based cross-sectional survey that recruited participants from five villages of Kyerwa district in Kagera region. Demographic, anthropometric, clinical, parasitological, types of houses inhabited and socio-economic status (SES) data were collected using electronic capture tools running on Open Data Kit (ODK). Risk factors associated with malaria infections were determined by univariate and multivariate logistic regression and the results were presented as crude (cOR) and adjusted Odds Ratio (aOR), with 95% confidence intervals (CI). Results: A total of 4,454 individuals were tested using malaria rapid diagnostic tests (RDTs) and 1,979 (44.4%) had a positive test. The prevalence of malaria varied from 14.4% to 68.5% with significant differences among the villages (p&lt;0.001). The prevalence and risk of malaria infections were significantly higher in males (aOR =1.25, 95% CI: 1.06-1.48, p=0.04), school children ((aged 5-10 years, aOR =4.09, 95% CI: 3.39-5.10, p&lt;0.001) and (10-15 years, aOR=4.40, 95% CI: 3.46-5.59, p&lt;0.001)) and among individuals who were not using bed nets (aOR =1.29, 95% CI: 1.10-1.42, p=0.002). Other risks of malaria infections included lower SES (aOR=1.27, 95% CI:1.04-1.55, p&lt;0.001) and living in houses with open windows (aOR=1.26, 95% CI: 1.03-1.54, p=0.024). Conclusion: This study showed high prevalence of malaria infections and high heterogeneity at micro-geographic levels. The risk of malaria infections was higher in school children, males, individuals who did not use bed nets, and among participants with low SES or living in poorly constructed houses. These findings provide important baseline data in an area with a high prevalence of artemisinin partial-resistant parasites and will be utilized in future studies to monitor the trends and potential spread of such parasites. Keywords: Malaria, asymptomatic infections, risk factors, partial artemisinin resistance, Tanzania.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.23296580">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.23296580" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.23296580">
        <p class="paperTitle">Bioinformatic Analysis of Defective Viral Genomes in SARS-CoV-2 and Its Impact on Population Infection Characteristics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.23296580" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.23296580" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, Z.; Peng, Q.; Song, J.; Zhang, H.; Wei, D.; Demongeot, J.</p>
        <p class="info">Score: 3.8, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.23296580' target='https://doi.org/10.1101/2023.10.05.23296580'> 10.1101/2023.10.05.23296580</a></p>
        <p class="abstract">DVGs (Defective Viral Genomes) and SIP (Semi-Infectious Particle) are commonly present in RNA virus infections. In this study, we analyzed high-throughput sequencing data and found that DVGs or SIPs are also widely present in SARS-CoV-2. Comparison of SARS-CoV-2 with various DNA viruses revealed that the SARS-CoV-2 genome is more susceptible to damage and has greater sequencing sample heterogeneity. Variability analysis at the whole-genome sequencing depth showed a higher coefficient of variation for SARS-CoV-2, and DVG analysis indicated a high proportion of splicing sites, suggesting significant genome heterogeneity and implying that most virus particles assembled are enveloped with incomplete RNA sequences. We further analyzed the characteristics of different strains in terms of sequencing depth and DVG content differences and found that as the virus evolves, the proportion of intact genomes in virus particles increases, which can be significantly reflected in third-generation sequencing data, while the proportion of DVG gradually decreases. Specifically, the proportion of intact genome of Omicron was greater than that of Delta and Alpha strains. This can well explain why Omicron strain is more infectious than Delta and Alpha strains. We also speculate that this improvement in completeness is due to the enhancement of virus assembly ability, as the Omicron strain can quickly realize the binding of RNA and capsid protein, thereby shortening the exposure time of exposed virus RNA in the host environment and greatly reducing its degradation level. Finally, by using mathematical modeling, we simulated how DVG effects under different environmental factors affect the infection characteristics and evolution of the population. We can explain well why the severity of symptoms is closely related to the amount of virus invasion and why the same strain causes huge differences in population infection characteristics under different environmental conditions. Our study provides a new approach for future virus research and vaccine development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.23295905">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.23295905" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.23295905">
        <p class="paperTitle">Liver abnormalities following SARS-CoV-2 infection in children under 10 years of age</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.23295905" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.23295905" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Terebuh, P.; Olaker, V. R.; Kendall, E. K.; Kaelber, D. C.; Xu, R.; Davis, P. B.</p>
        <p class="info">Score: 60.9, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.23295905' target='https://doi.org/10.1101/2023.09.21.23295905'> 10.1101/2023.09.21.23295905</a></p>
        <p class="abstract">ObjectiveBeginning in October 2021 in the US and elsewhere, cases of severe pediatric hepatitis of unknown etiology were identified in young children. While the adenovirus and adenovirus-associated virus have emerged as leading etiologic suspects, we attempted to investigate a potential role for SARS-CoV-2 in the development of subsequent liver abnormalities.

DesignWe conducted a study utilizing retrospective cohorts of de-identified, aggregated data from the electronic health records of over 100 million patients contributed by US health care organizations.

ResultsCompared to propensity-score-matched children with other respiratory infections, children aged 1-10 years with COVID-19 had a higher risk of elevated transaminases (Hazard ratio (HR) (95% Confidence interval (CI)) 2.16 (1.74-2.69)) or total bilirubin (HR (CI) 3.02 (1.91-4.78)), or new diagnoses of liver diseases (HR (CI) 1.67 (1.21-2.30)) from one to six months after infection. Patients with pre-existing liver abnormalities, liver abnormalities surrounding acute infection, younger age (1-4 years), or illness requiring hospitalization all had similarly elevated risk. Children who developed liver abnormalities following COVID-19 had more pre-existing conditions than those who developed abnormalities following other infections.

ConclusionThese results indicate that SARS-CoV-2 may prime the patient for subsequent development of liver infections or non-infectious liver diseases. While rare ([~]1 in 1,000), SARS-CoV-2 is a risk for subsequent abnormalities in liver function or the diagnosis of diseases of the liver.

What is already known on this topicClusters of severe hepatitis in children in 2022 coincident with the increase in COVID-19 infections in children raised the question of the contribution of SARS-CoV-2 to the hepatitis outbreak, though it was soon determined that SARS-CoV-2 was not the primary etiologic agent.

What this study addsSARS-CoV-2 may prime the patient for subsequent development of liver infections or non-infectious liver diseases.

How this study might affect research, practice or policyDespite the mild initial disease in children, there may be longer term consequences of COVID-19, such as liver abnormalities, that warrants further investigation.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
