<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Hematologic abnormalities after COVID-19 vaccination: A large Korean population-based cohort study
Authors: CHOI, H. S.; KIM, M.-H.; CHOI, M. G.; PARK, J. H.; CHUN, E. M.
Score: 1666.4, Published: 2023-11-22 DOI: 10.1101/2023.11.15.23298565
Adverse hematologic events have been reported after COVID-19 vaccination. The objective of this study was to investigate whether hematologic abnormalities develop after COVID-19 vaccination. Retrospective cohort analyses of data from the Korean National Health Insurance Service (KNHIS) database were conducted from July 2022 to August 2023.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Hematologic abnormalities after COVID-19 vaccination: A large Korean population-based cohort study
Authors: CHOI, H. S.; KIM, M.-H.; CHOI, M. G.; PARK, J. H.; CHUN, E. M.
Score: 1666.4, Published: 2023-11-22 DOI: 10.1101/2023.11.15.23298565
Adverse hematologic events have been reported after COVID-19 vaccination. The objective of this study was to investigate whether hematologic abnormalities develop after COVID-19 vaccination. Retrospective cohort analyses of data from the Korean National Health Insurance Service (KNHIS) database were conducted from July 2022 to August 2023." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-13T10:39:57+00:00" />
<meta property="article:modified_time" content="2023-12-13T10:39:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Hematologic abnormalities after COVID-19 vaccination: A large Korean population-based cohort study
Authors: CHOI, H. S.; KIM, M.-H.; CHOI, M. G.; PARK, J. H.; CHUN, E. M.
Score: 1666.4, Published: 2023-11-22 DOI: 10.1101/2023.11.15.23298565
Adverse hematologic events have been reported after COVID-19 vaccination. The objective of this study was to investigate whether hematologic abnormalities develop after COVID-19 vaccination. Retrospective cohort analyses of data from the Korean National Health Insurance Service (KNHIS) database were conducted from July 2022 to August 2023."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Hematologic abnormalities after COVID-19 vaccination: A large Korean population-based cohort study\nAuthors: CHOI, H. S.; KIM, M.-H.; CHOI, M. G.; PARK, J. H.; CHUN, E. M.\nScore: 1666.4, Published: 2023-11-22 DOI: 10.1101/2023.11.15.23298565\nAdverse hematologic events have been reported after COVID-19 vaccination. The objective of this study was to investigate whether hematologic abnormalities develop after COVID-19 vaccination. Retrospective cohort analyses of data from the Korean National Health Insurance Service (KNHIS) database were conducted from July 2022 to August 2023.",
  "keywords": [
    
  ],
  "articleBody": " Hematologic abnormalities after COVID-19 vaccination: A large Korean population-based cohort study\nAuthors: CHOI, H. S.; KIM, M.-H.; CHOI, M. G.; PARK, J. H.; CHUN, E. M.\nScore: 1666.4, Published: 2023-11-22 DOI: 10.1101/2023.11.15.23298565\nAdverse hematologic events have been reported after COVID-19 vaccination. The objective of this study was to investigate whether hematologic abnormalities develop after COVID-19 vaccination. Retrospective cohort analyses of data from the Korean National Health Insurance Service (KNHIS) database were conducted from July 2022 to August 2023. We randomly selected data of half of those living in Seoul City as of January 1, 2021 with their diagnostic records up to December 31, 2021. The included participants were vaccinated and nonvaccinated persons aged 20 years or older (n= 4,203,887). Hematologic abnormalities after COVID-19 vaccination were identified as nutritional anemia, hemolytic anemia, aplastic anemia, coagulation defects, and neutropenia using International Classification of Diseases, Tenth Revision codes after index date. Incidence rates of hematologic abnormalities in the vaccination group 3 months after vaccination were significantly higher than those in the nonvaccinated group: 14.79 vs. 9.59 (P\u003c.001) for nutritional anemia, 7.83 vs. 5.00 (P\u003c.001) for aplastic anemia, and 4.85 vs. 1.85 (P\u003c.001) for coagulation defects. COVID-19 mRNA vaccine was associated with higher development of nutritional anemia (odds ratio [OR], 1.230 [95% CI, 1.129-1.339], P\u003c.001) and aplastic anemia (OR, 1.242 [95% CI, 1.110-1.390], P\u003c.001) than the viral vector vaccine. The risk of coagulation defects was increased (OR, 1.986 [95% CI, 1.523-2.589], P\u003c.001) after vaccination, and there was no risk difference between mRNA vaccine and viral vector vaccine (OR, 1.075 [95% CI, 0.936-1.233], P=.306). In conclusions, COVID-19 vaccination increased the risk of hematologic abnormalities. When administering the COVID-19 vaccine, careful observation will be necessary after vaccination.\nFungal pleural infection due to Microascus gracilis with pulmonary aspergillosis after COVID-19 pneumonia\nAuthors: Hu, Z.; Mao, L.; Deng, T.; Su, B.; Yang, Y.; Dong, B.; Xu, Q.; Yang, S.\nScore: 77.6, Published: 2023-12-08 DOI: 10.1101/2023.12.07.23298951\nAbstractO_ST_ABSBackgroundC_ST_ABSScopulariopsis/Microascus is a rare but devastating pathogen due to its intrinsic resistance to nearly all available antifungal agents. Microascus gracilis, an ascomycetous mould in the order Microascales, family Microascaceae, has recently emerged as a significant invasive pathogen causing opportunistic infections. Objectives and MethodsWe present a case of pleural infection caused by M. gracilis with pulmonary aspergillosis in an immunocompromised man after COVID-19 pneumonia. To further understand the characteristics of the pathogen isolated from the patient, we identified the strain through mycological characteristics, matrix-assisted laser desorption/ionization (MALDI) time-of-flight (TOF) mass spectrometry (MALDI-TOF MS) and internal transcribed spacer (ITS)-based sequencing, and performed in vitro drug susceptibility testing against common antifungal agents. Moreover, we assessed lymphocyte subsets and programmed cell death protein 1 (PD-1) expression in peripheral blood and pleural effusion to monitor the efficacy of therapy with thymosin--1 and intravenous immunoglobulin. ResultsFilamentous fungi isolated from pleural fluid were identified as M. gracilis based on classical morphology, mass spectrometry and molecular biology methods. The susceptibility results in vitro revealed that multiple antifungal agents were inactive against the strain. Adjuvant immunomodulatory treatment successfully increased the levels of CD3+ T and CD4+ T cells while decreasing the levels of CD3+PD-1+ and CD4+PD-1+ T cells in both peripheral blood and pleural effusion. ConclusionsThe immunocompromised host with opportunistic M. gracilis infection, rapid and accurate recognition through direct microscopic testing with calcofluor white and MOLDI-TOF MS, is the key to achieving a definite diagnosis, and a combination of antifungal therapy with immunomodulatory therapy is vital for improving survival.\nLack of association between HLA and asymptomatic SARS-CoV-2 infection\nAuthors: Marchal, A.; Cirulli, E. T.; Neveux, I.; Bellos, E.; Thwaites, R. S.; Schiabor Barrett, K. M.; Zhang, Y.; Nemes-Bokun, I.; Kalinova, M.; Catchpole, A.; Tangye, S. G.; Spaan, A. N.; Lack, J. B.; Ghosn, J.; Burdet, C.; Gorochov, G.; Tubach, F.; Hausfater, P.; COVID Human Genetic Effort, ; COVIDeF Study Group, ; French COVID Cohort Study Group, ; CoV-Contact Cohort, ; COVID-STORM Clinicians, ; COVID Clinicians, ; Orchestra Working Group, ; Amsterdam UMC Covid-19 Biobank, ; NIAID-USUHS COVID Study Group, ; Dalgard, C. L.; Zhang, S.-Y.; Zhang, Q.; Chiu, C.; Fellay, J.; Grzymski, J. J.; San\nScore: 47.9, Published: 2023-12-08 DOI: 10.1101/2023.12.06.23299623\nHuman genetic studies of critical COVID-19 pneumonia have revealed the essential role of type I interferon-dependent innate immunity to SARS-CoV-2 infection. Conversely, an association between the HLA-B*15:01 allele and asymptomatic SARS-CoV-2 infection in unvaccinated individuals was recently reported, suggesting a contribution of pre-existing T cell-dependent adaptive immunity. We report a lack of association of classical HLA alleles, including HLA-B*15:01, with pre-omicron asymptomatic SARS-CoV-2 infection in unvaccinated participants in a prospective population-based study in the US (191 asymptomatic vs. 945 symptomatic COVID-19 cases). Moreover, we found no such association in the international COVID Human Genetic Effort cohort (206 asymptomatic vs. 574 mild or moderate COVID-19 cases and 1,625 severe or critical COVID-19 cases). Finally, in the Human Challenge Characterisation study, the three HLA-B*15:01 individuals infected with SARS-CoV-2 developed symptoms. As with other acute primary infections, no classical HLA alleles favoring an asymptomatic course of SARS-CoV-2 infection were identified. These findings suggest that memory T-cell immunity to seasonal coronaviruses does not strongly influence the outcome of SARS-CoV-2 infection in unvaccinated individuals.\nThe spectrum of non-fatal immune-related adverse events following COVID-19 vaccination: The population-based cohort study in Seoul, South Korea\nAuthors: Suh, J. H.; Kim, H. J.; Kim, M.-H.; Choi, M. G.; Chun, E. M.\nScore: 501.6, Published: 2023-11-22 DOI: 10.1101/2023.11.15.23298566\nObjectiveTo examine the incidence rate and risk of non-fatal irAEs, including gynecological, hematological, dermatological, ophthalmological, otologic, and dental problems following the COVID-19 vaccination. MethodsWe conducted a population-based cohort study from the National Health Insurance Service (NHIS) database in Seoul, South Korea. The non-fatal irAEs included gynecological, hematological, dermatological, ophthalmological, ear, and periodontal problems as reported by the Vaccine Adverse Event Reporting Center. The cumulative incidence rate per 10,000 population, Odds ratio, and Hazard ratio (HR) with 95% Confidence Interval (CI) were measured to assess the non-fatal irAEs after COVID-19 vaccination. ResultsThe cIR of non-fatal irAEs for three months was significantly higher in vaccinated subjects than in non-vaccinated subjects, except for endometriosis. The vaccination significantly increased the risks of all the non-fatal irAEs except for visual impairment. The risk of inner ear disease showed the highest HRs (HR [95% CI] = 2.368 [2.153-2.604]) among the non-fatal irAEs following COVID-19 vaccination. Among the vaccinated subjects, heterologous vaccination was associated with the increased risk of most of the non-fatal irAEs. ConclusionsThe three-month risks of incidental non-fatal irAEs are substantially higher in the COVID-19 vaccinated subjects than in non-vaccinated controls. Our findings suggested that vaccinated subjects with predisposition are potentially vulnerable to the occurrence of diverse irAEs although the COVID-19 vaccines may not be fatal.\nPrimary exposure to Zika virus increases risk of symptomatic dengue virus infection with serotypes 2, 3, and 4 but not serotype 1\nAuthors: Zambrana, J. V.; Hasund, C. M.; Aogo, R. A.; Bos, S.; Gonzalez, K.; Collado, D.; Miranda, T.; Kuan, G.; Arguello, S.; Balmaseda, A.; Katzelnick, L.; Harris, E.\nScore: 11.3, Published: 2023-11-30 DOI: 10.1101/2023.11.29.23299187\nInfection with any of the four dengue virus serotypes (DENV1-4) can protect against or enhance subsequent dengue depending on pre-existing antibodies and the subsequent infecting serotype. Additionally, primary infection with the related flavivirus Zika virus (ZIKV) has been shown to increase DENV2 disease. Here, we measured how prior DENV and ZIKV immunity influenced risk of disease caused by all four serotypes in a pediatric Nicaraguan cohort. Of 3,412 participants in 2022, 10.6% experienced symptomatic DENV infections caused by DENV1 (n=139), DENV4 (n=133), DENV3 (n=54), DENV2 (n=9), or an undetermined serotype (n=39). Longitudinal clinical and serological data were used to define infection histories, and generalized linear and additive models adjusted for age, sex, time since the last infection, cohort year, and repeat measurements were used to predict disease risk. Compared to flavivirus-naive participants, primary ZIKV infection increased disease risk of DENV4 (relative risk = 2.62, 95% confidence interval: 1.48-4.63) and DENV3 (2.90, 1.34-6.27) but not DENV1 (1.20, 0.72-1.99). Primary DENV infection or a DENV followed by ZIKV infection also increased DENV4 risk. We re-analyzed 19 years of cohort data and demonstrated that prior flavivirus-immunity and pre- existing antibody titer differentially affected disease risk for incoming serotypes, increasing risk of DENV2 and DENV4, protecting against DENV1, and protecting at high titers but enhancing at low titers against DENV3. We thus find that prior ZIKV infection, like prior DENV infection, increases risk of certain DENV serotypes. Cross-reactivity among flaviviruses should be carefully considered when assessing vaccine safety and efficacy. One-Sentence SummaryDengue disease risk is differentially modulated depending on pre- existing immunity to dengue and Zika virus infections and the secondary infecting serotype.\nInfection Inspection: Using the power of citizen science to help with image-based prediction of antibiotic resistance in Escherichia coli\nAuthors: Farrar, A.; Feehily, C.; Turner, P.; Zagajewski, A.; Chatzimichail, S.; Crook, D.; Andersson, M.; Oakley, S.; Barrett, L.; El Sayyed, H.; Fowler, P. W.; Nellaker, C.; Kapanidis, A. N.; Stoesser, N.\nScore: 10.0, Published: 2023-12-11 DOI: 10.1101/2023.12.11.23299807\nAntibiotic resistance is an urgent global health challenge, necessitating rapid diagnostic tools to combat its escalating threat. This study introduces innovative approaches for expedited bacterial antimicrobial resistance profiling, addressing the critical need for swift clinical responses. Between February and April 2023, we conducted the Infection Inspection project, a citizen science initiative in which the public could participate in advancing an antimicrobial susceptibility testing method based on single-cell images of cellular phenotypes in response to ciprofloxacin exposure. A total of 5,273 users participated, classifying 1,045,199 images. Notably, aggregated user accuracy in image classification reached 66.8%, lower than our deep learning model's performance at 75.3%, but accuracy increased for both users and the model when ciprofloxacin treatment was greater than a strain's own minimum inhibitory concentration. We used the users' classifications to elucidate which visual features influence classification decisions, most importantly the degree of DNA compaction and heterogeneity. We paired our classification data with an image feature analysis which showed that most of the incorrect classifications were due to cellular features that varied from the expected response. This understanding informs ongoing efforts to enhance the robustness of our deep learning-based bacterial classifier and diagnostic methodology. Our successful engagement with the public through citizen science is another demonstration of the potential for collaborative efforts in scientific research, specifically increasing public awareness and advocacy on the pressing issue of antibiotic resistance, and empowering individuals to actively contribute to the development of novel diagnostics.\nNanopore sequencing of influenza A and B in Oxfordshire and the United Kingdom, 2022-23\nAuthors: Cane, J.; Sanderson, N.; Barnett, S.; Vaughan, A.; Pott, M.; Kapel, N.; Morgan, M.; Jesuthasan, G.; Samuel, R.; Ehsaan, M.; Boothe, H.; Haduli, E.; Studley, R.; Rourke, E.; Diamond, I.; Fowler, T.; Watson, C.; Stoesser, N.; Walker, A. S.; Street, T.; Eyre, D. W.\nScore: 9.6, Published: 2023-11-22 DOI: 10.1101/2023.11.21.23298840\nObjectivesWe evaluated Nanopore sequencing for influenza surveillance. MethodsInfluenza A and B PCR-positive samples from hospital patients in Oxfordshire, UK, and a UK-wide population survey from winter 2022-23 underwent Nanopore sequencing following targeted rt-PCR amplification. ResultsFrom 941 infections, successful sequencing was achieved in 292/388(75%) available Oxfordshire samples: 231(79%) A/H3N2, 53(18%) A/H1N1, and 8(3%) B/Victoria and in 53/113(47%) UK-wide samples. Sequencing was more successful at lower Ct values. Most same-sample replicate sequences had identical haemagglutinin segments (124/141;88%); a subset of samples also Illumina sequenced were very similar to Nanopore sequences. Comparison of Oxfordshire and UK-wide sequences showed frequent inter-regional transmission. Infections were closely-related to 2022-23 vaccine strains. Only one sample had a neuraminidase inhibitor resistance mutation. 849/941(90%) Oxfordshire infections were community-acquired. 63/88(72%) potentially healthcare-associated cases shared a hospital ward with [\u0026ge;]1 known infectious case. 33 epidemiologically-plausible transmission links had sequencing data for both source and recipient: 8 were within [\u0026le;]5 SNPs, of these, 5(63%) involved potential sources that were also hospital-acquired. ConclusionsNanopore influenza sequencing was reproducible and antiviral resistance rare. Inter-regional transmission was common; most infections were genomically similar. Hospital-acquired infections are likely an important source of nosocomial transmission and should be prioritised for infection prevention and control. HighlightsO_LINanopore sequencing is a reproducible tool for influenza surveillance C_LIO_LIInter-regional transmission of influenza was common across the UK C_LIO_LIInfluenza anti-viral resistance was rare C_LIO_LIIn 1 year most infections were genetically similar, hindering transmission studies C_LIO_LIHospital-acquired infections are likely a key source of nosocomial transmission C_LI\nAn amplicon-based Illumina and nanopore sequencing workflow for Chikungunya virus West Africa genotype.\nAuthors: Dieng, I.; Ndiaye, M.; Kane, M.; Balde, D.; Mbanne, M.; Diop, S. M. B. S.; Sankhe, S.; Dia, M.; Dieng, M.; Doukanda, S. F. M.; Faye, O.; Sall, A. A.; Dia, N.; Fall, G.; Faye, O.; Diagne, M. M.\nScore: 2.5, Published: 2023-12-09 DOI: 10.1101/2023.12.07.23299611\nThe Chikungunya virus, a global arbovirus, is currently causing a major outbreak in the Western African region, with the highest cases reported in Senegal and Burkina Faso. Recent molecular evolution analyses reveal that the strain responsible for the epidemic belongs to the West African genotype, with new mutations potentially impacting viral replication, antigenicity, and host adaptation. Real-time genomic monitoring is needed to track the virus's spread in new regions. A scalable West African genotype amplicon-based Whole Genome Sequencing for multiple Next Generation Sequencing platforms has been developed to support genomic investigations and identify epidemiological links during the virus's ongoing spread. This technology will help identify potential threats and support real-time genomic investigations in the ongoing spread of the virus.\nA LOW-COST CULTURE- AND DNA EXTRACTION-FREE METHOD FOR THE MOLECULAR DETECTION OF PNEUMOCOCCAL CARRIAGE IN SALIVA\nAuthors: Peno, C.; Lin, T.-Y.; Hislop, M. S.; Yolda-Carr, D.; Farjado, K.; York, A.; Pitzer, V. E.; Weinberger, D. M.; Bei, A. K.; Allicock, O. M.; Wyllie, A. L.\nScore: 2.1, Published: 2023-12-09 DOI: 10.1101/2023.12.07.23299679\nBackgroundMolecular methods have improved the sensitivity of detection of pneumococcal carriage in saliva. However, they typically require sample culture-enrichment and nucleic acid extraction, prior to performing the detection assay. These factors may limit scalability for extensive surveillance of pneumococcus, particularly in low-resource settings. Here, we evaluated the performance of a DNA-extraction-free method for the detection of pneumococcus in saliva. MethodsWe developed a streamlined qPCR-based protocol for the detection of pneumococcus, omitting culture-enrichment and DNA extraction. Using saliva samples collected from children attending childcare centers (New Haven, CT, USA), we evaluated detection of pneumococcus using the saliva lysates as compared to purified DNA extracted from culture-enriched aliquots of the paired samples using qPCR targeting the pneumococcal piaB gene. ResultsOf 759 saliva samples tested from 92 children, pneumococcus was detected in 358 (47.2%) saliva lysates prepared using the extraction-free protocol and in 369 (48.6%) DNA extracted from the culture-enriched samples. We observed a near-perfect agreement between the two protocols (Cohens kappa: 0.92; 95%CI: 0.90-0.95). While we also observed a high correlation between the qPCR CT values generated by the two methods (r=0.93, p\u003c0.0001), the CT values generated from the extraction-free, saliva lysates were higher (lower concentration) than those obtained from DNA extracted from culture-enriched samples ({Delta}CT = 6.68, p\u003c0.00001). ConclusionsFor pneumococcal carriage surveillance in children, our findings suggest that a DNA extraction-free approach may offer a cost-effective alternative to the resource-intensive culture-enrichment method. While, as expected, we observed higher qPCR CT values (lower bacterial load) in the absence of culture-enrichment, the overall rate of detection remained unaffected.\nEffectiveness and durability of mRNA-1273 BA.4/BA.5 bivalent vaccine (mRNA-1273.222) against SARS-CoV-2 BA.4/BA.5 and XBB sublineages\nAuthors: Ackerson, B. K.; Bruxvoort, K. J.; Qian, L.; Sy, L. S.; Qiu, S.; Tubert, J. E.; Lee, G. S.; Ku, J. H.; Florea, A.; Luo, Y.; Bathala, R.; Stern, J.; Choi, S. K.; Takhar, H. S.; Aragones, M.; Marks, M. A.; Anderson, E. J.; Zhou, C. K.; Sun, T.; Talarico, C. A.; Tseng, H. F.\nScore: 1.8, Published: 2023-12-11 DOI: 10.1101/2023.12.11.23299663\nBackground: Emerging SARS-CoV-2 sublineages continue to cause serious COVID-19 disease, but most individuals have not received COVID-19 vaccine for \u003e1 year. Assessment of long-term effectiveness of bivalent COVID-19 vaccines against circulating sublineages is important to inform the potential need for vaccination with updated vaccines. Methods: In this test-negative study at Kaiser Permanente Southern California, sequencing-confirmed BA.4/BA.5- or XBB-related SARS-CoV-2-positive cases during 9/1/2022-6/30/2023 were matched 1:3 to SARS-CoV-2-negative controls. We assessed mRNA-1273 bivalent relative (rVE) and absolute vaccine effectiveness (VE) compared to [\u0026ge;]2 or 0 doses of original monovalent vaccine, respectively. Outcomes were BA.4/BA.5- or XBB-related infection, emergency department/urgent care (ED/UC) encounters, and hospitalization. Results: The rVE analysis included 20,966 cases and 62,898 controls. rVE (95%CI) against BA.4/BA.5 at 14-60 days and 121-180 days was 52.7% (46.9-57.8%) and 35.5% (-2.8%-59.5%) for infection, and 59.3% (49.7-67.0%) and 33.2% (-28.2-68.0%) for ED/UC encounters. For BA.4/BA.5-related hospitalizations, rVE was 71.3% (44.9-85.1%) and 52.0% (-1.2-77.3%) at 14-60 days and 61-120 days, respectively. rVE against XBB at 14-60 days and 121-180 days, was 48.8% (33.4-60.7%) and -3.9% (-18.1-11.3%) for infection, 70.7% (52.4-82.0%) and 15.7% (-6.0-33.2%) for ED/UC encounters, and 87.9% (43.8-97.4%) and 57.1% (17.0-77.8%) for hospitalization. Results for VE and subgroup analyses (age, immunocompromise, and previous SARS-CoV-2 infection) were similar to rVE analyses. Conclusions: rVE of mRNA-1273 bivalent vaccine against BA.4/BA.5 and XBB infections, ED/UC encounters, and hospitalizations waned over time. Periodic adjustment of vaccines to target emerging variants and revaccination may be important in reducing COVID-19 morbidity and mortality.\n",
  "wordCount" : "2830",
  "inLanguage": "en",
  "datePublished": "2023-12-13T10:39:57Z",
  "dateModified": "2023-12-13T10:39:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 13, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.23298565">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.23298565" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.23298565">
        <p class="paperTitle">Hematologic abnormalities after COVID-19 vaccination: A large Korean population-based cohort study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.23298565" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.23298565" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: CHOI, H. S.; KIM, M.-H.; CHOI, M. G.; PARK, J. H.; CHUN, E. M.</p>
        <p class="info">Score: 1666.4, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.23298565' target='https://doi.org/10.1101/2023.11.15.23298565'> 10.1101/2023.11.15.23298565</a></p>
        <p class="abstract">Adverse hematologic events have been reported after COVID-19 vaccination. The objective of this study was to investigate whether hematologic abnormalities develop after COVID-19 vaccination. Retrospective cohort analyses of data from the Korean National Health Insurance Service (KNHIS) database were conducted from July 2022 to August 2023. We randomly selected data of half of those living in Seoul City as of January 1, 2021 with their diagnostic records up to December 31, 2021. The included participants were vaccinated and nonvaccinated persons aged 20 years or older (n= 4,203,887). Hematologic abnormalities after COVID-19 vaccination were identified as nutritional anemia, hemolytic anemia, aplastic anemia, coagulation defects, and neutropenia using International Classification of Diseases, Tenth Revision codes after index date. Incidence rates of hematologic abnormalities in the vaccination group 3 months after vaccination were significantly higher than those in the nonvaccinated group: 14.79 vs. 9.59 (P&lt;.001) for nutritional anemia, 7.83 vs. 5.00 (P&lt;.001) for aplastic anemia, and 4.85 vs. 1.85 (P&lt;.001) for coagulation defects. COVID-19 mRNA vaccine was associated with higher development of nutritional anemia (odds ratio [OR], 1.230 [95% CI, 1.129-1.339], P&lt;.001) and aplastic anemia (OR, 1.242 [95% CI, 1.110-1.390], P&lt;.001) than the viral vector vaccine. The risk of coagulation defects was increased (OR, 1.986 [95% CI, 1.523-2.589], P&lt;.001) after vaccination, and there was no risk difference between mRNA vaccine and viral vector vaccine (OR, 1.075 [95% CI, 0.936-1.233], P=.306). In conclusions, COVID-19 vaccination increased the risk of hematologic abnormalities. When administering the COVID-19 vaccine, careful observation will be necessary after vaccination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.23298951">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.23298951" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.23298951">
        <p class="paperTitle">Fungal pleural infection due to Microascus gracilis with pulmonary aspergillosis after COVID-19 pneumonia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.23298951" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.23298951" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, Z.; Mao, L.; Deng, T.; Su, B.; Yang, Y.; Dong, B.; Xu, Q.; Yang, S.</p>
        <p class="info">Score: 77.6, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.23298951' target='https://doi.org/10.1101/2023.12.07.23298951'> 10.1101/2023.12.07.23298951</a></p>
        <p class="abstract">AbstractO_ST_ABSBackgroundC_ST_ABSScopulariopsis/Microascus is a rare but devastating pathogen due to its intrinsic resistance to nearly all available antifungal agents. Microascus gracilis, an ascomycetous mould in the order Microascales, family Microascaceae, has recently emerged as a significant invasive pathogen causing opportunistic infections.

Objectives and MethodsWe present a case of pleural infection caused by M. gracilis with pulmonary aspergillosis in an immunocompromised man after COVID-19 pneumonia. To further understand the characteristics of the pathogen isolated from the patient, we identified the strain through mycological characteristics, matrix-assisted laser desorption/ionization (MALDI) time-of-flight (TOF) mass spectrometry (MALDI-TOF MS) and internal transcribed spacer (ITS)-based sequencing, and performed in vitro drug susceptibility testing against common antifungal agents. Moreover, we assessed lymphocyte subsets and programmed cell death protein 1 (PD-1) expression in peripheral blood and pleural effusion to monitor the efficacy of therapy with thymosin--1 and intravenous immunoglobulin.

ResultsFilamentous fungi isolated from pleural fluid were identified as M. gracilis based on classical morphology, mass spectrometry and molecular biology methods. The susceptibility results in vitro revealed that multiple antifungal agents were inactive against the strain. Adjuvant immunomodulatory treatment successfully increased the levels of CD3&#43; T and CD4&#43; T cells while decreasing the levels of CD3&#43;PD-1&#43; and CD4&#43;PD-1&#43; T cells in both peripheral blood and pleural effusion.

ConclusionsThe immunocompromised host with opportunistic M. gracilis infection, rapid and accurate recognition through direct microscopic testing with calcofluor white and MOLDI-TOF MS, is the key to achieving a definite diagnosis, and a combination of antifungal therapy with immunomodulatory therapy is vital for improving survival.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.23299623">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.23299623" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.23299623">
        <p class="paperTitle">Lack of association between HLA and asymptomatic SARS-CoV-2 infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.23299623" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.23299623" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Marchal, A.; Cirulli, E. T.; Neveux, I.; Bellos, E.; Thwaites, R. S.; Schiabor Barrett, K. M.; Zhang, Y.; Nemes-Bokun, I.; Kalinova, M.; Catchpole, A.; Tangye, S. G.; Spaan, A. N.; Lack, J. B.; Ghosn, J.; Burdet, C.; Gorochov, G.; Tubach, F.; Hausfater, P.; COVID Human Genetic Effort,  ; COVIDeF Study Group,  ; French COVID Cohort Study Group,  ; CoV-Contact Cohort,  ; COVID-STORM Clinicians,  ; COVID Clinicians,  ; Orchestra Working Group,  ; Amsterdam UMC Covid-19 Biobank,  ; NIAID-USUHS COVID Study Group,  ; Dalgard, C. L.; Zhang, S.-Y.; Zhang, Q.; Chiu, C.; Fellay, J.; Grzymski, J. J.; San</p>
        <p class="info">Score: 47.9, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.23299623' target='https://doi.org/10.1101/2023.12.06.23299623'> 10.1101/2023.12.06.23299623</a></p>
        <p class="abstract">Human genetic studies of critical COVID-19 pneumonia have revealed the essential role of type I interferon-dependent innate immunity to SARS-CoV-2 infection. Conversely, an association between the HLA-B*15:01 allele and asymptomatic SARS-CoV-2 infection in unvaccinated individuals was recently reported, suggesting a contribution of pre-existing T cell-dependent adaptive immunity. We report a lack of association of classical HLA alleles, including HLA-B*15:01, with pre-omicron asymptomatic SARS-CoV-2 infection in unvaccinated participants in a prospective population-based study in the US (191 asymptomatic vs. 945 symptomatic COVID-19 cases). Moreover, we found no such association in the international COVID Human Genetic Effort cohort (206 asymptomatic vs. 574 mild or moderate COVID-19 cases and 1,625 severe or critical COVID-19 cases). Finally, in the Human Challenge Characterisation study, the three HLA-B*15:01 individuals infected with SARS-CoV-2 developed symptoms. As with other acute primary infections, no classical HLA alleles favoring an asymptomatic course of SARS-CoV-2 infection were identified. These findings suggest that memory T-cell immunity to seasonal coronaviruses does not strongly influence the outcome of SARS-CoV-2 infection in unvaccinated individuals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.23298566">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.23298566" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.23298566">
        <p class="paperTitle">The spectrum of non-fatal immune-related adverse events following COVID-19 vaccination: The population-based cohort study in Seoul, South Korea</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.23298566" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.23298566" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Suh, J. H.; Kim, H. J.; Kim, M.-H.; Choi, M. G.; Chun, E. M.</p>
        <p class="info">Score: 501.6, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.23298566' target='https://doi.org/10.1101/2023.11.15.23298566'> 10.1101/2023.11.15.23298566</a></p>
        <p class="abstract">ObjectiveTo examine the incidence rate and risk of non-fatal irAEs, including gynecological, hematological, dermatological, ophthalmological, otologic, and dental problems following the COVID-19 vaccination.

MethodsWe conducted a population-based cohort study from the National Health Insurance Service (NHIS) database in Seoul, South Korea. The non-fatal irAEs included gynecological, hematological, dermatological, ophthalmological, ear, and periodontal problems as reported by the Vaccine Adverse Event Reporting Center. The cumulative incidence rate per 10,000 population, Odds ratio, and Hazard ratio (HR) with 95% Confidence Interval (CI) were measured to assess the non-fatal irAEs after COVID-19 vaccination.

ResultsThe cIR of non-fatal irAEs for three months was significantly higher in vaccinated subjects than in non-vaccinated subjects, except for endometriosis. The vaccination significantly increased the risks of all the non-fatal irAEs except for visual impairment. The risk of inner ear disease showed the highest HRs (HR [95% CI] = 2.368 [2.153-2.604]) among the non-fatal irAEs following COVID-19 vaccination. Among the vaccinated subjects, heterologous vaccination was associated with the increased risk of most of the non-fatal irAEs.

ConclusionsThe three-month risks of incidental non-fatal irAEs are substantially higher in the COVID-19 vaccinated subjects than in non-vaccinated controls. Our findings suggested that vaccinated subjects with predisposition are potentially vulnerable to the occurrence of diverse irAEs although the COVID-19 vaccines may not be fatal.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.23299187">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.23299187" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.23299187">
        <p class="paperTitle">Primary exposure to Zika virus increases risk of symptomatic dengue virus infection with serotypes 2, 3, and 4 but not serotype 1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.23299187" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.23299187" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zambrana, J. V.; Hasund, C. M.; Aogo, R. A.; Bos, S.; Gonzalez, K.; Collado, D.; Miranda, T.; Kuan, G.; Arguello, S.; Balmaseda, A.; Katzelnick, L.; Harris, E.</p>
        <p class="info">Score: 11.3, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.23299187' target='https://doi.org/10.1101/2023.11.29.23299187'> 10.1101/2023.11.29.23299187</a></p>
        <p class="abstract">Infection with any of the four dengue virus serotypes (DENV1-4) can protect against or enhance subsequent dengue depending on pre-existing antibodies and the subsequent infecting serotype. Additionally, primary infection with the related flavivirus Zika virus (ZIKV) has been shown to increase DENV2 disease. Here, we measured how prior DENV and ZIKV immunity influenced risk of disease caused by all four serotypes in a pediatric Nicaraguan cohort. Of 3,412 participants in 2022, 10.6% experienced symptomatic DENV infections caused by DENV1 (n=139), DENV4 (n=133), DENV3 (n=54), DENV2 (n=9), or an undetermined serotype (n=39). Longitudinal clinical and serological data were used to define infection histories, and generalized linear and additive models adjusted for age, sex, time since the last infection, cohort year, and repeat measurements were used to predict disease risk. Compared to flavivirus-naive participants, primary ZIKV infection increased disease risk of DENV4 (relative risk = 2.62, 95% confidence interval: 1.48-4.63) and DENV3 (2.90, 1.34-6.27) but not DENV1 (1.20, 0.72-1.99). Primary DENV infection or a DENV followed by ZIKV infection also increased DENV4 risk. We re-analyzed 19 years of cohort data and demonstrated that prior flavivirus-immunity and pre- existing antibody titer differentially affected disease risk for incoming serotypes, increasing risk of DENV2 and DENV4, protecting against DENV1, and protecting at high titers but enhancing at low titers against DENV3. We thus find that prior ZIKV infection, like prior DENV infection, increases risk of certain DENV serotypes. Cross-reactivity among flaviviruses should be carefully considered when assessing vaccine safety and efficacy.

One-Sentence SummaryDengue disease risk is differentially modulated depending on pre- existing immunity to dengue and Zika virus infections and the secondary infecting serotype.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.23299807">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.23299807" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.23299807">
        <p class="paperTitle">Infection Inspection: Using the power of citizen science to help with image-based prediction of antibiotic resistance in Escherichia coli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.23299807" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.23299807" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Farrar, A.; Feehily, C.; Turner, P.; Zagajewski, A.; Chatzimichail, S.; Crook, D.; Andersson, M.; Oakley, S.; Barrett, L.; El Sayyed, H.; Fowler, P. W.; Nellaker, C.; Kapanidis, A. N.; Stoesser, N.</p>
        <p class="info">Score: 10.0, Published: 2023-12-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.23299807' target='https://doi.org/10.1101/2023.12.11.23299807'> 10.1101/2023.12.11.23299807</a></p>
        <p class="abstract">Antibiotic resistance is an urgent global health challenge, necessitating rapid diagnostic tools to combat its escalating threat. This study introduces innovative approaches for expedited bacterial antimicrobial resistance profiling, addressing the critical need for swift clinical responses. Between February and April 2023, we conducted the Infection Inspection project, a citizen science initiative in which the public could participate in advancing an antimicrobial susceptibility testing method based on single-cell images of cellular phenotypes in response to ciprofloxacin exposure. A total of 5,273 users participated, classifying 1,045,199 images. Notably, aggregated user accuracy in image classification reached 66.8%, lower than our deep learning model&#39;s performance at 75.3%, but accuracy increased for both users and the model when ciprofloxacin treatment was greater than a strain&#39;s own minimum inhibitory concentration. We used the users&#39; classifications to elucidate which visual features influence classification decisions, most importantly the degree of DNA compaction and heterogeneity. We paired our classification data with an image feature analysis which showed that most of the incorrect classifications were due to cellular features that varied from the expected response. This understanding informs ongoing efforts to enhance the robustness of our deep learning-based bacterial classifier and diagnostic methodology. Our successful engagement with the public through citizen science is another demonstration of the potential for collaborative efforts in scientific research, specifically increasing public awareness and advocacy on the pressing issue of antibiotic resistance, and empowering individuals to actively contribute to the development of novel diagnostics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.23298840">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.23298840" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.23298840">
        <p class="paperTitle">Nanopore sequencing of influenza A and B in Oxfordshire and the United Kingdom, 2022-23</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.23298840" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.23298840" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cane, J.; Sanderson, N.; Barnett, S.; Vaughan, A.; Pott, M.; Kapel, N.; Morgan, M.; Jesuthasan, G.; Samuel, R.; Ehsaan, M.; Boothe, H.; Haduli, E.; Studley, R.; Rourke, E.; Diamond, I.; Fowler, T.; Watson, C.; Stoesser, N.; Walker, A. S.; Street, T.; Eyre, D. W.</p>
        <p class="info">Score: 9.6, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.23298840' target='https://doi.org/10.1101/2023.11.21.23298840'> 10.1101/2023.11.21.23298840</a></p>
        <p class="abstract">ObjectivesWe evaluated Nanopore sequencing for influenza surveillance.

MethodsInfluenza A and B PCR-positive samples from hospital patients in Oxfordshire, UK, and a UK-wide population survey from winter 2022-23 underwent Nanopore sequencing following targeted rt-PCR amplification.

ResultsFrom 941 infections, successful sequencing was achieved in 292/388(75%) available Oxfordshire samples: 231(79%) A/H3N2, 53(18%) A/H1N1, and 8(3%) B/Victoria and in 53/113(47%) UK-wide samples. Sequencing was more successful at lower Ct values. Most same-sample replicate sequences had identical haemagglutinin segments (124/141;88%); a subset of samples also Illumina sequenced were very similar to Nanopore sequences.

Comparison of Oxfordshire and UK-wide sequences showed frequent inter-regional transmission. Infections were closely-related to 2022-23 vaccine strains. Only one sample had a neuraminidase inhibitor resistance mutation.

849/941(90%) Oxfordshire infections were community-acquired. 63/88(72%) potentially healthcare-associated cases shared a hospital ward with [&amp;ge;]1 known infectious case. 33 epidemiologically-plausible transmission links had sequencing data for both source and recipient: 8 were within [&amp;le;]5 SNPs, of these, 5(63%) involved potential sources that were also hospital-acquired.

ConclusionsNanopore influenza sequencing was reproducible and antiviral resistance rare. Inter-regional transmission was common; most infections were genomically similar. Hospital-acquired infections are likely an important source of nosocomial transmission and should be prioritised for infection prevention and control.

HighlightsO_LINanopore sequencing is a reproducible tool for influenza surveillance
C_LIO_LIInter-regional transmission of influenza was common across the UK
C_LIO_LIInfluenza anti-viral resistance was rare
C_LIO_LIIn 1 year most infections were genetically similar, hindering transmission studies
C_LIO_LIHospital-acquired infections are likely a key source of nosocomial transmission
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.23299611">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.23299611" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.23299611">
        <p class="paperTitle">An amplicon-based Illumina and nanopore sequencing workflow for Chikungunya virus West Africa genotype.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.23299611" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.23299611" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dieng, I.; Ndiaye, M.; Kane, M.; Balde, D.; Mbanne, M.; Diop, S. M. B. S.; Sankhe, S.; Dia, M.; Dieng, M.; Doukanda, S. F. M.; Faye, O.; Sall, A. A.; Dia, N.; Fall, G.; Faye, O.; Diagne, M. M.</p>
        <p class="info">Score: 2.5, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.23299611' target='https://doi.org/10.1101/2023.12.07.23299611'> 10.1101/2023.12.07.23299611</a></p>
        <p class="abstract">The Chikungunya virus, a global arbovirus, is currently causing a major outbreak in the Western African region, with the highest cases reported in Senegal and Burkina Faso. Recent molecular evolution analyses reveal that the strain responsible for the epidemic belongs to the West African genotype, with new mutations potentially impacting viral replication, antigenicity, and host adaptation. Real-time genomic monitoring is needed to track the virus&#39;s spread in new regions. A scalable West African genotype amplicon-based Whole Genome Sequencing for multiple Next Generation Sequencing platforms has been developed to support genomic investigations and identify epidemiological links during the virus&#39;s ongoing spread. This technology will help identify potential threats and support real-time genomic investigations in the ongoing spread of the virus.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.23299679">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.23299679" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.23299679">
        <p class="paperTitle">A LOW-COST CULTURE- AND DNA EXTRACTION-FREE METHOD FOR THE MOLECULAR DETECTION OF PNEUMOCOCCAL CARRIAGE IN SALIVA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.23299679" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.23299679" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peno, C.; Lin, T.-Y.; Hislop, M. S.; Yolda-Carr, D.; Farjado, K.; York, A.; Pitzer, V. E.; Weinberger, D. M.; Bei, A. K.; Allicock, O. M.; Wyllie, A. L.</p>
        <p class="info">Score: 2.1, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.23299679' target='https://doi.org/10.1101/2023.12.07.23299679'> 10.1101/2023.12.07.23299679</a></p>
        <p class="abstract">BackgroundMolecular methods have improved the sensitivity of detection of pneumococcal carriage in saliva. However, they typically require sample culture-enrichment and nucleic acid extraction, prior to performing the detection assay. These factors may limit scalability for extensive surveillance of pneumococcus, particularly in low-resource settings. Here, we evaluated the performance of a DNA-extraction-free method for the detection of pneumococcus in saliva.

MethodsWe developed a streamlined qPCR-based protocol for the detection of pneumococcus, omitting culture-enrichment and DNA extraction. Using saliva samples collected from children attending childcare centers (New Haven, CT, USA), we evaluated detection of pneumococcus using the saliva lysates as compared to purified DNA extracted from culture-enriched aliquots of the paired samples using qPCR targeting the pneumococcal piaB gene.

ResultsOf 759 saliva samples tested from 92 children, pneumococcus was detected in 358 (47.2%) saliva lysates prepared using the extraction-free protocol and in 369 (48.6%) DNA extracted from the culture-enriched samples. We observed a near-perfect agreement between the two protocols (Cohens kappa: 0.92; 95%CI: 0.90-0.95). While we also observed a high correlation between the qPCR CT values generated by the two methods (r=0.93, p&lt;0.0001), the CT values generated from the extraction-free, saliva lysates were higher (lower concentration) than those obtained from DNA extracted from culture-enriched samples ({Delta}CT = 6.68, p&lt;0.00001).

ConclusionsFor pneumococcal carriage surveillance in children, our findings suggest that a DNA extraction-free approach may offer a cost-effective alternative to the resource-intensive culture-enrichment method. While, as expected, we observed higher qPCR CT values (lower bacterial load) in the absence of culture-enrichment, the overall rate of detection remained unaffected.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.23299663">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.23299663" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.23299663">
        <p class="paperTitle">Effectiveness and durability of mRNA-1273 BA.4/BA.5 bivalent vaccine (mRNA-1273.222) against SARS-CoV-2 BA.4/BA.5 and XBB sublineages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.23299663" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.23299663" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ackerson, B. K.; Bruxvoort, K. J.; Qian, L.; Sy, L. S.; Qiu, S.; Tubert, J. E.; Lee, G. S.; Ku, J. H.; Florea, A.; Luo, Y.; Bathala, R.; Stern, J.; Choi, S. K.; Takhar, H. S.; Aragones, M.; Marks, M. A.; Anderson, E. J.; Zhou, C. K.; Sun, T.; Talarico, C. A.; Tseng, H. F.</p>
        <p class="info">Score: 1.8, Published: 2023-12-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.23299663' target='https://doi.org/10.1101/2023.12.11.23299663'> 10.1101/2023.12.11.23299663</a></p>
        <p class="abstract">Background: Emerging SARS-CoV-2 sublineages continue to cause serious COVID-19 disease, but most individuals have not received COVID-19 vaccine for &gt;1 year. Assessment of long-term effectiveness of bivalent COVID-19 vaccines against circulating sublineages is important to inform the potential need for vaccination with updated vaccines. Methods: In this test-negative study at Kaiser Permanente Southern California, sequencing-confirmed BA.4/BA.5- or XBB-related SARS-CoV-2-positive cases during 9/1/2022-6/30/2023 were matched 1:3 to SARS-CoV-2-negative controls. We assessed mRNA-1273 bivalent relative (rVE) and absolute vaccine effectiveness (VE) compared to [&amp;ge;]2 or 0 doses of original monovalent vaccine, respectively. Outcomes were BA.4/BA.5- or XBB-related infection, emergency department/urgent care (ED/UC) encounters, and hospitalization. Results: The rVE analysis included 20,966 cases and 62,898 controls. rVE (95%CI) against BA.4/BA.5 at 14-60 days and 121-180 days was 52.7% (46.9-57.8%) and 35.5% (-2.8%-59.5%) for infection, and 59.3% (49.7-67.0%) and 33.2% (-28.2-68.0%) for ED/UC encounters. For BA.4/BA.5-related hospitalizations, rVE was 71.3% (44.9-85.1%) and 52.0% (-1.2-77.3%) at 14-60 days and 61-120 days, respectively. rVE against XBB at 14-60 days and 121-180 days, was 48.8% (33.4-60.7%) and -3.9% (-18.1-11.3%) for infection, 70.7% (52.4-82.0%) and 15.7% (-6.0-33.2%) for ED/UC encounters, and 87.9% (43.8-97.4%) and 57.1% (17.0-77.8%) for hospitalization. Results for VE and subgroup analyses (age, immunocompromise, and previous SARS-CoV-2 infection) were similar to rVE analyses. Conclusions: rVE of mRNA-1273 bivalent vaccine against BA.4/BA.5 and XBB infections, ED/UC encounters, and hospitalizations waned over time. Periodic adjustment of vaccines to target emerging variants and revaccination may be important in reducing COVID-19 morbidity and mortality.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
