<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Immunological imprinting shapes the specificity of human antibody responses against SARS-CoV-2 variants
Authors: Johnston, T. S.; Li, S. H.; Painter, M. M.; Atkinson, R. K.; Douek, N. R.; Reeg, D. B.; Doueck, D. C.; Wherry, E. J.; Hensley, S. E.
Score: 24.5, Published: 2024-01-09 DOI: 10.1101/2024.01.08.24301002
The spike glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) continues to accumulate substitutions, leading to breakthrough infections of vaccinated individuals and prompting the development of updated booster vaccines.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Immunological imprinting shapes the specificity of human antibody responses against SARS-CoV-2 variants
Authors: Johnston, T. S.; Li, S. H.; Painter, M. M.; Atkinson, R. K.; Douek, N. R.; Reeg, D. B.; Doueck, D. C.; Wherry, E. J.; Hensley, S. E.
Score: 24.5, Published: 2024-01-09 DOI: 10.1101/2024.01.08.24301002
The spike glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) continues to accumulate substitutions, leading to breakthrough infections of vaccinated individuals and prompting the development of updated booster vaccines." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-10T10:40:05+00:00" />
<meta property="article:modified_time" content="2024-01-10T10:40:05+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Immunological imprinting shapes the specificity of human antibody responses against SARS-CoV-2 variants
Authors: Johnston, T. S.; Li, S. H.; Painter, M. M.; Atkinson, R. K.; Douek, N. R.; Reeg, D. B.; Doueck, D. C.; Wherry, E. J.; Hensley, S. E.
Score: 24.5, Published: 2024-01-09 DOI: 10.1101/2024.01.08.24301002
The spike glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) continues to accumulate substitutions, leading to breakthrough infections of vaccinated individuals and prompting the development of updated booster vaccines."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Immunological imprinting shapes the specificity of human antibody responses against SARS-CoV-2 variants\nAuthors: Johnston, T. S.; Li, S. H.; Painter, M. M.; Atkinson, R. K.; Douek, N. R.; Reeg, D. B.; Doueck, D. C.; Wherry, E. J.; Hensley, S. E.\nScore: 24.5, Published: 2024-01-09 DOI: 10.1101/2024.01.08.24301002\nThe spike glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) continues to accumulate substitutions, leading to breakthrough infections of vaccinated individuals and prompting the development of updated booster vaccines.",
  "keywords": [
    
  ],
  "articleBody": " Immunological imprinting shapes the specificity of human antibody responses against SARS-CoV-2 variants\nAuthors: Johnston, T. S.; Li, S. H.; Painter, M. M.; Atkinson, R. K.; Douek, N. R.; Reeg, D. B.; Doueck, D. C.; Wherry, E. J.; Hensley, S. E.\nScore: 24.5, Published: 2024-01-09 DOI: 10.1101/2024.01.08.24301002\nThe spike glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) continues to accumulate substitutions, leading to breakthrough infections of vaccinated individuals and prompting the development of updated booster vaccines. Here, we determined the specificity and functionality of antibody and B cell responses following exposure to BA.5 and XBB variants in individuals who received ancestral SARS-CoV-2 mRNA vaccines. BA.5 exposures elicited antibody responses that primarily targeted epitopes conserved between the BA.5 and ancestral spike, with poor reactivity to the XBB.1.5 variant. XBB exposures also elicited antibody responses that targeted epitopes conserved between the XBB.1.5 and ancestral spike. However, unlike BA.5, a single XBB exposure elicited low levels of XBB.1.5-specific antibodies and B cells in some individuals. Pre-existing cross-reactive B cells and antibodies were correlated with stronger overall responses to XBB but weaker XBB-specific responses, suggesting that baseline immunity influences the activation of variant-specific SARS-CoV-2 responses.\nVariant-specific humoral immune response to SARS-CoV-2 escape mutants arising in clinically severe, prolonged infection\nAuthors: Guenther, T.; Schoefbaenker, M.; Lorentzen, E. U.; Romberg, M.-L.; Hennies, M. T.; Neddermeyer, R.; Mueller, M. M.; Mellmann, A.; Lenz, G.; Stelljes, M.; Hrincius, E. R.; Vollenberg, R.; Ludwig, S.; Tepasse, P.-R.; Kuehn, J. E.\nScore: 23.4, Published: 2024-01-07 DOI: 10.1101/2024.01.06.24300890\nNeutralising antibodies against the SARS-CoV-2 spike (S) protein are major determinants of protective immunity, though insufficient antibody responses may cause the emergence of escape mutants. We studied the intra-host evolution and the humoral immune response in a B-cell depleted, haemato-oncologic patient experiencing clinically severe, prolonged SARS-CoV-2 infection with a virus of lineage B.1.177.81. Bamlanivimab treatment early in infection was associated with the emergence of the bamlanivimab escape mutation S:S494P. Ten days before virus elimination, additional mutations within the N-terminal domain (NTD) and the receptor binding domain (RBD) of S were observed, of which the triple mutant S:Delta141-4 E484K S494P became dominant. Routine serology revealed no evidence of an antibody response in the patient. A detailed analysis of the variant-specific immune response by pseudotyped virus neutralisation test (pVNT), surrogate VNT (sVNT), and immunoglobulin-capture EIA showed that the onset of an IgM-dominated antibody response coincided with the appearance of escape mutations. The formation of neutralising antibodies against S:Delta141-4 E484K S494P correlated with virus elimination. One year later, the patient experienced clinically mild re-infection with Omicron BA.1.18, which was treated with sotrovimab and resulted in a massive increase of Omicron-reactive antibodies. In conclusion, the onset of an IgM-dominated endogenous immune response in an immunocompromised patient coincided with the appearance of additional mutations in the NTD and RBD of S in a bamlanivimab-resistant virus. Although virus elimination was ultimately achieved, this humoral immune response escaped detection by routine diagnosis and created a situation temporarily favouring the emergence of escape variants with known epidemiological relevance.\nReverse mutational scanning of spike BA.2.86 identifies the epitopes contributing to immune escape from polyclonal sera\nAuthors: Bdeir, N.; Lüddecke, T.; Maass, H.; Schmelz, S.; Jacobsen, H.; Metzdorf, K.; Cossmann, A.; Stankov, M.; Hoffmann, M.; Pöhlmann, S.; Blankenfeldt, W.; Jablonka, A.; Behrens, G. M. N.; Cicin-Sain, L.\nScore: 22.0, Published: 2024-01-03 DOI: 10.1101/2024.01.03.23300575\nThe recently detected Omicron BA.2.86 lineage contains more than 30 amino acid mutations relative to BA.2. Here, we identify the epitopes driving immune escape of BA.2.86 and its derivative JN.1 (BA.2.86 + S455L) lineage. We investigated the cross-reactive humoral immunity within a cohort of health care workers against Omicron BA.2.86 and JN.1 by employing pseudo-viral mutants. We demonstrate that BA.2.86 and especially JN.1 evaded neutralization by serum antibodies of fully vaccinated individuals. To discern the contribution of individual epitope mutations to immune escape, we constructed a library of 33 BA.2.86 mutants, each of which harbored a single revertant mutation going back to BA.2. This library was used in a reverse mutational scanning approach to define serum neutralization titers against each epitope separately. The mutations within the receptor binding domain (RBD) at position K356T and to a lesser extent the mutations N460K, V483{Delta}, A484K, and F486P enhanced the immune escape. More surprisingly, the mutation 16insMPLF within the spike N-terminal domain (NTD) and the mutation P621S in S1/S2 significantly contributed to antibody escape of BA.2.86. Upon XBB.1.5 booster vaccination, neutralization titers against JN.1 and BA.2.86 improved relative to all ancestral strains, and the residual immune escape was driven by mutations at positions 16insMPLF, {Delta}144Y, E544K, P621S, and A484K.\nEscalating combinations of enhanced infectivity and immune escape define SARS-CoV-2 Omicron lineage replacement\nAuthors: Chen, N. F. G.; Pham, K.; Chaguza, C.; Lopes, R.; Klaassen, F.; Weinberger, D. M.; Pitzer, V. E.; Warren, J. L.; Grubaugh, N. D.; Hahn, A. M.\nScore: 7.3, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300790\nIn 2022, consecutive sweeps of the highly transmissible SARS-CoV-2 Omicron-family maintained high viral transmission levels despite extensive antigen exposure on the population level resulting from both vaccinations and infections. To better understand variant fitness in the context of the highly dynamic immunity landscape of 2022, we aimed to dissect the interplay between immunity and fitness advantages of emerging SARS-CoV-2 Omicron lineages on the population-level. We evaluated the relative contribution of higher intrinsic transmissibility or immune escape on the fitness of emerging lineages by analyzing data collected through our local genomic surveillance program from Connecticut, USA. We compared growth rates, estimated infections, effective reproductive rates, average viral copy numbers, and likelihood for causing vaccine break-through infections. Using these population-level data, we find that newly emerging Omicron lineages reach dominance through a specific combination of enhanced intrinsic transmissibility and immune escape that varies over time depending on the state of the host-population. Using similar frameworks that integrate whole genome sequencing together with clinical, laboratory, and epidemiological data can advance our knowledge on host-pathogen dynamics in the post-emergence phase that can be applied to other communicable diseases beyond SARS-CoV-2.\nDiverse and weakly immunogenic var gene expression facilitates malaria infection\nAuthors: Bhardwaj, I.; Nyarko, P. B.; Ashn, A. B.; Cohen, C.; Ceesay, S.; Achan, J.; Dabira, E.; Nakajima, R.; Jain, A.; Taghavian, O.; Jasinskas, A.; Felgner, P. L.; D'Alessandro, U.; Bousema, T.; Travassos, M.; Radulescu, O.; Claessens, A.\nScore: 5.2, Published: 2024-01-02 DOI: 10.1101/2023.12.27.23300577\nPlasmodium falciparum is believed to escape immunity via antigenic variation, mediated in part by 60 var genes. These genes undergo mutually exclusive expression and encode the PfEMP1 surface antigen. The frequency of var switching and the immunogenicity of each expressed PfEMP1 remain unclear. To this end, we carried out a Controlled Human Malaria Infection (CHMI) study with 19 adult African volunteers in The Gambia to gain insight into the effect of naturally acquired immunity on the expressed var gene repertoire during early phase of an infection. Our findings demonstrated a strong correlation between the diversity of var expression, quantified through entropy, and infection outcome. Low-immunity individuals were characterised by high var entropy profiles, higher parasitaemia, and lower sero-recognised PfEMP1 domains compared to high-immunity individuals. For the first time we recorded the probability of var gene switching in vitro and of turnover in vivo, enabling us to estimate both intrinsic switching and negative-selection effects. These processes are rapid, resulting in estimated turnover/switching probabilities of 69% - 97% and 7% - 57% per generation, in vivo and in vitro, respectively. Var (PfEMP1) expression triggered time-dependent humoral immune responses in low immunity individuals, with many PfEMP1 domains remaining weakly immunogenic. We conclude that the role of intrinsic var switching is to reset and maintain a diverse var repertoire. The high var switching rates and weak PfEMP1 immunogenicity benefit parasite survival during the CHMI.\nMalaria species prevalence among asymptomatic individuals in four regions of Mainland Tanzania\nAuthors: Popkin-Hall, Z. R.; Seth, M. D.; Madebe, R. A.; Budodo, R.; Bakari, C.; Francis, F.; Pereus, D.; Giesbrecht, D. J.; Mandara, C. I.; Mbwambo, D.; Aaron, S.; Lusasi, A.; Lazaro, S.; Bailey, J. A.; Juliano, J. J.; Gutman, J. R.; Ishengoma, D. S.\nScore: 3.8, Published: 2023-12-30 DOI: 10.1101/2023.12.28.23300584\nRecent studies point to the need to incorporate non-falciparum species detection into malaria surveillance activities in sub-Saharan Africa, where 95% of malaria cases occur. Although Plasmodium falciparum infection is typically more severe, diagnosis, treatment, and control for P. malariae, P. ovale spp., and P. vivax may be more challenging. The prevalence of these species throughout sub-Saharan Africa is poorly defined. Tanzania has geographically heterogeneous transmission levels but an overall high malaria burden. In order to estimate the prevalence of malaria species in Mainland Tanzania, 1,428 samples were randomly selected from 6,005 asymptomatic isolates collected in cross-sectional community surveys across four regions and analyzed via qPCR to detect each Plasmodium species. P. falciparum was most prevalent, with P. malariae and P. ovale spp. detected at lower prevalence (\u003c5%) in all four regions. P. vivax was not detected. Malaria elimination efforts in Tanzania will need to account for these non-falciparum species.\nThoughts and perspectives of metagenome sequencing as a diagnostic tool for infectious disease: an interpretive phenomenological study\nAuthors: Trivett, H.; Darby, A. C.; Oyebode, O.\nScore: 3.2, Published: 2024-01-03 DOI: 10.1101/2024.01.02.24300703\nBackgroundEffective infectious disease diagnostics (IDD) are vital for informing clinical decision-making regarding the treatment and patient management of disease and infections. Conventional clinical methods rely upon culture-dependent techniques, and there has been little shift in the acceptance and integration of culture-independent sequencing methods into routine clinical IDD. This study explored stakeholders experiences within IDD, including those working in clinical settings and those conducting research at the forefront of microbial genomics. We aimed to identify factors driving the development and implementation of metagenome sequencing as a routine diagnostic. MethodsVirtual semi-structured interviews were conducted with purposively selected individuals involved in IDD. The interviews explored the experiences of implementing metagenome sequencing as a diagnostic tool and decisions about which diagnostics are used for identifying bacteria-causing infections. Thematic analysis was used to analyse the data, and an Interpretive Phenomenological approach was used throughout. ResultsTen individuals were interviewed between July 2021 and October 2021, including Clinical scientists, consultants, and professors in academia. Their experience ranged from no knowledge of metagenome sequencing to an expert understanding of the phenomenon. Five themes emerged: Diagnostic Choice, Infrastructure, Open Data Sharing, COVID-19, and Communication. Participants recognised the need for new diagnostics to be implemented to overcome the limitations of current diagnostic approaches but highlighted the barriers to integrating new diagnostics into clinical settings, such as the impact on clinical decision-making, accreditation, and cost. However, participants felt that lessons could be learnt from using metagenomics in COVID-19 and how other diagnostic platforms have been integrated into clinical settings over the last 20 years. ConclusionsThe study provided clear evidence to address the knowledge gap in current literature and practice for developing and implementing metagenome sequencing as a potential IDD. The knowledge of new and upcoming genomic diagnostic testing is not equally distributed throughout the UK, impacting the understanding and drive to integrate metagenome sequencing into routine clinical diagnostics. Improvements in access to new diagnostics could improve patient treatment and management and positively impact public health.\nTrends in SARS-CoV-2 seroprevalence among pregnant women attending first antenatal care visits in Zambia: a repeated cross-sectional survey, 2021-2022\nAuthors: Heilmann, E.; Tembo, T.; Fwoloshi, S.; Kabamba, B.; Chilambe, F.; Kalenga, K.; Siwingwa, M.; Mulube, C.; Seffren, V.; Bolton-Moore, C.; Simwanza, J.; Yingst, S.; Yadav, R.; Rogier, E.; Auld, A. F.; Agolory, S.; Kapina, M.; Gutman, J. R.; Savory, T.; Kangale, C.; Mulenga, L. B.; Sikazwe, I.; Hines, J. Z.\nScore: 2.5, Published: 2024-01-03 DOI: 10.1101/2024.01.02.24300729\nBackgroundSARS-CoV-2 serosurveys help estimate the extent of transmission and guide allocation of COVID-19 vaccines. We measured SARS-CoV-2 seroprevalence among women attending ANC clinics to assess exposure trends over time in Zambia. MethodsWe conducted repeated cross-sectional surveys among pregnant women aged 15-49 years attending their first ANC visits in four districts of Zambia (two urban and two rural) during September 2021-September 2022. Serologic testing was done using a multiplex bead assay which detects IgG antibodies to the nucleocapsid protein and the spike protein receptor-binding domain (RBD). We calculated monthly SARS-CoV-2 seroprevalence by district. We also categorized seropositive results as infection alone, infection and vaccination, or vaccination alone based on COVID-19 vaccination status and anti-RBD and anti-nucleocapsid test results. FindingsAmong 8,304 participants, 5,296 (63.8%) were cumulatively seropositive for SARS-CoV-2 antibodies. SARS-CoV-2 seroprevalence primarily increased from September 2021 to September 2022 in three districts (Lusaka: 61.8-100.0%, Chongwe: 39.6-94.7%, Chipata: 56.5-95.0%), but in Chadiza, seroprevalence increased from 27.8% in September 2021 to 77.2% in April 2022 before gradually dropping to 56.6% in July 2022. Among 5,906 participants with a valid COVID-19 vaccination status, infection alone accounted for antibody responses in 77.7% (4,590) of participants. InterpretationMost women attending ANC had evidence of prior SARS-CoV-2 infection and most SARS-CoV-2 seropositivity was infection-induced. Capturing COVID-19 vaccination status and using a multiplex bead assay with anti-nucleocapsid and anti-RBD targets facilitated distinguishing infection-induced versus vaccine-induced antibody responses during a period of increasing COVID-19 vaccine coverage in Zambia. Declining seroprevalence in Chadiza may indicate waning antibodies and a need for booster vaccines. ANC clinics have a potential role in ongoing SARS-CoV-2 serosurveillance and can continue to provide insights into SARS-CoV-2 antibody dynamics to inform near real-time public health responses.\nPhylogeography and reassortment patterns of human influenza A viruses in sub-Saharan Africa\nAuthors: Owuor, D. C.; de Laurent, Z. R.; Oketch, J. W.; Murunga, N.; Otieno, J. R.; Chaves, S. S.; Nokes, J.; Agoti, C. N.\nScore: 2.1, Published: 2024-01-09 DOI: 10.1101/2024.01.07.24300955\nBackground. The role of sub-Saharan Africa in the global spread of influenza viruses remains unclear due to insufficient spatiotemporal sequence data. Methods. Here, we analyzed 222 codon-complete sequences of influenza A viruses (IAVs) sampled between 2011 and 2013 from five countries across sub-Saharan Africa (Kenya, Zambia, Mali, Gambia, and South Africa); these genomes were compared with 1,209 contemporaneous global genomes using phylogeographical approaches. Results. The spread of influenza in sub-Saharan Africa was characterized by (i) multiple introductions of IAVs into the region over consecutive influenza seasons, with viral importations originating from multiple global geographical regions, some of which persisted in circulation as intra-subtype reassortants for multiple seasons, (ii) virus transfer between sub-Saharan African countries, and (iii) virus export from sub-Saharan Africa to other geographical regions. Conclusion. Despite sparse data from influenza surveillance in sub-Saharan Africa, our findings support the notion that influenza viruses persist as temporally structured migrating metapopulations in which new virus strains can emerge in any geographical region, including in sub-Saharan Africa; these lineages may have been capable of dissemination to other continents through a globally migrating virus population. Further knowledge of the viral lineages that circulate within understudied sub-Saharan Africa regions is required to inform vaccination strategies in those regions.\nBeyond Borders: Spatial Disparities in the Mortality Burden of the Covid-19 pandemic across 569 European Regions (2020-2021)\nAuthors: Bonnet, F.; Grigoriev, P.; Sauerberg, M.; Alliger, I.; Muhlichen, M.; Camarda, C.-G.\nScore: 34.5, Published: 2023-12-18 DOI: 10.1101/2023.12.18.23300149\nThis article presents a detailed analysis of the global mortality burden of the COVID-19 pandemic across 569 regions in 25 European countries. We produce sex-specific excess mortality and present our results using Age-Standardised Years of Life Lost (ASYLL) in 2020 and 2021, as well as the cumulative impact over the two pandemic years. Employing a robust forecasting approach that considers regional diversity and provides confidence intervals, we find notable losses in 362 regions in 2020 (440 regions in 2021). Conversely, only seven regions experienced gains in 2020 (four regions in 2021). Most importantly, we estimate that eight regions suffered losses exceeding 20 years of life per 1,000 population in 2020, whereas this number increased to 75 regions in 2021. The contiguity of the regions investigated in our study also reveals the changing geographical patterns of the pandemic. While the highest excess mortality values were concentrated in the early COVID-19 outbreak areas during the initial pandemic year, a clear East-West gradient appeared in 2021, with regions of Slovakia, Hungary, and Latvia experiencing the highest losses. This research underscores the importance of regional analyses for a nuanced comprehension of the pandemics impact.\n",
  "wordCount" : "2664",
  "inLanguage": "en",
  "datePublished": "2024-01-10T10:40:05Z",
  "dateModified": "2024-01-10T10:40:05Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 10, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.24301002">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.24301002" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.24301002">
        <p class="paperTitle">Immunological imprinting shapes the specificity of human antibody responses against SARS-CoV-2 variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.24301002" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.24301002" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnston, T. S.; Li, S. H.; Painter, M. M.; Atkinson, R. K.; Douek, N. R.; Reeg, D. B.; Doueck, D. C.; Wherry, E. J.; Hensley, S. E.</p>
        <p class="info">Score: 24.5, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.24301002' target='https://doi.org/10.1101/2024.01.08.24301002'> 10.1101/2024.01.08.24301002</a></p>
        <p class="abstract">The spike glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) continues to accumulate substitutions, leading to breakthrough infections of vaccinated individuals and prompting the development of updated booster vaccines. Here, we determined the specificity and functionality of antibody and B cell responses following exposure to BA.5 and XBB variants in individuals who received ancestral SARS-CoV-2 mRNA vaccines. BA.5 exposures elicited antibody responses that primarily targeted epitopes conserved between the BA.5 and ancestral spike, with poor reactivity to the XBB.1.5 variant. XBB exposures also elicited antibody responses that targeted epitopes conserved between the XBB.1.5 and ancestral spike. However, unlike BA.5, a single XBB exposure elicited low levels of XBB.1.5-specific antibodies and B cells in some individuals. Pre-existing cross-reactive B cells and antibodies were correlated with stronger overall responses to XBB but weaker XBB-specific responses, suggesting that baseline immunity influences the activation of variant-specific SARS-CoV-2 responses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.06.24300890">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.06.24300890" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.06.24300890">
        <p class="paperTitle">Variant-specific humoral immune response to SARS-CoV-2 escape mutants arising in clinically severe, prolonged infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.06.24300890" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.06.24300890" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guenther, T.; Schoefbaenker, M.; Lorentzen, E. U.; Romberg, M.-L.; Hennies, M. T.; Neddermeyer, R.; Mueller, M. M.; Mellmann, A.; Lenz, G.; Stelljes, M.; Hrincius, E. R.; Vollenberg, R.; Ludwig, S.; Tepasse, P.-R.; Kuehn, J. E.</p>
        <p class="info">Score: 23.4, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.06.24300890' target='https://doi.org/10.1101/2024.01.06.24300890'> 10.1101/2024.01.06.24300890</a></p>
        <p class="abstract">Neutralising antibodies against the SARS-CoV-2 spike (S) protein are major determinants of protective immunity, though insufficient antibody responses may cause the emergence of escape mutants. We studied the intra-host evolution and the humoral immune response in a B-cell depleted, haemato-oncologic patient experiencing clinically severe, prolonged SARS-CoV-2 infection with a virus of lineage B.1.177.81.

Bamlanivimab treatment early in infection was associated with the emergence of the bamlanivimab escape mutation S:S494P. Ten days before virus elimination, additional mutations within the N-terminal domain (NTD) and the receptor binding domain (RBD) of S were observed, of which the triple mutant S:Delta141-4 E484K S494P became dominant. Routine serology revealed no evidence of an antibody response in the patient. A detailed analysis of the variant-specific immune response by pseudotyped virus neutralisation test (pVNT), surrogate VNT (sVNT), and immunoglobulin-capture EIA showed that the onset of an IgM-dominated antibody response coincided with the appearance of escape mutations. The formation of neutralising antibodies against S:Delta141-4 E484K S494P correlated with virus elimination. One year later, the patient experienced clinically mild re-infection with Omicron BA.1.18, which was treated with sotrovimab and resulted in a massive increase of Omicron-reactive antibodies.

In conclusion, the onset of an IgM-dominated endogenous immune response in an immunocompromised patient coincided with the appearance of additional mutations in the NTD and RBD of S in a bamlanivimab-resistant virus. Although virus elimination was ultimately achieved, this humoral immune response escaped detection by routine diagnosis and created a situation temporarily favouring the emergence of escape variants with known epidemiological relevance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.23300575">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.23300575" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.23300575">
        <p class="paperTitle">Reverse mutational scanning of spike BA.2.86 identifies the epitopes contributing to immune escape from polyclonal sera</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.23300575" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.23300575" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bdeir, N.; Lüddecke, T.; Maass, H.; Schmelz, S.; Jacobsen, H.; Metzdorf, K.; Cossmann, A.; Stankov, M.; Hoffmann, M.; Pöhlmann, S.; Blankenfeldt, W.; Jablonka, A.; Behrens, G. M. N.; Cicin-Sain, L.</p>
        <p class="info">Score: 22.0, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.23300575' target='https://doi.org/10.1101/2024.01.03.23300575'> 10.1101/2024.01.03.23300575</a></p>
        <p class="abstract">The recently detected Omicron BA.2.86 lineage contains more than 30 amino acid mutations relative to BA.2. Here, we identify the epitopes driving immune escape of BA.2.86 and its derivative JN.1 (BA.2.86 &#43; S455L) lineage. We investigated the cross-reactive humoral immunity within a cohort of health care workers against Omicron BA.2.86 and JN.1 by employing pseudo-viral mutants. We demonstrate that BA.2.86 and especially JN.1 evaded neutralization by serum antibodies of fully vaccinated individuals. To discern the contribution of individual epitope mutations to immune escape, we constructed a library of 33 BA.2.86 mutants, each of which harbored a single revertant mutation going back to BA.2. This library was used in a reverse mutational scanning approach to define serum neutralization titers against each epitope separately. The mutations within the receptor binding domain (RBD) at position K356T and to a lesser extent the mutations N460K, V483{Delta}, A484K, and F486P enhanced the immune escape. More surprisingly, the mutation 16insMPLF within the spike N-terminal domain (NTD) and the mutation P621S in S1/S2 significantly contributed to antibody escape of BA.2.86. Upon XBB.1.5 booster vaccination, neutralization titers against JN.1 and BA.2.86 improved relative to all ancestral strains, and the residual immune escape was driven by mutations at positions 16insMPLF, {Delta}144Y, E544K, P621S, and A484K.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.24300790">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.24300790" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.24300790">
        <p class="paperTitle">Escalating combinations of enhanced infectivity and immune escape define SARS-CoV-2 Omicron lineage replacement</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.24300790" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.24300790" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, N. F. G.; Pham, K.; Chaguza, C.; Lopes, R.; Klaassen, F.; Weinberger, D. M.; Pitzer, V. E.; Warren, J. L.; Grubaugh, N. D.; Hahn, A. M.</p>
        <p class="info">Score: 7.3, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.24300790' target='https://doi.org/10.1101/2024.01.03.24300790'> 10.1101/2024.01.03.24300790</a></p>
        <p class="abstract">In 2022, consecutive sweeps of the highly transmissible SARS-CoV-2 Omicron-family maintained high viral transmission levels despite extensive antigen exposure on the population level resulting from both vaccinations and infections. To better understand variant fitness in the context of the highly dynamic immunity landscape of 2022, we aimed to dissect the interplay between immunity and fitness advantages of emerging SARS-CoV-2 Omicron lineages on the population-level. We evaluated the relative contribution of higher intrinsic transmissibility or immune escape on the fitness of emerging lineages by analyzing data collected through our local genomic surveillance program from Connecticut, USA. We compared growth rates, estimated infections, effective reproductive rates, average viral copy numbers, and likelihood for causing vaccine break-through infections. Using these population-level data, we find that newly emerging Omicron lineages reach dominance through a specific combination of enhanced intrinsic transmissibility and immune escape that varies over time depending on the state of the host-population. Using similar frameworks that integrate whole genome sequencing together with clinical, laboratory, and epidemiological data can advance our knowledge on host-pathogen dynamics in the post-emergence phase that can be applied to other communicable diseases beyond SARS-CoV-2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.23300577">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.23300577" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.23300577">
        <p class="paperTitle">Diverse and weakly immunogenic var gene expression facilitates malaria infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.23300577" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.23300577" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bhardwaj, I.; Nyarko, P. B.; Ashn, A. B.; Cohen, C.; Ceesay, S.; Achan, J.; Dabira, E.; Nakajima, R.; Jain, A.; Taghavian, O.; Jasinskas, A.; Felgner, P. L.; D&#39;Alessandro, U.; Bousema, T.; Travassos, M.; Radulescu, O.; Claessens, A.</p>
        <p class="info">Score: 5.2, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.23300577' target='https://doi.org/10.1101/2023.12.27.23300577'> 10.1101/2023.12.27.23300577</a></p>
        <p class="abstract">Plasmodium falciparum is believed to escape immunity via antigenic variation, mediated in part by 60 var genes. These genes undergo mutually exclusive expression and encode the PfEMP1 surface antigen. The frequency of var switching and the immunogenicity of each expressed PfEMP1 remain unclear. To this end, we carried out a Controlled Human Malaria Infection (CHMI) study with 19 adult African volunteers in The Gambia to gain insight into the effect of naturally acquired immunity on the expressed var gene repertoire during early phase of an infection. Our findings demonstrated a strong correlation between the diversity of var expression, quantified through entropy, and infection outcome. Low-immunity individuals were characterised by high var entropy profiles, higher parasitaemia, and lower sero-recognised PfEMP1 domains compared to high-immunity individuals. For the first time we recorded the probability of var gene switching in vitro and of turnover in vivo, enabling us to estimate both intrinsic switching and negative-selection effects. These processes are rapid, resulting in estimated turnover/switching probabilities of 69% - 97% and 7% - 57% per generation, in vivo and in vitro, respectively. Var (PfEMP1) expression triggered time-dependent humoral immune responses in low immunity individuals, with many PfEMP1 domains remaining weakly immunogenic. We conclude that the role of intrinsic var switching is to reset and maintain a diverse var repertoire. The high var switching rates and weak PfEMP1 immunogenicity benefit parasite survival during the CHMI.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.23300584">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.23300584" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.23300584">
        <p class="paperTitle">Malaria species prevalence among asymptomatic individuals in four regions of Mainland Tanzania</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.23300584" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.23300584" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Popkin-Hall, Z. R.; Seth, M. D.; Madebe, R. A.; Budodo, R.; Bakari, C.; Francis, F.; Pereus, D.; Giesbrecht, D. J.; Mandara, C. I.; Mbwambo, D.; Aaron, S.; Lusasi, A.; Lazaro, S.; Bailey, J. A.; Juliano, J. J.; Gutman, J. R.; Ishengoma, D. S.</p>
        <p class="info">Score: 3.8, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.23300584' target='https://doi.org/10.1101/2023.12.28.23300584'> 10.1101/2023.12.28.23300584</a></p>
        <p class="abstract">Recent studies point to the need to incorporate non-falciparum species detection into malaria surveillance activities in sub-Saharan Africa, where 95% of malaria cases occur. Although Plasmodium falciparum infection is typically more severe, diagnosis, treatment, and control for P. malariae, P. ovale spp., and P. vivax may be more challenging. The prevalence of these species throughout sub-Saharan Africa is poorly defined. Tanzania has geographically heterogeneous transmission levels but an overall high malaria burden. In order to estimate the prevalence of malaria species in Mainland Tanzania, 1,428 samples were randomly selected from 6,005 asymptomatic isolates collected in cross-sectional community surveys across four regions and analyzed via qPCR to detect each Plasmodium species. P. falciparum was most prevalent, with P. malariae and P. ovale spp. detected at lower prevalence (&lt;5%) in all four regions. P. vivax was not detected. Malaria elimination efforts in Tanzania will need to account for these non-falciparum species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.24300703">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.24300703" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.24300703">
        <p class="paperTitle">Thoughts and perspectives of metagenome sequencing as a diagnostic tool for infectious disease: an interpretive phenomenological study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.24300703" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.24300703" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Trivett, H.; Darby, A. C.; Oyebode, O.</p>
        <p class="info">Score: 3.2, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.24300703' target='https://doi.org/10.1101/2024.01.02.24300703'> 10.1101/2024.01.02.24300703</a></p>
        <p class="abstract">BackgroundEffective infectious disease diagnostics (IDD) are vital for informing clinical decision-making regarding the treatment and patient management of disease and infections. Conventional clinical methods rely upon culture-dependent techniques, and there has been little shift in the acceptance and integration of culture-independent sequencing methods into routine clinical IDD. This study explored stakeholders experiences within IDD, including those working in clinical settings and those conducting research at the forefront of microbial genomics. We aimed to identify factors driving the development and implementation of metagenome sequencing as a routine diagnostic.

MethodsVirtual semi-structured interviews were conducted with purposively selected individuals involved in IDD. The interviews explored the experiences of implementing metagenome sequencing as a diagnostic tool and decisions about which diagnostics are used for identifying bacteria-causing infections. Thematic analysis was used to analyse the data, and an Interpretive Phenomenological approach was used throughout.

ResultsTen individuals were interviewed between July 2021 and October 2021, including Clinical scientists, consultants, and professors in academia. Their experience ranged from no knowledge of metagenome sequencing to an expert understanding of the phenomenon. Five themes emerged: Diagnostic Choice, Infrastructure, Open Data Sharing, COVID-19, and Communication. Participants recognised the need for new diagnostics to be implemented to overcome the limitations of current diagnostic approaches but highlighted the barriers to integrating new diagnostics into clinical settings, such as the impact on clinical decision-making, accreditation, and cost. However, participants felt that lessons could be learnt from using metagenomics in COVID-19 and how other diagnostic platforms have been integrated into clinical settings over the last 20 years.

ConclusionsThe study provided clear evidence to address the knowledge gap in current literature and practice for developing and implementing metagenome sequencing as a potential IDD. The knowledge of new and upcoming genomic diagnostic testing is not equally distributed throughout the UK, impacting the understanding and drive to integrate metagenome sequencing into routine clinical diagnostics. Improvements in access to new diagnostics could improve patient treatment and management and positively impact public health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.24300729">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.24300729" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.24300729">
        <p class="paperTitle">Trends in SARS-CoV-2 seroprevalence among pregnant women attending first antenatal care visits in Zambia: a repeated cross-sectional survey, 2021-2022</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.24300729" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.24300729" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heilmann, E.; Tembo, T.; Fwoloshi, S.; Kabamba, B.; Chilambe, F.; Kalenga, K.; Siwingwa, M.; Mulube, C.; Seffren, V.; Bolton-Moore, C.; Simwanza, J.; Yingst, S.; Yadav, R.; Rogier, E.; Auld, A. F.; Agolory, S.; Kapina, M.; Gutman, J. R.; Savory, T.; Kangale, C.; Mulenga, L. B.; Sikazwe, I.; Hines, J. Z.</p>
        <p class="info">Score: 2.5, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.24300729' target='https://doi.org/10.1101/2024.01.02.24300729'> 10.1101/2024.01.02.24300729</a></p>
        <p class="abstract">BackgroundSARS-CoV-2 serosurveys help estimate the extent of transmission and guide allocation of COVID-19 vaccines. We measured SARS-CoV-2 seroprevalence among women attending ANC clinics to assess exposure trends over time in Zambia.

MethodsWe conducted repeated cross-sectional surveys among pregnant women aged 15-49 years attending their first ANC visits in four districts of Zambia (two urban and two rural) during September 2021-September 2022. Serologic testing was done using a multiplex bead assay which detects IgG antibodies to the nucleocapsid protein and the spike protein receptor-binding domain (RBD). We calculated monthly SARS-CoV-2 seroprevalence by district. We also categorized seropositive results as infection alone, infection and vaccination, or vaccination alone based on COVID-19 vaccination status and anti-RBD and anti-nucleocapsid test results.

FindingsAmong 8,304 participants, 5,296 (63.8%) were cumulatively seropositive for SARS-CoV-2 antibodies. SARS-CoV-2 seroprevalence primarily increased from September 2021 to September 2022 in three districts (Lusaka: 61.8-100.0%, Chongwe: 39.6-94.7%, Chipata: 56.5-95.0%), but in Chadiza, seroprevalence increased from 27.8% in September 2021 to 77.2% in April 2022 before gradually dropping to 56.6% in July 2022. Among 5,906 participants with a valid COVID-19 vaccination status, infection alone accounted for antibody responses in 77.7% (4,590) of participants.

InterpretationMost women attending ANC had evidence of prior SARS-CoV-2 infection and most SARS-CoV-2 seropositivity was infection-induced. Capturing COVID-19 vaccination status and using a multiplex bead assay with anti-nucleocapsid and anti-RBD targets facilitated distinguishing infection-induced versus vaccine-induced antibody responses during a period of increasing COVID-19 vaccine coverage in Zambia. Declining seroprevalence in Chadiza may indicate waning antibodies and a need for booster vaccines. ANC clinics have a potential role in ongoing SARS-CoV-2 serosurveillance and can continue to provide insights into SARS-CoV-2 antibody dynamics to inform near real-time public health responses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.07.24300955">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.07.24300955" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.07.24300955">
        <p class="paperTitle">Phylogeography and reassortment patterns of human influenza A viruses in sub-Saharan Africa</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.07.24300955" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.07.24300955" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Owuor, D. C.; de Laurent, Z. R.; Oketch, J. W.; Murunga, N.; Otieno, J. R.; Chaves, S. S.; Nokes, J.; Agoti, C. N.</p>
        <p class="info">Score: 2.1, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.07.24300955' target='https://doi.org/10.1101/2024.01.07.24300955'> 10.1101/2024.01.07.24300955</a></p>
        <p class="abstract">Background. The role of sub-Saharan Africa in the global spread of influenza viruses remains unclear due to insufficient spatiotemporal sequence data. Methods. Here, we analyzed 222 codon-complete sequences of influenza A viruses (IAVs) sampled between 2011 and 2013 from five countries across sub-Saharan Africa (Kenya, Zambia, Mali, Gambia, and South Africa); these genomes were compared with 1,209 contemporaneous global genomes using phylogeographical approaches. Results. The spread of influenza in sub-Saharan Africa was characterized by (i) multiple introductions of IAVs into the region over consecutive influenza seasons, with viral importations originating from multiple global geographical regions, some of which persisted in circulation as intra-subtype reassortants for multiple seasons, (ii) virus transfer between sub-Saharan African countries, and (iii) virus export from sub-Saharan Africa to other geographical regions. Conclusion. Despite sparse data from influenza surveillance in sub-Saharan Africa, our findings support the notion that influenza viruses persist as temporally structured migrating metapopulations in which new virus strains can emerge in any geographical region, including in sub-Saharan Africa; these lineages may have been capable of dissemination to other continents through a globally migrating virus population. Further knowledge of the viral lineages that circulate within understudied sub-Saharan Africa regions is required to inform vaccination strategies in those regions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.23300149">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.23300149" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.23300149">
        <p class="paperTitle">Beyond Borders: Spatial Disparities in the Mortality Burden of the Covid-19 pandemic across 569 European Regions (2020-2021)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.23300149" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.23300149" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bonnet, F.; Grigoriev, P.; Sauerberg, M.; Alliger, I.; Muhlichen, M.; Camarda, C.-G.</p>
        <p class="info">Score: 34.5, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.23300149' target='https://doi.org/10.1101/2023.12.18.23300149'> 10.1101/2023.12.18.23300149</a></p>
        <p class="abstract">This article presents a detailed analysis of the global mortality burden of the COVID-19 pandemic across 569 regions in 25 European countries. We produce sex-specific excess mortality and present our results using Age-Standardised Years of Life Lost (ASYLL) in 2020 and 2021, as well as the cumulative impact over the two pandemic years. Employing a robust forecasting approach that considers regional diversity and provides confidence intervals, we find notable losses in 362 regions in 2020 (440 regions in 2021). Conversely, only seven regions experienced gains in 2020 (four regions in 2021). Most importantly, we estimate that eight regions suffered losses exceeding 20 years of life per 1,000 population in 2020, whereas this number increased to 75 regions in 2021. The contiguity of the regions investigated in our study also reveals the changing geographical patterns of the pandemic. While the highest excess mortality values were concentrated in the early COVID-19 outbreak areas during the initial pandemic year, a clear East-West gradient appeared in 2021, with regions of Slovakia, Hungary, and Latvia experiencing the highest losses. This research underscores the importance of regional analyses for a nuanced comprehension of the pandemics impact.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
