<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Early Biological Markers of Post-Acute Sequelae of SARS-CoV-2 Infection
Authors: Lu, S.; Peluso, M. J.; Glidden, D. V.; Davidson, M.; Lugtu, K.; Pineda-Ramirez, J.; Tassetto, M.; Garcia-Knight, M.; Zhang, A.; Goldberg, S. A.; Chen, J.; Forbes, M.; Park, S.; Martinez, A.; So, M.; Donovan, A.; Viswanathan, B.; Hoh, R.; Donohue, K.; McIlwain, D.; Gaudilliere, B.; Anglin, K.; Yee, B.; Chenna, A. A.; Winslow, J.; Petropoulos, C.; Briggs-Hagen, M.; Andino, R.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Early Biological Markers of Post-Acute Sequelae of SARS-CoV-2 Infection
Authors: Lu, S.; Peluso, M. J.; Glidden, D. V.; Davidson, M.; Lugtu, K.; Pineda-Ramirez, J.; Tassetto, M.; Garcia-Knight, M.; Zhang, A.; Goldberg, S. A.; Chen, J.; Forbes, M.; Park, S.; Martinez, A.; So, M.; Donovan, A.; Viswanathan, B.; Hoh, R.; Donohue, K.; McIlwain, D.; Gaudilliere, B.; Anglin, K.; Yee, B.; Chenna, A. A.; Winslow, J.; Petropoulos, C.; Briggs-Hagen, M.; Andino, R." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-26T10:37:30+00:00" />
<meta property="article:modified_time" content="2023-07-26T10:37:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Early Biological Markers of Post-Acute Sequelae of SARS-CoV-2 Infection
Authors: Lu, S.; Peluso, M. J.; Glidden, D. V.; Davidson, M.; Lugtu, K.; Pineda-Ramirez, J.; Tassetto, M.; Garcia-Knight, M.; Zhang, A.; Goldberg, S. A.; Chen, J.; Forbes, M.; Park, S.; Martinez, A.; So, M.; Donovan, A.; Viswanathan, B.; Hoh, R.; Donohue, K.; McIlwain, D.; Gaudilliere, B.; Anglin, K.; Yee, B.; Chenna, A. A.; Winslow, J.; Petropoulos, C.; Briggs-Hagen, M.; Andino, R."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Early Biological Markers of Post-Acute Sequelae of SARS-CoV-2 Infection\nAuthors: Lu, S.; Peluso, M. J.; Glidden, D. V.; Davidson, M.; Lugtu, K.; Pineda-Ramirez, J.; Tassetto, M.; Garcia-Knight, M.; Zhang, A.; Goldberg, S. A.; Chen, J.; Forbes, M.; Park, S.; Martinez, A.; So, M.; Donovan, A.; Viswanathan, B.; Hoh, R.; Donohue, K.; McIlwain, D.; Gaudilliere, B.; Anglin, K.; Yee, B.; Chenna, A. A.; Winslow, J.; Petropoulos, C.; Briggs-Hagen, M.; Andino, R.",
  "keywords": [
    
  ],
  "articleBody": " Early Biological Markers of Post-Acute Sequelae of SARS-CoV-2 Infection\nAuthors: Lu, S.; Peluso, M. J.; Glidden, D. V.; Davidson, M.; Lugtu, K.; Pineda-Ramirez, J.; Tassetto, M.; Garcia-Knight, M.; Zhang, A.; Goldberg, S. A.; Chen, J.; Forbes, M.; Park, S.; Martinez, A.; So, M.; Donovan, A.; Viswanathan, B.; Hoh, R.; Donohue, K.; McIlwain, D.; Gaudilliere, B.; Anglin, K.; Yee, B.; Chenna, A. A.; Winslow, J.; Petropoulos, C.; Briggs-Hagen, M.; Andino, R.; Midgley, C.; Martin, J. N.; Saydeh, S.; Kelly, J. D.\nScore: 314.0, Published: 2023-07-16 DOI: 10.1101/2023.07.14.23292649\nTo understand the roles of acute phase viral dynamics and host immune responses in PASC, we enrolled 136 participants within 5 days of their first positive SARS-CoV-2 real-time PCR. Participants self-collected nasal specimens up to 21 times within the first 28 days after symptom onset; Interviewer-administered clinical questionnaires and blood samples were collected at enrollment and days 9, 14, 21, 28, and month 4 and 8 post-symptom. Defining PASC as the presence of any symptom new or worse since infection reported at their 4-month visit, we compared viral markers (quantity and duration of viral RNA load, infectious viral load, and plasma N-antigen level) and host immune markers (IL-6, IL-10, TNF-, IFN-, IFN-{gamma}, MCP, IP-10, and Spike IgG) over the acute period. In comparison to those who fully recovered, those who developed PASC demonstrated significantly higher maximum levels of SARS-CoV-2 RNA, infectious virus, and N-antigen, longer duration of viral shedding, and lower Spike-specific IgG levels within the first 10 days of the acute phase of illness. No significant differences were identified among a panel of host immune markers, though there was a trend toward higher initial levels of certain markers (e.g., MCP-1, IFN-, and IFN-{gamma}) in those who went on to develop PASC. Early viral dynamics and the associated host immune responses play a role in the pathogenesis of PASC. These findings highlight the importance of understanding the early biological markers from acute SARS-CoV-2 infection in the natural history of PASC. Onset Sentence SummaryEarly viral dynamics and the associated host immune responses play a role in the pathogenesis of PASC.\nComparison of the analytical and clinical sensitivity of thirty-four rapid antigen tests with the most prevalent SARS-CoV-2 variants of concern during the COVID-19 pandemic in the UK\nAuthors: Byrne, R. L.; Aljayyoussi, G.; Greenland - Bews, C.; Kontogianni, K.; Wooding, D.; Williams, C. T.; de Vos, M.; Body, R.; Adams, E. R.; Escadafel, C.; Edwards, T. R.; Cubas Atienzar, A. I.\nScore: 69.8, Published: 2023-07-24 DOI: 10.1101/2023.07.24.23293072\nBackground The continued emergence of SARS-CoV-2 variants of concern (VOC) requires timely analytical and clinical evaluation of antigen-based rapid diagnostic tests (Ag-RDTs) especially those that are recommended for at home use. Methods The limit of detection (LOD) of 34 Ag-RDTs was evaluated using the most encountered SARS-CoV-2 VOC viral isolates (Alpha, Delta, Gamma, Omicron BA.1, Omicron BA.5) and the wild type (WT). Clinical sensitivity was further evaluated for five Ag-RDT utilising retrospective samples (Alpha, Delta, Omicron BA.1) and one Ag-RDT utilising prospective clinical samples (Delta and Omicron BA.1). Findings For the WT, Alpha, Delta, Gamma and Omicron (BA.1) variants 22, 32, 29, 31 and 32 of the 34 Ag-RDTs evaluated met the World Health Organisations (WHO) target product profile (TPP), respectively. Of the 31 Ag-RDTs included for Omicron BA.5 evaluation 29 met the WHO TPP. Additionally, the LODs for samples spiked with Omicron BA.5 were significantly lower than all other VOCs included (p\u003c0.001). In the retrospective clinical evaluation when comparing RNA copies/mL, the Ag-RDTs detected Alpha and Omicron (BA.1) more sensitively than the Delta VOC. Samples with high RT-qPCR Cts (Ct\u003e25) resulted in reduced test sensitivities across all variants. We used linear regression to model the 50% and 95% LOD of clinical samples and observed statistically similar results for all tests. In the prospective clinical samples, the sensitivity was statistically similar for the Delta VOC 71.9% (CI 95% 53.3-86.6%) and Omicron VOC 84.4% (CI95% 75.3-91.2%). Interpretation Test performance differs between SARS-CoV-2 VOCs, and high sensitivity was achieved when testing the Omicron BA.5 VOC compared to the WHO Ag-RDT requirements. Continuous evaluations must be performed to monitor test performance.\nRapid Identification of Bacterial isolates Using Microfluidic Adaptive Channels and Multiplexed Fluorescence Microscopy\nAuthors: Chatzimichail, S.; Turner, P. B.; Feehily, C.; Farrar, A. E.; Crook, D.; Andersson, M.; Oakley, S.; Barrett, L.; El Sayyed, H.; Kyropoulos, J.; Nellaker, C.; Stoesser, N.; Kapanidis, A. N.\nScore: 17.0, Published: 2023-07-16 DOI: 10.1101/2023.07.16.23292615\nWe demonstrate the rapid capture, enrichment, and identification of bacterial pathogens using Adaptive Channel Bacterial Capture (ACBC) devices. Using controlled tuning of device backpressure in polydimethylsiloxane (PDMS) devices, we enable the controlled formation of capture regions capable of trapping bacteria from low cell density samples with near 100% capture efficiency. The technical demands to prepare such devices are much lower compared to conventional methods for bacterial trapping and can be achieved with simple benchtop fabrication methods. We demonstrate the capture and identification of seven species of bacteria with bacterial concentrations lower than 1000 cells/mL, including common Gram-negative and Gram-positive pathogens such as Escherichia coli and Staphylococcus aureus. We further demonstrate that species identification of the trapped bacteria can be undertaken in the order of one-hour using multiplexed 16S rRNA-FISH with identification accuracies of 73-99% with unsupervised classification methods.\nIntranasal lavage with hypochlorous acid safely reduces the symptoms in the ambulatory patient with COVID-19.\nAuthors: Abner, M. L.\nScore: 6.5, Published: 2023-07-18 DOI: 10.1101/2023.07.17.23292426\nOBJECTIVEThis study was designed to investigate intranasal lavage with a hypochlorous acid solution in the reduction of symptoms in the ambulatory COVID-19 patient. STUDY DESIGNStudy approval granted by the Institutional Review Board of Reading Hospital (IRB 036-20), with informed consent obtained from all adult participants(age\u003e18 years). SETTINGAll enrollees, taken from the same ambulatory testing facility, received nasopharyngeal swabs for COVID-19 testing by reverse transcription polymerase chain (RT-PCR) or the COVID-19 antigen specific test (Binax NOW, Abbott Lab) METHODSConvenience sampling methodology was utilized. Each enrollee was provided with the study devices which included a Nasaflo Neti Pot (NeilMed Pharmaceutical, Inc.), and the hypochlorous acid solution (Vashe Wound Solution, Urgo Medical North America, LLC). Participants were instructed to irrigate each nostril with 120 cc (four ounces) of the solution for ten consecutive days, and record the presence or absence of symptoms in a scripted diary log. RESULTSThe study included 88 patients of which 74 (84.1%) completed the ten days of nasal lavage. All data analysis was conducted using SPSS version 25.0. Chi square test of association found no significant difference related to gender, age group race, ethnicity, residence, or living arrangements (all p-values \u003e 0.05). There were no statistical differences in any of the co-morbid conditions. Mild adverse reactions included burning, epistaxis, and oral metallic taste. No enrollees required mechanical ventilation. There were no deaths. CONCLUSIONThis study suggests the feasibility and safety of using intranasal lavage with a hypochlorous acid solution in relieving symptoms in the ambulatory Covid-19 patient.\nIncubation-period estimates of Omicron (BA.1) variant from Taiwan, December 2021-January 2022, and its comparison to other SARS-CoV-2 variants: a statistical modeling, systematic search and meta-analysis\nAuthors: Akhmetzhanov, A. R.; Cheng, H.-Y.; Dushoff, J.\nScore: 5.8, Published: 2023-07-24 DOI: 10.1101/2023.07.20.23292983\nBackground: The ongoing COVID-19 pandemic has seen several variants of concern, including the Omicron (BA.1) variant which emerged in October 2021. Accurately estimating the incubation period of these variants is crucial for predicting disease spread and formulating effective public health strategies. However, existing estimates often conflict because of biases arising from the dynamic nature of epidemic growth and selective inclusion of cases. This study aims to accurately estimate of the Omicron (BA.1) variant incubation period based on data from Taiwan, where disease incidence remained low and contact tracing was comprehensive during the first months of the Omicron outbreak. Methods: We reviewed 100 contact-tracing records for cases of the Omicron BA.1 variant reported between December 2021 and January 2022, and found enough information to analyze 70 of these. The incubation period distribution was estimated by fitting data on exposure and symptom onset within a Bayesian mixture model using gamma, Weibull, and lognormal distributions as candidates. Additionally, a systematic literature search was conducted to accumulate data for estimates of the incubation period for Omicron (BA.1/2, BA.4/5) subvariants, which was then used for meta-analysis and comparison. Results: The mean incubation period was estimated at 3.5 days (95% credible interval: 3.1-4.0 days), with no clear differences when stratified by vaccination status or age. This estimate aligns closely with the pooled mean of 3.4 days (3.0-3.8 days) obtained from a meta-analysis of other published studies on Omicron subvariants. Conclusions: The relatively shorter incubation period of the Omicron variant, as compared to previous SARS-CoV2 variants, implies its potential for rapid spread but also opens the possibility for individuals to voluntarily adopt shorter, more resource-efficient quarantine periods. Continual updates to incubation period estimates, utilizing data from comprehensive contact tracing, are crucial for effectively guiding these voluntary actions and adjusting high socio-economic cost interventions.\nSARS-CoV-2 virologic rebound with nirmatrelvir-ritonavir therapy\nAuthors: Edelstein, G. E.; Boucau, J.; Uddin, R.; Marino, C.; Liew, M. Y.; Barry, M.; Choudhary, M. C.; Gilbert, R. F.; Reynolds, Z.; Li, Y.; Tien, D.; Sagar, S.; Vyas, T. D.; Kawano, Y.; Sparks, J. A.; Hammond, S. P.; Wallace, Z.; Vyas, J. M.; Barczak, A. K.; Lemieux, J. E.; Li, J. Z.; Siedner, M. J.\nScore: 176.4, Published: 2023-06-27 DOI: 10.1101/2023.06.23.23288598\nObjectiveTo compare the frequency of replication-competent virologic rebound with and without nirmatrelvir-ritonavir treatment for acute COVID-19. Secondary aims were to estimate the validity of symptoms to detect rebound and the incidence of emergent nirmatrelvir-resistance mutations after rebound. DesignObservational cohort study. SettingMulticenter healthcare system in Boston, Massachusetts. ParticipantsWe enrolled ambulatory adults with a positive COVID-19 test and/or a prescription for nirmatrelvir-ritonavir. ExposuresReceipt of 5 days of nirmatrelvir-ritonavir treatment versus no COVID-19 therapy. Main Outcome and MeasuresThe primary outcome was COVID-19 virologic rebound, defined as either (1) a positive SARS-CoV-2 viral culture following a prior negative culture or (2) two consecutive viral loads [\u0026ge;]4.0 log10 copies/milliliter after a prior reduction in viral load to \u003c4.0 log10 copies/milliliter. ResultsCompared with untreated individuals (n=55), those taking nirmatrelvir-ritonavir (n=72) were older, received more COVID-19 vaccinations, and were more commonly immunosuppressed. Fifteen individuals (20.8%) taking nirmatrelvir-ritonavir experienced virologic rebound versus one (1.8%) of the untreated (absolute difference 19.0% [95%CI 9.0-29.0%], P=0.001). In multivariable models, only N-R was associated with VR (AOR 10.02, 95%CI 1.13-88.74). VR occurred more commonly among those with earlier nirmatrelvirritonavir initiation (29.0%, 16.7% and 0% when initiated days 0, 1, and [\u0026ge;]2 after diagnosis, respectively, P=0.089). Among participants on N-R, those experiencing rebound had prolonged shedding of replication-competent virus compared to those that did not rebound (median: 14 vs 3 days). Only 8/16 with virologic rebound reported worsening symptoms (50%, 95%CI 25%-75%); 2 were completely asymptomatic. We detected no post-rebound nirmatrelvir-resistance mutations in the NSP5 protease gene. Conclusions and RelevanceVirologic rebound occurred in approximately one in five people taking nirmatrelvir-ritonavir and often occurred without worsening symptoms. Because it is associated with replication-competent viral shedding, close monitoring and potential isolation of those who rebound should be considered.\nMucosal and Systemic Immune Correlates of Viral Control following SARS-CoV-2 Infection Challenge in Seronegative Adults\nAuthors: Wagstaffe, H. R.; Thwaites, R. S.; Reynaldi, A.; Sidhu, J. K.; McKendry, R.; Ascough, S.; Papargyris, L.; Collins, A. M.; Xu, J.; Lemm, N.-M.; Siggins, M. K.; Chain, B. M.; Killingley, B.; Kalinova, M.; Mann, A.; Catchpole, A.; Davenport, M. P.; Openshaw, P. J. M.; Chiu, C.\nScore: 4.5, Published: 2023-07-23 DOI: 10.1101/2023.07.21.23292994\nHuman infection challenge permits characterisation of the associated immune response in unparalleled depth, enabling evaluation of early pre-symptomatic immune changes and the dynamic immune factors important for viral clearance. Here, 34 healthy young adult volunteers, seronegative to SARS-CoV-2, were inoculated with a D614G-containing pre-Alpha SARS-CoV-2 strain. Nasal and systemic soluble mediator and antibody responses, and peripheral blood T cell and B cell responses were measured by MesoScale Discovery and flow cytometry just before and up to 1 year after intra-nasal inoculation. In the 18 (53%) participants who became infected, both nasal and systemic mediator responses were dominated by interferons (IFN) but with divergent kinetics. T cell activation and proliferation in blood peaked at day 10 in CD4+ T cells and day 14 in CD8+ T cells, returning to baseline by day 28. Following infection, antigen-specific T cells were largely CD38+Ki67+ and displayed central and effector memory phenotypes. T cells contracted after viral clearance with expanded antigen-specific memory T cell populations persisting past day 28. Both mucosal and systemic antibodies became detectable around day 10 but nasal antibodies plateaued after day 14 while circulating antibodies continued to rise. Using piecewise linear regression modelling, viral load related closely to the induction of type I IFN responses, moreover, CD8+ T cell responses and early IgA responses were strongly associated with viral clearance. Detailed analysis of innate and adaptive immune responses to primary SARS-CoV-2 infection following human challenge thus revealed the relationship between immune kinetics and viral load as factors associated with resolution of infection.\nRisk of SARS-CoV-2 infection and hospitalization in individuals with natural, vaccine-induced and hybrid immunity: a retrospective population-based cohort study from Estonia\nAuthors: Uuskula, A.; Pisarev, H.; Tisler, A.; Meister, T.; Suija, K.; Huik, K.; Abroi, A.; Kalda, R.; Kolde, R.; Fischer, K.\nScore: 3.9, Published: 2023-07-23 DOI: 10.1101/2023.07.18.23292858\nA large proportion of the worlds population has some form of immunity against SARS-CoV- 2, through either infection ( natural), vaccination or both ( hybrid). This retrospective cohort study used data on SARS-CoV-2, vaccination, and hospitalization from national health system from February 2020 to June 2022 and Cox regression modelling to compare those with natural immunity to those with no (Cohort1, n=92917), hybrid (Cohort2, n=46813), and vaccine (Cohort3, n=252414) immunity. In Cohort 1, those with natural immunity were at lower risk for infection during the Delta (aHR 0.17, 95%CI 0.15-0.18) and higher risk (aHR 1.24, 95%CI 1.18-1.32) during the Omicron period than those with no immunity. Natural immunity conferred substantial protection against COVID-19-hospitalization. Cohort 2 - in comparison to natural immunity hybrid immunity offered strong protection during the Delta (aHR 0.61, 95%CI 0.46-0.80) but not the Omicron (aHR 1.05, 95%CI 0.93-1.1) period. COVID-19-hospitalization was extremely rare among individuals with hybrid immunity. In Cohort 3, individuals with vaccine-induced immunity were at higher risk than those with natural immunity for infection (Delta aHR 4.90, 95%CI 4.48-5.36; Omicron 1.13, 95%CI 1.06-1.21) and hospitalization (Delta aHR 7.19, 95%CI 4.02-12.84). These results show that risk of infection and severe COVID-19 are driven by personal immunity history and the variant of SARS-CoV-2 causing infection.\nIncreased circulating fibronectin, depletion of natural IgM and heightened EBV, HSV-1 reactivation in ME/CFS and long COVID\nAuthors: Liu, Z.; Hollmann, C.; Kalanidhi, S.; Grothey, A.; Keating, S.; Mena-Palomo, I.; Lamer, S.; Schlosser, A.; Kaiping, A.; Scheller, C.; Sotzny, F.; Horn, A.; Nuernberger, C.; Cejka, V.; Afshar, B.; Bahmer, T.; Schreiber, S.; Janne Vehreschild, J.; Milljukov, O.; Schaefer, C.; Kretzler, L.; Keil, T.; Reese, J.-P.; Eichner, F. A.; Schmidbauer, L.; Heuschmann, P. U.; Stoerk, S.; Morbach, C.; Riemekasten, G.; Beyersdorf, N.; Scheibenbogen, C.; Naviaux, R. K.; Williams, M.; Ariza, M. E.; Prusty, B. K.\nScore: 474.2, Published: 2023-06-29 DOI: 10.1101/2023.06.23.23291827\nMyalgic Encephalomyelitis/ Chronic Fatigue syndrome (ME/CFS) is a complex, debilitating, long-term illness without a diagnostic biomarker. ME/CFS patients share overlapping symptoms with long COVID patients, an observation which has strengthened the infectious origin hypothesis of ME/CFS. However, the exact sequence of events leading to disease development is largely unknown for both clinical conditions. Here we show antibody response to herpesvirus dUTPases, particularly to that of Epstein-Barr virus (EBV) and HSV-1, increased circulating fibronectin (FN1) levels in serum and depletion of natural IgM against fibronectin ((n)IgM-FN1) are common factors for both severe ME/CFS and long COVID. We provide evidence for herpesvirus dUTPases-mediated alterations in host cell cytoskeleton, mitochondrial dysfunction and OXPHOS. Our data show altered active immune complexes, immunoglobulin-mediated mitochondrial fragmentation as well as adaptive IgM production in ME/CFS patients. Our findings provide mechanistic insight into both ME/CFS and long COVID development. Finding of increased circulating FN1 and depletion of (n)IgM-FN1 as a biomarker for the severity of both ME/CFS and long COVID has an immediate implication in diagnostics and development of treatment modalities.\nHealth-seeking behaviour and beliefs around sore throat in The Gambia: a qualitative study\nAuthors: Suau Sans, M.; Manneh, M.; Ceesay, I.; Bittaye, A.; de Crombrugghe, G.; Keeley, A. J.; de Silva, T. I.; Palmer, J.; Armitage, E. P.; Marks, M.\nScore: 2.8, Published: 2023-07-23 DOI: 10.1101/2023.07.17.23292793\nGroup A Streptococcus (StrepA) bacteria causes a broad spectrum of diseases. The most common manifestations of StrepA infection are sore throat and pus-producing skin infections such as impetigo. Complications of StrepA infection lead to inflammation in the bones, muscles, joints, and internal organs causing acute rheumatic fever and rheumatic heart disease (RHD). In The Gambia, the RHD burden is thought to be very high. However, epidemiological data is minimal, and StrepA control programmes do not exist. This study aims to explore common beliefs and practices related to sore throats among primary caregivers of children, and healthcare providers in a community with a high StrepA disease burden. This will inform the design of preventative strategies for StrepA-related sequelae. Four informal conversations with providers and fifteen semi-structured interviews with caregivers were conducted in the peri-urban area of Sukuta, The Gambia. Sampling was purposive and gradual, beginning from households identified to have recently experienced sore throat through a parallel cohort study. Themes explored in qualitative analysis included: sore throat causal attributions and diagnoses, care practises, health-seeking behaviour, and perceived barriers to using the biomedical sector. Sore throats were typically perceived to affect one child in a family, disproportionately or exclusively. Sore throats are rarely perceived as life-threatening, and awareness of links between sore throat and ARF or RHD was not reported among caregivers or providers in this study population. Most cases of sore throat are initially managed at home using traditional medicine which delays resort to antibiotics, though in two instances of severe pain with the presence of exudate, fear that the childs life was at risk prompted care-seeking through the formal health system. Our findings can inform the development of tailored strategies to increase community knowledge of the potential long-term consequences of sore throats and appropriate care-seeking, alongside improvements in the health system, to prevent StrepA sequelae effectively.\n",
  "wordCount" : "3028",
  "inLanguage": "en",
  "datePublished": "2023-07-26T10:37:30Z",
  "dateModified": "2023-07-26T10:37:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta"><span>updated on July 26, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.23292649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.23292649" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.23292649">
        <p class="paperTitle">Early Biological Markers of Post-Acute Sequelae of SARS-CoV-2 Infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.23292649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.23292649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lu, S.; Peluso, M. J.; Glidden, D. V.; Davidson, M.; Lugtu, K.; Pineda-Ramirez, J.; Tassetto, M.; Garcia-Knight, M.; Zhang, A.; Goldberg, S. A.; Chen, J.; Forbes, M.; Park, S.; Martinez, A.; So, M.; Donovan, A.; Viswanathan, B.; Hoh, R.; Donohue, K.; McIlwain, D.; Gaudilliere, B.; Anglin, K.; Yee, B.; Chenna, A. A.; Winslow, J.; Petropoulos, C.; Briggs-Hagen, M.; Andino, R.; Midgley, C.; Martin, J. N.; Saydeh, S.; Kelly, J. D.</p>
        <p class="info">Score: 314.0, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.23292649' target='https://doi.org/10.1101/2023.07.14.23292649'> 10.1101/2023.07.14.23292649</a></p>
        <p class="abstract">To understand the roles of acute phase viral dynamics and host immune responses in PASC, we enrolled 136 participants within 5 days of their first positive SARS-CoV-2 real-time PCR. Participants self-collected nasal specimens up to 21 times within the first 28 days after symptom onset; Interviewer-administered clinical questionnaires and blood samples were collected at enrollment and days 9, 14, 21, 28, and month 4 and 8 post-symptom. Defining PASC as the presence of any symptom new or worse since infection reported at their 4-month visit, we compared viral markers (quantity and duration of viral RNA load, infectious viral load, and plasma N-antigen level) and host immune markers (IL-6, IL-10, TNF-, IFN-, IFN-{gamma}, MCP, IP-10, and Spike IgG) over the acute period. In comparison to those who fully recovered, those who developed PASC demonstrated significantly higher maximum levels of SARS-CoV-2 RNA, infectious virus, and N-antigen, longer duration of viral shedding, and lower Spike-specific IgG levels within the first 10 days of the acute phase of illness. No significant differences were identified among a panel of host immune markers, though there was a trend toward higher initial levels of certain markers (e.g., MCP-1, IFN-, and IFN-{gamma}) in those who went on to develop PASC. Early viral dynamics and the associated host immune responses play a role in the pathogenesis of PASC. These findings highlight the importance of understanding the early biological markers from acute SARS-CoV-2 infection in the natural history of PASC.

Onset Sentence SummaryEarly viral dynamics and the associated host immune responses play a role in the pathogenesis of PASC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.23293072">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.23293072" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.23293072">
        <p class="paperTitle">Comparison of the analytical and clinical sensitivity of thirty-four rapid antigen tests with the most prevalent SARS-CoV-2 variants of concern during the COVID-19 pandemic in the UK</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.23293072" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.23293072" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Byrne, R. L.; Aljayyoussi, G.; Greenland - Bews, C.; Kontogianni, K.; Wooding, D.; Williams, C. T.; de Vos, M.; Body, R.; Adams, E. R.; Escadafel, C.; Edwards, T. R.; Cubas Atienzar, A. I.</p>
        <p class="info">Score: 69.8, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.23293072' target='https://doi.org/10.1101/2023.07.24.23293072'> 10.1101/2023.07.24.23293072</a></p>
        <p class="abstract">Background The continued emergence of SARS-CoV-2 variants of concern (VOC) requires timely analytical and clinical evaluation of antigen-based rapid diagnostic tests (Ag-RDTs) especially those that are recommended for at home use. Methods The limit of detection (LOD) of 34 Ag-RDTs was evaluated using the most encountered SARS-CoV-2 VOC viral isolates (Alpha, Delta, Gamma, Omicron BA.1, Omicron BA.5) and the wild type (WT). Clinical sensitivity was further evaluated for five Ag-RDT utilising retrospective samples (Alpha, Delta, Omicron BA.1) and one Ag-RDT utilising prospective clinical samples (Delta and Omicron BA.1). Findings For the WT, Alpha, Delta, Gamma and Omicron (BA.1) variants 22, 32, 29, 31 and 32 of the 34 Ag-RDTs evaluated met the World Health Organisations (WHO) target product profile (TPP), respectively. Of the 31 Ag-RDTs included for Omicron BA.5 evaluation 29 met the WHO TPP. Additionally, the LODs for samples spiked with Omicron BA.5 were significantly lower than all other VOCs included (p&lt;0.001). In the retrospective clinical evaluation when comparing RNA copies/mL, the Ag-RDTs detected Alpha and Omicron (BA.1) more sensitively than the Delta VOC. Samples with high RT-qPCR Cts (Ct&gt;25) resulted in reduced test sensitivities across all variants. We used linear regression to model the 50% and 95% LOD of clinical samples and observed statistically similar results for all tests. In the prospective clinical samples, the sensitivity was statistically similar for the Delta VOC 71.9% (CI 95% 53.3-86.6%) and Omicron VOC 84.4% (CI95% 75.3-91.2%). Interpretation Test performance differs between SARS-CoV-2 VOCs, and high sensitivity was achieved when testing the Omicron BA.5 VOC compared to the WHO Ag-RDT requirements. Continuous evaluations must be performed to monitor test performance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.23292615">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.23292615" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.23292615">
        <p class="paperTitle">Rapid Identification of Bacterial isolates Using Microfluidic Adaptive Channels and Multiplexed Fluorescence Microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.23292615" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.23292615" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chatzimichail, S.; Turner, P. B.; Feehily, C.; Farrar, A. E.; Crook, D.; Andersson, M.; Oakley, S.; Barrett, L.; El Sayyed, H.; Kyropoulos, J.; Nellaker, C.; Stoesser, N.; Kapanidis, A. N.</p>
        <p class="info">Score: 17.0, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.23292615' target='https://doi.org/10.1101/2023.07.16.23292615'> 10.1101/2023.07.16.23292615</a></p>
        <p class="abstract">We demonstrate the rapid capture, enrichment, and identification of bacterial pathogens using Adaptive Channel Bacterial Capture (ACBC) devices. Using controlled tuning of device backpressure in polydimethylsiloxane (PDMS) devices, we enable the controlled formation of capture regions capable of trapping bacteria from low cell density samples with near 100% capture efficiency. The technical demands to prepare such devices are much lower compared to conventional methods for bacterial trapping and can be achieved with simple benchtop fabrication methods. We demonstrate the capture and identification of seven species of bacteria with bacterial concentrations lower than 1000 cells/mL, including common Gram-negative and Gram-positive pathogens such as Escherichia coli and Staphylococcus aureus. We further demonstrate that species identification of the trapped bacteria can be undertaken in the order of one-hour using multiplexed 16S rRNA-FISH with identification accuracies of 73-99% with unsupervised classification methods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.23292426">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.23292426" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.23292426">
        <p class="paperTitle">Intranasal lavage with hypochlorous acid safely reduces the symptoms in the ambulatory patient with COVID-19.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.23292426" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.23292426" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abner, M. L.</p>
        <p class="info">Score: 6.5, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.23292426' target='https://doi.org/10.1101/2023.07.17.23292426'> 10.1101/2023.07.17.23292426</a></p>
        <p class="abstract">OBJECTIVEThis study was designed to investigate intranasal lavage with a hypochlorous acid solution in the reduction of symptoms in the ambulatory COVID-19 patient.

STUDY DESIGNStudy approval granted by the Institutional Review Board of Reading Hospital (IRB 036-20), with informed consent obtained from all adult participants(age&gt;18 years).

SETTINGAll enrollees, taken from the same ambulatory testing facility, received nasopharyngeal swabs for COVID-19 testing by reverse transcription polymerase chain (RT-PCR) or the COVID-19 antigen specific test (Binax NOW, Abbott Lab)

METHODSConvenience sampling methodology was utilized. Each enrollee was provided with the study devices which included a Nasaflo Neti Pot (NeilMed Pharmaceutical, Inc.), and the hypochlorous acid solution (Vashe Wound Solution, Urgo Medical North America, LLC). Participants were instructed to irrigate each nostril with 120 cc (four ounces) of the solution for ten consecutive days, and record the presence or absence of symptoms in a scripted diary log.

RESULTSThe study included 88 patients of which 74 (84.1%) completed the ten days of nasal lavage. All data analysis was conducted using SPSS version 25.0.

Chi square test of association found no significant difference related to gender, age group race, ethnicity, residence, or living arrangements (all p-values &gt; 0.05). There were no statistical differences in any of the co-morbid conditions. Mild adverse reactions included burning, epistaxis, and oral metallic taste. No enrollees required mechanical ventilation. There were no deaths.

CONCLUSIONThis study suggests the feasibility and safety of using intranasal lavage with a hypochlorous acid solution in relieving symptoms in the ambulatory Covid-19 patient.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.23292983">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.23292983" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.23292983">
        <p class="paperTitle">Incubation-period estimates of Omicron (BA.1) variant from Taiwan, December 2021-January 2022, and its comparison to other SARS-CoV-2 variants: a statistical modeling, systematic search and meta-analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.23292983" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.23292983" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Akhmetzhanov, A. R.; Cheng, H.-Y.; Dushoff, J.</p>
        <p class="info">Score: 5.8, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.23292983' target='https://doi.org/10.1101/2023.07.20.23292983'> 10.1101/2023.07.20.23292983</a></p>
        <p class="abstract">Background: The ongoing COVID-19 pandemic has seen several variants of concern, including the Omicron (BA.1) variant which emerged in October 2021. Accurately estimating the incubation period of these variants is crucial for predicting disease spread and formulating effective public health strategies. However, existing estimates often conflict because of biases arising from the dynamic nature of epidemic growth and selective inclusion of cases. This study aims to accurately estimate of the Omicron (BA.1) variant incubation period based on data from Taiwan, where disease incidence remained low and contact tracing was comprehensive during the first months of the Omicron outbreak. Methods: We reviewed 100 contact-tracing records for cases of the Omicron BA.1 variant reported between December 2021 and January 2022, and found enough information to analyze 70 of these. The incubation period distribution was estimated by fitting data on exposure and symptom onset within a Bayesian mixture model using gamma, Weibull, and lognormal distributions as candidates. Additionally, a systematic literature search was conducted to accumulate data for estimates of the incubation period for Omicron (BA.1/2, BA.4/5) subvariants, which was then used for meta-analysis and comparison. Results: The mean incubation period was estimated at 3.5 days (95% credible interval: 3.1-4.0 days), with no clear differences when stratified by vaccination status or age. This estimate aligns closely with the pooled mean of 3.4 days (3.0-3.8 days) obtained from a meta-analysis of other published studies on Omicron subvariants. Conclusions: The relatively shorter incubation period of the Omicron variant, as compared to previous SARS-CoV2 variants, implies its potential for rapid spread but also opens the possibility for individuals to voluntarily adopt shorter, more resource-efficient quarantine periods. Continual updates to incubation period estimates, utilizing data from comprehensive contact tracing, are crucial for effectively guiding these voluntary actions and adjusting high socio-economic cost interventions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.23288598">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.23288598" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.23288598">
        <p class="paperTitle">SARS-CoV-2 virologic rebound with nirmatrelvir-ritonavir therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.23288598" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.23288598" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Edelstein, G. E.; Boucau, J.; Uddin, R.; Marino, C.; Liew, M. Y.; Barry, M.; Choudhary, M. C.; Gilbert, R. F.; Reynolds, Z.; Li, Y.; Tien, D.; Sagar, S.; Vyas, T. D.; Kawano, Y.; Sparks, J. A.; Hammond, S. P.; Wallace, Z.; Vyas, J. M.; Barczak, A. K.; Lemieux, J. E.; Li, J. Z.; Siedner, M. J.</p>
        <p class="info">Score: 176.4, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.23288598' target='https://doi.org/10.1101/2023.06.23.23288598'> 10.1101/2023.06.23.23288598</a></p>
        <p class="abstract">ObjectiveTo compare the frequency of replication-competent virologic rebound with and without nirmatrelvir-ritonavir treatment for acute COVID-19. Secondary aims were to estimate the validity of symptoms to detect rebound and the incidence of emergent nirmatrelvir-resistance mutations after rebound.

DesignObservational cohort study.

SettingMulticenter healthcare system in Boston, Massachusetts.

ParticipantsWe enrolled ambulatory adults with a positive COVID-19 test and/or a prescription for nirmatrelvir-ritonavir.

ExposuresReceipt of 5 days of nirmatrelvir-ritonavir treatment versus no COVID-19 therapy.

Main Outcome and MeasuresThe primary outcome was COVID-19 virologic rebound, defined as either (1) a positive SARS-CoV-2 viral culture following a prior negative culture or (2) two consecutive viral loads [&amp;ge;]4.0 log10 copies/milliliter after a prior reduction in viral load to &lt;4.0 log10 copies/milliliter.

ResultsCompared with untreated individuals (n=55), those taking nirmatrelvir-ritonavir (n=72) were older, received more COVID-19 vaccinations, and were more commonly immunosuppressed. Fifteen individuals (20.8%) taking nirmatrelvir-ritonavir experienced virologic rebound versus one (1.8%) of the untreated (absolute difference 19.0% [95%CI 9.0-29.0%], P=0.001). In multivariable models, only N-R was associated with VR (AOR 10.02, 95%CI 1.13-88.74). VR occurred more commonly among those with earlier nirmatrelvirritonavir initiation (29.0%, 16.7% and 0% when initiated days 0, 1, and [&amp;ge;]2 after diagnosis, respectively, P=0.089). Among participants on N-R, those experiencing rebound had prolonged shedding of replication-competent virus compared to those that did not rebound (median: 14 vs 3 days). Only 8/16 with virologic rebound reported worsening symptoms (50%, 95%CI 25%-75%); 2 were completely asymptomatic. We detected no post-rebound nirmatrelvir-resistance mutations in the NSP5 protease gene.

Conclusions and RelevanceVirologic rebound occurred in approximately one in five people taking nirmatrelvir-ritonavir and often occurred without worsening symptoms. Because it is associated with replication-competent viral shedding, close monitoring and potential isolation of those who rebound should be considered.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.23292994">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.23292994" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.23292994">
        <p class="paperTitle">Mucosal and Systemic Immune Correlates of Viral Control following SARS-CoV-2 Infection Challenge in Seronegative Adults</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.23292994" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.23292994" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wagstaffe, H. R.; Thwaites, R. S.; Reynaldi, A.; Sidhu, J. K.; McKendry, R.; Ascough, S.; Papargyris, L.; Collins, A. M.; Xu, J.; Lemm, N.-M.; Siggins, M. K.; Chain, B. M.; Killingley, B.; Kalinova, M.; Mann, A.; Catchpole, A.; Davenport, M. P.; Openshaw, P. J. M.; Chiu, C.</p>
        <p class="info">Score: 4.5, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.23292994' target='https://doi.org/10.1101/2023.07.21.23292994'> 10.1101/2023.07.21.23292994</a></p>
        <p class="abstract">Human infection challenge permits characterisation of the associated immune response in unparalleled depth, enabling evaluation of early pre-symptomatic immune changes and the dynamic immune factors important for viral clearance. Here, 34 healthy young adult volunteers, seronegative to SARS-CoV-2, were inoculated with a D614G-containing pre-Alpha SARS-CoV-2 strain. Nasal and systemic soluble mediator and antibody responses, and peripheral blood T cell and B cell responses were measured by MesoScale Discovery and flow cytometry just before and up to 1 year after intra-nasal inoculation. In the 18 (53%) participants who became infected, both nasal and systemic mediator responses were dominated by interferons (IFN) but with divergent kinetics. T cell activation and proliferation in blood peaked at day 10 in CD4&#43; T cells and day 14 in CD8&#43; T cells, returning to baseline by day 28. Following infection, antigen-specific T cells were largely CD38&#43;Ki67&#43; and displayed central and effector memory phenotypes. T cells contracted after viral clearance with expanded antigen-specific memory T cell populations persisting past day 28. Both mucosal and systemic antibodies became detectable around day 10 but nasal antibodies plateaued after day 14 while circulating antibodies continued to rise. Using piecewise linear regression modelling, viral load related closely to the induction of type I IFN responses, moreover, CD8&#43; T cell responses and early IgA responses were strongly associated with viral clearance. Detailed analysis of innate and adaptive immune responses to primary SARS-CoV-2 infection following human challenge thus revealed the relationship between immune kinetics and viral load as factors associated with resolution of infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.23292858">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.23292858" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.23292858">
        <p class="paperTitle">Risk of SARS-CoV-2 infection and hospitalization in individuals with natural, vaccine-induced and hybrid immunity: a retrospective population-based cohort study from Estonia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.23292858" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.23292858" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Uuskula, A.; Pisarev, H.; Tisler, A.; Meister, T.; Suija, K.; Huik, K.; Abroi, A.; Kalda, R.; Kolde, R.; Fischer, K.</p>
        <p class="info">Score: 3.9, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.23292858' target='https://doi.org/10.1101/2023.07.18.23292858'> 10.1101/2023.07.18.23292858</a></p>
        <p class="abstract">A large proportion of the worlds population has some form of immunity against SARS-CoV- 2, through either infection ( natural), vaccination or both ( hybrid). This retrospective cohort study used data on SARS-CoV-2, vaccination, and hospitalization from national health system from February 2020 to June 2022 and Cox regression modelling to compare those with natural immunity to those with no (Cohort1, n=92917), hybrid (Cohort2, n=46813), and vaccine (Cohort3, n=252414) immunity. In Cohort 1, those with natural immunity were at lower risk for infection during the Delta (aHR 0.17, 95%CI 0.15-0.18) and higher risk (aHR 1.24, 95%CI 1.18-1.32) during the Omicron period than those with no immunity. Natural immunity conferred substantial protection against COVID-19-hospitalization. Cohort 2 - in comparison to natural immunity hybrid immunity offered strong protection during the Delta (aHR 0.61, 95%CI 0.46-0.80) but not the Omicron (aHR 1.05, 95%CI 0.93-1.1) period. COVID-19-hospitalization was extremely rare among individuals with hybrid immunity. In Cohort 3, individuals with vaccine-induced immunity were at higher risk than those with natural immunity for infection (Delta aHR 4.90, 95%CI 4.48-5.36; Omicron 1.13, 95%CI 1.06-1.21) and hospitalization (Delta aHR 7.19, 95%CI 4.02-12.84). These results show that risk of infection and severe COVID-19 are driven by personal immunity history and the variant of SARS-CoV-2 causing infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.23291827">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.23291827" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.23291827">
        <p class="paperTitle">Increased circulating fibronectin, depletion of natural IgM and heightened EBV, HSV-1 reactivation in ME/CFS and long COVID</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.23291827" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.23291827" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, Z.; Hollmann, C.; Kalanidhi, S.; Grothey, A.; Keating, S.; Mena-Palomo, I.; Lamer, S.; Schlosser, A.; Kaiping, A.; Scheller, C.; Sotzny, F.; Horn, A.; Nuernberger, C.; Cejka, V.; Afshar, B.; Bahmer, T.; Schreiber, S.; Janne Vehreschild, J.; Milljukov, O.; Schaefer, C.; Kretzler, L.; Keil, T.; Reese, J.-P.; Eichner, F. A.; Schmidbauer, L.; Heuschmann, P. U.; Stoerk, S.; Morbach, C.; Riemekasten, G.; Beyersdorf, N.; Scheibenbogen, C.; Naviaux, R. K.; Williams, M.; Ariza, M. E.; Prusty, B. K.</p>
        <p class="info">Score: 474.2, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.23291827' target='https://doi.org/10.1101/2023.06.23.23291827'> 10.1101/2023.06.23.23291827</a></p>
        <p class="abstract">Myalgic Encephalomyelitis/ Chronic Fatigue syndrome (ME/CFS) is a complex, debilitating, long-term illness without a diagnostic biomarker. ME/CFS patients share overlapping symptoms with long COVID patients, an observation which has strengthened the infectious origin hypothesis of ME/CFS. However, the exact sequence of events leading to disease development is largely unknown for both clinical conditions. Here we show antibody response to herpesvirus dUTPases, particularly to that of Epstein-Barr virus (EBV) and HSV-1, increased circulating fibronectin (FN1) levels in serum and depletion of natural IgM against fibronectin ((n)IgM-FN1) are common factors for both severe ME/CFS and long COVID. We provide evidence for herpesvirus dUTPases-mediated alterations in host cell cytoskeleton, mitochondrial dysfunction and OXPHOS. Our data show altered active immune complexes, immunoglobulin-mediated mitochondrial fragmentation as well as adaptive IgM production in ME/CFS patients. Our findings provide mechanistic insight into both ME/CFS and long COVID development. Finding of increased circulating FN1 and depletion of (n)IgM-FN1 as a biomarker for the severity of both ME/CFS and long COVID has an immediate implication in diagnostics and development of treatment modalities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.23292793">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.23292793" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.23292793">
        <p class="paperTitle">Health-seeking behaviour and beliefs around sore throat in The Gambia: a qualitative study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.23292793" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.23292793" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Suau Sans, M.; Manneh, M.; Ceesay, I.; Bittaye, A.; de Crombrugghe, G.; Keeley, A. J.; de Silva, T. I.; Palmer, J.; Armitage, E. P.; Marks, M.</p>
        <p class="info">Score: 2.8, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.23292793' target='https://doi.org/10.1101/2023.07.17.23292793'> 10.1101/2023.07.17.23292793</a></p>
        <p class="abstract">Group A Streptococcus (StrepA) bacteria causes a broad spectrum of diseases. The most common manifestations of StrepA infection are sore throat and pus-producing skin infections such as impetigo. Complications of StrepA infection lead to inflammation in the bones, muscles, joints, and internal organs causing acute rheumatic fever and rheumatic heart disease (RHD). In The Gambia, the RHD burden is thought to be very high. However, epidemiological data is minimal, and StrepA control programmes do not exist. This study aims to explore common beliefs and practices related to sore throats among primary caregivers of children, and healthcare providers in a community with a high StrepA disease burden. This will inform the design of preventative strategies for StrepA-related sequelae.

Four informal conversations with providers and fifteen semi-structured interviews with caregivers were conducted in the peri-urban area of Sukuta, The Gambia. Sampling was purposive and gradual, beginning from households identified to have recently experienced sore throat through a parallel cohort study. Themes explored in qualitative analysis included: sore throat causal attributions and diagnoses, care practises, health-seeking behaviour, and perceived barriers to using the biomedical sector.

Sore throats were typically perceived to affect one child in a family, disproportionately or exclusively. Sore throats are rarely perceived as life-threatening, and awareness of links between sore throat and ARF or RHD was not reported among caregivers or providers in this study population. Most cases of sore throat are initially managed at home using traditional medicine which delays resort to antibiotics, though in two instances of severe pain with the presence of exudate, fear that the childs life was at risk prompted care-seeking through the formal health system.

Our findings can inform the development of tailored strategies to increase community knowledge of the potential long-term consequences of sore throats and appropriate care-seeking, alongside improvements in the health system, to prevent StrepA sequelae effectively.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
