<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Risk of Coronavirus Disease 2019 (COVID-19) among Those Up-to-Date and Not Up-to-Date on COVID-19 Vaccination
Authors: Shrestha, N. K.; Burke, P. C.; Nowacki, A. S.; Gordon, S. M.
Score: 5160.8, Published: 2023-06-12 DOI: 10.1101/2023.06.09.23290893
BackgroundThe CDC recently defined being &#34;up-to-date&#34; on COVID-19 vaccination as having received at least one dose of a COVID-19 bivalent vaccine. The purpose of this study was to compare the risk of COVID-19 among those &#34;up-to-date&#34; and &#34;not up-to-date&#34; on COVID-19 vaccination.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Risk of Coronavirus Disease 2019 (COVID-19) among Those Up-to-Date and Not Up-to-Date on COVID-19 Vaccination
Authors: Shrestha, N. K.; Burke, P. C.; Nowacki, A. S.; Gordon, S. M.
Score: 5160.8, Published: 2023-06-12 DOI: 10.1101/2023.06.09.23290893
BackgroundThe CDC recently defined being &#34;up-to-date&#34; on COVID-19 vaccination as having received at least one dose of a COVID-19 bivalent vaccine. The purpose of this study was to compare the risk of COVID-19 among those &#34;up-to-date&#34; and &#34;not up-to-date&#34; on COVID-19 vaccination." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-06-28T11:45:43+00:00" />
<meta property="article:modified_time" content="2023-06-28T11:45:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Risk of Coronavirus Disease 2019 (COVID-19) among Those Up-to-Date and Not Up-to-Date on COVID-19 Vaccination
Authors: Shrestha, N. K.; Burke, P. C.; Nowacki, A. S.; Gordon, S. M.
Score: 5160.8, Published: 2023-06-12 DOI: 10.1101/2023.06.09.23290893
BackgroundThe CDC recently defined being &#34;up-to-date&#34; on COVID-19 vaccination as having received at least one dose of a COVID-19 bivalent vaccine. The purpose of this study was to compare the risk of COVID-19 among those &#34;up-to-date&#34; and &#34;not up-to-date&#34; on COVID-19 vaccination."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Risk of Coronavirus Disease 2019 (COVID-19) among Those Up-to-Date and Not Up-to-Date on COVID-19 Vaccination\nAuthors: Shrestha, N. K.; Burke, P. C.; Nowacki, A. S.; Gordon, S. M.\nScore: 5160.8, Published: 2023-06-12 DOI: 10.1101/2023.06.09.23290893\nBackgroundThe CDC recently defined being \u0026#34;up-to-date\u0026#34; on COVID-19 vaccination as having received at least one dose of a COVID-19 bivalent vaccine. The purpose of this study was to compare the risk of COVID-19 among those \u0026#34;up-to-date\u0026#34; and \u0026#34;not up-to-date\u0026#34; on COVID-19 vaccination.",
  "keywords": [
    
  ],
  "articleBody": " Risk of Coronavirus Disease 2019 (COVID-19) among Those Up-to-Date and Not Up-to-Date on COVID-19 Vaccination\nAuthors: Shrestha, N. K.; Burke, P. C.; Nowacki, A. S.; Gordon, S. M.\nScore: 5160.8, Published: 2023-06-12 DOI: 10.1101/2023.06.09.23290893\nBackgroundThe CDC recently defined being \"up-to-date\" on COVID-19 vaccination as having received at least one dose of a COVID-19 bivalent vaccine. The purpose of this study was to compare the risk of COVID-19 among those \"up-to-date\" and \"not up-to-date\" on COVID-19 vaccination. MethodsEmployees of Cleveland Clinic in employment when the COVID-19 bivalent vaccine first became available, and still employed when the XBB lineages became dominant, were included. Cumulative incidence of COVID-19 since the XBB lineages became dominant was compared across the \"up-to-date\" and \"not up-to-date\" states, by treating COVID-19 bivalent vaccination as a time-dependent covariate whose value changed on receipt of the vaccine. Risk of COVID-19 by vaccination status was also compared using multivariable Cox proportional hazards regression adjusting for propensity to get tested for COVID-19, age, sex, and phase of most recent prior SARS-CoV-2 infection. ResultsCOVID-19 occurred in 1475 (3%) of 48 344 employees during the 100-day study period. The cumulative incidence of COVID-19 was lower in the \"not up-to-date\" than in the \"up-to-date\" state. On multivariable analysis, not being \"up-to-date\" with COVID-19 vaccination was associated with lower risk of COVID-19 (HR, 0.77; 95% C.I., 0.69-0.86; P-value, \u003c0.001). Results were very similar when those 65 years and older were only considered \"up-to-date\" after receiving 2 doses of the bivalent vaccine. ConclusionsSince the XBB lineages became dominant, adults \"not up-to-date\" by the CDC definition have a lower risk of COVID-19 than those \"up-to-date\" on COVID-19 vaccination, bringing into question the value of this risk classification definition. SummaryAmong 48 344 working-aged Cleveland Clinic employees, those not \"up-to-date\" on COVID-19 vaccination had a lower risk of COVID-19 than those \"up-to-date\". The current CDC definition provides a meaningless classification of risk of COVID-19 in the adult population.\nInternal tremors and vibrations in long COVID: a cross-sectional study\nAuthors: Zhou, T.; McAlpine, L.; Michelsen, T.; Caraballo, C.; Sawano, M.; Bhattacharjee, B.; Lu, Y.; Iwasaki, A.; Krumholz, H.\nScore: 127.9, Published: 2023-06-19 DOI: 10.1101/2023.06.19.23291598\nBackgroundInternal tremors and vibrations symptoms have been described as part of neurologic disorders but have not been fully described as a part of long COVID. We compared demographics, socioeconomic characteristics, pre-pandemic comorbidities, and new-onset conditions between people with internal tremors and vibrations as part of their long COVID symptoms and people with long COVID but without these symptoms. MethodsA cross-sectional study, Listen to Immune, Symptom and Treatment Experiences Now (LISTEN), collected data from adults with long COVID. The study sample included 423 participants enrolled between May 2022 and June 2023. ResultsThe 423 participants had a median age of 46 years (interquartile range, 38-56), 74% were female, 87% were Non-Hispanic White, and 158 (37%) reported \"internal tremors, or buzzing/vibration\" as a long COVID symptom. Before long COVID, the groups had similar comorbidities. Post-COVID, participants with internal tremors and vibrations had significantly worse health as measured by the Euro-QoL visual analogue scale (median: 40 vs. 50 points, P = 0.007), higher rates of financial difficulties caused by the pandemic and housing insecurity (P \u003c 0.001 for each), and were significantly more likely to have new-onset conditions of mast cell disorders (11% vs. 2.6%), neurologic conditions (22% vs. 8.3%), anxiety (20% vs. 8.7%), and trauma- or stress-related mental health disorders (12% vs. 3.4%) compared with those without internal tremors (Bonferroni-adjusted P \u003c 0.05 for each). Participants with internal tremors also reported significantly higher rates of cardiovascular, gastrointestinal, integumentary, and neurologic long COVID symptoms compared with those without internal tremors (Bonferroni- adjusted P \u003c 0.05 for each). Conclusions and RelevanceAmong people with long COVID, those with internal tremors and vibrations have more associated symptoms and worse health status, suggesting it may be associated with a severe phenotype of the condition.\nMetformin reduces SARS-CoV-2 in a Phase 3 Randomized Placebo Controlled Clinical Trial\nAuthors: Bramante, C. T.; Beckman, K. B.; Mehta, T.; Krager, A. B.; Odde, D. J.; Tignanelli, C. J.; Buse, J. B.; Johnson, D. M.; Watson, R. H. B.; Daniel, J. J.; Liebovitz, D. M.; Nicklas, J. M.; Cohen, K.; Puskarich, M. A.; Belani, H. K.; Siegel, L. K.; Klatt, N. R.; Anderson, B.; Hartman, K. M.; Rao, V.; Hagan, A. A.; Patel, B.; Fenno, S. L.; Avula, N.; Reddy, N. V.; Erickson, S. M.; Fricton, R. D.; Lee, S.; Griffiths, G.; Pullen, M. F.; Thompson, J. L.; Sherwood, N.; Murray, T. A.; Rose, M. R.; Boulware, D. R.; Huling, J. D.\nScore: 1201.7, Published: 2023-06-07 DOI: 10.1101/2023.06.06.23290989\nCurrent antiviral treatment options for SARS-CoV-2 infections are not available globally, cannot be used with many medications, and are limited to virus-specific targets.1-3 Biophysical modeling of SARS-CoV-2 replication predicted that protein translation is an especially attractive target for antiviral therapy.4 Literature review identified metformin, widely known as a treatment for diabetes, as a potential suppressor of protein translation via targeting of the host mTor pathway.5 In vitro, metformin has antiviral activity against RNA viruses including SARS-CoV-2.6,7 In the COVID-OUT phase 3, randomized, placebo-controlled trial of outpatient treatment of COVID-19, metformin had a 42% reduction in ER visits/hospitalizations/death through 14 days; a 58% reduction in hospitalizations/death through 28 days, and a 42% reduction in Long COVID through 10 months.8,9 Here we show viral load analysis of specimens collected in the COVID-OUT trial that the mean SARS-CoV-2 viral load was reduced 3.6-fold with metformin relative to placebo (-0.56 log10 copies/mL; 95%CI, -1.05 to -0.06, p=0.027) while there was no virologic effect for ivermectin or fluvoxamine vs placebo. The metformin effect was consistent across subgroups and with emerging data.10,11 Our results demonstrate, consistent with model predictions, that a safe, widely available,12 well-tolerated, and inexpensive oral medication, metformin, can be repurposed to significantly reduce SARS-CoV-2 viral load.\nViral clearance as a surrogate of clinical efficacy for COVID-19 therapies in outpatients: A systematic review and meta-analysis\nAuthors: Elias, K. M.; Khan, s. R.; Stadler, E.; Schlub, T. E.; Cromer, D.; Polizzotto, M. N.; Kent, S.; Turner, T.; Davenport, M. P.; Khoury, D. S.\nScore: 48.1, Published: 2023-06-19 DOI: 10.1101/2023.06.18.23291566\nBackgroundSurrogates of antiviral efficacy are needed for COVID-19. We investigated the relationship between the virological effect of treatment and clinical efficacy as measured by progression to severe disease in unvaccinated outpatients treated for mild to moderate COVID-19. MethodsWe searched PubMed, Scopus and medRxiv from inception to 27th September 2022, for randomised controlled trials (RCTs) which tested potential treatments for COVID-19 in non-hospitalized patients. We included studies that reported both clinical and virological outcomes. Clinical outcomes were the rate of disease progression (generally hospitalization or death within 28 days of commencing treatment) and virological outcomes were viral load (viral RNA copies in upper respiratory tract swabs) within the first 7 days of treatment. Studies were excluded if they did not report on the outcome of a primary randomised controlled trial, or if results were reported in a more complete form in another publication. Risk of Bias assessment was performed using the RoB 2.0 tool. We used generalised linear models with random effects to assess the association between outcomes and account for study heterogeneity. FindingsWe identified 1372 unique studies of which 14 (with a total of 9257 participants) met inclusion criteria. Larger virological treatment effects at both day 3 and day 5 were associated with decreased odds of progression to hospitalisation or death in unvaccinated ambulatory subjects. The odds ratio (OR) for each extra two-fold reduction in viral load in treated compared to control subjects was 0.54 on both days 3 and 5 post treatment (day 3 95% CI 0.38 to 0.74, day 5 95%CI 0.41 to 0.72). There was no relationship between the odds of hospitalisation or death and virological treatment effect at day 7 (OR 0.91, 95%CI 0.74 to 1.13). InterpretationThis review provides evidence that treatment-induced acceleration of viral clearance within the first 5 days after treatment is a surrogate of clinical efficacy to prevent hospitalisation with COVID-19. Limitations included the aggregation of studies with differing designs, and evidence of risk of bias in some virological outcomes. These findings support the use of viral clearance as an early phase clinical trial endpoint of therapeutic efficacy. FundingThe authors were supported by the Australian Government Department of Health, Medical Research Future Fund, National Health and Medical Research Council and the University of New South Wales.\nHigh burden of viruses and bacterial pathobionts drives heightened nasal innate immunity in children with and without SARS-CoV-2\nAuthors: Watkins, T. A.; Cheemarla, N. R.; Hansel, K.; Green, A. B.; Amat, J. A. R.; Lozano, R.; Dudgeon, S. N.; Landry, M. L.; Schulz, W. L.; Foxman, E. F.\nScore: 24.2, Published: 2023-06-20 DOI: 10.1101/2023.06.17.23291498\nRecent work indicates that heightened nasal innate immunity in children may impact SARS-CoV-2 pathogenesis. Here, we identified drivers of nasal innate immune activation in children using cytokine profiling and multiplex pathogen detection in 291 pediatric nasopharyngeal samples from the 2022 Omicron surge. Nasal viruses and bacterial pathobionts were highly prevalent, especially in younger children (81% of symptomatic and 37% asymptomatic children overall; 91% and 62% in subjects \u003c5 yrs). For SARS-CoV-2, viral load was highest in young children, and viral load in single infections or combined viral loads in coinfections best predicted nasal CXCL10, a biomarker of the mucosal interferon response. Bacterial pathobionts correlated with high nasal IL-1{beta} and TNF, but not CXCL10, and viral-bacterial coinfections showed a combined immunophenotype. These findings reveal virus and bacteria as drivers of heightened nasal innate immunity in children and suggest that frequent host-pathogen interactions shape responses to respiratory viruses in this age group.\nRapid nanopore metagenomic sequencing and predictive susceptibility testing of positive blood cultures from intensive care patients with sepsis\nAuthors: Harris, P. N. A.; Bauer, M. J.; Lueftinger, L.; Beisken, S.; Forde, B. M.; Balch, R.; Cotta, M. O.; Schlapbach, L.; Raman, S.; Shekar, K.; Kruger, P.; Lipman, J.; Bialasiewicz, S.; Coin, L. J. M.; Roberts, J. A.; Paterson, D. L.; Irwin, A. D.\nScore: 18.1, Published: 2023-06-22 DOI: 10.1101/2023.06.15.23291261\nBackgroundDirect metagenomic sequencing from positive blood culture (BC) broths, to identify bacteria and predict antimicrobial susceptibility, has been previously demonstrated using Illumina-based methods, but is relatively slow. We aimed to evaluate this approach using nanopore sequencing to provide more rapid results. MethodsPatients with suspected sepsis in 4 intensive care units were prospectively enrolled. Human-depleted DNA was extracted from positive BC broths and sequenced using nanopore (MinION). Species abundance was estimated using Kraken2, and a cloud-based artificial intelligence (AI) system (AREScloud) provided in silico antimicrobial susceptibility testing (AST) from assembled contigs. These results were compared to conventional identification and phenotypic AST. ResultsGenus-level agreement between conventional methods and metagenomic whole genome sequencing (MG-WGS) was 96.2% (50/52), but increased to 100% in monomicrobial infections. In total, 262 high quality AREScloud AST predictions across 24 samples were made, exhibiting categorical agreement (CA) of 89.3%, with major error (MA) and very major error (VME) rates of 10.5% and 12.1%, respectively. Over 90% CA was achieved for some taxa (e.g. Staphylococcus aureus), but was suboptimal for Pseudomonas aeruginosa (CA 50%). In 470 AST predictions across 42 samples, with both high quality and exploratory-only predictions, overall CA, ME and VME rates were 87.7%, 8.3% and 28.4%. VME rates were inflated by false susceptibility calls in a small number of species / antibiotic combinations with few representative resistant isolates. Time to reporting from MG-WGS could be achieved within 8-16 hours from blood culture positivity. ConclusionsDirect metagenomic sequencing from positive BC broths is feasible and can provide accurate predictive AST for some species and antibiotics, but is sub-optimal for a subset of common pathogens, with unacceptably high VME rates. Nanopore-based approaches may be faster but improvements in accuracy are required before it can be considered for clinical use. New developments in nanopore sequencing technology, and training of AI algorithms on larger and more diverse datasets may improve performance.\nThe association of neutralizing antibodies with protection against symptomatic dengue virus infection varies by serotype, prior immunity, and assay conditions\nAuthors: Bos, S.; Graber, A. L.; Cardona-Ospina, J. A.; Duarte, E. M.; Zambrana, J. V.; Ruiz Salinas, J. A.; Mercado-Hernandez, R.; Singh, T.; Katzelnick, L. C.; de Silva, A.; Kuan, G.; Balmaseda, A.; Harris, E.\nScore: 12.7, Published: 2023-06-20 DOI: 10.1101/2023.06.20.23291522\nThe four dengue virus serotypes (DENV1-4) are the most prevalent arboviruses in humans and a major public health concern worldwide. Identification of the immune mechanisms that modulate DENV infection outcome is critical for development of a safe and effective vaccine. Neutralizing antibodies (nAbs) are considered an essential component of the protective response, yet measurement of their potency is primarily performed using a single cellular substrate and partially mature virions. This approach does not capture the full breadth of neutralizing activity and may lead to biased estimations of nAb potency and repertoire. Here, we evaluated the nAb response associated with protection against dengue cases using samples collected after one or more DENV infections but prior to a subsequent symptomatic versus inapparent DENV1, DENV2 or DENV3 infection from a long-standing pediatric cohort study in Nicaragua. We compared nAb titers in pre-inapparent and pre-symptomatic infection samples in Vero cells with or without DC-SIGN expression, and with either mature or partially mature virions. This method allowed us to measure the magnitude and characteristics of the nAb response associated with outcome and revealed that correlation of nAb titers with protection from symptomatic infection is dependent on the individuals prior DENV immune status as well as the subsequent infecting DENV serotype. Further, the nAb potency and the protective NT50 cutoff was greatly impacted by virion maturation state and cell substrate. These results have important implications for determination of antibody correlates of protection for vaccines and natural infections.\nTracing the origin of SARS-CoV-2 Omicron-like Spike sequences detected in wastewater\nAuthors: Shafer, M. M.; Bobholz, M. J.; Vuyk, W. C.; Gregory, D. A.; Roguet, A.; Haddock Soto, L. A.; Rushford, C.; Janssen, K. H.; Ries, H. J.; Pilch, H. E.; Mullen, P. A.; Fahney, R. B.; Wei, W.; Lambert, M.; Wenzel, J.; Halfmann, P.; Kawaoka, Y.; Wilson, N. A.; Friedrich, T. C.; Pray, I. W.; Westergaard, R.; O'Connor, D. H.; Johnson, M.\nScore: 1124.1, Published: 2023-05-30 DOI: 10.1101/2022.10.28.22281553\nBackgroundThe origin of divergent SARS-CoV-2 spike sequences found in wastewater, but not in clinical surveillance, remains unclear. These \"cryptic\" wastewater sequences have harbored many of the same mutations that later emerged in Omicron lineages. We first detected a cryptic lineage in municipal wastewater in Wisconsin in January 2022. Named the \"Wisconsin Lineage\", we sought to determine this viruss geographic origin and characterize its persistence and evolution over time. MethodsWe systematically sampled maintenance holes to trace the Wisconsin Lineages origin. We sequenced spike RBD domains, and where possible, whole viral genomes, to characterize the evolution of this lineage over the 13 consecutive months that it was detectable. FindingsThe persistence of the Wisconsin Lineage signal allowed us to trace it from a central wastewater plant to a single facility, with a high concentration of viral RNA. The viral sequences contained a combination of fixed nucleotide substitutions characteristic of Pango lineage B.1.234, which circulated in Wisconsin at low levels from October 2020 to February 2021, while mutations in the spike gene resembled those subsequently found in Omicron variants. InterpretationWe propose that prolonged detection of the Wisconsin Lineage in wastewater represents persistent shedding of SARS-CoV-2 from an infected individual, with ongoing within-host viral evolution leading to an ancestral B.1.234 virus accumulating \"Omicron-like\" mutations. FundingThe Rockefeller Foundation, Wisconsin Department of Health Services, Centers for Disease Control and Prevention (CDC), National Institute on Drug Abuse (NIDA), and the Center for Research on Influenza Pathogenesis and Transmission. Research in context Evidence before this studyTo identify other studies characterizing unusual wastewater-specific SARS-CoV-2 lineages, we conducted a PubMed search using the keywords \"cryptic SARS-CoV-2 lineages,\" \"novel SARS-CoV-2 lineages,\" and \"wastewater\" on May 9, 2023. From the 18 papers retrieved, only two reported detecting wastewater-specific cryptic lineages. These lineages were identified by investigators from this research team in wastewater from California, Missouri, and New York City, but none could be definitively traced to a specific origin. A third study in Nevada identified a unique recombinant variant (designated Pango lineage XL) in wastewater, which was also discovered in two clinical samples from the same community. However, it remained unclear whether the clinical samples were collected from the same individual(s) responsible for the virus detected in the wastewater. To our knowledge, no prior study has successfully traced cryptic SARS-CoV-2 wastewater lineages back to a specific location. How and where evolutionarily advanced cryptic lineages are introduced into wastewater has remained uncertain. Added value of this studyThis study documents the presence and likely source of a novel and highly divergent cryptic SARS-CoV-2 lineage detected in Wisconsin wastewater for 13 months. In contrast to previously reported cryptic lineages, we successfully traced the Wisconsin Lineage to a single facility serving about 30 people. The exceptionally high viral RNA loads at the source account for the detection of these sequences in wastewater collected at the downstream municipal wastewater treatment facility. Implications of all the available evidenceMany SARS-CoV-2 lineages have been found exclusively in wastewater. The high number of unusual mutations found in these wastewater-specific cryptic sequences raises the possibility that they originate from individual prolonged shedders or even non-human sources. The Wisconsin Lineages persistence in wastewater, single-facility origin, and heavily mutated Omicron-like genotype support the hypothesis that cryptic wastewater lineages arise from persistently infected humans. As these divergent sequences contain amino acid changes that eventually emerge in circulating viruses, increased global monitoring of such lineages in wastewater could help anticipate future circulating mutations and/or variants of concern.\nExpansion of a low-cost, saliva-based PCR test for the detection of mpox virus\nAuthors: Thomas, R. J.; Yolda-Carr, D.; Fajardo, K.; Allicock, O. M.; Steel, S. A.; Zepeda, T.; Brownlee, M.; Saladi, S.; Parkin, J.; Wyllie, A. L.\nScore: 10.2, Published: 2023-06-20 DOI: 10.1101/2023.06.12.23291295\nBackgroundCurrent recommendations for the diagnosis of mpox rely on lesion-swabs as the gold-standard specimen type even though many patients experience symptoms prior to lesion-onset. Earlier detection could bolster the mpox response by mitigating transmission and facilitating access to antiviral treatments. MethodsWe first compared five PCR assays for their detection of mpox DNA extracted from 30 saliva specimens in collection devices with a stabilizing buffer. Next, we investigated the stability of mpox detection in five raw, unsupplemented saliva samples diluted 1:10 in mpox-negative saliva, after storage at 4{degrees}C, room temperature ([~]19{degrees}C), 30{degrees}C, and 40{degrees}C for 72 hours. We also investigated the stability of virus detection through simulated shipping conditions. Lastly, we performed amplicon sequencing on seven saliva samples and assessed concordance of the PCR assays against mpox virus sequences. ResultsDespite identifying three different substitutions in the CDCs Monkeypox Virus Generic Real-Time PCR Tests forward and reverse primers, we observed no difference in the mean cycle threshold values generated between assays. However, one gene target for one assay performed better for overall detection when validated. Detection following storage at 4{degrees}C, [~]19{degrees}C, and 30{degrees}C remained relatively stable for 24-48 hours but this declined by 72 hours. At 40{degrees}C, detection was stable at 24 hours but declined by 48 hours. Detection following simulated summer and winter shipping temperature profiles also remained stable. ConclusionsFindings of this pilot investigation support a flexible, saliva-based, extraction-free PCR test as a promising approach for the low-cost detection of mpox virus. With stability observed for 24-48 hours as well as over simulated shipping temperatures, saliva-based sampling and simplified testing could reduce mpox diagnostic costs, increase access to testing and address hurdles in low- and middle-income countries. Future studies should build upon this and assess the temporal dynamics of mpox virus in saliva.\nCurrent sampling and sequencing biases of Lassa mammarenavirus limit inference from phylogeography and molecular epidemiology in Lassa fever endemic regions.\nAuthors: Arruda, L. B.; Free, H. B.; Simons, D.; Ansumana, R.; Elton, L.; Haider, N.; Honeyborne, I.; Asogun, D.; McHugh, T. D.; Ntoumi, F.; Zumla, A.; Kock, R.\nScore: 8.0, Published: 2023-06-22 DOI: 10.1101/2023.06.20.23291686\nLassa fever (LF) is a potentially lethal viral haemorrhagic infection of humans caused by Lassa mammarenavirus (LASV). It is an important endemic zoonotic disease in West Africa with growing evidence for increasing frequency and sizes of outbreaks. Phylogeographic and molecular epidemiology methods have projected expansion of the Lassa fever endemic zone in the context of future global change. The Natal multimammate mouse (Mastomys natalensis) is the predominant LASV reservoir, with few studies investigating the role of other animal species. To explore host sequencing biases, all LASV nucleotide sequences and associated metadata available on GenBank (n = 2,298) were retrieved. Most data originated from Nigeria (54%), Guinea (20%) and Sierra Leone (14%). Data from non-human hosts (n = 703) were limited and only 69 sequences encompassed complete genes. We found a strong positive correlation between the number of confirmed human cases and sequences at the country level (r = 0.93 (95% Confidence Interval = 0.71 - 0.98), p \u003c 0.001) but no correlation exists between confirmed cases and the number of available rodent sequences (r = -0.019 (95% C.I. -0.71 - 0.69), p = 0.96). Spatial modelling of sequencing effort highlighted current biases in locations of available sequences, with increased effort observed in Southern Guinea and Southern Nigeria. Phylogenetic analyses showed geographic clustering of LASV lineages, suggestive of isolated events of human-to-rodent transmission and the emergence of currently circulating strains of LASV from the year 1498 in Nigeria. Overall, the current study highlights significant geographic limitations in LASV surveillance, particularly, in non-human hosts. Further investigation of the non-human reservoir of LASV, alongside expanded surveillance, are required for precise characterisation of the emergence and dispersal of LASV. Accurate surveillance of LASV circulation in non-human hosts is vital to guide early detection and initiation of public health interventions for future Lassa fever outbreaks.\n",
  "wordCount" : "3608",
  "inLanguage": "en",
  "datePublished": "2023-06-28T11:45:43Z",
  "dateModified": "2023-06-28T11:45:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta"><span>updated on June 28, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.09.23290893">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.09.23290893" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.09.23290893">
        <p class="paperTitle">Risk of Coronavirus Disease 2019 (COVID-19) among Those Up-to-Date and Not Up-to-Date on COVID-19 Vaccination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.09.23290893" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.09.23290893" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shrestha, N. K.; Burke, P. C.; Nowacki, A. S.; Gordon, S. M.</p>
        <p class="info">Score: 5160.8, Published: 2023-06-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.09.23290893' target='https://doi.org/10.1101/2023.06.09.23290893'> 10.1101/2023.06.09.23290893</a></p>
        <p class="abstract">BackgroundThe CDC recently defined being &#34;up-to-date&#34; on COVID-19 vaccination as having received at least one dose of a COVID-19 bivalent vaccine. The purpose of this study was to compare the risk of COVID-19 among those &#34;up-to-date&#34; and &#34;not up-to-date&#34; on COVID-19 vaccination.

MethodsEmployees of Cleveland Clinic in employment when the COVID-19 bivalent vaccine first became available, and still employed when the XBB lineages became dominant, were included.

Cumulative incidence of COVID-19 since the XBB lineages became dominant was compared across the &#34;up-to-date&#34; and &#34;not up-to-date&#34; states, by treating COVID-19 bivalent vaccination as a time-dependent covariate whose value changed on receipt of the vaccine. Risk of COVID-19 by vaccination status was also compared using multivariable Cox proportional hazards regression adjusting for propensity to get tested for COVID-19, age, sex, and phase of most recent prior SARS-CoV-2 infection.

ResultsCOVID-19 occurred in 1475 (3%) of 48 344 employees during the 100-day study period. The cumulative incidence of COVID-19 was lower in the &#34;not up-to-date&#34; than in the &#34;up-to-date&#34; state. On multivariable analysis, not being &#34;up-to-date&#34; with COVID-19 vaccination was associated with lower risk of COVID-19 (HR, 0.77; 95% C.I., 0.69-0.86; P-value, &lt;0.001). Results were very similar when those 65 years and older were only considered &#34;up-to-date&#34; after receiving 2 doses of the bivalent vaccine.

ConclusionsSince the XBB lineages became dominant, adults &#34;not up-to-date&#34; by the CDC definition have a lower risk of COVID-19 than those &#34;up-to-date&#34; on COVID-19 vaccination, bringing into question the value of this risk classification definition.

SummaryAmong 48 344 working-aged Cleveland Clinic employees, those not &#34;up-to-date&#34; on COVID-19 vaccination had a lower risk of COVID-19 than those &#34;up-to-date&#34;. The current CDC definition provides a meaningless classification of risk of COVID-19 in the adult population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.23291598">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.23291598" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.23291598">
        <p class="paperTitle">Internal tremors and vibrations in long COVID: a cross-sectional study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.23291598" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.23291598" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, T.; McAlpine, L.; Michelsen, T.; Caraballo, C.; Sawano, M.; Bhattacharjee, B.; Lu, Y.; Iwasaki, A.; Krumholz, H.</p>
        <p class="info">Score: 127.9, Published: 2023-06-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.23291598' target='https://doi.org/10.1101/2023.06.19.23291598'> 10.1101/2023.06.19.23291598</a></p>
        <p class="abstract">BackgroundInternal tremors and vibrations symptoms have been described as part of neurologic disorders but have not been fully described as a part of long COVID. We compared demographics, socioeconomic characteristics, pre-pandemic comorbidities, and new-onset conditions between people with internal tremors and vibrations as part of their long COVID symptoms and people with long COVID but without these symptoms.

MethodsA cross-sectional study, Listen to Immune, Symptom and Treatment Experiences Now (LISTEN), collected data from adults with long COVID. The study sample included 423 participants enrolled between May 2022 and June 2023.

ResultsThe 423 participants had a median age of 46 years (interquartile range, 38-56), 74% were female, 87% were Non-Hispanic White, and 158 (37%) reported &#34;internal tremors, or buzzing/vibration&#34; as a long COVID symptom. Before long COVID, the groups had similar comorbidities. Post-COVID, participants with internal tremors and vibrations had significantly worse health as measured by the Euro-QoL visual analogue scale (median: 40 vs. 50 points, P = 0.007), higher rates of financial difficulties caused by the pandemic and housing insecurity (P &lt; 0.001 for each), and were significantly more likely to have new-onset conditions of mast cell disorders (11% vs. 2.6%), neurologic conditions (22% vs. 8.3%), anxiety (20% vs. 8.7%), and trauma- or stress-related mental health disorders (12% vs. 3.4%) compared with those without internal tremors (Bonferroni-adjusted P &lt; 0.05 for each). Participants with internal tremors also reported significantly higher rates of cardiovascular, gastrointestinal, integumentary, and neurologic long COVID symptoms compared with those without internal tremors (Bonferroni- adjusted P &lt; 0.05 for each).

Conclusions and RelevanceAmong people with long COVID, those with internal tremors and vibrations have more associated symptoms and worse health status, suggesting it may be associated with a severe phenotype of the condition.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.06.23290989">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.06.23290989" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.06.23290989">
        <p class="paperTitle">Metformin reduces SARS-CoV-2 in a Phase 3 Randomized Placebo Controlled Clinical Trial</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.06.23290989" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.06.23290989" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bramante, C. T.; Beckman, K. B.; Mehta, T.; Krager, A. B.; Odde, D. J.; Tignanelli, C. J.; Buse, J. B.; Johnson, D. M.; Watson, R. H. B.; Daniel, J. J.; Liebovitz, D. M.; Nicklas, J. M.; Cohen, K.; Puskarich, M. A.; Belani, H. K.; Siegel, L. K.; Klatt, N. R.; Anderson, B.; Hartman, K. M.; Rao, V.; Hagan, A. A.; Patel, B.; Fenno, S. L.; Avula, N.; Reddy, N. V.; Erickson, S. M.; Fricton, R. D.; Lee, S.; Griffiths, G.; Pullen, M. F.; Thompson, J. L.; Sherwood, N.; Murray, T. A.; Rose, M. R.; Boulware, D. R.; Huling, J. D.</p>
        <p class="info">Score: 1201.7, Published: 2023-06-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.06.23290989' target='https://doi.org/10.1101/2023.06.06.23290989'> 10.1101/2023.06.06.23290989</a></p>
        <p class="abstract">Current antiviral treatment options for SARS-CoV-2 infections are not available globally, cannot be used with many medications, and are limited to virus-specific targets.1-3 Biophysical modeling of SARS-CoV-2 replication predicted that protein translation is an especially attractive target for antiviral therapy.4 Literature review identified metformin, widely known as a treatment for diabetes, as a potential suppressor of protein translation via targeting of the host mTor pathway.5 In vitro, metformin has antiviral activity against RNA viruses including SARS-CoV-2.6,7 In the COVID-OUT phase 3, randomized, placebo-controlled trial of outpatient treatment of COVID-19, metformin had a 42% reduction in ER visits/hospitalizations/death through 14 days; a 58% reduction in hospitalizations/death through 28 days, and a 42% reduction in Long COVID through 10 months.8,9 Here we show viral load analysis of specimens collected in the COVID-OUT trial that the mean SARS-CoV-2 viral load was reduced 3.6-fold with metformin relative to placebo (-0.56 log10 copies/mL; 95%CI, -1.05 to -0.06, p=0.027) while there was no virologic effect for ivermectin or fluvoxamine vs placebo. The metformin effect was consistent across subgroups and with emerging data.10,11 Our results demonstrate, consistent with model predictions, that a safe, widely available,12 well-tolerated, and inexpensive oral medication, metformin, can be repurposed to significantly reduce SARS-CoV-2 viral load.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.18.23291566">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.18.23291566" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.18.23291566">
        <p class="paperTitle">Viral clearance as a surrogate of clinical efficacy for COVID-19 therapies in outpatients: A systematic review and meta-analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.18.23291566" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.18.23291566" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Elias, K. M.; Khan, s. R.; Stadler, E.; Schlub, T. E.; Cromer, D.; Polizzotto, M. N.; Kent, S.; Turner, T.; Davenport, M. P.; Khoury, D. S.</p>
        <p class="info">Score: 48.1, Published: 2023-06-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.18.23291566' target='https://doi.org/10.1101/2023.06.18.23291566'> 10.1101/2023.06.18.23291566</a></p>
        <p class="abstract">BackgroundSurrogates of antiviral efficacy are needed for COVID-19. We investigated the relationship between the virological effect of treatment and clinical efficacy as measured by progression to severe disease in unvaccinated outpatients treated for mild to moderate COVID-19.

MethodsWe searched PubMed, Scopus and medRxiv from inception to 27th September 2022, for randomised controlled trials (RCTs) which tested potential treatments for COVID-19 in non-hospitalized patients. We included studies that reported both clinical and virological outcomes. Clinical outcomes were the rate of disease progression (generally hospitalization or death within 28 days of commencing treatment) and virological outcomes were viral load (viral RNA copies in upper respiratory tract swabs) within the first 7 days of treatment. Studies were excluded if they did not report on the outcome of a primary randomised controlled trial, or if results were reported in a more complete form in another publication. Risk of Bias assessment was performed using the RoB 2.0 tool. We used generalised linear models with random effects to assess the association between outcomes and account for study heterogeneity.

FindingsWe identified 1372 unique studies of which 14 (with a total of 9257 participants) met inclusion criteria. Larger virological treatment effects at both day 3 and day 5 were associated with decreased odds of progression to hospitalisation or death in unvaccinated ambulatory subjects. The odds ratio (OR) for each extra two-fold reduction in viral load in treated compared to control subjects was 0.54 on both days 3 and 5 post treatment (day 3 95% CI 0.38 to 0.74, day 5 95%CI 0.41 to 0.72). There was no relationship between the odds of hospitalisation or death and virological treatment effect at day 7 (OR 0.91, 95%CI 0.74 to 1.13).

InterpretationThis review provides evidence that treatment-induced acceleration of viral clearance within the first 5 days after treatment is a surrogate of clinical efficacy to prevent hospitalisation with COVID-19. Limitations included the aggregation of studies with differing designs, and evidence of risk of bias in some virological outcomes. These findings support the use of viral clearance as an early phase clinical trial endpoint of therapeutic efficacy.

FundingThe authors were supported by the Australian Government Department of Health, Medical Research Future Fund, National Health and Medical Research Council and the University of New South Wales.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.17.23291498">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.17.23291498" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.17.23291498">
        <p class="paperTitle">High burden of viruses and bacterial pathobionts drives heightened nasal innate immunity in children with and without SARS-CoV-2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.17.23291498" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.17.23291498" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Watkins, T. A.; Cheemarla, N. R.; Hansel, K.; Green, A. B.; Amat, J. A. R.; Lozano, R.; Dudgeon, S. N.; Landry, M. L.; Schulz, W. L.; Foxman, E. F.</p>
        <p class="info">Score: 24.2, Published: 2023-06-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.17.23291498' target='https://doi.org/10.1101/2023.06.17.23291498'> 10.1101/2023.06.17.23291498</a></p>
        <p class="abstract">Recent work indicates that heightened nasal innate immunity in children may impact SARS-CoV-2 pathogenesis. Here, we identified drivers of nasal innate immune activation in children using cytokine profiling and multiplex pathogen detection in 291 pediatric nasopharyngeal samples from the 2022 Omicron surge. Nasal viruses and bacterial pathobionts were highly prevalent, especially in younger children (81% of symptomatic and 37% asymptomatic children overall; 91% and 62% in subjects &lt;5 yrs). For SARS-CoV-2, viral load was highest in young children, and viral load in single infections or combined viral loads in coinfections best predicted nasal CXCL10, a biomarker of the mucosal interferon response. Bacterial pathobionts correlated with high nasal IL-1{beta} and TNF, but not CXCL10, and viral-bacterial coinfections showed a combined immunophenotype. These findings reveal virus and bacteria as drivers of heightened nasal innate immunity in children and suggest that frequent host-pathogen interactions shape responses to respiratory viruses in this age group.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.15.23291261">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.15.23291261" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.15.23291261">
        <p class="paperTitle">Rapid nanopore metagenomic sequencing and predictive susceptibility testing of positive blood cultures from intensive care patients with sepsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.15.23291261" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.15.23291261" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harris, P. N. A.; Bauer, M. J.; Lueftinger, L.; Beisken, S.; Forde, B. M.; Balch, R.; Cotta, M. O.; Schlapbach, L.; Raman, S.; Shekar, K.; Kruger, P.; Lipman, J.; Bialasiewicz, S.; Coin, L. J. M.; Roberts, J. A.; Paterson, D. L.; Irwin, A. D.</p>
        <p class="info">Score: 18.1, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.15.23291261' target='https://doi.org/10.1101/2023.06.15.23291261'> 10.1101/2023.06.15.23291261</a></p>
        <p class="abstract">BackgroundDirect metagenomic sequencing from positive blood culture (BC) broths, to identify bacteria and predict antimicrobial susceptibility, has been previously demonstrated using Illumina-based methods, but is relatively slow. We aimed to evaluate this approach using nanopore sequencing to provide more rapid results.

MethodsPatients with suspected sepsis in 4 intensive care units were prospectively enrolled. Human-depleted DNA was extracted from positive BC broths and sequenced using nanopore (MinION). Species abundance was estimated using Kraken2, and a cloud-based artificial intelligence (AI) system (AREScloud) provided in silico antimicrobial susceptibility testing (AST) from assembled contigs. These results were compared to conventional identification and phenotypic AST.

ResultsGenus-level agreement between conventional methods and metagenomic whole genome sequencing (MG-WGS) was 96.2% (50/52), but increased to 100% in monomicrobial infections. In total, 262 high quality AREScloud AST predictions across 24 samples were made, exhibiting categorical agreement (CA) of 89.3%, with major error (MA) and very major error (VME) rates of 10.5% and 12.1%, respectively. Over 90% CA was achieved for some taxa (e.g. Staphylococcus aureus), but was suboptimal for Pseudomonas aeruginosa (CA 50%). In 470 AST predictions across 42 samples, with both high quality and exploratory-only predictions, overall CA, ME and VME rates were 87.7%, 8.3% and 28.4%. VME rates were inflated by false susceptibility calls in a small number of species / antibiotic combinations with few representative resistant isolates. Time to reporting from MG-WGS could be achieved within 8-16 hours from blood culture positivity.

ConclusionsDirect metagenomic sequencing from positive BC broths is feasible and can provide accurate predictive AST for some species and antibiotics, but is sub-optimal for a subset of common pathogens, with unacceptably high VME rates. Nanopore-based approaches may be faster but improvements in accuracy are required before it can be considered for clinical use. New developments in nanopore sequencing technology, and training of AI algorithms on larger and more diverse datasets may improve performance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.23291522">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.23291522" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.23291522">
        <p class="paperTitle">The association of neutralizing antibodies with protection against symptomatic dengue virus infection varies by serotype, prior immunity, and assay conditions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.23291522" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.23291522" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bos, S.; Graber, A. L.; Cardona-Ospina, J. A.; Duarte, E. M.; Zambrana, J. V.; Ruiz Salinas, J. A.; Mercado-Hernandez, R.; Singh, T.; Katzelnick, L. C.; de Silva, A.; Kuan, G.; Balmaseda, A.; Harris, E.</p>
        <p class="info">Score: 12.7, Published: 2023-06-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.23291522' target='https://doi.org/10.1101/2023.06.20.23291522'> 10.1101/2023.06.20.23291522</a></p>
        <p class="abstract">The four dengue virus serotypes (DENV1-4) are the most prevalent arboviruses in humans and a major public health concern worldwide. Identification of the immune mechanisms that modulate DENV infection outcome is critical for development of a safe and effective vaccine. Neutralizing antibodies (nAbs) are considered an essential component of the protective response, yet measurement of their potency is primarily performed using a single cellular substrate and partially mature virions. This approach does not capture the full breadth of neutralizing activity and may lead to biased estimations of nAb potency and repertoire. Here, we evaluated the nAb response associated with protection against dengue cases using samples collected after one or more DENV infections but prior to a subsequent symptomatic versus inapparent DENV1, DENV2 or DENV3 infection from a long-standing pediatric cohort study in Nicaragua. We compared nAb titers in pre-inapparent and pre-symptomatic infection samples in Vero cells with or without DC-SIGN expression, and with either mature or partially mature virions. This method allowed us to measure the magnitude and characteristics of the nAb response associated with outcome and revealed that correlation of nAb titers with protection from symptomatic infection is dependent on the individuals prior DENV immune status as well as the subsequent infecting DENV serotype. Further, the nAb potency and the protective NT50 cutoff was greatly impacted by virion maturation state and cell substrate. These results have important implications for determination of antibody correlates of protection for vaccines and natural infections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.10.28.22281553">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.10.28.22281553" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.10.28.22281553">
        <p class="paperTitle">Tracing the origin of SARS-CoV-2 Omicron-like Spike sequences detected in wastewater</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.10.28.22281553" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.10.28.22281553" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shafer, M. M.; Bobholz, M. J.; Vuyk, W. C.; Gregory, D. A.; Roguet, A.; Haddock Soto, L. A.; Rushford, C.; Janssen, K. H.; Ries, H. J.; Pilch, H. E.; Mullen, P. A.; Fahney, R. B.; Wei, W.; Lambert, M.; Wenzel, J.; Halfmann, P.; Kawaoka, Y.; Wilson, N. A.; Friedrich, T. C.; Pray, I. W.; Westergaard, R.; O&#39;Connor, D. H.; Johnson, M.</p>
        <p class="info">Score: 1124.1, Published: 2023-05-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.10.28.22281553' target='https://doi.org/10.1101/2022.10.28.22281553'> 10.1101/2022.10.28.22281553</a></p>
        <p class="abstract">BackgroundThe origin of divergent SARS-CoV-2 spike sequences found in wastewater, but not in clinical surveillance, remains unclear. These &#34;cryptic&#34; wastewater sequences have harbored many of the same mutations that later emerged in Omicron lineages. We first detected a cryptic lineage in municipal wastewater in Wisconsin in January 2022. Named the &#34;Wisconsin Lineage&#34;, we sought to determine this viruss geographic origin and characterize its persistence and evolution over time.

MethodsWe systematically sampled maintenance holes to trace the Wisconsin Lineages origin. We sequenced spike RBD domains, and where possible, whole viral genomes, to characterize the evolution of this lineage over the 13 consecutive months that it was detectable.

FindingsThe persistence of the Wisconsin Lineage signal allowed us to trace it from a central wastewater plant to a single facility, with a high concentration of viral RNA. The viral sequences contained a combination of fixed nucleotide substitutions characteristic of Pango lineage B.1.234, which circulated in Wisconsin at low levels from October 2020 to February 2021, while mutations in the spike gene resembled those subsequently found in Omicron variants.

InterpretationWe propose that prolonged detection of the Wisconsin Lineage in wastewater represents persistent shedding of SARS-CoV-2 from an infected individual, with ongoing within-host viral evolution leading to an ancestral B.1.234 virus accumulating &#34;Omicron-like&#34; mutations.

FundingThe Rockefeller Foundation, Wisconsin Department of Health Services, Centers for Disease Control and Prevention (CDC), National Institute on Drug Abuse (NIDA), and the Center for Research on Influenza Pathogenesis and Transmission.

Research in context

Evidence before this studyTo identify other studies characterizing unusual wastewater-specific SARS-CoV-2 lineages, we conducted a PubMed search using the keywords &#34;cryptic SARS-CoV-2 lineages,&#34; &#34;novel SARS-CoV-2 lineages,&#34; and &#34;wastewater&#34; on May 9, 2023. From the 18 papers retrieved, only two reported detecting wastewater-specific cryptic lineages. These lineages were identified by investigators from this research team in wastewater from California, Missouri, and New York City, but none could be definitively traced to a specific origin.

A third study in Nevada identified a unique recombinant variant (designated Pango lineage XL) in wastewater, which was also discovered in two clinical samples from the same community. However, it remained unclear whether the clinical samples were collected from the same individual(s) responsible for the virus detected in the wastewater. To our knowledge, no prior study has successfully traced cryptic SARS-CoV-2 wastewater lineages back to a specific location. How and where evolutionarily advanced cryptic lineages are introduced into wastewater has remained uncertain.

Added value of this studyThis study documents the presence and likely source of a novel and highly divergent cryptic SARS-CoV-2 lineage detected in Wisconsin wastewater for 13 months. In contrast to previously reported cryptic lineages, we successfully traced the Wisconsin Lineage to a single facility serving about 30 people. The exceptionally high viral RNA loads at the source account for the detection of these sequences in wastewater collected at the downstream municipal wastewater treatment facility.

Implications of all the available evidenceMany SARS-CoV-2 lineages have been found exclusively in wastewater. The high number of unusual mutations found in these wastewater-specific cryptic sequences raises the possibility that they originate from individual prolonged shedders or even non-human sources. The Wisconsin Lineages persistence in wastewater, single-facility origin, and heavily mutated Omicron-like genotype support the hypothesis that cryptic wastewater lineages arise from persistently infected humans. As these divergent sequences contain amino acid changes that eventually emerge in circulating viruses, increased global monitoring of such lineages in wastewater could help anticipate future circulating mutations and/or variants of concern.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.12.23291295">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.12.23291295" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.12.23291295">
        <p class="paperTitle">Expansion of a low-cost, saliva-based PCR test for the detection of mpox virus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.12.23291295" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.12.23291295" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thomas, R. J.; Yolda-Carr, D.; Fajardo, K.; Allicock, O. M.; Steel, S. A.; Zepeda, T.; Brownlee, M.; Saladi, S.; Parkin, J.; Wyllie, A. L.</p>
        <p class="info">Score: 10.2, Published: 2023-06-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.12.23291295' target='https://doi.org/10.1101/2023.06.12.23291295'> 10.1101/2023.06.12.23291295</a></p>
        <p class="abstract">BackgroundCurrent recommendations for the diagnosis of mpox rely on lesion-swabs as the gold-standard specimen type even though many patients experience symptoms prior to lesion-onset. Earlier detection could bolster the mpox response by mitigating transmission and facilitating access to antiviral treatments.

MethodsWe first compared five PCR assays for their detection of mpox DNA extracted from 30 saliva specimens in collection devices with a stabilizing buffer. Next, we investigated the stability of mpox detection in five raw, unsupplemented saliva samples diluted 1:10 in mpox-negative saliva, after storage at 4{degrees}C, room temperature ([~]19{degrees}C), 30{degrees}C, and 40{degrees}C for 72 hours. We also investigated the stability of virus detection through simulated shipping conditions. Lastly, we performed amplicon sequencing on seven saliva samples and assessed concordance of the PCR assays against mpox virus sequences.

ResultsDespite identifying three different substitutions in the CDCs Monkeypox Virus Generic Real-Time PCR Tests forward and reverse primers, we observed no difference in the mean cycle threshold values generated between assays. However, one gene target for one assay performed better for overall detection when validated. Detection following storage at 4{degrees}C, [~]19{degrees}C, and 30{degrees}C remained relatively stable for 24-48 hours but this declined by 72 hours. At 40{degrees}C, detection was stable at 24 hours but declined by 48 hours. Detection following simulated summer and winter shipping temperature profiles also remained stable.

ConclusionsFindings of this pilot investigation support a flexible, saliva-based, extraction-free PCR test as a promising approach for the low-cost detection of mpox virus. With stability observed for 24-48 hours as well as over simulated shipping temperatures, saliva-based sampling and simplified testing could reduce mpox diagnostic costs, increase access to testing and address hurdles in low- and middle-income countries. Future studies should build upon this and assess the temporal dynamics of mpox virus in saliva.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.23291686">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.23291686" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.23291686">
        <p class="paperTitle">Current sampling and sequencing biases of Lassa mammarenavirus limit inference from phylogeography and molecular epidemiology in Lassa fever endemic regions.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.23291686" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.23291686" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arruda, L. B.; Free, H. B.; Simons, D.; Ansumana, R.; Elton, L.; Haider, N.; Honeyborne, I.; Asogun, D.; McHugh, T. D.; Ntoumi, F.; Zumla, A.; Kock, R.</p>
        <p class="info">Score: 8.0, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.23291686' target='https://doi.org/10.1101/2023.06.20.23291686'> 10.1101/2023.06.20.23291686</a></p>
        <p class="abstract">Lassa fever (LF) is a potentially lethal viral haemorrhagic infection of humans caused by Lassa mammarenavirus (LASV). It is an important endemic zoonotic disease in West Africa with growing evidence for increasing frequency and sizes of outbreaks. Phylogeographic and molecular epidemiology methods have projected expansion of the Lassa fever endemic zone in the context of future global change. The Natal multimammate mouse (Mastomys natalensis) is the predominant LASV reservoir, with few studies investigating the role of other animal species. To explore host sequencing biases, all LASV nucleotide sequences and associated metadata available on GenBank (n = 2,298) were retrieved. Most data originated from Nigeria (54%), Guinea (20%) and Sierra Leone (14%). Data from non-human hosts (n = 703) were limited and only 69 sequences encompassed complete genes. We found a strong positive correlation between the number of confirmed human cases and sequences at the country level (r = 0.93 (95% Confidence Interval = 0.71 - 0.98), p &lt; 0.001) but no correlation exists between confirmed cases and the number of available rodent sequences (r = -0.019 (95% C.I. -0.71 - 0.69), p = 0.96). Spatial modelling of sequencing effort highlighted current biases in locations of available sequences, with increased effort observed in Southern Guinea and Southern Nigeria. Phylogenetic analyses showed geographic clustering of LASV lineages, suggestive of isolated events of human-to-rodent transmission and the emergence of currently circulating strains of LASV from the year 1498 in Nigeria. Overall, the current study highlights significant geographic limitations in LASV surveillance, particularly, in non-human hosts. Further investigation of the non-human reservoir of LASV, alongside expanded surveillance, are required for precise characterisation of the emergence and dispersal of LASV. Accurate surveillance of LASV circulation in non-human hosts is vital to guide early detection and initiation of public health interventions for future Lassa fever outbreaks.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
