<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>infectious diseases</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>infectious diseases | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Vaccine Effectiveness Against Long COVID in Children: A Report from the RECOVER EHR Cohort
Authors: Razzaghi, H.; Forrest, C.; Hirabayshi, K.; Wu, Q.; Allen, A.; Rao, S.; Chen, Y.; Bunnell, T.; Chrischilles, E. A.; Cowell, L. G.; Cummins, M. R.; Hanauer, D. A.; Higginbotham, M. J.; Horne, B.; Horowitz, C. R.; Jhaveri, R.; Kim, S.; Mishkin, A.; Muszynski, J.; Naggie, S.; Pajor, N. M.; Paranjape, A.; Schwenk, H. T.; Sills, M.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/infectious-diseases/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="infectious diseases" />
<meta property="og:description" content="Vaccine Effectiveness Against Long COVID in Children: A Report from the RECOVER EHR Cohort
Authors: Razzaghi, H.; Forrest, C.; Hirabayshi, K.; Wu, Q.; Allen, A.; Rao, S.; Chen, Y.; Bunnell, T.; Chrischilles, E. A.; Cowell, L. G.; Cummins, M. R.; Hanauer, D. A.; Higginbotham, M. J.; Horne, B.; Horowitz, C. R.; Jhaveri, R.; Kim, S.; Mishkin, A.; Muszynski, J.; Naggie, S.; Pajor, N. M.; Paranjape, A.; Schwenk, H. T.; Sills, M." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/infectious-diseases/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-01T10:37:47+00:00" />
<meta property="article:modified_time" content="2023-10-01T10:37:47+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="infectious diseases"/>
<meta name="twitter:description" content="Vaccine Effectiveness Against Long COVID in Children: A Report from the RECOVER EHR Cohort
Authors: Razzaghi, H.; Forrest, C.; Hirabayshi, K.; Wu, Q.; Allen, A.; Rao, S.; Chen, Y.; Bunnell, T.; Chrischilles, E. A.; Cowell, L. G.; Cummins, M. R.; Hanauer, D. A.; Higginbotham, M. J.; Horne, B.; Horowitz, C. R.; Jhaveri, R.; Kim, S.; Mishkin, A.; Muszynski, J.; Naggie, S.; Pajor, N. M.; Paranjape, A.; Schwenk, H. T.; Sills, M."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "infectious diseases",
      "item": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "infectious diseases",
  "name": "infectious diseases",
  "description": "Vaccine Effectiveness Against Long COVID in Children: A Report from the RECOVER EHR Cohort\nAuthors: Razzaghi, H.; Forrest, C.; Hirabayshi, K.; Wu, Q.; Allen, A.; Rao, S.; Chen, Y.; Bunnell, T.; Chrischilles, E. A.; Cowell, L. G.; Cummins, M. R.; Hanauer, D. A.; Higginbotham, M. J.; Horne, B.; Horowitz, C. R.; Jhaveri, R.; Kim, S.; Mishkin, A.; Muszynski, J.; Naggie, S.; Pajor, N. M.; Paranjape, A.; Schwenk, H. T.; Sills, M.",
  "keywords": [
    
  ],
  "articleBody": " Vaccine Effectiveness Against Long COVID in Children: A Report from the RECOVER EHR Cohort\nAuthors: Razzaghi, H.; Forrest, C.; Hirabayshi, K.; Wu, Q.; Allen, A.; Rao, S.; Chen, Y.; Bunnell, T.; Chrischilles, E. A.; Cowell, L. G.; Cummins, M. R.; Hanauer, D. A.; Higginbotham, M. J.; Horne, B.; Horowitz, C. R.; Jhaveri, R.; Kim, S.; Mishkin, A.; Muszynski, J.; Naggie, S.; Pajor, N. M.; Paranjape, A.; Schwenk, H. T.; Sills, M. R.; Tedla, Y. G.; Williams, D. A.; Bailey, C.\nScore: 103.3, Published: 2023-09-28 DOI: 10.1101/2023.09.27.23296100\nObjective Vaccination reduces the risk of acute COVID-19 in children, but it is less clear whether it protects against long COVID. We estimated vaccine effectiveness (VE) against long COVID in children aged 5 to 17 years. Methods This retrospective cohort study used data from 17 health systems in the RECOVER PCORnet electronic health record (EHR) Program for visits between vaccine availability, and October 29, 2022. Conditional logistic regression was used to estimate VE against long COVID with matching on age group (5 to 11, 12 to 17) and time period and adjustment for sex, ethnicity, health system, comorbidity burden, and pre-exposure health care utilization. We examined both probable (symptom-based) and diagnosed long COVID in the year following vaccination. Results The vaccination rate was 56% in the cohort of 1,037,936 children. The incidence of probably long COVID was 4.5% among patients with COVID-19, while diagnosed long COVID was 0.7%. Adjusted vaccine effectiveness within 12 months was 35.4% (95 CI 24.5 - 44.5) against probable long COVID and 41.7% (15.0- 60.0) against diagnosed long COVID. VE was higher for adolescents 50.3% [36.3 - 61.0]) than children aged 5-11 (23.8% [4.9 -39.0]). VE was higher at 6 months (61.4% [51.0 - 69.6]), but decreased to 10.6% (-26.8 - 37.0%) at 18 months. Discussion This large retrospective study shows a moderate protective effect of SARS-CoV-2 vaccination against long COVID. The effect is stronger in adolescents, who have higher risk of long COVID, and wanes over time. Understanding VE mechanism against long COVID requires more study, including EHR sources and prospective data. Discussion This large retrospective study shows a moderate protective effect of SARS-CoV-2 vaccination against long COVID. The effect is stronger in adolescents, who have higher risk of long COVID, and wanes over time. Understanding VE mechanism against long COVID requires more study, including EHR sources and prospective data.\nLiver abnormalities following SARS-CoV-2 infection in children under 10 years of age\nAuthors: Terebuh, P.; Olaker, V. R.; Kendall, E. K.; Kaelber, D. C.; Xu, R.; Davis, P. B.\nScore: 58.6, Published: 2023-09-23 DOI: 10.1101/2023.09.21.23295905\nObjectiveBeginning in October 2021 in the US and elsewhere, cases of severe pediatric hepatitis of unknown etiology were identified in young children. While the adenovirus and adenovirus-associated virus have emerged as leading etiologic suspects, we attempted to investigate a potential role for SARS-CoV-2 in the development of subsequent liver abnormalities. DesignWe conducted a study utilizing retrospective cohorts of de-identified, aggregated data from the electronic health records of over 100 million patients contributed by US health care organizations. ResultsCompared to propensity-score-matched children with other respiratory infections, children aged 1-10 years with COVID-19 had a higher risk of elevated transaminases (Hazard ratio (HR) (95% Confidence interval (CI)) 2.16 (1.74-2.69)) or total bilirubin (HR (CI) 3.02 (1.91-4.78)), or new diagnoses of liver diseases (HR (CI) 1.67 (1.21-2.30)) from one to six months after infection. Patients with pre-existing liver abnormalities, liver abnormalities surrounding acute infection, younger age (1-4 years), or illness requiring hospitalization all had similarly elevated risk. Children who developed liver abnormalities following COVID-19 had more pre-existing conditions than those who developed abnormalities following other infections. ConclusionThese results indicate that SARS-CoV-2 may prime the patient for subsequent development of liver infections or non-infectious liver diseases. While rare ([~]1 in 1,000), SARS-CoV-2 is a risk for subsequent abnormalities in liver function or the diagnosis of diseases of the liver. What is already known on this topicClusters of severe hepatitis in children in 2022 coincident with the increase in COVID-19 infections in children raised the question of the contribution of SARS-CoV-2 to the hepatitis outbreak, though it was soon determined that SARS-CoV-2 was not the primary etiologic agent. What this study addsSARS-CoV-2 may prime the patient for subsequent development of liver infections or non-infectious liver diseases. How this study might affect research, practice or policyDespite the mild initial disease in children, there may be longer term consequences of COVID-19, such as liver abnormalities, that warrants further investigation.\nSafety and Immunogenicity of XBB.1.5-Containing mRNA Vaccines\nAuthors: Chalkias, S.; McGhee, N.; Whatley, J. L.; Essink, B.; Brosz, A.; Tomassini, J. E.; Girard, B.; Wu, K.; Edwards, D. K.; Nasir, A.; Lee, D.; Avena, L. E.; Feng, J.; Deng, W.; Montefiori, D. C.; Baden, L. R.; Miller, J. M.; Das, R.\nScore: 3372.1, Published: 2023-09-07 DOI: 10.1101/2023.08.22.23293434\nBackgroundSubvariants of the severe acute respiratory syndrome coronavirus (SARS-CoV-2) omicron XBB-lineage have the potential to escape immunity provided by prior infection or vaccination. For Covid-19 immunizations beginning in the Fall 2023, the U.S. FDA has recommended updating to a monovalent omicron XBB.1.5-containing vaccine. MethodsIn this ongoing, phase 2/3 study participants were randomized 1:1 to receive 50-{micro}g doses of mRNA-1273.815 monovalent (50-{micro}g omicron XBB.1.5 spike mRNA) or mRNA-1273.231 bivalent (25-{micro}g omicron XBB.1.5 and 25-{micro}g omicron BA.4/BA.5 spike mRNAs) vaccines, administered as 5th doses, to adults who previously received a primary series and 3rd dose of an original mRNA coronavirus disease 2019 (Covid-19) vaccine, and a 4th dose of a bivalent (omicron BA.4/BA.5 and original SARS-CoV-2) vaccine. Interim safety and immunogenicity data 15 days post-vaccination are presented. ResultsIn April 2023, participants received mRNA-1273.815 (n=50) and mRNA-1273.231 (n=51). The median intervals from the prior dose of BA.4/BA.5-containing bivalent vaccine were 8.2 and 8.3 months for the mRNA-1273.815 and mRNA-1273.231 groups, respectively. Both vaccines increased neutralizing antibody (nAb) geometric mean titers against all variants tested at day 15 post-booster nAb compared to pre-booster levels. Geometric mean fold-rises from pre-booster titers after the monovalent booster were numerically higher against XBB.1.5, XBB.1.16 and SARS-CoV-2 (D614G) than those of the bivalent booster and were comparable against BA.4/BA.5 and BQ1.1 variants for both vaccines. The monovalent vaccine also elicited nAb responses against omicron XBB.2.3.2, EG.5.1, FL.1.5.1 and BA.2.86 that were similar to those against XBB.1.5 in a subset (n=20) of participants. The occurrence of solicited adverse reactions and unsolicited adverse events were overall similar to those previously reported for the original mRNA-1273 50-{micro}g and omicron BA.4/BA.5-containing bivalent mRNA-1273 vaccines. ConclusionIn this interim analysis, XBB.1.5-containing monovalent and bivalent vaccines elicited potent neutralizing responses against variants of the omicron XBB-lineage (XBB.1.5, XBB.1.6, XBB.2.3.2, EG.5.1, and FL.1.5.1) as well as the recently emerged BA.2.86 variant. The safety profile of the XBB.1.5-containing vaccine was consistent with those of prior vaccines. These results overall indicate that the XBB.1.5-containing mRNA-1273.815 vaccine has the potential to provide protection against these emerging variants and support the Covid-19 vaccine update in 2023-2024 to a monovalent XBB.1.5-containing vaccine.\nQuantity of SARS-CoV-2 RNA copies exhaled per minute during natural breathing over the course of COVID-19 infection\nAuthors: Lane, G.; Zhou, G.; Hultquist, J. F.; Simons, L. M.; Lorenzo-Redondo, R.; Ozer, E. A.; McCarthy, D. M.; Ison, M. G.; Achenbach, C. J.; Wang, X.; Wai, C. M.; Wyatt, E.; Aalsburg, A.; Yang, Q.; Noto, T.; Alisoltani, A.; Ysselstein, D.; Awatramani, R.; Murphy, R.; Theron, G.; Zelano, C.\nScore: 621.9, Published: 2023-09-08 DOI: 10.1101/2023.09.06.23295138\nSARS-CoV-2 is spread through exhaled breath of infected individuals. A fundamental question in understanding transmission of SARS-CoV-2 is how much virus an individual is exhaling into the environment while they breathe, over the course of their infection. Research on viral load dynamics during COVID-19 infection has focused on internal swab specimens, which provide a measure of viral loads inside the respiratory tract, but not on breath. Therefore, the dynamics of viral shedding on exhaled breath over the course of infection are poorly understood. Here, we collected exhaled breath specimens from COVID-19 patients and used RTq-PCR to show that numbers of exhaled SARS-CoV-2 RNA copies during COVID-19 infection do not decrease significantly until day 8 from symptom-onset. COVID-19-positive participants exhaled an average of 80 SARS-CoV-2 viral RNA copies per minute during the first 8 days of infection, with significant variability both between and within individuals, including spikes over 800 copies a minute in some patients. After day 8, there was a steep drop to levels nearing the limit of detection, persisting for up to 20 days. We further found that levels of exhaled viral RNA increased with self-rated symptom-severity, though individual variation was high. Levels of exhaled viral RNA did not differ across age, sex, time of day, vaccination status or viral variant. Our data provide a fine-grained, direct measure of the number of SARS-CoV-2 viral copies exhaled per minute during natural breathing--including 312 breath specimens collected multiple times daily over the course of infection--in order to fill an important gap in our understanding of the time course of exhaled viral loads in COVID-19.\nRisk of Coronavirus Disease 2019 (COVID-19) among Those Up-to-Date and Not Up-to-Date on COVID-19 Vaccination by US CDC Criteria\nAuthors: Shrestha, N. K.; Burke, P. C.; Nowacki, A. S.; Gordon, S. M.\nScore: 5134.0, Published: 2023-09-14 DOI: 10.1101/2023.06.09.23290893\nBackgroundThe CDC recently defined being \"up-to-date\" on COVID-19 vaccination as having received at least one dose of a COVID-19 bivalent vaccine. The purpose of this study was to compare the risk of COVID-19 among those \"up-to-date\" and \"not up-to-date\". MethodsEmployees of Cleveland Clinic in Ohio, USA, in employment when the COVID-19 bivalent vaccine first became available, and still employed when the XBB lineages became dominant, were included. Cumulative incidence of COVID-19 since the XBB lineages became dominant was compared across the \"up-to-date\" and \"not up-to-date\" states, by treating COVID-19 bivalent vaccination as a time- dependent covariate whose value changed on receipt of the vaccine. Risk of COVID-19 by vaccination status was also evaluated using multivariable Cox proportional hazards regression adjusting for propensity to get tested for COVID-19, age, sex, most recent prior SARS-CoV-2 infection, and number of prior vaccine doses. ResultsCOVID-19 occurred in 1475 (3%) of 48 344 employees during the 100-day study period. The cumulative incidence of COVID-19 was lower in the \"not up-to-date\" than the \"up-to-date\" state. On multivariable analysis, being \"up-to-date\" was not associated with lower risk of COVID-19 (HR, 1.05; 95% C.I., 0.88-1.25; P-value, 0.58). Results were very similar when those 65 years and older were only considered \"up-to-date\" after 2 doses of the bivalent vaccine. ConclusionsSince the XBB lineages became dominant, adults \"up-to-date\" on COVID-19 vaccination by the CDC definition do not have a lower risk of COVID-19 than those \"not up-to-date\", bringing into question the value of this risk classification definition. SummaryAmong 48 344 working-aged Cleveland Clinic employees, those \"up-to-date\" on COVID-19 vaccination did not have a lower risk of COVID-19 than those not \"up-to-date\". The current CDC definition provides a meaningless classification of risk of COVID-19 in the adult population.\nHuman origin ascertained for SARS-CoV-2 Omicron-like spike sequences detected in wastewater: a targeted surveillance study of a cryptic lineage in an urban sewershed\nAuthors: Shafer, M. M.; Bobholz, M. J.; Vuyk, W. C.; Gregory, D. A.; Roguet, A.; Haddock Soto, L. A.; Rushford, C.; Janssen, K. H.; Emmen, I. E.; Ries, H. J.; Pilch, H. E.; Mullen, P. A.; Fahney, R. B.; Wei, W.; Lambert, M.; Wenzel, J.; Halfmann, P.; Kawaoka, Y.; Wilson, N. A.; Friedrich, T. C.; Pray, I. W.; Westergaard, R.; O'Connor, D. H.; Johnson, M.\nScore: 1111.8, Published: 2023-09-22 DOI: 10.1101/2022.10.28.22281553\nBackgroundThe origin of novel SARS-CoV-2 spike sequences found in wastewater, without corresponding detection in clinical specimens, remains unclear. We sought to determine the origin of one such \"cryptic\" wastewater lineage by tracking and characterizing its persistence and genomic evolution over time. MethodsWe first detected a cryptic lineage in Wisconsin municipal wastewater in January 2022. By systematically sampling wastewater from targeted sub-sewershed lines and maintenance holes using compositing autosamplers, we traced this lineage (labeled WI-CL-001) to its source at a single commercial building. There we detected WI-CL-001 at concentrations as high as 2.7 x 109 genome copies per liter (gc/L) via RT-dPCR. In addition to using metagenomic 12s rRNA sequencing to determine the viruss host species, we also sequenced SARS-CoV-2 spike receptor binding domains (RBDs), and where possible, whole viral genomes to identify and characterize the evolution of this lineage over the 13 consecutive months that it was detectable. FindingsThe vast majority of 12s rRNAs sequenced from wastewater leaving the identified source building were human. Additionally, we generated over 100 viral RBD and whole genome sequences from wastewater samples containing the cryptic lineage collected between January 2022 and January 2023. These sequences contained a combination of fixed nucleotide substitutions characteristic of Pango lineage B.1.234, which circulated in humans in Wisconsin at low levels from October 2020 to February 2021. Despite this, mutations in the spike gene, and elsewhere, resembled those subsequently found in Omicron variants. InterpretationWe propose that prolonged detection of WI-CL-001 in wastewater represents persistent shedding of SARS-CoV-2 from a single human initially infected by an ancestral B.1.234 virus. The accumulation of convergent \"Omicron-like\" mutations in WI-CL-001s ancestral B.1.234 genome likely reflects persistent infection and extensive within-host evolution. FundingThe Rockefeller Foundation, Wisconsin Department of Health Services, Centers for Disease Control and Prevention (CDC), National Institute on Drug Abuse (NIDA), and the Center for Research on Influenza Pathogenesis and Transmission. Research in contextO_ST_ABSEvidence before this studyC_ST_ABSTo identify other studies that characterized unusual wastewater-specific SARS-CoV-2 lineages, we conducted a PubMed search using the keywords \"cryptic SARS-CoV-2 lineages\" or \"novel SARS-CoV-2 lineages\" in addition to \"wastewater\" on May 9, 2023. From the 18 papers retrieved, only two reported wastewater-specific cryptic lineages. These lineages were identified by members of our author team in wastewater from California, Missouri, and New York City. None of these could be definitively traced to a specific source. A third study in Nevada identified a unique recombinant variant (designated Pango lineage XL) in wastewater, which was also discovered in two clinical specimens from the same community. However, it was unclear whether the clinical specimens collected were from the same individual(s) responsible for the virus detected in the wastewater. To our knowledge, no prior study has successfully traced novel SARS-CoV-2 lineages detected in wastewater back to a specific location. How and where cryptic lineages are introduced into wastewater is not known. The added value of this studyThis study documents the presence and likely source of a novel and highly divergent cryptic SARS-CoV-2 lineage detected in Wisconsin wastewater for 13 months. In contrast to previously reported cryptic lineages, we successfully traced the lineage (WI-CL-001) to a single commercial building with approximately 30 employees. The exceptionally high viral RNA concentrations at the source building facilitated the tracing effort and allowed for the sequencing of WI-CL-001s whole genome, expanding our view of the lineages mutational landscape beyond the spike gene. Implications of all the available evidenceWI-CL-001s persistence in wastewater, its heavily mutated Omicron-like genotype, and its identified point source at a human-occupied commercial building all support the hypothesis that cryptic wastewater lineages can arise from persistently infected humans. Because cryptic wastewater lineages have some amino acid changes that subsequently emerge in circulating viruses, increased global monitoring of such lineages could help forecast variants that may arise in the future.\nComparative performance of COVID-19 test methods in healthcare workers during the omicron wave\nAuthors: Tornberg, E. C.; Tomlinson, A.; Oshiro, N. T. T.; Derfalie, E.; Curlin, M.\nScore: 14.5, Published: 2023-09-26 DOI: 10.1101/2023.09.25.23296099\nIntroductionThe COVID-19 pandemic presents unique requirements for accessible, reliable testing, and many testing platforms and sampling techniques have been developed. However, not all test methods have been systematically compared to each other or a common gold standard, and the performance of tests developed in the early epidemic have not been consistently re-evaluated in the context of newly emerging SARS-CoV-2 variants. MethodsWe conducted a repeated measures study with adult healthcare workers presenting for SARS-CoV-2 testing. Participants were tested using seven test modalities: PCR with samples from the nasopharynx, oropharynx, and saliva; and BinaxNOW and iHealth antigen-based rapid detection tests (AgRDT) sampling the oropharynx and the nares. Test sensitivity was compared using any positive PCR test as the gold standard. Results325 individuals participated in the study. PCR tests were the most sensitive with saliva PCR at 0.957 {+/-} 0.048, nasopharyngeal PCR at 0.877 {+/-} 0.075, and oropharyngeal PCR at 0.849 {+/-} 0.082. Standard nasal rapid antigen tests were less sensitive but roughly equivalent at 0.613 {+/-} 0.110 for BinaxNOW brand and 0.627 {+/-} 0.109 for iHealth. Oropharyngeal rapid antigen tests were the least sensitive with BinaxNOW and iHealth brands at 0.400 {+/-} 0.111 and 0.311 {+/-} 0.105 respectively. ConclusionPCR remains the most sensitive testing modality for COVID-19, with saliva PCR being significantly more sensitive than oropharyngeal PCR and equivalent to nasopharyngeal PCR. Saliva testing has patient comfort and financial benefits, making it a preferred testing modality. Nasal AgRDTs are less sensitive than PCR though more accessible and convenient.\nPre-dominance of dengue non-cross-reacting SARS-CoV-2 spike antibodies during the Omicron era and their role in the ADE-mediated surge of Dengue virus serotype 3\nAuthors: Sarker, S.; Dutta, C.; Mallick, A.; Das, S.; Das Chowdhury, C.; De, A.; Gorai, S.; Biswas, S.\nScore: 40.5, Published: 2023-09-10 DOI: 10.1101/2023.09.08.23295136\nIn India, the SARS-CoV-2 Delta wave (2020-21) gradually faded away with the advent of the Omicron variants (2021-present). Dengue incidences were observably less in Southeast Asia during the active years of the pandemic (2020-21). However, Dengue virus type 3 (DV3) cases were increasingly reported in India and many other dengue-endemic countries concurrent with the progression of the Omicron wave since 2022. This observation prompted us to investigate the current state of cross-reactivity between prevalent SARS-CoV-2 variants and different DV serotypes. Fifty-five COVID-19 serum samples (collected between January-September, 2022) and three pre-pandemic healthy serum samples were tested for DV or SARS-CoV-2 Immunoglobulin G/Immunoglobulin M (IgG/IgM) using the lateral flow immunoassays (LFIAs). The SARS-CoV-2 antibody (Ab)-positive samples were further tested for their ability to cross-neutralize DV types 1-4 (DV1-4) in Huh7 cell lines. Cross-reactivity between SARS-CoV-2 and DV diminished with the shift from Delta to Omicron prevalence. COVID-19 serum samples that were DV cross-reactive neutralized all DV serotypes, including DV3. However, Omicron wave serum samples were predominantly DV non-cross-reactive (about 70%) in LFIAs and coincided with the prevalence of BA.2 Omicron variant. They also cross-neutralized DV1, 2 and 4 but enhanced DV3 infectivity as evident from increased DV3 titres in virus neutralization test (VNT) compared to control serum samples. In conclusion, DV non-cross-reactive COVID-19 serum samples are becoming increasingly prominent in the present times. These non-cross-reactive serums could neutralize DV1, 2 and 4 but they are contributing to the surge in DV3 cases worldwide by means of ADE.\nPositive selection underlies repeated knockout of ORF8 in SARS-CoV-2 evolution\nAuthors: Wagner, C.; Kistler, K. E.; Perchetti, G. A.; Baker, N.; Frisbie, L. A.; Torres, L. M.; Aragona, F.; Yun, C.; Figgins, M.; Greninger, A. L.; Cox, A.; Oltean, H. N.; Roychoudhury, P.; Bedford, T. D.\nScore: 10.6, Published: 2023-09-23 DOI: 10.1101/2023.09.21.23295927\nKnockout of the ORF8 protein has repeatedly spread through the global viral population during SARS-CoV-2 evolution. Here we use both regional and global pathogen sequencing to explore the selection pressures underlying its loss. In Washington State, we identified transmission clusters with ORF8 knockout throughout SARS-CoV-2 evolution, not just on novel, high fitness viral backbones. Indeed, ORF8 is truncated more frequently and knockouts circulate for longer than for any other gene. Using a global phylogeny, we find evidence of positive selection to explain this phenomenon: nonsense mutations resulting in shortened protein products occur more frequently and are associated with faster clade growth rates than synonymous mutations in ORF8. Loss of ORF8 is also associated with reduced clinical severity, highlighting the diverse clinical impacts of SARS-CoV-2 evolution.\nLongitudinal analysis within one hospital in sub-Saharan Africa over 20 years reveals repeated replacements of dominant clones of Klebsiella pneumoniae and stresses the importance to include temporal patterns for vaccine design considerations\nAuthors: Heinz, E.; Pearse, O.; Zuza, A.; Bilima, S.; Msefula, C.; Musicha, P.; Siyabu, P.; Tewesa, E.; Graf, F. E.; Lester, R.; Lissauer, S.; Cornick, J.; Lewis, J. M.; Kawaza, K.; Thomson, N. R.; Feasey, N. A.\nScore: 8.2, Published: 2023-09-27 DOI: 10.1101/2023.09.26.23296137\nInfections caused by multidrug-resistant gram-negative bacteria present a severe threat to global public health. The WHO defines drug-resistant Klebsiella pneumoniae as a priority pathogen for which alternative treatments are needed given the limited treatment options and the rapid acquisition of novel resistance mechanisms by this species. Longitudinal descriptions of genomic epidemiology of Klebsiella pneumoniae can inform management strategies but data from sub-Saharan Africa are lacking. We present a longitudinal analysis of all invasive K. pneumoniae isolates from a single hospital in Blantyre, Malawi, southern Africa, from 1998-2020, combining clinical data with genome sequence analysis of the isolates. We show that after a dramatic increase in the number of infections from 2016 K. pneumoniae becomes hyperendemic, driven by an increase in neonatal infections. Genomic data show repeated waves of clonal expansion of different, often ward-restricted, lineages, suggestive of hospital associated transmission. We describe temporal trends in resistance and surface antigens, of relevance for vaccine development. Our data highlight a clear need for new interventions to prevent rather than treat K. pneumoniae infections in our setting. Whilst one option may be a vaccine, the majority of cases could be avoided by an increased focus on and investment in infection prevention and control measures, which would reduce all healthcare associated infections and not just one.\n",
  "wordCount" : "3561",
  "inLanguage": "en",
  "datePublished": "2023-10-01T10:37:47Z",
  "dateModified": "2023-10-01T10:37:47Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/infectious-diseases/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      infectious diseases
    </h1>
    <div class="post-meta"><span>updated on October 1, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.23296100">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.23296100" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.23296100">
        <p class="paperTitle">Vaccine Effectiveness Against Long COVID in Children: A Report from the RECOVER EHR Cohort</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.23296100" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.23296100" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Razzaghi, H.; Forrest, C.; Hirabayshi, K.; Wu, Q.; Allen, A.; Rao, S.; Chen, Y.; Bunnell, T.; Chrischilles, E. A.; Cowell, L. G.; Cummins, M. R.; Hanauer, D. A.; Higginbotham, M. J.; Horne, B.; Horowitz, C. R.; Jhaveri, R.; Kim, S.; Mishkin, A.; Muszynski, J.; Naggie, S.; Pajor, N. M.; Paranjape, A.; Schwenk, H. T.; Sills, M. R.; Tedla, Y. G.; Williams, D. A.; Bailey, C.</p>
        <p class="info">Score: 103.3, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.23296100' target='https://doi.org/10.1101/2023.09.27.23296100'> 10.1101/2023.09.27.23296100</a></p>
        <p class="abstract">Objective Vaccination reduces the risk of acute COVID-19 in children, but it is less clear whether it protects against long COVID. We estimated vaccine effectiveness (VE) against long COVID in children aged 5 to 17 years. Methods This retrospective cohort study used data from 17 health systems in the RECOVER PCORnet electronic health record (EHR) Program for visits between vaccine availability, and October 29, 2022. Conditional logistic regression was used to estimate VE against long COVID with matching on age group (5 to 11, 12 to 17) and time period and adjustment for sex, ethnicity, health system, comorbidity burden, and pre-exposure health care utilization. We examined both probable (symptom-based) and diagnosed long COVID in the year following vaccination. Results The vaccination rate was 56% in the cohort of 1,037,936 children. The incidence of probably long COVID was 4.5% among patients with COVID-19, while diagnosed long COVID was 0.7%. Adjusted vaccine effectiveness within 12 months was 35.4% (95 CI 24.5 - 44.5) against probable long COVID and 41.7% (15.0- 60.0) against diagnosed long COVID. VE was higher for adolescents 50.3% [36.3 - 61.0]) than children aged 5-11 (23.8% [4.9 -39.0]). VE was higher at 6 months (61.4% [51.0 - 69.6]), but decreased to 10.6% (-26.8 - 37.0%) at 18 months. Discussion This large retrospective study shows a moderate protective effect of SARS-CoV-2 vaccination against long COVID. The effect is stronger in adolescents, who have higher risk of long COVID, and wanes over time. Understanding VE mechanism against long COVID requires more study, including EHR sources and prospective data. Discussion This large retrospective study shows a moderate protective effect of SARS-CoV-2 vaccination against long COVID. The effect is stronger in adolescents, who have higher risk of long COVID, and wanes over time. Understanding VE mechanism against long COVID requires more study, including EHR sources and prospective data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.23295905">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.23295905" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.23295905">
        <p class="paperTitle">Liver abnormalities following SARS-CoV-2 infection in children under 10 years of age</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.23295905" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.23295905" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Terebuh, P.; Olaker, V. R.; Kendall, E. K.; Kaelber, D. C.; Xu, R.; Davis, P. B.</p>
        <p class="info">Score: 58.6, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.23295905' target='https://doi.org/10.1101/2023.09.21.23295905'> 10.1101/2023.09.21.23295905</a></p>
        <p class="abstract">ObjectiveBeginning in October 2021 in the US and elsewhere, cases of severe pediatric hepatitis of unknown etiology were identified in young children. While the adenovirus and adenovirus-associated virus have emerged as leading etiologic suspects, we attempted to investigate a potential role for SARS-CoV-2 in the development of subsequent liver abnormalities.

DesignWe conducted a study utilizing retrospective cohorts of de-identified, aggregated data from the electronic health records of over 100 million patients contributed by US health care organizations.

ResultsCompared to propensity-score-matched children with other respiratory infections, children aged 1-10 years with COVID-19 had a higher risk of elevated transaminases (Hazard ratio (HR) (95% Confidence interval (CI)) 2.16 (1.74-2.69)) or total bilirubin (HR (CI) 3.02 (1.91-4.78)), or new diagnoses of liver diseases (HR (CI) 1.67 (1.21-2.30)) from one to six months after infection. Patients with pre-existing liver abnormalities, liver abnormalities surrounding acute infection, younger age (1-4 years), or illness requiring hospitalization all had similarly elevated risk. Children who developed liver abnormalities following COVID-19 had more pre-existing conditions than those who developed abnormalities following other infections.

ConclusionThese results indicate that SARS-CoV-2 may prime the patient for subsequent development of liver infections or non-infectious liver diseases. While rare ([~]1 in 1,000), SARS-CoV-2 is a risk for subsequent abnormalities in liver function or the diagnosis of diseases of the liver.

What is already known on this topicClusters of severe hepatitis in children in 2022 coincident with the increase in COVID-19 infections in children raised the question of the contribution of SARS-CoV-2 to the hepatitis outbreak, though it was soon determined that SARS-CoV-2 was not the primary etiologic agent.

What this study addsSARS-CoV-2 may prime the patient for subsequent development of liver infections or non-infectious liver diseases.

How this study might affect research, practice or policyDespite the mild initial disease in children, there may be longer term consequences of COVID-19, such as liver abnormalities, that warrants further investigation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.23293434">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.23293434" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.23293434">
        <p class="paperTitle">Safety and Immunogenicity of XBB.1.5-Containing mRNA Vaccines</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.23293434" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.23293434" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chalkias, S.; McGhee, N.; Whatley, J. L.; Essink, B.; Brosz, A.; Tomassini, J. E.; Girard, B.; Wu, K.; Edwards, D. K.; Nasir, A.; Lee, D.; Avena, L. E.; Feng, J.; Deng, W.; Montefiori, D. C.; Baden, L. R.; Miller, J. M.; Das, R.</p>
        <p class="info">Score: 3372.1, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.23293434' target='https://doi.org/10.1101/2023.08.22.23293434'> 10.1101/2023.08.22.23293434</a></p>
        <p class="abstract">BackgroundSubvariants of the severe acute respiratory syndrome coronavirus (SARS-CoV-2) omicron XBB-lineage have the potential to escape immunity provided by prior infection or vaccination. For Covid-19 immunizations beginning in the Fall 2023, the U.S. FDA has recommended updating to a monovalent omicron XBB.1.5-containing vaccine.

MethodsIn this ongoing, phase 2/3 study participants were randomized 1:1 to receive 50-{micro}g doses of mRNA-1273.815 monovalent (50-{micro}g omicron XBB.1.5 spike mRNA) or mRNA-1273.231 bivalent (25-{micro}g omicron XBB.1.5 and 25-{micro}g omicron BA.4/BA.5 spike mRNAs) vaccines, administered as 5th doses, to adults who previously received a primary series and 3rd dose of an original mRNA coronavirus disease 2019 (Covid-19) vaccine, and a 4th dose of a bivalent (omicron BA.4/BA.5 and original SARS-CoV-2) vaccine. Interim safety and immunogenicity data 15 days post-vaccination are presented.

ResultsIn April 2023, participants received mRNA-1273.815 (n=50) and mRNA-1273.231 (n=51). The median intervals from the prior dose of BA.4/BA.5-containing bivalent vaccine were 8.2 and 8.3 months for the mRNA-1273.815 and mRNA-1273.231 groups, respectively. Both vaccines increased neutralizing antibody (nAb) geometric mean titers against all variants tested at day 15 post-booster nAb compared to pre-booster levels. Geometric mean fold-rises from pre-booster titers after the monovalent booster were numerically higher against XBB.1.5, XBB.1.16 and SARS-CoV-2 (D614G) than those of the bivalent booster and were comparable against BA.4/BA.5 and BQ1.1 variants for both vaccines. The monovalent vaccine also elicited nAb responses against omicron XBB.2.3.2, EG.5.1, FL.1.5.1 and BA.2.86 that were similar to those against XBB.1.5 in a subset (n=20) of participants. The occurrence of solicited adverse reactions and unsolicited adverse events were overall similar to those previously reported for the original mRNA-1273 50-{micro}g and omicron BA.4/BA.5-containing bivalent mRNA-1273 vaccines.

ConclusionIn this interim analysis, XBB.1.5-containing monovalent and bivalent vaccines elicited potent neutralizing responses against variants of the omicron XBB-lineage (XBB.1.5, XBB.1.6, XBB.2.3.2, EG.5.1, and FL.1.5.1) as well as the recently emerged BA.2.86 variant. The safety profile of the XBB.1.5-containing vaccine was consistent with those of prior vaccines. These results overall indicate that the XBB.1.5-containing mRNA-1273.815 vaccine has the potential to provide protection against these emerging variants and support the Covid-19 vaccine update in 2023-2024 to a monovalent XBB.1.5-containing vaccine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.23295138">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.23295138" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.23295138">
        <p class="paperTitle">Quantity of SARS-CoV-2 RNA copies exhaled per minute during natural breathing over the course of COVID-19 infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.23295138" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.23295138" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lane, G.; Zhou, G.; Hultquist, J. F.; Simons, L. M.; Lorenzo-Redondo, R.; Ozer, E. A.; McCarthy, D. M.; Ison, M. G.; Achenbach, C. J.; Wang, X.; Wai, C. M.; Wyatt, E.; Aalsburg, A.; Yang, Q.; Noto, T.; Alisoltani, A.; Ysselstein, D.; Awatramani, R.; Murphy, R.; Theron, G.; Zelano, C.</p>
        <p class="info">Score: 621.9, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.23295138' target='https://doi.org/10.1101/2023.09.06.23295138'> 10.1101/2023.09.06.23295138</a></p>
        <p class="abstract">SARS-CoV-2 is spread through exhaled breath of infected individuals. A fundamental question in understanding transmission of SARS-CoV-2 is how much virus an individual is exhaling into the environment while they breathe, over the course of their infection. Research on viral load dynamics during COVID-19 infection has focused on internal swab specimens, which provide a measure of viral loads inside the respiratory tract, but not on breath. Therefore, the dynamics of viral shedding on exhaled breath over the course of infection are poorly understood. Here, we collected exhaled breath specimens from COVID-19 patients and used RTq-PCR to show that numbers of exhaled SARS-CoV-2 RNA copies during COVID-19 infection do not decrease significantly until day 8 from symptom-onset. COVID-19-positive participants exhaled an average of 80 SARS-CoV-2 viral RNA copies per minute during the first 8 days of infection, with significant variability both between and within individuals, including spikes over 800 copies a minute in some patients. After day 8, there was a steep drop to levels nearing the limit of detection, persisting for up to 20 days. We further found that levels of exhaled viral RNA increased with self-rated symptom-severity, though individual variation was high. Levels of exhaled viral RNA did not differ across age, sex, time of day, vaccination status or viral variant. Our data provide a fine-grained, direct measure of the number of SARS-CoV-2 viral copies exhaled per minute during natural breathing--including 312 breath specimens collected multiple times daily over the course of infection--in order to fill an important gap in our understanding of the time course of exhaled viral loads in COVID-19.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.09.23290893">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.09.23290893" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.09.23290893">
        <p class="paperTitle">Risk of Coronavirus Disease 2019 (COVID-19) among Those Up-to-Date and Not Up-to-Date on COVID-19 Vaccination by US CDC Criteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.09.23290893" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.09.23290893" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shrestha, N. K.; Burke, P. C.; Nowacki, A. S.; Gordon, S. M.</p>
        <p class="info">Score: 5134.0, Published: 2023-09-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.09.23290893' target='https://doi.org/10.1101/2023.06.09.23290893'> 10.1101/2023.06.09.23290893</a></p>
        <p class="abstract">BackgroundThe CDC recently defined being &#34;up-to-date&#34; on COVID-19 vaccination as having received at least one dose of a COVID-19 bivalent vaccine. The purpose of this study was to compare the risk of COVID-19 among those &#34;up-to-date&#34; and &#34;not up-to-date&#34;.

MethodsEmployees of Cleveland Clinic in Ohio, USA, in employment when the COVID-19 bivalent vaccine first became available, and still employed when the XBB lineages became dominant, were included. Cumulative incidence of COVID-19 since the XBB lineages became dominant was compared across the &#34;up-to-date&#34; and &#34;not up-to-date&#34; states, by treating COVID-19 bivalent vaccination as a time- dependent covariate whose value changed on receipt of the vaccine. Risk of COVID-19 by vaccination status was also evaluated using multivariable Cox proportional hazards regression adjusting for propensity to get tested for COVID-19, age, sex, most recent prior SARS-CoV-2 infection, and number of prior vaccine doses.

ResultsCOVID-19 occurred in 1475 (3%) of 48 344 employees during the 100-day study period. The cumulative incidence of COVID-19 was lower in the &#34;not up-to-date&#34; than the &#34;up-to-date&#34; state. On multivariable analysis, being &#34;up-to-date&#34; was not associated with lower risk of COVID-19 (HR, 1.05; 95% C.I., 0.88-1.25; P-value, 0.58). Results were very similar when those 65 years and older were only considered &#34;up-to-date&#34; after 2 doses of the bivalent vaccine.

ConclusionsSince the XBB lineages became dominant, adults &#34;up-to-date&#34; on COVID-19 vaccination by the CDC definition do not have a lower risk of COVID-19 than those &#34;not up-to-date&#34;, bringing into question the value of this risk classification definition.

SummaryAmong 48 344 working-aged Cleveland Clinic employees, those &#34;up-to-date&#34; on COVID-19 vaccination did not have a lower risk of COVID-19 than those not &#34;up-to-date&#34;. The current CDC definition provides a meaningless classification of risk of COVID-19 in the adult population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.10.28.22281553">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.10.28.22281553" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.10.28.22281553">
        <p class="paperTitle">Human origin ascertained for SARS-CoV-2 Omicron-like spike sequences detected in wastewater: a targeted surveillance study of a cryptic lineage in an urban sewershed</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.10.28.22281553" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.10.28.22281553" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shafer, M. M.; Bobholz, M. J.; Vuyk, W. C.; Gregory, D. A.; Roguet, A.; Haddock Soto, L. A.; Rushford, C.; Janssen, K. H.; Emmen, I. E.; Ries, H. J.; Pilch, H. E.; Mullen, P. A.; Fahney, R. B.; Wei, W.; Lambert, M.; Wenzel, J.; Halfmann, P.; Kawaoka, Y.; Wilson, N. A.; Friedrich, T. C.; Pray, I. W.; Westergaard, R.; O&#39;Connor, D. H.; Johnson, M.</p>
        <p class="info">Score: 1111.8, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.10.28.22281553' target='https://doi.org/10.1101/2022.10.28.22281553'> 10.1101/2022.10.28.22281553</a></p>
        <p class="abstract">BackgroundThe origin of novel SARS-CoV-2 spike sequences found in wastewater, without corresponding detection in clinical specimens, remains unclear. We sought to determine the origin of one such &#34;cryptic&#34; wastewater lineage by tracking and characterizing its persistence and genomic evolution over time.

MethodsWe first detected a cryptic lineage in Wisconsin municipal wastewater in January 2022. By systematically sampling wastewater from targeted sub-sewershed lines and maintenance holes using compositing autosamplers, we traced this lineage (labeled WI-CL-001) to its source at a single commercial building. There we detected WI-CL-001 at concentrations as high as 2.7 x 109 genome copies per liter (gc/L) via RT-dPCR. In addition to using metagenomic 12s rRNA sequencing to determine the viruss host species, we also sequenced SARS-CoV-2 spike receptor binding domains (RBDs), and where possible, whole viral genomes to identify and characterize the evolution of this lineage over the 13 consecutive months that it was detectable.

FindingsThe vast majority of 12s rRNAs sequenced from wastewater leaving the identified source building were human. Additionally, we generated over 100 viral RBD and whole genome sequences from wastewater samples containing the cryptic lineage collected between January 2022 and January 2023. These sequences contained a combination of fixed nucleotide substitutions characteristic of Pango lineage B.1.234, which circulated in humans in Wisconsin at low levels from October 2020 to February 2021. Despite this, mutations in the spike gene, and elsewhere, resembled those subsequently found in Omicron variants.

InterpretationWe propose that prolonged detection of WI-CL-001 in wastewater represents persistent shedding of SARS-CoV-2 from a single human initially infected by an ancestral B.1.234 virus. The accumulation of convergent &#34;Omicron-like&#34; mutations in WI-CL-001s ancestral B.1.234 genome likely reflects persistent infection and extensive within-host evolution.

FundingThe Rockefeller Foundation, Wisconsin Department of Health Services, Centers for Disease Control and Prevention (CDC), National Institute on Drug Abuse (NIDA), and the Center for Research on Influenza Pathogenesis and Transmission.

Research in contextO_ST_ABSEvidence before this studyC_ST_ABSTo identify other studies that characterized unusual wastewater-specific SARS-CoV-2 lineages, we conducted a PubMed search using the keywords &#34;cryptic SARS-CoV-2 lineages&#34; or &#34;novel SARS-CoV-2 lineages&#34; in addition to &#34;wastewater&#34; on May 9, 2023. From the 18 papers retrieved, only two reported wastewater-specific cryptic lineages. These lineages were identified by members of our author team in wastewater from California, Missouri, and New York City. None of these could be definitively traced to a specific source.

A third study in Nevada identified a unique recombinant variant (designated Pango lineage XL) in wastewater, which was also discovered in two clinical specimens from the same community. However, it was unclear whether the clinical specimens collected were from the same individual(s) responsible for the virus detected in the wastewater. To our knowledge, no prior study has successfully traced novel SARS-CoV-2 lineages detected in wastewater back to a specific location. How and where cryptic lineages are introduced into wastewater is not known.

The added value of this studyThis study documents the presence and likely source of a novel and highly divergent cryptic SARS-CoV-2 lineage detected in Wisconsin wastewater for 13 months. In contrast to previously reported cryptic lineages, we successfully traced the lineage (WI-CL-001) to a single commercial building with approximately 30 employees. The exceptionally high viral RNA concentrations at the source building facilitated the tracing effort and allowed for the sequencing of WI-CL-001s whole genome, expanding our view of the lineages mutational landscape beyond the spike gene.

Implications of all the available evidenceWI-CL-001s persistence in wastewater, its heavily mutated Omicron-like genotype, and its identified point source at a human-occupied commercial building all support the hypothesis that cryptic wastewater lineages can arise from persistently infected humans. Because cryptic wastewater lineages have some amino acid changes that subsequently emerge in circulating viruses, increased global monitoring of such lineages could help forecast variants that may arise in the future.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.23296099">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.23296099" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.23296099">
        <p class="paperTitle">Comparative performance of COVID-19 test methods in healthcare workers during the omicron wave</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.23296099" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.23296099" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tornberg, E. C.; Tomlinson, A.; Oshiro, N. T. T.; Derfalie, E.; Curlin, M.</p>
        <p class="info">Score: 14.5, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.23296099' target='https://doi.org/10.1101/2023.09.25.23296099'> 10.1101/2023.09.25.23296099</a></p>
        <p class="abstract">IntroductionThe COVID-19 pandemic presents unique requirements for accessible, reliable testing, and many testing platforms and sampling techniques have been developed. However, not all test methods have been systematically compared to each other or a common gold standard, and the performance of tests developed in the early epidemic have not been consistently re-evaluated in the context of newly emerging SARS-CoV-2 variants.

MethodsWe conducted a repeated measures study with adult healthcare workers presenting for SARS-CoV-2 testing. Participants were tested using seven test modalities: PCR with samples from the nasopharynx, oropharynx, and saliva; and BinaxNOW and iHealth antigen-based rapid detection tests (AgRDT) sampling the oropharynx and the nares. Test sensitivity was compared using any positive PCR test as the gold standard.

Results325 individuals participated in the study. PCR tests were the most sensitive with saliva PCR at 0.957 {&#43;/-} 0.048, nasopharyngeal PCR at 0.877 {&#43;/-} 0.075, and oropharyngeal PCR at 0.849 {&#43;/-} 0.082. Standard nasal rapid antigen tests were less sensitive but roughly equivalent at 0.613 {&#43;/-} 0.110 for BinaxNOW brand and 0.627 {&#43;/-} 0.109 for iHealth. Oropharyngeal rapid antigen tests were the least sensitive with BinaxNOW and iHealth brands at 0.400 {&#43;/-} 0.111 and 0.311 {&#43;/-} 0.105 respectively.

ConclusionPCR remains the most sensitive testing modality for COVID-19, with saliva PCR being significantly more sensitive than oropharyngeal PCR and equivalent to nasopharyngeal PCR. Saliva testing has patient comfort and financial benefits, making it a preferred testing modality. Nasal AgRDTs are less sensitive than PCR though more accessible and convenient.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.23295136">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.23295136" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.23295136">
        <p class="paperTitle">Pre-dominance of dengue non-cross-reacting SARS-CoV-2 spike antibodies during the Omicron era and their role in the ADE-mediated surge of Dengue virus serotype 3</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.23295136" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.23295136" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sarker, S.; Dutta, C.; Mallick, A.; Das, S.; Das Chowdhury, C.; De, A.; Gorai, S.; Biswas, S.</p>
        <p class="info">Score: 40.5, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.23295136' target='https://doi.org/10.1101/2023.09.08.23295136'> 10.1101/2023.09.08.23295136</a></p>
        <p class="abstract">In India, the SARS-CoV-2 Delta wave (2020-21) gradually faded away with the advent of the Omicron variants (2021-present). Dengue incidences were observably less in Southeast Asia during the active years of the pandemic (2020-21). However, Dengue virus type 3 (DV3) cases were increasingly reported in India and many other dengue-endemic countries concurrent with the progression of the Omicron wave since 2022. This observation prompted us to investigate the current state of cross-reactivity between prevalent SARS-CoV-2 variants and different DV serotypes.

Fifty-five COVID-19 serum samples (collected between January-September, 2022) and three pre-pandemic healthy serum samples were tested for DV or SARS-CoV-2 Immunoglobulin G/Immunoglobulin M (IgG/IgM) using the lateral flow immunoassays (LFIAs). The SARS-CoV-2 antibody (Ab)-positive samples were further tested for their ability to cross-neutralize DV types 1-4 (DV1-4) in Huh7 cell lines.

Cross-reactivity between SARS-CoV-2 and DV diminished with the shift from Delta to Omicron prevalence. COVID-19 serum samples that were DV cross-reactive neutralized all DV serotypes, including DV3. However, Omicron wave serum samples were predominantly DV non-cross-reactive (about 70%) in LFIAs and coincided with the prevalence of BA.2 Omicron variant. They also cross-neutralized DV1, 2 and 4 but enhanced DV3 infectivity as evident from increased DV3 titres in virus neutralization test (VNT) compared to control serum samples.

In conclusion, DV non-cross-reactive COVID-19 serum samples are becoming increasingly prominent in the present times. These non-cross-reactive serums could neutralize DV1, 2 and 4 but they are contributing to the surge in DV3 cases worldwide by means of ADE.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.23295927">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.23295927" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.23295927">
        <p class="paperTitle">Positive selection underlies repeated knockout of ORF8 in SARS-CoV-2 evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.23295927" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.23295927" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wagner, C.; Kistler, K. E.; Perchetti, G. A.; Baker, N.; Frisbie, L. A.; Torres, L. M.; Aragona, F.; Yun, C.; Figgins, M.; Greninger, A. L.; Cox, A.; Oltean, H. N.; Roychoudhury, P.; Bedford, T. D.</p>
        <p class="info">Score: 10.6, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.23295927' target='https://doi.org/10.1101/2023.09.21.23295927'> 10.1101/2023.09.21.23295927</a></p>
        <p class="abstract">Knockout of the ORF8 protein has repeatedly spread through the global viral population during SARS-CoV-2 evolution. Here we use both regional and global pathogen sequencing to explore the selection pressures underlying its loss. In Washington State, we identified transmission clusters with ORF8 knockout throughout SARS-CoV-2 evolution, not just on novel, high fitness viral backbones. Indeed, ORF8 is truncated more frequently and knockouts circulate for longer than for any other gene. Using a global phylogeny, we find evidence of positive selection to explain this phenomenon: nonsense mutations resulting in shortened protein products occur more frequently and are associated with faster clade growth rates than synonymous mutations in ORF8. Loss of ORF8 is also associated with reduced clinical severity, highlighting the diverse clinical impacts of SARS-CoV-2 evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.23296137">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.23296137" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.23296137">
        <p class="paperTitle">Longitudinal analysis within one hospital in sub-Saharan Africa over 20 years reveals repeated replacements of dominant clones of Klebsiella pneumoniae and stresses the importance to include temporal patterns for vaccine design considerations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.23296137" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.23296137" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heinz, E.; Pearse, O.; Zuza, A.; Bilima, S.; Msefula, C.; Musicha, P.; Siyabu, P.; Tewesa, E.; Graf, F. E.; Lester, R.; Lissauer, S.; Cornick, J.; Lewis, J. M.; Kawaza, K.; Thomson, N. R.; Feasey, N. A.</p>
        <p class="info">Score: 8.2, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.23296137' target='https://doi.org/10.1101/2023.09.26.23296137'> 10.1101/2023.09.26.23296137</a></p>
        <p class="abstract">Infections caused by multidrug-resistant gram-negative bacteria present a severe threat to global public health. The WHO defines drug-resistant Klebsiella pneumoniae as a priority pathogen for which alternative treatments are needed given the limited treatment options and the rapid acquisition of novel resistance mechanisms by this species. Longitudinal descriptions of genomic epidemiology of Klebsiella pneumoniae can inform management strategies but data from sub-Saharan Africa are lacking. We present a longitudinal analysis of all invasive K. pneumoniae isolates from a single hospital in Blantyre, Malawi, southern Africa, from 1998-2020, combining clinical data with genome sequence analysis of the isolates. We show that after a dramatic increase in the number of infections from 2016 K. pneumoniae becomes hyperendemic, driven by an increase in neonatal infections. Genomic data show repeated waves of clonal expansion of different, often ward-restricted, lineages, suggestive of hospital associated transmission. We describe temporal trends in resistance and surface antigens, of relevance for vaccine development. Our data highlight a clear need for new interventions to prevent rather than treat K. pneumoniae infections in our setting. Whilst one option may be a vaccine, the majority of cases could be avoided by an increased focus on and investment in infection prevention and control measures, which would reduce all healthcare associated infections and not just one.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
