<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>epidemiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>epidemiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Risk of severe outcomes among Omicron sub-lineages BA.4.6, BA.2.75 and BQ.1 compared to BA.5 in England
Authors: Seghezzo, G.; Nash, S. G.; Abdul Aziz, N.; Hope, R.; Lopez Bernal, J.; Gallagher, E.; Dabrera, G.; Thelwall, S.
Score: 151.1, Published: 2023-07-16 DOI: 10.1101/2023.07.14.23292656
Since the emergence of Omicron variant of SARS-CoV-2 in late 2021, a number of sub-lineages have arisen and circulated internationally. Little is known about the relative severity of Omicron sub-lineages BA.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/epidemiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="epidemiology" />
<meta property="og:description" content="Risk of severe outcomes among Omicron sub-lineages BA.4.6, BA.2.75 and BQ.1 compared to BA.5 in England
Authors: Seghezzo, G.; Nash, S. G.; Abdul Aziz, N.; Hope, R.; Lopez Bernal, J.; Gallagher, E.; Dabrera, G.; Thelwall, S.
Score: 151.1, Published: 2023-07-16 DOI: 10.1101/2023.07.14.23292656
Since the emergence of Omicron variant of SARS-CoV-2 in late 2021, a number of sub-lineages have arisen and circulated internationally. Little is known about the relative severity of Omicron sub-lineages BA." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/epidemiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-26T10:37:30+00:00" />
<meta property="article:modified_time" content="2023-07-26T10:37:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="epidemiology"/>
<meta name="twitter:description" content="Risk of severe outcomes among Omicron sub-lineages BA.4.6, BA.2.75 and BQ.1 compared to BA.5 in England
Authors: Seghezzo, G.; Nash, S. G.; Abdul Aziz, N.; Hope, R.; Lopez Bernal, J.; Gallagher, E.; Dabrera, G.; Thelwall, S.
Score: 151.1, Published: 2023-07-16 DOI: 10.1101/2023.07.14.23292656
Since the emergence of Omicron variant of SARS-CoV-2 in late 2021, a number of sub-lineages have arisen and circulated internationally. Little is known about the relative severity of Omicron sub-lineages BA."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "epidemiology",
      "item": "https://trxiv.yorks0n.com/posts/epidemiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "epidemiology",
  "name": "epidemiology",
  "description": "Risk of severe outcomes among Omicron sub-lineages BA.4.6, BA.2.75 and BQ.1 compared to BA.5 in England\nAuthors: Seghezzo, G.; Nash, S. G.; Abdul Aziz, N.; Hope, R.; Lopez Bernal, J.; Gallagher, E.; Dabrera, G.; Thelwall, S.\nScore: 151.1, Published: 2023-07-16 DOI: 10.1101/2023.07.14.23292656\nSince the emergence of Omicron variant of SARS-CoV-2 in late 2021, a number of sub-lineages have arisen and circulated internationally. Little is known about the relative severity of Omicron sub-lineages BA.",
  "keywords": [
    
  ],
  "articleBody": " Risk of severe outcomes among Omicron sub-lineages BA.4.6, BA.2.75 and BQ.1 compared to BA.5 in England\nAuthors: Seghezzo, G.; Nash, S. G.; Abdul Aziz, N.; Hope, R.; Lopez Bernal, J.; Gallagher, E.; Dabrera, G.; Thelwall, S.\nScore: 151.1, Published: 2023-07-16 DOI: 10.1101/2023.07.14.23292656\nSince the emergence of Omicron variant of SARS-CoV-2 in late 2021, a number of sub-lineages have arisen and circulated internationally. Little is known about the relative severity of Omicron sub-lineages BA.2.75, BA.4.6 and BQ.1. We undertook a case-control analysis to determine the clinical severity of these lineages relative to BA.5, using whole genome sequenced, PCR-confirmed infections, between 1 August 2022 to 27 November 2022, among those who presented to emergency care in England 14 days after and up to one day prior to the positive specimen. A total of 10,375 episodes were included in the analysis, of which 5,207 (50.2%) were admitted to hospital or died. Multivariable conditional regression analyses found no evidence for greater odds of hospital admission or death among those with BA.2.75 (OR= 0.96, 95% CI: 0.84 to 1.09), and BA.4.6 (OR= 1.02, 95% CI: 0.88 to 1.17) or BQ.1 (OR= 1.03, 95 % CI: 0.94 to 1.13) compared to BA.5. Future lineages may not follow the same trend and there remains a need for continued surveillance of COVID-19 variants and their clinical outcomes to inform the public health response.\nProteomic prediction of common and rare diseases\nAuthors: Carrasco-Zanini, J.; Pietzner, M.; Davitte, J.; Surendran, P.; Croteau-Chonka, D. C.; Robins, C.; Torralbo, A.; Tomlinson, C.; Fitzpatrick, N.; Ytsma, C.; Kanno, T.; Gade, S.; Freitag, D.; Ziebell, F.; Denaxas, S.; Betts, J. C.; Wareham, N. J.; Hemingway, H.; Scott, R. A.; Langenberg, C.\nScore: 19.7, Published: 2023-07-23 DOI: 10.1101/2023.07.18.23292811\nBackgroundFor many diseases there are delays in diagnosis due to a lack of objective biomarkers for disease onset. Whether measuring thousands of proteins offers predictive information across a wide range of diseases is unknown. MethodsIn 41,931 individuals from the UK Biobank Pharma Proteomics Project (UKB-PPP), we integrated [~]3000 plasma proteins with clinical information to derive sparse prediction models for the 10-year incidence of 218 common and rare diseases (81 - 6038 cases). We compared prediction models based on proteins with a) basic clinical information alone, b) basic clinical information + 37 clinical biomarkers, and c) genome-wide polygenic risk scores. ResultsFor 67 pathologically diverse diseases, a model including as few as 5 to 20 proteins was superior to clinical models (median delta C-index = 0.07; range = 0.02 - 0.31) and to clinical models with biomarkers for 52 diseases. In multiple myeloma, for example, a set of 5 proteins significantly improved prediction over basic clinical information (delta C-index = 0.25 (95% confidence interval 0.20 - 0.29)). At a 5% false positive rate (FPR), proteomic prediction (5 proteins) identified individuals at high risk of multiple myeloma (detection rate (DR) = 50%), non-Hodgkin lymphoma (DR = 55%) and motor neuron disease (DR = 29%). At a 20% FPR, proteomic prediction identified individuals at high-risk for pulmonary fibrosis (DR= 80%) and dilated cardiomyopathy (DR = 75%). ConclusionsSparse plasma protein signatures offer novel, clinically useful prediction of common and rare diseases, through disease-specific proteins and protein predictors shared across multiple diseases. (Funded by Medical Research Council, NIHR, Wellcome Trust.)\nInterpretation of wide confidence intervals in meta-analytic estimates: Is the 'Absence of Evidence' 'Evidence of Absence'?\nAuthors: Miller, S. L.; Tuia, J.; Prasad, V.\nScore: 12.5, Published: 2023-07-14 DOI: 10.1101/2023.07.11.23292513\nIntroductionRecently, a Cochrane review by Jefferson et al. on physical interventions to slow the spread of respiratory viruses concluded that, \"Wearing masks in the community probably makes little or no difference to the outcome of laboratory-confirmed influenza/SARS-CoV-2 compared to not wearing masks\", though this finding had a wide confidence interval. Cochrane issued a rare clarifying statement, fueling controversy. We sought to contextualize the findings of the review by Jefferson et al. MethodsWe searched for consecutive reviews by Cochrane published on or before March 9th, 2023. We included studies where a central finding showed an intervention offered no statistically significant benefit, and ascertained the language used by reviewers to describe that result. We compare this to the report by Jefferson et al., and deemed it consistent or inconsistent with the language of their report. ResultsWe found between November 21st, 2022, and March 9th, 2023, there were 20 Cochrane reviews that met the inclusion criteria. We found that 95% (n = 19) of the reviews used language that was consistent with Jeffersons findings, while 5% (n = 1) used language inconsistent with Jeffersons conclusion, describing the effect of the intervention on the outcome as \"unclear\". DiscussionMost reviews performed by Cochrane conclude that interventions which fail to show statistically significant benefits make \"no difference\" have \"no effect\" or do not \"increase or decrease\" the outcome, and this occurs despite wide confidence intervals. The conclusions by Jefferson et al. are consistent with Cochrane reporting guidelines and clarification from the organization was unjustified.\nImmune boosting bridges leaky and polarized vaccination models\nAuthors: Park, S. W.; Li, M.; Metcalf, J.; Grenfell, B.; Dushoff, J.\nScore: 14.3, Published: 2023-07-18 DOI: 10.1101/2023.07.14.23292670\nTwo different epidemiological models of vaccination are commonly used in dynamical modeling studies. The leaky vaccination model assumes that all vaccinated individuals experience a reduced force of infection by the same amount. The polarized vaccination model assumes that some fraction of vaccinated individuals are completely protected, while the remaining fraction remains completely susceptible; this seemingly extreme assumption causes the polarized model to always predict lower final epidemic size than the leaky model under the same vaccine efficacy. However, the leaky model also makes an implicit, unrealistic assumption: vaccinated individuals who are exposed to infection but not infected remain just as susceptible as they were prior to exposures (i.e., independent of previous exposures). To resolve the independence assumption, we introduce an immune boosting mechanism, through which vaccinated, yet susceptible, individuals can gain protection without developing a transmissible infection. The boosting model further predicts identical epidemic dynamics as the polarized vaccination model, thereby bridging the differences between two models. We further develop a generalized vaccination model to explore how the assumptions of immunity affect epidemic dynamics and estimates of vaccine effectiveness. Significance statementDifferent assumptions about the long- and medium-term effects of protective vaccination can predict sharply different epidemiological dynamics. However, there has been limited discussion about which assumptions are more realistic and therefore more appropriate for making public health decisions. Here, we show that the differences between the two most common assumptions (the \"leaky\" and \"polarized\" vaccination models) are bridged by immune boosting, a mechanism by which individuals who resist infectious challenge due to partial immunity have their immunity increased. We demonstrate that this mechanism has important implications for measuring vaccine effectiveness. Our study challenges fundamental assumptions about commonly used vaccination models and provides a novel framework for understanding the epidemiological impact of vaccination.\nGenomic Analysis and Surveillance of Respiratory Syncytial Virus (RSV) Using Wastewater-Based Epidemiology (WBE)\nAuthors: Allen, D. M.; Reyne, M. I.; Allingham, P.; Levickas, A.; Bell, S. H.; Lock, J.; Coey, J. D.; Carson, S.; Lee, A. J.; McSparron, C.; Nejad, B. F.; McKenna, J.; Shannon, M.; Li, K.; Curran, T.; Broadbent, L. J.; Downey, D. G.; Power, U. F.; Groves, H. E.; McKinley, J. M.; McGrath, J. W.; Bamford, C. G. G.; Gilpin, D. F.\nScore: 10.6, Published: 2023-07-24 DOI: 10.1101/2023.07.21.23293016\nRespiratory syncytial virus (RSV) causes severe infections in infants, immunocompromised or elderly individuals resulting in annual epidemics of respiratory disease. Currently, limited clinical RSV surveillance and the lack of predictable RSV seasonal dynamics and limits the public health response. Wastewater-based epidemiology (WBE) has the capacity to determine levels of health-associated biomarkers and has recently been used globally as a key metric in determining prevalence of SARS-CoV-2 in the community. However, the application of genomic WBE for the surveillance of other respiratory viruses is limited. In this study, we present an integrated genomic WBE approach, using RT-qPCR and partial sequencing of the G gene to monitor RSV levels and variants in the community across 2 years encompassing two periods of high RSV clinical positivity in Northern Ireland. We report increasing detection of RSV in wastewater concomitant with increasing numbers of RSV positive clinical cases. Furthermore, analysis of wastewater-derived RSV sequences permitted subtyping, genotyping, and identification of distinct circulating lineages within and between seasons. Altogether, our genomic WBE platform has the potential to complement ongoing global surveillance efforts and aid the management of RSV by informing the timely deployment of pharmaceutical and non-pharmaceutical interventions.\nEvaluation of the impact of COVID-19 pandemic on hospital admission related to common infections\nAuthors: Fahmi, A.; Palin, V.; Zhong, X.; Yang, Y.-T.; Watts, S.; Ashcroft, D. M.; Goldacre, B.; Mackenna, B.; Fisher, L.; Massey, J.; Mehrkar, A.; Bacon, S.; the OpenSAFELY collaborative, ; Hand, K.; van Staa, T.\nScore: 10.1, Published: 2023-07-18 DOI: 10.1101/2023.07.16.23292723\nBackgroundAntimicrobial resistance (AMR) is a multifaceted global challenge, partly driven by inappropriate antibiotic prescribing. The COVID-19 pandemic impacted antibiotic prescribing for common bacterial infections. This highlights the need to examine risk of hospital admissions related to common infections, excluding COVID-19 infections during the pandemic. MethodsWith the approval of NHS England, we accessed electronic health records from The Phoenix Partnership (TPP) through OpenSAFELY platform. We included patients with primary care diagnosis of common infections, including lower respiratory tract infection (LRTI), upper respiratory tract infections (URTI), and lower urinary tract infection (UTI), from January 2019 to August 2022. We excluded patients with a COVID-19 record 90 days before to 30 days after the infection diagnosis. Using Cox proportional-hazard regression models, we predicted risk of infection-related hospital admission in 30 days follow-up period after the diagnosis. ResultsWe found 12,745,165 infection diagnoses from January 2019 to August 2022. Of them, 80,395 (2.05%) cases were admitted to hospital in the follow-up period. Counts of hospital admission for infections dropped during COVID-19, e.g., LRTI from 3,950 in December 2019 to 520 in April 2020. Comparing those prescribed an antibiotic to those without, reduction in risk of hospital admission were largest with LRTI (adjusted odds ratio (OR) of 0.35; 95% CI, 0.35-0.36) and UTI (adjusted OR 0.45; 95% CI, 0.44-0.46), compared to URTI (adjusted OR 1.04; 95% CI, 1.03-1.06). ConclusionLarge effectiveness of antibiotics in preventing complications related to LRTI and UTI can support better targeting of antibiotics to patients with higher complication risks. Key messages- The main drivers of infection-related hospital admission are age, Charlson comorbidity index, and history of prior antibiotics. - Antibiotics are more effective in preventing hospital admission related to infections such as lower respiratory tract infection and urinary tract infection, rather than upper respiratory tract infection. - Common antibiotic types are associated with more reduction in the risk of infection-related hospital admission.\nThe historical fingerprint and future impact of climate change on childhood malaria in Africa\nAuthors: Carlson, C. J.; Carleton, T. A.; Odoulami, R.; Trisos, C. H.\nScore: 15.1, Published: 2023-07-25 DOI: 10.1101/2023.07.16.23292713\nThe health burden of anthropogenic climate change is growing exponentially, but present-day impacts remain difficult to measure 1-3. Here, we leverage a recently-published comprehensive dataset of 50,425 population surveys 4 to investigate whether human-caused climate change has increased the burden of childhood malaria across sub-Saharan Africa. In historical data, we find that prevalence shows a robust response to temperature and extreme precipitation, consistent with expectations from previous empirical and epidemiological work. Comparing historical climate reconstructions to counterfactual simulations without anthropogenic warming, we find two-to-one odds that human-caused climate change has increased the overall prevalence of childhood malaria across sub-Saharan Africa since 1901. We estimate that by 2014, human-caused climate change was responsible for an average of 84 excess cases of malaria per 100,000 children ages 2 to 10, with higher elevation and cooler regions in southern and east Africa having greater increases. Under future climate change, we project increasing temperatures could plausibly accelerate the eradication of malaria in west and central Africa, where the present-day burden is highest, leading to continent-wide average reductions of 89 (low greenhouse gas emissions, SSP1-RCP2.6) to 1,750 (high emissions, SSP5-RCP8.5) cases per 100,000 children by the end of the century. However, we find that limiting future global warming to below 2{degrees}C (SSP1-RCP2.6) compared to[~] 3{degrees}C (SSP2-RCP4.5) could prevent an average of 496 excess cases in southern Africa, and 40 excess cases in east Africa,per 100,000 children by 2100. Our study resolves a decades-old debate about one of the earliest health impacts of global warming, and provides a template for future work measuring the true global burden of climate change.\nA Parkinson's disease genetic risk score associates with blood DNAm on chromosome 17\nAuthors: Casazza, W.; Schaffner, S. L.; Artaud, F.; Domenighetti, C.; Schulze-Hentrich, J. M.; Lesage, S.; Brice, A.; Corvol, J.-C.; Mostafavi, S.; Kobor, M. S.; Elbaz, A.; Dennis, J. K.\nScore: 1.1, Published: 2023-07-24 DOI: 10.1101/2023.07.21.23293014\nAlthough Parkinson's disease (PD) coincides with altered immune functioning, there are few reproducible associations between blood DNA methylation (DNAm) and PD case-control status. Integrative analyses of genotype and blood DNAm can address this gap and can help us characterize the biological function of PD genetic risk loci. First, we tested for associations between a PD genetic risk score (GRS) and DNAm. Our GRS included 36 independent genome-wide significant variants from the largest GWAS of PD to date. Our discovery sample was TERRE, consisting of French agricultural workers (71 PD cases and 147 controls). The GRS associated with DNAm at 85 CpG sites, with 19 associations replicated in an independent sample (DIG-PD). The majority of CpG sites (73) are within a 1.5 Mb window on chromosome 17, and 36 CpG sites annotate to MAPT and KANSL1, neighboring genes that affect neurodegeneration. All associations were invariant to non-genetic factors, including exposure to commercial-grade pesticides, and omitting chromosome 17 variants from the GRS had little effect on association. Second, we compared our findings to the relationship between individual PD risk loci and blood DNAm using blood mQTL from a large independent meta-analysis (GoDMC). We found 79 CpG sites that colocalized with PD loci, and via summary Mendelian randomization analysis, we show 25/79 CpG sites where DNAm causally affects PD risk. The nine largest causal effects are within chromosome 17, including an effect within MAPT. Thus, all integrative analyses prioritized DNAm on chromosome 17, drawing from multiple independent data sets, meriting further study of this region.\nUnderstanding the fine-scale heterogeneity and spatial drivers of malaria transmission using model-based geostatistical methods in Kenya\nAuthors: Mategula, D.; Gichuki, J.\nScore: 1.1, Published: 2023-07-19 DOI: 10.1101/2023.07.17.23292805\nMalaria remains a public health concern. Monitoring the fine-scale heterogeneity of the malaria burden enables more targeted control efforts. Although malaria indicator surveys (MIS) have been crucial in evaluating the progress of malaria control interventions, they are only designed to provide a cross-sectional national and regional malaria disease burden. Recent advances in geostatistical methods allow us to interpolate national survey data to describe subnational disease burden that is crucial in informing targeted control. A binomial geostatistical model employing Markov chain Monte Carlo (MCMC) parameter estimation methods is used to understand the spatial drivers of malaria risk in Kenya and to predict malaria risk at a fine-scale resolution, including identifying hotspots. A total of 11,549 children aged six months to 14 years from 207 clusters were sampled in this survey and used in the present analysis. The national malaria prevalence based on the data was 8.4%, with the highest in the lake endemic zone (18.1 %) and the lowest in the low-risk zone (\u003c1 %). The analysis shows that elevation, proportion of ITN distributed, rainfall, temperature and urbanization covariates are all significant predictors of malaria transmission. The 5x5 Km resolution maps show that malaria is heterogeneous in Kenya, with hotspot areas in the lake endemic area, the coastal areas, and some parts of the shores of Lake Turkana and Kajiado. The high-resolution malaria prevalence maps produced as part of the analysis have shown that Kenya has additional malaria hotspots, especially in areas least expected. These findings call for a rethinking of malaria burden classification in some regions for effective planning, implementation, resource mobilization, monitoring, and evaluation of malaria interventions in the country.\nCommon infections and neuroimaging markers of dementia in three UK cohort studies\nAuthors: Green, R. E.; Sudre, C. H.; Warren-Gash, C.; Butt, J.; Waterboer, T.; The Insight 46 study team, ; Hughes, A. D.; Schott, J. M.; Richards, M.; Chaturvedi, N.; Williams, D. M.\nScore: 14.6, Published: 2023-07-12 DOI: 10.1101/2023.07.12.23292538\nIntroductionWe aimed to investigate associations between common infections and neuroimaging markers of dementia risk (brain volume, hippocampal volume, white matter lesions) across three population-based studies. MethodsWe tested associations between serology measures (pathogen serostatus, cumulative burden, continuous antibody responses) and outcomes using linear regression, including adjustments for total intracranial volume and scanner/clinic information (basic model), age, sex, ethnicity, education, socioeconomic position, alcohol, BMI, and smoking (fully adjusted model). Interactions between serology measures and APOE genotype were tested. Findings were meta-analysed across cohorts (Nmain=2632; NAPOE-interaction=1810). ResultsSeropositivity to JC virus associated with smaller brain volumes in basic models ({beta}=-3.89ml[-5.81,-1.97],padjusted\u003c0.05); these were largely attenuated in fully adjusted models ({beta}=-1.59ml[-3.55,0.36], p=0.11). No other relationships were robust to multiple testing correction and sensitivity analyses, but several suggestive associations were observed. DiscussionWe did not find clear evidence for relationships between common infections and markers of dementia risk. Some suggestive findings warrant testing for replication.\n",
  "wordCount" : "2861",
  "inLanguage": "en",
  "datePublished": "2023-07-26T10:37:30Z",
  "dateModified": "2023-07-26T10:37:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/epidemiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      epidemiology
    </h1>
    <div class="post-meta"><span>updated on July 26, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.23292656">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.23292656" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.23292656">
        <p class="paperTitle">Risk of severe outcomes among Omicron sub-lineages BA.4.6, BA.2.75 and BQ.1 compared to BA.5 in England</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.23292656" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.23292656" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Seghezzo, G.; Nash, S. G.; Abdul Aziz, N.; Hope, R.; Lopez Bernal, J.; Gallagher, E.; Dabrera, G.; Thelwall, S.</p>
        <p class="info">Score: 151.1, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.23292656' target='https://doi.org/10.1101/2023.07.14.23292656'> 10.1101/2023.07.14.23292656</a></p>
        <p class="abstract">Since the emergence of Omicron variant of SARS-CoV-2 in late 2021, a number of sub-lineages have arisen and circulated internationally. Little is known about the relative severity of Omicron sub-lineages BA.2.75, BA.4.6 and BQ.1. We undertook a case-control analysis to determine the clinical severity of these lineages relative to BA.5, using whole genome sequenced, PCR-confirmed infections, between 1 August 2022 to 27 November 2022, among those who presented to emergency care in England 14 days after and up to one day prior to the positive specimen. A total of 10,375 episodes were included in the analysis, of which 5,207 (50.2%) were admitted to hospital or died. Multivariable conditional regression analyses found no evidence for greater odds of hospital admission or death among those with BA.2.75 (OR= 0.96, 95% CI: 0.84 to 1.09), and BA.4.6 (OR= 1.02, 95% CI: 0.88 to 1.17) or BQ.1 (OR= 1.03, 95 % CI: 0.94 to 1.13) compared to BA.5. Future lineages may not follow the same trend and there remains a need for continued surveillance of COVID-19 variants and their clinical outcomes to inform the public health response.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.23292811">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.23292811" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.23292811">
        <p class="paperTitle">Proteomic prediction of common and rare diseases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.23292811" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.23292811" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carrasco-Zanini, J.; Pietzner, M.; Davitte, J.; Surendran, P.; Croteau-Chonka, D. C.; Robins, C.; Torralbo, A.; Tomlinson, C.; Fitzpatrick, N.; Ytsma, C.; Kanno, T.; Gade, S.; Freitag, D.; Ziebell, F.; Denaxas, S.; Betts, J. C.; Wareham, N. J.; Hemingway, H.; Scott, R. A.; Langenberg, C.</p>
        <p class="info">Score: 19.7, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.23292811' target='https://doi.org/10.1101/2023.07.18.23292811'> 10.1101/2023.07.18.23292811</a></p>
        <p class="abstract">BackgroundFor many diseases there are delays in diagnosis due to a lack of objective biomarkers for disease onset. Whether measuring thousands of proteins offers predictive information across a wide range of diseases is unknown.

MethodsIn 41,931 individuals from the UK Biobank Pharma Proteomics Project (UKB-PPP), we integrated [~]3000 plasma proteins with clinical information to derive sparse prediction models for the 10-year incidence of 218 common and rare diseases (81 - 6038 cases). We compared prediction models based on proteins with a) basic clinical information alone, b) basic clinical information &#43; 37 clinical biomarkers, and c) genome-wide polygenic risk scores.

ResultsFor 67 pathologically diverse diseases, a model including as few as 5 to 20 proteins was superior to clinical models (median delta C-index = 0.07; range = 0.02 - 0.31) and to clinical models with biomarkers for 52 diseases. In multiple myeloma, for example, a set of 5 proteins significantly improved prediction over basic clinical information (delta C-index = 0.25 (95% confidence interval 0.20 - 0.29)). At a 5% false positive rate (FPR), proteomic prediction (5 proteins) identified individuals at high risk of multiple myeloma (detection rate (DR) = 50%), non-Hodgkin lymphoma (DR = 55%) and motor neuron disease (DR = 29%). At a 20% FPR, proteomic prediction identified individuals at high-risk for pulmonary fibrosis (DR= 80%) and dilated cardiomyopathy (DR = 75%).

ConclusionsSparse plasma protein signatures offer novel, clinically useful prediction of common and rare diseases, through disease-specific proteins and protein predictors shared across multiple diseases.

(Funded by Medical Research Council, NIHR, Wellcome Trust.)</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.23292513">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.23292513" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.23292513">
        <p class="paperTitle">Interpretation of wide confidence intervals in meta-analytic estimates: Is the &#39;Absence of Evidence&#39; &#39;Evidence of Absence&#39;?</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.23292513" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.23292513" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miller, S. L.; Tuia, J.; Prasad, V.</p>
        <p class="info">Score: 12.5, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.23292513' target='https://doi.org/10.1101/2023.07.11.23292513'> 10.1101/2023.07.11.23292513</a></p>
        <p class="abstract">IntroductionRecently, a Cochrane review by Jefferson et al. on physical interventions to slow the spread of respiratory viruses concluded that, &#34;Wearing masks in the community probably makes little or no difference to the outcome of laboratory-confirmed influenza/SARS-CoV-2 compared to not wearing masks&#34;, though this finding had a wide confidence interval. Cochrane issued a rare clarifying statement, fueling controversy. We sought to contextualize the findings of the review by Jefferson et al.

MethodsWe searched for consecutive reviews by Cochrane published on or before March 9th, 2023. We included studies where a central finding showed an intervention offered no statistically significant benefit, and ascertained the language used by reviewers to describe that result. We compare this to the report by Jefferson et al., and deemed it consistent or inconsistent with the language of their report.

ResultsWe found between November 21st, 2022, and March 9th, 2023, there were 20 Cochrane reviews that met the inclusion criteria. We found that 95% (n = 19) of the reviews used language that was consistent with Jeffersons findings, while 5% (n = 1) used language inconsistent with Jeffersons conclusion, describing the effect of the intervention on the outcome as &#34;unclear&#34;.

DiscussionMost reviews performed by Cochrane conclude that interventions which fail to show statistically significant benefits make &#34;no difference&#34; have &#34;no effect&#34; or do not &#34;increase or decrease&#34; the outcome, and this occurs despite wide confidence intervals. The conclusions by Jefferson et al. are consistent with Cochrane reporting guidelines and clarification from the organization was unjustified.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.23292670">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.23292670" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.23292670">
        <p class="paperTitle">Immune boosting bridges leaky and polarized vaccination models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.23292670" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.23292670" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Park, S. W.; Li, M.; Metcalf, J.; Grenfell, B.; Dushoff, J.</p>
        <p class="info">Score: 14.3, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.23292670' target='https://doi.org/10.1101/2023.07.14.23292670'> 10.1101/2023.07.14.23292670</a></p>
        <p class="abstract">Two different epidemiological models of vaccination are commonly used in dynamical modeling studies. The leaky vaccination model assumes that all vaccinated individuals experience a reduced force of infection by the same amount. The polarized vaccination model assumes that some fraction of vaccinated individuals are completely protected, while the remaining fraction remains completely susceptible; this seemingly extreme assumption causes the polarized model to always predict lower final epidemic size than the leaky model under the same vaccine efficacy. However, the leaky model also makes an implicit, unrealistic assumption: vaccinated individuals who are exposed to infection but not infected remain just as susceptible as they were prior to exposures (i.e., independent of previous exposures). To resolve the independence assumption, we introduce an immune boosting mechanism, through which vaccinated, yet susceptible, individuals can gain protection without developing a transmissible infection. The boosting model further predicts identical epidemic dynamics as the polarized vaccination model, thereby bridging the differences between two models. We further develop a generalized vaccination model to explore how the assumptions of immunity affect epidemic dynamics and estimates of vaccine effectiveness.

Significance statementDifferent assumptions about the long- and medium-term effects of protective vaccination can predict sharply different epidemiological dynamics. However, there has been limited discussion about which assumptions are more realistic and therefore more appropriate for making public health decisions. Here, we show that the differences between the two most common assumptions (the &#34;leaky&#34; and &#34;polarized&#34; vaccination models) are bridged by immune boosting, a mechanism by which individuals who resist infectious challenge due to partial immunity have their immunity increased. We demonstrate that this mechanism has important implications for measuring vaccine effectiveness. Our study challenges fundamental assumptions about commonly used vaccination models and provides a novel framework for understanding the epidemiological impact of vaccination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.23293016">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.23293016" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.23293016">
        <p class="paperTitle">Genomic Analysis and Surveillance of Respiratory Syncytial Virus (RSV) Using Wastewater-Based Epidemiology (WBE)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.23293016" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.23293016" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Allen, D. M.; Reyne, M. I.; Allingham, P.; Levickas, A.; Bell, S. H.; Lock, J.; Coey, J. D.; Carson, S.; Lee, A. J.; McSparron, C.; Nejad, B. F.; McKenna, J.; Shannon, M.; Li, K.; Curran, T.; Broadbent, L. J.; Downey, D. G.; Power, U. F.; Groves, H. E.; McKinley, J. M.; McGrath, J. W.; Bamford, C. G. G.; Gilpin, D. F.</p>
        <p class="info">Score: 10.6, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.23293016' target='https://doi.org/10.1101/2023.07.21.23293016'> 10.1101/2023.07.21.23293016</a></p>
        <p class="abstract">Respiratory syncytial virus (RSV) causes severe infections in infants, immunocompromised or elderly individuals resulting in annual epidemics of respiratory disease. Currently, limited clinical RSV surveillance and the lack of predictable RSV seasonal dynamics and limits the public health response. Wastewater-based epidemiology (WBE) has the capacity to determine levels of health-associated biomarkers and has recently been used globally as a key metric in determining prevalence of SARS-CoV-2 in the community. However, the application of genomic WBE for the surveillance of other respiratory viruses is limited. In this study, we present an integrated genomic WBE approach, using RT-qPCR and partial sequencing of the G gene to monitor RSV levels and variants in the community across 2 years encompassing two periods of high RSV clinical positivity in Northern Ireland. We report increasing detection of RSV in wastewater concomitant with increasing numbers of RSV positive clinical cases. Furthermore, analysis of wastewater-derived RSV sequences permitted subtyping, genotyping, and identification of distinct circulating lineages within and between seasons. Altogether, our genomic WBE platform has the potential to complement ongoing global surveillance efforts and aid the management of RSV by informing the timely deployment of pharmaceutical and non-pharmaceutical interventions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.23292723">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.23292723" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.23292723">
        <p class="paperTitle">Evaluation of the impact of COVID-19 pandemic on hospital admission related to common infections</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.23292723" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.23292723" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fahmi, A.; Palin, V.; Zhong, X.; Yang, Y.-T.; Watts, S.; Ashcroft, D. M.; Goldacre, B.; Mackenna, B.; Fisher, L.; Massey, J.; Mehrkar, A.; Bacon, S.; the OpenSAFELY collaborative,  ; Hand, K.; van Staa, T.</p>
        <p class="info">Score: 10.1, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.23292723' target='https://doi.org/10.1101/2023.07.16.23292723'> 10.1101/2023.07.16.23292723</a></p>
        <p class="abstract">BackgroundAntimicrobial resistance (AMR) is a multifaceted global challenge, partly driven by inappropriate antibiotic prescribing. The COVID-19 pandemic impacted antibiotic prescribing for common bacterial infections. This highlights the need to examine risk of hospital admissions related to common infections, excluding COVID-19 infections during the pandemic.

MethodsWith the approval of NHS England, we accessed electronic health records from The Phoenix Partnership (TPP) through OpenSAFELY platform. We included patients with primary care diagnosis of common infections, including lower respiratory tract infection (LRTI), upper respiratory tract infections (URTI), and lower urinary tract infection (UTI), from January 2019 to August 2022. We excluded patients with a COVID-19 record 90 days before to 30 days after the infection diagnosis. Using Cox proportional-hazard regression models, we predicted risk of infection-related hospital admission in 30 days follow-up period after the diagnosis.

ResultsWe found 12,745,165 infection diagnoses from January 2019 to August 2022. Of them, 80,395 (2.05%) cases were admitted to hospital in the follow-up period. Counts of hospital admission for infections dropped during COVID-19, e.g., LRTI from 3,950 in December 2019 to 520 in April 2020. Comparing those prescribed an antibiotic to those without, reduction in risk of hospital admission were largest with LRTI (adjusted odds ratio (OR) of 0.35; 95% CI, 0.35-0.36) and UTI (adjusted OR 0.45; 95% CI, 0.44-0.46), compared to URTI (adjusted OR 1.04; 95% CI, 1.03-1.06).

ConclusionLarge effectiveness of antibiotics in preventing complications related to LRTI and UTI can support better targeting of antibiotics to patients with higher complication risks.

Key messages- The main drivers of infection-related hospital admission are age, Charlson comorbidity index, and history of prior antibiotics.
- Antibiotics are more effective in preventing hospital admission related to infections such as lower respiratory tract infection and urinary tract infection, rather than upper respiratory tract infection.
- Common antibiotic types are associated with more reduction in the risk of infection-related hospital admission.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.23292713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.23292713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.23292713">
        <p class="paperTitle">The historical fingerprint and future impact of climate change on childhood malaria in Africa</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.23292713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.23292713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carlson, C. J.; Carleton, T. A.; Odoulami, R.; Trisos, C. H.</p>
        <p class="info">Score: 15.1, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.23292713' target='https://doi.org/10.1101/2023.07.16.23292713'> 10.1101/2023.07.16.23292713</a></p>
        <p class="abstract">The health burden of anthropogenic climate change is growing exponentially, but present-day impacts remain difficult to measure 1-3. Here, we leverage a recently-published comprehensive dataset of 50,425 population surveys 4 to investigate whether human-caused climate change has increased the burden of childhood malaria across sub-Saharan Africa. In historical data, we find that prevalence shows a robust response to temperature and extreme precipitation, consistent with expectations from previous empirical and epidemiological work. Comparing historical climate reconstructions to counterfactual simulations without anthropogenic warming, we find two-to-one odds that human-caused climate change has increased the overall prevalence of childhood malaria across sub-Saharan Africa since 1901. We estimate that by 2014, human-caused climate change was responsible for an average of 84 excess cases of malaria per 100,000 children ages 2 to 10, with higher elevation and cooler regions in southern and east Africa having greater increases. Under future climate change, we project increasing temperatures could plausibly accelerate the eradication of malaria in west and central Africa, where the present-day burden is highest, leading to continent-wide average reductions of 89 (low greenhouse gas emissions, SSP1-RCP2.6) to 1,750 (high emissions, SSP5-RCP8.5) cases per 100,000 children by the end of the century. However, we find that limiting future global warming to below 2{degrees}C (SSP1-RCP2.6) compared to[~] 3{degrees}C (SSP2-RCP4.5) could prevent an average of 496 excess cases in southern Africa, and 40 excess cases in east Africa,per 100,000 children by 2100. Our study resolves a decades-old debate about one of the earliest health impacts of global warming, and provides a template for future work measuring the true global burden of climate change.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.23293014">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.23293014" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.23293014">
        <p class="paperTitle">A Parkinson&#39;s disease genetic risk score associates with blood DNAm on chromosome 17</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.23293014" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.23293014" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Casazza, W.; Schaffner, S. L.; Artaud, F.; Domenighetti, C.; Schulze-Hentrich, J. M.; Lesage, S.; Brice, A.; Corvol, J.-C.; Mostafavi, S.; Kobor, M. S.; Elbaz, A.; Dennis, J. K.</p>
        <p class="info">Score: 1.1, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.23293014' target='https://doi.org/10.1101/2023.07.21.23293014'> 10.1101/2023.07.21.23293014</a></p>
        <p class="abstract">Although Parkinson&#39;s disease (PD) coincides with altered immune functioning, there are few reproducible associations between blood DNA methylation (DNAm) and PD case-control status. Integrative analyses of genotype and blood DNAm can address this gap and can help us characterize the biological function of PD genetic risk loci. First, we tested for associations between a PD genetic risk score (GRS) and DNAm. Our GRS included 36 independent genome-wide significant variants from the largest GWAS of PD to date. Our discovery sample was TERRE, consisting of French agricultural workers (71 PD cases and 147 controls). The GRS associated with DNAm at 85 CpG sites, with 19 associations replicated in an independent sample (DIG-PD). The majority of CpG sites (73) are within a 1.5 Mb window on chromosome 17, and 36 CpG sites annotate to MAPT and KANSL1, neighboring genes that affect neurodegeneration. All associations were invariant to non-genetic factors, including exposure to commercial-grade pesticides, and omitting chromosome 17 variants from the GRS had little effect on association. Second, we compared our findings to the relationship between individual PD risk loci and blood DNAm using blood mQTL from a large independent meta-analysis (GoDMC). We found 79 CpG sites that colocalized with PD loci, and via summary Mendelian randomization analysis, we show 25/79 CpG sites where DNAm causally affects PD risk. The nine largest causal effects are within chromosome 17, including an effect within MAPT. Thus, all integrative analyses prioritized DNAm on chromosome 17, drawing from multiple independent data sets, meriting further study of this region.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.23292805">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.23292805" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.23292805">
        <p class="paperTitle">Understanding the fine-scale heterogeneity and spatial drivers of malaria transmission using model-based geostatistical methods in Kenya</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.23292805" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.23292805" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mategula, D.; Gichuki, J.</p>
        <p class="info">Score: 1.1, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.23292805' target='https://doi.org/10.1101/2023.07.17.23292805'> 10.1101/2023.07.17.23292805</a></p>
        <p class="abstract">Malaria remains a public health concern. Monitoring the fine-scale heterogeneity of the malaria burden enables more targeted control efforts. Although malaria indicator surveys (MIS) have been crucial in evaluating the progress of malaria control interventions, they are only designed to provide a cross-sectional national and regional malaria disease burden. Recent advances in geostatistical methods allow us to interpolate national survey data to describe subnational disease burden that is crucial in informing targeted control.

A binomial geostatistical model employing Markov chain Monte Carlo (MCMC) parameter estimation methods is used to understand the spatial drivers of malaria risk in Kenya and to predict malaria risk at a fine-scale resolution, including identifying hotspots.

A total of 11,549 children aged six months to 14 years from 207 clusters were sampled in this survey and used in the present analysis. The national malaria prevalence based on the data was 8.4%, with the highest in the lake endemic zone (18.1 %) and the lowest in the low-risk zone (&lt;1 %). The analysis shows that elevation, proportion of ITN distributed, rainfall, temperature and urbanization covariates are all significant predictors of malaria transmission. The 5x5 Km resolution maps show that malaria is heterogeneous in Kenya, with hotspot areas in the lake endemic area, the coastal areas, and some parts of the shores of Lake Turkana and Kajiado.

The high-resolution malaria prevalence maps produced as part of the analysis have shown that Kenya has additional malaria hotspots, especially in areas least expected. These findings call for a rethinking of malaria burden classification in some regions for effective planning, implementation, resource mobilization, monitoring, and evaluation of malaria interventions in the country.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.23292538">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.23292538" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.23292538">
        <p class="paperTitle">Common infections and neuroimaging markers of dementia in three UK cohort studies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.23292538" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.23292538" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Green, R. E.; Sudre, C. H.; Warren-Gash, C.; Butt, J.; Waterboer, T.; The Insight 46 study team,  ; Hughes, A. D.; Schott, J. M.; Richards, M.; Chaturvedi, N.; Williams, D. M.</p>
        <p class="info">Score: 14.6, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.23292538' target='https://doi.org/10.1101/2023.07.12.23292538'> 10.1101/2023.07.12.23292538</a></p>
        <p class="abstract">IntroductionWe aimed to investigate associations between common infections and neuroimaging markers of dementia risk (brain volume, hippocampal volume, white matter lesions) across three population-based studies.

MethodsWe tested associations between serology measures (pathogen serostatus, cumulative burden, continuous antibody responses) and outcomes using linear regression, including adjustments for total intracranial volume and scanner/clinic information (basic model), age, sex, ethnicity, education, socioeconomic position, alcohol, BMI, and smoking (fully adjusted model). Interactions between serology measures and APOE genotype were tested. Findings were meta-analysed across cohorts (Nmain=2632; NAPOE-interaction=1810).

ResultsSeropositivity to JC virus associated with smaller brain volumes in basic models ({beta}=-3.89ml[-5.81,-1.97],padjusted&lt;0.05); these were largely attenuated in fully adjusted models ({beta}=-1.59ml[-3.55,0.36], p=0.11). No other relationships were robust to multiple testing correction and sensitivity analyses, but several suggestive associations were observed.

DiscussionWe did not find clear evidence for relationships between common infections and markers of dementia risk. Some suggestive findings warrant testing for replication.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
