<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>epidemiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>epidemiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Proteomic prediction of common and rare diseases
Authors: Carrasco-Zanini, J.; Pietzner, M.; Davitte, J.; Surendran, P.; Croteau-Chonka, D. C.; Robins, C.; Torralbo, A.; Tomlinson, C.; Fitzpatrick, N.; Ytsma, C.; Kanno, T.; Gade, S.; Freitag, D.; Ziebell, F.; Denaxas, S.; Betts, J. C.; Wareham, N. J.; Hemingway, H.; Scott, R. A.; Langenberg, C.
Score: 33.9, Published: 2023-07-23 DOI: 10.1101/2023.07.18.23292811
BackgroundFor many diseases there are delays in diagnosis due to a lack of objective biomarkers for disease onset.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/epidemiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="epidemiology" />
<meta property="og:description" content="Proteomic prediction of common and rare diseases
Authors: Carrasco-Zanini, J.; Pietzner, M.; Davitte, J.; Surendran, P.; Croteau-Chonka, D. C.; Robins, C.; Torralbo, A.; Tomlinson, C.; Fitzpatrick, N.; Ytsma, C.; Kanno, T.; Gade, S.; Freitag, D.; Ziebell, F.; Denaxas, S.; Betts, J. C.; Wareham, N. J.; Hemingway, H.; Scott, R. A.; Langenberg, C.
Score: 33.9, Published: 2023-07-23 DOI: 10.1101/2023.07.18.23292811
BackgroundFor many diseases there are delays in diagnosis due to a lack of objective biomarkers for disease onset." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/epidemiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-02T10:38:09+00:00" />
<meta property="article:modified_time" content="2023-08-02T10:38:09+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="epidemiology"/>
<meta name="twitter:description" content="Proteomic prediction of common and rare diseases
Authors: Carrasco-Zanini, J.; Pietzner, M.; Davitte, J.; Surendran, P.; Croteau-Chonka, D. C.; Robins, C.; Torralbo, A.; Tomlinson, C.; Fitzpatrick, N.; Ytsma, C.; Kanno, T.; Gade, S.; Freitag, D.; Ziebell, F.; Denaxas, S.; Betts, J. C.; Wareham, N. J.; Hemingway, H.; Scott, R. A.; Langenberg, C.
Score: 33.9, Published: 2023-07-23 DOI: 10.1101/2023.07.18.23292811
BackgroundFor many diseases there are delays in diagnosis due to a lack of objective biomarkers for disease onset."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "epidemiology",
      "item": "https://trxiv.yorks0n.com/posts/epidemiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "epidemiology",
  "name": "epidemiology",
  "description": "Proteomic prediction of common and rare diseases\nAuthors: Carrasco-Zanini, J.; Pietzner, M.; Davitte, J.; Surendran, P.; Croteau-Chonka, D. C.; Robins, C.; Torralbo, A.; Tomlinson, C.; Fitzpatrick, N.; Ytsma, C.; Kanno, T.; Gade, S.; Freitag, D.; Ziebell, F.; Denaxas, S.; Betts, J. C.; Wareham, N. J.; Hemingway, H.; Scott, R. A.; Langenberg, C.\nScore: 33.9, Published: 2023-07-23 DOI: 10.1101/2023.07.18.23292811\nBackgroundFor many diseases there are delays in diagnosis due to a lack of objective biomarkers for disease onset.",
  "keywords": [
    
  ],
  "articleBody": " Proteomic prediction of common and rare diseases\nAuthors: Carrasco-Zanini, J.; Pietzner, M.; Davitte, J.; Surendran, P.; Croteau-Chonka, D. C.; Robins, C.; Torralbo, A.; Tomlinson, C.; Fitzpatrick, N.; Ytsma, C.; Kanno, T.; Gade, S.; Freitag, D.; Ziebell, F.; Denaxas, S.; Betts, J. C.; Wareham, N. J.; Hemingway, H.; Scott, R. A.; Langenberg, C.\nScore: 33.9, Published: 2023-07-23 DOI: 10.1101/2023.07.18.23292811\nBackgroundFor many diseases there are delays in diagnosis due to a lack of objective biomarkers for disease onset. Whether measuring thousands of proteins offers predictive information across a wide range of diseases is unknown. MethodsIn 41,931 individuals from the UK Biobank Pharma Proteomics Project (UKB-PPP), we integrated [~]3000 plasma proteins with clinical information to derive sparse prediction models for the 10-year incidence of 218 common and rare diseases (81 - 6038 cases). We compared prediction models based on proteins with a) basic clinical information alone, b) basic clinical information + 37 clinical biomarkers, and c) genome-wide polygenic risk scores. ResultsFor 67 pathologically diverse diseases, a model including as few as 5 to 20 proteins was superior to clinical models (median delta C-index = 0.07; range = 0.02 - 0.31) and to clinical models with biomarkers for 52 diseases. In multiple myeloma, for example, a set of 5 proteins significantly improved prediction over basic clinical information (delta C-index = 0.25 (95% confidence interval 0.20 - 0.29)). At a 5% false positive rate (FPR), proteomic prediction (5 proteins) identified individuals at high risk of multiple myeloma (detection rate (DR) = 50%), non-Hodgkin lymphoma (DR = 55%) and motor neuron disease (DR = 29%). At a 20% FPR, proteomic prediction identified individuals at high-risk for pulmonary fibrosis (DR= 80%) and dilated cardiomyopathy (DR = 75%). ConclusionsSparse plasma protein signatures offer novel, clinically useful prediction of common and rare diseases, through disease-specific proteins and protein predictors shared across multiple diseases. (Funded by Medical Research Council, NIHR, Wellcome Trust.)\nCOVID-19 mitigation behaviors and policies limited SARS-CoV-2 transmission in the United States from September 2020 through November 2021\nAuthors: Lopez, V. K.; Kada, S.; Prasad, P. V.; Chin, T.; Cadwell, B. L.; Healy, J. M.; Slayton, R. B.; Biggerstaff, M.; Johansson, M. A.\nScore: 12.3, Published: 2023-07-23 DOI: 10.1101/2023.07.19.23292882\nUnited States jurisdictions implemented varied policies to slow SARS-CoV-2 transmission. Understanding patterns of these policies alongside individuals behaviors can inform effective outbreak response. To do so, we estimated the time-varying reproduction number (Rt), a weekly measure of real-time transmission using US COVID-19 cases from September 2020-November 2021. We then assessed the association between Rt and policies, personal COVID-19 mitigation behaviors, variants, immunity, and social vulnerability indicators using two multi-level regression models. First, we fit a model with state-level policy stringency according to the Oxford Stringency Index, a composite indicator reflecting the strictness of COVID-19 policies and strength of pandemic-related communication. Our second model included a subset of specific policies. We found that personal mitigation behaviors and vaccination were more strongly associated with decreased transmission than policies. Importantly, transmission was reduced not by a single measure, but by various layered measures. These results underscore the need for policy, behavior change, and risk communication integration to reduce virus transmission during epidemics.\nThe Impact of the UK COVID-19 Lockdown on the Screening, Diagnostics and Incidence of Breast, Colorectal, Lung and Prostate Cancer in the UK: a Population-Based Cohort Study\nAuthors: Barclay, N. L.; Pineda-Moncusi, M. L.; Jodicke, A. M.; Prieto-Alhambra, D.; Raventos, B.; Newby, D.; Delmestri, A.; Man, W. Y.; Chen, X.; Catala, M.\nScore: 7.3, Published: 2023-07-26 DOI: 10.1101/2023.07.21.23292937\nObjectivesThis study aimed to assess the impact of the COVID-19 lockdown on the screening and diagnosis of breast, colorectal, lung, and prostate cancer. The study also investigated whether the rates returned to pre-pandemic levels by December 2021. DesignCohort study. SettingElectronic health records from UK primary care Clinical Practice Research Datalink (CPRD) GOLD database. ParticipantsThe study included individuals registered with CPRD GOLD between January 2017 and December 2021, with at least 365 days of prior observation. Main outcome measuresThe study focused on screening, diagnostic tests, referrals and diagnoses of first-ever breast, colorectal, lung, and prostate cancer. Incidence rates (IR) were stratified by age, sex and region, and incidence rate ratios (IRR) were calculated to compare rates during and after lockdown with the reference period before lockdown. Forecasted rates were estimated using negative binomial regression models. ResultsAmong 5,191,650 eligible participants, the initial lockdown resulted in reduced screening and diagnostic tests for all cancers, which remained dramatically reduced across the whole observation period for almost all tests investigated. For cancer incidence rates, there were significant IRR reductions in breast (0.69), colorectal (0.74), and prostate (0.71) cancers. However, the reduction in lung cancer incidence (0.92) was non-significant. Extrapolating to the entire UK population, an estimated 18,000 breast, 13,000 colorectal, 10,000 lung, and 21,000 prostate cancer diagnoses were missed from March 2020 to December 2021. ConclusionThe national COVID-19 lockdown in the UK had a substantial impact on cancer screening, diagnostic tests, referrals and diagnoses. Although incidence rates started to recover after the lockdown, they remained significantly lower than pre-pandemic levels for breast and prostate cancers and associated tests. Delays in diagnosis are likely to have adverse consequences on cancer stage, treatment initiation, mortality rates, and years of life lost. Urgent strategies are needed to identify undiagnosed cases and address the long-term implications of delayed diagnoses. WHAT IS ALREADY KNOWN ON THIS TOPICO_LIBreast, colorectal, lung, and prostate cancer are the most common causes of cancer death in the UK. C_LIO_LIThe COVID-19 pandemic led to the postponement of cancer screening programs and reductions in diagnostic tests, resulting in delays in diagnosis and treatment initiation, impacting prognosis and mortality rates. C_LIO_LIComprehensive data on the impact of changing social restrictions and post-lockdown periods is lacking in the UK, along with an assessment of specific screening pathways and patient experiences within the healthcare system. C_LI WHAT THIS STUDY ADDSO_LIThe first UK national COVID-19 lockdown resulted in reductions in screening, diagnostic tests, and referrals, particularly for mammograms, colonoscopies, and visits to breast surgeons, leading to underdiagnosis of breast, colorectal, and prostate cancers. Despite some increase in rates after the lockdown, they remained significantly lower than pre-pandemic levels by December 2021, particularly for prostate cancer. C_LIO_LIMost affected populations were women aged 60-79 years for breast and colorectal cancer; men aged 60-79 years for lung cancer; and men aged 40-59 years for prostate cancer. C_LIO_LIDelays in diagnosis are likely to have consequences on cancer stage at diagnosis, treatment initiation, mortality rates, and total years of life lost. Strategies such as public awareness campaigns, targeted screening programs, and improved coordination between primary care and hospitals are needed to address the backlog and identify the potential [~]62,000 missed cancer cases in the UK. C_LI\nInteractions between climate change, urban infrastructure and mobility are driving dengue emergence in Vietnam\nAuthors: Gibb, R.; Colon-Gonzalez, F. J.; Lan, P. T.; Huong, P. T.; Nam, V. S.; Duoc, V. T.; Do, H. T.; Dong, N. T.; Chien, V. C.; Trang, L. T. T.; Quoc, D. K.; Hoa, T. M.; Tai, N. H.; Hang, T. T.; Tsarouchi, G.; Ainscoe, E.; Harpham, Q.; Hofmann, B.; Lumbroso, D.; Brady, O. J.; Lowe, R.\nScore: 7.0, Published: 2023-08-01 DOI: 10.1101/2023.07.25.23293110\nDengue is expanding globally, but how dengue emergence is shaped locally by interactions between climatic and socio-environmental factors is not well understood. Here, we investigate the drivers of dengue incidence and emergence in Vietnam, through analyzing 23-years of monthly district-level case data spanning a period of significant socioeconomic change (1998-2020). We show that urban infrastructure factors (sanitation, water supply and long-term urban growth) predict local spatial patterns of dengue incidence, while human mobility is a more influential driver in subtropical northern regions than the endemic south. Temperature is the dominant factor shaping dengues geographical distribution and dynamics, and using long-term reanalysis temperature data we show that recent warming (since 1950) has generally expanded transmission risk throughout Vietnam, and most strongly in current dengue emergence hotspots (e.g. southern central regions and Ha Noi). In contrast, effects of hydrometeorology are complex, multi-scalar and dependent on local context: risk increases under both short-term precipitation excess and long-term drought, but improvements in water supply largely mitigate drought-associated risks except under extreme conditions. Our findings challenge the assumption that dengue is an urban disease, instead suggesting that incidence peaks in transitional landscapes with intermediate infrastructure provision, and provide evidence that interactions between recent climate change and mobility have contributed to dengues ongoing expansion throughout Vietnam.\nLineage replacement and evolution captured by the United Kingdom Covid Infection Survey\nAuthors: Lythgoe, K. A.; Golubchik, T.; Hall, M.; House, T.; Cahuantzi, R.; MacIntyre-Cockett, G.; Fryer, H.; Thomson, L.; Nurtay, A.; Ghafari, M.; Buck, D.; Green, A.; Trebes, A.; Piazza, P.; Lonie, L. J.; Studley, R.; Rourke, E.; Smith, D.; Bashton, M.; Nelson, A.; Crown, M.; McCann, C.; Young, G. R.; de Santos, R. A. N.; Richards, Z.; Tariq, A.; Wellcome Sanger Institute COVID-19 Surveillance Team, ; COVID-19 Infection Survey Group, ; The COVID-19 Genomics UK (COG-UK) consortium, ; Fraser, C.; Diamond, I.; Barrett, J.; Walker, A. S.; Bonsall, D.\nScore: 15.2, Published: 2023-07-25 DOI: 10.1101/2022.01.05.21268323\nThe Office for National Statistics COVID-19 Infection Survey (ONS-CIS) is the largest surveillance study of SARS-CoV-2 positivity in the community, and collected data on the United Kingdom (UK) epidemic from April 2020 until March 2023 before being paused. Here, we report on the epidemiological and evolutionary dynamics of SARS-CoV-2 determined by analysing the sequenced samples collected by the ONS-CIS during this period. We observed a series of sweeps or partial sweeps, with each sweeping lineage having a distinct growth advantage compared to their predecessors. The sweeps also generated an alternating pattern in which most samples had either S-gene target failure (SGTF) or non- SGTF over time. Evolution was characterised by steadily increasing divergence and diversity within lineages, but with step increases in divergence associated with each sweeping major lineage. This led to a faster overall rate of evolution when measured at the between-lineage level compared to within lineages, and fluctuating levels of diversity. These observations highlight the value of viral sequencing integrated into community surveillance studies to monitor the viral epidemiology and evolution of SARS-CoV-2, and potentially other pathogens, particularly in the current phase of the pandemic with routine RT-PCR testing now ended in the community.\nRobust use of phenotypic heterogeneity at drug target genes for mechanistic insights: application of cis-multivariable Mendelian randomization to GLP1R gene region\nAuthors: Patel, A.; Gill, D.; Shungin, D.; Mantzoros, C. S.; Knudsen, L. B.; Bowden, J.; Burgess, S.\nScore: 5.0, Published: 2023-07-25 DOI: 10.1101/2023.07.20.23292958\nPhenotypic heterogeneity at genomic loci encoding drug targets can be exploited by multivariable Mendelian randomization to provide insight on the pathways by which pharmacological interventions may affect disease risk. However, statistical inference in such investigations may be poor if overdispersion heterogeneity in measured genetic associations is unaccounted for. In this work, we first develop conditional F-statistics for dimension-reduced genetic associations that enable more accurate measurement of phenotypic heterogeneity. We then develop a novel extension for two-sample multivariable Mendelian randomization that accounts for overdispersion heterogeneity in dimension-reduced genetic associations. Our empirical focus is to use genetic variants in the GLP1R gene region to understand the mechanism by which GLP1R agonism affects coronary artery disease (CAD) risk. Colocalization analyses indicate that distinct variants in the GLP1R gene region are associated with body mass index and type 2 diabetes. Multivariable Mendelian randomization analyses that were corrected for overdispersion heterogeneity suggest that bodyweight lowering rather than type 2 diabetes liability lowering effects of GLP1R agonism are more likely contributing to reduced CAD risk. Tissue-specific analyses prioritised brain tissue as the most likely to be relevant for CAD risk, of the tissues considered. We hope the multivariable Mendelian randomization approach illustrated here is widely applicable to better understand mechanisms linking drug targets to diseases outcomes, and hence to guide drug development efforts.\nThe historical fingerprint and future impact of climate change on childhood malaria in Africa\nAuthors: Carlson, C. J.; Carleton, T. A.; Odoulami, R.; Trisos, C. H.\nScore: 15.6, Published: 2023-07-25 DOI: 10.1101/2023.07.16.23292713\nThe health burden of anthropogenic climate change is growing exponentially, but present-day impacts remain difficult to measure 1-3. Here, we leverage a recently-published comprehensive dataset of 50,425 population surveys 4 to investigate whether human-caused climate change has increased the burden of childhood malaria across sub-Saharan Africa. In historical data, we find that prevalence shows a robust response to temperature and extreme precipitation, consistent with expectations from previous empirical and epidemiological work. Comparing historical climate reconstructions to counterfactual simulations without anthropogenic warming, we find two-to-one odds that human-caused climate change has increased the overall prevalence of childhood malaria across sub-Saharan Africa since 1901. We estimate that by 2014, human-caused climate change was responsible for an average of 84 excess cases of malaria per 100,000 children ages 2 to 10, with higher elevation and cooler regions in southern and east Africa having greater increases. Under future climate change, we project increasing temperatures could plausibly accelerate the eradication of malaria in west and central Africa, where the present-day burden is highest, leading to continent-wide average reductions of 89 (low greenhouse gas emissions, SSP1-RCP2.6) to 1,750 (high emissions, SSP5-RCP8.5) cases per 100,000 children by the end of the century. However, we find that limiting future global warming to below 2{degrees}C (SSP1-RCP2.6) compared to[~] 3{degrees}C (SSP2-RCP4.5) could prevent an average of 496 excess cases in southern Africa, and 40 excess cases in east Africa,per 100,000 children by 2100. Our study resolves a decades-old debate about one of the earliest health impacts of global warming, and provides a template for future work measuring the true global burden of climate change.\nPatient-Related Metadata Reported in Sequencing Studies of SARS-CoV-2: Protocol for a Scoping Review and Bibliometric Analysis\nAuthors: OConnor, K.; Weissenbacher, D.; Elyaderani, A.; Scotch, M.; Gonzalez-Hernandez, G.\nScore: 2.2, Published: 2023-07-16 DOI: 10.1101/2023.07.14.23292681\nBackgroundSince the onset of the COVID-19 pandemic, there has been an unprecedented effort in genomic epidemiology to sequence the SARS-CoV-2 virus and examine its molecular evolution. This has been facilitated by the availability of publicly accessible databases, GISAID and GenBank, which collectively hold millions of SARS-CoV-2 sequence records. However, genomic epidemiology seeks to go beyond phylogenetic analysis by linking genetic information to patient demographics and disease outcomes, enabling a comprehensive understanding of transmission dynamics and disease impact. While these repositories include some patient-related information, such as the location of the infected host, the granularity of this data and the inclusion of demographic and clinical details are inconsistent. Additionally, the extent to which patient-related metadata is reported in published sequencing studies remains largely unexplored. Therefore, it is essential to assess the extent and quality of patient-related metadata reported in SARS-CoV-2 sequencing studies. Moreover, there is limited linkage between published articles and sequence repositories, hindering the identification of relevant studies. Traditional search strategies based on keywords may miss relevant articles. To overcome these challenges, this study proposes the use of an automated classifier to identify relevant articles. ObjectiveThis study aims to conduct a systematic and comprehensive scoping review, along with a bibliometric analysis, to assess the reporting of patient-related metadata in SARS-CoV-2 sequencing studies. MethodsThe NIHs LitCovid collection will be used for the machine learning classification, while an independent search will be conducted in PubMed. Data extraction will be conducted using Covidence, and the extracted data will be synthesized and summarized to quantify the availability of patient metadata in the published literature of SARS-CoV-2 sequencing studies. For the bibliometric analysis, relevant data points, such as author affiliations, journal information, and citation metrics, will be extracted. ResultsThe study will report findings on the extent and types of patient-related metadata reported in genomic viral sequencing studies of SARS-CoV-2. The scoping review will identify gaps in the reporting of patient metadata and make recommendations for improving the quality and consistency of reporting in this area. The bibliometric analysis will uncover trends and patterns in the reporting of patient-related metadata, such as differences in reporting based on study types or geographic regions. Co-occurrence networks of author keywords will also be presented to highlight frequent themes and their associations with patient metadata reporting. ConclusionThis study will contribute to advancing knowledge in the field of genomic epidemiology by providing a comprehensive overview of the reporting of patient-related metadata in SARS-CoV-2 sequencing studies. The insights gained from this study may help improve the quality and consistency of reporting patient metadata, enhancing the utility of sequence metadata and facilitating future research on infectious diseases. The findings may also inform the development of machine learning methods to automatically extract patient-related information from sequencing studies.\nEpidemiological and health economic implications of symptom propagation in respiratory pathogens: A mathematical modelling investigation\nAuthors: Asplin, P.; Keeling, M. J.; Mancy, R.; Hill, E. M.\nScore: 5.1, Published: 2023-07-13 DOI: 10.1101/2023.07.12.23292544\nBackgroundRespiratory pathogens inflict a substantial burden on public health and the economy. Although the severity of symptoms caused by these pathogens can vary from asymptomatic to fatal, the factors that determine symptom severity are not fully understood. Correlations in symptoms between infector-infectee pairs, for which evidence is accumulating, can generate large-scale clusters of severe infections that could be devastating to those most at risk, whilst also conceivably leading to chains of mild or asymptomatic infections that generate widespread immunity with minimal cost to public health. Although this effect could be harnessed to amplify the impact of interventions that reduce symptom severity, the mechanistic representation of symptom propagation within mathematical and health economic modelling of respiratory diseases is understudied. Methods and FindingsWe propose a novel framework for incorporating different levels of symptom propagation into models of infectious disease transmission via a single parameter, . Varying tunes the model from having no symptom propagation ( = 0, as typically assumed) to one where symptoms always propagate ( = 1). For parameters corresponding to three respiratory pathogens -- seasonal influenza, pandemic influenza and SARS-CoV-2 -- we explored how symptom propagation impacted the relative epidemiological and health-economic performance of three interventions, conceptualised as vaccines with different actions: symptom-attenuating (labelled SA), infection-blocking (IB) and infection-blocking admitting only mild breakthrough infections (IB MB). In the absence of interventions, with fixed underlying epidemiological parameters, stronger symptom propagation increased the proportion of cases that were severe. For SA and IB MB, interventions were more effective at reducing prevalence (all infections and severe cases) for higher strengths of symptom propagation. For IB, symptom propagation had no impact on effectiveness, and for seasonal influenza this intervention type was more effective than SA at reducing severe infections for all levels of symptom propagation. For pandemic influenza and SARS-CoV-2, at low intervention uptake, SA was more effective than IB for all levels of symptom propagation; for high uptake, SA only became more effective under strong symptom propagation. Health economic assessments found that for SA-type interventions, the amount one could spend on control whilst maintaining a cost-effective intervention (termed threshold unit intervention cost) was very sensitive to the strength of symptom propagation. ConclusionsOverall, the preferred intervention type depended on the combination of the strength of symptom propagation and uptake. Given the importance of determining robust public health responses, we highlight the need to gather further data on symptom propagation, with our modelling framework acting as a template for future analysis.\nMarburg Virus Disease outbreaks, mathematical models, and disease parameters: a Systematic Review\nAuthors: Cuomo-Dannenburg, G.; McCain, K.; McCabe, R.; Unwin, H. J. T.; Doohan, P.; Nash, R. K.; Hicks, J. T.; Charniga, K.; Geismar, C.; Lambert, B.; Nikitin, D.; Skarp, J. E.; Wardle, J.; Pathogen Epidemiology Review Group, ; Kont, M.; Bhatia, S.; Imai, N.; van Elsland, S. L.; Cori, A.; Morgenstern, C.\nScore: 17.8, Published: 2023-07-12 DOI: 10.1101/2023.07.10.23292424\nBackgroundRecent Marburg virus disease (MVD) outbreaks in Equatorial Guinea and Tanzania highlighted the importance of better understanding this highly lethal infectious pathogen. Past epidemics of Ebola, COVID-19, and other pathogens have re-emphasised the usefulness of mathematical models in guiding public health responses during outbreaks. MethodsWe conducted a systematic review, registered with PROSPERO (CRD42023393345) and reported according to PRISMA guidelines, of peer-reviewed papers reporting historical out-breaks, modelling studies and epidemiological parameters focused on MVD, including contextual information. We searched PubMed and Web of Science until 31st March 2023. Two reviewers evaluated all titles and abstracts, with consensus-based decision-making. To ensure agreement, 31% (13/42) of studies were double-extracted and a custom-designed quality assessment questionnaire was used to assess the risk of bias. FindingsWe present detailed outbreak, model and parameter information on 970 reported cases and 818 deaths from MVD until 31 March 2023. Analysis of historical outbreaks and sero-prevalence estimates suggests the possibility of undetected MVD outbreaks, asymptomatic transmission and/or cross-reactivity with other pathogens. Only one study presented a mathematical model of MVD transmission. We estimate an unadjusted, pooled total random effect case fatality ratio for MVD of 61.9% (95% CI: 38.8-80.6%, I2=93%). We identify key epidemiological parameters relating to transmission and natural history for which there are few estimates. InterpretationThis review provides a comprehensive overview of the epidemiology of MVD, identifying key knowledge gaps about this pathogen. The extensive collection of knowledge gathered here will be crucial in developing mathematical models for use in the early stages of future outbreaks of MVD. All data are published alongside this article with functionality to easily update the database as new data become available. FundingMRC Centre for Global Infectious Disease Analysis Research in ContextO_LIEvidence before this study We searched Web of Science and PubMed up to 31 March 2023 using the search terms Marburg virus, epidemiology, outbreaks, models, transmissibility, severity, delays, risk factors, mutation rates and seroprevalence. We found five systematic reviews, all of which considered MVD alongside Ebola virus disease (EVD). One modelling study of Marburg virus disease (MVD) focused on animals, and not on computational models to understand past or project future disease transmission. One systematic review collated risk factors for transmission based on four MVD studies, but did not report attack rates due to missing underlying MVD estimates; another systematic review pooled estimates of MVD case fatality ratios (CFR): 53.8% (95% CI: 26.5-80.0%) and seroprevalence: 1.2% (95% CI: 0.5-2.0%). No systematic review covered transmission models of MVD, and the impact of public health and social measures is unknown. C_LIO_LIAdded value of this study We provide a comprehensive summary of the available, peer-reviewed literature of historical outbreaks, transmission models and parameters for MVD. Meta-analysis of existing estimates of CFRs, and our original estimates based on historical outbreak information, illustrate the severity of MVD with our pooled random effect estimated CFR of 61.9% (95% CI: 38.8-80.6%, I2=93%). We demonstrate the sparsity of evidence on MVD transmission and disease dynamics, particularly on transmissibility and natural history, which are key input parameters for computational models supporting outbreak response. Our work highlights key areas where further disease characterization is necessary. C_LIO_LIImplications of all the available evidence Previous outbreaks of infectious pathogens emphasized the usefulness of computational modelling in assessing epidemic trajectories and the impact of mitigation strategies. Our study provides necessary information for using mathematical models in future outbreaks of MVD, identifies uncertainties and knowledge gaps in MVD transmission and natural history, and highlights the severity of MVD. C_LI\n",
  "wordCount" : "3822",
  "inLanguage": "en",
  "datePublished": "2023-08-02T10:38:09Z",
  "dateModified": "2023-08-02T10:38:09Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/epidemiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      epidemiology
    </h1>
    <div class="post-meta"><span>updated on August 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.23292811">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.23292811" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.23292811">
        <p class="paperTitle">Proteomic prediction of common and rare diseases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.23292811" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.23292811" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carrasco-Zanini, J.; Pietzner, M.; Davitte, J.; Surendran, P.; Croteau-Chonka, D. C.; Robins, C.; Torralbo, A.; Tomlinson, C.; Fitzpatrick, N.; Ytsma, C.; Kanno, T.; Gade, S.; Freitag, D.; Ziebell, F.; Denaxas, S.; Betts, J. C.; Wareham, N. J.; Hemingway, H.; Scott, R. A.; Langenberg, C.</p>
        <p class="info">Score: 33.9, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.23292811' target='https://doi.org/10.1101/2023.07.18.23292811'> 10.1101/2023.07.18.23292811</a></p>
        <p class="abstract">BackgroundFor many diseases there are delays in diagnosis due to a lack of objective biomarkers for disease onset. Whether measuring thousands of proteins offers predictive information across a wide range of diseases is unknown.

MethodsIn 41,931 individuals from the UK Biobank Pharma Proteomics Project (UKB-PPP), we integrated [~]3000 plasma proteins with clinical information to derive sparse prediction models for the 10-year incidence of 218 common and rare diseases (81 - 6038 cases). We compared prediction models based on proteins with a) basic clinical information alone, b) basic clinical information &#43; 37 clinical biomarkers, and c) genome-wide polygenic risk scores.

ResultsFor 67 pathologically diverse diseases, a model including as few as 5 to 20 proteins was superior to clinical models (median delta C-index = 0.07; range = 0.02 - 0.31) and to clinical models with biomarkers for 52 diseases. In multiple myeloma, for example, a set of 5 proteins significantly improved prediction over basic clinical information (delta C-index = 0.25 (95% confidence interval 0.20 - 0.29)). At a 5% false positive rate (FPR), proteomic prediction (5 proteins) identified individuals at high risk of multiple myeloma (detection rate (DR) = 50%), non-Hodgkin lymphoma (DR = 55%) and motor neuron disease (DR = 29%). At a 20% FPR, proteomic prediction identified individuals at high-risk for pulmonary fibrosis (DR= 80%) and dilated cardiomyopathy (DR = 75%).

ConclusionsSparse plasma protein signatures offer novel, clinically useful prediction of common and rare diseases, through disease-specific proteins and protein predictors shared across multiple diseases.

(Funded by Medical Research Council, NIHR, Wellcome Trust.)</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.23292882">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.23292882" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.23292882">
        <p class="paperTitle">COVID-19 mitigation behaviors and policies limited SARS-CoV-2 transmission in the United States from September 2020 through November 2021</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.23292882" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.23292882" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lopez, V. K.; Kada, S.; Prasad, P. V.; Chin, T.; Cadwell, B. L.; Healy, J. M.; Slayton, R. B.; Biggerstaff, M.; Johansson, M. A.</p>
        <p class="info">Score: 12.3, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.23292882' target='https://doi.org/10.1101/2023.07.19.23292882'> 10.1101/2023.07.19.23292882</a></p>
        <p class="abstract">United States jurisdictions implemented varied policies to slow SARS-CoV-2 transmission. Understanding patterns of these policies alongside individuals behaviors can inform effective outbreak response. To do so, we estimated the time-varying reproduction number (Rt), a weekly measure of real-time transmission using US COVID-19 cases from September 2020-November 2021. We then assessed the association between Rt and policies, personal COVID-19 mitigation behaviors, variants, immunity, and social vulnerability indicators using two multi-level regression models. First, we fit a model with state-level policy stringency according to the Oxford Stringency Index, a composite indicator reflecting the strictness of COVID-19 policies and strength of pandemic-related communication. Our second model included a subset of specific policies. We found that personal mitigation behaviors and vaccination were more strongly associated with decreased transmission than policies. Importantly, transmission was reduced not by a single measure, but by various layered measures. These results underscore the need for policy, behavior change, and risk communication integration to reduce virus transmission during epidemics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.23292937">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.23292937" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.23292937">
        <p class="paperTitle">The Impact of the UK COVID-19 Lockdown on the Screening, Diagnostics and Incidence of Breast, Colorectal, Lung and Prostate Cancer in the UK: a Population-Based Cohort Study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.23292937" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.23292937" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Barclay, N. L.; Pineda-Moncusi, M. L.; Jodicke, A. M.; Prieto-Alhambra, D.; Raventos, B.; Newby, D.; Delmestri, A.; Man, W. Y.; Chen, X.; Catala, M.</p>
        <p class="info">Score: 7.3, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.23292937' target='https://doi.org/10.1101/2023.07.21.23292937'> 10.1101/2023.07.21.23292937</a></p>
        <p class="abstract">ObjectivesThis study aimed to assess the impact of the COVID-19 lockdown on the screening and diagnosis of breast, colorectal, lung, and prostate cancer. The study also investigated whether the rates returned to pre-pandemic levels by December 2021.

DesignCohort study.

SettingElectronic health records from UK primary care Clinical Practice Research Datalink (CPRD) GOLD database.

ParticipantsThe study included individuals registered with CPRD GOLD between January 2017 and December 2021, with at least 365 days of prior observation.

Main outcome measuresThe study focused on screening, diagnostic tests, referrals and diagnoses of first-ever breast, colorectal, lung, and prostate cancer. Incidence rates (IR) were stratified by age, sex and region, and incidence rate ratios (IRR) were calculated to compare rates during and after lockdown with the reference period before lockdown. Forecasted rates were estimated using negative binomial regression models.

ResultsAmong 5,191,650 eligible participants, the initial lockdown resulted in reduced screening and diagnostic tests for all cancers, which remained dramatically reduced across the whole observation period for almost all tests investigated. For cancer incidence rates, there were significant IRR reductions in breast (0.69), colorectal (0.74), and prostate (0.71) cancers. However, the reduction in lung cancer incidence (0.92) was non-significant. Extrapolating to the entire UK population, an estimated 18,000 breast, 13,000 colorectal, 10,000 lung, and 21,000 prostate cancer diagnoses were missed from March 2020 to December 2021.

ConclusionThe national COVID-19 lockdown in the UK had a substantial impact on cancer screening, diagnostic tests, referrals and diagnoses. Although incidence rates started to recover after the lockdown, they remained significantly lower than pre-pandemic levels for breast and prostate cancers and associated tests. Delays in diagnosis are likely to have adverse consequences on cancer stage, treatment initiation, mortality rates, and years of life lost. Urgent strategies are needed to identify undiagnosed cases and address the long-term implications of delayed diagnoses.

WHAT IS ALREADY KNOWN ON THIS TOPICO_LIBreast, colorectal, lung, and prostate cancer are the most common causes of cancer death in the UK.
C_LIO_LIThe COVID-19 pandemic led to the postponement of cancer screening programs and reductions in diagnostic tests, resulting in delays in diagnosis and treatment initiation, impacting prognosis and mortality rates.
C_LIO_LIComprehensive data on the impact of changing social restrictions and post-lockdown periods is lacking in the UK, along with an assessment of specific screening pathways and patient experiences within the healthcare system.
C_LI

WHAT THIS STUDY ADDSO_LIThe first UK national COVID-19 lockdown resulted in reductions in screening, diagnostic tests, and referrals, particularly for mammograms, colonoscopies, and visits to breast surgeons, leading to underdiagnosis of breast, colorectal, and prostate cancers. Despite some increase in rates after the lockdown, they remained significantly lower than pre-pandemic levels by December 2021, particularly for prostate cancer.
C_LIO_LIMost affected populations were women aged 60-79 years for breast and colorectal cancer; men aged 60-79 years for lung cancer; and men aged 40-59 years for prostate cancer.
C_LIO_LIDelays in diagnosis are likely to have consequences on cancer stage at diagnosis, treatment initiation, mortality rates, and total years of life lost. Strategies such as public awareness campaigns, targeted screening programs, and improved coordination between primary care and hospitals are needed to address the backlog and identify the potential [~]62,000 missed cancer cases in the UK.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.23293110">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.23293110" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.23293110">
        <p class="paperTitle">Interactions between climate change, urban infrastructure and mobility are driving dengue emergence in Vietnam</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.23293110" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.23293110" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gibb, R.; Colon-Gonzalez, F. J.; Lan, P. T.; Huong, P. T.; Nam, V. S.; Duoc, V. T.; Do, H. T.; Dong, N. T.; Chien, V. C.; Trang, L. T. T.; Quoc, D. K.; Hoa, T. M.; Tai, N. H.; Hang, T. T.; Tsarouchi, G.; Ainscoe, E.; Harpham, Q.; Hofmann, B.; Lumbroso, D.; Brady, O. J.; Lowe, R.</p>
        <p class="info">Score: 7.0, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.23293110' target='https://doi.org/10.1101/2023.07.25.23293110'> 10.1101/2023.07.25.23293110</a></p>
        <p class="abstract">Dengue is expanding globally, but how dengue emergence is shaped locally by interactions between climatic and socio-environmental factors is not well understood. Here, we investigate the drivers of dengue incidence and emergence in Vietnam, through analyzing 23-years of monthly district-level case data spanning a period of significant socioeconomic change (1998-2020). We show that urban infrastructure factors (sanitation, water supply and long-term urban growth) predict local spatial patterns of dengue incidence, while human mobility is a more influential driver in subtropical northern regions than the endemic south. Temperature is the dominant factor shaping dengues geographical distribution and dynamics, and using long-term reanalysis temperature data we show that recent warming (since 1950) has generally expanded transmission risk throughout Vietnam, and most strongly in current dengue emergence hotspots (e.g. southern central regions and Ha Noi). In contrast, effects of hydrometeorology are complex, multi-scalar and dependent on local context: risk increases under both short-term precipitation excess and long-term drought, but improvements in water supply largely mitigate drought-associated risks except under extreme conditions. Our findings challenge the assumption that dengue is an urban disease, instead suggesting that incidence peaks in transitional landscapes with intermediate infrastructure provision, and provide evidence that interactions between recent climate change and mobility have contributed to dengues ongoing expansion throughout Vietnam.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.01.05.21268323">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.01.05.21268323" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.01.05.21268323">
        <p class="paperTitle">Lineage replacement and evolution captured by the United Kingdom Covid Infection Survey</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.01.05.21268323" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.01.05.21268323" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lythgoe, K. A.; Golubchik, T.; Hall, M.; House, T.; Cahuantzi, R.; MacIntyre-Cockett, G.; Fryer, H.; Thomson, L.; Nurtay, A.; Ghafari, M.; Buck, D.; Green, A.; Trebes, A.; Piazza, P.; Lonie, L. J.; Studley, R.; Rourke, E.; Smith, D.; Bashton, M.; Nelson, A.; Crown, M.; McCann, C.; Young, G. R.; de Santos, R. A. N.; Richards, Z.; Tariq, A.; Wellcome Sanger Institute COVID-19 Surveillance Team,  ; COVID-19 Infection Survey Group,  ; The COVID-19 Genomics UK (COG-UK) consortium,  ; Fraser, C.; Diamond, I.; Barrett, J.; Walker, A. S.; Bonsall, D.</p>
        <p class="info">Score: 15.2, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.01.05.21268323' target='https://doi.org/10.1101/2022.01.05.21268323'> 10.1101/2022.01.05.21268323</a></p>
        <p class="abstract">The Office for National Statistics COVID-19 Infection Survey (ONS-CIS) is the largest surveillance study of SARS-CoV-2 positivity in the community, and collected data on the United Kingdom (UK) epidemic from April 2020 until March 2023 before being paused. Here, we report on the epidemiological and evolutionary dynamics of SARS-CoV-2 determined by analysing the sequenced samples collected by the ONS-CIS during this period. We observed a series of sweeps or partial sweeps, with each sweeping lineage having a distinct growth advantage compared to their predecessors. The sweeps also generated an alternating pattern in which most samples had either S-gene target failure (SGTF) or non- SGTF over time. Evolution was characterised by steadily increasing divergence and diversity within lineages, but with step increases in divergence associated with each sweeping major lineage. This led to a faster overall rate of evolution when measured at the between-lineage level compared to within lineages, and fluctuating levels of diversity. These observations highlight the value of viral sequencing integrated into community surveillance studies to monitor the viral epidemiology and evolution of SARS-CoV-2, and potentially other pathogens, particularly in the current phase of the pandemic with routine RT-PCR testing now ended in the community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.23292958">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.23292958" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.23292958">
        <p class="paperTitle">Robust use of phenotypic heterogeneity at drug target genes for mechanistic insights: application of cis-multivariable Mendelian randomization to GLP1R gene region</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.23292958" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.23292958" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Patel, A.; Gill, D.; Shungin, D.; Mantzoros, C. S.; Knudsen, L. B.; Bowden, J.; Burgess, S.</p>
        <p class="info">Score: 5.0, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.23292958' target='https://doi.org/10.1101/2023.07.20.23292958'> 10.1101/2023.07.20.23292958</a></p>
        <p class="abstract">Phenotypic heterogeneity at genomic loci encoding drug targets can be exploited by multivariable Mendelian randomization to provide insight on the pathways by which pharmacological interventions may affect disease risk. However, statistical inference in such investigations may be poor if overdispersion heterogeneity in measured genetic associations is unaccounted for. In this work, we first develop conditional F-statistics for dimension-reduced genetic associations that enable more accurate measurement of phenotypic heterogeneity. We then develop a novel extension for two-sample multivariable Mendelian randomization that accounts for overdispersion heterogeneity in dimension-reduced genetic associations. Our empirical focus is to use genetic variants in the GLP1R gene region to understand the mechanism by which GLP1R agonism affects coronary artery disease (CAD) risk. Colocalization analyses indicate that distinct variants in the GLP1R gene region are associated with body mass index and type 2 diabetes. Multivariable Mendelian randomization analyses that were corrected for overdispersion heterogeneity suggest that bodyweight lowering rather than type 2 diabetes liability lowering effects of GLP1R agonism are more likely contributing to reduced CAD risk. Tissue-specific analyses prioritised brain tissue as the most likely to be relevant for CAD risk, of the tissues considered. We hope the multivariable Mendelian randomization approach illustrated here is widely applicable to better understand mechanisms linking drug targets to diseases outcomes, and hence to guide drug development efforts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.23292713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.23292713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.23292713">
        <p class="paperTitle">The historical fingerprint and future impact of climate change on childhood malaria in Africa</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.23292713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.23292713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carlson, C. J.; Carleton, T. A.; Odoulami, R.; Trisos, C. H.</p>
        <p class="info">Score: 15.6, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.23292713' target='https://doi.org/10.1101/2023.07.16.23292713'> 10.1101/2023.07.16.23292713</a></p>
        <p class="abstract">The health burden of anthropogenic climate change is growing exponentially, but present-day impacts remain difficult to measure 1-3. Here, we leverage a recently-published comprehensive dataset of 50,425 population surveys 4 to investigate whether human-caused climate change has increased the burden of childhood malaria across sub-Saharan Africa. In historical data, we find that prevalence shows a robust response to temperature and extreme precipitation, consistent with expectations from previous empirical and epidemiological work. Comparing historical climate reconstructions to counterfactual simulations without anthropogenic warming, we find two-to-one odds that human-caused climate change has increased the overall prevalence of childhood malaria across sub-Saharan Africa since 1901. We estimate that by 2014, human-caused climate change was responsible for an average of 84 excess cases of malaria per 100,000 children ages 2 to 10, with higher elevation and cooler regions in southern and east Africa having greater increases. Under future climate change, we project increasing temperatures could plausibly accelerate the eradication of malaria in west and central Africa, where the present-day burden is highest, leading to continent-wide average reductions of 89 (low greenhouse gas emissions, SSP1-RCP2.6) to 1,750 (high emissions, SSP5-RCP8.5) cases per 100,000 children by the end of the century. However, we find that limiting future global warming to below 2{degrees}C (SSP1-RCP2.6) compared to[~] 3{degrees}C (SSP2-RCP4.5) could prevent an average of 496 excess cases in southern Africa, and 40 excess cases in east Africa,per 100,000 children by 2100. Our study resolves a decades-old debate about one of the earliest health impacts of global warming, and provides a template for future work measuring the true global burden of climate change.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.23292681">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.23292681" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.23292681">
        <p class="paperTitle">Patient-Related Metadata Reported in Sequencing Studies of SARS-CoV-2: Protocol for a Scoping Review and Bibliometric Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.23292681" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.23292681" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: OConnor, K.; Weissenbacher, D.; Elyaderani, A.; Scotch, M.; Gonzalez-Hernandez, G.</p>
        <p class="info">Score: 2.2, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.23292681' target='https://doi.org/10.1101/2023.07.14.23292681'> 10.1101/2023.07.14.23292681</a></p>
        <p class="abstract">BackgroundSince the onset of the COVID-19 pandemic, there has been an unprecedented effort in genomic epidemiology to sequence the SARS-CoV-2 virus and examine its molecular evolution. This has been facilitated by the availability of publicly accessible databases, GISAID and GenBank, which collectively hold millions of SARS-CoV-2 sequence records. However, genomic epidemiology seeks to go beyond phylogenetic analysis by linking genetic information to patient demographics and disease outcomes, enabling a comprehensive understanding of transmission dynamics and disease impact.

While these repositories include some patient-related information, such as the location of the infected host, the granularity of this data and the inclusion of demographic and clinical details are inconsistent. Additionally, the extent to which patient-related metadata is reported in published sequencing studies remains largely unexplored. Therefore, it is essential to assess the extent and quality of patient-related metadata reported in SARS-CoV-2 sequencing studies.

Moreover, there is limited linkage between published articles and sequence repositories, hindering the identification of relevant studies. Traditional search strategies based on keywords may miss relevant articles. To overcome these challenges, this study proposes the use of an automated classifier to identify relevant articles.

ObjectiveThis study aims to conduct a systematic and comprehensive scoping review, along with a bibliometric analysis, to assess the reporting of patient-related metadata in SARS-CoV-2 sequencing studies.

MethodsThe NIHs LitCovid collection will be used for the machine learning classification, while an independent search will be conducted in PubMed. Data extraction will be conducted using Covidence, and the extracted data will be synthesized and summarized to quantify the availability of patient metadata in the published literature of SARS-CoV-2 sequencing studies. For the bibliometric analysis, relevant data points, such as author affiliations, journal information, and citation metrics, will be extracted.

ResultsThe study will report findings on the extent and types of patient-related metadata reported in genomic viral sequencing studies of SARS-CoV-2. The scoping review will identify gaps in the reporting of patient metadata and make recommendations for improving the quality and consistency of reporting in this area. The bibliometric analysis will uncover trends and patterns in the reporting of patient-related metadata, such as differences in reporting based on study types or geographic regions. Co-occurrence networks of author keywords will also be presented to highlight frequent themes and their associations with patient metadata reporting.

ConclusionThis study will contribute to advancing knowledge in the field of genomic epidemiology by providing a comprehensive overview of the reporting of patient-related metadata in SARS-CoV-2 sequencing studies. The insights gained from this study may help improve the quality and consistency of reporting patient metadata, enhancing the utility of sequence metadata and facilitating future research on infectious diseases. The findings may also inform the development of machine learning methods to automatically extract patient-related information from sequencing studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.23292544">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.23292544" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.23292544">
        <p class="paperTitle">Epidemiological and health economic implications of symptom propagation in respiratory pathogens: A mathematical modelling investigation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.23292544" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.23292544" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Asplin, P.; Keeling, M. J.; Mancy, R.; Hill, E. M.</p>
        <p class="info">Score: 5.1, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.23292544' target='https://doi.org/10.1101/2023.07.12.23292544'> 10.1101/2023.07.12.23292544</a></p>
        <p class="abstract">BackgroundRespiratory pathogens inflict a substantial burden on public health and the economy. Although the severity of symptoms caused by these pathogens can vary from asymptomatic to fatal, the factors that determine symptom severity are not fully understood. Correlations in symptoms between infector-infectee pairs, for which evidence is accumulating, can generate large-scale clusters of severe infections that could be devastating to those most at risk, whilst also conceivably leading to chains of mild or asymptomatic infections that generate widespread immunity with minimal cost to public health. Although this effect could be harnessed to amplify the impact of interventions that reduce symptom severity, the mechanistic representation of symptom propagation within mathematical and health economic modelling of respiratory diseases is understudied.

Methods and FindingsWe propose a novel framework for incorporating different levels of symptom propagation into models of infectious disease transmission via a single parameter, . Varying  tunes the model from having no symptom propagation ( = 0, as typically assumed) to one where symptoms always propagate ( = 1). For parameters corresponding to three respiratory pathogens -- seasonal influenza, pandemic influenza and SARS-CoV-2 -- we explored how symptom propagation impacted the relative epidemiological and health-economic performance of three interventions, conceptualised as vaccines with different actions: symptom-attenuating (labelled SA), infection-blocking (IB) and infection-blocking admitting only mild breakthrough infections (IB MB).

In the absence of interventions, with fixed underlying epidemiological parameters, stronger symptom propagation increased the proportion of cases that were severe. For SA and IB MB, interventions were more effective at reducing prevalence (all infections and severe cases) for higher strengths of symptom propagation. For IB, symptom propagation had no impact on effectiveness, and for seasonal influenza this intervention type was more effective than SA at reducing severe infections for all levels of symptom propagation. For pandemic influenza and SARS-CoV-2, at low intervention uptake, SA was more effective than IB for all levels of symptom propagation; for high uptake, SA only became more effective under strong symptom propagation. Health economic assessments found that for SA-type interventions, the amount one could spend on control whilst maintaining a cost-effective intervention (termed threshold unit intervention cost) was very sensitive to the strength of symptom propagation.

ConclusionsOverall, the preferred intervention type depended on the combination of the strength of symptom propagation and uptake. Given the importance of determining robust public health responses, we highlight the need to gather further data on symptom propagation, with our modelling framework acting as a template for future analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.23292424">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.23292424" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.23292424">
        <p class="paperTitle">Marburg Virus Disease outbreaks, mathematical models, and disease parameters: a Systematic Review</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.23292424" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.23292424" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cuomo-Dannenburg, G.; McCain, K.; McCabe, R.; Unwin, H. J. T.; Doohan, P.; Nash, R. K.; Hicks, J. T.; Charniga, K.; Geismar, C.; Lambert, B.; Nikitin, D.; Skarp, J. E.; Wardle, J.; Pathogen Epidemiology Review Group,  ; Kont, M.; Bhatia, S.; Imai, N.; van Elsland, S. L.; Cori, A.; Morgenstern, C.</p>
        <p class="info">Score: 17.8, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.23292424' target='https://doi.org/10.1101/2023.07.10.23292424'> 10.1101/2023.07.10.23292424</a></p>
        <p class="abstract">BackgroundRecent Marburg virus disease (MVD) outbreaks in Equatorial Guinea and Tanzania highlighted the importance of better understanding this highly lethal infectious pathogen. Past epidemics of Ebola, COVID-19, and other pathogens have re-emphasised the usefulness of mathematical models in guiding public health responses during outbreaks.

MethodsWe conducted a systematic review, registered with PROSPERO (CRD42023393345) and reported according to PRISMA guidelines, of peer-reviewed papers reporting historical out-breaks, modelling studies and epidemiological parameters focused on MVD, including contextual information. We searched PubMed and Web of Science until 31st March 2023. Two reviewers evaluated all titles and abstracts, with consensus-based decision-making. To ensure agreement, 31% (13/42) of studies were double-extracted and a custom-designed quality assessment questionnaire was used to assess the risk of bias.

FindingsWe present detailed outbreak, model and parameter information on 970 reported cases and 818 deaths from MVD until 31 March 2023. Analysis of historical outbreaks and sero-prevalence estimates suggests the possibility of undetected MVD outbreaks, asymptomatic transmission and/or cross-reactivity with other pathogens. Only one study presented a mathematical model of MVD transmission. We estimate an unadjusted, pooled total random effect case fatality ratio for MVD of 61.9% (95% CI: 38.8-80.6%, I2=93%). We identify key epidemiological parameters relating to transmission and natural history for which there are few estimates.

InterpretationThis review provides a comprehensive overview of the epidemiology of MVD, identifying key knowledge gaps about this pathogen. The extensive collection of knowledge gathered here will be crucial in developing mathematical models for use in the early stages of future outbreaks of MVD. All data are published alongside this article with functionality to easily update the database as new data become available.

FundingMRC Centre for Global Infectious Disease Analysis

Research in ContextO_LIEvidence before this study
We searched Web of Science and PubMed up to 31 March 2023 using the search terms Marburg virus, epidemiology, outbreaks, models, transmissibility, severity, delays, risk factors, mutation rates and seroprevalence. We found five systematic reviews, all of which considered MVD alongside Ebola virus disease (EVD). One modelling study of Marburg virus disease (MVD) focused on animals, and not on computational models to understand past or project future disease transmission. One systematic review collated risk factors for transmission based on four MVD studies, but did not report attack rates due to missing underlying MVD estimates; another systematic review pooled estimates of MVD case fatality ratios (CFR): 53.8% (95% CI: 26.5-80.0%) and seroprevalence: 1.2% (95% CI: 0.5-2.0%). No systematic review covered transmission models of MVD, and the impact of public health and social measures is unknown.
C_LIO_LIAdded value of this study
We provide a comprehensive summary of the available, peer-reviewed literature of historical outbreaks, transmission models and parameters for MVD. Meta-analysis of existing estimates of CFRs, and our original estimates based on historical outbreak information, illustrate the severity of MVD with our pooled random effect estimated CFR of 61.9% (95% CI: 38.8-80.6%, I2=93%). We demonstrate the sparsity of evidence on MVD transmission and disease dynamics, particularly on transmissibility and natural history, which are key input parameters for computational models supporting outbreak response. Our work highlights key areas where further disease characterization is necessary.
C_LIO_LIImplications of all the available evidence
Previous outbreaks of infectious pathogens emphasized the usefulness of computational modelling in assessing epidemic trajectories and the impact of mitigation strategies. Our study provides necessary information for using mathematical models in future outbreaks of MVD, identifies uncertainties and knowledge gaps in MVD transmission and natural history, and highlights the severity of MVD.
C_LI</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
