<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>epidemiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>epidemiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Long COVID in a highly vaccinated population infected during a SARS-CoV-2 Omicron wave - Australia, 2022
Authors: Woldegiorgis, M.; Cadby, G.; Ngeh, S.; Korda, R.; Armstrong, P.; Maticevic, J.; Knight, P.; Jardine, A.; Bloomfield, L.; Effler, P.
Score: 2012.2, Published: 2023-08-09 DOI: 10.1101/2023.08.06.23293706
ObjectiveTo characterise Long COVID in a highly vaccinated population infected by Omicron. DesignFollow-up survey of persons testing positive for SARS-CoV-2 in Western Australia, 16 July-3 August 2022.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/epidemiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="epidemiology" />
<meta property="og:description" content="Long COVID in a highly vaccinated population infected during a SARS-CoV-2 Omicron wave - Australia, 2022
Authors: Woldegiorgis, M.; Cadby, G.; Ngeh, S.; Korda, R.; Armstrong, P.; Maticevic, J.; Knight, P.; Jardine, A.; Bloomfield, L.; Effler, P.
Score: 2012.2, Published: 2023-08-09 DOI: 10.1101/2023.08.06.23293706
ObjectiveTo characterise Long COVID in a highly vaccinated population infected by Omicron. DesignFollow-up survey of persons testing positive for SARS-CoV-2 in Western Australia, 16 July-3 August 2022." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/epidemiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-20T10:35:54+00:00" />
<meta property="article:modified_time" content="2023-08-20T10:35:54+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="epidemiology"/>
<meta name="twitter:description" content="Long COVID in a highly vaccinated population infected during a SARS-CoV-2 Omicron wave - Australia, 2022
Authors: Woldegiorgis, M.; Cadby, G.; Ngeh, S.; Korda, R.; Armstrong, P.; Maticevic, J.; Knight, P.; Jardine, A.; Bloomfield, L.; Effler, P.
Score: 2012.2, Published: 2023-08-09 DOI: 10.1101/2023.08.06.23293706
ObjectiveTo characterise Long COVID in a highly vaccinated population infected by Omicron. DesignFollow-up survey of persons testing positive for SARS-CoV-2 in Western Australia, 16 July-3 August 2022."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "epidemiology",
      "item": "https://trxiv.yorks0n.com/posts/epidemiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "epidemiology",
  "name": "epidemiology",
  "description": "Long COVID in a highly vaccinated population infected during a SARS-CoV-2 Omicron wave - Australia, 2022\nAuthors: Woldegiorgis, M.; Cadby, G.; Ngeh, S.; Korda, R.; Armstrong, P.; Maticevic, J.; Knight, P.; Jardine, A.; Bloomfield, L.; Effler, P.\nScore: 2012.2, Published: 2023-08-09 DOI: 10.1101/2023.08.06.23293706\nObjectiveTo characterise Long COVID in a highly vaccinated population infected by Omicron. DesignFollow-up survey of persons testing positive for SARS-CoV-2 in Western Australia, 16 July-3 August 2022.",
  "keywords": [
    
  ],
  "articleBody": " Long COVID in a highly vaccinated population infected during a SARS-CoV-2 Omicron wave - Australia, 2022\nAuthors: Woldegiorgis, M.; Cadby, G.; Ngeh, S.; Korda, R.; Armstrong, P.; Maticevic, J.; Knight, P.; Jardine, A.; Bloomfield, L.; Effler, P.\nScore: 2012.2, Published: 2023-08-09 DOI: 10.1101/2023.08.06.23293706\nObjectiveTo characterise Long COVID in a highly vaccinated population infected by Omicron. DesignFollow-up survey of persons testing positive for SARS-CoV-2 in Western Australia, 16 July-3 August 2022. SettingCommunity Participants22,744 persons with COVID-19 who had agreed to participate in research at the time of diagnosis were texted a survey link 90 days later; non-responders were telephoned. Post stratification weights were applied to responses from 11,697 (51.4%) participants, 94.0% of whom had received \u003e= 3 vaccine doses. Main outcome measuresPrevalence of Long COVID - defined as reporting new or ongoing COVID-19 illness-related symptoms or health issues 90 days post diagnosis; associated health care utilisation, reductions in work/study and risk factors were assessed using log-binomial regression. Results18.2% (n=2,130) of respondents met case definition for Long COVID. Female sex, being 50-69 years of age, pre-existing health issues, residing in a rural or remote area, and receiving fewer vaccine doses were significant independent predictors of Long COVID (p \u003c 0.05). Persons with Long COVID reported a median of 6 symptoms, most commonly fatigue (70.6%) and difficulty concentrating (59.6%); 38.2% consulted a GP and 1.6% reported hospitalisation in the month prior to the survey due to ongoing symptoms. Of 1,778 respondents with Long COVID who were working/studying before their COVID-19 diagnosis, 17.9% reported reducing/discontinuing work/study. Conclusion90 days post Omicron infection, almost 1 in 5 respondents reported Long COVID symptoms; 1 in 15 of all persons with COVID-19 sought healthcare for associated health concerns \u003e=2 months after the acute illness. Significance of the studyO_ST_ABSThe knownC_ST_ABSThe prevalence of Long COVID varies widely across studies conducted in diverse settings globally (range: 9%-81%). The newIn a highly vaccinated population (94% with \u003e=3 vaccine doses), almost 20% of persons infected with the SARS-CoV-2 Omicron variant reported symptoms consistent with Long COVID 90 days post diagnosis. Long COVID was associated with sustained negative impacts on work/study and a substantial utilisation of GP services 2-3 months after the acute illness; however, ED presentations and hospitalisations for Long COVID were rare. The implicationsGP clinics play a significant role in managing the burden of Long COVID in Australia.\nDiabetes following SARS-CoV-2 infection: Incidence, persistence, and implications of COVID-19 vaccination. A cohort study of fifteen million people.\nAuthors: Taylor, K.; Eastwood, S.; Walker, V.; Cezard, G.; Knight, R.; Al Arab, M.; Wei, Y.; Horne, E. M. F.; Teece, L.; Forbes, H.; Walker, A.; Fisher, L.; Massey, J.; Hopcroft, L. E. M.; Palmer, T.; Cuitun Coronado, J.; Ip, S.; Davy, S.; Dillingham, I.; Morton, C.; Greaves, F.; MacLeod, J.; Goldacre, B.; Wood, A.; Chaturvedi, N.; Sterne, J. A. C.; Denholm, R.; CONVALESCENCE Long-COVID study, ; Longitudinal Health and Wellbeing and Data and Connectivity UK COVID-19 National Core Studies, ; OpenSAFELY collaborative, Score: 828.9, Published: 2023-08-09 DOI: 10.1101/2023.08.07.23293778\nBackgroundType 2 diabetes (T2DM) incidence is increased after diagnosis of COVID-19. The impact of vaccination on this increase, for how long it persists, and the effect of COVID-19 on other types of diabetes remain unclear. MethodsWith NHS England approval, we studied diabetes incidence following COVID-19 diagnosis in pre-vaccination (N=15,211,471, January 2020-December 2021), vaccinated (N =11,822,640), and unvaccinated (N=2,851,183) cohorts (June-December 2021), using linked electronic health records. We estimated adjusted hazard ratios (aHRs) comparing diabetes incidence post-COVID-19 diagnosis with incidence before or without diagnosis up to 102 weeks post-diagnosis. Results were stratified by COVID-19 severity (hospitalised/non-hospitalised) and diabetes type. FindingsIn the pre-vaccination cohort, aHRS for T2DM incidence after COVID-19 (compared to before or without diagnosis) declined from 3.01 (95% CI: 2.76,3.28) in weeks 1-4 to 1.24 (1.12,1.38) in weeks 53-102. aHRS were higher in unvaccinated than vaccinated people (4.86 (3.69,6.41)) versus 1.42 (1.24,1.62) in weeks 1-4) and for hospitalised COVID-19 (pre-vaccination cohort 21.1 (18.8,23.7) in weeks 1-4 declining to 2.04 (1.65,2.51) in weeks 52-102), than non-hospitalised COVID-19 (1.45 (1.27,1.64) in weeks 1-4, 1.10 (0.98,1.23) in weeks 52-102). T2DM persisted for 4 months after COVID-19 for [~]73% of those diagnosed. Patterns were similar for Type 1 diabetes, though excess incidence did not persist beyond a year post-COVID-19. InterpretationElevated T2DM incidence after COVID-19 is greater, and persists longer, in hospitalised than non-hospitalised people. It is markedly less apparent post-vaccination. Testing for T2DM after severe COVID-19 and promotion of vaccination are important tools in addressing this public health problem. Research in contextO_ST_ABSEvidence before this studyC_ST_ABSWe searched PubMed for population-based observational studies published between December 1st 2019 and July 12th 2023 examining associations between SARS-CoV-2 infection or COVID-19 diagnosis (search string: SARS-CoV-2 or COVID* or coronavirus*) and subsequent incident diabetes (search term: diabetes). Of nineteen relevant studies; eight had a composite outcome of diabetes types, six stratified by diabetes type and five pertained to type-1-diabetes (T1DM) only. We did not identify any studies relating to gestational or other types of diabetes. Eleven studies were from the US, three from the UK, two from Germany, one from Canada, one from Denmark and one from South Korea. Most studies described cumulative relative risks (for infection versus no infection) one to two years post-SARS-CoV-2 infection of 1.2 to 2.6, though four studies found no associations with T1DM after the post-acute period. All studies lacked the power to compare diabetes relative risk by type, severity, and vaccination status in population subgroups. One study examined relative risks by vaccination status, but this used a composite outcome of diabetes and hyperlipidaemia and was conducted in a predominantly white male population. Two studies of T1DM found no evidence of elevated risk beyond 30 days after COVID-19 diagnosis, whilst two reported elevated risks at six months. Two studies of type 2 diabetes (T2DM) examined relative risks by time period post-infection: one study of US insurance claims reported a persistent association six months post-infection, whereas a large UK population-based study reported no associations after 12 weeks. However, the latter study used only primary care data, therefore COVID-19 cases were likely to have been under-ascertained. No large studies have investigated the persistence of diabetes diagnosed following COVID-19; key to elucidating the role of stress/steroid-induced hyperglycaemia. Added value of this studyThis study, which is the largest to address the question to date, analysed linked primary and secondary care health records with SARS-CoV-2 testing and COVID-19 vaccination data for 15 million people living in England. This enabled us to compare the elevation in diabetes incidence after COVID-19 diagnosis by diabetes type, COVID-19 severity and vaccination status, overall and in population subgroups. Importantly, excess diabetes incidence by time period since infection could also be quantified. Since healthcare in the UK is universal and free-at-the-point-of-delivery, almost the entire population is registered with primary care. Therefore the findings are likely to be generalisable. We found that, before availability of COVID-19 vaccination, a COVID-19 diagnosis (vs. no diagnosis) was associated with increased T2DM incidence which remained elevated by approximately 30% beyond one year after diagnosis. Though still present (with around 30% excess incidence at eight weeks), these associations were substantially attenuated in unvaccinated compared with vaccinated people. Excess incidence was greater in people hospitalised with COVID-19 than those who were not hospitalised after diagnosis. T1DM incidence was elevated up to, but not beyond, a year post COVID-19. Around 73% of people diagnosed with incident T2DM after COVID-19 still had evidence of diabetes four months after infection. Implications of all the available evidenceThere is a 30-50% elevated T2DM incidence post-COVID-19, but we report the novel finding that there is elevated incidence beyond one-year post-diagnosis. Elevated T1DM incidence did not appear to persist beyond a year, which may explain why previous studies disagree. For the first time in a general-population dataset, we demonstrate that COVID-19 vaccination reduces, but does not entirely ameliorate, excess diabetes incidence after COVID-19. This supports a policy of universal vaccination and suggests that other public health activities, such as enhanced diabetes screening after severe COVID-19, may be warranted, particularly in unvaccinated people.\nInfluence of Prior SARS-CoV-2 Infection on COVID-19 Severity: Evidence from the National COVID Cohort Collaborative\nAuthors: Hendrix, N.; Sidky, H.; Sahner, D.; N3C Consortium, Score: 560.5, Published: 2023-08-06 DOI: 10.1101/2023.08.03.23293612\nBackgroundAs SARS-CoV-2 has transitioned from a pandemic to endemic disease, the majority of new infections have been among previously infected individuals. To manage the risks and benefits of ongoing COVID-19 policies, it is important to understand whether prior infection modifies the severity of subsequent infections. MethodsWe used data from first and second COVID-19 episodes in the National COVID Cohort Collaborative (N3C), a collection of health systems who provide de-identified electronic health records for research purposes. Our analysis was a sequential series of nested trial emulations. In the first of two analytic stages, we created a month-specific model of the probability of prior infection for each individual. In the second stage, we used an ordinal logistic regression with inverse probability weights calculated in the first stage to simulate a series of monthly trials comparing severity between the cohorts of first and second infections. In addition to cohort-wide effect estimates, we also conducted analyses among race/ethnicity, sex, and age subgroups. ResultsFrom an initial cohort of 7,446,481 combined first and second infections, we identified a cohort of 2,227,484 infections, among which 7.6% were second infections. Ninety-four percent of patients with two recorded infections experienced mild disease for both. The overall odds ratio (OR) for more severe disease with prior infection was 1.06 (95% confidence interval [CI]: 1.03 - 1.10). Monthly point estimates of the OR ranged from 0.56 (95% CI: 0.37 - 0.84) in October 2020 to 1.64 (95% CI: 1.33 - 2.00) in February 2023. In most subgroups, the effect of prior infection was significant. In 8 out of 10 subgroups, the maximum monthly OR occurred after the minimum monthly OR, suggesting that protection has waned throughout the pandemic. ConclusionOverall, prior infection was associated with a significant slightly elevated risk of severe disease. This effect varied month to month. As the pandemic proceeded, the effect of prior infection tended to evolve from generally protective during the pre-Omicron era to unprotective during the Omicron era. This points to the need for continued strategies to avert and minimize the harms of COVID-19, rather than relying upon immunity acquired through previous infection. Key PointsO_ST_ABSQuestionC_ST_ABSDoes prior infection with SARS-CoV-2 affect the severity of subsequent COVID-19 episodes? FindingsWe observed a mild protective effect of prior infection during the early and mid-stages of the pandemic that waned after the rise of the Omicron variants, ultimately resulting in loss of protection or a tendency toward more severe second infections. MeaningPrior infection alone is likely not enough to avert the worst public health harms of endemic SARS-CoV-2. Interventions to avoid infection and reduce the severity of COVID-19 will still be important in the post-pandemic era.\nBlood protein levels predict leading incident diseases and mortality in UK Biobank\nAuthors: Gadd, D. A.; Hillary, R. F.; Kuncheva, Z.; Mangelis, T.; Cheng, Y.; Dissanayake, M.; Admanit, R.; Gagnon, J.; Lin, T.; Ferber, K.; Runz, H.; Biogen Biobank Team, ; Marioni, R. E.; Foley, C. N.; Sun, B. B.\nScore: 36.6, Published: 2023-08-04 DOI: 10.1101/2023.05.01.23288879\nThe circulating proteome offers insights into the biological pathways that underlie disease. Here, we test relationships between 1,468 Olink protein levels and the incidence of 23 age-related diseases and mortality, over 16 years of electronic health linkage in the UK Biobank (N=47,600). We report 3,201 associations between 961 protein levels and 21 incident outcomes, identifying proteomic indicators of multiple morbidities. Next, protein-based scores (ProteinScores) are developed using penalised Cox regression. When applied to test sets, six ProteinScores improve Area Under the Curve (AUC) estimates for the 10-year onset of incident outcomes beyond age, sex and a comprehensive set of 24 lifestyle factors, clinically-relevant biomarkers and physical measures. Furthermore, the ProteinScore for type 2 diabetes outperformed a polygenic risk score, a metabolomic score and HbA1c - a clinical marker used to monitor and diagnose type 2 diabetes. These data characterise early proteomic contributions to major age-related disease and demonstrate the value of the plasma proteome for risk stratification.\nAssociation of Life's Essential 8 with Cardiovascular Events and Mortality: The Cardiovascular Disease Lifetime Risk Pooling Project (LRPP)\nAuthors: Ning, H.; Allen, N.; Lloyd-Jones, D. M.; Perak, A. M.; Siddique, J.; Wilkins, J. T.\nScore: 14.3, Published: 2023-08-14 DOI: 10.1101/2023.08.08.23293855\nBackgroundThe American Heart Association (AHA) recently launched updated cardiovascular health (CVH) metrics, termed Lifes Essential 8 (LE8). Compared to Lifes Simple 7 (LS7), the new approach added sleep health as an eighth metric and updated the remaining 7 metrics. Whether the updated LE8 score outperforms the original LS7 score in predicting cardiovascular disease (CVD) is not known. We examined the association of LE8 scores with CVD, subtype CVD events and all-cause mortality. MethodsWe pooled individual-level data from 6 contemporary US-based cohorts from the Cardiovascular Lifetime Risk Pooling Project (LRPP). Total LE8 score (0-100 points), LE8 score without sleep (0-100 points), as well as prior LS7 scores (0-14 points), were calculated separately. We used multivariable-adjusted Cox models to evaluate the association of LE8 with CVD, CVD subtypes, and all-cause mortality among younger, middle, and older aged adult participants. Reclassification was defined based on the concordant/discordant categories of LS7 and LE8 scores quartile rankings across age groups. ResultsOur sample consisted of 32,896 US adults (7836 [23.8%] Black; 14941 [45.4%] men) followed for 642,000 person-years; of whom 9,391 developed CVD events. Each 10-point higher overall LE8 score was associated with 23-40% lower CVD risk across age groups. Reclassification of CVH from LS7 to LE8 was related to heath behaviors as well as health factors and was significantly associated with CVD risk. ConclusionsThese findings support the improved utility of the LE8 algorithm for assessing overall cardiovascular health and future CVD risk.\nOne Health Genomic Surveillance of Avian and Human Influenza A Viruses Through Environmental Wastewater Monitoring\nAuthors: Lee, A. J.; Carson, S.; Reyne, M. I.; Marshall, A.; Moody, D.; Allen, D.; Allingham, P.; Levickas, A.; Fitzgerald, A.; Bell, S. H.; Lock, J.; Lock, J. D.; McSparron, C.; Nejad, B. F.; Courtney, D. G.; Einarsson, G. G.; McKenna, J. P.; Fairley, D. J.; Curran, T.; McKinley, J. M.; Gilpin, D. F.; Lemon, K.; McGrath, J. W.; Bamford, C. G.\nScore: 12.7, Published: 2023-08-13 DOI: 10.1101/2023.08.08.23293833\nBackgroundInfluenza A viruses (IAV) are significant pathogens of humans and other animals. Although endemic in humans and birds, novel IAV strains can emerge, jump species, and cause epidemics, like the latest variant of H5N1. Wastewater-based epidemiology (WBE) has very recently been shown to detect human IAV but whether it can detect avian-origin IAV, and if whole genome sequencing (WGS) can be used to discriminate circulating strains of IAV in wastewater remains unknown. MethodsUsing a pan-IAV RT-qPCR assay, six wastewater treatment works (WWTWs) across Northern Ireland (NI), were screened from August to December 2022. A WGS approach using Oxford Nanopore technology was employed to sequence positive samples. Phylogenetic analysis of sequences relative to currently circulating human and avian IAVs was performed. FindingsWe detected a dynamic IAV signal in wastewater from September 2022 onwards across NI. \"Meta\" whole genome sequences were generated displaying homology to both human and avian IAV strains. The relative proportion of human versus avian-origin IAV reads differed across time and sample site. A diversity in subtypes and lineages was detected (e.g. H1N1, H3N2, and several avian). Avian segment 8 related to those found in recent H5N1 clade 2.3.4.4b was identified. InterpretationWBE affords a means to monitor circulating human and avian IAV strains and provide crucial genetic information. As such WBE can provide rapid, cost-effective, year-round \"one-health\" IAV surveillance to help control epidemic and pandemic threats. FundingThis study was funded by the Department of Health for Northern Ireland as part of the Northern Ireland Wastewater Surveillance Programme. HighlightsO_LIDynamic IAV RT-qPCR signal in wastewater detected across NI. C_LIO_LINanopore-based WGS reveals presence of both human and avian IAVs in wastewater. C_LIO_LIAvian IAV sequence similarity to gull-associated H13/H16 and recent H5N1 isolates. C_LIO_LICo-detection of distinct clades of human H1N1 and H3N2 subtypes. C_LI Author SummaryInfluenza A virus (IAV) is a major pathogen of humans and other animals and causes regular epidemics and devastating pandemics. Recently, a novel variant of highly-pathogenic H5N1 avian influenza has emerged spreading across the world killing millions of birds and infecting mammals, enhancing its pandemic potential. Strengthening global surveillance systems for human and animal IAV is thus a major priority. Wastewater-based epidemiology (WBE) has been applied to track SARS-CoV-2 and IAV in humans but whether this approach could work for avian IAV is not known. Here, we develop a \"one-health\" method to survey pan-IAV levels and genetically characterise the viruses. Through this we highlight co-detection of human and avian IAVs in wastewater, with homology to recent H5N1 isolates. Our work demonstrates the potential for WBE to help defend against not only human infections but emerging, zoonotic IAVs of pandemic potential. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=133 SRC=\"FIGDIR/small/23293833v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (26K): org.highwire.dtl.DTLVardef@13bd526org.highwire.dtl.DTLVardef@126db74org.highwire.dtl.DTLVardef@11b4dborg.highwire.dtl.DTLVardef@78a64f_HPS_FORMAT_FIGEXP M_FIG C_FIG\nProteomic prediction of common and rare diseases\nAuthors: Carrasco-Zanini, J.; Pietzner, M.; Davitte, J.; Surendran, P.; Croteau-Chonka, D. C.; Robins, C.; Torralbo, A.; Tomlinson, C.; Fitzpatrick, N.; Ytsma, C.; Kanno, T.; Gade, S.; Freitag, D.; Ziebell, F.; Denaxas, S.; Betts, J. C.; Wareham, N. J.; Hemingway, H.; Scott, R. A.; Langenberg, C.\nScore: 43.2, Published: 2023-07-23 DOI: 10.1101/2023.07.18.23292811\nBackgroundFor many diseases there are delays in diagnosis due to a lack of objective biomarkers for disease onset. Whether measuring thousands of proteins offers predictive information across a wide range of diseases is unknown. MethodsIn 41,931 individuals from the UK Biobank Pharma Proteomics Project (UKB-PPP), we integrated [~]3000 plasma proteins with clinical information to derive sparse prediction models for the 10-year incidence of 218 common and rare diseases (81 - 6038 cases). We compared prediction models based on proteins with a) basic clinical information alone, b) basic clinical information + 37 clinical biomarkers, and c) genome-wide polygenic risk scores. ResultsFor 67 pathologically diverse diseases, a model including as few as 5 to 20 proteins was superior to clinical models (median delta C-index = 0.07; range = 0.02 - 0.31) and to clinical models with biomarkers for 52 diseases. In multiple myeloma, for example, a set of 5 proteins significantly improved prediction over basic clinical information (delta C-index = 0.25 (95% confidence interval 0.20 - 0.29)). At a 5% false positive rate (FPR), proteomic prediction (5 proteins) identified individuals at high risk of multiple myeloma (detection rate (DR) = 50%), non-Hodgkin lymphoma (DR = 55%) and motor neuron disease (DR = 29%). At a 20% FPR, proteomic prediction identified individuals at high-risk for pulmonary fibrosis (DR= 80%) and dilated cardiomyopathy (DR = 75%). ConclusionsSparse plasma protein signatures offer novel, clinically useful prediction of common and rare diseases, through disease-specific proteins and protein predictors shared across multiple diseases. (Funded by Medical Research Council, NIHR, Wellcome Trust.)\nComparing methods to predict baseline mortality for excess mortality calculations\nAuthors: Ferenci, T.\nScore: 10.2, Published: 2023-08-14 DOI: 10.1101/2022.07.18.22277746\nBackground: The World Health Organizations excess mortality estimates presented in May 2022 stirred controversy, one point of which was the high estimate for Germany, later attributed to the used spline-model. This paper aims to reproduce the problem using synthetic datasets, thus allowing the investigation of its sensitivity to parameters, both of the mortality curve and of the used method, thereby shedding light on the conditions that gave rise to this error and its possible remedies. Methods: A negative binomial model was used accounting for long-term change, seasonality and flu seasons and heat waves. Simulated mortality curves from this model were then analysed with simple methods (mean, linear trend), with the WHOs method and with the method of Acosta and Irizarry. Results: The performance of the WHOs method with its original parametrization is indeed very poor, however it can be profoundly improved by a better choice of parameters. The Acosta-Irizarry method outperformed WHOs method despite being also based on splines, but it was also dependent on its parameters. Linear extrapolation could produce very good results, but is highly dependent on the choice of the starting year, while average was the worst in almost all cases. Conclusions: Splines are not inherently unsuitable for predicting baseline mortality, but care should be taken, in particular, results suggest that the key issue is the rigidity of the splines. Model diagnostics must be performed before accepting any result, and used methods should be validated. Further research is warranted to understand how these results can be generalized to other scenarios.\nChallenges in using data on fathers/partners to study prenatal exposures and offspring health\nAuthors: Easey, K.; Gkatzionis, A.; Millard, L. A. C.; Tilling, K.; Lawlor, D. A.; Sharp, G.\nScore: 3.2, Published: 2023-08-13 DOI: 10.1101/2023.08.08.23293816\nIntroductionPaternal exposures (and other non-maternal factors) around pregnancy could have important effects on offspring health. One challenge in research of paternal effects is that study samples with data on partners are usually a subgroup of those with data on mothers, which could introduce selection bias and limit generalisability of the findings. Here, we use maternal and father/partner data on prenatal behaviours to explore the extent to which selection in partner analyses might bias findings. MethodsWe characterise the availability of data on father/partner and mother health behaviours (smoking, alcohol consumption, caffeine consumption and physical activity) in the prenatal period from three UK cohort studies: the Avon Longitudinal Study of Parents and Children (ALSPAC), Born in Bradford (BiB) and the Millennium Cohort Study (MCS). We assess the extent of sample selection in these cohorts by comparing the characteristics of families where the father/partner does and does not participate. Using the association of parental smoking during pregnancy and child birthweight as an exemplar, we used simulations to explore the extent to which missing father/partner data may bias estimates. ResultsIn all three cohorts, data on prenatal health behaviours of fathers/partners were less detailed and collected at fewer timepoints than data on prenatal health behaviours of mothers. Partners of mothers who had a lower socioeconomic position were less likely to participate. Estimates of the association between maternal smoking and offspring birthweight were similar in samples with and without participating partners in all three cohorts. In simulations based on ALSPAC data, there was little evidence of selection bias in associations of maternal smoking with birthweight, and although bias was observed for father/partner smoking, its magnitude was relatively small. DiscussionUsing real and simulated data, we show that bias due to selected recruitment of partners into ALSPAC, BiB and MCS had a relatively small impact on estimates of the effects of maternal and partner smoking on offspring birthweight. In other applications, the impact of selection bias will depend on both the analysis model and the selection mechanism. We have shown how to use a simulation study to assess that and recommend that applied researchers working with partner data use simulations and other sensitivity analyses to assess the robustness of their results.\nAn epidemiological study of season of birth, mental health, and neuroimaging in the UK Biobank\nAuthors: Viejo-Romero, M.; Whalley, H. C.; Shen, X.; Stolicyn, A.; de Nooij, L.; Smith, D. J.; Howard, D. M.\nScore: 3.0, Published: 2023-08-13 DOI: 10.1101/2023.08.09.23293866\nEnvironmental exposures during the perinatal period are known to have a long-term effect on adult physical and mental health. One such influential environmental exposure is the time of year of birth which affects the amount of daylight, nutrients, and viral load that an individual is exposed to in the key developmental period. Here we investigate associations between season of birth (seasonality), four mental health traits (n=135,541) and multi-modal neuroimaging measures (n=33,815) within the UK Biobank. Summer births were associated with probable recurrent Major Depressive Disorder ({beta}=0.024, pcorr=0.048), greater mean cortical thickness in temporal and occipital lobes and in the middle temporal, fusiform, superior temporal, and lingual gyri regions ({beta} range=0.013 - 0.020, pcorr\u003c0.05). Winter births were associated with greater white matter integrity globally, in the association fibers, thalamic radiations, and six individual tracts ({beta} range=-0.010 to -0.021, pcorr\u003c0.05). Results of sensitivity analyses that adjusted for birth weight were similar, with additional associations found between winter birth and frontal, occipital and cingulate lobe surface areas, as well as fractional anisotropy in the forceps minor. Sensitivity analyses also revealed an additional association between summer birth and greater cingulate thickness. Overall, results suggest that seasonality affects brain structure in later life and may have a role in lifetime recurrent Major Depressive Disorder. The small effect sizes observed here warrant further research to validate the results in the context of different latitudes and co-examine genetic and epigenetic measures to potentially reveal informative biological pathways.\n",
  "wordCount" : "4074",
  "inLanguage": "en",
  "datePublished": "2023-08-20T10:35:54Z",
  "dateModified": "2023-08-20T10:35:54Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/epidemiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      epidemiology
    </h1>
    <div class="post-meta"><span>updated on August 20, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.23293706">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.23293706" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.23293706">
        <p class="paperTitle">Long COVID in a highly vaccinated population infected during a SARS-CoV-2 Omicron wave - Australia, 2022</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.23293706" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.23293706" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Woldegiorgis, M.; Cadby, G.; Ngeh, S.; Korda, R.; Armstrong, P.; Maticevic, J.; Knight, P.; Jardine, A.; Bloomfield, L.; Effler, P.</p>
        <p class="info">Score: 2012.2, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.23293706' target='https://doi.org/10.1101/2023.08.06.23293706'> 10.1101/2023.08.06.23293706</a></p>
        <p class="abstract">ObjectiveTo characterise Long COVID in a highly vaccinated population infected by Omicron.

DesignFollow-up survey of persons testing positive for SARS-CoV-2 in Western Australia, 16 July-3 August 2022.

SettingCommunity

Participants22,744 persons with COVID-19 who had agreed to participate in research at the time of diagnosis were texted a survey link 90 days later; non-responders were telephoned. Post stratification weights were applied to responses from 11,697 (51.4%) participants, 94.0% of whom had received &gt;= 3 vaccine doses.

Main outcome measuresPrevalence of  Long COVID - defined as reporting new or ongoing COVID-19 illness-related symptoms or health issues 90 days post diagnosis; associated health care utilisation, reductions in work/study and risk factors were assessed using log-binomial regression.

Results18.2% (n=2,130) of respondents met case definition for Long COVID. Female sex, being 50-69 years of age, pre-existing health issues, residing in a rural or remote area, and receiving fewer vaccine doses were significant independent predictors of Long COVID (p &lt; 0.05). Persons with Long COVID reported a median of 6 symptoms, most commonly fatigue (70.6%) and difficulty concentrating (59.6%); 38.2% consulted a GP and 1.6% reported hospitalisation in the month prior to the survey due to ongoing symptoms. Of 1,778 respondents with Long COVID who were working/studying before their COVID-19 diagnosis, 17.9% reported reducing/discontinuing work/study.

Conclusion90 days post Omicron infection, almost 1 in 5 respondents reported Long COVID symptoms; 1 in 15 of all persons with COVID-19 sought healthcare for associated health concerns &gt;=2 months after the acute illness.

Significance of the studyO_ST_ABSThe knownC_ST_ABSThe prevalence of Long COVID varies widely across studies conducted in diverse settings globally (range: 9%-81%).

The newIn a highly vaccinated population (94% with &gt;=3 vaccine doses), almost 20% of persons infected with the SARS-CoV-2 Omicron variant reported symptoms consistent with Long COVID 90 days post diagnosis. Long COVID was associated with sustained negative impacts on work/study and a substantial utilisation of GP services 2-3 months after the acute illness; however, ED presentations and hospitalisations for Long COVID were rare.

The implicationsGP clinics play a significant role in managing the burden of Long COVID in Australia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.23293778">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.23293778" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.23293778">
        <p class="paperTitle">Diabetes following SARS-CoV-2 infection: Incidence, persistence, and implications of COVID-19 vaccination. A cohort study of fifteen million people.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.23293778" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.23293778" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taylor, K.; Eastwood, S.; Walker, V.; Cezard, G.; Knight, R.; Al Arab, M.; Wei, Y.; Horne, E. M. F.; Teece, L.; Forbes, H.; Walker, A.; Fisher, L.; Massey, J.; Hopcroft, L. E. M.; Palmer, T.; Cuitun Coronado, J.; Ip, S.; Davy, S.; Dillingham, I.; Morton, C.; Greaves, F.; MacLeod, J.; Goldacre, B.; Wood, A.; Chaturvedi, N.; Sterne, J. A. C.; Denholm, R.; CONVALESCENCE Long-COVID study,  ; Longitudinal Health and Wellbeing and Data and Connectivity UK COVID-19 National Core Studies,  ; OpenSAFELY collaborative,  </p>
        <p class="info">Score: 828.9, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.23293778' target='https://doi.org/10.1101/2023.08.07.23293778'> 10.1101/2023.08.07.23293778</a></p>
        <p class="abstract">BackgroundType 2 diabetes (T2DM) incidence is increased after diagnosis of COVID-19. The impact of vaccination on this increase, for how long it persists, and the effect of COVID-19 on other types of diabetes remain unclear.

MethodsWith NHS England approval, we studied diabetes incidence following COVID-19 diagnosis in pre-vaccination (N=15,211,471, January 2020-December 2021), vaccinated (N =11,822,640), and unvaccinated (N=2,851,183) cohorts (June-December 2021), using linked electronic health records. We estimated adjusted hazard ratios (aHRs) comparing diabetes incidence post-COVID-19 diagnosis with incidence before or without diagnosis up to 102 weeks post-diagnosis. Results were stratified by COVID-19 severity (hospitalised/non-hospitalised) and diabetes type.

FindingsIn the pre-vaccination cohort, aHRS for T2DM incidence after COVID-19 (compared to before or without diagnosis) declined from 3.01 (95% CI: 2.76,3.28) in weeks 1-4 to 1.24 (1.12,1.38) in weeks 53-102. aHRS were higher in unvaccinated than vaccinated people (4.86 (3.69,6.41)) versus 1.42 (1.24,1.62) in weeks 1-4) and for hospitalised COVID-19 (pre-vaccination cohort 21.1 (18.8,23.7) in weeks 1-4 declining to 2.04 (1.65,2.51) in weeks 52-102), than non-hospitalised COVID-19 (1.45 (1.27,1.64) in weeks 1-4, 1.10 (0.98,1.23) in weeks 52-102). T2DM persisted for 4 months after COVID-19 for [~]73% of those diagnosed. Patterns were similar for Type 1 diabetes, though excess incidence did not persist beyond a year post-COVID-19.

InterpretationElevated T2DM incidence after COVID-19 is greater, and persists longer, in hospitalised than non-hospitalised people. It is markedly less apparent post-vaccination. Testing for T2DM after severe COVID-19 and promotion of vaccination are important tools in addressing this public health problem.

Research in contextO_ST_ABSEvidence before this studyC_ST_ABSWe searched PubMed for population-based observational studies published between December 1st 2019 and July 12th 2023 examining associations between SARS-CoV-2 infection or COVID-19 diagnosis (search string: SARS-CoV-2 or COVID* or coronavirus*) and subsequent incident diabetes (search term: diabetes). Of nineteen relevant studies; eight had a composite outcome of diabetes types, six stratified by diabetes type and five pertained to type-1-diabetes (T1DM) only. We did not identify any studies relating to gestational or other types of diabetes. Eleven studies were from the US, three from the UK, two from Germany, one from Canada, one from Denmark and one from South Korea.

Most studies described cumulative relative risks (for infection versus no infection) one to two years post-SARS-CoV-2 infection of 1.2 to 2.6, though four studies found no associations with T1DM after the post-acute period. All studies lacked the power to compare diabetes relative risk by type, severity, and vaccination status in population subgroups. One study examined relative risks by vaccination status, but this used a composite outcome of diabetes and hyperlipidaemia and was conducted in a predominantly white male population.

Two studies of T1DM found no evidence of elevated risk beyond 30 days after COVID-19 diagnosis, whilst two reported elevated risks at six months. Two studies of type 2 diabetes (T2DM) examined relative risks by time period post-infection: one study of US insurance claims reported a persistent association six months post-infection, whereas a large UK population-based study reported no associations after 12 weeks. However, the latter study used only primary care data, therefore COVID-19 cases were likely to have been under-ascertained.

No large studies have investigated the persistence of diabetes diagnosed following COVID-19; key to elucidating the role of stress/steroid-induced hyperglycaemia.

Added value of this studyThis study, which is the largest to address the question to date, analysed linked primary and secondary care health records with SARS-CoV-2 testing and COVID-19 vaccination data for 15 million people living in England. This enabled us to compare the elevation in diabetes incidence after COVID-19 diagnosis by diabetes type, COVID-19 severity and vaccination status, overall and in population subgroups. Importantly, excess diabetes incidence by time period since infection could also be quantified. Since healthcare in the UK is universal and free-at-the-point-of-delivery, almost the entire population is registered with primary care. Therefore the findings are likely to be generalisable.

We found that, before availability of COVID-19 vaccination, a COVID-19 diagnosis (vs. no diagnosis) was associated with increased T2DM incidence which remained elevated by approximately 30% beyond one year after diagnosis. Though still present (with around 30% excess incidence at eight weeks), these associations were substantially attenuated in unvaccinated compared with vaccinated people. Excess incidence was greater in people hospitalised with COVID-19 than those who were not hospitalised after diagnosis. T1DM incidence was elevated up to, but not beyond, a year post COVID-19. Around 73% of people diagnosed with incident T2DM after COVID-19 still had evidence of diabetes four months after infection.

Implications of all the available evidenceThere is a 30-50% elevated T2DM incidence post-COVID-19, but we report the novel finding that there is elevated incidence beyond one-year post-diagnosis. Elevated T1DM incidence did not appear to persist beyond a year, which may explain why previous studies disagree. For the first time in a general-population dataset, we demonstrate that COVID-19 vaccination reduces, but does not entirely ameliorate, excess diabetes incidence after COVID-19. This supports a policy of universal vaccination and suggests that other public health activities, such as enhanced diabetes screening after severe COVID-19, may be warranted, particularly in unvaccinated people.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.23293612">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.23293612" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.23293612">
        <p class="paperTitle">Influence of Prior SARS-CoV-2 Infection on COVID-19 Severity: Evidence from the National COVID Cohort Collaborative</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.23293612" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.23293612" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hendrix, N.; Sidky, H.; Sahner, D.; N3C Consortium,  </p>
        <p class="info">Score: 560.5, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.23293612' target='https://doi.org/10.1101/2023.08.03.23293612'> 10.1101/2023.08.03.23293612</a></p>
        <p class="abstract">BackgroundAs SARS-CoV-2 has transitioned from a pandemic to endemic disease, the majority of new infections have been among previously infected individuals. To manage the risks and benefits of ongoing COVID-19 policies, it is important to understand whether prior infection modifies the severity of subsequent infections.

MethodsWe used data from first and second COVID-19 episodes in the National COVID Cohort Collaborative (N3C), a collection of health systems who provide de-identified electronic health records for research purposes. Our analysis was a sequential series of nested trial emulations. In the first of two analytic stages, we created a month-specific model of the probability of prior infection for each individual. In the second stage, we used an ordinal logistic regression with inverse probability weights calculated in the first stage to simulate a series of monthly trials comparing severity between the cohorts of first and second infections. In addition to cohort-wide effect estimates, we also conducted analyses among race/ethnicity, sex, and age subgroups.

ResultsFrom an initial cohort of 7,446,481 combined first and second infections, we identified a cohort of 2,227,484 infections, among which 7.6% were second infections. Ninety-four percent of patients with two recorded infections experienced mild disease for both. The overall odds ratio (OR) for more severe disease with prior infection was 1.06 (95% confidence interval [CI]: 1.03 - 1.10). Monthly point estimates of the OR ranged from 0.56 (95% CI: 0.37 - 0.84) in October 2020 to 1.64 (95% CI: 1.33 - 2.00) in February 2023. In most subgroups, the effect of prior infection was significant. In 8 out of 10 subgroups, the maximum monthly OR occurred after the minimum monthly OR, suggesting that protection has waned throughout the pandemic.

ConclusionOverall, prior infection was associated with a significant slightly elevated risk of severe disease. This effect varied month to month. As the pandemic proceeded, the effect of prior infection tended to evolve from generally protective during the pre-Omicron era to unprotective during the Omicron era. This points to the need for continued strategies to avert and minimize the harms of COVID-19, rather than relying upon immunity acquired through previous infection.

Key PointsO_ST_ABSQuestionC_ST_ABSDoes prior infection with SARS-CoV-2 affect the severity of subsequent COVID-19 episodes?

FindingsWe observed a mild protective effect of prior infection during the early and mid-stages of the pandemic that waned after the rise of the Omicron variants, ultimately resulting in loss of protection or a tendency toward more severe second infections.

MeaningPrior infection alone is likely not enough to avert the worst public health harms of endemic SARS-CoV-2. Interventions to avoid infection and reduce the severity of COVID-19 will still be important in the post-pandemic era.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.01.23288879">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.01.23288879" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.01.23288879">
        <p class="paperTitle">Blood protein levels predict leading incident diseases and mortality in UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.01.23288879" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.01.23288879" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gadd, D. A.; Hillary, R. F.; Kuncheva, Z.; Mangelis, T.; Cheng, Y.; Dissanayake, M.; Admanit, R.; Gagnon, J.; Lin, T.; Ferber, K.; Runz, H.; Biogen Biobank Team,  ; Marioni, R. E.; Foley, C. N.; Sun, B. B.</p>
        <p class="info">Score: 36.6, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.01.23288879' target='https://doi.org/10.1101/2023.05.01.23288879'> 10.1101/2023.05.01.23288879</a></p>
        <p class="abstract">The circulating proteome offers insights into the biological pathways that underlie disease. Here, we test relationships between 1,468 Olink protein levels and the incidence of 23 age-related diseases and mortality, over 16 years of electronic health linkage in the UK Biobank (N=47,600). We report 3,201 associations between 961 protein levels and 21 incident outcomes, identifying proteomic indicators of multiple morbidities. Next, protein-based scores (ProteinScores) are developed using penalised Cox regression. When applied to test sets, six ProteinScores improve Area Under the Curve (AUC) estimates for the 10-year onset of incident outcomes beyond age, sex and a comprehensive set of 24 lifestyle factors, clinically-relevant biomarkers and physical measures. Furthermore, the ProteinScore for type 2 diabetes outperformed a polygenic risk score, a metabolomic score and HbA1c - a clinical marker used to monitor and diagnose type 2 diabetes. These data characterise early proteomic contributions to major age-related disease and demonstrate the value of the plasma proteome for risk stratification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.23293855">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.23293855" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.23293855">
        <p class="paperTitle">Association of Life&#39;s Essential 8 with Cardiovascular Events and Mortality: The Cardiovascular Disease Lifetime Risk Pooling Project (LRPP)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.23293855" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.23293855" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ning, H.; Allen, N.; Lloyd-Jones, D. M.; Perak, A. M.; Siddique, J.; Wilkins, J. T.</p>
        <p class="info">Score: 14.3, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.23293855' target='https://doi.org/10.1101/2023.08.08.23293855'> 10.1101/2023.08.08.23293855</a></p>
        <p class="abstract">BackgroundThe American Heart Association (AHA) recently launched updated cardiovascular health (CVH) metrics, termed Lifes Essential 8 (LE8). Compared to Lifes Simple 7 (LS7), the new approach added sleep health as an eighth metric and updated the remaining 7 metrics. Whether the updated LE8 score outperforms the original LS7 score in predicting cardiovascular disease (CVD) is not known. We examined the association of LE8 scores with CVD, subtype CVD events and all-cause mortality.

MethodsWe pooled individual-level data from 6 contemporary US-based cohorts from the Cardiovascular Lifetime Risk Pooling Project (LRPP). Total LE8 score (0-100 points), LE8 score without sleep (0-100 points), as well as prior LS7 scores (0-14 points), were calculated separately. We used multivariable-adjusted Cox models to evaluate the association of LE8 with CVD, CVD subtypes, and all-cause mortality among younger, middle, and older aged adult participants. Reclassification was defined based on the concordant/discordant categories of LS7 and LE8 scores quartile rankings across age groups.

ResultsOur sample consisted of 32,896 US adults (7836 [23.8%] Black; 14941 [45.4%] men) followed for 642,000 person-years; of whom 9,391 developed CVD events. Each 10-point higher overall LE8 score was associated with 23-40% lower CVD risk across age groups. Reclassification of CVH from LS7 to LE8 was related to heath behaviors as well as health factors and was significantly associated with CVD risk.

ConclusionsThese findings support the improved utility of the LE8 algorithm for assessing overall cardiovascular health and future CVD risk.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.23293833">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.23293833" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.23293833">
        <p class="paperTitle">One Health Genomic Surveillance of Avian and Human Influenza A Viruses Through Environmental Wastewater Monitoring</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.23293833" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.23293833" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, A. J.; Carson, S.; Reyne, M. I.; Marshall, A.; Moody, D.; Allen, D.; Allingham, P.; Levickas, A.; Fitzgerald, A.; Bell, S. H.; Lock, J.; Lock, J. D.; McSparron, C.; Nejad, B. F.; Courtney, D. G.; Einarsson, G. G.; McKenna, J. P.; Fairley, D. J.; Curran, T.; McKinley, J. M.; Gilpin, D. F.; Lemon, K.; McGrath, J. W.; Bamford, C. G.</p>
        <p class="info">Score: 12.7, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.23293833' target='https://doi.org/10.1101/2023.08.08.23293833'> 10.1101/2023.08.08.23293833</a></p>
        <p class="abstract">BackgroundInfluenza A viruses (IAV) are significant pathogens of humans and other animals. Although endemic in humans and birds, novel IAV strains can emerge, jump species, and cause epidemics, like the latest variant of H5N1. Wastewater-based epidemiology (WBE) has very recently been shown to detect human IAV but whether it can detect avian-origin IAV, and if whole genome sequencing (WGS) can be used to discriminate circulating strains of IAV in wastewater remains unknown.

MethodsUsing a pan-IAV RT-qPCR assay, six wastewater treatment works (WWTWs) across Northern Ireland (NI), were screened from August to December 2022. A WGS approach using Oxford Nanopore technology was employed to sequence positive samples. Phylogenetic analysis of sequences relative to currently circulating human and avian IAVs was performed.

FindingsWe detected a dynamic IAV signal in wastewater from September 2022 onwards across NI. &#34;Meta&#34; whole genome sequences were generated displaying homology to both human and avian IAV strains. The relative proportion of human versus avian-origin IAV reads differed across time and sample site. A diversity in subtypes and lineages was detected (e.g. H1N1, H3N2, and several avian). Avian segment 8 related to those found in recent H5N1 clade 2.3.4.4b was identified.

InterpretationWBE affords a means to monitor circulating human and avian IAV strains and provide crucial genetic information. As such WBE can provide rapid, cost-effective, year-round &#34;one-health&#34; IAV surveillance to help control epidemic and pandemic threats.

FundingThis study was funded by the Department of Health for Northern Ireland as part of the Northern Ireland Wastewater Surveillance Programme.

HighlightsO_LIDynamic IAV RT-qPCR signal in wastewater detected across NI.
C_LIO_LINanopore-based WGS reveals presence of both human and avian IAVs in wastewater.
C_LIO_LIAvian IAV sequence similarity to gull-associated H13/H16 and recent H5N1 isolates.
C_LIO_LICo-detection of distinct clades of human H1N1 and H3N2 subtypes.
C_LI

Author SummaryInfluenza A virus (IAV) is a major pathogen of humans and other animals and causes regular epidemics and devastating pandemics. Recently, a novel variant of highly-pathogenic H5N1 avian influenza has emerged spreading across the world killing millions of birds and infecting mammals, enhancing its pandemic potential. Strengthening global surveillance systems for human and animal IAV is thus a major priority. Wastewater-based epidemiology (WBE) has been applied to track SARS-CoV-2 and IAV in humans but whether this approach could work for avian IAV is not known. Here, we develop a &#34;one-health&#34; method to survey pan-IAV levels and genetically characterise the viruses. Through this we highlight co-detection of human and avian IAVs in wastewater, with homology to recent H5N1 isolates. Our work demonstrates the potential for WBE to help defend against not only human infections but emerging, zoonotic IAVs of pandemic potential.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=133 SRC=&#34;FIGDIR/small/23293833v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (26K):
org.highwire.dtl.DTLVardef@13bd526org.highwire.dtl.DTLVardef@126db74org.highwire.dtl.DTLVardef@11b4dborg.highwire.dtl.DTLVardef@78a64f_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.23292811">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.23292811" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.23292811">
        <p class="paperTitle">Proteomic prediction of common and rare diseases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.23292811" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.23292811" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carrasco-Zanini, J.; Pietzner, M.; Davitte, J.; Surendran, P.; Croteau-Chonka, D. C.; Robins, C.; Torralbo, A.; Tomlinson, C.; Fitzpatrick, N.; Ytsma, C.; Kanno, T.; Gade, S.; Freitag, D.; Ziebell, F.; Denaxas, S.; Betts, J. C.; Wareham, N. J.; Hemingway, H.; Scott, R. A.; Langenberg, C.</p>
        <p class="info">Score: 43.2, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.23292811' target='https://doi.org/10.1101/2023.07.18.23292811'> 10.1101/2023.07.18.23292811</a></p>
        <p class="abstract">BackgroundFor many diseases there are delays in diagnosis due to a lack of objective biomarkers for disease onset. Whether measuring thousands of proteins offers predictive information across a wide range of diseases is unknown.

MethodsIn 41,931 individuals from the UK Biobank Pharma Proteomics Project (UKB-PPP), we integrated [~]3000 plasma proteins with clinical information to derive sparse prediction models for the 10-year incidence of 218 common and rare diseases (81 - 6038 cases). We compared prediction models based on proteins with a) basic clinical information alone, b) basic clinical information &#43; 37 clinical biomarkers, and c) genome-wide polygenic risk scores.

ResultsFor 67 pathologically diverse diseases, a model including as few as 5 to 20 proteins was superior to clinical models (median delta C-index = 0.07; range = 0.02 - 0.31) and to clinical models with biomarkers for 52 diseases. In multiple myeloma, for example, a set of 5 proteins significantly improved prediction over basic clinical information (delta C-index = 0.25 (95% confidence interval 0.20 - 0.29)). At a 5% false positive rate (FPR), proteomic prediction (5 proteins) identified individuals at high risk of multiple myeloma (detection rate (DR) = 50%), non-Hodgkin lymphoma (DR = 55%) and motor neuron disease (DR = 29%). At a 20% FPR, proteomic prediction identified individuals at high-risk for pulmonary fibrosis (DR= 80%) and dilated cardiomyopathy (DR = 75%).

ConclusionsSparse plasma protein signatures offer novel, clinically useful prediction of common and rare diseases, through disease-specific proteins and protein predictors shared across multiple diseases.

(Funded by Medical Research Council, NIHR, Wellcome Trust.)</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.07.18.22277746">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.07.18.22277746" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.07.18.22277746">
        <p class="paperTitle">Comparing methods to predict baseline mortality for excess mortality calculations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.07.18.22277746" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.07.18.22277746" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ferenci, T.</p>
        <p class="info">Score: 10.2, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.07.18.22277746' target='https://doi.org/10.1101/2022.07.18.22277746'> 10.1101/2022.07.18.22277746</a></p>
        <p class="abstract">Background: The World Health Organizations excess mortality estimates presented in May 2022 stirred controversy, one point of which was the high estimate for Germany, later attributed to the used spline-model. This paper aims to reproduce the problem using synthetic datasets, thus allowing the investigation of its sensitivity to parameters, both of the mortality curve and of the used method, thereby shedding light on the conditions that gave rise to this error and its possible remedies. Methods: A negative binomial model was used accounting for long-term change, seasonality and flu seasons and heat waves. Simulated mortality curves from this model were then analysed with simple methods (mean, linear trend), with the WHOs method and with the method of Acosta and Irizarry. Results: The performance of the WHOs method with its original parametrization is indeed very poor, however it can be profoundly improved by a better choice of parameters. The Acosta-Irizarry method outperformed WHOs method despite being also based on splines, but it was also dependent on its parameters. Linear extrapolation could produce very good results, but is highly dependent on the choice of the starting year, while average was the worst in almost all cases. Conclusions: Splines are not inherently unsuitable for predicting baseline mortality, but care should be taken, in particular, results suggest that the key issue is the rigidity of the splines. Model diagnostics must be performed before accepting any result, and used methods should be validated. Further research is warranted to understand how these results can be generalized to other scenarios.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.23293816">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.23293816" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.23293816">
        <p class="paperTitle">Challenges in using data on fathers/partners to study prenatal exposures and offspring health</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.23293816" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.23293816" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Easey, K.; Gkatzionis, A.; Millard, L. A. C.; Tilling, K.; Lawlor, D. A.; Sharp, G.</p>
        <p class="info">Score: 3.2, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.23293816' target='https://doi.org/10.1101/2023.08.08.23293816'> 10.1101/2023.08.08.23293816</a></p>
        <p class="abstract">IntroductionPaternal exposures (and other non-maternal factors) around pregnancy could have important effects on offspring health. One challenge in research of paternal effects is that study samples with data on partners are usually a subgroup of those with data on mothers, which could introduce selection bias and limit generalisability of the findings. Here, we use maternal and father/partner data on prenatal behaviours to explore the extent to which selection in partner analyses might bias findings.

MethodsWe characterise the availability of data on father/partner and mother health behaviours (smoking, alcohol consumption, caffeine consumption and physical activity) in the prenatal period from three UK cohort studies: the Avon Longitudinal Study of Parents and Children (ALSPAC), Born in Bradford (BiB) and the Millennium Cohort Study (MCS). We assess the extent of sample selection in these cohorts by comparing the characteristics of families where the father/partner does and does not participate. Using the association of parental smoking during pregnancy and child birthweight as an exemplar, we used simulations to explore the extent to which missing father/partner data may bias estimates.

ResultsIn all three cohorts, data on prenatal health behaviours of fathers/partners were less detailed and collected at fewer timepoints than data on prenatal health behaviours of mothers. Partners of mothers who had a lower socioeconomic position were less likely to participate. Estimates of the association between maternal smoking and offspring birthweight were similar in samples with and without participating partners in all three cohorts. In simulations based on ALSPAC data, there was little evidence of selection bias in associations of maternal smoking with birthweight, and although bias was observed for father/partner smoking, its magnitude was relatively small.

DiscussionUsing real and simulated data, we show that bias due to selected recruitment of partners into ALSPAC, BiB and MCS had a relatively small impact on estimates of the effects of maternal and partner smoking on offspring birthweight. In other applications, the impact of selection bias will depend on both the analysis model and the selection mechanism. We have shown how to use a simulation study to assess that and recommend that applied researchers working with partner data use simulations and other sensitivity analyses to assess the robustness of their results.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.23293866">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.23293866" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.23293866">
        <p class="paperTitle">An epidemiological study of season of birth, mental health, and neuroimaging in the UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.23293866" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.23293866" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Viejo-Romero, M.; Whalley, H. C.; Shen, X.; Stolicyn, A.; de Nooij, L.; Smith, D. J.; Howard, D. M.</p>
        <p class="info">Score: 3.0, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.23293866' target='https://doi.org/10.1101/2023.08.09.23293866'> 10.1101/2023.08.09.23293866</a></p>
        <p class="abstract">Environmental exposures during the perinatal period are known to have a long-term effect on adult physical and mental health. One such influential environmental exposure is the time of year of birth which affects the amount of daylight, nutrients, and viral load that an individual is exposed to in the key developmental period. Here we investigate associations between season of birth (seasonality), four mental health traits (n=135,541) and multi-modal neuroimaging measures (n=33,815) within the UK Biobank. Summer births were associated with probable recurrent Major Depressive Disorder ({beta}=0.024, pcorr=0.048), greater mean cortical thickness in temporal and occipital lobes and in the middle temporal, fusiform, superior temporal, and lingual gyri regions ({beta} range=0.013 - 0.020, pcorr&lt;0.05). Winter births were associated with greater white matter integrity globally, in the association fibers, thalamic radiations, and six individual tracts ({beta} range=-0.010 to -0.021, pcorr&lt;0.05). Results of sensitivity analyses that adjusted for birth weight were similar, with additional associations found between winter birth and frontal, occipital and cingulate lobe surface areas, as well as fractional anisotropy in the forceps minor. Sensitivity analyses also revealed an additional association between summer birth and greater cingulate thickness. Overall, results suggest that seasonality affects brain structure in later life and may have a role in lifetime recurrent Major Depressive Disorder. The small effect sizes observed here warrant further research to validate the results in the context of different latitudes and co-examine genetic and epigenetic measures to potentially reveal informative biological pathways.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
