<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>epidemiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>epidemiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Unreliability in Simulations of COVID-19 Cases and Deaths Based on Transmission Models
Authors: Kakeya, H.; Itoh, M.; Kamijima, Y.; Nitta, T.; Umeno, Y.
Score: 401.3, Published: 2024-02-04 DOI: 10.1101/2024.02.02.24302123
Two papers authored by the same research group were published in academic journals in October 2023, both of which simulate counterfactual COVID-19 cases and deaths using transmission models. One paper estimates that the COVID-19 cases and deaths from Feb 17 to Nov 30, 2021 in Japan would have been as many as 63.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/epidemiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="epidemiology" />
<meta property="og:description" content="Unreliability in Simulations of COVID-19 Cases and Deaths Based on Transmission Models
Authors: Kakeya, H.; Itoh, M.; Kamijima, Y.; Nitta, T.; Umeno, Y.
Score: 401.3, Published: 2024-02-04 DOI: 10.1101/2024.02.02.24302123
Two papers authored by the same research group were published in academic journals in October 2023, both of which simulate counterfactual COVID-19 cases and deaths using transmission models. One paper estimates that the COVID-19 cases and deaths from Feb 17 to Nov 30, 2021 in Japan would have been as many as 63." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/epidemiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-14T10:38:48+00:00" />
<meta property="article:modified_time" content="2024-02-14T10:38:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="epidemiology"/>
<meta name="twitter:description" content="Unreliability in Simulations of COVID-19 Cases and Deaths Based on Transmission Models
Authors: Kakeya, H.; Itoh, M.; Kamijima, Y.; Nitta, T.; Umeno, Y.
Score: 401.3, Published: 2024-02-04 DOI: 10.1101/2024.02.02.24302123
Two papers authored by the same research group were published in academic journals in October 2023, both of which simulate counterfactual COVID-19 cases and deaths using transmission models. One paper estimates that the COVID-19 cases and deaths from Feb 17 to Nov 30, 2021 in Japan would have been as many as 63."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "epidemiology",
      "item": "https://trxiv.yorks0n.com/posts/epidemiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "epidemiology",
  "name": "epidemiology",
  "description": "Unreliability in Simulations of COVID-19 Cases and Deaths Based on Transmission Models\nAuthors: Kakeya, H.; Itoh, M.; Kamijima, Y.; Nitta, T.; Umeno, Y.\nScore: 401.3, Published: 2024-02-04 DOI: 10.1101/2024.02.02.24302123\nTwo papers authored by the same research group were published in academic journals in October 2023, both of which simulate counterfactual COVID-19 cases and deaths using transmission models. One paper estimates that the COVID-19 cases and deaths from Feb 17 to Nov 30, 2021 in Japan would have been as many as 63.",
  "keywords": [
    
  ],
  "articleBody": " Unreliability in Simulations of COVID-19 Cases and Deaths Based on Transmission Models\nAuthors: Kakeya, H.; Itoh, M.; Kamijima, Y.; Nitta, T.; Umeno, Y.\nScore: 401.3, Published: 2024-02-04 DOI: 10.1101/2024.02.02.24302123\nTwo papers authored by the same research group were published in academic journals in October 2023, both of which simulate counterfactual COVID-19 cases and deaths using transmission models. One paper estimates that the COVID-19 cases and deaths from Feb 17 to Nov 30, 2021 in Japan would have been as many as 63.3 million and 364 thousand respectively had the vaccination not been implemented, where the 95% confidence interval is claimed to be less than 1% of the estimated value. It also claims that the cases and deaths could have been reduced by 54% and 48% respectively had the vaccination been implemented 14 days earlier. The other paper estimates that the number of cases in early 2022, Tokyo would have been larger than the number of populations in the age group under 49 in the absence of the vaccination program. In this paper, we reexamine the results given by these papers to find that the simulation results do not explain the real-world data in Japan including prefectures with early/late vaccination schedules. The cause of discrepancy is identified as low reliability of model parameters that immensely affect the simulation results of case and death counts. Leaders of public healthcare are required to discern the reliability and credibility of simulation studies and to prepare for variety of possible scenarios when reliable predictions are not available.\nResults reporting for clinical trials led by medical universities and university hospitals in the Nordic countries was often missing or delayed\nAuthors: Nilsonne, G.; Wieschowski, S.; DeVito, N. J.; Salholz-Hillel, M.; Bruckner, T.; Klas, K.; Suljic, T.; Yerunkar, S.; Olsson, N.; Cruz, C.; Strzebonska, K.; Smabrekke, L.; Wasylewski, M. T.; Bengtsson, J.; Ringsten, M.; Schuster, A.; Krawczyk, T.; Paraskevas, T.; Ahnstrom, L.; Raittio, E.; Herczeg, L.; Hesselberg, J.-O.; Karlsson, S.; Borana, R.; Bruschettini, M.; Mulinari, S.; Lizarraga, K.; Siebert, M.; Hildebrand, N.; Ramakrishnan, S.; Janiaud, P.; Zavalis, E.; Franzen, D. L.; Boesen, K.; Hemkens, L. G.; Naudet, F.; Possmark, S.; Willen, R. M.; Ioannidis, J. P.; Strech, D.; Axfors, C.\nScore: 27.6, Published: 2024-02-05 DOI: 10.1101/2024.02.04.24301363\nObjectiveTo systematically evaluate timely reporting of clinical trial results at medical universities and university hospitals in the Nordic countries. Study Design and SettingIn this cross-sectional study, we included trials (regardless of intervention) registered in the EU Clinical Trials Registry and/or ClinicalTrials.gov, completed 2016-2019, and led by a university with medical faculty or university hospital in Denmark, Finland, Iceland, Norway, or Sweden. We identified summary results posted at the trial registries, and conducted systematic manual searches for results publications (e.g., journal articles, preprints). We present proportions with 95% confidence intervals (CI), and medians with interquartile range (IQR). Protocol: https://osf.io/wua3r ResultsAmong 2,113 included clinical trials, 1,638 (77.5%, 95%CI 75.9-79.2%) reported any results during our follow-up; 1,092 (51.7%, 95%CI 49.5-53.8%) reported any results within 2 years of the global completion date; and 42 (2%, 95%CI 1.5-2.7%) posted summary results in the registry within 1 year. Median time from global completion date to results reporting was 698 days (IQR 1,123). 856/1,681 (50.9%) of ClinicalTrials.gov-registrations were prospective. Denmark contributed approximately half of all trials. Reporting performance varied widely between institutions. ConclusionMissing and delayed results reporting of academically-led clinical trials is a pervasive problem in the Nordic countries. We relied on trial registry information, which can be incomplete. Institutions, funders, and policy makers need to support trial teams, ensure regulation adherence, and secure trial reporting before results are permanently lost. What is new?- Many Nordic registered clinical trials were reported late or not at all. - Almost one in four trials remained unreported at the end of our search period. - About half of registered trials had reported results two years after completion. - Only 2% of trials posted summary results in the registry one year after completion. - Concerted action is needed to improve reporting of Nordic clinical trials.\nBidirectional relationship between sleep problems and long COVID: a longitudinal analysis of data from the COVIDENCE UK study\nAuthors: Vivaldi, G.; Talaei, M.; Blaikley, J.; Jackson, C.; Pfeffer, P. E.; Shaheen, S. O.; Martineau, A. R.\nScore: 10.2, Published: 2024-02-08 DOI: 10.1101/2024.02.08.24302486\nBackgroundStudies into the bidirectional relationship between sleep and long COVID have been limited by retrospective pre-infection sleep data and infrequent post-infection follow-up. We therefore used prospectively collected monthly data to evaluate how pre-infection sleep characteristics affect risk of long COVID, and to track changes in sleep duration during the year after SARS-CoV-2 infection. MethodsCOVIDENCE UK is a prospective, population-based UK study of COVID-19 in adults. We included non-hospitalised participants with evidence of SARS-CoV-2 infection, and estimated odds ratios (ORs) for the association between pre-infection sleep characteristics and long COVID using logistic regression, adjusting for potential confounders. We assessed changes in sleep duration after infection using multilevel mixed models. We defined long COVID as unresolved symptoms at least 12 weeks after infection. We defined sleep quality according to age-dependent combinations of sleep duration and efficiency. COVIDENCE UK is registered with ClinicalTrials.gov, NCT04330599. FindingsWe included 3994 participants in our long COVID risk analysis, of whom 327 (8.2%) reported long COVID. We found an inverse relationship between pre-infection sleep quality and risk of long COVID (medium vs good quality: OR 1.37 [95% CI 1.04-1.81]; medium-low vs good: 1.55 [1.12-2.16]; low vs good: 1.94 [1.11-3.38]). Greater variability in pre-infection sleep efficiency was also associated with long COVID (OR per percentage-point increase 1.06 [1.01-1.11]). We assessed post-infection sleep duration in 6860 participants, observing a 0.11 h (95% CI 0.08-0.13) increase in the first month after infection compared with pre-infection, with larger increases for more severe infections. After 1 month, sleep duration largely returned to pre-infection levels, although fluctuations in duration lasted up to 6 months after infection among people reporting long COVID. InterpretationOur findings highlight the bidirectional relationship between sleep and long COVID. While poor-quality sleep before SARS-CoV-2 infection associates with increased risk of long COVID thereafter, changes in sleep duration after infection in these non-hospitalised cases were modest and generally quick to resolve. FundingBarts Charity.\nMetabolomic Age (MileAge) predicts health and lifespan: a comparison of multiple machine learning algorithms\nAuthors: Mutz, J.; Iniesta, R.; Lewis, C. M.\nScore: 6.9, Published: 2024-02-11 DOI: 10.1101/2024.02.10.24302617\nBackgroundMolecular ageing clocks estimate an individuals biological age. Our aim was to compare multiple machine learning algorithms for developing ageing clocks from nuclear magnetic resonance (NMR) spectroscopy metabolomics data. To validate how well each ageing clock predicted age-related morbidity and lifespan, we assessed their associations with multiple health indicators (e.g., telomere length and frailty) and all-cause mortality. MethodsThe UK Biobank is a multicentre observational health study of middle-aged and older adults. The Nightingale Health platform was used to quantify 168 circulating plasma metabolites at the baseline assessment from 2006 to 2010. We trained and internally validated 17 machine learning algorithms including regularised regression, kernel-based methods and ensembles. Metabolomic age (MileAge) delta was defined as the difference between predicted and chronological age. ResultsThe sample included 101,359 participants (mean age = 56.53 years, SD = 8.10). Most metabolite levels varied by chronological age. The nested cross-validation mean absolute error (MAE) ranged from 5.31 to 6.36 years. 31.76% of participants had an age-bias adjusted MileAge more than one standard deviation (3.75 years) above or below the mean. A Cubist rule-based regression model overall performed best at predicting health outcomes. The all-cause mortality hazard ratio (HR) comparing individuals with a MileAge delta more than one standard deviation above and below the mean was HR = 1.52 (95% CI 1.41-1.64, p \u003c 0.001) over a median follow-up of 13.87 years. Individuals with an older MileAge were frailer, had shorter telomeres, were more likely to have a chronic illness and rated their health worse. ConclusionsMetabolomic ageing clocks derived from multiple machine learning algorithms were robustly associated with health indicators and mortality. Our metabolomic ageing clock (MileAge) derived from a Cubist rule-based regression model can be incorporated in research, and may find applications in health assessments, risk stratification and proactive health tracking.\nA Stacking Framework for Polygenic Risk Prediction in Admixed Individuals\nAuthors: Liao, K.; Zöllner, S.\nScore: 7.1, Published: 2024-02-03 DOI: 10.1101/2024.01.31.24302103\n1.1Polygenic risk scores (PRS) are summaries of an individuals personalized genetic risk for a trait or disease. However, PRS often perform poorly for phenotype prediction when the ancestry of the target population does not match the population in which GWAS effect sizes were estimated. For many populations this can be addressed by performing GWAS in the target population. However, admixed individuals (whose genomes can be traced to multiple ancestral populations) lie on an ancestry continuum and are not easily represented as a discrete population. Here, we propose slaPRS (stacking local ancestry PRS), which incorporates multiple ancestry GWAS to alleviate the ancestry dependence of PRS in admixed samples. slaPRS uses ensemble learning (stacking) to combine local population specific PRS in regions across the genome. We compare slaPRS to single population PRS and a method that combines single population PRS globally. In simulations, slaPRS outperformed existing approaches and reduced the ancestry dependence of PRS in African Americans. In lipid traits from African British individuals (UK Biobank), slaPRS again improved on single population PRS while performing comparably to the globally combined PRS. slaPRS provides a data-driven and flexible framework to incorporate multiple population-specific GWAS and local ancestry in samples of admixed ancestry.\nEffect of Paxlovid Treatment on Long COVID Onset: An EHR-Based Target Trial Emulation from N3C\nAuthors: Preiss, A.; Bhatia, A.; Zang, C.; Aragon, L. V.; Baratta, J. M.; Baskaran, M.; Blancero, F.; Brannock, M. D.; Chew, R. F.; Diaz, I.; Fitzgerald, M.; Kelly, E. P.; Zhou, A. G.; Weiner, M. G.; Carton, T. W.; Wang, F.; Kaushal, R.; Chute, C. G.; Haendel, M.; Moffitt, R.; Pfaff, E.; N3C, ; RECOVER, Score: 76.3, Published: 2024-01-22 DOI: 10.1101/2024.01.20.24301525\nPreventing and treating post-acute sequelae of SARS-CoV-2 infection (PASC), commonly known as Long COVID, has become a public health priority. In this study, we examined whether treatment with Paxlovid in the acute phase of COVID-19 helps prevent the onset of PASC. We used electronic health records from the National Covid Cohort Collaborative (N3C) to define a cohort of 426,461 patients who had COVID-19 since April 1, 2022, and were eligible for Paxlovid treatment due to risk for progression to severe COVID-19. We used the target trial emulation (TTE) framework to estimate the effect of Paxlovid treatment on PASC incidence. Our primary outcome measure was a PASC computable phenotype. Secondary outcomes were the onset of novel cognitive, fatigue, and respiratory symptoms in the post-acute period. Paxlovid treatment did not have a significant effect on overall PASC incidence (relative risk [RR] = 0.99, 95% confidence interval [CI] 0.96-1.01). However, its effect varied across the cognitive (RR = 0.85, 95% CI 0.79-0.90), fatigue (RR = 0.93, 95% CI 0.89-0.96), and respiratory (RR = 0.99, 95% CI 0.95-1.02) symptom clusters, suggesting that Paxlovid treatment may help prevent post-acute cognitive and fatigue symptoms more than others.\nHealthy lifestyle for the prevention of post-COVID-19 multisystem sequelae, hospitalization, and death: a prospective cohort study\nAuthors: Wang, Y.; Su, B.; Alcalde-Herraiz, M.; Barclay, N. L.; Tian, Y.; Li, C.; Wareham, N. J.; Paredes, R.; Xie, J.; PRIETO-ALHAMBRA, D.\nScore: 8.8, Published: 2024-01-31 DOI: 10.1101/2024.01.30.24302040\nBackgroundPost-COVID complications are emerging as a global public health crisis. Effective prevention strategies are needed to inform patients, clinicians and policy makers, and to reduce their cumulative burden. We aimed to investigate whether a habitual healthy lifestyle predated pandemic is associated with lower risks of multisystem sequelae and other adverse outcomes of COVID-19, and whether the potential protective effects are independent of pre-existing comorbidities. MethodsThe prospective population-based cohort study enrolled participants with SARS-CoV-2 infection confirmed by a positive polymerase chain reaction test result between March 1, 2020, and March 1, 2022. Participants with no history of the related outcome one year before infection were included and followed up for 210 days. Exposures included ten modifiable healthy lifestyle factors including past or never smoking, moderate alcohol intake ([\u0026le;]4 times week), body mass index \u003c30 kg/m2, at least 150 minutes of moderate or 75 minutes of vigorous physical activity per week, less sedentary time (\u003c4 hours per day), healthy sleep duration (7-9 hours per day), adequate intake of fruit and vegetables ([\u0026ge;]400 g/day), adequate oily fish intake ([\u0026ge;]1 portion/week), moderate intake of red meat ([\u0026le;]4 portions week) and processed meat ([\u0026le;]4 portions week). Outcomes included multisystem COVID-19 sequelae (consisting of 75 diseases/symptoms in 10 organ systems), death, and hospital admission following SARS-CoV-2 infection, confirmed by hospital inpatient and death records. Risk was reported in relative scale (hazard ratio [HR]) and absolute scale (absolute risk reduction [ARR]) during both the acute (the first 30 days) and post-acute (30-210 days) phases of infection using Cox models. FindingsA total of 68,896 participants (mean [SD] age, 66.6 [8.4]; 32,098 women [46.6%]) with COVID-19 were included. A favorable lifestyle (6-10 healthy lifestyle factors; 46.4%) was associated with a 36% lower risk of multisystem sequelae of COVID-19 (HR, 0.64; 95% CI, 0.58-0.69; ARR, 7.08%; 95% CI, 5.98-8.09), compared with unfavorable lifestyle (0-4 factors; 12.3%). Risk reductions were observed across all 10 prespecified organ systems including cardiovascular, coagulation, metabolic and endocrine, gastrointestinal, kidney, mental health, musculoskeletal, neurologic, and respiratory disorders, and general symptoms of fatigue and malaise. This beneficial effect was largely attributable to direct effects of healthy lifestyle, with mediation proportion ranging from 44% to 93% across organ systems. A favorable lifestyle was also associated with lower risk of post-COVID death (HR, 0.59; 95% CI, 0.52-0.66; ARR, 1.99%; 95% CI, 1.61-2.32) and hospitalization (HR, 0.78; 95% CI, 0.73-0.84; ARR, 6.14%; 95% CI, 4.48-7.68). These associations were observed after accounting for potential misclassification of lifestyle factors, and during acute and post-acute infection, in those tested positive in the hospital and community setting, and independent of vaccination status or SARS-CoV-2 variant. InterpretationAdherence to a healthy lifestyle predated pandemic was associated with substantially lower risk of complications across organ systems, death, and hospitalization following COVID-19, regardless of phases of infection, vaccination status, test setting, and SARS-CoV-2 variants, and independent of comorbidities. These findings illustrate the benefits of adhering to a healthy lifestyle to reduce the long-term adverse health consequences following SARS-CoV-2 infection. Research in contextO_ST_ABSEvidence before this studyC_ST_ABSWe searched PubMed and MEDLINE for articles published between March 1, 2020, and December 1, 2023, using the search terms \"healthy lifestyle\", \"risk factor\", \"post-COVID condition\", \"long COVID\", \"post-acute sequelae\", \"prevention\", \"management\", and \"treatment\", with no language restrictions. Previous evidence on the prevention and management of long COVID has mainly focused on vaccination and pharmaceutical approaches, including antivirals (e.g., molnupiravir and nirmatrelvir) and other drugs (e.g., metformin). Vaccination before infection or use of antivirals in selected high-risk patients during acute infection only partially mediates the risk of COVID-19 sequelae. Evidence for the non-pharmaceutical prevention strategies are lacking. We identified only two publications on the association between healthy lifestyle and post-COVID condition, and one meta-analysis of the risk factors for long COVID symptoms. A cross-sectional study of 1981 women suggested an inverse association between healthy lifestyle factors and self-reported symptoms following infection of non-Omicron variants, which was mainly driven by BMI and sleep duration. Another study suggested an inverse prospective association between healthy lifestyle prior to infection and post-COVID cardiovascular events. High BMI and smoking are risk factors for long COVID mainly in hospitalized patients. We did not find any study that assessed the association between a composite healthy lifestyle and subsequent post-COVID complications or sequelae across organ systems, hospitalization, and death. Added value of this studyIn a prospective, population-based cohort of 68,896 participants with COVID-19, adherence to a healthy lifestyle prior to infection was associated with a substantially lower risk of multisystem sequelae (by 20%-36%), death (by 26%-41%), and hospital admission (by 13%-22%) following COVID-19. The reduced risk of sequelae was evident across 10 prespecified organ systems, including cardiovascular, coagulation and hematologic, metabolic and endocrine, gastrointestinal, kidney, mental health, musculoskeletal, neurologic, and respiratory disorders, as well as general symptoms of fatigue and malaise. The reduced risk of multisystem sequelae, hospitalization, and death associated with a healthy lifestyle was consistently observed across participants, regardless of their vaccination status, disease severity, and major SARS-CoV-2 variants, and largely independent of relevant comorbidities. Adherence to a healthy lifestyle prior to infection was consistently and directly associated with reduced risk of sequelae and other adverse health outcomes following COVID-19. Implications of all the available evidenceThe inverse association of healthy lifestyle with multisystem sequelae was even larger than those observed in previous studies of pharmaceutical interventions in non-hospitalized patients. Considering the restricted scope of currently available therapies, such as antivirals (only selected patients at higher risk are qualified during the acute infection) and limited efficacy of vaccination in preventing long COVID, adherence to a healthy lifestyle, in combination with vaccination and, if necessary, potential medications, emerges as practical prevention and care strategies to mitigate the long-term health consequences of SARS-CoV-2 infection. These strategies are of significant clinical and public health importance in reducing the overall burden of post-COVID conditions and improving preparedness for future pandemics.\nSafety of BNT162b2 mRNA COVID-19 vaccine batches: A nationwide cohort study\nAuthors: Hviid, A.; Bech Svalgaard, I.\nScore: 18.8, Published: 2024-01-24 DOI: 10.1101/2024.01.22.24301520\nBackgroundThe safety of the BNT162b2 mRNA COVID-19 vaccine has been extensively evaluated since the global rollout began. While serious adverse events are rare, safety issues continue to arise. This study evaluates the claim that earlier small vaccine batches were associated with higher rates of serious adverse events compared to later batches. MethodsA nationwide cohort study was conducted in Denmark, comprising individuals vaccinated with the BNT162b2 vaccine from 52 pre-defined batches classified into three pre-defined groups. Vaccinated individuals were matched 1:1 between batch groups on age, sex, and vaccination priority group. The study outcomes, included 27 serious adverse events, 2 negative control outcomes and all-cause mortality. Cox regression was used to estimate hazard ratios (HRs) comparing rates between batch groups in the 28-days following vaccination. We conducted two comparisons of the early small batches to two groups of larger batches used later in the pandemic. ResultsIn the study period, 9,983,448 vaccinations were administered from batches in the three pre-defined groups. Slightly increased rates of arrhythmia were observed in both study comparisons, HRs 1.25 (95% CI,1.05-1.50) and 1.15 (1.00-1.31), respectively, but sensitivity analyses did not robustly support these associations. For the remaining outcomes, increased rates in both study comparisons were not observed. ConclusionThis nationwide cohort study provides reassurance regarding the safety of the BNT162b2 vaccine across different batches used in Denmark. The findings support the overall safety of the vaccine, with no clinically relevant variations in serious adverse event rates between batches.\nInequalities in local funding cuts to environmental and regulatory service expenditure in England from 2009 to 2020\nAuthors: Murrell, L.; Fahy, K.; Clough, H. E.; Gibb, R.; Zhang, X.; Chattaway, M.; Green, M. A.; Buchan, I.; Barr, B.; Hungerford, D.\nScore: 3.0, Published: 2024-01-26 DOI: 10.1101/2024.01.26.24301656\nBackgroundLocal authorities have been subject to substantial reductions in funding, placing strain on local services. Environmental and Regulatory (ER) services provide essential functions including infectious disease prevention and control via Food Safety and Animal and Public Health Infectious Disease Control services (APHIDC). This study investigates inequalities in local funding cuts to these services. MethodsWe used a generalised estimating equation model to estimate the annual percent change of ER service expenditure between 2009/10 and 2020/21 in addition to Food Safety and APHIDC expenditure change overall, and as a share of total ER expenditure. Models analysed trends by deprivation level, local authority structure and population density. ResultsAreas of higher deprivation had the largest reduction in expenditure, with ER and Food Safety and APHIDC cuts of 2% and of 23% respectively, compared to a 1% and 8% reduction in the least deprived areas. The share of ER expenditure spent on Food Safety and APHIDC decreased by 13% in the most deprived authorities compared to 6% in the least deprived areas. Environmental and Regulatory services saw the largest cuts in unitary authorities, declining by 2%. London boroughs had the greatest reductions in Food Safety and APHIDC expenditure, decreasing by 10%. Both ER and Food Safety and APHIDC expenditure decreased with increasing population density. ConclusionThe unequal distribution of cuts shows the need for increased and equitable investment into these services to enable resilience to emerging infectious disease threats, and to prevent widening of health inequalities. Key MessagesO_ST_ABSWhat is already known on this topicC_ST_ABSO_LIAusterity measures have led to substantial reductions in local funding placing increased pressure on local authority services in England, research shows reductions vary by deprivation level of an area, rural - urban classification and local authority structure. C_LIO_LIIt is unknown how local funding cuts to Environmental and Regulatory services, which provide essential services for public health protection, vary by these characteristics. C_LI What this study addsO_LIWe investigate inequalities in austerity-enforced changes in Environmental and Regulatory service expenditure and sub-spending lines of Food Safety and Animal and Public Health Infectious Disease Control over time by deprivation, local authority structure and population density from 2009/10 to 2020/21. C_LIO_LIThe largest cuts were in the more deprived areas and with increased population density for both Environmental and Regulatory and Food safety and Animal and Public Health Infectious Disease Control services. The largest cuts in Environmental and Regulatory services were seen in unitary authorities whereas Food safety and Animal and Public Health Infectious Disease Control saw largest cuts in London boroughs. C_LI How this study might affect research, practice or policyO_LIThis research provides strong evidence of inequalities in local authority service expenditure in Environmental and Regulatory services and highlights where investment should be focused, in order to protect environmental and public health. C_LI\nUnderestimation of SARS-CoV-2 in wastewater due to single or double mutations in the N1 qPCR probe binding region\nAuthors: Sun, J.; Yang, M. I.; Peng, J.; Lopez, J. J.; Khan, I.; Chan, R.; Edwards, E. A.; Peng, H.\nScore: 127.4, Published: 2024-02-04 DOI: 10.1101/2024.02.03.24302274\nWastewater surveillance using RT-qPCR has now been widely adopted to track circulating levels of SARS-CoV-2 virus in many sewer sheds. The CDC qPCR assays targeting two regions (N1 and N2) within the N gene are commonly used, but a discrepancy between the two biomarkers has been noticed by many groups using this method since late 2021. The reason is presumed to be due to mutations in regions targeted by the qPCR probe. In this study, we systematically investigated and unequivocally confirmed that the underlying reason for this discrepancy was mutations in the N1 probe target, and that a single mutation could cause a significant drop in signal. We first confirmed the proportion of related mutations in wastewater samples (Jan 2021-Dec 2022) using nested PCR and LC-MS. Based on relative proportion of N1 alleles, we separated the wastewater data into four time periods corresponding to different variant waves: Period I (Alpha and Delta waves with 0 mutation), Period II (BA.1/BA.2 wave with a single mutation found in all Omicron strains), Period III (BA. 5.2* wave with two mutations), and Period IV (BQ.1* wave with two mutations). Significantly lower N1 copies relative to N2 copies in samples from Periods II-IV compared to those from Period I was observed in wastewater. To further pinpoint the extent to which each mutation impacted N1 quantification, we compared the qPCR response among different synthetic oligomers with corresponding mutations. This study highlighted the impact of even just one or two mutations on qPCR-based wastewater surveillance of SARS-CoV-2.\n",
  "wordCount" : "3893",
  "inLanguage": "en",
  "datePublished": "2024-02-14T10:38:48Z",
  "dateModified": "2024-02-14T10:38:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/epidemiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      epidemiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.24302123">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.24302123" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.24302123">
        <p class="paperTitle">Unreliability in Simulations of COVID-19 Cases and Deaths Based on Transmission Models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.24302123" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.24302123" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kakeya, H.; Itoh, M.; Kamijima, Y.; Nitta, T.; Umeno, Y.</p>
        <p class="info">Score: 401.3, Published: 2024-02-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.24302123' target='https://doi.org/10.1101/2024.02.02.24302123'> 10.1101/2024.02.02.24302123</a></p>
        <p class="abstract">Two papers authored by the same research group were published in academic journals in October 2023, both of which simulate counterfactual COVID-19 cases and deaths using transmission models. One paper estimates that the COVID-19 cases and deaths from Feb 17 to Nov 30, 2021 in Japan would have been as many as 63.3 million and 364 thousand respectively had the vaccination not been implemented, where the 95% confidence interval is claimed to be less than 1% of the estimated value. It also claims that the cases and deaths could have been reduced by 54% and 48% respectively had the vaccination been implemented 14 days earlier. The other paper estimates that the number of cases in early 2022, Tokyo would have been larger than the number of populations in the age group under 49 in the absence of the vaccination program. In this paper, we reexamine the results given by these papers to find that the simulation results do not explain the real-world data in Japan including prefectures with early/late vaccination schedules. The cause of discrepancy is identified as low reliability of model parameters that immensely affect the simulation results of case and death counts. Leaders of public healthcare are required to discern the reliability and credibility of simulation studies and to prepare for variety of possible scenarios when reliable predictions are not available.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.04.24301363">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.04.24301363" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.04.24301363">
        <p class="paperTitle">Results reporting for clinical trials led by medical universities and university hospitals in the Nordic countries was often missing or delayed</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.04.24301363" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.04.24301363" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nilsonne, G.; Wieschowski, S.; DeVito, N. J.; Salholz-Hillel, M.; Bruckner, T.; Klas, K.; Suljic, T.; Yerunkar, S.; Olsson, N.; Cruz, C.; Strzebonska, K.; Smabrekke, L.; Wasylewski, M. T.; Bengtsson, J.; Ringsten, M.; Schuster, A.; Krawczyk, T.; Paraskevas, T.; Ahnstrom, L.; Raittio, E.; Herczeg, L.; Hesselberg, J.-O.; Karlsson, S.; Borana, R.; Bruschettini, M.; Mulinari, S.; Lizarraga, K.; Siebert, M.; Hildebrand, N.; Ramakrishnan, S.; Janiaud, P.; Zavalis, E.; Franzen, D. L.; Boesen, K.; Hemkens, L. G.; Naudet, F.; Possmark, S.; Willen, R. M.; Ioannidis, J. P.; Strech, D.; Axfors, C.</p>
        <p class="info">Score: 27.6, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.04.24301363' target='https://doi.org/10.1101/2024.02.04.24301363'> 10.1101/2024.02.04.24301363</a></p>
        <p class="abstract">ObjectiveTo systematically evaluate timely reporting of clinical trial results at medical universities and university hospitals in the Nordic countries.

Study Design and SettingIn this cross-sectional study, we included trials (regardless of intervention) registered in the EU Clinical Trials Registry and/or ClinicalTrials.gov, completed 2016-2019, and led by a university with medical faculty or university hospital in Denmark, Finland, Iceland, Norway, or Sweden. We identified summary results posted at the trial registries, and conducted systematic manual searches for results publications (e.g., journal articles, preprints). We present proportions with 95% confidence intervals (CI), and medians with interquartile range (IQR). Protocol: https://osf.io/wua3r

ResultsAmong 2,113 included clinical trials, 1,638 (77.5%, 95%CI 75.9-79.2%) reported any results during our follow-up; 1,092 (51.7%, 95%CI 49.5-53.8%) reported any results within 2 years of the global completion date; and 42 (2%, 95%CI 1.5-2.7%) posted summary results in the registry within 1 year. Median time from global completion date to results reporting was 698 days (IQR 1,123). 856/1,681 (50.9%) of ClinicalTrials.gov-registrations were prospective. Denmark contributed approximately half of all trials. Reporting performance varied widely between institutions.

ConclusionMissing and delayed results reporting of academically-led clinical trials is a pervasive problem in the Nordic countries. We relied on trial registry information, which can be incomplete. Institutions, funders, and policy makers need to support trial teams, ensure regulation adherence, and secure trial reporting before results are permanently lost.

What is new?- Many Nordic registered clinical trials were reported late or not at all.
- Almost one in four trials remained unreported at the end of our search period.
- About half of registered trials had reported results two years after completion.
- Only 2% of trials posted summary results in the registry one year after completion.
- Concerted action is needed to improve reporting of Nordic clinical trials.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.08.24302486">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.08.24302486" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.08.24302486">
        <p class="paperTitle">Bidirectional relationship between sleep problems and long COVID: a longitudinal analysis of data from the COVIDENCE UK study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.08.24302486" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.08.24302486" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vivaldi, G.; Talaei, M.; Blaikley, J.; Jackson, C.; Pfeffer, P. E.; Shaheen, S. O.; Martineau, A. R.</p>
        <p class="info">Score: 10.2, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.08.24302486' target='https://doi.org/10.1101/2024.02.08.24302486'> 10.1101/2024.02.08.24302486</a></p>
        <p class="abstract">BackgroundStudies into the bidirectional relationship between sleep and long COVID have been limited by retrospective pre-infection sleep data and infrequent post-infection follow-up. We therefore used prospectively collected monthly data to evaluate how pre-infection sleep characteristics affect risk of long COVID, and to track changes in sleep duration during the year after SARS-CoV-2 infection.

MethodsCOVIDENCE UK is a prospective, population-based UK study of COVID-19 in adults. We included non-hospitalised participants with evidence of SARS-CoV-2 infection, and estimated odds ratios (ORs) for the association between pre-infection sleep characteristics and long COVID using logistic regression, adjusting for potential confounders. We assessed changes in sleep duration after infection using multilevel mixed models. We defined long COVID as unresolved symptoms at least 12 weeks after infection. We defined sleep quality according to age-dependent combinations of sleep duration and efficiency. COVIDENCE UK is registered with ClinicalTrials.gov, NCT04330599.

FindingsWe included 3994 participants in our long COVID risk analysis, of whom 327 (8.2%) reported long COVID. We found an inverse relationship between pre-infection sleep quality and risk of long COVID (medium vs good quality: OR 1.37 [95% CI 1.04-1.81]; medium-low vs good: 1.55 [1.12-2.16]; low vs good: 1.94 [1.11-3.38]). Greater variability in pre-infection sleep efficiency was also associated with long COVID (OR per percentage-point increase 1.06 [1.01-1.11]). We assessed post-infection sleep duration in 6860 participants, observing a 0.11 h (95% CI 0.08-0.13) increase in the first month after infection compared with pre-infection, with larger increases for more severe infections. After 1 month, sleep duration largely returned to pre-infection levels, although fluctuations in duration lasted up to 6 months after infection among people reporting long COVID.

InterpretationOur findings highlight the bidirectional relationship between sleep and long COVID. While poor-quality sleep before SARS-CoV-2 infection associates with increased risk of long COVID thereafter, changes in sleep duration after infection in these non-hospitalised cases were modest and generally quick to resolve.

FundingBarts Charity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.10.24302617">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.10.24302617" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.10.24302617">
        <p class="paperTitle">Metabolomic Age (MileAge) predicts health and lifespan: a comparison of multiple machine learning algorithms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.10.24302617" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.10.24302617" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mutz, J.; Iniesta, R.; Lewis, C. M.</p>
        <p class="info">Score: 6.9, Published: 2024-02-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.10.24302617' target='https://doi.org/10.1101/2024.02.10.24302617'> 10.1101/2024.02.10.24302617</a></p>
        <p class="abstract">BackgroundMolecular ageing clocks estimate an individuals biological age. Our aim was to compare multiple machine learning algorithms for developing ageing clocks from nuclear magnetic resonance (NMR) spectroscopy metabolomics data. To validate how well each ageing clock predicted age-related morbidity and lifespan, we assessed their associations with multiple health indicators (e.g., telomere length and frailty) and all-cause mortality.

MethodsThe UK Biobank is a multicentre observational health study of middle-aged and older adults. The Nightingale Health platform was used to quantify 168 circulating plasma metabolites at the baseline assessment from 2006 to 2010. We trained and internally validated 17 machine learning algorithms including regularised regression, kernel-based methods and ensembles. Metabolomic age (MileAge) delta was defined as the difference between predicted and chronological age.

ResultsThe sample included 101,359 participants (mean age = 56.53 years, SD = 8.10). Most metabolite levels varied by chronological age. The nested cross-validation mean absolute error (MAE) ranged from 5.31 to 6.36 years. 31.76% of participants had an age-bias adjusted MileAge more than one standard deviation (3.75 years) above or below the mean. A Cubist rule-based regression model overall performed best at predicting health outcomes. The all-cause mortality hazard ratio (HR) comparing individuals with a MileAge delta more than one standard deviation above and below the mean was HR = 1.52 (95% CI 1.41-1.64, p &lt; 0.001) over a median follow-up of 13.87 years. Individuals with an older MileAge were frailer, had shorter telomeres, were more likely to have a chronic illness and rated their health worse.

ConclusionsMetabolomic ageing clocks derived from multiple machine learning algorithms were robustly associated with health indicators and mortality. Our metabolomic ageing clock (MileAge) derived from a Cubist rule-based regression model can be incorporated in research, and may find applications in health assessments, risk stratification and proactive health tracking.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.24302103">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.24302103" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.24302103">
        <p class="paperTitle">A Stacking Framework for Polygenic Risk Prediction in Admixed Individuals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.24302103" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.24302103" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liao, K.; Zöllner, S.</p>
        <p class="info">Score: 7.1, Published: 2024-02-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.24302103' target='https://doi.org/10.1101/2024.01.31.24302103'> 10.1101/2024.01.31.24302103</a></p>
        <p class="abstract">1.1Polygenic risk scores (PRS) are summaries of an individuals personalized genetic risk for a trait or disease. However, PRS often perform poorly for phenotype prediction when the ancestry of the target population does not match the population in which GWAS effect sizes were estimated. For many populations this can be addressed by performing GWAS in the target population. However, admixed individuals (whose genomes can be traced to multiple ancestral populations) lie on an ancestry continuum and are not easily represented as a discrete population.

Here, we propose slaPRS (stacking local ancestry PRS), which incorporates multiple ancestry GWAS to alleviate the ancestry dependence of PRS in admixed samples. slaPRS uses ensemble learning (stacking) to combine local population specific PRS in regions across the genome. We compare slaPRS to single population PRS and a method that combines single population PRS globally. In simulations, slaPRS outperformed existing approaches and reduced the ancestry dependence of PRS in African Americans. In lipid traits from African British individuals (UK Biobank), slaPRS again improved on single population PRS while performing comparably to the globally combined PRS. slaPRS provides a data-driven and flexible framework to incorporate multiple population-specific GWAS and local ancestry in samples of admixed ancestry.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.24301525">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.24301525" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.24301525">
        <p class="paperTitle">Effect of Paxlovid Treatment on Long COVID Onset: An EHR-Based Target Trial Emulation from N3C</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.24301525" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.24301525" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Preiss, A.; Bhatia, A.; Zang, C.; Aragon, L. V.; Baratta, J. M.; Baskaran, M.; Blancero, F.; Brannock, M. D.; Chew, R. F.; Diaz, I.; Fitzgerald, M.; Kelly, E. P.; Zhou, A. G.; Weiner, M. G.; Carton, T. W.; Wang, F.; Kaushal, R.; Chute, C. G.; Haendel, M.; Moffitt, R.; Pfaff, E.; N3C,  ; RECOVER,  </p>
        <p class="info">Score: 76.3, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.24301525' target='https://doi.org/10.1101/2024.01.20.24301525'> 10.1101/2024.01.20.24301525</a></p>
        <p class="abstract">Preventing and treating post-acute sequelae of SARS-CoV-2 infection (PASC), commonly known as Long COVID, has become a public health priority. In this study, we examined whether treatment with Paxlovid in the acute phase of COVID-19 helps prevent the onset of PASC. We used electronic health records from the National Covid Cohort Collaborative (N3C) to define a cohort of 426,461 patients who had COVID-19 since April 1, 2022, and were eligible for Paxlovid treatment due to risk for progression to severe COVID-19. We used the target trial emulation (TTE) framework to estimate the effect of Paxlovid treatment on PASC incidence. Our primary outcome measure was a PASC computable phenotype. Secondary outcomes were the onset of novel cognitive, fatigue, and respiratory symptoms in the post-acute period. Paxlovid treatment did not have a significant effect on overall PASC incidence (relative risk [RR] = 0.99, 95% confidence interval [CI] 0.96-1.01). However, its effect varied across the cognitive (RR = 0.85, 95% CI 0.79-0.90), fatigue (RR = 0.93, 95% CI 0.89-0.96), and respiratory (RR = 0.99, 95% CI 0.95-1.02) symptom clusters, suggesting that Paxlovid treatment may help prevent post-acute cognitive and fatigue symptoms more than others.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.24302040">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.24302040" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.24302040">
        <p class="paperTitle">Healthy lifestyle for the prevention of post-COVID-19 multisystem sequelae, hospitalization, and death: a prospective cohort study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.24302040" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.24302040" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Y.; Su, B.; Alcalde-Herraiz, M.; Barclay, N. L.; Tian, Y.; Li, C.; Wareham, N. J.; Paredes, R.; Xie, J.; PRIETO-ALHAMBRA, D.</p>
        <p class="info">Score: 8.8, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.24302040' target='https://doi.org/10.1101/2024.01.30.24302040'> 10.1101/2024.01.30.24302040</a></p>
        <p class="abstract">BackgroundPost-COVID complications are emerging as a global public health crisis. Effective prevention strategies are needed to inform patients, clinicians and policy makers, and to reduce their cumulative burden. We aimed to investigate whether a habitual healthy lifestyle predated pandemic is associated with lower risks of multisystem sequelae and other adverse outcomes of COVID-19, and whether the potential protective effects are independent of pre-existing comorbidities.

MethodsThe prospective population-based cohort study enrolled participants with SARS-CoV-2 infection confirmed by a positive polymerase chain reaction test result between March 1, 2020, and March 1, 2022. Participants with no history of the related outcome one year before infection were included and followed up for 210 days. Exposures included ten modifiable healthy lifestyle factors including past or never smoking, moderate alcohol intake ([&amp;le;]4 times week), body mass index &lt;30 kg/m2, at least 150 minutes of moderate or 75 minutes of vigorous physical activity per week, less sedentary time (&lt;4 hours per day), healthy sleep duration (7-9 hours per day), adequate intake of fruit and vegetables ([&amp;ge;]400 g/day), adequate oily fish intake ([&amp;ge;]1 portion/week), moderate intake of red meat ([&amp;le;]4 portions week) and processed meat ([&amp;le;]4 portions week). Outcomes included multisystem COVID-19 sequelae (consisting of 75 diseases/symptoms in 10 organ systems), death, and hospital admission following SARS-CoV-2 infection, confirmed by hospital inpatient and death records. Risk was reported in relative scale (hazard ratio [HR]) and absolute scale (absolute risk reduction [ARR]) during both the acute (the first 30 days) and post-acute (30-210 days) phases of infection using Cox models.

FindingsA total of 68,896 participants (mean [SD] age, 66.6 [8.4]; 32,098 women [46.6%]) with COVID-19 were included. A favorable lifestyle (6-10 healthy lifestyle factors; 46.4%) was associated with a 36% lower risk of multisystem sequelae of COVID-19 (HR, 0.64; 95% CI, 0.58-0.69; ARR, 7.08%; 95% CI, 5.98-8.09), compared with unfavorable lifestyle (0-4 factors; 12.3%). Risk reductions were observed across all 10 prespecified organ systems including cardiovascular, coagulation, metabolic and endocrine, gastrointestinal, kidney, mental health, musculoskeletal, neurologic, and respiratory disorders, and general symptoms of fatigue and malaise. This beneficial effect was largely attributable to direct effects of healthy lifestyle, with mediation proportion ranging from 44% to 93% across organ systems. A favorable lifestyle was also associated with lower risk of post-COVID death (HR, 0.59; 95% CI, 0.52-0.66; ARR, 1.99%; 95% CI, 1.61-2.32) and hospitalization (HR, 0.78; 95% CI, 0.73-0.84; ARR, 6.14%; 95% CI, 4.48-7.68). These associations were observed after accounting for potential misclassification of lifestyle factors, and during acute and post-acute infection, in those tested positive in the hospital and community setting, and independent of vaccination status or SARS-CoV-2 variant.

InterpretationAdherence to a healthy lifestyle predated pandemic was associated with substantially lower risk of complications across organ systems, death, and hospitalization following COVID-19, regardless of phases of infection, vaccination status, test setting, and SARS-CoV-2 variants, and independent of comorbidities. These findings illustrate the benefits of adhering to a healthy lifestyle to reduce the long-term adverse health consequences following SARS-CoV-2 infection.

Research in contextO_ST_ABSEvidence before this studyC_ST_ABSWe searched PubMed and MEDLINE for articles published between March 1, 2020, and December 1, 2023, using the search terms &#34;healthy lifestyle&#34;, &#34;risk factor&#34;, &#34;post-COVID condition&#34;, &#34;long COVID&#34;, &#34;post-acute sequelae&#34;, &#34;prevention&#34;, &#34;management&#34;, and &#34;treatment&#34;, with no language restrictions. Previous evidence on the prevention and management of long COVID has mainly focused on vaccination and pharmaceutical approaches, including antivirals (e.g., molnupiravir and nirmatrelvir) and other drugs (e.g., metformin). Vaccination before infection or use of antivirals in selected high-risk patients during acute infection only partially mediates the risk of COVID-19 sequelae. Evidence for the non-pharmaceutical prevention strategies are lacking. We identified only two publications on the association between healthy lifestyle and post-COVID condition, and one meta-analysis of the risk factors for long COVID symptoms. A cross-sectional study of 1981 women suggested an inverse association between healthy lifestyle factors and self-reported symptoms following infection of non-Omicron variants, which was mainly driven by BMI and sleep duration. Another study suggested an inverse prospective association between healthy lifestyle prior to infection and post-COVID cardiovascular events. High BMI and smoking are risk factors for long COVID mainly in hospitalized patients. We did not find any study that assessed the association between a composite healthy lifestyle and subsequent post-COVID complications or sequelae across organ systems, hospitalization, and death.

Added value of this studyIn a prospective, population-based cohort of 68,896 participants with COVID-19, adherence to a healthy lifestyle prior to infection was associated with a substantially lower risk of multisystem sequelae (by 20%-36%), death (by 26%-41%), and hospital admission (by 13%-22%) following COVID-19. The reduced risk of sequelae was evident across 10 prespecified organ systems, including cardiovascular, coagulation and hematologic, metabolic and endocrine, gastrointestinal, kidney, mental health, musculoskeletal, neurologic, and respiratory disorders, as well as general symptoms of fatigue and malaise. The reduced risk of multisystem sequelae, hospitalization, and death associated with a healthy lifestyle was consistently observed across participants, regardless of their vaccination status, disease severity, and major SARS-CoV-2 variants, and largely independent of relevant comorbidities. Adherence to a healthy lifestyle prior to infection was consistently and directly associated with reduced risk of sequelae and other adverse health outcomes following COVID-19.

Implications of all the available evidenceThe inverse association of healthy lifestyle with multisystem sequelae was even larger than those observed in previous studies of pharmaceutical interventions in non-hospitalized patients. Considering the restricted scope of currently available therapies, such as antivirals (only selected patients at higher risk are qualified during the acute infection) and limited efficacy of vaccination in preventing long COVID, adherence to a healthy lifestyle, in combination with vaccination and, if necessary, potential medications, emerges as practical prevention and care strategies to mitigate the long-term health consequences of SARS-CoV-2 infection. These strategies are of significant clinical and public health importance in reducing the overall burden of post-COVID conditions and improving preparedness for future pandemics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.24301520">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.24301520" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.24301520">
        <p class="paperTitle">Safety of BNT162b2 mRNA COVID-19 vaccine batches: A nationwide cohort study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.24301520" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.24301520" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hviid, A.; Bech Svalgaard, I.</p>
        <p class="info">Score: 18.8, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.24301520' target='https://doi.org/10.1101/2024.01.22.24301520'> 10.1101/2024.01.22.24301520</a></p>
        <p class="abstract">BackgroundThe safety of the BNT162b2 mRNA COVID-19 vaccine has been extensively evaluated since the global rollout began. While serious adverse events are rare, safety issues continue to arise. This study evaluates the claim that earlier small vaccine batches were associated with higher rates of serious adverse events compared to later batches.

MethodsA nationwide cohort study was conducted in Denmark, comprising individuals vaccinated with the BNT162b2 vaccine from 52 pre-defined batches classified into three pre-defined groups. Vaccinated individuals were matched 1:1 between batch groups on age, sex, and vaccination priority group. The study outcomes, included 27 serious adverse events, 2 negative control outcomes and all-cause mortality. Cox regression was used to estimate hazard ratios (HRs) comparing rates between batch groups in the 28-days following vaccination. We conducted two comparisons of the early small batches to two groups of larger batches used later in the pandemic.

ResultsIn the study period, 9,983,448 vaccinations were administered from batches in the three pre-defined groups. Slightly increased rates of arrhythmia were observed in both study comparisons, HRs 1.25 (95% CI,1.05-1.50) and 1.15 (1.00-1.31), respectively, but sensitivity analyses did not robustly support these associations. For the remaining outcomes, increased rates in both study comparisons were not observed.

ConclusionThis nationwide cohort study provides reassurance regarding the safety of the BNT162b2 vaccine across different batches used in Denmark. The findings support the overall safety of the vaccine, with no clinically relevant variations in serious adverse event rates between batches.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.24301656">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.24301656" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.24301656">
        <p class="paperTitle">Inequalities in local funding cuts to environmental and regulatory service expenditure in England from 2009 to 2020</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.24301656" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.24301656" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Murrell, L.; Fahy, K.; Clough, H. E.; Gibb, R.; Zhang, X.; Chattaway, M.; Green, M. A.; Buchan, I.; Barr, B.; Hungerford, D.</p>
        <p class="info">Score: 3.0, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.24301656' target='https://doi.org/10.1101/2024.01.26.24301656'> 10.1101/2024.01.26.24301656</a></p>
        <p class="abstract">BackgroundLocal authorities have been subject to substantial reductions in funding, placing strain on local services. Environmental and Regulatory (ER) services provide essential functions including infectious disease prevention and control via Food Safety and Animal and Public Health Infectious Disease Control services (APHIDC). This study investigates inequalities in local funding cuts to these services.

MethodsWe used a generalised estimating equation model to estimate the annual percent change of ER service expenditure between 2009/10 and 2020/21 in addition to Food Safety and APHIDC expenditure change overall, and as a share of total ER expenditure. Models analysed trends by deprivation level, local authority structure and population density.

ResultsAreas of higher deprivation had the largest reduction in expenditure, with ER and Food Safety and APHIDC cuts of 2% and of 23% respectively, compared to a 1% and 8% reduction in the least deprived areas. The share of ER expenditure spent on Food Safety and APHIDC decreased by 13% in the most deprived authorities compared to 6% in the least deprived areas. Environmental and Regulatory services saw the largest cuts in unitary authorities, declining by 2%. London boroughs had the greatest reductions in Food Safety and APHIDC expenditure, decreasing by 10%. Both ER and Food Safety and APHIDC expenditure decreased with increasing population density.

ConclusionThe unequal distribution of cuts shows the need for increased and equitable investment into these services to enable resilience to emerging infectious disease threats, and to prevent widening of health inequalities.

Key MessagesO_ST_ABSWhat is already known on this topicC_ST_ABSO_LIAusterity measures have led to substantial reductions in local funding placing increased pressure on local authority services in England, research shows reductions vary by deprivation level of an area, rural - urban classification and local authority structure.
C_LIO_LIIt is unknown how local funding cuts to Environmental and Regulatory services, which provide essential services for public health protection, vary by these characteristics.
C_LI

What this study addsO_LIWe investigate inequalities in austerity-enforced changes in Environmental and Regulatory service expenditure and sub-spending lines of Food Safety and Animal and Public Health Infectious Disease Control over time by deprivation, local authority structure and population density from 2009/10 to 2020/21.
C_LIO_LIThe largest cuts were in the more deprived areas and with increased population density for both Environmental and Regulatory and Food safety and Animal and Public Health Infectious Disease Control services. The largest cuts in Environmental and Regulatory services were seen in unitary authorities whereas Food safety and Animal and Public Health Infectious Disease Control saw largest cuts in London boroughs.
C_LI

How this study might affect research, practice or policyO_LIThis research provides strong evidence of inequalities in local authority service expenditure in Environmental and Regulatory services and highlights where investment should be focused, in order to protect environmental and public health.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.03.24302274">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.03.24302274" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.03.24302274">
        <p class="paperTitle">Underestimation of SARS-CoV-2 in wastewater due to single or double mutations in the N1 qPCR probe binding region</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.03.24302274" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.03.24302274" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sun, J.; Yang, M. I.; Peng, J.; Lopez, J. J.; Khan, I.; Chan, R.; Edwards, E. A.; Peng, H.</p>
        <p class="info">Score: 127.4, Published: 2024-02-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.03.24302274' target='https://doi.org/10.1101/2024.02.03.24302274'> 10.1101/2024.02.03.24302274</a></p>
        <p class="abstract">Wastewater surveillance using RT-qPCR has now been widely adopted to track circulating levels of SARS-CoV-2 virus in many sewer sheds. The CDC qPCR assays targeting two regions (N1 and N2) within the N gene are commonly used, but a discrepancy between the two biomarkers has been noticed by many groups using this method since late 2021. The reason is presumed to be due to mutations in regions targeted by the qPCR probe. In this study, we systematically investigated and unequivocally confirmed that the underlying reason for this discrepancy was mutations in the N1 probe target, and that a single mutation could cause a significant drop in signal. We first confirmed the proportion of related mutations in wastewater samples (Jan 2021-Dec 2022) using nested PCR and LC-MS. Based on relative proportion of N1 alleles, we separated the wastewater data into four time periods corresponding to different variant waves: Period I (Alpha and Delta waves with 0 mutation), Period II (BA.1/BA.2 wave with a single mutation found in all Omicron strains), Period III (BA. 5.2* wave with two mutations), and Period IV (BQ.1* wave with two mutations). Significantly lower N1 copies relative to N2 copies in samples from Periods II-IV compared to those from Period I was observed in wastewater. To further pinpoint the extent to which each mutation impacted N1 quantification, we compared the qPCR response among different synthetic oligomers with corresponding mutations. This study highlighted the impact of even just one or two mutations on qPCR-based wastewater surveillance of SARS-CoV-2.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
