<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>epidemiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>epidemiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Influence of Prior SARS-CoV-2 Infection on COVID-19 Severity: Evidence from the National COVID Cohort Collaborative
Authors: Hendrix, N.; Sidky, H.; Sahner, D.; N3C Consortium, Score: 363.7, Published: 2023-08-06 DOI: 10.1101/2023.08.03.23293612
BackgroundAs SARS-CoV-2 has transitioned from a pandemic to endemic disease, the majority of new infections have been among previously infected individuals. To manage the risks and benefits of ongoing COVID-19 policies, it is important to understand whether prior infection modifies the severity of subsequent infections.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/epidemiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="epidemiology" />
<meta property="og:description" content="Influence of Prior SARS-CoV-2 Infection on COVID-19 Severity: Evidence from the National COVID Cohort Collaborative
Authors: Hendrix, N.; Sidky, H.; Sahner, D.; N3C Consortium, Score: 363.7, Published: 2023-08-06 DOI: 10.1101/2023.08.03.23293612
BackgroundAs SARS-CoV-2 has transitioned from a pandemic to endemic disease, the majority of new infections have been among previously infected individuals. To manage the risks and benefits of ongoing COVID-19 policies, it is important to understand whether prior infection modifies the severity of subsequent infections." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/epidemiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-09T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-08-09T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="epidemiology"/>
<meta name="twitter:description" content="Influence of Prior SARS-CoV-2 Infection on COVID-19 Severity: Evidence from the National COVID Cohort Collaborative
Authors: Hendrix, N.; Sidky, H.; Sahner, D.; N3C Consortium, Score: 363.7, Published: 2023-08-06 DOI: 10.1101/2023.08.03.23293612
BackgroundAs SARS-CoV-2 has transitioned from a pandemic to endemic disease, the majority of new infections have been among previously infected individuals. To manage the risks and benefits of ongoing COVID-19 policies, it is important to understand whether prior infection modifies the severity of subsequent infections."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "epidemiology",
      "item": "https://trxiv.yorks0n.com/posts/epidemiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "epidemiology",
  "name": "epidemiology",
  "description": "Influence of Prior SARS-CoV-2 Infection on COVID-19 Severity: Evidence from the National COVID Cohort Collaborative\nAuthors: Hendrix, N.; Sidky, H.; Sahner, D.; N3C Consortium, Score: 363.7, Published: 2023-08-06 DOI: 10.1101/2023.08.03.23293612\nBackgroundAs SARS-CoV-2 has transitioned from a pandemic to endemic disease, the majority of new infections have been among previously infected individuals. To manage the risks and benefits of ongoing COVID-19 policies, it is important to understand whether prior infection modifies the severity of subsequent infections.",
  "keywords": [
    
  ],
  "articleBody": " Influence of Prior SARS-CoV-2 Infection on COVID-19 Severity: Evidence from the National COVID Cohort Collaborative\nAuthors: Hendrix, N.; Sidky, H.; Sahner, D.; N3C Consortium, Score: 363.7, Published: 2023-08-06 DOI: 10.1101/2023.08.03.23293612\nBackgroundAs SARS-CoV-2 has transitioned from a pandemic to endemic disease, the majority of new infections have been among previously infected individuals. To manage the risks and benefits of ongoing COVID-19 policies, it is important to understand whether prior infection modifies the severity of subsequent infections. MethodsWe used data from first and second COVID-19 episodes in the National COVID Cohort Collaborative (N3C), a collection of health systems who provide de-identified electronic health records for research purposes. Our analysis was a sequential series of nested trial emulations. In the first of two analytic stages, we created a month-specific model of the probability of prior infection for each individual. In the second stage, we used an ordinal logistic regression with inverse probability weights calculated in the first stage to simulate a series of monthly trials comparing severity between the cohorts of first and second infections. In addition to cohort-wide effect estimates, we also conducted analyses among race/ethnicity, sex, and age subgroups. ResultsFrom an initial cohort of 7,446,481 combined first and second infections, we identified a cohort of 2,227,484 infections, among which 7.6% were second infections. Ninety-four percent of patients with two recorded infections experienced mild disease for both. The overall odds ratio (OR) for more severe disease with prior infection was 1.06 (95% confidence interval [CI]: 1.03 - 1.10). Monthly point estimates of the OR ranged from 0.56 (95% CI: 0.37 - 0.84) in October 2020 to 1.64 (95% CI: 1.33 - 2.00) in February 2023. In most subgroups, the effect of prior infection was significant. In 8 out of 10 subgroups, the maximum monthly OR occurred after the minimum monthly OR, suggesting that protection has waned throughout the pandemic. ConclusionOverall, prior infection was associated with a significant slightly elevated risk of severe disease. This effect varied month to month. As the pandemic proceeded, the effect of prior infection tended to evolve from generally protective during the pre-Omicron era to unprotective during the Omicron era. This points to the need for continued strategies to avert and minimize the harms of COVID-19, rather than relying upon immunity acquired through previous infection. Key PointsO_ST_ABSQuestionC_ST_ABSDoes prior infection with SARS-CoV-2 affect the severity of subsequent COVID-19 episodes? FindingsWe observed a mild protective effect of prior infection during the early and mid-stages of the pandemic that waned after the rise of the Omicron variants, ultimately resulting in loss of protection or a tendency toward more severe second infections. MeaningPrior infection alone is likely not enough to avert the worst public health harms of endemic SARS-CoV-2. Interventions to avoid infection and reduce the severity of COVID-19 will still be important in the post-pandemic era.\nPrevalence of long-term symptoms varies by using different post-COVID-19 definitions in positively and negatively tested adults: the PRIME post-COVID study\nAuthors: Pagen, D. M.; van Bilsen, C. J.; Brinkhues, S.; Van Herck, M.; Konings, K.; den Heijer, C. D.; ter Waarbeek, H. L.; Spruit, M. A.; Hoebe, C. J.; Dukers-Muijrers, N. H.\nScore: 77.4, Published: 2023-08-02 DOI: 10.1101/2023.07.27.23293244\nBackgroundLong-term symptoms after a SARS-CoV-2 infection (i.e., post-COVID-19 condition or long COVID), constitute a substantial public health problem. Yet, the prevalence remains currently unclear as different case definitions are used, and negatively tested controls are lacking. We aimed to estimate post-COVID-19 condition prevalence using six definitions. MethodsThe Prevalence, Risk factors, and Impact Evaluation (PRIME) post-COVID-19 condition study is a population-based sample of COVID-19 tested adults. End 2021, 61,655 adults were invited to complete an online questionnaire, including 44 symptoms plus a severity score (0-10) per symptom. The prevalence was calculated in both positively and negatively tested adults, stratified by time since their COVID-19 test (3-5, 6-11 or [\u0026ge;]12 months ago). ResultsIn positives (n=7,405; 75.6%), the prevalence of long-term symptoms was between 26.9% and 64.1% using the six definitions, while in negatives (n=2,392; 24.4%) the prevalence varied between 11.4% and 32.5%. The prevalence of long-term symptoms potentially accountable to COVID-19 ranged from 17.9% to 26.3%. ConclusionThere is a (substantial) variation in prevalence estimates by using different definitions as is current practice, showing limited overlap between definitions, indicating that the essential post-COVID-19 condition criteria are still unclear. Including negatives is important to determine long-term symptoms accountable to COVID-19. Trial registration ClinicalTrials.gov Identifier: NCT05128695.\nPost-acute health care costs following SARS-CoV-2 infection: A retrospective cohort study of among 531,182 matched adults\nAuthors: McNaughton, C. D.; Austin, P. C.; Li, Z.; Sivaswamy, A.; Fang, T.; Abdel-Qadir, H.; Udell, J. A.; Wodchis, W.; Lee, D. S.; Mostarac, I.; Atzema, C. L.\nScore: 22.3, Published: 2023-08-04 DOI: 10.1101/2023.08.02.23293563\nPost-acute health care costs following SARS-CoV-2 infection are not known. Beginning 56 days following SARS-CoV-2 polymerase chain reaction (PCR) testing, we compared person-specific total and component health care costs across their distribution for the following year (test-positive versus test-negative, matched people; January 1, 2020-March 31, 2021). For 531,182 individuals, mean person-specific total health care costs were $513.83 (95% CI $387.37-$638.40) higher for test-positive females and $459.10 (95% CI $304.60-$615.32) higher for test-positive males, or \u003e10% increase in mean per-capita costs, driven by hospitalization, long-term care, and complex continuing care costs. At the 99th percentile of each subgroup, person-specific health care costs were $12,533.00 (95% CI $9,008.50-$16,473.00) higher for test-positive females and $14,604.00 (95% CI $9,565.50-$19,506.50) for test-positive males, driven by hospitalization, specialist (males), and homecare costs (females). Cancer costs were lower. Six-month and 1-year costs differences were similar. These findings can inform planning for post-acute SARS-CoV-2 health care costs.\nIdentifying proteomic risk factors for cancer using prospective and exome analyses: 1,463 circulating proteins and risk of 19 cancers in the UK Biobank\nAuthors: Papier, K.; Atkins, J. R.; Tong, T. Y.; Gaitskell, K.; Desai, T.; Ogamba, C. F.; Parsaeian, M.; Reeves, G. K.; Mills, I. G.; Key, T. J.; Smith-Byrne, K.; Travis, R. C.\nScore: 20.1, Published: 2023-08-05 DOI: 10.1101/2023.07.28.23293330\nBackgroundProteins are essential for the development and progression of cancer and for the human bodys defense against tumor onset. The availability of a large panel of protein measurements and whole exome sequence data in the UK Biobank has enabled the simultaneous examination of plasma protein associations with risk across multiple cancer sites and their potential role in cancer etiology. MethodsWe investigated the associations of plasma proteins with incidence of 19 cancers and 9 cancer subsites in up to 44,645 middle-aged adults in the UK Biobank, who had measurements of 1,463 plasma proteins generated using Olink Explore Proximity Extension Assay in baseline blood samples (2006-2010). Using multivariable-adjusted Cox regression, we estimated the risk of each protein with each cancer overall and by time-to-diagnosis after correction for multiple-testing. Identified protein-cancer associations were further assessed in an analysis of cancer risk using cis-pQTL and exome-wide protein genetic scores (exGS) in all UK Biobank participants (n=337,543). ResultsWe identified 371 proteins associated with the risk of at least one incident cancer, represented by a total of 621 protein-cancer associations. These proteins were associated with cancers of the blood (201 proteins), liver (131), kidney (51), lung (28), esophagus (22), colorectum (15), stomach (8), breast (5), prostate (3), endometrium (3), ovary (2), bladder (1), head and neck (1), and brain (1). 100 of these 621 protein-cancer associations persisted for cases diagnosed more than seven years after blood draw. Of these 621 associations, there was further support from cis-pQTL analyses for the etiological role of TNFRSF14 in risk of non-Hodgkin lymphoma (NHL), and from whole exome protein score (exGS) analyses for 28 other protein-cancer associations, including SRP14 and risk of leukemia. Proteins with directionally concordant evidence from long time-to-diagnosis analyses and from both cis-pQTL and exGS analyses were SFTPA2 for lung cancer, TNFRSF1B and CD74 for NHL, and ADAM8 for leukemia. ConclusionsFor the first time using an integrated multi-omics and cross-cancer approach, we have comprehensively assessed the plasma proteome in relation to cancer risk and identified multiple novel etiological candidates. Differences in the levels of many circulating proteins were detectable more than seven years before cancer diagnosis; while some of these are likely to be markers of early cancer processes that may inform risk stratification, and/or risk factors, concordant evidence from genetic analyses suggests that some may have a role in cancer development.\nLineage replacement and evolution captured by the United Kingdom Covid Infection Survey\nAuthors: Lythgoe, K. A.; Golubchik, T.; Hall, M.; House, T.; Cahuantzi, R.; MacIntyre-Cockett, G.; Fryer, H.; Thomson, L.; Nurtay, A.; Ghafari, M.; Buck, D.; Green, A.; Trebes, A.; Piazza, P.; Lonie, L. J.; Studley, R.; Rourke, E.; Smith, D.; Bashton, M.; Nelson, A.; Crown, M.; McCann, C.; Young, G. R.; de Santos, R. A. N.; Richards, Z.; Tariq, A.; Wellcome Sanger Institute COVID-19 Surveillance Team, ; COVID-19 Infection Survey Group, ; The COVID-19 Genomics UK (COG-UK) consortium, ; Fraser, C.; Diamond, I.; Barrett, J.; Walker, A. S.; Bonsall, D.\nScore: 30.5, Published: 2023-07-25 DOI: 10.1101/2022.01.05.21268323\nThe Office for National Statistics COVID-19 Infection Survey (ONS-CIS) is the largest surveillance study of SARS-CoV-2 positivity in the community, and collected data on the United Kingdom (UK) epidemic from April 2020 until March 2023 before being paused. Here, we report on the epidemiological and evolutionary dynamics of SARS-CoV-2 determined by analysing the sequenced samples collected by the ONS-CIS during this period. We observed a series of sweeps or partial sweeps, with each sweeping lineage having a distinct growth advantage compared to their predecessors. The sweeps also generated an alternating pattern in which most samples had either S-gene target failure (SGTF) or non- SGTF over time. Evolution was characterised by steadily increasing divergence and diversity within lineages, but with step increases in divergence associated with each sweeping major lineage. This led to a faster overall rate of evolution when measured at the between-lineage level compared to within lineages, and fluctuating levels of diversity. These observations highlight the value of viral sequencing integrated into community surveillance studies to monitor the viral epidemiology and evolution of SARS-CoV-2, and potentially other pathogens, particularly in the current phase of the pandemic with routine RT-PCR testing now ended in the community.\nBlood protein levels predict leading incident diseases and mortality in UK Biobank\nAuthors: Gadd, D. A.; Hillary, R. F.; Kuncheva, Z.; Mangelis, T.; Cheng, Y.; Dissanayake, M.; Admanit, R.; Gagnon, J.; Lin, T.; Ferber, K.; Runz, H.; Biogen Biobank Team, ; Marioni, R. E.; Foley, C. N.; Sun, B. B.\nScore: 20.0, Published: 2023-08-04 DOI: 10.1101/2023.05.01.23288879\nThe circulating proteome offers insights into the biological pathways that underlie disease. Here, we test relationships between 1,468 Olink protein levels and the incidence of 23 age-related diseases and mortality, over 16 years of electronic health linkage in the UK Biobank (N=47,600). We report 3,201 associations between 961 protein levels and 21 incident outcomes, identifying proteomic indicators of multiple morbidities. Next, protein-based scores (ProteinScores) are developed using penalised Cox regression. When applied to test sets, six ProteinScores improve Area Under the Curve (AUC) estimates for the 10-year onset of incident outcomes beyond age, sex and a comprehensive set of 24 lifestyle factors, clinically-relevant biomarkers and physical measures. Furthermore, the ProteinScore for type 2 diabetes outperformed a polygenic risk score, a metabolomic score and HbA1c - a clinical marker used to monitor and diagnose type 2 diabetes. These data characterise early proteomic contributions to major age-related disease and demonstrate the value of the plasma proteome for risk stratification.\nUS Exceptionalism? International Trends in Midlife Mortality\nAuthors: Dowd, J. B.; Doniec, K.; Zhang, L.; Tilstra, A.\nScore: 9.8, Published: 2023-07-27 DOI: 10.1101/2023.07.25.23293099\nBackgroundRising midlife mortality in the United States (US) has raised concerns, particularly the increase in \"deaths of despair\" (due to drugs, alcohol, and suicide). While life expectancy is also stalling in other countries such as the UK, whether midlife mortality is rising outside the US is not known. MethodsWe document trends in midlife mortality in the US, UK and a group of 16 high-income countries in Western Europe, Australia, Canada, New Zealand, and Japan, as well as 7 Central and Eastern European (CEE) countries from 1990-2019. We use annual mortality data from the World Health Organization Mortality Database to analyze sex and age-specific (25-44, 45-54, and 55-64) age-standardized death rates across 13 major cause-of-death categories. FindingsUS midlife mortality rates worsened since 1990 for several causes of death including drug- related, alcohol-related, suicide, metabolic disease, nervous system disease, respiratory disease, and infectious/parasitic diseases. Deaths due to homicide, transport accidents, and cardiovascular disease declined overall since 1990 but saw recent increases or stalling of improvements. Midlife mortality has also recently increased in the UK for 45-54-year-olds, and in Canada, Poland, and Sweden among 25-44-year-olds. ConclusionThe US is increasingly falling behind not only high-income but also CEE countries heavily impacted by the post-Soviet mortality crisis of the 1990s. While levels of midlife mortality in the UK are substantially lower than in the US overall, there are signs that UK midlife mortality is worsening relative to the rest of Europe.\nExploring pleiotropy in Mendelian randomisation analyses: What are genetic variants associated with \"cigarette smoking initiation\" really capturing?\nAuthors: Reed, Z. E.; Wootton, R. E.; Khouja, J. N.; Richardson, T. G.; Sanderson, E. G.; Davey Smith, G.; Munafo, M. R.\nScore: 6.8, Published: 2023-08-08 DOI: 10.1101/2023.08.04.23293638\nBackground: Genetic variants used as instruments for exposures in Mendelian randomisation (MR) analyses may also have horizontal pleiotropic effects (i.e., influence outcomes via pathways other than through the exposure), which can undermine the validity of results. We examined the extent to which horizontal pleiotropy may be present, using smoking behaviours as an example. Methods: We first ran a phenome-wide association study in UK Biobank, using a genetic instrument for smoking initiation. From the most strongly associated phenotypes, we selected those that we considered could either plausibly or not plausibly be caused by smoking. We next examined the association between genetic instruments for smoking initiation, smoking heaviness and lifetime smoking and these phenotypes in both UK Biobank and the Avon Longitudinal Study of Parents and Children (ALSPAC). We conducted negative control analyses among never smokers, including children in ALSPAC. Results: We found evidence that smoking-related genetic instruments (mainly for smoking initiation and lifetime smoking) were associated with phenotypes not plausibly caused by smoking in UK Biobank and (to a lesser extent) ALSPAC, although this may reflect the much smaller sample size in ALSPAC. We also observed associations with several phenotypes among never smokers. Conclusion: Our results suggest that genetic instruments for smoking-related phenotypes demonstrate horizontal pleiotropy. When using genetic variants - particularly those for complex behavioural exposures - in genetically-informed causal inference analyses (e.g., MR) it is important to include negative control outcomes where possible, and other triangulation approaches, to avoid arriving at incorrect conclusions.\nFacilitating youth diabetes studies with the most comprehensive epidemiological dataset available through a public web portal\nAuthors: McDonough, C.; Li, Y. C.; Vangeepuram, N.; Liu, B.; Pandey, G.\nScore: 4.2, Published: 2023-08-04 DOI: 10.1101/2023.08.02.23293517\nThe prevalence of type 2 diabetes mellitus (DM) and prediabetes (preDM) is rapidly increasing among youth, posing significant health and economic consequences. To address this growing concern, we created the most comprehensive youth-focused diabetes dataset to date derived from National Health and Nutrition Examination Survey (NHANES) data from 1999 to 2018. The dataset, consisting of 15,149 youth aged 12 to 19 years, encompasses preDM/DM relevant variables from sociodemographic, health status, diet, and other lifestyle behavior domains. An interactive web portal, POND (Prediabetes/diabetes in youth ONline Dashboard), was developed to provide public access to the dataset, allowing users to explore variables potentially associated with youth preDM/DM. Leveraging statistical and machine learning methods, we conducted two case studies, revealing established and lesser-known variables linked to youth preDM/DM. This dataset and portal can facilitate future studies to inform prevention and management strategies for youth prediabetes and diabetes.\nVaccine effectiveness against hospitalisation and comparative odds of hospital admission and severe outcomes with BQ.1, CH.1.1. and XBB.1.5 in England.\nAuthors: Kirsebom, F. C. M.; Harman, K.; Lunt, R. J.; Andrews, N.; Groves, N.; Aziz, N. I. B. A.; Hope, R.; Stowe, J.; Chand, M.; Ramsay, M.; Dabrera, G.; Kall, M.; Lopez Bernal, J.\nScore: 3.1, Published: 2023-08-05 DOI: 10.1101/2023.07.28.23293333\nBackgroundSince the first emergence of Omicron BA.1 in England in November 2021, numerous sub-lineages have evolved. In September 2022, BA.5 dominated. The prevalence of BQ.1 increased from October, while the prevalence of CH.1.1 and XBB.1.5 increased from December 2022 and January 2023, respectively. Little is known about the effectiveness of the vaccines against hospitalisation with these sub-lineages, nor the relative severity. MethodsA test-negative case-control study was used to estimate the incremental effectiveness of the bivalent BA.1 booster vaccines against hospitalisation, relative to those with waned immunity where the last dose was at least 6 months prior. The odds of hospital admission for those testing PCR positive on the day of an attendance to accident and emergency departments and the odds of intensive care unit admission or death amongst COVID-19 admissions were compared between variants. Additionally, a Cox proportional hazards survival regression was used to investigate length of stay amongst hospitalised cases by variant. FindingsThere was no difference in incremental vaccine effectiveness against hospitalisation with BQ.1, CH.1.1 or XBB.1.5, nor was there a difference in the severity of these variants. Effectiveness against hospitalisation was 48.0% (95% C.I.; 38.5-56.0%), 29.7% (95% C.I.; 7.5-46.6%) and 52.7% (95% C.I.; 24.6-70.4%) against BQ.1, CH.1.1 and XBB.1.5, respectively, at 5 to 9 weeks post booster vaccination. Compared to BQ.1, the odds of hospital admission were 0.87 (95% C.I.; 0.77-0.99) and 0.88 (95% C.I.; 0.75-1.02) for CH.1.1 and XBB.1.5 cases attending accident and emergency departments, respectively. There was no significant difference in the odds of admission to intensive care units or death for those with CH.1.1 (OR 0.96, 95% C.I.; 0.71-1.30) or XBB.1.5 (OR 0.67, 95% C.I.; 0.44-1.02) compared to BQ.1. There was also no significant difference in the length of hospital stay by variant. InterpretationTogether, these results provide reassuring evidence that the bivalent BA.1 booster vaccines provide similar protection against hospitalisation with BQ.1, CH.1.1 and XBB.1.5, and that the emergent CH.1.1 and XBB.1.5 sub-lineages do not cause more severe disease than BQ.1. FundingNone.\n",
  "wordCount" : "2998",
  "inLanguage": "en",
  "datePublished": "2023-08-09T10:39:33Z",
  "dateModified": "2023-08-09T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/epidemiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      epidemiology
    </h1>
    <div class="post-meta"><span>updated on August 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.23293612">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.23293612" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.23293612">
        <p class="paperTitle">Influence of Prior SARS-CoV-2 Infection on COVID-19 Severity: Evidence from the National COVID Cohort Collaborative</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.23293612" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.23293612" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hendrix, N.; Sidky, H.; Sahner, D.; N3C Consortium,  </p>
        <p class="info">Score: 363.7, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.23293612' target='https://doi.org/10.1101/2023.08.03.23293612'> 10.1101/2023.08.03.23293612</a></p>
        <p class="abstract">BackgroundAs SARS-CoV-2 has transitioned from a pandemic to endemic disease, the majority of new infections have been among previously infected individuals. To manage the risks and benefits of ongoing COVID-19 policies, it is important to understand whether prior infection modifies the severity of subsequent infections.

MethodsWe used data from first and second COVID-19 episodes in the National COVID Cohort Collaborative (N3C), a collection of health systems who provide de-identified electronic health records for research purposes. Our analysis was a sequential series of nested trial emulations. In the first of two analytic stages, we created a month-specific model of the probability of prior infection for each individual. In the second stage, we used an ordinal logistic regression with inverse probability weights calculated in the first stage to simulate a series of monthly trials comparing severity between the cohorts of first and second infections. In addition to cohort-wide effect estimates, we also conducted analyses among race/ethnicity, sex, and age subgroups.

ResultsFrom an initial cohort of 7,446,481 combined first and second infections, we identified a cohort of 2,227,484 infections, among which 7.6% were second infections. Ninety-four percent of patients with two recorded infections experienced mild disease for both. The overall odds ratio (OR) for more severe disease with prior infection was 1.06 (95% confidence interval [CI]: 1.03 - 1.10). Monthly point estimates of the OR ranged from 0.56 (95% CI: 0.37 - 0.84) in October 2020 to 1.64 (95% CI: 1.33 - 2.00) in February 2023. In most subgroups, the effect of prior infection was significant. In 8 out of 10 subgroups, the maximum monthly OR occurred after the minimum monthly OR, suggesting that protection has waned throughout the pandemic.

ConclusionOverall, prior infection was associated with a significant slightly elevated risk of severe disease. This effect varied month to month. As the pandemic proceeded, the effect of prior infection tended to evolve from generally protective during the pre-Omicron era to unprotective during the Omicron era. This points to the need for continued strategies to avert and minimize the harms of COVID-19, rather than relying upon immunity acquired through previous infection.

Key PointsO_ST_ABSQuestionC_ST_ABSDoes prior infection with SARS-CoV-2 affect the severity of subsequent COVID-19 episodes?

FindingsWe observed a mild protective effect of prior infection during the early and mid-stages of the pandemic that waned after the rise of the Omicron variants, ultimately resulting in loss of protection or a tendency toward more severe second infections.

MeaningPrior infection alone is likely not enough to avert the worst public health harms of endemic SARS-CoV-2. Interventions to avoid infection and reduce the severity of COVID-19 will still be important in the post-pandemic era.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.23293244">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.23293244" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.23293244">
        <p class="paperTitle">Prevalence of long-term symptoms varies by using different post-COVID-19 definitions in positively and negatively tested adults: the PRIME post-COVID study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.23293244" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.23293244" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pagen, D. M.; van Bilsen, C. J.; Brinkhues, S.; Van Herck, M.; Konings, K.; den Heijer, C. D.; ter Waarbeek, H. L.; Spruit, M. A.; Hoebe, C. J.; Dukers-Muijrers, N. H.</p>
        <p class="info">Score: 77.4, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.23293244' target='https://doi.org/10.1101/2023.07.27.23293244'> 10.1101/2023.07.27.23293244</a></p>
        <p class="abstract">BackgroundLong-term symptoms after a SARS-CoV-2 infection (i.e., post-COVID-19 condition or long COVID), constitute a substantial public health problem. Yet, the prevalence remains currently unclear as different case definitions are used, and negatively tested controls are lacking. We aimed to estimate post-COVID-19 condition prevalence using six definitions.

MethodsThe Prevalence, Risk factors, and Impact Evaluation (PRIME) post-COVID-19 condition study is a population-based sample of COVID-19 tested adults. End 2021, 61,655 adults were invited to complete an online questionnaire, including 44 symptoms plus a severity score (0-10) per symptom. The prevalence was calculated in both positively and negatively tested adults, stratified by time since their COVID-19 test (3-5, 6-11 or [&amp;ge;]12 months ago).

ResultsIn positives (n=7,405; 75.6%), the prevalence of long-term symptoms was between 26.9% and 64.1% using the six definitions, while in negatives (n=2,392; 24.4%) the prevalence varied between 11.4% and 32.5%. The prevalence of long-term symptoms potentially accountable to COVID-19 ranged from 17.9% to 26.3%.

ConclusionThere is a (substantial) variation in prevalence estimates by using different definitions as is current practice, showing limited overlap between definitions, indicating that the essential post-COVID-19 condition criteria are still unclear. Including negatives is important to determine long-term symptoms accountable to COVID-19.

Trial registration ClinicalTrials.gov Identifier: NCT05128695.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.23293563">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.23293563" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.23293563">
        <p class="paperTitle">Post-acute health care costs following SARS-CoV-2 infection: A retrospective cohort study of among 531,182 matched adults</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.23293563" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.23293563" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McNaughton, C. D.; Austin, P. C.; Li, Z.; Sivaswamy, A.; Fang, T.; Abdel-Qadir, H.; Udell, J. A.; Wodchis, W.; Lee, D. S.; Mostarac, I.; Atzema, C. L.</p>
        <p class="info">Score: 22.3, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.23293563' target='https://doi.org/10.1101/2023.08.02.23293563'> 10.1101/2023.08.02.23293563</a></p>
        <p class="abstract">Post-acute health care costs following SARS-CoV-2 infection are not known. Beginning 56 days following SARS-CoV-2 polymerase chain reaction (PCR) testing, we compared person-specific total and component health care costs across their distribution for the following year (test-positive versus test-negative, matched people; January 1, 2020-March 31, 2021). For 531,182 individuals, mean person-specific total health care costs were $513.83 (95% CI $387.37-$638.40) higher for test-positive females and $459.10 (95% CI $304.60-$615.32) higher for test-positive males, or &gt;10% increase in mean per-capita costs, driven by hospitalization, long-term care, and complex continuing care costs. At the 99th percentile of each subgroup, person-specific health care costs were $12,533.00 (95% CI $9,008.50-$16,473.00) higher for test-positive females and $14,604.00 (95% CI $9,565.50-$19,506.50) for test-positive males, driven by hospitalization, specialist (males), and homecare costs (females). Cancer costs were lower. Six-month and 1-year costs differences were similar. These findings can inform planning for post-acute SARS-CoV-2 health care costs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.23293330">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.23293330" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.23293330">
        <p class="paperTitle">Identifying proteomic risk factors for cancer using prospective and exome analyses: 1,463 circulating proteins and risk of 19 cancers in the UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.23293330" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.23293330" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Papier, K.; Atkins, J. R.; Tong, T. Y.; Gaitskell, K.; Desai, T.; Ogamba, C. F.; Parsaeian, M.; Reeves, G. K.; Mills, I. G.; Key, T. J.; Smith-Byrne, K.; Travis, R. C.</p>
        <p class="info">Score: 20.1, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.23293330' target='https://doi.org/10.1101/2023.07.28.23293330'> 10.1101/2023.07.28.23293330</a></p>
        <p class="abstract">BackgroundProteins are essential for the development and progression of cancer and for the human bodys defense against tumor onset. The availability of a large panel of protein measurements and whole exome sequence data in the UK Biobank has enabled the simultaneous examination of plasma protein associations with risk across multiple cancer sites and their potential role in cancer etiology.

MethodsWe investigated the associations of plasma proteins with incidence of 19 cancers and 9 cancer subsites in up to 44,645 middle-aged adults in the UK Biobank, who had measurements of 1,463 plasma proteins generated using Olink Explore Proximity Extension Assay in baseline blood samples (2006-2010). Using multivariable-adjusted Cox regression, we estimated the risk of each protein with each cancer overall and by time-to-diagnosis after correction for multiple-testing. Identified protein-cancer associations were further assessed in an analysis of cancer risk using cis-pQTL and exome-wide protein genetic scores (exGS) in all UK Biobank participants (n=337,543).

ResultsWe identified 371 proteins associated with the risk of at least one incident cancer, represented by a total of 621 protein-cancer associations. These proteins were associated with cancers of the blood (201 proteins), liver (131), kidney (51), lung (28), esophagus (22), colorectum (15), stomach (8), breast (5), prostate (3), endometrium (3), ovary (2), bladder (1), head and neck (1), and brain (1). 100 of these 621 protein-cancer associations persisted for cases diagnosed more than seven years after blood draw. Of these 621 associations, there was further support from cis-pQTL analyses for the etiological role of TNFRSF14 in risk of non-Hodgkin lymphoma (NHL), and from whole exome protein score (exGS) analyses for 28 other protein-cancer associations, including SRP14 and risk of leukemia. Proteins with directionally concordant evidence from long time-to-diagnosis analyses and from both cis-pQTL and exGS analyses were SFTPA2 for lung cancer, TNFRSF1B and CD74 for NHL, and ADAM8 for leukemia.

ConclusionsFor the first time using an integrated multi-omics and cross-cancer approach, we have comprehensively assessed the plasma proteome in relation to cancer risk and identified multiple novel etiological candidates. Differences in the levels of many circulating proteins were detectable more than seven years before cancer diagnosis; while some of these are likely to be markers of early cancer processes that may inform risk stratification, and/or risk factors, concordant evidence from genetic analyses suggests that some may have a role in cancer development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.01.05.21268323">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.01.05.21268323" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.01.05.21268323">
        <p class="paperTitle">Lineage replacement and evolution captured by the United Kingdom Covid Infection Survey</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.01.05.21268323" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.01.05.21268323" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lythgoe, K. A.; Golubchik, T.; Hall, M.; House, T.; Cahuantzi, R.; MacIntyre-Cockett, G.; Fryer, H.; Thomson, L.; Nurtay, A.; Ghafari, M.; Buck, D.; Green, A.; Trebes, A.; Piazza, P.; Lonie, L. J.; Studley, R.; Rourke, E.; Smith, D.; Bashton, M.; Nelson, A.; Crown, M.; McCann, C.; Young, G. R.; de Santos, R. A. N.; Richards, Z.; Tariq, A.; Wellcome Sanger Institute COVID-19 Surveillance Team,  ; COVID-19 Infection Survey Group,  ; The COVID-19 Genomics UK (COG-UK) consortium,  ; Fraser, C.; Diamond, I.; Barrett, J.; Walker, A. S.; Bonsall, D.</p>
        <p class="info">Score: 30.5, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.01.05.21268323' target='https://doi.org/10.1101/2022.01.05.21268323'> 10.1101/2022.01.05.21268323</a></p>
        <p class="abstract">The Office for National Statistics COVID-19 Infection Survey (ONS-CIS) is the largest surveillance study of SARS-CoV-2 positivity in the community, and collected data on the United Kingdom (UK) epidemic from April 2020 until March 2023 before being paused. Here, we report on the epidemiological and evolutionary dynamics of SARS-CoV-2 determined by analysing the sequenced samples collected by the ONS-CIS during this period. We observed a series of sweeps or partial sweeps, with each sweeping lineage having a distinct growth advantage compared to their predecessors. The sweeps also generated an alternating pattern in which most samples had either S-gene target failure (SGTF) or non- SGTF over time. Evolution was characterised by steadily increasing divergence and diversity within lineages, but with step increases in divergence associated with each sweeping major lineage. This led to a faster overall rate of evolution when measured at the between-lineage level compared to within lineages, and fluctuating levels of diversity. These observations highlight the value of viral sequencing integrated into community surveillance studies to monitor the viral epidemiology and evolution of SARS-CoV-2, and potentially other pathogens, particularly in the current phase of the pandemic with routine RT-PCR testing now ended in the community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.01.23288879">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.01.23288879" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.01.23288879">
        <p class="paperTitle">Blood protein levels predict leading incident diseases and mortality in UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.01.23288879" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.01.23288879" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gadd, D. A.; Hillary, R. F.; Kuncheva, Z.; Mangelis, T.; Cheng, Y.; Dissanayake, M.; Admanit, R.; Gagnon, J.; Lin, T.; Ferber, K.; Runz, H.; Biogen Biobank Team,  ; Marioni, R. E.; Foley, C. N.; Sun, B. B.</p>
        <p class="info">Score: 20.0, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.01.23288879' target='https://doi.org/10.1101/2023.05.01.23288879'> 10.1101/2023.05.01.23288879</a></p>
        <p class="abstract">The circulating proteome offers insights into the biological pathways that underlie disease. Here, we test relationships between 1,468 Olink protein levels and the incidence of 23 age-related diseases and mortality, over 16 years of electronic health linkage in the UK Biobank (N=47,600). We report 3,201 associations between 961 protein levels and 21 incident outcomes, identifying proteomic indicators of multiple morbidities. Next, protein-based scores (ProteinScores) are developed using penalised Cox regression. When applied to test sets, six ProteinScores improve Area Under the Curve (AUC) estimates for the 10-year onset of incident outcomes beyond age, sex and a comprehensive set of 24 lifestyle factors, clinically-relevant biomarkers and physical measures. Furthermore, the ProteinScore for type 2 diabetes outperformed a polygenic risk score, a metabolomic score and HbA1c - a clinical marker used to monitor and diagnose type 2 diabetes. These data characterise early proteomic contributions to major age-related disease and demonstrate the value of the plasma proteome for risk stratification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.23293099">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.23293099" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.23293099">
        <p class="paperTitle">US Exceptionalism? International Trends in Midlife Mortality</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.23293099" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.23293099" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dowd, J. B.; Doniec, K.; Zhang, L.; Tilstra, A.</p>
        <p class="info">Score: 9.8, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.23293099' target='https://doi.org/10.1101/2023.07.25.23293099'> 10.1101/2023.07.25.23293099</a></p>
        <p class="abstract">BackgroundRising midlife mortality in the United States (US) has raised concerns, particularly the increase in &#34;deaths of despair&#34; (due to drugs, alcohol, and suicide). While life expectancy is also stalling in other countries such as the UK, whether midlife mortality is rising outside the US is not known.

MethodsWe document trends in midlife mortality in the US, UK and a group of 16 high-income countries in Western Europe, Australia, Canada, New Zealand, and Japan, as well as 7 Central and Eastern European (CEE) countries from 1990-2019. We use annual mortality data from the World Health Organization Mortality Database to analyze sex and age-specific (25-44, 45-54, and 55-64) age-standardized death rates across 13 major cause-of-death categories.

FindingsUS midlife mortality rates worsened since 1990 for several causes of death including drug- related, alcohol-related, suicide, metabolic disease, nervous system disease, respiratory disease, and infectious/parasitic diseases. Deaths due to homicide, transport accidents, and cardiovascular disease declined overall since 1990 but saw recent increases or stalling of improvements. Midlife mortality has also recently increased in the UK for 45-54-year-olds, and in Canada, Poland, and Sweden among 25-44-year-olds.

ConclusionThe US is increasingly falling behind not only high-income but also CEE countries heavily impacted by the post-Soviet mortality crisis of the 1990s. While levels of midlife mortality in the UK are substantially lower than in the US overall, there are signs that UK midlife mortality is worsening relative to the rest of Europe.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.23293638">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.23293638" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.23293638">
        <p class="paperTitle">Exploring pleiotropy in Mendelian randomisation analyses: What are genetic variants associated with &#34;cigarette smoking initiation&#34; really capturing?</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.23293638" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.23293638" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reed, Z. E.; Wootton, R. E.; Khouja, J. N.; Richardson, T. G.; Sanderson, E. G.; Davey Smith, G.; Munafo, M. R.</p>
        <p class="info">Score: 6.8, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.23293638' target='https://doi.org/10.1101/2023.08.04.23293638'> 10.1101/2023.08.04.23293638</a></p>
        <p class="abstract">Background: Genetic variants used as instruments for exposures in Mendelian randomisation (MR) analyses may also have horizontal pleiotropic effects (i.e., influence outcomes via pathways other than through the exposure), which can undermine the validity of results. We examined the extent to which horizontal pleiotropy may be present, using smoking behaviours as an example. Methods: We first ran a phenome-wide association study in UK Biobank, using a genetic instrument for smoking initiation. From the most strongly associated phenotypes, we selected those that we considered could either plausibly or not plausibly be caused by smoking. We next examined the association between genetic instruments for smoking initiation, smoking heaviness and lifetime smoking and these phenotypes in both UK Biobank and the Avon Longitudinal Study of Parents and Children (ALSPAC). We conducted negative control analyses among never smokers, including children in ALSPAC. Results: We found evidence that smoking-related genetic instruments (mainly for smoking initiation and lifetime smoking) were associated with phenotypes not plausibly caused by smoking in UK Biobank and (to a lesser extent) ALSPAC, although this may reflect the much smaller sample size in ALSPAC. We also observed associations with several phenotypes among never smokers. Conclusion: Our results suggest that genetic instruments for smoking-related phenotypes demonstrate horizontal pleiotropy. When using genetic variants - particularly those for complex behavioural exposures - in genetically-informed causal inference analyses (e.g., MR) it is important to include negative control outcomes where possible, and other triangulation approaches, to avoid arriving at incorrect conclusions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.23293517">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.23293517" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.23293517">
        <p class="paperTitle">Facilitating youth diabetes studies with the most comprehensive epidemiological dataset available through a public web portal</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.23293517" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.23293517" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McDonough, C.; Li, Y. C.; Vangeepuram, N.; Liu, B.; Pandey, G.</p>
        <p class="info">Score: 4.2, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.23293517' target='https://doi.org/10.1101/2023.08.02.23293517'> 10.1101/2023.08.02.23293517</a></p>
        <p class="abstract">The prevalence of type 2 diabetes mellitus (DM) and prediabetes (preDM) is rapidly increasing among youth, posing significant health and economic consequences. To address this growing concern, we created the most comprehensive youth-focused diabetes dataset to date derived from National Health and Nutrition Examination Survey (NHANES) data from 1999 to 2018. The dataset, consisting of 15,149 youth aged 12 to 19 years, encompasses preDM/DM relevant variables from sociodemographic, health status, diet, and other lifestyle behavior domains. An interactive web portal, POND (Prediabetes/diabetes in youth ONline Dashboard), was developed to provide public access to the dataset, allowing users to explore variables potentially associated with youth preDM/DM. Leveraging statistical and machine learning methods, we conducted two case studies, revealing established and lesser-known variables linked to youth preDM/DM. This dataset and portal can facilitate future studies to inform prevention and management strategies for youth prediabetes and diabetes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.23293333">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.23293333" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.23293333">
        <p class="paperTitle">Vaccine effectiveness against hospitalisation and comparative odds of hospital admission and severe outcomes with BQ.1, CH.1.1. and XBB.1.5 in England.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.23293333" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.23293333" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kirsebom, F. C. M.; Harman, K.; Lunt, R. J.; Andrews, N.; Groves, N.; Aziz, N. I. B. A.; Hope, R.; Stowe, J.; Chand, M.; Ramsay, M.; Dabrera, G.; Kall, M.; Lopez Bernal, J.</p>
        <p class="info">Score: 3.1, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.23293333' target='https://doi.org/10.1101/2023.07.28.23293333'> 10.1101/2023.07.28.23293333</a></p>
        <p class="abstract">BackgroundSince the first emergence of Omicron BA.1 in England in November 2021, numerous sub-lineages have evolved. In September 2022, BA.5 dominated. The prevalence of BQ.1 increased from October, while the prevalence of CH.1.1 and XBB.1.5 increased from December 2022 and January 2023, respectively. Little is known about the effectiveness of the vaccines against hospitalisation with these sub-lineages, nor the relative severity.

MethodsA test-negative case-control study was used to estimate the incremental effectiveness of the bivalent BA.1 booster vaccines against hospitalisation, relative to those with waned immunity where the last dose was at least 6 months prior. The odds of hospital admission for those testing PCR positive on the day of an attendance to accident and emergency departments and the odds of intensive care unit admission or death amongst COVID-19 admissions were compared between variants. Additionally, a Cox proportional hazards survival regression was used to investigate length of stay amongst hospitalised cases by variant.

FindingsThere was no difference in incremental vaccine effectiveness against hospitalisation with BQ.1, CH.1.1 or XBB.1.5, nor was there a difference in the severity of these variants. Effectiveness against hospitalisation was 48.0% (95% C.I.; 38.5-56.0%), 29.7% (95% C.I.; 7.5-46.6%) and 52.7% (95% C.I.; 24.6-70.4%) against BQ.1, CH.1.1 and XBB.1.5, respectively, at 5 to 9 weeks post booster vaccination. Compared to BQ.1, the odds of hospital admission were 0.87 (95% C.I.; 0.77-0.99) and 0.88 (95% C.I.; 0.75-1.02) for CH.1.1 and XBB.1.5 cases attending accident and emergency departments, respectively. There was no significant difference in the odds of admission to intensive care units or death for those with CH.1.1 (OR 0.96, 95% C.I.; 0.71-1.30) or XBB.1.5 (OR 0.67, 95% C.I.; 0.44-1.02) compared to BQ.1. There was also no significant difference in the length of hospital stay by variant.

InterpretationTogether, these results provide reassuring evidence that the bivalent BA.1 booster vaccines provide similar protection against hospitalisation with BQ.1, CH.1.1 and XBB.1.5, and that the emergent CH.1.1 and XBB.1.5 sub-lineages do not cause more severe disease than BQ.1.

FundingNone.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
