<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>epidemiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>epidemiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Lineage replacement and evolution captured by the United Kingdom Covid Infection Survey
Authors: Lythgoe, K. A.; Golubchik, T.; Hall, M.; House, T.; Cahuantzi, R.; MacIntyre-Cockett, G.; Fryer, H.; Thomson, L.; Nurtay, A.; Ghafari, M.; Buck, D.; Green, A.; Trebes, A.; Piazza, P.; Lonie, L. J.; Studley, R.; Rourke, E.; Smith, D.; Bashton, M.; Nelson, A.; Crown, M.; McCann, C.; Young, G. R.; de Santos, R. A. N.; Richards, Z.; Tariq, A.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/epidemiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="epidemiology" />
<meta property="og:description" content="Lineage replacement and evolution captured by the United Kingdom Covid Infection Survey
Authors: Lythgoe, K. A.; Golubchik, T.; Hall, M.; House, T.; Cahuantzi, R.; MacIntyre-Cockett, G.; Fryer, H.; Thomson, L.; Nurtay, A.; Ghafari, M.; Buck, D.; Green, A.; Trebes, A.; Piazza, P.; Lonie, L. J.; Studley, R.; Rourke, E.; Smith, D.; Bashton, M.; Nelson, A.; Crown, M.; McCann, C.; Young, G. R.; de Santos, R. A. N.; Richards, Z.; Tariq, A." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/epidemiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-06T10:36:48+00:00" />
<meta property="article:modified_time" content="2023-08-06T10:36:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="epidemiology"/>
<meta name="twitter:description" content="Lineage replacement and evolution captured by the United Kingdom Covid Infection Survey
Authors: Lythgoe, K. A.; Golubchik, T.; Hall, M.; House, T.; Cahuantzi, R.; MacIntyre-Cockett, G.; Fryer, H.; Thomson, L.; Nurtay, A.; Ghafari, M.; Buck, D.; Green, A.; Trebes, A.; Piazza, P.; Lonie, L. J.; Studley, R.; Rourke, E.; Smith, D.; Bashton, M.; Nelson, A.; Crown, M.; McCann, C.; Young, G. R.; de Santos, R. A. N.; Richards, Z.; Tariq, A."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "epidemiology",
      "item": "https://trxiv.yorks0n.com/posts/epidemiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "epidemiology",
  "name": "epidemiology",
  "description": "Lineage replacement and evolution captured by the United Kingdom Covid Infection Survey\nAuthors: Lythgoe, K. A.; Golubchik, T.; Hall, M.; House, T.; Cahuantzi, R.; MacIntyre-Cockett, G.; Fryer, H.; Thomson, L.; Nurtay, A.; Ghafari, M.; Buck, D.; Green, A.; Trebes, A.; Piazza, P.; Lonie, L. J.; Studley, R.; Rourke, E.; Smith, D.; Bashton, M.; Nelson, A.; Crown, M.; McCann, C.; Young, G. R.; de Santos, R. A. N.; Richards, Z.; Tariq, A.",
  "keywords": [
    
  ],
  "articleBody": " Lineage replacement and evolution captured by the United Kingdom Covid Infection Survey\nAuthors: Lythgoe, K. A.; Golubchik, T.; Hall, M.; House, T.; Cahuantzi, R.; MacIntyre-Cockett, G.; Fryer, H.; Thomson, L.; Nurtay, A.; Ghafari, M.; Buck, D.; Green, A.; Trebes, A.; Piazza, P.; Lonie, L. J.; Studley, R.; Rourke, E.; Smith, D.; Bashton, M.; Nelson, A.; Crown, M.; McCann, C.; Young, G. R.; de Santos, R. A. N.; Richards, Z.; Tariq, A.; Wellcome Sanger Institute COVID-19 Surveillance Team, ; COVID-19 Infection Survey Group, ; The COVID-19 Genomics UK (COG-UK) consortium, ; Fraser, C.; Diamond, I.; Barrett, J.; Walker, A. S.; Bonsall, D.\nScore: 27.7, Published: 2023-07-25 DOI: 10.1101/2022.01.05.21268323\nThe Office for National Statistics COVID-19 Infection Survey (ONS-CIS) is the largest surveillance study of SARS-CoV-2 positivity in the community, and collected data on the United Kingdom (UK) epidemic from April 2020 until March 2023 before being paused. Here, we report on the epidemiological and evolutionary dynamics of SARS-CoV-2 determined by analysing the sequenced samples collected by the ONS-CIS during this period. We observed a series of sweeps or partial sweeps, with each sweeping lineage having a distinct growth advantage compared to their predecessors. The sweeps also generated an alternating pattern in which most samples had either S-gene target failure (SGTF) or non- SGTF over time. Evolution was characterised by steadily increasing divergence and diversity within lineages, but with step increases in divergence associated with each sweeping major lineage. This led to a faster overall rate of evolution when measured at the between-lineage level compared to within lineages, and fluctuating levels of diversity. These observations highlight the value of viral sequencing integrated into community surveillance studies to monitor the viral epidemiology and evolution of SARS-CoV-2, and potentially other pathogens, particularly in the current phase of the pandemic with routine RT-PCR testing now ended in the community.\nBlood protein levels predict leading incident diseases and mortality in UK Biobank\nAuthors: Gadd, D. A.; Hillary, R. F.; Kuncheva, Z.; Mangelis, T.; Cheng, Y.; Dissanayake, M.; Admanit, R.; Gagnon, J.; Lin, T.; Ferber, K.; Runz, H.; Biogen Biobank Team, ; Marioni, R. E.; Foley, C. N.; Sun, B. B.\nScore: 19.1, Published: 2023-08-04 DOI: 10.1101/2023.05.01.23288879\nThe circulating proteome offers insights into the biological pathways that underlie disease. Here, we test relationships between 1,468 Olink protein levels and the incidence of 23 age-related diseases and mortality, over 16 years of electronic health linkage in the UK Biobank (N=47,600). We report 3,201 associations between 961 protein levels and 21 incident outcomes, identifying proteomic indicators of multiple morbidities. Next, protein-based scores (ProteinScores) are developed using penalised Cox regression. When applied to test sets, six ProteinScores improve Area Under the Curve (AUC) estimates for the 10 year onset of incident outcomes beyond age, sex and a comprehensive set of 24 lifestyle factors, clinically relevant biomarkers and physical measures. Furthermore, the ProteinScore for type 2 diabetes outperformed a polygenic risk score, a metabolomic score and HbA1c, a clinical marker used to monitor and diagnose type 2 diabetes. These data characterise early proteomic contributions to major age-related disease and demonstrate the value of the plasma proteome for risk stratification.\nInteractions between climate change, urban infrastructure and mobility are driving dengue emergence in Vietnam\nAuthors: Gibb, R.; Colon-Gonzalez, F. J.; Lan, P. T.; Huong, P. T.; Nam, V. S.; Duoc, V. T.; Do, H. T.; Dong, N. T.; Chien, V. C.; Trang, L. T. T.; Quoc, D. K.; Hoa, T. M.; Tai, N. H.; Hang, T. T.; Tsarouchi, G.; Ainscoe, E.; Harpham, Q.; Hofmann, B.; Lumbroso, D.; Brady, O. J.; Lowe, R.\nScore: 7.0, Published: 2023-08-01 DOI: 10.1101/2023.07.25.23293110\nDengue is expanding globally, but how dengue emergence is shaped locally by interactions between climatic and socio-environmental factors is not well understood. Here, we investigate the drivers of dengue incidence and emergence in Vietnam, through analyzing 23-years of monthly district-level case data spanning a period of significant socioeconomic change (1998-2020). We show that urban infrastructure factors (sanitation, water supply and long-term urban growth) predict local spatial patterns of dengue incidence, while human mobility is a more influential driver in subtropical northern regions than the endemic south. Temperature is the dominant factor shaping dengues geographical distribution and dynamics, and using long-term reanalysis temperature data we show that recent warming (since 1950) has generally expanded transmission risk throughout Vietnam, and most strongly in current dengue emergence hotspots (e.g. southern central regions and Ha Noi). In contrast, effects of hydrometeorology are complex, multi-scalar and dependent on local context: risk increases under both short-term precipitation excess and long-term drought, but improvements in water supply largely mitigate drought-associated risks except under extreme conditions. Our findings challenge the assumption that dengue is an urban disease, instead suggesting that incidence peaks in transitional landscapes with intermediate infrastructure provision, and provide evidence that interactions between recent climate change and mobility have contributed to dengues ongoing expansion throughout Vietnam.\nPrevalence of long-term symptoms varies by using different post-COVID-19 definitions in positively and negatively tested adults: the PRIME post-COVID study\nAuthors: Pagen, D. M.; van Bilsen, C. J.; Brinkhues, S.; Van Herck, M.; Konings, K.; den Heijer, C. D.; ter Waarbeek, H. L.; Spruit, M. A.; Hoebe, C. J.; Dukers-Muijrers, N. H.\nScore: 5.9, Published: 2023-08-02 DOI: 10.1101/2023.07.27.23293244\nBackgroundLong-term symptoms after a SARS-CoV-2 infection (i.e., post-COVID-19 condition or long COVID), constitute a substantial public health problem. Yet, the prevalence remains currently unclear as different case definitions are used, and negatively tested controls are lacking. We aimed to estimate post-COVID-19 condition prevalence using six definitions. MethodsThe Prevalence, Risk factors, and Impact Evaluation (PRIME) post-COVID-19 condition study is a population-based sample of COVID-19 tested adults. End 2021, 61,655 adults were invited to complete an online questionnaire, including 44 symptoms plus a severity score (0-10) per symptom. The prevalence was calculated in both positively and negatively tested adults, stratified by time since their COVID-19 test (3-5, 6-11 or [\u0026ge;]12 months ago). ResultsIn positives (n=7,405; 75.6%), the prevalence of long-term symptoms was between 26.9% and 64.1% using the six definitions, while in negatives (n=2,392; 24.4%) the prevalence varied between 11.4% and 32.5%. The prevalence of long-term symptoms potentially accountable to COVID-19 ranged from 17.9% to 26.3%. ConclusionThere is a (substantial) variation in prevalence estimates by using different definitions as is current practice, showing limited overlap between definitions, indicating that the essential post-COVID-19 condition criteria are still unclear. Including negatives is important to determine long-term symptoms accountable to COVID-19. Trial registration ClinicalTrials.gov Identifier: NCT05128695.\nThe Impact of the UK COVID-19 Lockdown on the Screening, Diagnostics and Incidence of Breast, Colorectal, Lung and Prostate Cancer in the UK: a Population-Based Cohort Study\nAuthors: Barclay, N. L.; Pineda-Moncusi, M. L.; Jodicke, A. M.; Prieto-Alhambra, D.; Raventos, B.; Newby, D.; Delmestri, A.; Man, W. Y.; Chen, X.; Catala, M.\nScore: 7.3, Published: 2023-07-26 DOI: 10.1101/2023.07.21.23292937\nObjectivesThis study aimed to assess the impact of the COVID-19 lockdown on the screening and diagnosis of breast, colorectal, lung, and prostate cancer. The study also investigated whether the rates returned to pre-pandemic levels by December 2021. DesignCohort study. SettingElectronic health records from UK primary care Clinical Practice Research Datalink (CPRD) GOLD database. ParticipantsThe study included individuals registered with CPRD GOLD between January 2017 and December 2021, with at least 365 days of prior observation. Main outcome measuresThe study focused on screening, diagnostic tests, referrals and diagnoses of first-ever breast, colorectal, lung, and prostate cancer. Incidence rates (IR) were stratified by age, sex and region, and incidence rate ratios (IRR) were calculated to compare rates during and after lockdown with the reference period before lockdown. Forecasted rates were estimated using negative binomial regression models. ResultsAmong 5,191,650 eligible participants, the initial lockdown resulted in reduced screening and diagnostic tests for all cancers, which remained dramatically reduced across the whole observation period for almost all tests investigated. For cancer incidence rates, there were significant IRR reductions in breast (0.69), colorectal (0.74), and prostate (0.71) cancers. However, the reduction in lung cancer incidence (0.92) was non-significant. Extrapolating to the entire UK population, an estimated 18,000 breast, 13,000 colorectal, 10,000 lung, and 21,000 prostate cancer diagnoses were missed from March 2020 to December 2021. ConclusionThe national COVID-19 lockdown in the UK had a substantial impact on cancer screening, diagnostic tests, referrals and diagnoses. Although incidence rates started to recover after the lockdown, they remained significantly lower than pre-pandemic levels for breast and prostate cancers and associated tests. Delays in diagnosis are likely to have adverse consequences on cancer stage, treatment initiation, mortality rates, and years of life lost. Urgent strategies are needed to identify undiagnosed cases and address the long-term implications of delayed diagnoses. WHAT IS ALREADY KNOWN ON THIS TOPICO_LIBreast, colorectal, lung, and prostate cancer are the most common causes of cancer death in the UK. C_LIO_LIThe COVID-19 pandemic led to the postponement of cancer screening programs and reductions in diagnostic tests, resulting in delays in diagnosis and treatment initiation, impacting prognosis and mortality rates. C_LIO_LIComprehensive data on the impact of changing social restrictions and post-lockdown periods is lacking in the UK, along with an assessment of specific screening pathways and patient experiences within the healthcare system. C_LI WHAT THIS STUDY ADDSO_LIThe first UK national COVID-19 lockdown resulted in reductions in screening, diagnostic tests, and referrals, particularly for mammograms, colonoscopies, and visits to breast surgeons, leading to underdiagnosis of breast, colorectal, and prostate cancers. Despite some increase in rates after the lockdown, they remained significantly lower than pre-pandemic levels by December 2021, particularly for prostate cancer. C_LIO_LIMost affected populations were women aged 60-79 years for breast and colorectal cancer; men aged 60-79 years for lung cancer; and men aged 40-59 years for prostate cancer. C_LIO_LIDelays in diagnosis are likely to have consequences on cancer stage at diagnosis, treatment initiation, mortality rates, and total years of life lost. Strategies such as public awareness campaigns, targeted screening programs, and improved coordination between primary care and hospitals are needed to address the backlog and identify the potential [~]62,000 missed cancer cases in the UK. C_LI\nProteomic prediction of common and rare diseases\nAuthors: Carrasco-Zanini, J.; Pietzner, M.; Davitte, J.; Surendran, P.; Croteau-Chonka, D. C.; Robins, C.; Torralbo, A.; Tomlinson, C.; Fitzpatrick, N.; Ytsma, C.; Kanno, T.; Gade, S.; Freitag, D.; Ziebell, F.; Denaxas, S.; Betts, J. C.; Wareham, N. J.; Hemingway, H.; Scott, R. A.; Langenberg, C.\nScore: 33.5, Published: 2023-07-23 DOI: 10.1101/2023.07.18.23292811\nBackgroundFor many diseases there are delays in diagnosis due to a lack of objective biomarkers for disease onset. Whether measuring thousands of proteins offers predictive information across a wide range of diseases is unknown. MethodsIn 41,931 individuals from the UK Biobank Pharma Proteomics Project (UKB-PPP), we integrated [~]3000 plasma proteins with clinical information to derive sparse prediction models for the 10-year incidence of 218 common and rare diseases (81 - 6038 cases). We compared prediction models based on proteins with a) basic clinical information alone, b) basic clinical information + 37 clinical biomarkers, and c) genome-wide polygenic risk scores. ResultsFor 67 pathologically diverse diseases, a model including as few as 5 to 20 proteins was superior to clinical models (median delta C-index = 0.07; range = 0.02 - 0.31) and to clinical models with biomarkers for 52 diseases. In multiple myeloma, for example, a set of 5 proteins significantly improved prediction over basic clinical information (delta C-index = 0.25 (95% confidence interval 0.20 - 0.29)). At a 5% false positive rate (FPR), proteomic prediction (5 proteins) identified individuals at high risk of multiple myeloma (detection rate (DR) = 50%), non-Hodgkin lymphoma (DR = 55%) and motor neuron disease (DR = 29%). At a 20% FPR, proteomic prediction identified individuals at high-risk for pulmonary fibrosis (DR= 80%) and dilated cardiomyopathy (DR = 75%). ConclusionsSparse plasma protein signatures offer novel, clinically useful prediction of common and rare diseases, through disease-specific proteins and protein predictors shared across multiple diseases. (Funded by Medical Research Council, NIHR, Wellcome Trust.)\nThe Lancet peer reviewers and the COVID-19 pandemic\nAuthors: Masukume, G.; Grech, V.\nScore: 1.6, Published: 2023-08-04 DOI: 10.1101/2023.08.02.23293558\nIntroduction: Peer review is paramount to the scholarly article paradigm, helping to ensure the integrity and credibility of research. The Lancet played a crucial role in disseminating key information on the COVID-19 pandemic, publishing early clinical descriptions, risk factors for death, and effectiveness of measures like physical distancing and masks. Notably, The Lancet was the world's most cited journal for COVID-19 research, emphasising its significant impact on disseminating critical findings during the pandemic. Methods: Geographic data for The Lancet's peer reviewers in 2019 (pre-pandemic) and 2020 (pandemic) were analysed at the country level, ranking reviewer countries. A test of proportions compared reviewer numbers between the years. Results: In 2020, China emerged as one of the top ten reviewer countries for the first time, with a significant increase from 1% (25 of 1843) in 2019 to 3% (54 of 1850), p=0.001. Italy also entered the top five reviewer countries, rising from 4% (67) to 5% (90), p=0.065. Reviewers from Africa 43 (2%) and South America 31 (2%) represented their continents in 2020. The top ten reviewer nations for The Lancet in 2020 largely mirrored the top ten countries in global COVID-19 research output. Conclusion: During the COVID-19 pandemic's acute phase in 2020, The Lancet, the world's most cited journal for COVID-19 research, featured peer reviewers who were largely representative of global COVID-19 research output. Notably, reviewers from China, the first country affected by COVID-19, increased significantly. However, underrepresentation of some continents persisted. To foster global idea exchange and enhance pandemic preparedness, research capacity worldwide must expand, broadening the reviewer pool; a vital step given uncertainties in future pandemic geographic origin.\nPredicting Long COVID in the National COVID Cohort Collaborative Using Super Learner\nAuthors: Butzin-Dozier, Z.; Ji, Y.; Li, H.; Coyle, J.; Shi, J.; Phillips, R. V.; Mertens, A.; Pirracchio, R.; van der Laan, M. J.; Patel, R. C.; Colford, J. M.; Hubbard, A. E.; on behalf of the National COVID Cohort Collaborative (N3C) Consortium*, Score: 1.0, Published: 2023-08-04 DOI: 10.1101/2023.07.27.23293272\nPost-acute Sequelae of COVID-19 (PASC), also known as Long COVID, is a broad grouping of a range of long-term symptoms following acute COVID-19 infection. An understanding of characteristics that are predictive of future PASC is valuable, as this can inform the identification of high-risk individuals and future preventative efforts. However, current knowledge regarding PASC risk factors is limited. Using a sample of 55,257 participants from the National COVID Cohort Collaborative, as part of the NIH Long COVID Computational Challenge, we sought to predict individual risk of PASC diagnosis from a curated set of clinically informed covariates. We predicted individual PASC status, given covariate information, using Super Learner (an ensemble machine learning algorithm also known as stacking) to learn the optimal, AUC-maximizing combination of gradient boosting and random forest algorithms. We were able to predict individual PASC diagnoses accurately (AUC 0.947). Temporally, we found that baseline characteristics were most predictive of future PASC diagnosis, compared with characteristics immediately before, during, or after COVID-19 infection. This finding supports the hypothesis that clinicians may be able to accurately assess the risk of PASC in patients prior to acute COVID diagnosis, which could improve early interventions and preventive care. We found that medical utilization, demographics and anthropometry, and respiratory factors were most predictive of PASC diagnosis. This highlights the importance of respiratory characteristics in PASC risk assessment. The methods outlined here provide an open-source, applied example of using Super Learner to predict PASC status using electronic health record data, which can be replicated across a variety of settings.\n",
  "wordCount" : "2565",
  "inLanguage": "en",
  "datePublished": "2023-08-06T10:36:48Z",
  "dateModified": "2023-08-06T10:36:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/epidemiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      epidemiology
    </h1>
    <div class="post-meta"><span>updated on August 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.01.05.21268323">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.01.05.21268323" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.01.05.21268323">
        <p class="paperTitle">Lineage replacement and evolution captured by the United Kingdom Covid Infection Survey</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.01.05.21268323" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.01.05.21268323" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lythgoe, K. A.; Golubchik, T.; Hall, M.; House, T.; Cahuantzi, R.; MacIntyre-Cockett, G.; Fryer, H.; Thomson, L.; Nurtay, A.; Ghafari, M.; Buck, D.; Green, A.; Trebes, A.; Piazza, P.; Lonie, L. J.; Studley, R.; Rourke, E.; Smith, D.; Bashton, M.; Nelson, A.; Crown, M.; McCann, C.; Young, G. R.; de Santos, R. A. N.; Richards, Z.; Tariq, A.; Wellcome Sanger Institute COVID-19 Surveillance Team,  ; COVID-19 Infection Survey Group,  ; The COVID-19 Genomics UK (COG-UK) consortium,  ; Fraser, C.; Diamond, I.; Barrett, J.; Walker, A. S.; Bonsall, D.</p>
        <p class="info">Score: 27.7, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.01.05.21268323' target='https://doi.org/10.1101/2022.01.05.21268323'> 10.1101/2022.01.05.21268323</a></p>
        <p class="abstract">The Office for National Statistics COVID-19 Infection Survey (ONS-CIS) is the largest surveillance study of SARS-CoV-2 positivity in the community, and collected data on the United Kingdom (UK) epidemic from April 2020 until March 2023 before being paused. Here, we report on the epidemiological and evolutionary dynamics of SARS-CoV-2 determined by analysing the sequenced samples collected by the ONS-CIS during this period. We observed a series of sweeps or partial sweeps, with each sweeping lineage having a distinct growth advantage compared to their predecessors. The sweeps also generated an alternating pattern in which most samples had either S-gene target failure (SGTF) or non- SGTF over time. Evolution was characterised by steadily increasing divergence and diversity within lineages, but with step increases in divergence associated with each sweeping major lineage. This led to a faster overall rate of evolution when measured at the between-lineage level compared to within lineages, and fluctuating levels of diversity. These observations highlight the value of viral sequencing integrated into community surveillance studies to monitor the viral epidemiology and evolution of SARS-CoV-2, and potentially other pathogens, particularly in the current phase of the pandemic with routine RT-PCR testing now ended in the community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.01.23288879">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.01.23288879" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.01.23288879">
        <p class="paperTitle">Blood protein levels predict leading incident diseases and mortality in UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.01.23288879" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.01.23288879" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gadd, D. A.; Hillary, R. F.; Kuncheva, Z.; Mangelis, T.; Cheng, Y.; Dissanayake, M.; Admanit, R.; Gagnon, J.; Lin, T.; Ferber, K.; Runz, H.; Biogen Biobank Team,  ; Marioni, R. E.; Foley, C. N.; Sun, B. B.</p>
        <p class="info">Score: 19.1, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.01.23288879' target='https://doi.org/10.1101/2023.05.01.23288879'> 10.1101/2023.05.01.23288879</a></p>
        <p class="abstract">The circulating proteome offers insights into the biological pathways that underlie disease. Here, we test relationships between 1,468 Olink protein levels and the incidence of 23 age-related diseases and mortality, over 16 years of electronic health linkage in the UK Biobank (N=47,600). We report 3,201 associations between 961 protein levels and 21 incident outcomes, identifying proteomic indicators of multiple morbidities. Next, protein-based scores (ProteinScores) are developed using penalised Cox regression. When applied to test sets, six ProteinScores improve Area Under the Curve (AUC) estimates for the 10 year onset of incident outcomes beyond age, sex and a comprehensive set of 24 lifestyle factors, clinically relevant biomarkers and physical measures. Furthermore, the ProteinScore for type 2 diabetes outperformed a polygenic risk score, a metabolomic score and HbA1c, a clinical marker used to monitor and diagnose type 2 diabetes. These data characterise early proteomic contributions to major age-related disease and demonstrate the value of the plasma proteome for risk stratification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.23293110">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.23293110" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.23293110">
        <p class="paperTitle">Interactions between climate change, urban infrastructure and mobility are driving dengue emergence in Vietnam</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.23293110" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.23293110" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gibb, R.; Colon-Gonzalez, F. J.; Lan, P. T.; Huong, P. T.; Nam, V. S.; Duoc, V. T.; Do, H. T.; Dong, N. T.; Chien, V. C.; Trang, L. T. T.; Quoc, D. K.; Hoa, T. M.; Tai, N. H.; Hang, T. T.; Tsarouchi, G.; Ainscoe, E.; Harpham, Q.; Hofmann, B.; Lumbroso, D.; Brady, O. J.; Lowe, R.</p>
        <p class="info">Score: 7.0, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.23293110' target='https://doi.org/10.1101/2023.07.25.23293110'> 10.1101/2023.07.25.23293110</a></p>
        <p class="abstract">Dengue is expanding globally, but how dengue emergence is shaped locally by interactions between climatic and socio-environmental factors is not well understood. Here, we investigate the drivers of dengue incidence and emergence in Vietnam, through analyzing 23-years of monthly district-level case data spanning a period of significant socioeconomic change (1998-2020). We show that urban infrastructure factors (sanitation, water supply and long-term urban growth) predict local spatial patterns of dengue incidence, while human mobility is a more influential driver in subtropical northern regions than the endemic south. Temperature is the dominant factor shaping dengues geographical distribution and dynamics, and using long-term reanalysis temperature data we show that recent warming (since 1950) has generally expanded transmission risk throughout Vietnam, and most strongly in current dengue emergence hotspots (e.g. southern central regions and Ha Noi). In contrast, effects of hydrometeorology are complex, multi-scalar and dependent on local context: risk increases under both short-term precipitation excess and long-term drought, but improvements in water supply largely mitigate drought-associated risks except under extreme conditions. Our findings challenge the assumption that dengue is an urban disease, instead suggesting that incidence peaks in transitional landscapes with intermediate infrastructure provision, and provide evidence that interactions between recent climate change and mobility have contributed to dengues ongoing expansion throughout Vietnam.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.23293244">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.23293244" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.23293244">
        <p class="paperTitle">Prevalence of long-term symptoms varies by using different post-COVID-19 definitions in positively and negatively tested adults: the PRIME post-COVID study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.23293244" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.23293244" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pagen, D. M.; van Bilsen, C. J.; Brinkhues, S.; Van Herck, M.; Konings, K.; den Heijer, C. D.; ter Waarbeek, H. L.; Spruit, M. A.; Hoebe, C. J.; Dukers-Muijrers, N. H.</p>
        <p class="info">Score: 5.9, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.23293244' target='https://doi.org/10.1101/2023.07.27.23293244'> 10.1101/2023.07.27.23293244</a></p>
        <p class="abstract">BackgroundLong-term symptoms after a SARS-CoV-2 infection (i.e., post-COVID-19 condition or long COVID), constitute a substantial public health problem. Yet, the prevalence remains currently unclear as different case definitions are used, and negatively tested controls are lacking. We aimed to estimate post-COVID-19 condition prevalence using six definitions.

MethodsThe Prevalence, Risk factors, and Impact Evaluation (PRIME) post-COVID-19 condition study is a population-based sample of COVID-19 tested adults. End 2021, 61,655 adults were invited to complete an online questionnaire, including 44 symptoms plus a severity score (0-10) per symptom. The prevalence was calculated in both positively and negatively tested adults, stratified by time since their COVID-19 test (3-5, 6-11 or [&amp;ge;]12 months ago).

ResultsIn positives (n=7,405; 75.6%), the prevalence of long-term symptoms was between 26.9% and 64.1% using the six definitions, while in negatives (n=2,392; 24.4%) the prevalence varied between 11.4% and 32.5%. The prevalence of long-term symptoms potentially accountable to COVID-19 ranged from 17.9% to 26.3%.

ConclusionThere is a (substantial) variation in prevalence estimates by using different definitions as is current practice, showing limited overlap between definitions, indicating that the essential post-COVID-19 condition criteria are still unclear. Including negatives is important to determine long-term symptoms accountable to COVID-19.

Trial registration ClinicalTrials.gov Identifier: NCT05128695.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.23292937">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.23292937" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.23292937">
        <p class="paperTitle">The Impact of the UK COVID-19 Lockdown on the Screening, Diagnostics and Incidence of Breast, Colorectal, Lung and Prostate Cancer in the UK: a Population-Based Cohort Study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.23292937" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.23292937" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Barclay, N. L.; Pineda-Moncusi, M. L.; Jodicke, A. M.; Prieto-Alhambra, D.; Raventos, B.; Newby, D.; Delmestri, A.; Man, W. Y.; Chen, X.; Catala, M.</p>
        <p class="info">Score: 7.3, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.23292937' target='https://doi.org/10.1101/2023.07.21.23292937'> 10.1101/2023.07.21.23292937</a></p>
        <p class="abstract">ObjectivesThis study aimed to assess the impact of the COVID-19 lockdown on the screening and diagnosis of breast, colorectal, lung, and prostate cancer. The study also investigated whether the rates returned to pre-pandemic levels by December 2021.

DesignCohort study.

SettingElectronic health records from UK primary care Clinical Practice Research Datalink (CPRD) GOLD database.

ParticipantsThe study included individuals registered with CPRD GOLD between January 2017 and December 2021, with at least 365 days of prior observation.

Main outcome measuresThe study focused on screening, diagnostic tests, referrals and diagnoses of first-ever breast, colorectal, lung, and prostate cancer. Incidence rates (IR) were stratified by age, sex and region, and incidence rate ratios (IRR) were calculated to compare rates during and after lockdown with the reference period before lockdown. Forecasted rates were estimated using negative binomial regression models.

ResultsAmong 5,191,650 eligible participants, the initial lockdown resulted in reduced screening and diagnostic tests for all cancers, which remained dramatically reduced across the whole observation period for almost all tests investigated. For cancer incidence rates, there were significant IRR reductions in breast (0.69), colorectal (0.74), and prostate (0.71) cancers. However, the reduction in lung cancer incidence (0.92) was non-significant. Extrapolating to the entire UK population, an estimated 18,000 breast, 13,000 colorectal, 10,000 lung, and 21,000 prostate cancer diagnoses were missed from March 2020 to December 2021.

ConclusionThe national COVID-19 lockdown in the UK had a substantial impact on cancer screening, diagnostic tests, referrals and diagnoses. Although incidence rates started to recover after the lockdown, they remained significantly lower than pre-pandemic levels for breast and prostate cancers and associated tests. Delays in diagnosis are likely to have adverse consequences on cancer stage, treatment initiation, mortality rates, and years of life lost. Urgent strategies are needed to identify undiagnosed cases and address the long-term implications of delayed diagnoses.

WHAT IS ALREADY KNOWN ON THIS TOPICO_LIBreast, colorectal, lung, and prostate cancer are the most common causes of cancer death in the UK.
C_LIO_LIThe COVID-19 pandemic led to the postponement of cancer screening programs and reductions in diagnostic tests, resulting in delays in diagnosis and treatment initiation, impacting prognosis and mortality rates.
C_LIO_LIComprehensive data on the impact of changing social restrictions and post-lockdown periods is lacking in the UK, along with an assessment of specific screening pathways and patient experiences within the healthcare system.
C_LI

WHAT THIS STUDY ADDSO_LIThe first UK national COVID-19 lockdown resulted in reductions in screening, diagnostic tests, and referrals, particularly for mammograms, colonoscopies, and visits to breast surgeons, leading to underdiagnosis of breast, colorectal, and prostate cancers. Despite some increase in rates after the lockdown, they remained significantly lower than pre-pandemic levels by December 2021, particularly for prostate cancer.
C_LIO_LIMost affected populations were women aged 60-79 years for breast and colorectal cancer; men aged 60-79 years for lung cancer; and men aged 40-59 years for prostate cancer.
C_LIO_LIDelays in diagnosis are likely to have consequences on cancer stage at diagnosis, treatment initiation, mortality rates, and total years of life lost. Strategies such as public awareness campaigns, targeted screening programs, and improved coordination between primary care and hospitals are needed to address the backlog and identify the potential [~]62,000 missed cancer cases in the UK.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.23292811">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.23292811" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.23292811">
        <p class="paperTitle">Proteomic prediction of common and rare diseases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.23292811" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.23292811" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carrasco-Zanini, J.; Pietzner, M.; Davitte, J.; Surendran, P.; Croteau-Chonka, D. C.; Robins, C.; Torralbo, A.; Tomlinson, C.; Fitzpatrick, N.; Ytsma, C.; Kanno, T.; Gade, S.; Freitag, D.; Ziebell, F.; Denaxas, S.; Betts, J. C.; Wareham, N. J.; Hemingway, H.; Scott, R. A.; Langenberg, C.</p>
        <p class="info">Score: 33.5, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.23292811' target='https://doi.org/10.1101/2023.07.18.23292811'> 10.1101/2023.07.18.23292811</a></p>
        <p class="abstract">BackgroundFor many diseases there are delays in diagnosis due to a lack of objective biomarkers for disease onset. Whether measuring thousands of proteins offers predictive information across a wide range of diseases is unknown.

MethodsIn 41,931 individuals from the UK Biobank Pharma Proteomics Project (UKB-PPP), we integrated [~]3000 plasma proteins with clinical information to derive sparse prediction models for the 10-year incidence of 218 common and rare diseases (81 - 6038 cases). We compared prediction models based on proteins with a) basic clinical information alone, b) basic clinical information &#43; 37 clinical biomarkers, and c) genome-wide polygenic risk scores.

ResultsFor 67 pathologically diverse diseases, a model including as few as 5 to 20 proteins was superior to clinical models (median delta C-index = 0.07; range = 0.02 - 0.31) and to clinical models with biomarkers for 52 diseases. In multiple myeloma, for example, a set of 5 proteins significantly improved prediction over basic clinical information (delta C-index = 0.25 (95% confidence interval 0.20 - 0.29)). At a 5% false positive rate (FPR), proteomic prediction (5 proteins) identified individuals at high risk of multiple myeloma (detection rate (DR) = 50%), non-Hodgkin lymphoma (DR = 55%) and motor neuron disease (DR = 29%). At a 20% FPR, proteomic prediction identified individuals at high-risk for pulmonary fibrosis (DR= 80%) and dilated cardiomyopathy (DR = 75%).

ConclusionsSparse plasma protein signatures offer novel, clinically useful prediction of common and rare diseases, through disease-specific proteins and protein predictors shared across multiple diseases.

(Funded by Medical Research Council, NIHR, Wellcome Trust.)</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.23293558">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.23293558" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.23293558">
        <p class="paperTitle">The Lancet peer reviewers and the COVID-19 pandemic</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.23293558" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.23293558" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Masukume, G.; Grech, V.</p>
        <p class="info">Score: 1.6, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.23293558' target='https://doi.org/10.1101/2023.08.02.23293558'> 10.1101/2023.08.02.23293558</a></p>
        <p class="abstract">Introduction: Peer review is paramount to the scholarly article paradigm, helping to ensure the integrity and credibility of research. The Lancet played a crucial role in disseminating key information on the COVID-19 pandemic, publishing early clinical descriptions, risk factors for death, and effectiveness of measures like physical distancing and masks. Notably, The Lancet was the world&#39;s most cited journal for COVID-19 research, emphasising its significant impact on disseminating critical findings during the pandemic. Methods: Geographic data for The Lancet&#39;s peer reviewers in 2019 (pre-pandemic) and 2020 (pandemic) were analysed at the country level, ranking reviewer countries. A test of proportions compared reviewer numbers between the years. Results: In 2020, China emerged as one of the top ten reviewer countries for the first time, with a significant increase from 1% (25 of 1843) in 2019 to 3% (54 of 1850), p=0.001. Italy also entered the top five reviewer countries, rising from 4% (67) to 5% (90), p=0.065. Reviewers from Africa 43 (2%) and South America 31 (2%) represented their continents in 2020. The top ten reviewer nations for The Lancet in 2020 largely mirrored the top ten countries in global COVID-19 research output. Conclusion: During the COVID-19 pandemic&#39;s acute phase in 2020, The Lancet, the world&#39;s most cited journal for COVID-19 research, featured peer reviewers who were largely representative of global COVID-19 research output. Notably, reviewers from China, the first country affected by COVID-19, increased significantly. However, underrepresentation of some continents persisted. To foster global idea exchange and enhance pandemic preparedness, research capacity worldwide must expand, broadening the reviewer pool; a vital step given uncertainties in future pandemic geographic origin.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.23293272">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.23293272" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.23293272">
        <p class="paperTitle">Predicting Long COVID in the National COVID Cohort Collaborative Using Super Learner</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.23293272" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.23293272" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Butzin-Dozier, Z.; Ji, Y.; Li, H.; Coyle, J.; Shi, J.; Phillips, R. V.; Mertens, A.; Pirracchio, R.; van der Laan, M. J.; Patel, R. C.; Colford, J. M.; Hubbard, A. E.; on behalf of the National COVID Cohort Collaborative (N3C) Consortium*,  </p>
        <p class="info">Score: 1.0, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.23293272' target='https://doi.org/10.1101/2023.07.27.23293272'> 10.1101/2023.07.27.23293272</a></p>
        <p class="abstract">Post-acute Sequelae of COVID-19 (PASC), also known as Long COVID, is a broad grouping of a range of long-term symptoms following acute COVID-19 infection. An understanding of characteristics that are predictive of future PASC is valuable, as this can inform the identification of high-risk individuals and future preventative efforts. However, current knowledge regarding PASC risk factors is limited. Using a sample of 55,257 participants from the National COVID Cohort Collaborative, as part of the NIH Long COVID Computational Challenge, we sought to predict individual risk of PASC diagnosis from a curated set of clinically informed covariates. We predicted individual PASC status, given covariate information, using Super Learner (an ensemble machine learning algorithm also known as stacking) to learn the optimal, AUC-maximizing combination of gradient boosting and random forest algorithms. We were able to predict individual PASC diagnoses accurately (AUC 0.947). Temporally, we found that baseline characteristics were most predictive of future PASC diagnosis, compared with characteristics immediately before, during, or after COVID-19 infection. This finding supports the hypothesis that clinicians may be able to accurately assess the risk of PASC in patients prior to acute COVID diagnosis, which could improve early interventions and preventive care. We found that medical utilization, demographics and anthropometry, and respiratory factors were most predictive of PASC diagnosis. This highlights the importance of respiratory characteristics in PASC risk assessment. The methods outlined here provide an open-source, applied example of using Super Learner to predict PASC status using electronic health record data, which can be replicated across a variety of settings.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
