<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>epidemiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>epidemiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Long COVID in a highly vaccinated population infected during a SARS-CoV-2 Omicron wave - Australia, 2022
Authors: Woldegiorgis, M.; Cadby, G.; Ngeh, S.; Korda, R.; Armstrong, P.; Maticevic, J.; Knight, P.; Jardine, A.; Bloomfield, L.; Effler, P.
Score: 2370.5, Published: 2023-08-09 DOI: 10.1101/2023.08.06.23293706
ObjectiveTo characterise Long COVID in a highly vaccinated population infected by Omicron. DesignFollow-up survey of persons testing positive for SARS-CoV-2 in Western Australia, 16 July-3 August 2022.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/epidemiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="epidemiology" />
<meta property="og:description" content="Long COVID in a highly vaccinated population infected during a SARS-CoV-2 Omicron wave - Australia, 2022
Authors: Woldegiorgis, M.; Cadby, G.; Ngeh, S.; Korda, R.; Armstrong, P.; Maticevic, J.; Knight, P.; Jardine, A.; Bloomfield, L.; Effler, P.
Score: 2370.5, Published: 2023-08-09 DOI: 10.1101/2023.08.06.23293706
ObjectiveTo characterise Long COVID in a highly vaccinated population infected by Omicron. DesignFollow-up survey of persons testing positive for SARS-CoV-2 in Western Australia, 16 July-3 August 2022." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/epidemiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-03T10:37:18+00:00" />
<meta property="article:modified_time" content="2023-09-03T10:37:18+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="epidemiology"/>
<meta name="twitter:description" content="Long COVID in a highly vaccinated population infected during a SARS-CoV-2 Omicron wave - Australia, 2022
Authors: Woldegiorgis, M.; Cadby, G.; Ngeh, S.; Korda, R.; Armstrong, P.; Maticevic, J.; Knight, P.; Jardine, A.; Bloomfield, L.; Effler, P.
Score: 2370.5, Published: 2023-08-09 DOI: 10.1101/2023.08.06.23293706
ObjectiveTo characterise Long COVID in a highly vaccinated population infected by Omicron. DesignFollow-up survey of persons testing positive for SARS-CoV-2 in Western Australia, 16 July-3 August 2022."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "epidemiology",
      "item": "https://trxiv.yorks0n.com/posts/epidemiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "epidemiology",
  "name": "epidemiology",
  "description": "Long COVID in a highly vaccinated population infected during a SARS-CoV-2 Omicron wave - Australia, 2022\nAuthors: Woldegiorgis, M.; Cadby, G.; Ngeh, S.; Korda, R.; Armstrong, P.; Maticevic, J.; Knight, P.; Jardine, A.; Bloomfield, L.; Effler, P.\nScore: 2370.5, Published: 2023-08-09 DOI: 10.1101/2023.08.06.23293706\nObjectiveTo characterise Long COVID in a highly vaccinated population infected by Omicron. DesignFollow-up survey of persons testing positive for SARS-CoV-2 in Western Australia, 16 July-3 August 2022.",
  "keywords": [
    
  ],
  "articleBody": " Long COVID in a highly vaccinated population infected during a SARS-CoV-2 Omicron wave - Australia, 2022\nAuthors: Woldegiorgis, M.; Cadby, G.; Ngeh, S.; Korda, R.; Armstrong, P.; Maticevic, J.; Knight, P.; Jardine, A.; Bloomfield, L.; Effler, P.\nScore: 2370.5, Published: 2023-08-09 DOI: 10.1101/2023.08.06.23293706\nObjectiveTo characterise Long COVID in a highly vaccinated population infected by Omicron. DesignFollow-up survey of persons testing positive for SARS-CoV-2 in Western Australia, 16 July-3 August 2022. SettingCommunity Participants22,744 persons with COVID-19 who had agreed to participate in research at the time of diagnosis were texted a survey link 90 days later; non-responders were telephoned. Post stratification weights were applied to responses from 11,697 (51.4%) participants, 94.0% of whom had received \u003e= 3 vaccine doses. Main outcome measuresPrevalence of Long COVID - defined as reporting new or ongoing COVID-19 illness-related symptoms or health issues 90 days post diagnosis; associated health care utilisation, reductions in work/study and risk factors were assessed using log-binomial regression. Results18.2% (n=2,130) of respondents met case definition for Long COVID. Female sex, being 50-69 years of age, pre-existing health issues, residing in a rural or remote area, and receiving fewer vaccine doses were significant independent predictors of Long COVID (p \u003c 0.05). Persons with Long COVID reported a median of 6 symptoms, most commonly fatigue (70.6%) and difficulty concentrating (59.6%); 38.2% consulted a GP and 1.6% reported hospitalisation in the month prior to the survey due to ongoing symptoms. Of 1,778 respondents with Long COVID who were working/studying before their COVID-19 diagnosis, 17.9% reported reducing/discontinuing work/study. Conclusion90 days post Omicron infection, almost 1 in 5 respondents reported Long COVID symptoms; 1 in 15 of all persons with COVID-19 sought healthcare for associated health concerns \u003e=2 months after the acute illness. Significance of the studyO_ST_ABSThe knownC_ST_ABSThe prevalence of Long COVID varies widely across studies conducted in diverse settings globally (range: 9%-81%). The newIn a highly vaccinated population (94% with \u003e=3 vaccine doses), almost 20% of persons infected with the SARS-CoV-2 Omicron variant reported symptoms consistent with Long COVID 90 days post diagnosis. Long COVID was associated with sustained negative impacts on work/study and a substantial utilisation of GP services 2-3 months after the acute illness; however, ED presentations and hospitalisations for Long COVID were rare. The implicationsGP clinics play a significant role in managing the burden of Long COVID in Australia.\nInfluence of Prior SARS-CoV-2 Infection on COVID-19 Severity: Evidence from the National COVID Cohort Collaborative\nAuthors: Hendrix, N.; Sidky, H.; Sahner, D.; N3C Consortium, Score: 761.1, Published: 2023-08-06 DOI: 10.1101/2023.08.03.23293612\nBackgroundAs SARS-CoV-2 has transitioned from a pandemic to endemic disease, the majority of new infections have been among previously infected individuals. To manage the risks and benefits of ongoing COVID-19 policies, it is important to understand whether prior infection modifies the severity of subsequent infections. MethodsWe used data from first and second COVID-19 episodes in the National COVID Cohort Collaborative (N3C), a collection of health systems who provide de-identified electronic health records for research purposes. Our analysis was a sequential series of nested trial emulations. In the first of two analytic stages, we created a month-specific model of the probability of prior infection for each individual. In the second stage, we used an ordinal logistic regression with inverse probability weights calculated in the first stage to simulate a series of monthly trials comparing severity between the cohorts of first and second infections. In addition to cohort-wide effect estimates, we also conducted analyses among race/ethnicity, sex, and age subgroups. ResultsFrom an initial cohort of 7,446,481 combined first and second infections, we identified a cohort of 2,227,484 infections, among which 7.6% were second infections. Ninety-four percent of patients with two recorded infections experienced mild disease for both. The overall odds ratio (OR) for more severe disease with prior infection was 1.06 (95% confidence interval [CI]: 1.03 - 1.10). Monthly point estimates of the OR ranged from 0.56 (95% CI: 0.37 - 0.84) in October 2020 to 1.64 (95% CI: 1.33 - 2.00) in February 2023. In most subgroups, the effect of prior infection was significant. In 8 out of 10 subgroups, the maximum monthly OR occurred after the minimum monthly OR, suggesting that protection has waned throughout the pandemic. ConclusionOverall, prior infection was associated with a significant slightly elevated risk of severe disease. This effect varied month to month. As the pandemic proceeded, the effect of prior infection tended to evolve from generally protective during the pre-Omicron era to unprotective during the Omicron era. This points to the need for continued strategies to avert and minimize the harms of COVID-19, rather than relying upon immunity acquired through previous infection. Key PointsO_ST_ABSQuestionC_ST_ABSDoes prior infection with SARS-CoV-2 affect the severity of subsequent COVID-19 episodes? FindingsWe observed a mild protective effect of prior infection during the early and mid-stages of the pandemic that waned after the rise of the Omicron variants, ultimately resulting in loss of protection or a tendency toward more severe second infections. MeaningPrior infection alone is likely not enough to avert the worst public health harms of endemic SARS-CoV-2. Interventions to avoid infection and reduce the severity of COVID-19 will still be important in the post-pandemic era.\nHigh proportions of post-exertional malaise and orthostatic intolerance in people living with post-COVID-19 condition: the PRIME post-COVID study\nAuthors: Pagen, D. M.; Van Herck, M.; van Bilsen, C. J.; Brinkhues, S.; Konings, K.; den Heijer, C. D.; Spruit, M. A.; Hoebe, C. J.; Dukers-Muijrers, N. H.\nScore: 300.3, Published: 2023-08-23 DOI: 10.1101/2023.08.17.23294204\nBackgroundExercise-based treatments can be harmful in people who were SARS-CoV-2 positive and living with post-COVID-19 condition (PL-PCC) and who have post-exertional malaise (PEM) or orthostatic intolerance (OI). Nevertheless, PEM and OI are not routinely assessed by clinicians. We estimated PEM and OI proportions in PL-PCC, as well in people not living with PCC (PnL-PCC) and negatives (i.e., never reported a SARS-CoV-2 positive test), and identified associated factors. MethodsParticipants from the PRIME post-COVID study were included. PEM and OI were assessed using validated questionnaires. PCC was defined as feeling unrecovered after SARS-CoV-2 infection. Multivariable regression analyses to study PEM and OI were stratified for sex. ResultsData from 3,783 participants was analyzed. In PL-PCC, proportion of PEM was 48.1% and 41.2%, and proportion of OI was 29.3% and 27.9% in women and in men, respectively. Proportions were higher in PL-PCC compared to negatives, for PEM in women OR=4.38 [95%CI:3.01-6.38]; in men OR=4.78 [95%CI:3.13-7.29]; for OI in women 3.06 [95%CI:1.97-4.76]; in men 2.71 [95%CI:1.75-4.21]. Associated factors were age [\u0026le;]60 years, [\u0026ge;]1 comorbidities and living alone. ConclusionsHigh proportions of PEM and OI are observed in PL-PCC. Standard screening for PEM and OI is recommended in PL-PCC, to promote appropriate therapies. Trial registration ClinicalTrials.gov identifier:NCT05128695\nAssociation between dietary inflammatory index score and incident dementia: results from the Framingham heart Study offspring cohort\nAuthors: Melo van Lent, D.; Gokingco Mesa, H.; Short, M. I.; Gonzales, M. M.; Aparicio, H. J.; Salinas, J.; Yuan, C.; Jacques, P. F.; Beiser, A.; Seshadri, S.; Jacob, M. E.; Himali, J. J.\nScore: 16.8, Published: 2023-08-22 DOI: 10.1101/2023.08.21.23294374\nBackgroundThe Dietary Inflammatory Index (DII), has been specifically designed to capture the inflammatory content of diet and has shown association with neurodegenerative disease related outcomes. But literature is limited on the role of diet-driven inflammation measured by the DII on incident all-cause dementia and Alzheimers disease dementia (AD). ObjectiveWe evaluated whether higher DII scores were associated with increased incidence of all-cause dementia and AD over 22.3 years of follow-up in the community-based Framingham Heart Study (FHS) Offspring cohort. Design, Setting, and ParticipantsObservational longitudinal study in the FHS Offspring cohort. Dementia surveillance for present study: until 2020. Data were analyzed from December 2020 to June 2022. Participants completed a validated 126-item food frequency questionnaires (FFQ), administered at FHS examination cycle 7 (1998-2001) and examination cycle 5 (1991-1995), and/or 6 (1995-1998). Individuals aged \u003c60 years, with prevalent dementia, no dementia follow-up, other relevant neurological diseases, and/or no FFQ data were excluded. ExposureA DII score (based on the published method by Shivappa et al. 2014) was created based on previous studies linking individual dietary factors to six inflammatory markers (i.e. C-reactive protein, interleukin (IL)-1{beta}, IL-4, IL-6, IL-10, and tumor necrosis factor-alpha), consisting of 36 components. A cumulative DII score was calculated by averaging across a maximum of three FFQs. Main outcomes and measuresIncident all-cause dementia and AD. ResultsWe included 1487 participants (mean{+/-}SD, age in years 69 {+/-} 6; 53{middle dot}2% women; 31{middle dot}6% college graduates]). 246 participants developed all-cause dementia (including AD n=187) over a median follow up time of 13{middle dot}1 years. Higher DII scores were associated with an increased incidence of all-cause dementia and AD following adjustment for age and sex (Hazard ratio (HR) 1{middle dot}16, 95% confidence interval (CI) 1{middle dot}07 to 1{middle dot}25, p\u003c.001; HR 1{middle dot}16, 95% CI 1{middle dot}06 to 1{middle dot}26, p=.001). The relationships remained after additional adjustment for demographic, lifestyle, and clinical covariates (HR 1{middle dot}21, 95% CI 1{middle dot}10 to 1{middle dot}33, p\u003c0.001; HR1{middle dot}20, 95% CI1{middle dot}07 to 1{middle dot}35, p=.001). Conclusion and relevanceHigher DII scores were associated with a higher risk of incident all-cause dementia and AD. Although these promising findings need to be replicated and further validated, our results suggest that diets which correlate with low DII scores may prevent late-life dementia.\nGenetic Insights into the Microevolutionary Dynamics and Early Introductions of Human Monkeypox Virus in Mexico\nAuthors: Gomez-Sanchez, I.; Castelan-Sanchez, H. G.; Martinez-Castilla, L. P.; Hurtado-Ramirez, J. M.; Lopez-Leal, G.\nScore: 10.5, Published: 2023-08-28 DOI: 10.1101/2023.08.26.23294674\nThe recent global outbreak of Monkeypox, caused by the Monkeypox virus (MPVX) emerged in Europe in 2022 and rapidly spread to over 40 countries. The Americas are currently facing the highest impact, reporting over 50,000 cases by early 2023. Here, we analyze 880 MPXV isolates worldwide to gain insights into the evolutionary patterns and initial introduction events of the virus in Mexico. We found that MPXV entered Mexico on multiple occasions, from the United Kingdom, Portugal, and Canada, and subsequently spread locally in different regions of Mexico. Additionally, we show that MPXV has an open pangenome, highlighting the role of gene turnover in shaping its genomic diversity, rather than SNPs variations, which do not contribute significantly to genome diversity. Although the genome presents multiple SNP sites in coding regions, these remain under purifying selection, suggesting their evolutionary conservation. One notable exception is the amino acid site 63 of the protein encoded by the Cop-A4L gene, intricately related to viral maturity, for which we picked up a strong signal of positive selection. Ancestral state reconstruction deduced that the ancestral state at site 63 corresponds to the amino acid valine, present only in isolates of clade I. However, the isolates of the current outbreak evolved for threonine at site 63. Finally, our findings contribute to the knowledge of the evolutionary processes of the Monkeypox virus.\nImproving estimates of epidemiological quantities by combining reported cases with wastewater data: a statistical framework with applications to COVID-19 in Aotearoa New Zealand\nAuthors: Watson, L. M.; Plank, M. J.; Armstrong, B.; Chapman, J.; Hewitt, J.; Morris, H.; Orsi, A.; Bunce, M.; Donnelly, C. A.; Steyn, N.\nScore: 71.2, Published: 2023-08-16 DOI: 10.1101/2023.08.14.23294060\nBackgroundTimely and informed public health responses to infectious diseases such as COVID-19 necessitate reliable information about infection dynamics. The case ascertainment rate (CAR), the proportion of infections that are reported as cases, is typically much less than one and varies with testing practices and behaviours, making reported cases unreliable as the sole source of data. The concentration of viral RNA in wastewater samples provides an alternate measure of infection prevalence that is not affected by human behaviours. Here, we investigated how these two data sources can be combined to inform estimates of the instantaneous reproduction number, R, and track changes in the CAR over time. MethodsWe constructed a state-space model that we solved using sequential Monte Carlo methods. The observed data are the levels of SARS-CoV-2 in wastewater and reported case incidence. The hidden states that we estimate are R and CAR. Model parameters are estimated using the particle marginal Metropolis Hastings algorithm. FindingsWe analysed data from 1 January 2022 to 31 March 2023 from Aotearoa New Zealand. Our model estimates that R peaked at 2.76 (95% CrI 2.20, 3.83) around 18 February 2022 and the CAR peaked around 12 March 2022. Accounting for reduced CAR, we estimate that New Zealands second Omicron wave in July 2022 was similar in size to the first, despite fewer reported cases. We estimate that the CAR in the BA.5 Omicron wave in July 2022 was approximately 50% lower than in the BA.1/BA.2 Omicron wave in March 2022. The CAR in subsequent waves around November 2022 and April 2023 was estimated to be comparable to that in the second Omicron wave. InterpretationThis work on wastewater-based epidemiology (WBE) can be used to give insight into key epidemiological quantities. Estimating R, CAR, and cumulative number of infections provides useful information for planning public health responses and understanding the state of immunity in the population. This model is a useful disease surveillance tool, improving situational awareness of infectious disease dynamics in real-time, which may be increasingly useful as intensive pandemic surveillance programmes are wound down. FundingNew Zealand Ministry of Health, Department of Prime Minister and Cabinet, the Royal Society Te Ap[a]rangi, Imperial College London, and University of Oxford. Research in ContextO_ST_ABSEvidence before this studyC_ST_ABSThere has been a substantial increase in the number of publications focusing on wastewater-based epidemiology (WBE) in recent years, particularly during the COVID-19 pandemic. We searched PubMed for \"wastewater based epidemiology\" and found fewer than 10 papers per year prior to 2014 with a drastic increase to 463 in 2022. Approximately 52% of the WBE publications are related to COVID-19 (\"wastewater based epidemiology\" AND (\"SARS-CoV-2\" OR \"COVID-19\")). Many studies have focused on detecting SARS-CoV-2 in wastewater systems but only 10 have estimated the reproduction number (\"wastewater based epidemiology\" AND (\"SARS-CoV-2\" OR \"COVID-19\") AND \"reproduction number\"). No previous work has combined WBE with reported case data to estimate (relative) case ascertainment rate (\"waste-water based epidemiology\" AND (\"SARS-CoV-2\" OR \"COVID-19\") AND \"case ascertainment rate\"). Previous work has estimated the reproduction number from reported cases assuming constant under-ascertainment but the issue of time-varying case ascertainment has not yet been addressed, except to demonstrate the effect of a pre-determined change in case ascertainment. Added value of this studyWe present a model that, for the first time, enables reported case information to be combined with wastewater data to estimate epidemiology quantities. This work further demonstrates the utility of WBE; the reproduction number can be estimated in the absence of reported case information (although results are more reliable when case data are included), and wastewater data include information that, when combined with case data, can be used to estimate the time-varying relative case ascertainment rate. Implications of all the available evidenceIn order to make informed and timely public health decisions about infectious diseases, it is important to understand the number of infections in the community. WBE provides a useful source of data that is not impacted by time-varying testing practices. Wastewater data can be quantitatively combined with case information to better understand the state of an epidemic. In order to determine the absolute case ascertainment rate (rather than the relative rate calculated in this work), there is a need for infection prevalence surveys to calibrate model results against.\nStrategic use of SARS-CoV-2 wastewater concentration data could enhance, but not replace, high-resolution community prevalence survey programmes\nAuthors: Mills, C.; Chadeau-Hyam, M.; Elliott, P. L.; Donnelly, C. A.\nScore: 9.1, Published: 2023-08-20 DOI: 10.1101/2023.08.17.23293589\nWastewater-based epidemiology (WBE) has been proposed as a tool for public health authorities to monitor community transmission of SARS-CoV-2 and other agents. Here, we review the utility of WBE for estimating SARS-CoV-2 prevalence using wastewater data from the Environmental Monitoring for Health Protection (EMHP) programme and prevalence data from the REal-time Assessment of Community Transmission-1 (REACT-1) study in England. Our analysis shows a temporally evolving relationship between wastewater and prevalence which limits the utility of WBE for estimating SARS-CoV-2 prevalence in high spatial resolution without a concurrent prevalence survey. We further characterise WBE for SARS-CoV-2 prevalence as i) vaccination-coverage-dependent and ii) variant-specific. Our work provides a gesopatial framework to map wastewater concentrations to public health boundaries, enabling public health authorities to interpret the relationship between wastewater and prevalence. We demonstrate that WBE can improve the cost efficiency and accuracy of community prevalence surveys which on their own may have incomplete geographic coverage or small sample sizes.\nFirst-Time Prescribing of GLP-1 Receptor Agonists from 2018-2023: A Descriptive Analysis\nAuthors: Rodriguez, P. J.; Gratzl, S.; Goodwin Cartwright, B. M.; Brar, R.; Gluckman, T. J.; Stucky, N.\nScore: 8.1, Published: 2023-08-25 DOI: 10.1101/2023.08.22.23294277\nAimsLimited recent data exist on prescribing patterns and patient characteristics for glucagon-like peptide 1 receptor agonists (GLP-1 RAs), an important drug class for patients with type 2 diabetes mellitus (T2D) and/or overweight or obesity. We describe trends in first-time prescribing of GLP-1 RAs. Materials and MethodsUsing aggregated US electronic health record data, we identified first-time prescriptions of GLP-1 RAs between January 2018 and June 2023 for adults receiving regular care. We describe prescribing volumes stratified by patient characteristics, specific drug, and FDA-labelled indication. The 5.5-year analysis period was divided into six-month periods. ResultsIn total, 274,562 patients were newly prescribed a GLP-1 RA, with a significant increase over time (January to June, 2018: 9,642 versus 2023: 66,569; p \u003c 0.001). Overall, 181,860 (66.2%) patients had T2D and 229,715 (83.7%) had obesity or overweight. The proportion with T2D decreased over time (January through June, 2018: 84.1% versus 2023: 49%; p \u003c 0.001), while the proportion with overweight or obesity increased (January through June, 2018: 75.9% versus 2023: 90.7%; p \u003c.001). Of prescriptions with a known FDA-labelled indication (74.2%), 87% were labelled for T2D and 13% were labelled for overweight or obesity. Patients first prescribed a GLP-1 RA labelled for T2D were 59.1% female with a mean (SD) age of 58.6 (13) years, while those prescribed a GLP-1 RA labelled for overweight or obesity were 82.9% female, with a mean (SD) age of 48.2 (12.2) years. ConclusionsWe observed an increase in first-time prescribing of GLP-1 RAs overall, and a shift away from a predominately T2D population.\nOne Health Genomic Surveillance of Avian and Human Influenza A Viruses Through Environmental Wastewater Monitoring\nAuthors: Lee, A. J.; Carson, S.; Reyne, M. I.; Marshall, A.; Moody, D.; Allen, D.; Allingham, P.; Levickas, A.; Fitzgerald, A.; Bell, S. H.; Lock, J.; Lock, J. D.; McSparron, C.; Nejad, B. F.; Courtney, D. G.; Einarsson, G. G.; McKenna, J. P.; Fairley, D. J.; Curran, T.; McKinley, J. M.; Gilpin, D. F.; Lemon, K.; McGrath, J. W.; Bamford, C. G.\nScore: 15.3, Published: 2023-08-13 DOI: 10.1101/2023.08.08.23293833\nBackgroundInfluenza A viruses (IAV) are significant pathogens of humans and other animals. Although endemic in humans and birds, novel IAV strains can emerge, jump species, and cause epidemics, like the latest variant of H5N1. Wastewater-based epidemiology (WBE) has very recently been shown to detect human IAV but whether it can detect avian-origin IAV, and if whole genome sequencing (WGS) can be used to discriminate circulating strains of IAV in wastewater remains unknown. MethodsUsing a pan-IAV RT-qPCR assay, six wastewater treatment works (WWTWs) across Northern Ireland (NI), were screened from August to December 2022. A WGS approach using Oxford Nanopore technology was employed to sequence positive samples. Phylogenetic analysis of sequences relative to currently circulating human and avian IAVs was performed. FindingsWe detected a dynamic IAV signal in wastewater from September 2022 onwards across NI. \"Meta\" whole genome sequences were generated displaying homology to both human and avian IAV strains. The relative proportion of human versus avian-origin IAV reads differed across time and sample site. A diversity in subtypes and lineages was detected (e.g. H1N1, H3N2, and several avian). Avian segment 8 related to those found in recent H5N1 clade 2.3.4.4b was identified. InterpretationWBE affords a means to monitor circulating human and avian IAV strains and provide crucial genetic information. As such WBE can provide rapid, cost-effective, year-round \"one-health\" IAV surveillance to help control epidemic and pandemic threats. FundingThis study was funded by the Department of Health for Northern Ireland as part of the Northern Ireland Wastewater Surveillance Programme. HighlightsO_LIDynamic IAV RT-qPCR signal in wastewater detected across NI. C_LIO_LINanopore-based WGS reveals presence of both human and avian IAVs in wastewater. C_LIO_LIAvian IAV sequence similarity to gull-associated H13/H16 and recent H5N1 isolates. C_LIO_LICo-detection of distinct clades of human H1N1 and H3N2 subtypes. C_LI Author SummaryInfluenza A virus (IAV) is a major pathogen of humans and other animals and causes regular epidemics and devastating pandemics. Recently, a novel variant of highly-pathogenic H5N1 avian influenza has emerged spreading across the world killing millions of birds and infecting mammals, enhancing its pandemic potential. Strengthening global surveillance systems for human and animal IAV is thus a major priority. Wastewater-based epidemiology (WBE) has been applied to track SARS-CoV-2 and IAV in humans but whether this approach could work for avian IAV is not known. Here, we develop a \"one-health\" method to survey pan-IAV levels and genetically characterise the viruses. Through this we highlight co-detection of human and avian IAVs in wastewater, with homology to recent H5N1 isolates. Our work demonstrates the potential for WBE to help defend against not only human infections but emerging, zoonotic IAVs of pandemic potential. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=133 SRC=\"FIGDIR/small/23293833v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (26K): org.highwire.dtl.DTLVardef@a7b3c4org.highwire.dtl.DTLVardef@b2c12org.highwire.dtl.DTLVardef@17a81faorg.highwire.dtl.DTLVardef@1eb5c70_HPS_FORMAT_FIGEXP M_FIG C_FIG\nMEX-PD: A National Network for the Epidemiological \u0026 Genetic Research of Parkinson's Disease\nAuthors: Lazaro-Figueroa, A.; Reyes-Perez, P.; Morelos-Figaredo, E.; Guerra-Galicia, C. M.; Estrada-Bellmann, I.; Salinas-Barboza, K.; Matuk, Y.; Gandarilla-Martinez, N. A.; Caballero-Sanchez, U.; Flores-Ocampo, V.; Montes-Alcantara, P.; Espinosa-Mendez, I.; Zayas-Del Moral, A.; Gaspar-Martinez, E.; Vazquez-Guevara, D.; Rodriguez-Violante, M.; Inca-Martinez, M.; Mata, I. F.; Alcauter, S.; Renteria, M. E.; Medina-Rivera, A.; Ruiz-Contreras, A. E.\nScore: 4.3, Published: 2023-08-31 DOI: 10.1101/2023.08.28.23294700\nBackground Parkinson's Disease (PD) has a complex etiology, involving genetic and environmental factors. Most of our current understanding of the disease comes from studies in populations with mostly European ancestry, representing challenges in generalizing findings to other populations with different genetic, social, and environmental contexts. There are scarce studies focused in Latin American populations. The Mexican population is genetically diverse because its admixture from Native American, European, and African ancestries, coupled with the unique environmental conditions, stressing the relevance of establishing genetic studies in this population. Thus, we have established the Mexican Parkinson's Research Network (MEX-PD), a consortium to research the clinical, genetical, environmental, and neurophysiological bases of the phenotypic diversity in Mexican PD patients. Objectives Describing how MEX-PD was established, the methods and instruments and presenting the first results. Methods Patients and controls were recruited from medical centers in 20 states of Mexico. Initial recruitment included neurological evaluation, cognitive assessment, and DNA collection. Results MEX-PD has registered 302 controls and 262 PD patients with a mean age of diagnosis of 61 years (SD=10.86). There were 19.8% PD patients identified with early onset. Levodopa was the most common pharmacological treatment. Conclusions MEX-PD contributes to understand PD nationally. The information gathered here will allow us to understand the prevalence of mental health, neurological symptoms, and cognitive function in the PD Mexican population and how genetical and environmental factors contributes to those outcomes. These will advocate for personalized treatments and improving quality of life in the Mexican population.\n",
  "wordCount" : "3864",
  "inLanguage": "en",
  "datePublished": "2023-09-03T10:37:18Z",
  "dateModified": "2023-09-03T10:37:18Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/epidemiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      epidemiology
    </h1>
    <div class="post-meta"><span>updated on September 3, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.23293706">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.23293706" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.23293706">
        <p class="paperTitle">Long COVID in a highly vaccinated population infected during a SARS-CoV-2 Omicron wave - Australia, 2022</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.23293706" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.23293706" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Woldegiorgis, M.; Cadby, G.; Ngeh, S.; Korda, R.; Armstrong, P.; Maticevic, J.; Knight, P.; Jardine, A.; Bloomfield, L.; Effler, P.</p>
        <p class="info">Score: 2370.5, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.23293706' target='https://doi.org/10.1101/2023.08.06.23293706'> 10.1101/2023.08.06.23293706</a></p>
        <p class="abstract">ObjectiveTo characterise Long COVID in a highly vaccinated population infected by Omicron.

DesignFollow-up survey of persons testing positive for SARS-CoV-2 in Western Australia, 16 July-3 August 2022.

SettingCommunity

Participants22,744 persons with COVID-19 who had agreed to participate in research at the time of diagnosis were texted a survey link 90 days later; non-responders were telephoned. Post stratification weights were applied to responses from 11,697 (51.4%) participants, 94.0% of whom had received &gt;= 3 vaccine doses.

Main outcome measuresPrevalence of  Long COVID - defined as reporting new or ongoing COVID-19 illness-related symptoms or health issues 90 days post diagnosis; associated health care utilisation, reductions in work/study and risk factors were assessed using log-binomial regression.

Results18.2% (n=2,130) of respondents met case definition for Long COVID. Female sex, being 50-69 years of age, pre-existing health issues, residing in a rural or remote area, and receiving fewer vaccine doses were significant independent predictors of Long COVID (p &lt; 0.05). Persons with Long COVID reported a median of 6 symptoms, most commonly fatigue (70.6%) and difficulty concentrating (59.6%); 38.2% consulted a GP and 1.6% reported hospitalisation in the month prior to the survey due to ongoing symptoms. Of 1,778 respondents with Long COVID who were working/studying before their COVID-19 diagnosis, 17.9% reported reducing/discontinuing work/study.

Conclusion90 days post Omicron infection, almost 1 in 5 respondents reported Long COVID symptoms; 1 in 15 of all persons with COVID-19 sought healthcare for associated health concerns &gt;=2 months after the acute illness.

Significance of the studyO_ST_ABSThe knownC_ST_ABSThe prevalence of Long COVID varies widely across studies conducted in diverse settings globally (range: 9%-81%).

The newIn a highly vaccinated population (94% with &gt;=3 vaccine doses), almost 20% of persons infected with the SARS-CoV-2 Omicron variant reported symptoms consistent with Long COVID 90 days post diagnosis. Long COVID was associated with sustained negative impacts on work/study and a substantial utilisation of GP services 2-3 months after the acute illness; however, ED presentations and hospitalisations for Long COVID were rare.

The implicationsGP clinics play a significant role in managing the burden of Long COVID in Australia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.23293612">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.23293612" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.23293612">
        <p class="paperTitle">Influence of Prior SARS-CoV-2 Infection on COVID-19 Severity: Evidence from the National COVID Cohort Collaborative</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.23293612" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.23293612" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hendrix, N.; Sidky, H.; Sahner, D.; N3C Consortium,  </p>
        <p class="info">Score: 761.1, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.23293612' target='https://doi.org/10.1101/2023.08.03.23293612'> 10.1101/2023.08.03.23293612</a></p>
        <p class="abstract">BackgroundAs SARS-CoV-2 has transitioned from a pandemic to endemic disease, the majority of new infections have been among previously infected individuals. To manage the risks and benefits of ongoing COVID-19 policies, it is important to understand whether prior infection modifies the severity of subsequent infections.

MethodsWe used data from first and second COVID-19 episodes in the National COVID Cohort Collaborative (N3C), a collection of health systems who provide de-identified electronic health records for research purposes. Our analysis was a sequential series of nested trial emulations. In the first of two analytic stages, we created a month-specific model of the probability of prior infection for each individual. In the second stage, we used an ordinal logistic regression with inverse probability weights calculated in the first stage to simulate a series of monthly trials comparing severity between the cohorts of first and second infections. In addition to cohort-wide effect estimates, we also conducted analyses among race/ethnicity, sex, and age subgroups.

ResultsFrom an initial cohort of 7,446,481 combined first and second infections, we identified a cohort of 2,227,484 infections, among which 7.6% were second infections. Ninety-four percent of patients with two recorded infections experienced mild disease for both. The overall odds ratio (OR) for more severe disease with prior infection was 1.06 (95% confidence interval [CI]: 1.03 - 1.10). Monthly point estimates of the OR ranged from 0.56 (95% CI: 0.37 - 0.84) in October 2020 to 1.64 (95% CI: 1.33 - 2.00) in February 2023. In most subgroups, the effect of prior infection was significant. In 8 out of 10 subgroups, the maximum monthly OR occurred after the minimum monthly OR, suggesting that protection has waned throughout the pandemic.

ConclusionOverall, prior infection was associated with a significant slightly elevated risk of severe disease. This effect varied month to month. As the pandemic proceeded, the effect of prior infection tended to evolve from generally protective during the pre-Omicron era to unprotective during the Omicron era. This points to the need for continued strategies to avert and minimize the harms of COVID-19, rather than relying upon immunity acquired through previous infection.

Key PointsO_ST_ABSQuestionC_ST_ABSDoes prior infection with SARS-CoV-2 affect the severity of subsequent COVID-19 episodes?

FindingsWe observed a mild protective effect of prior infection during the early and mid-stages of the pandemic that waned after the rise of the Omicron variants, ultimately resulting in loss of protection or a tendency toward more severe second infections.

MeaningPrior infection alone is likely not enough to avert the worst public health harms of endemic SARS-CoV-2. Interventions to avoid infection and reduce the severity of COVID-19 will still be important in the post-pandemic era.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.23294204">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.23294204" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.23294204">
        <p class="paperTitle">High proportions of post-exertional malaise and orthostatic intolerance in people living with post-COVID-19 condition: the PRIME post-COVID study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.23294204" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.23294204" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pagen, D. M.; Van Herck, M.; van Bilsen, C. J.; Brinkhues, S.; Konings, K.; den Heijer, C. D.; Spruit, M. A.; Hoebe, C. J.; Dukers-Muijrers, N. H.</p>
        <p class="info">Score: 300.3, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.23294204' target='https://doi.org/10.1101/2023.08.17.23294204'> 10.1101/2023.08.17.23294204</a></p>
        <p class="abstract">BackgroundExercise-based treatments can be harmful in people who were SARS-CoV-2 positive and living with post-COVID-19 condition (PL-PCC) and who have post-exertional malaise (PEM) or orthostatic intolerance (OI). Nevertheless, PEM and OI are not routinely assessed by clinicians. We estimated PEM and OI proportions in PL-PCC, as well in people not living with PCC (PnL-PCC) and negatives (i.e., never reported a SARS-CoV-2 positive test), and identified associated factors.

MethodsParticipants from the PRIME post-COVID study were included. PEM and OI were assessed using validated questionnaires. PCC was defined as feeling unrecovered after SARS-CoV-2 infection. Multivariable regression analyses to study PEM and OI were stratified for sex.

ResultsData from 3,783 participants was analyzed. In PL-PCC, proportion of PEM was 48.1% and 41.2%, and proportion of OI was 29.3% and 27.9% in women and in men, respectively. Proportions were higher in PL-PCC compared to negatives, for PEM in women OR=4.38 [95%CI:3.01-6.38]; in men OR=4.78 [95%CI:3.13-7.29]; for OI in women 3.06 [95%CI:1.97-4.76]; in men 2.71 [95%CI:1.75-4.21]. Associated factors were age [&amp;le;]60 years, [&amp;ge;]1 comorbidities and living alone.

ConclusionsHigh proportions of PEM and OI are observed in PL-PCC. Standard screening for PEM and OI is recommended in PL-PCC, to promote appropriate therapies.

Trial registration ClinicalTrials.gov identifier:NCT05128695</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.23294374">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.23294374" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.23294374">
        <p class="paperTitle">Association between dietary inflammatory index score and incident dementia: results from the Framingham heart Study offspring cohort</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.23294374" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.23294374" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Melo van Lent, D.; Gokingco Mesa, H.; Short, M. I.; Gonzales, M. M.; Aparicio, H. J.; Salinas, J.; Yuan, C.; Jacques, P. F.; Beiser, A.; Seshadri, S.; Jacob, M. E.; Himali, J. J.</p>
        <p class="info">Score: 16.8, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.23294374' target='https://doi.org/10.1101/2023.08.21.23294374'> 10.1101/2023.08.21.23294374</a></p>
        <p class="abstract">BackgroundThe Dietary Inflammatory Index (DII), has been specifically designed to capture the inflammatory content of diet and has shown association with neurodegenerative disease related outcomes. But literature is limited on the role of diet-driven inflammation measured by the DII on incident all-cause dementia and Alzheimers disease dementia (AD).

ObjectiveWe evaluated whether higher DII scores were associated with increased incidence of all-cause dementia and AD over 22.3 years of follow-up in the community-based Framingham Heart Study (FHS) Offspring cohort.

Design, Setting, and ParticipantsObservational longitudinal study in the FHS Offspring cohort. Dementia surveillance for present study: until 2020. Data were analyzed from December 2020 to June 2022. Participants completed a validated 126-item food frequency questionnaires (FFQ), administered at FHS examination cycle 7 (1998-2001) and examination cycle 5 (1991-1995), and/or 6 (1995-1998). Individuals aged &lt;60 years, with prevalent dementia, no dementia follow-up, other relevant neurological diseases, and/or no FFQ data were excluded.

ExposureA DII score (based on the published method by Shivappa et al. 2014) was created based on previous studies linking individual dietary factors to six inflammatory markers (i.e. C-reactive protein, interleukin (IL)-1{beta}, IL-4, IL-6, IL-10, and tumor necrosis factor-alpha), consisting of 36 components. A cumulative DII score was calculated by averaging across a maximum of three FFQs.

Main outcomes and measuresIncident all-cause dementia and AD.

ResultsWe included 1487 participants (mean{&#43;/-}SD, age in years 69 {&#43;/-} 6; 53{middle dot}2% women; 31{middle dot}6% college graduates]). 246 participants developed all-cause dementia (including AD n=187) over a median follow up time of 13{middle dot}1 years. Higher DII scores were associated with an increased incidence of all-cause dementia and AD following adjustment for age and sex (Hazard ratio (HR) 1{middle dot}16, 95% confidence interval (CI) 1{middle dot}07 to 1{middle dot}25, p&lt;.001; HR 1{middle dot}16, 95% CI 1{middle dot}06 to 1{middle dot}26, p=.001). The relationships remained after additional adjustment for demographic, lifestyle, and clinical covariates (HR 1{middle dot}21, 95% CI 1{middle dot}10 to 1{middle dot}33, p&lt;0.001; HR1{middle dot}20, 95% CI1{middle dot}07 to 1{middle dot}35, p=.001).

Conclusion and relevanceHigher DII scores were associated with a higher risk of incident all-cause dementia and AD. Although these promising findings need to be replicated and further validated, our results suggest that diets which correlate with low DII scores may prevent late-life dementia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.26.23294674">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.26.23294674" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.26.23294674">
        <p class="paperTitle">Genetic Insights into the Microevolutionary Dynamics and Early Introductions of Human Monkeypox Virus in Mexico</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.26.23294674" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.26.23294674" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gomez-Sanchez, I.; Castelan-Sanchez, H. G.; Martinez-Castilla, L. P.; Hurtado-Ramirez, J. M.; Lopez-Leal, G.</p>
        <p class="info">Score: 10.5, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.26.23294674' target='https://doi.org/10.1101/2023.08.26.23294674'> 10.1101/2023.08.26.23294674</a></p>
        <p class="abstract">The recent global outbreak of Monkeypox, caused by the Monkeypox virus (MPVX) emerged in Europe in 2022 and rapidly spread to over 40 countries. The Americas are currently facing the highest impact, reporting over 50,000 cases by early 2023. Here, we analyze 880 MPXV isolates worldwide to gain insights into the evolutionary patterns and initial introduction events of the virus in Mexico. We found that MPXV entered Mexico on multiple occasions, from the United Kingdom, Portugal, and Canada, and subsequently spread locally in different regions of Mexico. Additionally, we show that MPXV has an open pangenome, highlighting the role of gene turnover in shaping its genomic diversity, rather than SNPs variations, which do not contribute significantly to genome diversity. Although the genome presents multiple SNP sites in coding regions, these remain under purifying selection, suggesting their evolutionary conservation. One notable exception is the amino acid site 63 of the protein encoded by the Cop-A4L gene, intricately related to viral maturity, for which we picked up a strong signal of positive selection. Ancestral state reconstruction deduced that the ancestral state at site 63 corresponds to the amino acid valine, present only in isolates of clade I. However, the isolates of the current outbreak evolved for threonine at site 63. Finally, our findings contribute to the knowledge of the evolutionary processes of the Monkeypox virus.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.23294060">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.23294060" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.23294060">
        <p class="paperTitle">Improving estimates of epidemiological quantities by combining reported cases with wastewater data: a statistical framework with applications to COVID-19 in Aotearoa New Zealand</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.23294060" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.23294060" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Watson, L. M.; Plank, M. J.; Armstrong, B.; Chapman, J.; Hewitt, J.; Morris, H.; Orsi, A.; Bunce, M.; Donnelly, C. A.; Steyn, N.</p>
        <p class="info">Score: 71.2, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.23294060' target='https://doi.org/10.1101/2023.08.14.23294060'> 10.1101/2023.08.14.23294060</a></p>
        <p class="abstract">BackgroundTimely and informed public health responses to infectious diseases such as COVID-19 necessitate reliable information about infection dynamics. The case ascertainment rate (CAR), the proportion of infections that are reported as cases, is typically much less than one and varies with testing practices and behaviours, making reported cases unreliable as the sole source of data. The concentration of viral RNA in wastewater samples provides an alternate measure of infection prevalence that is not affected by human behaviours. Here, we investigated how these two data sources can be combined to inform estimates of the instantaneous reproduction number, R, and track changes in the CAR over time.

MethodsWe constructed a state-space model that we solved using sequential Monte Carlo methods. The observed data are the levels of SARS-CoV-2 in wastewater and reported case incidence. The hidden states that we estimate are R and CAR. Model parameters are estimated using the particle marginal Metropolis Hastings algorithm.

FindingsWe analysed data from 1 January 2022 to 31 March 2023 from Aotearoa New Zealand. Our model estimates that R peaked at 2.76 (95% CrI 2.20, 3.83) around 18 February 2022 and the CAR peaked around 12 March 2022. Accounting for reduced CAR, we estimate that New Zealands second Omicron wave in July 2022 was similar in size to the first, despite fewer reported cases. We estimate that the CAR in the BA.5 Omicron wave in July 2022 was approximately 50% lower than in the BA.1/BA.2 Omicron wave in March 2022. The CAR in subsequent waves around November 2022 and April 2023 was estimated to be comparable to that in the second Omicron wave.

InterpretationThis work on wastewater-based epidemiology (WBE) can be used to give insight into key epidemiological quantities. Estimating R, CAR, and cumulative number of infections provides useful information for planning public health responses and understanding the state of immunity in the population. This model is a useful disease surveillance tool, improving situational awareness of infectious disease dynamics in real-time, which may be increasingly useful as intensive pandemic surveillance programmes are wound down.

FundingNew Zealand Ministry of Health, Department of Prime Minister and Cabinet, the Royal Society Te Ap[a]rangi, Imperial College London, and University of Oxford.

Research in ContextO_ST_ABSEvidence before this studyC_ST_ABSThere has been a substantial increase in the number of publications focusing on wastewater-based epidemiology (WBE) in recent years, particularly during the COVID-19 pandemic. We searched PubMed for &#34;wastewater based epidemiology&#34; and found fewer than 10 papers per year prior to 2014 with a drastic increase to 463 in 2022. Approximately 52% of the WBE publications are related to COVID-19 (&#34;wastewater based epidemiology&#34; AND (&#34;SARS-CoV-2&#34; OR &#34;COVID-19&#34;)). Many studies have focused on detecting SARS-CoV-2 in wastewater systems but only 10 have estimated the reproduction number (&#34;wastewater based epidemiology&#34; AND (&#34;SARS-CoV-2&#34; OR &#34;COVID-19&#34;) AND &#34;reproduction number&#34;). No previous work has combined WBE with reported case data to estimate (relative) case ascertainment rate (&#34;waste-water based epidemiology&#34; AND (&#34;SARS-CoV-2&#34; OR &#34;COVID-19&#34;) AND &#34;case ascertainment rate&#34;). Previous work has estimated the reproduction number from reported cases assuming constant under-ascertainment but the issue of time-varying case ascertainment has not yet been addressed, except to demonstrate the effect of a pre-determined change in case ascertainment.

Added value of this studyWe present a model that, for the first time, enables reported case information to be combined with wastewater data to estimate epidemiology quantities. This work further demonstrates the utility of WBE; the reproduction number can be estimated in the absence of reported case information (although results are more reliable when case data are included), and wastewater data include information that, when combined with case data, can be used to estimate the time-varying relative case ascertainment rate.

Implications of all the available evidenceIn order to make informed and timely public health decisions about infectious diseases, it is important to understand the number of infections in the community. WBE provides a useful source of data that is not impacted by time-varying testing practices. Wastewater data can be quantitatively combined with case information to better understand the state of an epidemic. In order to determine the absolute case ascertainment rate (rather than the relative rate calculated in this work), there is a need for infection prevalence surveys to calibrate model results against.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.23293589">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.23293589" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.23293589">
        <p class="paperTitle">Strategic use of SARS-CoV-2 wastewater concentration data could enhance, but not replace, high-resolution community prevalence survey programmes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.23293589" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.23293589" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mills, C.; Chadeau-Hyam, M.; Elliott, P. L.; Donnelly, C. A.</p>
        <p class="info">Score: 9.1, Published: 2023-08-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.23293589' target='https://doi.org/10.1101/2023.08.17.23293589'> 10.1101/2023.08.17.23293589</a></p>
        <p class="abstract">Wastewater-based epidemiology (WBE) has been proposed as a tool for public health authorities to monitor community transmission of SARS-CoV-2 and other agents. Here, we review the utility of WBE for estimating SARS-CoV-2 prevalence using wastewater data from the Environmental Monitoring for Health Protection (EMHP) programme and prevalence data from the REal-time Assessment of Community Transmission-1 (REACT-1) study in England. Our analysis shows a temporally evolving relationship between wastewater and prevalence which limits the utility of WBE for estimating SARS-CoV-2 prevalence in high spatial resolution without a concurrent prevalence survey. We further characterise WBE for SARS-CoV-2 prevalence as i) vaccination-coverage-dependent and ii) variant-specific. Our work provides a gesopatial framework to map wastewater concentrations to public health boundaries, enabling public health authorities to interpret the relationship between wastewater and prevalence. We demonstrate that WBE can improve the cost efficiency and accuracy of community prevalence surveys which on their own may have incomplete geographic coverage or small sample sizes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.23294277">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.23294277" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.23294277">
        <p class="paperTitle">First-Time Prescribing of GLP-1 Receptor Agonists from 2018-2023: A Descriptive Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.23294277" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.23294277" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rodriguez, P. J.; Gratzl, S.; Goodwin Cartwright, B. M.; Brar, R.; Gluckman, T. J.; Stucky, N.</p>
        <p class="info">Score: 8.1, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.23294277' target='https://doi.org/10.1101/2023.08.22.23294277'> 10.1101/2023.08.22.23294277</a></p>
        <p class="abstract">AimsLimited recent data exist on prescribing patterns and patient characteristics for glucagon-like peptide 1 receptor agonists (GLP-1 RAs), an important drug class for patients with type 2 diabetes mellitus (T2D) and/or overweight or obesity. We describe trends in first-time prescribing of GLP-1 RAs.

Materials and MethodsUsing aggregated US electronic health record data, we identified first-time prescriptions of GLP-1 RAs between January 2018 and June 2023 for adults receiving regular care. We describe prescribing volumes stratified by patient characteristics, specific drug, and FDA-labelled indication. The 5.5-year analysis period was divided into six-month periods.

ResultsIn total, 274,562 patients were newly prescribed a GLP-1 RA, with a significant increase over time (January to June, 2018: 9,642 versus 2023: 66,569; p &lt; 0.001). Overall, 181,860 (66.2%) patients had T2D and 229,715 (83.7%) had obesity or overweight. The proportion with T2D decreased over time (January through June, 2018: 84.1% versus 2023: 49%; p &lt; 0.001), while the proportion with overweight or obesity increased (January through June, 2018: 75.9% versus 2023: 90.7%; p &lt;.001). Of prescriptions with a known FDA-labelled indication (74.2%), 87% were labelled for T2D and 13% were labelled for overweight or obesity. Patients first prescribed a GLP-1 RA labelled for T2D were 59.1% female with a mean (SD) age of 58.6 (13) years, while those prescribed a GLP-1 RA labelled for overweight or obesity were 82.9% female, with a mean (SD) age of 48.2 (12.2) years.

ConclusionsWe observed an increase in first-time prescribing of GLP-1 RAs overall, and a shift away from a predominately T2D population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.23293833">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.23293833" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.23293833">
        <p class="paperTitle">One Health Genomic Surveillance of Avian and Human Influenza A Viruses Through Environmental Wastewater Monitoring</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.23293833" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.23293833" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, A. J.; Carson, S.; Reyne, M. I.; Marshall, A.; Moody, D.; Allen, D.; Allingham, P.; Levickas, A.; Fitzgerald, A.; Bell, S. H.; Lock, J.; Lock, J. D.; McSparron, C.; Nejad, B. F.; Courtney, D. G.; Einarsson, G. G.; McKenna, J. P.; Fairley, D. J.; Curran, T.; McKinley, J. M.; Gilpin, D. F.; Lemon, K.; McGrath, J. W.; Bamford, C. G.</p>
        <p class="info">Score: 15.3, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.23293833' target='https://doi.org/10.1101/2023.08.08.23293833'> 10.1101/2023.08.08.23293833</a></p>
        <p class="abstract">BackgroundInfluenza A viruses (IAV) are significant pathogens of humans and other animals. Although endemic in humans and birds, novel IAV strains can emerge, jump species, and cause epidemics, like the latest variant of H5N1. Wastewater-based epidemiology (WBE) has very recently been shown to detect human IAV but whether it can detect avian-origin IAV, and if whole genome sequencing (WGS) can be used to discriminate circulating strains of IAV in wastewater remains unknown.

MethodsUsing a pan-IAV RT-qPCR assay, six wastewater treatment works (WWTWs) across Northern Ireland (NI), were screened from August to December 2022. A WGS approach using Oxford Nanopore technology was employed to sequence positive samples. Phylogenetic analysis of sequences relative to currently circulating human and avian IAVs was performed.

FindingsWe detected a dynamic IAV signal in wastewater from September 2022 onwards across NI. &#34;Meta&#34; whole genome sequences were generated displaying homology to both human and avian IAV strains. The relative proportion of human versus avian-origin IAV reads differed across time and sample site. A diversity in subtypes and lineages was detected (e.g. H1N1, H3N2, and several avian). Avian segment 8 related to those found in recent H5N1 clade 2.3.4.4b was identified.

InterpretationWBE affords a means to monitor circulating human and avian IAV strains and provide crucial genetic information. As such WBE can provide rapid, cost-effective, year-round &#34;one-health&#34; IAV surveillance to help control epidemic and pandemic threats.

FundingThis study was funded by the Department of Health for Northern Ireland as part of the Northern Ireland Wastewater Surveillance Programme.

HighlightsO_LIDynamic IAV RT-qPCR signal in wastewater detected across NI.
C_LIO_LINanopore-based WGS reveals presence of both human and avian IAVs in wastewater.
C_LIO_LIAvian IAV sequence similarity to gull-associated H13/H16 and recent H5N1 isolates.
C_LIO_LICo-detection of distinct clades of human H1N1 and H3N2 subtypes.
C_LI

Author SummaryInfluenza A virus (IAV) is a major pathogen of humans and other animals and causes regular epidemics and devastating pandemics. Recently, a novel variant of highly-pathogenic H5N1 avian influenza has emerged spreading across the world killing millions of birds and infecting mammals, enhancing its pandemic potential. Strengthening global surveillance systems for human and animal IAV is thus a major priority. Wastewater-based epidemiology (WBE) has been applied to track SARS-CoV-2 and IAV in humans but whether this approach could work for avian IAV is not known. Here, we develop a &#34;one-health&#34; method to survey pan-IAV levels and genetically characterise the viruses. Through this we highlight co-detection of human and avian IAVs in wastewater, with homology to recent H5N1 isolates. Our work demonstrates the potential for WBE to help defend against not only human infections but emerging, zoonotic IAVs of pandemic potential.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=133 SRC=&#34;FIGDIR/small/23293833v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (26K):
org.highwire.dtl.DTLVardef@a7b3c4org.highwire.dtl.DTLVardef@b2c12org.highwire.dtl.DTLVardef@17a81faorg.highwire.dtl.DTLVardef@1eb5c70_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.23294700">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.23294700" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.23294700">
        <p class="paperTitle">MEX-PD: A National Network for the Epidemiological &amp; Genetic Research of Parkinson&#39;s Disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.23294700" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.23294700" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lazaro-Figueroa, A.; Reyes-Perez, P.; Morelos-Figaredo, E.; Guerra-Galicia, C. M.; Estrada-Bellmann, I.; Salinas-Barboza, K.; Matuk, Y.; Gandarilla-Martinez, N. A.; Caballero-Sanchez, U.; Flores-Ocampo, V.; Montes-Alcantara, P.; Espinosa-Mendez, I.; Zayas-Del Moral, A.; Gaspar-Martinez, E.; Vazquez-Guevara, D.; Rodriguez-Violante, M.; Inca-Martinez, M.; Mata, I. F.; Alcauter, S.; Renteria, M. E.; Medina-Rivera, A.; Ruiz-Contreras, A. E.</p>
        <p class="info">Score: 4.3, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.23294700' target='https://doi.org/10.1101/2023.08.28.23294700'> 10.1101/2023.08.28.23294700</a></p>
        <p class="abstract">Background Parkinson&#39;s Disease (PD) has a complex etiology, involving genetic and environmental factors. Most of our current understanding of the disease comes from studies in populations with mostly European ancestry, representing challenges in generalizing findings to other populations with different genetic, social, and environmental contexts. There are scarce studies focused in Latin American populations. The Mexican population is genetically diverse because its admixture from Native American, European, and African ancestries, coupled with the unique environmental conditions, stressing the relevance of establishing genetic studies in this population. Thus, we have established the Mexican Parkinson&#39;s Research Network (MEX-PD), a consortium to research the clinical, genetical, environmental, and neurophysiological bases of the phenotypic diversity in Mexican PD patients. Objectives Describing how MEX-PD was established, the methods and instruments and presenting the first results. Methods Patients and controls were recruited from medical centers in 20 states of Mexico. Initial recruitment included neurological evaluation, cognitive assessment, and DNA collection. Results MEX-PD has registered 302 controls and 262 PD patients with a mean age of diagnosis of 61 years (SD=10.86). There were 19.8% PD patients identified with early onset. Levodopa was the most common pharmacological treatment. Conclusions MEX-PD contributes to understand PD nationally. The information gathered here will allow us to understand the prevalence of mental health, neurological symptoms, and cognitive function in the PD Mexican population and how genetical and environmental factors contributes to those outcomes. These will advocate for personalized treatments and improving quality of life in the Mexican population.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
