<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>epidemiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>epidemiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Long COVID in a highly vaccinated population infected during a SARS-CoV-2 Omicron wave - Australia, 2022
Authors: Woldegiorgis, M.; Cadby, G.; Ngeh, S.; Korda, R.; Armstrong, P.; Maticevic, J.; Knight, P.; Jardine, A.; Bloomfield, L.; Effler, P.
Score: 2178.5, Published: 2023-08-09 DOI: 10.1101/2023.08.06.23293706
ObjectiveTo characterise Long COVID in a highly vaccinated population infected by Omicron. DesignFollow-up survey of persons testing positive for SARS-CoV-2 in Western Australia, 16 July-3 August 2022.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/epidemiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="epidemiology" />
<meta property="og:description" content="Long COVID in a highly vaccinated population infected during a SARS-CoV-2 Omicron wave - Australia, 2022
Authors: Woldegiorgis, M.; Cadby, G.; Ngeh, S.; Korda, R.; Armstrong, P.; Maticevic, J.; Knight, P.; Jardine, A.; Bloomfield, L.; Effler, P.
Score: 2178.5, Published: 2023-08-09 DOI: 10.1101/2023.08.06.23293706
ObjectiveTo characterise Long COVID in a highly vaccinated population infected by Omicron. DesignFollow-up survey of persons testing positive for SARS-CoV-2 in Western Australia, 16 July-3 August 2022." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/epidemiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-23T10:37:56+00:00" />
<meta property="article:modified_time" content="2023-08-23T10:37:56+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="epidemiology"/>
<meta name="twitter:description" content="Long COVID in a highly vaccinated population infected during a SARS-CoV-2 Omicron wave - Australia, 2022
Authors: Woldegiorgis, M.; Cadby, G.; Ngeh, S.; Korda, R.; Armstrong, P.; Maticevic, J.; Knight, P.; Jardine, A.; Bloomfield, L.; Effler, P.
Score: 2178.5, Published: 2023-08-09 DOI: 10.1101/2023.08.06.23293706
ObjectiveTo characterise Long COVID in a highly vaccinated population infected by Omicron. DesignFollow-up survey of persons testing positive for SARS-CoV-2 in Western Australia, 16 July-3 August 2022."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "epidemiology",
      "item": "https://trxiv.yorks0n.com/posts/epidemiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "epidemiology",
  "name": "epidemiology",
  "description": "Long COVID in a highly vaccinated population infected during a SARS-CoV-2 Omicron wave - Australia, 2022\nAuthors: Woldegiorgis, M.; Cadby, G.; Ngeh, S.; Korda, R.; Armstrong, P.; Maticevic, J.; Knight, P.; Jardine, A.; Bloomfield, L.; Effler, P.\nScore: 2178.5, Published: 2023-08-09 DOI: 10.1101/2023.08.06.23293706\nObjectiveTo characterise Long COVID in a highly vaccinated population infected by Omicron. DesignFollow-up survey of persons testing positive for SARS-CoV-2 in Western Australia, 16 July-3 August 2022.",
  "keywords": [
    
  ],
  "articleBody": " Long COVID in a highly vaccinated population infected during a SARS-CoV-2 Omicron wave - Australia, 2022\nAuthors: Woldegiorgis, M.; Cadby, G.; Ngeh, S.; Korda, R.; Armstrong, P.; Maticevic, J.; Knight, P.; Jardine, A.; Bloomfield, L.; Effler, P.\nScore: 2178.5, Published: 2023-08-09 DOI: 10.1101/2023.08.06.23293706\nObjectiveTo characterise Long COVID in a highly vaccinated population infected by Omicron. DesignFollow-up survey of persons testing positive for SARS-CoV-2 in Western Australia, 16 July-3 August 2022. SettingCommunity Participants22,744 persons with COVID-19 who had agreed to participate in research at the time of diagnosis were texted a survey link 90 days later; non-responders were telephoned. Post stratification weights were applied to responses from 11,697 (51.4%) participants, 94.0% of whom had received \u003e= 3 vaccine doses. Main outcome measuresPrevalence of Long COVID - defined as reporting new or ongoing COVID-19 illness-related symptoms or health issues 90 days post diagnosis; associated health care utilisation, reductions in work/study and risk factors were assessed using log-binomial regression. Results18.2% (n=2,130) of respondents met case definition for Long COVID. Female sex, being 50-69 years of age, pre-existing health issues, residing in a rural or remote area, and receiving fewer vaccine doses were significant independent predictors of Long COVID (p \u003c 0.05). Persons with Long COVID reported a median of 6 symptoms, most commonly fatigue (70.6%) and difficulty concentrating (59.6%); 38.2% consulted a GP and 1.6% reported hospitalisation in the month prior to the survey due to ongoing symptoms. Of 1,778 respondents with Long COVID who were working/studying before their COVID-19 diagnosis, 17.9% reported reducing/discontinuing work/study. Conclusion90 days post Omicron infection, almost 1 in 5 respondents reported Long COVID symptoms; 1 in 15 of all persons with COVID-19 sought healthcare for associated health concerns \u003e=2 months after the acute illness. Significance of the studyO_ST_ABSThe knownC_ST_ABSThe prevalence of Long COVID varies widely across studies conducted in diverse settings globally (range: 9%-81%). The newIn a highly vaccinated population (94% with \u003e=3 vaccine doses), almost 20% of persons infected with the SARS-CoV-2 Omicron variant reported symptoms consistent with Long COVID 90 days post diagnosis. Long COVID was associated with sustained negative impacts on work/study and a substantial utilisation of GP services 2-3 months after the acute illness; however, ED presentations and hospitalisations for Long COVID were rare. The implicationsGP clinics play a significant role in managing the burden of Long COVID in Australia.\nImproving estimates of epidemiological quantities by combining reported cases with wastewater data: a statistical framework with applications to COVID-19 in Aotearoa New Zealand\nAuthors: Watson, L. M.; Plank, M. J.; Armstrong, B.; Chapman, J.; Hewitt, J.; Morris, H.; Orsi, A.; Bunce, M.; Donnelly, C. A.; Steyn, N.\nScore: 62.4, Published: 2023-08-16 DOI: 10.1101/2023.08.14.23294060\nBackgroundTimely and informed public health responses to infectious diseases such as COVID-19 necessitate reliable information about infection dynamics. The case ascertainment rate (CAR), the proportion of infections that are reported as cases, is typically much less than one and varies with testing practices and behaviours, making reported cases unreliable as the sole source of data. The concentration of viral RNA in wastewater samples provides an alternate measure of infection prevalence that is not affected by human behaviours. Here, we investigated how these two data sources can be combined to inform estimates of the instantaneous reproduction number, R, and track changes in the CAR over time. MethodsWe constructed a state-space model that we solved using sequential Monte Carlo methods. The observed data are the levels of SARS-CoV-2 in wastewater and reported case incidence. The hidden states that we estimate are R and CAR. Model parameters are estimated using the particle marginal Metropolis Hastings algorithm. FindingsWe analysed data from 1 January 2022 to 31 March 2023 from Aotearoa New Zealand. Our model estimates that R peaked at 2.76 (95% CrI 2.20, 3.83) around 18 February 2022 and the CAR peaked around 12 March 2022. Accounting for reduced CAR, we estimate that New Zealands second Omicron wave in July 2022 was similar in size to the first, despite fewer reported cases. We estimate that the CAR in the BA.5 Omicron wave in July 2022 was approximately 50% lower than in the BA.1/BA.2 Omicron wave in March 2022. The CAR in subsequent waves around November 2022 and April 2023 was estimated to be comparable to that in the second Omicron wave. InterpretationThis work on wastewater-based epidemiology (WBE) can be used to give insight into key epidemiological quantities. Estimating R, CAR, and cumulative number of infections provides useful information for planning public health responses and understanding the state of immunity in the population. This model is a useful disease surveillance tool, improving situational awareness of infectious disease dynamics in real-time, which may be increasingly useful as intensive pandemic surveillance programmes are wound down. FundingNew Zealand Ministry of Health, Department of Prime Minister and Cabinet, the Royal Society Te Ap[a]rangi, Imperial College London, and University of Oxford. Research in ContextO_ST_ABSEvidence before this studyC_ST_ABSThere has been a substantial increase in the number of publications focusing on wastewater-based epidemiology (WBE) in recent years, particularly during the COVID-19 pandemic. We searched PubMed for \"wastewater based epidemiology\" and found fewer than 10 papers per year prior to 2014 with a drastic increase to 463 in 2022. Approximately 52% of the WBE publications are related to COVID-19 (\"wastewater based epidemiology\" AND (\"SARS-CoV-2\" OR \"COVID-19\")). Many studies have focused on detecting SARS-CoV-2 in wastewater systems but only 10 have estimated the reproduction number (\"wastewater based epidemiology\" AND (\"SARS-CoV-2\" OR \"COVID-19\") AND \"reproduction number\"). No previous work has combined WBE with reported case data to estimate (relative) case ascertainment rate (\"waste-water based epidemiology\" AND (\"SARS-CoV-2\" OR \"COVID-19\") AND \"case ascertainment rate\"). Previous work has estimated the reproduction number from reported cases assuming constant under-ascertainment but the issue of time-varying case ascertainment has not yet been addressed, except to demonstrate the effect of a pre-determined change in case ascertainment. Added value of this studyWe present a model that, for the first time, enables reported case information to be combined with wastewater data to estimate epidemiology quantities. This work further demonstrates the utility of WBE; the reproduction number can be estimated in the absence of reported case information (although results are more reliable when case data are included), and wastewater data include information that, when combined with case data, can be used to estimate the time-varying relative case ascertainment rate. Implications of all the available evidenceIn order to make informed and timely public health decisions about infectious diseases, it is important to understand the number of infections in the community. WBE provides a useful source of data that is not impacted by time-varying testing practices. Wastewater data can be quantitatively combined with case information to better understand the state of an epidemic. In order to determine the absolute case ascertainment rate (rather than the relative rate calculated in this work), there is a need for infection prevalence surveys to calibrate model results against.\nInfluence of Prior SARS-CoV-2 Infection on COVID-19 Severity: Evidence from the National COVID Cohort Collaborative\nAuthors: Hendrix, N.; Sidky, H.; Sahner, D.; N3C Consortium, Score: 568.4, Published: 2023-08-06 DOI: 10.1101/2023.08.03.23293612\nBackgroundAs SARS-CoV-2 has transitioned from a pandemic to endemic disease, the majority of new infections have been among previously infected individuals. To manage the risks and benefits of ongoing COVID-19 policies, it is important to understand whether prior infection modifies the severity of subsequent infections. MethodsWe used data from first and second COVID-19 episodes in the National COVID Cohort Collaborative (N3C), a collection of health systems who provide de-identified electronic health records for research purposes. Our analysis was a sequential series of nested trial emulations. In the first of two analytic stages, we created a month-specific model of the probability of prior infection for each individual. In the second stage, we used an ordinal logistic regression with inverse probability weights calculated in the first stage to simulate a series of monthly trials comparing severity between the cohorts of first and second infections. In addition to cohort-wide effect estimates, we also conducted analyses among race/ethnicity, sex, and age subgroups. ResultsFrom an initial cohort of 7,446,481 combined first and second infections, we identified a cohort of 2,227,484 infections, among which 7.6% were second infections. Ninety-four percent of patients with two recorded infections experienced mild disease for both. The overall odds ratio (OR) for more severe disease with prior infection was 1.06 (95% confidence interval [CI]: 1.03 - 1.10). Monthly point estimates of the OR ranged from 0.56 (95% CI: 0.37 - 0.84) in October 2020 to 1.64 (95% CI: 1.33 - 2.00) in February 2023. In most subgroups, the effect of prior infection was significant. In 8 out of 10 subgroups, the maximum monthly OR occurred after the minimum monthly OR, suggesting that protection has waned throughout the pandemic. ConclusionOverall, prior infection was associated with a significant slightly elevated risk of severe disease. This effect varied month to month. As the pandemic proceeded, the effect of prior infection tended to evolve from generally protective during the pre-Omicron era to unprotective during the Omicron era. This points to the need for continued strategies to avert and minimize the harms of COVID-19, rather than relying upon immunity acquired through previous infection. Key PointsO_ST_ABSQuestionC_ST_ABSDoes prior infection with SARS-CoV-2 affect the severity of subsequent COVID-19 episodes? FindingsWe observed a mild protective effect of prior infection during the early and mid-stages of the pandemic that waned after the rise of the Omicron variants, ultimately resulting in loss of protection or a tendency toward more severe second infections. MeaningPrior infection alone is likely not enough to avert the worst public health harms of endemic SARS-CoV-2. Interventions to avoid infection and reduce the severity of COVID-19 will still be important in the post-pandemic era.\nDiabetes following SARS-CoV-2 infection: Incidence, persistence, and implications of COVID-19 vaccination. A cohort study of fifteen million people.\nAuthors: Taylor, K.; Eastwood, S.; Walker, V.; Cezard, G.; Knight, R.; Al Arab, M.; Wei, Y.; Horne, E. M. F.; Teece, L.; Forbes, H.; Walker, A.; Fisher, L.; Massey, J.; Hopcroft, L. E. M.; Palmer, T.; Cuitun Coronado, J.; Ip, S.; Davy, S.; Dillingham, I.; Morton, C.; Greaves, F.; MacLeod, J.; Goldacre, B.; Wood, A.; Chaturvedi, N.; Sterne, J. A. C.; Denholm, R.; CONVALESCENCE Long-COVID study, ; Longitudinal Health and Wellbeing and Data and Connectivity UK COVID-19 National Core Studies, ; OpenSAFELY collaborative, Score: 831.8, Published: 2023-08-09 DOI: 10.1101/2023.08.07.23293778\nBackgroundType 2 diabetes (T2DM) incidence is increased after diagnosis of COVID-19. The impact of vaccination on this increase, for how long it persists, and the effect of COVID-19 on other types of diabetes remain unclear. MethodsWith NHS England approval, we studied diabetes incidence following COVID-19 diagnosis in pre-vaccination (N=15,211,471, January 2020-December 2021), vaccinated (N =11,822,640), and unvaccinated (N=2,851,183) cohorts (June-December 2021), using linked electronic health records. We estimated adjusted hazard ratios (aHRs) comparing diabetes incidence post-COVID-19 diagnosis with incidence before or without diagnosis up to 102 weeks post-diagnosis. Results were stratified by COVID-19 severity (hospitalised/non-hospitalised) and diabetes type. FindingsIn the pre-vaccination cohort, aHRS for T2DM incidence after COVID-19 (compared to before or without diagnosis) declined from 3.01 (95% CI: 2.76,3.28) in weeks 1-4 to 1.24 (1.12,1.38) in weeks 53-102. aHRS were higher in unvaccinated than vaccinated people (4.86 (3.69,6.41)) versus 1.42 (1.24,1.62) in weeks 1-4) and for hospitalised COVID-19 (pre-vaccination cohort 21.1 (18.8,23.7) in weeks 1-4 declining to 2.04 (1.65,2.51) in weeks 52-102), than non-hospitalised COVID-19 (1.45 (1.27,1.64) in weeks 1-4, 1.10 (0.98,1.23) in weeks 52-102). T2DM persisted for 4 months after COVID-19 for [~]73% of those diagnosed. Patterns were similar for Type 1 diabetes, though excess incidence did not persist beyond a year post-COVID-19. InterpretationElevated T2DM incidence after COVID-19 is greater, and persists longer, in hospitalised than non-hospitalised people. It is markedly less apparent post-vaccination. Testing for T2DM after severe COVID-19 and promotion of vaccination are important tools in addressing this public health problem. Research in contextO_ST_ABSEvidence before this studyC_ST_ABSWe searched PubMed for population-based observational studies published between December 1st 2019 and July 12th 2023 examining associations between SARS-CoV-2 infection or COVID-19 diagnosis (search string: SARS-CoV-2 or COVID* or coronavirus*) and subsequent incident diabetes (search term: diabetes). Of nineteen relevant studies; eight had a composite outcome of diabetes types, six stratified by diabetes type and five pertained to type-1-diabetes (T1DM) only. We did not identify any studies relating to gestational or other types of diabetes. Eleven studies were from the US, three from the UK, two from Germany, one from Canada, one from Denmark and one from South Korea. Most studies described cumulative relative risks (for infection versus no infection) one to two years post-SARS-CoV-2 infection of 1.2 to 2.6, though four studies found no associations with T1DM after the post-acute period. All studies lacked the power to compare diabetes relative risk by type, severity, and vaccination status in population subgroups. One study examined relative risks by vaccination status, but this used a composite outcome of diabetes and hyperlipidaemia and was conducted in a predominantly white male population. Two studies of T1DM found no evidence of elevated risk beyond 30 days after COVID-19 diagnosis, whilst two reported elevated risks at six months. Two studies of type 2 diabetes (T2DM) examined relative risks by time period post-infection: one study of US insurance claims reported a persistent association six months post-infection, whereas a large UK population-based study reported no associations after 12 weeks. However, the latter study used only primary care data, therefore COVID-19 cases were likely to have been under-ascertained. No large studies have investigated the persistence of diabetes diagnosed following COVID-19; key to elucidating the role of stress/steroid-induced hyperglycaemia. Added value of this studyThis study, which is the largest to address the question to date, analysed linked primary and secondary care health records with SARS-CoV-2 testing and COVID-19 vaccination data for 15 million people living in England. This enabled us to compare the elevation in diabetes incidence after COVID-19 diagnosis by diabetes type, COVID-19 severity and vaccination status, overall and in population subgroups. Importantly, excess diabetes incidence by time period since infection could also be quantified. Since healthcare in the UK is universal and free-at-the-point-of-delivery, almost the entire population is registered with primary care. Therefore the findings are likely to be generalisable. We found that, before availability of COVID-19 vaccination, a COVID-19 diagnosis (vs. no diagnosis) was associated with increased T2DM incidence which remained elevated by approximately 30% beyond one year after diagnosis. Though still present (with around 30% excess incidence at eight weeks), these associations were substantially attenuated in unvaccinated compared with vaccinated people. Excess incidence was greater in people hospitalised with COVID-19 than those who were not hospitalised after diagnosis. T1DM incidence was elevated up to, but not beyond, a year post COVID-19. Around 73% of people diagnosed with incident T2DM after COVID-19 still had evidence of diabetes four months after infection. Implications of all the available evidenceThere is a 30-50% elevated T2DM incidence post-COVID-19, but we report the novel finding that there is elevated incidence beyond one-year post-diagnosis. Elevated T1DM incidence did not appear to persist beyond a year, which may explain why previous studies disagree. For the first time in a general-population dataset, we demonstrate that COVID-19 vaccination reduces, but does not entirely ameliorate, excess diabetes incidence after COVID-19. This supports a policy of universal vaccination and suggests that other public health activities, such as enhanced diabetes screening after severe COVID-19, may be warranted, particularly in unvaccinated people.\nEmployment outcomes of people with Long Covid symptoms: community-based cohort study\nAuthors: Ayoubkhani, D.; Zaccardi, F.; Pouwels, K. B.; Walker, A. S.; Houston, D.; Alwan, N. A.; Martin, J.; Khunti, K.; Nafilyan, V.\nScore: 69.7, Published: 2023-08-15 DOI: 10.1101/2023.03.21.23287524\nBackgroundEvidence on the long-term employment consequences of SARS-CoV-2 infection is lacking. We used data from a large, community-based sample in the UK to estimate associations between Long Covid and subsequent employment outcomes. MethodsThis was an observational, longitudinal study using a pre-post design. We included survey participants from 3 February 2021 to 30 September 2022 when they were aged 16 to 64 years and not in full-time education. Using conditional logit modelling, we explored the time-varying relationship between Long Covid status [\u0026ge;]12 weeks after a first test-confirmed SARS-CoV-2 infection (reference: pre-infection) and labour market inactivity (neither working nor looking for work) or workplace absence lasting [\u0026ge;]4 weeks. ResultsOf 206,299 included participants (mean age 45 years, 54% female, 92% white), 15% were ever inactive in the labour market and 10% were ever long-term absent during follow-up. Compared with pre-infection, inactivity was higher in participants reporting Long Covid 30 to \u003c40 weeks (adjusted odds ratio (aOR): 1.45; 95% CI: 1.17 to 1.81) or 40 to \u003c52 weeks (1.34; 1.05 to 1.72) post-infection. Combining with official statistics on Long Covid prevalence, our estimates translate to 27,000 (95% CI: 6,000 to 47,000) working-age adults in the UK being inactive because of Long Covid in July 2022. ConclusionsLong Covid is likely to have contributed to reduced levels of participation in the UK labour market, though it is unlikely to be the sole driver. Further research is required to quantify the contribution of other factors, such as indirect health effects of the pandemic.\nAssociation of Life's Essential 8 with Cardiovascular Events and Mortality: The Cardiovascular Disease Lifetime Risk Pooling Project (LRPP)\nAuthors: Ning, H.; Allen, N.; Lloyd-Jones, D. M.; Perak, A. M.; Siddique, J.; Wilkins, J. T.\nScore: 14.3, Published: 2023-08-14 DOI: 10.1101/2023.08.08.23293855\nBackgroundThe American Heart Association (AHA) recently launched updated cardiovascular health (CVH) metrics, termed Lifes Essential 8 (LE8). Compared to Lifes Simple 7 (LS7), the new approach added sleep health as an eighth metric and updated the remaining 7 metrics. Whether the updated LE8 score outperforms the original LS7 score in predicting cardiovascular disease (CVD) is not known. We examined the association of LE8 scores with CVD, subtype CVD events and all-cause mortality. MethodsWe pooled individual-level data from 6 contemporary US-based cohorts from the Cardiovascular Lifetime Risk Pooling Project (LRPP). Total LE8 score (0-100 points), LE8 score without sleep (0-100 points), as well as prior LS7 scores (0-14 points), were calculated separately. We used multivariable-adjusted Cox models to evaluate the association of LE8 with CVD, CVD subtypes, and all-cause mortality among younger, middle, and older aged adult participants. Reclassification was defined based on the concordant/discordant categories of LS7 and LE8 scores quartile rankings across age groups. ResultsOur sample consisted of 32,896 US adults (7836 [23.8%] Black; 14941 [45.4%] men) followed for 642,000 person-years; of whom 9,391 developed CVD events. Each 10-point higher overall LE8 score was associated with 23-40% lower CVD risk across age groups. Reclassification of CVH from LS7 to LE8 was related to heath behaviors as well as health factors and was significantly associated with CVD risk. ConclusionsThese findings support the improved utility of the LE8 algorithm for assessing overall cardiovascular health and future CVD risk.\nSleep Regularity and Mortality: A Prospective Analysis in the UK Biobank\nAuthors: Cribb, L.; Sha, R.; Yiallourou, S.; Grima, N. A.; Cavuoto, M.; Baril, A.-A.; Pase, M. P.\nScore: 58.3, Published: 2023-08-15 DOI: 10.1101/2023.04.14.23288550\nBackgroundIrregular sleep-wake timing may cause circadian disruption leading to several chronic age-related diseases. We examined the relationship between sleep regularity and risk of all-cause, cardiovascular disease (CVD), and cancer mortality in 88,975 participants from the prospective UK Biobank cohort. MethodsThe sleep regularity index (SRI) was calculated as the probability of an individual being in the same state (asleep or awake) at any two time points 24 hours apart, averaged over 7-days of accelerometry (range 0-100, with 100 being perfectly regular). The SRI was related to the risk of mortality in time-to-event models. FindingsThe mean sample age was 62 years (SD, 8), 56% were women, and the median SRI was 60 (SD, 10). There were 3010 deaths during a mean follow-up of 7.1 years. Following adjustments for demographic and clinical variables, we identified a non-linear relationship between the SRI and all-cause mortality hazard (p [global test of spline term] \u003c 0{middle dot}001). Hazard Ratios, relative to the median SRI, were 1{middle dot}53 (95% confidence interval [CI]: 1{middle dot}41, 1{middle dot}66) for participants with SRI at the 5th percentile (SRI = 41) and 0{middle dot}90 (95% CI: 0{middle dot}81, 1{middle dot}00) for those with SRI at the 95th percentile (SRI = 75), respectively. Findings for CVD mortality and cancer mortality followed a similar pattern. ConclusionsIrregular sleep-wake patterns are associated with higher mortality risk. FundingNational Health and Medical Research Council of Australia (GTN2009264; GTN1158384), National Institute on Aging (AG062531), Alzheimers Association (2018-AARG-591358), and the Banting Fellowship Program (#454104).\nOne Health Genomic Surveillance of Avian and Human Influenza A Viruses Through Environmental Wastewater Monitoring\nAuthors: Lee, A. J.; Carson, S.; Reyne, M. I.; Marshall, A.; Moody, D.; Allen, D.; Allingham, P.; Levickas, A.; Fitzgerald, A.; Bell, S. H.; Lock, J.; Lock, J. D.; McSparron, C.; Nejad, B. F.; Courtney, D. G.; Einarsson, G. G.; McKenna, J. P.; Fairley, D. J.; Curran, T.; McKinley, J. M.; Gilpin, D. F.; Lemon, K.; McGrath, J. W.; Bamford, C. G.\nScore: 12.7, Published: 2023-08-13 DOI: 10.1101/2023.08.08.23293833\nBackgroundInfluenza A viruses (IAV) are significant pathogens of humans and other animals. Although endemic in humans and birds, novel IAV strains can emerge, jump species, and cause epidemics, like the latest variant of H5N1. Wastewater-based epidemiology (WBE) has very recently been shown to detect human IAV but whether it can detect avian-origin IAV, and if whole genome sequencing (WGS) can be used to discriminate circulating strains of IAV in wastewater remains unknown. MethodsUsing a pan-IAV RT-qPCR assay, six wastewater treatment works (WWTWs) across Northern Ireland (NI), were screened from August to December 2022. A WGS approach using Oxford Nanopore technology was employed to sequence positive samples. Phylogenetic analysis of sequences relative to currently circulating human and avian IAVs was performed. FindingsWe detected a dynamic IAV signal in wastewater from September 2022 onwards across NI. \"Meta\" whole genome sequences were generated displaying homology to both human and avian IAV strains. The relative proportion of human versus avian-origin IAV reads differed across time and sample site. A diversity in subtypes and lineages was detected (e.g. H1N1, H3N2, and several avian). Avian segment 8 related to those found in recent H5N1 clade 2.3.4.4b was identified. InterpretationWBE affords a means to monitor circulating human and avian IAV strains and provide crucial genetic information. As such WBE can provide rapid, cost-effective, year-round \"one-health\" IAV surveillance to help control epidemic and pandemic threats. FundingThis study was funded by the Department of Health for Northern Ireland as part of the Northern Ireland Wastewater Surveillance Programme. HighlightsO_LIDynamic IAV RT-qPCR signal in wastewater detected across NI. C_LIO_LINanopore-based WGS reveals presence of both human and avian IAVs in wastewater. C_LIO_LIAvian IAV sequence similarity to gull-associated H13/H16 and recent H5N1 isolates. C_LIO_LICo-detection of distinct clades of human H1N1 and H3N2 subtypes. C_LI Author SummaryInfluenza A virus (IAV) is a major pathogen of humans and other animals and causes regular epidemics and devastating pandemics. Recently, a novel variant of highly-pathogenic H5N1 avian influenza has emerged spreading across the world killing millions of birds and infecting mammals, enhancing its pandemic potential. Strengthening global surveillance systems for human and animal IAV is thus a major priority. Wastewater-based epidemiology (WBE) has been applied to track SARS-CoV-2 and IAV in humans but whether this approach could work for avian IAV is not known. Here, we develop a \"one-health\" method to survey pan-IAV levels and genetically characterise the viruses. Through this we highlight co-detection of human and avian IAVs in wastewater, with homology to recent H5N1 isolates. Our work demonstrates the potential for WBE to help defend against not only human infections but emerging, zoonotic IAVs of pandemic potential. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=133 SRC=\"FIGDIR/small/23293833v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (26K): org.highwire.dtl.DTLVardef@a7b3c4org.highwire.dtl.DTLVardef@b2c12org.highwire.dtl.DTLVardef@17a81faorg.highwire.dtl.DTLVardef@1eb5c70_HPS_FORMAT_FIGEXP M_FIG C_FIG\nStrategic use of SARS-CoV-2 wastewater concentration data could enhance, but not replace, high-resolution community prevalence survey programmes\nAuthors: Mills, C.; Chadeau-Hyam, M.; Elliott, P. L.; Donnelly, C. A.\nScore: 3.7, Published: 2023-08-20 DOI: 10.1101/2023.08.17.23293589\nWastewater-based epidemiology (WBE) has been proposed as a tool for public health authorities to monitor community transmission of SARS-CoV-2 and other agents. Here, we review the utility of WBE for estimating SARS-CoV-2 prevalence using wastewater data from the Environmental Monitoring for Health Protection (EMHP) programme and prevalence data from the REal-time Assessment of Community Transmission-1 (REACT-1) study in England. Our analysis shows a temporally evolving relationship between wastewater and prevalence which limits the utility of WBE for estimating SARS-CoV-2 prevalence in high spatial resolution without a concurrent prevalence survey. We further characterise WBE for SARS-CoV-2 prevalence as i) vaccination-coverage-dependent and ii) variant-specific. Our work provides a gesopatial framework to map wastewater concentrations to public health boundaries, enabling public health authorities to interpret the relationship between wastewater and prevalence. We demonstrate that WBE can improve the cost efficiency and accuracy of community prevalence surveys which on their own may have incomplete geographic coverage or small sample sizes.\nUnraveling COVID-19: Descriptive Analytics in a Middle-Income Country, Paving the Path Forward\nAuthors: Bansilan, N. P.; Rabajante, J. F.\nScore: 3.0, Published: 2023-08-20 DOI: 10.1101/2023.08.16.23294160\nThe outbreak of COVID-19 unleashed an unprecedented global pandemic, leaving a profound impact on lives and economies worldwide. Recognizing its severity, the World Health Organization swiftly declared it a public health emergency of international concern. Tragically, the Philippines reported the first death case outside China, leading to a surge in cases following the first instance of local transmission. In response to this crisis, collaborative efforts have been underway to control the disease and minimize its health and socio-economic impacts. The COVID-19 epidemic curve holds vital insights into the history of exposure, transmission, testing, tracing, social distancing measures, community lockdowns, quarantine, isolation, and treatment, offering a comprehensive perspective on the nations response. One approach to gaining crucial insights is through meticulous analysis of available datasets, empowering us to inform future strategies and responses effectively. This paper aims to provide descriptive data analytics of the COVID-19 pandemic in the Philippines, summarizing the countrys fight by visualizing epidemiological and mobility datasets, revisiting scientific papers and news articles, and creating a timeline of the key issues faced during the pandemic. By leveraging these multifaceted analyses, policymakers and health authorities can make informed decisions to enhance preparedness, expand inter-agency cooperation, and combat future public health crises effectively. This study seeks to serve as a valuable resource, guiding nations worldwide in comprehending and responding to the challenges posed by COVID-19 and beyond.\n",
  "wordCount" : "4355",
  "inLanguage": "en",
  "datePublished": "2023-08-23T10:37:56Z",
  "dateModified": "2023-08-23T10:37:56Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/epidemiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      epidemiology
    </h1>
    <div class="post-meta"><span>updated on August 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.23293706">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.23293706" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.23293706">
        <p class="paperTitle">Long COVID in a highly vaccinated population infected during a SARS-CoV-2 Omicron wave - Australia, 2022</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.23293706" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.23293706" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Woldegiorgis, M.; Cadby, G.; Ngeh, S.; Korda, R.; Armstrong, P.; Maticevic, J.; Knight, P.; Jardine, A.; Bloomfield, L.; Effler, P.</p>
        <p class="info">Score: 2178.5, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.23293706' target='https://doi.org/10.1101/2023.08.06.23293706'> 10.1101/2023.08.06.23293706</a></p>
        <p class="abstract">ObjectiveTo characterise Long COVID in a highly vaccinated population infected by Omicron.

DesignFollow-up survey of persons testing positive for SARS-CoV-2 in Western Australia, 16 July-3 August 2022.

SettingCommunity

Participants22,744 persons with COVID-19 who had agreed to participate in research at the time of diagnosis were texted a survey link 90 days later; non-responders were telephoned. Post stratification weights were applied to responses from 11,697 (51.4%) participants, 94.0% of whom had received &gt;= 3 vaccine doses.

Main outcome measuresPrevalence of  Long COVID - defined as reporting new or ongoing COVID-19 illness-related symptoms or health issues 90 days post diagnosis; associated health care utilisation, reductions in work/study and risk factors were assessed using log-binomial regression.

Results18.2% (n=2,130) of respondents met case definition for Long COVID. Female sex, being 50-69 years of age, pre-existing health issues, residing in a rural or remote area, and receiving fewer vaccine doses were significant independent predictors of Long COVID (p &lt; 0.05). Persons with Long COVID reported a median of 6 symptoms, most commonly fatigue (70.6%) and difficulty concentrating (59.6%); 38.2% consulted a GP and 1.6% reported hospitalisation in the month prior to the survey due to ongoing symptoms. Of 1,778 respondents with Long COVID who were working/studying before their COVID-19 diagnosis, 17.9% reported reducing/discontinuing work/study.

Conclusion90 days post Omicron infection, almost 1 in 5 respondents reported Long COVID symptoms; 1 in 15 of all persons with COVID-19 sought healthcare for associated health concerns &gt;=2 months after the acute illness.

Significance of the studyO_ST_ABSThe knownC_ST_ABSThe prevalence of Long COVID varies widely across studies conducted in diverse settings globally (range: 9%-81%).

The newIn a highly vaccinated population (94% with &gt;=3 vaccine doses), almost 20% of persons infected with the SARS-CoV-2 Omicron variant reported symptoms consistent with Long COVID 90 days post diagnosis. Long COVID was associated with sustained negative impacts on work/study and a substantial utilisation of GP services 2-3 months after the acute illness; however, ED presentations and hospitalisations for Long COVID were rare.

The implicationsGP clinics play a significant role in managing the burden of Long COVID in Australia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.23294060">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.23294060" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.23294060">
        <p class="paperTitle">Improving estimates of epidemiological quantities by combining reported cases with wastewater data: a statistical framework with applications to COVID-19 in Aotearoa New Zealand</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.23294060" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.23294060" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Watson, L. M.; Plank, M. J.; Armstrong, B.; Chapman, J.; Hewitt, J.; Morris, H.; Orsi, A.; Bunce, M.; Donnelly, C. A.; Steyn, N.</p>
        <p class="info">Score: 62.4, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.23294060' target='https://doi.org/10.1101/2023.08.14.23294060'> 10.1101/2023.08.14.23294060</a></p>
        <p class="abstract">BackgroundTimely and informed public health responses to infectious diseases such as COVID-19 necessitate reliable information about infection dynamics. The case ascertainment rate (CAR), the proportion of infections that are reported as cases, is typically much less than one and varies with testing practices and behaviours, making reported cases unreliable as the sole source of data. The concentration of viral RNA in wastewater samples provides an alternate measure of infection prevalence that is not affected by human behaviours. Here, we investigated how these two data sources can be combined to inform estimates of the instantaneous reproduction number, R, and track changes in the CAR over time.

MethodsWe constructed a state-space model that we solved using sequential Monte Carlo methods. The observed data are the levels of SARS-CoV-2 in wastewater and reported case incidence. The hidden states that we estimate are R and CAR. Model parameters are estimated using the particle marginal Metropolis Hastings algorithm.

FindingsWe analysed data from 1 January 2022 to 31 March 2023 from Aotearoa New Zealand. Our model estimates that R peaked at 2.76 (95% CrI 2.20, 3.83) around 18 February 2022 and the CAR peaked around 12 March 2022. Accounting for reduced CAR, we estimate that New Zealands second Omicron wave in July 2022 was similar in size to the first, despite fewer reported cases. We estimate that the CAR in the BA.5 Omicron wave in July 2022 was approximately 50% lower than in the BA.1/BA.2 Omicron wave in March 2022. The CAR in subsequent waves around November 2022 and April 2023 was estimated to be comparable to that in the second Omicron wave.

InterpretationThis work on wastewater-based epidemiology (WBE) can be used to give insight into key epidemiological quantities. Estimating R, CAR, and cumulative number of infections provides useful information for planning public health responses and understanding the state of immunity in the population. This model is a useful disease surveillance tool, improving situational awareness of infectious disease dynamics in real-time, which may be increasingly useful as intensive pandemic surveillance programmes are wound down.

FundingNew Zealand Ministry of Health, Department of Prime Minister and Cabinet, the Royal Society Te Ap[a]rangi, Imperial College London, and University of Oxford.

Research in ContextO_ST_ABSEvidence before this studyC_ST_ABSThere has been a substantial increase in the number of publications focusing on wastewater-based epidemiology (WBE) in recent years, particularly during the COVID-19 pandemic. We searched PubMed for &#34;wastewater based epidemiology&#34; and found fewer than 10 papers per year prior to 2014 with a drastic increase to 463 in 2022. Approximately 52% of the WBE publications are related to COVID-19 (&#34;wastewater based epidemiology&#34; AND (&#34;SARS-CoV-2&#34; OR &#34;COVID-19&#34;)). Many studies have focused on detecting SARS-CoV-2 in wastewater systems but only 10 have estimated the reproduction number (&#34;wastewater based epidemiology&#34; AND (&#34;SARS-CoV-2&#34; OR &#34;COVID-19&#34;) AND &#34;reproduction number&#34;). No previous work has combined WBE with reported case data to estimate (relative) case ascertainment rate (&#34;waste-water based epidemiology&#34; AND (&#34;SARS-CoV-2&#34; OR &#34;COVID-19&#34;) AND &#34;case ascertainment rate&#34;). Previous work has estimated the reproduction number from reported cases assuming constant under-ascertainment but the issue of time-varying case ascertainment has not yet been addressed, except to demonstrate the effect of a pre-determined change in case ascertainment.

Added value of this studyWe present a model that, for the first time, enables reported case information to be combined with wastewater data to estimate epidemiology quantities. This work further demonstrates the utility of WBE; the reproduction number can be estimated in the absence of reported case information (although results are more reliable when case data are included), and wastewater data include information that, when combined with case data, can be used to estimate the time-varying relative case ascertainment rate.

Implications of all the available evidenceIn order to make informed and timely public health decisions about infectious diseases, it is important to understand the number of infections in the community. WBE provides a useful source of data that is not impacted by time-varying testing practices. Wastewater data can be quantitatively combined with case information to better understand the state of an epidemic. In order to determine the absolute case ascertainment rate (rather than the relative rate calculated in this work), there is a need for infection prevalence surveys to calibrate model results against.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.23293612">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.23293612" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.23293612">
        <p class="paperTitle">Influence of Prior SARS-CoV-2 Infection on COVID-19 Severity: Evidence from the National COVID Cohort Collaborative</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.23293612" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.23293612" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hendrix, N.; Sidky, H.; Sahner, D.; N3C Consortium,  </p>
        <p class="info">Score: 568.4, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.23293612' target='https://doi.org/10.1101/2023.08.03.23293612'> 10.1101/2023.08.03.23293612</a></p>
        <p class="abstract">BackgroundAs SARS-CoV-2 has transitioned from a pandemic to endemic disease, the majority of new infections have been among previously infected individuals. To manage the risks and benefits of ongoing COVID-19 policies, it is important to understand whether prior infection modifies the severity of subsequent infections.

MethodsWe used data from first and second COVID-19 episodes in the National COVID Cohort Collaborative (N3C), a collection of health systems who provide de-identified electronic health records for research purposes. Our analysis was a sequential series of nested trial emulations. In the first of two analytic stages, we created a month-specific model of the probability of prior infection for each individual. In the second stage, we used an ordinal logistic regression with inverse probability weights calculated in the first stage to simulate a series of monthly trials comparing severity between the cohorts of first and second infections. In addition to cohort-wide effect estimates, we also conducted analyses among race/ethnicity, sex, and age subgroups.

ResultsFrom an initial cohort of 7,446,481 combined first and second infections, we identified a cohort of 2,227,484 infections, among which 7.6% were second infections. Ninety-four percent of patients with two recorded infections experienced mild disease for both. The overall odds ratio (OR) for more severe disease with prior infection was 1.06 (95% confidence interval [CI]: 1.03 - 1.10). Monthly point estimates of the OR ranged from 0.56 (95% CI: 0.37 - 0.84) in October 2020 to 1.64 (95% CI: 1.33 - 2.00) in February 2023. In most subgroups, the effect of prior infection was significant. In 8 out of 10 subgroups, the maximum monthly OR occurred after the minimum monthly OR, suggesting that protection has waned throughout the pandemic.

ConclusionOverall, prior infection was associated with a significant slightly elevated risk of severe disease. This effect varied month to month. As the pandemic proceeded, the effect of prior infection tended to evolve from generally protective during the pre-Omicron era to unprotective during the Omicron era. This points to the need for continued strategies to avert and minimize the harms of COVID-19, rather than relying upon immunity acquired through previous infection.

Key PointsO_ST_ABSQuestionC_ST_ABSDoes prior infection with SARS-CoV-2 affect the severity of subsequent COVID-19 episodes?

FindingsWe observed a mild protective effect of prior infection during the early and mid-stages of the pandemic that waned after the rise of the Omicron variants, ultimately resulting in loss of protection or a tendency toward more severe second infections.

MeaningPrior infection alone is likely not enough to avert the worst public health harms of endemic SARS-CoV-2. Interventions to avoid infection and reduce the severity of COVID-19 will still be important in the post-pandemic era.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.23293778">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.23293778" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.23293778">
        <p class="paperTitle">Diabetes following SARS-CoV-2 infection: Incidence, persistence, and implications of COVID-19 vaccination. A cohort study of fifteen million people.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.23293778" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.23293778" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taylor, K.; Eastwood, S.; Walker, V.; Cezard, G.; Knight, R.; Al Arab, M.; Wei, Y.; Horne, E. M. F.; Teece, L.; Forbes, H.; Walker, A.; Fisher, L.; Massey, J.; Hopcroft, L. E. M.; Palmer, T.; Cuitun Coronado, J.; Ip, S.; Davy, S.; Dillingham, I.; Morton, C.; Greaves, F.; MacLeod, J.; Goldacre, B.; Wood, A.; Chaturvedi, N.; Sterne, J. A. C.; Denholm, R.; CONVALESCENCE Long-COVID study,  ; Longitudinal Health and Wellbeing and Data and Connectivity UK COVID-19 National Core Studies,  ; OpenSAFELY collaborative,  </p>
        <p class="info">Score: 831.8, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.23293778' target='https://doi.org/10.1101/2023.08.07.23293778'> 10.1101/2023.08.07.23293778</a></p>
        <p class="abstract">BackgroundType 2 diabetes (T2DM) incidence is increased after diagnosis of COVID-19. The impact of vaccination on this increase, for how long it persists, and the effect of COVID-19 on other types of diabetes remain unclear.

MethodsWith NHS England approval, we studied diabetes incidence following COVID-19 diagnosis in pre-vaccination (N=15,211,471, January 2020-December 2021), vaccinated (N =11,822,640), and unvaccinated (N=2,851,183) cohorts (June-December 2021), using linked electronic health records. We estimated adjusted hazard ratios (aHRs) comparing diabetes incidence post-COVID-19 diagnosis with incidence before or without diagnosis up to 102 weeks post-diagnosis. Results were stratified by COVID-19 severity (hospitalised/non-hospitalised) and diabetes type.

FindingsIn the pre-vaccination cohort, aHRS for T2DM incidence after COVID-19 (compared to before or without diagnosis) declined from 3.01 (95% CI: 2.76,3.28) in weeks 1-4 to 1.24 (1.12,1.38) in weeks 53-102. aHRS were higher in unvaccinated than vaccinated people (4.86 (3.69,6.41)) versus 1.42 (1.24,1.62) in weeks 1-4) and for hospitalised COVID-19 (pre-vaccination cohort 21.1 (18.8,23.7) in weeks 1-4 declining to 2.04 (1.65,2.51) in weeks 52-102), than non-hospitalised COVID-19 (1.45 (1.27,1.64) in weeks 1-4, 1.10 (0.98,1.23) in weeks 52-102). T2DM persisted for 4 months after COVID-19 for [~]73% of those diagnosed. Patterns were similar for Type 1 diabetes, though excess incidence did not persist beyond a year post-COVID-19.

InterpretationElevated T2DM incidence after COVID-19 is greater, and persists longer, in hospitalised than non-hospitalised people. It is markedly less apparent post-vaccination. Testing for T2DM after severe COVID-19 and promotion of vaccination are important tools in addressing this public health problem.

Research in contextO_ST_ABSEvidence before this studyC_ST_ABSWe searched PubMed for population-based observational studies published between December 1st 2019 and July 12th 2023 examining associations between SARS-CoV-2 infection or COVID-19 diagnosis (search string: SARS-CoV-2 or COVID* or coronavirus*) and subsequent incident diabetes (search term: diabetes). Of nineteen relevant studies; eight had a composite outcome of diabetes types, six stratified by diabetes type and five pertained to type-1-diabetes (T1DM) only. We did not identify any studies relating to gestational or other types of diabetes. Eleven studies were from the US, three from the UK, two from Germany, one from Canada, one from Denmark and one from South Korea.

Most studies described cumulative relative risks (for infection versus no infection) one to two years post-SARS-CoV-2 infection of 1.2 to 2.6, though four studies found no associations with T1DM after the post-acute period. All studies lacked the power to compare diabetes relative risk by type, severity, and vaccination status in population subgroups. One study examined relative risks by vaccination status, but this used a composite outcome of diabetes and hyperlipidaemia and was conducted in a predominantly white male population.

Two studies of T1DM found no evidence of elevated risk beyond 30 days after COVID-19 diagnosis, whilst two reported elevated risks at six months. Two studies of type 2 diabetes (T2DM) examined relative risks by time period post-infection: one study of US insurance claims reported a persistent association six months post-infection, whereas a large UK population-based study reported no associations after 12 weeks. However, the latter study used only primary care data, therefore COVID-19 cases were likely to have been under-ascertained.

No large studies have investigated the persistence of diabetes diagnosed following COVID-19; key to elucidating the role of stress/steroid-induced hyperglycaemia.

Added value of this studyThis study, which is the largest to address the question to date, analysed linked primary and secondary care health records with SARS-CoV-2 testing and COVID-19 vaccination data for 15 million people living in England. This enabled us to compare the elevation in diabetes incidence after COVID-19 diagnosis by diabetes type, COVID-19 severity and vaccination status, overall and in population subgroups. Importantly, excess diabetes incidence by time period since infection could also be quantified. Since healthcare in the UK is universal and free-at-the-point-of-delivery, almost the entire population is registered with primary care. Therefore the findings are likely to be generalisable.

We found that, before availability of COVID-19 vaccination, a COVID-19 diagnosis (vs. no diagnosis) was associated with increased T2DM incidence which remained elevated by approximately 30% beyond one year after diagnosis. Though still present (with around 30% excess incidence at eight weeks), these associations were substantially attenuated in unvaccinated compared with vaccinated people. Excess incidence was greater in people hospitalised with COVID-19 than those who were not hospitalised after diagnosis. T1DM incidence was elevated up to, but not beyond, a year post COVID-19. Around 73% of people diagnosed with incident T2DM after COVID-19 still had evidence of diabetes four months after infection.

Implications of all the available evidenceThere is a 30-50% elevated T2DM incidence post-COVID-19, but we report the novel finding that there is elevated incidence beyond one-year post-diagnosis. Elevated T1DM incidence did not appear to persist beyond a year, which may explain why previous studies disagree. For the first time in a general-population dataset, we demonstrate that COVID-19 vaccination reduces, but does not entirely ameliorate, excess diabetes incidence after COVID-19. This supports a policy of universal vaccination and suggests that other public health activities, such as enhanced diabetes screening after severe COVID-19, may be warranted, particularly in unvaccinated people.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.03.21.23287524">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.03.21.23287524" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.03.21.23287524">
        <p class="paperTitle">Employment outcomes of people with Long Covid symptoms: community-based cohort study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.03.21.23287524" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.03.21.23287524" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ayoubkhani, D.; Zaccardi, F.; Pouwels, K. B.; Walker, A. S.; Houston, D.; Alwan, N. A.; Martin, J.; Khunti, K.; Nafilyan, V.</p>
        <p class="info">Score: 69.7, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.03.21.23287524' target='https://doi.org/10.1101/2023.03.21.23287524'> 10.1101/2023.03.21.23287524</a></p>
        <p class="abstract">BackgroundEvidence on the long-term employment consequences of SARS-CoV-2 infection is lacking. We used data from a large, community-based sample in the UK to estimate associations between Long Covid and subsequent employment outcomes.

MethodsThis was an observational, longitudinal study using a pre-post design. We included survey participants from 3 February 2021 to 30 September 2022 when they were aged 16 to 64 years and not in full-time education. Using conditional logit modelling, we explored the time-varying relationship between Long Covid status [&amp;ge;]12 weeks after a first test-confirmed SARS-CoV-2 infection (reference: pre-infection) and labour market inactivity (neither working nor looking for work) or workplace absence lasting [&amp;ge;]4 weeks.

ResultsOf 206,299 included participants (mean age 45 years, 54% female, 92% white), 15% were ever inactive in the labour market and 10% were ever long-term absent during follow-up. Compared with pre-infection, inactivity was higher in participants reporting Long Covid 30 to &lt;40 weeks (adjusted odds ratio (aOR): 1.45; 95% CI: 1.17 to 1.81) or 40 to &lt;52 weeks (1.34; 1.05 to 1.72) post-infection. Combining with official statistics on Long Covid prevalence, our estimates translate to 27,000 (95% CI: 6,000 to 47,000) working-age adults in the UK being inactive because of Long Covid in July 2022.

ConclusionsLong Covid is likely to have contributed to reduced levels of participation in the UK labour market, though it is unlikely to be the sole driver. Further research is required to quantify the contribution of other factors, such as indirect health effects of the pandemic.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.23293855">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.23293855" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.23293855">
        <p class="paperTitle">Association of Life&#39;s Essential 8 with Cardiovascular Events and Mortality: The Cardiovascular Disease Lifetime Risk Pooling Project (LRPP)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.23293855" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.23293855" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ning, H.; Allen, N.; Lloyd-Jones, D. M.; Perak, A. M.; Siddique, J.; Wilkins, J. T.</p>
        <p class="info">Score: 14.3, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.23293855' target='https://doi.org/10.1101/2023.08.08.23293855'> 10.1101/2023.08.08.23293855</a></p>
        <p class="abstract">BackgroundThe American Heart Association (AHA) recently launched updated cardiovascular health (CVH) metrics, termed Lifes Essential 8 (LE8). Compared to Lifes Simple 7 (LS7), the new approach added sleep health as an eighth metric and updated the remaining 7 metrics. Whether the updated LE8 score outperforms the original LS7 score in predicting cardiovascular disease (CVD) is not known. We examined the association of LE8 scores with CVD, subtype CVD events and all-cause mortality.

MethodsWe pooled individual-level data from 6 contemporary US-based cohorts from the Cardiovascular Lifetime Risk Pooling Project (LRPP). Total LE8 score (0-100 points), LE8 score without sleep (0-100 points), as well as prior LS7 scores (0-14 points), were calculated separately. We used multivariable-adjusted Cox models to evaluate the association of LE8 with CVD, CVD subtypes, and all-cause mortality among younger, middle, and older aged adult participants. Reclassification was defined based on the concordant/discordant categories of LS7 and LE8 scores quartile rankings across age groups.

ResultsOur sample consisted of 32,896 US adults (7836 [23.8%] Black; 14941 [45.4%] men) followed for 642,000 person-years; of whom 9,391 developed CVD events. Each 10-point higher overall LE8 score was associated with 23-40% lower CVD risk across age groups. Reclassification of CVH from LS7 to LE8 was related to heath behaviors as well as health factors and was significantly associated with CVD risk.

ConclusionsThese findings support the improved utility of the LE8 algorithm for assessing overall cardiovascular health and future CVD risk.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.04.14.23288550">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.04.14.23288550" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.04.14.23288550">
        <p class="paperTitle">Sleep Regularity and Mortality: A Prospective Analysis in the UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.04.14.23288550" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.04.14.23288550" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cribb, L.; Sha, R.; Yiallourou, S.; Grima, N. A.; Cavuoto, M.; Baril, A.-A.; Pase, M. P.</p>
        <p class="info">Score: 58.3, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.04.14.23288550' target='https://doi.org/10.1101/2023.04.14.23288550'> 10.1101/2023.04.14.23288550</a></p>
        <p class="abstract">BackgroundIrregular sleep-wake timing may cause circadian disruption leading to several chronic age-related diseases. We examined the relationship between sleep regularity and risk of all-cause, cardiovascular disease (CVD), and cancer mortality in 88,975 participants from the prospective UK Biobank cohort.

MethodsThe sleep regularity index (SRI) was calculated as the probability of an individual being in the same state (asleep or awake) at any two time points 24 hours apart, averaged over 7-days of accelerometry (range 0-100, with 100 being perfectly regular). The SRI was related to the risk of mortality in time-to-event models.

FindingsThe mean sample age was 62 years (SD, 8), 56% were women, and the median SRI was 60 (SD, 10). There were 3010 deaths during a mean follow-up of 7.1 years. Following adjustments for demographic and clinical variables, we identified a non-linear relationship between the SRI and all-cause mortality hazard (p [global test of spline term] &lt; 0{middle dot}001). Hazard Ratios, relative to the median SRI, were 1{middle dot}53 (95% confidence interval [CI]: 1{middle dot}41, 1{middle dot}66) for participants with SRI at the 5th percentile (SRI = 41) and 0{middle dot}90 (95% CI: 0{middle dot}81, 1{middle dot}00) for those with SRI at the 95th percentile (SRI = 75), respectively. Findings for CVD mortality and cancer mortality followed a similar pattern.

ConclusionsIrregular sleep-wake patterns are associated with higher mortality risk.

FundingNational Health and Medical Research Council of Australia (GTN2009264; GTN1158384), National Institute on Aging (AG062531), Alzheimers Association (2018-AARG-591358), and the Banting Fellowship Program (#454104).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.23293833">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.23293833" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.23293833">
        <p class="paperTitle">One Health Genomic Surveillance of Avian and Human Influenza A Viruses Through Environmental Wastewater Monitoring</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.23293833" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.23293833" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, A. J.; Carson, S.; Reyne, M. I.; Marshall, A.; Moody, D.; Allen, D.; Allingham, P.; Levickas, A.; Fitzgerald, A.; Bell, S. H.; Lock, J.; Lock, J. D.; McSparron, C.; Nejad, B. F.; Courtney, D. G.; Einarsson, G. G.; McKenna, J. P.; Fairley, D. J.; Curran, T.; McKinley, J. M.; Gilpin, D. F.; Lemon, K.; McGrath, J. W.; Bamford, C. G.</p>
        <p class="info">Score: 12.7, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.23293833' target='https://doi.org/10.1101/2023.08.08.23293833'> 10.1101/2023.08.08.23293833</a></p>
        <p class="abstract">BackgroundInfluenza A viruses (IAV) are significant pathogens of humans and other animals. Although endemic in humans and birds, novel IAV strains can emerge, jump species, and cause epidemics, like the latest variant of H5N1. Wastewater-based epidemiology (WBE) has very recently been shown to detect human IAV but whether it can detect avian-origin IAV, and if whole genome sequencing (WGS) can be used to discriminate circulating strains of IAV in wastewater remains unknown.

MethodsUsing a pan-IAV RT-qPCR assay, six wastewater treatment works (WWTWs) across Northern Ireland (NI), were screened from August to December 2022. A WGS approach using Oxford Nanopore technology was employed to sequence positive samples. Phylogenetic analysis of sequences relative to currently circulating human and avian IAVs was performed.

FindingsWe detected a dynamic IAV signal in wastewater from September 2022 onwards across NI. &#34;Meta&#34; whole genome sequences were generated displaying homology to both human and avian IAV strains. The relative proportion of human versus avian-origin IAV reads differed across time and sample site. A diversity in subtypes and lineages was detected (e.g. H1N1, H3N2, and several avian). Avian segment 8 related to those found in recent H5N1 clade 2.3.4.4b was identified.

InterpretationWBE affords a means to monitor circulating human and avian IAV strains and provide crucial genetic information. As such WBE can provide rapid, cost-effective, year-round &#34;one-health&#34; IAV surveillance to help control epidemic and pandemic threats.

FundingThis study was funded by the Department of Health for Northern Ireland as part of the Northern Ireland Wastewater Surveillance Programme.

HighlightsO_LIDynamic IAV RT-qPCR signal in wastewater detected across NI.
C_LIO_LINanopore-based WGS reveals presence of both human and avian IAVs in wastewater.
C_LIO_LIAvian IAV sequence similarity to gull-associated H13/H16 and recent H5N1 isolates.
C_LIO_LICo-detection of distinct clades of human H1N1 and H3N2 subtypes.
C_LI

Author SummaryInfluenza A virus (IAV) is a major pathogen of humans and other animals and causes regular epidemics and devastating pandemics. Recently, a novel variant of highly-pathogenic H5N1 avian influenza has emerged spreading across the world killing millions of birds and infecting mammals, enhancing its pandemic potential. Strengthening global surveillance systems for human and animal IAV is thus a major priority. Wastewater-based epidemiology (WBE) has been applied to track SARS-CoV-2 and IAV in humans but whether this approach could work for avian IAV is not known. Here, we develop a &#34;one-health&#34; method to survey pan-IAV levels and genetically characterise the viruses. Through this we highlight co-detection of human and avian IAVs in wastewater, with homology to recent H5N1 isolates. Our work demonstrates the potential for WBE to help defend against not only human infections but emerging, zoonotic IAVs of pandemic potential.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=133 SRC=&#34;FIGDIR/small/23293833v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (26K):
org.highwire.dtl.DTLVardef@a7b3c4org.highwire.dtl.DTLVardef@b2c12org.highwire.dtl.DTLVardef@17a81faorg.highwire.dtl.DTLVardef@1eb5c70_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.23293589">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.23293589" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.23293589">
        <p class="paperTitle">Strategic use of SARS-CoV-2 wastewater concentration data could enhance, but not replace, high-resolution community prevalence survey programmes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.23293589" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.23293589" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mills, C.; Chadeau-Hyam, M.; Elliott, P. L.; Donnelly, C. A.</p>
        <p class="info">Score: 3.7, Published: 2023-08-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.23293589' target='https://doi.org/10.1101/2023.08.17.23293589'> 10.1101/2023.08.17.23293589</a></p>
        <p class="abstract">Wastewater-based epidemiology (WBE) has been proposed as a tool for public health authorities to monitor community transmission of SARS-CoV-2 and other agents. Here, we review the utility of WBE for estimating SARS-CoV-2 prevalence using wastewater data from the Environmental Monitoring for Health Protection (EMHP) programme and prevalence data from the REal-time Assessment of Community Transmission-1 (REACT-1) study in England. Our analysis shows a temporally evolving relationship between wastewater and prevalence which limits the utility of WBE for estimating SARS-CoV-2 prevalence in high spatial resolution without a concurrent prevalence survey. We further characterise WBE for SARS-CoV-2 prevalence as i) vaccination-coverage-dependent and ii) variant-specific. Our work provides a gesopatial framework to map wastewater concentrations to public health boundaries, enabling public health authorities to interpret the relationship between wastewater and prevalence. We demonstrate that WBE can improve the cost efficiency and accuracy of community prevalence surveys which on their own may have incomplete geographic coverage or small sample sizes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.23294160">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.23294160" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.23294160">
        <p class="paperTitle">Unraveling COVID-19: Descriptive Analytics in a Middle-Income Country, Paving the Path Forward</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.23294160" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.23294160" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bansilan, N. P.; Rabajante, J. F.</p>
        <p class="info">Score: 3.0, Published: 2023-08-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.23294160' target='https://doi.org/10.1101/2023.08.16.23294160'> 10.1101/2023.08.16.23294160</a></p>
        <p class="abstract">The outbreak of COVID-19 unleashed an unprecedented global pandemic, leaving a profound impact on lives and economies worldwide. Recognizing its severity, the World Health Organization swiftly declared it a public health emergency of international concern. Tragically, the Philippines reported the first death case outside China, leading to a surge in cases following the first instance of local transmission. In response to this crisis, collaborative efforts have been underway to control the disease and minimize its health and socio-economic impacts. The COVID-19 epidemic curve holds vital insights into the history of exposure, transmission, testing, tracing, social distancing measures, community lockdowns, quarantine, isolation, and treatment, offering a comprehensive perspective on the nations response. One approach to gaining crucial insights is through meticulous analysis of available datasets, empowering us to inform future strategies and responses effectively. This paper aims to provide descriptive data analytics of the COVID-19 pandemic in the Philippines, summarizing the countrys fight by visualizing epidemiological and mobility datasets, revisiting scientific papers and news articles, and creating a timeline of the key issues faced during the pandemic. By leveraging these multifaceted analyses, policymakers and health authorities can make informed decisions to enhance preparedness, expand inter-agency cooperation, and combat future public health crises effectively. This study seeks to serve as a valuable resource, guiding nations worldwide in comprehending and responding to the challenges posed by COVID-19 and beyond.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
