<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Simultaneous multi-site editing of individual genomes using retron arrays
Authors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.
Score: 29.4, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549397
Our understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Simultaneous multi-site editing of individual genomes using retron arrays
Authors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.
Score: 29.4, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549397
Our understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-26T10:37:30+00:00" />
<meta property="article:modified_time" content="2023-07-26T10:37:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Simultaneous multi-site editing of individual genomes using retron arrays
Authors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.
Score: 29.4, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549397
Our understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Simultaneous multi-site editing of individual genomes using retron arrays\nAuthors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.\nScore: 29.4, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549397\nOur understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites.",
  "keywords": [
    
  ],
  "articleBody": " Simultaneous multi-site editing of individual genomes using retron arrays\nAuthors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.\nScore: 29.4, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549397\nOur understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites. Unfortunately, building cells with multiple, non-adjacent precise mutations remains a laborious cycle of editing, isolating an edited cell, and editing again. Here, we describe a technology for precisely modifying multiple sites on a single genome simultaneously. This technology - termed a multitron - is built from a heavily modified retron, in which multiple donor-encoding msds are produced from a single transcript. The multitron architecture is compatible with both recombineering in prokaryotic cells and CRISPR editing in eukaryotic cells. We demonstrate applications for this approach in molecular recording, genetic element minimization, and metabolic engineering.\nEliminating Malaria Vectors with Precision Guided Sterile Males\nAuthors: Smidler, A. L.; Apte, R. A.; Pai, J. J.; Chow, M. L.; Chen, S.; Mondal, A.; Sanchez C., H. M.; Antoshechkin, I.; Marshall, J. M.; Akbari, O. S.\nScore: 13.3, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549947\nControlling the principal African malaria vector, the mosquito Anopheles gambiae, is considered essential to curtail malaria transmission. However existing vector control technologies rely on insecticides, which are becoming increasingly ineffective. Sterile insect technique (SIT) is a powerful suppression approach that has successfully eradicated a number of insect pests, yet the A. gambiae toolkit lacks the requisite technologies for its implementation. SIT relies on iterative mass-releases of non-biting, non-driving, sterile males which seek out and mate with monandrous wild females. Once mated, females are permanently sterilized due to mating-induced refractoriness, which results in population suppression of the subsequent generation. However, sterilization by traditional methods renders males unfit, making the creation of precise genetic sterilization methods imperative. Here we develop precision guided Sterile Insect Technique (pgSIT) in the mosquito A. gambiae for inducible, programmed male-sterilization and female-elimination for wide scale use in SIT campaigns. Using a binary CRISPR strategy, we cross separate engineered Cas9 and gRNA strains to disrupt male-fertility and female-essential genes, yielding \u003e99.5% male-sterility and \u003e99.9% female-lethality in hybrid progeny. We demonstrate that these genetically sterilized males have good longevity, are able to induce population suppression in cage trials, and are predicted to eliminate wild A. gambiae populations using mathematical models, making them ideal candidates for release. This work provides a valuable addition to the malaria genetic biocontrol toolkit, for the first time enabling scalable SIT-like confinable suppression in the species.\nFlow in fetoplacental microvessels in vitro enhances perfusion, barrier function, and matrix stability\nAuthors: Cherubini, M.; Erickson, S.; Padmanaban, P.; Haberkant, P.; Stein, F.; Sastre, V. B.; Haase, K.\nScore: 9.1, Published: 2023-07-21 DOI: 10.1101/2023.07.19.549736\nProper placental vascularization is vital for pregnancy outcomes, but assessing it with animal models and human explants has limitations. Here, we present a 3D in vitro model of human placenta terminal villi that includes fetal mesenchyme and vascular endothelium. By co-culturing HUVEC, placental fibroblasts, and pericytes in a macro-fluidic chip with a flow reservoir, we generate fully perfusable fetal microvessels. Pressure-driven flow is crucial for the growth and remodeling of these microvessels, resulting in early formation of interconnected placental vascular networks and maintained viability. Computational fluid dynamics simulations predict shear forces, which increase microtissue stiffness, decrease diffusivity and enhance barrier function as shear stress rises. Mass-spec analysis reveals the deposition of numerous extracellular proteins, with flow notably enhancing the expression of matrix stability regulators, proteins associated with actin dynamics, and cytoskeleton organization. Our model provides a powerful tool for deducing complex in vivo parameters, such as shear stress on developing vascularized placental tissue, and holds promise for unraveling gestational disorders related to the vasculature.\nTowards Modular Engineering of Cell Signalling: Topographically-Textured Microparticles Induce Osteogenesis via Activation of Canonical Hedgehog Signalling\nAuthors: Ghuloum, F. I.; Stevens, L. A.; Johnson, C. A.; Riobo-Del Galdo, N. A.; Amer, M. H.\nScore: 6.5, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549481\nPolymer microparticles possess great potential as functional building blocks for advanced bottom-up engineering of complex tissues. Tailoring the three-dimensional architectural features of culture substrates has been shown to induce osteogenesis in mesenchymal stem cells in vitro, but the molecular mechanisms underpinning this remain unclear. This study proposes a mechanism linking the activation of Hedgehog signalling to the osteoinductive effect of surface-engineered, topographically-textured polymeric microparticles. In this study, mesenchymal progenitor C3H10T1/2 cells were cultured on smooth and dimpled poly(D,L-lactide) microparticles to assess differences in viability, cellular morphology, proliferation, and expression of a range of Hedgehog signalling components and osteogenesis-related genes. Dimpled microparticles induced osteogenesis and activated the Hedgehog signalling pathway relative to smooth microparticles and 2D-cultured controls without the addition of exogenous biochemical factors. We observed upregulation of the osteogenesis markers Runt-related transcription factor2 (Runx2) and bone gamma-carboxyglutamate protein 2 (Bglap2), as well as the Hedgehog signalling components, glioma associated oncogene homolog 1 (Gli1), Patched1 (Ptch1), and Smoothened (Smo). Treatment with the Smo antagonist KAAD-cyclopamine confirmed the involvement of Smo in Gli1 target gene activation, with a significant reduction in the expression of Gli1, Runx2 and Bglap2 (p[\u0026le;]0.001) following KAAD-cyclopamine treatment. Overall, our study demonstrates the role of the topographical microenvironment in the modulation of Hedgehog signalling, highlighting the potential for tailoring substrate topographical design to offer cell-instructive 3D microenvironments. Topographically-textured microparticles allow the modulation of Hedgehog signalling in vitro without adding exogenous biochemical agonists, thereby eliminating potential confounding artefacts in high-throughput drug screening applications. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=134 HEIGHT=200 SRC=\"FIGDIR/small/549481v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (42K): org.highwire.dtl.DTLVardef@d9d7d3org.highwire.dtl.DTLVardef@1e87feeorg.highwire.dtl.DTLVardef@127199aorg.highwire.dtl.DTLVardef@20572_HPS_FORMAT_FIGEXP M_FIG C_FIG\nResponse Surface Methodology-Based High-Throughput BioLector Fermentation Screening of Rosetta-2 for Enhanced Production of Insulin Variant: Towards Improved Production Efficiency\nAuthors: Ayan, E.; Aytekin, A. O.; Kati, A.; Demirci, H.\nScore: 5.5, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549622\nBackgroundIn order to obtain high yields of recombinant insulin and advancing therapeutic avenues for diabetes patients, the development of innovative designer insulin analogs have critical importance. The modified insulin analog presents a cost-effective remedy by being produced as inclusion bodies (IBs) within Escherichia coli BL21 (DE3) Rosetta-2 strain. This approach to production not only offers reduced production time but also yields high recovery rates. The prime aim of this investigation was to optimize the composition of the cultivation media, thereby accomplishing higher cell density fermentation of the proinsulin, that is fused with a cleavable hexa histidine purification tag. ResultVarious factors, including carbon and nitrogen sources, salts, metal ions, and pH, were systematically investigated through experimental screening using the BioLector multiwell bright plate. Additionally, computational analysis employing the Plackett-Burman Design within the Design Expert software was utilized to assess their effectiveness in terms of insulin concentration as a surrogate measure of insulin yield. Among the tested variables, glucose, glycerol, MgSO4, and lower Luria-Bertani mix concentration have a significant influence on insulin production, as determined by the screening experiment. Subsequently, the Central Composite Design approach was operated to further evaluate and optimize the precise levels of these influential variables. This systematic methodology achieved an optimized cultivation media formulation, resulting in a remarkable enhancement of insulin production, with levels reaching up to 13 mg/ml when applied in BioLector fermentation. ConclusionThe formulated cultivation media exhibited suitability for promoting high cell density fermentation of the modified insulin, thereby facilitating the attainment of optimal yields for the proinsulin expression.\nSimple and rewireable biomolecular building blocks for DNA machine-learning algorithms\nAuthors: Lee, R. C.; Corsano, A.; Tseng, C. Y.; Chou, L. Y. T.\nScore: 5.3, Published: 2023-07-23 DOI: 10.1101/2023.07.20.549967\nDeep learning algorithms, such as neural networks, enable the processing of complex datasets with many related variables, and have applications in disease diagnosis, cell profiling, and drug discovery. Beyond its use in electronic computers, neural networks have been implemented using programmable biomolecules such as DNA. This confers unique advantages such as greater portability, ability to operate without electricity, and direct analysis of patterns of biomolecules in solution. Analogous to past bottlenecks in electronic computers, the computing power of DNA-based neural networks is limited by the ability to add more computing units, i.e. neurons. This limitation exists because current architectures require many nucleic acids to model a single neuron. Each addition of a neuron to the network compounds existing problems such as long assembly times, high background signal, and cross-talk between components. Here we test three strategies to solve this limitation and improve the scalability of DNA-based neural networks: (i) enzymatic synthesis to generate high-purity neurons, (ii) spatial patterning of neuron clusters based on their network position, and (iii) encoding neuron connectivity on a universal single-stranded DNA backbone. We show that neurons implemented via these strategies activate quickly, with high signal-to-background ratio, and respond to varying input concentrations and weights. Using this neuron design, we implemented basic neural network motifs such as cascading, fan-in, and fan-out circuits. Since this design is modular, easy to synthesize, and compatible with multiple neural network architectures, we envision it will help scale DNA-based neural networks in a variety of settings. This will enable portable computing power for applications such as portable diagnostics, compact data storage, and autonomous decision making for lab-on-a-chips.\nModeling Mechanical Activation of Macrophages During Pulmonary Fibrogenesis for Targeted Anti-Fibrosis Therapy\nAuthors: Xu, Y.; Ying, L.; Lang, J. K.; Hinz, B.; Zhao, R.\nScore: 3.4, Published: 2023-07-21 DOI: 10.1101/2023.07.19.549794\nPulmonary fibrosis, as seen in idiopathic pulmonary fibrosis (IPF) and COVID-induced pulmonary fibrosis, is an often-fatal lung disease. Increased numbers of immune cells such as macrophages were shown to accumulate in the fibrotic lung, but it is unclear how they contribute to the development of fibrosis. To recapitulate the macrophage mechanical activation in the fibrotic lung tissue microenvironment, we developed a fibrotic microtissue model with cocultured human macrophages and fibroblasts. We show that profibrotic macrophages seeded on topographically controlled stromal tissue constructs become mechanically activated. The resulting co-alignment of macrophages, collagen fibers and fibroblasts promote widespread fibrogenesis in micro-engineered lung tissues. Anti-fibrosis treatment using pirfenidone disrupts the polarization and mechanical activation of profibrotic macrophages, leading to fibrosis inhibition. Pirfenidone inhibits the mechanical activation of macrophages by suppressing integrin M{beta}2 (CD11b/CD18) and Rho-associated kinase 2, which is a previously unknown mechanism of action of the drug. Together, these results demonstrate a potential pulmonary fibrogenesis mechanism at the tissue level contributed by mechanically activated macrophages. We propose the coculture, force-sensing microtissue model as a powerful tool to study the complex immune-stromal cell interactions and the mechanism of action of anti-fibrosis drugs.\nOn automated model discovery and a universal material subroutine\nAuthors: Peirlinck, M.; Linka, K.; Hurtado, J. A.; Kuhl, E.\nScore: 3.2, Published: 2023-07-25 DOI: 10.1101/2023.07.19.549749\nConstitutive modeling is the cornerstone of computational and structural mechanics. In a finite element analysis, the constitutive model is encoded in the material subroutine, a function that maps local strains onto stresses. This function is called within every finite element, at each integration point, within every time step, at each Newton iteration. Today's finite element packages offer large libraries of material subroutines to choose from. However, the scientific criteria for model selection remain highly subjective and prone to user bias. Here we fully automate the process of model selection, autonomously discover the best model and parameters from experimental data, encode all possible discoverable models into a single material subroutine, and seamlessly integrate this universal material subroutine into a finite element analysis. We prototype this strategy for tension, compression, and shear data from human brain tissue and perform a hyperelastic model discovery from twelve possible terms. These terms feature the first and second invariants, raised to the first and second powers, embedded in the identity, exponential, and logarithmic functions, generating 4096 models in total. We demonstrate how to integrate these models into a single universal material subroutine that features the classical neo Hooke, Blatz Ko, Mooney Rivlin, Demiray, Gent, and Holzapfel models as special cases. Finite element simulations with our universal material subroutine show that it specializes well to these widely used models, generalizes well to newly discovered models, and agrees excellently with both experimental data and previous simulations. It also performs well within realistic finite element simulations and accurately predicts stress concentrations in the human brain for six different head impact scenarios. We anticipate that integrating automated model discovery into a universal material subroutine will generalize naturally to more complex anisotropic, compressible, and inelastic materials and to other nonlinear finite element platforms. Replacing dozens of individual material subroutines by a single universal material subroutine that is populated directly via automated model discovery-entirely without human interaction-makes finite element analyses more accessible, more robust, and less vulnerable to human error. This could forever change how we simulate materials and structures.\nIn vivo fate of systemically administered encapsulin protein nanocages and implications for their use in targeted drug delivery\nAuthors: Rennie, C.; Sives, C.; Boyton, I.; Diaz, D.; Gorrie, C. A.; Vittorio, O.; Collins-Praino, L.; Care, A.\nScore: 7.5, Published: 2023-07-18 DOI: 10.1101/2023.07.16.549228\nEncapsulins, self-assembling protein nanocages derived from prokaryotes, are promising nanoparticle-based drug delivery systems (NDDS). However, the in vivo behavior and fate of encapsulins are poorly understood. In this pre-clinical study, we probe the interactions between the model encapsulin from Thermotoga maritima (TmEnc) and key biological barriers encountered by NDDS. Here, a purified TmEnc formulation that exhibited colloidal stability, storability, and blood compatibility was intravenously injected into BALB/c mice. TmEnc had an excellent nanosafety profile, with no abnormal weight loss or gross pathology observed, and only temporary alterations in toxicity biomarkers detected. Notably, TmEnc demonstrated immunogenic properties, inducing the generation of nanocage-specific IgM and IgG antibodies, but without any prolonged pro-inflammatory effects. An absence of antibody cross-reactivity also suggested immune-orthogonality among encapsulins systems. Moreover, TmEnc formed a serum-derived protein corona on its surface which changed dynamically and appeared to play a role in immune recognition. TmEncs biodistribution profile further revealed its sequestration from the blood circulation by the liver and then biodegraded within Kupffer cells, thus indicating clearance via the mononuclear phagocyte system. Collectively, these findings provide critical insights into how encapsulins behave in vivo, thereby informing their future design, modification, and application in targeted drug delivery. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=187 SRC=\"FIGDIR/small/549228v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (33K): org.highwire.dtl.DTLVardef@b6e267org.highwire.dtl.DTLVardef@ca161borg.highwire.dtl.DTLVardef@1715e9corg.highwire.dtl.DTLVardef@1fa7f97_HPS_FORMAT_FIGEXP M_FIG C_FIG\nFLECS Technology for High-Throughput Screening of Hypercontractile Cellular Phenotypes in Fibrosis: A Function-First Approach to Anti-Fibrotic Drug Discovery\nAuthors: Wang, Y.; Cortes, E.; Huang, R.; Wan, J.; Hinz, B.; Damoiseaux, R.; Pushkarsky, I.\nScore: 2.1, Published: 2023-07-25 DOI: 10.1101/2023.07.21.549589\nThe pivotal role of myofibroblast contractility in the pathophysiology of fibrosis is widely recognized, yet HTS approaches are not available to quantify this critically important function in drug discovery. We develop, validate, and scale-up a HTS platform that quantifies contractile function of primary human lung myofibroblasts upon treatment with pro-fibrotic TGF-{beta}1. With the fully automated assay we screened a library of 40,000 novel small molecules in under 80 h of total assay run-time. We identified 42 hit compounds that inhibited the TGF-{beta}1-induced contractile phenotype of myofibroblasts, and enriched for 19 that specifically target myofibroblasts but not phenotypically related smooth muscle cells. Selected hits were validated in an ex vivo lung tissue models for their inhibitory effects on fibrotic gene upregulation by TGF-{beta}1. Our results demonstrate that integrating a functional contraction test into the drug screening process is key to identify compounds with targeted and diverse activity as potential anti-fibrotic agents.\n",
  "wordCount" : "2548",
  "inLanguage": "en",
  "datePublished": "2023-07-26T10:37:30Z",
  "dateModified": "2023-07-26T10:37:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on July 26, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549397">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549397" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549397">
        <p class="paperTitle">Simultaneous multi-site editing of individual genomes using retron arrays</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549397" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549397" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.</p>
        <p class="info">Score: 29.4, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549397' target='https://doi.org/10.1101/2023.07.17.549397'> 10.1101/2023.07.17.549397</a></p>
        <p class="abstract">Our understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites. Unfortunately, building cells with multiple, non-adjacent precise mutations remains a laborious cycle of editing, isolating an edited cell, and editing again. Here, we describe a technology for precisely modifying multiple sites on a single genome simultaneously. This technology - termed a multitron - is built from a heavily modified retron, in which multiple donor-encoding msds are produced from a single transcript. The multitron architecture is compatible with both recombineering in prokaryotic cells and CRISPR editing in eukaryotic cells. We demonstrate applications for this approach in molecular recording, genetic element minimization, and metabolic engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549947">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549947" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549947">
        <p class="paperTitle">Eliminating Malaria Vectors with Precision Guided Sterile Males</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549947" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549947" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smidler, A. L.; Apte, R. A.; Pai, J. J.; Chow, M. L.; Chen, S.; Mondal, A.; Sanchez C., H. M.; Antoshechkin, I.; Marshall, J. M.; Akbari, O. S.</p>
        <p class="info">Score: 13.3, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549947' target='https://doi.org/10.1101/2023.07.20.549947'> 10.1101/2023.07.20.549947</a></p>
        <p class="abstract">Controlling the principal African malaria vector, the mosquito Anopheles gambiae, is considered essential to curtail malaria transmission. However existing vector control technologies rely on insecticides, which are becoming increasingly ineffective. Sterile insect technique (SIT) is a powerful suppression approach that has successfully eradicated a number of insect pests, yet the A. gambiae toolkit lacks the requisite technologies for its implementation. SIT relies on iterative mass-releases of non-biting, non-driving, sterile males which seek out and mate with monandrous wild females. Once mated, females are permanently sterilized due to mating-induced refractoriness, which results in population suppression of the subsequent generation. However, sterilization by traditional methods renders males unfit, making the creation of precise genetic sterilization methods imperative. Here we develop precision guided Sterile Insect Technique (pgSIT) in the mosquito A. gambiae for inducible, programmed male-sterilization and female-elimination for wide scale use in SIT campaigns. Using a binary CRISPR strategy, we cross separate engineered Cas9 and gRNA strains to disrupt male-fertility and female-essential genes, yielding &gt;99.5% male-sterility and &gt;99.9% female-lethality in hybrid progeny. We demonstrate that these genetically sterilized males have good longevity, are able to induce population suppression in cage trials, and are predicted to eliminate wild A. gambiae populations using mathematical models, making them ideal candidates for release. This work provides a valuable addition to the malaria genetic biocontrol toolkit, for the first time enabling scalable SIT-like confinable suppression in the species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549736">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549736" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549736">
        <p class="paperTitle">Flow in fetoplacental microvessels in vitro enhances perfusion, barrier function, and matrix stability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549736" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549736" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cherubini, M.; Erickson, S.; Padmanaban, P.; Haberkant, P.; Stein, F.; Sastre, V. B.; Haase, K.</p>
        <p class="info">Score: 9.1, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549736' target='https://doi.org/10.1101/2023.07.19.549736'> 10.1101/2023.07.19.549736</a></p>
        <p class="abstract">Proper placental vascularization is vital for pregnancy outcomes, but assessing it with animal models and human explants has limitations. Here, we present a 3D in vitro model of human placenta terminal villi that includes fetal mesenchyme and vascular endothelium. By co-culturing HUVEC, placental fibroblasts, and pericytes in a macro-fluidic chip with a flow reservoir, we generate fully perfusable fetal microvessels. Pressure-driven flow is crucial for the growth and remodeling of these microvessels, resulting in early formation of interconnected placental vascular networks and maintained viability. Computational fluid dynamics simulations predict shear forces, which increase microtissue stiffness, decrease diffusivity and enhance barrier function as shear stress rises. Mass-spec analysis reveals the deposition of numerous extracellular proteins, with flow notably enhancing the expression of matrix stability regulators, proteins associated with actin dynamics, and cytoskeleton organization. Our model provides a powerful tool for deducing complex in vivo parameters, such as shear stress on developing vascularized placental tissue, and holds promise for unraveling gestational disorders related to the vasculature.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549481">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549481" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549481">
        <p class="paperTitle">Towards Modular Engineering of Cell Signalling: Topographically-Textured Microparticles Induce Osteogenesis via Activation of Canonical Hedgehog Signalling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549481" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549481" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ghuloum, F. I.; Stevens, L. A.; Johnson, C. A.; Riobo-Del Galdo, N. A.; Amer, M. H.</p>
        <p class="info">Score: 6.5, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549481' target='https://doi.org/10.1101/2023.07.19.549481'> 10.1101/2023.07.19.549481</a></p>
        <p class="abstract">Polymer microparticles possess great potential as functional building blocks for advanced bottom-up engineering of complex tissues. Tailoring the three-dimensional architectural features of culture substrates has been shown to induce osteogenesis in mesenchymal stem cells in vitro, but the molecular mechanisms underpinning this remain unclear. This study proposes a mechanism linking the activation of Hedgehog signalling to the osteoinductive effect of surface-engineered, topographically-textured polymeric microparticles. In this study, mesenchymal progenitor C3H10T1/2 cells were cultured on smooth and dimpled poly(D,L-lactide) microparticles to assess differences in viability, cellular morphology, proliferation, and expression of a range of Hedgehog signalling components and osteogenesis-related genes. Dimpled microparticles induced osteogenesis and activated the Hedgehog signalling pathway relative to smooth microparticles and 2D-cultured controls without the addition of exogenous biochemical factors. We observed upregulation of the osteogenesis markers Runt-related transcription factor2 (Runx2) and bone gamma-carboxyglutamate protein 2 (Bglap2), as well as the Hedgehog signalling components, glioma associated oncogene homolog 1 (Gli1), Patched1 (Ptch1), and Smoothened (Smo). Treatment with the Smo antagonist KAAD-cyclopamine confirmed the involvement of Smo in Gli1 target gene activation, with a significant reduction in the expression of Gli1, Runx2 and Bglap2 (p[&amp;le;]0.001) following KAAD-cyclopamine treatment. Overall, our study demonstrates the role of the topographical microenvironment in the modulation of Hedgehog signalling, highlighting the potential for tailoring substrate topographical design to offer cell-instructive 3D microenvironments. Topographically-textured microparticles allow the modulation of Hedgehog signalling in vitro without adding exogenous biochemical agonists, thereby eliminating potential confounding artefacts in high-throughput drug screening applications.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=134 HEIGHT=200 SRC=&#34;FIGDIR/small/549481v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (42K):
org.highwire.dtl.DTLVardef@d9d7d3org.highwire.dtl.DTLVardef@1e87feeorg.highwire.dtl.DTLVardef@127199aorg.highwire.dtl.DTLVardef@20572_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549622">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549622" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549622">
        <p class="paperTitle">Response Surface Methodology-Based High-Throughput BioLector Fermentation Screening of Rosetta-2 for Enhanced Production of Insulin Variant: Towards Improved Production Efficiency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549622" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549622" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ayan, E.; Aytekin, A. O.; Kati, A.; Demirci, H.</p>
        <p class="info">Score: 5.5, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549622' target='https://doi.org/10.1101/2023.07.19.549622'> 10.1101/2023.07.19.549622</a></p>
        <p class="abstract">BackgroundIn order to obtain high yields of recombinant insulin and advancing therapeutic avenues for diabetes patients, the development of innovative designer insulin analogs have critical importance. The modified insulin analog presents a cost-effective remedy by being produced as inclusion bodies (IBs) within Escherichia coli BL21 (DE3) Rosetta-2 strain. This approach to production not only offers reduced production time but also yields high recovery rates. The prime aim of this investigation was to optimize the composition of the cultivation media, thereby accomplishing higher cell density fermentation of the proinsulin, that is fused with a cleavable hexa histidine purification tag.

ResultVarious factors, including carbon and nitrogen sources, salts, metal ions, and pH, were systematically investigated through experimental screening using the BioLector multiwell bright plate. Additionally, computational analysis employing the Plackett-Burman Design within the Design Expert software was utilized to assess their effectiveness in terms of insulin concentration as a surrogate measure of insulin yield. Among the tested variables, glucose, glycerol, MgSO4, and lower Luria-Bertani mix concentration have a significant influence on insulin production, as determined by the screening experiment. Subsequently, the Central Composite Design approach was operated to further evaluate and optimize the precise levels of these influential variables. This systematic methodology achieved an optimized cultivation media formulation, resulting in a remarkable enhancement of insulin production, with levels reaching up to 13 mg/ml when applied in BioLector fermentation.

ConclusionThe formulated cultivation media exhibited suitability for promoting high cell density fermentation of the modified insulin, thereby facilitating the attainment of optimal yields for the proinsulin expression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549967">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549967" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549967">
        <p class="paperTitle">Simple and rewireable biomolecular building blocks for DNA machine-learning algorithms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549967" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549967" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, R. C.; Corsano, A.; Tseng, C. Y.; Chou, L. Y. T.</p>
        <p class="info">Score: 5.3, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549967' target='https://doi.org/10.1101/2023.07.20.549967'> 10.1101/2023.07.20.549967</a></p>
        <p class="abstract">Deep learning algorithms, such as neural networks, enable the processing of complex datasets with many related variables, and have applications in disease diagnosis, cell profiling, and drug discovery. Beyond its use in electronic computers, neural networks have been implemented using programmable biomolecules such as DNA. This confers unique advantages such as greater portability, ability to operate without electricity, and direct analysis of patterns of biomolecules in solution. Analogous to past bottlenecks in electronic computers, the computing power of DNA-based neural networks is limited by the ability to add more computing units, i.e. neurons. This limitation exists because current architectures require many nucleic acids to model a single neuron. Each addition of a neuron to the network compounds existing problems such as long assembly times, high background signal, and cross-talk between components. Here we test three strategies to solve this limitation and improve the scalability of DNA-based neural networks: (i) enzymatic synthesis to generate high-purity neurons, (ii) spatial patterning of neuron clusters based on their network position, and (iii) encoding neuron connectivity on a universal single-stranded DNA backbone. We show that neurons implemented via these strategies activate quickly, with high signal-to-background ratio, and respond to varying input concentrations and weights. Using this neuron design, we implemented basic neural network motifs such as cascading, fan-in, and fan-out circuits. Since this design is modular, easy to synthesize, and compatible with multiple neural network architectures, we envision it will help scale DNA-based neural networks in a variety of settings. This will enable portable computing power for applications such as portable diagnostics, compact data storage, and autonomous decision making for lab-on-a-chips.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549794">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549794" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549794">
        <p class="paperTitle">Modeling Mechanical Activation of Macrophages During Pulmonary Fibrogenesis for Targeted Anti-Fibrosis Therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549794" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549794" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, Y.; Ying, L.; Lang, J. K.; Hinz, B.; Zhao, R.</p>
        <p class="info">Score: 3.4, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549794' target='https://doi.org/10.1101/2023.07.19.549794'> 10.1101/2023.07.19.549794</a></p>
        <p class="abstract">Pulmonary fibrosis, as seen in idiopathic pulmonary fibrosis (IPF) and COVID-induced pulmonary fibrosis, is an often-fatal lung disease. Increased numbers of immune cells such as macrophages were shown to accumulate in the fibrotic lung, but it is unclear how they contribute to the development of fibrosis. To recapitulate the macrophage mechanical activation in the fibrotic lung tissue microenvironment, we developed a fibrotic microtissue model with cocultured human macrophages and fibroblasts. We show that profibrotic macrophages seeded on topographically controlled stromal tissue constructs become mechanically activated. The resulting co-alignment of macrophages, collagen fibers and fibroblasts promote widespread fibrogenesis in micro-engineered lung tissues. Anti-fibrosis treatment using pirfenidone disrupts the polarization and mechanical activation of profibrotic macrophages, leading to fibrosis inhibition. Pirfenidone inhibits the mechanical activation of macrophages by suppressing integrin M{beta}2 (CD11b/CD18) and Rho-associated kinase 2, which is a previously unknown mechanism of action of the drug. Together, these results demonstrate a potential pulmonary fibrogenesis mechanism at the tissue level contributed by mechanically activated macrophages. We propose the coculture, force-sensing microtissue model as a powerful tool to study the complex immune-stromal cell interactions and the mechanism of action of anti-fibrosis drugs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549749">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549749" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549749">
        <p class="paperTitle">On automated model discovery and a universal material subroutine</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549749" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549749" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peirlinck, M.; Linka, K.; Hurtado, J. A.; Kuhl, E.</p>
        <p class="info">Score: 3.2, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549749' target='https://doi.org/10.1101/2023.07.19.549749'> 10.1101/2023.07.19.549749</a></p>
        <p class="abstract">Constitutive modeling is the cornerstone of computational and structural mechanics. In a finite element analysis, the constitutive model is encoded in the material subroutine, a function that maps local strains onto stresses. This function is called within every finite element, at each integration point, within every time step, at each Newton iteration. Today&#39;s finite element packages offer large libraries of material subroutines to choose from. However, the scientific criteria for model selection remain highly subjective and prone to user bias. Here we fully automate the process of model selection, autonomously discover the best model and parameters from experimental data, encode all possible discoverable models into a single material subroutine, and seamlessly integrate this universal material subroutine into a finite element analysis. We prototype this strategy for tension, compression, and shear data from human brain tissue and perform a hyperelastic model discovery from twelve possible terms. These terms feature the first and second invariants, raised to the first and second powers, embedded in the identity, exponential, and logarithmic functions, generating 4096 models in total. We demonstrate how to integrate these models into a single universal material subroutine that features the classical neo Hooke, Blatz Ko, Mooney Rivlin, Demiray, Gent, and Holzapfel models as special cases. Finite element simulations with our universal material subroutine show that it specializes well to these widely used models, generalizes well to newly discovered models, and agrees excellently with both experimental data and previous simulations. It also performs well within realistic finite element simulations and accurately predicts stress concentrations in the human brain for six different head impact scenarios. We anticipate that integrating automated model discovery into a universal material subroutine will generalize naturally to more complex anisotropic, compressible, and inelastic materials and to other nonlinear finite element platforms. Replacing dozens of individual material subroutines by a single universal material subroutine that is populated directly via automated model discovery-entirely without human interaction-makes finite element analyses more accessible, more robust, and less vulnerable to human error. This could forever change how we simulate materials and structures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.549228">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.549228" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.549228">
        <p class="paperTitle">In vivo fate of systemically administered encapsulin protein nanocages and implications for their use in targeted drug delivery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.549228" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.549228" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rennie, C.; Sives, C.; Boyton, I.; Diaz, D.; Gorrie, C. A.; Vittorio, O.; Collins-Praino, L.; Care, A.</p>
        <p class="info">Score: 7.5, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.549228' target='https://doi.org/10.1101/2023.07.16.549228'> 10.1101/2023.07.16.549228</a></p>
        <p class="abstract">Encapsulins, self-assembling protein nanocages derived from prokaryotes, are promising nanoparticle-based drug delivery systems (NDDS). However, the in vivo behavior and fate of encapsulins are poorly understood. In this pre-clinical study, we probe the interactions between the model encapsulin from Thermotoga maritima (TmEnc) and key biological barriers encountered by NDDS. Here, a purified TmEnc formulation that exhibited colloidal stability, storability, and blood compatibility was intravenously injected into BALB/c mice. TmEnc had an excellent nanosafety profile, with no abnormal weight loss or gross pathology observed, and only temporary alterations in toxicity biomarkers detected. Notably, TmEnc demonstrated immunogenic properties, inducing the generation of nanocage-specific IgM and IgG antibodies, but without any prolonged pro-inflammatory effects. An absence of antibody cross-reactivity also suggested immune-orthogonality among encapsulins systems. Moreover, TmEnc formed a serum-derived protein corona on its surface which changed dynamically and appeared to play a role in immune recognition. TmEncs biodistribution profile further revealed its sequestration from the blood circulation by the liver and then biodegraded within Kupffer cells, thus indicating clearance via the mononuclear phagocyte system. Collectively, these findings provide critical insights into how encapsulins behave in vivo, thereby informing their future design, modification, and application in targeted drug delivery.

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=187 SRC=&#34;FIGDIR/small/549228v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (33K):
org.highwire.dtl.DTLVardef@b6e267org.highwire.dtl.DTLVardef@ca161borg.highwire.dtl.DTLVardef@1715e9corg.highwire.dtl.DTLVardef@1fa7f97_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.549589">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.549589" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.549589">
        <p class="paperTitle">FLECS Technology for High-Throughput Screening of Hypercontractile Cellular Phenotypes in Fibrosis: A Function-First Approach to Anti-Fibrotic Drug Discovery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.549589" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.549589" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Y.; Cortes, E.; Huang, R.; Wan, J.; Hinz, B.; Damoiseaux, R.; Pushkarsky, I.</p>
        <p class="info">Score: 2.1, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.549589' target='https://doi.org/10.1101/2023.07.21.549589'> 10.1101/2023.07.21.549589</a></p>
        <p class="abstract">The pivotal role of myofibroblast contractility in the pathophysiology of fibrosis is widely recognized, yet HTS approaches are not available to quantify this critically important function in drug discovery. We develop, validate, and scale-up a HTS platform that quantifies contractile function of primary human lung myofibroblasts upon treatment with pro-fibrotic TGF-{beta}1. With the fully automated assay we screened a library of 40,000 novel small molecules in under 80 h of total assay run-time. We identified 42 hit compounds that inhibited the TGF-{beta}1-induced contractile phenotype of myofibroblasts, and enriched for 19 that specifically target myofibroblasts but not phenotypically related smooth muscle cells. Selected hits were validated in an ex vivo lung tissue models for their inhibitory effects on fibrotic gene upregulation by TGF-{beta}1. Our results demonstrate that integrating a functional contraction test into the drug screening process is key to identify compounds with targeted and diverse activity as potential anti-fibrotic agents.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
