<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Far-red chemigenetic biosensors for multi-dimensional and super-resolved kinase activity imaging
Authors: Frei, M. S.; Sanchez, S. A.; Liu, L.; Schneider, F.; Wang, Z.; Hakozaki, H.; Li, Y.; Lyons, A. C.; Rohm, T. V.; Olefsky, J. M.; Shi, L.; Schöneberg, J.; Fraser, S. E.; Mehta, S.; Wang, Y.; Zhang, J.
Score: 35.1, Published: 2024-02-11 DOI: 10.1101/2024.02.10.579766
Fluorescent biosensors revolutionized biomedical science by enabling the direct measurement of signaling activities in living cells, yet the current technology is limited in resolution and dimensionality.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Far-red chemigenetic biosensors for multi-dimensional and super-resolved kinase activity imaging
Authors: Frei, M. S.; Sanchez, S. A.; Liu, L.; Schneider, F.; Wang, Z.; Hakozaki, H.; Li, Y.; Lyons, A. C.; Rohm, T. V.; Olefsky, J. M.; Shi, L.; Schöneberg, J.; Fraser, S. E.; Mehta, S.; Wang, Y.; Zhang, J.
Score: 35.1, Published: 2024-02-11 DOI: 10.1101/2024.02.10.579766
Fluorescent biosensors revolutionized biomedical science by enabling the direct measurement of signaling activities in living cells, yet the current technology is limited in resolution and dimensionality." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-18T10:38:00+00:00" />
<meta property="article:modified_time" content="2024-02-18T10:38:00+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Far-red chemigenetic biosensors for multi-dimensional and super-resolved kinase activity imaging
Authors: Frei, M. S.; Sanchez, S. A.; Liu, L.; Schneider, F.; Wang, Z.; Hakozaki, H.; Li, Y.; Lyons, A. C.; Rohm, T. V.; Olefsky, J. M.; Shi, L.; Schöneberg, J.; Fraser, S. E.; Mehta, S.; Wang, Y.; Zhang, J.
Score: 35.1, Published: 2024-02-11 DOI: 10.1101/2024.02.10.579766
Fluorescent biosensors revolutionized biomedical science by enabling the direct measurement of signaling activities in living cells, yet the current technology is limited in resolution and dimensionality."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Far-red chemigenetic biosensors for multi-dimensional and super-resolved kinase activity imaging\nAuthors: Frei, M. S.; Sanchez, S. A.; Liu, L.; Schneider, F.; Wang, Z.; Hakozaki, H.; Li, Y.; Lyons, A. C.; Rohm, T. V.; Olefsky, J. M.; Shi, L.; Schöneberg, J.; Fraser, S. E.; Mehta, S.; Wang, Y.; Zhang, J.\nScore: 35.1, Published: 2024-02-11 DOI: 10.1101/2024.02.10.579766\nFluorescent biosensors revolutionized biomedical science by enabling the direct measurement of signaling activities in living cells, yet the current technology is limited in resolution and dimensionality.",
  "keywords": [
    
  ],
  "articleBody": " Far-red chemigenetic biosensors for multi-dimensional and super-resolved kinase activity imaging\nAuthors: Frei, M. S.; Sanchez, S. A.; Liu, L.; Schneider, F.; Wang, Z.; Hakozaki, H.; Li, Y.; Lyons, A. C.; Rohm, T. V.; Olefsky, J. M.; Shi, L.; Schöneberg, J.; Fraser, S. E.; Mehta, S.; Wang, Y.; Zhang, J.\nScore: 35.1, Published: 2024-02-11 DOI: 10.1101/2024.02.10.579766\nFluorescent biosensors revolutionized biomedical science by enabling the direct measurement of signaling activities in living cells, yet the current technology is limited in resolution and dimensionality. Here, we introduce highly sensitive chemigenetic kinase activity biosensors that combine the genetically encodable self-labeling protein tag HaloTag7 with bright far-red-emitting synthetic fluorophores. This technology enables five-color biosensor multiplexing, 4D activity imaging, and functional super-resolution imaging via stimulated emission depletion (STED) microscopy.\nFeature Reuse and Scaling: Understanding Transfer Learning with Protein Language Models\nAuthors: Li, F.-Z.; Amini, A. P.; Yue, Y.; Yang, K. K.; Lu, A. X.\nScore: 90.7, Published: 2024-02-14 DOI: 10.1101/2024.02.05.578959\nLarge pretrained protein language models (PLMs) have improved protein property and structure prediction from sequences via transfer learning, in which weights and representations from PLMs are repurposed for downstream tasks. Although PLMs have shown great promise, currently there is little understanding of how the features learned by pretraining relate to and are useful for downstream tasks. We perform a systematic analysis of transfer learning using PLMs, conducting 370 experiments across a comprehensive suite of factors including different downstream tasks, architectures, model sizes, model depths, and pretraining time. We observe that while almost all downstream tasks do benefit from pretrained models compared to naive sequence representations, for the majority of tasks performance does not scale with pretraining, and instead relies on low-level features learned early in pretraining. Our results point to a mismatch between current PLM pretraining paradigms and most applications of these models, indicating a need for better pretraining methods.\nRepurposing CRISPR-Cas13 systems for robust mRNA trans-splicing\nAuthors: Fiflis, D. N.; Rey, N.; Lavanya, H. V.; Sewell, B.; Mitchell-Dick, A.; Milo, S.; Clements, K.; Benkert, A.; Rosales, A.; Fergione, S.; Asokan, A.\nScore: 15.7, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579165\nType VI CRISPR enzymes have been developed as programmable RNA-guided Cas proteins for eukaryotic RNA editing. Notably, Cas13 has been utilized for site-targeted single base edits, demethylation, RNA cleavage or knockdown and alternative splicing. However, the ability to edit large stretches of mRNA transcripts remains a significant challenge. Here, we demonstrate that CRISPR-Cas13 systems can be repurposed to assist trans-splicing of exogenous RNA fragments into an endogenous pre-mRNA transcript, a method termed CRISPR Assisted mRNA Fragment Trans-splicing (CRAFT). Using split reporter-based assays, we evaluate orthogonal Cas13 systems, optimize guide RNA length and screen for optimal trans-splicing site(s) across a range of intronic targets. We achieve markedly improved editing of large 5 and 3 segments in different endogenous mRNAs across various mammalian cell types compared to other spliceosome-mediated trans-splicing methods. CRAFT can serve as a versatile platform for attachment of protein tags, studying the impact of multiple mutations/single nucleotide polymorphisms, modification of untranslated regions (UTRs) or replacing large segments of mRNA transcripts.\nDeveloping and Investigating a Nanovibration Intervention for the Prevention/Reversal of Bone Loss Following Spinal Cord Injury\nAuthors: Williams, J. A.; Campsie, P.; Gibson, R.; Johnson-Love, O.; Werber, A. S.; Sprott, M.; Meechan, R.; Huesa, C.; Windmill, J. F. C.; Purcell, M.; Coupaud, S.; Dalby, M. J.; Childs, P. G.; Riddell, J. S.; Reid, S.\nScore: 6.3, Published: 2024-02-14 DOI: 10.1101/2024.02.12.578222\nOsteoporosis disrupts the fine-tuned balance between bone formation and resorption leading to reductions in bone quantity and quality, ultimately leading to increased fracture risk. Prevention and treatment of osteoporotic fractures is essential, for reductions in mortality, morbidity and the economic burden, particularly considering the ageing global population. Extreme bone loss that mimics time-accelerated osteoporosis develops in the paralysed limbs following complete spinal cord injury (SCI). In vitro nanoscale vibration (1 kHz, 30- or 90 nm amplitude) has been shown to drive differentiation of mesenchymal stem cells towards osteoblast-like phenotypes, enhancing osteogenesis, and inhibiting osteoclastogenesis, simultaneously. Here we develop and characterise a wearable device designed to deliver continuous nano-amplitude vibration to the hindlimb long bones of rats with complete SCI. We investigate whether a clinically feasible dose of nanovibration (4-hours/day, 5-days/week for 6 weeks) is effective at reversing the established SCI-induced osteoporosis. Laser interferometry and finite element analysis confirmed transmission of nanovibration into the bone, and micro-computed tomography and serum bone formation and resorption markers assessed effectiveness. The intervention did not reverse SCI-induced osteoporosis. However, serum analysis indicated an elevated concentration of the bone formation marker procollagen type 1 N-terminal propeptide (P1NP) in rats receiving 40 nm amplitude nanovibration, suggesting increased synthesis of type 1 collagen, the major organic component of bone. Therefore, enhanced doses of nanovibrational stimulus may yet prove beneficial in attenuating/reversing osteoporosis, particularly in less severe forms of osteoporosis.\nNovel approach for carbon-wise utilization of lignin-related compounds by synergistically employing anaerobic and aerobic bacteria\nAuthors: Merilainen, E.; Efimova, E.; Santala, V.; Santala, S.\nScore: 5.7, Published: 2024-02-16 DOI: 10.1101/2024.02.14.580265\nLignin is a highly abundant but strongly underutilized natural resource that could serve as a sustainable feedstock to produce chemicals by microbial cell factories. However, the production from lignin-related aromatics is hindered by limited substrate range and inefficient catabolism of the production hosts. Particularly, the aerobic demethylation reactions are energy-limited and cause growth inhibition and loss of CO2. Here, we present a novel approach for carbon-wise utilization of lignin-related aromatics by the integration of anaerobic and aerobic metabolisms. In practice, we employed an acetogenic bacterium Acetobacterium woodii for anaerobic O-demethylation of aromatic compounds, which distinctively differs from the aerobic demethylation; in the process, the carbon from the methoxyl groups is fixated together with CO2 to form acetate while the aromatic ring remains unchanged. These accessible end-metabolites were then utilized by an aerobic bacterium Acinetobacter baylyi ADP1. Finally, we demonstrated the production of muconic acid from guaiacol, an abundant but inaccessible substrate to most microbes, with a nearly equimolar yield with only a minor genetic engineering and without the need for additional organic carbon source. This study highlights the power of synergistic integration of distinctive metabolic features of bacteria, thus unlocking new opportunities for harnessing microbial cocultures in upgrading challenging feedstocks.\nThermally activated antibiotic production by probiotic bacteria for pathogen elimination\nAuthors: Dey, S.; Seyfert, C. E.; Fink-Straube, C.; Kany, A. M.; Mueller, R.; Sankaran, S.\nScore: 5.3, Published: 2024-02-15 DOI: 10.1101/2024.02.13.579303\nEngineered bacterial therapeutics is a rapidly emerging field in which bacteria are genetically engineered to produce and deliver therapeutic compounds at the body site of use. They offer the advantages of being able to produce biopharmaceutical drugs on the spot and control drug release time and dosage through genetic switches to treat a variety of diseases, including infections, inflammatory diseases, and cancer. However, a major challenge in the field is achieving drug production rates for effectively treating diseases. This is especially true when combining genetic switches with heterologous gene expression. This study presents an expression strategy to overcome this challenge as part of developing thermo-switchable production of a novel antibiotic, darobactin, in probiotic Escherichia coli Nissle 1917. While thermo-switchable promoters produced ineffective levels of darobactin, the T7 promoter enabled production of pathogen-inhibitory levels although it was highly leaky. Thus, parts from both switches were combined to create a thermo-amplifier circuit that shows no detectable leakiness below 37 oC and releases sufficient darobactin at 40oC to inhibit the growth of a critically prioritized Pseudomonas aeruginosa pathogen.\nBio-orthogonal Glycan Imaging of Culture Cells and Whole Animal C. elegans with Expansion Microscopy\nAuthors: Kuo, J. C.-H.; Colville, M. J.; Sorkin, M. R.; Kuo, J. L. K.; Huang, L.-T.; Thornlow, D. N.; Beacham, G. M.; Hollopeter, G.; DeLisa, M. P.; Alabi, C. A.; Paszek, M. J.\nScore: 8.5, Published: 2024-02-02 DOI: 10.1101/2024.02.01.578333\nComplex carbohydrates called glycans play crucial roles in the regulation of cell and tissue physiology, but how glycans map to nanoscale anatomical features must still be resolved. Here, we present the first nanoscale map of mucin-type O-glycans throughout the entirety of the Caenorhabditis elegans model organism. We construct a library of multifunctional linkers to probe and anchor metabolically labelled glycans in expansion microscopy (ExM), an imaging modality that overcomes the diffraction limit of conventional optical microscopes through the physical expansion of samples embedded in a polyelectrolyte gel matrix. A flexible strategy is demonstrated for the chemical synthesis of linkers with a broad inventory of bio-orthogonal functional groups, fluorophores, anchorage chemistries, and linker arms. Employing C. elegans as a test bed, we resolve metabolically labelled O-glycans on the gut microvilli and other nanoscale anatomical features using our ExM reagents and optimized protocols. We use transmission electron microscopy images of C. elegans nano-anatomy as ground truth data to validate the fidelity and isotropy of gel expansion. We construct whole organism maps of C. elegans O-glycosylation in the first larval stage and identify O-glycan \"hotspots\" in unexpected anatomical locations, including the body wall furrows. Beyond C. elegans, we provide validated ExM protocols for nanoscale imaging of metabolically labelled glycans on cultured mammalian cells. Together, our results suggest the broad applicability of the multifunctional reagents for imaging glycans and other metabolically labelled biomolecules at enhanced resolutions with ExM. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=95 SRC=\"FIGDIR/small/578333v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (43K): org.highwire.dtl.DTLVardef@68fb6aorg.highwire.dtl.DTLVardef@52166aorg.highwire.dtl.DTLVardef@13dbfaeorg.highwire.dtl.DTLVardef@f352ca_HPS_FORMAT_FIGEXP M_FIG C_FIG\nFar-red and sensitive sensor for monitoring real-time H2O2 dynamics with subcellular resolution and inmulti-parametric imaging applications.\nAuthors: Lee, J. D.; Nguyen, A.; Jin, Z. R.; Moghadasi, A.; Gibbs, C. E.; Wait, S. J.; Evitts, K. M.; Asencio, A.; Bremner, S. B.; Zuniga, S.; Chavan, V.; Williams, A.; Smith, N.; Regnier, M.; Young, J. E.; Mack, D. L.; Nance, E.; Boyle, P. M.; Berndt, A.\nScore: 5.0, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579232\nH2O2 is a key oxidant in mammalian biology and a pleiotropic signaling molecule at the physiological level, and its excessive accumulation in conjunction with decreased cellular reduction capacity is often found to be a common pathological marker. Here, we present a red fluorescent Genetically Encoded H2O2 Indicator (GEHI) allowing versatile optogenetic dissection of redox biology. Our new GEHI, oROS-HT, is a chemigenetic sensor utilizing a HaloTag and Janelia Fluor (JF) rhodamine dye as fluorescent reporters. We developed oROS-HT through a structure-guided approach aided by classic protein structures and recent protein structure prediction tools. Optimized with JF635, oROS-HT is a sensor with 635 nm excitation and 650 nm emission peaks, allowing it to retain its brightness while monitoring intracellular H2O2 dynamics. Furthermore, it enables multi-color imaging in combination with blue-green fluorescent sensors for orthogonal analytes and low auto-fluorescence interference in biological tissues. Other advantages of oROS-HT over alternative GEHIs are its fast kinetics, oxygen-independent maturation, low pH sensitivity, lack of photo-artifact, and lack of intracellular aggregation. Here, we demonstrated efficient subcellular targeting and how oROS-HT can map inter and intracellular H2O2 diffusion at subcellular resolution. Lastly, we used oROS-HT with the green fluorescent calcium indicator Fluo-4 to investigate the transient effect of the anti-inflammatory agent auranofin on cellular redox physiology and calcium levels via multi-parametric, dual-color imaging.\nStructure-guided engineering of a fast genetically encoded sensor for real-time H2O2 monitoring.\nAuthors: Lee, J. D.; Won, W.; Kimball, K.; Wang, Y.; Yeboah, F.; Evitts, K.; Neiswanger, C.; Schattauer, S.; Rappleye, M.; Bremner, S. B.; Chun, C.; Smith, N.; Mack, D. L.; Young, J. E.; Lee, C. J.; Chavkin, C.; Berndt, A.\nScore: 6.7, Published: 2024-02-04 DOI: 10.1101/2024.01.31.578117\nHydrogen Peroxide (H2O2) is a central oxidant in redox biology due to its pleiotropic role in physiology and pathology. However, real-time monitoring of H2O2 in living cells and tissues remains a challenge. We address this gap with the development of an optogenetic hydRogen perOxide Sensor (oROS), leveraging the bacterial peroxide binding domain OxyR. Previously engineered OxyR-based fluorescent peroxide sensors lack the necessary sensitivity or response speed for effective real-time monitoring. By structurally redesigning the fusion of Escherichia coli (E. coli) ecOxyR with a circularly permutated green fluorescent protein (cpGFP), we created a novel, green-fluorescent peroxide sensor oROS-G. oROS-G exhibits high sensitivity and fast on-and-off kinetics, ideal for monitoring intracellular H2O2 dynamics. We successfully tracked real-time transient and steady-state H2O2 levels in diverse biological systems, including human stem cell-derived neurons and cardiomyocytes, primary neurons and astrocytes, and mouse neurons and astrocytes in ex vivo brain slices. These applications demonstrate oROSs capabilities to monitor H2O2 as a secondary response to pharmacologically induced oxidative stress, G-protein coupled receptor (GPCR)-induced cell signaling, and when adapting to varying metabolic stress. We showcased the increased oxidative stress in astrocytes via A{beta}-putriscine-MAOB axis, highlighting the sensors relevance in validating neurodegenerative disease models. oROS is a versatile tool, offering a window into the dynamic landscape of H2O2 signaling. This advancement paves the way for a deeper understanding of redox physiology, with significant implications for diseases associated with oxidative stress, such as cancer, neurodegenerative disorders, and cardiovascular diseases.\nIntegrating Safety, Security, Sustainability and Social Responsibility Principles into the U.S Bioeconomy\nAuthors: Gronvall, G. K.; Attal-Juncqua, A.; Getz, J.; Morhard, R.\nScore: 8.0, Published: 2024-02-08 DOI: 10.1101/2024.02.05.578954\nBioindustrial manufacturing is undergoing rapid expansion and investment and is seen as integral to nations economic progress. Ensuring that bioindustrial manufacturing benefits society as the field expands is of critical, urgent importance. To better understand the industrys ethical trajectory and to shape policy, we explored the views of biotechnology leaders on 4 aspects of ethical and social responsibility: - Safety, Security, Social Responsibility, and Sustainability, what we have termed \"4S Principles\". We identified policy actions governments and other stakeholders may take to maximize societal benefits in industrial biotechnology.\n",
  "wordCount" : "2232",
  "inLanguage": "en",
  "datePublished": "2024-02-18T10:38:00Z",
  "dateModified": "2024-02-18T10:38:00Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 18, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.10.579766">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.10.579766" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.10.579766">
        <p class="paperTitle">Far-red chemigenetic biosensors for multi-dimensional and super-resolved kinase activity imaging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.10.579766" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.10.579766" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Frei, M. S.; Sanchez, S. A.; Liu, L.; Schneider, F.; Wang, Z.; Hakozaki, H.; Li, Y.; Lyons, A. C.; Rohm, T. V.; Olefsky, J. M.; Shi, L.; Schöneberg, J.; Fraser, S. E.; Mehta, S.; Wang, Y.; Zhang, J.</p>
        <p class="info">Score: 35.1, Published: 2024-02-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.10.579766' target='https://doi.org/10.1101/2024.02.10.579766'> 10.1101/2024.02.10.579766</a></p>
        <p class="abstract">Fluorescent biosensors revolutionized biomedical science by enabling the direct measurement of signaling activities in living cells, yet the current technology is limited in resolution and dimensionality. Here, we introduce highly sensitive chemigenetic kinase activity biosensors that combine the genetically encodable self-labeling protein tag HaloTag7 with bright far-red-emitting synthetic fluorophores. This technology enables five-color biosensor multiplexing, 4D activity imaging, and functional super-resolution imaging via stimulated emission depletion (STED) microscopy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578959">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578959" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578959">
        <p class="paperTitle">Feature Reuse and Scaling: Understanding Transfer Learning with Protein Language Models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578959" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578959" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, F.-Z.; Amini, A. P.; Yue, Y.; Yang, K. K.; Lu, A. X.</p>
        <p class="info">Score: 90.7, Published: 2024-02-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578959' target='https://doi.org/10.1101/2024.02.05.578959'> 10.1101/2024.02.05.578959</a></p>
        <p class="abstract">Large pretrained protein language models (PLMs) have improved protein property and structure prediction from sequences via transfer learning, in which weights and representations from PLMs are repurposed for downstream tasks. Although PLMs have shown great promise, currently there is little understanding of how the features learned by pretraining relate to and are useful for downstream tasks. We perform a systematic analysis of transfer learning using PLMs, conducting 370 experiments across a comprehensive suite of factors including different downstream tasks, architectures, model sizes, model depths, and pretraining time. We observe that while almost all downstream tasks do benefit from pretrained models compared to naive sequence representations, for the majority of tasks performance does not scale with pretraining, and instead relies on low-level features learned early in pretraining. Our results point to a mismatch between current PLM pretraining paradigms and most applications of these models, indicating a need for better pretraining methods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579165">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579165" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579165">
        <p class="paperTitle">Repurposing CRISPR-Cas13 systems for robust mRNA trans-splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579165" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579165" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fiflis, D. N.; Rey, N.; Lavanya, H. V.; Sewell, B.; Mitchell-Dick, A.; Milo, S.; Clements, K.; Benkert, A.; Rosales, A.; Fergione, S.; Asokan, A.</p>
        <p class="info">Score: 15.7, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579165' target='https://doi.org/10.1101/2024.02.06.579165'> 10.1101/2024.02.06.579165</a></p>
        <p class="abstract">Type VI CRISPR enzymes have been developed as programmable RNA-guided Cas proteins for eukaryotic RNA editing. Notably, Cas13 has been utilized for site-targeted single base edits, demethylation, RNA cleavage or knockdown and alternative splicing. However, the ability to edit large stretches of mRNA transcripts remains a significant challenge. Here, we demonstrate that CRISPR-Cas13 systems can be repurposed to assist trans-splicing of exogenous RNA fragments into an endogenous pre-mRNA transcript, a method termed CRISPR Assisted mRNA Fragment Trans-splicing (CRAFT). Using split reporter-based assays, we evaluate orthogonal Cas13 systems, optimize guide RNA length and screen for optimal trans-splicing site(s) across a range of intronic targets. We achieve markedly improved editing of large 5 and 3 segments in different endogenous mRNAs across various mammalian cell types compared to other spliceosome-mediated trans-splicing methods. CRAFT can serve as a versatile platform for attachment of protein tags, studying the impact of multiple mutations/single nucleotide polymorphisms, modification of untranslated regions (UTRs) or replacing large segments of mRNA transcripts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.12.578222">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.12.578222" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.12.578222">
        <p class="paperTitle">Developing and Investigating a Nanovibration Intervention for the Prevention/Reversal of Bone Loss Following Spinal Cord Injury</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.12.578222" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.12.578222" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Williams, J. A.; Campsie, P.; Gibson, R.; Johnson-Love, O.; Werber, A. S.; Sprott, M.; Meechan, R.; Huesa, C.; Windmill, J. F. C.; Purcell, M.; Coupaud, S.; Dalby, M. J.; Childs, P. G.; Riddell, J. S.; Reid, S.</p>
        <p class="info">Score: 6.3, Published: 2024-02-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.12.578222' target='https://doi.org/10.1101/2024.02.12.578222'> 10.1101/2024.02.12.578222</a></p>
        <p class="abstract">Osteoporosis disrupts the fine-tuned balance between bone formation and resorption leading to reductions in bone quantity and quality, ultimately leading to increased fracture risk. Prevention and treatment of osteoporotic fractures is essential, for reductions in mortality, morbidity and the economic burden, particularly considering the ageing global population. Extreme bone loss that mimics time-accelerated osteoporosis develops in the paralysed limbs following complete spinal cord injury (SCI). In vitro nanoscale vibration (1 kHz, 30- or 90 nm amplitude) has been shown to drive differentiation of mesenchymal stem cells towards osteoblast-like phenotypes, enhancing osteogenesis, and inhibiting osteoclastogenesis, simultaneously. Here we develop and characterise a wearable device designed to deliver continuous nano-amplitude vibration to the hindlimb long bones of rats with complete SCI. We investigate whether a clinically feasible dose of nanovibration (4-hours/day, 5-days/week for 6 weeks) is effective at reversing the established SCI-induced osteoporosis. Laser interferometry and finite element analysis confirmed transmission of nanovibration into the bone, and micro-computed tomography and serum bone formation and resorption markers assessed effectiveness. The intervention did not reverse SCI-induced osteoporosis. However, serum analysis indicated an elevated concentration of the bone formation marker procollagen type 1 N-terminal propeptide (P1NP) in rats receiving 40 nm amplitude nanovibration, suggesting increased synthesis of type 1 collagen, the major organic component of bone. Therefore, enhanced doses of nanovibrational stimulus may yet prove beneficial in attenuating/reversing osteoporosis, particularly in less severe forms of osteoporosis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.14.580265">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.14.580265" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.14.580265">
        <p class="paperTitle">Novel approach for carbon-wise utilization of lignin-related compounds by synergistically employing anaerobic and aerobic bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.14.580265" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.14.580265" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Merilainen, E.; Efimova, E.; Santala, V.; Santala, S.</p>
        <p class="info">Score: 5.7, Published: 2024-02-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.14.580265' target='https://doi.org/10.1101/2024.02.14.580265'> 10.1101/2024.02.14.580265</a></p>
        <p class="abstract">Lignin is a highly abundant but strongly underutilized natural resource that could serve as a sustainable feedstock to produce chemicals by microbial cell factories. However, the production from lignin-related aromatics is hindered by limited substrate range and inefficient catabolism of the production hosts. Particularly, the aerobic demethylation reactions are energy-limited and cause growth inhibition and loss of CO2. Here, we present a novel approach for carbon-wise utilization of lignin-related aromatics by the integration of anaerobic and aerobic metabolisms. In practice, we employed an acetogenic bacterium Acetobacterium woodii for anaerobic O-demethylation of aromatic compounds, which distinctively differs from the aerobic demethylation; in the process, the carbon from the methoxyl groups is fixated together with CO2 to form acetate while the aromatic ring remains unchanged. These accessible end-metabolites were then utilized by an aerobic bacterium Acinetobacter baylyi ADP1. Finally, we demonstrated the production of muconic acid from guaiacol, an abundant but inaccessible substrate to most microbes, with a nearly equimolar yield with only a minor genetic engineering and without the need for additional organic carbon source. This study highlights the power of synergistic integration of distinctive metabolic features of bacteria, thus unlocking new opportunities for harnessing microbial cocultures in upgrading challenging feedstocks.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.13.579303">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.13.579303" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.13.579303">
        <p class="paperTitle">Thermally activated antibiotic production by probiotic bacteria for pathogen elimination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.13.579303" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.13.579303" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dey, S.; Seyfert, C. E.; Fink-Straube, C.; Kany, A. M.; Mueller, R.; Sankaran, S.</p>
        <p class="info">Score: 5.3, Published: 2024-02-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.13.579303' target='https://doi.org/10.1101/2024.02.13.579303'> 10.1101/2024.02.13.579303</a></p>
        <p class="abstract">Engineered bacterial therapeutics is a rapidly emerging field in which bacteria are genetically engineered to produce and deliver therapeutic compounds at the body site of use. They offer the advantages of being able to produce biopharmaceutical drugs on the spot and control drug release time and dosage through genetic switches to treat a variety of diseases, including infections, inflammatory diseases, and cancer. However, a major challenge in the field is achieving drug production rates for effectively treating diseases. This is especially true when combining genetic switches with heterologous gene expression. This study presents an expression strategy to overcome this challenge as part of developing thermo-switchable production of a novel antibiotic, darobactin, in probiotic Escherichia coli Nissle 1917. While thermo-switchable promoters produced ineffective levels of darobactin, the T7 promoter enabled production of pathogen-inhibitory levels although it was highly leaky. Thus, parts from both switches were combined to create a thermo-amplifier circuit that shows no detectable leakiness below 37 oC and releases sufficient darobactin at 40oC to inhibit the growth of a critically prioritized Pseudomonas aeruginosa pathogen.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578333">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578333" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578333">
        <p class="paperTitle">Bio-orthogonal Glycan Imaging of Culture Cells and Whole Animal C. elegans with Expansion Microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578333" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578333" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kuo, J. C.-H.; Colville, M. J.; Sorkin, M. R.; Kuo, J. L. K.; Huang, L.-T.; Thornlow, D. N.; Beacham, G. M.; Hollopeter, G.; DeLisa, M. P.; Alabi, C. A.; Paszek, M. J.</p>
        <p class="info">Score: 8.5, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578333' target='https://doi.org/10.1101/2024.02.01.578333'> 10.1101/2024.02.01.578333</a></p>
        <p class="abstract">Complex carbohydrates called glycans play crucial roles in the regulation of cell and tissue physiology, but how glycans map to nanoscale anatomical features must still be resolved. Here, we present the first nanoscale map of mucin-type O-glycans throughout the entirety of the Caenorhabditis elegans model organism. We construct a library of multifunctional linkers to probe and anchor metabolically labelled glycans in expansion microscopy (ExM), an imaging modality that overcomes the diffraction limit of conventional optical microscopes through the physical expansion of samples embedded in a polyelectrolyte gel matrix. A flexible strategy is demonstrated for the chemical synthesis of linkers with a broad inventory of bio-orthogonal functional groups, fluorophores, anchorage chemistries, and linker arms. Employing C. elegans as a test bed, we resolve metabolically labelled O-glycans on the gut microvilli and other nanoscale anatomical features using our ExM reagents and optimized protocols. We use transmission electron microscopy images of C. elegans nano-anatomy as ground truth data to validate the fidelity and isotropy of gel expansion. We construct whole organism maps of C. elegans O-glycosylation in the first larval stage and identify O-glycan &#34;hotspots&#34; in unexpected anatomical locations, including the body wall furrows. Beyond C. elegans, we provide validated ExM protocols for nanoscale imaging of metabolically labelled glycans on cultured mammalian cells. Together, our results suggest the broad applicability of the multifunctional reagents for imaging glycans and other metabolically labelled biomolecules at enhanced resolutions with ExM.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=95 SRC=&#34;FIGDIR/small/578333v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (43K):
org.highwire.dtl.DTLVardef@68fb6aorg.highwire.dtl.DTLVardef@52166aorg.highwire.dtl.DTLVardef@13dbfaeorg.highwire.dtl.DTLVardef@f352ca_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579232">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579232" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579232">
        <p class="paperTitle">Far-red and sensitive sensor for monitoring real-time H2O2 dynamics with subcellular resolution and inmulti-parametric imaging applications.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579232" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579232" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, J. D.; Nguyen, A.; Jin, Z. R.; Moghadasi, A.; Gibbs, C. E.; Wait, S. J.; Evitts, K. M.; Asencio, A.; Bremner, S. B.; Zuniga, S.; Chavan, V.; Williams, A.; Smith, N.; Regnier, M.; Young, J. E.; Mack, D. L.; Nance, E.; Boyle, P. M.; Berndt, A.</p>
        <p class="info">Score: 5.0, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579232' target='https://doi.org/10.1101/2024.02.06.579232'> 10.1101/2024.02.06.579232</a></p>
        <p class="abstract">H2O2 is a key oxidant in mammalian biology and a pleiotropic signaling molecule at the physiological level, and its excessive accumulation in conjunction with decreased cellular reduction capacity is often found to be a common pathological marker. Here, we present a red fluorescent Genetically Encoded H2O2 Indicator (GEHI) allowing versatile optogenetic dissection of redox biology. Our new GEHI, oROS-HT, is a chemigenetic sensor utilizing a HaloTag and Janelia Fluor (JF) rhodamine dye as fluorescent reporters. We developed oROS-HT through a structure-guided approach aided by classic protein structures and recent protein structure prediction tools. Optimized with JF635, oROS-HT is a sensor with 635 nm excitation and 650 nm emission peaks, allowing it to retain its brightness while monitoring intracellular H2O2 dynamics. Furthermore, it enables multi-color imaging in combination with blue-green fluorescent sensors for orthogonal analytes and low auto-fluorescence interference in biological tissues. Other advantages of oROS-HT over alternative GEHIs are its fast kinetics, oxygen-independent maturation, low pH sensitivity, lack of photo-artifact, and lack of intracellular aggregation. Here, we demonstrated efficient subcellular targeting and how oROS-HT can map inter and intracellular H2O2 diffusion at subcellular resolution. Lastly, we used oROS-HT with the green fluorescent calcium indicator Fluo-4 to investigate the transient effect of the anti-inflammatory agent auranofin on cellular redox physiology and calcium levels via multi-parametric, dual-color imaging.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.578117">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.578117" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.578117">
        <p class="paperTitle">Structure-guided engineering of a fast genetically encoded sensor for real-time H2O2 monitoring.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.578117" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.578117" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, J. D.; Won, W.; Kimball, K.; Wang, Y.; Yeboah, F.; Evitts, K.; Neiswanger, C.; Schattauer, S.; Rappleye, M.; Bremner, S. B.; Chun, C.; Smith, N.; Mack, D. L.; Young, J. E.; Lee, C. J.; Chavkin, C.; Berndt, A.</p>
        <p class="info">Score: 6.7, Published: 2024-02-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.578117' target='https://doi.org/10.1101/2024.01.31.578117'> 10.1101/2024.01.31.578117</a></p>
        <p class="abstract">Hydrogen Peroxide (H2O2) is a central oxidant in redox biology due to its pleiotropic role in physiology and pathology. However, real-time monitoring of H2O2 in living cells and tissues remains a challenge. We address this gap with the development of an optogenetic hydRogen perOxide Sensor (oROS), leveraging the bacterial peroxide binding domain OxyR. Previously engineered OxyR-based fluorescent peroxide sensors lack the necessary sensitivity or response speed for effective real-time monitoring. By structurally redesigning the fusion of Escherichia coli (E. coli) ecOxyR with a circularly permutated green fluorescent protein (cpGFP), we created a novel, green-fluorescent peroxide sensor oROS-G. oROS-G exhibits high sensitivity and fast on-and-off kinetics, ideal for monitoring intracellular H2O2 dynamics. We successfully tracked real-time transient and steady-state H2O2 levels in diverse biological systems, including human stem cell-derived neurons and cardiomyocytes, primary neurons and astrocytes, and mouse neurons and astrocytes in ex vivo brain slices. These applications demonstrate oROSs capabilities to monitor H2O2 as a secondary response to pharmacologically induced oxidative stress, G-protein coupled receptor (GPCR)-induced cell signaling, and when adapting to varying metabolic stress. We showcased the increased oxidative stress in astrocytes via A{beta}-putriscine-MAOB axis, highlighting the sensors relevance in validating neurodegenerative disease models. oROS is a versatile tool, offering a window into the dynamic landscape of H2O2 signaling. This advancement paves the way for a deeper understanding of redox physiology, with significant implications for diseases associated with oxidative stress, such as cancer, neurodegenerative disorders, and cardiovascular diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578954">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578954" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578954">
        <p class="paperTitle">Integrating Safety, Security, Sustainability and Social Responsibility Principles into the U.S Bioeconomy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578954" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578954" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gronvall, G. K.; Attal-Juncqua, A.; Getz, J.; Morhard, R.</p>
        <p class="info">Score: 8.0, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578954' target='https://doi.org/10.1101/2024.02.05.578954'> 10.1101/2024.02.05.578954</a></p>
        <p class="abstract">Bioindustrial manufacturing is undergoing rapid expansion and investment and is seen as integral to nations economic progress. Ensuring that bioindustrial manufacturing benefits society as the field expands is of critical, urgent importance. To better understand the industrys ethical trajectory and to shape policy, we explored the views of biotechnology leaders on 4 aspects of ethical and social responsibility: - Safety, Security, Social Responsibility, and Sustainability, what we have termed &#34;4S Principles&#34;. We identified policy actions governments and other stakeholders may take to maximize societal benefits in industrial biotechnology.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
