<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Molecular de-extinction of antibiotics enabled by deep learning
Authors: Wan, F.; Torres, M. D. T.; Peng, J.; de la Fuente, C.
Score: 87.9, Published: 2023-10-01 DOI: 10.1101/2023.10.01.560353
Molecular de-extinction is an emerging field that aims to resurrect molecules to solve present-day problems such as antibiotic resistance. Here, we introduce a deep learning approach called Antibiotic Peptide de-Extinction (APEX) to mine the proteomes of all available extinct organisms (the &#34;extinctome&#34;) searching for encrypted peptide (EP) antibiotics.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Molecular de-extinction of antibiotics enabled by deep learning
Authors: Wan, F.; Torres, M. D. T.; Peng, J.; de la Fuente, C.
Score: 87.9, Published: 2023-10-01 DOI: 10.1101/2023.10.01.560353
Molecular de-extinction is an emerging field that aims to resurrect molecules to solve present-day problems such as antibiotic resistance. Here, we introduce a deep learning approach called Antibiotic Peptide de-Extinction (APEX) to mine the proteomes of all available extinct organisms (the &#34;extinctome&#34;) searching for encrypted peptide (EP) antibiotics." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-11T10:39:46+00:00" />
<meta property="article:modified_time" content="2023-10-11T10:39:46+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Molecular de-extinction of antibiotics enabled by deep learning
Authors: Wan, F.; Torres, M. D. T.; Peng, J.; de la Fuente, C.
Score: 87.9, Published: 2023-10-01 DOI: 10.1101/2023.10.01.560353
Molecular de-extinction is an emerging field that aims to resurrect molecules to solve present-day problems such as antibiotic resistance. Here, we introduce a deep learning approach called Antibiotic Peptide de-Extinction (APEX) to mine the proteomes of all available extinct organisms (the &#34;extinctome&#34;) searching for encrypted peptide (EP) antibiotics."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Molecular de-extinction of antibiotics enabled by deep learning\nAuthors: Wan, F.; Torres, M. D. T.; Peng, J.; de la Fuente, C.\nScore: 87.9, Published: 2023-10-01 DOI: 10.1101/2023.10.01.560353\nMolecular de-extinction is an emerging field that aims to resurrect molecules to solve present-day problems such as antibiotic resistance. Here, we introduce a deep learning approach called Antibiotic Peptide de-Extinction (APEX) to mine the proteomes of all available extinct organisms (the \u0026#34;extinctome\u0026#34;) searching for encrypted peptide (EP) antibiotics.",
  "keywords": [
    
  ],
  "articleBody": " Molecular de-extinction of antibiotics enabled by deep learning\nAuthors: Wan, F.; Torres, M. D. T.; Peng, J.; de la Fuente, C.\nScore: 87.9, Published: 2023-10-01 DOI: 10.1101/2023.10.01.560353\nMolecular de-extinction is an emerging field that aims to resurrect molecules to solve present-day problems such as antibiotic resistance. Here, we introduce a deep learning approach called Antibiotic Peptide de-Extinction (APEX) to mine the proteomes of all available extinct organisms (the \"extinctome\") searching for encrypted peptide (EP) antibiotics. APEX mined a total of 10,311,899 EPs and identified 37,176 sequences predicted to have broad-spectrum antimicrobial activity, 11,035 of which were not found in extant organisms. Chemical synthesis and experimental validation yielded archaic EPs (AEPs) with activity against dangerous bacterial pathogens. Most peptides killed bacteria by depolarizing their cytoplasmic membrane, contrary to known antimicrobial peptides, which target the outer membrane. Notably, lead peptides, including those derived from the woolly mammoth, ancient sea cow, giant sloth, and extinct giant elk, exhibited anti-infective activity in preclinical mouse models. We propose molecular de-extinction, accelerated by deep learning, as a framework for discovering therapeutic molecules.\nA Drug-Free Pathogen Capture and Neutralizing Nasal Spray to Prevent Emerging Respiratory Infections\nAuthors: Joseph, J.; Baby, H. M.; Quintero, J. R.; Kenney, D.; Mebratu, Y.; Bhatia, E.; Shah, P.; Swain, K.; Kaur, S.; Li, X. L.; Mwangi, J.; Snapper, O.; Nair, R.; Agus, E.; Ranganathan, S.; Kage, J.; Gao, J.; Luo, J. N.; Yu, A.; Douam, F.; Tesfaigzi, Y.; Karp, J.; Joshi, N.\nScore: 49.7, Published: 2023-10-03 DOI: 10.1101/2023.10.02.560602\nRespiratory infections pose a global health crisis. Vaccines are pathogen specific, and new vaccines are needed for mutants and emerging pathogens. Here, we report a \"drug free\" prophylactic platform - a \"Pathogen Capture and Neutralizing Spray\" (PCANS) that acts via a multi-pronged approach to prevent a broad spectrum of respiratory infections. PCANS forms a protective coating in the nasal cavity that enhances the capture of large respiratory droplets. The coating acts as a physical barrier against a broad spectrum of viruses and bacteria, and rapidly neutralizes them, reducing the pathogen load by \u003e99.99%. In mice, PCANS showed nasal retention for at least 8 h and was safe for daily administration. A single prophylactic dose of PCANS protected mice against supra-lethal dosages of a mouse-adapted H1N1 Influenza virus (PR8), reduced lung viral titer by \u003e99.99%, improved survival, and suppressed pathological manifestations. Together, our data suggest PCANS as a promising daily-use prophylactic approach against current and emerging respiratory infections.\nSpiral NeuroString: High-Density Soft Bioelectronic Fibers for Multimodal Sensing and Stimulation\nAuthors: Khatib, M.; Zhao, E. T.; Wei, S.; Abramson, A.; Bishop, E. B.; Chen, C.-H.; Thomas, A.-L.; Xu, C.; Park, J.; Lee, Y.; Hamnett, R.; Yu, W.; Root, S. E.; Yuan, L.; Chakhtoura, D.; Kim, K. K.; Zhong, D.; Nishio, Y.; Zhao, C.; Wu, C.; Jiang, Y.; Zhang, A.; Li, J.; Wang, W.; Salimi-Jazi, F.; Rafeeqi, T. A.; Hemed, N. M.; Tok, J. B.-H.; Chen, X.; Kaltschmidt, J. A.; Dunn, J.; Bao, Z.\nScore: 17.1, Published: 2023-10-03 DOI: 10.1101/2023.10.02.560482\nBioelectronic fibers hold promise for both research and clinical applications due to their compactness, ease of implantation, and ability to incorporate various functionalities such as sensing and stimulation. However, existing devices suffer from bulkiness, rigidity, limited functionality, and low density of active components. These limitations stem from the difficulty to incorporate many components on one-dimensional (1D) fiber devices due to the incompatibility of conventional microfabrication methods (e.g., photolithography) with curved, thin and long fiber structures. Herein, we introduce a fabrication approach, \"spiral transformation'', to convert two-dimensional (2D) films containing microfabricated devices into 1D soft fibers. This approach allows for the creation of high density multimodal soft bioelectronic fibers, termed Spiral NeuroString (S-NeuroString), while enabling precise control over the longitudinal, angular, and radial positioning and distribution of the functional components. We show the utility of S-NeuroString for motility mapping, serotonin sensing, and tissue stimulation within the dynamic and soft gastrointestinal (GI) system, as well as for single-unit recordings in the brain. The described bioelectronic fibers hold great promises for next-generation multifunctional implantable electronics.\nClick editing enables programmable genome writing using DNA polymerases and HUH endonucleases\nAuthors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.\nScore: 109.9, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557440\nGenome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e.g. CRISPR-Cas) to permit the installation of a range of edits including substitutions, insertions, and deletions. Click editors (CEs) leverage the \"click\"-like bioconjugation ability of HUH endonucleases (HUHes) with single stranded DNA substrates to covalently tether \"click DNA\" (clkDNA) templates encoding user-specifiable edits at targeted genomic loci. Through iterative optimization of the modular components of CEs (DNA polymerase and HUHe orthologs, architectural modifications, etc.) and their clkDNAs (template configurations, repair evading substitutions, etc.), we demonstrate the ability to install precise genome edits with minimal indels and no unwanted byproduct insertions. Since clkDNAs can be ordered as simple DNA oligonucleotides for cents per base, it is possible to screen many different clkDNA parameters rapidly and inexpensively to maximize edit efficiency. Together, click editing is a precise and highly versatile platform for modifying genomes with a simple workflow and broad utility across diverse biological applications.\nBioengineered human colon organoids with in vivo-like complexity and function\nAuthors: Mitrofanova, O.; Broguiere, N.; Nikolaev, M.; Lutolf, M. P.\nScore: 8.8, Published: 2023-10-06 DOI: 10.1101/2023.10.05.560991\nOrganoids and microphysiological systems, such as organs-on-a-chip, have emerged as powerful tools for modeling human gut physiology and disease in vitro. However, although physiologically relevant, these systems often lack the environmental milieu, spatial organization, cell-type diversity, and maturity necessary for mimicking adult human intestinal mucosa. To instead generate models closely resembling the in vivo cell-type composition and spatial compartmentalization, we herein integrated organoid and organ-on-a-chip technology to develop a primary human stem cell-derived organoid model, called mini-colons. The luminal access and flow in human mini-colons removes shed cells to greatly enhance tissue longevity and differentiation over physically inaccessible human intestinal organoids that accumulate trapped cellular debris and waste. By establishing a gradient of growth factors, we replicated and sustained in vivo-like cell fate patterning and concurrent differentiation to secretory cell types and colonocytes. These long-lived human mini-colons contain abundant mucus-producing Goblet cells that lubricate the colonic epithelial lining. The stem and proliferative progenitor cells are also realistically confined to the crypts, facilitating stable homeostatic tissue turnover and preserving tissue integrity for several weeks. Also signifying mini-colon in vivo-like maturation, single-cell RNA sequencing showed emerging mature colonocytes and absorptive BEST4+ colonocytes. This methodology could be expanded to generate microtissues derived from the small intestine and incorporate additional microenvironmental components, thus emulating the intricate complexity of the native gut in an in vitro setting. Our bioengineered human organoids provide a highly accurate, long-lived, functional platform to systematically study human gut physiology and pathology, and for the development of novel therapeutic strategies.\nRewiring native post-transcriptional carbon regulators to build multi-layered genetic circuits and optimize engineered microbes for bioproduction\nAuthors: Simmons, T. R.; Partipilo, G.; Stankes, A. C.; Srivastava, R.; Buchser, R.; Chiu, D.; Keitz, B. K.; Contreras, L. M.\nScore: 7.8, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560922\nAs the bioprocessing and synthetic biology spaces rapidly expand, post-transcriptional regulation is emerging as driver for maximizing signal response rate and for minimizing cost per protein within cells. Designing robust post-transcriptional control systems that have precise tunability and can achieve diverse regulatory outcomes are paramount to advance the field. Herein, we develop a new approach for engineered post-transcriptional control in bacteria by rewiring a native regulatory system in Escherichia coli, the Carbon Storage Regulatory (Csr) Network, to create tunable, complex genetic circuits. First, by co-opting native components of the Csr Network to regulate translation of a target mRNA transcript, we establish a Csr-regulated Buffer Gate. Next, by rationally engineering the interactions between our synthetic construct and the native components of the Csr Network, we expand our original design into a genetic toolbox of 12 Buffer Gates that achieve precise tunability across a 10-fold range of target gene expression. Subsequently, to further regulatory capabilities using this approach, we develop a Csr-regulated NOT Gate through engineering a Csr-activated sequence into our synthetic constructs. We then build upon the Csr Buffer and Not Gates to create post-transcriptional dual input Boolean OR, NOR, AND and NAND Logic Gates, as well as a genetic pulse circuit. As a third step, we demonstrate portability of our Csr-regulated Buffer Gates into three industrially relevant bacteria by recapitulating Buffer Gate activity simply by leveraging the conserved homologous Csr Network in each species. Lastly, as a demonstration of downstream application, we apply our system to a proof-of-concept synthetic mevalonate pathway. Using our engineered constructs, we optimize mevalonate production in E. coli resulting in a three-fold increase in production relative to a transcriptionally controlled mevalonate pathway. As a whole, we establish a novel approach to rewire post-transcriptional regulatory networks for complex bacterial computation that can be utilized for efficient bioproduction in engineered microbes.\nBuilding on-chip cytoskeletal circuits via branched microtubule networks\nAuthors: Zaferani, M.; Song, R.; Petry, S.; Stone, H. A.\nScore: 9.0, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560515\nControllable platforms to engineer robust cytoskeletal scaffolds have the potential to create novel on-chip nanotechnologies. Inspired by axons, we combined the branching microtubule (MT) nucleation pathway with microfabrication to develop \"cytoskeletal circuits\". This active matter platform allows control over the adaptive self-organization of uniformly polarized MT arrays via geometric features of microstructures designed within a microfluidic confinement. We build and characterize basic elements, including turns and divisions, as well as complex regulatory elements, such as biased division and MT diodes, to construct various MT architectures on a chip. Our platform could be used in diverse applications, ranging from efficient on-chip molecular transport to mechanical nano-actuators. Further, cytoskeletal circuits can serve as a tool to study how the physical environment contributes to MT architecture in living cells. SignificanceMicrotubules have essential functions within the cell, including providing a robust railroad for motor-driven cargo transport. The unique properties of microtubules have stimulated attempts to harness these characteristics for targeted delivery of molecular complexes, novel material design, and developing nanotechnologies with precision comparable to living organisms. However, Previous efforts mainly focused on microtubules with fixed length and layout and no controlled MT generation, setting a limit on designing MT architecture. In this study, we integrated nanofabrication with microtubule branching reactions borrowed directly from the cells toolkit to construct cytoskeletal circuits and generate microtubule architectures from scratch. That is, our system enables control over microtubule growth and autocatalytic nucleation on a microfluidic chip with micro/nanostructures patterned within.\nTrehalose-based coacervates for local bioactive protein delivery to the central nervous system\nAuthors: Hassan, L. F.; Sen, R.; O'Shea, T. M.\nScore: 5.0, Published: 2023-10-07 DOI: 10.1101/2023.10.05.561124\nTherapeutic outcomes of local biomolecule delivery to the central nervous system (CNS) using bulk biomaterials are limited by inadequate drug loading, neuropil disruption, and severe foreign body responses. Effective CNS delivery requires addressing these issues and developing well-tolerated, highly-loaded carriers that are dispersible within local neural parenchyma. Here, we synthesized biodegradable trehalose-based polyelectrolyte oligomers using facile A2:B3:AR thiol-ene Michael addition reactions that form complex coacervates upon mixing of oppositely charged oligomers. Coacervates permit high con-centration loading and controlled release of bioactive growth factors, enzymes, and anti-bodies, with modular formulation parameters that confer tunable release kinetics. Coacervates are cytocompatible and readily bind to cultured neural cells in vitro but remain extracellularly. Coacervates serve as effective vehicles for precisely delivering biomolecules, including bioactive neurotrophins, to the mouse striatum following intraparenchymal injection. These results support the use of trehalose-based coacervates as part of therapeutic protein delivery strategies for CNS disorders.\nOptimizing Cryo-EM Structural Analysis of Gi-coupling Receptors via Engineered Gt and Nb35 Application\nAuthors: Oshima, H. S.; Sano, F. K.; Akasaka, H.; Iwama, A.; Shihoya, W.; Nureki, O.\nScore: 4.6, Published: 2023-10-07 DOI: 10.1101/2023.10.07.561347\nCryo-EM single particle analysis has recently facilitated the high-resolution structural determination of numerous GPCR-G complexes. Diverse methodologies have been devised with this trend, and in the case of GPCR-Gi complexes, scFv16, an antibody that recognizes the intricate interface of the complex, has been mainly implemented to stabilize the complex. However, owing to their flexibility and heterogeneity, structural determinations of GPCR-Gi complexes remain both challenging and resource-intensive. By employing eGt, which exhibits binding affinity to modified nanobody Nb35, the cryo-EM structure of Rhodopsin-eGt complex was previously reported. Using this modified G protein, we determined the structure of the ETB-eGt complex bound to the modified Nb35. The determined structure of ETB receptor was the same as the previously reported ETB-Gi complex, and the resulting dataset demonstrated significantly improved anisotropy. This modified G protein will be utilized for the structural determination of other GPCR-Gi complexes.\nMultichannel bioelectronic sensing using engineered Escherichia coli\nAuthors: Zhang, X.; Ajo-Franklin, C.\nScore: 6.0, Published: 2023-09-30 DOI: 10.1101/2023.09.30.560307\nBy engineering extracellular electron transfer (EET) to be dependent on an analyte, researchers have developed whole cell bioelectronic sensors that sense hazards to human and environmental health1. However, these sensors regulate a single electron transfer pathway as an electrochemical channel, limiting the sensing information to a single analyte. To increase information content, we developed a multichannel bioelectronic sensor through which different chemicals regulate distinct extracellular electron transfer pathways within a single Escherichia coli cell. One channel utilizes the flavin synthesis pathway from Bacillus subtilis2 and the other a set of cytochromes constructing the Mtr pathway from Shewanella oneidensis3. We demonstrate an arsenite responsive promoter can control the Mtr pathway through activation of cytochrome CymA expression and a cadmium responsive promoter can control the flavin synthesis pathway4,5. The redox potential of flavin-mediated EET is different from that of CymA-mediated one6. This allowed for development of a redox-potential-dependent algorithm that distinguishes variable input signals of each analyte mediated by two EET pathways in vivo. This approach enables a 2-bit binary signal readout for real-time tracking throughout the entire sensing duration. Our multichannel bioelectronic sensor was able to accurately sense and distinguish different heavy metals in Brays Bayou water samples with a response time comparable to that in clean water. This multichannel bioelectronic sensors allow for simultaneous detection of different chemicals, significantly expanding information transmission and helping to safeguard human and environmental health.\n",
  "wordCount" : "2368",
  "inLanguage": "en",
  "datePublished": "2023-10-11T10:39:46Z",
  "dateModified": "2023-10-11T10:39:46Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on October 11, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.560353">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.560353" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.560353">
        <p class="paperTitle">Molecular de-extinction of antibiotics enabled by deep learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.560353" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.560353" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wan, F.; Torres, M. D. T.; Peng, J.; de la Fuente, C.</p>
        <p class="info">Score: 87.9, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.560353' target='https://doi.org/10.1101/2023.10.01.560353'> 10.1101/2023.10.01.560353</a></p>
        <p class="abstract">Molecular de-extinction is an emerging field that aims to resurrect molecules to solve present-day problems such as antibiotic resistance. Here, we introduce a deep learning approach called Antibiotic Peptide de-Extinction (APEX) to mine the proteomes of all available extinct organisms (the &#34;extinctome&#34;) searching for encrypted peptide (EP) antibiotics. APEX mined a total of 10,311,899 EPs and identified 37,176 sequences predicted to have broad-spectrum antimicrobial activity, 11,035 of which were not found in extant organisms. Chemical synthesis and experimental validation yielded archaic EPs (AEPs) with activity against dangerous bacterial pathogens. Most peptides killed bacteria by depolarizing their cytoplasmic membrane, contrary to known antimicrobial peptides, which target the outer membrane. Notably, lead peptides, including those derived from the woolly mammoth, ancient sea cow, giant sloth, and extinct giant elk, exhibited anti-infective activity in preclinical mouse models. We propose molecular de-extinction, accelerated by deep learning, as a framework for discovering therapeutic molecules.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560602">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560602" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560602">
        <p class="paperTitle">A Drug-Free Pathogen Capture and Neutralizing Nasal Spray to Prevent Emerging Respiratory Infections</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560602" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560602" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Joseph, J.; Baby, H. M.; Quintero, J. R.; Kenney, D.; Mebratu, Y.; Bhatia, E.; Shah, P.; Swain, K.; Kaur, S.; Li, X. L.; Mwangi, J.; Snapper, O.; Nair, R.; Agus, E.; Ranganathan, S.; Kage, J.; Gao, J.; Luo, J. N.; Yu, A.; Douam, F.; Tesfaigzi, Y.; Karp, J.; Joshi, N.</p>
        <p class="info">Score: 49.7, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560602' target='https://doi.org/10.1101/2023.10.02.560602'> 10.1101/2023.10.02.560602</a></p>
        <p class="abstract">Respiratory infections pose a global health crisis. Vaccines are pathogen specific, and new vaccines are needed for mutants and emerging pathogens. Here, we report a &#34;drug free&#34; prophylactic platform - a &#34;Pathogen Capture and Neutralizing Spray&#34; (PCANS) that acts via a multi-pronged approach to prevent a broad spectrum of respiratory infections. PCANS forms a protective coating in the nasal cavity that enhances the capture of large respiratory droplets. The coating acts as a physical barrier against a broad spectrum of viruses and bacteria, and rapidly neutralizes them, reducing the pathogen load by &gt;99.99%. In mice, PCANS showed nasal retention for at least 8 h and was safe for daily administration. A single prophylactic dose of PCANS protected mice against supra-lethal dosages of a mouse-adapted H1N1 Influenza virus (PR8), reduced lung viral titer by &gt;99.99%, improved survival, and suppressed pathological manifestations. Together, our data suggest PCANS as a promising daily-use prophylactic approach against current and emerging respiratory infections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560482">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560482" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560482">
        <p class="paperTitle">Spiral NeuroString: High-Density Soft Bioelectronic Fibers for Multimodal Sensing and Stimulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560482" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560482" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khatib, M.; Zhao, E. T.; Wei, S.; Abramson, A.; Bishop, E. B.; Chen, C.-H.; Thomas, A.-L.; Xu, C.; Park, J.; Lee, Y.; Hamnett, R.; Yu, W.; Root, S. E.; Yuan, L.; Chakhtoura, D.; Kim, K. K.; Zhong, D.; Nishio, Y.; Zhao, C.; Wu, C.; Jiang, Y.; Zhang, A.; Li, J.; Wang, W.; Salimi-Jazi, F.; Rafeeqi, T. A.; Hemed, N. M.; Tok, J. B.-H.; Chen, X.; Kaltschmidt, J. A.; Dunn, J.; Bao, Z.</p>
        <p class="info">Score: 17.1, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560482' target='https://doi.org/10.1101/2023.10.02.560482'> 10.1101/2023.10.02.560482</a></p>
        <p class="abstract">Bioelectronic fibers hold promise for both research and clinical applications due to their compactness, ease of implantation, and ability to incorporate various functionalities such as sensing and stimulation. However, existing devices suffer from bulkiness, rigidity, limited functionality, and low density of active components. These limitations stem from the difficulty to incorporate many components on one-dimensional (1D) fiber devices due to the incompatibility of conventional microfabrication methods (e.g., photolithography) with curved, thin and long fiber structures. Herein, we introduce a fabrication approach, &#34;spiral transformation&#39;&#39;, to convert two-dimensional (2D) films containing microfabricated devices into 1D soft fibers. This approach allows for the creation of high density multimodal soft bioelectronic fibers, termed Spiral NeuroString (S-NeuroString), while enabling precise control over the longitudinal, angular, and radial positioning and distribution of the functional components. We show the utility of S-NeuroString for motility mapping, serotonin sensing, and tissue stimulation within the dynamic and soft gastrointestinal (GI) system, as well as for single-unit recordings in the brain. The described bioelectronic fibers hold great promises for next-generation multifunctional implantable electronics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557440">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557440" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557440">
        <p class="paperTitle">Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557440" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557440" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.</p>
        <p class="info">Score: 109.9, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557440' target='https://doi.org/10.1101/2023.09.12.557440'> 10.1101/2023.09.12.557440</a></p>
        <p class="abstract">Genome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e.g. CRISPR-Cas) to permit the installation of a range of edits including substitutions, insertions, and deletions. Click editors (CEs) leverage the &#34;click&#34;-like bioconjugation ability of HUH endonucleases (HUHes) with single stranded DNA substrates to covalently tether &#34;click DNA&#34; (clkDNA) templates encoding user-specifiable edits at targeted genomic loci. Through iterative optimization of the modular components of CEs (DNA polymerase and HUHe orthologs, architectural modifications, etc.) and their clkDNAs (template configurations, repair evading substitutions, etc.), we demonstrate the ability to install precise genome edits with minimal indels and no unwanted byproduct insertions. Since clkDNAs can be ordered as simple DNA oligonucleotides for cents per base, it is possible to screen many different clkDNA parameters rapidly and inexpensively to maximize edit efficiency. Together, click editing is a precise and highly versatile platform for modifying genomes with a simple workflow and broad utility across diverse biological applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.560991">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.560991" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.560991">
        <p class="paperTitle">Bioengineered human colon organoids with in vivo-like complexity and function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.560991" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.560991" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mitrofanova, O.; Broguiere, N.; Nikolaev, M.; Lutolf, M. P.</p>
        <p class="info">Score: 8.8, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.560991' target='https://doi.org/10.1101/2023.10.05.560991'> 10.1101/2023.10.05.560991</a></p>
        <p class="abstract">Organoids and microphysiological systems, such as organs-on-a-chip, have emerged as powerful tools for modeling human gut physiology and disease in vitro. However, although physiologically relevant, these systems often lack the environmental milieu, spatial organization, cell-type diversity, and maturity necessary for mimicking adult human intestinal mucosa. To instead generate models closely resembling the in vivo cell-type composition and spatial compartmentalization, we herein integrated organoid and organ-on-a-chip technology to develop a primary human stem cell-derived organoid model, called mini-colons. The luminal access and flow in human mini-colons removes shed cells to greatly enhance tissue longevity and differentiation over physically inaccessible human intestinal organoids that accumulate trapped cellular debris and waste. By establishing a gradient of growth factors, we replicated and sustained in vivo-like cell fate patterning and concurrent differentiation to secretory cell types and colonocytes. These long-lived human mini-colons contain abundant mucus-producing Goblet cells that lubricate the colonic epithelial lining. The stem and proliferative progenitor cells are also realistically confined to the crypts, facilitating stable homeostatic tissue turnover and preserving tissue integrity for several weeks. Also signifying mini-colon in vivo-like maturation, single-cell RNA sequencing showed emerging mature colonocytes and absorptive BEST4&#43; colonocytes. This methodology could be expanded to generate microtissues derived from the small intestine and incorporate additional microenvironmental components, thus emulating the intricate complexity of the native gut in an in vitro setting. Our bioengineered human organoids provide a highly accurate, long-lived, functional platform to systematically study human gut physiology and pathology, and for the development of novel therapeutic strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560922">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560922" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560922">
        <p class="paperTitle">Rewiring native post-transcriptional carbon regulators to build multi-layered genetic circuits and optimize engineered microbes for bioproduction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560922" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560922" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Simmons, T. R.; Partipilo, G.; Stankes, A. C.; Srivastava, R.; Buchser, R.; Chiu, D.; Keitz, B. K.; Contreras, L. M.</p>
        <p class="info">Score: 7.8, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560922' target='https://doi.org/10.1101/2023.10.04.560922'> 10.1101/2023.10.04.560922</a></p>
        <p class="abstract">As the bioprocessing and synthetic biology spaces rapidly expand, post-transcriptional regulation is emerging as driver for maximizing signal response rate and for minimizing cost per protein within cells. Designing robust post-transcriptional control systems that have precise tunability and can achieve diverse regulatory outcomes are paramount to advance the field. Herein, we develop a new approach for engineered post-transcriptional control in bacteria by rewiring a native regulatory system in Escherichia coli, the Carbon Storage Regulatory (Csr) Network, to create tunable, complex genetic circuits. First, by co-opting native components of the Csr Network to regulate translation of a target mRNA transcript, we establish a Csr-regulated Buffer Gate. Next, by rationally engineering the interactions between our synthetic construct and the native components of the Csr Network, we expand our original design into a genetic toolbox of 12 Buffer Gates that achieve precise tunability across a 10-fold range of target gene expression. Subsequently, to further regulatory capabilities using this approach, we develop a Csr-regulated NOT Gate through engineering a Csr-activated sequence into our synthetic constructs. We then build upon the Csr Buffer and Not Gates to create post-transcriptional dual input Boolean OR, NOR, AND and NAND Logic Gates, as well as a genetic pulse circuit. As a third step, we demonstrate portability of our Csr-regulated Buffer Gates into three industrially relevant bacteria by recapitulating Buffer Gate activity simply by leveraging the conserved homologous Csr Network in each species. Lastly, as a demonstration of downstream application, we apply our system to a proof-of-concept synthetic mevalonate pathway. Using our engineered constructs, we optimize mevalonate production in E. coli resulting in a three-fold increase in production relative to a transcriptionally controlled mevalonate pathway. As a whole, we establish a novel approach to rewire post-transcriptional regulatory networks for complex bacterial computation that can be utilized for efficient bioproduction in engineered microbes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560515">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560515" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560515">
        <p class="paperTitle">Building on-chip cytoskeletal circuits via branched microtubule networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560515" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560515" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zaferani, M.; Song, R.; Petry, S.; Stone, H. A.</p>
        <p class="info">Score: 9.0, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560515' target='https://doi.org/10.1101/2023.10.02.560515'> 10.1101/2023.10.02.560515</a></p>
        <p class="abstract">Controllable platforms to engineer robust cytoskeletal scaffolds have the potential to create novel on-chip nanotechnologies. Inspired by axons, we combined the branching microtubule (MT) nucleation pathway with microfabrication to develop &#34;cytoskeletal circuits&#34;. This active matter platform allows control over the adaptive self-organization of uniformly polarized MT arrays via geometric features of microstructures designed within a microfluidic confinement. We build and characterize basic elements, including turns and divisions, as well as complex regulatory elements, such as biased division and MT diodes, to construct various MT architectures on a chip. Our platform could be used in diverse applications, ranging from efficient on-chip molecular transport to mechanical nano-actuators. Further, cytoskeletal circuits can serve as a tool to study how the physical environment contributes to MT architecture in living cells.

SignificanceMicrotubules have essential functions within the cell, including providing a robust railroad for motor-driven cargo transport. The unique properties of microtubules have stimulated attempts to harness these characteristics for targeted delivery of molecular complexes, novel material design, and developing nanotechnologies with precision comparable to living organisms. However, Previous efforts mainly focused on microtubules with fixed length and layout and no controlled MT generation, setting a limit on designing MT architecture. In this study, we integrated nanofabrication with microtubule branching reactions borrowed directly from the cells toolkit to construct cytoskeletal circuits and generate microtubule architectures from scratch. That is, our system enables control over microtubule growth and autocatalytic nucleation on a microfluidic chip with micro/nanostructures patterned within.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561124">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561124" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561124">
        <p class="paperTitle">Trehalose-based coacervates for local bioactive protein delivery to the central nervous system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561124" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561124" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hassan, L. F.; Sen, R.; O&#39;Shea, T. M.</p>
        <p class="info">Score: 5.0, Published: 2023-10-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561124' target='https://doi.org/10.1101/2023.10.05.561124'> 10.1101/2023.10.05.561124</a></p>
        <p class="abstract">Therapeutic outcomes of local biomolecule delivery to the central nervous system (CNS) using bulk biomaterials are limited by inadequate drug loading, neuropil disruption, and severe foreign body responses. Effective CNS delivery requires addressing these issues and developing well-tolerated, highly-loaded carriers that are dispersible within local neural parenchyma. Here, we synthesized biodegradable trehalose-based polyelectrolyte oligomers using facile A2:B3:AR thiol-ene Michael addition reactions that form complex coacervates upon mixing of oppositely charged oligomers. Coacervates permit high con-centration loading and controlled release of bioactive growth factors, enzymes, and anti-bodies, with modular formulation parameters that confer tunable release kinetics. Coacervates are cytocompatible and readily bind to cultured neural cells in vitro but remain extracellularly. Coacervates serve as effective vehicles for precisely delivering biomolecules, including bioactive neurotrophins, to the mouse striatum following intraparenchymal injection. These results support the use of trehalose-based coacervates as part of therapeutic protein delivery strategies for CNS disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.07.561347">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.07.561347" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.07.561347">
        <p class="paperTitle">Optimizing Cryo-EM Structural Analysis of Gi-coupling Receptors via Engineered Gt and Nb35 Application</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.07.561347" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.07.561347" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oshima, H. S.; Sano, F. K.; Akasaka, H.; Iwama, A.; Shihoya, W.; Nureki, O.</p>
        <p class="info">Score: 4.6, Published: 2023-10-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.07.561347' target='https://doi.org/10.1101/2023.10.07.561347'> 10.1101/2023.10.07.561347</a></p>
        <p class="abstract">Cryo-EM single particle analysis has recently facilitated the high-resolution structural determination of numerous GPCR-G complexes. Diverse methodologies have been devised with this trend, and in the case of GPCR-Gi complexes, scFv16, an antibody that recognizes the intricate interface of the complex, has been mainly implemented to stabilize the complex. However, owing to their flexibility and heterogeneity, structural determinations of GPCR-Gi complexes remain both challenging and resource-intensive. By employing eGt, which exhibits binding affinity to modified nanobody Nb35, the cryo-EM structure of Rhodopsin-eGt complex was previously reported. Using this modified G protein, we determined the structure of the ETB-eGt complex bound to the modified Nb35. The determined structure of ETB receptor was the same as the previously reported ETB-Gi complex, and the resulting dataset demonstrated significantly improved anisotropy. This modified G protein will be utilized for the structural determination of other GPCR-Gi complexes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.30.560307">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.30.560307" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.30.560307">
        <p class="paperTitle">Multichannel bioelectronic sensing using engineered Escherichia coli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.30.560307" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.30.560307" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, X.; Ajo-Franklin, C.</p>
        <p class="info">Score: 6.0, Published: 2023-09-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.30.560307' target='https://doi.org/10.1101/2023.09.30.560307'> 10.1101/2023.09.30.560307</a></p>
        <p class="abstract">By engineering extracellular electron transfer (EET) to be dependent on an analyte, researchers have developed whole cell bioelectronic sensors that sense hazards to human and environmental health1. However, these sensors regulate a single electron transfer pathway as an electrochemical channel, limiting the sensing information to a single analyte. To increase information content, we developed a multichannel bioelectronic sensor through which different chemicals regulate distinct extracellular electron transfer pathways within a single Escherichia coli cell. One channel utilizes the flavin synthesis pathway from Bacillus subtilis2 and the other a set of cytochromes constructing the Mtr pathway from Shewanella oneidensis3. We demonstrate an arsenite responsive promoter can control the Mtr pathway through activation of cytochrome CymA expression and a cadmium responsive promoter can control the flavin synthesis pathway4,5. The redox potential of flavin-mediated EET is different from that of CymA-mediated one6. This allowed for development of a redox-potential-dependent algorithm that distinguishes variable input signals of each analyte mediated by two EET pathways in vivo. This approach enables a 2-bit binary signal readout for real-time tracking throughout the entire sensing duration. Our multichannel bioelectronic sensor was able to accurately sense and distinguish different heavy metals in Brays Bayou water samples with a response time comparable to that in clean water. This multichannel bioelectronic sensors allow for simultaneous detection of different chemicals, significantly expanding information transmission and helping to safeguard human and environmental health.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
