<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Single-cell precision nanotechnology in vivo
Authors: MOLBAY, M.; Kick, B.; Zhao, S.; Todorov, M. I.; Ohn, T.-L.; Roth, S.; Simats, A.; Singh, V.; Khalin, I.; Pan, C.; Singh Bhatia, H.; Hellal, F.; Zeidler, R.; Liesz, A.; Plesnila, N.; Dietz, H.; Erturk, A.
Score: 124.7, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550304
Targeting nanoparticle therapeutics with cellular accuracy in whole organisms could open breakthrough opportunities in precision medicine. However, evaluating and fine-tuning the biodistribution of such systems in the whole organism at the cellular level remains a major obstacle.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Single-cell precision nanotechnology in vivo
Authors: MOLBAY, M.; Kick, B.; Zhao, S.; Todorov, M. I.; Ohn, T.-L.; Roth, S.; Simats, A.; Singh, V.; Khalin, I.; Pan, C.; Singh Bhatia, H.; Hellal, F.; Zeidler, R.; Liesz, A.; Plesnila, N.; Dietz, H.; Erturk, A.
Score: 124.7, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550304
Targeting nanoparticle therapeutics with cellular accuracy in whole organisms could open breakthrough opportunities in precision medicine. However, evaluating and fine-tuning the biodistribution of such systems in the whole organism at the cellular level remains a major obstacle." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-02T10:38:09+00:00" />
<meta property="article:modified_time" content="2023-08-02T10:38:09+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Single-cell precision nanotechnology in vivo
Authors: MOLBAY, M.; Kick, B.; Zhao, S.; Todorov, M. I.; Ohn, T.-L.; Roth, S.; Simats, A.; Singh, V.; Khalin, I.; Pan, C.; Singh Bhatia, H.; Hellal, F.; Zeidler, R.; Liesz, A.; Plesnila, N.; Dietz, H.; Erturk, A.
Score: 124.7, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550304
Targeting nanoparticle therapeutics with cellular accuracy in whole organisms could open breakthrough opportunities in precision medicine. However, evaluating and fine-tuning the biodistribution of such systems in the whole organism at the cellular level remains a major obstacle."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Single-cell precision nanotechnology in vivo\nAuthors: MOLBAY, M.; Kick, B.; Zhao, S.; Todorov, M. I.; Ohn, T.-L.; Roth, S.; Simats, A.; Singh, V.; Khalin, I.; Pan, C.; Singh Bhatia, H.; Hellal, F.; Zeidler, R.; Liesz, A.; Plesnila, N.; Dietz, H.; Erturk, A.\nScore: 124.7, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550304\nTargeting nanoparticle therapeutics with cellular accuracy in whole organisms could open breakthrough opportunities in precision medicine. However, evaluating and fine-tuning the biodistribution of such systems in the whole organism at the cellular level remains a major obstacle.",
  "keywords": [
    
  ],
  "articleBody": " Single-cell precision nanotechnology in vivo\nAuthors: MOLBAY, M.; Kick, B.; Zhao, S.; Todorov, M. I.; Ohn, T.-L.; Roth, S.; Simats, A.; Singh, V.; Khalin, I.; Pan, C.; Singh Bhatia, H.; Hellal, F.; Zeidler, R.; Liesz, A.; Plesnila, N.; Dietz, H.; Erturk, A.\nScore: 124.7, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550304\nTargeting nanoparticle therapeutics with cellular accuracy in whole organisms could open breakthrough opportunities in precision medicine. However, evaluating and fine-tuning the biodistribution of such systems in the whole organism at the cellular level remains a major obstacle. Here, we constructed targetable DNA origami, and analyzed biodistribution in transparent mice, in addition to studying tolerability, clearance kinetics, and immune response parameters. Untargeted DNA origami primarily accumulated in the spleen and the liver, while an immune cell-targeting variant successfully attached to immune cells throughout the body. A cancer cell-targeting mimetic co-localized on solid-tumor metastasis in the liver and the lung. These findings indicate that DNA origami can be directed in vivo, providing an important proof-of-concept and highlights the potential of high-resolution tissue-clearing imaging technologies in their development. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=121 SRC=\"FIGDIR/small/550304v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (44K): org.highwire.dtl.DTLVardef@1a2a3c8org.highwire.dtl.DTLVardef@aec749org.highwire.dtl.DTLVardef@9dc36aorg.highwire.dtl.DTLVardef@18896ba_HPS_FORMAT_FIGEXP M_FIG C_FIG HighlightsO_LIThis study demonstrates the potential of DNA origami-based drug delivery systems as versatile tool for for targeted delivery, which could be used to treat a range of diseases with applications. C_LIO_LIThe immune compatibility, half-life, targeting efficiency, and the biodistribution evaluation of DNA origami indicate its potential for systemic drug delivery. C_LIO_LIOur approach enables the assessment of biodistribution of nanoparticles in the intact body with a sensitivity to the single-cell level, highlighting the high-resolution tissue clearing technoloies in revealing DNA origamis feasibility for drug targeting. C_LI\nEfficient encoding of large antigenic spaces by epitope prioritization with Dolphyn\nAuthors: Liebhoff, A.-M.; Venkataraman, T.; Morgenlander, W. R.; Na, M.; Kula, T.; Waugh, K.; Morrison, C.; Rewers, M.; Longman, R.; Round, J.; Elledge, S.; Ruczinski, I.; Langmead, B.; Larman, H. B.\nScore: 4.9, Published: 2023-07-31 DOI: 10.1101/2023.07.30.551179\nWe investigated a relatively underexplored component of the gut-immune axis by profiling the antibody response to gut phages using Phage Immunoprecipitation Sequencing (PhIP-Seq). To enhance this approach, we developed Dolphyn, a novel method that uses machine learning to select peptides from protein sets and compresses the proteome through epitope-stitching. Dolphyn improves the fraction of gut phage library peptides bound by antibodies from 10% to 31% in healthy individuals, while also reducing the number of synthesized peptides by 78%. In our study on gut phages, we discovered that the immune system develops antibodies to bacteria-infecting viruses in the human gut, particularly E.coli-infecting Myoviridae. Cost-effective PhIP-Seq libraries designed with Dolphyn enable the assessment of a wider range of proteins in a single experiment, thus facilitating the study of the gut-immune axis.\nA gel-coated air-liquid-interface culture system with tunable substrate stiffness matching healthy and diseased lung tissues\nAuthors: He, Z.; Chu, C.; Dickson, R.; Cai, L.\nScore: 3.9, Published: 2023-07-30 DOI: 10.1101/2023.07.29.550710\nSince its invention in the late 1980s, the air-liquid-interface (ALI) culture system has been the standard in vitro model for studying human airway biology and pulmonary diseases. However, in a conventional ALI system, cells are cultured on a porous plastic membrane that is much stiffer than human airway tissues. Here, we develop a gel-ALI culture system by simply coating the plastic membrane with a thin layer of hydrogel with tunable stiffness matching that of healthy and fibrotic airway tissues. We determine the optimum gel thickness that does not impair the transport of nutrients and biomolecules essential to cell growth. We show that the gel-ALI system allows human bronchial epithelial cells (HBECs) to proliferate and differentiate into a pseudostratified epithelium. Further, we discover that HBECs migrate significantly faster on hydrogel substrates with stiffness matching that of fibrotic lung tissues, highlighting the importance of mechanical cues to human airway remodeling. The developed gel-ALI system provides a facile approach to studying the effects of mechanical cues on human airway biology and in modeling pulmonary diseases.\nImmunometabolic cues recompose and reprogram the microenvironment around biomaterials\nAuthors: Maduka, C. V.; Schmitter-Sanchez, A. D.; Makela, A. V.; Ural, E.; Stivers, K. B.; Pope, H.; Kuhnert, M. M.; Habeeb, O. M.; Tundo, A.; Alhaj, M.; Kiselev, A.; Chen, S.; Olive, A. J.; Hankenson, K. D. D.; Narayan, R.; Park, S.; Elisseeff, J. H.; Contag, C. H.\nScore: 3.8, Published: 2023-07-31 DOI: 10.1101/2023.07.30.551180\nCirculating monocytes infiltrate and coordinate immune responses in various inflamed tissues, such as those surrounding implanted biomaterials, affecting therapeutic, diagnostic, tissue engineering and regenerative applications. Here, we show that immunometabolic cues in the biomaterial microenvironment govern CCR2- and CX3CR1-dependent trafficking of immune cells, including neutrophils and monocytes; ultimately, this affects the composition and activation states of macrophage and dendritic cell populations. Furthermore, immunometabolic cues around implants orchestrate the relative composition of proinflammatory, transitory and anti-inflammatory CCR2+, CX3CR1+ and CCR2+CX3CR1+ immune cell populations. Consequently, modifying immunometabolism by glycolytic inhibition drives a pro-regenerative microenvironment in part by myeloid cells around amorphous polylactide implants. In addition to, Arginase 1-expressing myeloid cells, T helper 2 cells and {gamma}{delta}+ T-cells producing IL-4 significantly contribute to shaping the metabolically reprogramed, pro-regenerative microenvironment around crystalline polylactide biomaterials. Taken together, we find that local metabolic states regulate inflammatory processes in the biomaterial microenvironment, with implications for translational medicine.\nOn automated model discovery and a universal material subroutine\nAuthors: Peirlinck, M.; Linka, K.; Hurtado, J. A.; Kuhl, E.\nScore: 5.9, Published: 2023-07-25 DOI: 10.1101/2023.07.19.549749\nConstitutive modeling is the cornerstone of computational and structural mechanics. In a finite element analysis, the constitutive model is encoded in the material subroutine, a function that maps local strains onto stresses. This function is called within every finite element, at each integration point, within every time step, at each Newton iteration. Todays finite element packages offer large libraries of material subroutines to choose from. However, the scientific criteria for model selection remain highly subjective and prone to user bias. Here we fully automate the process of model selection, autonomously discover the best model and parameters from experimental data, encode all possible discoverable models into a single material subroutine, and seamlessly integrate this universal material subroutine into a finite element analysis. We prototype this strategy for tension, compression, and shear data from human brain tissue and perform a hyperelastic model discovery from twelve possible terms. These terms feature the first and second invariants, raised to the first and second powers, embedded in the identity, exponential, and logarithmic functions, generating 22x2x3 = 4096 models in total. We demonstrate how to integrate these models into a single universal material subroutine that features the classical neo Hooke, Blatz Ko, Mooney Rivlin, Demiray, Gent, and Holzapfel models as special cases. Finite element simulations with our universal material subroutine show that it specializes well to these widely used models, generalizes well to newly discovered models, and agrees excellently with both experimental data and previous simulations. It also performs well within realistic finite element simulations and accurately predicts stress concentrations in the human brain for six different head impact scenarios. We anticipate that integrating automated model discovery into a universal material subroutine will generalize naturally to more complex anisotropic, compressible, and inelastic materials and to other nonlinear finite element platforms. Replacing dozens of individual material subroutines by a single universal material subroutine that is populated directly via automated model discovery--entirely without human interaction--makes finite element analyses more accessible, more robust, and less vulnerable to human error. This could forever change how we simulate materials and structures.\nLarge-FOV 3D localization microscopy by spatially variant point spread function generation\nAuthors: Xiao, D.; Kedem Orange, R.; Opatovski, N.; Parizat, A.; Nehme, E.; Alalouf, O.; Shechtman, Y.\nScore: 2.5, Published: 2023-07-31 DOI: 10.1101/2023.07.30.551150\nAccurate characterization of the microscopic point spread function (PSF) is crucial for achieving high-performance localization microscopy (LM). Traditionally, LM assumes a spatially-invariant PSF to simplify the modeling of the imaging system. However, for large fields of view (FOV) imaging, it becomes important to account for the spatially variant nature of the PSF. In this work, we propose an accurate and fast principal component analysis (PCA)-based field-dependent 3D PSF generator (PPG3D) and localizer for LM. Through simulations and experimental 3D single molecule localization microscopy (SMLM), we demonstrate the effectiveness of PPG3D, enabling super-resolution imaging of mitochondria and microtubules with high fidelity over a large FOV. A comparison of PPG3D with three other shift-invariant and shift-variant PSF generators for 3D LM reveals a three-fold improvement in accuracy and an operation speed approximately one hundred times faster. Given its user-friendliness and conciseness, we believe that PPG3D holds great potential for widespread application in SMLM and other imaging modalities.\nFibroblasts-dependent maturation and phenotype exacerbation of dystrophic hiPSC-derived MYOtissues enables muscle strength evaluation for gene therapy screening\nAuthors: Palmieri, L.; Moula, M.; Jaber, A.; El Khoury, R.; Brochiet, G.; Bigot, A.; Israeli, D.; Richard, I.; Albini, S.\nScore: 2.5, Published: 2023-07-28 DOI: 10.1101/2023.07.26.550063\nDuchenne muscular dystrophy (DMD) is a lethal muscle wasting disease caused by absence of dystrophin, a protein essential to preserve muscle integrity continuously challenged by contractions. Gene therapy utilizing adeno-associated virus (AAV) to deliver truncated forms of dystrophin ({micro}Dys) is currently the most promising therapeutic approach. However, the therapeutic outcome in treated patients has not been as successful as anticipated by animal studies, underscoring the need of improved and high-throughput models for accurate and fast prediction of human response. Here, we describe the generation of MYOtissues, a 3D muscle platform based on direct myogenic conversion of human induced pluripotent stem cells (iPSC), whose structural and functional maturation is enhanced by fibroblasts incorporation. MYOtissues derived from DMD-iPSC including DMD fibroblasts, exacerbated pathogenic hallmarks such as fibrosis and muscle force loss. As a proof of concept, we showed that AAV-mediated {micro}Dys gene transfer improved muscle resistance and membrane stability in DMD-MYOtissues, highlighting the suitability of our system for gene therapy screening.\nGenerating an organ-deficient animal model using a multi-targeted CRISPR-Cas9 system\nAuthors: Lim, J. J.-Y.; Yuri, S.; Kawai, T.; Isotani, A.\nScore: 2.2, Published: 2023-08-01 DOI: 10.1101/2023.07.31.551401\nGene-knockout animal models with organ-deficient phenotypes used for blastocyst complementation are generally not viable. Animals need to be maintained as heterozygous mutants, and homozygous mutant embryos yield only one-fourth of all embryos. In this study, we generated organ-deficient embryos using the CRISPR-Cas9-sgRNAms system that induces cell death with a single-guide RNA (sgRNAms) targeting multiple sites in the genome. The Cas9-sgRNAms system interrupted cell proliferation and induced cell ablation in vitro. The mouse model had Cas9 driven by the Foxn1 promoter with a ubiquitous expression cassette of sgRNAms at the Rosa26 locus (Foxn1Cas9; Rosa26_ms). It showed an athymic phenotype similar to that of nude mice but was not hairless. Eventually, a rat cell-derived thymus in an interspecific chimera was generated by blastocyst complementation of Foxn1Cas9; Rosa26_ms mouse embryos with rat embryonic stem cells. Theoretically, half of the total embryos have the Cas9-sgRNAms system because Rosa26_ms could be maintained as homozygous.\nSimultaneous multi-site editing of individual genomes using retron arrays\nAuthors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.\nScore: 29.7, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549397\nOur understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites. Unfortunately, building cells with multiple, non-adjacent precise mutations remains a laborious cycle of editing, isolating an edited cell, and editing again. Here, we describe a technology for precisely modifying multiple sites on a single genome simultaneously. This technology - termed a multitron - is built from a heavily modified retron, in which multiple donor-encoding msds are produced from a single transcript. The multitron architecture is compatible with both recombineering in prokaryotic cells and CRISPR editing in eukaryotic cells. We demonstrate applications for this approach in molecular recording, genetic element minimization, and metabolic engineering.\nSoft robotics-enabled large animal model of HFpEF hemodynamics for device testing\nAuthors: Rosalia, L.; Ozturk, C.; Wang, S.; Quevedo, D.; Saeed, M.; Mauskapf, A.; Roche, E.\nScore: 2.1, Published: 2023-07-29 DOI: 10.1101/2023.07.26.550654\nHeart failure with preserved ejection fraction (HFpEF) is a major challenge in cardiovascular medicine, accounting for approximately 50% of all cases of heart failure. Due to the lack of effective therapies for this condition, the mortality associated with HFpEF remains higher than that of most cancers. Despite the ongoing efforts, no medical device has yet received FDA approval. This is largely due to the lack of an in vivo model of the HFpEF hemodynamics, resulting in the inability to evaluate device effectiveness in vivo prior to clinical trials. Here, we describe the development of a highly tunable porcine model of HFpEF hemodynamics using implantable soft robotic sleeves, where controlled actuation of a left ventricular and an aortic sleeve can recapitulate changes in ventricular compliance and afterload associated with a broad spectrum of HFpEF hemodynamic phenotypes. We demonstrate the feasibility of the proposed model in preclinical testing by evaluating the hemodynamic response of the model post-implantation of an interatrial shunt device, which was found to be consistent with findings from in silico studies and clinical trials. This work addresses several of the limitations associated with previous models of HFpEF, such as their limited hemodynamic fidelity, elevated costs, lengthy development time, and low throughput. By showcasing exceptional versatility and tunability, the proposed platform has the potential to revolutionize the current approach for HFpEF device development and selection, with the goal of improving the quality of life for the 32 million people affected by HFpEF worldwide.\n",
  "wordCount" : "2241",
  "inLanguage": "en",
  "datePublished": "2023-08-02T10:38:09Z",
  "dateModified": "2023-08-02T10:38:09Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on August 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550304">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550304" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550304">
        <p class="paperTitle">Single-cell precision nanotechnology in vivo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550304" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550304" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: MOLBAY, M.; Kick, B.; Zhao, S.; Todorov, M. I.; Ohn, T.-L.; Roth, S.; Simats, A.; Singh, V.; Khalin, I.; Pan, C.; Singh Bhatia, H.; Hellal, F.; Zeidler, R.; Liesz, A.; Plesnila, N.; Dietz, H.; Erturk, A.</p>
        <p class="info">Score: 124.7, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550304' target='https://doi.org/10.1101/2023.07.24.550304'> 10.1101/2023.07.24.550304</a></p>
        <p class="abstract">Targeting nanoparticle therapeutics with cellular accuracy in whole organisms could open breakthrough opportunities in precision medicine. However, evaluating and fine-tuning the biodistribution of such systems in the whole organism at the cellular level remains a major obstacle. Here, we constructed targetable DNA origami, and analyzed biodistribution in transparent mice, in addition to studying tolerability, clearance kinetics, and immune response parameters. Untargeted DNA origami primarily accumulated in the spleen and the liver, while an immune cell-targeting variant successfully attached to immune cells throughout the body. A cancer cell-targeting mimetic co-localized on solid-tumor metastasis in the liver and the lung. These findings indicate that DNA origami can be directed in vivo, providing an important proof-of-concept and highlights the potential of high-resolution tissue-clearing imaging technologies in their development.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=121 SRC=&#34;FIGDIR/small/550304v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (44K):
org.highwire.dtl.DTLVardef@1a2a3c8org.highwire.dtl.DTLVardef@aec749org.highwire.dtl.DTLVardef@9dc36aorg.highwire.dtl.DTLVardef@18896ba_HPS_FORMAT_FIGEXP  M_FIG C_FIG HighlightsO_LIThis study demonstrates the potential of DNA origami-based drug delivery systems as versatile tool for for targeted delivery, which could be used to treat a range of diseases with applications.
C_LIO_LIThe immune compatibility, half-life, targeting efficiency, and the biodistribution evaluation of DNA origami indicate its potential for systemic drug delivery.
C_LIO_LIOur approach enables the assessment of biodistribution of nanoparticles in the intact body with a sensitivity to the single-cell level, highlighting the high-resolution tissue clearing technoloies in revealing DNA origamis feasibility for drug targeting.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.30.551179">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.30.551179" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.30.551179">
        <p class="paperTitle">Efficient encoding of large antigenic spaces by epitope prioritization with Dolphyn</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.30.551179" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.30.551179" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liebhoff, A.-M.; Venkataraman, T.; Morgenlander, W. R.; Na, M.; Kula, T.; Waugh, K.; Morrison, C.; Rewers, M.; Longman, R.; Round, J.; Elledge, S.; Ruczinski, I.; Langmead, B.; Larman, H. B.</p>
        <p class="info">Score: 4.9, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.30.551179' target='https://doi.org/10.1101/2023.07.30.551179'> 10.1101/2023.07.30.551179</a></p>
        <p class="abstract">We investigated a relatively underexplored component of the gut-immune axis by profiling the antibody response to gut phages using Phage Immunoprecipitation Sequencing (PhIP-Seq). To enhance this approach, we developed Dolphyn, a novel method that uses machine learning to select peptides from protein sets and compresses the proteome through epitope-stitching. Dolphyn improves the fraction of gut phage library peptides bound by antibodies from 10% to 31% in healthy individuals, while also reducing the number of synthesized peptides by 78%. In our study on gut phages, we discovered that the immune system develops antibodies to bacteria-infecting viruses in the human gut, particularly E.coli-infecting Myoviridae. Cost-effective PhIP-Seq libraries designed with Dolphyn enable the assessment of a wider range of proteins in a single experiment, thus facilitating the study of the gut-immune axis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.29.550710">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.29.550710" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.29.550710">
        <p class="paperTitle">A gel-coated air-liquid-interface culture system with tunable substrate stiffness matching healthy and diseased lung tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.29.550710" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.29.550710" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: He, Z.; Chu, C.; Dickson, R.; Cai, L.</p>
        <p class="info">Score: 3.9, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.29.550710' target='https://doi.org/10.1101/2023.07.29.550710'> 10.1101/2023.07.29.550710</a></p>
        <p class="abstract">Since its invention in the late 1980s, the air-liquid-interface (ALI) culture system has been the standard in vitro model for studying human airway biology and pulmonary diseases. However, in a conventional ALI system, cells are cultured on a porous plastic membrane that is much stiffer than human airway tissues. Here, we develop a gel-ALI culture system by simply coating the plastic membrane with a thin layer of hydrogel with tunable stiffness matching that of healthy and fibrotic airway tissues. We determine the optimum gel thickness that does not impair the transport of nutrients and biomolecules essential to cell growth. We show that the gel-ALI system allows human bronchial epithelial cells (HBECs) to proliferate and differentiate into a pseudostratified epithelium. Further, we discover that HBECs migrate significantly faster on hydrogel substrates with stiffness matching that of fibrotic lung tissues, highlighting the importance of mechanical cues to human airway remodeling. The developed gel-ALI system provides a facile approach to studying the effects of mechanical cues on human airway biology and in modeling pulmonary diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.30.551180">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.30.551180" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.30.551180">
        <p class="paperTitle">Immunometabolic cues recompose and reprogram the microenvironment around biomaterials</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.30.551180" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.30.551180" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Maduka, C. V.; Schmitter-Sanchez, A. D.; Makela, A. V.; Ural, E.; Stivers, K. B.; Pope, H.; Kuhnert, M. M.; Habeeb, O. M.; Tundo, A.; Alhaj, M.; Kiselev, A.; Chen, S.; Olive, A. J.; Hankenson, K. D. D.; Narayan, R.; Park, S.; Elisseeff, J. H.; Contag, C. H.</p>
        <p class="info">Score: 3.8, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.30.551180' target='https://doi.org/10.1101/2023.07.30.551180'> 10.1101/2023.07.30.551180</a></p>
        <p class="abstract">Circulating monocytes infiltrate and coordinate immune responses in various inflamed tissues, such as those surrounding implanted biomaterials, affecting therapeutic, diagnostic, tissue engineering and regenerative applications. Here, we show that immunometabolic cues in the biomaterial microenvironment govern CCR2- and CX3CR1-dependent trafficking of immune cells, including neutrophils and monocytes; ultimately, this affects the composition and activation states of macrophage and dendritic cell populations. Furthermore, immunometabolic cues around implants orchestrate the relative composition of proinflammatory, transitory and anti-inflammatory CCR2&#43;, CX3CR1&#43; and CCR2&#43;CX3CR1&#43; immune cell populations. Consequently, modifying immunometabolism by glycolytic inhibition drives a pro-regenerative microenvironment in part by myeloid cells around amorphous polylactide implants. In addition to, Arginase 1-expressing myeloid cells, T helper 2 cells and {gamma}{delta}&#43; T-cells producing IL-4 significantly contribute to shaping the metabolically reprogramed, pro-regenerative microenvironment around crystalline polylactide biomaterials. Taken together, we find that local metabolic states regulate inflammatory processes in the biomaterial microenvironment, with implications for translational medicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549749">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549749" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549749">
        <p class="paperTitle">On automated model discovery and a universal material subroutine</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549749" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549749" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peirlinck, M.; Linka, K.; Hurtado, J. A.; Kuhl, E.</p>
        <p class="info">Score: 5.9, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549749' target='https://doi.org/10.1101/2023.07.19.549749'> 10.1101/2023.07.19.549749</a></p>
        <p class="abstract">Constitutive modeling is the cornerstone of computational and structural mechanics. In a finite element analysis, the constitutive model is encoded in the material subroutine, a function that maps local strains onto stresses. This function is called within every finite element, at each integration point, within every time step, at each Newton iteration. Todays finite element packages offer large libraries of material subroutines to choose from. However, the scientific criteria for model selection remain highly subjective and prone to user bias. Here we fully automate the process of model selection, autonomously discover the best model and parameters from experimental data, encode all possible discoverable models into a single material subroutine, and seamlessly integrate this universal material subroutine into a finite element analysis. We prototype this strategy for tension, compression, and shear data from human brain tissue and perform a hyperelastic model discovery from twelve possible terms. These terms feature the first and second invariants, raised to the first and second powers, embedded in the identity, exponential, and logarithmic functions, generating 22x2x3 = 4096 models in total. We demonstrate how to integrate these models into a single universal material subroutine that features the classical neo Hooke, Blatz Ko, Mooney Rivlin, Demiray, Gent, and Holzapfel models as special cases. Finite element simulations with our universal material subroutine show that it specializes well to these widely used models, generalizes well to newly discovered models, and agrees excellently with both experimental data and previous simulations. It also performs well within realistic finite element simulations and accurately predicts stress concentrations in the human brain for six different head impact scenarios. We anticipate that integrating automated model discovery into a universal material subroutine will generalize naturally to more complex anisotropic, compressible, and inelastic materials and to other nonlinear finite element platforms. Replacing dozens of individual material subroutines by a single universal material subroutine that is populated directly via automated model discovery--entirely without human interaction--makes finite element analyses more accessible, more robust, and less vulnerable to human error. This could forever change how we simulate materials and structures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.30.551150">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.30.551150" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.30.551150">
        <p class="paperTitle">Large-FOV 3D localization microscopy by spatially variant point spread function generation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.30.551150" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.30.551150" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xiao, D.; Kedem Orange, R.; Opatovski, N.; Parizat, A.; Nehme, E.; Alalouf, O.; Shechtman, Y.</p>
        <p class="info">Score: 2.5, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.30.551150' target='https://doi.org/10.1101/2023.07.30.551150'> 10.1101/2023.07.30.551150</a></p>
        <p class="abstract">Accurate characterization of the microscopic point spread function (PSF) is crucial for achieving high-performance localization microscopy (LM). Traditionally, LM assumes a spatially-invariant PSF to simplify the modeling of the imaging system. However, for large fields of view (FOV) imaging, it becomes important to account for the spatially variant nature of the PSF. In this work, we propose an accurate and fast principal component analysis (PCA)-based field-dependent 3D PSF generator (PPG3D) and localizer for LM. Through simulations and experimental 3D single molecule localization microscopy (SMLM), we demonstrate the effectiveness of PPG3D, enabling super-resolution imaging of mitochondria and microtubules with high fidelity over a large FOV. A comparison of PPG3D with three other shift-invariant and shift-variant PSF generators for 3D LM reveals a three-fold improvement in accuracy and an operation speed approximately one hundred times faster. Given its user-friendliness and conciseness, we believe that PPG3D holds great potential for widespread application in SMLM and other imaging modalities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550063">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550063" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550063">
        <p class="paperTitle">Fibroblasts-dependent maturation and phenotype exacerbation of dystrophic hiPSC-derived MYOtissues enables muscle strength evaluation for gene therapy screening</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550063" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550063" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Palmieri, L.; Moula, M.; Jaber, A.; El Khoury, R.; Brochiet, G.; Bigot, A.; Israeli, D.; Richard, I.; Albini, S.</p>
        <p class="info">Score: 2.5, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550063' target='https://doi.org/10.1101/2023.07.26.550063'> 10.1101/2023.07.26.550063</a></p>
        <p class="abstract">Duchenne muscular dystrophy (DMD) is a lethal muscle wasting disease caused by absence of dystrophin, a protein essential to preserve muscle integrity continuously challenged by contractions. Gene therapy utilizing adeno-associated virus (AAV) to deliver truncated forms of dystrophin ({micro}Dys) is currently the most promising therapeutic approach. However, the therapeutic outcome in treated patients has not been as successful as anticipated by animal studies, underscoring the need of improved and high-throughput models for accurate and fast prediction of human response. Here, we describe the generation of MYOtissues, a 3D muscle platform based on direct myogenic conversion of human induced pluripotent stem cells (iPSC), whose structural and functional maturation is enhanced by fibroblasts incorporation. MYOtissues derived from DMD-iPSC including DMD fibroblasts, exacerbated pathogenic hallmarks such as fibrosis and muscle force loss. As a proof of concept, we showed that AAV-mediated {micro}Dys gene transfer improved muscle resistance and membrane stability in DMD-MYOtissues, highlighting the suitability of our system for gene therapy screening.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551401">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551401" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551401">
        <p class="paperTitle">Generating an organ-deficient animal model using a multi-targeted CRISPR-Cas9 system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551401" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551401" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lim, J. J.-Y.; Yuri, S.; Kawai, T.; Isotani, A.</p>
        <p class="info">Score: 2.2, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551401' target='https://doi.org/10.1101/2023.07.31.551401'> 10.1101/2023.07.31.551401</a></p>
        <p class="abstract">Gene-knockout animal models with organ-deficient phenotypes used for blastocyst complementation are generally not viable. Animals need to be maintained as heterozygous mutants, and homozygous mutant embryos yield only one-fourth of all embryos. In this study, we generated organ-deficient embryos using the CRISPR-Cas9-sgRNAms system that induces cell death with a single-guide RNA (sgRNAms) targeting multiple sites in the genome. The Cas9-sgRNAms system interrupted cell proliferation and induced cell ablation in vitro. The mouse model had Cas9 driven by the Foxn1 promoter with a ubiquitous expression cassette of sgRNAms at the Rosa26 locus (Foxn1Cas9; Rosa26_ms). It showed an athymic phenotype similar to that of nude mice but was not hairless. Eventually, a rat cell-derived thymus in an interspecific chimera was generated by blastocyst complementation of Foxn1Cas9; Rosa26_ms mouse embryos with rat embryonic stem cells. Theoretically, half of the total embryos have the Cas9-sgRNAms system because Rosa26_ms could be maintained as homozygous.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549397">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549397" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549397">
        <p class="paperTitle">Simultaneous multi-site editing of individual genomes using retron arrays</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549397" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549397" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.</p>
        <p class="info">Score: 29.7, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549397' target='https://doi.org/10.1101/2023.07.17.549397'> 10.1101/2023.07.17.549397</a></p>
        <p class="abstract">Our understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites. Unfortunately, building cells with multiple, non-adjacent precise mutations remains a laborious cycle of editing, isolating an edited cell, and editing again. Here, we describe a technology for precisely modifying multiple sites on a single genome simultaneously. This technology - termed a multitron - is built from a heavily modified retron, in which multiple donor-encoding msds are produced from a single transcript. The multitron architecture is compatible with both recombineering in prokaryotic cells and CRISPR editing in eukaryotic cells. We demonstrate applications for this approach in molecular recording, genetic element minimization, and metabolic engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550654">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550654" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550654">
        <p class="paperTitle">Soft robotics-enabled large animal model of HFpEF hemodynamics for device testing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550654" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550654" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rosalia, L.; Ozturk, C.; Wang, S.; Quevedo, D.; Saeed, M.; Mauskapf, A.; Roche, E.</p>
        <p class="info">Score: 2.1, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550654' target='https://doi.org/10.1101/2023.07.26.550654'> 10.1101/2023.07.26.550654</a></p>
        <p class="abstract">Heart failure with preserved ejection fraction (HFpEF) is a major challenge in cardiovascular medicine, accounting for approximately 50% of all cases of heart failure. Due to the lack of effective therapies for this condition, the mortality associated with HFpEF remains higher than that of most cancers. Despite the ongoing efforts, no medical device has yet received FDA approval. This is largely due to the lack of an in vivo model of the HFpEF hemodynamics, resulting in the inability to evaluate device effectiveness in vivo prior to clinical trials. Here, we describe the development of a highly tunable porcine model of HFpEF hemodynamics using implantable soft robotic sleeves, where controlled actuation of a left ventricular and an aortic sleeve can recapitulate changes in ventricular compliance and afterload associated with a broad spectrum of HFpEF hemodynamic phenotypes. We demonstrate the feasibility of the proposed model in preclinical testing by evaluating the hemodynamic response of the model post-implantation of an interatrial shunt device, which was found to be consistent with findings from in silico studies and clinical trials. This work addresses several of the limitations associated with previous models of HFpEF, such as their limited hemodynamic fidelity, elevated costs, lengthy development time, and low throughput. By showcasing exceptional versatility and tunability, the proposed platform has the potential to revolutionize the current approach for HFpEF device development and selection, with the goal of improving the quality of life for the 32 million people affected by HFpEF worldwide.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
