<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Molecular de-extinction of antibiotics enabled by deep learning
Authors: Wan, F.; Torres, M. D. T.; Peng, J.; de la Fuente, C.
Score: 87.3, Published: 2023-10-01 DOI: 10.1101/2023.10.01.560353
Molecular de-extinction is an emerging field that aims to resurrect molecules to solve present-day problems such as antibiotic resistance. Here, we introduce a deep learning approach called Antibiotic Peptide de-Extinction (APEX) to mine the proteomes of all available extinct organisms (the &#34;extinctome&#34;) searching for encrypted peptide (EP) antibiotics.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Molecular de-extinction of antibiotics enabled by deep learning
Authors: Wan, F.; Torres, M. D. T.; Peng, J.; de la Fuente, C.
Score: 87.3, Published: 2023-10-01 DOI: 10.1101/2023.10.01.560353
Molecular de-extinction is an emerging field that aims to resurrect molecules to solve present-day problems such as antibiotic resistance. Here, we introduce a deep learning approach called Antibiotic Peptide de-Extinction (APEX) to mine the proteomes of all available extinct organisms (the &#34;extinctome&#34;) searching for encrypted peptide (EP) antibiotics." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-08T10:37:16+00:00" />
<meta property="article:modified_time" content="2023-10-08T10:37:16+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Molecular de-extinction of antibiotics enabled by deep learning
Authors: Wan, F.; Torres, M. D. T.; Peng, J.; de la Fuente, C.
Score: 87.3, Published: 2023-10-01 DOI: 10.1101/2023.10.01.560353
Molecular de-extinction is an emerging field that aims to resurrect molecules to solve present-day problems such as antibiotic resistance. Here, we introduce a deep learning approach called Antibiotic Peptide de-Extinction (APEX) to mine the proteomes of all available extinct organisms (the &#34;extinctome&#34;) searching for encrypted peptide (EP) antibiotics."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Molecular de-extinction of antibiotics enabled by deep learning\nAuthors: Wan, F.; Torres, M. D. T.; Peng, J.; de la Fuente, C.\nScore: 87.3, Published: 2023-10-01 DOI: 10.1101/2023.10.01.560353\nMolecular de-extinction is an emerging field that aims to resurrect molecules to solve present-day problems such as antibiotic resistance. Here, we introduce a deep learning approach called Antibiotic Peptide de-Extinction (APEX) to mine the proteomes of all available extinct organisms (the \u0026#34;extinctome\u0026#34;) searching for encrypted peptide (EP) antibiotics.",
  "keywords": [
    
  ],
  "articleBody": " Molecular de-extinction of antibiotics enabled by deep learning\nAuthors: Wan, F.; Torres, M. D. T.; Peng, J.; de la Fuente, C.\nScore: 87.3, Published: 2023-10-01 DOI: 10.1101/2023.10.01.560353\nMolecular de-extinction is an emerging field that aims to resurrect molecules to solve present-day problems such as antibiotic resistance. Here, we introduce a deep learning approach called Antibiotic Peptide de-Extinction (APEX) to mine the proteomes of all available extinct organisms (the \"extinctome\") searching for encrypted peptide (EP) antibiotics. APEX mined a total of 10,311,899 EPs and identified 37,176 sequences predicted to have broad-spectrum antimicrobial activity, 11,035 of which were not found in extant organisms. Chemical synthesis and experimental validation yielded archaic EPs (AEPs) with activity against dangerous bacterial pathogens. Most peptides killed bacteria by depolarizing their cytoplasmic membrane, contrary to known antimicrobial peptides, which target the outer membrane. Notably, lead peptides, including those derived from the woolly mammoth, ancient sea cow, giant sloth, and extinct giant elk, exhibited anti-infective activity in preclinical mouse models. We propose molecular de-extinction, accelerated by deep learning, as a framework for discovering therapeutic molecules.\nProtein generation with evolutionary diffusion: sequence is all you need\nAuthors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.\nScore: 200.1, Published: 2023-09-12 DOI: 10.1101/2023.09.11.556673\nDeep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design. However, current state-of-the-art models generate protein structures, which limits the scope of their training data and restricts generations to a small and biased subset of protein design space. Here, we introduce a general-purpose diffusion framework, EvoDiff, that combines evolutionary-scale data with the distinct conditioning capabilities of diffusion models for controllable protein generation in sequence space. EvoDiff generates high-fidelity, diverse, and structurally-plausible proteins that cover natural sequence and functional space. Critically, EvoDiff can generate proteins inaccessible to structure-based models, such as those with disordered regions, while maintaining the ability to design scaffolds for functional structural motifs, demonstrating the universality of our sequence-based formulation. We envision that EvoDiff will expand capabilities in protein engineering beyond the structure-function paradigm toward programmable, sequence-first design.\nSpiral NeuroString: High-Density Soft Bioelectronic Fibers for Multimodal Sensing and Stimulation\nAuthors: Khatib, M.; Zhao, E. T.; Wei, S.; Abramson, A.; Bishop, E. B.; Chen, C.-H.; Thomas, A.-L.; Xu, C.; Park, J.; Lee, Y.; Hamnett, R.; Yu, W.; Root, S. E.; Yuan, L.; Chakhtoura, D.; Kim, K. K.; Zhong, D.; Nishio, Y.; Zhao, C.; Wu, C.; Jiang, Y.; Zhang, A.; Li, J.; Wang, W.; Salimi-Jazi, F.; Rafeeqi, T. A.; Hemed, N. M.; Tok, J. B.-H.; Chen, X.; Kaltschmidt, J. A.; Dunn, J.; Bao, Z.\nScore: 16.2, Published: 2023-10-03 DOI: 10.1101/2023.10.02.560482\nBioelectronic fibers hold promise for both research and clinical applications due to their compactness, ease of implantation, and ability to incorporate various functionalities such as sensing and stimulation. However, existing devices suffer from bulkiness, rigidity, limited functionality, and low density of active components. These limitations stem from the difficulty to incorporate many components on one-dimensional (1D) fiber devices due to the incompatibility of conventional microfabrication methods (e.g., photolithography) with curved, thin and long fiber structures. Herein, we introduce a fabrication approach, spiral transformation'', to convert two-dimensional (2D) films containing microfabricated devices into 1D soft fibers. This approach allows for the creation of high density multimodal soft bioelectronic fibers, termed Spiral NeuroString (S-NeuroString), while enabling precise control over the longitudinal, angular, and radial positioning and distribution of the functional components. We show the utility of S-NeuroString for motility mapping, serotonin sensing, and tissue stimulation within the dynamic and soft gastrointestinal (GI) system, as well as for single-unit recordings in the brain. The described bioelectronic fibers hold great promises for next-generation multifunctional implantable electronics.\nNext-generation genetic sexing strain establishment in the agricultural pest Ceratitis capitata\nAuthors: Davydova, S.; Liu, J.; Kandul, N. P.; Braswell, W. E.; Akbari, O. S.; Meccariello, A.\nScore: 10.0, Published: 2023-10-01 DOI: 10.1101/2023.09.29.560088\nTephritid fruit fly pests pose an increasing threat to the agricultural industry due to their global dispersion and a highly invasive nature. Here we showcase the feasibility of an early-detection SEPARATOR sex sorting approach through using the non-model Tephritid pest, Ceratitis capitata. This system relies on female-only fluorescent marker expression, accomplished through the use of a sex-specific intron of the highly-conserved transformer gene from C. capitata and Anastrepha ludens. The herein characterized strains have 100% desired phenotype outcomes, allowing accurate male-female separation during early development. Overall, we describe an antibiotic and temperature-independent sex-sorting system in C. capitata, which, moving forward, may be implemented in other non-model Tephritid pest species. This strategy can facilitate the establishment of genetic sexing systems with endogenous elements exclusively, which, on a wider scale, can improve pest population control strategies like sterile insect technique.\nBuilding on-chip cytoskeletal circuits via branched microtubule networks\nAuthors: Zaferani, M.; Song, R.; Petry, S.; Stone, H. A.\nScore: 9.0, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560515\nControllable platforms to engineer robust cytoskeletal scaffolds have the potential to create novel on-chip nanotechnologies. Inspired by axons, we combined the branching microtubule (MT) nucleation pathway with microfabrication to develop \"cytoskeletal circuits\". This active matter platform allows control over the adaptive self-organization of uniformly polarized MT arrays via geometric features of microstructures designed within a microfluidic confinement. We build and characterize basic elements, including turns and divisions, as well as complex regulatory elements, such as biased division and MT diodes, to construct various MT architectures on a chip. Our platform could be used in diverse applications, ranging from efficient on-chip molecular transport to mechanical nano-actuators. Further, cytoskeletal circuits can serve as a tool to study how the physical environment contributes to MT architecture in living cells. SignificanceMicrotubules have essential functions within the cell, including providing a robust railroad for motor-driven cargo transport. The unique properties of microtubules have stimulated attempts to harness these characteristics for targeted delivery of molecular complexes, novel material design, and developing nanotechnologies with precision comparable to living organisms. However, Previous efforts mainly focused on microtubules with fixed length and layout and no controlled MT generation, setting a limit on designing MT architecture. In this study, we integrated nanofabrication with microtubule branching reactions borrowed directly from the cells toolkit to construct cytoskeletal circuits and generate microtubule architectures from scratch. That is, our system enables control over microtubule growth and autocatalytic nucleation on a microfluidic chip with micro/nanostructures patterned within.\nA Drug-Free Pathogen Capture and Neutralizing Nasal Spray to Prevent Emerging Respiratory Infections\nAuthors: Joseph, J.; Baby, H. M.; Quintero, J. R.; Kenney, D.; Mebratu, Y.; Bhatia, E.; Shah, P.; Swain, K.; Kaur, S.; Li, X. L.; Mwangi, J.; Snapper, O.; Nair, R.; Agus, E.; Ranganathan, S.; Kage, J.; Gao, J.; Luo, J. N.; Yu, A.; Douam, F.; Tesfaigzi, Y.; Karp, J.; Joshi, N.\nScore: 8.0, Published: 2023-10-03 DOI: 10.1101/2023.10.02.560602\nRespiratory infections pose a global health crisis. Vaccines are pathogen specific, and new vaccines are needed for mutants and emerging pathogens. Here, we report a drug free prophylactic platform - a Pathogen Capture and Neutralizing Spray (PCANS) that acts via a multi-pronged approach to prevent a broad spectrum of respiratory infections. PCANS forms a protective coating in the nasal cavity that enhances the capture of large respiratory droplets. The coating acts as a physical barrier against a broad spectrum of viruses and bacteria, and rapidly neutralizes them, reducing the pathogen load by \u003e99.99%. In mice, PCANS showed nasal retention for at least 8 h and was safe for daily administration. A single prophylactic dose of PCANS protected mice against supra-lethal dosages of a mouse-adapted H1N1 Influenza virus (PR8), reduced lung viral titer by \u003e99.99%, improved survival, and suppressed pathological manifestations. Together, our data suggest PCANS as a promising daily-use prophylactic approach against current and emerging respiratory infections.\nClick editing enables programmable genome writing using DNA polymerases and HUH endonucleases\nAuthors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.\nScore: 106.5, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557440\nGenome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e.g. CRISPR-Cas) to permit the installation of a range of edits including substitutions, insertions, and deletions. Click editors (CEs) leverage the \"click\"-like bioconjugation ability of HUH endonucleases (HUHes) with single stranded DNA substrates to covalently tether \"click DNA\" (clkDNA) templates encoding user-specifiable edits at targeted genomic loci. Through iterative optimization of the modular components of CEs (DNA polymerase and HUHe orthologs, architectural modifications, etc.) and their clkDNAs (template configurations, repair evading substitutions, etc.), we demonstrate the ability to install precise genome edits with minimal indels and no unwanted byproduct insertions. Since clkDNAs can be ordered as simple DNA oligonucleotides for cents per base, it is possible to screen many different clkDNA parameters rapidly and inexpensively to maximize edit efficiency. Together, click editing is a precise and highly versatile platform for modifying genomes with a simple workflow and broad utility across diverse biological applications.\nRewiring native post-transcriptional carbon regulators to build multi-layered genetic circuits and optimize engineered microbes for bioproduction\nAuthors: Simmons, T. R.; Partipilo, G.; Stankes, A. C.; Srivastava, R.; Buchser, R.; Chiu, D.; Keitz, B. K.; Contreras, L. M.\nScore: 6.8, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560922\nAs the bioprocessing and synthetic biology spaces rapidly expand, post-transcriptional regulation is emerging as driver for maximizing signal response rate and for minimizing cost per protein within cells. Designing robust post-transcriptional control systems that have precise tunability and can achieve diverse regulatory outcomes are paramount to advance the field. Herein, we develop a new approach for engineered post-transcriptional control in bacteria by rewiring a native regulatory system in Escherichia coli, the Carbon Storage Regulatory (Csr) Network, to create tunable, complex genetic circuits. First, by co-opting native components of the Csr Network to regulate translation of a target mRNA transcript, we establish a Csr-regulated Buffer Gate. Next, by rationally engineering the interactions between our synthetic construct and the native components of the Csr Network, we expand our original design into a genetic toolbox of 12 Buffer Gates that achieve precise tunability across a 10-fold range of target gene expression. Subsequently, to further regulatory capabilities using this approach, we develop a Csr-regulated NOT Gate through engineering a Csr-activated sequence into our synthetic constructs. We then build upon the Csr Buffer and Not Gates to create post-transcriptional dual input Boolean OR, NOR, AND and NAND Logic Gates, as well as a genetic pulse circuit. As a third step, we demonstrate portability of our Csr-regulated Buffer Gates into three industrially relevant bacteria by recapitulating Buffer Gate activity simply by leveraging the conserved homologous Csr Network in each species. Lastly, as a demonstration of downstream application, we apply our system to a proof-of-concept synthetic mevalonate pathway. Using our engineered constructs, we optimize mevalonate production in E. coli resulting in a three-fold increase in production relative to a transcriptionally controlled mevalonate pathway. As a whole, we establish a novel approach to rewire post-transcriptional regulatory networks for complex bacterial computation that can be utilized for efficient bioproduction in engineered microbes.\nMultichannel bioelectronic sensing using engineered Escherichia coli\nAuthors: Zhang, X.; Ajo-Franklin, C.\nScore: 6.0, Published: 2023-09-30 DOI: 10.1101/2023.09.30.560307\nBy engineering extracellular electron transfer (EET) to be dependent on an analyte, researchers have developed whole cell bioelectronic sensors that sense hazards to human and environmental health1. However, these sensors regulate a single electron transfer pathway as an electrochemical channel, limiting the sensing information to a single analyte. To increase information content, we developed a multichannel bioelectronic sensor through which different chemicals regulate distinct extracellular electron transfer pathways within a single Escherichia coli cell. One channel utilizes the flavin synthesis pathway from Bacillus subtilis2 and the other a set of cytochromes constructing the Mtr pathway from Shewanella oneidensis3. We demonstrate an arsenite responsive promoter can control the Mtr pathway through activation of cytochrome CymA expression and a cadmium responsive promoter can control the flavin synthesis pathway4,5. The redox potential of flavin-mediated EET is different from that of CymA-mediated one6. This allowed for development of a redox-potential-dependent algorithm that distinguishes variable input signals of each analyte mediated by two EET pathways in vivo. This approach enables a 2-bit binary signal readout for real-time tracking throughout the entire sensing duration. Our multichannel bioelectronic sensor was able to accurately sense and distinguish different heavy metals in Brays Bayou water samples with a response time comparable to that in clean water. This multichannel bioelectronic sensors allow for simultaneous detection of different chemicals, significantly expanding information transmission and helping to safeguard human and environmental health.\nAccurate Identification of Motoneuron Discharges from Ultrasound Images Across the Full Muscle Cross-Section\nAuthors: Lubel, E.; Rohlen, R.; Grandi Sgambato, B.; Barsakcioglu, D.; Ibanez, J.; Tang, M.-x.; Farina, D.\nScore: 5.2, Published: 2023-10-02 DOI: 10.1101/2023.09.29.560220\nObjective: Non-invasive identification of motoneuron (MN) activity is commonly done using (EMG). However, surface EMG (sEMG) signals detect only superficial sources, at less than approximately 10-mm depth. Intramuscular EMG can detect deep sources, but it is limited to sources within a few mm of the detection site. Conversely, ultrasound (US) images have high spatial resolution across the whole muscle cross-section. The activity of MNs can be extracted from US images due to the movements that MN activation generates in the innervated muscle fibers. Current US-based decomposition methods can accurately identify the location and average twitch induced by MN activity. However, they cannot accurately detect MN discharge times. Methods: Here, we present a method based on the convolutive blind source separation of US images to estimate MN discharge times with high accuracy. The method was validated across 10 participants using concomitant sEMG decomposition as the ground truth. Results: 140 unique MN spike trains were identified from US images, with a rate of agreement (RoA) with sEMG decomposition of 87.4 {+/-} 10.3 %. Over 50% of these MN spike trains had a RoA greater than 90%. Furthermore, with US, we identified additional MUs well beyond the sEMG detection volume, at up to \u003e30 mm below the skin. Conclusion: The proposed method can identify discharges of MNs innervating muscle fibers in a large range of depths within the muscle from US images. Significance: The proposed methodology can non-invasively interface with the outer layers of the central nervous system innervating muscles across the full cross-section.\n",
  "wordCount" : "2409",
  "inLanguage": "en",
  "datePublished": "2023-10-08T10:37:16Z",
  "dateModified": "2023-10-08T10:37:16Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on October 8, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.560353">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.560353" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.560353">
        <p class="paperTitle">Molecular de-extinction of antibiotics enabled by deep learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.560353" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.560353" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wan, F.; Torres, M. D. T.; Peng, J.; de la Fuente, C.</p>
        <p class="info">Score: 87.3, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.560353' target='https://doi.org/10.1101/2023.10.01.560353'> 10.1101/2023.10.01.560353</a></p>
        <p class="abstract">Molecular de-extinction is an emerging field that aims to resurrect molecules to solve present-day problems such as antibiotic resistance. Here, we introduce a deep learning approach called Antibiotic Peptide de-Extinction (APEX) to mine the proteomes of all available extinct organisms (the &#34;extinctome&#34;) searching for encrypted peptide (EP) antibiotics. APEX mined a total of 10,311,899 EPs and identified 37,176 sequences predicted to have broad-spectrum antimicrobial activity, 11,035 of which were not found in extant organisms. Chemical synthesis and experimental validation yielded archaic EPs (AEPs) with activity against dangerous bacterial pathogens. Most peptides killed bacteria by depolarizing their cytoplasmic membrane, contrary to known antimicrobial peptides, which target the outer membrane. Notably, lead peptides, including those derived from the woolly mammoth, ancient sea cow, giant sloth, and extinct giant elk, exhibited anti-infective activity in preclinical mouse models. We propose molecular de-extinction, accelerated by deep learning, as a framework for discovering therapeutic molecules.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.556673">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.556673" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.556673">
        <p class="paperTitle">Protein generation with evolutionary diffusion: sequence is all you need</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.556673" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.556673" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.</p>
        <p class="info">Score: 200.1, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.556673' target='https://doi.org/10.1101/2023.09.11.556673'> 10.1101/2023.09.11.556673</a></p>
        <p class="abstract">Deep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design. However, current state-of-the-art models generate protein structures, which limits the scope of their training data and restricts generations to a small and biased subset of protein design space. Here, we introduce a general-purpose diffusion framework, EvoDiff, that combines evolutionary-scale data with the distinct conditioning capabilities of diffusion models for controllable protein generation in sequence space. EvoDiff generates high-fidelity, diverse, and structurally-plausible proteins that cover natural sequence and functional space. Critically, EvoDiff can generate proteins inaccessible to structure-based models, such as those with disordered regions, while maintaining the ability to design scaffolds for functional structural motifs, demonstrating the universality of our sequence-based formulation. We envision that EvoDiff will expand capabilities in protein engineering beyond the structure-function paradigm toward programmable, sequence-first design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560482">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560482" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560482">
        <p class="paperTitle">Spiral NeuroString: High-Density Soft Bioelectronic Fibers for Multimodal Sensing and Stimulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560482" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560482" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khatib, M.; Zhao, E. T.; Wei, S.; Abramson, A.; Bishop, E. B.; Chen, C.-H.; Thomas, A.-L.; Xu, C.; Park, J.; Lee, Y.; Hamnett, R.; Yu, W.; Root, S. E.; Yuan, L.; Chakhtoura, D.; Kim, K. K.; Zhong, D.; Nishio, Y.; Zhao, C.; Wu, C.; Jiang, Y.; Zhang, A.; Li, J.; Wang, W.; Salimi-Jazi, F.; Rafeeqi, T. A.; Hemed, N. M.; Tok, J. B.-H.; Chen, X.; Kaltschmidt, J. A.; Dunn, J.; Bao, Z.</p>
        <p class="info">Score: 16.2, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560482' target='https://doi.org/10.1101/2023.10.02.560482'> 10.1101/2023.10.02.560482</a></p>
        <p class="abstract">Bioelectronic fibers hold promise for both research and clinical applications due to their compactness, ease of implantation, and ability to incorporate various functionalities such as sensing and stimulation. However, existing devices suffer from bulkiness, rigidity, limited functionality, and low density of active components. These limitations stem from the difficulty to incorporate many components on one-dimensional (1D) fiber devices due to the incompatibility of conventional microfabrication methods (e.g., photolithography) with curved, thin and long fiber structures. Herein, we introduce a fabrication approach, spiral transformation&#39;&#39;, to convert two-dimensional (2D) films containing microfabricated devices into 1D soft fibers. This approach allows for the creation of high density multimodal soft bioelectronic fibers, termed Spiral NeuroString (S-NeuroString), while enabling precise control over the longitudinal, angular, and radial positioning and distribution of the functional components. We show the utility of S-NeuroString for motility mapping, serotonin sensing, and tissue stimulation within the dynamic and soft gastrointestinal (GI) system, as well as for single-unit recordings in the brain. The described bioelectronic fibers hold great promises for next-generation multifunctional implantable electronics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560088">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560088" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560088">
        <p class="paperTitle">Next-generation genetic sexing strain establishment in the agricultural pest Ceratitis capitata</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560088" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560088" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davydova, S.; Liu, J.; Kandul, N. P.; Braswell, W. E.; Akbari, O. S.; Meccariello, A.</p>
        <p class="info">Score: 10.0, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560088' target='https://doi.org/10.1101/2023.09.29.560088'> 10.1101/2023.09.29.560088</a></p>
        <p class="abstract">Tephritid fruit fly pests pose an increasing threat to the agricultural industry due to their global dispersion and a highly invasive nature. Here we showcase the feasibility of an early-detection SEPARATOR sex sorting approach through using the non-model Tephritid pest, Ceratitis capitata. This system relies on female-only fluorescent marker expression, accomplished through the use of a sex-specific intron of the highly-conserved transformer gene from C. capitata and Anastrepha ludens. The herein characterized strains have 100% desired phenotype outcomes, allowing accurate male-female separation during early development. Overall, we describe an antibiotic and temperature-independent sex-sorting system in C. capitata, which, moving forward, may be implemented in other non-model Tephritid pest species. This strategy can facilitate the establishment of genetic sexing systems with endogenous elements exclusively, which, on a wider scale, can improve pest population control strategies like sterile insect technique.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560515">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560515" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560515">
        <p class="paperTitle">Building on-chip cytoskeletal circuits via branched microtubule networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560515" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560515" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zaferani, M.; Song, R.; Petry, S.; Stone, H. A.</p>
        <p class="info">Score: 9.0, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560515' target='https://doi.org/10.1101/2023.10.02.560515'> 10.1101/2023.10.02.560515</a></p>
        <p class="abstract">Controllable platforms to engineer robust cytoskeletal scaffolds have the potential to create novel on-chip nanotechnologies. Inspired by axons, we combined the branching microtubule (MT) nucleation pathway with microfabrication to develop &#34;cytoskeletal circuits&#34;. This active matter platform allows control over the adaptive self-organization of uniformly polarized MT arrays via geometric features of microstructures designed within a microfluidic confinement. We build and characterize basic elements, including turns and divisions, as well as complex regulatory elements, such as biased division and MT diodes, to construct various MT architectures on a chip. Our platform could be used in diverse applications, ranging from efficient on-chip molecular transport to mechanical nano-actuators. Further, cytoskeletal circuits can serve as a tool to study how the physical environment contributes to MT architecture in living cells.

SignificanceMicrotubules have essential functions within the cell, including providing a robust railroad for motor-driven cargo transport. The unique properties of microtubules have stimulated attempts to harness these characteristics for targeted delivery of molecular complexes, novel material design, and developing nanotechnologies with precision comparable to living organisms. However, Previous efforts mainly focused on microtubules with fixed length and layout and no controlled MT generation, setting a limit on designing MT architecture. In this study, we integrated nanofabrication with microtubule branching reactions borrowed directly from the cells toolkit to construct cytoskeletal circuits and generate microtubule architectures from scratch. That is, our system enables control over microtubule growth and autocatalytic nucleation on a microfluidic chip with micro/nanostructures patterned within.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560602">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560602" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560602">
        <p class="paperTitle">A Drug-Free Pathogen Capture and Neutralizing Nasal Spray to Prevent Emerging Respiratory Infections</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560602" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560602" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Joseph, J.; Baby, H. M.; Quintero, J. R.; Kenney, D.; Mebratu, Y.; Bhatia, E.; Shah, P.; Swain, K.; Kaur, S.; Li, X. L.; Mwangi, J.; Snapper, O.; Nair, R.; Agus, E.; Ranganathan, S.; Kage, J.; Gao, J.; Luo, J. N.; Yu, A.; Douam, F.; Tesfaigzi, Y.; Karp, J.; Joshi, N.</p>
        <p class="info">Score: 8.0, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560602' target='https://doi.org/10.1101/2023.10.02.560602'> 10.1101/2023.10.02.560602</a></p>
        <p class="abstract">Respiratory infections pose a global health crisis. Vaccines are pathogen specific, and new vaccines are needed for mutants and emerging pathogens. Here, we report a drug free prophylactic platform - a Pathogen Capture and Neutralizing Spray (PCANS) that acts via a multi-pronged approach to prevent a broad spectrum of respiratory infections. PCANS forms a protective coating in the nasal cavity that enhances the capture of large respiratory droplets. The coating acts as a physical barrier against a broad spectrum of viruses and bacteria, and rapidly neutralizes them, reducing the pathogen load by &gt;99.99%. In mice, PCANS showed nasal retention for at least 8 h and was safe for daily administration. A single prophylactic dose of PCANS protected mice against supra-lethal dosages of a mouse-adapted H1N1 Influenza virus (PR8), reduced lung viral titer by &gt;99.99%, improved survival, and suppressed pathological manifestations. Together, our data suggest PCANS as a promising daily-use prophylactic approach against current and emerging respiratory infections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557440">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557440" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557440">
        <p class="paperTitle">Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557440" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557440" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.</p>
        <p class="info">Score: 106.5, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557440' target='https://doi.org/10.1101/2023.09.12.557440'> 10.1101/2023.09.12.557440</a></p>
        <p class="abstract">Genome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e.g. CRISPR-Cas) to permit the installation of a range of edits including substitutions, insertions, and deletions. Click editors (CEs) leverage the &#34;click&#34;-like bioconjugation ability of HUH endonucleases (HUHes) with single stranded DNA substrates to covalently tether &#34;click DNA&#34; (clkDNA) templates encoding user-specifiable edits at targeted genomic loci. Through iterative optimization of the modular components of CEs (DNA polymerase and HUHe orthologs, architectural modifications, etc.) and their clkDNAs (template configurations, repair evading substitutions, etc.), we demonstrate the ability to install precise genome edits with minimal indels and no unwanted byproduct insertions. Since clkDNAs can be ordered as simple DNA oligonucleotides for cents per base, it is possible to screen many different clkDNA parameters rapidly and inexpensively to maximize edit efficiency. Together, click editing is a precise and highly versatile platform for modifying genomes with a simple workflow and broad utility across diverse biological applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560922">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560922" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560922">
        <p class="paperTitle">Rewiring native post-transcriptional carbon regulators to build multi-layered genetic circuits and optimize engineered microbes for bioproduction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560922" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560922" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Simmons, T. R.; Partipilo, G.; Stankes, A. C.; Srivastava, R.; Buchser, R.; Chiu, D.; Keitz, B. K.; Contreras, L. M.</p>
        <p class="info">Score: 6.8, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560922' target='https://doi.org/10.1101/2023.10.04.560922'> 10.1101/2023.10.04.560922</a></p>
        <p class="abstract">As the bioprocessing and synthetic biology spaces rapidly expand, post-transcriptional regulation is emerging as driver for maximizing signal response rate and for minimizing cost per protein within cells. Designing robust post-transcriptional control systems that have precise tunability and can achieve diverse regulatory outcomes are paramount to advance the field. Herein, we develop a new approach for engineered post-transcriptional control in bacteria by rewiring a native regulatory system in Escherichia coli, the Carbon Storage Regulatory (Csr) Network, to create tunable, complex genetic circuits. First, by co-opting native components of the Csr Network to regulate translation of a target mRNA transcript, we establish a Csr-regulated Buffer Gate. Next, by rationally engineering the interactions between our synthetic construct and the native components of the Csr Network, we expand our original design into a genetic toolbox of 12 Buffer Gates that achieve precise tunability across a 10-fold range of target gene expression. Subsequently, to further regulatory capabilities using this approach, we develop a Csr-regulated NOT Gate through engineering a Csr-activated sequence into our synthetic constructs. We then build upon the Csr Buffer and Not Gates to create post-transcriptional dual input Boolean OR, NOR, AND and NAND Logic Gates, as well as a genetic pulse circuit. As a third step, we demonstrate portability of our Csr-regulated Buffer Gates into three industrially relevant bacteria by recapitulating Buffer Gate activity simply by leveraging the conserved homologous Csr Network in each species. Lastly, as a demonstration of downstream application, we apply our system to a proof-of-concept synthetic mevalonate pathway. Using our engineered constructs, we optimize mevalonate production in E. coli resulting in a three-fold increase in production relative to a transcriptionally controlled mevalonate pathway. As a whole, we establish a novel approach to rewire post-transcriptional regulatory networks for complex bacterial computation that can be utilized for efficient bioproduction in engineered microbes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.30.560307">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.30.560307" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.30.560307">
        <p class="paperTitle">Multichannel bioelectronic sensing using engineered Escherichia coli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.30.560307" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.30.560307" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, X.; Ajo-Franklin, C.</p>
        <p class="info">Score: 6.0, Published: 2023-09-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.30.560307' target='https://doi.org/10.1101/2023.09.30.560307'> 10.1101/2023.09.30.560307</a></p>
        <p class="abstract">By engineering extracellular electron transfer (EET) to be dependent on an analyte, researchers have developed whole cell bioelectronic sensors that sense hazards to human and environmental health1. However, these sensors regulate a single electron transfer pathway as an electrochemical channel, limiting the sensing information to a single analyte. To increase information content, we developed a multichannel bioelectronic sensor through which different chemicals regulate distinct extracellular electron transfer pathways within a single Escherichia coli cell. One channel utilizes the flavin synthesis pathway from Bacillus subtilis2 and the other a set of cytochromes constructing the Mtr pathway from Shewanella oneidensis3. We demonstrate an arsenite responsive promoter can control the Mtr pathway through activation of cytochrome CymA expression and a cadmium responsive promoter can control the flavin synthesis pathway4,5. The redox potential of flavin-mediated EET is different from that of CymA-mediated one6. This allowed for development of a redox-potential-dependent algorithm that distinguishes variable input signals of each analyte mediated by two EET pathways in vivo. This approach enables a 2-bit binary signal readout for real-time tracking throughout the entire sensing duration. Our multichannel bioelectronic sensor was able to accurately sense and distinguish different heavy metals in Brays Bayou water samples with a response time comparable to that in clean water. This multichannel bioelectronic sensors allow for simultaneous detection of different chemicals, significantly expanding information transmission and helping to safeguard human and environmental health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560220">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560220" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560220">
        <p class="paperTitle">Accurate Identification of Motoneuron Discharges from Ultrasound Images Across the Full Muscle Cross-Section</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560220" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560220" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lubel, E.; Rohlen, R.; Grandi Sgambato, B.; Barsakcioglu, D.; Ibanez, J.; Tang, M.-x.; Farina, D.</p>
        <p class="info">Score: 5.2, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560220' target='https://doi.org/10.1101/2023.09.29.560220'> 10.1101/2023.09.29.560220</a></p>
        <p class="abstract">Objective: Non-invasive identification of motoneuron (MN) activity is commonly done using (EMG). However, surface EMG (sEMG) signals detect only superficial sources, at less than approximately 10-mm depth. Intramuscular EMG can detect deep sources, but it is limited to sources within a few mm of the detection site. Conversely, ultrasound (US) images have high spatial resolution across the whole muscle cross-section. The activity of MNs can be extracted from US images due to the movements that MN activation generates in the innervated muscle fibers. Current US-based decomposition methods can accurately identify the location and average twitch induced by MN activity. However, they cannot accurately detect MN discharge times. Methods: Here, we present a method based on the convolutive blind source separation of US images to estimate MN discharge times with high accuracy. The method was validated across 10 participants using concomitant sEMG decomposition as the ground truth. Results: 140 unique MN spike trains were identified from US images, with a rate of agreement (RoA) with sEMG decomposition of 87.4 {&#43;/-} 10.3 %. Over 50% of these MN spike trains had a RoA greater than 90%. Furthermore, with US, we identified additional MUs well beyond the sEMG detection volume, at up to &gt;30 mm below the skin. Conclusion: The proposed method can identify discharges of MNs innervating muscle fibers in a large range of depths within the muscle from US images. Significance: The proposed methodology can non-invasively interface with the outer layers of the central nervous system innervating muscles across the full cross-section.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
