<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases
Authors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.
Score: 109.3, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557440
Genome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases
Authors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.
Score: 109.3, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557440
Genome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-01T10:37:47+00:00" />
<meta property="article:modified_time" content="2023-10-01T10:37:47+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases
Authors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.
Score: 109.3, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557440
Genome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases\nAuthors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.\nScore: 109.3, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557440\nGenome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e.",
  "keywords": [
    
  ],
  "articleBody": " Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases\nAuthors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.\nScore: 109.3, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557440\nGenome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e.g. CRISPR-Cas) to permit the installation of a range of edits including substitutions, insertions, and deletions. Click editors (CEs) leverage the \"click\"-like bioconjugation ability of HUH endonucleases (HUHes) with single stranded DNA substrates to covalently tether \"click DNA\" (clkDNA) templates encoding user-specifiable edits at targeted genomic loci. Through iterative optimization of the modular components of CEs (DNA polymerase and HUHe orthologs, architectural modifications, etc.) and their clkDNAs (template configurations, repair evading substitutions, etc.), we demonstrate the ability to install precise genome edits with minimal indels and no unwanted byproduct insertions. Since clkDNAs can be ordered as simple DNA oligonucleotides for cents per base, it is possible to screen many different clkDNA parameters rapidly and inexpensively to maximize edit efficiency. Together, click editing is a precise and highly versatile platform for modifying genomes with a simple workflow and broad utility across diverse biological applications.\nHaloTag display enables quantitative single-particle characterization and functionalization of engineered extracellular vesicles\nAuthors: Mitrut, R. E.; Stranford, D. M.; Chan, J. M.; Bailey, M. D.; Luo, M.; Meade, T. J.; Wang, M.; Leonard, J. N.\nScore: 10.3, Published: 2023-09-27 DOI: 10.1101/2023.09.25.559433\nExtracellular vesicles (EVs) play key roles in diverse biological processes, transport biomolecules between cells, and have been engineered for therapeutic applications. A useful EV bioengineering strategy is to express engineered proteins on the EV surface to confer targeting, bioactivity, and other properties. Measuring how incorporation varies across a population of EVs is important for characterizing such materials and understanding their function, yet it remains challenging to quantitatively characterize the absolute number of engineered proteins incorporated at single-EV resolution. To address these needs, we developed a HaloTag-based characterization platform in which dyes or other synthetic species can be covalently and stoichiometrically attached to engineered proteins on the EV surface. To evaluate this system, we employed several orthogonal quantification methods, including flow cytometry and fluorescence microscopy, and found that HaloTag-mediated quantification is generally robust across EV analysis methods. We compared HaloTag-labeling to antibody-labeling of EVs using single vesicle flow cytometry, enabling us to quantify the substantial degree to which antibody labeling can underestimate the absolute number of proteins present on an EV. Finally, we demonstrate use of HaloTag to compare between protein designs for EV bioengineering. Overall, the HaloTag system is a useful EV characterization tool which complements and expands existing methods.\nOpenCell: A Low-cost, Open-Source, 3-in-1 device for DNA Extraction\nAuthors: Gupta, A.; Yu, J.; Challita, E. J.; Standeven, J.; Bhamla, M. S.\nScore: 8.1, Published: 2023-09-25 DOI: 10.1101/2023.09.18.558349\nHigh-cost DNA extraction procedures pose significant challenges for budget-constrained laboratories. To address this, we introduce OpenCell, an economical, open-source, 3-in-1 laboratory device that combines the functionalities of a bead homogenizer, a microcentrifuge, and a vortex mixer. OpenCell utilizes modular attachments that magnetically connect to a central rotating brushless motor. This motor couples to an epicyclic gearing mechanism, enabling efficient bead homogenization, vortex mixing, and centrifugation within one compact unit. OpenCells design incorporates multiple redundant safety features, ensuring both the devices and operators safety. Additional features such as RPM measurement, programmable timers, battery operation, and optional speed control make OpenCell a reliable and reproducible laboratory instrument. In our study, OpenCell successfully isolated DNA from Spinacia oleracea (spinach), with an average yield of 2.3 g and an A260/A280 ratio of 1.77, demonstrating its effectiveness for downstream applications such as Polymerase Chain Reaction (PCR) amplification. With its compact size (20 cm x 28 cm x 6.7 cm) and lightweight design (0.8 kg), comparable to the size and weight of a laptop, OpenCell is portable, making it an attractive component of a lab-in-a-backpack for resource-constrained environments in low-and-middle-income countries and synthetic biology in remote field stations. Leveraging the accessibility of 3D printing and off-the-shelf components, OpenCell can be manufactured and assembled at a low unit cost of less than $50, providing an affordable alternative to expensive laboratory equipment costing over $4000. OpenCell aims to overcome the barriers to entry in synthetic biology research and contribute to the growing collection of frugal and open hardware.\nEngineering antigen-specific tolerance to an artificial protein hydrogel\nAuthors: Rapp, P. B.; Baccile, J. A.; Galimidi, R. P.; Vielmetter, J.\nScore: 8.1, Published: 2023-09-23 DOI: 10.1101/2023.09.23.559120\nArtificial protein hydrogels are an emerging class of biomaterials with numerous prospective applications in tissue engineering and regenerative medicine. These materials are likely to be immunogenic due their frequent incorporation of novel amino acid sequence domains, which often serve a functional role within the material itself. We engineered injectable \"self\" and \"non-self\" artificial protein hydrogels which were predicted to have divergent immune outcomes in vivo on the basis of their primary amino acid sequence. Following implantation in mouse, the non-self gels raised significantly higher anti-gel antibody titers than the corresponding self gels. Prophylactic administration of a fusion antibody targeting the non-self hydrogel epitopes to DEC-205, an endocytic receptor involved in Treg induction, fully suppressed the elevated antibody titer against the non-self gels. These results suggest that the clinical immune response to artificial protein biomaterials, including those that contain highly antigenic sequence domains, can be tuned through the induction of antigen-specific tolerance.\nPredicting Long and Short Duration Beta Bursts from Subthalamic Nucleus Local Field Potential Activity in Parkinson's Disease\nAuthors: Abdi-Sargezeh, B.; Shirani, S.; Sharma, A.; Starr, P.; Little, S.; Oswal, A.\nScore: 6.9, Published: 2023-09-23 DOI: 10.1101/2023.09.22.558764\nNeural activities within the beta frequency range (13-30 Hz) are not stationary, but occur in transient packets known as beta bursts. Parkinsons disease (PD) is characterized by the occurrence of beta bursts of increased duration and amplitude within the cortico-basal ganglia network. The pathophysiological importance of beta bursts is exemplified by the fact that they serve as a clinically useful feedback signal in beta amplitude triggered adaptive Deep Brain Stimulation (aDBS). Prolonged duration beta bursts are closely associated with motor impairments in PD, whilst bursts of shorter duration may have a physiological role. Consequently, we aimed to develop a deep learning-based pipeline capable of predicting long (\u003e 150ms) and short (\u003c 150ms) duration beta bursts from subthalamic nucleus local field potential (LFP) recordings. Our approach achieved promising accuracy values of 87% and 85.2% in two patients implanted with a DBS device that was capable of long-term wireless LFP sensing. Our findings highlight the feasibility of prolonged beta burst prediction and could inform the development of a new type of intelligent DBS approach with the capability of delivering stimulation only during the occurrence of prolonged bursts.\nA general approach for selection of epitope-directed binders to proteins\nAuthors: zhou, j.; Le, C. Q.; Zhang, Y.; Wells, J. A.\nScore: 7.5, Published: 2023-09-23 DOI: 10.1101/2022.10.24.513434\nDirecting antibodies to a particular epitope among many possible on a target protein is a significant challenge. Here we present a simple and general method for epitope-directed selection (EDS) using a differential phage selection strategy. This involves engineering the protein of interest (POI) with the epitope of interest (EOI) mutated using a systematic bioinformatics algorithm to guide the local design of an EOI decoy variant. Using several alternating rounds of negative selection with the EOI decoy variant followed by positive selection on the wild-type (WT) POI, we were able to identify highly specific and potent antibodies to five different EOI antigens that bind and functionally block known sites of proteolysis. Among these we developed highly specific antibodies that target the proteolytic site on the CUB domain containing protein 1 (CDCP1) to prevent its proteolysis allowing us to study the cellular maturation of this event that triggers malignancy. We generated antibodies that recognize the junction between the pro and catalytic domains for four different matrix metalloproteases (MMPs), such as MMP1, MMP3, and MMP9, that selectively block activation of each of these enzymes and impairs cell migration. We targeted a proteolytic epitope on the cell surface receptor, EPH Receptor A2, that is known to transform it from a tumor suppressor to an oncoprotein. We believe the EDS method greatly facilitates the generation antibodies to specific EOIs on a wide range of proteins and enzymes for broad therapeutic and diagnostic applications. SignificanceWe have developed a highly efficient platform to facilitate the directed selection in vitro of antibodies to a wide range of functional epitopes on proteins. This method uses a bioinformatic program to guide mutations in the local site of interest to create a decoy antigen that can effectively remove antibodies not binding the site of interest by negative selection, followed by positive selection with the WT antigen to identify antibodies to the epitope of interest. We demonstrate the generality and versatility of this method by successfully producing functional antibodies to block specific proteolytically sensitive epitopes on five different proteins including enzymes important in cancer. The epitope-directed selection (EDS) approach greatly facilitates the identification of binders to specific sites of interest on proteins to probe function and as potential immunotherapeutics.\nSingle-cell Herpes Simplex Virus type-1 infection of neurons using drop-based microfluidics reveals heterogeneous replication kinetics\nAuthors: Fredrikson, J. P.; Domanico, L. F.; Pratt, S. L.; Loveday, E. K.; Taylor, M. P.; Chang, C. B.\nScore: 9.3, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558333\nSingle-cell analyses of viral infections often reveal heterogeneity that is not detected by traditional population-level studies. This study applies drop-based microfluidics to investigate the dynamics of HSV-1 infection of neurons at the single-cell level. We used micron-scale Matrigel beads, termed microgels, to culture individual murine Superior Cervical ganglia (SCG) neurons or epithelial cells. Microgel-cultured cells are subsequently enclosed in individual media-in-oil droplets with a dual fluorescent-reporter HSV-1, enabling real-time observation of viral gene expression and replication. Infection within drops revealed that the kinetics of initial viral gene expression and replication were dependent on the inoculating dose. Notably, increasing inoculating doses led to earlier onset of viral gene expression and more frequent productive viral replication. These observations provide crucial insights into the complexity of HSV-1 infection in neurons and emphasize the importance of studying single-cell outcomes of viral infection. The innovative techniques presented here for cell culture and infection in drops provide a foundation for future virology and neurobiology investigations.\nProtein generation with evolutionary diffusion: sequence is all you need\nAuthors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.\nScore: 186.0, Published: 2023-09-12 DOI: 10.1101/2023.09.11.556673\nDeep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design. However, current state-of-the-art models generate protein structures, which limits the scope of their training data and restricts generations to a small and biased subset of protein design space. Here, we introduce a general-purpose diffusion framework, EvoDiff, that combines evolutionary-scale data with the distinct conditioning capabilities of diffusion models for controllable protein generation in sequence space. EvoDiff generates high-fidelity, diverse, and structurally-plausible proteins that cover natural sequence and functional space. Critically, EvoDiff can generate proteins inaccessible to structure-based models, such as those with disordered regions, while maintaining the ability to design scaffolds for functional structural motifs, demonstrating the universality of our sequence-based formulation. We envision that EvoDiff will expand capabilities in protein engineering beyond the structure-function paradigm toward programmable, sequence-first design.\nMulti-modal profiling of biostabilized human skin modules reveals a coordinated ecosystem response to injected mRNA-1273 COVID-19 vaccine.\nAuthors: Scholaert, M.; Peries, M.; Braun, E.; Martin, J.; Serhan, N.; Loste, A.; Bruner, A.; Basso, L.; Chaput, B.; Merle, E.; Descargues, P.; Pages, E.; Gaudenzio, N.\nScore: 2.8, Published: 2023-09-25 DOI: 10.1101/2023.09.22.558940\nThe field of vaccination is witnessing a remarkable surge in the development of innovative strategies. There is a need to develop technological platforms capable of generating human data prior to progressing to clinical trials. Here we introduce VaxSkin, a flexible solution designed for the comprehensive monitoring of the natural human skin ecosystems response to vaccines over time. Based on bioengineering to repurpose surgical resections, it allows a comprehensive analysis of the response to vaccines at both organ and single-cell levels. Upon injection of the mRNA-1273 COVID-19 vaccine, we characterized precise sequential molecular events triggered upon detection of the exogenous substance. We also found that the vaccine consistently targets DC/macrophages and mast cells, regardless of the administration route, while promoting specific cell-cell communications in surrounding immune cell subsets. Given its direct translational relevance, VaxSkin provides a multiscale vision of skin vaccination that could pave the way toward the development of new vaccination development strategies.\nSingle-cell mechanical analysis reveals viscoelastic similarities between normal and neoplastic brain cells\nAuthors: Onwudiwe, K.; Najera, J.; Holen, L.; Burchett, A. A.; Rodriguez, D.; Zarodniuk, M.; Siri, S.; Datta, M.\nScore: 2.5, Published: 2023-09-25 DOI: 10.1101/2023.09.23.559055\nUnderstanding cancer cell mechanics allows for the identification of novel disease mechanisms, diagnostic biomarkers, and targeted therapies. In this study, we utilized our previously established fluid shear stress assay to investigate and compare the viscoelastic properties of normal immortalized human astrocytes (IHAs) and invasive human glioblastoma (GBM) cells when subjected to physiological levels of shear stress that are present in the brain microenvironment. We used a parallel-flow microfluidic shear system and a camera-coupled optical microscope to expose single cells to fluid shear stress and monitor the resulting deformation in real-time, respectively. From the video-rate imaging, we fed cell deformation information from digital image correlation into a three-parameter generalized Maxwell model to quantify the nuclear and cytoplasmic viscoelastic properties of single cells. We further quantified actin cytoskeleton density and alignment in IHAs and GBM cells via immunofluorescence microscopy and image analysis techniques. Results from our study show that contrary to the behavior of many extracranial cells, normal and cancerous brain cells do not exhibit significant differences in their viscoelastic behavior. Moreover, we also found that the viscoelastic properties of the nucleus and cytoplasm as well as the actin cytoskeletal densities of both brain cell types are similar. Our work suggests that malignant GBM cells exhibit unique mechanical behaviors not seen in other cancer cell types. These results warrant future study to elucidate the distinct biophysical characteristics of the brain and reveal novel mechanical attributes of GBM and other primary brain tumors.\n",
  "wordCount" : "2416",
  "inLanguage": "en",
  "datePublished": "2023-10-01T10:37:47Z",
  "dateModified": "2023-10-01T10:37:47Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on October 1, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557440">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557440" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557440">
        <p class="paperTitle">Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557440" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557440" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.</p>
        <p class="info">Score: 109.3, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557440' target='https://doi.org/10.1101/2023.09.12.557440'> 10.1101/2023.09.12.557440</a></p>
        <p class="abstract">Genome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e.g. CRISPR-Cas) to permit the installation of a range of edits including substitutions, insertions, and deletions. Click editors (CEs) leverage the &#34;click&#34;-like bioconjugation ability of HUH endonucleases (HUHes) with single stranded DNA substrates to covalently tether &#34;click DNA&#34; (clkDNA) templates encoding user-specifiable edits at targeted genomic loci. Through iterative optimization of the modular components of CEs (DNA polymerase and HUHe orthologs, architectural modifications, etc.) and their clkDNAs (template configurations, repair evading substitutions, etc.), we demonstrate the ability to install precise genome edits with minimal indels and no unwanted byproduct insertions. Since clkDNAs can be ordered as simple DNA oligonucleotides for cents per base, it is possible to screen many different clkDNA parameters rapidly and inexpensively to maximize edit efficiency. Together, click editing is a precise and highly versatile platform for modifying genomes with a simple workflow and broad utility across diverse biological applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559433">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559433" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559433">
        <p class="paperTitle">HaloTag display enables quantitative single-particle characterization and functionalization of engineered extracellular vesicles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559433" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559433" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mitrut, R. E.; Stranford, D. M.; Chan, J. M.; Bailey, M. D.; Luo, M.; Meade, T. J.; Wang, M.; Leonard, J. N.</p>
        <p class="info">Score: 10.3, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559433' target='https://doi.org/10.1101/2023.09.25.559433'> 10.1101/2023.09.25.559433</a></p>
        <p class="abstract">Extracellular vesicles (EVs) play key roles in diverse biological processes, transport biomolecules between cells, and have been engineered for therapeutic applications. A useful EV bioengineering strategy is to express engineered proteins on the EV surface to confer targeting, bioactivity, and other properties. Measuring how incorporation varies across a population of EVs is important for characterizing such materials and understanding their function, yet it remains challenging to quantitatively characterize the absolute number of engineered proteins incorporated at single-EV resolution. To address these needs, we developed a HaloTag-based characterization platform in which dyes or other synthetic species can be covalently and stoichiometrically attached to engineered proteins on the EV surface. To evaluate this system, we employed several orthogonal quantification methods, including flow cytometry and fluorescence microscopy, and found that HaloTag-mediated quantification is generally robust across EV analysis methods. We compared HaloTag-labeling to antibody-labeling of EVs using single vesicle flow cytometry, enabling us to quantify the substantial degree to which antibody labeling can underestimate the absolute number of proteins present on an EV. Finally, we demonstrate use of HaloTag to compare between protein designs for EV bioengineering. Overall, the HaloTag system is a useful EV characterization tool which complements and expands existing methods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558349">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558349" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558349">
        <p class="paperTitle">OpenCell: A Low-cost, Open-Source, 3-in-1 device for DNA Extraction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558349" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558349" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gupta, A.; Yu, J.; Challita, E. J.; Standeven, J.; Bhamla, M. S.</p>
        <p class="info">Score: 8.1, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558349' target='https://doi.org/10.1101/2023.09.18.558349'> 10.1101/2023.09.18.558349</a></p>
        <p class="abstract">High-cost DNA extraction procedures pose significant challenges for budget-constrained laboratories. To address this, we introduce OpenCell, an economical, open-source, 3-in-1 laboratory device that combines the functionalities of a bead homogenizer, a microcentrifuge, and a vortex mixer. OpenCell utilizes modular attachments that magnetically connect to a central rotating brushless motor. This motor couples to an epicyclic gearing mechanism, enabling efficient bead homogenization, vortex mixing, and centrifugation within one compact unit. OpenCells design incorporates multiple redundant safety features, ensuring both the devices and operators safety. Additional features such as RPM measurement, programmable timers, battery operation, and optional speed control make OpenCell a reliable and reproducible laboratory instrument. In our study, OpenCell successfully isolated DNA from Spinacia oleracea (spinach), with an average yield of 2.3 g and an A260/A280 ratio of 1.77, demonstrating its effectiveness for downstream applications such as Polymerase Chain Reaction (PCR) amplification. With its compact size (20 cm x 28 cm x 6.7 cm) and lightweight design (0.8 kg), comparable to the size and weight of a laptop, OpenCell is portable, making it an attractive component of a  lab-in-a-backpack for resource-constrained environments in low-and-middle-income countries and synthetic biology in remote field stations. Leveraging the accessibility of 3D printing and off-the-shelf components, OpenCell can be manufactured and assembled at a low unit cost of less than $50, providing an affordable alternative to expensive laboratory equipment costing over $4000. OpenCell aims to overcome the barriers to entry in synthetic biology research and contribute to the growing collection of frugal and open hardware.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.23.559120">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.23.559120" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.23.559120">
        <p class="paperTitle">Engineering antigen-specific tolerance to an artificial protein hydrogel</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.23.559120" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.23.559120" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rapp, P. B.; Baccile, J. A.; Galimidi, R. P.; Vielmetter, J.</p>
        <p class="info">Score: 8.1, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.23.559120' target='https://doi.org/10.1101/2023.09.23.559120'> 10.1101/2023.09.23.559120</a></p>
        <p class="abstract">Artificial protein hydrogels are an emerging class of biomaterials with numerous prospective applications in tissue engineering and regenerative medicine. These materials are likely to be immunogenic due their frequent incorporation of novel amino acid sequence domains, which often serve a functional role within the material itself. We engineered injectable &#34;self&#34; and &#34;non-self&#34; artificial protein hydrogels which were predicted to have divergent immune outcomes in vivo on the basis of their primary amino acid sequence. Following implantation in mouse, the non-self gels raised significantly higher anti-gel antibody titers than the corresponding self gels. Prophylactic administration of a fusion antibody targeting the non-self hydrogel epitopes to DEC-205, an endocytic receptor involved in Treg induction, fully suppressed the elevated antibody titer against the non-self gels. These results suggest that the clinical immune response to artificial protein biomaterials, including those that contain highly antigenic sequence domains, can be tuned through the induction of antigen-specific tolerance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.558764">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.558764" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.558764">
        <p class="paperTitle">Predicting Long and Short Duration Beta Bursts from Subthalamic Nucleus Local Field Potential Activity in Parkinson&#39;s Disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.558764" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.558764" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abdi-Sargezeh, B.; Shirani, S.; Sharma, A.; Starr, P.; Little, S.; Oswal, A.</p>
        <p class="info">Score: 6.9, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.558764' target='https://doi.org/10.1101/2023.09.22.558764'> 10.1101/2023.09.22.558764</a></p>
        <p class="abstract">Neural activities within the beta frequency range (13-30 Hz) are not stationary, but occur in transient packets known as beta bursts. Parkinsons disease (PD) is characterized by the occurrence of beta bursts of increased duration and amplitude within the cortico-basal ganglia network. The pathophysiological importance of beta bursts is exemplified by the fact that they serve as a clinically useful feedback signal in beta amplitude triggered adaptive Deep Brain Stimulation (aDBS). Prolonged duration beta bursts are closely associated with motor impairments in PD, whilst bursts of shorter duration may have a physiological role. Consequently, we aimed to develop a deep learning-based pipeline capable of predicting long (&gt; 150ms) and short (&lt; 150ms) duration beta bursts from subthalamic nucleus local field potential (LFP) recordings. Our approach achieved promising accuracy values of 87% and 85.2% in two patients implanted with a DBS device that was capable of long-term wireless LFP sensing. Our findings highlight the feasibility of prolonged beta burst prediction and could inform the development of a new type of intelligent DBS approach with the capability of delivering stimulation only during the occurrence of prolonged bursts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.10.24.513434">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.10.24.513434" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.10.24.513434">
        <p class="paperTitle">A general approach for selection of epitope-directed binders to proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.10.24.513434" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.10.24.513434" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: zhou, j.; Le, C. Q.; Zhang, Y.; Wells, J. A.</p>
        <p class="info">Score: 7.5, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.10.24.513434' target='https://doi.org/10.1101/2022.10.24.513434'> 10.1101/2022.10.24.513434</a></p>
        <p class="abstract">Directing antibodies to a particular epitope among many possible on a target protein is a significant challenge. Here we present a simple and general method for epitope-directed selection (EDS) using a differential phage selection strategy. This involves engineering the protein of interest (POI) with the epitope of interest (EOI) mutated using a systematic bioinformatics algorithm to guide the local design of an EOI decoy variant. Using several alternating rounds of negative selection with the EOI decoy variant followed by positive selection on the wild-type (WT) POI, we were able to identify highly specific and potent antibodies to five different EOI antigens that bind and functionally block known sites of proteolysis. Among these we developed highly specific antibodies that target the proteolytic site on the CUB domain containing protein 1 (CDCP1) to prevent its proteolysis allowing us to study the cellular maturation of this event that triggers malignancy. We generated antibodies that recognize the junction between the pro and catalytic domains for four different matrix metalloproteases (MMPs), such as MMP1, MMP3, and MMP9, that selectively block activation of each of these enzymes and impairs cell migration. We targeted a proteolytic epitope on the cell surface receptor, EPH Receptor A2, that is known to transform it from a tumor suppressor to an oncoprotein. We believe the EDS method greatly facilitates the generation antibodies to specific EOIs on a wide range of proteins and enzymes for broad therapeutic and diagnostic applications.

SignificanceWe have developed a highly efficient platform to facilitate the directed selection in vitro of antibodies to a wide range of functional epitopes on proteins. This method uses a bioinformatic program to guide mutations in the local site of interest to create a decoy antigen that can effectively remove antibodies not binding the site of interest by negative selection, followed by positive selection with the WT antigen to identify antibodies to the epitope of interest. We demonstrate the generality and versatility of this method by successfully producing functional antibodies to block specific proteolytically sensitive epitopes on five different proteins including enzymes important in cancer. The epitope-directed selection (EDS) approach greatly facilitates the identification of binders to specific sites of interest on proteins to probe function and as potential immunotherapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558333">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558333" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558333">
        <p class="paperTitle">Single-cell Herpes Simplex Virus type-1 infection of neurons using drop-based microfluidics reveals heterogeneous replication kinetics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558333" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558333" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fredrikson, J. P.; Domanico, L. F.; Pratt, S. L.; Loveday, E. K.; Taylor, M. P.; Chang, C. B.</p>
        <p class="info">Score: 9.3, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558333' target='https://doi.org/10.1101/2023.09.18.558333'> 10.1101/2023.09.18.558333</a></p>
        <p class="abstract">Single-cell analyses of viral infections often reveal heterogeneity that is not detected by traditional population-level studies. This study applies drop-based microfluidics to investigate the dynamics of HSV-1 infection of neurons at the single-cell level. We used micron-scale Matrigel beads, termed microgels, to culture individual murine Superior Cervical ganglia (SCG) neurons or epithelial cells. Microgel-cultured cells are subsequently enclosed in individual media-in-oil droplets with a dual fluorescent-reporter HSV-1, enabling real-time observation of viral gene expression and replication. Infection within drops revealed that the kinetics of initial viral gene expression and replication were dependent on the inoculating dose. Notably, increasing inoculating doses led to earlier onset of viral gene expression and more frequent productive viral replication. These observations provide crucial insights into the complexity of HSV-1 infection in neurons and emphasize the importance of studying single-cell outcomes of viral infection. The innovative techniques presented here for cell culture and infection in drops provide a foundation for future virology and neurobiology investigations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.556673">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.556673" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.556673">
        <p class="paperTitle">Protein generation with evolutionary diffusion: sequence is all you need</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.556673" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.556673" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.</p>
        <p class="info">Score: 186.0, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.556673' target='https://doi.org/10.1101/2023.09.11.556673'> 10.1101/2023.09.11.556673</a></p>
        <p class="abstract">Deep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design. However, current state-of-the-art models generate protein structures, which limits the scope of their training data and restricts generations to a small and biased subset of protein design space. Here, we introduce a general-purpose diffusion framework, EvoDiff, that combines evolutionary-scale data with the distinct conditioning capabilities of diffusion models for controllable protein generation in sequence space. EvoDiff generates high-fidelity, diverse, and structurally-plausible proteins that cover natural sequence and functional space. Critically, EvoDiff can generate proteins inaccessible to structure-based models, such as those with disordered regions, while maintaining the ability to design scaffolds for functional structural motifs, demonstrating the universality of our sequence-based formulation. We envision that EvoDiff will expand capabilities in protein engineering beyond the structure-function paradigm toward programmable, sequence-first design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.558940">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.558940" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.558940">
        <p class="paperTitle">Multi-modal profiling of biostabilized human skin modules reveals a coordinated ecosystem response to injected mRNA-1273 COVID-19 vaccine.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.558940" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.558940" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Scholaert, M.; Peries, M.; Braun, E.; Martin, J.; Serhan, N.; Loste, A.; Bruner, A.; Basso, L.; Chaput, B.; Merle, E.; Descargues, P.; Pages, E.; Gaudenzio, N.</p>
        <p class="info">Score: 2.8, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.558940' target='https://doi.org/10.1101/2023.09.22.558940'> 10.1101/2023.09.22.558940</a></p>
        <p class="abstract">The field of vaccination is witnessing a remarkable surge in the development of innovative strategies. There is a need to develop technological platforms capable of generating human data prior to progressing to clinical trials. Here we introduce VaxSkin, a flexible solution designed for the comprehensive monitoring of the natural human skin ecosystems response to vaccines over time. Based on bioengineering to repurpose surgical resections, it allows a comprehensive analysis of the response to vaccines at both organ and single-cell levels. Upon injection of the mRNA-1273 COVID-19 vaccine, we characterized precise sequential molecular events triggered upon detection of the exogenous substance. We also found that the vaccine consistently targets DC/macrophages and mast cells, regardless of the administration route, while promoting specific cell-cell communications in surrounding immune cell subsets. Given its direct translational relevance, VaxSkin provides a multiscale vision of skin vaccination that could pave the way toward the development of new vaccination development strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.23.559055">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.23.559055" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.23.559055">
        <p class="paperTitle">Single-cell mechanical analysis reveals viscoelastic similarities between normal and neoplastic brain cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.23.559055" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.23.559055" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Onwudiwe, K.; Najera, J.; Holen, L.; Burchett, A. A.; Rodriguez, D.; Zarodniuk, M.; Siri, S.; Datta, M.</p>
        <p class="info">Score: 2.5, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.23.559055' target='https://doi.org/10.1101/2023.09.23.559055'> 10.1101/2023.09.23.559055</a></p>
        <p class="abstract">Understanding cancer cell mechanics allows for the identification of novel disease mechanisms, diagnostic biomarkers, and targeted therapies. In this study, we utilized our previously established fluid shear stress assay to investigate and compare the viscoelastic properties of normal immortalized human astrocytes (IHAs) and invasive human glioblastoma (GBM) cells when subjected to physiological levels of shear stress that are present in the brain microenvironment. We used a parallel-flow microfluidic shear system and a camera-coupled optical microscope to expose single cells to fluid shear stress and monitor the resulting deformation in real-time, respectively. From the video-rate imaging, we fed cell deformation information from digital image correlation into a three-parameter generalized Maxwell model to quantify the nuclear and cytoplasmic viscoelastic properties of single cells. We further quantified actin cytoskeleton density and alignment in IHAs and GBM cells via immunofluorescence microscopy and image analysis techniques. Results from our study show that contrary to the behavior of many extracranial cells, normal and cancerous brain cells do not exhibit significant differences in their viscoelastic behavior. Moreover, we also found that the viscoelastic properties of the nucleus and cytoplasm as well as the actin cytoskeletal densities of both brain cell types are similar. Our work suggests that malignant GBM cells exhibit unique mechanical behaviors not seen in other cancer cell types. These results warrant future study to elucidate the distinct biophysical characteristics of the brain and reveal novel mechanical attributes of GBM and other primary brain tumors.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
