<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Direct 3D-bioprinting of hiPSC-derived cardiomyocytes to generate functional cardiac tissues
Authors: Esser, T. U.; Anspach, A.; Muenzebrock, K. A.; Kah, D.; Schruefer, S.; Schenk, J.; Heinze, K. G.; Schubert, D. W.; Fabry, B.; Engel, F. B.
Score: 44.8, Published: 2023-06-12 DOI: 10.1101/2023.06.12.544557
3D-bioprinting is a promising technology to produce human tissues as drug screening tool or for organ repair. However, direct printing of living cells has proven difficult. Here, we present a method to directly 3D-bioprint human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes embedded in a collagen-hyaluronic acid ink generating centimeter-sized functional ring- and ventricle-shaped cardiac tissues in an accurate and reproducible manner.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Direct 3D-bioprinting of hiPSC-derived cardiomyocytes to generate functional cardiac tissues
Authors: Esser, T. U.; Anspach, A.; Muenzebrock, K. A.; Kah, D.; Schruefer, S.; Schenk, J.; Heinze, K. G.; Schubert, D. W.; Fabry, B.; Engel, F. B.
Score: 44.8, Published: 2023-06-12 DOI: 10.1101/2023.06.12.544557
3D-bioprinting is a promising technology to produce human tissues as drug screening tool or for organ repair. However, direct printing of living cells has proven difficult. Here, we present a method to directly 3D-bioprint human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes embedded in a collagen-hyaluronic acid ink generating centimeter-sized functional ring- and ventricle-shaped cardiac tissues in an accurate and reproducible manner." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-02T10:39:32+00:00" />
<meta property="article:modified_time" content="2023-07-02T10:39:32+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Direct 3D-bioprinting of hiPSC-derived cardiomyocytes to generate functional cardiac tissues
Authors: Esser, T. U.; Anspach, A.; Muenzebrock, K. A.; Kah, D.; Schruefer, S.; Schenk, J.; Heinze, K. G.; Schubert, D. W.; Fabry, B.; Engel, F. B.
Score: 44.8, Published: 2023-06-12 DOI: 10.1101/2023.06.12.544557
3D-bioprinting is a promising technology to produce human tissues as drug screening tool or for organ repair. However, direct printing of living cells has proven difficult. Here, we present a method to directly 3D-bioprint human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes embedded in a collagen-hyaluronic acid ink generating centimeter-sized functional ring- and ventricle-shaped cardiac tissues in an accurate and reproducible manner."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Direct 3D-bioprinting of hiPSC-derived cardiomyocytes to generate functional cardiac tissues\nAuthors: Esser, T. U.; Anspach, A.; Muenzebrock, K. A.; Kah, D.; Schruefer, S.; Schenk, J.; Heinze, K. G.; Schubert, D. W.; Fabry, B.; Engel, F. B.\nScore: 44.8, Published: 2023-06-12 DOI: 10.1101/2023.06.12.544557\n3D-bioprinting is a promising technology to produce human tissues as drug screening tool or for organ repair. However, direct printing of living cells has proven difficult. Here, we present a method to directly 3D-bioprint human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes embedded in a collagen-hyaluronic acid ink generating centimeter-sized functional ring- and ventricle-shaped cardiac tissues in an accurate and reproducible manner.",
  "keywords": [
    
  ],
  "articleBody": " Direct 3D-bioprinting of hiPSC-derived cardiomyocytes to generate functional cardiac tissues\nAuthors: Esser, T. U.; Anspach, A.; Muenzebrock, K. A.; Kah, D.; Schruefer, S.; Schenk, J.; Heinze, K. G.; Schubert, D. W.; Fabry, B.; Engel, F. B.\nScore: 44.8, Published: 2023-06-12 DOI: 10.1101/2023.06.12.544557\n3D-bioprinting is a promising technology to produce human tissues as drug screening tool or for organ repair. However, direct printing of living cells has proven difficult. Here, we present a method to directly 3D-bioprint human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes embedded in a collagen-hyaluronic acid ink generating centimeter-sized functional ring- and ventricle-shaped cardiac tissues in an accurate and reproducible manner. The printed tissues contained hiPSC-derived cardiomyocytes with well-organized sarcomeres and exhibited spontaneous and regular contractions, which persisted for several months and were able to contract against passive resistance. Importantly, beating frequencies of the printed cardiac tissues could be modulated by pharmacological stimulation. This approach opens up new possibilities for generating complex functional cardiac tissues as models for advanced drug screening or as tissue grafts for organ repair or replacement.\nProtein design using structure-based residue preferences\nAuthors: Ding, D.; Shaw, A. Y.; Sinai, S.; Rollins, N. J.; Prywes, N.; Savage, D.; Laub, M. T.; Marks, D. S.\nScore: 48.5, Published: 2023-06-14 DOI: 10.1101/2022.10.31.514613\nRecent developments in protein design have adapted large neural networks with up to 100s of millions of parameters to learn complex sequence-function mappings. However, it is unclear which dependencies between residues are critical for determining protein function, and a better empirical understanding could enable high quality models that are also more data- and resource-efficient. Here, we observe that the per residue amino acid preferences - without considering interactions between mutations are sufficient to explain much, and sometimes virtually all of the combinatorial mutation effects across 7 datasets (R2 [~] 78-98%), including one generated here. These preference parameters (20*N, where N is the number of mutated residues) can be learned from as few as [~]5*20*N observations to predict a much larger number (potentially up to 20N) of combinatorial variant effects with high accuracy (Pearson r \u003e 0.8). We hypothesized that the local structural dependencies surrounding a residue could be sufficient to learn these required mutation preferences, and developed an unsupervised design approach, which we term CoVES for Combinatorial Variant Effects from Structure. We show that CoVES outperforms not just model free sampling approaches but also complicated, high-capacity autoregressive neural networks in generating functional and diverse sequence variants for two example proteins. This simple, biologically-rooted model can be an effective alternative to high-capacity, out of domain models for the design of functional proteins.\nDe Novo Generation and Prioritization of Target-Binding Peptide Motifs from Sequence Alone\nAuthors: Bhat, S.; Palepu, K.; Yudistyra, V.; Hong, L.; Kavirayuni, V. S.; Chen, T.; Zhao, L.; Wang, T.; Vincoff, S.; Chatterjee, P.\nScore: 21.2, Published: 2023-06-28 DOI: 10.1101/2023.06.26.546591\nDesigning binders to target undruggable proteins presents a formidable challenge in drug discovery, requiring innovative approaches to overcome the lack of putative binding sites. Recently, generative models have been trained to design binding proteins from the three-dimensional structure of a target protein alone, but thus exclude design to disordered or conformationally unstable targets. In this work, we provide a generalizable algorithmic framework to design short, target-binding peptide motifs, requiring only the amino acid sequence of the target protein. To do this, we propose a process to generate naturalistic peptide candidates through Gaussian perturbation of the peptidic latent space of the state-of-the-art ESM-2 protein language model, and subsequently screen these de novo linear sequences for target-selective interaction activity via a CLIP-based contrastive learning architecture. By integrating these generative and discriminative steps, we create a Peptide Prioritization via CLIP (PepPrCLIP) pipeline and validate highly-ranked, target-specific peptide motifs experimentally via fusion to E3 ubiquitin ligase domains, demonstrating functionally potent degradation of conventionally undruggable targets in vitro. Overall, our design strategy provides a modular toolkit for designing short binding motifs to any target protein without the reliance on stable and ordered tertiary structure, enabling generation of programmable modulators to undruggable and disordered proteins such as transcription factors and fusion oncoproteins.\nOpenSIM: open source microscope add-on for structured illumination microscopy\nAuthors: Hannebelle, M. T. M.; Raeth, E.; Leitao, S.; Lukes, T.; Pospisil, J.; Toniolo, C.; Venzin, O. F.; Chrisnandy, A.; Swain, P. P.; Ronceray, N.; Lutolf, M. P.; Oates, A. C.; Hagen, G. M.; Lasser, T.; Radenovic, A.; Mckinney, J. D.; Fantner, G. E.\nScore: 17.4, Published: 2023-06-19 DOI: 10.1101/2023.06.16.545316\nSuper-resolution techniques expand the abilities of researchers who have the knowledge and resources to either build or purchase a system. This excludes the part of the research community without these capabilities. Here we introduce the openSIM add-on to upgrade existing optical microscopes to Structured Illumination super-resolution Microscopes (SIM). The openSIM is an open-hardware system, designed and documented to be easily duplicated by other laboratories, making super-resolution modality accessible to facilitate innovative research. The add-on approach gives a performance improvement for pre-existing lab equipment without the need to build a completely new system.\nHierarchical design of pseudosymmetric protein nanoparticles\nAuthors: Dowling, Q.; Park, Y.-J.; Gerstenmaier, N.; Yang, E.; Wargacki, A.; Hsia, Y.; Fries, C.; Ravichandran, R.; Walkey, C.; Burrell, A.; Veesler, D.; Baker, D.; King, N.\nScore: 48.0, Published: 2023-06-18 DOI: 10.1101/2023.06.16.545393\nDiscrete protein assemblies ranging from hundreds of kilodaltons to hundreds of megadaltons in size are a ubiquitous feature of biological systems and perform highly specialized functions 1-3. Despite remarkable recent progress in accurately designing new self-assembling proteins, the size and complexity of these assemblies has been limited by a reliance on strict symmetry 4,5. Inspired by the pseudosymmetry observed in bacterial microcompartments and viral capsids, we developed a hierarchical computational method for designing large pseudosymmetric self-assembling protein nanomaterials. We computationally designed pseudosymmetric heterooligomeric components and used them to create discrete, cage-like protein assemblies with icosahedral symmetry containing 240, 540, and 960 subunits. At 49, 71, and 96 nm diameter, these nanoparticles are the largest bounded computationally designed protein assemblies generated to date. More broadly, by moving beyond strict symmetry, our work represents an important step towards the accurate design of arbitrary self-assembling nanoscale protein objects.\nA human primary airway microphysiological system infected with SARS-CoV-2 distinguishes the treatment efficacy between nirmatrelvir and repurposed compounds fluvoxamine and amodiaquine\nAuthors: Lopez Quezada, L.; Mba Medie, F.; Gabriel, E. P.; Luu, R. J.; Rubio, L. D.; Mulhern, T. J.; Borenstein, J. T.; Fisher, C. R.; Gard, A. L.\nScore: 7.3, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546790\nThe COVID-19 pandemic necessitated a rapid mobilization of resources toward the development of safe and efficacious vaccines and therapeutics. Finding effective treatments to stem the wave of infected individuals needing hospitalization and reduce the risk of adverse events was paramount. For scientists and healthcare professionals addressing this challenge, the need to rapidly identify medical countermeasures became urgent, and many compounds in clinical use for other indications were repurposed for COVID-19 clinical trials after preliminary preclinical data demonstrated antiviral activity against SARS-CoV-2. Two repurposed compounds, fluvoxamine and amodiaquine, showed efficacy in reducing SARS-CoV-2 viral loads in preclinical experiments, but ultimately failed in clinical trials, highlighting the need for improved predictive preclinical tools that can be rapidly deployed for events such as pandemic emerging infectious diseases. The PREDICT96-ALI platform is a high-throughput, high-fidelity microphysiological system (MPS) that recapitulates primary human tracheobronchial tissue and supports highly robust and reproducible viral titers of SARS-CoV-2 variants Delta and Omicron. When amodiaquine and fluvoxamine were tested in PREDICT96-ALI, neither compound demonstrated an antiviral response, consistent with clinical outcomes and in contrast with prior reports assessing the efficacy of these compounds in other human cell-based in vitro platforms. These results highlight the unique prognostic capability of the PREDICT96-ALI proximal airway MPS to assess the potential antiviral response of lead compounds.\nEngineering in vitro models of cystic fibrosis lung disease using neutrophil extracellular trap inspired biomaterials\nAuthors: Boboltz, A.; Yang, S.; Duncan, G.\nScore: 5.5, Published: 2023-06-28 DOI: 10.1101/2023.06.26.546583\nCystic fibrosis (CF) is a muco-obstructive lung disease where inflammatory responses due to chronic infection result in the accumulation of neutrophil extracellular traps (NETs) in the airways. NETs are web-like complexes comprised mainly of decondensed chromatin that function to capture and kill bacteria. Prior studies have established excess release of NETs in CF airways increases viscoelasticity of mucus secretions and reduces mucociliary clearance. Despite the pivotal role of NETs in CF disease pathogenesis, current in vitro models of this disease do not account for their contribution. Motivated by this, we developed a new approach to study the pathobiological effects of NETs in CF by combining synthetic NET-like biomaterials, composed of DNA and histones, with an in vitro human airway epithelial cell culture model. To determine the impact of synthetic NETs on airway clearance function, we incorporated synthetic NETs into mucin hydrogels and cell culture derived airway mucus to assess their rheological and transport properties. We found that the addition of synthetic NETs significantly increases mucin hydrogel and native mucus viscoelasticity. As a result, mucociliary transport in vitro was significantly reduced with the addition of mucus containing synthetic NETs. Given the prevalence of bacterial infection in the CF lung, we also evaluated the growth of Pseudomonas aeruginosa in mucus with or without synthetic NETs. We found mucus containing synthetic NETs promoted microcolony growth and prolonged bacterial survival. Together, this work establishes a new biomaterial enabled approach to study innate immunity mediated airway dysfunction in CF.\nHigh-Resolution Additive Manufacturing of a Biodegradable Elastomer with a Low-Cost LCD 3D Printer\nAuthors: Karamzadeh, V.; Shen, M.; Ravanbakhsh, H.; Kashani, A. S.; Savoji, H.; Radisic, M.; Juncker, D.\nScore: 6.4, Published: 2023-06-15 DOI: 10.1101/2023.06.15.545079\nArtificial organs and organs-on-a-chip are of great clinical and scientific interest and have recently been made by additive manufacturing, but depend on, and benefit from, biocompatible, biodegradable, and soft materials. Poly(octamethylene maleate (anhydride) citrate (POMaC) meets these criteria and has gained popularity, and as in principle, it can be photocured and is amenable to vat-photopolymerization (VP) 3D printing, but only low-resolution structures have been produced so fa. Here, we introduce a VP-POMaC ink and demonstrate 3D printing of high resolution (80 {micro}m) and complex 3D structures using low-cost ([~]US$300) liquid-crystal display (LCD) printers. The ink includes POMaC, a diluent and porogen additive to reduce viscosity within the range of VP, and a crosslinker to speed up reaction kinetics. The mechanical properties of the cured ink were tuned to match the elastic moduli of different tissues simply by varying the porogen concentration. The biocompatibility was assessed by cell culture which yielded 80% viability and the potential for tissue engineering illustrated with a 3D printed gyroid seeded with cells. VP-POMaC and low-cost LCD printers make the additive manufacturing of high resolution, elastomeric, and biodegradable constructs widely accessible, paving the way for a myriad of applications in tissue engineering, implants, organ-on-a-chip, wearables, and soft robotics.\nSpatiotemporal Control of Genomics and Epigenomics by Ultrasound\nAuthors: Wu, Y.; Huang, Z.; Liu, Y.; Yoon, C.-W.; Sun, K.; Situ, Y.; Ho, P.; Yuan, Z.; Zhu, L.; Eyquem, J.; Zhao, Y.; Liu, T.; Kwong, G. A.; Chien, S.; Wang, Y.\nScore: 9.8, Published: 2023-06-23 DOI: 10.1101/2023.06.21.544125\nCRISPR (clustered regularly interspaced short palindromic repeats) is a revolutionary technology for genome editing. Its derived technologies such as CRISPR activation (CRISPRa) and CRISPR interference (CRISPRi) further allow transcriptional and epigenetic modulations. Focused ultrasound (FUS) can penetrate deep in biological tissues and induce mild hyperthermia in a confined region to activate heat-sensitive genes. Here we engineer a set of CRISPR(a/i) tools containing heat-sensitive genetic modules controllable by FUS for the regulation of genome and epigenome in live cells and animals. We demonstrated the capabilities of FUS-inducible CRISPRa, CRISPRi, and CRISPR (FUS-CRISPR(a/i)) to upregulate, repress, and knockout exogenous and/or endogenous genes, respectively, in different cell types. We further targeted FUS-CRISPR to telomeres in tumor cells to induce telomere disruption, inhibiting tumor growth and enhancing tumor susceptibility to killing by chimeric antigen receptor (CAR)-T cells. FUS-CRISPR-mediated telomere disruption for tumor priming combined with CAR-T therapy demonstrated synergistic therapeutic effects in xenograft mouse models. The FUS-CRISPR(a/i) toolbox allows the remote, noninvasive, and spatiotemporal control of genomic and epigenomic reprogramming in vivo, with extended applications in cancer treatment.\nRobust estimation of brain stimulation evoked responses using magnetoencephalography\nAuthors: Oswal, A.; Abdi-Sargezeh, B.; Ozkurt, T. E.; Taulu, S.; Sarangmat, N.; Green, A. L.; Litvak, V.\nScore: 4.8, Published: 2023-06-27 DOI: 10.1101/2023.06.25.546459\nMagnetoencephalography (MEG) recordings are often contaminated by interference that can exceed the amplitude of physiological brain activity by several orders of magnitude. Furthermore, activity of interference sources may leak into the activity of brain signals of interest, resulting in source estimation inaccuracies. This problem is particularly apparent when using MEG to interrogate the effects of brain stimulation on large scale cortical networks. This technical report offers two contributions. Firstly, using phantom MEG recordings we describe an approach for validating the estimation accuracy of brain stimulation evoked responses. Secondly, we propose a novel denoising method for suppressing the leakage of stimulation related signal into recorded brain activity. This approach leverages spatial and temporal domain projectors for signal arising from prespecified anatomical regions of interest. We highlight its advantages compared to the benchmark - spatiotemporal signal space separation (tSSS) - and show that it can more accurately reveal brain stimulation evoked responses.\n",
  "wordCount" : "2186",
  "inLanguage": "en",
  "datePublished": "2023-07-02T10:39:32Z",
  "dateModified": "2023-07-02T10:39:32Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on July 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.12.544557">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.12.544557" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.12.544557">
        <p class="paperTitle">Direct 3D-bioprinting of hiPSC-derived cardiomyocytes to generate functional cardiac tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.12.544557" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.12.544557" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Esser, T. U.; Anspach, A.; Muenzebrock, K. A.; Kah, D.; Schruefer, S.; Schenk, J.; Heinze, K. G.; Schubert, D. W.; Fabry, B.; Engel, F. B.</p>
        <p class="info">Score: 44.8, Published: 2023-06-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.12.544557' target='https://doi.org/10.1101/2023.06.12.544557'> 10.1101/2023.06.12.544557</a></p>
        <p class="abstract">3D-bioprinting is a promising technology to produce human tissues as drug screening tool or for organ repair. However, direct printing of living cells has proven difficult. Here, we present a method to directly 3D-bioprint human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes embedded in a collagen-hyaluronic acid ink generating centimeter-sized functional ring- and ventricle-shaped cardiac tissues in an accurate and reproducible manner. The printed tissues contained hiPSC-derived cardiomyocytes with well-organized sarcomeres and exhibited spontaneous and regular contractions, which persisted for several months and were able to contract against passive resistance. Importantly, beating frequencies of the printed cardiac tissues could be modulated by pharmacological stimulation. This approach opens up new possibilities for generating complex functional cardiac tissues as models for advanced drug screening or as tissue grafts for organ repair or replacement.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.10.31.514613">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.10.31.514613" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.10.31.514613">
        <p class="paperTitle">Protein design using structure-based residue preferences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.10.31.514613" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.10.31.514613" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ding, D.; Shaw, A. Y.; Sinai, S.; Rollins, N. J.; Prywes, N.; Savage, D.; Laub, M. T.; Marks, D. S.</p>
        <p class="info">Score: 48.5, Published: 2023-06-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.10.31.514613' target='https://doi.org/10.1101/2022.10.31.514613'> 10.1101/2022.10.31.514613</a></p>
        <p class="abstract">Recent developments in protein design have adapted large neural networks with up to 100s of millions of parameters to learn complex sequence-function mappings. However, it is unclear which dependencies between residues are critical for determining protein function, and a better empirical understanding could enable high quality models that are also more data- and resource-efficient. Here, we observe that the per residue amino acid preferences - without considering interactions between mutations are sufficient to explain much, and sometimes virtually all of the combinatorial mutation effects across 7 datasets (R2 [~] 78-98%), including one generated here. These preference parameters (20*N, where N is the number of mutated residues) can be learned from as few as [~]5*20*N observations to predict a much larger number (potentially up to 20N) of combinatorial variant effects with high accuracy (Pearson r &gt; 0.8). We hypothesized that the local structural dependencies surrounding a residue could be sufficient to learn these required mutation preferences, and developed an unsupervised design approach, which we term CoVES for  Combinatorial Variant Effects from Structure. We show that CoVES outperforms not just model free sampling approaches but also complicated, high-capacity autoregressive neural networks in generating functional and diverse sequence variants for two example proteins. This simple, biologically-rooted model can be an effective alternative to high-capacity, out of domain models for the design of functional proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546591">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546591" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546591">
        <p class="paperTitle">De Novo Generation and Prioritization of Target-Binding Peptide Motifs from Sequence Alone</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546591" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546591" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bhat, S.; Palepu, K.; Yudistyra, V.; Hong, L.; Kavirayuni, V. S.; Chen, T.; Zhao, L.; Wang, T.; Vincoff, S.; Chatterjee, P.</p>
        <p class="info">Score: 21.2, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546591' target='https://doi.org/10.1101/2023.06.26.546591'> 10.1101/2023.06.26.546591</a></p>
        <p class="abstract">Designing binders to target undruggable proteins presents a formidable challenge in drug discovery, requiring innovative approaches to overcome the lack of putative binding sites. Recently, generative models have been trained to design binding proteins from the three-dimensional structure of a target protein alone, but thus exclude design to disordered or conformationally unstable targets. In this work, we provide a generalizable algorithmic framework to design short, target-binding peptide motifs, requiring only the amino acid sequence of the target protein. To do this, we propose a process to generate naturalistic peptide candidates through Gaussian perturbation of the peptidic latent space of the state-of-the-art ESM-2 protein language model, and subsequently screen these de novo linear sequences for target-selective interaction activity via a CLIP-based contrastive learning architecture. By integrating these generative and discriminative steps, we create a Peptide Prioritization via CLIP (PepPrCLIP) pipeline and validate highly-ranked, target-specific peptide motifs experimentally via fusion to E3 ubiquitin ligase domains, demonstrating functionally potent degradation of conventionally undruggable targets in vitro. Overall, our design strategy provides a modular toolkit for designing short binding motifs to any target protein without the reliance on stable and ordered tertiary structure, enabling generation of programmable modulators to undruggable and disordered proteins such as transcription factors and fusion oncoproteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.16.545316">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.16.545316" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.16.545316">
        <p class="paperTitle">OpenSIM: open source microscope add-on for structured illumination microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.16.545316" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.16.545316" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hannebelle, M. T. M.; Raeth, E.; Leitao, S.; Lukes, T.; Pospisil, J.; Toniolo, C.; Venzin, O. F.; Chrisnandy, A.; Swain, P. P.; Ronceray, N.; Lutolf, M. P.; Oates, A. C.; Hagen, G. M.; Lasser, T.; Radenovic, A.; Mckinney, J. D.; Fantner, G. E.</p>
        <p class="info">Score: 17.4, Published: 2023-06-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.16.545316' target='https://doi.org/10.1101/2023.06.16.545316'> 10.1101/2023.06.16.545316</a></p>
        <p class="abstract">Super-resolution techniques expand the abilities of researchers who have the knowledge and resources to either build or purchase a system. This excludes the part of the research community without these capabilities. Here we introduce the openSIM add-on to upgrade existing optical microscopes to Structured Illumination super-resolution Microscopes (SIM). The openSIM is an open-hardware system, designed and documented to be easily duplicated by other laboratories, making super-resolution modality accessible to facilitate innovative research. The add-on approach gives a performance improvement for pre-existing lab equipment without the need to build a completely new system.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.16.545393">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.16.545393" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.16.545393">
        <p class="paperTitle">Hierarchical design of pseudosymmetric protein nanoparticles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.16.545393" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.16.545393" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dowling, Q.; Park, Y.-J.; Gerstenmaier, N.; Yang, E.; Wargacki, A.; Hsia, Y.; Fries, C.; Ravichandran, R.; Walkey, C.; Burrell, A.; Veesler, D.; Baker, D.; King, N.</p>
        <p class="info">Score: 48.0, Published: 2023-06-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.16.545393' target='https://doi.org/10.1101/2023.06.16.545393'> 10.1101/2023.06.16.545393</a></p>
        <p class="abstract">Discrete protein assemblies ranging from hundreds of kilodaltons to hundreds of megadaltons in size are a ubiquitous feature of biological systems and perform highly specialized functions 1-3. Despite remarkable recent progress in accurately designing new self-assembling proteins, the size and complexity of these assemblies has been limited by a reliance on strict symmetry 4,5. Inspired by the pseudosymmetry observed in bacterial microcompartments and viral capsids, we developed a hierarchical computational method for designing large pseudosymmetric self-assembling protein nanomaterials. We computationally designed pseudosymmetric heterooligomeric components and used them to create discrete, cage-like protein assemblies with icosahedral symmetry containing 240, 540, and 960 subunits. At 49, 71, and 96 nm diameter, these nanoparticles are the largest bounded computationally designed protein assemblies generated to date. More broadly, by moving beyond strict symmetry, our work represents an important step towards the accurate design of arbitrary self-assembling nanoscale protein objects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546790">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546790" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546790">
        <p class="paperTitle">A human primary airway microphysiological system infected with SARS-CoV-2 distinguishes the treatment efficacy between nirmatrelvir and repurposed compounds fluvoxamine and amodiaquine</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546790" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546790" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lopez Quezada, L.; Mba Medie, F.; Gabriel, E. P.; Luu, R. J.; Rubio, L. D.; Mulhern, T. J.; Borenstein, J. T.; Fisher, C. R.; Gard, A. L.</p>
        <p class="info">Score: 7.3, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546790' target='https://doi.org/10.1101/2023.06.27.546790'> 10.1101/2023.06.27.546790</a></p>
        <p class="abstract">The COVID-19 pandemic necessitated a rapid mobilization of resources toward the development of safe and efficacious vaccines and therapeutics. Finding effective treatments to stem the wave of infected individuals needing hospitalization and reduce the risk of adverse events was paramount. For scientists and healthcare professionals addressing this challenge, the need to rapidly identify medical countermeasures became urgent, and many compounds in clinical use for other indications were repurposed for COVID-19 clinical trials after preliminary preclinical data demonstrated antiviral activity against SARS-CoV-2. Two repurposed compounds, fluvoxamine and amodiaquine, showed efficacy in reducing SARS-CoV-2 viral loads in preclinical experiments, but ultimately failed in clinical trials, highlighting the need for improved predictive preclinical tools that can be rapidly deployed for events such as pandemic emerging infectious diseases. The PREDICT96-ALI platform is a high-throughput, high-fidelity microphysiological system (MPS) that recapitulates primary human tracheobronchial tissue and supports highly robust and reproducible viral titers of SARS-CoV-2 variants Delta and Omicron. When amodiaquine and fluvoxamine were tested in PREDICT96-ALI, neither compound demonstrated an antiviral response, consistent with clinical outcomes and in contrast with prior reports assessing the efficacy of these compounds in other human cell-based in vitro platforms. These results highlight the unique prognostic capability of the PREDICT96-ALI proximal airway MPS to assess the potential antiviral response of lead compounds.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546583">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546583" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546583">
        <p class="paperTitle">Engineering in vitro models of cystic fibrosis lung disease using neutrophil extracellular trap inspired biomaterials</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546583" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546583" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boboltz, A.; Yang, S.; Duncan, G.</p>
        <p class="info">Score: 5.5, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546583' target='https://doi.org/10.1101/2023.06.26.546583'> 10.1101/2023.06.26.546583</a></p>
        <p class="abstract">Cystic fibrosis (CF) is a muco-obstructive lung disease where inflammatory responses due to chronic infection result in the accumulation of neutrophil extracellular traps (NETs) in the airways. NETs are web-like complexes comprised mainly of decondensed chromatin that function to capture and kill bacteria. Prior studies have established excess release of NETs in CF airways increases viscoelasticity of mucus secretions and reduces mucociliary clearance. Despite the pivotal role of NETs in CF disease pathogenesis, current in vitro models of this disease do not account for their contribution. Motivated by this, we developed a new approach to study the pathobiological effects of NETs in CF by combining synthetic NET-like biomaterials, composed of DNA and histones, with an in vitro human airway epithelial cell culture model. To determine the impact of synthetic NETs on airway clearance function, we incorporated synthetic NETs into mucin hydrogels and cell culture derived airway mucus to assess their rheological and transport properties. We found that the addition of synthetic NETs significantly increases mucin hydrogel and native mucus viscoelasticity. As a result, mucociliary transport in vitro was significantly reduced with the addition of mucus containing synthetic NETs. Given the prevalence of bacterial infection in the CF lung, we also evaluated the growth of Pseudomonas aeruginosa in mucus with or without synthetic NETs. We found mucus containing synthetic NETs promoted microcolony growth and prolonged bacterial survival. Together, this work establishes a new biomaterial enabled approach to study innate immunity mediated airway dysfunction in CF.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.15.545079">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.15.545079" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.15.545079">
        <p class="paperTitle">High-Resolution Additive Manufacturing of a Biodegradable Elastomer with a Low-Cost LCD 3D Printer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.15.545079" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.15.545079" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Karamzadeh, V.; Shen, M.; Ravanbakhsh, H.; Kashani, A. S.; Savoji, H.; Radisic, M.; Juncker, D.</p>
        <p class="info">Score: 6.4, Published: 2023-06-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.15.545079' target='https://doi.org/10.1101/2023.06.15.545079'> 10.1101/2023.06.15.545079</a></p>
        <p class="abstract">Artificial organs and organs-on-a-chip are of great clinical and scientific interest and have recently been made by additive manufacturing, but depend on, and benefit from, biocompatible, biodegradable, and soft materials. Poly(octamethylene maleate (anhydride) citrate (POMaC) meets these criteria and has gained popularity, and as in principle, it can be photocured and is amenable to vat-photopolymerization (VP) 3D printing, but only low-resolution structures have been produced so fa. Here, we introduce a VP-POMaC ink and demonstrate 3D printing of high resolution (80 {micro}m) and complex 3D structures using low-cost ([~]US$300) liquid-crystal display (LCD) printers. The ink includes POMaC, a diluent and porogen additive to reduce viscosity within the range of VP, and a crosslinker to speed up reaction kinetics. The mechanical properties of the cured ink were tuned to match the elastic moduli of different tissues simply by varying the porogen concentration. The biocompatibility was assessed by cell culture which yielded 80% viability and the potential for tissue engineering illustrated with a 3D printed gyroid seeded with cells. VP-POMaC and low-cost LCD printers make the additive manufacturing of high resolution, elastomeric, and biodegradable constructs widely accessible, paving the way for a myriad of applications in tissue engineering, implants, organ-on-a-chip, wearables, and soft robotics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.544125">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.544125" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.544125">
        <p class="paperTitle">Spatiotemporal Control of Genomics and Epigenomics by Ultrasound</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.544125" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.544125" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wu, Y.; Huang, Z.; Liu, Y.; Yoon, C.-W.; Sun, K.; Situ, Y.; Ho, P.; Yuan, Z.; Zhu, L.; Eyquem, J.; Zhao, Y.; Liu, T.; Kwong, G. A.; Chien, S.; Wang, Y.</p>
        <p class="info">Score: 9.8, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.544125' target='https://doi.org/10.1101/2023.06.21.544125'> 10.1101/2023.06.21.544125</a></p>
        <p class="abstract">CRISPR (clustered regularly interspaced short palindromic repeats) is a revolutionary technology for genome editing. Its derived technologies such as CRISPR activation (CRISPRa) and CRISPR interference (CRISPRi) further allow transcriptional and epigenetic modulations. Focused ultrasound (FUS) can penetrate deep in biological tissues and induce mild hyperthermia in a confined region to activate heat-sensitive genes. Here we engineer a set of CRISPR(a/i) tools containing heat-sensitive genetic modules controllable by FUS for the regulation of genome and epigenome in live cells and animals. We demonstrated the capabilities of FUS-inducible CRISPRa, CRISPRi, and CRISPR (FUS-CRISPR(a/i)) to upregulate, repress, and knockout exogenous and/or endogenous genes, respectively, in different cell types. We further targeted FUS-CRISPR to telomeres in tumor cells to induce telomere disruption, inhibiting tumor growth and enhancing tumor susceptibility to killing by chimeric antigen receptor (CAR)-T cells. FUS-CRISPR-mediated telomere disruption for tumor priming combined with CAR-T therapy demonstrated synergistic therapeutic effects in xenograft mouse models. The FUS-CRISPR(a/i) toolbox allows the remote, noninvasive, and spatiotemporal control of genomic and epigenomic reprogramming in vivo, with extended applications in cancer treatment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.25.546459">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.25.546459" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.25.546459">
        <p class="paperTitle">Robust estimation of brain stimulation evoked responses using magnetoencephalography</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.25.546459" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.25.546459" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oswal, A.; Abdi-Sargezeh, B.; Ozkurt, T. E.; Taulu, S.; Sarangmat, N.; Green, A. L.; Litvak, V.</p>
        <p class="info">Score: 4.8, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.25.546459' target='https://doi.org/10.1101/2023.06.25.546459'> 10.1101/2023.06.25.546459</a></p>
        <p class="abstract">Magnetoencephalography (MEG) recordings are often contaminated by interference that can exceed the amplitude of physiological brain activity by several orders of magnitude. Furthermore, activity of interference sources may  leak into the activity of brain signals of interest, resulting in source estimation inaccuracies. This problem is particularly apparent when using MEG to interrogate the effects of brain stimulation on large scale cortical networks.

This technical report offers two contributions. Firstly, using phantom MEG recordings we describe an approach for validating the estimation accuracy of brain stimulation evoked responses. Secondly, we propose a novel denoising method for suppressing the leakage of stimulation related signal into recorded brain activity. This approach leverages spatial and temporal domain projectors for signal arising from prespecified anatomical regions of interest. We highlight its advantages compared to the benchmark - spatiotemporal signal space separation (tSSS) - and show that it can more accurately reveal brain stimulation evoked responses.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
