<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Direct 3D-bioprinting of hiPSC-derived cardiomyocytes to generate functional cardiac tissues
Authors: Esser, T. U.; Anspach, A.; Muenzebrock, K. A.; Kah, D.; Schruefer, S.; Schenk, J.; Heinze, K. G.; Schubert, D. W.; Fabry, B.; Engel, F. B.
Score: 78.6, Published: 2023-06-12 DOI: 10.1101/2023.06.12.544557
3D-bioprinting is a promising technology to produce human tissues as drug screening tool or for organ repair. However, direct printing of living cells has proven difficult. Here, we present a method to directly 3D-bioprint human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes embedded in a collagen-hyaluronic acid ink generating centimeter-sized functional ring- and ventricle-shaped cardiac tissues in an accurate and reproducible manner.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Direct 3D-bioprinting of hiPSC-derived cardiomyocytes to generate functional cardiac tissues
Authors: Esser, T. U.; Anspach, A.; Muenzebrock, K. A.; Kah, D.; Schruefer, S.; Schenk, J.; Heinze, K. G.; Schubert, D. W.; Fabry, B.; Engel, F. B.
Score: 78.6, Published: 2023-06-12 DOI: 10.1101/2023.06.12.544557
3D-bioprinting is a promising technology to produce human tissues as drug screening tool or for organ repair. However, direct printing of living cells has proven difficult. Here, we present a method to directly 3D-bioprint human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes embedded in a collagen-hyaluronic acid ink generating centimeter-sized functional ring- and ventricle-shaped cardiac tissues in an accurate and reproducible manner." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-05T10:42:43+00:00" />
<meta property="article:modified_time" content="2023-07-05T10:42:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Direct 3D-bioprinting of hiPSC-derived cardiomyocytes to generate functional cardiac tissues
Authors: Esser, T. U.; Anspach, A.; Muenzebrock, K. A.; Kah, D.; Schruefer, S.; Schenk, J.; Heinze, K. G.; Schubert, D. W.; Fabry, B.; Engel, F. B.
Score: 78.6, Published: 2023-06-12 DOI: 10.1101/2023.06.12.544557
3D-bioprinting is a promising technology to produce human tissues as drug screening tool or for organ repair. However, direct printing of living cells has proven difficult. Here, we present a method to directly 3D-bioprint human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes embedded in a collagen-hyaluronic acid ink generating centimeter-sized functional ring- and ventricle-shaped cardiac tissues in an accurate and reproducible manner."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Direct 3D-bioprinting of hiPSC-derived cardiomyocytes to generate functional cardiac tissues\nAuthors: Esser, T. U.; Anspach, A.; Muenzebrock, K. A.; Kah, D.; Schruefer, S.; Schenk, J.; Heinze, K. G.; Schubert, D. W.; Fabry, B.; Engel, F. B.\nScore: 78.6, Published: 2023-06-12 DOI: 10.1101/2023.06.12.544557\n3D-bioprinting is a promising technology to produce human tissues as drug screening tool or for organ repair. However, direct printing of living cells has proven difficult. Here, we present a method to directly 3D-bioprint human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes embedded in a collagen-hyaluronic acid ink generating centimeter-sized functional ring- and ventricle-shaped cardiac tissues in an accurate and reproducible manner.",
  "keywords": [
    
  ],
  "articleBody": " Direct 3D-bioprinting of hiPSC-derived cardiomyocytes to generate functional cardiac tissues\nAuthors: Esser, T. U.; Anspach, A.; Muenzebrock, K. A.; Kah, D.; Schruefer, S.; Schenk, J.; Heinze, K. G.; Schubert, D. W.; Fabry, B.; Engel, F. B.\nScore: 78.6, Published: 2023-06-12 DOI: 10.1101/2023.06.12.544557\n3D-bioprinting is a promising technology to produce human tissues as drug screening tool or for organ repair. However, direct printing of living cells has proven difficult. Here, we present a method to directly 3D-bioprint human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes embedded in a collagen-hyaluronic acid ink generating centimeter-sized functional ring- and ventricle-shaped cardiac tissues in an accurate and reproducible manner. The printed tissues contained hiPSC-derived cardiomyocytes with well-organized sarcomeres and exhibited spontaneous and regular contractions, which persisted for several months and were able to contract against passive resistance. Importantly, beating frequencies of the printed cardiac tissues could be modulated by pharmacological stimulation. This approach opens up new possibilities for generating complex functional cardiac tissues as models for advanced drug screening or as tissue grafts for organ repair or replacement.\nProtein design using structure-based residue preferences\nAuthors: Ding, D.; Shaw, A. Y.; Sinai, S.; Rollins, N. J.; Prywes, N.; Savage, D.; Laub, M. T.; Marks, D. S.\nScore: 48.5, Published: 2023-06-14 DOI: 10.1101/2022.10.31.514613\nRecent developments in protein design have adapted large neural networks with up to 100s of millions of parameters to learn complex sequence-function mappings. However, it is unclear which dependencies between residues are critical for determining protein function, and a better empirical understanding could enable high quality models that are also more data- and resource-efficient. Here, we observe that the per residue amino acid preferences - without considering interactions between mutations are sufficient to explain much, and sometimes virtually all of the combinatorial mutation effects across 7 datasets (R2 [~] 78-98%), including one generated here. These preference parameters (20*N, where N is the number of mutated residues) can be learned from as few as [~]5*20*N observations to predict a much larger number (potentially up to 20N) of combinatorial variant effects with high accuracy (Pearson r \u003e 0.8). We hypothesized that the local structural dependencies surrounding a residue could be sufficient to learn these required mutation preferences, and developed an unsupervised design approach, which we term CoVES for Combinatorial Variant Effects from Structure. We show that CoVES outperforms not just model free sampling approaches but also complicated, high-capacity autoregressive neural networks in generating functional and diverse sequence variants for two example proteins. This simple, biologically-rooted model can be an effective alternative to high-capacity, out of domain models for the design of functional proteins.\nDe Novo Generation and Prioritization of Target-Binding Peptide Motifs from Sequence Alone\nAuthors: Bhat, S.; Palepu, K.; Yudistyra, V.; Hong, L.; Kavirayuni, V. S.; Chen, T.; Zhao, L.; Wang, T.; Vincoff, S.; Chatterjee, P.\nScore: 21.2, Published: 2023-06-28 DOI: 10.1101/2023.06.26.546591\nDesigning binders to target undruggable proteins presents a formidable challenge in drug discovery, requiring innovative approaches to overcome the lack of putative binding sites. Recently, generative models have been trained to design binding proteins from the three-dimensional structure of a target protein alone, but thus exclude design to disordered or conformationally unstable targets. In this work, we provide a generalizable algorithmic framework to design short, target-binding peptide motifs, requiring only the amino acid sequence of the target protein. To do this, we propose a process to generate naturalistic peptide candidates through Gaussian perturbation of the peptidic latent space of the state-of-the-art ESM-2 protein language model, and subsequently screen these de novo linear sequences for target-selective interaction activity via a CLIP-based contrastive learning architecture. By integrating these generative and discriminative steps, we create a Peptide Prioritization via CLIP (PepPrCLIP) pipeline and validate highly-ranked, target-specific peptide motifs experimentally via fusion to E3 ubiquitin ligase domains, demonstrating functionally potent degradation of conventionally undruggable targets in vitro. Overall, our design strategy provides a modular toolkit for designing short binding motifs to any target protein without the reliance on stable and ordered tertiary structure, enabling generation of programmable modulators to undruggable and disordered proteins such as transcription factors and fusion oncoproteins.\nConfigurational entropy is an intrinsic driver of tissue structural heterogeneity\nAuthors: Srivastava, V.; Hu, J. L.; Garbe, J. C.; Veytsman, B.; Shalabi, S. F.; Yllanes, D.; Thomson, M.; LaBarge, M. A.; Huber, G.; Gartner, Z. J.\nScore: 12.9, Published: 2023-07-02 DOI: 10.1101/2023.07.01.546933\nTissues comprise ordered arrangements of cells that can be surprisingly disordered in their details. How the properties of single cells and their microenvironment contribute to the balance between order and disorder at the tissue-scale remains poorly understood. Here, we address this question using the self-organization of human mammary organoids as a model. We find that organoids behave like a dynamic structural ensemble at the steady state. We apply a maximum entropy formalism to derive the ensemble distribution from three measurable parameters - the degeneracy of structural states, interfacial energy, and tissue activity (the energy associated with positional fluctuations). We link these parameters with the molecular and microenvironmental factors that control them to precisely engineer the ensemble across multiple conditions. Our analysis reveals that the entropy associated with structural degeneracy sets a theoretical limit to tissue order and provides new insight for tissue engineering, development, and our understanding of disease progression.\nDesign, construction and optimization of formaldehyde growth biosensors with broad application in Biotechnology\nAuthors: Schann, K.; Bakker, J.; Boinot, M.; Kuschel, P.; He, H.; Nattermann, M.; Erb, T.; Bar-Even, A.; Wenk, S.\nScore: 12.3, Published: 2023-07-01 DOI: 10.1101/2023.06.29.547045\nFormaldehyde is a key metabolite in natural and synthetic one-carbon metabolism as well as an important environmental toxin with high toxicity at low concentrations. To engineer efficient formaldehyde producing enzymes and to detect formaldehyde in industrial or environmental samples, it is important to establish highly sensitive, easy to use and affordable formaldehyde detection methods. Here, we transformed the workhorse bacterium Escherichia coli into biosensors that can detect a broad range of formaldehyde concentrations. Based on natural and promiscuous formaldehyde assimilation enzymes, we designed and engineered three different E. coli strains that depend on formaldehyde assimilation for cellular growth. After in depth characterization of these biosensors, we show that the formaldehyde sensitivity can be improved through adaptive laboratory evolution or modification of metabolic branch points. The metabolic engineering strategy presented in this work allowed the creation of E. coli biosensors that can detect formaldehyde in a concentration range from ~30 M to ~13 mM. Using the most sensitive strain, we benchmarked the in vivo activities of different, widely used NAD-dependent methanol dehydrogenases, the rate-limiting enzyme in synthetic methylotrophy. We also show that the strains can grow upon external addition of formaldehyde indicating their potential use for applications beyond enzyme engineering. The formaldehyde biosensors developed in this study are fully genomic and can be used as plug and play devices for screening large enzyme libraries. Thus, they have the potential to greatly advance enzyme engineering and might even be used for environmental monitoring or analysis of industrial probes.\nOpenSIM: open source microscope add-on for structured illumination microscopy\nAuthors: Hannebelle, M. T. M.; Raeth, E.; Leitao, S.; Lukes, T.; Pospisil, J.; Toniolo, C.; Venzin, O. F.; Chrisnandy, A.; Swain, P. P.; Ronceray, N.; Lutolf, M. P.; Oates, A. C.; Hagen, G. M.; Lasser, T.; Radenovic, A.; Mckinney, J. D.; Fantner, G. E.\nScore: 18.2, Published: 2023-06-19 DOI: 10.1101/2023.06.16.545316\nSuper-resolution techniques expand the abilities of researchers who have the knowledge and resources to either build or purchase a system. This excludes the part of the research community without these capabilities. Here we introduce the openSIM add-on to upgrade existing optical microscopes to Structured Illumination super-resolution Microscopes (SIM). The openSIM is an open-hardware system, designed and documented to be easily duplicated by other laboratories, making super-resolution modality accessible to facilitate innovative research. The add-on approach gives a performance improvement for pre-existing lab equipment without the need to build a completely new system.\nA human primary airway microphysiological system infected with SARS-CoV-2 distinguishes the treatment efficacy between nirmatrelvir and repurposed compounds fluvoxamine and amodiaquine\nAuthors: Lopez Quezada, L.; Mba Medie, F.; Gabriel, E. P.; Luu, R. J.; Rubio, L. D.; Mulhern, T. J.; Borenstein, J. T.; Fisher, C. R.; Gard, A. L.\nScore: 7.3, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546790\nThe COVID-19 pandemic necessitated a rapid mobilization of resources toward the development of safe and efficacious vaccines and therapeutics. Finding effective treatments to stem the wave of infected individuals needing hospitalization and reduce the risk of adverse events was paramount. For scientists and healthcare professionals addressing this challenge, the need to rapidly identify medical countermeasures became urgent, and many compounds in clinical use for other indications were repurposed for COVID-19 clinical trials after preliminary preclinical data demonstrated antiviral activity against SARS-CoV-2. Two repurposed compounds, fluvoxamine and amodiaquine, showed efficacy in reducing SARS-CoV-2 viral loads in preclinical experiments, but ultimately failed in clinical trials, highlighting the need for improved predictive preclinical tools that can be rapidly deployed for events such as pandemic emerging infectious diseases. The PREDICT96-ALI platform is a high-throughput, high-fidelity microphysiological system (MPS) that recapitulates primary human tracheobronchial tissue and supports highly robust and reproducible viral titers of SARS-CoV-2 variants Delta and Omicron. When amodiaquine and fluvoxamine were tested in PREDICT96-ALI, neither compound demonstrated an antiviral response, consistent with clinical outcomes and in contrast with prior reports assessing the efficacy of these compounds in other human cell-based in vitro platforms. These results highlight the unique prognostic capability of the PREDICT96-ALI proximal airway MPS to assess the potential antiviral response of lead compounds.\nCRISPR/Cas9 DNA synthesis disruption as a tool to control cell proliferation in vitro and in vivo\nAuthors: Sartori-Maldonado, R.; Montaser, H.; Soppa, I.; Balaz, M.; Puttonen, H.; Eurola, S.; Otonkoski, T.; Saarimaki-Vire, J.; Wartiovaara, K.\nScore: 6.9, Published: 2023-06-26 DOI: 10.1101/2023.06.26.546441\nCell therapies hold a lot of promise for regenerative medicine and gene therapy. However, many living cell products entail a fundamental biological risk of unwanted growth. Here, we describe a novel metabolic safety system to control cell proliferation without exogenous genetic elements. The method allows robust cell culture and manufacture while mitigating the risk of uncontrolled growth of transplanted cells. Using CRISPR/Cas9, we inactivated the enzyme in charge of the de novo thymidylate synthesis (TYMS) in several cell lines, including human induced pluripotent stem cells. This resulted in cells that proliferate when supplemented with exogenous dTMP but fail to grow in its absence. Under dTMP supplementation, TYMS-/- hiPSCs produce mature teratomas in vivo and differentiate normally into potentially therapeutic cell types, such us pancreatic beta cells. After terminal differentiation, the cells no longer require dTMP to remain functional, as seen by prolonged in vivo production of human insulin in mice.\nEngineering in vitro models of cystic fibrosis lung disease using neutrophil extracellular trap inspired biomaterials\nAuthors: Boboltz, A.; Yang, S.; Duncan, G.\nScore: 5.5, Published: 2023-06-28 DOI: 10.1101/2023.06.26.546583\nCystic fibrosis (CF) is a muco-obstructive lung disease where inflammatory responses due to chronic infection result in the accumulation of neutrophil extracellular traps (NETs) in the airways. NETs are web-like complexes comprised mainly of decondensed chromatin that function to capture and kill bacteria. Prior studies have established excess release of NETs in CF airways increases viscoelasticity of mucus secretions and reduces mucociliary clearance. Despite the pivotal role of NETs in CF disease pathogenesis, current in vitro models of this disease do not account for their contribution. Motivated by this, we developed a new approach to study the pathobiological effects of NETs in CF by combining synthetic NET-like biomaterials, composed of DNA and histones, with an in vitro human airway epithelial cell culture model. To determine the impact of synthetic NETs on airway clearance function, we incorporated synthetic NETs into mucin hydrogels and cell culture derived airway mucus to assess their rheological and transport properties. We found that the addition of synthetic NETs significantly increases mucin hydrogel and native mucus viscoelasticity. As a result, mucociliary transport in vitro was significantly reduced with the addition of mucus containing synthetic NETs. Given the prevalence of bacterial infection in the CF lung, we also evaluated the growth of Pseudomonas aeruginosa in mucus with or without synthetic NETs. We found mucus containing synthetic NETs promoted microcolony growth and prolonged bacterial survival. Together, this work establishes a new biomaterial enabled approach to study innate immunity mediated airway dysfunction in CF.\nEfficient Visual Perception of Human-Robot Walking Environments Using Semi-Supervised Learning\nAuthors: Kuzmenko, D.; Tsepa, O.; Kurbis, G.; Mihailidis, A.; Laschowski, B.\nScore: 3.7, Published: 2023-06-29 DOI: 10.1101/2023.06.28.546903\nConvolutional neural networks trained using supervised learning can improve visual perception for human-robot walking. These advances have been possible due to large-scale datasets like ExoNet and StairNet - the largest open-source image datasets of real-world walking environments. However, these datasets require vast amounts of manually annotated data, the development of which is time consuming and labor intensive. Here we present a novel semi-supervised learning system (ExoNet-SSL) that uses over 1.2 million unlabelled images from ExoNet to improve training efficiency. We developed a deep learning model based on mobile vision transformers and trained the model using semi-supervised learning for image classification. Compared to standard supervised learning (98.4%), our ExoNet-SSL system was able to maintain high prediction accuracy (98.8%) when tested on previously unseen environments, while requiring 35% fewer labelled images during training. These results show that semi-supervised learning can improve training efficiency by leveraging large amounts of unlabelled data and minimize the size requirements for manually annotated images. Future research will focus on model deployment for onboard real-time inference and control of human-robot walking.\n",
  "wordCount" : "2230",
  "inLanguage": "en",
  "datePublished": "2023-07-05T10:42:43Z",
  "dateModified": "2023-07-05T10:42:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on July 5, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.12.544557">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.12.544557" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.12.544557">
        <p class="paperTitle">Direct 3D-bioprinting of hiPSC-derived cardiomyocytes to generate functional cardiac tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.12.544557" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.12.544557" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Esser, T. U.; Anspach, A.; Muenzebrock, K. A.; Kah, D.; Schruefer, S.; Schenk, J.; Heinze, K. G.; Schubert, D. W.; Fabry, B.; Engel, F. B.</p>
        <p class="info">Score: 78.6, Published: 2023-06-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.12.544557' target='https://doi.org/10.1101/2023.06.12.544557'> 10.1101/2023.06.12.544557</a></p>
        <p class="abstract">3D-bioprinting is a promising technology to produce human tissues as drug screening tool or for organ repair. However, direct printing of living cells has proven difficult. Here, we present a method to directly 3D-bioprint human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes embedded in a collagen-hyaluronic acid ink generating centimeter-sized functional ring- and ventricle-shaped cardiac tissues in an accurate and reproducible manner. The printed tissues contained hiPSC-derived cardiomyocytes with well-organized sarcomeres and exhibited spontaneous and regular contractions, which persisted for several months and were able to contract against passive resistance. Importantly, beating frequencies of the printed cardiac tissues could be modulated by pharmacological stimulation. This approach opens up new possibilities for generating complex functional cardiac tissues as models for advanced drug screening or as tissue grafts for organ repair or replacement.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.10.31.514613">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.10.31.514613" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.10.31.514613">
        <p class="paperTitle">Protein design using structure-based residue preferences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.10.31.514613" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.10.31.514613" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ding, D.; Shaw, A. Y.; Sinai, S.; Rollins, N. J.; Prywes, N.; Savage, D.; Laub, M. T.; Marks, D. S.</p>
        <p class="info">Score: 48.5, Published: 2023-06-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.10.31.514613' target='https://doi.org/10.1101/2022.10.31.514613'> 10.1101/2022.10.31.514613</a></p>
        <p class="abstract">Recent developments in protein design have adapted large neural networks with up to 100s of millions of parameters to learn complex sequence-function mappings. However, it is unclear which dependencies between residues are critical for determining protein function, and a better empirical understanding could enable high quality models that are also more data- and resource-efficient. Here, we observe that the per residue amino acid preferences - without considering interactions between mutations are sufficient to explain much, and sometimes virtually all of the combinatorial mutation effects across 7 datasets (R2 [~] 78-98%), including one generated here. These preference parameters (20*N, where N is the number of mutated residues) can be learned from as few as [~]5*20*N observations to predict a much larger number (potentially up to 20N) of combinatorial variant effects with high accuracy (Pearson r &gt; 0.8). We hypothesized that the local structural dependencies surrounding a residue could be sufficient to learn these required mutation preferences, and developed an unsupervised design approach, which we term CoVES for  Combinatorial Variant Effects from Structure. We show that CoVES outperforms not just model free sampling approaches but also complicated, high-capacity autoregressive neural networks in generating functional and diverse sequence variants for two example proteins. This simple, biologically-rooted model can be an effective alternative to high-capacity, out of domain models for the design of functional proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546591">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546591" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546591">
        <p class="paperTitle">De Novo Generation and Prioritization of Target-Binding Peptide Motifs from Sequence Alone</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546591" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546591" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bhat, S.; Palepu, K.; Yudistyra, V.; Hong, L.; Kavirayuni, V. S.; Chen, T.; Zhao, L.; Wang, T.; Vincoff, S.; Chatterjee, P.</p>
        <p class="info">Score: 21.2, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546591' target='https://doi.org/10.1101/2023.06.26.546591'> 10.1101/2023.06.26.546591</a></p>
        <p class="abstract">Designing binders to target undruggable proteins presents a formidable challenge in drug discovery, requiring innovative approaches to overcome the lack of putative binding sites. Recently, generative models have been trained to design binding proteins from the three-dimensional structure of a target protein alone, but thus exclude design to disordered or conformationally unstable targets. In this work, we provide a generalizable algorithmic framework to design short, target-binding peptide motifs, requiring only the amino acid sequence of the target protein. To do this, we propose a process to generate naturalistic peptide candidates through Gaussian perturbation of the peptidic latent space of the state-of-the-art ESM-2 protein language model, and subsequently screen these de novo linear sequences for target-selective interaction activity via a CLIP-based contrastive learning architecture. By integrating these generative and discriminative steps, we create a Peptide Prioritization via CLIP (PepPrCLIP) pipeline and validate highly-ranked, target-specific peptide motifs experimentally via fusion to E3 ubiquitin ligase domains, demonstrating functionally potent degradation of conventionally undruggable targets in vitro. Overall, our design strategy provides a modular toolkit for designing short binding motifs to any target protein without the reliance on stable and ordered tertiary structure, enabling generation of programmable modulators to undruggable and disordered proteins such as transcription factors and fusion oncoproteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.01.546933">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.01.546933" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.01.546933">
        <p class="paperTitle">Configurational entropy is an intrinsic driver of tissue structural heterogeneity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.01.546933" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.01.546933" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Srivastava, V.; Hu, J. L.; Garbe, J. C.; Veytsman, B.; Shalabi, S. F.; Yllanes, D.; Thomson, M.; LaBarge, M. A.; Huber, G.; Gartner, Z. J.</p>
        <p class="info">Score: 12.9, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.01.546933' target='https://doi.org/10.1101/2023.07.01.546933'> 10.1101/2023.07.01.546933</a></p>
        <p class="abstract">Tissues comprise ordered arrangements of cells that can be surprisingly disordered in their details. How the properties of single cells and their microenvironment contribute to the balance between order and disorder at the tissue-scale remains poorly understood. Here, we address this question using the self-organization of human mammary organoids as a model. We find that organoids behave like a dynamic structural ensemble at the steady state. We apply a maximum entropy formalism to derive the ensemble distribution from three measurable parameters - the degeneracy of structural states, interfacial energy, and tissue activity (the energy associated with positional fluctuations). We link these parameters with the molecular and microenvironmental factors that control them to precisely engineer the ensemble across multiple conditions. Our analysis reveals that the entropy associated with structural degeneracy sets a theoretical limit to tissue order and provides new insight for tissue engineering, development, and our understanding of disease progression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547045">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547045" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547045">
        <p class="paperTitle">Design, construction and optimization of formaldehyde growth biosensors with broad application in Biotechnology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547045" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547045" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schann, K.; Bakker, J.; Boinot, M.; Kuschel, P.; He, H.; Nattermann, M.; Erb, T.; Bar-Even, A.; Wenk, S.</p>
        <p class="info">Score: 12.3, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547045' target='https://doi.org/10.1101/2023.06.29.547045'> 10.1101/2023.06.29.547045</a></p>
        <p class="abstract">Formaldehyde is a key metabolite in natural and synthetic one-carbon metabolism as well as an important environmental toxin with high toxicity at low concentrations. To engineer efficient formaldehyde producing enzymes and to detect formaldehyde in industrial or environmental samples, it is important to establish highly sensitive, easy to use and affordable formaldehyde detection methods. Here, we transformed the workhorse bacterium Escherichia coli into biosensors that can detect a broad range of formaldehyde concentrations. Based on natural and promiscuous formaldehyde assimilation enzymes, we designed and engineered three different E. coli strains that depend on formaldehyde assimilation for cellular growth. After in depth characterization of these biosensors, we show that the formaldehyde sensitivity can be improved through adaptive laboratory evolution or modification of metabolic branch points. The metabolic engineering strategy presented in this work allowed the creation of E. coli biosensors that can detect formaldehyde in a concentration range from ~30 M to ~13 mM. Using the most sensitive strain, we benchmarked the in vivo activities of different, widely used NAD-dependent methanol dehydrogenases, the rate-limiting enzyme in synthetic methylotrophy. We also show that the strains can grow upon external addition of formaldehyde indicating their potential use for applications beyond enzyme engineering. The formaldehyde biosensors developed in this study are fully genomic and can be used as plug and play devices for screening large enzyme libraries. Thus, they have the potential to greatly advance enzyme engineering and might even be used for environmental monitoring or analysis of industrial probes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.16.545316">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.16.545316" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.16.545316">
        <p class="paperTitle">OpenSIM: open source microscope add-on for structured illumination microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.16.545316" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.16.545316" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hannebelle, M. T. M.; Raeth, E.; Leitao, S.; Lukes, T.; Pospisil, J.; Toniolo, C.; Venzin, O. F.; Chrisnandy, A.; Swain, P. P.; Ronceray, N.; Lutolf, M. P.; Oates, A. C.; Hagen, G. M.; Lasser, T.; Radenovic, A.; Mckinney, J. D.; Fantner, G. E.</p>
        <p class="info">Score: 18.2, Published: 2023-06-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.16.545316' target='https://doi.org/10.1101/2023.06.16.545316'> 10.1101/2023.06.16.545316</a></p>
        <p class="abstract">Super-resolution techniques expand the abilities of researchers who have the knowledge and resources to either build or purchase a system. This excludes the part of the research community without these capabilities. Here we introduce the openSIM add-on to upgrade existing optical microscopes to Structured Illumination super-resolution Microscopes (SIM). The openSIM is an open-hardware system, designed and documented to be easily duplicated by other laboratories, making super-resolution modality accessible to facilitate innovative research. The add-on approach gives a performance improvement for pre-existing lab equipment without the need to build a completely new system.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546790">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546790" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546790">
        <p class="paperTitle">A human primary airway microphysiological system infected with SARS-CoV-2 distinguishes the treatment efficacy between nirmatrelvir and repurposed compounds fluvoxamine and amodiaquine</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546790" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546790" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lopez Quezada, L.; Mba Medie, F.; Gabriel, E. P.; Luu, R. J.; Rubio, L. D.; Mulhern, T. J.; Borenstein, J. T.; Fisher, C. R.; Gard, A. L.</p>
        <p class="info">Score: 7.3, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546790' target='https://doi.org/10.1101/2023.06.27.546790'> 10.1101/2023.06.27.546790</a></p>
        <p class="abstract">The COVID-19 pandemic necessitated a rapid mobilization of resources toward the development of safe and efficacious vaccines and therapeutics. Finding effective treatments to stem the wave of infected individuals needing hospitalization and reduce the risk of adverse events was paramount. For scientists and healthcare professionals addressing this challenge, the need to rapidly identify medical countermeasures became urgent, and many compounds in clinical use for other indications were repurposed for COVID-19 clinical trials after preliminary preclinical data demonstrated antiviral activity against SARS-CoV-2. Two repurposed compounds, fluvoxamine and amodiaquine, showed efficacy in reducing SARS-CoV-2 viral loads in preclinical experiments, but ultimately failed in clinical trials, highlighting the need for improved predictive preclinical tools that can be rapidly deployed for events such as pandemic emerging infectious diseases. The PREDICT96-ALI platform is a high-throughput, high-fidelity microphysiological system (MPS) that recapitulates primary human tracheobronchial tissue and supports highly robust and reproducible viral titers of SARS-CoV-2 variants Delta and Omicron. When amodiaquine and fluvoxamine were tested in PREDICT96-ALI, neither compound demonstrated an antiviral response, consistent with clinical outcomes and in contrast with prior reports assessing the efficacy of these compounds in other human cell-based in vitro platforms. These results highlight the unique prognostic capability of the PREDICT96-ALI proximal airway MPS to assess the potential antiviral response of lead compounds.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546441">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546441" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546441">
        <p class="paperTitle">CRISPR/Cas9 DNA synthesis disruption as a tool to control cell proliferation in vitro and in vivo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546441" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546441" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sartori-Maldonado, R.; Montaser, H.; Soppa, I.; Balaz, M.; Puttonen, H.; Eurola, S.; Otonkoski, T.; Saarimaki-Vire, J.; Wartiovaara, K.</p>
        <p class="info">Score: 6.9, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546441' target='https://doi.org/10.1101/2023.06.26.546441'> 10.1101/2023.06.26.546441</a></p>
        <p class="abstract">Cell therapies hold a lot of promise for regenerative medicine and gene therapy. However, many living cell products entail a fundamental biological risk of unwanted growth. Here, we describe a novel metabolic safety system to control cell proliferation without exogenous genetic elements. The method allows robust cell culture and manufacture while mitigating the risk of uncontrolled growth of transplanted cells.

Using CRISPR/Cas9, we inactivated the enzyme in charge of the de novo thymidylate synthesis (TYMS) in several cell lines, including human induced pluripotent stem cells. This resulted in cells that proliferate when supplemented with exogenous dTMP but fail to grow in its absence. Under dTMP supplementation, TYMS-/- hiPSCs produce mature teratomas in vivo and differentiate normally into potentially therapeutic cell types, such us pancreatic beta cells. After terminal differentiation, the cells no longer require dTMP to remain functional, as seen by prolonged in vivo production of human insulin in mice.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546583">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546583" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546583">
        <p class="paperTitle">Engineering in vitro models of cystic fibrosis lung disease using neutrophil extracellular trap inspired biomaterials</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546583" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546583" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boboltz, A.; Yang, S.; Duncan, G.</p>
        <p class="info">Score: 5.5, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546583' target='https://doi.org/10.1101/2023.06.26.546583'> 10.1101/2023.06.26.546583</a></p>
        <p class="abstract">Cystic fibrosis (CF) is a muco-obstructive lung disease where inflammatory responses due to chronic infection result in the accumulation of neutrophil extracellular traps (NETs) in the airways. NETs are web-like complexes comprised mainly of decondensed chromatin that function to capture and kill bacteria. Prior studies have established excess release of NETs in CF airways increases viscoelasticity of mucus secretions and reduces mucociliary clearance. Despite the pivotal role of NETs in CF disease pathogenesis, current in vitro models of this disease do not account for their contribution. Motivated by this, we developed a new approach to study the pathobiological effects of NETs in CF by combining synthetic NET-like biomaterials, composed of DNA and histones, with an in vitro human airway epithelial cell culture model. To determine the impact of synthetic NETs on airway clearance function, we incorporated synthetic NETs into mucin hydrogels and cell culture derived airway mucus to assess their rheological and transport properties. We found that the addition of synthetic NETs significantly increases mucin hydrogel and native mucus viscoelasticity. As a result, mucociliary transport in vitro was significantly reduced with the addition of mucus containing synthetic NETs. Given the prevalence of bacterial infection in the CF lung, we also evaluated the growth of Pseudomonas aeruginosa in mucus with or without synthetic NETs. We found mucus containing synthetic NETs promoted microcolony growth and prolonged bacterial survival. Together, this work establishes a new biomaterial enabled approach to study innate immunity mediated airway dysfunction in CF.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546903">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546903" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546903">
        <p class="paperTitle">Efficient Visual Perception of Human-Robot Walking Environments Using Semi-Supervised Learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546903" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546903" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kuzmenko, D.; Tsepa, O.; Kurbis, G.; Mihailidis, A.; Laschowski, B.</p>
        <p class="info">Score: 3.7, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546903' target='https://doi.org/10.1101/2023.06.28.546903'> 10.1101/2023.06.28.546903</a></p>
        <p class="abstract">Convolutional neural networks trained using supervised learning can improve visual perception for human-robot walking. These advances have been possible due to large-scale datasets like ExoNet and StairNet - the largest open-source image datasets of real-world walking environments. However, these datasets require vast amounts of manually annotated data, the development of which is time consuming and labor intensive. Here we present a novel semi-supervised learning system (ExoNet-SSL) that uses over 1.2 million unlabelled images from ExoNet to improve training efficiency. We developed a deep learning model based on mobile vision transformers and trained the model using semi-supervised learning for image classification. Compared to standard supervised learning (98.4%), our ExoNet-SSL system was able to maintain high prediction accuracy (98.8%) when tested on previously unseen environments, while requiring 35% fewer labelled images during training. These results show that semi-supervised learning can improve training efficiency by leveraging large amounts of unlabelled data and minimize the size requirements for manually annotated images. Future research will focus on model deployment for onboard real-time inference and control of human-robot walking.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
