<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Protein generation with evolutionary diffusion: sequence is all you need
Authors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.
Score: 172.6, Published: 2023-09-12 DOI: 10.1101/2023.09.11.556673
Deep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Protein generation with evolutionary diffusion: sequence is all you need
Authors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.
Score: 172.6, Published: 2023-09-12 DOI: 10.1101/2023.09.11.556673
Deep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-17T10:36:50+00:00" />
<meta property="article:modified_time" content="2023-09-17T10:36:50+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Protein generation with evolutionary diffusion: sequence is all you need
Authors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.
Score: 172.6, Published: 2023-09-12 DOI: 10.1101/2023.09.11.556673
Deep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Protein generation with evolutionary diffusion: sequence is all you need\nAuthors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.\nScore: 172.6, Published: 2023-09-12 DOI: 10.1101/2023.09.11.556673\nDeep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design.",
  "keywords": [
    
  ],
  "articleBody": " Protein generation with evolutionary diffusion: sequence is all you need\nAuthors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.\nScore: 172.6, Published: 2023-09-12 DOI: 10.1101/2023.09.11.556673\nDeep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design. However, current state-of-the-art models generate protein structures, which limits the scope of their training data and restricts generations to a small and biased subset of protein design space. Here, we introduce a general-purpose diffusion framework, EvoDiff, that combines evolutionary-scale data with the distinct conditioning capabilities of diffusion models for controllable protein generation in sequence space. EvoDiff generates high-fidelity, diverse, and structurally-plausible proteins that cover natural sequence and functional space. Critically, EvoDiff can generate proteins inaccessible to structure-based models, such as those with disordered regions, while maintaining the ability to design scaffolds for functional structural motifs, demonstrating the universality of our sequence-based formulation. We envision that EvoDiff will expand capabilities in protein engineering beyond the structure-function paradigm toward programmable, sequence-first design.\nClick editing enables programmable genome writing using DNA polymerases and HUH endonucleases\nAuthors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.\nScore: 83.0, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557440\nGenome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e.g. CRISPR-Cas) to permit the installation of a range of edits including substitutions, insertions, and deletions. Click editors (CEs) leverage the \"click\"-like bioconjugation ability of HUH endonucleases (HUHes) with single stranded DNA substrates to covalently tether \"click DNA\" (clkDNA) templates encoding user-specifiable edits at targeted genomic loci. Through iterative optimization of the modular components of CEs (DNA polymerase and HUHe orthologs, architectural modifications, etc.) and their clkDNAs (template configurations, repair evading substitutions, etc.), we demonstrate the ability to install precise genome edits with minimal indels and no unwanted byproduct insertions. Since clkDNAs can be ordered as simple DNA oligonucleotides for cents per base, it is possible to screen many different clkDNA parameters rapidly and inexpensively to maximize edit efficiency. Together, click editing is a precise and highly versatile platform for modifying genomes with a simple workflow and broad utility across diverse biological applications.\nCentimeter-scale perfusable cultured meat with densely packed, highly aligned muscle fibers via hollow fiber bioreactor\nAuthors: Nie, M.; Shima, A.; Takeuchi, S.\nScore: 12.8, Published: 2023-09-04 DOI: 10.1101/2023.09.04.555230\nThe development of in-vitro biofabrication methods for producing cultured meat based on animal cells has been advancing, but replicating the texture of traditional meat in centimeter-scale has been a challenge. To address this, a method using a hollow fiber bioreactor (HFB) has been developed. The HFB contains semipermeable hollow fibers that act as artificial circulatory systems to deliver nutrients and oxygen uniformly to the tissue, along with microfabricated anchors for inducing cell alignment. With active perfusion, the biofabricated centimeter-scale chick muscle tissue shows elevated levels of marker protein expression and sarcomere formation across the whole tissue, along with improved texture and flavor. In the future, further scaling up of this approach using industrial robots has the potential to transform not only the cultured meat industry but also the tissue engineering fields aiming for the formation of large-scale artificial organs.\nHuman gut metagenomic mining reveals an untapped source of peptide antibiotics\nAuthors: Torres, M. D. T.; Brooks, E. F.; Cesaro, A.; Sberro, H.; Nicolaou, C.; Bhatt, A. S.; de la Fuente, C.\nScore: 32.0, Published: 2023-09-03 DOI: 10.1101/2023.08.31.555711\nDrug-resistant bacteria are outpacing traditional antibiotic discovery efforts. Here, we computationally mined 444,054 families of putative small proteins from 1,773 human gut metagenomes, identifying 323 peptide antibiotics encoded in small open reading frames (smORFs). To test our computational predictions, 78 peptides were synthesized and screened for antimicrobial activity in vitro, with 59% displaying activity against either pathogens or commensals. Since these peptides were unique compared to previously reported antimicrobial peptides, we termed them smORF-encoded peptides (SEPs). SEPs killed bacteria by targeting their membrane, synergized with each other, and modulated gut commensals, indicating that they may play a role in reconfiguring microbiome communities in addition to counteracting pathogens. The lead candidates were anti-infective in both murine skin abscess and deep thigh infection models. Notably, prevotellin-2 from Prevotella copri presented activity comparable to the commonly used antibiotic polymyxin B. We report the discovery of hundreds of peptide sequences in the human gut.\nIn vivo selection in non-human primates identifies superior AAV capsids for on-target CSF delivery to spinal cord\nAuthors: Hanlon, K. S.; Cheng, M.; De La Cruz, D.; Patel, N.; Santoscoy, M. C.; Gong, Y.; Ng, C.; Nguyen, D. M.; Nammour, J.; Clark, S. W.; Kozarsky, K.; Maguire, C. A.\nScore: 5.2, Published: 2023-09-13 DOI: 10.1101/2023.09.13.557506\nSystemic administration of adeno-associated virus (AAV) vectors for spinal cord gene therapy has challenges including toxicity at high doses and pre-existing immunity that reduces efficacy. Intrathecal delivery of AAV vectors into the cerebral spinal fluid (CSF) can avoid many of the issues of systemic delivery, although achieving broad distribution of the vector and transgene expression throughout the spinal cord is challenging and vector entry to the periphery occurs, sometimes initiating hepatotoxicity. Here we performed two rounds of in vivo biopanning in non-human primates (NHPs) with an AAV9 peptide display library injected intrathecally and performed insert sequencing on DNA isolated from either whole tissue (conventional selection), isolated nuclei, or nuclei from transgene-expressing cells. A subsequent barcoded pool of candidates and AAV9 was compared at the DNA (biodistribution) and RNA (expression) level in spinal cord and liver of intrathecally injected NHPs. Most of the candidates displayed enhanced biodistribution compared to AAV9 at all levels of spinal cord ranging from 2 to 265-fold. Nuclear isolation or expression-based selection yielded 4 of 7 candidate capsids with enhanced transgene expression in spinal cord (up to 2.4-fold), while no capsid obtained by conventional selection achieved that level. Furthermore, several capsids displayed lower biodistribution to the liver of up to 1,250-fold, compared to AAV9, providing a remarkable on target/off target biodistribution ratio. These capsids may have potential for gene therapy programs directed at the spinal cord and the selection method described here should be useful in clinically relevant large animal models.\nSpyCombinator Assembly of Bispecific Binders\nAuthors: Driscoll, C. L.; Keeble, A. H.; Howarth, M.\nScore: 5.9, Published: 2023-09-03 DOI: 10.1101/2023.08.31.555700\nBispecific antibodies are a successful and expanding therapeutic class, bridging two cell-types or engaging two different molecules on the same cell. Bispecifics unlock avenues towards synergy, resistance evasion, and signaling bias. Standard approaches to generate bispecifics are complicated by the need for disulfide reduction/oxidation or cloning of each binder molecule in two different formats. Here we present a modular approach to bispecifics using SpyTag/SpyCatcher spontaneous amidation, where all binders are cloned in the same format, bearing a SpyTag. Two SpyTag-fused antigen-binding modules can be precisely conjugated onto DoubleCatcher, a tandem SpyCatcher where the second Catcher is unreactive until unveiling of reactivity using site-specific protease. Assembly on DoubleCatcher is efficient in phosphate-buffered saline at 37 {degrees}C, with half-times less than 5 min for both SpyCatcher arms and over 97% bispecific homogeneity. We engineer a panel of DoubleCatchers, locked through disulfide bonds to direct binders to project in different directions from the hub. We establish a generalized methodology for one-pot assembly and purification of bispecifics in 96-well plate format. A panel of Fab, affibody or nanobody binders recognizing different sites on HER2 were coupled to DoubleCatcher, revealing unexpected combinations with anti-proliferative or pro-proliferative activity on HER2-addicted cancer cells. Bispecific activity depended sensitively on both the order of the binders within the assembly and the geometry of DoubleCatcher scaffolds. These findings support the need for straightforward assembly in different formats. SpyCombinator provides a simple and scalable tool to discover synergy in bispecific activity, through modulating receptor organization and geometry.\nMAGIK: A rapid and efficient method to create lineage-specific reporters in human pluripotent stem cells\nAuthors: Haideri, T.; Lin, J.; Bao, X.; Lian, X. L.\nScore: 3.6, Published: 2023-09-10 DOI: 10.1101/2023.09.07.556774\nPrecise insertion of a fluorescent protein into a lineage-specific gene in human pluripotent stem cells (hPSCs) presents challenges due to the low knockin efficiency and difficulties in selecting the correctly targeted cells. Here we introduce the ModRNA-based Activation for Gene Insertion and Knockin (MAGIK) approach to enhance knockin efficacy in hPSCs. MAGIK operates in two steps: first, it employs a Cas9-2A-p53DD modRNA with a mini-donor plasmid (without a drug-selection cassette) to significantly enhance efficiency; second, a dCas9 activator modRNA and a dgRNA are used to temporarily activate the successfully targeted gene, allowing for live cell sorting without single cell cloning. Consequently, MAGIK eliminates the need for drug selection cassettes or labor-intensive single cell colony screening, expediting precise genetic integration. We have demonstrated that MAGIK can be utilized to insert fluorescent proteins into various genes, including SOX17, NKX6.1, NKX2.5 and PDX1, across multiple hPSC lines, showcasing its robustness. This innovative MAGIK approach streamlines the process and provides a promising solution for targeted genetic modifications in hPSCs.\nCRISPR-Cas9 Extracellular Vesicles for Treating Hearing Loss\nAuthors: Pan, X.; Huang, P.; Ali, S.; Hutchinson, T. E.; Erwin, N.; Greenberg, Z. F.; Ding, Z.; Li, Y.; Fernadez, N. E.; Staecker, H.; He, M.\nScore: 3.0, Published: 2023-09-15 DOI: 10.1101/2023.09.14.557853\nThe treatment of inner ear disorders remains challenging due to the intrinsic anatomical barriers. The majority treatments and delivery approaches for accessing inner hair cells are still engaged with surgical intervention, which is highly invasive and inconsistent in terms of efficacy and safety. In order to address this challenge for crossing anatomical barriers, we report an extracellular vesicle (EVs) -based delivery approach to inner hair cells, which enables carrying CRISPR/Cas9 ribonucleoprotein (RNP)-sgRNA complex in high-throughput and high efficiency. The novel Microfluidic Droplet-based Electroporation System (uDES) is developed to efficiently load cargos into EVs via millisecond pulsed, low-voltage electroporation within flow-through droplets as enormous bioreactors in a continuous-flow and scalable manner. The observed loading efficiency of CRISPR/Cas9 RNA complex into EVs (RNP-EVs) is 10-fold higher than current bulk cuvette electroporation with hundred-fold increase of processing throughput. The low-voltage electroporation minimized the Joule heating influence on nanosized EVs, which retained the native surface membrane properties of cargo-loaded EVs. Both ex vivo and in vivo testing in Shaker-1 mice model demonstrated the high biocompatibility and biodistribution of produced RNP-EVs in the mouse cochlea penetrating inner hair cells. In contrast, the CRISPR/Cas9 RNP lipid-like nanoparticles (RNP-LNPs) control group was unable to penetrate anatomical barriers to access inner hair cells. In the Shaker-1 mouse model, uDES produced RNP-EVs demonstrated much higher editing efficiency at Myo7ash1 mRNA level and showed significant hearing recovery in the Myo7aWT/Sh1 mice via Auditory Brainstem Response (ABR) testing. The report work will present a new solution to advance gene therapy in treating sensorineural hearing loss.\nThe colocatome as a spatial -omic reveals shared microenvironment features between tumour-stroma assembloids and lung cancer specimens\nAuthors: Bouchard, G.; Zhang, W.; Li, I.; Ilerten, I.; Shrager, J. B.; Bhattacharya, A.; Li, Y.; Trope, W.; Osawa, M. G.; Kuo, C.; Tian, L.; Giaccia, A. J.; Plevritis, S. K.\nScore: 3.0, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557278\nComputational frameworks to quantify and compare microenvironment spatial features of in-vitro patient-derived models and clinical specimens are needed. Here, we acquired and analysed multiplexed immunofluorescence images of human lung adenocarcinoma (LUAD) alongside tumour- stroma assembloids constructed with organoids and fibroblasts harvested from the leading edge (Tumour-Adjacent Fibroblasts;TAFs) or core (Tumour Core Fibroblasts;TCFs) of human LUAD. We introduce the concept of the \"colocatome\" as a spatial -omic dimension to catalogue all proximate and distant colocalisations between malignant and fibroblast subpopulations in both the assembloids and clinical specimens. The colocatome expands upon the colocalisation quotient (CLQ) through a nomalisation strategy that involves permutation analysis and thereby allows comparisons of CLQs under different conditions. Using colocatome analysis, we report that both TAFs and TCFs protected cancer cells from targeted oncogene treatment by uniquely reorganising the tumour-stroma cytoarchitecture, rather than by promoting cellular heterogeneity or selection. Moreover, we show that the assembloids colocatome recapitulates the tumour-stroma cytoarchitecture defining the tumour microenvironment of LUAD clinical samples and thereby can serve as a functional spatial readout to guide translational discoveries. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=190 SRC=\"FIGDIR/small/557278v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (60K): org.highwire.dtl.DTLVardef@7bf7a1org.highwire.dtl.DTLVardef@be7274org.highwire.dtl.DTLVardef@14c5572org.highwire.dtl.DTLVardef@7dd5fd_HPS_FORMAT_FIGEXP M_FIG C_FIG\nImage-based parametric finite element modelling for studying contact mechanics in human knee joints\nAuthors: Readioff, R.; Seil, R.; Mouton, C.; Marks, L.; Barrera, O.\nScore: 2.8, Published: 2023-09-10 DOI: 10.1101/2023.09.07.556747\nPurposeThis study presents a framework for generating patient-specific finite element models, parameterised and optimised for contact mechanics from computed tomography (CT) scans, by avoiding the segmentation step usually employed to transform medical images into 3D models. Two morphological parameters affecting contact mechanics were investigated in the framework development: tibial cartilage thickness and tibial spine height. This study explores the effect of the interplay of these parameters in load sharing between meniscus and articulating cartilage, meniscal posterior and anterior roots strain and menisci kine-matics. MethodsMorphological measurements from four knee CT scans were collected, such as the maximum thickness of the tibial cartilage (ranging from 1.1 to 5.2 mm), the height of the tibial spine (ranging from 3.55 to 10.1 mm), and the width of the tibial plateau in both the coronal (ranging from 27.3 to 36.17 mm) and sagittal (ranging from 31.79 to 53.77 mm) planes. These measurements were taken for the lateral tibial plateau for both left and right knees. Subsequently, three finite element (FE) models were generated, comprising lateral tibial plateaus, lateral femoral condyle and lateral meniscus. The tibial cartilage thickness was kept at a constant value of 1 mm while varying the tibial spine height within the range measured from the CT images. This resulted in three FE models with varying spine heights, categorised as large (height = 7.42 mm), medium (height = 4.25 mm), and small (height = 1.63 mm) tibial spine heights. The menisci in the FE models were generated to be congruent with the tibial plateau. For the first time, this study advances the representations of the knee menisci microstructure in FE modelling, such that we have generated meniscus FE models with three layers of a hyperelastic model in which layer thickness and layer-specific hyperelastic material parameters are derived from our previous experimental work. ResultsThe load sharing between the meniscus and articular cartilage was not sensitive to the varying tibial spine heights. In all three FE models, cartilage carried more than 90% of the applied load. However, the meniscus kinematics and root strains varied considerably with changing tibial spine heights. The small tibial spine height model predicted the highest meniscus movements (8.12 and 9.33 mm in the radial and circumferential directions, respectively) and the highest root strain (21.92 and 22.19 mm/mm in the anterior and posterior roots, respectively). ConclusionOur framework can generate finite element models of patients knees using clinical data (i.e., CT scans) without the need for lengthy image segmentation. This process is not only time-efficient but also independent of imaging operators. The models converge quickly ({inverted question}30 minutes on 2 cores) using an implicit solver with non-linear geometry and have the capability to predict contact mechanics between the articulating surfaces, meniscus kinematics and root strains. The modelling strategy presented here can provide valuable insights into predicting changes in the mechanics of soft tissues in the knee joint. It is particularly useful for investigating injury and surgical mechanisms related to the meniscus.\n",
  "wordCount" : "2596",
  "inLanguage": "en",
  "datePublished": "2023-09-17T10:36:50Z",
  "dateModified": "2023-09-17T10:36:50Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on September 17, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.556673">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.556673" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.556673">
        <p class="paperTitle">Protein generation with evolutionary diffusion: sequence is all you need</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.556673" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.556673" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.</p>
        <p class="info">Score: 172.6, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.556673' target='https://doi.org/10.1101/2023.09.11.556673'> 10.1101/2023.09.11.556673</a></p>
        <p class="abstract">Deep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design. However, current state-of-the-art models generate protein structures, which limits the scope of their training data and restricts generations to a small and biased subset of protein design space. Here, we introduce a general-purpose diffusion framework, EvoDiff, that combines evolutionary-scale data with the distinct conditioning capabilities of diffusion models for controllable protein generation in sequence space. EvoDiff generates high-fidelity, diverse, and structurally-plausible proteins that cover natural sequence and functional space. Critically, EvoDiff can generate proteins inaccessible to structure-based models, such as those with disordered regions, while maintaining the ability to design scaffolds for functional structural motifs, demonstrating the universality of our sequence-based formulation. We envision that EvoDiff will expand capabilities in protein engineering beyond the structure-function paradigm toward programmable, sequence-first design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557440">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557440" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557440">
        <p class="paperTitle">Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557440" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557440" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.</p>
        <p class="info">Score: 83.0, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557440' target='https://doi.org/10.1101/2023.09.12.557440'> 10.1101/2023.09.12.557440</a></p>
        <p class="abstract">Genome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e.g. CRISPR-Cas) to permit the installation of a range of edits including substitutions, insertions, and deletions. Click editors (CEs) leverage the &#34;click&#34;-like bioconjugation ability of HUH endonucleases (HUHes) with single stranded DNA substrates to covalently tether &#34;click DNA&#34; (clkDNA) templates encoding user-specifiable edits at targeted genomic loci. Through iterative optimization of the modular components of CEs (DNA polymerase and HUHe orthologs, architectural modifications, etc.) and their clkDNAs (template configurations, repair evading substitutions, etc.), we demonstrate the ability to install precise genome edits with minimal indels and no unwanted byproduct insertions. Since clkDNAs can be ordered as simple DNA oligonucleotides for cents per base, it is possible to screen many different clkDNA parameters rapidly and inexpensively to maximize edit efficiency. Together, click editing is a precise and highly versatile platform for modifying genomes with a simple workflow and broad utility across diverse biological applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.555230">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.555230" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.555230">
        <p class="paperTitle">Centimeter-scale perfusable cultured meat with densely packed, highly aligned muscle fibers via hollow fiber bioreactor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.555230" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.555230" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nie, M.; Shima, A.; Takeuchi, S.</p>
        <p class="info">Score: 12.8, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.555230' target='https://doi.org/10.1101/2023.09.04.555230'> 10.1101/2023.09.04.555230</a></p>
        <p class="abstract">The development of in-vitro biofabrication methods for producing cultured meat based on animal cells has been advancing, but replicating the texture of traditional meat in centimeter-scale has been a challenge. To address this, a method using a hollow fiber bioreactor (HFB) has been developed. The HFB contains semipermeable hollow fibers that act as artificial circulatory systems to deliver nutrients and oxygen uniformly to the tissue, along with microfabricated anchors for inducing cell alignment. With active perfusion, the biofabricated centimeter-scale chick muscle tissue shows elevated levels of marker protein expression and sarcomere formation across the whole tissue, along with improved texture and flavor. In the future, further scaling up of this approach using industrial robots has the potential to transform not only the cultured meat industry but also the tissue engineering fields aiming for the formation of large-scale artificial organs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555711">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555711" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555711">
        <p class="paperTitle">Human gut metagenomic mining reveals an untapped source of peptide antibiotics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555711" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555711" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Torres, M. D. T.; Brooks, E. F.; Cesaro, A.; Sberro, H.; Nicolaou, C.; Bhatt, A. S.; de la Fuente, C.</p>
        <p class="info">Score: 32.0, Published: 2023-09-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555711' target='https://doi.org/10.1101/2023.08.31.555711'> 10.1101/2023.08.31.555711</a></p>
        <p class="abstract">Drug-resistant bacteria are outpacing traditional antibiotic discovery efforts. Here, we computationally mined 444,054 families of putative small proteins from 1,773 human gut metagenomes, identifying 323 peptide antibiotics encoded in small open reading frames (smORFs). To test our computational predictions, 78 peptides were synthesized and screened for antimicrobial activity in vitro, with 59% displaying activity against either pathogens or commensals. Since these peptides were unique compared to previously reported antimicrobial peptides, we termed them smORF-encoded peptides (SEPs). SEPs killed bacteria by targeting their membrane, synergized with each other, and modulated gut commensals, indicating that they may play a role in reconfiguring microbiome communities in addition to counteracting pathogens. The lead candidates were anti-infective in both murine skin abscess and deep thigh infection models. Notably, prevotellin-2 from Prevotella copri presented activity comparable to the commonly used antibiotic polymyxin B. We report the discovery of hundreds of peptide sequences in the human gut.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557506">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557506" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557506">
        <p class="paperTitle">In vivo selection in non-human primates identifies superior AAV capsids for on-target CSF delivery to spinal cord</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557506" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557506" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hanlon, K. S.; Cheng, M.; De La Cruz, D.; Patel, N.; Santoscoy, M. C.; Gong, Y.; Ng, C.; Nguyen, D. M.; Nammour, J.; Clark, S. W.; Kozarsky, K.; Maguire, C. A.</p>
        <p class="info">Score: 5.2, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557506' target='https://doi.org/10.1101/2023.09.13.557506'> 10.1101/2023.09.13.557506</a></p>
        <p class="abstract">Systemic administration of adeno-associated virus (AAV) vectors for spinal cord gene therapy has challenges including toxicity at high doses and pre-existing immunity that reduces efficacy. Intrathecal delivery of AAV vectors into the cerebral spinal fluid (CSF) can avoid many of the issues of systemic delivery, although achieving broad distribution of the vector and transgene expression throughout the spinal cord is challenging and vector entry to the periphery occurs, sometimes initiating hepatotoxicity. Here we performed two rounds of in vivo biopanning in non-human primates (NHPs) with an AAV9 peptide display library injected intrathecally and performed insert sequencing on DNA isolated from either whole tissue (conventional selection), isolated nuclei, or nuclei from transgene-expressing cells. A subsequent barcoded pool of candidates and AAV9 was compared at the DNA (biodistribution) and RNA (expression) level in spinal cord and liver of intrathecally injected NHPs. Most of the candidates displayed enhanced biodistribution compared to AAV9 at all levels of spinal cord ranging from 2 to 265-fold. Nuclear isolation or expression-based selection yielded 4 of 7 candidate capsids with enhanced transgene expression in spinal cord (up to 2.4-fold), while no capsid obtained by conventional selection achieved that level. Furthermore, several capsids displayed lower biodistribution to the liver of up to 1,250-fold, compared to AAV9, providing a remarkable on target/off target biodistribution ratio. These capsids may have potential for gene therapy programs directed at the spinal cord and the selection method described here should be useful in clinically relevant large animal models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555700">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555700" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555700">
        <p class="paperTitle">SpyCombinator Assembly of Bispecific Binders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555700" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555700" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Driscoll, C. L.; Keeble, A. H.; Howarth, M.</p>
        <p class="info">Score: 5.9, Published: 2023-09-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555700' target='https://doi.org/10.1101/2023.08.31.555700'> 10.1101/2023.08.31.555700</a></p>
        <p class="abstract">Bispecific antibodies are a successful and expanding therapeutic class, bridging two cell-types or engaging two different molecules on the same cell. Bispecifics unlock avenues towards synergy, resistance evasion, and signaling bias. Standard approaches to generate bispecifics are complicated by the need for disulfide reduction/oxidation or cloning of each binder molecule in two different formats. Here we present a modular approach to bispecifics using SpyTag/SpyCatcher spontaneous amidation, where all binders are cloned in the same format, bearing a SpyTag. Two SpyTag-fused antigen-binding modules can be precisely conjugated onto DoubleCatcher, a tandem SpyCatcher where the second Catcher is unreactive until unveiling of reactivity using site-specific protease. Assembly on DoubleCatcher is efficient in phosphate-buffered saline at 37 {degrees}C, with half-times less than 5 min for both SpyCatcher arms and over 97% bispecific homogeneity. We engineer a panel of DoubleCatchers, locked through disulfide bonds to direct binders to project in different directions from the hub. We establish a generalized methodology for one-pot assembly and purification of bispecifics in 96-well plate format. A panel of Fab, affibody or nanobody binders recognizing different sites on HER2 were coupled to DoubleCatcher, revealing unexpected combinations with anti-proliferative or pro-proliferative activity on HER2-addicted cancer cells. Bispecific activity depended sensitively on both the order of the binders within the assembly and the geometry of DoubleCatcher scaffolds. These findings support the need for straightforward assembly in different formats. SpyCombinator provides a simple and scalable tool to discover synergy in bispecific activity, through modulating receptor organization and geometry.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556774">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556774" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556774">
        <p class="paperTitle">MAGIK: A rapid and efficient method to create lineage-specific reporters in human pluripotent stem cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556774" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556774" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Haideri, T.; Lin, J.; Bao, X.; Lian, X. L.</p>
        <p class="info">Score: 3.6, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556774' target='https://doi.org/10.1101/2023.09.07.556774'> 10.1101/2023.09.07.556774</a></p>
        <p class="abstract">Precise insertion of a fluorescent protein into a lineage-specific gene in human pluripotent stem cells (hPSCs) presents challenges due to the low knockin efficiency and difficulties in selecting the correctly targeted cells. Here we introduce the ModRNA-based Activation for Gene Insertion and Knockin (MAGIK) approach to enhance knockin efficacy in hPSCs. MAGIK operates in two steps: first, it employs a Cas9-2A-p53DD modRNA with a mini-donor plasmid (without a drug-selection cassette) to significantly enhance efficiency; second, a dCas9 activator modRNA and a dgRNA are used to temporarily activate the successfully targeted gene, allowing for live cell sorting without single cell cloning. Consequently, MAGIK eliminates the need for drug selection cassettes or labor-intensive single cell colony screening, expediting precise genetic integration. We have demonstrated that MAGIK can be utilized to insert fluorescent proteins into various genes, including SOX17, NKX6.1, NKX2.5 and PDX1, across multiple hPSC lines, showcasing its robustness. This innovative MAGIK approach streamlines the process and provides a promising solution for targeted genetic modifications in hPSCs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557853">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557853" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557853">
        <p class="paperTitle">CRISPR-Cas9 Extracellular Vesicles for Treating Hearing Loss</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557853" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557853" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pan, X.; Huang, P.; Ali, S.; Hutchinson, T. E.; Erwin, N.; Greenberg, Z. F.; Ding, Z.; Li, Y.; Fernadez, N. E.; Staecker, H.; He, M.</p>
        <p class="info">Score: 3.0, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557853' target='https://doi.org/10.1101/2023.09.14.557853'> 10.1101/2023.09.14.557853</a></p>
        <p class="abstract">The treatment of inner ear disorders remains challenging due to the intrinsic anatomical barriers. The majority treatments and delivery approaches for accessing inner hair cells are still engaged with surgical intervention, which is highly invasive and inconsistent in terms of efficacy and safety. In order to address this challenge for crossing anatomical barriers, we report an extracellular vesicle (EVs) -based delivery approach to inner hair cells, which enables carrying CRISPR/Cas9 ribonucleoprotein (RNP)-sgRNA complex in high-throughput and high efficiency. The novel Microfluidic Droplet-based Electroporation System (uDES) is developed to efficiently load cargos into EVs via millisecond pulsed, low-voltage electroporation within flow-through droplets as enormous bioreactors in a continuous-flow and scalable manner. The observed loading efficiency of CRISPR/Cas9 RNA complex into EVs (RNP-EVs) is 10-fold higher than current bulk cuvette electroporation with hundred-fold increase of processing throughput. The low-voltage electroporation minimized the Joule heating influence on nanosized EVs, which retained the native surface membrane properties of cargo-loaded EVs. Both ex vivo and in vivo testing in Shaker-1 mice model demonstrated the high biocompatibility and biodistribution of produced RNP-EVs in the mouse cochlea penetrating inner hair cells. In contrast, the CRISPR/Cas9 RNP lipid-like nanoparticles (RNP-LNPs) control group was unable to penetrate anatomical barriers to access inner hair cells. In the Shaker-1 mouse model, uDES produced RNP-EVs demonstrated much higher editing efficiency at Myo7ash1 mRNA level and showed significant hearing recovery in the Myo7aWT/Sh1 mice via Auditory Brainstem Response (ABR) testing. The report work will present a new solution to advance gene therapy in treating sensorineural hearing loss.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557278">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557278" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557278">
        <p class="paperTitle">The colocatome as a spatial -omic reveals shared microenvironment features between tumour-stroma assembloids and lung cancer specimens</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557278" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557278" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bouchard, G.; Zhang, W.; Li, I.; Ilerten, I.; Shrager, J. B.; Bhattacharya, A.; Li, Y.; Trope, W.; Osawa, M. G.; Kuo, C.; Tian, L.; Giaccia, A. J.; Plevritis, S. K.</p>
        <p class="info">Score: 3.0, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557278' target='https://doi.org/10.1101/2023.09.11.557278'> 10.1101/2023.09.11.557278</a></p>
        <p class="abstract">Computational frameworks to quantify and compare microenvironment spatial features of in-vitro patient-derived models and clinical specimens are needed. Here, we acquired and analysed multiplexed immunofluorescence images of human lung adenocarcinoma (LUAD) alongside tumour- stroma assembloids constructed with organoids and fibroblasts harvested from the leading edge (Tumour-Adjacent Fibroblasts;TAFs) or core (Tumour Core Fibroblasts;TCFs) of human LUAD. We introduce the concept of the &#34;colocatome&#34; as a spatial -omic dimension to catalogue all proximate and distant colocalisations between malignant and fibroblast subpopulations in both the assembloids and clinical specimens. The colocatome expands upon the colocalisation quotient (CLQ) through a nomalisation strategy that involves permutation analysis and thereby allows comparisons of CLQs under different conditions. Using colocatome analysis, we report that both TAFs and TCFs protected cancer cells from targeted oncogene treatment by uniquely reorganising the tumour-stroma cytoarchitecture, rather than by promoting cellular heterogeneity or selection. Moreover, we show that the assembloids colocatome recapitulates the tumour-stroma cytoarchitecture defining the tumour microenvironment of LUAD clinical samples and thereby can serve as a functional spatial readout to guide translational discoveries.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=190 SRC=&#34;FIGDIR/small/557278v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (60K):
org.highwire.dtl.DTLVardef@7bf7a1org.highwire.dtl.DTLVardef@be7274org.highwire.dtl.DTLVardef@14c5572org.highwire.dtl.DTLVardef@7dd5fd_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556747">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556747" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556747">
        <p class="paperTitle">Image-based parametric finite element modelling for studying contact mechanics in human knee joints</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556747" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556747" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Readioff, R.; Seil, R.; Mouton, C.; Marks, L.; Barrera, O.</p>
        <p class="info">Score: 2.8, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556747' target='https://doi.org/10.1101/2023.09.07.556747'> 10.1101/2023.09.07.556747</a></p>
        <p class="abstract">PurposeThis study presents a framework for generating patient-specific finite element models, parameterised and optimised for contact mechanics from computed tomography (CT) scans, by avoiding the segmentation step usually employed to transform medical images into 3D models. Two morphological parameters affecting contact mechanics were investigated in the framework development: tibial cartilage thickness and tibial spine height. This study explores the effect of the interplay of these parameters in load sharing between meniscus and articulating cartilage, meniscal posterior and anterior roots strain and menisci kine-matics.

MethodsMorphological measurements from four knee CT scans were collected, such as the maximum thickness of the tibial cartilage (ranging from 1.1 to 5.2 mm), the height of the tibial spine (ranging from 3.55 to 10.1 mm), and the width of the tibial plateau in both the coronal (ranging from 27.3 to 36.17 mm) and sagittal (ranging from 31.79 to 53.77 mm) planes. These measurements were taken for the lateral tibial plateau for both left and right knees. Subsequently, three finite element (FE) models were generated, comprising lateral tibial plateaus, lateral femoral condyle and lateral meniscus. The tibial cartilage thickness was kept at a constant value of 1 mm while varying the tibial spine height within the range measured from the CT images. This resulted in three FE models with varying spine heights, categorised as large (height = 7.42 mm), medium (height = 4.25 mm), and small (height = 1.63 mm) tibial spine heights. The menisci in the FE models were generated to be congruent with the tibial plateau. For the first time, this study advances the representations of the knee menisci microstructure in FE modelling, such that we have generated meniscus FE models with three layers of a hyperelastic model in which layer thickness and layer-specific hyperelastic material parameters are derived from our previous experimental work.

ResultsThe load sharing between the meniscus and articular cartilage was not sensitive to the varying tibial spine heights. In all three FE models, cartilage carried more than 90% of the applied load. However, the meniscus kinematics and root strains varied considerably with changing tibial spine heights. The small tibial spine height model predicted the highest meniscus movements (8.12 and 9.33 mm in the radial and circumferential directions, respectively) and the highest root strain (21.92 and 22.19 mm/mm in the anterior and posterior roots, respectively).

ConclusionOur framework can generate finite element models of patients knees using clinical data (i.e., CT scans) without the need for lengthy image segmentation. This process is not only time-efficient but also independent of imaging operators. The models converge quickly ({inverted question}30 minutes on 2 cores) using an implicit solver with non-linear geometry and have the capability to predict contact mechanics between the articulating surfaces, meniscus kinematics and root strains. The modelling strategy presented here can provide valuable insights into predicting changes in the mechanics of soft tissues in the knee joint. It is particularly useful for investigating injury and surgical mechanisms related to the meniscus.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
