<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Simultaneous multi-site editing of individual genomes using retron arrays
Authors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.
Score: 23.8, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549397
Our understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Simultaneous multi-site editing of individual genomes using retron arrays
Authors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.
Score: 23.8, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549397
Our understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-19T10:38:24+00:00" />
<meta property="article:modified_time" content="2023-07-19T10:38:24+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Simultaneous multi-site editing of individual genomes using retron arrays
Authors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.
Score: 23.8, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549397
Our understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Simultaneous multi-site editing of individual genomes using retron arrays\nAuthors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.\nScore: 23.8, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549397\nOur understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites.",
  "keywords": [
    
  ],
  "articleBody": " Simultaneous multi-site editing of individual genomes using retron arrays\nAuthors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.\nScore: 23.8, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549397\nOur understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites. Unfortunately, building cells with multiple, non-adjacent precise mutations remains a laborious cycle of editing, isolating an edited cell, and editing again. Here, we describe a technology for precisely modifying multiple sites on a single genome simultaneously. This technology - termed a multitron - is built from a heavily modified retron, in which multiple donor-encoding msds are produced from a single transcript. The multitron architecture is compatible with both recombineering in prokaryotic cells and CRISPR editing in eukaryotic cells. We demonstrate applications for this approach in molecular recording, genetic element minimization, and metabolic engineering.\nSingle cell susceptibility to SARS-CoV-2 infection is driven by variable cell states\nAuthors: Reffsin, S.; Miller, J.; Ayyanathan, K.; Dunagin, M. C.; Jain, N.; Schultz, D. C.; Cherry, S.; Raj, A.\nScore: 29.8, Published: 2023-07-07 DOI: 10.1101/2023.07.06.547955\nThe ability of a virus to infect a cell type is at least in part determined by the presence of host factors required for the viral life cycle. However, even within cell types that express known factors needed for infection, not every cell is equally susceptible, suggesting that our knowledge of the full spectrum of factors that promote infection is incomplete. Profiling the most susceptible subsets of cells within a population may reveal additional factors that promote infection. However, because viral infection dramatically alters the state of the cell, new approaches are needed to reveal the state of these cells prior to infection with virus. Here, we used single-cell clone tracing to retrospectively identify and characterize lung epithelial cells that are highly susceptible to infection with SARS-CoV-2. The transcriptional state of these highly susceptible cells includes markers of retinoic acid signaling and epithelial differentiation. Loss of candidate factors identified by our approach revealed that many of these factors play roles in viral entry. Moreover, a subset of these factors exert control over the infectable cell state itself, regulating the expression of key factors associated with viral infection and entry. Analysis of patient samples revealed the heterogeneous expression of these factors across both cells and patients in vivo. Further, the expression of these factors is upregulated in particular inflammatory pathologies. Altogether, our results show that the variable expression of intrinsic cell states is a major determinant of whether a cell can be infected by SARS-CoV-2.\nPyLabRobot: An Open-Source, Hardware Agnostic Interface for Liquid-Handling Robots and Accessories\nAuthors: Wierenga, R.; Golas, S.; Ho, W.; Coley, C.; Esvelt, K.\nScore: 14.5, Published: 2023-07-15 DOI: 10.1101/2023.07.10.547733\nLiquid handling robots are often limited by proprietary programming interfaces that are only compatible with a single type of robot and operating system, restricting method sharing and slowing development. Here we present PyLabRobot, an open-source, cross-platform Python interface capable of programming diverse liquid-handling robots, including Hamilton STARs, Tecan EVOs, and Opentron OT-2s. PyLabRobot provides a universal set of commands and representations for deck layout and labware, enabling the control of diverse accessory devices. The interface is extensible and can work with any robot that manipulates liquids within a Cartesian coordinate system. We validated the system through unit tests and several application demonstrations, including a browser-based simulator, a position calibration tool, and a path-teaching tool for complex movements. PyLabRobot provides a flexible, open, and collaborative programming environment for laboratory automation. Figure Abstract O_FIG O_LINKSMALLFIG WIDTH=124 HEIGHT=200 SRC=\"FIGDIR/small/547733v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (34K): org.highwire.dtl.DTLVardef@a21b41org.highwire.dtl.DTLVardef@ab6365org.highwire.dtl.DTLVardef@abcdborg.highwire.dtl.DTLVardef@f4e160_HPS_FORMAT_FIGEXP M_FIG C_FIG PyLabRobot overcomes the limitations of proprietary robotic systems. (a) Scientists with access to liquid-handling robots are currently limited by proprietary interfaces that require specialized knowledge, hinder cross-platform operability, and restrict sharing of methods among different robot types. For complex tasks, many researchers need assistance from a specialist familiar with their particular system, most notably when creating or editing protocols. (b) PyLabRobot (https://github.com/PyLabRobot/pylabrobot) offers a single interface that allows any person with basic Python skills to program diverse types of liquid-handling robots and share protocols freely, fostering a more collaborative and efficient research environment. The Python API makes it easy to interact with a large scientific computing ecosystem and allows users to leverage large language models for programming assistance.\nIdentifying novel radioprotective drugs via salivary gland tissue chip screening\nAuthors: Piraino, L.; Chen, C.; Mereness, J.; Dunman, P.; Ovitt, C.; Benoit, D.; DeLouise, L. A.\nScore: 10.7, Published: 2023-07-13 DOI: 10.1101/2023.07.12.548707\nDuring head and neck cancer treatment, off-target ionizing radiation damage to the salivary glands commonly causes a permanent loss of secretory function. Due to the resulting decrease in saliva production, patients have trouble eating, speaking and are predisposed to oral infections and tooth decay. While the radioprotective antioxidant drug Amifostine is approved to prevent radiation-induced hyposalivation, it has intolerable side effects that limit its use, motivating the discovery of alternative therapeutics. To address this issue, we previously developed a salivary gland mimetic (SGm) tissue chip platform. Here, we leverage this SGm tissue chip for high-content drug discovery. First, we developed in-chip assays to quantify glutathione and cellular senescence ({beta}-galactosidase), which are biomarkers of radiation damage, and we validated radioprotection using WR-1065, the active form of Amifostine. Following validation, we tested other reported radioprotective drugs, including, Edaravone, Tempol, N-acetylcysteine (NAC), Rapamycin, Ex-Rad, and Palifermin, confirming that all drugs but NAC and Ex-Rad exhibited robust radioprotection. Next, a Selleck Chemicals library of 438 FDA-approved drugs was screened for radioprotection. We discovered 25 hits, with most of the drugs identified with mechanisms of action other than antioxidant activity. Hits were down-selected using EC50 values and pharmacokinetics and pharmacodynamics data from the PubChem database leading to testing of Phenylbutazone (anti-inflammatory), Enoxacin (antibiotic), and Doripenem (antibiotic) for in vivo radioprotection in mice using retroductal injections. Results confirm that Phenylbutazone and Enoxacin exhibited equivalent radioprotection to Amifostine. This body of work demonstrates the development and validation of assays using a SGm tissue chip platform for high-content drug screening and the successful in vitro discovery and in vivo validation of novel radioprotective drugs with nonantioxidant primary indications pointing to possible, yet unknown novel mechanisms of radioprotection.\nIn vivo fate of systemically administered encapsulin protein nanocages and implications for their use in targeted drug delivery\nAuthors: Rennie, C.; Sives, C.; Boyton, I.; Diaz, D.; Gorrie, C. A.; Vittorio, O.; Collins-Praino, L.; Care, A.\nScore: 6.6, Published: 2023-07-18 DOI: 10.1101/2023.07.16.549228\nEncapsulins, self-assembling protein nanocages derived from prokaryotes, are promising nanoparticle-based drug delivery systems (NDDS). However, the in vivo behavior and fate of encapsulins are poorly understood. In this pre-clinical study, we probe the interactions between the model encapsulin from Thermotoga maritima (TmEnc) and key biological barriers encountered by NDDS. Here, a purified TmEnc formulation that exhibited colloidal stability, storability, and blood compatibility was intravenously injected into BALB/c mice. TmEnc had an excellent nanosafety profile, with no abnormal weight loss or gross pathology observed, and only temporary alterations in toxicity biomarkers detected. Notably, TmEnc demonstrated immunogenic properties, inducing the generation of nanocage-specific IgM and IgG antibodies but without any prolonged pro-inflammatory effects. An absence of antibody cross-reactivity also suggested immune-orthogonality among encapsulins systems. Moreover, TmEnc formed a serum-derived protein corona on its surface which changed dynamically and appeared to play a role in immune recognition. TmEnc's biodistribution profile further revealed its sequestration from the blood circulation by the liver and then biodegraded within Kupffer cells, thus indicating clearance via the mononuclear phagocyte system. Collectively, these findings provide critical insights into how encapsulins behave in vivo, thereby informing their future design, modification, and application in targeted drug delivery.\nSolid tumor-on-chip model for efficacy and safety assessment of CAR-T cell therapy\nAuthors: Maulana, T. I.; Teufel, C.; Cipriano, M.; Lazarevski, L.; van den Hil, F. E.; Orlova, V.; Koch, A.; Alb, M.; Hudecek, M.; Loskill, P.\nScore: 4.6, Published: 2023-07-15 DOI: 10.1101/2023.07.13.548856\nThe non-clinical assessment of CAR-T cells demands innovative models that are capable of predicting safety and efficacy in the clinical setting. Here, we present a novel solid tumor-on-chip model that allows CAR-T cell perfusion and integrates the vasculature and tumor lesions to recapitulate key events of CAR-T cell performance including extravasation, tumor infiltration and cytokine release. We assessed CAR-T cells targeting the ROR1 antigen against tumor aggregates that were derived from a breast cancer cell line and primary breast cancer organoids. The data show the temporal kinetic of ROR1 CAR-T cell migration and expansion, lytic activity and cytokine production over the course of 8 days, and reveal a correlation between anti-tumor efficacy and ROR1 antigen density on tumor cells. CAR-modified T cells extravasated faster, infiltrated tumor lesions stronger, persisted longer and in higher numbers than non-CAR modified T cells. Intriguingly, we detected cytokine release levels and kinetics typically observed in patients who developed cytokine release syndrome, and administered dasatinib as a pharmacologic OFF switch to control this inflammatory response. The data illustrate the ability of this tumor-on-chip platform to assess parameters associated with therapeutic outcome and the potential to aid in patient stratification and monitoring of CAR-T cell therapy.\nMicrofluidic Quaking-Induced Conversion (Micro-QuIC) for Rapid On-Site Amplification and Detection of Misfolded Proteins\nAuthors: Lee, D. J.; Christenson, P. R.; Rowden, G.; Lindquist, N. C.; Larsen, P. A.; Oh, S.-H.\nScore: 3.9, Published: 2023-07-18 DOI: 10.1101/2023.07.17.549283\nProtein misfolding diseases, such as prion diseases, Alzheimer's, and Parkinson's, share a common molecular mechanism involving the misfolding and aggregation of specific proteins. There is an urgent need for point-of-care (POC) diagnostic technologies that can accurately detect these misfolded proteins, facilitating early diagnosis and intervention. Here, we introduce the Microfluidic Quaking Induced Conversion (Micro-QuIC), a novel acoustofluidic platform for the rapid and sensitive detection of protein misfolding diseases. We demonstrate the utility of our technology using chronic wasting disease (CWD) as a model system, as samples from wild white-tailed deer are readily accessible, and CWD shares similarities with human protein misfolding diseases. Acoustofluidic mixing enables homogeneous mixing of reagents in a high-Reynolds-number regime, significantly accelerating the turnaround time for CWD diagnosis. Our Micro-QuIC assay amplifies prions by an order of magnitude faster than the current gold standard, real-time quaking-induced conversion (RT-QuIC). Furthermore, we integrated Micro-QuIC with a gold nanoparticle-based, naked-eye detection method, which enables visual discrimination between CWD positive and negative samples without the need for a bulky fluorescence detection module. This integration creates a rapid, POC testing platform capable of detecting misfolded proteins associated with a variety of protein misfolding diseases.\nBiosensor-Assisted Laboratory Evolution of Malonyl-CoA production in Saccharomyces cerevisiae\nAuthors: Ferreira, R.; Hedin, K. A.; Nielsen, J.; David, F.\nScore: 3.9, Published: 2023-07-17 DOI: 10.1101/2023.07.16.549225\nThe production of bio-based chemicals and fuels through microbial engineering offers a promising and sustainable alternative to petroleum-based fuels and chemicals, with the potential for scalability. However, engineering microbes and continuously evolving them to enhance the production of industrially relevant products is a complex and challenging task, requiring precise selection of genetic traits to achieve desired outcomes. In this study, we report the development of a novel counter-selectable growth-sensitive malonyl-CoA platform strain by coupling the malonyl-CoA repressor FapR from Bacillus subtilis to essential gene promoters involved in glucose growth and the plasma membrane arginine permease. This platform strain was then coupled with a CRISPR-dCas9 guide-RNA (gRNA) library, which after multiple rounds of dilutions and library sequencing, resulted in the enrichment for gRNAs that increased fluxes towards malonyl-CoA. The enriched gRNAs were validated for their effects on growth enhancement, gene regulation, and the production of an industrially relevant malonyl-CoA product, namely 3-hydroxypropionic acid. This study highlights an innovative approach to microbial engineering and opens up avenues for further exploration in the field of laboratory continuous evolution.\nMicroengineering 3D Collagen Matrices with Tumor-Mimetic Gradients in Fiber Alignment\nAuthors: Joshi, I. M.; Mansouri, M.; Ahmed, A.; Simon, R.; Esmaili, P. B.; Desa, D. E.; Elias, T.; Brown, E.; Abhyankar, V. V.\nScore: 5.0, Published: 2023-07-10 DOI: 10.1101/2023.07.09.548253\nIn the tumor microenvironment (TME), collagen fibers facilitate tumor cell migration through the extracellular matrix. Previous studies have focused on studying the responses of cells on uniformly aligned or randomly aligned collagen fibers. However, the in vivo environment also features spatial gradients in alignment, which arise from the local reorganization of the matrix architecture due to cell-induced traction forces. Although there has been extensive research on how cells respond to graded biophysical cues, such as stiffness, porosity, and ligand density, the cellular responses to physiological fiber alignment gradients have been largely unexplored. This is due, in part, to a lack of robust experimental techniques to create controlled alignment gradients in natural materials. In this study, we image tumor biopsy samples and characterize the alignment gradients present in the TME. To replicate physiological gradients, we introduce a first-of-its-kind biofabrication technique that utilizes a microfluidic channel with constricting and expanding geometry to engineer 3D collagen hydrogels with tunable fiber alignment gradients that range from sub-millimeter to millimeter length scales. Our modular approach allows easy access to the microengineered gradient gels, and we demonstrate that HUVECs migrate in response to the fiber architecture. We provide preliminary evidence suggesting that MDA-MB-231 cell aggregates, patterned onto a specific location on the alignment gradient, exhibit preferential migration towards increasing alignment. This finding suggests that alignment gradients could serve as an additional taxis cue in the ECM. Importantly, our study represents the first successful engineering of continuous gradients of fiber alignment in soft, natural materials. We anticipate that our user-friendly platform, which needs no specialized equipment, will offer new experimental capabilities to study the impact of fiber-based contact guidance on directed cell migration.\nAcidic nanoparticles restore lysosomal acidification and rescue metabolic dysfunction in pancreatic β-cells under lipotoxic condition\nAuthors: Lo, C. H.; O'Connor, L. M.; Loi, G. W. Z.; Saipuljumri, E. N.; Indajang, J.; Lopes, K. M.; Shirihai, O. S.; Grinstaff, M. W.; Zeng, J.\nScore: 3.0, Published: 2023-07-12 DOI: 10.1101/2023.07.11.548395\nType 2 diabetes (T2D) is one of the most common metabolic disorders worldwide without effective treatments. Lysosomal acidification impairment, autophagy defect, and mitochondrial dysfunction all associate with T2D, although the exact pathogenic mechanisms remain under investigation. Here, we report a series of biodegradable poly(butylene tetrafluorosuccinate-co-succinate) (PBFSU) polyesters consisting of an 1,4-butanediol linker and varying ratios of tetrafluorosuccinic acid (TFSA) and succinic acid (SA) as component acids to engineer new lysosome acidifying nanoparticles (NPs). The TFSA component significantly modulates the degradation rate and acidification capability of the PBFSU NPs, where NPs consisting entirely of TFSA, which we termed TFSA NPs, re-acidify impaired lysosomes with the strongest reduction in luminal pH. Further, we show that in both mRNA expression of human T2D pancreatic islets and protein expression in INS-1 {beta}-cells under lipotoxic condition that there is a significant downregulation of genes associated with lysosomal vacuolar (H+)-ATPase (V-ATPase) subunits responsible for maintaining lysosomal acidification. Treatment of TFSA NPs mediates palmitate-induced lipotoxicity in INS-1 {beta}-cells by restoring lysosomal acidification, autophagic function, and mitochondria functions, as well as promoting glucose-stimulated insulin secretion. Administration of TFSA NPs to high-fat diet T2D mice improves glucose clearance and reduces insulin resistance. Together, the data document the therapeutic potential of lysosome acidifying TFSA NPs for T2D. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=168 SRC=\"FIGDIR/small/548395v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (42K): org.highwire.dtl.DTLVardef@1de9456org.highwire.dtl.DTLVardef@1dcb6c1org.highwire.dtl.DTLVardef@162755aorg.highwire.dtl.DTLVardef@1069fd0_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "2547",
  "inLanguage": "en",
  "datePublished": "2023-07-19T10:38:24Z",
  "dateModified": "2023-07-19T10:38:24Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on July 19, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549397">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549397" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549397">
        <p class="paperTitle">Simultaneous multi-site editing of individual genomes using retron arrays</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549397" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549397" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.</p>
        <p class="info">Score: 23.8, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549397' target='https://doi.org/10.1101/2023.07.17.549397'> 10.1101/2023.07.17.549397</a></p>
        <p class="abstract">Our understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites. Unfortunately, building cells with multiple, non-adjacent precise mutations remains a laborious cycle of editing, isolating an edited cell, and editing again. Here, we describe a technology for precisely modifying multiple sites on a single genome simultaneously. This technology - termed a multitron - is built from a heavily modified retron, in which multiple donor-encoding msds are produced from a single transcript. The multitron architecture is compatible with both recombineering in prokaryotic cells and CRISPR editing in eukaryotic cells. We demonstrate applications for this approach in molecular recording, genetic element minimization, and metabolic engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.547955">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.547955" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.547955">
        <p class="paperTitle">Single cell susceptibility to SARS-CoV-2 infection is driven by variable cell states</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.547955" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.547955" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reffsin, S.; Miller, J.; Ayyanathan, K.; Dunagin, M. C.; Jain, N.; Schultz, D. C.; Cherry, S.; Raj, A.</p>
        <p class="info">Score: 29.8, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.547955' target='https://doi.org/10.1101/2023.07.06.547955'> 10.1101/2023.07.06.547955</a></p>
        <p class="abstract">The ability of a virus to infect a cell type is at least in part determined by the presence of host factors required for the viral life cycle. However, even within cell types that express known factors needed for infection, not every cell is equally susceptible, suggesting that our knowledge of the full spectrum of factors that promote infection is incomplete. Profiling the most susceptible subsets of cells within a population may reveal additional factors that promote infection. However, because viral infection dramatically alters the state of the cell, new approaches are needed to reveal the state of these cells prior to infection with virus. Here, we used single-cell clone tracing to retrospectively identify and characterize lung epithelial cells that are highly susceptible to infection with SARS-CoV-2. The transcriptional state of these highly susceptible cells includes markers of retinoic acid signaling and epithelial differentiation. Loss of candidate factors identified by our approach revealed that many of these factors play roles in viral entry. Moreover, a subset of these factors exert control over the infectable cell state itself, regulating the expression of key factors associated with viral infection and entry. Analysis of patient samples revealed the heterogeneous expression of these factors across both cells and patients in vivo. Further, the expression of these factors is upregulated in particular inflammatory pathologies. Altogether, our results show that the variable expression of intrinsic cell states is a major determinant of whether a cell can be infected by SARS-CoV-2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.547733">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.547733" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.547733">
        <p class="paperTitle">PyLabRobot: An Open-Source, Hardware Agnostic Interface for Liquid-Handling Robots and Accessories</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.547733" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.547733" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wierenga, R.; Golas, S.; Ho, W.; Coley, C.; Esvelt, K.</p>
        <p class="info">Score: 14.5, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.547733' target='https://doi.org/10.1101/2023.07.10.547733'> 10.1101/2023.07.10.547733</a></p>
        <p class="abstract">Liquid handling robots are often limited by proprietary programming interfaces that are only compatible with a single type of robot and operating system, restricting method sharing and slowing development. Here we present PyLabRobot, an open-source, cross-platform Python interface capable of programming diverse liquid-handling robots, including Hamilton STARs, Tecan EVOs, and Opentron OT-2s. PyLabRobot provides a universal set of commands and representations for deck layout and labware, enabling the control of diverse accessory devices. The interface is extensible and can work with any robot that manipulates liquids within a Cartesian coordinate system. We validated the system through unit tests and several application demonstrations, including a browser-based simulator, a position calibration tool, and a path-teaching tool for complex movements. PyLabRobot provides a flexible, open, and collaborative programming environment for laboratory automation.

Figure Abstract

O_FIG O_LINKSMALLFIG WIDTH=124 HEIGHT=200 SRC=&#34;FIGDIR/small/547733v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (34K):
org.highwire.dtl.DTLVardef@a21b41org.highwire.dtl.DTLVardef@ab6365org.highwire.dtl.DTLVardef@abcdborg.highwire.dtl.DTLVardef@f4e160_HPS_FORMAT_FIGEXP  M_FIG C_FIG PyLabRobot overcomes the limitations of proprietary robotic systems. (a) Scientists with access to liquid-handling robots are currently limited by proprietary interfaces that require specialized knowledge, hinder cross-platform operability, and restrict sharing of methods among different robot types. For complex tasks, many researchers need assistance from a specialist familiar with their particular system, most notably when creating or editing protocols. (b) PyLabRobot (https://github.com/PyLabRobot/pylabrobot) offers a single interface that allows any person with basic Python skills to program diverse types of liquid-handling robots and share protocols freely, fostering a more collaborative and efficient research environment. The Python API makes it easy to interact with a large scientific computing ecosystem and allows users to leverage large language models for programming assistance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548707">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548707" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548707">
        <p class="paperTitle">Identifying novel radioprotective drugs via salivary gland tissue chip screening</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548707" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548707" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Piraino, L.; Chen, C.; Mereness, J.; Dunman, P.; Ovitt, C.; Benoit, D.; DeLouise, L. A.</p>
        <p class="info">Score: 10.7, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548707' target='https://doi.org/10.1101/2023.07.12.548707'> 10.1101/2023.07.12.548707</a></p>
        <p class="abstract">During head and neck cancer treatment, off-target ionizing radiation damage to the salivary glands commonly causes a permanent loss of secretory function. Due to the resulting decrease in saliva production, patients have trouble eating, speaking and are predisposed to oral infections and tooth decay. While the radioprotective antioxidant drug Amifostine is approved to prevent radiation-induced hyposalivation, it has intolerable side effects that limit its use, motivating the discovery of alternative therapeutics. To address this issue, we previously developed a salivary gland mimetic (SGm) tissue chip platform. Here, we leverage this SGm tissue chip for high-content drug discovery. First, we developed in-chip assays to quantify glutathione and cellular senescence ({beta}-galactosidase), which are biomarkers of radiation damage, and we validated radioprotection using WR-1065, the active form of Amifostine. Following validation, we tested other reported radioprotective drugs, including, Edaravone, Tempol, N-acetylcysteine (NAC), Rapamycin, Ex-Rad, and Palifermin, confirming that all drugs but NAC and Ex-Rad exhibited robust radioprotection. Next, a Selleck Chemicals library of 438 FDA-approved drugs was screened for radioprotection. We discovered 25 hits, with most of the drugs identified with mechanisms of action other than antioxidant activity. Hits were down-selected using EC50 values and pharmacokinetics and pharmacodynamics data from the PubChem database leading to testing of Phenylbutazone (anti-inflammatory), Enoxacin (antibiotic), and Doripenem (antibiotic) for in vivo radioprotection in mice using retroductal injections. Results confirm that Phenylbutazone and Enoxacin exhibited equivalent radioprotection to Amifostine. This body of work demonstrates the development and validation of assays using a SGm tissue chip platform for high-content drug screening and the successful in vitro discovery and in vivo validation of novel radioprotective drugs with nonantioxidant primary indications pointing to possible, yet unknown novel mechanisms of radioprotection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.549228">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.549228" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.549228">
        <p class="paperTitle">In vivo fate of systemically administered encapsulin protein nanocages and implications for their use in targeted drug delivery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.549228" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.549228" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rennie, C.; Sives, C.; Boyton, I.; Diaz, D.; Gorrie, C. A.; Vittorio, O.; Collins-Praino, L.; Care, A.</p>
        <p class="info">Score: 6.6, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.549228' target='https://doi.org/10.1101/2023.07.16.549228'> 10.1101/2023.07.16.549228</a></p>
        <p class="abstract">Encapsulins, self-assembling protein nanocages derived from prokaryotes, are promising nanoparticle-based drug delivery systems (NDDS). However, the in vivo behavior and fate of encapsulins are poorly understood. In this pre-clinical study, we probe the interactions between the model encapsulin from Thermotoga maritima (TmEnc) and key biological barriers encountered by NDDS. Here, a purified TmEnc formulation that exhibited colloidal stability, storability, and blood compatibility was intravenously injected into BALB/c mice. TmEnc had an excellent nanosafety profile, with no abnormal weight loss or gross pathology observed, and only temporary alterations in toxicity biomarkers detected. Notably, TmEnc demonstrated immunogenic properties, inducing the generation of nanocage-specific IgM and IgG antibodies but without any prolonged pro-inflammatory effects. An absence of antibody cross-reactivity also suggested immune-orthogonality among encapsulins systems. Moreover, TmEnc formed a serum-derived protein corona on its surface which changed dynamically and appeared to play a role in immune recognition. TmEnc&#39;s biodistribution profile further revealed its sequestration from the blood circulation by the liver and then biodegraded within Kupffer cells, thus indicating clearance via the mononuclear phagocyte system. Collectively, these findings provide critical insights into how encapsulins behave in vivo, thereby informing their future design, modification, and application in targeted drug delivery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548856">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548856" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548856">
        <p class="paperTitle">Solid tumor-on-chip model for efficacy and safety assessment of CAR-T cell therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548856" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548856" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Maulana, T. I.; Teufel, C.; Cipriano, M.; Lazarevski, L.; van den Hil, F. E.; Orlova, V.; Koch, A.; Alb, M.; Hudecek, M.; Loskill, P.</p>
        <p class="info">Score: 4.6, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548856' target='https://doi.org/10.1101/2023.07.13.548856'> 10.1101/2023.07.13.548856</a></p>
        <p class="abstract">The non-clinical assessment of CAR-T cells demands innovative models that are capable of predicting safety and efficacy in the clinical setting. Here, we present a novel solid tumor-on-chip model that allows CAR-T cell perfusion and integrates the vasculature and tumor lesions to recapitulate key events of CAR-T cell performance including extravasation, tumor infiltration and cytokine release. We assessed CAR-T cells targeting the ROR1 antigen against tumor aggregates that were derived from a breast cancer cell line and primary breast cancer organoids. The data show the temporal kinetic of ROR1 CAR-T cell migration and expansion, lytic activity and cytokine production over the course of 8 days, and reveal a correlation between anti-tumor efficacy and ROR1 antigen density on tumor cells. CAR-modified T cells extravasated faster, infiltrated tumor lesions stronger, persisted longer and in higher numbers than non-CAR modified T cells. Intriguingly, we detected cytokine release levels and kinetics typically observed in patients who developed cytokine release syndrome, and administered dasatinib as a pharmacologic OFF switch to control this inflammatory response. The data illustrate the ability of this tumor-on-chip platform to assess parameters associated with therapeutic outcome and the potential to aid in patient stratification and monitoring of CAR-T cell therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549283">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549283" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549283">
        <p class="paperTitle">Microfluidic Quaking-Induced Conversion (Micro-QuIC) for Rapid On-Site Amplification and Detection of Misfolded Proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549283" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549283" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, D. J.; Christenson, P. R.; Rowden, G.; Lindquist, N. C.; Larsen, P. A.; Oh, S.-H.</p>
        <p class="info">Score: 3.9, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549283' target='https://doi.org/10.1101/2023.07.17.549283'> 10.1101/2023.07.17.549283</a></p>
        <p class="abstract">Protein misfolding diseases, such as prion diseases, Alzheimer&#39;s, and Parkinson&#39;s, share a common molecular mechanism involving the misfolding and aggregation of specific proteins. There is an urgent need for point-of-care (POC) diagnostic technologies that can accurately detect these misfolded proteins, facilitating early diagnosis and intervention. Here, we introduce the Microfluidic Quaking Induced Conversion (Micro-QuIC), a novel acoustofluidic platform for the rapid and sensitive detection of protein misfolding diseases. We demonstrate the utility of our technology using chronic wasting disease (CWD) as a model system, as samples from wild white-tailed deer are readily accessible, and CWD shares similarities with human protein misfolding diseases. Acoustofluidic mixing enables homogeneous mixing of reagents in a high-Reynolds-number regime, significantly accelerating the turnaround time for CWD diagnosis. Our Micro-QuIC assay amplifies prions by an order of magnitude faster than the current gold standard, real-time quaking-induced conversion (RT-QuIC). Furthermore, we integrated Micro-QuIC with a gold nanoparticle-based, naked-eye detection method, which enables visual discrimination between CWD positive and negative samples without the need for a bulky fluorescence detection module. This integration creates a rapid, POC testing platform capable of detecting misfolded proteins associated with a variety of protein misfolding diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.549225">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.549225" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.549225">
        <p class="paperTitle">Biosensor-Assisted Laboratory Evolution of Malonyl-CoA production in Saccharomyces cerevisiae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.549225" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.549225" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ferreira, R.; Hedin, K. A.; Nielsen, J.; David, F.</p>
        <p class="info">Score: 3.9, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.549225' target='https://doi.org/10.1101/2023.07.16.549225'> 10.1101/2023.07.16.549225</a></p>
        <p class="abstract">The production of bio-based chemicals and fuels through microbial engineering offers a promising and sustainable alternative to petroleum-based fuels and chemicals, with the potential for scalability. However, engineering microbes and continuously evolving them to enhance the production of industrially relevant products is a complex and challenging task, requiring precise selection of genetic traits to achieve desired outcomes. In this study, we report the development of a novel counter-selectable growth-sensitive malonyl-CoA platform strain by coupling the malonyl-CoA repressor FapR from Bacillus subtilis to essential gene promoters involved in glucose growth and the plasma membrane arginine permease. This platform strain was then coupled with a CRISPR-dCas9 guide-RNA (gRNA) library, which after multiple rounds of dilutions and library sequencing, resulted in the enrichment for gRNAs that increased fluxes towards malonyl-CoA. The enriched gRNAs were validated for their effects on growth enhancement, gene regulation, and the production of an industrially relevant malonyl-CoA product, namely 3-hydroxypropionic acid. This study highlights an innovative approach to microbial engineering and opens up avenues for further exploration in the field of laboratory continuous evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.09.548253">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.09.548253" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.09.548253">
        <p class="paperTitle">Microengineering 3D Collagen Matrices with Tumor-Mimetic Gradients in Fiber Alignment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.09.548253" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.09.548253" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Joshi, I. M.; Mansouri, M.; Ahmed, A.; Simon, R.; Esmaili, P. B.; Desa, D. E.; Elias, T.; Brown, E.; Abhyankar, V. V.</p>
        <p class="info">Score: 5.0, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.09.548253' target='https://doi.org/10.1101/2023.07.09.548253'> 10.1101/2023.07.09.548253</a></p>
        <p class="abstract">In the tumor microenvironment (TME), collagen fibers facilitate tumor cell migration through the extracellular matrix. Previous studies have focused on studying the responses of cells on uniformly aligned or randomly aligned collagen fibers. However, the in vivo environment also features spatial gradients in alignment, which arise from the local reorganization of the matrix architecture due to cell-induced traction forces. Although there has been extensive research on how cells respond to graded biophysical cues, such as stiffness, porosity, and ligand density, the cellular responses to physiological fiber alignment gradients have been largely unexplored. This is due, in part, to a lack of robust experimental techniques to create controlled alignment gradients in natural materials. In this study, we image tumor biopsy samples and characterize the alignment gradients present in the TME. To replicate physiological gradients, we introduce a first-of-its-kind biofabrication technique that utilizes a microfluidic channel with constricting and expanding geometry to engineer 3D collagen hydrogels with tunable fiber alignment gradients that range from sub-millimeter to millimeter length scales. Our modular approach allows easy access to the microengineered gradient gels, and we demonstrate that HUVECs migrate in response to the fiber architecture. We provide preliminary evidence suggesting that MDA-MB-231 cell aggregates, patterned onto a specific location on the alignment gradient, exhibit preferential migration towards increasing alignment. This finding suggests that alignment gradients could serve as an additional taxis cue in the ECM. Importantly, our study represents the first successful engineering of continuous gradients of fiber alignment in soft, natural materials. We anticipate that our user-friendly platform, which needs no specialized equipment, will offer new experimental capabilities to study the impact of fiber-based contact guidance on directed cell migration.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548395">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548395" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548395">
        <p class="paperTitle">Acidic nanoparticles restore lysosomal acidification and rescue metabolic dysfunction in pancreatic β-cells under lipotoxic condition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548395" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548395" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lo, C. H.; O&#39;Connor, L. M.; Loi, G. W. Z.; Saipuljumri, E. N.; Indajang, J.; Lopes, K. M.; Shirihai, O. S.; Grinstaff, M. W.; Zeng, J.</p>
        <p class="info">Score: 3.0, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548395' target='https://doi.org/10.1101/2023.07.11.548395'> 10.1101/2023.07.11.548395</a></p>
        <p class="abstract">Type 2 diabetes (T2D) is one of the most common metabolic disorders worldwide without effective treatments. Lysosomal acidification impairment, autophagy defect, and mitochondrial dysfunction all associate with T2D, although the exact pathogenic mechanisms remain under investigation. Here, we report a series of biodegradable poly(butylene tetrafluorosuccinate-co-succinate) (PBFSU) polyesters consisting of an 1,4-butanediol linker and varying ratios of tetrafluorosuccinic acid (TFSA) and succinic acid (SA) as component acids to engineer new lysosome acidifying nanoparticles (NPs). The TFSA component significantly modulates the degradation rate and acidification capability of the PBFSU NPs, where NPs consisting entirely of TFSA, which we termed TFSA NPs, re-acidify impaired lysosomes with the strongest reduction in luminal pH. Further, we show that in both mRNA expression of human T2D pancreatic islets and protein expression in INS-1 {beta}-cells under lipotoxic condition that there is a significant downregulation of genes associated with lysosomal vacuolar (H&#43;)-ATPase (V-ATPase) subunits responsible for maintaining lysosomal acidification. Treatment of TFSA NPs mediates palmitate-induced lipotoxicity in INS-1 {beta}-cells by restoring lysosomal acidification, autophagic function, and mitochondria functions, as well as promoting glucose-stimulated insulin secretion. Administration of TFSA NPs to high-fat diet T2D mice improves glucose clearance and reduces insulin resistance. Together, the data document the therapeutic potential of lysosome acidifying TFSA NPs for T2D.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=168 SRC=&#34;FIGDIR/small/548395v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (42K):
org.highwire.dtl.DTLVardef@1de9456org.highwire.dtl.DTLVardef@1dcb6c1org.highwire.dtl.DTLVardef@162755aorg.highwire.dtl.DTLVardef@1069fd0_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
