<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Engineering mice for female-biased progeny without impacting genetic integrity and litter size
Authors: Yosef, I.; Mahata, T.; Chen, Y.; Bar-Joseph, H.; Shalgi, R.; Munitz, A.; Gerlic, M.; Qimron, U.
Score: 36.6, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568055
Generating mammalian gametes with a skewed sex ratio has thus far eluded empirical confirmation. The utilization of such genetically engineered organisms would offer the potential to curtail the necessity for culling animals of undesirable sex, mitigate resource wastage, and alleviate superfluous labor burdens.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Engineering mice for female-biased progeny without impacting genetic integrity and litter size
Authors: Yosef, I.; Mahata, T.; Chen, Y.; Bar-Joseph, H.; Shalgi, R.; Munitz, A.; Gerlic, M.; Qimron, U.
Score: 36.6, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568055
Generating mammalian gametes with a skewed sex ratio has thus far eluded empirical confirmation. The utilization of such genetically engineered organisms would offer the potential to curtail the necessity for culling animals of undesirable sex, mitigate resource wastage, and alleviate superfluous labor burdens." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-26T10:37:33+00:00" />
<meta property="article:modified_time" content="2023-11-26T10:37:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Engineering mice for female-biased progeny without impacting genetic integrity and litter size
Authors: Yosef, I.; Mahata, T.; Chen, Y.; Bar-Joseph, H.; Shalgi, R.; Munitz, A.; Gerlic, M.; Qimron, U.
Score: 36.6, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568055
Generating mammalian gametes with a skewed sex ratio has thus far eluded empirical confirmation. The utilization of such genetically engineered organisms would offer the potential to curtail the necessity for culling animals of undesirable sex, mitigate resource wastage, and alleviate superfluous labor burdens."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Engineering mice for female-biased progeny without impacting genetic integrity and litter size\nAuthors: Yosef, I.; Mahata, T.; Chen, Y.; Bar-Joseph, H.; Shalgi, R.; Munitz, A.; Gerlic, M.; Qimron, U.\nScore: 36.6, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568055\nGenerating mammalian gametes with a skewed sex ratio has thus far eluded empirical confirmation. The utilization of such genetically engineered organisms would offer the potential to curtail the necessity for culling animals of undesirable sex, mitigate resource wastage, and alleviate superfluous labor burdens.",
  "keywords": [
    
  ],
  "articleBody": " Engineering mice for female-biased progeny without impacting genetic integrity and litter size\nAuthors: Yosef, I.; Mahata, T.; Chen, Y.; Bar-Joseph, H.; Shalgi, R.; Munitz, A.; Gerlic, M.; Qimron, U.\nScore: 36.6, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568055\nGenerating mammalian gametes with a skewed sex ratio has thus far eluded empirical confirmation. The utilization of such genetically engineered organisms would offer the potential to curtail the necessity for culling animals of undesirable sex, mitigate resource wastage, and alleviate superfluous labor burdens. In this study, we introduce a transgenic male mouse lineage, which consistently yields predominantly female progeny (comprising up to 91% of the total offspring). This accomplishment was made possible by integrating a controllable genetic cassette onto the Y chromosome. The cassette encodes dCas9 and RNA guides that selectively silence a spermatid maturation gene. After the separation of X and Y gametes during meiosis, gametes containing an X chromosome develop normally, while those harboring the engineered Y chromosome, subjected to dCas9 silencing of the spermatid maturation gene, do not mature properly. Indeed, some spermatozoa from the transgenic mice exhibit a unique morphology, associated with the absence of the maturation gene. Notably, the resultant female offspring do not inherit the genetically engineered Y chromosome and are thus not genetically modified. Importantly, the litter size of the transgenic mice remains unchanged compared to the wild type. These findings represent a groundbreaking demonstration of genetic engineering's potential to yield sex-biased litters of full size without compromising genetic integrity, marking a pioneering advancement in this field of study.\nEngineering self-deliverable ribonucleoproteins for genome editing in the brain\nAuthors: Chen, K.; Stahl, E. C.; Kang, M. H.; Xu, B.; Allen, R.; Trinidad, M.; Doudna, J. A.\nScore: 25.4, Published: 2023-11-15 DOI: 10.1101/2023.11.15.567251\nThe delivery of CRISPR ribonucleoproteins (RNPs) for genome editing in vitro and in vivo has important advantages over other delivery methods, including reduced off-target and immunogenic effects1. However, effective delivery of RNPs remains challenging in certain cell types due to low efficiency and cell toxicity. To address these issues, we engineered self-deliverable RNPs that can promote efficient cellular uptake and carry out robust genome editing without the need for helper materials or biomolecules. Screening of cell-penetrating peptides (CPPs) fused to CRISPR-Cas9 protein identified potent constructs capable of efficient genome editing of neural progenitor cells. Further engineering of these fusion proteins identified a C-terminal Cas9 fusion with three copies of A22p, a peptide derived from human semaphorin-3a, that exhibited substantially improved editing efficacy compared to other constructs. We found that self-deliverable Cas9 RNPs generated robust genome edits in clinically relevant genes when injected directly into the mouse striatum. Overall, self-deliverable Cas9 proteins provide a facile and effective platform for genome editing in vitro and in vivo.\nPolymer-mediated oligonucleotide delivery enables construction of barcoded 3D cultures for spatial single-cell analysis\nAuthors: King, J. J.; Mowla, A.; Kretzmann, J. A.; Norret, M.; Kadolsky, U. D.; Iqbal, M.; Saxena, A.; Mamsa, S. S. A.; Amos, S. E.; Choi, Y. S.; Kennedy, B. F.; Iyer, K. S.; Smith, N. M.; Evans, C. W.\nScore: 7.1, Published: 2023-11-21 DOI: 10.1101/2023.11.20.567985\nSpatial transcriptomics has been widely used to capture gene expression profiles, realised as a two-dimensional (2D) projection of RNA captured from tissue sections. Wree-dimensional (3D) cultures such as spheroids and organoids are highly promising alternatives to oversimplified and homogeneous 2D cell culture models, but existing spatial transcriptomic platforms do not currently have sufficient resolution for robust analysis of 3D cultures. We present a transfection-based method for fluorescent DNA barcoding of cell populations, and the subsequent construction of spheroidal cellular architectures using barcoded cells in a layer-by-layer approach. For the first time, changes in gene expression throughout this 3D culture architecture are interrogated using multiplex single-cell RNA sequencing in which DNA barcodes are used to encode the spatial positioning of cells. We show that transfection with fluorophore-conjugated barcode oligonucleotides enables both imaging and sequencing at single-cell resolution, providing spatial maps of gene expression and drug response. Additionally, we show that fluorophore-conjugated DNA barcodes support correlative imaging studies such as mechano-microscopy to capture information about spatially-varying mechanical heterogeneity in 3D cultures. We ability to create customised, encoded cellular assemblies is a general approach that can resolve spatial differences in gene expression in 3D cell culture models.\nUnusual Structural Properties of Switchback DNA\nAuthors: Madhanagopal, B. R.; Talbot, H.; Rodriguez, A.; Louis, J. M.; Zeghal, H.; Vangaveti, S.; Reddy, K.; Chandrasekaran, A. R.\nScore: 7.0, Published: 2023-11-17 DOI: 10.1101/2023.11.15.563609\nSynthetic DNA motifs form the basis of nucleic acid nanotechnology, and their biochemical and biophysical properties determine their applications. Here, we present a detailed characterization of switchback DNA, a globally left-handed structure composed of two parallel DNA strands. Compared to a conventional duplex, switchback DNA shows lower thermodynamic stability and requires higher magnesium concentration for assembly, but exhibits a higher biostability against some nucleases. Strand competition and strand displacement experiments show that component sequences have an absolute preference for duplex complements instead of their switchback partners. Further, we hypothesize a potential role for switchback DNA as an alternate structure for short-tandem repeats involved in repeat-expansion diseases. Together with small molecule binding experiments and cell studies, our results open new avenues for synthetic DNA motifs in biology and nanotechnology.\nIsogenic iPSC-derived proximal and distal lung-on-chip models: Tissue- and virus-specific immune responses in human lungs\nAuthors: Yadav, S.; Fujimoto, K.; Takenaga, T.; Takahashi, S.; Muramoto, Y.; Mikawa, R.; Noda, T.; Gotoh, S.; Yokokawa, R.\nScore: 3.0, Published: 2023-11-24 DOI: 10.1101/2023.11.24.568532\nMicro-physiological systems (MPS) are set to play a vital role in preclinical studies, particularly in the context of future viral pandemics. Nonetheless, the development of MPS is often impeded by the scarcity of reliable cell sources, especially when seeking various organs or tissues from a single patient for comparative analysis of the host immune response. Herein, we developed human airway-on-chip and alveolus-on-chip models using induced pluripotent stem cell (iPSC)-derived isogenic lung progenitor cells. Both models demonstrated the replication of two different respiratory viruses, namely SARS-CoV-2 and Influenza, as well as related cellular damage and innate immune responses-on-chip. Our findings reveal distinct immune responses to SARS-CoV-2 in the proximal and distal lung-on-chip models. The airway chips exhibited a robust interferon (IFN)-dependent immune response, whereas the alveolus chips exhibited dysregulated IFN activation but a significantly upregulated chemokine pathway. In contrast, Influenza virus infection induced a more pronounced immune response and cellular damage in both chip models compared to SARS-CoV-2. Thus, iPSC-derived lung-on-chip models may aid in quickly gaining insights into viral pathology and screening potential drugs for future pandemics.\nHigh-throughput, label-free detection of DNA origami in single-cell suspensions using origamiFISH-Flow\nAuthors: Alexander, S.; Wang, W. X.; Tseng, C.-Y.; Douglas, T. R.; Chou, L. Y. T.\nScore: 10.8, Published: 2023-11-02 DOI: 10.1101/2023.10.31.564845\nStructural DNA nanotechnology enables custom fabrication of nanoscale devices and promises diverse biological applications. However, the effects of design on DNA nanostructure (DN)-cell interactions in vitro and in vivo are not yet well-characterized. origamiFISH is a recently developed technique for imaging DNs in cells and tissues. Compared to the use of fluorescent tags, origamiFISH offers label-free and structure-agnostic detection of DNs with significantly improved sensitivity. Here, we extend the origamiFISH technique to quantifying DNs in single-cell suspensions, including nonadherent cells such as subsets of immune cells, via readout by flow cytometry. This method, referred to as origamiFISH-Flow, is high-throughput (e.g., 10,000 cells per second) and compatible with immunostaining for concurrent cell-type and -state characterization. We demonstrate that origamiFISH-Flow enhances signal-to-noise ratio by up to 20-fold compared to dye labeling approaches, leading to the capture of \u003e25-fold more DN+ cells at low, single-picomolar DN uptake concentrations. We additionally show the use of origamiFISH-Flow to profile cell-type and shape-specific DN uptake patterns across cell lines and splenocytes and quantify in vivo DN accumulation in lymphoid organs. Together, origamiFISH-Flow offers a new tool to interrogate DN interactions with cells and tissues, while providing insights for tailoring their designs in bio-applications.\nMicroplate Format Protein Nanopatterning for High-Throughput Screening of Cellular Microenvironments\nAuthors: Shahrokhtash, A.; Sivertsen, M. v. T.; Laursen, S. H.; Sutherland, D. S.\nScore: 2.8, Published: 2023-11-19 DOI: 10.1101/2023.11.19.567703\nAn advanced protein nanopatterned cell culture platform is engineered to emulate the extracellular matrixs complexity, enabling precise nanoscale biomolecule copatterning to mimic environments analogous to native tissue for cellular assays. Nanopatterns fabricated through sparse colloidal lithography, with 100 nm to 800 nm features in separate wells, are seamlessly integrated into standard microplate formats (96-well/384-well). Robust patterns are built from fully PEGylated, passivated thin glass coverslips optimized for minimal nonspecific interactions. Biotin-avidin binding and click chemistry to ensure the accurate and robust localization of bioligands. The transparent, metal-free substrates are free of topographical interference, rendering them ideal for diverse fluorescence microscopy techniques encompassing single-molecule TIRFM and extensive high-throughput imaging. The structural stability of these nanopatterns persists beyond a year in storage and long-term in cell culture conditions, endorsing their application for prolonged experimental studies and potential for widespread academic and industrial use. The platform has been demonstrated for nanopatterning an array of biomolecules, from small molecules to proteins, DNA, and extracellular matrix components, instrumental for studying cell signaling. Experiments with C2C12 cells demonstrated the exceptional specificity of the nanopatterned microplates, with nonspecific adhesion remaining below 2% and the platforms ability to elicit size-dependent cellular reactions when interfaced with nanopatterned fibronectin.\nTranswell-based microphysiological platform for high-resolution imaging of airway tissues\nAuthors: Kurmashev, A.; Boos, J. A.; Swart, L.; Laventie, B.-J.; Junne, T.; Suetterlin, R.; Jenal, U.; Hierlemann, A.\nScore: 2.2, Published: 2023-11-22 DOI: 10.1101/2023.11.22.567838\nTranswell-based airway models have become increasingly important to study the effects of respiratory diseases and drug treatment at the air-liquid interface of the lung epithelial barrier. However, the underlying mechanisms at tissue and cell level often remain unclear, as transwell inserts feature limited live-cell imaging compatibility. Here, we report on a novel microphysiological platform for the cultivation of transwell-based lung tissues providing the possibility to alternate between air-liquid and liquid-liquid interfaces. While the air-liquid interface recapitulates physiological conditions for the lung model, the liquid-liquid interface enables live-imaging of the tissue at high spatiotemporal resolution. The plastics-based microfluidic platform enables insertion and recuperation of the transwell inserts, which allows for tissue cultivation and analysis under standardized well plate conditions. We used the device to monitor infections of Pseudomonas aeruginosa in human stem-cell-derived bronchial epithelial tissue. We continuously imaged the progression of a P. aeruginosa infection in real time at high resolution, which provided insights into bacterial spreading and invasion on the apical tissue surface, as well as insights into tissue breaching and destruction over time. The airway tissue culture system is a powerful tool to visualize and elucidate key processes of developing respiratory diseases and to facilitate drug testing and development.\nMouse Genome Editing from Scratch\nAuthors: Popova, J.; Bets, V.; Omelina, E.; Boldyreva, L.; Kozhevnikova, E.\nScore: 2.2, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567793\nMouse genome modification requires costly equipment and highly skilled personnel to manipulate zygotes. A number of zygote electroporation techniques were reported to be highly efficient in gene delivery. One of these methods called i-GONAD (improved Genome-editing via Oviductal Nucleic Acids Delivery) describes electroporation-based gene transfer to zygotes in utero. Here we adopted this technology for mouse genome-editing from scratch minimizing the cost of equipment, operator skill and animal use. We chose the CRISPR/Cas9 system as a genome editing tool and i-GONAD as a gene delivery method to produce IL10 gene knockout in C57BL/6 mice. Four animals out of 13 delivered pups (30.8%) were genetically compromised at IL10 gene locus suggesting the feasibility of the approach. This report provides one of the possible technical settings for those who aim at establishing in-house mouse transgenesis pipeline at minimal cost from scratch. For citationPopova J.V., Bets V.D., Omelina E.S., Boldyreva L.V., Kozhevnikova E.N. Mouse Genome Editing from Scratch. Journal of Experimental Biology. Year; issue (number): page. DOI\nPredicting Cortical Bone Resorption in Disuse Condition Caused by Transient Muscle Paralysis\nAuthors: Shekhar, H.; Singh, S.; Prasad, J.\nScore: 2.1, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568078\nLoad removal from the load-bearing bone, such as in the case of extended space travel and prolonged bed rest, harms bone health and leads to severe bone loss. However, the constitutive idea relating the quantity of bone loss to the absence of physiological loading is poorly understood. This work attempts to develop a mathematical model that predicts cortical bone loss at three sections: distal, mid-section, and proximal along the length of a mouse tibia. Load-induced interstitial fluid flow-based dissipation energy density has been adopted as a stimulus to trigger mechanotransduction. The developed model takes the loss of stimulus due to the disuse of bone as an input and predicts the quantity of bone loss with spatial accuracy. In the loaded bone, the fluid velocity is found to be maximum at the endocortical surface and negligible at the periosteal surface due to the permeable and impermeable boundary conditions, respectively, at the two surfaces. In case of disuse of bone, there is no fluid flow, which leads to a more significant loss of fluid flow at the endocortical surface. This loss of fluid flow coincides with the botulinum toxin (Botox)-induced endocortical bone loss data in the literature. Motivated by this fact, we hypothesized that the bone loss site would be the site of maximum stimulus loss due to disuse. To test the hypothesis, we calculated stimulus loss, i.e., loss of dissipation energy density due to bone disuse, based on the poroelastic analysis of the bone using a finite element method. A novel mathematical model has been then developed that successfully relates this loss of stimulus to the in-vivo bone loss data in the literature. According to the developed model, the site-specific mineral rate is found to be proportional to the square root of the loss of dissipation energy density. To the authors best knowledge, this model is the first of its kind to compute site-specific bone loss. The developed model can be extended to predict bone loss due to other disuse conditions such as long space travel, prolonged bed rest, etc.\n",
  "wordCount" : "2319",
  "inLanguage": "en",
  "datePublished": "2023-11-26T10:37:33Z",
  "dateModified": "2023-11-26T10:37:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 26, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568055">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568055" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568055">
        <p class="paperTitle">Engineering mice for female-biased progeny without impacting genetic integrity and litter size</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568055" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568055" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yosef, I.; Mahata, T.; Chen, Y.; Bar-Joseph, H.; Shalgi, R.; Munitz, A.; Gerlic, M.; Qimron, U.</p>
        <p class="info">Score: 36.6, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568055' target='https://doi.org/10.1101/2023.11.21.568055'> 10.1101/2023.11.21.568055</a></p>
        <p class="abstract">Generating mammalian gametes with a skewed sex ratio has thus far eluded empirical confirmation. The utilization of such genetically engineered organisms would offer the potential to curtail the necessity for culling animals of undesirable sex, mitigate resource wastage, and alleviate superfluous labor burdens. In this study, we introduce a transgenic male mouse lineage, which consistently yields predominantly female progeny (comprising up to 91% of the total offspring). This accomplishment was made possible by integrating a controllable genetic cassette onto the Y chromosome. The cassette encodes dCas9 and RNA guides that selectively silence a spermatid maturation gene. After the separation of X and Y gametes during meiosis, gametes containing an X chromosome develop normally, while those harboring the engineered Y chromosome, subjected to dCas9 silencing of the spermatid maturation gene, do not mature properly. Indeed, some spermatozoa from the transgenic mice exhibit a unique morphology, associated with the absence of the maturation gene. Notably, the resultant female offspring do not inherit the genetically engineered Y chromosome and are thus not genetically modified. Importantly, the litter size of the transgenic mice remains unchanged compared to the wild type. These findings represent a groundbreaking demonstration of genetic engineering&#39;s potential to yield sex-biased litters of full size without compromising genetic integrity, marking a pioneering advancement in this field of study.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567251">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567251" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567251">
        <p class="paperTitle">Engineering self-deliverable ribonucleoproteins for genome editing in the brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567251" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567251" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, K.; Stahl, E. C.; Kang, M. H.; Xu, B.; Allen, R.; Trinidad, M.; Doudna, J. A.</p>
        <p class="info">Score: 25.4, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567251' target='https://doi.org/10.1101/2023.11.15.567251'> 10.1101/2023.11.15.567251</a></p>
        <p class="abstract">The delivery of CRISPR ribonucleoproteins (RNPs) for genome editing in vitro and in vivo has important advantages over other delivery methods, including reduced off-target and immunogenic effects1. However, effective delivery of RNPs remains challenging in certain cell types due to low efficiency and cell toxicity. To address these issues, we engineered self-deliverable RNPs that can promote efficient cellular uptake and carry out robust genome editing without the need for helper materials or biomolecules. Screening of cell-penetrating peptides (CPPs) fused to CRISPR-Cas9 protein identified potent constructs capable of efficient genome editing of neural progenitor cells. Further engineering of these fusion proteins identified a C-terminal Cas9 fusion with three copies of A22p, a peptide derived from human semaphorin-3a, that exhibited substantially improved editing efficacy compared to other constructs. We found that self-deliverable Cas9 RNPs generated robust genome edits in clinically relevant genes when injected directly into the mouse striatum. Overall, self-deliverable Cas9 proteins provide a facile and effective platform for genome editing in vitro and in vivo.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567985">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567985" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567985">
        <p class="paperTitle">Polymer-mediated oligonucleotide delivery enables construction of barcoded 3D cultures for spatial single-cell analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567985" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567985" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: King, J. J.; Mowla, A.; Kretzmann, J. A.; Norret, M.; Kadolsky, U. D.; Iqbal, M.; Saxena, A.; Mamsa, S. S. A.; Amos, S. E.; Choi, Y. S.; Kennedy, B. F.; Iyer, K. S.; Smith, N. M.; Evans, C. W.</p>
        <p class="info">Score: 7.1, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567985' target='https://doi.org/10.1101/2023.11.20.567985'> 10.1101/2023.11.20.567985</a></p>
        <p class="abstract">Spatial transcriptomics has been widely used to capture gene expression profiles, realised as a two-dimensional (2D) projection of RNA captured from tissue sections. Wree-dimensional (3D) cultures such as spheroids and organoids are highly promising alternatives to oversimplified and homogeneous 2D cell culture models, but existing spatial transcriptomic platforms do not currently have sufficient resolution for robust analysis of 3D cultures. We present a transfection-based method for fluorescent DNA barcoding of cell populations, and the subsequent construction of spheroidal cellular architectures using barcoded cells in a layer-by-layer approach. For the first time, changes in gene expression throughout this 3D culture architecture are interrogated using multiplex single-cell RNA sequencing in which DNA barcodes are used to encode the spatial positioning of cells. We show that transfection with fluorophore-conjugated barcode oligonucleotides enables both imaging and sequencing at single-cell resolution, providing spatial maps of gene expression and drug response. Additionally, we show that fluorophore-conjugated DNA barcodes support correlative imaging studies such as mechano-microscopy to capture information about spatially-varying mechanical heterogeneity in 3D cultures. We ability to create customised, encoded cellular assemblies is a general approach that can resolve spatial differences in gene expression in 3D cell culture models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.563609">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.563609" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.563609">
        <p class="paperTitle">Unusual Structural Properties of Switchback DNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.563609" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.563609" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Madhanagopal, B. R.; Talbot, H.; Rodriguez, A.; Louis, J. M.; Zeghal, H.; Vangaveti, S.; Reddy, K.; Chandrasekaran, A. R.</p>
        <p class="info">Score: 7.0, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.563609' target='https://doi.org/10.1101/2023.11.15.563609'> 10.1101/2023.11.15.563609</a></p>
        <p class="abstract">Synthetic DNA motifs form the basis of nucleic acid nanotechnology, and their biochemical and biophysical properties determine their applications. Here, we present a detailed characterization of switchback DNA, a globally left-handed structure composed of two parallel DNA strands. Compared to a conventional duplex, switchback DNA shows lower thermodynamic stability and requires higher magnesium concentration for assembly, but exhibits a higher biostability against some nucleases. Strand competition and strand displacement experiments show that component sequences have an absolute preference for duplex complements instead of their switchback partners. Further, we hypothesize a potential role for switchback DNA as an alternate structure for short-tandem repeats involved in repeat-expansion diseases. Together with small molecule binding experiments and cell studies, our results open new avenues for synthetic DNA motifs in biology and nanotechnology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.24.568532">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.24.568532" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.24.568532">
        <p class="paperTitle">Isogenic iPSC-derived proximal and distal lung-on-chip models: Tissue- and virus-specific immune responses in human lungs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.24.568532" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.24.568532" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yadav, S.; Fujimoto, K.; Takenaga, T.; Takahashi, S.; Muramoto, Y.; Mikawa, R.; Noda, T.; Gotoh, S.; Yokokawa, R.</p>
        <p class="info">Score: 3.0, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.24.568532' target='https://doi.org/10.1101/2023.11.24.568532'> 10.1101/2023.11.24.568532</a></p>
        <p class="abstract">Micro-physiological systems (MPS) are set to play a vital role in preclinical studies, particularly in the context of future viral pandemics. Nonetheless, the development of MPS is often impeded by the scarcity of reliable cell sources, especially when seeking various organs or tissues from a single patient for comparative analysis of the host immune response. Herein, we developed human airway-on-chip and alveolus-on-chip models using induced pluripotent stem cell (iPSC)-derived isogenic lung progenitor cells. Both models demonstrated the replication of two different respiratory viruses, namely SARS-CoV-2 and Influenza, as well as related cellular damage and innate immune responses-on-chip. Our findings reveal distinct immune responses to SARS-CoV-2 in the proximal and distal lung-on-chip models. The airway chips exhibited a robust interferon (IFN)-dependent immune response, whereas the alveolus chips exhibited dysregulated IFN activation but a significantly upregulated chemokine pathway. In contrast, Influenza virus infection induced a more pronounced immune response and cellular damage in both chip models compared to SARS-CoV-2. Thus, iPSC-derived lung-on-chip models may aid in quickly gaining insights into viral pathology and screening potential drugs for future pandemics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.564845">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.564845" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.564845">
        <p class="paperTitle">High-throughput, label-free detection of DNA origami in single-cell suspensions using origamiFISH-Flow</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.564845" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.564845" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alexander, S.; Wang, W. X.; Tseng, C.-Y.; Douglas, T. R.; Chou, L. Y. T.</p>
        <p class="info">Score: 10.8, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.564845' target='https://doi.org/10.1101/2023.10.31.564845'> 10.1101/2023.10.31.564845</a></p>
        <p class="abstract">Structural DNA nanotechnology enables custom fabrication of nanoscale devices and promises diverse biological applications. However, the effects of design on DNA nanostructure (DN)-cell interactions in vitro and in vivo are not yet well-characterized. origamiFISH is a recently developed technique for imaging DNs in cells and tissues. Compared to the use of fluorescent tags, origamiFISH offers label-free and structure-agnostic detection of DNs with significantly improved sensitivity. Here, we extend the origamiFISH technique to quantifying DNs in single-cell suspensions, including nonadherent cells such as subsets of immune cells, via readout by flow cytometry. This method, referred to as origamiFISH-Flow, is high-throughput (e.g., 10,000 cells per second) and compatible with immunostaining for concurrent cell-type and -state characterization. We demonstrate that origamiFISH-Flow enhances signal-to-noise ratio by up to 20-fold compared to dye labeling approaches, leading to the capture of &gt;25-fold more DN&#43; cells at low, single-picomolar DN uptake concentrations. We additionally show the use of origamiFISH-Flow to profile cell-type and shape-specific DN uptake patterns across cell lines and splenocytes and quantify in vivo DN accumulation in lymphoid organs. Together, origamiFISH-Flow offers a new tool to interrogate DN interactions with cells and tissues, while providing insights for tailoring their designs in bio-applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.19.567703">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.19.567703" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.19.567703">
        <p class="paperTitle">Microplate Format Protein Nanopatterning for High-Throughput Screening of Cellular Microenvironments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.19.567703" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.19.567703" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shahrokhtash, A.; Sivertsen, M. v. T.; Laursen, S. H.; Sutherland, D. S.</p>
        <p class="info">Score: 2.8, Published: 2023-11-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.19.567703' target='https://doi.org/10.1101/2023.11.19.567703'> 10.1101/2023.11.19.567703</a></p>
        <p class="abstract">An advanced protein nanopatterned cell culture platform is engineered to emulate the extracellular matrixs complexity, enabling precise nanoscale biomolecule copatterning to mimic environments analogous to native tissue for cellular assays. Nanopatterns fabricated through sparse colloidal lithography, with 100 nm to 800 nm features in separate wells, are seamlessly integrated into standard microplate formats (96-well/384-well). Robust patterns are built from fully PEGylated, passivated thin glass coverslips optimized for minimal nonspecific interactions. Biotin-avidin binding and click chemistry to ensure the accurate and robust localization of bioligands. The transparent, metal-free substrates are free of topographical interference, rendering them ideal for diverse fluorescence microscopy techniques encompassing single-molecule TIRFM and extensive high-throughput imaging. The structural stability of these nanopatterns persists beyond a year in storage and long-term in cell culture conditions, endorsing their application for prolonged experimental studies and potential for widespread academic and industrial use. The platform has been demonstrated for nanopatterning an array of biomolecules, from small molecules to proteins, DNA, and extracellular matrix components, instrumental for studying cell signaling. Experiments with C2C12 cells demonstrated the exceptional specificity of the nanopatterned microplates, with nonspecific adhesion remaining below 2% and the platforms ability to elicit size-dependent cellular reactions when interfaced with nanopatterned fibronectin.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.567838">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.567838" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.567838">
        <p class="paperTitle">Transwell-based microphysiological platform for high-resolution imaging of airway tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.567838" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.567838" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kurmashev, A.; Boos, J. A.; Swart, L.; Laventie, B.-J.; Junne, T.; Suetterlin, R.; Jenal, U.; Hierlemann, A.</p>
        <p class="info">Score: 2.2, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.567838' target='https://doi.org/10.1101/2023.11.22.567838'> 10.1101/2023.11.22.567838</a></p>
        <p class="abstract">Transwell-based airway models have become increasingly important to study the effects of respiratory diseases and drug treatment at the air-liquid interface of the lung epithelial barrier. However, the underlying mechanisms at tissue and cell level often remain unclear, as transwell inserts feature limited live-cell imaging compatibility. Here, we report on a novel microphysiological platform for the cultivation of transwell-based lung tissues providing the possibility to alternate between air-liquid and liquid-liquid interfaces. While the air-liquid interface recapitulates physiological conditions for the lung model, the liquid-liquid interface enables live-imaging of the tissue at high spatiotemporal resolution. The plastics-based microfluidic platform enables insertion and recuperation of the transwell inserts, which allows for tissue cultivation and analysis under standardized well plate conditions. We used the device to monitor infections of Pseudomonas aeruginosa in human stem-cell-derived bronchial epithelial tissue. We continuously imaged the progression of a P. aeruginosa infection in real time at high resolution, which provided insights into bacterial spreading and invasion on the apical tissue surface, as well as insights into tissue breaching and destruction over time. The airway tissue culture system is a powerful tool to visualize and elucidate key processes of developing respiratory diseases and to facilitate drug testing and development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567793">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567793" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567793">
        <p class="paperTitle">Mouse Genome Editing from Scratch</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567793" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567793" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Popova, J.; Bets, V.; Omelina, E.; Boldyreva, L.; Kozhevnikova, E.</p>
        <p class="info">Score: 2.2, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567793' target='https://doi.org/10.1101/2023.11.20.567793'> 10.1101/2023.11.20.567793</a></p>
        <p class="abstract">Mouse genome modification requires costly equipment and highly skilled personnel to manipulate zygotes. A number of zygote electroporation techniques were reported to be highly efficient in gene delivery. One of these methods called i-GONAD (improved Genome-editing via Oviductal Nucleic Acids Delivery) describes electroporation-based gene transfer to zygotes in utero. Here we adopted this technology for mouse genome-editing from scratch minimizing the cost of equipment, operator skill and animal use. We chose the CRISPR/Cas9 system as a genome editing tool and i-GONAD as a gene delivery method to produce IL10 gene knockout in C57BL/6 mice. Four animals out of 13 delivered pups (30.8%) were genetically compromised at IL10 gene locus suggesting the feasibility of the approach. This report provides one of the possible technical settings for those who aim at establishing in-house mouse transgenesis pipeline at minimal cost from scratch.

For citationPopova J.V., Bets V.D., Omelina E.S., Boldyreva L.V., Kozhevnikova E.N. Mouse Genome Editing from Scratch. Journal of Experimental Biology. Year; issue (number): page. DOI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568078">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568078" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568078">
        <p class="paperTitle">Predicting Cortical Bone Resorption in Disuse Condition Caused by Transient Muscle Paralysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568078" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568078" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shekhar, H.; Singh, S.; Prasad, J.</p>
        <p class="info">Score: 2.1, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568078' target='https://doi.org/10.1101/2023.11.21.568078'> 10.1101/2023.11.21.568078</a></p>
        <p class="abstract">Load removal from the load-bearing bone, such as in the case of extended space travel and prolonged bed rest, harms bone health and leads to severe bone loss. However, the constitutive idea relating the quantity of bone loss to the absence of physiological loading is poorly understood. This work attempts to develop a mathematical model that predicts cortical bone loss at three sections:  distal,  mid-section, and  proximal along the length of a mouse tibia. Load-induced interstitial fluid flow-based dissipation energy density has been adopted as a stimulus to trigger mechanotransduction. The developed model takes the loss of stimulus due to the disuse of bone as an input and predicts the quantity of bone loss with spatial accuracy.

In the loaded bone, the fluid velocity is found to be maximum at the endocortical surface and negligible at the periosteal surface due to the permeable and impermeable boundary conditions, respectively, at the two surfaces. In case of disuse of bone, there is no fluid flow, which leads to a more significant loss of fluid flow at the endocortical surface. This loss of fluid flow coincides with the botulinum toxin (Botox)-induced endocortical bone loss data in the literature. Motivated by this fact, we hypothesized that the bone loss site would be the site of maximum stimulus loss due to disuse.

To test the hypothesis, we calculated stimulus loss, i.e., loss of dissipation energy density due to bone disuse, based on the poroelastic analysis of the bone using a finite element method. A novel mathematical model has been then developed that successfully relates this loss of stimulus to the in-vivo bone loss data in the literature. According to the developed model, the site-specific mineral rate is found to be proportional to the square root of the loss of dissipation energy density. To the authors best knowledge, this model is the first of its kind to compute site-specific bone loss. The developed model can be extended to predict bone loss due to other disuse conditions such as long space travel, prolonged bed rest, etc.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
