<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Surface Patterned Omniphobic Tiles (SPOTs): a versatile platform for scalable liquid handling
Authors: Shiri, S.; Qazi, M. J.; Tan, S.; Albo, J.; Chen, A.; Fukuda, R.; Jain, M. S.; Menesses, M.; Nchinda, N.; Ahmed, G.; Gallegos, A.; Gangishetty, M.; Congreve, D.; Cira, N. J.
Score: 23.4, Published: 2024-01-22 DOI: 10.1101/2024.01.17.575712
Manipulating liquids is a ubiquitous need for experiments across numerous scientific disciplines. To overcome limitations of current methods, we introduce Surface Patterned Omniphobic Tiles (SPOTs).">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Surface Patterned Omniphobic Tiles (SPOTs): a versatile platform for scalable liquid handling
Authors: Shiri, S.; Qazi, M. J.; Tan, S.; Albo, J.; Chen, A.; Fukuda, R.; Jain, M. S.; Menesses, M.; Nchinda, N.; Ahmed, G.; Gallegos, A.; Gangishetty, M.; Congreve, D.; Cira, N. J.
Score: 23.4, Published: 2024-01-22 DOI: 10.1101/2024.01.17.575712
Manipulating liquids is a ubiquitous need for experiments across numerous scientific disciplines. To overcome limitations of current methods, we introduce Surface Patterned Omniphobic Tiles (SPOTs)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-24T10:40:04+00:00" />
<meta property="article:modified_time" content="2024-01-24T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Surface Patterned Omniphobic Tiles (SPOTs): a versatile platform for scalable liquid handling
Authors: Shiri, S.; Qazi, M. J.; Tan, S.; Albo, J.; Chen, A.; Fukuda, R.; Jain, M. S.; Menesses, M.; Nchinda, N.; Ahmed, G.; Gallegos, A.; Gangishetty, M.; Congreve, D.; Cira, N. J.
Score: 23.4, Published: 2024-01-22 DOI: 10.1101/2024.01.17.575712
Manipulating liquids is a ubiquitous need for experiments across numerous scientific disciplines. To overcome limitations of current methods, we introduce Surface Patterned Omniphobic Tiles (SPOTs)."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Surface Patterned Omniphobic Tiles (SPOTs): a versatile platform for scalable liquid handling\nAuthors: Shiri, S.; Qazi, M. J.; Tan, S.; Albo, J.; Chen, A.; Fukuda, R.; Jain, M. S.; Menesses, M.; Nchinda, N.; Ahmed, G.; Gallegos, A.; Gangishetty, M.; Congreve, D.; Cira, N. J.\nScore: 23.4, Published: 2024-01-22 DOI: 10.1101/2024.01.17.575712\nManipulating liquids is a ubiquitous need for experiments across numerous scientific disciplines. To overcome limitations of current methods, we introduce Surface Patterned Omniphobic Tiles (SPOTs).",
  "keywords": [
    
  ],
  "articleBody": " Surface Patterned Omniphobic Tiles (SPOTs): a versatile platform for scalable liquid handling\nAuthors: Shiri, S.; Qazi, M. J.; Tan, S.; Albo, J.; Chen, A.; Fukuda, R.; Jain, M. S.; Menesses, M.; Nchinda, N.; Ahmed, G.; Gallegos, A.; Gangishetty, M.; Congreve, D.; Cira, N. J.\nScore: 23.4, Published: 2024-01-22 DOI: 10.1101/2024.01.17.575712\nManipulating liquids is a ubiquitous need for experiments across numerous scientific disciplines. To overcome limitations of current methods, we introduce Surface Patterned Omniphobic Tiles (SPOTs). This platform combines geometry and surface engineering, building on discontinuous wetting approaches to leverage capillarity for metering liquids. The SPOTs platform allows manipulation of hundreds to thousands of independent experiments without expensive equipment or large consumable costs. These devices can handle a wide range of liquid types and volumes (\u003c10 nanoliters to \u003e10 microliters) with better precision than pipetting. The platform is inexpensive and easy to fabricate, fast and intuitive to use, and cross-compatible with existing microwell plate layouts. We demonstrate how these capabilities facilitate diverse experiments including testing antibiotic combinations for synergy and antagonism, material screening of perovskites, and genotyping microbial isolates. We anticipate SPOTs will enable users from disparate domains to quickly and easily run a wide range of high-throughput experiments.\nRapid and reversible dissolution of biomolecular condensates using light-controlled recruitment of a solubility tag\nAuthors: Brumbaugh-Reed, E. H.; Aoki, K.; Toettcher, J. E.\nScore: 17.6, Published: 2024-01-17 DOI: 10.1101/2024.01.16.575860\nBiomolecular condensates are broadly implicated in both normal cellular regulation and disease. Consequently, several chemical biology and optogenetic approaches have been developed to induce phase separation of a protein of interest. However, few tools are available to perform the converse function--dissolving a condensate of interest on demand. Such a tool would aid in testing whether the condensate plays specific functional roles, a major question in cell biology and drug development. Here we report an optogenetic approach to selectively dissolve a condensate of interest in a reversible and spatially controlled manner. We show that light-gated recruitment of maltose-binding protein (MBP), a commonly used solubilizing domain in protein purification, results in rapid and controlled dissolution of condensates formed from proteins of interest. Our optogenetic MBP-based dissolution strategy (OptoMBP) is rapid, reversible, and can be spatially controlled with subcellular precision. We also provide a proof-of-principle application of OptoMBP, showing that disrupting condensation of the oncogenic fusion protein FUS-CHOP results in reversion of FUS-CHOP driven transcriptional changes. We envision that the OptoMBP system could be broadly useful for disrupting constitutive protein condensates to probe their biological functions.\nDNA microbeads for spatio-temporally controlled morphogen release within organoids\nAuthors: Afting, C.; Walther, T.; Wittbrodt, J.; GoÌˆpfrich, K.\nScore: 17.0, Published: 2024-01-12 DOI: 10.1101/2024.01.10.575045\nOrganoids have proven to be powerful in vitro model systems that mimic features of the corresponding tissue in vivo. However, across tissue types and species, organoids still often fail to reach full maturity and function, because biochemical cues cannot be provided from within the organoid to guide their development. The establishment of such tools has been identified as a major goal of the field. Here, we introduce DNA microbeads as a novel tool for implementing spatio-temporally controlled morphogen gradients inside of organoids at any point in their life cycle. The DNA microbeads are formed in a simple one-pot process, they can be stored for a year and their viscoelastic behavior and surface modification is tunable to mimic the corresponding tissue. Employing medaka retinal organoids and early embryos, we show that DNA microbeads can be integrated into embryos and organoids by microinjection and erased in a non-invasive manner with light. Coupling a recombinant surrogate Wnt to the DNA microbeads we demonstrate the spatio-temporally controlled release of the morphogen from the microinjection site, which leads to the formation of retinal pigmented epithelium while maintaining neuroretinal ganglion cells. We were thus able to bioengineer retinal organoids to more closely mirror the cell type diversity of in vivo retinas. The DNA microbead technology can easily be adapted to other organoid applications for improved tissue mimicry.\nGenerative AI for Cardiac Organoid Florescence Generation\nAuthors: Kandula, A. K. R.; Phamornratanakun, T.; Gomez, A. H.; Bhoi, R.; El-Mokahal, M.; Feng, Y.; Yang, H.\nScore: 4.5, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575724\nhPSC-derived cardiac organoid is the most recent three-dimensional tissue structure that mimics the structure and functionality of the human heart and plays a pivotal role in modeling heart development and disease. The hPSC-derived cardiac organoids are commonly characterized by bright-field microscopic imaging. Although the bright-field microscope provides essential information about hPSC-derived cardiac organoids, such as morphology, size, and general structure, it does not extend our understanding of cardiac organoids on cell type-specific distribution and structure. Then, fluorescence microscopic imaging is required to identify the specific cardiovascular cell types in the hPSC-derived cardiac organoids by fluorescence immunostaining of fixed organoid samples or fluorescence reporter imaging of live organoids. Both approaches require extra steps of experiments and do not provide general information on all hPSC-derived cardiac organoids from different hPSC lines, which limits the biomedical applications of hPSC-derived cardiac organoids. This research addresses this limitation by proposing a comprehensive workflow for colorizing grayscale cardiac organoid images from bright-field microscopic imaging using conditional Generative Adversarial Networks (GANs) to provide cell type-specific information. By infusing these grayscale images with accurate fluorescence colorization, our approach aims to unlock the hidden wealth of cell type, distribution, and network, for bettering characterizing hPSC-derived cardiac organoids.\nSaponin Nanoparticle Adjuvants Incorporating Toll-Like Receptor Agonists Improve Vaccine Immunomodulation\nAuthors: Ou, B. S.; Baillet, J.; Filsinger Interrante, M. V.; Adamska, J. Z.; Zhou, X.; Saouaf, O.; Yan, J.; Klich, J.; Jons, C. K.; Meany, E.; Valdez, A. S.; Carter, L.; Pulendran, B.; King, N. P.; Appel, E.\nScore: 149.9, Published: 2023-12-31 DOI: 10.1101/2023.07.16.549249\nOver the past few decades, the development of potent and safe immune-activating adjuvant technologies has become the heart of intensive research in the constant fight against highly mutative and immune evasive viruses such as influenza, SARS-CoV-2, and HIV. Herein, we developed a highly modular saponin-based nanoparticle platform incorporating toll-like receptor agonists (TLRas) including TLR1/2a, TLR4a, TLR7/8a adjuvants and their mixtures. These various TLRa-SNP adjuvant constructs induce unique acute cytokine and immune-signaling profiles, leading to specific Th-responses that could be of interest depending on the target disease for prevention. In a murine vaccine study, the adjuvants greatly improved the potency, durability, breadth, and neutralization of both COVID-19 and HIV vaccine candidates, suggesting the potential broad application of these adjuvant constructs to a range of different antigens. Overall, this work demonstrates a modular TLRa-SNP adjuvant platform which could improve the design of vaccines for and dramatically impact modern vaccine development.\nEpigenetic control of multiple genes with a single lentiviral vector encoding transcriptional repressors fused to compact zinc finger arrays\nAuthors: Monteferrario, D.; David, M.; Tadi, S. K.; Zhou, Y.; Marchetti, I.; Jeanneau, C.; Saviane, G.; Dupont, C. F.; Martelli, A. E.; Truong, L.; Eshleman, J.; Ng, C.; Huston, M.; Davis, G. D.; Fontenot, J. D.; Reik, A.; de la Rosa, M.; Fenard, D.\nScore: 4.0, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576049\nGene silencing without gene editing holds great potential for the development of safe therapeutic applications. Here, we describe a novel strategy to concomitantly repress multiple genes using zinc finger proteins fused to Kruppel-Associated Box repression domains (ZF-Rs). This was achieved via the optimization of a lentiviral system tailored for the delivery of ZF-Rs in hematopoietic cells. We showed that an optimal design of the lentiviral backbone is crucial to multiplex up to three ZF-Rs or two ZF-Rs and a chimeric antigen receptor. ZF-R expression had no impact on the integrity and functionality of transduced cells. Furthermore, gene repression in ZF-R-expressing T cells was highly efficient in vitro and in vivo during the entire monitoring period (up to ten weeks), and it was accompanied by epigenetic remodeling events. Finally, we described an approach to improve ZF-R specificity to illustrate the path towards the generation of ZF-Rs with a safe clinical profile. In conclusion, we successfully developed an epigenetic-based cell engineering approach for concomitant modulation of multiple gene expressions that bypass the risks associated with DNA editing.\nA General-Purpose Non-invasive Neurotechnology Research Platform\nAuthors: Schalk, G.; Shao, S.; Xiao, K.; Li, J.; Xie, J.; Guan, Y.; Wu, Z.; Chen, L.; Zhong, X.; Xu, C.; Li, G.; Yu, H.\nScore: 4.3, Published: 2024-01-17 DOI: 10.1101/2024.01.01.573494\nThis article describes our initial work toward a general-purpose platform for non-invasive neurotechnology research. This platform consists of a multi-modal wireless recording device, an associated software API, and full integration into BCI2000 software. The device is placed on the forehead and features two electroencephalographic (EEG) sensors, an inertial movement sensor (IMU), a photoplethysmogram (PPG) sensor, a microphone, and vibration-based feedback. Herein, we demonstrate different technical characteristics of our platform and its use in the context of sleep monitoring/modulation, simultaneous and synchronized recordings from different hardware, and evoked potentials. With further development and widespread dissemination, our platform could become an important tool for research into new non-invasive neurotechnology protocols in humans.\nSimple visualization of submicroscopic protein clusters with a phase-separation-based fluorescent reporter.\nAuthors: Mumford, T. R.; Rae, D.; Brackhahn, E.; Idris, A.; Gonzalez-Martinez, D.; Pal, A. A.; Chung, M. C.; Guan, J.; Rhoades, E.; Bugaj, L. J.\nScore: 50.8, Published: 2024-01-13 DOI: 10.1101/2022.07.13.499962\nProtein clustering plays numerous roles in cell physiology and disease. However, protein oligomers can be difficult to detect because they are often too small to appear as puncta in conventional fluorescence microscopy. Here we describe a fluorescent reporter strategy that detects protein clusters with high sensitivity, called CluMPS (Clusters Magnified by Phase Separation). A CluMPS reporter detects and visually amplifies even small clusters of a binding partner, generating large, quantifiable fluorescence condensates. We use computational modeling and optogenetic clustering to demonstrate that CluMPS can detect small oligomers and behaves rationally according to key system parameters. CluMPS detected small aggregates of pathological proteins where the corresponding GFP fusions appeared diffuse. CluMPS also detected and tracked clusters of unmodified and tagged endogenous proteins, and orthogonal CluMPS probes could be multiplexed in cells. CluMPS provides a powerful yet straightforward approach to observe higher-order protein assembly in its native cellular context.\nBlack Soldier Fly Bioconversion to Cultivated Meat Media Components Using Blue Catfish Gut Microbiome\nAuthors: Timoneda, A.; Amirvaresi, A.; Ovissipour, R.\nScore: 2.0, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576372\nDeveloping low-cost media is one of the major challenges in the cellular agriculture domain. Thus, this study aimed to develop low-cost media for cell-cultivated seafood using gut-microbial community-assisted fermentation. Black soldier fly larvae (Hermetia illucens) were used as the substrate and exposed to gut microbial communities isolated from Blue catfish (Ictalurus furcatus). In the first step, BSFL slurry was subjected to enzymatic digestion, using pepsin and trypsin to mimic animal digestive processes. The results showed a 2.8% degree of hydrolysis after digestion with pepsin and an additional 5.9% after digestion with trypsin. In the second step, two fermentation approaches were tested, including the direct addition of gut homogenate to the hydrolysates (fermentation A) and the establishment of microbial cultures from the gut homogenate before fermentation (fermentation B). Both fermentations resulted in similar protein content and degree of hydrolysis. Fermentation led to a decrease in species richness, with the loss of important chitinase and protease-producing genera such as Pseudomonas and Clostridiaceae. However, there was an increase in Paraclostridium and members of the Enterobacteriaceae. In addition, the effect of fermented hydrolysates from BSFL on the proliferation of zebrafish embryo fibroblasts was tested in comparison to fetal bovine serum (FBS) in in vitro cell cultivation. Lower concentrations of FBS resulted in decreased cell density and altered cell morphology. The supplementation of hydrolysate B at high peptide concentrations had cytotoxic effects on the cells, while at lower peptide concentrations, it improved cell proliferation only in cultures with 2.5% FBS.\nExploring Mechanisms of Lipid Nanoparticle-Mucus Interactions in Healthy and Cystic Fibrosis Conditions\nAuthors: Tafech, B.; Rokhforouz, M.-R.; Leung, J.; Sung, M.; Lin, P.; Sin, D.; Lauster, D.; Block, S.; Quon, B.; Tam, Y.; Cullis, P.; Feng, J. J.; Hedtrich, S.\nScore: 2.0, Published: 2024-01-22 DOI: 10.1101/2024.01.18.575680\nMucus forms the first defense line of human lungs, and as such hampers the efficient delivery of therapeutics to the underlying epithelium. This holds particularly true for genetic cargo such as CRISPR-based gene editing tools which cannot readily surmount the mucosal barrier. While lipid nanoparticles (LNPs) emerged as versatile non-viral gene delivery systems that could help overcome the delivery challenge, many knowledge gaps remain, especially for diseased states such as cystic fibrosis (CF). This study provides fundamental insights into Cas9 mRNA or ribonucleoprotein-loaded LNP-mucus interactions in healthy and diseased states by assessing the impact of the genetic cargo, mucin sialylation, mucin concentration, ionic strength, pH, and polyethylene glycol (PEG) concentration and nature on LNP diffusivity leveraging experimental approaches and Brownian dynamics simulations. Taken together, this study identifies key mucus and LNP characteristics that are critical to enabling a rational LNP design for transmucosal delivery.\n",
  "wordCount" : "2128",
  "inLanguage": "en",
  "datePublished": "2024-01-24T10:40:04Z",
  "dateModified": "2024-01-24T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 24, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.575712">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.575712" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.575712">
        <p class="paperTitle">Surface Patterned Omniphobic Tiles (SPOTs): a versatile platform for scalable liquid handling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.575712" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.575712" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shiri, S.; Qazi, M. J.; Tan, S.; Albo, J.; Chen, A.; Fukuda, R.; Jain, M. S.; Menesses, M.; Nchinda, N.; Ahmed, G.; Gallegos, A.; Gangishetty, M.; Congreve, D.; Cira, N. J.</p>
        <p class="info">Score: 23.4, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.575712' target='https://doi.org/10.1101/2024.01.17.575712'> 10.1101/2024.01.17.575712</a></p>
        <p class="abstract">Manipulating liquids is a ubiquitous need for experiments across numerous scientific disciplines. To overcome limitations of current methods, we introduce Surface Patterned Omniphobic Tiles (SPOTs). This platform combines geometry and surface engineering, building on discontinuous wetting approaches to leverage capillarity for metering liquids. The SPOTs platform allows manipulation of hundreds to thousands of independent experiments without expensive equipment or large consumable costs. These devices can handle a wide range of liquid types and volumes (&lt;10 nanoliters to &gt;10 microliters) with better precision than pipetting. The platform is inexpensive and easy to fabricate, fast and intuitive to use, and cross-compatible with existing microwell plate layouts. We demonstrate how these capabilities facilitate diverse experiments including testing antibiotic combinations for synergy and antagonism, material screening of perovskites, and genotyping microbial isolates. We anticipate SPOTs will enable users from disparate domains to quickly and easily run a wide range of high-throughput experiments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575860">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575860" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575860">
        <p class="paperTitle">Rapid and reversible dissolution of biomolecular condensates using light-controlled recruitment of a solubility tag</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575860" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575860" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brumbaugh-Reed, E. H.; Aoki, K.; Toettcher, J. E.</p>
        <p class="info">Score: 17.6, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575860' target='https://doi.org/10.1101/2024.01.16.575860'> 10.1101/2024.01.16.575860</a></p>
        <p class="abstract">Biomolecular condensates are broadly implicated in both normal cellular regulation and disease. Consequently, several chemical biology and optogenetic approaches have been developed to induce phase separation of a protein of interest. However, few tools are available to perform the converse function--dissolving a condensate of interest on demand. Such a tool would aid in testing whether the condensate plays specific functional roles, a major question in cell biology and drug development. Here we report an optogenetic approach to selectively dissolve a condensate of interest in a reversible and spatially controlled manner. We show that light-gated recruitment of maltose-binding protein (MBP), a commonly used solubilizing domain in protein purification, results in rapid and controlled dissolution of condensates formed from proteins of interest. Our optogenetic MBP-based dissolution strategy (OptoMBP) is rapid, reversible, and can be spatially controlled with subcellular precision. We also provide a proof-of-principle application of OptoMBP, showing that disrupting condensation of the oncogenic fusion protein FUS-CHOP results in reversion of FUS-CHOP driven transcriptional changes. We envision that the OptoMBP system could be broadly useful for disrupting constitutive protein condensates to probe their biological functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575045">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575045" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575045">
        <p class="paperTitle">DNA microbeads for spatio-temporally controlled morphogen release within organoids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575045" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575045" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Afting, C.; Walther, T.; Wittbrodt, J.; GoÌˆpfrich, K.</p>
        <p class="info">Score: 17.0, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575045' target='https://doi.org/10.1101/2024.01.10.575045'> 10.1101/2024.01.10.575045</a></p>
        <p class="abstract">Organoids have proven to be powerful in vitro model systems that mimic features of the corresponding tissue in vivo. However, across tissue types and species, organoids still often fail to reach full maturity and function, because biochemical cues cannot be provided from within the organoid to guide their development. The establishment of such tools has been identified as a major goal of the field. Here, we introduce DNA microbeads as a novel tool for implementing spatio-temporally controlled morphogen gradients inside of organoids at any point in their life cycle. The DNA microbeads are formed in a simple one-pot process, they can be stored for a year and their viscoelastic behavior and surface modification is tunable to mimic the corresponding tissue. Employing medaka retinal organoids and early embryos, we show that DNA microbeads can be integrated into embryos and organoids by microinjection and erased in a non-invasive manner with light. Coupling a recombinant surrogate Wnt to the DNA microbeads we demonstrate the spatio-temporally controlled release of the morphogen from the microinjection site, which leads to the formation of retinal pigmented epithelium while maintaining neuroretinal ganglion cells. We were thus able to bioengineer retinal organoids to more closely mirror the cell type diversity of in vivo retinas. The DNA microbead technology can easily be adapted to other organoid applications for improved tissue mimicry.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575724">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575724" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575724">
        <p class="paperTitle">Generative AI for Cardiac Organoid Florescence Generation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575724" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575724" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kandula, A. K. R.; Phamornratanakun, T.; Gomez, A. H.; Bhoi, R.; El-Mokahal, M.; Feng, Y.; Yang, H.</p>
        <p class="info">Score: 4.5, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575724' target='https://doi.org/10.1101/2024.01.15.575724'> 10.1101/2024.01.15.575724</a></p>
        <p class="abstract">hPSC-derived cardiac organoid is the most recent three-dimensional tissue structure that mimics the structure and functionality of the human heart and plays a pivotal role in modeling heart development and disease. The hPSC-derived cardiac organoids are commonly characterized by bright-field microscopic imaging. Although the bright-field microscope provides essential information about hPSC-derived cardiac organoids, such as morphology, size, and general structure, it does not extend our understanding of cardiac organoids on cell type-specific distribution and structure. Then, fluorescence microscopic imaging is required to identify the specific cardiovascular cell types in the hPSC-derived cardiac organoids by fluorescence immunostaining of fixed organoid samples or fluorescence reporter imaging of live organoids. Both approaches require extra steps of experiments and do not provide general information on all hPSC-derived cardiac organoids from different hPSC lines, which limits the biomedical applications of hPSC-derived cardiac organoids. This research addresses this limitation by proposing a comprehensive workflow for colorizing grayscale cardiac organoid images from bright-field microscopic imaging using conditional Generative Adversarial Networks (GANs) to provide cell type-specific information. By infusing these grayscale images with accurate fluorescence colorization, our approach aims to unlock the hidden wealth of cell type, distribution, and network, for bettering characterizing hPSC-derived cardiac organoids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.549249">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.549249" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.549249">
        <p class="paperTitle">Saponin Nanoparticle Adjuvants Incorporating Toll-Like Receptor Agonists Improve Vaccine Immunomodulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.549249" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.549249" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ou, B. S.; Baillet, J.; Filsinger Interrante, M. V.; Adamska, J. Z.; Zhou, X.; Saouaf, O.; Yan, J.; Klich, J.; Jons, C. K.; Meany, E.; Valdez, A. S.; Carter, L.; Pulendran, B.; King, N. P.; Appel, E.</p>
        <p class="info">Score: 149.9, Published: 2023-12-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.549249' target='https://doi.org/10.1101/2023.07.16.549249'> 10.1101/2023.07.16.549249</a></p>
        <p class="abstract">Over the past few decades, the development of potent and safe immune-activating adjuvant technologies has become the heart of intensive research in the constant fight against highly mutative and immune evasive viruses such as influenza, SARS-CoV-2, and HIV. Herein, we developed a highly modular saponin-based nanoparticle platform incorporating toll-like receptor agonists (TLRas) including TLR1/2a, TLR4a, TLR7/8a adjuvants and their mixtures. These various TLRa-SNP adjuvant constructs induce unique acute cytokine and immune-signaling profiles, leading to specific Th-responses that could be of interest depending on the target disease for prevention. In a murine vaccine study, the adjuvants greatly improved the potency, durability, breadth, and neutralization of both COVID-19 and HIV vaccine candidates, suggesting the potential broad application of these adjuvant constructs to a range of different antigens. Overall, this work demonstrates a modular TLRa-SNP adjuvant platform which could improve the design of vaccines for and dramatically impact modern vaccine development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576049">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576049" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576049">
        <p class="paperTitle">Epigenetic control of multiple genes with a single lentiviral vector encoding transcriptional repressors fused to compact zinc finger arrays</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576049" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576049" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Monteferrario, D.; David, M.; Tadi, S. K.; Zhou, Y.; Marchetti, I.; Jeanneau, C.; Saviane, G.; Dupont, C. F.; Martelli, A. E.; Truong, L.; Eshleman, J.; Ng, C.; Huston, M.; Davis, G. D.; Fontenot, J. D.; Reik, A.; de la Rosa, M.; Fenard, D.</p>
        <p class="info">Score: 4.0, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576049' target='https://doi.org/10.1101/2024.01.17.576049'> 10.1101/2024.01.17.576049</a></p>
        <p class="abstract">Gene silencing without gene editing holds great potential for the development of safe therapeutic applications. Here, we describe a novel strategy to concomitantly repress multiple genes using zinc finger proteins fused to Kruppel-Associated Box repression domains (ZF-Rs). This was achieved via the optimization of a lentiviral system tailored for the delivery of ZF-Rs in hematopoietic cells. We showed that an optimal design of the lentiviral backbone is crucial to multiplex up to three ZF-Rs or two ZF-Rs and a chimeric antigen receptor. ZF-R expression had no impact on the integrity and functionality of transduced cells. Furthermore, gene repression in ZF-R-expressing T cells was highly efficient in vitro and in vivo during the entire monitoring period (up to ten weeks), and it was accompanied by epigenetic remodeling events. Finally, we described an approach to improve ZF-R specificity to illustrate the path towards the generation of ZF-Rs with a safe clinical profile. In conclusion, we successfully developed an epigenetic-based cell engineering approach for concomitant modulation of multiple gene expressions that bypass the risks associated with DNA editing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.01.573494">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.01.573494" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.01.573494">
        <p class="paperTitle">A General-Purpose Non-invasive Neurotechnology Research Platform</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.01.573494" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.01.573494" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schalk, G.; Shao, S.; Xiao, K.; Li, J.; Xie, J.; Guan, Y.; Wu, Z.; Chen, L.; Zhong, X.; Xu, C.; Li, G.; Yu, H.</p>
        <p class="info">Score: 4.3, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.01.573494' target='https://doi.org/10.1101/2024.01.01.573494'> 10.1101/2024.01.01.573494</a></p>
        <p class="abstract">This article describes our initial work toward a general-purpose platform for non-invasive neurotechnology research. This platform consists of a multi-modal wireless recording device, an associated software API, and full integration into BCI2000 software. The device is placed on the forehead and features two electroencephalographic (EEG) sensors, an inertial movement sensor (IMU), a photoplethysmogram (PPG) sensor, a microphone, and vibration-based feedback. Herein, we demonstrate different technical characteristics of our platform and its use in the context of sleep monitoring/modulation, simultaneous and synchronized recordings from different hardware, and evoked potentials. With further development and widespread dissemination, our platform could become an important tool for research into new non-invasive neurotechnology protocols in humans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.07.13.499962">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.07.13.499962" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.07.13.499962">
        <p class="paperTitle">Simple visualization of submicroscopic protein clusters with a phase-separation-based fluorescent reporter.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.07.13.499962" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.07.13.499962" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mumford, T. R.; Rae, D.; Brackhahn, E.; Idris, A.; Gonzalez-Martinez, D.; Pal, A. A.; Chung, M. C.; Guan, J.; Rhoades, E.; Bugaj, L. J.</p>
        <p class="info">Score: 50.8, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.07.13.499962' target='https://doi.org/10.1101/2022.07.13.499962'> 10.1101/2022.07.13.499962</a></p>
        <p class="abstract">Protein clustering plays numerous roles in cell physiology and disease. However, protein oligomers can be difficult to detect because they are often too small to appear as puncta in conventional fluorescence microscopy. Here we describe a fluorescent reporter strategy that detects protein clusters with high sensitivity, called CluMPS (Clusters Magnified by Phase Separation). A CluMPS reporter detects and visually amplifies even small clusters of a binding partner, generating large, quantifiable fluorescence condensates. We use computational modeling and optogenetic clustering to demonstrate that CluMPS can detect small oligomers and behaves rationally according to key system parameters. CluMPS detected small aggregates of pathological proteins where the corresponding GFP fusions appeared diffuse. CluMPS also detected and tracked clusters of unmodified and tagged endogenous proteins, and orthogonal CluMPS probes could be multiplexed in cells. CluMPS provides a powerful yet straightforward approach to observe higher-order protein assembly in its native cellular context.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576372">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576372" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576372">
        <p class="paperTitle">Black Soldier Fly Bioconversion to Cultivated Meat Media Components Using Blue Catfish Gut Microbiome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576372" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576372" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Timoneda, A.; Amirvaresi, A.; Ovissipour, R.</p>
        <p class="info">Score: 2.0, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576372' target='https://doi.org/10.1101/2024.01.20.576372'> 10.1101/2024.01.20.576372</a></p>
        <p class="abstract">Developing low-cost media is one of the major challenges in the cellular agriculture domain. Thus, this study aimed to develop low-cost media for cell-cultivated seafood using gut-microbial community-assisted fermentation. Black soldier fly larvae (Hermetia illucens) were used as the substrate and exposed to gut microbial communities isolated from Blue catfish (Ictalurus furcatus). In the first step, BSFL slurry was subjected to enzymatic digestion, using pepsin and trypsin to mimic animal digestive processes. The results showed a 2.8% degree of hydrolysis after digestion with pepsin and an additional 5.9% after digestion with trypsin. In the second step, two fermentation approaches were tested, including the direct addition of gut homogenate to the hydrolysates (fermentation A) and the establishment of microbial cultures from the gut homogenate before fermentation (fermentation B). Both fermentations resulted in similar protein content and degree of hydrolysis. Fermentation led to a decrease in species richness, with the loss of important chitinase and protease-producing genera such as Pseudomonas and Clostridiaceae. However, there was an increase in Paraclostridium and members of the Enterobacteriaceae. In addition, the effect of fermented hydrolysates from BSFL on the proliferation of zebrafish embryo fibroblasts was tested in comparison to fetal bovine serum (FBS) in in vitro cell cultivation. Lower concentrations of FBS resulted in decreased cell density and altered cell morphology. The supplementation of hydrolysate B at high peptide concentrations had cytotoxic effects on the cells, while at lower peptide concentrations, it improved cell proliferation only in cultures with 2.5% FBS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.575680">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.575680" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.575680">
        <p class="paperTitle">Exploring Mechanisms of Lipid Nanoparticle-Mucus Interactions in Healthy and Cystic Fibrosis Conditions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.575680" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.575680" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tafech, B.; Rokhforouz, M.-R.; Leung, J.; Sung, M.; Lin, P.; Sin, D.; Lauster, D.; Block, S.; Quon, B.; Tam, Y.; Cullis, P.; Feng, J. J.; Hedtrich, S.</p>
        <p class="info">Score: 2.0, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.575680' target='https://doi.org/10.1101/2024.01.18.575680'> 10.1101/2024.01.18.575680</a></p>
        <p class="abstract">Mucus forms the first defense line of human lungs, and as such hampers the efficient delivery of therapeutics to the underlying epithelium. This holds particularly true for genetic cargo such as CRISPR-based gene editing tools which cannot readily surmount the mucosal barrier. While lipid nanoparticles (LNPs) emerged as versatile non-viral gene delivery systems that could help overcome the delivery challenge, many knowledge gaps remain, especially for diseased states such as cystic fibrosis (CF). This study provides fundamental insights into Cas9 mRNA or ribonucleoprotein-loaded LNP-mucus interactions in healthy and diseased states by assessing the impact of the genetic cargo, mucin sialylation, mucin concentration, ionic strength, pH, and polyethylene glycol (PEG) concentration and nature on LNP diffusivity leveraging experimental approaches and Brownian dynamics simulations. Taken together, this study identifies key mucus and LNP characteristics that are critical to enabling a rational LNP design for transmucosal delivery.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
