<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Flow in fetoplacental microvessels in vitro enhances perfusion, barrier function, and matrix stability
Authors: Cherubini, M.; Erickson, S.; Padmanaban, P.; Haberkant, P.; Stein, F.; Sastre, V. B.; Haase, K.
Score: 9.1, Published: 2023-07-21 DOI: 10.1101/2023.07.19.549736
Proper placental vascularization is vital for pregnancy outcomes, but assessing it with animal models and human explants has limitations. Here, we present a 3D in vitro model of human placenta terminal villi that includes fetal mesenchyme and vascular endothelium.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Flow in fetoplacental microvessels in vitro enhances perfusion, barrier function, and matrix stability
Authors: Cherubini, M.; Erickson, S.; Padmanaban, P.; Haberkant, P.; Stein, F.; Sastre, V. B.; Haase, K.
Score: 9.1, Published: 2023-07-21 DOI: 10.1101/2023.07.19.549736
Proper placental vascularization is vital for pregnancy outcomes, but assessing it with animal models and human explants has limitations. Here, we present a 3D in vitro model of human placenta terminal villi that includes fetal mesenchyme and vascular endothelium." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-30T10:36:57+00:00" />
<meta property="article:modified_time" content="2023-07-30T10:36:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Flow in fetoplacental microvessels in vitro enhances perfusion, barrier function, and matrix stability
Authors: Cherubini, M.; Erickson, S.; Padmanaban, P.; Haberkant, P.; Stein, F.; Sastre, V. B.; Haase, K.
Score: 9.1, Published: 2023-07-21 DOI: 10.1101/2023.07.19.549736
Proper placental vascularization is vital for pregnancy outcomes, but assessing it with animal models and human explants has limitations. Here, we present a 3D in vitro model of human placenta terminal villi that includes fetal mesenchyme and vascular endothelium."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Flow in fetoplacental microvessels in vitro enhances perfusion, barrier function, and matrix stability\nAuthors: Cherubini, M.; Erickson, S.; Padmanaban, P.; Haberkant, P.; Stein, F.; Sastre, V. B.; Haase, K.\nScore: 9.1, Published: 2023-07-21 DOI: 10.1101/2023.07.19.549736\nProper placental vascularization is vital for pregnancy outcomes, but assessing it with animal models and human explants has limitations. Here, we present a 3D in vitro model of human placenta terminal villi that includes fetal mesenchyme and vascular endothelium.",
  "keywords": [
    
  ],
  "articleBody": " Flow in fetoplacental microvessels in vitro enhances perfusion, barrier function, and matrix stability\nAuthors: Cherubini, M.; Erickson, S.; Padmanaban, P.; Haberkant, P.; Stein, F.; Sastre, V. B.; Haase, K.\nScore: 9.1, Published: 2023-07-21 DOI: 10.1101/2023.07.19.549736\nProper placental vascularization is vital for pregnancy outcomes, but assessing it with animal models and human explants has limitations. Here, we present a 3D in vitro model of human placenta terminal villi that includes fetal mesenchyme and vascular endothelium. By co-culturing HUVEC, placental fibroblasts, and pericytes in a macro-fluidic chip with a flow reservoir, we generate fully perfusable fetal microvessels. Pressure-driven flow is crucial for the growth and remodeling of these microvessels, resulting in early formation of interconnected placental vascular networks and maintained viability. Computational fluid dynamics simulations predict shear forces, which increase microtissue stiffness, decrease diffusivity and enhance barrier function as shear stress rises. Mass-spec analysis reveals the deposition of numerous extracellular proteins, with flow notably enhancing the expression of matrix stability regulators, proteins associated with actin dynamics, and cytoskeleton organization. Our model provides a powerful tool for deducing complex in vivo parameters, such as shear stress on developing vascularized placental tissue, and holds promise for unraveling gestational disorders related to the vasculature.\nSimple and rewireable biomolecular building blocks for DNA machine-learning algorithms\nAuthors: Lee, R. C.; Corsano, A.; Tseng, C. Y.; Chou, L. Y. T.\nScore: 7.0, Published: 2023-07-23 DOI: 10.1101/2023.07.20.549967\nDeep learning algorithms, such as neural networks, enable the processing of complex datasets with many related variables, and have applications in disease diagnosis, cell profiling, and drug discovery. Beyond its use in electronic computers, neural networks have been implemented using programmable biomolecules such as DNA. This confers unique advantages such as greater portability, ability to operate without electricity, and direct analysis of patterns of biomolecules in solution. Analogous to past bottlenecks in electronic computers, the computing power of DNA-based neural networks is limited by the ability to add more computing units, i.e. neurons. This limitation exists because current architectures require many nucleic acids to model a single neuron. Each addition of a neuron to the network compounds existing problems such as long assembly times, high background signal, and cross-talk between components. Here we test three strategies to solve this limitation and improve the scalability of DNA-based neural networks: (i) enzymatic synthesis to generate high-purity neurons, (ii) spatial patterning of neuron clusters based on their network position, and (iii) encoding neuron connectivity on a universal single-stranded DNA backbone. We show that neurons implemented via these strategies activate quickly, with high signal-to-background ratio, and respond to varying input concentrations and weights. Using this neuron design, we implemented basic neural network motifs such as cascading, fan-in, and fan-out circuits. Since this design is modular, easy to synthesize, and compatible with multiple neural network architectures, we envision it will help scale DNA-based neural networks in a variety of settings. This will enable portable computing power for applications such as portable diagnostics, compact data storage, and autonomous decision making for lab-on-a-chips.\nOn automated model discovery and a universal material subroutine\nAuthors: Peirlinck, M.; Linka, K.; Hurtado, J. A.; Kuhl, E.\nScore: 5.9, Published: 2023-07-25 DOI: 10.1101/2023.07.19.549749\nConstitutive modeling is the cornerstone of computational and structural mechanics. In a finite element analysis, the constitutive model is encoded in the material subroutine, a function that maps local strains onto stresses. This function is called within every finite element, at each integration point, within every time step, at each Newton iteration. Todays finite element packages offer large libraries of material subroutines to choose from. However, the scientific criteria for model selection remain highly subjective and prone to user bias. Here we fully automate the process of model selection, autonomously discover the best model and parameters from experimental data, encode all possible discoverable models into a single material subroutine, and seamlessly integrate this universal material subroutine into a finite element analysis. We prototype this strategy for tension, compression, and shear data from human brain tissue and perform a hyperelastic model discovery from twelve possible terms. These terms feature the first and second invariants, raised to the first and second powers, embedded in the identity, exponential, and logarithmic functions, generating 22x2x3 = 4096 models in total. We demonstrate how to integrate these models into a single universal material subroutine that features the classical neo Hooke, Blatz Ko, Mooney Rivlin, Demiray, Gent, and Holzapfel models as special cases. Finite element simulations with our universal material subroutine show that it specializes well to these widely used models, generalizes well to newly discovered models, and agrees excellently with both experimental data and previous simulations. It also performs well within realistic finite element simulations and accurately predicts stress concentrations in the human brain for six different head impact scenarios. We anticipate that integrating automated model discovery into a universal material subroutine will generalize naturally to more complex anisotropic, compressible, and inelastic materials and to other nonlinear finite element platforms. Replacing dozens of individual material subroutines by a single universal material subroutine that is populated directly via automated model discovery--entirely without human interaction--makes finite element analyses more accessible, more robust, and less vulnerable to human error. This could forever change how we simulate materials and structures.\nEliminating Malaria Vectors with Precision Guided Sterile Males\nAuthors: Smidler, A. L.; Apte, R. A.; Pai, J. J.; Chow, M. L.; Chen, S.; Mondal, A.; Sanchez C., H. M.; Antoshechkin, I.; Marshall, J. M.; Akbari, O. S.\nScore: 13.5, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549947\nControlling the principal African malaria vector, the mosquito Anopheles gambiae, is considered essential to curtail malaria transmission. However existing vector control technologies rely on insecticides, which are becoming increasingly ineffective. Sterile insect technique (SIT) is a powerful suppression approach that has successfully eradicated a number of insect pests, yet the A. gambiae toolkit lacks the requisite technologies for its implementation. SIT relies on iterative mass-releases of non-biting, non-driving, sterile males which seek out and mate with monandrous wild females. Once mated, females are permanently sterilized due to mating-induced refractoriness, which results in population suppression of the subsequent generation. However, sterilization by traditional methods renders males unfit, making the creation of precise genetic sterilization methods imperative. Here we develop precision guided Sterile Insect Technique (pgSIT) in the mosquito A. gambiae for inducible, programmed male-sterilization and female-elimination for wide scale use in SIT campaigns. Using a binary CRISPR strategy, we cross separate engineered Cas9 and gRNA strains to disrupt male-fertility and female-essential genes, yielding \u003e99.5% male-sterility and \u003e99.9% female-lethality in hybrid progeny. We demonstrate that these genetically sterilized males have good longevity, are able to induce population suppression in cage trials, and are predicted to eliminate wild A. gambiae populations using mathematical models, making them ideal candidates for release. This work provides a valuable addition to the malaria genetic biocontrol toolkit, for the first time enabling scalable SIT-like confinable suppression in the species.\nSingle-cell precision nanotechnology in vivo\nAuthors: MOLBAY, M.; Kick, B.; Zhao, S.; Todorov, M. I.; Ohn, T.-L.; Roth, S.; Simats, A.; Singh, V.; Khalin, I.; Pan, C.; Singh Bhatia, H.; Hellal, F.; Zeidler, R.; Liesz, A.; Plesnila, N.; Dietz, H.; Erturk, A.\nScore: 4.7, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550304\nTargeting nanoparticle therapeutics with cellular accuracy in whole organisms could open breakthrough opportunities in precision medicine. However, evaluating and fine-tuning the biodistribution of such systems in the whole organism at the cellular level remains a major obstacle. Here, we constructed targetable DNA origami, and analyzed biodistribution in transparent mice, in addition to studying tolerability, clearance kinetics, and immune response parameters. Untargeted DNA origami primarily accumulated in the spleen and the liver, while an immune cell-targeting variant successfully attached to immune cells throughout the body. A cancer cell-targeting mimetic co-localized on solid-tumor metastasis in the liver and the lung. These findings indicate that DNA origami can be directed in vivo, providing an important proof-of-concept and highlights the potential of high-resolution tissue-clearing imaging technologies in their development. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=121 SRC=\"FIGDIR/small/550304v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (44K): org.highwire.dtl.DTLVardef@e6f63corg.highwire.dtl.DTLVardef@12ffe6dorg.highwire.dtl.DTLVardef@1e8e54aorg.highwire.dtl.DTLVardef@1256361_HPS_FORMAT_FIGEXP M_FIG C_FIG HighlightsO_LIThis study demonstrates the potential of DNA origami-based drug delivery systems as versatile tool for for targeted delivery, which could be used to treat a range of diseases with applications. C_LIO_LIThe immune compatibility, half-life, targeting efficiency, and the biodistribution evaluation of DNA origami indicate its potential for systemic drug delivery. C_LIO_LIOur approach enables the assessment of biodistribution of nanoparticles in the intact body with a sensitivity to the single-cell level, highlighting the high-resolution tissue clearing technoloies in revealing DNA origamis feasibility for drug targeting. C_LI\nResponse Surface Methodology-Based High-Throughput BioLector Fermentation Screening of Rosetta-2 for Enhanced Production of Insulin Variant: Towards Improved Production Efficiency\nAuthors: Ayan, E.; Aytekin, A. O.; Kati, A.; Demirci, H.\nScore: 5.5, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549622\nBackgroundIn order to obtain high yields of recombinant insulin and advancing therapeutic avenues for diabetes patients, the development of innovative designer insulin analogs have critical importance. The modified insulin analog presents a cost-effective remedy by being produced as inclusion bodies (IBs) within Escherichia coli BL21 (DE3) Rosetta-2 strain. This approach to production not only offers reduced production time but also yields high recovery rates. The prime aim of this investigation was to optimize the composition of the cultivation media, thereby accomplishing higher cell density fermentation of the proinsulin, that is fused with a cleavable hexa histidine purification tag. ResultVarious factors, including carbon and nitrogen sources, salts, metal ions, and pH, were systematically investigated through experimental screening using the BioLector multiwell bright plate. Additionally, computational analysis employing the Plackett-Burman Design within the Design Expert software was utilized to assess their effectiveness in terms of insulin concentration as a surrogate measure of insulin yield. Among the tested variables, glucose, glycerol, MgSO4, and lower Luria-Bertani mix concentration have a significant influence on insulin production, as determined by the screening experiment. Subsequently, the Central Composite Design approach was operated to further evaluate and optimize the precise levels of these influential variables. This systematic methodology achieved an optimized cultivation media formulation, resulting in a remarkable enhancement of insulin production, with levels reaching up to 13 mg/ml when applied in BioLector fermentation. ConclusionThe formulated cultivation media exhibited suitability for promoting high cell density fermentation of the modified insulin, thereby facilitating the attainment of optimal yields for the proinsulin expression.\nFibroblasts-dependent maturation and phenotype exacerbation of dystrophic hiPSC-derived MYOtissues enables muscle strength evaluation for gene therapy screening\nAuthors: Palmieri, L.; Moula, M.; Jaber, A.; El Khoury, R.; Brochiet, G.; Bigot, A.; Israeli, D.; Richard, I.; Albini, S.\nScore: 2.5, Published: 2023-07-28 DOI: 10.1101/2023.07.26.550063\nDuchenne muscular dystrophy (DMD) is a lethal muscle wasting disease caused by absence of dystrophin, a protein essential to preserve muscle integrity continuously challenged by contractions. Gene therapy utilizing adeno-associated virus (AAV) to deliver truncated forms of dystrophin (uDys) is currently the most promising therapeutic approach. However, the therapeutic outcome in treated patients has not been as successful as anticipated by animal studies, underscoring the need of improved and high-throughput models for accurate and fast prediction of human response. Here, we describe the generation of MYOtissues, a 3D muscle platform based on direct myogenic conversion of human induced pluripotent stem cells (iPSC), whose structural and functional maturation is enhanced by fibroblasts incorporation. MYOtissues derived from DMD-iPSC including DMD fibroblasts, exacerbated pathogenic hallmarks such as fibrosis and muscle force loss. As a proof of concept, we showed that AAV-mediated uDys gene transfer improved muscle resistance and membrane stability in DMD-MYOtissues, highlighting the suitability of our system for gene therapy screening.\nSimultaneous multi-site editing of individual genomes using retron arrays\nAuthors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.\nScore: 29.1, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549397\nOur understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites. Unfortunately, building cells with multiple, non-adjacent precise mutations remains a laborious cycle of editing, isolating an edited cell, and editing again. Here, we describe a technology for precisely modifying multiple sites on a single genome simultaneously. This technology - termed a multitron - is built from a heavily modified retron, in which multiple donor-encoding msds are produced from a single transcript. The multitron architecture is compatible with both recombineering in prokaryotic cells and CRISPR editing in eukaryotic cells. We demonstrate applications for this approach in molecular recording, genetic element minimization, and metabolic engineering.\nSoft robotics-enabled large animal model of HFpEF hemodynamics for device testing\nAuthors: Rosalia, L.; Ozturk, C.; Wang, S.; Quevedo, D.; Saeed, M.; Mauskapf, A.; Roche, E.\nScore: 2.1, Published: 2023-07-29 DOI: 10.1101/2023.07.26.550654\nHeart failure with preserved ejection fraction (HFpEF) is a major challenge in cardiovascular medicine, accounting for approximately 50% of all cases of heart failure. Due to the lack of effective therapies for this condition, the mortality associated with HFpEF remains higher than that of most cancers. Despite the ongoing efforts, no medical device has yet received FDA approval. This is largely due to the lack of an in vivo model of the HFpEF hemodynamics, resulting in the inability to evaluate device effectiveness in vivo prior to clinical trials. Here, we describe the development of a highly tunable porcine model of HFpEF hemodynamics using implantable soft robotic sleeves, where controlled actuation of a left ventricular and an aortic sleeve can recapitulate changes in ventricular compliance and afterload associated with a broad spectrum of HFpEF hemodynamic phenotypes. We demonstrate the feasibility of the proposed model in preclinical testing by evaluating the hemodynamic response of the model post-implantation of an interatrial shunt device, which was found to be consistent with findings from in silico studies and clinical trials. This work addresses several of the limitations associated with previous models of HFpEF, such as their limited hemodynamic fidelity, elevated costs, lengthy development time, and low throughput. By showcasing exceptional versatility and tunability, the proposed platform has the potential to revolutionize the current approach for HFpEF device development and selection, with the goal of improving the quality of life for the 32 million people affected by HFpEF worldwide.\nAutologous iPSC- and MSC-derived Chondrocyte Implants for Cartilage Repair in a Miniature Pig Model\nAuthors: Lee, M.-S.; Sivapatham, A.; Leiferman, E. M.; Jiao, H.; Lu, Y.; Nemke, B. W.; Leiferman, M.; Markel, M. D.; Li, W.-J.\nScore: 2.1, Published: 2023-07-29 DOI: 10.1101/2023.07.25.550409\nInduced pluripotent stem cell (iPSC)-derived mesenchymal stem cells (iMSCs) have greater potential for generating chondrocytes without hypertrophic and fibrotic phenotypes compared to bone marrow-derived mesenchymal stem/stromal cells (BMSCs). However, there is a lack of research demonstrating the use of autologous iMSCs for repairing articular chondral lesions in large animal models. In this study, we aimed to evaluate the effectiveness of autologous miniature pig (minipig) iMSC-chondrocyte (iMSC-Ch)-laden implants in comparison to autologous BMSC-chondrocyte (BMSC-Ch)-laden implants for cartilage repair in porcine femoral condyles. iMSCs and BMSCs were seeded into fibrin glue/nanofiber constructs and cultured with chondrogenic induction media for 7 days before implantation. To assess the regenerative capacity of the cells, 19 skeletally mature Yucatan minipigs were randomly divided into microfracture control, acellular scaffold, iMSC, and BMSC subgroups. A cylindrical defect measuring 7 mm in diameter and 0.6 mm in depth was created on the articular cartilage surface without violating the subchondral bone. The defects were then left untreated or treated with acellular or cellular implants. Both cellular implant-treated groups exhibited enhanced joint repair compared to the microfracture and acellular control groups. Immunofluorescence analysis yielded significant findings, showing that cartilage treated with iMSC-Ch implants exhibited higher expression of COL2A1 and minimal to no expression of COL1A1 and COL10A1, in contrast to the BMSC-Ch-treated group. This indicates that the iMSC-Ch implants generated more hyaline cartilage-like tissue compared to the BMSC-Ch implants. These results contribute to filling the knowledge gap regarding the potential of autologous iPSC derivatives for cartilage repair in translational animal models.\n",
  "wordCount" : "2634",
  "inLanguage": "en",
  "datePublished": "2023-07-30T10:36:57Z",
  "dateModified": "2023-07-30T10:36:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on July 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549736">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549736" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549736">
        <p class="paperTitle">Flow in fetoplacental microvessels in vitro enhances perfusion, barrier function, and matrix stability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549736" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549736" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cherubini, M.; Erickson, S.; Padmanaban, P.; Haberkant, P.; Stein, F.; Sastre, V. B.; Haase, K.</p>
        <p class="info">Score: 9.1, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549736' target='https://doi.org/10.1101/2023.07.19.549736'> 10.1101/2023.07.19.549736</a></p>
        <p class="abstract">Proper placental vascularization is vital for pregnancy outcomes, but assessing it with animal models and human explants has limitations. Here, we present a 3D in vitro model of human placenta terminal villi that includes fetal mesenchyme and vascular endothelium. By co-culturing HUVEC, placental fibroblasts, and pericytes in a macro-fluidic chip with a flow reservoir, we generate fully perfusable fetal microvessels. Pressure-driven flow is crucial for the growth and remodeling of these microvessels, resulting in early formation of interconnected placental vascular networks and maintained viability. Computational fluid dynamics simulations predict shear forces, which increase microtissue stiffness, decrease diffusivity and enhance barrier function as shear stress rises. Mass-spec analysis reveals the deposition of numerous extracellular proteins, with flow notably enhancing the expression of matrix stability regulators, proteins associated with actin dynamics, and cytoskeleton organization. Our model provides a powerful tool for deducing complex in vivo parameters, such as shear stress on developing vascularized placental tissue, and holds promise for unraveling gestational disorders related to the vasculature.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549967">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549967" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549967">
        <p class="paperTitle">Simple and rewireable biomolecular building blocks for DNA machine-learning algorithms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549967" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549967" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, R. C.; Corsano, A.; Tseng, C. Y.; Chou, L. Y. T.</p>
        <p class="info">Score: 7.0, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549967' target='https://doi.org/10.1101/2023.07.20.549967'> 10.1101/2023.07.20.549967</a></p>
        <p class="abstract">Deep learning algorithms, such as neural networks, enable the processing of complex datasets with many related variables, and have applications in disease diagnosis, cell profiling, and drug discovery. Beyond its use in electronic computers, neural networks have been implemented using programmable biomolecules such as DNA. This confers unique advantages such as greater portability, ability to operate without electricity, and direct analysis of patterns of biomolecules in solution. Analogous to past bottlenecks in electronic computers, the computing power of DNA-based neural networks is limited by the ability to add more computing units, i.e. neurons. This limitation exists because current architectures require many nucleic acids to model a single neuron. Each addition of a neuron to the network compounds existing problems such as long assembly times, high background signal, and cross-talk between components. Here we test three strategies to solve this limitation and improve the scalability of DNA-based neural networks: (i) enzymatic synthesis to generate high-purity neurons, (ii) spatial patterning of neuron clusters based on their network position, and (iii) encoding neuron connectivity on a universal single-stranded DNA backbone. We show that neurons implemented via these strategies activate quickly, with high signal-to-background ratio, and respond to varying input concentrations and weights. Using this neuron design, we implemented basic neural network motifs such as cascading, fan-in, and fan-out circuits. Since this design is modular, easy to synthesize, and compatible with multiple neural network architectures, we envision it will help scale DNA-based neural networks in a variety of settings. This will enable portable computing power for applications such as portable diagnostics, compact data storage, and autonomous decision making for lab-on-a-chips.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549749">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549749" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549749">
        <p class="paperTitle">On automated model discovery and a universal material subroutine</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549749" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549749" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peirlinck, M.; Linka, K.; Hurtado, J. A.; Kuhl, E.</p>
        <p class="info">Score: 5.9, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549749' target='https://doi.org/10.1101/2023.07.19.549749'> 10.1101/2023.07.19.549749</a></p>
        <p class="abstract">Constitutive modeling is the cornerstone of computational and structural mechanics. In a finite element analysis, the constitutive model is encoded in the material subroutine, a function that maps local strains onto stresses. This function is called within every finite element, at each integration point, within every time step, at each Newton iteration. Todays finite element packages offer large libraries of material subroutines to choose from. However, the scientific criteria for model selection remain highly subjective and prone to user bias. Here we fully automate the process of model selection, autonomously discover the best model and parameters from experimental data, encode all possible discoverable models into a single material subroutine, and seamlessly integrate this universal material subroutine into a finite element analysis. We prototype this strategy for tension, compression, and shear data from human brain tissue and perform a hyperelastic model discovery from twelve possible terms. These terms feature the first and second invariants, raised to the first and second powers, embedded in the identity, exponential, and logarithmic functions, generating 22x2x3 = 4096 models in total. We demonstrate how to integrate these models into a single universal material subroutine that features the classical neo Hooke, Blatz Ko, Mooney Rivlin, Demiray, Gent, and Holzapfel models as special cases. Finite element simulations with our universal material subroutine show that it specializes well to these widely used models, generalizes well to newly discovered models, and agrees excellently with both experimental data and previous simulations. It also performs well within realistic finite element simulations and accurately predicts stress concentrations in the human brain for six different head impact scenarios. We anticipate that integrating automated model discovery into a universal material subroutine will generalize naturally to more complex anisotropic, compressible, and inelastic materials and to other nonlinear finite element platforms. Replacing dozens of individual material subroutines by a single universal material subroutine that is populated directly via automated model discovery--entirely without human interaction--makes finite element analyses more accessible, more robust, and less vulnerable to human error. This could forever change how we simulate materials and structures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549947">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549947" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549947">
        <p class="paperTitle">Eliminating Malaria Vectors with Precision Guided Sterile Males</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549947" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549947" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smidler, A. L.; Apte, R. A.; Pai, J. J.; Chow, M. L.; Chen, S.; Mondal, A.; Sanchez C., H. M.; Antoshechkin, I.; Marshall, J. M.; Akbari, O. S.</p>
        <p class="info">Score: 13.5, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549947' target='https://doi.org/10.1101/2023.07.20.549947'> 10.1101/2023.07.20.549947</a></p>
        <p class="abstract">Controlling the principal African malaria vector, the mosquito Anopheles gambiae, is considered essential to curtail malaria transmission. However existing vector control technologies rely on insecticides, which are becoming increasingly ineffective. Sterile insect technique (SIT) is a powerful suppression approach that has successfully eradicated a number of insect pests, yet the A. gambiae toolkit lacks the requisite technologies for its implementation. SIT relies on iterative mass-releases of non-biting, non-driving, sterile males which seek out and mate with monandrous wild females. Once mated, females are permanently sterilized due to mating-induced refractoriness, which results in population suppression of the subsequent generation. However, sterilization by traditional methods renders males unfit, making the creation of precise genetic sterilization methods imperative. Here we develop precision guided Sterile Insect Technique (pgSIT) in the mosquito A. gambiae for inducible, programmed male-sterilization and female-elimination for wide scale use in SIT campaigns. Using a binary CRISPR strategy, we cross separate engineered Cas9 and gRNA strains to disrupt male-fertility and female-essential genes, yielding &gt;99.5% male-sterility and &gt;99.9% female-lethality in hybrid progeny. We demonstrate that these genetically sterilized males have good longevity, are able to induce population suppression in cage trials, and are predicted to eliminate wild A. gambiae populations using mathematical models, making them ideal candidates for release. This work provides a valuable addition to the malaria genetic biocontrol toolkit, for the first time enabling scalable SIT-like confinable suppression in the species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550304">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550304" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550304">
        <p class="paperTitle">Single-cell precision nanotechnology in vivo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550304" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550304" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: MOLBAY, M.; Kick, B.; Zhao, S.; Todorov, M. I.; Ohn, T.-L.; Roth, S.; Simats, A.; Singh, V.; Khalin, I.; Pan, C.; Singh Bhatia, H.; Hellal, F.; Zeidler, R.; Liesz, A.; Plesnila, N.; Dietz, H.; Erturk, A.</p>
        <p class="info">Score: 4.7, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550304' target='https://doi.org/10.1101/2023.07.24.550304'> 10.1101/2023.07.24.550304</a></p>
        <p class="abstract">Targeting nanoparticle therapeutics with cellular accuracy in whole organisms could open breakthrough opportunities in precision medicine. However, evaluating and fine-tuning the biodistribution of such systems in the whole organism at the cellular level remains a major obstacle. Here, we constructed targetable DNA origami, and analyzed biodistribution in transparent mice, in addition to studying tolerability, clearance kinetics, and immune response parameters. Untargeted DNA origami primarily accumulated in the spleen and the liver, while an immune cell-targeting variant successfully attached to immune cells throughout the body. A cancer cell-targeting mimetic co-localized on solid-tumor metastasis in the liver and the lung. These findings indicate that DNA origami can be directed in vivo, providing an important proof-of-concept and highlights the potential of high-resolution tissue-clearing imaging technologies in their development.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=121 SRC=&#34;FIGDIR/small/550304v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (44K):
org.highwire.dtl.DTLVardef@e6f63corg.highwire.dtl.DTLVardef@12ffe6dorg.highwire.dtl.DTLVardef@1e8e54aorg.highwire.dtl.DTLVardef@1256361_HPS_FORMAT_FIGEXP  M_FIG C_FIG HighlightsO_LIThis study demonstrates the potential of DNA origami-based drug delivery systems as versatile tool for for targeted delivery, which could be used to treat a range of diseases with applications.
C_LIO_LIThe immune compatibility, half-life, targeting efficiency, and the biodistribution evaluation of DNA origami indicate its potential for systemic drug delivery.
C_LIO_LIOur approach enables the assessment of biodistribution of nanoparticles in the intact body with a sensitivity to the single-cell level, highlighting the high-resolution tissue clearing technoloies in revealing DNA origamis feasibility for drug targeting.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549622">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549622" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549622">
        <p class="paperTitle">Response Surface Methodology-Based High-Throughput BioLector Fermentation Screening of Rosetta-2 for Enhanced Production of Insulin Variant: Towards Improved Production Efficiency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549622" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549622" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ayan, E.; Aytekin, A. O.; Kati, A.; Demirci, H.</p>
        <p class="info">Score: 5.5, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549622' target='https://doi.org/10.1101/2023.07.19.549622'> 10.1101/2023.07.19.549622</a></p>
        <p class="abstract">BackgroundIn order to obtain high yields of recombinant insulin and advancing therapeutic avenues for diabetes patients, the development of innovative designer insulin analogs have critical importance. The modified insulin analog presents a cost-effective remedy by being produced as inclusion bodies (IBs) within Escherichia coli BL21 (DE3) Rosetta-2 strain. This approach to production not only offers reduced production time but also yields high recovery rates. The prime aim of this investigation was to optimize the composition of the cultivation media, thereby accomplishing higher cell density fermentation of the proinsulin, that is fused with a cleavable hexa histidine purification tag.

ResultVarious factors, including carbon and nitrogen sources, salts, metal ions, and pH, were systematically investigated through experimental screening using the BioLector multiwell bright plate. Additionally, computational analysis employing the Plackett-Burman Design within the Design Expert software was utilized to assess their effectiveness in terms of insulin concentration as a surrogate measure of insulin yield. Among the tested variables, glucose, glycerol, MgSO4, and lower Luria-Bertani mix concentration have a significant influence on insulin production, as determined by the screening experiment. Subsequently, the Central Composite Design approach was operated to further evaluate and optimize the precise levels of these influential variables. This systematic methodology achieved an optimized cultivation media formulation, resulting in a remarkable enhancement of insulin production, with levels reaching up to 13 mg/ml when applied in BioLector fermentation.

ConclusionThe formulated cultivation media exhibited suitability for promoting high cell density fermentation of the modified insulin, thereby facilitating the attainment of optimal yields for the proinsulin expression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550063">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550063" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550063">
        <p class="paperTitle">Fibroblasts-dependent maturation and phenotype exacerbation of dystrophic hiPSC-derived MYOtissues enables muscle strength evaluation for gene therapy screening</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550063" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550063" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Palmieri, L.; Moula, M.; Jaber, A.; El Khoury, R.; Brochiet, G.; Bigot, A.; Israeli, D.; Richard, I.; Albini, S.</p>
        <p class="info">Score: 2.5, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550063' target='https://doi.org/10.1101/2023.07.26.550063'> 10.1101/2023.07.26.550063</a></p>
        <p class="abstract">Duchenne muscular dystrophy (DMD) is a lethal muscle wasting disease caused by absence of dystrophin, a protein essential to preserve muscle integrity continuously challenged by contractions. Gene therapy utilizing adeno-associated virus (AAV) to deliver truncated forms of dystrophin (uDys) is currently the most promising therapeutic approach. However, the therapeutic outcome in treated patients has not been as successful as anticipated by animal studies, underscoring the need of improved and high-throughput models for accurate and fast prediction of human response. Here, we describe the generation of MYOtissues, a 3D muscle platform based on direct myogenic conversion of human induced pluripotent stem cells (iPSC), whose structural and functional maturation is enhanced by fibroblasts incorporation. MYOtissues derived from DMD-iPSC including DMD fibroblasts, exacerbated pathogenic hallmarks such as fibrosis and muscle force loss. As a proof of concept, we showed that AAV-mediated uDys gene transfer improved muscle resistance and membrane stability in DMD-MYOtissues, highlighting the suitability of our system for gene therapy screening.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549397">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549397" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549397">
        <p class="paperTitle">Simultaneous multi-site editing of individual genomes using retron arrays</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549397" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549397" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.</p>
        <p class="info">Score: 29.1, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549397' target='https://doi.org/10.1101/2023.07.17.549397'> 10.1101/2023.07.17.549397</a></p>
        <p class="abstract">Our understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites. Unfortunately, building cells with multiple, non-adjacent precise mutations remains a laborious cycle of editing, isolating an edited cell, and editing again. Here, we describe a technology for precisely modifying multiple sites on a single genome simultaneously. This technology - termed a multitron - is built from a heavily modified retron, in which multiple donor-encoding msds are produced from a single transcript. The multitron architecture is compatible with both recombineering in prokaryotic cells and CRISPR editing in eukaryotic cells. We demonstrate applications for this approach in molecular recording, genetic element minimization, and metabolic engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550654">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550654" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550654">
        <p class="paperTitle">Soft robotics-enabled large animal model of HFpEF hemodynamics for device testing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550654" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550654" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rosalia, L.; Ozturk, C.; Wang, S.; Quevedo, D.; Saeed, M.; Mauskapf, A.; Roche, E.</p>
        <p class="info">Score: 2.1, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550654' target='https://doi.org/10.1101/2023.07.26.550654'> 10.1101/2023.07.26.550654</a></p>
        <p class="abstract">Heart failure with preserved ejection fraction (HFpEF) is a major challenge in cardiovascular medicine, accounting for approximately 50% of all cases of heart failure. Due to the lack of effective therapies for this condition, the mortality associated with HFpEF remains higher than that of most cancers. Despite the ongoing efforts, no medical device has yet received FDA approval. This is largely due to the lack of an in vivo model of the HFpEF hemodynamics, resulting in the inability to evaluate device effectiveness in vivo prior to clinical trials. Here, we describe the development of a highly tunable porcine model of HFpEF hemodynamics using implantable soft robotic sleeves, where controlled actuation of a left ventricular and an aortic sleeve can recapitulate changes in ventricular compliance and afterload associated with a broad spectrum of HFpEF hemodynamic phenotypes. We demonstrate the feasibility of the proposed model in preclinical testing by evaluating the hemodynamic response of the model post-implantation of an interatrial shunt device, which was found to be consistent with findings from in silico studies and clinical trials. This work addresses several of the limitations associated with previous models of HFpEF, such as their limited hemodynamic fidelity, elevated costs, lengthy development time, and low throughput. By showcasing exceptional versatility and tunability, the proposed platform has the potential to revolutionize the current approach for HFpEF device development and selection, with the goal of improving the quality of life for the 32 million people affected by HFpEF worldwide.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550409">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550409" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550409">
        <p class="paperTitle">Autologous iPSC- and MSC-derived Chondrocyte Implants for Cartilage Repair in a Miniature Pig Model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550409" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550409" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, M.-S.; Sivapatham, A.; Leiferman, E. M.; Jiao, H.; Lu, Y.; Nemke, B. W.; Leiferman, M.; Markel, M. D.; Li, W.-J.</p>
        <p class="info">Score: 2.1, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550409' target='https://doi.org/10.1101/2023.07.25.550409'> 10.1101/2023.07.25.550409</a></p>
        <p class="abstract">Induced pluripotent stem cell (iPSC)-derived mesenchymal stem cells (iMSCs) have greater potential for generating chondrocytes without hypertrophic and fibrotic phenotypes compared to bone marrow-derived mesenchymal stem/stromal cells (BMSCs). However, there is a lack of research demonstrating the use of autologous iMSCs for repairing articular chondral lesions in large animal models. In this study, we aimed to evaluate the effectiveness of autologous miniature pig (minipig) iMSC-chondrocyte (iMSC-Ch)-laden implants in comparison to autologous BMSC-chondrocyte (BMSC-Ch)-laden implants for cartilage repair in porcine femoral condyles. iMSCs and BMSCs were seeded into fibrin glue/nanofiber constructs and cultured with chondrogenic induction media for 7 days before implantation. To assess the regenerative capacity of the cells, 19 skeletally mature Yucatan minipigs were randomly divided into microfracture control, acellular scaffold, iMSC, and BMSC subgroups. A cylindrical defect measuring 7 mm in diameter and 0.6 mm in depth was created on the articular cartilage surface without violating the subchondral bone. The defects were then left untreated or treated with acellular or cellular implants. Both cellular implant-treated groups exhibited enhanced joint repair compared to the microfracture and acellular control groups. Immunofluorescence analysis yielded significant findings, showing that cartilage treated with iMSC-Ch implants exhibited higher expression of COL2A1 and minimal to no expression of COL1A1 and COL10A1, in contrast to the BMSC-Ch-treated group. This indicates that the iMSC-Ch implants generated more hyaline cartilage-like tissue compared to the BMSC-Ch implants. These results contribute to filling the knowledge gap regarding the potential of autologous iPSC derivatives for cartilage repair in translational animal models.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
