<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Volumetric imaging of an intact organism by a distributed molecular network
Authors: Weinstein, J. A.; Qian, N.
Score: 42.1, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553025
Lymphatic, nervous, and tumoral tissues, among others, exhibit physiology that emerges from three-dimensional interactions between genetically unique cells. A technology capable of volumetrically imaging transcriptomes, genotypes, and morphologies in a single de novo measurement would therefore provide a critical view into the biological complexity of living systems.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Volumetric imaging of an intact organism by a distributed molecular network
Authors: Weinstein, J. A.; Qian, N.
Score: 42.1, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553025
Lymphatic, nervous, and tumoral tissues, among others, exhibit physiology that emerges from three-dimensional interactions between genetically unique cells. A technology capable of volumetrically imaging transcriptomes, genotypes, and morphologies in a single de novo measurement would therefore provide a critical view into the biological complexity of living systems." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-23T10:37:56+00:00" />
<meta property="article:modified_time" content="2023-08-23T10:37:56+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Volumetric imaging of an intact organism by a distributed molecular network
Authors: Weinstein, J. A.; Qian, N.
Score: 42.1, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553025
Lymphatic, nervous, and tumoral tissues, among others, exhibit physiology that emerges from three-dimensional interactions between genetically unique cells. A technology capable of volumetrically imaging transcriptomes, genotypes, and morphologies in a single de novo measurement would therefore provide a critical view into the biological complexity of living systems."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Volumetric imaging of an intact organism by a distributed molecular network\nAuthors: Weinstein, J. A.; Qian, N.\nScore: 42.1, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553025\nLymphatic, nervous, and tumoral tissues, among others, exhibit physiology that emerges from three-dimensional interactions between genetically unique cells. A technology capable of volumetrically imaging transcriptomes, genotypes, and morphologies in a single de novo measurement would therefore provide a critical view into the biological complexity of living systems.",
  "keywords": [
    
  ],
  "articleBody": " Volumetric imaging of an intact organism by a distributed molecular network\nAuthors: Weinstein, J. A.; Qian, N.\nScore: 42.1, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553025\nLymphatic, nervous, and tumoral tissues, among others, exhibit physiology that emerges from three-dimensional interactions between genetically unique cells. A technology capable of volumetrically imaging transcriptomes, genotypes, and morphologies in a single de novo measurement would therefore provide a critical view into the biological complexity of living systems. Here we achieve this by extending DNA microscopy, an imaging modality that encodes a spatio-genetic map of a specimen via a massive distributed network of DNA molecules inside it, to three dimensions and multiple length scales in developing zebrafish embryos.\nTransferrin Receptor Targeting Chimeras (TransTACs) for Membrane Protein Degradation\nAuthors: Zhang, D.; Duque-Jimenez, J.; Brixi, G.; Facchinetti, F.; Rhee, K.; Feng, W. W.; JaÌˆnne, P. A.; Zhou, X.\nScore: 40.8, Published: 2023-08-13 DOI: 10.1101/2023.08.10.552782\nCancer cells require high levels of iron for rapid proliferation, leading to a significant upregulation of the iron carrier protein Transferrin Receptor (TfR) on their cell surface. Leveraging this phenomenon and the exceptionally fast endocytosis rate of TfR, we introduce Transferrin Receptor TArgeting Chimeras (TransTAC), a novel molecular archetype for membrane protein degradation in cancers and other cell types. TransTACs repurpose the naturally recycling receptor TfR1 for protein degradation. To accomplish this, we utilized a combination of protein engineering strategies to redirect the target protein from recycling-endosome trafficking to lysosomal degradation. We show that TransTACs can highly efficiently degrade a diverse range of single-pass, multi-pass, native, or synthetic membrane proteins, establishing new possibilities for targeted cancer therapy.\nThe Enderstruder: An accessible open-source syringe extruder compatible with Ender series 3D printers\nAuthors: Cordova, D. J.; Rodriguez, A. A.; Woodward, S. C.; Crosby, C. O.\nScore: 6.3, Published: 2023-08-17 DOI: 10.1101/2023.08.15.553241\nBioprinting has enabled the precise spatiotemporal deposition of cell-containing bioinks, opening new avenues of research in tissue engineering and regenerative medicine. Although several open-source syringe extruder designs have been published and adopted by end users, only one has been specifically adapted for the Ender series, an affordable and open-source line of thermoplastic 3D printers. Here, we introduce the Enderstruder, a cost-effective extruder attachment that uses a standard 10 mL BD syringe, positions the stepper motor at the level of the gantry, enhances x-axis stability with a linear rail, and uses the originally included stepper motor, resulting in reduced cost and simplified assembly. Furthermore, we present an iterative process to fine-tune printing profiles for high-viscosity biomaterial inks. To facilitate the implementation of our work by other groups, we provide fully editable Cura profiles for five commonly used biomaterials. We employ the Enderstruder to print established calibration patterns and complex shapes using each of these five materials to validate and characterize our design. By presenting the Enderstruder and its iterative development process, this study contributes to the growing repository of open-source bioprinting solutions, fostering greater accessibility and affordability for researchers in tissue engineering. Specifications table O_TBL View this table: org.highwire.dtl.DTLVardef@9e65d6org.highwire.dtl.DTLVardef@12feefaorg.highwire.dtl.DTLVardef@1f26e0eorg.highwire.dtl.DTLVardef@11774cdorg.highwire.dtl.DTLVardef@f55f17_HPS_FORMAT_FIGEXP M_TBL C_TBL\nA Hybrid Transistor with Transcriptionally Controlled Computation and Plasticity\nAuthors: Gao, Y.; Zhou, Y.; Ji, X.; Graham, A. J.; Dundas, C. M.; Mahfoud, I. E. M.; Tibbett, B. M.; Tan, B.; Partipilo, G.; Dodabalapur, A.; Rivnay, J.; Keitz, B. K.\nScore: 5.4, Published: 2023-08-18 DOI: 10.1101/2023.08.16.553547\nOrganic electrochemical transistors (OECTs) are ideal devices for translating biological signals into electrical readouts and have applications in bioelectronics, biosensing, and neuromorphic computing. Despite their potential, developing programmable and modular methods for living systems to interface with OECTs has proven challenging. Here we describe hybrid OECTs containing the model electroactive bacterium Shewanella oneidensis that enable the transduction of biological computations to electrical responses. Specifically, we fabricated planar p-type OECTs and demonstrated that channel de-doping is driven by extracellular electron transfer (EET) from S. oneidensis. Leveraging this mechanistic understanding and our ability to control EET flux via transcriptional regulation, we used plasmid-based Boolean logic gates to translate biological computation into current changes within the OECT. Finally, we demonstrated EET-driven changes to OECT synaptic plasticity. This work enables fundamental EET studies and OECT- based biosensing and biocomputing systems with genetically controllable and modular design elements.\nConserved and tissue-specific immune responses to biologic scaffold implantation\nAuthors: DeStefano, S.; Josyula, A.; Faust, M.; Fertil, D.; Lokwani, R.; Ngo, T. B.; Sadtler, K.\nScore: 5.4, Published: 2023-08-17 DOI: 10.1101/2023.08.15.553390\n1.Upon implantation into a patient, any biomaterial induces a cascade of immune responses that influences the outcome of that device. This cascade depends upon several factors, including the composition of the material itself and the location in which the material is implanted. There is still significant uncertainty around the role of different tissue microenvironments in the immune response to biomaterials and how that may alter downstream scaffold remodeling and integration. In this study, we present a study evaluating the immune response to decellularized extracellular matrix materials within the intraperitoneal cavity, the subcutaneous space, and in a traumatic skeletal muscle injury microenvironment. All different locations induced robust cellular recruitment, specifically of macrophages and eosinophils. The latter was most prominent in the subcutaneous space. Intraperitoneal implants uniquely recruited B cells that may alter downstream reactivity as adaptive immunity has been strongly implicated in the outcome of scaffold remodeling. These data suggest that the location of tissue implants should be taken together with the composition of the material itself when designing devices for downline therapeutics.\nSingle-cell precision nanotechnology in vivo\nAuthors: MOLBAY, M.; Kick, B.; Zhao, S.; Todorov, M. I.; Ohn, T.-L.; Roth, S.; Simats, A.; Singh, V.; Khalin, I.; Pan, C.; Singh Bhatia, H.; Hellal, F.; Zeidler, R.; Liesz, A.; Plesnila, N.; Dietz, H.; Erturk, A.\nScore: 150.6, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550304\nTargeting nanoparticle therapeutics with cellular accuracy in whole organisms could open breakthrough opportunities in precision medicine. However, evaluating and fine-tuning the biodistribution of such systems in the whole organism at the cellular level remains a major obstacle. Here, we constructed targetable DNA origami, and analyzed biodistribution in transparent mice, in addition to studying tolerability, clearance kinetics, and immune response parameters. Untargeted DNA origami primarily accumulated in the spleen and the liver, while an immune cell-targeting variant successfully attached to immune cells throughout the body. A cancer cell-targeting mimetic co-localized on solid-tumor metastasis in the liver and the lung. These findings indicate that DNA origami can be directed in vivo, providing an important proof-of-concept and highlights the potential of high-resolution tissue-clearing imaging technologies in their development. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=121 SRC=\"FIGDIR/small/550304v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (44K): org.highwire.dtl.DTLVardef@ce216borg.highwire.dtl.DTLVardef@f40bb4org.highwire.dtl.DTLVardef@2c6a66org.highwire.dtl.DTLVardef@1fd7fde_HPS_FORMAT_FIGEXP M_FIG C_FIG HighlightsO_LIThis study demonstrates the potential of DNA origami-based drug delivery systems as versatile tool for for targeted delivery, which could be used to treat a range of diseases with applications. C_LIO_LIThe immune compatibility, half-life, targeting efficiency, and the biodistribution evaluation of DNA origami indicate its potential for systemic drug delivery. C_LIO_LIOur approach enables the assessment of biodistribution of nanoparticles in the intact body with a sensitivity to the single-cell level, highlighting the high-resolution tissue clearing technoloies in revealing DNA origamis feasibility for drug targeting. C_LI\nSecond Harmonic Generation Imaging Reveals Entanglement of Collagen Fibers in the Elephant Trunk Skin Dermis\nAuthors: Schulz, A.; Plotczyk, M.; Sordilla, S.; Boyle, M.; Singal, K.; Reidenberg, J.; Hu, D.; Higgins, C.\nScore: 8.7, Published: 2023-08-15 DOI: 10.1101/2023.08.11.553031\nForm-function relationships often have tradeoffs: if a material is tough, it is often inflexible, and vice versa. This is particularly relevant for the elephant trunk, where the skin should be protective yet elastic. To investigate how this is achieved, we used classical histochemical staining and second harmonic generation microscopy to describe the morphology and composition of elephant trunk skin. We report structure at the macro and micro scales, from the thickness of the dermis to the interaction of 10 m thick collagen fibers. We analyzed several sites along the length of the trunk, to compare and contrast the dorsal-ventral and proximal-distal skin morphologies and compositions. We find the dorsal skin of the elephant trunk can have keratin armor layers over 2mm thick, which is nearly 100 times the thickness of the equivalent layer in human skin. We also found that the structural support layer (the dermis) of elephant trunk contains a distribution of collagen-I (COL1) fibers in both perpendicular and parallel arrangement. The bimodal distribution of collagen is seen across all portions of the trunk, and is dissimilar from that of human skin where one orientation dominates within a body site. We hypothesize that this distribution of COL1 in the elephant trunk allows both flexibility and load-bearing capabilities. Additionally, when viewing individual fiber interaction of 10 m thick collagen, we find the fiber crossings per unit volume are five times more common than in human skin, suggesting that the fibers are entangled. We surmise that these intriguing structures permit both flexibility and strength in the elephant trunk. The complex nature of the elephant skin may inspire the design of materials that can combine strength and flexibility.\nBioengineering of a human innervated cardiac muscle model\nAuthors: Schneider, L. V.; Bao, G.; Methi, A.; Jensen, O.; Schmoll, K. A.; Setya, M. G.; Sakib, M. S.; Fahud, A. L.; Brockmoeller, J.; Fischer, A.; Liaw, N. Y.; Zimmermann, W.-H.; Zafeiriou, M.-P.\nScore: 4.5, Published: 2023-08-19 DOI: 10.1101/2023.08.18.552653\nCardiac autonomic neurons control cardiac contractility. Dysregulation of the autonomic nervous system can lead to sympathetic overdrive resulting in heart failure and an increased incidence of fatal arrhythmias. Here, we introduce innervated engineered human myocardium (iEHM), a novel model of neuro-cardiac junctions, constructed by fusion of a bioengineered neural organoid (BENO) patterned to autonomic nervous system and engineered human myocardium (EHM). Projections of sympathetic neurons into engineered human myocardium formed presynaptic terminals in close proximity to cardiomyocytes and an extensive vascular network co-developing in the tissues. Contractile responses to optogenetic stimulation of the accordingly engineered neuronal component demonstrated functionality of the engineered neuro-cardiac junctions in iEHM. This model will serve as a human surrogate system to delineate neuron and cardiac cell contribution to brain and heart diseases and is an important step towards engineering a human brain to heart axis in a dish.\nDNA Nanostructure-Guided Assembly of Proteins into Programmable Shapes\nAuthors: Lu, Q.; Xu, Y.; Poppleton, E.; Zhou, K.; Sulc, P.; Stephanopoulos, N.; Ke, Y.\nScore: 5.3, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552347\nThe development of methods to synthesize protein oligomers with precisely controlled number and configuration of components will enable artificial protein complexes and nanostructures with diverse biological and medical applications. Using DNA hybridization to link protein-DNA conjugates provides for a programmability that can be difficult to achieve with other methods, such as engineering multiple orthogonal protein-protein interfaces. However, it is still difficult to construct well-defined protein assemblies, and the challenge is magnified if only a single type of building block is used. To overcome this hurdle, we use a DNA origami as an \"assembler\" to guide the linking of protein-DNA conjugates using a series of oligonucleotide hybridization and displacement operations. We constructed several isomeric protein nanostructures on a DNA origami platform by using a C3-symmetric building block comprised of a protein trimer modified with DNA handles. By changing the number of protein-DNA building blocks attached to the origami, and the sequence of the linking and displacement strands added, we were able to produce dimers, two types of trimer structures, and three types of tetramer assemblies. Our approach expands the scope for the precise design and assembly of protein-based nanostructures, and will enable the formulation of functional protein complexes with stoichiometric and geometric control.\nHigh-throughput generic single-entity sequencing using droplet microfluidics\nAuthors: Wang, G.; Zhao, L.; Shi, Y.; Qu, F.; Ding, Y.; Liu, W.; Liu, C.; Luo, G.; Li, M.; Bai, X.; Li, L.; Ho, Y.-P.; Yu, J.\nScore: 2.5, Published: 2023-08-15 DOI: 10.1101/2023.08.15.549386\nSingle-cell sequencing has revolutionized our understanding of cellular heterogeneity by providing a micro-level perspective over the past decades. Although heterogeneity is essential for various biological communities, the currently demonstrated platform predominantly focuses on eukaryotic cells without cell walls and their transcriptomics 1,2, leaving significant gaps in the study of omics from other single biological entities such as bacteria and viruses. Due to the difficulty of isolating and acquiring their DNA3, contemporary methodologies for the characterization of generic biological entities remain conspicuously constrained, with low throughput4, compromised lysis efficiency5, and highly fragmented genomes6. Herein, we present the Generic Single Entity Sequencing platform (GSE-Seq), which boasts ample versatility, high throughput, and high coverage, and is enabled by an innovative workflow, addressing the critical challenges in single entities sequencing: (1) one-step manufacturing of massive barcodes, (2) degradable hydrogel-based in situ sample processing and whole genome amplification, (3) integrated in-drop library preparation, (4) compatible long-read sequencing. By GSE-Seq, we have achieved a significant milestone by enabling high-throughput, long-read single-entity profiling of dsDNA and ssDNA from single virus sequencing (SV-seq) and single bacteria sequencing (SB-seq) of the human gut and marine sediment for the first time. Notably, our analysis uncovered previously overlooked viral and bacterial dark matter and phage-host interactions. In summary, the presented conceptually new workflow offers a toolbox based on droplet microfluidics to tackle the persistent challenges in high-throughput profiling to generic applications, which hold immense promise for diverse biological entities, especially hard-to-lyse cells.\n",
  "wordCount" : "2175",
  "inLanguage": "en",
  "datePublished": "2023-08-23T10:37:56Z",
  "dateModified": "2023-08-23T10:37:56Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on August 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553025">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553025" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553025">
        <p class="paperTitle">Volumetric imaging of an intact organism by a distributed molecular network</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553025" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553025" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weinstein, J. A.; Qian, N.</p>
        <p class="info">Score: 42.1, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553025' target='https://doi.org/10.1101/2023.08.11.553025'> 10.1101/2023.08.11.553025</a></p>
        <p class="abstract">Lymphatic, nervous, and tumoral tissues, among others, exhibit physiology that emerges from three-dimensional interactions between genetically unique cells. A technology capable of volumetrically imaging transcriptomes, genotypes, and morphologies in a single de novo measurement would therefore provide a critical view into the biological complexity of living systems. Here we achieve this by extending DNA microscopy, an imaging modality that encodes a spatio-genetic map of a specimen via a massive distributed network of DNA molecules inside it, to three dimensions and multiple length scales in developing zebrafish embryos.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552782">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552782" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552782">
        <p class="paperTitle">Transferrin Receptor Targeting Chimeras (TransTACs) for Membrane Protein Degradation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552782" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552782" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, D.; Duque-Jimenez, J.; Brixi, G.; Facchinetti, F.; Rhee, K.; Feng, W. W.; JaÌˆnne, P. A.; Zhou, X.</p>
        <p class="info">Score: 40.8, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552782' target='https://doi.org/10.1101/2023.08.10.552782'> 10.1101/2023.08.10.552782</a></p>
        <p class="abstract">Cancer cells require high levels of iron for rapid proliferation, leading to a significant upregulation of the iron carrier protein Transferrin Receptor (TfR) on their cell surface. Leveraging this phenomenon and the exceptionally fast endocytosis rate of TfR, we introduce Transferrin Receptor TArgeting Chimeras (TransTAC), a novel molecular archetype for membrane protein degradation in cancers and other cell types. TransTACs repurpose the naturally recycling receptor TfR1 for protein degradation. To accomplish this, we utilized a combination of protein engineering strategies to redirect the target protein from recycling-endosome trafficking to lysosomal degradation. We show that TransTACs can highly efficiently degrade a diverse range of single-pass, multi-pass, native, or synthetic membrane proteins, establishing new possibilities for targeted cancer therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553241">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553241" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553241">
        <p class="paperTitle">The Enderstruder: An accessible open-source syringe extruder compatible with Ender series 3D printers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553241" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553241" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cordova, D. J.; Rodriguez, A. A.; Woodward, S. C.; Crosby, C. O.</p>
        <p class="info">Score: 6.3, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553241' target='https://doi.org/10.1101/2023.08.15.553241'> 10.1101/2023.08.15.553241</a></p>
        <p class="abstract">Bioprinting has enabled the precise spatiotemporal deposition of cell-containing bioinks, opening new avenues of research in tissue engineering and regenerative medicine. Although several open-source syringe extruder designs have been published and adopted by end users, only one has been specifically adapted for the Ender series, an affordable and open-source line of thermoplastic 3D printers. Here, we introduce the Enderstruder, a cost-effective extruder attachment that uses a standard 10 mL BD syringe, positions the stepper motor at the level of the gantry, enhances x-axis stability with a linear rail, and uses the originally included stepper motor, resulting in reduced cost and simplified assembly. Furthermore, we present an iterative process to fine-tune printing profiles for high-viscosity biomaterial inks. To facilitate the implementation of our work by other groups, we provide fully editable Cura profiles for five commonly used biomaterials. We employ the Enderstruder to print established calibration patterns and complex shapes using each of these five materials to validate and characterize our design. By presenting the Enderstruder and its iterative development process, this study contributes to the growing repository of open-source bioprinting solutions, fostering greater accessibility and affordability for researchers in tissue engineering.

Specifications table

O_TBL View this table:
org.highwire.dtl.DTLVardef@9e65d6org.highwire.dtl.DTLVardef@12feefaorg.highwire.dtl.DTLVardef@1f26e0eorg.highwire.dtl.DTLVardef@11774cdorg.highwire.dtl.DTLVardef@f55f17_HPS_FORMAT_FIGEXP  M_TBL C_TBL</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553547">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553547" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553547">
        <p class="paperTitle">A Hybrid Transistor with Transcriptionally Controlled Computation and Plasticity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553547" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553547" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gao, Y.; Zhou, Y.; Ji, X.; Graham, A. J.; Dundas, C. M.; Mahfoud, I. E. M.; Tibbett, B. M.; Tan, B.; Partipilo, G.; Dodabalapur, A.; Rivnay, J.; Keitz, B. K.</p>
        <p class="info">Score: 5.4, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553547' target='https://doi.org/10.1101/2023.08.16.553547'> 10.1101/2023.08.16.553547</a></p>
        <p class="abstract">Organic electrochemical transistors (OECTs) are ideal devices for translating biological signals into electrical readouts and have applications in bioelectronics, biosensing, and neuromorphic computing. Despite their potential, developing programmable and modular methods for living systems to interface with OECTs has proven challenging. Here we describe hybrid OECTs containing the model electroactive bacterium Shewanella oneidensis that enable the transduction of biological computations to electrical responses. Specifically, we fabricated planar p-type OECTs and demonstrated that channel de-doping is driven by extracellular electron transfer (EET) from S. oneidensis. Leveraging this mechanistic understanding and our ability to control EET flux via transcriptional regulation, we used plasmid-based Boolean logic gates to translate biological computation into current changes within the OECT. Finally, we demonstrated EET-driven changes to OECT synaptic plasticity. This work enables fundamental EET studies and OECT- based biosensing and biocomputing systems with genetically controllable and modular design elements.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553390">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553390" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553390">
        <p class="paperTitle">Conserved and tissue-specific immune responses to biologic scaffold implantation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553390" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553390" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: DeStefano, S.; Josyula, A.; Faust, M.; Fertil, D.; Lokwani, R.; Ngo, T. B.; Sadtler, K.</p>
        <p class="info">Score: 5.4, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553390' target='https://doi.org/10.1101/2023.08.15.553390'> 10.1101/2023.08.15.553390</a></p>
        <p class="abstract">1.Upon implantation into a patient, any biomaterial induces a cascade of immune responses that influences the outcome of that device. This cascade depends upon several factors, including the composition of the material itself and the location in which the material is implanted. There is still significant uncertainty around the role of different tissue microenvironments in the immune response to biomaterials and how that may alter downstream scaffold remodeling and integration. In this study, we present a study evaluating the immune response to decellularized extracellular matrix materials within the intraperitoneal cavity, the subcutaneous space, and in a traumatic skeletal muscle injury microenvironment. All different locations induced robust cellular recruitment, specifically of macrophages and eosinophils. The latter was most prominent in the subcutaneous space. Intraperitoneal implants uniquely recruited B cells that may alter downstream reactivity as adaptive immunity has been strongly implicated in the outcome of scaffold remodeling. These data suggest that the location of tissue implants should be taken together with the composition of the material itself when designing devices for downline therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550304">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550304" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550304">
        <p class="paperTitle">Single-cell precision nanotechnology in vivo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550304" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550304" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: MOLBAY, M.; Kick, B.; Zhao, S.; Todorov, M. I.; Ohn, T.-L.; Roth, S.; Simats, A.; Singh, V.; Khalin, I.; Pan, C.; Singh Bhatia, H.; Hellal, F.; Zeidler, R.; Liesz, A.; Plesnila, N.; Dietz, H.; Erturk, A.</p>
        <p class="info">Score: 150.6, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550304' target='https://doi.org/10.1101/2023.07.24.550304'> 10.1101/2023.07.24.550304</a></p>
        <p class="abstract">Targeting nanoparticle therapeutics with cellular accuracy in whole organisms could open breakthrough opportunities in precision medicine. However, evaluating and fine-tuning the biodistribution of such systems in the whole organism at the cellular level remains a major obstacle. Here, we constructed targetable DNA origami, and analyzed biodistribution in transparent mice, in addition to studying tolerability, clearance kinetics, and immune response parameters. Untargeted DNA origami primarily accumulated in the spleen and the liver, while an immune cell-targeting variant successfully attached to immune cells throughout the body. A cancer cell-targeting mimetic co-localized on solid-tumor metastasis in the liver and the lung. These findings indicate that DNA origami can be directed in vivo, providing an important proof-of-concept and highlights the potential of high-resolution tissue-clearing imaging technologies in their development.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=121 SRC=&#34;FIGDIR/small/550304v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (44K):
org.highwire.dtl.DTLVardef@ce216borg.highwire.dtl.DTLVardef@f40bb4org.highwire.dtl.DTLVardef@2c6a66org.highwire.dtl.DTLVardef@1fd7fde_HPS_FORMAT_FIGEXP  M_FIG C_FIG HighlightsO_LIThis study demonstrates the potential of DNA origami-based drug delivery systems as versatile tool for for targeted delivery, which could be used to treat a range of diseases with applications.
C_LIO_LIThe immune compatibility, half-life, targeting efficiency, and the biodistribution evaluation of DNA origami indicate its potential for systemic drug delivery.
C_LIO_LIOur approach enables the assessment of biodistribution of nanoparticles in the intact body with a sensitivity to the single-cell level, highlighting the high-resolution tissue clearing technoloies in revealing DNA origamis feasibility for drug targeting.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553031">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553031" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553031">
        <p class="paperTitle">Second Harmonic Generation Imaging Reveals Entanglement of Collagen Fibers in the Elephant Trunk Skin Dermis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553031" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553031" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schulz, A.; Plotczyk, M.; Sordilla, S.; Boyle, M.; Singal, K.; Reidenberg, J.; Hu, D.; Higgins, C.</p>
        <p class="info">Score: 8.7, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553031' target='https://doi.org/10.1101/2023.08.11.553031'> 10.1101/2023.08.11.553031</a></p>
        <p class="abstract">Form-function relationships often have tradeoffs: if a material is tough, it is often inflexible, and vice versa. This is particularly relevant for the elephant trunk, where the skin should be protective yet elastic. To investigate how this is achieved, we used classical histochemical staining and second harmonic generation microscopy to describe the morphology and composition of elephant trunk skin. We report structure at the macro and micro scales, from the thickness of the dermis to the interaction of 10 m thick collagen fibers. We analyzed several sites along the length of the trunk, to compare and contrast the dorsal-ventral and proximal-distal skin morphologies and compositions. We find the dorsal skin of the elephant trunk can have keratin armor layers over 2mm thick, which is nearly 100 times the thickness of the equivalent layer in human skin. We also found that the structural support layer (the dermis) of elephant trunk contains a distribution of collagen-I (COL1) fibers in both perpendicular and parallel arrangement. The bimodal distribution of collagen is seen across all portions of the trunk, and is dissimilar from that of human skin where one orientation dominates within a body site. We hypothesize that this distribution of COL1 in the elephant trunk allows both flexibility and load-bearing capabilities. Additionally, when viewing individual fiber interaction of 10 m thick collagen, we find the fiber crossings per unit volume are five times more common than in human skin, suggesting that the fibers are entangled. We surmise that these intriguing structures permit both flexibility and strength in the elephant trunk. The complex nature of the elephant skin may inspire the design of materials that can combine strength and flexibility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.552653">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.552653" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.552653">
        <p class="paperTitle">Bioengineering of a human innervated cardiac muscle model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.552653" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.552653" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schneider, L. V.; Bao, G.; Methi, A.; Jensen, O.; Schmoll, K. A.; Setya, M. G.; Sakib, M. S.; Fahud, A. L.; Brockmoeller, J.; Fischer, A.; Liaw, N. Y.; Zimmermann, W.-H.; Zafeiriou, M.-P.</p>
        <p class="info">Score: 4.5, Published: 2023-08-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.552653' target='https://doi.org/10.1101/2023.08.18.552653'> 10.1101/2023.08.18.552653</a></p>
        <p class="abstract">Cardiac autonomic neurons control cardiac contractility. Dysregulation of the autonomic nervous system can lead to sympathetic overdrive resulting in heart failure and an increased incidence of fatal arrhythmias. Here, we introduce innervated engineered human myocardium (iEHM), a novel model of neuro-cardiac junctions, constructed by fusion of a bioengineered neural organoid (BENO) patterned to autonomic nervous system and engineered human myocardium (EHM). Projections of sympathetic neurons into engineered human myocardium formed presynaptic terminals in close proximity to cardiomyocytes and an extensive vascular network co-developing in the tissues. Contractile responses to optogenetic stimulation of the accordingly engineered neuronal component demonstrated functionality of the engineered neuro-cardiac junctions in iEHM. This model will serve as a human surrogate system to delineate neuron and cardiac cell contribution to brain and heart diseases and is an important step towards engineering a human brain to heart axis in a dish.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552347">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552347" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552347">
        <p class="paperTitle">DNA Nanostructure-Guided Assembly of Proteins into Programmable Shapes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552347" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552347" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lu, Q.; Xu, Y.; Poppleton, E.; Zhou, K.; Sulc, P.; Stephanopoulos, N.; Ke, Y.</p>
        <p class="info">Score: 5.3, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552347' target='https://doi.org/10.1101/2023.08.07.552347'> 10.1101/2023.08.07.552347</a></p>
        <p class="abstract">The development of methods to synthesize protein oligomers with precisely controlled number and configuration of components will enable artificial protein complexes and nanostructures with diverse biological and medical applications. Using DNA hybridization to link protein-DNA conjugates provides for a programmability that can be difficult to achieve with other methods, such as engineering multiple orthogonal protein-protein interfaces. However, it is still difficult to construct well-defined protein assemblies, and the challenge is magnified if only a single type of building block is used. To overcome this hurdle, we use a DNA origami as an &#34;assembler&#34; to guide the linking of protein-DNA conjugates using a series of oligonucleotide hybridization and displacement operations. We constructed several isomeric protein nanostructures on a DNA origami platform by using a C3-symmetric building block comprised of a protein trimer modified with DNA handles. By changing the number of protein-DNA building blocks attached to the origami, and the sequence of the linking and displacement strands added, we were able to produce dimers, two types of trimer structures, and three types of tetramer assemblies. Our approach expands the scope for the precise design and assembly of protein-based nanostructures, and will enable the formulation of functional protein complexes with stoichiometric and geometric control.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.549386">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.549386" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.549386">
        <p class="paperTitle">High-throughput generic single-entity sequencing using droplet microfluidics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.549386" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.549386" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, G.; Zhao, L.; Shi, Y.; Qu, F.; Ding, Y.; Liu, W.; Liu, C.; Luo, G.; Li, M.; Bai, X.; Li, L.; Ho, Y.-P.; Yu, J.</p>
        <p class="info">Score: 2.5, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.549386' target='https://doi.org/10.1101/2023.08.15.549386'> 10.1101/2023.08.15.549386</a></p>
        <p class="abstract">Single-cell sequencing has revolutionized our understanding of cellular heterogeneity by providing a micro-level perspective over the past decades. Although heterogeneity is essential for various biological communities, the currently demonstrated platform predominantly focuses on eukaryotic cells without cell walls and their transcriptomics 1,2, leaving significant gaps in the study of omics from other single biological entities such as bacteria and viruses. Due to the difficulty of isolating and acquiring their DNA3, contemporary methodologies for the characterization of generic biological entities remain conspicuously constrained, with low throughput4, compromised lysis efficiency5, and highly fragmented genomes6. Herein, we present the Generic Single Entity Sequencing platform (GSE-Seq), which boasts ample versatility, high throughput, and high coverage, and is enabled by an innovative workflow, addressing the critical challenges in single entities sequencing: (1) one-step manufacturing of massive barcodes, (2) degradable hydrogel-based in situ sample processing and whole genome amplification, (3) integrated in-drop library preparation, (4) compatible long-read sequencing. By GSE-Seq, we have achieved a significant milestone by enabling high-throughput, long-read single-entity profiling of dsDNA and ssDNA from single virus sequencing (SV-seq) and single bacteria sequencing (SB-seq) of the human gut and marine sediment for the first time. Notably, our analysis uncovered previously overlooked viral and bacterial dark matter and phage-host interactions. In summary, the presented conceptually new workflow offers a toolbox based on droplet microfluidics to tackle the persistent challenges in high-throughput profiling to generic applications, which hold immense promise for diverse biological entities, especially hard-to-lyse cells.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
