<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Structure-conditioned masked language models for protein sequence design generalize beyond the native sequence space
Authors: Akpinaroglu, D.; Seki, K.; Guo, A.; Zhu, E.; Kelly, M. J.; Kortemme, T.
Score: 23.3, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571823
Machine learning has revolutionized computational protein design, enabling significant progress in protein backbone generation and sequence design. Here, we introduce Frame2seq, a structure-conditioned masked language model for protein sequence design. Frame2seq generates sequences in a single pass, achieves 49.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Structure-conditioned masked language models for protein sequence design generalize beyond the native sequence space
Authors: Akpinaroglu, D.; Seki, K.; Guo, A.; Zhu, E.; Kelly, M. J.; Kortemme, T.
Score: 23.3, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571823
Machine learning has revolutionized computational protein design, enabling significant progress in protein backbone generation and sequence design. Here, we introduce Frame2seq, a structure-conditioned masked language model for protein sequence design. Frame2seq generates sequences in a single pass, achieves 49." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-03T10:39:43+00:00" />
<meta property="article:modified_time" content="2024-01-03T10:39:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Structure-conditioned masked language models for protein sequence design generalize beyond the native sequence space
Authors: Akpinaroglu, D.; Seki, K.; Guo, A.; Zhu, E.; Kelly, M. J.; Kortemme, T.
Score: 23.3, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571823
Machine learning has revolutionized computational protein design, enabling significant progress in protein backbone generation and sequence design. Here, we introduce Frame2seq, a structure-conditioned masked language model for protein sequence design. Frame2seq generates sequences in a single pass, achieves 49."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Structure-conditioned masked language models for protein sequence design generalize beyond the native sequence space\nAuthors: Akpinaroglu, D.; Seki, K.; Guo, A.; Zhu, E.; Kelly, M. J.; Kortemme, T.\nScore: 23.3, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571823\nMachine learning has revolutionized computational protein design, enabling significant progress in protein backbone generation and sequence design. Here, we introduce Frame2seq, a structure-conditioned masked language model for protein sequence design. Frame2seq generates sequences in a single pass, achieves 49.",
  "keywords": [
    
  ],
  "articleBody": " Structure-conditioned masked language models for protein sequence design generalize beyond the native sequence space\nAuthors: Akpinaroglu, D.; Seki, K.; Guo, A.; Zhu, E.; Kelly, M. J.; Kortemme, T.\nScore: 23.3, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571823\nMachine learning has revolutionized computational protein design, enabling significant progress in protein backbone generation and sequence design. Here, we introduce Frame2seq, a structure-conditioned masked language model for protein sequence design. Frame2seq generates sequences in a single pass, achieves 49.1% sequence recovery on the CATH 4.2 test dataset, and accurately estimates the error in its own predictions, outperforming the autoregressive ProteinMPNN model with over six times faster inference. To probe the ability of Frame2seq to generate novel designs beyond the native-like sequence space it was trained on, we experimentally test 26 Frame2seq designs for de novo backbones with low identity to the starting sequences. We show that Frame2seq successfully designs soluble (22/26), monomeric, folded, and stable proteins (17/26), including a design with 0% sequence identity to native. The speed and accuracy of Frame2seq will accelerate exploration of novel sequence space across diverse design tasks, including challenging applications such as multi-objective optimization.\nCellular Senescence Impairs Tendon Extracellular Matrix Remodeling in Response to Mechanical Unloading\nAuthors: Stowe, E. J.; Keller, M. R.; Connizzo, B. K.\nScore: 10.6, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572594\nMusculoskeletal injuries, including tendinopathies, present a significant clinical burden for aging populations. While the biological drivers of age-related declines in tendon function are poorly understood, it is well accepted that dysregulation of extracellular matrix (ECM) remodeling plays a role in chronic tendon degeneration. Senescent cells, which have been associated with multiple degenerative pathologies in musculoskeletal tissues, secrete a highly pro-inflammatory senescence-associated secretory phenotype (SASP) that has potential to promote ECM breakdown. However, the role of senescent cells in the dysregulation of tendon ECM homeostasis is largely unknown. To assess this directly, we developed an in vitro model of induced cellular senescence in murine tendon explants. This novel technique enables us to study the isolated interactions of senescent cells and their native ECM without interference from age-related systemic changes. We document multiple biomarkers of cellular senescence in induced tendon explants including cell cycle arrest, apoptosis resistance, and SASP production. We then utilize this in vitro senescence model to compare the ECM remodeling response of young, naturally aged, and senescent tendons to an altered mechanical stimulus. We found that both senescence and aging independently led to alterations in ECM-related gene expression, reductions in protein synthesis, and tissue compositional changes. Furthermore, MMP activity was sustained, thus shifting the remodeling balance of aged and senescent tissues towards degradation over production. Together, this demonstrates that cellular senescence plays a role in the altered mechano-response of aged tendons and likely contributes to poor clinical outcomes in aging populations.\nPiggybacking functionalized DNA nanostructures into live cell nuclei\nAuthors: Roozbahani, G. M.; Colosi, P.; Oravecz, A.; Sorokina, E. M.; Pfeifer, W.; Shokri, S.; Wei, Y.; Didier, P.; DeLuca, M.; Arya, G.; Tora, L.; Lakadamyali, M.; Poirier, M. G.; Castro, C.\nScore: 3.7, Published: 2024-01-01 DOI: 10.1101/2023.12.30.573746\nDNA origami (DO) are promising tools for in vitro or in vivo applications including drug delivery; biosensing, detecting biomolecules; and probing chromatin sub-structures. Targeting these nanodevices to mammalian cell nuclei could provide impactful approaches for probing visualizing and controlling important biological processes in live cells. Here we present an approach to deliver DO strucures into live cell nuclei. We show that labelled DOs do not undergo detectable structural degradation in cell culture media or human cell extracts for 24 hr. To deliver DO platforms into the nuclei of human U2OS cells, we conjugated 30 nm long DO nanorods with an antibody raised against the largest subunit of RNA Polymerase II (Pol II), a key enzyme involved in gene transcription. We find that DOs remain structurally intact in cells for 24hr, including within the nucleus. Using fluorescence microscopy we demonstrate that the electroporated anti-Pol II antibody conjugated DOs are efficiently piggybacked into nuclei and exihibit sub-diffusive motion inside the nucleus. Our results reveal that functionalizing DOs with an antibody raised against a nuclear factor is a highly effective method for the delivery of nanodevices into live cell nuclei.\nHigh-resolution low-cost LCD 3D printing of microfluidics\nAuthors: Shafique, H.; Karamzadeh, V.; Kim, G.; Morocz, Y.; Sohrabi-Kashani, A.; Shen, M. L.; Juncker, D.\nScore: 2.8, Published: 2024-01-02 DOI: 10.1101/2023.12.31.573772\nThe fabrication of microfluidic devices has progressed from cleanroom manufacturing to replica molding in polymers, and more recently to direct manufacturing by subtractive (e.g., laser machining) and additive (e.g., 3D printing) techniques, notably digital light processing (DLP) photopolymerization. However, many methods require technical expertise and while DLP 3D printers remain expensive at a cost ~15-30K USD with ~8M pixels that are 25-40 m in size. Here, we introduce (i) the use of low-cost (~150-600 USD) liquid crystal display (LCD) photopolymerization 3D printing with ~8M-58M pixels that are 18-35 m in size for direct microfluidic device fabrication and (ii) a poly(ethylene glycol) diacrylate-based ink developed for LCD 3D printing (PLInk). We optimized PLInk for high resolution, fast 3D printing and biocompatibility while considering the illumination inhomogeneity and low power density of LCD 3D printers. We made lateral features as small as 75 m, 22-m-thick embedded membranes, and circular channels with a 110 m radius. We 3D printed microfluidic devices previously manufactured by other methods, including an embedded 3D micromixer, a membrane microvalve, and an autonomous capillaric circuit (CC) deployed for interferon-{gamma} detection with excellent performance (limit of detection: 12 pg mL-1, CV: 6.8%), and we demonstrated compatibility with cell culture. Finally, large area manufacturing was illustrated by printing 42 CCs with embedded microchannels in \u003c45 min. LCD 3D printing together with tailored inks pave the way for democratizing access to high-resolution manufacturing of ready-to-use microfluidic devices by anyone, anywhere.\nCell-Free Optimized Production of Protoporphyrin IX\nAuthors: Garcia, D. C.; Davies, J. P.; Rhea, K.; Lee, M. S.; Lux, M. W.\nScore: 2.2, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573540\nBackgroundThe asymmetric and aromatic structures of porphyrins enable semiconducting properties allowing them to absorb light to initiate complex photocatalytic activity. These properties coupled with biological origins have garnered these molecules and their derivatives wide interest as a biotechnological platform. However, porphyrin production in cells is challenging due to limited titers, long production timescales, and difficult purification. ResultsA cell-free metabolic engineering platform was constructed to produce protoporphyrin IX (PPIX) from E. coli crude extracts. Using acoustic liquid-handling, design of experiments for high-throughput buffer optimization and co-culturing techniques for extract production, our cell-free reactions effectively produced 0.109 mg/mL quantities of porphyrins. ConclusionsThe use of cell-free metabolic engineering as a bioproduction platform could improve the production of toxic or inefficient biomolecules such as PPIX. The engineering strategies applied in this study provide a roadmap towards increasing the scale of cell-free metabolic engineering by elucidating batch to batch variability, as well as the need for batch specific optimization of reaction conditions.\nRationally designed modular STAT-activating scaffolds enforce cell-intrinsic transcriptional programs augmenting the anti-tumor potency of CAR T cells\nAuthors: Saxby, C. P.; Ishida, T. K.; Rosser, J. M.; Curtis, B. C.; Baldwin, M. L.; Spurrell, C. H.; Giles, J.; Valdez, A.; Johnson, A. J.; Gustafson, J.; Appelbaum, J. S.; Jensen, M. C.\nScore: 2.2, Published: 2023-12-26 DOI: 10.1101/2023.12.22.573170\nChimeric antigen receptor (CAR)-expressing T cells can mediate anti-tumor responses in a variety of preclinical models and clinical settings, however, strategies to enhance anti-tumor potency is the subject of intense investigation. Signals emanating from gamma-c cytokine receptors modulate the transcriptional state of activated T cells impacting proliferation, survival, differentiation, and effector functioning through the STAT family of transcription factors. Design of ligand-independent cell-intrinsic cytokine STAT activation scaffolds is a conceptually attractive strategy to provide CAR T cells with a surrogate for exogenous cytokine support. Here, we designed a series of ligand-autonomous STAT inducer (LASI) scaffolds comprised of an extracellular identification tag, a homodimerizing transmembrane domain, and a membrane proximal IL7R Box1 domain followed by STAT5 and/or STAT3 docking sequences derived from IL7R and IL21R, respectively. We constructed LASI scaffolds having STAT5 (LASI-5), STAT3 (LASI-3), and combined STAT5 and STAT3 (LASI-5+3) docking domains and then interrogated their impact in primary human CD8+ anti-CD19 (4-1BB:zeta) CAR T cells. While LASI-5 expression had limited effects on CAR T cells, LASI-3 transcriptional programming was found to be indispensable to achieving anti-tumor functional enhancement associated with limited terminal differentiation, heightened T cell proliferation in response to antigen, and dampened expression of exhaustion-associated genes. Moreover, CAR T cells supplemented with LASI-3 or 5+3 displayed superior potency against human leukemia tumors in NSG mice. LASI-5+3 mediated the highest magnitude of CAR T cell engraftment in vivo that evolved into a fatal lymphoproliferative syndrome. However, the same efficacy enhancement was achieved with LASI-3 without the lymphoproliferative complication. Our findings provide a rationale for utilization of constitutively expressed LASI-3 to enhance the anti-tumor potency of CAR T cells, the need to regulate the activity of LASI-5+3, and a generalizable scaffold design for studying additional combinations of STAT family transcription factors.\nThermo-Responsive Polymers Targeting Inflammation in Murine Colitis\nAuthors: Zhang, S.; Jin, A. T.; Tang, W.; Zhang, R. Y.; Jing, L.; Zhou, Y.; Zhang, H.; Lennerz, J. K.; Korzenik, J. R.; Langer, R.; Traverso, G.\nScore: 2.0, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573545\nTargeting the site of inflammation is an ideal approach for treating inflammatory bowel disease (IBD). Inflammation targeting enables maximal drug-on-target effects while minimizing off-target side effects. Negatively charged drug carriers have been shown to facilitate drug delivery to inflamed colon mucosa after local administration. To modulate the negative charges and integrate responsiveness to stimuli, here we describe thermo-responsive, inflammation-targeting (TRIT) hydrogels based on functionalized poly(N-isopropylacrylamide-co-methacrylic acid) (PNIPAM-MAA). We show that both chemical modification types and polymer molecular weights affect the resultant microgels adhesion to the inflamed colon in dextran sulfate sodium (DSS)-induced murine colitis in vivo. Further, we quantified the correlations between microgels adhesion and colitis severity for individual mice, demonstrating that the microgels adhesion correlated directly with weight loss percentage in DSS-treated mice. By exploiting charge-mediated interaction and thermo-responsiveness, TRIT hydrogels represent a promising strategy to target inflamed colon mucosa as a drug delivery platform for colonic IBD treatment. TeaserThis study developed thermo-responsive, inflammation-targeting (TRIT) hydrogels that harness charge-mediated interaction and sol-to-gel transition to target inflamed colon mucosa as a new approach for treating inflammatory bowel disease.\nDual carbon sequestration with photosynthetic living materials\nAuthors: Dranseike, D.; Cui, Y.; Ling, A. S.; Donat, F.; Bernhard, S.; Bernero, M.; Areeckal, A.; Qin, X.-H.; Oakey, J. S.; Dillenburger, B.; Studart, A. R.; Tibbitt, M. W.\nScore: 3.5, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572991\nNatural ecosystems offer efficient pathways for carbon sequestration, serving as a resilient approach to remove CO2 from the atmosphere with minimal environmental impact. However, the control of living systems outside of their native environments is often challenging. Here, we engineered a photosynthetic living material for dual CO2 sequestration by immobilizing photosynthetic microorganisms within a printable polymeric network. The carbon concentrating mechanism of the cyanobacteria enabled accumulation of CO2 within the cell, resulting in biomass production. Additionally, the metabolic production of OH- ions in the surrounding medium created an environment for the formation of insoluble carbonates via microbially-induced calcium carbonate precipitation (MICP). Digital design and fabrication of the living material ensured sufficient access to light and nutrient transport of the encapsulated cyanobacteria, which were essential for long-term viability (more than one year) as well as efficient photosynthesis and carbon sequestration. The photosynthetic living materials sequestered approximately 2.5 mg of CO2 per gram of hydrogel material over 30 days via dual carbon sequestration, with 2.2 {+/-} 0.9 mg stored as insoluble carbonates. Over an extended incubation period of 400 days, the living materials sequestered 26 {+/-} 7 mg of CO2 per gram of hydrogel material in the form of stable minerals. These findings highlight the potential of photosynthetic living materials for scalable carbon sequestration, carbon-neutral infrastructure, and green building materials. The simplicity of maintenance, coupled with its scalability nature, suggests broad applications of photosynthetic living materials as a complementary strategy to mitigate CO2 emissions.\nDe novo multi-mechanism antimicrobial peptide design via multimodal deep learning\nAuthors: Wang, Y.; Gong, H.; Li, X.; Li, L.; Zhao, Y.; Bao, P.; Kong, Q.; Wan, B.; Zhang, Y.; Zhang, J.; Ni, J.; Han, Z.; Nan, X.; Ju, K.; Sun, L.; Chang, H.; Zheng, M.; Yu, Y.; Yang, X.; Zuo, X.; Li, Y.\nScore: 1.8, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573846\nArtificial intelligence (AI)-driven discovery of antimicrobial peptides (AMPs) is yet to fully utilise their three-dimensional (3D) structural characteristics, microbial specie-specific antimicrobial activities and mechanisms. Here, we constructed a QLAPD database comprising the sequence, structures and antimicrobial properties of 12,914 AMPs. QLAPD underlies a multimodal, multitask, multilabel, and conditionally controlled AMP discovery (M3-CAD) pipeline, which is proposed for the de novo design of multi-mechanism AMPs to combat multidrug-resistant organisms (MDROs). This pipeline integrates the generation, regression, and classification modules, using a innovative 3D voxel coloring method to capture the nuanced physicochemical context of amino acids, significantly enhancing structural characterizations. QL-AMP-1, discovered by M3-CAD, which possesses four antimicrobial mechanisms, exhibited low toxicity and significant activity against MDROs. The skin wound infection model demonstrates its considerable antimicrobial effects and negligible toxicity. Altogether, integrating 3D features, specie-specific antimicrobial activities and mechanisms enhanced AI-driven AMP discovery, making the M3-CAD pipeline a viable tool for de novo AMP design.\nIntraoperative Ablation Control Based on Real-time Necrosis Monitoring Feedback: Numerical Evaluation\nAuthors: Murakami, R.; Mori, S.; Zhang, H.\nScore: 1.8, Published: 2024-01-02 DOI: 10.1101/2023.12.31.573805\nAblation therapy is a type of minimally invasive treatment, utilized for various organs including the brain, heart, and kidneys. The accuracy of the ablation process is critically important to avoid both insufficient and excessive ablation, which may result in compromised efficacy or complications. The thermal ablation is formulated by two theoretical models: the heat transfer (HT) and necrosis formation (NF) models. In modern medical practices, feed-forward (FF) and temperature feedback (TFB) controls are primarily used as ablation control methodologies. FF involves pre-therapy procedure planning based on previous experiences and theoretical knowledge without monitoring the intraoperative tissue response, hence, it can't compensate for discrepancies in the assumed HT or NF models. These discrepancies can arise due to individual patient's tissue characteristic differences and specific environmental conditions. Conversely, TFB control is based on the intraoperative temperature profile. It estimates the resulting heat damage based on the monitored temperature distribution and assumed NF model. Therefore, TFB can make necessary adjustments even if there is an error in the assumed HT model. TFB is thus seen as a more robust control method against modeling errors in the HT model. Still, TFB is limited as it assumes a fixed NF model, irrespective of the patient or the ablation technique used. An ideal solution to these limitations would be to actively monitor heat damage to the tissue during the operation and utilize this data to control ablation. This strategy is defined as necrosis feedback (NFB) in this study. Such real-time necrosis monitoring modalities making NFB possible are emerging, however, there is an absence of a generalized study that discusses the integration and quantifies the significance of the real-time necrosis monitor techniques for ablation therapy. Such an investigation is expected to clarify the universal principles of how these techniques would improve ablation therapy. In this study, we examine the potential of NFB in suppressing errors associated with the NF model as NFB is theoretically capable of monitoring and suppressing the errors associated with the NF models in its closed control loop. We simulate and compare the performances of TFB and NFB with artificially generated modeling errors using the finite element method (FEM). The results show that NFB provides more accurate ablation control than TFB when NF-oriented errors are applied, indicating NFB's potential to improve the ablation control accuracy and highlighting the value of the ongoing research to make real-time necrosis monitoring a clinically viable option.\n",
  "wordCount" : "2622",
  "inLanguage": "en",
  "datePublished": "2024-01-03T10:39:43Z",
  "dateModified": "2024-01-03T10:39:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 3, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571823">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571823" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571823">
        <p class="paperTitle">Structure-conditioned masked language models for protein sequence design generalize beyond the native sequence space</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571823" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571823" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Akpinaroglu, D.; Seki, K.; Guo, A.; Zhu, E.; Kelly, M. J.; Kortemme, T.</p>
        <p class="info">Score: 23.3, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571823' target='https://doi.org/10.1101/2023.12.15.571823'> 10.1101/2023.12.15.571823</a></p>
        <p class="abstract">Machine learning has revolutionized computational protein design, enabling significant progress in protein backbone generation and sequence design. Here, we introduce Frame2seq, a structure-conditioned masked language model for protein sequence design. Frame2seq generates sequences in a single pass, achieves 49.1% sequence recovery on the CATH 4.2 test dataset, and accurately estimates the error in its own predictions, outperforming the autoregressive ProteinMPNN model with over six times faster inference. To probe the ability of Frame2seq to generate novel designs beyond the native-like sequence space it was trained on, we experimentally test 26 Frame2seq designs for de novo backbones with low identity to the starting sequences. We show that Frame2seq successfully designs soluble (22/26), monomeric, folded, and stable proteins (17/26), including a design with 0% sequence identity to native. The speed and accuracy of Frame2seq will accelerate exploration of novel sequence space across diverse design tasks, including challenging applications such as multi-objective optimization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572594">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572594" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572594">
        <p class="paperTitle">Cellular Senescence Impairs Tendon Extracellular Matrix Remodeling in Response to Mechanical Unloading</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572594" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572594" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stowe, E. J.; Keller, M. R.; Connizzo, B. K.</p>
        <p class="info">Score: 10.6, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572594' target='https://doi.org/10.1101/2023.12.22.572594'> 10.1101/2023.12.22.572594</a></p>
        <p class="abstract">Musculoskeletal injuries, including tendinopathies, present a significant clinical burden for aging populations. While the biological drivers of age-related declines in tendon function are poorly understood, it is well accepted that dysregulation of extracellular matrix (ECM) remodeling plays a role in chronic tendon degeneration. Senescent cells, which have been associated with multiple degenerative pathologies in musculoskeletal tissues, secrete a highly pro-inflammatory senescence-associated secretory phenotype (SASP) that has potential to promote ECM breakdown. However, the role of senescent cells in the dysregulation of tendon ECM homeostasis is largely unknown. To assess this directly, we developed an in vitro model of induced cellular senescence in murine tendon explants. This novel technique enables us to study the isolated interactions of senescent cells and their native ECM without interference from age-related systemic changes. We document multiple biomarkers of cellular senescence in induced tendon explants including cell cycle arrest, apoptosis resistance, and SASP production. We then utilize this in vitro senescence model to compare the ECM remodeling response of young, naturally aged, and senescent tendons to an altered mechanical stimulus. We found that both senescence and aging independently led to alterations in ECM-related gene expression, reductions in protein synthesis, and tissue compositional changes. Furthermore, MMP activity was sustained, thus shifting the remodeling balance of aged and senescent tissues towards degradation over production. Together, this demonstrates that cellular senescence plays a role in the altered mechano-response of aged tendons and likely contributes to poor clinical outcomes in aging populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573746">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573746" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573746">
        <p class="paperTitle">Piggybacking functionalized DNA nanostructures into live cell nuclei</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573746" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573746" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roozbahani, G. M.; Colosi, P.; Oravecz, A.; Sorokina, E. M.; Pfeifer, W.; Shokri, S.; Wei, Y.; Didier, P.; DeLuca, M.; Arya, G.; Tora, L.; Lakadamyali, M.; Poirier, M. G.; Castro, C.</p>
        <p class="info">Score: 3.7, Published: 2024-01-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573746' target='https://doi.org/10.1101/2023.12.30.573746'> 10.1101/2023.12.30.573746</a></p>
        <p class="abstract">DNA origami (DO) are promising tools for in vitro or in vivo applications including drug delivery; biosensing, detecting biomolecules; and probing chromatin sub-structures. Targeting these nanodevices to mammalian cell nuclei could provide impactful approaches for probing visualizing and controlling important biological processes in live cells. Here we present an approach to deliver DO strucures into live cell nuclei. We show that labelled DOs do not undergo detectable structural degradation in cell culture media or human cell extracts for 24 hr. To deliver DO platforms into the nuclei of human U2OS cells, we conjugated 30 nm long DO nanorods with an antibody raised against the largest subunit of RNA Polymerase II (Pol II), a key enzyme involved in gene transcription. We find that DOs remain structurally intact in cells for 24hr, including within the nucleus. Using fluorescence microscopy we demonstrate that the electroporated anti-Pol II antibody conjugated DOs are efficiently piggybacked into nuclei and exihibit sub-diffusive motion inside the nucleus. Our results reveal that functionalizing DOs with an antibody raised against a nuclear factor is a highly effective method for the delivery of nanodevices into live cell nuclei.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.31.573772">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.31.573772" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.31.573772">
        <p class="paperTitle">High-resolution low-cost LCD 3D printing of microfluidics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.31.573772" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.31.573772" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shafique, H.; Karamzadeh, V.; Kim, G.; Morocz, Y.; Sohrabi-Kashani, A.; Shen, M. L.; Juncker, D.</p>
        <p class="info">Score: 2.8, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.31.573772' target='https://doi.org/10.1101/2023.12.31.573772'> 10.1101/2023.12.31.573772</a></p>
        <p class="abstract">The fabrication of microfluidic devices has progressed from cleanroom manufacturing to replica molding in polymers, and more recently to direct manufacturing by subtractive (e.g., laser machining) and additive (e.g., 3D printing) techniques, notably digital light processing (DLP) photopolymerization. However, many methods require technical expertise and while DLP 3D printers remain expensive at a cost ~15-30K USD with ~8M pixels that are 25-40 m in size. Here, we introduce (i) the use of low-cost (~150-600 USD) liquid crystal display (LCD) photopolymerization 3D printing with ~8M-58M pixels that are 18-35 m in size for direct microfluidic device fabrication and (ii) a poly(ethylene glycol) diacrylate-based ink developed for LCD 3D printing (PLInk). We optimized PLInk for high resolution, fast 3D printing and biocompatibility while considering the illumination inhomogeneity and low power density of LCD 3D printers. We made lateral features as small as 75 m, 22-m-thick embedded membranes, and circular channels with a 110 m radius. We 3D printed microfluidic devices previously manufactured by other methods, including an embedded 3D micromixer, a membrane microvalve, and an autonomous capillaric circuit (CC) deployed for interferon-{gamma} detection with excellent performance (limit of detection: 12 pg mL-1, CV: 6.8%), and we demonstrated compatibility with cell culture. Finally, large area manufacturing was illustrated by printing 42 CCs with embedded microchannels in &lt;45 min. LCD 3D printing together with tailored inks pave the way for democratizing access to high-resolution manufacturing of ready-to-use microfluidic devices by anyone, anywhere.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573540">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573540" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573540">
        <p class="paperTitle">Cell-Free Optimized Production of Protoporphyrin IX</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573540" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573540" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garcia, D. C.; Davies, J. P.; Rhea, K.; Lee, M. S.; Lux, M. W.</p>
        <p class="info">Score: 2.2, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573540' target='https://doi.org/10.1101/2023.12.28.573540'> 10.1101/2023.12.28.573540</a></p>
        <p class="abstract">BackgroundThe asymmetric and aromatic structures of porphyrins enable semiconducting properties allowing them to absorb light to initiate complex photocatalytic activity. These properties coupled with biological origins have garnered these molecules and their derivatives wide interest as a biotechnological platform. However, porphyrin production in cells is challenging due to limited titers, long production timescales, and difficult purification.

ResultsA cell-free metabolic engineering platform was constructed to produce protoporphyrin IX (PPIX) from E. coli crude extracts. Using acoustic liquid-handling, design of experiments for high-throughput buffer optimization and co-culturing techniques for extract production, our cell-free reactions effectively produced 0.109 mg/mL quantities of porphyrins.

ConclusionsThe use of cell-free metabolic engineering as a bioproduction platform could improve the production of toxic or inefficient biomolecules such as PPIX. The engineering strategies applied in this study provide a roadmap towards increasing the scale of cell-free metabolic engineering by elucidating batch to batch variability, as well as the need for batch specific optimization of reaction conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573170">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573170" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573170">
        <p class="paperTitle">Rationally designed modular STAT-activating scaffolds enforce cell-intrinsic transcriptional programs augmenting the anti-tumor potency of CAR T cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573170" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573170" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saxby, C. P.; Ishida, T. K.; Rosser, J. M.; Curtis, B. C.; Baldwin, M. L.; Spurrell, C. H.; Giles, J.; Valdez, A.; Johnson, A. J.; Gustafson, J.; Appelbaum, J. S.; Jensen, M. C.</p>
        <p class="info">Score: 2.2, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573170' target='https://doi.org/10.1101/2023.12.22.573170'> 10.1101/2023.12.22.573170</a></p>
        <p class="abstract">Chimeric antigen receptor (CAR)-expressing T cells can mediate anti-tumor responses in a variety of preclinical models and clinical settings, however, strategies to enhance anti-tumor potency is the subject of intense investigation. Signals emanating from gamma-c cytokine receptors modulate the transcriptional state of activated T cells impacting proliferation, survival, differentiation, and effector functioning through the STAT family of transcription factors. Design of ligand-independent cell-intrinsic cytokine STAT activation scaffolds is a conceptually attractive strategy to provide CAR T cells with a surrogate for exogenous cytokine support. Here, we designed a series of ligand-autonomous STAT inducer (LASI) scaffolds comprised of an extracellular identification tag, a homodimerizing transmembrane domain, and a membrane proximal IL7R Box1 domain followed by STAT5 and/or STAT3 docking sequences derived from IL7R and IL21R, respectively. We constructed LASI scaffolds having STAT5 (LASI-5), STAT3 (LASI-3), and combined STAT5 and STAT3 (LASI-5&#43;3) docking domains and then interrogated their impact in primary human CD8&#43; anti-CD19 (4-1BB:zeta) CAR T cells. While LASI-5 expression had limited effects on CAR T cells, LASI-3 transcriptional programming was found to be indispensable to achieving anti-tumor functional enhancement associated with limited terminal differentiation, heightened T cell proliferation in response to antigen, and dampened expression of exhaustion-associated genes. Moreover, CAR T cells supplemented with LASI-3 or 5&#43;3 displayed superior potency against human leukemia tumors in NSG mice. LASI-5&#43;3 mediated the highest magnitude of CAR T cell engraftment in vivo that evolved into a fatal lymphoproliferative syndrome. However, the same efficacy enhancement was achieved with LASI-3 without the lymphoproliferative complication. Our findings provide a rationale for utilization of constitutively expressed LASI-3 to enhance the anti-tumor potency of CAR T cells, the need to regulate the activity of LASI-5&#43;3, and a generalizable scaffold design for studying additional combinations of STAT family transcription factors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573545">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573545" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573545">
        <p class="paperTitle">Thermo-Responsive Polymers Targeting Inflammation in Murine Colitis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573545" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573545" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, S.; Jin, A. T.; Tang, W.; Zhang, R. Y.; Jing, L.; Zhou, Y.; Zhang, H.; Lennerz, J. K.; Korzenik, J. R.; Langer, R.; Traverso, G.</p>
        <p class="info">Score: 2.0, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573545' target='https://doi.org/10.1101/2023.12.28.573545'> 10.1101/2023.12.28.573545</a></p>
        <p class="abstract">Targeting the site of inflammation is an ideal approach for treating inflammatory bowel disease (IBD). Inflammation targeting enables maximal drug-on-target effects while minimizing off-target side effects. Negatively charged drug carriers have been shown to facilitate drug delivery to inflamed colon mucosa after local administration. To modulate the negative charges and integrate responsiveness to stimuli, here we describe thermo-responsive, inflammation-targeting (TRIT) hydrogels based on functionalized poly(N-isopropylacrylamide-co-methacrylic acid) (PNIPAM-MAA). We show that both chemical modification types and polymer molecular weights affect the resultant microgels adhesion to the inflamed colon in dextran sulfate sodium (DSS)-induced murine colitis in vivo. Further, we quantified the correlations between microgels adhesion and colitis severity for individual mice, demonstrating that the microgels adhesion correlated directly with weight loss percentage in DSS-treated mice. By exploiting charge-mediated interaction and thermo-responsiveness, TRIT hydrogels represent a promising strategy to target inflamed colon mucosa as a drug delivery platform for colonic IBD treatment.

TeaserThis study developed thermo-responsive, inflammation-targeting (TRIT) hydrogels that harness charge-mediated interaction and sol-to-gel transition to target inflamed colon mucosa as a new approach for treating inflammatory bowel disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572991">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572991" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572991">
        <p class="paperTitle">Dual carbon sequestration with photosynthetic living materials</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572991" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572991" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dranseike, D.; Cui, Y.; Ling, A. S.; Donat, F.; Bernhard, S.; Bernero, M.; Areeckal, A.; Qin, X.-H.; Oakey, J. S.; Dillenburger, B.; Studart, A. R.; Tibbitt, M. W.</p>
        <p class="info">Score: 3.5, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572991' target='https://doi.org/10.1101/2023.12.22.572991'> 10.1101/2023.12.22.572991</a></p>
        <p class="abstract">Natural ecosystems offer efficient pathways for carbon sequestration, serving as a resilient approach to remove CO2 from the atmosphere with minimal environmental impact. However, the control of living systems outside of their native environments is often challenging. Here, we engineered a photosynthetic living material for dual CO2 sequestration by immobilizing photosynthetic microorganisms within a printable polymeric network. The carbon concentrating mechanism of the cyanobacteria enabled accumulation of CO2 within the cell, resulting in biomass production. Additionally, the metabolic production of OH- ions in the surrounding medium created an environment for the formation of insoluble carbonates via microbially-induced calcium carbonate precipitation (MICP). Digital design and fabrication of the living material ensured sufficient access to light and nutrient transport of the encapsulated cyanobacteria, which were essential for long-term viability (more than one year) as well as efficient photosynthesis and carbon sequestration. The photosynthetic living materials sequestered approximately 2.5 mg of CO2 per gram of hydrogel material over 30 days via dual carbon sequestration, with 2.2 {&#43;/-} 0.9 mg stored as insoluble carbonates. Over an extended incubation period of 400 days, the living materials sequestered 26 {&#43;/-} 7 mg of CO2 per gram of hydrogel material in the form of stable minerals. These findings highlight the potential of photosynthetic living materials for scalable carbon sequestration, carbon-neutral infrastructure, and green building materials. The simplicity of maintenance, coupled with its scalability nature, suggests broad applications of photosynthetic living materials as a complementary strategy to mitigate CO2 emissions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573846">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573846" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573846">
        <p class="paperTitle">De novo multi-mechanism antimicrobial peptide design via multimodal deep learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573846" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573846" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Y.; Gong, H.; Li, X.; Li, L.; Zhao, Y.; Bao, P.; Kong, Q.; Wan, B.; Zhang, Y.; Zhang, J.; Ni, J.; Han, Z.; Nan, X.; Ju, K.; Sun, L.; Chang, H.; Zheng, M.; Yu, Y.; Yang, X.; Zuo, X.; Li, Y.</p>
        <p class="info">Score: 1.8, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573846' target='https://doi.org/10.1101/2024.01.02.573846'> 10.1101/2024.01.02.573846</a></p>
        <p class="abstract">Artificial intelligence (AI)-driven discovery of antimicrobial peptides (AMPs) is yet to fully utilise their three-dimensional (3D) structural characteristics, microbial specie-specific antimicrobial activities and mechanisms. Here, we constructed a QLAPD database comprising the sequence, structures and antimicrobial properties of 12,914 AMPs. QLAPD underlies a multimodal, multitask, multilabel, and conditionally controlled AMP discovery (M3-CAD) pipeline, which is proposed for the de novo design of multi-mechanism AMPs to combat multidrug-resistant organisms (MDROs). This pipeline integrates the generation, regression, and classification modules, using a innovative 3D voxel coloring method to capture the nuanced physicochemical context of amino acids, significantly enhancing structural characterizations. QL-AMP-1, discovered by M3-CAD, which possesses four antimicrobial mechanisms, exhibited low toxicity and significant activity against MDROs. The skin wound infection model demonstrates its considerable antimicrobial effects and negligible toxicity. Altogether, integrating 3D features, specie-specific antimicrobial activities and mechanisms enhanced AI-driven AMP discovery, making the M3-CAD pipeline a viable tool for de novo AMP design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.31.573805">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.31.573805" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.31.573805">
        <p class="paperTitle">Intraoperative Ablation Control Based on Real-time Necrosis Monitoring Feedback: Numerical Evaluation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.31.573805" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.31.573805" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Murakami, R.; Mori, S.; Zhang, H.</p>
        <p class="info">Score: 1.8, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.31.573805' target='https://doi.org/10.1101/2023.12.31.573805'> 10.1101/2023.12.31.573805</a></p>
        <p class="abstract">Ablation therapy is a type of minimally invasive treatment, utilized for various organs including the brain, heart, and kidneys. The accuracy of the ablation process is critically important to avoid both insufficient and excessive ablation, which may result in compromised efficacy or complications. The thermal ablation is formulated by two theoretical models: the heat transfer (HT) and necrosis formation (NF) models. In modern medical practices, feed-forward (FF) and temperature feedback (TFB) controls are primarily used as ablation control methodologies. FF involves pre-therapy procedure planning based on previous experiences and theoretical knowledge without monitoring the intraoperative tissue response, hence, it can&#39;t compensate for discrepancies in the assumed HT or NF models. These discrepancies can arise due to individual patient&#39;s tissue characteristic differences and specific environmental conditions. Conversely, TFB control is based on the intraoperative temperature profile. It estimates the resulting heat damage based on the monitored temperature distribution and assumed NF model. Therefore, TFB can make necessary adjustments even if there is an error in the assumed HT model. TFB is thus seen as a more robust control method against modeling errors in the HT model. Still, TFB is limited as it assumes a fixed NF model, irrespective of the patient or the ablation technique used. An ideal solution to these limitations would be to actively monitor heat damage to the tissue during the operation and utilize this data to control ablation. This strategy is defined as necrosis feedback (NFB) in this study. Such real-time necrosis monitoring modalities making NFB possible are emerging, however, there is an absence of a generalized study that discusses the integration and quantifies the significance of the real-time necrosis monitor techniques for ablation therapy. Such an investigation is expected to clarify the universal principles of how these techniques would improve ablation therapy. In this study, we examine the potential of NFB in suppressing errors associated with the NF model as NFB is theoretically capable of monitoring and suppressing the errors associated with the NF models in its closed control loop. We simulate and compare the performances of TFB and NFB with artificially generated modeling errors using the finite element method (FEM). The results show that NFB provides more accurate ablation control than TFB when NF-oriented errors are applied, indicating NFB&#39;s potential to improve the ablation control accuracy and highlighting the value of the ongoing research to make real-time necrosis monitoring a clinically viable option.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
