<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="SonoPrint: Acoustically Assisted Volumetric 3D Printing for Composites
Authors: Agrawal, P.; Zhuang, S.; Dreher, S.; Mitter, S.; Ahmed, D.
Score: 9.6, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552292
Advancements in additive manufacturing in composites have transformed various fields in aerospace, medical devices, tissue engineering, and electronics, enabling fine-tuning material properties by reinforcing internal particles and adjusting their type, orientation, and volume fraction. This capability opens new possibilities for tailoring materials to specific applications and optimizing the performance of 3D-printed objects.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="SonoPrint: Acoustically Assisted Volumetric 3D Printing for Composites
Authors: Agrawal, P.; Zhuang, S.; Dreher, S.; Mitter, S.; Ahmed, D.
Score: 9.6, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552292
Advancements in additive manufacturing in composites have transformed various fields in aerospace, medical devices, tissue engineering, and electronics, enabling fine-tuning material properties by reinforcing internal particles and adjusting their type, orientation, and volume fraction. This capability opens new possibilities for tailoring materials to specific applications and optimizing the performance of 3D-printed objects." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-13T10:36:01+00:00" />
<meta property="article:modified_time" content="2023-08-13T10:36:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="SonoPrint: Acoustically Assisted Volumetric 3D Printing for Composites
Authors: Agrawal, P.; Zhuang, S.; Dreher, S.; Mitter, S.; Ahmed, D.
Score: 9.6, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552292
Advancements in additive manufacturing in composites have transformed various fields in aerospace, medical devices, tissue engineering, and electronics, enabling fine-tuning material properties by reinforcing internal particles and adjusting their type, orientation, and volume fraction. This capability opens new possibilities for tailoring materials to specific applications and optimizing the performance of 3D-printed objects."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "SonoPrint: Acoustically Assisted Volumetric 3D Printing for Composites\nAuthors: Agrawal, P.; Zhuang, S.; Dreher, S.; Mitter, S.; Ahmed, D.\nScore: 9.6, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552292\nAdvancements in additive manufacturing in composites have transformed various fields in aerospace, medical devices, tissue engineering, and electronics, enabling fine-tuning material properties by reinforcing internal particles and adjusting their type, orientation, and volume fraction. This capability opens new possibilities for tailoring materials to specific applications and optimizing the performance of 3D-printed objects.",
  "keywords": [
    
  ],
  "articleBody": " SonoPrint: Acoustically Assisted Volumetric 3D Printing for Composites\nAuthors: Agrawal, P.; Zhuang, S.; Dreher, S.; Mitter, S.; Ahmed, D.\nScore: 9.6, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552292\nAdvancements in additive manufacturing in composites have transformed various fields in aerospace, medical devices, tissue engineering, and electronics, enabling fine-tuning material properties by reinforcing internal particles and adjusting their type, orientation, and volume fraction. This capability opens new possibilities for tailoring materials to specific applications and optimizing the performance of 3D-printed objects. Existing reinforcement strategies are restricted to pattern types, alignment areas, and particle characteristics. Alternatively, acoustics provide versatility by controlling particles independent of their size, geometry, and charge and can create intricate pattern formations. Despite the potential of acoustics in most 3D printing, limitation arises from the scattering of the acoustic field between the polymerized hard layers and the unpolymerized resin, leading to undesirable patterning formation. However, this challenge can be addressed by adopting a novel approach that involves simultaneous reinforcement and printing the entire structure. Here, we present SonoPrint, an acoustically-assisted volumetric 3D printer that produces mechanically tunable composite geometries by patterning reinforcement microparticles within the fabricated structure. SonoPrint creates a standing wave field that produces a targeted particle motif in the photosensitive resin while simultaneously printing the object in just a few minutes. We have also demonstrated various patterning configurations such as lines, radial lines, circles, rhombuses, quadrilaterals, and hexagons using microscopic particles such as glass, metal, and polystyrene particles. Furthermore, we fabricated diverse composites using different resins, achieving 87 microns feature size. We have shown that the printed structure with patterned microparticles increased their tensile and compression strength by [~]38% and [~]75%, respectively.\nnapari-threedee: a toolkit for human-in-the-loop 3D image analysis in napari\nAuthors: Yamauchi, K. A.; Burt, A.\nScore: 10.6, Published: 2023-07-30 DOI: 10.1101/2023.07.28.550950\nWith recent advances in three-dimensional bioimaging acquisition and analysis methods, imaging data are an important tool for unveiling the complex architecture of biological organisms. However, extracting insights from these complex datasets requires interactive exploration as well as annotation for training supervised analytical methods. Here we present napari-threedee, a library of composable components and napari plugins for interactively exploring and annotating image data in 3D. We demonstrate how napari-threedee can be used for human-in-the-loop 3D image analysis in diverse image data including fluorescence microscopy and cryoelectron tomography.\nMetabolic impact of heterologous protein production in Pseudomonas putida: Insights into carbon and energy flux control\nAuthors: Vogeleer, P.; Millard, P.; Ortega Arbulu, A.-S.; Pfluger-Grau, K.; Kremling, A.; Letisse, F.\nScore: 6.5, Published: 2023-07-27 DOI: 10.1101/2023.07.27.550675\nFor engineered microorganisms, the expression of heterologous proteins that are often useless to host cells represents a burden on resources, which have to be shared with normal cellular processes. Within a certain metabolic leeway, this competitive process has no impact on growth. However, once this leeway, or free capacity, is fully utilized, the extra load becomes a metabolic burden that inhibits cellular processes and triggers a broad cellular response, reducing cell growth and often hindering the production of heterologous proteins. In this study, we sought to characterize the metabolic rearrangements occurring in the central metabolism of Pseudomonas putida at different levels of metabolic load. To this end, we constructed a P. putida KT2440 strain that expressed two genes encoding fluorescent proteins, one in the genome under constitutive expression to monitor the free capacity, and the other on an inducible plasmid to probe heterologous protein production. We found that metabolic fluxes are profoundly reshuffled, notably at the level of periplasmic pathways, as soon as the metabolic load exceeds the free capacity. Heterologous protein production leads to the decoupling of anabolism and catabolism, resulting in large excess energy production relative to the requirements of protein biosynthesis. Finally, heterologous protein production was found to exert a stronger control on carbon fluxes than on energy fluxes, indicating that the flexible nature of P. putidas central metabolic network is solicited to sustain energy production. HighlightsHeterologous protein production in P. putida reshuffles the periplasmic metabolism. Increased protein production progressively decouples catabolism from anabolism. Protein production exerts a stronger control on energy than on carbon fluxes. Glucose is directed towards ATP production to meet the elevated energy demands.\nSingle-cell precision nanotechnology in vivo\nAuthors: MOLBAY, M.; Kick, B.; Zhao, S.; Todorov, M. I.; Ohn, T.-L.; Roth, S.; Simats, A.; Singh, V.; Khalin, I.; Pan, C.; Singh Bhatia, H.; Hellal, F.; Zeidler, R.; Liesz, A.; Plesnila, N.; Dietz, H.; Erturk, A.\nScore: 144.9, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550304\nTargeting nanoparticle therapeutics with cellular accuracy in whole organisms could open breakthrough opportunities in precision medicine. However, evaluating and fine-tuning the biodistribution of such systems in the whole organism at the cellular level remains a major obstacle. Here, we constructed targetable DNA origami, and analyzed biodistribution in transparent mice, in addition to studying tolerability, clearance kinetics, and immune response parameters. Untargeted DNA origami primarily accumulated in the spleen and the liver, while an immune cell-targeting variant successfully attached to immune cells throughout the body. A cancer cell-targeting mimetic co-localized on solid-tumor metastasis in the liver and the lung. These findings indicate that DNA origami can be directed in vivo, providing an important proof-of-concept and highlights the potential of high-resolution tissue-clearing imaging technologies in their development. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=121 SRC=\"FIGDIR/small/550304v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (44K): org.highwire.dtl.DTLVardef@ce216borg.highwire.dtl.DTLVardef@f40bb4org.highwire.dtl.DTLVardef@2c6a66org.highwire.dtl.DTLVardef@1fd7fde_HPS_FORMAT_FIGEXP M_FIG C_FIG HighlightsO_LIThis study demonstrates the potential of DNA origami-based drug delivery systems as versatile tool for for targeted delivery, which could be used to treat a range of diseases with applications. C_LIO_LIThe immune compatibility, half-life, targeting efficiency, and the biodistribution evaluation of DNA origami indicate its potential for systemic drug delivery. C_LIO_LIOur approach enables the assessment of biodistribution of nanoparticles in the intact body with a sensitivity to the single-cell level, highlighting the high-resolution tissue clearing technoloies in revealing DNA origamis feasibility for drug targeting. C_LI\nStiffness anisotropy coordinates supracellular contractility driving long-range myotube-ECM alignment\nAuthors: Skillin, N. P.; Kirkpatrick, B. E.; Herbert, K. M.; Nelson, B. R.; Hach, G. K.; Gunay, K. A.; Khan, R. M.; DelRio, F. W.; White, T. J.; Anseth, K. S.\nScore: 4.1, Published: 2023-08-11 DOI: 10.1101/2023.08.08.552197\nIn skeletal muscle tissue, injury-related changes in stiffness activate muscle stem cells through mechanosensitive signaling pathways. Functional muscle tissue regeneration also requires the effective coordination of myoblast proliferation, migration, polarization, differentiation, and fusion across multiple length scales. Here, we demonstrate that substrate stiffness anisotropy coordinates contractility-driven collective cellular dynamics resulting in C2C12 myotube alignment over millimeter-scale distances. When cultured on mechanically anisotropic liquid crystalline polymer networks (LCNs) lacking topographic features that could confer contact guidance, C2C12 myoblasts collectively polarize in the stiffest direction of the substrate. Cellular coordination is amplified through reciprocal cell-ECM dynamics that emerge during fusion, driving global myotube-ECM ordering. Conversely, myotube alignment was restricted to small local domains with no directional preference on mechanically isotropic LCNs of same chemical formulation. These findings reveal a role for stiffness anisotropy in coordinating emergent collective cellular dynamics, with implications for understanding skeletal muscle tissue development and regeneration.\nHeart-on-a-chip model of immune-induced cardiac dysfunction reveals the involvement of free mitochondrial DNA and therapeutic effects of endothelial exosomes\nAuthors: Lu, X. Z.; Rafatian, N.; Zhao, Y.; Wagner, K. T.; Beroncal, E. L.; Li, B.; Lee, C.; Chen, J.; Churcher, E.; Vosoughi, D.; Wang, Y.; Baker, A.; Trahtemberg, U.; Li, B.; Pierro, A.; Andreazza, A.; dos Santos, C. C.; Radisic, M.\nScore: 3.9, Published: 2023-08-09 DOI: 10.1101/2023.08.09.552495\nCardiovascular disease continues to take more human lives than all cancer combined, prompting the need for improved research models and treatment options. Despite a significant progress in development of mature heart-on-a-chip models of fibrosis and cardiomyopathies starting from induced pluripotent stem cells (iPSCs), human cell-based models of myocardial inflammation are lacking. Here, we bioengineered a vascularized heart-on-a-chip system with circulating immune cells to model SARS-CoV-2-induced acute myocarditis. Briefly, we observed hallmarks of COVID-19-induced myocardial inflammation in the heart-on-a-chip model, as the presence of immune cells augmented the expression levels of proinflammatory cytokines, triggered progressive impairment of contractile function and altered intracellular calcium transient activities. An elevation of circulating cell-free mitochondrial DNA (ccf-mtDNA) was measured first in the in vitro heart-on-a-chip model and then validated in COVID-19 patients with low left ventricular ejection fraction (LVEF), demonstrating that mitochondrial damage is an important pathophysiological hallmark of inflammation induced cardiac dysfunction. Leveraging this platform in the context of SARS-CoV-2 induced myocardial inflammation, we established that administration of human umbilical vein-derived EVs effectively rescued the contractile deficit, normalized intracellular calcium handling, elevated the contraction force and reduced the ccf- mtDNA and chemokine release via TLR-NF-kB signaling axis.\nNuclear Factor Kappa B Over-Activation in the Intervertebral Disc Leads to Macrophage Recruitment and Severe Disc Degeneration\nAuthors: Burt, K. G.; Kim, M. K. M.; Viola, D. C.; Abraham, A. C.; Chahine, N. O.\nScore: 2.5, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552274\nObjectiveLow back pain (LBP) is the leading cause of global disability and is thought to be driven primarily by intervertebral disc (IVD) degeneration (DD). Persistent upregulation of catabolic enzymes and inflammatory mediators have been associated with severe cases of DD. Nuclear factor kappa B (NF-{kappa}B) is a master transcription regulator of immune responses and is over expressed during inflammatory-driven musculoskeletal diseases, including DD. However, its role in triggering DD is unknown. Therefore, this study investigated the effect of NF-{kappa}B pathway over-activation on IVD integrity and DD pathology. MethodsUsing skeletally mature mouse model, we genetically targeted IVD cells for canonical NF-{kappa}B pathway activation via expression of a constitutively active form of inhibitor of {kappa}B kinase B (IKK{beta}), and assessed changes in IVD cellularity, structural integrity including histology, disc height, and extracellular matrix (ECM) biochemistry, biomechanics, expression of inflammatory, catabolic, and neurotropic mediators, and changes in macrophage subsets, longitudinally up to 6-months post activation. ResultsProlonged NF-{kappa}B activation led to severe structural degeneration, with a loss of glycosaminoglycan (GAG) content and complete loss of nucleus pulposus (NP) cellularity. Structural and compositional changes decreased IVD height and compressive mechanical properties with prolonged NF-{kappa}B activation. These alterations were accompanied by increases in gene expression of inflammatory molecules (Il1b, Il6, Nos2), chemokines (Mcp1, Mif), catabolic enzymes (Mmp3, Mmp9, Adamts4), and neurotrophic factors (Bdnf, Ngf) within IVD tissue. Increased recruitment of activated F4/80+ macrophages exhibited a greater abundance of pro-inflammatory (CD38+) over inflammatory-resolving (CD206+) macrophage subsets in the IVD, with temporal changes in the relative abundance of macrophage subsets over time, providing evidence for temporal regulation of macrophage polarization in DD in vivo, where macrophages participate in resolving the inflammatory cascade but promote fibrotic transformation of the IVD matrix. We further show that NF-{kappa}B driven secretory factors from IVD cells increase macrophage migration and inflammatory activation, and that the secretome of inflammatory-resolving macrophages mitigates effects of NF-{kappa}B overactivation. ConclusionOverall the observed results suggest prolonged NF-{kappa}B activation can induce severe DD, acting through increases in inflammatory cytokines, chemotactic proteins, catabolic enzymes, and the recruitment and inflammatory activation of a macrophage cell populations, that can be mitigated with inflammatory-resolving macrophage secretome.\nSpatial and temporal control of 3D hydrogel viscoelasticity through phototuning\nAuthors: Crandell, P.; Stowers, R.\nScore: 2.2, Published: 2023-08-08 DOI: 10.1101/2023.08.07.551544\nThe mechanical properties of the extracellular environment can regulate a variety of cellular functions, such as spreading, migration, proliferation, and even differentiation and phenotypic determination. Much effort has been directed at understanding the effects of the extracellular matrix (ECM) elastic modulus and more recently, stress relaxation, on cellular processes. In physiological contexts like development, wound healing, and fibrotic disease progression, ECM mechanical properties change substantially over time or space. Dynamically tunable hydrogel platforms have been developed to spatiotemporally modulate a gels elastic modulus. However, dynamically altering the stress relaxation rate of a hydrogel remains a challenge. Here, we present a strategy to tune hydrogel stress relaxation rates in time or space using a light-triggered tethering of poly(ethylene glycol) (PEG) to alginate. We show that stress relaxation rate can be tuned without altering the elastic modulus of the hydrogel. We found that cells are capable of sensing and responding to dynamic stress relaxation rate changes, both morphologically and through differences in proliferation rates. We also exploited the light-based technique to generate spatial patterns of stress relaxation rates in 3D hydrogels. We anticipate that user-directed control of 3D hydrogel stress relaxation rate will be a powerful tool that enables studies that mimic dynamic ECM contexts, or as a means to guide cell fate in space and time for tissue engineering applications.\nSimple and rewireable biomolecular building blocks for DNA machine-learning algorithms\nAuthors: Lee, R. C.; Corsano, A.; Tseng, C. Y.; Chou, L. Y. T.\nScore: 10.5, Published: 2023-07-23 DOI: 10.1101/2023.07.20.549967\nDeep learning algorithms, such as neural networks, enable the processing of complex datasets with many related variables, and have applications in disease diagnosis, cell profiling, and drug discovery. Beyond its use in electronic computers, neural networks have been implemented using programmable biomolecules such as DNA. This confers unique advantages such as greater portability, ability to operate without electricity, and direct analysis of patterns of biomolecules in solution. Analogous to past bottlenecks in electronic computers, the computing power of DNA-based neural networks is limited by the ability to add more computing units, i.e. neurons. This limitation exists because current architectures require many nucleic acids to model a single neuron. Each addition of a neuron to the network compounds existing problems such as long assembly times, high background signal, and cross-talk between components. Here we test three strategies to solve this limitation and improve the scalability of DNA-based neural networks: (i) enzymatic synthesis to generate high-purity neurons, (ii) spatial patterning of neuron clusters based on their network position, and (iii) encoding neuron connectivity on a universal single-stranded DNA backbone. We show that neurons implemented via these strategies activate quickly, with high signal-to-background ratio, and respond to varying input concentrations and weights. Using this neuron design, we implemented basic neural network motifs such as cascading, fan-in, and fan-out circuits. Since this design is modular, easy to synthesize, and compatible with multiple neural network architectures, we envision it will help scale DNA-based neural networks in a variety of settings. This will enable portable computing power for applications such as portable diagnostics, compact data storage, and autonomous decision making for lab-on-a-chips.\nMicrobiome single cell atlases generated with a commercial instrument\nAuthors: Li, X.; Xu, L.; Demaree, B.; Noecker, C.; Bisanz, J. E.; Weisgerber, D. W.; Modavi, C.; Turnbaugh, P. J.; Abate, A. R.\nScore: 2.0, Published: 2023-08-10 DOI: 10.1101/2023.08.08.551713\nSingle cell sequencing is useful for resolving complex systems into their composite cell types and computationally mining them for unique features that are masked in pooled sequencing. However, while commercial instruments have made single cell analysis widespread for mammalian cells, analogous tools for microbes are limited. Here, we present EASi-seq (Easily Accessible Single microbe sequencing). By adapting the single cell workflow of the commercial Mission Bio Tapestri instrument, this method allows for efficient sequencing of individual microbes' genomes. EASi-seq allows thousands of microbes to be sequenced per run and, as we show, can generate detailed atlases of human and environmental microbiomes. The ability to capture large shotgun genome datasets from thousands of single microbes provides new opportunities in discovering and analyzing species subpopulations. To facilitate this, we develop a companion bioinformatic pipeline that clusters microbes by similarity, improving whole genome assembly, strain identification, taxonomic classification, and gene annotation. In addition, we demonstrate integration of metagenomic contigs with the EASi-seq datasets to reduce capture bias and increase coverage. Overall, EASi-seq enables high quality single cell genomic data for microbiome samples using an accessible workflow that can be run on a commercially available platform.\n",
  "wordCount" : "2571",
  "inLanguage": "en",
  "datePublished": "2023-08-13T10:36:01Z",
  "dateModified": "2023-08-13T10:36:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on August 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552292">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552292" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552292">
        <p class="paperTitle">SonoPrint: Acoustically Assisted Volumetric 3D Printing for Composites</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552292" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552292" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Agrawal, P.; Zhuang, S.; Dreher, S.; Mitter, S.; Ahmed, D.</p>
        <p class="info">Score: 9.6, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552292' target='https://doi.org/10.1101/2023.08.07.552292'> 10.1101/2023.08.07.552292</a></p>
        <p class="abstract">Advancements in additive manufacturing in composites have transformed various fields in aerospace, medical devices, tissue engineering, and electronics, enabling fine-tuning material properties by reinforcing internal particles and adjusting their type, orientation, and volume fraction. This capability opens new possibilities for tailoring materials to specific applications and optimizing the performance of 3D-printed objects. Existing reinforcement strategies are restricted to pattern types, alignment areas, and particle characteristics. Alternatively, acoustics provide versatility by controlling particles independent of their size, geometry, and charge and can create intricate pattern formations. Despite the potential of acoustics in most 3D printing, limitation arises from the scattering of the acoustic field between the polymerized hard layers and the unpolymerized resin, leading to undesirable patterning formation. However, this challenge can be addressed by adopting a novel approach that involves simultaneous reinforcement and printing the entire structure. Here, we present SonoPrint, an acoustically-assisted volumetric 3D printer that produces mechanically tunable composite geometries by patterning reinforcement microparticles within the fabricated structure. SonoPrint creates a standing wave field that produces a targeted particle motif in the photosensitive resin while simultaneously printing the object in just a few minutes. We have also demonstrated various patterning configurations such as lines, radial lines, circles, rhombuses, quadrilaterals, and hexagons using microscopic particles such as glass, metal, and polystyrene particles. Furthermore, we fabricated diverse composites using different resins, achieving 87 microns feature size. We have shown that the printed structure with patterned microparticles increased their tensile and compression strength by [~]38% and [~]75%, respectively.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550950">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550950" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550950">
        <p class="paperTitle">napari-threedee: a toolkit for human-in-the-loop 3D image analysis in napari</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550950" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550950" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yamauchi, K. A.; Burt, A.</p>
        <p class="info">Score: 10.6, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550950' target='https://doi.org/10.1101/2023.07.28.550950'> 10.1101/2023.07.28.550950</a></p>
        <p class="abstract">With recent advances in three-dimensional bioimaging acquisition and analysis methods, imaging data are an important tool for unveiling the complex architecture of biological organisms. However, extracting insights from these complex datasets requires interactive exploration as well as annotation for training supervised analytical methods. Here we present napari-threedee, a library of composable components and napari plugins for interactively exploring and annotating image data in 3D. We demonstrate how napari-threedee can be used for human-in-the-loop 3D image analysis in diverse image data including fluorescence microscopy and cryoelectron tomography.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550675">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550675" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550675">
        <p class="paperTitle">Metabolic impact of heterologous protein production in Pseudomonas putida: Insights into carbon and energy flux control</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550675" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550675" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vogeleer, P.; Millard, P.; Ortega Arbulu, A.-S.; Pfluger-Grau, K.; Kremling, A.; Letisse, F.</p>
        <p class="info">Score: 6.5, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550675' target='https://doi.org/10.1101/2023.07.27.550675'> 10.1101/2023.07.27.550675</a></p>
        <p class="abstract">For engineered microorganisms, the expression of heterologous proteins that are often useless to host cells represents a burden on resources, which have to be shared with normal cellular processes. Within a certain metabolic leeway, this competitive process has no impact on growth. However, once this leeway, or free capacity, is fully utilized, the extra load becomes a metabolic burden that inhibits cellular processes and triggers a broad cellular response, reducing cell growth and often hindering the production of heterologous proteins. In this study, we sought to characterize the metabolic rearrangements occurring in the central metabolism of Pseudomonas putida at different levels of metabolic load. To this end, we constructed a P. putida KT2440 strain that expressed two genes encoding fluorescent proteins, one in the genome under constitutive expression to monitor the free capacity, and the other on an inducible plasmid to probe heterologous protein production. We found that metabolic fluxes are profoundly reshuffled, notably at the level of periplasmic pathways, as soon as the metabolic load exceeds the free capacity. Heterologous protein production leads to the decoupling of anabolism and catabolism, resulting in large excess energy production relative to the requirements of protein biosynthesis. Finally, heterologous protein production was found to exert a stronger control on carbon fluxes than on energy fluxes, indicating that the flexible nature of P. putidas central metabolic network is solicited to sustain energy production.

HighlightsHeterologous protein production in P. putida reshuffles the periplasmic metabolism. Increased protein production progressively decouples catabolism from anabolism. Protein production exerts a stronger control on energy than on carbon fluxes. Glucose is directed towards ATP production to meet the elevated energy demands.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550304">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550304" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550304">
        <p class="paperTitle">Single-cell precision nanotechnology in vivo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550304" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550304" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: MOLBAY, M.; Kick, B.; Zhao, S.; Todorov, M. I.; Ohn, T.-L.; Roth, S.; Simats, A.; Singh, V.; Khalin, I.; Pan, C.; Singh Bhatia, H.; Hellal, F.; Zeidler, R.; Liesz, A.; Plesnila, N.; Dietz, H.; Erturk, A.</p>
        <p class="info">Score: 144.9, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550304' target='https://doi.org/10.1101/2023.07.24.550304'> 10.1101/2023.07.24.550304</a></p>
        <p class="abstract">Targeting nanoparticle therapeutics with cellular accuracy in whole organisms could open breakthrough opportunities in precision medicine. However, evaluating and fine-tuning the biodistribution of such systems in the whole organism at the cellular level remains a major obstacle. Here, we constructed targetable DNA origami, and analyzed biodistribution in transparent mice, in addition to studying tolerability, clearance kinetics, and immune response parameters. Untargeted DNA origami primarily accumulated in the spleen and the liver, while an immune cell-targeting variant successfully attached to immune cells throughout the body. A cancer cell-targeting mimetic co-localized on solid-tumor metastasis in the liver and the lung. These findings indicate that DNA origami can be directed in vivo, providing an important proof-of-concept and highlights the potential of high-resolution tissue-clearing imaging technologies in their development.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=121 SRC=&#34;FIGDIR/small/550304v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (44K):
org.highwire.dtl.DTLVardef@ce216borg.highwire.dtl.DTLVardef@f40bb4org.highwire.dtl.DTLVardef@2c6a66org.highwire.dtl.DTLVardef@1fd7fde_HPS_FORMAT_FIGEXP  M_FIG C_FIG HighlightsO_LIThis study demonstrates the potential of DNA origami-based drug delivery systems as versatile tool for for targeted delivery, which could be used to treat a range of diseases with applications.
C_LIO_LIThe immune compatibility, half-life, targeting efficiency, and the biodistribution evaluation of DNA origami indicate its potential for systemic drug delivery.
C_LIO_LIOur approach enables the assessment of biodistribution of nanoparticles in the intact body with a sensitivity to the single-cell level, highlighting the high-resolution tissue clearing technoloies in revealing DNA origamis feasibility for drug targeting.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552197">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552197" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552197">
        <p class="paperTitle">Stiffness anisotropy coordinates supracellular contractility driving long-range myotube-ECM alignment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552197" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552197" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Skillin, N. P.; Kirkpatrick, B. E.; Herbert, K. M.; Nelson, B. R.; Hach, G. K.; Gunay, K. A.; Khan, R. M.; DelRio, F. W.; White, T. J.; Anseth, K. S.</p>
        <p class="info">Score: 4.1, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552197' target='https://doi.org/10.1101/2023.08.08.552197'> 10.1101/2023.08.08.552197</a></p>
        <p class="abstract">In skeletal muscle tissue, injury-related changes in stiffness activate muscle stem cells through mechanosensitive signaling pathways. Functional muscle tissue regeneration also requires the effective coordination of myoblast proliferation, migration, polarization, differentiation, and fusion across multiple length scales. Here, we demonstrate that substrate stiffness anisotropy coordinates contractility-driven collective cellular dynamics resulting in C2C12 myotube alignment over millimeter-scale distances. When cultured on mechanically anisotropic liquid crystalline polymer networks (LCNs) lacking topographic features that could confer contact guidance, C2C12 myoblasts collectively polarize in the stiffest direction of the substrate. Cellular coordination is amplified through reciprocal cell-ECM dynamics that emerge during fusion, driving global myotube-ECM ordering. Conversely, myotube alignment was restricted to small local domains with no directional preference on mechanically isotropic LCNs of same chemical formulation. These findings reveal a role for stiffness anisotropy in coordinating emergent collective cellular dynamics, with implications for understanding skeletal muscle tissue development and regeneration.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552495">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552495" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552495">
        <p class="paperTitle">Heart-on-a-chip model of immune-induced cardiac dysfunction reveals the involvement of free mitochondrial DNA and therapeutic effects of endothelial exosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552495" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552495" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lu, X. Z.; Rafatian, N.; Zhao, Y.; Wagner, K. T.; Beroncal, E. L.; Li, B.; Lee, C.; Chen, J.; Churcher, E.; Vosoughi, D.; Wang, Y.; Baker, A.; Trahtemberg, U.; Li, B.; Pierro, A.; Andreazza, A.; dos Santos, C. C.; Radisic, M.</p>
        <p class="info">Score: 3.9, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552495' target='https://doi.org/10.1101/2023.08.09.552495'> 10.1101/2023.08.09.552495</a></p>
        <p class="abstract">Cardiovascular disease continues to take more human lives than all cancer combined, prompting the need for improved research models and treatment options. Despite a significant progress in development of mature heart-on-a-chip models of fibrosis and cardiomyopathies starting from induced pluripotent stem cells (iPSCs), human cell-based models of myocardial inflammation are lacking. Here, we bioengineered a vascularized heart-on-a-chip system with circulating immune cells to model SARS-CoV-2-induced acute myocarditis. Briefly, we observed hallmarks of COVID-19-induced myocardial inflammation in the heart-on-a-chip model, as the presence of immune cells augmented the expression levels of proinflammatory cytokines, triggered progressive impairment of contractile function and altered intracellular calcium transient activities. An elevation of circulating cell-free mitochondrial DNA (ccf-mtDNA) was measured first in the in vitro heart-on-a-chip model and then validated in COVID-19 patients with low left ventricular ejection fraction (LVEF), demonstrating that mitochondrial damage is an important pathophysiological hallmark of inflammation induced cardiac dysfunction. Leveraging this platform in the context of SARS-CoV-2 induced myocardial inflammation, we established that administration of human umbilical vein-derived EVs effectively rescued the contractile deficit, normalized intracellular calcium handling, elevated the contraction force and reduced the ccf- mtDNA and chemokine release via TLR-NF-kB signaling axis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552274">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552274" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552274">
        <p class="paperTitle">Nuclear Factor Kappa B Over-Activation in the Intervertebral Disc Leads to Macrophage Recruitment and Severe Disc Degeneration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552274" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552274" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Burt, K. G.; Kim, M. K. M.; Viola, D. C.; Abraham, A. C.; Chahine, N. O.</p>
        <p class="info">Score: 2.5, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552274' target='https://doi.org/10.1101/2023.08.07.552274'> 10.1101/2023.08.07.552274</a></p>
        <p class="abstract">ObjectiveLow back pain (LBP) is the leading cause of global disability and is thought to be driven primarily by intervertebral disc (IVD) degeneration (DD). Persistent upregulation of catabolic enzymes and inflammatory mediators have been associated with severe cases of DD. Nuclear factor kappa B (NF-{kappa}B) is a master transcription regulator of immune responses and is over expressed during inflammatory-driven musculoskeletal diseases, including DD. However, its role in triggering DD is unknown. Therefore, this study investigated the effect of NF-{kappa}B pathway over-activation on IVD integrity and DD pathology.

MethodsUsing skeletally mature mouse model, we genetically targeted IVD cells for canonical NF-{kappa}B pathway activation via expression of a constitutively active form of inhibitor of {kappa}B kinase B (IKK{beta}), and assessed changes in IVD cellularity, structural integrity including histology, disc height, and extracellular matrix (ECM) biochemistry, biomechanics, expression of inflammatory, catabolic, and neurotropic mediators, and changes in macrophage subsets, longitudinally up to 6-months post activation.

ResultsProlonged NF-{kappa}B activation led to severe structural degeneration, with a loss of glycosaminoglycan (GAG) content and complete loss of nucleus pulposus (NP) cellularity. Structural and compositional changes decreased IVD height and compressive mechanical properties with prolonged NF-{kappa}B activation. These alterations were accompanied by increases in gene expression of inflammatory molecules (Il1b, Il6, Nos2), chemokines (Mcp1, Mif), catabolic enzymes (Mmp3, Mmp9, Adamts4), and neurotrophic factors (Bdnf, Ngf) within IVD tissue. Increased recruitment of activated F4/80&#43; macrophages exhibited a greater abundance of pro-inflammatory (CD38&#43;) over inflammatory-resolving (CD206&#43;) macrophage subsets in the IVD, with temporal changes in the relative abundance of macrophage subsets over time, providing evidence for temporal regulation of macrophage polarization in DD in vivo, where macrophages participate in resolving the inflammatory cascade but promote fibrotic transformation of the IVD matrix. We further show that NF-{kappa}B driven secretory factors from IVD cells increase macrophage migration and inflammatory activation, and that the secretome of inflammatory-resolving macrophages mitigates effects of NF-{kappa}B overactivation.

ConclusionOverall the observed results suggest prolonged NF-{kappa}B activation can induce severe DD, acting through increases in inflammatory cytokines, chemotactic proteins, catabolic enzymes, and the recruitment and inflammatory activation of a macrophage cell populations, that can be mitigated with inflammatory-resolving macrophage secretome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.551544">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.551544" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.551544">
        <p class="paperTitle">Spatial and temporal control of 3D hydrogel viscoelasticity through phototuning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.551544" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.551544" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Crandell, P.; Stowers, R.</p>
        <p class="info">Score: 2.2, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.551544' target='https://doi.org/10.1101/2023.08.07.551544'> 10.1101/2023.08.07.551544</a></p>
        <p class="abstract">The mechanical properties of the extracellular environment can regulate a variety of cellular functions, such as spreading, migration, proliferation, and even differentiation and phenotypic determination. Much effort has been directed at understanding the effects of the extracellular matrix (ECM) elastic modulus and more recently, stress relaxation, on cellular processes. In physiological contexts like development, wound healing, and fibrotic disease progression, ECM mechanical properties change substantially over time or space. Dynamically tunable hydrogel platforms have been developed to spatiotemporally modulate a gels elastic modulus. However, dynamically altering the stress relaxation rate of a hydrogel remains a challenge. Here, we present a strategy to tune hydrogel stress relaxation rates in time or space using a light-triggered tethering of poly(ethylene glycol) (PEG) to alginate. We show that stress relaxation rate can be tuned without altering the elastic modulus of the hydrogel. We found that cells are capable of sensing and responding to dynamic stress relaxation rate changes, both morphologically and through differences in proliferation rates. We also exploited the light-based technique to generate spatial patterns of stress relaxation rates in 3D hydrogels. We anticipate that user-directed control of 3D hydrogel stress relaxation rate will be a powerful tool that enables studies that mimic dynamic ECM contexts, or as a means to guide cell fate in space and time for tissue engineering applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549967">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549967" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549967">
        <p class="paperTitle">Simple and rewireable biomolecular building blocks for DNA machine-learning algorithms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549967" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549967" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, R. C.; Corsano, A.; Tseng, C. Y.; Chou, L. Y. T.</p>
        <p class="info">Score: 10.5, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549967' target='https://doi.org/10.1101/2023.07.20.549967'> 10.1101/2023.07.20.549967</a></p>
        <p class="abstract">Deep learning algorithms, such as neural networks, enable the processing of complex datasets with many related variables, and have applications in disease diagnosis, cell profiling, and drug discovery. Beyond its use in electronic computers, neural networks have been implemented using programmable biomolecules such as DNA. This confers unique advantages such as greater portability, ability to operate without electricity, and direct analysis of patterns of biomolecules in solution. Analogous to past bottlenecks in electronic computers, the computing power of DNA-based neural networks is limited by the ability to add more computing units, i.e. neurons. This limitation exists because current architectures require many nucleic acids to model a single neuron. Each addition of a neuron to the network compounds existing problems such as long assembly times, high background signal, and cross-talk between components. Here we test three strategies to solve this limitation and improve the scalability of DNA-based neural networks: (i) enzymatic synthesis to generate high-purity neurons, (ii) spatial patterning of neuron clusters based on their network position, and (iii) encoding neuron connectivity on a universal single-stranded DNA backbone. We show that neurons implemented via these strategies activate quickly, with high signal-to-background ratio, and respond to varying input concentrations and weights. Using this neuron design, we implemented basic neural network motifs such as cascading, fan-in, and fan-out circuits. Since this design is modular, easy to synthesize, and compatible with multiple neural network architectures, we envision it will help scale DNA-based neural networks in a variety of settings. This will enable portable computing power for applications such as portable diagnostics, compact data storage, and autonomous decision making for lab-on-a-chips.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.551713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.551713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.551713">
        <p class="paperTitle">Microbiome single cell atlases generated with a commercial instrument</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.551713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.551713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, X.; Xu, L.; Demaree, B.; Noecker, C.; Bisanz, J. E.; Weisgerber, D. W.; Modavi, C.; Turnbaugh, P. J.; Abate, A. R.</p>
        <p class="info">Score: 2.0, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.551713' target='https://doi.org/10.1101/2023.08.08.551713'> 10.1101/2023.08.08.551713</a></p>
        <p class="abstract">Single cell sequencing is useful for resolving complex systems into their composite cell types and computationally mining them for unique features that are masked in pooled sequencing. However, while commercial instruments have made single cell analysis widespread for mammalian cells, analogous tools for microbes are limited. Here, we present EASi-seq (Easily Accessible Single microbe sequencing). By adapting the single cell workflow of the commercial Mission Bio Tapestri instrument, this method allows for efficient sequencing of individual microbes&#39; genomes. EASi-seq allows thousands of microbes to be sequenced per run and, as we show, can generate detailed atlases of human and environmental microbiomes. The ability to capture large shotgun genome datasets from thousands of single microbes provides new opportunities in discovering and analyzing species subpopulations. To facilitate this, we develop a companion bioinformatic pipeline that clusters microbes by similarity, improving whole genome assembly, strain identification, taxonomic classification, and gene annotation. In addition, we demonstrate integration of metagenomic contigs with the EASi-seq datasets to reduce capture bias and increase coverage. Overall, EASi-seq enables high quality single cell genomic data for microbiome samples using an accessible workflow that can be run on a commercially available platform.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
