<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR-Cas9 RNP
Authors: Chen, K.; Han, H.; Zhao, S.; Xu, B.; Yin, B.; Trinidad, M.; Burgstone, B. W.; Murthy, N.; Doudna, J. A.
Score: 25.7, Published: 2023-11-15 DOI: 10.1101/2023.11.15.566339
Lipid nanoparticle (LNP) delivery of CRISPR ribonucleoproteins (RNPs) has the potential to enable high-efficiency in vivo genome editing with low toxicity and an easily manufactured technology, if RNP efficacy can be maintained during LNP production.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR-Cas9 RNP
Authors: Chen, K.; Han, H.; Zhao, S.; Xu, B.; Yin, B.; Trinidad, M.; Burgstone, B. W.; Murthy, N.; Doudna, J. A.
Score: 25.7, Published: 2023-11-15 DOI: 10.1101/2023.11.15.566339
Lipid nanoparticle (LNP) delivery of CRISPR ribonucleoproteins (RNPs) has the potential to enable high-efficiency in vivo genome editing with low toxicity and an easily manufactured technology, if RNP efficacy can be maintained during LNP production." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-22T10:41:31+00:00" />
<meta property="article:modified_time" content="2023-11-22T10:41:31+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR-Cas9 RNP
Authors: Chen, K.; Han, H.; Zhao, S.; Xu, B.; Yin, B.; Trinidad, M.; Burgstone, B. W.; Murthy, N.; Doudna, J. A.
Score: 25.7, Published: 2023-11-15 DOI: 10.1101/2023.11.15.566339
Lipid nanoparticle (LNP) delivery of CRISPR ribonucleoproteins (RNPs) has the potential to enable high-efficiency in vivo genome editing with low toxicity and an easily manufactured technology, if RNP efficacy can be maintained during LNP production."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR-Cas9 RNP\nAuthors: Chen, K.; Han, H.; Zhao, S.; Xu, B.; Yin, B.; Trinidad, M.; Burgstone, B. W.; Murthy, N.; Doudna, J. A.\nScore: 25.7, Published: 2023-11-15 DOI: 10.1101/2023.11.15.566339\nLipid nanoparticle (LNP) delivery of CRISPR ribonucleoproteins (RNPs) has the potential to enable high-efficiency in vivo genome editing with low toxicity and an easily manufactured technology, if RNP efficacy can be maintained during LNP production.",
  "keywords": [
    
  ],
  "articleBody": " Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR-Cas9 RNP\nAuthors: Chen, K.; Han, H.; Zhao, S.; Xu, B.; Yin, B.; Trinidad, M.; Burgstone, B. W.; Murthy, N.; Doudna, J. A.\nScore: 25.7, Published: 2023-11-15 DOI: 10.1101/2023.11.15.566339\nLipid nanoparticle (LNP) delivery of CRISPR ribonucleoproteins (RNPs) has the potential to enable high-efficiency in vivo genome editing with low toxicity and an easily manufactured technology, if RNP efficacy can be maintained during LNP production. In this study, we engineered a thermostable Cas9 from Geobacillus stearothermophilus (GeoCas9) using directed evolution to generate iGeoCas9 evolved variants capable of robust genome editing of cells and organs. iGeoCas9s were significantly better at editing cells than wild-type GeoCas9, with genome editing levels \u003e100X greater than those induced by the native GeoCas9 enzyme. Furthermore, iGeoCas9 RNP:LNP complexes edited a variety of cell lines and induced homology-directed repair (HDR) in cells receiving co-delivered single-stranded DNA (ssDNA) templates. Using tissue-selective LNP formulations, we observed genome editing of 35-56% efficiency in the liver or lungs of mice that received intravenous injections of iGeoCas9 RNP:LNPs. In particular, iGeoCas9 complexed to acid-degradable LNPs edited lung tissue in vivo with an average of 35% efficiency, a significant improvement over editing efficiencies observed previously using viral or non-viral delivery strategies. These results show that thermostable Cas9 RNP:LNP complexes are a powerful alternative to mRNA:LNP delivery vehicles, expanding the therapeutic potential of genome editing.\nEngineering self-deliverable ribonucleoproteins for genome editing in the brain\nAuthors: Chen, K.; Stahl, E. C.; Kang, M. H.; Xu, B.; Allen, R.; Trinidad, M.; Doudna, J. A.\nScore: 22.9, Published: 2023-11-15 DOI: 10.1101/2023.11.15.567251\nThe delivery of CRISPR ribonucleoproteins (RNPs) for genome editing in vitro and in vivo has important advantages over other delivery methods, including reduced off-target and immunogenic effects1. However, effective delivery of RNPs remains challenging in certain cell types due to low efficiency and cell toxicity. To address these issues, we engineered self-deliverable RNPs that can promote efficient cellular uptake and carry out robust genome editing without the need for helper materials or biomolecules. Screening of cell-penetrating peptides (CPPs) fused to CRISPR-Cas9 protein identified potent constructs capable of efficient genome editing of neural progenitor cells. Further engineering of these fusion proteins identified a C-terminal Cas9 fusion with three copies of A22p, a peptide derived from human semaphorin-3a, that exhibited substantially improved editing efficacy compared to other constructs. We found that self-deliverable Cas9 RNPs generated robust genome edits in clinically relevant genes when injected directly into the mouse striatum. Overall, self-deliverable Cas9 proteins provide a facile and effective platform for genome editing in vitro and in vivo.\nUnusual Structural Properties of Switchback DNA\nAuthors: Madhanagopal, B. R.; Talbot, H.; Rodriguez, A.; Louis, J. M.; Zeghal, H.; Vangaveti, S.; Reddy, K.; Chandrasekaran, A. R.\nScore: 7.0, Published: 2023-11-17 DOI: 10.1101/2023.11.15.563609\nSynthetic DNA motifs form the basis of nucleic acid nanotechnology, and their biochemical and biophysical properties determine their applications. Here, we present a detailed characterization of switchback DNA, a globally left-handed structure composed of two parallel DNA strands. Compared to a conventional duplex, switchback DNA shows lower thermodynamic stability and requires higher magnesium concentration for assembly, but exhibits a higher biostability against some nucleases. Strand competition and strand displacement experiments show that component sequences have an absolute preference for duplex complements instead of their switchback partners. Further, we hypothesize a potential role for switchback DNA as an alternate structure for short-tandem repeats involved in repeat-expansion diseases. Together with small molecule binding experiments and cell studies, our results open new avenues for synthetic DNA motifs in biology and nanotechnology.\nUltrasonic reporters of calcium for deep tissue imaging of cellular signals\nAuthors: Jin, Z.; Lakshmanan, A.; Zhang, R.; Tran, T. A.; Rabut, C.; Dutka, P.; Duan, M.; Hurt, R. C.; Malounda, D.; Yao, Y.; Shapiro, M. G.\nScore: 76.6, Published: 2023-11-12 DOI: 10.1101/2023.11.09.566364\nCalcium imaging has enabled major biological discoveries. However, the scattering of light by tissue limits the use of standard fluorescent calcium indicators in living animals. To address this limitation, we introduce the first genetically encoded ultrasonic reporter of calcium (URoC). Based on a unique class of air-filled protein nanostructures called gas vesicles, we engineered URoC to produce elevated nonlinear ultrasound signal upon binding to calcium ions. With URoC expressed in mammalian cells, we demonstrate noninvasive ultrasound imaging of calcium signaling in vivo during drug-induced receptor activation. URoC brings the depth and resolution advantages of ultrasound to the in vivo imaging of dynamic cellular function and paves the way for acoustic biosensing of a broader variety of biological signals.\nHyperspectral Oblique Plane Microscopy Enables Spontaneous, Label-Free Imaging of Biological Dynamic Processes in Live Animals\nAuthors: Guo, K.; Kalyviotis, K.; Pantazis, P.; Rowlands, C. J.\nScore: 11.2, Published: 2023-11-10 DOI: 10.1101/2023.03.15.532804\nSpontaneous Raman imaging has emerged as powerful label-free technique for investigating the molecular composition of biological specimens. Although Raman imaging can facilitate understanding of complex biological phenomena in vivo, current imaging modalities are limited in speed and sample compatibility. Here, we introduce a single-objective light-sheet microscope, named{lambda} -OPM, which records Raman images on a timescale of minutes to milliseconds. To demonstrate its function, we use{lambda} -OPM to map and identify micro-plastic particles based on their Raman spectral characteristics. In live zebrafish embryos, we show that{lambda} -OPM can capture wound dynamics at five-minute intervals, revealing rapid changes in cellular and extracellular matrix composition in the wounded region. Finally, we use{lambda} -OPM to obtain Raman scattering maps of a zebrafish embryos beating heart at an effective 28 frames per second, recording compositional changes at different points in the cardiac cycle.\nOptimizing Niosome Formulations for Enhanced Cellular Applications: A Comparative Case Study with L-α-lecithin Liposomes\nAuthors: Cakir, N.; Ozturk, N.; Kara, A.; Zarrabi, A.; Mustafaoglu, N.\nScore: 3.0, Published: 2023-11-16 DOI: 10.1101/2023.11.14.567080\nNiosomes, emerging as nonionic surfactant-derived amphiphilic nanoparticles, hold substantial promise in the realm of biomedical research. This study addresses the need for a comprehensive exploration of niosome production optimization for biological applications, while also establishing meaningful comparisons with the well-established liposomal counterparts. Beyond conventional stability assessments, our motivation centers on discerning not only critical niosome process parameters but also on devising cost-effective, scalable alternatives to liposomes through comparative studies of liposomes and niosomes, rather than solely emphasizing niosomal stability advantages. The primary objective of this study is to formulate and characterize a diverse array of niosomal nanoparticles, with a prime focus on their process-related parameters, physicochemical characteristics, cellular uptake, and toxicity performances. To establish the niosomes as their research twins of liposomes, the gap in the research field is picked as the starting point. The study is designed with stringent criteria based on the limitations of vasculature-tissue barriers. The proposed encompassing size (100-200 nm), polydispersity below 0.5, and zeta potential within the range of -10 to 10 mV are set for this purpose. These criteria serve as the initial screening parameters, streamlining the selection of niosome formulations with the potential to overcome the barriers. Through meticulous physicochemical characterization, we synthesized 10 optimized formulations aligned with the targeted size, polydispersity, and zeta potential ranges. In this physicochemical critical process parameter screening, short and long-term stability, shelf-life aggregation profiles, and the reproducibility of formulations were also assessed to confidently report the potential niosomal formulations for further drug delivery purposes. The statistical evaluations and analytical screening of process parameters obtained from the DoE interface indicated that most formulations maintain their critical criteria for at least 21 days, with three formulations remaining stable for 35 days. Reproducibility tests further validate the consistency of eight out of ten formulations regarding size, polydispersity, and surface charge. The F-score confirms high similarity between predicted and observed physicochemical properties (F-score = 0.83) for reproducibility tests. Concurrently, we explore the pivotal process parameters governing niosome preparation and their consequential impact on physicochemical attributes. Further, physiochemically selected niosomal carriers are simultaneously exposed to cellular applications with L--lecithin liposomes including cellular toxicity and cellular uptake. In cellular toxicity, the selected niosomes from physicochemical screening were exposed to two different cancerous cell lines belonging to glioblastoma multiform (U-87 MG) and lymphoblast-like cell line (NFS-60). The cellular uptake profiles in U-87 MG and simultaneous comparison with liposomes revealed non-toxicity across all formulations and promising cellular uptake performance in four formulations, either similar to or better than liposomes. Overall, this study holds potential implications of niosomes for advancing reliable drug delivery strategies, enhancing treatment efficacy, and ensuring safety in various therapeutic applications. Besides, demonstrating the scientific records of physiochemically controlled niosomes similarity to a type of liposomes in cellular interactions and scalable production will ultimately expand their applications in the field of biomedical research.\nMicroplate Format Protein Nanopatterning for High-Throughput Screening of Cellular Microenvironments\nAuthors: Shahrokhtash, A.; Sivertsen, M. v. T.; Laursen, S. H.; Sutherland, D. S.\nScore: 2.8, Published: 2023-11-19 DOI: 10.1101/2023.11.19.567703\nAn advanced protein nanopatterned cell culture platform is engineered to emulate the extracellular matrix's complexity, enabling precise nanoscale biomolecule copatterning to mimic environments analogous to native tissue for cellular assays. Nanopatterns fabricated through sparse colloidal lithography, with 100 nm to 800 nm features in separate wells, are seamlessly integrated into standard microplate formats (96-well/384-well). Robust patterns are built from fully PEGylated, passivated thin glass coverslips optimized for minimal nonspecific interactions. Biotin-avidin binding and click chemistry to ensure the accurate and robust localization of bioligands. The transparent, metal-free substrates are free of topographical interference, rendering them ideal for diverse fluorescence microscopy techniques encompassing single-molecule TIRFM and extensive high-throughput imaging. The structural stability of these nanopatterns persists beyond a year in storage and long-term in cell culture conditions, endorsing their application for prolonged experimental studies and potential for widespread academic and industrial use. The platform has been demonstrated for nanopatterning an array of biomolecules, from small molecules to proteins, DNA, and extracellular matrix components, instrumental for studying cell signaling. Experiments with C2C12 cells demonstrated the exceptional specificity of the nanopatterned microplates, with nonspecific adhesion remaining below 2% and the platform's ability to elicit size-dependent cellular reactions when interfaced with nanopatterned fibronectin.\nDONOR VARIABILITY IN HUMAN MESENCHYMAL STEM CELL OSTEOGENIC RESPONSE AS A FUNCTION OF PASSAGE CONDITIONS AND DONOR SEX\nAuthors: Kolliopoulos, V.; Tiffany, A.; Polanek, M.; Harley, B.\nScore: 2.8, Published: 2023-11-15 DOI: 10.1101/2023.11.12.566781\nContemporary tissue engineering efforts often seek to use mesenchymal stem cells (MSCs) due to their potential to differentiate to various tissue-specific cells and generate a pro-regenerative secretome. While MSC differentiation and therapeutic potential can differ as a function of matrix environment, it may also be widely influenced as a function of donor-to-donor variability. Further, effects of passage number and donor sex may further convolute the identification of clinically effective MSC-mediated regeneration technologies. We report efforts to adapt a well-defined mineralized collagen scaffold platform to study the influence of MSC proliferation and osteogenic potential as a function of passage number and donor sex. Mineralized collagen scaffolds broadly support MSC osteogenic differentiation and regenerative potency in the absence of traditional osteogenic supplements for a wide range of MSCs (rabbit, rat, porcine, human). We obtained a library of bone marrow and adipose tissue derived stem cells to examine donor-variability of regenerative potency in mineralized collagen scaffolds. MSCs displayed reduced proliferative capacity as a function of passage duration. Further, MSCs showed significant sex-based differences. Notably, MSCs from male donors displayed significantly higher metabolic activity and proliferation while MSCs from female donor displayed significantly higher osteogenic response via increased alkaline phosphate activity, osteoprotegerin release, and mineral formation in vitro. Our study highlights the essentiality of considering MSC donor sex and culture expansion in future studies of biomaterial regenerative potential.\nPolymer-mediated oligonucleotide delivery enables construction of barcoded 3D cultures for spatial single-cell analysis\nAuthors: King, J. J.; Mowla, A.; Kretzmann, J. A.; Norret, M.; Kadolsky, U. D.; Iqbal, M.; Saxena, A.; Mamsa, S. S. A.; Amos, S. E.; Choi, Y. S.; Kennedy, B. F.; Iyer, K. S.; Smith, N. M.; Evans, C. W.\nScore: 2.2, Published: 2023-11-21 DOI: 10.1101/2023.11.20.567985\nSpatial transcriptomics has been widely used to capture gene expression profiles, realised as a two-dimensional (2D) projection of RNA captured from tissue sections. Three-dimensional (3D) cultures such as spheroids and organoids are highly promising alternatives to oversimplified and homogeneous 2D cell culture models, but existing spatial transcriptomic platforms do not currently have sufficient resolution for robust analysis of 3D cultures. We present a transfection-based method for fluorescent DNA barcoding of cell populations, and the subsequent construction of spheroidal cellular architectures using barcoded cells in a layer-by-layer approach. For the first time, changes in gene expression throughout this 3D culture architecture are interrogated using multiplex single-cell RNA sequencing in which DNA barcodes are used to encode the spatial positioning of cells. We show that transfection with fluorophore-conjugated barcode oligonucleotides enables both imaging and sequencing at single-cell resolution, providing spatial maps of gene expression and drug response. Additionally, we show that fluorophore-conjugated DNA barcodes support correlative imaging studies such as mechano-microscopy to capture information about spatially-varying mechanical heterogeneity in 3D cultures. The ability to create customised, encoded cellular assemblies is a general approach that can resolve spatial differences in gene expression in 3D cell culture models.\nMouse Genome Editing from Scratch\nAuthors: Popova, J.; Bets, V.; Omelina, E.; Boldyreva, L.; Kozhevnikova, E.\nScore: 2.2, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567793\nMouse genome modification requires costly equipment and highly skilled personnel to manipulate zygotes. A number of zygote electroporation techniques were reported to be highly efficient in gene delivery. One of these methods called i-GONAD (improved Genome-editing via Oviductal Nucleic Acids Delivery) describes electroporation-based gene transfer to zygotes in utero. Here we adopted this technology for mouse genome-editing from scratch minimizing the cost of equipment, operator skill and animal use. We chose the CRISPR/Cas9 system as a genome editing tool and i-GONAD as a gene delivery method to produce IL10 gene knockout in C57BL/6 mice. Four animals out of 13 delivered pups (30.8%) were genetically compromised at IL10 gene locus suggesting the feasibility of the approach. This report provides one of the possible technical settings for those who aim at establishing in-house mouse transgenesis pipeline at minimal cost from scratch.\n",
  "wordCount" : "2319",
  "inLanguage": "en",
  "datePublished": "2023-11-22T10:41:31Z",
  "dateModified": "2023-11-22T10:41:31Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 22, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.566339">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.566339" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.566339">
        <p class="paperTitle">Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR-Cas9 RNP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.566339" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.566339" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, K.; Han, H.; Zhao, S.; Xu, B.; Yin, B.; Trinidad, M.; Burgstone, B. W.; Murthy, N.; Doudna, J. A.</p>
        <p class="info">Score: 25.7, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.566339' target='https://doi.org/10.1101/2023.11.15.566339'> 10.1101/2023.11.15.566339</a></p>
        <p class="abstract">Lipid nanoparticle (LNP) delivery of CRISPR ribonucleoproteins (RNPs) has the potential to enable high-efficiency in vivo genome editing with low toxicity and an easily manufactured technology, if RNP efficacy can be maintained during LNP production. In this study, we engineered a thermostable Cas9 from Geobacillus stearothermophilus (GeoCas9) using directed evolution to generate iGeoCas9 evolved variants capable of robust genome editing of cells and organs. iGeoCas9s were significantly better at editing cells than wild-type GeoCas9, with genome editing levels &gt;100X greater than those induced by the native GeoCas9 enzyme. Furthermore, iGeoCas9 RNP:LNP complexes edited a variety of cell lines and induced homology-directed repair (HDR) in cells receiving co-delivered single-stranded DNA (ssDNA) templates. Using tissue-selective LNP formulations, we observed genome editing of 35-56% efficiency in the liver or lungs of mice that received intravenous injections of iGeoCas9 RNP:LNPs. In particular, iGeoCas9 complexed to acid-degradable LNPs edited lung tissue in vivo with an average of 35% efficiency, a significant improvement over editing efficiencies observed previously using viral or non-viral delivery strategies. These results show that thermostable Cas9 RNP:LNP complexes are a powerful alternative to mRNA:LNP delivery vehicles, expanding the therapeutic potential of genome editing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567251">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567251" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567251">
        <p class="paperTitle">Engineering self-deliverable ribonucleoproteins for genome editing in the brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567251" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567251" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, K.; Stahl, E. C.; Kang, M. H.; Xu, B.; Allen, R.; Trinidad, M.; Doudna, J. A.</p>
        <p class="info">Score: 22.9, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567251' target='https://doi.org/10.1101/2023.11.15.567251'> 10.1101/2023.11.15.567251</a></p>
        <p class="abstract">The delivery of CRISPR ribonucleoproteins (RNPs) for genome editing in vitro and in vivo has important advantages over other delivery methods, including reduced off-target and immunogenic effects1. However, effective delivery of RNPs remains challenging in certain cell types due to low efficiency and cell toxicity. To address these issues, we engineered self-deliverable RNPs that can promote efficient cellular uptake and carry out robust genome editing without the need for helper materials or biomolecules. Screening of cell-penetrating peptides (CPPs) fused to CRISPR-Cas9 protein identified potent constructs capable of efficient genome editing of neural progenitor cells. Further engineering of these fusion proteins identified a C-terminal Cas9 fusion with three copies of A22p, a peptide derived from human semaphorin-3a, that exhibited substantially improved editing efficacy compared to other constructs. We found that self-deliverable Cas9 RNPs generated robust genome edits in clinically relevant genes when injected directly into the mouse striatum. Overall, self-deliverable Cas9 proteins provide a facile and effective platform for genome editing in vitro and in vivo.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.563609">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.563609" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.563609">
        <p class="paperTitle">Unusual Structural Properties of Switchback DNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.563609" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.563609" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Madhanagopal, B. R.; Talbot, H.; Rodriguez, A.; Louis, J. M.; Zeghal, H.; Vangaveti, S.; Reddy, K.; Chandrasekaran, A. R.</p>
        <p class="info">Score: 7.0, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.563609' target='https://doi.org/10.1101/2023.11.15.563609'> 10.1101/2023.11.15.563609</a></p>
        <p class="abstract">Synthetic DNA motifs form the basis of nucleic acid nanotechnology, and their biochemical and biophysical properties determine their applications. Here, we present a detailed characterization of switchback DNA, a globally left-handed structure composed of two parallel DNA strands. Compared to a conventional duplex, switchback DNA shows lower thermodynamic stability and requires higher magnesium concentration for assembly, but exhibits a higher biostability against some nucleases. Strand competition and strand displacement experiments show that component sequences have an absolute preference for duplex complements instead of their switchback partners. Further, we hypothesize a potential role for switchback DNA as an alternate structure for short-tandem repeats involved in repeat-expansion diseases. Together with small molecule binding experiments and cell studies, our results open new avenues for synthetic DNA motifs in biology and nanotechnology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566364">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566364" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566364">
        <p class="paperTitle">Ultrasonic reporters of calcium for deep tissue imaging of cellular signals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566364" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566364" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jin, Z.; Lakshmanan, A.; Zhang, R.; Tran, T. A.; Rabut, C.; Dutka, P.; Duan, M.; Hurt, R. C.; Malounda, D.; Yao, Y.; Shapiro, M. G.</p>
        <p class="info">Score: 76.6, Published: 2023-11-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566364' target='https://doi.org/10.1101/2023.11.09.566364'> 10.1101/2023.11.09.566364</a></p>
        <p class="abstract">Calcium imaging has enabled major biological discoveries. However, the scattering of light by tissue limits the use of standard fluorescent calcium indicators in living animals. To address this limitation, we introduce the first genetically encoded ultrasonic reporter of calcium (URoC). Based on a unique class of air-filled protein nanostructures called gas vesicles, we engineered URoC to produce elevated nonlinear ultrasound signal upon binding to calcium ions. With URoC expressed in mammalian cells, we demonstrate noninvasive ultrasound imaging of calcium signaling in vivo during drug-induced receptor activation. URoC brings the depth and resolution advantages of ultrasound to the in vivo imaging of dynamic cellular function and paves the way for acoustic biosensing of a broader variety of biological signals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.03.15.532804">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.03.15.532804" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.03.15.532804">
        <p class="paperTitle">Hyperspectral Oblique Plane Microscopy Enables Spontaneous, Label-Free Imaging of Biological Dynamic Processes in Live Animals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.03.15.532804" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.03.15.532804" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guo, K.; Kalyviotis, K.; Pantazis, P.; Rowlands, C. J.</p>
        <p class="info">Score: 11.2, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.03.15.532804' target='https://doi.org/10.1101/2023.03.15.532804'> 10.1101/2023.03.15.532804</a></p>
        <p class="abstract">Spontaneous Raman imaging has emerged as powerful label-free technique for investigating the molecular composition of biological specimens. Although Raman imaging can facilitate understanding of complex biological phenomena in vivo, current imaging modalities are limited in speed and sample compatibility. Here, we introduce a single-objective light-sheet microscope, named{lambda} -OPM, which records Raman images on a timescale of minutes to milliseconds. To demonstrate its function, we use{lambda} -OPM to map and identify micro-plastic particles based on their Raman spectral characteristics. In live zebrafish embryos, we show that{lambda} -OPM can capture wound dynamics at five-minute intervals, revealing rapid changes in cellular and extracellular matrix composition in the wounded region. Finally, we use{lambda} -OPM to obtain Raman scattering maps of a zebrafish embryos beating heart at an effective 28 frames per second, recording compositional changes at different points in the cardiac cycle.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567080">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567080" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567080">
        <p class="paperTitle">Optimizing Niosome Formulations for Enhanced Cellular Applications: A Comparative Case Study with L-α-lecithin Liposomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567080" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567080" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cakir, N.; Ozturk, N.; Kara, A.; Zarrabi, A.; Mustafaoglu, N.</p>
        <p class="info">Score: 3.0, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567080' target='https://doi.org/10.1101/2023.11.14.567080'> 10.1101/2023.11.14.567080</a></p>
        <p class="abstract">Niosomes, emerging as nonionic surfactant-derived amphiphilic nanoparticles, hold substantial promise in the realm of biomedical research. This study addresses the need for a comprehensive exploration of niosome production optimization for biological applications, while also establishing meaningful comparisons with the well-established liposomal counterparts. Beyond conventional stability assessments, our motivation centers on discerning not only critical niosome process parameters but also on devising cost-effective, scalable alternatives to liposomes through comparative studies of liposomes and niosomes, rather than solely emphasizing niosomal stability advantages.

The primary objective of this study is to formulate and characterize a diverse array of niosomal nanoparticles, with a prime focus on their process-related parameters, physicochemical characteristics, cellular uptake, and toxicity performances. To establish the niosomes as their research twins of liposomes, the gap in the research field is picked as the starting point. The study is designed with stringent criteria based on the limitations of vasculature-tissue barriers. The proposed encompassing size (100-200 nm), polydispersity below 0.5, and zeta potential within the range of -10 to 10 mV are set for this purpose. These criteria serve as the initial screening parameters, streamlining the selection of niosome formulations with the potential to overcome the barriers. Through meticulous physicochemical characterization, we synthesized 10 optimized formulations aligned with the targeted size, polydispersity, and zeta potential ranges. In this physicochemical critical process parameter screening, short and long-term stability, shelf-life aggregation profiles, and the reproducibility of formulations were also assessed to confidently report the potential niosomal formulations for further drug delivery purposes. The statistical evaluations and analytical screening of process parameters obtained from the DoE interface indicated that most formulations maintain their critical criteria for at least 21 days, with three formulations remaining stable for 35 days. Reproducibility tests further validate the consistency of eight out of ten formulations regarding size, polydispersity, and surface charge. The F-score confirms high similarity between predicted and observed physicochemical properties (F-score = 0.83) for reproducibility tests.

Concurrently, we explore the pivotal process parameters governing niosome preparation and their consequential impact on physicochemical attributes. Further, physiochemically selected niosomal carriers are simultaneously exposed to cellular applications with L--lecithin liposomes including cellular toxicity and cellular uptake. In cellular toxicity, the selected niosomes from physicochemical screening were exposed to two different cancerous cell lines belonging to glioblastoma multiform (U-87 MG) and lymphoblast-like cell line (NFS-60). The cellular uptake profiles in U-87 MG and simultaneous comparison with liposomes revealed non-toxicity across all formulations and promising cellular uptake performance in four formulations, either similar to or better than liposomes.

Overall, this study holds potential implications of niosomes for advancing reliable drug delivery strategies, enhancing treatment efficacy, and ensuring safety in various therapeutic applications. Besides, demonstrating the scientific records of physiochemically controlled niosomes similarity to a type of liposomes in cellular interactions and scalable production will ultimately expand their applications in the field of biomedical research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.19.567703">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.19.567703" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.19.567703">
        <p class="paperTitle">Microplate Format Protein Nanopatterning for High-Throughput Screening of Cellular Microenvironments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.19.567703" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.19.567703" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shahrokhtash, A.; Sivertsen, M. v. T.; Laursen, S. H.; Sutherland, D. S.</p>
        <p class="info">Score: 2.8, Published: 2023-11-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.19.567703' target='https://doi.org/10.1101/2023.11.19.567703'> 10.1101/2023.11.19.567703</a></p>
        <p class="abstract">An advanced protein nanopatterned cell culture platform is engineered to emulate the extracellular matrix&#39;s complexity, enabling precise nanoscale biomolecule copatterning to mimic environments analogous to native tissue for cellular assays. Nanopatterns fabricated through sparse colloidal lithography, with 100 nm to 800 nm features in separate wells, are seamlessly integrated into standard microplate formats (96-well/384-well). Robust patterns are built from fully PEGylated, passivated thin glass coverslips optimized for minimal nonspecific interactions. Biotin-avidin binding and click chemistry to ensure the accurate and robust localization of bioligands. The transparent, metal-free substrates are free of topographical interference, rendering them ideal for diverse fluorescence microscopy techniques encompassing single-molecule TIRFM and extensive high-throughput imaging. The structural stability of these nanopatterns persists beyond a year in storage and long-term in cell culture conditions, endorsing their application for prolonged experimental studies and potential for widespread academic and industrial use. The platform has been demonstrated for nanopatterning an array of biomolecules, from small molecules to proteins, DNA, and extracellular matrix components, instrumental for studying cell signaling. Experiments with C2C12 cells demonstrated the exceptional specificity of the nanopatterned microplates, with nonspecific adhesion remaining below 2% and the platform&#39;s ability to elicit size-dependent cellular reactions when interfaced with nanopatterned fibronectin.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.12.566781">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.12.566781" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.12.566781">
        <p class="paperTitle">DONOR VARIABILITY IN HUMAN MESENCHYMAL STEM CELL OSTEOGENIC RESPONSE AS A FUNCTION OF PASSAGE CONDITIONS AND DONOR SEX</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.12.566781" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.12.566781" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kolliopoulos, V.; Tiffany, A.; Polanek, M.; Harley, B.</p>
        <p class="info">Score: 2.8, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.12.566781' target='https://doi.org/10.1101/2023.11.12.566781'> 10.1101/2023.11.12.566781</a></p>
        <p class="abstract">Contemporary tissue engineering efforts often seek to use mesenchymal stem cells (MSCs) due to their potential to differentiate to various tissue-specific cells and generate a pro-regenerative secretome. While MSC differentiation and therapeutic potential can differ as a function of matrix environment, it may also be widely influenced as a function of donor-to-donor variability. Further, effects of passage number and donor sex may further convolute the identification of clinically effective MSC-mediated regeneration technologies. We report efforts to adapt a well-defined mineralized collagen scaffold platform to study the influence of MSC proliferation and osteogenic potential as a function of passage number and donor sex. Mineralized collagen scaffolds broadly support MSC osteogenic differentiation and regenerative potency in the absence of traditional osteogenic supplements for a wide range of MSCs (rabbit, rat, porcine, human). We obtained a library of bone marrow and adipose tissue derived stem cells to examine donor-variability of regenerative potency in mineralized collagen scaffolds. MSCs displayed reduced proliferative capacity as a function of passage duration. Further, MSCs showed significant sex-based differences. Notably, MSCs from male donors displayed significantly higher metabolic activity and proliferation while MSCs from female donor displayed significantly higher osteogenic response via increased alkaline phosphate activity, osteoprotegerin release, and mineral formation in vitro. Our study highlights the essentiality of considering MSC donor sex and culture expansion in future studies of biomaterial regenerative potential.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567985">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567985" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567985">
        <p class="paperTitle">Polymer-mediated oligonucleotide delivery enables construction of barcoded 3D cultures for spatial single-cell analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567985" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567985" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: King, J. J.; Mowla, A.; Kretzmann, J. A.; Norret, M.; Kadolsky, U. D.; Iqbal, M.; Saxena, A.; Mamsa, S. S. A.; Amos, S. E.; Choi, Y. S.; Kennedy, B. F.; Iyer, K. S.; Smith, N. M.; Evans, C. W.</p>
        <p class="info">Score: 2.2, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567985' target='https://doi.org/10.1101/2023.11.20.567985'> 10.1101/2023.11.20.567985</a></p>
        <p class="abstract">Spatial transcriptomics has been widely used to capture gene expression profiles, realised as a two-dimensional (2D) projection of RNA captured from tissue sections. Three-dimensional (3D) cultures such as spheroids and organoids are highly promising alternatives to oversimplified and homogeneous 2D cell culture models, but existing spatial transcriptomic platforms do not currently have sufficient resolution for robust analysis of 3D cultures. We present a transfection-based method for fluorescent DNA barcoding of cell populations, and the subsequent construction of spheroidal cellular architectures using barcoded cells in a layer-by-layer approach. For the first time, changes in gene expression throughout this 3D culture architecture are interrogated using multiplex single-cell RNA sequencing in which DNA barcodes are used to encode the spatial positioning of cells. We show that transfection with fluorophore-conjugated barcode oligonucleotides enables both imaging and sequencing at single-cell resolution, providing spatial maps of gene expression and drug response. Additionally, we show that fluorophore-conjugated DNA barcodes support correlative imaging studies such as mechano-microscopy to capture information about spatially-varying mechanical heterogeneity in 3D cultures. The ability to create customised, encoded cellular assemblies is a general approach that can resolve spatial differences in gene expression in 3D cell culture models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567793">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567793" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567793">
        <p class="paperTitle">Mouse Genome Editing from Scratch</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567793" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567793" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Popova, J.; Bets, V.; Omelina, E.; Boldyreva, L.; Kozhevnikova, E.</p>
        <p class="info">Score: 2.2, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567793' target='https://doi.org/10.1101/2023.11.20.567793'> 10.1101/2023.11.20.567793</a></p>
        <p class="abstract">Mouse genome modification requires costly equipment and highly skilled personnel to manipulate zygotes. A number of zygote electroporation techniques were reported to be highly efficient in gene delivery. One of these methods called i-GONAD (improved Genome-editing via Oviductal Nucleic Acids Delivery) describes electroporation-based gene transfer to zygotes in utero. Here we adopted this technology for mouse genome-editing from scratch minimizing the cost of equipment, operator skill and animal use. We chose the CRISPR/Cas9 system as a genome editing tool and i-GONAD as a gene delivery method to produce IL10 gene knockout in C57BL/6 mice. Four animals out of 13 delivered pups (30.8%) were genetically compromised at IL10 gene locus suggesting the feasibility of the approach. This report provides one of the possible technical settings for those who aim at establishing in-house mouse transgenesis pipeline at minimal cost from scratch.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
