<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Single cell susceptibility to SARS-CoV-2 infection is driven by variable cell states
Authors: Reffsin, S.; Miller, J.; Ayyanathan, K.; Dunagin, M. C.; Jain, N.; Schultz, D. C.; Cherry, S.; Raj, A.
Score: 23.8, Published: 2023-07-07 DOI: 10.1101/2023.07.06.547955
The ability of a virus to infect a cell type is at least in part determined by the presence of host factors required for the viral life cycle. However, even within cell types that express known factors needed for infection, not every cell is equally susceptible, suggesting that our knowledge of the full spectrum of factors that promote infection is incomplete.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Single cell susceptibility to SARS-CoV-2 infection is driven by variable cell states
Authors: Reffsin, S.; Miller, J.; Ayyanathan, K.; Dunagin, M. C.; Jain, N.; Schultz, D. C.; Cherry, S.; Raj, A.
Score: 23.8, Published: 2023-07-07 DOI: 10.1101/2023.07.06.547955
The ability of a virus to infect a cell type is at least in part determined by the presence of host factors required for the viral life cycle. However, even within cell types that express known factors needed for infection, not every cell is equally susceptible, suggesting that our knowledge of the full spectrum of factors that promote infection is incomplete." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-12T10:41:10+00:00" />
<meta property="article:modified_time" content="2023-07-12T10:41:10+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Single cell susceptibility to SARS-CoV-2 infection is driven by variable cell states
Authors: Reffsin, S.; Miller, J.; Ayyanathan, K.; Dunagin, M. C.; Jain, N.; Schultz, D. C.; Cherry, S.; Raj, A.
Score: 23.8, Published: 2023-07-07 DOI: 10.1101/2023.07.06.547955
The ability of a virus to infect a cell type is at least in part determined by the presence of host factors required for the viral life cycle. However, even within cell types that express known factors needed for infection, not every cell is equally susceptible, suggesting that our knowledge of the full spectrum of factors that promote infection is incomplete."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Single cell susceptibility to SARS-CoV-2 infection is driven by variable cell states\nAuthors: Reffsin, S.; Miller, J.; Ayyanathan, K.; Dunagin, M. C.; Jain, N.; Schultz, D. C.; Cherry, S.; Raj, A.\nScore: 23.8, Published: 2023-07-07 DOI: 10.1101/2023.07.06.547955\nThe ability of a virus to infect a cell type is at least in part determined by the presence of host factors required for the viral life cycle. However, even within cell types that express known factors needed for infection, not every cell is equally susceptible, suggesting that our knowledge of the full spectrum of factors that promote infection is incomplete.",
  "keywords": [
    
  ],
  "articleBody": " Single cell susceptibility to SARS-CoV-2 infection is driven by variable cell states\nAuthors: Reffsin, S.; Miller, J.; Ayyanathan, K.; Dunagin, M. C.; Jain, N.; Schultz, D. C.; Cherry, S.; Raj, A.\nScore: 23.8, Published: 2023-07-07 DOI: 10.1101/2023.07.06.547955\nThe ability of a virus to infect a cell type is at least in part determined by the presence of host factors required for the viral life cycle. However, even within cell types that express known factors needed for infection, not every cell is equally susceptible, suggesting that our knowledge of the full spectrum of factors that promote infection is incomplete. Profiling the most susceptible subsets of cells within a population may reveal additional factors that promote infection. However, because viral infection dramatically alters the state of the cell, new approaches are needed to reveal the state of these cells prior to infection with virus. Here, we used single-cell clone tracing to retrospectively identify and characterize lung epithelial cells that are highly susceptible to infection with SARS-CoV-2. The transcriptional state of these highly susceptible cells includes markers of retinoic acid signaling and epithelial differentiation. Loss of candidate factors identified by our approach revealed that many of these factors play roles in viral entry. Moreover, a subset of these factors exert control over the infectable cell state itself, regulating the expression of key factors associated with viral infection and entry. Analysis of patient samples revealed the heterogeneous expression of these factors across both cells and patients in vivo. Further, the expression of these factors is upregulated in particular inflammatory pathologies. Altogether, our results show that the variable expression of intrinsic cell states is a major determinant of whether a cell can be infected by SARS-CoV-2.\nSEM2: A computational framework to model multiscale mechanics with subcellular elements\nAuthors: Chattaraj, S.; Torre, M.; Kalcher, C.; stukowski, A.; Morganti, S.; Reali, A.; Pasqualini, F.\nScore: 14.8, Published: 2023-07-08 DOI: 10.1101/2023.07.07.548118\nModeling multiscale mechanics in shape-shifting biological tissues in embryos, traditional, or engineered cell culture platforms (organoids, organs-on-chips) is both important and challenging. In fact, it is difficult to model relevant tissue-level structural changes mediated by discrete events at the cellular and subcellular levels, such as migration and proliferation. To accomplish this, we leveraged the subcellular element modeling (SEM) method, where ensembles of coarse-grained particles interacting via empirically defined potentials are used to model individual cells while preserving cell rheology. However, an explicit treatment of multiscale mechanics in SEM was missing. Here, we introduced SEM2, an extended version of the open-source software SEM++ and LAMMPS, enabling new analyses and visualization of particle-level stress and strain. We demonstrated various functionalities of SEM2 by simulating cell creep, migration, and proliferation in scenarios that recapitulate classical and engineered cell culture platforms. For every scenario, we highlight key mechanobiology that emerges spontaneously from particle interactions and discuss recent experimental evidence as qualitative validations of our simulations. The SEM2 code is available on GitHub at https://github.com/Synthetic-Physiology-Lab/sem2 .\nOptimizing CRE and PhiC31 mediated recombination in Aedes aegypti\nAuthors: Carabajal Paladino, L. Z.; Wilson, R.; Tng, P. Y. L.; Dhokiya, V.; Keen, E.; Cuber, P.; Larner, W.; Rooney, S.; Nicholls, M.; Uglow, A.; Williams, L.; Anderson, M. A. E.; Basu, S.; Leftwich, P. T.; Alphey, L.\nScore: 11.0, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548128\nGenetic manipulation of Aedes aegypti is key to developing a deeper understanding of this insects biology, vector-virus interactions and makes future genetic control strategies possible. Despite some advances, this process remains laborious and requires highly skilled researchers and specialist equipment. Here we present two improved methods for genetic manipulation in this species. Use of transgenic lines which express Cre recombinase allowed, by simple crossing schemes, germline or somatic recombination of transgenes, which could be utilized for numerous genetic manipulations. PhiC31 integrase based methods for site-specific integration of genetic elements was also improved, by developing a plasmid which expresses PhiC31 when injected into early embryos, eliminating the need to use costly and unstable mRNA as is the current standard.\nConfigurational entropy is an intrinsic driver of tissue structural heterogeneity\nAuthors: Srivastava, V.; Hu, J. L.; Garbe, J. C.; Veytsman, B.; Shalabi, S. F.; Yllanes, D.; Thomson, M.; LaBarge, M. A.; Huber, G.; Gartner, Z. J.\nScore: 16.9, Published: 2023-07-02 DOI: 10.1101/2023.07.01.546933\nTissues comprise ordered arrangements of cells that can be surprisingly disordered in their details. How the properties of single cells and their microenvironment contribute to the balance between order and disorder at the tissue-scale remains poorly understood. Here, we address this question using the self-organization of human mammary organoids as a model. We find that organoids behave like a dynamic structural ensemble at the steady state. We apply a maximum entropy formalism to derive the ensemble distribution from three measurable parameters - the degeneracy of structural states, interfacial energy, and tissue activity (the energy associated with positional fluctuations). We link these parameters with the molecular and microenvironmental factors that control them to precisely engineer the ensemble across multiple conditions. Our analysis reveals that the entropy associated with structural degeneracy sets a theoretical limit to tissue order and provides new insight for tissue engineering, development, and our understanding of disease progression.\nDesign, construction and optimization of formaldehyde growth biosensors with broad application in Biotechnology\nAuthors: Schann, K.; Bakker, J.; Boinot, M.; Kuschel, P.; He, H.; Nattermann, M.; Erb, T.; Bar-Even, A.; Wenk, S.\nScore: 12.3, Published: 2023-07-01 DOI: 10.1101/2023.06.29.547045\nFormaldehyde is a key metabolite in natural and synthetic one-carbon metabolism as well as an important environmental toxin with high toxicity at low concentrations. To engineer efficient formaldehyde producing enzymes and to detect formaldehyde in industrial or environmental samples, it is important to establish highly sensitive, easy to use and affordable formaldehyde detection methods. Here, we transformed the workhorse bacterium Escherichia coli into biosensors that can detect a broad range of formaldehyde concentrations. Based on natural and promiscuous formaldehyde assimilation enzymes, we designed and engineered three different E. coli strains that depend on formaldehyde assimilation for cellular growth. After in depth characterization of these biosensors, we show that the formaldehyde sensitivity can be improved through adaptive laboratory evolution or modification of metabolic branch points. The metabolic engineering strategy presented in this work allowed the creation of E. coli biosensors that can detect formaldehyde in a concentration range from [~]30 M to [~]13 mM. Using the most sensitive strain, we benchmarked the in vivo activities of different, widely used NAD-dependent methanol dehydrogenases, the rate-limiting enzyme in synthetic methylotrophy. We also show that the strains can grow upon external addition of formaldehyde indicating their potential use for applications beyond enzyme engineering. The formaldehyde biosensors developed in this study are fully genomic and can be used as plug and play devices for screening large enzyme libraries. Thus, they have the potential to greatly advance enzyme engineering and might even be used for environmental monitoring or analysis of industrial probes. Highlights- Conversion of E. coli into three different formaldehyde growth biosensors - Biosensors are fully genomic and grow robustly when formaldehyde is present - Biosensors can detect formaldehyde concentrations ranging from [~]30 M to [~]13 mM - Benchmarking of biotechnological relevant methanol dehydrogenases reveals potential of biosensors for enzyme engineering - Biosensors grow upon direct addition of formaldehyde indicating potential use in environmental or industrial settings\nThe Use of Fluid-phase 3D Printing to Pattern Alginate-gelatin Hydrogel Properties to Guide Cell Growth and Behaviour In Vitro\nAuthors: Souza, A.; McCarthy, K.; Rodriguez, B. J.; REYNAUD, E. G.\nScore: 3.2, Published: 2023-07-08 DOI: 10.1101/2023.07.08.547691\n3D (bio)printing technology has boosted the advancement of the biomedical field. However, tissue engineering is in its infancy and (bio)printing biomimetic constructions for tissue formation in vitro is still a default. As a new methodology to improve in vitro studies, we suggest the use of a cross-linkable aqueous support bath to pattern the characteristics of the scaffolds during the 3D printing process. Using fluid-phase, different molecules can be added to specific locations of the substrate promoting cell behaviour guidance and compartmentalization. Moreover, mechanical aspects can be customized by changing the type or concentration of the solution in which the (bio)printing is acquired. In this study, we first assessed different formulations of alginate/gelatin to improve cell colonization in our printings. On formulations with lower gelatin content, the U2OS cells increased 2.83 times the cell growth. In addition, the alginate-gelatin hydrogel presented a good printability in both air and fluid-phase, however the fluid-phase printings showed better printing fidelity as it diminished the collapsing and the spreading of the hydrogel strand. Next, the fluid-phase methodology was used to guide cell colonization in our printings. First, different stiffness were created by crosslinking the hydrogel with different concentrations of CaCl2 during the printing process. As a result, the U2OS cells were compartmentalized on the stiffer parts of the printings. In addition, using fluid-phase to add RGD molecules to specific parts of the hydrogel has also promoted guidance on cell growth. Finally, our results showed that by combining stiffer alginate-gelatin hydrogel with RGD increasing concentrations we can create a synergetic effect and boost cell growth by up to 3.17-fold. This work presents a new printing process for tailoring multiple parameters in hydrogel substrates by using fluid-phase to generate a more faithful replication of the in vivo environment.\nMosaic-PICASSO: accurate crosstalk removal for multiplex fluorescence imaging\nAuthors: Cang, H.; Liu, Y.; Xing, J.\nScore: 2.5, Published: 2023-07-11 DOI: 10.1101/2023.07.06.547878\nUltra-multiplexed fluorescence imaging has revolutionized our understanding of biological systems, enabling the simultaneous visualization and quantification of multiple targets within biological specimens. A recent breakthrough in this field is PICASSO, a mutual-information-based technique capable of demixing up to 15 fluorophores without their spectra, thereby significantly simplifying the application of ultra-multiplexed fluorescence imaging. However, this study has identified a limitation of mutual information-based techniques. They do not differentiate between spatial colocalization and spectral mixing. Consequently, mutual information-based demixing may incorrectly interpret spatially co- localized targets as non-colocalized, leading to overcorrection. We found that selecting regions within a multiplex image with low spatial similarity for measuring spectroscopic mixing results in more accurate demixing. This method effectively minimizes overcorrections and promises to accelerate the broader adoption of ultra-multiplex imaging.\n3D bioprinting directly affects proteomic signature and myogenic maturation in muscle pericytes-derived human myo-substitute.\nAuthors: Reggio, A.; Fuoco, C.; De Paolis, F.; Testa, S.; Celikkin, N.; Bernardini, S.; Baldi, J.; Biagini, R.; Seliktar, D.; Cirillo, C.; Grumati, P.; Cannata, S.; Costantini, M.; Gargioli, C.\nScore: 2.4, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548389\nSkeletal muscle tissue engineering (SMTE) has recently emerged to address major clinical challenges such as volumetric muscle loss. Here, we report a rotary wet-spinning (RoWS) biofabrication technique for producing human myo-substitutes with biomimetic architectures and functions. We show how the proposed technique may be used to establish a well-tailored, anisotropic microenvironment that promotes exceptional myogenic differentiation of human skeletal muscle-derived pericytes (hPeri). Using high-resolution mass spectrometry-based proteomics with the integration of literature-derived signaling networks, we uncovered that i) 3D biomimetic matrix environment (PEG-Fibrinogen) confers a lower mitogenicity microenvironment compared to standard 2D cultures, favoring the formation of contractile-competent bundles of pericytes-derived myotubes in an anchoring-independent 3D state, and ii) the bioprinting method promotes an upregulation of muscle matrix structural protein besides increasing contractile machinery proteins with respect to 3D bulk cultures. Finally, in vivo investigations demonstrate that the 3D bioprinted myo-substitute is fully compatible with the host ablated muscular tissue, exhibiting myo-substitute engraftment and muscle regeneration in a mouse VML model. Overall, the results show that 3D bioprinting has a superior capability for controlling the myogenic differentiation process on a macroscale and, with future refining, may have the potential to be translated into clinical practice.\nPotency of CRISPR-Cas Antifungals Is Enhanced by Co-targeting DNA Repair and Growth Regulatory Machinery at the Genetic Level\nAuthors: Mendoza, B. J.; Zheng, X.; Clements, J. C.; Cotter, C.; Trinh, C.\nScore: 2.2, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548496\nFungal pathogens are virulent and resistant to antibiotic treatment. Due to their adaptability, specificity, and efficiency, CRISPR-Cas systems can be harnessed for neutralizing these fungal pathogens. However, the conventional design of CRISPR-Cas antimicrobials, based on induction of DNA double-strand-breaks (DSBs), is potentially less effective in fungi due to robust eukaryotic DNA repair machinery. Here, we report a novel design principle to formulate more effective CRISPR-Cas antifungals by co-targeting essential genes with DNA repair defensive genes that remove the fungis ability to repair the DSB sites of essential genes. By evaluating this design on the model fungus Saccharomyces cerevisiae, we demonstrated that essential and defensive gene co-targeting is more effective than either essential or defensive gene targeting alone. The top-performing CRISPR-Cas antifungals performed as effectively as the antibiotic Geneticin. Fast growth kinetics of S. cerevisiae induced resistance to CRISPR-Cas antifungals where genetic mutations mostly occurred in defensive genes and guide RNA sequences.\nUse of nanomaterials-modified carbon microfiber electrode material for superior electrochemical performance in lake sediment inoculated microbial fuel cells\nAuthors: Somasiri, M.; Amandani, T.; Basnayake, C.; Ahsan, A.; Herath, A. C.; Dilangani, G. P.; Bandara, S.; Nazeer, Z.; Thilini, N.; Kyazze, G.; Fernando, E. Y.\nScore: 2.2, Published: 2023-07-10 DOI: 10.1101/2023.07.10.548441\nThe use of novel and inexpensive catalysts as replacements for platinum is desirable. In this study, nanomaterials and the conductive polymer polyaniline (PANI) were directly chemically synthesized on carbon microfiber electrodes to improve the performance of lake sediment inoculated MFCs. Nanomaterial of MnO2, MnO2/polyaniline (PANI), ZnO/NiO and ZnO/NiO/PANI attachments were directly chemically synthesized on the carbon material and used as cathode electrodes. The maximum power densities recorded for the different treatments were; MnO2 78.5 mW/m2, MnO2/PANI (Polyaniline) 141.6 mW/m2, ZnO/NiO 67.6 mW/m2, and ZnO/NiO/PANI 129.4 mW/m2. Current and power densities were more than six-fold higher in ZnO/NiO/PANI and MnO2/PANI nanoparticle modified cathodes compared to the control MFCs with no catalyst more than 2.5 fold higher compared to Pt loaded conventional cathodes. This study demonstrates that nanomaterials-incorporated carbon microfiber cathodes bring about significant enhancements to power densities and may potentially have applications in cost-effective MFCs.\n",
  "wordCount" : "2298",
  "inLanguage": "en",
  "datePublished": "2023-07-12T10:41:10Z",
  "dateModified": "2023-07-12T10:41:10Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on July 12, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.547955">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.547955" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.547955">
        <p class="paperTitle">Single cell susceptibility to SARS-CoV-2 infection is driven by variable cell states</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.547955" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.547955" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reffsin, S.; Miller, J.; Ayyanathan, K.; Dunagin, M. C.; Jain, N.; Schultz, D. C.; Cherry, S.; Raj, A.</p>
        <p class="info">Score: 23.8, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.547955' target='https://doi.org/10.1101/2023.07.06.547955'> 10.1101/2023.07.06.547955</a></p>
        <p class="abstract">The ability of a virus to infect a cell type is at least in part determined by the presence of host factors required for the viral life cycle. However, even within cell types that express known factors needed for infection, not every cell is equally susceptible, suggesting that our knowledge of the full spectrum of factors that promote infection is incomplete. Profiling the most susceptible subsets of cells within a population may reveal additional factors that promote infection. However, because viral infection dramatically alters the state of the cell, new approaches are needed to reveal the state of these cells prior to infection with virus. Here, we used single-cell clone tracing to retrospectively identify and characterize lung epithelial cells that are highly susceptible to infection with SARS-CoV-2. The transcriptional state of these highly susceptible cells includes markers of retinoic acid signaling and epithelial differentiation. Loss of candidate factors identified by our approach revealed that many of these factors play roles in viral entry. Moreover, a subset of these factors exert control over the infectable cell state itself, regulating the expression of key factors associated with viral infection and entry. Analysis of patient samples revealed the heterogeneous expression of these factors across both cells and patients in vivo. Further, the expression of these factors is upregulated in particular inflammatory pathologies. Altogether, our results show that the variable expression of intrinsic cell states is a major determinant of whether a cell can be infected by SARS-CoV-2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548118">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548118" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548118">
        <p class="paperTitle">SEM2: A computational framework to model multiscale mechanics with subcellular elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548118" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548118" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chattaraj, S.; Torre, M.; Kalcher, C.; stukowski, A.; Morganti, S.; Reali, A.; Pasqualini, F.</p>
        <p class="info">Score: 14.8, Published: 2023-07-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548118' target='https://doi.org/10.1101/2023.07.07.548118'> 10.1101/2023.07.07.548118</a></p>
        <p class="abstract">Modeling multiscale mechanics in shape-shifting biological tissues in embryos, traditional, or engineered cell culture platforms (organoids, organs-on-chips) is both important and challenging. In fact, it is difficult to model relevant tissue-level structural changes mediated by discrete events at the cellular and subcellular levels, such as migration and proliferation. To accomplish this, we leveraged the subcellular element modeling (SEM) method, where ensembles of coarse-grained particles interacting via empirically defined potentials are used to model individual cells while preserving cell rheology. However, an explicit treatment of multiscale mechanics in SEM was missing. Here, we introduced SEM2, an extended version of the open-source software SEM&#43;&#43; and LAMMPS, enabling new analyses and visualization of particle-level stress and strain. We demonstrated various functionalities of SEM2 by simulating cell creep, migration, and proliferation in scenarios that recapitulate classical and engineered cell culture platforms. For every scenario, we highlight key mechanobiology that emerges spontaneously from particle interactions and discuss recent experimental evidence as qualitative validations of our simulations. The SEM2 code is available on GitHub at https://github.com/Synthetic-Physiology-Lab/sem2 .</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548128">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548128" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548128">
        <p class="paperTitle">Optimizing CRE and PhiC31 mediated recombination in Aedes aegypti</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548128" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548128" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carabajal Paladino, L. Z.; Wilson, R.; Tng, P. Y. L.; Dhokiya, V.; Keen, E.; Cuber, P.; Larner, W.; Rooney, S.; Nicholls, M.; Uglow, A.; Williams, L.; Anderson, M. A. E.; Basu, S.; Leftwich, P. T.; Alphey, L.</p>
        <p class="info">Score: 11.0, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548128' target='https://doi.org/10.1101/2023.07.07.548128'> 10.1101/2023.07.07.548128</a></p>
        <p class="abstract">Genetic manipulation of Aedes aegypti is key to developing a deeper understanding of this insects biology, vector-virus interactions and makes future genetic control strategies possible. Despite some advances, this process remains laborious and requires highly skilled researchers and specialist equipment. Here we present two improved methods for genetic manipulation in this species. Use of transgenic lines which express Cre recombinase allowed, by simple crossing schemes, germline or somatic recombination of transgenes, which could be utilized for numerous genetic manipulations. PhiC31 integrase based methods for site-specific integration of genetic elements was also improved, by developing a plasmid which expresses PhiC31 when injected into early embryos, eliminating the need to use costly and unstable mRNA as is the current standard.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.01.546933">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.01.546933" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.01.546933">
        <p class="paperTitle">Configurational entropy is an intrinsic driver of tissue structural heterogeneity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.01.546933" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.01.546933" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Srivastava, V.; Hu, J. L.; Garbe, J. C.; Veytsman, B.; Shalabi, S. F.; Yllanes, D.; Thomson, M.; LaBarge, M. A.; Huber, G.; Gartner, Z. J.</p>
        <p class="info">Score: 16.9, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.01.546933' target='https://doi.org/10.1101/2023.07.01.546933'> 10.1101/2023.07.01.546933</a></p>
        <p class="abstract">Tissues comprise ordered arrangements of cells that can be surprisingly disordered in their details. How the properties of single cells and their microenvironment contribute to the balance between order and disorder at the tissue-scale remains poorly understood. Here, we address this question using the self-organization of human mammary organoids as a model. We find that organoids behave like a dynamic structural ensemble at the steady state. We apply a maximum entropy formalism to derive the ensemble distribution from three measurable parameters - the degeneracy of structural states, interfacial energy, and tissue activity (the energy associated with positional fluctuations). We link these parameters with the molecular and microenvironmental factors that control them to precisely engineer the ensemble across multiple conditions. Our analysis reveals that the entropy associated with structural degeneracy sets a theoretical limit to tissue order and provides new insight for tissue engineering, development, and our understanding of disease progression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547045">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547045" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547045">
        <p class="paperTitle">Design, construction and optimization of formaldehyde growth biosensors with broad application in Biotechnology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547045" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547045" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schann, K.; Bakker, J.; Boinot, M.; Kuschel, P.; He, H.; Nattermann, M.; Erb, T.; Bar-Even, A.; Wenk, S.</p>
        <p class="info">Score: 12.3, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547045' target='https://doi.org/10.1101/2023.06.29.547045'> 10.1101/2023.06.29.547045</a></p>
        <p class="abstract">Formaldehyde is a key metabolite in natural and synthetic one-carbon metabolism as well as an important environmental toxin with high toxicity at low concentrations. To engineer efficient formaldehyde producing enzymes and to detect formaldehyde in industrial or environmental samples, it is important to establish highly sensitive, easy to use and affordable formaldehyde detection methods. Here, we transformed the workhorse bacterium Escherichia coli into biosensors that can detect a broad range of formaldehyde concentrations. Based on natural and promiscuous formaldehyde assimilation enzymes, we designed and engineered three different E. coli strains that depend on formaldehyde assimilation for cellular growth. After in depth characterization of these biosensors, we show that the formaldehyde sensitivity can be improved through adaptive laboratory evolution or modification of metabolic branch points. The metabolic engineering strategy presented in this work allowed the creation of E. coli biosensors that can detect formaldehyde in a concentration range from [~]30 M to [~]13 mM. Using the most sensitive strain, we benchmarked the in vivo activities of different, widely used NAD-dependent methanol dehydrogenases, the rate-limiting enzyme in synthetic methylotrophy. We also show that the strains can grow upon external addition of formaldehyde indicating their potential use for applications beyond enzyme engineering. The formaldehyde biosensors developed in this study are fully genomic and can be used as plug and play devices for screening large enzyme libraries. Thus, they have the potential to greatly advance enzyme engineering and might even be used for environmental monitoring or analysis of industrial probes.

Highlights- Conversion of E. coli into three different formaldehyde growth biosensors
- Biosensors are fully genomic and grow robustly when formaldehyde is present
- Biosensors can detect formaldehyde concentrations ranging from [~]30 M to [~]13 mM
- Benchmarking of biotechnological relevant methanol dehydrogenases reveals potential of biosensors for enzyme engineering
- Biosensors grow upon direct addition of formaldehyde indicating potential use in environmental or industrial settings</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.08.547691">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.08.547691" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.08.547691">
        <p class="paperTitle">The Use of Fluid-phase 3D Printing to Pattern Alginate-gelatin Hydrogel Properties to Guide Cell Growth and Behaviour In Vitro</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.08.547691" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.08.547691" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Souza, A.; McCarthy, K.; Rodriguez, B. J.; REYNAUD, E. G.</p>
        <p class="info">Score: 3.2, Published: 2023-07-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.08.547691' target='https://doi.org/10.1101/2023.07.08.547691'> 10.1101/2023.07.08.547691</a></p>
        <p class="abstract">3D (bio)printing technology has boosted the advancement of the biomedical field. However, tissue engineering is in its infancy and (bio)printing biomimetic constructions for tissue formation in vitro is still a default. As a new methodology to improve in vitro studies, we suggest the use of a cross-linkable aqueous support bath to pattern the characteristics of the scaffolds during the 3D printing process. Using fluid-phase, different molecules can be added to specific locations of the substrate promoting cell behaviour guidance and compartmentalization. Moreover, mechanical aspects can be customized by changing the type or concentration of the solution in which the (bio)printing is acquired. In this study, we first assessed different formulations of alginate/gelatin to improve cell colonization in our printings. On formulations with lower gelatin content, the U2OS cells increased 2.83 times the cell growth. In addition, the alginate-gelatin hydrogel presented a good printability in both air and fluid-phase, however the fluid-phase printings showed better printing fidelity as it diminished the collapsing and the spreading of the hydrogel strand. Next, the fluid-phase methodology was used to guide cell colonization in our printings. First, different stiffness were created by crosslinking the hydrogel with different concentrations of CaCl2 during the printing process. As a result, the U2OS cells were compartmentalized on the stiffer parts of the printings. In addition, using fluid-phase to add RGD molecules to specific parts of the hydrogel has also promoted guidance on cell growth. Finally, our results showed that by combining stiffer alginate-gelatin hydrogel with RGD increasing concentrations we can create a synergetic effect and boost cell growth by up to 3.17-fold. This work presents a new printing process for tailoring multiple parameters in hydrogel substrates by using fluid-phase to generate a more faithful replication of the in vivo environment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.547878">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.547878" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.547878">
        <p class="paperTitle">Mosaic-PICASSO: accurate crosstalk removal for multiplex fluorescence imaging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.547878" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.547878" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cang, H.; Liu, Y.; Xing, J.</p>
        <p class="info">Score: 2.5, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.547878' target='https://doi.org/10.1101/2023.07.06.547878'> 10.1101/2023.07.06.547878</a></p>
        <p class="abstract">Ultra-multiplexed fluorescence imaging has revolutionized our understanding of biological systems, enabling the simultaneous visualization and quantification of multiple targets within biological specimens. A recent breakthrough in this field is PICASSO, a mutual-information-based technique capable of demixing up to 15 fluorophores without their spectra, thereby significantly simplifying the application of ultra-multiplexed fluorescence imaging. However, this study has identified a limitation of mutual information-based techniques. They do not differentiate between spatial colocalization and spectral mixing. Consequently, mutual information-based demixing may incorrectly interpret spatially co- localized targets as non-colocalized, leading to overcorrection. We found that selecting regions within a multiplex image with low spatial similarity for measuring spectroscopic mixing results in more accurate demixing. This method effectively minimizes overcorrections and promises to accelerate the broader adoption of ultra-multiplex imaging.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548389">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548389" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548389">
        <p class="paperTitle">3D bioprinting directly affects proteomic signature and myogenic maturation in muscle pericytes-derived human myo-substitute.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548389" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548389" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reggio, A.; Fuoco, C.; De Paolis, F.; Testa, S.; Celikkin, N.; Bernardini, S.; Baldi, J.; Biagini, R.; Seliktar, D.; Cirillo, C.; Grumati, P.; Cannata, S.; Costantini, M.; Gargioli, C.</p>
        <p class="info">Score: 2.4, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548389' target='https://doi.org/10.1101/2023.07.10.548389'> 10.1101/2023.07.10.548389</a></p>
        <p class="abstract">Skeletal muscle tissue engineering (SMTE) has recently emerged to address major clinical challenges such as volumetric muscle loss. Here, we report a rotary wet-spinning (RoWS) biofabrication technique for producing human myo-substitutes with biomimetic architectures and functions. We show how the proposed technique may be used to establish a well-tailored, anisotropic microenvironment that promotes exceptional myogenic differentiation of human skeletal muscle-derived pericytes (hPeri). Using high-resolution mass spectrometry-based proteomics with the integration of literature-derived signaling networks, we uncovered that i) 3D biomimetic matrix environment (PEG-Fibrinogen) confers a lower mitogenicity microenvironment compared to standard 2D cultures, favoring the formation of contractile-competent bundles of pericytes-derived myotubes in an anchoring-independent 3D state, and ii) the bioprinting method promotes an upregulation of muscle matrix structural protein besides increasing contractile machinery proteins with respect to 3D bulk cultures. Finally, in vivo investigations demonstrate that the 3D bioprinted myo-substitute is fully compatible with the host ablated muscular tissue, exhibiting myo-substitute engraftment and muscle regeneration in a mouse VML model. Overall, the results show that 3D bioprinting has a superior capability for controlling the myogenic differentiation process on a macroscale and, with future refining, may have the potential to be translated into clinical practice.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548496">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548496" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548496">
        <p class="paperTitle">Potency of CRISPR-Cas Antifungals Is Enhanced by Co-targeting DNA Repair and Growth Regulatory Machinery at the Genetic Level</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548496" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548496" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mendoza, B. J.; Zheng, X.; Clements, J. C.; Cotter, C.; Trinh, C.</p>
        <p class="info">Score: 2.2, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548496' target='https://doi.org/10.1101/2023.07.11.548496'> 10.1101/2023.07.11.548496</a></p>
        <p class="abstract">Fungal pathogens are virulent and resistant to antibiotic treatment. Due to their adaptability, specificity, and efficiency, CRISPR-Cas systems can be harnessed for neutralizing these fungal pathogens. However, the conventional design of CRISPR-Cas antimicrobials, based on induction of DNA double-strand-breaks (DSBs), is potentially less effective in fungi due to robust eukaryotic DNA repair machinery. Here, we report a novel design principle to formulate more effective CRISPR-Cas antifungals by co-targeting essential genes with DNA repair defensive genes that remove the fungis ability to repair the DSB sites of essential genes. By evaluating this design on the model fungus Saccharomyces cerevisiae, we demonstrated that essential and defensive gene co-targeting is more effective than either essential or defensive gene targeting alone. The top-performing CRISPR-Cas antifungals performed as effectively as the antibiotic Geneticin. Fast growth kinetics of S. cerevisiae induced resistance to CRISPR-Cas antifungals where genetic mutations mostly occurred in defensive genes and guide RNA sequences.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548441">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548441" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548441">
        <p class="paperTitle">Use of nanomaterials-modified carbon microfiber electrode material for superior electrochemical performance in lake sediment inoculated microbial fuel cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548441" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548441" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Somasiri, M.; Amandani, T.; Basnayake, C.; Ahsan, A.; Herath, A. C.; Dilangani, G. P.; Bandara, S.; Nazeer, Z.; Thilini, N.; Kyazze, G.; Fernando, E. Y.</p>
        <p class="info">Score: 2.2, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548441' target='https://doi.org/10.1101/2023.07.10.548441'> 10.1101/2023.07.10.548441</a></p>
        <p class="abstract">The use of novel and inexpensive catalysts as replacements for platinum is desirable. In this study, nanomaterials and the conductive polymer polyaniline (PANI) were directly chemically synthesized on carbon microfiber electrodes to improve the performance of lake sediment inoculated MFCs. Nanomaterial of MnO2, MnO2/polyaniline (PANI), ZnO/NiO and ZnO/NiO/PANI attachments were directly chemically synthesized on the carbon material and used as cathode electrodes. The maximum power densities recorded for the different treatments were; MnO2 78.5 mW/m2, MnO2/PANI (Polyaniline) 141.6 mW/m2, ZnO/NiO 67.6 mW/m2, and ZnO/NiO/PANI 129.4 mW/m2. Current and power densities were more than six-fold higher in ZnO/NiO/PANI and MnO2/PANI nanoparticle modified cathodes compared to the control MFCs with no catalyst more than 2.5 fold higher compared to Pt loaded conventional cathodes. This study demonstrates that nanomaterials-incorporated carbon microfiber cathodes bring about significant enhancements to power densities and may potentially have applications in cost-effective MFCs.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
