<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A general platform for targeting MHC-II antigens via a single loop
Authors: Du, H.; Liu, J.; Jude, K. M.; Yang, X.; Li, Y.; Bell, B.; Yang, H.; Kassardjian, A.; Mobedi, A.; Parekh, U.; Sperberg, A.; Julien, J.-P.; Mellins, E.; Garcia, K. C.; Huang, P.
Score: 39.3, Published: 2024-01-30 DOI: 10.1101/2024.01.26.577489
Class-II major histocompatibility complexes (MHC-IIs) are central to the communications between CD4&#43; T cells and antigen presenting cells (APCs), but intrinsic structural features associated with MHC-II make it difficult to develop a general targeting system with high affinity and antigen specificity.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="A general platform for targeting MHC-II antigens via a single loop
Authors: Du, H.; Liu, J.; Jude, K. M.; Yang, X.; Li, Y.; Bell, B.; Yang, H.; Kassardjian, A.; Mobedi, A.; Parekh, U.; Sperberg, A.; Julien, J.-P.; Mellins, E.; Garcia, K. C.; Huang, P.
Score: 39.3, Published: 2024-01-30 DOI: 10.1101/2024.01.26.577489
Class-II major histocompatibility complexes (MHC-IIs) are central to the communications between CD4&#43; T cells and antigen presenting cells (APCs), but intrinsic structural features associated with MHC-II make it difficult to develop a general targeting system with high affinity and antigen specificity." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-07T10:40:15+00:00" />
<meta property="article:modified_time" content="2024-02-07T10:40:15+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="A general platform for targeting MHC-II antigens via a single loop
Authors: Du, H.; Liu, J.; Jude, K. M.; Yang, X.; Li, Y.; Bell, B.; Yang, H.; Kassardjian, A.; Mobedi, A.; Parekh, U.; Sperberg, A.; Julien, J.-P.; Mellins, E.; Garcia, K. C.; Huang, P.
Score: 39.3, Published: 2024-01-30 DOI: 10.1101/2024.01.26.577489
Class-II major histocompatibility complexes (MHC-IIs) are central to the communications between CD4&#43; T cells and antigen presenting cells (APCs), but intrinsic structural features associated with MHC-II make it difficult to develop a general targeting system with high affinity and antigen specificity."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "A general platform for targeting MHC-II antigens via a single loop\nAuthors: Du, H.; Liu, J.; Jude, K. M.; Yang, X.; Li, Y.; Bell, B.; Yang, H.; Kassardjian, A.; Mobedi, A.; Parekh, U.; Sperberg, A.; Julien, J.-P.; Mellins, E.; Garcia, K. C.; Huang, P.\nScore: 39.3, Published: 2024-01-30 DOI: 10.1101/2024.01.26.577489\nClass-II major histocompatibility complexes (MHC-IIs) are central to the communications between CD4\u0026#43; T cells and antigen presenting cells (APCs), but intrinsic structural features associated with MHC-II make it difficult to develop a general targeting system with high affinity and antigen specificity.",
  "keywords": [
    
  ],
  "articleBody": " A general platform for targeting MHC-II antigens via a single loop\nAuthors: Du, H.; Liu, J.; Jude, K. M.; Yang, X.; Li, Y.; Bell, B.; Yang, H.; Kassardjian, A.; Mobedi, A.; Parekh, U.; Sperberg, A.; Julien, J.-P.; Mellins, E.; Garcia, K. C.; Huang, P.\nScore: 39.3, Published: 2024-01-30 DOI: 10.1101/2024.01.26.577489\nClass-II major histocompatibility complexes (MHC-IIs) are central to the communications between CD4+ T cells and antigen presenting cells (APCs), but intrinsic structural features associated with MHC-II make it difficult to develop a general targeting system with high affinity and antigen specificity. Here, we introduce a protein platform, Targeted Recognition of Antigen-MHC Complex Reporter for MHC-II (TRACeR-II), to enable the rapid development of peptide-specific MHC-II binders. TRACeR-II has a small helical bundle scaffold and uses an unconventional mechanism to recognize antigens via a single loop. This unique antigen-recognition mechanism renders this platform highly versatile and amenable to direct structural modeling of the interactions with the antigen. We demonstrate that TRACeR-II binders can be rapidly evolved across multiple alleles, while computational protein design can produce specific binding sequences for a SARS-CoV-2 peptide of unknown complex structure. TRACeR-II sheds light on a simple and straightforward approach to address the MHC peptide targeting challenge, without relying on combinatorial selection on complementarity determining region (CDR) loops. It presents a promising basis for further exploration in immune response modulation as well as a broad range of theragnostic applications.\nGrayscale 4D Biomaterial Customization at High Resolution and Scale\nAuthors: Batalov, I.; Filteau, J. R.; Francis, R. M.; Jaindl, G.; Orr, L.; Rapp, T. L.; Yang, S.; Filteau, J. A.; Xie, W.; Bretherton, R. C.; Glaser, A. K.; Liu, J. T. C.; Stevens, K. R.; DeForest, C. A.\nScore: 32.7, Published: 2024-02-03 DOI: 10.1101/2024.01.31.578280\nHydrogel biomaterials have proven indispensable for three-dimensional (3D) cell culture but have fallen short in replicating the innate physiochemical complexity of real tissue. Though traditional photolithography affords localized material manipulation, robust methods that govern when, where, and to what extent such phototailoring occurs throughout materials would be profoundly enabling towards fabricating more-realistic 3D tissue constructs. Here, we introduce \"grayscale image z-stack-guided multiphoton optical-lithography\" (GIZMO) as a generalizable and intuitive strategy to rapidly photomodulate materials in full 3D non-binary patterns at submicron resolutions spanning large volumes (\u003emm3). Highlighting its versatility, we employ GIZMO to variably photopattern biomolecule release from, protein immobilization to, and degradation within hydrogels based on biologically derived or synthetic grayscale image stacks with unprecedented complexity. We anticipate that GIZMO will enable new opportunities to probe and manipulate cell fates, as well as to engineer complex functional tissue.\nA combination of two-enzyme system and enzyme engineering improved the activity of a new PET hydrolase identified from soil bacterial genome\nAuthors: Mabashi-Asazuma, H.; Hirai, M.; Sakurai, S.; Ide, K.; Kogawa, M.; Matsushita, A.; Hosokawa, M.; Tsuda, S.\nScore: 8.8, Published: 2024-02-02 DOI: 10.1101/2024.02.01.578500\nWe here report a novel PET hydrolase originating from a soil microbial genome sequence. This enzyme, bbPET0069, exhibits characteristics resembling a cutinase-like Type I PET-degrading enzyme but lacks disulfide bonds. Notably, bbPET0069 displayed remarkable synergy with Candida antarctica lipase B (CALB), demonstrating rapid and efficient PET degradation. To improve the PET degradation activity of bbPET0069, we employed a three-dimensional (3D) structural modeling to identify mutation sites around its substrate binding domain combined with a protein language model for effective mutation prediction. Through three initial rounds of directed evolution, we achieved a significant enhancement in PET degradation with CALB, resulting in a 12.6-fold increase compared to wild-type bbPET0069 without CALB. We confirmed its PET degradation activity in PET nanoparticles and films, and our proposed approach enabled efficient PET degradation to terephthalic acid monomers up to 95.5%. Our approach, which integrates a two-enzyme system with protein engineering, demonstrates the potential for enhancing the activity of emerging PET-degradation enzymes, which may possess unique attributes. Graphical AbstractA novel PET hydrolase, bbPET0069, was identified from a soil microbial genome. bbPET0069 and CALB showed remarkable synergy in PET degradation. Using surface feature analysis, PET degradation activity of bbPET0069 was significantly improved. This combination of a two-enzyme system and surface feature analysis holds promise for enhancing emerging PET-degradation enzymes. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=130 SRC=\"FIGDIR/small/578500v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (27K): org.highwire.dtl.DTLVardef@1a60a45org.highwire.dtl.DTLVardef@1ec776eorg.highwire.dtl.DTLVardef@19500c6org.highwire.dtl.DTLVardef@166068c_HPS_FORMAT_FIGEXP M_FIG C_FIG\nRefining Spatial Proteomics by Mass Spectrometry: An Efficient Workflow Tailored for Archival Tissue\nAuthors: Daucke, R.; Rift, C. V.; Bager, N. S.; Saxena, K.; Koffeldt, P. R.; Woessmann, J.; Petrosius, V.; Rugiu, E. S.; Kristensen, B. W.; Klausen, P.; Schoof, E. M.\nScore: 9.3, Published: 2024-01-29 DOI: 10.1101/2024.01.25.577263\nFormalin-fixed paraffin-embedded (FFPE) tissue, while excellent for preserving tissue for extended periods of time, poses a challenge when extracting molecular information. We therefore developed an easily adaptable and highly efficient workflow for extracting high levels of proteins from low-input material. We compared sensitivity between two stains, EpCAM and H\u0026E, across material inputs of 1,166 and [~]800,000 {micro}m2. In the context of EpCAM-stained tissue, our investigations unveiled a range from [~]1,200 unique protein groups at the lowest input to [~]5,900 at the highest. For H\u0026E, the spectrum covers [~]900 to [~]5,200 protein groups. We found an optimal balance between maximizing detected proteins and minimizing input material to be within the range of [~]50,000 to [~]100,000 {micro}m2. With this knowledge, we tested the spatial capabilities by isolating specific cell populations, through Laser Capture Microdissection (LCM), from three different tissue types, where we were able to identify tissue-specific signatures and prominent clustering of all cell populations.\nMechanically Tunable, Compostable, Healable and Scalable Engineered Living Materials\nAuthors: Manjula-Basavanna, A.; Joshi, N.; Duraj-Thatte, A.\nScore: 6.0, Published: 2024-02-02 DOI: 10.1101/2024.01.31.578234\nNovel design strategies are essential to realize the full potential of Engineered Living Materials (ELMs), including their biodegradability, manufacturability, sustainability, and ability to tailor functional properties. Toward these goals, we present Mechanically Engineered Living Material with Compostability, Healability, and Scalability (MECHS) - a material that integrates these features in the form of a stretchable plastic that is simultaneously flushable, compostable, and exhibits the characteristics of paper. This plastic/paper-like material is produced directly from cultured bacterial biomass (40%) producing engineered curli protein nanofibers in scalable quantities (0.5-1 g L-1). The elongation at break (1-160%) and Youngs modulus (6-450 MPa) of MECHS was tuned to more than two orders of magnitude. By genetically encoded covalent crosslinking of curli nanofibers, we increase the Youngs modulus by two times. MECHS biodegrades completely in 15-75 days, while its mechanical properties are comparable to petrochemical plastics and thus may find use as compostable materials for primary packaging.\nCollagen organization and structure in FLBN5-/- mice using label-free microscopy: implications for pelvic organ prolapse\nAuthors: Jennings, C.; Markel, A.; Domingo, M.; Miller, K.; Bayer, C.; Parekh, S. H.\nScore: 5.5, Published: 2024-02-01 DOI: 10.1101/2024.01.31.578106\nPelvic organ prolapse (POP) is a gynecological disorder described by the descent of superior pelvic organs into or out of the vagina as a consequence of disrupted muscles and tissue. A thorough understanding of the etiology of POP is limited by the availability of clinically relevant samples, restricting longitudinal POP studies on soft-tissue biomechanics and structure to POP-induced models such as fibulin-5 knockout (FBLN5-/-) mice. Despite being a principal constituent in the extracellular matrix, little is known about structural perturbations to collagen networks in the FBLN5-/- mouse cervix. We identify significantly different collagen network populations in normal and prolapsed cervical cross-sections using two label-free, nonlinear microscopy techniques. Collagen in the prolapsed mouse cervix tends to be more isotropic, and displays reduced alignment persistence via 2-D Fourier Transform analysis of images acquired using second harmonic generation microscopy. Furthermore, coherent Raman hyperspectral imaging revealed elevated disorder in the secondary structure of collagen in prolapsed tissues. Our results underscore the need for in situ multimodal monitoring of collagen organization to improve POP predictive capabilities.\nFocused Ultrasound Blood-Brain Barrier Opening Arrests the Growth and Formation of Cerebral Cavernous Malformations\nAuthors: Fisher, D. G.; Sharifi, K. A.; Shah, I. M.; Gorick, C. M.; Breza, V. R.; Debski, A. C.; Hoch, M.; Cruz, T.; Samuels, J. D.; Sheehan, J.; Schlesinger, D.; Moore, D.; Lukens, J. R.; Tvrdik, P.; Miller, G. W.; Price, R. J.\nScore: 4.8, Published: 2024-02-04 DOI: 10.1101/2024.01.31.577810\nBACKGROUNDCerebral cavernous malformations (CCM) are vascular lesions within the central nervous system, consisting of dilated and hemorrhage-prone capillaries. CCMs can cause debilitating neurological symptoms, and surgical excision or stereotactic radiosurgery are the only current treatment options. Meanwhile, transient blood-brain barrier opening (BBBO) with focused ultrasound (FUS) and microbubbles is now understood to exert potentially beneficial bioeffects, such as stimulation of neurogenesis and clearance of amyloid-{beta}. Here, we tested whether FUS BBBO could be deployed therapeutically to control CCM formation and progression in a clinically-representative murine model. METHODSCCMs were induced in mice by postnatal, endothelial-specific Krit1 ablation. FUS was applied for BBBO with fixed peak-negative pressures (PNPs; 0.2-0.6 MPa) or passive cavitation detection-modulated PNPs. Magnetic resonance imaging (MRI) was used to target FUS treatments, evaluate safety, and measure longitudinal changes in CCM growth after BBBO. RESULTSFUS BBBO elicited gadolinium accumulation primarily at the perilesional boundaries of CCMs, rather than lesion cores. Passive cavitation detection and gadolinium contrast enhancement were comparable in CCM and wild-type mice, indicating that Krit1 ablation does not confer differential sensitivity to FUS BBBO. Acutely, CCMs exposed to FUS BBBO remained structurally stable, with no signs of hemorrhage. Longitudinal MRI revealed that FUS BBBO halted the growth of 94% of CCMs treated in the study. At 1 month, FUS BBBO-treated lesions lost, on average, 9% of their pre-sonication volume. In contrast, non-sonicated control lesions grew to 670% of their initial volume. Lesion control with FUS BBBO was accompanied by a marked reduction in the area and mesenchymal appearance of Krit mutant endothelium. Strikingly, in mice receiving multiple BBBO treatments with fixed PNPs, de novo CCM formation was significantly reduced by 81%. Mock treatment plans on MRIs of patients with surgically inaccessible lesions revealed their lesions are amenable to FUS BBBO with current clinical technology. CONCLUSIONSOur results establish FUS BBBO as a novel, non-invasive modality that can safely arrest murine CCM growth and prevent their de novo formation. As an incisionless, MR image-guided therapy with the ability to target eloquent brain locations, FUS BBBO offers an unparalleled potential to revolutionize the therapeutic experience and enhance the accessibility of treatments for CCM patients.\nProlonged airway explant culture enables study of health, disease, and viral pathogenesis\nAuthors: Lee-Ferris, R. E.; Okuda, K.; Galiger, J. R.; Schworer, S. A.; Rogers, T. D.; Dang, H.; Gilmore, R.; Edwards, C.; Nakano, S.; Cawley, A. M.; Pickles, R. J.; Gallant, S. C.; Crisci, E.; Rivier, L.; Hagood, J. S.; O'Neal, W. K.; Baric, R. S.; Grubb, B. R.; Boucher, R. C.; Randell, S. H.\nScore: 4.3, Published: 2024-02-05 DOI: 10.1101/2024.02.03.578756\nIn vitro models play a major role in studying airway physiology and disease. However, the native lungs complex tissue architecture and non-epithelial cell lineages are not preserved in these models. Ex vivo tissue models could overcome in vitro limitations, but methods for long-term maintenance of ex vivo tissue has not been established. We describe methods to culture human large airway explants, small airway explants, and precision-cut lung slices for at least 14 days. Human airway explants recapitulate genotype-specific electrophysiology, characteristic epithelial, endothelial, stromal and immune cell populations, and model viral infection after 14 days in culture. These methods also maintain mouse, rabbit, and pig tracheal explants. Notably, intact airway tissue can be cryopreserved, thawed, and used to generate explants with recovery of function 14 days post-thaw. These studies highlight the broad applications of airway tissue explants and their use as translational intermediates between in vitro and in vivo studies.\nThe T-cell niche tunes immune function through modulation of the cytoskeleton and TCR-antigen forces\nAuthors: Kellner, A. V.; Hunter, R.; Do, P.; Eggert, J.; Jaffe, M.; Geitgey, D. K.; Lee, M.; Hamilton, J. A. G.; Ross, A. J.; Ank, R. S.; Bender, R. L.; Ma, R.; Porter, C. C.; Dreaden, E. C.; Au-Yeung, B. B.; Haynes, K. A.; Henry, C. J.; Salaita, K.\nScore: 3.5, Published: 2024-02-02 DOI: 10.1101/2024.01.31.578101\nObesity is a major public health crisis given its rampant growth and association with an increased risk for cancer. Interestingly, patients with obesity tend to have an increased tumor burden and decreased T-cell function. It remains unclear how obesity compromises T-cell mediated immunity. To address this question, we modeled the adipocyte niche using the secretome released from adipocytes as well as the niche of stromal cells and investigated how these factors modulated T-cell function. We found that the secretomes altered antigen-specific T-cell receptor (TCR) triggering and activation. RNA-sequencing analysis identified thousands of gene targets modulated by the secretome including those associated with cytoskeletal regulation and actin polymerization. We next used molecular force probes to show that T-cells exposed to the adipocyte niche display dampened force transmission to the TCR-antigen complex and conversely, stromal cell secreted factors lead to significantly enhanced TCR forces. These results were then validated in diet-induced obese mice. Importantly, secretome-mediated TCR force modulation mirrored the changes in T-cell functional responses in human T-cells using the FDA-approved immunotherapy, blinatumomab. Thus, this work shows that the adipocyte niche contributes to T-cell dysfunction through cytoskeletal modulation and reduces TCR triggering by dampening TCR forces consistent with the mechanosensor model of T-cell activation.\nBio-orthogonal Glycan Imaging of Culture Cells and Whole Animal C. elegans with Expansion Microscopy\nAuthors: Kuo, J. C.-H.; Colville, M. J.; Sorkin, M. R.; Kuo, J. L. K.; Huang, L.-T.; Thornlow, D. N.; Beacham, G. M.; Hollopeter, G.; DeLisa, M. P.; Alabi, C. A.; Paszek, M. J.\nScore: 3.4, Published: 2024-02-02 DOI: 10.1101/2024.02.01.578333\nComplex carbohydrates called glycans play crucial roles in the regulation of cell and tissue physiology, but how glycans map to nanoscale anatomical features must still be resolved. Here, we present the first nanoscale map of mucin-type O-glycans throughout the entirety of the Caenorhabditis elegans model organism. We construct a library of multifunctional linkers to probe and anchor metabolically labelled glycans in expansion microscopy (ExM), an imaging modality that overcomes the diffraction limit of conventional optical microscopes through the physical expansion of samples embedded in a polyelectrolyte gel matrix. A flexible strategy is demonstrated for the chemical synthesis of linkers with a broad inventory of bio-orthogonal functional groups, fluorophores, anchorage chemistries, and linker arms. Employing C. elegans as a test bed, we resolve metabolically labelled O-glycans on the gut microvilli and other nanoscale anatomical features using our ExM reagents and optimized protocols. We use transmission electron microscopy images of C. elegans nano-anatomy as ground truth data to validate the fidelity and isotropy of gel expansion. We construct whole organism maps of C. elegans O-glycosylation in the first larval stage and identify O-glycan \"hotspots\" in unexpected anatomical locations, including the body wall furrows. Beyond C. elegans, we provide validated ExM protocols for nanoscale imaging of metabolically labelled glycans on cultured mammalian cells. Together, our results suggest the broad applicability of the multifunctional reagents for imaging glycans and other metabolically labelled biomolecules at enhanced resolutions with ExM. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=95 SRC=\"FIGDIR/small/578333v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (43K): org.highwire.dtl.DTLVardef@403eaeorg.highwire.dtl.DTLVardef@1d3293aorg.highwire.dtl.DTLVardef@37de58org.highwire.dtl.DTLVardef@1e696cf_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "2486",
  "inLanguage": "en",
  "datePublished": "2024-02-07T10:40:15Z",
  "dateModified": "2024-02-07T10:40:15Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 7, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577489">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577489" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577489">
        <p class="paperTitle">A general platform for targeting MHC-II antigens via a single loop</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577489" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577489" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Du, H.; Liu, J.; Jude, K. M.; Yang, X.; Li, Y.; Bell, B.; Yang, H.; Kassardjian, A.; Mobedi, A.; Parekh, U.; Sperberg, A.; Julien, J.-P.; Mellins, E.; Garcia, K. C.; Huang, P.</p>
        <p class="info">Score: 39.3, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577489' target='https://doi.org/10.1101/2024.01.26.577489'> 10.1101/2024.01.26.577489</a></p>
        <p class="abstract">Class-II major histocompatibility complexes (MHC-IIs) are central to the communications between CD4&#43; T cells and antigen presenting cells (APCs), but intrinsic structural features associated with MHC-II make it difficult to develop a general targeting system with high affinity and antigen specificity. Here, we introduce a protein platform, Targeted Recognition of Antigen-MHC Complex Reporter for MHC-II (TRACeR-II), to enable the rapid development of peptide-specific MHC-II binders. TRACeR-II has a small helical bundle scaffold and uses an unconventional mechanism to recognize antigens via a single loop. This unique antigen-recognition mechanism renders this platform highly versatile and amenable to direct structural modeling of the interactions with the antigen. We demonstrate that TRACeR-II binders can be rapidly evolved across multiple alleles, while computational protein design can produce specific binding sequences for a SARS-CoV-2 peptide of unknown complex structure. TRACeR-II sheds light on a simple and straightforward approach to address the MHC peptide targeting challenge, without relying on combinatorial selection on complementarity determining region (CDR) loops. It presents a promising basis for further exploration in immune response modulation as well as a broad range of theragnostic applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.578280">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.578280" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.578280">
        <p class="paperTitle">Grayscale 4D Biomaterial Customization at High Resolution and Scale</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.578280" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.578280" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Batalov, I.; Filteau, J. R.; Francis, R. M.; Jaindl, G.; Orr, L.; Rapp, T. L.; Yang, S.; Filteau, J. A.; Xie, W.; Bretherton, R. C.; Glaser, A. K.; Liu, J. T. C.; Stevens, K. R.; DeForest, C. A.</p>
        <p class="info">Score: 32.7, Published: 2024-02-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.578280' target='https://doi.org/10.1101/2024.01.31.578280'> 10.1101/2024.01.31.578280</a></p>
        <p class="abstract">Hydrogel biomaterials have proven indispensable for three-dimensional (3D) cell culture but have fallen short in replicating the innate physiochemical complexity of real tissue. Though traditional photolithography affords localized material manipulation, robust methods that govern when, where, and to what extent such phototailoring occurs throughout materials would be profoundly enabling towards fabricating more-realistic 3D tissue constructs. Here, we introduce &#34;grayscale image z-stack-guided multiphoton optical-lithography&#34; (GIZMO) as a generalizable and intuitive strategy to rapidly photomodulate materials in full 3D non-binary patterns at submicron resolutions spanning large volumes (&gt;mm3). Highlighting its versatility, we employ GIZMO to variably photopattern biomolecule release from, protein immobilization to, and degradation within hydrogels based on biologically derived or synthetic grayscale image stacks with unprecedented complexity. We anticipate that GIZMO will enable new opportunities to probe and manipulate cell fates, as well as to engineer complex functional tissue.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578500">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578500" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578500">
        <p class="paperTitle">A combination of two-enzyme system and enzyme engineering improved the activity of a new PET hydrolase identified from soil bacterial genome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578500" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578500" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mabashi-Asazuma, H.; Hirai, M.; Sakurai, S.; Ide, K.; Kogawa, M.; Matsushita, A.; Hosokawa, M.; Tsuda, S.</p>
        <p class="info">Score: 8.8, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578500' target='https://doi.org/10.1101/2024.02.01.578500'> 10.1101/2024.02.01.578500</a></p>
        <p class="abstract">We here report a novel PET hydrolase originating from a soil microbial genome sequence. This enzyme, bbPET0069, exhibits characteristics resembling a cutinase-like Type I PET-degrading enzyme but lacks disulfide bonds. Notably, bbPET0069 displayed remarkable synergy with Candida antarctica lipase B (CALB), demonstrating rapid and efficient PET degradation. To improve the PET degradation activity of bbPET0069, we employed a three-dimensional (3D) structural modeling to identify mutation sites around its substrate binding domain combined with a protein language model for effective mutation prediction. Through three initial rounds of directed evolution, we achieved a significant enhancement in PET degradation with CALB, resulting in a 12.6-fold increase compared to wild-type bbPET0069 without CALB. We confirmed its PET degradation activity in PET nanoparticles and films, and our proposed approach enabled efficient PET degradation to terephthalic acid monomers up to 95.5%. Our approach, which integrates a two-enzyme system with protein engineering, demonstrates the potential for enhancing the activity of emerging PET-degradation enzymes, which may possess unique attributes.

Graphical AbstractA novel PET hydrolase, bbPET0069, was identified from a soil microbial genome. bbPET0069 and CALB showed remarkable synergy in PET degradation. Using surface feature analysis, PET degradation activity of bbPET0069 was significantly improved. This combination of a two-enzyme system and surface feature analysis holds promise for enhancing emerging PET-degradation enzymes.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=130 SRC=&#34;FIGDIR/small/578500v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (27K):
org.highwire.dtl.DTLVardef@1a60a45org.highwire.dtl.DTLVardef@1ec776eorg.highwire.dtl.DTLVardef@19500c6org.highwire.dtl.DTLVardef@166068c_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.577263">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.577263" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.577263">
        <p class="paperTitle">Refining Spatial Proteomics by Mass Spectrometry: An Efficient Workflow Tailored for Archival Tissue</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.577263" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.577263" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Daucke, R.; Rift, C. V.; Bager, N. S.; Saxena, K.; Koffeldt, P. R.; Woessmann, J.; Petrosius, V.; Rugiu, E. S.; Kristensen, B. W.; Klausen, P.; Schoof, E. M.</p>
        <p class="info">Score: 9.3, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.577263' target='https://doi.org/10.1101/2024.01.25.577263'> 10.1101/2024.01.25.577263</a></p>
        <p class="abstract">Formalin-fixed paraffin-embedded (FFPE) tissue, while excellent for preserving tissue for extended periods of time, poses a challenge when extracting molecular information. We therefore developed an easily adaptable and highly efficient workflow for extracting high levels of proteins from low-input material. We compared sensitivity between two stains, EpCAM and H&amp;E, across material inputs of 1,166 and [~]800,000 {micro}m2. In the context of EpCAM-stained tissue, our investigations unveiled a range from [~]1,200 unique protein groups at the lowest input to [~]5,900 at the highest. For H&amp;E, the spectrum covers [~]900 to [~]5,200 protein groups. We found an optimal balance between maximizing detected proteins and minimizing input material to be within the range of [~]50,000 to [~]100,000 {micro}m2. With this knowledge, we tested the spatial capabilities by isolating specific cell populations, through Laser Capture Microdissection (LCM), from three different tissue types, where we were able to identify tissue-specific signatures and prominent clustering of all cell populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.578234">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.578234" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.578234">
        <p class="paperTitle">Mechanically Tunable, Compostable, Healable and Scalable Engineered Living Materials</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.578234" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.578234" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Manjula-Basavanna, A.; Joshi, N.; Duraj-Thatte, A.</p>
        <p class="info">Score: 6.0, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.578234' target='https://doi.org/10.1101/2024.01.31.578234'> 10.1101/2024.01.31.578234</a></p>
        <p class="abstract">Novel design strategies are essential to realize the full potential of Engineered Living Materials (ELMs), including their biodegradability, manufacturability, sustainability, and ability to tailor functional properties. Toward these goals, we present Mechanically Engineered Living Material with Compostability, Healability, and Scalability (MECHS) - a material that integrates these features in the form of a stretchable plastic that is simultaneously flushable, compostable, and exhibits the characteristics of paper. This plastic/paper-like material is produced directly from cultured bacterial biomass (40%) producing engineered curli protein nanofibers in scalable quantities (0.5-1 g L-1). The elongation at break (1-160%) and Youngs modulus (6-450 MPa) of MECHS was tuned to more than two orders of magnitude. By genetically encoded covalent crosslinking of curli nanofibers, we increase the Youngs modulus by two times. MECHS biodegrades completely in 15-75 days, while its mechanical properties are comparable to petrochemical plastics and thus may find use as compostable materials for primary packaging.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.578106">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.578106" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.578106">
        <p class="paperTitle">Collagen organization and structure in FLBN5-/- mice using label-free microscopy: implications for pelvic organ prolapse</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.578106" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.578106" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jennings, C.; Markel, A.; Domingo, M.; Miller, K.; Bayer, C.; Parekh, S. H.</p>
        <p class="info">Score: 5.5, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.578106' target='https://doi.org/10.1101/2024.01.31.578106'> 10.1101/2024.01.31.578106</a></p>
        <p class="abstract">Pelvic organ prolapse (POP) is a gynecological disorder described by the descent of superior pelvic organs into or out of the vagina as a consequence of disrupted muscles and tissue. A thorough understanding of the etiology of POP is limited by the availability of clinically relevant samples, restricting longitudinal POP studies on soft-tissue biomechanics and structure to POP-induced models such as fibulin-5 knockout (FBLN5-/-) mice. Despite being a principal constituent in the extracellular matrix, little is known about structural perturbations to collagen networks in the FBLN5-/- mouse cervix. We identify significantly different collagen network populations in normal and prolapsed cervical cross-sections using two label-free, nonlinear microscopy techniques. Collagen in the prolapsed mouse cervix tends to be more isotropic, and displays reduced alignment persistence via 2-D Fourier Transform analysis of images acquired using second harmonic generation microscopy. Furthermore, coherent Raman hyperspectral imaging revealed elevated disorder in the secondary structure of collagen in prolapsed tissues. Our results underscore the need for in situ multimodal monitoring of collagen organization to improve POP predictive capabilities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.577810">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.577810" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.577810">
        <p class="paperTitle">Focused Ultrasound Blood-Brain Barrier Opening Arrests the Growth and Formation of Cerebral Cavernous Malformations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.577810" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.577810" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fisher, D. G.; Sharifi, K. A.; Shah, I. M.; Gorick, C. M.; Breza, V. R.; Debski, A. C.; Hoch, M.; Cruz, T.; Samuels, J. D.; Sheehan, J.; Schlesinger, D.; Moore, D.; Lukens, J. R.; Tvrdik, P.; Miller, G. W.; Price, R. J.</p>
        <p class="info">Score: 4.8, Published: 2024-02-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.577810' target='https://doi.org/10.1101/2024.01.31.577810'> 10.1101/2024.01.31.577810</a></p>
        <p class="abstract">BACKGROUNDCerebral cavernous malformations (CCM) are vascular lesions within the central nervous system, consisting of dilated and hemorrhage-prone capillaries. CCMs can cause debilitating neurological symptoms, and surgical excision or stereotactic radiosurgery are the only current treatment options. Meanwhile, transient blood-brain barrier opening (BBBO) with focused ultrasound (FUS) and microbubbles is now understood to exert potentially beneficial bioeffects, such as stimulation of neurogenesis and clearance of amyloid-{beta}. Here, we tested whether FUS BBBO could be deployed therapeutically to control CCM formation and progression in a clinically-representative murine model.

METHODSCCMs were induced in mice by postnatal, endothelial-specific Krit1 ablation. FUS was applied for BBBO with fixed peak-negative pressures (PNPs; 0.2-0.6 MPa) or passive cavitation detection-modulated PNPs. Magnetic resonance imaging (MRI) was used to target FUS treatments, evaluate safety, and measure longitudinal changes in CCM growth after BBBO.

RESULTSFUS BBBO elicited gadolinium accumulation primarily at the perilesional boundaries of CCMs, rather than lesion cores. Passive cavitation detection and gadolinium contrast enhancement were comparable in CCM and wild-type mice, indicating that Krit1 ablation does not confer differential sensitivity to FUS BBBO. Acutely, CCMs exposed to FUS BBBO remained structurally stable, with no signs of hemorrhage. Longitudinal MRI revealed that FUS BBBO halted the growth of 94% of CCMs treated in the study. At 1 month, FUS BBBO-treated lesions lost, on average, 9% of their pre-sonication volume. In contrast, non-sonicated control lesions grew to 670% of their initial volume. Lesion control with FUS BBBO was accompanied by a marked reduction in the area and mesenchymal appearance of Krit mutant endothelium. Strikingly, in mice receiving multiple BBBO treatments with fixed PNPs, de novo CCM formation was significantly reduced by 81%. Mock treatment plans on MRIs of patients with surgically inaccessible lesions revealed their lesions are amenable to FUS BBBO with current clinical technology.

CONCLUSIONSOur results establish FUS BBBO as a novel, non-invasive modality that can safely arrest murine CCM growth and prevent their de novo formation. As an incisionless, MR image-guided therapy with the ability to target eloquent brain locations, FUS BBBO offers an unparalleled potential to revolutionize the therapeutic experience and enhance the accessibility of treatments for CCM patients.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.03.578756">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.03.578756" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.03.578756">
        <p class="paperTitle">Prolonged airway explant culture enables study of health, disease, and viral pathogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.03.578756" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.03.578756" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee-Ferris, R. E.; Okuda, K.; Galiger, J. R.; Schworer, S. A.; Rogers, T. D.; Dang, H.; Gilmore, R.; Edwards, C.; Nakano, S.; Cawley, A. M.; Pickles, R. J.; Gallant, S. C.; Crisci, E.; Rivier, L.; Hagood, J. S.; O&#39;Neal, W. K.; Baric, R. S.; Grubb, B. R.; Boucher, R. C.; Randell, S. H.</p>
        <p class="info">Score: 4.3, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.03.578756' target='https://doi.org/10.1101/2024.02.03.578756'> 10.1101/2024.02.03.578756</a></p>
        <p class="abstract">In vitro models play a major role in studying airway physiology and disease. However, the native lungs complex tissue architecture and non-epithelial cell lineages are not preserved in these models. Ex vivo tissue models could overcome in vitro limitations, but methods for long-term maintenance of ex vivo tissue has not been established. We describe methods to culture human large airway explants, small airway explants, and precision-cut lung slices for at least 14 days. Human airway explants recapitulate genotype-specific electrophysiology, characteristic epithelial, endothelial, stromal and immune cell populations, and model viral infection after 14 days in culture. These methods also maintain mouse, rabbit, and pig tracheal explants. Notably, intact airway tissue can be cryopreserved, thawed, and used to generate explants with recovery of function 14 days post-thaw. These studies highlight the broad applications of airway tissue explants and their use as translational intermediates between in vitro and in vivo studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.578101">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.578101" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.578101">
        <p class="paperTitle">The T-cell niche tunes immune function through modulation of the cytoskeleton and TCR-antigen forces</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.578101" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.578101" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kellner, A. V.; Hunter, R.; Do, P.; Eggert, J.; Jaffe, M.; Geitgey, D. K.; Lee, M.; Hamilton, J. A. G.; Ross, A. J.; Ank, R. S.; Bender, R. L.; Ma, R.; Porter, C. C.; Dreaden, E. C.; Au-Yeung, B. B.; Haynes, K. A.; Henry, C. J.; Salaita, K.</p>
        <p class="info">Score: 3.5, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.578101' target='https://doi.org/10.1101/2024.01.31.578101'> 10.1101/2024.01.31.578101</a></p>
        <p class="abstract">Obesity is a major public health crisis given its rampant growth and association with an increased risk for cancer. Interestingly, patients with obesity tend to have an increased tumor burden and decreased T-cell function. It remains unclear how obesity compromises T-cell mediated immunity. To address this question, we modeled the adipocyte niche using the secretome released from adipocytes as well as the niche of stromal cells and investigated how these factors modulated T-cell function. We found that the secretomes altered antigen-specific T-cell receptor (TCR) triggering and activation. RNA-sequencing analysis identified thousands of gene targets modulated by the secretome including those associated with cytoskeletal regulation and actin polymerization. We next used molecular force probes to show that T-cells exposed to the adipocyte niche display dampened force transmission to the TCR-antigen complex and conversely, stromal cell secreted factors lead to significantly enhanced TCR forces. These results were then validated in diet-induced obese mice. Importantly, secretome-mediated TCR force modulation mirrored the changes in T-cell functional responses in human T-cells using the FDA-approved immunotherapy, blinatumomab. Thus, this work shows that the adipocyte niche contributes to T-cell dysfunction through cytoskeletal modulation and reduces TCR triggering by dampening TCR forces consistent with the mechanosensor model of T-cell activation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578333">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578333" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578333">
        <p class="paperTitle">Bio-orthogonal Glycan Imaging of Culture Cells and Whole Animal C. elegans with Expansion Microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578333" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578333" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kuo, J. C.-H.; Colville, M. J.; Sorkin, M. R.; Kuo, J. L. K.; Huang, L.-T.; Thornlow, D. N.; Beacham, G. M.; Hollopeter, G.; DeLisa, M. P.; Alabi, C. A.; Paszek, M. J.</p>
        <p class="info">Score: 3.4, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578333' target='https://doi.org/10.1101/2024.02.01.578333'> 10.1101/2024.02.01.578333</a></p>
        <p class="abstract">Complex carbohydrates called glycans play crucial roles in the regulation of cell and tissue physiology, but how glycans map to nanoscale anatomical features must still be resolved. Here, we present the first nanoscale map of mucin-type O-glycans throughout the entirety of the Caenorhabditis elegans model organism. We construct a library of multifunctional linkers to probe and anchor metabolically labelled glycans in expansion microscopy (ExM), an imaging modality that overcomes the diffraction limit of conventional optical microscopes through the physical expansion of samples embedded in a polyelectrolyte gel matrix. A flexible strategy is demonstrated for the chemical synthesis of linkers with a broad inventory of bio-orthogonal functional groups, fluorophores, anchorage chemistries, and linker arms. Employing C. elegans as a test bed, we resolve metabolically labelled O-glycans on the gut microvilli and other nanoscale anatomical features using our ExM reagents and optimized protocols. We use transmission electron microscopy images of C. elegans nano-anatomy as ground truth data to validate the fidelity and isotropy of gel expansion. We construct whole organism maps of C. elegans O-glycosylation in the first larval stage and identify O-glycan &#34;hotspots&#34; in unexpected anatomical locations, including the body wall furrows. Beyond C. elegans, we provide validated ExM protocols for nanoscale imaging of metabolically labelled glycans on cultured mammalian cells. Together, our results suggest the broad applicability of the multifunctional reagents for imaging glycans and other metabolically labelled biomolecules at enhanced resolutions with ExM.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=95 SRC=&#34;FIGDIR/small/578333v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (43K):
org.highwire.dtl.DTLVardef@403eaeorg.highwire.dtl.DTLVardef@1d3293aorg.highwire.dtl.DTLVardef@37de58org.highwire.dtl.DTLVardef@1e696cf_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
