<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Surface Patterned Omniphobic Tiles (SPOTs): a versatile platform for scalable liquid handling
Authors: Shiri, S.; Qazi, M. J.; Tan, S.; Albo, J.; Chen, A.; Fukuda, R.; Jain, M. S.; Menesses, M.; Nchinda, N.; Ahmed, G.; Gallegos, A.; Gangishetty, M.; Congreve, D.; Cira, N. J.
Score: 23.9, Published: 2024-01-22 DOI: 10.1101/2024.01.17.575712
Manipulating liquids is a ubiquitous need for experiments across numerous scientific disciplines. To overcome limitations of current methods, we introduce Surface Patterned Omniphobic Tiles (SPOTs).">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Surface Patterned Omniphobic Tiles (SPOTs): a versatile platform for scalable liquid handling
Authors: Shiri, S.; Qazi, M. J.; Tan, S.; Albo, J.; Chen, A.; Fukuda, R.; Jain, M. S.; Menesses, M.; Nchinda, N.; Ahmed, G.; Gallegos, A.; Gangishetty, M.; Congreve, D.; Cira, N. J.
Score: 23.9, Published: 2024-01-22 DOI: 10.1101/2024.01.17.575712
Manipulating liquids is a ubiquitous need for experiments across numerous scientific disciplines. To overcome limitations of current methods, we introduce Surface Patterned Omniphobic Tiles (SPOTs)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-28T10:37:59+00:00" />
<meta property="article:modified_time" content="2024-01-28T10:37:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Surface Patterned Omniphobic Tiles (SPOTs): a versatile platform for scalable liquid handling
Authors: Shiri, S.; Qazi, M. J.; Tan, S.; Albo, J.; Chen, A.; Fukuda, R.; Jain, M. S.; Menesses, M.; Nchinda, N.; Ahmed, G.; Gallegos, A.; Gangishetty, M.; Congreve, D.; Cira, N. J.
Score: 23.9, Published: 2024-01-22 DOI: 10.1101/2024.01.17.575712
Manipulating liquids is a ubiquitous need for experiments across numerous scientific disciplines. To overcome limitations of current methods, we introduce Surface Patterned Omniphobic Tiles (SPOTs)."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Surface Patterned Omniphobic Tiles (SPOTs): a versatile platform for scalable liquid handling\nAuthors: Shiri, S.; Qazi, M. J.; Tan, S.; Albo, J.; Chen, A.; Fukuda, R.; Jain, M. S.; Menesses, M.; Nchinda, N.; Ahmed, G.; Gallegos, A.; Gangishetty, M.; Congreve, D.; Cira, N. J.\nScore: 23.9, Published: 2024-01-22 DOI: 10.1101/2024.01.17.575712\nManipulating liquids is a ubiquitous need for experiments across numerous scientific disciplines. To overcome limitations of current methods, we introduce Surface Patterned Omniphobic Tiles (SPOTs).",
  "keywords": [
    
  ],
  "articleBody": " Surface Patterned Omniphobic Tiles (SPOTs): a versatile platform for scalable liquid handling\nAuthors: Shiri, S.; Qazi, M. J.; Tan, S.; Albo, J.; Chen, A.; Fukuda, R.; Jain, M. S.; Menesses, M.; Nchinda, N.; Ahmed, G.; Gallegos, A.; Gangishetty, M.; Congreve, D.; Cira, N. J.\nScore: 23.9, Published: 2024-01-22 DOI: 10.1101/2024.01.17.575712\nManipulating liquids is a ubiquitous need for experiments across numerous scientific disciplines. To overcome limitations of current methods, we introduce Surface Patterned Omniphobic Tiles (SPOTs). This platform combines geometry and surface engineering, building on discontinuous wetting approaches to leverage capillarity for metering liquids. The SPOTs platform allows manipulation of hundreds to thousands of independent experiments without expensive equipment or large consumable costs. These devices can handle a wide range of liquid types and volumes (\u003c10 nanoliters to \u003e10 microliters) with better precision than pipetting. The platform is inexpensive and easy to fabricate, fast and intuitive to use, and cross-compatible with existing microwell plate layouts. We demonstrate how these capabilities facilitate diverse experiments including testing antibiotic combinations for synergy and antagonism, material screening of perovskites, and genotyping microbial isolates. We anticipate SPOTs will enable users from disparate domains to quickly and easily run a wide range of high-throughput experiments.\nAn experimental census of retrons for DNA production and genome editing\nAuthors: Khan, A. G.; Rojas-Montero, M.; Gonzalez-Delgado, A.; Lopez, S. C.; Fang, R. F.; Shipman, S. L.\nScore: 23.8, Published: 2024-01-26 DOI: 10.1101/2024.01.25.577267\nRetrons are bacterial immune systems that use reverse transcribed DNA as a detector of phage infection. They are also increasingly deployed as a component of biotechnology. For genome editing, for instance, retrons are modified so that the reverse transcribed DNA (RT-DNA) encodes an editing donor. Retrons are commonly found in bacterial genomes; thousands of unique retrons have now been predicted bioinformatically. However, only a small number have been characterized experimentally. Here, we add substantially to the corpus of experimentally studied retrons. We synthesized \u003e100 previously untested retrons to identify the natural sequence of RT-DNA they produce, quantify their RT-DNA production, and test the relative efficacy of editing using retron-derived donors to edit bacterial genomes, phage genomes, and human genomes. We add 62 new empirically determined, natural RT-DNAs, which are not predictable from the retron sequence alone. We report a large diversity in RT-DNA production and editing rates across retrons, finding that top performing editors outperform those used in previous studies, and are drawn from a subset of the retron phylogeny.\nDNA microbeads for spatio-temporally controlled morphogen release within organoids\nAuthors: Afting, C.; Walther, T.; Wittbrodt, J.; Göpfrich, K.\nScore: 17.5, Published: 2024-01-12 DOI: 10.1101/2024.01.10.575045\nOrganoids have proven to be powerful in vitro model systems that mimic features of the corresponding tissue in vivo. However, across tissue types and species, organoids still often fail to reach full maturity and function, because biochemical cues cannot be provided from within the organoid to guide their development. The establishment of such tools has been identified as a major goal of the field. Here, we introduce DNA microbeads as a novel tool for implementing spatio-temporally controlled morphogen gradients inside of organoids at any point in their life cycle. The DNA microbeads are formed in a simple one-pot process, they can be stored for a year and their viscoelastic behavior and surface modification is tunable to mimic the corresponding tissue. Employing medaka retinal organoids and early embryos, we show that DNA microbeads can be integrated into embryos and organoids by microinjection and erased in a non-invasive manner with light. Coupling a recombinant surrogate Wnt to the DNA microbeads we demonstrate the spatio-temporally controlled release of the morphogen from the microinjection site, which leads to the formation of retinal pigmented epithelium while maintaining neuroretinal ganglion cells. We were thus able to bioengineer retinal organoids to more closely mirror the cell type diversity of in vivo retinas. The DNA microbead technology can easily be adapted to other organoid applications for improved tissue mimicry.\nTreatment of a genetic liver disease in mice through transient prime editor expression\nAuthors: Rothgangl, T.; Ioannidi, E. I.; Weber, Y.; Talas, A.; Boeck, D.; Matsushita, M.; Villiger, E. A.; Schmidheini, L.; Moon, J.; Lin, P. J. C.; Fan, S. H. Y.; Marquart, K. F.; Schwerdel, C.; Rimann, N.; Faccin, E.; Cremonesi, A.; Villiger, L.; Muramatsu, H.; Vadovics, M.; Thoeny, B.; Kopf, M.; Haeberle, J.; Pardi, N.; Tam, Y. K.; Schwank, G.\nScore: 9.2, Published: 2024-01-24 DOI: 10.1101/2024.01.22.575834\nPrime editing is a versatile genome editing technology that does not rely on DNA double-strand break formation and homology-directed repair (HDR). This makes it a promising tool for correcting pathogenic mutations in tissues consisting predominantly of postmitotic cells, such as the liver. While recent studies have already demonstrated proof-of-concept for in vivo prime editing, the use of viral delivery vectors resulted in prolonged prime editor (PE) expression, posing challenges for clinical application. Here, we developed an in vivo prime editing approach where we delivered the pegRNA using self-complementary adeno-associated viral (scAAV) vectors and the prime editor using nucleoside-modified mRNA encapsulated in lipid nanoparticles (LNPs). This methodology led to transient expression of the PE for 48h and 26% editing at the Dnmt1 locus using AAV doses of 2.5x1013 vector genomes (vg)/kg and a single dose of 3mg/kg mRNA-LNP. When targeting the pathogenic mutation in the Pahenu2 mouse model of phenylketonuria (PKU), we achieved 4.3% gene correction using an AAV dose of 2.5x1013 vg/kg and three doses of 2 mg/kg mRNA-LNP. Editing was specific to the liver and the intended locus, and was sufficient to reduce blood L-phenylalanine (Phe) levels from over 1500 umol/l to below the therapeutic threshold of 600 umol/l. Our study demonstrates the feasibility of in vivo gene correction in the liver with transient PE expression, bringing prime editing closer to clinical application.\nQuantification of local matrix deposition during muscle stem cell activation using engineered hydrogels\nAuthors: Duran, P.; Yang, B. A.; Plaster, E.; Eiken, M.; Loebel, C.; Aguilar, C. A.\nScore: 5.7, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576326\nAdult stem cells occupy a niche that contributes to their function, but how stem cells remodel their microenvironment remains an open-ended question. Herein, biomaterials-based systems and metabolic labeling were utilized to evaluate how skeletal muscle stem cells deposit extracellular matrix. Muscle stem cells and committed myoblasts were observed to generate less nascent matrix than muscle resident fibro-adipogenic progenitors. When cultured on substrates that matched the stiffness of physiological uninjured and injured muscles, the increased nascent matrix deposition was associated with stem cell activation. Reducing the ability to deposit nascent matrix in muscle stem cells attenuated function and mimicked impairments observed from muscle stem cells isolated from old aged muscles, which could be rescued with therapeutic supplementation of insulin-like growth factors. These results highlight how nascent matrix production is critical for maintaining healthy stem cell function.\nAutomated Synthesis of Wireframe DNA Nanotubes\nAuthors: Islas, P.; Platnich, C.; Gidi, Y.; Karimi, R.; Ginot, L.; Cosa, G.; Sleiman, H.\nScore: 4.0, Published: 2024-01-23 DOI: 10.1101/2024.01.18.576297\nDNA nanotechnology has revolutionized our ability to position matter at the nanoscale, but the preparation of DNA-based architectures remains laborious. To facilitate the formation of custom structures, we report a fully automated method to produce sequence- and size-defined DNA nanotubes. By programming the sequential addition of desired building blocks, rigid DX-tile-based DNA nanotubes (and flexible wireframe DNA structures) are attained, where the total number of possible constructs increases as a power function of the number of different units available. Using single-molecule fluorescence imaging, the kinetics and yield of each synthetic step can be quantitatively determined, revealing differences in self-assembly dynamics as the nanotube is built up from the solid support and providing new insights into DNA self-assembly. The exploitation of automation for both assembly and analysis (tthrough an ad-hoc developed K-means clustering algorithm) facilitates a workflow wherein the synthesis parameters may be iteratively improved upon, demonstrating how a single-molecule assembly-analysis-optimization sequence can be used to generate complex, non-covalent materials in good yield. The presented synthetic strategy is generalizable, making use of equipment already available in most standard laboratories and represents the first fully automated supramolecular assembly on a solid support.\nEpigenetic control of multiple genes with a single lentiviral vector encoding transcriptional repressors fused to compact zinc finger arrays\nAuthors: Monteferrario, D.; David, M.; Tadi, S. K.; Zhou, Y.; Marchetti, I.; Jeanneau, C.; Saviane, G.; Dupont, C. F.; Martelli, A. E.; Truong, L.; Eshleman, J.; Ng, C.; Huston, M.; Davis, G. D.; Fontenot, J. D.; Reik, A.; de la Rosa, M.; Fenard, D.\nScore: 4.0, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576049\nGene silencing without gene editing holds great potential for the development of safe therapeutic applications. Here, we describe a novel strategy to concomitantly repress multiple genes using zinc finger proteins fused to Kruppel-Associated Box repression domains (ZF-Rs). This was achieved via the optimization of a lentiviral system tailored for the delivery of ZF-Rs in hematopoietic cells. We showed that an optimal design of the lentiviral backbone is crucial to multiplex up to three ZF-Rs or two ZF-Rs and a chimeric antigen receptor. ZF-R expression had no impact on the integrity and functionality of transduced cells. Furthermore, gene repression in ZF-R-expressing T cells was highly efficient in vitro and in vivo during the entire monitoring period (up to ten weeks), and it was accompanied by epigenetic remodeling events. Finally, we described an approach to improve ZF-R specificity to illustrate the path towards the generation of ZF-Rs with a safe clinical profile. In conclusion, we successfully developed an epigenetic-based cell engineering approach for concomitant modulation of multiple gene expressions that bypass the risks associated with DNA editing.\nHuman vascular organoids with a mosaic AKT1 mutation recapitulate Proteus syndrome\nAuthors: He, S.; Zhu, Y.; Chauhan, S.; Tavakol, D. N.; Lee, J. H.; Berris, R. B.-L.; Xu, C.; Lee, J.; Lee, C.; Cai, S.; McElroy, S.; Vunjak-Novakovic, G.; Tomer, R.; Azizi, E.; Xu, B.; Lao, Y.-H.; Leong, K. W.\nScore: 2.5, Published: 2024-01-27 DOI: 10.1101/2024.01.26.577324\nVascular malformation, a key clinical phenotype of Proteus syndrome, lacks effective models for pathophysiological study and drug development due to limited patient sample access. To bridge this gap, we built a human vascular organoid model replicating Proteus syndrome's vasculature. Using CRISPR/Cas9 genome editing and gene overexpression, we created induced pluripotent stem cells (iPSCs) embodying the Proteus syndrome-specific AKTE17K point mutation for organoid generation. Our findings revealed that AKT overactivation in these organoids resulted in smaller sizes yet increased vascular connectivity, although with less stable connections. This could be due to the significant vasculogenesis induced by AKT overactivation. This phenomenon likely stems from boosted vasculogenesis triggered by AKT overactivation, leading to increased vascular sprouting. Additionally, a notable increase in dysfunctional PDGFRbeta+ mural cells, impaired in matrix secretion, was observed in these AKT-overactivated organoids. The application of AKT inhibitors (ARQ092, AZD5363, or GDC0068) reversed the vascular malformations; the inhibitors' effectiveness was directly linked to reduced connectivity in the organoids. In summary, our study introduces an innovative in vitro model combining organoid technology and gene editing to explore vascular pathophysiology in Proteus syndrome. This model not only simulates Proteus syndrome vasculature but also holds potential for mimicking vasculatures of other genetically driven diseases. It represents an advance in drug development for rare diseases, historically plagued by slow progress.\nMicrofluidic platform enables shear-less aerosolization of lipid nanoparticles for messenger RNA inhalation\nAuthors: Kim, J.; Jozic', A.; Bloom, E.; Jones, B.; Marra, M.; Murthy, N. T. V.; Eygeris, Y.; Sahay, G.\nScore: 3.8, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576136\nLeveraging the extensive surface area of the lungs for gene therapy, inhalation route offers distinct advantages for delivery. Clinical nebulizers that employ vibrating mesh technology are the standard choice for converting liquid medicines into aerosols. However, they have limitations when it comes to delivering mRNA through inhalation, including severe damage to nanoparticles due to shearing forces. Here, we introduce a novel microfluidic aerosolization platform (MAP) that preserves the structural and physicochemical integrity of lipid nanoparticles, enabling safe and efficient mRNA delivery to the respiratory system. Our results demonstrated the superiority of the novel MAP over the conventional vibrating mesh nebulizer, as it avoided problems such as particle aggregation, loss of mRNA encapsulation, and deformation of nanoparticle morphology. Notably, aerosolized nanoparticles generated by the microfluidic device led to enhanced transfection efficiency across various cell lines. In vivo experiments with mice that inhaled these aerosolized nanoparticles revealed successful, lung-specific mRNA transfection without observable signs of toxicity. This pioneering MAP represents a significant advancement for the pulmonary gene therapy, enabling precise and effective delivery of aerosolized nanoparticles.\nGenerative AI for Cardiac Organoid Florescence Generation\nAuthors: Kandula, A. K. R.; Phamornratanakun, T.; Gomez, A. H.; Bhoi, R.; El-Mokahal, M.; Feng, Y.; Yang, H.\nScore: 4.5, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575724\nhPSC-derived cardiac organoid is the most recent three-dimensional tissue structure that mimics the structure and functionality of the human heart and plays a pivotal role in modeling heart development and disease. The hPSC-derived cardiac organoids are commonly characterized by bright-field microscopic imaging. Although the bright-field microscope provides essential information about hPSC-derived cardiac organoids, such as morphology, size, and general structure, it does not extend our understanding of cardiac organoids on cell type-specific distribution and structure. Then, fluorescence microscopic imaging is required to identify the specific cardiovascular cell types in the hPSC-derived cardiac organoids by fluorescence immunostaining of fixed organoid samples or fluorescence reporter imaging of live organoids. Both approaches require extra steps of experiments and do not provide general information on all hPSC-derived cardiac organoids from different hPSC lines, which limits the biomedical applications of hPSC-derived cardiac organoids. This research addresses this limitation by proposing a comprehensive workflow for colorizing grayscale cardiac organoid images from bright-field microscopic imaging using conditional Generative Adversarial Networks (GANs) to provide cell type-specific information. By infusing these grayscale images with accurate fluorescence colorization, our approach aims to unlock the hidden wealth of cell type, distribution, and network, for bettering characterizing hPSC-derived cardiac organoids.\n",
  "wordCount" : "2290",
  "inLanguage": "en",
  "datePublished": "2024-01-28T10:37:59Z",
  "dateModified": "2024-01-28T10:37:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 28, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.575712">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.575712" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.575712">
        <p class="paperTitle">Surface Patterned Omniphobic Tiles (SPOTs): a versatile platform for scalable liquid handling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.575712" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.575712" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shiri, S.; Qazi, M. J.; Tan, S.; Albo, J.; Chen, A.; Fukuda, R.; Jain, M. S.; Menesses, M.; Nchinda, N.; Ahmed, G.; Gallegos, A.; Gangishetty, M.; Congreve, D.; Cira, N. J.</p>
        <p class="info">Score: 23.9, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.575712' target='https://doi.org/10.1101/2024.01.17.575712'> 10.1101/2024.01.17.575712</a></p>
        <p class="abstract">Manipulating liquids is a ubiquitous need for experiments across numerous scientific disciplines. To overcome limitations of current methods, we introduce Surface Patterned Omniphobic Tiles (SPOTs). This platform combines geometry and surface engineering, building on discontinuous wetting approaches to leverage capillarity for metering liquids. The SPOTs platform allows manipulation of hundreds to thousands of independent experiments without expensive equipment or large consumable costs. These devices can handle a wide range of liquid types and volumes (&lt;10 nanoliters to &gt;10 microliters) with better precision than pipetting. The platform is inexpensive and easy to fabricate, fast and intuitive to use, and cross-compatible with existing microwell plate layouts. We demonstrate how these capabilities facilitate diverse experiments including testing antibiotic combinations for synergy and antagonism, material screening of perovskites, and genotyping microbial isolates. We anticipate SPOTs will enable users from disparate domains to quickly and easily run a wide range of high-throughput experiments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.577267">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.577267" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.577267">
        <p class="paperTitle">An experimental census of retrons for DNA production and genome editing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.577267" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.577267" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khan, A. G.; Rojas-Montero, M.; Gonzalez-Delgado, A.; Lopez, S. C.; Fang, R. F.; Shipman, S. L.</p>
        <p class="info">Score: 23.8, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.577267' target='https://doi.org/10.1101/2024.01.25.577267'> 10.1101/2024.01.25.577267</a></p>
        <p class="abstract">Retrons are bacterial immune systems that use reverse transcribed DNA as a detector of phage infection. They are also increasingly deployed as a component of biotechnology. For genome editing, for instance, retrons are modified so that the reverse transcribed DNA (RT-DNA) encodes an editing donor. Retrons are commonly found in bacterial genomes; thousands of unique retrons have now been predicted bioinformatically. However, only a small number have been characterized experimentally. Here, we add substantially to the corpus of experimentally studied retrons. We synthesized &gt;100 previously untested retrons to identify the natural sequence of RT-DNA they produce, quantify their RT-DNA production, and test the relative efficacy of editing using retron-derived donors to edit bacterial genomes, phage genomes, and human genomes. We add 62 new empirically determined, natural RT-DNAs, which are not predictable from the retron sequence alone. We report a large diversity in RT-DNA production and editing rates across retrons, finding that top performing editors outperform those used in previous studies, and are drawn from a subset of the retron phylogeny.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575045">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575045" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575045">
        <p class="paperTitle">DNA microbeads for spatio-temporally controlled morphogen release within organoids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575045" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575045" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Afting, C.; Walther, T.; Wittbrodt, J.; Göpfrich, K.</p>
        <p class="info">Score: 17.5, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575045' target='https://doi.org/10.1101/2024.01.10.575045'> 10.1101/2024.01.10.575045</a></p>
        <p class="abstract">Organoids have proven to be powerful in vitro model systems that mimic features of the corresponding tissue in vivo. However, across tissue types and species, organoids still often fail to reach full maturity and function, because biochemical cues cannot be provided from within the organoid to guide their development. The establishment of such tools has been identified as a major goal of the field. Here, we introduce DNA microbeads as a novel tool for implementing spatio-temporally controlled morphogen gradients inside of organoids at any point in their life cycle. The DNA microbeads are formed in a simple one-pot process, they can be stored for a year and their viscoelastic behavior and surface modification is tunable to mimic the corresponding tissue. Employing medaka retinal organoids and early embryos, we show that DNA microbeads can be integrated into embryos and organoids by microinjection and erased in a non-invasive manner with light. Coupling a recombinant surrogate Wnt to the DNA microbeads we demonstrate the spatio-temporally controlled release of the morphogen from the microinjection site, which leads to the formation of retinal pigmented epithelium while maintaining neuroretinal ganglion cells. We were thus able to bioengineer retinal organoids to more closely mirror the cell type diversity of in vivo retinas. The DNA microbead technology can easily be adapted to other organoid applications for improved tissue mimicry.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.575834">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.575834" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.575834">
        <p class="paperTitle">Treatment of a genetic liver disease in mice through transient prime editor expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.575834" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.575834" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rothgangl, T.; Ioannidi, E. I.; Weber, Y.; Talas, A.; Boeck, D.; Matsushita, M.; Villiger, E. A.; Schmidheini, L.; Moon, J.; Lin, P. J. C.; Fan, S. H. Y.; Marquart, K. F.; Schwerdel, C.; Rimann, N.; Faccin, E.; Cremonesi, A.; Villiger, L.; Muramatsu, H.; Vadovics, M.; Thoeny, B.; Kopf, M.; Haeberle, J.; Pardi, N.; Tam, Y. K.; Schwank, G.</p>
        <p class="info">Score: 9.2, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.575834' target='https://doi.org/10.1101/2024.01.22.575834'> 10.1101/2024.01.22.575834</a></p>
        <p class="abstract">Prime editing is a versatile genome editing technology that does not rely on DNA double-strand break formation and homology-directed repair (HDR). This makes it a promising tool for correcting pathogenic mutations in tissues consisting predominantly of postmitotic cells, such as the liver. While recent studies have already demonstrated proof-of-concept for in vivo prime editing, the use of viral delivery vectors resulted in prolonged prime editor (PE) expression, posing challenges for clinical application. Here, we developed an in vivo prime editing approach where we delivered the pegRNA using self-complementary adeno-associated viral (scAAV) vectors and the prime editor using nucleoside-modified mRNA encapsulated in lipid nanoparticles (LNPs). This methodology led to transient expression of the PE for 48h and 26% editing at the Dnmt1 locus using AAV doses of 2.5x1013 vector genomes (vg)/kg and a single dose of 3mg/kg mRNA-LNP. When targeting the pathogenic mutation in the Pahenu2 mouse model of phenylketonuria (PKU), we achieved 4.3% gene correction using an AAV dose of 2.5x1013 vg/kg and three doses of 2 mg/kg mRNA-LNP. Editing was specific to the liver and the intended locus, and was sufficient to reduce blood L-phenylalanine (Phe) levels from over 1500 umol/l to below the therapeutic threshold of 600 umol/l. Our study demonstrates the feasibility of in vivo gene correction in the liver with transient PE expression, bringing prime editing closer to clinical application.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576326">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576326" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576326">
        <p class="paperTitle">Quantification of local matrix deposition during muscle stem cell activation using engineered hydrogels</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576326" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576326" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duran, P.; Yang, B. A.; Plaster, E.; Eiken, M.; Loebel, C.; Aguilar, C. A.</p>
        <p class="info">Score: 5.7, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576326' target='https://doi.org/10.1101/2024.01.20.576326'> 10.1101/2024.01.20.576326</a></p>
        <p class="abstract">Adult stem cells occupy a niche that contributes to their function, but how stem cells remodel their microenvironment remains an open-ended question. Herein, biomaterials-based systems and metabolic labeling were utilized to evaluate how skeletal muscle stem cells deposit extracellular matrix. Muscle stem cells and committed myoblasts were observed to generate less nascent matrix than muscle resident fibro-adipogenic progenitors. When cultured on substrates that matched the stiffness of physiological uninjured and injured muscles, the increased nascent matrix deposition was associated with stem cell activation. Reducing the ability to deposit nascent matrix in muscle stem cells attenuated function and mimicked impairments observed from muscle stem cells isolated from old aged muscles, which could be rescued with therapeutic supplementation of insulin-like growth factors. These results highlight how nascent matrix production is critical for maintaining healthy stem cell function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576297">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576297" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576297">
        <p class="paperTitle">Automated Synthesis of Wireframe DNA Nanotubes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576297" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576297" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Islas, P.; Platnich, C.; Gidi, Y.; Karimi, R.; Ginot, L.; Cosa, G.; Sleiman, H.</p>
        <p class="info">Score: 4.0, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576297' target='https://doi.org/10.1101/2024.01.18.576297'> 10.1101/2024.01.18.576297</a></p>
        <p class="abstract">DNA nanotechnology has revolutionized our ability to position matter at the nanoscale, but the preparation of DNA-based architectures remains laborious. To facilitate the formation of custom structures, we report a fully automated method to produce sequence- and size-defined DNA nanotubes. By programming the sequential addition of desired building blocks, rigid DX-tile-based DNA nanotubes (and flexible wireframe DNA structures) are attained, where the total number of possible constructs increases as a power function of the number of different units available. Using single-molecule fluorescence imaging, the kinetics and yield of each synthetic step can be quantitatively determined, revealing differences in self-assembly dynamics as the nanotube is built up from the solid support and providing new insights into DNA self-assembly. The exploitation of automation for both assembly and analysis (tthrough an ad-hoc developed K-means clustering algorithm) facilitates a workflow wherein the synthesis parameters may be iteratively improved upon, demonstrating how a single-molecule  assembly-analysis-optimization sequence can be used to generate complex, non-covalent materials in good yield. The presented synthetic strategy is generalizable, making use of equipment already available in most standard laboratories and represents the first fully automated supramolecular assembly on a solid support.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576049">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576049" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576049">
        <p class="paperTitle">Epigenetic control of multiple genes with a single lentiviral vector encoding transcriptional repressors fused to compact zinc finger arrays</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576049" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576049" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Monteferrario, D.; David, M.; Tadi, S. K.; Zhou, Y.; Marchetti, I.; Jeanneau, C.; Saviane, G.; Dupont, C. F.; Martelli, A. E.; Truong, L.; Eshleman, J.; Ng, C.; Huston, M.; Davis, G. D.; Fontenot, J. D.; Reik, A.; de la Rosa, M.; Fenard, D.</p>
        <p class="info">Score: 4.0, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576049' target='https://doi.org/10.1101/2024.01.17.576049'> 10.1101/2024.01.17.576049</a></p>
        <p class="abstract">Gene silencing without gene editing holds great potential for the development of safe therapeutic applications. Here, we describe a novel strategy to concomitantly repress multiple genes using zinc finger proteins fused to Kruppel-Associated Box repression domains (ZF-Rs). This was achieved via the optimization of a lentiviral system tailored for the delivery of ZF-Rs in hematopoietic cells. We showed that an optimal design of the lentiviral backbone is crucial to multiplex up to three ZF-Rs or two ZF-Rs and a chimeric antigen receptor. ZF-R expression had no impact on the integrity and functionality of transduced cells. Furthermore, gene repression in ZF-R-expressing T cells was highly efficient in vitro and in vivo during the entire monitoring period (up to ten weeks), and it was accompanied by epigenetic remodeling events. Finally, we described an approach to improve ZF-R specificity to illustrate the path towards the generation of ZF-Rs with a safe clinical profile. In conclusion, we successfully developed an epigenetic-based cell engineering approach for concomitant modulation of multiple gene expressions that bypass the risks associated with DNA editing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577324">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577324" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577324">
        <p class="paperTitle">Human vascular organoids with a mosaic AKT1 mutation recapitulate Proteus syndrome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577324" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577324" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: He, S.; Zhu, Y.; Chauhan, S.; Tavakol, D. N.; Lee, J. H.; Berris, R. B.-L.; Xu, C.; Lee, J.; Lee, C.; Cai, S.; McElroy, S.; Vunjak-Novakovic, G.; Tomer, R.; Azizi, E.; Xu, B.; Lao, Y.-H.; Leong, K. W.</p>
        <p class="info">Score: 2.5, Published: 2024-01-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577324' target='https://doi.org/10.1101/2024.01.26.577324'> 10.1101/2024.01.26.577324</a></p>
        <p class="abstract">Vascular malformation, a key clinical phenotype of Proteus syndrome, lacks effective models for pathophysiological study and drug development due to limited patient sample access. To bridge this gap, we built a human vascular organoid model replicating Proteus syndrome&#39;s vasculature. Using CRISPR/Cas9 genome editing and gene overexpression, we created induced pluripotent stem cells (iPSCs) embodying the Proteus syndrome-specific AKTE17K point mutation for organoid generation. Our findings revealed that AKT overactivation in these organoids resulted in smaller sizes yet increased vascular connectivity, although with less stable connections. This could be due to the significant vasculogenesis induced by AKT overactivation. This phenomenon likely stems from boosted vasculogenesis triggered by AKT overactivation, leading to increased vascular sprouting. Additionally, a notable increase in dysfunctional PDGFRbeta&#43; mural cells, impaired in matrix secretion, was observed in these AKT-overactivated organoids. The application of AKT inhibitors (ARQ092, AZD5363, or GDC0068) reversed the vascular malformations; the inhibitors&#39; effectiveness was directly linked to reduced connectivity in the organoids. In summary, our study introduces an innovative in vitro model combining organoid technology and gene editing to explore vascular pathophysiology in Proteus syndrome. This model not only simulates Proteus syndrome vasculature but also holds potential for mimicking vasculatures of other genetically driven diseases. It represents an advance in drug development for rare diseases, historically plagued by slow progress.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576136">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576136" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576136">
        <p class="paperTitle">Microfluidic platform enables shear-less aerosolization of lipid nanoparticles for messenger RNA inhalation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576136" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576136" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, J.; Jozic&#39;, A.; Bloom, E.; Jones, B.; Marra, M.; Murthy, N. T. V.; Eygeris, Y.; Sahay, G.</p>
        <p class="info">Score: 3.8, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576136' target='https://doi.org/10.1101/2024.01.17.576136'> 10.1101/2024.01.17.576136</a></p>
        <p class="abstract">Leveraging the extensive surface area of the lungs for gene therapy, inhalation route offers distinct advantages for delivery. Clinical nebulizers that employ vibrating mesh technology are the standard choice for converting liquid medicines into aerosols. However, they have limitations when it comes to delivering mRNA through inhalation, including severe damage to nanoparticles due to shearing forces. Here, we introduce a novel microfluidic aerosolization platform (MAP) that preserves the structural and physicochemical integrity of lipid nanoparticles, enabling safe and efficient mRNA delivery to the respiratory system. Our results demonstrated the superiority of the novel MAP over the conventional vibrating mesh nebulizer, as it avoided problems such as particle aggregation, loss of mRNA encapsulation, and deformation of nanoparticle morphology. Notably, aerosolized nanoparticles generated by the microfluidic device led to enhanced transfection efficiency across various cell lines. In vivo experiments with mice that inhaled these aerosolized nanoparticles revealed successful, lung-specific mRNA transfection without observable signs of toxicity. This pioneering MAP represents a significant advancement for the pulmonary gene therapy, enabling precise and effective delivery of aerosolized nanoparticles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575724">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575724" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575724">
        <p class="paperTitle">Generative AI for Cardiac Organoid Florescence Generation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575724" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575724" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kandula, A. K. R.; Phamornratanakun, T.; Gomez, A. H.; Bhoi, R.; El-Mokahal, M.; Feng, Y.; Yang, H.</p>
        <p class="info">Score: 4.5, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575724' target='https://doi.org/10.1101/2024.01.15.575724'> 10.1101/2024.01.15.575724</a></p>
        <p class="abstract">hPSC-derived cardiac organoid is the most recent three-dimensional tissue structure that mimics the structure and functionality of the human heart and plays a pivotal role in modeling heart development and disease. The hPSC-derived cardiac organoids are commonly characterized by bright-field microscopic imaging. Although the bright-field microscope provides essential information about hPSC-derived cardiac organoids, such as morphology, size, and general structure, it does not extend our understanding of cardiac organoids on cell type-specific distribution and structure. Then, fluorescence microscopic imaging is required to identify the specific cardiovascular cell types in the hPSC-derived cardiac organoids by fluorescence immunostaining of fixed organoid samples or fluorescence reporter imaging of live organoids. Both approaches require extra steps of experiments and do not provide general information on all hPSC-derived cardiac organoids from different hPSC lines, which limits the biomedical applications of hPSC-derived cardiac organoids. This research addresses this limitation by proposing a comprehensive workflow for colorizing grayscale cardiac organoid images from bright-field microscopic imaging using conditional Generative Adversarial Networks (GANs) to provide cell type-specific information. By infusing these grayscale images with accurate fluorescence colorization, our approach aims to unlock the hidden wealth of cell type, distribution, and network, for bettering characterizing hPSC-derived cardiac organoids.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
