<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="PRO-LDM: Protein Sequence Generation with Conditional Latent Diffusion Models
Authors: Jiang, Z.; Zhang, S.; Huang, R.; Mo, S.; Zhu, L.; Li, P.; Zhang, Z.; Chen, X.; Long, Y.; Xu, R.; Qing, R.
Score: 14.4, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554145
AbstractsProtein design with deep-learning based algorithms represents an emerging and highly promising field in molecular biology. Yet it remains a challenging task due to the complexity of protein sequences. Recent developments in deep generative models such as diffusion model have demonstrated impressive performance in protein structure design, yet left much to be desired in creating new sequences.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="PRO-LDM: Protein Sequence Generation with Conditional Latent Diffusion Models
Authors: Jiang, Z.; Zhang, S.; Huang, R.; Mo, S.; Zhu, L.; Li, P.; Zhang, Z.; Chen, X.; Long, Y.; Xu, R.; Qing, R.
Score: 14.4, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554145
AbstractsProtein design with deep-learning based algorithms represents an emerging and highly promising field in molecular biology. Yet it remains a challenging task due to the complexity of protein sequences. Recent developments in deep generative models such as diffusion model have demonstrated impressive performance in protein structure design, yet left much to be desired in creating new sequences." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-27T10:37:02+00:00" />
<meta property="article:modified_time" content="2023-08-27T10:37:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="PRO-LDM: Protein Sequence Generation with Conditional Latent Diffusion Models
Authors: Jiang, Z.; Zhang, S.; Huang, R.; Mo, S.; Zhu, L.; Li, P.; Zhang, Z.; Chen, X.; Long, Y.; Xu, R.; Qing, R.
Score: 14.4, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554145
AbstractsProtein design with deep-learning based algorithms represents an emerging and highly promising field in molecular biology. Yet it remains a challenging task due to the complexity of protein sequences. Recent developments in deep generative models such as diffusion model have demonstrated impressive performance in protein structure design, yet left much to be desired in creating new sequences."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "PRO-LDM: Protein Sequence Generation with Conditional Latent Diffusion Models\nAuthors: Jiang, Z.; Zhang, S.; Huang, R.; Mo, S.; Zhu, L.; Li, P.; Zhang, Z.; Chen, X.; Long, Y.; Xu, R.; Qing, R.\nScore: 14.4, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554145\nAbstractsProtein design with deep-learning based algorithms represents an emerging and highly promising field in molecular biology. Yet it remains a challenging task due to the complexity of protein sequences. Recent developments in deep generative models such as diffusion model have demonstrated impressive performance in protein structure design, yet left much to be desired in creating new sequences.",
  "keywords": [
    
  ],
  "articleBody": " PRO-LDM: Protein Sequence Generation with Conditional Latent Diffusion Models\nAuthors: Jiang, Z.; Zhang, S.; Huang, R.; Mo, S.; Zhu, L.; Li, P.; Zhang, Z.; Chen, X.; Long, Y.; Xu, R.; Qing, R.\nScore: 14.4, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554145\nAbstractsProtein design with deep-learning based algorithms represents an emerging and highly promising field in molecular biology. Yet it remains a challenging task due to the complexity of protein sequences. Recent developments in deep generative models such as diffusion model have demonstrated impressive performance in protein structure design, yet left much to be desired in creating new sequences. Here we present PRO-LDM: an efficient protein sequence design framework that utilizes diffusion model in the latent space, utilizing the joint autoencoder as the backbone, to capture latent variable distribution and generate meaningful embeddings of protein sequences. PRO-LDM is capable of (1) learning feature representations in natural proteins both at amino-acid level and sequence level; (2) generating new sequences with natural properties and high diversity; and (3) conditionally designing new protein sequences with designated fitness. Our model presents a feasible pathway a generalizable new tool to extract both physical and chemical properties of amino acids, as well as evolutionary information within the protein sequences, for protein sequence design and functional optimization.\nBioengineering of a human innervated cardiac muscle model\nAuthors: Schneider, L. V.; Bao, G.; Methi, A.; Jensen, O.; Schmoll, K. A.; Setya, M. G.; Sakib, M. S.; Fahud, A. L.; Brockmoeller, J.; Fischer, A.; Liaw, N. Y.; Zimmermann, W.-H.; Zafeiriou, M.-P.\nScore: 9.7, Published: 2023-08-19 DOI: 10.1101/2023.08.18.552653\nCardiac autonomic neurons control cardiac contractility. Dysregulation of the autonomic nervous system can lead to sympathetic overdrive resulting in heart failure and an increased incidence of fatal arrhythmias. Here, we introduce innervated engineered human myocardium (iEHM), a novel model of neuro-cardiac junctions, constructed by fusion of a bioengineered neural organoid (BENO) patterned to autonomic nervous system and engineered human myocardium (EHM). Projections of sympathetic neurons into engineered human myocardium formed presynaptic terminals in close proximity to cardiomyocytes and an extensive vascular network co-developing in the tissues. Contractile responses to optogenetic stimulation of the accordingly engineered neuronal component demonstrated functionality of neuro-cardiac junctions in iEHM. This model will serve as a human surrogate system to delineate neuron and cardiac cell contribution to brain and heart diseases and is an important step towards engineering a human brain to heart axis in a dish.\nOpen-source milligram-scale, four channel, automated protein purification system\nAuthors: Puccinelli, R. R.; Sama, S. S.; Worthington, C. M.; Puschnik, A. S.; Pak, J. E.; Gomez-Sjoberg, R.\nScore: 11.2, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552685\nLiquid chromatography purification of multiple recombinant proteins, in parallel, could catalyze research and discovery if the processes are fast and approach the robustness of traditional, \"one-protein-at-a-time\" purification. Here, we report an automated, four channel chromatography platform that we have designed and validated for parallelized protein purification at milligram scales. The device can purify up to four proteins (each with its own single column), has inputs for up to eight buffers or solvents that can be directed to any of the four columns via a network of software-driven valves, and includes an automated fraction collector with ten positions for 1.5 or 5.0 mL collection tubes and four positions for 50 mL collection tubes for each column output. The control software can be accessed either via Python scripting, giving users full access to all steps of the purification process, or via a simple-to-navigate touch screen graphical user interface that does not require knowledge of the command line or any programming language. Using our instrument, we report milligram-scale, parallelized, single-column purification of a panel of mammalian cell expressed coronavirus (SARS-CoV-2, HCoV-229E, HCoV-OC43, HCoV-229E) trimeric Spike and monomeric Receptor Binding Domain (RBD) antigens, and monoclonal antibodies targeting SARS-CoV-2 Spike (S) and Influenza Hemagglutinin (HA). We include a detailed hardware build guide, and have made the controlling software open source, to allow others to build and customize their own protein purifier systems.\nSecond Harmonic Generation Imaging Reveals Entanglement of Collagen Fibers in the Elephant Trunk Skin Dermis\nAuthors: Schulz, A.; Plotczyk, M.; Sordilla, S.; Boyle, M.; Singal, K.; Reidenberg, J.; Hu, D.; Higgins, C.\nScore: 11.5, Published: 2023-08-15 DOI: 10.1101/2023.08.11.553031\nForm-function relationships often have tradeoffs: if a material is tough, it is often inflexible, and vice versa. This is particularly relevant for the elephant trunk, where the skin should be protective yet elastic. To investigate how this is achieved, we used classical histochemical staining and second harmonic generation microscopy to describe the morphology and composition of elephant trunk skin. We report structure at the macro and micro scales, from the thickness of the dermis to the interaction of 10 m thick collagen fibers. We analyzed several sites along the length of the trunk, to compare and contrast the dorsal-ventral and proximal-distal skin morphologies and compositions. We find the dorsal skin of the elephant trunk can have keratin armor layers over 2mm thick, which is nearly 100 times the thickness of the equivalent layer in human skin. We also found that the structural support layer (the dermis) of elephant trunk contains a distribution of collagen-I (COL1) fibers in both perpendicular and parallel arrangement. The bimodal distribution of collagen is seen across all portions of the trunk, and is dissimilar from that of human skin where one orientation dominates within a body site. We hypothesize that this distribution of COL1 in the elephant trunk allows both flexibility and load-bearing capabilities. Additionally, when viewing individual fiber interaction of 10 m thick collagen, we find the fiber crossings per unit volume are five times more common than in human skin, suggesting that the fibers are entangled. We surmise that these intriguing structures permit both flexibility and strength in the elephant trunk. The complex nature of the elephant skin may inspire the design of materials that can combine strength and flexibility.\nDNA Nanostructure-Guided Assembly of Proteins into Programmable Shapes\nAuthors: Lu, Q.; Xu, Y.; Poppleton, E.; Zhou, K.; Sulc, P.; Stephanopoulos, N.; Ke, Y.\nScore: 5.3, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552347\nThe development of methods to synthesize protein oligomers with precisely controlled number and configuration of components will enable artificial protein complexes and nanostructures with diverse biological and medical applications. Using DNA hybridization to link protein-DNA conjugates provides for a programmability that can be difficult to achieve with other methods, such as engineering multiple orthogonal protein-protein interfaces. However, it is still difficult to construct well-defined protein assemblies, and the challenge is magnified if only a single type of building block is used. To overcome this hurdle, we use a DNA origami as an \"assembler\" to guide the linking of protein-DNA conjugates using a series of oligonucleotide hybridization and displacement operations. We constructed several isomeric protein nanostructures on a DNA origami platform by using a C3-symmetric building block comprised of a protein trimer modified with DNA handles. By changing the number of protein-DNA building blocks attached to the origami, and the sequence of the linking and displacement strands added, we were able to produce dimers, two types of trimer structures, and three types of tetramer assemblies. Our approach expands the scope for the precise design and assembly of protein-based nanostructures, and will enable the formulation of functional protein complexes with stoichiometric and geometric control.\nVolumetric imaging of an intact organism by a distributed molecular network\nAuthors: Weinstein, J. A.; Qian, N.\nScore: 42.1, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553025\nLymphatic, nervous, and tumoral tissues, among others, exhibit physiology that emerges from three-dimensional interactions between genetically unique cells. A technology capable of volumetrically imaging transcriptomes, genotypes, and morphologies in a single de novo measurement would therefore provide a critical view into the biological complexity of living systems. Here we achieve this by extending DNA microscopy, an imaging modality that encodes a spatio-genetic map of a specimen via a massive distributed network of DNA molecules inside it, to three dimensions and multiple length scales in developing zebrafish embryos.\nEngineering of a graphene oxide-based two-dimensional platform for immune activation and modulation\nAuthors: Despotopoulou, D.; Stylianou, M.; Arellano, L. M.; Kisby, T.; Lozano, N.; Kostarelos, K.\nScore: 2.0, Published: 2023-08-22 DOI: 10.1101/2023.08.22.553542\nNanoscale-based tools for immunomodulation are expected to offer a rich battery of options for more targeted and safer approaches to achieve clinically effective manipulation of the local and systemic immune environment. In this study, we aimed to design nanoscale constructs based on graphene oxide (GO) nanosheets as platform carriers for the TLR7/8 agonist Resiquimod (R848). The non-covalent complexation of R848 molecules on the GO surface resulted in stable complexes by preserving their biological activity. The physicochemical properties, molecular quantification, as well as the overall performance of the complex were systematically investigated. We hypothesized the formation of GO:drug nano-constructs with strong colloidal stability over time, due to the strong {pi}-{pi} interactions between the R848 molecules and the GO surface, and identified that R848 loading efficiency consistently ranged around 75% (of starting molecules), quantified by HPLC and UV-Vis. The 2D morphology of the thin nanosheets was retained after complexation, determined by various (AFM and SEM) microscopic techniques. Based on the surface physicochemical characterization of the complexes by Raman, FTIR, XPS, and XRD, the formation of non-covalent interactions among the GO surface and the R848 molecules was confirmed. Most importantly, GO:R848 complexes did not compromise the biological activity of R848, and effectively activated macrophages in vitro. Collectively, this study demonstrates that thin GO sheets can act as platforms for the non-covalent association with small TLR7/8 agonist molecules, forming stable and highly reproducible complexes, that could be exploited as effective immunomodulatory agents.\nSpatially resolved molecular analysis of host response to medical device implantation using the 3D OrbiSIMS\nAuthors: Suvannapruk, W.; Fisher, L. E.; Luckett, J. C.; Edney, M. K.; Kotowska, A. M.; Kim, D.-H.; Scurr, D. J.; Ghaemmaghami, A.; Alexander, M. R.\nScore: 2.0, Published: 2023-08-21 DOI: 10.1101/2023.08.18.553860\nA key goal for implanted medical devices is that they do not elicit a detrimental immune response. Macrophages play critical roles in modulation of the host immune response and are the major cells responsible for persistent inflammatory reactions to implanted biomaterials. We investigate two novel immune-instructive polymers that stimulate pro- or anti-inflammatory responses from macrophages in vitro. These also modulate in vivo foreign body responses (FBR) when implanted subcutaneously in mice as coatings on biomedical grade silicone rubber. The tissue surrounding the implant is mechanically sectioned and imaged to assess the response of the polymers compared to silicone rubber. Immunofluorescent staining reveals responses consistent with pro- or anti-inflammatory responses previously described for these polymers. We apply 3D OrbiSIMS analysis to provide spatial analysis of the metabolite signature in the tissue surrounding the implant for the first time, providing molecular histology insight into the metabolite response in the host tissue. For the pro-inflammatory coating, monoacylglycerols (MG) and diacylglycerols (DG) are observed at increased intensity, while for the anti-inflammatory coating the number of phospholipid species detected decrease and pyridine and pyrimidine levels were elevated. These findings link to observations of small molecule signature from single cell studies of M2 macrophages in vitro where cell and tissue ion intensities were found to correlate suggesting potential for prediction. This illustrates the power of metabolite characterization by the 3D OrbiSIMS to gain insight into the mechanism of bio-instructive materials as medical devices and to inform on the FBR to biomaterials.\nFrontal hip exoskeleton assistance does not appear promising for reducing the metabolic cost of walking: A preliminary experimental study\nAuthors: Kim, J.; Raitor, M.; Liu, C. K.; Collins, S. H.\nScore: 1.5, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554212\nBackgroundDuring walking, humans exert a substantial hip abduction moment to maintain balance and prevent pelvic drop. This significant torque requirement suggests that assisting the frontal hip muscles could be a promising strategy to reduce the energy expenditure associated with walking. A previous musculoskeletal simulation study also predicted that providing hip abduction assistance through an exoskeleton could potentially result in a large reduction in whole-body metabolic rate. However, to date, no study has experimentally assessed the metabolic cost of walking with frontal hip assistance. MethodsIn this case study involving a single subject (N = 1), a tethered hip exoskeleton emulator was used to assess the feasibility of reducing metabolic expenditure through frontal-plane hip assistance. Human-in-the-loop optimization was conducted separately under torque and position control to determine energetically optimal assistance parameters for each control scheme. ResultsThe optimized profiles in both control schemes did not reduce metabolic rate compared to walking with assistance turned off. The optimal peak torque magnitude was found to be close to zero, suggesting that any hip abduction torque would increase metabolic rate. Both bio-inspired and simulation-inspired profiles substantially increased metabolic cost. ConclusionFrontal hip assistance does not appear to be promising in reducing the metabolic rate of walking. This could be attributed to the need for maintaining balance, as humans may refrain from relaxing certain muscles as a precaution against unexpected disturbances during walking. An investigation of different control architectures is needed to determine if frontal-plane hip assistance can yield successful results.\nMetabolic Engineering on a 3D-Printed Microfluidic Platform: A New Approach for Modular Co-Metabolic pathways\nAuthors: Sun, Y.; Helalat, S. H.; Seder, I.; Tellez, R. C.; Amani, M.\nScore: 1.5, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554264\nMetabolic engineering of cell factories often requires extensive modification of host cellular machinery, leading to numerous challenges such as metabolic burden, intermediate metabolite toxicity, and inadequate endogenous fluxes. To overcome the limitations, we presented an innovative approach for metabolic engineering, by constructing modular biosynthetic pathways on a 3D-printed microfluidic platform. Several new techniques have been developed, including novel designs of chip configurations, effective methods for enzyme immobilization on printed resins, and proper ways to regenerate cofactors in redox reactions. As a proof of concept, we built xylose consumption and CO2 fixation pathways in the microfluidic chips and successfully demonstrated that the platform was able to convert xylose and enable the rapid growth of Saccharomyces cerevisiae, which otherwise will not grow with xylose as the only carbon source. Overall, the 3D-printed microfluidic platform presents a much simpler and more efficient cell-free strategy for developing modular, optimized biosynthetic pathways.\n",
  "wordCount" : "2328",
  "inLanguage": "en",
  "datePublished": "2023-08-27T10:37:02Z",
  "dateModified": "2023-08-27T10:37:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on August 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554145">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554145" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554145">
        <p class="paperTitle">PRO-LDM: Protein Sequence Generation with Conditional Latent Diffusion Models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554145" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554145" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, Z.; Zhang, S.; Huang, R.; Mo, S.; Zhu, L.; Li, P.; Zhang, Z.; Chen, X.; Long, Y.; Xu, R.; Qing, R.</p>
        <p class="info">Score: 14.4, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554145' target='https://doi.org/10.1101/2023.08.22.554145'> 10.1101/2023.08.22.554145</a></p>
        <p class="abstract">AbstractsProtein design with deep-learning based algorithms represents an emerging and highly promising field in molecular biology. Yet it remains a challenging task due to the complexity of protein sequences. Recent developments in deep generative models such as diffusion model have demonstrated impressive performance in protein structure design, yet left much to be desired in creating new sequences. Here we present PRO-LDM: an efficient protein sequence design framework that utilizes diffusion model in the latent space, utilizing the joint autoencoder as the backbone, to capture latent variable distribution and generate meaningful embeddings of protein sequences. PRO-LDM is capable of (1) learning feature representations in natural proteins both at amino-acid level and sequence level; (2) generating new sequences with natural properties and high diversity; and (3) conditionally designing new protein sequences with designated fitness. Our model presents a feasible pathway a generalizable new tool to extract both physical and chemical properties of amino acids, as well as evolutionary information within the protein sequences, for protein sequence design and functional optimization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.552653">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.552653" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.552653">
        <p class="paperTitle">Bioengineering of a human innervated cardiac muscle model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.552653" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.552653" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schneider, L. V.; Bao, G.; Methi, A.; Jensen, O.; Schmoll, K. A.; Setya, M. G.; Sakib, M. S.; Fahud, A. L.; Brockmoeller, J.; Fischer, A.; Liaw, N. Y.; Zimmermann, W.-H.; Zafeiriou, M.-P.</p>
        <p class="info">Score: 9.7, Published: 2023-08-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.552653' target='https://doi.org/10.1101/2023.08.18.552653'> 10.1101/2023.08.18.552653</a></p>
        <p class="abstract">Cardiac autonomic neurons control cardiac contractility. Dysregulation of the autonomic nervous system can lead to sympathetic overdrive resulting in heart failure and an increased incidence of fatal arrhythmias. Here, we introduce innervated engineered human myocardium (iEHM), a novel model of neuro-cardiac junctions, constructed by fusion of a bioengineered neural organoid (BENO) patterned to autonomic nervous system and engineered human myocardium (EHM). Projections of sympathetic neurons into engineered human myocardium formed presynaptic terminals in close proximity to cardiomyocytes and an extensive vascular network co-developing in the tissues. Contractile responses to optogenetic stimulation of the accordingly engineered neuronal component demonstrated functionality of neuro-cardiac junctions in iEHM. This model will serve as a human surrogate system to delineate neuron and cardiac cell contribution to brain and heart diseases and is an important step towards engineering a human brain to heart axis in a dish.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552685">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552685" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552685">
        <p class="paperTitle">Open-source milligram-scale, four channel, automated protein purification system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552685" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552685" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Puccinelli, R. R.; Sama, S. S.; Worthington, C. M.; Puschnik, A. S.; Pak, J. E.; Gomez-Sjoberg, R.</p>
        <p class="info">Score: 11.2, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552685' target='https://doi.org/10.1101/2023.08.09.552685'> 10.1101/2023.08.09.552685</a></p>
        <p class="abstract">Liquid chromatography purification of multiple recombinant proteins, in parallel, could catalyze research and discovery if the processes are fast and approach the robustness of traditional, &#34;one-protein-at-a-time&#34; purification. Here, we report an automated, four channel chromatography platform that we have designed and validated for parallelized protein purification at milligram scales. The device can purify up to four proteins (each with its own single column), has inputs for up to eight buffers or solvents that can be directed to any of the four columns via a network of software-driven valves, and includes an automated fraction collector with ten positions for 1.5 or 5.0 mL collection tubes and four positions for 50 mL collection tubes for each column output. The control software can be accessed either via Python scripting, giving users full access to all steps of the purification process, or via a simple-to-navigate touch screen graphical user interface that does not require knowledge of the command line or any programming language. Using our instrument, we report milligram-scale, parallelized, single-column purification of a panel of mammalian cell expressed coronavirus (SARS-CoV-2, HCoV-229E, HCoV-OC43, HCoV-229E) trimeric Spike and monomeric Receptor Binding Domain (RBD) antigens, and monoclonal antibodies targeting SARS-CoV-2 Spike (S) and Influenza Hemagglutinin (HA). We include a detailed hardware build guide, and have made the controlling software open source, to allow others to build and customize their own protein purifier systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553031">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553031" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553031">
        <p class="paperTitle">Second Harmonic Generation Imaging Reveals Entanglement of Collagen Fibers in the Elephant Trunk Skin Dermis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553031" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553031" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schulz, A.; Plotczyk, M.; Sordilla, S.; Boyle, M.; Singal, K.; Reidenberg, J.; Hu, D.; Higgins, C.</p>
        <p class="info">Score: 11.5, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553031' target='https://doi.org/10.1101/2023.08.11.553031'> 10.1101/2023.08.11.553031</a></p>
        <p class="abstract">Form-function relationships often have tradeoffs: if a material is tough, it is often inflexible, and vice versa. This is particularly relevant for the elephant trunk, where the skin should be protective yet elastic. To investigate how this is achieved, we used classical histochemical staining and second harmonic generation microscopy to describe the morphology and composition of elephant trunk skin. We report structure at the macro and micro scales, from the thickness of the dermis to the interaction of 10 m thick collagen fibers. We analyzed several sites along the length of the trunk, to compare and contrast the dorsal-ventral and proximal-distal skin morphologies and compositions. We find the dorsal skin of the elephant trunk can have keratin armor layers over 2mm thick, which is nearly 100 times the thickness of the equivalent layer in human skin. We also found that the structural support layer (the dermis) of elephant trunk contains a distribution of collagen-I (COL1) fibers in both perpendicular and parallel arrangement. The bimodal distribution of collagen is seen across all portions of the trunk, and is dissimilar from that of human skin where one orientation dominates within a body site. We hypothesize that this distribution of COL1 in the elephant trunk allows both flexibility and load-bearing capabilities. Additionally, when viewing individual fiber interaction of 10 m thick collagen, we find the fiber crossings per unit volume are five times more common than in human skin, suggesting that the fibers are entangled. We surmise that these intriguing structures permit both flexibility and strength in the elephant trunk. The complex nature of the elephant skin may inspire the design of materials that can combine strength and flexibility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552347">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552347" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552347">
        <p class="paperTitle">DNA Nanostructure-Guided Assembly of Proteins into Programmable Shapes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552347" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552347" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lu, Q.; Xu, Y.; Poppleton, E.; Zhou, K.; Sulc, P.; Stephanopoulos, N.; Ke, Y.</p>
        <p class="info">Score: 5.3, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552347' target='https://doi.org/10.1101/2023.08.07.552347'> 10.1101/2023.08.07.552347</a></p>
        <p class="abstract">The development of methods to synthesize protein oligomers with precisely controlled number and configuration of components will enable artificial protein complexes and nanostructures with diverse biological and medical applications. Using DNA hybridization to link protein-DNA conjugates provides for a programmability that can be difficult to achieve with other methods, such as engineering multiple orthogonal protein-protein interfaces. However, it is still difficult to construct well-defined protein assemblies, and the challenge is magnified if only a single type of building block is used. To overcome this hurdle, we use a DNA origami as an &#34;assembler&#34; to guide the linking of protein-DNA conjugates using a series of oligonucleotide hybridization and displacement operations. We constructed several isomeric protein nanostructures on a DNA origami platform by using a C3-symmetric building block comprised of a protein trimer modified with DNA handles. By changing the number of protein-DNA building blocks attached to the origami, and the sequence of the linking and displacement strands added, we were able to produce dimers, two types of trimer structures, and three types of tetramer assemblies. Our approach expands the scope for the precise design and assembly of protein-based nanostructures, and will enable the formulation of functional protein complexes with stoichiometric and geometric control.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553025">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553025" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553025">
        <p class="paperTitle">Volumetric imaging of an intact organism by a distributed molecular network</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553025" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553025" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weinstein, J. A.; Qian, N.</p>
        <p class="info">Score: 42.1, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553025' target='https://doi.org/10.1101/2023.08.11.553025'> 10.1101/2023.08.11.553025</a></p>
        <p class="abstract">Lymphatic, nervous, and tumoral tissues, among others, exhibit physiology that emerges from three-dimensional interactions between genetically unique cells. A technology capable of volumetrically imaging transcriptomes, genotypes, and morphologies in a single de novo measurement would therefore provide a critical view into the biological complexity of living systems. Here we achieve this by extending DNA microscopy, an imaging modality that encodes a spatio-genetic map of a specimen via a massive distributed network of DNA molecules inside it, to three dimensions and multiple length scales in developing zebrafish embryos.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.553542">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.553542" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.553542">
        <p class="paperTitle">Engineering of a graphene oxide-based two-dimensional platform for immune activation and modulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.553542" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.553542" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Despotopoulou, D.; Stylianou, M.; Arellano, L. M.; Kisby, T.; Lozano, N.; Kostarelos, K.</p>
        <p class="info">Score: 2.0, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.553542' target='https://doi.org/10.1101/2023.08.22.553542'> 10.1101/2023.08.22.553542</a></p>
        <p class="abstract">Nanoscale-based tools for immunomodulation are expected to offer a rich battery of options for more targeted and safer approaches to achieve clinically effective manipulation of the local and systemic immune environment. In this study, we aimed to design nanoscale constructs based on graphene oxide (GO) nanosheets as platform carriers for the TLR7/8 agonist Resiquimod (R848). The non-covalent complexation of R848 molecules on the GO surface resulted in stable complexes by preserving their biological activity. The physicochemical properties, molecular quantification, as well as the overall performance of the complex were systematically investigated. We hypothesized the formation of GO:drug nano-constructs with strong colloidal stability over time, due to the strong {pi}-{pi} interactions between the R848 molecules and the GO surface, and identified that R848 loading efficiency consistently ranged around 75% (of starting molecules), quantified by HPLC and UV-Vis. The 2D morphology of the thin nanosheets was retained after complexation, determined by various (AFM and SEM) microscopic techniques. Based on the surface physicochemical characterization of the complexes by Raman, FTIR, XPS, and XRD, the formation of non-covalent interactions among the GO surface and the R848 molecules was confirmed. Most importantly, GO:R848 complexes did not compromise the biological activity of R848, and effectively activated macrophages in vitro. Collectively, this study demonstrates that thin GO sheets can act as platforms for the non-covalent association with small TLR7/8 agonist molecules, forming stable and highly reproducible complexes, that could be exploited as effective immunomodulatory agents.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553860">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553860" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553860">
        <p class="paperTitle">Spatially resolved molecular analysis of host response to medical device implantation using the 3D OrbiSIMS</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553860" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553860" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Suvannapruk, W.; Fisher, L. E.; Luckett, J. C.; Edney, M. K.; Kotowska, A. M.; Kim, D.-H.; Scurr, D. J.; Ghaemmaghami, A.; Alexander, M. R.</p>
        <p class="info">Score: 2.0, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553860' target='https://doi.org/10.1101/2023.08.18.553860'> 10.1101/2023.08.18.553860</a></p>
        <p class="abstract">A key goal for implanted medical devices is that they do not elicit a detrimental immune response. Macrophages play critical roles in modulation of the host immune response and are the major cells responsible for persistent inflammatory reactions to implanted biomaterials. We investigate two novel immune-instructive polymers that stimulate pro- or anti-inflammatory responses from macrophages in vitro. These also modulate in vivo foreign body responses (FBR) when implanted subcutaneously in mice as coatings on biomedical grade silicone rubber. The tissue surrounding the implant is mechanically sectioned and imaged to assess the response of the polymers compared to silicone rubber. Immunofluorescent staining reveals responses consistent with pro- or anti-inflammatory responses previously described for these polymers. We apply 3D OrbiSIMS analysis to provide spatial analysis of the metabolite signature in the tissue surrounding the implant for the first time, providing molecular histology insight into the metabolite response in the host tissue. For the pro-inflammatory coating, monoacylglycerols (MG) and diacylglycerols (DG) are observed at increased intensity, while for the anti-inflammatory coating the number of phospholipid species detected decrease and pyridine and pyrimidine levels were elevated. These findings link to observations of small molecule signature from single cell studies of M2 macrophages in vitro where cell and tissue ion intensities were found to correlate suggesting potential for prediction. This illustrates the power of metabolite characterization by the 3D OrbiSIMS to gain insight into the mechanism of bio-instructive materials as medical devices and to inform on the FBR to biomaterials.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554212">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554212" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554212">
        <p class="paperTitle">Frontal hip exoskeleton assistance does not appear promising for reducing the metabolic cost of walking: A preliminary experimental study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554212" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554212" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, J.; Raitor, M.; Liu, C. K.; Collins, S. H.</p>
        <p class="info">Score: 1.5, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554212' target='https://doi.org/10.1101/2023.08.22.554212'> 10.1101/2023.08.22.554212</a></p>
        <p class="abstract">BackgroundDuring walking, humans exert a substantial hip abduction moment to maintain balance and prevent pelvic drop. This significant torque requirement suggests that assisting the frontal hip muscles could be a promising strategy to reduce the energy expenditure associated with walking. A previous musculoskeletal simulation study also predicted that providing hip abduction assistance through an exoskeleton could potentially result in a large reduction in whole-body metabolic rate. However, to date, no study has experimentally assessed the metabolic cost of walking with frontal hip assistance.

MethodsIn this case study involving a single subject (N = 1), a tethered hip exoskeleton emulator was used to assess the feasibility of reducing metabolic expenditure through frontal-plane hip assistance. Human-in-the-loop optimization was conducted separately under torque and position control to determine energetically optimal assistance parameters for each control scheme.

ResultsThe optimized profiles in both control schemes did not reduce metabolic rate compared to walking with assistance turned off. The optimal peak torque magnitude was found to be close to zero, suggesting that any hip abduction torque would increase metabolic rate. Both bio-inspired and simulation-inspired profiles substantially increased metabolic cost.

ConclusionFrontal hip assistance does not appear to be promising in reducing the metabolic rate of walking. This could be attributed to the need for maintaining balance, as humans may refrain from relaxing certain muscles as a precaution against unexpected disturbances during walking. An investigation of different control architectures is needed to determine if frontal-plane hip assistance can yield successful results.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554264">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554264" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554264">
        <p class="paperTitle">Metabolic Engineering on a 3D-Printed Microfluidic Platform: A New Approach for Modular Co-Metabolic pathways</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554264" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554264" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sun, Y.; Helalat, S. H.; Seder, I.; Tellez, R. C.; Amani, M.</p>
        <p class="info">Score: 1.5, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554264' target='https://doi.org/10.1101/2023.08.22.554264'> 10.1101/2023.08.22.554264</a></p>
        <p class="abstract">Metabolic engineering of cell factories often requires extensive modification of host cellular machinery, leading to numerous challenges such as metabolic burden, intermediate metabolite toxicity, and inadequate endogenous fluxes. To overcome the limitations, we presented an innovative approach for metabolic engineering, by constructing modular biosynthetic pathways on a 3D-printed microfluidic platform. Several new techniques have been developed, including novel designs of chip configurations, effective methods for enzyme immobilization on printed resins, and proper ways to regenerate cofactors in redox reactions. As a proof of concept, we built xylose consumption and CO2 fixation pathways in the microfluidic chips and successfully demonstrated that the platform was able to convert xylose and enable the rapid growth of Saccharomyces cerevisiae, which otherwise will not grow with xylose as the only carbon source. Overall, the 3D-printed microfluidic platform presents a much simpler and more efficient cell-free strategy for developing modular, optimized biosynthetic pathways.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
