<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Human neuron subtype programming through combinatorial patterning with scRNA-seq readouts
Authors: Lin, H.-C.; Janssens, J.; Kroell, A.-S.; Hornauer, P.; Santel, M.; Okamoto, R.; Karava, K.; Priouret, M.; Garcia, M. P.; Schroeter, M.; Camp, J. G.; Treutlein, B.
Score: 47.2, Published: 2023-12-20 DOI: 10.1101/2023.12.12.571318
Human neurons programmed through transcription factor (TF) overexpression model neuronal differentiation and neurological diseases. However, programming specific neuron types remains challenging. Here, we modulate developmental signaling pathways combined with TF overexpression to explore the spectrum of neuron subtypes generated from pluripotent stem cells.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Human neuron subtype programming through combinatorial patterning with scRNA-seq readouts
Authors: Lin, H.-C.; Janssens, J.; Kroell, A.-S.; Hornauer, P.; Santel, M.; Okamoto, R.; Karava, K.; Priouret, M.; Garcia, M. P.; Schroeter, M.; Camp, J. G.; Treutlein, B.
Score: 47.2, Published: 2023-12-20 DOI: 10.1101/2023.12.12.571318
Human neurons programmed through transcription factor (TF) overexpression model neuronal differentiation and neurological diseases. However, programming specific neuron types remains challenging. Here, we modulate developmental signaling pathways combined with TF overexpression to explore the spectrum of neuron subtypes generated from pluripotent stem cells." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-27T10:39:30+00:00" />
<meta property="article:modified_time" content="2023-12-27T10:39:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Human neuron subtype programming through combinatorial patterning with scRNA-seq readouts
Authors: Lin, H.-C.; Janssens, J.; Kroell, A.-S.; Hornauer, P.; Santel, M.; Okamoto, R.; Karava, K.; Priouret, M.; Garcia, M. P.; Schroeter, M.; Camp, J. G.; Treutlein, B.
Score: 47.2, Published: 2023-12-20 DOI: 10.1101/2023.12.12.571318
Human neurons programmed through transcription factor (TF) overexpression model neuronal differentiation and neurological diseases. However, programming specific neuron types remains challenging. Here, we modulate developmental signaling pathways combined with TF overexpression to explore the spectrum of neuron subtypes generated from pluripotent stem cells."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Human neuron subtype programming through combinatorial patterning with scRNA-seq readouts\nAuthors: Lin, H.-C.; Janssens, J.; Kroell, A.-S.; Hornauer, P.; Santel, M.; Okamoto, R.; Karava, K.; Priouret, M.; Garcia, M. P.; Schroeter, M.; Camp, J. G.; Treutlein, B.\nScore: 47.2, Published: 2023-12-20 DOI: 10.1101/2023.12.12.571318\nHuman neurons programmed through transcription factor (TF) overexpression model neuronal differentiation and neurological diseases. However, programming specific neuron types remains challenging. Here, we modulate developmental signaling pathways combined with TF overexpression to explore the spectrum of neuron subtypes generated from pluripotent stem cells.",
  "keywords": [
    
  ],
  "articleBody": " Human neuron subtype programming through combinatorial patterning with scRNA-seq readouts\nAuthors: Lin, H.-C.; Janssens, J.; Kroell, A.-S.; Hornauer, P.; Santel, M.; Okamoto, R.; Karava, K.; Priouret, M.; Garcia, M. P.; Schroeter, M.; Camp, J. G.; Treutlein, B.\nScore: 47.2, Published: 2023-12-20 DOI: 10.1101/2023.12.12.571318\nHuman neurons programmed through transcription factor (TF) overexpression model neuronal differentiation and neurological diseases. However, programming specific neuron types remains challenging. Here, we modulate developmental signaling pathways combined with TF overexpression to explore the spectrum of neuron subtypes generated from pluripotent stem cells. We screened 480 morphogen signaling modulations coupled with NGN2 or ASCL1/DLX2 induction using a multiplexed single-cell transcriptomic readout. Analysis of 700,000 cells identified diverse excitatory and inhibitory neurons patterned along the anterior-posterior and dorsal-ventral axes of neural tube development. We inferred signaling and TF interaction networks guiding differentiation of forebrain, midbrain, hindbrain, spinal cord, peripheral sympathetic and sensory neurons. Our approach provides a strategy for cell subtype programming and to investigate how cooperative signaling drives neuronal fate.\nEncapsulation of AAVs into protein vault nanoparticles as a novel solution to gene therapy's neutralizing antibody problem\nAuthors: Collins, L. T.; Beatty, W.; Moyo, B.; Alves-Bezerra, M.; Hurley, A.; Lagor, W.; Bao, G.; Ponnazhagan, S.; McNally, R.; Rome, L.; Curiel, D.\nScore: 22.2, Published: 2023-12-19 DOI: 10.1101/2023.11.29.569229\nAlthough adeno-associated virus (AAV) has enjoyed enormous success as a delivery modality for gene therapy, it continues to suffer from the high prevalence of preexisting neutralizing antibodies in human populations, limiting who can receive potentially life-saving treatments. In this regard, AAV therapies generally also must be administered as a single dose since neutralizing antibodies develop in patients who receive the virus. Strategies for circumventing these issues remain limited. As a novel solution, we employed SpyTag-SpyCatcher molecular glue technology to facilitate packaging of AAVs inside of recombinant protein vault nanoparticles. Vaults are endogenous proteinaceous organelles produced by mammalian cells, so they are recognized as self by the immune system. They therefore can shield packaged molecules from neutralizing antibodies. Vault has previously been utilized to deliver drugs and proteins into cells, but our study represents the first time anyone has packaged an entire virus inside of vault. We showed that our vaultAAV (VAAV) delivery vehicle transduces cells in the presence of anti-AAV neutralizing serum. VAAV is positioned as a new gene therapy delivery platform with potential to overcome the neutralizing antibody problem and perhaps even allow administration of multiple doses, expanding the scope of AAV treatments.\nElectroadhesive hydrogel interface for prolonged mucosal theranostics\nAuthors: Ying, B.; Nan, K.; Zhu, Q.; Khuu, T.; Ro, H.; Qin, S.; Wang, S.; Jiang, K.; Chen, Y.; Bao, G.; Jenkins, J.; Pettinari, A.; Kuosmanen, J.; Ishida, K.; Lopez, A.; Li, J.; Hayward, A.; Langer, R.; Traverso, G.\nScore: 7.4, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572448\nEstablishing a robust and intimate mucosal interface that allows medical devices to remain within lumen-confined organs for extended periods has valuable applications, particularly for gastrointestinal (GI) theranostics. Here, we report the development of e-GLUE, an electroadhesive hydrogel interface for robust and prolonged mucosal retention following electrical activation. Notably, this novel mucosal adhesion mechanism can increase the adhesion energy of hydrogels on the mucosa by up to 30-fold and enable in vivo GI retention of e-GLUE devices for up to 30 days. Strong mucosal adhesion occurs within one minute of electrical activation, despite the presence of luminal fluid, mucus exposure, and organ motility, thereby ensuring compatibility with complex in vivo environments. In swine studies, we demonstrate the utility of e-GLUE for mucosal hemostasis, sustained local delivery of therapeutics, and intimate biosensing in the GI tract. This system can enable improved treatments for various health conditions, including gastrointestinal bleeding, inflammatory bowel disease, and diagnostic applications in the GI tract and beyond.\nSingle cell transcriptomics reveals chondrocyte differentiation dynamics in vivo and in vitro\nAuthors: Lawrence, J. E. G.; Woods, S.; Roberts, K.; Sumanaweera, D. N.; Balogh, P.; Predeus, A. V.; He, P.; Li, T.; Polanski, K.; Prigmore, E.; Tuck, E.; Mamanova, L.; Zhou, D.; Webb, S.; Jardine, L.; He, X.; Barker, R. A.; Haniffa, M.; Flanagan, A. M.; Young, M. D.; Behjati, S.; Bayraktar, O.; Kimber, S. J.; Teichmann, S.\nScore: 6.8, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572425\nThe consistent production of in vitro chondrocytes that faithfully recapitulate in vivo development would be of great benefit for musculoskeletal disease modelling and regenerative medicine. Current efforts are often limited by off-target differentiation, resulting in a heterogeneous product. Furthermore, the lack of comparison to human embryonic tissue, precludes detailed evaluation of in vitro cells. Here, we perform single-cell RNA sequencing of embryonic long bones dissected from first trimester hind limbs from a range of gestational ages. We combine this with publicly available data to form a detailed atlas of endochondral ossification, which we then use to evaluate a series of published in vitro chondrogenesis protocols, finding substantial variability in cell states produced by each. We apply single-nuclear RNA sequencing to one protocol to enable direct comparison between in vitro and in vivo, and perform trajectory alignment between the two to reveal differentiation dynamics at the single-cell level, shedding new light on off-target differentiation in vitro. Using this information, we inhibit the activity of FOXO1, a transcription factor predicted to be active in embryonic bone development and in chondrogenic cells in vitro, and increase chondrocyte transcripts in vitro. This study therefore presents a new framework for evaluating tissue engineering protocols, using single-cell data from human development to drive improvement and bring the prospect of true engineered cartilage closer to reality.\nCellular Senescence Impairs Tendon Extracellular Matrix Remodeling in Response to Mechanical Unloading\nAuthors: Stowe, E. J.; Keller, M. R.; Connizzo, B. K.\nScore: 6.4, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572594\nMusculoskeletal injuries, including tendinopathies, present a significant clinical burden for aging populations. While the biological drivers of age-related declines in tendon function are poorly understood, it is well accepted that dysregulation of extracellular matrix (ECM) remodeling plays a role in chronic tendon degeneration. Senescent cells, which have been associated with multiple degenerative pathologies in musculoskeletal tissues, secrete a highly pro-inflammatory senescence-associated secretory phenotype (SASP) that has potential to promote ECM breakdown. However, the role of senescent cells in the dysregulation of tendon ECM homeostasis is largely unknown. To assess this directly, we developed an in vitro model of induced cellular senescence in murine tendon explants. This novel technique enables us to study the isolated interactions of senescent cells and their native ECM without interference from age-related systemic changes. We document multiple biomarkers of cellular senescence in induced tendon explants including cell cycle arrest, apoptosis resistance, and SASP production. We then utilize this in vitro senescence model to compare the ECM remodeling response of young, naturally aged, and senescent tendons to an altered mechanical stimulus. We found that both senescence and aging independently led to alterations in ECM-related gene expression, reductions in protein synthesis, and tissue compositional changes. Furthermore, MMP activity was sustained, thus shifting the remodeling balance of aged and senescent tissues towards degradation over production. Together, this demonstrates that cellular senescence plays a role in the altered mechano-response of aged tendons and likely contributes to poor clinical outcomes in aging populations.\nAllosteric Modulation of Fluorescence Revealed by Hydrogen Bond Dynamics in a Genetically Encoded Maltose Biosensor\nAuthors: BERKSOZ, M.; ATILGAN, C.\nScore: 6.4, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572600\nGenetically encoded fluorescent biosensors (GEFBs) proved to be reliable tracers for many metabolites and cellular processes. In the simplest case, a fluorescent protein (FP) is genetically fused to a sensing protein which undergoes a conformational change upon ligand binding. This drives a rearrangement in the chromophore environment and changes the spectral properties of the FP. Structural determinants of successful biosensors are revealed only in hindsight when the crystal structures of both ligand-bound and ligand-free forms are available. This makes the development of new biosensors for desired analytes a long trial-and-error process. In the current study, we conducted {micro}s-long all atom molecular dynamics (MD) simulations of a maltose biosensor in both the apo (dark) and holo (bright) forms. We performed detailed hydrogen bond occupancy analyses to shed light on the mechanism of ligand induced conformational change in the sensor protein and its allosteric effect on the chromophore environment. We find that two strong indicators for distinguishing bright and dark states of biosensors are due to substantial changes in hydrogen bond dynamics in the system and solvent accessibility of the chromophore.\nReinforcement Learning for Control of Human Locomotion in Simulation\nAuthors: Dashkovets, A.; Laschowski, B.\nScore: 4.2, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572447\nControl of robotic leg prostheses and exoskeletons is an open challenge. Computer modeling and simulation can be used to study the dynamics and control of human walking and extract principles that can be programmed into robotic legs to behave similar to biological legs. In this study, we present the development of an efficient two-layer Q-learning algorithm, with k-d trees, that operates over continuous action spaces and a reward model that estimates the degree of muscle activation similarity between the agent and human state-to-action pairs and state-to-action sequences. We used a human musculoskeletal model acting in a high-dimensional, physics-based simulation environment to train and evaluate our algorithm to simulate biomimetic walking. We used imitation learning and artificial bio-mechanics data to accelerate training via expert demonstrations and used experimental human data to compare and validate our predictive simulations, achieving 79% accuracy. Also, when compared to the previous state-of-the-art that used deep deterministic policy gradient, our algorithm was significantly more efficient with lower computational and memory storage requirements (i.e., requiring 7 times less RAM and 87 times less CPU compute), which can benefit real-time embedded computing. Overall, our new two-layer Q-learning algorithm using sequential data for continuous imitation of human locomotion serves as a first step towards the development of bioinspired controllers for robotic prosthetic legs and exoskeletons. Future work will focus on improving the prediction accuracy compared to experimental data and expanding our simulations to other locomotor activities.\nBiodistribution of DNA-origami nanostructures in live zebrafish embryos with single-cell resolution\nAuthors: Kolonelou, C.; Bräutigam, L.; Edwards, S.; Engström, E.; Dias, J. M.; Spratt, J.; Karampelias, C.; Wennmalm, S.; Brismar, H.; Andersson, O.; Teixeira, A. I.\nScore: 4.1, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572973\nDNA origami-based nanotechnology is a versatile tool for exploring fundamental biological questions and holds significant promise for future biomedical applications. However, the development of DNA origami-based therapeutic agents is hindered by the challenge of translating in vitro performance into effective applications in vivo. Here, we exploit the optical transparency of the embryonic zebrafish to track intravenously injected, fluorescently labelled wireframe DNA origami nanostructures. Our approach integrated long-term, high-resolution imaging of transgenic live embryos with single-cell RNA sequencing, to elucidate the biodistribution of DNA nanostructures over time, up to 3 days post-injection (dpi). Notably, we observed rapid accumulation of nanostructures in the caudal hematopoietic tissue (CHT), akin to the fetal liver in mammals. We tested the effects of coating the nanostructures with an oligolysine PEG copolymer (K-PEG), a widely used strategy to enhance their stability. The K-PEG coating mitigated the accumulation rate in CHT, enabling higher percentages of the nanostructures to engage with other tissues. Additionally, our findings highlighted the pivotal role of scavenger endothelial cells in DNA origami clearance, with K-PEG offering sustained protection for the nanostructures at the CHT. Furthermore, by monitoring DNA origami in a transgenic zebrafish line designed for targeted macrophage ablation, we found that macrophages contribute to nanostructure clearance at later time points. This study introduces a framework for the analyses of the biodistribution and clearance of DNA origami nanostructures in vivo with single cell resolution and establishes a foundation for the investigation of DNA origami-based nanomedicines in animal models.\nMagnetoelectric Nanodiscs Enable Wireless Transgene-Free Neuromodulation\nAuthors: Kim, Y. J.; Driscoll, N.; Kent, N.; Vargas Paniagua, E.; Tabet, A.; Koehler, F.; Manthey, M.; Sahasrabudhe, A.; Signorelli, L.; Gregurec, D.; Anikeeva, P.\nScore: 2.5, Published: 2023-12-25 DOI: 10.1101/2023.12.24.573272\nDeep-brain stimulation (DBS) with implanted electrodes revolutionized treatment of movement disorders and empowered neuroscience studies. Identifying less invasive alternatives to DBS may further extend its clinical and research applications. Nanomaterial-mediated transduction of magnetic fields into electric potentials offers an alternative to invasive DBS. Here, we synthesize magnetoelectric nanodiscs (MENDs) with a core-double shell Fe3O4-CoFe2O4-BaTiO3 architecture with efficient magnetoelectric coupling. We find robust responses to magnetic field stimulation in neurons decorated with MENDs at a density of 1 {micro}g/mm2 despite individual-particle potentials below the neuronal excitation threshold. We propose a model for repetitive subthreshold depolarization, which combined with cable theory, corroborates our findings in vitro and informs magnetoelectric stimulation in vivo. MENDs injected into the ventral tegmental area of genetically intact mice at concentrations of 1 mg/mL enable remote control of reward behavior, setting the stage for mechanistic optimization of magnetoelectric neuromodulation and inspiring its future applications in fundamental and translational neuroscience.\nDevelopment of a robotic cluster for automated and scalable cell therapy manufacturing\nAuthors: Melocchi, A.; Schmittlein, B.; Jones, A. L.; Ainane, Y.; Rizvi, A.; Chan, D.; Dickey, E.; Pool, K.; Harsono, K.; Szymkiewicz, D.; Scarfogliero, U.; Bhatia, V.; Sivanantham, A.; Kreciglowa, N.; Hunter, A.; Gomez, M.; Tanner, A.; Uboldi, M.; Batish, A.; Balcerek, J.; Kutova-Stoilova, M.; Paruthiyil, S.; Acevedo, L. A.; Stadnitskiy, R.; Carmichael, S.; Aulbach, H.; Hewitt, M.; De Mollerat Du Jeu, X.; di Robilant, B.; Parietti, F.; Esensten, J. H.\nScore: 2.5, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572854\nThe production of commercial autologous cell therapies such as chimeric antigen receptor T cells requires complex manual manufacturing processes. Skilled labor costs and challenges in manufacturing scale-out have contributed to high prices for these products. Here, we present a robotic system that uses industry-standard cell therapy manufacturing equipment to automate the steps involved in cell therapy manufacturing. The robotic cluster consists of a robotic arm and customized modules, allowing the robot to manipulate a variety of standard cell therapy instruments and materials such as incubators, bioreactors, and reagent bags. This system enables existing manual manufacturing processes to be rapidly adapted to robotic manufacturing, without having to adopt a completely new technology platform. Proof-of-concept for the robotic cluster expansion module was demonstrated by activating and expanding human CD8+ T cells. The robotic cultures showed comparable cell yields, viability, and identity to those manually performed. Such modular robotic solutions may support scale-up and scale-out of cell therapies that are developed using classical manual methods in academic laboratories and biotechnology companies.\n",
  "wordCount" : "2370",
  "inLanguage": "en",
  "datePublished": "2023-12-27T10:39:30Z",
  "dateModified": "2023-12-27T10:39:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 27, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571318">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571318" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571318">
        <p class="paperTitle">Human neuron subtype programming through combinatorial patterning with scRNA-seq readouts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571318" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571318" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lin, H.-C.; Janssens, J.; Kroell, A.-S.; Hornauer, P.; Santel, M.; Okamoto, R.; Karava, K.; Priouret, M.; Garcia, M. P.; Schroeter, M.; Camp, J. G.; Treutlein, B.</p>
        <p class="info">Score: 47.2, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571318' target='https://doi.org/10.1101/2023.12.12.571318'> 10.1101/2023.12.12.571318</a></p>
        <p class="abstract">Human neurons programmed through transcription factor (TF) overexpression model neuronal differentiation and neurological diseases. However, programming specific neuron types remains challenging. Here, we modulate developmental signaling pathways combined with TF overexpression to explore the spectrum of neuron subtypes generated from pluripotent stem cells. We screened 480 morphogen signaling modulations coupled with NGN2 or ASCL1/DLX2 induction using a multiplexed single-cell transcriptomic readout. Analysis of 700,000 cells identified diverse excitatory and inhibitory neurons patterned along the anterior-posterior and dorsal-ventral axes of neural tube development. We inferred signaling and TF interaction networks guiding differentiation of forebrain, midbrain, hindbrain, spinal cord, peripheral sympathetic and sensory neurons. Our approach provides a strategy for cell subtype programming and to investigate how cooperative signaling drives neuronal fate.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569229">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569229" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569229">
        <p class="paperTitle">Encapsulation of AAVs into protein vault nanoparticles as a novel solution to gene therapy&#39;s neutralizing antibody problem</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569229" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569229" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Collins, L. T.; Beatty, W.; Moyo, B.; Alves-Bezerra, M.; Hurley, A.; Lagor, W.; Bao, G.; Ponnazhagan, S.; McNally, R.; Rome, L.; Curiel, D.</p>
        <p class="info">Score: 22.2, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569229' target='https://doi.org/10.1101/2023.11.29.569229'> 10.1101/2023.11.29.569229</a></p>
        <p class="abstract">Although adeno-associated virus (AAV) has enjoyed enormous success as a delivery modality for gene therapy, it continues to suffer from the high prevalence of preexisting neutralizing antibodies in human populations, limiting who can receive potentially life-saving treatments. In this regard, AAV therapies generally also must be administered as a single dose since neutralizing antibodies develop in patients who receive the virus. Strategies for circumventing these issues remain limited. As a novel solution, we employed SpyTag-SpyCatcher molecular glue technology to facilitate packaging of AAVs inside of recombinant protein vault nanoparticles. Vaults are endogenous proteinaceous organelles produced by mammalian cells, so they are recognized as self by the immune system. They therefore can shield packaged molecules from neutralizing antibodies. Vault has previously been utilized to deliver drugs and proteins into cells, but our study represents the first time anyone has packaged an entire virus inside of vault. We showed that our vaultAAV (VAAV) delivery vehicle transduces cells in the presence of anti-AAV neutralizing serum. VAAV is positioned as a new gene therapy delivery platform with potential to overcome the neutralizing antibody problem and perhaps even allow administration of multiple doses, expanding the scope of AAV treatments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572448">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572448" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572448">
        <p class="paperTitle">Electroadhesive hydrogel interface for prolonged mucosal theranostics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572448" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572448" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ying, B.; Nan, K.; Zhu, Q.; Khuu, T.; Ro, H.; Qin, S.; Wang, S.; Jiang, K.; Chen, Y.; Bao, G.; Jenkins, J.; Pettinari, A.; Kuosmanen, J.; Ishida, K.; Lopez, A.; Li, J.; Hayward, A.; Langer, R.; Traverso, G.</p>
        <p class="info">Score: 7.4, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572448' target='https://doi.org/10.1101/2023.12.19.572448'> 10.1101/2023.12.19.572448</a></p>
        <p class="abstract">Establishing a robust and intimate mucosal interface that allows medical devices to remain within lumen-confined organs for extended periods has valuable applications, particularly for gastrointestinal (GI) theranostics. Here, we report the development of e-GLUE, an electroadhesive hydrogel interface for robust and prolonged mucosal retention following electrical activation. Notably, this novel mucosal adhesion mechanism can increase the adhesion energy of hydrogels on the mucosa by up to 30-fold and enable in vivo GI retention of e-GLUE devices for up to 30 days. Strong mucosal adhesion occurs within one minute of electrical activation, despite the presence of luminal fluid, mucus exposure, and organ motility, thereby ensuring compatibility with complex in vivo environments. In swine studies, we demonstrate the utility of e-GLUE for mucosal hemostasis, sustained local delivery of therapeutics, and intimate biosensing in the GI tract. This system can enable improved treatments for various health conditions, including gastrointestinal bleeding, inflammatory bowel disease, and diagnostic applications in the GI tract and beyond.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572425">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572425" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572425">
        <p class="paperTitle">Single cell transcriptomics reveals chondrocyte differentiation dynamics in vivo and in vitro</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572425" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572425" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lawrence, J. E. G.; Woods, S.; Roberts, K.; Sumanaweera, D. N.; Balogh, P.; Predeus, A. V.; He, P.; Li, T.; Polanski, K.; Prigmore, E.; Tuck, E.; Mamanova, L.; Zhou, D.; Webb, S.; Jardine, L.; He, X.; Barker, R. A.; Haniffa, M.; Flanagan, A. M.; Young, M. D.; Behjati, S.; Bayraktar, O.; Kimber, S. J.; Teichmann, S.</p>
        <p class="info">Score: 6.8, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572425' target='https://doi.org/10.1101/2023.12.20.572425'> 10.1101/2023.12.20.572425</a></p>
        <p class="abstract">The consistent production of in vitro chondrocytes that faithfully recapitulate in vivo development would be of great benefit for musculoskeletal disease modelling and regenerative medicine. Current efforts are often limited by off-target differentiation, resulting in a heterogeneous product. Furthermore, the lack of comparison to human embryonic tissue, precludes detailed evaluation of in vitro cells. Here, we perform single-cell RNA sequencing of embryonic long bones dissected from first trimester hind limbs from a range of gestational ages. We combine this with publicly available data to form a detailed atlas of endochondral ossification, which we then use to evaluate a series of published in vitro chondrogenesis protocols, finding substantial variability in cell states produced by each. We apply single-nuclear RNA sequencing to one protocol to enable direct comparison between in vitro and in vivo, and perform trajectory alignment between the two to reveal differentiation dynamics at the single-cell level, shedding new light on off-target differentiation in vitro. Using this information, we inhibit the activity of FOXO1, a transcription factor predicted to be active in embryonic bone development and in chondrogenic cells in vitro, and increase chondrocyte transcripts in vitro. This study therefore presents a new framework for evaluating tissue engineering protocols, using single-cell data from human development to drive improvement and bring the prospect of true engineered cartilage closer to reality.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572594">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572594" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572594">
        <p class="paperTitle">Cellular Senescence Impairs Tendon Extracellular Matrix Remodeling in Response to Mechanical Unloading</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572594" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572594" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stowe, E. J.; Keller, M. R.; Connizzo, B. K.</p>
        <p class="info">Score: 6.4, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572594' target='https://doi.org/10.1101/2023.12.22.572594'> 10.1101/2023.12.22.572594</a></p>
        <p class="abstract">Musculoskeletal injuries, including tendinopathies, present a significant clinical burden for aging populations. While the biological drivers of age-related declines in tendon function are poorly understood, it is well accepted that dysregulation of extracellular matrix (ECM) remodeling plays a role in chronic tendon degeneration. Senescent cells, which have been associated with multiple degenerative pathologies in musculoskeletal tissues, secrete a highly pro-inflammatory senescence-associated secretory phenotype (SASP) that has potential to promote ECM breakdown. However, the role of senescent cells in the dysregulation of tendon ECM homeostasis is largely unknown. To assess this directly, we developed an in vitro model of induced cellular senescence in murine tendon explants. This novel technique enables us to study the isolated interactions of senescent cells and their native ECM without interference from age-related systemic changes. We document multiple biomarkers of cellular senescence in induced tendon explants including cell cycle arrest, apoptosis resistance, and SASP production. We then utilize this in vitro senescence model to compare the ECM remodeling response of young, naturally aged, and senescent tendons to an altered mechanical stimulus. We found that both senescence and aging independently led to alterations in ECM-related gene expression, reductions in protein synthesis, and tissue compositional changes. Furthermore, MMP activity was sustained, thus shifting the remodeling balance of aged and senescent tissues towards degradation over production. Together, this demonstrates that cellular senescence plays a role in the altered mechano-response of aged tendons and likely contributes to poor clinical outcomes in aging populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572600">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572600" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572600">
        <p class="paperTitle">Allosteric Modulation of Fluorescence Revealed by Hydrogen Bond Dynamics in a Genetically Encoded Maltose Biosensor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572600" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572600" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: BERKSOZ, M.; ATILGAN, C.</p>
        <p class="info">Score: 6.4, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572600' target='https://doi.org/10.1101/2023.12.20.572600'> 10.1101/2023.12.20.572600</a></p>
        <p class="abstract">Genetically encoded fluorescent biosensors (GEFBs) proved to be reliable tracers for many metabolites and cellular processes. In the simplest case, a fluorescent protein (FP) is genetically fused to a sensing protein which undergoes a conformational change upon ligand binding. This drives a rearrangement in the chromophore environment and changes the spectral properties of the FP. Structural determinants of successful biosensors are revealed only in hindsight when the crystal structures of both ligand-bound and ligand-free forms are available. This makes the development of new biosensors for desired analytes a long trial-and-error process. In the current study, we conducted {micro}s-long all atom molecular dynamics (MD) simulations of a maltose biosensor in both the apo (dark) and holo (bright) forms. We performed detailed hydrogen bond occupancy analyses to shed light on the mechanism of ligand induced conformational change in the sensor protein and its allosteric effect on the chromophore environment. We find that two strong indicators for distinguishing bright and dark states of biosensors are due to substantial changes in hydrogen bond dynamics in the system and solvent accessibility of the chromophore.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572447">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572447" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572447">
        <p class="paperTitle">Reinforcement Learning for Control of Human Locomotion in Simulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572447" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572447" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dashkovets, A.; Laschowski, B.</p>
        <p class="info">Score: 4.2, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572447' target='https://doi.org/10.1101/2023.12.19.572447'> 10.1101/2023.12.19.572447</a></p>
        <p class="abstract">Control of robotic leg prostheses and exoskeletons is an open challenge. Computer modeling and simulation can be used to study the dynamics and control of human walking and extract principles that can be programmed into robotic legs to behave similar to biological legs. In this study, we present the development of an efficient two-layer Q-learning algorithm, with k-d trees, that operates over continuous action spaces and a reward model that estimates the degree of muscle activation similarity between the agent and human state-to-action pairs and state-to-action sequences. We used a human musculoskeletal model acting in a high-dimensional, physics-based simulation environment to train and evaluate our algorithm to simulate biomimetic walking. We used imitation learning and artificial bio-mechanics data to accelerate training via expert demonstrations and used experimental human data to compare and validate our predictive simulations, achieving 79% accuracy. Also, when compared to the previous state-of-the-art that used deep deterministic policy gradient, our algorithm was significantly more efficient with lower computational and memory storage requirements (i.e., requiring 7 times less RAM and 87 times less CPU compute), which can benefit real-time embedded computing. Overall, our new two-layer Q-learning algorithm using sequential data for continuous imitation of human locomotion serves as a first step towards the development of bioinspired controllers for robotic prosthetic legs and exoskeletons. Future work will focus on improving the prediction accuracy compared to experimental data and expanding our simulations to other locomotor activities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572973">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572973" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572973">
        <p class="paperTitle">Biodistribution of DNA-origami nanostructures in live zebrafish embryos with single-cell resolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572973" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572973" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kolonelou, C.; Bräutigam, L.; Edwards, S.; Engström, E.; Dias, J. M.; Spratt, J.; Karampelias, C.; Wennmalm, S.; Brismar, H.; Andersson, O.; Teixeira, A. I.</p>
        <p class="info">Score: 4.1, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572973' target='https://doi.org/10.1101/2023.12.22.572973'> 10.1101/2023.12.22.572973</a></p>
        <p class="abstract">DNA origami-based nanotechnology is a versatile tool for exploring fundamental biological questions and holds significant promise for future biomedical applications. However, the development of DNA origami-based therapeutic agents is hindered by the challenge of translating in vitro performance into effective applications in vivo. Here, we exploit the optical transparency of the embryonic zebrafish to track intravenously injected, fluorescently labelled wireframe DNA origami nanostructures. Our approach integrated long-term, high-resolution imaging of transgenic live embryos with single-cell RNA sequencing, to elucidate the biodistribution of DNA nanostructures over time, up to 3 days post-injection (dpi). Notably, we observed rapid accumulation of nanostructures in the caudal hematopoietic tissue (CHT), akin to the fetal liver in mammals. We tested the effects of coating the nanostructures with an oligolysine PEG copolymer (K-PEG), a widely used strategy to enhance their stability. The K-PEG coating mitigated the accumulation rate in CHT, enabling higher percentages of the nanostructures to engage with other tissues. Additionally, our findings highlighted the pivotal role of scavenger endothelial cells in DNA origami clearance, with K-PEG offering sustained protection for the nanostructures at the CHT. Furthermore, by monitoring DNA origami in a transgenic zebrafish line designed for targeted macrophage ablation, we found that macrophages contribute to nanostructure clearance at later time points. This study introduces a framework for the analyses of the biodistribution and clearance of DNA origami nanostructures in vivo with single cell resolution and establishes a foundation for the investigation of DNA origami-based nanomedicines in animal models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.24.573272">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.24.573272" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.24.573272">
        <p class="paperTitle">Magnetoelectric Nanodiscs Enable Wireless Transgene-Free Neuromodulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.24.573272" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.24.573272" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, Y. J.; Driscoll, N.; Kent, N.; Vargas Paniagua, E.; Tabet, A.; Koehler, F.; Manthey, M.; Sahasrabudhe, A.; Signorelli, L.; Gregurec, D.; Anikeeva, P.</p>
        <p class="info">Score: 2.5, Published: 2023-12-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.24.573272' target='https://doi.org/10.1101/2023.12.24.573272'> 10.1101/2023.12.24.573272</a></p>
        <p class="abstract">Deep-brain stimulation (DBS) with implanted electrodes revolutionized treatment of movement disorders and empowered neuroscience studies. Identifying less invasive alternatives to DBS may further extend its clinical and research applications. Nanomaterial-mediated transduction of magnetic fields into electric potentials offers an alternative to invasive DBS. Here, we synthesize magnetoelectric nanodiscs (MENDs) with a core-double shell Fe3O4-CoFe2O4-BaTiO3 architecture with efficient magnetoelectric coupling. We find robust responses to magnetic field stimulation in neurons decorated with MENDs at a density of 1 {micro}g/mm2 despite individual-particle potentials below the neuronal excitation threshold. We propose a model for repetitive subthreshold depolarization, which combined with cable theory, corroborates our findings in vitro and informs magnetoelectric stimulation in vivo. MENDs injected into the ventral tegmental area of genetically intact mice at concentrations of 1 mg/mL enable remote control of reward behavior, setting the stage for mechanistic optimization of magnetoelectric neuromodulation and inspiring its future applications in fundamental and translational neuroscience.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572854">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572854" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572854">
        <p class="paperTitle">Development of a robotic cluster for automated and scalable cell therapy manufacturing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572854" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572854" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Melocchi, A.; Schmittlein, B.; Jones, A. L.; Ainane, Y.; Rizvi, A.; Chan, D.; Dickey, E.; Pool, K.; Harsono, K.; Szymkiewicz, D.; Scarfogliero, U.; Bhatia, V.; Sivanantham, A.; Kreciglowa, N.; Hunter, A.; Gomez, M.; Tanner, A.; Uboldi, M.; Batish, A.; Balcerek, J.; Kutova-Stoilova, M.; Paruthiyil, S.; Acevedo, L. A.; Stadnitskiy, R.; Carmichael, S.; Aulbach, H.; Hewitt, M.; De Mollerat Du Jeu, X.; di Robilant, B.; Parietti, F.; Esensten, J. H.</p>
        <p class="info">Score: 2.5, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572854' target='https://doi.org/10.1101/2023.12.21.572854'> 10.1101/2023.12.21.572854</a></p>
        <p class="abstract">The production of commercial autologous cell therapies such as chimeric antigen receptor T cells requires complex manual manufacturing processes. Skilled labor costs and challenges in manufacturing scale-out have contributed to high prices for these products. Here, we present a robotic system that uses industry-standard cell therapy manufacturing equipment to automate the steps involved in cell therapy manufacturing. The robotic cluster consists of a robotic arm and customized modules, allowing the robot to manipulate a variety of standard cell therapy instruments and materials such as incubators, bioreactors, and reagent bags. This system enables existing manual manufacturing processes to be rapidly adapted to robotic manufacturing, without having to adopt a completely new technology platform. Proof-of-concept for the robotic cluster expansion module was demonstrated by activating and expanding human CD8&#43; T cells. The robotic cultures showed comparable cell yields, viability, and identity to those manually performed. Such modular robotic solutions may support scale-up and scale-out of cell therapies that are developed using classical manual methods in academic laboratories and biotechnology companies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
