<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Protein generation with evolutionary diffusion: sequence is all you need
Authors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.
Score: 199.8, Published: 2023-09-12 DOI: 10.1101/2023.09.11.556673
Deep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Protein generation with evolutionary diffusion: sequence is all you need
Authors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.
Score: 199.8, Published: 2023-09-12 DOI: 10.1101/2023.09.11.556673
Deep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-04T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-10-04T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Protein generation with evolutionary diffusion: sequence is all you need
Authors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.
Score: 199.8, Published: 2023-09-12 DOI: 10.1101/2023.09.11.556673
Deep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Protein generation with evolutionary diffusion: sequence is all you need\nAuthors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.\nScore: 199.8, Published: 2023-09-12 DOI: 10.1101/2023.09.11.556673\nDeep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design.",
  "keywords": [
    
  ],
  "articleBody": " Protein generation with evolutionary diffusion: sequence is all you need\nAuthors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.\nScore: 199.8, Published: 2023-09-12 DOI: 10.1101/2023.09.11.556673\nDeep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design. However, current state-of-the-art models generate protein structures, which limits the scope of their training data and restricts generations to a small and biased subset of protein design space. Here, we introduce a general-purpose diffusion framework, EvoDiff, that combines evolutionary-scale data with the distinct conditioning capabilities of diffusion models for controllable protein generation in sequence space. EvoDiff generates high-fidelity, diverse, and structurally-plausible proteins that cover natural sequence and functional space. Critically, EvoDiff can generate proteins inaccessible to structure-based models, such as those with disordered regions, while maintaining the ability to design scaffolds for functional structural motifs, demonstrating the universality of our sequence-based formulation. We envision that EvoDiff will expand capabilities in protein engineering beyond the structure-function paradigm toward programmable, sequence-first design.\nMolecular de-extinction of antibiotics enabled by deep learning\nAuthors: Wan, F.; Torres, M. D. T.; Peng, J.; de la Fuente, C.\nScore: 18.1, Published: 2023-10-01 DOI: 10.1101/2023.10.01.560353\nMolecular de-extinction is an emerging field that aims to resurrect molecules to solve present-day problems such as antibiotic resistance. Here, we introduce a deep learning approach called Antibiotic Peptide de-Extinction (APEX) to mine the proteomes of all available extinct organisms (the extinctome) searching for encrypted peptide (EP) antibiotics. APEX mined a total of 10,311,899 EPs and identified 37,176 sequences predicted to have broad-spectrum antimicrobial activity, 11,035 of which were not found in extant organisms. Chemical synthesis and experimental validation yielded archaic EPs (AEPs) with activity against dangerous bacterial pathogens. Most peptides killed bacteria by depolarizing their cytoplasmic membrane, contrary to known antimicrobial peptides, which target the outer membrane. Notably, lead peptides, including those derived from the woolly mammoth, ancient sea cow, giant sloth, and extinct giant elk, exhibited anti-infective activity in preclinical mouse models. We propose molecular de-extinction, accelerated by deep learning, as a framework for discovering therapeutic molecules.\nClick editing enables programmable genome writing using DNA polymerases and HUH endonucleases\nAuthors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.\nScore: 106.3, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557440\nGenome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e.g. CRISPR-Cas) to permit the installation of a range of edits including substitutions, insertions, and deletions. Click editors (CEs) leverage the \"click\"-like bioconjugation ability of HUH endonucleases (HUHes) with single stranded DNA substrates to covalently tether \"click DNA\" (clkDNA) templates encoding user-specifiable edits at targeted genomic loci. Through iterative optimization of the modular components of CEs (DNA polymerase and HUHe orthologs, architectural modifications, etc.) and their clkDNAs (template configurations, repair evading substitutions, etc.), we demonstrate the ability to install precise genome edits with minimal indels and no unwanted byproduct insertions. Since clkDNAs can be ordered as simple DNA oligonucleotides for cents per base, it is possible to screen many different clkDNA parameters rapidly and inexpensively to maximize edit efficiency. Together, click editing is a precise and highly versatile platform for modifying genomes with a simple workflow and broad utility across diverse biological applications.\nHaloTag display enables quantitative single-particle characterization and functionalization of engineered extracellular vesicles\nAuthors: Mitrut, R. E.; Stranford, D. M.; Chan, J. M.; Bailey, M. D.; Luo, M.; Meade, T. J.; Wang, M.; Leonard, J. N.\nScore: 10.8, Published: 2023-09-27 DOI: 10.1101/2023.09.25.559433\nExtracellular vesicles (EVs) play key roles in diverse biological processes, transport biomolecules between cells, and have been engineered for therapeutic applications. A useful EV bioengineering strategy is to express engineered proteins on the EV surface to confer targeting, bioactivity, and other properties. Measuring how incorporation varies across a population of EVs is important for characterizing such materials and understanding their function, yet it remains challenging to quantitatively characterize the absolute number of engineered proteins incorporated at single-EV resolution. To address these needs, we developed a HaloTag-based characterization platform in which dyes or other synthetic species can be covalently and stoichiometrically attached to engineered proteins on the EV surface. To evaluate this system, we employed several orthogonal quantification methods, including flow cytometry and fluorescence microscopy, and found that HaloTag-mediated quantification is generally robust across EV analysis methods. We compared HaloTag-labeling to antibody-labeling of EVs using single vesicle flow cytometry, enabling us to quantify the substantial degree to which antibody labeling can underestimate the absolute number of proteins present on an EV. Finally, we demonstrate use of HaloTag to compare between protein designs for EV bioengineering. Overall, the HaloTag system is a useful EV characterization tool which complements and expands existing methods.\nNext-generation genetic sexing strain establishment in the agricultural pest Ceratitis capitata\nAuthors: Davydova, S.; Liu, J.; Kandul, N. P.; Braswell, W. E.; Akbari, O. S.; Meccariello, A.\nScore: 8.6, Published: 2023-10-01 DOI: 10.1101/2023.09.29.560088\nTephritid fruit fly pests pose an increasing threat to the agricultural industry due to their global dispersion and a highly invasive nature. Here we showcase the feasibility of an early-detection SEPARATOR sex sorting approach through using the non-model Tephritid pest, Ceratitis capitata. This system relies on female-only fluorescent marker expression, accomplished through the use of a sex-specific intron of the highly-conserved transformer gene from C. capitata and Anastrepha ludens. The herein characterized strains have 100% desired phenotype outcomes, allowing accurate male-female separation during early development. Overall, we describe an antibiotic and temperature-independent sex-sorting system in C. capitata, which, moving forward, may be implemented in other non-model Tephritid pest species. This strategy can facilitate the establishment of genetic sexing systems with endogenous elements exclusively, which, on a wider scale, can improve pest population control strategies like sterile insect technique.\nOpenCell: A Low-cost, Open-Source, 3-in-1 device for DNA Extraction\nAuthors: Gupta, A.; Yu, J.; Challita, E. J.; Standeven, J.; Bhamla, M. S.\nScore: 9.1, Published: 2023-09-25 DOI: 10.1101/2023.09.18.558349\nHigh-cost DNA extraction procedures pose significant challenges for budget-constrained laboratories. To address this, we introduce OpenCell, an economical, open-source, 3-in-1 laboratory device that combines the functionalities of a bead homogenizer, a microcentrifuge, and a vortex mixer. OpenCell utilizes modular attachments that magnetically connect to a central rotating brushless motor. This motor couples to an epicyclic gearing mechanism, enabling efficient bead homogenization, vortex mixing, and centrifugation within one compact unit. OpenCells design incorporates multiple redundant safety features, ensuring both the devices and operators safety. Additional features such as RPM measurement, programmable timers, battery operation, and optional speed control make OpenCell a reliable and reproducible laboratory instrument. In our study, OpenCell successfully isolated DNA from Spinacia oleracea (spinach), with an average yield of 2.3 g and an A260/A280 ratio of 1.77, demonstrating its effectiveness for downstream applications such as Polymerase Chain Reaction (PCR) amplification. With its compact size (20 cm x 28 cm x 6.7 cm) and lightweight design (0.8 kg), comparable to the size and weight of a laptop, OpenCell is portable, making it an attractive component of a lab-in-a-backpack for resource-constrained environments in low-and-middle-income countries and synthetic biology in remote field stations. Leveraging the accessibility of 3D printing and off-the-shelf components, OpenCell can be manufactured and assembled at a low unit cost of less than $50, providing an affordable alternative to expensive laboratory equipment costing over $4000. OpenCell aims to overcome the barriers to entry in synthetic biology research and contribute to the growing collection of frugal and open hardware.\nSpiral NeuroString: High-Density Soft Bioelectronic Fibers for Multimodal Sensing and Stimulation\nAuthors: Khatib, M.; Zhao, E. T.; Wei, S.; Abramson, A.; Bishop, E. B.; Chen, C.-H.; Thomas, A.-L.; Xu, C.; Park, J.; Lee, Y.; Hamnett, R.; Yu, W.; Root, S. E.; Yuan, L.; Chakhtoura, D.; Kim, K. K.; Zhong, D.; Nishio, Y.; Zhao, C.; Wu, C.; Jiang, Y.; Zhang, A.; Li, J.; Wang, W.; Salimi-Jazi, F.; Rafeeqi, T. A.; Hemed, N. M.; Tok, J. B.-H.; Chen, X.; Kaltschmidt, J. A.; Dunn, J.; Bao, Z.\nScore: 3.8, Published: 2023-10-03 DOI: 10.1101/2023.10.02.560482\nBioelectronic fibers hold promise for both research and clinical applications due to their compactness, ease of implantation, and ability to incorporate various functionalities such as sensing and stimulation. However, existing devices suffer from bulkiness, rigidity, limited functionality, and low density of active components. These limitations stem from the difficulty to incorporate many components on one-dimensional (1D) fiber devices due to the incompatibility of conventional microfabrication methods (e.g., photolithography) with curved, thin and long fiber structures. Herein, we introduce a fabrication approach, spiral transformation'', to convert two-dimensional (2D) films containing microfabricated devices into 1D soft fibers. This approach allows for the creation of high density multimodal soft bioelectronic fibers, termed Spiral NeuroString (S-NeuroString), while enabling precise control over the longitudinal, angular, and radial positioning and distribution of the functional components. We show the utility of S-NeuroString for motility mapping, serotonin sensing, and tissue stimulation within the dynamic and soft gastrointestinal (GI) system, as well as for single-unit recordings in the brain. The described bioelectronic fibers hold great promises for next-generation multifunctional implantable electronics.\nSingle-cell Herpes Simplex Virus type-1 infection of neurons using drop-based microfluidics reveals heterogeneous replication kinetics\nAuthors: Fredrikson, J. P.; Domanico, L. F.; Pratt, S. L.; Loveday, E. K.; Taylor, M. P.; Chang, C. B.\nScore: 9.3, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558333\nSingle-cell analyses of viral infections often reveal heterogeneity that is not detected by traditional population-level studies. This study applies drop-based microfluidics to investigate the dynamics of HSV-1 infection of neurons at the single-cell level. We used micron-scale Matrigel beads, termed microgels, to culture individual murine Superior Cervical ganglia (SCG) neurons or epithelial cells. Microgel-cultured cells are subsequently enclosed in individual media-in-oil droplets with a dual fluorescent-reporter HSV-1, enabling real-time observation of viral gene expression and replication. Infection within drops revealed that the kinetics of initial viral gene expression and replication were dependent on the inoculating dose. Notably, increasing inoculating doses led to earlier onset of viral gene expression and more frequent productive viral replication. These observations provide crucial insights into the complexity of HSV-1 infection in neurons and emphasize the importance of studying single-cell outcomes of viral infection. The innovative techniques presented here for cell culture and infection in drops provide a foundation for future virology and neurobiology investigations.\nBuilding on-chip cytoskeletal circuits via branched microtubule networks\nAuthors: Zaferani, M.; Song, R.; Petry, S.; Stone, H. A.\nScore: 3.1, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560515\nControllable platforms to engineer robust cytoskeletal scaffolds have the potential to create novel on-chip nanotechnologies. Inspired by axons, we combined the branching microtubule (MT) nucleation pathway with microfabrication to develop \"cytoskeletal circuits\". This active matter platform allows control over the adaptive self-organization of uniformly polarized MT arrays via geometric features of microstructures designed within a microfluidic confinement. We build and characterize basic elements, including turns and divisions, as well as complex regulatory elements, such as biased division and MT diodes, to construct various MT architectures on a chip. Our platform could be used in diverse applications, ranging from efficient on-chip molecular transport to mechanical nano-actuators. Further, cytoskeletal circuits can serve as a tool to study how the physical environment contributes to MT architecture in living cells.\nMonitoring lasting changes to brain tissue integrity through mechanical properties following adolescent exercise intervention in a rat model of Fetal Alcohol Spectrum Disorders\nAuthors: Milbocker, K. A.; Williams, L. T.; Caban-Rivera, D. A.; Smith, I. F.; Kurtz, S.; McGarry, M. D.; Wattrisse, B.; Van Houten, E. E.; Johnson, C. L.; Klintsova, A. Y.\nScore: 2.8, Published: 2023-09-29 DOI: 10.1101/2023.09.26.559571\nBackground Fetal Alcohol Spectrum Disorders (FASD) encompass a group of highly prevalent conditions resulting from prenatal alcohol exposure. Alcohol exposure during the third trimester of pregnancy overlapping with the brain growth spurt is detrimental to white matter growth and myelination, particularly in the corpus callosum, ultimately affecting tissue integrity in adolescence. Traditional neuroimaging techniques have been essential for assessing neurodevelopment in affected youth; however, these methods are limited in their capacity to track subtle microstructural alterations to white matter, thus restricting their effectiveness in monitoring therapeutic intervention. In this preliminary study we use a highly sensitive and clinically translatable Magnetic Resonance Elastography (MRE) protocol for assessing brain tissue microstructure through its mechanical properties following an exercise intervention in a rat model of FASD. Methods Rat pups were divided into two groups: alcohol-exposed (AE) pups which received alcohol in milk substitute (5.25 g/kg/day) via intragastric intubation on postnatal days (PD) four through nine during the rat brain growth spurt (Dobbing and Sands, 1979), or sham-intubated (SI) controls. In adolescence, on PD 30, half AE and SI rats were randomly assigned to either a modified home cage with free access to a running wheel or to a new home cage for 12 days (Gursky and Klintsova, 2017). Previous studies conducted in the lab have shown that 12 days of voluntary exercise intervention in adolescence immediately ameliorated callosal myelination in AE rats (Milbocker et al., 2022, 2023). MRE was used to measure longitudinal changes to mechanical properties of the whole brain and the corpus callosum at intervention termination and one-month post-intervention. Histological quantification of precursor and myelinating oligoglia in corpus callosum was performed one-month post-intervention. Results Prior to intervention, AE rats had lower forebrain stiffness in adolescence compared to SI controls (p = 0.02). Exercise intervention immediately mitigated this effect in AE rats, resulting in higher forebrain stiffness post-intervention in adolescence. Similarly, we discovered that forebrain damping ratio was lowest in AE rats in adolescence (p \u003c 0.01), irrespective of intervention exposure. One-month post-intervention in adulthood, AE and SI rats exhibited comparable forebrain stiffness and damping ratio (p \u003e 0.05). Taken together, these MRE data suggest that adolescent exercise intervention supports neurodevelopmental catch-up in AE rats. Analysis of the stiffness and damping ratio of the body of corpus callosum revealed that these measures increased with age. Finally, histological quantification of myelinating oligodendrocytes one-month post-intervention revealed a negative rebound effect of exercise cessation on the total estimate of these cells in the body of corpus callosum, irrespective of treatment group which was not convergent with noninvasive MRE measures. Conclusions This is the first application of MRE to measure changes in brain mechanical properties in a rodent model of FASD. MRE successfully captured alcohol-related changes to forebrain stiffness and damping ratio in adolescence. These preliminary findings expand upon results from previous studies which used traditional diffusion neuroimaging to identify structural changes to the adolescent brain in rodent models of FASD (Milbocker et al., 2022; Newville et al., 2017). Additionally, in vivo MRE identified an exercise-related alteration to forebrain stiffness that occurred in adolescence, immediately post-intervention.\n",
  "wordCount" : "2473",
  "inLanguage": "en",
  "datePublished": "2023-10-04T10:39:33Z",
  "dateModified": "2023-10-04T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on October 4, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.556673">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.556673" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.556673">
        <p class="paperTitle">Protein generation with evolutionary diffusion: sequence is all you need</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.556673" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.556673" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.</p>
        <p class="info">Score: 199.8, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.556673' target='https://doi.org/10.1101/2023.09.11.556673'> 10.1101/2023.09.11.556673</a></p>
        <p class="abstract">Deep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design. However, current state-of-the-art models generate protein structures, which limits the scope of their training data and restricts generations to a small and biased subset of protein design space. Here, we introduce a general-purpose diffusion framework, EvoDiff, that combines evolutionary-scale data with the distinct conditioning capabilities of diffusion models for controllable protein generation in sequence space. EvoDiff generates high-fidelity, diverse, and structurally-plausible proteins that cover natural sequence and functional space. Critically, EvoDiff can generate proteins inaccessible to structure-based models, such as those with disordered regions, while maintaining the ability to design scaffolds for functional structural motifs, demonstrating the universality of our sequence-based formulation. We envision that EvoDiff will expand capabilities in protein engineering beyond the structure-function paradigm toward programmable, sequence-first design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.560353">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.560353" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.560353">
        <p class="paperTitle">Molecular de-extinction of antibiotics enabled by deep learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.560353" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.560353" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wan, F.; Torres, M. D. T.; Peng, J.; de la Fuente, C.</p>
        <p class="info">Score: 18.1, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.560353' target='https://doi.org/10.1101/2023.10.01.560353'> 10.1101/2023.10.01.560353</a></p>
        <p class="abstract">Molecular de-extinction is an emerging field that aims to resurrect molecules to solve present-day problems such as antibiotic resistance. Here, we introduce a deep learning approach called Antibiotic Peptide de-Extinction (APEX) to mine the proteomes of all available extinct organisms (the extinctome) searching for encrypted peptide (EP) antibiotics. APEX mined a total of 10,311,899 EPs and identified 37,176 sequences predicted to have broad-spectrum antimicrobial activity, 11,035 of which were not found in extant organisms. Chemical synthesis and experimental validation yielded archaic EPs (AEPs) with activity against dangerous bacterial pathogens. Most peptides killed bacteria by depolarizing their cytoplasmic membrane, contrary to known antimicrobial peptides, which target the outer membrane. Notably, lead peptides, including those derived from the woolly mammoth, ancient sea cow, giant sloth, and extinct giant elk, exhibited anti-infective activity in preclinical mouse models. We propose molecular de-extinction, accelerated by deep learning, as a framework for discovering therapeutic molecules.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557440">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557440" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557440">
        <p class="paperTitle">Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557440" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557440" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.</p>
        <p class="info">Score: 106.3, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557440' target='https://doi.org/10.1101/2023.09.12.557440'> 10.1101/2023.09.12.557440</a></p>
        <p class="abstract">Genome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e.g. CRISPR-Cas) to permit the installation of a range of edits including substitutions, insertions, and deletions. Click editors (CEs) leverage the &#34;click&#34;-like bioconjugation ability of HUH endonucleases (HUHes) with single stranded DNA substrates to covalently tether &#34;click DNA&#34; (clkDNA) templates encoding user-specifiable edits at targeted genomic loci. Through iterative optimization of the modular components of CEs (DNA polymerase and HUHe orthologs, architectural modifications, etc.) and their clkDNAs (template configurations, repair evading substitutions, etc.), we demonstrate the ability to install precise genome edits with minimal indels and no unwanted byproduct insertions. Since clkDNAs can be ordered as simple DNA oligonucleotides for cents per base, it is possible to screen many different clkDNA parameters rapidly and inexpensively to maximize edit efficiency. Together, click editing is a precise and highly versatile platform for modifying genomes with a simple workflow and broad utility across diverse biological applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559433">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559433" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559433">
        <p class="paperTitle">HaloTag display enables quantitative single-particle characterization and functionalization of engineered extracellular vesicles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559433" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559433" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mitrut, R. E.; Stranford, D. M.; Chan, J. M.; Bailey, M. D.; Luo, M.; Meade, T. J.; Wang, M.; Leonard, J. N.</p>
        <p class="info">Score: 10.8, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559433' target='https://doi.org/10.1101/2023.09.25.559433'> 10.1101/2023.09.25.559433</a></p>
        <p class="abstract">Extracellular vesicles (EVs) play key roles in diverse biological processes, transport biomolecules between cells, and have been engineered for therapeutic applications. A useful EV bioengineering strategy is to express engineered proteins on the EV surface to confer targeting, bioactivity, and other properties. Measuring how incorporation varies across a population of EVs is important for characterizing such materials and understanding their function, yet it remains challenging to quantitatively characterize the absolute number of engineered proteins incorporated at single-EV resolution. To address these needs, we developed a HaloTag-based characterization platform in which dyes or other synthetic species can be covalently and stoichiometrically attached to engineered proteins on the EV surface. To evaluate this system, we employed several orthogonal quantification methods, including flow cytometry and fluorescence microscopy, and found that HaloTag-mediated quantification is generally robust across EV analysis methods. We compared HaloTag-labeling to antibody-labeling of EVs using single vesicle flow cytometry, enabling us to quantify the substantial degree to which antibody labeling can underestimate the absolute number of proteins present on an EV. Finally, we demonstrate use of HaloTag to compare between protein designs for EV bioengineering. Overall, the HaloTag system is a useful EV characterization tool which complements and expands existing methods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560088">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560088" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560088">
        <p class="paperTitle">Next-generation genetic sexing strain establishment in the agricultural pest Ceratitis capitata</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560088" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560088" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davydova, S.; Liu, J.; Kandul, N. P.; Braswell, W. E.; Akbari, O. S.; Meccariello, A.</p>
        <p class="info">Score: 8.6, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560088' target='https://doi.org/10.1101/2023.09.29.560088'> 10.1101/2023.09.29.560088</a></p>
        <p class="abstract">Tephritid fruit fly pests pose an increasing threat to the agricultural industry due to their global dispersion and a highly invasive nature. Here we showcase the feasibility of an early-detection SEPARATOR sex sorting approach through using the non-model Tephritid pest, Ceratitis capitata. This system relies on female-only fluorescent marker expression, accomplished through the use of a sex-specific intron of the highly-conserved transformer gene from C. capitata and Anastrepha ludens. The herein characterized strains have 100% desired phenotype outcomes, allowing accurate male-female separation during early development. Overall, we describe an antibiotic and temperature-independent sex-sorting system in C. capitata, which, moving forward, may be implemented in other non-model Tephritid pest species. This strategy can facilitate the establishment of genetic sexing systems with endogenous elements exclusively, which, on a wider scale, can improve pest population control strategies like sterile insect technique.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558349">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558349" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558349">
        <p class="paperTitle">OpenCell: A Low-cost, Open-Source, 3-in-1 device for DNA Extraction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558349" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558349" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gupta, A.; Yu, J.; Challita, E. J.; Standeven, J.; Bhamla, M. S.</p>
        <p class="info">Score: 9.1, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558349' target='https://doi.org/10.1101/2023.09.18.558349'> 10.1101/2023.09.18.558349</a></p>
        <p class="abstract">High-cost DNA extraction procedures pose significant challenges for budget-constrained laboratories. To address this, we introduce OpenCell, an economical, open-source, 3-in-1 laboratory device that combines the functionalities of a bead homogenizer, a microcentrifuge, and a vortex mixer. OpenCell utilizes modular attachments that magnetically connect to a central rotating brushless motor. This motor couples to an epicyclic gearing mechanism, enabling efficient bead homogenization, vortex mixing, and centrifugation within one compact unit. OpenCells design incorporates multiple redundant safety features, ensuring both the devices and operators safety. Additional features such as RPM measurement, programmable timers, battery operation, and optional speed control make OpenCell a reliable and reproducible laboratory instrument. In our study, OpenCell successfully isolated DNA from Spinacia oleracea (spinach), with an average yield of 2.3 g and an A260/A280 ratio of 1.77, demonstrating its effectiveness for downstream applications such as Polymerase Chain Reaction (PCR) amplification. With its compact size (20 cm x 28 cm x 6.7 cm) and lightweight design (0.8 kg), comparable to the size and weight of a laptop, OpenCell is portable, making it an attractive component of a  lab-in-a-backpack for resource-constrained environments in low-and-middle-income countries and synthetic biology in remote field stations. Leveraging the accessibility of 3D printing and off-the-shelf components, OpenCell can be manufactured and assembled at a low unit cost of less than $50, providing an affordable alternative to expensive laboratory equipment costing over $4000. OpenCell aims to overcome the barriers to entry in synthetic biology research and contribute to the growing collection of frugal and open hardware.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560482">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560482" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560482">
        <p class="paperTitle">Spiral NeuroString: High-Density Soft Bioelectronic Fibers for Multimodal Sensing and Stimulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560482" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560482" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khatib, M.; Zhao, E. T.; Wei, S.; Abramson, A.; Bishop, E. B.; Chen, C.-H.; Thomas, A.-L.; Xu, C.; Park, J.; Lee, Y.; Hamnett, R.; Yu, W.; Root, S. E.; Yuan, L.; Chakhtoura, D.; Kim, K. K.; Zhong, D.; Nishio, Y.; Zhao, C.; Wu, C.; Jiang, Y.; Zhang, A.; Li, J.; Wang, W.; Salimi-Jazi, F.; Rafeeqi, T. A.; Hemed, N. M.; Tok, J. B.-H.; Chen, X.; Kaltschmidt, J. A.; Dunn, J.; Bao, Z.</p>
        <p class="info">Score: 3.8, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560482' target='https://doi.org/10.1101/2023.10.02.560482'> 10.1101/2023.10.02.560482</a></p>
        <p class="abstract">Bioelectronic fibers hold promise for both research and clinical applications due to their compactness, ease of implantation, and ability to incorporate various functionalities such as sensing and stimulation. However, existing devices suffer from bulkiness, rigidity, limited functionality, and low density of active components. These limitations stem from the difficulty to incorporate many components on one-dimensional (1D) fiber devices due to the incompatibility of conventional microfabrication methods (e.g., photolithography) with curved, thin and long fiber structures. Herein, we introduce a fabrication approach, spiral transformation&#39;&#39;, to convert two-dimensional (2D) films containing microfabricated devices into 1D soft fibers. This approach allows for the creation of high density multimodal soft bioelectronic fibers, termed Spiral NeuroString (S-NeuroString), while enabling precise control over the longitudinal, angular, and radial positioning and distribution of the functional components. We show the utility of S-NeuroString for motility mapping, serotonin sensing, and tissue stimulation within the dynamic and soft gastrointestinal (GI) system, as well as for single-unit recordings in the brain. The described bioelectronic fibers hold great promises for next-generation multifunctional implantable electronics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558333">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558333" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558333">
        <p class="paperTitle">Single-cell Herpes Simplex Virus type-1 infection of neurons using drop-based microfluidics reveals heterogeneous replication kinetics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558333" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558333" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fredrikson, J. P.; Domanico, L. F.; Pratt, S. L.; Loveday, E. K.; Taylor, M. P.; Chang, C. B.</p>
        <p class="info">Score: 9.3, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558333' target='https://doi.org/10.1101/2023.09.18.558333'> 10.1101/2023.09.18.558333</a></p>
        <p class="abstract">Single-cell analyses of viral infections often reveal heterogeneity that is not detected by traditional population-level studies. This study applies drop-based microfluidics to investigate the dynamics of HSV-1 infection of neurons at the single-cell level. We used micron-scale Matrigel beads, termed microgels, to culture individual murine Superior Cervical ganglia (SCG) neurons or epithelial cells. Microgel-cultured cells are subsequently enclosed in individual media-in-oil droplets with a dual fluorescent-reporter HSV-1, enabling real-time observation of viral gene expression and replication. Infection within drops revealed that the kinetics of initial viral gene expression and replication were dependent on the inoculating dose. Notably, increasing inoculating doses led to earlier onset of viral gene expression and more frequent productive viral replication. These observations provide crucial insights into the complexity of HSV-1 infection in neurons and emphasize the importance of studying single-cell outcomes of viral infection. The innovative techniques presented here for cell culture and infection in drops provide a foundation for future virology and neurobiology investigations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560515">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560515" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560515">
        <p class="paperTitle">Building on-chip cytoskeletal circuits via branched microtubule networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560515" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560515" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zaferani, M.; Song, R.; Petry, S.; Stone, H. A.</p>
        <p class="info">Score: 3.1, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560515' target='https://doi.org/10.1101/2023.10.02.560515'> 10.1101/2023.10.02.560515</a></p>
        <p class="abstract">Controllable platforms to engineer robust cytoskeletal scaffolds have the potential to create novel on-chip nanotechnologies. Inspired by axons, we combined the branching microtubule (MT) nucleation pathway with microfabrication to develop &#34;cytoskeletal circuits&#34;. This active matter platform allows control over the adaptive self-organization of uniformly polarized MT arrays via geometric features of microstructures designed within a microfluidic confinement. We build and characterize basic elements, including turns and divisions, as well as complex regulatory elements, such as biased division and MT diodes, to construct various MT architectures on a chip. Our platform could be used in diverse applications, ranging from efficient on-chip molecular transport to mechanical nano-actuators. Further, cytoskeletal circuits can serve as a tool to study how the physical environment contributes to MT architecture in living cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559571">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559571" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559571">
        <p class="paperTitle">Monitoring lasting changes to brain tissue integrity through mechanical properties following adolescent exercise intervention in a rat model of Fetal Alcohol Spectrum Disorders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559571" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559571" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Milbocker, K. A.; Williams, L. T.; Caban-Rivera, D. A.; Smith, I. F.; Kurtz, S.; McGarry, M. D.; Wattrisse, B.; Van Houten, E. E.; Johnson, C. L.; Klintsova, A. Y.</p>
        <p class="info">Score: 2.8, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559571' target='https://doi.org/10.1101/2023.09.26.559571'> 10.1101/2023.09.26.559571</a></p>
        <p class="abstract">Background Fetal Alcohol Spectrum Disorders (FASD) encompass a group of highly prevalent conditions resulting from prenatal alcohol exposure. Alcohol exposure during the third trimester of pregnancy overlapping with the brain growth spurt is detrimental to white matter growth and myelination, particularly in the corpus callosum, ultimately affecting tissue integrity in adolescence. Traditional neuroimaging techniques have been essential for assessing neurodevelopment in affected youth; however, these methods are limited in their capacity to track subtle microstructural alterations to white matter, thus restricting their effectiveness in monitoring therapeutic intervention. In this preliminary study we use a highly sensitive and clinically translatable Magnetic Resonance Elastography (MRE) protocol for assessing brain tissue microstructure through its mechanical properties following an exercise intervention in a rat model of FASD. Methods Rat pups were divided into two groups: alcohol-exposed (AE) pups which received alcohol in milk substitute (5.25 g/kg/day) via intragastric intubation on postnatal days (PD) four through nine during the rat brain growth spurt (Dobbing and Sands, 1979), or sham-intubated (SI) controls. In adolescence, on PD 30, half AE and SI rats were randomly assigned to either a modified home cage with free access to a running wheel or to a new home cage for 12 days (Gursky and Klintsova, 2017). Previous studies conducted in the lab have shown that 12 days of voluntary exercise intervention in adolescence immediately ameliorated callosal myelination in AE rats (Milbocker et al., 2022, 2023). MRE was used to measure longitudinal changes to mechanical properties of the whole brain and the corpus callosum at intervention termination and one-month post-intervention. Histological quantification of precursor and myelinating oligoglia in corpus callosum was performed one-month post-intervention. Results Prior to intervention, AE rats had lower forebrain stiffness in adolescence compared to SI controls (p = 0.02). Exercise intervention immediately mitigated this effect in AE rats, resulting in higher forebrain stiffness post-intervention in adolescence. Similarly, we discovered that forebrain damping ratio was lowest in AE rats in adolescence (p &lt; 0.01), irrespective of intervention exposure. One-month post-intervention in adulthood, AE and SI rats exhibited comparable forebrain stiffness and damping ratio (p &gt; 0.05). Taken together, these MRE data suggest that adolescent exercise intervention supports neurodevelopmental catch-up in AE rats. Analysis of the stiffness and damping ratio of the body of corpus callosum revealed that these measures increased with age. Finally, histological quantification of myelinating oligodendrocytes one-month post-intervention revealed a negative rebound effect of exercise cessation on the total estimate of these cells in the body of corpus callosum, irrespective of treatment group which was not convergent with noninvasive MRE measures. Conclusions This is the first application of MRE to measure changes in brain mechanical properties in a rodent model of FASD. MRE successfully captured alcohol-related changes to forebrain stiffness and damping ratio in adolescence. These preliminary findings expand upon results from previous studies which used traditional diffusion neuroimaging to identify structural changes to the adolescent brain in rodent models of FASD (Milbocker et al., 2022; Newville et al., 2017). Additionally, in vivo MRE identified an exercise-related alteration to forebrain stiffness that occurred in adolescence, immediately post-intervention.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
