<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Antigen-display exosomes provide adjuvant-free protection against SARS-CoV-2 disease at nanogram levels of spike protein
Authors: Guo, C.; Sachithanandham, J.; Zhong, W.; Craney, M.; Villano, J.; Pekosz, A.; Gould, S.
Score: 11.4, Published: 2024-01-05 DOI: 10.1101/2024.01.04.574272
As the only bionormal nanovesicle, exosomes have high potential as a nanovesicle for delivering vaccines and therapeutics. We show here that the loading of type-1 membrane proteins into the exosome membrane is induced by exosome membrane anchor domains, EMADs, that maximize protein delivery to the plasma membrane, minimize protein sorting to other compartments, and direct proteins into exosome membranes.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Antigen-display exosomes provide adjuvant-free protection against SARS-CoV-2 disease at nanogram levels of spike protein
Authors: Guo, C.; Sachithanandham, J.; Zhong, W.; Craney, M.; Villano, J.; Pekosz, A.; Gould, S.
Score: 11.4, Published: 2024-01-05 DOI: 10.1101/2024.01.04.574272
As the only bionormal nanovesicle, exosomes have high potential as a nanovesicle for delivering vaccines and therapeutics. We show here that the loading of type-1 membrane proteins into the exosome membrane is induced by exosome membrane anchor domains, EMADs, that maximize protein delivery to the plasma membrane, minimize protein sorting to other compartments, and direct proteins into exosome membranes." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-07T10:38:59+00:00" />
<meta property="article:modified_time" content="2024-01-07T10:38:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Antigen-display exosomes provide adjuvant-free protection against SARS-CoV-2 disease at nanogram levels of spike protein
Authors: Guo, C.; Sachithanandham, J.; Zhong, W.; Craney, M.; Villano, J.; Pekosz, A.; Gould, S.
Score: 11.4, Published: 2024-01-05 DOI: 10.1101/2024.01.04.574272
As the only bionormal nanovesicle, exosomes have high potential as a nanovesicle for delivering vaccines and therapeutics. We show here that the loading of type-1 membrane proteins into the exosome membrane is induced by exosome membrane anchor domains, EMADs, that maximize protein delivery to the plasma membrane, minimize protein sorting to other compartments, and direct proteins into exosome membranes."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Antigen-display exosomes provide adjuvant-free protection against SARS-CoV-2 disease at nanogram levels of spike protein\nAuthors: Guo, C.; Sachithanandham, J.; Zhong, W.; Craney, M.; Villano, J.; Pekosz, A.; Gould, S.\nScore: 11.4, Published: 2024-01-05 DOI: 10.1101/2024.01.04.574272\nAs the only bionormal nanovesicle, exosomes have high potential as a nanovesicle for delivering vaccines and therapeutics. We show here that the loading of type-1 membrane proteins into the exosome membrane is induced by exosome membrane anchor domains, EMADs, that maximize protein delivery to the plasma membrane, minimize protein sorting to other compartments, and direct proteins into exosome membranes.",
  "keywords": [
    
  ],
  "articleBody": " Antigen-display exosomes provide adjuvant-free protection against SARS-CoV-2 disease at nanogram levels of spike protein\nAuthors: Guo, C.; Sachithanandham, J.; Zhong, W.; Craney, M.; Villano, J.; Pekosz, A.; Gould, S.\nScore: 11.4, Published: 2024-01-05 DOI: 10.1101/2024.01.04.574272\nAs the only bionormal nanovesicle, exosomes have high potential as a nanovesicle for delivering vaccines and therapeutics. We show here that the loading of type-1 membrane proteins into the exosome membrane is induced by exosome membrane anchor domains, EMADs, that maximize protein delivery to the plasma membrane, minimize protein sorting to other compartments, and direct proteins into exosome membranes. Using SARS-CoV-2 spike as an example and EMAD13 as our most effective exosome membrane anchor, we show that cells expressing a spike-EMAD13 fusion protein produced exosomes that carry dense arrays of spike trimers on 50% of all exosomes. Moreover, we find that immunization with spike-EMAD13 exosomes induced strong neutralizing antibody responses and protected hamsters against SARS-CoV-2 disease at doses of just 0.5-5 ng of spike protein, without adjuvant, demonstrating that antigen-display exosomes are particularly immunogenic, with important implications for both structural and expression-dependent vaccines.\nPiggybacking functionalized DNA nanostructures into live cell nuclei\nAuthors: Roozbahani, G. M.; Colosi, P.; Oravecz, A.; Sorokina, E. M.; Pfeifer, W.; Shokri, S.; Wei, Y.; Didier, P.; DeLuca, M.; Arya, G.; Tora, L.; Lakadamyali, M.; Poirier, M. G.; Castro, C.\nScore: 8.6, Published: 2024-01-01 DOI: 10.1101/2023.12.30.573746\nDNA origami (DO) are promising tools for in vitro or in vivo applications including drug delivery; biosensing, detecting biomolecules; and probing chromatin sub-structures. Targeting these nanodevices to mammalian cell nuclei could provide impactful approaches for probing visualizing and controlling important biological processes in live cells. Here we present an approach to deliver DO strucures into live cell nuclei. We show that labelled DOs do not undergo detectable structural degradation in cell culture media or human cell extracts for 24 hr. To deliver DO platforms into the nuclei of human U2OS cells, we conjugated 30 nm long DO nanorods with an antibody raised against the largest subunit of RNA Polymerase II (Pol II), a key enzyme involved in gene transcription. We find that DOs remain structurally intact in cells for 24hr, including within the nucleus. Using fluorescence microscopy we demonstrate that the electroporated anti-Pol II antibody conjugated DOs are efficiently piggybacked into nuclei and exihibit sub-diffusive motion inside the nucleus. Our results reveal that functionalizing DOs with an antibody raised against a nuclear factor is a highly effective method for the delivery of nanodevices into live cell nuclei.\nContrast-enhanced photon-counting micro-CT of tumor xenograft models\nAuthors: Li, M.; Guo, X.; Verma, A.; Rudkouskaya, A.; McKenna, A.; Intes, X. R. M.; Wang, G.; Barroso, M.\nScore: 5.2, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574097\nPhoton-counting micro computed tomography (micro-CT) offers new potential in preclinical imaging, particularly in distinguishing materials. It becomes especially helpful when combined with contrast agents, enabling the differentiation of tumors from surrounding tissues. There are mainly two types of contrast agents in the market for micro-CT: small molecule-based and nanoparticle-based. However, despite their widespread use in liver tumor studies, there is a notable gap in research on the application of these commercially available agents for photon-counting micro-CT in breast and ovarian tumors. Herein, we explored the effectiveness of these agents in differentiating tumor xenografts from various origins (AU565, MDA-MB-231, and SKOV-3) in nude mice, using photon-counting micro-CT. Specifically, ISOVUE-370 (a small molecule-based agent) and Exitrone Nano 12000 (a nanoparticle-based agent) were investigated in this context. To improve tumor visualization, we proposed a novel color visualization method for photon-counting micro-CT, which changes color tones to highlight contrast media distribution, offering a robust alternative to traditional material decomposition methods with less computational demand. Our in vivo experiments confirm its effectiveness, showing distinct enhancement characteristics for each contrast agent. Qualitative and quantitative analyses suggested that Exitrone Nano 12000 provides superior vasculature enhancement and better quantitative consistency across scans, while ISOVUE-370 gives more comprehensive tumor enhancement but with a significant variance between scans due to its short blood half-time. This variability leads to high sensitivity to timing and individual differences among mice. Further, a paired t-test on mean and standard deviation values within tumor volumes showed significant differences between the AU565 and SKOV-3 tumor models with the nanoparticle-based (p-values \u003c 0.02), attributable to their distinct vascularity, as confirmed by immunohistochemistry. These findings underscore the utility of photon-counting micro-CT in non-invasively assessing the morphology and anatomy of different tumor xenografts, which is crucial for tumor characterization and longitudinal monitoring of tumor development and response to treatments.\nHigh-resolution low-cost LCD 3D printing of microfluidics\nAuthors: Shafique, H.; Karamzadeh, V.; Kim, G.; Morocz, Y.; Sohrabi-Kashani, A.; Shen, M. L.; Juncker, D.\nScore: 4.6, Published: 2024-01-02 DOI: 10.1101/2023.12.31.573772\nThe fabrication of microfluidic devices has progressed from cleanroom manufacturing to replica molding in polymers, and more recently to direct manufacturing by subtractive (e.g., laser machining) and additive (e.g., 3D printing) techniques, notably digital light processing (DLP) photopolymerization. However, many methods require technical expertise and while DLP 3D printers remain expensive at a cost [~]15-30K USD with [~]8M pixels that are 25-40 {micro}m in size. Here, we introduce (i) the use of low-cost ([~]150-600 USD) liquid crystal display (LCD) photopolymerization 3D printing with [~]8M-58M pixels that are 18-35 {micro}m in size for direct microfluidic device fabrication and (ii) a poly(ethylene glycol) diacrylate-based ink developed for LCD 3D printing (PLInk). We optimized PLInk for high resolution, fast 3D printing and biocompatibility while considering the illumination inhomogeneity and low power density of LCD 3D printers. We made lateral features as small as 75 {micro}m, 22-{micro}m-thick embedded membranes, and circular channels with a 110 {micro}m radius. We 3D printed microfluidic devices previously manufactured by other methods, including an embedded 3D micromixer, a membrane microvalve, and an autonomous capillaric circuit (CC) deployed for interferon-{gamma} detection with excellent performance (limit of detection: 12 pg mL-1, CV: 6.8%), and we demonstrated compatibility with cell culture. Finally, large area manufacturing was illustrated by printing 42 CCs with embedded microchannels in \u003c45 min. LCD 3D printing together with tailored inks pave the way for democratizing access to high-resolution manufacturing of ready-to-use microfluidic devices by anyone, anywhere.\nHow Older Adults Maintain Lateral Balance While Walking on Narrowing Paths\nAuthors: Kazanski, M. E.; Cusumano, J. P.; Dingwell, J. B.\nScore: 3.0, Published: 2024-01-04 DOI: 10.1101/2024.01.04.574192\nBackgroundOlder adults have difficulty maintaining side-to-side balance while navigating daily environments. Losing balance in such circumstances can lead to falls. We need to better understand how older adults adapt lateral balance to navigate environment-imposed task constraints. Research QuestionHow do older adults adjust mediolateral balance while walking along continually-narrowing paths, and what are the stability implications of these adjustments? MethodsEighteen older (71.6{+/-}6.0 years) and twenty younger (21.7{+/-}2.6 years) healthy adults traversed 25m-long paths that gradually narrowed from 45cm to 5cm. Participants switched onto an adjacent path when they chose. We quantified participants lateral center-of-mass dynamics and lateral Margins of Stability (MoSL) as paths narrowed. We quantified lateral Probability of Instability (PoIL) as the cumulative probability that participants would take a laterally unstable (MoSL\u003c0) step as they walked. We also extracted these outcomes where participants switched paths. ResultsAs paths narrowed, all participants exhibited progressively smaller average MoSL and increasingly larger PoIL. However, their MoSL variability was largest at both the narrowest and widest path sections. Older adults exhibited consistently both larger average and more variable MoSL across path widths. Taken into account together, these resulted in either comparable or somewhat larger PoIL as paths narrowed. Older adults left the narrowing paths sooner, on average, than younger. As they did so, older adults exhibited significantly larger average and more variable MoSL, but somewhat smaller PoIL than younger. SignificanceOur results directly challenge the predominant interpretation that larger average MoSL as indicating \"greater stability\", which we argue is inconsistent with the principles underlying its derivation. In contrast, analyzing step-to-step gait dynamics, together with estimating PoIL allows one to properly quantify instability risk. Furthermore, the adaptive strategies uncovered using these methods suggest potential targets for future interventions to reduce falls in older adults.\nMicrobiome-induced Increases and Decreases in Bone Tissue Strength can be Initiated After Skeletal Maturity\nAuthors: Liu, C.; Cyphert, E. L.; Stephen, S. J.; Wang, B.; Morales, A.; Nixon, J.; Natsoulas, N.; Garcia, M.; Blazquez Carmona, P.; Vill, A.; Donnelly, E.; Brito, I. L.; Vashishth, D.; Hernandez, C. J.\nScore: 3.0, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574074\nRecent studies in mice have indicated that the gut microbiome can regulate bone tissue strength. However, prior work involved modifications to the gut microbiome in growing animals and it is unclear if the same changes in the microbiome, applied later in life, would change matrix strength. Here we changed the composition of the gut microbiome before and/or after skeletal maturity (16 weeks of age) using oral antibiotics (ampicillin + neomycin). Male and female mice (n=143 total, n=12-17/group/sex) were allocated into five study groups:1) Unaltered, 2) Continuous (dosing 4-24 weeks of age), 3) Delayed (dosing only 16-24 weeks of age), 4) Initial (dosing 4-16 weeks of age, suspended at 16 weeks), and 5) Reconstituted (dosing from 4-16 weeks following by fecal microbiota transplant from Unaltered donors). Animals were euthanized at 24 weeks of age. In males, bone matrix strength in the femur was 25-35% less than expected from geometry in mice from the Continuous (p= 0.001), Delayed (p= 0.005), and Initial (p=0.040) groups as compared to Unaltered. Reconstitution of the gut microbiota, however, led to a bone matrix strength similar to Unaltered animals (p=0.929). In females, microbiome-induced changes in bone matrix strength followed the same trend as males but were not significantly different, demonstrating sex-related differences in the response of bone matrix to the gut microbiota. Minor differences in chemical composition of bone matrix were observed (Raman spectroscopy). Our findings indicate that microbiome-induced impairment of bone matrix in males can be initiated and/or reversed after skeletal maturity. The portion of the femoral cortical bone formed after skeletal maturity (16 weeks) is small; however, this suggests that microbiome-induced changes in bone matrix occur without osteoblast/osteoclast turnover using an, as of yet unidentified mechanism. These findings add to evidence that the mechanical properties of bone matrix can be altered in the adult skeleton.\nA Combinatorial Strategy for HRV 3C Protease Engineering to Achieve the N-terminal Free Cleavage\nAuthors: Yi, L.; Zhang, G.; Mei, M.; Fan, X.; Zhou, Y.; Zhang, F.\nScore: 2.8, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574269\nHuman rhinovirus 3C protease (HRV 3C-P) has a high specificity against the substrate of LEVLFQG at P1' site, which plays an important role in biotechnology and academia as a fusion tag removal tool. However, a non-ignorable limitation is that an extra residue of Gly would remain at the N terminus of the recombinant target protein after cleavage with HRV 3C-P, thus potentially causing protein mis-functionality or immunogenicity. Here, we developed a combinatorial strategy by integrating structure-guided library design and high-throughput screening of eYESS approach for HRV 3C-P engineering to expand its P1' specificity. Finally, a C3 variant was obtained, exhibiting a broad substrate P1' specificity to recognize 20 different amino acids with the highest activity against LEVLFQM (kcat/KM = 3.72 +/- 0.04 mM-1.s-1). Further biochemical and NGS-mediated substrate profiling analysis showed that C3 variant still kept its substrate stringency at P1 site and good residue tolerance at P2'site, but with an expanded P1' specificity. Structural simulation of C3 indicated a reconstructed S1' binding pocket as well as new interactions with the substrates. Overall, our studies here prompt not only the practical applications and understanding of substrate recognition mechanisms of HRV 3C-P, also provide new tools for other enzyme engineering.\nHuman cell surface-AAV interactomes identify LRP6 as blood-brain-barrier transcytosis receptor and immune cytokine IL3 as AAV9 binder\nAuthors: Shay, T. F.; Jang, S.; Chen, X.; Walker, B.; Tebbutt, C.; Wolfe, D. A.; Brittain, T. J.; Arokiaraj, C. M.; Sullivan, E. E.; Ding, X.; Wang, T.-Y.; Lei, Y.; Chuapoco, M. R.; Chou, T.-F.; Gradinaru, V.\nScore: 2.5, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574399\nAdeno-associated viruses (AAVs) are foundational gene delivery tools for basic science and clinical therapeutics. However, lack of mechanistic insight, especially for engineered vectors created by directed evolution, can hamper their application. Here, we adapted an unbiased human cell microarray platform to determine the extracellular and cell surface interactomes of natural and engineered AAVs. We identified a naturally-evolved and serotype-specific interaction between the AAV9 capsid and human interleukin 3 (IL3), with possible roles in host immune modulation, as well as lab-evolved low-density-lipoprotein-receptor-related-protein 6 (LRP6) interactions specific to engineered capsids that cross the blood-brain barrier in non-human primates after intravenous administration. The unbiased cell microarray screening approach also allowed us to identify off-target tissue binding interactions of engineered brain-enriched AAV capsids that may inform vector peripheral organ tropism and side effects. These results allow confident application of engineered AAVs in diverse organisms and unlock future target-informed engineering of improved viral and non-viral vectors for non-invasive therapeutic delivery to the brain.\nHuman cell surface-AAV interactomes identify LRP6 as blood-brain-barrier transcytosis receptor and immune cytokine IL3 as AAV9 binder\nAuthors: Shay, T. F.; Jang, S.; Chen, X.; Walker, B.; Tebbutt, C.; Wolfe, D. A.; Brittain, T. J.; Arokiaraj, C. M.; Sullivan, E. E.; Ding, X.; Wang, T.-Y.; Lei, Y.; Chuapoco, M. R.; Chou, T.-F.; Gradinaru, V.\nScore: 2.5, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574399\nAdeno-associated viruses (AAVs) are foundational gene delivery tools for basic science and clinical therapeutics. However, lack of mechanistic insight, especially for engineered vectors created by directed evolution, can hamper their application. Here, we adapted an unbiased human cell microarray platform to determine the extracellular and cell surface interactomes of natural and engineered AAVs. We identified a naturally-evolved and serotype-specific interaction between the AAV9 capsid and human interleukin 3 (IL3), with possible roles in host immune modulation, as well as lab-evolved low-density-lipoprotein-receptor-related-protein 6 (LRP6) interactions specific to engineered capsids that cross the blood-brain barrier in non-human primates after intravenous administration. The unbiased cell microarray screening approach also allowed us to identify off-target tissue binding interactions of engineered brain-enriched AAV capsids that may inform vector peripheral organ tropism and side effects. These results allow confident application of engineered AAVs in diverse organisms and unlock future target-informed engineering of improved viral and non-viral vectors for non-invasive therapeutic delivery to the brain.\nInsights into Yeast Response to Chemotherapeutic Agent through Time Series Genome-Scale Metabolic Models\nAuthors: Karabekmez, M. E.\nScore: 2.5, Published: 2024-01-03 DOI: 10.1101/2024.01.02.573982\nBackgroundOrganism-specific genome-scale metabolic models (GSMMs) can unveil molecular mechanisms within cells and are commonly used in diverse applications, from synthetic biology, biotechnology, and systems biology to metabolic engineering. There are limited studies incorporating time-series transcriptomics in GSMM simulations. Yeast is an easy-to-manipulate model organism for tumor research; MethodsHere, a novel approach (TS-GSMM) was proposed to integrate time-series transcriptomics with GSMMs to narrow down the feasible solution space of all possible flux distributions and attain time-series flux samples. The flux samples were clustered using machine learning techniques, and the clusters functional analysis was performed using reaction set enrichment analysis; ResultsA time series transcriptomics response of Yeast cells to a chemotherapeutic reagent - doxorubicin - was mapped onto a Yeast GSMM. Eleven flux clusters were obtained with our approach, and pathway dynamics were displayed. Induction of fluxes related to bicarbonate formation and transport, ergosterol and spermidine transport, and ATP production were captured; ConclusionsIntegrating time-series transcriptomics data with GSMMs is a promising approach to reveal pathway dynamics without any kinetic modeling and detects pathways that cannot be identified through transcriptomics-only analysis. The codes are available at https://github.com/karabekmez/TS-GSMM.\n",
  "wordCount" : "2553",
  "inLanguage": "en",
  "datePublished": "2024-01-07T10:38:59Z",
  "dateModified": "2024-01-07T10:38:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 7, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574272">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574272" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574272">
        <p class="paperTitle">Antigen-display exosomes provide adjuvant-free protection against SARS-CoV-2 disease at nanogram levels of spike protein</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574272" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574272" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guo, C.; Sachithanandham, J.; Zhong, W.; Craney, M.; Villano, J.; Pekosz, A.; Gould, S.</p>
        <p class="info">Score: 11.4, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574272' target='https://doi.org/10.1101/2024.01.04.574272'> 10.1101/2024.01.04.574272</a></p>
        <p class="abstract">As the only bionormal nanovesicle, exosomes have high potential as a nanovesicle for delivering vaccines and therapeutics. We show here that the loading of type-1 membrane proteins into the exosome membrane is induced by exosome membrane anchor domains, EMADs, that maximize protein delivery to the plasma membrane, minimize protein sorting to other compartments, and direct proteins into exosome membranes. Using SARS-CoV-2 spike as an example and EMAD13 as our most effective exosome membrane anchor, we show that cells expressing a spike-EMAD13 fusion protein produced exosomes that carry dense arrays of spike trimers on 50% of all exosomes. Moreover, we find that immunization with spike-EMAD13 exosomes induced strong neutralizing antibody responses and protected hamsters against SARS-CoV-2 disease at doses of just 0.5-5 ng of spike protein, without adjuvant, demonstrating that antigen-display exosomes are particularly immunogenic, with important implications for both structural and expression-dependent vaccines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573746">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573746" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573746">
        <p class="paperTitle">Piggybacking functionalized DNA nanostructures into live cell nuclei</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573746" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573746" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roozbahani, G. M.; Colosi, P.; Oravecz, A.; Sorokina, E. M.; Pfeifer, W.; Shokri, S.; Wei, Y.; Didier, P.; DeLuca, M.; Arya, G.; Tora, L.; Lakadamyali, M.; Poirier, M. G.; Castro, C.</p>
        <p class="info">Score: 8.6, Published: 2024-01-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573746' target='https://doi.org/10.1101/2023.12.30.573746'> 10.1101/2023.12.30.573746</a></p>
        <p class="abstract">DNA origami (DO) are promising tools for in vitro or in vivo applications including drug delivery; biosensing, detecting biomolecules; and probing chromatin sub-structures. Targeting these nanodevices to mammalian cell nuclei could provide impactful approaches for probing visualizing and controlling important biological processes in live cells. Here we present an approach to deliver DO strucures into live cell nuclei. We show that labelled DOs do not undergo detectable structural degradation in cell culture media or human cell extracts for 24 hr. To deliver DO platforms into the nuclei of human U2OS cells, we conjugated 30 nm long DO nanorods with an antibody raised against the largest subunit of RNA Polymerase II (Pol II), a key enzyme involved in gene transcription. We find that DOs remain structurally intact in cells for 24hr, including within the nucleus. Using fluorescence microscopy we demonstrate that the electroporated anti-Pol II antibody conjugated DOs are efficiently piggybacked into nuclei and exihibit sub-diffusive motion inside the nucleus. Our results reveal that functionalizing DOs with an antibody raised against a nuclear factor is a highly effective method for the delivery of nanodevices into live cell nuclei.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574097">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574097" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574097">
        <p class="paperTitle">Contrast-enhanced photon-counting micro-CT of tumor xenograft models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574097" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574097" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, M.; Guo, X.; Verma, A.; Rudkouskaya, A.; McKenna, A.; Intes, X. R. M.; Wang, G.; Barroso, M.</p>
        <p class="info">Score: 5.2, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574097' target='https://doi.org/10.1101/2024.01.03.574097'> 10.1101/2024.01.03.574097</a></p>
        <p class="abstract">Photon-counting micro computed tomography (micro-CT) offers new potential in preclinical imaging, particularly in distinguishing materials. It becomes especially helpful when combined with contrast agents, enabling the differentiation of tumors from surrounding tissues. There are mainly two types of contrast agents in the market for micro-CT: small molecule-based and nanoparticle-based. However, despite their widespread use in liver tumor studies, there is a notable gap in research on the application of these commercially available agents for photon-counting micro-CT in breast and ovarian tumors. Herein, we explored the effectiveness of these agents in differentiating tumor xenografts from various origins (AU565, MDA-MB-231, and SKOV-3) in nude mice, using photon-counting micro-CT. Specifically, ISOVUE-370 (a small molecule-based agent) and Exitrone Nano 12000 (a nanoparticle-based agent) were investigated in this context. To improve tumor visualization, we proposed a novel color visualization method for photon-counting micro-CT, which changes color tones to highlight contrast media distribution, offering a robust alternative to traditional material decomposition methods with less computational demand. Our in vivo experiments confirm its effectiveness, showing distinct enhancement characteristics for each contrast agent. Qualitative and quantitative analyses suggested that Exitrone Nano 12000 provides superior vasculature enhancement and better quantitative consistency across scans, while ISOVUE-370 gives more comprehensive tumor enhancement but with a significant variance between scans due to its short blood half-time. This variability leads to high sensitivity to timing and individual differences among mice. Further, a paired t-test on mean and standard deviation values within tumor volumes showed significant differences between the AU565 and SKOV-3 tumor models with the nanoparticle-based (p-values &lt; 0.02), attributable to their distinct vascularity, as confirmed by immunohistochemistry. These findings underscore the utility of photon-counting micro-CT in non-invasively assessing the morphology and anatomy of different tumor xenografts, which is crucial for tumor characterization and longitudinal monitoring of tumor development and response to treatments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.31.573772">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.31.573772" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.31.573772">
        <p class="paperTitle">High-resolution low-cost LCD 3D printing of microfluidics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.31.573772" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.31.573772" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shafique, H.; Karamzadeh, V.; Kim, G.; Morocz, Y.; Sohrabi-Kashani, A.; Shen, M. L.; Juncker, D.</p>
        <p class="info">Score: 4.6, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.31.573772' target='https://doi.org/10.1101/2023.12.31.573772'> 10.1101/2023.12.31.573772</a></p>
        <p class="abstract">The fabrication of microfluidic devices has progressed from cleanroom manufacturing to replica molding in polymers, and more recently to direct manufacturing by subtractive (e.g., laser machining) and additive (e.g., 3D printing) techniques, notably digital light processing (DLP) photopolymerization. However, many methods require technical expertise and while DLP 3D printers remain expensive at a cost [~]15-30K USD with [~]8M pixels that are 25-40 {micro}m in size. Here, we introduce (i) the use of low-cost ([~]150-600 USD) liquid crystal display (LCD) photopolymerization 3D printing with [~]8M-58M pixels that are 18-35 {micro}m in size for direct microfluidic device fabrication and (ii) a poly(ethylene glycol) diacrylate-based ink developed for LCD 3D printing (PLInk). We optimized PLInk for high resolution, fast 3D printing and biocompatibility while considering the illumination inhomogeneity and low power density of LCD 3D printers. We made lateral features as small as 75 {micro}m, 22-{micro}m-thick embedded membranes, and circular channels with a 110 {micro}m radius. We 3D printed microfluidic devices previously manufactured by other methods, including an embedded 3D micromixer, a membrane microvalve, and an autonomous capillaric circuit (CC) deployed for interferon-{gamma} detection with excellent performance (limit of detection: 12 pg mL-1, CV: 6.8%), and we demonstrated compatibility with cell culture. Finally, large area manufacturing was illustrated by printing 42 CCs with embedded microchannels in &lt;45 min. LCD 3D printing together with tailored inks pave the way for democratizing access to high-resolution manufacturing of ready-to-use microfluidic devices by anyone, anywhere.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574192">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574192" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574192">
        <p class="paperTitle">How Older Adults Maintain Lateral Balance While Walking on Narrowing Paths</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574192" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574192" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kazanski, M. E.; Cusumano, J. P.; Dingwell, J. B.</p>
        <p class="info">Score: 3.0, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574192' target='https://doi.org/10.1101/2024.01.04.574192'> 10.1101/2024.01.04.574192</a></p>
        <p class="abstract">BackgroundOlder adults have difficulty maintaining side-to-side balance while navigating daily environments. Losing balance in such circumstances can lead to falls. We need to better understand how older adults adapt lateral balance to navigate environment-imposed task constraints.

Research QuestionHow do older adults adjust mediolateral balance while walking along continually-narrowing paths, and what are the stability implications of these adjustments?

MethodsEighteen older (71.6{&#43;/-}6.0 years) and twenty younger (21.7{&#43;/-}2.6 years) healthy adults traversed 25m-long paths that gradually narrowed from 45cm to 5cm. Participants switched onto an adjacent path when they chose. We quantified participants lateral center-of-mass dynamics and lateral Margins of Stability (MoSL) as paths narrowed. We quantified lateral Probability of Instability (PoIL) as the cumulative probability that participants would take a laterally unstable (MoSL&lt;0) step as they walked. We also extracted these outcomes where participants switched paths.

ResultsAs paths narrowed, all participants exhibited progressively smaller average MoSL and increasingly larger PoIL. However, their MoSL variability was largest at both the narrowest and widest path sections. Older adults exhibited consistently both larger average and more variable MoSL across path widths. Taken into account together, these resulted in either comparable or somewhat larger PoIL as paths narrowed. Older adults left the narrowing paths sooner, on average, than younger. As they did so, older adults exhibited significantly larger average and more variable MoSL, but somewhat smaller PoIL than younger.

SignificanceOur results directly challenge the predominant interpretation that larger average MoSL as indicating &#34;greater stability&#34;, which we argue is inconsistent with the principles underlying its derivation. In contrast, analyzing step-to-step gait dynamics, together with estimating PoIL allows one to properly quantify instability risk. Furthermore, the adaptive strategies uncovered using these methods suggest potential targets for future interventions to reduce falls in older adults.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574074" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574074">
        <p class="paperTitle">Microbiome-induced Increases and Decreases in Bone Tissue Strength can be Initiated After Skeletal Maturity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, C.; Cyphert, E. L.; Stephen, S. J.; Wang, B.; Morales, A.; Nixon, J.; Natsoulas, N.; Garcia, M.; Blazquez Carmona, P.; Vill, A.; Donnelly, E.; Brito, I. L.; Vashishth, D.; Hernandez, C. J.</p>
        <p class="info">Score: 3.0, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574074' target='https://doi.org/10.1101/2024.01.03.574074'> 10.1101/2024.01.03.574074</a></p>
        <p class="abstract">Recent studies in mice have indicated that the gut microbiome can regulate bone tissue strength. However, prior work involved modifications to the gut microbiome in growing animals and it is unclear if the same changes in the microbiome, applied later in life, would change matrix strength. Here we changed the composition of the gut microbiome before and/or after skeletal maturity (16 weeks of age) using oral antibiotics (ampicillin &#43; neomycin). Male and female mice (n=143 total, n=12-17/group/sex) were allocated into five study groups:1) Unaltered, 2) Continuous (dosing 4-24 weeks of age), 3) Delayed (dosing only 16-24 weeks of age), 4) Initial (dosing 4-16 weeks of age, suspended at 16 weeks), and 5) Reconstituted (dosing from 4-16 weeks following by fecal microbiota transplant from Unaltered donors). Animals were euthanized at 24 weeks of age. In males, bone matrix strength in the femur was 25-35% less than expected from geometry in mice from the Continuous (p= 0.001), Delayed (p= 0.005), and Initial (p=0.040) groups as compared to Unaltered. Reconstitution of the gut microbiota, however, led to a bone matrix strength similar to Unaltered animals (p=0.929). In females, microbiome-induced changes in bone matrix strength followed the same trend as males but were not significantly different, demonstrating sex-related differences in the response of bone matrix to the gut microbiota. Minor differences in chemical composition of bone matrix were observed (Raman spectroscopy). Our findings indicate that microbiome-induced impairment of bone matrix in males can be initiated and/or reversed after skeletal maturity. The portion of the femoral cortical bone formed after skeletal maturity (16 weeks) is small; however, this suggests that microbiome-induced changes in bone matrix occur without osteoblast/osteoclast turnover using an, as of yet unidentified mechanism. These findings add to evidence that the mechanical properties of bone matrix can be altered in the adult skeleton.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574269">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574269" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574269">
        <p class="paperTitle">A Combinatorial Strategy for HRV 3C Protease Engineering to Achieve the N-terminal Free Cleavage</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574269" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574269" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yi, L.; Zhang, G.; Mei, M.; Fan, X.; Zhou, Y.; Zhang, F.</p>
        <p class="info">Score: 2.8, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574269' target='https://doi.org/10.1101/2024.01.05.574269'> 10.1101/2024.01.05.574269</a></p>
        <p class="abstract">Human rhinovirus 3C protease (HRV 3C-P) has a high specificity against the substrate of LEVLFQG at P1&#39; site, which plays an important role in biotechnology and academia as a fusion tag removal tool. However, a non-ignorable limitation is that an extra residue of Gly would remain at the N terminus of the recombinant target protein after cleavage with HRV 3C-P, thus potentially causing protein mis-functionality or immunogenicity. Here, we developed a combinatorial strategy by integrating structure-guided library design and high-throughput screening of eYESS approach for HRV 3C-P engineering to expand its P1&#39; specificity. Finally, a C3 variant was obtained, exhibiting a broad substrate P1&#39; specificity to recognize 20 different amino acids with the highest activity against LEVLFQM (kcat/KM = 3.72 &#43;/- 0.04 mM-1.s-1). Further biochemical and NGS-mediated substrate profiling analysis showed that C3 variant still kept its substrate stringency at P1 site and good residue tolerance at P2&#39;site, but with an expanded P1&#39; specificity. Structural simulation of C3 indicated a reconstructed S1&#39; binding pocket as well as new interactions with the substrates. Overall, our studies here prompt not only the practical applications and understanding of substrate recognition mechanisms of HRV 3C-P, also provide new tools for other enzyme engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574399">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574399" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574399">
        <p class="paperTitle">Human cell surface-AAV interactomes identify LRP6 as blood-brain-barrier transcytosis receptor and immune cytokine IL3 as AAV9 binder</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574399" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574399" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shay, T. F.; Jang, S.; Chen, X.; Walker, B.; Tebbutt, C.; Wolfe, D. A.; Brittain, T. J.; Arokiaraj, C. M.; Sullivan, E. E.; Ding, X.; Wang, T.-Y.; Lei, Y.; Chuapoco, M. R.; Chou, T.-F.; Gradinaru, V.</p>
        <p class="info">Score: 2.5, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574399' target='https://doi.org/10.1101/2024.01.05.574399'> 10.1101/2024.01.05.574399</a></p>
        <p class="abstract">Adeno-associated viruses (AAVs) are foundational gene delivery tools for basic science and clinical therapeutics. However, lack of mechanistic insight, especially for engineered vectors created by directed evolution, can hamper their application. Here, we adapted an unbiased human cell microarray platform to determine the extracellular and cell surface interactomes of natural and engineered AAVs. We identified a naturally-evolved and serotype-specific interaction between the AAV9 capsid and human interleukin 3 (IL3), with possible roles in host immune modulation, as well as lab-evolved low-density-lipoprotein-receptor-related-protein 6 (LRP6) interactions specific to engineered capsids that cross the blood-brain barrier in non-human primates after intravenous administration. The unbiased cell microarray screening approach also allowed us to identify off-target tissue binding interactions of engineered brain-enriched AAV capsids that may inform vector peripheral organ tropism and side effects. These results allow confident application of engineered AAVs in diverse organisms and unlock future target-informed engineering of improved viral and non-viral vectors for non-invasive therapeutic delivery to the brain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574399">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574399" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574399">
        <p class="paperTitle">Human cell surface-AAV interactomes identify LRP6 as blood-brain-barrier transcytosis receptor and immune cytokine IL3 as AAV9 binder</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574399" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574399" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shay, T. F.; Jang, S.; Chen, X.; Walker, B.; Tebbutt, C.; Wolfe, D. A.; Brittain, T. J.; Arokiaraj, C. M.; Sullivan, E. E.; Ding, X.; Wang, T.-Y.; Lei, Y.; Chuapoco, M. R.; Chou, T.-F.; Gradinaru, V.</p>
        <p class="info">Score: 2.5, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574399' target='https://doi.org/10.1101/2024.01.05.574399'> 10.1101/2024.01.05.574399</a></p>
        <p class="abstract">Adeno-associated viruses (AAVs) are foundational gene delivery tools for basic science and clinical therapeutics. However, lack of mechanistic insight, especially for engineered vectors created by directed evolution, can hamper their application. Here, we adapted an unbiased human cell microarray platform to determine the extracellular and cell surface interactomes of natural and engineered AAVs. We identified a naturally-evolved and serotype-specific interaction between the AAV9 capsid and human interleukin 3 (IL3), with possible roles in host immune modulation, as well as lab-evolved low-density-lipoprotein-receptor-related-protein 6 (LRP6) interactions specific to engineered capsids that cross the blood-brain barrier in non-human primates after intravenous administration. The unbiased cell microarray screening approach also allowed us to identify off-target tissue binding interactions of engineered brain-enriched AAV capsids that may inform vector peripheral organ tropism and side effects. These results allow confident application of engineered AAVs in diverse organisms and unlock future target-informed engineering of improved viral and non-viral vectors for non-invasive therapeutic delivery to the brain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573982">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573982" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573982">
        <p class="paperTitle">Insights into Yeast Response to Chemotherapeutic Agent through Time Series Genome-Scale Metabolic Models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573982" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573982" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Karabekmez, M. E.</p>
        <p class="info">Score: 2.5, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573982' target='https://doi.org/10.1101/2024.01.02.573982'> 10.1101/2024.01.02.573982</a></p>
        <p class="abstract">BackgroundOrganism-specific genome-scale metabolic models (GSMMs) can unveil molecular mechanisms within cells and are commonly used in diverse applications, from synthetic biology, biotechnology, and systems biology to metabolic engineering. There are limited studies incorporating time-series transcriptomics in GSMM simulations. Yeast is an easy-to-manipulate model organism for tumor research;

MethodsHere, a novel approach (TS-GSMM) was proposed to integrate time-series transcriptomics with GSMMs to narrow down the feasible solution space of all possible flux distributions and attain time-series flux samples. The flux samples were clustered using machine learning techniques, and the clusters functional analysis was performed using reaction set enrichment analysis;

ResultsA time series transcriptomics response of Yeast cells to a chemotherapeutic reagent - doxorubicin - was mapped onto a Yeast GSMM. Eleven flux clusters were obtained with our approach, and pathway dynamics were displayed. Induction of fluxes related to bicarbonate formation and transport, ergosterol and spermidine transport, and ATP production were captured;

ConclusionsIntegrating time-series transcriptomics data with GSMMs is a promising approach to reveal pathway dynamics without any kinetic modeling and detects pathways that cannot be identified through transcriptomics-only analysis. The codes are available at https://github.com/karabekmez/TS-GSMM.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
