<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Simultaneous multi-site editing of individual genomes using retron arrays
Authors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.
Score: 27.2, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549397
Our understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Simultaneous multi-site editing of individual genomes using retron arrays
Authors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.
Score: 27.2, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549397
Our understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-23T10:36:34+00:00" />
<meta property="article:modified_time" content="2023-07-23T10:36:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Simultaneous multi-site editing of individual genomes using retron arrays
Authors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.
Score: 27.2, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549397
Our understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Simultaneous multi-site editing of individual genomes using retron arrays\nAuthors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.\nScore: 27.2, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549397\nOur understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites.",
  "keywords": [
    
  ],
  "articleBody": " Simultaneous multi-site editing of individual genomes using retron arrays\nAuthors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.\nScore: 27.2, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549397\nOur understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites. Unfortunately, building cells with multiple, non-adjacent precise mutations remains a laborious cycle of editing, isolating an edited cell, and editing again. Here, we describe a technology for precisely modifying multiple sites on a single genome simultaneously. This technology - termed a multitron - is built from a heavily modified retron, in which multiple donor-encoding msds are produced from a single transcript. The multitron architecture is compatible with both recombineering in prokaryotic cells and CRISPR editing in eukaryotic cells. We demonstrate applications for this approach in molecular recording, genetic element minimization, and metabolic engineering.\nEliminating Malaria Vectors with Precision Guided Sterile Males\nAuthors: Smidler, A. L.; Apte, R. A.; Pai, J. J.; Chow, M. L.; Chen, S.; Mondal, A.; Sanchez C., H. M.; Antoshechkin, I.; Marshall, J. M.; Akbari, O. S.\nScore: 10.2, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549947\nControlling the principal African malaria vector, the mosquito Anopheles gambiae, is considered essential to curtail malaria transmission. However existing vector control technologies rely on insecticides, which are becoming increasingly ineffective. Sterile insect technique (SIT) is a powerful suppression approach that has successfully eradicated a number of insect pests, yet the A. gambiae toolkit lacks the requisite technologies for its implementation. SIT relies on iterative mass-releases of non-biting, non-driving, sterile males which seek out and mate with monandrous wild females. Once mated, females are permanently sterilized due to mating-induced refractoriness, which results in population suppression of the subsequent generation. However, sterilization by traditional methods renders males unfit, making the creation of precise genetic sterilization methods imperative. Here we develop precision guided Sterile Insect Technique (pgSIT) in the mosquito A. gambiae for inducible, programmed male-sterilization and female-elimination for wide scale use in SIT campaigns. Using a binary CRISPR strategy, we cross separate engineered Cas9 and gRNA strains to disrupt male-fertility and female-essential genes, yielding \u003e99.5% male-sterility and \u003e99.9% female-lethality in hybrid progeny. We demonstrate that these genetically sterilized males have good longevity, are able to induce population suppression in cage trials, and are predicted to eliminate wild A. gambiae populations using mathematical models, making them ideal candidates for release. This work provides a valuable addition to the malaria genetic biocontrol toolkit, for the first time enabling scalable SIT-like confinable suppression in the species.\nIn vivo fate of systemically administered encapsulin protein nanocages and implications for their use in targeted drug delivery\nAuthors: Rennie, C.; Sives, C.; Boyton, I.; Diaz, D.; Gorrie, C. A.; Vittorio, O.; Collins-Praino, L.; Care, A.\nScore: 7.5, Published: 2023-07-18 DOI: 10.1101/2023.07.16.549228\nEncapsulins, self-assembling protein nanocages derived from prokaryotes, are promising nanoparticle-based drug delivery systems (NDDS). However, the in vivo behavior and fate of encapsulins are poorly understood. In this pre-clinical study, we probe the interactions between the model encapsulin from Thermotoga maritima (TmEnc) and key biological barriers encountered by NDDS. Here, a purified TmEnc formulation that exhibited colloidal stability, storability, and blood compatibility was intravenously injected into BALB/c mice. TmEnc had an excellent nanosafety profile, with no abnormal weight loss or gross pathology observed, and only temporary alterations in toxicity biomarkers detected. Notably, TmEnc demonstrated immunogenic properties, inducing the generation of nanocage-specific IgM and IgG antibodies, but without any prolonged pro-inflammatory effects. An absence of antibody cross-reactivity also suggested immune-orthogonality among encapsulins systems. Moreover, TmEnc formed a serum-derived protein corona on its surface which changed dynamically and appeared to play a role in immune recognition. TmEncs biodistribution profile further revealed its sequestration from the blood circulation by the liver and then biodegraded within Kupffer cells, thus indicating clearance via the mononuclear phagocyte system. Collectively, these findings provide critical insights into how encapsulins behave in vivo, thereby informing their future design, modification, and application in targeted drug delivery. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=187 SRC=\"FIGDIR/small/549228v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (33K): org.highwire.dtl.DTLVardef@af6321org.highwire.dtl.DTLVardef@951830org.highwire.dtl.DTLVardef@67090corg.highwire.dtl.DTLVardef@832967_HPS_FORMAT_FIGEXP M_FIG C_FIG\nTowards Modular Engineering of Cell Signalling: Topographically-Textured Microparticles Induce Osteogenesis via Activation of Canonical Hedgehog Signalling\nAuthors: Ghuloum, F. I.; Stevens, L. A.; Johnson, C. A.; Riobo-Del Galdo, N. A.; Amer, M. H.\nScore: 5.7, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549481\nPolymer microparticles possess great potential as functional building blocks for advanced bottom-up engineering of complex tissues. Tailoring the three-dimensional architectural features of culture substrates has been shown to induce osteogenesis in mesenchymal stem cells in vitro, but the molecular mechanisms underpinning this remain unclear. This study proposes a mechanism linking the activation of Hedgehog signalling to the osteoinductive effect of surface-engineered, topographically-textured polymeric microparticles. In this study, mesenchymal progenitor C3H10T1/2 cells were cultured on smooth and dimpled poly(D,L-lactide) microparticles to assess differences in viability, cellular morphology, proliferation, and expression of a range of Hedgehog signalling components and osteogenesis-related genes. Dimpled microparticles induced osteogenesis and activated the Hedgehog signalling pathway relative to smooth microparticles and 2D-cultured controls without the addition of exogenous biochemical factors. We observed upregulation of the osteogenesis markers Runt-related transcription factor2 (Runx2) and bone gamma-carboxyglutamate protein 2 (Bglap2), as well as the Hedgehog signalling components, glioma associated oncogene homolog 1 (Gli1), Patched1 (Ptch1), and Smoothened (Smo). Treatment with the Smo antagonist KAAD-cyclopamine confirmed the involvement of Smo in Gli1 target gene activation, with a significant reduction in the expression of Gli1, Runx2 and Bglap2 (p[\u0026le;]0.001) following KAAD-cyclopamine treatment. Overall, our study demonstrates the role of the topographical microenvironment in the modulation of Hedgehog signalling, highlighting the potential for tailoring substrate topographical design to offer cell-instructive 3D microenvironments. Topographically-textured microparticles allow the modulation of Hedgehog signalling in vitro without adding exogenous biochemical agonists, thereby eliminating potential confounding artefacts in high-throughput drug screening applications. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=134 HEIGHT=200 SRC=\"FIGDIR/small/549481v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (42K): org.highwire.dtl.DTLVardef@e01c27org.highwire.dtl.DTLVardef@1587e56org.highwire.dtl.DTLVardef@ce44e5org.highwire.dtl.DTLVardef@d1f01_HPS_FORMAT_FIGEXP M_FIG C_FIG\nMicrofluidic Quaking-Induced Conversion (Micro-QuIC) for Rapid On-Site Amplification and Detection of Misfolded Proteins\nAuthors: Lee, D. J.; Christenson, P. R.; Rowden, G.; Lindquist, N. C.; Larsen, P. A.; Oh, S.-H.\nScore: 5.5, Published: 2023-07-18 DOI: 10.1101/2023.07.17.549283\nProtein misfolding diseases, such as prion diseases, Alzheimers, and Parkinsons, share a common molecular mechanism involving the misfolding and aggregation of specific proteins. There is an urgent need for point-of-care (POC) diagnostic technologies that can accurately detect these misfolded proteins, facilitating early diagnosis and intervention. Here, we introduce the Microfluidic Quaking Induced Conversion (Micro-QuIC), a novel acoustofluidic platform for the rapid and sensitive detection of protein misfolding diseases. We demonstrate the utility of our technology using chronic wasting disease (CWD) as a model system, as samples from wild white-tailed deer are readily accessible, and CWD shares similarities with human protein misfolding diseases. Acoustofluidic mixing enables homogeneous mixing of reagents in a high-Reynolds-number regime, significantly accelerating the turnaround time for CWD diagnosis. Our Micro-QuIC assay amplifies prions by an order of magnitude faster than the current gold standard, real-time quaking-induced conversion (RT-QuIC). Furthermore, we integrated Micro-QuIC with a gold nanoparticle-based, naked-eye detection method, which enables visual discrimination between CWD positive and negative samples without the need for a bulky fluorescence detection module. This integration creates a rapid, POC testing platform capable of detecting misfolded proteins associated with a variety of protein misfolding diseases. TOC graphic O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=109 SRC=\"FIGDIR/small/549283v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (39K): org.highwire.dtl.DTLVardef@7d9c08org.highwire.dtl.DTLVardef@8ef24borg.highwire.dtl.DTLVardef@c556bcorg.highwire.dtl.DTLVardef@1c16279_HPS_FORMAT_FIGEXP M_FIG C_FIG\nA monochromatically excitable green-red dual-fluorophore fusion incorporating a new large Stokes shift fluorescent protein\nAuthors: Ejike, J. O.; Sadoine, M.; Shen, Y.; Ishikawa, Y.; Sunal, E.; Hänsch, S.; Hamacher, A. B.; Frommer, W. B.; Wudick, M. M.; Campbell, R. E.; Kleist, T. J.\nScore: 3.9, Published: 2023-07-17 DOI: 10.1101/2023.07.16.549156\nGenetically encoded sensors enable quantitative imaging of analytes in live cells. State-of-the-art sensors are commonly constructed by combining ligand-binding domains with one or more sensitized fluorescent protein (FP) domains. Sensors based on a single FP are susceptible to artifacts caused by differing expression levels or sensor distribution in vivo. Hence, our lab developed dual-FP Matryoshka technology introduced by a single cassette that contains a stable large Stokes shift (LSS) reference FP nested within a reporter FP (cpEGFP), allowing simple construction of intensiometric sensors with the capacity for ratiometric quantification. The first-generation Green-Orange (GO) Matryoshka cassette established proof of concept but required custom optical setups to maximize achievable dynamic range. Here, we present a genetically encoded calcium sensor that employs optimized second-generation Green-Apple (GA) Matryoshka technology that incorporates a newly designed red LSSmApple fluorophore. LSSmApple provides improved excitation spectrum overlap with cpEGFP, allowing for monochromatic co-excitation with blue light. The exceptionally large Stokes shift of LSSmApple results in improved emission spectrum separation from cpEGFP, which minimizes fluorophore bleed-through and facilitates imaging using standard dichroics and red fluorescent protein (RFP) emission filters. We developed an image analysis pipeline for yeast (Saccharomyces cerevisiae) timelapse imaging that utilizes LSSmApple to segment and track cells for high-throughput quantitative analysis. In summary, we engineered a new fluorescent protein, constructed a genetically encoded calcium indicator (GA-MatryoshCaMP6s), and performed calcium imaging in yeast as a demonstration.\nBiosensor-Assisted Laboratory Evolution of Malonyl-CoA production in Saccharomyces cerevisiae\nAuthors: Ferreira, R.; Hedin, K. A.; Nielsen, J.; David, F.\nScore: 3.9, Published: 2023-07-17 DOI: 10.1101/2023.07.16.549225\nThe production of bio-based chemicals and fuels through microbial engineering offers a promising and sustainable alternative to petroleum-based fuels and chemicals, with the potential for scalability. However, engineering microbes and continuously evolving them to enhance the production of industrially relevant products is a complex and challenging task, requiring precise selection of genetic traits to achieve desired outcomes. In this study, we report the development of a novel counter-selectable growth-sensitive malonyl-CoA platform strain by coupling the malonyl-CoA repressor FapR from Bacillus subtilis to essential gene promoters involved in glucose growth and the plasma membrane arginine permease. This platform strain was then coupled with a CRISPR-dCas9 guide-RNA (gRNA) library, which after multiple rounds of dilutions and library sequencing, resulted in the enrichment for gRNAs that increased fluxes towards malonyl-CoA. The enriched gRNAs were validated for their effects on growth enhancement, gene regulation, and the production of an industrially relevant malonyl-CoA product, namely 3-hydroxypropionic acid. This study highlights an innovative approach to microbial engineering and opens up avenues for further exploration in the field of laboratory continuous evolution.\nSolid tumor-on-chip model for efficacy and safety assessment of CAR-T cell therapy\nAuthors: Maulana, T. I.; Teufel, C.; Cipriano, M.; Lazarevski, L.; van den Hil, F. E.; Orlova, V.; Koch, A.; Alb, M.; Hudecek, M.; Loskill, P.\nScore: 4.6, Published: 2023-07-15 DOI: 10.1101/2023.07.13.548856\nThe non-clinical assessment of CAR-T cells demands innovative models that are capable of predicting safety and efficacy in the clinical setting. Here, we present a novel solid tumor-on-chip model that allows CAR-T cell perfusion and integrates the vasculature and tumor lesions to recapitulate key events of CAR-T cell performance including extravasation, tumor infiltration and cytokine release. We assessed CAR-T cells targeting the ROR1 antigen against tumor aggregates that were derived from a breast cancer cell line and primary breast cancer organoids. The data show the temporal kinetic of ROR1 CAR-T cell migration and expansion, lytic activity and cytokine production over the course of 8 days, and reveal a correlation between anti-tumor efficacy and ROR1 antigen density on tumor cells. CAR-modified T cells extravasated faster, infiltrated tumor lesions stronger, persisted longer and in higher numbers than non-CAR modified T cells. Intriguingly, we detected cytokine release levels and kinetics typically observed in patients who developed cytokine release syndrome, and administered dasatinib as a pharmacologic OFF switch to control this inflammatory response. The data illustrate the ability of this tumor-on-chip platform to assess parameters associated withherapeutic outcome and the potential to aid in patient stratification and monitoring of CAR-T cell therapy.\nModeling Mechanical Activation of Macrophages During Pulmonary Fibrogenesis for Targeted Anti-Fibrosis Therapy\nAuthors: Xu, Y.; Ying, L.; Lang, J. K.; Hinz, B.; Zhao, R.\nScore: 3.4, Published: 2023-07-21 DOI: 10.1101/2023.07.19.549794\nPulmonary fibrosis, as seen in idiopathic pulmonary fibrosis (IPF) and COVID-induced pulmonary fibrosis, is an often-fatal lung disease. Increased numbers of immune cells such as macrophages were shown to accumulate in the fibrotic lung, but it is unclear how they contribute to the development of fibrosis. To recapitulate the macrophage mechanical activation in the fibrotic lung tissue microenvironment, we developed a fibrotic microtissue model with cocultured human macrophages and fibroblasts. We show that profibrotic macrophages seeded on topographically controlled stromal tissue constructs become mechanically activated. The resulting co-alignment of macrophages, collagen fibers and fibroblasts promote widespread fibrogenesis in micro- engineered lung tissues. Anti-fibrosis treatment using pirfenidone disrupts the polarization and mechanical activation of profibrotic macrophages, leading to fibrosis inhibition. Pirfenidone inhibits the mechanical activation of macrophages by suppressing integrin M{beta}2 (CD11b/CD18) and Rho-associated kinase 2, which is a previously unknown mechanism of action of the drug. Together, these results demonstrate a potential pulmonary fibrogenesis mechanism at the tissue level contributed by mechanically activated macrophages. We propose the coculture, force-sensing microtissue model as a powerful tool to study the complex immune-stromal cell interactions and the mechanism of action of anti-fibrosis drugs.\nReprogramming Yarrowia lipolytica metabolism for efficient synthesis of itaconic acid from flask to semi-pilot scale\nAuthors: Fu, J.; Zaghen, S.; Lu, H.; Konzock, O.; Poorinmohammad, N.; Kornberg, A.; Koseto, D.; Wentzel, A.; Bartolomeo, F. D.; Kerkhoven, E. J.\nScore: 2.2, Published: 2023-07-18 DOI: 10.1101/2023.07.17.549194\nItaconic acid is an emerging platform chemical with extensive applications. It is currently produced by Aspergillus terreus through biological fermentation. However, A. terreus is a fungal pathogen and needs additional morphology controls, and therefore the production remains problematic. Here, we reprogrammed the GRAS yeast Yarrowia lipolytica metabolism for competitive itaconic acid production. After redirecting the flux of lipid accumulation as carbon sink, we evaluated itaconic acid production both inside and outside the mitochondria, and fine modulated its synthetic pathway. We then mimicked the regulation of nitrogen limitation in nitrogen replete conditions through down regulation of IDH by weak promoter changing, RNAi, or CRISPRi. Ultimately, we optimized fermentation parameters for fed-batch cultivations, and produced itaconic acid with titres of 130.1 g/L in 1L bioreactors and 94.8 g/L in a 50L bioreactor on semi-pilot scale. Our finds provide effective approaches for harnessing GRAS microorganism for competitive industrial itaconic acid production.\n",
  "wordCount" : "2369",
  "inLanguage": "en",
  "datePublished": "2023-07-23T10:36:34Z",
  "dateModified": "2023-07-23T10:36:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on July 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549397">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549397" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549397">
        <p class="paperTitle">Simultaneous multi-site editing of individual genomes using retron arrays</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549397" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549397" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gonzalez-Delgado, A.; Lopez, S. C.; Rojas-Montero, M.; Fishman, C. B.; Shipman, S. L.</p>
        <p class="info">Score: 27.2, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549397' target='https://doi.org/10.1101/2023.07.17.549397'> 10.1101/2023.07.17.549397</a></p>
        <p class="abstract">Our understanding of genomics is limited by the scale of our genomic technologies. While libraries of genomic manipulations scaffolded on CRISPR gRNAs have been transformative, these existing approaches are typically multiplexed across genomes. Yet much of the complexity of real genomes is encoded within a genome across sites. Unfortunately, building cells with multiple, non-adjacent precise mutations remains a laborious cycle of editing, isolating an edited cell, and editing again. Here, we describe a technology for precisely modifying multiple sites on a single genome simultaneously. This technology - termed a multitron - is built from a heavily modified retron, in which multiple donor-encoding msds are produced from a single transcript. The multitron architecture is compatible with both recombineering in prokaryotic cells and CRISPR editing in eukaryotic cells. We demonstrate applications for this approach in molecular recording, genetic element minimization, and metabolic engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549947">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549947" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549947">
        <p class="paperTitle">Eliminating Malaria Vectors with Precision Guided Sterile Males</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549947" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549947" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smidler, A. L.; Apte, R. A.; Pai, J. J.; Chow, M. L.; Chen, S.; Mondal, A.; Sanchez C., H. M.; Antoshechkin, I.; Marshall, J. M.; Akbari, O. S.</p>
        <p class="info">Score: 10.2, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549947' target='https://doi.org/10.1101/2023.07.20.549947'> 10.1101/2023.07.20.549947</a></p>
        <p class="abstract">Controlling the principal African malaria vector, the mosquito Anopheles gambiae, is considered essential to curtail malaria transmission. However existing vector control technologies rely on insecticides, which are becoming increasingly ineffective. Sterile insect technique (SIT) is a powerful suppression approach that has successfully eradicated a number of insect pests, yet the A. gambiae toolkit lacks the requisite technologies for its implementation. SIT relies on iterative mass-releases of non-biting, non-driving, sterile males which seek out and mate with monandrous wild females. Once mated, females are permanently sterilized due to mating-induced refractoriness, which results in population suppression of the subsequent generation. However, sterilization by traditional methods renders males unfit, making the creation of precise genetic sterilization methods imperative. Here we develop precision guided Sterile Insect Technique (pgSIT) in the mosquito A. gambiae for inducible, programmed male-sterilization and female-elimination for wide scale use in SIT campaigns. Using a binary CRISPR strategy, we cross separate engineered Cas9 and gRNA strains to disrupt male-fertility and female-essential genes, yielding &gt;99.5% male-sterility and &gt;99.9% female-lethality in hybrid progeny. We demonstrate that these genetically sterilized males have good longevity, are able to induce population suppression in cage trials, and are predicted to eliminate wild A. gambiae populations using mathematical models, making them ideal candidates for release. This work provides a valuable addition to the malaria genetic biocontrol toolkit, for the first time enabling scalable SIT-like confinable suppression in the species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.549228">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.549228" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.549228">
        <p class="paperTitle">In vivo fate of systemically administered encapsulin protein nanocages and implications for their use in targeted drug delivery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.549228" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.549228" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rennie, C.; Sives, C.; Boyton, I.; Diaz, D.; Gorrie, C. A.; Vittorio, O.; Collins-Praino, L.; Care, A.</p>
        <p class="info">Score: 7.5, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.549228' target='https://doi.org/10.1101/2023.07.16.549228'> 10.1101/2023.07.16.549228</a></p>
        <p class="abstract">Encapsulins, self-assembling protein nanocages derived from prokaryotes, are promising nanoparticle-based drug delivery systems (NDDS). However, the in vivo behavior and fate of encapsulins are poorly understood. In this pre-clinical study, we probe the interactions between the model encapsulin from Thermotoga maritima (TmEnc) and key biological barriers encountered by NDDS. Here, a purified TmEnc formulation that exhibited colloidal stability, storability, and blood compatibility was intravenously injected into BALB/c mice. TmEnc had an excellent nanosafety profile, with no abnormal weight loss or gross pathology observed, and only temporary alterations in toxicity biomarkers detected. Notably, TmEnc demonstrated immunogenic properties, inducing the generation of nanocage-specific IgM and IgG antibodies, but without any prolonged pro-inflammatory effects. An absence of antibody cross-reactivity also suggested immune-orthogonality among encapsulins systems. Moreover, TmEnc formed a serum-derived protein corona on its surface which changed dynamically and appeared to play a role in immune recognition. TmEncs biodistribution profile further revealed its sequestration from the blood circulation by the liver and then biodegraded within Kupffer cells, thus indicating clearance via the mononuclear phagocyte system. Collectively, these findings provide critical insights into how encapsulins behave in vivo, thereby informing their future design, modification, and application in targeted drug delivery.

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=187 SRC=&#34;FIGDIR/small/549228v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (33K):
org.highwire.dtl.DTLVardef@af6321org.highwire.dtl.DTLVardef@951830org.highwire.dtl.DTLVardef@67090corg.highwire.dtl.DTLVardef@832967_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549481">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549481" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549481">
        <p class="paperTitle">Towards Modular Engineering of Cell Signalling: Topographically-Textured Microparticles Induce Osteogenesis via Activation of Canonical Hedgehog Signalling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549481" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549481" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ghuloum, F. I.; Stevens, L. A.; Johnson, C. A.; Riobo-Del Galdo, N. A.; Amer, M. H.</p>
        <p class="info">Score: 5.7, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549481' target='https://doi.org/10.1101/2023.07.19.549481'> 10.1101/2023.07.19.549481</a></p>
        <p class="abstract">Polymer microparticles possess great potential as functional building blocks for advanced bottom-up engineering of complex tissues. Tailoring the three-dimensional architectural features of culture substrates has been shown to induce osteogenesis in mesenchymal stem cells in vitro, but the molecular mechanisms underpinning this remain unclear. This study proposes a mechanism linking the activation of Hedgehog signalling to the osteoinductive effect of surface-engineered, topographically-textured polymeric microparticles. In this study, mesenchymal progenitor C3H10T1/2 cells were cultured on smooth and dimpled poly(D,L-lactide) microparticles to assess differences in viability, cellular morphology, proliferation, and expression of a range of Hedgehog signalling components and osteogenesis-related genes. Dimpled microparticles induced osteogenesis and activated the Hedgehog signalling pathway relative to smooth microparticles and 2D-cultured controls without the addition of exogenous biochemical factors. We observed upregulation of the osteogenesis markers Runt-related transcription factor2 (Runx2) and bone gamma-carboxyglutamate protein 2 (Bglap2), as well as the Hedgehog signalling components, glioma associated oncogene homolog 1 (Gli1), Patched1 (Ptch1), and Smoothened (Smo). Treatment with the Smo antagonist KAAD-cyclopamine confirmed the involvement of Smo in Gli1 target gene activation, with a significant reduction in the expression of Gli1, Runx2 and Bglap2 (p[&amp;le;]0.001) following KAAD-cyclopamine treatment. Overall, our study demonstrates the role of the topographical microenvironment in the modulation of Hedgehog signalling, highlighting the potential for tailoring substrate topographical design to offer cell-instructive 3D microenvironments. Topographically-textured microparticles allow the modulation of Hedgehog signalling in vitro without adding exogenous biochemical agonists, thereby eliminating potential confounding artefacts in high-throughput drug screening applications.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=134 HEIGHT=200 SRC=&#34;FIGDIR/small/549481v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (42K):
org.highwire.dtl.DTLVardef@e01c27org.highwire.dtl.DTLVardef@1587e56org.highwire.dtl.DTLVardef@ce44e5org.highwire.dtl.DTLVardef@d1f01_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549283">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549283" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549283">
        <p class="paperTitle">Microfluidic Quaking-Induced Conversion (Micro-QuIC) for Rapid On-Site Amplification and Detection of Misfolded Proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549283" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549283" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, D. J.; Christenson, P. R.; Rowden, G.; Lindquist, N. C.; Larsen, P. A.; Oh, S.-H.</p>
        <p class="info">Score: 5.5, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549283' target='https://doi.org/10.1101/2023.07.17.549283'> 10.1101/2023.07.17.549283</a></p>
        <p class="abstract">Protein misfolding diseases, such as prion diseases, Alzheimers, and Parkinsons, share a common molecular mechanism involving the misfolding and aggregation of specific proteins. There is an urgent need for point-of-care (POC) diagnostic technologies that can accurately detect these misfolded proteins, facilitating early diagnosis and intervention. Here, we introduce the Microfluidic Quaking Induced Conversion (Micro-QuIC), a novel acoustofluidic platform for the rapid and sensitive detection of protein misfolding diseases. We demonstrate the utility of our technology using chronic wasting disease (CWD) as a model system, as samples from wild white-tailed deer are readily accessible, and CWD shares similarities with human protein misfolding diseases. Acoustofluidic mixing enables homogeneous mixing of reagents in a high-Reynolds-number regime, significantly accelerating the turnaround time for CWD diagnosis. Our Micro-QuIC assay amplifies prions by an order of magnitude faster than the current gold standard, real-time quaking-induced conversion (RT-QuIC). Furthermore, we integrated Micro-QuIC with a gold nanoparticle-based, naked-eye detection method, which enables visual discrimination between CWD positive and negative samples without the need for a bulky fluorescence detection module. This integration creates a rapid, POC testing platform capable of detecting misfolded proteins associated with a variety of protein misfolding diseases.

TOC graphic

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=109 SRC=&#34;FIGDIR/small/549283v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (39K):
org.highwire.dtl.DTLVardef@7d9c08org.highwire.dtl.DTLVardef@8ef24borg.highwire.dtl.DTLVardef@c556bcorg.highwire.dtl.DTLVardef@1c16279_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.549156">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.549156" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.549156">
        <p class="paperTitle">A monochromatically excitable green-red dual-fluorophore fusion incorporating a new large Stokes shift fluorescent protein</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.549156" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.549156" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ejike, J. O.; Sadoine, M.; Shen, Y.; Ishikawa, Y.; Sunal, E.; Hänsch, S.; Hamacher, A. B.; Frommer, W. B.; Wudick, M. M.; Campbell, R. E.; Kleist, T. J.</p>
        <p class="info">Score: 3.9, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.549156' target='https://doi.org/10.1101/2023.07.16.549156'> 10.1101/2023.07.16.549156</a></p>
        <p class="abstract">Genetically encoded sensors enable quantitative imaging of analytes in live cells. State-of-the-art sensors are commonly constructed by combining ligand-binding domains with one or more sensitized fluorescent protein (FP) domains. Sensors based on a single FP are susceptible to artifacts caused by differing expression levels or sensor distribution in vivo. Hence, our lab developed dual-FP Matryoshka technology introduced by a single cassette that contains a stable large Stokes shift (LSS) reference FP nested within a reporter FP (cpEGFP), allowing simple construction of intensiometric sensors with the capacity for ratiometric quantification. The first-generation Green-Orange (GO) Matryoshka cassette established proof of concept but required custom optical setups to maximize achievable dynamic range. Here, we present a genetically encoded calcium sensor that employs optimized second-generation Green-Apple (GA) Matryoshka technology that incorporates a newly designed red LSSmApple fluorophore. LSSmApple provides improved excitation spectrum overlap with cpEGFP, allowing for monochromatic co-excitation with blue light. The exceptionally large Stokes shift of LSSmApple results in improved emission spectrum separation from cpEGFP, which minimizes fluorophore bleed-through and facilitates imaging using standard dichroics and red fluorescent protein (RFP) emission filters. We developed an image analysis pipeline for yeast (Saccharomyces cerevisiae) timelapse imaging that utilizes LSSmApple to segment and track cells for high-throughput quantitative analysis. In summary, we engineered a new fluorescent protein, constructed a genetically encoded calcium indicator (GA-MatryoshCaMP6s), and performed calcium imaging in yeast as a demonstration.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.549225">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.549225" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.549225">
        <p class="paperTitle">Biosensor-Assisted Laboratory Evolution of Malonyl-CoA production in Saccharomyces cerevisiae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.549225" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.549225" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ferreira, R.; Hedin, K. A.; Nielsen, J.; David, F.</p>
        <p class="info">Score: 3.9, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.549225' target='https://doi.org/10.1101/2023.07.16.549225'> 10.1101/2023.07.16.549225</a></p>
        <p class="abstract">The production of bio-based chemicals and fuels through microbial engineering offers a promising and sustainable alternative to petroleum-based fuels and chemicals, with the potential for scalability. However, engineering microbes and continuously evolving them to enhance the production of industrially relevant products is a complex and challenging task, requiring precise selection of genetic traits to achieve desired outcomes. In this study, we report the development of a novel counter-selectable growth-sensitive malonyl-CoA platform strain by coupling the malonyl-CoA repressor FapR from Bacillus subtilis to essential gene promoters involved in glucose growth and the plasma membrane arginine permease. This platform strain was then coupled with a CRISPR-dCas9 guide-RNA (gRNA) library, which after multiple rounds of dilutions and library sequencing, resulted in the enrichment for gRNAs that increased fluxes towards malonyl-CoA. The enriched gRNAs were validated for their effects on growth enhancement, gene regulation, and the production of an industrially relevant malonyl-CoA product, namely 3-hydroxypropionic acid. This study highlights an innovative approach to microbial engineering and opens up avenues for further exploration in the field of laboratory continuous evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548856">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548856" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548856">
        <p class="paperTitle">Solid tumor-on-chip model for efficacy and safety assessment of CAR-T cell therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548856" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548856" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Maulana, T. I.; Teufel, C.; Cipriano, M.; Lazarevski, L.; van den Hil, F. E.; Orlova, V.; Koch, A.; Alb, M.; Hudecek, M.; Loskill, P.</p>
        <p class="info">Score: 4.6, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548856' target='https://doi.org/10.1101/2023.07.13.548856'> 10.1101/2023.07.13.548856</a></p>
        <p class="abstract">The non-clinical assessment of CAR-T cells demands innovative models that are capable of predicting safety and efficacy in the clinical setting. Here, we present a novel solid tumor-on-chip model that allows CAR-T cell perfusion and integrates the vasculature and tumor lesions to recapitulate key events of CAR-T cell performance including extravasation, tumor infiltration and cytokine release. We assessed CAR-T cells targeting the ROR1 antigen against tumor aggregates that were derived from a breast cancer cell line and primary breast cancer organoids. The data show the temporal kinetic of ROR1 CAR-T cell migration and expansion, lytic activity and cytokine production over the course of 8 days, and reveal a correlation between anti-tumor efficacy and ROR1 antigen density on tumor cells. CAR-modified T cells extravasated faster, infiltrated tumor lesions stronger, persisted longer and in higher numbers than non-CAR modified T cells. Intriguingly, we detected cytokine release levels and kinetics typically observed in patients who developed cytokine release syndrome, and administered dasatinib as a pharmacologic OFF switch to control this inflammatory response. The data illustrate the ability of this tumor-on-chip platform to assess parameters associated withherapeutic outcome and the potential to aid in patient stratification and monitoring of CAR-T cell therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549794">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549794" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549794">
        <p class="paperTitle">Modeling Mechanical Activation of Macrophages During Pulmonary Fibrogenesis for Targeted Anti-Fibrosis Therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549794" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549794" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, Y.; Ying, L.; Lang, J. K.; Hinz, B.; Zhao, R.</p>
        <p class="info">Score: 3.4, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549794' target='https://doi.org/10.1101/2023.07.19.549794'> 10.1101/2023.07.19.549794</a></p>
        <p class="abstract">Pulmonary fibrosis, as seen in idiopathic pulmonary fibrosis (IPF) and COVID-induced pulmonary fibrosis, is an often-fatal lung disease. Increased numbers of immune cells such as macrophages were shown to accumulate in the fibrotic lung, but it is unclear how they contribute to the development of fibrosis. To recapitulate the macrophage mechanical activation in the fibrotic lung tissue microenvironment, we developed a fibrotic microtissue model with cocultured human macrophages and fibroblasts. We show that profibrotic macrophages seeded on topographically controlled stromal tissue constructs become mechanically activated. The resulting co-alignment of macrophages, collagen fibers and fibroblasts promote widespread fibrogenesis in micro- engineered lung tissues. Anti-fibrosis treatment using pirfenidone disrupts the polarization and mechanical activation of profibrotic macrophages, leading to fibrosis inhibition. Pirfenidone inhibits the mechanical activation of macrophages by suppressing integrin M{beta}2 (CD11b/CD18) and Rho-associated kinase 2, which is a previously unknown mechanism of action of the drug. Together, these results demonstrate a potential pulmonary fibrogenesis mechanism at the tissue level contributed by mechanically activated macrophages. We propose the coculture, force-sensing microtissue model as a powerful tool to study the complex immune-stromal cell interactions and the mechanism of action of anti-fibrosis drugs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549194">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549194" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549194">
        <p class="paperTitle">Reprogramming Yarrowia lipolytica metabolism for efficient synthesis of itaconic acid from flask to semi-pilot scale</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549194" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549194" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fu, J.; Zaghen, S.; Lu, H.; Konzock, O.; Poorinmohammad, N.; Kornberg, A.; Koseto, D.; Wentzel, A.; Bartolomeo, F. D.; Kerkhoven, E. J.</p>
        <p class="info">Score: 2.2, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549194' target='https://doi.org/10.1101/2023.07.17.549194'> 10.1101/2023.07.17.549194</a></p>
        <p class="abstract">Itaconic acid is an emerging platform chemical with extensive applications. It is currently produced by Aspergillus terreus through biological fermentation. However, A. terreus is a fungal pathogen and needs additional morphology controls, and therefore the production remains problematic. Here, we reprogrammed the GRAS yeast Yarrowia lipolytica metabolism for competitive itaconic acid production. After redirecting the flux of lipid accumulation as carbon sink, we evaluated itaconic acid production both inside and outside the mitochondria, and fine modulated its synthetic pathway. We then mimicked the regulation of nitrogen limitation in nitrogen replete conditions through down regulation of IDH by weak promoter changing, RNAi, or CRISPRi. Ultimately, we optimized fermentation parameters for fed-batch cultivations, and produced itaconic acid with titres of 130.1 g/L in 1L bioreactors and 94.8 g/L in a 50L bioreactor on semi-pilot scale. Our finds provide effective approaches for harnessing GRAS microorganism for competitive industrial itaconic acid production.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
