<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Saponin Nanoparticle Adjuvants Incorporating Toll-Like Receptor Agonists Improve Vaccine Immunomodulation
Authors: Ou, B. S.; Baillet, J.; Filsinger Interrante, M. V.; Adamska, J. Z.; Zhou, X.; Saouaf, O.; Yan, J.; Klich, J.; Jons, C. K.; Meany, E.; Valdez, A. S.; Carter, L.; Pulendran, B.; King, N. P.; Appel, E.
Score: 141.8, Published: 2023-12-31 DOI: 10.1101/2023.07.16.549249
Over the past few decades, the development of potent and safe immune-activating adjuvant technologies has become the heart of intensive research in the constant fight against highly mutative and immune evasive viruses such as influenza, SARS-CoV-2, and HIV.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Saponin Nanoparticle Adjuvants Incorporating Toll-Like Receptor Agonists Improve Vaccine Immunomodulation
Authors: Ou, B. S.; Baillet, J.; Filsinger Interrante, M. V.; Adamska, J. Z.; Zhou, X.; Saouaf, O.; Yan, J.; Klich, J.; Jons, C. K.; Meany, E.; Valdez, A. S.; Carter, L.; Pulendran, B.; King, N. P.; Appel, E.
Score: 141.8, Published: 2023-12-31 DOI: 10.1101/2023.07.16.549249
Over the past few decades, the development of potent and safe immune-activating adjuvant technologies has become the heart of intensive research in the constant fight against highly mutative and immune evasive viruses such as influenza, SARS-CoV-2, and HIV." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-14T10:38:40+00:00" />
<meta property="article:modified_time" content="2024-01-14T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Saponin Nanoparticle Adjuvants Incorporating Toll-Like Receptor Agonists Improve Vaccine Immunomodulation
Authors: Ou, B. S.; Baillet, J.; Filsinger Interrante, M. V.; Adamska, J. Z.; Zhou, X.; Saouaf, O.; Yan, J.; Klich, J.; Jons, C. K.; Meany, E.; Valdez, A. S.; Carter, L.; Pulendran, B.; King, N. P.; Appel, E.
Score: 141.8, Published: 2023-12-31 DOI: 10.1101/2023.07.16.549249
Over the past few decades, the development of potent and safe immune-activating adjuvant technologies has become the heart of intensive research in the constant fight against highly mutative and immune evasive viruses such as influenza, SARS-CoV-2, and HIV."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Saponin Nanoparticle Adjuvants Incorporating Toll-Like Receptor Agonists Improve Vaccine Immunomodulation\nAuthors: Ou, B. S.; Baillet, J.; Filsinger Interrante, M. V.; Adamska, J. Z.; Zhou, X.; Saouaf, O.; Yan, J.; Klich, J.; Jons, C. K.; Meany, E.; Valdez, A. S.; Carter, L.; Pulendran, B.; King, N. P.; Appel, E.\nScore: 141.8, Published: 2023-12-31 DOI: 10.1101/2023.07.16.549249\nOver the past few decades, the development of potent and safe immune-activating adjuvant technologies has become the heart of intensive research in the constant fight against highly mutative and immune evasive viruses such as influenza, SARS-CoV-2, and HIV.",
  "keywords": [
    
  ],
  "articleBody": " Saponin Nanoparticle Adjuvants Incorporating Toll-Like Receptor Agonists Improve Vaccine Immunomodulation\nAuthors: Ou, B. S.; Baillet, J.; Filsinger Interrante, M. V.; Adamska, J. Z.; Zhou, X.; Saouaf, O.; Yan, J.; Klich, J.; Jons, C. K.; Meany, E.; Valdez, A. S.; Carter, L.; Pulendran, B.; King, N. P.; Appel, E.\nScore: 141.8, Published: 2023-12-31 DOI: 10.1101/2023.07.16.549249\nOver the past few decades, the development of potent and safe immune-activating adjuvant technologies has become the heart of intensive research in the constant fight against highly mutative and immune evasive viruses such as influenza, SARS-CoV-2, and HIV. Herein, we developed a highly modular saponin-based nanoparticle platform incorporating toll-like receptor agonists (TLRas) including TLR1/2a, TLR4a, TLR7/8a adjuvants and their mixtures. These various TLRa-SNP adjuvant constructs induce unique acute cytokine and immune-signaling profiles, leading to specific Th-responses that could be of interest depending on the target disease for prevention. In a murine vaccine study, the adjuvants greatly improved the potency, durability, breadth, and neutralization of both COVID-19 and HIV vaccine candidates, suggesting the potential broad application of these adjuvant constructs to a range of different antigens. Overall, this work demonstrates a modular TLRa-SNP adjuvant platform which could improve the design of vaccines for and dramatically impact modern vaccine development.\nUltrasound-Driven Programmable Artificial Muscles\nAuthors: Shi, Z.; Zhang, Z.; Ahmed, D.\nScore: 8.6, Published: 2024-01-09 DOI: 10.1101/2024.01.08.574699\nDespite their promise for technological and medical innovations, the advancement of artificial muscles faces challenges in dynamic programmability, wireless control, scalability, and responsiveness. Ultrasound offers a promising yet underutilized solution, providing effective and robust actuation. We introduce an innovative design paradigm for soft artificial muscles, utilizing tens of thousands of resonant microbubbles for targeted activation, enabling a wide range of programmable motions. We developed a comprehensive theoretical model and conducted various applications: from actuating soft grippers and powering centimeter-scale biomimetic stingraybots for propulsion to demonstrating the multimodal shape transformation of a functional surface and equipping static objects with transformative robotic skins for mobility. Our approach is marked by its compactness, high force intensity, rapid responsiveness, and lightweightness. The customizable, scalable design of our artificial muscles offers both immediate and long-term impact on robotics, wearable technology, haptics, prosthetic development, and the development of avant-garde biomedical devices like soft surgical tools. One-Sentence SummaryWe introduced a first-of-its-kind, programmable artificial muscle using ultrasound-activated microbubbles, providing a compact, responsive, and scalable solution.\nSpatial genomics of AAVs reveals mechanism of transcriptional crosstalk that enables targeted delivery of large genetic cargo\nAuthors: Coughlin, G. M.; Borsos, M.; Appling, N.; Barcelona, B. H.; Mayfield, A. M. H.; Mackey, E. D.; Eser, R. A.; Chen, X.; Kumar, S. R.; Gradinaru, V.\nScore: 8.2, Published: 2023-12-24 DOI: 10.1101/2023.12.23.573214\nIntegrating cell type-specific regulatory elements (e.g. enhancers) with recombinant adeno-associated viruses (AAVs) can provide broad and efficient genetic access to specific cell types. However, the packaging capacity of AAVs restricts the size of both the enhancers and the cargo that can be delivered. Transcriptional crosstalk offers a novel paradigm for cell type-specific expression of large cargo, by separating distally-acting regulatory elements into a second AAV genome. Here, we identify and profile transcriptional crosstalk in AAV genomes carrying 11 different enhancers active in mouse brain. To understand transcriptional crosstalk, we develop spatial genomics methods to identify and localize AAV genomes and their concatemeric forms in cultured cells and in tissue. Using these methods, we construct detailed views of the dynamics of AAV transduction and demonstrate that transcriptional crosstalk is dependent upon concatemer formation. Finally, we leverage transcriptional crosstalk to drive expression of a large Cas9 cargo in a cell type-specific manner with systemically-administered engineered AAVs and demonstrate AAV-delivered, minimally-invasive, cell type-specific gene editing in wildtype animals that recapitulates known disease phenotypes. HighlightsO_LITranscriptional crosstalk between enhancers and promoters delivered in trans by AAVs is a generalized phenomenon. C_LIO_LISpatial genomics techniques, AAV-Zombie and SpECTr, reveal that AAV genome concatemerization facilitates transcriptional crosstalk. C_LIO_LITranscriptional crosstalk can be leveraged for minimally-invasive, targeted AAV delivery of large cargo, including machinery for CRISPR-based gene editing and manipulation. C_LIO_LITranscriptional crosstalk enables cell-type specific gene disruption in wildtype animals, recapitulating behavioural phenotypes of genetic knockouts. C_LI\nDevelopment of a robotic cluster for automated and scalable cell therapy manufacturing\nAuthors: Melocchi, A.; Schmittlein, B.; Jones, A. L.; Ainane, Y.; Rizvi, A.; Chan, D.; Dickey, E.; Pool, K.; Harsono, K.; Szymkiewicz, D.; Scarfogliero, U.; Bhatia, V.; Sivanantham, A.; Kreciglowa, N.; Hunter, A.; Gomez, M.; Tanner, A.; Uboldi, M.; Batish, A.; Balcerek, J.; Kutova-Stoilova, M.; Paruthiyil, S.; Acevedo, L. A.; Stadnitskiy, R.; Carmichael, S.; Aulbach, H.; Hewitt, M.; De Mollerat Du Jeu, X.; di Robilant, B.; Parietti, F.; Esensten, J. H.\nScore: 11.5, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572854\nThe production of commercial autologous cell therapies such as chimeric antigen receptor T cells requires complex manual manufacturing processes. Skilled labor costs and challenges in manufacturing scale-out have contributed to high prices for these products. Here, we present a robotic system that uses industry-standard cell therapy manufacturing equipment to automate the steps involved in cell therapy manufacturing. The robotic cluster consists of a robotic arm and customized modules, allowing the robot to manipulate a variety of standard cell therapy instruments and materials such as incubators, bioreactors, and reagent bags. This system enables existing manual manufacturing processes to be rapidly adapted to robotic manufacturing, without having to adopt a completely new technology platform. Proof-of-concept for the robotic clusters expansion module was demonstrated by activating and expanding human CD8+ T cells. The robotic cultures showed comparable cell yields, viability, and identity to those manually performed. Such modular robotic solutions may support scale-up and scale-out of cell therapies that are developed using classical manual methods in academic laboratories and biotechnology companies.\nDemonstration of chemotherapeutic mediated lymphatic changes in meningeal lymphatics in vitro, ex vivo, and in vivo\nAuthors: Roberts, L. M.; Hammel, J. H.; Azar, F.; Feng, T.-Y.; Cunningham, J. J.; Rutkowski, M. R.; Munson, J.\nScore: 5.5, Published: 2024-01-08 DOI: 10.1101/2024.01.06.574460\nSystemic chemotherapeutics target cancer cells but are also known to impact other cells away from the tumor. Questions remain whether systemic chemotherapy crosses the blood-brain barrier and causes inflammation in the periphery that impacts the central nervous system (CNS) downstream. The meningeal lymphatics are a critical component that drain cerebrospinal fluid from the CNS to the cervical lymph nodes for immunosurveillence. To develop new tools for understanding chemotherapy-mediated effects on the meningeal lymphatics, we present two novel models that examine cellular and tissue level changes. Our in vitro tissue engineered model of a meningeal lymphatic vessel lumen, using a simple tissue culture insert system with both lymphatic endothelial and meningeal cells, examines cell disruption. Our ex vivo model culturing mouse meningeal layers probes structural changes and remodeling, correlating to an explant tissue level. To gain a holistic understanding, we compare our in vitro and ex vivo models to in vivo studies for validation and a three-tier methodology for examining the chemotherapeutic response of the meningeal lymphatics. We have demonstrated that the meningeal lymphatics can be disrupted by systemic chemotherapy but show differential responses to platinum and taxane chemotherapies, emphasizing the need for further study of off-target impacts in the CNS.\nReinforcement Learning for Control of Human Locomotion in Simulation\nAuthors: Dashkovets, A.; Laschowski, B.\nScore: 6.4, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572447\nControl of robotic leg prostheses and exoskeletons is an open challenge. Computer modeling and simulation can be used to study the dynamics and control of human walking and extract principles that can be programmed into robotic legs to behave similar to biological legs. In this study, we present the development of an efficient two-layer Q-learning algorithm, with k-d trees, that operates over continuous action spaces and a reward model that estimates the degree of muscle activation similarity between the agent and human state-to-action pairs and state-to-action sequences. We used a human musculoskeletal model acting in a high-dimensional, physics-based simulation environment to train and evaluate our algorithm to simulate biomimetic walking. We used imitation learning and artificial bio-mechanics data to accelerate training via expert demonstrations and used experimental human data to compare and validate our predictive simulations, achieving 79% accuracy. Also, when compared to the previous state-of-the-art that used deep deterministic policy gradient, our algorithm was significantly more efficient with lower computational and memory storage requirements (i.e., requiring 7 times less RAM and 87 times less CPU compute), which can benefit real-time embedded computing. Overall, our new two-layer Q-learning algorithm using sequential data for continuous imitation of human locomotion serves as a first step towards the development of bioinspired controllers for robotic prosthetic legs and exoskeletons. Future work will focus on improving the prediction accuracy compared to experimental data and expanding our simulations to other locomotor activities.\nSpatial transcriptomic analysis drives PET imaging of tight junction protein expression in pancreatic cancer theranostics\nAuthors: Wang, J.; Seo, J. W.; Kare, A. J.; Schneider, M.; Tumbale, S. K.; Wu, B.; Raie, M. N.; Pandrala, M.; Iagaru, A.; Brunsing, R. L.; Charville, G. W.; Park, W. G.; Ferrara, K. W.\nScore: 3.5, Published: 2024-01-08 DOI: 10.1101/2024.01.07.574209\nWe apply spatial transcriptomics and proteomics to select pancreatic cancer surface receptor targets for molecular imaging and theranostics using an approach that can be applied to many cancers. Selected cancer surfaceome epithelial markers were spatially correlated and provided specific cancer localization, whereas the spatial correlation between cancer markers and immune- cell or fibroblast markers was low. While molecular imaging of cancer-associated fibroblasts and integrins has been proposed for pancreatic cancer, our data point to the tight junction protein claudin-4 as a theranostic target. Claudin-4 expression increased [~]16 fold in cancer as compared with normal pancreas, and the tight junction localization conferred low background for imaging in normal tissue. We developed a peptide-based molecular imaging agent targeted to claudin-4 with accumulation to [~]25% injected activity per cc (IA/cc) in metastases and [~]18% IA/cc in tumors. Our work motivates a new approach for data-driven selection of molecular targets.\nHigh-resolution low-cost LCD 3D printing of microfluidics\nAuthors: Shafique, H.; Karamzadeh, V.; Kim, G.; Morocz, Y.; Sohrabi-Kashani, A.; Shen, M. L.; Juncker, D.\nScore: 7.7, Published: 2024-01-02 DOI: 10.1101/2023.12.31.573772\nThe fabrication of microfluidic devices has progressed from cleanroom manufacturing to replica molding in polymers, and more recently to direct manufacturing by subtractive (e.g., laser machining) and additive (e.g., 3D printing) techniques, notably digital light processing (DLP) photopolymerization. However, many methods require technical expertise and while DLP 3D printers remain expensive at a cost [~]15-30K USD with [~]8M pixels that are 25-40 {micro}m in size. Here, we introduce (i) the use of low-cost ([~]150-600 USD) liquid crystal display (LCD) photopolymerization 3D printing with [~]8M-58M pixels that are 18-35 {micro}m in size for direct microfluidic device fabrication and (ii) a poly(ethylene glycol) diacrylate-based ink developed for LCD 3D printing (PLInk). We optimized PLInk for high resolution, fast 3D printing and biocompatibility while considering the illumination inhomogeneity and low power density of LCD 3D printers. We made lateral features as small as 75 {micro}m, 22-{micro}m-thick embedded membranes, and circular channels with a 110 {micro}m radius. We 3D printed microfluidic devices previously manufactured by other methods, including an embedded 3D micromixer, a membrane microvalve, and an autonomous capillaric circuit (CC) deployed for interferon-{gamma} detection with excellent performance (limit of detection: 12 pg mL-1, CV: 6.8%), and we demonstrated compatibility with cell culture. Finally, large area manufacturing was illustrated by printing 42 CCs with embedded microchannels in \u003c45 min. LCD 3D printing together with tailored inks pave the way for democratizing access to high-resolution manufacturing of ready-to-use microfluidic devices by anyone, anywhere.\nImaging the structural connectome with hybrid diffusion MRI-microscopy tractography\nAuthors: Zhu, S.; Huszar, I. N.; Cottaar, M.; Daubney, G.; Eichert, N.; Hanayik, T.; Khrapitchev, A. A.; Mars, R. B.; Mollink, J.; Sallet, J.; Scott, C.; Smart, A.; Jbabdi, S.; Miller, K. L.; Howard, A. F.\nScore: 2.5, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574641\nNeuroanatomical tract tracing methods are fundamental in providing \"gold standard\" estimates of brain connectivity. However, tracer methods cannot be performed in humans and even in animals, we can only study projections from typically one or two injection sites per animal sacrificed. Orientation-sensitive microscopy techniques such as PLI provide an alternative where they can visualise detailed fibre orientations at the micron-scale across the whole brain. However, these methods are often most informative on orientations within the 2D imaging plane, with less reliable or missing through-plane information, restricting 3D tract reconstruction. Conversely, dMRI can estimate fibre orientations in 3D but at low resolution, which leads to many false positive and negative estimates of fibre trajectories. To facilitate reconstruction of the microscopy-informed connectome, we develop a data-fusion method that complements 2D microscopy with through-plane information from diffusion MRI to construct 3D hybrid orientations that are both maximally informed by the high-resolution microscopy, have whole-brain coverage and can be input into existing tractography pipelines. Diffusion MRI can be readily acquired prior to microscopy meaning the same method is translatable across species, including in humans. Here we apply our method to an existing open-access macaque dataset and demonstrate (1) whole-brain microscopy-informed tractography (2) the advantages of hybrid tractography in two known tractography challenges, the gyral bias and bottleneck problem (3) how hybrid tractography appears to outperform diffusion-only tractography when compared to tracer data and (4) the generalisability of our hybrid method to different microscopy contrasts, facilitating wider translation.\nPredicting Serotonin Detection with DNA-Carbon Nanotube Sensors Across Multiple Spectral Wavelengths\nAuthors: Kelich, P.; Adams, J.; Jeong, S.; Navarro, N.; Landry, M. P.; Vukovic, L.\nScore: 2.5, Published: 2024-01-07 DOI: 10.1101/2024.01.06.574485\nOwing to the value of DNA-wrapped single-walled carbon nanotube (SWNT) based sensors for chemically-specific imaging in biology, we explore machine learning (ML) predictions DNA-SWNT serotonin sensor responsivity as a function of DNA sequence based on the whole SWNT fluorescence spectra. Our analysis reveals the crucial role of DNA sequence in the binding modes of DNA-SWNTs to serotonin, with a smaller influence of SWNT chirality. Regression ML models trained on existing datasets predict the change in the fluorescence emission in response to serotonin, {Delta}F/F, at over a hundred wavelengths for new DNA-SWNT conjugates, successfully identifying some high- and low-response DNA sequences. Despite successful predictions, we also show that the finite size of the training dataset leads to limitations on prediction accuracy. Nevertheless, incorporating entire spectra into ML models enhances prediction robustness and facilitates the discovery of novel DNA-SWNT sensors. Our approaches show promise for identifying new chemical systems with specific sensing response characteristics, marking a valuable advancement in DNA-based system discovery.\n",
  "wordCount" : "2381",
  "inLanguage": "en",
  "datePublished": "2024-01-14T10:38:40Z",
  "dateModified": "2024-01-14T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.549249">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.549249" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.549249">
        <p class="paperTitle">Saponin Nanoparticle Adjuvants Incorporating Toll-Like Receptor Agonists Improve Vaccine Immunomodulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.549249" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.549249" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ou, B. S.; Baillet, J.; Filsinger Interrante, M. V.; Adamska, J. Z.; Zhou, X.; Saouaf, O.; Yan, J.; Klich, J.; Jons, C. K.; Meany, E.; Valdez, A. S.; Carter, L.; Pulendran, B.; King, N. P.; Appel, E.</p>
        <p class="info">Score: 141.8, Published: 2023-12-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.549249' target='https://doi.org/10.1101/2023.07.16.549249'> 10.1101/2023.07.16.549249</a></p>
        <p class="abstract">Over the past few decades, the development of potent and safe immune-activating adjuvant technologies has become the heart of intensive research in the constant fight against highly mutative and immune evasive viruses such as influenza, SARS-CoV-2, and HIV. Herein, we developed a highly modular saponin-based nanoparticle platform incorporating toll-like receptor agonists (TLRas) including TLR1/2a, TLR4a, TLR7/8a adjuvants and their mixtures. These various TLRa-SNP adjuvant constructs induce unique acute cytokine and immune-signaling profiles, leading to specific Th-responses that could be of interest depending on the target disease for prevention. In a murine vaccine study, the adjuvants greatly improved the potency, durability, breadth, and neutralization of both COVID-19 and HIV vaccine candidates, suggesting the potential broad application of these adjuvant constructs to a range of different antigens. Overall, this work demonstrates a modular TLRa-SNP adjuvant platform which could improve the design of vaccines for and dramatically impact modern vaccine development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574699">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574699" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574699">
        <p class="paperTitle">Ultrasound-Driven Programmable Artificial Muscles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574699" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574699" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shi, Z.; Zhang, Z.; Ahmed, D.</p>
        <p class="info">Score: 8.6, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574699' target='https://doi.org/10.1101/2024.01.08.574699'> 10.1101/2024.01.08.574699</a></p>
        <p class="abstract">Despite their promise for technological and medical innovations, the advancement of artificial muscles faces challenges in dynamic programmability, wireless control, scalability, and responsiveness. Ultrasound offers a promising yet underutilized solution, providing effective and robust actuation. We introduce an innovative design paradigm for soft artificial muscles, utilizing tens of thousands of resonant microbubbles for targeted activation, enabling a wide range of programmable motions. We developed a comprehensive theoretical model and conducted various applications: from actuating soft grippers and powering centimeter-scale biomimetic stingraybots for propulsion to demonstrating the multimodal shape transformation of a functional surface and equipping static objects with transformative robotic skins for mobility. Our approach is marked by its compactness, high force intensity, rapid responsiveness, and lightweightness. The customizable, scalable design of our artificial muscles offers both immediate and long-term impact on robotics, wearable technology, haptics, prosthetic development, and the development of avant-garde biomedical devices like soft surgical tools.

One-Sentence SummaryWe introduced a first-of-its-kind, programmable artificial muscle using ultrasound-activated microbubbles, providing a compact, responsive, and scalable solution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573214">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573214" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573214">
        <p class="paperTitle">Spatial genomics of AAVs reveals mechanism of transcriptional crosstalk that enables targeted delivery of large genetic cargo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573214" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573214" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Coughlin, G. M.; Borsos, M.; Appling, N.; Barcelona, B. H.; Mayfield, A. M. H.; Mackey, E. D.; Eser, R. A.; Chen, X.; Kumar, S. R.; Gradinaru, V.</p>
        <p class="info">Score: 8.2, Published: 2023-12-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573214' target='https://doi.org/10.1101/2023.12.23.573214'> 10.1101/2023.12.23.573214</a></p>
        <p class="abstract">Integrating cell type-specific regulatory elements (e.g. enhancers) with recombinant adeno-associated viruses (AAVs) can provide broad and efficient genetic access to specific cell types. However, the packaging capacity of AAVs restricts the size of both the enhancers and the cargo that can be delivered. Transcriptional crosstalk offers a novel paradigm for cell type-specific expression of large cargo, by separating distally-acting regulatory elements into a second AAV genome. Here, we identify and profile transcriptional crosstalk in AAV genomes carrying 11 different enhancers active in mouse brain. To understand transcriptional crosstalk, we develop spatial genomics methods to identify and localize AAV genomes and their concatemeric forms in cultured cells and in tissue. Using these methods, we construct detailed views of the dynamics of AAV transduction and demonstrate that transcriptional crosstalk is dependent upon concatemer formation. Finally, we leverage transcriptional crosstalk to drive expression of a large Cas9 cargo in a cell type-specific manner with systemically-administered engineered AAVs and demonstrate AAV-delivered, minimally-invasive, cell type-specific gene editing in wildtype animals that recapitulates known disease phenotypes.

HighlightsO_LITranscriptional crosstalk between enhancers and promoters delivered in trans by AAVs is a generalized phenomenon.
C_LIO_LISpatial genomics techniques, AAV-Zombie and SpECTr, reveal that AAV genome concatemerization facilitates transcriptional crosstalk.
C_LIO_LITranscriptional crosstalk can be leveraged for minimally-invasive, targeted AAV delivery of large cargo, including machinery for CRISPR-based gene editing and manipulation.
C_LIO_LITranscriptional crosstalk enables cell-type specific gene disruption in wildtype animals, recapitulating behavioural phenotypes of genetic knockouts.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572854">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572854" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572854">
        <p class="paperTitle">Development of a robotic cluster for automated and scalable cell therapy manufacturing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572854" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572854" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Melocchi, A.; Schmittlein, B.; Jones, A. L.; Ainane, Y.; Rizvi, A.; Chan, D.; Dickey, E.; Pool, K.; Harsono, K.; Szymkiewicz, D.; Scarfogliero, U.; Bhatia, V.; Sivanantham, A.; Kreciglowa, N.; Hunter, A.; Gomez, M.; Tanner, A.; Uboldi, M.; Batish, A.; Balcerek, J.; Kutova-Stoilova, M.; Paruthiyil, S.; Acevedo, L. A.; Stadnitskiy, R.; Carmichael, S.; Aulbach, H.; Hewitt, M.; De Mollerat Du Jeu, X.; di Robilant, B.; Parietti, F.; Esensten, J. H.</p>
        <p class="info">Score: 11.5, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572854' target='https://doi.org/10.1101/2023.12.21.572854'> 10.1101/2023.12.21.572854</a></p>
        <p class="abstract">The production of commercial autologous cell therapies such as chimeric antigen receptor T cells requires complex manual manufacturing processes. Skilled labor costs and challenges in manufacturing scale-out have contributed to high prices for these products. Here, we present a robotic system that uses industry-standard cell therapy manufacturing equipment to automate the steps involved in cell therapy manufacturing. The robotic cluster consists of a robotic arm and customized modules, allowing the robot to manipulate a variety of standard cell therapy instruments and materials such as incubators, bioreactors, and reagent bags. This system enables existing manual manufacturing processes to be rapidly adapted to robotic manufacturing, without having to adopt a completely new technology platform. Proof-of-concept for the robotic clusters expansion module was demonstrated by activating and expanding human CD8&#43; T cells. The robotic cultures showed comparable cell yields, viability, and identity to those manually performed. Such modular robotic solutions may support scale-up and scale-out of cell therapies that are developed using classical manual methods in academic laboratories and biotechnology companies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.06.574460">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.06.574460" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.06.574460">
        <p class="paperTitle">Demonstration of chemotherapeutic mediated lymphatic changes in meningeal lymphatics in vitro, ex vivo, and in vivo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.06.574460" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.06.574460" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roberts, L. M.; Hammel, J. H.; Azar, F.; Feng, T.-Y.; Cunningham, J. J.; Rutkowski, M. R.; Munson, J.</p>
        <p class="info">Score: 5.5, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.06.574460' target='https://doi.org/10.1101/2024.01.06.574460'> 10.1101/2024.01.06.574460</a></p>
        <p class="abstract">Systemic chemotherapeutics target cancer cells but are also known to impact other cells away from the tumor. Questions remain whether systemic chemotherapy crosses the blood-brain barrier and causes inflammation in the periphery that impacts the central nervous system (CNS) downstream. The meningeal lymphatics are a critical component that drain cerebrospinal fluid from the CNS to the cervical lymph nodes for immunosurveillence. To develop new tools for understanding chemotherapy-mediated effects on the meningeal lymphatics, we present two novel models that examine cellular and tissue level changes. Our in vitro tissue engineered model of a meningeal lymphatic vessel lumen, using a simple tissue culture insert system with both lymphatic endothelial and meningeal cells, examines cell disruption. Our ex vivo model culturing mouse meningeal layers probes structural changes and remodeling, correlating to an explant tissue level. To gain a holistic understanding, we compare our in vitro and ex vivo models to in vivo studies for validation and a three-tier methodology for examining the chemotherapeutic response of the meningeal lymphatics. We have demonstrated that the meningeal lymphatics can be disrupted by systemic chemotherapy but show differential responses to platinum and taxane chemotherapies, emphasizing the need for further study of off-target impacts in the CNS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572447">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572447" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572447">
        <p class="paperTitle">Reinforcement Learning for Control of Human Locomotion in Simulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572447" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572447" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dashkovets, A.; Laschowski, B.</p>
        <p class="info">Score: 6.4, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572447' target='https://doi.org/10.1101/2023.12.19.572447'> 10.1101/2023.12.19.572447</a></p>
        <p class="abstract">Control of robotic leg prostheses and exoskeletons is an open challenge. Computer modeling and simulation can be used to study the dynamics and control of human walking and extract principles that can be programmed into robotic legs to behave similar to biological legs. In this study, we present the development of an efficient two-layer Q-learning algorithm, with k-d trees, that operates over continuous action spaces and a reward model that estimates the degree of muscle activation similarity between the agent and human state-to-action pairs and state-to-action sequences. We used a human musculoskeletal model acting in a high-dimensional, physics-based simulation environment to train and evaluate our algorithm to simulate biomimetic walking. We used imitation learning and artificial bio-mechanics data to accelerate training via expert demonstrations and used experimental human data to compare and validate our predictive simulations, achieving 79% accuracy. Also, when compared to the previous state-of-the-art that used deep deterministic policy gradient, our algorithm was significantly more efficient with lower computational and memory storage requirements (i.e., requiring 7 times less RAM and 87 times less CPU compute), which can benefit real-time embedded computing. Overall, our new two-layer Q-learning algorithm using sequential data for continuous imitation of human locomotion serves as a first step towards the development of bioinspired controllers for robotic prosthetic legs and exoskeletons. Future work will focus on improving the prediction accuracy compared to experimental data and expanding our simulations to other locomotor activities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.07.574209">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.07.574209" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.07.574209">
        <p class="paperTitle">Spatial transcriptomic analysis drives PET imaging of tight junction protein expression in pancreatic cancer theranostics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.07.574209" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.07.574209" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, J.; Seo, J. W.; Kare, A. J.; Schneider, M.; Tumbale, S. K.; Wu, B.; Raie, M. N.; Pandrala, M.; Iagaru, A.; Brunsing, R. L.; Charville, G. W.; Park, W. G.; Ferrara, K. W.</p>
        <p class="info">Score: 3.5, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.07.574209' target='https://doi.org/10.1101/2024.01.07.574209'> 10.1101/2024.01.07.574209</a></p>
        <p class="abstract">We apply spatial transcriptomics and proteomics to select pancreatic cancer surface receptor targets for molecular imaging and theranostics using an approach that can be applied to many cancers. Selected cancer surfaceome epithelial markers were spatially correlated and provided specific cancer localization, whereas the spatial correlation between cancer markers and immune- cell or fibroblast markers was low. While molecular imaging of cancer-associated fibroblasts and integrins has been proposed for pancreatic cancer, our data point to the tight junction protein claudin-4 as a theranostic target. Claudin-4 expression increased [~]16 fold in cancer as compared with normal pancreas, and the tight junction localization conferred low background for imaging in normal tissue. We developed a peptide-based molecular imaging agent targeted to claudin-4 with accumulation to [~]25% injected activity per cc (IA/cc) in metastases and [~]18% IA/cc in tumors. Our work motivates a new approach for data-driven selection of molecular targets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.31.573772">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.31.573772" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.31.573772">
        <p class="paperTitle">High-resolution low-cost LCD 3D printing of microfluidics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.31.573772" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.31.573772" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shafique, H.; Karamzadeh, V.; Kim, G.; Morocz, Y.; Sohrabi-Kashani, A.; Shen, M. L.; Juncker, D.</p>
        <p class="info">Score: 7.7, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.31.573772' target='https://doi.org/10.1101/2023.12.31.573772'> 10.1101/2023.12.31.573772</a></p>
        <p class="abstract">The fabrication of microfluidic devices has progressed from cleanroom manufacturing to replica molding in polymers, and more recently to direct manufacturing by subtractive (e.g., laser machining) and additive (e.g., 3D printing) techniques, notably digital light processing (DLP) photopolymerization. However, many methods require technical expertise and while DLP 3D printers remain expensive at a cost [~]15-30K USD with [~]8M pixels that are 25-40 {micro}m in size. Here, we introduce (i) the use of low-cost ([~]150-600 USD) liquid crystal display (LCD) photopolymerization 3D printing with [~]8M-58M pixels that are 18-35 {micro}m in size for direct microfluidic device fabrication and (ii) a poly(ethylene glycol) diacrylate-based ink developed for LCD 3D printing (PLInk). We optimized PLInk for high resolution, fast 3D printing and biocompatibility while considering the illumination inhomogeneity and low power density of LCD 3D printers. We made lateral features as small as 75 {micro}m, 22-{micro}m-thick embedded membranes, and circular channels with a 110 {micro}m radius. We 3D printed microfluidic devices previously manufactured by other methods, including an embedded 3D micromixer, a membrane microvalve, and an autonomous capillaric circuit (CC) deployed for interferon-{gamma} detection with excellent performance (limit of detection: 12 pg mL-1, CV: 6.8%), and we demonstrated compatibility with cell culture. Finally, large area manufacturing was illustrated by printing 42 CCs with embedded microchannels in &lt;45 min. LCD 3D printing together with tailored inks pave the way for democratizing access to high-resolution manufacturing of ready-to-use microfluidic devices by anyone, anywhere.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574641">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574641" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574641">
        <p class="paperTitle">Imaging the structural connectome with hybrid diffusion MRI-microscopy tractography</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574641" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574641" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhu, S.; Huszar, I. N.; Cottaar, M.; Daubney, G.; Eichert, N.; Hanayik, T.; Khrapitchev, A. A.; Mars, R. B.; Mollink, J.; Sallet, J.; Scott, C.; Smart, A.; Jbabdi, S.; Miller, K. L.; Howard, A. F.</p>
        <p class="info">Score: 2.5, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574641' target='https://doi.org/10.1101/2024.01.08.574641'> 10.1101/2024.01.08.574641</a></p>
        <p class="abstract">Neuroanatomical tract tracing methods are fundamental in providing &#34;gold standard&#34; estimates of brain connectivity. However, tracer methods cannot be performed in humans and even in animals, we can only study projections from typically one or two injection sites per animal sacrificed. Orientation-sensitive microscopy techniques such as PLI provide an alternative where they can visualise detailed fibre orientations at the micron-scale across the whole brain. However, these methods are often most informative on orientations within the 2D imaging plane, with less reliable or missing through-plane information, restricting 3D tract reconstruction. Conversely, dMRI can estimate fibre orientations in 3D but at low resolution, which leads to many false positive and negative estimates of fibre trajectories.

To facilitate reconstruction of the microscopy-informed connectome, we develop a data-fusion method that complements 2D microscopy with through-plane information from diffusion MRI to construct 3D hybrid orientations that are both maximally informed by the high-resolution microscopy, have whole-brain coverage and can be input into existing tractography pipelines. Diffusion MRI can be readily acquired prior to microscopy meaning the same method is translatable across species, including in humans. Here we apply our method to an existing open-access macaque dataset and demonstrate (1) whole-brain microscopy-informed tractography (2) the advantages of hybrid tractography in two known tractography challenges, the gyral bias and bottleneck problem (3) how hybrid tractography appears to outperform diffusion-only tractography when compared to tracer data and (4) the generalisability of our hybrid method to different microscopy contrasts, facilitating wider translation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.06.574485">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.06.574485" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.06.574485">
        <p class="paperTitle">Predicting Serotonin Detection with DNA-Carbon Nanotube Sensors Across Multiple Spectral Wavelengths</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.06.574485" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.06.574485" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kelich, P.; Adams, J.; Jeong, S.; Navarro, N.; Landry, M. P.; Vukovic, L.</p>
        <p class="info">Score: 2.5, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.06.574485' target='https://doi.org/10.1101/2024.01.06.574485'> 10.1101/2024.01.06.574485</a></p>
        <p class="abstract">Owing to the value of DNA-wrapped single-walled carbon nanotube (SWNT) based sensors for chemically-specific imaging in biology, we explore machine learning (ML) predictions DNA-SWNT serotonin sensor responsivity as a function of DNA sequence based on the whole SWNT fluorescence spectra. Our analysis reveals the crucial role of DNA sequence in the binding modes of DNA-SWNTs to serotonin, with a smaller influence of SWNT chirality. Regression ML models trained on existing datasets predict the change in the fluorescence emission in response to serotonin, {Delta}F/F, at over a hundred wavelengths for new DNA-SWNT conjugates, successfully identifying some high- and low-response DNA sequences. Despite successful predictions, we also show that the finite size of the training dataset leads to limitations on prediction accuracy. Nevertheless, incorporating entire spectra into ML models enhances prediction robustness and facilitates the discovery of novel DNA-SWNT sensors. Our approaches show promise for identifying new chemical systems with specific sensing response characteristics, marking a valuable advancement in DNA-based system discovery.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
