<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Reengineering of a flavin-binding fluorescent protein using ProteinMPNN
Authors: Nikolaev, A.; Kuzmin, A.; Markeeva, E.; Kuznetsova, E.; Semenov, O.; Anuchina, A.; Remeeva, A.; Gushchin, I.
Score: 19.2, Published: 2023-08-27 DOI: 10.1101/2023.08.25.554855
Recent advances in machine learning techniques have led to development of a number of protein design and engineering approaches. One of them, ProteinMPNN, predicts an amino acid sequence that would fold and match user-defined backbone structure. In this short report, we test whether ProteinMPNN can be used to reengineer a flavin-binding fluorescent protein, CagFbFP.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Reengineering of a flavin-binding fluorescent protein using ProteinMPNN
Authors: Nikolaev, A.; Kuzmin, A.; Markeeva, E.; Kuznetsova, E.; Semenov, O.; Anuchina, A.; Remeeva, A.; Gushchin, I.
Score: 19.2, Published: 2023-08-27 DOI: 10.1101/2023.08.25.554855
Recent advances in machine learning techniques have led to development of a number of protein design and engineering approaches. One of them, ProteinMPNN, predicts an amino acid sequence that would fold and match user-defined backbone structure. In this short report, we test whether ProteinMPNN can be used to reengineer a flavin-binding fluorescent protein, CagFbFP." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-30T10:38:23+00:00" />
<meta property="article:modified_time" content="2023-08-30T10:38:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Reengineering of a flavin-binding fluorescent protein using ProteinMPNN
Authors: Nikolaev, A.; Kuzmin, A.; Markeeva, E.; Kuznetsova, E.; Semenov, O.; Anuchina, A.; Remeeva, A.; Gushchin, I.
Score: 19.2, Published: 2023-08-27 DOI: 10.1101/2023.08.25.554855
Recent advances in machine learning techniques have led to development of a number of protein design and engineering approaches. One of them, ProteinMPNN, predicts an amino acid sequence that would fold and match user-defined backbone structure. In this short report, we test whether ProteinMPNN can be used to reengineer a flavin-binding fluorescent protein, CagFbFP."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Reengineering of a flavin-binding fluorescent protein using ProteinMPNN\nAuthors: Nikolaev, A.; Kuzmin, A.; Markeeva, E.; Kuznetsova, E.; Semenov, O.; Anuchina, A.; Remeeva, A.; Gushchin, I.\nScore: 19.2, Published: 2023-08-27 DOI: 10.1101/2023.08.25.554855\nRecent advances in machine learning techniques have led to development of a number of protein design and engineering approaches. One of them, ProteinMPNN, predicts an amino acid sequence that would fold and match user-defined backbone structure. In this short report, we test whether ProteinMPNN can be used to reengineer a flavin-binding fluorescent protein, CagFbFP.",
  "keywords": [
    
  ],
  "articleBody": " Reengineering of a flavin-binding fluorescent protein using ProteinMPNN\nAuthors: Nikolaev, A.; Kuzmin, A.; Markeeva, E.; Kuznetsova, E.; Semenov, O.; Anuchina, A.; Remeeva, A.; Gushchin, I.\nScore: 19.2, Published: 2023-08-27 DOI: 10.1101/2023.08.25.554855\nRecent advances in machine learning techniques have led to development of a number of protein design and engineering approaches. One of them, ProteinMPNN, predicts an amino acid sequence that would fold and match user-defined backbone structure. In this short report, we test whether ProteinMPNN can be used to reengineer a flavin-binding fluorescent protein, CagFbFP. We fixed the native backbone conformation and the identity of 20 amino acids interacting with the chromophore (flavin mononucleotide, FMN), while letting ProteinMPNN predict the rest of the sequence. The software package suggested replacing 36-48 out of the remaining 86 amino acids. The three designs that we tested experimentally displayed different expression levels, yet all were able to bind FMN and displayed fluorescence, thermal stability and other properties similar to those of CagFbFP. Our results demonstrate that ProteinMPNN can be used to generate diverging unnatural variants of fluorescent proteins, and, more generally, to reengineer proteins without losing their ligand-binding capabilities.\nPRO-LDM: Protein Sequence Generation with Conditional Latent Diffusion Models\nAuthors: Jiang, Z.; Zhang, S.; Huang, R.; Mo, S.; Zhu, L.; Li, P.; Zhang, Z.; Chen, X.; Long, Y.; Xu, R.; Qing, R.\nScore: 14.0, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554145\nAbstractsProtein design with deep-learning based algorithms represents an emerging and highly promising field in molecular biology. Yet it remains a challenging task due to the complexity of protein sequences. Recent developments in deep generative models such as diffusion model have demonstrated impressive performance in protein structure design, yet left much to be desired in creating new sequences. Here we present PRO-LDM: an efficient protein sequence design framework that utilizes diffusion model in the latent space, utilizing the joint autoencoder as the backbone, to capture latent variable distribution and generate meaningful embeddings of protein sequences. PRO-LDM is capable of (1) learning feature representations in natural proteins both at amino-acid level and sequence level; (2) generating new sequences with natural properties and high diversity; and (3) conditionally designing new protein sequences with designated fitness. Our model presents a feasible pathway a generalizable new tool to extract both physical and chemical properties of amino acids, as well as evolutionary information within the protein sequences, for protein sequence design and functional optimization.\nBioengineering of a human innervated cardiac muscle model\nAuthors: Schneider, L. V.; Bao, G.; Methi, A.; Jensen, O.; Schmoll, K. A.; Setya, M. G.; Sakib, M. S.; Fahud, A. L.; Brockmoeller, J.; Fischer, A.; Liaw, N. Y.; Zimmermann, W.-H.; Zafeiriou, M.-P.\nScore: 12.4, Published: 2023-08-19 DOI: 10.1101/2023.08.18.552653\nCardiac autonomic neurons control cardiac contractility. Dysregulation of the autonomic nervous system can lead to sympathetic overdrive resulting in heart failure and an increased incidence of fatal arrhythmias. Here, we introduce innervated engineered human myocardium (iEHM), a novel model of neuro-cardiac junctions, constructed by fusion of a bioengineered neural organoid (BENO) patterned to autonomic nervous system and engineered human myocardium (EHM). Projections of sympathetic neurons into engineered human myocardium formed presynaptic terminals in close proximity to cardiomyocytes and an extensive vascular network co-developing in the tissues. Contractile responses to optogenetic stimulation of the accordingly engineered neuronal component demonstrated functionality of neuro-cardiac junctions in iEHM. This model will serve as a human surrogate system to delineate neuron and cardiac cell contribution to brain and heart diseases and is an important step towards engineering a human brain to heart axis in a dish.\nOpen-source milligram-scale, four channel, automated protein purification system\nAuthors: Puccinelli, R. R.; Sama, S. S.; Worthington, C. M.; Puschnik, A. S.; Pak, J. E.; Gomez-Sjoberg, R.\nScore: 11.2, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552685\nLiquid chromatography purification of multiple recombinant proteins, in parallel, could catalyze research and discovery if the processes are fast and approach the robustness of traditional, \"one-protein-at-a-time\" purification. Here, we report an automated, four channel chromatography platform that we have designed and validated for parallelized protein purification at milligram scales. The device can purify up to four proteins (each with its own single column), has inputs for up to eight buffers or solvents that can be directed to any of the four columns via a network of software-driven valves, and includes an automated fraction collector with ten positions for 1.5 or 5.0 mL collection tubes and four positions for 50 mL collection tubes for each column output. The control software can be accessed either via Python scripting, giving users full access to all steps of the purification process, or via a simple-to-navigate touch screen graphical user interface that does not require knowledge of the command line or any programming language. Using our instrument, we report milligram-scale, parallelized, single-column purification of a panel of mammalian cell expressed coronavirus (SARS-CoV-2, HCoV-229E, HCoV-OC43, HCoV-229E) trimeric Spike and monomeric Receptor Binding Domain (RBD) antigens, and monoclonal antibodies targeting SARS-CoV-2 Spike (S) and Influenza Hemagglutinin (HA). We include a detailed hardware build guide, and have made the controlling software open source, to allow others to build and customize their own protein purifier systems.\nStructural Determinants of Tendon Function During Development and Their Sensitivity to Mechanical Stimulation\nAuthors: Peterson, B. E.; Canonicco Castro, M. L.; McCarthy, H.; Buckley, N.; Dunne, N.; Rolfe, R. A.; Murphy, P.; Szczesny, S.\nScore: 4.3, Published: 2023-08-27 DOI: 10.1101/2023.08.25.554314\nThe load-bearing capabilities of tendon are acquired during neonatal stages of development, characterized by an abrupt increase in multiscale mechanical properties. While prior work has identified numerous changes within the collagenous structure during these developmental periods, the primary structural elements that give rise to this abrupt mechanical functionality, and their mechanobiological sensitivity, remains unclear. To address this gap in knowledge, we leveraged a combination of ultrastructural imaging, biochemical/thermodynamic assays, multiscale mechanical testing, and shear lag modeling to probe the dynamic structure-function relationships and establish their sensitivity to mechanical stimulation during tenogenesis. Mechanical testing and modeling suggested that the rapid increase in multiscale mechanics can be explained by an increasing fibril length and intrafibrillar crosslinking. To test this, we inhibited collagen crosslinking during development and observed a drastic reduction in multiscale mechanical capabilities that was explained by a reduction in both fibril modulus and length. Using muscle paralysis to investigate mechanosensitivity, we observed a significantly impaired multiscale mechanical response despite small changes in fibril diameter and fibril area fraction. While there was no change in crosslinking density, there was a decrease in thermal stability with flaccid paralysis, and our shear-lag model suggested that flaccid paralysis produces a reduction in fibril length and intrafibrillar crosslinking. Together, these data suggest that both intrafibrillar crosslink formation and fibril elongation are critical to the formation of load-bearing capabilities in tenogenesis and are sensitive to musculoskeletal activity. These findings provide critical insights into the biological mechanisms that give rise to load-bearing soft tissue.\nSecond Harmonic Generation Imaging Reveals Entanglement of Collagen Fibers in the Elephant Trunk Skin Dermis\nAuthors: Schulz, A.; Plotczyk, M.; Sordilla, S.; Boyle, M.; Singal, K.; Reidenberg, J.; Hu, D.; Higgins, C.\nScore: 11.5, Published: 2023-08-15 DOI: 10.1101/2023.08.11.553031\nForm-function relationships often have tradeoffs: if a material is tough, it is often inflexible, and vice versa. This is particularly relevant for the elephant trunk, where the skin should be protective yet elastic. To investigate how this is achieved, we used classical histochemical staining and second harmonic generation microscopy to describe the morphology and composition of elephant trunk skin. We report structure at the macro and micro scales, from the thickness of the dermis to the interaction of 10 m thick collagen fibers. We analyzed several sites along the length of the trunk, to compare and contrast the dorsal-ventral and proximal-distal skin morphologies and compositions. We find the dorsal skin of the elephant trunk can have keratin armor layers over 2mm thick, which is nearly 100 times the thickness of the equivalent layer in human skin. We also found that the structural support layer (the dermis) of elephant trunk contains a distribution of collagen-I (COL1) fibers in both perpendicular and parallel arrangement. The bimodal distribution of collagen is seen across all portions of the trunk, and is dissimilar from that of human skin where one orientation dominates within a body site. We hypothesize that this distribution of COL1 in the elephant trunk allows both flexibility and load-bearing capabilities. Additionally, when viewing individual fiber interaction of 10 m thick collagen, we find the fiber crossings per unit volume are five times more common than in human skin, suggesting that the fibers are entangled. We surmise that these intriguing structures permit both flexibility and strength in the elephant trunk. The complex nature of the elephant skin may inspire the design of materials that can combine strength and flexibility.\nEngineering of a graphene oxide-based two-dimensional platform for immune activation and modulation\nAuthors: Despotopoulou, D.; Stylianou, M.; Arellano, L. M.; Kisby, T.; Lozano, N.; Kostarelos, K.\nScore: 2.0, Published: 2023-08-22 DOI: 10.1101/2023.08.22.553542\nNanoscale-based tools for immunomodulation are expected to offer a rich battery of options for more targeted and safer approaches to achieve clinically effective manipulation of the local and systemic immune environment. In this study, we aimed to design nanoscale constructs based on graphene oxide (GO) nanosheets as platform carriers for the TLR7/8 agonist Resiquimod (R848). The non-covalent complexation of R848 molecules on the GO surface resulted in stable complexes by preserving their biological activity. The physicochemical properties, molecular quantification, as well as the overall performance of the complex were systematically investigated. We hypothesized the formation of GO:drug nano-constructs with strong colloidal stability over time, due to the strong {pi}-{pi} interactions between the R848 molecules and the GO surface, and identified that R848 loading efficiency consistently ranged around 75% (of starting molecules), quantified by HPLC and UV-Vis. The 2D morphology of the thin nanosheets was retained after complexation, determined by various (AFM and SEM) microscopic techniques. Based on the surface physicochemical characterization of the complexes by Raman, FTIR, XPS, and XRD, the formation of non-covalent interactions among the GO surface and the R848 molecules was confirmed. Most importantly, GO:R848 complexes did not compromise the biological activity of R848, and effectively activated macrophages in vitro. Collectively, this study demonstrates that thin GO sheets can act as platforms for the non-covalent association with small TLR7/8 agonist molecules, forming stable and highly reproducible complexes, that could be exploited as effective immunomodulatory agents.\nHuman plasma cells engineered to secrete bispecifics drive effective in vivo leukemia killing\nAuthors: Hill, T. F.; Narvekar, P.; Asher, G.; Camp, N.; Thomas, K. R.; Tasian, S. K.; Rawlings, D. J.; James, R. G.\nScore: 1.8, Published: 2023-08-27 DOI: 10.1101/2023.08.24.554523\nBispecific antibodies are an important tool for the management and treatment of acute leukemias. Advances in genome-engineering have enabled the generation of human plasma cells that secrete therapeutic proteins and are capable of long-term in vivo engraftment in humanized mouse models. As a next step towards clinical translation of engineered plasma cells (ePCs) towards cancer therapy, here we describe approaches for the expression and secretion of bispecific antibodies by human plasma cells. We show that human ePCs expressing either fragment crystallizable domain deficient anti-CD19 x anti-CD3 (blinatumomab) or anti-CD33 x anti-CD3 bispecific antibodies mediate T cell activation and direct T cell killing of specific primary human cell subsets and B-acute lymphoblastic leukemia or acute myeloid leukemia cell lines in vitro. We demonstrate that knockout of the self-expressed antigen, CD19, boosts anti-CD19 bispecific secretion by ePCs and prevents self-targeting. Further, anti-CD19 bispecific-ePCs elicited tumor eradication in vivo following local delivery in flank-implanted Raji lymphoma cells. Finally, immunodeficient mice engrafted with anti-CD19 bispecific-ePCs and autologous T cells potently prevented in vivo growth of CD19+ acute lymphoblastic leukemia in patient-derived xenografts. Collectively, these findings support further development of ePCs for use as a durable, local delivery system for the treatment of acute leukemias, and potentially other cancers.\nTESOGENASE, An Engineered Nuclease Editor for Enhanced Targeted Genome Integration\nAuthors: Nam, H.; Xie, K.; Majumdar, I.; Yang, S.; Starzyk, J.; Lee, D.; Shan, R.; Li, J.; Wu, H.\nScore: 1.5, Published: 2023-08-29 DOI: 10.1101/2023.08.28.553855\nNon-viral DNA donor template has been widely used for targeted genomic integration by homologous recombination (HR). This process has become more efficient with RNA guided endonuclease editor system such as CRISPR/Cas9. Circular single stranded DNA (cssDNA) has been harnessed previously as a genome engineering catalyst (GATALYST) for efficient and safe targeted gene knock-in. However, the engineering efficiency is bottlenecked by the nucleoplasm trafficking and genomic tethering of cssDNA donor, especially for extra-large transgene integration. Here we developed enGager, enhanced GATALYST associated genome editor system by fusion of nucleus localization signal (NLS) peptide tagged Cas9 with various single stranded DNA binding protein modules through a GFP reporter Knock-in screening. The enGager system assembles an integrative genome integration machinery by forming tripartite complex for engineered nuclease editors, sgRNA and ssDNA donors, thereby facilitate the nucleus trafficking of DNA donors and increase their active local concentration at the targeted genomic site. When applied for genome integration with cssDNA donor templates to diverse genomic loci in various cell types, these enGagers outperform unfused editors. The enhancement of integration efficiency ranges from 1.5- to more than 6-fold, with the effect being more prominent for \u003e 4Kb transgene knock-in in primary cells. We further demonstrated that enGager mediated enhancement for genome integration is ssDNA, but less dsDNA dependent. Using one of the mini-enGagers, we demonstrated large chimeric antigen receptor (CAR) transgene integration in primary T cells with exceptional efficiency and anti-tumor function. These tripartite editors with ssDNA optimized genome engineering system (TESOGENASE) add a set of novel endonuclease editors into the gene-editing toolbox for potential cell and gene therapeutic development based on ssDNA mediated non-viral genome engineering.\nControlled disturbed flows on the endothelium result in increased inflammation, nuclear lamin disruption and heterochromatin condensation\nAuthors: Paddillaya, N.; Mahulkar, S.; Arakeri, J.; Gundiah, N.\nScore: 1.5, Published: 2023-08-28 DOI: 10.1101/2023.08.27.555040\nEarly atherosclerotic lesions often develop in arterial bifurcations and curvatures that experience complex disturbed flows. Lab-on-chip devices primarily adopt laminar unidirectional and oscillatory flows to characterize endothelial mechanobiology and test therapeutics. Such flows do not reproduce the time-varying, bidirectional, and complicated shear patterns on arterial walls. We fabricated an endothelium-on-chip device and developed a semi-analytical model to produce any prescribed wall shear pattern characterized by a shear rosette. We quantified responses of Human Aortic Endothelial Cell (HAEC) monolayers subjected to unidirectional laminar (Uni-L) and oscillatory (Uni-O) flows in straight microfluidic channels, and a circular shear rosette (Bi-O), defined by bidirectional oscillatory flows, in the new device. Immunofluorescence results show well-defined stress fibers and VE-cadherin under laminar flows. Cells were more cuboidal with significantly larger nuclear areas in the device as compared to other groups. Lamin A/C disruption and heterochromatin reorganization were evident in nuclei of cells in the Bi-O group but were absent in the Uni-L and No-flow control groups. Cells showed significantly elevated actin, VE-cadherin, inflammatory NF-kB, and lamin expressions in the device compared to other groups. A device to create controlled disturbed flows offers a platform to assess mechanobiology, has significant potential in personalized medicine, and reduces reliance on animal trials.\n",
  "wordCount" : "2494",
  "inLanguage": "en",
  "datePublished": "2023-08-30T10:38:23Z",
  "dateModified": "2023-08-30T10:38:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on August 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554855">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554855" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554855">
        <p class="paperTitle">Reengineering of a flavin-binding fluorescent protein using ProteinMPNN</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554855" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554855" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nikolaev, A.; Kuzmin, A.; Markeeva, E.; Kuznetsova, E.; Semenov, O.; Anuchina, A.; Remeeva, A.; Gushchin, I.</p>
        <p class="info">Score: 19.2, Published: 2023-08-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554855' target='https://doi.org/10.1101/2023.08.25.554855'> 10.1101/2023.08.25.554855</a></p>
        <p class="abstract">Recent advances in machine learning techniques have led to development of a number of protein design and engineering approaches. One of them, ProteinMPNN, predicts an amino acid sequence that would fold and match user-defined backbone structure. In this short report, we test whether ProteinMPNN can be used to reengineer a flavin-binding fluorescent protein, CagFbFP. We fixed the native backbone conformation and the identity of 20 amino acids interacting with the chromophore (flavin mononucleotide, FMN), while letting ProteinMPNN predict the rest of the sequence. The software package suggested replacing 36-48 out of the remaining 86 amino acids. The three designs that we tested experimentally displayed different expression levels, yet all were able to bind FMN and displayed fluorescence, thermal stability and other properties similar to those of CagFbFP. Our results demonstrate that ProteinMPNN can be used to generate diverging unnatural variants of fluorescent proteins, and, more generally, to reengineer proteins without losing their ligand-binding capabilities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554145">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554145" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554145">
        <p class="paperTitle">PRO-LDM: Protein Sequence Generation with Conditional Latent Diffusion Models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554145" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554145" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, Z.; Zhang, S.; Huang, R.; Mo, S.; Zhu, L.; Li, P.; Zhang, Z.; Chen, X.; Long, Y.; Xu, R.; Qing, R.</p>
        <p class="info">Score: 14.0, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554145' target='https://doi.org/10.1101/2023.08.22.554145'> 10.1101/2023.08.22.554145</a></p>
        <p class="abstract">AbstractsProtein design with deep-learning based algorithms represents an emerging and highly promising field in molecular biology. Yet it remains a challenging task due to the complexity of protein sequences. Recent developments in deep generative models such as diffusion model have demonstrated impressive performance in protein structure design, yet left much to be desired in creating new sequences. Here we present PRO-LDM: an efficient protein sequence design framework that utilizes diffusion model in the latent space, utilizing the joint autoencoder as the backbone, to capture latent variable distribution and generate meaningful embeddings of protein sequences. PRO-LDM is capable of (1) learning feature representations in natural proteins both at amino-acid level and sequence level; (2) generating new sequences with natural properties and high diversity; and (3) conditionally designing new protein sequences with designated fitness. Our model presents a feasible pathway a generalizable new tool to extract both physical and chemical properties of amino acids, as well as evolutionary information within the protein sequences, for protein sequence design and functional optimization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.552653">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.552653" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.552653">
        <p class="paperTitle">Bioengineering of a human innervated cardiac muscle model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.552653" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.552653" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schneider, L. V.; Bao, G.; Methi, A.; Jensen, O.; Schmoll, K. A.; Setya, M. G.; Sakib, M. S.; Fahud, A. L.; Brockmoeller, J.; Fischer, A.; Liaw, N. Y.; Zimmermann, W.-H.; Zafeiriou, M.-P.</p>
        <p class="info">Score: 12.4, Published: 2023-08-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.552653' target='https://doi.org/10.1101/2023.08.18.552653'> 10.1101/2023.08.18.552653</a></p>
        <p class="abstract">Cardiac autonomic neurons control cardiac contractility. Dysregulation of the autonomic nervous system can lead to sympathetic overdrive resulting in heart failure and an increased incidence of fatal arrhythmias. Here, we introduce innervated engineered human myocardium (iEHM), a novel model of neuro-cardiac junctions, constructed by fusion of a bioengineered neural organoid (BENO) patterned to autonomic nervous system and engineered human myocardium (EHM). Projections of sympathetic neurons into engineered human myocardium formed presynaptic terminals in close proximity to cardiomyocytes and an extensive vascular network co-developing in the tissues. Contractile responses to optogenetic stimulation of the accordingly engineered neuronal component demonstrated functionality of neuro-cardiac junctions in iEHM. This model will serve as a human surrogate system to delineate neuron and cardiac cell contribution to brain and heart diseases and is an important step towards engineering a human brain to heart axis in a dish.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552685">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552685" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552685">
        <p class="paperTitle">Open-source milligram-scale, four channel, automated protein purification system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552685" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552685" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Puccinelli, R. R.; Sama, S. S.; Worthington, C. M.; Puschnik, A. S.; Pak, J. E.; Gomez-Sjoberg, R.</p>
        <p class="info">Score: 11.2, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552685' target='https://doi.org/10.1101/2023.08.09.552685'> 10.1101/2023.08.09.552685</a></p>
        <p class="abstract">Liquid chromatography purification of multiple recombinant proteins, in parallel, could catalyze research and discovery if the processes are fast and approach the robustness of traditional, &#34;one-protein-at-a-time&#34; purification. Here, we report an automated, four channel chromatography platform that we have designed and validated for parallelized protein purification at milligram scales. The device can purify up to four proteins (each with its own single column), has inputs for up to eight buffers or solvents that can be directed to any of the four columns via a network of software-driven valves, and includes an automated fraction collector with ten positions for 1.5 or 5.0 mL collection tubes and four positions for 50 mL collection tubes for each column output. The control software can be accessed either via Python scripting, giving users full access to all steps of the purification process, or via a simple-to-navigate touch screen graphical user interface that does not require knowledge of the command line or any programming language. Using our instrument, we report milligram-scale, parallelized, single-column purification of a panel of mammalian cell expressed coronavirus (SARS-CoV-2, HCoV-229E, HCoV-OC43, HCoV-229E) trimeric Spike and monomeric Receptor Binding Domain (RBD) antigens, and monoclonal antibodies targeting SARS-CoV-2 Spike (S) and Influenza Hemagglutinin (HA). We include a detailed hardware build guide, and have made the controlling software open source, to allow others to build and customize their own protein purifier systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554314">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554314" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554314">
        <p class="paperTitle">Structural Determinants of Tendon Function During Development and Their Sensitivity to Mechanical Stimulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554314" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554314" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peterson, B. E.; Canonicco Castro, M. L.; McCarthy, H.; Buckley, N.; Dunne, N.; Rolfe, R. A.; Murphy, P.; Szczesny, S.</p>
        <p class="info">Score: 4.3, Published: 2023-08-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554314' target='https://doi.org/10.1101/2023.08.25.554314'> 10.1101/2023.08.25.554314</a></p>
        <p class="abstract">The load-bearing capabilities of tendon are acquired during neonatal stages of development, characterized by an abrupt increase in multiscale mechanical properties. While prior work has identified numerous changes within the collagenous structure during these developmental periods, the primary structural elements that give rise to this abrupt mechanical functionality, and their mechanobiological sensitivity, remains unclear. To address this gap in knowledge, we leveraged a combination of ultrastructural imaging, biochemical/thermodynamic assays, multiscale mechanical testing, and shear lag modeling to probe the dynamic structure-function relationships and establish their sensitivity to mechanical stimulation during tenogenesis. Mechanical testing and modeling suggested that the rapid increase in multiscale mechanics can be explained by an increasing fibril length and intrafibrillar crosslinking. To test this, we inhibited collagen crosslinking during development and observed a drastic reduction in multiscale mechanical capabilities that was explained by a reduction in both fibril modulus and length. Using muscle paralysis to investigate mechanosensitivity, we observed a significantly impaired multiscale mechanical response despite small changes in fibril diameter and fibril area fraction. While there was no change in crosslinking density, there was a decrease in thermal stability with flaccid paralysis, and our shear-lag model suggested that flaccid paralysis produces a reduction in fibril length and intrafibrillar crosslinking. Together, these data suggest that both intrafibrillar crosslink formation and fibril elongation are critical to the formation of load-bearing capabilities in tenogenesis and are sensitive to musculoskeletal activity. These findings provide critical insights into the biological mechanisms that give rise to load-bearing soft tissue.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553031">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553031" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553031">
        <p class="paperTitle">Second Harmonic Generation Imaging Reveals Entanglement of Collagen Fibers in the Elephant Trunk Skin Dermis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553031" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553031" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schulz, A.; Plotczyk, M.; Sordilla, S.; Boyle, M.; Singal, K.; Reidenberg, J.; Hu, D.; Higgins, C.</p>
        <p class="info">Score: 11.5, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553031' target='https://doi.org/10.1101/2023.08.11.553031'> 10.1101/2023.08.11.553031</a></p>
        <p class="abstract">Form-function relationships often have tradeoffs: if a material is tough, it is often inflexible, and vice versa. This is particularly relevant for the elephant trunk, where the skin should be protective yet elastic. To investigate how this is achieved, we used classical histochemical staining and second harmonic generation microscopy to describe the morphology and composition of elephant trunk skin. We report structure at the macro and micro scales, from the thickness of the dermis to the interaction of 10 m thick collagen fibers. We analyzed several sites along the length of the trunk, to compare and contrast the dorsal-ventral and proximal-distal skin morphologies and compositions. We find the dorsal skin of the elephant trunk can have keratin armor layers over 2mm thick, which is nearly 100 times the thickness of the equivalent layer in human skin. We also found that the structural support layer (the dermis) of elephant trunk contains a distribution of collagen-I (COL1) fibers in both perpendicular and parallel arrangement. The bimodal distribution of collagen is seen across all portions of the trunk, and is dissimilar from that of human skin where one orientation dominates within a body site. We hypothesize that this distribution of COL1 in the elephant trunk allows both flexibility and load-bearing capabilities. Additionally, when viewing individual fiber interaction of 10 m thick collagen, we find the fiber crossings per unit volume are five times more common than in human skin, suggesting that the fibers are entangled. We surmise that these intriguing structures permit both flexibility and strength in the elephant trunk. The complex nature of the elephant skin may inspire the design of materials that can combine strength and flexibility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.553542">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.553542" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.553542">
        <p class="paperTitle">Engineering of a graphene oxide-based two-dimensional platform for immune activation and modulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.553542" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.553542" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Despotopoulou, D.; Stylianou, M.; Arellano, L. M.; Kisby, T.; Lozano, N.; Kostarelos, K.</p>
        <p class="info">Score: 2.0, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.553542' target='https://doi.org/10.1101/2023.08.22.553542'> 10.1101/2023.08.22.553542</a></p>
        <p class="abstract">Nanoscale-based tools for immunomodulation are expected to offer a rich battery of options for more targeted and safer approaches to achieve clinically effective manipulation of the local and systemic immune environment. In this study, we aimed to design nanoscale constructs based on graphene oxide (GO) nanosheets as platform carriers for the TLR7/8 agonist Resiquimod (R848). The non-covalent complexation of R848 molecules on the GO surface resulted in stable complexes by preserving their biological activity. The physicochemical properties, molecular quantification, as well as the overall performance of the complex were systematically investigated. We hypothesized the formation of GO:drug nano-constructs with strong colloidal stability over time, due to the strong {pi}-{pi} interactions between the R848 molecules and the GO surface, and identified that R848 loading efficiency consistently ranged around 75% (of starting molecules), quantified by HPLC and UV-Vis. The 2D morphology of the thin nanosheets was retained after complexation, determined by various (AFM and SEM) microscopic techniques. Based on the surface physicochemical characterization of the complexes by Raman, FTIR, XPS, and XRD, the formation of non-covalent interactions among the GO surface and the R848 molecules was confirmed. Most importantly, GO:R848 complexes did not compromise the biological activity of R848, and effectively activated macrophages in vitro. Collectively, this study demonstrates that thin GO sheets can act as platforms for the non-covalent association with small TLR7/8 agonist molecules, forming stable and highly reproducible complexes, that could be exploited as effective immunomodulatory agents.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554523">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554523" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554523">
        <p class="paperTitle">Human plasma cells engineered to secrete bispecifics drive effective in vivo leukemia killing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554523" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554523" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hill, T. F.; Narvekar, P.; Asher, G.; Camp, N.; Thomas, K. R.; Tasian, S. K.; Rawlings, D. J.; James, R. G.</p>
        <p class="info">Score: 1.8, Published: 2023-08-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554523' target='https://doi.org/10.1101/2023.08.24.554523'> 10.1101/2023.08.24.554523</a></p>
        <p class="abstract">Bispecific antibodies are an important tool for the management and treatment of acute leukemias. Advances in genome-engineering have enabled the generation of human plasma cells that secrete therapeutic proteins and are capable of long-term in vivo engraftment in humanized mouse models. As a next step towards clinical translation of engineered plasma cells (ePCs) towards cancer therapy, here we describe approaches for the expression and secretion of bispecific antibodies by human plasma cells. We show that human ePCs expressing either fragment crystallizable domain deficient anti-CD19 x anti-CD3 (blinatumomab) or anti-CD33 x anti-CD3 bispecific antibodies mediate T cell activation and direct T cell killing of specific primary human cell subsets and B-acute lymphoblastic leukemia or acute myeloid leukemia cell lines in vitro. We demonstrate that knockout of the self-expressed antigen, CD19, boosts anti-CD19 bispecific secretion by ePCs and prevents self-targeting. Further, anti-CD19 bispecific-ePCs elicited tumor eradication in vivo following local delivery in flank-implanted Raji lymphoma cells. Finally, immunodeficient mice engrafted with anti-CD19 bispecific-ePCs and autologous T cells potently prevented in vivo growth of CD19&#43; acute lymphoblastic leukemia in patient-derived xenografts. Collectively, these findings support further development of ePCs for use as a durable, local delivery system for the treatment of acute leukemias, and potentially other cancers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.553855">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.553855" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.553855">
        <p class="paperTitle">TESOGENASE, An Engineered Nuclease Editor for Enhanced Targeted Genome Integration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.553855" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.553855" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nam, H.; Xie, K.; Majumdar, I.; Yang, S.; Starzyk, J.; Lee, D.; Shan, R.; Li, J.; Wu, H.</p>
        <p class="info">Score: 1.5, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.553855' target='https://doi.org/10.1101/2023.08.28.553855'> 10.1101/2023.08.28.553855</a></p>
        <p class="abstract">Non-viral DNA donor template has been widely used for targeted genomic integration by homologous recombination (HR). This process has become more efficient with RNA guided endonuclease editor system such as CRISPR/Cas9. Circular single stranded DNA (cssDNA) has been harnessed previously as a genome engineering catalyst (GATALYST) for efficient and safe targeted gene knock-in. However, the engineering efficiency is bottlenecked by the nucleoplasm trafficking and genomic tethering of cssDNA donor, especially for extra-large transgene integration. Here we developed enGager, enhanced GATALYST associated genome editor system by fusion of nucleus localization signal (NLS) peptide tagged Cas9 with various single stranded DNA binding protein modules through a GFP reporter Knock-in screening. The enGager system assembles an integrative genome integration machinery by forming tripartite complex for engineered nuclease editors, sgRNA and ssDNA donors, thereby facilitate the nucleus trafficking of DNA donors and increase their active local concentration at the targeted genomic site. When applied for genome integration with cssDNA donor templates to diverse genomic loci in various cell types, these enGagers outperform unfused editors. The enhancement of integration efficiency ranges from 1.5- to more than 6-fold, with the effect being more prominent for &gt; 4Kb transgene knock-in in primary cells. We further demonstrated that enGager mediated enhancement for genome integration is ssDNA, but less dsDNA dependent. Using one of the mini-enGagers, we demonstrated large chimeric antigen receptor (CAR) transgene integration in primary T cells with exceptional efficiency and anti-tumor function. These tripartite editors with ssDNA optimized genome engineering system (TESOGENASE) add a set of novel endonuclease editors into the gene-editing toolbox for potential cell and gene therapeutic development based on ssDNA mediated non-viral genome engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.27.555040">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.27.555040" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.27.555040">
        <p class="paperTitle">Controlled disturbed flows on the endothelium result in increased inflammation, nuclear lamin disruption and heterochromatin condensation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.27.555040" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.27.555040" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Paddillaya, N.; Mahulkar, S.; Arakeri, J.; Gundiah, N.</p>
        <p class="info">Score: 1.5, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.27.555040' target='https://doi.org/10.1101/2023.08.27.555040'> 10.1101/2023.08.27.555040</a></p>
        <p class="abstract">Early atherosclerotic lesions often develop in arterial bifurcations and curvatures that experience complex disturbed flows. Lab-on-chip devices primarily adopt laminar unidirectional and oscillatory flows to characterize endothelial mechanobiology and test therapeutics. Such flows do not reproduce the time-varying, bidirectional, and complicated shear patterns on arterial walls. We fabricated an endothelium-on-chip device and developed a semi-analytical model to produce any prescribed wall shear pattern characterized by a shear rosette. We quantified responses of Human Aortic Endothelial Cell (HAEC) monolayers subjected to unidirectional laminar (Uni-L) and oscillatory (Uni-O) flows in straight microfluidic channels, and a circular shear rosette (Bi-O), defined by bidirectional oscillatory flows, in the new device. Immunofluorescence results show well-defined stress fibers and VE-cadherin under laminar flows. Cells were more cuboidal with significantly larger nuclear areas in the device as compared to other groups. Lamin A/C disruption and heterochromatin reorganization were evident in nuclei of cells in the Bi-O group but were absent in the Uni-L and No-flow control groups. Cells showed significantly elevated actin, VE-cadherin, inflammatory NF-kB, and lamin expressions in the device compared to other groups. A device to create controlled disturbed flows offers a platform to assess mechanobiology, has significant potential in personalized medicine, and reduces reliance on animal trials.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
