<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Human neuron subtype programming through combinatorial patterning with scRNA-seq readouts
Authors: Lin, H.-C.; Janssens, J.; Kroell, A.-S.; Hornauer, P.; Santel, M.; Okamoto, R.; Karava, K.; Priouret, M.; Garcia, M. P.; Schroeter, M.; Camp, J. G.; Treutlein, B.
Score: 46.7, Published: 2023-12-20 DOI: 10.1101/2023.12.12.571318
Human neurons programmed from induced pluripotent stem cells (iPSCs) through transcription factor (TF) overexpression are widely used to study neuronal differentiation mechanisms and to model neurological diseases.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Human neuron subtype programming through combinatorial patterning with scRNA-seq readouts
Authors: Lin, H.-C.; Janssens, J.; Kroell, A.-S.; Hornauer, P.; Santel, M.; Okamoto, R.; Karava, K.; Priouret, M.; Garcia, M. P.; Schroeter, M.; Camp, J. G.; Treutlein, B.
Score: 46.7, Published: 2023-12-20 DOI: 10.1101/2023.12.12.571318
Human neurons programmed from induced pluripotent stem cells (iPSCs) through transcription factor (TF) overexpression are widely used to study neuronal differentiation mechanisms and to model neurological diseases." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-24T10:38:02+00:00" />
<meta property="article:modified_time" content="2023-12-24T10:38:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Human neuron subtype programming through combinatorial patterning with scRNA-seq readouts
Authors: Lin, H.-C.; Janssens, J.; Kroell, A.-S.; Hornauer, P.; Santel, M.; Okamoto, R.; Karava, K.; Priouret, M.; Garcia, M. P.; Schroeter, M.; Camp, J. G.; Treutlein, B.
Score: 46.7, Published: 2023-12-20 DOI: 10.1101/2023.12.12.571318
Human neurons programmed from induced pluripotent stem cells (iPSCs) through transcription factor (TF) overexpression are widely used to study neuronal differentiation mechanisms and to model neurological diseases."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Human neuron subtype programming through combinatorial patterning with scRNA-seq readouts\nAuthors: Lin, H.-C.; Janssens, J.; Kroell, A.-S.; Hornauer, P.; Santel, M.; Okamoto, R.; Karava, K.; Priouret, M.; Garcia, M. P.; Schroeter, M.; Camp, J. G.; Treutlein, B.\nScore: 46.7, Published: 2023-12-20 DOI: 10.1101/2023.12.12.571318\nHuman neurons programmed from induced pluripotent stem cells (iPSCs) through transcription factor (TF) overexpression are widely used to study neuronal differentiation mechanisms and to model neurological diseases.",
  "keywords": [
    
  ],
  "articleBody": " Human neuron subtype programming through combinatorial patterning with scRNA-seq readouts\nAuthors: Lin, H.-C.; Janssens, J.; Kroell, A.-S.; Hornauer, P.; Santel, M.; Okamoto, R.; Karava, K.; Priouret, M.; Garcia, M. P.; Schroeter, M.; Camp, J. G.; Treutlein, B.\nScore: 46.7, Published: 2023-12-20 DOI: 10.1101/2023.12.12.571318\nHuman neurons programmed from induced pluripotent stem cells (iPSCs) through transcription factor (TF) overexpression are widely used to study neuronal differentiation mechanisms and to model neurological diseases. However, heterogeneity in TF-induced neurons can confound results that are sensitive to neuron subtypes. Engineering of neuron subtypes remains challenging without comprehensive interrogation of developmental signaling pathways that establish diverse neuronal identities. Here, we systematically screened for neuron subtypes through combinatorial morphogen treatments on TF-induced neurons. Using high-throughput multiplex single-cell transcriptomics, we analyze 480 morphogen combinations in nearly 700,000 neurons. Patterned TF-induced neurons are diverse excitatory and inhibitory neuron subtypes mapped to the anterior-posterior and dorsal-ventral axes of neural tube development, including dorsal and ventral forebrain, midbrain, hindbrain, spinal cord, peripheral sympathetic and sensory neurons. We also adopt a computational framework to identify combinatorial signalings and upregulated transcription factors that drive neurons toward specific subtypes. Our systematic approach could serve as a general strategy for cell subtype engineering and to investigate how cooperative signaling drives cell fate acquisition.\nElectroadhesive hydrogel interface for prolonged mucosal theranostics\nAuthors: Ying, B.; Nan, K.; Zhu, Q.; Khuu, T.; Ro, H.; Qin, S.; Wang, S.; Jiang, K.; Chen, Y.; Bao, G.; Jenkins, J.; Pettinari, A.; Kuosmanen, J.; Ishida, K.; Lopez, A.; Li, J.; Hayward, A.; Langer, R.; Traverso, G.\nScore: 7.4, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572448\nEstablishing a robust and intimate mucosal interface that allows medical devices to remain within lumen-confined organs for extended periods has valuable applications, particularly for gastrointestinal (GI) theranostics. Here, we report the development of e-GLUE, an electroadhesive hydrogel interface for robust and prolonged mucosal retention following electrical activation. Notably, this novel mucosal adhesion mechanism can increase the adhesion energy of hydrogels on the mucosa by up to 30-fold and enable in vivo GI retention of e-GLUE devices for up to 30 days. Strong mucosal adhesion occurs within one minute of electrical activation, despite the presence of luminal fluid, mucus exposure, and organ motility, thereby ensuring compatibility with complex in vivo environments. In swine studies, we demonstrate the utility of e-GLUE for mucosal hemostasis, sustained local delivery of therapeutics, and intimate biosensing in the GI tract. This system can enable improved treatments for various health conditions, including gastrointestinal bleeding, inflammatory bowel disease, and diagnostic applications in the GI tract and beyond.\nSingle cell transcriptomics reveals chondrocyte differentiation dynamics in vivo and in vitro\nAuthors: Lawrence, J. E. G.; Woods, S.; Roberts, K.; Sumanaweera, D. N.; Balogh, P.; Predeus, A. V.; He, P.; Li, T.; Polanski, K.; Prigmore, E.; Tuck, E.; Mamanova, L.; Zhou, D.; Webb, S.; Jardine, L.; He, X.; Barker, R. A.; Haniffa, M.; Flanagan, A. M.; Young, M. D.; Behjati, S.; Bayraktar, O.; Kimber, S. J.; Teichmann, S.\nScore: 4.2, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572425\nThe consistent production of in vitro chondrocytes that faithfully recapitulate in vivo development would be of great benefit for musculoskeletal disease modelling and regenerative medicine. Current efforts are often limited by off-target differentiation, resulting in a heterogeneous product. Furthermore, the lack of comparison to human embryonic tissue, precludes detailed evaluation of in vitro cells. Here, we perform single-cell RNA sequencing of embryonic long bones dissected from first trimester hind limbs from a range of gestational ages. We combine this with publicly available data to form a detailed atlas of endochondral ossification, which we then use to evaluate a series of published in vitro chondrogenesis protocols, finding substantial variability in cell states produced by each. We apply single-nuclear RNA sequencing to one protocol to enable direct comparison between in vitro and in vivo, and perform trajectory alignment between the two to reveal differentiation dynamics at the single-cell level, shedding new light on off-target differentiation in vitro. Using this information, we inhibit the activity of FOXO1, a transcription factor predicted to be active in embryonic bone development and in chondrogenic cells in vitro, and increase chondrocyte transcripts in vitro. This study therefore presents a new framework for evaluating tissue engineering protocols, using single-cell data from human development to drive improvement and bring the prospect of true engineered cartilage closer to reality.\nCellular Senescence Impairs Tendon Extracellular Matrix Remodeling in Response to Mechanical Unloading\nAuthors: Stowe, E. J.; Keller, M. R.; Connizzo, B. K.\nScore: 4.2, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572594\nMusculoskeletal injuries, including tendinopathies, present a significant clinical burden for aging populations. While the biological drivers of age-related declines in tendon function are poorly understood, it is well accepted that dysregulation of extracellular matrix (ECM) remodeling plays a role in chronic tendon degeneration. Senescent cells, which have been associated with multiple degenerative pathologies in musculoskeletal tissues, secrete a highly pro-inflammatory senescence-associated secretory phenotype (SASP) that has potential to promote ECM breakdown. However, the role of senescent cells in the dysregulation of tendon ECM homeostasis is largely unknown. To assess this directly, we developed an in vitro model of induced cellular senescence in murine tendon explants. This novel technique enables us to study the isolated interactions of senescent cells and their native ECM without interference from age-related systemic changes. We document multiple biomarkers of cellular senescence in induced tendon explants including cell cycle arrest, apoptosis resistance, and SASP production. We then utilize this in vitro senescence model to compare the ECM remodeling response of young, naturally aged, and senescent tendons to an altered mechanical stimulus. We found that both senescence and aging independently led to alterations in ECM-related gene expression, reductions in protein synthesis, and tissue compositional changes. Furthermore, MMP activity was sustained, thus shifting the remodeling balance of aged and senescent tissues towards degradation over production. Together, this demonstrates that cellular senescence plays a role in the altered mechano-response of aged tendons and likely contributes to poor clinical outcomes in aging populations.\nReinforcement Learning for Control of Human Locomotion in Simulation\nAuthors: Dashkovets, A.; Laschowski, B.\nScore: 4.2, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572447\nControl of robotic leg prostheses and exoskeletons is an open challenge. Computer modeling and simulation can be used to study the dynamics and control of human walking and extract principles that can be programmed into robotic legs to behave similar to biological legs. In this study, we present the development of an efficient two-layer Q-learning algorithm, with k-d trees, that operates over continuous action spaces and a reward model that estimates the degree of muscle activation similarity between the agent and human state-to-action pairs and state-to-action sequences. We used a human musculoskeletal model acting in a high-dimensional, physics-based simulation environment to train and evaluate our algorithm to simulate biomimetic walking. We used imitation learning and artificial bio-mechanics data to accelerate training via expert demonstrations and used experimental human data to compare and validate our predictive simulations, achieving 79% accuracy. Also, when compared to the previous state-of-the-art that used deep deterministic policy gradient, our algorithm was significantly more efficient with lower computational and memory storage requirements (i.e., requiring 7 times less RAM and 87 times less CPU compute), which can benefit real-time embedded computing. Overall, our new two-layer Q-learning algorithm using sequential data for continuous imitation of human locomotion serves as a first step towards the development of bioinspired controllers for robotic prosthetic legs and exoskeletons. Future work will focus on improving the prediction accuracy compared to experimental data and expanding our simulations to other locomotor activities.\nStructure-conditioned masked language models for protein sequence design generalize beyond the native sequence space\nAuthors: Akpinaroglu, D.; Seki, K.; Guo, A.; Zhu, E.; Kelly, M. J.; Kortemme, T.\nScore: 18.1, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571823\nMachine learning has revolutionized computational protein design, enabling significant progress in protein backbone generation and sequence design. Here, we introduce Frame2seq, a structure-conditioned masked language model for protein sequence design. Frame2seq generates sequences in a single pass, achieves 49.1% sequence recovery on the CATH 4.2 test dataset, and accurately estimates the error in its own predictions, outperforming the autoregressive ProteinMPNN model with over six times faster inference. To probe the ability of Frame2seq to generate novel designs beyond the native-like sequence space it was trained on, we experimentally test 26 Frame2seq designs for de novo backbones with low identity to the starting sequences. We show that Frame2seq successfully designs soluble (22/26), monomeric, folded, and stable proteins (17/26), including a design with 0% sequence identity to native. The speed and accuracy of Frame2seq will accelerate exploration of novel sequence space across diverse design tasks, including challenging applications such as multi-objective optimization.\nAllosteric Modulation of Fluorescence Revealed by Hydrogen Bond Dynamics in a Genetically Encoded Maltose Biosensor\nAuthors: BERKSOZ, M.; ATILGAN, C.\nScore: 3.9, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572600\nGenetically encoded fluorescent biosensors (GEFBs) proved to be reliable tracers for many metabolites and cellular processes. In the simplest case, a fluorescent protein (FP) is genetically fused to a sensing protein which undergoes a conformational change upon ligand binding. This drives a rearrangement in the chromophore environment and changes the spectral properties of the FP. Structural determinants of successful biosensors are revealed only in hindsight when the crystal structures of both ligand-bound and ligand-free forms are available. This makes the development of new biosensors for desired analytes a long trial-and-error process. In the current study, we conducted microsecond-long all atom molecular dynamics (MD) simulations of a maltose biosensor in both the apo (dark) and holo (bright) forms. We performed detailed hydrogen bond occupancy analyses to shed light on the mechanism of ligand induced conformational change in the sensor protein and its allosteric effect on the chromophore environment. We find that two strong indicators for distinguishing bright and dark states of biosensors are due to substantial changes in hydrogen bond dynamics in the system and solvent accessibility of the chromophore.\nEncapsulation of AAVs into protein vault nanoparticles as a novel solution to gene therapy's neutralizing antibody problem\nAuthors: Collins, L. T.; Beatty, W.; Moyo, B.; Alves-Bezerra, M.; Hurley, A.; Lagor, W.; Bao, G.; Ponnazhagan, S.; McNally, R.; Rome, L.; Curiel, D.\nScore: 16.2, Published: 2023-12-19 DOI: 10.1101/2023.11.29.569229\nAlthough adeno-associated virus (AAV) has enjoyed enormous success as a delivery modality for gene therapy, it continues to suffer from the high prevalence of preexisting neutralizing antibodies in human populations, limiting who can receive potentially life-saving treatments. In this regard, AAV therapies generally also must be administered as a single dose since neutralizing antibodies develop in patients who receive the virus. Strategies for circumventing these issues remain limited. As a novel solution, we employed SpyTag-SpyCatcher molecular glue technology to facilitate packaging of AAVs inside of recombinant protein vault nanoparticles. Vaults are endogenous proteinaceous organelles produced by mammalian cells, so they are recognized as self by the immune system. They therefore can shield packaged molecules from neutralizing antibodies. Vault has previously been utilized to deliver drugs and proteins into cells, but our study represents the first time anyone has packaged an entire virus inside of vault. We showed that our vaultAAV (VAAV) delivery vehicle transduces cells in the presence of anti-AAV neutralizing serum. VAAV is positioned as a new gene therapy delivery platform with potential to overcome the neutralizing antibody problem and perhaps even allow administration of multiple doses, expanding the scope of AAV treatments.\nBiodistribution of DNA-origami nanostructures in live zebrafish embryos with single-cell resolution\nAuthors: Kolonelou, C.; Bräutigam, L.; Edwards, S.; Engström, E.; Dias, J. M.; Spratt, J.; Karampelias, C.; Wennmalm, S.; Brismar, H.; Andersson, O.; Teixeira, A. I.\nScore: 3.2, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572973\nDNA origami-based nanotechnology is a versatile tool for exploring fundamental biological questions and holds significant promise for future biomedical applications. However, the development of DNA origami-based therapeutic agents is hindered by the challenge of translating in vitro performance into effective applications in vivo. Here, we exploit the optical transparency of the embryonic zebrafish to track intravenously injected, fluorescently labelled wireframe DNA origami nanostructures. Our approach integrated long-term, high-resolution imaging of transgenic live embryos with single-cell RNA sequencing, to elucidate the biodistribution of DNA nanostructures over time, up to 3 days post-injection (dpi). Notably, we observed rapid accumulation of nanostructures in the caudal hematopoietic tissue (CHT), akin to the fetal liver in mammals. We tested the effects of coating the nanostructures with an oligolysine PEG copolymer (K-PEG), a widely used strategy to enhance their stability. The K-PEG coating mitigated the accumulation rate in CHT, enabling higher percentages of the nanostructures to engage with other tissues. Additionally, our findings highlighted the pivotal role of scavenger endothelial cells in DNA origami clearance, with K-PEG offering sustained protection for the nanostructures at the CHT. Furthermore, by monitoring DNA origami in a transgenic zebrafish line designed for targeted macrophage ablation, we found that macrophages contribute to nanostructure clearance at later time points. This study introduces a framework for the analyses of the biodistribution and clearance of DNA origami nanostructures in vivo with single cell resolution and establishes a foundation for the investigation of DNA origami-based nanomedicines in animal models.\nEngineered Protein Nanosheets for the Scale up of Mesenchymal Stem Cell Culture on Bioemulsions\nAuthors: Bosch-Fortea, M.; Nadal, C.; Chrysanthou, A.; Marciano, D.; Kanso, H.; Nguyen, N.; Gautrot, J.\nScore: 2.6, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572135\nThe rapid progress in cell therapies and stem cell technologies requires the development of novel bioprocessing and biomanufacturing pipelines able to cope with the scale up of cell manufacturing. In this respect, microdroplet technologies have already revolutionised the field of biotechnologies, but remain ill-suited to the culture of adherent cells. In this report, we describe the engineering of albumins with cell adhesive peptides for the stabilisation of microdroplets enabling the scale up of mesenchymal stem cell (MSC) expansion. We characterise the modified albumins prior to study their self-assembly at liquid-liquid interfaces via interfacial shear rheology, and mechanical strengthening through the formation of crosslinked nanosheets. The biofunctionalisation of these protein nanosheets is then characterised by fluorescence microscopy. In turn, the ability of the resulting bioactive microdroplets to promote rapid cell adhesion and expansion is examined and the extensive deposition of matrix associated with such cultures is characterised. The culture of MSCs is then scaled up 100 fold, first at the surface of fluorinated oil emulsions, then on plant-based emulsions stabilised by engineered protein nanosheets and the phenotype of resulting cells is characterised. The microdroplet culture system presented displays attractive advantages over existing technologies, in terms of simplicity of processing, compatibility with regulatory expectations and costs of production, and offers exciting opportunities for translation to cell manufacturing, for cell therapies and cultivated meat applications.\n",
  "wordCount" : "2412",
  "inLanguage": "en",
  "datePublished": "2023-12-24T10:38:02Z",
  "dateModified": "2023-12-24T10:38:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 24, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571318">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571318" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571318">
        <p class="paperTitle">Human neuron subtype programming through combinatorial patterning with scRNA-seq readouts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571318" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571318" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lin, H.-C.; Janssens, J.; Kroell, A.-S.; Hornauer, P.; Santel, M.; Okamoto, R.; Karava, K.; Priouret, M.; Garcia, M. P.; Schroeter, M.; Camp, J. G.; Treutlein, B.</p>
        <p class="info">Score: 46.7, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571318' target='https://doi.org/10.1101/2023.12.12.571318'> 10.1101/2023.12.12.571318</a></p>
        <p class="abstract">Human neurons programmed from induced pluripotent stem cells (iPSCs) through transcription factor (TF) overexpression are widely used to study neuronal differentiation mechanisms and to model neurological diseases. However, heterogeneity in TF-induced neurons can confound results that are sensitive to neuron subtypes. Engineering of neuron subtypes remains challenging without comprehensive interrogation of developmental signaling pathways that establish diverse neuronal identities. Here, we systematically screened for neuron subtypes through combinatorial morphogen treatments on TF-induced neurons. Using high-throughput multiplex single-cell transcriptomics, we analyze 480 morphogen combinations in nearly 700,000 neurons. Patterned TF-induced neurons are diverse excitatory and inhibitory neuron subtypes mapped to the anterior-posterior and dorsal-ventral axes of neural tube development, including dorsal and ventral forebrain, midbrain, hindbrain, spinal cord, peripheral sympathetic and sensory neurons. We also adopt a computational framework to identify combinatorial signalings and upregulated transcription factors that drive neurons toward specific subtypes. Our systematic approach could serve as a general strategy for cell subtype engineering and to investigate how cooperative signaling drives cell fate acquisition.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572448">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572448" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572448">
        <p class="paperTitle">Electroadhesive hydrogel interface for prolonged mucosal theranostics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572448" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572448" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ying, B.; Nan, K.; Zhu, Q.; Khuu, T.; Ro, H.; Qin, S.; Wang, S.; Jiang, K.; Chen, Y.; Bao, G.; Jenkins, J.; Pettinari, A.; Kuosmanen, J.; Ishida, K.; Lopez, A.; Li, J.; Hayward, A.; Langer, R.; Traverso, G.</p>
        <p class="info">Score: 7.4, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572448' target='https://doi.org/10.1101/2023.12.19.572448'> 10.1101/2023.12.19.572448</a></p>
        <p class="abstract">Establishing a robust and intimate mucosal interface that allows medical devices to remain within lumen-confined organs for extended periods has valuable applications, particularly for gastrointestinal (GI) theranostics. Here, we report the development of e-GLUE, an electroadhesive hydrogel interface for robust and prolonged mucosal retention following electrical activation. Notably, this novel mucosal adhesion mechanism can increase the adhesion energy of hydrogels on the mucosa by up to 30-fold and enable in vivo GI retention of e-GLUE devices for up to 30 days. Strong mucosal adhesion occurs within one minute of electrical activation, despite the presence of luminal fluid, mucus exposure, and organ motility, thereby ensuring compatibility with complex in vivo environments. In swine studies, we demonstrate the utility of e-GLUE for mucosal hemostasis, sustained local delivery of therapeutics, and intimate biosensing in the GI tract. This system can enable improved treatments for various health conditions, including gastrointestinal bleeding, inflammatory bowel disease, and diagnostic applications in the GI tract and beyond.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572425">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572425" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572425">
        <p class="paperTitle">Single cell transcriptomics reveals chondrocyte differentiation dynamics in vivo and in vitro</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572425" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572425" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lawrence, J. E. G.; Woods, S.; Roberts, K.; Sumanaweera, D. N.; Balogh, P.; Predeus, A. V.; He, P.; Li, T.; Polanski, K.; Prigmore, E.; Tuck, E.; Mamanova, L.; Zhou, D.; Webb, S.; Jardine, L.; He, X.; Barker, R. A.; Haniffa, M.; Flanagan, A. M.; Young, M. D.; Behjati, S.; Bayraktar, O.; Kimber, S. J.; Teichmann, S.</p>
        <p class="info">Score: 4.2, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572425' target='https://doi.org/10.1101/2023.12.20.572425'> 10.1101/2023.12.20.572425</a></p>
        <p class="abstract">The consistent production of in vitro chondrocytes that faithfully recapitulate in vivo development would be of great benefit for musculoskeletal disease modelling and regenerative medicine. Current efforts are often limited by off-target differentiation, resulting in a heterogeneous product. Furthermore, the lack of comparison to human embryonic tissue, precludes detailed evaluation of in vitro cells. Here, we perform single-cell RNA sequencing of embryonic long bones dissected from first trimester hind limbs from a range of gestational ages. We combine this with publicly available data to form a detailed atlas of endochondral ossification, which we then use to evaluate a series of published in vitro chondrogenesis protocols, finding substantial variability in cell states produced by each. We apply single-nuclear RNA sequencing to one protocol to enable direct comparison between in vitro and in vivo, and perform trajectory alignment between the two to reveal differentiation dynamics at the single-cell level, shedding new light on off-target differentiation in vitro. Using this information, we inhibit the activity of FOXO1, a transcription factor predicted to be active in embryonic bone development and in chondrogenic cells in vitro, and increase chondrocyte transcripts in vitro. This study therefore presents a new framework for evaluating tissue engineering protocols, using single-cell data from human development to drive improvement and bring the prospect of true engineered cartilage closer to reality.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572594">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572594" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572594">
        <p class="paperTitle">Cellular Senescence Impairs Tendon Extracellular Matrix Remodeling in Response to Mechanical Unloading</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572594" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572594" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stowe, E. J.; Keller, M. R.; Connizzo, B. K.</p>
        <p class="info">Score: 4.2, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572594' target='https://doi.org/10.1101/2023.12.22.572594'> 10.1101/2023.12.22.572594</a></p>
        <p class="abstract">Musculoskeletal injuries, including tendinopathies, present a significant clinical burden for aging populations. While the biological drivers of age-related declines in tendon function are poorly understood, it is well accepted that dysregulation of extracellular matrix (ECM) remodeling plays a role in chronic tendon degeneration. Senescent cells, which have been associated with multiple degenerative pathologies in musculoskeletal tissues, secrete a highly pro-inflammatory senescence-associated secretory phenotype (SASP) that has potential to promote ECM breakdown. However, the role of senescent cells in the dysregulation of tendon ECM homeostasis is largely unknown. To assess this directly, we developed an in vitro model of induced cellular senescence in murine tendon explants. This novel technique enables us to study the isolated interactions of senescent cells and their native ECM without interference from age-related systemic changes. We document multiple biomarkers of cellular senescence in induced tendon explants including cell cycle arrest, apoptosis resistance, and SASP production. We then utilize this in vitro senescence model to compare the ECM remodeling response of young, naturally aged, and senescent tendons to an altered mechanical stimulus. We found that both senescence and aging independently led to alterations in ECM-related gene expression, reductions in protein synthesis, and tissue compositional changes. Furthermore, MMP activity was sustained, thus shifting the remodeling balance of aged and senescent tissues towards degradation over production. Together, this demonstrates that cellular senescence plays a role in the altered mechano-response of aged tendons and likely contributes to poor clinical outcomes in aging populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572447">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572447" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572447">
        <p class="paperTitle">Reinforcement Learning for Control of Human Locomotion in Simulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572447" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572447" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dashkovets, A.; Laschowski, B.</p>
        <p class="info">Score: 4.2, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572447' target='https://doi.org/10.1101/2023.12.19.572447'> 10.1101/2023.12.19.572447</a></p>
        <p class="abstract">Control of robotic leg prostheses and exoskeletons is an open challenge. Computer modeling and simulation can be used to study the dynamics and control of human walking and extract principles that can be programmed into robotic legs to behave similar to biological legs. In this study, we present the development of an efficient two-layer Q-learning algorithm, with k-d trees, that operates over continuous action spaces and a reward model that estimates the degree of muscle activation similarity between the agent and human state-to-action pairs and state-to-action sequences. We used a human musculoskeletal model acting in a high-dimensional, physics-based simulation environment to train and evaluate our algorithm to simulate biomimetic walking. We used imitation learning and artificial bio-mechanics data to accelerate training via expert demonstrations and used experimental human data to compare and validate our predictive simulations, achieving 79% accuracy. Also, when compared to the previous state-of-the-art that used deep deterministic policy gradient, our algorithm was significantly more efficient with lower computational and memory storage requirements (i.e., requiring 7 times less RAM and 87 times less CPU compute), which can benefit real-time embedded computing. Overall, our new two-layer Q-learning algorithm using sequential data for continuous imitation of human locomotion serves as a first step towards the development of bioinspired controllers for robotic prosthetic legs and exoskeletons. Future work will focus on improving the prediction accuracy compared to experimental data and expanding our simulations to other locomotor activities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571823">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571823" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571823">
        <p class="paperTitle">Structure-conditioned masked language models for protein sequence design generalize beyond the native sequence space</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571823" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571823" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Akpinaroglu, D.; Seki, K.; Guo, A.; Zhu, E.; Kelly, M. J.; Kortemme, T.</p>
        <p class="info">Score: 18.1, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571823' target='https://doi.org/10.1101/2023.12.15.571823'> 10.1101/2023.12.15.571823</a></p>
        <p class="abstract">Machine learning has revolutionized computational protein design, enabling significant progress in protein backbone generation and sequence design. Here, we introduce Frame2seq, a structure-conditioned masked language model for protein sequence design. Frame2seq generates sequences in a single pass, achieves 49.1% sequence recovery on the CATH 4.2 test dataset, and accurately estimates the error in its own predictions, outperforming the autoregressive ProteinMPNN model with over six times faster inference. To probe the ability of Frame2seq to generate novel designs beyond the native-like sequence space it was trained on, we experimentally test 26 Frame2seq designs for de novo backbones with low identity to the starting sequences. We show that Frame2seq successfully designs soluble (22/26), monomeric, folded, and stable proteins (17/26), including a design with 0% sequence identity to native. The speed and accuracy of Frame2seq will accelerate exploration of novel sequence space across diverse design tasks, including challenging applications such as multi-objective optimization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572600">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572600" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572600">
        <p class="paperTitle">Allosteric Modulation of Fluorescence Revealed by Hydrogen Bond Dynamics in a Genetically Encoded Maltose Biosensor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572600" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572600" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: BERKSOZ, M.; ATILGAN, C.</p>
        <p class="info">Score: 3.9, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572600' target='https://doi.org/10.1101/2023.12.20.572600'> 10.1101/2023.12.20.572600</a></p>
        <p class="abstract">Genetically encoded fluorescent biosensors (GEFBs) proved to be reliable tracers for many metabolites and cellular processes. In the simplest case, a fluorescent protein (FP) is genetically fused to a sensing protein which undergoes a conformational change upon ligand binding. This drives a rearrangement in the chromophore environment and changes the spectral properties of the FP. Structural determinants of successful biosensors are revealed only in hindsight when the crystal structures of both ligand-bound and ligand-free forms are available. This makes the development of new biosensors for desired analytes a long trial-and-error process. In the current study, we conducted microsecond-long all atom molecular dynamics (MD) simulations of a maltose biosensor in both the apo (dark) and holo (bright) forms. We performed detailed hydrogen bond occupancy analyses to shed light on the mechanism of ligand induced conformational change in the sensor protein and its allosteric effect on the chromophore environment. We find that two strong indicators for distinguishing bright and dark states of biosensors are due to substantial changes in hydrogen bond dynamics in the system and solvent accessibility of the chromophore.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569229">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569229" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569229">
        <p class="paperTitle">Encapsulation of AAVs into protein vault nanoparticles as a novel solution to gene therapy&#39;s neutralizing antibody problem</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569229" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569229" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Collins, L. T.; Beatty, W.; Moyo, B.; Alves-Bezerra, M.; Hurley, A.; Lagor, W.; Bao, G.; Ponnazhagan, S.; McNally, R.; Rome, L.; Curiel, D.</p>
        <p class="info">Score: 16.2, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569229' target='https://doi.org/10.1101/2023.11.29.569229'> 10.1101/2023.11.29.569229</a></p>
        <p class="abstract">Although adeno-associated virus (AAV) has enjoyed enormous success as a delivery modality for gene therapy, it continues to suffer from the high prevalence of preexisting neutralizing antibodies in human populations, limiting who can receive potentially life-saving treatments. In this regard, AAV therapies generally also must be administered as a single dose since neutralizing antibodies develop in patients who receive the virus. Strategies for circumventing these issues remain limited. As a novel solution, we employed SpyTag-SpyCatcher molecular glue technology to facilitate packaging of AAVs inside of recombinant protein vault nanoparticles. Vaults are endogenous proteinaceous organelles produced by mammalian cells, so they are recognized as self by the immune system. They therefore can shield packaged molecules from neutralizing antibodies. Vault has previously been utilized to deliver drugs and proteins into cells, but our study represents the first time anyone has packaged an entire virus inside of vault. We showed that our vaultAAV (VAAV) delivery vehicle transduces cells in the presence of anti-AAV neutralizing serum. VAAV is positioned as a new gene therapy delivery platform with potential to overcome the neutralizing antibody problem and perhaps even allow administration of multiple doses, expanding the scope of AAV treatments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572973">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572973" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572973">
        <p class="paperTitle">Biodistribution of DNA-origami nanostructures in live zebrafish embryos with single-cell resolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572973" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572973" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kolonelou, C.; Bräutigam, L.; Edwards, S.; Engström, E.; Dias, J. M.; Spratt, J.; Karampelias, C.; Wennmalm, S.; Brismar, H.; Andersson, O.; Teixeira, A. I.</p>
        <p class="info">Score: 3.2, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572973' target='https://doi.org/10.1101/2023.12.22.572973'> 10.1101/2023.12.22.572973</a></p>
        <p class="abstract">DNA origami-based nanotechnology is a versatile tool for exploring fundamental biological questions and holds significant promise for future biomedical applications. However, the development of DNA origami-based therapeutic agents is hindered by the challenge of translating in vitro performance into effective applications in vivo. Here, we exploit the optical transparency of the embryonic zebrafish to track intravenously injected, fluorescently labelled wireframe DNA origami nanostructures. Our approach integrated long-term, high-resolution imaging of transgenic live embryos with single-cell RNA sequencing, to elucidate the biodistribution of DNA nanostructures over time, up to 3 days post-injection (dpi). Notably, we observed rapid accumulation of nanostructures in the caudal hematopoietic tissue (CHT), akin to the fetal liver in mammals. We tested the effects of coating the nanostructures with an oligolysine PEG copolymer (K-PEG), a widely used strategy to enhance their stability. The K-PEG coating mitigated the accumulation rate in CHT, enabling higher percentages of the nanostructures to engage with other tissues. Additionally, our findings highlighted the pivotal role of scavenger endothelial cells in DNA origami clearance, with K-PEG offering sustained protection for the nanostructures at the CHT. Furthermore, by monitoring DNA origami in a transgenic zebrafish line designed for targeted macrophage ablation, we found that macrophages contribute to nanostructure clearance at later time points. This study introduces a framework for the analyses of the biodistribution and clearance of DNA origami nanostructures in vivo with single cell resolution and establishes a foundation for the investigation of DNA origami-based nanomedicines in animal models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572135">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572135" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572135">
        <p class="paperTitle">Engineered Protein Nanosheets for the Scale up of Mesenchymal Stem Cell Culture on Bioemulsions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572135" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572135" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bosch-Fortea, M.; Nadal, C.; Chrysanthou, A.; Marciano, D.; Kanso, H.; Nguyen, N.; Gautrot, J.</p>
        <p class="info">Score: 2.6, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572135' target='https://doi.org/10.1101/2023.12.18.572135'> 10.1101/2023.12.18.572135</a></p>
        <p class="abstract">The rapid progress in cell therapies and stem cell technologies requires the development of novel bioprocessing and biomanufacturing pipelines able to cope with the scale up of cell manufacturing. In this respect, microdroplet technologies have already revolutionised the field of biotechnologies, but remain ill-suited to the culture of adherent cells. In this report, we describe the engineering of albumins with cell adhesive peptides for the stabilisation of microdroplets enabling the scale up of mesenchymal stem cell (MSC) expansion. We characterise the modified albumins prior to study their self-assembly at liquid-liquid interfaces via interfacial shear rheology, and mechanical strengthening through the formation of crosslinked nanosheets. The biofunctionalisation of these protein nanosheets is then characterised by fluorescence microscopy. In turn, the ability of the resulting bioactive microdroplets to promote rapid cell adhesion and expansion is examined and the extensive deposition of matrix associated with such cultures is characterised. The culture of MSCs is then scaled up 100 fold, first at the surface of fluorinated oil emulsions, then on plant-based emulsions stabilised by engineered protein nanosheets and the phenotype of resulting cells is characterised. The microdroplet culture system presented displays attractive advantages over existing technologies, in terms of simplicity of processing, compatibility with regulatory expectations and costs of production, and offers exciting opportunities for translation to cell manufacturing, for cell therapies and cultivated meat applications.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
