<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases
Authors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.
Score: 104.7, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557440
Genome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases
Authors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.
Score: 104.7, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557440
Genome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-27T10:39:11+00:00" />
<meta property="article:modified_time" content="2023-09-27T10:39:11+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases
Authors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.
Score: 104.7, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557440
Genome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases\nAuthors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.\nScore: 104.7, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557440\nGenome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e.",
  "keywords": [
    
  ],
  "articleBody": " Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases\nAuthors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.\nScore: 104.7, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557440\nGenome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e.g. CRISPR-Cas) to permit the installation of a range of edits including substitutions, insertions, and deletions. Click editors (CEs) leverage the \"click\"-like bioconjugation ability of HUH endonucleases (HUHes) with single stranded DNA substrates to covalently tether \"click DNA\" (clkDNA) templates encoding user-specifiable edits at targeted genomic loci. Through iterative optimization of the modular components of CEs (DNA polymerase and HUHe orthologs, architectural modifications, etc.) and their clkDNAs (template configurations, repair evading substitutions, etc.), we demonstrate the ability to install precise genome edits with minimal indels and no unwanted byproduct insertions. Since clkDNAs can be ordered as simple DNA oligonucleotides for cents per base, it is possible to screen many different clkDNA parameters rapidly and inexpensively to maximize edit efficiency. Together, click editing is a precise and highly versatile platform for modifying genomes with a simple workflow and broad utility across diverse biological applications.\nACE2-Coated Virus-Like Particles Effectively Block SARS-CoV-2 Infection\nAuthors: Bayraktar, C.; Kayabolen, A.; Odabas, A.; Durgun, A.; Kok, I.; Sevinc, K.; Supramaniam, A.; Idris, A.; Bagci-Onder, T.\nScore: 22.5, Published: 2023-09-20 DOI: 10.1101/2023.09.19.558424\nA large body of research accumulated over the past three years dedicated to our understanding and fighting COVID-19. Blocking the interaction between SARS-CoV-2 Spike and ACE2 receptor has been considered an effective strategy as anti-SARS-CoV-2 therapeutics. In this study, we developed ACE2-coated virus-like particles (ACE2-VLPs), which can be utilized to prevent viral entry into host cells and efficiently neutralize the virus. These ACE2-VLPs exhibited high neutralization capacity even when applied at low doses, and displayed superior efficacy compared to extracellular vesicles carrying ACE2, in the in vitro pseudoviral assays. ACE2-VLPs were stable under different environmental temperatures, and they were effective in blocking all tested variants of concern in vitro. Finally, ACE2-VLPs displayed marked neutralization capacity against Omicron BA.1 in the Vero E6 cells. Based on their superior efficacy compared to extracellular vesicles, and their demonstrated success against live virus, ACE2-VLPs can be considered as vital candidates for treating SARS-CoV-2. This novel therapeutic approach of VLP coating with receptor particles can serve as proof-of-concept for designing effective neutralization strategies for other viral diseases in the future. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=78 SRC=\"FIGDIR/small/558424v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (16K): org.highwire.dtl.DTLVardef@e36657org.highwire.dtl.DTLVardef@153e78borg.highwire.dtl.DTLVardef@1867b42org.highwire.dtl.DTLVardef@655eb_HPS_FORMAT_FIGEXP M_FIG In our study, we demonstrate the prevention of SARS-CoV-2 infection through the use of Ace2-coated VLPs. C_FIG\nComplete substitution with modified nucleotides suppresses the early interferon response and increases the potency of self-amplifying RNA\nAuthors: McGee, J. E.; Kirsch, J. R.; Kenney, D.; Chavez, E.; Shih, T.-Y.; Douam, F.; Wong, W. W.; Grinstaff, M. W.\nScore: 25.3, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557994\nSelf-amplifying RNA (saRNA) will revolutionize vaccines and in situ therapeutics by enabling protein expression for longer duration at lower doses. However, a major barrier to saRNA efficacy is the potent early interferon response triggered upon cellular entry, resulting in saRNA degradation and translational inhibition. Substitution of mRNA with modified nucleotides (modNTPs), such as N1-methylpseudouridine (N1m{Psi}), reduce the interferon response and enhance expression levels. Multiple attempts to use modNTPs in saRNA have been unsuccessful, leading to the conclusion that modNTPs are incompatible with saRNA, thus hindering further development. Here, contrary to the common dogma in the field, we identify multiple modNTPs that when incorporated into saRNA at 100% substitution confer immune evasion and enhance expression potency. Transfection efficiency enhances by roughly an order of magnitude in difficult to transfect cell types compared to unmodified saRNA, and interferon production reduces by \u003e8 fold compared to unmodified saRNA in human peripheral blood mononuclear cells (PBMCs). Furthermore, we demonstrate expression of viral antigens in vitro and observe significant protection against lethal challenge with a mouse-adapted SARS-CoV-2 strain in vivo. A modified saRNA vaccine, at 100-fold lower dose than a modified mRNA vaccine, results in a statistically improved performance to unmodified saRNA and statistically equivalent performance to modified mRNA. This discovery considerably broadens the potential scope of self-amplifying RNA, enabling entry into previously impossible cell types, as well as the potential to apply saRNA technology to non-vaccine modalities such as cell therapy and protein replacement.\nProtein generation with evolutionary diffusion: sequence is all you need\nAuthors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.\nScore: 186.5, Published: 2023-09-12 DOI: 10.1101/2023.09.11.556673\nDeep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design. However, current state-of-the-art models generate protein structures, which limits the scope of their training data and restricts generations to a small and biased subset of protein design space. Here, we introduce a general-purpose diffusion framework, EvoDiff, that combines evolutionary-scale data with the distinct conditioning capabilities of diffusion models for controllable protein generation in sequence space. EvoDiff generates high-fidelity, diverse, and structurally-plausible proteins that cover natural sequence and functional space. Critically, EvoDiff can generate proteins inaccessible to structure-based models, such as those with disordered regions, while maintaining the ability to design scaffolds for functional structural motifs, demonstrating the universality of our sequence-based formulation. We envision that EvoDiff will expand capabilities in protein engineering beyond the structure-function paradigm toward programmable, sequence-first design.\nHigh-affinity binding to the SARS-CoV-2 spike trimer by a nanostructured, trivalent protein-DNA synthetic antibody\nAuthors: Xu, Y.; Zheng, R.; Prasad, A.; Liu, M.; Wan, Z.; Zhou, X.; Porter, R. M.; Sample, M.; Poppleton, E.; Procyk, J.; Liu, H.; Li, Y.; Wang, S.; Yan, H.; Sulc, P.; Stephanopoulos, N.\nScore: 11.1, Published: 2023-09-19 DOI: 10.1101/2023.09.18.558353\nMultivalency enables nanostructures to bind molecular targets with high affinity. Although antibodies can be generated against a wide range of antigens, their shape and size cannot be tuned to match a given target. DNA nanotechnology provides an attractive approach for designing customized multivalent scaffolds due to the addressability and programmability of the nanostructure shape and size. Here, we design a nanoscale synthetic antibody (\"nano-synbody\") based on a three-helix bundle DNA nanostructure with one, two, or three identical arms terminating in a mini-binder protein that targets the SARS-CoV-2 spike protein. The nano-synbody was designed to match the valence and distance between the three receptor binding domains (RBDs) in the spike trimer, in order to enhance affinity. The protein-DNA nano-synbody shows tight binding to the wild-type, Delta, and several Omicron variants of the SARS-CoV-2 spike trimer, with affinity increasing as the number of arms increases from one to three. The effectiveness of the nano-synbody was also verified using a pseudovirus neutralization assay, with the three-arm nanostructure inhibiting two Omicron variants against which the structures with only one or two arms are ineffective. The structure of the three-arm nano-synbody bound to the Omicron variant spike trimer was solved by negative-stain transmission electron microscopy reconstruction, and shows the protein-DNA nanostructure with all three arms attached to the RBD domains, confirming the intended trivalent attachment. The ability to tune the size and shape of the nano-synbody, as well as its potential ability to attach two or more different binding ligands, will enable the high-affinity targeting of a range of proteins not possible with traditional antibodies.\nExtracellular matrix scaffold-assisted tumor vaccines induce tumor regression and long-term immune memory\nAuthors: Pal, S.; Chaudhari, R.; Baurceanu, I.; Hill, B. J.; Nagy, B. A.; Wolf, M.\nScore: 11.9, Published: 2023-09-15 DOI: 10.1101/2023.09.12.557449\nInjectable scaffold delivery is an immune engineering strategy to enhance the efficacy and reliability of cancer vaccine immunotherapy. The composition and structure of the biomaterial scaffold determines both vaccine release kinetics and inherent immune stimulation via the scaffold host response. Extracellular matrix (ECM) scaffolds prepared from decellularized tissues initiate an acute alternative inflammatory response following implantation, which facilitates wound healing following tumor resection and promotes local cancer immune surveillance. However, it remains unknown whether this environment is compatible with generating protective anti-tumor cytotoxic immunity with local immunotherapy delivery. Here, we engineered an ECM scaffold-assisted therapeutic cancer vaccine that maintained an immune microenvironment consistent with tissue reconstruction. Immune adjuvants MPLA, GM-CSF, and CDA were screened in a cancer vaccine formulated for decellularized small intestinal submucosa (SIS) ECM scaffold co-delivery. Though MPLA and GM-CSF showed the greatest increase in local myeloid cell infiltration, we found that the STING pathway adjuvant CDA was the most potent inducer of cytotoxic immunity with SIS-ECM scaffold delivery. Further, CDA did not diminish hallmark ECM immune responses needed in wound healing such as high Il4 cytokine expression. SIS scaffold delivery enhanced therapeutic vaccine efficacy using CDA and the antigen ovalbumin, curing greater than 50% of established EG.7 tumors in young mice and 75% in 24-week-old mature mice, compared to soluble components alone (0% cured). SIS-ECM scaffold assisted vaccination extended antigen exposure, was dependent on CD8+ cytotoxic T cells, and generated long term anti-tumor memory at least 7 months post-vaccination in both young and mature-aged mice. This study shows that an ECM scaffold is a promising delivery vehicle to enhance cancer vaccine efficacy while being orthogonal to characteristics of pro-healing immune hallmarks.\nOptimal Sampling Rate for 3D Single Molecule Localization\nAuthors: Chang, H.; Fu, S.; Li, Y.\nScore: 6.7, Published: 2023-09-21 DOI: 10.1101/2023.09.19.558416\nResolution of single molecule localization microscopy (SMLM) depends on the localization accuracy, which can be improved by utilizing engineered point spread functions (PSF) with delicate shapes. However, the intrinsic pixelation effect of the detector sensor will deteriorate PSFs under different sampling rates. The influence of the pixelation effect to the achieved 3D localization accuracy for different PSF shapes under different signal to background ratio (SBR) and pixel dependent readout noise has not been investigated in detail so far. In this work, we proposed a framework to characterize the 3D localization accuracy of pixelated PSF at different sampling rates. Four different PSFs (astigmatic PSF, double helix (DH) PSF, Tetrapod PSF and 4Pi PSF) were evaluated and the pixel size with optimal 3D localization performance were derived. This work provides a theoretical guide for the optimal design of sampling rate for 3D super resolution imaging.\nEngineering antigen-specific tolerance to an artificial protein hydrogel\nAuthors: Rapp, P. B.; Baccile, J. A.; Galimidi, R. P.; Vielmetter, J.\nScore: 6.1, Published: 2023-09-23 DOI: 10.1101/2023.09.23.559120\nArtificial protein hydrogels are an emerging class of biomaterials with numerous prospective applications in tissue engineering and regenerative medicine. These materials are likely to be immunogenic due their frequent incorporation of novel amino acid sequence domains, which often serve a functional role within the material itself. We engineered injectable ''self'' and ''non-self'' artificial protein hydrogels which were predicted to have divergent immune outcomes in vivo on the basis of their primary amino acid sequence. Following implantation in mouse, the non-self gels raised significantly higher anti-gel antibody titers than the corresponding self gels. Prophylactic administration of a fusion antibody targeting the non-self hydrogel epitopes to DEC-205, an endocytic receptor involved in Treg induction, fully suppressed the elevated antibody titer against the non-self gels. These results suggest that the clinical immune response to artificial protein biomaterials, including those that contain highly antigenic sequence domains, can be tuned through the induction of antigen-specific tolerance.\nPredicting Long and Short Duration Beta Bursts from Subthalamic Nucleus Local Field Potential Activity in Parkinson's Disease\nAuthors: Abdi-Sargezeh, B.; Shirani, S.; Sharma, A.; Starr, P.; Little, S.; Oswal, A.\nScore: 6.1, Published: 2023-09-23 DOI: 10.1101/2023.09.22.558764\nNeural activities within the beta frequency range (13-30 Hz) are not stationary, but occur in transient packets known as beta bursts. Parkinson's disease (PD) is characterized by the occurrence of beta bursts of increased duration and amplitude within the cortico-basal ganglia network. The pathophysiological importance of beta bursts is exemplified by the fact that they serve as a clinically useful feedback signal in beta amplitude triggered adaptive Deep Brain Stimulation (aDBS). Prolonged duration beta bursts are closely associated with motor impairments in PD, whilst bursts of shorter duration may have a physiological role. Consequently, we aimed to develop a deep learning-based pipeline capable of predicting long (\u003e150ms) and short (\u003c150ms) duration beta bursts from subthalamic nucleus local field potential (LFP) recordings. Our approach achieved promising accuracy values of 87% and 85.2% in two patients implanted with a DBS device that was capable of long-term wireless LFP sensing. Our findings highlight the feasibility of prolonged beta burst prediction and could inform the development of a new type of intelligent DBS approach with the capability of delivering stimulation only during the occurrence of prolonged bursts.\nA general approach for selection of epitope-directed binders to proteins\nAuthors: zhou, j.; Le, C. Q.; Zhang, Y.; Wells, J. A.\nScore: 7.5, Published: 2023-09-23 DOI: 10.1101/2022.10.24.513434\nDirecting antibodies to a particular epitope among many possible on a target protein is a significant challenge. Here we present a simple and general method for epitope-directed selection (EDS) using a differential phage selection strategy. This involves engineering the protein of interest (POI) with the epitope of interest (EOI) mutated using a systematic bioinformatics algorithm to guide the local design of an EOI decoy variant. Using several alternating rounds of negative selection with the EOI decoy variant followed by positive selection on the wild-type (WT) POI, we were able to identify highly specific and potent antibodies to five different EOI antigens that bind and functionally block known sites of proteolysis. Among these we developed highly specific antibodies that target the proteolytic site on the CUB domain containing protein 1 (CDCP1) to prevent its proteolysis allowing us to study the cellular maturation of this event that triggers malignancy. We generated antibodies that recognize the junction between the pro and catalytic domains for four different matrix metalloproteases (MMPs), such as MMP1, MMP3, and MMP9, that selectively block activation of each of these enzymes and impairs cell migration. We targeted a proteolytic epitope on the cell surface receptor, EPH Receptor A2, that is known to transform it from a tumor suppressor to an oncoprotein. We believe the EDS method greatly facilitates the generation antibodies to specific EOIs on a wide range of proteins and enzymes for broad therapeutic and diagnostic applications. SignificanceWe have developed a highly efficient platform to facilitate the directed selection in vitro of antibodies to a wide range of functional epitopes on proteins. This method uses a bioinformatic program to guide mutations in the local site of interest to create a decoy antigen that can effectively remove antibodies not binding the site of interest by negative selection, followed by positive selection with the WT antigen to identify antibodies to the epitope of interest. We demonstrate the generality and versatility of this method by successfully producing functional antibodies to block specific proteolytically sensitive epitopes on five different proteins including enzymes important in cancer. The epitope-directed selection (EDS) approach greatly facilitates the identification of binders to specific sites of interest on proteins to probe function and as potential immunotherapeutics.\n",
  "wordCount" : "2525",
  "inLanguage": "en",
  "datePublished": "2023-09-27T10:39:11Z",
  "dateModified": "2023-09-27T10:39:11Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on September 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557440">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557440" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557440">
        <p class="paperTitle">Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557440" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557440" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.</p>
        <p class="info">Score: 104.7, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557440' target='https://doi.org/10.1101/2023.09.12.557440'> 10.1101/2023.09.12.557440</a></p>
        <p class="abstract">Genome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e.g. CRISPR-Cas) to permit the installation of a range of edits including substitutions, insertions, and deletions. Click editors (CEs) leverage the &#34;click&#34;-like bioconjugation ability of HUH endonucleases (HUHes) with single stranded DNA substrates to covalently tether &#34;click DNA&#34; (clkDNA) templates encoding user-specifiable edits at targeted genomic loci. Through iterative optimization of the modular components of CEs (DNA polymerase and HUHe orthologs, architectural modifications, etc.) and their clkDNAs (template configurations, repair evading substitutions, etc.), we demonstrate the ability to install precise genome edits with minimal indels and no unwanted byproduct insertions. Since clkDNAs can be ordered as simple DNA oligonucleotides for cents per base, it is possible to screen many different clkDNA parameters rapidly and inexpensively to maximize edit efficiency. Together, click editing is a precise and highly versatile platform for modifying genomes with a simple workflow and broad utility across diverse biological applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558424">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558424" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558424">
        <p class="paperTitle">ACE2-Coated Virus-Like Particles Effectively Block SARS-CoV-2 Infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558424" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558424" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bayraktar, C.; Kayabolen, A.; Odabas, A.; Durgun, A.; Kok, I.; Sevinc, K.; Supramaniam, A.; Idris, A.; Bagci-Onder, T.</p>
        <p class="info">Score: 22.5, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558424' target='https://doi.org/10.1101/2023.09.19.558424'> 10.1101/2023.09.19.558424</a></p>
        <p class="abstract">A large body of research accumulated over the past three years dedicated to our understanding and fighting COVID-19. Blocking the interaction between SARS-CoV-2 Spike and ACE2 receptor has been considered an effective strategy as anti-SARS-CoV-2 therapeutics. In this study, we developed ACE2-coated virus-like particles (ACE2-VLPs), which can be utilized to prevent viral entry into host cells and efficiently neutralize the virus. These ACE2-VLPs exhibited high neutralization capacity even when applied at low doses, and displayed superior efficacy compared to extracellular vesicles carrying ACE2, in the in vitro pseudoviral assays. ACE2-VLPs were stable under different environmental temperatures, and they were effective in blocking all tested variants of concern in vitro. Finally, ACE2-VLPs displayed marked neutralization capacity against Omicron BA.1 in the Vero E6 cells. Based on their superior efficacy compared to extracellular vesicles, and their demonstrated success against live virus, ACE2-VLPs can be considered as vital candidates for treating SARS-CoV-2. This novel therapeutic approach of VLP coating with receptor particles can serve as proof-of-concept for designing effective neutralization strategies for other viral diseases in the future.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=78 SRC=&#34;FIGDIR/small/558424v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (16K):
org.highwire.dtl.DTLVardef@e36657org.highwire.dtl.DTLVardef@153e78borg.highwire.dtl.DTLVardef@1867b42org.highwire.dtl.DTLVardef@655eb_HPS_FORMAT_FIGEXP  M_FIG In our study, we demonstrate the prevention of SARS-CoV-2 infection through the use of Ace2-coated VLPs.

C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557994">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557994" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557994">
        <p class="paperTitle">Complete substitution with modified nucleotides suppresses the early interferon response and increases the potency of self-amplifying RNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557994" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557994" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McGee, J. E.; Kirsch, J. R.; Kenney, D.; Chavez, E.; Shih, T.-Y.; Douam, F.; Wong, W. W.; Grinstaff, M. W.</p>
        <p class="info">Score: 25.3, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557994' target='https://doi.org/10.1101/2023.09.15.557994'> 10.1101/2023.09.15.557994</a></p>
        <p class="abstract">Self-amplifying RNA (saRNA) will revolutionize vaccines and in situ therapeutics by enabling protein expression for longer duration at lower doses. However, a major barrier to saRNA efficacy is the potent early interferon response triggered upon cellular entry, resulting in saRNA degradation and translational inhibition. Substitution of mRNA with modified nucleotides (modNTPs), such as N1-methylpseudouridine (N1m{Psi}), reduce the interferon response and enhance expression levels. Multiple attempts to use modNTPs in saRNA have been unsuccessful, leading to the conclusion that modNTPs are incompatible with saRNA, thus hindering further development. Here, contrary to the common dogma in the field, we identify multiple modNTPs that when incorporated into saRNA at 100% substitution confer immune evasion and enhance expression potency. Transfection efficiency enhances by roughly an order of magnitude in difficult to transfect cell types compared to unmodified saRNA, and interferon production reduces by &gt;8 fold compared to unmodified saRNA in human peripheral blood mononuclear cells (PBMCs). Furthermore, we demonstrate expression of viral antigens in vitro and observe significant protection against lethal challenge with a mouse-adapted SARS-CoV-2 strain in vivo. A modified saRNA vaccine, at 100-fold lower dose than a modified mRNA vaccine, results in a statistically improved performance to unmodified saRNA and statistically equivalent performance to modified mRNA. This discovery considerably broadens the potential scope of self-amplifying RNA, enabling entry into previously impossible cell types, as well as the potential to apply saRNA technology to non-vaccine modalities such as cell therapy and protein replacement.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.556673">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.556673" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.556673">
        <p class="paperTitle">Protein generation with evolutionary diffusion: sequence is all you need</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.556673" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.556673" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.</p>
        <p class="info">Score: 186.5, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.556673' target='https://doi.org/10.1101/2023.09.11.556673'> 10.1101/2023.09.11.556673</a></p>
        <p class="abstract">Deep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design. However, current state-of-the-art models generate protein structures, which limits the scope of their training data and restricts generations to a small and biased subset of protein design space. Here, we introduce a general-purpose diffusion framework, EvoDiff, that combines evolutionary-scale data with the distinct conditioning capabilities of diffusion models for controllable protein generation in sequence space. EvoDiff generates high-fidelity, diverse, and structurally-plausible proteins that cover natural sequence and functional space. Critically, EvoDiff can generate proteins inaccessible to structure-based models, such as those with disordered regions, while maintaining the ability to design scaffolds for functional structural motifs, demonstrating the universality of our sequence-based formulation. We envision that EvoDiff will expand capabilities in protein engineering beyond the structure-function paradigm toward programmable, sequence-first design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558353">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558353" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558353">
        <p class="paperTitle">High-affinity binding to the SARS-CoV-2 spike trimer by a nanostructured, trivalent protein-DNA synthetic antibody</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558353" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558353" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, Y.; Zheng, R.; Prasad, A.; Liu, M.; Wan, Z.; Zhou, X.; Porter, R. M.; Sample, M.; Poppleton, E.; Procyk, J.; Liu, H.; Li, Y.; Wang, S.; Yan, H.; Sulc, P.; Stephanopoulos, N.</p>
        <p class="info">Score: 11.1, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558353' target='https://doi.org/10.1101/2023.09.18.558353'> 10.1101/2023.09.18.558353</a></p>
        <p class="abstract">Multivalency enables nanostructures to bind molecular targets with high affinity. Although antibodies can be generated against a wide range of antigens, their shape and size cannot be tuned to match a given target. DNA nanotechnology provides an attractive approach for designing customized multivalent scaffolds due to the addressability and programmability of the nanostructure shape and size. Here, we design a nanoscale synthetic antibody (&#34;nano-synbody&#34;) based on a three-helix bundle DNA nanostructure with one, two, or three identical arms terminating in a mini-binder protein that targets the SARS-CoV-2 spike protein. The nano-synbody was designed to match the valence and distance between the three receptor binding domains (RBDs) in the spike trimer, in order to enhance affinity. The protein-DNA nano-synbody shows tight binding to the wild-type, Delta, and several Omicron variants of the SARS-CoV-2 spike trimer, with affinity increasing as the number of arms increases from one to three. The effectiveness of the nano-synbody was also verified using a pseudovirus neutralization assay, with the three-arm nanostructure inhibiting two Omicron variants against which the structures with only one or two arms are ineffective. The structure of the three-arm nano-synbody bound to the Omicron variant spike trimer was solved by negative-stain transmission electron microscopy reconstruction, and shows the protein-DNA nanostructure with all three arms attached to the RBD domains, confirming the intended trivalent attachment. The ability to tune the size and shape of the nano-synbody, as well as its potential ability to attach two or more different binding ligands, will enable the high-affinity targeting of a range of proteins not possible with traditional antibodies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557449">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557449" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557449">
        <p class="paperTitle">Extracellular matrix scaffold-assisted tumor vaccines induce tumor regression and long-term immune memory</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557449" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557449" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pal, S.; Chaudhari, R.; Baurceanu, I.; Hill, B. J.; Nagy, B. A.; Wolf, M.</p>
        <p class="info">Score: 11.9, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557449' target='https://doi.org/10.1101/2023.09.12.557449'> 10.1101/2023.09.12.557449</a></p>
        <p class="abstract">Injectable scaffold delivery is an immune engineering strategy to enhance the efficacy and reliability of cancer vaccine immunotherapy. The composition and structure of the biomaterial scaffold determines both vaccine release kinetics and inherent immune stimulation via the scaffold host response. Extracellular matrix (ECM) scaffolds prepared from decellularized tissues initiate an acute alternative inflammatory response following implantation, which facilitates wound healing following tumor resection and promotes local cancer immune surveillance. However, it remains unknown whether this environment is compatible with generating protective anti-tumor cytotoxic immunity with local immunotherapy delivery. Here, we engineered an ECM scaffold-assisted therapeutic cancer vaccine that maintained an immune microenvironment consistent with tissue reconstruction. Immune adjuvants MPLA, GM-CSF, and CDA were screened in a cancer vaccine formulated for decellularized small intestinal submucosa (SIS) ECM scaffold co-delivery. Though MPLA and GM-CSF showed the greatest increase in local myeloid cell infiltration, we found that the STING pathway adjuvant CDA was the most potent inducer of cytotoxic immunity with SIS-ECM scaffold delivery. Further, CDA did not diminish hallmark ECM immune responses needed in wound healing such as high Il4 cytokine expression. SIS scaffold delivery enhanced therapeutic vaccine efficacy using CDA and the antigen ovalbumin, curing greater than 50% of established EG.7 tumors in young mice and 75% in 24-week-old mature mice, compared to soluble components alone (0% cured). SIS-ECM scaffold assisted vaccination extended antigen exposure, was dependent on CD8&#43; cytotoxic T cells, and generated long term anti-tumor memory at least 7 months post-vaccination in both young and mature-aged mice. This study shows that an ECM scaffold is a promising delivery vehicle to enhance cancer vaccine efficacy while being orthogonal to characteristics of pro-healing immune hallmarks.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558416">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558416" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558416">
        <p class="paperTitle">Optimal Sampling Rate for 3D Single Molecule Localization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558416" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558416" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chang, H.; Fu, S.; Li, Y.</p>
        <p class="info">Score: 6.7, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558416' target='https://doi.org/10.1101/2023.09.19.558416'> 10.1101/2023.09.19.558416</a></p>
        <p class="abstract">Resolution of single molecule localization microscopy (SMLM) depends on the localization accuracy, which can be improved by utilizing engineered point spread functions (PSF) with delicate shapes. However, the intrinsic pixelation effect of the detector sensor will deteriorate PSFs under different sampling rates. The influence of the pixelation effect to the achieved 3D localization accuracy for different PSF shapes under different signal to background ratio (SBR) and pixel dependent readout noise has not been investigated in detail so far. In this work, we proposed a framework to characterize the 3D localization accuracy of pixelated PSF at different sampling rates. Four different PSFs (astigmatic PSF, double helix (DH) PSF, Tetrapod PSF and 4Pi PSF) were evaluated and the pixel size with optimal 3D localization performance were derived. This work provides a theoretical guide for the optimal design of sampling rate for 3D super resolution imaging.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.23.559120">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.23.559120" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.23.559120">
        <p class="paperTitle">Engineering antigen-specific tolerance to an artificial protein hydrogel</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.23.559120" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.23.559120" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rapp, P. B.; Baccile, J. A.; Galimidi, R. P.; Vielmetter, J.</p>
        <p class="info">Score: 6.1, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.23.559120' target='https://doi.org/10.1101/2023.09.23.559120'> 10.1101/2023.09.23.559120</a></p>
        <p class="abstract">Artificial protein hydrogels are an emerging class of biomaterials with numerous prospective applications in tissue engineering and regenerative medicine. These materials are likely to be immunogenic due their frequent incorporation of novel amino acid sequence domains, which often serve a functional role within the material itself. We engineered injectable &#39;&#39;self&#39;&#39; and &#39;&#39;non-self&#39;&#39; artificial protein hydrogels which were predicted to have divergent immune outcomes in vivo on the basis of their primary amino acid sequence. Following implantation in mouse, the non-self gels raised significantly higher anti-gel antibody titers than the corresponding self gels. Prophylactic administration of a fusion antibody targeting the non-self hydrogel epitopes to DEC-205, an endocytic receptor involved in Treg induction, fully suppressed the elevated antibody titer against the non-self gels. These results suggest that the clinical immune response to artificial protein biomaterials, including those that contain highly antigenic sequence domains, can be tuned through the induction of antigen-specific tolerance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.558764">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.558764" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.558764">
        <p class="paperTitle">Predicting Long and Short Duration Beta Bursts from Subthalamic Nucleus Local Field Potential Activity in Parkinson&#39;s Disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.558764" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.558764" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abdi-Sargezeh, B.; Shirani, S.; Sharma, A.; Starr, P.; Little, S.; Oswal, A.</p>
        <p class="info">Score: 6.1, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.558764' target='https://doi.org/10.1101/2023.09.22.558764'> 10.1101/2023.09.22.558764</a></p>
        <p class="abstract">Neural activities within the beta frequency range (13-30 Hz) are not stationary, but occur in transient packets known as beta bursts. Parkinson&#39;s disease (PD) is characterized by the occurrence of beta bursts of increased duration and amplitude within the cortico-basal ganglia network. The pathophysiological importance of beta bursts is exemplified by the fact that they serve as a clinically useful feedback signal in beta amplitude triggered adaptive Deep Brain Stimulation (aDBS). Prolonged duration beta bursts are closely associated with motor impairments in PD, whilst bursts of shorter duration may have a physiological role. Consequently, we aimed to develop a deep learning-based pipeline capable of predicting long (&gt;150ms) and short (&lt;150ms) duration beta bursts from subthalamic nucleus local field potential (LFP) recordings. Our approach achieved promising accuracy values of 87% and 85.2% in two patients implanted with a DBS device that was capable of long-term wireless LFP sensing. Our findings highlight the feasibility of prolonged beta burst prediction and could inform the development of a new type of intelligent DBS approach with the capability of delivering stimulation only during the occurrence of prolonged bursts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.10.24.513434">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.10.24.513434" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.10.24.513434">
        <p class="paperTitle">A general approach for selection of epitope-directed binders to proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.10.24.513434" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.10.24.513434" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: zhou, j.; Le, C. Q.; Zhang, Y.; Wells, J. A.</p>
        <p class="info">Score: 7.5, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.10.24.513434' target='https://doi.org/10.1101/2022.10.24.513434'> 10.1101/2022.10.24.513434</a></p>
        <p class="abstract">Directing antibodies to a particular epitope among many possible on a target protein is a significant challenge. Here we present a simple and general method for epitope-directed selection (EDS) using a differential phage selection strategy. This involves engineering the protein of interest (POI) with the epitope of interest (EOI) mutated using a systematic bioinformatics algorithm to guide the local design of an EOI decoy variant. Using several alternating rounds of negative selection with the EOI decoy variant followed by positive selection on the wild-type (WT) POI, we were able to identify highly specific and potent antibodies to five different EOI antigens that bind and functionally block known sites of proteolysis. Among these we developed highly specific antibodies that target the proteolytic site on the CUB domain containing protein 1 (CDCP1) to prevent its proteolysis allowing us to study the cellular maturation of this event that triggers malignancy. We generated antibodies that recognize the junction between the pro and catalytic domains for four different matrix metalloproteases (MMPs), such as MMP1, MMP3, and MMP9, that selectively block activation of each of these enzymes and impairs cell migration. We targeted a proteolytic epitope on the cell surface receptor, EPH Receptor A2, that is known to transform it from a tumor suppressor to an oncoprotein. We believe the EDS method greatly facilitates the generation antibodies to specific EOIs on a wide range of proteins and enzymes for broad therapeutic and diagnostic applications.

SignificanceWe have developed a highly efficient platform to facilitate the directed selection in vitro of antibodies to a wide range of functional epitopes on proteins. This method uses a bioinformatic program to guide mutations in the local site of interest to create a decoy antigen that can effectively remove antibodies not binding the site of interest by negative selection, followed by positive selection with the WT antigen to identify antibodies to the epitope of interest. We demonstrate the generality and versatility of this method by successfully producing functional antibodies to block specific proteolytically sensitive epitopes on five different proteins including enzymes important in cancer. The epitope-directed selection (EDS) approach greatly facilitates the identification of binders to specific sites of interest on proteins to probe function and as potential immunotherapeutics.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
