<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Massively parallel sample preparation for multiplexed single-cell proteomics using nPOP
Authors: Leduc, A.; Koury, L.; Cantlon, J.; Slavov, N.
Score: 38.4, Published: 2023-11-28 DOI: 10.1101/2023.11.27.568927
Single-cell proteomics by mass spectrometry (MS) allows quantifying proteins with high specificity and sensitivity. To increase its throughput, we developed nPOP, a method for parallel preparation of thousands of single cells in nanoliter volume droplets deposited on glass slides. Here, we describe its protocol with emphasis on its flexibility to prepare samples for different multiplexed MS methods.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Massively parallel sample preparation for multiplexed single-cell proteomics using nPOP
Authors: Leduc, A.; Koury, L.; Cantlon, J.; Slavov, N.
Score: 38.4, Published: 2023-11-28 DOI: 10.1101/2023.11.27.568927
Single-cell proteomics by mass spectrometry (MS) allows quantifying proteins with high specificity and sensitivity. To increase its throughput, we developed nPOP, a method for parallel preparation of thousands of single cells in nanoliter volume droplets deposited on glass slides. Here, we describe its protocol with emphasis on its flexibility to prepare samples for different multiplexed MS methods." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-13T10:39:57+00:00" />
<meta property="article:modified_time" content="2023-12-13T10:39:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Massively parallel sample preparation for multiplexed single-cell proteomics using nPOP
Authors: Leduc, A.; Koury, L.; Cantlon, J.; Slavov, N.
Score: 38.4, Published: 2023-11-28 DOI: 10.1101/2023.11.27.568927
Single-cell proteomics by mass spectrometry (MS) allows quantifying proteins with high specificity and sensitivity. To increase its throughput, we developed nPOP, a method for parallel preparation of thousands of single cells in nanoliter volume droplets deposited on glass slides. Here, we describe its protocol with emphasis on its flexibility to prepare samples for different multiplexed MS methods."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Massively parallel sample preparation for multiplexed single-cell proteomics using nPOP\nAuthors: Leduc, A.; Koury, L.; Cantlon, J.; Slavov, N.\nScore: 38.4, Published: 2023-11-28 DOI: 10.1101/2023.11.27.568927\nSingle-cell proteomics by mass spectrometry (MS) allows quantifying proteins with high specificity and sensitivity. To increase its throughput, we developed nPOP, a method for parallel preparation of thousands of single cells in nanoliter volume droplets deposited on glass slides. Here, we describe its protocol with emphasis on its flexibility to prepare samples for different multiplexed MS methods.",
  "keywords": [
    
  ],
  "articleBody": " Massively parallel sample preparation for multiplexed single-cell proteomics using nPOP\nAuthors: Leduc, A.; Koury, L.; Cantlon, J.; Slavov, N.\nScore: 38.4, Published: 2023-11-28 DOI: 10.1101/2023.11.27.568927\nSingle-cell proteomics by mass spectrometry (MS) allows quantifying proteins with high specificity and sensitivity. To increase its throughput, we developed nPOP, a method for parallel preparation of thousands of single cells in nanoliter volume droplets deposited on glass slides. Here, we describe its protocol with emphasis on its flexibility to prepare samples for different multiplexed MS methods. An implementation with plexDIA demonstrates accurate quantification of about 3,000 - 3,700 proteins per human cell. The protocol is implemented on the CellenONE instrument and uses readily available consumables, which should facilitate broad adoption. nPOP can be applied to all samples that can be processed to a single-cell suspension. It takes 1 or 2 days to prepare over 3,000 single cells. We provide metrics and software for quality control that can support the robust scaling of nPOP to higher plex reagents for achieving reliable high-throughput single-cell protein analysis. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=48 SRC=\"FIGDIR/small/568927v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (33K): org.highwire.dtl.DTLVardef@c82330org.highwire.dtl.DTLVardef@17e5f2forg.highwire.dtl.DTLVardef@19b200borg.highwire.dtl.DTLVardef@154e336_HPS_FORMAT_FIGEXP M_FIG C_FIG\nDevelopment of a Nanoparticle-Based Tendon-Targeting Drug Delivery System to Pharmacologically Modulate Tendon Healing\nAuthors: Adjei-Sowah, E. E.; Chandrasiri, I.; Xiao, B.; Liu, Y.; Ackerman, J.; Soto, C.; Nichols, A. E.; Nolan, K.; Benoit, D. S. W.; Loiselle, A. E.\nScore: 9.4, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569204\nTendon regeneration following acute injury is marred by a fibrotic healing response that prevents complete functional recovery. Despite the high frequency of tendon injuries and the poor outcomes, including functional deficits and elevated risk of re-injury, there are currently no pharmacological therapies in clinical use to enhance the healing process. Several promising pharmacotherapies have been identified; however, systemic treatments lack tendon specificity, resulting in poor tendon biodistribution and perhaps explaining the largely limited beneficial effects of these treatments on the tendon healing process. To address this major unmet need, we leveraged our existing spatial transcriptomics dataset of the tendon healing process to identify an area of the healing tendon that is enriched for expression of Acp5. Acp5 encodes tartrate-resistant acid phosphatase (TRAP), and we demonstrate robust TRAP activity in the healing tendon. This unexpected finding allowed us to refine and apply our existing TRAP binding peptide (TBP) functionalized nanoparticle (NP) drug delivery system (DDS) to facilitate improved delivery of systemic treatments to the healing tendon. To demonstrate the translational potential of this drug delivery system, we delivered the S100a4 inhibitor, Niclosamide to the healing tendon. We have previously shown that genetic knockdown of S100a4 enhances tendon healing. While systemic delivery of Niclosamide did not affect the healing process, relative to controls, TBP-NP delivery of Niclosamide enhanced both functional and mechanical outcome measures. Collectively, these data identify a novel tendon-targeting drug delivery system and demonstrate the translational potential of this approach to enhance the tendon healing process.\nBiodegradable silica nanoparticles for efficient linear DNA gene delivery\nAuthors: Ramos-Valle, A.; Kirst, H.; Fanarraga, M. L.\nScore: 5.9, Published: 2023-12-05 DOI: 10.1101/2023.12.05.569925\nTargeting, safety, scalability, and storage stability of vectors are still challenges in the field of nucleic acid delivery for gene therapy. Silica-based nanoparticles have been widely studied as gene carriers, exhibiting key features such as biocompatibility, simplistic synthesis and enabling easy surface modifications for targeting. However, the ability of the formulation to incorporate DNA is limited, which restricts the number of DNA molecules that can be incorporated into the particle, thereby reducing gene expression. Here we use polymerase chain reaction (PCR)-generated linear DNA molecules to augment the coding sequences of gene-carrying nanoparticles, thereby maximizing nucleic acid loading and minimizing the size of these nanocarriers. This approach results in a remarkable 16-fold increase in protein expression six days post-transfection in cells transfected with particles carrying the linear DNA compared with particles bearing circular plasmid DNA. The study also showed that the use of linear DNA entrapped in DNA@SiO2 resulted in a much more efficient level of gene expression compared to standard transfection reagents. The system developed in this study features simplicity, scalability, and increased transfection efficiency and gene expression over existing approaches, enabled by improved embedment capabilities for linear DNA, compared to conventional methods such as lipids or polymers, which generally show greater transfection efficiency with plasmid DNA. Therefore, this novel methodology can find applications not only in gene therapy but also in research settings for high throughput gene expression screenings.\nA Ratiometric Nonfluorescent CRISPR Assay Utilizing Cas12a-Induced Plasmid Supercoil Relaxation\nAuthors: Mohammad, N.; Talton, L.; Dalgan, S.; Wei, Q.\nScore: 5.3, Published: 2023-12-07 DOI: 10.1101/2023.12.04.570032\nMost CRISPR-based biosensors rely on labeled reporter molecules and expensive equipment for signal readout. A recent alternative approach quantifies analyte concentration by sizing native {lambda} DNA reporters, providing a simple and label-free solution for ultrasensitive detection. However, this method faces challenges in accurately quantifying size reduction of long DNA reporters via gel electrophoresis due to the sensitivity of DNA band shift to other interferences such as gel distortion. To overcome these limitations, here we developed a simple and robust ratiometric signaling strategy using CRISPR-Cas12a-induced supercoil relaxation of dsDNA plasmid reporters. In the presence of target, we observed that the fraction of supercoiled plasmid DNA decreased, and the amount of relaxed conformation (circular) increased over time. The relative percentage of supercoiled DNA to the relaxed circular DNA was analyzed by gel electrophoresis to generate an intensity-based ratiometric signal for more accurate target concentration quantification. This simple and inexpensive method is [~]100 times more sensitive when compared with the typical fluorescent reporter system. This self-referenced strategy solves the potential application limitations of previously demonstrated DNA sizing-based CRISPR-Dx without compromising the sensitivity. Finally, we demonstrated the applicability of ratiometric sensing strategy using model DNA targets such as AAV and HPV 16, highlighting its feasibility for point-of-use CRISPR-Dx applications. Table of Contents (TOC) O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=117 SRC=\"FIGDIR/small/570032v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (41K): org.highwire.dtl.DTLVardef@2ac1ccorg.highwire.dtl.DTLVardef@1195d17org.highwire.dtl.DTLVardef@2ec9fcorg.highwire.dtl.DTLVardef@1cef5da_HPS_FORMAT_FIGEXP M_FIG C_FIG\nDemocratizing biomedical simulation through automated model discovery and a universal material subroutine\nAuthors: Peirlinck, M.; Linka, K.; Hurtado, J. A.; Holzapfel, G. A.; Kuhl, E.\nScore: 4.7, Published: 2023-12-08 DOI: 10.1101/2023.12.06.570487\nPersonalized computational simulations have emerged as a vital tool to understand the biomechanical factors of a disease, predict disease progression, and design personalized intervention. Material modeling is critical for realistic biomedical simulations, and poor model selection can have life-threatening consequences for the patient. However, selecting the best model requires a profound domain knowledge and is limited to a few highly specialized experts in the field. Here we explore the feasibility of eliminating user involvement and automate the process of material modeling in finite element analyses. We leverage recent developments in constitutive neural networks, machine learning, and artificial intelligence to discover the best constitutive model from thousands of possible combinations of a few functional building blocks. We integrate all discoverable models into the finite element workflow by creating a universal material subroutine that contains more than 60,000 models, made up of 16 individual terms. We prototype this workflow using biaxial extension tests from healthy human arteries as input and stress and stretch profiles across the human aortic arch as output. Our results suggest that constitutive neural networks can robustly discover various flavors of arterial models from data, feed these models directly into a finite element simulation, and predict stress and strain profiles that compare favorably to the classical Holzapfel model. Replacing dozens of individual material subroutines by a single universal material subroutine-populated directly via automated model discovery-will make finite element simulations more user-friendly, more robust, and less vulnerable to human error. Democratizing finite element simulation by automating model selection could induce a paradigm shift in physics-based modeling, broaden access to simulation technologies, and empower individuals with varying levels of expertise and diverse backgrounds to actively participate in scientific discovery and push the boundaries of biomedical simulation.\nNoninvasive Injectable Optical Nanosensor-Hydrogel Hybrids Detect Doxorubicin in Living Mice\nAuthors: Cohen, Z.; Alpert, D. J.; Weisel, A. C.; Roach, A.; Rahman, S.; Gaikwad, P. V.; Nicoll, S. B.; Williams, R. M.\nScore: 3.8, Published: 2023-12-07 DOI: 10.1101/2023.12.05.570137\nWhile the tissue-transparent fluorescence of single-walled carbon nanotubes (SWCNTs) imparts substantial potential for use in non-invasive biosensors, development of non-invasive systems is yet to be realized. Here, we investigated the functionality of a SWCNT-based nanosensor in several injectable SWCNT-hydrogel systems, ultimately finding SWCNT encapsulation in a sulfonated methylcellulose hydrogel optimal for detection of ions, small molecules, and proteins. We found that the hydrogel system and nanosensor signal were stable for several weeks in live mice. We then found that this system successfully detects local injections of the chemotherapeutic agent doxorubicin in mice. We anticipate future studies to adapt this device for detection of other analytes in animals and, ultimately, patients.\nYoung adults accelerate their arms significantly faster and earlier than old adults resulting in improved center of mass dynamics during an overground slip perturbation\nAuthors: Lee-Confer, J.; Lo, M. K.; Troy, K.\nScore: 3.7, Published: 2023-12-10 DOI: 10.1101/2023.12.09.570848\nArm motions reduce the likelihood of falling from a slip in young adults, and injuries to older adults from falls are known to occur from a frontal plane fall. Therefore, the purpose of this study was to quantify the frontal plane trunk and arm motion in older and younger adults during a slip. Eleven older adults (age: 72.0 +/- 5.0) and 11 younger adults (age: 25.17 +/- 5.94) were subjected to a slip during walking. 27% of the older adults and 11% of the younger adults experienced a fall from the slip incident. The arm excursion did not significantly differ between groups, however younger adults average peak arm abduction occurred 310 ms prior to older adults (p = 0.03). There were no significant differences in peak trunk between groups. The arm abduction acceleration was significantly higher in younger adults compared to older adults (3593.2 +/- 1144.8 vs. 2309.8 +/- 1428.5 degrees/s2, p = 0.03). Furthermore, young adults exhibited a significantly reduced CoM excursion compared to old adults (4.55 +/- 3.45 vs. 10.27 +/- 6.62 cm, p \u003c 0.01). Young adults may have exhibited lower fall rates due to the significantly higher acceleration of their arms compared to old adults.\nEngineering mice for female-biased progeny without impacting genetic integrity and litter size\nAuthors: Yosef, I.; Mahata, T.; Chen, Y.; Bar-Joseph, H.; Shalgi, R.; Munitz, A.; Gerlic, M.; Qimron, U.\nScore: 43.8, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568055\nGenerating mammalian gametes with a skewed sex ratio has thus far eluded empirical confirmation. The utilization of such genetically engineered organisms would offer the potential to curtail the necessity for culling animals of undesirable sex, mitigate resource wastage, and alleviate superfluous labor burdens. In this study, we introduce a transgenic male mouse lineage, which consistently yields predominantly female progeny (comprising up to 91% of the total offspring). This accomplishment was made possible by integrating a controllable genetic cassette onto the Y chromosome. The cassette encodes dCas9 and RNA guides that selectively silence a spermatid maturation gene. After the separation of X and Y gametes during meiosis, gametes containing an X chromosome develop normally, while those harboring the engineered Y chromosome, subjected to dCas9 silencing of the spermatid maturation gene, do not mature properly. Indeed, some spermatozoa from the transgenic mice exhibit a unique morphology, associated with the absence of the maturation gene. Notably, the resultant female offspring do not inherit the genetically engineered Y chromosome and are thus not genetically modified. Importantly, the litter size of the transgenic mice remains unchanged compared to the wild type. These findings represent a groundbreaking demonstration of genetic engineerings potential to yield sex-biased litters of full size without compromising genetic integrity, marking a pioneering advancement in this field of study. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=139 SRC=\"FIGDIR/small/568055v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (28K): org.highwire.dtl.DTLVardef@18f9ccborg.highwire.dtl.DTLVardef@17f0577org.highwire.dtl.DTLVardef@3fd5bborg.highwire.dtl.DTLVardef@e900bc_HPS_FORMAT_FIGEXP M_FIG Graphical Abstract A proof of concept for the first mammal producing sexed semen may revolutionize the way for breeding animals such as cows for the dairy industry and chickens for the eggs industry. C_FIG\nA high-fidelity CRISPR-Cas13 system improves abnormalities associated with C9ORF72-linked ALS/FTD\nAuthors: McCallister, T. X.; Lim, C. K. W.; Terpstra, W. M.; C., M. A. Z.; Zhang, S.; Powell, J. E.; Gaj, T.\nScore: 3.0, Published: 2023-12-12 DOI: 10.1101/2023.12.12.571328\nAn abnormal expansion of a GGGGCC hexanucleotide repeat in the C9ORF72 gene is the most common genetic cause of amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD), two debilitating neurodegenerative disorders driven in part by gain-of-function mechanisms involving transcribed forms of the repeat expansion. By utilizing a Cas13 variant with reduced collateral effects, we developed a high-fidelity RNA-targeting CRISPR-based system for C9ORF72-linked ALS/FTD. When delivered to the brain of a transgenic rodent model, this Cas13-based platform effectively curbed the expression of the GGGGCC repeat-containing RNA without affecting normal C9ORF72 levels, which in turn decreased the formation of RNA foci and reversed transcriptional deficits. This high-fidelity Cas13 variant possessed improved transcriptome-wide specificity compared to its native form and mediated efficient targeting in motor neuron-like cells derived from a patient with ALS. Our results lay the foundation for the implementation of RNA-targeting CRISPR technologies for C9ORF72-linked ALS/FTD.\nHighly efficient transgenic mouse production using piggyBac and its application to rapid phenotyping at the founder generation\nAuthors: Okamura, E.; Mizuno, S.; Matsumoto, S.; Murata, K.; Tanimoto, Y.; Tra, D. T. H.; Suzuki, H.; Kang, W.; Ema, T.; Morimoto, K.; Kato, K.; Matsumoto, T.; Masuyama, N.; Kijima, Y.; Morimura, T.; Sugiyama, F.; Takahashi, S.; Mizutani, E.; Woltjen, K.; Yachie, N.; Ema, M.\nScore: 2.5, Published: 2023-12-10 DOI: 10.1101/2023.12.10.570953\nPronuclear microinjection is the most popular method for producing transgenic (Tg) animals. Because the production efficiency is typically less than 20%, phenotypic characterization of Tg animals is generally performed on the next generation (F1) onwards. However, apart from in rodents, in many animal species with long generation times, it is desirable to perform phenotyping in the founder (F0) generation. In this study, we attempted to optimize a method of Tg mouse production to achieve higher Tg production efficiency using piggyBac transposon systems and established optimal conditions under which almost all individuals in the F0 generation were Tg. We also succeeded in generating bacterial artificial chromosome Tg mice with efficiency of approximately 70%. By combining this method with genome editing technology, we established a new strategy to perform phenotyping of mice with tissue-specific knockout using the F0 generation. Taking the obtained findings together, by using this method, experimental research using Tg animals can be carried out more efficiently.\n",
  "wordCount" : "2383",
  "inLanguage": "en",
  "datePublished": "2023-12-13T10:39:57Z",
  "dateModified": "2023-12-13T10:39:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 13, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568927">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568927" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568927">
        <p class="paperTitle">Massively parallel sample preparation for multiplexed single-cell proteomics using nPOP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568927" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568927" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Leduc, A.; Koury, L.; Cantlon, J.; Slavov, N.</p>
        <p class="info">Score: 38.4, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568927' target='https://doi.org/10.1101/2023.11.27.568927'> 10.1101/2023.11.27.568927</a></p>
        <p class="abstract">Single-cell proteomics by mass spectrometry (MS) allows quantifying proteins with high specificity and sensitivity. To increase its throughput, we developed nPOP, a method for parallel preparation of thousands of single cells in nanoliter volume droplets deposited on glass slides. Here, we describe its protocol with emphasis on its flexibility to prepare samples for different multiplexed MS methods. An implementation with plexDIA demonstrates accurate quantification of about 3,000 - 3,700 proteins per human cell. The protocol is implemented on the CellenONE instrument and uses readily available consumables, which should facilitate broad adoption. nPOP can be applied to all samples that can be processed to a single-cell suspension. It takes 1 or 2 days to prepare over 3,000 single cells. We provide metrics and software for quality control that can support the robust scaling of nPOP to higher plex reagents for achieving reliable high-throughput single-cell protein analysis.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=48 SRC=&#34;FIGDIR/small/568927v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (33K):
org.highwire.dtl.DTLVardef@c82330org.highwire.dtl.DTLVardef@17e5f2forg.highwire.dtl.DTLVardef@19b200borg.highwire.dtl.DTLVardef@154e336_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569204">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569204" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569204">
        <p class="paperTitle">Development of a Nanoparticle-Based Tendon-Targeting Drug Delivery System to Pharmacologically Modulate Tendon Healing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569204" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569204" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Adjei-Sowah, E. E.; Chandrasiri, I.; Xiao, B.; Liu, Y.; Ackerman, J.; Soto, C.; Nichols, A. E.; Nolan, K.; Benoit, D. S. W.; Loiselle, A. E.</p>
        <p class="info">Score: 9.4, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569204' target='https://doi.org/10.1101/2023.11.29.569204'> 10.1101/2023.11.29.569204</a></p>
        <p class="abstract">Tendon regeneration following acute injury is marred by a fibrotic healing response that prevents complete functional recovery. Despite the high frequency of tendon injuries and the poor outcomes, including functional deficits and elevated risk of re-injury, there are currently no pharmacological therapies in clinical use to enhance the healing process. Several promising pharmacotherapies have been identified; however, systemic treatments lack tendon specificity, resulting in poor tendon biodistribution and perhaps explaining the largely limited beneficial effects of these treatments on the tendon healing process. To address this major unmet need, we leveraged our existing spatial transcriptomics dataset of the tendon healing process to identify an area of the healing tendon that is enriched for expression of Acp5. Acp5 encodes tartrate-resistant acid phosphatase (TRAP), and we demonstrate robust TRAP activity in the healing tendon. This unexpected finding allowed us to refine and apply our existing TRAP binding peptide (TBP) functionalized nanoparticle (NP) drug delivery system (DDS) to facilitate improved delivery of systemic treatments to the healing tendon. To demonstrate the translational potential of this drug delivery system, we delivered the S100a4 inhibitor, Niclosamide to the healing tendon. We have previously shown that genetic knockdown of S100a4 enhances tendon healing. While systemic delivery of Niclosamide did not affect the healing process, relative to controls, TBP-NP delivery of Niclosamide enhanced both functional and mechanical outcome measures. Collectively, these data identify a novel tendon-targeting drug delivery system and demonstrate the translational potential of this approach to enhance the tendon healing process.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.569925">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.569925" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.569925">
        <p class="paperTitle">Biodegradable silica nanoparticles for efficient linear DNA gene delivery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.569925" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.569925" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ramos-Valle, A.; Kirst, H.; Fanarraga, M. L.</p>
        <p class="info">Score: 5.9, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.569925' target='https://doi.org/10.1101/2023.12.05.569925'> 10.1101/2023.12.05.569925</a></p>
        <p class="abstract">Targeting, safety, scalability, and storage stability of vectors are still challenges in the field of nucleic acid delivery for gene therapy. Silica-based nanoparticles have been widely studied as gene carriers, exhibiting key features such as biocompatibility, simplistic synthesis and enabling easy surface modifications for targeting. However, the ability of the formulation to incorporate DNA is limited, which restricts the number of DNA molecules that can be incorporated into the particle, thereby reducing gene expression.

Here we use polymerase chain reaction (PCR)-generated linear DNA molecules to augment the coding sequences of gene-carrying nanoparticles, thereby maximizing nucleic acid loading and minimizing the size of these nanocarriers. This approach results in a remarkable 16-fold increase in protein expression six days post-transfection in cells transfected with particles carrying the linear DNA compared with particles bearing circular plasmid DNA. The study also showed that the use of linear DNA entrapped in DNA@SiO2 resulted in a much more efficient level of gene expression compared to standard transfection reagents. The system developed in this study features simplicity, scalability, and increased transfection efficiency and gene expression over existing approaches, enabled by improved embedment capabilities for linear DNA, compared to conventional methods such as lipids or polymers, which generally show greater transfection efficiency with plasmid DNA. Therefore, this novel methodology can find applications not only in gene therapy but also in research settings for high throughput gene expression screenings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.570032">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.570032" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.570032">
        <p class="paperTitle">A Ratiometric Nonfluorescent CRISPR Assay Utilizing Cas12a-Induced Plasmid Supercoil Relaxation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.570032" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.570032" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mohammad, N.; Talton, L.; Dalgan, S.; Wei, Q.</p>
        <p class="info">Score: 5.3, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.570032' target='https://doi.org/10.1101/2023.12.04.570032'> 10.1101/2023.12.04.570032</a></p>
        <p class="abstract">Most CRISPR-based biosensors rely on labeled reporter molecules and expensive equipment for signal readout. A recent alternative approach quantifies analyte concentration by sizing native {lambda} DNA reporters, providing a simple and label-free solution for ultrasensitive detection. However, this method faces challenges in accurately quantifying size reduction of long DNA reporters via gel electrophoresis due to the sensitivity of DNA band shift to other interferences such as gel distortion. To overcome these limitations, here we developed a simple and robust ratiometric signaling strategy using CRISPR-Cas12a-induced supercoil relaxation of dsDNA plasmid reporters. In the presence of target, we observed that the fraction of supercoiled plasmid DNA decreased, and the amount of relaxed conformation (circular) increased over time. The relative percentage of supercoiled DNA to the relaxed circular DNA was analyzed by gel electrophoresis to generate an intensity-based ratiometric signal for more accurate target concentration quantification. This simple and inexpensive method is [~]100 times more sensitive when compared with the typical fluorescent reporter system. This self-referenced strategy solves the potential application limitations of previously demonstrated DNA sizing-based CRISPR-Dx without compromising the sensitivity. Finally, we demonstrated the applicability of ratiometric sensing strategy using model DNA targets such as AAV and HPV 16, highlighting its feasibility for point-of-use CRISPR-Dx applications.

Table of Contents (TOC)

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=117 SRC=&#34;FIGDIR/small/570032v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (41K):
org.highwire.dtl.DTLVardef@2ac1ccorg.highwire.dtl.DTLVardef@1195d17org.highwire.dtl.DTLVardef@2ec9fcorg.highwire.dtl.DTLVardef@1cef5da_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.570487">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.570487" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.570487">
        <p class="paperTitle">Democratizing biomedical simulation through automated model discovery and a universal material subroutine</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.570487" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.570487" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peirlinck, M.; Linka, K.; Hurtado, J. A.; Holzapfel, G. A.; Kuhl, E.</p>
        <p class="info">Score: 4.7, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.570487' target='https://doi.org/10.1101/2023.12.06.570487'> 10.1101/2023.12.06.570487</a></p>
        <p class="abstract">Personalized computational simulations have emerged as a vital tool to understand the biomechanical factors of a disease, predict disease progression, and design personalized intervention. Material modeling is critical for realistic biomedical simulations, and poor model selection can have life-threatening consequences for the patient. However, selecting the best model requires a profound domain knowledge and is limited to a few highly specialized experts in the field. Here we explore the feasibility of eliminating user involvement and automate the process of material modeling in finite element analyses. We leverage recent developments in constitutive neural networks, machine learning, and artificial intelligence to discover the best constitutive model from thousands of possible combinations of a few functional building blocks. We integrate all discoverable models into the finite element workflow by creating a universal material subroutine that contains more than 60,000 models, made up of 16 individual terms. We prototype this workflow using biaxial extension tests from healthy human arteries as input and stress and stretch profiles across the human aortic arch as output. Our results suggest that constitutive neural networks can robustly discover various flavors of arterial models from data, feed these models directly into a finite element simulation, and predict stress and strain profiles that compare favorably to the classical Holzapfel model. Replacing dozens of individual material subroutines by a single universal material subroutine-populated directly via automated model discovery-will make finite element simulations more user-friendly, more robust, and less vulnerable to human error. Democratizing finite element simulation by automating model selection could induce a paradigm shift in physics-based modeling, broaden access to simulation technologies, and empower individuals with varying levels of expertise and diverse backgrounds to actively participate in scientific discovery and push the boundaries of biomedical simulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570137">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570137" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570137">
        <p class="paperTitle">Noninvasive Injectable Optical Nanosensor-Hydrogel Hybrids Detect Doxorubicin in Living Mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570137" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570137" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cohen, Z.; Alpert, D. J.; Weisel, A. C.; Roach, A.; Rahman, S.; Gaikwad, P. V.; Nicoll, S. B.; Williams, R. M.</p>
        <p class="info">Score: 3.8, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570137' target='https://doi.org/10.1101/2023.12.05.570137'> 10.1101/2023.12.05.570137</a></p>
        <p class="abstract">While the tissue-transparent fluorescence of single-walled carbon nanotubes (SWCNTs) imparts substantial potential for use in non-invasive biosensors, development of non-invasive systems is yet to be realized. Here, we investigated the functionality of a SWCNT-based nanosensor in several injectable SWCNT-hydrogel systems, ultimately finding SWCNT encapsulation in a sulfonated methylcellulose hydrogel optimal for detection of ions, small molecules, and proteins. We found that the hydrogel system and nanosensor signal were stable for several weeks in live mice. We then found that this system successfully detects local injections of the chemotherapeutic agent doxorubicin in mice. We anticipate future studies to adapt this device for detection of other analytes in animals and, ultimately, patients.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.09.570848">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.09.570848" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.09.570848">
        <p class="paperTitle">Young adults accelerate their arms significantly faster and earlier than old adults resulting in improved center of mass dynamics during an overground slip perturbation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.09.570848" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.09.570848" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee-Confer, J.; Lo, M. K.; Troy, K.</p>
        <p class="info">Score: 3.7, Published: 2023-12-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.09.570848' target='https://doi.org/10.1101/2023.12.09.570848'> 10.1101/2023.12.09.570848</a></p>
        <p class="abstract">Arm motions reduce the likelihood of falling from a slip in young adults, and injuries to older adults from falls are known to occur from a frontal plane fall. Therefore, the purpose of this study was to quantify the frontal plane trunk and arm motion in older and younger adults during a slip. Eleven older adults (age: 72.0 &#43;/- 5.0) and 11 younger adults (age: 25.17 &#43;/- 5.94) were subjected to a slip during walking. 27% of the older adults and 11% of the younger adults experienced a fall from the slip incident. The arm excursion did not significantly differ between groups, however younger adults average peak arm abduction occurred 310 ms prior to older adults (p = 0.03). There were no significant differences in peak trunk between groups. The arm abduction acceleration was significantly higher in younger adults compared to older adults (3593.2 &#43;/- 1144.8 vs. 2309.8 &#43;/- 1428.5 degrees/s2, p = 0.03). Furthermore, young adults exhibited a significantly reduced CoM excursion compared to old adults (4.55 &#43;/- 3.45 vs. 10.27 &#43;/- 6.62 cm, p &lt; 0.01). Young adults may have exhibited lower fall rates due to the significantly higher acceleration of their arms compared to old adults.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568055">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568055" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568055">
        <p class="paperTitle">Engineering mice for female-biased progeny without impacting genetic integrity and litter size</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568055" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568055" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yosef, I.; Mahata, T.; Chen, Y.; Bar-Joseph, H.; Shalgi, R.; Munitz, A.; Gerlic, M.; Qimron, U.</p>
        <p class="info">Score: 43.8, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568055' target='https://doi.org/10.1101/2023.11.21.568055'> 10.1101/2023.11.21.568055</a></p>
        <p class="abstract">Generating mammalian gametes with a skewed sex ratio has thus far eluded empirical confirmation. The utilization of such genetically engineered organisms would offer the potential to curtail the necessity for culling animals of undesirable sex, mitigate resource wastage, and alleviate superfluous labor burdens. In this study, we introduce a transgenic male mouse lineage, which consistently yields predominantly female progeny (comprising up to 91% of the total offspring). This accomplishment was made possible by integrating a controllable genetic cassette onto the Y chromosome. The cassette encodes dCas9 and RNA guides that selectively silence a spermatid maturation gene. After the separation of X and Y gametes during meiosis, gametes containing an X chromosome develop normally, while those harboring the engineered Y chromosome, subjected to dCas9 silencing of the spermatid maturation gene, do not mature properly. Indeed, some spermatozoa from the transgenic mice exhibit a unique morphology, associated with the absence of the maturation gene. Notably, the resultant female offspring do not inherit the genetically engineered Y chromosome and are thus not genetically modified. Importantly, the litter size of the transgenic mice remains unchanged compared to the wild type. These findings represent a groundbreaking demonstration of genetic engineerings potential to yield sex-biased litters of full size without compromising genetic integrity, marking a pioneering advancement in this field of study.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=139 SRC=&#34;FIGDIR/small/568055v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (28K):
org.highwire.dtl.DTLVardef@18f9ccborg.highwire.dtl.DTLVardef@17f0577org.highwire.dtl.DTLVardef@3fd5bborg.highwire.dtl.DTLVardef@e900bc_HPS_FORMAT_FIGEXP  M_FIG Graphical Abstract

A proof of concept for the first mammal producing sexed semen may revolutionize the way for breeding animals such as cows for the dairy industry and chickens for the eggs industry.

C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571328">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571328" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571328">
        <p class="paperTitle">A high-fidelity CRISPR-Cas13 system improves abnormalities associated with C9ORF72-linked ALS/FTD</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571328" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571328" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McCallister, T. X.; Lim, C. K. W.; Terpstra, W. M.; C., M. A. Z.; Zhang, S.; Powell, J. E.; Gaj, T.</p>
        <p class="info">Score: 3.0, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571328' target='https://doi.org/10.1101/2023.12.12.571328'> 10.1101/2023.12.12.571328</a></p>
        <p class="abstract">An abnormal expansion of a GGGGCC hexanucleotide repeat in the C9ORF72 gene is the most common genetic cause of amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD), two debilitating neurodegenerative disorders driven in part by gain-of-function mechanisms involving transcribed forms of the repeat expansion. By utilizing a Cas13 variant with reduced collateral effects, we developed a high-fidelity RNA-targeting CRISPR-based system for C9ORF72-linked ALS/FTD. When delivered to the brain of a transgenic rodent model, this Cas13-based platform effectively curbed the expression of the GGGGCC repeat-containing RNA without affecting normal C9ORF72 levels, which in turn decreased the formation of RNA foci and reversed transcriptional deficits. This high-fidelity Cas13 variant possessed improved transcriptome-wide specificity compared to its native form and mediated efficient targeting in motor neuron-like cells derived from a patient with ALS. Our results lay the foundation for the implementation of RNA-targeting CRISPR technologies for C9ORF72-linked ALS/FTD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.10.570953">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.10.570953" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.10.570953">
        <p class="paperTitle">Highly efficient transgenic mouse production using piggyBac and its application to rapid phenotyping at the founder generation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.10.570953" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.10.570953" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Okamura, E.; Mizuno, S.; Matsumoto, S.; Murata, K.; Tanimoto, Y.; Tra, D. T. H.; Suzuki, H.; Kang, W.; Ema, T.; Morimoto, K.; Kato, K.; Matsumoto, T.; Masuyama, N.; Kijima, Y.; Morimura, T.; Sugiyama, F.; Takahashi, S.; Mizutani, E.; Woltjen, K.; Yachie, N.; Ema, M.</p>
        <p class="info">Score: 2.5, Published: 2023-12-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.10.570953' target='https://doi.org/10.1101/2023.12.10.570953'> 10.1101/2023.12.10.570953</a></p>
        <p class="abstract">Pronuclear microinjection is the most popular method for producing transgenic (Tg) animals. Because the production efficiency is typically less than 20%, phenotypic characterization of Tg animals is generally performed on the next generation (F1) onwards. However, apart from in rodents, in many animal species with long generation times, it is desirable to perform phenotyping in the founder (F0) generation. In this study, we attempted to optimize a method of Tg mouse production to achieve higher Tg production efficiency using piggyBac transposon systems and established optimal conditions under which almost all individuals in the F0 generation were Tg. We also succeeded in generating bacterial artificial chromosome Tg mice with efficiency of approximately 70%. By combining this method with genome editing technology, we established a new strategy to perform phenotyping of mice with tissue-specific knockout using the F0 generation. Taking the obtained findings together, by using this method, experimental research using Tg animals can be carried out more efficiently.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
