<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Direct 3D-bioprinting of hiPSC-derived cardiomyocytes to generate functional cardiac tissues
Authors: Esser, T. U.; Anspach, A.; Muenzebrock, K. A.; Kah, D.; Schruefer, S.; Schenk, J.; Heinze, K. G.; Schubert, D. W.; Fabry, B.; Engel, F. B.
Score: 78.6, Published: 2023-06-12 DOI: 10.1101/2023.06.12.544557
3D-bioprinting is a promising technology to produce human tissues as drug screening tool or for organ repair. However, direct printing of living cells has proven difficult. Here, we present a method to directly 3D-bioprint human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes embedded in a collagen-hyaluronic acid ink generating centimeter-sized functional ring- and ventricle-shaped cardiac tissues in an accurate and reproducible manner.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Direct 3D-bioprinting of hiPSC-derived cardiomyocytes to generate functional cardiac tissues
Authors: Esser, T. U.; Anspach, A.; Muenzebrock, K. A.; Kah, D.; Schruefer, S.; Schenk, J.; Heinze, K. G.; Schubert, D. W.; Fabry, B.; Engel, F. B.
Score: 78.6, Published: 2023-06-12 DOI: 10.1101/2023.06.12.544557
3D-bioprinting is a promising technology to produce human tissues as drug screening tool or for organ repair. However, direct printing of living cells has proven difficult. Here, we present a method to directly 3D-bioprint human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes embedded in a collagen-hyaluronic acid ink generating centimeter-sized functional ring- and ventricle-shaped cardiac tissues in an accurate and reproducible manner." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-09T10:40:17+00:00" />
<meta property="article:modified_time" content="2023-07-09T10:40:17+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Direct 3D-bioprinting of hiPSC-derived cardiomyocytes to generate functional cardiac tissues
Authors: Esser, T. U.; Anspach, A.; Muenzebrock, K. A.; Kah, D.; Schruefer, S.; Schenk, J.; Heinze, K. G.; Schubert, D. W.; Fabry, B.; Engel, F. B.
Score: 78.6, Published: 2023-06-12 DOI: 10.1101/2023.06.12.544557
3D-bioprinting is a promising technology to produce human tissues as drug screening tool or for organ repair. However, direct printing of living cells has proven difficult. Here, we present a method to directly 3D-bioprint human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes embedded in a collagen-hyaluronic acid ink generating centimeter-sized functional ring- and ventricle-shaped cardiac tissues in an accurate and reproducible manner."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Direct 3D-bioprinting of hiPSC-derived cardiomyocytes to generate functional cardiac tissues\nAuthors: Esser, T. U.; Anspach, A.; Muenzebrock, K. A.; Kah, D.; Schruefer, S.; Schenk, J.; Heinze, K. G.; Schubert, D. W.; Fabry, B.; Engel, F. B.\nScore: 78.6, Published: 2023-06-12 DOI: 10.1101/2023.06.12.544557\n3D-bioprinting is a promising technology to produce human tissues as drug screening tool or for organ repair. However, direct printing of living cells has proven difficult. Here, we present a method to directly 3D-bioprint human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes embedded in a collagen-hyaluronic acid ink generating centimeter-sized functional ring- and ventricle-shaped cardiac tissues in an accurate and reproducible manner.",
  "keywords": [
    
  ],
  "articleBody": " Direct 3D-bioprinting of hiPSC-derived cardiomyocytes to generate functional cardiac tissues\nAuthors: Esser, T. U.; Anspach, A.; Muenzebrock, K. A.; Kah, D.; Schruefer, S.; Schenk, J.; Heinze, K. G.; Schubert, D. W.; Fabry, B.; Engel, F. B.\nScore: 78.6, Published: 2023-06-12 DOI: 10.1101/2023.06.12.544557\n3D-bioprinting is a promising technology to produce human tissues as drug screening tool or for organ repair. However, direct printing of living cells has proven difficult. Here, we present a method to directly 3D-bioprint human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes embedded in a collagen-hyaluronic acid ink generating centimeter-sized functional ring- and ventricle-shaped cardiac tissues in an accurate and reproducible manner. The printed tissues contained hiPSC-derived cardiomyocytes with well-organized sarcomeres and exhibited spontaneous and regular contractions, which persisted for several months and were able to contract against passive resistance. Importantly, beating frequencies of the printed cardiac tissues could be modulated by pharmacological stimulation. This approach opens up new possibilities for generating complex functional cardiac tissues as models for advanced drug screening or as tissue grafts for organ repair or replacement.\nConfigurational entropy is an intrinsic driver of tissue structural heterogeneity\nAuthors: Srivastava, V.; Hu, J. L.; Garbe, J. C.; Veytsman, B.; Shalabi, S. F.; Yllanes, D.; Thomson, M.; LaBarge, M. A.; Huber, G.; Gartner, Z. J.\nScore: 16.9, Published: 2023-07-02 DOI: 10.1101/2023.07.01.546933\nTissues comprise ordered arrangements of cells that can be surprisingly disordered in their details. How the properties of single cells and their microenvironment contribute to the balance between order and disorder at the tissue-scale remains poorly understood. Here, we address this question using the self-organization of human mammary organoids as a model. We find that organoids behave like a dynamic structural ensemble at the steady state. We apply a maximum entropy formalism to derive the ensemble distribution from three measurable parameters - the degeneracy of structural states, interfacial energy, and tissue activity (the energy associated with positional fluctuations). We link these parameters with the molecular and microenvironmental factors that control them to precisely engineer the ensemble across multiple conditions. Our analysis reveals that the entropy associated with structural degeneracy sets a theoretical limit to tissue order and provides new insight for tissue engineering, development, and our understanding of disease progression.\nSEM2: A computational framework to model multiscale mechanics with subcellular elements\nAuthors: Chattaraj, S.; Torre, M.; Kalcher, C.; stukowski, A.; Morganti, S.; Reali, A.; Pasqualini, F.\nScore: 12.1, Published: 2023-07-08 DOI: 10.1101/2023.07.07.548118\nModeling multiscale mechanics in shape-shifting biological tissues in embryos, traditional, or engineered cell culture platforms (organoids, organs-on-chips) is both important and challenging. In fact, it is difficult to model relevant tissue-level structural changes mediated by discrete events at the cellular and subcellular levels, such as migration and proliferation. To accomplish this, we leveraged the subcellular element modeling (SEM) method, where ensembles of coarse-grained particles interacting via empirically defined potentials are used to model individual cells while preserving cell rheology. However, an explicit treatment of multiscale mechanics in SEM was missing. Here, we introduced SEM2, an extended version of the open-source software SEM++ and LAMMPS, enabling new analyses and visualization of particle-level stress and strain. We demonstrated various functionalities of SEM2 by simulating cell creep, migration, and proliferation in scenarios that recapitulate classical and engineered cell culture platforms. For every scenario, we highlight key mechanobiology that emerges spontaneously from particle interactions and discuss recent experimental evidence as qualitative validations of our simulations. The SEM2 code is available on GitHub at https://github.com/Synthetic-Physiology-Lab/sem2 .\nDesign, construction and optimization of formaldehyde growth biosensors with broad application in Biotechnology\nAuthors: Schann, K.; Bakker, J.; Boinot, M.; Kuschel, P.; He, H.; Nattermann, M.; Erb, T.; Bar-Even, A.; Wenk, S.\nScore: 12.3, Published: 2023-07-01 DOI: 10.1101/2023.06.29.547045\nFormaldehyde is a key metabolite in natural and synthetic one-carbon metabolism as well as an important environmental toxin with high toxicity at low concentrations. To engineer efficient formaldehyde producing enzymes and to detect formaldehyde in industrial or environmental samples, it is important to establish highly sensitive, easy to use and affordable formaldehyde detection methods. Here, we transformed the workhorse bacterium Escherichia coli into biosensors that can detect a broad range of formaldehyde concentrations. Based on natural and promiscuous formaldehyde assimilation enzymes, we designed and engineered three different E. coli strains that depend on formaldehyde assimilation for cellular growth. After in depth characterization of these biosensors, we show that the formaldehyde sensitivity can be improved through adaptive laboratory evolution or modification of metabolic branch points. The metabolic engineering strategy presented in this work allowed the creation of E. coli biosensors that can detect formaldehyde in a concentration range from [~]30 M to [~]13 mM. Using the most sensitive strain, we benchmarked the in vivo activities of different, widely used NAD-dependent methanol dehydrogenases, the rate-limiting enzyme in synthetic methylotrophy. We also show that the strains can grow upon external addition of formaldehyde indicating their potential use for applications beyond enzyme engineering. The formaldehyde biosensors developed in this study are fully genomic and can be used as plug and play devices for screening large enzyme libraries. Thus, they have the potential to greatly advance enzyme engineering and might even be used for environmental monitoring or analysis of industrial probes. Highlights- Conversion of E. coli into three different formaldehyde growth biosensors - Biosensors are fully genomic and grow robustly when formaldehyde is present - Biosensors can detect formaldehyde concentrations ranging from [~]30 M to [~]13 mM - Benchmarking of biotechnological relevant methanol dehydrogenases reveals potential of biosensors for enzyme engineering - Biosensors grow upon direct addition of formaldehyde indicating potential use in environmental or industrial settings\nCRISPR/Cas9 DNA synthesis disruption as a tool to control cell proliferation in vitro and in vivo\nAuthors: Sartori-Maldonado, R.; Montaser, H.; Soppa, I.; Balaz, M.; Puttonen, H.; Eurola, S.; Otonkoski, T.; Saarimaki-Vire, J.; Wartiovaara, K.\nScore: 7.4, Published: 2023-06-26 DOI: 10.1101/2023.06.26.546441\nCell therapies hold a lot of promise for regenerative medicine and gene therapy. However, many living cell products entail a fundamental biological risk of unwanted growth. Here, we describe a novel metabolic safety system to control cell proliferation without exogenous genetic elements. The method allows robust cell culture and manufacture while mitigating the risk of uncontrolled growth of transplanted cells. Using CRISPR/Cas9, we inactivated the enzyme in charge of the de novo thymidylate synthesis (TYMS) in several cell lines, including human induced pluripotent stem cells. This resulted in cells that proliferate when supplemented with exogenous dTMP but fail to grow in its absence. Under dTMP supplementation, TYMS-/- hiPSCs produce mature teratomas in vivo and differentiate normally into potentially therapeutic cell types, such us pancreatic beta cells. After terminal differentiation, the cells no longer require dTMP to remain functional, as seen by prolonged in vivo production of human insulin in mice.\nSingle cell susceptibility to SARS-CoV-2 infection is driven by variable cell states\nAuthors: Reffsin, S.; Miller, J.; Ayyanathan, K.; Dunagin, M. C.; Jain, N.; Schultz, D. C.; Cherry, S.; Raj, A.\nScore: 5.5, Published: 2023-07-07 DOI: 10.1101/2023.07.06.547955\nThe ability of a virus to infect a cell type is at least in part determined by the presence of host factors required for the viral life cycle. However, even within cell types that express known factors needed for infection, not every cell is equally susceptible, suggesting that our knowledge of the full spectrum of factors that promote infection is incomplete. Profiling the most susceptible subsets of cells within a population may reveal additional factors that promote infection. However, because viral infection dramatically alters the state of the cell, new approaches are needed to reveal the state of these cells prior to infection with virus. Here, we used single-cell clone tracing to retrospectively identify and characterize lung epithelial cells that are highly susceptible to infection with SARS-CoV-2. The transcriptional state of these highly susceptible cells includes markers of retinoic acid signaling and epithelial differentiation. Loss of candidate factors identified by our approach revealed that many of these factors play roles in viral entry. Moreover, a subset of these factors exert control over the infectable cell state itself, regulating the expression of key factors associated with viral infection and entry. Analysis of patient samples revealed the heterogeneous expression of these factors across both cells and patients in vivo. Further, the expression of these factors is upregulated in particular inflammatory pathologies. Altogether, our results show that the variable expression of intrinsic cell states is a major determinant of whether a cell can be infected by SARS-CoV-2.\nReproducing Left Ventricular Twist by Mimicking Myocardial Fiber Orientation Using 3D Bioprinting-assisted Tissue Assembly\nAuthors: Hwang, D. G.; Choi, H.; Yong, U.; Kim, D.; Kang, W.; Park, S.-M.; Jang, J.\nScore: 3.0, Published: 2023-07-04 DOI: 10.1101/2023.07.03.547595\nBackground: Left ventricular twist, an opposite rotation of the apex and base, is caused by myocardial fiber orientation, a unique structural feature of the myocardium, and contributes to the effective ejection fraction of the native heart. Reproducing this structural-functional relationship in an in vitro heart model remains challenging due to the lack of synchrony between layers when mimicking the fiber orientations of each layer. Methods: We employed a hierarchical approach for creating multilayered and multiaxial fibers in a chamber-like structure, as follows: 3D bioprinting-assisted tissue assembly, fabrication of uniaxially aligned engineered heart tissue as a building block, and assembly of them to create a myocardial fiber orientation in a chamber-like structure. Result: The EHT module confirmed uniaxial alignment and cardiac functions such as contractility and electrophysiological properties. By fabricating the assembly platform by 3D bioprinting, it is possible to guide building blocks in various directions as intended, confirming the versatility of this method. The assembly platform allows structural and functional synchrony of assembled tissues while controlling and maintaining predefined cellular alignment. Furthermore, various shapes and sizes of EHT modules and assembly platform were fabricated for mimicking myocardial fiber orientation in a chamber-like structure. The resulting structure exhibited three layers and three orientations representing myocardial fiber orientation. Moreover, the left ventricular twist was confirmed by measuring basal and apical rotations. Conclusions: Recapitulation of the microscale structure of the left ventricle enabled the identification of information not discernible from the existing macroscale structure. This understanding of the structure-function relationship of the heart can provide insights into the mechanisms underlying cardiac structure, function, and related diseases. Furthermore, the versatility of the 3D bioprinting-assisted tissue assembly allows for the creation of organs and tissue collections with complex structural and functional features by fabricating and assembling modules that meet the specific requirements of target tissues and organs.\nPlant Cellulose as a Substrate for 3D Neural Stem Cell Culture\nAuthors: Couvrette, L.; Walker, K.; Bui, T.; Pelling, A.\nScore: 2.8, Published: 2023-07-02 DOI: 10.1101/2023.06.30.547242\nNeural stem cell (NSC) based therapies are at the forefront of regenerative medicine strategies for various neural defects and injuries such as stroke, traumatic brain injury and spinal cord injury. For several clinical applications, NSC therapies require biocompatible scaffolds to support cell survival and to direct differentiation. Here, we investigate decellularized plant tissue as a novel scaffold for three-dimensional (3D) in vitro culture of NSCs. Plant cellulose scaffolds were shown to support attachment and proliferation of adult rat hippocampal neural stem cells (NSCs). Further, NSCs differentiated on the cellulose scaffold had significant increases in their expression of neuron-specific beta-III tubulin and glial fibrillary acidic protein compared to 2D culture on a polystyrene plate, indicating that the scaffold may enhance differentiation of NSCs towards astrocytic and neuronal lineages. Our findings suggest that plant-derived cellulose scaffolds have the potential to be used in neural tissue engineering and can be harnessed to direct differentiation of NSCs.\nComparison of robotic automated and manual injection methods in zebrafish embryos for high throughput RNA silencing using CRISPR-CasRx\nAuthors: Abugattas-Nunez Del Prado, J.; Ding, Y.; de Sonneville, J.; van der Kolk, K.-J.; Moreno-Mateos, M. A.; Malaga-Trillo, E.; Spaink, H. P.\nScore: 2.5, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547651\nRecently, the CRISPR-RfxCas13d (CasRx) system was proven to induce efficient mRNA knockdown in animal embryos. Here we compared the efficiency of CasRx-based RNA depletion with that of Cas9-mediated DNA targeting under the same conditions, using automated robotic and manual injection methods. As a proof-of-principle target, we used the no tail (tbxta) gene in zebrafish embryos, for which knockdown and knockout embryonic phenotypes were easy to be scored. Both Cas9 and CasRx systems induced loss of function phenotypes of tbxta gene. Higher percentage of severe phenotype was observed using Cas9 protein compared to the mRNA while the efficiency was similar in terms of Cas13d protein and mRNA. In addition, both the robotic and manual injection approaches yielded similar percentages of phenotypes and mortality rates. Therefore, our study not only showcases the potential of RNA-targeting CRISPR effectors for precise and potent gene knockdown, but also emphasizes automated microinjection in zebrafish embryos as an excellent alternative to manual methods for achieving gene knockdown at a high throughput level.\nThis microtubule does not exist: Super-resolution microscopy image generation by a diffusion model\nAuthors: Saguy, A.; Nahimov, T.; Lehrman, M.; Alalouf, O.; Shechtman, Y.\nScore: 2.2, Published: 2023-07-07 DOI: 10.1101/2023.07.06.548004\nGenerative models, such as diffusion models, have made significant advancements in recent years, enabling the synthesis of high-quality realistic data across various domains. Here, we explore the adaptation of a diffusion model to super-resolution microscopy images and train it on images from a publicly available database. We show that the generated images resemble experimental images, and that the generated images do not copy existing images from the training set. Additionally, we compare the performance of a deep-learning-based deconvolution method when trained on our generated data versus training on mathematical model based data and show superior reconstruction quality in means of spatial resolution. These findings demonstrate the potential contribution of generative diffusion models for microscopy tasks and pave the way for their future application in this field.\n",
  "wordCount" : "2273",
  "inLanguage": "en",
  "datePublished": "2023-07-09T10:40:17Z",
  "dateModified": "2023-07-09T10:40:17Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on July 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.12.544557">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.12.544557" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.12.544557">
        <p class="paperTitle">Direct 3D-bioprinting of hiPSC-derived cardiomyocytes to generate functional cardiac tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.12.544557" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.12.544557" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Esser, T. U.; Anspach, A.; Muenzebrock, K. A.; Kah, D.; Schruefer, S.; Schenk, J.; Heinze, K. G.; Schubert, D. W.; Fabry, B.; Engel, F. B.</p>
        <p class="info">Score: 78.6, Published: 2023-06-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.12.544557' target='https://doi.org/10.1101/2023.06.12.544557'> 10.1101/2023.06.12.544557</a></p>
        <p class="abstract">3D-bioprinting is a promising technology to produce human tissues as drug screening tool or for organ repair. However, direct printing of living cells has proven difficult. Here, we present a method to directly 3D-bioprint human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes embedded in a collagen-hyaluronic acid ink generating centimeter-sized functional ring- and ventricle-shaped cardiac tissues in an accurate and reproducible manner. The printed tissues contained hiPSC-derived cardiomyocytes with well-organized sarcomeres and exhibited spontaneous and regular contractions, which persisted for several months and were able to contract against passive resistance. Importantly, beating frequencies of the printed cardiac tissues could be modulated by pharmacological stimulation. This approach opens up new possibilities for generating complex functional cardiac tissues as models for advanced drug screening or as tissue grafts for organ repair or replacement.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.01.546933">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.01.546933" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.01.546933">
        <p class="paperTitle">Configurational entropy is an intrinsic driver of tissue structural heterogeneity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.01.546933" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.01.546933" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Srivastava, V.; Hu, J. L.; Garbe, J. C.; Veytsman, B.; Shalabi, S. F.; Yllanes, D.; Thomson, M.; LaBarge, M. A.; Huber, G.; Gartner, Z. J.</p>
        <p class="info">Score: 16.9, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.01.546933' target='https://doi.org/10.1101/2023.07.01.546933'> 10.1101/2023.07.01.546933</a></p>
        <p class="abstract">Tissues comprise ordered arrangements of cells that can be surprisingly disordered in their details. How the properties of single cells and their microenvironment contribute to the balance between order and disorder at the tissue-scale remains poorly understood. Here, we address this question using the self-organization of human mammary organoids as a model. We find that organoids behave like a dynamic structural ensemble at the steady state. We apply a maximum entropy formalism to derive the ensemble distribution from three measurable parameters - the degeneracy of structural states, interfacial energy, and tissue activity (the energy associated with positional fluctuations). We link these parameters with the molecular and microenvironmental factors that control them to precisely engineer the ensemble across multiple conditions. Our analysis reveals that the entropy associated with structural degeneracy sets a theoretical limit to tissue order and provides new insight for tissue engineering, development, and our understanding of disease progression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548118">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548118" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548118">
        <p class="paperTitle">SEM2: A computational framework to model multiscale mechanics with subcellular elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548118" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548118" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chattaraj, S.; Torre, M.; Kalcher, C.; stukowski, A.; Morganti, S.; Reali, A.; Pasqualini, F.</p>
        <p class="info">Score: 12.1, Published: 2023-07-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548118' target='https://doi.org/10.1101/2023.07.07.548118'> 10.1101/2023.07.07.548118</a></p>
        <p class="abstract">Modeling multiscale mechanics in shape-shifting biological tissues in embryos, traditional, or engineered cell culture platforms (organoids, organs-on-chips) is both important and challenging. In fact, it is difficult to model relevant tissue-level structural changes mediated by discrete events at the cellular and subcellular levels, such as migration and proliferation. To accomplish this, we leveraged the subcellular element modeling (SEM) method, where ensembles of coarse-grained particles interacting via empirically defined potentials are used to model individual cells while preserving cell rheology. However, an explicit treatment of multiscale mechanics in SEM was missing. Here, we introduced SEM2, an extended version of the open-source software SEM&#43;&#43; and LAMMPS, enabling new analyses and visualization of particle-level stress and strain. We demonstrated various functionalities of SEM2 by simulating cell creep, migration, and proliferation in scenarios that recapitulate classical and engineered cell culture platforms. For every scenario, we highlight key mechanobiology that emerges spontaneously from particle interactions and discuss recent experimental evidence as qualitative validations of our simulations. The SEM2 code is available on GitHub at https://github.com/Synthetic-Physiology-Lab/sem2 .</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547045">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547045" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547045">
        <p class="paperTitle">Design, construction and optimization of formaldehyde growth biosensors with broad application in Biotechnology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547045" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547045" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schann, K.; Bakker, J.; Boinot, M.; Kuschel, P.; He, H.; Nattermann, M.; Erb, T.; Bar-Even, A.; Wenk, S.</p>
        <p class="info">Score: 12.3, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547045' target='https://doi.org/10.1101/2023.06.29.547045'> 10.1101/2023.06.29.547045</a></p>
        <p class="abstract">Formaldehyde is a key metabolite in natural and synthetic one-carbon metabolism as well as an important environmental toxin with high toxicity at low concentrations. To engineer efficient formaldehyde producing enzymes and to detect formaldehyde in industrial or environmental samples, it is important to establish highly sensitive, easy to use and affordable formaldehyde detection methods. Here, we transformed the workhorse bacterium Escherichia coli into biosensors that can detect a broad range of formaldehyde concentrations. Based on natural and promiscuous formaldehyde assimilation enzymes, we designed and engineered three different E. coli strains that depend on formaldehyde assimilation for cellular growth. After in depth characterization of these biosensors, we show that the formaldehyde sensitivity can be improved through adaptive laboratory evolution or modification of metabolic branch points. The metabolic engineering strategy presented in this work allowed the creation of E. coli biosensors that can detect formaldehyde in a concentration range from [~]30 M to [~]13 mM. Using the most sensitive strain, we benchmarked the in vivo activities of different, widely used NAD-dependent methanol dehydrogenases, the rate-limiting enzyme in synthetic methylotrophy. We also show that the strains can grow upon external addition of formaldehyde indicating their potential use for applications beyond enzyme engineering. The formaldehyde biosensors developed in this study are fully genomic and can be used as plug and play devices for screening large enzyme libraries. Thus, they have the potential to greatly advance enzyme engineering and might even be used for environmental monitoring or analysis of industrial probes.

Highlights- Conversion of E. coli into three different formaldehyde growth biosensors
- Biosensors are fully genomic and grow robustly when formaldehyde is present
- Biosensors can detect formaldehyde concentrations ranging from [~]30 M to [~]13 mM
- Benchmarking of biotechnological relevant methanol dehydrogenases reveals potential of biosensors for enzyme engineering
- Biosensors grow upon direct addition of formaldehyde indicating potential use in environmental or industrial settings</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546441">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546441" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546441">
        <p class="paperTitle">CRISPR/Cas9 DNA synthesis disruption as a tool to control cell proliferation in vitro and in vivo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546441" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546441" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sartori-Maldonado, R.; Montaser, H.; Soppa, I.; Balaz, M.; Puttonen, H.; Eurola, S.; Otonkoski, T.; Saarimaki-Vire, J.; Wartiovaara, K.</p>
        <p class="info">Score: 7.4, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546441' target='https://doi.org/10.1101/2023.06.26.546441'> 10.1101/2023.06.26.546441</a></p>
        <p class="abstract">Cell therapies hold a lot of promise for regenerative medicine and gene therapy. However, many living cell products entail a fundamental biological risk of unwanted growth. Here, we describe a novel metabolic safety system to control cell proliferation without exogenous genetic elements. The method allows robust cell culture and manufacture while mitigating the risk of uncontrolled growth of transplanted cells.

Using CRISPR/Cas9, we inactivated the enzyme in charge of the de novo thymidylate synthesis (TYMS) in several cell lines, including human induced pluripotent stem cells. This resulted in cells that proliferate when supplemented with exogenous dTMP but fail to grow in its absence. Under dTMP supplementation, TYMS-/- hiPSCs produce mature teratomas in vivo and differentiate normally into potentially therapeutic cell types, such us pancreatic beta cells. After terminal differentiation, the cells no longer require dTMP to remain functional, as seen by prolonged in vivo production of human insulin in mice.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.547955">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.547955" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.547955">
        <p class="paperTitle">Single cell susceptibility to SARS-CoV-2 infection is driven by variable cell states</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.547955" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.547955" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reffsin, S.; Miller, J.; Ayyanathan, K.; Dunagin, M. C.; Jain, N.; Schultz, D. C.; Cherry, S.; Raj, A.</p>
        <p class="info">Score: 5.5, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.547955' target='https://doi.org/10.1101/2023.07.06.547955'> 10.1101/2023.07.06.547955</a></p>
        <p class="abstract">The ability of a virus to infect a cell type is at least in part determined by the presence of host factors required for the viral life cycle. However, even within cell types that express known factors needed for infection, not every cell is equally susceptible, suggesting that our knowledge of the full spectrum of factors that promote infection is incomplete. Profiling the most susceptible subsets of cells within a population may reveal additional factors that promote infection. However, because viral infection dramatically alters the state of the cell, new approaches are needed to reveal the state of these cells prior to infection with virus. Here, we used single-cell clone tracing to retrospectively identify and characterize lung epithelial cells that are highly susceptible to infection with SARS-CoV-2. The transcriptional state of these highly susceptible cells includes markers of retinoic acid signaling and epithelial differentiation. Loss of candidate factors identified by our approach revealed that many of these factors play roles in viral entry. Moreover, a subset of these factors exert control over the infectable cell state itself, regulating the expression of key factors associated with viral infection and entry. Analysis of patient samples revealed the heterogeneous expression of these factors across both cells and patients in vivo. Further, the expression of these factors is upregulated in particular inflammatory pathologies. Altogether, our results show that the variable expression of intrinsic cell states is a major determinant of whether a cell can be infected by SARS-CoV-2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547595">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547595" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547595">
        <p class="paperTitle">Reproducing Left Ventricular Twist by Mimicking Myocardial Fiber Orientation Using 3D Bioprinting-assisted Tissue Assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547595" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547595" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hwang, D. G.; Choi, H.; Yong, U.; Kim, D.; Kang, W.; Park, S.-M.; Jang, J.</p>
        <p class="info">Score: 3.0, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547595' target='https://doi.org/10.1101/2023.07.03.547595'> 10.1101/2023.07.03.547595</a></p>
        <p class="abstract">Background: Left ventricular twist, an opposite rotation of the apex and base, is caused by myocardial fiber orientation, a unique structural feature of the myocardium, and contributes to the effective ejection fraction of the native heart. Reproducing this structural-functional relationship in an in vitro heart model remains challenging due to the lack of synchrony between layers when mimicking the fiber orientations of each layer. Methods: We employed a hierarchical approach for creating multilayered and multiaxial fibers in a chamber-like structure, as follows: 3D bioprinting-assisted tissue assembly, fabrication of uniaxially aligned engineered heart tissue as a building block, and assembly of them to create a myocardial fiber orientation in a chamber-like structure. Result: The EHT module confirmed uniaxial alignment and cardiac functions such as contractility and electrophysiological properties. By fabricating the assembly platform by 3D bioprinting, it is possible to guide building blocks in various directions as intended, confirming the versatility of this method. The assembly platform allows structural and functional synchrony of assembled tissues while controlling and maintaining predefined cellular alignment. Furthermore, various shapes and sizes of EHT modules and assembly platform were fabricated for mimicking myocardial fiber orientation in a chamber-like structure. The resulting structure exhibited three layers and three orientations representing myocardial fiber orientation. Moreover, the left ventricular twist was confirmed by measuring basal and apical rotations. Conclusions: Recapitulation of the microscale structure of the left ventricle enabled the identification of information not discernible from the existing macroscale structure. This understanding of the structure-function relationship of the heart can provide insights into the mechanisms underlying cardiac structure, function, and related diseases. Furthermore, the versatility of the 3D bioprinting-assisted tissue assembly allows for the creation of organs and tissue collections with complex structural and functional features by fabricating and assembling modules that meet the specific requirements of target tissues and organs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547242">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547242" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547242">
        <p class="paperTitle">Plant Cellulose as a Substrate for 3D Neural Stem Cell Culture</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547242" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547242" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Couvrette, L.; Walker, K.; Bui, T.; Pelling, A.</p>
        <p class="info">Score: 2.8, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547242' target='https://doi.org/10.1101/2023.06.30.547242'> 10.1101/2023.06.30.547242</a></p>
        <p class="abstract">Neural stem cell (NSC) based therapies are at the forefront of regenerative medicine strategies for various neural defects and injuries such as stroke, traumatic brain injury and spinal cord injury. For several clinical applications, NSC therapies require biocompatible scaffolds to support cell survival and to direct differentiation. Here, we investigate decellularized plant tissue as a novel scaffold for three-dimensional (3D) in vitro culture of NSCs. Plant cellulose scaffolds were shown to support attachment and proliferation of adult rat hippocampal neural stem cells (NSCs). Further, NSCs differentiated on the cellulose scaffold had significant increases in their expression of neuron-specific beta-III tubulin and glial fibrillary acidic protein compared to 2D culture on a polystyrene plate, indicating that the scaffold may enhance differentiation of NSCs towards astrocytic and neuronal lineages. Our findings suggest that plant-derived cellulose scaffolds have the potential to be used in neural tissue engineering and can be harnessed to direct differentiation of NSCs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547651">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547651" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547651">
        <p class="paperTitle">Comparison of robotic automated and manual injection methods in zebrafish embryos for high throughput RNA silencing using CRISPR-CasRx</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547651" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547651" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abugattas-Nunez Del Prado, J.; Ding, Y.; de Sonneville, J.; van der Kolk, K.-J.; Moreno-Mateos, M. A.; Malaga-Trillo, E.; Spaink, H. P.</p>
        <p class="info">Score: 2.5, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547651' target='https://doi.org/10.1101/2023.07.04.547651'> 10.1101/2023.07.04.547651</a></p>
        <p class="abstract">Recently, the CRISPR-RfxCas13d (CasRx) system was proven to induce efficient mRNA knockdown in animal embryos. Here we compared the efficiency of CasRx-based RNA depletion with that of Cas9-mediated DNA targeting under the same conditions, using automated robotic and manual injection methods. As a proof-of-principle target, we used the no tail (tbxta) gene in zebrafish embryos, for which knockdown and knockout embryonic phenotypes were easy to be scored. Both Cas9 and CasRx systems induced loss of function phenotypes of tbxta gene. Higher percentage of severe phenotype was observed using Cas9 protein compared to the mRNA while the efficiency was similar in terms of Cas13d protein and mRNA. In addition, both the robotic and manual injection approaches yielded similar percentages of phenotypes and mortality rates. Therefore, our study not only showcases the potential of RNA-targeting CRISPR effectors for precise and potent gene knockdown, but also emphasizes automated microinjection in zebrafish embryos as an excellent alternative to manual methods for achieving gene knockdown at a high throughput level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.548004">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.548004" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.548004">
        <p class="paperTitle">This microtubule does not exist: Super-resolution microscopy image generation by a diffusion model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.548004" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.548004" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saguy, A.; Nahimov, T.; Lehrman, M.; Alalouf, O.; Shechtman, Y.</p>
        <p class="info">Score: 2.2, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.548004' target='https://doi.org/10.1101/2023.07.06.548004'> 10.1101/2023.07.06.548004</a></p>
        <p class="abstract">Generative models, such as diffusion models, have made significant advancements in recent years, enabling the synthesis of high-quality realistic data across various domains. Here, we explore the adaptation of a diffusion model to super-resolution microscopy images and train it on images from a publicly available database. We show that the generated images resemble experimental images, and that the generated images do not copy existing images from the training set. Additionally, we compare the performance of a deep-learning-based deconvolution method when trained on our generated data versus training on mathematical model based data and show superior reconstruction quality in means of spatial resolution. These findings demonstrate the potential contribution of generative diffusion models for microscopy tasks and pave the way for their future application in this field.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
