<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Young adults accelerate their arms significantly faster and earlier than old adults resulting in improved center of mass dynamics during an overground slip perturbation
Authors: Lee-Confer, J.; Lo, M. K.; Troy, K.
Score: 92.9, Published: 2023-12-10 DOI: 10.1101/2023.12.09.570848
Arm motions reduce the likelihood of falling from a slip in young adults, and injuries to older adults from falls are known to occur from a frontal plane fall. Therefore, the purpose of this study was to quantify the frontal plane trunk and arm motion in older and younger adults during a slip.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Young adults accelerate their arms significantly faster and earlier than old adults resulting in improved center of mass dynamics during an overground slip perturbation
Authors: Lee-Confer, J.; Lo, M. K.; Troy, K.
Score: 92.9, Published: 2023-12-10 DOI: 10.1101/2023.12.09.570848
Arm motions reduce the likelihood of falling from a slip in young adults, and injuries to older adults from falls are known to occur from a frontal plane fall. Therefore, the purpose of this study was to quantify the frontal plane trunk and arm motion in older and younger adults during a slip." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-20T10:36:27+00:00" />
<meta property="article:modified_time" content="2023-12-20T10:36:27+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Young adults accelerate their arms significantly faster and earlier than old adults resulting in improved center of mass dynamics during an overground slip perturbation
Authors: Lee-Confer, J.; Lo, M. K.; Troy, K.
Score: 92.9, Published: 2023-12-10 DOI: 10.1101/2023.12.09.570848
Arm motions reduce the likelihood of falling from a slip in young adults, and injuries to older adults from falls are known to occur from a frontal plane fall. Therefore, the purpose of this study was to quantify the frontal plane trunk and arm motion in older and younger adults during a slip."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Young adults accelerate their arms significantly faster and earlier than old adults resulting in improved center of mass dynamics during an overground slip perturbation\nAuthors: Lee-Confer, J.; Lo, M. K.; Troy, K.\nScore: 92.9, Published: 2023-12-10 DOI: 10.1101/2023.12.09.570848\nArm motions reduce the likelihood of falling from a slip in young adults, and injuries to older adults from falls are known to occur from a frontal plane fall. Therefore, the purpose of this study was to quantify the frontal plane trunk and arm motion in older and younger adults during a slip.",
  "keywords": [
    
  ],
  "articleBody": " Young adults accelerate their arms significantly faster and earlier than old adults resulting in improved center of mass dynamics during an overground slip perturbation\nAuthors: Lee-Confer, J.; Lo, M. K.; Troy, K.\nScore: 92.9, Published: 2023-12-10 DOI: 10.1101/2023.12.09.570848\nArm motions reduce the likelihood of falling from a slip in young adults, and injuries to older adults from falls are known to occur from a frontal plane fall. Therefore, the purpose of this study was to quantify the frontal plane trunk and arm motion in older and younger adults during a slip. Eleven older adults (age: 72.0 {+/-} 5.0) and 11 younger adults (age: 25.17 {+/-} 5.94) were subjected to a slip during walking. 27% of the older adults and 11% of the younger adults experienced a fall from the slip incident. The arm excursion did not significantly differ between groups, however younger adults average peak arm abduction occurred 310 ms prior to older adults (p = 0.03). There were no significant differences in peak trunk between groups. The arm abduction acceleration was significantly higher in younger adults compared to older adults (3593.2 {+/-} 1144.8 vs. 2309.8 {+/-} 1428.5 degrees/s2, p = 0.03). Furthermore, young adults exhibited a significantly reduced CoM excursion compared to old adults (4.55 {+/-} 3.45 vs. 10.27 {+/-} 6.62 cm, p \u003c 0.01). Young adults may have exhibited lower fall rates due to the significantly higher acceleration of their arms compared to old adults.\nStructure-conditioned masked language models for protein sequence design generalize beyond the native sequence space\nAuthors: Akpinaroglu, D.; Seki, K.; Guo, A.; Zhu, E.; Kelly, M. J.; Kortemme, T.\nScore: 18.3, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571823\nMachine learning has revolutionized computational protein design, enabling significant progress in protein backbone generation and sequence design. Here, we introduce Frame2seq, a structure-conditioned masked language model for protein sequence design. Frame2seq generates sequences in a single pass, achieves 49.1% sequence recovery on the CATH 4.2 test dataset, and accurately estimates the error in its own predictions, outperforming the autoregressive ProteinMPNN model with over six times faster inference. To probe the ability of Frame2seq to generate novel designs beyond the native-like sequence space it was trained on, we experimentally test 26 Frame2seq designs for de novo backbones with low identity to the starting sequences. We show that Frame2seq successfully designs soluble (22/26), monomeric, folded, and stable proteins (17/26), including a design with 0% sequence identity to native. The speed and accuracy of Frame2seq will accelerate exploration of novel sequence space across diverse design tasks, including challenging applications such as multi-objective optimization.\nTopoDiff: Improving Protein Backbone Generationwith Topology-aware Latent Encoding\nAuthors: Zhang, Y.; Ma, Z.; Gong, H.\nScore: 7.0, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571602\nThe de novo design of protein structures is an intriguing research topic in the field of protein engineering. Recent breakthroughs in diffusion-based generative models have demonstrated substantial promise in tackling this task, notably in the generation of diverse and realistic protein structures. While existing models predominantly focus on unconditional generation or fine-grained conditioning at the residue level, the holistic, top-down approaches to control the overall topological arrangements are still insufficiently explored. In response, we introduce TopoDiff, a diffusion-based framework augmented by a global-structure encoding module, which is capable of unsupervisedly learning a compact latent representation of natural protein topologies with interpretable characteristics and simultaneously harnessing this learned information for controllable protein structure generation. We also propose a novel metric specifically designed to assess the coverage of sampled proteins with respect to the natural protein space. In comparative analyses with existing models, our generative model not only demonstrates comparable performance on established metrics but also exhibits better coverage across the recognized topology landscape. In summary, TopoDiff emerges as a novel solution towards enhancing the controllability and comprehensiveness of de novo protein structure generation, presenting new possibilities for innovative applications in protein engineering and beyond.\nA high-fidelity CRISPR-Cas13 system improves abnormalities associated with C9ORF72-linked ALS/FTD\nAuthors: McCallister, T. X.; Lim, C. K. W.; Terpstra, W. M.; C., M. A. Z.; Zhang, S.; Powell, J. E.; Gaj, T.\nScore: 5.8, Published: 2023-12-12 DOI: 10.1101/2023.12.12.571328\nAn abnormal expansion of a GGGGCC hexanucleotide repeat in the C9ORF72 gene is the most common genetic cause of amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD), two debilitating neurodegenerative disorders driven in part by gain-of-function mechanisms involving transcribed forms of the repeat expansion. By utilizing a Cas13 variant with reduced collateral effects, we developed a high-fidelity RNA-targeting CRISPR-based system for C9ORF72-linked ALS/FTD. When delivered to the brain of a transgenic rodent model, this Cas13-based platform effectively curbed the expression of the GGGGCC repeat-containing RNA without affecting normal C9ORF72 levels, which in turn decreased the formation of RNA foci and reversed transcriptional deficits. This high-fidelity Cas13 variant possessed improved transcriptome-wide specificity compared to its native form and mediated efficient targeting in motor neuron-like cells derived from a patient with ALS. Our results lay the foundation for the implementation of RNA-targeting CRISPR technologies for C9ORF72-linked ALS/FTD.\nAn Integrated Optogenetic and Bioelectronic Platform for Regulating Cardiomyocyte Function\nAuthors: Bolonduro, O. A.; Chen, Z.; Lai, Y.-R.; Cote, M.; Rao, A. A.; Liu, H.; Tzanakakis, E. S.; Timko, B. P.\nScore: 3.6, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571704\nWe report an integrated optogenetic and bioelectronic platform for stable and long-term modulation and monitoring of cardiomyocyte function in vitro. Optogenetic inputs were achieved through expression of a photoactivatable adenylyl cyclase (bPAC), that when activated by blue light caused a dose-dependent and time-limited increase in autonomous cardiomyocyte beat rate. Bioelectronic readouts were achieved through an integrated planar multi-electrode array (MEA) that provided real-time readouts of electrophysiological activity from 32 spatially-distinct locations. Irradiation at 27 W/mm2 resulted in a ca. 14% increase in beat rate within 20-25 minutes, which remained stable for at least 2 hours. The beating rate could be cycled through repeated \"on\" and \"off\" states, and its magnitude was a monotonic function of irradiation intensity. Our integrated platform opens new avenues in bioelectronic medicine, including closedloop feedback systems, with potential applications for cardiac regulation including arrhythmia diagnosis and intervention. TeaserA system that integrates optogenetic stimulation and bioelectronic recording capabilities allows for on-demand regulation of cardiac cell function.\nEngineered Protein Nanosheets for the Scale up of Mesenchymal Stem Cell Culture on Bioemulsions\nAuthors: Bosch-Fortea, M.; Nadal, C.; Chrysanthou, A.; Marciano, D.; Kanso, H.; Nguyen, N.; Gautrot, J.\nScore: 2.6, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572135\nThe rapid progress in cell therapies and stem cell technologies requires the development of novel bioprocessing and biomanufacturing pipelines able to cope with the scale up of cell manufacturing. In this respect, microdroplet technologies have already revolutionised the field of biotechnologies, but remain ill-suited to the culture of adherent cells. In this report, we describe the engineering of albumins with cell adhesive peptides for the stabilisation of microdroplets enabling the scale up of mesenchymal stem cell (MSC) expansion. We characterise the modified albumins prior to study their self-assembly at liquid-liquid interfaces via interfacial shear rheology, and mechanical strengthening through the formation of crosslinked nanosheets. The biofunctionalisation of these protein nanosheets is then characterised by fluorescence microscopy. In turn, the ability of the resulting bioactive microdroplets to promote rapid cell adhesion and expansion is examined and the extensive deposition of matrix associated with such cultures is characterised. The culture of MSCs is then scaled up 100 fold, first at the surface of fluorinated oil emulsions, then on plant-based emulsions stabilised by engineered protein nanosheets and the phenotype of resulting cells is characterised. The microdroplet culture system presented displays attractive advantages over existing technologies, in terms of simplicity of processing, compatibility with regulatory expectations and costs of production, and offers exciting opportunities for translation to cell manufacturing, for cell therapies and cultivated meat applications.\npyQCM-BraTaDio: A tool for visualization, data mining, and modeling of Quartz crystal microbalance with dissipation data\nAuthors: Pardi, B. M.; Ahmed, S. T.; Jonguitud Flores, S.; Flores, W.; Mears, L. L. M.; Yanez Soto, B.; Andresen Eguiluz, R. C.\nScore: 2.5, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571789\nHere, we present a Python based software that allows for the rapid visualization, data mining, and basic model applications of quartz crystal microbalance with dissipation data. Our implementation begins with a Tkinter GUI to prompt the user for all required information, such as file name/location, selection of baseline time, and overtones for visualization (with customization capabilities). These inputs are then fed to a workflow that will use the baseline time to scrub and temporally shift data using the Pandas and Numpy libraries and carry out the plot options for visualization. The last stage consists of an interactive plot, that presents the data and allows the user to select ranges in MatPlotLib-generated panels, followed by application of data models, including Sauerbrey, thin films in liquid, among others, that are carried out with NumPy and SciPy. The implementation of this software allows for simple and expedited data analysis, in lieu of time consuming and labor-intensive spreadsheet analysis. Metadata O_TBL View this table: org.highwire.dtl.DTLVardef@1d8793aorg.highwire.dtl.DTLVardef@ff020corg.highwire.dtl.DTLVardef@1df6ea6org.highwire.dtl.DTLVardef@1d95b67org.highwire.dtl.DTLVardef@ffec0b_HPS_FORMAT_FIGEXP M_TBL C_TBL\nAdaptive LDA classifier enhances real-time control of an EEG Brain-computer interface for imagined-speech decoding\nAuthors: Wu, S.; Bhadra, K.; Giraud, A.-L.; Marchesotti, S.\nScore: 2.5, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571206\nBrain-Computer Interfaces (BCI) aim to establish a pathway between the brain and an external device without the involvement of the motor system, relying exclusively on neural signals. Such systems have the potential to provide a means of communication for patients who have lost the ability to speak due to a neurological disorder. Traditional methodologies for decoding imagined speech directly from brain signals often deploy static classifiers, that is, decoders that are computed once at the beginning of the experiment and that remain unchanged throughout the BCI use. However, this approach might be inadequate in effectively handling the non-stationary nature of electroencephalography (EEG) signals and the learning that accompanies BCI use as parameters are expected to change, all the more in a real-time setting. To address this limitation, we have developed an adaptive classifier that updates its parameters based on the incoming data in real time. We first identified optimal parameters (the update coefficient, UC) to be used in an adaptive Linear Discriminant Analysis (LDA) classifier, using a previously recorded EEG dataset, acquired while healthy participants controlled a binary BCI based on imagined syllable decoding. We subsequently tested the effectiveness of this optimization in a real-time BCI-control setting. Twenty healthy participants performed two BCI-control sessions based on the imagery of two syllables, using a static LDA and the other the adaptive LDA classifier, in randomized order. In this real-time BCI-control task, the adaptive classifier led to better performances than the static one. Furthermore, the optimal parameters for the adaptive classifier were closely aligned in both datasets acquired with the same syllable imagery task. These findings highlight the effectiveness and re-liability of adaptive LDA classifiers for real-time imagined speech decoding, and its interest for non-invasive EEG-based BCI notably characterized by low decoding accuracies.\nBattery-Free Optoelectronic Patch for Photodynamic and Light Therapies in Treating Bacteria-Infected Wounds\nAuthors: Xue, Z.; Chou, W.; Xu, Y.; Cheng, Z.; Ren, X.; Sun, T.; Tong, W.; Xie, Y.; Chen, J.; Zhang, N.; Sheng, X.; Wang, Y.; Zhao, H.; Yang, J.; Ding, H.\nScore: 2.2, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571417\nLight therapy is an effective approach for the treatment of a variety of challenging dermatological conditions. In contrast to existing methods involving high doses and large areas of illumination, alternative strategies based on wearable designs that utilize a low light dose over an extended period provide a precise and convenient treatment. In this study, we present a battery-free, skin-integrated optoelectronic patch that incorporates a coil-powered circuit, an array of microscale violet and red light emitting diodes (LEDs), and polymer microneedles (MNs) loaded with 5-aminolevulinic acid (5-ALA). This polymer MNs, based on the biodegradable composite materials of polyvinyl alcohol (PVA) and hyaluronic acid (HA), serves as light waveguides for optical access and a medium for drug release into deeper skin layers. Unlike conventional clinical photomedical appliances with a rigid and fixed light source, this flexible design allows for a conformable light source that can be applied directly to the skin. In animal models with bacterial-infected wounds, the experimental group with the combination treatment of metronomic photodynamic and light therapies reduced 2.48 log10 CFU mL-1 in bactericidal level compared to the control group, indicating an effective anti-infective response. Furthermore, post-treatment analysis revealed the activation of proregenerative genes in monocyte and macrophage cell populations, suggesting enhanced tissue regeneration, neovascularization, and dermal recovery. Overall, this optoelectronic patch design broadens the scope for targeting deep skin lesions, and provides an alternative with the functionality of standard clinical light therapy methods.\nNAD: Noise-augmented direct sequencing of target nucleic acids by augmenting with noise and selective sampling\nAuthors: Shim, H.\nScore: 2.0, Published: 2023-12-15 DOI: 10.1101/2023.12.14.571721\nNext-generation sequencing necessitates a minimum quantity and concentration of DNA/RNA samples, typically achieved through amplification using the PCR technique. However, this amplification step introduces several drawbacks to biological insights, including PCR bias and the loss of epigenetic information. The advent of long-read sequencing technologies facilitates direct sequencing, with the primary constraint being the limited amount of DNA/RNA present in biological samples. Here, we present a novel method called Noise-Augmented Direct (NAD) sequencing that enables the direct sequencing of target DNA even when it falls below the minimum quantity and concentration required for long-read sequencing by augmenting with noise DNA and adaptive sampling. Adaptive sampling is an emerging technology of nanopore sequencing, allowing the enhanced sequencing of target DNA by selectively depleting noise DNA. In this study, we use the DNA standard of the Lambda phage genome as the noise DNA to augment samples containing low amounts of bacterial genomes (1 ng to 300 ng). The results with cost-effective flow cells indicate that NAD sequencing successfully detects the target DNA with an input quantity as low as 1 ng, and the bacterial genome of Salmonella enterica can be assembled to 30% completion at an accuracy of 98% with an input quantity of 3 ng. With high throughput flow cells, the bacterial genome of Pseudonomas aeruginas was assembled to near completion (99.9%) at an accuracy of 99.97% with an input quantity of 300 ng. This proof-of-concept study demonstrates the potential of NAD sequencing in enhancing the robustness of long-read sequencing with small input DNA/RNA samples with noise augmentation and adaptive sampling.\n",
  "wordCount" : "2366",
  "inLanguage": "en",
  "datePublished": "2023-12-20T10:36:27Z",
  "dateModified": "2023-12-20T10:36:27Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 20, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.09.570848">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.09.570848" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.09.570848">
        <p class="paperTitle">Young adults accelerate their arms significantly faster and earlier than old adults resulting in improved center of mass dynamics during an overground slip perturbation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.09.570848" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.09.570848" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee-Confer, J.; Lo, M. K.; Troy, K.</p>
        <p class="info">Score: 92.9, Published: 2023-12-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.09.570848' target='https://doi.org/10.1101/2023.12.09.570848'> 10.1101/2023.12.09.570848</a></p>
        <p class="abstract">Arm motions reduce the likelihood of falling from a slip in young adults, and injuries to older adults from falls are known to occur from a frontal plane fall. Therefore, the purpose of this study was to quantify the frontal plane trunk and arm motion in older and younger adults during a slip. Eleven older adults (age: 72.0 {&#43;/-} 5.0) and 11 younger adults (age: 25.17 {&#43;/-} 5.94) were subjected to a slip during walking. 27% of the older adults and 11% of the younger adults experienced a fall from the slip incident. The arm excursion did not significantly differ between groups, however younger adults average peak arm abduction occurred 310 ms prior to older adults (p = 0.03). There were no significant differences in peak trunk between groups. The arm abduction acceleration was significantly higher in younger adults compared to older adults (3593.2 {&#43;/-} 1144.8 vs. 2309.8 {&#43;/-} 1428.5 degrees/s2, p = 0.03). Furthermore, young adults exhibited a significantly reduced CoM excursion compared to old adults (4.55 {&#43;/-} 3.45 vs. 10.27 {&#43;/-} 6.62 cm, p &lt; 0.01). Young adults may have exhibited lower fall rates due to the significantly higher acceleration of their arms compared to old adults.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571823">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571823" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571823">
        <p class="paperTitle">Structure-conditioned masked language models for protein sequence design generalize beyond the native sequence space</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571823" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571823" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Akpinaroglu, D.; Seki, K.; Guo, A.; Zhu, E.; Kelly, M. J.; Kortemme, T.</p>
        <p class="info">Score: 18.3, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571823' target='https://doi.org/10.1101/2023.12.15.571823'> 10.1101/2023.12.15.571823</a></p>
        <p class="abstract">Machine learning has revolutionized computational protein design, enabling significant progress in protein backbone generation and sequence design. Here, we introduce Frame2seq, a structure-conditioned masked language model for protein sequence design. Frame2seq generates sequences in a single pass, achieves 49.1% sequence recovery on the CATH 4.2 test dataset, and accurately estimates the error in its own predictions, outperforming the autoregressive ProteinMPNN model with over six times faster inference. To probe the ability of Frame2seq to generate novel designs beyond the native-like sequence space it was trained on, we experimentally test 26 Frame2seq designs for de novo backbones with low identity to the starting sequences. We show that Frame2seq successfully designs soluble (22/26), monomeric, folded, and stable proteins (17/26), including a design with 0% sequence identity to native. The speed and accuracy of Frame2seq will accelerate exploration of novel sequence space across diverse design tasks, including challenging applications such as multi-objective optimization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571602">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571602" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571602">
        <p class="paperTitle">TopoDiff: Improving Protein Backbone Generationwith Topology-aware Latent Encoding</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571602" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571602" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Y.; Ma, Z.; Gong, H.</p>
        <p class="info">Score: 7.0, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571602' target='https://doi.org/10.1101/2023.12.13.571602'> 10.1101/2023.12.13.571602</a></p>
        <p class="abstract">The de novo design of protein structures is an intriguing research topic in the field of protein engineering. Recent breakthroughs in diffusion-based generative models have demonstrated substantial promise in tackling this task, notably in the generation of diverse and realistic protein structures. While existing models predominantly focus on unconditional generation or fine-grained conditioning at the residue level, the holistic, top-down approaches to control the overall topological arrangements are still insufficiently explored. In response, we introduce TopoDiff, a diffusion-based framework augmented by a global-structure encoding module, which is capable of unsupervisedly learning a compact latent representation of natural protein topologies with interpretable characteristics and simultaneously harnessing this learned information for controllable protein structure generation. We also propose a novel metric specifically designed to assess the coverage of sampled proteins with respect to the natural protein space. In comparative analyses with existing models, our generative model not only demonstrates comparable performance on established metrics but also exhibits better coverage across the recognized topology landscape. In summary, TopoDiff emerges as a novel solution towards enhancing the controllability and comprehensiveness of de novo protein structure generation, presenting new possibilities for innovative applications in protein engineering and beyond.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571328">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571328" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571328">
        <p class="paperTitle">A high-fidelity CRISPR-Cas13 system improves abnormalities associated with C9ORF72-linked ALS/FTD</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571328" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571328" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McCallister, T. X.; Lim, C. K. W.; Terpstra, W. M.; C., M. A. Z.; Zhang, S.; Powell, J. E.; Gaj, T.</p>
        <p class="info">Score: 5.8, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571328' target='https://doi.org/10.1101/2023.12.12.571328'> 10.1101/2023.12.12.571328</a></p>
        <p class="abstract">An abnormal expansion of a GGGGCC hexanucleotide repeat in the C9ORF72 gene is the most common genetic cause of amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD), two debilitating neurodegenerative disorders driven in part by gain-of-function mechanisms involving transcribed forms of the repeat expansion. By utilizing a Cas13 variant with reduced collateral effects, we developed a high-fidelity RNA-targeting CRISPR-based system for C9ORF72-linked ALS/FTD. When delivered to the brain of a transgenic rodent model, this Cas13-based platform effectively curbed the expression of the GGGGCC repeat-containing RNA without affecting normal C9ORF72 levels, which in turn decreased the formation of RNA foci and reversed transcriptional deficits. This high-fidelity Cas13 variant possessed improved transcriptome-wide specificity compared to its native form and mediated efficient targeting in motor neuron-like cells derived from a patient with ALS. Our results lay the foundation for the implementation of RNA-targeting CRISPR technologies for C9ORF72-linked ALS/FTD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571704">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571704" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571704">
        <p class="paperTitle">An Integrated Optogenetic and Bioelectronic Platform for Regulating Cardiomyocyte Function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571704" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571704" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bolonduro, O. A.; Chen, Z.; Lai, Y.-R.; Cote, M.; Rao, A. A.; Liu, H.; Tzanakakis, E. S.; Timko, B. P.</p>
        <p class="info">Score: 3.6, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571704' target='https://doi.org/10.1101/2023.12.15.571704'> 10.1101/2023.12.15.571704</a></p>
        <p class="abstract">We report an integrated optogenetic and bioelectronic platform for stable and long-term modulation and monitoring of cardiomyocyte function in vitro. Optogenetic inputs were achieved through expression of a photoactivatable adenylyl cyclase (bPAC), that when activated by blue light caused a dose-dependent and time-limited increase in autonomous cardiomyocyte beat rate. Bioelectronic readouts were achieved through an integrated planar multi-electrode array (MEA) that provided real-time readouts of electrophysiological activity from 32 spatially-distinct locations. Irradiation at 27 W/mm2 resulted in a ca. 14% increase in beat rate within 20-25 minutes, which remained stable for at least 2 hours. The beating rate could be cycled through repeated &#34;on&#34; and &#34;off&#34; states, and its magnitude was a monotonic function of irradiation intensity. Our integrated platform opens new avenues in bioelectronic medicine, including closedloop feedback systems, with potential applications for cardiac regulation including arrhythmia diagnosis and intervention.

TeaserA system that integrates optogenetic stimulation and bioelectronic recording capabilities allows for on-demand regulation of cardiac cell function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572135">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572135" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572135">
        <p class="paperTitle">Engineered Protein Nanosheets for the Scale up of Mesenchymal Stem Cell Culture on Bioemulsions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572135" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572135" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bosch-Fortea, M.; Nadal, C.; Chrysanthou, A.; Marciano, D.; Kanso, H.; Nguyen, N.; Gautrot, J.</p>
        <p class="info">Score: 2.6, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572135' target='https://doi.org/10.1101/2023.12.18.572135'> 10.1101/2023.12.18.572135</a></p>
        <p class="abstract">The rapid progress in cell therapies and stem cell technologies requires the development of novel bioprocessing and biomanufacturing pipelines able to cope with the scale up of cell manufacturing. In this respect, microdroplet technologies have already revolutionised the field of biotechnologies, but remain ill-suited to the culture of adherent cells. In this report, we describe the engineering of albumins with cell adhesive peptides for the stabilisation of microdroplets enabling the scale up of mesenchymal stem cell (MSC) expansion. We characterise the modified albumins prior to study their self-assembly at liquid-liquid interfaces via interfacial shear rheology, and mechanical strengthening through the formation of crosslinked nanosheets. The biofunctionalisation of these protein nanosheets is then characterised by fluorescence microscopy. In turn, the ability of the resulting bioactive microdroplets to promote rapid cell adhesion and expansion is examined and the extensive deposition of matrix associated with such cultures is characterised. The culture of MSCs is then scaled up 100 fold, first at the surface of fluorinated oil emulsions, then on plant-based emulsions stabilised by engineered protein nanosheets and the phenotype of resulting cells is characterised. The microdroplet culture system presented displays attractive advantages over existing technologies, in terms of simplicity of processing, compatibility with regulatory expectations and costs of production, and offers exciting opportunities for translation to cell manufacturing, for cell therapies and cultivated meat applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571789">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571789" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571789">
        <p class="paperTitle">pyQCM-BraTaDio: A tool for visualization, data mining, and modeling of Quartz crystal microbalance with dissipation data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571789" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571789" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pardi, B. M.; Ahmed, S. T.; Jonguitud Flores, S.; Flores, W.; Mears, L. L. M.; Yanez Soto, B.; Andresen Eguiluz, R. C.</p>
        <p class="info">Score: 2.5, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571789' target='https://doi.org/10.1101/2023.12.15.571789'> 10.1101/2023.12.15.571789</a></p>
        <p class="abstract">Here, we present a Python based software that allows for the rapid visualization, data mining, and basic model applications of quartz crystal microbalance with dissipation data. Our implementation begins with a Tkinter GUI to prompt the user for all required information, such as file name/location, selection of baseline time, and overtones for visualization (with customization capabilities). These inputs are then fed to a workflow that will use the baseline time to scrub and temporally shift data using the Pandas and Numpy libraries and carry out the plot options for visualization. The last stage consists of an interactive plot, that presents the data and allows the user to select ranges in MatPlotLib-generated panels, followed by application of data models, including Sauerbrey, thin films in liquid, among others, that are carried out with NumPy and SciPy. The implementation of this software allows for simple and expedited data analysis, in lieu of time consuming and labor-intensive spreadsheet analysis.

Metadata

O_TBL View this table:
org.highwire.dtl.DTLVardef@1d8793aorg.highwire.dtl.DTLVardef@ff020corg.highwire.dtl.DTLVardef@1df6ea6org.highwire.dtl.DTLVardef@1d95b67org.highwire.dtl.DTLVardef@ffec0b_HPS_FORMAT_FIGEXP  M_TBL C_TBL</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571206">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571206" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571206">
        <p class="paperTitle">Adaptive LDA classifier enhances real-time control of an EEG Brain-computer interface for imagined-speech decoding</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571206" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571206" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wu, S.; Bhadra, K.; Giraud, A.-L.; Marchesotti, S.</p>
        <p class="info">Score: 2.5, Published: 2023-12-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571206' target='https://doi.org/10.1101/2023.12.12.571206'> 10.1101/2023.12.12.571206</a></p>
        <p class="abstract">Brain-Computer Interfaces (BCI) aim to establish a pathway between the brain and an external device without the involvement of the motor system, relying exclusively on neural signals. Such systems have the potential to provide a means of communication for patients who have lost the ability to speak due to a neurological disorder. Traditional methodologies for decoding imagined speech directly from brain signals often deploy static classifiers, that is, decoders that are computed once at the beginning of the experiment and that remain unchanged throughout the BCI use. However, this approach might be inadequate in effectively handling the non-stationary nature of electroencephalography (EEG) signals and the learning that accompanies BCI use as parameters are expected to change, all the more in a real-time setting. To address this limitation, we have developed an adaptive classifier that updates its parameters based on the incoming data in real time. We first identified optimal parameters (the update coefficient, UC) to be used in an adaptive Linear Discriminant Analysis (LDA) classifier, using a previously recorded EEG dataset, acquired while healthy participants controlled a binary BCI based on imagined syllable decoding. We subsequently tested the effectiveness of this optimization in a real-time BCI-control setting. Twenty healthy participants performed two BCI-control sessions based on the imagery of two syllables, using a static LDA and the other the adaptive LDA classifier, in randomized order. In this real-time BCI-control task, the adaptive classifier led to better performances than the static one. Furthermore, the optimal parameters for the adaptive classifier were closely aligned in both datasets acquired with the same syllable imagery task. These findings highlight the effectiveness and re-liability of adaptive LDA classifiers for real-time imagined speech decoding, and its interest for non-invasive EEG-based BCI notably characterized by low decoding accuracies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571417">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571417" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571417">
        <p class="paperTitle">Battery-Free Optoelectronic Patch for Photodynamic and Light Therapies in Treating Bacteria-Infected Wounds</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571417" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571417" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xue, Z.; Chou, W.; Xu, Y.; Cheng, Z.; Ren, X.; Sun, T.; Tong, W.; Xie, Y.; Chen, J.; Zhang, N.; Sheng, X.; Wang, Y.; Zhao, H.; Yang, J.; Ding, H.</p>
        <p class="info">Score: 2.2, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571417' target='https://doi.org/10.1101/2023.12.13.571417'> 10.1101/2023.12.13.571417</a></p>
        <p class="abstract">Light therapy is an effective approach for the treatment of a variety of challenging dermatological conditions. In contrast to existing methods involving high doses and large areas of illumination, alternative strategies based on wearable designs that utilize a low light dose over an extended period provide a precise and convenient treatment. In this study, we present a battery-free, skin-integrated optoelectronic patch that incorporates a coil-powered circuit, an array of microscale violet and red light emitting diodes (LEDs), and polymer microneedles (MNs) loaded with 5-aminolevulinic acid (5-ALA). This polymer MNs, based on the biodegradable composite materials of polyvinyl alcohol (PVA) and hyaluronic acid (HA), serves as light waveguides for optical access and a medium for drug release into deeper skin layers. Unlike conventional clinical photomedical appliances with a rigid and fixed light source, this flexible design allows for a conformable light source that can be applied directly to the skin. In animal models with bacterial-infected wounds, the experimental group with the combination treatment of metronomic photodynamic and light therapies reduced 2.48 log10 CFU mL-1 in bactericidal level compared to the control group, indicating an effective anti-infective response. Furthermore, post-treatment analysis revealed the activation of proregenerative genes in monocyte and macrophage cell populations, suggesting enhanced tissue regeneration, neovascularization, and dermal recovery. Overall, this optoelectronic patch design broadens the scope for targeting deep skin lesions, and provides an alternative with the functionality of standard clinical light therapy methods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571721">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571721" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571721">
        <p class="paperTitle">NAD: Noise-augmented direct sequencing of target nucleic acids by augmenting with noise and selective sampling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571721" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571721" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shim, H.</p>
        <p class="info">Score: 2.0, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571721' target='https://doi.org/10.1101/2023.12.14.571721'> 10.1101/2023.12.14.571721</a></p>
        <p class="abstract">Next-generation sequencing necessitates a minimum quantity and concentration of DNA/RNA samples, typically achieved through amplification using the PCR technique. However, this amplification step introduces several drawbacks to biological insights, including PCR bias and the loss of epigenetic information. The advent of long-read sequencing technologies facilitates direct sequencing, with the primary constraint being the limited amount of DNA/RNA present in biological samples. Here, we present a novel method called Noise-Augmented Direct (NAD) sequencing that enables the direct sequencing of target DNA even when it falls below the minimum quantity and concentration required for long-read sequencing by augmenting with noise DNA and adaptive sampling. Adaptive sampling is an emerging technology of nanopore sequencing, allowing the enhanced sequencing of target DNA by selectively depleting noise DNA. In this study, we use the DNA standard of the Lambda phage genome as the noise DNA to augment samples containing low amounts of bacterial genomes (1 ng to 300 ng). The results with cost-effective flow cells indicate that NAD sequencing successfully detects the target DNA with an input quantity as low as 1 ng, and the bacterial genome of Salmonella enterica can be assembled to 30% completion at an accuracy of 98% with an input quantity of 3 ng. With high throughput flow cells, the bacterial genome of Pseudonomas aeruginas was assembled to near completion (99.9%) at an accuracy of 99.97% with an input quantity of 300 ng. This proof-of-concept study demonstrates the potential of NAD sequencing in enhancing the robustness of long-read sequencing with small input DNA/RNA samples with noise augmentation and adaptive sampling.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
