<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Complete substitution with modified nucleotides suppresses the early interferon response and increases the potency of self-amplifying RNA
Authors: McGee, J. E.; Kirsch, J. R.; Kenney, D.; Chavez, E.; Shih, T.-Y.; Douam, F.; Wong, W. W.; Grinstaff, M. W.
Score: 25.3, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557994
Self-amplifying RNA (saRNA) will revolutionize vaccines and in situ therapeutics by enabling protein expression for longer duration at lower doses. However, a major barrier to saRNA efficacy is the potent early interferon response triggered upon cellular entry, resulting in saRNA degradation and translational inhibition.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Complete substitution with modified nucleotides suppresses the early interferon response and increases the potency of self-amplifying RNA
Authors: McGee, J. E.; Kirsch, J. R.; Kenney, D.; Chavez, E.; Shih, T.-Y.; Douam, F.; Wong, W. W.; Grinstaff, M. W.
Score: 25.3, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557994
Self-amplifying RNA (saRNA) will revolutionize vaccines and in situ therapeutics by enabling protein expression for longer duration at lower doses. However, a major barrier to saRNA efficacy is the potent early interferon response triggered upon cellular entry, resulting in saRNA degradation and translational inhibition." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-24T10:37:06+00:00" />
<meta property="article:modified_time" content="2023-09-24T10:37:06+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Complete substitution with modified nucleotides suppresses the early interferon response and increases the potency of self-amplifying RNA
Authors: McGee, J. E.; Kirsch, J. R.; Kenney, D.; Chavez, E.; Shih, T.-Y.; Douam, F.; Wong, W. W.; Grinstaff, M. W.
Score: 25.3, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557994
Self-amplifying RNA (saRNA) will revolutionize vaccines and in situ therapeutics by enabling protein expression for longer duration at lower doses. However, a major barrier to saRNA efficacy is the potent early interferon response triggered upon cellular entry, resulting in saRNA degradation and translational inhibition."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Complete substitution with modified nucleotides suppresses the early interferon response and increases the potency of self-amplifying RNA\nAuthors: McGee, J. E.; Kirsch, J. R.; Kenney, D.; Chavez, E.; Shih, T.-Y.; Douam, F.; Wong, W. W.; Grinstaff, M. W.\nScore: 25.3, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557994\nSelf-amplifying RNA (saRNA) will revolutionize vaccines and in situ therapeutics by enabling protein expression for longer duration at lower doses. However, a major barrier to saRNA efficacy is the potent early interferon response triggered upon cellular entry, resulting in saRNA degradation and translational inhibition.",
  "keywords": [
    
  ],
  "articleBody": " Complete substitution with modified nucleotides suppresses the early interferon response and increases the potency of self-amplifying RNA\nAuthors: McGee, J. E.; Kirsch, J. R.; Kenney, D.; Chavez, E.; Shih, T.-Y.; Douam, F.; Wong, W. W.; Grinstaff, M. W.\nScore: 25.3, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557994\nSelf-amplifying RNA (saRNA) will revolutionize vaccines and in situ therapeutics by enabling protein expression for longer duration at lower doses. However, a major barrier to saRNA efficacy is the potent early interferon response triggered upon cellular entry, resulting in saRNA degradation and translational inhibition. Substitution of mRNA with modified nucleotides (modNTPs), such as N1-methylpseudouridine (N1m{Psi}), reduce the interferon response and enhance expression levels. Multiple attempts to use modNTPs in saRNA have been unsuccessful, leading to the conclusion that modNTPs are incompatible with saRNA, thus hindering further development. Here, contrary to the common dogma in the field, we identify multiple modNTPs that when incorporated into saRNA at 100% substitution confer immune evasion and enhance expression potency. Transfection efficiency enhances by roughly an order of magnitude in difficult to transfect cell types compared to unmodified saRNA, and interferon production reduces by \u003e8 fold compared to unmodified saRNA in human peripheral blood mononuclear cells (PBMCs). Furthermore, we demonstrate expression of viral antigens in vitro and observe significant protection against lethal challenge with a mouse-adapted SARS-CoV-2 strain in vivo. A modified saRNA vaccine, at 100-fold lower dose than a modified mRNA vaccine, results in a statistically improved performance to unmodified saRNA and statistically equivalent performance to modified mRNA. This discovery considerably broadens the potential scope of self-amplifying RNA, enabling entry into previously impossible cell types, as well as the potential to apply saRNA technology to non-vaccine modalities such as cell therapy and protein replacement.\nACE2-Coated Virus-Like Particles Effectively Block SARS-CoV-2 Infection\nAuthors: Bayraktar, C.; Kayabolen, A.; Odabas, A.; Durgun, A.; Kok, I.; Sevinc, K.; Supramaniam, A.; Idris, A.; Bagci-Onder, T.\nScore: 22.5, Published: 2023-09-20 DOI: 10.1101/2023.09.19.558424\nA large body of research accumulated over the past three years dedicated to our understanding and fighting COVID-19. Blocking the interaction between SARS-CoV-2 Spike and ACE2 receptor has been considered an effective strategy as anti-SARS-CoV-2 therapeutics. In this study, we developed ACE2-coated virus-like particles (ACE2-VLPs), which can be utilized to prevent viral entry into host cells and efficiently neutralize the virus. These ACE2-VLPs exhibited high neutralization capacity even when applied at low doses, and displayed superior efficacy compared to extracellular vesicles carrying ACE2, in the in vitro pseudoviral assays. ACE2-VLPs were stable under different environmental temperatures, and they were effective in blocking all tested variants of concern in vitro. Finally, ACE2-VLPs displayed marked neutralization capacity against Omicron BA.1 in the Vero E6 cells. Based on their superior efficacy compared to extracellular vesicles, and their demonstrated success against live virus, ACE2-VLPs can be considered as vital candidates for treating SARS-CoV-2. This novel therapeutic approach of VLP coating with receptor particles can serve as proof-of-concept for designing effective neutralization strategies for other viral diseases in the future. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=78 SRC=\"FIGDIR/small/558424v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (16K): org.highwire.dtl.DTLVardef@1a50421org.highwire.dtl.DTLVardef@b21ea7org.highwire.dtl.DTLVardef@1fa905borg.highwire.dtl.DTLVardef@11b5ceb_HPS_FORMAT_FIGEXP M_FIG In our study, we demonstrate the prevention of SARS-CoV-2 infection through the use of Ace2-coated VLPs. C_FIG\nClick editing enables programmable genome writing using DNA polymerases and HUH endonucleases\nAuthors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.\nScore: 95.2, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557440\nGenome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e.g. CRISPR-Cas) to permit the installation of a range of edits including substitutions, insertions, and deletions. Click editors (CEs) leverage the \"click\"-like bioconjugation ability of HUH endonucleases (HUHes) with single stranded DNA substrates to covalently tether \"click DNA\" (clkDNA) templates encoding user-specifiable edits at targeted genomic loci. Through iterative optimization of the modular components of CEs (DNA polymerase and HUHe orthologs, architectural modifications, etc.) and their clkDNAs (template configurations, repair evading substitutions, etc.), we demonstrate the ability to install precise genome edits with minimal indels and no unwanted byproduct insertions. Since clkDNAs can be ordered as simple DNA oligonucleotides for cents per base, it is possible to screen many different clkDNA parameters rapidly and inexpensively to maximize edit efficiency. Together, click editing is a precise and highly versatile platform for modifying genomes with a simple workflow and broad utility across diverse biological applications.\nProtein generation with evolutionary diffusion: sequence is all you need\nAuthors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.\nScore: 186.8, Published: 2023-09-12 DOI: 10.1101/2023.09.11.556673\nDeep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design. However, current state-of-the-art models generate protein structures, which limits the scope of their training data and restricts generations to a small and biased subset of protein design space. Here, we introduce a general-purpose diffusion framework, EvoDiff, that combines evolutionary-scale data with the distinct conditioning capabilities of diffusion models for controllable protein generation in sequence space. EvoDiff generates high-fidelity, diverse, and structurally-plausible proteins that cover natural sequence and functional space. Critically, EvoDiff can generate proteins inaccessible to structure-based models, such as those with disordered regions, while maintaining the ability to design scaffolds for functional structural motifs, demonstrating the universality of our sequence-based formulation. We envision that EvoDiff will expand capabilities in protein engineering beyond the structure-function paradigm toward programmable, sequence-first design.\nRapid optogenetic clustering of a cytoplasmic BcLOV4 variant\nAuthors: Huang, Z.; Benman, W.; Dong, L.; Bugaj, L. J.\nScore: 11.8, Published: 2023-09-17 DOI: 10.1101/2023.09.14.557726\nProtein clustering is a powerful form of optogenetic control, yet there is currently only one protein --Cry2--whose light-induced clustering has been harnessed for these purposes. Recently, the photoreceptor BcLOV4 was found to form protein clusters in mammalian cells in response to blue light, although clustering coincided with its translocation to the plasma membrane, potentially constraining its application as an optogenetic clustering module. Herein we identify key amino acids that couple clustering to membrane binding, allowing us to engineer a variant of BcLOV4 that clusters in the cytoplasm and does not associate with the membrane in response to blue light. This variant, BcLOVclust, clustered over many cycles with dramatically faster clustering and de-clustering kinetics compared to Cry2. The magnitude of BcLOVclust clustering could be strengthened by appending an intrinsically disordered region from the fused in sarcoma (FUS) protein, or by optimizing the fluorescent protein to which it was fused. BcLOVclust retained the temperature sensitivity of BcLOV4 such that light induced clustering was transient, and the rate of spontaneous declustering increased with temperature. At low temperatures, BcLOVclust and Cry2 could be multiplexed in the same cells, allowing light control of independent protein condensates. BcLOVclust could also be applied to control signaling proteins and stress granules in mammalian cells. Thus BcLOVclust provides an alternative to Cry2 for optogenetic clustering and a method for multiplexed clustering. While its usage is currently suited for organisms that can be cultured below [~]30 {degrees}C, a deeper understanding of BcLOVclust thermal response will further enable its use at physiological mammalian temperatures. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=163 SRC=\"FIGDIR/small/557726v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (31K): org.highwire.dtl.DTLVardef@10b7de8org.highwire.dtl.DTLVardef@17732e2org.highwire.dtl.DTLVardef@add2c1org.highwire.dtl.DTLVardef@1e7a541_HPS_FORMAT_FIGEXP M_FIG C_FIG HighlightsO_LILight-responsive clustering of BcLOV4 can be decoupled from its membrane association C_LIO_LIBcLOVclust clusters in the cytoplasm with faster ON and OFF kinetics than Cry2 C_LIO_LIBcLOVclust and Cry2 can be multiplexed in the same cell C_LIO_LIBcLOVclust can control RhoA activity and stress granule formation C_LI\nHigh-affinity binding to the SARS-CoV-2 spike trimer by a nanostructured, trivalent protein-DNA synthetic antibody\nAuthors: Xu, Y.; Zheng, R.; Prasad, A.; Liu, M.; Wan, Z.; Zhou, X.; Porter, R. M.; Sample, M.; Poppleton, E.; Procyk, J.; Liu, H.; Li, Y.; Wang, S.; Yan, H.; Sulc, P.; Stephanopoulos, N.\nScore: 10.3, Published: 2023-09-19 DOI: 10.1101/2023.09.18.558353\nMultivalency enables nanostructures to bind molecular targets with high affinity. Although antibodies can be generated against a wide range of antigens, their shape and size cannot be tuned to match a given target. DNA nanotechnology provides an attractive approach for designing customized multivalent scaffolds due to the addressability and programmability of the nanostructure shape and size. Here, we design a nanoscale synthetic antibody (\"nano-synbody\") based on a three-helix bundle DNA nanostructure with one, two, or three identical arms terminating in a mini-binder protein that targets the SARS-CoV-2 spike protein. The nano-synbody was designed to match the valence and distance between the three receptor binding domains (RBDs) in the spike trimer, in order to enhance affinity. The protein-DNA nano-synbody shows tight binding to the wild-type, Delta, and several Omicron variants of the SARS-CoV-2 spike trimer, with affinity increasing as the number of arms increases from one to three. The effectiveness of the nano-synbody was also verified using a pseudovirus neutralization assay, with the three-arm nanostructure inhibiting two Omicron variants against which the structures with only one or two arms are ineffective. The structure of the three-arm nano-synbody bound to the Omicron variant spike trimer was solved by negative-stain transmission electron microscopy reconstruction, and shows the protein-DNA nanostructure with all three arms attached to the RBD domains, confirming the intended trivalent attachment. The ability to tune the size and shape of the nano-synbody, as well as its potential ability to attach two or more different binding ligands, will enable the high-affinity targeting of a range of proteins not possible with traditional antibodies.\nExtracellular matrix scaffold-assisted tumor vaccines induce tumor regression and long-term immune memory\nAuthors: Pal, S.; Chaudhari, R.; Baurceanu, I.; Hill, B. J.; Nagy, B. A.; Wolf, M.\nScore: 11.9, Published: 2023-09-15 DOI: 10.1101/2023.09.12.557449\nInjectable scaffold delivery is an immune engineering strategy to enhance the efficacy and reliability of cancer vaccine immunotherapy. The composition and structure of the biomaterial scaffold determines both vaccine release kinetics and inherent immune stimulation via the scaffold host response. Extracellular matrix (ECM) scaffolds prepared from decellularized tissues initiate an acute alternative inflammatory response following implantation, which facilitates wound healing following tumor resection and promotes local cancer immune surveillance. However, it remains unknown whether this environment is compatible with generating protective anti-tumor cytotoxic immunity with local immunotherapy delivery. Here, we engineered an ECM scaffold-assisted therapeutic cancer vaccine that maintained an immune microenvironment consistent with tissue reconstruction. Immune adjuvants MPLA, GM-CSF, and CDA were screened in a cancer vaccine formulated for decellularized small intestinal submucosa (SIS) ECM scaffold co-delivery. Though MPLA and GM-CSF showed the greatest increase in local myeloid cell infiltration, we found that the STING pathway adjuvant CDA was the most potent inducer of cytotoxic immunity with SIS-ECM scaffold delivery. Further, CDA did not diminish hallmark ECM immune responses needed in wound healing such as high Il4 cytokine expression. SIS scaffold delivery enhanced therapeutic vaccine efficacy using CDA and the antigen ovalbumin, curing greater than 50% of established EG.7 tumors in young mice and 75% in 24-week-old mature mice, compared to soluble components alone (0% cured). SIS-ECM scaffold assisted vaccination extended antigen exposure, was dependent on CD8+ cytotoxic T cells, and generated long term anti-tumor memory at least 7 months post-vaccination in both young and mature-aged mice. This study shows that an ECM scaffold is a promising delivery vehicle to enhance cancer vaccine efficacy while being orthogonal to characteristics of pro-healing immune hallmarks.\nEngineered transcription-associated Cas9 targeting in eukaryotic cells\nAuthors: Goldberg, G. W.; Kogenaru, M.; Keegan, S.; Haase, M. A. B.; Kagermazova, L.; Arias, M. A.; Onyebeke, K.; Adams, S.; Fenyo, D.; Noyes, M. B.; Boeke, J. D.\nScore: 7.5, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558319\nDNA targeting Class 2 CRISPR-Cas effector nucleases, including the well-studied Cas9 proteins, evolved protospacer-adjacent motif (PAM) and guide RNA interactions that sequentially license their binding and cleavage activities at protospacer target sites. Both interactions are nucleic acid sequence specific but function constitutively; thus, they provide intrinsic spatial control over DNA targeting activities but naturally lack temporal control. Here we show that engineered Cas9 fusion proteins which bind to nascent RNAs near a protospacer can facilitate spatiotemporal coupling between transcription and DNA targeting at that protospacer: Transcription-associated Cas9 Targeting (TraCT). Engineered TraCT is enabled when suboptimal PAM interactions limit basal activity in vivo and when one or more nascent RNA substrates are still tethered to the actively transcribing target DNA in cis. We further show that this phenomenon can be exploited for selective editing at one of two identical targets in distinct gene loci, or, in diploid allelic loci that are differentially transcribed. Our work demonstrates that temporal control over Cas9s targeting activity at specific DNA sites may be engineered without modifying Cas9s core domains and guide RNA components or their expression levels. More broadly, it establishes RNA binding in cis as a mechanism that can conditionally stimulate CRISPR-Cas DNA targeting in eukaryotes.\nOptimal Sampling Rate for 3D Single Molecule Localization\nAuthors: Chang, H.; Fu, S.; Li, Y.\nScore: 6.4, Published: 2023-09-21 DOI: 10.1101/2023.09.19.558416\nResolution of single molecule localization microscopy (SMLM) depends on the localization accuracy, which can be improved by utilizing engineered point spread functions (PSF) with delicate shapes. However, the intrinsic pixelation effect of the detector sensor will deteriorate PSFs under different sampling rates. The influence of the pixelation effect to the achieved 3D localization accuracy for different PSF shapes under different signal to background ratio (SBR) and pixel dependent readout noise has not been investigated in detail so far. In this work, we proposed a framework to characterize the 3D localization accuracy of pixelated PSF at different sampling rates. Four different PSFs (astigmatic PSF, double helix (DH) PSF, Tetrapod PSF and 4Pi PSF) were evaluated and the pixel size with optimal 3D localization performance were derived. This work provides a theoretical guide for the optimal design of sampling rate for 3D super resolution imaging.\nSingle-cell Herpes Simplex Virus type-1 infection of neurons using drop-based microfluidics reveals heterogeneous replication kinetics\nAuthors: Fredrikson, J. P.; Domanico, L. F.; Pratt, S. L.; Loveday, E. K.; Taylor, M. P.; Chang, C. B.\nScore: 5.1, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558333\nSingle-cell analyses of viral infections often reveal heterogeneity that is not detected by traditional population-level studies. This study applies drop-based microfluidics to investigate the dynamics of HSV-1 infection of neurons at the single-cell level. We used micron-scale Matrigel beads, termed microgels, to culture individual murine Superior Cervical ganglia (SCG) neurons or epithelial cells. Microgel-cultured cells are subsequently enclosed in individual media-in-oil droplets with a dual fluorescent-reporter HSV-1, enabling real-time observation of viral gene expression and replication. Infection within drops revealed that the kinetics of initial viral gene expression and replication were dependent on the inoculating dose. Notably, increasing inoculating doses led to earlier onset of viral gene expression and more frequent productive viral replication. These observations provide crucial insights into the complexity of HSV-1 infection in neurons and emphasize the importance of studying single-cell outcomes of viral infection. The innovative techniques presented here for cell culture and infection in drops provide a foundation for future virology and neurobiology investigations.\n",
  "wordCount" : "2515",
  "inLanguage": "en",
  "datePublished": "2023-09-24T10:37:06Z",
  "dateModified": "2023-09-24T10:37:06Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on September 24, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557994">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557994" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557994">
        <p class="paperTitle">Complete substitution with modified nucleotides suppresses the early interferon response and increases the potency of self-amplifying RNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557994" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557994" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McGee, J. E.; Kirsch, J. R.; Kenney, D.; Chavez, E.; Shih, T.-Y.; Douam, F.; Wong, W. W.; Grinstaff, M. W.</p>
        <p class="info">Score: 25.3, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557994' target='https://doi.org/10.1101/2023.09.15.557994'> 10.1101/2023.09.15.557994</a></p>
        <p class="abstract">Self-amplifying RNA (saRNA) will revolutionize vaccines and in situ therapeutics by enabling protein expression for longer duration at lower doses. However, a major barrier to saRNA efficacy is the potent early interferon response triggered upon cellular entry, resulting in saRNA degradation and translational inhibition. Substitution of mRNA with modified nucleotides (modNTPs), such as N1-methylpseudouridine (N1m{Psi}), reduce the interferon response and enhance expression levels. Multiple attempts to use modNTPs in saRNA have been unsuccessful, leading to the conclusion that modNTPs are incompatible with saRNA, thus hindering further development. Here, contrary to the common dogma in the field, we identify multiple modNTPs that when incorporated into saRNA at 100% substitution confer immune evasion and enhance expression potency. Transfection efficiency enhances by roughly an order of magnitude in difficult to transfect cell types compared to unmodified saRNA, and interferon production reduces by &gt;8 fold compared to unmodified saRNA in human peripheral blood mononuclear cells (PBMCs). Furthermore, we demonstrate expression of viral antigens in vitro and observe significant protection against lethal challenge with a mouse-adapted SARS-CoV-2 strain in vivo. A modified saRNA vaccine, at 100-fold lower dose than a modified mRNA vaccine, results in a statistically improved performance to unmodified saRNA and statistically equivalent performance to modified mRNA. This discovery considerably broadens the potential scope of self-amplifying RNA, enabling entry into previously impossible cell types, as well as the potential to apply saRNA technology to non-vaccine modalities such as cell therapy and protein replacement.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558424">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558424" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558424">
        <p class="paperTitle">ACE2-Coated Virus-Like Particles Effectively Block SARS-CoV-2 Infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558424" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558424" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bayraktar, C.; Kayabolen, A.; Odabas, A.; Durgun, A.; Kok, I.; Sevinc, K.; Supramaniam, A.; Idris, A.; Bagci-Onder, T.</p>
        <p class="info">Score: 22.5, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558424' target='https://doi.org/10.1101/2023.09.19.558424'> 10.1101/2023.09.19.558424</a></p>
        <p class="abstract">A large body of research accumulated over the past three years dedicated to our understanding and fighting COVID-19. Blocking the interaction between SARS-CoV-2 Spike and ACE2 receptor has been considered an effective strategy as anti-SARS-CoV-2 therapeutics. In this study, we developed ACE2-coated virus-like particles (ACE2-VLPs), which can be utilized to prevent viral entry into host cells and efficiently neutralize the virus. These ACE2-VLPs exhibited high neutralization capacity even when applied at low doses, and displayed superior efficacy compared to extracellular vesicles carrying ACE2, in the in vitro pseudoviral assays. ACE2-VLPs were stable under different environmental temperatures, and they were effective in blocking all tested variants of concern in vitro. Finally, ACE2-VLPs displayed marked neutralization capacity against Omicron BA.1 in the Vero E6 cells. Based on their superior efficacy compared to extracellular vesicles, and their demonstrated success against live virus, ACE2-VLPs can be considered as vital candidates for treating SARS-CoV-2. This novel therapeutic approach of VLP coating with receptor particles can serve as proof-of-concept for designing effective neutralization strategies for other viral diseases in the future.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=78 SRC=&#34;FIGDIR/small/558424v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (16K):
org.highwire.dtl.DTLVardef@1a50421org.highwire.dtl.DTLVardef@b21ea7org.highwire.dtl.DTLVardef@1fa905borg.highwire.dtl.DTLVardef@11b5ceb_HPS_FORMAT_FIGEXP  M_FIG In our study, we demonstrate the prevention of SARS-CoV-2 infection through the use of Ace2-coated VLPs.

C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557440">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557440" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557440">
        <p class="paperTitle">Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557440" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557440" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: da Silva, J. F.; Tou, C. J.; King, E. M.; Eller, M. L.; Ma, L.; Rufino-Ramos, D.; Kleinstiver, B. P.</p>
        <p class="info">Score: 95.2, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557440' target='https://doi.org/10.1101/2023.09.12.557440'> 10.1101/2023.09.12.557440</a></p>
        <p class="abstract">Genome editing technologies that install diverse edits can widely enable genetic studies and new therapeutics. Here we develop click editing, a genome writing platform that couples the advantageous properties of DNA-dependent DNA polymerases with RNA-programmable nickases (e.g. CRISPR-Cas) to permit the installation of a range of edits including substitutions, insertions, and deletions. Click editors (CEs) leverage the &#34;click&#34;-like bioconjugation ability of HUH endonucleases (HUHes) with single stranded DNA substrates to covalently tether &#34;click DNA&#34; (clkDNA) templates encoding user-specifiable edits at targeted genomic loci. Through iterative optimization of the modular components of CEs (DNA polymerase and HUHe orthologs, architectural modifications, etc.) and their clkDNAs (template configurations, repair evading substitutions, etc.), we demonstrate the ability to install precise genome edits with minimal indels and no unwanted byproduct insertions. Since clkDNAs can be ordered as simple DNA oligonucleotides for cents per base, it is possible to screen many different clkDNA parameters rapidly and inexpensively to maximize edit efficiency. Together, click editing is a precise and highly versatile platform for modifying genomes with a simple workflow and broad utility across diverse biological applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.556673">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.556673" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.556673">
        <p class="paperTitle">Protein generation with evolutionary diffusion: sequence is all you need</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.556673" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.556673" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.</p>
        <p class="info">Score: 186.8, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.556673' target='https://doi.org/10.1101/2023.09.11.556673'> 10.1101/2023.09.11.556673</a></p>
        <p class="abstract">Deep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design. However, current state-of-the-art models generate protein structures, which limits the scope of their training data and restricts generations to a small and biased subset of protein design space. Here, we introduce a general-purpose diffusion framework, EvoDiff, that combines evolutionary-scale data with the distinct conditioning capabilities of diffusion models for controllable protein generation in sequence space. EvoDiff generates high-fidelity, diverse, and structurally-plausible proteins that cover natural sequence and functional space. Critically, EvoDiff can generate proteins inaccessible to structure-based models, such as those with disordered regions, while maintaining the ability to design scaffolds for functional structural motifs, demonstrating the universality of our sequence-based formulation. We envision that EvoDiff will expand capabilities in protein engineering beyond the structure-function paradigm toward programmable, sequence-first design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557726">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557726" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557726">
        <p class="paperTitle">Rapid optogenetic clustering of a cytoplasmic BcLOV4 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557726" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557726" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, Z.; Benman, W.; Dong, L.; Bugaj, L. J.</p>
        <p class="info">Score: 11.8, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557726' target='https://doi.org/10.1101/2023.09.14.557726'> 10.1101/2023.09.14.557726</a></p>
        <p class="abstract">Protein clustering is a powerful form of optogenetic control, yet there is currently only one protein --Cry2--whose light-induced clustering has been harnessed for these purposes. Recently, the photoreceptor BcLOV4 was found to form protein clusters in mammalian cells in response to blue light, although clustering coincided with its translocation to the plasma membrane, potentially constraining its application as an optogenetic clustering module. Herein we identify key amino acids that couple clustering to membrane binding, allowing us to engineer a variant of BcLOV4 that clusters in the cytoplasm and does not associate with the membrane in response to blue light. This variant, BcLOVclust, clustered over many cycles with dramatically faster clustering and de-clustering kinetics compared to Cry2. The magnitude of BcLOVclust clustering could be strengthened by appending an intrinsically disordered region from the fused in sarcoma (FUS) protein, or by optimizing the fluorescent protein to which it was fused. BcLOVclust retained the temperature sensitivity of BcLOV4 such that light induced clustering was transient, and the rate of spontaneous declustering increased with temperature. At low temperatures, BcLOVclust and Cry2 could be multiplexed in the same cells, allowing light control of independent protein condensates. BcLOVclust could also be applied to control signaling proteins and stress granules in mammalian cells. Thus BcLOVclust provides an alternative to Cry2 for optogenetic clustering and a method for multiplexed clustering. While its usage is currently suited for organisms that can be cultured below [~]30 {degrees}C, a deeper understanding of BcLOVclust thermal response will further enable its use at physiological mammalian temperatures.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=163 SRC=&#34;FIGDIR/small/557726v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (31K):
org.highwire.dtl.DTLVardef@10b7de8org.highwire.dtl.DTLVardef@17732e2org.highwire.dtl.DTLVardef@add2c1org.highwire.dtl.DTLVardef@1e7a541_HPS_FORMAT_FIGEXP  M_FIG C_FIG HighlightsO_LILight-responsive clustering of BcLOV4 can be decoupled from its membrane association
C_LIO_LIBcLOVclust clusters in the cytoplasm with faster ON and OFF kinetics than Cry2
C_LIO_LIBcLOVclust and Cry2 can be multiplexed in the same cell
C_LIO_LIBcLOVclust can control RhoA activity and stress granule formation
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558353">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558353" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558353">
        <p class="paperTitle">High-affinity binding to the SARS-CoV-2 spike trimer by a nanostructured, trivalent protein-DNA synthetic antibody</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558353" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558353" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, Y.; Zheng, R.; Prasad, A.; Liu, M.; Wan, Z.; Zhou, X.; Porter, R. M.; Sample, M.; Poppleton, E.; Procyk, J.; Liu, H.; Li, Y.; Wang, S.; Yan, H.; Sulc, P.; Stephanopoulos, N.</p>
        <p class="info">Score: 10.3, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558353' target='https://doi.org/10.1101/2023.09.18.558353'> 10.1101/2023.09.18.558353</a></p>
        <p class="abstract">Multivalency enables nanostructures to bind molecular targets with high affinity. Although antibodies can be generated against a wide range of antigens, their shape and size cannot be tuned to match a given target. DNA nanotechnology provides an attractive approach for designing customized multivalent scaffolds due to the addressability and programmability of the nanostructure shape and size. Here, we design a nanoscale synthetic antibody (&#34;nano-synbody&#34;) based on a three-helix bundle DNA nanostructure with one, two, or three identical arms terminating in a mini-binder protein that targets the SARS-CoV-2 spike protein. The nano-synbody was designed to match the valence and distance between the three receptor binding domains (RBDs) in the spike trimer, in order to enhance affinity. The protein-DNA nano-synbody shows tight binding to the wild-type, Delta, and several Omicron variants of the SARS-CoV-2 spike trimer, with affinity increasing as the number of arms increases from one to three. The effectiveness of the nano-synbody was also verified using a pseudovirus neutralization assay, with the three-arm nanostructure inhibiting two Omicron variants against which the structures with only one or two arms are ineffective. The structure of the three-arm nano-synbody bound to the Omicron variant spike trimer was solved by negative-stain transmission electron microscopy reconstruction, and shows the protein-DNA nanostructure with all three arms attached to the RBD domains, confirming the intended trivalent attachment. The ability to tune the size and shape of the nano-synbody, as well as its potential ability to attach two or more different binding ligands, will enable the high-affinity targeting of a range of proteins not possible with traditional antibodies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557449">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557449" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557449">
        <p class="paperTitle">Extracellular matrix scaffold-assisted tumor vaccines induce tumor regression and long-term immune memory</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557449" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557449" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pal, S.; Chaudhari, R.; Baurceanu, I.; Hill, B. J.; Nagy, B. A.; Wolf, M.</p>
        <p class="info">Score: 11.9, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557449' target='https://doi.org/10.1101/2023.09.12.557449'> 10.1101/2023.09.12.557449</a></p>
        <p class="abstract">Injectable scaffold delivery is an immune engineering strategy to enhance the efficacy and reliability of cancer vaccine immunotherapy. The composition and structure of the biomaterial scaffold determines both vaccine release kinetics and inherent immune stimulation via the scaffold host response. Extracellular matrix (ECM) scaffolds prepared from decellularized tissues initiate an acute alternative inflammatory response following implantation, which facilitates wound healing following tumor resection and promotes local cancer immune surveillance. However, it remains unknown whether this environment is compatible with generating protective anti-tumor cytotoxic immunity with local immunotherapy delivery. Here, we engineered an ECM scaffold-assisted therapeutic cancer vaccine that maintained an immune microenvironment consistent with tissue reconstruction. Immune adjuvants MPLA, GM-CSF, and CDA were screened in a cancer vaccine formulated for decellularized small intestinal submucosa (SIS) ECM scaffold co-delivery. Though MPLA and GM-CSF showed the greatest increase in local myeloid cell infiltration, we found that the STING pathway adjuvant CDA was the most potent inducer of cytotoxic immunity with SIS-ECM scaffold delivery. Further, CDA did not diminish hallmark ECM immune responses needed in wound healing such as high Il4 cytokine expression. SIS scaffold delivery enhanced therapeutic vaccine efficacy using CDA and the antigen ovalbumin, curing greater than 50% of established EG.7 tumors in young mice and 75% in 24-week-old mature mice, compared to soluble components alone (0% cured). SIS-ECM scaffold assisted vaccination extended antigen exposure, was dependent on CD8&#43; cytotoxic T cells, and generated long term anti-tumor memory at least 7 months post-vaccination in both young and mature-aged mice. This study shows that an ECM scaffold is a promising delivery vehicle to enhance cancer vaccine efficacy while being orthogonal to characteristics of pro-healing immune hallmarks.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558319">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558319" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558319">
        <p class="paperTitle">Engineered transcription-associated Cas9 targeting in eukaryotic cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558319" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558319" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goldberg, G. W.; Kogenaru, M.; Keegan, S.; Haase, M. A. B.; Kagermazova, L.; Arias, M. A.; Onyebeke, K.; Adams, S.; Fenyo, D.; Noyes, M. B.; Boeke, J. D.</p>
        <p class="info">Score: 7.5, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558319' target='https://doi.org/10.1101/2023.09.18.558319'> 10.1101/2023.09.18.558319</a></p>
        <p class="abstract">DNA targeting Class 2 CRISPR-Cas effector nucleases, including the well-studied Cas9 proteins, evolved protospacer-adjacent motif (PAM) and guide RNA interactions that sequentially license their binding and cleavage activities at protospacer target sites. Both interactions are nucleic acid sequence specific but function constitutively; thus, they provide intrinsic spatial control over DNA targeting activities but naturally lack temporal control. Here we show that engineered Cas9 fusion proteins which bind to nascent RNAs near a protospacer can facilitate spatiotemporal coupling between transcription and DNA targeting at that protospacer: Transcription-associated Cas9 Targeting (TraCT). Engineered TraCT is enabled when suboptimal PAM interactions limit basal activity in vivo and when one or more nascent RNA substrates are still tethered to the actively transcribing target DNA in cis. We further show that this phenomenon can be exploited for selective editing at one of two identical targets in distinct gene loci, or, in diploid allelic loci that are differentially transcribed. Our work demonstrates that temporal control over Cas9s targeting activity at specific DNA sites may be engineered without modifying Cas9s core domains and guide RNA components or their expression levels. More broadly, it establishes RNA binding in cis as a mechanism that can conditionally stimulate CRISPR-Cas DNA targeting in eukaryotes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558416">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558416" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558416">
        <p class="paperTitle">Optimal Sampling Rate for 3D Single Molecule Localization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558416" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558416" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chang, H.; Fu, S.; Li, Y.</p>
        <p class="info">Score: 6.4, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558416' target='https://doi.org/10.1101/2023.09.19.558416'> 10.1101/2023.09.19.558416</a></p>
        <p class="abstract">Resolution of single molecule localization microscopy (SMLM) depends on the localization accuracy, which can be improved by utilizing engineered point spread functions (PSF) with delicate shapes. However, the intrinsic pixelation effect of the detector sensor will deteriorate PSFs under different sampling rates. The influence of the pixelation effect to the achieved 3D localization accuracy for different PSF shapes under different signal to background ratio (SBR) and pixel dependent readout noise has not been investigated in detail so far. In this work, we proposed a framework to characterize the 3D localization accuracy of pixelated PSF at different sampling rates. Four different PSFs (astigmatic PSF, double helix (DH) PSF, Tetrapod PSF and 4Pi PSF) were evaluated and the pixel size with optimal 3D localization performance were derived. This work provides a theoretical guide for the optimal design of sampling rate for 3D super resolution imaging.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558333">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558333" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558333">
        <p class="paperTitle">Single-cell Herpes Simplex Virus type-1 infection of neurons using drop-based microfluidics reveals heterogeneous replication kinetics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558333" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558333" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fredrikson, J. P.; Domanico, L. F.; Pratt, S. L.; Loveday, E. K.; Taylor, M. P.; Chang, C. B.</p>
        <p class="info">Score: 5.1, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558333' target='https://doi.org/10.1101/2023.09.18.558333'> 10.1101/2023.09.18.558333</a></p>
        <p class="abstract">Single-cell analyses of viral infections often reveal heterogeneity that is not detected by traditional population-level studies. This study applies drop-based microfluidics to investigate the dynamics of HSV-1 infection of neurons at the single-cell level. We used micron-scale Matrigel beads, termed microgels, to culture individual murine Superior Cervical ganglia (SCG) neurons or epithelial cells. Microgel-cultured cells are subsequently enclosed in individual media-in-oil droplets with a dual fluorescent-reporter HSV-1, enabling real-time observation of viral gene expression and replication. Infection within drops revealed that the kinetics of initial viral gene expression and replication were dependent on the inoculating dose. Notably, increasing inoculating doses led to earlier onset of viral gene expression and more frequent productive viral replication. These observations provide crucial insights into the complexity of HSV-1 infection in neurons and emphasize the importance of studying single-cell outcomes of viral infection. The innovative techniques presented here for cell culture and infection in drops provide a foundation for future virology and neurobiology investigations.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
