<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Human gut metagenomic mining reveals an untapped source of peptide antibiotics
Authors: Torres, M. D. T.; Brooks, E. F.; Cesaro, A.; Sberro, H.; Nicolaou, C.; Bhatt, A. S.; de la Fuente, C.
Score: 32.0, Published: 2023-09-03 DOI: 10.1101/2023.08.31.555711
Drug-resistant bacteria are outpacing traditional antibiotic discovery efforts. Here, we computationally mined 444,054 families of putative small proteins from 1,773 human gut metagenomes, identifying 323 peptide antibiotics encoded in small open reading frames (smORFs).">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Human gut metagenomic mining reveals an untapped source of peptide antibiotics
Authors: Torres, M. D. T.; Brooks, E. F.; Cesaro, A.; Sberro, H.; Nicolaou, C.; Bhatt, A. S.; de la Fuente, C.
Score: 32.0, Published: 2023-09-03 DOI: 10.1101/2023.08.31.555711
Drug-resistant bacteria are outpacing traditional antibiotic discovery efforts. Here, we computationally mined 444,054 families of putative small proteins from 1,773 human gut metagenomes, identifying 323 peptide antibiotics encoded in small open reading frames (smORFs)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-13T10:39:03+00:00" />
<meta property="article:modified_time" content="2023-09-13T10:39:03+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Human gut metagenomic mining reveals an untapped source of peptide antibiotics
Authors: Torres, M. D. T.; Brooks, E. F.; Cesaro, A.; Sberro, H.; Nicolaou, C.; Bhatt, A. S.; de la Fuente, C.
Score: 32.0, Published: 2023-09-03 DOI: 10.1101/2023.08.31.555711
Drug-resistant bacteria are outpacing traditional antibiotic discovery efforts. Here, we computationally mined 444,054 families of putative small proteins from 1,773 human gut metagenomes, identifying 323 peptide antibiotics encoded in small open reading frames (smORFs)."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Human gut metagenomic mining reveals an untapped source of peptide antibiotics\nAuthors: Torres, M. D. T.; Brooks, E. F.; Cesaro, A.; Sberro, H.; Nicolaou, C.; Bhatt, A. S.; de la Fuente, C.\nScore: 32.0, Published: 2023-09-03 DOI: 10.1101/2023.08.31.555711\nDrug-resistant bacteria are outpacing traditional antibiotic discovery efforts. Here, we computationally mined 444,054 families of putative small proteins from 1,773 human gut metagenomes, identifying 323 peptide antibiotics encoded in small open reading frames (smORFs).",
  "keywords": [
    
  ],
  "articleBody": " Human gut metagenomic mining reveals an untapped source of peptide antibiotics\nAuthors: Torres, M. D. T.; Brooks, E. F.; Cesaro, A.; Sberro, H.; Nicolaou, C.; Bhatt, A. S.; de la Fuente, C.\nScore: 32.0, Published: 2023-09-03 DOI: 10.1101/2023.08.31.555711\nDrug-resistant bacteria are outpacing traditional antibiotic discovery efforts. Here, we computationally mined 444,054 families of putative small proteins from 1,773 human gut metagenomes, identifying 323 peptide antibiotics encoded in small open reading frames (smORFs). To test our computational predictions, 78 peptides were synthesized and screened for antimicrobial activity in vitro, with 59% displaying activity against either pathogens or commensals. Since these peptides were unique compared to previously reported antimicrobial peptides, we termed them smORF-encoded peptides (SEPs). SEPs killed bacteria by targeting their membrane, synergized with each other, and modulated gut commensals, indicating that they may play a role in reconfiguring microbiome communities in addition to counteracting pathogens. The lead candidates were anti-infective in both murine skin abscess and deep thigh infection models. Notably, prevotellin-2 from Prevotella copri presented activity comparable to the commonly used antibiotic polymyxin B. We report the discovery of hundreds of peptide sequences in the human gut.\nProtein generation with evolutionary diffusion: sequence is all you need\nAuthors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.\nScore: 15.4, Published: 2023-09-12 DOI: 10.1101/2023.09.11.556673\nDeep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design. However, current state-of-the-art models generate protein structures, which severely limits the scope of their training data and restricts generations to a small and biased subset of protein design space. Here, we introduce a general-purpose diffusion framework, EvoDiff, that combines evolutionary-scale data with the distinct conditioning capabilities of diffusion models for controllable protein generation in sequence space. EvoDiff generates high-fidelity, diverse, and structurally-plausible proteins that cover natural sequence and functional space. Critically, EvoDiff can generate proteins inaccessible to structure-based models, such as those with disordered regions, while maintaining the ability to design scaffolds for functional structural motifs, demonstrating the universality of our sequence-based formulation. We envision that EvoDiff will expand capabilities in protein engineering beyond the structure-function paradigm toward programmable, sequence-first design.\nScalable bio-platform to recover critical metals fromcomplex waste sources\nAuthors: Good, N. M.; Kang-Yun, C. S.; Su, M. Z.; Zytnick, A. M.; Vu, H. N.; Grace, J. M.; Nguyen, H. H.; Park, D. M.; Skovran, E.; Fan, M.; Martinez-Gomez, N. C.\nScore: 10.4, Published: 2023-09-06 DOI: 10.1101/2023.09.02.556034\nChemical methods for extraction and refinement of technologically critical rare earth elements (REEs) are energy intensive, hazardous, and environmentally destructive. Current bio-based extraction systems rely on extremophilic organisms and generate many of the same detrimental effects as chemical methodologies. The mesophilic methylotrophic bacterium Methylobacterium extorquens AM1 was previously shown to grow using electronic waste by naturally acquiring REEs to power methanol metabolism. Here we show that growth using electronic waste as a sole REE source is scalable up to 10 L with consistent metal yields, without the use of harsh acids or high temperatures. Addition of organic acids increases REE leaching in a nonspecific manner. REE-specific bioleaching can be engineered through the overproduction of REE-binding ligands (called lanthanophores) and pyrroloquinoline quinone. REE bioaccumulation increases with leachate concentration and is highly specific. REEs are stored intracellularly in polyphosphate granules, and genetic engineering to eliminate exopolyphosphatase activity increases metal accumulation, confirming the link between phosphate metabolism and biological REE use. Finally, we report the innate ability of M. extorquens to grow using other complex REE sources including pulverized smart phones, demonstrating the flexibility and potential for use as a recovery platform for these critical metals.\nCentimeter-scale perfusable cultured meat with densely packed, highly aligned muscle fibers via hollow fiber bioreactor\nAuthors: Nie, M.; Shima, A.; Takeuchi, S.\nScore: 12.8, Published: 2023-09-04 DOI: 10.1101/2023.09.04.555230\nThe development of in-vitro biofabrication methods for producing cultured meat based on animal cells has been advancing, but replicating the texture of traditional meat in centimeter-scale has been a challenge. To address this, a method using a hollow fiber bioreactor (HFB) has been developed. The HFB contains semipermeable hollow fibers that act as artificial circulatory systems to deliver nutrients and oxygen uniformly to the tissue, along with microfabricated anchors for inducing cell alignment. With active perfusion, the biofabricated centimeter-scale chick muscle tissue shows elevated levels of marker protein expression and sarcomere formation across the whole tissue, along with improved texture and flavor. In the future, further scaling up of this approach using industrial robots has the potential to transform not only the cultured meat industry but also the tissue engineering fields aiming for the formation of large-scale artificial organs.\nSpyCombinator Assembly of Bispecific Binders\nAuthors: Driscoll, C. L.; Keeble, A. H.; Howarth, M.\nScore: 5.4, Published: 2023-09-03 DOI: 10.1101/2023.08.31.555700\nBispecific antibodies are a successful and expanding therapeutic class, bridging two cell-types or engaging two different molecules on the same cell. Bispecifics unlock avenues towards synergy, resistance evasion, and signaling bias. Standard approaches to generate bispecifics are complicated by the need for disulfide reduction/oxidation or cloning of each binder molecule in two different formats. Here we present a modular approach to bispecifics using SpyTag/SpyCatcher spontaneous amidation, where all binders are cloned in the same format, bearing a SpyTag. Two SpyTag-fused antigen-binding modules can be precisely conjugated onto DoubleCatcher, a tandem SpyCatcher where the second Catcher is unreactive until unveiling of reactivity using site-specific protease. Assembly on DoubleCatcher is efficient in phosphate-buffered saline at 37 {degrees}C, with half-times less than 5 min for both SpyCatcher arms and over 97% bispecific homogeneity. We engineer a panel of DoubleCatchers, locked through disulfide bonds to direct binders to project in different directions from the hub. We establish a generalized methodology for one-pot assembly and purification of bispecifics in 96-well plate format. A panel of Fab, affibody or nanobody binders recognizing different sites on HER2 were coupled to DoubleCatcher, revealing unexpected combinations with anti-proliferative or pro-proliferative activity on HER2-addicted cancer cells. Bispecific activity depended sensitively on both the order of the binders within the assembly and the geometry of DoubleCatcher scaffolds. These findings support the need for straightforward assembly in different formats. SpyCombinator provides a simple and scalable tool to discover synergy in bispecific activity, through modulating receptor organization and geometry.\nTopological evolution of sprouting vascular networks: from day-by-day analysis to general growth rules.\nAuthors: Rojek, K. O.; Wrzos, A.; Zukowski, S.; Bogdan, M.; Lisicki, M.; Szymczak, P.; Guzowski, J.\nScore: 3.0, Published: 2023-09-06 DOI: 10.1101/2023.09.02.555959\nEngineering tissues with an embedded vasculature of well-controlled topology remains one of the basic problems in biofabrication. Still, little is known about the evolution of topological characteristics of vascular networks over time. Here, we perform a high-throughput day-by-day analysis of tens of microvasculatures that sprout from endothelial-cell coated micrometric beads embedded in an external fibrin gel. We use the bead-assays to systematically analyze (i) macroscopic observables such as the overall length and area of the sprouts, (ii) microscopic observables such as the lengths of segments or the branching angles and their distributions, as well as (iii) general measures of network complexity such as the average number of bifurcations per branch. We develop a custom angiogenic image analysis toolkit and track the evolution of the networks for at least 14 days of culture under various conditions, e.g., in the presence of fibroblasts or with added endothelial growth factor (VEGF). We find that the evolution always consists of three stages: (i) an inactive stage in which cells remain bound to the beads, (ii) a sprouting stage in which the sprouts rapidly elongate and bifurcate, and (iii) the maturation stage in which the growth slows down. We show that higher concentrations of VEGF lead to an earlier onset of sprouting and to a higher number of primary branches, yet without significantly affecting the speed of growth of the individual sprouts. We find that the mean branching angle is weakly dependent on VEGF and typically in the range of 60-75 degrees suggesting that, by comparison with the available Laplacian growth models, the sprouts tend to follow local VEGF gradients. Finally, we observe an exponential distribution of segment lengths, which we interpret as a signature of stochastic branching at a constant bifurcation rate (per unit branch length). Our results, due to high statistical relevance, may serve as a benchmark for predictive models and reveal how the external means of control, such as VEGF concentration, could be used to control the morphology of the vascular networks. We provide guidelines for the fabrication of optimized microvasculatures with potential applications in drug testing or regenerative medicine.\nImage-based parametric finite element modelling for studying contact mechanics in human knee joints\nAuthors: Readioff, R.; Seil, R.; Mouton, C.; Marks, L.; Barrera, O.\nScore: 2.8, Published: 2023-09-10 DOI: 10.1101/2023.09.07.556747\nPurpose: This study presents a framework for generating patient-specific finite element models, parameterised and optimised for contact mechanics from computed tomography (CT) scans, by avoiding the segmentation step usually employed to transform medical images into 3D models. Two morphological parameters affecting contact mechanics were investigated in the framework development: tibial cartilage thickness and tibial spine height. This study explores the effect of the interplay of these parameters in load sharing between meniscus and articulating cartilage, meniscal posterior and anterior roots strain and menisci kinematics. Methods: Morphological measurements from four knee CT scans were collected, such as the maximum thickness of the tibial cartilage (ranging from 1.1 to 5.2 mm), the height of the tibial spine (ranging from 3.55 to 10.1 mm), and the width of the tibial plateau in both the coronal (ranging from 27.3 to 36.17 mm) and sagittal (ranging from 31.79 to 53.77 mm) planes. These measurements were taken for the lateral tibial plateau for both left and right knees. Subsequently, three finite element (FE) models were generated, comprising lateral tibial plateaus, lateral femoral condyle and lateral meniscus. The tibial cartilage thickness was kept at a constant value of 1 mm while varying the tibial spine height within the range measured from the CT images. This resulted in three FE models with varying spine heights, categorised as large (height = 7.42 mm), medium (height = 4.25 mm), and small (height = 1.63 mm) tibial spine heights. The menisci in the FE models were generated to be congruent with the tibial plateau. For the first time, this study advances the representations of the knee menisci microstructure in FE modelling, such that we have generated meniscus FE models with three layers of a hyperelastic model in which layer thickness and layer-specific hyperelastic material parameters are derived from our previous experimental work. Results: The load sharing between the meniscus and articular cartilage was not sensitive to the varying tibial spine heights. In all three FE models, cartilage carried more than 90% of the applied load. However, the meniscus kinematics and root strains varied considerably with changing tibial spine heights. The small tibial spine height model predicted the highest meniscus movements (8.12 and 9.33 mm in the radial and circumferential directions, respectively) and the highest root strain (21.92 and 22.19 mm/mm in the anterior and posterior roots, respectively). Conclusion: Our framework can generate finite element models of patients' knees using clinical data (i.e., CT scans) without the need for lengthy image segmentation. This process is not only time-efficient but also independent of imaging operators. The models converge quickly (~30 minutes on 2 cores) using an implicit solver with non-linear geometry and have the capability to predict contact mechanics between the articulating surfaces, meniscus kinematics and root strains. The modelling strategy presented here can provide valuable insights into predicting changes in the mechanics of soft tissues in the knee joint. It is particularly useful for investigating injury and surgical mechanisms related to the meniscus.\nLVING Reveals the Intracellular Structure of Cell Growth\nAuthors: Pradeep, S.; Zangle, T. A.\nScore: 2.4, Published: 2023-09-10 DOI: 10.1101/2023.09.08.553132\nThe continuous balance of growth and degradation inside cells maintains homeostasis. Disturbance of the balance by internal or external factors cause state of disease. Effective disease treatments seek to restore this balance. Here, we present a method based on quantitative phase imaging (QPI) based measurements of cell mass and the velocity of mass transport to quantify the balance of growth and degradation within intracellular control volumes. The result, which we call Lagrangian velocimetry for intracellular net growth (LVING), provides high resolution maps of intracellular biomass production and degradation. We use LVING to quantify the growth in different regions of the cell during phases of the cell cycle. LVING can also be used to quantitatively compare the effect of range of chemotherapy drug doses on subcellular growth processes. Finally, we applied LVING to characterize the effect of autophagy on the growth machinery inside cells. Overall, LVING reveals both the structure and distribution of basal growth within cells, as well as the disruptions to this structure that occur during alterations in cell state.\nGenome-wide Perturbation Analysis Screening System for Exosomes-Related Genes Based on the CRISPR/Cas9 Platform\nAuthors: Lu, L.; Yin, J.; Chen, N.; Guo, X.; Han, C.; Wang, M.; Du, H.; Li, H.; Pan, X.; Gao, M.; Wang, N.; Qi, D.; Wang, J.; Dong, F.; Li, T.; Ge, X.\nScore: 2.2, Published: 2023-09-08 DOI: 10.1101/2023.09.07.556640\nAs reported previously, exosomes have significant impacts on the physiological and pathological state in vivo. Exosomes have been extensively studied as a drug delivery carrier and some exosomes drugs have been undergoing clinical research. The mechanisms of exosome production, transport, and secretion remain to be studied in depth. Thus, here, we developed a novel CRISPR-UMI-based gene screening system, with which we can reveal the exosomes biogenesis, transport, and uptake mechanisms on a genome-wide scale. This system consists of two parts: one is a gene knockout component; the other part is a unique molecular identifiers (UMI) labeling component that can label the exosomes produced by the knockout cells, in which each sgRNA corresponds to a specific UMI. In this way, by detecting the UMI loaded in the exosomes, we can trace the knockout gene. In this study, we first verified the function of each component using plasmids and lentiviruses respectively. Next, we simulated the infection of cells with lentiviral libraries using a single lentivirus to validate the functionality of the screening system. Finally, we constructed a CRISPR-UMI-based library targeting 15 genes (genes with clear effects on exosomes biogenesis) to further validate the reliability of the screening system. The development of this screening system is of indispensable importance for the in-depth study of the mechanisms of exosome production and secretion, as well as for the improvement of exosomes production and the advancement of exosomes industrialization.\nDropBlot: single-cell western blotting of chemically fixed cancer cells\nAuthors: Liu, Y.; Herr, A. E.\nScore: 2.2, Published: 2023-09-06 DOI: 10.1101/2023.09.04.556277\nTo further realize proteomics of archived tissues for translational research, we introduce a hybrid microfluidic platform for high-specificity, high-sensitivity protein detection from individual chemically fixed cells. To streamline processing-to-analysis workflows and minimize signal loss, DropBlot serially integrates sample preparation using droplet-based antigen retrieval from single fixed cells with unified analysis-on-a-chip comprising microwell-based antigen extraction followed by chip-based single-cell western blotting. A water-in-oil droplet formulation proves robust to the harsh chemical (SDS, 6M urea) and thermal conditions (98{degrees}C, 1-2 hr.) required for sufficient antigen retrieval, and the electromechanical conditions required for electrotransfer of retrieved antigen from microwell-encapsulated droplets to single-cell electrophoresis. Protein-target retrieval was demonstrated for unfixed, paraformaldehyde-(PFA), and methanol-fixed cells. We observed higher protein electrophoresis separation resolution from PFA-fixed cells with sufficient immunoreactivity confirmed for key targets (HER2, GAPDH, EpCAM, Vimentin) from both fixation chemistries. Multiple forms of EpCAM and Vimentin were detected, a hallmark strength of western-blot analysis. DropBlot of PFA-fixed human-derived breast tumor specimens (n = 5) showed antigen retrieval from cells archived frozen for 6 yrs. DropBlot could provide a precision integrated workflow for single-cell resolution protein-biomarker mining of precious biospecimen repositories.\n",
  "wordCount" : "2657",
  "inLanguage": "en",
  "datePublished": "2023-09-13T10:39:03Z",
  "dateModified": "2023-09-13T10:39:03Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on September 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555711">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555711" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555711">
        <p class="paperTitle">Human gut metagenomic mining reveals an untapped source of peptide antibiotics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555711" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555711" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Torres, M. D. T.; Brooks, E. F.; Cesaro, A.; Sberro, H.; Nicolaou, C.; Bhatt, A. S.; de la Fuente, C.</p>
        <p class="info">Score: 32.0, Published: 2023-09-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555711' target='https://doi.org/10.1101/2023.08.31.555711'> 10.1101/2023.08.31.555711</a></p>
        <p class="abstract">Drug-resistant bacteria are outpacing traditional antibiotic discovery efforts. Here, we computationally mined 444,054 families of putative small proteins from 1,773 human gut metagenomes, identifying 323 peptide antibiotics encoded in small open reading frames (smORFs). To test our computational predictions, 78 peptides were synthesized and screened for antimicrobial activity in vitro, with 59% displaying activity against either pathogens or commensals. Since these peptides were unique compared to previously reported antimicrobial peptides, we termed them smORF-encoded peptides (SEPs). SEPs killed bacteria by targeting their membrane, synergized with each other, and modulated gut commensals, indicating that they may play a role in reconfiguring microbiome communities in addition to counteracting pathogens. The lead candidates were anti-infective in both murine skin abscess and deep thigh infection models. Notably, prevotellin-2 from Prevotella copri presented activity comparable to the commonly used antibiotic polymyxin B. We report the discovery of hundreds of peptide sequences in the human gut.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.556673">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.556673" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.556673">
        <p class="paperTitle">Protein generation with evolutionary diffusion: sequence is all you need</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.556673" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.556673" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alamdari, S.; Thakkar, N.; van den Berg, R.; Lu, A. X.; Fusi, N.; Amini, A. P.; Yang, K. K.</p>
        <p class="info">Score: 15.4, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.556673' target='https://doi.org/10.1101/2023.09.11.556673'> 10.1101/2023.09.11.556673</a></p>
        <p class="abstract">Deep generative models are increasingly powerful tools for the in silico design of novel proteins. Recently, a family of generative models called diffusion models has demonstrated the ability to generate biologically plausible proteins that are dissimilar to any actual proteins seen in nature, enabling unprecedented capability and control in de novo protein design. However, current state-of-the-art models generate protein structures, which severely limits the scope of their training data and restricts generations to a small and biased subset of protein design space. Here, we introduce a general-purpose diffusion framework, EvoDiff, that combines evolutionary-scale data with the distinct conditioning capabilities of diffusion models for controllable protein generation in sequence space. EvoDiff generates high-fidelity, diverse, and structurally-plausible proteins that cover natural sequence and functional space. Critically, EvoDiff can generate proteins inaccessible to structure-based models, such as those with disordered regions, while maintaining the ability to design scaffolds for functional structural motifs, demonstrating the universality of our sequence-based formulation. We envision that EvoDiff will expand capabilities in protein engineering beyond the structure-function paradigm toward programmable, sequence-first design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.02.556034">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.02.556034" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.02.556034">
        <p class="paperTitle">Scalable bio-platform to recover critical metals fromcomplex waste sources</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.02.556034" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.02.556034" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Good, N. M.; Kang-Yun, C. S.; Su, M. Z.; Zytnick, A. M.; Vu, H. N.; Grace, J. M.; Nguyen, H. H.; Park, D. M.; Skovran, E.; Fan, M.; Martinez-Gomez, N. C.</p>
        <p class="info">Score: 10.4, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.02.556034' target='https://doi.org/10.1101/2023.09.02.556034'> 10.1101/2023.09.02.556034</a></p>
        <p class="abstract">Chemical methods for extraction and refinement of technologically critical rare earth elements (REEs) are energy intensive, hazardous, and environmentally destructive. Current bio-based extraction systems rely on extremophilic organisms and generate many of the same detrimental effects as chemical methodologies. The mesophilic methylotrophic bacterium Methylobacterium extorquens AM1 was previously shown to grow using electronic waste by naturally acquiring REEs to power methanol metabolism. Here we show that growth using electronic waste as a sole REE source is scalable up to 10 L with consistent metal yields, without the use of harsh acids or high temperatures. Addition of organic acids increases REE leaching in a nonspecific manner. REE-specific bioleaching can be engineered through the overproduction of REE-binding ligands (called lanthanophores) and pyrroloquinoline quinone. REE bioaccumulation increases with leachate concentration and is highly specific. REEs are stored intracellularly in polyphosphate granules, and genetic engineering to eliminate exopolyphosphatase activity increases metal accumulation, confirming the link between phosphate metabolism and biological REE use. Finally, we report the innate ability of M. extorquens to grow using other complex REE sources including pulverized smart phones, demonstrating the flexibility and potential for use as a recovery platform for these critical metals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.555230">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.555230" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.555230">
        <p class="paperTitle">Centimeter-scale perfusable cultured meat with densely packed, highly aligned muscle fibers via hollow fiber bioreactor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.555230" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.555230" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nie, M.; Shima, A.; Takeuchi, S.</p>
        <p class="info">Score: 12.8, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.555230' target='https://doi.org/10.1101/2023.09.04.555230'> 10.1101/2023.09.04.555230</a></p>
        <p class="abstract">The development of in-vitro biofabrication methods for producing cultured meat based on animal cells has been advancing, but replicating the texture of traditional meat in centimeter-scale has been a challenge. To address this, a method using a hollow fiber bioreactor (HFB) has been developed. The HFB contains semipermeable hollow fibers that act as artificial circulatory systems to deliver nutrients and oxygen uniformly to the tissue, along with microfabricated anchors for inducing cell alignment. With active perfusion, the biofabricated centimeter-scale chick muscle tissue shows elevated levels of marker protein expression and sarcomere formation across the whole tissue, along with improved texture and flavor. In the future, further scaling up of this approach using industrial robots has the potential to transform not only the cultured meat industry but also the tissue engineering fields aiming for the formation of large-scale artificial organs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555700">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555700" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555700">
        <p class="paperTitle">SpyCombinator Assembly of Bispecific Binders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555700" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555700" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Driscoll, C. L.; Keeble, A. H.; Howarth, M.</p>
        <p class="info">Score: 5.4, Published: 2023-09-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555700' target='https://doi.org/10.1101/2023.08.31.555700'> 10.1101/2023.08.31.555700</a></p>
        <p class="abstract">Bispecific antibodies are a successful and expanding therapeutic class, bridging two cell-types or engaging two different molecules on the same cell. Bispecifics unlock avenues towards synergy, resistance evasion, and signaling bias. Standard approaches to generate bispecifics are complicated by the need for disulfide reduction/oxidation or cloning of each binder molecule in two different formats. Here we present a modular approach to bispecifics using SpyTag/SpyCatcher spontaneous amidation, where all binders are cloned in the same format, bearing a SpyTag. Two SpyTag-fused antigen-binding modules can be precisely conjugated onto DoubleCatcher, a tandem SpyCatcher where the second Catcher is unreactive until unveiling of reactivity using site-specific protease. Assembly on DoubleCatcher is efficient in phosphate-buffered saline at 37 {degrees}C, with half-times less than 5 min for both SpyCatcher arms and over 97% bispecific homogeneity. We engineer a panel of DoubleCatchers, locked through disulfide bonds to direct binders to project in different directions from the hub. We establish a generalized methodology for one-pot assembly and purification of bispecifics in 96-well plate format. A panel of Fab, affibody or nanobody binders recognizing different sites on HER2 were coupled to DoubleCatcher, revealing unexpected combinations with anti-proliferative or pro-proliferative activity on HER2-addicted cancer cells. Bispecific activity depended sensitively on both the order of the binders within the assembly and the geometry of DoubleCatcher scaffolds. These findings support the need for straightforward assembly in different formats. SpyCombinator provides a simple and scalable tool to discover synergy in bispecific activity, through modulating receptor organization and geometry.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.02.555959">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.02.555959" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.02.555959">
        <p class="paperTitle">Topological evolution of sprouting vascular networks: from day-by-day analysis to general growth rules.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.02.555959" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.02.555959" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rojek, K. O.; Wrzos, A.; Zukowski, S.; Bogdan, M.; Lisicki, M.; Szymczak, P.; Guzowski, J.</p>
        <p class="info">Score: 3.0, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.02.555959' target='https://doi.org/10.1101/2023.09.02.555959'> 10.1101/2023.09.02.555959</a></p>
        <p class="abstract">Engineering tissues with an embedded vasculature of well-controlled topology remains one of the basic problems in biofabrication. Still, little is known about the evolution of topological characteristics of vascular networks over time. Here, we perform a high-throughput day-by-day analysis of tens of microvasculatures that sprout from endothelial-cell coated micrometric beads embedded in an external fibrin gel. We use the bead-assays to systematically analyze (i)  macroscopic observables such as the overall length and area of the sprouts, (ii)  microscopic observables such as the lengths of segments or the branching angles and their distributions, as well as (iii) general measures of network complexity such as the average number of bifurcations per branch. We develop a custom angiogenic image analysis toolkit and track the evolution of the networks for at least 14 days of culture under various conditions, e.g., in the presence of fibroblasts or with added endothelial growth factor (VEGF). We find that the evolution always consists of three stages: (i) an inactive stage in which cells remain bound to the beads, (ii) a sprouting stage in which the sprouts rapidly elongate and bifurcate, and (iii) the maturation stage in which the growth slows down. We show that higher concentrations of VEGF lead to an earlier onset of sprouting and to a higher number of primary branches, yet without significantly affecting the speed of growth of the individual sprouts. We find that the mean branching angle is weakly dependent on VEGF and typically in the range of 60-75 degrees suggesting that, by comparison with the available Laplacian growth models, the sprouts tend to follow local VEGF gradients. Finally, we observe an exponential distribution of segment lengths, which we interpret as a signature of stochastic branching at a constant bifurcation rate (per unit branch length). Our results, due to high statistical relevance, may serve as a benchmark for predictive models and reveal how the external means of control, such as VEGF concentration, could be used to control the morphology of the vascular networks. We provide guidelines for the fabrication of optimized microvasculatures with potential applications in drug testing or regenerative medicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556747">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556747" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556747">
        <p class="paperTitle">Image-based parametric finite element modelling for studying contact mechanics in human knee joints</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556747" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556747" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Readioff, R.; Seil, R.; Mouton, C.; Marks, L.; Barrera, O.</p>
        <p class="info">Score: 2.8, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556747' target='https://doi.org/10.1101/2023.09.07.556747'> 10.1101/2023.09.07.556747</a></p>
        <p class="abstract">Purpose: This study presents a framework for generating patient-specific finite element models, parameterised and optimised for contact mechanics from computed tomography (CT) scans, by avoiding the segmentation step usually employed to transform medical images into 3D models. Two morphological parameters affecting contact mechanics were investigated in the framework development: tibial cartilage thickness and tibial spine height. This study explores the effect of the interplay of these parameters in load sharing between meniscus and articulating cartilage, meniscal posterior and anterior roots strain and menisci kinematics. Methods: Morphological measurements from four knee CT scans were collected, such as the maximum thickness of the tibial cartilage (ranging from 1.1 to 5.2 mm), the height of the tibial spine (ranging from 3.55 to 10.1 mm), and the width of the tibial plateau in both the coronal (ranging from 27.3 to 36.17 mm) and sagittal (ranging from 31.79 to 53.77 mm) planes. These measurements were taken for the lateral tibial plateau for both left and right knees. Subsequently, three finite element (FE) models were generated, comprising lateral tibial plateaus, lateral femoral condyle and lateral meniscus. The tibial cartilage thickness was kept at a constant value of 1 mm while varying the tibial spine height within the range measured from the CT images. This resulted in three FE models with varying spine heights, categorised as large (height = 7.42 mm), medium (height = 4.25 mm), and small (height = 1.63 mm) tibial spine heights. The menisci in the FE models were generated to be congruent with the tibial plateau. For the first time, this study advances the representations of the knee menisci microstructure in FE modelling, such that we have generated meniscus FE models with three layers of a hyperelastic model in which layer thickness and layer-specific hyperelastic material parameters are derived from our previous experimental work. Results: The load sharing between the meniscus and articular cartilage was not sensitive to the varying tibial spine heights. In all three FE models, cartilage carried more than 90% of the applied load. However, the meniscus kinematics and root strains varied considerably with changing tibial spine heights. The small tibial spine height model predicted the highest meniscus movements (8.12 and 9.33 mm in the radial and circumferential directions, respectively) and the highest root strain (21.92 and 22.19 mm/mm in the anterior and posterior roots, respectively). Conclusion: Our framework can generate finite element models of patients&#39; knees using clinical data (i.e., CT scans) without the need for lengthy image segmentation. This process is not only time-efficient but also independent of imaging operators. The models converge quickly (~30 minutes on 2 cores) using an implicit solver with non-linear geometry and have the capability to predict contact mechanics between the articulating surfaces, meniscus kinematics and root strains. The modelling strategy presented here can provide valuable insights into predicting changes in the mechanics of soft tissues in the knee joint. It is particularly useful for investigating injury and surgical mechanisms related to the meniscus.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.553132">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.553132" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.553132">
        <p class="paperTitle">LVING Reveals the Intracellular Structure of Cell Growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.553132" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.553132" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pradeep, S.; Zangle, T. A.</p>
        <p class="info">Score: 2.4, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.553132' target='https://doi.org/10.1101/2023.09.08.553132'> 10.1101/2023.09.08.553132</a></p>
        <p class="abstract">The continuous balance of growth and degradation inside cells maintains homeostasis. Disturbance of the balance by internal or external factors cause state of disease. Effective disease treatments seek to restore this balance. Here, we present a method based on quantitative phase imaging (QPI) based measurements of cell mass and the velocity of mass transport to quantify the balance of growth and degradation within intracellular control volumes. The result, which we call Lagrangian velocimetry for intracellular net growth (LVING), provides high resolution maps of intracellular biomass production and degradation. We use LVING to quantify the growth in different regions of the cell during phases of the cell cycle. LVING can also be used to quantitatively compare the effect of range of chemotherapy drug doses on subcellular growth processes. Finally, we applied LVING to characterize the effect of autophagy on the growth machinery inside cells. Overall, LVING reveals both the structure and distribution of basal growth within cells, as well as the disruptions to this structure that occur during alterations in cell state.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556640">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556640" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556640">
        <p class="paperTitle">Genome-wide Perturbation Analysis Screening System for Exosomes-Related Genes Based on the CRISPR/Cas9 Platform</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556640" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556640" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lu, L.; Yin, J.; Chen, N.; Guo, X.; Han, C.; Wang, M.; Du, H.; Li, H.; Pan, X.; Gao, M.; Wang, N.; Qi, D.; Wang, J.; Dong, F.; Li, T.; Ge, X.</p>
        <p class="info">Score: 2.2, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556640' target='https://doi.org/10.1101/2023.09.07.556640'> 10.1101/2023.09.07.556640</a></p>
        <p class="abstract">As reported previously, exosomes have significant impacts on the physiological and pathological state in vivo. Exosomes have been extensively studied as a drug delivery carrier and some exosomes drugs have been undergoing clinical research. The mechanisms of exosome production, transport, and secretion remain to be studied in depth. Thus, here, we developed a novel CRISPR-UMI-based gene screening system, with which we can reveal the exosomes biogenesis, transport, and uptake mechanisms on a genome-wide scale. This system consists of two parts: one is a gene knockout component; the other part is a unique molecular identifiers (UMI) labeling component that can label the exosomes produced by the knockout cells, in which each sgRNA corresponds to a specific UMI. In this way, by detecting the UMI loaded in the exosomes, we can trace the knockout gene. In this study, we first verified the function of each component using plasmids and lentiviruses respectively. Next, we simulated the infection of cells with lentiviral libraries using a single lentivirus to validate the functionality of the screening system. Finally, we constructed a CRISPR-UMI-based library targeting 15 genes (genes with clear effects on exosomes biogenesis) to further validate the reliability of the screening system. The development of this screening system is of indispensable importance for the in-depth study of the mechanisms of exosome production and secretion, as well as for the improvement of exosomes production and the advancement of exosomes industrialization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556277">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556277" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556277">
        <p class="paperTitle">DropBlot: single-cell western blotting of chemically fixed cancer cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556277" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556277" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, Y.; Herr, A. E.</p>
        <p class="info">Score: 2.2, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556277' target='https://doi.org/10.1101/2023.09.04.556277'> 10.1101/2023.09.04.556277</a></p>
        <p class="abstract">To further realize proteomics of archived tissues for translational research, we introduce a hybrid microfluidic platform for high-specificity, high-sensitivity protein detection from individual chemically fixed cells. To streamline processing-to-analysis workflows and minimize signal loss, DropBlot serially integrates sample preparation using droplet-based antigen retrieval from single fixed cells with unified analysis-on-a-chip comprising microwell-based antigen extraction followed by chip-based single-cell western blotting. A water-in-oil droplet formulation proves robust to the harsh chemical (SDS, 6M urea) and thermal conditions (98{degrees}C, 1-2 hr.) required for sufficient antigen retrieval, and the electromechanical conditions required for electrotransfer of retrieved antigen from microwell-encapsulated droplets to single-cell electrophoresis. Protein-target retrieval was demonstrated for unfixed, paraformaldehyde-(PFA), and methanol-fixed cells. We observed higher protein electrophoresis separation resolution from PFA-fixed cells with sufficient immunoreactivity confirmed for key targets (HER2, GAPDH, EpCAM, Vimentin) from both fixation chemistries. Multiple forms of EpCAM and Vimentin were detected, a hallmark strength of western-blot analysis. DropBlot of PFA-fixed human-derived breast tumor specimens (n = 5) showed antigen retrieval from cells archived frozen for 6 yrs. DropBlot could provide a precision integrated workflow for single-cell resolution protein-biomarker mining of precious biospecimen repositories.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
