<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Transferrin Receptor Targeting Chimeras (TransTACs) for Membrane Protein Degradation
Authors: Zhang, D.; Duque-Jimenez, J.; Brixi, G.; Facchinetti, F.; Rhee, K.; Feng, W. W.; Jänne, P. A.; Zhou, X.
Score: 40.3, Published: 2023-08-13 DOI: 10.1101/2023.08.10.552782
Cancer cells require high levels of iron for rapid proliferation, leading to a significant upregulation of the iron carrier protein Transferrin Receptor (TfR) on their cell surface. Leveraging this phenomenon and the exceptionally fast endocytosis rate of TfR, we introduce Transferrin Receptor TArgeting Chimeras (TransTAC), a novel molecular archetype for membrane protein degradation in cancers and other cell types.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Transferrin Receptor Targeting Chimeras (TransTACs) for Membrane Protein Degradation
Authors: Zhang, D.; Duque-Jimenez, J.; Brixi, G.; Facchinetti, F.; Rhee, K.; Feng, W. W.; Jänne, P. A.; Zhou, X.
Score: 40.3, Published: 2023-08-13 DOI: 10.1101/2023.08.10.552782
Cancer cells require high levels of iron for rapid proliferation, leading to a significant upregulation of the iron carrier protein Transferrin Receptor (TfR) on their cell surface. Leveraging this phenomenon and the exceptionally fast endocytosis rate of TfR, we introduce Transferrin Receptor TArgeting Chimeras (TransTAC), a novel molecular archetype for membrane protein degradation in cancers and other cell types." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-20T10:35:54+00:00" />
<meta property="article:modified_time" content="2023-08-20T10:35:54+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Transferrin Receptor Targeting Chimeras (TransTACs) for Membrane Protein Degradation
Authors: Zhang, D.; Duque-Jimenez, J.; Brixi, G.; Facchinetti, F.; Rhee, K.; Feng, W. W.; Jänne, P. A.; Zhou, X.
Score: 40.3, Published: 2023-08-13 DOI: 10.1101/2023.08.10.552782
Cancer cells require high levels of iron for rapid proliferation, leading to a significant upregulation of the iron carrier protein Transferrin Receptor (TfR) on their cell surface. Leveraging this phenomenon and the exceptionally fast endocytosis rate of TfR, we introduce Transferrin Receptor TArgeting Chimeras (TransTAC), a novel molecular archetype for membrane protein degradation in cancers and other cell types."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Transferrin Receptor Targeting Chimeras (TransTACs) for Membrane Protein Degradation\nAuthors: Zhang, D.; Duque-Jimenez, J.; Brixi, G.; Facchinetti, F.; Rhee, K.; Feng, W. W.; Jänne, P. A.; Zhou, X.\nScore: 40.3, Published: 2023-08-13 DOI: 10.1101/2023.08.10.552782\nCancer cells require high levels of iron for rapid proliferation, leading to a significant upregulation of the iron carrier protein Transferrin Receptor (TfR) on their cell surface. Leveraging this phenomenon and the exceptionally fast endocytosis rate of TfR, we introduce Transferrin Receptor TArgeting Chimeras (TransTAC), a novel molecular archetype for membrane protein degradation in cancers and other cell types.",
  "keywords": [
    
  ],
  "articleBody": " Transferrin Receptor Targeting Chimeras (TransTACs) for Membrane Protein Degradation\nAuthors: Zhang, D.; Duque-Jimenez, J.; Brixi, G.; Facchinetti, F.; Rhee, K.; Feng, W. W.; Jänne, P. A.; Zhou, X.\nScore: 40.3, Published: 2023-08-13 DOI: 10.1101/2023.08.10.552782\nCancer cells require high levels of iron for rapid proliferation, leading to a significant upregulation of the iron carrier protein Transferrin Receptor (TfR) on their cell surface. Leveraging this phenomenon and the exceptionally fast endocytosis rate of TfR, we introduce Transferrin Receptor TArgeting Chimeras (TransTAC), a novel molecular archetype for membrane protein degradation in cancers and other cell types. TransTACs repurpose the naturally recycling receptor TfR1 for protein degradation. To accomplish this, we utilized a combination of protein engineering strategies to redirect the target protein from recycling-endosome trafficking to lysosomal degradation. We show that TransTACs can highly efficiently degrade a diverse range of single-pass, multi-pass, native, or synthetic membrane proteins, establishing new possibilities for targeted cancer therapy.\nVolumetric imaging of an intact organism by a distributed molecular network\nAuthors: Weinstein, J. A.; Qian, N.\nScore: 40.1, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553025\nLymphatic, nervous, and tumoral tissues, among others, exhibit physiology that emerges from three-dimensional interactions between genetically unique cells. A technology capable of volumetrically imaging transcriptomes, genotypes, and morphologies in a single de novo measurement would therefore provide a critical view into the biological complexity of living systems. Here we achieve this by extending DNA microscopy, an imaging modality that encodes a spatio-genetic map of a specimen via a massive distributed network of DNA molecules inside it, to three dimensions and multiple length scales in developing zebrafish embryos.\nSimulated musculoskeletal optimization for sprinting and marathon running\nAuthors: Van Wouwe, T.; Hicks, J.; Delp, S.; Liu, K.\nScore: 11.4, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552222\nMusculoskeletal geometry and muscle volumes vary widely in the population and are intricately linked to the performance of tasks ranging from walking and running to jumping and sprinting. However, our ability to understand how these parameters affect task performance has been limited due to the high computational cost of modelling the necessary complexity of the musculoskeletal system and solving the requisite multi-dimensional optimization problem. For example, sprinting and running are fundamental to many forms of sport, but past research on the relationships between musculoskeletal geometry, muscle volumes, and running performance has been limited to observational studies, which have not established cause-effect relationships, and simulation studies with simplified representations of musculoskeletal geometry. In this study, we developed a novel musculoskeletal simulator that is differentiable with respect to musculoskeletal geometry and muscle volumes. This simulator enabled us to find the optimal body segment dimensions and optimal distribution of added muscle volume for sprinting and marathon running. Our simulation results replicate experimental observations, such as increased muscle mass in sprinters, as well a mass in the lower end of the healthy BMI range and a higher leg-length-to-height ratio in marathon runners. The simulations also reveal new relationships, for example showing that hip musculature is vital to both sprinting and marathon running. We found hip flexor and extensor moment arms were maximized to optimize sprint and marathon running performance, and hip muscles the main target when we simulated strength training for sprinters. Our simulation results can help sprint and marathon runners customize strength training, and our simulator can be extended to other athletic tasks, such as jumping, or to non-athletic applications, such as designing interventions to improve mobility in older adults or individuals with movement disorders. AUTHOR SUMMARYOur study addresses the challenge of determining optimal musculoskeletal parameters for tasks like sprinting and marathon running. Existing research has been limited to observational studies and simplified simulations. To overcome these limitations, we developed a differentiable musculoskeletal simulator to optimize running performance. We replicated past findings and uncovered new insights. We confirmed the benefits of increased muscle mass for sprinters and identified key factors for marathon runners, such a mass in the lower end of the healthy BMI range and an increased leg-length-to-height ratio. Hip musculature was found to be critical for both sprinting and marathon running. Our simulation results have practical implications. They can inform customized strength training for sprinters and marathon runners. Additionally, the simulator can be extended to other athletic tasks, benefiting various sporting events. Beyond athletics, our open-source simulator has broader applications. It can determine minimal strength requirements for daily activities, guide strength training in the elderly, and estimate the effects of simulated musculoskeletal surgery.\nConditional Protein Denoising Diffusion Generates Programmable Endonucleases\nAuthors: Zhou, B.; Zheng, L.; Wu, B.; Yi, K.; Zhong, B.; Lio, P.; Hong, L.\nScore: 8.7, Published: 2023-08-14 DOI: 10.1101/2023.08.10.552783\nComputation or deep learning-based functional protein generation methods address the urgent demand for novel biocatalysts, allowing for precise tailoring of functionalities to meet specific requirements. This emergence leads to the creation of highly efficient and specialized proteins with wide-ranging applications in scientific, technological, and biomedical domains. This study establishes a conditional protein diffusion model, namely CPDiffusion, to deliver diverse protein sequences with desired functions. While the model is free from extensive training data and the sampling process involves little guidance on the type of generated amino acids, CPDiffusion effectively secures essential highly conserved residues that are crucial for protein functionalities. We employed CPDiffusion and generated 27 artificially designed Argonaute proteins, programmable endonucleases applied for easy-to-implement and high-throughput screenings in gene editing and molecular diagnostics, that mutated approximately 200 - 400 amino acids with 40% sequence identities to those from nature. Experimental tests demonstrate the solubility of all 27 artificially-designed proteins (AP), with 24 of them displaying DNA cleavage activity. Remarkably, 74% of active APs exhibited superior activity compared to the template protein, and the most effective one showcased a remarkable nearly nine-fold enhancement of enzymatic activity. Moreover, 37% of APs exhibited enhanced thermostability. These findings emphasize CPDiffusions remarkable capability to generate long-sequence proteins in a single step while retaining or enhancing intricate functionality. This approach facilitates the design of intricate enzymes featuring multi-domain molecular structures through in silico generation and throughput, all accomplished without the need for supervision from labeled data.\nCrystal ribcage: a platform for probing real-time lung function at cellular resolution in health and disease\nAuthors: Banerji, R.; Grifno, G. N.; Shi, L.; Smolen, D.; LeBourdais, R.; Muhvich, J.; Eberman, C.; Hiller, B.; Lee, J.; Regan, K.; Zheng, S.; Zhang, S. S.; Jiang, J.; Raslan, A. A.; Breda, J. C.; Pihl, R.; Traber, K.; Mazzilli, S.; Ligresti, G.; Mizgerd, J. P.; Suki, B.; Nia, H. T.\nScore: 13.8, Published: 2023-08-09 DOI: 10.1101/2022.10.28.514251\nUnderstanding the dynamic pathogenesis and treatment response in pulmonary diseases requires probing the lung at cellular resolution in real-time. Despite recent progress in intravital imaging, optical imaging of the lung during active respiration and circulation has remained challenging. Here, we introduce the crystal ribcage: a transparent ribcage that (i) allows truly multiscale optical imaging of the lung in health and disease from whole-organ to single cell, (ii) enables the modulation of lung biophysics and immunity through intravascular, intrapulmonary, intraparenchymal, and optogenetic interventions, and (iii) preserves the 3-D architecture, air-liquid interface, cellular diversity, and respiratory-circulatory functions of the lung. Utilizing these unprecedented capabilities on murine models of primary and metastatic lung tumors, respiratory infection, pulmonary fibrosis, emphysema, and acute lung injury we probed how disease progression remodels the respiratory-circulatory functions at the single alveolus and capillary levels. In cancer, we identified the earliest stage of tumorigenesis that compromises alveolar and capillary functions, a key state with consequences on tumor progression and treatment response. In pneumonia, we mapped mutual links between the recruited immune cells and the alveolar-capillary functions. We found that neutrophil migration is strongly and reversibly responsive to vascular pressure with implications for understanding of how lung physiology, altered by disease and anatomical location, affects immune cell activities. The crystal ribcage and its broad applications presented here will facilitate further studies of real-time remodeling of the alveoli and capillaries during pathogenesis of nearly any pulmonary disease, leading to the identification of new targets for treatment strategies.\nSecond Harmonic Generation Imaging Reveals Entanglement of Collagen Fibers in the Elephant Trunk Skin Dermis\nAuthors: Schulz, A.; Plotczyk, M.; Sordilla, S.; Boyle, M.; Singal, K.; Reidenberg, J.; Hu, D.; Higgins, C.\nScore: 6.9, Published: 2023-08-15 DOI: 10.1101/2023.08.11.553031\nForm-function relationships often have tradeoffs: if a material is tough, it is often inflexible, and vice versa. This is particularly relevant for the elephant trunk, where the skin should be protective yet elastic. To investigate how this is achieved, we used classical histochemical staining and second harmonic generation microscopy to describe the morphology and composition of elephant trunk skin. We report structure at the macro and micro scales, from the thickness of the dermis to the interaction of 10 m thick collagen fibers. We analyzed several sites along the length of the trunk, to compare and contrast the dorsal-ventral and proximal-distal skin morphologies and compositions. We find the dorsal skin of the elephant trunk can have keratin armor layers over 2mm thick, which is nearly 100 times the thickness of the equivalent layer in human skin. We also found that the structural support layer (the dermis) of elephant trunk contains a distribution of collagen-I (COL1) fibers in both perpendicular and parallel arrangement. The bimodal distribution of collagen is seen across all portions of the trunk, and is dissimilar from that of human skin where one orientation dominates within a body site. We hypothesize that this distribution of COL1 in the elephant trunk allows both flexibility and load-bearing capabilities. Additionally, when viewing individual fiber interaction of 10 m thick collagen, we find the fiber crossings per unit volume are five times more common than in human skin, suggesting that the fibers are entangled. We surmise that these intriguing structures permit both flexibility and strength in the elephant trunk. The complex nature of the elephant skin may inspire the design of materials that can combine strength and flexibility.\nThe Enderstruder: An accessible open-source syringe extruder compatible with Ender series 3D printers\nAuthors: Cordova, D. J.; Rodriguez, A. A.; Woodward, S. C.; Crosby, C. O.\nScore: 6.3, Published: 2023-08-17 DOI: 10.1101/2023.08.15.553241\nBioprinting has enabled the precise spatiotemporal deposition of cell-containing bioinks, opening new avenues of research in tissue engineering and regenerative medicine. Although several open-source syringe extruder designs have been published and adopted by end users, only one has been specifically adapted for the Ender series, an affordable and open-source line of thermoplastic 3D printers. Here, we introduce the Enderstruder, a cost-effective extruder attachment that uses a standard 10 mL BD syringe, positions the stepper motor at the level of the gantry, enhances x-axis stability with a linear rail, and uses the originally included stepper motor, resulting in reduced cost and simplified assembly. Furthermore, we present an iterative process to fine-tune printing profiles for high-viscosity biomaterial inks. To facilitate the implementation of our work by other groups, we provide fully editable Cura profiles for five commonly used biomaterials. We employ the Enderstruder to print established calibration patterns and complex shapes using each of these five materials to validate and characterize our design. By presenting the Enderstruder and its iterative development process, this study contributes to the growing repository of open-source bioprinting solutions, fostering greater accessibility and affordability for researchers in tissue engineering.\nConserved and tissue-specific immune responses to biologic scaffold implantation\nAuthors: DeStefano, S.; Josyula, A.; Faust, M.; Fertil, D.; Lokwani, R.; Ngo, T. B.; Sadtler, K.\nScore: 5.4, Published: 2023-08-17 DOI: 10.1101/2023.08.15.553390\nUpon implantation into a patient, any biomaterial induces a cascade of immune responses that influences the outcome of that device. This cascade depends upon several factors, including the composition of the material itself and the location in which the material is implanted. There is still significant uncertainty around the role of different tissue microenvironments in the immune response to biomaterials and how that may alter downstream scaffold remodeling and integration. In this study, we present a study evaluating the immune response to decellularized extracellular matrix materials within the intraperitoneal cavity, the subcutaneous space, and in a traumatic skeletal muscle injury microenvironment. All different locations induced robust cellular recruitment, specifically of macrophages and eosinophils. The latter was most prominent in the subcutaneous space. Intraperitoneal implants uniquely recruited B cells that may alter downstream reactivity as adaptive immunity has been strongly implicated in the outcome of scaffold remodeling. These data suggest that the location of tissue implants should be taken together with the composition of the material itself when designing devices for downline therapeutics.\nA Hybrid Transistor with Transcriptionally Controlled Computation and Plasticity\nAuthors: Gao, Y.; Zhou, Y.; Ji, X.; Graham, A. J.; Dundas, C. M.; Mahfoud, I. E. M.; Tibbett, B. M.; Tan, B.; Partipilo, G.; Dodabalapur, A.; Rivnay, J.; Keitz, B. K.\nScore: 4.6, Published: 2023-08-18 DOI: 10.1101/2023.08.16.553547\nOrganic electrochemical transistors (OECTs) are ideal devices for translating biological signals into electrical readouts and have applications in bioelectronics, biosensing, and neuromorphic computing. Despite their potential, developing programmable and modular methods for living systems to interface with OECTs has proven challenging. Here we describe hybrid OECTs containing the model electroactive bacterium Shewanella oneidensis that enable the transduction of biological computations to electrical responses. Specifically, we fabricated planar p-type OECTs and demonstrated that channel de-doping is driven by extracellular electron transfer (EET) from S. oneidensis. Leveraging this mechanistic understanding and our ability to control EET flux via transcriptional regulation, we used plasmid-based Boolean logic gates to translate biological computation into current changes within the OECT. Finally, we demonstrated EET-driven changes to OECT synaptic plasticity. This work enables fundamental EET studies and OECT-based biosensing and biocomputing systems with genetically controllable and modular design elements.\nSingle-cell precision nanotechnology in vivo\nAuthors: MOLBAY, M.; Kick, B.; Zhao, S.; Todorov, M. I.; Ohn, T.-L.; Roth, S.; Simats, A.; Singh, V.; Khalin, I.; Pan, C.; Singh Bhatia, H.; Hellal, F.; Zeidler, R.; Liesz, A.; Plesnila, N.; Dietz, H.; Erturk, A.\nScore: 148.3, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550304\nTargeting nanoparticle therapeutics with cellular accuracy in whole organisms could open breakthrough opportunities in precision medicine. However, evaluating and fine-tuning the biodistribution of such systems in the whole organism at the cellular level remains a major obstacle. Here, we constructed targetable DNA origami, and analyzed biodistribution in transparent mice, in addition to studying tolerability, clearance kinetics, and immune response parameters. Untargeted DNA origami primarily accumulated in the spleen and the liver, while an immune cell-targeting variant successfully attached to immune cells throughout the body. A cancer cell-targeting mimetic co-localized on solid-tumor metastasis in the liver and the lung. These findings indicate that DNA origami can be directed in vivo, providing an important proof-of-concept and highlights the potential of high-resolution tissue-clearing imaging technologies in their development. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=121 SRC=\"FIGDIR/small/550304v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (44K): org.highwire.dtl.DTLVardef@ce216borg.highwire.dtl.DTLVardef@f40bb4org.highwire.dtl.DTLVardef@2c6a66org.highwire.dtl.DTLVardef@1fd7fde_HPS_FORMAT_FIGEXP M_FIG C_FIG HighlightsO_LIThis study demonstrates the potential of DNA origami-based drug delivery systems as versatile tool for for targeted delivery, which could be used to treat a range of diseases with applications. C_LIO_LIThe immune compatibility, half-life, targeting efficiency, and the biodistribution evaluation of DNA origami indicate its potential for systemic drug delivery. C_LIO_LIOur approach enables the assessment of biodistribution of nanoparticles in the intact body with a sensitivity to the single-cell level, highlighting the high-resolution tissue clearing technoloies in revealing DNA origamis feasibility for drug targeting. C_LI\n",
  "wordCount" : "2517",
  "inLanguage": "en",
  "datePublished": "2023-08-20T10:35:54Z",
  "dateModified": "2023-08-20T10:35:54Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta"><span>updated on August 20, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552782">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552782" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552782">
        <p class="paperTitle">Transferrin Receptor Targeting Chimeras (TransTACs) for Membrane Protein Degradation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552782" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552782" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, D.; Duque-Jimenez, J.; Brixi, G.; Facchinetti, F.; Rhee, K.; Feng, W. W.; Jänne, P. A.; Zhou, X.</p>
        <p class="info">Score: 40.3, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552782' target='https://doi.org/10.1101/2023.08.10.552782'> 10.1101/2023.08.10.552782</a></p>
        <p class="abstract">Cancer cells require high levels of iron for rapid proliferation, leading to a significant upregulation of the iron carrier protein Transferrin Receptor (TfR) on their cell surface. Leveraging this phenomenon and the exceptionally fast endocytosis rate of TfR, we introduce Transferrin Receptor TArgeting Chimeras (TransTAC), a novel molecular archetype for membrane protein degradation in cancers and other cell types. TransTACs repurpose the naturally recycling receptor TfR1 for protein degradation. To accomplish this, we utilized a combination of protein engineering strategies to redirect the target protein from recycling-endosome trafficking to lysosomal degradation. We show that TransTACs can highly efficiently degrade a diverse range of single-pass, multi-pass, native, or synthetic membrane proteins, establishing new possibilities for targeted cancer therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553025">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553025" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553025">
        <p class="paperTitle">Volumetric imaging of an intact organism by a distributed molecular network</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553025" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553025" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weinstein, J. A.; Qian, N.</p>
        <p class="info">Score: 40.1, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553025' target='https://doi.org/10.1101/2023.08.11.553025'> 10.1101/2023.08.11.553025</a></p>
        <p class="abstract">Lymphatic, nervous, and tumoral tissues, among others, exhibit physiology that emerges from three-dimensional interactions between genetically unique cells. A technology capable of volumetrically imaging transcriptomes, genotypes, and morphologies in a single de novo measurement would therefore provide a critical view into the biological complexity of living systems. Here we achieve this by extending DNA microscopy, an imaging modality that encodes a spatio-genetic map of a specimen via a massive distributed network of DNA molecules inside it, to three dimensions and multiple length scales in developing zebrafish embryos.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552222">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552222" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552222">
        <p class="paperTitle">Simulated musculoskeletal optimization for sprinting and marathon running</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552222" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552222" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Van Wouwe, T.; Hicks, J.; Delp, S.; Liu, K.</p>
        <p class="info">Score: 11.4, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552222' target='https://doi.org/10.1101/2023.08.07.552222'> 10.1101/2023.08.07.552222</a></p>
        <p class="abstract">Musculoskeletal geometry and muscle volumes vary widely in the population and are intricately linked to the performance of tasks ranging from walking and running to jumping and sprinting. However, our ability to understand how these parameters affect task performance has been limited due to the high computational cost of modelling the necessary complexity of the musculoskeletal system and solving the requisite multi-dimensional optimization problem. For example, sprinting and running are fundamental to many forms of sport, but past research on the relationships between musculoskeletal geometry, muscle volumes, and running performance has been limited to observational studies, which have not established cause-effect relationships, and simulation studies with simplified representations of musculoskeletal geometry. In this study, we developed a novel musculoskeletal simulator that is differentiable with respect to musculoskeletal geometry and muscle volumes. This simulator enabled us to find the optimal body segment dimensions and optimal distribution of added muscle volume for sprinting and marathon running. Our simulation results replicate experimental observations, such as increased muscle mass in sprinters, as well a mass in the lower end of the healthy BMI range and a higher leg-length-to-height ratio in marathon runners. The simulations also reveal new relationships, for example showing that hip musculature is vital to both sprinting and marathon running. We found hip flexor and extensor moment arms were maximized to optimize sprint and marathon running performance, and hip muscles the main target when we simulated strength training for sprinters. Our simulation results can help sprint and marathon runners customize strength training, and our simulator can be extended to other athletic tasks, such as jumping, or to non-athletic applications, such as designing interventions to improve mobility in older adults or individuals with movement disorders.

AUTHOR SUMMARYOur study addresses the challenge of determining optimal musculoskeletal parameters for tasks like sprinting and marathon running. Existing research has been limited to observational studies and simplified simulations. To overcome these limitations, we developed a differentiable musculoskeletal simulator to optimize running performance. We replicated past findings and uncovered new insights. We confirmed the benefits of increased muscle mass for sprinters and identified key factors for marathon runners, such a mass in the lower end of the healthy BMI range and an increased leg-length-to-height ratio. Hip musculature was found to be critical for both sprinting and marathon running.

Our simulation results have practical implications. They can inform customized strength training for sprinters and marathon runners. Additionally, the simulator can be extended to other athletic tasks, benefiting various sporting events. Beyond athletics, our open-source simulator has broader applications. It can determine minimal strength requirements for daily activities, guide strength training in the elderly, and estimate the effects of simulated musculoskeletal surgery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552783">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552783" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552783">
        <p class="paperTitle">Conditional Protein Denoising Diffusion Generates Programmable Endonucleases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552783" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552783" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, B.; Zheng, L.; Wu, B.; Yi, K.; Zhong, B.; Lio, P.; Hong, L.</p>
        <p class="info">Score: 8.7, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552783' target='https://doi.org/10.1101/2023.08.10.552783'> 10.1101/2023.08.10.552783</a></p>
        <p class="abstract">Computation or deep learning-based functional protein generation methods address the urgent demand for novel biocatalysts, allowing for precise tailoring of functionalities to meet specific requirements. This emergence leads to the creation of highly efficient and specialized proteins with wide-ranging applications in scientific, technological, and biomedical domains. This study establishes a conditional protein diffusion model, namely CPDiffusion, to deliver diverse protein sequences with desired functions. While the model is free from extensive training data and the sampling process involves little guidance on the type of generated amino acids, CPDiffusion effectively secures essential highly conserved residues that are crucial for protein functionalities. We employed CPDiffusion and generated 27 artificially designed Argonaute proteins, programmable endonucleases applied for easy-to-implement and high-throughput screenings in gene editing and molecular diagnostics, that mutated approximately 200 - 400 amino acids with 40% sequence identities to those from nature. Experimental tests demonstrate the solubility of all 27 artificially-designed proteins (AP), with 24 of them displaying DNA cleavage activity. Remarkably, 74% of active APs exhibited superior activity compared to the template protein, and the most effective one showcased a remarkable nearly nine-fold enhancement of enzymatic activity. Moreover, 37% of APs exhibited enhanced thermostability. These findings emphasize CPDiffusions remarkable capability to generate long-sequence proteins in a single step while retaining or enhancing intricate functionality. This approach facilitates the design of intricate enzymes featuring multi-domain molecular structures through in silico generation and throughput, all accomplished without the need for supervision from labeled data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.10.28.514251">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.10.28.514251" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.10.28.514251">
        <p class="paperTitle">Crystal ribcage: a platform for probing real-time lung function at cellular resolution in health and disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.10.28.514251" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.10.28.514251" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Banerji, R.; Grifno, G. N.; Shi, L.; Smolen, D.; LeBourdais, R.; Muhvich, J.; Eberman, C.; Hiller, B.; Lee, J.; Regan, K.; Zheng, S.; Zhang, S. S.; Jiang, J.; Raslan, A. A.; Breda, J. C.; Pihl, R.; Traber, K.; Mazzilli, S.; Ligresti, G.; Mizgerd, J. P.; Suki, B.; Nia, H. T.</p>
        <p class="info">Score: 13.8, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.10.28.514251' target='https://doi.org/10.1101/2022.10.28.514251'> 10.1101/2022.10.28.514251</a></p>
        <p class="abstract">Understanding the dynamic pathogenesis and treatment response in pulmonary diseases requires probing the lung at cellular resolution in real-time. Despite recent progress in intravital imaging, optical imaging of the lung during active respiration and circulation has remained challenging. Here, we introduce the crystal ribcage: a transparent ribcage that (i) allows truly multiscale optical imaging of the lung in health and disease from whole-organ to single cell, (ii) enables the modulation of lung biophysics and immunity through intravascular, intrapulmonary, intraparenchymal, and optogenetic interventions, and (iii) preserves the 3-D architecture, air-liquid interface, cellular diversity, and respiratory-circulatory functions of the lung. Utilizing these unprecedented capabilities on murine models of primary and metastatic lung tumors, respiratory infection, pulmonary fibrosis, emphysema, and acute lung injury we probed how disease progression remodels the respiratory-circulatory functions at the single alveolus and capillary levels. In cancer, we identified the earliest stage of tumorigenesis that compromises alveolar and capillary functions, a key state with consequences on tumor progression and treatment response. In pneumonia, we mapped mutual links between the recruited immune cells and the alveolar-capillary functions. We found that neutrophil migration is strongly and reversibly responsive to vascular pressure with implications for understanding of how lung physiology, altered by disease and anatomical location, affects immune cell activities. The crystal ribcage and its broad applications presented here will facilitate further studies of real-time remodeling of the alveoli and capillaries during pathogenesis of nearly any pulmonary disease, leading to the identification of new targets for treatment strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553031">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553031" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553031">
        <p class="paperTitle">Second Harmonic Generation Imaging Reveals Entanglement of Collagen Fibers in the Elephant Trunk Skin Dermis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553031" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553031" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schulz, A.; Plotczyk, M.; Sordilla, S.; Boyle, M.; Singal, K.; Reidenberg, J.; Hu, D.; Higgins, C.</p>
        <p class="info">Score: 6.9, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553031' target='https://doi.org/10.1101/2023.08.11.553031'> 10.1101/2023.08.11.553031</a></p>
        <p class="abstract">Form-function relationships often have tradeoffs: if a material is tough, it is often inflexible, and vice versa. This is particularly relevant for the elephant trunk, where the skin should be protective yet elastic. To investigate how this is achieved, we used classical histochemical staining and second harmonic generation microscopy to describe the morphology and composition of elephant trunk skin. We report structure at the macro and micro scales, from the thickness of the dermis to the interaction of 10 m thick collagen fibers. We analyzed several sites along the length of the trunk, to compare and contrast the dorsal-ventral and proximal-distal skin morphologies and compositions. We find the dorsal skin of the elephant trunk can have keratin armor layers over 2mm thick, which is nearly 100 times the thickness of the equivalent layer in human skin. We also found that the structural support layer (the dermis) of elephant trunk contains a distribution of collagen-I (COL1) fibers in both perpendicular and parallel arrangement. The bimodal distribution of collagen is seen across all portions of the trunk, and is dissimilar from that of human skin where one orientation dominates within a body site. We hypothesize that this distribution of COL1 in the elephant trunk allows both flexibility and load-bearing capabilities. Additionally, when viewing individual fiber interaction of 10 m thick collagen, we find the fiber crossings per unit volume are five times more common than in human skin, suggesting that the fibers are entangled. We surmise that these intriguing structures permit both flexibility and strength in the elephant trunk. The complex nature of the elephant skin may inspire the design of materials that can combine strength and flexibility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553241">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553241" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553241">
        <p class="paperTitle">The Enderstruder: An accessible open-source syringe extruder compatible with Ender series 3D printers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553241" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553241" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cordova, D. J.; Rodriguez, A. A.; Woodward, S. C.; Crosby, C. O.</p>
        <p class="info">Score: 6.3, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553241' target='https://doi.org/10.1101/2023.08.15.553241'> 10.1101/2023.08.15.553241</a></p>
        <p class="abstract">Bioprinting has enabled the precise spatiotemporal deposition of cell-containing bioinks, opening new avenues of research in tissue engineering and regenerative medicine. Although several open-source syringe extruder designs have been published and adopted by end users, only one has been specifically adapted for the Ender series, an affordable and open-source line of thermoplastic 3D printers. Here, we introduce the Enderstruder, a cost-effective extruder attachment that uses a standard 10 mL BD syringe, positions the stepper motor at the level of the gantry, enhances x-axis stability with a linear rail, and uses the originally included stepper motor, resulting in reduced cost and simplified assembly. Furthermore, we present an iterative process to fine-tune printing profiles for high-viscosity biomaterial inks. To facilitate the implementation of our work by other groups, we provide fully editable Cura profiles for five commonly used biomaterials. We employ the Enderstruder to print established calibration patterns and complex shapes using each of these five materials to validate and characterize our design. By presenting the Enderstruder and its iterative development process, this study contributes to the growing repository of open-source bioprinting solutions, fostering greater accessibility and affordability for researchers in tissue engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553390">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553390" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553390">
        <p class="paperTitle">Conserved and tissue-specific immune responses to biologic scaffold implantation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553390" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553390" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: DeStefano, S.; Josyula, A.; Faust, M.; Fertil, D.; Lokwani, R.; Ngo, T. B.; Sadtler, K.</p>
        <p class="info">Score: 5.4, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553390' target='https://doi.org/10.1101/2023.08.15.553390'> 10.1101/2023.08.15.553390</a></p>
        <p class="abstract">Upon implantation into a patient, any biomaterial induces a cascade of immune responses that influences the outcome of that device. This cascade depends upon several factors, including the composition of the material itself and the location in which the material is implanted. There is still significant uncertainty around the role of different tissue microenvironments in the immune response to biomaterials and how that may alter downstream scaffold remodeling and integration. In this study, we present a study evaluating the immune response to decellularized extracellular matrix materials within the intraperitoneal cavity, the subcutaneous space, and in a traumatic skeletal muscle injury microenvironment. All different locations induced robust cellular recruitment, specifically of macrophages and eosinophils. The latter was most prominent in the subcutaneous space. Intraperitoneal implants uniquely recruited B cells that may alter downstream reactivity as adaptive immunity has been strongly implicated in the outcome of scaffold remodeling. These data suggest that the location of tissue implants should be taken together with the composition of the material itself when designing devices for downline therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553547">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553547" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553547">
        <p class="paperTitle">A Hybrid Transistor with Transcriptionally Controlled Computation and Plasticity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553547" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553547" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gao, Y.; Zhou, Y.; Ji, X.; Graham, A. J.; Dundas, C. M.; Mahfoud, I. E. M.; Tibbett, B. M.; Tan, B.; Partipilo, G.; Dodabalapur, A.; Rivnay, J.; Keitz, B. K.</p>
        <p class="info">Score: 4.6, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553547' target='https://doi.org/10.1101/2023.08.16.553547'> 10.1101/2023.08.16.553547</a></p>
        <p class="abstract">Organic electrochemical transistors (OECTs) are ideal devices for translating biological signals into electrical readouts and have applications in bioelectronics, biosensing, and neuromorphic computing. Despite their potential, developing programmable and modular methods for living systems to interface with OECTs has proven challenging. Here we describe hybrid OECTs containing the model electroactive bacterium Shewanella oneidensis that enable the transduction of biological computations to electrical responses. Specifically, we fabricated planar p-type OECTs and demonstrated that channel de-doping is driven by extracellular electron transfer (EET) from S. oneidensis. Leveraging this mechanistic understanding and our ability to control EET flux via transcriptional regulation, we used plasmid-based Boolean logic gates to translate biological computation into current changes within the OECT. Finally, we demonstrated EET-driven changes to OECT synaptic plasticity. This work enables fundamental EET studies and OECT-based biosensing and biocomputing systems with genetically controllable and modular design elements.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550304">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550304" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550304">
        <p class="paperTitle">Single-cell precision nanotechnology in vivo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550304" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550304" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: MOLBAY, M.; Kick, B.; Zhao, S.; Todorov, M. I.; Ohn, T.-L.; Roth, S.; Simats, A.; Singh, V.; Khalin, I.; Pan, C.; Singh Bhatia, H.; Hellal, F.; Zeidler, R.; Liesz, A.; Plesnila, N.; Dietz, H.; Erturk, A.</p>
        <p class="info">Score: 148.3, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550304' target='https://doi.org/10.1101/2023.07.24.550304'> 10.1101/2023.07.24.550304</a></p>
        <p class="abstract">Targeting nanoparticle therapeutics with cellular accuracy in whole organisms could open breakthrough opportunities in precision medicine. However, evaluating and fine-tuning the biodistribution of such systems in the whole organism at the cellular level remains a major obstacle. Here, we constructed targetable DNA origami, and analyzed biodistribution in transparent mice, in addition to studying tolerability, clearance kinetics, and immune response parameters. Untargeted DNA origami primarily accumulated in the spleen and the liver, while an immune cell-targeting variant successfully attached to immune cells throughout the body. A cancer cell-targeting mimetic co-localized on solid-tumor metastasis in the liver and the lung. These findings indicate that DNA origami can be directed in vivo, providing an important proof-of-concept and highlights the potential of high-resolution tissue-clearing imaging technologies in their development.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=121 SRC=&#34;FIGDIR/small/550304v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (44K):
org.highwire.dtl.DTLVardef@ce216borg.highwire.dtl.DTLVardef@f40bb4org.highwire.dtl.DTLVardef@2c6a66org.highwire.dtl.DTLVardef@1fd7fde_HPS_FORMAT_FIGEXP  M_FIG C_FIG HighlightsO_LIThis study demonstrates the potential of DNA origami-based drug delivery systems as versatile tool for for targeted delivery, which could be used to treat a range of diseases with applications.
C_LIO_LIThe immune compatibility, half-life, targeting efficiency, and the biodistribution evaluation of DNA origami indicate its potential for systemic drug delivery.
C_LIO_LIOur approach enables the assessment of biodistribution of nanoparticles in the intact body with a sensitivity to the single-cell level, highlighting the high-resolution tissue clearing technoloies in revealing DNA origamis feasibility for drug targeting.
C_LI</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
