<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioengineering</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioengineering | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Feature Reuse and Scaling: Understanding Transfer Learning with Protein Language Models
Authors: Li, F.-Z.; Amini, A. P.; Yue, Y.; Yang, K. K.; Lu, A. X.
Score: 80.9, Published: 2024-02-08 DOI: 10.1101/2024.02.05.578959
Large pretrained protein language models (PLMs) have improved protein property and structure prediction from sequences via transfer learning, in which weights and representations from PLMs are repurposed for downstream tasks. Although PLMs have shown great promise, currently there is little understanding of how the features learned by pretraining relate to and are useful for downstream tasks.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioengineering/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioengineering" />
<meta property="og:description" content="Feature Reuse and Scaling: Understanding Transfer Learning with Protein Language Models
Authors: Li, F.-Z.; Amini, A. P.; Yue, Y.; Yang, K. K.; Lu, A. X.
Score: 80.9, Published: 2024-02-08 DOI: 10.1101/2024.02.05.578959
Large pretrained protein language models (PLMs) have improved protein property and structure prediction from sequences via transfer learning, in which weights and representations from PLMs are repurposed for downstream tasks. Although PLMs have shown great promise, currently there is little understanding of how the features learned by pretraining relate to and are useful for downstream tasks." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioengineering/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-11T10:38:55+00:00" />
<meta property="article:modified_time" content="2024-02-11T10:38:55+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioengineering"/>
<meta name="twitter:description" content="Feature Reuse and Scaling: Understanding Transfer Learning with Protein Language Models
Authors: Li, F.-Z.; Amini, A. P.; Yue, Y.; Yang, K. K.; Lu, A. X.
Score: 80.9, Published: 2024-02-08 DOI: 10.1101/2024.02.05.578959
Large pretrained protein language models (PLMs) have improved protein property and structure prediction from sequences via transfer learning, in which weights and representations from PLMs are repurposed for downstream tasks. Although PLMs have shown great promise, currently there is little understanding of how the features learned by pretraining relate to and are useful for downstream tasks."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioengineering",
      "item": "https://trxiv.yorks0n.com/posts/bioengineering/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioengineering",
  "name": "bioengineering",
  "description": "Feature Reuse and Scaling: Understanding Transfer Learning with Protein Language Models\nAuthors: Li, F.-Z.; Amini, A. P.; Yue, Y.; Yang, K. K.; Lu, A. X.\nScore: 80.9, Published: 2024-02-08 DOI: 10.1101/2024.02.05.578959\nLarge pretrained protein language models (PLMs) have improved protein property and structure prediction from sequences via transfer learning, in which weights and representations from PLMs are repurposed for downstream tasks. Although PLMs have shown great promise, currently there is little understanding of how the features learned by pretraining relate to and are useful for downstream tasks.",
  "keywords": [
    
  ],
  "articleBody": " Feature Reuse and Scaling: Understanding Transfer Learning with Protein Language Models\nAuthors: Li, F.-Z.; Amini, A. P.; Yue, Y.; Yang, K. K.; Lu, A. X.\nScore: 80.9, Published: 2024-02-08 DOI: 10.1101/2024.02.05.578959\nLarge pretrained protein language models (PLMs) have improved protein property and structure prediction from sequences via transfer learning, in which weights and representations from PLMs are repurposed for downstream tasks. Although PLMs have shown great promise, currently there is little understanding of how the features learned by pretraining relate to and are useful for downstream tasks. We perform a systematic analysis of transfer learning using PLMs, conducting 370 experiments across a comprehensive suite of factors including different downstream tasks, architectures, model sizes, model depths, and pretraining time. We observe that while almost all downstream tasks do benefit from pretrained models compared to naive sequence representations, for the majority of tasks performance does not scale with pretraining, and instead relies on low-level features learned early in pretraining. Our results point to a mismatch between current PLM pretraining paradigms and most applications of these models, indicating a need for better pretraining methods.\nGrayscale 4D Biomaterial Customization at High Resolution and Scale\nAuthors: Batalov, I.; Filteau, J. R.; Francis, R. M.; Jaindl, G.; Orr, L.; Rapp, T. L.; Yang, S.; Filteau, J. A.; Xie, W.; Bretherton, R. C.; Glaser, A. K.; Liu, J. T. C.; Stevens, K. R.; DeForest, C. A.\nScore: 38.9, Published: 2024-02-03 DOI: 10.1101/2024.01.31.578280\nHydrogel biomaterials have proven indispensable for three-dimensional (3D) cell culture but have fallen short in replicating the innate physiochemical complexity of real tissue. Though traditional photolithography affords localized material manipulation, robust methods that govern when, where, and to what extent such phototailoring occurs throughout materials would be profoundly enabling towards fabricating more-realistic 3D tissue constructs. Here, we introduce \"grayscale image z-stack-guided multiphoton optical-lithography\" (GIZMO) as a generalizable and intuitive strategy to rapidly photomodulate materials in full 3D non-binary patterns at submicron resolutions spanning large volumes (\u003emm3). Highlighting its versatility, we employ GIZMO to variably photopattern biomolecule release from, protein immobilization to, and degradation within hydrogels based on biologically derived or synthetic grayscale image stacks with unprecedented complexity. We anticipate that GIZMO will enable new opportunities to probe and manipulate cell fates, as well as to engineer complex functional tissue.\nSimple visualization of submicroscopic protein clusters with a phase-separation-based fluorescent reporter.\nAuthors: Mumford, T. R.; Rae, D.; Brackhahn, E.; Idris, A.; Gonzalez-Martinez, D.; Pal, A. A.; Chung, M. C.; Guan, J.; Rhoades, E.; Bugaj, L. J.\nScore: 61.0, Published: 2024-01-13 DOI: 10.1101/2022.07.13.499962\nProtein clustering plays numerous roles in cell physiology and disease. However, protein oligomers can be difficult to detect because they are often too small to appear as puncta in conventional fluorescence microscopy. Here we describe a fluorescent reporter strategy that detects protein clusters with high sensitivity, called CluMPS (Clusters Magnified by Phase Separation). A CluMPS reporter detects and visually amplifies even small clusters of a binding partner, generating large, quantifiable fluorescence condensates. We use computational modeling and optogenetic clustering to demonstrate that CluMPS can detect small oligomers and behaves rationally according to key system parameters. CluMPS detected small aggregates of pathological proteins where the corresponding GFP fusions appeared diffuse. CluMPS also detected and tracked clusters of unmodified and tagged endogenous proteins, and orthogonal CluMPS probes could be multiplexed in cells. CluMPS provides a powerful yet straightforward approach to observe higher-order protein assembly in its native cellular context.\nA real-time, high-performance brain-computer interface for finger decoding and quadcopter control\nAuthors: Willsey, M. S.; Shah, N. P.; Avansino, D. T.; Hahn, N. V.; Jamiolkowski, R. M.; Kamdar, F. B.; Hochberg, L. R.; Willett, F. R.; Henderson, J. M.\nScore: 7.4, Published: 2024-02-08 DOI: 10.1101/2024.02.06.578107\nPeople with paralysis express unmet needs for peer support, leisure activities, and sporting activities. Many within the general population rely on social media and massively multiplayer video games to address these needs. We developed a high-performance finger brain-computer-interface system allowing continuous control of 3 independent finger groups with 2D thumb movements. The system was tested in a human research participant over sequential trials requiring fingers to reach and hold on targets, with an average acquisition rate of 76 targets/minute and completion time of 1.58 {+/-} 0.06 seconds. Performance compared favorably to previous animal studies, despite a 2-fold increase in the decoded degrees-of-freedom (DOF). Finger positions were then used for 4-DOF velocity control of a virtual quadcopter, demonstrating functionality over both fixed and random obstacle courses. This approach shows promise for controlling multiple-DOF end-effectors, such as robotic fingers or digital interfaces for work, entertainment, and socialization.\nEfficient stripe artefact removal by a variational method: application to light-sheet microscopy, FIB-SEM and remote sensing images\nAuthors: Rottmayer, N.; Redenbach, C.; Fahrbach, F.\nScore: 6.5, Published: 2024-02-07 DOI: 10.1101/2024.02.02.578531\nLight-sheet fluorescence microscopy (LSFM) is used to capture volume images of biological specimens. It offers high contrast deep inside densely fluorescence labelled samples, fast acquisition speed and minimal harmful effects on the sample. However, the resulting images often show strong stripe artifacts originating from light-matter interactions. We propose a robust variational method suitable for removing stripes which outperforms existing methods and offers flexibility through two adjustable parameters. This tool is widely applicable to improve visual quality as well as facilitate downstream processing and analysis of images acquired on systems that do not provide hardware-based destriping methods. An evaluation of methods is performed on LSFM, FIB-SEM and remote sensing data, supplemented by synthetic LSFM images. The latter is obtained by simulating the imaging process on virtual samples.\nMeasurement of adhesion and traction of cells at high yield (MATCHY) reveals an energetic ratchet driving nephron condensation\nAuthors: Liu, J.; Prahl, L. S.; Huang, A.; Hughes, A. J.\nScore: 6.1, Published: 2024-02-08 DOI: 10.1101/2024.02.07.579368\nEngineering of embryonic strategies for tissue-building has extraordinary promise for regenerative medicine. This has led to a resurgence in interest in the relationship between cell biophysical properties and morphological transitions. However, mapping gene or protein expression data to cell biophysical properties to physical morphogenesis remains challenging with current techniques. Here we present MATCHY (multiplexed adhesion and traction of cells at high yield). MATCHY advances the multiplexing and throughput capabilities of existing traction force and cell-cell adhesion assays using microfabrication and an automated computation scheme with machine learning-driven cell segmentation. Both biophysical assays are coupled with serial downstream immunofluorescence to extract cell type/signaling state information. MATCHY is especially suited to complex primary tissue-, organoid-, or biopsy-derived cell mixtures since it does not rely on a priori knowledge of cell surface markers, cell sorting, or use of lineage-specific reporter animals. We first validate MATCHY on canine kidney epithelial cells engineered for RET tyrosine kinase expression and quantify a relationship between downstream signaling and cell traction. We go on to create a biophysical atlas of primary cells dissociated from the mouse embryonic kidney and use MATCHY to identify distinct biophysical states along the nephron differentiation trajectory. Our data complement expression-level knowledge of adhesion molecule changes that accompany nephron differentiation with quantitative biophysical information. These data reveal an 'energetic ratchet' that explains spatial nephron progenitor cell condensation from the niche as they differentiate, which we validate through agent-based computational simulation. MATCHY offers automated cell biophysical characterization at \u003e10,000-cell throughput, a highly enabling advance for fundamental studies and new synthetic tissue design strategies for regenerative medicine.\nRepurposing CRISPR-Cas13 systems for robust mRNA trans-splicing\nAuthors: Fiflis, D. N.; Rey, N.; Lavanya, H. V.; Sewell, B.; Mitchell-Dick, A.; Milo, S.; Clements, K.; Benkert, A.; Rosales, A.; Fergione, S.; Asokan, A.\nScore: 5.8, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579165\nType VI CRISPR enzymes have been developed as programmable RNA-guided Cas proteins for eukaryotic RNA editing. Notably, Cas13 has been utilized for site-targeted single base edits, demethylation, RNA cleavage or knockdown and alternative splicing. However, the ability to edit large stretches of mRNA transcripts remains a significant challenge. Here, we demonstrate that CRISPR-Cas13 systems can be repurposed to assist trans-splicing of exogenous RNA fragments into an endogenous pre-mRNA transcript, a method termed CRISPR Assisted mRNA Fragment Trans-splicing (CRAFT). Using split reporter-based assays, we evaluate orthogonal Cas13 systems, optimize guide RNA length and screen for optimal trans-splicing site(s) across a range of intronic targets. We achieve markedly improved editing of large 5 and 3 segments in different endogenous mRNAs across various mammalian cell types compared to other spliceosome-mediated trans-splicing methods. CRAFT can serve as a versatile platform for attachment of protein tags, studying the impact of multiple mutations/single nucleotide polymorphisms, modification of untranslated regions (UTRs) or replacing large segments of mRNA transcripts.\nFocused Ultrasound Blood-Brain Barrier Opening Arrests the Growth and Formation of Cerebral Cavernous Malformations\nAuthors: Fisher, D. G.; Sharifi, K. A.; Shah, I. M.; Gorick, C. M.; Breza, V. R.; Debski, A. C.; Hoch, M.; Cruz, T.; Samuels, J. D.; Sheehan, J.; Schlesinger, D.; Moore, D.; Lukens, J. R.; Tvrdik, P.; Miller, G. W.; Price, R. J.\nScore: 4.8, Published: 2024-02-04 DOI: 10.1101/2024.01.31.577810\nBACKGROUNDCerebral cavernous malformations (CCM) are vascular lesions within the central nervous system, consisting of dilated and hemorrhage-prone capillaries. CCMs can cause debilitating neurological symptoms, and surgical excision or stereotactic radiosurgery are the only current treatment options. Meanwhile, transient blood-brain barrier opening (BBBO) with focused ultrasound (FUS) and microbubbles is now understood to exert potentially beneficial bioeffects, such as stimulation of neurogenesis and clearance of amyloid-{beta}. Here, we tested whether FUS BBBO could be deployed therapeutically to control CCM formation and progression in a clinically-representative murine model. METHODSCCMs were induced in mice by postnatal, endothelial-specific Krit1 ablation. FUS was applied for BBBO with fixed peak-negative pressures (PNPs; 0.2-0.6 MPa) or passive cavitation detection-modulated PNPs. Magnetic resonance imaging (MRI) was used to target FUS treatments, evaluate safety, and measure longitudinal changes in CCM growth after BBBO. RESULTSFUS BBBO elicited gadolinium accumulation primarily at the perilesional boundaries of CCMs, rather than lesion cores. Passive cavitation detection and gadolinium contrast enhancement were comparable in CCM and wild-type mice, indicating that Krit1 ablation does not confer differential sensitivity to FUS BBBO. Acutely, CCMs exposed to FUS BBBO remained structurally stable, with no signs of hemorrhage. Longitudinal MRI revealed that FUS BBBO halted the growth of 94% of CCMs treated in the study. At 1 month, FUS BBBO-treated lesions lost, on average, 9% of their pre-sonication volume. In contrast, non-sonicated control lesions grew to 670% of their initial volume. Lesion control with FUS BBBO was accompanied by a marked reduction in the area and mesenchymal appearance of Krit mutant endothelium. Strikingly, in mice receiving multiple BBBO treatments with fixed PNPs, de novo CCM formation was significantly reduced by 81%. Mock treatment plans on MRIs of patients with surgically inaccessible lesions revealed their lesions are amenable to FUS BBBO with current clinical technology. CONCLUSIONSOur results establish FUS BBBO as a novel, non-invasive modality that can safely arrest murine CCM growth and prevent their de novo formation. As an incisionless, MR image-guided therapy with the ability to target eloquent brain locations, FUS BBBO offers an unparalleled potential to revolutionize the therapeutic experience and enhance the accessibility of treatments for CCM patients.\nIntegrating Safety, Security, Sustainability and Social Responsibility Principles into the U.S Bioeconomy\nAuthors: Gronvall, G. K.; Attal-Juncqua, A.; Getz, J.; Morhard, R.\nScore: 4.7, Published: 2024-02-08 DOI: 10.1101/2024.02.05.578954\nBioindustrial manufacturing is undergoing rapid expansion and investment and is seen as integral to nations economic progress. Ensuring that bioindustrial manufacturing benefits society as the field expands is of critical, urgent importance. To better understand the industrys ethical trajectory and to shape policy, we explored the views of biotechnology leaders on 4 aspects of ethical and social responsibility: - Safety, Security, Social Responsibility, and Sustainability, what we have termed \"4S Principles\". We identified policy actions governments and other stakeholders may take to maximize societal benefits in industrial biotechnology.\nLaser Scanning Confocal printing, a new simple method for micro-device fabrication (LSCprint method).\nAuthors: Megias, D.; Alonso, J. G.; Julia, J. O.; Mata, G.; Martinez, M. P.\nScore: 4.3, Published: 2024-02-09 DOI: 10.1101/2024.02.08.579477\nMicro-devices and lab-on-a-chip technologies have revolutionized cell biology research, enabling a plethora of applications from single-cell sorting to organ-on-a-chip assays. However, their construction remains laborious, requiring specialized equipment and trained personnel, thereby restricting their accessibility to specialized laboratories. The conventional protocol for micro-device printing involves intricate steps, including master cast/mold production and device fabrication, leading to high costs and time consumption. Here, we present a novel, simplified method utilizing a laser scanning confocal microscope (LSCM) and commercially available photosensitive resins. By using the UV or violet excitation laser lines of an LSCM, we eliminate the need for external suppliers and complex equipment, enabling any conventional cell biology laboratory to fabricate micro-devices swiftly and inexpensively. Our method not only enhances the capabilities of standard confocal microscopes but also democratises microfluidic device fabrication, making it accessible to non-specialized laboratories. With minimal reagent consumption and high scalability, our approach offers a cost-effective solution for rapid prototyping and production of micro-devices, circumventing previous barriers to widespread adoption. Moreover, our method allows direct printing of micro-devices onto substrates, eliminating the need for molds and intermediate steps, thus facilitating greater design flexibility and accessibility for non-specialized laboratories.\n",
  "wordCount" : "2160",
  "inLanguage": "en",
  "datePublished": "2024-02-11T10:38:55Z",
  "dateModified": "2024-02-11T10:38:55Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioengineering/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioengineering
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 11, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578959">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578959" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578959">
        <p class="paperTitle">Feature Reuse and Scaling: Understanding Transfer Learning with Protein Language Models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578959" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578959" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, F.-Z.; Amini, A. P.; Yue, Y.; Yang, K. K.; Lu, A. X.</p>
        <p class="info">Score: 80.9, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578959' target='https://doi.org/10.1101/2024.02.05.578959'> 10.1101/2024.02.05.578959</a></p>
        <p class="abstract">Large pretrained protein language models (PLMs) have improved protein property and structure prediction from sequences via transfer learning, in which weights and representations from PLMs are repurposed for downstream tasks. Although PLMs have shown great promise, currently there is little understanding of how the features learned by pretraining relate to and are useful for downstream tasks. We perform a systematic analysis of transfer learning using PLMs, conducting 370 experiments across a comprehensive suite of factors including different downstream tasks, architectures, model sizes, model depths, and pretraining time. We observe that while almost all downstream tasks do benefit from pretrained models compared to naive sequence representations, for the majority of tasks performance does not scale with pretraining, and instead relies on low-level features learned early in pretraining. Our results point to a mismatch between current PLM pretraining paradigms and most applications of these models, indicating a need for better pretraining methods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.578280">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.578280" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.578280">
        <p class="paperTitle">Grayscale 4D Biomaterial Customization at High Resolution and Scale</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.578280" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.578280" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Batalov, I.; Filteau, J. R.; Francis, R. M.; Jaindl, G.; Orr, L.; Rapp, T. L.; Yang, S.; Filteau, J. A.; Xie, W.; Bretherton, R. C.; Glaser, A. K.; Liu, J. T. C.; Stevens, K. R.; DeForest, C. A.</p>
        <p class="info">Score: 38.9, Published: 2024-02-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.578280' target='https://doi.org/10.1101/2024.01.31.578280'> 10.1101/2024.01.31.578280</a></p>
        <p class="abstract">Hydrogel biomaterials have proven indispensable for three-dimensional (3D) cell culture but have fallen short in replicating the innate physiochemical complexity of real tissue. Though traditional photolithography affords localized material manipulation, robust methods that govern when, where, and to what extent such phototailoring occurs throughout materials would be profoundly enabling towards fabricating more-realistic 3D tissue constructs. Here, we introduce &#34;grayscale image z-stack-guided multiphoton optical-lithography&#34; (GIZMO) as a generalizable and intuitive strategy to rapidly photomodulate materials in full 3D non-binary patterns at submicron resolutions spanning large volumes (&gt;mm3). Highlighting its versatility, we employ GIZMO to variably photopattern biomolecule release from, protein immobilization to, and degradation within hydrogels based on biologically derived or synthetic grayscale image stacks with unprecedented complexity. We anticipate that GIZMO will enable new opportunities to probe and manipulate cell fates, as well as to engineer complex functional tissue.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.07.13.499962">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.07.13.499962" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.07.13.499962">
        <p class="paperTitle">Simple visualization of submicroscopic protein clusters with a phase-separation-based fluorescent reporter.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.07.13.499962" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.07.13.499962" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mumford, T. R.; Rae, D.; Brackhahn, E.; Idris, A.; Gonzalez-Martinez, D.; Pal, A. A.; Chung, M. C.; Guan, J.; Rhoades, E.; Bugaj, L. J.</p>
        <p class="info">Score: 61.0, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.07.13.499962' target='https://doi.org/10.1101/2022.07.13.499962'> 10.1101/2022.07.13.499962</a></p>
        <p class="abstract">Protein clustering plays numerous roles in cell physiology and disease. However, protein oligomers can be difficult to detect because they are often too small to appear as puncta in conventional fluorescence microscopy. Here we describe a fluorescent reporter strategy that detects protein clusters with high sensitivity, called CluMPS (Clusters Magnified by Phase Separation). A CluMPS reporter detects and visually amplifies even small clusters of a binding partner, generating large, quantifiable fluorescence condensates. We use computational modeling and optogenetic clustering to demonstrate that CluMPS can detect small oligomers and behaves rationally according to key system parameters. CluMPS detected small aggregates of pathological proteins where the corresponding GFP fusions appeared diffuse. CluMPS also detected and tracked clusters of unmodified and tagged endogenous proteins, and orthogonal CluMPS probes could be multiplexed in cells. CluMPS provides a powerful yet straightforward approach to observe higher-order protein assembly in its native cellular context.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.578107">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.578107" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.578107">
        <p class="paperTitle">A real-time, high-performance brain-computer interface for finger decoding and quadcopter control</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.578107" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.578107" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Willsey, M. S.; Shah, N. P.; Avansino, D. T.; Hahn, N. V.; Jamiolkowski, R. M.; Kamdar, F. B.; Hochberg, L. R.; Willett, F. R.; Henderson, J. M.</p>
        <p class="info">Score: 7.4, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.578107' target='https://doi.org/10.1101/2024.02.06.578107'> 10.1101/2024.02.06.578107</a></p>
        <p class="abstract">People with paralysis express unmet needs for peer support, leisure activities, and sporting activities. Many within the general population rely on social media and massively multiplayer video games to address these needs. We developed a high-performance finger brain-computer-interface system allowing continuous control of 3 independent finger groups with 2D thumb movements. The system was tested in a human research participant over sequential trials requiring fingers to reach and hold on targets, with an average acquisition rate of 76 targets/minute and completion time of 1.58 {&#43;/-} 0.06 seconds. Performance compared favorably to previous animal studies, despite a 2-fold increase in the decoded degrees-of-freedom (DOF). Finger positions were then used for 4-DOF velocity control of a virtual quadcopter, demonstrating functionality over both fixed and random obstacle courses. This approach shows promise for controlling multiple-DOF end-effectors, such as robotic fingers or digital interfaces for work, entertainment, and socialization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.578531">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.578531" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.578531">
        <p class="paperTitle">Efficient stripe artefact removal by a variational method: application to light-sheet microscopy, FIB-SEM and remote sensing images</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.578531" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.578531" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rottmayer, N.; Redenbach, C.; Fahrbach, F.</p>
        <p class="info">Score: 6.5, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.578531' target='https://doi.org/10.1101/2024.02.02.578531'> 10.1101/2024.02.02.578531</a></p>
        <p class="abstract">Light-sheet fluorescence microscopy (LSFM) is used to capture volume images of biological specimens. It offers high contrast deep inside densely fluorescence labelled samples, fast acquisition speed and minimal harmful effects on the sample. However, the resulting images often show strong stripe artifacts originating from light-matter interactions. We propose a robust variational method suitable for removing stripes which outperforms existing methods and offers flexibility through two adjustable parameters. This tool is widely applicable to improve visual quality as well as facilitate downstream processing and analysis of images acquired on systems that do not provide hardware-based destriping methods. An evaluation of methods is performed on LSFM, FIB-SEM and remote sensing data, supplemented by synthetic LSFM images. The latter is obtained by simulating the imaging process on virtual samples.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.579368">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.579368" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.579368">
        <p class="paperTitle">Measurement of adhesion and traction of cells at high yield (MATCHY) reveals an energetic ratchet driving nephron condensation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.579368" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.579368" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, J.; Prahl, L. S.; Huang, A.; Hughes, A. J.</p>
        <p class="info">Score: 6.1, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.579368' target='https://doi.org/10.1101/2024.02.07.579368'> 10.1101/2024.02.07.579368</a></p>
        <p class="abstract">Engineering of embryonic strategies for tissue-building has extraordinary promise for regenerative medicine. This has led to a resurgence in interest in the relationship between cell biophysical properties and morphological transitions. However, mapping gene or protein expression data to cell biophysical properties to physical morphogenesis remains challenging with current techniques. Here we present MATCHY (multiplexed adhesion and traction of cells at high yield). MATCHY advances the multiplexing and throughput capabilities of existing traction force and cell-cell adhesion assays using microfabrication and an automated computation scheme with machine learning-driven cell segmentation. Both biophysical assays are coupled with serial downstream immunofluorescence to extract cell type/signaling state information. MATCHY is especially suited to complex primary tissue-, organoid-, or biopsy-derived cell mixtures since it does not rely on a priori knowledge of cell surface markers, cell sorting, or use of lineage-specific reporter animals. We first validate MATCHY on canine kidney epithelial cells engineered for RET tyrosine kinase expression and quantify a relationship between downstream signaling and cell traction. We go on to create a biophysical atlas of primary cells dissociated from the mouse embryonic kidney and use MATCHY to identify distinct biophysical states along the nephron differentiation trajectory. Our data complement expression-level knowledge of adhesion molecule changes that accompany nephron differentiation with quantitative biophysical information. These data reveal an &#39;energetic ratchet&#39; that explains spatial nephron progenitor cell condensation from the niche as they differentiate, which we validate through agent-based computational simulation. MATCHY offers automated cell biophysical characterization at &gt;10,000-cell throughput, a highly enabling advance for fundamental studies and new synthetic tissue design strategies for regenerative medicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579165">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579165" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579165">
        <p class="paperTitle">Repurposing CRISPR-Cas13 systems for robust mRNA trans-splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579165" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579165" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fiflis, D. N.; Rey, N.; Lavanya, H. V.; Sewell, B.; Mitchell-Dick, A.; Milo, S.; Clements, K.; Benkert, A.; Rosales, A.; Fergione, S.; Asokan, A.</p>
        <p class="info">Score: 5.8, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579165' target='https://doi.org/10.1101/2024.02.06.579165'> 10.1101/2024.02.06.579165</a></p>
        <p class="abstract">Type VI CRISPR enzymes have been developed as programmable RNA-guided Cas proteins for eukaryotic RNA editing. Notably, Cas13 has been utilized for site-targeted single base edits, demethylation, RNA cleavage or knockdown and alternative splicing. However, the ability to edit large stretches of mRNA transcripts remains a significant challenge. Here, we demonstrate that CRISPR-Cas13 systems can be repurposed to assist trans-splicing of exogenous RNA fragments into an endogenous pre-mRNA transcript, a method termed CRISPR Assisted mRNA Fragment Trans-splicing (CRAFT). Using split reporter-based assays, we evaluate orthogonal Cas13 systems, optimize guide RNA length and screen for optimal trans-splicing site(s) across a range of intronic targets. We achieve markedly improved editing of large 5 and 3 segments in different endogenous mRNAs across various mammalian cell types compared to other spliceosome-mediated trans-splicing methods. CRAFT can serve as a versatile platform for attachment of protein tags, studying the impact of multiple mutations/single nucleotide polymorphisms, modification of untranslated regions (UTRs) or replacing large segments of mRNA transcripts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.577810">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.577810" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.577810">
        <p class="paperTitle">Focused Ultrasound Blood-Brain Barrier Opening Arrests the Growth and Formation of Cerebral Cavernous Malformations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.577810" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.577810" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fisher, D. G.; Sharifi, K. A.; Shah, I. M.; Gorick, C. M.; Breza, V. R.; Debski, A. C.; Hoch, M.; Cruz, T.; Samuels, J. D.; Sheehan, J.; Schlesinger, D.; Moore, D.; Lukens, J. R.; Tvrdik, P.; Miller, G. W.; Price, R. J.</p>
        <p class="info">Score: 4.8, Published: 2024-02-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.577810' target='https://doi.org/10.1101/2024.01.31.577810'> 10.1101/2024.01.31.577810</a></p>
        <p class="abstract">BACKGROUNDCerebral cavernous malformations (CCM) are vascular lesions within the central nervous system, consisting of dilated and hemorrhage-prone capillaries. CCMs can cause debilitating neurological symptoms, and surgical excision or stereotactic radiosurgery are the only current treatment options. Meanwhile, transient blood-brain barrier opening (BBBO) with focused ultrasound (FUS) and microbubbles is now understood to exert potentially beneficial bioeffects, such as stimulation of neurogenesis and clearance of amyloid-{beta}. Here, we tested whether FUS BBBO could be deployed therapeutically to control CCM formation and progression in a clinically-representative murine model.

METHODSCCMs were induced in mice by postnatal, endothelial-specific Krit1 ablation. FUS was applied for BBBO with fixed peak-negative pressures (PNPs; 0.2-0.6 MPa) or passive cavitation detection-modulated PNPs. Magnetic resonance imaging (MRI) was used to target FUS treatments, evaluate safety, and measure longitudinal changes in CCM growth after BBBO.

RESULTSFUS BBBO elicited gadolinium accumulation primarily at the perilesional boundaries of CCMs, rather than lesion cores. Passive cavitation detection and gadolinium contrast enhancement were comparable in CCM and wild-type mice, indicating that Krit1 ablation does not confer differential sensitivity to FUS BBBO. Acutely, CCMs exposed to FUS BBBO remained structurally stable, with no signs of hemorrhage. Longitudinal MRI revealed that FUS BBBO halted the growth of 94% of CCMs treated in the study. At 1 month, FUS BBBO-treated lesions lost, on average, 9% of their pre-sonication volume. In contrast, non-sonicated control lesions grew to 670% of their initial volume. Lesion control with FUS BBBO was accompanied by a marked reduction in the area and mesenchymal appearance of Krit mutant endothelium. Strikingly, in mice receiving multiple BBBO treatments with fixed PNPs, de novo CCM formation was significantly reduced by 81%. Mock treatment plans on MRIs of patients with surgically inaccessible lesions revealed their lesions are amenable to FUS BBBO with current clinical technology.

CONCLUSIONSOur results establish FUS BBBO as a novel, non-invasive modality that can safely arrest murine CCM growth and prevent their de novo formation. As an incisionless, MR image-guided therapy with the ability to target eloquent brain locations, FUS BBBO offers an unparalleled potential to revolutionize the therapeutic experience and enhance the accessibility of treatments for CCM patients.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578954">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578954" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578954">
        <p class="paperTitle">Integrating Safety, Security, Sustainability and Social Responsibility Principles into the U.S Bioeconomy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578954" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578954" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gronvall, G. K.; Attal-Juncqua, A.; Getz, J.; Morhard, R.</p>
        <p class="info">Score: 4.7, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578954' target='https://doi.org/10.1101/2024.02.05.578954'> 10.1101/2024.02.05.578954</a></p>
        <p class="abstract">Bioindustrial manufacturing is undergoing rapid expansion and investment and is seen as integral to nations economic progress. Ensuring that bioindustrial manufacturing benefits society as the field expands is of critical, urgent importance. To better understand the industrys ethical trajectory and to shape policy, we explored the views of biotechnology leaders on 4 aspects of ethical and social responsibility: - Safety, Security, Social Responsibility, and Sustainability, what we have termed &#34;4S Principles&#34;. We identified policy actions governments and other stakeholders may take to maximize societal benefits in industrial biotechnology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.08.579477">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.08.579477" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.08.579477">
        <p class="paperTitle">Laser Scanning Confocal printing, a new simple method for micro-device fabrication (LSCprint method).</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.08.579477" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.08.579477" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Megias, D.; Alonso, J. G.; Julia, J. O.; Mata, G.; Martinez, M. P.</p>
        <p class="info">Score: 4.3, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.08.579477' target='https://doi.org/10.1101/2024.02.08.579477'> 10.1101/2024.02.08.579477</a></p>
        <p class="abstract">Micro-devices and lab-on-a-chip technologies have revolutionized cell biology research, enabling a plethora of applications from single-cell sorting to organ-on-a-chip assays. However, their construction remains laborious, requiring specialized equipment and trained personnel, thereby restricting their accessibility to specialized laboratories. The conventional protocol for micro-device printing involves intricate steps, including master cast/mold production and device fabrication, leading to high costs and time consumption. Here, we present a novel, simplified method utilizing a laser scanning confocal microscope (LSCM) and commercially available photosensitive resins. By using the UV or violet excitation laser lines of an LSCM, we eliminate the need for external suppliers and complex equipment, enabling any conventional cell biology laboratory to fabricate micro-devices swiftly and inexpensively. Our method not only enhances the capabilities of standard confocal microscopes but also democratises microfluidic device fabrication, making it accessible to non-specialized laboratories. With minimal reagent consumption and high scalability, our approach offers a cost-effective solution for rapid prototyping and production of micro-devices, circumventing previous barriers to widespread adoption. Moreover, our method allows direct printing of micro-devices onto substrates, eliminating the need for molds and intermediate steps, thus facilitating greater design flexibility and accessibility for non-specialized laboratories.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
