<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Evo-devo dynamics of hominin brain size
Authors: Gonzalez-Forero, M.
Score: 33.4, Published: 2023-08-28 DOI: 10.1101/2023.03.20.533421
Brain size tripled in the human lineage over four million years, but why this occurred remains uncertain. To advance our understanding of what caused hominin-brain expansion, I mechanistically replicate it in-silico by modelling the evolutionary and developmental (evo-devo) dynamics of hominin-brain size. I show that, starting from australop-ithecine brain and body sizes, the model recovers the evolution of brain and body sizes of seven hominin species, the evolution of the hominin brain-body allometry, and major patterns of human development and evolution.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Evo-devo dynamics of hominin brain size
Authors: Gonzalez-Forero, M.
Score: 33.4, Published: 2023-08-28 DOI: 10.1101/2023.03.20.533421
Brain size tripled in the human lineage over four million years, but why this occurred remains uncertain. To advance our understanding of what caused hominin-brain expansion, I mechanistically replicate it in-silico by modelling the evolutionary and developmental (evo-devo) dynamics of hominin-brain size. I show that, starting from australop-ithecine brain and body sizes, the model recovers the evolution of brain and body sizes of seven hominin species, the evolution of the hominin brain-body allometry, and major patterns of human development and evolution." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-03T10:37:18+00:00" />
<meta property="article:modified_time" content="2023-09-03T10:37:18+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Evo-devo dynamics of hominin brain size
Authors: Gonzalez-Forero, M.
Score: 33.4, Published: 2023-08-28 DOI: 10.1101/2023.03.20.533421
Brain size tripled in the human lineage over four million years, but why this occurred remains uncertain. To advance our understanding of what caused hominin-brain expansion, I mechanistically replicate it in-silico by modelling the evolutionary and developmental (evo-devo) dynamics of hominin-brain size. I show that, starting from australop-ithecine brain and body sizes, the model recovers the evolution of brain and body sizes of seven hominin species, the evolution of the hominin brain-body allometry, and major patterns of human development and evolution."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Evo-devo dynamics of hominin brain size\nAuthors: Gonzalez-Forero, M.\nScore: 33.4, Published: 2023-08-28 DOI: 10.1101/2023.03.20.533421\nBrain size tripled in the human lineage over four million years, but why this occurred remains uncertain. To advance our understanding of what caused hominin-brain expansion, I mechanistically replicate it in-silico by modelling the evolutionary and developmental (evo-devo) dynamics of hominin-brain size. I show that, starting from australop-ithecine brain and body sizes, the model recovers the evolution of brain and body sizes of seven hominin species, the evolution of the hominin brain-body allometry, and major patterns of human development and evolution.",
  "keywords": [
    
  ],
  "articleBody": " Evo-devo dynamics of hominin brain size\nAuthors: Gonzalez-Forero, M.\nScore: 33.4, Published: 2023-08-28 DOI: 10.1101/2023.03.20.533421\nBrain size tripled in the human lineage over four million years, but why this occurred remains uncertain. To advance our understanding of what caused hominin-brain expansion, I mechanistically replicate it in-silico by modelling the evolutionary and developmental (evo-devo) dynamics of hominin-brain size. I show that, starting from australop-ithecine brain and body sizes, the model recovers the evolution of brain and body sizes of seven hominin species, the evolution of the hominin brain-body allometry, and major patterns of human development and evolution. Analysis shows that in this model the brain expands because it is \"socio-genetically\" correlated with developmentally late preovulatory ovarian follicles, not because brain size is directly selected for. The socio-genetic correlation causing the recovered hominin brain expansion is generated over development by ecology and possibly culture. Thus, in this model, direct selection that does not favour brain expansion provides a force that developmental constraints divert causing hominin-brain expansion.\nAn efficient and robust ABC approach to infer the rate and strength of adaptation\nAuthors: Murga-Moreno, J.; Casillas, S.; Barbadilla, A.; Uricchio, L.; Enard, D.\nScore: 16.2, Published: 2023-08-31 DOI: 10.1101/2023.08.29.555322\nInferring the effects of positive selection on genomes remains a critical step in characterizing the ultimate and proximate causes of adaptation across species, and quantifying positive selection remains a challenge due to the confounding effects of many other evolutionary processes. Robust and efficient approaches for adaptation inference could help characterize the rate and strength of adaptation in non-model species for which demographic history, mutational processes, and recombination patterns are not currently well-described. Here, we introduce an efficient and user-friendly extension of of the McDonald-Kreitman test (ABC-MK) for quantifying long-term protein adaptation in specific lineages of interest.. We characterize the performance of our approach with forward simulations and find that it is robust to many demographic perturbations positive selection configurations, demonstrating its suitability for applications to non-model genomes. We apply ABC-MK to the human proteome and a set of known Virus Interacting Proteins (VIPs) to test the long-term adaptation in genes interacting with VIPs, testing for specific pathogenic players associated with human evolution. We find substantially stronger signatures of positive selection on RNA-VIPs than DNA-VIPs, suggesting that RNA-VIPs may be an important driver of human adaptation over deep evolutionary time scales.\nThe highly rugged yet navigable regulatory landscape of the bacterial transcription factor TetR\nAuthors: Westmann, C. A.; Goldbach, L.; Wagner, A.\nScore: 12.2, Published: 2023-08-27 DOI: 10.1101/2023.08.25.554764\nTranscription factor binding sites (TFBSs) are important sources of evolutionary innovations. Understanding how evolution navigates the sequence space of such sites can be achieved by mapping TFBS adaptive landscapes. In such a landscape, an individual location corresponds to a TFBS bound by a transcription factor. The elevation at that location corresponds to the strength of transcriptional regulation conveyed by the sequence. We developed an in vivo massively parallel reporter assay to map the landscape of bacterial TFBSs. We applied this assay to the TetR repressor, for which few TFBSs are known. We quantify the strength of transcriptional repression for 17,765 TFBSs and show that the resulting landscape is highly rugged, with 2,092 peaks. Only a few peaks convey stronger repression than the wild type. Non-additive (epistatic) interactions between mutations are frequent. Despite these hallmarks of ruggedness, most high peaks are evolutionarily accessible. They have large basins of attraction and are reached by around 20% of populations evolving on the landscape. Which high peak is reached during evolution is unpredictable and contingent on the mutational path taken. This first in-depth analysis of a prokaryotic gene regulator reveals a landscape that is navigable but much more rugged than the landscapes of eukaryotic regulators. SignificanceUnderstanding how evolution explores the vast space of genotypic possibilities is a fundamental question in evolutionary biology. The mapping of genotypes to quantitative traits (such as phenotypes and fitness) allows us to delineate adaptive landscapes and their topological properties, shedding light on how evolution can navigate such vast spaces. In this study, we focused on mapping a transcription factor binding site (TFBS) landscape to gene expression levels, as changes in gene expression patterns play a crucial role in biological innovation. We developed a massively parallel reporter assay and mapped the first comprehensive in vivo gene regulatory landscape for a bacterial transcriptional regulator, TetR. Surprisingly, this landscape is way more rugged than those observed in eukaryotic regulators. Despite its ruggedness, the landscape remains highly navigable through adaptive evolution. Our study presents the first high-resolution landscape for a bacterial TFBS, offering valuable insights into the evolution of TFBS in vivo. Moreover, it holds promise as a framework for discovering new genetic components for synthetic biological systems.\nRemote homolog detection places insect chemoreceptors in a cryptic protein superfamily spanning the tree of life\nAuthors: Himmel, N. J.; Moi, D.; Benton, R.\nScore: 10.4, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555482\nMany proteins exist in the so-called 'twilight zone' of sequence alignment, where low pairwise sequence identity makes it difficult to determine homology and phylogeny. As protein tertiary structure is often more conserved, recent advances in ab initio protein folding have made structure-based identification of putative homologs feasible. However, structural screening and phylogenetics are in their infancy, particularly for twilight zone proteins. We present a pipeline for the identification and characterization of distant homologs, and apply it to 7-transmembrane domain ion channels (7TMICs), a protein group founded by insect Odorant and Gustatory receptors. Previous sequence and limited structure-based searches identified putatively-related proteins, mainly in other animals and plants. However, very few 7TMICs have been identified in non-animal, non-plant taxa. Moreover, these proteins' remarkable sequence dissimilarity made it uncertain if disparate 7TMIC types (Gr/Or, Grl, GRL, DUF3537, PHTF and GrlHz) are homologous or convergent, leaving their evolutionary history unresolved. Our pipeline identified thousands of new 7TMICs in archaea, bacteria and unicellular eukaryotes. Using graph-based analyses and protein language models to extract family-wide signatures, we demonstrate that 7TMICs have structure and sequence similarity, supporting homology. Through sequence and structure-based phylogenetics, we classify eukaryotic 7TMICs into two families (Class-A and Class-B), which are the result of a gene duplication predating the split(s) leading to Amorphea (animals, fungi and allies) and Diaphoretickes (plants and allies). Our work reveals 7TMICs as a cryptic superfamily with origins close to the evolution of cellular life. More generally, this study serves as a methodological proof of principle for the identification of extremely distant protein homologs.\nGene loss and cis-regulatory novelty shaped core histone gene evolution in the apiculate yeast Hanseniaspora uvarum\nAuthors: Haase, M. A. B.; Steenwyk, J. L.; Boeke, J. D.\nScore: 10.3, Published: 2023-09-01 DOI: 10.1101/2023.08.28.551515\nAcross eukaryotic species, core histone genes display a remarkable diversity of cis-regulatory mechanisms despite their protein sequence conservation. However, the dynamics and significance of this regulatory turnover are not well understood. Here we describe the evolutionary history of core histone gene regulation across 400 million years in Saccharomycotina yeasts, revealing diverse lineage-specific solutions to core histone expression. We further characterize the emergence of a novel regulatory mode in the Hanseniaspora genus, which coincided with the loss of one copy of its paralogous core histones genes from its fast-evolving lineage. By analyzing the growth dynamics using live cell imaging of genetically modified Hanseniaspora uvarum, we observed a regulatory decoupling of core histone synthesis from DNA synthesis and propose that this may be adaptive for its rapid cell cycle progression. Overall, our findings imply that the frequent turnover of core histone cis-regulatory mechanisms likely provides distinct adaptive solutions for specific life histories.\nChloramphenicol reduces phage resistance evolution by suppressing bacterial cell surface mutants\nAuthors: Parab, L.; Romeyer Dherbey, J.; Rivera, N.; Schwarz, M.; Bertels, F.\nScore: 9.8, Published: 2023-08-28 DOI: 10.1101/2023.08.28.552763\nBacteriophages infect Gram-negative bacteria by attaching to molecules present on the bacterial outer membrane, often lipopolysaccharides (LPS). Modification of the LPS can lead to phage resistance. LPS modifications also impact antibiotic susceptibility, allowing for phage-antibiotic synergism. The mechanism for these synergistic interactions is unclear. Here, we show that antibiotics affect the evolution of phage resistance using phage {Phi}X174 and Escherichia coli C wildtype. We use a collection of E. coli C LPS mutants, each of which is resistant to {Phi}X174, and has either a \"rough\" or \"deep-rough\" LPS phenotype. Growth of deep rough mutants is inhibited at subinhibitory chloramphenicol concentrations. In contrast, gentamicin has no major effect on growth. Hypothesis testing shows that treating E. coli C wildtype with {Phi}X174 and chloramphenicol eliminates deep rough mutants, and reduces phage resistance evolution. Our results show that differential survival of phage resistant mutants with antibiotics explains phage-antibiotic synergism in our model system.\nAcoelomorph flatworm monophyly is a severe long-branch attraction artefact obscuring a clade of Acoela and Xenoturbellida\nAuthors: Redmond, A. K.\nScore: 9.7, Published: 2023-08-27 DOI: 10.1101/2023.08.27.555014\nAcoelomorpha is a broadly accepted clade of bilaterian animals made up of the fast-evolving, morphologically simple, mainly marine flatworm lineages Acoela and Nemertodermatida. Phylogenomic studies support Acoelomorphas close relationship with the slowly evolving and similarly simplistic Xenoturbella, together forming the phylum Xenacoelomorpha. The phylogenetic placement of Xenacoelomorpha amongst bilaterians is controversial, with some studies supporting Xenacoelomorpha as the sister group to all other bilaterians, implying that their simplicity may be representative of early bilaterians. Others propose that this placement is a long branch attraction artefact resulting from the fast-evolving Acoelomorpha, and instead suggest that they are the secondarily simplified sister group of the deuterostome clade Ambulacraria. Perhaps as a result of this debate, internal xenacoelomorph relationships have been somewhat overlooked at a phylogenomic scale. Here, I employ both empirical and simulation approaches to detect and overcome phylogenomic errors to reassess the relationship between Xenoturbella and the fast evolving acoelomorph flatworms. I conclude that subphylum Acoelomorpha is a long-branch attraction artefact obscuring a previously undiscovered clade comprising Xenoturbella and Acoela, for which I propose the name Xenacoela. These analyses are also consistent with the Nephrozoa hypothesis deriving from systematic error, and instead generally favour a close, but unclear, relationship of Xenacoelomorpha with deuterostomes. This study provides a template for future efforts aimed at discovering and correcting unrecognised long-branch attraction artefacts throughout the tree of life.\nA cell-type atlas from a scyphozoan jellyfish Aurelia coerulea (formerly sp.1) provides insights into changes of cell-type diversity in the transition from polyps to medusae\nAuthors: Link, O.; Jahnel, S. M.; Janicek, K.; Kraus, J.; Montenegro, J. D.; Zimmerman, B.; Cole, A. G.; Technau, U.\nScore: 12.4, Published: 2023-08-29 DOI: 10.1101/2023.08.24.554571\nWe provide here a detailed single cell transcriptomic atlas covering the life cycle of the cosmopolitan scyphozoan Aurelia coerulea. We show that there is an increase in cell type diversity in the medusa stage, which is reflected by an increase in the number of unique transcripts expressed. We highlight parallels in both cell complement and specification pathways between Aurelia and the sea anemone Nematostella, two lineages separated by more than 500 MY. We find that several cell types of the neuroglandular lineages are specified by the same transcription factors that arose by gene duplications in the common ancestor of medusozoans and anthozoans. This confirms gene duplications as a source for the increase of cellular complexity during the evolution of cnidarian neuroglandular lineages dates to the common ancestor of all cnidarians. We validated spatial expression domains corresponding to molecular profiles of smooth and striated muscles and followed the development of the striated muscle in the medusa. Our data suggests that striated muscles in the subumbrella of the medusa derive directly from smooth muscles in the jellyfish, and that these two muscle cell types are characterized by distinct molecular profiles. Further, reconstruction of putative cell trajectories indicates multipotent cell states derive from tissues at the boundary between epithelial layers and highlight molecular characteristics not typical of other cnidarians. TeaserSingle cell transcriptomic atlas across the jellyfish life cycle reveals increased cellular complexity associated with the medusa form.\nLarge-scale analyses reveal the contribution of adaptive evolution in pathogenic and non-pathogenic fungal species\nAuthors: Pereira, D.; Bolton, M. D.; Friesen, T. L.; Stephan, W.; Dutheil, J. Y.; Stukenbrock, E. H.\nScore: 7.3, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555124\nGenome studies of fungal pathogens have presented evidence for exceptionally high rates of evolution. It has been proposed that rapid adaptation is a hallmark of pathogen evolution that facilitates the invasion of new host niches and the overcoming of intervention strategies such as fungicide applications and drug treatments. To which extent high levels of genetic variation within and between species correlate with adaptive protein evolution in fungi more generally has so far not been explored. In this study, we addressed the contribution of adaptive evolution relative to genetic drift in 20 fungal species, hereby exploring genetic variation in 2,478 fungal genomes. We reannotated positions of protein-coding genes to obtain a high-quality dataset of 234,427 full-length core gene and 25,612 accessory gene alignments. We applied an extension of the McDonald-Kreitman test that models the distributions of fitness effects to infer the rate of adaptive ({omega}A) and non-adaptive ({omega}NA) non-synonymous substitutions in protein-coding genes. To explore the relevance of recombination on local adaptation rates, we inferred the population genomic recombination rate for all 20 species. Our analyses reveal extensive variation in rates of adaptation and show that high rates of adaptation are not a hallmark of a pathogenic lifestyle. Up to 83% of non-synonymous substitutions are adaptive in the species Parastagonospora nodorum. However, non-synonymous substitutions in other species, including the prominent rice-infecting pathogen Magnaporthe oryzae, are predominantly non-adaptive (neutral or slightly deleterious). Correlating adaptation measures with effective population size and recombination rate, we show that effective population size is a primary determinant of adaptive evolution in fungi. At the genome scale, recombination rate variation explains variation in both {omega}A and {omega}NA. Finally, we demonstrate the robustness of our estimates using simulations. We underline the value of population genetic principles in studies of fungal evolution, and we highlight the importance of demographic processes in adaptive evolution of pathogenic and non-pathogenic species.\nOne Method to Sequence Them All? Comparison between Whole-Genome Sequencing (WGS) and Target Enrichment (TE) of museum specimens from the moth families Epicopeiidae and Sematuridae (Lepidoptera)\nAuthors: Call, E.; Twort, V. G.; Espeland, M.; Wahlberg, N.\nScore: 15.1, Published: 2023-08-25 DOI: 10.1101/2023.08.21.553699\nThere are various possibilities for sequencing highly degraded DNA, such as target enrichment (TE), or whole-genome sequencing (WGS). Here we compare TE and WGS methods using old museum specimens of two families of moths in the superfamily Geometroidea: Epicopeiidae and Sematuridae. Until recently, the relationships of these two families were unclear, as few studies had been done. Recently two studies used the TE approach, either on relatively fresh specimens, or on old museum specimens. Here, we aim to increase the sampling of the families Epicopeiidae and Sematuridae from museum specimens using the WGS method. We show that both sequencing methods give comparable results, but, unsurprisingly, WGS recovers more data. By combining TE and WGS data, we confirm that Sematuridae are sister to Pseudobistonidae+Epicopeiidae. Relationships of genera within the families are well supported. With the costs of WGS decreasing, we suggest that using low-coverage whole genome sequencing is becoming an increasingly viable option in the phylogenomics of insects.\n",
  "wordCount" : "2504",
  "inLanguage": "en",
  "datePublished": "2023-09-03T10:37:18Z",
  "dateModified": "2023-09-03T10:37:18Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on September 3, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.03.20.533421">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.03.20.533421" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.03.20.533421">
        <p class="paperTitle">Evo-devo dynamics of hominin brain size</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.03.20.533421" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.03.20.533421" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gonzalez-Forero, M.</p>
        <p class="info">Score: 33.4, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.03.20.533421' target='https://doi.org/10.1101/2023.03.20.533421'> 10.1101/2023.03.20.533421</a></p>
        <p class="abstract">Brain size tripled in the human lineage over four million years, but why this occurred remains uncertain. To advance our understanding of what caused hominin-brain expansion, I mechanistically replicate it in-silico by modelling the evolutionary and developmental (evo-devo) dynamics of hominin-brain size. I show that, starting from australop-ithecine brain and body sizes, the model recovers the evolution of brain and body sizes of seven hominin species, the evolution of the hominin brain-body allometry, and major patterns of human development and evolution. Analysis shows that in this model the brain expands because it is &#34;socio-genetically&#34; correlated with developmentally late preovulatory ovarian follicles, not because brain size is directly selected for. The socio-genetic correlation causing the recovered hominin brain expansion is generated over development by ecology and possibly culture. Thus, in this model, direct selection that does not favour brain expansion provides a force that developmental constraints divert causing hominin-brain expansion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555322">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555322" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555322">
        <p class="paperTitle">An efficient and robust ABC approach to infer the rate and strength of adaptation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555322" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555322" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Murga-Moreno, J.; Casillas, S.; Barbadilla, A.; Uricchio, L.; Enard, D.</p>
        <p class="info">Score: 16.2, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555322' target='https://doi.org/10.1101/2023.08.29.555322'> 10.1101/2023.08.29.555322</a></p>
        <p class="abstract">Inferring the effects of positive selection on genomes remains a critical step in characterizing the ultimate and proximate causes of adaptation across species, and quantifying positive selection remains a challenge due to the confounding effects of many other evolutionary processes. Robust and efficient approaches for adaptation inference could help characterize the rate and strength of adaptation in non-model species for which demographic history, mutational processes, and recombination patterns are not currently well-described. Here, we introduce an efficient and user-friendly extension of of the McDonald-Kreitman test (ABC-MK) for quantifying long-term protein adaptation in specific lineages of interest.. We characterize the performance of our approach with forward simulations and find that it is robust to many demographic perturbations positive selection configurations, demonstrating its suitability for applications to non-model genomes. We apply ABC-MK to the human proteome and a set of known Virus Interacting Proteins (VIPs) to test the long-term adaptation in genes interacting with VIPs, testing for specific pathogenic players associated with human evolution. We find substantially stronger signatures of positive selection on RNA-VIPs than DNA-VIPs, suggesting that RNA-VIPs may be an important driver of human adaptation over deep evolutionary time scales.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554764">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554764" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554764">
        <p class="paperTitle">The highly rugged yet navigable regulatory landscape of the bacterial transcription factor TetR</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554764" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554764" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Westmann, C. A.; Goldbach, L.; Wagner, A.</p>
        <p class="info">Score: 12.2, Published: 2023-08-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554764' target='https://doi.org/10.1101/2023.08.25.554764'> 10.1101/2023.08.25.554764</a></p>
        <p class="abstract">Transcription factor binding sites (TFBSs) are important sources of evolutionary innovations. Understanding how evolution navigates the sequence space of such sites can be achieved by mapping TFBS adaptive landscapes. In such a landscape, an individual location corresponds to a TFBS bound by a transcription factor. The elevation at that location corresponds to the strength of transcriptional regulation conveyed by the sequence. We developed an in vivo massively parallel reporter assay to map the landscape of bacterial TFBSs. We applied this assay to the TetR repressor, for which few TFBSs are known. We quantify the strength of transcriptional repression for 17,765 TFBSs and show that the resulting landscape is highly rugged, with 2,092 peaks. Only a few peaks convey stronger repression than the wild type. Non-additive (epistatic) interactions between mutations are frequent. Despite these hallmarks of ruggedness, most high peaks are evolutionarily accessible. They have large basins of attraction and are reached by around 20% of populations evolving on the landscape. Which high peak is reached during evolution is unpredictable and contingent on the mutational path taken. This first in-depth analysis of a prokaryotic gene regulator reveals a landscape that is navigable but much more rugged than the landscapes of eukaryotic regulators.

SignificanceUnderstanding how evolution explores the vast space of genotypic possibilities is a fundamental question in evolutionary biology. The mapping of genotypes to quantitative traits (such as phenotypes and fitness) allows us to delineate adaptive landscapes and their topological properties, shedding light on how evolution can navigate such vast spaces. In this study, we focused on mapping a transcription factor binding site (TFBS) landscape to gene expression levels, as changes in gene expression patterns play a crucial role in biological innovation. We developed a massively parallel reporter assay and mapped the first comprehensive in vivo gene regulatory landscape for a bacterial transcriptional regulator, TetR. Surprisingly, this landscape is way more rugged than those observed in eukaryotic regulators. Despite its ruggedness, the landscape remains highly navigable through adaptive evolution. Our study presents the first high-resolution landscape for a bacterial TFBS, offering valuable insights into the evolution of TFBS in vivo. Moreover, it holds promise as a framework for discovering new genetic components for synthetic biological systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555482">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555482" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555482">
        <p class="paperTitle">Remote homolog detection places insect chemoreceptors in a cryptic protein superfamily spanning the tree of life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555482" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555482" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Himmel, N. J.; Moi, D.; Benton, R.</p>
        <p class="info">Score: 10.4, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555482' target='https://doi.org/10.1101/2023.08.30.555482'> 10.1101/2023.08.30.555482</a></p>
        <p class="abstract">Many proteins exist in the so-called &#39;twilight zone&#39; of sequence alignment, where low pairwise sequence identity makes it difficult to determine homology and phylogeny. As protein tertiary structure is often more conserved, recent advances in ab initio protein folding have made structure-based identification of putative homologs feasible. However, structural screening and phylogenetics are in their infancy, particularly for twilight zone proteins. We present a pipeline for the identification and characterization of distant homologs, and apply it to 7-transmembrane domain ion channels (7TMICs), a protein group founded by insect Odorant and Gustatory receptors. Previous sequence and limited structure-based searches identified putatively-related proteins, mainly in other animals and plants. However, very few 7TMICs have been identified in non-animal, non-plant taxa. Moreover, these proteins&#39; remarkable sequence dissimilarity made it uncertain if disparate 7TMIC types (Gr/Or, Grl, GRL, DUF3537, PHTF and GrlHz) are homologous or convergent, leaving their evolutionary history unresolved. Our pipeline identified thousands of new 7TMICs in archaea, bacteria and unicellular eukaryotes. Using graph-based analyses and protein language models to extract family-wide signatures, we demonstrate that 7TMICs have structure and sequence similarity, supporting homology. Through sequence and structure-based phylogenetics, we classify eukaryotic 7TMICs into two families (Class-A and Class-B), which are the result of a gene duplication predating the split(s) leading to Amorphea (animals, fungi and allies) and Diaphoretickes (plants and allies). Our work reveals 7TMICs as a cryptic superfamily with origins close to the evolution of cellular life. More generally, this study serves as a methodological proof of principle for the identification of extremely distant protein homologs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.551515">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.551515" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.551515">
        <p class="paperTitle">Gene loss and cis-regulatory novelty shaped core histone gene evolution in the apiculate yeast Hanseniaspora uvarum</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.551515" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.551515" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Haase, M. A. B.; Steenwyk, J. L.; Boeke, J. D.</p>
        <p class="info">Score: 10.3, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.551515' target='https://doi.org/10.1101/2023.08.28.551515'> 10.1101/2023.08.28.551515</a></p>
        <p class="abstract">Across eukaryotic species, core histone genes display a remarkable diversity of cis-regulatory mechanisms despite their protein sequence conservation. However, the dynamics and significance of this regulatory turnover are not well understood. Here we describe the evolutionary history of core histone gene regulation across 400 million years in Saccharomycotina yeasts, revealing diverse lineage-specific solutions to core histone expression. We further characterize the emergence of a novel regulatory mode in the Hanseniaspora genus, which coincided with the loss of one copy of its paralogous core histones genes from its fast-evolving lineage. By analyzing the growth dynamics using live cell imaging of genetically modified Hanseniaspora uvarum, we observed a regulatory decoupling of core histone synthesis from DNA synthesis and propose that this may be adaptive for its rapid cell cycle progression. Overall, our findings imply that the frequent turnover of core histone cis-regulatory mechanisms likely provides distinct adaptive solutions for specific life histories.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.552763">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.552763" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.552763">
        <p class="paperTitle">Chloramphenicol reduces phage resistance evolution by suppressing bacterial cell surface mutants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.552763" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.552763" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parab, L.; Romeyer Dherbey, J.; Rivera, N.; Schwarz, M.; Bertels, F.</p>
        <p class="info">Score: 9.8, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.552763' target='https://doi.org/10.1101/2023.08.28.552763'> 10.1101/2023.08.28.552763</a></p>
        <p class="abstract">Bacteriophages infect Gram-negative bacteria by attaching to molecules present on the bacterial outer membrane, often lipopolysaccharides (LPS). Modification of the LPS can lead to phage resistance. LPS modifications also impact antibiotic susceptibility, allowing for phage-antibiotic synergism. The mechanism for these synergistic interactions is unclear. Here, we show that antibiotics affect the evolution of phage resistance using phage {Phi}X174 and Escherichia coli C wildtype. We use a collection of E. coli C LPS mutants, each of which is resistant to {Phi}X174, and has either a &#34;rough&#34; or &#34;deep-rough&#34; LPS phenotype. Growth of deep rough mutants is inhibited at subinhibitory chloramphenicol concentrations. In contrast, gentamicin has no major effect on growth. Hypothesis testing shows that treating E. coli C wildtype with {Phi}X174 and chloramphenicol eliminates deep rough mutants, and reduces phage resistance evolution. Our results show that differential survival of phage resistant mutants with antibiotics explains phage-antibiotic synergism in our model system.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.27.555014">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.27.555014" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.27.555014">
        <p class="paperTitle">Acoelomorph flatworm monophyly is a severe long-branch attraction artefact obscuring a clade of Acoela and Xenoturbellida</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.27.555014" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.27.555014" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Redmond, A. K.</p>
        <p class="info">Score: 9.7, Published: 2023-08-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.27.555014' target='https://doi.org/10.1101/2023.08.27.555014'> 10.1101/2023.08.27.555014</a></p>
        <p class="abstract">Acoelomorpha is a broadly accepted clade of bilaterian animals made up of the fast-evolving, morphologically simple, mainly marine flatworm lineages Acoela and Nemertodermatida. Phylogenomic studies support Acoelomorphas close relationship with the slowly evolving and similarly simplistic Xenoturbella, together forming the phylum Xenacoelomorpha. The phylogenetic placement of Xenacoelomorpha amongst bilaterians is controversial, with some studies supporting Xenacoelomorpha as the sister group to all other bilaterians, implying that their simplicity may be representative of early bilaterians. Others propose that this placement is a long branch attraction artefact resulting from the fast-evolving Acoelomorpha, and instead suggest that they are the secondarily simplified sister group of the deuterostome clade Ambulacraria. Perhaps as a result of this debate, internal xenacoelomorph relationships have been somewhat overlooked at a phylogenomic scale. Here, I employ both empirical and simulation approaches to detect and overcome phylogenomic errors to reassess the relationship between Xenoturbella and the fast evolving acoelomorph flatworms. I conclude that subphylum Acoelomorpha is a long-branch attraction artefact obscuring a previously undiscovered clade comprising Xenoturbella and Acoela, for which I propose the name Xenacoela. These analyses are also consistent with the Nephrozoa hypothesis deriving from systematic error, and instead generally favour a close, but unclear, relationship of Xenacoelomorpha with deuterostomes. This study provides a template for future efforts aimed at discovering and correcting unrecognised long-branch attraction artefacts throughout the tree of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554571">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554571" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554571">
        <p class="paperTitle">A cell-type atlas from a scyphozoan jellyfish Aurelia coerulea (formerly sp.1) provides insights into changes of cell-type diversity in the transition from polyps to medusae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554571" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554571" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Link, O.; Jahnel, S. M.; Janicek, K.; Kraus, J.; Montenegro, J. D.; Zimmerman, B.; Cole, A. G.; Technau, U.</p>
        <p class="info">Score: 12.4, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554571' target='https://doi.org/10.1101/2023.08.24.554571'> 10.1101/2023.08.24.554571</a></p>
        <p class="abstract">We provide here a detailed single cell transcriptomic atlas covering the life cycle of the cosmopolitan scyphozoan Aurelia coerulea. We show that there is an increase in cell type diversity in the medusa stage, which is reflected by an increase in the number of unique transcripts expressed. We highlight parallels in both cell complement and specification pathways between Aurelia and the sea anemone Nematostella, two lineages separated by more than 500 MY. We find that several cell types of the neuroglandular lineages are specified by the same transcription factors that arose by gene duplications in the common ancestor of medusozoans and anthozoans. This confirms gene duplications as a source for the increase of cellular complexity during the evolution of cnidarian neuroglandular lineages dates to the common ancestor of all cnidarians. We validated spatial expression domains corresponding to molecular profiles of smooth and striated muscles and followed the development of the striated muscle in the medusa. Our data suggests that striated muscles in the subumbrella of the medusa derive directly from smooth muscles in the jellyfish, and that these two muscle cell types are characterized by distinct molecular profiles. Further, reconstruction of putative cell trajectories indicates multipotent cell states derive from tissues at the boundary between epithelial layers and highlight molecular characteristics not typical of other cnidarians.

TeaserSingle cell transcriptomic atlas across the jellyfish life cycle reveals increased cellular complexity associated with the medusa form.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555124">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555124" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555124">
        <p class="paperTitle">Large-scale analyses reveal the contribution of adaptive evolution in pathogenic and non-pathogenic fungal species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555124" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555124" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pereira, D.; Bolton, M. D.; Friesen, T. L.; Stephan, W.; Dutheil, J. Y.; Stukenbrock, E. H.</p>
        <p class="info">Score: 7.3, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555124' target='https://doi.org/10.1101/2023.08.28.555124'> 10.1101/2023.08.28.555124</a></p>
        <p class="abstract">Genome studies of fungal pathogens have presented evidence for exceptionally high rates of evolution. It has been proposed that rapid adaptation is a hallmark of pathogen evolution that facilitates the invasion of new host niches and the overcoming of intervention strategies such as fungicide applications and drug treatments. To which extent high levels of genetic variation within and between species correlate with adaptive protein evolution in fungi more generally has so far not been explored. In this study, we addressed the contribution of adaptive evolution relative to genetic drift in 20 fungal species, hereby exploring genetic variation in 2,478 fungal genomes. We reannotated positions of protein-coding genes to obtain a high-quality dataset of 234,427 full-length core gene and 25,612 accessory gene alignments. We applied an extension of the McDonald-Kreitman test that models the distributions of fitness effects to infer the rate of adaptive ({omega}A) and non-adaptive ({omega}NA) non-synonymous substitutions in protein-coding genes. To explore the relevance of recombination on local adaptation rates, we inferred the population genomic recombination rate for all 20 species. Our analyses reveal extensive variation in rates of adaptation and show that high rates of adaptation are not a hallmark of a pathogenic lifestyle. Up to 83% of non-synonymous substitutions are adaptive in the species Parastagonospora nodorum. However, non-synonymous substitutions in other species, including the prominent rice-infecting pathogen Magnaporthe oryzae, are predominantly non-adaptive (neutral or slightly deleterious). Correlating adaptation measures with effective population size and recombination rate, we show that effective population size is a primary determinant of adaptive evolution in fungi. At the genome scale, recombination rate variation explains variation in both {omega}A and {omega}NA. Finally, we demonstrate the robustness of our estimates using simulations. We underline the value of population genetic principles in studies of fungal evolution, and we highlight the importance of demographic processes in adaptive evolution of pathogenic and non-pathogenic species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.553699">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.553699" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.553699">
        <p class="paperTitle">One Method to Sequence Them All? Comparison between Whole-Genome Sequencing (WGS) and Target Enrichment (TE) of museum specimens from the moth families Epicopeiidae and Sematuridae (Lepidoptera)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.553699" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.553699" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Call, E.; Twort, V. G.; Espeland, M.; Wahlberg, N.</p>
        <p class="info">Score: 15.1, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.553699' target='https://doi.org/10.1101/2023.08.21.553699'> 10.1101/2023.08.21.553699</a></p>
        <p class="abstract">There are various possibilities for sequencing highly degraded DNA, such as target enrichment (TE), or whole-genome sequencing (WGS). Here we compare TE and WGS methods using old museum specimens of two families of moths in the superfamily Geometroidea: Epicopeiidae and Sematuridae. Until recently, the relationships of these two families were unclear, as few studies had been done. Recently two studies used the TE approach, either on relatively fresh specimens, or on old museum specimens. Here, we aim to increase the sampling of the families Epicopeiidae and Sematuridae from museum specimens using the WGS method. We show that both sequencing methods give comparable results, but, unsurprisingly, WGS recovers more data. By combining TE and WGS data, we confirm that Sematuridae are sister to Pseudobistonidae&#43;Epicopeiidae. Relationships of genera within the families are well supported. With the costs of WGS decreasing, we suggest that using low-coverage whole genome sequencing is becoming an increasingly viable option in the phylogenomics of insects.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
