<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Single-fly assemblies fill major phylogenomic gaps across the Drosophilidae Tree of Life
Authors: Kim, B. Y.; Gellert, H. R.; Church, S. H.; Suvorov, A.; Anderson, S. S.; Barmina, O.; Beskid, S. G.; Comeault, A. A.; Crown, K. N.; Diamond, S. E.; Dorus, S.; Fujichika, T.; Hemker, J. A.; Hrcek, J.; Kankare, M.; Katoh, T.; Magnacca, K. N.; Martin, R. A.; Matsunaga, T.; Medeiros, M. J.; Miller, D. E.; Pitnick, S.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Single-fly assemblies fill major phylogenomic gaps across the Drosophilidae Tree of Life
Authors: Kim, B. Y.; Gellert, H. R.; Church, S. H.; Suvorov, A.; Anderson, S. S.; Barmina, O.; Beskid, S. G.; Comeault, A. A.; Crown, K. N.; Diamond, S. E.; Dorus, S.; Fujichika, T.; Hemker, J. A.; Hrcek, J.; Kankare, M.; Katoh, T.; Magnacca, K. N.; Martin, R. A.; Matsunaga, T.; Medeiros, M. J.; Miller, D. E.; Pitnick, S." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-04T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-10-04T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Single-fly assemblies fill major phylogenomic gaps across the Drosophilidae Tree of Life
Authors: Kim, B. Y.; Gellert, H. R.; Church, S. H.; Suvorov, A.; Anderson, S. S.; Barmina, O.; Beskid, S. G.; Comeault, A. A.; Crown, K. N.; Diamond, S. E.; Dorus, S.; Fujichika, T.; Hemker, J. A.; Hrcek, J.; Kankare, M.; Katoh, T.; Magnacca, K. N.; Martin, R. A.; Matsunaga, T.; Medeiros, M. J.; Miller, D. E.; Pitnick, S."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Single-fly assemblies fill major phylogenomic gaps across the Drosophilidae Tree of Life\nAuthors: Kim, B. Y.; Gellert, H. R.; Church, S. H.; Suvorov, A.; Anderson, S. S.; Barmina, O.; Beskid, S. G.; Comeault, A. A.; Crown, K. N.; Diamond, S. E.; Dorus, S.; Fujichika, T.; Hemker, J. A.; Hrcek, J.; Kankare, M.; Katoh, T.; Magnacca, K. N.; Martin, R. A.; Matsunaga, T.; Medeiros, M. J.; Miller, D. E.; Pitnick, S.",
  "keywords": [
    
  ],
  "articleBody": " Single-fly assemblies fill major phylogenomic gaps across the Drosophilidae Tree of Life\nAuthors: Kim, B. Y.; Gellert, H. R.; Church, S. H.; Suvorov, A.; Anderson, S. S.; Barmina, O.; Beskid, S. G.; Comeault, A. A.; Crown, K. N.; Diamond, S. E.; Dorus, S.; Fujichika, T.; Hemker, J. A.; Hrcek, J.; Kankare, M.; Katoh, T.; Magnacca, K. N.; Martin, R. A.; Matsunaga, T.; Medeiros, M. J.; Miller, D. E.; Pitnick, S.; Simoni, S.; Steenwinkel, T. E.; Schiffer, M.; Syed, Z. A.; Takahashi, A.; Wei, K. H.-C.; Yokoyama, T.; Eisen, M. B.; Kopp, A.; Matute, D.; Obbard, D. J.; O'Grady, P. M.; Price, D. K.; Toda, M. J.; Werner, T.; Petrov, D. A.\nScore: 45.7, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560517\nLong-read sequencing is driving rapid progress in genome assembly across all major groups of life, including species of the family Drosophilidae, a longtime model system for genetics, genomics, and evolution. We previously developed a cost-effective hybrid Oxford Nanopore (ONT) long-read and Illumina short-read sequencing approach and used it to assemble 101 drosophilid genomes from laboratory cultures, greatly increasing the number of genome assemblies for this taxonomic group. The next major challenge is to address the laboratory culture bias in taxon sampling by sequencing genomes of species that cannot easily be reared in the lab. Here, we build upon our previous methods to perform amplification-free ONT sequencing of single wild flies obtained either directly from the field or from ethanol-preserved specimens in museum collections, greatly improving the representation of lesser studied drosophilid taxa in whole-genome data. Using Illumina Novaseq X Plus and ONT P2 sequencers with R10.4.1 chemistry, we set a new benchmark for inexpensive hybrid genome assembly at US $150 per genome while assembling genomes from as little as 35 ng of genomic DNA from a single fly. We present 183 new genome assemblies for 179 species as a resource for drosophilid systematics, phylogenetics, and comparative genomics. Of these genomes, 62 are from pooled lab strains and 121 from single adult flies. Despite the sample limitations of working with small insects, most single-fly diploid assemblies are comparable in contiguity (\u003e1Mb contig N50), completeness (\u003e98% complete dipteran BUSCOs), and accuracy (\u003eQV40 genome-wide with ONT R10.4.1) to assemblies from inbred lines. We present a well-resolved multi-locus phylogeny for 360 drosophilid and 4 outgroup species encompassing all publicly available (as of August 2023) genomes for this group. Finally, we present a Progressive Cactus whole-genome, reference-free alignment built from a subset of 298 suitably high-quality drosophilid genomes. The new assemblies and alignment, along with updated laboratory protocols and computational pipelines, are released as an open resource and as a tool for studying evolution at the scale of an entire insect family.\nDeep learning reveals the role of copy number variation in the genetic architecture of a highly polymorphic sexual trait\nAuthors: van der Bijl, W.; Shu, J.; Goberdhan, V. S.; Sherin, L.; Cortazar-Chinarro, M.; Corral-Lopez, A.; Mank, J. E.\nScore: 23.1, Published: 2023-09-30 DOI: 10.1101/2023.09.29.560175\nThe extraordinary variation in male guppy coloration has been a powerful model for studying the interplay of natural and sexual selection. Many guppy populations exhibit substantial Y-linkage of color traits, and this has hampered the identification of the genetic architecture underlying male guppy color, as well as clouded our understanding of how this exceptional level of diversity is maintained. Here we used a population with low levels of Y-linkage for color variation to identify the heritability and genetic basis of male color variation using convolutional neural networks for high-resolution phenotyping coupled with selection experiments and controlled pedigrees and whole-genome resequencing for Genome Wide Association Study (GWAS) of colour. Our phenotypic and genomic results converge to show that color patterning in guppies is a combination of many heritable features, each with a partially overlapping genetic architecture. Unusually, our GWAS results suggest that copy number variation (CNV) is responsible for much of the variation in colour in guppies, providing a potential mechanism for the maintenance of variation of this classic model trait.\nWorking together to control mutation: how collective peroxide detoxification determines microbial mutation rate plasticity\nAuthors: Green, R.; Wang, H.; Botchey, C.; Zhang, N.; Wadsworth, C.; McBain, A.; Paszek, P.; Krasovec, R.; Knight, C. G.\nScore: 22.2, Published: 2023-09-27 DOI: 10.1101/2023.09.27.557722\nMutagenesis is responsive to many environmental factors. Evolution therefore depends on the environment not only for selection but also in determining the variation available in a population. One such environmental dependency is the inverse relationship between mutation rates and population density in many microbial species. Here we determine the mechanism responsible for this mutation rate plasticity. Using dynamical computational modelling and in vivo mutation rate estimation we show that the negative relationship between mutation rate and population density arises from the collective ability of microbial populations to control concentrations of hydrogen peroxide. We demonstrate a loss of this density-associated mutation rate plasticity when Escherichia coli populations are deficient in the degradation of hydrogen peroxide. We further show that the reduction in mutation rate in denser populations is restored in peroxide degradation-deficient cells by the presence of wild-type cells in a mixed population. Together, these model-guided experiments provide a mechanistic explanation for density-associated mutation rate plasticity, applicable across all domains of life, and frames mutation rate as a dynamic trait shaped by microbial community composition.\nThe MUC19 gene in Denisovans, Neanderthals, and Modern Humans: An Evolutionary History of Recurrent Introgression and Natural Selection\nAuthors: Villanea, F. A.; Peede, D.; Kaufman, E. J.; Anorve-Garibay, V.; Dillon, K. E. W.; Zeloni, R.; Marnetto, d.; Moorjani, P.; Jay, F.; Valdmanis, P. N.; Avila-Arcos, M. C.; Huerta-Sanchez, E.\nScore: 38.4, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559202\nAll humans carry a small fraction of archaic ancestry across the genome, the legacy of gene flow from Neanderthals, Denisovans, and other hominids into the ancestors of modern humans. While the effects of Neanderthal ancestry on human fitness and health have been explored more thoroughly, there are fewer examples of adaptive introgression of Denisovan variants. Here, we study the gene MUC19, for which some modern humans carry a Denisovan-like haplotype. MUC19 is a mucin, a glycoprotein that forms gels with various biological functions, from lubrication to immunity. We find the diagnostic variants for the Denisovan-like MUC19 haplotype at high frequencies in admixed Latin American individuals among global population, and at highest frequency in 23 ancient Indigenous American individuals, all predating population admixture with Europeans and Africans. We find that some Neanderthals--Vindija and Chagyrskaya--carry the Denisovan-like MUC19 haplotype, and that it was likely introgressed into human populations through Neanderthal introgression rather than Denisovan introgression. Finally, we find that the Denisovan-like MUC19 haplotype carries a higher copy number of a 30 base-pair variable number tandem repeat relative to the Human-like haplotype, and that copy numbers of this repeat are exceedingly high in American populations. Our results suggest that the Denisovan-like MUC19 haplotype served as the raw genetic material for positive selection as American populations adapted to novel environments during their movement from Beringia into North and then South America.\nPhased assembly of neo-sex chromosomes reveals extensive Y degeneration and rapid genome evolution in Rumex hastatulus\nAuthors: Sacchi, B. M.; Humphries, Z.; Kruzlicova, J.; Bodlakova, M.; Pyne, C.; Choudhury, B.; Gong, Y.; Bacovsky, V.; Hobza, R.; Barrett, S. C. H.; Wright, S. I.\nScore: 14.5, Published: 2023-09-28 DOI: 10.1101/2023.09.26.559509\nY chromosomes are thought to undergo progressive degeneration due to stepwise loss of recombination and subsequent reduction in selection efficiency. However, the timescales over which degeneration occurs and the evolutionary forces driving degeneration remain unclear. In order to characterize the evolution of sex chromosomes on multiple timescales, we generated a high-quality phased genome assembly of the massive older (7-9 MYA) and neo (\u003c200,000 years) sex chromosomes in the XYY cytotype of the plant Rumex hastatulus, along with a hermaphroditic outgroup Rumex salicifolius. Our assemblies confirmed the neo-sex chromosomes were formed by two key events: an X-autosome fusion and a reciprocal translocation between the homologous autosome and the Y chromosome. The enormous sex-linked regions of the X (296 MB) and the two Y chromosomes (503 MB) both arose in a large repeat-rich genome with low recombination, however the complete loss of recombination on the Y still led to over 30% gene loss and massive rearrangements over a short timescale. In the older sex-linked region, there has been a major increase in the abundance of transposable elements, including into and near genes. In the neo sex-linked regions, we observe evidence of extensive chromosome rearrangements before gene degeneration and loss. Overall, we observe extensive degeneration during the first 10 million years of Y chromosome evolution, but not immediate genomic degeneration on very short timescales. Our results highlight that even when sex chromosomes emerge from repetitive regions of already-low recombination, the complete loss of recombination on the Y chromosome still leads to a substantial increase in repetitive element content and gene degeneration.\nPolyploidy is widespread in Microsporidia\nAuthors: Khalaf, A.; Jaron, K. S.; Blaxter, M.; Lawniczak, M.\nScore: 13.9, Published: 2023-09-30 DOI: 10.1101/2023.09.29.560119\nMicrosporidia are obligate intracellular eukaryotic parasites with an extremely broad host range. They have both economic and public health importance. Ploidy in microsporidia is variable, with a few species formally identified as diploid and one as polyploid. Given the increase in the number of studies sequencing microsporidian genomes, it is now possible to assess ploidy levels across all currently-explored microsporidian diversity. We estimate ploidy for all microsporidian datasets available on the Sequence Read Archive (SRA) using k-mer-based analyses, and demonstrate that polyploidy is widespread in Microsporidia, and that ploidy change is dynamic in the group. Using genome-wide heterozygosity estimates, we also show that polyploid microsporidian genomes are relatively homozygous, and we discuss the implications of these findings on the timing of polyploidisation events and their origin.\nGenome analysis of the Jomon dogs reveals the oldest domestic dog lineage in Eastern Eurasia\nAuthors: Terai, Y.; Xiaokaiti, X.; Gojobori, J.; Arakawa, N.; Sato, T.; Kasai, K.; Machida, K.; Yamazaki, K.; Yamaji, N.; Hongo, H.; Gakuhari, T.\nScore: 10.7, Published: 2023-10-01 DOI: 10.1101/2023.09.29.560089\nDog is the oldest domesticated animal that established close relationships with humans. Due to its ancient origin, when, where, and whether a single or dual domestication event occurred is still under debate. The dogs in the Jomon period (Jomon dogs) in the Japanese archipelago had little change in morphology from 10,000 to 3,000 years ago. Therefore, we expected that the ancient genome of the Jomon dogs would provide a clue to reveal the characteristics of the ancient East Asian dogs. Here, we have sequenced the genomes of three 6000-year-old Jomon dogs, one 3000-4000-year-old Jomon dog, and four late 8th century dogs excavated in Japan. Our analyses suggest that the Jomon dogs are a distinct lineage from the previously known ancient dogs and are one of the oldest among the dogs in East Eurasian lineages. In addition, the genome of the Jomon dogs contained 9.5% of the genome of Japanese wolf ancestry due to a single introgression event. We estimated the proportion of the Jomon dog lineage genome in the genomes of dogs, which indicates that the genomic composition derived from the Jomon dog lineage is one of the major sources of modern dog genomes. Furthermore, we estimated the early admixture events of dogs in East Eurasia by analyzing the ancient genomes of the Jomon dogs. Due to the admixture events, the Jomon dog-derived genome has been one of the genomic sources of a wide range of modern dogs.\nAssessing the role of mitonuclear interactions on mitochondrial function and organismal fitness in natural Drosophila populations\nAuthors: Bettinazzi, S.; Liang, J.; Rodriguez, E.; Bonneau, M.; Holt, R.; Whitehead, B.; Dowling, D.; Lane, N.; Camus, F.\nScore: 9.6, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559268\nMitochondrial metabolism is regulated by a series of enzyme complexes, whose function depends on effective interactions between proteins and RNA encoded by the mitochondrial and nuclear genomes. These epistatic interactions are in turn highly sensitive to the environment. Many studies have found that mitochondrial haplotype frequencies of various taxa associate with latitude or altitude, leading to the hypothesis that mitochondrial genomes may respond to thermal selection and contribute to local adaptation. We used a Drosophila melanogaster panel comprising native (coadapted) populations from the extremes of the Australian east-coast cline, and generated mitonuclear cybrid populations. Our results indicate a strong phenotypic impact of mitonuclear interactions in cybrid lines, involving an apparent trade-off between aerobic capacity and key fitness aspects such as reproduction, growth, and survival. Overall, our study shows that naturally-occurring mitonuclear disruptions can have a meaningful impact on phenotypes, potentially influencing future ecological adaptation and population persistence.\nA genome-scale Opisthokonta tree of life: toward phylogenomic resolution of ancient divergences\nAuthors: Liu, H.; Steenwyk, J. L.; Zhou, X.; Schultz, D. T.; Kocot, K. M.; Shen, X.-X.; Rokas, A.; Li, Y.\nScore: 7.3, Published: 2023-09-21 DOI: 10.1101/2023.09.20.556338\nAbstractAncient divergences within Opisthokonta--a major lineage that includes organisms in the kingdoms Animalia, Fungi, and their unicellular relatives-- remain contentious, hindering investigations of the evolutionary processes that gave rise to two kingdoms and the repeated emergence of iconic phenotypes like multicellularity. Here, we use genome-scale amounts of data to reconstruct the most taxon-rich Opisthokonta tree of life to date (348 species) and place divergences in geologic time, suggesting a Mesoproterozoic origin ([~] 1.11 billion years ago). By dissecting multiple dimensions of phylogenomic error, such as the influence of taxon sampling and model complexity, we found that deep divergences within Holozoa remain unresolved and suggest Pluriformea is either sister to Ichthyosporea and Filozoa (Pluriformea-sister hypothesis) or is monophyletic to Ichthyosporea, forming the Teretosporea lineage (Teretosporea-sister hypothesis). A combination of information theory and sensitivity analyses revealed that the inferred unicellular Holozoa relationships are largely robust to common sources of analytical error, such as insufficient model complexity, and suggest that previous reports likely suffered from insufficient taxon sampling. Our study presents a robust Opisthokonta phylogenomic framework, highlights the challenges in resolving the relationships of unicellular Holozoa, and paves the way for illuminating ancient evolutionary episodes concerning the origin of two kingdoms.\nStrong environmental memory revealed by experimental evolution in static and fluctuating environments\nAuthors: Abreu, C. I.; Mathur, S.; Petrov, D. A.\nScore: 42.1, Published: 2023-09-22 DOI: 10.1101/2023.09.14.557739\nEvolution in a static environment, such as a laboratory setting with constant and uniform conditions, often proceeds via large-effect beneficial mutations that may become maladaptive in other environments. Conversely, natural settings require populations to endure environmental fluctuations. A sensible assumption is that the fitness of a lineage in a fluctuating environment is the time-average of its fitness over the sequence of static conditions it encounters. However, transitions between conditions may pose entirely new challenges, which could cause deviations from this time-average. To test this, we tracked hundreds of thousands of barcoded yeast lineages evolving in static and fluctuating conditions and subsequently isolated 900 mutants for pooled fitness assays in 15 environments. We find that fitness in fluctuating environments indeed often deviates from the expectation based on static components, leading to fitness non-additivity. Moreover, closer examination reveals that fitness in one component of a fluctuating environment is often strongly influenced by the previous component. We show that this environmental memory is especially common for mutants with high variance in fitness across tested environments, even if the components of the focal fluctuating environment are excluded from this variance. We employ a simple mathematical model and whole-genome sequencing to propose mechanisms underlying this effect, including lag time evolution and sensing mutations. Our results demonstrate that environmental fluctuations have large impacts on fitness and suggest that variance in static environments can explain these impacts.\n",
  "wordCount" : "2571",
  "inLanguage": "en",
  "datePublished": "2023-10-04T10:39:33Z",
  "dateModified": "2023-10-04T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on October 4, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560517">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560517" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560517">
        <p class="paperTitle">Single-fly assemblies fill major phylogenomic gaps across the Drosophilidae Tree of Life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560517" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560517" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, B. Y.; Gellert, H. R.; Church, S. H.; Suvorov, A.; Anderson, S. S.; Barmina, O.; Beskid, S. G.; Comeault, A. A.; Crown, K. N.; Diamond, S. E.; Dorus, S.; Fujichika, T.; Hemker, J. A.; Hrcek, J.; Kankare, M.; Katoh, T.; Magnacca, K. N.; Martin, R. A.; Matsunaga, T.; Medeiros, M. J.; Miller, D. E.; Pitnick, S.; Simoni, S.; Steenwinkel, T. E.; Schiffer, M.; Syed, Z. A.; Takahashi, A.; Wei, K. H.-C.; Yokoyama, T.; Eisen, M. B.; Kopp, A.; Matute, D.; Obbard, D. J.; O&#39;Grady, P. M.; Price, D. K.; Toda, M. J.; Werner, T.; Petrov, D. A.</p>
        <p class="info">Score: 45.7, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560517' target='https://doi.org/10.1101/2023.10.02.560517'> 10.1101/2023.10.02.560517</a></p>
        <p class="abstract">Long-read sequencing is driving rapid progress in genome assembly across all major groups of life, including species of the family Drosophilidae, a longtime model system for genetics, genomics, and evolution. We previously developed a cost-effective hybrid Oxford Nanopore (ONT) long-read and Illumina short-read sequencing approach and used it to assemble 101 drosophilid genomes from laboratory cultures, greatly increasing the number of genome assemblies for this taxonomic group. The next major challenge is to address the laboratory culture bias in taxon sampling by sequencing genomes of species that cannot easily be reared in the lab. Here, we build upon our previous methods to perform amplification-free ONT sequencing of single wild flies obtained either directly from the field or from ethanol-preserved specimens in museum collections, greatly improving the representation of lesser studied drosophilid taxa in whole-genome data. Using Illumina Novaseq X Plus and ONT P2 sequencers with R10.4.1 chemistry, we set a new benchmark for inexpensive hybrid genome assembly at US $150 per genome while assembling genomes from as little as 35 ng of genomic DNA from a single fly. We present 183 new genome assemblies for 179 species as a resource for drosophilid systematics, phylogenetics, and comparative genomics. Of these genomes, 62 are from pooled lab strains and 121 from single adult flies. Despite the sample limitations of working with small insects, most single-fly diploid assemblies are comparable in contiguity (&gt;1Mb contig N50), completeness (&gt;98% complete dipteran BUSCOs), and accuracy (&gt;QV40 genome-wide with ONT R10.4.1) to assemblies from inbred lines. We present a well-resolved multi-locus phylogeny for 360 drosophilid and 4 outgroup species encompassing all publicly available (as of August 2023) genomes for this group. Finally, we present a Progressive Cactus whole-genome, reference-free alignment built from a subset of 298 suitably high-quality drosophilid genomes. The new assemblies and alignment, along with updated laboratory protocols and computational pipelines, are released as an open resource and as a tool for studying evolution at the scale of an entire insect family.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560175">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560175" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560175">
        <p class="paperTitle">Deep learning reveals the role of copy number variation in the genetic architecture of a highly polymorphic sexual trait</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560175" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560175" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: van der Bijl, W.; Shu, J.; Goberdhan, V. S.; Sherin, L.; Cortazar-Chinarro, M.; Corral-Lopez, A.; Mank, J. E.</p>
        <p class="info">Score: 23.1, Published: 2023-09-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560175' target='https://doi.org/10.1101/2023.09.29.560175'> 10.1101/2023.09.29.560175</a></p>
        <p class="abstract">The extraordinary variation in male guppy coloration has been a powerful model for studying the interplay of natural and sexual selection. Many guppy populations exhibit substantial Y-linkage of color traits, and this has hampered the identification of the genetic architecture underlying male guppy color, as well as clouded our understanding of how this exceptional level of diversity is maintained. Here we used a population with low levels of Y-linkage for color variation to identify the heritability and genetic basis of male color variation using convolutional neural networks for high-resolution phenotyping coupled with selection experiments and controlled pedigrees and whole-genome resequencing for Genome Wide Association Study (GWAS) of colour. Our phenotypic and genomic results converge to show that color patterning in guppies is a combination of many heritable features, each with a partially overlapping genetic architecture. Unusually, our GWAS results suggest that copy number variation (CNV) is responsible for much of the variation in colour in guppies, providing a potential mechanism for the maintenance of variation of this classic model trait.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.557722">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.557722" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.557722">
        <p class="paperTitle">Working together to control mutation: how collective peroxide detoxification determines microbial mutation rate plasticity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.557722" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.557722" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Green, R.; Wang, H.; Botchey, C.; Zhang, N.; Wadsworth, C.; McBain, A.; Paszek, P.; Krasovec, R.; Knight, C. G.</p>
        <p class="info">Score: 22.2, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.557722' target='https://doi.org/10.1101/2023.09.27.557722'> 10.1101/2023.09.27.557722</a></p>
        <p class="abstract">Mutagenesis is responsive to many environmental factors. Evolution therefore depends on the environment not only for selection but also in determining the variation available in a population. One such environmental dependency is the inverse relationship between mutation rates and population density in many microbial species. Here we determine the mechanism responsible for this mutation rate plasticity. Using dynamical computational modelling and in vivo mutation rate estimation we show that the negative relationship between mutation rate and population density arises from the collective ability of microbial populations to control concentrations of hydrogen peroxide. We demonstrate a loss of this density-associated mutation rate plasticity when Escherichia coli populations are deficient in the degradation of hydrogen peroxide. We further show that the reduction in mutation rate in denser populations is restored in peroxide degradation-deficient cells by the presence of wild-type cells in a mixed population. Together, these model-guided experiments provide a mechanistic explanation for density-associated mutation rate plasticity, applicable across all domains of life, and frames mutation rate as a dynamic trait shaped by microbial community composition.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559202">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559202" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559202">
        <p class="paperTitle">The MUC19 gene in Denisovans, Neanderthals, and Modern Humans: An Evolutionary History of Recurrent Introgression and Natural Selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559202" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559202" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Villanea, F. A.; Peede, D.; Kaufman, E. J.; Anorve-Garibay, V.; Dillon, K. E. W.; Zeloni, R.; Marnetto, d.; Moorjani, P.; Jay, F.; Valdmanis, P. N.; Avila-Arcos, M. C.; Huerta-Sanchez, E.</p>
        <p class="info">Score: 38.4, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559202' target='https://doi.org/10.1101/2023.09.25.559202'> 10.1101/2023.09.25.559202</a></p>
        <p class="abstract">All humans carry a small fraction of archaic ancestry across the genome, the legacy of gene flow from Neanderthals, Denisovans, and other hominids into the ancestors of modern humans. While the effects of Neanderthal ancestry on human fitness and health have been explored more thoroughly, there are fewer examples of adaptive introgression of Denisovan variants. Here, we study the gene MUC19, for which some modern humans carry a Denisovan-like haplotype. MUC19 is a mucin, a glycoprotein that forms gels with various biological functions, from lubrication to immunity. We find the diagnostic variants for the Denisovan-like MUC19 haplotype at high frequencies in admixed Latin American individuals among global population, and at highest frequency in 23 ancient Indigenous American individuals, all predating population admixture with Europeans and Africans. We find that some Neanderthals--Vindija and Chagyrskaya--carry the Denisovan-like MUC19 haplotype, and that it was likely introgressed into human populations through Neanderthal introgression rather than Denisovan introgression. Finally, we find that the Denisovan-like MUC19 haplotype carries a higher copy number of a 30 base-pair variable number tandem repeat relative to the Human-like haplotype, and that copy numbers of this repeat are exceedingly high in American populations. Our results suggest that the Denisovan-like MUC19 haplotype served as the raw genetic material for positive selection as American populations adapted to novel environments during their movement from Beringia into North and then South America.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559509">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559509" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559509">
        <p class="paperTitle">Phased assembly of neo-sex chromosomes reveals extensive Y degeneration and rapid genome evolution in Rumex hastatulus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559509" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559509" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sacchi, B. M.; Humphries, Z.; Kruzlicova, J.; Bodlakova, M.; Pyne, C.; Choudhury, B.; Gong, Y.; Bacovsky, V.; Hobza, R.; Barrett, S. C. H.; Wright, S. I.</p>
        <p class="info">Score: 14.5, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559509' target='https://doi.org/10.1101/2023.09.26.559509'> 10.1101/2023.09.26.559509</a></p>
        <p class="abstract">Y chromosomes are thought to undergo progressive degeneration due to stepwise loss of recombination and subsequent reduction in selection efficiency. However, the timescales over which degeneration occurs and the evolutionary forces driving degeneration remain unclear. In order to characterize the evolution of sex chromosomes on multiple timescales, we generated a high-quality phased genome assembly of the massive older (7-9 MYA) and neo (&lt;200,000 years) sex chromosomes in the XYY cytotype of the plant Rumex hastatulus, along with a hermaphroditic outgroup Rumex salicifolius. Our assemblies confirmed the neo-sex chromosomes were formed by two key events: an X-autosome fusion and a reciprocal translocation between the homologous autosome and the Y chromosome. The enormous sex-linked regions of the X (296 MB) and the two Y chromosomes (503 MB) both arose in a large repeat-rich genome with low recombination, however the complete loss of recombination on the Y still led to over 30% gene loss and massive rearrangements over a short timescale. In the older sex-linked region, there has been a major increase in the abundance of transposable elements, including into and near genes. In the neo sex-linked regions, we observe evidence of extensive chromosome rearrangements before gene degeneration and loss. Overall, we observe extensive degeneration during the first 10 million years of Y chromosome evolution, but not immediate genomic degeneration on very short timescales. Our results highlight that even when sex chromosomes emerge from repetitive regions of already-low recombination, the complete loss of recombination on the Y chromosome still leads to a substantial increase in repetitive element content and gene degeneration.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560119">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560119" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560119">
        <p class="paperTitle">Polyploidy is widespread in Microsporidia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560119" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560119" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khalaf, A.; Jaron, K. S.; Blaxter, M.; Lawniczak, M.</p>
        <p class="info">Score: 13.9, Published: 2023-09-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560119' target='https://doi.org/10.1101/2023.09.29.560119'> 10.1101/2023.09.29.560119</a></p>
        <p class="abstract">Microsporidia are obligate intracellular eukaryotic parasites with an extremely broad host range. They have both economic and public health importance. Ploidy in microsporidia is variable, with a few species formally identified as diploid and one as polyploid. Given the increase in the number of studies sequencing microsporidian genomes, it is now possible to assess ploidy levels across all currently-explored microsporidian diversity. We estimate ploidy for all microsporidian datasets available on the Sequence Read Archive (SRA) using k-mer-based analyses, and demonstrate that polyploidy is widespread in Microsporidia, and that ploidy change is dynamic in the group. Using genome-wide heterozygosity estimates, we also show that polyploid microsporidian genomes are relatively homozygous, and we discuss the implications of these findings on the timing of polyploidisation events and their origin.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560089">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560089" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560089">
        <p class="paperTitle">Genome analysis of the Jomon dogs reveals the oldest domestic dog lineage in Eastern Eurasia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560089" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560089" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Terai, Y.; Xiaokaiti, X.; Gojobori, J.; Arakawa, N.; Sato, T.; Kasai, K.; Machida, K.; Yamazaki, K.; Yamaji, N.; Hongo, H.; Gakuhari, T.</p>
        <p class="info">Score: 10.7, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560089' target='https://doi.org/10.1101/2023.09.29.560089'> 10.1101/2023.09.29.560089</a></p>
        <p class="abstract">Dog is the oldest domesticated animal that established close relationships with humans. Due to its ancient origin, when, where, and whether a single or dual domestication event occurred is still under debate. The dogs in the Jomon period (Jomon dogs) in the Japanese archipelago had little change in morphology from 10,000 to 3,000 years ago. Therefore, we expected that the ancient genome of the Jomon dogs would provide a clue to reveal the characteristics of the ancient East Asian dogs. Here, we have sequenced the genomes of three 6000-year-old Jomon dogs, one 3000-4000-year-old Jomon dog, and four late 8th century dogs excavated in Japan. Our analyses suggest that the Jomon dogs are a distinct lineage from the previously known ancient dogs and are one of the oldest among the dogs in East Eurasian lineages. In addition, the genome of the Jomon dogs contained 9.5% of the genome of Japanese wolf ancestry due to a single introgression event. We estimated the proportion of the Jomon dog lineage genome in the genomes of dogs, which indicates that the genomic composition derived from the Jomon dog lineage is one of the major sources of modern dog genomes. Furthermore, we estimated the early admixture events of dogs in East Eurasia by analyzing the ancient genomes of the Jomon dogs. Due to the admixture events, the Jomon dog-derived genome has been one of the genomic sources of a wide range of modern dogs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559268">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559268" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559268">
        <p class="paperTitle">Assessing the role of mitonuclear interactions on mitochondrial function and organismal fitness in natural Drosophila populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559268" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559268" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bettinazzi, S.; Liang, J.; Rodriguez, E.; Bonneau, M.; Holt, R.; Whitehead, B.; Dowling, D.; Lane, N.; Camus, F.</p>
        <p class="info">Score: 9.6, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559268' target='https://doi.org/10.1101/2023.09.25.559268'> 10.1101/2023.09.25.559268</a></p>
        <p class="abstract">Mitochondrial metabolism is regulated by a series of enzyme complexes, whose function depends on effective interactions between proteins and RNA encoded by the mitochondrial and nuclear genomes. These epistatic interactions are in turn highly sensitive to the environment. Many studies have found that mitochondrial haplotype frequencies of various taxa associate with latitude or altitude, leading to the hypothesis that mitochondrial genomes may respond to thermal selection and contribute to local adaptation. We used a Drosophila melanogaster panel comprising native (coadapted) populations from the extremes of the Australian east-coast cline, and generated mitonuclear cybrid populations. Our results indicate a strong phenotypic impact of mitonuclear interactions in cybrid lines, involving an apparent trade-off between aerobic capacity and key fitness aspects such as reproduction, growth, and survival. Overall, our study shows that naturally-occurring mitonuclear disruptions can have a meaningful impact on phenotypes, potentially influencing future ecological adaptation and population persistence.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.556338">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.556338" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.556338">
        <p class="paperTitle">A genome-scale Opisthokonta tree of life: toward phylogenomic resolution of ancient divergences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.556338" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.556338" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, H.; Steenwyk, J. L.; Zhou, X.; Schultz, D. T.; Kocot, K. M.; Shen, X.-X.; Rokas, A.; Li, Y.</p>
        <p class="info">Score: 7.3, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.556338' target='https://doi.org/10.1101/2023.09.20.556338'> 10.1101/2023.09.20.556338</a></p>
        <p class="abstract">AbstractAncient divergences within Opisthokonta--a major lineage that includes organisms in the kingdoms Animalia, Fungi, and their unicellular relatives-- remain contentious, hindering investigations of the evolutionary processes that gave rise to two kingdoms and the repeated emergence of iconic phenotypes like multicellularity. Here, we use genome-scale amounts of data to reconstruct the most taxon-rich Opisthokonta tree of life to date (348 species) and place divergences in geologic time, suggesting a Mesoproterozoic origin ([~] 1.11 billion years ago). By dissecting multiple dimensions of phylogenomic error, such as the influence of taxon sampling and model complexity, we found that deep divergences within Holozoa remain unresolved and suggest Pluriformea is either sister to Ichthyosporea and Filozoa (Pluriformea-sister hypothesis) or is monophyletic to Ichthyosporea, forming the Teretosporea lineage (Teretosporea-sister hypothesis). A combination of information theory and sensitivity analyses revealed that the inferred unicellular Holozoa relationships are largely robust to common sources of analytical error, such as insufficient model complexity, and suggest that previous reports likely suffered from insufficient taxon sampling. Our study presents a robust Opisthokonta phylogenomic framework, highlights the challenges in resolving the relationships of unicellular Holozoa, and paves the way for illuminating ancient evolutionary episodes concerning the origin of two kingdoms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557739">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557739" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557739">
        <p class="paperTitle">Strong environmental memory revealed by experimental evolution in static and fluctuating environments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557739" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557739" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abreu, C. I.; Mathur, S.; Petrov, D. A.</p>
        <p class="info">Score: 42.1, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557739' target='https://doi.org/10.1101/2023.09.14.557739'> 10.1101/2023.09.14.557739</a></p>
        <p class="abstract">Evolution in a static environment, such as a laboratory setting with constant and uniform conditions, often proceeds via large-effect beneficial mutations that may become maladaptive in other environments. Conversely, natural settings require populations to endure environmental fluctuations. A sensible assumption is that the fitness of a lineage in a fluctuating environment is the time-average of its fitness over the sequence of static conditions it encounters. However, transitions between conditions may pose entirely new challenges, which could cause deviations from this time-average. To test this, we tracked hundreds of thousands of barcoded yeast lineages evolving in static and fluctuating conditions and subsequently isolated 900 mutants for pooled fitness assays in 15 environments. We find that fitness in fluctuating environments indeed often deviates from the expectation based on static components, leading to fitness non-additivity. Moreover, closer examination reveals that fitness in one component of a fluctuating environment is often strongly influenced by the previous component. We show that this environmental memory is especially common for mutants with high variance in fitness across tested environments, even if the components of the focal fluctuating environment are excluded from this variance. We employ a simple mathematical model and whole-genome sequencing to propose mechanisms underlying this effect, including lag time evolution and sensing mutations. Our results demonstrate that environmental fluctuations have large impacts on fitness and suggest that variance in static environments can explain these impacts.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
