<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Consistent accumulation of transposable elements in species of the Hawaiian Tetragnatha spiny-leg adaptive radiation across the archipelago chronosequence
Authors: Yang, H.; Goubert, C.; Cotoras, D.; Graham, N.; Cerca, J.; Gillespie, R. G.
Score: 10.6, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574070
The ecological and phenotypic diversity observed in oceanic island radiations presents an evolutionary paradox: a high level of genetic variation is typically required for diversification, but species colonizing a new island typically suffer from founder effects.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Consistent accumulation of transposable elements in species of the Hawaiian Tetragnatha spiny-leg adaptive radiation across the archipelago chronosequence
Authors: Yang, H.; Goubert, C.; Cotoras, D.; Graham, N.; Cerca, J.; Gillespie, R. G.
Score: 10.6, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574070
The ecological and phenotypic diversity observed in oceanic island radiations presents an evolutionary paradox: a high level of genetic variation is typically required for diversification, but species colonizing a new island typically suffer from founder effects." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-07T10:38:59+00:00" />
<meta property="article:modified_time" content="2024-01-07T10:38:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Consistent accumulation of transposable elements in species of the Hawaiian Tetragnatha spiny-leg adaptive radiation across the archipelago chronosequence
Authors: Yang, H.; Goubert, C.; Cotoras, D.; Graham, N.; Cerca, J.; Gillespie, R. G.
Score: 10.6, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574070
The ecological and phenotypic diversity observed in oceanic island radiations presents an evolutionary paradox: a high level of genetic variation is typically required for diversification, but species colonizing a new island typically suffer from founder effects."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Consistent accumulation of transposable elements in species of the Hawaiian Tetragnatha spiny-leg adaptive radiation across the archipelago chronosequence\nAuthors: Yang, H.; Goubert, C.; Cotoras, D.; Graham, N.; Cerca, J.; Gillespie, R. G.\nScore: 10.6, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574070\nThe ecological and phenotypic diversity observed in oceanic island radiations presents an evolutionary paradox: a high level of genetic variation is typically required for diversification, but species colonizing a new island typically suffer from founder effects.",
  "keywords": [
    
  ],
  "articleBody": " Consistent accumulation of transposable elements in species of the Hawaiian Tetragnatha spiny-leg adaptive radiation across the archipelago chronosequence\nAuthors: Yang, H.; Goubert, C.; Cotoras, D.; Graham, N.; Cerca, J.; Gillespie, R. G.\nScore: 10.6, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574070\nThe ecological and phenotypic diversity observed in oceanic island radiations presents an evolutionary paradox: a high level of genetic variation is typically required for diversification, but species colonizing a new island typically suffer from founder effects. This reduction in population size leads to a reduction in genetic diversity, which ultimately results in a reduction in the efficiency of natural selection. Then, what is the source of genetic variation which acts as the raw material for ecological and phenotypic diversification in oceanic archipelagos? Transposable elements (TEs) are mobile genetic elements that have been linked to the generation of genetic diversity, and evidence suggests that TE activity and accumulation along the genome can result from reductions in population size. Here, we use the Hawaiian spiny-leg spider radiation (Tetragnatha) to test whether TE accumulation increases due to demographic processes associated with island colonization. We sequenced and quantified TEs in 23 individuals from the spiny-leg radiation and 4 individuals from its sister radiation, the Hawaiian web-building Tetragnatha. Our results show that founder effects resulting from colonization of new islands have not resulted in TE accumulation over evolutionary time. Specifically, we found no evidence for increase in abundance of specific TE superfamilies, nor an accumulation of young TEs in lineages which have recently colonized a new island or are present in islands with active volcanoes. We also found that the DNA/hAT transposon superfamily is by far the most abundant TE superfamily in the Tetragnatha radiation. This work shows that TE abundance has remained constant for the spiny-leg radiation across the archipelago chronosequence, and TE accumulation is not affected by population oscillations associated with island colonization events. Therefore, despite their known role in the generation of genetic diversity, TE activity does not appear to be the mechanism to explain the evolutionary paradox of the insular Tetragnatha spiny-leg radiation.\nThe role of transposon activity in shaping cis-regulatory element evolution after whole genome duplication\nAuthors: Monsen, O.; Gronvold, L.; Datsomor, A.; Harvey, T. N.; Kijas, J.; Suh, A. S.-J.; Hvidsten, T. R.; Sandve, S. R.\nScore: 9.8, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573861\nBackgroundTwo of the most potent drivers of genome evolution in eukaryotes are whole genome duplications (WGD) and transposable element (TE) activity. These two mutational forces can also play synergistic roles; WGDs result in both cellular stress and functional redundancy, which would allow TEs to escape host-silencing mechanisms and effectively spread with reduced impact on fitness. As TEs can function as, or evolve into, TE-derived cis-regulatory elements (TE-CREs), bursts of TE-activity following WGD are likely to impact evolution of gene regulation. However, the role of TEs in genome regulatory remodelling after WGDs is unclear. Here we used the genome of Atlantic salmon, which is known to have experienced massive expansion of TEs after a WGD [~]100 Mya, as a model system to explore the synergistic roles of TEs and WGDs on genome regulatory evolution. ResultsWe identified 61,309 putative TE-CREs in Atlantic salmon using chromatin accessibility data from brain and liver. Of these, 82% were tissue specific to liver (43%) or brain (39%) and TE-CREs originating from retroelements were twice as common as those originating from DNA elements. Signatures of selection shaping TE-CRE evolution were evident from depletion of TEs in open chromatin, a bias in tissue-shared TE-CREs towards older TE-insertions, as well as tissue-specific processes shaping the TE-CRE repertoire. The DTT elements (Tc1-Mariners), which exploded in numbers at the time of the WGD, were significantly less prone to evolve into TE-CREs and significantly less potent in driving or repressing transcription compared to other TE-derived sequences. A minority of TEs (16% of consensus TEs) accounted for the origin of 46% of all TE-CREs, but these CRE-superspreaders were not temporally associated with the WGD. Rather, the majority of TE-CREs, including those found to be significantly associated with gene regulatory evolution and thus found to drive or repress transcription, evolved from TE activity occurring across tens of millions of years following the WGD event. ConclusionOur results do not support a WGD-associated TE-CRE rewiring of gene regulation. Instead we find that TEs from diverse superfamilies have been particularly effective in spreading TE-CREs and shaping gene regulatory networks under tissue-specific selection pressures, across millions of years following the salmonid WGD.\nRetention of an endosymbiont for the production of a single molecule\nAuthors: Garber, A.; Garcia de la Filia Molina, A.; Vea, I. M.; Mongue, A. J.; Ross, L.; McCutcheon, J. P.\nScore: 8.3, Published: 2024-01-05 DOI: 10.1101/2024.01.04.574232\nSap-feeding insects often maintain multiple nutritional endosymbionts, which act in concert to produce compounds essential for insect survival. Many mealybugs have endosymbionts in a nested configuration: one or two bacterial species reside within the cytoplasm of another bacterium, and together these bacteria have genomes which encode interdependent but complete sets of genes needed to produce key nutritional molecules. Here we show that the mealybug Pseudococcus viburni has three endosymbionts, one of which contributes only two genes that produce a single host nutrition-related molecule. All three bacterial endosymbionts have tiny genomes, suggesting that they have been co-evolving inside their insect host for millions of years.\nHorizontal transfer and recombination fuel Ty4 retrotransposon evolution in Saccharomyces.\nAuthors: Chen, J.; Garfinkel, D. J.; Bergman, C. M.\nScore: 7.2, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572574\nHorizontal transposon transfer (HTT) plays an important role in the evolution of eukaryotic genomes, however the detailed evolutionary history and impact of most HTT events remain to be elucidated. To better understand the process of HTT in closely-related microbial eukaryotes, we studied Ty4 retrotransposon subfamily content and sequence evolution across the genus Saccharomyces using short- and long-read whole genome sequence data, including new PacBio genome assemblies for two S. mikatae strains. We find evidence for multiple independent HTT events introducing the Tsu4 subfamily into specific lineages of S. paradoxus, S. cerevisiae, S. eubayanus, S. kudriavzevii and the ancestor of the S. mikatae/S. jurei species pair. In both S. mikatae and S. kudriavzevii , we identified novel Ty4 clades that were independently generated through recombination between resident and horizontally-transferred subfamilies. Our results reveal that recurrent HTT and lineage-specific extinction events lead to a complex pattern of Ty4 subfamily content across the genus Saccharomyces. Moreover, our results demonstrate how HTT can lead to coexistence of related retrotransposon subfamilies in the same genome that can fuel evolution of new retrotransposon clades via recombination.\nDistinct ancient structural polymorphisms control heterodichogamy in walnuts and hickories\nAuthors: Groh, J. S.; Vik, D. C.; Stevens, K. A.; Brown, P. J.; Langley, C. H.; Coop, G.\nScore: 21.0, Published: 2023-12-23 DOI: 10.1101/2023.12.23.573205\nThe maintenance of stable mating type polymorphisms is a classic example of balancing selection, underlying the nearly ubiquitous 50/50 sex ratio in species with separate sexes. One lesser known but intriguing example of a balanced mating polymorphism in angiosperms is heterodichogamy - polymorphism for opposing directions of dichogamy (temporal separation of male and female function in hermaphrodites) within a flowering season. This mating system is common throughout Juglandaceae, the family that includes globally important and iconic nut and timber crops - walnuts (Juglans), as well as pecan and other hickories (Carya). We show that two distinct genetic systems control heterodichogamous mating systems in these genera. In each genus, we find ancient (\u003e50Mya) structural variants that segregate as trans-species polymorphisms. The Juglans locus maps to a ca. 20kb structural variant adjacent to gene with likely function in the trehalose-6-phosphate signaling pathway, which is known to regulate flowering time in Arabidopsis and inflorescence architecture in maize. Notably, an insertion in the dominant (protogynous) haplotype contains 8-12 complex tandem duplicates of the 3 UTR of the gene, and transpecies polymorphism within the gene is restricted to the 3 UTR. In contrast, the Carya heterodichogamy locus maps to a ca. 200-450 kb cluster of tightly linked polymorphisms across 20 genes, some of which have known roles in flowering and are differentially expressed between morphs in floral primordia. The dominant (protogynous) haplotype in pecan, which is nearly always heterozygous and appears to rarely recombine, shows reduced genetic diversity and is over twice as long as its recessive counterpart due to proliferation of multiple transposable element families. We did not detect either genetic system in another heterodichogamous genus within the Juglandaceae, suggesting that additional genetic systems for heterodichogamy may yet remain undiscovered.\nFunctional consequences of reductive protein evolution in a minimal eukaryotic genome\nAuthors: Jiang, J.; Qu, R.; Grigorescu, M.; Zhao, W.; Reinke, A. W.\nScore: 3.0, Published: 2024-01-01 DOI: 10.1101/2023.12.31.573788\nMicrosporidia are parasites with the smallest known eukaryotic genomes. The extent of protein loss in these organisms has been well documented, but much less is known about how compaction of microsporidia proteins affects their function. Taking a comparative genomic approach, we identified microsporidia orthologs of budding yeast proteins and show that these orthologs are enriched for essential yeast genes. We show that the median microsporidia protein is 21% shorter than its yeast counterpart and although extensive protein loss occurred after the divergence of microsporidia, reduced protein sizes were already present in microsporidian relatives. Microsporidia proteins are shorter through reduced domain lengths, diminished linker lengths, and domain loss, with 21% of microsporidia orthologs having lost domains present in yeast. On average, 34% of microsporidia orthologs have lost C-terminal residues essential for function in yeast, including 13 essential domains lost per genome. We also found that microsporidia display distinct phylogenetic patterns of domain loss, with losses occurring in a clade-specific manner. To investigate conservation of function, we used yeast complementation assays to test orthologs from several microsporidia species and their relative Rozella allomycis. These experiments reveal that most microsporidia proteins cannot complement their yeast orthologs, the ability to complement is about three-fold less than observed for R. allomycis orthologs, and proteins that do not complement are more reduced in length than their yeast orthologs. Altogether, our results demonstrate the drastic reduction of microsporidia proteins and show that these reductions have resulted in functional divergence from their fungal ancestors.\nThe recombination landscape of introgression in yeast\nAuthors: Schwarzkopf, E. J.; Brandt, N.; Smukowski Heil, C.\nScore: 2.9, Published: 2024-01-05 DOI: 10.1101/2024.01.04.574263\nMeiotic recombination is an evolutionary force that acts by breaking up genomic linkage, increasing the efficacy of selection. Recombination is initiated with a double-strand break which is resolved via a crossover, which involves the reciprocal exchange of genetic material between homologous chromosomes, or a non-crossover, which results in small tracts of non-reciprocal exchange of genetic material. Crossover and non-crossover rates vary between species, populations, individuals, and across the genome. In recent years, recombination rate has been associated with the distribution of ancestry derived from past interspecific hybridization (introgression) in a variety of species. We explore this interaction of recombination and introgression by sequencing spores and detecting crossovers and non-crossovers from two crosses of the yeast Saccharomyces uvarum. One cross is between strains which each contain introgression from their sister species, S. eubayanus, while the other cross has no introgression present. We find that the recombination landscape is significantly different between S. uvarum crosses, and that most of these differences can be explained by the presence of introgression in one cross. Crossovers are reduced and non-crossovers are increased in heterozygous introgression compared to syntenic regions in the cross without introgression. This translates to reduced allele shuffling within introgressed regions, and an overall reduction of shuffling on most chromosomes with introgression compared to the syntenic regions and chromosomes without introgression. Our results suggest that hybridization can significantly influence the recombination landscape, and that the reduction in allele shuffling contributes to the initial purging of introgression in the generations following a hybridization event.\nAncestral Sequence Reconstruction as a tool to detect and study de novo gene emergence\nAuthors: Vakirlis, N.; Acar, O.; Cherupally, V. K.; Carvunis, A.-R.\nScore: 2.5, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573862\nNew protein-coding genes can evolve from previously non-coding genomic regions through a process known as de novo gene emergence. Evidence suggests that this process has likely occurred throughout evolution and across the tree of life. Yet, confidently identifying de novo emerged genes remains challenging. Ancestral Sequence Reconstruction (ASR) is a promising approach for inferring whether a gene has emerged de novo or not, as it can enable us to inspect whether a given genomic locus ancestrally harbored protein-coding capacity. However, the use of ASR in the context of de novo emergence is still in its infancy and its capabilities, limitations, and overall potential are largely unknown. Notably, it is difficult to formally evaluate the protein-coding capacity of ancestral sequences, particularly when new gene candidates are short. How well-suited is ASR as a tool for the detection and study of de novo genes? Here, we address this question by designing an ASR workflow incorporating different tools and sets of parameters and by introducing a formal criterion that allows to estimate, within a desired level of confidence, when protein-coding capacity originated at a particular locus. Applying this workflow on [~]2,600 short, annotated budding yeast genes (\u003c1,000 nucleotides), we found that ASR robustly predicts an ancient origin for most widely conserved genes, which constitute \"easy\" cases. For less robust cases, we calculated a randomization-based empirical P-value estimating whether the observed conservation between the extant and ancestral reading frame could be attributed to chance. This formal criterion allowed us to pinpoint a branch of origin for most of the less robust cases, identifying 33 genes that can unequivocally be considered de novo originated since the split of the Saccharomyces genus, including 20 S. cerevisiae-specific genes. We find that the remaining, equivocal cases, may be explained by different evolutionary scenarios including rapid evolution and multiple losses, as well as a very recent de novo origin. Overall, our findings suggest that ASR is a valuable tool to study de novo gene emergence but should be applied with caution and awareness of its limitations.\nQuantitative species delimitation, tests for gene flow, and migration models uncover complex, recent speciation in tree squirrels\nAuthors: Abreu, E. F.; Prado, J. R.; Maldonado, J. E.; Wilson, D. E.; Percequillo, A. R.; Pavan, S. E.\nScore: 2.2, Published: 2024-01-05 DOI: 10.1101/2024.01.04.574244\nAccurate estimates of species diversity are essential for all biodiversity research. Delimiting species and understanding the underlaying processes of speciation are also central components of systematic biology that outline our comprehension of the evolutionary mechanisms generating biodiversity. We obtained genomic data (Ultraconserved Elements and single nucleotide polymorphisms) for a widespread genus of South American tree squirrels (genus Guerlinguetus) to explore alternative hypotheses on species limits and to clarify recent and rapid speciation on continental-scale and dynamically evolving landscapes. Using a multilayered genomic approach that integrates fine-scale population genetic analyses with quantitative molecular species delimitation methods, we observed that (i) the most likely number of species within Guerlinguetus is six, contrasting with both classic morphological revision and mitochondrial species delimitation; (ii) incongruencies in species relationships still persist, which might be a response to population migration and gene flow taking place in lowlands of eastern Amazonia; and (iii) effective migration surfaces detected important geographic barriers associated with the major Amazonian riverine systems and the mountain ranges of the Guiana shield. In conclusion, we uncovered unexpected species diversity of keystone mammals that are critical in tree-seed predation and dispersal in one of the most fragile and threated ecosystems of the world, the tropical rainforests of South America. Our results corroborate recent findings suggesting that much of the extant species-level diversity in Amazonia is young, dating back to the Quaternary, while also reinforcing long-established hypotheses on the role of rivers and climate-driven forest dynamics triggering Amazonian speciation.\nPost-zygotic reproductive isolation is not correlated with chromosome number in plants.\nAuthors: Finch, G. S.; Yang, Y.; Barker, M. S.\nScore: 2.0, Published: 2024-01-05 DOI: 10.1101/2024.01.05.573914\nThe evolution of chromosome numbers is an important but not fully understood aspect of eukaryotic evolution. Although we understand the types of karyotypic changes that can lead to chromosome gain and loss, we still do not understand why chromosome numbers in many plants and animals have an average of n = 9. A recent hypothesis proposed that chromosome number reduction following whole genome duplication (WGD) in angiosperms is driven by an interaction between chromosome number and the strength of reproductive isolation among populations. Chromosome number is expected to determine the maximum number of independently assorting Bateson-Dobzhansky-Muller incompatibilities (BDMIs). Selection against restricted gene flow among populations would result in selection for reduced chromosome number. Here we test for an interaction between chromosome number and post-zygotic reproductive isolation across a broad sample of land plants. We additionally tested for the indirect effects of WGD in generating post-zygotic reproductive isolation. Such an effect is expected if reproductive isolation is driven largely by reciprocal gene loss of WGD-derived paralogs. We found no correlation between post-zygotic reproductive isolation and chromosome number, WGD age, or degree of fractionation, suggesting that the accumulation of genic incompatibilities is likely not a major driver of post-WGD chromosome number reduction.\n",
  "wordCount" : "2826",
  "inLanguage": "en",
  "datePublished": "2024-01-07T10:38:59Z",
  "dateModified": "2024-01-07T10:38:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 7, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574070">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574070" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574070">
        <p class="paperTitle">Consistent accumulation of transposable elements in species of the Hawaiian Tetragnatha spiny-leg adaptive radiation across the archipelago chronosequence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574070" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574070" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, H.; Goubert, C.; Cotoras, D.; Graham, N.; Cerca, J.; Gillespie, R. G.</p>
        <p class="info">Score: 10.6, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574070' target='https://doi.org/10.1101/2024.01.03.574070'> 10.1101/2024.01.03.574070</a></p>
        <p class="abstract">The ecological and phenotypic diversity observed in oceanic island radiations presents an evolutionary paradox: a high level of genetic variation is typically required for diversification, but species colonizing a new island typically suffer from founder effects. This reduction in population size leads to a reduction in genetic diversity, which ultimately results in a reduction in the efficiency of natural selection. Then, what is the source of genetic variation which acts as the raw material for ecological and phenotypic diversification in oceanic archipelagos? Transposable elements (TEs) are mobile genetic elements that have been linked to the generation of genetic diversity, and evidence suggests that TE activity and accumulation along the genome can result from reductions in population size. Here, we use the Hawaiian spiny-leg spider radiation (Tetragnatha) to test whether TE accumulation increases due to demographic processes associated with island colonization. We sequenced and quantified TEs in 23 individuals from the spiny-leg radiation and 4 individuals from its sister radiation, the Hawaiian web-building Tetragnatha. Our results show that founder effects resulting from colonization of new islands have not resulted in TE accumulation over evolutionary time. Specifically, we found no evidence for increase in abundance of specific TE superfamilies, nor an accumulation of  young TEs in lineages which have recently colonized a new island or are present in islands with active volcanoes. We also found that the DNA/hAT transposon superfamily is by far the most abundant TE superfamily in the Tetragnatha radiation. This work shows that TE abundance has remained constant for the spiny-leg radiation across the archipelago chronosequence, and TE accumulation is not affected by population oscillations associated with island colonization events. Therefore, despite their known role in the generation of genetic diversity, TE activity does not appear to be the mechanism to explain the evolutionary paradox of the insular Tetragnatha spiny-leg radiation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573861">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573861" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573861">
        <p class="paperTitle">The role of transposon activity in shaping cis-regulatory element evolution after whole genome duplication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573861" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573861" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Monsen, O.; Gronvold, L.; Datsomor, A.; Harvey, T. N.; Kijas, J.; Suh, A. S.-J.; Hvidsten, T. R.; Sandve, S. R.</p>
        <p class="info">Score: 9.8, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573861' target='https://doi.org/10.1101/2024.01.02.573861'> 10.1101/2024.01.02.573861</a></p>
        <p class="abstract">BackgroundTwo of the most potent drivers of genome evolution in eukaryotes are whole genome duplications (WGD) and transposable element (TE) activity. These two mutational forces can also play synergistic roles; WGDs result in both cellular stress and functional redundancy, which would allow TEs to escape host-silencing mechanisms and effectively spread with reduced impact on fitness. As TEs can function as, or evolve into, TE-derived cis-regulatory elements (TE-CREs), bursts of TE-activity following WGD are likely to impact evolution of gene regulation. However, the role of TEs in genome regulatory remodelling after WGDs is unclear. Here we used the genome of Atlantic salmon, which is known to have experienced massive expansion of TEs after a WGD [~]100 Mya, as a model system to explore the synergistic roles of TEs and WGDs on genome regulatory evolution.

ResultsWe identified 61,309 putative TE-CREs in Atlantic salmon using chromatin accessibility data from brain and liver. Of these, 82% were tissue specific to liver (43%) or brain (39%) and TE-CREs originating from retroelements were twice as common as those originating from DNA elements. Signatures of selection shaping TE-CRE evolution were evident from depletion of TEs in open chromatin, a bias in tissue-shared TE-CREs towards older TE-insertions, as well as tissue-specific processes shaping the TE-CRE repertoire. The DTT elements (Tc1-Mariners), which exploded in numbers at the time of the WGD, were significantly less prone to evolve into TE-CREs and significantly less potent in driving or repressing transcription compared to other TE-derived sequences. A minority of TEs (16% of consensus TEs) accounted for the origin of 46% of all TE-CREs, but these  CRE-superspreaders were not temporally associated with the WGD. Rather, the majority of TE-CREs, including those found to be significantly associated with gene regulatory evolution and thus found to drive or repress transcription, evolved from TE activity occurring across tens of millions of years following the WGD event.

ConclusionOur results do not support a WGD-associated TE-CRE rewiring of gene regulation. Instead we find that TEs from diverse superfamilies have been particularly effective in spreading TE-CREs and shaping gene regulatory networks under tissue-specific selection pressures, across millions of years following the salmonid WGD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574232">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574232" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574232">
        <p class="paperTitle">Retention of an endosymbiont for the production of a single molecule</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574232" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574232" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garber, A.; Garcia de la Filia Molina, A.; Vea, I. M.; Mongue, A. J.; Ross, L.; McCutcheon, J. P.</p>
        <p class="info">Score: 8.3, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574232' target='https://doi.org/10.1101/2024.01.04.574232'> 10.1101/2024.01.04.574232</a></p>
        <p class="abstract">Sap-feeding insects often maintain multiple nutritional endosymbionts, which act in concert to produce compounds essential for insect survival. Many mealybugs have endosymbionts in a nested configuration: one or two bacterial species reside within the cytoplasm of another bacterium, and together these bacteria have genomes which encode interdependent but complete sets of genes needed to produce key nutritional molecules. Here we show that the mealybug Pseudococcus viburni has three endosymbionts, one of which contributes only two genes that produce a single host nutrition-related molecule. All three bacterial endosymbionts have tiny genomes, suggesting that they have been co-evolving inside their insect host for millions of years.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572574">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572574" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572574">
        <p class="paperTitle">Horizontal transfer and recombination fuel Ty4 retrotransposon evolution in Saccharomyces.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572574" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572574" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, J.; Garfinkel, D. J.; Bergman, C. M.</p>
        <p class="info">Score: 7.2, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572574' target='https://doi.org/10.1101/2023.12.20.572574'> 10.1101/2023.12.20.572574</a></p>
        <p class="abstract">Horizontal transposon transfer (HTT) plays an important role in the evolution of eukaryotic genomes, however the detailed evolutionary history and impact of most HTT events remain to be elucidated. To better understand the process of HTT in closely-related microbial eukaryotes, we studied Ty4 retrotransposon subfamily content and sequence evolution across the genus Saccharomyces using short- and long-read whole genome sequence data, including new PacBio genome assemblies for two S. mikatae strains. We find evidence for multiple independent HTT events introducing the Tsu4 subfamily into specific lineages of S. paradoxus, S. cerevisiae, S. eubayanus, S. kudriavzevii and the ancestor of the S. mikatae/S. jurei species pair. In both S. mikatae and S. kudriavzevii , we identified novel Ty4 clades that were independently generated through recombination between resident and horizontally-transferred subfamilies. Our results reveal that recurrent HTT and lineage-specific extinction events lead to a complex pattern of Ty4 subfamily content across the genus Saccharomyces. Moreover, our results demonstrate how HTT can lead to coexistence of related retrotransposon subfamilies in the same genome that can fuel evolution of new retrotransposon clades via recombination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573205">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573205" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573205">
        <p class="paperTitle">Distinct ancient structural polymorphisms control heterodichogamy in walnuts and hickories</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573205" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573205" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Groh, J. S.; Vik, D. C.; Stevens, K. A.; Brown, P. J.; Langley, C. H.; Coop, G.</p>
        <p class="info">Score: 21.0, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573205' target='https://doi.org/10.1101/2023.12.23.573205'> 10.1101/2023.12.23.573205</a></p>
        <p class="abstract">The maintenance of stable mating type polymorphisms is a classic example of balancing selection, underlying the nearly ubiquitous 50/50 sex ratio in species with separate sexes. One lesser known but intriguing example of a balanced mating polymorphism in angiosperms is heterodichogamy - polymorphism for opposing directions of dichogamy (temporal separation of male and female function in hermaphrodites) within a flowering season. This mating system is common throughout Juglandaceae, the family that includes globally important and iconic nut and timber crops - walnuts (Juglans), as well as pecan and other hickories (Carya). We show that two distinct genetic systems control heterodichogamous mating systems in these genera. In each genus, we find ancient (&gt;50Mya) structural variants that segregate as trans-species polymorphisms. The Juglans locus maps to a ca. 20kb structural variant adjacent to gene with likely function in the trehalose-6-phosphate signaling pathway, which is known to regulate flowering time in Arabidopsis and inflorescence architecture in maize. Notably, an insertion in the dominant (protogynous) haplotype contains 8-12 complex tandem duplicates of the 3 UTR of the gene, and transpecies polymorphism within the gene is restricted to the 3 UTR. In contrast, the Carya heterodichogamy locus maps to a ca. 200-450 kb cluster of tightly linked polymorphisms across 20 genes, some of which have known roles in flowering and are differentially expressed between morphs in floral primordia. The dominant (protogynous) haplotype in pecan, which is nearly always heterozygous and appears to rarely recombine, shows reduced genetic diversity and is over twice as long as its recessive counterpart due to proliferation of multiple transposable element families. We did not detect either genetic system in another heterodichogamous genus within the Juglandaceae, suggesting that additional genetic systems for heterodichogamy may yet remain undiscovered.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.31.573788">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.31.573788" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.31.573788">
        <p class="paperTitle">Functional consequences of reductive protein evolution in a minimal eukaryotic genome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.31.573788" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.31.573788" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, J.; Qu, R.; Grigorescu, M.; Zhao, W.; Reinke, A. W.</p>
        <p class="info">Score: 3.0, Published: 2024-01-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.31.573788' target='https://doi.org/10.1101/2023.12.31.573788'> 10.1101/2023.12.31.573788</a></p>
        <p class="abstract">Microsporidia are parasites with the smallest known eukaryotic genomes. The extent of protein loss in these organisms has been well documented, but much less is known about how compaction of microsporidia proteins affects their function. Taking a comparative genomic approach, we identified microsporidia orthologs of budding yeast proteins and show that these orthologs are enriched for essential yeast genes. We show that the median microsporidia protein is 21% shorter than its yeast counterpart and although extensive protein loss occurred after the divergence of microsporidia, reduced protein sizes were already present in microsporidian relatives. Microsporidia proteins are shorter through reduced domain lengths, diminished linker lengths, and domain loss, with 21% of microsporidia orthologs having lost domains present in yeast. On average, 34% of microsporidia orthologs have lost C-terminal residues essential for function in yeast, including 13 essential domains lost per genome. We also found that microsporidia display distinct phylogenetic patterns of domain loss, with losses occurring in a clade-specific manner. To investigate conservation of function, we used yeast complementation assays to test orthologs from several microsporidia species and their relative Rozella allomycis. These experiments reveal that most microsporidia proteins cannot complement their yeast orthologs, the ability to complement is about three-fold less than observed for R. allomycis orthologs, and proteins that do not complement are more reduced in length than their yeast orthologs. Altogether, our results demonstrate the drastic reduction of microsporidia proteins and show that these reductions have resulted in functional divergence from their fungal ancestors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574263">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574263" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574263">
        <p class="paperTitle">The recombination landscape of introgression in yeast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574263" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574263" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schwarzkopf, E. J.; Brandt, N.; Smukowski Heil, C.</p>
        <p class="info">Score: 2.9, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574263' target='https://doi.org/10.1101/2024.01.04.574263'> 10.1101/2024.01.04.574263</a></p>
        <p class="abstract">Meiotic recombination is an evolutionary force that acts by breaking up genomic linkage, increasing the efficacy of selection. Recombination is initiated with a double-strand break which is resolved via a crossover, which involves the reciprocal exchange of genetic material between homologous chromosomes, or a non-crossover, which results in small tracts of non-reciprocal exchange of genetic material. Crossover and non-crossover rates vary between species, populations, individuals, and across the genome. In recent years, recombination rate has been associated with the distribution of ancestry derived from past interspecific hybridization (introgression) in a variety of species. We explore this interaction of recombination and introgression by sequencing spores and detecting crossovers and non-crossovers from two crosses of the yeast Saccharomyces uvarum. One cross is between strains which each contain introgression from their sister species, S. eubayanus, while the other cross has no introgression present. We find that the recombination landscape is significantly different between S. uvarum crosses, and that most of these differences can be explained by the presence of introgression in one cross. Crossovers are reduced and non-crossovers are increased in heterozygous introgression compared to syntenic regions in the cross without introgression. This translates to reduced allele shuffling within introgressed regions, and an overall reduction of shuffling on most chromosomes with introgression compared to the syntenic regions and chromosomes without introgression. Our results suggest that hybridization can significantly influence the recombination landscape, and that the reduction in allele shuffling contributes to the initial purging of introgression in the generations following a hybridization event.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573862">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573862" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573862">
        <p class="paperTitle">Ancestral Sequence Reconstruction as a tool to detect and study de novo gene emergence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573862" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573862" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vakirlis, N.; Acar, O.; Cherupally, V. K.; Carvunis, A.-R.</p>
        <p class="info">Score: 2.5, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573862' target='https://doi.org/10.1101/2024.01.02.573862'> 10.1101/2024.01.02.573862</a></p>
        <p class="abstract">New protein-coding genes can evolve from previously non-coding genomic regions through a process known as de novo gene emergence. Evidence suggests that this process has likely occurred throughout evolution and across the tree of life. Yet, confidently identifying de novo emerged genes remains challenging. Ancestral Sequence Reconstruction (ASR) is a promising approach for inferring whether a gene has emerged de novo or not, as it can enable us to inspect whether a given genomic locus ancestrally harbored protein-coding capacity. However, the use of ASR in the context of de novo emergence is still in its infancy and its capabilities, limitations, and overall potential are largely unknown. Notably, it is difficult to formally evaluate the protein-coding capacity of ancestral sequences, particularly when new gene candidates are short. How well-suited is ASR as a tool for the detection and study of de novo genes? Here, we address this question by designing an ASR workflow incorporating different tools and sets of parameters and by introducing a formal criterion that allows to estimate, within a desired level of confidence, when protein-coding capacity originated at a particular locus. Applying this workflow on [~]2,600 short, annotated budding yeast genes (&lt;1,000 nucleotides), we found that ASR robustly predicts an ancient origin for most widely conserved genes, which constitute &#34;easy&#34; cases. For less robust cases, we calculated a randomization-based empirical P-value estimating whether the observed conservation between the extant and ancestral reading frame could be attributed to chance. This formal criterion allowed us to pinpoint a branch of origin for most of the less robust cases, identifying 33 genes that can unequivocally be considered de novo originated since the split of the Saccharomyces genus, including 20 S. cerevisiae-specific genes. We find that the remaining, equivocal cases, may be explained by different evolutionary scenarios including rapid evolution and multiple losses, as well as a very recent de novo origin. Overall, our findings suggest that ASR is a valuable tool to study de novo gene emergence but should be applied with caution and awareness of its limitations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574244">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574244" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574244">
        <p class="paperTitle">Quantitative species delimitation, tests for gene flow, and migration models uncover complex, recent speciation in tree squirrels</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574244" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574244" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abreu, E. F.; Prado, J. R.; Maldonado, J. E.; Wilson, D. E.; Percequillo, A. R.; Pavan, S. E.</p>
        <p class="info">Score: 2.2, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574244' target='https://doi.org/10.1101/2024.01.04.574244'> 10.1101/2024.01.04.574244</a></p>
        <p class="abstract">Accurate estimates of species diversity are essential for all biodiversity research. Delimiting species and understanding the underlaying processes of speciation are also central components of systematic biology that outline our comprehension of the evolutionary mechanisms generating biodiversity. We obtained genomic data (Ultraconserved Elements and single nucleotide polymorphisms) for a widespread genus of South American tree squirrels (genus Guerlinguetus) to explore alternative hypotheses on species limits and to clarify recent and rapid speciation on continental-scale and dynamically evolving landscapes. Using a multilayered genomic approach that integrates fine-scale population genetic analyses with quantitative molecular species delimitation methods, we observed that (i) the most likely number of species within Guerlinguetus is six, contrasting with both classic morphological revision and mitochondrial species delimitation; (ii) incongruencies in species relationships still persist, which might be a response to population migration and gene flow taking place in lowlands of eastern Amazonia; and (iii) effective migration surfaces detected important geographic barriers associated with the major Amazonian riverine systems and the mountain ranges of the Guiana shield. In conclusion, we uncovered unexpected species diversity of keystone mammals that are critical in tree-seed predation and dispersal in one of the most fragile and threated ecosystems of the world, the tropical rainforests of South America. Our results corroborate recent findings suggesting that much of the extant species-level diversity in Amazonia is young, dating back to the Quaternary, while also reinforcing long-established hypotheses on the role of rivers and climate-driven forest dynamics triggering Amazonian speciation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.573914">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.573914" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.573914">
        <p class="paperTitle">Post-zygotic reproductive isolation is not correlated with chromosome number in plants.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.573914" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.573914" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Finch, G. S.; Yang, Y.; Barker, M. S.</p>
        <p class="info">Score: 2.0, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.573914' target='https://doi.org/10.1101/2024.01.05.573914'> 10.1101/2024.01.05.573914</a></p>
        <p class="abstract">The evolution of chromosome numbers is an important but not fully understood aspect of eukaryotic evolution. Although we understand the types of karyotypic changes that can lead to chromosome gain and loss, we still do not understand why chromosome numbers in many plants and animals have an average of n = 9. A recent hypothesis proposed that chromosome number reduction following whole genome duplication (WGD) in angiosperms is driven by an interaction between chromosome number and the strength of reproductive isolation among populations. Chromosome number is expected to determine the maximum number of independently assorting Bateson-Dobzhansky-Muller incompatibilities (BDMIs). Selection against restricted gene flow among populations would result in selection for reduced chromosome number. Here we test for an interaction between chromosome number and post-zygotic reproductive isolation across a broad sample of land plants. We additionally tested for the indirect effects of WGD in generating post-zygotic reproductive isolation. Such an effect is expected if reproductive isolation is driven largely by reciprocal gene loss of WGD-derived paralogs. We found no correlation between post-zygotic reproductive isolation and chromosome number, WGD age, or degree of fractionation, suggesting that the accumulation of genic incompatibilities is likely not a major driver of post-WGD chromosome number reduction.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
