<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Functional Adaptations of Endogenous Retroviruses to the Drosophila Host Underlie their Evolutionary Diversification
Authors: Senti, K.-A.; Handler, D.; Rafanel, B.; Kosiol, C.; Schloetterer, C.; Brennecke, J.
Score: 52.7, Published: 2023-08-04 DOI: 10.1101/2023.08.03.551782
Transposable elements profoundly affect the biology and evolution of their hosts, yet their own evolutionary dynamics remain poorly understood. Here, we investigate insect endogenous retroviruses (iERVs), a monophyletic group of LTR retrotransposons that have acquired the trait of infectivity, likely through capture of a Baculovirus envelope gene.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Functional Adaptations of Endogenous Retroviruses to the Drosophila Host Underlie their Evolutionary Diversification
Authors: Senti, K.-A.; Handler, D.; Rafanel, B.; Kosiol, C.; Schloetterer, C.; Brennecke, J.
Score: 52.7, Published: 2023-08-04 DOI: 10.1101/2023.08.03.551782
Transposable elements profoundly affect the biology and evolution of their hosts, yet their own evolutionary dynamics remain poorly understood. Here, we investigate insect endogenous retroviruses (iERVs), a monophyletic group of LTR retrotransposons that have acquired the trait of infectivity, likely through capture of a Baculovirus envelope gene." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-13T10:36:01+00:00" />
<meta property="article:modified_time" content="2023-08-13T10:36:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Functional Adaptations of Endogenous Retroviruses to the Drosophila Host Underlie their Evolutionary Diversification
Authors: Senti, K.-A.; Handler, D.; Rafanel, B.; Kosiol, C.; Schloetterer, C.; Brennecke, J.
Score: 52.7, Published: 2023-08-04 DOI: 10.1101/2023.08.03.551782
Transposable elements profoundly affect the biology and evolution of their hosts, yet their own evolutionary dynamics remain poorly understood. Here, we investigate insect endogenous retroviruses (iERVs), a monophyletic group of LTR retrotransposons that have acquired the trait of infectivity, likely through capture of a Baculovirus envelope gene."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Functional Adaptations of Endogenous Retroviruses to the Drosophila Host Underlie their Evolutionary Diversification\nAuthors: Senti, K.-A.; Handler, D.; Rafanel, B.; Kosiol, C.; Schloetterer, C.; Brennecke, J.\nScore: 52.7, Published: 2023-08-04 DOI: 10.1101/2023.08.03.551782\nTransposable elements profoundly affect the biology and evolution of their hosts, yet their own evolutionary dynamics remain poorly understood. Here, we investigate insect endogenous retroviruses (iERVs), a monophyletic group of LTR retrotransposons that have acquired the trait of infectivity, likely through capture of a Baculovirus envelope gene.",
  "keywords": [
    
  ],
  "articleBody": " Functional Adaptations of Endogenous Retroviruses to the Drosophila Host Underlie their Evolutionary Diversification\nAuthors: Senti, K.-A.; Handler, D.; Rafanel, B.; Kosiol, C.; Schloetterer, C.; Brennecke, J.\nScore: 52.7, Published: 2023-08-04 DOI: 10.1101/2023.08.03.551782\nTransposable elements profoundly affect the biology and evolution of their hosts, yet their own evolutionary dynamics remain poorly understood. Here, we investigate insect endogenous retroviruses (iERVs), a monophyletic group of LTR retrotransposons that have acquired the trait of infectivity, likely through capture of a Baculovirus envelope gene. In Drosophila ovaries, iERVs with functional envelope have adapted their cis-regulatory sequences to be expressed in any somatic cell type, from where they infect the germline. Strikingly, related retroviruses show distinct expression patterns, indicating niche partitioning. In contrast, all non-infectious iERVs that emerged through secondary envelope-loss are specifically expressed in the germline. Co-evolving with iERVs, the genome-protecting piRNA pathway has assimilated iERV promoter and sequence information into piRNA clusters, underscoring the functional significance of iERV expression in somatic niches. We propose that the evolutionary innovation of cell-to-cell infectivity has triggered the adaptive radiation of iERVs through trait diversification and antagonistic virus-host interactions, processes that likely underpin niche-specific expression of endogenous retroviruses in vertebrates as well.\nAn evolutionary timescale for Bacteria calibrated using the Great Oxidation Event\nAuthors: Davin, A. A.; Woodcroft, B. J.; Soo, R. M.; Murali, R.; Schrempf, D.; Clark, J.; Boussau, B.; Moody, E. R. R.; Szantho, L. L.; Richy, E.; Pisani, D.; Hemp, J.; Fischer, W. W.; Donoghue, P.; Spang, A.; Hugenholtz, P.; Williams, T. A.; SzoÌˆllosi, G. J.\nScore: 23.9, Published: 2023-08-11 DOI: 10.1101/2023.08.08.552427\nMost of life's diversity and history is microbial but it has left a meagre fossil record, greatly hindering understanding of evolution in deep time. However, the co-evolution of life and the Earth system has left signatures of bacterial metabolism in the geochemical record, most conspicuously the Great Oxidation Event (GOE) ~2.33 billion years ago (Ga, (Poulton et al. 2021)), in which oxygenic photosynthesis and tectonism (Eguchi, Seales, and Dasgupta 2019) transformed Earth's biosphere from dominantly anaerobic to aerobic. Here, we combine machine learning and phylogenetic reconciliation to infer ancestral transitions to aerobic lifestyles during bacterial evolution. Linking these transitions to the GOE provides new constraints to infer the timetree of Bacteria. We find that extant bacterial phyla are truly ancient, having radiated in the Archaean and the Proterozoic: the oldest include Bacillota (Firmicutes), which radiated 3.1-3.7 Ga, Cyanobacteria (3.3-3.5 Ga) and Patescibacteria (3-3.5 Ga). We show that most bacterial phyla were ancestrally anaerobic and that most transitions to an aerobic lifestyle post-dated the GOE. Our analyses trace oxygen production and consumption back to Cyanobacteria. From that starting point, horizontal transfer seeded aerobic lifestyles across bacterial diversity over hundreds of millions of years. Our analyses demonstrate that the diversification of aerobes proceeded in two waves corresponding to the GOE and to a second sustained rise in atmospheric O2 at the dawn of the Palezoic (Krause et al. 2022).\nTetrapod vocal evolution: higher frequencies and faster rates of evolution in mammalian vocalizations\nAuthors: Munoz, M. I.; Marsot, M.; Ellers, J.; Halfwerk, W.\nScore: 20.4, Published: 2023-08-09 DOI: 10.1101/2023.08.09.552622\nUsing the voice to produce sound is a widespread form of communication and plays an important role in contexts as diverse as parent-offspring interactions and courtship. Variation in the tempo and mode of vocal signal evolution has been studied in a phylogenetic context within orders or classes, but understanding vocal signal evolution ultimately requires comparison across all major lineages involved. Here we used comparative analyses to investigate the evolution of dominant frequency (i.e., the frequency with the highest energy content) and its association with body weight across 873 species of mammals, birds and frogs. In agreement with previous studies, we found that the negative allometric relationship between body weight and vocal frequency is a general feature of vocal systems. In addition, we found mammals to consistently vocalize at higher frequencies, and to evolve their vocalizations at around 6-fold faster rates than those of birds and frogs. Although all three groups strongly rely on vocal communication, our findings show that only mammals have extensively explored the spectral acoustic space available to them. We argue that such high vocal diversity of mammals is made possible by their unique hearing system, which evolved in the context of a small, parental-caring, nocturnal and insectivore ancestor, and has allowed them to detect, and therefore to evolve, a richer array of frequencies than other tetrapods.\nTissue-specific and cis-regulatory changes underlie parallel, adaptive gene expression evolution in house mice\nAuthors: Durkin, S. M.; Ballinger, M. A.; Nachman, M. W.\nScore: 14.4, Published: 2023-08-01 DOI: 10.1101/2023.08.01.551413\nChanges in gene regulation have long been appreciated as a driving force of adaptive evolution, however the relative contributions of cis- and trans-acting changes to gene regulation over short evolutionary timescales remain unclear. Instances of recent, parallel phenotypic evolution provide an opportunity to assess whether parallel patterns are seen at the level of gene expression, and to assess the relative contribution of cis- and trans-changes to gene regulation in the early stages of divergence. Here, we studied gene expression in liver and brown adipose tissue in two wild-derived strains of house mice that independently adapted to cold, northern environments, and we compared them to a strain of house mice from a warm, southern environment. To investigate gene regulatory evolution, we studied expression in parents and allele-specific expression in F1 hybrids of crosses between warm-adapted and cold-adapted strains. First, we found that the different cold-adapted mice showed both unique and shared changes in expression, but that the proportion of shared changes (i.e. parallelism) was greater than expected by chance. Second, we discovered that expression evolution occurred largely at tissue-specific and cis-regulated genes, and that these genes were over-represented in parallel cases of evolution. Finally, we integrated the expression data with scans for selection in natural populations and found substantial parallelism in the two northern populations for genes under selection. Furthermore, selection outliers were associated with cis-regulated genes more than expected by chance; cis-regulated genes under selection influenced phenotypes such as body size, fat content, immune functioning, and skeletal morphology. These results demonstrate that parallel patterns of gene expression in mice that have independently adapted to cold environments are driven largely by tissue-specific and cis-regulatory changes, providing insight into the mechanisms of adaptive gene regulatory evolution at the earliest stages of divergence. Author summaryThe parallel movement of organisms into novel environments provides an opportunity to understand the molecular basis of adaptation and the repeatability of this process. Mutations affecting the expression of genes are known to underlie much of adaptive evolution. Such mutations can arise in cis-(near the gene of interest) or in trans-(at a distant locus), but the relative contribution of these different kinds of changes to adaptation is poorly understood, especially during very recent divergence. Here, we compared evolved gene expression differences between a warm-adapted house mouse strain and two different cold-adapted strains that have independently evolved similar phenotypic traits, such as increased body size and decreased extremity length during the last few hundred years. Using crosses between warm-adapted and cold-adapted mice, we found that mutations acting in a context specific manner (cis-regulatory and tissue-specific changes) predominate expression divergence and are more likely to be involved in parallel evolution. We used population level selection scans in wild animals to identify regions of the genome under selection and combined these findings with the gene expression data to identify candidate genes underlying adaptation to novel environments. Together, our work describes the gene regulatory dynamics of rapid environmental adaptation, and the repeatability of these patterns over multiple instances of adaptation.\nGene expression is the main driver of purifying selection in large penguin populations\nAuthors: Trucchi, E.; Massa, P.; Giannelli, F.; Fernandes, F. A. N.; Ancona, L.; Stenseth, N. C.; Ferrer Obiol, J.; Paris, J.; Bertorelle, G.; Le Bohec, C.\nScore: 9.2, Published: 2023-08-08 DOI: 10.1101/2023.08.08.552445\nPurifying selection is the most pervasive type of selection, as it constantly removes deleterious mutations arising in populations, directly scaling with population size. Highly expressed genes appear to accumulate fewer deleterious mutations between divergent species lineages, pointing towards gene expression as an additional driver of purifying selection. However, estimates of the effect of gene expression on segregating deleterious variants in natural populations are lacking, as well as an understanding of the relative contribution of population size and gene expression to overall purifying selection pressure. Here, we analyse genomic and transcriptomic data from two natural populations of closely related sister species with different demographic histories, the Emperor (Aptenodytes forsteri) and the King penguins (A. patagonicus), and demonstrate that purifying selection at the population-level depends on the level of gene expression, with larger effects than population size. Deleterious segregating variants spread less in the population when they are in genes with higher expression rate. Leveraging realistic forward simulations, we estimate that the top 10% of the most highly expressed genes in a genome experience a selection pressure corresponding to an average selection coefficient of -0.1, which decreases to a selection coefficient of -0.01 for the top 50%. Gene expression appears to be a fundamental driver of purifying selection in natural populations, also effective at small population size. We suggest gene expression could be used as a proxy for gene selection coefficients (i.e., distribution of fitness effects), which are notoriously difficult to derive in non-model species under real-world conditions.\nWhole-genomes from the extinct Xerces Blue butterfly can help identify declining insect species\nAuthors: de-Dios, T.; Fontsere, C.; Renom, P.; Stiller, J.; Llovera, L.; Uliano-Silva, M.; Sanchez-Gracia, A.; Wright, C.; Lizano, E.; Navarro, A.; Civit, S.; Robbins, R. K.; Blaxter, M.; Marques-Bonet, T.; Vila, R.; Lalueza-Fox, C.\nScore: 111.5, Published: 2023-07-14 DOI: 10.1101/2021.11.08.467457\nThe Xerces Blue (Glaucopsyche xerces) is considered to be the first butterfly to become extinct at global scale in historical times. It was notable for its chalky lavender wings with conspicuous white spots on the ventral wings. The last individuals were collected in their restricted habitat, in the dunes near the Presidio military base in San Francisco, in 1941. We sequenced the genomes of four 80 to 100-year-old Xerces Blue, and seven historical and one modern specimens of its closest relative, the Silvery Blue (G. lygdamus). We compared these to a novel annotated genome of the Green-Underside Blue (G. alexis). Phylogenetic relationships inferred from complete mitochondrial genomes indicate that Xerces Blue was a distinct species that diverged from the Silvery Blue lineage at least 850,000 years ago. Using nuclear genomes, both species experienced population growth during the Eemian interglacial period, but the Xerces Blue decreased to a very low effective population size subsequently, a trend opposite to that observed in the Silvery Blue. Runs of homozygosity and deleterious load in the Xerces Blue were significantly greater than in the Silvery Blue, suggesting a higher incidence of inbreeding. These signals of population decline observed in Xerces Blue could be used to identify and monitor other insects threatened by human activities, whose extinction patterns are still not well known.\nFruitless mating with the exes: the irreversible parthenogenesis in a stick insect\nAuthors: Nozaki, T.; Chikami, Y.; Yano, K.; Sato, R.; Suetsugu, K.; Kaneko, S.\nScore: 206.9, Published: 2023-07-30 DOI: 10.1101/2023.07.28.550994\nParthenogenetic lineages, common in many animals, have sparked debate about their evolutionary persistence. Even after the loss of male individuals, males can survive on the genomes; in fact, rarely occurring males were described in many asexual lineages. Theoretical predictions suggest that genes related to male sexual traits in parthenogenetic lineages will decay over long evolutionary times free from the pressure of stabilizing selection. However, in many cases, such rare males are potentially functional. In this study, we investigated the functionality of rare males in the Japanese parthenogenetic stick insect, Ramulus mikado. Rarely occurring males in the species exhibited the external and anatomical morphology typical in stick insect males, and vigorously and successfully performed mating behaviors with conspecific females; nevertheless, no paternal genotypes were detected in the offspring; all embryos were genetically identical to their mothers. Our histological observation demonstrated that females received a spermatophore without sperms in their copulatory pouch, and revealed that in a male, spermatogenesis was completely deformed. These results suggest the irreversible clonal reproduction of R. mikado due to compromised male function. Our present study provides the first case of evident dysfunction in rare males of stick insects, illuminating the degenerative evolution of sexual traits in asexual lineages.\nTransitions in sex determination mechanisms through parental antagonism\nAuthors: Schenkel, M. A.\nScore: 4.5, Published: 2023-08-06 DOI: 10.1101/2023.08.04.551945\nParental antagonism (PA) occurs when the fitness effects of a gene depend on the parent from which it is inherited. Such genes may become enriched on sex chromosomes, due to their biased inheritance patterns. Although various sex determination (SD) genes exhibit parent-of-origin effects themselves, and between-parent conflict over offspring sex may affect SD, PA itself has not been considered as a driver of SD transitions. Here, I present a model to investigate the scope for transitions in SD mechanisms through PA. My model assumes an ancestral SD locus linked to a PA gene, as well as an autosomal PA gene in whose vicinity a novel SD gene arises. Transitions between functionally-homologous genes are found to depend on the fitness effects of both PA genes and their linkage to nearby SD genes. Transitions between male and female heterogamety by the invasion of a dominant SD gene are however nearly unconstrained. This also allows for back-and-forth dynamics where the ancestral SD and novel SD genes constantly evolve to be dominant over each other. These results further underline the malleability of SD mechanisms, and the need to consider parent-of-origin effects in driving transitions in SD, through proximate and/or ultimate means.\nDeclines in Neanderthal and Denisovan population sizes are probably not an artifact of subdivision\nAuthors: Rogers, A. R.\nScore: 8.9, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551046\nIn a subdivided population, effective population size (Ne) reflects not only the number of individuals, but also the rate and pattern of gene flow. If we sample two haploid genomes from the same deme, estimates of Ne will be small in the recent past but large in the distant past, even if there were no changes in census size or in the rate or pattern of gene flow. This article asks whether this effect can plausibly account for observed declines in effective sizes of archaic populations. Archaic human populations (Neanderthals and Denisovans) had very low heterozygosity. Under the hypothesis of equilibrium, heterozygosity alone allows us to estimate that each of these populations had a total effective size (including all subdivisions) of about 3600 individuals. Both archaic populations also exhibit a pronounced decline in Ne over a period of about 400 thousand years. Under the island model of population structure, these data imply an implausible level of separation among archaic demes. Each deme receives a migrant every 14 generations, and FST = 0.69. A different problem arises when one considers a larger number of demes distributed across a 2-dimensional landscape and exchanging mates only with nearest neighbors. Because of the constraint on global population size, the number of demes can be large only if each one is small. Small groups tend to be exogamous, so the rate of local gene flow must be high. With 144 demes, each of size 25, this model converged even faster than the island model did--too fast to generate the pattern of declining Ne described above. It seems unlikely that any model assuming equilibrium can be consistent with the pattern observed in archaic DNA. That pattern apparently reflects change either in the number of individuals or in the rate or pattern of gene flow.\nFacilitating taxonomy and phylogenetics: An informative and cost-effective protocol integrating long amplicon PCRs and third-generation sequencing\nAuthors: Gajski, D.; Wolff, J.; Melcher, A.; Weber, S.; Prost, S.; Krehenwinkel, H.; Kennedy, S.\nScore: 3.5, Published: 2023-08-06 DOI: 10.1101/2023.08.03.551825\nPhylogenetic inference has become a standard technique in integrative taxonomy and systematics, as well as in biogeography and ecology. DNA barcodes are often used for phylogenetic inference, despite being strongly limited due to their low number of informative sites. Also, because current DNA barcodes are based on a fraction of a single, fast-evolving gene, they are highly unsuitable for resolving deeper phylogenetic relationships due to saturation. In recent years, methods that analyse hundreds and thousands of loci at once have improved the resolution of the Tree of Life, but these methods require resources, experience and molecular laboratories that most taxonomists do not have. This paper introduces a PCR-based protocol that produces long amplicons of both slow- and fast-evolving unlinked mitochondrial and nuclear gene regions, which can be sequenced by the affordable and portable ONT MinION platform with low infrastructure or funding requirements. As a proof of concept, we inferred a phylogeny of a sample of 63 spider species from 20 families using our proposed protocol. The results were overall consistent with the results from approaches based on hundreds and thousands of loci, while requiring just a fraction of the cost and labour of such approaches, making our protocol accessible to taxonomists worldwide. HighlightsO_LIDNA barcoding is an invaluable tool for fast and accurate taxonomic classification C_LIO_LIExisting DNA barcodes are still insufficient for obtaining well-supported phylogenies C_LIO_LIWe present a protocol that produces long amplicons of unlinked loci for spiders C_LIO_LIAmplicons are sequenced at very low cost per specimen with ONT MinION C_LIO_LIOur recovered phylogeny is largely consistent with that of high-cost approaches. C_LI\n",
  "wordCount" : "2886",
  "inLanguage": "en",
  "datePublished": "2023-08-13T10:36:01Z",
  "dateModified": "2023-08-13T10:36:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on August 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551782">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551782" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551782">
        <p class="paperTitle">Functional Adaptations of Endogenous Retroviruses to the Drosophila Host Underlie their Evolutionary Diversification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551782" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551782" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Senti, K.-A.; Handler, D.; Rafanel, B.; Kosiol, C.; Schloetterer, C.; Brennecke, J.</p>
        <p class="info">Score: 52.7, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551782' target='https://doi.org/10.1101/2023.08.03.551782'> 10.1101/2023.08.03.551782</a></p>
        <p class="abstract">Transposable elements profoundly affect the biology and evolution of their hosts, yet their own evolutionary dynamics remain poorly understood. Here, we investigate insect endogenous retroviruses (iERVs), a monophyletic group of LTR retrotransposons that have acquired the trait of infectivity, likely through capture of a Baculovirus envelope gene. In Drosophila ovaries, iERVs with functional envelope have adapted their cis-regulatory sequences to be expressed in any somatic cell type, from where they infect the germline. Strikingly, related retroviruses show distinct expression patterns, indicating niche partitioning. In contrast, all non-infectious iERVs that emerged through secondary envelope-loss are specifically expressed in the germline. Co-evolving with iERVs, the genome-protecting piRNA pathway has assimilated iERV promoter and sequence information into piRNA clusters, underscoring the functional significance of iERV expression in somatic niches. We propose that the evolutionary innovation of cell-to-cell infectivity has triggered the adaptive radiation of iERVs through trait diversification and antagonistic virus-host interactions, processes that likely underpin niche-specific expression of endogenous retroviruses in vertebrates as well.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552427">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552427" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552427">
        <p class="paperTitle">An evolutionary timescale for Bacteria calibrated using the Great Oxidation Event</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552427" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552427" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davin, A. A.; Woodcroft, B. J.; Soo, R. M.; Murali, R.; Schrempf, D.; Clark, J.; Boussau, B.; Moody, E. R. R.; Szantho, L. L.; Richy, E.; Pisani, D.; Hemp, J.; Fischer, W. W.; Donoghue, P.; Spang, A.; Hugenholtz, P.; Williams, T. A.; SzoÌˆllosi, G. J.</p>
        <p class="info">Score: 23.9, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552427' target='https://doi.org/10.1101/2023.08.08.552427'> 10.1101/2023.08.08.552427</a></p>
        <p class="abstract">Most of life&#39;s diversity and history is microbial but it has left a meagre fossil record, greatly hindering understanding of evolution in deep time. However, the co-evolution of life and the Earth system has left signatures of bacterial metabolism in the geochemical record, most conspicuously the Great Oxidation Event (GOE) ~2.33 billion years ago (Ga, (Poulton et al. 2021)), in which oxygenic photosynthesis and tectonism (Eguchi, Seales, and Dasgupta 2019) transformed Earth&#39;s biosphere from dominantly anaerobic to aerobic. Here, we combine machine learning and phylogenetic reconciliation to infer ancestral transitions to aerobic lifestyles during bacterial evolution. Linking these transitions to the GOE provides new constraints to infer the timetree of Bacteria. We find that extant bacterial phyla are truly ancient, having radiated in the Archaean and the Proterozoic: the oldest include Bacillota (Firmicutes), which radiated 3.1-3.7 Ga, Cyanobacteria (3.3-3.5 Ga) and Patescibacteria (3-3.5 Ga). We show that most bacterial phyla were ancestrally anaerobic and that most transitions to an aerobic lifestyle post-dated the GOE. Our analyses trace oxygen production and consumption back to Cyanobacteria. From that starting point, horizontal transfer seeded aerobic lifestyles across bacterial diversity over hundreds of millions of years. Our analyses demonstrate that the diversification of aerobes proceeded in two waves corresponding to the GOE and to a second sustained rise in atmospheric O2 at the dawn of the Palezoic (Krause et al. 2022).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552622">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552622" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552622">
        <p class="paperTitle">Tetrapod vocal evolution: higher frequencies and faster rates of evolution in mammalian vocalizations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552622" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552622" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Munoz, M. I.; Marsot, M.; Ellers, J.; Halfwerk, W.</p>
        <p class="info">Score: 20.4, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552622' target='https://doi.org/10.1101/2023.08.09.552622'> 10.1101/2023.08.09.552622</a></p>
        <p class="abstract">Using the voice to produce sound is a widespread form of communication and plays an important role in contexts as diverse as parent-offspring interactions and courtship. Variation in the tempo and mode of vocal signal evolution has been studied in a phylogenetic context within orders or classes, but understanding vocal signal evolution ultimately requires comparison across all major lineages involved. Here we used comparative analyses to investigate the evolution of dominant frequency (i.e., the frequency with the highest energy content) and its association with body weight across 873 species of mammals, birds and frogs. In agreement with previous studies, we found that the negative allometric relationship between body weight and vocal frequency is a general feature of vocal systems. In addition, we found mammals to consistently vocalize at higher frequencies, and to evolve their vocalizations at around 6-fold faster rates than those of birds and frogs. Although all three groups strongly rely on vocal communication, our findings show that only mammals have extensively explored the spectral acoustic space available to them. We argue that such high vocal diversity of mammals is made possible by their unique hearing system, which evolved in the context of a small, parental-caring, nocturnal and insectivore ancestor, and has allowed them to detect, and therefore to evolve, a richer array of frequencies than other tetrapods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551413">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551413" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551413">
        <p class="paperTitle">Tissue-specific and cis-regulatory changes underlie parallel, adaptive gene expression evolution in house mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551413" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551413" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Durkin, S. M.; Ballinger, M. A.; Nachman, M. W.</p>
        <p class="info">Score: 14.4, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551413' target='https://doi.org/10.1101/2023.08.01.551413'> 10.1101/2023.08.01.551413</a></p>
        <p class="abstract">Changes in gene regulation have long been appreciated as a driving force of adaptive evolution, however the relative contributions of cis- and trans-acting changes to gene regulation over short evolutionary timescales remain unclear. Instances of recent, parallel phenotypic evolution provide an opportunity to assess whether parallel patterns are seen at the level of gene expression, and to assess the relative contribution of cis- and trans-changes to gene regulation in the early stages of divergence. Here, we studied gene expression in liver and brown adipose tissue in two wild-derived strains of house mice that independently adapted to cold, northern environments, and we compared them to a strain of house mice from a warm, southern environment. To investigate gene regulatory evolution, we studied expression in parents and allele-specific expression in F1 hybrids of crosses between warm-adapted and cold-adapted strains. First, we found that the different cold-adapted mice showed both unique and shared changes in expression, but that the proportion of shared changes (i.e. parallelism) was greater than expected by chance. Second, we discovered that expression evolution occurred largely at tissue-specific and cis-regulated genes, and that these genes were over-represented in parallel cases of evolution. Finally, we integrated the expression data with scans for selection in natural populations and found substantial parallelism in the two northern populations for genes under selection. Furthermore, selection outliers were associated with cis-regulated genes more than expected by chance; cis-regulated genes under selection influenced phenotypes such as body size, fat content, immune functioning, and skeletal morphology. These results demonstrate that parallel patterns of gene expression in mice that have independently adapted to cold environments are driven largely by tissue-specific and cis-regulatory changes, providing insight into the mechanisms of adaptive gene regulatory evolution at the earliest stages of divergence.

Author summaryThe parallel movement of organisms into novel environments provides an opportunity to understand the molecular basis of adaptation and the repeatability of this process. Mutations affecting the expression of genes are known to underlie much of adaptive evolution. Such mutations can arise in cis-(near the gene of interest) or in trans-(at a distant locus), but the relative contribution of these different kinds of changes to adaptation is poorly understood, especially during very recent divergence. Here, we compared evolved gene expression differences between a warm-adapted house mouse strain and two different cold-adapted strains that have independently evolved similar phenotypic traits, such as increased body size and decreased extremity length during the last few hundred years. Using crosses between warm-adapted and cold-adapted mice, we found that mutations acting in a context specific manner (cis-regulatory and tissue-specific changes) predominate expression divergence and are more likely to be involved in parallel evolution. We used population level selection scans in wild animals to identify regions of the genome under selection and combined these findings with the gene expression data to identify candidate genes underlying adaptation to novel environments. Together, our work describes the gene regulatory dynamics of rapid environmental adaptation, and the repeatability of these patterns over multiple instances of adaptation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552445">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552445" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552445">
        <p class="paperTitle">Gene expression is the main driver of purifying selection in large penguin populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552445" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552445" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Trucchi, E.; Massa, P.; Giannelli, F.; Fernandes, F. A. N.; Ancona, L.; Stenseth, N. C.; Ferrer Obiol, J.; Paris, J.; Bertorelle, G.; Le Bohec, C.</p>
        <p class="info">Score: 9.2, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552445' target='https://doi.org/10.1101/2023.08.08.552445'> 10.1101/2023.08.08.552445</a></p>
        <p class="abstract">Purifying selection is the most pervasive type of selection, as it constantly removes deleterious mutations arising in populations, directly scaling with population size. Highly expressed genes appear to accumulate fewer deleterious mutations between divergent species lineages, pointing towards gene expression as an additional driver of purifying selection. However, estimates of the effect of gene expression on segregating deleterious variants in natural populations are lacking, as well as an understanding of the relative contribution of population size and gene expression to overall purifying selection pressure. Here, we analyse genomic and transcriptomic data from two natural populations of closely related sister species with different demographic histories, the Emperor (Aptenodytes forsteri) and the King penguins (A. patagonicus), and demonstrate that purifying selection at the population-level depends on the level of gene expression, with larger effects than population size. Deleterious segregating variants spread less in the population when they are in genes with higher expression rate. Leveraging realistic forward simulations, we estimate that the top 10% of the most highly expressed genes in a genome experience a selection pressure corresponding to an average selection coefficient of -0.1, which decreases to a selection coefficient of -0.01 for the top 50%. Gene expression appears to be a fundamental driver of purifying selection in natural populations, also effective at small population size. We suggest gene expression could be used as a proxy for gene selection coefficients (i.e., distribution of fitness effects), which are notoriously difficult to derive in non-model species under real-world conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2021.11.08.467457">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2021.11.08.467457" aria-expanded="false" aria-controls="flush-collapse10.1101/2021.11.08.467457">
        <p class="paperTitle">Whole-genomes from the extinct Xerces Blue butterfly can help identify declining insect species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2021.11.08.467457" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2021.11.08.467457" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de-Dios, T.; Fontsere, C.; Renom, P.; Stiller, J.; Llovera, L.; Uliano-Silva, M.; Sanchez-Gracia, A.; Wright, C.; Lizano, E.; Navarro, A.; Civit, S.; Robbins, R. K.; Blaxter, M.; Marques-Bonet, T.; Vila, R.; Lalueza-Fox, C.</p>
        <p class="info">Score: 111.5, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2021.11.08.467457' target='https://doi.org/10.1101/2021.11.08.467457'> 10.1101/2021.11.08.467457</a></p>
        <p class="abstract">The Xerces Blue (Glaucopsyche xerces) is considered to be the first butterfly to become extinct at global scale in historical times. It was notable for its chalky lavender wings with conspicuous white spots on the ventral wings. The last individuals were collected in their restricted habitat, in the dunes near the Presidio military base in San Francisco, in 1941. We sequenced the genomes of four 80 to 100-year-old Xerces Blue, and seven historical and one modern specimens of its closest relative, the Silvery Blue (G. lygdamus). We compared these to a novel annotated genome of the Green-Underside Blue (G. alexis). Phylogenetic relationships inferred from complete mitochondrial genomes indicate that Xerces Blue was a distinct species that diverged from the Silvery Blue lineage at least 850,000 years ago. Using nuclear genomes, both species experienced population growth during the Eemian interglacial period, but the Xerces Blue decreased to a very low effective population size subsequently, a trend opposite to that observed in the Silvery Blue. Runs of homozygosity and deleterious load in the Xerces Blue were significantly greater than in the Silvery Blue, suggesting a higher incidence of inbreeding. These signals of population decline observed in Xerces Blue could be used to identify and monitor other insects threatened by human activities, whose extinction patterns are still not well known.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550994">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550994" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550994">
        <p class="paperTitle">Fruitless mating with the exes: the irreversible parthenogenesis in a stick insect</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550994" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550994" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nozaki, T.; Chikami, Y.; Yano, K.; Sato, R.; Suetsugu, K.; Kaneko, S.</p>
        <p class="info">Score: 206.9, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550994' target='https://doi.org/10.1101/2023.07.28.550994'> 10.1101/2023.07.28.550994</a></p>
        <p class="abstract">Parthenogenetic lineages, common in many animals, have sparked debate about their evolutionary persistence. Even after the loss of male individuals, males can survive on the genomes; in fact, rarely occurring males were described in many asexual lineages. Theoretical predictions suggest that genes related to male sexual traits in parthenogenetic lineages will decay over long evolutionary times free from the pressure of stabilizing selection. However, in many cases, such rare males are potentially functional. In this study, we investigated the functionality of rare males in the Japanese parthenogenetic stick insect, Ramulus mikado. Rarely occurring males in the species exhibited the external and anatomical morphology typical in stick insect males, and vigorously and successfully performed mating behaviors with conspecific females; nevertheless, no paternal genotypes were detected in the offspring; all embryos were genetically identical to their mothers. Our histological observation demonstrated that females received a spermatophore without sperms in their copulatory pouch, and revealed that in a male, spermatogenesis was completely deformed. These results suggest the irreversible clonal reproduction of R. mikado due to compromised male function. Our present study provides the first case of evident dysfunction in rare males of stick insects, illuminating the degenerative evolution of sexual traits in asexual lineages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.551945">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.551945" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.551945">
        <p class="paperTitle">Transitions in sex determination mechanisms through parental antagonism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.551945" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.551945" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schenkel, M. A.</p>
        <p class="info">Score: 4.5, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.551945' target='https://doi.org/10.1101/2023.08.04.551945'> 10.1101/2023.08.04.551945</a></p>
        <p class="abstract">Parental antagonism (PA) occurs when the fitness effects of a gene depend on the parent from which it is inherited. Such genes may become enriched on sex chromosomes, due to their biased inheritance patterns. Although various sex determination (SD) genes exhibit parent-of-origin effects themselves, and between-parent conflict over offspring sex may affect SD, PA itself has not been considered as a driver of SD transitions. Here, I present a model to investigate the scope for transitions in SD mechanisms through PA. My model assumes an ancestral SD locus linked to a PA gene, as well as an autosomal PA gene in whose vicinity a novel SD gene arises. Transitions between functionally-homologous genes are found to depend on the fitness effects of both PA genes and their linkage to nearby SD genes. Transitions between male and female heterogamety by the invasion of a dominant SD gene are however nearly unconstrained. This also allows for back-and-forth dynamics where the ancestral SD and novel SD genes constantly evolve to be dominant over each other. These results further underline the malleability of SD mechanisms, and the need to consider parent-of-origin effects in driving transitions in SD, through proximate and/or ultimate means.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.551046">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.551046" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.551046">
        <p class="paperTitle">Declines in Neanderthal and Denisovan population sizes are probably not an artifact of subdivision</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.551046" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.551046" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rogers, A. R.</p>
        <p class="info">Score: 8.9, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.551046' target='https://doi.org/10.1101/2023.07.28.551046'> 10.1101/2023.07.28.551046</a></p>
        <p class="abstract">In a subdivided population, effective population size (Ne) reflects not only the number of individuals, but also the rate and pattern of gene flow. If we sample two haploid genomes from the same deme, estimates of Ne will be small in the recent past but large in the distant past, even if there were no changes in census size or in the rate or pattern of gene flow. This article asks whether this effect can plausibly account for observed declines in effective sizes of archaic populations.

Archaic human populations (Neanderthals and Denisovans) had very low heterozygosity. Under the hypothesis of equilibrium, heterozygosity alone allows us to estimate that each of these populations had a total effective size (including all subdivisions) of about 3600 individuals.

Both archaic populations also exhibit a pronounced decline in Ne over a period of about 400 thousand years. Under the island model of population structure, these data imply an implausible level of separation among archaic demes. Each deme receives a migrant every 14 generations, and FST = 0.69. A different problem arises when one considers a larger number of demes distributed across a 2-dimensional landscape and exchanging mates only with nearest neighbors. Because of the constraint on global population size, the number of demes can be large only if each one is small. Small groups tend to be exogamous, so the rate of local gene flow must be high. With 144 demes, each of size 25, this model converged even faster than the island model did--too fast to generate the pattern of declining Ne described above.

It seems unlikely that any model assuming equilibrium can be consistent with the pattern observed in archaic DNA. That pattern apparently reflects change either in the number of individuals or in the rate or pattern of gene flow.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551825">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551825" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551825">
        <p class="paperTitle">Facilitating taxonomy and phylogenetics: An informative and cost-effective protocol integrating long amplicon PCRs and third-generation sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551825" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551825" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gajski, D.; Wolff, J.; Melcher, A.; Weber, S.; Prost, S.; Krehenwinkel, H.; Kennedy, S.</p>
        <p class="info">Score: 3.5, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551825' target='https://doi.org/10.1101/2023.08.03.551825'> 10.1101/2023.08.03.551825</a></p>
        <p class="abstract">Phylogenetic inference has become a standard technique in integrative taxonomy and systematics, as well as in biogeography and ecology. DNA barcodes are often used for phylogenetic inference, despite being strongly limited due to their low number of informative sites. Also, because current DNA barcodes are based on a fraction of a single, fast-evolving gene, they are highly unsuitable for resolving deeper phylogenetic relationships due to saturation. In recent years, methods that analyse hundreds and thousands of loci at once have improved the resolution of the Tree of Life, but these methods require resources, experience and molecular laboratories that most taxonomists do not have. This paper introduces a PCR-based protocol that produces long amplicons of both slow- and fast-evolving unlinked mitochondrial and nuclear gene regions, which can be sequenced by the affordable and portable ONT MinION platform with low infrastructure or funding requirements. As a proof of concept, we inferred a phylogeny of a sample of 63 spider species from 20 families using our proposed protocol. The results were overall consistent with the results from approaches based on hundreds and thousands of loci, while requiring just a fraction of the cost and labour of such approaches, making our protocol accessible to taxonomists worldwide.

HighlightsO_LIDNA barcoding is an invaluable tool for fast and accurate taxonomic classification
C_LIO_LIExisting DNA barcodes are still insufficient for obtaining well-supported phylogenies
C_LIO_LIWe present a protocol that produces long amplicons of unlinked loci for spiders
C_LIO_LIAmplicons are sequenced at very low cost per specimen with ONT MinION
C_LIO_LIOur recovered phylogeny is largely consistent with that of high-cost approaches.
C_LI</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
