<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Evolutionary change is remarkably constant across timescales
Authors: De Lisle, S.; Svensson, E. I.
Score: 28.2, Published: 2023-12-03 DOI: 10.1101/2023.12.02.569704
Apparent time-scaling of evolutionary rates presents an evolutionary dilemma; rates of molecular, phenotypic, and lineage diversification scale strongly negatively with timeframe of measurement, raising questions about the processes driving this general pattern. Here we show that much of this apparent temporal scaling is an inevitable outcome of plotting a ratio against its denominator.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Evolutionary change is remarkably constant across timescales
Authors: De Lisle, S.; Svensson, E. I.
Score: 28.2, Published: 2023-12-03 DOI: 10.1101/2023.12.02.569704
Apparent time-scaling of evolutionary rates presents an evolutionary dilemma; rates of molecular, phenotypic, and lineage diversification scale strongly negatively with timeframe of measurement, raising questions about the processes driving this general pattern. Here we show that much of this apparent temporal scaling is an inevitable outcome of plotting a ratio against its denominator." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-06T10:40:04+00:00" />
<meta property="article:modified_time" content="2023-12-06T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Evolutionary change is remarkably constant across timescales
Authors: De Lisle, S.; Svensson, E. I.
Score: 28.2, Published: 2023-12-03 DOI: 10.1101/2023.12.02.569704
Apparent time-scaling of evolutionary rates presents an evolutionary dilemma; rates of molecular, phenotypic, and lineage diversification scale strongly negatively with timeframe of measurement, raising questions about the processes driving this general pattern. Here we show that much of this apparent temporal scaling is an inevitable outcome of plotting a ratio against its denominator."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Evolutionary change is remarkably constant across timescales\nAuthors: De Lisle, S.; Svensson, E. I.\nScore: 28.2, Published: 2023-12-03 DOI: 10.1101/2023.12.02.569704\nApparent time-scaling of evolutionary rates presents an evolutionary dilemma; rates of molecular, phenotypic, and lineage diversification scale strongly negatively with timeframe of measurement, raising questions about the processes driving this general pattern. Here we show that much of this apparent temporal scaling is an inevitable outcome of plotting a ratio against its denominator.",
  "keywords": [
    
  ],
  "articleBody": " Evolutionary change is remarkably constant across timescales\nAuthors: De Lisle, S.; Svensson, E. I.\nScore: 28.2, Published: 2023-12-03 DOI: 10.1101/2023.12.02.569704\nApparent time-scaling of evolutionary rates presents an evolutionary dilemma; rates of molecular, phenotypic, and lineage diversification scale strongly negatively with timeframe of measurement, raising questions about the processes driving this general pattern. Here we show that much of this apparent temporal scaling is an inevitable outcome of plotting a ratio against its denominator. Highly unlikely relationships between timescale and amount of evolutionary change are required to produce anything other than artefactual negative relationships between rate and time. Unscaled magnitudes of evolution are largely unrelated to time, are easily explained by existing evolutionary models, and reframe the widely-held view of disparate pace of micro- and macroevolution. Time-independence of evolutionary change raises new questions about the factors that generate such temporal consistency in evolution.\nGene expression shifts in Emperor penguin adaptation to the extreme Antarctic environment\nAuthors: Paris, J. R.; Fernandes, F. A. N.; Pirri, F.; Greco, S.; Gerdol, M.; Pallavicini, A.; Benoiste, M.; Cornec, C.; Zane, L.; Haas, B.; Le Bohec, C.; Trucchi, E.\nScore: 14.0, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569211\nGene expression can accelerate ecological divergence by rapidly tweaking the response of an organism to novel environments, with more divergent environments exerting stronger selection and, supposedly, requiring faster adaptive responses. Organisms adapted to extreme environments provide ideal systems to test this hypothesis, particularly when compared to related species with milder ecological niches. The Emperor penguin (Aptenodytes forsteri) is the only warm-blooded vertebrate breeding in the harsh Antarctic winter, in stark contrast with the less cold-adapted sister species, the King penguin (A. patagonicus). Assembling the first de novo transcriptomes and analysing multi-tissue (brain, kidney, liver, muscle, skin) RNAseq data from natural populations of both species, we quantified the shifts in tissue-enhanced genes, co-expression gene networks, and differentially expressed genes characterising Emperor penguin adaptation to extreme Antarctic ecology. Our analyses revealed the crucial role played by muscle and liver in temperature homeostasis, fasting and whole-body energy metabolism (glucose/insulin regulation, lipid metabolism, fatty acid beta-oxidation, and blood coagulation). Repatterning at the regulatory level appears as more important in the brain of the Emperor penguin, showing the lowest signature of differential gene expression but the largest co-expression gene network shift. Nevertheless, over-expressed genes related to mTOR signalling in the brain and the liver support their central role in cold and fasting responses. Besides contributing to understanding the genetics underlying complex traits, like body energy reservoir management, our results provide a first insight into the role of gene expression in adaptation to one of the most extreme environmental conditions endured by an endotherm.\nExamining the role of oxygen-binding proteins on the early evolution of multicellularity\nAuthors: Wong, W.; Bravo, P.; Yunker, P. J.; Ratcliff, W. C.; Burnetti, A. J.\nScore: 13.4, Published: 2023-12-04 DOI: 10.1101/2023.12.01.569647\nOxygen availability is a key factor in the evolution of multicellularity, as larger and more sophisticated organisms often require mechanisms allowing efficient oxygen delivery to their tissues. One such mechanism is the presence of oxygen-binding proteins, such as globins and hemerythrins, which arose in the ancestor of bilaterian animals. Despite their importance, the precise mechanisms by which oxygen-binding proteins influenced the early stages of multicellular evolution under varying environmental oxygen levels are not yet clear. We addressed this knowledge gap by heterologously expressing the oxygen binding proteins myoglobin and myohemerythrin in snowflake yeast, a model system of simple, undifferentiated multicellularity. These proteins increased the depth and rate of oxygen diffusion, increasing the fitness of snowflake yeast growing aerobically. Experiments show that, paradoxically, oxygen-binding proteins confer a greater fitness benefit for larger organisms under high, not low, O2 conditions. We show via biophysical modeling that this is because facilitated diffusion is more efficient when oxygen is abundant, transporting a greater quantity of O2 which can be used for metabolism. By alleviating anatomical diffusion limitations to oxygen consumption, the evolution of O2-binding proteins in the oxygen-rich Neoproterozoic may have been a key breakthrough enabling the evolution of increasingly large, complex multicellular metazoan lineages.\nHot springs viruses at Yellowstone National Park have ancient origins and are adapted to their thermophilic hosts.\nAuthors: Benites, L. F.; Stephens, T. G.; Van Etten, J.; James, T.; Christian, W. C.; Barry, K.; Grigoriev, I. V.; McDermott, T. R.; Bhattacharya, D.\nScore: 17.8, Published: 2023-11-17 DOI: 10.1101/2023.11.15.567249\nGeothermal springs in areas such as Yellowstone National Park (YNP), USA house unicellular red algae that dominate the microbial biomass. Little is known about the viruses that infect the biota. Here we used metagenomics to characterize the multi-kingdom infecting virus community associated with red algal mats in three neighboring habitats (creek, endolithic, soil) at Lemonade Creek, YNP to determine their taxonomic composition, predicted gene functions, extent of horizontal gene transfer, and potential links to hosts. We find that despite proximity, each habitat houses a unique collection of viruses, with the giant viruses, Megaviricetes, dominant in all three. The early branching phylogenetic position of genes encoded on metagenome assembled virus genomes (vMAGs) suggests that the YNP lineages are of ancient origins and not the result of multiple invasions from mesophilic habitats. The existence of genomic footprints of adaptation to thermophily in the vMAGs is consistent with this idea. Our study is the first analysis of viruses associated with polyextremophilic red algae that form extensive microbial mats in YNP and are common worldwide at geothermal sites. Although more recent than the earliest forms of prokaryotic life, these eukaryotic algal mats originated ca. 1.5 Bya, and therefore span a significant period of the planets history. Our data are therefore relevant to understanding biotic interactions on the early Earth.\nSelective sweep probabilities in spatially expanding populations\nAuthors: Stein, A.; Kizhuttil, R.; Bak, M.; Noble, R. J.\nScore: 13.0, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568915\nEvolution during range expansions is an important feature of many biological systems including tumours, microbial communities, and invasive species. A selective sweep is a fundamental process, in which an advantageous mutation evades clonal interference and spreads through the population to fixation. However, most theoretical investigations of selective sweeps have assumed constant population size or have ignored spatial structure. Here we use mathematical modelling and analysis to investigate selective sweep probabilities in populations that grow with constant radial expansion speed. We derive probability distributions for the arrival time and location of the first surviving mutant and hence find simple approximate and exact expressions for selective sweep probabilities in one, two and three dimensions, which are independent of mutation rate. Namely, the selective sweep probability is approximately (1-cwt/cm)d, where cwt and cm are the wildtype and mutant radial expansion speeds, and d is the spatial dimension. Using agent-based simulations, we show that our analytical results accurately predict selective sweep frequencies in the two-dimensional spatial Moran process. We further compare our results with those obtained for alternative growth laws. Parameterizing our model for human tumours, we find that selective sweeps are predicted to be rare except during very early solid tumour growth, thus providing a general, pan-cancer explanation for findings from recent sequencing studies.\nEvolution of hybrid inviability associated with chromosome fusions\nAuthors: Boman, J.; Näsvall, K.; Vila, R.; Wiklund, C.; Backström, N.\nScore: 10.0, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569355\nChromosomal rearrangements, such as inversions, have received considerable attention in the speciation literature due to their hampering effects on recombination. However, less is known about how other rearrangements, such as chromosome fissions and fusions, can affect the evolution of reproductive isolation. Here, we used crosses between populations of the wood white butterfly (Leptidea sinapis) with different karyotypes to identify genomic regions associated with hybrid inviability. By contrasting allele frequencies between F2 hybrids that survived until adulthood with individuals of the same cohort that succumbed to hybrid incompatibilities, we show that candidate loci for hybrid inviability mainly are situated in fast-evolving regions with reduced recombination rates, especially in regions where chromosome fusions have occurred. Our results show that the extensive variation in chromosome numbers observed across the tree of life can be involved in speciation by being hotspots for the early evolution of postzygotic reproductive isolation.\nWhole-genome sequencing of major malaria vectors reveals the evolution of new insecticide resistance variants in a longitudinal study in Burkina Faso\nAuthors: Kientega, M.; Clarkson, C. S.; Traore, N.; Hui, T.-Y. J.; O'Loughlin, S. M.; Millogo, A.-A.; Yao, F. A.; Epopa, P. S.; Belem, A. M. G.; Jon, B. H.; Miles, A.; Burt, A.; Diabate, A.\nScore: 9.9, Published: 2023-11-25 DOI: 10.1101/2023.11.20.567800\nIntensive deployment of insecticide based malaria vector control tools results in the rapid evolution of phenotypes resistant to these chemicals. Understanding this process on the genomic level is essential for the deployment of successful interventions. Using whole genome sequencing data of 1409 individual An. Gambiae s.l. collected from 2012 to 2017, we investigated the change in genetic structure and the evolution of the insecticide resistance variants in natural populations over time and space. The results showed similar and constant nucleotide diversity and negative Tajimas D between An. gambiae s.s. and An. coluzzii. PCA and FST showed a clear genetic structure in the An. gambiae s.l. species. Genome-wide FST and H12 scans identified genomic regions under divergent selection and also having an implication in the adaptations to ecological changes. Novel voltage-gated sodium channel pyrethroid resistance target-site alleles (V402L, I1527T) were identified at increasing frequencies alongside the established kdr alleles (Vgsc-L995F, Vgsc-L995S and N1570Y) within the An. gambiae s.l. populations. Organophosphate metabolic resistance markers were also identified, at increasing frequencies, within the An. gambiae s.s. populations from 2012 to 2017, including the SNP Ace1-G280S and its associated duplication. Variants simultaneously identified in the same vector populations raise concerns about the long-term efficacy of new-generation bednets and the recently introduced organophosphate pirimiphos-methyl indoor residual spraying. These findings highlighted the benefit of genomic malaria vector surveillance for the detection of new insecticide resistance variants, the monitoring of the existing resistance variants, and also to get insights into the evolutionary processes driving insecticide resistance. Author SummaryGenomic surveillance of malaria vectors is crucial for understanding the genetic variation in natural vector populations and also guiding the implementation of novel and innovative vector control tools. Application of sequencing technologies in vector studies provide insights on the genetic and evolutionary phenomena of vectors that could have impact on vector control strategies. By analyzing the whole genome data of 1409 wild An. gambiae s.l. mosquito collected between 2012 and 2017, we showed an emergence of novel insecticide resistance markers alongside increasing frequencies of existing insecticide resistance variants over time in Burkina Faso. We showed the benefit of genomic surveillance of malaria vectors for the monitoring of the insecticide resistance variants and also providing insights into the evolutionary processes driving insecticide resistance.\nAllelic gene conversion softens selective sweeps\nAuthors: Schrider, D. R.\nScore: 7.2, Published: 2023-12-05 DOI: 10.1101/2023.12.05.570141\nThe prominence of positive selection, in which beneficial mutations are favored by natural selection and rapidly increase in frequency, is a subject of intense debate. Positive selection can result in selective sweeps, in which the haplotype(s) bearing the adaptive allele \"sweep\" through the population, thereby removing much of the genetic diversity from the region surrounding the target of selection. Two models of selective sweeps have been proposed: classical sweeps, or \"hard sweeps\", in which a single copy of the adaptive allele sweeps to fixation, and \"soft sweeps\", in which multiple distinct copies of the adaptive allele leave descendants after the sweep. Soft sweeps can be the outcome of recurrent mutation to the adaptive allele, or the presence of standing genetic variation consisting of multiple copies of the adaptive allele prior to the onset of selection. Importantly, soft sweeps will be common when populations can rapidly adapt to novel selective pressures, either because of a high mutation rate or because adaptive alleles are already present. The prevalence of soft sweeps is especially controversial, and it has been noted that selection on standing variation or recurrent mutations may not always produce soft sweeps. Here, we show that the inverse is true: selection on single-origin de novo mutations may often result in an outcome that is indistinguishable from a soft sweep. This is made possible by allelic gene conversion, which \"softens\" hard sweeps by copying the adaptive allele onto multiple genetic backgrounds, a process we refer to as a \"pseudo-soft\" sweep. We carried out a simulation study examining the impact of gene conversion on sweeps from a single de novo variant in models of human, Drosophila, and Arabidopsis populations. The fraction of simulations in which gene conversion had produced multiple haplotypes with the adaptive allele upon fixation was appreciable. Indeed, under realistic demographic histories and gene conversion rates, even if selection always acts on a single-origin mutation, sweeps involving multiple haplotypes are more likely than hard sweeps in large populations, especially when selection is not extremely strong. Thus, even when the mutation rate is low or there is no standing variation, hard sweeps are expected to be the exception rather than the rule in large populations. These results also imply that the presence of signatures of soft sweeps does not necessarily mean that adaptation has been especially rapid or is not mutation limited.\nPatterns of physical activity in hunter-gatherer children compared with US and UK children.\nAuthors: Kretschmer, L.; Dyble, M.; Chaudhary, N.; Bann, D.; Salali, G. D.\nScore: 6.9, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569171\nSedentary lifestyles, mismatched with our active foraging history, contribute to escalating rates of non-communicable diseases. Contemporary hunter-gatherers appear to be highly active, but little is known about physical activity levels in hunter-gatherer children. We analysed 150 days of accelerometer data from 51 BaYaka hunter-gatherer children (aged 3-18) in the Republic of Congo, comparing it with British and American children (MCS and NHANES). BaYaka children were highly active, engaging in over 3 hours of moderate-to-vigorous physical activity (MVPA) daily, surpassing British adolescents by over 70 minutes. In US children activity declined with age; while in BaYaka children activity increased with age, unaffected by gender. Reflecting their foraging lifestyle, activity patterns varied within and between days, yet all children consistently rose with the sun. These findings highlight the impact of a foraging upbringing on childrens activity levels, providing a benchmark for understanding childhood physical activity and wellbeing.\nSelection shapes the genomic landscape of introgressed ancestry in a pair of sympatric sea urchin species\nAuthors: Glasenapp, M.; Pogson, G.\nScore: 4.3, Published: 2023-12-03 DOI: 10.1101/2023.12.01.566927\nA growing number of recent studies have demonstrated that introgression is common across the tree of life. However, we still have a limited understanding of the fate and fitness consequence of introgressed variation at the whole-genome scale across diverse taxonomic groups. Here, we implemented a phylogenetic hidden Markov model to identify and characterize introgressed genomic regions in a pair of well-diverged, non-sister sea urchin species: Strongylocentrotus pallidus and S. droebachiensis. Despite the old age of introgression, a sizable fraction of the genome (1% - 5%) exhibited introgressed ancestry, including numerous genes showing signals of historical positive selection that may represent cases of adaptive introgression. There were two chromosomes with considerably reduced introgression and a negative correlation between introgression and gene density, consistent with selection against introgression. Testing for nonrandom patterns among introgressed regions relative to the non-introgressed genome-wide background revealed that introgressed regions had significantly reduced nucleotide divergence (dXY) and overlapped fewer protein-coding genes, coding bases, and genes with a history of positive selection. Additionally, genes residing within introgressed regions showed slower rates of evolution (dN, dS, dN/dS) than random distributions of genes without introgressed ancestry. Overall, our findings are consistent with widespread selection against introgressed ancestry across the genome and suggest that slowly evolving, low-divergence genomic regions are more likely to move between species and avoid negative selection following hybridization and introgression.\n",
  "wordCount" : "2603",
  "inLanguage": "en",
  "datePublished": "2023-12-06T10:40:04Z",
  "dateModified": "2023-12-06T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 6, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.02.569704">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.02.569704" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.02.569704">
        <p class="paperTitle">Evolutionary change is remarkably constant across timescales</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.02.569704" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.02.569704" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: De Lisle, S.; Svensson, E. I.</p>
        <p class="info">Score: 28.2, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.02.569704' target='https://doi.org/10.1101/2023.12.02.569704'> 10.1101/2023.12.02.569704</a></p>
        <p class="abstract">Apparent time-scaling of evolutionary rates presents an evolutionary dilemma; rates of molecular, phenotypic, and lineage diversification scale strongly negatively with timeframe of measurement, raising questions about the processes driving this general pattern. Here we show that much of this apparent temporal scaling is an inevitable outcome of plotting a ratio against its denominator. Highly unlikely relationships between timescale and amount of evolutionary change are required to produce anything other than artefactual negative relationships between rate and time. Unscaled magnitudes of evolution are largely unrelated to time, are easily explained by existing evolutionary models, and reframe the widely-held view of disparate pace of micro- and macroevolution. Time-independence of evolutionary change raises new questions about the factors that generate such temporal consistency in evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569211">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569211" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569211">
        <p class="paperTitle">Gene expression shifts in Emperor penguin adaptation to the extreme Antarctic environment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569211" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569211" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Paris, J. R.; Fernandes, F. A. N.; Pirri, F.; Greco, S.; Gerdol, M.; Pallavicini, A.; Benoiste, M.; Cornec, C.; Zane, L.; Haas, B.; Le Bohec, C.; Trucchi, E.</p>
        <p class="info">Score: 14.0, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569211' target='https://doi.org/10.1101/2023.11.29.569211'> 10.1101/2023.11.29.569211</a></p>
        <p class="abstract">Gene expression can accelerate ecological divergence by rapidly tweaking the response of an organism to novel environments, with more divergent environments exerting stronger selection and, supposedly, requiring faster adaptive responses. Organisms adapted to extreme environments provide ideal systems to test this hypothesis, particularly when compared to related species with milder ecological niches. The Emperor penguin (Aptenodytes forsteri) is the only warm-blooded vertebrate breeding in the harsh Antarctic winter, in stark contrast with the less cold-adapted sister species, the King penguin (A. patagonicus). Assembling the first de novo transcriptomes and analysing multi-tissue (brain, kidney, liver, muscle, skin) RNAseq data from natural populations of both species, we quantified the shifts in tissue-enhanced genes, co-expression gene networks, and differentially expressed genes characterising Emperor penguin adaptation to extreme Antarctic ecology. Our analyses revealed the crucial role played by muscle and liver in temperature homeostasis, fasting and whole-body energy metabolism (glucose/insulin regulation, lipid metabolism, fatty acid beta-oxidation, and blood coagulation). Repatterning at the regulatory level appears as more important in the brain of the Emperor penguin, showing the lowest signature of differential gene expression but the largest co-expression gene network shift. Nevertheless, over-expressed genes related to mTOR signalling in the brain and the liver support their central role in cold and fasting responses. Besides contributing to understanding the genetics underlying complex traits, like body energy reservoir management, our results provide a first insight into the role of gene expression in adaptation to one of the most extreme environmental conditions endured by an endotherm.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569647">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569647" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569647">
        <p class="paperTitle">Examining the role of oxygen-binding proteins on the early evolution of multicellularity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569647" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569647" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wong, W.; Bravo, P.; Yunker, P. J.; Ratcliff, W. C.; Burnetti, A. J.</p>
        <p class="info">Score: 13.4, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569647' target='https://doi.org/10.1101/2023.12.01.569647'> 10.1101/2023.12.01.569647</a></p>
        <p class="abstract">Oxygen availability is a key factor in the evolution of multicellularity, as larger and more sophisticated organisms often require mechanisms allowing efficient oxygen delivery to their tissues. One such mechanism is the presence of oxygen-binding proteins, such as globins and hemerythrins, which arose in the ancestor of bilaterian animals. Despite their importance, the precise mechanisms by which oxygen-binding proteins influenced the early stages of multicellular evolution under varying environmental oxygen levels are not yet clear. We addressed this knowledge gap by heterologously expressing the oxygen binding proteins myoglobin and myohemerythrin in snowflake yeast, a model system of simple, undifferentiated multicellularity. These proteins increased the depth and rate of oxygen diffusion, increasing the fitness of snowflake yeast growing aerobically. Experiments show that, paradoxically, oxygen-binding proteins confer a greater fitness benefit for larger organisms under high, not low, O2 conditions. We show via biophysical modeling that this is because facilitated diffusion is more efficient when oxygen is abundant, transporting a greater quantity of O2 which can be used for metabolism. By alleviating anatomical diffusion limitations to oxygen consumption, the evolution of O2-binding proteins in the oxygen-rich Neoproterozoic may have been a key breakthrough enabling the evolution of increasingly large, complex multicellular metazoan lineages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567249">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567249" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567249">
        <p class="paperTitle">Hot springs viruses at Yellowstone National Park have ancient origins and are adapted to their thermophilic hosts.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567249" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567249" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benites, L. F.; Stephens, T. G.; Van Etten, J.; James, T.; Christian, W. C.; Barry, K.; Grigoriev, I. V.; McDermott, T. R.; Bhattacharya, D.</p>
        <p class="info">Score: 17.8, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567249' target='https://doi.org/10.1101/2023.11.15.567249'> 10.1101/2023.11.15.567249</a></p>
        <p class="abstract">Geothermal springs in areas such as Yellowstone National Park (YNP), USA house unicellular red algae that dominate the microbial biomass. Little is known about the viruses that infect the biota. Here we used metagenomics to characterize the multi-kingdom infecting virus community associated with red algal mats in three neighboring habitats (creek, endolithic, soil) at Lemonade Creek, YNP to determine their taxonomic composition, predicted gene functions, extent of horizontal gene transfer, and potential links to hosts. We find that despite proximity, each habitat houses a unique collection of viruses, with the giant viruses, Megaviricetes, dominant in all three. The early branching phylogenetic position of genes encoded on metagenome assembled virus genomes (vMAGs) suggests that the YNP lineages are of ancient origins and not the result of multiple invasions from mesophilic habitats. The existence of genomic footprints of adaptation to thermophily in the vMAGs is consistent with this idea. Our study is the first analysis of viruses associated with polyextremophilic red algae that form extensive microbial mats in YNP and are common worldwide at geothermal sites. Although more recent than the earliest forms of prokaryotic life, these eukaryotic algal mats originated ca. 1.5 Bya, and therefore span a significant period of the planets history. Our data are therefore relevant to understanding biotic interactions on the early Earth.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568915">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568915" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568915">
        <p class="paperTitle">Selective sweep probabilities in spatially expanding populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568915" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568915" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stein, A.; Kizhuttil, R.; Bak, M.; Noble, R. J.</p>
        <p class="info">Score: 13.0, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568915' target='https://doi.org/10.1101/2023.11.27.568915'> 10.1101/2023.11.27.568915</a></p>
        <p class="abstract">Evolution during range expansions is an important feature of many biological systems including tumours, microbial communities, and invasive species. A selective sweep is a fundamental process, in which an advantageous mutation evades clonal interference and spreads through the population to fixation. However, most theoretical investigations of selective sweeps have assumed constant population size or have ignored spatial structure. Here we use mathematical modelling and analysis to investigate selective sweep probabilities in populations that grow with constant radial expansion speed. We derive probability distributions for the arrival time and location of the first surviving mutant and hence find simple approximate and exact expressions for selective sweep probabilities in one, two and three dimensions, which are independent of mutation rate. Namely, the selective sweep probability is approximately (1-cwt/cm)d, where cwt and cm are the wildtype and mutant radial expansion speeds, and d is the spatial dimension. Using agent-based simulations, we show that our analytical results accurately predict selective sweep frequencies in the two-dimensional spatial Moran process. We further compare our results with those obtained for alternative growth laws. Parameterizing our model for human tumours, we find that selective sweeps are predicted to be rare except during very early solid tumour growth, thus providing a general, pan-cancer explanation for findings from recent sequencing studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569355">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569355" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569355">
        <p class="paperTitle">Evolution of hybrid inviability associated with chromosome fusions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569355" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569355" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boman, J.; Näsvall, K.; Vila, R.; Wiklund, C.; Backström, N.</p>
        <p class="info">Score: 10.0, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569355' target='https://doi.org/10.1101/2023.11.30.569355'> 10.1101/2023.11.30.569355</a></p>
        <p class="abstract">Chromosomal rearrangements, such as inversions, have received considerable attention in the speciation literature due to their hampering effects on recombination. However, less is known about how other rearrangements, such as chromosome fissions and fusions, can affect the evolution of reproductive isolation. Here, we used crosses between populations of the wood white butterfly (Leptidea sinapis) with different karyotypes to identify genomic regions associated with hybrid inviability. By contrasting allele frequencies between F2 hybrids that survived until adulthood with individuals of the same cohort that succumbed to hybrid incompatibilities, we show that candidate loci for hybrid inviability mainly are situated in fast-evolving regions with reduced recombination rates, especially in regions where chromosome fusions have occurred. Our results show that the extensive variation in chromosome numbers observed across the tree of life can be involved in speciation by being hotspots for the early evolution of postzygotic reproductive isolation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567800">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567800" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567800">
        <p class="paperTitle">Whole-genome sequencing of major malaria vectors reveals the evolution of new insecticide resistance variants in a longitudinal study in Burkina Faso</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567800" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567800" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kientega, M.; Clarkson, C. S.; Traore, N.; Hui, T.-Y. J.; O&#39;Loughlin, S. M.; Millogo, A.-A.; Yao, F. A.; Epopa, P. S.; Belem, A. M. G.; Jon, B. H.; Miles, A.; Burt, A.; Diabate, A.</p>
        <p class="info">Score: 9.9, Published: 2023-11-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567800' target='https://doi.org/10.1101/2023.11.20.567800'> 10.1101/2023.11.20.567800</a></p>
        <p class="abstract">Intensive deployment of insecticide based malaria vector control tools results in the rapid evolution of phenotypes resistant to these chemicals. Understanding this process on the genomic level is essential for the deployment of successful interventions. Using whole genome sequencing data of 1409 individual An. Gambiae s.l. collected from 2012 to 2017, we investigated the change in genetic structure and the evolution of the insecticide resistance variants in natural populations over time and space. The results showed similar and constant nucleotide diversity and negative Tajimas D between An. gambiae s.s. and An. coluzzii. PCA and FST showed a clear genetic structure in the An. gambiae s.l. species. Genome-wide FST and H12 scans identified genomic regions under divergent selection and also having an implication in the adaptations to ecological changes. Novel voltage-gated sodium channel pyrethroid resistance target-site alleles (V402L, I1527T) were identified at increasing frequencies alongside the established kdr alleles (Vgsc-L995F, Vgsc-L995S and N1570Y) within the An. gambiae s.l. populations. Organophosphate metabolic resistance markers were also identified, at increasing frequencies, within the An. gambiae s.s. populations from 2012 to 2017, including the SNP Ace1-G280S and its associated duplication. Variants simultaneously identified in the same vector populations raise concerns about the long-term efficacy of new-generation bednets and the recently introduced organophosphate pirimiphos-methyl indoor residual spraying. These findings highlighted the benefit of genomic malaria vector surveillance for the detection of new insecticide resistance variants, the monitoring of the existing resistance variants, and also to get insights into the evolutionary processes driving insecticide resistance.

Author SummaryGenomic surveillance of malaria vectors is crucial for understanding the genetic variation in natural vector populations and also guiding the implementation of novel and innovative vector control tools. Application of sequencing technologies in vector studies provide insights on the genetic and evolutionary phenomena of vectors that could have impact on vector control strategies. By analyzing the whole genome data of 1409 wild An. gambiae s.l. mosquito collected between 2012 and 2017, we showed an emergence of novel insecticide resistance markers alongside increasing frequencies of existing insecticide resistance variants over time in Burkina Faso. We showed the benefit of genomic surveillance of malaria vectors for the monitoring of the insecticide resistance variants and also providing insights into the evolutionary processes driving insecticide resistance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570141">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570141" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570141">
        <p class="paperTitle">Allelic gene conversion softens selective sweeps</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570141" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570141" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schrider, D. R.</p>
        <p class="info">Score: 7.2, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570141' target='https://doi.org/10.1101/2023.12.05.570141'> 10.1101/2023.12.05.570141</a></p>
        <p class="abstract">The prominence of positive selection, in which beneficial mutations are favored by natural selection and rapidly increase in frequency, is a subject of intense debate. Positive selection can result in selective sweeps, in which the haplotype(s) bearing the adaptive allele &#34;sweep&#34; through the population, thereby removing much of the genetic diversity from the region surrounding the target of selection. Two models of selective sweeps have been proposed: classical sweeps, or &#34;hard sweeps&#34;, in which a single copy of the adaptive allele sweeps to fixation, and &#34;soft sweeps&#34;, in which multiple distinct copies of the adaptive allele leave descendants after the sweep. Soft sweeps can be the outcome of recurrent mutation to the adaptive allele, or the presence of standing genetic variation consisting of multiple copies of the adaptive allele prior to the onset of selection. Importantly, soft sweeps will be common when populations can rapidly adapt to novel selective pressures, either because of a high mutation rate or because adaptive alleles are already present. The prevalence of soft sweeps is especially controversial, and it has been noted that selection on standing variation or recurrent mutations may not always produce soft sweeps. Here, we show that the inverse is true: selection on single-origin de novo mutations may often result in an outcome that is indistinguishable from a soft sweep. This is made possible by allelic gene conversion, which &#34;softens&#34; hard sweeps by copying the adaptive allele onto multiple genetic backgrounds, a process we refer to as a &#34;pseudo-soft&#34; sweep. We carried out a simulation study examining the impact of gene conversion on sweeps from a single de novo variant in models of human, Drosophila, and Arabidopsis populations. The fraction of simulations in which gene conversion had produced multiple haplotypes with the adaptive allele upon fixation was appreciable. Indeed, under realistic demographic histories and gene conversion rates, even if selection always acts on a single-origin mutation, sweeps involving multiple haplotypes are more likely than hard sweeps in large populations, especially when selection is not extremely strong. Thus, even when the mutation rate is low or there is no standing variation, hard sweeps are expected to be the exception rather than the rule in large populations. These results also imply that the presence of signatures of soft sweeps does not necessarily mean that adaptation has been especially rapid or is not mutation limited.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569171">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569171" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569171">
        <p class="paperTitle">Patterns of physical activity in hunter-gatherer children compared with US and UK children.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569171" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569171" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kretschmer, L.; Dyble, M.; Chaudhary, N.; Bann, D.; Salali, G. D.</p>
        <p class="info">Score: 6.9, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569171' target='https://doi.org/10.1101/2023.11.29.569171'> 10.1101/2023.11.29.569171</a></p>
        <p class="abstract">Sedentary lifestyles, mismatched with our active foraging history, contribute to escalating rates of non-communicable diseases. Contemporary hunter-gatherers appear to be highly active, but little is known about physical activity levels in hunter-gatherer children. We analysed 150 days of accelerometer data from 51 BaYaka hunter-gatherer children (aged 3-18) in the Republic of Congo, comparing it with British and American children (MCS and NHANES). BaYaka children were highly active, engaging in over 3 hours of moderate-to-vigorous physical activity (MVPA) daily, surpassing British adolescents by over 70 minutes. In US children activity declined with age; while in BaYaka children activity increased with age, unaffected by gender. Reflecting their foraging lifestyle, activity patterns varied within and between days, yet all children consistently rose with the sun. These findings highlight the impact of a foraging upbringing on childrens activity levels, providing a benchmark for understanding childhood physical activity and wellbeing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.566927">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.566927" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.566927">
        <p class="paperTitle">Selection shapes the genomic landscape of introgressed ancestry in a pair of sympatric sea urchin species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.566927" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.566927" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Glasenapp, M.; Pogson, G.</p>
        <p class="info">Score: 4.3, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.566927' target='https://doi.org/10.1101/2023.12.01.566927'> 10.1101/2023.12.01.566927</a></p>
        <p class="abstract">A growing number of recent studies have demonstrated that introgression is common across the tree of life. However, we still have a limited understanding of the fate and fitness consequence of introgressed variation at the whole-genome scale across diverse taxonomic groups. Here, we implemented a phylogenetic hidden Markov model to identify and characterize introgressed genomic regions in a pair of well-diverged, non-sister sea urchin species: Strongylocentrotus pallidus and S. droebachiensis. Despite the old age of introgression, a sizable fraction of the genome (1% - 5%) exhibited introgressed ancestry, including numerous genes showing signals of historical positive selection that may represent cases of adaptive introgression. There were two chromosomes with considerably reduced introgression and a negative correlation between introgression and gene density, consistent with selection against introgression. Testing for nonrandom patterns among introgressed regions relative to the non-introgressed genome-wide background revealed that introgressed regions had significantly reduced nucleotide divergence (dXY) and overlapped fewer protein-coding genes, coding bases, and genes with a history of positive selection. Additionally, genes residing within introgressed regions showed slower rates of evolution (dN, dS, dN/dS) than random distributions of genes without introgressed ancestry. Overall, our findings are consistent with widespread selection against introgressed ancestry across the genome and suggest that slowly evolving, low-divergence genomic regions are more likely to move between species and avoid negative selection following hybridization and introgression.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
