<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Opsin gene duplication in Lepidoptera: retrotransposition, sex linkage, and gene expression
Authors: Mulhair, P. O.; Crowley, L.; Boyes, D. H.; Lewis, O. T.; Holland, P. W. H.
Score: 35.3, Published: 2023-08-16 DOI: 10.1101/2023.08.16.552946
Colour vision in insects is determined by signalling cascades, central to which are opsin proteins, resulting in sensitivity to light at different wavelengths. In certain insect groups, lineage specific evolution of opsin genes, in terms of copy number, shifts in expression patterns, and functional amino acid substitutions, has resulted in changes in colour vision with subsequent behavioural and niche adaptations.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Opsin gene duplication in Lepidoptera: retrotransposition, sex linkage, and gene expression
Authors: Mulhair, P. O.; Crowley, L.; Boyes, D. H.; Lewis, O. T.; Holland, P. W. H.
Score: 35.3, Published: 2023-08-16 DOI: 10.1101/2023.08.16.552946
Colour vision in insects is determined by signalling cascades, central to which are opsin proteins, resulting in sensitivity to light at different wavelengths. In certain insect groups, lineage specific evolution of opsin genes, in terms of copy number, shifts in expression patterns, and functional amino acid substitutions, has resulted in changes in colour vision with subsequent behavioural and niche adaptations." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-23T10:37:56+00:00" />
<meta property="article:modified_time" content="2023-08-23T10:37:56+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Opsin gene duplication in Lepidoptera: retrotransposition, sex linkage, and gene expression
Authors: Mulhair, P. O.; Crowley, L.; Boyes, D. H.; Lewis, O. T.; Holland, P. W. H.
Score: 35.3, Published: 2023-08-16 DOI: 10.1101/2023.08.16.552946
Colour vision in insects is determined by signalling cascades, central to which are opsin proteins, resulting in sensitivity to light at different wavelengths. In certain insect groups, lineage specific evolution of opsin genes, in terms of copy number, shifts in expression patterns, and functional amino acid substitutions, has resulted in changes in colour vision with subsequent behavioural and niche adaptations."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Opsin gene duplication in Lepidoptera: retrotransposition, sex linkage, and gene expression\nAuthors: Mulhair, P. O.; Crowley, L.; Boyes, D. H.; Lewis, O. T.; Holland, P. W. H.\nScore: 35.3, Published: 2023-08-16 DOI: 10.1101/2023.08.16.552946\nColour vision in insects is determined by signalling cascades, central to which are opsin proteins, resulting in sensitivity to light at different wavelengths. In certain insect groups, lineage specific evolution of opsin genes, in terms of copy number, shifts in expression patterns, and functional amino acid substitutions, has resulted in changes in colour vision with subsequent behavioural and niche adaptations.",
  "keywords": [
    
  ],
  "articleBody": " Opsin gene duplication in Lepidoptera: retrotransposition, sex linkage, and gene expression\nAuthors: Mulhair, P. O.; Crowley, L.; Boyes, D. H.; Lewis, O. T.; Holland, P. W. H.\nScore: 35.3, Published: 2023-08-16 DOI: 10.1101/2023.08.16.552946\nColour vision in insects is determined by signalling cascades, central to which are opsin proteins, resulting in sensitivity to light at different wavelengths. In certain insect groups, lineage specific evolution of opsin genes, in terms of copy number, shifts in expression patterns, and functional amino acid substitutions, has resulted in changes in colour vision with subsequent behavioural and niche adaptations. Lepidoptera are a fascinating model to address whether evolutionary change in opsin content and sequence evolution are associated with changes in vision phenotype. Until recently, the lack of high quality genome data representing broad sampling across the lepidopteran phylogeny has greatly limited our ability to accurately address this question. Here, we annotate opsin genes in 219 lepidopteran genomes representing 33 families, reconstruct their evolutionary history, and analyse shifts in selective pressures and expression between genes and species. We discover 44 duplication events in opsin genes across [~]300 million years of lepidopteran evolution. While many duplication events are species or family specific, we find retention of an ancient long-wavelength sensitive (LW) opsin duplication derived by retrotransposition within the speciose superfamily Noctuoidea (in the families Nolidae, Erebidae, and Noctuidae). This conserved LW retrogene shows life stage specific expression suggesting visual sensitivities or other sensory functions specific to the early larval stage. This study provides a comprehensive order-wide view of opsin evolution across Lepidoptera, showcasing high rates of opsin duplications and changes in expression patterns.\nEvidence for ecological tuning of novel anuran biofluorescent signals\nAuthors: Whitcher, C.; Ron, S. R.; Ayala-Varela, F.; Crawford, A.; Herrera-Alva, V.; Castillo-Urbina, E.; Grazziotin, F.; Bowman, R. M.; Lemmon, A. R.; Moriarty Lemmon, E.\nScore: 37.5, Published: 2023-07-28 DOI: 10.1101/2023.07.25.550432\nOur study assesses the variability of amphibian biofluorescence and provides insight into its potential functions and role in anuran evolution. Via a field survey across South America, we discovered and documented patterns of biofluorescence in tropical amphibians. We more than tripled the number of species that have been tested for this trait and added representatives from previously untested anuran families. We found evidence for ecological tuning (i.e., the specific adaptation of a signal to the environment in which it is received) of the novel anuran biofluorescent signals. Across groups, the fluorescence excitation peak matches the wavelengths most available at twilight, the light environment in which most frog species are active. Additionally, biofluorescence emission spans both wavelengths of low availability in twilight and the peak sensitivity of green-sensitive rods in the anuran eye, likely increasing contrast of this signal for a conspecific receiver. With evidence of tuning to the ecology and sensory systems of frogs, our results suggest frog biofluorescence is likely functioning in anuran communication.\nSox21b underlies the rapid diversification of a novel male genital structure between Drosophila species\nAuthors: Ridgway, A. M.; Hood, E.; Jimenez, J. F.; Nunes, M. D. S.; McGregor, A. P.\nScore: 21.5, Published: 2023-08-18 DOI: 10.1101/2023.08.17.552955\nThe emergence and subsequent diversification of morphological novelties is a major feature of animal evolution1-9. However, in most cases little is known about the molecular basis of the evolution of novel structures and the genetic mechanisms underlying their diversification. The epandrial posterior lobes of the male genital arch is a novelty of some species of the Drosophila melanogaster subgroup10-13. The posterior lobes grasp the ovipositor of the female and then integrate between her abdominal tergites, and therefore these structures are important for copulation and species-recognition10-12,14-17. The posterior lobes evolved from co-option of a Hox regulated gene network from the posterior spiracles10 and have since diversified in shape and size in the D. simulans clade in particular over the last 240,000 years driven by sexual selection18-21. The genetic basis of this diversification is highly polygenic but to the best of our knowledge none of the causative genes have yet been identified despite extensive mapping22-30. Identifying the genes underlying the diversification of these secondary sexual structures is essential to understanding the basis of changes in their morphology and the evolutionary impact on copulation and species recognition. Here, we show that the transcription factor encoded by Sox21b negatively regulates posterior lobe size during development. This is consistent with higher and expanded expression of Sox21b in D. mauritiana, which develops smaller posterior lobes compared to D. simulans. We tested this by generating reciprocal hemizygotes and confirmed that changes in Sox21b underlie posterior lobe evolution between these two species. Furthermore, we found that differences in posterior lobe size caused by the species-specific allele of Sox21b significantly affect the duration of copulation. Taken together, our study reveals the genetic basis for the sexual selection driven diversification of a novel morphological structure and its functional impact on copulatory behaviour. HighlightsO_LISox21b regulates the development of the epandrial posterior lobes, a recently evolved novel structure of some species of the Drosophila melanogaster subgroup, which has subsequently rapidly diversified in size and shape. C_LIO_LID. mauritiana has smaller posterior lobes than D. simulans and more expansive expression of Sox21b in the developing genitalia. Using a reciprocal hemizygosity test, we show that variation in Sox21b underlies the diversification of epandrial posterior lobe size and shape between D. simulans and D. mauritiana. C_LIO_LIBehavioural tests show that the species allele of Sox21b causes differences in the duration of copulation in otherwise genetically identical backgrounds. C_LIO_LISox21b has evolved between D. simulans and D. mauritiana, and contributed to the divergence of a morphological novelty and copulatory behaviour between these two species. C_LI\nAn effector deletion leads to the breakdown of partial grapevine resistance to downy mildew\nAuthors: Paineau, M.; Minio, A.; Mestre, P.; Fabre, F.; Mazet, I. D.; Couture, C.; Legeai, F.; Dumartinet, T.; Cantu, D.; Delmotte, F.\nScore: 14.9, Published: 2023-08-18 DOI: 10.1101/2023.08.17.553663\nGrapevine downy mildew, caused by the oomycete Plasmopara viticola, is a globally destructive disease that particularly affect the Eurasian wine grape V. vinifera. While genetically resistant varieties are becoming more accessible, populations of P. viticola are demonstrating rapid adaptability, successfully over-coming these resistances. Here we aimed to identify the avirulence genes involved in the interaction with the Rpv3.1-mediated resistance in grapevine. We sequenced the full genome of 136 P. viticola strains sampled in a natural population of Bordeaux (France) and characterized their development on both resistant and sensitive cultivars. The genome-wide association study allowed the identification of a genomic region associated with the breakdown of Rpv3.1 grapevine resistance (avrRpv3.1 locus). A diploid-aware reassembly of the P. viticola INRA-Pv221 genome allowed to detect structural variations in this locus, including a major 30 Kbp deletion. At the avrRpv3.1 locus, virulent P. viticola strains presented deletion on both haplotypes indicating that avirulence is recessive. The deletion involves two closely-related genes that encode proteins containing 800-900 amino acids with a signal peptide. The structure of the predicted proteins contains repeats of the LWY-fold structural modules, typical of oomycete effectors. Moreover, when these proteins were transiently expressed, they induced cell death in grapevines carrying Rpv3.1 resistance, confirming their avirulence nature. The first description of candidate effectors of P. viticola involved in the interaction with resistance gene provides valuable insights into the genetic mechanisms that enable P. viticola to adapt to grapevine resistance, laying a foundation for developing strategies to manage this damaging crop pathogen.\nAn evolutionary timescale for Bacteria calibrated using the Great Oxidation Event\nAuthors: Davin, A. A.; Woodcroft, B. J.; Soo, R. M.; Murali, R.; Schrempf, D.; Clark, J.; Boussau, B.; Moody, E. R. R.; Szantho, L. L.; Richy, E.; Pisani, D.; Hemp, J.; Fischer, W. W.; Donoghue, P.; Spang, A.; Hugenholtz, P.; Williams, T. A.; Szöllosi, G. J.\nScore: 41.5, Published: 2023-08-11 DOI: 10.1101/2023.08.08.552427\nMost of lifes diversity and history is microbial but it has left a meagre fossil record, greatly hindering understanding of evolution in deep time. However, the co-evolution of life and the Earth system has left signatures of bacterial metabolism in the geochemical record, most conspicuously the Great Oxidation Event (GOE) [~]2.33 billion years ago (Ga, (Poulton et al. 2021)), in which oxygenic photosynthesis and tectonism (Eguchi, Seales, and Dasgupta 2019) transformed Earths biosphere from dominantly anaerobic to aerobic. Here, we combine machine learning and phylogenetic reconciliation to infer ancestral transitions to aerobic lifestyles during bacterial evolution. Linking these transitions to the GOE provides new constraints to infer the timetree of Bacteria. We find that extant bacterial phyla are truly ancient, having radiated in the Archaean and the Proterozoic: the oldest include Bacillota (Firmicutes), which radiated 3.1-3.7 Ga, Cyanobacteria (3.3-3.5 Ga) and Patescibacteria (3-3.5 Ga). We show that most bacterial phyla were ancestrally anaerobic and that most transitions to an aerobic lifestyle post-dated the GOE. Our analyses trace oxygen production and consumption back to Cyanobacteria. From that starting point, horizontal transfer seeded aerobic lifestyles across bacterial diversity over hundreds of millions of years. Our analyses demonstrate that the diversification of aerobes proceeded in two waves corresponding to the GOE and to a second sustained rise in atmospheric O2 at the dawn of the Palezoic (Krause et al. 2022).\nPolygenic architecture of flowering time and its relationship with local environments in the grass Brachypodium distachyon.\nAuthors: Minadakis, N.; Kaderli, L.; Horvath, R.; Xu, W.; Thieme, M.; Woods, D. P.; Roulin, A. C.\nScore: 10.2, Published: 2023-08-16 DOI: 10.1101/2023.07.11.548268\nSynchronizing the timing of reproduction with the environment is crucial in the wild. Among the multiple mechanisms annual plants evolved to sense their environment, the requirement of cold-mediated vernalization is a major process that prevents individuals from flowering during winter. In many annual plants including crops, both a long and short vernalization requirement can be observed within species, resulting in so-called early-(spring) and late (winter)-flowering genotypes. Here, using the grass model Brachypodium distachyon, we explored the link between vernalization requirement, flowering time, environmental variation, and diversity at flowering genes by combining measurements under greenhouse and outdoor conditions. These experiments confirmed that B. distachyon natural accessions display large differences regarding vernalization requirements and ultimately flowering time. We underline significant, albeit quantitative effects of current environmental conditions on flowering time. Population genomics in 332 natural accessions revealed that eight well-characterized flowering-time genes contribute significantly to flowering time variation and display signs of polygenic selection. Flowering-time genes, however, do not colocalize with GWAs peaks obtained with outdoor measurements, indicating that flowering-time genes may not largely contribute to flowering time variation in the wild. Altogether, our study fosters our understanding of the polygenic architecture of flowering time in a natural grass system and opens new avenues of research to investigate the gene-by-environment interaction at play for this trait.\nThe genome of Litomosoides sigmodontis illuminates the origins of Y chromosomes in filarial nematodes\nAuthors: Stevens, L.; Kieninger, M.; Chan, B.; Wood, J. M. D.; Gonzalez de la Rosa, P.; Allen, J.; Blaxter, M.\nScore: 12.5, Published: 2023-08-03 DOI: 10.1101/2023.08.02.550553\nHeteromorphic sex chromosomes are usually thought to have originated from a pair of autosomes that acquired a sex-determining locus and subsequently stopped recombining, leading to degeneration of the sex-limited chromosome. In contrast, in rhabditid nematodes, sex is determined by an X-chromosome counting mechanism and males are hemizygous for one or more X chromosomes (XX/X0). Some species of filarial nematodes, including important parasites of humans, have heteromorphic XX/XY systems. It has been assumed that sex is determined by a Y-linked locus in these species. However, karyotypic analyses suggested that filarial Y chromosomes are derived from the unfused autosome following an X-to-autosome fusion. Here, we generated a chromosome-level reference genome for Litomosoides sigmodontis, a filarial nematode with the ancestral filarial karyotype and sex determination mechanism (XX/X0). We mapped the assembled chromosomes to the rhabditid nematode ancestral linkage (or Nigon) elements. We found that the L. sigmodontis X chromosome was formed from a fusion of NigonX (the ancestrally X-linked element) and NigonD (ancestrally autosomal) that occurred in the last common ancestor of all filarial nematodes. In the two filarial lineages with XY systems, the X chromosomes were formed from two recent and independent fusions of the ancestral X chromosome with different autosomal Nigon elements. In both lineages, the region shared by the neo-X and neo-Y chromosomes is within the ancestrally autosomal portion of the X, confirming that the filarial Y chromosomes are derived from unfused autosomes. Sex determination in XY filarial nematodes therefore likely continues to operate via the ancestral X-chromosome counting mechanism, rather than via a neo-Y-linked sex-determining locus.\nFruitless mating with the exes: the irreversible parthenogenesis in a stick insect\nAuthors: Nozaki, T.; Chikami, Y.; Yano, K.; Sato, R.; Suetsugu, K.; Kaneko, S.\nScore: 233.7, Published: 2023-07-30 DOI: 10.1101/2023.07.28.550994\nParthenogenetic lineages, common in many animals, have sparked debate about their evolutionary persistence. Even after the loss of male individuals, males can survive on the genomes; in fact, rarely occurring males were described in many asexual lineages. Theoretical predictions suggest that genes related to male sexual traits in parthenogenetic lineages will decay over long evolutionary times free from the pressure of stabilizing selection. However, in many cases, such rare males are potentially functional. In this study, we investigated the functionality of rare males in the Japanese parthenogenetic stick insect, Ramulus mikado. Rarely occurring males in the species exhibited the external and anatomical morphology typical in stick insect males, and vigorously and successfully performed mating behaviors with conspecific females; nevertheless, no paternal genotypes were detected in the offspring; all embryos were genetically identical to their mothers. Our histological observation demonstrated that females received a spermatophore without sperms in their copulatory pouch, and revealed that in a male, spermatogenesis was completely deformed. These results suggest the irreversible clonal reproduction of R. mikado due to compromised male function. Our present study provides the first case of evident dysfunction in rare males of stick insects, illuminating the degenerative evolution of sexual traits in asexual lineages.\nThe pace of mitochondrial molecular evolution varies with seasonal migration distance\nAuthors: Pegan, T. M.; Berv, J. S.; Gulson-Castillo, E. R.; Kimmitt, A. A.; Winger, B. M.\nScore: 7.4, Published: 2023-08-03 DOI: 10.1101/2023.08.01.551470\nAnimals that engage in long-distance seasonal migration experience strong selective pressures on their metabolic performance and life history, with potential consequences for molecular evolution. Species with slow life histories typically show lower rates of synonymous substitution (dS) than \"fast\" species. Previous work has suggested that long-distance seasonal migrants have a slower life history strategy than short-distance migrants, raising the possibility that rates of molecular evolution may covary with migration distance. Additionally, long-distance migrants may face strong selection on metabolically important mitochondrial genes owing to their long-distance flights. Using over 1000 mitochondrial genomes, we assessed the relationship between migration distance and mitochondrial molecular evolution in 39 boreal-breeding migratory bird species. We show that migration distance correlates negatively with dS, suggesting that the slow life history associated with long-distance migration is reflected in rates of molecular evolution. Mitochondrial genes in every study species exhibited evidence of purifying selection, but the strength of selection was greater in short-distance migrants, contrary to our predictions. This result may indicate selection for cold tolerance on mitochondrial evolution among species overwintering at high latitudes. Our study demonstrates that the pervasive correlation between life history and molecular evolutionary rates exists in the context of differential adaptations to seasonality.\nTransmission bottleneck size estimation from de novo viral genetic variation\nAuthors: Shi, T.; Harris, J.; Martin, M. A.; Koelle, K.\nScore: 11.8, Published: 2023-08-14 DOI: 10.1101/2023.08.14.553219\nSequencing of viral infections has become increasingly common over the last decade. Deep sequencing data in particular have proven useful in characterizing the roles that genetic drift and natural selection play in shaping within-host viral populations. They have also been used to estimate transmission bottleneck sizes from identified donor-recipient pairs. These bottleneck sizes quantify the number of viral particles that establish genetic lineages in the recipient host and are important to estimate due to their impact on viral evolution. Current approaches for estimating bottleneck sizes exclusively consider the subset of viral sites that are observed as polymorphic in the donor individual. However, allele frequencies can change dramatically over the course of an individuals infection, such that sites that are polymorphic in the donor at the time of transmission may not be polymorphic in the donor at the time of sampling and allele frequencies at donor-polymorphic sites may change dramatically over the course of a recipients infection. Because of this, transmission bottleneck sizes estimated using allele frequencies observed at a donors polymorphic sites may be considerable underestimates of true bottleneck sizes. Here, we present a new statistical approach for instead estimating bottleneck sizes using patterns of viral genetic variation that arose de novo within a recipient individual. Specifically, our approach makes use of the number of clonal viral variants observed in a transmission pair, defined as the number of viral sites that are monomorphic in both the donor and the recipient but carry different alleles. We first test our approach on a simulated dataset and then apply it to both influenza A virus sequence data and SARS-CoV-2 sequence data from identified transmission pairs. Our results confirm the existence of extremely tight transmission bottlenecks for these two respiratory viruses, using an approach that does not tend to underestimate transmission bottleneck sizes.\n",
  "wordCount" : "2843",
  "inLanguage": "en",
  "datePublished": "2023-08-23T10:37:56Z",
  "dateModified": "2023-08-23T10:37:56Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on August 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.552946">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.552946" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.552946">
        <p class="paperTitle">Opsin gene duplication in Lepidoptera: retrotransposition, sex linkage, and gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.552946" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.552946" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mulhair, P. O.; Crowley, L.; Boyes, D. H.; Lewis, O. T.; Holland, P. W. H.</p>
        <p class="info">Score: 35.3, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.552946' target='https://doi.org/10.1101/2023.08.16.552946'> 10.1101/2023.08.16.552946</a></p>
        <p class="abstract">Colour vision in insects is determined by signalling cascades, central to which are opsin proteins, resulting in sensitivity to light at different wavelengths. In certain insect groups, lineage specific evolution of opsin genes, in terms of copy number, shifts in expression patterns, and functional amino acid substitutions, has resulted in changes in colour vision with subsequent behavioural and niche adaptations. Lepidoptera are a fascinating model to address whether evolutionary change in opsin content and sequence evolution are associated with changes in vision phenotype. Until recently, the lack of high quality genome data representing broad sampling across the lepidopteran phylogeny has greatly limited our ability to accurately address this question. Here, we annotate opsin genes in 219 lepidopteran genomes representing 33 families, reconstruct their evolutionary history, and analyse shifts in selective pressures and expression between genes and species. We discover 44 duplication events in opsin genes across [~]300 million years of lepidopteran evolution. While many duplication events are species or family specific, we find retention of an ancient long-wavelength sensitive (LW) opsin duplication derived by retrotransposition within the speciose superfamily Noctuoidea (in the families Nolidae, Erebidae, and Noctuidae). This conserved LW retrogene shows life stage specific expression suggesting visual sensitivities or other sensory functions specific to the early larval stage. This study provides a comprehensive order-wide view of opsin evolution across Lepidoptera, showcasing high rates of opsin duplications and changes in expression patterns.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550432">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550432" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550432">
        <p class="paperTitle">Evidence for ecological tuning of novel anuran biofluorescent signals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550432" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550432" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Whitcher, C.; Ron, S. R.; Ayala-Varela, F.; Crawford, A.; Herrera-Alva, V.; Castillo-Urbina, E.; Grazziotin, F.; Bowman, R. M.; Lemmon, A. R.; Moriarty Lemmon, E.</p>
        <p class="info">Score: 37.5, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550432' target='https://doi.org/10.1101/2023.07.25.550432'> 10.1101/2023.07.25.550432</a></p>
        <p class="abstract">Our study assesses the variability of amphibian biofluorescence and provides insight into its potential functions and role in anuran evolution. Via a field survey across South America, we discovered and documented patterns of biofluorescence in tropical amphibians. We more than tripled the number of species that have been tested for this trait and added representatives from previously untested anuran families. We found evidence for ecological tuning (i.e., the specific adaptation of a signal to the environment in which it is received) of the novel anuran biofluorescent signals. Across groups, the fluorescence excitation peak matches the wavelengths most available at twilight, the light environment in which most frog species are active. Additionally, biofluorescence emission spans both wavelengths of low availability in twilight and the peak sensitivity of green-sensitive rods in the anuran eye, likely increasing contrast of this signal for a conspecific receiver. With evidence of tuning to the ecology and sensory systems of frogs, our results suggest frog biofluorescence is likely functioning in anuran communication.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.552955">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.552955" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.552955">
        <p class="paperTitle">Sox21b underlies the rapid diversification of a novel male genital structure between Drosophila species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.552955" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.552955" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ridgway, A. M.; Hood, E.; Jimenez, J. F.; Nunes, M. D. S.; McGregor, A. P.</p>
        <p class="info">Score: 21.5, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.552955' target='https://doi.org/10.1101/2023.08.17.552955'> 10.1101/2023.08.17.552955</a></p>
        <p class="abstract">The emergence and subsequent diversification of morphological novelties is a major feature of animal evolution1-9. However, in most cases little is known about the molecular basis of the evolution of novel structures and the genetic mechanisms underlying their diversification. The epandrial posterior lobes of the male genital arch is a novelty of some species of the Drosophila melanogaster subgroup10-13. The posterior lobes grasp the ovipositor of the female and then integrate between her abdominal tergites, and therefore these structures are important for copulation and species-recognition10-12,14-17. The posterior lobes evolved from co-option of a Hox regulated gene network from the posterior spiracles10 and have since diversified in shape and size in the D. simulans clade in particular over the last 240,000 years driven by sexual selection18-21. The genetic basis of this diversification is highly polygenic but to the best of our knowledge none of the causative genes have yet been identified despite extensive mapping22-30. Identifying the genes underlying the diversification of these secondary sexual structures is essential to understanding the basis of changes in their morphology and the evolutionary impact on copulation and species recognition. Here, we show that the transcription factor encoded by Sox21b negatively regulates posterior lobe size during development. This is consistent with higher and expanded expression of Sox21b in D. mauritiana, which develops smaller posterior lobes compared to D. simulans. We tested this by generating reciprocal hemizygotes and confirmed that changes in Sox21b underlie posterior lobe evolution between these two species. Furthermore, we found that differences in posterior lobe size caused by the species-specific allele of Sox21b significantly affect the duration of copulation. Taken together, our study reveals the genetic basis for the sexual selection driven diversification of a novel morphological structure and its functional impact on copulatory behaviour.

HighlightsO_LISox21b regulates the development of the epandrial posterior lobes, a recently evolved novel structure of some species of the Drosophila melanogaster subgroup, which has subsequently rapidly diversified in size and shape.
C_LIO_LID. mauritiana has smaller posterior lobes than D. simulans and more expansive expression of Sox21b in the developing genitalia. Using a reciprocal hemizygosity test, we show that variation in Sox21b underlies the diversification of epandrial posterior lobe size and shape between D. simulans and D. mauritiana.
C_LIO_LIBehavioural tests show that the species allele of Sox21b causes differences in the duration of copulation in otherwise genetically identical backgrounds.
C_LIO_LISox21b has evolved between D. simulans and D. mauritiana, and contributed to the divergence of a morphological novelty and copulatory behaviour between these two species.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553663">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553663" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553663">
        <p class="paperTitle">An effector deletion leads to the breakdown of partial grapevine resistance to downy mildew</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553663" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553663" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Paineau, M.; Minio, A.; Mestre, P.; Fabre, F.; Mazet, I. D.; Couture, C.; Legeai, F.; Dumartinet, T.; Cantu, D.; Delmotte, F.</p>
        <p class="info">Score: 14.9, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553663' target='https://doi.org/10.1101/2023.08.17.553663'> 10.1101/2023.08.17.553663</a></p>
        <p class="abstract">Grapevine downy mildew, caused by the oomycete Plasmopara viticola, is a globally destructive disease that particularly affect the Eurasian wine grape V. vinifera. While genetically resistant varieties are becoming more accessible, populations of P. viticola are demonstrating rapid adaptability, successfully over-coming these resistances. Here we aimed to identify the avirulence genes involved in the interaction with the Rpv3.1-mediated resistance in grapevine. We sequenced the full genome of 136 P. viticola strains sampled in a natural population of Bordeaux (France) and characterized their development on both resistant and sensitive cultivars. The genome-wide association study allowed the identification of a genomic region associated with the breakdown of Rpv3.1 grapevine resistance (avrRpv3.1 locus). A diploid-aware reassembly of the P. viticola INRA-Pv221 genome allowed to detect structural variations in this locus, including a major 30 Kbp deletion. At the avrRpv3.1 locus, virulent P. viticola strains presented deletion on both haplotypes indicating that avirulence is recessive. The deletion involves two closely-related genes that encode proteins containing 800-900 amino acids with a signal peptide. The structure of the predicted proteins contains repeats of the LWY-fold structural modules, typical of oomycete effectors. Moreover, when these proteins were transiently expressed, they induced cell death in grapevines carrying Rpv3.1 resistance, confirming their avirulence nature. The first description of candidate effectors of P. viticola involved in the interaction with resistance gene provides valuable insights into the genetic mechanisms that enable P. viticola to adapt to grapevine resistance, laying a foundation for developing strategies to manage this damaging crop pathogen.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552427">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552427" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552427">
        <p class="paperTitle">An evolutionary timescale for Bacteria calibrated using the Great Oxidation Event</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552427" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552427" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davin, A. A.; Woodcroft, B. J.; Soo, R. M.; Murali, R.; Schrempf, D.; Clark, J.; Boussau, B.; Moody, E. R. R.; Szantho, L. L.; Richy, E.; Pisani, D.; Hemp, J.; Fischer, W. W.; Donoghue, P.; Spang, A.; Hugenholtz, P.; Williams, T. A.; Szöllosi, G. J.</p>
        <p class="info">Score: 41.5, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552427' target='https://doi.org/10.1101/2023.08.08.552427'> 10.1101/2023.08.08.552427</a></p>
        <p class="abstract">Most of lifes diversity and history is microbial but it has left a meagre fossil record, greatly hindering understanding of evolution in deep time. However, the co-evolution of life and the Earth system has left signatures of bacterial metabolism in the geochemical record, most conspicuously the Great Oxidation Event (GOE) [~]2.33 billion years ago (Ga, (Poulton et al. 2021)), in which oxygenic photosynthesis and tectonism (Eguchi, Seales, and Dasgupta 2019) transformed Earths biosphere from dominantly anaerobic to aerobic. Here, we combine machine learning and phylogenetic reconciliation to infer ancestral transitions to aerobic lifestyles during bacterial evolution. Linking these transitions to the GOE provides new constraints to infer the timetree of Bacteria. We find that extant bacterial phyla are truly ancient, having radiated in the Archaean and the Proterozoic: the oldest include Bacillota (Firmicutes), which radiated 3.1-3.7 Ga, Cyanobacteria (3.3-3.5 Ga) and Patescibacteria (3-3.5 Ga). We show that most bacterial phyla were ancestrally anaerobic and that most transitions to an aerobic lifestyle post-dated the GOE. Our analyses trace oxygen production and consumption back to Cyanobacteria. From that starting point, horizontal transfer seeded aerobic lifestyles across bacterial diversity over hundreds of millions of years. Our analyses demonstrate that the diversification of aerobes proceeded in two waves corresponding to the GOE and to a second sustained rise in atmospheric O2 at the dawn of the Palezoic (Krause et al. 2022).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548268">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548268" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548268">
        <p class="paperTitle">Polygenic architecture of flowering time and its relationship with local environments in the grass Brachypodium distachyon.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548268" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548268" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Minadakis, N.; Kaderli, L.; Horvath, R.; Xu, W.; Thieme, M.; Woods, D. P.; Roulin, A. C.</p>
        <p class="info">Score: 10.2, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548268' target='https://doi.org/10.1101/2023.07.11.548268'> 10.1101/2023.07.11.548268</a></p>
        <p class="abstract">Synchronizing the timing of reproduction with the environment is crucial in the wild. Among the multiple mechanisms annual plants evolved to sense their environment, the requirement of cold-mediated vernalization is a major process that prevents individuals from flowering during winter. In many annual plants including crops, both a long and short vernalization requirement can be observed within species, resulting in so-called early-(spring) and late (winter)-flowering genotypes. Here, using the grass model Brachypodium distachyon, we explored the link between vernalization requirement, flowering time, environmental variation, and diversity at flowering genes by combining measurements under greenhouse and outdoor conditions. These experiments confirmed that B. distachyon natural accessions display large differences regarding vernalization requirements and ultimately flowering time. We underline significant, albeit quantitative effects of current environmental conditions on flowering time. Population genomics in 332 natural accessions revealed that eight well-characterized flowering-time genes contribute significantly to flowering time variation and display signs of polygenic selection. Flowering-time genes, however, do not colocalize with GWAs peaks obtained with outdoor measurements, indicating that flowering-time genes may not largely contribute to flowering time variation in the wild. Altogether, our study fosters our understanding of the polygenic architecture of flowering time in a natural grass system and opens new avenues of research to investigate the gene-by-environment interaction at play for this trait.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.550553">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.550553" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.550553">
        <p class="paperTitle">The genome of Litomosoides sigmodontis illuminates the origins of Y chromosomes in filarial nematodes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.550553" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.550553" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stevens, L.; Kieninger, M.; Chan, B.; Wood, J. M. D.; Gonzalez de la Rosa, P.; Allen, J.; Blaxter, M.</p>
        <p class="info">Score: 12.5, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.550553' target='https://doi.org/10.1101/2023.08.02.550553'> 10.1101/2023.08.02.550553</a></p>
        <p class="abstract">Heteromorphic sex chromosomes are usually thought to have originated from a pair of autosomes that acquired a sex-determining locus and subsequently stopped recombining, leading to degeneration of the sex-limited chromosome. In contrast, in rhabditid nematodes, sex is determined by an X-chromosome counting mechanism and males are hemizygous for one or more X chromosomes (XX/X0). Some species of filarial nematodes, including important parasites of humans, have heteromorphic XX/XY systems. It has been assumed that sex is determined by a Y-linked locus in these species. However, karyotypic analyses suggested that filarial Y chromosomes are derived from the unfused autosome following an X-to-autosome fusion. Here, we generated a chromosome-level reference genome for Litomosoides sigmodontis, a filarial nematode with the ancestral filarial karyotype and sex determination mechanism (XX/X0). We mapped the assembled chromosomes to the rhabditid nematode ancestral linkage (or Nigon) elements. We found that the L. sigmodontis X chromosome was formed from a fusion of NigonX (the ancestrally X-linked element) and NigonD (ancestrally autosomal) that occurred in the last common ancestor of all filarial nematodes. In the two filarial lineages with XY systems, the X chromosomes were formed from two recent and independent fusions of the ancestral X chromosome with different autosomal Nigon elements. In both lineages, the region shared by the neo-X and neo-Y chromosomes is within the ancestrally autosomal portion of the X, confirming that the filarial Y chromosomes are derived from unfused autosomes. Sex determination in XY filarial nematodes therefore likely continues to operate via the ancestral X-chromosome counting mechanism, rather than via a neo-Y-linked sex-determining locus.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550994">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550994" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550994">
        <p class="paperTitle">Fruitless mating with the exes: the irreversible parthenogenesis in a stick insect</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550994" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550994" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nozaki, T.; Chikami, Y.; Yano, K.; Sato, R.; Suetsugu, K.; Kaneko, S.</p>
        <p class="info">Score: 233.7, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550994' target='https://doi.org/10.1101/2023.07.28.550994'> 10.1101/2023.07.28.550994</a></p>
        <p class="abstract">Parthenogenetic lineages, common in many animals, have sparked debate about their evolutionary persistence. Even after the loss of male individuals, males can survive on the genomes; in fact, rarely occurring males were described in many asexual lineages. Theoretical predictions suggest that genes related to male sexual traits in parthenogenetic lineages will decay over long evolutionary times free from the pressure of stabilizing selection. However, in many cases, such rare males are potentially functional. In this study, we investigated the functionality of rare males in the Japanese parthenogenetic stick insect, Ramulus mikado. Rarely occurring males in the species exhibited the external and anatomical morphology typical in stick insect males, and vigorously and successfully performed mating behaviors with conspecific females; nevertheless, no paternal genotypes were detected in the offspring; all embryos were genetically identical to their mothers. Our histological observation demonstrated that females received a spermatophore without sperms in their copulatory pouch, and revealed that in a male, spermatogenesis was completely deformed. These results suggest the irreversible clonal reproduction of R. mikado due to compromised male function. Our present study provides the first case of evident dysfunction in rare males of stick insects, illuminating the degenerative evolution of sexual traits in asexual lineages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551470">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551470" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551470">
        <p class="paperTitle">The pace of mitochondrial molecular evolution varies with seasonal migration distance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551470" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551470" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pegan, T. M.; Berv, J. S.; Gulson-Castillo, E. R.; Kimmitt, A. A.; Winger, B. M.</p>
        <p class="info">Score: 7.4, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551470' target='https://doi.org/10.1101/2023.08.01.551470'> 10.1101/2023.08.01.551470</a></p>
        <p class="abstract">Animals that engage in long-distance seasonal migration experience strong selective pressures on their metabolic performance and life history, with potential consequences for molecular evolution. Species with slow life histories typically show lower rates of synonymous substitution (dS) than &#34;fast&#34; species. Previous work has suggested that long-distance seasonal migrants have a slower life history strategy than short-distance migrants, raising the possibility that rates of molecular evolution may covary with migration distance. Additionally, long-distance migrants may face strong selection on metabolically important mitochondrial genes owing to their long-distance flights. Using over 1000 mitochondrial genomes, we assessed the relationship between migration distance and mitochondrial molecular evolution in 39 boreal-breeding migratory bird species. We show that migration distance correlates negatively with dS, suggesting that the slow life history associated with long-distance migration is reflected in rates of molecular evolution. Mitochondrial genes in every study species exhibited evidence of purifying selection, but the strength of selection was greater in short-distance migrants, contrary to our predictions. This result may indicate selection for cold tolerance on mitochondrial evolution among species overwintering at high latitudes. Our study demonstrates that the pervasive correlation between life history and molecular evolutionary rates exists in the context of differential adaptations to seasonality.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.553219">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.553219" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.553219">
        <p class="paperTitle">Transmission bottleneck size estimation from de novo viral genetic variation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.553219" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.553219" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shi, T.; Harris, J.; Martin, M. A.; Koelle, K.</p>
        <p class="info">Score: 11.8, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.553219' target='https://doi.org/10.1101/2023.08.14.553219'> 10.1101/2023.08.14.553219</a></p>
        <p class="abstract">Sequencing of viral infections has become increasingly common over the last decade. Deep sequencing data in particular have proven useful in characterizing the roles that genetic drift and natural selection play in shaping within-host viral populations. They have also been used to estimate transmission bottleneck sizes from identified donor-recipient pairs. These bottleneck sizes quantify the number of viral particles that establish genetic lineages in the recipient host and are important to estimate due to their impact on viral evolution. Current approaches for estimating bottleneck sizes exclusively consider the subset of viral sites that are observed as polymorphic in the donor individual. However, allele frequencies can change dramatically over the course of an individuals infection, such that sites that are polymorphic in the donor at the time of transmission may not be polymorphic in the donor at the time of sampling and allele frequencies at donor-polymorphic sites may change dramatically over the course of a recipients infection. Because of this, transmission bottleneck sizes estimated using allele frequencies observed at a donors polymorphic sites may be considerable underestimates of true bottleneck sizes. Here, we present a new statistical approach for instead estimating bottleneck sizes using patterns of viral genetic variation that arose de novo within a recipient individual. Specifically, our approach makes use of the number of clonal viral variants observed in a transmission pair, defined as the number of viral sites that are monomorphic in both the donor and the recipient but carry different alleles. We first test our approach on a simulated dataset and then apply it to both influenza A virus sequence data and SARS-CoV-2 sequence data from identified transmission pairs. Our results confirm the existence of extremely tight transmission bottlenecks for these two respiratory viruses, using an approach that does not tend to underestimate transmission bottleneck sizes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
