<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Changes in the cellular makeup of motor patterning circuits drive courtship song evolution in Drosophila
Authors: Ye, D.; Walsh, J. T.; Junker, I. P.; Ding, Y.
Score: 16.9, Published: 2024-01-24 DOI: 10.1101/2024.01.23.576861
How evolutionary changes in genes and neurons encode species variation in complex motor behaviors are largely unknown. Here, we develop genetic tools that permit a neural circuit comparison between the model species Drosophila melanogaster and the closely-related species D.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Changes in the cellular makeup of motor patterning circuits drive courtship song evolution in Drosophila
Authors: Ye, D.; Walsh, J. T.; Junker, I. P.; Ding, Y.
Score: 16.9, Published: 2024-01-24 DOI: 10.1101/2024.01.23.576861
How evolutionary changes in genes and neurons encode species variation in complex motor behaviors are largely unknown. Here, we develop genetic tools that permit a neural circuit comparison between the model species Drosophila melanogaster and the closely-related species D." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-28T10:37:59+00:00" />
<meta property="article:modified_time" content="2024-01-28T10:37:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Changes in the cellular makeup of motor patterning circuits drive courtship song evolution in Drosophila
Authors: Ye, D.; Walsh, J. T.; Junker, I. P.; Ding, Y.
Score: 16.9, Published: 2024-01-24 DOI: 10.1101/2024.01.23.576861
How evolutionary changes in genes and neurons encode species variation in complex motor behaviors are largely unknown. Here, we develop genetic tools that permit a neural circuit comparison between the model species Drosophila melanogaster and the closely-related species D."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Changes in the cellular makeup of motor patterning circuits drive courtship song evolution in Drosophila\nAuthors: Ye, D.; Walsh, J. T.; Junker, I. P.; Ding, Y.\nScore: 16.9, Published: 2024-01-24 DOI: 10.1101/2024.01.23.576861\nHow evolutionary changes in genes and neurons encode species variation in complex motor behaviors are largely unknown. Here, we develop genetic tools that permit a neural circuit comparison between the model species Drosophila melanogaster and the closely-related species D.",
  "keywords": [
    
  ],
  "articleBody": " Changes in the cellular makeup of motor patterning circuits drive courtship song evolution in Drosophila\nAuthors: Ye, D.; Walsh, J. T.; Junker, I. P.; Ding, Y.\nScore: 16.9, Published: 2024-01-24 DOI: 10.1101/2024.01.23.576861\nHow evolutionary changes in genes and neurons encode species variation in complex motor behaviors are largely unknown. Here, we develop genetic tools that permit a neural circuit comparison between the model species Drosophila melanogaster and the closely-related species D. yakuba, who has undergone a lineage-specific loss of sine song, one of the two major types of male courtship song in Drosophila. Neuroanatomical comparison of song patterning neurons called TN1 across the phylogeny demonstrates a link between the loss of sine song and a reduction both in the number of TN1 neurons and the neurites serving the sine circuit connectivity. Optogenetic activation confirms that TN1 neurons in D. yakuba have lost the ability to drive sine song, while maintaining the ability to drive the singing wing posture. Single-cell transcriptomic comparison shows that D. yakuba specifically lacks a cell type corresponding to TN1A neurons, the TN1 subtype that is essential for sine song. Genetic and developmental manipulation reveals a functional divergence of the sex determination gene doublesex in D. yakuba to reduce TN1 number by promoting apoptosis. Our work illustrates the contribution of motor patterning circuits and cell type changes in behavioral evolution, and uncovers the evolutionary lability of sex determination genes to reconfigure the cellular makeup of neural circuits.\nEffect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.\nAuthors: Cain, F. E.; Yamaguchi, K.\nScore: 15.9, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576074\nThe effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe. In this study 40 participants took part in a simple eye test in increasing luminance to examine if there was a difference in capacity to see in low light conditions between blue and brown-eyed individuals after a brief adaptation period. Blue eyed individuals were identified to have significantly better ability to see in lower lighting after a short adaption period than brown eyed individuals making it likely depigmented irises provide an adaptive advantage (p=0.046). Superior ability to see in low light conditions could be the result of increased straylight in depigmented irises, which in light luminance is disadvantageous but in low light conditions may provide an advantage. More research is needed to determine the specific association between melanin content and low-light visual acuity. Furthermore, more research is needed to establish that the improved capacity of blue-eyed individuals to see in low light settings seen in this study is attributable to iris pigmentation rather than corresponding pigmentation elsewhere.\nDeep learning reveals the role of copy number variation in the genetic architecture of a highly polymorphic sexual trait\nAuthors: van der Bijl, W.; Shu, J.; Goberdhan, V. S.; Sherin, L.; Cortazar-Chinarro, M.; Corral-Lopez, A.; Mank, J. E.\nScore: 24.6, Published: 2024-01-23 DOI: 10.1101/2023.09.29.560175\nThe extraordinary variation in male guppy coloration has proven a powerful model for studying the interplay of natural and sexual selection. Many guppy populations exhibit substantial Y-linkage of color traits, and this has hampered the identification of the genetic architecture underlying male guppy color, as well as clouded our understanding of how this exceptional level of diversity is maintained. Here we identify the heritability and genetic basis of male color variation using convolutional neural networks for high-resolution phenotyping coupled with selection experiments, controlled pedigrees and whole-genome resequencing for a Genome Wide Association Study (GWAS) of color. Our phenotypic and genomic results converge to show that color patterning in guppies is a combination of many heritable features, each with a partially overlapping genetic architecture spanning the entire genome. Unusually, our GWAS results suggest that copy number variation, particularly copies shared between the Y chromosome and the remainder of the genome, is responsible for much of the variation in color in guppies, providing a potential mechanism for the maintenance of variation of this classic model trait.\nParasite contingency loci and the evolution of host specificity: Simple sequence repeats mediate Bartonella adaptation to a wild rodent host\nAuthors: Rodriguez-Pastor, R.; Knossow, N.; Shahar, N.; Hasik, A. Z.; Deatherage, D. E.; Gutierrez, R.; Harrus, S.; Zaman, L. E.; Lenski, R. E.; Barrick, J. E.; Hawlena, H.\nScore: 6.8, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576196\nParasites can adapt to better exploit their hosts on many timescales, ranging from within a single infection to across serial infections of multiple hosts. However, little is known about how the genomes of parasites in natural communities evolve when they face diverse hosts. We investigated how Bartonella bacteria that circulate in rodent communities in the dunes of the Negev Desert in Israel adapt to different hosts. We propagated 15 Bartonella populations through infections of either a single rodent species (Gerbillus andersoni or Gerbillus pyramidum) or alternating between the two. After 20 rodent passages, strains with de novo mutations outcompeted the ancestor in most lines. Mutations in two mononucleotide simple sequence repeats (SSRs) dominated the evolutionary dynamics and caused frameshifts in the same adhesin gene. These mutations appeared exclusively in populations that encountered G. andersoni, and they improved the ability to exploit this host. Similar SSRs in other genes are conserved and exhibit ON/OFF variation in Bartonella isolates from the Negev Desert dunes. Our results suggest that SSR-based contingency loci are important not only for rapidly and reversibly generating antigenic variation to escape immune responses but may also play a prominent role in the evolution of host specificity.\nStronger evidence for relaxed selection than adaptive evolution in high-elevation animal mtDNA\nAuthors: Iverson, E. N. K.; Criswell, A.; Havird, J. C.\nScore: 6.4, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576402\nMitochondrial (mt) genes are the subject of many adaptive hypotheses due to the key role of mitochondria in energy production and metabolism. One widespread adaptive hypothesis is that selection imposed by life at high elevation leads to the rapid fixation of beneficial alleles in mtDNA, reflected in the increased rates of mtDNA evolution documented in many high-elevation species. However, the assumption that fast mtDNA evolution is caused by positive, rather than relaxed purifying selection has rarely been tested. Here, we calculated the dN/dS ratio, a metric of nonsynonymous substitution bias, and explicitly tested for relaxed selection in the mtDNA of over 700 species of terrestrial vertebrates, freshwater fishes, and arthropods, with information on elevation and latitudinal range limits, range sizes, and body sizes. We confirmed that mitochondrial genomes of high-elevation taxa have slightly higher dN/dS ratios compared to low-elevation relatives. High-elevation species tend to have smaller ranges, which predict higher dN/dS ratios and more relaxed selection across species and clades, while absolute elevation and latitude do not predict higher dN/dS. We also find a positive relationship between body mass and dN/dS, supporting a role for small effective population size leading to relaxed selection. We conclude that higher mt dN/dS among high-elevation species is more likely to reflect relaxed selection due to smaller ranges and reduced effective population size than adaptation to the environment. Our results highlight the importance of rigorously testing adaptive stories against non-adaptive alternative hypotheses, especially in mt genomes.\nEarly acting inbreeding depression can evolve as an inbreeding avoidance mechanism\nAuthors: Brandvain, Y. J.; Thomson, L.; Pyhajarvi, T.\nScore: 9.8, Published: 2024-01-25 DOI: 10.1101/2023.03.20.533524\nDespite the potential for mechanical, developmental and/or chemical mechanisms to prevent self-fertilization, incidental self-fertilization is inevitable in many predominantly outcrossing species. In such cases, inbreeding can compromise individual fitness. Unquestionably, much of this inbreeding depression is maladaptive. However, we show that when reproductive compensation allows for the replacement of inviable embryos lost early in development, selection can favor deleterious recessive variants that induce \"self-sacrificial\" death of inbred embryos. Our theoretical results provide numerous testable predictions which could challenge the assumption that inbreeding depression is always maladaptive. Our work is applicable any species that cannot fully avoid inbreeding, exhibits substantial inbreeding depression, and has the potential to compensate embryos lost early in development. In addition to its general applicability our theory suggests that self-sacrificial variants might be responsible for the remarkably low realized selfing rates of gymnosperms with high primary selfing rates, as gymnosperms exhibit strong inbreeding depression, have effective reproductive compensation mechanisms, and cannot evolve chemical self-incompatibility.\nCell type and regulatory analysis in amphioxus illuminates evolutionary origin of the vertebrate head\nAuthors: Markos, A.; Kubovciak, J.; Mikula Mrstakova, S.; Paces, J.; Zitova, A.; Kozmik, Z.; Machacova, S.; Kozmik, Z.; Kozmikova, I.\nScore: 6.2, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576194\nTo shed light on the enigmatic origin of the vertebrate head, our study employs an integrated approach that combines single-cell transcriptomics, perturbations in signalling pathways, and cis-regulatory analysis in amphioxus, a close relative of chordate common ancestor. Through cell type characterization, we identified the presence of a prechordal plate, pre-migratory and migratory neural crest-like cell populations in the developing amphioxus embryo. Functional analysis established conserved roles of the Nodal and Hedgehog signalling pathways in prechordal plate, and of Wnt signalling pathway in neural crest development. Furthermore, the trans-species transgenic experiments provided evidence of cis-regulatory level homology within the chordate lineage. Our findings provide evidence that the key features of vertebrate head development can be traced back to the common ancestor of all chordates. One Sentence SummaryCell populations forming the vertebrate head are present in the close relative of chordate common ancestor.\nRecurrent Evolutionary Innovations in Rodent and Primate Schlafen Genes\nAuthors: Mordier, J.; Fraisse, M.; Cohen-Tannoudji, M.; Molaro, A.\nScore: 20.0, Published: 2024-01-13 DOI: 10.1101/2024.01.12.575368\nSCHLAFEN proteins are a large family of RNase-related enzymes carrying essential immune and developmental functions. Despite these important roles, Schlafen genes display varying degrees of evolutionary conservation in mammals. While this appears to influence their molecular activities, a detailed understanding of these evolutionary innovations is still lacking. Here, we used in depth phylogenomic approaches to characterize the evolutionary trajectories and selective forces shaping mammalian Schlafen genes. We traced lineage-specific Schlafen amplifications and found that recent duplicates evolved under distinct selective forces, supporting repeated sub-functionalization cycles. Codon-level natural selection analyses in primates and rodents, identified recurrent positive selection over Schlafen protein domains engaged in viral interactions. Combining crystal structures with machine learning predictions, we discovered a novel class of rapidly evolving residues enriched at the contact interface of SCHLAFEN protein dimers. Our results suggest that inter Schlafen compatibilities are under strong selective pressures and are likely to impact their molecular functions. We posit that cycles of genetic conflicts with pathogens and between paralogs drove Schlafens recurrent evolutionary innovations in mammals.\nMaternal manipulation of offspring size can trigger the evolution of eusociality in promiscuous species\nAuthors: Rees-Baylis, E.; Pen, I.; Kreider, J. J.\nScore: 4.5, Published: 2024-01-26 DOI: 10.1101/2024.01.23.576864\nEusocial organisms typically live in colonies with one reproductive queen supported by thousands of sterile workers. It is widely believed that monogamous mating is a precondition for the evolution of eusociality. Here, we present a theoretical model that simulates a realistic scenario for the evolution of eusociality. In the model, mothers can evolve control over resource allocation to offspring, affecting offspring body size. The offspring can evolve body-size-dependent dispersal, by which they disperse to breed or stay at the nest as helpers. We demonstrate that eusociality evolves, even if mothers are not strictly monogamous, if mothers can constrain their offspring's reproduction by manipulation. We also observe the evolution of social polymorphism with small individuals that help and larger individuals that disperse to breed. Our model unifies the traditional kin selection and maternal manipulation explanations for the evolution of eusociality and demonstrates that, contrary to current consensus belief, eusociality can evolve despite highly promiscuous mating.\nRelaxed purifying selection is associated with an accumulation of transposable elements in flies\nAuthors: Merel, V.; Tricou, T.; Burlet, N.; Haudry, A. V.\nScore: 4.2, Published: 2024-01-23 DOI: 10.1101/2024.01.23.576885\nAlthough the mechanisms driving genome size evolution have not yet been fully deciphered, one potentially important factor is the dynamic of accumulation in mobile selfish genetic elements named transposable elements (TEs). Because the majority of these sequences are neutral or slightly deleterious, a negative correlation between genome size and selection efficacy is expected. Nevertheless, previous studies relying on empirical data from closely related species have yielded inconsistent conclusions, leaving this matter contentious. Here, we reconstructed a phylogeny based on whole genome data (2,242 genes) for 82 lineages representing 77 Drosophilid species. We studied correlations between genome size, TE content and measures of selection efficacy. We highlighted a strong phylogenetic inertia on genome size, and confirmed that TEs are the major components of the genome size. Using an integrative approach controlling for shared history, we found genome-wide ratios of non-synonymous over synonymous divergence (dN/dS) to be strongly positively correlated to genome size and TE content, especially in GC poor genes. This work provides evidence for TE proliferation in the genome of flies when purifying selection is reduced and the genetic drift is increased. In the end, this study emphasizes the critical importance of controlling for GC heterogeneity when testing for the controversial correlation between evolutionary rates and genome size.\n",
  "wordCount" : "2185",
  "inLanguage": "en",
  "datePublished": "2024-01-28T10:37:59Z",
  "dateModified": "2024-01-28T10:37:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 28, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576861">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576861" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576861">
        <p class="paperTitle">Changes in the cellular makeup of motor patterning circuits drive courtship song evolution in Drosophila</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576861" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576861" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ye, D.; Walsh, J. T.; Junker, I. P.; Ding, Y.</p>
        <p class="info">Score: 16.9, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576861' target='https://doi.org/10.1101/2024.01.23.576861'> 10.1101/2024.01.23.576861</a></p>
        <p class="abstract">How evolutionary changes in genes and neurons encode species variation in complex motor behaviors are largely unknown. Here, we develop genetic tools that permit a neural circuit comparison between the model species Drosophila melanogaster and the closely-related species D. yakuba, who has undergone a lineage-specific loss of sine song, one of the two major types of male courtship song in Drosophila. Neuroanatomical comparison of song patterning neurons called TN1 across the phylogeny demonstrates a link between the loss of sine song and a reduction both in the number of TN1 neurons and the neurites serving the sine circuit connectivity. Optogenetic activation confirms that TN1 neurons in D. yakuba have lost the ability to drive sine song, while maintaining the ability to drive the singing wing posture. Single-cell transcriptomic comparison shows that D. yakuba specifically lacks a cell type corresponding to TN1A neurons, the TN1 subtype that is essential for sine song. Genetic and developmental manipulation reveals a functional divergence of the sex determination gene doublesex in D. yakuba to reduce TN1 number by promoting apoptosis. Our work illustrates the contribution of motor patterning circuits and cell type changes in behavioral evolution, and uncovers the evolutionary lability of sex determination genes to reconfigure the cellular makeup of neural circuits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576074" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576074">
        <p class="paperTitle">Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cain, F. E.; Yamaguchi, K.</p>
        <p class="info">Score: 15.9, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576074' target='https://doi.org/10.1101/2024.01.17.576074'> 10.1101/2024.01.17.576074</a></p>
        <p class="abstract">The effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe. In this study 40 participants took part in a simple eye test in increasing luminance to examine if there was a difference in capacity to see in low light conditions between blue and brown-eyed individuals after a brief adaptation period. Blue eyed individuals were identified to have significantly better ability to see in lower lighting after a short adaption period than brown eyed individuals making it likely depigmented irises provide an adaptive advantage (p=0.046). Superior ability to see in low light conditions could be the result of increased straylight in depigmented irises, which in light luminance is disadvantageous but in low light conditions may provide an advantage. More research is needed to determine the specific association between melanin content and low-light visual acuity. Furthermore, more research is needed to establish that the improved capacity of blue-eyed individuals to see in low light settings seen in this study is attributable to iris pigmentation rather than corresponding pigmentation elsewhere.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560175">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560175" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560175">
        <p class="paperTitle">Deep learning reveals the role of copy number variation in the genetic architecture of a highly polymorphic sexual trait</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560175" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560175" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: van der Bijl, W.; Shu, J.; Goberdhan, V. S.; Sherin, L.; Cortazar-Chinarro, M.; Corral-Lopez, A.; Mank, J. E.</p>
        <p class="info">Score: 24.6, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560175' target='https://doi.org/10.1101/2023.09.29.560175'> 10.1101/2023.09.29.560175</a></p>
        <p class="abstract">The extraordinary variation in male guppy coloration has proven a powerful model for studying the interplay of natural and sexual selection. Many guppy populations exhibit substantial Y-linkage of color traits, and this has hampered the identification of the genetic architecture underlying male guppy color, as well as clouded our understanding of how this exceptional level of diversity is maintained. Here we identify the heritability and genetic basis of male color variation using convolutional neural networks for high-resolution phenotyping coupled with selection experiments, controlled pedigrees and whole-genome resequencing for a Genome Wide Association Study (GWAS) of color. Our phenotypic and genomic results converge to show that color patterning in guppies is a combination of many heritable features, each with a partially overlapping genetic architecture spanning the entire genome. Unusually, our GWAS results suggest that copy number variation, particularly copies shared between the Y chromosome and the remainder of the genome, is responsible for much of the variation in color in guppies, providing a potential mechanism for the maintenance of variation of this classic model trait.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576196">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576196" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576196">
        <p class="paperTitle">Parasite contingency loci and the evolution of host specificity: Simple sequence repeats mediate Bartonella adaptation to a wild rodent host</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576196" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576196" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rodriguez-Pastor, R.; Knossow, N.; Shahar, N.; Hasik, A. Z.; Deatherage, D. E.; Gutierrez, R.; Harrus, S.; Zaman, L. E.; Lenski, R. E.; Barrick, J. E.; Hawlena, H.</p>
        <p class="info">Score: 6.8, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576196' target='https://doi.org/10.1101/2024.01.18.576196'> 10.1101/2024.01.18.576196</a></p>
        <p class="abstract">Parasites can adapt to better exploit their hosts on many timescales, ranging from within a single infection to across serial infections of multiple hosts. However, little is known about how the genomes of parasites in natural communities evolve when they face diverse hosts. We investigated how Bartonella bacteria that circulate in rodent communities in the dunes of the Negev Desert in Israel adapt to different hosts. We propagated 15 Bartonella populations through infections of either a single rodent species (Gerbillus andersoni or Gerbillus pyramidum) or alternating between the two. After 20 rodent passages, strains with de novo mutations outcompeted the ancestor in most lines. Mutations in two mononucleotide simple sequence repeats (SSRs) dominated the evolutionary dynamics and caused frameshifts in the same adhesin gene. These mutations appeared exclusively in populations that encountered G. andersoni, and they improved the ability to exploit this host. Similar SSRs in other genes are conserved and exhibit ON/OFF variation in Bartonella isolates from the Negev Desert dunes. Our results suggest that SSR-based contingency loci are important not only for rapidly and reversibly generating antigenic variation to escape immune responses but may also play a prominent role in the evolution of host specificity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576402">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576402" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576402">
        <p class="paperTitle">Stronger evidence for relaxed selection than adaptive evolution in high-elevation animal mtDNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576402" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576402" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Iverson, E. N. K.; Criswell, A.; Havird, J. C.</p>
        <p class="info">Score: 6.4, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576402' target='https://doi.org/10.1101/2024.01.20.576402'> 10.1101/2024.01.20.576402</a></p>
        <p class="abstract">Mitochondrial (mt) genes are the subject of many adaptive hypotheses due to the key role of mitochondria in energy production and metabolism. One widespread adaptive hypothesis is that selection imposed by life at high elevation leads to the rapid fixation of beneficial alleles in mtDNA, reflected in the increased rates of mtDNA evolution documented in many high-elevation species. However, the assumption that fast mtDNA evolution is caused by positive, rather than relaxed purifying selection has rarely been tested. Here, we calculated the dN/dS ratio, a metric of nonsynonymous substitution bias, and explicitly tested for relaxed selection in the mtDNA of over 700 species of terrestrial vertebrates, freshwater fishes, and arthropods, with information on elevation and latitudinal range limits, range sizes, and body sizes. We confirmed that mitochondrial genomes of high-elevation taxa have slightly higher dN/dS ratios compared to low-elevation relatives. High-elevation species tend to have smaller ranges, which predict higher dN/dS ratios and more relaxed selection across species and clades, while absolute elevation and latitude do not predict higher dN/dS. We also find a positive relationship between body mass and dN/dS, supporting a role for small effective population size leading to relaxed selection. We conclude that higher mt dN/dS among high-elevation species is more likely to reflect relaxed selection due to smaller ranges and reduced effective population size than adaptation to the environment. Our results highlight the importance of rigorously testing adaptive stories against non-adaptive alternative hypotheses, especially in mt genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.03.20.533524">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.03.20.533524" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.03.20.533524">
        <p class="paperTitle">Early acting inbreeding depression can evolve as an inbreeding avoidance mechanism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.03.20.533524" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.03.20.533524" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brandvain, Y. J.; Thomson, L.; Pyhajarvi, T.</p>
        <p class="info">Score: 9.8, Published: 2024-01-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.03.20.533524' target='https://doi.org/10.1101/2023.03.20.533524'> 10.1101/2023.03.20.533524</a></p>
        <p class="abstract">Despite the potential for mechanical, developmental and/or chemical mechanisms to prevent self-fertilization, incidental self-fertilization is inevitable in many predominantly outcrossing species. In such cases, inbreeding can compromise individual fitness. Unquestionably, much of this inbreeding depression is maladaptive. However, we show that when reproductive compensation allows for the replacement of inviable embryos lost early in development, selection can favor deleterious recessive variants that induce &#34;self-sacrificial&#34; death of inbred embryos. Our theoretical results provide numerous testable predictions which could challenge the assumption that inbreeding depression is always maladaptive. Our work is applicable any species that cannot fully avoid inbreeding, exhibits substantial inbreeding depression, and has the potential to compensate embryos lost early in development. In addition to its general applicability our theory suggests that self-sacrificial variants might be responsible for the remarkably low realized selfing rates of gymnosperms with high primary selfing rates, as gymnosperms exhibit strong inbreeding depression, have effective reproductive compensation mechanisms, and cannot evolve chemical self-incompatibility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576194">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576194" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576194">
        <p class="paperTitle">Cell type and regulatory analysis in amphioxus illuminates evolutionary origin of the vertebrate head</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576194" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576194" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Markos, A.; Kubovciak, J.; Mikula Mrstakova, S.; Paces, J.; Zitova, A.; Kozmik, Z.; Machacova, S.; Kozmik, Z.; Kozmikova, I.</p>
        <p class="info">Score: 6.2, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576194' target='https://doi.org/10.1101/2024.01.18.576194'> 10.1101/2024.01.18.576194</a></p>
        <p class="abstract">To shed light on the enigmatic origin of the vertebrate head, our study employs an integrated approach that combines single-cell transcriptomics, perturbations in signalling pathways, and cis-regulatory analysis in amphioxus, a close relative of chordate common ancestor. Through cell type characterization, we identified the presence of a prechordal plate, pre-migratory and migratory neural crest-like cell populations in the developing amphioxus embryo. Functional analysis established conserved roles of the Nodal and Hedgehog signalling pathways in prechordal plate, and of Wnt signalling pathway in neural crest development. Furthermore, the trans-species transgenic experiments provided evidence of cis-regulatory level homology within the chordate lineage. Our findings provide evidence that the key features of vertebrate head development can be traced back to the common ancestor of all chordates.

One Sentence SummaryCell populations forming the vertebrate head are present in the close relative of chordate common ancestor.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575368">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575368" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575368">
        <p class="paperTitle">Recurrent Evolutionary Innovations in Rodent and Primate Schlafen Genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575368" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575368" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mordier, J.; Fraisse, M.; Cohen-Tannoudji, M.; Molaro, A.</p>
        <p class="info">Score: 20.0, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575368' target='https://doi.org/10.1101/2024.01.12.575368'> 10.1101/2024.01.12.575368</a></p>
        <p class="abstract">SCHLAFEN proteins are a large family of RNase-related enzymes carrying essential immune and developmental functions. Despite these important roles, Schlafen genes display varying degrees of evolutionary conservation in mammals. While this appears to influence their molecular activities, a detailed understanding of these evolutionary innovations is still lacking. Here, we used in depth phylogenomic approaches to characterize the evolutionary trajectories and selective forces shaping mammalian Schlafen genes. We traced lineage-specific Schlafen amplifications and found that recent duplicates evolved under distinct selective forces, supporting repeated sub-functionalization cycles. Codon-level natural selection analyses in primates and rodents, identified recurrent positive selection over Schlafen protein domains engaged in viral interactions. Combining crystal structures with machine learning predictions, we discovered a novel class of rapidly evolving residues enriched at the contact interface of SCHLAFEN protein dimers. Our results suggest that inter Schlafen compatibilities are under strong selective pressures and are likely to impact their molecular functions. We posit that cycles of genetic conflicts with pathogens and between paralogs drove Schlafens recurrent evolutionary innovations in mammals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576864">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576864" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576864">
        <p class="paperTitle">Maternal manipulation of offspring size can trigger the evolution of eusociality in promiscuous species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576864" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576864" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rees-Baylis, E.; Pen, I.; Kreider, J. J.</p>
        <p class="info">Score: 4.5, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576864' target='https://doi.org/10.1101/2024.01.23.576864'> 10.1101/2024.01.23.576864</a></p>
        <p class="abstract">Eusocial organisms typically live in colonies with one reproductive queen supported by thousands of sterile workers. It is widely believed that monogamous mating is a precondition for the evolution of eusociality. Here, we present a theoretical model that simulates a realistic scenario for the evolution of eusociality. In the model, mothers can evolve control over resource allocation to offspring, affecting offspring body size. The offspring can evolve body-size-dependent dispersal, by which they disperse to breed or stay at the nest as helpers. We demonstrate that eusociality evolves, even if mothers are not strictly monogamous, if mothers can constrain their offspring&#39;s reproduction by manipulation. We also observe the evolution of social polymorphism with small individuals that help and larger individuals that disperse to breed. Our model unifies the traditional kin selection and maternal manipulation explanations for the evolution of eusociality and demonstrates that, contrary to current consensus belief, eusociality can evolve despite highly promiscuous mating.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576885">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576885" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576885">
        <p class="paperTitle">Relaxed purifying selection is associated with an accumulation of transposable elements in flies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576885" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576885" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Merel, V.; Tricou, T.; Burlet, N.; Haudry, A. V.</p>
        <p class="info">Score: 4.2, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576885' target='https://doi.org/10.1101/2024.01.23.576885'> 10.1101/2024.01.23.576885</a></p>
        <p class="abstract">Although the mechanisms driving genome size evolution have not yet been fully deciphered, one potentially important factor is the dynamic of accumulation in mobile selfish genetic elements named transposable elements (TEs). Because the majority of these sequences are neutral or slightly deleterious, a negative correlation between genome size and selection efficacy is expected. Nevertheless, previous studies relying on empirical data from closely related species have yielded inconsistent conclusions, leaving this matter contentious. Here, we reconstructed a phylogeny based on whole genome data (2,242 genes) for 82 lineages representing 77 Drosophilid species. We studied correlations between genome size, TE content and measures of selection efficacy. We highlighted a strong phylogenetic inertia on genome size, and confirmed that TEs are the major components of the genome size. Using an integrative approach controlling for shared history, we found genome-wide ratios of non-synonymous over synonymous divergence (dN/dS) to be strongly positively correlated to genome size and TE content, especially in GC poor genes. This work provides evidence for TE proliferation in the genome of flies when purifying selection is reduced and the genetic drift is increased. In the end, this study emphasizes the critical importance of controlling for GC heterogeneity when testing for the controversial correlation between evolutionary rates and genome size.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
