<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.
Authors: Cain, F. E.; Yamaguchi, K.
Score: 147.0, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576074
The effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.
Authors: Cain, F. E.; Yamaguchi, K.
Score: 147.0, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576074
The effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-14T10:38:48+00:00" />
<meta property="article:modified_time" content="2024-02-14T10:38:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.
Authors: Cain, F. E.; Yamaguchi, K.
Score: 147.0, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576074
The effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.\nAuthors: Cain, F. E.; Yamaguchi, K.\nScore: 147.0, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576074\nThe effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe.",
  "keywords": [
    
  ],
  "articleBody": " Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.\nAuthors: Cain, F. E.; Yamaguchi, K.\nScore: 147.0, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576074\nThe effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe. In this study 40 participants took part in a simple eye test in increasing luminance to examine if there was a difference in capacity to see in low light conditions between blue and brown-eyed individuals after a brief adaptation period. Blue eyed individuals were identified to have significantly better ability to see in lower lighting after a short adaption period than brown eyed individuals making it likely depigmented irises provide an adaptive advantage (p=0.046). Superior ability to see in low light conditions could be the result of increased straylight in depigmented irises, which in light luminance is disadvantageous but in low light conditions may provide an advantage. More research is needed to determine the specific association between melanin content and low-light visual acuity. Furthermore, more research is needed to establish that the improved capacity of blue-eyed individuals to see in low light settings seen in this study is attributable to iris pigmentation rather than corresponding pigmentation elsewhere.\nBacterial ecology and evolution converge on seasonal and decadal scales\nAuthors: Rohwer, R. R.; Kirkpatrick, M.; Garcia, S. L.; Kellom, M.; McMahon, K. D.; Baker, B. J.\nScore: 83.3, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579087\nEcology and evolution are distinct theories, but the short lifespans and large population sizes of microbes allow evolution to unfold along contemporary ecological time scales. To document this in a natural system, we collected a two-decade, 471-metagenome time series from a single site in a freshwater lake, which we refer to as the TYMEFLIES dataset. This massive sampling and sequencing effort resulted in the reconstruction of 30,389 metagenomic-assembled genomes (MAGs) over 50% complete, which dereplicated into 2,855 distinct genomes (\u003e96% nucleotide sequence identity). We found both ecological and evolutionary processes occurred at seasonal time scales. There were recurring annual patterns at the species level in abundances, nucleotide diversities ({pi}), and single nucleotide variant (SNV) profiles for the majority of all taxa. During annual blooms, we observed both higher and lower nucleotide diversity, indicating that both ecological differentiation and competition drove evolutionary dynamics. Overlayed upon seasonal patterns, we observed long-term change in 20% of the species SNV profiles including gradual changes, step changes, and disturbances followed by resilience. Most abrupt changes occurred in a single species, suggesting evolutionary drivers are highly specific. Nevertheless, seven members of the abundant Nanopelagicaceae family experienced abrupt change in 2012, an unusually hot and dry year. This shift coincided with increased numbers of genes under selection involved in amino acid and nucleic acid metabolism, suggesting fundamental organic nitrogen compounds drive strain differentiation in the most globally abundant freshwater family. Overall, we observed seasonal and decadal trends in both interspecific ecological and intraspecific evolutionary processes. The convergence of microbial ecology and evolution on the same time scales demonstrates that understanding microbiomes requires a new unified approach that views ecology and evolution as a single continuum.\nUnselfish meiotic drive maintains heterozygosity in a parthenogenetic ant\nAuthors: Lacy, K. D.; Hart, T.; Kronauer, D. J. C.\nScore: 32.2, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579553\nAccording to Mendel's second law, chromosomes segregate randomly in meiosis. Non-random segregation is primarily known for cases of selfish meiotic drive in females, in which particular alleles bias their own transmission into the oocyte1,2. Here, we report a rare example of unselfish meiotic drive for crossover inheritance in the clonal raider ant, Ooceraea biroi. This species produces diploid offspring parthenogenetically via fusion of two haploid nuclei from the same meiosis3. This process should cause rapid genotypic degeneration due to loss of heterozygosity, which results if crossover recombination is followed by random (Mendelian) segregation of chromosomes4,5. However, by comparing whole genomes of mothers and daughters, we show that loss of heterozygosity is exceedingly rare, raising the possibility that crossovers are infrequent or absent in O. biroi meiosis. Using a combination of cytology and whole genome sequencing, we show that crossover recombination is, in fact, common, but that loss of heterozygosity is avoided because crossover products are faithfully co-inherited. This results from a programmed violation of Mendel's law of segregation, such that crossover products segregate together rather than randomly. This discovery highlights an extreme example of cellular ''memory'' of crossovers, which could be a common yet cryptic feature of chromosomal segregation.\nA long non-coding RNA at the cortex locus controls adaptive colouration in butterflies\nAuthors: Livraghi, L.; Hanly, J. J.; Evans, E.; Wright, C. J.; Loh, L. S.; Mazo-Vargas, A.; Kamrava, K.; Carter, A. T.; van der Heijden, E. S.; Reed, R. D.; Papa, R.; Jiggins, C. D.; Martin, A.\nScore: 26.2, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579710\nEvolutionary variation in the wing pigmentation of butterflies and moths offers striking examples of adaptation by crypsis and mimicry. The cortex locus has been independently mapped as the locus controlling colour polymorphisms in 14 lepidopteran species, suggesting it acts as a genomic hotspot for the diversification of wing patterns, but functional validation through protein-coding knockouts has proven difficult to obtain. Our study unveils the role of a novel long non-coding RNA (lncRNA) which we name ivory, transcribed from the cortex locus, in modulating colour patterning in butterflies. Strikingly, ivory expression prefigures most melanic patterns during pupal development, suggesting an early developmental role in specifying scale identity. To test this, we generated CRISPR mosaic knock-outs in five nymphalid butterfly species and show that ivory mutagenesis yields transformations of dark pigmented scales into white or light-coloured scales. Genotyping of Vanessa cardui germline mutants associates these phenotypes to small on-target deletions at the conserved first exon of ivory. In contrast, cortex germline mutant butterflies with confirmed null alleles lack any wing phenotype, and exclude a colour patterning role for this adjacent gene. Overall, these results show that a lncRNA acts as a master switch of colour pattern specification, and played key roles in the adaptive diversification of colour patterns in butterflies.\nCompensatory mutations potentiate constructive neutral evolution by gene duplication\nAuthors: Despres, P. C.; Dube, A. K.; Grenier, J.; Picard, M.-E.; Shi, R.; Landry, C. R.\nScore: 20.7, Published: 2024-02-13 DOI: 10.1101/2024.02.12.579783\nProtein functions generally depend on their assembly into complexes. During evolution, some complexes have transitioned from homomers encoded by a single gene to heteromers encoded by duplicate genes. This transition could occur without adaptive evolution through intermolecular compensatory mutations. Here, we experimentally duplicate and evolve an homodimeric enzyme to examine if and how this could happen. We identify hundreds of deleterious mutations that inactivate individual homodimers but produce functional enzymes when co-expressed as duplicated proteins that heterodimerize. The structure of one such heteromer reveals how both losses of function are buffered through the introduction of asymmetry in the complex that allows them to subfunctionalize. Constructive neutral evolution can thus occur by gene duplication followed by only one deleterious mutation per duplicate.\nChronospaces: an R package for the statistical exploration of divergence times reveals extreme dependence on molecular clocks and gene choice\nAuthors: Mongiardino Koch, N.; Milla Carmona, P.\nScore: 18.6, Published: 2024-02-08 DOI: 10.1101/2024.02.04.578835\nMuch of our understanding of the history of life hinges upon time calibration, the process of assigning absolute times to cladogenetic events. Bayesian approaches to time scaling phylogenetic trees have dramatically grown in complexity, and hinge today upon numerous methodological choices. Arriving at objective justifications for all of these is difficult and time consuming. Thus, divergence times are routinely inferred under only one or a handful of parametric conditions, often chosen arbitrarily. Progress towards building robust biological timescales necessitate the development of better methods to visualize and quantify the sensitivity of results to these decisions. Here, we present an R package that assists in this endeavor through the use of chronospaces, i.e., graphical representations summarizing variation in the node ages contained in time-calibrated trees. We further test this approach using three empirical datasets spanning widely differing timeframes. Our results reveal large differences in the impact of many common methodological decisions, with the choice of clock (uncorrelated vs. autocorrelated) and loci having strong effects on inferred ages. Other decisions have comparatively minor consequences, including the use of the computationally intensive site-heterogeneous model CAT-GTR. Notably, these conclusions are as valid for Cenozoic divergences as they are for the deepest eukaryote nodes. The package chronospace implements a range of graphical and analytical tools that assist in the exploration of sensitivity and the prioritization of computational resources in the inference of divergence times.\nAtypical landscape of transposable elements in the large genome of Aedes aegypti\nAuthors: Daron, J.; Bergman, A.; Lopez-Maestre, H.; LAMBRECHTS, L.\nScore: 12.6, Published: 2024-02-08 DOI: 10.1101/2024.02.07.579293\nTransposable elements (TEs) contribute significantly to variation in genome size among eukaryotic species, but the factors influencing TE accumulation and diversification are only partially understood. Most of our current knowledge about TE organization, dynamics and evolution derives from investigations in model organisms with a relatively small genome size such as Drosophila melanogaster or Arabidopsis thaliana. Whether the observed patterns hold true in larger genomes remains to be determined. The Diptera order is an ideal taxon to address this question, because it includes a forty-year model of TE biology (D. melanogaster) as well as mosquito species with significantly larger genomes. Here, we use a comparative genomics approach to characterize the genomic forces that have shaped the TE content of the Aedes aegypti genome (1.3 Gb) relative to the Anopheles coluzzii genome (300 Mb) and the D. melanogaster genome (180 Mb). Leveraging a newly developed high-quality TE library for Ae. aegypti, our results reveal a contrasted pattern of TE organization in Ae. aegypti compared to An. coluzzii and D. melanogaster. Our analyses suggest that the substantial TE fraction observed in the Ae. aegypti genome reflect both a high rate of TE transposition and a low rate of TE elimination. Together, our results indicate that TE organization and evolutionary dynamics in the large genome of Ae. aegypti are distinct from those of other dipterans with smaller genomes.\nThe impact of non-neutral synonymous mutations when inferring selection on non-synonymous mutations\nAuthors: Martinez i Zurita, A.; Kyriazis, C.; Lohmueller, K. E.\nScore: 10.4, Published: 2024-02-08 DOI: 10.1101/2024.02.07.579314\nThe distribution of fitness effects (DFE) describes the proportions of new mutations that have different effects on reproductive fitness. Accurate measurements of the DFE are important because the DFE is a fundamental parameter in evolutionary genetics and has implications for our understanding of other phenomena like complex disease or inbreeding depression. Current computational methods to infer the DFE for nonsynonymous mutations from natural variation first estimate demographic parameters from synonymous variants to control for the effects of demography and background selection. Then, conditional on these parameters, the DFE is then inferred for nonsynonymous mutations. This approach relies on the assumption that synonymous variants are neutrally evolving. However, some evidence points toward synonymous mutations having measurable effects on fitness. To test whether selection on synonymous mutations affects inference of the DFE of nonsynonymous mutations, we simulated several possible models of selection on synonymous mutations using SLiM and attempted to recover the DFE of nonsynonymous mutations using Fit{partial}a{partial}i, a common method for DFE inference. Our results show that the presence of selection on synonymous variants leads to incorrect inferences of recent population growth. Furthermore, under certain parameter combinations, inferences of the DFE can have an inflated proportion of highly deleterious nonsynonymous mutations. However, this bias can be eliminated if the correct demographic parameters are used for DFE inference instead of the biased ones inferred from synonymous variants. Our work demonstrates how unmodeled selection on synonymous mutations may affect downstream inferences of the DFE.\nLineage-resolved analysis of embryonic gene expression evolution in C. elegans and C. briggsae\nAuthors: Large, C. R. L.; Khanal, R.; Hillier, L. W.; Huynh, C.; Kubo, C.; Kim, J.; Waterston, R.; Murray, J. I.\nScore: 14.1, Published: 2024-02-06 DOI: 10.1101/2024.02.03.578695\nWhat constraints govern the evolution of gene expression patterns across development remains a fundamental question of evolutionary biology. The advent of single-cell sequencing opens the possibility of learning these constraints by systematically profiling homologous cells across different organisms. The nematode C. elegans is a well-studied model for embryonic development, and its invariant lineage that is conserved with other Caenorhabditis species makes it an ideal model to directly compare gene expression between homologous progenitor and terminal cell types across evolution. We have measured the spatiotemporal divergence of gene expression across embryogenesis by collecting, annotating, and comparing the transcriptomes of homologous embryonic progenitors and terminal cell types, using a dataset comprising \u003e200,000 C. elegans cells and \u003e190,000 C. briggsae cells. We find a high level of similarity in gene expression programs between the species despite tens of millions of years of evolutionary divergence, consistent with their conserved developmental lineages. Even still, thousands of genes show divergence in their cell-type specific expression patterns, and these are enriched for categories involved in environmental response and behavior. Comparing the degree of expression conservation across cell types reveals that certain cell types such as neurons, have diverged more than others such as the intestine and body wall muscle. Taken together, this work identifies likely constraints on the evolution of developmental gene expression and provides a powerful resource for addressing diverse evolutionary questions.\nA single giant cell at the origin of an evolutionary innovation\nAuthors: Badiane, A.; Viala, S.; Armisen, D.; Mignerey, I.; Forcet-Vauchel, C.; De Jesus Villanueva, C. N.; Van Belleghem, S.; Crumiere, A.; Santos, E.; Francesconi, M.; Khila, A.\nScore: 8.4, Published: 2024-02-09 DOI: 10.1101/2024.02.08.579433\nKey innovations play a central role in the diversification of lineages, yet our understanding of the mechanisms underlying their emergence remains fragmented. The propelling fan is a key innovation associated with the diversification of Rhagovelia water striders into fast flowing streams. Here, we show that a single giant cell is responsible for the development of the fan throughout embryogenesis. RNA interference against the fan-specific gene family gsha/mogsha depleted the fan in the embryo but did not alter the giant cell, indicating that these genes are not required for specifying the identity of this cell. gsha/mogsha instead seem to activate the expression of cuticular genes during early development, suggesting a role of these genes in regulating the accumulation of fan structural proteins. We also show that Hox genes act to block the giant cell in the fore- and rear-legs, thus restricting its fate to the mid-legs - the pair of legs that generates movement on water. In addition, Hox expression is specifically excluded from the giant cell, thus allowing the expression of the fans developmental genetic program. This study reveals that a single cell can be sufficient to generate innovative traits that have been central for a lineage to acquire new ecological opportunities and burst into diversification. One-sentence summaryA single giant cell orchestrates the developmental genetic program of a key innovation\n",
  "wordCount" : "2494",
  "inLanguage": "en",
  "datePublished": "2024-02-14T10:38:48Z",
  "dateModified": "2024-02-14T10:38:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576074" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576074">
        <p class="paperTitle">Effect of iris pigmentation of blue and brown eyed individuals with European ancestry on ability to see in low light conditions after a short-term dark adaption period.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cain, F. E.; Yamaguchi, K.</p>
        <p class="info">Score: 147.0, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576074' target='https://doi.org/10.1101/2024.01.17.576074'> 10.1101/2024.01.17.576074</a></p>
        <p class="abstract">The effect of iris depigmentation on ability to see in low light conditions has not been thoroughly investigated as an adaptive advantage that could have contributed to the evolution and persistence of blue eyes in Europe. In this study 40 participants took part in a simple eye test in increasing luminance to examine if there was a difference in capacity to see in low light conditions between blue and brown-eyed individuals after a brief adaptation period. Blue eyed individuals were identified to have significantly better ability to see in lower lighting after a short adaption period than brown eyed individuals making it likely depigmented irises provide an adaptive advantage (p=0.046). Superior ability to see in low light conditions could be the result of increased straylight in depigmented irises, which in light luminance is disadvantageous but in low light conditions may provide an advantage. More research is needed to determine the specific association between melanin content and low-light visual acuity. Furthermore, more research is needed to establish that the improved capacity of blue-eyed individuals to see in low light settings seen in this study is attributable to iris pigmentation rather than corresponding pigmentation elsewhere.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579087">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579087" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579087">
        <p class="paperTitle">Bacterial ecology and evolution converge on seasonal and decadal scales</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579087" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579087" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rohwer, R. R.; Kirkpatrick, M.; Garcia, S. L.; Kellom, M.; McMahon, K. D.; Baker, B. J.</p>
        <p class="info">Score: 83.3, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579087' target='https://doi.org/10.1101/2024.02.06.579087'> 10.1101/2024.02.06.579087</a></p>
        <p class="abstract">Ecology and evolution are distinct theories, but the short lifespans and large population sizes of microbes allow evolution to unfold along contemporary ecological time scales. To document this in a natural system, we collected a two-decade, 471-metagenome time series from a single site in a freshwater lake, which we refer to as the TYMEFLIES dataset. This massive sampling and sequencing effort resulted in the reconstruction of 30,389 metagenomic-assembled genomes (MAGs) over 50% complete, which dereplicated into 2,855 distinct genomes (&gt;96% nucleotide sequence identity). We found both ecological and evolutionary processes occurred at seasonal time scales. There were recurring annual patterns at the species level in abundances, nucleotide diversities ({pi}), and single nucleotide variant (SNV) profiles for the majority of all taxa. During annual blooms, we observed both higher and lower nucleotide diversity, indicating that both ecological differentiation and competition drove evolutionary dynamics. Overlayed upon seasonal patterns, we observed long-term change in 20% of the species SNV profiles including gradual changes, step changes, and disturbances followed by resilience. Most abrupt changes occurred in a single species, suggesting evolutionary drivers are highly specific. Nevertheless, seven members of the abundant Nanopelagicaceae family experienced abrupt change in 2012, an unusually hot and dry year. This shift coincided with increased numbers of genes under selection involved in amino acid and nucleic acid metabolism, suggesting fundamental organic nitrogen compounds drive strain differentiation in the most globally abundant freshwater family. Overall, we observed seasonal and decadal trends in both interspecific ecological and intraspecific evolutionary processes. The convergence of microbial ecology and evolution on the same time scales demonstrates that understanding microbiomes requires a new unified approach that views ecology and evolution as a single continuum.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579553">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579553" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579553">
        <p class="paperTitle">Unselfish meiotic drive maintains heterozygosity in a parthenogenetic ant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579553" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579553" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lacy, K. D.; Hart, T.; Kronauer, D. J. C.</p>
        <p class="info">Score: 32.2, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579553' target='https://doi.org/10.1101/2024.02.09.579553'> 10.1101/2024.02.09.579553</a></p>
        <p class="abstract">According to Mendel&#39;s second law, chromosomes segregate randomly in meiosis. Non-random segregation is primarily known for cases of selfish meiotic drive in females, in which particular alleles bias their own transmission into the oocyte1,2. Here, we report a rare example of unselfish meiotic drive for crossover inheritance in the clonal raider ant, Ooceraea biroi. This species produces diploid offspring parthenogenetically via fusion of two haploid nuclei from the same meiosis3. This process should cause rapid genotypic degeneration due to loss of heterozygosity, which results if crossover recombination is followed by random (Mendelian) segregation of chromosomes4,5. However, by comparing whole genomes of mothers and daughters, we show that loss of heterozygosity is exceedingly rare, raising the possibility that crossovers are infrequent or absent in O. biroi meiosis. Using a combination of cytology and whole genome sequencing, we show that crossover recombination is, in fact, common, but that loss of heterozygosity is avoided because crossover products are faithfully co-inherited. This results from a programmed violation of Mendel&#39;s law of segregation, such that crossover products segregate together rather than randomly. This discovery highlights an extreme example of cellular &#39;&#39;memory&#39;&#39; of crossovers, which could be a common yet cryptic feature of chromosomal segregation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579710">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579710" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579710">
        <p class="paperTitle">A long non-coding RNA at the cortex locus controls adaptive colouration in butterflies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579710" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579710" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Livraghi, L.; Hanly, J. J.; Evans, E.; Wright, C. J.; Loh, L. S.; Mazo-Vargas, A.; Kamrava, K.; Carter, A. T.; van der Heijden, E. S.; Reed, R. D.; Papa, R.; Jiggins, C. D.; Martin, A.</p>
        <p class="info">Score: 26.2, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579710' target='https://doi.org/10.1101/2024.02.09.579710'> 10.1101/2024.02.09.579710</a></p>
        <p class="abstract">Evolutionary variation in the wing pigmentation of butterflies and moths offers striking examples of adaptation by crypsis and mimicry. The cortex locus has been independently mapped as the locus controlling colour polymorphisms in 14 lepidopteran species, suggesting it acts as a genomic hotspot for the diversification of wing patterns, but functional validation through protein-coding knockouts has proven difficult to obtain. Our study unveils the role of a novel long non-coding RNA (lncRNA) which we name ivory, transcribed from the cortex locus, in modulating colour patterning in butterflies. Strikingly, ivory expression prefigures most melanic patterns during pupal development, suggesting an early developmental role in specifying scale identity. To test this, we generated CRISPR mosaic knock-outs in five nymphalid butterfly species and show that ivory mutagenesis yields transformations of dark pigmented scales into white or light-coloured scales. Genotyping of Vanessa cardui germline mutants associates these phenotypes to small on-target deletions at the conserved first exon of ivory. In contrast, cortex germline mutant butterflies with confirmed null alleles lack any wing phenotype, and exclude a colour patterning role for this adjacent gene. Overall, these results show that a lncRNA acts as a master switch of colour pattern specification, and played key roles in the adaptive diversification of colour patterns in butterflies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.12.579783">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.12.579783" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.12.579783">
        <p class="paperTitle">Compensatory mutations potentiate constructive neutral evolution by gene duplication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.12.579783" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.12.579783" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Despres, P. C.; Dube, A. K.; Grenier, J.; Picard, M.-E.; Shi, R.; Landry, C. R.</p>
        <p class="info">Score: 20.7, Published: 2024-02-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.12.579783' target='https://doi.org/10.1101/2024.02.12.579783'> 10.1101/2024.02.12.579783</a></p>
        <p class="abstract">Protein functions generally depend on their assembly into complexes. During evolution, some complexes have transitioned from homomers encoded by a single gene to heteromers encoded by duplicate genes. This transition could occur without adaptive evolution through intermolecular compensatory mutations. Here, we experimentally duplicate and evolve an homodimeric enzyme to examine if and how this could happen. We identify hundreds of deleterious mutations that inactivate individual homodimers but produce functional enzymes when co-expressed as duplicated proteins that heterodimerize. The structure of one such heteromer reveals how both losses of function are buffered through the introduction of asymmetry in the complex that allows them to subfunctionalize. Constructive neutral evolution can thus occur by gene duplication followed by only one deleterious mutation per duplicate.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.04.578835">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.04.578835" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.04.578835">
        <p class="paperTitle">Chronospaces: an R package for the statistical exploration of divergence times reveals extreme dependence on molecular clocks and gene choice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.04.578835" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.04.578835" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mongiardino Koch, N.; Milla Carmona, P.</p>
        <p class="info">Score: 18.6, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.04.578835' target='https://doi.org/10.1101/2024.02.04.578835'> 10.1101/2024.02.04.578835</a></p>
        <p class="abstract">Much of our understanding of the history of life hinges upon time calibration, the process of assigning absolute times to cladogenetic events. Bayesian approaches to time scaling phylogenetic trees have dramatically grown in complexity, and hinge today upon numerous methodological choices. Arriving at objective justifications for all of these is difficult and time consuming. Thus, divergence times are routinely inferred under only one or a handful of parametric conditions, often chosen arbitrarily. Progress towards building robust biological timescales necessitate the development of better methods to visualize and quantify the sensitivity of results to these decisions. Here, we present an R package that assists in this endeavor through the use of chronospaces, i.e., graphical representations summarizing variation in the node ages contained in time-calibrated trees. We further test this approach using three empirical datasets spanning widely differing timeframes. Our results reveal large differences in the impact of many common methodological decisions, with the choice of clock (uncorrelated vs. autocorrelated) and loci having strong effects on inferred ages. Other decisions have comparatively minor consequences, including the use of the computationally intensive site-heterogeneous model CAT-GTR. Notably, these conclusions are as valid for Cenozoic divergences as they are for the deepest eukaryote nodes. The package chronospace implements a range of graphical and analytical tools that assist in the exploration of sensitivity and the prioritization of computational resources in the inference of divergence times.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.579293">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.579293" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.579293">
        <p class="paperTitle">Atypical landscape of transposable elements in the large genome of Aedes aegypti</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.579293" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.579293" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Daron, J.; Bergman, A.; Lopez-Maestre, H.; LAMBRECHTS, L.</p>
        <p class="info">Score: 12.6, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.579293' target='https://doi.org/10.1101/2024.02.07.579293'> 10.1101/2024.02.07.579293</a></p>
        <p class="abstract">Transposable elements (TEs) contribute significantly to variation in genome size among eukaryotic species, but the factors influencing TE accumulation and diversification are only partially understood. Most of our current knowledge about TE organization, dynamics and evolution derives from investigations in model organisms with a relatively small genome size such as Drosophila melanogaster or Arabidopsis thaliana. Whether the observed patterns hold true in larger genomes remains to be determined. The Diptera order is an ideal taxon to address this question, because it includes a forty-year model of TE biology (D. melanogaster) as well as mosquito species with significantly larger genomes. Here, we use a comparative genomics approach to characterize the genomic forces that have shaped the TE content of the Aedes aegypti genome (1.3 Gb) relative to the Anopheles coluzzii genome (300 Mb) and the D. melanogaster genome (180 Mb). Leveraging a newly developed high-quality TE library for Ae. aegypti, our results reveal a contrasted pattern of TE organization in Ae. aegypti compared to An. coluzzii and D. melanogaster. Our analyses suggest that the substantial TE fraction observed in the Ae. aegypti genome reflect both a high rate of TE transposition and a low rate of TE elimination. Together, our results indicate that TE organization and evolutionary dynamics in the large genome of Ae. aegypti are distinct from those of other dipterans with smaller genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.579314">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.579314" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.579314">
        <p class="paperTitle">The impact of non-neutral synonymous mutations when inferring selection on non-synonymous mutations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.579314" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.579314" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinez i Zurita, A.; Kyriazis, C.; Lohmueller, K. E.</p>
        <p class="info">Score: 10.4, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.579314' target='https://doi.org/10.1101/2024.02.07.579314'> 10.1101/2024.02.07.579314</a></p>
        <p class="abstract">The distribution of fitness effects (DFE) describes the proportions of new mutations that have different effects on reproductive fitness. Accurate measurements of the DFE are important because the DFE is a fundamental parameter in evolutionary genetics and has implications for our understanding of other phenomena like complex disease or inbreeding depression. Current computational methods to infer the DFE for nonsynonymous mutations from natural variation first estimate demographic parameters from synonymous variants to control for the effects of demography and background selection. Then, conditional on these parameters, the DFE is then inferred for nonsynonymous mutations. This approach relies on the assumption that synonymous variants are neutrally evolving. However, some evidence points toward synonymous mutations having measurable effects on fitness. To test whether selection on synonymous mutations affects inference of the DFE of nonsynonymous mutations, we simulated several possible models of selection on synonymous mutations using SLiM and attempted to recover the DFE of nonsynonymous mutations using Fit{partial}a{partial}i, a common method for DFE inference. Our results show that the presence of selection on synonymous variants leads to incorrect inferences of recent population growth. Furthermore, under certain parameter combinations, inferences of the DFE can have an inflated proportion of highly deleterious nonsynonymous mutations. However, this bias can be eliminated if the correct demographic parameters are used for DFE inference instead of the biased ones inferred from synonymous variants. Our work demonstrates how unmodeled selection on synonymous mutations may affect downstream inferences of the DFE.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.03.578695">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.03.578695" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.03.578695">
        <p class="paperTitle">Lineage-resolved analysis of embryonic gene expression evolution in C. elegans and C. briggsae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.03.578695" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.03.578695" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Large, C. R. L.; Khanal, R.; Hillier, L. W.; Huynh, C.; Kubo, C.; Kim, J.; Waterston, R.; Murray, J. I.</p>
        <p class="info">Score: 14.1, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.03.578695' target='https://doi.org/10.1101/2024.02.03.578695'> 10.1101/2024.02.03.578695</a></p>
        <p class="abstract">What constraints govern the evolution of gene expression patterns across development remains a fundamental question of evolutionary biology. The advent of single-cell sequencing opens the possibility of learning these constraints by systematically profiling homologous cells across different organisms. The nematode C. elegans is a well-studied model for embryonic development, and its invariant lineage that is conserved with other Caenorhabditis species makes it an ideal model to directly compare gene expression between homologous progenitor and terminal cell types across evolution. We have measured the spatiotemporal divergence of gene expression across embryogenesis by collecting, annotating, and comparing the transcriptomes of homologous embryonic progenitors and terminal cell types, using a dataset comprising &gt;200,000 C. elegans cells and &gt;190,000 C. briggsae cells. We find a high level of similarity in gene expression programs between the species despite tens of millions of years of evolutionary divergence, consistent with their conserved developmental lineages. Even still, thousands of genes show divergence in their cell-type specific expression patterns, and these are enriched for categories involved in environmental response and behavior. Comparing the degree of expression conservation across cell types reveals that certain cell types such as neurons, have diverged more than others such as the intestine and body wall muscle. Taken together, this work identifies likely constraints on the evolution of developmental gene expression and provides a powerful resource for addressing diverse evolutionary questions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.08.579433">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.08.579433" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.08.579433">
        <p class="paperTitle">A single giant cell at the origin of an evolutionary innovation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.08.579433" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.08.579433" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Badiane, A.; Viala, S.; Armisen, D.; Mignerey, I.; Forcet-Vauchel, C.; De Jesus Villanueva, C. N.; Van Belleghem, S.; Crumiere, A.; Santos, E.; Francesconi, M.; Khila, A.</p>
        <p class="info">Score: 8.4, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.08.579433' target='https://doi.org/10.1101/2024.02.08.579433'> 10.1101/2024.02.08.579433</a></p>
        <p class="abstract">Key innovations play a central role in the diversification of lineages, yet our understanding of the mechanisms underlying their emergence remains fragmented. The propelling fan is a key innovation associated with the diversification of Rhagovelia water striders into fast flowing streams. Here, we show that a single giant cell is responsible for the development of the fan throughout embryogenesis. RNA interference against the fan-specific gene family gsha/mogsha depleted the fan in the embryo but did not alter the giant cell, indicating that these genes are not required for specifying the identity of this cell. gsha/mogsha instead seem to activate the expression of cuticular genes during early development, suggesting a role of these genes in regulating the accumulation of fan structural proteins. We also show that Hox genes act to block the giant cell in the fore- and rear-legs, thus restricting its fate to the mid-legs - the pair of legs that generates movement on water. In addition, Hox expression is specifically excluded from the giant cell, thus allowing the expression of the fans developmental genetic program. This study reveals that a single cell can be sufficient to generate innovative traits that have been central for a lineage to acquire new ecological opportunities and burst into diversification.

One-sentence summaryA single giant cell orchestrates the developmental genetic program of a key innovation</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
