<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Functional Adaptations of Endogenous Retroviruses to the Drosophila Host Underlie their Evolutionary Diversification
Authors: Senti, K.-A.; Handler, D.; Rafanel, B.; Kosiol, C.; Schloetterer, C.; Brennecke, J.
Score: 50.6, Published: 2023-08-04 DOI: 10.1101/2023.08.03.551782
Transposable elements profoundly affect the biology and evolution of their hosts, yet their own evolutionary dynamics remain poorly understood. Here, we investigate insect endogenous retroviruses (iERVs), a monophyletic group of LTR retrotransposons that have acquired the trait of infectivity, likely through capture of a Baculovirus envelope gene.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Functional Adaptations of Endogenous Retroviruses to the Drosophila Host Underlie their Evolutionary Diversification
Authors: Senti, K.-A.; Handler, D.; Rafanel, B.; Kosiol, C.; Schloetterer, C.; Brennecke, J.
Score: 50.6, Published: 2023-08-04 DOI: 10.1101/2023.08.03.551782
Transposable elements profoundly affect the biology and evolution of their hosts, yet their own evolutionary dynamics remain poorly understood. Here, we investigate insect endogenous retroviruses (iERVs), a monophyletic group of LTR retrotransposons that have acquired the trait of infectivity, likely through capture of a Baculovirus envelope gene." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-09T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-08-09T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Functional Adaptations of Endogenous Retroviruses to the Drosophila Host Underlie their Evolutionary Diversification
Authors: Senti, K.-A.; Handler, D.; Rafanel, B.; Kosiol, C.; Schloetterer, C.; Brennecke, J.
Score: 50.6, Published: 2023-08-04 DOI: 10.1101/2023.08.03.551782
Transposable elements profoundly affect the biology and evolution of their hosts, yet their own evolutionary dynamics remain poorly understood. Here, we investigate insect endogenous retroviruses (iERVs), a monophyletic group of LTR retrotransposons that have acquired the trait of infectivity, likely through capture of a Baculovirus envelope gene."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Functional Adaptations of Endogenous Retroviruses to the Drosophila Host Underlie their Evolutionary Diversification\nAuthors: Senti, K.-A.; Handler, D.; Rafanel, B.; Kosiol, C.; Schloetterer, C.; Brennecke, J.\nScore: 50.6, Published: 2023-08-04 DOI: 10.1101/2023.08.03.551782\nTransposable elements profoundly affect the biology and evolution of their hosts, yet their own evolutionary dynamics remain poorly understood. Here, we investigate insect endogenous retroviruses (iERVs), a monophyletic group of LTR retrotransposons that have acquired the trait of infectivity, likely through capture of a Baculovirus envelope gene.",
  "keywords": [
    
  ],
  "articleBody": " Functional Adaptations of Endogenous Retroviruses to the Drosophila Host Underlie their Evolutionary Diversification\nAuthors: Senti, K.-A.; Handler, D.; Rafanel, B.; Kosiol, C.; Schloetterer, C.; Brennecke, J.\nScore: 50.6, Published: 2023-08-04 DOI: 10.1101/2023.08.03.551782\nTransposable elements profoundly affect the biology and evolution of their hosts, yet their own evolutionary dynamics remain poorly understood. Here, we investigate insect endogenous retroviruses (iERVs), a monophyletic group of LTR retrotransposons that have acquired the trait of infectivity, likely through capture of a Baculovirus envelope gene. In Drosophila ovaries, iERVs with functional envelope have adapted their cis-regulatory sequences to be expressed in any somatic cell type, from where they infect the germline. Strikingly, related retroviruses show distinct expression patterns, indicating niche partitioning. In contrast, all non-infectious iERVs that emerged through secondary envelope-loss are specifically expressed in the germline. Co-evolving with iERVs, the genome-protecting piRNA pathway has assimilated iERV promoter and sequence information into piRNA clusters, underscoring the functional significance of iERV expression in somatic niches. We propose that the evolutionary innovation of cell-to-cell infectivity has triggered the adaptive radiation of iERVs through trait diversification and antagonistic virus-host interactions, processes that likely underpin niche-specific expression of endogenous retroviruses in vertebrates as well.\nFruitless mating with the exes: the irreversible parthenogenesis in a stick insect\nAuthors: Nozaki, T.; Chikami, Y.; Yano, K.; Sato, R.; Suetsugu, K.; Kaneko, S.\nScore: 204.2, Published: 2023-07-30 DOI: 10.1101/2023.07.28.550994\nParthenogenetic lineages, common in many animals, have sparked debate about their evolutionary persistence. Even after the loss of male individuals, males can survive on the genomes; in fact, rarely occurring males were described in many asexual lineages. Theoretical predictions suggest that genes related to male sexual traits in parthenogenetic lineages will decay over long evolutionary times free from the pressure of stabilizing selection. However, in many cases, such rare males are potentially functional. In this study, we investigated the functionality of rare males in the Japanese parthenogenetic stick insect, Ramulus mikado. Rarely occurring males in the species exhibited the external and anatomical morphology typical in stick insect males, and vigorously and successfully performed mating behaviors with conspecific females; nevertheless, no paternal genotypes were detected in the offspring; all embryos were genetically identical to their mothers. Our histological observation demonstrated that females received a spermatophore without sperms in their copulatory pouch, and revealed that in a male, spermatogenesis was completely deformed. These results suggest the irreversible clonal reproduction of R. mikado due to compromised male function. Our present study provides the first case of evident dysfunction in rare males of stick insects, illuminating the degenerative evolution of sexual traits in asexual lineages.\nTissue-specific and cis-regulatory changes underlie parallel, adaptive gene expression evolution in house mice\nAuthors: Durkin, S. M.; Ballinger, M. A.; Nachman, M. W.\nScore: 14.4, Published: 2023-08-01 DOI: 10.1101/2023.08.01.551413\nChanges in gene regulation have long been appreciated as a driving force of adaptive evolution, however the relative contributions of cis- and trans-acting changes to gene regulation over short evolutionary timescales remain unclear. Instances of recent, parallel phenotypic evolution provide an opportunity to assess whether parallel patterns are seen at the level of gene expression, and to assess the relative contribution of cis- and trans-changes to gene regulation in the early stages of divergence. Here, we studied gene expression in liver and brown adipose tissue in two wild-derived strains of house mice that independently adapted to cold, northern environments, and we compared them to a strain of house mice from a warm, southern environment. To investigate gene regulatory evolution, we studied expression in parents and allele-specific expression in F1 hybrids of crosses between warm-adapted and cold-adapted strains. First, we found that the different cold-adapted mice showed both unique and shared changes in expression, but that the proportion of shared changes (i.e. parallelism) was greater than expected by chance. Second, we discovered that expression evolution occurred largely at tissue-specific and cis-regulated genes, and that these genes were over-represented in parallel cases of evolution. Finally, we integrated the expression data with scans for selection in natural populations and found substantial parallelism in the two northern populations for genes under selection. Furthermore, selection outliers were associated with cis-regulated genes more than expected by chance; cis-regulated genes under selection influenced phenotypes such as body size, fat content, immune functioning, and skeletal morphology. These results demonstrate that parallel patterns of gene expression in mice that have independently adapted to cold environments are driven largely by tissue-specific and cis-regulatory changes, providing insight into the mechanisms of adaptive gene regulatory evolution at the earliest stages of divergence. Author summaryThe parallel movement of organisms into novel environments provides an opportunity to understand the molecular basis of adaptation and the repeatability of this process. Mutations affecting the expression of genes are known to underlie much of adaptive evolution. Such mutations can arise in cis-(near the gene of interest) or in trans-(at a distant locus), but the relative contribution of these different kinds of changes to adaptation is poorly understood, especially during very recent divergence. Here, we compared evolved gene expression differences between a warm-adapted house mouse strain and two different cold-adapted strains that have independently evolved similar phenotypic traits, such as increased body size and decreased extremity length during the last few hundred years. Using crosses between warm-adapted and cold-adapted mice, we found that mutations acting in a context specific manner (cis-regulatory and tissue-specific changes) predominate expression divergence and are more likely to be involved in parallel evolution. We used population level selection scans in wild animals to identify regions of the genome under selection and combined these findings with the gene expression data to identify candidate genes underlying adaptation to novel environments. Together, our work describes the gene regulatory dynamics of rapid environmental adaptation, and the repeatability of these patterns over multiple instances of adaptation.\nEmergence and spread of the barley net blotch pathogen coincided with crop domestication and cultivation history\nAuthors: Taliadoros, D.; Feurtey, A.; Wyatt, N.; Gladieux, P.; Friesen, T.; Stukenbrock, E. H.\nScore: 13.5, Published: 2023-07-29 DOI: 10.1101/2023.07.28.550921\nFungal pathogens cause devastating disease in crops. Understanding the evolutionary origin of pathogens is essential to the prediction of future disease emergence and the potential of pathogens to disperse. The fungus Pyrenophora teres f. teres causes net form net blotch (NFNB), an economically significant disease of barley. In this study, we have used 104 P. teres f. teres genomes from four continents to explore the population structure and demographic history of the fungal pathogen. We showed that P. teres f. teres is structured into populations that tend to be geographically restricted to different regions. Using Multiple Sequentially Markovian Coalescent and machine learning approaches we demonstrated that the demographic history of the pathogen correlates with the history of barley, highlighting the importance of human migration and trade in spreading the pathogen. Exploring signatures of natural selection, we identified several population-specific selective sweeps that colocalized with genomic regions enriched in putative virulence genes, and loci previously identified as determinants of virulence specificities by quantitative trait locus analyses. This reflects rapid adaptation to local hosts and environmental conditions of P. teres f. teres as it spread with barley. Our research highlights how human activities can contribute to the spread of pathogens that significantly impact the productivity of field crops.\nLocal adaptation to an altitudinal gradient: the interplay between constitutive trait variation and phenotypic plasticity in Mimulus laciniatus\nAuthors: Syrotchen, J. M.; Ferris, K. G.\nScore: 12.5, Published: 2023-08-02 DOI: 10.1101/2023.08.02.551729\nOrganisms can adapt to environmental heterogeneity through two mechanisms: (1) constitutive expression of population genetic variation or (2) phenotypic plasticity. We investigated whether patterns of plastic and constitutively expressed traits along elevational and latitudinal clines in a North American endemic plant, Mimulus laciniatus, were consistent with local adaptation. We grew inbred lines of M. laciniatus from across the species range to measure constitutive and plastic trait expression in several traits previously shown to be involved in adaptation to M. laciniatuss rocky outcrop microhabitat: flowering time, size-related traits, and leaf shape. We then tested for a relationship between variation in trait expression and population elevation and latitude, and examined correlation in trait expression between plastic and constitutive traits. We found that population variation in constitutive leaf lobing is associated with latitude, and identified multiple traits that show patterns consistent with local adaptation to elevation: critical photoperiod, flowering time, flower size, leaf lobing, and leaf lobing plasticity. These trends occur along multiple geographically independent altitudinal clines indicating that selection is a more likely cause of this pattern than gene flow among nearby populations with similar trait values. We did not find a correlation between constitutive and plastic leaf lobing at the individual genotype level, suggesting that while both traits appear to be adaptive, they operate under independent genetic controls. This indicates that both having more highly lobed leaves and greater leaf shape plasticity are likely adaptive at high elevation within M. laciniatus. Our data strongly suggest that traits known to be under divergent selection between M. laciniatus and close relative Mimulus guttatus are also under locally varying selection within M. laciniatus.\nGenomic analysis supports Cape Lion population connectivity prior to colonial eradication and extinction\nAuthors: de Flamingh, A.; Gnoske, T. P.; Rivera-Colon, A. G.; Simeonovski, V. A.; Kerbis Peterhans, J. C.; Yamaguchi, N.; Witt, K. E.; Catchen, J. M.; Roca, A.; Malhi, R. S.\nScore: 6.9, Published: 2023-08-02 DOI: 10.1101/2023.07.31.551389\nExtinct Cape lions (Panthera leo melanochaitus) formerly ranged throughout the grassland plains of the \"Cape Flats\" in what is today known as the Western Cape Province, South Africa. Cape lions were likely eradicated because of overhunting and habitat loss after European colonization. European naturalists originally described Cape lions as \"Black-maned lions\" and claimed that they were phenotypically distinct. However, other depictions and historical descriptions of lions from the Cape report mixed or light coloration and without black or extensively developed manes. These findings suggest that, rather than forming a distinct population, Cape lions may have had phenotypic and genotypic variation similar to other African lions. Here we investigate extinct Cape lion genome characteristics, population dynamics and genetic distinctiveness prior to their extinction. We generated genomic data from two historic Cape lions to compare to 118 existing high-coverage mitogenomes, and low-coverage nuclear genomes of 53 lions from 13 African countries. We show that, before their eradication, lions from the Cape Flats had diverse mitogenomes and nuclear genomes that clustered with lions from both southern and eastern Africa. Cape lions had high genome-wide heterozygosity and low inbreeding coefficients, indicating that populations in the Cape Flats went extinct so rapidly that genomic effects associated with long-term small population size and isolation were not detectable. Our findings do not support the characterization of Cape lions as phylogeographically distinct, as originally put forth by some European naturalists, but rather highlights how alternative knowledge-systems, e.g., Indigenous perspectives, could potentially further inform interpretations of species life histories.\nDeep estimation of the intensity and timing of selection from ancient genomes\nAuthors: Laval, G.; Patin, E.; Quintana-Murci, L.; Kerner, G.\nScore: 10.2, Published: 2023-07-28 DOI: 10.1101/2023.07.27.550703\nLeveraging past allele frequencies has proven to be key to identify the impact of natural selection across time. However, this approach often suffers from imprecise estimations of the intensity (s) and timing (T) of selection particularly when ancient samples are scarce in specific epochs. Here, we aimed at bypassing the computation of past allele frequencies by implementing new convolutional neural networks (CNNs) algorithms that directly use ancient genotypes sampled across time to refine the estimations of selection parameters. Using computer simulations, we first show that genotype-based CNNs consistently outperform an approximate Bayesian computation (ABC) approach based on past allele frequency trajectories, regardless of the selection model assumed and of the amount of ancient genotypes available. When applying this method to empirical data from modern and ancient Europeans, we confirmed the reported excess of selection events in post-Neolithic Europe, independently of the continental subregion studied. Furthermore, we substantially refined the ABC-based estimations of s and T for a set of positively-and negatively-selected variants recently identified, including iconic cases of positive selection and experimentally validated disease-risk variants. Thanks to our CNN predictions we provide support to the history of recent and strong selection in northern Europe associated to the Black Death pandemic and confirm the heavy burden recently imposed by tuberculosis in Europe. These findings collectively support that detecting the imprints of natural selection on ancient genomes are crucial for unraveling the past history of severe human diseases.\nArtificial selection optimizes pollutant-degrading bacterial communities\nAuthors: Arias-Sanchez, F. I.; Vessman, B.; Haym, A.; Alberti, G.; Mitri, S.\nScore: 5.3, Published: 2023-07-27 DOI: 10.1101/2023.07.27.550627\nArtificial selection is a promising way to improve microbial community functions, but previous experiments have only shown moderate success. Here, we experimentally evaluate a new method that was inspired by genetic algorithms to artificially select small bacterial communities of known species composition based on their degradation of an industrial pollutant. Starting from 29 randomly generated four-species communities, we repeatedly grew communities for four days, selected the 10 best-degrading communities, and rearranged them into 29 new communities with species compositions that resembled those of the most successful ones. The best community after 18 such rounds of selection degraded the pollutant better than the best community in the first round. It featured member species that degrade well, species that degrade badly alone but improve community degradation, and free-rider species that did not contribute to community degradation. Most species in the evolved communities did not differ significantly from their ancestors, suggesting that genetic evolution plays a small role at this time scale. These experiments show that artificial selection on microbial communities can work in principle, and inform on how to improve future experiments.\nThe evolution of borneol repellency in culicine mosquitoes\nAuthors: Vainer, Y.; Wang, Y.; Huff, R. M.; Ghaninia, M.; Coutinho-Abreu, I. V.; Sar-Shalom, E.; Ruiz, C.; Perets, D.; Yakir, E.; Rajamanickam, D.; Warburg, A.; Papathanos, P. A.; Akbari, O.; Ignell, R.; Riffell, J. A.; Pitts, J.; Bohbot, J. D.\nScore: 4.8, Published: 2023-08-04 DOI: 10.1101/2023.08.01.548337\nInsects have developed remarkable adaptations to effectively interact with plant secondary metabolites and utilize them as cues to identify suitable hosts. Consequently, humans have used aromatic plants for centuries to repel mosquitoes. The repellent effects of plant volatile compounds are mediated through olfactory structures present in the antennae, and maxillary palps of mosquitoes. Mosquito maxillary palps contain capitate-peg sensilla, which house three olfactory sensory neurons, of which two are mainly tuned to either carbon dioxide or octenol - two animal host odorants. However, the third neuron, which expresses the OR49 receptor, has remained without a known ecologically-relevant odorant since its initial discovery. In this study, we used odorant mixtures and terpenoid-rich Cannabis essential oils to investigate the activation of OR49. Our results demonstrate that two monoterpenoids, borneol and camphor, selectively activate OR49, and OR9-expressing neurons, as well as the MD3 glomerulus in the antennal lobe. We confirm that borneol repels female mosquitoes, and knocking out the gene encoding the OR49 receptor suppresses the response of the corresponding olfactory sensory neuron. Importantly, this molecular mechanism of action is conserved across culicine mosquito species, underscoring its significance in their olfactory systems.\nGenomic data suggest parallel dental vestigialization within the xenarthran radiation\nAuthors: Emerling, C. A.; Gibb, G. C.; Tilak, M.-K.; Hughes, J. J.; Kuch, M.; Duggan, A. T.; Poinar, H. N.; Nachman, M. W.; Delsuc, F.\nScore: 15.8, Published: 2023-08-08 DOI: 10.1101/2022.12.09.519446\nThe recent influx of genomic data has provided greater insights into the molecular basis for regressive evolution, or vestigialization, through gene loss and pseudogenization. As such, the analysis of gene degradation patterns has the potential to provide insights into the evolutionary history of regressed anatomical traits. We specifically applied these principles to the xenarthran radiation (anteaters, sloths, armadillos), which is characterized by taxa with a gradation in regressed dental phenotypes. Whether the pattern among extant xenarthrans is due to an ancient and gradual decay of dental morphology or occurred repeatedly in parallel is unknown. We tested these competing hypotheses by examining 11 core dental genes in most living species of Xenarthra, characterizing shared inactivating mutations and patterns of relaxed selection during their radiation. Here we report evidence of independent and distinct events of dental gene loss in the major xenarthran subclades. First, we found strong evidence of complete enamel loss in the common ancestor of sloths and anteaters, suggested by the inactivation of five enamel-associated genes (AMELX, AMTN, MMP20, ENAM, ACP4). Next, whereas dental regression appears to have halted in sloths, presumably a critical event that ultimately permitted adaptation to an herbivorous lifestyle, anteaters continued losing genes on the path towards complete tooth loss. Echoes of this event are recorded in the genomes of all living anteaters, being marked by a 2-bp deletion in a gene critical for dentinogenesis (DSPP) and a putative shared 1-bp insertion in a gene linked to tooth retention (ODAPH). By contrast, in the two major armadillo clades, genes pertaining to the dento-gingival junction and amelogenesis appear to have been independently inactivated prior to losing all or some enamel. These genomic data provide evidence for multiple pathways and rates of anatomical regression, and underscore the utility of using pseudogenes to reconstruct evolutionary history when fossils are sparse.\n",
  "wordCount" : "2845",
  "inLanguage": "en",
  "datePublished": "2023-08-09T10:39:33Z",
  "dateModified": "2023-08-09T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on August 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551782">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551782" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551782">
        <p class="paperTitle">Functional Adaptations of Endogenous Retroviruses to the Drosophila Host Underlie their Evolutionary Diversification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551782" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551782" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Senti, K.-A.; Handler, D.; Rafanel, B.; Kosiol, C.; Schloetterer, C.; Brennecke, J.</p>
        <p class="info">Score: 50.6, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551782' target='https://doi.org/10.1101/2023.08.03.551782'> 10.1101/2023.08.03.551782</a></p>
        <p class="abstract">Transposable elements profoundly affect the biology and evolution of their hosts, yet their own evolutionary dynamics remain poorly understood. Here, we investigate insect endogenous retroviruses (iERVs), a monophyletic group of LTR retrotransposons that have acquired the trait of infectivity, likely through capture of a Baculovirus envelope gene. In Drosophila ovaries, iERVs with functional envelope have adapted their cis-regulatory sequences to be expressed in any somatic cell type, from where they infect the germline. Strikingly, related retroviruses show distinct expression patterns, indicating niche partitioning. In contrast, all non-infectious iERVs that emerged through secondary envelope-loss are specifically expressed in the germline. Co-evolving with iERVs, the genome-protecting piRNA pathway has assimilated iERV promoter and sequence information into piRNA clusters, underscoring the functional significance of iERV expression in somatic niches. We propose that the evolutionary innovation of cell-to-cell infectivity has triggered the adaptive radiation of iERVs through trait diversification and antagonistic virus-host interactions, processes that likely underpin niche-specific expression of endogenous retroviruses in vertebrates as well.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550994">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550994" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550994">
        <p class="paperTitle">Fruitless mating with the exes: the irreversible parthenogenesis in a stick insect</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550994" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550994" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nozaki, T.; Chikami, Y.; Yano, K.; Sato, R.; Suetsugu, K.; Kaneko, S.</p>
        <p class="info">Score: 204.2, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550994' target='https://doi.org/10.1101/2023.07.28.550994'> 10.1101/2023.07.28.550994</a></p>
        <p class="abstract">Parthenogenetic lineages, common in many animals, have sparked debate about their evolutionary persistence. Even after the loss of male individuals, males can survive on the genomes; in fact, rarely occurring males were described in many asexual lineages. Theoretical predictions suggest that genes related to male sexual traits in parthenogenetic lineages will decay over long evolutionary times free from the pressure of stabilizing selection. However, in many cases, such rare males are potentially functional. In this study, we investigated the functionality of rare males in the Japanese parthenogenetic stick insect, Ramulus mikado. Rarely occurring males in the species exhibited the external and anatomical morphology typical in stick insect males, and vigorously and successfully performed mating behaviors with conspecific females; nevertheless, no paternal genotypes were detected in the offspring; all embryos were genetically identical to their mothers. Our histological observation demonstrated that females received a spermatophore without sperms in their copulatory pouch, and revealed that in a male, spermatogenesis was completely deformed. These results suggest the irreversible clonal reproduction of R. mikado due to compromised male function. Our present study provides the first case of evident dysfunction in rare males of stick insects, illuminating the degenerative evolution of sexual traits in asexual lineages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551413">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551413" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551413">
        <p class="paperTitle">Tissue-specific and cis-regulatory changes underlie parallel, adaptive gene expression evolution in house mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551413" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551413" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Durkin, S. M.; Ballinger, M. A.; Nachman, M. W.</p>
        <p class="info">Score: 14.4, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551413' target='https://doi.org/10.1101/2023.08.01.551413'> 10.1101/2023.08.01.551413</a></p>
        <p class="abstract">Changes in gene regulation have long been appreciated as a driving force of adaptive evolution, however the relative contributions of cis- and trans-acting changes to gene regulation over short evolutionary timescales remain unclear. Instances of recent, parallel phenotypic evolution provide an opportunity to assess whether parallel patterns are seen at the level of gene expression, and to assess the relative contribution of cis- and trans-changes to gene regulation in the early stages of divergence. Here, we studied gene expression in liver and brown adipose tissue in two wild-derived strains of house mice that independently adapted to cold, northern environments, and we compared them to a strain of house mice from a warm, southern environment. To investigate gene regulatory evolution, we studied expression in parents and allele-specific expression in F1 hybrids of crosses between warm-adapted and cold-adapted strains. First, we found that the different cold-adapted mice showed both unique and shared changes in expression, but that the proportion of shared changes (i.e. parallelism) was greater than expected by chance. Second, we discovered that expression evolution occurred largely at tissue-specific and cis-regulated genes, and that these genes were over-represented in parallel cases of evolution. Finally, we integrated the expression data with scans for selection in natural populations and found substantial parallelism in the two northern populations for genes under selection. Furthermore, selection outliers were associated with cis-regulated genes more than expected by chance; cis-regulated genes under selection influenced phenotypes such as body size, fat content, immune functioning, and skeletal morphology. These results demonstrate that parallel patterns of gene expression in mice that have independently adapted to cold environments are driven largely by tissue-specific and cis-regulatory changes, providing insight into the mechanisms of adaptive gene regulatory evolution at the earliest stages of divergence.

Author summaryThe parallel movement of organisms into novel environments provides an opportunity to understand the molecular basis of adaptation and the repeatability of this process. Mutations affecting the expression of genes are known to underlie much of adaptive evolution. Such mutations can arise in cis-(near the gene of interest) or in trans-(at a distant locus), but the relative contribution of these different kinds of changes to adaptation is poorly understood, especially during very recent divergence. Here, we compared evolved gene expression differences between a warm-adapted house mouse strain and two different cold-adapted strains that have independently evolved similar phenotypic traits, such as increased body size and decreased extremity length during the last few hundred years. Using crosses between warm-adapted and cold-adapted mice, we found that mutations acting in a context specific manner (cis-regulatory and tissue-specific changes) predominate expression divergence and are more likely to be involved in parallel evolution. We used population level selection scans in wild animals to identify regions of the genome under selection and combined these findings with the gene expression data to identify candidate genes underlying adaptation to novel environments. Together, our work describes the gene regulatory dynamics of rapid environmental adaptation, and the repeatability of these patterns over multiple instances of adaptation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550921">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550921" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550921">
        <p class="paperTitle">Emergence and spread of the barley net blotch pathogen coincided with crop domestication and cultivation history</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550921" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550921" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taliadoros, D.; Feurtey, A.; Wyatt, N.; Gladieux, P.; Friesen, T.; Stukenbrock, E. H.</p>
        <p class="info">Score: 13.5, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550921' target='https://doi.org/10.1101/2023.07.28.550921'> 10.1101/2023.07.28.550921</a></p>
        <p class="abstract">Fungal pathogens cause devastating disease in crops. Understanding the evolutionary origin of pathogens is essential to the prediction of future disease emergence and the potential of pathogens to disperse. The fungus Pyrenophora teres f. teres causes net form net blotch (NFNB), an economically significant disease of barley. In this study, we have used 104 P. teres f. teres genomes from four continents to explore the population structure and demographic history of the fungal pathogen. We showed that P. teres f. teres is structured into populations that tend to be geographically restricted to different regions. Using Multiple Sequentially Markovian Coalescent and machine learning approaches we demonstrated that the demographic history of the pathogen correlates with the history of barley, highlighting the importance of human migration and trade in spreading the pathogen. Exploring signatures of natural selection, we identified several population-specific selective sweeps that colocalized with genomic regions enriched in putative virulence genes, and loci previously identified as determinants of virulence specificities by quantitative trait locus analyses. This reflects rapid adaptation to local hosts and environmental conditions of P. teres f. teres as it spread with barley. Our research highlights how human activities can contribute to the spread of pathogens that significantly impact the productivity of field crops.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551729">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551729" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551729">
        <p class="paperTitle">Local adaptation to an altitudinal gradient: the interplay between constitutive trait variation and phenotypic plasticity in Mimulus laciniatus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551729" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551729" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Syrotchen, J. M.; Ferris, K. G.</p>
        <p class="info">Score: 12.5, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551729' target='https://doi.org/10.1101/2023.08.02.551729'> 10.1101/2023.08.02.551729</a></p>
        <p class="abstract">Organisms can adapt to environmental heterogeneity through two mechanisms: (1) constitutive expression of population genetic variation or (2) phenotypic plasticity. We investigated whether patterns of plastic and constitutively expressed traits along elevational and latitudinal clines in a North American endemic plant, Mimulus laciniatus, were consistent with local adaptation. We grew inbred lines of M. laciniatus from across the species range to measure constitutive and plastic trait expression in several traits previously shown to be involved in adaptation to M. laciniatuss rocky outcrop microhabitat: flowering time, size-related traits, and leaf shape. We then tested for a relationship between variation in trait expression and population elevation and latitude, and examined correlation in trait expression between plastic and constitutive traits. We found that population variation in constitutive leaf lobing is associated with latitude, and identified multiple traits that show patterns consistent with local adaptation to elevation: critical photoperiod, flowering time, flower size, leaf lobing, and leaf lobing plasticity. These trends occur along multiple geographically independent altitudinal clines indicating that selection is a more likely cause of this pattern than gene flow among nearby populations with similar trait values. We did not find a correlation between constitutive and plastic leaf lobing at the individual genotype level, suggesting that while both traits appear to be adaptive, they operate under independent genetic controls. This indicates that both having more highly lobed leaves and greater leaf shape plasticity are likely adaptive at high elevation within M. laciniatus. Our data strongly suggest that traits known to be under divergent selection between M. laciniatus and close relative Mimulus guttatus are also under locally varying selection within M. laciniatus.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551389">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551389" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551389">
        <p class="paperTitle">Genomic analysis supports Cape Lion population connectivity prior to colonial eradication and extinction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551389" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551389" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de Flamingh, A.; Gnoske, T. P.; Rivera-Colon, A. G.; Simeonovski, V. A.; Kerbis Peterhans, J. C.; Yamaguchi, N.; Witt, K. E.; Catchen, J. M.; Roca, A.; Malhi, R. S.</p>
        <p class="info">Score: 6.9, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551389' target='https://doi.org/10.1101/2023.07.31.551389'> 10.1101/2023.07.31.551389</a></p>
        <p class="abstract">Extinct Cape lions (Panthera leo melanochaitus) formerly ranged throughout the grassland plains of the &#34;Cape Flats&#34; in what is today known as the Western Cape Province, South Africa. Cape lions were likely eradicated because of overhunting and habitat loss after European colonization. European naturalists originally described Cape lions as &#34;Black-maned lions&#34; and claimed that they were phenotypically distinct. However, other depictions and historical descriptions of lions from the Cape report mixed or light coloration and without black or extensively developed manes. These findings suggest that, rather than forming a distinct population, Cape lions may have had phenotypic and genotypic variation similar to other African lions. Here we investigate extinct Cape lion genome characteristics, population dynamics and genetic distinctiveness prior to their extinction. We generated genomic data from two historic Cape lions to compare to 118 existing high-coverage mitogenomes, and low-coverage nuclear genomes of 53 lions from 13 African countries. We show that, before their eradication, lions from the Cape Flats had diverse mitogenomes and nuclear genomes that clustered with lions from both southern and eastern Africa. Cape lions had high genome-wide heterozygosity and low inbreeding coefficients, indicating that populations in the Cape Flats went extinct so rapidly that genomic effects associated with long-term small population size and isolation were not detectable. Our findings do not support the characterization of Cape lions as phylogeographically distinct, as originally put forth by some European naturalists, but rather highlights how alternative knowledge-systems, e.g., Indigenous perspectives, could potentially further inform interpretations of species life histories.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550703">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550703" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550703">
        <p class="paperTitle">Deep estimation of the intensity and timing of selection from ancient genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550703" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550703" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Laval, G.; Patin, E.; Quintana-Murci, L.; Kerner, G.</p>
        <p class="info">Score: 10.2, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550703' target='https://doi.org/10.1101/2023.07.27.550703'> 10.1101/2023.07.27.550703</a></p>
        <p class="abstract">Leveraging past allele frequencies has proven to be key to identify the impact of natural selection across time. However, this approach often suffers from imprecise estimations of the intensity (s) and timing (T) of selection particularly when ancient samples are scarce in specific epochs. Here, we aimed at bypassing the computation of past allele frequencies by implementing new convolutional neural networks (CNNs) algorithms that directly use ancient genotypes sampled across time to refine the estimations of selection parameters. Using computer simulations, we first show that genotype-based CNNs consistently outperform an approximate Bayesian computation (ABC) approach based on past allele frequency trajectories, regardless of the selection model assumed and of the amount of ancient genotypes available. When applying this method to empirical data from modern and ancient Europeans, we confirmed the reported excess of selection events in post-Neolithic Europe, independently of the continental subregion studied. Furthermore, we substantially refined the ABC-based estimations of s and T for a set of positively-and negatively-selected variants recently identified, including iconic cases of positive selection and experimentally validated disease-risk variants. Thanks to our CNN predictions we provide support to the history of recent and strong selection in northern Europe associated to the Black Death pandemic and confirm the heavy burden recently imposed by tuberculosis in Europe. These findings collectively support that detecting the imprints of natural selection on ancient genomes are crucial for unraveling the past history of severe human diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550627">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550627" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550627">
        <p class="paperTitle">Artificial selection optimizes pollutant-degrading bacterial communities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550627" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550627" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arias-Sanchez, F. I.; Vessman, B.; Haym, A.; Alberti, G.; Mitri, S.</p>
        <p class="info">Score: 5.3, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550627' target='https://doi.org/10.1101/2023.07.27.550627'> 10.1101/2023.07.27.550627</a></p>
        <p class="abstract">Artificial selection is a promising way to improve microbial community functions, but previous experiments have only shown moderate success. Here, we experimentally evaluate a new method that was inspired by genetic algorithms to artificially select small bacterial communities of known species composition based on their degradation of an industrial pollutant. Starting from 29 randomly generated four-species communities, we repeatedly grew communities for four days, selected the 10 best-degrading communities, and rearranged them into 29 new communities with species compositions that resembled those of the most successful ones. The best community after 18 such rounds of selection degraded the pollutant better than the best community in the first round. It featured member species that degrade well, species that degrade badly alone but improve community degradation, and free-rider species that did not contribute to community degradation. Most species in the evolved communities did not differ significantly from their ancestors, suggesting that genetic evolution plays a small role at this time scale. These experiments show that artificial selection on microbial communities can work in principle, and inform on how to improve future experiments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.548337">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.548337" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.548337">
        <p class="paperTitle">The evolution of borneol repellency in culicine mosquitoes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.548337" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.548337" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vainer, Y.; Wang, Y.; Huff, R. M.; Ghaninia, M.; Coutinho-Abreu, I. V.; Sar-Shalom, E.; Ruiz, C.; Perets, D.; Yakir, E.; Rajamanickam, D.; Warburg, A.; Papathanos, P. A.; Akbari, O.; Ignell, R.; Riffell, J. A.; Pitts, J.; Bohbot, J. D.</p>
        <p class="info">Score: 4.8, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.548337' target='https://doi.org/10.1101/2023.08.01.548337'> 10.1101/2023.08.01.548337</a></p>
        <p class="abstract">Insects have developed remarkable adaptations to effectively interact with plant secondary metabolites and utilize them as cues to identify suitable hosts. Consequently, humans have used aromatic plants for centuries to repel mosquitoes. The repellent effects of plant volatile compounds are mediated through olfactory structures present in the antennae, and maxillary palps of mosquitoes. Mosquito maxillary palps contain capitate-peg sensilla, which house three olfactory sensory neurons, of which two are mainly tuned to either carbon dioxide or octenol - two animal host odorants. However, the third neuron, which expresses the OR49 receptor, has remained without a known ecologically-relevant odorant since its initial discovery. In this study, we used odorant mixtures and terpenoid-rich Cannabis essential oils to investigate the activation of OR49. Our results demonstrate that two monoterpenoids, borneol and camphor, selectively activate OR49, and OR9-expressing neurons, as well as the MD3 glomerulus in the antennal lobe. We confirm that borneol repels female mosquitoes, and knocking out the gene encoding the OR49 receptor suppresses the response of the corresponding olfactory sensory neuron. Importantly, this molecular mechanism of action is conserved across culicine mosquito species, underscoring its significance in their olfactory systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.12.09.519446">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.12.09.519446" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.12.09.519446">
        <p class="paperTitle">Genomic data suggest parallel dental vestigialization within the xenarthran radiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.12.09.519446" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.12.09.519446" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Emerling, C. A.; Gibb, G. C.; Tilak, M.-K.; Hughes, J. J.; Kuch, M.; Duggan, A. T.; Poinar, H. N.; Nachman, M. W.; Delsuc, F.</p>
        <p class="info">Score: 15.8, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.12.09.519446' target='https://doi.org/10.1101/2022.12.09.519446'> 10.1101/2022.12.09.519446</a></p>
        <p class="abstract">The recent influx of genomic data has provided greater insights into the molecular basis for regressive evolution, or vestigialization, through gene loss and pseudogenization. As such, the analysis of gene degradation patterns has the potential to provide insights into the evolutionary history of regressed anatomical traits. We specifically applied these principles to the xenarthran radiation (anteaters, sloths, armadillos), which is characterized by taxa with a gradation in regressed dental phenotypes. Whether the pattern among extant xenarthrans is due to an ancient and gradual decay of dental morphology or occurred repeatedly in parallel is unknown. We tested these competing hypotheses by examining 11 core dental genes in most living species of Xenarthra, characterizing shared inactivating mutations and patterns of relaxed selection during their radiation. Here we report evidence of independent and distinct events of dental gene loss in the major xenarthran subclades. First, we found strong evidence of complete enamel loss in the common ancestor of sloths and anteaters, suggested by the inactivation of five enamel-associated genes (AMELX, AMTN, MMP20, ENAM, ACP4). Next, whereas dental regression appears to have halted in sloths, presumably a critical event that ultimately permitted adaptation to an herbivorous lifestyle, anteaters continued losing genes on the path towards complete tooth loss. Echoes of this event are recorded in the genomes of all living anteaters, being marked by a 2-bp deletion in a gene critical for dentinogenesis (DSPP) and a putative shared 1-bp insertion in a gene linked to tooth retention (ODAPH). By contrast, in the two major armadillo clades, genes pertaining to the dento-gingival junction and amelogenesis appear to have been independently inactivated prior to losing all or some enamel. These genomic data provide evidence for multiple pathways and rates of anatomical regression, and underscore the utility of using pseudogenes to reconstruct evolutionary history when fossils are sparse.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
