<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Evidence for ecological tuning of novel anuran biofluorescent signals
Authors: Whitcher, C.; Ron, S. R.; Ayala-Varela, F.; Crawford, A.; Herrera-Alva, V.; Castillo-Urbina, E.; Grazziotin, F.; Bowman, R. M.; Lemmon, A. R.; Moriarty Lemmon, E.
Score: 51.4, Published: 2023-07-28 DOI: 10.1101/2023.07.25.550432
Our study assesses the variability of amphibian biofluorescence and provides insight into its potential functions and role in anuran evolution. Via a field survey across South America, we discovered and documented patterns of biofluorescence in tropical amphibians.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Evidence for ecological tuning of novel anuran biofluorescent signals
Authors: Whitcher, C.; Ron, S. R.; Ayala-Varela, F.; Crawford, A.; Herrera-Alva, V.; Castillo-Urbina, E.; Grazziotin, F.; Bowman, R. M.; Lemmon, A. R.; Moriarty Lemmon, E.
Score: 51.4, Published: 2023-07-28 DOI: 10.1101/2023.07.25.550432
Our study assesses the variability of amphibian biofluorescence and provides insight into its potential functions and role in anuran evolution. Via a field survey across South America, we discovered and documented patterns of biofluorescence in tropical amphibians." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-27T10:37:02+00:00" />
<meta property="article:modified_time" content="2023-08-27T10:37:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Evidence for ecological tuning of novel anuran biofluorescent signals
Authors: Whitcher, C.; Ron, S. R.; Ayala-Varela, F.; Crawford, A.; Herrera-Alva, V.; Castillo-Urbina, E.; Grazziotin, F.; Bowman, R. M.; Lemmon, A. R.; Moriarty Lemmon, E.
Score: 51.4, Published: 2023-07-28 DOI: 10.1101/2023.07.25.550432
Our study assesses the variability of amphibian biofluorescence and provides insight into its potential functions and role in anuran evolution. Via a field survey across South America, we discovered and documented patterns of biofluorescence in tropical amphibians."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Evidence for ecological tuning of novel anuran biofluorescent signals\nAuthors: Whitcher, C.; Ron, S. R.; Ayala-Varela, F.; Crawford, A.; Herrera-Alva, V.; Castillo-Urbina, E.; Grazziotin, F.; Bowman, R. M.; Lemmon, A. R.; Moriarty Lemmon, E.\nScore: 51.4, Published: 2023-07-28 DOI: 10.1101/2023.07.25.550432\nOur study assesses the variability of amphibian biofluorescence and provides insight into its potential functions and role in anuran evolution. Via a field survey across South America, we discovered and documented patterns of biofluorescence in tropical amphibians.",
  "keywords": [
    
  ],
  "articleBody": " Evidence for ecological tuning of novel anuran biofluorescent signals\nAuthors: Whitcher, C.; Ron, S. R.; Ayala-Varela, F.; Crawford, A.; Herrera-Alva, V.; Castillo-Urbina, E.; Grazziotin, F.; Bowman, R. M.; Lemmon, A. R.; Moriarty Lemmon, E.\nScore: 51.4, Published: 2023-07-28 DOI: 10.1101/2023.07.25.550432\nOur study assesses the variability of amphibian biofluorescence and provides insight into its potential functions and role in anuran evolution. Via a field survey across South America, we discovered and documented patterns of biofluorescence in tropical amphibians. We more than tripled the number of species that have been tested for this trait and added representatives from previously untested anuran families. We found evidence for ecological tuning (i.e., the specific adaptation of a signal to the environment in which it is received) of the novel anuran biofluorescent signals. Across groups, the fluorescence excitation peak matches the wavelengths most available at twilight, the light environment in which most frog species are active. Additionally, biofluorescence emission spans both wavelengths of low availability in twilight and the peak sensitivity of green-sensitive rods in the anuran eye, likely increasing contrast of this signal for a conspecific receiver. With evidence of tuning to the ecology and sensory systems of frogs, our results suggest frog biofluorescence is likely functioning in anuran communication.\nRapid turnover of centromeric DNA reveals signatures of genetic conflict in Drosophila\nAuthors: Courret, C.; Hemmer, L.; Wei, X.; Patel, P. D.; Santinello, B.; Geng, X.; Chang, C.-H.; Mellone, B.; Larracuente, A. M.\nScore: 15.7, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554357\nDespite their essential function in chromosome segregation, centromeres reside in rapidly evolving, repeat-rich genomic regions. Across organisms, centromeres are rich in selfish genetic elements like transposable elements and satellite DNAs that can bias their transmission through meiosis, but still need to contribute to centromere function. To gain insight into the balance between conflict and cooperation at centromeric DNA, we take advantage of the close evolutionary relationships within the Drosophila simulans clade - D. simulans, D. sechellia, and D. mauritiana - and their relative, D. melanogaster. We discovered dramatic centromere reorganization involving recurrent shifts between retroelements and satellite DNAs over short evolutionary timescales. We also reveal the recent origin (\u003c240 Kya) of truly telocentric chromosomes in D. sechellia, where X and dot centromeres now sit on telomere-specific retroelements. This rapid centromere turnover is consistent with genetic conflicts in the female germline and has implications for centromeric DNA function and karyotype evolution.\nAn effector deletion leads to the breakdown of partial grapevine resistance to downy mildew\nAuthors: Paineau, M.; Minio, A.; Mestre, P.; Fabre, F.; Mazet, I. D.; Couture, C.; Legeai, F.; Dumartinet, T.; Cantu, D.; Delmotte, F.\nScore: 17.6, Published: 2023-08-18 DOI: 10.1101/2023.08.17.553663\nGrapevine downy mildew, caused by the oomycete Plasmopara viticola, is a globally destructive disease that particularly affect the Eurasian wine grape V. vinifera. While genetically resistant varieties are becoming more accessible, populations of P. viticola are demonstrating rapid adaptability, successfully over-coming these resistances. Here we aimed to identify the avirulence genes involved in the interaction with the Rpv3.1-mediated resistance in grapevine. We sequenced the full genome of 136 P. viticola strains sampled in a natural population of Bordeaux (France) and characterized their development on both resistant and sensitive cultivars. The genome-wide association study allowed the identification of a genomic region associated with the breakdown of Rpv3.1 grapevine resistance (avrRpv3.1 locus). A diploid-aware reassembly of the P. viticola INRA-Pv221 genome allowed to detect structural variations in this locus, including a major 30 Kbp deletion. At the avrRpv3.1 locus, virulent P. viticola strains presented deletion on both haplotypes indicating that avirulence is recessive. The deletion involves two closely-related genes that encode proteins containing 800-900 amino acids with a signal peptide. The structure of the predicted proteins contains repeats of the LWY-fold structural modules, typical of oomycete effectors. Moreover, when these proteins were transiently expressed, they induced cell death in grapevines carrying Rpv3.1 resistance, confirming their avirulence nature. The first description of candidate effectors of P. viticola involved in the interaction with resistance gene provides valuable insights into the genetic mechanisms that enable P. viticola to adapt to grapevine resistance, laying a foundation for developing strategies to manage this damaging crop pathogen.\nGeo-genomic predictors of genetree heterogeneity in an Amazonian bird (Thamnophilus aethiops)\nAuthors: Musher, L. J.; Del Rio, G.; Marcondes, R. S.; Brumfield, R. T.; Bravo, G.; Thom, G.\nScore: 9.0, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554279\nCan knowledge about genome architecture inform biogeographic and phylogenetic inference? Selection, drift, recombination, and gene flow interact to produce a genomic landscape of divergence wherein patterns of differentiation and genealogy vary nonrandomly across the genomes of diverging populations. For instance, genealogical patterns that arise due to gene flow should be more likely to occur on smaller chromosomes, which experience high recombination, whereas those tracking histories of geographic isolation (reduced gene flow caused by a barrier) and divergence should be more likely to occur on larger and sex chromosomes. In Amazonia, populations of many bird species diverge and introgress across rivers, resulting in reticulated genomic signals. Herein, we used reduced representation genomic data to disentangle the evolutionary history of four populations of an Amazonian antbird, Thamnophilus aethiops, whose biogeographic history was associated with the dynamic evolution of the Madeira River Basin. Specifically, we evaluate whether a large river capture event ca. 200 kya, gave rise to reticulated genealogies in the genome by making spatially explicit predictions about isolation and gene flow based on knowledge about genomic processes. We first estimated chromosome-level phylogenies and recovered two primary topologies across the genome. The first topology (T1) was most consistent with predictions about population divergence, and was recovered for the Z chromosome. The second (T2), was consistent with predictions about gene flow upon secondary contact. To evaluate support for these topologies, we trained a convolutional neural network to classify our data into alternative diversification models and estimate demographic parameters. The best-fit model was concordant with T1 and included gene flow between non-sister taxa. Finally, we modeled levels of divergence and introgression as functions of chromosome length, and found that smaller chromosomes experienced higher gene flow. Given that (1) gene-trees supporting T2 were more likely to occur on smaller chromosomes and (2) we found lower levels of introgression on larger chromosomes (and especially the Z-chromosome), we argue that T1 represents the history of population divergence across rivers and T2 the history of secondary contact due to barrier loss. Our results suggest that a significant portion of genomic heterogeneity arises due to extrinsic biogeographic processes such as river capture interacting with intrinsic processes associated with genome architecture. Future biogeographic studies would benefit from accounting for genomic processes, as different parts of the genome reveal contrasting, albeit complementary histories, all of which are relevant for disentangling the intricate geogenomic mechanisms of biotic diversification.\nOne Method to Sequence Them All? Comparison between Whole-Genome Sequencing (WGS) and Target Enrichment (TE) of museum specimens from the moth families Epicopeiidae and Sematuridae (Lepidoptera)\nAuthors: Call, E.; Twort, V. G.; Espeland, M.; Wahlberg, N.\nScore: 8.5, Published: 2023-08-25 DOI: 10.1101/2023.08.21.553699\nThere are various possibilities for sequencing highly degraded DNA, such as target enrichment (TE), or whole-genome sequencing (WGS). Here we compare TE and WGS methods using old museum specimens of two families of moths in the superfamily Geometroidea: Epicopeiidae and Sematuridae. Until recently, the relationships of these two families were unclear, as few studies had been done. Recently two studies used the TE approach, either on relatively fresh specimens, or on old museum specimens. Here, we aim to increase the sampling of the families Epicopeiidae and Sematuridae from museum specimens using the WGS method. We show that both sequencing methods give comparable results, but, unsurprisingly, WGS recovers more data. By combining TE and WGS data, we confirm that Sematuridae are sister to Pseudobistonidae+Epicopeiidae. Relationships of genera within the families are well supported. With the costs of WGS decreasing, we suggest that using low-coverage whole genome sequencing is becoming an increasingly viable option in the phylogenomics of insects.\nA cell-type atlas from a scyphozoan jellyfish Aurelia coerulea (formerly sp.1) provides insights into changes of cell-type diversity in the transition from polyps to medusae\nAuthors: Link, O.; Jahnel, S. M.; Janicek, K.; Kraus, J.; Montenegro, J. D.; Zimmerman, B.; Cole, A. G.; Technau, U.\nScore: 7.7, Published: 2023-08-26 DOI: 10.1101/2023.08.24.554571\nWe provide here a detailed single cell transcriptomic atlas covering the life cycle of the cosmopolitan scyphozoan Aurelia coerulea. We show that there is an increase in cell type diversity in the medusa stage, which is reflected by an increase in the number of unique transcripts expressed. We highlight parallels in both cell complement and specification pathways between Aurelia and the sea anemone Nematostella, two lineages separated by more than 500 MY. We find that several cell types of the neuroglandular lineages are specified by the same transcription factors that arose by gene duplications in the common ancestor of medusozoans and anthozoans. This confirms gene duplications as a source for the increase of cellular complexity during the evolution of cnidarian neuroglandular lineages dates to the common ancestor of all cnidarians. We validated spatial expression domains corresponding to molecular profiles of smooth and striated muscles and followed the development of the striated muscle in the medusa. Our data suggests that striated muscles in the subumbrella of the medusa derive directly from smooth muscles in the jellyfish, and that these two muscle cell types are characterized by distinct molecular profiles. Further, reconstruction of putative cell trajectories indicates multipotent cell states derive from tissues at the boundary between epithelial layers and highlight molecular characteristics not typical of other cnidarians.\nPhenotypic lags influence rapid evolution throughout a drought cycle\nAuthors: Branch, H. A.; Anstett, D. N.; Angert, A. L.\nScore: 6.5, Published: 2023-08-21 DOI: 10.1101/2023.08.18.553905\nClimate anomalies pose strong selection which can lead to rapid evolution. These global mean trends occur on a backdrop of interannual variability that might weaken or even reverse selection. However, the impact of climatic interannual variability on rapid evolution is rarely considered. We study evolution through a seven-year period encompassing a severe drought across 12 populations of Mimulus cardinalis (scarlet monkeyflower). Plants were grown in a common greenhouse environment under wet and dry treatments, where specific leaf area and date of flowering were measured. We compare the ability of different climate metrics to explain the rapid evolution of trait values, examining different time-periods, including the collection year, prior years, and cumulative metrics across sequential years. We find that anomalies in mean annual precipitation best describe rapid evolution over our study period. Past climates, of one-to two-years ago, are often related to trait values in a conflicting direction to collection-year climate. Uncovering these complex climatic impacts on evolution is critical to better predict and interpret the impacts of climate change.\nAdaptation to oral infection in D. melanogaster through evolution of both resistance and disease tolerance mechanisms\nAuthors: Paulo, T. F.; Akyaw, P. A.; Paixao, T.; Sucena, E.\nScore: 6.5, Published: 2023-08-24 DOI: 10.1101/2023.08.23.554397\nPathogens exert strong selection on hosts, that evolve and deploy different defensive strategies, namely minimizing pathogen exposure (avoidance), directly promoting pathogen elimination (resistance), and/or managing the deleterious effects of illness (disease tolerance). However, how the response to pathogens partitions across these processes has never been directly assessed in a single system, let alone in the context of known adaptive trajectories under controlled selection regimes. Here, an experimental evolution system composed of D. melanogaster and its natural pathogen P. entomophila is used to determine the role of behavioural traits, and of resistance and disease tolerance mechanisms on host adaption. We compare a population adapted to oral infection with P. entomophila (BactOral) with its control population to find no evidence for behavioural change but measurable differences in both resistance and disease tolerance. In BactOral we identify a relative decrease in bacterial loads correlated with an increase in gut production of specific AMPs, but no differences in bacterial intake, in differential gut cell renewal rate, or in the rate of defecation, pointing to a strengthening in resistance. Additionally, we posit that disease tolerance also contributes to the adaptive response of BactOral through a tighter control of its immune response and of the deleterious effects of exposure. This study reveals a genetically complex and mechanistically multi-layered response, possibly reflecting the structure of adaptation to infection in natural populations.\nOpsin gene duplication in Lepidoptera: retrotransposition, sex linkage, and gene expression\nAuthors: Mulhair, P. O.; Crowley, L.; Boyes, D. H.; Lewis, O. T.; Holland, P. W. H.\nScore: 34.9, Published: 2023-08-16 DOI: 10.1101/2023.08.16.552946\nColour vision in insects is determined by signalling cascades, central to which are opsin proteins, resulting in sensitivity to light at different wavelengths. In certain insect groups, lineage specific evolution of opsin genes, in terms of copy number, shifts in expression patterns, and functional amino acid substitutions, has resulted in changes in colour vision with subsequent behavioural and niche adaptations. Lepidoptera are a fascinating model to address whether evolutionary change in opsin content and sequence evolution are associated with changes in vision phenotype. Until recently, the lack of high quality genome data representing broad sampling across the lepidopteran phylogeny has greatly limited our ability to accurately address this question. Here, we annotate opsin genes in 219 lepidopteran genomes representing 33 families, reconstruct their evolutionary history, and analyse shifts in selective pressures and expression between genes and species. We discover 44 duplication events in opsin genes across [~]300 million years of lepidopteran evolution. While many duplication events are species or family specific, we find retention of an ancient long-wavelength sensitive (LW) opsin duplication derived by retrotransposition within the speciose superfamily Noctuoidea (in the families Nolidae, Erebidae, and Noctuidae). This conserved LW retrogene shows life stage specific expression suggesting visual sensitivities or other sensory functions specific to the early larval stage. This study provides a comprehensive order-wide view of opsin evolution across Lepidoptera, showcasing high rates of opsin duplications and changes in expression patterns.\nAfrican bush pigs exhibit porous species boundaries and appeared in Madagascar concurrently with human arrival\nAuthors: Balboa, R. F.; Bertola, L. D.; Bruniche-Olsen, A.; Rasmussen, M. S.; Liu, X.; Besnard, G.; Salmona, J.; Santander, C. G.; He, S.; Zinner, D.; Pedrono, M.; Muwanika, V.; Masembe, C.; Schubert, M.; Kuja, J.; Quinn, L.; Garcia-Erill, G.; Rakotoarivony, R.; Henrique, M.; Lin, L.; Wang, X.; Heaton, M. P.; Smith, T. P. L.; Hanghoj, K.; Sinding, M.-H. S.; Atickem, A.; Chikhi, L.; Roos, C.; Gaubert, P.; Siegismund, H. R.; Moltke, I.; Albrechtsen, A.; Heller, R.\nScore: 4.7, Published: 2023-08-24 DOI: 10.1101/2023.08.23.553838\nSeveral African mammals exhibit a phylogeographic pattern where closely related taxa are split between West/Central and East/Southern Africa, but their evolutionary relationships and histories remain controversial. Bushpigs (Potamochoerus larvatus) and red river hogs (P. porcus) are recognised as separate species due to morphological distinctions, a perceived lack of interbreeding at contact, and putatively old divergence times, but historically, they were considered conspecific. Moreover, the presence of Malagasy bushpigs as the sole large terrestrial mammal shared with the African mainland raises intriguing questions about its origin and arrival in Madagascar. Analyses of 67 whole genomes revealed a genetic continuum between the two species, with putative signatures of historical gene flow, variable FST values, and a recent divergence time (\u003c500,000 years). Thus, our study challenges key arguments for splitting Potamochoerus into two species and suggests their speciation might be incomplete. Our findings also indicate that Malagasy bushpigs diverged from southern African populations and underwent a limited bottleneck 1,000-5,000 years ago, concurrent with human arrival in Madagascar. These results shed new light on the evolutionary history of an iconic and widespread African genus and provide insight into the longstanding biogeographic puzzle surrounding the bushpig's presence in Madagascar.\n",
  "wordCount" : "2588",
  "inLanguage": "en",
  "datePublished": "2023-08-27T10:37:02Z",
  "dateModified": "2023-08-27T10:37:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on August 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550432">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550432" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550432">
        <p class="paperTitle">Evidence for ecological tuning of novel anuran biofluorescent signals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550432" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550432" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Whitcher, C.; Ron, S. R.; Ayala-Varela, F.; Crawford, A.; Herrera-Alva, V.; Castillo-Urbina, E.; Grazziotin, F.; Bowman, R. M.; Lemmon, A. R.; Moriarty Lemmon, E.</p>
        <p class="info">Score: 51.4, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550432' target='https://doi.org/10.1101/2023.07.25.550432'> 10.1101/2023.07.25.550432</a></p>
        <p class="abstract">Our study assesses the variability of amphibian biofluorescence and provides insight into its potential functions and role in anuran evolution. Via a field survey across South America, we discovered and documented patterns of biofluorescence in tropical amphibians. We more than tripled the number of species that have been tested for this trait and added representatives from previously untested anuran families. We found evidence for ecological tuning (i.e., the specific adaptation of a signal to the environment in which it is received) of the novel anuran biofluorescent signals. Across groups, the fluorescence excitation peak matches the wavelengths most available at twilight, the light environment in which most frog species are active. Additionally, biofluorescence emission spans both wavelengths of low availability in twilight and the peak sensitivity of green-sensitive rods in the anuran eye, likely increasing contrast of this signal for a conspecific receiver. With evidence of tuning to the ecology and sensory systems of frogs, our results suggest frog biofluorescence is likely functioning in anuran communication.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554357">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554357" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554357">
        <p class="paperTitle">Rapid turnover of centromeric DNA reveals signatures of genetic conflict in Drosophila</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554357" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554357" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Courret, C.; Hemmer, L.; Wei, X.; Patel, P. D.; Santinello, B.; Geng, X.; Chang, C.-H.; Mellone, B.; Larracuente, A. M.</p>
        <p class="info">Score: 15.7, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554357' target='https://doi.org/10.1101/2023.08.22.554357'> 10.1101/2023.08.22.554357</a></p>
        <p class="abstract">Despite their essential function in chromosome segregation, centromeres reside in rapidly evolving, repeat-rich genomic regions. Across organisms, centromeres are rich in selfish genetic elements like transposable elements and satellite DNAs that can bias their transmission through meiosis, but still need to contribute to centromere function. To gain insight into the balance between conflict and cooperation at centromeric DNA, we take advantage of the close evolutionary relationships within the Drosophila simulans clade - D. simulans, D. sechellia, and D. mauritiana - and their relative, D. melanogaster. We discovered dramatic centromere reorganization involving recurrent shifts between retroelements and satellite DNAs over short evolutionary timescales. We also reveal the recent origin (&lt;240 Kya) of truly telocentric chromosomes in D. sechellia, where X and dot centromeres now sit on telomere-specific retroelements. This rapid centromere turnover is consistent with genetic conflicts in the female germline and has implications for centromeric DNA function and karyotype evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553663">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553663" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553663">
        <p class="paperTitle">An effector deletion leads to the breakdown of partial grapevine resistance to downy mildew</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553663" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553663" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Paineau, M.; Minio, A.; Mestre, P.; Fabre, F.; Mazet, I. D.; Couture, C.; Legeai, F.; Dumartinet, T.; Cantu, D.; Delmotte, F.</p>
        <p class="info">Score: 17.6, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553663' target='https://doi.org/10.1101/2023.08.17.553663'> 10.1101/2023.08.17.553663</a></p>
        <p class="abstract">Grapevine downy mildew, caused by the oomycete Plasmopara viticola, is a globally destructive disease that particularly affect the Eurasian wine grape V. vinifera. While genetically resistant varieties are becoming more accessible, populations of P. viticola are demonstrating rapid adaptability, successfully over-coming these resistances. Here we aimed to identify the avirulence genes involved in the interaction with the Rpv3.1-mediated resistance in grapevine. We sequenced the full genome of 136 P. viticola strains sampled in a natural population of Bordeaux (France) and characterized their development on both resistant and sensitive cultivars. The genome-wide association study allowed the identification of a genomic region associated with the breakdown of Rpv3.1 grapevine resistance (avrRpv3.1 locus). A diploid-aware reassembly of the P. viticola INRA-Pv221 genome allowed to detect structural variations in this locus, including a major 30 Kbp deletion. At the avrRpv3.1 locus, virulent P. viticola strains presented deletion on both haplotypes indicating that avirulence is recessive. The deletion involves two closely-related genes that encode proteins containing 800-900 amino acids with a signal peptide. The structure of the predicted proteins contains repeats of the LWY-fold structural modules, typical of oomycete effectors. Moreover, when these proteins were transiently expressed, they induced cell death in grapevines carrying Rpv3.1 resistance, confirming their avirulence nature. The first description of candidate effectors of P. viticola involved in the interaction with resistance gene provides valuable insights into the genetic mechanisms that enable P. viticola to adapt to grapevine resistance, laying a foundation for developing strategies to manage this damaging crop pathogen.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554279">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554279" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554279">
        <p class="paperTitle">Geo-genomic predictors of genetree heterogeneity in an Amazonian bird (Thamnophilus aethiops)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554279" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554279" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Musher, L. J.; Del Rio, G.; Marcondes, R. S.; Brumfield, R. T.; Bravo, G.; Thom, G.</p>
        <p class="info">Score: 9.0, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554279' target='https://doi.org/10.1101/2023.08.22.554279'> 10.1101/2023.08.22.554279</a></p>
        <p class="abstract">Can knowledge about genome architecture inform biogeographic and phylogenetic inference? Selection, drift, recombination, and gene flow interact to produce a genomic landscape of divergence wherein patterns of differentiation and genealogy vary nonrandomly across the genomes of diverging populations. For instance, genealogical patterns that arise due to gene flow should be more likely to occur on smaller chromosomes, which experience high recombination, whereas those tracking histories of geographic isolation (reduced gene flow caused by a barrier) and divergence should be more likely to occur on larger and sex chromosomes. In Amazonia, populations of many bird species diverge and introgress across rivers, resulting in reticulated genomic signals. Herein, we used reduced representation genomic data to disentangle the evolutionary history of four populations of an Amazonian antbird, Thamnophilus aethiops, whose biogeographic history was associated with the dynamic evolution of the Madeira River Basin. Specifically, we evaluate whether a large river capture event ca. 200 kya, gave rise to reticulated genealogies in the genome by making spatially explicit predictions about isolation and gene flow based on knowledge about genomic processes. We first estimated chromosome-level phylogenies and recovered two primary topologies across the genome. The first topology (T1) was most consistent with predictions about population divergence, and was recovered for the Z chromosome. The second (T2), was consistent with predictions about gene flow upon secondary contact. To evaluate support for these topologies, we trained a convolutional neural network to classify our data into alternative diversification models and estimate demographic parameters. The best-fit model was concordant with T1 and included gene flow between non-sister taxa. Finally, we modeled levels of divergence and introgression as functions of chromosome length, and found that smaller chromosomes experienced higher gene flow. Given that (1) gene-trees supporting T2 were more likely to occur on smaller chromosomes and (2) we found lower levels of introgression on larger chromosomes (and especially the Z-chromosome), we argue that T1 represents the history of population divergence across rivers and T2 the history of secondary contact due to barrier loss. Our results suggest that a significant portion of genomic heterogeneity arises due to extrinsic biogeographic processes such as river capture interacting with intrinsic processes associated with genome architecture. Future biogeographic studies would benefit from accounting for genomic processes, as different parts of the genome reveal contrasting, albeit complementary histories, all of which are relevant for disentangling the intricate geogenomic mechanisms of biotic diversification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.553699">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.553699" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.553699">
        <p class="paperTitle">One Method to Sequence Them All? Comparison between Whole-Genome Sequencing (WGS) and Target Enrichment (TE) of museum specimens from the moth families Epicopeiidae and Sematuridae (Lepidoptera)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.553699" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.553699" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Call, E.; Twort, V. G.; Espeland, M.; Wahlberg, N.</p>
        <p class="info">Score: 8.5, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.553699' target='https://doi.org/10.1101/2023.08.21.553699'> 10.1101/2023.08.21.553699</a></p>
        <p class="abstract">There are various possibilities for sequencing highly degraded DNA, such as target enrichment (TE), or whole-genome sequencing (WGS). Here we compare TE and WGS methods using old museum specimens of two families of moths in the superfamily Geometroidea: Epicopeiidae and Sematuridae. Until recently, the relationships of these two families were unclear, as few studies had been done. Recently two studies used the TE approach, either on relatively fresh specimens, or on old museum specimens. Here, we aim to increase the sampling of the families Epicopeiidae and Sematuridae from museum specimens using the WGS method. We show that both sequencing methods give comparable results, but, unsurprisingly, WGS recovers more data. By combining TE and WGS data, we confirm that Sematuridae are sister to Pseudobistonidae&#43;Epicopeiidae. Relationships of genera within the families are well supported. With the costs of WGS decreasing, we suggest that using low-coverage whole genome sequencing is becoming an increasingly viable option in the phylogenomics of insects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554571">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554571" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554571">
        <p class="paperTitle">A cell-type atlas from a scyphozoan jellyfish Aurelia coerulea (formerly sp.1) provides insights into changes of cell-type diversity in the transition from polyps to medusae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554571" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554571" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Link, O.; Jahnel, S. M.; Janicek, K.; Kraus, J.; Montenegro, J. D.; Zimmerman, B.; Cole, A. G.; Technau, U.</p>
        <p class="info">Score: 7.7, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554571' target='https://doi.org/10.1101/2023.08.24.554571'> 10.1101/2023.08.24.554571</a></p>
        <p class="abstract">We provide here a detailed single cell transcriptomic atlas covering the life cycle of the cosmopolitan scyphozoan Aurelia coerulea. We show that there is an increase in cell type diversity in the medusa stage, which is reflected by an increase in the number of unique transcripts expressed. We highlight parallels in both cell complement and specification pathways between Aurelia and the sea anemone Nematostella, two lineages separated by more than 500 MY. We find that several cell types of the neuroglandular lineages are specified by the same transcription factors that arose by gene duplications in the common ancestor of medusozoans and anthozoans. This confirms gene duplications as a source for the increase of cellular complexity during the evolution of cnidarian neuroglandular lineages dates to the common ancestor of all cnidarians. We validated spatial expression domains corresponding to molecular profiles of smooth and striated muscles and followed the development of the striated muscle in the medusa. Our data suggests that striated muscles in the subumbrella of the medusa derive directly from smooth muscles in the jellyfish, and that these two muscle cell types are characterized by distinct molecular profiles. Further, reconstruction of putative cell trajectories indicates multipotent cell states derive from tissues at the boundary between epithelial layers and highlight molecular characteristics not typical of other cnidarians.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553905">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553905" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553905">
        <p class="paperTitle">Phenotypic lags influence rapid evolution throughout a drought cycle</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553905" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553905" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Branch, H. A.; Anstett, D. N.; Angert, A. L.</p>
        <p class="info">Score: 6.5, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553905' target='https://doi.org/10.1101/2023.08.18.553905'> 10.1101/2023.08.18.553905</a></p>
        <p class="abstract">Climate anomalies pose strong selection which can lead to rapid evolution. These global mean trends occur on a backdrop of interannual variability that might weaken or even reverse selection. However, the impact of climatic interannual variability on rapid evolution is rarely considered. We study evolution through a seven-year period encompassing a severe drought across 12 populations of Mimulus cardinalis (scarlet monkeyflower). Plants were grown in a common greenhouse environment under wet and dry treatments, where specific leaf area and date of flowering were measured. We compare the ability of different climate metrics to explain the rapid evolution of trait values, examining different time-periods, including the collection year, prior years, and cumulative metrics across sequential years. We find that anomalies in mean annual precipitation best describe rapid evolution over our study period. Past climates, of one-to two-years ago, are often related to trait values in a conflicting direction to collection-year climate. Uncovering these complex climatic impacts on evolution is critical to better predict and interpret the impacts of climate change.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554397">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554397" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554397">
        <p class="paperTitle">Adaptation to oral infection in D. melanogaster through evolution of both resistance and disease tolerance mechanisms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554397" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554397" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Paulo, T. F.; Akyaw, P. A.; Paixao, T.; Sucena, E.</p>
        <p class="info">Score: 6.5, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554397' target='https://doi.org/10.1101/2023.08.23.554397'> 10.1101/2023.08.23.554397</a></p>
        <p class="abstract">Pathogens exert strong selection on hosts, that evolve and deploy different defensive strategies, namely minimizing pathogen exposure (avoidance), directly promoting pathogen elimination (resistance), and/or managing the deleterious effects of illness (disease tolerance). However, how the response to pathogens partitions across these processes has never been directly assessed in a single system, let alone in the context of known adaptive trajectories under controlled selection regimes. Here, an experimental evolution system composed of D. melanogaster and its natural pathogen P. entomophila is used to determine the role of behavioural traits, and of resistance and disease tolerance mechanisms on host adaption. We compare a population adapted to oral infection with P. entomophila (BactOral) with its control population to find no evidence for behavioural change but measurable differences in both resistance and disease tolerance. In BactOral we identify a relative decrease in bacterial loads correlated with an increase in gut production of specific AMPs, but no differences in bacterial intake, in differential gut cell renewal rate, or in the rate of defecation, pointing to a strengthening in resistance. Additionally, we posit that disease tolerance also contributes to the adaptive response of BactOral through a tighter control of its immune response and of the deleterious effects of exposure. This study reveals a genetically complex and mechanistically multi-layered response, possibly reflecting the structure of adaptation to infection in natural populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.552946">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.552946" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.552946">
        <p class="paperTitle">Opsin gene duplication in Lepidoptera: retrotransposition, sex linkage, and gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.552946" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.552946" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mulhair, P. O.; Crowley, L.; Boyes, D. H.; Lewis, O. T.; Holland, P. W. H.</p>
        <p class="info">Score: 34.9, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.552946' target='https://doi.org/10.1101/2023.08.16.552946'> 10.1101/2023.08.16.552946</a></p>
        <p class="abstract">Colour vision in insects is determined by signalling cascades, central to which are opsin proteins, resulting in sensitivity to light at different wavelengths. In certain insect groups, lineage specific evolution of opsin genes, in terms of copy number, shifts in expression patterns, and functional amino acid substitutions, has resulted in changes in colour vision with subsequent behavioural and niche adaptations. Lepidoptera are a fascinating model to address whether evolutionary change in opsin content and sequence evolution are associated with changes in vision phenotype. Until recently, the lack of high quality genome data representing broad sampling across the lepidopteran phylogeny has greatly limited our ability to accurately address this question. Here, we annotate opsin genes in 219 lepidopteran genomes representing 33 families, reconstruct their evolutionary history, and analyse shifts in selective pressures and expression between genes and species. We discover 44 duplication events in opsin genes across [~]300 million years of lepidopteran evolution. While many duplication events are species or family specific, we find retention of an ancient long-wavelength sensitive (LW) opsin duplication derived by retrotransposition within the speciose superfamily Noctuoidea (in the families Nolidae, Erebidae, and Noctuidae). This conserved LW retrogene shows life stage specific expression suggesting visual sensitivities or other sensory functions specific to the early larval stage. This study provides a comprehensive order-wide view of opsin evolution across Lepidoptera, showcasing high rates of opsin duplications and changes in expression patterns.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.553838">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.553838" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.553838">
        <p class="paperTitle">African bush pigs exhibit porous species boundaries and appeared in Madagascar concurrently with human arrival</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.553838" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.553838" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Balboa, R. F.; Bertola, L. D.; Bruniche-Olsen, A.; Rasmussen, M. S.; Liu, X.; Besnard, G.; Salmona, J.; Santander, C. G.; He, S.; Zinner, D.; Pedrono, M.; Muwanika, V.; Masembe, C.; Schubert, M.; Kuja, J.; Quinn, L.; Garcia-Erill, G.; Rakotoarivony, R.; Henrique, M.; Lin, L.; Wang, X.; Heaton, M. P.; Smith, T. P. L.; Hanghoj, K.; Sinding, M.-H. S.; Atickem, A.; Chikhi, L.; Roos, C.; Gaubert, P.; Siegismund, H. R.; Moltke, I.; Albrechtsen, A.; Heller, R.</p>
        <p class="info">Score: 4.7, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.553838' target='https://doi.org/10.1101/2023.08.23.553838'> 10.1101/2023.08.23.553838</a></p>
        <p class="abstract">Several African mammals exhibit a phylogeographic pattern where closely related taxa are split between West/Central and East/Southern Africa, but their evolutionary relationships and histories remain controversial. Bushpigs (Potamochoerus larvatus) and red river hogs (P. porcus) are recognised as separate species due to morphological distinctions, a perceived lack of interbreeding at contact, and putatively old divergence times, but historically, they were considered conspecific. Moreover, the presence of Malagasy bushpigs as the sole large terrestrial mammal shared with the African mainland raises intriguing questions about its origin and arrival in Madagascar. Analyses of 67 whole genomes revealed a genetic continuum between the two species, with putative signatures of historical gene flow, variable FST values, and a recent divergence time (&lt;500,000 years). Thus, our study challenges key arguments for splitting Potamochoerus into two species and suggests their speciation might be incomplete. Our findings also indicate that Malagasy bushpigs diverged from southern African populations and underwent a limited bottleneck 1,000-5,000 years ago, concurrent with human arrival in Madagascar. These results shed new light on the evolutionary history of an iconic and widespread African genus and provide insight into the longstanding biogeographic puzzle surrounding the bushpig&#39;s presence in Madagascar.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
