<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Genomic and ecological factors shaping specialism and generalism across an entire subphylum
Authors: Opulente, D. A.; LaBella, A. L.; Harrison, M.-C.; Wolters, J. F.; Liu, C.; Li, Y.; Kominek, J.; Steenwyk, J. L.; Stoneman, H. R.; VanDenAvond, J.; Miller, C. R.; Langdon, Q. K.; Silva, M.; Goncalves, C.; Ubbelohde, E. J.; Li, Y.; Buh, K. V.; Jarzyna, M.; Haase, M. A. B.; Rosa, C. A.; Cadez, N.; Libkind, D.; DeVirgilio, J.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="Genomic and ecological factors shaping specialism and generalism across an entire subphylum
Authors: Opulente, D. A.; LaBella, A. L.; Harrison, M.-C.; Wolters, J. F.; Liu, C.; Li, Y.; Kominek, J.; Steenwyk, J. L.; Stoneman, H. R.; VanDenAvond, J.; Miller, C. R.; Langdon, Q. K.; Silva, M.; Goncalves, C.; Ubbelohde, E. J.; Li, Y.; Buh, K. V.; Jarzyna, M.; Haase, M. A. B.; Rosa, C. A.; Cadez, N.; Libkind, D.; DeVirgilio, J." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-02T10:39:32+00:00" />
<meta property="article:modified_time" content="2023-07-02T10:39:32+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="Genomic and ecological factors shaping specialism and generalism across an entire subphylum
Authors: Opulente, D. A.; LaBella, A. L.; Harrison, M.-C.; Wolters, J. F.; Liu, C.; Li, Y.; Kominek, J.; Steenwyk, J. L.; Stoneman, H. R.; VanDenAvond, J.; Miller, C. R.; Langdon, Q. K.; Silva, M.; Goncalves, C.; Ubbelohde, E. J.; Li, Y.; Buh, K. V.; Jarzyna, M.; Haase, M. A. B.; Rosa, C. A.; Cadez, N.; Libkind, D.; DeVirgilio, J."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "Genomic and ecological factors shaping specialism and generalism across an entire subphylum\nAuthors: Opulente, D. A.; LaBella, A. L.; Harrison, M.-C.; Wolters, J. F.; Liu, C.; Li, Y.; Kominek, J.; Steenwyk, J. L.; Stoneman, H. R.; VanDenAvond, J.; Miller, C. R.; Langdon, Q. K.; Silva, M.; Goncalves, C.; Ubbelohde, E. J.; Li, Y.; Buh, K. V.; Jarzyna, M.; Haase, M. A. B.; Rosa, C. A.; Cadez, N.; Libkind, D.; DeVirgilio, J.",
  "keywords": [
    
  ],
  "articleBody": " Genomic and ecological factors shaping specialism and generalism across an entire subphylum\nAuthors: Opulente, D. A.; LaBella, A. L.; Harrison, M.-C.; Wolters, J. F.; Liu, C.; Li, Y.; Kominek, J.; Steenwyk, J. L.; Stoneman, H. R.; VanDenAvond, J.; Miller, C. R.; Langdon, Q. K.; Silva, M.; Goncalves, C.; Ubbelohde, E. J.; Li, Y.; Buh, K. V.; Jarzyna, M.; Haase, M. A. B.; Rosa, C. A.; Cadez, N.; Libkind, D.; DeVirgilio, J. H.; Hulfachor, A. B.; Kurtzman, C. P.; Sampaio, J. P.; Goncalves, P.; Zhou, X.; Shen, X.-X.; Groenewald, M.; Rokas, A.; Hittinger, C. T.\nScore: 72.2, Published: 2023-06-28 DOI: 10.1101/2023.06.19.545611\nOrganisms exhibit extensive variation in ecological niche breadth, from very narrow (specialists) to very broad (generalists). Paradigms proposed to explain this variation either invoke trade-offs between performance efficiency and breadth or underlying intrinsic or extrinsic factors. We assembled genomic (1,154 yeast strains from 1,049 species), metabolic (quantitative measures of growth of 843 species in 24 conditions), and ecological (environmental ontology of 1,088 species) data from nearly all known species of the ancient fungal subphylum Saccharomycotina to examine niche breadth evolution. We found large interspecific differences in carbon breadth stem from intrinsic differences in genes encoding specific metabolic pathways but no evidence of trade-offs and a limited role of extrinsic ecological factors. These comprehensive data argue that intrinsic factors driving microbial niche breadth variation.\n241,000 to 335,000 Years Old Rock Engravings Made by Homo naledi in the Rising Star Cave system, South Africa.\nAuthors: Berger, L. R.; Hawks, J.; Fuentes, A.; van Rooyen, D.; Tsikoane, M.; Ramalepa, M.; Nkwe, S.; Molopyane, K.\nScore: 1758.0, Published: 2023-06-05 DOI: 10.1101/2023.06.01.543133\nThe production of painted, etched or engraved designs on cave walls or other surfaces is recognized as a major cognitive step in human evolution. Such intentional designs, which are widely interpreted as signifying, recording, and transmitting information in a durable manner were once considered exclusive to Late Pleistocene Homo sapiens. Recent work has demonstrated that other hominin groups also made such marks, including Neanderthals (Rodriguez-Vidal et al., 2014; Hoffmann et al., 2018), and possibly Middle-Pleistocene Homo erectus (Joordens et al., 2015). Such durable signs indicate an intentionality characteristic of meaning-making (Kissel and Fuentes 2018) which has been argued to require significant levels of cognitive abilities not found in species with smaller brain sizes (Parkington, 2010). In fact, the evolution of such meaning-making symbols is thought to be a core aspect of what it means to be \"human\" (Henshilwood, 2009). Here we present the first known example of abstract patterns and shapes engraved within the Dinaledi subsystem of the Rising Star Cave in South Africa. We identified markings incised into the dolomitic limestone walls of the cave. The engravings described here are deeply impressed cross-hatchings and other geometric shapes. The surfaces bearing these engravings appear to have been prepared and smoothed. In some areas there is residue that creates a sheen on the surface possibly indicating repeated handling or rubbing of the rock, and there is evidence of the application of dirt or sand to the surface by non-natural processes. Homo naledi entered this part of the cave system and buried bodies within the both the Dinaledi Chamber and adjacent Hill Antechamber between 241 and 335 ka (Dirks et al., 2017; Robbins et al., 2021, Berger et al, 2023a). The engravings described here are found on a pillar in the Hill Antechamber that extends into the natural fissure corridor that links the two chambers and we associate them with H. naledi.\nChance favors the prepared genomes: horizontal transfer shapes the emergence of antibiotic resistance mutations in core genes.\nAuthors: Coluzzi, C.; Guillemet, M.; Mazzamurro, F.; Touchon, M.; Godfroid, M.; Achaz, G.; Glaser, P.; Rocha, E. P. C.\nScore: 52.5, Published: 2023-06-22 DOI: 10.1101/2023.06.20.545734\nBacterial lineages vary in the frequency with which they acquire novel traits, like antibiotic resistance or virulence. While previous studies have highlighted the impact of the genetic background on the successful acquisition of novel traits through horizontal gene transfer, the impact of the latter on the subsequent evolution of bacterial genomes by point mutations remains poorly understood. Here, we studied the evolution of resistance to quinolones in thousands of Escherichia coli genomes. Resistance-conferring point mutations in the core genes are frequent and accumulate very quickly. We searched for gene gains and losses significantly associated with the subsequent acquisition of these resistance mutations. This revealed 60 groups of genes in genetic linkage whose gain or loss induced a change in the probability of subsequently becoming resistant to quinolones by point mutations in gyrA and parC. Although some of these chronologies may reflect epidemiological trends, most of these groups encoded functions that were previously associated with antibiotic resistance, tolerance, or persistence, often specifically under quinolone treatment. A lot of the largest groups were found in prophages or plasmids, and they usually increased the likelihood of subsequent resistance mutations. Conversely groups of lost genes were typically small and chromosomal. Quinolone resistance was among the first resistances acquired in the extant lineages of E. coli and its acquisition was associated with an increased likelihood of acquiring other types of resistances, including to aminoglycosides and beta-lactams. Our findings suggest that gene flow shapes the subsequent fixation rate of adaptive mutations in core genes. Given the substantial gene flow within bacterial genomes, interactions between horizontal transfer and point mutations in core genes may be key to the success of adaptation processes.\nDeep Divergences Among Cryptic Clades of Epipedobates Poison Frogs\nAuthors: Lopez-Hervas, K.; Santos, J. C.; Ron, S. R.; Betancourth-Cundar, M.; Cannatella, D. C.; Tarvin, R. D.\nScore: 23.9, Published: 2023-06-30 DOI: 10.1101/2023.06.29.547117\nPoison frogs (Dendrobatidae) are famous for their aposematic species, having a combination of diverse color patterns and defensive skin toxins, yet most species in this family are inconspicuous (cryptic) and considered non-aposematic. Epipedobates is among the youngest genus-level clades of Dendrobatidae that includes both aposematic and cryptic species. Using mitochondrial and nuclear markers, we demonstrate deep genetic divergences among cryptic species of Epipedobates but relatively shallow genetic divergences among conspicuous species. Our phylogenetic analysis includes broad geographic sampling of the cryptic lineages typically identified as E. boulengeri and E. espinosai, which reveals two putative new species, one in west-central Colombia (E. sp. 1) and the other in north-central Ecuador (E. aff. espinosai). We conclude that E. darwinwallacei is a junior subjective synonym of E. espinosai. We also clarify the geographic distributions of cryptic Epipedobates species including the widespread E. boulengeri. We provide a qualitative assessment of the phenotypic diversity in each nominal species, with a focus on the color and pattern of cryptic species. We conclude that Epipedobates contains eight known valid species, six of which are cryptic. A relaxed molecular clock analysis suggests that the most recent common ancestor of Epipedobates is ~11.1 million years old, which nearly doubles previous estimates. Last, genetic information points to a center of species diversity in the Choco at the southwestern border of Colombia with Ecuador.\nrphylopic: An R package for fetching, transforming, and visualising PhyloPic silhouettes\nAuthors: Gearty, W.; Jones, L. A.\nScore: 23.8, Published: 2023-06-26 DOI: 10.1101/2023.06.22.546191\nO_LIData visualisation is vital for data exploration, analysis, and communication in research. Moreover, it can bridge gaps between researchers and the general public by making research findings more accessible and engaging. Today, researchers increasingly conduct their data analyses in programming languages such as R and Python. The availability of data visualisation tools within these environments supports the generation of reproducible data analyses and visualisation workflows. However, resources from external databases are frequently required for data visualisation necessitating integration between these databases and the platforms in which researchers conduct their analyses. C_LIO_LIHere, we introduce rphylopic, an R package for fetching, transforming, and visualising silhouettes of organisms from the PhyloPic database. In addition to making over 7,000 organism silhouettes available within the R programming language, rphylopic empowers users to modify the appearance of these silhouettes for ultimate customisability when coding production-quality visualisations in both base R and ggplot2 workflows. C_LIO_LIIn this work, we provide details about how the package can be installed, its implementation, and potential use cases. For the latter, we showcase three examples across the ecology and evolutionary biology spectrum. C_LIO_LIOur hope is that rphylopic will make it easier for biologists to develop more accessible and engaging data visualisations by making external resources readily accessible, customisable, and usable within R. In turn, by integrating into existing workflows, rphylopic helps to ensure that science is reproducible and accessible. C_LI\nReconstructing the evolutionary history of human, simian, and prosimian immunodeficiency viruses\nAuthors: Ghafari, M.; Simmonds, P.; Katzourakis, A.\nScore: 19.1, Published: 2023-06-28 DOI: 10.1101/2023.06.28.546833\nWhile paleovirological evidence supports the idea that the ancestors of modern simian (SIV) and prosimian (pSIV) immunodeficiency viruses have been evolving in nonhuman primates for millions of years, standard molecular clock methods calibrated using contemporary sequences significantly underestimate their times of origin. This discrepancy is attributed to the time-dependent nature of evolutionary rate estimates whereby the rate of virus evolution varies with respect to the time window of its measurement. While biogeographical calibrations may provide better estimates across intermediate timescales, there is currently no unifying framework that can allow inference of lentiviral ages across all relevant timescales. Here we showed that by correcting for such time-dependent rate effects using the recently developed Prisoner-of-War evolutionary model, we can successfully reconstruct the evolutionary history of human, simian, and prosimian immunodeficiency viruses across vastly different timescales and allowed estimates of the age of lineages that gave rise to HIV-1 and HIV-2. The model also predicted that the most recent common ancestor of SIV and pSIV lived around 21 million years ago, suggesting that the most likely explanation for the origins of primate lentiviruses is a terrestrial transfer of lentiviruses between African and Malagasy primates during the last episode of colonisation of Madagascar. Infections have entered via a land bridge in a group of mammals or through a nonprimate vector or transfer mediated by an aerial vector species. We also found that the most recent common ancestor of SIVs lived nearly a million years ago and that some SIV lineages codiverged with their hosts for several hundreds of thousands of years. The predicted long evolutionary timescales of SIVs and potential for close virus/host co-adaptation is consistent with their reduced or minimal pathogenicity in their native hosts, contrasting with the very recent evolutionary origins of highly pathogenic HIV strains in humans.\nAncient Sheep Genomes reveal four Millennia of North European Short-Tailed Sheep in the Baltic Sea region\nAuthors: Larsson, M. N.; Morell Miranda, P.; Pan, L.; Basak Vural, K.; Kaptan, D.; Rodrigues Soares, A. E.; Kivikero, H.; Kantanen, J.; Somel, M.; Özer, F.; Johansson, A. M.; Stora, J.; Günther, T.\nScore: 18.5, Published: 2023-06-27 DOI: 10.1101/2023.06.26.544912\nSheep are among the earliest domesticated livestock species, with a wide variety of breeds present today. However, it remains unclear how far back this breed diversity goes, with formal documentation only dating back a few centuries. North European short-tailed breeds are often assumed to be among the oldest domestic sheep populations, even thought to represent relicts of the earliest sheep expansions during the Neolithic period reaching Scandinavia less than 6000 years ago. This study sequenced the genomes (up to 11.6X) of five sheep remains from the Baltic islands of Gotland and [A]land, dating from Late Neolithic ([~]4100 calBP) to historical times ([~]1600 CE). Our findings indicate that these ancient sheep largely possessed the genetic characteristics of modern North European short-tailed breeds, suggesting a substantial degree of long-term continuity of this breed type in the Baltic Sea region. Despite the wide temporal spread, population genetic analyses show high levels of affinity between the ancient genomes and they also exhibit higher genetic diversity when compared to modern breeds, implying a loss of diversity in recent centuries associated with breed formation. Finally, we see a potential signature of an even earlier, genetically different form of sheep in Scandinavia as these samples do not represent the first sheep in Northern Europe. Our results shed light on the development of breeds in Northern Europe specifically as well as the development of genetic diversity in sheep breeds, and their expansion from the domestication center in general.\nA multi-layered integrative analysis reveals a cholesterol metabolic program in outer radial glia with implications for human brain evolution\nAuthors: Moriano, J.; Leonardi, O.; Vitriolo, A.; Testa, G.; Boeckx, C.\nScore: 17.9, Published: 2023-06-26 DOI: 10.1101/2023.06.23.546307\nThe definition of molecular and cellular mechanisms contributing to evolutionary divergences in brain ontogenetic trajectories is essential to formulate hypotheses about the emergence of our species. Yet the functional dissection of evolutionary modifications derived in the Homo sapiens lineage at an appropriate level of granularity remains particularly challenging. Capitalizing on recent single-cell sequencing efforts that have massively profiled neural stem cells from the developing human cortex, we develop an integrative computational framework in which we perform (i) trajectory inference and gene regulatory network reconstruction, (ii) (pseudo)time-informed non-negative matrix factorization for learning the dynamics of gene expression programs, and (iii) paleogenomic analysis for a higher-resolution mapping of the regulatory landscape where our species acquired derived mutations in comparison to our closest relatives. We provide evidence for cell type-specific activation and regulation of gene expression programs during indirect neurogenesis. In particular, our analysis uncovers a zinc-finger transcription factor, KLF6, as a key regulator of a cholesterol metabolic program specifically in outer radial glia. Our strategy allows us to further probe whether the (semi)discrete gene expression programs identified have been under selective pressures in our species lineage. A cartography of the regulatory landscape impacted by Homo sapiens-derived transcription factor binding site disruptions reveals signals of selection clustering around regulatory regions associated with GLI3, a well-known regulator of the radial glial cell cycle. As a whole, our study contributes to the evidence of significant changes impacting metabolic pathways in recent human brain evolution.\nStanding genetic variation and de novo mutations underlie parallel evolution of island bird phenotypes\nAuthors: Estandia, A.; Sendell-Price, A. T.; Robertson, B. C.; Clegg, S. M.\nScore: 17.9, Published: 2023-06-29 DOI: 10.1101/2023.06.29.546893\nParallel evolution occurs when the same trait evolves in closely related lineages in response to similar ecological contexts and provides some of the best examples of determinism in evolutionary biology. However, from a genetic standpoint, this process can be driven by either new mutations that appear independently in each diverging population or by selection on existing genetic variation common to both lineages. Small bodied birds, for example, tend to increase in size after they colonise a new island, following what is known as the 'island rule'. Such is the case of the Silvereye, a prolific natural coloniser of southwest Pacific islands. Island forms of this bird species increase in body size after they establish, with the pattern and pace of change consistent with directional natural selection and evident even in the most recent colonisations within the last 200 years. The system provides an exceptional opportunity to explore the genomic basis of repeated body size evolution. We sequenced 377 whole genomes from 31 different Silvereye populations, which revealed that both mechanisms are at play: in some lineages, new mutations are highly associated with body and bill size, but there are also highly associated polymorphisms present across all populations. Our research sheds light on the genomic basis of repeated body size evolution and emphasises that multiple molecular mechanisms can underlie similar evolutionary trajectories even within a single taxon.\nBurials and engravings in a small-brained hominin, Homo naledi, from the late Pleistocene: contexts and evolutionary implications\nAuthors: Fuentes, A.; Kissel, M.; Spikins, P.; Molopyane, K.; Hawks, J.; Berger, L. R.\nScore: 1457.8, Published: 2023-06-05 DOI: 10.1101/2023.06.01.543135\nData from recent explorations in the Dinaledi subsystem illustrates one of the earliest examples of a mortuary practice in hominins and offers the earliest evidence of multiple interments and funerary actions, as well as evidence of the early creation of meaning making by a hominin. The hominin undertaking these behaviors was the small-brained Homo naledi. These data call into question several key assumptions about behavioral and cognitive evolution in Pleistocene hominins. The evidence from Dinaledi push back the temporal origins of mortuary and funerary behaviors and associate the creation of meaning making with a small-brained species and thus challenge key assumptions about the role and importance of encephalization in human evolution. This suggests that the hominin socio-cognitive niche and its relation to meaning-making activities is more diverse than previously thought. The association of these activities in subterranean spaces accessed and modified by the small brained species Homo naledi impacts assertations that technological and cognitive advances in human evolution are associated solely with the evolution of larger brains. One-Sentence SummaryBurials and related meaning making in a small-brained hominin alter our understandings of human evolution.\n",
  "wordCount" : "2768",
  "inLanguage": "en",
  "datePublished": "2023-07-02T10:39:32Z",
  "dateModified": "2023-07-02T10:39:32Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on July 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545611">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545611" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545611">
        <p class="paperTitle">Genomic and ecological factors shaping specialism and generalism across an entire subphylum</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545611" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545611" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Opulente, D. A.; LaBella, A. L.; Harrison, M.-C.; Wolters, J. F.; Liu, C.; Li, Y.; Kominek, J.; Steenwyk, J. L.; Stoneman, H. R.; VanDenAvond, J.; Miller, C. R.; Langdon, Q. K.; Silva, M.; Goncalves, C.; Ubbelohde, E. J.; Li, Y.; Buh, K. V.; Jarzyna, M.; Haase, M. A. B.; Rosa, C. A.; Cadez, N.; Libkind, D.; DeVirgilio, J. H.; Hulfachor, A. B.; Kurtzman, C. P.; Sampaio, J. P.; Goncalves, P.; Zhou, X.; Shen, X.-X.; Groenewald, M.; Rokas, A.; Hittinger, C. T.</p>
        <p class="info">Score: 72.2, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545611' target='https://doi.org/10.1101/2023.06.19.545611'> 10.1101/2023.06.19.545611</a></p>
        <p class="abstract">Organisms exhibit extensive variation in ecological niche breadth, from very narrow (specialists) to very broad (generalists). Paradigms proposed to explain this variation either invoke trade-offs between performance efficiency and breadth or underlying intrinsic or extrinsic factors. We assembled genomic (1,154 yeast strains from 1,049 species), metabolic (quantitative measures of growth of 843 species in 24 conditions), and ecological (environmental ontology of 1,088 species) data from nearly all known species of the ancient fungal subphylum Saccharomycotina to examine niche breadth evolution. We found large interspecific differences in carbon breadth stem from intrinsic differences in genes encoding specific metabolic pathways but no evidence of trade-offs and a limited role of extrinsic ecological factors. These comprehensive data argue that intrinsic factors driving microbial niche breadth variation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.01.543133">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.01.543133" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.01.543133">
        <p class="paperTitle">241,000 to 335,000 Years Old Rock Engravings Made by Homo naledi in the Rising Star Cave system, South Africa.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.01.543133" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.01.543133" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Berger, L. R.; Hawks, J.; Fuentes, A.; van Rooyen, D.; Tsikoane, M.; Ramalepa, M.; Nkwe, S.; Molopyane, K.</p>
        <p class="info">Score: 1758.0, Published: 2023-06-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.01.543133' target='https://doi.org/10.1101/2023.06.01.543133'> 10.1101/2023.06.01.543133</a></p>
        <p class="abstract">The production of painted, etched or engraved designs on cave walls or other surfaces is recognized as a major cognitive step in human evolution. Such intentional designs, which are widely interpreted as signifying, recording, and transmitting information in a durable manner were once considered exclusive to Late Pleistocene Homo sapiens. Recent work has demonstrated that other hominin groups also made such marks, including Neanderthals (Rodriguez-Vidal et al., 2014; Hoffmann et al., 2018), and possibly Middle-Pleistocene Homo erectus (Joordens et al., 2015). Such durable signs indicate an intentionality characteristic of meaning-making (Kissel and Fuentes 2018) which has been argued to require significant levels of cognitive abilities not found in species with smaller brain sizes (Parkington, 2010). In fact, the evolution of such meaning-making symbols is thought to be a core aspect of what it means to be &#34;human&#34; (Henshilwood, 2009). Here we present the first known example of abstract patterns and shapes engraved within the Dinaledi subsystem of the Rising Star Cave in South Africa. We identified markings incised into the dolomitic limestone walls of the cave. The engravings described here are deeply impressed cross-hatchings and other geometric shapes. The surfaces bearing these engravings appear to have been prepared and smoothed. In some areas there is residue that creates a sheen on the surface possibly indicating repeated handling or rubbing of the rock, and there is evidence of the application of dirt or sand to the surface by non-natural processes. Homo naledi entered this part of the cave system and buried bodies within the both the Dinaledi Chamber and adjacent Hill Antechamber between 241 and 335 ka (Dirks et al., 2017; Robbins et al., 2021, Berger et al, 2023a). The engravings described here are found on a pillar in the Hill Antechamber that extends into the natural fissure corridor that links the two chambers and we associate them with H. naledi.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545734">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545734" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545734">
        <p class="paperTitle">Chance favors the prepared genomes: horizontal transfer shapes the emergence of antibiotic resistance mutations in core genes.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545734" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545734" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Coluzzi, C.; Guillemet, M.; Mazzamurro, F.; Touchon, M.; Godfroid, M.; Achaz, G.; Glaser, P.; Rocha, E. P. C.</p>
        <p class="info">Score: 52.5, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545734' target='https://doi.org/10.1101/2023.06.20.545734'> 10.1101/2023.06.20.545734</a></p>
        <p class="abstract">Bacterial lineages vary in the frequency with which they acquire novel traits, like antibiotic resistance or virulence. While previous studies have highlighted the impact of the genetic background on the successful acquisition of novel traits through horizontal gene transfer, the impact of the latter on the subsequent evolution of bacterial genomes by point mutations remains poorly understood. Here, we studied the evolution of resistance to quinolones in thousands of Escherichia coli genomes. Resistance-conferring point mutations in the core genes are frequent and accumulate very quickly. We searched for gene gains and losses significantly associated with the subsequent acquisition of these resistance mutations. This revealed 60 groups of genes in genetic linkage whose gain or loss induced a change in the probability of subsequently becoming resistant to quinolones by point mutations in gyrA and parC. Although some of these chronologies may reflect epidemiological trends, most of these groups encoded functions that were previously associated with antibiotic resistance, tolerance, or persistence, often specifically under quinolone treatment. A lot of the largest groups were found in prophages or plasmids, and they usually increased the likelihood of subsequent resistance mutations. Conversely groups of lost genes were typically small and chromosomal. Quinolone resistance was among the first resistances acquired in the extant lineages of E. coli and its acquisition was associated with an increased likelihood of acquiring other types of resistances, including to aminoglycosides and beta-lactams. Our findings suggest that gene flow shapes the subsequent fixation rate of adaptive mutations in core genes. Given the substantial gene flow within bacterial genomes, interactions between horizontal transfer and point mutations in core genes may be key to the success of adaptation processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547117">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547117" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547117">
        <p class="paperTitle">Deep Divergences Among Cryptic Clades of Epipedobates Poison Frogs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547117" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547117" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lopez-Hervas, K.; Santos, J. C.; Ron, S. R.; Betancourth-Cundar, M.; Cannatella, D. C.; Tarvin, R. D.</p>
        <p class="info">Score: 23.9, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547117' target='https://doi.org/10.1101/2023.06.29.547117'> 10.1101/2023.06.29.547117</a></p>
        <p class="abstract">Poison frogs (Dendrobatidae) are famous for their aposematic species, having a combination of diverse color patterns and defensive skin toxins, yet most species in this family are inconspicuous (cryptic) and considered non-aposematic. Epipedobates is among the youngest genus-level clades of Dendrobatidae that includes both aposematic and cryptic species. Using mitochondrial and nuclear markers, we demonstrate deep genetic divergences among cryptic species of Epipedobates but relatively shallow genetic divergences among conspicuous species. Our phylogenetic analysis includes broad geographic sampling of the cryptic lineages typically identified as E. boulengeri and E. espinosai, which reveals two putative new species, one in west-central Colombia (E. sp. 1) and the other in north-central Ecuador (E. aff. espinosai). We conclude that E. darwinwallacei is a junior subjective synonym of E. espinosai. We also clarify the geographic distributions of cryptic Epipedobates species including the widespread E. boulengeri. We provide a qualitative assessment of the phenotypic diversity in each nominal species, with a focus on the color and pattern of cryptic species. We conclude that Epipedobates contains eight known valid species, six of which are cryptic. A relaxed molecular clock analysis suggests that the most recent common ancestor of Epipedobates is ~11.1 million years old, which nearly doubles previous estimates. Last, genetic information points to a center of species diversity in the Choco at the southwestern border of Colombia with Ecuador.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546191">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546191" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546191">
        <p class="paperTitle">rphylopic: An R package for fetching, transforming, and visualising PhyloPic silhouettes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546191" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546191" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gearty, W.; Jones, L. A.</p>
        <p class="info">Score: 23.8, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546191' target='https://doi.org/10.1101/2023.06.22.546191'> 10.1101/2023.06.22.546191</a></p>
        <p class="abstract">O_LIData visualisation is vital for data exploration, analysis, and communication in research. Moreover, it can bridge gaps between researchers and the general public by making research findings more accessible and engaging. Today, researchers increasingly conduct their data analyses in programming languages such as R and Python. The availability of data visualisation tools within these environments supports the generation of reproducible data analyses and visualisation workflows. However, resources from external databases are frequently required for data visualisation necessitating integration between these databases and the platforms in which researchers conduct their analyses.
C_LIO_LIHere, we introduce rphylopic, an R package for fetching, transforming, and visualising silhouettes of organisms from the PhyloPic database. In addition to making over 7,000 organism silhouettes available within the R programming language, rphylopic empowers users to modify the appearance of these silhouettes for ultimate customisability when coding production-quality visualisations in both base R and ggplot2 workflows.
C_LIO_LIIn this work, we provide details about how the package can be installed, its implementation, and potential use cases. For the latter, we showcase three examples across the ecology and evolutionary biology spectrum.
C_LIO_LIOur hope is that rphylopic will make it easier for biologists to develop more accessible and engaging data visualisations by making external resources readily accessible, customisable, and usable within R. In turn, by integrating into existing workflows, rphylopic helps to ensure that science is reproducible and accessible.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546833">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546833" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546833">
        <p class="paperTitle">Reconstructing the evolutionary history of human, simian, and prosimian immunodeficiency viruses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546833" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546833" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ghafari, M.; Simmonds, P.; Katzourakis, A.</p>
        <p class="info">Score: 19.1, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546833' target='https://doi.org/10.1101/2023.06.28.546833'> 10.1101/2023.06.28.546833</a></p>
        <p class="abstract">While paleovirological evidence supports the idea that the ancestors of modern simian (SIV) and prosimian (pSIV) immunodeficiency viruses have been evolving in nonhuman primates for millions of years, standard molecular clock methods calibrated using contemporary sequences significantly underestimate their times of origin. This discrepancy is attributed to the time-dependent nature of evolutionary rate estimates whereby the rate of virus evolution varies with respect to the time window of its measurement. While biogeographical calibrations may provide better estimates across intermediate timescales, there is currently no unifying framework that can allow inference of lentiviral ages across all relevant timescales. Here we showed that by correcting for such time-dependent rate effects using the recently developed Prisoner-of-War evolutionary model, we can successfully reconstruct the evolutionary history of human, simian, and prosimian immunodeficiency viruses across vastly different timescales and allowed estimates of the age of lineages that gave rise to HIV-1 and HIV-2. The model also predicted that the most recent common ancestor of SIV and pSIV lived around 21 million years ago, suggesting that the most likely explanation for the origins of primate lentiviruses is a terrestrial transfer of lentiviruses between African and Malagasy primates during the last episode of colonisation of Madagascar. Infections have entered via a land bridge in a group of mammals or through a nonprimate vector or transfer mediated by an aerial vector species. We also found that the most recent common ancestor of SIVs lived nearly a million years ago and that some SIV lineages codiverged with their hosts for several hundreds of thousands of years. The predicted long evolutionary timescales of SIVs and potential for close virus/host co-adaptation is consistent with their reduced or minimal pathogenicity in their native hosts, contrasting with the very recent evolutionary origins of highly pathogenic HIV strains in humans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.544912">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.544912" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.544912">
        <p class="paperTitle">Ancient Sheep Genomes reveal four Millennia of North European Short-Tailed Sheep in the Baltic Sea region</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.544912" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.544912" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Larsson, M. N.; Morell Miranda, P.; Pan, L.; Basak Vural, K.; Kaptan, D.; Rodrigues Soares, A. E.; Kivikero, H.; Kantanen, J.; Somel, M.; Özer, F.; Johansson, A. M.; Stora, J.; Günther, T.</p>
        <p class="info">Score: 18.5, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.544912' target='https://doi.org/10.1101/2023.06.26.544912'> 10.1101/2023.06.26.544912</a></p>
        <p class="abstract">Sheep are among the earliest domesticated livestock species, with a wide variety of breeds present today. However, it remains unclear how far back this breed diversity goes, with formal documentation only dating back a few centuries. North European short-tailed breeds are often assumed to be among the oldest domestic sheep populations, even thought to represent relicts of the earliest sheep expansions during the Neolithic period reaching Scandinavia less than 6000 years ago. This study sequenced the genomes (up to 11.6X) of five sheep remains from the Baltic islands of Gotland and [A]land, dating from Late Neolithic ([~]4100 calBP) to historical times ([~]1600 CE). Our findings indicate that these ancient sheep largely possessed the genetic characteristics of modern North European short-tailed breeds, suggesting a substantial degree of long-term continuity of this breed type in the Baltic Sea region. Despite the wide temporal spread, population genetic analyses show high levels of affinity between the ancient genomes and they also exhibit higher genetic diversity when compared to modern breeds, implying a loss of diversity in recent centuries associated with breed formation. Finally, we see a potential signature of an even earlier, genetically different form of sheep in Scandinavia as these samples do not represent the first sheep in Northern Europe. Our results shed light on the development of breeds in Northern Europe specifically as well as the development of genetic diversity in sheep breeds, and their expansion from the domestication center in general.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546307">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546307" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546307">
        <p class="paperTitle">A multi-layered integrative analysis reveals a cholesterol metabolic program in outer radial glia with implications for human brain evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546307" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546307" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moriano, J.; Leonardi, O.; Vitriolo, A.; Testa, G.; Boeckx, C.</p>
        <p class="info">Score: 17.9, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546307' target='https://doi.org/10.1101/2023.06.23.546307'> 10.1101/2023.06.23.546307</a></p>
        <p class="abstract">The definition of molecular and cellular mechanisms contributing to evolutionary divergences in brain ontogenetic trajectories is essential to formulate hypotheses about the emergence of our species. Yet the functional dissection of evolutionary modifications derived in the Homo sapiens lineage at an appropriate level of granularity remains particularly challenging. Capitalizing on recent single-cell sequencing efforts that have massively profiled neural stem cells from the developing human cortex, we develop an integrative computational framework in which we perform (i) trajectory inference and gene regulatory network reconstruction, (ii) (pseudo)time-informed non-negative matrix factorization for learning the dynamics of gene expression programs, and (iii) paleogenomic analysis for a higher-resolution mapping of the regulatory landscape where our species acquired derived mutations in comparison to our closest relatives. We provide evidence for cell type-specific activation and regulation of gene expression programs during indirect neurogenesis. In particular, our analysis uncovers a zinc-finger transcription factor, KLF6, as a key regulator of a cholesterol metabolic program specifically in outer radial glia. Our strategy allows us to further probe whether the (semi)discrete gene expression programs identified have been under selective pressures in our species lineage. A cartography of the regulatory landscape impacted by Homo sapiens-derived transcription factor binding site disruptions reveals signals of selection clustering around regulatory regions associated with GLI3, a well-known regulator of the radial glial cell cycle. As a whole, our study contributes to the evidence of significant changes impacting metabolic pathways in recent human brain evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.546893">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.546893" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.546893">
        <p class="paperTitle">Standing genetic variation and de novo mutations underlie parallel evolution of island bird phenotypes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.546893" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.546893" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Estandia, A.; Sendell-Price, A. T.; Robertson, B. C.; Clegg, S. M.</p>
        <p class="info">Score: 17.9, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.546893' target='https://doi.org/10.1101/2023.06.29.546893'> 10.1101/2023.06.29.546893</a></p>
        <p class="abstract">Parallel evolution occurs when the same trait evolves in closely related lineages in response to similar ecological contexts and provides some of the best examples of determinism in evolutionary biology. However, from a genetic standpoint, this process can be driven by either new mutations that appear independently in each diverging population or by selection on existing genetic variation common to both lineages. Small bodied birds, for example, tend to increase in size after they colonise a new island, following what is known as the &#39;island rule&#39;. Such is the case of the Silvereye, a prolific natural coloniser of southwest Pacific islands. Island forms of this bird species increase in body size after they establish, with the pattern and pace of change consistent with directional natural selection and evident even in the most recent colonisations within the last 200 years. The system provides an exceptional opportunity to explore the genomic basis of repeated body size evolution. We sequenced 377 whole genomes from 31 different Silvereye populations, which revealed that both mechanisms are at play: in some lineages, new mutations are highly associated with body and bill size, but there are also highly associated polymorphisms present across all populations. Our research sheds light on the genomic basis of repeated body size evolution and emphasises that multiple molecular mechanisms can underlie similar evolutionary trajectories even within a single taxon.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.01.543135">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.01.543135" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.01.543135">
        <p class="paperTitle">Burials and engravings in a small-brained hominin, Homo naledi, from the late Pleistocene: contexts and evolutionary implications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.01.543135" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.01.543135" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fuentes, A.; Kissel, M.; Spikins, P.; Molopyane, K.; Hawks, J.; Berger, L. R.</p>
        <p class="info">Score: 1457.8, Published: 2023-06-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.01.543135' target='https://doi.org/10.1101/2023.06.01.543135'> 10.1101/2023.06.01.543135</a></p>
        <p class="abstract">Data from recent explorations in the Dinaledi subsystem illustrates one of the earliest examples of a mortuary practice in hominins and offers the earliest evidence of multiple interments and funerary actions, as well as evidence of the early creation of meaning making by a hominin. The hominin undertaking these behaviors was the small-brained Homo naledi. These data call into question several key assumptions about behavioral and cognitive evolution in Pleistocene hominins. The evidence from Dinaledi push back the temporal origins of mortuary and funerary behaviors and associate the creation of meaning making with a small-brained species and thus challenge key assumptions about the role and importance of encephalization in human evolution. This suggests that the hominin socio-cognitive niche and its relation to meaning-making activities is more diverse than previously thought. The association of these activities in subterranean spaces accessed and modified by the small brained species Homo naledi impacts assertations that technological and cognitive advances in human evolution are associated solely with the evolution of larger brains.

One-Sentence SummaryBurials and related meaning making in a small-brained hominin alter our understandings of human evolution.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
