<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times
Authors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times
Authors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-11T10:39:46+00:00" />
<meta property="article:modified_time" content="2023-10-11T10:39:46+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times
Authors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times\nAuthors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E.",
  "keywords": [
    
  ],
  "articleBody": " The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times\nAuthors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E.\nScore: 95.2, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561165\nInfectious diseases have had devastating impacts on human populations throughout history. Still, the origins and past dynamics of human pathogens remain poorly understood (1). To create the first spatiotemporal map of diverse ancient human microorganisms and parasites, we screened shotgun sequencing data from 1,313 ancient human remains covering 35,000 years of Eurasian history for ancient DNA deriving from bacteria, viruses, and parasites. We demonstrate the widespread presence of ancient microbial DNA in human remains, identifying over 2,400 individual species hits in 896 samples. We report a wide range of pathogens detected for the first time in ancient human remains, including the food-borne pathogens Yersinia enterocolitica and Shigella spp., the animal-borne Leptospira interrogans, and the malaria-causing parasite Plasmodium vivax. Our findings extend the spatiotemporal range of previously described ancient pathogens such as Yersinia pestis, the causative agent of plague, Hepatitis B virus, and Borrelia recurrentis, the cause of louse-borne relapsing fever (LBRF). For LRBF we increase the known distribution from a single medieval genome to 31 cases across Eurasia covering 5,000 years. Grouping the ancient microbial species according to their type of transmission (zoonotic, anthroponotic, sapronotic, opportunistic, and other), we find that most categories are identified throughout the entire sample period, while zoonotic pathogens, which are transmitted from living animals to humans or which have made a host jump into humans from animals in the timeframe of this study, are only detected from ~6,500 years ago. The incidence of zoonotic pathogens increased in our samples some 1,000 years later before reaching the highest detection rates ~5,000 years ago, and was associated with a human genetic ancestry component characteristic of pastoralist populations from the Eurasian Steppe. Our results provide the first direct evidence for an epidemiological transition to an increased burden of zoonotic infectious diseases following the domestication of animals (2). However, they also reveal that the spread of these pathogens first becomes frequent thousands of years after increased animal-human contact, likely coinciding with the pastoralist migrations from the Eurasian Steppe (3,4). This study provides the first spatiotemporal map of past human pathogens using genomic paleoepidemiology, and the first direct evidence for an epidemiological transition of increased zoonotic infectious disease burden after the onset of agriculture, through historical times.\nThe amino acid sequence determines protein abundance through its conformational stability and reduced synthesis cost.\nAuthors: Buric, F.; Viknander, S.; Fu, X.; Lemke, O.; Zrimec, J.; Szyrwiel, L.; Muelleder, M.; Ralser, M.; Zelezniak, A.\nScore: 43.0, Published: 2023-10-04 DOI: 10.1101/2023.10.02.560091\nUnderstanding what drives protein abundance is essential to biology, medicine, and biotechnology. Driven by evolutionary selection, the amino acid sequence is tailored to meet the required abundance of proteomes, underscoring the intricate relationship between sequence and functional demand. Yet, the specific role of amino acid sequences in determining proteome abundance remains elusive. Here, we demonstrate that the amino acid sequence predicts abundance by shaping a proteins conformational stability. We show that increasing the abundance provides metabolic cost benefits, underscoring the evolutionary advantage of maintaining a highly abundant and stable proteome. Specifically, using a deep learning model (BERT), we predict 56% of protein abundance variation in Saccharomyces cerevisiae solely based on amino acid sequence. The model reveals latent factors linking sequence features to protein stability. To probe these relationships, we introduce MGEM (Mutation Guided by an Embedded Manifold), a methodology for guiding protein abundance through sequence modifications. We find that mutations increasing abundance significantly alter protein polarity and hydrophobicity, underscoring a connection between protein stability and abundance. Through molecular dynamics simulations and in vivo experiments in yeast, we confirm that abundance-enhancing mutations result in longer-lasting and more stable protein expression. Importantly, these sequence changes also reduce metabolic costs of protein synthesis, elucidating the evolutionary advantage of cost-effective, high-abundance, stable proteomes. Our findings support the role of amino acid sequence as a pivotal determinant of protein abundance and stability, revealing an evolutionary optimization for metabolic efficiency.\nRecurrent duplication and diversification of a vital DNA repair gene family across Drosophila\nAuthors: Brand, C. L.; Oliver, G. T.; Farkas, I. Z.; Levine, M. T.\nScore: 34.2, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560779\nMaintaining genome integrity is vital for organismal survival and reproduction. Essential, broadly conserved DNA repair pathways actively preserve genome integrity. However, many DNA repair proteins evolve adaptively. Ecological forces like UV exposure are classically cited as drivers of DNA repair evolution. Intrinsic forces like repetitive DNA, which can also imperil genome integrity, have received less attention. We recently reported that a Drosophila melanogaster-specific DNA satellite array triggered species-specific, adaptive evolution of a DNA repair protein called Spartan/MH. The Spartan family of proteases cleave hazardous, covalent crosslinks that form between DNA and proteins (\"DNA-protein crosslink repair\"). Appreciating that DNA satellites are both ubiquitous and universally fast-evolving, we hypothesized that satellite DNA turnover spurs evolution of DNA-protein crosslink repair beyond D. melanogaster. This hypothesis predicts pervasive Spartan gene family diversification across the Drosophila phylogeny. To study the evolutionary history of the Drosophila Spartan gene family, we conducted population genetic, molecular evolution, phylogenomic, and tissue-specific expression analyses. We uncovered widespread signals of positive selection across multiple Spartan family genes and across multiple evolutionary timescales. We also detected recurrent Spartan family gene duplication, divergence, and gene loss. Finally, we found that ovary-enriched parent genes consistently birthed testis-enriched daughter genes. To account for Drosophila-wide, Spartan family diversification, we introduce a mechanistic model of antagonistic coevolution that links DNA satellite evolution and adaptive regulation of Spartan protease activity. This framework, combined with a recent explosion of genome assemblies that encompass repeat-rich genomic regions, promises to accelerate our understanding of how DNA repeats drive recurrent evolutionary innovation to preserve genome integrity.\nSingle-fly assemblies fill major phylogenomic gaps across the Drosophilidae Tree of Life\nAuthors: Kim, B. Y.; Gellert, H. R.; Church, S. H.; Suvorov, A.; Anderson, S. S.; Barmina, O.; Beskid, S. G.; Comeault, A. A.; Crown, K. N.; Diamond, S. E.; Dorus, S.; Fujichika, T.; Hemker, J. A.; Hrcek, J.; Kankare, M.; Katoh, T.; Magnacca, K. N.; Martin, R. A.; Matsunaga, T.; Medeiros, M. J.; Miller, D. E.; Pitnick, S.; Simoni, S.; Steenwinkel, T. E.; Schiffer, M.; Syed, Z. A.; Takahashi, A.; Wei, K. H.-C.; Yokoyama, T.; Eisen, M. B.; Kopp, A.; Matute, D.; Obbard, D. J.; O'Grady, P. M.; Price, D. K.; Toda, M. J.; Werner, T.; Petrov, D. A.\nScore: 58.9, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560517\nLong-read sequencing is driving rapid progress in genome assembly across all major groups of life, including species of the family Drosophilidae, a longtime model system for genetics, genomics, and evolution. We previously developed a cost-effective hybrid Oxford Nanopore (ONT) long-read and Illumina short-read sequencing approach and used it to assemble 101 drosophilid genomes from laboratory cultures, greatly increasing the number of genome assemblies for this taxonomic group. The next major challenge is to address the laboratory culture bias in taxon sampling by sequencing genomes of species that cannot easily be reared in the lab. Here, we build upon our previous methods to perform amplification-free ONT sequencing of single wild flies obtained either directly from the field or from ethanol-preserved specimens in museum collections, greatly improving the representation of lesser studied drosophilid taxa in whole-genome data. Using Illumina Novaseq X Plus and ONT P2 sequencers with R10.4.1 chemistry, we set a new benchmark for inexpensive hybrid genome assembly at US $150 per genome while assembling genomes from as little as 35 ng of genomic DNA from a single fly. We present 183 new genome assemblies for 179 species as a resource for drosophilid systematics, phylogenetics, and comparative genomics. Of these genomes, 62 are from pooled lab strains and 121 from single adult flies. Despite the sample limitations of working with small insects, most single-fly diploid assemblies are comparable in contiguity (\u003e1Mb contig N50), completeness (\u003e98% complete dipteran BUSCOs), and accuracy (\u003eQV40 genome-wide with ONT R10.4.1) to assemblies from inbred lines. We present a well-resolved multi-locus phylogeny for 360 drosophilid and 4 outgroup species encompassing all publicly available (as of August 2023) genomes for this group. Finally, we present a Progressive Cactus whole-genome, reference-free alignment built from a subset of 298 suitably high-quality drosophilid genomes. The new assemblies and alignment, along with updated laboratory protocols and computational pipelines, are released as an open resource and as a tool for studying evolution at the scale of an entire insect family.\nDarwin's Overlooked Radiation: genomic evidence points to the early stages of a radiation in the Galapagos prickly pear cactus (Opuntia, Cactaceae)\nAuthors: Zapata, F.; Cerca, J.; McCarney, D.; Henriquez, C. L.; Tiamiyu, B. B.; McCormack, J.; Reckling, K. R.; Chaves, J. A.; Rivas-Torres, G.\nScore: 15.0, Published: 2023-10-05 DOI: 10.1101/2023.10.03.560736\nThe Galapagos Islands are a prime example of a natural laboratory for the study of evolutionary radiations. While much attention has been devoted to iconic species like Darwin's finches, the islands offer an equally unique but often overlooked opportunity for plant radiations. Yet, compared to their animal counterparts, our understanding of the patterns and processes underpinning Galapagos plant radiations remains relatively limited. We present evidence of the early stages of a radiation in prickly-pear cactus (Opuntia, Cactaceae), a plant lineage widespread across the archipelago. Phylogenomic and population genomic analyses show that notwithstanding overall low genetic differentiation across populations, there is marked geographic structure that is broadly consistent with current taxonomy and the dynamic paleogeography of the Galapagos. Because such low genetic differentiation stands in stark contrast to the exceptional eco-phenotypic diversity displayed by cacti across islands, it is plausible that phenotypic plasticity precedes genetic divergence and is the source of adaptive evolution, or that introgression between populations facilitates local adaptation. Models of population relationships including admixture indicate that gene flow is common between certain islands, likely facilitated by dispersal via animals known to feed on Opuntia flowers, fruits, and seeds across the archipelago. Scans of genetic differentiation between populations reveal candidate loci associated with seed traits and environmental stressors, suggesting that a combination of biotic interactions and abiotic pressures due to the harsh conditions characterizing island life in a volcanic, equatorial archipelago may underlie the diversification of prickly-pear cacti. Considered in concert, these results are relevant to both the mechanisms of plant eco-phenotypic differentiation and the evolutionary history and conservation of the Galapagos biota.\nIntegrating Genomics, Collections, and Community Science to Reveal Speciation in a Variable Monitor Lizard (Varanus tristis)\nAuthors: Pavon-Vazquez, C. J.; Fitch, A. J.; Doughty, P.; Donnellan, S. C.; Keogh, J. S.\nScore: 12.5, Published: 2023-10-05 DOI: 10.1101/2023.10.03.560725\n--The accurate characterization of species diversity is a vital prerequisite for ecological and evolutionary research, as well as conservation. Thus, it is necessary to generate robust hypotheses of species limits based on the inference of evolutionary processes. Integrative species delimitation, the inference of species limits based on multiple sources of evidence, can provide unique insight into species diversity and the processes behind it. However, the application of integrative approaches in non-model organisms is often limited by the amount of data that is available. Here, we show how data relevant for species delimitation can be bolstered by incorporating information from tissue collections, museum specimens, and observations made by the wider community. We show how to integrate these data under a hypothesis-driven, integrative framework by identifying the processes generating genetic and phenotypic variation in Varanus tristis, a widespread and variable complex of Australian monitor lizards. Using genomic, morphometric (linear and geometric), coloration, spatial, and environmental data we show that disparity in this complex is inconsistent with intraspecific variation and instead suggests that speciation has occurred. Based on our results, we identify the environmental factors that may have been responsible for the geographic sorting of variation. Our workflow provides a guideline for the integrative analysis of several types of data to identify the occurrence and causes of speciation. Furthermore, our study highlights how community science and machine learning--two tools used here--can be used to accelerate taxonomic research.\nGenome analysis of the Jomon dogs reveals the oldest domestic dog lineage in Eastern Eurasia\nAuthors: Terai, Y.; Xiaokaiti, X.; Gojobori, J.; Arakawa, N.; Sato, T.; Kasai, K.; Machida, K.; Yamazaki, K.; Yamaji, N.; Hongo, H.; Gakuhari, T.\nScore: 22.7, Published: 2023-10-01 DOI: 10.1101/2023.09.29.560089\nDog is the oldest domesticated animal that established close relationships with humans. Due to its ancient origin, when, where, and whether a single or dual domestication event occurred is still under debate. The dogs in the Jomon period (Jomon dogs) in the Japanese archipelago had little change in morphology from 10,000 to 3,000 years ago. Therefore, we expected that the ancient genome of the Jomon dogs would provide a clue to reveal the characteristics of the ancient East Asian dogs. Here, we have sequenced the genomes of three 6000-year-old Jomon dogs, one 3000-4000-year-old Jomon dog, and four late 8th century dogs excavated in Japan. Our analyses suggest that the Jomon dogs are a distinct lineage from the previously known ancient dogs and are one of the oldest among the dogs in East Eurasian lineages. In addition, the genome of the Jomon dogs contained 9.5% of the genome of Japanese wolf ancestry due to a single introgression event. We estimated the proportion of the Jomon dog lineage genome in the genomes of dogs, which indicates that the genomic composition derived from the Jomon dog lineage is one of the major sources of modern dog genomes. Furthermore, we estimated the early admixture events of dogs in East Eurasia by analyzing the ancient genomes of the Jomon dogs. Due to the admixture events, the Jomon dog-derived genome has been one of the genomic sources of a wide range of modern dogs.\nRapid evolution of pesticide resistance via adaptation and interspecific introgression in a major North American crop pest\nAuthors: North, H. L.; Fu, Z.; Metz, R.; Stull, M. A.; Johnson, C. D.; Shirley, X.; Crumley, K.; Reisig, D.; Kerns, D. L.; Gilligan, T.; Walsh, T.; Jiggins, C. D.; Sword, G.\nScore: 11.7, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560869\nInsect crop pests threaten global food security. This threat is amplified through the spread of non-native species and the evolution of pesticide resistance, which can be introduced to a population though de novo mutation or gene flow. We investigate these processes in an economically important noctuid crop pest, Helicoverpa zea, which has rapidly evolved resistance to several pesticides. Its sister species Helicoverpa armigera, first detected as an invasive species in Brazil in 2013, introduced the pyrethroid resistance gene CYP337B3 to South American H. zea via introgression. To understand whether this contributes to pesticide resistance in North America, we sequenced 237 H. zea genomes across 10 sample sites in the US. First, we report H. armigera introgression into the North American H. zea population. Two individuals sampled in Texas in 2019 carry H. armigera haplotypes in a 4Mbp region containing CYP337B3. Second, we show that the remarkable dispersal ability of H. zea results in a panmictic North American population. Third, we detect signatures of selection in non-admixed H. zea, identifying a selective sweep at a second pesticide resistance locus with a similar name: CYP333B3. We estimate that its derived allele conferred a ~4.9% fitness advantage and show that this estimate explains independently observed rare nonsynonymous CYP333B3 mutations approaching fixation over a ~20-year period. We also detect putative signatures of selection at a kinesin gene associated with Bt resistance. Our results show that pesticide resistance in H. zea evolved rapidly and recently via two independent mechanisms: interspecific introgression and rapid intraspecific adaptation.\nTrade-offs, trade-ups, and high mutational parallelism underlie microbial adaptation to extreme feast/famine\nAuthors: Behringer, M. G.; Ho, W.-C.; Miller, S. F.; Worthan, S. B.; Cen, Z.; Stikeleather, R.; Lynch, M.\nScore: 11.3, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560893\nMicrobes are robust organisms capable of rapidly adapting to complex stress, enabling the colonization of harsh environments. In nature, microbes are regularly challenged by starvation, which is a particularly complex stress because resource limitation often co-occurs with changes in pH, osmolarity, and toxin accumulation created by metabolic waste. Often overlooked are the additional complications introduced by eventual resource replenishment as successful microbes must withstand rapid environmental shifts before swiftly capitalizing on replenished resources to avoid invasion by competing species. To understand how microbes navigate trade-offs between growth and survival, ultimately adapting to thrive in environments with extreme fluctuations, we experimentally evolved 16 Escherichia coli populations for 900 days to repeated feast/famine cycles of 100-day starvation before resource replenishment. Using longitudinal population-genomic analysis, we found that evolution in response to extreme feast/famine is characterized by narrow adaptive trajectories with high mutational parallelism and notable mutational order. Genetic reconstructions reveal that early mutations result in trade-offs for biofilm and motility but trade-ups for growth and survival, as these mutations conferred correlated advantages during both short-term and long-term culture. Our results demonstrate how microbes can navigate the adaptive landscapes of regularly fluctuating conditions and ultimately follow mutational trajectories that confer benefits across diverse environments.\nTradeoffs between phage resistance and nitrogen fixation drive the evolution of genes essential for cyanobacterial heterocyst functionality\nAuthors: Kolan, D.; Cattan-Tsaushu, E.; Enav, H.; Freiman, Z.; Malinsky-Rushansky, N.; Ninio, S.; Avrani, S.\nScore: 10.8, Published: 2023-10-05 DOI: 10.1101/2023.10.04.560878\nHarmful blooms caused by diazotrophic (nitrogen-fixing) cyanobacteria are becoming increasingly frequent and negatively impact aquatic environments worldwide. Cyanophages (viruses infecting cyanobacteria) can potentially regulate cyanobacterial blooms, yet cyanobacteria can rapidly acquire mutations that provide protection against phage infection. Here, we provide novel insights into cyanophage:cyanobacteria interactions by characterizing the resistance to phages in two species of diazotrophic cyanobacteria: Nostoc sp. and Cylindrospermopsis raciborskii. Our results demonstrate that phage resistance is associated with a fitness tradeoff by which resistant cyanobacteria have reduced ability to fix nitrogen and/or to survive nitrogen starvation. Furthermore, we use whole genome sequence analysis of 58 Nostoc resistant strains to identify several mutations associated with phage resistance, including in cell surface-related genes, and regulatory genes involved in development and function of heterocysts (cells specialized in nitrogen fixation). Finally, we employ phylogenetic analyses to show that most of these resistance genes are accessory genes whose evolution is impacted by lateral gene transfer events. Together, these results further our understanding of the interplay between diazotrophic cyanobacteria and their phages, and suggest that a tradeoff between phage resistance and nitrogen fixation affects the evolution of cell surface-related genes and of genes involved in heterocyst differentiation and nitrogen fixation.\n",
  "wordCount" : "3093",
  "inLanguage": "en",
  "datePublished": "2023-10-11T10:39:46Z",
  "dateModified": "2023-10-11T10:39:46Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on October 11, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561165">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561165" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561165">
        <p class="paperTitle">The landscape of ancient human pathogens in Eurasia from the Stone Age to historical times</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561165" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561165" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sikora, M.; Canteri, E.; Fernandez-Guerra, A.; Oskolkov, N.; Agren, R.; Hansson, L.; Irving-Pease, E. K.; Muehlemann, B.; Holtsmark Nielsen, S.; Scorrano, G.; Allentoft, M. E.; Seersholm, F. V.; Schroeder, H.; Gaunitz, C.; Stenderup, J.; Vinner, L.; Jones, T. C.; Nystedt, B.; Parkhill, J.; Fugger, L.; Racimo, F.; Kristiansen, K.; Iversen, A. K. N.; Willerslev, E.</p>
        <p class="info">Score: 95.2, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561165' target='https://doi.org/10.1101/2023.10.06.561165'> 10.1101/2023.10.06.561165</a></p>
        <p class="abstract">Infectious diseases have had devastating impacts on human populations throughout history. Still, the origins and past dynamics of human pathogens remain poorly understood (1). To create the first spatiotemporal map of diverse ancient human microorganisms and parasites, we screened shotgun sequencing data from 1,313 ancient human remains covering 35,000 years of Eurasian history for ancient DNA deriving from bacteria, viruses, and parasites. We demonstrate the widespread presence of ancient microbial DNA in human remains, identifying over 2,400 individual species hits in 896 samples. We report a wide range of pathogens detected for the first time in ancient human remains, including the food-borne pathogens Yersinia enterocolitica and Shigella spp., the animal-borne Leptospira interrogans, and the malaria-causing parasite Plasmodium vivax. Our findings extend the spatiotemporal range of previously described ancient pathogens such as Yersinia pestis, the causative agent of plague, Hepatitis B virus, and Borrelia recurrentis, the cause of louse-borne relapsing fever (LBRF). For LRBF we increase the known distribution from a single medieval genome to 31 cases across Eurasia covering 5,000 years. Grouping the ancient microbial species according to their type of transmission (zoonotic, anthroponotic, sapronotic, opportunistic, and other), we find that most categories are identified throughout the entire sample period, while zoonotic pathogens, which are transmitted from living animals to humans or which have made a host jump into humans from animals in the timeframe of this study, are only detected from ~6,500 years ago. The incidence of zoonotic pathogens increased in our samples some 1,000 years later before reaching the highest detection rates ~5,000 years ago, and was associated with a human genetic ancestry component characteristic of pastoralist populations from the Eurasian Steppe. Our results provide the first direct evidence for an epidemiological transition to an increased burden of zoonotic infectious diseases following the domestication of animals (2). However, they also reveal that the spread of these pathogens first becomes frequent thousands of years after increased animal-human contact, likely coinciding with the pastoralist migrations from the Eurasian Steppe (3,4). This study provides the first spatiotemporal map of past human pathogens using genomic paleoepidemiology, and the first direct evidence for an epidemiological transition of increased zoonotic infectious disease burden after the onset of agriculture, through historical times.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560091">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560091" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560091">
        <p class="paperTitle">The amino acid sequence determines protein abundance through its conformational stability and reduced synthesis cost.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560091" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560091" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Buric, F.; Viknander, S.; Fu, X.; Lemke, O.; Zrimec, J.; Szyrwiel, L.; Muelleder, M.; Ralser, M.; Zelezniak, A.</p>
        <p class="info">Score: 43.0, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560091' target='https://doi.org/10.1101/2023.10.02.560091'> 10.1101/2023.10.02.560091</a></p>
        <p class="abstract">Understanding what drives protein abundance is essential to biology, medicine, and biotechnology. Driven by evolutionary selection, the amino acid sequence is tailored to meet the required abundance of proteomes, underscoring the intricate relationship between sequence and functional demand. Yet, the specific role of amino acid sequences in determining proteome abundance remains elusive. Here, we demonstrate that the amino acid sequence predicts abundance by shaping a proteins conformational stability. We show that increasing the abundance provides metabolic cost benefits, underscoring the evolutionary advantage of maintaining a highly abundant and stable proteome. Specifically, using a deep learning model (BERT), we predict 56% of protein abundance variation in Saccharomyces cerevisiae solely based on amino acid sequence. The model reveals latent factors linking sequence features to protein stability. To probe these relationships, we introduce MGEM (Mutation Guided by an Embedded Manifold), a methodology for guiding protein abundance through sequence modifications. We find that mutations increasing abundance significantly alter protein polarity and hydrophobicity, underscoring a connection between protein stability and abundance. Through molecular dynamics simulations and in vivo experiments in yeast, we confirm that abundance-enhancing mutations result in longer-lasting and more stable protein expression. Importantly, these sequence changes also reduce metabolic costs of protein synthesis, elucidating the evolutionary advantage of cost-effective, high-abundance, stable proteomes. Our findings support the role of amino acid sequence as a pivotal determinant of protein abundance and stability, revealing an evolutionary optimization for metabolic efficiency.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560779">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560779" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560779">
        <p class="paperTitle">Recurrent duplication and diversification of a vital DNA repair gene family across Drosophila</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560779" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560779" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brand, C. L.; Oliver, G. T.; Farkas, I. Z.; Levine, M. T.</p>
        <p class="info">Score: 34.2, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560779' target='https://doi.org/10.1101/2023.10.04.560779'> 10.1101/2023.10.04.560779</a></p>
        <p class="abstract">Maintaining genome integrity is vital for organismal survival and reproduction. Essential, broadly conserved DNA repair pathways actively preserve genome integrity. However, many DNA repair proteins evolve adaptively. Ecological forces like UV exposure are classically cited as drivers of DNA repair evolution. Intrinsic forces like repetitive DNA, which can also imperil genome integrity, have received less attention. We recently reported that a Drosophila melanogaster-specific DNA satellite array triggered species-specific, adaptive evolution of a DNA repair protein called Spartan/MH. The Spartan family of proteases cleave hazardous, covalent crosslinks that form between DNA and proteins (&#34;DNA-protein crosslink repair&#34;). Appreciating that DNA satellites are both ubiquitous and universally fast-evolving, we hypothesized that satellite DNA turnover spurs evolution of DNA-protein crosslink repair beyond D. melanogaster. This hypothesis predicts pervasive Spartan gene family diversification across the Drosophila phylogeny. To study the evolutionary history of the Drosophila Spartan gene family, we conducted population genetic, molecular evolution, phylogenomic, and tissue-specific expression analyses. We uncovered widespread signals of positive selection across multiple Spartan family genes and across multiple evolutionary timescales. We also detected recurrent Spartan family gene duplication, divergence, and gene loss. Finally, we found that ovary-enriched parent genes consistently birthed testis-enriched daughter genes. To account for Drosophila-wide, Spartan family diversification, we introduce a mechanistic model of antagonistic coevolution that links DNA satellite evolution and adaptive regulation of Spartan protease activity. This framework, combined with a recent explosion of genome assemblies that encompass repeat-rich genomic regions, promises to accelerate our understanding of how DNA repeats drive recurrent evolutionary innovation to preserve genome integrity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560517">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560517" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560517">
        <p class="paperTitle">Single-fly assemblies fill major phylogenomic gaps across the Drosophilidae Tree of Life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560517" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560517" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, B. Y.; Gellert, H. R.; Church, S. H.; Suvorov, A.; Anderson, S. S.; Barmina, O.; Beskid, S. G.; Comeault, A. A.; Crown, K. N.; Diamond, S. E.; Dorus, S.; Fujichika, T.; Hemker, J. A.; Hrcek, J.; Kankare, M.; Katoh, T.; Magnacca, K. N.; Martin, R. A.; Matsunaga, T.; Medeiros, M. J.; Miller, D. E.; Pitnick, S.; Simoni, S.; Steenwinkel, T. E.; Schiffer, M.; Syed, Z. A.; Takahashi, A.; Wei, K. H.-C.; Yokoyama, T.; Eisen, M. B.; Kopp, A.; Matute, D.; Obbard, D. J.; O&#39;Grady, P. M.; Price, D. K.; Toda, M. J.; Werner, T.; Petrov, D. A.</p>
        <p class="info">Score: 58.9, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560517' target='https://doi.org/10.1101/2023.10.02.560517'> 10.1101/2023.10.02.560517</a></p>
        <p class="abstract">Long-read sequencing is driving rapid progress in genome assembly across all major groups of life, including species of the family Drosophilidae, a longtime model system for genetics, genomics, and evolution. We previously developed a cost-effective hybrid Oxford Nanopore (ONT) long-read and Illumina short-read sequencing approach and used it to assemble 101 drosophilid genomes from laboratory cultures, greatly increasing the number of genome assemblies for this taxonomic group. The next major challenge is to address the laboratory culture bias in taxon sampling by sequencing genomes of species that cannot easily be reared in the lab. Here, we build upon our previous methods to perform amplification-free ONT sequencing of single wild flies obtained either directly from the field or from ethanol-preserved specimens in museum collections, greatly improving the representation of lesser studied drosophilid taxa in whole-genome data. Using Illumina Novaseq X Plus and ONT P2 sequencers with R10.4.1 chemistry, we set a new benchmark for inexpensive hybrid genome assembly at US $150 per genome while assembling genomes from as little as 35 ng of genomic DNA from a single fly. We present 183 new genome assemblies for 179 species as a resource for drosophilid systematics, phylogenetics, and comparative genomics. Of these genomes, 62 are from pooled lab strains and 121 from single adult flies. Despite the sample limitations of working with small insects, most single-fly diploid assemblies are comparable in contiguity (&gt;1Mb contig N50), completeness (&gt;98% complete dipteran BUSCOs), and accuracy (&gt;QV40 genome-wide with ONT R10.4.1) to assemblies from inbred lines. We present a well-resolved multi-locus phylogeny for 360 drosophilid and 4 outgroup species encompassing all publicly available (as of August 2023) genomes for this group. Finally, we present a Progressive Cactus whole-genome, reference-free alignment built from a subset of 298 suitably high-quality drosophilid genomes. The new assemblies and alignment, along with updated laboratory protocols and computational pipelines, are released as an open resource and as a tool for studying evolution at the scale of an entire insect family.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560736">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560736" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560736">
        <p class="paperTitle">Darwin&#39;s Overlooked Radiation: genomic evidence points to the early stages of a radiation in the Galapagos prickly pear cactus (Opuntia, Cactaceae)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560736" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560736" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zapata, F.; Cerca, J.; McCarney, D.; Henriquez, C. L.; Tiamiyu, B. B.; McCormack, J.; Reckling, K. R.; Chaves, J. A.; Rivas-Torres, G.</p>
        <p class="info">Score: 15.0, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560736' target='https://doi.org/10.1101/2023.10.03.560736'> 10.1101/2023.10.03.560736</a></p>
        <p class="abstract">The Galapagos Islands are a prime example of a natural laboratory for the study of evolutionary radiations. While much attention has been devoted to iconic species like Darwin&#39;s finches, the islands offer an equally unique but often overlooked opportunity for plant radiations. Yet, compared to their animal counterparts, our understanding of the patterns and processes underpinning Galapagos plant radiations remains relatively limited. We present evidence of the early stages of a radiation in prickly-pear cactus (Opuntia, Cactaceae), a plant lineage widespread across the archipelago. Phylogenomic and population genomic analyses show that notwithstanding overall low genetic differentiation across populations, there is marked geographic structure that is broadly consistent with current taxonomy and the dynamic paleogeography of the Galapagos. Because such low genetic differentiation stands in stark contrast to the exceptional eco-phenotypic diversity displayed by cacti across islands, it is plausible that phenotypic plasticity precedes genetic divergence and is the source of adaptive evolution, or that introgression between populations facilitates local adaptation. Models of population relationships including admixture indicate that gene flow is common between certain islands, likely facilitated by dispersal via animals known to feed on Opuntia flowers, fruits, and seeds across the archipelago. Scans of genetic differentiation between populations reveal candidate loci associated with seed traits and environmental stressors, suggesting that a combination of biotic interactions and abiotic pressures due to the harsh conditions characterizing island life in a volcanic, equatorial archipelago may underlie the diversification of prickly-pear cacti. Considered in concert, these results are relevant to both the mechanisms of plant eco-phenotypic differentiation and the evolutionary history and conservation of the Galapagos biota.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560725">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560725" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560725">
        <p class="paperTitle">Integrating Genomics, Collections, and Community Science to Reveal Speciation in a Variable Monitor Lizard (Varanus tristis)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560725" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560725" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pavon-Vazquez, C. J.; Fitch, A. J.; Doughty, P.; Donnellan, S. C.; Keogh, J. S.</p>
        <p class="info">Score: 12.5, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560725' target='https://doi.org/10.1101/2023.10.03.560725'> 10.1101/2023.10.03.560725</a></p>
        <p class="abstract">--The accurate characterization of species diversity is a vital prerequisite for ecological and evolutionary research, as well as conservation. Thus, it is necessary to generate robust hypotheses of species limits based on the inference of evolutionary processes. Integrative species delimitation, the inference of species limits based on multiple sources of evidence, can provide unique insight into species diversity and the processes behind it. However, the application of integrative approaches in non-model organisms is often limited by the amount of data that is available. Here, we show how data relevant for species delimitation can be bolstered by incorporating information from tissue collections, museum specimens, and observations made by the wider community. We show how to integrate these data under a hypothesis-driven, integrative framework by identifying the processes generating genetic and phenotypic variation in Varanus tristis, a widespread and variable complex of Australian monitor lizards. Using genomic, morphometric (linear and geometric), coloration, spatial, and environmental data we show that disparity in this complex is inconsistent with intraspecific variation and instead suggests that speciation has occurred. Based on our results, we identify the environmental factors that may have been responsible for the geographic sorting of variation. Our workflow provides a guideline for the integrative analysis of several types of data to identify the occurrence and causes of speciation. Furthermore, our study highlights how community science and machine learning--two tools used here--can be used to accelerate taxonomic research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560089">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560089" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560089">
        <p class="paperTitle">Genome analysis of the Jomon dogs reveals the oldest domestic dog lineage in Eastern Eurasia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560089" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560089" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Terai, Y.; Xiaokaiti, X.; Gojobori, J.; Arakawa, N.; Sato, T.; Kasai, K.; Machida, K.; Yamazaki, K.; Yamaji, N.; Hongo, H.; Gakuhari, T.</p>
        <p class="info">Score: 22.7, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560089' target='https://doi.org/10.1101/2023.09.29.560089'> 10.1101/2023.09.29.560089</a></p>
        <p class="abstract">Dog is the oldest domesticated animal that established close relationships with humans. Due to its ancient origin, when, where, and whether a single or dual domestication event occurred is still under debate. The dogs in the Jomon period (Jomon dogs) in the Japanese archipelago had little change in morphology from 10,000 to 3,000 years ago. Therefore, we expected that the ancient genome of the Jomon dogs would provide a clue to reveal the characteristics of the ancient East Asian dogs. Here, we have sequenced the genomes of three 6000-year-old Jomon dogs, one 3000-4000-year-old Jomon dog, and four late 8th century dogs excavated in Japan. Our analyses suggest that the Jomon dogs are a distinct lineage from the previously known ancient dogs and are one of the oldest among the dogs in East Eurasian lineages. In addition, the genome of the Jomon dogs contained 9.5% of the genome of Japanese wolf ancestry due to a single introgression event. We estimated the proportion of the Jomon dog lineage genome in the genomes of dogs, which indicates that the genomic composition derived from the Jomon dog lineage is one of the major sources of modern dog genomes. Furthermore, we estimated the early admixture events of dogs in East Eurasia by analyzing the ancient genomes of the Jomon dogs. Due to the admixture events, the Jomon dog-derived genome has been one of the genomic sources of a wide range of modern dogs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560869">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560869" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560869">
        <p class="paperTitle">Rapid evolution of pesticide resistance via adaptation and interspecific introgression in a major North American crop pest</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560869" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560869" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: North, H. L.; Fu, Z.; Metz, R.; Stull, M. A.; Johnson, C. D.; Shirley, X.; Crumley, K.; Reisig, D.; Kerns, D. L.; Gilligan, T.; Walsh, T.; Jiggins, C. D.; Sword, G.</p>
        <p class="info">Score: 11.7, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560869' target='https://doi.org/10.1101/2023.10.04.560869'> 10.1101/2023.10.04.560869</a></p>
        <p class="abstract">Insect crop pests threaten global food security. This threat is amplified through the spread of non-native species and the evolution of pesticide resistance, which can be introduced to a population though de novo mutation or gene flow. We investigate these processes in an economically important noctuid crop pest, Helicoverpa zea, which has rapidly evolved resistance to several pesticides. Its sister species Helicoverpa armigera, first detected as an invasive species in Brazil in 2013, introduced the pyrethroid resistance gene CYP337B3 to South American H. zea via introgression. To understand whether this contributes to pesticide resistance in North America, we sequenced 237 H. zea genomes across 10 sample sites in the US. First, we report H. armigera introgression into the North American H. zea population. Two individuals sampled in Texas in 2019 carry H. armigera haplotypes in a 4Mbp region containing CYP337B3. Second, we show that the remarkable dispersal ability of H. zea results in a panmictic North American population. Third, we detect signatures of selection in non-admixed H. zea, identifying a selective sweep at a second pesticide resistance locus with a similar name: CYP333B3. We estimate that its derived allele conferred a ~4.9% fitness advantage and show that this estimate explains independently observed rare nonsynonymous CYP333B3 mutations approaching fixation over a ~20-year period. We also detect putative signatures of selection at a kinesin gene associated with Bt resistance. Our results show that pesticide resistance in H. zea evolved rapidly and recently via two independent mechanisms: interspecific introgression and rapid intraspecific adaptation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560893">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560893" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560893">
        <p class="paperTitle">Trade-offs, trade-ups, and high mutational parallelism underlie microbial adaptation to extreme feast/famine</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560893" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560893" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Behringer, M. G.; Ho, W.-C.; Miller, S. F.; Worthan, S. B.; Cen, Z.; Stikeleather, R.; Lynch, M.</p>
        <p class="info">Score: 11.3, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560893' target='https://doi.org/10.1101/2023.10.04.560893'> 10.1101/2023.10.04.560893</a></p>
        <p class="abstract">Microbes are robust organisms capable of rapidly adapting to complex stress, enabling the colonization of harsh environments. In nature, microbes are regularly challenged by starvation, which is a particularly complex stress because resource limitation often co-occurs with changes in pH, osmolarity, and toxin accumulation created by metabolic waste. Often overlooked are the additional complications introduced by eventual resource replenishment as successful microbes must withstand rapid environmental shifts before swiftly capitalizing on replenished resources to avoid invasion by competing species. To understand how microbes navigate trade-offs between growth and survival, ultimately adapting to thrive in environments with extreme fluctuations, we experimentally evolved 16 Escherichia coli populations for 900 days to repeated feast/famine cycles of 100-day starvation before resource replenishment. Using longitudinal population-genomic analysis, we found that evolution in response to extreme feast/famine is characterized by narrow adaptive trajectories with high mutational parallelism and notable mutational order. Genetic reconstructions reveal that early mutations result in trade-offs for biofilm and motility but trade-ups for growth and survival, as these mutations conferred correlated advantages during both short-term and long-term culture. Our results demonstrate how microbes can navigate the adaptive landscapes of regularly fluctuating conditions and ultimately follow mutational trajectories that confer benefits across diverse environments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560878">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560878" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560878">
        <p class="paperTitle">Tradeoffs between phage resistance and nitrogen fixation drive the evolution of genes essential for cyanobacterial heterocyst functionality</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560878" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560878" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kolan, D.; Cattan-Tsaushu, E.; Enav, H.; Freiman, Z.; Malinsky-Rushansky, N.; Ninio, S.; Avrani, S.</p>
        <p class="info">Score: 10.8, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560878' target='https://doi.org/10.1101/2023.10.04.560878'> 10.1101/2023.10.04.560878</a></p>
        <p class="abstract">Harmful blooms caused by diazotrophic (nitrogen-fixing) cyanobacteria are becoming increasingly frequent and negatively impact aquatic environments worldwide. Cyanophages (viruses infecting cyanobacteria) can potentially regulate cyanobacterial blooms, yet cyanobacteria can rapidly acquire mutations that provide protection against phage infection. Here, we provide novel insights into cyanophage:cyanobacteria interactions by characterizing the resistance to phages in two species of diazotrophic cyanobacteria: Nostoc sp. and Cylindrospermopsis raciborskii. Our results demonstrate that phage resistance is associated with a fitness tradeoff by which resistant cyanobacteria have reduced ability to fix nitrogen and/or to survive nitrogen starvation. Furthermore, we use whole genome sequence analysis of 58 Nostoc resistant strains to identify several mutations associated with phage resistance, including in cell surface-related genes, and regulatory genes involved in development and function of heterocysts (cells specialized in nitrogen fixation). Finally, we employ phylogenetic analyses to show that most of these resistance genes are accessory genes whose evolution is impacted by lateral gene transfer events. Together, these results further our understanding of the interplay between diazotrophic cyanobacteria and their phages, and suggest that a tradeoff between phage resistance and nitrogen fixation affects the evolution of cell surface-related genes and of genes involved in heterocyst differentiation and nitrogen fixation.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
