<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A Quantitative Genetic Model of Background Selection in Humans
Authors: Buffalo, V.; Kern, A. D.
Score: 24.0, Published: 2023-09-08 DOI: 10.1101/2023.09.07.556762
Across the human genome, there are large-scale fluctuations in genetic diversity caused by the indirect effects of selection. This &#34;linked selection signal&#34; reflects the impact of selection according to the physical placement of functional regions and recombination rates along chromosomes. Previous work has shown that purifying selection acting against the steady influx of new deleterious mutations at functional portions of the genome shapes patterns of genomic variation.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="A Quantitative Genetic Model of Background Selection in Humans
Authors: Buffalo, V.; Kern, A. D.
Score: 24.0, Published: 2023-09-08 DOI: 10.1101/2023.09.07.556762
Across the human genome, there are large-scale fluctuations in genetic diversity caused by the indirect effects of selection. This &#34;linked selection signal&#34; reflects the impact of selection according to the physical placement of functional regions and recombination rates along chromosomes. Previous work has shown that purifying selection acting against the steady influx of new deleterious mutations at functional portions of the genome shapes patterns of genomic variation." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-10T10:36:49+00:00" />
<meta property="article:modified_time" content="2023-09-10T10:36:49+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="A Quantitative Genetic Model of Background Selection in Humans
Authors: Buffalo, V.; Kern, A. D.
Score: 24.0, Published: 2023-09-08 DOI: 10.1101/2023.09.07.556762
Across the human genome, there are large-scale fluctuations in genetic diversity caused by the indirect effects of selection. This &#34;linked selection signal&#34; reflects the impact of selection according to the physical placement of functional regions and recombination rates along chromosomes. Previous work has shown that purifying selection acting against the steady influx of new deleterious mutations at functional portions of the genome shapes patterns of genomic variation."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "A Quantitative Genetic Model of Background Selection in Humans\nAuthors: Buffalo, V.; Kern, A. D.\nScore: 24.0, Published: 2023-09-08 DOI: 10.1101/2023.09.07.556762\nAcross the human genome, there are large-scale fluctuations in genetic diversity caused by the indirect effects of selection. This \u0026#34;linked selection signal\u0026#34; reflects the impact of selection according to the physical placement of functional regions and recombination rates along chromosomes. Previous work has shown that purifying selection acting against the steady influx of new deleterious mutations at functional portions of the genome shapes patterns of genomic variation.",
  "keywords": [
    
  ],
  "articleBody": " A Quantitative Genetic Model of Background Selection in Humans\nAuthors: Buffalo, V.; Kern, A. D.\nScore: 24.0, Published: 2023-09-08 DOI: 10.1101/2023.09.07.556762\nAcross the human genome, there are large-scale fluctuations in genetic diversity caused by the indirect effects of selection. This \"linked selection signal\" reflects the impact of selection according to the physical placement of functional regions and recombination rates along chromosomes. Previous work has shown that purifying selection acting against the steady influx of new deleterious mutations at functional portions of the genome shapes patterns of genomic variation. To date, statistical efforts to estimate purifying selection parameters from linked selection models have relied on classic Background Selection theory, which is only applicable when new mutations are so deleterious that they cannot fix in the population. Here, we develop a statistical method based on a quantitative genetics view of linked selection, that models how polygenic additive fitness variance distributed along the genome increases the rate of stochastic allele frequency change. By jointly predicting the equilibrium fitness variance and substitution rate due to both strong and weakly deleterious mutations, we estimate the distribution of fitness effects (DFE) and mutation rate across three geographically distinct human samples. While our model can accommodate weaker selection, we find evidence of strong selection operating similarly across all human samples. While our quantitative genetic model of linked selection fits better than previous models, substitution rates of the most constrained sites are poorly predicted. We find that a model incorporating selective interference better predicts observed divergence in conserved regions, but overall our results ultimately suggest considerable uncertainty remains about the processes generating fitness variation in humans.\nLarge-scale analyses reveal the contribution of adaptive evolution in pathogenic and non-pathogenic fungal species\nAuthors: Pereira, D.; Bolton, M. D.; Friesen, T. L.; Stephan, W.; Dutheil, J. Y.; Stukenbrock, E. H.\nScore: 26.3, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555124\nGenome studies of fungal pathogens have presented evidence for exceptionally high rates of evolution. It has been proposed that rapid adaptation is a hallmark of pathogen evolution that facilitates the invasion of new host niches and the overcoming of intervention strategies such as fungicide applications and drug treatments. To which extent high levels of genetic variation within and between species correlate with adaptive protein evolution in fungi more generally has so far not been explored. In this study, we addressed the contribution of adaptive evolution relative to genetic drift in 20 fungal species, hereby exploring genetic variation in 2,478 fungal genomes. We reannotated positions of protein-coding genes to obtain a high-quality dataset of 234,427 full-length core gene and 25,612 accessory gene alignments. We applied an extension of the McDonald-Kreitman test that models the distributions of fitness effects to infer the rate of adaptive ({omega}A) and non-adaptive ({omega}NA) non-synonymous substitutions in protein-coding genes. To explore the relevance of recombination on local adaptation rates, we inferred the population genomic recombination rate for all 20 species. Our analyses reveal extensive variation in rates of adaptation and show that high rates of adaptation are not a hallmark of a pathogenic lifestyle. Up to 83% of non-synonymous substitutions are adaptive in the species Parastagonospora nodorum. However, non-synonymous substitutions in other species, including the prominent rice-infecting pathogen Magnaporthe oryzae, are predominantly non-adaptive (neutral or slightly deleterious). Correlating adaptation measures with effective population size and recombination rate, we show that effective population size is a primary determinant of adaptive evolution in fungi. At the genome scale, recombination rate variation explains variation in both {omega}A and {omega}NA. Finally, we demonstrate the robustness of our estimates using simulations. We underline the value of population genetic principles in studies of fungal evolution, and we highlight the importance of demographic processes in adaptive evolution of pathogenic and non-pathogenic species.\nSpeciation in kleptoparasites of oak gall wasps often correlates with shifts into new tree habitats, tree organs, and gall morphospace\nAuthors: Ward, A.; Zhang, Y. M.; Brown, G. E.; Hippee, A. C.; Prior, K. M.; Rollins, S.; Sierra, N.; Sheikh, S. I.; Tribull, C. M.; Forbes, A. A.\nScore: 18.2, Published: 2023-09-07 DOI: 10.1101/2023.09.07.556376\nHost shifts to new plants can drive speciation for plant-feeding insects, but how commonly do host shifts also drive diversification for the parasites of those same insects? Oak gall wasps induce galls on oak trees and shifts to novel tree hosts and new tree organs have been implicated as drivers of oak gall wasp speciation. Gall wasps are themselves attacked by many insect parasites, which must find their hosts on the correct tree species and organ, but which also must navigate the morphologically variable galls with which they interact. Thus, we ask whether host shifts to new trees, organs, or gall morphologies correlate with gall parasite diversification. We delimit species and infer phylogenies for two genera of gall kleptoparasites, Synergus and Ceroptres, reared from a variety of North American oak galls. We find that most species were reared from galls induced by just one gall wasp species, and no parasite species was reared from galls of more than four species. Most kleptoparasite divergence events correlate with shifts to non-ancestral galls. These shifts often involved changes in tree habitat, gall location, and gall morphology. Host shifts are thus implicated in driving diversification for both oak gall wasps and their kleptoparasitic associates.\nDetecting shifts in the mode of chromosomal speciation across the cosmopolitan plant lineage Carex\nAuthors: Tribble, C. M.; Marquez Corro, J. I.; May, M. R.; Hipp, A. L.; Escudero, M.; Zenil-Ferguson, R.\nScore: 15.9, Published: 2023-09-10 DOI: 10.1101/2023.09.05.556433\nRecent studies of angiosperm diversification have focused on the role of polyploidy as a driver of diversification. However, we know far less about the effects of single changes in chromosome number---dysploidy, which can mediate lineage diversification by affecting recombination rates, linkage, and/or reproductive isolation. Modeling the effects of dysploidy on diversification is mathematically and computationally challenging because many states and parameters are required to track changes in individual chromosomes, especially in lineages that have high variability in chromosome number. Additionally, we expect the processes of diversification and dysploidy to vary across clades, which requires modeling process variation to disentangle the effects of the observed trait (chromosome number) from the effects of unobserved traits on diversification. In this work, we propose a new state-dependent diversification model of chromosome evolution that includes numerous character states and explicitly models heterogeneity in the diversification process. Our model includes parameters that functionally link diversification rates to dysploidy rates and differentiate between anagenetic and cladogenetic changes. We apply this model to Carex (Cyperaceae), a model lineage for understanding dysploidy and diversification, leveraging chromosome number information and the most recent time-calibrated phylogenetic tree for over 700 species and subspecies. We recover distinct modes of chromosomal speciation across Carex. In one mode, dysploidy occurs very frequently and drives faster diversification rates. In the other mode, dysploidy is rare and diversification is driven by other factors, unmeasured in our analysis. This study is the first to demonstrate that dysploidy drives diversification in plants while considering unmeasured factors affecting diversification.\nHuman deleterious mutation rate implies high fitness variance, with declining mean fitness compensated by rarer beneficial mutations of larger effect\nAuthors: Matheson, J. D.; Masel, J.; Bertram, J.\nScore: 15.5, Published: 2023-09-04 DOI: 10.1101/2023.09.01.555871\nEach new human has an expected Ud =2 - 10 new deleterious mutations. This deluge of deleterious mutations cannot all be purged, and therefore accumulate in a declining fitness ratchet. Using a novel simulation framework designed to efficiently handle genome-wide linkage disequilibria across many segregating sites, we find that rarer, beneficial mutations of larger effect are sufficient to compensate fitness declines due to the fixation of many slightly deleterious mutations. Drift barrier theory posits a similar asymmetric pattern of fixations to explain ratcheting genome size and complexity, but in our theory, the cause is Ud \u003e1 rather than small population size. In our simulations, Ud [~]2 - 10 generates high within-population variance in relative fitness; two individuals will typically differ in fitness by 15- 40%. Ud [~]2 - 10 also slows net adaptation by [~]13%-39%. Surprisingly, fixation rates are more sensitive to changes in the beneficial than the deleterious mutation rate, e.g. a 10% increase in overall mutation rate leads to faster adaptation; this puts to rest dysgenic fears about increasing mutation rates due to rising paternal age.\nA cell-type atlas from a scyphozoan jellyfish Aurelia coerulea (formerly sp.1) provides insights into changes of cell-type diversity in the transition from polyps to medusae\nAuthors: Link, O.; Jahnel, S. M.; Janicek, K.; Kraus, J.; Montenegro, J. D.; Zimmerman, B.; Cole, A. G.; Technau, U.\nScore: 23.5, Published: 2023-08-29 DOI: 10.1101/2023.08.24.554571\nWe provide here a detailed single cell transcriptomic atlas covering the life cycle of the cosmopolitan scyphozoan Aurelia coerulea. We show that there is an increase in cell type diversity in the medusa stage, which is reflected by an increase in the number of unique transcripts expressed. We highlight parallels in both cell complement and specification pathways between Aurelia and the sea anemone Nematostella, two lineages separated by more than 500 MY. We find that several cell types of the neuroglandular lineages are specified by the same transcription factors that arose by gene duplications in the common ancestor of medusozoans and anthozoans. This confirms gene duplications as a source for the increase of cellular complexity during the evolution of cnidarian neuroglandular lineages dates to the common ancestor of all cnidarians. We validated spatial expression domains corresponding to molecular profiles of smooth and striated muscles and followed the development of the striated muscle in the medusa. Our data suggests that striated muscles in the subumbrella of the medusa derive directly from smooth muscles, and yet these two muscle cell phenotypes are characterized by distinct molecular profiles. Further, reconstruction of putative cell trajectories indicates multipotent cell states derive from tissues at the boundary between epithelial layers and highlight molecular characteristics not typical of other cnidarians. TeaserSingle cell transcriptomic atlas across the jellyfish life cycle reveals increased cellular complexity associated with the medusa form.\nConserved switch genes regulate a novel cannibalistic morph after whole genome duplication\nAuthors: Wighard, S.; Sommer, R. J.; Witte, H.\nScore: 16.9, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554244\nDevelopmental plasticity facilitates morphological and behavioural novelty, but associated regulatory mechanisms remain elusive. Nematodes have emerged as a powerful model to study developmental plasticity and its evolution. Here, we show the predatory nematode Allodiplogaster sudhausi evolved an additional third mouth morph, concomitant with whole genome duplication (WGD) and a strong increase in body size. The three mouth morphs are induced by different diets; bacteria, fungi and nematodes. CRISPR experiments indicate that regulation of the third morph involves co-option of a conserved developmental switch gene, which through WGD resulted in two mouth-form regulators. Gene dosage studies revealed a diverged role of these developmental switches, with functional redundancy and quantitative effects in the two mouth-form decisions, respectively. The third morph is cannibalistic and kills kin, whereas the other two morphs do not. Thus, the recent evolution of a new morph relies on pre-existing regulatory mechanisms and adds behavioural and social complexity. One-Sentence SummaryExperimental genetics in a nematode reveals a key role for developmental plasticity in the evolution of nutritional diversity\nUncovering the mosaic evolution of carnivoran skeletal systems\nAuthors: Law, C. J.; Hlusko, L. J.; Tseng, J.\nScore: 11.9, Published: 2023-09-04 DOI: 10.1101/2023.09.03.556127\nThe diversity of vertebrate skeletal forms is often attributed to adaptations to distinct ecological factors such as diet, locomotion, and sensory environment. Although the adaptive evolution of cranial, appendicular, and vertebral skeletal systems is well studied in vertebrates, comprehensive investigations of all skeletal components are rarely performed simultaneously. Consequently, we know little of how modes of evolution differ among skeletal components. We collected 103 linear measurements from 208 osteological museum specimens to capture skeletal variation across the major components of the skeletal systems of 119 extant carnivores. With these data, we tested if ecological and phylogenetic effects led to distinct modes of evolution among the cranial, appendicular, and vertebral skeletal regions by fitting multivariate evolutionary models. We found mosaic evolution in which only the mandible, hindlimb, and posterior region of the vertebral column showed evidence of adaptation towards ecological regimes whereas the remaining skeletal components followed clade-specific evolutionary shifts. We hypothesize that the decoupled evolution of individual skeletal components may have led to the origination of distinct adaptive zones and morphologies among extant carnivoran families that reflect phylogenetic hierarchies. Overall, our work highlights the importance of examining multiple skeletal components simultaneously in ecomorphological analyses. Ongoing work integrating the fossil and paleoenvironmental record will further govern carnivoran diversity we see today and reveal the complexity of evolutionary processes in multicomponent systems.\nWhat is the best method for estimating ancestral states from discrete characters?\nAuthors: Keating, J. N.\nScore: 11.4, Published: 2023-09-02 DOI: 10.1101/2023.08.31.555762\nAncestral state estimation is a formal phylogenetic method for inferring the nature of ancestors and performing tests of character evolution. As such, it is among the most important tools available to evolutionary biologists. However, there are a profusion of methods available, the accuracy of which remains unclear. Here I use a simulation approach to test between parsimony and likelihood methods for estimating ancestral states from discrete binary characters. I simulate 500 characters using 15 different Markov generating models, a range of tree sizes (8-256 tips) and three topologies representing end members of tree symmetry and branch length heterogeneity. Simulated tip states were subjected to ancestral state estimation under the Equal Rates (ER) and All-Rates-Different (ARD) models, as well as under parsimony assuming accelerated transformations (ACCTRAN). The results demonstrate that both parsimony and likelihood approaches obtain high accuracy applied to trees with more tips. Parsimony performs poorly when trees contain long branches, whereas the ER model performs well across simulations and is reasonably robust to model violation. The ER model frequently outperforms the ARD model, even when data are simulated using unequal rates. Furthermore, the ER model exhibits less transition rate error when compared to ER models. These results suggest that ARD models may be overparameterized when character data is limited. Surprisingly, the difference in likelihood-based information criteria between models was found to be a poor predictor of difference in model error; better fitting models are not necessarily more accurate. However, there is a strong correlation between model uncertainty and model error; likelihood models with more certain ancestral state estimates are typically more accurate. Using empirical morphological datasets, I demonstrate that applying different methods often results in substantively different ancestral state estimates. The results of the simulation study highlight the importance of incorporating fossils in ancestral state estimation. Fossils increase the total number of tips, break long branches and are closer to internal nodes, thereby lowering average branch length and overall branch length heterogeneity of trees. These factors will all contribute to increasing the accuracy of ancestral state estimates, irrespective of the method used.\nThe genomic landscape, causes, and consequences of extensive phylogenomic discordance in Old World mice and rats\nAuthors: Thomas, G. W. C.; Hughes, J. J.; Kumon, T.; Berv, J. S.; Nordgren, C. E.; Lampson, M.; Levine, M.; Searle, J. B.; Good, J. M.\nScore: 10.7, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555178\nA species tree is a central concept in evolutionary biology whereby a single branching phylogeny reflects relationships among species. However, the phylogenies of different genomic regions often differ from the species tree. Although tree discordance is often widespread in phylogenomic studies, we still lack a clear understanding of how variation in phylogenetic patterns is shaped by genome biology or the extent to which discordance may compromise comparative studies. We characterized patterns of phylogenomic discordance across the murine rodents (Old World mice and rats) - a large and ecologically diverse group that gave rise to the mouse and rat model systems. Combining new linked-read genome assemblies for seven murine species with eleven published rodent genomes, we first used ultra-conserved elements (UCEs) to infer a robust species tree. We then used whole genomes to examine finer-scale patterns of discordance and found that phylogenies built from proximate chromosomal regions had similar phylogenies. However, there was no relationship between tree similarity and local recombination rates in house mice, suggesting that genetic linkage influences phylogenetic patterns over deeper timescales. This signal may be independent of contemporary recombination landscapes. We also detected a strong influence of linked selection whereby purifying selection at UCEs led to less discordance, while genes experiencing positive selection showed more discordant and variable phylogenetic signals. Finally, we show that assuming a single species tree can result in high error rates when testing for positive selection under different models. Collectively, our results highlight the complex relationship between phylogenetic inference and genome biology and underscore how failure to account for this complexity can mislead comparative genomic studies.\n",
  "wordCount" : "2762",
  "inLanguage": "en",
  "datePublished": "2023-09-10T10:36:49Z",
  "dateModified": "2023-09-10T10:36:49Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta"><span>updated on September 10, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556762">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556762" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556762">
        <p class="paperTitle">A Quantitative Genetic Model of Background Selection in Humans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556762" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556762" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Buffalo, V.; Kern, A. D.</p>
        <p class="info">Score: 24.0, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556762' target='https://doi.org/10.1101/2023.09.07.556762'> 10.1101/2023.09.07.556762</a></p>
        <p class="abstract">Across the human genome, there are large-scale fluctuations in genetic diversity caused by the indirect effects of selection. This &#34;linked selection signal&#34; reflects the impact of selection according to the physical placement of functional regions and recombination rates along chromosomes. Previous work has shown that purifying selection acting against the steady influx of new deleterious mutations at functional portions of the genome shapes patterns of genomic variation. To date, statistical efforts to estimate purifying selection parameters from linked selection models have relied on classic Background Selection theory, which is only applicable when new mutations are so deleterious that they cannot fix in the population. Here, we develop a statistical method based on a quantitative genetics view of linked selection, that models how polygenic additive fitness variance distributed along the genome increases the rate of stochastic allele frequency change. By jointly predicting the equilibrium fitness variance and substitution rate due to both strong and weakly deleterious mutations, we estimate the distribution of fitness effects (DFE) and mutation rate across three geographically distinct human samples. While our model can accommodate weaker selection, we find evidence of strong selection operating similarly across all human samples. While our quantitative genetic model of linked selection fits better than previous models, substitution rates of the most constrained sites are poorly predicted. We find that a model incorporating selective interference better predicts observed divergence in conserved regions, but overall our results ultimately suggest considerable uncertainty remains about the processes generating fitness variation in humans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555124">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555124" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555124">
        <p class="paperTitle">Large-scale analyses reveal the contribution of adaptive evolution in pathogenic and non-pathogenic fungal species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555124" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555124" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pereira, D.; Bolton, M. D.; Friesen, T. L.; Stephan, W.; Dutheil, J. Y.; Stukenbrock, E. H.</p>
        <p class="info">Score: 26.3, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555124' target='https://doi.org/10.1101/2023.08.28.555124'> 10.1101/2023.08.28.555124</a></p>
        <p class="abstract">Genome studies of fungal pathogens have presented evidence for exceptionally high rates of evolution. It has been proposed that rapid adaptation is a hallmark of pathogen evolution that facilitates the invasion of new host niches and the overcoming of intervention strategies such as fungicide applications and drug treatments. To which extent high levels of genetic variation within and between species correlate with adaptive protein evolution in fungi more generally has so far not been explored. In this study, we addressed the contribution of adaptive evolution relative to genetic drift in 20 fungal species, hereby exploring genetic variation in 2,478 fungal genomes. We reannotated positions of protein-coding genes to obtain a high-quality dataset of 234,427 full-length core gene and 25,612 accessory gene alignments. We applied an extension of the McDonald-Kreitman test that models the distributions of fitness effects to infer the rate of adaptive ({omega}A) and non-adaptive ({omega}NA) non-synonymous substitutions in protein-coding genes. To explore the relevance of recombination on local adaptation rates, we inferred the population genomic recombination rate for all 20 species. Our analyses reveal extensive variation in rates of adaptation and show that high rates of adaptation are not a hallmark of a pathogenic lifestyle. Up to 83% of non-synonymous substitutions are adaptive in the species Parastagonospora nodorum. However, non-synonymous substitutions in other species, including the prominent rice-infecting pathogen Magnaporthe oryzae, are predominantly non-adaptive (neutral or slightly deleterious). Correlating adaptation measures with effective population size and recombination rate, we show that effective population size is a primary determinant of adaptive evolution in fungi. At the genome scale, recombination rate variation explains variation in both {omega}A and {omega}NA. Finally, we demonstrate the robustness of our estimates using simulations. We underline the value of population genetic principles in studies of fungal evolution, and we highlight the importance of demographic processes in adaptive evolution of pathogenic and non-pathogenic species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556376">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556376" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556376">
        <p class="paperTitle">Speciation in kleptoparasites of oak gall wasps often correlates with shifts into new tree habitats, tree organs, and gall morphospace</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556376" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556376" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ward, A.; Zhang, Y. M.; Brown, G. E.; Hippee, A. C.; Prior, K. M.; Rollins, S.; Sierra, N.; Sheikh, S. I.; Tribull, C. M.; Forbes, A. A.</p>
        <p class="info">Score: 18.2, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556376' target='https://doi.org/10.1101/2023.09.07.556376'> 10.1101/2023.09.07.556376</a></p>
        <p class="abstract">Host shifts to new plants can drive speciation for plant-feeding insects, but how commonly do host shifts also drive diversification for the parasites of those same insects? Oak gall wasps induce galls on oak trees and shifts to novel tree hosts and new tree organs have been implicated as drivers of oak gall wasp speciation. Gall wasps are themselves attacked by many insect parasites, which must find their hosts on the correct tree species and organ, but which also must navigate the morphologically variable galls with which they interact. Thus, we ask whether host shifts to new trees, organs, or gall morphologies correlate with gall parasite diversification. We delimit species and infer phylogenies for two genera of gall kleptoparasites, Synergus and Ceroptres, reared from a variety of North American oak galls. We find that most species were reared from galls induced by just one gall wasp species, and no parasite species was reared from galls of more than four species. Most kleptoparasite divergence events correlate with shifts to non-ancestral galls. These shifts often involved changes in tree habitat, gall location, and gall morphology. Host shifts are thus implicated in driving diversification for both oak gall wasps and their kleptoparasitic associates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556433">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556433" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556433">
        <p class="paperTitle">Detecting shifts in the mode of chromosomal speciation across the cosmopolitan plant lineage Carex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556433" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556433" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tribble, C. M.; Marquez Corro, J. I.; May, M. R.; Hipp, A. L.; Escudero, M.; Zenil-Ferguson, R.</p>
        <p class="info">Score: 15.9, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556433' target='https://doi.org/10.1101/2023.09.05.556433'> 10.1101/2023.09.05.556433</a></p>
        <p class="abstract">Recent studies of angiosperm diversification have focused on the role of polyploidy as a driver of diversification. However, we know far less about the effects of single changes in chromosome number---dysploidy, which can mediate lineage diversification by affecting recombination rates, linkage, and/or reproductive isolation. Modeling the effects of dysploidy on diversification is mathematically and computationally challenging because many states and parameters are required to track changes in individual chromosomes, especially in lineages that have high variability in chromosome number. Additionally, we expect the processes of diversification and dysploidy to vary across clades, which requires modeling process variation to disentangle the effects of the observed trait (chromosome number) from the effects of unobserved traits on diversification. In this work, we propose a new state-dependent diversification model of chromosome evolution that includes numerous character states and explicitly models heterogeneity in the diversification process. Our model includes parameters that functionally link diversification rates to dysploidy rates and differentiate between anagenetic and cladogenetic changes. We apply this model to Carex (Cyperaceae), a model lineage for understanding dysploidy and diversification, leveraging chromosome number information and the most recent time-calibrated phylogenetic tree for over 700 species and subspecies. We recover distinct modes of chromosomal speciation across Carex. In one mode, dysploidy occurs very frequently and drives faster diversification rates. In the other mode, dysploidy is rare and diversification is driven by other factors, unmeasured in our analysis. This study is the first to demonstrate that dysploidy drives diversification in plants while considering unmeasured factors affecting diversification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555871">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555871" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555871">
        <p class="paperTitle">Human deleterious mutation rate implies high fitness variance, with declining mean fitness compensated by rarer beneficial mutations of larger effect</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555871" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555871" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Matheson, J. D.; Masel, J.; Bertram, J.</p>
        <p class="info">Score: 15.5, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555871' target='https://doi.org/10.1101/2023.09.01.555871'> 10.1101/2023.09.01.555871</a></p>
        <p class="abstract">Each new human has an expected Ud =2 - 10 new deleterious mutations. This deluge of deleterious mutations cannot all be purged, and therefore accumulate in a declining fitness ratchet. Using a novel simulation framework designed to efficiently handle genome-wide linkage disequilibria across many segregating sites, we find that rarer, beneficial mutations of larger effect are sufficient to compensate fitness declines due to the fixation of many slightly deleterious mutations. Drift barrier theory posits a similar asymmetric pattern of fixations to explain ratcheting genome size and complexity, but in our theory, the cause is Ud &gt;1 rather than small population size. In our simulations, Ud [~]2 - 10 generates high within-population variance in relative fitness; two individuals will typically differ in fitness by 15- 40%. Ud [~]2 - 10 also slows net adaptation by [~]13%-39%. Surprisingly, fixation rates are more sensitive to changes in the beneficial than the deleterious mutation rate, e.g. a 10% increase in overall mutation rate leads to faster adaptation; this puts to rest dysgenic fears about increasing mutation rates due to rising paternal age.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554571">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554571" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554571">
        <p class="paperTitle">A cell-type atlas from a scyphozoan jellyfish Aurelia coerulea (formerly sp.1) provides insights into changes of cell-type diversity in the transition from polyps to medusae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554571" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554571" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Link, O.; Jahnel, S. M.; Janicek, K.; Kraus, J.; Montenegro, J. D.; Zimmerman, B.; Cole, A. G.; Technau, U.</p>
        <p class="info">Score: 23.5, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554571' target='https://doi.org/10.1101/2023.08.24.554571'> 10.1101/2023.08.24.554571</a></p>
        <p class="abstract">We provide here a detailed single cell transcriptomic atlas covering the life cycle of the cosmopolitan scyphozoan Aurelia coerulea. We show that there is an increase in cell type diversity in the medusa stage, which is reflected by an increase in the number of unique transcripts expressed. We highlight parallels in both cell complement and specification pathways between Aurelia and the sea anemone Nematostella, two lineages separated by more than 500 MY. We find that several cell types of the neuroglandular lineages are specified by the same transcription factors that arose by gene duplications in the common ancestor of medusozoans and anthozoans. This confirms gene duplications as a source for the increase of cellular complexity during the evolution of cnidarian neuroglandular lineages dates to the common ancestor of all cnidarians. We validated spatial expression domains corresponding to molecular profiles of smooth and striated muscles and followed the development of the striated muscle in the medusa. Our data suggests that striated muscles in the subumbrella of the medusa derive directly from smooth muscles, and yet these two muscle cell phenotypes are characterized by distinct molecular profiles. Further, reconstruction of putative cell trajectories indicates multipotent cell states derive from tissues at the boundary between epithelial layers and highlight molecular characteristics not typical of other cnidarians.

TeaserSingle cell transcriptomic atlas across the jellyfish life cycle reveals increased cellular complexity associated with the medusa form.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554244">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554244" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554244">
        <p class="paperTitle">Conserved switch genes regulate a novel cannibalistic morph after whole genome duplication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554244" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554244" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wighard, S.; Sommer, R. J.; Witte, H.</p>
        <p class="info">Score: 16.9, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554244' target='https://doi.org/10.1101/2023.08.22.554244'> 10.1101/2023.08.22.554244</a></p>
        <p class="abstract">Developmental plasticity facilitates morphological and behavioural novelty, but associated regulatory mechanisms remain elusive. Nematodes have emerged as a powerful model to study developmental plasticity and its evolution. Here, we show the predatory nematode Allodiplogaster sudhausi evolved an additional third mouth morph, concomitant with whole genome duplication (WGD) and a strong increase in body size. The three mouth morphs are induced by different diets; bacteria, fungi and nematodes. CRISPR experiments indicate that regulation of the third morph involves co-option of a conserved developmental switch gene, which through WGD resulted in two mouth-form regulators. Gene dosage studies revealed a diverged role of these developmental switches, with functional redundancy and quantitative effects in the two mouth-form decisions, respectively. The third morph is cannibalistic and kills kin, whereas the other two morphs do not. Thus, the recent evolution of a new morph relies on pre-existing regulatory mechanisms and adds behavioural and social complexity.

One-Sentence SummaryExperimental genetics in a nematode reveals a key role for developmental plasticity in the evolution of nutritional diversity</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.03.556127">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.03.556127" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.03.556127">
        <p class="paperTitle">Uncovering the mosaic evolution of carnivoran skeletal systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.03.556127" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.03.556127" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Law, C. J.; Hlusko, L. J.; Tseng, J.</p>
        <p class="info">Score: 11.9, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.03.556127' target='https://doi.org/10.1101/2023.09.03.556127'> 10.1101/2023.09.03.556127</a></p>
        <p class="abstract">The diversity of vertebrate skeletal forms is often attributed to adaptations to distinct ecological factors such as diet, locomotion, and sensory environment. Although the adaptive evolution of cranial, appendicular, and vertebral skeletal systems is well studied in vertebrates, comprehensive investigations of all skeletal components are rarely performed simultaneously. Consequently, we know little of how modes of evolution differ among skeletal components. We collected 103 linear measurements from 208 osteological museum specimens to capture skeletal variation across the major components of the skeletal systems of 119 extant carnivores. With these data, we tested if ecological and phylogenetic effects led to distinct modes of evolution among the cranial, appendicular, and vertebral skeletal regions by fitting multivariate evolutionary models. We found mosaic evolution in which only the mandible, hindlimb, and posterior region of the vertebral column showed evidence of adaptation towards ecological regimes whereas the remaining skeletal components followed clade-specific evolutionary shifts. We hypothesize that the decoupled evolution of individual skeletal components may have led to the origination of distinct adaptive zones and morphologies among extant carnivoran families that reflect phylogenetic hierarchies. Overall, our work highlights the importance of examining multiple skeletal components simultaneously in ecomorphological analyses. Ongoing work integrating the fossil and paleoenvironmental record will further govern carnivoran diversity we see today and reveal the complexity of evolutionary processes in multicomponent systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555762">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555762" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555762">
        <p class="paperTitle">What is the best method for estimating ancestral states from discrete characters?</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555762" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555762" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Keating, J. N.</p>
        <p class="info">Score: 11.4, Published: 2023-09-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555762' target='https://doi.org/10.1101/2023.08.31.555762'> 10.1101/2023.08.31.555762</a></p>
        <p class="abstract">Ancestral state estimation is a formal phylogenetic method for inferring the nature of ancestors and performing tests of character evolution. As such, it is among the most important tools available to evolutionary biologists. However, there are a profusion of methods available, the accuracy of which remains unclear. Here I use a simulation approach to test between parsimony and likelihood methods for estimating ancestral states from discrete binary characters. I simulate 500 characters using 15 different Markov generating models, a range of tree sizes (8-256 tips) and three topologies representing end members of tree symmetry and branch length heterogeneity. Simulated tip states were subjected to ancestral state estimation under the Equal Rates (ER) and All-Rates-Different (ARD) models, as well as under parsimony assuming accelerated transformations (ACCTRAN). The results demonstrate that both parsimony and likelihood approaches obtain high accuracy applied to trees with more tips. Parsimony performs poorly when trees contain long branches, whereas the ER model performs well across simulations and is reasonably robust to model violation. The ER model frequently outperforms the ARD model, even when data are simulated using unequal rates. Furthermore, the ER model exhibits less transition rate error when compared to ER models. These results suggest that ARD models may be overparameterized when character data is limited. Surprisingly, the difference in likelihood-based information criteria between models was found to be a poor predictor of difference in model error; better fitting models are not necessarily more accurate. However, there is a strong correlation between model uncertainty and model error; likelihood models with more certain ancestral state estimates are typically more accurate. Using empirical morphological datasets, I demonstrate that applying different methods often results in substantively different ancestral state estimates. The results of the simulation study highlight the importance of incorporating fossils in ancestral state estimation. Fossils increase the total number of tips, break long branches and are closer to internal nodes, thereby lowering average branch length and overall branch length heterogeneity of trees. These factors will all contribute to increasing the accuracy of ancestral state estimates, irrespective of the method used.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555178">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555178" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555178">
        <p class="paperTitle">The genomic landscape, causes, and consequences of extensive phylogenomic discordance in Old World mice and rats</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555178" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555178" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thomas, G. W. C.; Hughes, J. J.; Kumon, T.; Berv, J. S.; Nordgren, C. E.; Lampson, M.; Levine, M.; Searle, J. B.; Good, J. M.</p>
        <p class="info">Score: 10.7, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555178' target='https://doi.org/10.1101/2023.08.28.555178'> 10.1101/2023.08.28.555178</a></p>
        <p class="abstract">A species tree is a central concept in evolutionary biology whereby a single branching phylogeny reflects relationships among species. However, the phylogenies of different genomic regions often differ from the species tree. Although tree discordance is often widespread in phylogenomic studies, we still lack a clear understanding of how variation in phylogenetic patterns is shaped by genome biology or the extent to which discordance may compromise comparative studies. We characterized patterns of phylogenomic discordance across the murine rodents (Old World mice and rats) - a large and ecologically diverse group that gave rise to the mouse and rat model systems. Combining new linked-read genome assemblies for seven murine species with eleven published rodent genomes, we first used ultra-conserved elements (UCEs) to infer a robust species tree. We then used whole genomes to examine finer-scale patterns of discordance and found that phylogenies built from proximate chromosomal regions had similar phylogenies. However, there was no relationship between tree similarity and local recombination rates in house mice, suggesting that genetic linkage influences phylogenetic patterns over deeper timescales. This signal may be independent of contemporary recombination landscapes. We also detected a strong influence of linked selection whereby purifying selection at UCEs led to less discordance, while genes experiencing positive selection showed more discordant and variable phylogenetic signals. Finally, we show that assuming a single species tree can result in high error rates when testing for positive selection under different models. Collectively, our results highlight the complex relationship between phylogenetic inference and genome biology and underscore how failure to account for this complexity can mislead comparative genomic studies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
