<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>evolutionary biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>evolutionary biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="New genome reveals molecular signatures of adaptation to nocturnality in moth-like butterflies (Hedylidae)
Authors: Singh, R. P.; Weng, Y.-M.; Sondhi, Y.; Plotkin, D.; Frandsen, P. B.; Kawahara, A. Y.
Score: 55.0, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568084
Nearly all animals have a preferred period of daily activity (diel-niche), which is strongly influenced by the light environment. Sensory systems, particularly vision, are adapted to light, and evolutionary transitions to novel light environments, especially light limited ones, can impose strong constraints on eye evolution, color, and motion vision.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/evolutionary-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="evolutionary biology" />
<meta property="og:description" content="New genome reveals molecular signatures of adaptation to nocturnality in moth-like butterflies (Hedylidae)
Authors: Singh, R. P.; Weng, Y.-M.; Sondhi, Y.; Plotkin, D.; Frandsen, P. B.; Kawahara, A. Y.
Score: 55.0, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568084
Nearly all animals have a preferred period of daily activity (diel-niche), which is strongly influenced by the light environment. Sensory systems, particularly vision, are adapted to light, and evolutionary transitions to novel light environments, especially light limited ones, can impose strong constraints on eye evolution, color, and motion vision." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/evolutionary-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-26T10:37:33+00:00" />
<meta property="article:modified_time" content="2023-11-26T10:37:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="evolutionary biology"/>
<meta name="twitter:description" content="New genome reveals molecular signatures of adaptation to nocturnality in moth-like butterflies (Hedylidae)
Authors: Singh, R. P.; Weng, Y.-M.; Sondhi, Y.; Plotkin, D.; Frandsen, P. B.; Kawahara, A. Y.
Score: 55.0, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568084
Nearly all animals have a preferred period of daily activity (diel-niche), which is strongly influenced by the light environment. Sensory systems, particularly vision, are adapted to light, and evolutionary transitions to novel light environments, especially light limited ones, can impose strong constraints on eye evolution, color, and motion vision."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "evolutionary biology",
      "item": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "evolutionary biology",
  "name": "evolutionary biology",
  "description": "New genome reveals molecular signatures of adaptation to nocturnality in moth-like butterflies (Hedylidae)\nAuthors: Singh, R. P.; Weng, Y.-M.; Sondhi, Y.; Plotkin, D.; Frandsen, P. B.; Kawahara, A. Y.\nScore: 55.0, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568084\nNearly all animals have a preferred period of daily activity (diel-niche), which is strongly influenced by the light environment. Sensory systems, particularly vision, are adapted to light, and evolutionary transitions to novel light environments, especially light limited ones, can impose strong constraints on eye evolution, color, and motion vision.",
  "keywords": [
    
  ],
  "articleBody": " New genome reveals molecular signatures of adaptation to nocturnality in moth-like butterflies (Hedylidae)\nAuthors: Singh, R. P.; Weng, Y.-M.; Sondhi, Y.; Plotkin, D.; Frandsen, P. B.; Kawahara, A. Y.\nScore: 55.0, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568084\nNearly all animals have a preferred period of daily activity (diel-niche), which is strongly influenced by the light environment. Sensory systems, particularly vision, are adapted to light, and evolutionary transitions to novel light environments, especially light limited ones, can impose strong constraints on eye evolution, color, and motion vision. The adaptive changes in sensory abilities of animals during these transitions, both at the genetic and neural levels, are largely unexplored. Butterflies and moths, with their diverse diel-niche shifts, are an ideal group for investigating the gene evolution linked to these transitions. While most butterflies are day-flying, hedylid butterflies are unique in being primarily nocturnal, and they represent an important evolutionary shift from diurnality to nocturnality in this clade. Here, we sequence the first high-quality Hedylidae genome and functionally annotate genes to understand genomic changes associated with shifts in diel niche. Comparing Hedylidae visual genes against day- and night-flying Lepidoptera species revealed that visual genes are highly conserved, with no major losses. However, hedylid butterfly opsins were more similar to nocturnal moths than their diurnal congeners. Tests on the evolutionary rates (dN/dS) confirmed that color vision opsins were under strong selection, similar to nocturnal moths. We propose that a convergent event of sequence evolution took place when these butterflies became nocturnal, approximately 98 million years ago.\nInduction of apoptosis by double-stranded RNA was present in the last common ancestor of cnidarian and bilaterian animals\nAuthors: Kozlovski, I.; Jaimes-Becerra, A.; Sharoni, T.; Lewandowska, M.; Karmi, O.; Moran, Y.\nScore: 20.0, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567790\nApoptosis, a major form of programmed cell death, is an essential component of host defense against invading intracellular pathogens. Viruses encode inhibitors of apoptosis to evade host responses during infection, and to support their own replication and survival. Therefore, hosts and their viruses are entangled in a constant evolutionary arms race to control apoptosis. Until now, apoptosis in the context of the antiviral immune system has been almost exclusively studied in vertebrates. This limited phyletic sampling makes it impossible to determine whether a similar mechanism existed in the last common ancestor of animals. Here, we established assays to probe apoptosis in the sea anemone Nematostella vectensis, a model species of Cnidaria, a phylum that diverged approximately 600 million years ago from the rest of animals. We show that polyinosinic:polycytidylic acid (poly I:C), a synthetic long double-stranded RNA mimicking viral RNA and a primary ligand for the vertebrate RLR melanoma differentiation-associated protein 5 (MDA5), is sufficient to induce apoptosis in N. vectensis. Furthermore, at the transcriptomic level, apoptosis related genes are significantly enriched upon poly(I:C) exposure in N. vectensis as well as bilaterian invertebrates. Our phylogenetic analysis of caspase family genes in N. vectensis reveals conservation of all four caspase genes involved in apoptosis in mammals and revealed a cnidarian-specific caspase gene which was strongly upregulated. Altogether, our findings suggest that apoptosis in response to a viral challenge is a functionally conserved mechanism that can be traced back to the last common ancestor of Bilateria and Cnidaria.\nChallenges in estimating species age from phylogenetic trees\nAuthors: Calderon del Cid, C.; Hauffe, T.; Carrillo, J. D.; Warnock, R. C. M.; Silvestro, D.\nScore: 15.5, Published: 2023-11-19 DOI: 10.1101/2023.11.19.567731\nAimSpecies age, the elapsed time since origination, can give an insight into how species longevity might influence eco-evolutionary dynamics and has been hypothesized to influence extinction risk. Traditionally, species ages have been measured in the fossil record. However, recently, numerous studies have attempted to estimate the ages of extant species from the branch lengths of time-calibrated phylogenies. This approach poses problems because phylogenetic trees contain direct information about species identity only at the tips and not along the branches. Here, we show that taxon sampling, extinction, and different assumptions about speciation modes can significantly alter the relationship between true species age and phylogenetic branch lengths, leading to high error rates. We find that these biases can lead to erroneous interpretations of eco-evolutionary patterns derived from the comparison between phylogenetic age and other traits, such as extinction risk. InnovationFor bifurcating speciation, which is the default assumption in most analyses, we propose a probabilistic approach to improve the estimation of species ages, based on the properties of a birth-death process. We show that our model can reduce the error by one order of magnitude under cases of high extinction. Main conclusionOur results call for caution in interpreting the relationship between phylogenetic ages and eco-evolutionary traits, and show that, under some assumptions, it is possible to obtain better approximations of species age by combining information from branch lengths with the expectations of a birth-death process.\nMass lysis of bacterial predators drives the enrichment of antibiotic resistance in soil microbial communities\nAuthors: Saha, S.; Kalathera, J.; Sumi, T. S.; Mane, V.; Zimmermann, S.; Waschina, S.; Pande, S.\nScore: 15.1, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567171\nWhile studies on anthropogenic activities and antibiotic resistance are numerous, the impact of microbial interactions on resistance in complex communities remains uncertain. Here we demonstrate a correlation between the presence of Myxococcus xanthus in natural soil communities and the abundance of antibiotic-resistant bacteria. Further, introducing M. xanthus isolates also enriches antibiotic resistance. This is due to the mass lysis of M. xanthus cells, which results in a toxic environment that fosters the proliferation of pre-existing resistant bacteria rather than de novo resistance evolution. Metagenomic analysis revealed that this enrichment is not limited to the tested antibiotics in culture-based methods, indicating its broader relevance. Crucially, these findings go beyond laboratory settings, showing M. xanthus introduction enriches resistant isolates in natural soil communities. Finally, we demonstrate that the mass lysis of M. xanthus cells during starvation-induced development--key aspect of the lifecycle of M. xanthus--also results in the enrichment of antibiotic resistance in soil communities. Together, we demonstrate how life-history traits in bacterial predators, like M. xanthus, significantly impact antibiotic resistomes in nature. This study also highlights the complex dynamics at play in the evolution and maintenance of antibiotic resistance, emphasizing the role of interspecies interactions in shaping antibiotic resistance profiles.\nThe explosive radiation of the Neotropical Tillandsia subgenus Tillandsia (Bromeliaceae) has been facilitated by pervasive hybridization\nAuthors: Yardeni, G.; Barfuss, M. H. J.; Till, W.; Thornton, M. R.; Groot Crego, C.; Lexer, C.; Leroy, T.; Paun, O.\nScore: 14.4, Published: 2023-11-17 DOI: 10.1101/2023.11.16.567341\nThe recent rapid radiation of Tillandsia subgenus Tillandsia (Bromeliaceae) provides an attractive system to study the drivers and limits of species diversification. This species-rich Neotropical monocot clade includes predominantly epiphytic species displaying vast phenotypic diversity. Recent in-depth phylogenomic work revealed that the subgenus originated within the last 7 MY while expanding through one major event from South into Central America within the last 5 MY. However, disagreements between phylogenies and lack of resolution at shallow nodes suggested that hybridization occurred throughout the radiation, together with frequent incomplete lineage sorting and/or considerable gene family evolution. We used whole-genome resequencing data and a newly available reference genome to explore the evolutionary history of 34 representative ingroup species employing both a tree-based and a network approach. Our results indicate that lineage co-occurrence does not predict relatedness and confirm significant deviations from a tree-like structure, coupled with pervasive gene tree discordance. Focusing on hybridization, ABBA-BABA and related statistics were used to infer the rates and relative timing of introgression, while topology weighting uncovered high heterogeneity of the phylogenetic signal along the genome. High rates of hybridization within and among clades suggest that, in contrast to previous hypotheses, the expansion of subgenus Tillandsia into Central America proceeded in several dispersal events, punctuated by episodes of diversification and gene flow. Network analysis revealed reticulation as a prominent propeller during radiation and establishment in different ecological niches. This work contributes a plant example of prevalent hybridization during rapid species diversification, supporting the hypothesis that interspecific gene flow facilitates explosive diversification.\nFunctional and evolutionary integration of a fungal gene with a bacterial operon\nAuthors: Sun, L.; David, K. T.; Wolters, J. F.; Karlen, S. D.; Goncalves, C.; Opulente, D. A.; LaBella, A. L.; Groenewald, M.; Zhou, X.; Shen, X.-X.; Rokas, A.; Hittinger, C. T.\nScore: 13.8, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568075\nSiderophores are crucial for iron-scavenging in microorganisms. While many yeasts can uptake siderophores produced by other organisms, they are typically unable to synthesize siderophores themselves. In contrast, Wickerhamiella/Starmerella (W/S) clade yeasts gained the capacity to make the siderophore enterobactin following the remarkable horizontal acquisition of a bacterial operon enabling enterobactin synthesis. Yet, how these yeasts absorb the iron bound by enterobactin remains unresolved. Here, we demonstrate that Enb1 is the key enterobactin importer in the W/S-clade species Starmerella bombicola. Through phylogenomic analyses, we show that ENB1 is present in all W/S clade yeast species that retained the enterobactin biosynthetic genes. Conversely, it is absent in species that lost the ent genes, except for Starmerella stellata, making this species the only cheater in the W/S clade that can utilize enterobactin without producing it. Through phylogenetic analyses, we infer that ENB1 is a fungal gene that likely existed in the W/S clade prior to the acquisition of the ent genes and subsequently experienced multiple gene losses and duplications. Through phylogenetic topology tests, we show that ENB1 likely underwent horizontal gene transfer from an ancient W/S clade yeast to the order Saccharomycetales, which includes the model yeast Saccharomyces cerevisiae, followed by extensive secondary losses. Taken together, these results suggest that the fungal ENB1 and bacterial ent genes were cooperatively integrated into a functional unit within the W/S clade that enabled adaptation to iron-limited environments. This integrated fungal-bacterial circuit and its dynamic evolution determines the extant distribution of yeast enterobactin producers and cheaters.\nPromoter capture drives the emergence of proto-genes in Escherichia coli\nAuthors: uz-Zaman, M. H.; D'Alton, S.; Barrick, J. E.; Ochman, H.\nScore: 14.3, Published: 2023-11-17 DOI: 10.1101/2023.11.15.567300\nThe phenomenon of de novo gene birth--the emergence of genes from non-genic sequences--has received considerable attention due to the widespread occurrence of genes that are unique to particular species or genomes. Most instances of de novo gene birth have been recognized through comparative analyses of genome sequences in eukaryotes, despite the abundance of novel, lineage-specific genes in bacteria and the relative ease with which bacteria can be studied in an experimental context. Here, we explore the genetic record of the Escherichia coli Long-Term Evolution Experiment (LTEE) for changes indicative of \"proto-genic\" phases of new gene birth in which non-genic sequences evolve stable transcription and/or translation. Over the time-span of the LTEE, non-genic regions are frequently transcribed, translated and differentially expressed, thereby serving as raw material for new gene emergence. Most proto-genes result either from insertion element activity or chromosomal translocations that fused pre-existing regulatory sequences to regions that were not expressed in the LTEE ancestor. Additionally, we identified instances of proto-gene emergence in which a previously unexpressed sequence was transcribed after formation of an upstream promoter. Tracing the origin of the causative mutations, we discovered that most occurred early in the history of the LTEE, often within the first 20,000 generations, and became fixed soon after emergence. Our findings show that proto-genes emerge frequently within evolving populations, persist stably, and can serve as potential substrates for new gene formation.\nFrequent transitions from night-to-day activity after mass extinctions\nAuthors: Shafer, M. E. R.; Nichols, A. L. A.; Schier, A. F.; Salzburger, W.\nScore: 25.2, Published: 2023-11-01 DOI: 10.1101/2023.10.27.564421\nWhy certain species survive large-scale extinction events while others do not is poorly understood. While the fossil record can provide insights into morphological adaptations that increase survival probabilities, it provides limited information regarding behavioural traits. Here we integrate behavioural data with phylogenetic comparative models to study the evolution of day-night activity patterns (nocturnality and diurnality) in bony fishes, which have persisted through the last four mass extinction events. Our findings in fish, combined with data across all other clades of vertebrates, provide four lines of evidence that nocturnality conferred an evolutionary advantage during mass extinctions, and that frequent nocturnal-to-diurnal transitions facilitated post-extinction diversification: First, phylogenetic reconstruction indicates that the last common ancestors of all vertebrates and of clades of bony vertebrates were nocturnal. Second, in the lineage of bony fishes, which contains over half of all vertebrate species, twice as many transitions between nocturnal and diurnal activity patterns have occurred compared to tetrapods. Third, within a specific ecological niche, different species exhibit distinct temporal activity patterns, suggesting widespread temporal niche partitioning. Fourth, independent bursts in night-to-day transitions followed large-scale extinction events during the last two geological eras in all four major bony vertebrate groups. These observations suggest that ancestral nocturnality and frequent transitions to diurnality helped vertebrates survive and diversify in the face of extinction events, such as those expected during the current \"6th mass extinction\" event caused by anthropogenic climate change.\nGene acquisition by giant transposons primes eukaryotes for rapid evolution via horizontal gene transfer\nAuthors: Urquhart, A. S.; Gluck-Thaler, E.; Vogan, A. A.\nScore: 11.5, Published: 2023-11-22 DOI: 10.1101/2023.11.22.568313\nHorizontal gene transfer (HGT) disseminates genetic information between species. The significance of HGT in eukaryotes is not well established, with evidence currently limited to isolated examples, typically absent of a mechanism. It has been proposed that mobile elements might be active agents of HGT in eukaryotes, as they are in prokaryotes. To test this hypothesis, we examined a gene cluster, which putatively contributes to formaldehyde resistance and is found within some members of the Starship superfamily of giant transposons. Remarkably, we found four instances where ssf has been independently acquired by distantly related Starships, and show that each acquisition event coincided with the element's horizontal transfer (at least 9 HGT events in total). Our results demonstrate that acquisition of host-beneficial cargo by Starships primes fungi for rapid and repeated adaptation via HGT, revealing and elevating the role of HGT in eukaryotic biology.\nDollo parsimony overestimates ancestral gene content reconstructions\nAuthors: Morante, A. G.; Gueguen, L.; Natsidis, P.; Telford, M. J.; Richter, D. J.\nScore: 10.0, Published: 2023-11-21 DOI: 10.1101/2023.11.20.567878\nAncestral reconstruction is a widely-used technique that has been applied to understand the history of gain and loss of gene families over evolutionary time scales, and to produce hypotheses on how these gains and losses may have influenced the evolutionary trajectories of extant organisms. Ancestral gene content can be reconstructed via different phylogenetic methods, such as maximum likelihood or Bayesian inference, but many current and previous studies employ Dollo parsimony. We hypothesize that Dollo parsimony is not appropriate for ancestral gene content reconstruction inferences based on sequence homology, as Dollo parsimony is derived from the assumption that a complex character can only be gained once in evolutionary history. This premise does not accurately model molecular sequence evolution, in which false orthology can result from sequence convergence (including random sequence similarity and parallel gene gains), non-orthologous homology or lateral gene transfer. The aim of this study is to test Dollo parsimonys suitability for ancestral gene content reconstruction and to compare its inferences with a maximum likelihood-based approach which allows a gene family to be gained more than once within a tree. In order to test our hypothesis, we first compared the performance of the two approaches on a series of artificial datasets each of 5,000 genes that were simulated according to a spectrum of evolutionary rates. Next, we reconstructed protein domain evolution on a phylogeny representing known eukaryotic diversity. We observed that Dollo parsimony produced numerous ancestral gene content overestimations, which were more pronounced at nodes closer to the root of the tree. These observations led us to the conclusion that, confirming our hypothesis, Dollo parsimony is not an appropriate method for ancestral reconstruction studies based on sequence homology.\n",
  "wordCount" : "2616",
  "inLanguage": "en",
  "datePublished": "2023-11-26T10:37:33Z",
  "dateModified": "2023-11-26T10:37:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/evolutionary-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      evolutionary biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 26, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568084">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568084" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568084">
        <p class="paperTitle">New genome reveals molecular signatures of adaptation to nocturnality in moth-like butterflies (Hedylidae)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568084" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568084" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Singh, R. P.; Weng, Y.-M.; Sondhi, Y.; Plotkin, D.; Frandsen, P. B.; Kawahara, A. Y.</p>
        <p class="info">Score: 55.0, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568084' target='https://doi.org/10.1101/2023.11.21.568084'> 10.1101/2023.11.21.568084</a></p>
        <p class="abstract">Nearly all animals have a preferred period of daily activity (diel-niche), which is strongly influenced by the light environment. Sensory systems, particularly vision, are adapted to light, and evolutionary transitions to novel light environments, especially light limited ones, can impose strong constraints on eye evolution, color, and motion vision. The adaptive changes in sensory abilities of animals during these transitions, both at the genetic and neural levels, are largely unexplored. Butterflies and moths, with their diverse diel-niche shifts, are an ideal group for investigating the gene evolution linked to these transitions. While most butterflies are day-flying, hedylid butterflies are unique in being primarily nocturnal, and they represent an important evolutionary shift from diurnality to nocturnality in this clade. Here, we sequence the first high-quality Hedylidae genome and functionally annotate genes to understand genomic changes associated with shifts in diel niche. Comparing Hedylidae visual genes against day- and night-flying Lepidoptera species revealed that visual genes are highly conserved, with no major losses. However, hedylid butterfly opsins were more similar to nocturnal moths than their diurnal congeners. Tests on the evolutionary rates (dN/dS) confirmed that color vision opsins were under strong selection, similar to nocturnal moths. We propose that a convergent event of sequence evolution took place when these butterflies became nocturnal, approximately 98 million years ago.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567790">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567790" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567790">
        <p class="paperTitle">Induction of apoptosis by double-stranded RNA was present in the last common ancestor of cnidarian and bilaterian animals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567790" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567790" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kozlovski, I.; Jaimes-Becerra, A.; Sharoni, T.; Lewandowska, M.; Karmi, O.; Moran, Y.</p>
        <p class="info">Score: 20.0, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567790' target='https://doi.org/10.1101/2023.11.20.567790'> 10.1101/2023.11.20.567790</a></p>
        <p class="abstract">Apoptosis, a major form of programmed cell death, is an essential component of host defense against invading intracellular pathogens. Viruses encode inhibitors of apoptosis to evade host responses during infection, and to support their own replication and survival. Therefore, hosts and their viruses are entangled in a constant evolutionary arms race to control apoptosis. Until now, apoptosis in the context of the antiviral immune system has been almost exclusively studied in vertebrates. This limited phyletic sampling makes it impossible to determine whether a similar mechanism existed in the last common ancestor of animals. Here, we established assays to probe apoptosis in the sea anemone Nematostella vectensis, a model species of Cnidaria, a phylum that diverged approximately 600 million years ago from the rest of animals. We show that polyinosinic:polycytidylic acid (poly I:C), a synthetic long double-stranded RNA mimicking viral RNA and a primary ligand for the vertebrate RLR melanoma differentiation-associated protein 5 (MDA5), is sufficient to induce apoptosis in N. vectensis. Furthermore, at the transcriptomic level, apoptosis related genes are significantly enriched upon poly(I:C) exposure in N. vectensis as well as bilaterian invertebrates. Our phylogenetic analysis of caspase family genes in N. vectensis reveals conservation of all four caspase genes involved in apoptosis in mammals and revealed a cnidarian-specific caspase gene which was strongly upregulated. Altogether, our findings suggest that apoptosis in response to a viral challenge is a functionally conserved mechanism that can be traced back to the last common ancestor of Bilateria and Cnidaria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.19.567731">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.19.567731" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.19.567731">
        <p class="paperTitle">Challenges in estimating species age from phylogenetic trees</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.19.567731" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.19.567731" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Calderon del Cid, C.; Hauffe, T.; Carrillo, J. D.; Warnock, R. C. M.; Silvestro, D.</p>
        <p class="info">Score: 15.5, Published: 2023-11-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.19.567731' target='https://doi.org/10.1101/2023.11.19.567731'> 10.1101/2023.11.19.567731</a></p>
        <p class="abstract">AimSpecies age, the elapsed time since origination, can give an insight into how species longevity might influence eco-evolutionary dynamics and has been hypothesized to influence extinction risk. Traditionally, species ages have been measured in the fossil record. However, recently, numerous studies have attempted to estimate the ages of extant species from the branch lengths of time-calibrated phylogenies. This approach poses problems because phylogenetic trees contain direct information about species identity only at the tips and not along the branches. Here, we show that taxon sampling, extinction, and different assumptions about speciation modes can significantly alter the relationship between true species age and phylogenetic branch lengths, leading to high error rates. We find that these biases can lead to erroneous interpretations of eco-evolutionary patterns derived from the comparison between phylogenetic age and other traits, such as extinction risk.

InnovationFor bifurcating speciation, which is the default assumption in most analyses, we propose a probabilistic approach to improve the estimation of species ages, based on the properties of a birth-death process. We show that our model can reduce the error by one order of magnitude under cases of high extinction.

Main conclusionOur results call for caution in interpreting the relationship between phylogenetic ages and eco-evolutionary traits, and show that, under some assumptions, it is possible to obtain better approximations of species age by combining information from branch lengths with the expectations of a birth-death process.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567171">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567171" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567171">
        <p class="paperTitle">Mass lysis of bacterial predators drives the enrichment of antibiotic resistance in soil microbial communities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567171" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567171" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saha, S.; Kalathera, J.; Sumi, T. S.; Mane, V.; Zimmermann, S.; Waschina, S.; Pande, S.</p>
        <p class="info">Score: 15.1, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567171' target='https://doi.org/10.1101/2023.11.20.567171'> 10.1101/2023.11.20.567171</a></p>
        <p class="abstract">While studies on anthropogenic activities and antibiotic resistance are numerous, the impact of microbial interactions on resistance in complex communities remains uncertain. Here we demonstrate a correlation between the presence of Myxococcus xanthus in natural soil communities and the abundance of antibiotic-resistant bacteria. Further, introducing M. xanthus isolates also enriches antibiotic resistance. This is due to the mass lysis of M. xanthus cells, which results in a toxic environment that fosters the proliferation of pre-existing resistant bacteria rather than de novo resistance evolution. Metagenomic analysis revealed that this enrichment is not limited to the tested antibiotics in culture-based methods, indicating its broader relevance. Crucially, these findings go beyond laboratory settings, showing M. xanthus introduction enriches resistant isolates in natural soil communities. Finally, we demonstrate that the mass lysis of M. xanthus cells during starvation-induced development--key aspect of the lifecycle of M. xanthus--also results in the enrichment of antibiotic resistance in soil communities. Together, we demonstrate how life-history traits in bacterial predators, like M. xanthus, significantly impact antibiotic resistomes in nature. This study also highlights the complex dynamics at play in the evolution and maintenance of antibiotic resistance, emphasizing the role of interspecies interactions in shaping antibiotic resistance profiles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567341">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567341" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567341">
        <p class="paperTitle">The explosive radiation of the Neotropical Tillandsia subgenus Tillandsia (Bromeliaceae) has been facilitated by pervasive hybridization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567341" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567341" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yardeni, G.; Barfuss, M. H. J.; Till, W.; Thornton, M. R.; Groot Crego, C.; Lexer, C.; Leroy, T.; Paun, O.</p>
        <p class="info">Score: 14.4, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567341' target='https://doi.org/10.1101/2023.11.16.567341'> 10.1101/2023.11.16.567341</a></p>
        <p class="abstract">The recent rapid radiation of Tillandsia subgenus Tillandsia (Bromeliaceae) provides an attractive system to study the drivers and limits of species diversification. This species-rich Neotropical monocot clade includes predominantly epiphytic species displaying vast phenotypic diversity. Recent in-depth phylogenomic work revealed that the subgenus originated within the last 7 MY while expanding through one major event from South into Central America within the last 5 MY. However, disagreements between phylogenies and lack of resolution at shallow nodes suggested that hybridization occurred throughout the radiation, together with frequent incomplete lineage sorting and/or considerable gene family evolution. We used whole-genome resequencing data and a newly available reference genome to explore the evolutionary history of 34 representative ingroup species employing both a tree-based and a network approach. Our results indicate that lineage co-occurrence does not predict relatedness and confirm significant deviations from a tree-like structure, coupled with pervasive gene tree discordance. Focusing on hybridization, ABBA-BABA and related statistics were used to infer the rates and relative timing of introgression, while topology weighting uncovered high heterogeneity of the phylogenetic signal along the genome. High rates of hybridization within and among clades suggest that, in contrast to previous hypotheses, the expansion of subgenus Tillandsia into Central America proceeded in several dispersal events, punctuated by episodes of diversification and gene flow. Network analysis revealed reticulation as a prominent propeller during radiation and establishment in different ecological niches. This work contributes a plant example of prevalent hybridization during rapid species diversification, supporting the hypothesis that interspecific gene flow facilitates explosive diversification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568075">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568075" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568075">
        <p class="paperTitle">Functional and evolutionary integration of a fungal gene with a bacterial operon</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568075" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568075" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sun, L.; David, K. T.; Wolters, J. F.; Karlen, S. D.; Goncalves, C.; Opulente, D. A.; LaBella, A. L.; Groenewald, M.; Zhou, X.; Shen, X.-X.; Rokas, A.; Hittinger, C. T.</p>
        <p class="info">Score: 13.8, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568075' target='https://doi.org/10.1101/2023.11.21.568075'> 10.1101/2023.11.21.568075</a></p>
        <p class="abstract">Siderophores are crucial for iron-scavenging in microorganisms. While many yeasts can uptake siderophores produced by other organisms, they are typically unable to synthesize siderophores themselves. In contrast, Wickerhamiella/Starmerella (W/S) clade yeasts gained the capacity to make the siderophore enterobactin following the remarkable horizontal acquisition of a bacterial operon enabling enterobactin synthesis. Yet, how these yeasts absorb the iron bound by enterobactin remains unresolved. Here, we demonstrate that Enb1 is the key enterobactin importer in the W/S-clade species Starmerella bombicola. Through phylogenomic analyses, we show that ENB1 is present in all W/S clade yeast species that retained the enterobactin biosynthetic genes. Conversely, it is absent in species that lost the ent genes, except for Starmerella stellata, making this species the only cheater in the W/S clade that can utilize enterobactin without producing it. Through phylogenetic analyses, we infer that ENB1 is a fungal gene that likely existed in the W/S clade prior to the acquisition of the ent genes and subsequently experienced multiple gene losses and duplications. Through phylogenetic topology tests, we show that ENB1 likely underwent horizontal gene transfer from an ancient W/S clade yeast to the order Saccharomycetales, which includes the model yeast Saccharomyces cerevisiae, followed by extensive secondary losses. Taken together, these results suggest that the fungal ENB1 and bacterial ent genes were cooperatively integrated into a functional unit within the W/S clade that enabled adaptation to iron-limited environments. This integrated fungal-bacterial circuit and its dynamic evolution determines the extant distribution of yeast enterobactin producers and cheaters.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567300">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567300" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567300">
        <p class="paperTitle">Promoter capture drives the emergence of proto-genes in Escherichia coli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567300" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567300" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: uz-Zaman, M. H.; D&#39;Alton, S.; Barrick, J. E.; Ochman, H.</p>
        <p class="info">Score: 14.3, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567300' target='https://doi.org/10.1101/2023.11.15.567300'> 10.1101/2023.11.15.567300</a></p>
        <p class="abstract">The phenomenon of de novo gene birth--the emergence of genes from non-genic sequences--has received considerable attention due to the widespread occurrence of genes that are unique to particular species or genomes. Most instances of de novo gene birth have been recognized through comparative analyses of genome sequences in eukaryotes, despite the abundance of novel, lineage-specific genes in bacteria and the relative ease with which bacteria can be studied in an experimental context. Here, we explore the genetic record of the Escherichia coli Long-Term Evolution Experiment (LTEE) for changes indicative of &#34;proto-genic&#34; phases of new gene birth in which non-genic sequences evolve stable transcription and/or translation. Over the time-span of the LTEE, non-genic regions are frequently transcribed, translated and differentially expressed, thereby serving as raw material for new gene emergence. Most proto-genes result either from insertion element activity or chromosomal translocations that fused pre-existing regulatory sequences to regions that were not expressed in the LTEE ancestor. Additionally, we identified instances of proto-gene emergence in which a previously unexpressed sequence was transcribed after formation of an upstream promoter. Tracing the origin of the causative mutations, we discovered that most occurred early in the history of the LTEE, often within the first 20,000 generations, and became fixed soon after emergence. Our findings show that proto-genes emerge frequently within evolving populations, persist stably, and can serve as potential substrates for new gene formation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564421">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564421" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564421">
        <p class="paperTitle">Frequent transitions from night-to-day activity after mass extinctions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564421" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564421" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shafer, M. E. R.; Nichols, A. L. A.; Schier, A. F.; Salzburger, W.</p>
        <p class="info">Score: 25.2, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564421' target='https://doi.org/10.1101/2023.10.27.564421'> 10.1101/2023.10.27.564421</a></p>
        <p class="abstract">Why certain species survive large-scale extinction events while others do not is poorly understood. While the fossil record can provide insights into morphological adaptations that increase survival probabilities, it provides limited information regarding behavioural traits. Here we integrate behavioural data with phylogenetic comparative models to study the evolution of day-night activity patterns (nocturnality and diurnality) in bony fishes, which have persisted through the last four mass extinction events. Our findings in fish, combined with data across all other clades of vertebrates, provide four lines of evidence that nocturnality conferred an evolutionary advantage during mass extinctions, and that frequent nocturnal-to-diurnal transitions facilitated post-extinction diversification: First, phylogenetic reconstruction indicates that the last common ancestors of all vertebrates and of clades of bony vertebrates were nocturnal. Second, in the lineage of bony fishes, which contains over half of all vertebrate species, twice as many transitions between nocturnal and diurnal activity patterns have occurred compared to tetrapods. Third, within a specific ecological niche, different species exhibit distinct temporal activity patterns, suggesting widespread temporal niche partitioning. Fourth, independent bursts in night-to-day transitions followed large-scale extinction events during the last two geological eras in all four major bony vertebrate groups. These observations suggest that ancestral nocturnality and frequent transitions to diurnality helped vertebrates survive and diversify in the face of extinction events, such as those expected during the current &#34;6th mass extinction&#34; event caused by anthropogenic climate change.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568313">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568313" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568313">
        <p class="paperTitle">Gene acquisition by giant transposons primes eukaryotes for rapid evolution via horizontal gene transfer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568313" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568313" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Urquhart, A. S.; Gluck-Thaler, E.; Vogan, A. A.</p>
        <p class="info">Score: 11.5, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568313' target='https://doi.org/10.1101/2023.11.22.568313'> 10.1101/2023.11.22.568313</a></p>
        <p class="abstract">Horizontal gene transfer (HGT) disseminates genetic information between species. The significance of HGT in eukaryotes is not well established, with evidence currently limited to isolated examples, typically absent of a mechanism. It has been proposed that mobile elements might be active agents of HGT in eukaryotes, as they are in prokaryotes. To test this hypothesis, we examined a gene cluster, which putatively contributes to formaldehyde resistance and is found within some members of the Starship superfamily of giant transposons. Remarkably, we found four instances where ssf has been independently acquired by distantly related Starships, and show that each acquisition event coincided with the element&#39;s horizontal transfer (at least 9 HGT events in total). Our results demonstrate that acquisition of host-beneficial cargo by Starships primes fungi for rapid and repeated adaptation via HGT, revealing and elevating the role of HGT in eukaryotic biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567878">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567878" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567878">
        <p class="paperTitle">Dollo parsimony overestimates ancestral gene content reconstructions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567878" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567878" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Morante, A. G.; Gueguen, L.; Natsidis, P.; Telford, M. J.; Richter, D. J.</p>
        <p class="info">Score: 10.0, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567878' target='https://doi.org/10.1101/2023.11.20.567878'> 10.1101/2023.11.20.567878</a></p>
        <p class="abstract">Ancestral reconstruction is a widely-used technique that has been applied to understand the history of gain and loss of gene families over evolutionary time scales, and to produce hypotheses on how these gains and losses may have influenced the evolutionary trajectories of extant organisms.

Ancestral gene content can be reconstructed via different phylogenetic methods, such as maximum likelihood or Bayesian inference, but many current and previous studies employ Dollo parsimony. We hypothesize that Dollo parsimony is not appropriate for ancestral gene content reconstruction inferences based on sequence homology, as Dollo parsimony is derived from the assumption that a complex character can only be gained once in evolutionary history. This premise does not accurately model molecular sequence evolution, in which false orthology can result from sequence convergence (including random sequence similarity and parallel gene gains), non-orthologous homology or lateral gene transfer. The aim of this study is to test Dollo parsimonys suitability for ancestral gene content reconstruction and to compare its inferences with a maximum likelihood-based approach which allows a gene family to be gained more than once within a tree.

In order to test our hypothesis, we first compared the performance of the two approaches on a series of artificial datasets each of 5,000 genes that were simulated according to a spectrum of evolutionary rates. Next, we reconstructed protein domain evolution on a phylogeny representing known eukaryotic diversity. We observed that Dollo parsimony produced numerous ancestral gene content overestimations, which were more pronounced at nodes closer to the root of the tree. These observations led us to the conclusion that, confirming our hypothesis, Dollo parsimony is not an appropriate method for ancestral reconstruction studies based on sequence homology.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
